{
 "cells": [
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "<style>\n",
    ".tit { \n",
    "    text-align: left; \n",
    "    font-family: Arial Black; \n",
    "    font-weight: bold; \n",
    "    font-size: 50px; \n",
    "    color: #06cc70; \n",
    "    text-shadow: -1px 0 #000, 0 1px #000, 3px 0 #000, 0 -1px #000;\n",
    "}\n",
    "</style>\n",
    "\n",
    "<h1 class=\"tit\">Data collection</h1>"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "\n",
    "The data collection process is a set of techniques and methods used to collect, extract, retrieve and store useful information for a specific purpose. This process is critical for a wide range of applications, including data analytics, artificial intelligence, and machine learning."
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "#### <span style=\"color:yellow\">Load the dependencies used in the notebook</span>"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Install packages that are requirements"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "%pip install -r ../requirements.txt"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "**Import libraries**"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "# For system operations\n",
    "import os, sys\n",
    "\n",
    "# For ignoring warnings\n",
    "import warnings\n",
    "\n",
    "# For saving objects\n",
    "import pickle\n",
    "\n",
    "# For scraping\n",
    "import requests\n",
    "\n",
    "# Numpy (for arrays and matrices)\n",
    "import numpy as np\n",
    "\n",
    "# Pandas (for dataframes)\n",
    "import pandas as pd\n",
    "\n",
    "# Matplotlib (for plotting)\n",
    "import matplotlib.pyplot as plt\n",
    "\n",
    "# tqdm (for progress bars)\n",
    "from tqdm import tqdm\n",
    "\n",
    "# BeautifulSoup (for scraping)\n",
    "from bs4 import BeautifulSoup\n",
    "\n",
    "# SQLAlchemy\n",
    "from sqlalchemy import create_engine\n",
    "from sqlalchemy.orm import sessionmaker\n",
    "\n",
    "# Selenium (for scraping) \n",
    "from selenium import webdriver\n",
    "from selenium.webdriver.common.by import By\n",
    "\n",
    "# Own modules\n",
    "# Add module olivia_finder to path\n",
    "sys.path.append('../')\n",
    "from olivia_finder.Util import *                                              # Utility functions                                     \n",
    "from olivia_finder.ProxyRequest import RequestHandler                         # Proxy request handler\n",
    "from olivia_finder.R.R_Package import R_Package                               # R package class                    \n",
    "from olivia_finder.R.R_ORM_Model import *                                     # R ORM module\n",
    "from olivia_finder.R.scraping.Cran_Scraper import Cran_Scraper                # CRAN scraper  \n",
    "from olivia_finder.R.scraping.Bioc_Scraper import Bioc_Scraper                # Bioconductor scraper\n"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# <ins>**1 - Using datasets**</ins>"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## **1.1 - Data from libraries.io**"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 1.1.0 - Previous notes (about libraries.io):"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "<a href=\"https://libraries.io/\">Libraries.io</a> is a platform that tracks and aggregates information about open source projects and software packages. \n",
    "\n",
    "Provides an overview of projects and packages, including information about their dependencies, development status, licenses, and supported platforms. This information can be useful for developers who are looking for specific projects and packages to include in their projects, or for those who simply want to explore the different options available. \n",
    "\n",
    "Furthermore, Libraries.io also allows users to monitor and receive notifications about changes and updates to specific projects and packages.\n",
    "\n",
    "Taking into account the material provided by <a href=\"https://libraries.io/\">libraries.io</a> as a source of information, a series of functions have been implemented to extract the data.\n",
    "Libraries.io provides us with a compressed file that contains a series of datasets in csv format, each one of them contains the data referring to a standard classification, specifically and according to the documentation:\n",
    "\n",
    "Libraries.io's data is organised into seven packages.\n",
    "\n",
    "| Term | Definition |\n",
    "| --- | --- |\n",
    "| Projects | A Libraries.io project is the definition of a package available from one of the 32 Package Managers that it supports. |\n",
    "| Versions | A Libraries.io version is a definition of an immutable published version of a Project from a package manager. Not all package managers have a concept of publishing versions, often relying directly on tags/branches from a source control. |\n",
    "| Tags | A Libraries.io tag is equivalent to a tag in git or other source control systems. |\n",
    "| Dependencies | Libraries.io dependencies belong to versions of a project, each version can have different sets of dependencies with different versions. Dependencies point at a specific version or range of versions of other projects, the resolution of that project version change over time as new versions are published and dependent on the specifics of the platform. Almost all package managers dependencies will be from the same package manager, although the extra `Dependency Platform` field exists otherwise. |\n",
    "| Repositories | A Libraries.io repository represents a publically accessible source code repository from either github.com, gitlab.com or bitbucket.org. |\n",
    "| Repository Dependencies | A Libraries.io repository dependency is where a dependency on a Project from a package manager has been specified in a manifest file, either as a manually added dependency committed by a user or listed as a generated dependency listed in a lockfile that has been automatically generated by a package manager and commited. |\n",
    "| Projects with related Repository fields | This is an alternative projects export that denormalizes a projects related source code repository inline to reduce the need to join between two data sets. |"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "We can download the data from the following url:\n",
    "\n",
    "- https://zenodo.org/record/3626071/files/libraries-1.6.0-2020-01-12.tar.gz "
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "<style>\n",
    ".green {color: green;}\n",
    "</style>\n",
    "**Packages**\n",
    "\n",
    "In relation to the previous note, for the analysis of the network of package dependencies in software repositories.\n",
    "\n",
    "We are going to focus on the information provided by the Projects file <span class=\"green\">projects-1.6.0-2020-01-12.csv</span>, where we can find information regarding the package and its metadata.\n",
    "\n",
    "It has the following structure:\n",
    "\n",
    "| Field Name | Data Type | Description |\n",
    "| --- | --- | --- |\n",
    "| ID  | integer | The unique primary key of the project in the Libraries.io database |\n",
    "| Platform | string | The name of the Package manager the project is available on. |\n",
    "| Name | string | The name of the project, unique by Platform (case sensitive). |\n",
    "| Created Timestamp | datetime | The timestamp of when Libraries.io first discovered the project. |\n",
    "| Updated Timestamp | datetime | The timestamp of when Libraries.io last saved a change to the project. |\n",
    "| Description | string | Description provided by the package manager, falling back to description from repository if empty. |\n",
    "| Keywords | string | Comma separated array of keywords if supported by package manager. |\n",
    "| Homepage URL | string | URL of webpage or repository as provided by package managers that support it. |\n",
    "| Licenses | string | Comma separated array of SPDX identifiers for licenses declared in package manager meta data or submitted manually by Libraries.io user via \"project suggestion\" feature. |\n",
    "| Repository URL | string | URL of source code repository declared in package manager metadata or submitted manually by Libraries.io user via \"project suggestion\" feature. |\n",
    "| Versions Count | integer | Number of published versions of the project found by Libraries.io. |\n",
    "| SourceRank | integer | Libraries.io defined score based on quality, popularity and community metrics. |\n",
    "| Latest Release Publish Timestamp | datetime | Time of the latest release detected by Libraries.io (ordered by semver, falling back to publish date for invalid semver). |\n",
    "| Latest Release Number | string | Version number of the latest release detected by Libraries.io (ordered by semver, falling back to publish date for invalid semver). |\n",
    "| Package Manager ID | integer | Unique ID of project from package manager API. |\n",
    "| Dependent Projects Count | integer | Number of other projects that declare the project as a dependency in one or more of their versions. |\n",
    "| Language | string | Primary programming language the project is written in, pulled from the repository if source is hosted on GitHub. |\n",
    "| Status | string | The current status of the project, as determined by Libraries.io |"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "**Dependencies**\n",
    "\n",
    "The dependencies file contains information about the dependencies of the packages, according to the documentation:\n",
    "\n",
    "- Libraries.io dependencies belong to versions of a project, each version can have different sets of dependencies with different versions. Dependencies point at a specific version or range of versions of other projects, the resolution of that project version change over time as new versions are published and dependent on the specifics of the platform.\n",
    "\n",
    "- Almost all package managers dependencies will be from the same package manager, although the extra `Dependency Platform` field exists otherwise\n",
    "\n",
    "|Field|Type|Description|\n",
    "|---|---|---|\n",
    "|ID|integer|The unique primary key of the dependency in the Libraries.io database|\n",
    "|Platform|string|The name of the package manager on which the dependency is available|\n",
    "|Project Name|string|The name of the project the dependency belongs to|\n",
    "|Project ID|integer|The project's unique primary key for this dependency in the Libraries.io database|\n",
    "|Version Number|string|The version number to which the dependency belongs|\n",
    "|Version ID|integer|The version's unique primary key for this dependency in the Libraries.io database|\n",
    "|Dependency Name|string|The name of the project that specifies the dependency|\n",
    "|Dependency Platform|string|The name of the package manager from which the project specifying the dependency is available|\n",
    "|Dependency Kind|string|The kind of dependency, often declared for the use phase, eg runtime, test, develop, compile.|\n",
    "|Optional Dependency|boolean|Is the dependency optional?|\n",
    "|Dependency Requirements|string|The version or range of versions that the dependency specifies, the resolution of that specific version is specific to the package manager.|\n",
    "|Dependency Project ID|integer|The unique project primary key for this dependency in the Libraries.io database|"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 1.1.1 - Data filtering"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "#### Extract package data from libraries.io projects csv"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Since this Notebook is focused on CRAN as a proof of concept and due to the large file size, we have isolated the CRAN data in a new dataset."
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "We generate a new csv that will contain only the CRAN data\n",
    "\n",
    "**Note:** \n",
    "\n",
    "The file ***projects-1.6.0-2020-01-12.csv*** is very large, for this reason it is not included in the repository, you can download the data from the link provided above\n",
    "\n",
    "We leave the code commented so as not to carry out this process every time the notebook is executed\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "# # Read the csv file and extract the data with column 'Platform' = 'CRAN' to a new dataframe\n",
    "data_path = '/media/dnllns/Externo/libraries_data/libraries-1.6.0-2020-01-12/projects-1.6.0-2020-01-12.csv'\n",
    "packages_df = pd.read_csv(data_path)\n",
    "packages_df = packages_df[packages_df['Platform'] == 'CRAN']\n",
    "\n",
    "# Save the new dataframe to a csv file\n",
    "out_path = 'data' + os.sep + 'datasets' + os.sep + 'R' + os.sep + 'libraries_io' + os.sep + 'cran_librariesio_packages.csv'\n",
    "packages_df.to_csv(out_path, index=False)"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Finally we visualize the generated csv"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>ID</th>\n",
       "      <th>Platform</th>\n",
       "      <th>Name</th>\n",
       "      <th>Created Timestamp</th>\n",
       "      <th>Updated Timestamp</th>\n",
       "      <th>Description</th>\n",
       "      <th>Keywords</th>\n",
       "      <th>Homepage URL</th>\n",
       "      <th>Licenses</th>\n",
       "      <th>Repository URL</th>\n",
       "      <th>...</th>\n",
       "      <th>SourceRank</th>\n",
       "      <th>Latest Release Publish Timestamp</th>\n",
       "      <th>Latest Release Number</th>\n",
       "      <th>Package Manager ID</th>\n",
       "      <th>Dependent Projects Count</th>\n",
       "      <th>Language</th>\n",
       "      <th>Status</th>\n",
       "      <th>Last synced Timestamp</th>\n",
       "      <th>Dependent Repositories Count</th>\n",
       "      <th>Repository ID</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>687281</td>\n",
       "      <td>CRAN</td>\n",
       "      <td>archivist</td>\n",
       "      <td>2015-03-02 00:48:14 UTC</td>\n",
       "      <td>2019-10-11 14:26:28 UTC</td>\n",
       "      <td>Tools for Storing, Restoring and Searching for...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://pbiecek.github.io/archivist/</td>\n",
       "      <td>GPL-2.0</td>\n",
       "      <td>https://github.com/pbiecek/archivist</td>\n",
       "      <td>...</td>\n",
       "      <td>14</td>\n",
       "      <td>2019-08-31 00:00:00 UTC</td>\n",
       "      <td>2.3.4</td>\n",
       "      <td>NaN</td>\n",
       "      <td>6</td>\n",
       "      <td>HTML</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2019-10-11 14:26:25 UTC</td>\n",
       "      <td>12</td>\n",
       "      <td>339318.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>687282</td>\n",
       "      <td>CRAN</td>\n",
       "      <td>ggthemes</td>\n",
       "      <td>2015-03-02 00:48:15 UTC</td>\n",
       "      <td>2019-05-13 09:52:57 UTC</td>\n",
       "      <td>Extra Themes, Scales and Geoms for 'ggplot2'</td>\n",
       "      <td>NaN</td>\n",
       "      <td>http://github.com/jrnold/ggthemes</td>\n",
       "      <td>GPL-2.0</td>\n",
       "      <td>https://github.com/jrnold/ggthemes</td>\n",
       "      <td>...</td>\n",
       "      <td>19</td>\n",
       "      <td>2019-05-13 00:00:00 UTC</td>\n",
       "      <td>4.2.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>64</td>\n",
       "      <td>R</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2019-05-13 09:52:55 UTC</td>\n",
       "      <td>232</td>\n",
       "      <td>339319.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows × 21 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "       ID Platform       Name        Created Timestamp  \\\n",
       "0  687281     CRAN  archivist  2015-03-02 00:48:14 UTC   \n",
       "1  687282     CRAN   ggthemes  2015-03-02 00:48:15 UTC   \n",
       "\n",
       "         Updated Timestamp                                        Description  \\\n",
       "0  2019-10-11 14:26:28 UTC  Tools for Storing, Restoring and Searching for...   \n",
       "1  2019-05-13 09:52:57 UTC       Extra Themes, Scales and Geoms for 'ggplot2'   \n",
       "\n",
       "   Keywords                          Homepage URL Licenses  \\\n",
       "0       NaN  https://pbiecek.github.io/archivist/  GPL-2.0   \n",
       "1       NaN     http://github.com/jrnold/ggthemes  GPL-2.0   \n",
       "\n",
       "                         Repository URL  ...  SourceRank  \\\n",
       "0  https://github.com/pbiecek/archivist  ...          14   \n",
       "1    https://github.com/jrnold/ggthemes  ...          19   \n",
       "\n",
       "   Latest Release Publish Timestamp Latest Release Number Package Manager ID  \\\n",
       "0           2019-08-31 00:00:00 UTC                 2.3.4                NaN   \n",
       "1           2019-05-13 00:00:00 UTC                 4.2.0                NaN   \n",
       "\n",
       "   Dependent Projects Count  Language Status    Last synced Timestamp  \\\n",
       "0                         6      HTML    NaN  2019-10-11 14:26:25 UTC   \n",
       "1                        64         R    NaN  2019-05-13 09:52:55 UTC   \n",
       "\n",
       "  Dependent Repositories Count  Repository ID  \n",
       "0                           12       339318.0  \n",
       "1                          232       339319.0  \n",
       "\n",
       "[2 rows x 21 columns]"
      ]
     },
     "execution_count": 9,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# Read the csv file generated from the previous step\n",
    "out_path = 'data' + os.sep + 'datasets' + os.sep + 'R' + os.sep + 'libraries_io' + os.sep + 'cran_librariesio_packages.csv'\n",
    "packages_df = pd.read_csv(out_path)\n",
    "packages_df.head(2)"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "We can see that in total we have 16695 packages in the CRAN csv"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Number of packages: 16695\n"
     ]
    }
   ],
   "source": [
    "# Count the number of projects in the dataframe\n",
    "print('Number of packages: {}'.format(len(packages_df)))"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "As an example, let's get the data of the A3 package"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>ID</th>\n",
       "      <th>Platform</th>\n",
       "      <th>Name</th>\n",
       "      <th>Created Timestamp</th>\n",
       "      <th>Updated Timestamp</th>\n",
       "      <th>Description</th>\n",
       "      <th>Keywords</th>\n",
       "      <th>Homepage URL</th>\n",
       "      <th>Licenses</th>\n",
       "      <th>Repository URL</th>\n",
       "      <th>...</th>\n",
       "      <th>SourceRank</th>\n",
       "      <th>Latest Release Publish Timestamp</th>\n",
       "      <th>Latest Release Number</th>\n",
       "      <th>Package Manager ID</th>\n",
       "      <th>Dependent Projects Count</th>\n",
       "      <th>Language</th>\n",
       "      <th>Status</th>\n",
       "      <th>Last synced Timestamp</th>\n",
       "      <th>Dependent Repositories Count</th>\n",
       "      <th>Repository ID</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>5143</th>\n",
       "      <td>692488</td>\n",
       "      <td>CRAN</td>\n",
       "      <td>A3</td>\n",
       "      <td>2015-03-02 01:14:48 UTC</td>\n",
       "      <td>2019-10-11 17:58:17 UTC</td>\n",
       "      <td>Accurate, Adaptable, and Accessible Error Metr...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>CNRI-Python-GPL-Compatible,CNRI-Python-GPL-Com...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>5</td>\n",
       "      <td>2015-08-16 00:00:00 UTC</td>\n",
       "      <td>1.0.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2019-10-11 17:58:17 UTC</td>\n",
       "      <td>0</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>1 rows × 21 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "          ID Platform Name        Created Timestamp        Updated Timestamp  \\\n",
       "5143  692488     CRAN   A3  2015-03-02 01:14:48 UTC  2019-10-11 17:58:17 UTC   \n",
       "\n",
       "                                            Description  Keywords  \\\n",
       "5143  Accurate, Adaptable, and Accessible Error Metr...       NaN   \n",
       "\n",
       "     Homepage URL                                           Licenses  \\\n",
       "5143          NaN  CNRI-Python-GPL-Compatible,CNRI-Python-GPL-Com...   \n",
       "\n",
       "     Repository URL  ...  SourceRank  Latest Release Publish Timestamp  \\\n",
       "5143            NaN  ...           5           2015-08-16 00:00:00 UTC   \n",
       "\n",
       "     Latest Release Number Package Manager ID  Dependent Projects Count  \\\n",
       "5143                 1.0.0                NaN                         0   \n",
       "\n",
       "      Language Status    Last synced Timestamp Dependent Repositories Count  \\\n",
       "5143       NaN    NaN  2019-10-11 17:58:17 UTC                            0   \n",
       "\n",
       "      Repository ID  \n",
       "5143            NaN  \n",
       "\n",
       "[1 rows x 21 columns]"
      ]
     },
     "execution_count": 12,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# Get the A3 package data from the df dataframe\n",
    "packages_df[packages_df['Name'] == 'A3']"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "#### Extract package-dependency data from libraries.io dependencies csv"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "As before, we are going to extract this information from the dependencies file"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>ID</th>\n",
       "      <th>Platform</th>\n",
       "      <th>Name</th>\n",
       "      <th>Created Timestamp</th>\n",
       "      <th>Updated Timestamp</th>\n",
       "      <th>Description</th>\n",
       "      <th>Keywords</th>\n",
       "      <th>Homepage URL</th>\n",
       "      <th>Licenses</th>\n",
       "      <th>Repository URL</th>\n",
       "      <th>...</th>\n",
       "      <th>SourceRank</th>\n",
       "      <th>Latest Release Publish Timestamp</th>\n",
       "      <th>Latest Release Number</th>\n",
       "      <th>Package Manager ID</th>\n",
       "      <th>Dependent Projects Count</th>\n",
       "      <th>Language</th>\n",
       "      <th>Status</th>\n",
       "      <th>Last synced Timestamp</th>\n",
       "      <th>Dependent Repositories Count</th>\n",
       "      <th>Repository ID</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>1</td>\n",
       "      <td>Alcatraz</td>\n",
       "      <td>21st digital Templates</td>\n",
       "      <td>2015-01-11 23:56:18 UTC</td>\n",
       "      <td>2020-01-13 01:02:01 UTC</td>\n",
       "      <td>A starting point for stripped down, structured...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://github.com/21stdigital/Xcode-Templates</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://github.com/21stdigital/Xcode-Templates</td>\n",
       "      <td>...</td>\n",
       "      <td>5</td>\n",
       "      <td>2020-01-13 01:01:47 UTC</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0</td>\n",
       "      <td>Objective-C</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2020-01-13 01:01:53 UTC</td>\n",
       "      <td>0</td>\n",
       "      <td>3945.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>2</td>\n",
       "      <td>Alcatraz</td>\n",
       "      <td>ACCodeSnippetRepository</td>\n",
       "      <td>2015-01-11 23:56:18 UTC</td>\n",
       "      <td>2020-01-13 01:01:44 UTC</td>\n",
       "      <td>Synchronize code snippets with a git repository.</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://github.com/acoomans/ACCodeSnippetRepos...</td>\n",
       "      <td>MIT</td>\n",
       "      <td>https://github.com/acoomans/ACCodeSnippetRepos...</td>\n",
       "      <td>...</td>\n",
       "      <td>11</td>\n",
       "      <td>2014-03-13 18:40:38 UTC</td>\n",
       "      <td>0.0.4</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0</td>\n",
       "      <td>Objective-C</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2020-01-13 01:01:44 UTC</td>\n",
       "      <td>0</td>\n",
       "      <td>3943.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows × 21 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "   ID  Platform                     Name        Created Timestamp  \\\n",
       "0   1  Alcatraz   21st digital Templates  2015-01-11 23:56:18 UTC   \n",
       "1   2  Alcatraz  ACCodeSnippetRepository  2015-01-11 23:56:18 UTC   \n",
       "\n",
       "         Updated Timestamp                                        Description  \\\n",
       "0  2020-01-13 01:02:01 UTC  A starting point for stripped down, structured...   \n",
       "1  2020-01-13 01:01:44 UTC   Synchronize code snippets with a git repository.   \n",
       "\n",
       "   Keywords                                       Homepage URL Licenses  \\\n",
       "0       NaN     https://github.com/21stdigital/Xcode-Templates      NaN   \n",
       "1       NaN  https://github.com/acoomans/ACCodeSnippetRepos...      MIT   \n",
       "\n",
       "                                      Repository URL  ...  SourceRank  \\\n",
       "0     https://github.com/21stdigital/Xcode-Templates  ...           5   \n",
       "1  https://github.com/acoomans/ACCodeSnippetRepos...  ...          11   \n",
       "\n",
       "   Latest Release Publish Timestamp Latest Release Number Package Manager ID  \\\n",
       "0           2020-01-13 01:01:47 UTC                   NaN                NaN   \n",
       "1           2014-03-13 18:40:38 UTC                 0.0.4                NaN   \n",
       "\n",
       "   Dependent Projects Count     Language Status    Last synced Timestamp  \\\n",
       "0                         0  Objective-C    NaN  2020-01-13 01:01:53 UTC   \n",
       "1                         0  Objective-C    NaN  2020-01-13 01:01:44 UTC   \n",
       "\n",
       "  Dependent Repositories Count  Repository ID  \n",
       "0                            0         3945.0  \n",
       "1                            0         3943.0  \n",
       "\n",
       "[2 rows x 21 columns]"
      ]
     },
     "execution_count": 17,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "data_path = '/media/dnllns/Externo/libraries_data/libraries-1.6.0-2020-01-12/projects-1.6.0-2020-01-12.csv'\n",
    "dependencies_df = pd.read_csv(data_path, nrows=10)\n",
    "dependencies_df.head(2)"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "This file is very large and the machine I have does not have enough RAM to open it, so the only solution is to cut it into smaller pieces.\n",
    "\n",
    "In a linux system we can use the following command to split the file into files of 10000 lines each"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [],
   "source": [
    "#!cat dependencies-1.6.0-2020-01-12.csv | parallel --header : --pipe -N10000 'cat >dependencies-1.6.0-2020-01-12/split_dependencies-1.6.0-2020-01-12_{#}.csv'"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "We can see that the file has been divided into 19040 smaller files."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "19040\n"
     ]
    }
   ],
   "source": [
    "## Count the number of files in the directory\n",
    "PATH_ = '/media/dnllns/Externo/libraries_data/libraries-1.6.0-2020-01-12/' \n",
    "!ls -l $PATH_\"dependencies-1.6.0-2020-01-12\" | wc -l"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "We process the chunks individually and load the dataframe"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "100%|██████████| 19039/19039 [1:12:52<00:00,  4.35it/s]    \n"
     ]
    }
   ],
   "source": [
    "warnings.filterwarnings(\"ignore\", category=FutureWarning)\n",
    "\n",
    "chunks_path = '/media/dnllns/Externo/libraries_data/libraries-1.6.0-2020-01-12/dependencies-1.6.0-2020-01-12/'\n",
    "files = os.listdir(chunks_path)\n",
    "\n",
    "# Process dependency files\n",
    "# If the row has a 'CRAN' value in the 'Dependency Platform' column, it is saved in a new dataframe\n",
    "dependencies_df = pd.DataFrame()\n",
    "file_count = 0\n",
    "pbar = tqdm(total=len(files))\n",
    "for file in files:\n",
    "    if file.endswith(\".csv\"):\n",
    "        file_count += 1\n",
    "        temp_dependencies_df = pd.read_csv(chunks_path + file)\n",
    "        dependencies_df = pd.concat([dependencies_df, temp_dependencies_df[temp_dependencies_df['Dependency Platform'] == 'CRAN']])\n",
    "    pbar.update(1)\n",
    "pbar.close()\n",
    "\n",
    "# Save the new dataframe to a csv file\n",
    "out_path = 'data' + os.sep + 'datasets' + os.sep + 'R' + os.sep + 'libraries_io' + os.sep + 'cran_librariesio_dependencies.csv'\n",
    "dependencies_df.to_csv(out_path, index=False)"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Once all the files have been processed we have a csv that contains the dependencies of the CRAN packages.\n",
    "\n",
    "\n",
    "We will show the dependencies of the A3 package of type imports and depends for version 1.0.0"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>ID</th>\n",
       "      <th>Platform</th>\n",
       "      <th>Project Name</th>\n",
       "      <th>Project ID</th>\n",
       "      <th>Version Number</th>\n",
       "      <th>Version ID</th>\n",
       "      <th>Dependency Name</th>\n",
       "      <th>Dependency Platform</th>\n",
       "      <th>Dependency Kind</th>\n",
       "      <th>Optional Dependency</th>\n",
       "      <th>Dependency Requirements</th>\n",
       "      <th>Dependency Project ID</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>66065047</td>\n",
       "      <td>CRAN</td>\n",
       "      <td>nmfem</td>\n",
       "      <td>2524222</td>\n",
       "      <td>1.0.0</td>\n",
       "      <td>11273453</td>\n",
       "      <td>R</td>\n",
       "      <td>CRAN</td>\n",
       "      <td>depends</td>\n",
       "      <td>False</td>\n",
       "      <td>*</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>66065056</td>\n",
       "      <td>CRAN</td>\n",
       "      <td>nmfem</td>\n",
       "      <td>2524222</td>\n",
       "      <td>1.0.0</td>\n",
       "      <td>11273453</td>\n",
       "      <td>plyr</td>\n",
       "      <td>CRAN</td>\n",
       "      <td>imports</td>\n",
       "      <td>False</td>\n",
       "      <td>*</td>\n",
       "      <td>691054.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "         ID Platform Project Name  Project ID Version Number  Version ID  \\\n",
       "0  66065047     CRAN        nmfem     2524222          1.0.0    11273453   \n",
       "1  66065056     CRAN        nmfem     2524222          1.0.0    11273453   \n",
       "\n",
       "  Dependency Name Dependency Platform Dependency Kind  Optional Dependency  \\\n",
       "0               R                CRAN         depends                False   \n",
       "1            plyr                CRAN         imports                False   \n",
       "\n",
       "  Dependency Requirements  Dependency Project ID  \n",
       "0                       *                    NaN  \n",
       "1                       *               691054.0  "
      ]
     },
     "execution_count": 19,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "data_path = 'data' + os.sep + 'datasets' + os.sep + 'R' + os.sep + 'libraries_io' + os.sep + 'cran_librariesio_dependencies.csv'\n",
    "dependencies_df = pd.read_csv(data_path)\n",
    "dependencies_df.head(2)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "#### Visualize the data"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Total packages:  16695\n",
      "Total packages with dependencies :  16174\n",
      "Total packages that are dependencies of others:  5501\n",
      "Total packages without dependencies:  1173\n"
     ]
    }
   ],
   "source": [
    "dependency_packages = set(dependencies_df['Dependency Name'].values)\n",
    "packages_with_dependencies = set(dependencies_df['Project Name'].values)\n",
    "total_packages = set(packages_df['Name'].values)\n",
    "packages_without_dependencies = total_packages - packages_with_dependencies\n",
    "print('Total packages: ', len(total_packages))\n",
    "print('Total packages with dependencies : ', len(dependency_packages | packages_with_dependencies))\n",
    "print('Total packages that are dependencies of others: ', len(dependency_packages))\n",
    "print('Total packages without dependencies: ', len(packages_without_dependencies))"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "We build a histogram to represent the analyzed information"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "image/png": "iVBORw0KGgoAAAANSUhEUgAAAxYAAAGGCAYAAADmRxfNAAAAOXRFWHRTb2Z0d2FyZQBNYXRwbG90bGliIHZlcnNpb24zLjMuNCwgaHR0cHM6Ly9tYXRwbG90bGliLm9yZy8QVMy6AAAACXBIWXMAAA9hAAAPYQGoP6dpAACEgElEQVR4nOzde3zP9f//8dt757F5Y2yzmlPEZnNcmLMcJkaoyLRyiD4fyrlJJCLnUlFCTqHW71PxiWosZ81pywgjRKKtKWsYdnz//vDd6+Ntw9bGbN2vl8suvJ+v5+v1er5e79fr/X493s+TyWKxWBARERERESkAm6IugIiIiIiIFH8KLEREREREpMAUWIiIiIiISIEpsBARERERkQJTYCEiIiIiIgWmwEJERERERApMgYWIiIiIiBSYAgsRERERESkwBRYiIiIiIlJgCixEROS2li9fjslkMv7s7Ox48MEH6d+/P+fOnSv0/bVp0wY/P79C366IiNxddkVdABERKR6WLVtG7dq1uXr1Ktu3b2f69Ols27aNH3/8kdKlSxd18UREpIgpsBARkTzx8/MjICAAgLZt25KZmcmUKVNYu3Ytffv2LeLSiYhIUVNTKBER+VuaNm0KwC+//MLkyZNp0qQJ5cuXp0yZMjRs2JAlS5ZgsVhyrPfJJ58QGBiIi4sLLi4u1K9fnyVLltx2X2vWrKFUqVI8//zzZGRkcO3aNUaPHk39+vUxm82UL1+ewMBA/vvf/+ZY96+//mLgwIGUL18eFxcXunTpws8//4zJZGLSpElWeY8fP05ISAju7u44Ojri4+PD+++/b5UnKyuLqVOnUqtWLZydnSlbtix169bl3XffzecZFBEpWVRjISIif8uJEycAqFixIlFRUbzwwgtUrlwZgN27d/PSSy9x7tw5Jk6caKwzceJEpkyZQs+ePRk9ejRms5lDhw7xyy+/3HI/c+fO5eWXX2bSpElMmDABgJSUFC5cuMCYMWN44IEHSEtL47vvvqNnz54sW7aMZ599FrgeBHTt2pXo6GgmTZpEw4YN2bVrF506dcqxnyNHjtCsWTMqV67MW2+9haenJxs2bGDYsGH88ccfvP766wDMmjXLKEurVq1IT0/n6NGj/PXXX4VyXkVEii2LiIjIbSxbtswCWHbv3m1JT0+3XLp0ybJ+/XpLxYoVLa6urpaEhASr/JmZmZb09HTLG2+8YXFzc7NkZWVZLBaL5eeff7bY2tpa+vbte9v9tW7d2lKnTh1LZmam5cUXX7Q4ODhYVq1addt1MjIyLOnp6ZaBAwdaGjRoYKR//fXXFsCyYMECq/zTp0+3AJbXX3/dSAsKCrI8+OCDluTkZKu8L774osXJycly4cIFi8VisQQHB1vq169/2/KIiPwTqSmUiIjkSdOmTbG3t8fV1ZXg4GA8PT359ttv8fDwYPPmzbRv3x6z2YytrS329vZMnDiRP//8k8TERAAiIyPJzMxk6NChd9zXtWvX6N69O6tXr2bjxo259uH4z3/+Q/PmzXFxccHOzg57e3uWLFlCXFyckWfbtm0A9OrVy2rdPn365Njfpk2b6NGjB6VKlSIjI8P469y5M9euXWP37t0ANG7cmAMHDjBkyBA2bNjAxYsX83ciRURKKAUWIiKSJx9//DH79u1j//79/Pbbbxw8eJDmzZuzd+9eOnbsCMDixYv5/vvv2bdvH+PHjwfg6tWrAJw/fx6ABx988I77SkxMZMOGDQQGBtKsWbMcy7/88kt69erFAw88wKpVq9i1axf79u1jwIABXLt2zcj3559/YmdnR/ny5a3W9/DwsHr9559/kpGRwbx587C3t7f669y5MwB//PEHAOPGjWPOnDns3r2bxx57DDc3N9q1a0d0dHSezqOISEmlPhYiIpInPj4+xqhQNwoPD8fe3p7169fj5ORkpK9du9YqX8WKFQE4e/Ys3t7et91X5cqVefvtt+nRowc9e/bkP//5j9W2V61aRbVq1fjss88wmUxGempqqtV23NzcyMjI4MKFC1bBRUJCglW+cuXKYWtrS2ho6C1rVKpVqwaAnZ0do0aNYtSoUfz111989913vPrqqwQFBfHrr79SqlSp2x6biEhJpRoLEREpkOxJ82xtbY20q1evsnLlSqt8HTt2xNbWlgULFuRpux07dmTDhg1s376d4OBgUlJSrPbp4OBgFVQkJCTkGBWqdevWAHz22WdW6eHh4VavS5UqRdu2bdm/fz9169YlICAgx5+bm1uOMpYtW5Ynn3ySoUOHcuHCBU6fPp2nYxMRKYlUYyEiIgXSpUsX3n77bUJCQhg8eDB//vknc+bMwdHR0Spf1apVefXVV5kyZQpXr16lT58+mM1mjhw5wh9//MHkyZNzbLtFixZs2rSJTp060bFjR7755hvMZjPBwcF8+eWXDBkyhCeffJJff/2VKVOmUKlSJY4fP26s36lTJ5o3b87o0aO5ePEijRo1YteuXXz88ccA2Nj87/e1d999lxYtWtCyZUv+/e9/U7VqVS5dusSJEydYt24dmzdvBqBr167GnB4VK1bkl19+4Z133qFKlSrUrFnzbpxiEZFiQYGFiIgUyKOPPsrSpUuZOXMmXbt25YEHHmDQoEG4u7szcOBAq7xvvPEGNWvWZN68efTt2xc7Oztq1qzJsGHDbrn9gIAAtm3bRvv27Xn00UfZsGED/fv3JzExkQ8//JClS5dSvXp1XnnlFc6ePWsVoNjY2LBu3TpGjx7NjBkzSEtLo3nz5qxatYqmTZtStmxZI6+vry8//PADU6ZMYcKECSQmJlK2bFlq1qxp9LOA65MDfvHFF3z00UdcvHgRT09POnTowGuvvYa9vX3hnVgRkWLGZLHkMnuRiIhICfbJJ5/Qt29fvv/++1w7h4uISP4psBARkRLt008/5dy5c/j7+2NjY8Pu3buZPXs2DRo0MIajFRGRglNTKBERKdFcXV0JDw9n6tSppKSkUKlSJfr168fUqVOLumgiIiWKaixERERERKTANNysiIiIiIgUmAILEREREREpMAUWIiIiIiJSYOq8XYiysrL47bffcHV1tZoNVkRERESkOLJYLFy6dAkvLy+rSUVzo8CiEP322294e3sXdTFERERERArVr7/+yoMPPnjbPAosCpGrqytw/cSXKVOmiEsjIiIiIlIwFy9exNvb23jOvR0FFoUou/lTmTJlFFiIiIiISImRl2b+6rwtIiIiIiIFpsBCREREREQKTIGFiIiIiIgUmPpYiIhIsZSVlUVaWlpRF0NEpFizt7fH1ta2ULalwEJERIqdtLQ0Tp06RVZWVlEXRUSk2Ctbtiyenp4FnodNgYWIiBQrFouF+Ph4bG1t8fb2vuOETSIikjuLxcKVK1dITEwEoFKlSgXangILEREpVjIyMrhy5QpeXl6UKlWqqIsjIlKsOTs7A5CYmIi7u3uBmkXpZx4RESlWMjMzAXBwcCjikoiIlAzZP9Kkp6cXaDsKLEREpFgqaFtgERG5rrA+TxVYiIiIiIhIgSmwEBERKaGqVq3KO++8U6RlmDRpEvXr1y/SMojIvaHO2yIiUiIEBNzb/UVH5z3vnZoZPPfccyxfvvy2669Zs4bu3bvnfaciIveYAgsREZG7LD4+3vj/Z599xsSJEzl27JiRlj0qi4hIcabAogS517/W3U35+SVQROR+5+npafzfbDZjMpms0hYsWMCcOXP49ddfqVatGhMmTCA0NBS43pwJoEePHgBUqVKF06dPc/LkSUaNGsXu3btJSUnBx8eH6dOn0759+zyXq1+/fvz11180aNCA999/n2vXrtGnTx/mzZtnjLoVERHB1KlTOXToELa2tgQGBvLuu+/y0EMPGds5e/YsY8aMYePGjaSmpuLj48P7779PkyZNcuzz1KlTdOjQgQ4dOvD+++8TExPDq6++yv79+0lPT6d+/frMnTuXhg0bGuscPXqU559/nujoaKpXr857771Hhw4drGpxzp07x6hRo9i4cSM2Nja0aNGCd9991zh/W7duJSwsjMOHD2Nvb0+dOnX45JNPqFKlSp7Pl4jcnvpYiIiIFKE1a9YwfPhwRo8ezaFDh3jhhRfo378/W7ZsAWDfvn0ALFu2jPj4eOP15cuX6dy5M9999x379+8nKCiIrl27cubMmXztf9OmTcTFxbFlyxY+/fRT1qxZw+TJk43lKSkpjBo1in379rFp0yZsbGzo0aOHMev55cuXad26Nb/99htfffUVBw4cICwsLNdZ0Q8dOkTz5s156qmnWLBgATY2Nly6dInnnnuOHTt2sHv3bmrWrEnnzp25dOkSAFlZWXTv3p1SpUqxZ88eFi1axPjx4622e+XKFdq2bYuLiwvbt29n586duLi40KlTJ9LS0sjIyKB79+60bt2agwcPsmvXLgYPHqyRxUQKmWosREREitCcOXPo168fQ4YMATBqIebMmUPbtm2pWLEiAGXLlrWq5ahXrx716tUzXk+dOpU1a9bw1Vdf8eKLL+Z5/w4ODixdupRSpUpRp04d3njjDV5++WWmTJmCjY0NTzzxhFX+JUuW4O7uzpEjR/Dz8+OTTz7h/Pnz7Nu3j/LlywNQo0aNHPvZtWsXwcHBjBs3jjFjxhjpjz76qFW+hQsXUq5cObZt20ZwcDAbN27k5MmTbN261Tj+N998kw4dOhjrhIeHY2Njw0cffWQEC8uWLaNs2bJs3bqVgIAAkpOTCQ4ONmpafHx88nyORCRvVGMhIiJShOLi4mjevLlVWvPmzYmLi7vteikpKYSFheHr60vZsmVxcXHh6NGj+a6xqFevntUM5oGBgVy+fJlff/0VgJMnTxISEkL16tUpU6YM1apVAzD2ExsbS4MGDYygIjdnzpyhffv2TJgwwSqogOuz/f7rX//i4Ycfxmw2YzabuXz5srH9Y8eO4e3tbRVUNW7c2GobMTExnDhxAldXV1xcXHBxcaF8+fJcu3aNkydPUr58efr162fU6rz77rtW/V5EpHCoxkJERKSI3dwkx2Kx3LGZzssvv8yGDRuYM2cONWrUwNnZmSeffJK0tLRCLVPXrl3x9vZm8eLFeHl5kZWVhZ+fn7GfvHQ8r1ixIl5eXoSHhzNw4EDKlCljLOvXrx/nz5/nnXfeoUqVKjg6OhIYGGhsPy/nIisri0aNGrF69epc9w3XazCGDRtGREQEn332GRMmTCAyMpKmTZvm7YSIyB2pxkJERKQI+fj4sHPnTqu0qKgoq6Y69vb2ZGZmWuXZsWMH/fr1o0ePHvj7++Pp6cnp06fzvf8DBw5w9epV4/Xu3btxcXHhwQcf5M8//yQuLo4JEybQrl07fHx8SEpKslq/bt26xMbGcuHChVvuw9nZmfXr1+Pk5ERQUJDRfyL7OIYNG0bnzp2pU6cOjo6O/PHHH8by2rVrc+bMGX7//XcjLbufSbaGDRty/Phx3N3dqVGjhtWf2Ww28jVo0IBx48YRFRVlNOMSkcKjwEJERKQIvfzyyyxfvpwPP/yQ48eP8/bbb/Pll19aNRmqWrUqmzZtIiEhwXiwr1GjBl9++SWxsbEcOHCAkJCQXDtM30laWhoDBw7kyJEjfPvtt7z++uu8+OKL2NjYUK5cOdzc3Fi0aBEnTpxg8+bNjBo1ymr9Pn364OnpSffu3fn+++/5+eef+eKLL9i1a5dVvtKlS/P1119jZ2fHY489xuXLl43jWLlyJXFxcezZs4e+ffta1YJ06NCBhx56iOeee46DBw/y/fffG523s2sy+vbtS4UKFXj88cfZsWMHp06dYtu2bQwfPpyzZ89y6tQpxo0bx65du/jll1/YuHEjP/30k/pZiBQyBRYiIiJFqHv37rz77rvMnj2bOnXqsHDhQpYtW0abNm2MPG+99RaRkZF4e3vToEEDAObOnUu5cuVo1qwZXbt2JSgoyGqI1rxq164dNWvWpFWrVvTq1YuuXbsyadIkAGxsbAgPDycmJgY/Pz9GjhzJ7NmzrdZ3cHBg48aNuLu707lzZ/z9/ZkxYwa2trY59uXi4sK3336LxWKhc+fOpKSksHTpUpKSkmjQoAGhoaEMGzYMd3d3Yx1bW1vWrl3L5cuXeeSRR3j++eeZMGECAE5OTgCUKlWK7du3U7lyZXr27ImPjw8DBgzg6tWrlClThlKlSnH06FGeeOIJHn74YQYPHsyLL77ICy+8kO/zJSK3ZrJYLJai2vn27duZPXs2MTExxMfH5zqraFxcHGPHjmXbtm1kZWVRp04d/t//+39UrlwZgNTUVMaMGcOnn37K1atXadeuHR988AEPPvigsY2kpCSGDRvGV199BUC3bt2YN28eZcuWNfKcOXOGoUOHsnnzZpydnQkJCWHOnDnGON55cfHiRcxmM8nJyVbtR+8VzWMhIv8E165d49SpU1SrVs14sJS/J3sei7Vr1xZ1UfLl+++/p0WLFpw4ccJqPg0R+Xtu97man+fbIu28nZKSQr169ejfv3+O4ezg+kgULVq0YODAgUyePBmz2UxcXJzVAY8YMYJ169YRHh6Om5sbo0ePJjg4mJiYGOPXkpCQEM6ePUtERAQAgwcPJjQ0lHXr1gGQmZlJly5dqFixIjt37uTPP//kueeew2KxMG/evHtwJkSkMAUsKhlRdvRgRdgicH2uDxcXF2rWrMmJEycYPnw4zZs3V1Ahcp8p0sDiscce47HHHrvl8vHjx9O5c2dmzZplpFWvXt34f3JyMkuWLGHlypXGTKOrVq3C29ub7777jqCgIOLi4oiIiGD37t3GDKCLFy8mMDCQY8eOUatWLTZu3MiRI0f49ddf8fLyAq5XO/fr148333yzSGofRERE5LpLly4RFhbGr7/+SoUKFWjfvj1vvfVWURdLRG5y3w43m5WVxddff01YWBhBQUHs37+fatWqMW7cOKO5VExMDOnp6XTs2NFYz8vLCz8/P6KioggKCmLXrl2YzWYjqABo2rQpZrOZqKgoatWqxa5du/Dz8zOCCoCgoCBSU1OJiYmhbdu2uZYxNTWV1NRU4/XFixcByMjIICMjozBPR57Y3bfvZv4VwemTEsTu/v1oy5ei+BwpDjIyMrBYLMaf/H3Lli0DuO/PY2hoKKGhoTnS7/dyixQX2Z+nuT3D5ue76L799k1MTOTy5cvMmDGDqVOnMnPmTCIiIujZsydbtmyhdevWJCQk4ODgQLly5azW9fDwICEhAYCEhASrTmDZ3N3drfJ4eHhYLS9XrhwODg5GntxMnz6dyZMn50iPjo6mdOnS+T7mgurT557v8q7Zs6eoSyDFWZ9yJeNm2KMbIVcmk4lSpUpx5cqVHEOwiohI/qWmppKWlsbBgwdzBOwpKSl53s59G1hkD5n3+OOPM3LkSADq169PVFQUH374Ia1bt77lujdPppPbxDp/J8/Nxo0bZzXs3sWLF/H29iYgIKBImk+NHn3Pd3nXDB1a1CWQ4mz00pJxMwztoRshN9euXePMmTOUKlVKnbdFRAqBra0tDg4O1KhRI9fO23l13wYWFSpUwM7ODl9fX6v0GycS8vT0JC0tjaSkJKtai8TERJo1a2bkuXFSnWznz583aik8PT1z/DKYlJREenp6jpqMGzk6OuLo6Jgj3c7ODrsiaJdUklpNlKRmXXLvZVAyboai+BwpDuzs7DCZTMafiIgUTPbnaW7PsPn5Lrpv57FwcHDgkUce4dixY1bpP/30E1WqVAGgUaNG2NvbExkZaSyPj4/n0KFDRmARGBhIcnIye/fuNfLs2bOH5ORkqzyHDh0iPj7eyLNx40YcHR1p1KjRXTtGEREREZGSokh/Drt8+TInTpwwXp86dYrY2FjKly9P5cqVefnll+nduzetWrWibdu2REREsG7dOrZu3QqA2Wxm4MCBjB49Gjc3N8qXL8+YMWPw9/c3Rony8fGhU6dODBo0iIULFwLXh5sNDg6mVq1aAHTs2BFfX19CQ0OZPXs2Fy5cYMyYMQwaNEgjQomIiIiI5EGR1lhER0fToEEDYxbRUaNG0aBBAyZOnAhAjx49+PDDD5k1axb+/v589NFHfPHFF7Ro0cLYxty5c+nevTu9evWiefPmlCpVinXr1lnN+Ll69Wr8/f3p2LEjHTt2pG7duqxcudJYbmtry9dff42TkxPNmzenV69edO/enTlz5tyjMyEiIiIiUrwV6czbJY1m3i48mnlbCkIT5JVsmnk7d8V1Fu38qFq1KiNGjGDEiBG3zDNp0iTWrl1LbGxsgfa1fPlyRowYwV9//VWg7dwreTk3hen06dNUq1aN/fv3U79+/XuyT7l7SsTM2yIiIoXlXgeU+Q38+vXrx4oVK4DrnSG9vb3p2bMnkydPLpIhyoujffv2WZ0rk8nEmjVrjPmt5N7x9vYmPj6eChUqFHVR5D6iwEJEROQe6dSpE8uWLSM9PZ0dO3bw/PPPk5KSwoIFC4q6aMVCxYoVi7oI8n9sbW3x9PQs6mLIfea+HRVKRESkpHF0dMTT0xNvb29CQkLo27ev0XRp1apVBAQE4OrqiqenJyEhISQmJlqtf/jwYbp06UKZMmVwdXWlZcuWnDx5Mtd9xcTE4O7uzptvvglAREQELVq0oGzZsri5uREcHJxj3aioKOrXr4+TkxMBAQGsXbsWk8lk1azoyJEjdO7cGRcXFzw8PAgNDeWPP/4wln/++ef4+/vj7OyMm5sb7du3v+UEW40aNeKtt94yXnfv3h07Oztj3PyEhARMJpMxQmTVqlV55513jP/D9f6YJpPJeJ1t5cqVVK1aFbPZzNNPP82lS5dyLUO25cuXU7lyZUqVKkWPHj34888/c+RZt24djRo1wsnJierVqzN58mSrWYlNJhMLFizgsccew9nZmWrVqvGf//zHahvnzp2jd+/elCtXDjc3Nx5//HFOnz5tLO/Xr5/Rz7NSpUq4ubkxdOhQ0tPTjTyJiYl07drV2Mfq1atzlDU5OZnBgwfj7u5OmTJlePTRRzlw4ICxfNKkSdSvX/+25ykrK4uZM2dSo0YNHB0dqVy5snE9nT59+q5eG1I8KbAQEREpIs7OzsYDY1paGlOmTOHAgQOsXbuWU6dO0a9fPyPvuXPnaNWqFU5OTmzevJmYmBgGDBhg9WCbbevWrbRr147Jkyczfvx44PrsuaNGjWLfvn1s2rQJGxsbevToYUxIe+nSJbp27Yq/vz8//PADU6ZMYezYsVbbjY+Pp3Xr1tSvX5/o6GgiIiL4/fff6dWrl7G8T58+DBgwgLi4OLZu3UrPnj1zzOSbrU2bNsZIjxaLhR07dlCuXDljvqotW7bg6elpjOJ4o3379gGwbNky4uPjjdcAJ0+eZO3ataxfv57169ezbds2ZsyYccv3Yc+ePQwYMIAhQ4YQGxtL27ZtmTp1qlWeDRs28MwzzzBs2DCOHDnCwoULWb58ufGgne21117jiSee4MCBAzzzzDP06dOHuLg4AK5cuULbtm1xcXFh+/bt7Ny5ExcXFzp16kRaWpqxjS1btnDy5Em2bNnCihUrWL58OcuXLzeW9+vXj9OnT7N582Y+//xzPvjgA6sg1GKx0KVLFxISEvjmm2+IiYmhYcOGtGvXjgsXLuT5PI0bN46ZM2fy2muvceTIET755JNbzu9V2NeGFE9qCiUiIlIE9u7dyyeffEK7du0AGDBggLGsevXqvPfeezRu3JjLly/j4uLC+++/j9lsJjw8HHt7ewAefvjhHNv973//S2hoKAsXLqRPnz5G+hNPPGGVb8mSJbi7u3PkyBH8/PxYvXo1JpOJxYsX4+TkhK+vL+fOnWPQoEHGOgsWLKBhw4ZMmzbNSFu6dCne3t789NNPXL58mYyMDHr27GnMOeXv73/Lc9CmTRuWLFlCVlYWP/74I7a2tjzzzDNs3bqVzp07s3XrVlq3bp3rutnNosqWLZujSU5WVhbLly/H1dUVgNDQUDZt2pQjCMj27rvvEhQUxCuvvAJcP69RUVFEREQYed58801eeeUVnnvuOeD6ezRlyhTCwsJ4/fXXjXxPPfUUzz//PABTpkwhMjKSefPm8cEHHxAeHo6NjQ0fffSRMbnjsmXLKFu2LFu3bqVjx44AlCtXjvnz52Nra0vt2rXp0qULmzZtYtCgQfz00098++237N69myZNmhjvpY+Pj1GGLVu28OOPP5KYmGhM5DtnzhzWrl3L559/zuDBg+94ni5dusS7777L/PnzjWN+6KGHrEbmvFFhXxtSPKnGQkRE5B5Zv349Li4uODk5ERgYSKtWrZg3bx4A+/fv5/HHH6dKlSq4urrSpk0bAM6cOQNAbGwsLVu2NIKK3OzZs4cnnniCFStWWAUVcP3X6ZCQEKpXr06ZMmWoVq2a1faPHTtG3bp1rUaEady4sdU2YmJi2LJlCy4uLsZf7dq1je3Xq1ePdu3a4e/vz1NPPcXixYtJSkq6ZXlbtWrFpUuX2L9/P9u2baN169a0bduWbdu2Adw2sLidqlWrGg/LAJUqVcrRrOxGcXFxBAYGWqXd/DomJoY33njD6tgHDRpEfHw8V65cueV6gYGBRo1FTEwMJ06cwNXV1dhG+fLluXbtmlWztDp16lgNm39j+ePi4rCzsyPghqEga9euTdmyZa3KevnyZdzc3KzKe+rUKav93O48xcXFkZqaagS+d1LY14YUT6qxEBERuUfatm3LggULsLe3x8vLywgSUlJSjLmWVq1aRcWKFTlz5gxBQUFGExlnZ+c7bv+hhx7Czc2NpUuX0qVLFxwcHIxlXbt2xdvbm8WLF+Pl5UVWVhZ+fn7G9i0Wi/Ererabm6lkZWXRtWtXZs6cmWPflSpVwtbWlsjISKKioti4cSPz5s1j/Pjx7NmzxwhkbmQ2m6lfvz5bt24lKiqKRx99lJYtWxIbG8vx48f56aefjAArP24Ovkwmk9HkKzd5aY6TlZXF5MmT6dmzZ45ldxr2OPu8ZmVl0ahRo1z7RNzYMf125c8u683v1c1lrVSpktHM7EY3BiC3209erreb91mY14YUT6qxEBERuUdKly5NjRo1qFKlitVD3dGjR/njjz+YMWMGLVu2pHbt2jl+Ya9bty47duyw6sR7swoVKrB582ZOnjxJ7969jbx//vkncXFxTJgwgXbt2uHj45Pj1+LatWtz8OBBUlNTjbTomyYVatiwIYcPH6Zq1arUqFHD6i97GFiTyUTz5s2ZPHky+/fvx8HBgTVr1tyyzG3atGHLli1s376dNm3aULZsWXx9fZk6dSru7u5WTXxuZm9vT2Zm5i2X55Wvry+7d++2Srv5dcOGDTl27FiO465RowY2Nja3XG/37t3GL/cNGzbk+PHjuLu759iG2WzOU1l9fHzIyMiwem+OHTtmNd9Gw4YNSUhIwM7OLsd+8jo8bM2aNXF2dmbTpk15yn83rg0pfhRYiIiIFLHKlSvj4ODAvHnz+Pnnn/nqq6+YMmWKVZ4XX3yRixcv8vTTTxMdHc3x48dZuXKlMWJSNnd3dzZv3szRo0fp06cPGRkZxghEixYt4sSJE2zevJlRo0ZZrRcSEkJWVhaDBw8mLi6ODRs2MGfOHOB/v44PHTqUCxcu0KdPH/bu3cvPP//Mxo0bGTBgAJmZmezZs4dp06YRHR3NmTNn+PLLLzl//vxtg4M2bdoQERGByWTC19fXSFu9evUdm0FVrVqVTZs2kZCQUKBmNcOGDSMiIoJZs2bx008/MX/+fKv+FQATJ07k448/ZtKkSRw+fJi4uDg+++wzJkyYYJXvP//5D0uXLuWnn37i9ddfZ+/evbz44osA9O3blwoVKvD444+zY8cOTp06xbZt2xg+fDhnz57NU1lr1apFp06dGDRoEHv27CEmJobnn3/eqoahffv2BAYG0r17dzZs2MDp06eJiopiwoQJOYLFW3FycmLs2LGEhYXx8ccfc/LkSXbv3s2SJUtyzX83rg0pfhRYiIiIFLGKFSuyfPly/vOf/+Dr68uMGTOMh/psbm5ubN68mcuXL9O6dWsaNWrE4sWLc+1z4enpyebNm/nxxx/p27cvFouF8PBwYmJi8PPzY+TIkcyePdtqnTJlyrBu3TpiY2OpX78+48ePZ+LEicD/mvp4eXnx/fffk5mZSVBQEH5+fgwfPhyz2YyNjQ1lypRh+/btdO7cmYcffpgJEybw1ltv8dhjj93y2Fu1agVA69atjQCmdevWZGZm3jGweOutt4iMjMTb25sGDRrc4SzfWtOmTfnoo4+YN28e9evXZ+PGjTkChqCgINavX09kZCSPPPIITZs25e233zY6ImebPHky4eHh1K1blxUrVrB69WojYCpVqhTbt2+ncuXK9OzZEx8fHwYMGMDVq1fvOKPxjZYtW4a3tzetW7emZ8+exrCy2UwmE9988w2tWrViwIABPPzwwzz99NOcPn36lqM65ea1115j9OjRTJw4ER8fH3r37n3Lvip349qQ4sdk0ThfhSY/U57fDQH3dtLZuyqPP6iI5Opez8B8t+R3Zud/imvXrnHq1CmqVat2x7btUjCrV6+mf//+JCcn57vN/T+RZgKX4up2n6v5eb5V520REREB4OOPP6Z69eo88MADHDhwgLFjx9KrVy8FFSKSJwosREREBLg+0/XEiRNJSEigUqVKPPXUU7ec+0FE5GYKLERERASAsLAwwsLCiroYxZZal8s/nTpvi4iIiIhIgSmwEBERERGRAlNgISIiIiIiBabAQkRERERECkyBhYiIiIiIFJgCCxERERERKTAFFiIiIiVAv379SsyMz23atGHEiBFFXYx7YtKkSdSvX7+oi5FnJpOJtWvX3rP9bd26FZPJxF9//XXP9nmzK1eu8MQTT1CmTJlCL0txe//vRPNYiIhIyRAQcG/3Fx2dr+z9+vVjxYoVANjZ2eHt7U3Pnj2ZPHkypUuXvhslvO9t3bqVtm3bkpSURNmyZQt128uXL2fEiBFF+kAqBdesWTPi4+Mxm81FVoYVK1awY8cOoqKiqFChwt8ui8lkYs2aNSXmB4DcKLAQERG5Rzp16sSyZctIT09nx44dPP/886SkpLBgwYKiLprkQ1paGg4ODkVdjH8EBwcHPD09i7QMJ0+exMfHBz8/vyItR15ZLBYyMzOxs7v3j/lqCiUiInKPODo64unpibe3NyEhIfTt29doVrJq1SoCAgJwdXXF09OTkJAQEhMTrdY/fPgwXbp0oUyZMri6utKyZUtOnjyZ675iYmJwd3fnzTffBCAiIoIWLVpQtmxZ3NzcCA4OzrFuVFQU9evXx8nJiYCAANauXYvJZCI2NtbIc+TIETp37oyLiwseHh6Ehobyxx9/GMs///xz/P39cXZ2xs3Njfbt25OSkpKjfKdPn6Zt27YAlCtXDpPJRL9+/YzlWVlZhIWFUb58eTw9PZk0aZLV+m+//Tb+/v6ULl0ab29vhgwZwuXLl4HrNSH9+/cnOTkZk8mEyWTKsX62kydP8vjjj+Ph4YGLiwuPPPII3333nVWeqlWrMnXqVPr164fZbGbQoEHG+WrVqhXOzs54e3szbNiwXI/1RjNmzMDDwwNXV1cGDhzItWvXcuRZtmwZPj4+ODk5Ubt2bT744AOr82YymQgPD6dZs2Y4OTlRp04dtm7darWNO71Pbdq0YdiwYbc9x8ePH6dVq1Y4OTnh6+tLZGRkjrKeO3eO3r17U65cOdzc3Hj88cc5ffq0sTy7id6cOXOoVKkSbm5uDB06lPT0dCNPamoqYWFheHt74+joSM2aNVmyZAmQe1OoO533Dz74gJo1a+Lk5ISHhwdPPvnkbd+TL774gjp16uDo6EjVqlV56623rM7TW2+9xfbt2zGZTLRp0+aW21mwYAEPPfQQDg4O1KpVi5UrVxrLqlatCkCPHj0wmUzG62wrV66katWqmM1mnn76aS5dumQss1gszJo1i+rVq+Ps7Ey9evX4/PPPjeXZ52jDhg0EBATg6OjIjh07OHDgAG3btsXV1ZUyZcrQqFEjovNZ05pfCixERESKiLOzs/GAlZaWxpQpUzhw4ABr167l1KlTVg/a586dMx7yNm/eTExMDAMGDCAjIyPHdrdu3Uq7du2YPHky48ePByAlJYVRo0axb98+Nm3ahI2NDT169CArKwuAS5cu0bVrV/z9/fnhhx+YMmUKY8eOtdpufHw8rVu3pn79+kRHRxMREcHvv/9Or169jOV9+vRhwIABxMXFsXXrVnr27InFYslRRm9vb7744gsAjh07Rnx8PO+++66xfMWKFZQuXZo9e/Ywa9Ys3njjDasHWxsbG9577z0OHTrEihUr2Lx5M2FhYcD15jPvvPMOZcqUIT4+nvj4eMaMGZPre3D58mU6d+7Md999x/79+wkKCqJr166cOXPGKt/s2bPx8/MjJiaG1157jR9//JGgoCB69uzJwYMH+eyzz9i5cycvvvhirvsB+H//7//x+uuv8+abbxIdHU2lSpWsggaAxYsXM378eN58803i4uKYNm0ar732mtGMLtvLL7/M6NGj2b9/P82aNaNbt278+eefeXqf8nKOs7Ky6NmzJ7a2tuzevZsPP/wwx/Vw5coV2rZti4uLC9u3b2fnzp24uLjQqVMn0tLSjHxbtmzh5MmTbNmyhRUrVrB8+XKWL19uLH/22WcJDw/nvffeIy4ujg8//BAXF5dcz+Gdznt0dDTDhg3jjTfe4NixY0RERNCqVatbvicxMTH06tWLp59+mh9//JFJkybx2muvGeX78ssvGTRoEIGBgcTHx/Pll1/mup01a9YwfPhwRo8ezaFDh3jhhRfo378/W7ZsAWDfvn3A9aAxPj7eeA3Xg9u1a9eyfv161q9fz7Zt25gxY4axfMKECSxbtowFCxZw+PBhRo4cyTPPPMO2bdusyhAWFsb06dOJi4ujbt269O3blwcffJB9+/YRExPDK6+8gr29/S3PRWEwWXK72++R7du3M3v2bGJiYoiPj79tu7MXXniBRYsWMXfuXKsOXampqYwZM4ZPP/2Uq1ev0q5dOz744AMefPBBI09SUhLDhg3jq6++AqBbt27MmzfPqj3nmTNnGDp0KJs3b8bZ2ZmQkBDmzJmTr6rOixcvYjabSU5OpkyZMvk6F4XhXjcvvpvuckAtJVzAopJxM0QP1o2Qm2vXrnHq1CmqVauGk5PT/xYUgz4Wf/31l1FDsXfvXjp37ky7du347LPPcuTft28fjRs35tKlS7i4uPDqq68SHh7OsWPHcn04yN5+//79CQ0NZeHChfTp0+eW5Tl//jzu7u78+OOP+Pn58eGHHzJhwgTOnj1rnNePPvqIQYMGsX//furXr8/EiRPZs2cPGzZsMLZz9uxZvL29OXbsGJcvX6ZRo0acPn2aKlWq3PGc3KqPRZs2bcjMzGTHjh1GWuPGjXn00UetHrhu9J///Id///vfxq/yBeljUadOHf79738bD6tVq1alQYMGrFmzxsjz7LPP4uzszMKFC420nTt30rp1a1JSUqyvzf/TrFkz6tWrZ9X0rWnTply7ds2oFapcuTIzZ860eu+mTp3KN998Q1RUFKdPn6ZatWrMmDHDeNDPyMigWrVqvPTSS4SFhd3xfXr44YfveI43btxI586dOX36tPFMFRERwWOPPWY8ry1dupRZs2YRFxeHyWQCrgfIZcuWZe3atXTs2JF+/fqxdetWTp48ia2tLQC9evXCxsaG8PBwfvrpJ2rVqkVkZCTt27fPcc5uvkbudN6/+eYb+vfvz9mzZ3F1db3TW03fvn05f/48GzduNNLCwsL4+uuvOXz4MAAjRowgNjY2R63QjZo3b06dOnVYtGiRkdarVy9SUlL4+uuvgdz7WEyaNInZs2eTkJBglDcsLIzt27eze/duUlJSqFChAps3byYwMNBY7/nnn+fKlSt88sknxjlau3Ytjz/+uJGnTJkyzJs3j+eee+6O5+GWn6vk7/m2SGssUlJSqFevHvPnz79tvrVr17Jnzx68vLxyLBsxYgRr1qwhPDycnTt3cvnyZYKDg8nMzDTyhISEEBsbS0REBBEREcTGxhIaGmosz8zMpEuXLqSkpLBz507Cw8P54osvGD16dOEdrIiI/OOtX78eFxcXnJycCAwMpFWrVsybNw+A/fv38/jjj1OlShVcXV2NJhfZv5zHxsbSsmXL2/7iuGfPHp544glWrFiRI6g4efIkISEhVK9enTJlylCtWjWr7R87doy6detaPVQ0btzYahsxMTFs2bIFFxcX46927drG9uvVq0e7du3w9/fnqaeeYvHixSQlJf2tc1W3bl2r15UqVbJqGrZlyxY6dOjAAw88gKurK88++yx//vnnHZsi3SwlJYWwsDB8fX0pW7YsLi4uHD16NEeNRcBNgWtMTAzLly+3OhdBQUFkZWVx6tSpXPcVFxdn9XAIWL0+f/48v/76KwMHDrTa7tSpU3M0W7txPTs7OwICAoiLizPKdrv3KdvtznFcXByVK1e2+qH25rLHxMRw4sQJXF1djf2UL1+ea9euWe2nTp06RlBx835iY2OxtbWldevWuZ6zm93pvHfo0IEqVapQvXp1QkNDWb16NVeuXLnl9uLi4mjevLlVWvPmzTl+/LjVs+Sd3Go72e/J7VStWtUqCLrx/Bw5coRr167RoUMHq2P++OOPc1wTN1+jo0aN4vnnn6d9+/bMmDHjls0mC1ORdt5+7LHHeOyxx26b59y5c7z44ots2LCBLl26WC1LTk5myZIlrFy50ohyV61ahbe3N9999x1BQUHExcURERHB7t27adKkCXC9mjEwMJBjx45Rq1YtNm7cyJEjR/j111+N4OWtt96iX79+vPnmm0VS+yAiIiVP27ZtWbBgAfb29nh5eRlBQkpKCh07dqRjx46sWrWKihUrcubMGYKCgowmJc7Oznfc/kMPPYSbmxtLly6lS5cuVrXuXbt2xdvbm8WLF+Pl5UVWVhZ+fn7G9i0Wi/Grc7abGzVkZWXRtWtXZs6cmWPflSpVwtbWlsjISKKioti4cSPz5s1j/Pjx7Nmzxwhk8urmAMpkMhnNtn755Rc6d+7Mv/71L6ZMmUL58uXZuXMnAwcOtGq7nxcvv/wyGzZsYM6cOdSoUQNnZ2eefPJJq6Y8QI6Ru7KysnjhhRcYNmxYjm1Wrlw5X2W4cZtw/Tkl+5kl240P5reS/f7d6X3KdrtznFuDlpuvj6ysLBo1asTq1atz5K1YsWKe9pOX6/rmfd7uvDs4OPDDDz+wdetWNm7cyMSJE5k0aRL79u3LdeSxvFz3eZXbdm5Oy83tzk/2v19//TUPPPCAVT5HR0er1zdfo5MmTSIkJISvv/6ab7/9ltdff53w8HB69OiRtwP6G+7rPhZZWVmEhoby8ssvU6dOnRzLY2JiSE9Pp2PHjkaal5cXfn5+REVFAbBr1y7MZrPVDdq0aVPMZrNVHj8/P6sakaCgIFJTU4mJiblbhyciIv8wpUuXpkaNGlSpUsXqYeLo0aP88ccfzJgxg5YtW1K7du0cHbfr1q3Ljh07bvvgnN1k4uTJk/Tu3dvI++effxIXF8eECRNo164dPj4+OWoSateuzcGDB0lNTTXSbu7o2bBhQw4fPkzVqlWpUaOG1V/2Q43JZKJ58+ZMnjyZ/fv34+DgYNWE6EbZgU9+fhnOLldGRgZvvfUWTZs25eGHH+a3337Lse28bHfHjh3069ePHj164O/vj6enp1Xn41vJPhc3n4caNWrcshm1j48Pu3fvtkq78bWHhwcPPPAAP//8c45t3hyY3bheRkYGMTExRq1EXt6nO/H19eXMmTNW53XXrl05zsHx48dxd3fPsZ+8Dsnq7+9PVlZWjv4Ct5KX825nZ0f79u2ZNWsWBw8e5PTp02zevPmWx7lz506rtKioKB5++OE8BXPZfHx8ct2Oj4+P8dre3j7f17qvry+Ojo6cOXMmx/F6e3vfcf2HH36YkSNHsnHjRnr27MmyZcvytf/8uq+Hm505cyZ2dna5RqUACQkJODg4UK5cOat0Dw8PEhISjDzu7u451nV3d7fK4+HhYbW8XLlyODg4GHlyk5qaavUBfPHiReD6DZ5bZ7q7rQhGFbtriuD0SQlid39/tOVZUXyOFAcZGRlYLBbjr8j8zX3fqiOzg4MD7733Hv/61784dOgQU6ZMMfJbLBaGDh3KvHnzePrpp3nllVcwm83s3r2bxo0bU6tWLWNbFStWZNOmTTz66KP06dOHTz/91BgJatGiRXh6enLmzBnGjRtntf0+ffowfvx4Bg8ezNixYzlz5gxz5syxKveQIUNYvHgxffr0YcyYMVSoUIETJ04QHh7O4sWLiY6OZtOmTXTs2BF3d3f27NnD+fPnqV27dq7HXblyZUwmE+vWraNz5844OzsbnXZv9f5aLBaqV69ORkYG7733Hl27duX777/nww8/tFqvSpUqXL58me+++4569epRqlQpSpUqlWN7NWrU4MsvvyQ4OBiTycTEiRPJysrKsf+bX4eFhREYGMiQIUMYNGgQpUuXJi4ujsjISKN5282GDRtGv379aNSoES1atGD16tUcPnyY6tWrG9t+/fXXGT58OK6urjz22GOkpqYSHR1NUlISo0aNMvK9//771KhRAx8fH+bOnUtSUhL9+/fP0/uU/cB8u3Pcrl07atWqxbPPPsucOXO4ePGiMRBA9nohISHMnj2bxx9/nMmTJ/Pggw9y5swZvvzyS15++WWrZlQ3n8vsf6tUqcJzzz3HgAEDePfdd6lXrx6//PILiYmJ9OrVyyqvxWK543lfv349P//8M61ataJcuXJ88803ZGVl8fDDD+d6rKNGjaJx48a88cYb9O7dm127djF//nzef/99q33ffAw3GzNmDL1796ZBgwa0a9eOdevW8eWXXxIZGWmsV7VqVb777juaNWuGo6Mj5cqVy3XbN6a5uLgwevRoRo4cSWZmJi1atODixYtERUXh4uLCc889l+McAVy9epWXX36ZJ598kmrVqnH27Fn27dt3y8EUstfN7Rk2P99F9+23b0xMDO+++y4//PBDnqqRbnRz1VNu6/+dPDebPn06kydPzpEeHR1dJJMd3aaPXrGzZ09Rl0CKsz7lSsbNsEc3Qq5MJhOlSpXiypUrVr/+Of9fk4F75Wo+2/Knp6eTkZGRax+AUqVK8eGHHzJ58mTmzZtHvXr1mDJlCr179+bq1atGZ+D169czYcIE2rRpg62tLf7+/jRo0ICUlBSr7bu6urJu3Toee+wxnn76aZYuXcqyZct4+eWX8ff3p2bNmsyePZvHHnuMa9eukZKSgq2tLZ999hkjR46kQYMG1KlTh7CwMAYMGEBWVhYpKSmYzWajeUmnTp1ITU3F29ubDh06cPXqVezs7Ni6dSvvvPMOly5dwtvbm2nTptGqVatcj7ts2bKMHz+eV155hQEDBtCnTx8WLlxIZmZmjnOVkZFBeno6KSkp1KxZk+nTpzNz5kxeffVVmjdvzuuvv87gwYNJSUnB3t6eevXqMXDgQHr37s2FCxcYN24cr776ao4yTJ06lSFDhtC8eXPc3NwYOXIkSUlJVvu3WCykpaVZleehhx7i22+/5Y033qBVq1ZYLBaqVatGz549b9nPIzg4mLFjxzJ27FhSU1Pp1q0bAwcOZNOmTcY6ffr0wdbWlnfffZexY8dSunRpfH19GTJkCCkpKUZ/gUmTJjF9+nQOHjxItWrVCA8Px9nZOU/vk8lkuuM5huvNy4cOHUqTJk2oXLkys2fPpkePHsY1A/Dtt9/y2muv0bNnTy5fvoyXlxetW7fG1tY2x3V5434yMzONtNmzZzNp0iSGDBnChQsX8Pb2ZvTo0aSkpBjD8Wa/r3c6746Ojnz++edMnjyZa9eu8dBDD7Fs2TKqVq2a6/tSq1YtPv74Y6ZOncrUqVPx9PRk/PjxPPXUU0b+m8ubmw4dOjBr1ixmz57N8OHDqVKlCgsWLOCRRx4x1ps6dSqvvvoqH330EV5eXhw+fJj09HTj/sqWlpaGxWIx0l555RXKli3LtGnTOH36NGazmfr169/yHGVvIzExkWeffZbExETc3Nzo1q0bYWFhuR5HamoqaWlpHDx4MEfgkZ9+S0U6KtSNbu4p/8477zBq1ChsbP7XWiszMxMbGxu8vb2Naq127dpx4cIFq1qLevXq0b17dyZPnszSpUsZNWpUjlEhypYty9y5c+nfvz8TJ07kv//9LwcOHDCWJyUlUb58eTZv3myMs32z3GosvL29+fPPP4ukX0aLFvd8l3fNTbWJIvnSYmnJuBl2DtCNkJtr165x5syZXEcvkcK1evVqBgwYwF9//ZXvtvByd5w+fZrq1avzww8/UL9+/aIujpQQ2aNCVa5cOddRodzc3PI0KtR9W2MRGhqaY9ixoKAgQkND6d+/PwCNGjXC3t6eyMhIqzG0Dx06xKxZs4DrIxgkJyezd+9eY3SLPXv2kJycTLNmzYw8b775JvHx8Uanpo0bN+Lo6EijRo1uWUZHR8ccHWfgetu+opjtsCS1mihJzbrk3sugZNwMRfE5UhzY2dkZk57lt0Zbbu/jjz+mevXqPPDAAxw4cIBXXnmFXr165dqESIpG9jWv618KU/b1lNszbH6+i4r0W+vy5cucOHHCeH3q1CliY2MpX748lStXxs3NzSq/vb09np6eRltSs9nMwIEDGT16NG5ubpQvX54xY8bg7+9vBCU+Pj506tSJQYMGGWMeDx48mODgYGM7HTt2xNfXl9DQUGbPns2FCxcYM2YMgwYN0ohQIiLyj5GQkMDEiRNJSEigUqVKPPXUU8bM3SIid1KkgUV0dLRVM6NRo0YB8Nxzz1nNyHg7c+fOxc7Ojl69ehkT5C1fvtyqJ//q1asZNmyYMXpUt27drObOsLW15euvvzbaWN44QZ6IiMg/RVhYmDF7tdyfqlatWrSDFojcxn3Tx6Ik0MzbhUczb0tBaObtku12M8SKiEj+lYiZt0VEREREpGRQYCEiIiIiIgWmwEJERERERApMgYWIiIiIiBSYAgsRERERESkwBRYiIiIlQL9+/ejevXtRF+Ouqlq1Ku+8885t80yaNOm+n5G6OL1Xp0+fxmQyERsbe8/2uXz5csqWLXvP9ieFR9O6iohIiRAdfW+HGQ4IyN9wwP369WPFihXA9Zlsvb296dmzJ5MnT6Z06dJ3o4glzr59+6zOlclkYs2aNUXykD5p0iTWrl17Tx+4/yl69+5N586di7oY8jcosBAREblHOnXqxLJly0hPT2fHjh08//zzpKSksGDBgqIuWrFQsWLFoi6C3APOzs44OzsXdTHkb1BTKBERkXvE0dERT09PvL29CQkJoW/fvqxduxaAVatWERAQgKurK56enoSEhJCYmGi1/uHDh+nSpQtlypTB1dWVli1bcvLkyVz3FRMTg7u7O2+++SYAERERtGjRgrJly+Lm5kZwcHCOdaOioqhfvz5OTk4EBASwdu3aHM1gjhw5QufOnXFxccHDw4PQ0FD++OMPY/nnn3+Ov78/zs7OuLm50b59e1JSUnItY6NGjXjrrbeM1927d8fOzo6LFy8CkJCQgMlk4tixY4B1U6iqVasC0KNHD0wmk/E628qVK6latSpms5mnn36aS5cuGctSU1MZNmwY7u7uODk50aJFC/bt22csz60pTva5yF4+efJkDhw4gMlkwmQysXz58lyPMTMzk1GjRhnnPSwsLMfM2RaLhVmzZlG9enWcnZ2pV68en3/+ubF869atmEwmvv76a+rVq4eTkxNNmjThxx9/tNpOVFQUrVq1wtnZGW9vb4YNG2Z17qtWrcq0adMYMGAArq6uVK5cmUWLFlltY+/evTRo0MC4Bvbv35/jmO50DbRp04Zhw4YRFhZG+fLl8fT0ZNKkSVbb+Ouvvxg8eDAeHh44OTnh5+fH+vXrb3n+161bR6NGjXBycqJ69epMnjyZjIwMY/mkSZOoXLkyjo6OeHl5MWzYsFzfD7m7FFiIiIgUEWdnZ9LT0wFIS0tjypQpHDhwgLVr13Lq1Cn69etn5D137hytWrXCycmJzZs3ExMTw4ABA6werrJt3bqVdu3aMXnyZMaPHw9ASkoKo0aNYt++fWzatAkbGxt69OhBVlYWAJcuXaJr1674+/vzww8/MGXKFMaOHWu13fj4eFq3bk39+vWJjo4mIiKC33//nV69ehnL+/Tpw4ABA4iLi2Pr1q307Nkzx4N0tjZt2rB161bg+sP1jh07KFeuHDt37gRgy5YteHp6UqtWrRzrZgcCy5YtIz4+3iowOHnyJGvXrmX9+vWsX7+ebdu2MWPGDGN5WFgYX3zxBStWrOCHH36gRo0aBAUFceHChVu/WTfo3bs3o0ePpk6dOsTHxxMfH0/v3r1zzfvWW2+xdOlSlixZws6dO7lw4QJr1qyxyjNhwgSWLVvGggULOHz4MCNHjuSZZ55h27ZtVvlefvll5syZw759+3B3d6dbt27G9fPjjz8SFBREz549OXjwIJ999hk7d+7kxRdfzFGe7IBhyJAh/Pvf/+bo0aPA9WskODiYWrVqERMTw6RJkxgzZozV+ne6BrKtWLGC0qVLs2fPHmbNmsUbb7xBZGQkAFlZWTz22GNERUWxatUqjhw5wowZM7C1tc31HG7YsIFnnnmGYcOGceTIERYuXMjy5cuNoPnzzz9n7ty5LFy4kOPHj7N27Vr8/f1v+f7J3aOmUCIiIkVg7969fPLJJ7Rr1w6AAQMGGMuqV6/Oe++9R+PGjbl8+TIuLi68//77mM1mwsPDsbe3B+Dhhx/Osd3//ve/hIaGsnDhQvr06WOkP/HEE1b5lixZgru7O0eOHMHPz4/Vq1djMplYvHgxTk5O+Pr6cu7cOQYNGmSss2DBAho2bMi0adOMtKVLl+Lt7c1PP/3E5cuXycjIoGfPnlSpUgXgtg94bdq0YcmSJWRlZfHjjz9ia2vLM888w9atW+ncuTNbt26ldevWua6b3SyqbNmyeHp6Wi3Lyspi+fLluLq6AhAaGsqmTZt48803jaZny5cv57HHHgNg8eLFREZGsmTJEl5++eVbljebs7MzLi4u2NnZ5dj3zd555x3GjRtnnP8PP/yQDRs2GMtTUlJ4++232bx5M4GBgcD193/nzp0sXLjQ6vhff/11OnToAFx/cH/wwQdZs2YNvXr1Yvbs2YSEhDBixAgAatasyXvvvUfr1q1ZsGABTk5OAHTu3JkhQ4YAMHbsWObOncvWrVupXbs2q1evJjMzk6VLl1KqVCnq1KnD2bNn+fe//22U4U7XQPY1WbduXV5//XWjLPPnz2fTpk106NCB7777jr179xIXF2fkr169+i3P4Ztvvskrr7zCc889Z+SdMmUKYWFhvP7665w5cwZPT0/at2+Pvb09lStXpnHjxrd9X+TuUI2FiIjIPbJ+/XpcXFxwcnIiMDCQVq1aMW/ePAD279/P448/TpUqVXB1daVNmzYAnDlzBoDY2FhatmxpBBW52bNnD0888QQrVqywCirg+q/4ISEhVK9enTJlylCtWjWr7R87doy6desaD6BAjoezmJgYtmzZgouLi/FXu3ZtY/v16tWjXbt2+Pv789RTT7F48WKSkpJuWd5WrVpx6dIl9u/fz7Zt22jdujVt27Y1fqm/XWBxO1WrVjWCCoBKlSoZzcpOnjxJeno6zZs3N5bb29vTuHFj4uLi8r2v20lOTiY+Pt4IGOB6x/2AgP8NNHDkyBGuXbtGhw4drM7rxx9/nKOp2o3bKV++PLVq1TLKHBMTw/Lly622ERQURFZWFqdOnTLWq1u3rvF/k8mEp6encW7i4uKoV68epUqVynWf2fu53TWQ237A+j2IjY3lwQcfzDUwzk1MTAxvvPGG1T4HDRpEfHw8V65c4amnnuLq1atUr16dQYMGsWbNmlxr8uTuU42FiIjIPdK2bVsWLFiAvb09Xl5eRpCQkpJCx44d6dixI6tWraJixYqcOXOGoKAg0tLSAPLUmfWhhx7Czc2NpUuX0qVLFxwcHIxlXbt2xdvbm8WLF+Pl5UVWVhZ+fn7G9i0Wi9GHINvNTZiysrLo2rUrM2fOzLHvSpUqYWtrS2RkJFFRUWzcuJF58+Yxfvx49uzZYwQyNzKbzdSvX5+tW7cSFRXFo48+SsuWLYmNjeX48eP89NNPRoCVHzcHXyaTyWjylX1MuR1rdpqNjU2OY89uclTYssv19ddf88ADD1gtc3R0vOP62WXOysrihRdeyLVvQeXKlY3/5+Xc3Km8t7sG8rKf/HbMzsrKYvLkyfTs2TPHMicnJ7y9vTl27BiRkZF89913DBkyhNmzZ7Nt27bbBuJS+FRjISIico+ULl2aGjVqUKVKFasHnqNHj/LHH38wY8YMWrZsSe3atXN03K5bty47duy47QNuhQoV2Lx5MydPnqR3795G3j///JO4uDgmTJhAu3bt8PHxyVGTULt2bQ4ePEhqaqqRFh1tPaRuw4YNOXz4MFWrVqVGjRpWf9nDwJpMJpo3b87kyZPZv38/Dg4OOfoU3KhNmzZs2bKF7du306ZNG8qWLYuvry9Tp07F3d0dHx+fW65rb29PZmbmLZfnpkaNGjg4OBj9OOB60BAdHW3sq2LFily6dMmq4/PNw8o6ODjccd9ms5lKlSqxe/duIy0jI4OYmBjjta+vL46Ojpw5cybHOfX29rba3o3bSUpK4qeffjJqC7Lfm5u3kX28eeHr68uBAwe4evVqrvu8cT+3uwbupG7dupw9e5affvopT/kbNmzIsWPHcj02G5vrj7LOzs5069aN9957j61bt7Jr164cndvl7lNgISIiUsQqV66Mg4MD8+bN4+eff+arr75iypQpVnlefPFFLl68yNNPP010dDTHjx9n5cqVxohJ2dzd3dm8eTNHjx6lT58+ZGRkUK5cOdzc3Fi0aBEnTpxg8+bNjBo1ymq9kJAQsrKyGDx4MHFxcWzYsIE5c+YA//tVfOjQoVy4cIE+ffqwd+9efv75ZzZu3MiAAQPIzMxkz549TJs2jejoaM6cOcOXX37J+fPnbxsctGnThoiICEwmE76+vkba6tWr79gMqmrVqmzatImEhITbNrm6UenSpfn3v//Nyy+/TEREBEeOHGHQoEFcuXKFgQMHAtCkSRNKlSrFq6++yokTJ/jkk09yjPpUtWpVTp06RWxsLH/88YdVQHaj4cOHM2PGDNasWcPRo0cZMmQIf/31l7Hc1dWVMWPGMHLkSFasWMHJkyfZv38/77//vjHvSbY33niDTZs2cejQIfr160eFChWMOTzGjh3Lrl27GDp0qFHj89VXX/HSSy/l6bzA9WvAxsaGgQMHcuTIEb755hvjGsh2p2sgL1q3bk2rVq144okniIyM5NSpU3z77bdERETkmn/ixIl8/PHHTJo0icOHDxMXF8dnn33GhAkTgOujSC1ZsoRDhw7x888/s3LlSpydnY1+PnLvKLAQEREpYhUrVmT58uX85z//wdfXlxkzZuR4oHNzc2Pz5s1cvnyZ1q1b06hRIxYvXpxrUw9PT082b97Mjz/+SN++fbFYLISHhxMTE4Ofnx8jR45k9uzZVuuUKVOGdevWERsbS/369Rk/fjwTJ04EMPpdeHl58f3335OZmUlQUBB+fn4MHz4cs9mMjY0NZcqUYfv27XTu3JmHH36YCRMm8NZbbxmdpHPTqlUr4PrDZnYA07p1azIzM+8YWLz11ltERkbi7e1NgwYN7nCW/2fGjBk88cQThIaG0rBhQ06cOMGGDRsoV64ccL3/wqpVq/jmm2/w9/fn008/zTFc6hNPPEGnTp1o27YtFStW5NNPP811X6NHj+bZZ5+lX79+BAYG4urqSo8ePazyTJkyhYkTJzJ9+nR8fHwICgpi3bp1OZqPzZgxg+HDh9OoUSPi4+P56quvjNqIunXrsm3bNo4fP07Lli1p0KABr732mlXzpDtxcXFh3bp1HDlyhAYNGjB+/PgcTZ7udA3k1RdffMEjjzxCnz598PX1JSws7JaBSVBQEOvXrycyMpJHHnmEpk2b8vbbbxuBQ9myZVm8eDHNmzenbt26bNq0iXXr1uHm5pbn8kjhMFny0qBO8uTixYuYzWaSk5MpU6bMPd9/wL2ddPauis7fhLYiVgIWlYybIXqwboTcXLt2jVOnTlGtWjWrjsZS+FavXk3//v1JTk7WhGVFaOvWrbRt25akpKQc8zuIFIbbfa7m5/lWnbdFREQEgI8//pjq1avzwAMPcODAAcaOHUuvXr0UVIhIniiwEBEREeD6TNcTJ04kISGBSpUq8dRTTxmTkImI3Em+A4urV69isViMMY5/+eUX1qxZg6+vLx07diz0AoqIiMi9ERYWRlhYWFEXQ27Spk2bPA0FK1LU8t15+/HHH+fjjz8G4K+//qJJkya89dZbPP744yxYsKDQCygiIiIiIve/fAcWP/zwAy1btgTg888/x8PDg19++YWPP/6Y9957r9ALKCIikhv9gisiUjgK6/M034HFlStXcHV1BWDjxo307NkTGxsbmjZtyi+//FIohRIREbkVW1tbAGPGaBERKZgrV64AOWdMz69897GoUaMGa9eupUePHmzYsIGRI0cCkJiYWCRDrIqIyD+LnZ0dpUqV4vz589jb2+dr7HwREfkfi8XClStXSExMpGzZssYPN39XvgOLiRMnEhISwsiRI3n00UcJDAwErtde5GdyGhERkb/DZDJRqVIlTp06pZpyEZFCULZsWTw9PQu8nXwHFk8++SQtWrQgPj6eevXqGent2rXLMZOkiIjI3eDg4EDNmjXVHEpEpIDs7e0LXFOR7W/NY+Hp6cnly5eJjIykVatWODs788gjj2AymfK1ne3btzN79mxiYmKIj49nzZo1dO/eHYD09HQmTJjAN998w88//4zZbKZ9+/bMmDEDLy8vYxupqamMGTOGTz/9lKtXr9KuXTs++OADHnzwQSNPUlISw4YN46uvvgKgW7duzJs3z2r2yjNnzjB06FA2b96Ms7MzISEhzJkzBwcHh79zikRE5C6zsbHRzNsiIveRfDdM/fPPP2nXrh0PP/wwnTt3Jj4+HoDnn3+e0aNH52tbKSkp1KtXj/nz5+dYduXKFX744Qdee+01fvjhB7788kt++uknunXrZpVvxIgRrFmzhvDwcHbu3Mnly5cJDg4mMzPTyBMSEkJsbCwRERFEREQQGxtLaGiosTwzM5MuXbqQkpLCzp07CQ8P54svvsj38YiIiIiI/FOZLPkcX+rZZ58lMTGRjz76CB8fHw4cOED16tXZuHEjI0eO5PDhw3+vICaTVY1Fbvbt20fjxo355ZdfqFy5MsnJyVSsWJGVK1fSu3dvAH777Te8vb355ptvCAoKIi4uDl9fX3bv3k2TJk0A2L17N4GBgRw9epRatWrx7bffEhwczK+//mrUhoSHh9OvX798dUq/ePEiZrOZ5OTkIunIHhBwz3d510RHF3UJpDgLWFQybobowboRRESkaOXn+TbfTaE2btzIhg0brJoaAdSsWfOud6JLTk7GZDIZTZhiYmJIT0+3mvHby8sLPz8/oqKiCAoKYteuXZjNZiOoAGjatClms5moqChq1arFrl278PPzs2piFRQURGpqKjExMbRt2zbX8qSmppKammq8vnjxIgAZGRlkZGQU5qHnid3fath2fyqC0ycliN3fa+V53ymKzxEREZEb5ee7KN/fvikpKZQqVSpH+h9//IGjo2N+N5dn165d45VXXiEkJMSIlhISEnBwcKBcuXJWeT08PEhISDDyuLu759ieu7u7VR4PDw+r5eXKlcPBwcHIk5vp06czefLkHOnR0dGULl06fwdYCPr0uee7vGv27CnqEkhx1qdcybgZ9uhGEBGRIpaSkpLnvPkOLFq1asXHH3/MlClTgOtNmLKyspg9e/Ytf9kvqPT0dJ5++mmysrL44IMP7pjfYrFYdSTPrVP538lzs3HjxjFq1Cjj9cWLF/H29iYgIKBImkKVpC4hQ4cWdQmkOBu9tGTcDEN76EYQEZGild0iJy/yHVjMnj2bNm3aEB0dTVpaGmFhYRw+fJgLFy7w/fff53dzd5Senk6vXr04deoUmzdvtnpg9/T0JC0tjaSkJKtai8TERJo1a2bk+f3333Ns9/z580YthaenZ45fBpOSkkhPT89Rk3EjR0fHXGtp7OzssCuCdkklqdVESWrWJfdeBiXjZiiKzxEREZEb5ee7KN+jQvn6+nLw4EEaN25Mhw4dSElJoWfPnuzfv5+HHnoov5u7reyg4vjx43z33Xe4ublZLW/UqBH29vZERkYaafHx8Rw6dMgILAIDA0lOTmbv3r1Gnj179pCcnGyV59ChQ8YIV3C9L4mjoyONGjUq1GMSERERESmJ/vY8Frn1Lcivy5cvc+LECeP1qVOniI2NpXz58nh5efHkk0/yww8/sH79ejIzM43+DuXLl8fBwQGz2czAgQMZPXo0bm5ulC9fnjFjxuDv70/79u0B8PHxoVOnTgwaNIiFCxcCMHjwYIKDg6lVqxYAHTt2xNfXl9DQUGbPns2FCxcYM2YMgwYNKpImTSIiIiIixU2+A4uDBw/mmm4ymXBycqJy5cp57sQdHR1t1S8ju7/Cc889x6RJk4wJ7erXr2+13pYtW2jTpg0Ac+fOxc7Ojl69ehkT5C1fvtxqBsHVq1czbNgwY/Sobt26Wc2dYWtry9dff82QIUNo3ry51QR5IiIiIiJyZ/mex8LGxsbo0Jy96o0dnO3t7enduzcLFy78x82IqnksCo/msZCC0DwWIiIihSM/z7f57mOxZs0aatasyaJFizhw4ACxsbEsWrSIWrVq8cknn7BkyRI2b97MhAkT/vYBiIiIiIhI8ZLvplBvvvkm7777LkFBQUZa3bp1efDBB3nttdfYu3cvpUuXZvTo0WpKJCIiIiLyD5HvGosff/yRKlWq5EivUqUKP/74I3C9T8SNIyyJiIiIiEjJlu/Aonbt2syYMYO0tDQjLT09nRkzZlC7dm0Azp07d9v5H0REREREpGTJd1Oo999/n27duvHggw9St25dTCYTBw8eJDMzk/Xr1wPw888/M2TIkEIvrIiIiIiI3J/yHVg0a9aM06dPs2rVKn766ScsFgtPPvkkISEhuLq6AhAaGlroBRURERERkfvX35ogz8XFhX/961+FXRYRERERESmm/lZgAXDkyBHOnDlj1dcCrk8+JyIiIiIi/yz5Dix+/vlnevTowY8//ojJZMoxSV5mZmbhllBERERERO57+R4Vavjw4VSrVo3ff/+dUqVKcfjwYbZv305AQABbt269C0UUEREREZH7Xb5rLHbt2sXmzZupWLEiNjY22NjY0KJFC6ZPn86wYcPYv3//3SiniIiIiIjcx/JdY5GZmYmLiwsAFSpU4LfffgOuT5B37Nixwi2diIiIiIgUC/musfDz8+PgwYNUr16dJk2aMGvWLBwcHFi0aBHVq1e/G2UUEREREZH7XL4DiwkTJpCSkgLA1KlTCQ4OpmXLlri5uREeHl7oBRQRERERkftfvgOLoKAg4//Vq1fnyJEjXLhwgXLlyhVqwUREREREpPjIdx+L6dOn50grX748FouFkJCQQimUiIiIiIgUL/kOLN555x0WLVpklZaZmUnv3r2JjY0trHKJiIiIiEgxku+mUN988w3t27enbNmy9OrVi/T0dHr37s3Ro0fZsmXL3SijiIiIiIjc5/IdWDRq1Ig1a9bw+OOP4+joyJIlSzh58iRbtmzBw8PjbpRRRERERETuc/luCgXQpk0bVq5cyZNPPsnp06fZtm2bggoRERERkX+wPNVY9OzZM9f0ihUrUrZsWQYPHmykffnll4VTMhERERERKTbyFFiYzeZc028celZERERERP658hRYLFu27G6XQ0REREREirF897E4deoUx48fz5F+/PhxTp8+XRhlEhERERGRYibfgUW/fv2IiorKkb5nzx769etXGGUSEREREZFiJt+Bxf79+2nevHmO9KZNm+Z7grzt27fTtWtXvLy8MJlMrF271mq5xWJh0qRJeHl54ezsTJs2bTh8+LBVntTUVF566SUqVKhA6dKl6datG2fPnrXKk5SURGhoKGazGbPZTGhoKH/99ZdVnjNnztC1a1dKly5NhQoVGDZsGGlpafk6HhERERGRf6p8BxYmk4lLly7lSE9OTiYzMzNf20pJSaFevXrMnz8/1+WzZs3i7bffZv78+ezbtw9PT086dOhgtf8RI0awZs0awsPD2blzJ5cvXyY4ONiqLCEhIcTGxhIREUFERASxsbGEhoYayzMzM+nSpQspKSns3LmT8PBwvvjiC0aPHp2v4xERERER+acyWSwWS35WCA4OplSpUnz66afY2toC1x/Me/fuTUpKCt9+++3fK4jJxJo1a+jevTtwvbbCy8uLESNGMHbsWOB67YSHhwczZ87khRdeIDk5mYoVK7Jy5Up69+4NwG+//Ya3tzfffPMNQUFBxMXF4evry+7du2nSpAkAu3fvJjAwkKNHj1KrVi2+/fZbgoOD+fXXX/Hy8gIgPDycfv36kZiYSJkyZfJ0DBcvXsRsNpOcnJzndQpTQMA93+VdEx1d1CWQ4ixgUcm4GaIH60YQEZGilZ/n23zXWMyaNYvNmzdTq1Yt+vfvT//+/alVqxbbt29n9uzZf7vQNzt16hQJCQl07NjRSHN0dKR169ZGH4+YmBjS09Ot8nh5eeHn52fk2bVrF2az2Qgq4HqzLbPZbJXHz8/PCCrg+lC6qampxMTEFNoxiYiIiIiUVHkabvZGvr6+HDx4kPnz53PgwAGcnZ159tlnefHFFylfvnyhFSwhIQEgx4zeHh4e/PLLL0YeBwcHypUrlyNP9voJCQm4u7vn2L67u7tVnpv3U65cORwcHIw8uUlNTSU1NdV4ffHiRQAyMjLIyMjI03EWJrt8v5v3ryI4fVKC2OX/o+2+VBSfIyIiIjfKz3fR3/r29fLyYtq0aX9n1XwzmUxWry0WS460m92cJ7f8fyfPzaZPn87kyZNzpEdHR1O6dOnblvFu6NPnnu/yrtmzp6hLIMVZn3Il42bYoxtBRESKWEpKSp7z/u2f9a5cucKZM2dyjJxUt27dv7tJK56ensD12oRKlSoZ6YmJiUbtgqenJ2lpaSQlJVnVWiQmJtKsWTMjz++//55j++fPn7fazs1f4ElJSaSnp+eoybjRuHHjGDVqlPH64sWLeHt7ExAQUCR9LEpSX/OhQ4u6BFKcjV5aMm6GoT10I4iISNHKbpGTF/kOLM6fP0///v1v2Uk7vyND3Uq1atXw9PQkMjKSBg0aAJCWlsa2bduYOXMmAI0aNcLe3p7IyEh69eoFQHx8PIcOHWLWrFkABAYGkpyczN69e2ncuDFw/VfA5ORkI/gIDAzkzTffJD4+3ghiNm7ciKOjI40aNbplGR0dHXF0dMyRbmdnh10RtEsqSa0mSlKzLrn3MigZN0NRfI6IiIjcKD/fRfnuvD1ixAiSkpLYvXs3zs7OREREsGLFCmrWrMlXX32Vr21dvnyZ2NhYY/6LU6dOERsby5kzZzCZTIwYMYJp06axZs0aDh06RL9+/ShVqhQhISEAmM1mBg4cyOjRo9m0aRP79+/nmWeewd/fn/bt2wPg4+NDp06dGDRoELt372b37t0MGjSI4OBgatWqBUDHjh3x9fUlNDSU/fv3s2nTJsaMGcOgQYOKpOZBRERERKS4yffPYZs3b+a///0vjzzyCDY2NlSpUoUOHTpQpkwZpk+fTpcuXfK8rejoaNq2bWu8zm5W9Nxzz7F8+XLCwsK4evUqQ4YMISkpiSZNmrBx40ZcXV2NdebOnYudnR29evXi6tWrtGvXjuXLlxtD4QKsXr2aYcOGGaNHdevWzWruDFtbW77++muGDBlC8+bNcXZ2JiQkhDlz5uT39IiIiIiI/CPlex6LMmXKcPDgQapWrUrVqlVZvXo1zZs359SpU9SpU4crV67crbLe9zSPReHRPBZSEJrHQkREpHDc1XksatWqxbFjxwCoX78+Cxcu5Ny5c3z44YdWnaxFREREROSfI99NoUaMGEF8fDwAr7/+OkFBQaxevRoHBweWL19e2OUTEREREZFiIN+BRd++fY3/N2jQgNOnT3P06FEqV65MhQoVCrVwIiIiIiJSPBRoLEOLxYKzszMNGzYsrPKIiIiIiEgxlO8+FgBLlizBz88PJycnnJyc8PPz46OPPirssomIiIiISDGR7xqL1157jblz5/LSSy8RGBgIwK5duxg5ciSnT59m6tSphV5IERERERG5v+U7sFiwYAGLFy+mT58+Rlq3bt2oW7cuL730kgILEREREZF/oHw3hcrMzCQglwkTGjVqREZGRqEUSkREREREipd8BxbPPPMMCxYsyJG+aNEiqxGjRERERETkn+NvjQq1ZMkSNm7cSNOmTQHYvXs3v/76K88++yyjRo0y8r399tuFU0oREREREbmv5TuwOHTokDG87MmTJwGoWLEiFStW5NChQ0Y+k8lUSEUUEREREZH7Xb4Diy1bttyNcoiIiIiISDH2t+axEBERERERuZECCxERERERKTAFFiIiIiIiUmAKLEREREREpMDyFFg0bNiQpKQkAN544w2uXLlyVwslIiIiIiLFS54Ci7i4OFJSUgCYPHkyly9fvquFEhERERGR4iVPw83Wr1+f/v3706JFCywWC3PmzMHFxSXXvBMnTizUAoqIiIiIyP0vT4HF8uXLef3111m/fj0mk4lvv/0WO7ucq5pMJgUWIiIiIiL/QHkKLGrVqkV4eDgANjY2bNq0CXd397taMBERERERKT7yPfN2VlbW3SiHiIiIiIgUY/kOLABOnjzJO++8Q1xcHCaTCR8fH4YPH85DDz1U2OUTEREREZFiIN/zWGzYsAFfX1/27t1L3bp18fPzY8+ePdSpU4fIyMi7UUYREREREbnP5bvG4pVXXmHkyJHMmDEjR/rYsWPp0KFDoRVORERERESKh3zXWMTFxTFw4MAc6QMGDODIkSOFUigRERERESle8h1YVKxYkdjY2BzpsbGxhT5SVEZGBhMmTKBatWo4OztTvXp13njjDasO5BaLhUmTJuHl5YWzszNt2rTh8OHDVttJTU3lpZdeokKFCpQuXZpu3bpx9uxZqzxJSUmEhoZiNpsxm82Ehoby119/FerxiIiIiIiUVPkOLAYNGsTgwYOZOXMmO3bsYOfOncyYMYMXXniBwYMHF2rhZs6cyYcffsj8+fOJi4tj1qxZzJ49m3nz5hl5Zs2axdtvv838+fPZt28fnp6edOjQgUuXLhl5RowYwZo1awgPD2fnzp1cvnyZ4OBgMjMzjTwhISHExsYSERFBREQEsbGxhIaGFurxiIiIiIiUVCaLxWLJzwoWi4V33nmHt956i99++w0ALy8vXn75ZYYNG4bJZCq0wgUHB+Ph4cGSJUuMtCeeeIJSpUqxcuVKLBYLXl5ejBgxgrFjxwLXayc8PDyYOXMmL7zwAsnJyVSsWJGVK1fSu3dvAH777Te8vb355ptvCAoKIi4uDl9fX3bv3k2TJk0A2L17N4GBgRw9epRatWrlqbwXL17EbDaTnJxMmTJlCu085FVAwD3f5V0THV3UJZDiLGBRybgZogfrRhARkaKVn+fbfNdYmEwmRo4cydmzZ0lOTiY5OZmzZ88yfPjwQg0qAFq0aMGmTZv46aefADhw4AA7d+6kc+fOAJw6dYqEhAQ6duxorOPo6Ejr1q2JiooCICYmhvT0dKs8Xl5e+Pn5GXl27dqF2Ww2ggqApk2bYjabjTwiIiIiInJrf2sei2yurq6FVY5cjR07luTkZGrXro2trS2ZmZm8+eab9OnTB4CEhAQAPDw8rNbz8PDgl19+MfI4ODhQrly5HHmy109ISMi1f4i7u7uRJzepqamkpqYary9evAhc7xuSkZGR38MtMLsCvZv3lyI4fVKC2BXso+2+URSfIyIiIjfKz3fRff3t+9lnn7Fq1So++eQT6tSpQ2xsLCNGjMDLy4vnnnvOyHdzTYnFYrlj7cnNeXLLf6ftTJ8+ncmTJ+dIj46OpnTp0rfd/93wf/FWibBnT1GXQIqzPuVKxs2wRzeCiIgUsZSUlDznva8Di5dffplXXnmFp59+GgB/f39++eUXpk+fznPPPYenpydwvcahUqVKxnqJiYlGLYanpydpaWkkJSVZ1VokJibSrFkzI8/vv/+eY//nz5/PURtyo3HjxjFq1Cjj9cWLF/H29iYgIKBI+liMHn3Pd3nXDB1a1CWQ4mz00pJxMwztoRtBRESKVnaLnLy4rwOLK1euYGNj3Q3E1tbWGG62WrVqeHp6EhkZSYMGDQBIS0tj27ZtzJw5E4BGjRphb29PZGQkvXr1AiA+Pp5Dhw4xa9YsAAIDA0lOTmbv3r00btwYuP5LYXJyshF85MbR0RFHR8cc6XZ2dtgVQbukktRqoiQ165J7L4OScTMUxeeIiIjIjfLzXZSvb63sTtALFy7k4YcfznfB8qtr1668+eabVK5cmTp16rB//37efvttBgwYAFxvvjRixAimTZtGzZo1qVmzJtOmTaNUqVKEhIQAYDabGThwIKNHj8bNzY3y5cszZswY/P39ad++PQA+Pj506tSJQYMGsXDhQgAGDx5McHBwnkeEEhERERH5J8tXYGFvb8+hQ4cKffSnW5k3bx6vvfYaQ4YMITExES8vL1544QUmTpxo5AkLC+Pq1asMGTKEpKQkmjRpwsaNG606ls+dOxc7Ozt69erF1atXadeuHcuXL8fW1tbIs3r1aoYNG2aMHtWtWzfmz59/T45TRERERKS4y/c8FqNHj8be3p4ZM2bcrTIVW5rHovBoHgspCM1jISIiUjjy83yb7wa8aWlpfPTRR0RGRhIQEJBj9KO33347v5sUEREREZFiLt+BxaFDh2jYsCGAMXFdtnvVREpERERERO4v+Q4stmzZcjfKISIiIiIixZjNnbPk7sSJE2zYsIGrV68C1yeTExERERGRf6Z8BxZ//vkn7dq14+GHH6Zz587Ex8cD8PzzzzO6JM3QJiIiIiIieZbvwGLkyJHY29tz5swZSpUqZaT37t2biIiIQi2ciIiIiIgUD/nuY7Fx40Y2bNjAgw8+aJVes2ZNfvnll0IrmIiIiIiIFB/5rrFISUmxqqnI9scff+Do6FgohRIRERERkeIl34FFq1at+Pjjj43XJpOJrKwsZs+eTdu2bQu1cCIiIiIiUjzkuynU7NmzadOmDdHR0aSlpREWFsbhw4e5cOEC33///d0oo4iIiIiI3OfyXWPh6+vLwYMHady4MR06dCAlJYWePXuyf/9+HnroobtRRhERERERuc/lu8YCwNPTk8mTJxd2WUREREREpJj6W4FFUlISS5YsIS4uDpPJhI+PD/3796d8+fKFXT4RERERESkG8t0Uatu2bVSrVo333nuPpKQkLly4wHvvvUe1atXYtm3b3SijiIiIiIjc5/JdYzF06FB69erFggULsLW1BSAzM5MhQ4YwdOhQDh06VOiFFBERERGR+1u+ayxOnjzJ6NGjjaACwNbWllGjRnHy5MlCLZyIiIiIiBQP+Q4sGjZsSFxcXI70uLg46tevXxhlEhERERGRYiZPTaEOHjxo/H/YsGEMHz6cEydO0LRpUwB2797N+++/z4wZM+5OKUVERERE5L5mslgsljtlsrGxwWQycaesJpOJzMzMQitccXPx4kXMZjPJycmUKVPmnu8/IOCe7/KuiY4u6hJIcRawqGTcDNGDdSOIiEjRys/zbZ5qLE6dOlUoBRMRERERkZIpT4FFlSpV7nY5RERERESkGPtbE+SdO3eO77//nsTERLKysqyWDRs2rFAKJiIiIiIixUe+A4tly5bxr3/9CwcHB9zc3DCZTMYyk8mkwEJERERE5B8o34HFxIkTmThxIuPGjcPGJt+j1YqIiIiISAmU78jgypUrPP300woqRERERETEkO/oYODAgfznP/+5G2UREREREZFiKt+BxfTp09m2bRtt2rThpZdeYtSoUVZ/he3cuXM888wzuLm5UapUKerXr09MTIyx3GKxMGnSJLy8vHB2dqZNmzYcPnzYahupqam89NJLVKhQgdKlS9OtWzfOnj1rlScpKYnQ0FDMZjNms5nQ0FD++uuvQj8eEREREZGSKN99LKZNm8aGDRuoVasWQI7O24UpKSmJ5s2b07ZtW7799lvc3d05efIkZcuWNfLMmjWLt99+m+XLl/Pwww8zdepUOnTowLFjx3B1dQVgxIgRrFu3jvDwcNzc3Bg9ejTBwcHExMRga2sLQEhICGfPniUiIgKAwYMHExoayrp16wr1mERERERESqI8zbx9o3LlyjF37lz69et3l4r0P6+88grff/89O3bsyHW5xWLBy8uLESNGMHbsWOB67YSHhwczZ87khRdeIDk5mYoVK7Jy5Up69+4NwG+//Ya3tzfffPMNQUFBxMXF4evry+7du2nSpAkAu3fvJjAwkKNHjxpB1J1o5u3Co5m3pSA087aIiEjhKPSZt2/k6OhI8+bN/3bh8uOrr74iKCiIp556im3btvHAAw8wZMgQBg0aBFyfETwhIYGOHTtala9169ZERUXxwgsvEBMTQ3p6ulUeLy8v/Pz8iIqKIigoiF27dmE2m42gAqBp06aYzWaioqJuGVikpqaSmppqvL548SIAGRkZZGRkFOq5yAu7vzUryf2pCE6flCB2f2+KnvtOUXyOiIiI3Cg/30X5/vYdPnw48+bN47333svvqvn2888/s2DBAkaNGsWrr77K3r17GTZsGI6Ojjz77LMkJCQA4OHhYbWeh4cHv/zyCwAJCQk4ODhQrly5HHmy109ISMDd3T3H/t3d3Y08uZk+fTqTJ0/OkR4dHU3p0qXzd7CFoE+fe77Lu2bPnqIugRRnfcqVjJthj24EEREpYikpKXnOm+/AYu/evWzevJn169dTp04d7O3trZZ/+eWX+d3kLWVlZREQEMC0adMAaNCgAYcPH2bBggU8++yzRr6b+3ZYLJY79ve4OU9u+e+0nXHjxll1WL948SLe3t4EBAQUSVOo0aPv+S7vmqFDi7oEUpyNXloyboahPXQjiIhI0cpukZMX+Q4sypYtS8+ePfO72t9SqVIlfH19rdJ8fHz44osvAPD09ASu1zhUqlTJyJOYmGjUYnh6epKWlkZSUpJVrUViYiLNmjUz8vz+++859n/+/PkctSE3cnR0xNHRMUe6nZ0ddkXQLqkktZooSc265N7LoGTcDEXxOSIiInKj/HwX5ftba9myZfld5W9r3rw5x44ds0r76aefqFKlCgDVqlXD09OTyMhIGjRoAEBaWhrbtm1j5syZADRq1Ah7e3siIyPp1asXAPHx8Rw6dIhZs2YBEBgYSHJyMnv37qVx48bA9SYIycnJRvAhIiIiIiK3dl//HDZy5EiaNWvGtGnT6NWrF3v37mXRokUsWrQIuN58acSIEUybNo2aNWtSs2ZNpk2bRqlSpQgJCQHAbDYzcOBARo8ejZubG+XLl2fMmDH4+/vTvn174HotSKdOnRg0aBALFy4Erg83GxwcnOcRoURERERE/snyHVhUq1bttv0Ofv755wIV6EaPPPIIa9asYdy4cbzxxhtUq1aNd955h759+xp5wsLCuHr1KkOGDCEpKYkmTZqwceNGYw4LgLlz52JnZ0evXr24evUq7dq1Y/ny5cYcFgCrV69m2LBhxuhR3bp1Y/78+YV2LCIiIiIiJVm+57F49913rV6np6ezf/9+IiIiePnll3nllVcKtYDFieaxKDyax0IKQvNYiIiIFI67Oo/F8OHDc01///33idbToIiIiIjIP5JNYW3oscceM0ZrEhERERGRf5ZC67z9+eefU758+cLanIiIiBRESWkfq9YQIsVGvgOLBg0aWHXetlgsJCQkcP78eT744INCLZyIiIiIiBQP+Q4sunfvbvXaxsaGihUr0qZNG2rXrl1Y5RIRERERkWIk34HF66+/fjfKISIiIiIixVihdd4WEREREZF/rjzXWNjY2Nx2Yjy4PhN2RkZGgQslIiIiIiLFS54DizVr1txyWVRUFPPmzSOfc+2JiIiIiEgJkefA4vHHH8+RdvToUcaNG8e6devo27cvU6ZMKdTCiYiIiIhI8fC3+lj89ttvDBo0iLp165KRkUFsbCwrVqygcuXKhV0+EREREREpBvIVWCQnJzN27Fhq1KjB4cOH2bRpE+vWrcPPz+9ulU9ERERERIqBPDeFmjVrFjNnzsTT05NPP/0016ZRIiIiIiLyz5TnwOKVV17B2dmZGjVqsGLFClasWJFrvi+//LLQCiciIiIiIsVDngOLZ5999o7DzYqIiIiIyD9TngOL5cuX38ViiIiIiIhIcaaZt0VEREREpMAUWIiIiIiISIEpsBARERERkQJTYCEiIiIiIgWmwEJERERERApMgYWIiIiIiBSYAgsRERERESkwBRYiIiIiIlJgCixERERERKTAilVgMX36dEwmEyNGjDDSLBYLkyZNwsvLC2dnZ9q0acPhw4et1ktNTeWll16iQoUKlC5dmm7dunH27FmrPElJSYSGhmI2mzGbzYSGhvLXX3/dg6MSERERESn+ik1gsW/fPhYtWkTdunWt0mfNmsXbb7/N/Pnz2bdvH56ennTo0IFLly4ZeUaMGMGaNWsIDw9n586dXL58meDgYDIzM408ISEhxMbGEhERQUREBLGxsYSGht6z4xMRERERKc6KRWBx+fJl+vbty+LFiylXrpyRbrFYeOeddxg/fjw9e/bEz8+PFStWcOXKFT755BMAkpOTWbJkCW+99Rbt27enQYMGrFq1ih9//JHvvvsOgLi4OCIiIvjoo48IDAwkMDCQxYsXs379eo4dO1YkxywiIiIiUpwUi8Bi6NChdOnShfbt21ulnzp1ioSEBDp27GikOTo60rp1a6KiogCIiYkhPT3dKo+Xlxd+fn5Gnl27dmE2m2nSpImRp2nTppjNZiOPiIiIiIjcml1RF+BOwsPD+eGHH9i3b1+OZQkJCQB4eHhYpXt4ePDLL78YeRwcHKxqOrLzZK+fkJCAu7t7ju27u7sbeXKTmppKamqq8frixYsAZGRkkJGRkZfDK1R29/27mXdFcPqkBLG7/z/a8qQoPkekBCkpXwq6D0SKVH6+i+7rT51ff/2V4cOHs3HjRpycnG6Zz2QyWb22WCw50m52c57c8t9pO9OnT2fy5Mk50qOjoylduvRt93839Olzz3d51+zZU9QlkOKsT7mScTPs0Y0gBVFSvhR0H4gUqZSUlDznva8Di5iYGBITE2nUqJGRlpmZyfbt25k/f77R/yEhIYFKlSoZeRITE41aDE9PT9LS0khKSrKqtUhMTKRZs2ZGnt9//z3H/s+fP5+jNuRG48aNY9SoUcbrixcv4u3tTUBAAGXKlPmbR/33jR59z3d51wwdWtQlkOJs9NKScTMM7aEbQQqgpHwp6AtBpEhlt8jJi/s6sGjXrh0//vijVVr//v2pXbs2Y8eOpXr16nh6ehIZGUmDBg0ASEtLY9u2bcycOROARo0aYW9vT2RkJL169QIgPj6eQ4cOMWvWLAACAwNJTk5m7969NG7cGLj+S2FycrIRfOTG0dERR0fHHOl2dnbYFUEVdEmqLS4pNfhSNDIoGTdDUXyOSAlSUr4UdB+IFKn8fBfd13erq6srfn5+VmmlS5fGzc3NSB8xYgTTpk2jZs2a1KxZk2nTplGqVClCQkIAMJvNDBw4kNGjR+Pm5kb58uUZM2YM/v7+RmdwHx8fOnXqxKBBg1i4cCEAgwcPJjg4mFq1at3DIxYRERERKZ7u68AiL8LCwrh69SpDhgwhKSmJJk2asHHjRlxdXY08c+fOxc7Ojl69enH16lXatWvH8uXLsbW1NfKsXr2aYcOGGaNHdevWjfnz59/z4xERERERKY5MFovFUtSFKCkuXryI2WwmOTm5SPpYBATc813eNdHRRV0CKc4CFpWMmyF6sG4EKYCS8qWgLwSRIpWf59tiMY+FiIiIiIjc3xRYiIiIiIhIgSmwEBERERGRAlNgISIiIiIiBabAQkRERERECkyBhYiIiIiIFJgCCxERERERKTAFFiIiIiIiUmAKLEREREREpMAUWIiIiIiISIEpsBARERERkQJTYCEiIiIiIgWmwEJERERERApMgYWIiIiIiBSYAgsRERERESkwBRYiIiIiIlJgCixERERERKTAFFiIiIiIiEiBKbAQEREREZECU2AhIiIiIiIFpsBCREREREQKTIGFiIiIiIgUmAILEREREREpMAUWIiIiIiJSYAosRERERESkwBRYiIiIiIhIgd3XgcX06dN55JFHcHV1xd3dne7du3Ps2DGrPBaLhUmTJuHl5YWzszNt2rTh8OHDVnlSU1N56aWXqFChAqVLl6Zbt26cPXvWKk9SUhKhoaGYzWbMZjOhoaH89ddfd/sQRURERERKhPs6sNi2bRtDhw5l9+7dREZGkpGRQceOHUlJSTHyzJo1i7fffpv58+ezb98+PD096dChA5cuXTLyjBgxgjVr1hAeHs7OnTu5fPkywcHBZGZmGnlCQkKIjY0lIiKCiIgIYmNjCQ0NvafHKyIiIiJSXJksFoulqAuRV+fPn8fd3Z1t27bRqlUrLBYLXl5ejBgxgrFjxwLXayc8PDyYOXMmL7zwAsnJyVSsWJGVK1fSu3dvAH777Te8vb355ptvCAoKIi4uDl9fX3bv3k2TJk0A2L17N4GBgRw9epRatWrlqXwXL17EbDaTnJxMmTJl7s5JuI2AgHu+y7smOrqoSyDFWcCiknEzRA/WjSAFUFK+FPSFIFKk8vN8a3ePylQokpOTAShfvjwAp06dIiEhgY4dOxp5HB0dad26NVFRUbzwwgvExMSQnp5ulcfLyws/Pz+ioqIICgpi165dmM1mI6gAaNq0KWazmaioqFsGFqmpqaSmphqvL168CEBGRgYZGRmFd+B5ZFes3s3bK4LTJyWIXfH6aLulovgckRKkpHwp6D4QKVL5+S4qNp86FouFUaNG0aJFC/z8/ABISEgAwMPDwyqvh4cHv/zyi5HHwcGBcuXK5ciTvX5CQgLu7u459unu7m7kyc306dOZPHlyjvTo6GhKly6dj6MrHH363PNd3jV79hR1CaQ461OuZNwMe3QjSEGUlC8F3QciRerGLgh3UmwCixdffJGDBw+yc+fOHMtMJpPVa4vFkiPtZjfnyS3/nbYzbtw4Ro0aZby+ePEi3t7eBAQEFElTqNGj7/ku75qhQ4u6BFKcjV5aMm6GoT10I0gBlJQvBX0hiBSp7BY5eVEsAouXXnqJr776iu3bt/Pggw8a6Z6ensD1GodKlSoZ6YmJiUYthqenJ2lpaSQlJVnVWiQmJtKsWTMjz++//55jv+fPn89RG3IjR0dHHB0dc6Tb2dlhVwRV0CWptrik1OBL0cigZNwMRfE5IiVISflS0H0gUqTy8110X48KZbFYePHFF/nyyy/ZvHkz1apVs1perVo1PD09iYyMNNLS0tLYtm2bETQ0atQIe3t7qzzx8fEcOnTIyBMYGEhycjJ79+418uzZs4fk5GQjj4iIiIiI3Np9/TPA0KFD+eSTT/jvf/+Lq6ur0d/BbDbj7OyMyWRixIgRTJs2jZo1a1KzZk2mTZtGqVKlCAkJMfIOHDiQ0aNH4+bmRvny5RkzZgz+/v60b98eAB8fHzp16sSgQYNYuHAhAIMHDyY4ODjPI0KJiIiIiPyT3deBxYIFCwBo06aNVfqyZcvo168fAGFhYVy9epUhQ4aQlJREkyZN2LhxI66urkb+uXPnYmdnR69evbh69Srt2rVj+fLl2NraGnlWr17NsGHDjNGjunXrxvz58+/uAYqIiIiIlBDFah6L+53msSg8GrZcCkLzWIhQcr4U9IUgUqTy83x7X/exEBERERGR4kGBhYiIiIiIFJgCCxERERERKTAFFiIiIiIiUmAKLEREREREpMAUWIiIiIiISIEpsBARERERkQJTYCEiIiIiIgV2X8+8LSIiIiLyd0VHl4yJIgMCisdEkaqxEBERERGRAlNgISIiIiIiBabAQkRERERECkyBhYiIiIiIFJgCCxERERERKTAFFiIiIiIiUmAKLEREREREpMAUWIiIiIiISIEpsBARERERkQJTYCEiIiIiIgWmwEJERERERApMgYWIiIiIiBSYAgsRERERESkwBRYiIiIiIlJgCixERERERKTAFFiIiIiIiEiBKbC4yQcffEC1atVwcnKiUaNG7Nixo6iLJCIiIiJy31NgcYPPPvuMESNGMH78ePbv30/Lli157LHHOHPmTFEXTURERETkvqbA4gZvv/02AwcO5Pnnn8fHx4d33nkHb29vFixYUNRFExERERG5rymw+D9paWnExMTQsWNHq/SOHTsSFRVVRKUSERERESke7Iq6APeLP/74g8zMTDw8PKzSPTw8SEhIyHWd1NRUUlNTjdfJyckAXLhwgYyMjLtX2H+ACxeKugRSrF0t6gIUjgu6EUT0hSAFcvFiUZegcBTl98HF/zuJFovljnkVWNzEZDJZvbZYLDnSsk2fPp3JkyfnSK9WrdpdKds/iZtbUZdApOi5jdCNIKIvBBGAor8PLl26hNlsvm0eBRb/p0KFCtja2uaonUhMTMxRi5Ft3LhxjBo1ynidlZXFhQsXcHNzu2UwIveHixcv4u3tza+//kqZMmWKujhSzOj6kYLQ9SN/l64dKQoWi4VLly7h5eV1x7wKLP6Pg4MDjRo1IjIykh49ehjpkZGRPP7447mu4+joiKOjo1Va2bJl72YxpZCVKVNGH87yt+n6kYLQ9SN/l64dudfuVFORTYHFDUaNGkVoaCgBAQEEBgayaNEizpw5w7/+9a+iLpqIiIiIyH1NgcUNevfuzZ9//skbb7xBfHw8fn5+fPPNN1SpUqWoiyYiIiIicl9TYHGTIUOGMGTIkKIuhtxljo6OvP766zmasonkha4fKQhdP/J36dqR+53Jkpexo0RERERERG5DE+SJiIiIiEiBKbAQEREREZECU2AhIiIiIiIFpsBCREREREQKTIGFiIiIiIgUmAILEREREREpMAUWIiIiIiJSYAosRERERESkwBRYiIjIfalfv3507969qIshIiJ5pMBCRET+tn79+mEymTCZTNjb21O9enXGjBlDSkpKURdNRETuMbuiLoCIiBRvnTp1YtmyZaSnp7Njxw6ef/55UlJSWLBgQVEXTURE7iHVWIiISIE4Ojri6emJt7c3ISEh9O3bl7Vr17Jq1SoCAgJwdXXF09OTkJAQEhMTrdY9fPgwXbp0oUyZMri6utKyZUtOnjyZ635iYmJwd3fnzTffBCAiIoIWLVpQtmxZ3NzcCA4OzrFuVFQU9evXx8nJiYCAANauXYvJZCI2NtbIc+TIETp37oyLiwseHh6Ehobyxx9/GMs///xz/P39cXZ2xs3Njfbt26tGRkQkFwosRESkUDk7O5Oenk5aWhpTpkzhwIEDrF27llOnTtGvXz8j37lz52jVqhVOTk5s3ryZmJgYBgwYQEZGRo5tbt26lXbt2jF58mTGjx8PQEpKCqNGjWLfvn1s2rQJGxsbevToQVZWFgCXLl2ia9eu+Pv788MPPzBlyhTGjh1rtd34+Hhat25N/fr1iY6OJiIigt9//51evXoZy/v06cOAAQOIi4tj69at9OzZE4vFcpfOnohI8aWmUCIiUmj27t3LJ598Qrt27RgwYICRXr16dd577z0aN27M5cuXcXFx4f3338dsNhMeHo69vT0ADz/8cI5t/ve//yU0NJSFCxfSp08fI/2JJ56wyrdkyRLc3d05cuQIfn5+rF69GpPJxOLFi3FycsLX15dz584xaNAgY50FCxbQsGFDpk2bZqQtXboUb29vfvrpJy5fvkxGRgY9e/akSpUqAPj7+xfOyRIRKWFUYyEiIgWyfv16XFxccHJyIjAwkFatWjFv3jz279/P448/TpUqVXB1daVNmzYAnDlzBoDY2FhatmxpBBW52bNnD0888QQrVqywCioATp48SUhICNWrV6dMmTJUq1bNavvHjh2jbt26ODk5Ges0btzYahsxMTFs2bIFFxcX46927drG9uvVq0e7du3w9/fnqaeeYvHixSQlJRXshImIlFAKLEREpEDatm1LbGwsx44d49q1a3z55ZeULl2ajh074uLiwqpVq9i3bx9r1qwBIC0tDbjeZOpOHnroIWrXrs3SpUuN9bJ17dqVP//8k8WLF7Nnzx727NljtX2LxYLJZLJa5+YmTFlZWXTt2pXY2Firv+PHj9OqVStsbW3/f3t3EwpbHMZx/DcWs5kaaZCXQpqYmSaJpWRhIfZehhULG0psLExTUprCanZsREopCxZCnQVZ0JG3NCOMspryUmQjxtzVndvk3rrdw4L7/dTZnJ7/WfxX59dznv/R5uam1tbW5PP5FIlEVFlZqaurq3/bLAD4xggWAABLHA6H3G63SktL092HWCym29tbhcNh1dfXy+PxvBvcrqqq0vb2tl5eXv747NzcXBmGocvLS7W3t6dr7+7uFI1GFQwG1djYKK/X+66T4PF4dHx8rOfn5/Q90zQzampqanR6eqqysjK53e6My+FwSJJsNpvq6uo0Ojqqg4MD2e32dEgCAPxCsAAAfLiSkhLZ7XZFIhHF43GtrKxobGwso6a/v1+Pj4/q6OiQaZo6Pz/X/Py8zs7OMury8/NlGIZisZgCgYBeX1+Vk5Mjl8ul6elpXVxcyDAMDQ0NZazr7OzU29ubent7FY1Gtb6+rsnJSUlKdzL6+vp0f3+vQCCgvb09xeNxbWxsqKenR8lkUru7uxofH5dpmrq+vtby8rJubm7k9Xo/cfcA4GsiWAAAPlxeXp5mZ2e1tLQkn8+ncDicfqn/yeVyyTAMPT09qaGhQbW1tZqZmfntzEVBQYEMw9DJyYm6urqUSqW0uLio/f19+f1+DQ4OamJiImON0+nU6uqqDg8PVV1drZGREYVCIUlKz10UFRVpZ2dHyWRSTU1N8vv9GhgYUHZ2trKysuR0OrW1taWWlhZVVFQoGAxqampKzc3Nn7RzAPB12VKcmQcA+E8sLCyou7tbDw8PfzXjAQD4exw3CwD4tubm5lReXq7i4mIdHR1peHhYbW1thAoA+AQECwDAt5VIJBQKhZRIJFRYWKjW1tb0n7sBAB+LT6EAAAAAWMbwNgAAAADLCBYAAAAALCNYAAAAALCMYAEAAADAMoIFAAAAAMsIFgAAAAAsI1gAAAAAsIxgAQAAAMAyggUAAAAAy34A/LtDMs37jEgAAAAASUVORK5CYII=",
      "text/plain": [
       "<Figure size 800x400 with 1 Axes>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "# Data to plot as a histogram\n",
    "data1 = [len(total_packages), 0, 0, 0]\n",
    "data2 = [0, len(dependency_packages | packages_with_dependencies), 0, 0]\n",
    "data3 = [0, 0, len(dependency_packages), 0]\n",
    "data4 = [0, 0, 0, len(packages_without_dependencies)]\n",
    "\n",
    "fig, ax = plt.subplots(figsize=(8, 4))\n",
    "index = np.arange(4)\n",
    "bar_width = 0.35\n",
    "opacity = 0.8\n",
    "rects1 = plt.bar(index, data1, bar_width, alpha=opacity, color='b', label='Total packages')\n",
    "rects2 = plt.bar(index+bar_width, data2, bar_width, alpha=opacity, color='g', label='Packages with dependencies')\n",
    "rects3 = plt.bar(index+2*bar_width, data3, bar_width, alpha=opacity, color='r', label='Packages that are dependencies of others')\n",
    "rects4 = plt.bar(index+3*bar_width, data4, bar_width, alpha=opacity, color='y', label='Packages without dependencies')\n",
    "plt.xlabel('Packages')\n",
    "plt.ylabel('Number of packages')\n",
    "plt.title('Packages')\n",
    "plt.xticks(index + bar_width, ())\n",
    "\n",
    "# Add a grid    \n",
    "plt.grid(axis='y', alpha=0.75)\n",
    "ax.legend(loc='upper right')\n",
    "plt.tight_layout()\n",
    "plt.show()\n"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "---"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# <ins>**2 - Web Scraping**</ins>"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## **2.1 - Implementation Details**"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "**ProxyRequest module**\n",
    "\n",
    "To carry out the task of obtaining data through web scraping, the module <span style=\"color:green; font-weight:bold\">ProxyRequest</span> with the class <span style=\"color:green; font-weight:bold\">RequestHandler</span> has been implemented to make the operation more transparent.\n",
    "\n",
    "\n",
    "The RequestHandler module allows us to make requests using different *proxies and useragents* with the aim of avoiding the denial of service by the server to our requests, which will happen if or if due to their high number.\n",
    "\n",
    "The implementation of the class is not defined in this notebook due to not extending the contents much.\n",
    "\n",
    "You can see the implementation of the module and more information at:\n",
    "\n",
    "- URL\n",
    "\n"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "**R_Scraper module**\n",
    "\n",
    "This module has been implemented with the objective of extracting the information of the packages housed on the **CRAN** & **Bioconductor** website.\n",
    "\n",
    "This websites contains a list of packages currently availables, as well as the packages that have been filed (orphanage or by express desire for the maintainer)\n",
    "\n",
    "The **Cran_Scraper** & **Bioc_Scraper** classes implemented allows us to extract the information stored in the HTML and process it with the aim of generating an object of the package class that we have previously implemented.\n",
    "\n",
    "You can see the implementation of the module in the next URL:\n",
    "\n",
    "- URL\n",
    "- URL"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## **2.2 - Scraping process**"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### **2.2.1 - Package processing** "
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "We do a test getting a package"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Name: MASS\n",
      "Description:\n",
      "Functions and datasets to support Venables and Ripley, \"Modern Applied Statistics with S\" (4th edition, 2002).\n",
      "Version: 7.3-58.2\n",
      "Publication date: 2023-01-23\n",
      "Mantainer: Brian Ripley <ripley@stats.ox.ac.uk>\n",
      "Authors:\n",
      "Brian Ripley [aut, cre, cph], Bill Venables [ctb], Douglas M. Bates [ctb], Kurt Hornik [trl] (partial port ca 1998), Albrecht Gebhardt [trl] (partial port ca 1998), David Firth [ctb]\n",
      "Dependencies:\n",
      "  - R (≥ 3.3.0) type: depends\n",
      "  - grDevices, type: depends\n",
      "  - graphics, type: depends\n",
      "  - stats, type: depends\n",
      "  - utils, type: depends\n",
      "  - methods, type: imports\n",
      "Requires compilation: True\n",
      "In CRAN: True\n",
      "In Bioconductor: None\n",
      "Licenses:\n",
      "GPL-2 | GPL-3\n",
      "Links:\n",
      "https://cran.r-project.org/package=MASS\n",
      "\n"
     ]
    }
   ],
   "source": [
    "cran_scraper = Cran_Scraper(RequestHandler())       # Create object of class CranScraper\n",
    "p = cran_scraper.build('MASS')                      # Get the MASS package data\n",
    "p.dump()                                            # Print the package data                     "
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "As can be seen thanks to this implementation, it is now quite simple to obtain an initialized package object with the website data of the repository to which it belongs.\n",
    "\n",
    "The only thing that should know his name."
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### **2.2.2 - Scraping CRAN website**"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Once all the tools have been obtained, the only thing left is to obtain the information of each of the packages."
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "#### **2.2.2.1 - Get a list of packages**"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "**<span style=\"color:yellow\">Get data from cran.r-project.org</span>**"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "We make a request to the CRAN web server and get the list of packages"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Request successful\n"
     ]
    }
   ],
   "source": [
    "# URL to get the list of packages from CRAN\n",
    "CRAN_pkgs_url = \"https://cran.r-project.org/web/packages/available_packages_by_name.html\"\n",
    "\n",
    "# Create object of class RequestHandler\n",
    "rh = RequestHandler()\n",
    "\n",
    "# Do the request\n",
    "response = rh.do_request(CRAN_pkgs_url, retry=True)\n",
    "\n",
    "# If the request was successful\n",
    "if response.status_code == 200:\n",
    "    print(\"Request successful\")\n",
    "else:\n",
    "    print(\"Error in request\")"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "If the request was correct, we will begin to extract the information we need"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "**<span style=\"color:yellow\">Get package names via scraping</span>**"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "We will use the BeautifullSoup4 library to do the scraping"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [],
   "source": [
    "# We parse the HTML content of the web page using BeautifulSoup\n",
    "# We extract the table that contains the information of the CRAN packages\n",
    "soup = BeautifulSoup(response.content, \"html.parser\")\n",
    "table = soup.find(\"table\")\n",
    "rows = table.find_all(\"tr\")"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "As you can see we have the information stored in a table, now we iterate each row to extract the name of the packages"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[<tr id=\"available-packages-A\"> <td></td> </tr>,\n",
       " <tr>\n",
       " <td><a href=\"../../web/packages/A3/index.html\">A3</a></td><td>Accurate, Adaptable, and Accessible Error Metrics for Predictive\n",
       " Models</td></tr>,\n",
       " <tr>\n",
       " <td><a href=\"../../web/packages/AATtools/index.html\">AATtools</a></td><td>Reliability and Scoring Routines for the Approach-Avoidance Task</td></tr>,\n",
       " <tr>\n",
       " <td><a href=\"../../web/packages/ABACUS/index.html\">ABACUS</a></td><td>Apps Based Activities for Communicating and Understanding\n",
       " Statistics</td></tr>,\n",
       " <tr>\n",
       " <td><a href=\"../../web/packages/abbreviate/index.html\">abbreviate</a></td><td>Readable String Abbreviation</td></tr>]"
      ]
     },
     "execution_count": 4,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "rows[:5]"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "The data could be pre-processed to remove unwanted debris, although we will handle this later."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "<tr id=\"available-packages-A\"> <td></td> </tr>"
      ]
     },
     "execution_count": 5,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "rows[0]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "<tr id=\"available-packages-A\"> <td></td> </tr>"
      ]
     },
     "execution_count": 6,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "rows.pop(0)"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "We import the Packages module, whose implementation is documented in the 'Data storage and Persistence' notebook\n",
    "\n",
    "We also import the Util module, which contains code to garnish the stdout output (Nothing relevant)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "packages = []\n",
    "\n",
    "# We iterate over each row of the table\n",
    "for row in rows:\n",
    "\n",
    "    # We extract the cells of the current row\n",
    "    cells = row.find_all(\"td\")\n",
    "\n",
    "    package_name = \"\"\n",
    "    # If there are cells in the row\n",
    "    if cells:\n",
    "        try: \n",
    "            # We extract the name of the package\n",
    "            # The name is in the first cell of the row\n",
    "            package_name = cells[0].find(\"a\").text   \n",
    "\n",
    "            # We add the package name to the list of packages\n",
    "            packages.append(package_name)\n",
    "\n",
    "            # Show success message\n",
    "            message = \"Package \" + package_name + \" added to the list\"\n",
    "            print_colored(message, GREEN)\n",
    "            \n",
    "\n",
    "        # If an error occurs, we show the error message\n",
    "        except Exception as e:\n",
    "            \n",
    "            # message string\n",
    "            message = \"Error processing package: \" + package_name\n",
    "            message += \"Exception: \" + e.__class__.__name__ \n",
    "            message += \"Error: \" + str(e)\n",
    "\n",
    "            # Print error message\n",
    "            print_colored(message, RED)\n",
    "            print(\"Continuing...\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Number of packages processed:  19248\n"
     ]
    }
   ],
   "source": [
    "# We show the number of packages that we have processed\n",
    "print(\"Number of packages processed: \", len(packages))"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "**<span style=\"color:yellow\">Store package names in a file</span>**\n",
    "\n",
    "We can store the list of packages in a file so as not to repeat this step more times unless we want to update. \n",
    "<span style=\"color:red;\">Be careful not to crush the file!</span>"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "# file_name = \"data\" + os.sep + \"CRAN_package_name_list.pkl\"\n",
    "\n",
    "# # Drop file if it exists\n",
    "# if os.path.exists(file_name):\n",
    "#     os.remove(file_name)\n",
    "\n",
    "# # We open the file in write mode\n",
    "# with open(file_name, \"wb\") as f:\n",
    "#     # We dump the packages list in the file\n",
    "#     pickle.dump(packages, f)"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "**<span style=\"color:yellow\">Inspect the data</span>**"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Check that the package file has been properly generated"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['dependencies-1.6.0-2020-01-12-cran.csv',\n",
       " 'bioconductor_scraped_dependencies.csv',\n",
       " 'bioconductor_network.png',\n",
       " 'bioconductor_dependencies_net.bz2',\n",
       " 'CRAN_scraped_dependencies.csv',\n",
       " 'packages_20230221121430.pkl',\n",
       " 'bioconductor_raw.html',\n",
       " 'bioconductor_package_name_list.pkl',\n",
       " 'bioconductor_selenium.html',\n",
       " 'CRAN_package_object_list.pkl',\n",
       " 'bioconductor_package_object_list.pkl',\n",
       " 'CRAN_package_name_list.pkl',\n",
       " 'CRAN_dependencies_net.bz2',\n",
       " 'projects-1.6.0-2020-01-12-cran.csv']"
      ]
     },
     "execution_count": 3,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "os.listdir(\"data\")"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "We are going to show some basic statistics to get a little better idea of ​​the data we have."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Compare the list of packages obtained by scraping with the list of packages obtained by the libraries.io dataset\n",
    "# -------------------------------------------------------------------------------------------------------------\n",
    "\n",
    "file_name = \"data\" + os.sep + \"CRAN_package_name_list.pkl\"\n",
    "\n",
    "# We load the list of packages obtained by scraping\n",
    "with open(file_name, \"rb\") as f:\n",
    "    packages_scraping = pickle.load(f)\n",
    "\n",
    "# We load the list of packages obtained by the libraries.io dataset, from the csv file\n",
    "packages_librariesio = pd.read_csv(\"data\" + os.sep + \"projects-1.6.0-2020-01-12-cran.csv\")\n",
    "\n",
    "# We get the names of the packages obtained by scraping\n",
    "packages_scraping_names = set(packages_scraping)\n",
    "\n",
    "# We get the names of the packages obtained by the libraries.io dataset\n",
    "packages_librariesio_names = set(packages_librariesio['Name'].values)\n",
    "\n",
    "# We get the packages that are in the libraries.io dataset, but not in the scraping dataset\n",
    "packages_librariesio_not_in_scraping = packages_librariesio_names - packages_scraping_names\n",
    "\n",
    "# We get the packages that are in the scraping dataset, but not in the libraries.io dataset\n",
    "packages_scraping_not_in_librariesio = packages_scraping_names - packages_librariesio_names"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Number of packages in the libraries.io dataset:  16695\n",
      "Number of packages in the scraping dataset:  19248\n",
      "Number of packages in the libraries.io dataset, but not in the scraping dataset:  4091\n",
      "Number of packages in the scraping dataset, but not in the libraries.io dataset:  6644\n",
      "Number of packages in both datasets:  12604\n"
     ]
    }
   ],
   "source": [
    "print(\"Number of packages in the libraries.io dataset: \", len(packages_librariesio_names))\n",
    "print(\"Number of packages in the scraping dataset: \", len(packages_scraping_names))\n",
    "print(\"Number of packages in the libraries.io dataset, but not in the scraping dataset: \", len(packages_librariesio_not_in_scraping))\n",
    "print(\"Number of packages in the scraping dataset, but not in the libraries.io dataset: \", len(packages_scraping_not_in_librariesio))\n",
    "\n",
    "# We show the number of packages that are in both datasets\n",
    "print(\"Number of packages in both datasets: \", len(packages_scraping_names & packages_librariesio_names))"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "We create a bar graph to visualize the size of the repositories"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 28,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "image/png": "iVBORw0KGgoAAAANSUhEUgAAAk4AAAGGCAYAAACNCg6xAAAAOXRFWHRTb2Z0d2FyZQBNYXRwbG90bGliIHZlcnNpb24zLjMuNCwgaHR0cHM6Ly9tYXRwbG90bGliLm9yZy8QVMy6AAAACXBIWXMAAA9hAAAPYQGoP6dpAABVYElEQVR4nO3dfVxO9+M/8Ncl3XdVihSlIiFS1Fjldkxzt2gTZZuWzYw298NQkmHG8DEz6+v+3jYaZmiSu5JK5aYkFI0aQrlLd+/fH36dj0vJqaUbn9fz8bgej65z3ud93u9zrptX533OuRRCCAEiIiIieql6Nd0AIiIiorqCwYmIiIhIJgYnIiIiIpkYnIiIiIhkYnAiIiIikonBiYiIiEgmBiciIiIimRiciIiIiGRicCIiIiKSicGJiOqMdevWQaFQSI/69evD3NwcH3/8Ma5fv17l6+vRowfatWtX5fUSUd1Vv6YbQERUUWvXrkXr1q3x+PFjHD16FPPnz8eRI0dw9uxZ6Orq1nTziOg1xuBERHVOu3bt4OzsDADo2bMnioqKEBwcjNDQUAwfPryGW0dErzMO1RFRnffmm28CAK5evYqgoCB07twZRkZG0NfXR8eOHbF69WqU9XvmW7ZsgYuLC/T09KCnpwdHR0esXr263HXt2rULOjo6+OSTT1BYWIi8vDxMmjQJjo6OMDAwgJGREVxcXPD777+XWvbevXsYOXIkjIyMoKenh/79++PKlStQKBSYPXu2StnU1FT4+PjAxMQEmpqaaNOmDVasWKFSpri4GHPnzkWrVq2gra0NQ0NDtG/fHsuWLavgFiQiuXjEiYjqvEuXLgEAGjVqhMjISHz22Wdo1qwZAODkyZP44osvcP36dQQEBEjLBAQEIDg4GJ6enpg0aRIMDAxw7tw5XL169YXrWbJkCaZMmYLZs2dj5syZAICHDx/izp07mDx5Mpo2bYr8/Hz89ddf8PT0xNq1a/HRRx8BeBpyBg4ciNjYWMyePRsdO3ZEVFQU3nnnnVLrSUpKgqurK5o1a4bFixfD1NQUBw4cwJdffonbt28jMDAQALBw4UKpLd26dUNBQQEuXLiAe/fuVcl2JaIyCCKiOmLt2rUCgDh58qQoKCgQ9+/fF3v37hWNGjUSSqVSZGVlqZQvKioSBQUFYs6cOcLY2FgUFxcLIYS4cuWKUFNTE8OHDy93fd27dxdt27YVRUVFwt/fX2hoaIhNmzaVu0xhYaEoKCgQI0eOFB06dJCm//HHHwKAWLlypUr5+fPnCwAiMDBQmubu7i7Mzc1FTk6OSll/f3+hpaUl7ty5I4QQYsCAAcLR0bHc9hBR1eJQHRHVOW+++SbU1dWhVCoxYMAAmJqa4s8//0Tjxo0RHh6O3r17w8DAAGpqalBXV0dAQACys7Nx8+ZNAEBYWBiKioowduzYl64rLy8PgwYNwubNm3Hw4MEyz6H65Zdf4ObmBj09PdSvXx/q6upYvXo1kpOTpTJHjhwBAHh5eaks6+3tXWp9hw4dwuDBg6Gjo4PCwkLp0a9fP+Tl5eHkyZMAgE6dOiExMRFjxozBgQMHkJubW7ENSUQVxuBERHXOhg0bEBMTg/j4eNy4cQNnzpyBm5sbTp06hT59+gAAQkJCcOLECcTExGDGjBkAgMePHwMAbt26BQAwNzd/6bpu3ryJAwcOwMXFBa6urqXm79y5E15eXmjatCk2bdqEqKgoxMTEwM/PD3l5eVK57Oxs1K9fH0ZGRirLN27cWOV5dnY2CgsLsXz5cqirq6s8+vXrBwC4ffs2AGD69OlYtGgRTp48ib59+8LY2Bi9evVCbGysrO1IRBXHc5yIqM5p06aNdFXds7Zt2wZ1dXXs3bsXWlpa0vTQ0FCVco0aNQIA/P3337CwsCh3Xc2aNcP333+PwYMHw9PTE7/88otK3Zs2bYK1tTW2b98OhUIhTX/y5IlKPcbGxigsLMSdO3dUwlNWVpZKuQYNGkBNTQ0ffvjhC4+IWVtbAwDq16+PiRMnYuLEibh37x7++usvfP3113B3d0dGRgZ0dHTK7RsRVRyPOBHRa6PkpphqamrStMePH2Pjxo0q5fr06QM1NTWsXLlSVr19+vTBgQMHcPToUQwYMAAPHz5UWaeGhoZKaMrKyip1VV337t0BANu3b1eZvm3bNpXnOjo66NmzJ+Lj49G+fXs4OzuXehgbG5dqo6GhId5//32MHTsWd+7cQXp6uqy+EVHF8IgTEb02+vfvj++//x4+Pj4YNWoUsrOzsWjRImhqaqqUs7Kywtdff43g4GA8fvwY3t7eMDAwQFJSEm7fvo2goKBSdXfp0gWHDh3CO++8gz59+mDfvn0wMDDAgAEDsHPnTowZMwbvv/8+MjIyEBwcDDMzM6SmpkrLv/POO3Bzc8OkSZOQm5sLJycnREVFYcOGDQCAevX++3/ssmXL0KVLF3Tt2hWff/45rKyscP/+fVy6dAl79uxBeHg4AGDgwIHSPa0aNWqEq1evYunSpbC0tETLli1fxSYmopo+O52ISK6Sq+piYmJeWGbNmjWiVatWQlNTUzRv3lzMnz9frF69WgAQaWlpKmU3bNgg3njjDaGlpSX09PREhw4dxNq1a6X5JVfVPevcuXPC1NRUdOzYUdy6dUsIIcSCBQuElZWV0NTUFG3atBEhISEiMDBQPP8Re+fOHfHxxx8LQ0NDoaOjI95++21x8uRJAUAsW7ZMpWxaWprw8/MTTZs2Ferq6qJRo0bC1dVVzJ07VyqzePFi4erqKho2bCg0NDREs2bNxMiRI0V6enpFNisRVYBCiDLuCkdERNViy5YtGD58OE6cOFHmyedEVLswOBERVZOtW7fi+vXrsLe3R7169XDy5El899136NChg3S7AiKq3XiOExFRNVEqldi2bRvmzp2Lhw8fwszMDL6+vpg7d25NN42IZOIRJyIiIiKZeDsCIiIiIpkYnIiIiIhkYnAiIiIikoknh9dhxcXFuHHjBpRKpcpdi4mIiP5XCCFw//59NGnSROVGsq8Kg1MdduPGjZf+zhYREdH/goyMDFk/3P1vMTjVYUqlEsDTF4u+vn4Nt4aIiKj65ebmwsLCQvpOfNUYnOqwkuE5fX19BiciIvqfVl2nrPDkcCIiIiKZGJyIiIiIZGJwIiIiIpKJwYmIiIhIJgYnIiIiIpkYnIiIiIhkYnAiIiIikonBiYiIiEgmBiciIiIimRiciIiIiGRicCIiIiKSib9V9xrovrY71LTVaroZ9BqIHRVb000gIqrVeMSJiIiISCYGJyIiIiKZGJyIiIiIZGJwIiIiIpKJwYmIiIhIJgYnIiIiIpkYnIiIiIhkYnAiIiIikonBiYiIiEgmBiciIiIimRiciIiIiGRicCIiIiKSicGJiIiISCYGJyIiIiKZGJyIiIiIZGJwIiIiIpKJwYmIiIhIJgYnIiIiIpkYnIiIiIhkYnAiIiIikonBiYiIiEgmBiciIiIimRicqkiPHj0wfvz4mm4GERERvUL1a7oBr4udO3dCXV29pptBREREr9D/dHAqKCiosrBjZGRUJfUQERFR7VUnh+p+/fVX2NvbQ1tbG8bGxujduzcePnwIAFizZg3atm0LTU1NmJmZwd/fX1pOoVDgp59+goeHB3R1dTF37lwUFRVh5MiRsLa2hra2Nlq1aoVly5aprM/X1xeDBg1CUFAQTExMoK+vj88++wz5+flSmeeH6qysrDBv3jz4+flBqVSiWbNm+Pnnn1XqjYyMhKOjI7S0tODs7IzQ0FAoFAokJCRU/UYjIiKif63OBafMzEx4e3vDz88PycnJiIiIgKenJ4QQWLlyJcaOHYtRo0bh7Nmz2L17N2xsbFSWDwwMhIeHB86ePQs/Pz8UFxfD3NwcO3bsQFJSEgICAvD1119jx44dKssdOnQIycnJOHz4MLZu3Ypdu3YhKCio3LYuXrwYzs7OiI+Px5gxY/D555/jwoULAID79+9j4MCBsLe3x+nTpxEcHIypU6dW7cYiIiKiKlXnhuoyMzNRWFgIT09PWFpaAgDs7e0BAHPnzsWkSZMwbtw4qfwbb7yhsryPjw/8/PxUpj0bgKytrREZGYkdO3bAy8tLmq6hoYE1a9ZAR0cHbdu2xZw5czBlyhQEBwejXr2y82e/fv0wZswYAMDUqVOxZMkSREREoHXr1ti8eTMUCgVCQkKgpaUFOzs7XL9+HZ9++ukL+/7kyRM8efJEep6bm1vutiIiIqKqVeeOODk4OKBXr16wt7fHkCFDEBISgrt37+LmzZu4ceMGevXqVe7yzs7Opab99NNPcHZ2RqNGjaCnp4eQkBBcu3at1Hp1dHSk5y4uLnjw4AEyMjJeuK727dtLfysUCpiamuLmzZsAgJSUFLRv3x5aWlpSmU6dOpXb9vnz58PAwEB6WFhYlFueiIiIqladC05qamoICwvDn3/+CTs7OyxfvhytWrXCP//8I2t5XV1dlec7duzAhAkT4Ofnh4MHDyIhIQEff/yxyvlL5VEoFC+c9/yJ5wqFAsXFxQAAIUSpZYUQ5a5r+vTpyMnJkR7lhTYiIiKqenUuOAFPA4ibmxuCgoIQHx8PDQ0NhIWFwcrKCocOHapQXceOHYOrqyvGjBmDDh06wMbGBpcvXy5VLjExEY8fP5aenzx5Enp6ejA3N69UH1q3bo0zZ86oDL3FxsaWu4ympib09fVVHkRERFR96lxwio6Oxrx58xAbG4tr165h586duHXrFtq0aYPZs2dj8eLF+M9//oPU1FScPn0ay5cvL7c+GxsbxMbG4sCBA7h48SJmzZqFmJiYUuXy8/MxcuRIJCUl4c8//0RgYCD8/f1feH7Ty/j4+KC4uBijRo1CcnIyDhw4gEWLFgEo/ygWERER1Zw6d3K4vr4+jh49iqVLlyI3NxeWlpZYvHgx+vbtCwDIy8vDkiVLMHnyZDRs2BDvv/9+ufWNHj0aCQkJGDp0KBQKBby9vTFmzBj8+eefKuV69eqFli1bolu3bnjy5AmGDRuG2bNn/6t+7NmzB59//jkcHR1hb2+PgIAA+Pj4qJz3RERERLWHQrzsxBqCr68v7t27h9DQ0Fe6ns2bN+Pjjz9GTk4OtLW1X1o+NzcXBgYGcFzqCDVttVfaNvrfEDuq/OFiIqLapuS7MCcnp1pOYalzR5xeJxs2bEDz5s3RtGlTJCYmYurUqfDy8pIVmoiIiKj6MTjVoKysLAQEBCArKwtmZmYYMmQIvvnmm5puFhEREb0Ah+rqMA7VUVXjUB0R1TXVPVRX566qIyIiIqopDE5EREREMjE4EREREcnE4EREREQkE4MTERERkUwMTkREREQyMTgRERERycTgRERERCQTgxMRERGRTAxORERERDIxOBERERHJxOBEREREJBODExEREZFMDE5EREREMjE4EREREcnE4EREREQkE4MTERERkUwMTkREREQyMTgRERERycTgRERERCRT/ZpuAFWBdUcANf2abgW9Bpx/rukWENHrLDa2plvw7/GIExEREZFMDE5EREREMjE4EREREcnE4EREREQkE4MTERERkUwMTkREREQyMTgRERERycTgRERERCQTgxMRERGRTAxORERERDIxOBERERHJxOBEREREJBODExEREZFMDE5EREREMjE4EREREcnE4EREREQkE4MTERERkUwMTkREREQyMTgRERERycTgRERERCQTgxMRERGRTAxORERERDIxOBERERHJxOBEREREJBODExEREZFMDE5EREREMjE4EREREcnE4EREREQkE4MTERERkUwMTkREREQyMTgRERERycTgRERERCQTgxMRERGRTAxORERERDIxOBERERHJxOBEREREJBODExEREZFMDE5EREREMjE4EREREcnE4EREREQkE4MTERERkUwMTkREREQyMTgRERERycTgRERERCQTgxMRERGRTAxORERERDIxOBERERHJxOBEREREJBODExEREZFMDE5EREREMjE4EREREcnE4EREREQkU5UFp3v37lVVVURERES1UqWC07fffovt27dLz728vGBsbIymTZsiMTGxyhpHREREVJtUKjitWrUKFhYWAICwsDCEhYXhzz//RN++fTFlypQqbSARERFRbVG/MgtlZmZKwWnv3r3w8vJCnz59YGVlhc6dO1dpA4mIiIhqi0oFpwYNGiAjIwMWFhbYv38/5s6dCwAQQqCoqKhKG0gvd+QIoK9f060gIiJ6/VUqOHl6esLHxwctW7ZEdnY2+vbtCwBISEiAjY1NlTaQ6FUTQqCwsJChn4ioFlJTU0P9+vWhUChquikAKhmclixZAisrK2RkZGDhwoXQ09MD8HQIb8yYMVXaQKJXKT8/H5mZmXj06FFNN4WIiF5AR0cHZmZm0NDQqOmmQCGEEBVd6OjRo3B1dUX9+qq5q7CwEJGRkejWrVuVNZBeLDc3FwYGBsjJyYE+x+oqrLi4GKmpqVBTU0OjRo2goaFRa/6jISKipyMC+fn5uHXrFoqKitCyZUvUq6d6XVt1fxdW6ohTz549kZmZCRMTE5XpOTk56NmzJ4c8qE7Iz89HcXExLCwsoKOjU9PNISKiMmhra0NdXR1Xr15Ffn4+tLS0arQ9lbodgRCizP/Ms7Ozoaur+68bRVSdnv/vhYiIapfa9DldoSNOnp6eAACFQgFfX19oampK84qKinDmzBm4urpWbQuJiIiIaokKRTgDAwMYGBhACAGlUik9NzAwgKmpKUaNGoVNmza9qrYSUTXw9fXFoEGDaroZFRIREQGFQvGvfvqpKup4kXXr1sHQ0PBf1/Mq20hE8lToiNPatWsBAFZWVpg8eTKH5ei15OxcveuLja1YeV9fX6xfvx4AUL9+fVhYWMDT0xNBQUH/s+9JV1dXZGZmwsDAoKabAisrK4wfPx7jx4+v6aYAAHr06AFHR0csXbq0WterUCiwa9euOhfCiV6mUoOGgYGB0NTUxF9//YVVq1bh/v37AIAbN27gwYMHVdpAIirtnXfeQWZmJq5cuYK5c+fixx9/xOTJk2u6WTVGQ0MDpqamvCqSiF65SgWnq1evwt7eHh4eHhg7dixu3boFAFi4cOH/9Ic3UXXR1NSEqakpLCws4OPjg+HDhyM0NBQAsGnTJjg7O0OpVMLU1BQ+Pj64efOmyvLnz59H//79oa+vD6VSia5du+Ly5ctlrisuLg4mJib45ptvAAD79+9Hly5dYGhoCGNjYwwYMKDUspGRkXB0dISWlhacnZ0RGhoKhUKBhIQEqUxSUhL69esHPT09NG7cGB9++CFu374tzf/1119hb28PbW1tGBsbo3fv3nj48GGZbXx+CKtkaOzAgQNo06YN9PT0pLD5MidOnICDgwO0tLTQuXNnnD17VmX+b7/9hrZt20JTUxNWVlZYvHixNK9Hjx64evUqJkyYAIVCUSrIVbQ9+/btg62tLbS1tdGzZ0+kp6erzM/Ozoa3tzfMzc2ho6MDe3t7bN26VZrv6+uLI0eOYNmyZVJ70tPTUVRUhJEjR8La2hra2tpo1aoVli1bVmqbdurUCbq6ujA0NISbmxuuXr0qzd+zZw+cnJygpaWF5s2bIygoCIWFhQCeHnUDgMGDB0OhUEjPExMT0bNnTyiVSujr68PJyQmxFT3kSlTDKhWcxo0bB2dnZ9y9exfa2trS9MGDB+PQoUNV1jgikkdbWxsFBQUAnt5mITg4GImJiQgNDUVaWhp8fX2lstevX0e3bt2gpaWF8PBwxMXFwc/PT/rSe1ZERAR69eqFoKAgzJgxAwDw8OFDTJw4ETExMTh06BDq1auHwYMHo7i4GABw//59DBw4EPb29jh9+jSCg4MxdepUlXozMzPRvXt3ODo6IjY2Fvv378c///wDLy8vab63tzf8/PyQnJyMiIgIeHp6oiK3nXv06BEWLVqEjRs34ujRo7h27Zqsf+ymTJmCRYsWISYmBiYmJnj33XelbRsXFwcvLy8MGzYMZ8+exezZszFr1iysW7cOALBz506Ym5tjzpw5yMzMVAlGFW1PRkYGPD090a9fPyQkJOCTTz7BtGnTVMrk5eXByckJe/fuxblz5zBq1Ch8+OGHiI6OBgAsW7YMLi4u+PTTT6X2WFhYoLi4GObm5tixYweSkpIQEBCAr7/+Gjt27ADw9J58gwYNQvfu3XHmzBlERUVh1KhRUhA8cOAAPvjgA3z55ZdISkrCqlWrsG7dOilcx8TEAHh6ekdmZqb0fPjw4TA3N0dMTAzi4uIwbdo0qKurv3SfENUqohKMjY3FhQsXhBBC6OnpicuXLwshhEhLSxPa2tqVqZIqIScnRwAQOTk5Nd2UOunx48ciKSlJPH78WGW6k1P1PipqxIgRwsPDQ3oeHR0tjI2NhZeXV5nlT506JQCI+/fvCyGEmD59urC2thb5+fnl1h8aGiqUSqXYsmVLue25efOmACDOnj0rhBBi5cqVwtjYWGW7hoSECAAiPj5eCCHErFmzRJ8+fVTqycjIEABESkqKiIuLEwBEenp6uesucfjwYQFA3L17VwghxNq1awUAcenSJanMihUrROPGjV9ax7Zt26Rp2dnZQltbW2zfvl0IIYSPj494++23VZabMmWKsLOzk55bWlqKJUuWqJSpTHumT58u2rRpI4qLi6VpU6dOVelnWfr16ycmTZokPe/evbsYN27cC8uXGDNmjHjvvfeEEE/7DUBERESUWbZr165i3rx5KtM2btwozMzMpOcAxK5du1TKKJVKsW7dupe2heh5L/q8FqL6vwsrdcSpuLi4zJtc/v3331AqlZWpkogqYO/evdDT04OWlhZcXFzQrVs3LF++HAAQHx8PDw8PWFpaQqlUokePHgCAa9euAXj6m5Jdu3Yt9z/96OhovPfee1i/fj28vb1V5l2+fBk+Pj5o3rw59PX1YW1trVJ/SkoK2rdvr3KTuk6dOqnUERcXh8OHD0NPT096tG7dWqrfwcEBvXr1gr29PYYMGYKQkBDcvXu3QttIR0cHLVq0kJ6bmZmVGrIsi4uLi/S3kZERWrVqheTkZABAcnIy3NzcVMq7ubkhNTX1pTf+rWh7kpOT8eabb6oM9z3bNuDpbWC++eYbtG/fHsbGxtDT08PBgwelfVGen376Cc7OzmjUqBH09PQQEhIiLWdkZARfX1+4u7tj4MCBWLZsmcrRs7i4OMyZM0dl/5Uc1Srv54smTpyITz75BL1798aCBQteODxMVJtVKji9/fbbKldoKBQKPHjwAIGBgejXr19VtY2IXqBnz55ISEhASkoK8vLysHPnTpiYmODhw4fo06cP9PT0sGnTJsTExGDXrl0Ang7hAVAZXn+RFi1aoHXr1lizZo20XImBAwciOzsbISEhiI6OloaFSsqJMm6QK54bYisuLsbAgQORkJCg8khNTUW3bt2gpqaGsLAw/Pnnn7Czs8Py5cvRqlUrpKWlyd5GzwdDhUJRoaG+55eV27eqao+cehcvXowlS5bgq6++Qnh4OBISEuDu7l5qnz1vx44dmDBhAvz8/HDw4EEkJCTg448/Vllu7dq1iIqKgqurK7Zv3w5bW1ucPHkSwNP9FxQUpLLvzp49i9TU1HLv6jx79mzp/Lrw8HDY2dlJr0+iuqJSwWnJkiU4cuQI7OzskJeXBx8fH1hZWeH69ev49ttvq7qNRPQcXV1d2NjYwNLSUuUL+cKFC7h9+zYWLFiArl27onXr1qWOarRv3x7Hjh2TztspS8OGDREeHo7Lly9j6NChUtns7GwkJydj5syZ6NWrF9q0aVPqSFDr1q1x5swZPHnyRJr2/AnAHTt2xPnz52FlZQUbGxuVR8ktFRQKBdzc3BAUFIT4+HhoaGhUy5dsSTgAgLt37+LixYvS0TA7OzscP35cpXxkZCRsbW2hpqYG4OkVflXxs1N2dnYqbXm+bQBw7NgxeHh44IMPPoCDgwOaN2+O1NRUlTJltefYsWNwdXXFmDFj0KFDB9jY2JR59KdDhw6YPn06IiMj0a5dO2zZsgXA0/2XkpJSat/Z2NhId3hWV1cvczvY2tpiwoQJOHjwIDw9PaXb3BDVFZUKTk2aNEFCQgKmTJmCzz77DB06dMCCBQsQHx9f6vfriKj6NGvWDBoaGli+fDmuXLmC3bt3Izg4WKWMv78/cnNzMWzYMMTGxiI1NRUbN25ESkqKSjkTExOEh4fjwoUL8Pb2RmFhIRo0aABjY2P8/PPPuHTpEsLDwzFx4kSV5Xx8fFBcXIxRo0YhOTkZBw4cwKJFiwD898jN2LFjcefOHXh7e+PUqVO4cuUKDh48CD8/PxQVFSE6Ohrz5s1DbGwsrl27hp07d+LWrVto06bNK9x6T82ZMweHDh3CuXPn4Ovri4YNG0r3Ipo0aRIOHTqE4OBgXLx4EevXr8cPP/ygcpK3lZUVjh49iuvXr6tcJVhRo0ePxuXLlzFx4kSkpKRgy5Yt0knoJWxsbBAWFobIyEgkJyfjs88+Q1ZWlkoZKysrREdHIz09Hbdv30ZxcTFsbGwQGxuLAwcO4OLFi5g1a5Z0AjcApKWlYfr06YiKisLVq1dx8OBBXLx4Udr+AQEB2LBhg3QEKTk5Gdu3b8fMmTNV1nvo0CFkZWXh7t27ePz4Mfz9/REREYGrV6/ixIkTiImJqZZ9SlSlquVMKnoleHL4v/O6nBz+vC1btggrKyuhqakpXFxcxO7du1VOzBZCiMTERNGnTx+ho6MjlEql6Nq1q3SRx/P137hxQ9ja2govLy9RWFgowsLCRJs2bYSmpqZo3769iIiIKHUi8IkTJ0T79u2FhoaGcHJyElu2bBEApItKhBDi4sWLYvDgwcLQ0FBoa2uL1q1bi/Hjx4vi4mKRlJQk3N3dRaNGjYSmpqawtbUVy5cvf2Gfyzo53MDAQKXMrl27RHkfeSV17NmzR7Rt21ZoaGiIN954QyQkJKiU+/XXX4WdnZ1QV1cXzZo1E999953K/KioKNG+fXuhqakpra8y7RFCiD179ggbGxuhqakpunbtKtasWaPSz+zsbOHh4SH09PSEiYmJmDlzpvjoo49U9l9KSop48803hba2tgAg0tLSRF5envD19RUGBgbC0NBQfP7552LatGnCwcFBCCFEVlaWGDRokDAzMxMaGhrC0tJSBAQEiKKiIqne/fv3C1dXV6GtrS309fVFp06dxM8//yzN3717t7CxsRH169cXlpaW4smTJ2LYsGHCwsJCaGhoiCZNmgh/f/8yT/Ylel5tOjlcIUTFB/3Xr1+Phg0bon///gCAr776Cj///DPs7OywdetWWFpaVkmoo/Ll5ubCwMAAOTk50NfXr+nm1Dl5eXlIS0uDtbV1jf/a9utu8+bN+Pjjj5GTkyPrHCsiomeV93ld3d+FlRqqmzdvnvThFxUVhR9++AELFy5Ew4YNMWHChCptIBHVPRs2bMDx48eRlpaG0NBQTJ06FV5eXgxNRFTnVei36kpkZGTAxsYGABAaGor3338fo0aNgpubm3TpMxH978rKykJAQACysrJgZmaGIUOGSDdHJCKqyyoVnPT09JCdnY1mzZrh4MGD0lEmLS0tPH78uEobSER1z1dffYWvvvqqpptBRFTlKhWc3n77bXzyySfo0KEDLl68KJ3rVHJ5MREREdHrqFLnOK1YsQIuLi64desWfvvtNxgbGwN4ejfZ5+8yTERERPS6qNQRJ0NDQ/zwww+lpgcFBf3rBhERERHVVpUKTiUePXqEa9eulbq9f/v27f9Vo4iIiIhqo0oFp1u3bsHX1xf79+8vc35V/NwAERERUW1TqXOcxo8fj3v37uHkyZPQ1tbG/v37sX79erRs2RK7d++u6jYSERER1QqVCk7h4eFYsmQJ3njjDdSrVw+Wlpb44IMPsHDhQsyfP7+q20hE1cjX11f6bba6IiIiAgqFAvfu3avpplSJ9PR0KBQKJCQk1HRTXpm6+DqTS6FQIDQ0tMrrnT17NhwdHau83sqysrLC0qVLa7oZ1a5SQ3UPHz6UfszXyMgIt27dgq2tLezt7XH69OkqbSBRdXP+2bla1xc7KrZC5X19fbF+/XoAQP369WFhYQFPT08EBQVBV1f3VTSx1nN1dUVmZiYMDAxquik1xtfXF/fu3XslX9jlsbKywvjx4zF+/PhqXW8JhUKBXbt2VUsIk9vXzMxMNGjQ4F+tqzr79TLr1q2TRpqokkecWrVqJf2SuqOjI1atWoXr16/jp59+gpmZWZU2kIhKe+edd5CZmYkrV65g7ty5+PHHHzF58uSablaN0dDQgKmpKRQKRU03hQimpqbQ1NSs6WbUSc9fbFYbVfocp8zMTABAYGAg9u/fj2bNmuE///kP5s2bV6UNJKLSNDU1YWpqCgsLC/j4+GD48OHSkYZNmzbB2dkZSqUSpqam8PHxwc2bN1WWP3/+PPr37w99fX0olUp07doVly9fLnNdcXFxMDExkX4yZf/+/ejSpQsMDQ1hbGyMAQMGlFo2MjISjo6O0NLSgrOzM0JDQ0sNPSUlJaFfv37Q09ND48aN8eGHH+L27dvS/F9//RX29vbQ1taGsbExevfujYcPH5bZxueH6tatWwdDQ0McOHAAbdq0gZ6enhQ2y3PkyBF06tQJmpqaMDMzw7Rp01BYWCjN79GjB7788kt89dVXMDIygqmpKWbPnq1SR05ODkaNGgUTExPo6+vjrbfeQmJiYrnrPXXqFDp06CBtr/j4eJX5RUVFGDlyJKytraGtrY1WrVph2bJl0vzZs2dj/fr1+P3336FQKKBQKBAREQEAmDp1KmxtbaGjo4PmzZtj1qxZKCgokJZNTExEz549oVQqoa+vDycnJ8TG/vcoaGRkJLp16wZtbW1YWFjgyy+/lPZDjx49cPXqVUyYMEFab0UFBQVJ2+qzzz5T+eIsayjI0dFR2uYlN1wePHgwFArFC2/AXDL0uXPnTvTs2RM6OjpwcHBAVFSUSrnffvsNbdu2haamJqysrLB48WJpXkX6+uxQndx1P+tl/dq4cSOsrKxgYGCAYcOG4f79+9I8IQQWLlyI5s2bQ1tbGw4ODvj1119fuC4AuHv3Lj766CM0aNAAOjo66Nu3L1JTUwE8fW+V/EB3Sb+ffc0/evQIfn5+UCqVaNasGX7++WeVuq9fv46hQ4eiQYMGMDY2hoeHB9LT06X5JUO28+fPR5MmTWBra1tuW2uDSgWn4cOHw9fXFwDQoUMHpKenIyYmBhkZGRg6dGhVto+IZNDW1pa+DPPz8xEcHIzExESEhoYiLS1Ner8CTz/IunXrBi0tLYSHhyMuLg5+fn4qAaFEREQEevXqhaCgIMyYMQPA06H6iRMnIiYmBocOHUK9evUwePBgFBcXAwDu37+PgQMHSkP3wcHBmDp1qkq9mZmZ6N69OxwdHREbG4v9+/fjn3/+gZeXlzTf29sbfn5+SE5ORkREBDw9PSGEkL1NHj16hEWLFmHjxo04evQorl27Vu5RuevXr6Nfv3544403kJiYiJUrV2L16tWYO3euSrn169dDV1cX0dHRWLhwIebMmYOwsDAAT7+0+vfvj6ysLOzbtw9xcXHo2LEjevXqhTt37pS53ocPH2LAgAFo1aoV4uLiMHv27FLtLC4uhrm5OXbs2IGkpCQEBATg66+/xo4dOwAAkydPhpeXlxQOMzMz4erqCgBQKpVYt24dkpKSsGzZMoSEhGDJkiVS3cOHD4e5uTliYmIQFxeHadOmQV1dHQBw9uxZuLu7w9PTE2fOnMH27dtx/Phx+Pv7AwB27twJc3NzzJkzR1pvRRw6dAjJyck4fPgwtm7dil27dlXofoAxMTEAgLVr1yIzM1N6/iIzZszA5MmTkZCQAFtbW3h7e0uv+7i4OHh5eWHYsGE4e/YsZs+ejVmzZmHdunVV0tfy1l2Rfl2+fBmhoaHYu3cv9u7diyNHjmDBggXS/JkzZ2Lt2rVYuXIlzp8/jwkTJuCDDz7AkSNHXtg2X19fxMbGYvfu3YiKioIQAv369UNBQQFcXV2xdOlS6OvrS/1+9vW5ePFiKeyPGTMGn3/+OS5cuADg6XuwZ8+e0NPTw9GjR3H8+HHpn5hnA3LJ6yAsLAx79+6t0HatEaISgoKCxMOHD0tNf/TokQgKCqpMlVQJOTk5AoDIycmp6abUSY8fPxZJSUni8ePHKtOdVjlV66OiRowYITw8PKTn0dHRwtjYWHh5eZVZ/tSpUwKAuH//vhBCiOnTpwtra2uRn59fbv2hoaFCqVSKLVu2lNuemzdvCgDi7NmzQgghVq5cKYyNjVW2a0hIiAAg4uPjhRBCzJo1S/Tp00elnoyMDAFApKSkiLi4OAFApKenl7vuEocPHxYAxN27d4UQQqxdu1YAEJcuXZLKrFixQjRu3PiFdXz99deiVatWori4WGUZPT09UVRUJIQQonv37qJLly4qy73xxhti6tSpQgghDh06JPT19UVeXp5KmRYtWohVq1aVud5Vq1YJIyMjlc/UlStXqmyvsowZM0a899570vPnXxcvsnDhQuHk9N/XnVKpFOvWrSuz7IcffihGjRqlMu3YsWOiXr160v61tLQUS5Yseel6nzdixIgy+/3s9i6rbgcHBxEYGCg9ByB27dpV7rrS0tIEAPF///d/0rTz588LACI5OVkIIYSPj494++23VZabMmWKsLOzk57L7euzbZKz7pfVUSIwMFDo6OiI3NxclTZ27txZCCHEgwcPhJaWloiMjFRZbuTIkcLb27vM9Vy8eFEAECdOnJCm3b59W2hra4sdO3YIIZ6+nwwMDEota2lpKT744APpeXFxsTAxMRErV64UQgixevXqUu+pJ0+eCG1tbXHgwAEhxNPXQePGjcWTJ09euC2EePHntRDV/11YqSNOQUFBePDgQanpjx494t3DiarB3r17oaenBy0tLbi4uKBbt25Yvnw5ACA+Ph4eHh6wtLSEUqlEjx49AADXrl0DACQkJKBr167SUYWyREdH47333sP69etL/YzS5cuX4ePjg+bNm0NfXx/W1tYq9aekpKB9+/bQ0tKSlunUqZNKHXFxcTh8+DD09PSkR+vWraX6HRwc0KtXL9jb22PIkCEICQnB3bt3K7SNdHR00KJFC+m5mZlZqSHLZyUnJ8PFxUVlCMbNzQ0PHjzA33//LU17/ga/z9YbFxeHBw8ewNjYWKVvaWlpLxwKTU5OhoODA3R0dKRpLi4upcr99NNPcHZ2RqNGjaCnp4eQkBBpm5fn119/RZcuXWBqago9PT3MmjVLZbmJEyfik08+Qe/evbFgwQKVdsbFxWHdunUqfXF3d0dxcTHS0tJeuu6XKavfDx48QEZGxr+uuyzP7ruS83FL9l1ycjLc3NxUyru5uSE1NbVK7k1Y3rorwsrKCkqlUqWuknqSkpKQl5eHt99+W2WfbdiwodzXX/369dG5c2dpmrGxMVq1aoXk5OQK9UuhUMDU1FTl/XDp0iUolUqpLUZGRsjLy1Npj729PTQ0NCq2IWpQpa6qE0KUOb6bmJgIIyOjf90oIipfz549sXLlSqirq6NJkyZSCHr48CH69OmDPn36YNOmTWjUqBGuXbsGd3d36dC4trb2S+tv0aIFjI2NsWbNGvTv31/lQ23gwIGwsLBASEgImjRpguLiYrRr106qv6zPB/HcEFtxcTEGDhyIb7/9ttS6zczMoKamhrCwMERGRuLgwYNYvnw5ZsyYgejoaCmovczzwVChUJQ71Fdeu5+dXla9JcOUxcXFMDMzk84vepahoeEL1/syO3bswIQJE7B48WK4uLhAqVTiu+++Q3R0dLnLnTx5EsOGDUNQUBDc3d1hYGCAbdu2qZy7M3v2bPj4+OCPP/7An3/+icDAQGzbtk0afv3ss8/w5Zdflqq7WbNmL213ZZVs73r16pXaPs+en1VRz+67knWU7Ds5r9t/o7x1V7aekrqeff0BwB9//IGmTZuqlHvRyeov6uOLvucr2h4nJyds3ry51HKNGjWS/q5rVwNXKDg1aNBAOjnM1tZWZaMWFRXhwYMHGD16dJU3kohU6erqwsbGptT0Cxcu4Pbt21iwYAEsLCwAQOVEX+Dpf4jr169HQUHBC486NWzYEDt37kSPHj0wdOhQ7NixA+rq6sjOzkZycjJWrVqFrl27AgCOHz+usmzr1q2xefNmPHnyRPqwfr4NHTt2xG+//QYrKyvUr1/2x5BCoYCbmxvc3NwQEBAAS0tL7Nq1CxMnTpSxhSrOzs4Ov/32m8oXRmRkJJRKZakvoRfp2LEjsrKyUL9+/ReeqFzWejdu3IjHjx9LofbkyZMqZY4dOwZXV1eMGTNGmvb8EQQNDY1SR0ZOnDgBS0tL6fw0ALh69WqpNtja2sLW1hYTJkyAt7c31q5di8GDB6Njx444f/58ma+18tYrV2JiYql+6+npwdzcHMDTL9dnzyXKzc0tdaRLXV29So4I2dnZlXotR0ZGwtbWFmpqagD+XV8rqjL9srOzg6amJq5du4bu3bvLXqawsBDR0dHSeXHZ2dm4ePEi2rRpA6Dy/e7YsSO2b98unfz/uqjQUN3SpUvx/fffQwiBoKAgLFmyRHr89NNPOH78OFasWPGq2kpEL9GsWTNoaGhg+fLluHLlCnbv3o3g4GCVMv7+/sjNzcWwYcMQGxuL1NRUbNy4UbrFSAkTExOEh4fjwoUL0omsJVfG/Pzzz7h06RLCw8NLBRkfHx8UFxdj1KhRSE5OxoEDB7Bo0SIA//1Pe+zYsbhz5w68vb1x6tQpXLlyBQcPHoSfnx+KiooQHR2NefPmITY2FteuXcPOnTtx69Yt6YP8VRgzZgwyMjLwxRdf4MKFC/j9998RGBiIiRMnol49eR+VvXv3houLCwYNGoQDBw4gPT0dkZGRmDlzZqnwWMLHxwf16tXDyJEjkZSUhH379knbq4SNjQ1iY2Nx4MABXLx4EbNmzSp1IrSVlRXOnDmDlJQU3L59GwUFBbCxscG1a9ewbds2XL58Gf/5z3+wa9cuaZnHjx/D398fERERuHr1Kk6cOIGYmBhpO0+dOhVRUVEYO3YsEhISkJqait27d+OLL75QWe/Ro0dx/fp1lasi5cjPz5f6XXK0y9/fX9reb731FjZu3Ihjx47h3LlzGDFihBRinl3/oUOHkJWVVeHh3GdNmjQJhw4dQnBwMC5evIj169fjhx9+UDkR+t/0taIq0y+lUonJkydjwoQJWL9+PS5fvoz4+HisWLFCuvfb81q2bAkPDw98+umnOH78OBITE/HBBx+gadOm8PDwkNry4MEDHDp0CLdv38ajR49ktWf48OFo2LAhPDw8cOzYMaSlpeHIkSMYN26cyvB3XVOh4DRixAj4+vri8OHD+PzzzzFixAjp4e3tXea4PBFVn0aNGmHdunX45ZdfYGdnhwULFpT6EjY2NkZ4eDgePHiA7t27w8nJCSEhIWUefTI1NUV4eDjOnj2L4cOHQwiBbdu2IS4uDu3atcOECRPw3XffqSyjr6+PPXv2ICEhAY6OjpgxYwYCAgIAQDrvqUmTJjhx4gSKiorg7u6Odu3aYdy4cTAwMEC9evWgr6+Po0ePol+/frC1tcXMmTOxePFi9O3b9xVtOaBp06bYt28fTp06BQcHB4wePRojR47EzJkzZdehUCiwb98+dOvWDX5+frC1tcWwYcOQnp6Oxo0bl7mMnp4e9uzZg6SkJHTo0AEzZswoNYQ5evRoeHp6YujQoejcuTOys7NVjj4BwKeffopWrVpJ50GdOHECHh4emDBhAvz9/eHo6IjIyEjMmjVLWkZNTQ3Z2dn46KOPYGtrCy8vL/Tt21c6V7V9+/Y4cuQIUlNT0bVrV3To0AGzZs1SuV/fnDlzkJ6ejhYtWqgMvygUCumKtBfp1asXWrZsiW7dusHLywsDBw5UudR9+vTp6NatGwYMGIB+/fph0KBBKuetAU+v6goLC4OFhQU6dOhQ7vrK07FjR+zYsQPbtm1Du3btEBAQgDlz5qhckfqivr4Kle1XcHAwAgICMH/+fLRp0wbu7u7Ys2dPuUPca9euhZOTEwYMGAAXFxcIIbBv3z7pM8HV1RWjR4/G0KFD0ahRIyxcuFBWW3R0dHD06FE0a9YMnp6eaNOmDfz8/PD48eM6fQRKIf7lIO7jx49LjTnX5Q1Sl+Tm5sLAwAA5OTnc5pWQl5eHtLQ0WFtbq5zITFVv8+bN0r1g5JxjRXVbeno6WrZsiaSkJLRs2bKmm0OvgfI+r6v7u7BSJ4c/evQIX331FXbs2IHs7OxS86trDJiIaqcNGzagefPmaNq0KRITEzF16lR4eXkxNP2P2L9/P0aNGsXQRK+lSgWnKVOm4PDhw/jxxx/x0UcfYcWKFbh+/TpWrVqlciMuIvrflJWVhYCAAGRlZcHMzAxDhgyR7jxOrz9eJESvs0oN1TVr1gwbNmxAjx49oK+vj9OnT8PGxgYbN27E1q1bsW/fvlfRVnoOh+r+HQ7VERHVDbVpqK5SN8C8c+eOdKKZvr6+9FMCXbp0wdGjR6uudURERES1SKWCU/PmzaUf6bOzs5N+L2nPnj0vvMkbERERUV1XqeD08ccfS7/2PX36dPz444/Q1NTEhAkTMGXKlCptINGrVpV3ByYioqpXmz6nK3Vy+IQJE6S/e/bsiQsXLiA2NhYtWrSAg4NDlTWO6FUquUfJo0ePeLUXEVEtVnLTzfJ+Y7O6VDg4FRcXY926ddi5cyfS09OhUChgbW2N999/v9SPXxLVZmpqajA0NJR+kFJHR0fWbzMREVH1EELg0aNHuHnzJgwNDUvdOb4mVCg4CSHw7rvvYt++fXBwcIC9vT2EEEhOToavry927tyJ0NDQV9RUoqpnamoKoHK/Uk5ERNXD0NBQ+ryuaRUKTuvWrcPRo0dx6NAh9OzZU2VeeHg4Bg0ahA0bNuCjjz6q0kZS+bqv7Q417ZpP4XWZZj1NGKgbQIEXH3H6behv1dgiIiICng7P1YYjTSUqFJy2bt2Kr7/+ulRoAp7+GOO0adOwefNmBieqc54UP8HNJ+UfdeK9noiIqEJX1Z05cwbvvPPOC+f37dtXutqOiIiI6HVToeB0586dF/7CNwA0btwYd+/e/deNIiIiIqqNKhScioqKUL/+i0f31NTUUFhY+K8bRURERFQbVfiqOl9fX2hqapY5/8mTJ1XSKCIiIqLaqELBacSIES8twxPDiYiI6HVVoeC0du3aV9UOIiIiolqvUr9VR0RERPS/iMGJiIiISCYGJyIiIiKZGJyIiIiIZGJwIiIiIpKJwYmIiIhIJgYnIiIiIpkYnIiIiIhkYnAiIiIikonBiYiIiEgmBiciIiIimRiciIiIiGRicCIiIiKSicGJiIiISCYGJyIiIiKZajQ49ejRA+PHjwcAWFlZYenSpdI8hUKB0NDQam3Pq1xneno6FAoFEhISXkn9RERE9OrVr+kGlIiJiYGurm6NtiEzMxMNGjR4JXVbWFggMzMTDRs2fCX1ExER0atXa4JTo0aN/nUdBQUFUFdXr/By+fn50NDQgKmp6b9uw4uoqam90vqJiIjo1as15zg9P1QHPD0C1LdvX2hra8Pa2hq//PKLNK9k6GvHjh3o0aMHtLS0sGnTJmRnZ8Pb2xvm5ubQ0dGBvb09tm7dqlJvjx494O/vj4kTJ6Jhw4Z4++23AZQeqrt+/TqGDh2KBg0awNjYGB4eHkhPT5fmR0REoFOnTtDV1YWhoSHc3Nxw9erVMvtX1lDdkSNH0KlTJ2hqasLMzAzTpk1DYWFh5TYgERERvXK1JjiVZdasWXjvvfeQmJiIDz74AN7e3khOTlYpM3XqVHz55ZdITk6Gu7s78vLy4OTkhL179+LcuXMYNWoUPvzwQ0RHR6sst379etSvXx8nTpzAqlWrSq370aNH6NmzJ/T09HD06FEcP34cenp6eOedd5Cfn4/CwkIMGjQI3bt3x5kzZxAVFYVRo0ZBoVDI6tv169fRr18/vPHGG0hMTMTKlSuxevVqzJ07t/IbjIiIiF6pWjNUV5YhQ4bgk08+AQAEBwcjLCwMy5cvx48//iiVGT9+PDw9PVWWmzx5svT3F198gf379+OXX35B586dpek2NjZYuHDhC9e9bds21KtXD//3f/8nhaG1a9fC0NAQERERcHZ2Rk5ODgYMGIAWLVoAANq0aSO7bz/++CMsLCzwww8/QKFQoHXr1rhx4wamTp2KgIAA1KtXOtM+efIET548kZ7n5ubKXh8RERH9e7X6iJOLi0up588fcXJ2dlZ5XlRUhG+++Qbt27eHsbEx9PT0cPDgQVy7dq3c5Z4XFxeHS5cuQalUQk9PD3p6ejAyMkJeXh4uX74MIyMj+Pr6wt3dHQMHDsSyZcuQmZkpu2/JyclwcXFROULl5uaGBw8e4O+//y5zmfnz58PAwEB6WFhYyF4fERER/Xu1OjiV5fmhsOevxFu8eDGWLFmCr776CuHh4UhISIC7uzvy8/PLXe55xcXFcHJyQkJCgsrj4sWL8PHxAfD0CFRUVBRcXV2xfft22Nra4uTJk7L6IYQo1RchRJl9LDF9+nTk5ORIj4yMDFnrIiIioqpRq4PT8yHk5MmTaN26dbnLHDt2DB4eHvjggw/g4OCA5s2bIzU1tcLr7tixI1JTU2FiYgIbGxuVh4GBgVSuQ4cOmD59OiIjI9GuXTts2bJFVv12dnaIjIyUwhIAREZGQqlUomnTpmUuo6mpCX19fZUHERERVZ9aHZx++eUXrFmzBhcvXkRgYCBOnToFf3//cpexsbFBWFgYIiMjkZycjM8++wxZWVkVXvfw4cPRsGFDeHh44NixY0hLS8ORI0cwbtw4/P3330hLS8P06dMRFRWFq1ev4uDBg7h48aJ0ntOpU6fQunVrXL9+vcz6x4wZg4yMDHzxxRe4cOECfv/9dwQGBmLixIllnt9ERERENa9WnxweFBSEbdu2YcyYMTA1NcXmzZthZ2dX7jKzZs1CWloa3N3doaOjg1GjRmHQoEHIycmp0Lp1dHRw9OhRTJ06FZ6enrh//z6aNm2KXr16QV9fH48fP8aFCxewfv16ZGdnw8zMDP7+/vjss88APL0qLyUlBQUFBWXW37RpU+zbtw9TpkyBg4MDjIyMMHLkSMycObNC7SQiIqLqoxDPjhVRnZKbmwsDAwM4LnWEmrZaTTfntRc7Kramm0BERM8p+S7MycmpllNYOCZEREREJBODExEREZFMDE5EREREMjE4EREREcnE4EREREQkE4MTERERkUwMTkREREQyMTgRERERycTgRERERCQTgxMRERGRTAxORERERDIxOBERERHJxOBEREREJBODExEREZFMDE5EREREMjE4EREREcnE4EREREQkE4MTERERkUwMTkREREQyMTgRERERycTgRERERCQTgxMRERGRTPVrugFUBdYdAdT0a7oVrz3nn2u6BUT0OouNrekWkBw84kREREQkE4MTERERkUwMTkREREQyMTgRERERycTgRERERCQTgxMRERGRTAxORERERDIxOBERERHJxOBEREREJBODExEREZFMDE5EREREMjE4EREREcnE4EREREQkE4MTERERkUwMTkREREQyMTgRERERycTgRERERCQTgxMRERGRTAxORERERDIxOBERERHJxOBEREREJBODExEREZFMDE5EREREMjE4EREREcnE4EREREQkE4MTERERkUwMTkREREQyMTgRERERycTgRERERCQTgxMRERGRTAxORERERDIxOBERERHJxOBEREREJBODExEREZFMDE5EREREMjE4EREREcnE4EREREQkE4MTERERkUwMTkREREQyMTgRERERycTgRERERCQTgxMRERGRTAxORERERDIxOBERERHJxOBEREREJBODExEREZFMDE5EREREMjE4EREREcnE4EREREQkE4MTERERkUwMTkREREQyMTgRERERycTgRERERCQTgxMRERGRTAxORERERDIxOBERERHJxOBEREREJFP9mm4A/XtHjgD6+jXdCiIiotcfjzgRERERycTgRERERCQTgxMRERGRTAxORERERDIxOBERERHJxOBEREREJBODExEREZFMDE5EREREMjE4EREREcnE4EREREQkE4MTERERkUwMTkREREQy8Ud+6zAhBAAgNze3hltCRERUM0q+A0u+E181Bqc6LDs7GwBgYWFRwy0hIiKqWffv34eBgcErXw+DUx1mZGQEALh27Vq1vFiqW25uLiwsLJCRkQF9ff2abk6Ve5379zr3DWD/6rrXuX+vc9+AsvsnhMD9+/fRpEmTamkDg1MdVq/e01PUDAwMXss3SAl9fX32r456nfsGsH913evcv9e5b0Dp/lXnwQOeHE5EREQkE4MTERERkUwMTnWYpqYmAgMDoampWdNNeSXYv7rrde4bwP7Vda9z/17nvgG1o38KUV3X7xERERHVcTziRERERCQTgxMRERGRTAxORERERDIxONVhP/74I6ytraGlpQUnJyccO3asppukYv78+XjjjTegVCphYmKCQYMGISUlRaWMr68vFAqFyuPNN99UKfPkyRN88cUXaNiwIXR1dfHuu+/i77//Vilz9+5dfPjhhzAwMICBgQE+/PBD3Lt375X2b/bs2aXabmpqKs0XQmD27Nlo0qQJtLW10aNHD5w/f75O9A0ArKysSvVPoVBg7NixAOrWvjt69CgGDhyIJk2aQKFQIDQ0VGV+de6ra9euYeDAgdDV1UXDhg3x5ZdfIj8//5X1r6CgAFOnToW9vT10dXXRpEkTfPTRR7hx44ZKHT169Ci1P4cNG1br+wdU72uxJvpX1vtQoVDgu+++k8rU1v0n53ugzr3/BNVJ27ZtE+rq6iIkJEQkJSWJcePGCV1dXXH16tWabprE3d1drF27Vpw7d04kJCSI/v37i2bNmokHDx5IZUaMGCHeeecdkZmZKT2ys7NV6hk9erRo2rSpCAsLE6dPnxY9e/YUDg4OorCwUCrzzjvviHbt2onIyEgRGRkp2rVrJwYMGPBK+xcYGCjatm2r0vabN29K8xcsWCCUSqX47bffxNmzZ8XQoUOFmZmZyM3NrfV9E0KImzdvqvQtLCxMABCHDx8WQtStfbdv3z4xY8YM8dtvvwkAYteuXSrzq2tfFRYWinbt2omePXuK06dPi7CwMNGkSRPh7+//yvp379490bt3b7F9+3Zx4cIFERUVJTp37iycnJxU6ujevbv49NNPVfbnvXv3VMrUxv4JUX2vxZrq37P9yszMFGvWrBEKhUJcvnxZKlNb95+c74G69v5jcKqjOnXqJEaPHq0yrXXr1mLatGk11KKXu3nzpgAgjhw5Ik0bMWKE8PDweOEy9+7dE+rq6mLbtm3StOvXr4t69eqJ/fv3CyGESEpKEgDEyZMnpTJRUVECgLhw4ULVd+T/CwwMFA4ODmXOKy4uFqampmLBggXStLy8PGFgYCB++uknIUTt7ltZxo0bJ1q0aCGKi4uFEHV33z3/xVSd+2rfvn2iXr164vr161KZrVu3Ck1NTZGTk/NK+leWU6dOCQAq/2h1795djBs37oXL1Ob+VddrsbbsPw8PD/HWW2+pTKsr++/574G6+P7jUF0dlJ+fj7i4OPTp00dlep8+fRAZGVlDrXq5nJwcAP/9jb0SERERMDExga2tLT799FPcvHlTmhcXF4eCggKVvjZp0gTt2rWT+hoVFQUDAwN07txZKvPmm2/CwMDglW+P1NRUNGnSBNbW1hg2bBiuXLkCAEhLS0NWVpZKuzU1NdG9e3epTbW9b8/Kz8/Hpk2b4OfnB4VCIU2vy/uuRHXuq6ioKLRr107lN7Xc3d3x5MkTxMXFvdJ+PisnJwcKhQKGhoYq0zdv3oyGDRuibdu2mDx5Mu7fvy/Nq+39q47XYm3Yf//88w/++OMPjBw5stS8urD/nv8eqIvvP/5WXR10+/ZtFBUVoXHjxirTGzdujKysrBpqVfmEEJg4cSK6dOmCdu3aSdP79u2LIUOGwNLSEmlpaZg1axbeeustxMXFQVNTE1lZWdDQ0ECDBg1U6nu2r1lZWTAxMSm1ThMTk1e6PTp37owNGzbA1tYW//zzD+bOnQtXV1ecP39eWm9Z++jq1atSu2tr354XGhqKe/fuwdfXV5pWl/fds6pzX2VlZZVaT4MGDaChoVFt/c3Ly8O0adPg4+Oj8ltfw4cPh7W1NUxNTXHu3DlMnz4diYmJCAsLk9peW/tXXa/F2rD/1q9fD6VSCU9PT5XpdWH/lfU9UBfffwxOddiz//kDT1+Uz0+rLfz9/XHmzBkcP35cZfrQoUOlv9u1awdnZ2dYWlrijz/+KPXB8Kzn+1pWv1/19ujbt6/0t729PVxcXNCiRQusX79eOjG1MvuoNvTteatXr0bfvn1V/lOry/uuLNW1r2qyvwUFBRg2bBiKi4vx448/qsz79NNPpb/btWuHli1bwtnZGadPn0bHjh0B1N7+VedrsaZfr2vWrMHw4cOhpaWlMr0u7L8XfQ+Utd7a/P7jUF0d1LBhQ6ipqZVKyDdv3iyVpmuDL774Art378bhw4dhbm5eblkzMzNYWloiNTUVAGBqaor8/HzcvXtXpdyzfTU1NcU///xTqq5bt25V6/bQ1dWFvb09UlNTpavryttHdaVvV69exV9//YVPPvmk3HJ1dd9V574yNTUttZ67d++ioKDglfe3oKAAXl5eSEtLQ1hYmMrRprJ07NgR6urqKvuzNvfvWa/qtVjT/Tt27BhSUlJe+l4Eat/+e9H3QF18/zE41UEaGhpwcnKSDsGWCAsLg6uraw21qjQhBPz9/bFz506Eh4fD2tr6pctkZ2cjIyMDZmZmAAAnJyeoq6ur9DUzMxPnzp2T+uri4oKcnBycOnVKKhMdHY2cnJxq3R5PnjxBcnIyzMzMpEPmz7Y7Pz8fR44ckdpUV/q2du1amJiYoH///uWWq6v7rjr3lYuLC86dO4fMzEypzMGDB6GpqQknJ6dX1seS0JSamoq//voLxsbGL13m/PnzKCgokPZnbe7f817Va7Gm+7d69Wo4OTnBwcHhpWVry/572fdAnXz/yT6NnGqVktsRrF69WiQlJYnx48cLXV1dkZ6eXtNNk3z++efCwMBAREREqFwi++jRIyGEEPfv3xeTJk0SkZGRIi0tTRw+fFi4uLiIpk2blroM1dzcXPz111/i9OnT4q233irzMtT27duLqKgoERUVJezt7V/5JfuTJk0SERER4sqVK+LkyZNiwIABQqlUSvtgwYIFwsDAQOzcuVOcPXtWeHt7l3mJbW3sW4mioiLRrFkzMXXqVJXpdW3f3b9/X8THx4v4+HgBQHz//fciPj5euqqsuvZVyeXQvXr1EqdPnxZ//fWXMDc3/9eXs5fXv4KCAvHuu+8Kc3NzkZCQoPJefPLkiRBCiEuXLomgoCARExMj0tLSxB9//CFat24tOnToUOv7V52vxZroX4mcnByho6MjVq5cWWr52rz/XvY9IETde/8xONVhK1asEJaWlkJDQ0N07NhR5TL/2gBAmY+1a9cKIYR49OiR6NOnj2jUqJFQV1cXzZo1EyNGjBDXrl1Tqefx48fC399fGBkZCW1tbTFgwIBSZbKzs8Xw4cOFUqkUSqVSDB8+XNy9e/eV9q/kXiPq6uqiSZMmwtPTU5w/f16aX1xcLAIDA4WpqanQ1NQU3bp1E2fPnq0TfStx4MABAUCkpKSoTK9r++7w4cNlvhZHjBghhKjefXX16lXRv39/oa2tLYyMjIS/v7/Iy8t7Zf1LS0t74Xux5J5c165dE926dRNGRkZCQ0NDtGjRQnz55Zel7oVUG/tX3a/F6u5fiVWrVgltbe1S92YSonbvv5d9DwhR995/iv/fMSIiIiJ6CZ7jRERERCQTgxMRERGRTAxORERERDIxOBERERHJxOBEREREJBODExEREZFMDE5EREREMjE4EREREcnE4EREtV56ejoUCgUSEhJquimSCxcu4M0334SWlhYcHR1rrB2+vr4YNGhQja2f6H8NgxMRvZSvry8UCgUWLFigMj00NBQKhaKGWlWzAgMDoauri5SUFBw6dKimm0NE1YTBiYhk0dLSwrfffou7d+/WdFOqTH5+fqWXvXz5Mrp06QJLS0sYGxtXYauIqDZjcCIiWXr37g1TU1PMnz//hWVmz55dathq6dKlsLKykp6XDC3NmzcPjRs3hqGhIYKCglBYWIgpU6bAyMgI5ubmWLNmTan6L1y4AFdXV2hpaaFt27aIiIhQmZ+UlIR+/fpBT08PjRs3xocffojbt29L83v06AF/f39MnDgRDRs2xNtvv11mP4qLizFnzhyYm5tDU1MTjo6O2L9/vzRfoVAgLi4Oc+bMgUKhwOzZs8usp2R9/v7+MDQ0hLGxMWbOnIlnfyJ006ZNcHZ2hlKphKmpKXx8fHDz5k2Ves6fP4/+/ftDX18fSqUSXbt2xeXLl8tcZ1xcHExMTPDNN98AAPbv348uXbpI6x8wYECpZSMjI+Ho6AgtLS04OztLRxKfHRp92bb99ddfYW9vD21tbRgbG6N37954+PBhmW0kqssYnIhIFjU1NcybNw/Lly/H33///a/qCg8Px40bN3D06FF8//33mD17NgYMGIAGDRogOjoao0ePxujRo5GRkaGy3JQpUzBp0iTEx8fD1dUV7777LrKzswEAmZmZ6N69OxwdHREbG4v9+/fjn3/+gZeXl0od69evR/369XHixAmsWrWqzPYtW7YMixcvxqJFi3DmzBm4u7vj3XffRWpqqrSutm3bYtKkScjMzMTkyZNf2NeS9UVHR+M///kPlixZgv/7v/+T5ufn5yM4OBiJiYkIDQ1FWloafH19pfnXr19Ht27doKWlhfDwcMTFxcHPzw+FhYWl1hUREYFevXohKCgIM2bMAAA8fPgQEydORExMDA4dOoR69eph8ODBKC4uBgDcv38fAwcOhL29PU6fPo3g4GBMnTpVpd6XbdvMzEx4e3vDz88PycnJiIiIgKenJ/gb8vRaEkRELzFixAjh4eEhhBDizTffFH5+fkIIIXbt2iWe/RgJDAwUDg4OKssuWbJEWFpaqtRlaWkpioqKpGmtWrUSXbt2lZ4XFhYKXV1dsXXrViGEEGlpaQKAWLBggVSmoKBAmJubi2+//VYIIcSsWbNEnz59VNadkZEhAIiUlBQhhBDdu3cXjo6OL+1vkyZNxDfffKMy7Y033hBjxoyRnjs4OIjAwMBy6+nevbto06aNKC4ulqZNnTpVtGnT5oXLnDp1SgAQ9+/fF0IIMX36dGFtbS3y8/PLLF+yb0JDQ4VSqRRbtmwpt003b94UAMTZs2eFEEKsXLlSGBsbi8ePH0tlQkJCBAARHx8vhHj5to2LixMARHp6ernrJnod8IgTEVXIt99+i/Xr1yMpKanSdbRt2xb16v3346dx48awt7eXnqupqcHY2LjUkJWLi4v0d/369eHs7Izk5GQAT4eoDh8+DD09PenRunVrAFAZmnJ2di63bbm5ubhx4wbc3NxUpru5uUnrqog333xT5QR6FxcXpKamoqioCAAQHx8PDw8PWFpaQqlUokePHgCAa9euAQASEhLQtWtXqKurv3Ad0dHReO+997B+/Xp4e3urzLt8+TJ8fHzQvHlz6Ovrw9raWqX+lJQUtG/fHlpaWtIynTp1UqnjZdvWwcEBvXr1gr29PYYMGYKQkJDX6lw4omcxOBFRhXTr1g3u7u74+uuvS82rV69eqeGZgoKCUuWeDwEKhaLMaSXDSeUpCSXFxcUYOHAgEhISVB6pqano1q2bVF5XV/eldT5bbwkhRJVfQfjw4UP06dMHenp62LRpE2JiYrBr1y4A/z1xXVtb+6X1tGjRAq1bt8aaNWtKnfA+cOBAZGdnIyQkBNHR0YiOjlapv6x+Pb8PX7Zt1dTUEBYWhj///BN2dnZYvnw5WrVqhbS0tMptGKJajMGJiCpswYIF2LNnDyIjI1WmN2rUCFlZWSpfvFV576WTJ09KfxcWFiIuLk468tGxY0ecP38eVlZWsLGxUXnIDUsAoK+vjyZNmuD48eMq0yMjI9GmTZt/1eaS5y1btoSamhouXLiA27dvY8GCBejatStat25d6ihb+/btcezYsTIDaImGDRsiPDwcly9fxtChQ6Wy2dnZSE5OxsyZM9GrVy+0adOm1JGg1q1b48yZM3jy5Ik0LTY2VqWMnG2rUCjg5uaGoKAgxMfHQ0NDQwqBRK8TBiciqjB7e3sMHz4cy5cvV5neo0cP3Lp1CwsXLsTly5exYsUK/Pnnn1W23hUrVmDXrl24cOECxo4di7t378LPzw8AMHbsWNy5cwfe3t44deoUrly5goMHD8LPz08aFpNrypQp+Pbbb7F9+3akpKRg2rRpSEhIwLhx4yrc5oyMDEycOBEpKSnYunUrli9fLtXTrFkzaGhoYPny5bhy5Qp2796N4OBgleX9/f2Rm5uLYcOGITY2Fqmpqdi4cSNSUlJUypmYmCA8PBwXLlyAt7c3CgsL0aBBAxgbG+Pnn3/GpUuXEB4ejokTJ6os5+Pjg+LiYowaNQrJyck4cOAAFi1aBOC/R91etm2jo6Mxb948xMbG4tq1a9i5cydu3bpVqaBJVNsxOBFRpQQHB5ca0mnTpg1+/PFHrFixAg4ODjh16lS5V5xV1IIFC/Dtt9/CwcEBx44dw++//46GDRsCAJo0aYITJ06gqKgI7u7uaNeuHcaNGwcDAwOV86nk+PLLLzFp0iRMmjQJ9vb22L9/P3bv3o2WLVtWuM0fffQRHj9+jE6dOmHs2LH44osvMGrUKABPj9CtW7cOv/zyC+zs7LBgwQIptJQwNjZGeHg4Hjx4gO7du8PJyQkhISFlnvNkamqK8PBwnD17FsOHD4cQAtu2bUNcXBzatWuHCRMm4LvvvlNZRl9fH3v27EFCQgIcHR0xY8YMBAQEAIB03tPLtq2+vj6OHj2Kfv36wdbWFjNnzsTixYvRt2/fCm8votpOIZ7/5CMioirRo0cPODo6YunSpTXdlArZvHkzPv74Y+Tk5Mg6x4rof0n9mm4AERHVrA0bNqB58+Zo2rQpEhMTMXXqVHh5eTE0EZWBwYmI6H9cVlYWAgICkJWVBTMzMwwZMkS68zgRqeJQHREREZFMPDmciIiISCYGJyIiIiKZGJyIiIiIZGJwIiIiIpKJwYmIiIhIJgYnIiIiIpkYnIiIiIhkYnAiIiIikonBiYiIiEim/weW4aBNl7k46QAAAABJRU5ErkJggg==",
      "text/plain": [
       "<Figure size 600x400 with 1 Axes>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "# Graphical representation of the previous results with a bar chart\n",
    "# ------------------------------------------------------------------\n",
    "\n",
    "# We create the data to represent in the bar chart\n",
    "data1 = [len(packages_librariesio_names), len(packages_scraping_names)]\n",
    "data2 = [len(packages_librariesio_not_in_scraping), len(packages_scraping_not_in_librariesio)]\n",
    "labels = [\"libraries.io\", \"scraping\"]\n",
    "plt.figure(figsize=(6, 4))\n",
    "bar_width = 0.35\n",
    "opacity = 0.8\n",
    "index = np.arange(len(labels))\n",
    "\n",
    "# We create the bars in horizontal position\n",
    "rects1 = plt.barh(index, data1, bar_width, alpha=opacity, color='b', label='Packages in both datasets')\n",
    "rects2 = plt.barh(index + bar_width, data2, bar_width, alpha=opacity, color='g', label='Packages in one dataset, but not in the other')\n",
    "\n",
    "# We add the labels to the bars\n",
    "plt.xlabel('Number of packages')\n",
    "plt.ylabel('Datasets')\n",
    "plt.title('Packages')\n",
    "plt.yticks(index + bar_width, labels)\n",
    "plt.legend()\n",
    "\n",
    "plt.tight_layout()\n",
    "plt.show()\n"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "#### **2.2.2.2 - Scrape packages**"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "The web scraping process is very expensive due to the multiple requests that are going to be made on the server, and the long time that this task requires."
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "**<span style=\"color:yellow\">Load data from file</span>**\n",
    "\n",
    "We get the package list from the pkl file"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [],
   "source": [
    "file_name = \"data\" + os.sep + \"CRAN_package_name_list.pkl\"\n",
    "\n",
    "# We open the file in read mode\n",
    "with open(file_name, \"rb\") as f:\n",
    "    # We load the list of packages\n",
    "    CRAN_package_name_list = pickle.load(f)"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "**<span style=\"color:yellow\">Do the scraping and save the objects file</span>**\n",
    "\n",
    "Once we have the list with the packages we have to obtain, we execute the next code and the scraping process begins."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "scraper = Cran_Scraper(RequestHandler())        # Create object of class CranScraper\n",
    "CRAN_packages_objects_list = scraper.build_list(CRAN_package_name_list)\n",
    "\n",
    "# Save the list of packages objects in a pickle file\n",
    "file_name = \"data\" + os.sep + \"CRAN_package_object_list.pkl\"\n",
    "\n",
    "# Drop file if it exists\n",
    "if os.path.exists(file_name):\n",
    "    os.remove(file_name)\n",
    "\n",
    "# We open the file in write mode\n",
    "with open(file_name, \"wb\") as f:\n",
    "    # We dump the packages list in the file\n",
    "    pickle.dump(CRAN_packages_objects_list, f)"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "**<span style=\"color:yellow\">See a package</span>**\n",
    "\n",
    "Once the process is finished we can obtain any **CRAN** package"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Name: A3\n",
      "Description:\n",
      "Supplies tools for tabulating and analyzing the results of predictive models. The methods employed are applicable to virtually any predictive model and make comparisons between different methodologies straightforward.\n",
      "Version: 1.0.0\n",
      "Publication date: 2015-08-16\n",
      "Mantainer: Scott Fortmann-Roe <scottfr@berkeley.edu>\n",
      "Authors:\n",
      "Scott Fortmann-Roe\n",
      "Dependencies:\n",
      "  - R (≥ 2.15.0) type: depends\n",
      "  - xtable, type: depends\n",
      "  - pbapply, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: True\n",
      "In Bioconductor: None\n",
      "Licenses:\n",
      "GPL-2 | GPL-3 [expanded from: GPL (≥ 2)]\n",
      "Links:\n",
      "https://cran.r-project.org/package=A3\n",
      "\n"
     ]
    }
   ],
   "source": [
    "p = CRAN_packages_objects_list[0]  # We get the first package object\n",
    "p.dump()"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### **2.2.3 - Scraping Bioconductor website**"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "**<span style=\"color:yellow\">Get data from bioconductor.org</span>**\n",
    "\n",
    "Making scraping at bioconductor webpage is more difficult.\n",
    "\n",
    "The problem is that the bioconductor web server loads the data in the HTML using JQuery, so these data can only be obtained  from the browser.\n",
    "\n",
    "The above can be verified, if you do the scraping process and then check the document *data/bioconductor_raw.html* you can see that the data table is empty."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Scraping the Bioconductor website\n",
    "url = \"https://www.bioconductor.org/packages/release/BiocViews.html#___Software\"\n",
    "response = requests.get(url)\n",
    "soup = BeautifulSoup(response.content, \"html.parser\")\n",
    "\n",
    "# Save the html file\n",
    "with open(\"data\" + os.sep + \"bioconductor_raw.html\", \"w\") as f:\n",
    "    f.write(str(soup))\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "!cat data/bioconductor_raw.html"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "**As an alternative we have several options:**\n",
    "- <span style=\"color: yellow\">Easy way:</span> We can manually save the HTML with the rendered data using our browser.\n",
    "\n",
    "    It is as simple as addressing the URL with the pack of packages and save the document on the disk.\n",
    "\n",
    "- <span style=\"color:yellow\">Automated way:</span> We can use some Python library like **Selenium** that provides us an headless browser that will be able to run Javascript code, now containing if data to extract. \n",
    "\n",
    "    - https://selenium-python.readthedocs.io/\n",
    "\n",
    "    The following two cells show this process using selenium\n",
    "    \n",
    "    It is necessary to select a browser driver and load it in the code. In this case the driver *gecko* has been used for the browser *Firefox*\n",
    "\n",
    "    - https://github.com/mozilla/geckodriver/releases\n",
    "    \n",
    "    - https://sites.google.com/a/chromium.org/chromedriver/downloads\n",
    "\n",
    "    The driver container folder must be available at PATH"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "**Add the folder containing the driver to the Path**"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Add the folder where the Firefox's selenium driver \n",
    "sys.path.append(\"./drivers/\")\n",
    "\n",
    "# Check the path\n",
    "for path in sys.path:\n",
    "    print(path)"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "**Save the data table on an HTML file using selenium**"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [],
   "source": [
    "\n",
    "driver = webdriver.Firefox()\n",
    "driver.get(\"https://www.bioconductor.org/packages/release/BiocViews.html#___Software\")\n",
    "table = driver.find_element(By.ID, \"biocViews_package_table\")\n",
    "table_content = table.get_attribute(\"innerHTML\")\n",
    "\n",
    "# Store the content of the table in an HTML file\n",
    "with open(\"data\" + os.sep + \"bioconductor_selenium.html\", \"w\") as f:\n",
    "    f.write(table_content)\n",
    "\n",
    "driver.close()"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Now if everything went well we have the data correctly loaded"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "!cat data/bioconductor_selenium.html"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "**<span style=\"color:yellow\">Do the scraping and build a package name list backup file</span>**\n",
    "\n",
    "Once we have the data, we can continue using ***BeautifulSoup*** to extract them"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Number of packages:  2183\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "['BiocGenerics', 'S4Vectors', 'BiocVersion', 'GenomeInfoDb', 'IRanges']"
      ]
     },
     "execution_count": 4,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "soup = BeautifulSoup(open(\"data\" + os.sep + \"bioconductor_selenium.html\"), \"html.parser\")\n",
    "\n",
    "# Process the HTML to obtain packages\n",
    "bioc_packages = []\n",
    "for row in soup.find_all(\"tr\"):\n",
    "    for cell in row.find_all(\"td\"):\n",
    "        if cell.find(\"a\"):\n",
    "            bioc_packages.append(cell.find(\"a\").text)\n",
    "\n",
    "print(\"Number of packages: \", len(bioc_packages))\n",
    "bioc_packages[:5]"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "**<span style=\"color:yellow\">Save the package name list as a file</span>**\n",
    "\n",
    "We save the data in a file so as not to repeat the process"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [],
   "source": [
    "# file_name = \"bioconductor_package_name_list.pkl\"\n",
    "# # Delete the file if it already exists\n",
    "# if os.path.exists(\"data\" + os.sep + file_name):\n",
    "#     os.remove(\"data\" + os.sep + file_name)\n",
    "\n",
    "# # Save the list of packages in a pkl file\n",
    "# with open(\"data\" + os.sep + file_name, \"wb\") as f:\n",
    "#     pickle.dump(bioc_packages, f)"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "**<span style=\"color:yellow\">Load the data from the names backup file</span>**\n",
    "\n",
    "Load the data from the file, so we avoid performing the process of obtaining again"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Number of packages:  2183\n"
     ]
    }
   ],
   "source": [
    "file_name = \"bioconductor_package_name_list.pkl\"\n",
    "# Load the list of packages from the pkl file\n",
    "with open(\"data\" + os.sep + file_name, \"rb\") as f:\n",
    "    bioc_packages = pickle.load(f)\n",
    "\n",
    "print(\"Number of packages: \", len(bioc_packages))"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "**We can easily obtain the data in the following way**"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [],
   "source": [
    "scraper = Bioc_Scraper(RequestHandler())"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Name: a4\n",
      "Description:\n",
      "Umbrella package is available for the entire Automated Affymetrix Array Analysis suite of package.\n",
      "Version: 1.46.0\n",
      "Publication date: None\n",
      "Mantainer: Laure Cougnaud <laure.cougnaud@openanalytics.eu>\n",
      "Authors:\n",
      "Author: Willem Talloen [aut], Tobias Verbeke [aut], Laure Cougnaud [cre]\n",
      "Dependencies:\n",
      "  - a4Base, type: depends\n",
      "  - a4Preproc, type: depends\n",
      "  - a4Classif, type: depends\n",
      "  - a4Core, type: depends\n",
      "  - a4Reporting, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/a4.html\n",
      "\n"
     ]
    }
   ],
   "source": [
    "p = scraper.build(\"a4\")\n",
    "p.dump()"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "**<span style=\"color:yellow\">Do the scraping of the package data and build objects backup file</span>**\n",
    "\n",
    "\n",
    "We proceed to obtain all the bioconductor packages"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mScraped package BiocGenerics: 1/2183\u001b[0m\n",
      "\u001b[32mScraped package S4Vectors: 2/2183\u001b[0m\n",
      "\u001b[32mScraped package BiocVersion: 3/2183\u001b[0m\n",
      "\u001b[32mScraped package GenomeInfoDb: 4/2183\u001b[0m\n",
      "\u001b[32mScraped package IRanges: 5/2183\u001b[0m\n",
      "\u001b[32mScraped package Biobase: 6/2183\u001b[0m\n",
      "\u001b[32mScraped package zlibbioc: 7/2183\u001b[0m\n",
      "\u001b[32mScraped package XVector: 8/2183\u001b[0m\n",
      "\u001b[32mScraped package Biostrings: 9/2183\u001b[0m\n",
      "\u001b[32mScraped package BiocParallel: 10/2183\u001b[0m\n",
      "\u001b[32mScraped package GenomicRanges: 11/2183\u001b[0m\n",
      "\u001b[32mScraped package DelayedArray: 12/2183\u001b[0m\n",
      "\u001b[32mScraped package SummarizedExperiment: 13/2183\u001b[0m\n",
      "\u001b[32mScraped package MatrixGenerics: 14/2183\u001b[0m\n",
      "\u001b[32mScraped package limma: 15/2183\u001b[0m\n",
      "\u001b[32mScraped package AnnotationDbi: 16/2183\u001b[0m\n",
      "\u001b[32mScraped package KEGGREST: 17/2183\u001b[0m\n",
      "\u001b[32mScraped package annotate: 18/2183\u001b[0m\n",
      "\u001b[32mScraped package BiocFileCache: 19/2183\u001b[0m\n",
      "\u001b[32mScraped package Rhtslib: 20/2183\u001b[0m\n",
      "\u001b[32mScraped package Rsamtools: 21/2183\u001b[0m\n",
      "\u001b[32mScraped package genefilter: 22/2183\u001b[0m\n",
      "\u001b[32mScraped package edgeR: 23/2183\u001b[0m\n",
      "\u001b[32mScraped package GenomicAlignments: 24/2183\u001b[0m\n",
      "\u001b[32mScraped package biomaRt: 25/2183\u001b[0m\n",
      "\u001b[32mScraped package rtracklayer: 26/2183\u001b[0m\n",
      "\u001b[32mScraped package Rhdf5lib: 27/2183\u001b[0m\n",
      "\u001b[32mScraped package graph: 28/2183\u001b[0m\n",
      "\u001b[32mScraped package DESeq2: 29/2183\u001b[0m\n",
      "\u001b[32mScraped package geneplotter: 30/2183\u001b[0m\n",
      "\u001b[32mScraped package GenomicFeatures: 31/2183\u001b[0m\n",
      "\u001b[32mScraped package rhdf5: 32/2183\u001b[0m\n",
      "\u001b[32mScraped package ggtree: 33/2183\u001b[0m\n",
      "\u001b[32mScraped package rhdf5filters: 34/2183\u001b[0m\n",
      "\u001b[32mScraped package treeio: 35/2183\u001b[0m\n",
      "\u001b[32mScraped package BiocIO: 36/2183\u001b[0m\n",
      "\u001b[32mScraped package qvalue: 37/2183\u001b[0m\n",
      "\u001b[32mScraped package preprocessCore: 38/2183\u001b[0m\n",
      "\u001b[32mScraped package enrichplot: 39/2183\u001b[0m\n",
      "\u001b[32mScraped package fgsea: 40/2183\u001b[0m\n",
      "\u001b[32mScraped package DelayedMatrixStats: 41/2183\u001b[0m\n",
      "\u001b[32mScraped package DOSE: 42/2183\u001b[0m\n",
      "\u001b[32mScraped package clusterProfiler: 43/2183\u001b[0m\n",
      "\u001b[32mScraped package GOSemSim: 44/2183\u001b[0m\n",
      "\u001b[32mScraped package sparseMatrixStats: 45/2183\u001b[0m\n",
      "\u001b[32mScraped package SingleCellExperiment: 46/2183\u001b[0m\n",
      "\u001b[32mScraped package ComplexHeatmap: 47/2183\u001b[0m\n",
      "\u001b[32mScraped package beachmat: 48/2183\u001b[0m\n",
      "\u001b[32mScraped package HDF5Array: 49/2183\u001b[0m\n",
      "\u001b[32mScraped package BSgenome: 50/2183\u001b[0m\n",
      "\u001b[32mScraped package multtest: 51/2183\u001b[0m\n",
      "\u001b[32mScraped package GEOquery: 52/2183\u001b[0m\n",
      "\u001b[32mScraped package BiocSingular: 53/2183\u001b[0m\n",
      "\u001b[32mScraped package RBGL: 54/2183\u001b[0m\n",
      "\u001b[32mScraped package Rgraphviz: 55/2183\u001b[0m\n",
      "\u001b[32mScraped package ProtGenerics: 56/2183\u001b[0m\n",
      "\u001b[32mScraped package ScaledMatrix: 57/2183\u001b[0m\n",
      "\u001b[32mScraped package impute: 58/2183\u001b[0m\n",
      "\u001b[32mScraped package AnnotationHub: 59/2183\u001b[0m\n",
      "\u001b[32mScraped package ensembldb: 60/2183\u001b[0m\n",
      "\u001b[32mScraped package affyio: 61/2183\u001b[0m\n",
      "\u001b[32mScraped package affy: 62/2183\u001b[0m\n",
      "\u001b[32mScraped package interactiveDisplayBase: 63/2183\u001b[0m\n",
      "\u001b[32mScraped package AnnotationFilter: 64/2183\u001b[0m\n",
      "\u001b[32mScraped package GSEABase: 65/2183\u001b[0m\n",
      "\u001b[32mScraped package BiocNeighbors: 66/2183\u001b[0m\n",
      "\u001b[32mScraped package scuttle: 67/2183\u001b[0m\n",
      "\u001b[32mScraped package VariantAnnotation: 68/2183\u001b[0m\n",
      "\u001b[32mScraped package BiocStyle: 69/2183\u001b[0m\n",
      "\u001b[32mScraped package sva: 70/2183\u001b[0m\n",
      "\u001b[32mScraped package ShortRead: 71/2183\u001b[0m\n",
      "\u001b[32mScraped package ExperimentHub: 72/2183\u001b[0m\n",
      "\u001b[32mScraped package scater: 73/2183\u001b[0m\n",
      "\u001b[32mScraped package vsn: 74/2183\u001b[0m\n",
      "\u001b[32mScraped package KEGGgraph: 75/2183\u001b[0m\n",
      "\u001b[32mScraped package biovizBase: 76/2183\u001b[0m\n",
      "\u001b[32mScraped package biomformat: 77/2183\u001b[0m\n",
      "\u001b[32mScraped package GSVA: 78/2183\u001b[0m\n",
      "\u001b[32mScraped package pcaMethods: 79/2183\u001b[0m\n",
      "\u001b[32mScraped package phyloseq: 80/2183\u001b[0m\n",
      "\u001b[32mScraped package pathview: 81/2183\u001b[0m\n",
      "\u001b[32mScraped package biocViews: 82/2183\u001b[0m\n",
      "\u001b[32mScraped package scran: 83/2183\u001b[0m\n",
      "\u001b[32mScraped package DNAcopy: 84/2183\u001b[0m\n",
      "\u001b[32mScraped package bluster: 85/2183\u001b[0m\n",
      "\u001b[32mScraped package EnhancedVolcano: 86/2183\u001b[0m\n",
      "\u001b[32mScraped package metapod: 87/2183\u001b[0m\n",
      "\u001b[32mScraped package Gviz: 88/2183\u001b[0m\n",
      "\u001b[32mScraped package apeglm: 89/2183\u001b[0m\n",
      "\u001b[32mScraped package tximport: 90/2183\u001b[0m\n",
      "\u001b[32mScraped package TCGAbiolinks: 91/2183\u001b[0m\n",
      "\u001b[32mScraped package batchelor: 92/2183\u001b[0m\n",
      "\u001b[32mScraped package topGO: 93/2183\u001b[0m\n",
      "\u001b[32mScraped package MSnbase: 94/2183\u001b[0m\n",
      "\u001b[32mScraped package OrganismDbi: 95/2183\u001b[0m\n",
      "\u001b[32mScraped package ConsensusClusterPlus: 96/2183\u001b[0m\n",
      "\u001b[32mScraped package SingleR: 97/2183\u001b[0m\n",
      "\u001b[32mScraped package ResidualMatrix: 98/2183\u001b[0m\n",
      "\u001b[32mScraped package mzR: 99/2183\u001b[0m\n",
      "\u001b[32mScraped package MultiAssayExperiment: 100/2183\u001b[0m\n",
      "\u001b[32mScraped package maftools: 101/2183\u001b[0m\n",
      "\u001b[32mScraped package illuminaio: 102/2183\u001b[0m\n",
      "\u001b[32mScraped package siggenes: 103/2183\u001b[0m\n",
      "\u001b[32mScraped package MsCoreUtils: 104/2183\u001b[0m\n",
      "\u001b[32mScraped package mixOmics: 105/2183\u001b[0m\n",
      "\u001b[32mScraped package monocle: 106/2183\u001b[0m\n",
      "\u001b[32mScraped package mzID: 107/2183\u001b[0m\n",
      "\u001b[32mScraped package AnnotationForge: 108/2183\u001b[0m\n",
      "\u001b[32mScraped package cytolib: 109/2183\u001b[0m\n",
      "\u001b[32mScraped package EBImage: 110/2183\u001b[0m\n",
      "\u001b[32mScraped package flowCore: 111/2183\u001b[0m\n",
      "\u001b[32mScraped package bumphunter: 112/2183\u001b[0m\n",
      "\u001b[32mScraped package graphite: 113/2183\u001b[0m\n",
      "\u001b[32mScraped package dada2: 114/2183\u001b[0m\n",
      "\u001b[32mScraped package RProtoBufLib: 115/2183\u001b[0m\n",
      "\u001b[32mScraped package Rsubread: 116/2183\u001b[0m\n",
      "\u001b[32mScraped package DirichletMultinomial: 117/2183\u001b[0m\n",
      "\u001b[32mScraped package seqLogo: 118/2183\u001b[0m\n",
      "\u001b[32mScraped package DropletUtils: 119/2183\u001b[0m\n",
      "\u001b[32mScraped package Category: 120/2183\u001b[0m\n",
      "\u001b[32mScraped package minfi: 121/2183\u001b[0m\n",
      "\u001b[32mScraped package ggbio: 122/2183\u001b[0m\n",
      "\u001b[32mScraped package DECIPHER: 123/2183\u001b[0m\n",
      "\u001b[32mScraped package regioneR: 124/2183\u001b[0m\n",
      "\u001b[32mScraped package ReactomePA: 125/2183\u001b[0m\n",
      "\u001b[32mScraped package GOstats: 126/2183\u001b[0m\n",
      "\u001b[32mScraped package gdsfmt: 127/2183\u001b[0m\n",
      "\u001b[32mScraped package aroma.light: 128/2183\u001b[0m\n",
      "\u001b[32mScraped package snpStats: 129/2183\u001b[0m\n",
      "\u001b[32mScraped package BiocCheck: 130/2183\u001b[0m\n",
      "\u001b[32mScraped package ChIPseeker: 131/2183\u001b[0m\n",
      "\u001b[32mScraped package marray: 132/2183\u001b[0m\n",
      "\u001b[32mScraped package EDASeq: 133/2183\u001b[0m\n",
      "\u001b[32mScraped package AUCell: 134/2183\u001b[0m\n",
      "\u001b[32mScraped package oligoClasses: 135/2183\u001b[0m\n",
      "\u001b[32mScraped package MAST: 136/2183\u001b[0m\n",
      "\u001b[32mScraped package microbiome: 137/2183\u001b[0m\n",
      "\u001b[32mScraped package TFBSTools: 138/2183\u001b[0m\n",
      "\u001b[32mScraped package CNEr: 139/2183\u001b[0m\n",
      "\u001b[32mScraped package gcrma: 140/2183\u001b[0m\n",
      "\u001b[32mScraped package affxparser: 141/2183\u001b[0m\n",
      "\u001b[32mScraped package oligo: 142/2183\u001b[0m\n",
      "\u001b[32mScraped package glmGamPoi: 143/2183\u001b[0m\n",
      "\u001b[32mScraped package ncdfFlow: 144/2183\u001b[0m\n",
      "\u001b[32mScraped package flowWorkspace: 145/2183\u001b[0m\n",
      "\u001b[32mScraped package MassSpecWavelet: 146/2183\u001b[0m\n",
      "\u001b[32mScraped package affyPLM: 147/2183\u001b[0m\n",
      "\u001b[32mScraped package methylumi: 148/2183\u001b[0m\n",
      "\u001b[32mScraped package globaltest: 149/2183\u001b[0m\n",
      "\u001b[32mScraped package flowViz: 150/2183\u001b[0m\n",
      "\u001b[32mScraped package SNPRelate: 151/2183\u001b[0m\n",
      "\u001b[32mScraped package xcms: 152/2183\u001b[0m\n",
      "\u001b[32mScraped package basilisk: 153/2183\u001b[0m\n",
      "\u001b[32mScraped package Glimma: 154/2183\u001b[0m\n",
      "\u001b[32mScraped package goseq: 155/2183\u001b[0m\n",
      "\u001b[32mScraped package basilisk.utils: 156/2183\u001b[0m\n",
      "\u001b[32mScraped package ggcyto: 157/2183\u001b[0m\n",
      "\u001b[32mScraped package bsseq: 158/2183\u001b[0m\n",
      "\u001b[32mScraped package msa: 159/2183\u001b[0m\n",
      "\u001b[32mScraped package InteractionSet: 160/2183\u001b[0m\n",
      "\u001b[32mScraped package dir.expiry: 161/2183\u001b[0m\n",
      "\u001b[32mScraped package systemPipeR: 162/2183\u001b[0m\n",
      "\u001b[32mScraped package TrajectoryUtils: 163/2183\u001b[0m\n",
      "\u001b[32mScraped package flowClust: 164/2183\u001b[0m\n",
      "\u001b[32mScraped package ROC: 165/2183\u001b[0m\n",
      "\u001b[32mScraped package lumi: 166/2183\u001b[0m\n",
      "\u001b[32mScraped package openCyto: 167/2183\u001b[0m\n",
      "\u001b[32mScraped package M3C: 168/2183\u001b[0m\n",
      "\u001b[32mScraped package flowStats: 169/2183\u001b[0m\n",
      "\u001b[32mScraped package DiffBind: 170/2183\u001b[0m\n",
      "\u001b[32mScraped package slingshot: 171/2183\u001b[0m\n",
      "\u001b[32mScraped package MsFeatures: 172/2183\u001b[0m\n",
      "\u001b[32mScraped package DEXSeq: 173/2183\u001b[0m\n",
      "\u001b[32mScraped package GenomicDataCommons: 174/2183\u001b[0m\n",
      "\u001b[32mScraped package lpsymphony: 175/2183\u001b[0m\n",
      "\u001b[32mScraped package tximeta: 176/2183\u001b[0m\n",
      "\u001b[32mScraped package DSS: 177/2183\u001b[0m\n",
      "\u001b[32mScraped package CytoML: 178/2183\u001b[0m\n",
      "\u001b[32mScraped package PCAtools: 179/2183\u001b[0m\n",
      "\u001b[32mScraped package gage: 180/2183\u001b[0m\n",
      "\u001b[32mScraped package widgetTools: 181/2183\u001b[0m\n",
      "\u001b[32mScraped package Wrench: 182/2183\u001b[0m\n",
      "\u001b[32mScraped package DynDoc: 183/2183\u001b[0m\n",
      "\u001b[32mScraped package motifmatchr: 184/2183\u001b[0m\n",
      "\u001b[32mScraped package metagenomeSeq: 185/2183\u001b[0m\n",
      "\u001b[32mScraped package tkWidgets: 186/2183\u001b[0m\n",
      "\u001b[32mScraped package bamsignals: 187/2183\u001b[0m\n",
      "\u001b[32mScraped package ChIPpeakAnno: 188/2183\u001b[0m\n",
      "\u001b[32mScraped package FlowSOM: 189/2183\u001b[0m\n",
      "\u001b[32mScraped package chromVAR: 190/2183\u001b[0m\n",
      "\u001b[32mScraped package missMethyl: 191/2183\u001b[0m\n",
      "\u001b[32mScraped package RcisTarget: 192/2183\u001b[0m\n",
      "\u001b[32mScraped package GenomicFiles: 193/2183\u001b[0m\n",
      "\u001b[32mScraped package GENIE3: 194/2183\u001b[0m\n",
      "\u001b[32mScraped package plyranges: 195/2183\u001b[0m\n",
      "\u001b[32mScraped package DMRcate: 196/2183\u001b[0m\n",
      "\u001b[32mScraped package RaggedExperiment: 197/2183\u001b[0m\n",
      "\u001b[32mScraped package RUVSeq: 198/2183\u001b[0m\n",
      "\u001b[32mScraped package STRINGdb: 199/2183\u001b[0m\n",
      "\u001b[32mScraped package infercnv: 200/2183\u001b[0m\n",
      "\u001b[32mScraped package singscore: 201/2183\u001b[0m\n",
      "\u001b[32mScraped package decontam: 202/2183\u001b[0m\n",
      "\u001b[32mScraped package Mfuzz: 203/2183\u001b[0m\n",
      "\u001b[32mScraped package SeqArray: 204/2183\u001b[0m\n",
      "\u001b[32mScraped package scDblFinder: 205/2183\u001b[0m\n",
      "\u001b[32mScraped package GreyListChIP: 206/2183\u001b[0m\n",
      "\u001b[32mScraped package survcomp: 207/2183\u001b[0m\n",
      "\u001b[32mScraped package karyoploteR: 208/2183\u001b[0m\n",
      "\u001b[32mScraped package ReportingTools: 209/2183\u001b[0m\n",
      "\u001b[32mScraped package wateRmelon: 210/2183\u001b[0m\n",
      "\u001b[32mScraped package ropls: 211/2183\u001b[0m\n",
      "\u001b[32mScraped package MultiDataSet: 212/2183\u001b[0m\n",
      "\u001b[32mScraped package ALDEx2: 213/2183\u001b[0m\n",
      "\u001b[32mScraped package ANCOMBC: 214/2183\u001b[0m\n",
      "\u001b[32mScraped package ggtreeExtra: 215/2183\u001b[0m\n",
      "\u001b[32mScraped package TCGAutils: 216/2183\u001b[0m\n",
      "\u001b[32mScraped package RCy3: 217/2183\u001b[0m\n",
      "\u001b[32mScraped package SpatialExperiment: 218/2183\u001b[0m\n",
      "\u001b[32mScraped package destiny: 219/2183\u001b[0m\n",
      "\u001b[32mScraped package Ringo: 220/2183\u001b[0m\n",
      "\u001b[32mScraped package BeadDataPackR: 221/2183\u001b[0m\n",
      "\u001b[32mScraped package beadarray: 222/2183\u001b[0m\n",
      "\u001b[32mScraped package Repitools: 223/2183\u001b[0m\n",
      "\u001b[32mScraped package dittoSeq: 224/2183\u001b[0m\n",
      "\u001b[32mScraped package fastseg: 225/2183\u001b[0m\n",
      "\u001b[32mScraped package AnVIL: 226/2183\u001b[0m\n",
      "\u001b[32mScraped package zinbwave: 227/2183\u001b[0m\n",
      "\u001b[32mScraped package BiocBaseUtils: 228/2183\u001b[0m\n",
      "\u001b[32mScraped package TreeSummarizedExperiment: 229/2183\u001b[0m\n",
      "\u001b[32mScraped package ChAMP: 230/2183\u001b[0m\n",
      "\u001b[32mScraped package Nebulosa: 231/2183\u001b[0m\n",
      "\u001b[32mScraped package GlobalAncova: 232/2183\u001b[0m\n",
      "\u001b[32mScraped package viper: 233/2183\u001b[0m\n",
      "\u001b[32mScraped package IHW: 234/2183\u001b[0m\n",
      "\u001b[32mScraped package mia: 235/2183\u001b[0m\n",
      "\u001b[32mScraped package quantsmooth: 236/2183\u001b[0m\n",
      "\u001b[32mScraped package motifStack: 237/2183\u001b[0m\n",
      "\u001b[32mScraped package ChemmineR: 238/2183\u001b[0m\n",
      "\u001b[32mScraped package GWASTools: 239/2183\u001b[0m\n",
      "\u001b[32mScraped package minet: 240/2183\u001b[0m\n",
      "\u001b[32mScraped package baySeq: 241/2183\u001b[0m\n",
      "\u001b[32mScraped package ballgown: 242/2183\u001b[0m\n",
      "\u001b[32mScraped package ArrayExpress: 243/2183\u001b[0m\n",
      "\u001b[32mScraped package genomation: 244/2183\u001b[0m\n",
      "\u001b[32mScraped package seqPattern: 245/2183\u001b[0m\n",
      "\u001b[32mScraped package NOISeq: 246/2183\u001b[0m\n",
      "\u001b[32mScraped package zellkonverter: 247/2183\u001b[0m\n",
      "\u001b[32mScraped package RTCGAToolbox: 248/2183\u001b[0m\n",
      "\u001b[32mScraped package arrayQualityMetrics: 249/2183\u001b[0m\n",
      "\u001b[32mScraped package chipseq: 250/2183\u001b[0m\n",
      "\u001b[32mScraped package RTCGA: 251/2183\u001b[0m\n",
      "\u001b[32mScraped package methylKit: 252/2183\u001b[0m\n",
      "\u001b[32mScraped package scds: 253/2183\u001b[0m\n",
      "\u001b[32mScraped package MotifDb: 254/2183\u001b[0m\n",
      "\u001b[32mScraped package DEGreport: 255/2183\u001b[0m\n",
      "\u001b[32mScraped package R4RNA: 256/2183\u001b[0m\n",
      "\u001b[32mScraped package variancePartition: 257/2183\u001b[0m\n",
      "\u001b[32mScraped package flowAI: 258/2183\u001b[0m\n",
      "\u001b[32mScraped package derfinder: 259/2183\u001b[0m\n",
      "\u001b[32mScraped package derfinderHelper: 260/2183\u001b[0m\n",
      "\u001b[32mScraped package rGREAT: 261/2183\u001b[0m\n",
      "\u001b[32mScraped package SeqVarTools: 262/2183\u001b[0m\n",
      "\u001b[32mScraped package CAMERA: 263/2183\u001b[0m\n",
      "\u001b[32mScraped package Maaslin2: 264/2183\u001b[0m\n",
      "\u001b[32mScraped package SPIA: 265/2183\u001b[0m\n",
      "\u001b[32mScraped package GenomicScores: 266/2183\u001b[0m\n",
      "\u001b[32mScraped package DepecheR: 267/2183\u001b[0m\n",
      "\u001b[32mScraped package sesame: 268/2183\u001b[0m\n",
      "\u001b[32mScraped package DEP: 269/2183\u001b[0m\n",
      "\u001b[32mScraped package Rdisop: 270/2183\u001b[0m\n",
      "\u001b[32mScraped package LEA: 271/2183\u001b[0m\n",
      "\u001b[32mScraped package EnrichedHeatmap: 272/2183\u001b[0m\n",
      "\u001b[32mScraped package safe: 273/2183\u001b[0m\n",
      "\u001b[32mScraped package SC3: 274/2183\u001b[0m\n",
      "\u001b[32mScraped package universalmotif: 275/2183\u001b[0m\n",
      "\u001b[32mScraped package annotatr: 276/2183\u001b[0m\n",
      "\u001b[32mScraped package GenVisR: 277/2183\u001b[0m\n",
      "\u001b[32mScraped package Rbowtie: 278/2183\u001b[0m\n",
      "\u001b[32mScraped package gwascat: 279/2183\u001b[0m\n",
      "\u001b[32mScraped package M3Drop: 280/2183\u001b[0m\n",
      "\u001b[32mScraped package scmap: 281/2183\u001b[0m\n",
      "\u001b[32mScraped package CATALYST: 282/2183\u001b[0m\n",
      "\u001b[32mScraped package pcaExplorer: 283/2183\u001b[0m\n",
      "\u001b[32mScraped package copynumber: 284/2183\u001b[0m\n",
      "\u001b[32mScraped package MutationalPatterns: 285/2183\u001b[0m\n",
      "\u001b[32mScraped package rGADEM: 286/2183\u001b[0m\n",
      "\u001b[32mScraped package MSstats: 287/2183\u001b[0m\n",
      "\u001b[32mScraped package mbkmeans: 288/2183\u001b[0m\n",
      "\u001b[32mScraped package UCell: 289/2183\u001b[0m\n",
      "\u001b[32mScraped package affycoretools: 290/2183\u001b[0m\n",
      "\u001b[32mScraped package Organism.dplyr: 291/2183\u001b[0m\n",
      "\u001b[32mScraped package Spectra: 292/2183\u001b[0m\n",
      "\u001b[32mScraped package splatter: 293/2183\u001b[0m\n",
      "\u001b[32mScraped package recount: 294/2183\u001b[0m\n",
      "\u001b[32mScraped package qusage: 295/2183\u001b[0m\n",
      "\u001b[32mScraped package DRIMSeq: 296/2183\u001b[0m\n",
      "\u001b[32mScraped package Heatplus: 297/2183\u001b[0m\n",
      "\u001b[32mScraped package EBSeq: 298/2183\u001b[0m\n",
      "\u001b[32mScraped package SRAdb: 299/2183\u001b[0m\n",
      "\u001b[32mScraped package AIMS: 300/2183\u001b[0m\n",
      "\u001b[32mScraped package fishpond: 301/2183\u001b[0m\n",
      "\u001b[32mScraped package tradeSeq: 302/2183\u001b[0m\n",
      "\u001b[32mScraped package ChIPQC: 303/2183\u001b[0m\n",
      "\u001b[32mScraped package genefu: 304/2183\u001b[0m\n",
      "\u001b[32mScraped package lfa: 305/2183\u001b[0m\n",
      "\u001b[32mScraped package sangerseqR: 306/2183\u001b[0m\n",
      "\u001b[32mScraped package piano: 307/2183\u001b[0m\n",
      "\u001b[32mScraped package TSCAN: 308/2183\u001b[0m\n",
      "\u001b[32mScraped package mygene: 309/2183\u001b[0m\n",
      "\u001b[32mScraped package csaw: 310/2183\u001b[0m\n",
      "\u001b[32mScraped package celda: 311/2183\u001b[0m\n",
      "\u001b[32mScraped package QFeatures: 312/2183\u001b[0m\n",
      "\u001b[32mScraped package supraHex: 313/2183\u001b[0m\n",
      "\u001b[32mScraped package progeny: 314/2183\u001b[0m\n",
      "\u001b[32mScraped package OmnipathR: 315/2183\u001b[0m\n",
      "\u001b[32mScraped package QuasR: 316/2183\u001b[0m\n",
      "\u001b[32mScraped package CGHbase: 317/2183\u001b[0m\n",
      "\u001b[32mScraped package MSstatsConvert: 318/2183\u001b[0m\n",
      "\u001b[32mScraped package ATACseqQC: 319/2183\u001b[0m\n",
      "\u001b[32mScraped package scde: 320/2183\u001b[0m\n",
      "\u001b[32mScraped package LoomExperiment: 321/2183\u001b[0m\n",
      "\u001b[32mScraped package MicrobiotaProcess: 322/2183\u001b[0m\n",
      "\u001b[32mScraped package MOFA2: 323/2183\u001b[0m\n",
      "\u001b[32mScraped package GENESIS: 324/2183\u001b[0m\n",
      "\u001b[32mScraped package cBioPortalData: 325/2183\u001b[0m\n",
      "\u001b[32mScraped package MAGeCKFlute: 326/2183\u001b[0m\n",
      "\u001b[32mScraped package scMerge: 327/2183\u001b[0m\n",
      "\u001b[32mScraped package ggmsa: 328/2183\u001b[0m\n",
      "\u001b[32mScraped package rols: 329/2183\u001b[0m\n",
      "\u001b[32mScraped package GDCRNATools: 330/2183\u001b[0m\n",
      "\u001b[32mScraped package clusterExperiment: 331/2183\u001b[0m\n",
      "\u001b[32mScraped package iClusterPlus: 332/2183\u001b[0m\n",
      "\u001b[32mScraped package powerTCR: 333/2183\u001b[0m\n",
      "\u001b[32mScraped package trackViewer: 334/2183\u001b[0m\n",
      "\u001b[32mScraped package UniProt.ws: 335/2183\u001b[0m\n",
      "\u001b[32mScraped package CGHcall: 336/2183\u001b[0m\n",
      "\u001b[32mScraped package simplifyEnrichment: 337/2183\u001b[0m\n",
      "\u001b[32mScraped package rWikiPathways: 338/2183\u001b[0m\n",
      "\u001b[32mScraped package MLInterfaces: 339/2183\u001b[0m\n",
      "\u001b[32mScraped package LBE: 340/2183\u001b[0m\n",
      "\u001b[32mScraped package rpx: 341/2183\u001b[0m\n",
      "\u001b[32mScraped package BumpyMatrix: 342/2183\u001b[0m\n",
      "\u001b[32mScraped package DEsingle: 343/2183\u001b[0m\n",
      "\u001b[32mScraped package TCC: 344/2183\u001b[0m\n",
      "\u001b[32mScraped package rrvgo: 345/2183\u001b[0m\n",
      "\u001b[32mScraped package EnrichmentBrowser: 346/2183\u001b[0m\n",
      "\u001b[32mScraped package Icens: 347/2183\u001b[0m\n",
      "\u001b[32mScraped package QDNAseq: 348/2183\u001b[0m\n",
      "\u001b[32mScraped package ctc: 349/2183\u001b[0m\n",
      "\u001b[32mScraped package decoupleR: 350/2183\u001b[0m\n",
      "\u001b[32mScraped package Rbowtie2: 351/2183\u001b[0m\n",
      "\u001b[32mScraped package iSEE: 352/2183\u001b[0m\n",
      "\u001b[32mScraped package maSigPro: 353/2183\u001b[0m\n",
      "\u001b[32mScraped package cmapR: 354/2183\u001b[0m\n",
      "\u001b[32mScraped package SGSeq: 355/2183\u001b[0m\n",
      "\u001b[32mScraped package GeneOverlap: 356/2183\u001b[0m\n",
      "\u001b[32mScraped package RankProd: 357/2183\u001b[0m\n",
      "\u001b[32mScraped package recount3: 358/2183\u001b[0m\n",
      "\u001b[32mScraped package soGGi: 359/2183\u001b[0m\n",
      "\u001b[32mScraped package scRepertoire: 360/2183\u001b[0m\n",
      "\u001b[32mScraped package cummeRbund: 361/2183\u001b[0m\n",
      "\u001b[32mScraped package cqn: 362/2183\u001b[0m\n",
      "\u001b[32mScraped package ELMER: 363/2183\u001b[0m\n",
      "\u001b[32mScraped package BioNet: 364/2183\u001b[0m\n",
      "\u001b[32mScraped package heatmaps: 365/2183\u001b[0m\n",
      "\u001b[32mScraped package GLAD: 366/2183\u001b[0m\n",
      "\u001b[32mScraped package made4: 367/2183\u001b[0m\n",
      "\u001b[32mScraped package pvca: 368/2183\u001b[0m\n",
      "\u001b[32mScraped package EpiDISH: 369/2183\u001b[0m\n",
      "\u001b[32mScraped package IsoformSwitchAnalyzeR: 370/2183\u001b[0m\n",
      "\u001b[32mScraped package annaffy: 371/2183\u001b[0m\n",
      "\u001b[32mScraped package diffcyt: 372/2183\u001b[0m\n",
      "\u001b[32mScraped package hpar: 373/2183\u001b[0m\n",
      "\u001b[32mScraped package HMMcopy: 374/2183\u001b[0m\n",
      "\u001b[32mScraped package pRoloc: 375/2183\u001b[0m\n",
      "\u001b[32mScraped package ReadqPCR: 376/2183\u001b[0m\n",
      "\u001b[32mScraped package InteractiveComplexHeatmap: 377/2183\u001b[0m\n",
      "\u001b[32mScraped package exomeCopy: 378/2183\u001b[0m\n",
      "\u001b[32mScraped package RedeR: 379/2183\u001b[0m\n",
      "\u001b[32mScraped package escape: 380/2183\u001b[0m\n",
      "\u001b[32mScraped package AnnotationHubData: 381/2183\u001b[0m\n",
      "\u001b[32mScraped package DEFormats: 382/2183\u001b[0m\n",
      "\u001b[32mScraped package microbiomeMarker: 383/2183\u001b[0m\n",
      "\u001b[32mScraped package NanoStringNCTools: 384/2183\u001b[0m\n",
      "\u001b[32mScraped package GeomxTools: 385/2183\u001b[0m\n",
      "\u001b[32mScraped package BUSpaRse: 386/2183\u001b[0m\n",
      "\u001b[32mScraped package multiMiR: 387/2183\u001b[0m\n",
      "\u001b[32mScraped package NormqPCR: 388/2183\u001b[0m\n",
      "\u001b[32mScraped package bambu: 389/2183\u001b[0m\n",
      "\u001b[32mScraped package bioDist: 390/2183\u001b[0m\n",
      "\u001b[32mScraped package OmicCircos: 391/2183\u001b[0m\n",
      "\u001b[32mScraped package muscat: 392/2183\u001b[0m\n",
      "\u001b[32mScraped package hopach: 393/2183\u001b[0m\n",
      "\u001b[32mScraped package CancerSubtypes: 394/2183\u001b[0m\n",
      "\u001b[32mScraped package GenomicInteractions: 395/2183\u001b[0m\n",
      "\u001b[32mScraped package MungeSumstats: 396/2183\u001b[0m\n",
      "\u001b[32mScraped package RnBeads: 397/2183\u001b[0m\n",
      "\u001b[32mScraped package flowClean: 398/2183\u001b[0m\n",
      "\u001b[32mScraped package lefser: 399/2183\u001b[0m\n",
      "\u001b[32mScraped package ChemmineOB: 400/2183\u001b[0m\n",
      "\u001b[32mScraped package Rqc: 401/2183\u001b[0m\n",
      "\u001b[32mScraped package DEGseq: 402/2183\u001b[0m\n",
      "\u001b[32mScraped package muscle: 403/2183\u001b[0m\n",
      "\u001b[32mScraped package DEqMS: 404/2183\u001b[0m\n",
      "\u001b[32mScraped package fmcsR: 405/2183\u001b[0m\n",
      "\u001b[32mScraped package MeSHDbi: 406/2183\u001b[0m\n",
      "\u001b[32mScraped package ReactomeGSA: 407/2183\u001b[0m\n",
      "\u001b[32mScraped package GEOmetadb: 408/2183\u001b[0m\n",
      "\u001b[32mScraped package flowDensity: 409/2183\u001b[0m\n",
      "\u001b[32mScraped package MsBackendMgf: 410/2183\u001b[0m\n",
      "\u001b[32mScraped package BiocSet: 411/2183\u001b[0m\n",
      "\u001b[32mScraped package GOfuncR: 412/2183\u001b[0m\n",
      "\u001b[32mScraped package quantro: 413/2183\u001b[0m\n",
      "\u001b[32mScraped package SIMLR: 414/2183\u001b[0m\n",
      "\u001b[32mScraped package philr: 415/2183\u001b[0m\n",
      "\u001b[32mScraped package cicero: 416/2183\u001b[0m\n",
      "\u001b[32mScraped package cn.mops: 417/2183\u001b[0m\n",
      "\u001b[32mScraped package MSstatsTMT: 418/2183\u001b[0m\n",
      "\u001b[32mScraped package PADOG: 419/2183\u001b[0m\n",
      "\u001b[32mScraped package Rhisat2: 420/2183\u001b[0m\n",
      "\u001b[32mScraped package CoreGx: 421/2183\u001b[0m\n",
      "\u001b[32mScraped package genbankr: 422/2183\u001b[0m\n",
      "\u001b[32mScraped package biobroom: 423/2183\u001b[0m\n",
      "\u001b[32mScraped package ExperimentHubData: 424/2183\u001b[0m\n",
      "\u001b[32mScraped package HTSFilter: 425/2183\u001b[0m\n",
      "\u001b[32mScraped package DeconRNASeq: 426/2183\u001b[0m\n",
      "\u001b[32mScraped package cleaver: 427/2183\u001b[0m\n",
      "\u001b[32mScraped package synergyfinder: 428/2183\u001b[0m\n",
      "\u001b[32mScraped package EGSEA: 429/2183\u001b[0m\n",
      "\u001b[32mScraped package ENmix: 430/2183\u001b[0m\n",
      "\u001b[32mScraped package cytomapper: 431/2183\u001b[0m\n",
      "\u001b[32mScraped package VennDetail: 432/2183\u001b[0m\n",
      "\u001b[32mScraped package MetaboCoreUtils: 433/2183\u001b[0m\n",
      "\u001b[32mScraped package stageR: 434/2183\u001b[0m\n",
      "\u001b[32mScraped package pathifier: 435/2183\u001b[0m\n",
      "\u001b[32mScraped package profileplyr: 436/2183\u001b[0m\n",
      "\u001b[32mScraped package BiocWorkflowTools: 437/2183\u001b[0m\n",
      "\u001b[32mScraped package densvis: 438/2183\u001b[0m\n",
      "\u001b[32mScraped package LOLA: 439/2183\u001b[0m\n",
      "\u001b[32mScraped package PharmacoGx: 440/2183\u001b[0m\n",
      "\u001b[32mScraped package msmsEDA: 441/2183\u001b[0m\n",
      "\u001b[32mScraped package CEMiTool: 442/2183\u001b[0m\n",
      "\u001b[32mScraped package orthogene: 443/2183\u001b[0m\n",
      "\u001b[32mScraped package PureCN: 444/2183\u001b[0m\n",
      "\u001b[32mScraped package ROTS: 445/2183\u001b[0m\n",
      "\u001b[32mScraped package singleCellTK: 446/2183\u001b[0m\n",
      "\u001b[32mScraped package OUTRIDER: 447/2183\u001b[0m\n",
      "\u001b[32mScraped package memes: 448/2183\u001b[0m\n",
      "\u001b[32mScraped package BayesSpace: 449/2183\u001b[0m\n",
      "\u001b[32mScraped package msmsTests: 450/2183\u001b[0m\n",
      "\u001b[32mScraped package HilbertVis: 451/2183\u001b[0m\n",
      "\u001b[32mScraped package proDA: 452/2183\u001b[0m\n",
      "\u001b[32mScraped package AgiMicroRna: 453/2183\u001b[0m\n",
      "\u001b[32mScraped package HiCcompare: 454/2183\u001b[0m\n",
      "\u001b[32mScraped package HTqPCR: 455/2183\u001b[0m\n",
      "\u001b[32mScraped package TOAST: 456/2183\u001b[0m\n",
      "\u001b[32mScraped package biocthis: 457/2183\u001b[0m\n",
      "\u001b[32mScraped package Linnorm: 458/2183\u001b[0m\n",
      "\u001b[32mScraped package miloR: 459/2183\u001b[0m\n",
      "\u001b[32mScraped package ORFik: 460/2183\u001b[0m\n",
      "\u001b[32mScraped package sigFeature: 461/2183\u001b[0m\n",
      "\u001b[32mScraped package rawrr: 462/2183\u001b[0m\n",
      "\u001b[32mScraped package SomaticSignatures: 463/2183\u001b[0m\n",
      "\u001b[32mScraped package sangeranalyseR: 464/2183\u001b[0m\n",
      "\u001b[32mScraped package scDD: 465/2183\u001b[0m\n",
      "\u001b[32mScraped package drawProteins: 466/2183\u001b[0m\n",
      "\u001b[32mScraped package MSnID: 467/2183\u001b[0m\n",
      "\u001b[32mScraped package rBiopaxParser: 468/2183\u001b[0m\n",
      "\u001b[32mScraped package sSeq: 469/2183\u001b[0m\n",
      "\u001b[32mScraped package quantiseqr: 470/2183\u001b[0m\n",
      "\u001b[32mScraped package plotgardener: 471/2183\u001b[0m\n",
      "\u001b[32mScraped package clustifyr: 472/2183\u001b[0m\n",
      "\u001b[32mScraped package Harman: 473/2183\u001b[0m\n",
      "\u001b[32mScraped package miaViz: 474/2183\u001b[0m\n",
      "\u001b[32mScraped package MethylSeekR: 475/2183\u001b[0m\n",
      "\u001b[32mScraped package snm: 476/2183\u001b[0m\n",
      "\u001b[32mScraped package StructuralVariantAnnotation: 477/2183\u001b[0m\n",
      "\u001b[32mScraped package flowUtils: 478/2183\u001b[0m\n",
      "\u001b[32mScraped package RNASeqPower: 479/2183\u001b[0m\n",
      "\u001b[32mScraped package SingleCellSignalR: 480/2183\u001b[0m\n",
      "\u001b[32mScraped package topconfects: 481/2183\u001b[0m\n",
      "\u001b[32mScraped package CoGAPS: 482/2183\u001b[0m\n",
      "\u001b[32mScraped package genomeIntervals: 483/2183\u001b[0m\n",
      "\u001b[32mScraped package ViSEAGO: 484/2183\u001b[0m\n",
      "\u001b[32mScraped package BRGenomics: 485/2183\u001b[0m\n",
      "\u001b[32mScraped package debrowser: 486/2183\u001b[0m\n",
      "\u001b[32mScraped package iCOBRA: 487/2183\u001b[0m\n",
      "\u001b[32mScraped package normr: 488/2183\u001b[0m\n",
      "\u001b[32mScraped package plier: 489/2183\u001b[0m\n",
      "\u001b[32mScraped package dmrseq: 490/2183\u001b[0m\n",
      "\u001b[32mScraped package DNABarcodes: 491/2183\u001b[0m\n",
      "\u001b[32mScraped package EBarrays: 492/2183\u001b[0m\n",
      "\u001b[32mScraped package miRBaseConverter: 493/2183\u001b[0m\n",
      "\u001b[32mScraped package MLSeq: 494/2183\u001b[0m\n",
      "\u001b[32mScraped package schex: 495/2183\u001b[0m\n",
      "\u001b[32mScraped package aCGH: 496/2183\u001b[0m\n",
      "\u001b[32mScraped package fabia: 497/2183\u001b[0m\n",
      "\u001b[32mScraped package splots: 498/2183\u001b[0m\n",
      "\u001b[32mScraped package tracktables: 499/2183\u001b[0m\n",
      "\u001b[32mScraped package cola: 500/2183\u001b[0m\n",
      "\u001b[32mScraped package CMA: 501/2183\u001b[0m\n",
      "\u001b[32mScraped package conumee: 502/2183\u001b[0m\n",
      "\u001b[32mScraped package CompoundDb: 503/2183\u001b[0m\n",
      "\u001b[32mScraped package IPO: 504/2183\u001b[0m\n",
      "\u001b[32mScraped package tidySummarizedExperiment: 505/2183\u001b[0m\n",
      "\u001b[32mScraped package convert: 506/2183\u001b[0m\n",
      "\u001b[32mScraped package microRNA: 507/2183\u001b[0m\n",
      "\u001b[32mScraped package qpgraph: 508/2183\u001b[0m\n",
      "\u001b[32mScraped package SpatialDecon: 509/2183\u001b[0m\n",
      "\u001b[32mScraped package tidybulk: 510/2183\u001b[0m\n",
      "\u001b[32mScraped package meshes: 511/2183\u001b[0m\n",
      "\u001b[32mScraped package CARNIVAL: 512/2183\u001b[0m\n",
      "\u001b[32mScraped package ensemblVEP: 513/2183\u001b[0m\n",
      "\u001b[32mScraped package pipeFrame: 514/2183\u001b[0m\n",
      "\u001b[32mScraped package wiggleplotr: 515/2183\u001b[0m\n",
      "\u001b[32mScraped package HiTC: 516/2183\u001b[0m\n",
      "\u001b[32mScraped package kebabs: 517/2183\u001b[0m\n",
      "\u001b[32mScraped package cellHTS2: 518/2183\u001b[0m\n",
      "\u001b[32mScraped package coRdon: 519/2183\u001b[0m\n",
      "\u001b[32mScraped package gtrellis: 520/2183\u001b[0m\n",
      "\u001b[32mScraped package SIAMCAT: 521/2183\u001b[0m\n",
      "\u001b[32mScraped package sigPathway: 522/2183\u001b[0m\n",
      "\u001b[32mScraped package BrowserViz: 523/2183\u001b[0m\n",
      "\u001b[32mScraped package GOSim: 524/2183\u001b[0m\n",
      "\u001b[32mScraped package matter: 525/2183\u001b[0m\n",
      "\u001b[32mScraped package MethylMix: 526/2183\u001b[0m\n",
      "\u001b[32mScraped package Rcpi: 527/2183\u001b[0m\n",
      "\u001b[32mScraped package TCseq: 528/2183\u001b[0m\n",
      "\u001b[32mScraped package polyester: 529/2183\u001b[0m\n",
      "\u001b[32mScraped package RTN: 530/2183\u001b[0m\n",
      "\u001b[32mScraped package Cardinal: 531/2183\u001b[0m\n",
      "\u001b[32mScraped package MetaboAnnotation: 532/2183\u001b[0m\n",
      "\u001b[32mScraped package miRNAtap: 533/2183\u001b[0m\n",
      "\u001b[32mScraped package mogsa: 534/2183\u001b[0m\n",
      "\u001b[32mScraped package BASiCS: 535/2183\u001b[0m\n",
      "\u001b[32mScraped package CelliD: 536/2183\u001b[0m\n",
      "\u001b[32mScraped package GeneTonic: 537/2183\u001b[0m\n",
      "\u001b[32mScraped package agilp: 538/2183\u001b[0m\n",
      "\u001b[32mScraped package BatchQC: 539/2183\u001b[0m\n",
      "\u001b[32mScraped package crlmm: 540/2183\u001b[0m\n",
      "\u001b[32mScraped package FELLA: 541/2183\u001b[0m\n",
      "\u001b[32mScraped package makecdfenv: 542/2183\u001b[0m\n",
      "\u001b[32mScraped package a4Preproc: 543/2183\u001b[0m\n",
      "\u001b[32mScraped package cellxgenedp: 544/2183\u001b[0m\n",
      "\u001b[32mScraped package FRASER: 545/2183\u001b[0m\n",
      "\u001b[32mScraped package lipidr: 546/2183\u001b[0m\n",
      "\u001b[32mScraped package BHC: 547/2183\u001b[0m\n",
      "\u001b[32mScraped package BioQC: 548/2183\u001b[0m\n",
      "\u001b[32mScraped package flowPeaks: 549/2183\u001b[0m\n",
      "\u001b[32mScraped package GSAR: 550/2183\u001b[0m\n",
      "\u001b[32mScraped package CRISPRseek: 551/2183\u001b[0m\n",
      "\u001b[32mScraped package flowMeans: 552/2183\u001b[0m\n",
      "\u001b[32mScraped package MEDIPS: 553/2183\u001b[0m\n",
      "\u001b[32mScraped package NormalyzerDE: 554/2183\u001b[0m\n",
      "\u001b[32mScraped package seqTools: 555/2183\u001b[0m\n",
      "\u001b[32mScraped package DAPAR: 556/2183\u001b[0m\n",
      "\u001b[32mScraped package edge: 557/2183\u001b[0m\n",
      "\u001b[32mScraped package Guitar: 558/2183\u001b[0m\n",
      "\u001b[32mScraped package ontoProc: 559/2183\u001b[0m\n",
      "\u001b[32mScraped package SamSPECTRAL: 560/2183\u001b[0m\n",
      "\u001b[32mScraped package tidySingleCellExperiment: 561/2183\u001b[0m\n",
      "\u001b[32mScraped package arrayQuality: 562/2183\u001b[0m\n",
      "\u001b[32mScraped package BRAIN: 563/2183\u001b[0m\n",
      "\u001b[32mScraped package ccrepe: 564/2183\u001b[0m\n",
      "\u001b[32mScraped package distinct: 565/2183\u001b[0m\n",
      "\u001b[32mScraped package methylGSA: 566/2183\u001b[0m\n",
      "\u001b[32mScraped package TCGAbiolinksGUI: 567/2183\u001b[0m\n",
      "\u001b[32mScraped package qrqc: 568/2183\u001b[0m\n",
      "\u001b[32mScraped package Rfastp: 569/2183\u001b[0m\n",
      "\u001b[32mScraped package snifter: 570/2183\u001b[0m\n",
      "\u001b[32mScraped package CellBench: 571/2183\u001b[0m\n",
      "\u001b[32mScraped package Modstrings: 572/2183\u001b[0m\n",
      "\u001b[32mScraped package omicade4: 573/2183\u001b[0m\n",
      "\u001b[32mScraped package ExploreModelMatrix: 574/2183\u001b[0m\n",
      "\u001b[32mScraped package imcRtools: 575/2183\u001b[0m\n",
      "\u001b[32mScraped package maser: 576/2183\u001b[0m\n",
      "\u001b[32mScraped package pandaR: 577/2183\u001b[0m\n",
      "\u001b[32mScraped package pdInfoBuilder: 578/2183\u001b[0m\n",
      "\u001b[32mScraped package regionReport: 579/2183\u001b[0m\n",
      "\u001b[32mScraped package scone: 580/2183\u001b[0m\n",
      "\u001b[32mScraped package a4Core: 581/2183\u001b[0m\n",
      "\u001b[32mScraped package exomePeak2: 582/2183\u001b[0m\n",
      "\u001b[32mScraped package metaMS: 583/2183\u001b[0m\n",
      "\u001b[32mScraped package yarn: 584/2183\u001b[0m\n",
      "\u001b[32mScraped package diffHic: 585/2183\u001b[0m\n",
      "\u001b[32mScraped package frma: 586/2183\u001b[0m\n",
      "\u001b[32mScraped package methylclock: 587/2183\u001b[0m\n",
      "\u001b[32mScraped package twilight: 588/2183\u001b[0m\n",
      "\u001b[32mScraped package zFPKM: 589/2183\u001b[0m\n",
      "\u001b[32mScraped package mosaics: 590/2183\u001b[0m\n",
      "\u001b[32mScraped package paxtoolsr: 591/2183\u001b[0m\n",
      "\u001b[32mScraped package puma: 592/2183\u001b[0m\n",
      "\u001b[32mScraped package QUBIC: 593/2183\u001b[0m\n",
      "\u001b[32mScraped package dupRadar: 594/2183\u001b[0m\n",
      "\u001b[32mScraped package ExpressionAtlas: 595/2183\u001b[0m\n",
      "\u001b[32mScraped package granulator: 596/2183\u001b[0m\n",
      "\u001b[32mScraped package PeacoQC: 597/2183\u001b[0m\n",
      "\u001b[32mScraped package r3Cseq: 598/2183\u001b[0m\n",
      "\u001b[32mScraped package VariantTools: 599/2183\u001b[0m\n",
      "\u001b[32mScraped package BicARE: 600/2183\u001b[0m\n",
      "\u001b[32mScraped package CAGEr: 601/2183\u001b[0m\n",
      "\u001b[32mScraped package CSAR: 602/2183\u001b[0m\n",
      "\u001b[32mScraped package hypeR: 603/2183\u001b[0m\n",
      "\u001b[32mScraped package megadepth: 604/2183\u001b[0m\n",
      "\u001b[32mScraped package SCnorm: 605/2183\u001b[0m\n",
      "\u001b[32mScraped package scry: 606/2183\u001b[0m\n",
      "\u001b[32mScraped package BiocPkgTools: 607/2183\u001b[0m\n",
      "\u001b[32mScraped package clusterStab: 608/2183\u001b[0m\n",
      "\u001b[32mScraped package deepSNV: 609/2183\u001b[0m\n",
      "\u001b[32mScraped package discordant: 610/2183\u001b[0m\n",
      "\u001b[32mScraped package girafe: 611/2183\u001b[0m\n",
      "\u001b[32mScraped package GOexpress: 612/2183\u001b[0m\n",
      "\u001b[32mScraped package gpls: 613/2183\u001b[0m\n",
      "\u001b[32mScraped package iCARE: 614/2183\u001b[0m\n",
      "\u001b[32mScraped package mgsa: 615/2183\u001b[0m\n",
      "\u001b[32mScraped package MWASTools: 616/2183\u001b[0m\n",
      "\u001b[32mScraped package velociraptor: 617/2183\u001b[0m\n",
      "\u001b[32mScraped package vidger: 618/2183\u001b[0m\n",
      "\u001b[32mScraped package animalcules: 619/2183\u001b[0m\n",
      "\u001b[32mScraped package chipenrich: 620/2183\u001b[0m\n",
      "\u001b[32mScraped package ChIPseqR: 621/2183\u001b[0m\n",
      "\u001b[32mScraped package CNTools: 622/2183\u001b[0m\n",
      "\u001b[32mScraped package EWCE: 623/2183\u001b[0m\n",
      "\u001b[32mScraped package MetaboSignal: 624/2183\u001b[0m\n",
      "\u001b[32mScraped package OCplus: 625/2183\u001b[0m\n",
      "\u001b[32mScraped package TreeAndLeaf: 626/2183\u001b[0m\n",
      "\u001b[32mScraped package biodb: 627/2183\u001b[0m\n",
      "\u001b[32mScraped package CopyNumberPlots: 628/2183\u001b[0m\n",
      "\u001b[32mScraped package gaia: 629/2183\u001b[0m\n",
      "\u001b[32mScraped package NanoStringDiff: 630/2183\u001b[0m\n",
      "\u001b[32mScraped package planet: 631/2183\u001b[0m\n",
      "\u001b[32mScraped package rebook: 632/2183\u001b[0m\n",
      "\u001b[32mScraped package RnaSeqSampleSize: 633/2183\u001b[0m\n",
      "\u001b[32mScraped package ChIPsim: 634/2183\u001b[0m\n",
      "\u001b[32mScraped package coseq: 635/2183\u001b[0m\n",
      "\u001b[32mScraped package ddCt: 636/2183\u001b[0m\n",
      "\u001b[32mScraped package eisaR: 637/2183\u001b[0m\n",
      "\u001b[32mScraped package geNetClassifier: 638/2183\u001b[0m\n",
      "\u001b[32mScraped package glmSparseNet: 639/2183\u001b[0m\n",
      "\u001b[32mScraped package GWENA: 640/2183\u001b[0m\n",
      "\u001b[32mScraped package miQC: 641/2183\u001b[0m\n",
      "\u001b[32mScraped package OrderedList: 642/2183\u001b[0m\n",
      "\u001b[32mScraped package statTarget: 643/2183\u001b[0m\n",
      "\u001b[32mScraped package struct: 644/2183\u001b[0m\n",
      "\u001b[32mScraped package tweeDEseq: 645/2183\u001b[0m\n",
      "\u001b[32mScraped package bacon: 646/2183\u001b[0m\n",
      "\u001b[32mScraped package GeneticsPed: 647/2183\u001b[0m\n",
      "\u001b[32mScraped package HIBAG: 648/2183\u001b[0m\n",
      "\u001b[32mScraped package motifbreakR: 649/2183\u001b[0m\n",
      "\u001b[32mScraped package openPrimeR: 650/2183\u001b[0m\n",
      "\u001b[32mScraped package RbcBook1: 651/2183\u001b[0m\n",
      "\u001b[32mScraped package TissueEnrich: 652/2183\u001b[0m\n",
      "\u001b[32mScraped package tricycle: 653/2183\u001b[0m\n",
      "\u001b[32mScraped package BgeeDB: 654/2183\u001b[0m\n",
      "\u001b[32mScraped package pathwayPCA: 655/2183\u001b[0m\n",
      "\u001b[32mScraped package PICS: 656/2183\u001b[0m\n",
      "\u001b[32mScraped package pmp: 657/2183\u001b[0m\n",
      "\u001b[32mScraped package a4Base: 658/2183\u001b[0m\n",
      "\u001b[32mScraped package biocGraph: 659/2183\u001b[0m\n",
      "\u001b[32mScraped package HelloRanges: 660/2183\u001b[0m\n",
      "\u001b[32mScraped package igvR: 661/2183\u001b[0m\n",
      "\u001b[32mScraped package proBatch: 662/2183\u001b[0m\n",
      "\u001b[32mScraped package PWMEnrich: 663/2183\u001b[0m\n",
      "\u001b[32mScraped package REDseq: 664/2183\u001b[0m\n",
      "\u001b[32mScraped package cogena: 665/2183\u001b[0m\n",
      "\u001b[32mScraped package CrispRVariants: 666/2183\u001b[0m\n",
      "\u001b[32mScraped package isobar: 667/2183\u001b[0m\n",
      "\u001b[32mScraped package phenopath: 668/2183\u001b[0m\n",
      "\u001b[32mScraped package PROcess: 669/2183\u001b[0m\n",
      "\u001b[32mScraped package RDRToolbox: 670/2183\u001b[0m\n",
      "\u001b[32mScraped package segmentSeq: 671/2183\u001b[0m\n",
      "\u001b[32mScraped package CellNOptR: 672/2183\u001b[0m\n",
      "\u001b[32mScraped package cydar: 673/2183\u001b[0m\n",
      "\u001b[32mScraped package Mulcom: 674/2183\u001b[0m\n",
      "\u001b[32mScraped package RandomWalkRestartMH: 675/2183\u001b[0m\n",
      "\u001b[32mScraped package ROntoTools: 676/2183\u001b[0m\n",
      "\u001b[32mScraped package alpine: 677/2183\u001b[0m\n",
      "\u001b[32mScraped package BioNERO: 678/2183\u001b[0m\n",
      "\u001b[32mScraped package esATAC: 679/2183\u001b[0m\n",
      "\u001b[32mScraped package FastqCleaner: 680/2183\u001b[0m\n",
      "\u001b[32mScraped package iBBiG: 681/2183\u001b[0m\n",
      "\u001b[32mScraped package ideal: 682/2183\u001b[0m\n",
      "\u001b[32mScraped package SplicingGraphs: 683/2183\u001b[0m\n",
      "\u001b[32mScraped package switchBox: 684/2183\u001b[0m\n",
      "\u001b[32mScraped package tRNA: 685/2183\u001b[0m\n",
      "\u001b[32mScraped package ABSSeq: 686/2183\u001b[0m\n",
      "\u001b[32mScraped package derfinderPlot: 687/2183\u001b[0m\n",
      "\u001b[32mScraped package HPAanalyze: 688/2183\u001b[0m\n",
      "\u001b[32mScraped package MMUPHin: 689/2183\u001b[0m\n",
      "\u001b[32mScraped package Structstrings: 690/2183\u001b[0m\n",
      "\u001b[32mScraped package ccfindR: 691/2183\u001b[0m\n",
      "\u001b[32mScraped package CiteFuse: 692/2183\u001b[0m\n",
      "\u001b[32mScraped package flowVS: 693/2183\u001b[0m\n",
      "\u001b[32mScraped package msqrob2: 694/2183\u001b[0m\n",
      "\u001b[32mScraped package sechm: 695/2183\u001b[0m\n",
      "\u001b[32mScraped package SpeCond: 696/2183\u001b[0m\n",
      "\u001b[32mScraped package SPOTlight: 697/2183\u001b[0m\n",
      "\u001b[32mScraped package chromPlot: 698/2183\u001b[0m\n",
      "\u001b[32mScraped package GDSArray: 699/2183\u001b[0m\n",
      "\u001b[32mScraped package hypergraph: 700/2183\u001b[0m\n",
      "\u001b[32mScraped package meshr: 701/2183\u001b[0m\n",
      "\u001b[32mScraped package OmaDB: 702/2183\u001b[0m\n",
      "\u001b[32mScraped package signatureSearch: 703/2183\u001b[0m\n",
      "\u001b[32mScraped package AneuFinder: 704/2183\u001b[0m\n",
      "\u001b[32mScraped package BiRewire: 705/2183\u001b[0m\n",
      "\u001b[32mScraped package clipper: 706/2183\u001b[0m\n",
      "\u001b[32mScraped package mfa: 707/2183\u001b[0m\n",
      "\u001b[32mScraped package MSstatsPTM: 708/2183\u001b[0m\n",
      "\u001b[32mScraped package NanoStringQCPro: 709/2183\u001b[0m\n",
      "\u001b[32mScraped package rsbml: 710/2183\u001b[0m\n",
      "\u001b[32mScraped package trio: 711/2183\u001b[0m\n",
      "\u001b[32mScraped package annotationTools: 712/2183\u001b[0m\n",
      "\u001b[32mScraped package chopsticks: 713/2183\u001b[0m\n",
      "\u001b[32mScraped package FEAST: 714/2183\u001b[0m\n",
      "\u001b[32mScraped package keggorthology: 715/2183\u001b[0m\n",
      "\u001b[32mScraped package PhosR: 716/2183\u001b[0m\n",
      "\u001b[32mScraped package PROPER: 717/2183\u001b[0m\n",
      "\u001b[32mScraped package shinyMethyl: 718/2183\u001b[0m\n",
      "\u001b[32mScraped package spicyR: 719/2183\u001b[0m\n",
      "\u001b[32mScraped package YAPSA: 720/2183\u001b[0m\n",
      "\u001b[32mScraped package Chicago: 721/2183\u001b[0m\n",
      "\u001b[32mScraped package chromstaR: 722/2183\u001b[0m\n",
      "\u001b[32mScraped package GRmetrics: 723/2183\u001b[0m\n",
      "\u001b[32mScraped package HubPub: 724/2183\u001b[0m\n",
      "\u001b[32mScraped package RRHO: 725/2183\u001b[0m\n",
      "\u001b[32mScraped package scPipe: 726/2183\u001b[0m\n",
      "\u001b[32mScraped package chimeraviz: 727/2183\u001b[0m\n",
      "\u001b[32mScraped package goProfiles: 728/2183\u001b[0m\n",
      "\u001b[32mScraped package interactiveDisplay: 729/2183\u001b[0m\n",
      "\u001b[32mScraped package SEtools: 730/2183\u001b[0m\n",
      "\u001b[32mScraped package Spaniel: 731/2183\u001b[0m\n",
      "\u001b[32mScraped package sparrow: 732/2183\u001b[0m\n",
      "\u001b[32mScraped package alevinQC: 733/2183\u001b[0m\n",
      "\u001b[32mScraped package ASpli: 734/2183\u001b[0m\n",
      "\u001b[32mScraped package bigPint: 735/2183\u001b[0m\n",
      "\u001b[32mScraped package dearseq: 736/2183\u001b[0m\n",
      "\u001b[32mScraped package FCBF: 737/2183\u001b[0m\n",
      "\u001b[32mScraped package GenomicDistributions: 738/2183\u001b[0m\n",
      "\u001b[32mScraped package gwasurvivr: 739/2183\u001b[0m\n",
      "\u001b[32mScraped package Pi: 740/2183\u001b[0m\n",
      "\u001b[32mScraped package psichomics: 741/2183\u001b[0m\n",
      "\u001b[32mScraped package SBGNview: 742/2183\u001b[0m\n",
      "\u001b[32mScraped package bayNorm: 743/2183\u001b[0m\n",
      "\u001b[32mScraped package bugsigdbr: 744/2183\u001b[0m\n",
      "\u001b[32mScraped package CNVRanger: 745/2183\u001b[0m\n",
      "\u001b[32mScraped package consensusSeekeR: 746/2183\u001b[0m\n",
      "\u001b[32mScraped package decompTumor2Sig: 747/2183\u001b[0m\n",
      "\u001b[32mScraped package HybridMTest: 748/2183\u001b[0m\n",
      "\u001b[32mScraped package iSEEu: 749/2183\u001b[0m\n",
      "\u001b[32mScraped package multiGSEA: 750/2183\u001b[0m\n",
      "\u001b[32mScraped package multiHiCcompare: 751/2183\u001b[0m\n",
      "\u001b[32mScraped package rain: 752/2183\u001b[0m\n",
      "\u001b[32mScraped package DNAshapeR: 753/2183\u001b[0m\n",
      "\u001b[32mScraped package MetaVolcanoR: 754/2183\u001b[0m\n",
      "\u001b[32mScraped package nondetects: 755/2183\u001b[0m\n",
      "\u001b[32mScraped package phenoTest: 756/2183\u001b[0m\n",
      "\u001b[32mScraped package Pigengene: 757/2183\u001b[0m\n",
      "\u001b[32mScraped package Prostar: 758/2183\u001b[0m\n",
      "\u001b[32mScraped package RMassBank: 759/2183\u001b[0m\n",
      "\u001b[32mScraped package CAGEfightR: 760/2183\u001b[0m\n",
      "\u001b[32mScraped package CODEX: 761/2183\u001b[0m\n",
      "\u001b[32mScraped package compcodeR: 762/2183\u001b[0m\n",
      "\u001b[32mScraped package FamAgg: 763/2183\u001b[0m\n",
      "\u001b[32mScraped package gmapR: 764/2183\u001b[0m\n",
      "\u001b[32mScraped package logicFS: 765/2183\u001b[0m\n",
      "\u001b[32mScraped package mnem: 766/2183\u001b[0m\n",
      "\u001b[32mScraped package monaLisa: 767/2183\u001b[0m\n",
      "\u001b[32mScraped package ngsReports: 768/2183\u001b[0m\n",
      "\u001b[32mScraped package PREDA: 769/2183\u001b[0m\n",
      "\u001b[32mScraped package ACME: 770/2183\u001b[0m\n",
      "\u001b[32mScraped package biosigner: 771/2183\u001b[0m\n",
      "\u001b[32mScraped package CytoTree: 772/2183\u001b[0m\n",
      "\u001b[32mScraped package diffcoexp: 773/2183\u001b[0m\n",
      "\u001b[32mScraped package DMRcaller: 774/2183\u001b[0m\n",
      "\u001b[32mScraped package erma: 775/2183\u001b[0m\n",
      "\u001b[32mScraped package flowCut: 776/2183\u001b[0m\n",
      "\u001b[32mScraped package flowFP: 777/2183\u001b[0m\n",
      "\u001b[32mScraped package HilbertCurve: 778/2183\u001b[0m\n",
      "\u001b[32mScraped package SCAN.UPC: 779/2183\u001b[0m\n",
      "\u001b[32mScraped package SpatialCPie: 780/2183\u001b[0m\n",
      "\u001b[32mScraped package tilingArray: 781/2183\u001b[0m\n",
      "\u001b[32mScraped package TRONCO: 782/2183\u001b[0m\n",
      "\u001b[32mScraped package ACE: 783/2183\u001b[0m\n",
      "\u001b[32mScraped package ASSIGN: 784/2183\u001b[0m\n",
      "\u001b[32mScraped package ccmap: 785/2183\u001b[0m\n",
      "\u001b[32mScraped package celaref: 786/2183\u001b[0m\n",
      "\u001b[32mScraped package countsimQC: 787/2183\u001b[0m\n",
      "\u001b[32mScraped package EmpiricalBrownsMethod: 788/2183\u001b[0m\n",
      "\u001b[32mScraped package GSEAlm: 789/2183\u001b[0m\n",
      "\u001b[32mScraped package lisaClust: 790/2183\u001b[0m\n",
      "\u001b[32mScraped package PAIRADISE: 791/2183\u001b[0m\n",
      "\u001b[32mScraped package phantasus: 792/2183\u001b[0m\n",
      "\u001b[32mScraped package rCGH: 793/2183\u001b[0m\n",
      "\u001b[32mScraped package RefPlus: 794/2183\u001b[0m\n",
      "\u001b[32mScraped package SPONGE: 795/2183\u001b[0m\n",
      "\u001b[32mScraped package BEclear: 796/2183\u001b[0m\n",
      "\u001b[32mScraped package BiSeq: 797/2183\u001b[0m\n",
      "\u001b[32mScraped package cliqueMS: 798/2183\u001b[0m\n",
      "\u001b[32mScraped package DaMiRseq: 799/2183\u001b[0m\n",
      "\u001b[32mScraped package epivizrServer: 800/2183\u001b[0m\n",
      "\u001b[32mScraped package FGNet: 801/2183\u001b[0m\n",
      "\u001b[32mScraped package InterCellar: 802/2183\u001b[0m\n",
      "\u001b[32mScraped package myvariant: 803/2183\u001b[0m\n",
      "\u001b[32mScraped package pRolocGUI: 804/2183\u001b[0m\n",
      "\u001b[32mScraped package RGSEA: 805/2183\u001b[0m\n",
      "\u001b[32mScraped package SPsimSeq: 806/2183\u001b[0m\n",
      "\u001b[32mScraped package TPP: 807/2183\u001b[0m\n",
      "\u001b[32mScraped package acde: 808/2183\u001b[0m\n",
      "\u001b[32mScraped package AllelicImbalance: 809/2183\u001b[0m\n",
      "\u001b[32mScraped package artMS: 810/2183\u001b[0m\n",
      "\u001b[32mScraped package ASSET: 811/2183\u001b[0m\n",
      "\u001b[32mScraped package crossmeta: 812/2183\u001b[0m\n",
      "\u001b[32mScraped package esetVis: 813/2183\u001b[0m\n",
      "\u001b[32mScraped package Herper: 814/2183\u001b[0m\n",
      "\u001b[32mScraped package HiCDCPlus: 815/2183\u001b[0m\n",
      "\u001b[32mScraped package hmdbQuery: 816/2183\u001b[0m\n",
      "\u001b[32mScraped package MesKit: 817/2183\u001b[0m\n",
      "\u001b[32mScraped package MLP: 818/2183\u001b[0m\n",
      "\u001b[32mScraped package msPurity: 819/2183\u001b[0m\n",
      "\u001b[32mScraped package ramwas: 820/2183\u001b[0m\n",
      "\u001b[32mScraped package scPCA: 821/2183\u001b[0m\n",
      "\u001b[32mScraped package ssize: 822/2183\u001b[0m\n",
      "\u001b[32mScraped package dcanr: 823/2183\u001b[0m\n",
      "\u001b[32mScraped package easier: 824/2183\u001b[0m\n",
      "\u001b[32mScraped package epiNEM: 825/2183\u001b[0m\n",
      "\u001b[32mScraped package MEAL: 826/2183\u001b[0m\n",
      "\u001b[32mScraped package PepsNMR: 827/2183\u001b[0m\n",
      "\u001b[32mScraped package PPInfer: 828/2183\u001b[0m\n",
      "\u001b[32mScraped package pqsfinder: 829/2183\u001b[0m\n",
      "\u001b[32mScraped package rcellminer: 830/2183\u001b[0m\n",
      "\u001b[32mScraped package rexposome: 831/2183\u001b[0m\n",
      "\u001b[32mScraped package structToolbox: 832/2183\u001b[0m\n",
      "\u001b[32mScraped package a4Reporting: 833/2183\u001b[0m\n",
      "\u001b[32mScraped package affycomp: 834/2183\u001b[0m\n",
      "\u001b[32mScraped package amplican: 835/2183\u001b[0m\n",
      "\u001b[32mScraped package CNORode: 836/2183\u001b[0m\n",
      "\u001b[32mScraped package coMET: 837/2183\u001b[0m\n",
      "\u001b[32mScraped package customProDB: 838/2183\u001b[0m\n",
      "\u001b[32mScraped package DiscoRhythm: 839/2183\u001b[0m\n",
      "\u001b[32mScraped package enrichTF: 840/2183\u001b[0m\n",
      "\u001b[32mScraped package scClassify: 841/2183\u001b[0m\n",
      "\u001b[32mScraped package a4: 842/2183\u001b[0m\n",
      "\u001b[32mScraped package AffyCompatible: 843/2183\u001b[0m\n",
      "\u001b[32mScraped package ASICS: 844/2183\u001b[0m\n",
      "\u001b[32mScraped package BiocSklearn: 845/2183\u001b[0m\n",
      "\u001b[32mScraped package Cepo: 846/2183\u001b[0m\n",
      "\u001b[32mScraped package GOsummaries: 847/2183\u001b[0m\n",
      "\u001b[32mScraped package h5vc: 848/2183\u001b[0m\n",
      "\u001b[32mScraped package MAIT: 849/2183\u001b[0m\n",
      "\u001b[32mScraped package MetaNeighbor: 850/2183\u001b[0m\n",
      "\u001b[32mScraped package ReactomeContentService4R: 851/2183\u001b[0m\n",
      "\u001b[32mScraped package RNAAgeCalc: 852/2183\u001b[0m\n",
      "\u001b[32mScraped package RNAmodR: 853/2183\u001b[0m\n",
      "\u001b[32mScraped package satuRn: 854/2183\u001b[0m\n",
      "\u001b[32mScraped package TitanCNA: 855/2183\u001b[0m\n",
      "\u001b[32mScraped package a4Classif: 856/2183\u001b[0m\n",
      "\u001b[32mScraped package AMOUNTAIN: 857/2183\u001b[0m\n",
      "\u001b[32mScraped package ClassifyR: 858/2183\u001b[0m\n",
      "\u001b[32mScraped package CopywriteR: 859/2183\u001b[0m\n",
      "\u001b[32mScraped package CoverageView: 860/2183\u001b[0m\n",
      "\u001b[32mScraped package CRImage: 861/2183\u001b[0m\n",
      "\u001b[32mScraped package farms: 862/2183\u001b[0m\n",
      "\u001b[32mScraped package ffpe: 863/2183\u001b[0m\n",
      "\u001b[32mScraped package GeoDiff: 864/2183\u001b[0m\n",
      "\u001b[32mScraped package gespeR: 865/2183\u001b[0m\n",
      "\u001b[32mScraped package ggspavis: 866/2183\u001b[0m\n",
      "\u001b[32mScraped package hiAnnotator: 867/2183\u001b[0m\n",
      "\u001b[32mScraped package LymphoSeq: 868/2183\u001b[0m\n",
      "\u001b[32mScraped package MACSr: 869/2183\u001b[0m\n",
      "\u001b[32mScraped package methrix: 870/2183\u001b[0m\n",
      "\u001b[32mScraped package MsBackendMassbank: 871/2183\u001b[0m\n",
      "\u001b[32mScraped package ndexr: 872/2183\u001b[0m\n",
      "\u001b[32mScraped package oposSOM: 873/2183\u001b[0m\n",
      "\u001b[32mScraped package rDGIdb: 874/2183\u001b[0m\n",
      "\u001b[32mScraped package scp: 875/2183\u001b[0m\n",
      "\u001b[32mScraped package signeR: 876/2183\u001b[0m\n",
      "\u001b[32mScraped package ADImpute: 877/2183\u001b[0m\n",
      "\u001b[32mScraped package CeTF: 878/2183\u001b[0m\n",
      "\u001b[32mScraped package CHETAH: 879/2183\u001b[0m\n",
      "\u001b[32mScraped package COMPASS: 880/2183\u001b[0m\n",
      "\u001b[32mScraped package condiments: 881/2183\u001b[0m\n",
      "\u001b[32mScraped package corral: 882/2183\u001b[0m\n",
      "\u001b[32mScraped package DeMixT: 883/2183\u001b[0m\n",
      "\u001b[32mScraped package easyRNASeq: 884/2183\u001b[0m\n",
      "\u001b[32mScraped package GeneStructureTools: 885/2183\u001b[0m\n",
      "\u001b[32mScraped package mCSEA: 886/2183\u001b[0m\n",
      "\u001b[32mScraped package metagene: 887/2183\u001b[0m\n",
      "\u001b[32mScraped package multiClust: 888/2183\u001b[0m\n",
      "\u001b[32mScraped package panelcn.mops: 889/2183\u001b[0m\n",
      "\u001b[32mScraped package plgem: 890/2183\u001b[0m\n",
      "\u001b[32mScraped package scAnnotatR: 891/2183\u001b[0m\n",
      "\u001b[32mScraped package timescape: 892/2183\u001b[0m\n",
      "\u001b[32mScraped package affylmGUI: 893/2183\u001b[0m\n",
      "\u001b[32mScraped package AlpsNMR: 894/2183\u001b[0m\n",
      "\u001b[32mScraped package bioassayR: 895/2183\u001b[0m\n",
      "\u001b[32mScraped package CellMixS: 896/2183\u001b[0m\n",
      "\u001b[32mScraped package DiffLogo: 897/2183\u001b[0m\n",
      "\u001b[32mScraped package EGAD: 898/2183\u001b[0m\n",
      "\u001b[32mScraped package GeneMeta: 899/2183\u001b[0m\n",
      "\u001b[32mScraped package geneplast: 900/2183\u001b[0m\n",
      "\u001b[32mScraped package hipathia: 901/2183\u001b[0m\n",
      "\u001b[32mScraped package MicrobiomeProfiler: 902/2183\u001b[0m\n",
      "\u001b[32mScraped package RCAS: 903/2183\u001b[0m\n",
      "\u001b[32mScraped package scDataviz: 904/2183\u001b[0m\n",
      "\u001b[32mScraped package SeqGSEA: 905/2183\u001b[0m\n",
      "\u001b[32mScraped package SpidermiR: 906/2183\u001b[0m\n",
      "\u001b[32mScraped package VanillaICE: 907/2183\u001b[0m\n",
      "\u001b[32mScraped package branchpointer: 908/2183\u001b[0m\n",
      "\u001b[32mScraped package cellbaseR: 909/2183\u001b[0m\n",
      "\u001b[32mScraped package EBSeqHMM: 910/2183\u001b[0m\n",
      "\u001b[32mScraped package erccdashboard: 911/2183\u001b[0m\n",
      "\u001b[32mScraped package fcoex: 912/2183\u001b[0m\n",
      "\u001b[32mScraped package groHMM: 913/2183\u001b[0m\n",
      "\u001b[32mScraped package hca: 914/2183\u001b[0m\n",
      "\u001b[32mScraped package IdeoViz: 915/2183\u001b[0m\n",
      "\u001b[32mScraped package maanova: 916/2183\u001b[0m\n",
      "\u001b[32mScraped package MEIGOR: 917/2183\u001b[0m\n",
      "\u001b[32mScraped package metaseqR2: 918/2183\u001b[0m\n",
      "\u001b[32mScraped package microbiomeExplorer: 919/2183\u001b[0m\n",
      "\u001b[32mScraped package NanoMethViz: 920/2183\u001b[0m\n",
      "\u001b[32mScraped package netbiov: 921/2183\u001b[0m\n",
      "\u001b[32mScraped package POMA: 922/2183\u001b[0m\n",
      "\u001b[32mScraped package projectR: 923/2183\u001b[0m\n",
      "\u001b[32mScraped package QuaternaryProd: 924/2183\u001b[0m\n",
      "\u001b[32mScraped package sevenbridges: 925/2183\u001b[0m\n",
      "\u001b[32mScraped package SharedObject: 926/2183\u001b[0m\n",
      "\u001b[32mScraped package anota: 927/2183\u001b[0m\n",
      "\u001b[32mScraped package APAlyzer: 928/2183\u001b[0m\n",
      "\u001b[32mScraped package BioNetStat: 929/2183\u001b[0m\n",
      "\u001b[32mScraped package BridgeDbR: 930/2183\u001b[0m\n",
      "\u001b[32mScraped package CGEN: 931/2183\u001b[0m\n",
      "\u001b[32mScraped package CluMSID: 932/2183\u001b[0m\n",
      "\u001b[32mScraped package crisprBowtie: 933/2183\u001b[0m\n",
      "\u001b[32mScraped package crisprScore: 934/2183\u001b[0m\n",
      "\u001b[32mScraped package diffuStats: 935/2183\u001b[0m\n",
      "\u001b[32mScraped package epivizrData: 936/2183\u001b[0m\n",
      "\u001b[32mScraped package ImmuneSpaceR: 937/2183\u001b[0m\n",
      "\u001b[32mScraped package IONiseR: 938/2183\u001b[0m\n",
      "\u001b[32mScraped package PSMatch: 939/2183\u001b[0m\n",
      "\u001b[32mScraped package qsea: 940/2183\u001b[0m\n",
      "\u001b[32mScraped package RiboProfiling: 941/2183\u001b[0m\n",
      "\u001b[32mScraped package ribosomeProfilingQC: 942/2183\u001b[0m\n",
      "\u001b[32mScraped package rqubic: 943/2183\u001b[0m\n",
      "\u001b[32mScraped package RSeqAn: 944/2183\u001b[0m\n",
      "\u001b[32mScraped package runibic: 945/2183\u001b[0m\n",
      "\u001b[32mScraped package scHOT: 946/2183\u001b[0m\n",
      "\u001b[32mScraped package subSeq: 947/2183\u001b[0m\n",
      "\u001b[32mScraped package BADER: 948/2183\u001b[0m\n",
      "\u001b[32mScraped package BCRANK: 949/2183\u001b[0m\n",
      "\u001b[32mScraped package cancerclass: 950/2183\u001b[0m\n",
      "\u001b[32mScraped package CoRegNet: 951/2183\u001b[0m\n",
      "\u001b[32mScraped package cosmosR: 952/2183\u001b[0m\n",
      "\u001b[32mScraped package EBcoexpress: 953/2183\u001b[0m\n",
      "\u001b[32mScraped package flowMerge: 954/2183\u001b[0m\n",
      "\u001b[32mScraped package flowTrans: 955/2183\u001b[0m\n",
      "\u001b[32mScraped package goTools: 956/2183\u001b[0m\n",
      "\u001b[32mScraped package GSEABenchmarkeR: 957/2183\u001b[0m\n",
      "\u001b[32mScraped package GUIDEseq: 958/2183\u001b[0m\n",
      "\u001b[32mScraped package methylSig: 959/2183\u001b[0m\n",
      "\u001b[32mScraped package netresponse: 960/2183\u001b[0m\n",
      "\u001b[32mScraped package nucleR: 961/2183\u001b[0m\n",
      "\u001b[32mScraped package PhyloProfile: 962/2183\u001b[0m\n",
      "\u001b[32mScraped package ProteoMM: 963/2183\u001b[0m\n",
      "\u001b[32mScraped package Rcade: 964/2183\u001b[0m\n",
      "\u001b[32mScraped package RegEnrich: 965/2183\u001b[0m\n",
      "\u001b[32mScraped package seqCNA: 966/2183\u001b[0m\n",
      "\u001b[32mScraped package SPEM: 967/2183\u001b[0m\n",
      "\u001b[32mScraped package TMixClust: 968/2183\u001b[0m\n",
      "\u001b[32mScraped package tRNAscanImport: 969/2183\u001b[0m\n",
      "\u001b[32mScraped package altcdfenvs: 970/2183\u001b[0m\n",
      "\u001b[32mScraped package anota2seq: 971/2183\u001b[0m\n",
      "\u001b[32mScraped package BANDITS: 972/2183\u001b[0m\n",
      "\u001b[32mScraped package bioCancer: 973/2183\u001b[0m\n",
      "\u001b[32mScraped package BiocOncoTK: 974/2183\u001b[0m\n",
      "\u001b[32mScraped package BioCor: 975/2183\u001b[0m\n",
      "\u001b[32mScraped package EMDomics: 976/2183\u001b[0m\n",
      "\u001b[32mScraped package epivizr: 977/2183\u001b[0m\n",
      "\u001b[32mScraped package hyperdraw: 978/2183\u001b[0m\n",
      "\u001b[32mScraped package metagene2: 979/2183\u001b[0m\n",
      "\u001b[32mScraped package metaSeq: 980/2183\u001b[0m\n",
      "\u001b[32mScraped package methylCC: 981/2183\u001b[0m\n",
      "\u001b[32mScraped package MineICA: 982/2183\u001b[0m\n",
      "\u001b[32mScraped package odseq: 983/2183\u001b[0m\n",
      "\u001b[32mScraped package qckitfastq: 984/2183\u001b[0m\n",
      "\u001b[32mScraped package SpectralTAD: 985/2183\u001b[0m\n",
      "\u001b[32mScraped package splineTimeR: 986/2183\u001b[0m\n",
      "\u001b[32mScraped package TFEA.ChIP: 987/2183\u001b[0m\n",
      "\u001b[32mScraped package timecourse: 988/2183\u001b[0m\n",
      "\u001b[32mScraped package vissE: 989/2183\u001b[0m\n",
      "\u001b[32mScraped package ABarray: 990/2183\u001b[0m\n",
      "\u001b[32mScraped package adSplit: 991/2183\u001b[0m\n",
      "\u001b[32mScraped package CGHnormaliter: 992/2183\u001b[0m\n",
      "\u001b[32mScraped package CGHregions: 993/2183\u001b[0m\n",
      "\u001b[32mScraped package dagLogo: 994/2183\u001b[0m\n",
      "\u001b[32mScraped package gaga: 995/2183\u001b[0m\n",
      "\u001b[32mScraped package GeneNetworkBuilder: 996/2183\u001b[0m\n",
      "\u001b[32mScraped package InterMineR: 997/2183\u001b[0m\n",
      "\u001b[32mScraped package limmaGUI: 998/2183\u001b[0m\n",
      "\u001b[32mScraped package lmdme: 999/2183\u001b[0m\n",
      "\u001b[32mScraped package metabolomicsWorkbenchR: 1000/2183\u001b[0m\n",
      "\u001b[32mScraped package methimpute: 1001/2183\u001b[0m\n",
      "\u001b[32mScraped package MIRA: 1002/2183\u001b[0m\n",
      "\u001b[32mScraped package msImpute: 1003/2183\u001b[0m\n",
      "\u001b[32mScraped package PERFect: 1004/2183\u001b[0m\n",
      "\u001b[32mScraped package PloGO2: 1005/2183\u001b[0m\n",
      "\u001b[32mScraped package SAIGEgds: 1006/2183\u001b[0m\n",
      "\u001b[32mScraped package TEQC: 1007/2183\u001b[0m\n",
      "\u001b[32mScraped package vbmp: 1008/2183\u001b[0m\n",
      "\u001b[32mScraped package affyContam: 1009/2183\u001b[0m\n",
      "\u001b[32mScraped package beadarraySNP: 1010/2183\u001b[0m\n",
      "\u001b[32mScraped package bigmelon: 1011/2183\u001b[0m\n",
      "\u001b[32mScraped package biscuiteer: 1012/2183\u001b[0m\n",
      "\u001b[32mScraped package BitSeq: 1013/2183\u001b[0m\n",
      "\u001b[32mScraped package cghMCR: 1014/2183\u001b[0m\n",
      "\u001b[32mScraped package ChromHeatMap: 1015/2183\u001b[0m\n",
      "\u001b[32mScraped package CHRONOS: 1016/2183\u001b[0m\n",
      "\u001b[32mScraped package debCAM: 1017/2183\u001b[0m\n",
      "\u001b[32mScraped package EventPointer: 1018/2183\u001b[0m\n",
      "\u001b[32mScraped package GEOexplorer: 1019/2183\u001b[0m\n",
      "\u001b[32mScraped package MethReg: 1020/2183\u001b[0m\n",
      "\u001b[32mScraped package methylPipe: 1021/2183\u001b[0m\n",
      "\u001b[32mScraped package MetNet: 1022/2183\u001b[0m\n",
      "\u001b[32mScraped package miRspongeR: 1023/2183\u001b[0m\n",
      "\u001b[32mScraped package moanin: 1024/2183\u001b[0m\n",
      "\u001b[32mScraped package OncoSimulR: 1025/2183\u001b[0m\n",
      "\u001b[32mScraped package pwrEWAS: 1026/2183\u001b[0m\n",
      "\u001b[32mScraped package qsmooth: 1027/2183\u001b[0m\n",
      "\u001b[32mScraped package RBioinf: 1028/2183\u001b[0m\n",
      "\u001b[32mScraped package rbsurv: 1029/2183\u001b[0m\n",
      "\u001b[32mScraped package REMP: 1030/2183\u001b[0m\n",
      "\u001b[32mScraped package snapCGH: 1031/2183\u001b[0m\n",
      "\u001b[32mScraped package spatialDE: 1032/2183\u001b[0m\n",
      "\u001b[32mScraped package TADCompare: 1033/2183\u001b[0m\n",
      "\u001b[32mScraped package timeOmics: 1034/2183\u001b[0m\n",
      "\u001b[32mScraped package atSNP: 1035/2183\u001b[0m\n",
      "\u001b[32mScraped package BufferedMatrix: 1036/2183\u001b[0m\n",
      "\u001b[32mScraped package circRNAprofiler: 1037/2183\u001b[0m\n",
      "\u001b[32mScraped package codelink: 1038/2183\u001b[0m\n",
      "\u001b[32mScraped package isomiRs: 1039/2183\u001b[0m\n",
      "\u001b[32mScraped package IVAS: 1040/2183\u001b[0m\n",
      "\u001b[32mScraped package metaCCA: 1041/2183\u001b[0m\n",
      "\u001b[32mScraped package MethylAid: 1042/2183\u001b[0m\n",
      "\u001b[32mScraped package mistyR: 1043/2183\u001b[0m\n",
      "\u001b[32mScraped package mitch: 1044/2183\u001b[0m\n",
      "\u001b[32mScraped package musicatk: 1045/2183\u001b[0m\n",
      "\u001b[32mScraped package NetPathMiner: 1046/2183\u001b[0m\n",
      "\u001b[32mScraped package pkgDepTools: 1047/2183\u001b[0m\n",
      "\u001b[32mScraped package rmelting: 1048/2183\u001b[0m\n",
      "\u001b[32mScraped package RTNsurvival: 1049/2183\u001b[0m\n",
      "\u001b[32mScraped package rTRM: 1050/2183\u001b[0m\n",
      "\u001b[32mScraped package savR: 1051/2183\u001b[0m\n",
      "\u001b[32mScraped package scruff: 1052/2183\u001b[0m\n",
      "\u001b[32mScraped package seqsetvis: 1053/2183\u001b[0m\n",
      "\u001b[32mScraped package abseqR: 1054/2183\u001b[0m\n",
      "\u001b[32mScraped package ADAM: 1055/2183\u001b[0m\n",
      "\u001b[32mScraped package Anaquin: 1056/2183\u001b[0m\n",
      "\u001b[32mScraped package BaseSpaceR: 1057/2183\u001b[0m\n",
      "\u001b[32mScraped package BloodGen3Module: 1058/2183\u001b[0m\n",
      "\u001b[32mScraped package CausalR: 1059/2183\u001b[0m\n",
      "\u001b[32mScraped package CFAssay: 1060/2183\u001b[0m\n",
      "\u001b[32mScraped package CINdex: 1061/2183\u001b[0m\n",
      "\u001b[32mScraped package consensusOV: 1062/2183\u001b[0m\n",
      "\u001b[32mScraped package DART: 1063/2183\u001b[0m\n",
      "\u001b[32mScraped package FitHiC: 1064/2183\u001b[0m\n",
      "\u001b[32mScraped package fmrs: 1065/2183\u001b[0m\n",
      "\u001b[32mScraped package intansv: 1066/2183\u001b[0m\n",
      "\u001b[32mScraped package interacCircos: 1067/2183\u001b[0m\n",
      "\u001b[32mScraped package LPE: 1068/2183\u001b[0m\n",
      "\u001b[32mScraped package MatrixQCvis: 1069/2183\u001b[0m\n",
      "\u001b[32mScraped package MetaCyto: 1070/2183\u001b[0m\n",
      "\u001b[32mScraped package MOMA: 1071/2183\u001b[0m\n",
      "\u001b[32mScraped package MoonlightR: 1072/2183\u001b[0m\n",
      "\u001b[32mScraped package mumosa: 1073/2183\u001b[0m\n",
      "\u001b[32mScraped package Oscope: 1074/2183\u001b[0m\n",
      "\u001b[32mScraped package PAA: 1075/2183\u001b[0m\n",
      "\u001b[32mScraped package qcmetrics: 1076/2183\u001b[0m\n",
      "\u001b[32mScraped package RCX: 1077/2183\u001b[0m\n",
      "\u001b[32mScraped package rhdf5client: 1078/2183\u001b[0m\n",
      "\u001b[32mScraped package riboSeqR: 1079/2183\u001b[0m\n",
      "\u001b[32mScraped package scmeth: 1080/2183\u001b[0m\n",
      "\u001b[32mScraped package TargetSearch: 1081/2183\u001b[0m\n",
      "\u001b[32mScraped package AffyRNADegradation: 1082/2183\u001b[0m\n",
      "\u001b[32mScraped package aggregateBioVar: 1083/2183\u001b[0m\n",
      "\u001b[32mScraped package ANF: 1084/2183\u001b[0m\n",
      "\u001b[32mScraped package apComplex: 1085/2183\u001b[0m\n",
      "\u001b[32mScraped package basecallQC: 1086/2183\u001b[0m\n",
      "\u001b[32mScraped package bgx: 1087/2183\u001b[0m\n",
      "\u001b[32mScraped package BPRMeth: 1088/2183\u001b[0m\n",
      "\u001b[32mScraped package canceR: 1089/2183\u001b[0m\n",
      "\u001b[32mScraped package Clonality: 1090/2183\u001b[0m\n",
      "\u001b[32mScraped package clst: 1091/2183\u001b[0m\n",
      "\u001b[32mScraped package CNVPanelizer: 1092/2183\u001b[0m\n",
      "\u001b[32mScraped package compEpiTools: 1093/2183\u001b[0m\n",
      "\u001b[32mScraped package doppelgangR: 1094/2183\u001b[0m\n",
      "\u001b[32mScraped package GeneExpressionSignature: 1095/2183\u001b[0m\n",
      "\u001b[32mScraped package gscreend: 1096/2183\u001b[0m\n",
      "\u001b[32mScraped package HGC: 1097/2183\u001b[0m\n",
      "\u001b[32mScraped package IsoCorrectoR: 1098/2183\u001b[0m\n",
      "\u001b[32mScraped package LOBSTAHS: 1099/2183\u001b[0m\n",
      "\u001b[32mScraped package Mergeomics: 1100/2183\u001b[0m\n",
      "\u001b[32mScraped package OSAT: 1101/2183\u001b[0m\n",
      "\u001b[32mScraped package PECA: 1102/2183\u001b[0m\n",
      "\u001b[32mScraped package Pviz: 1103/2183\u001b[0m\n",
      "\u001b[32mScraped package SCANVIS: 1104/2183\u001b[0m\n",
      "\u001b[32mScraped package scTensor: 1105/2183\u001b[0m\n",
      "\u001b[32mScraped package seqbias: 1106/2183\u001b[0m\n",
      "\u001b[32mScraped package snapcount: 1107/2183\u001b[0m\n",
      "\u001b[32mScraped package systemPipeShiny: 1108/2183\u001b[0m\n",
      "\u001b[32mScraped package waddR: 1109/2183\u001b[0m\n",
      "\u001b[32mScraped package AMARETTO: 1110/2183\u001b[0m\n",
      "\u001b[32mScraped package antiProfiles: 1111/2183\u001b[0m\n",
      "\u001b[32mScraped package ASEB: 1112/2183\u001b[0m\n",
      "\u001b[32mScraped package BaalChIP: 1113/2183\u001b[0m\n",
      "\u001b[32mScraped package Basic4Cseq: 1114/2183\u001b[0m\n",
      "\u001b[32mScraped package biodbKegg: 1115/2183\u001b[0m\n",
      "\u001b[32mScraped package casper: 1116/2183\u001b[0m\n",
      "\u001b[32mScraped package CellaRepertorium: 1117/2183\u001b[0m\n",
      "\u001b[32mScraped package ChromSCape: 1118/2183\u001b[0m\n",
      "\u001b[32mScraped package CNVfilteR: 1119/2183\u001b[0m\n",
      "\u001b[32mScraped package CORREP: 1120/2183\u001b[0m\n",
      "\u001b[32mScraped package cTRAP: 1121/2183\u001b[0m\n",
      "\u001b[32mScraped package CytoGLMM: 1122/2183\u001b[0m\n",
      "\u001b[32mScraped package DeepBlueR: 1123/2183\u001b[0m\n",
      "\u001b[32mScraped package DelayedDataFrame: 1124/2183\u001b[0m\n",
      "\u001b[32mScraped package flowSpecs: 1125/2183\u001b[0m\n",
      "\u001b[32mScraped package GeneBreak: 1126/2183\u001b[0m\n",
      "\u001b[32mScraped package GOTHiC: 1127/2183\u001b[0m\n",
      "\u001b[32mScraped package GraphAlignment: 1128/2183\u001b[0m\n",
      "\u001b[32mScraped package GSALightning: 1129/2183\u001b[0m\n",
      "\u001b[32mScraped package HiCBricks: 1130/2183\u001b[0m\n",
      "\u001b[32mScraped package hiReadsProcessor: 1131/2183\u001b[0m\n",
      "\u001b[32mScraped package idpr: 1132/2183\u001b[0m\n",
      "\u001b[32mScraped package INSPEcT: 1133/2183\u001b[0m\n",
      "\u001b[32mScraped package ISAnalytics: 1134/2183\u001b[0m\n",
      "\u001b[32mScraped package ldblock: 1135/2183\u001b[0m\n",
      "\u001b[32mScraped package lionessR: 1136/2183\u001b[0m\n",
      "\u001b[32mScraped package methylInheritance: 1137/2183\u001b[0m\n",
      "\u001b[32mScraped package MSPrep: 1138/2183\u001b[0m\n",
      "\u001b[32mScraped package MultiBaC: 1139/2183\u001b[0m\n",
      "\u001b[32mScraped package multiOmicsViz: 1140/2183\u001b[0m\n",
      "\u001b[32mScraped package NanoTube: 1141/2183\u001b[0m\n",
      "\u001b[32mScraped package openPrimeRui: 1142/2183\u001b[0m\n",
      "\u001b[32mScraped package PCAN: 1143/2183\u001b[0m\n",
      "\u001b[32mScraped package peakPantheR: 1144/2183\u001b[0m\n",
      "\u001b[32mScraped package qPLEXanalyzer: 1145/2183\u001b[0m\n",
      "\u001b[32mScraped package R453Plus1Toolbox: 1146/2183\u001b[0m\n",
      "\u001b[32mScraped package RCyjs: 1147/2183\u001b[0m\n",
      "\u001b[32mScraped package Risa: 1148/2183\u001b[0m\n",
      "\u001b[32mScraped package roar: 1149/2183\u001b[0m\n",
      "\u001b[32mScraped package RTNduals: 1150/2183\u001b[0m\n",
      "\u001b[32mScraped package SBMLR: 1151/2183\u001b[0m\n",
      "\u001b[32mScraped package SCBN: 1152/2183\u001b[0m\n",
      "\u001b[32mScraped package similaRpeak: 1153/2183\u001b[0m\n",
      "\u001b[32mScraped package SparseSignatures: 1154/2183\u001b[0m\n",
      "\u001b[32mScraped package transcriptogramer: 1155/2183\u001b[0m\n",
      "\u001b[32mScraped package Trendy: 1156/2183\u001b[0m\n",
      "\u001b[32mScraped package ADAMgui: 1157/2183\u001b[0m\n",
      "\u001b[32mScraped package adductomicsR: 1158/2183\u001b[0m\n",
      "\u001b[32mScraped package AGDEX: 1159/2183\u001b[0m\n",
      "\u001b[32mScraped package ASGSCA: 1160/2183\u001b[0m\n",
      "\u001b[32mScraped package atena: 1161/2183\u001b[0m\n",
      "\u001b[32mScraped package attract: 1162/2183\u001b[0m\n",
      "\u001b[32mScraped package AWFisher: 1163/2183\u001b[0m\n",
      "\u001b[32mScraped package BAC: 1164/2183\u001b[0m\n",
      "\u001b[32mScraped package BDMMAcorrect: 1165/2183\u001b[0m\n",
      "\u001b[32mScraped package CellTrails: 1166/2183\u001b[0m\n",
      "\u001b[32mScraped package clstutils: 1167/2183\u001b[0m\n",
      "\u001b[32mScraped package CNAnorm: 1168/2183\u001b[0m\n",
      "\u001b[32mScraped package CNORfeeder: 1169/2183\u001b[0m\n",
      "\u001b[32mScraped package crisprBase: 1170/2183\u001b[0m\n",
      "\u001b[32mScraped package DEWSeq: 1171/2183\u001b[0m\n",
      "\u001b[32mScraped package diggit: 1172/2183\u001b[0m\n",
      "\u001b[32mScraped package drugTargetInteractions: 1173/2183\u001b[0m\n",
      "\u001b[32mScraped package epivizrStandalone: 1174/2183\u001b[0m\n",
      "\u001b[32mScraped package gcapc: 1175/2183\u001b[0m\n",
      "\u001b[32mScraped package gcatest: 1176/2183\u001b[0m\n",
      "\u001b[32mScraped package GeneSelectMMD: 1177/2183\u001b[0m\n",
      "\u001b[32mScraped package genomes: 1178/2183\u001b[0m\n",
      "\u001b[32mScraped package goSTAG: 1179/2183\u001b[0m\n",
      "\u001b[32mScraped package gpart: 1180/2183\u001b[0m\n",
      "\u001b[32mScraped package GSEAmining: 1181/2183\u001b[0m\n",
      "\u001b[32mScraped package iterativeBMA: 1182/2183\u001b[0m\n",
      "\u001b[32mScraped package KEGGlincs: 1183/2183\u001b[0m\n",
      "\u001b[32mScraped package KnowSeq: 1184/2183\u001b[0m\n",
      "\u001b[32mScraped package LRBaseDbi: 1185/2183\u001b[0m\n",
      "\u001b[32mScraped package maigesPack: 1186/2183\u001b[0m\n",
      "\u001b[32mScraped package miRLAB: 1187/2183\u001b[0m\n",
      "\u001b[32mScraped package miRNAmeConverter: 1188/2183\u001b[0m\n",
      "\u001b[32mScraped package miRNApath: 1189/2183\u001b[0m\n",
      "\u001b[32mScraped package MODA: 1190/2183\u001b[0m\n",
      "\u001b[32mScraped package nullranges: 1191/2183\u001b[0m\n",
      "\u001b[32mScraped package OLIN: 1192/2183\u001b[0m\n",
      "\u001b[32mScraped package parody: 1193/2183\u001b[0m\n",
      "\u001b[32mScraped package proActiv: 1194/2183\u001b[0m\n",
      "\u001b[32mScraped package RUVcorr: 1195/2183\u001b[0m\n",
      "\u001b[32mScraped package shinyepico: 1196/2183\u001b[0m\n",
      "\u001b[32mScraped package survtype: 1197/2183\u001b[0m\n",
      "\u001b[32mScraped package TAPseq: 1198/2183\u001b[0m\n",
      "\u001b[32mScraped package tRNAdbImport: 1199/2183\u001b[0m\n",
      "\u001b[32mScraped package VariantFiltering: 1200/2183\u001b[0m\n",
      "\u001b[32mScraped package ADaCGH2: 1201/2183\u001b[0m\n",
      "\u001b[32mScraped package BadRegionFinder: 1202/2183\u001b[0m\n",
      "\u001b[32mScraped package BAGS: 1203/2183\u001b[0m\n",
      "\u001b[32mScraped package BasicSTARRseq: 1204/2183\u001b[0m\n",
      "\u001b[32mScraped package blacksheepr: 1205/2183\u001b[0m\n",
      "\u001b[32mScraped package blima: 1206/2183\u001b[0m\n",
      "\u001b[32mScraped package BLMA: 1207/2183\u001b[0m\n",
      "\u001b[32mScraped package bnbc: 1208/2183\u001b[0m\n",
      "\u001b[32mScraped package BubbleTree: 1209/2183\u001b[0m\n",
      "\u001b[32mScraped package cellity: 1210/2183\u001b[0m\n",
      "\u001b[32mScraped package ChIPComp: 1211/2183\u001b[0m\n",
      "\u001b[32mScraped package coGPS: 1212/2183\u001b[0m\n",
      "\u001b[32mScraped package COHCAP: 1213/2183\u001b[0m\n",
      "\u001b[32mScraped package ctsGE: 1214/2183\u001b[0m\n",
      "\u001b[32mScraped package CytoDx: 1215/2183\u001b[0m\n",
      "\u001b[32mScraped package daMA: 1216/2183\u001b[0m\n",
      "\u001b[32mScraped package dasper: 1217/2183\u001b[0m\n",
      "\u001b[32mScraped package DEGraph: 1218/2183\u001b[0m\n",
      "\u001b[32mScraped package DriverNet: 1219/2183\u001b[0m\n",
      "\u001b[32mScraped package EpiTxDb: 1220/2183\u001b[0m\n",
      "\u001b[32mScraped package garfield: 1221/2183\u001b[0m\n",
      "\u001b[32mScraped package gCrisprTools: 1222/2183\u001b[0m\n",
      "\u001b[32mScraped package GenomicSuperSignature: 1223/2183\u001b[0m\n",
      "\u001b[32mScraped package getDEE2: 1224/2183\u001b[0m\n",
      "\u001b[32mScraped package GGPA: 1225/2183\u001b[0m\n",
      "\u001b[32mScraped package GmicR: 1226/2183\u001b[0m\n",
      "\u001b[32mScraped package GOpro: 1227/2183\u001b[0m\n",
      "\u001b[32mScraped package idiogram: 1228/2183\u001b[0m\n",
      "\u001b[32mScraped package idr2d: 1229/2183\u001b[0m\n",
      "\u001b[32mScraped package IntEREst: 1230/2183\u001b[0m\n",
      "\u001b[32mScraped package MVCClass: 1231/2183\u001b[0m\n",
      "\u001b[32mScraped package NCIgraph: 1232/2183\u001b[0m\n",
      "\u001b[32mScraped package nethet: 1233/2183\u001b[0m\n",
      "\u001b[32mScraped package podkat: 1234/2183\u001b[0m\n",
      "\u001b[32mScraped package pwOmics: 1235/2183\u001b[0m\n",
      "\u001b[32mScraped package RBM: 1236/2183\u001b[0m\n",
      "\u001b[32mScraped package RSVSim: 1237/2183\u001b[0m\n",
      "\u001b[32mScraped package RUVnormalize: 1238/2183\u001b[0m\n",
      "\u001b[32mScraped package sights: 1239/2183\u001b[0m\n",
      "\u001b[32mScraped package sizepower: 1240/2183\u001b[0m\n",
      "\u001b[32mScraped package SLqPCR: 1241/2183\u001b[0m\n",
      "\u001b[32mScraped package STATegRa: 1242/2183\u001b[0m\n",
      "\u001b[32mScraped package Streamer: 1243/2183\u001b[0m\n",
      "\u001b[32mScraped package SWATH2stats: 1244/2183\u001b[0m\n",
      "\u001b[32mScraped package SwathXtend: 1245/2183\u001b[0m\n",
      "\u001b[32mScraped package transcriptR: 1246/2183\u001b[0m\n",
      "\u001b[32mScraped package tscR: 1247/2183\u001b[0m\n",
      "\u001b[32mScraped package XDE: 1248/2183\u001b[0m\n",
      "\u001b[32mScraped package affyILM: 1249/2183\u001b[0m\n",
      "\u001b[32mScraped package autonomics: 1250/2183\u001b[0m\n",
      "\u001b[32mScraped package bcSeq: 1251/2183\u001b[0m\n",
      "\u001b[32mScraped package BgeeCall: 1252/2183\u001b[0m\n",
      "\u001b[32mScraped package BiGGR: 1253/2183\u001b[0m\n",
      "\u001b[32mScraped package biobtreeR: 1254/2183\u001b[0m\n",
      "\u001b[32mScraped package BioMVCClass: 1255/2183\u001b[0m\n",
      "\u001b[32mScraped package breakpointR: 1256/2183\u001b[0m\n",
      "\u001b[32mScraped package brendaDb: 1257/2183\u001b[0m\n",
      "\u001b[32mScraped package BufferedMatrixMethods: 1258/2183\u001b[0m\n",
      "\u001b[32mScraped package BUS: 1259/2183\u001b[0m\n",
      "\u001b[32mScraped package cn.farms: 1260/2183\u001b[0m\n",
      "\u001b[32mScraped package CNORdt: 1261/2183\u001b[0m\n",
      "\u001b[32mScraped package CNORfuzzy: 1262/2183\u001b[0m\n",
      "\u001b[32mScraped package COCOA: 1263/2183\u001b[0m\n",
      "\u001b[32mScraped package COSNet: 1264/2183\u001b[0m\n",
      "\u001b[32mScraped package DIAlignR: 1265/2183\u001b[0m\n",
      "\u001b[32mScraped package fdrame: 1266/2183\u001b[0m\n",
      "\u001b[32mScraped package flowPloidy: 1267/2183\u001b[0m\n",
      "\u001b[32mScraped package GA4GHclient: 1268/2183\u001b[0m\n",
      "\u001b[32mScraped package GCSscore: 1269/2183\u001b[0m\n",
      "\u001b[32mScraped package GEM: 1270/2183\u001b[0m\n",
      "\u001b[32mScraped package GRridge: 1271/2183\u001b[0m\n",
      "\u001b[32mScraped package HEM: 1272/2183\u001b[0m\n",
      "\u001b[32mScraped package imageHTS: 1273/2183\u001b[0m\n",
      "\u001b[32mScraped package IMMAN: 1274/2183\u001b[0m\n",
      "\u001b[32mScraped package martini: 1275/2183\u001b[0m\n",
      "\u001b[32mScraped package MBASED: 1276/2183\u001b[0m\n",
      "\u001b[32mScraped package Metab: 1277/2183\u001b[0m\n",
      "\u001b[32mScraped package MGFR: 1278/2183\u001b[0m\n",
      "\u001b[32mScraped package MIMOSA: 1279/2183\u001b[0m\n",
      "\u001b[32mScraped package MPRAnalyze: 1280/2183\u001b[0m\n",
      "\u001b[32mScraped package MsBackendMsp: 1281/2183\u001b[0m\n",
      "\u001b[32mScraped package netSmooth: 1282/2183\u001b[0m\n",
      "\u001b[32mScraped package NewWave: 1283/2183\u001b[0m\n",
      "\u001b[32mScraped package NuPoP: 1284/2183\u001b[0m\n",
      "\u001b[32mScraped package PathNet: 1285/2183\u001b[0m\n",
      "\u001b[32mScraped package PathoStat: 1286/2183\u001b[0m\n",
      "\u001b[32mScraped package PhIPData: 1287/2183\u001b[0m\n",
      "\u001b[32mScraped package psygenet2r: 1288/2183\u001b[0m\n",
      "\u001b[32mScraped package QuartPAC: 1289/2183\u001b[0m\n",
      "\u001b[32mScraped package RITAN: 1290/2183\u001b[0m\n",
      "\u001b[32mScraped package rnaseqcomp: 1291/2183\u001b[0m\n",
      "\u001b[32mScraped package Rtreemix: 1292/2183\u001b[0m\n",
      "\u001b[32mScraped package sincell: 1293/2183\u001b[0m\n",
      "\u001b[32mScraped package specL: 1294/2183\u001b[0m\n",
      "\u001b[32mScraped package SynExtend: 1295/2183\u001b[0m\n",
      "\u001b[32mScraped package tanggle: 1296/2183\u001b[0m\n",
      "\u001b[32mScraped package TBSignatureProfiler: 1297/2183\u001b[0m\n",
      "\u001b[32mScraped package TNBC.CMS: 1298/2183\u001b[0m\n",
      "\u001b[32mScraped package AffiXcan: 1299/2183\u001b[0m\n",
      "\u001b[32mScraped package appreci8R: 1300/2183\u001b[0m\n",
      "\u001b[32mScraped package ARRmNormalization: 1301/2183\u001b[0m\n",
      "\u001b[32mScraped package AssessORF: 1302/2183\u001b[0m\n",
      "\u001b[32mScraped package BiFET: 1303/2183\u001b[0m\n",
      "\u001b[32mScraped package BioMM: 1304/2183\u001b[0m\n",
      "\u001b[32mScraped package BioTIP: 1305/2183\u001b[0m\n",
      "\u001b[32mScraped package biotmle: 1306/2183\u001b[0m\n",
      "\u001b[32mScraped package categoryCompare: 1307/2183\u001b[0m\n",
      "\u001b[32mScraped package CellMapper: 1308/2183\u001b[0m\n",
      "\u001b[32mScraped package ChIPexoQual: 1309/2183\u001b[0m\n",
      "\u001b[32mScraped package cleanUpdTSeq: 1310/2183\u001b[0m\n",
      "\u001b[32mScraped package compartmap: 1311/2183\u001b[0m\n",
      "\u001b[32mScraped package CSSP: 1312/2183\u001b[0m\n",
      "\u001b[32mScraped package dce: 1313/2183\u001b[0m\n",
      "\u001b[32mScraped package DegNorm: 1314/2183\u001b[0m\n",
      "\u001b[32mScraped package DelayedRandomArray: 1315/2183\u001b[0m\n",
      "\u001b[32mScraped package DEsubs: 1316/2183\u001b[0m\n",
      "\u001b[32mScraped package dyebias: 1317/2183\u001b[0m\n",
      "\u001b[32mScraped package ExiMiR: 1318/2183\u001b[0m\n",
      "\u001b[32mScraped package factDesign: 1319/2183\u001b[0m\n",
      "\u001b[32mScraped package FLAMES: 1320/2183\u001b[0m\n",
      "\u001b[32mScraped package flowBin: 1321/2183\u001b[0m\n",
      "\u001b[32mScraped package flowTime: 1322/2183\u001b[0m\n",
      "\u001b[32mScraped package frmaTools: 1323/2183\u001b[0m\n",
      "\u001b[32mScraped package GateFinder: 1324/2183\u001b[0m\n",
      "\u001b[32mScraped package GISPA: 1325/2183\u001b[0m\n",
      "\u001b[32mScraped package GSCA: 1326/2183\u001b[0m\n",
      "\u001b[32mScraped package gsean: 1327/2183\u001b[0m\n",
      "\u001b[32mScraped package HDTD: 1328/2183\u001b[0m\n",
      "\u001b[32mScraped package iCNV: 1329/2183\u001b[0m\n",
      "\u001b[32mScraped package iPAC: 1330/2183\u001b[0m\n",
      "\u001b[32mScraped package iterClust: 1331/2183\u001b[0m\n",
      "\u001b[32mScraped package les: 1332/2183\u001b[0m\n",
      "\u001b[32mScraped package massiR: 1333/2183\u001b[0m\n",
      "\u001b[32mScraped package metapone: 1334/2183\u001b[0m\n",
      "\u001b[32mScraped package NoRCE: 1335/2183\u001b[0m\n",
      "\u001b[32mScraped package panp: 1336/2183\u001b[0m\n",
      "\u001b[32mScraped package Path2PPI: 1337/2183\u001b[0m\n",
      "\u001b[32mScraped package pathRender: 1338/2183\u001b[0m\n",
      "\u001b[32mScraped package PhenStat: 1339/2183\u001b[0m\n",
      "\u001b[32mScraped package qpcrNorm: 1340/2183\u001b[0m\n",
      "\u001b[32mScraped package ramr: 1341/2183\u001b[0m\n",
      "\u001b[32mScraped package randPack: 1342/2183\u001b[0m\n",
      "\u001b[32mScraped package Rbwa: 1343/2183\u001b[0m\n",
      "\u001b[32mScraped package RCM: 1344/2183\u001b[0m\n",
      "\u001b[32mScraped package RNAmodR.AlkAnilineSeq: 1345/2183\u001b[0m\n",
      "\u001b[32mScraped package scCB2: 1346/2183\u001b[0m\n",
      "\u001b[32mScraped package SCOPE: 1347/2183\u001b[0m\n",
      "\u001b[32mScraped package scRecover: 1348/2183\u001b[0m\n",
      "\u001b[32mScraped package SeqSQC: 1349/2183\u001b[0m\n",
      "\u001b[32mScraped package SIM: 1350/2183\u001b[0m\n",
      "\u001b[32mScraped package SMITE: 1351/2183\u001b[0m\n",
      "\u001b[32mScraped package STAN: 1352/2183\u001b[0m\n",
      "\u001b[32mScraped package TargetScore: 1353/2183\u001b[0m\n",
      "\u001b[32mScraped package TimiRGeN: 1354/2183\u001b[0m\n",
      "\u001b[32mScraped package weaver: 1355/2183\u001b[0m\n",
      "\u001b[32mScraped package webbioc: 1356/2183\u001b[0m\n",
      "\u001b[32mScraped package wpm: 1357/2183\u001b[0m\n",
      "\u001b[32mScraped package yamss: 1358/2183\u001b[0m\n",
      "\u001b[32mScraped package AlphaBeta: 1359/2183\u001b[0m\n",
      "\u001b[32mScraped package annmap: 1360/2183\u001b[0m\n",
      "\u001b[32mScraped package arrayMvout: 1361/2183\u001b[0m\n",
      "\u001b[32mScraped package cbpManager: 1362/2183\u001b[0m\n",
      "\u001b[32mScraped package cellTree: 1363/2183\u001b[0m\n",
      "\u001b[32mScraped package cisPath: 1364/2183\u001b[0m\n",
      "\u001b[32mScraped package clustComp: 1365/2183\u001b[0m\n",
      "\u001b[32mScraped package ClusterJudge: 1366/2183\u001b[0m\n",
      "\u001b[32mScraped package CNVgears: 1367/2183\u001b[0m\n",
      "\u001b[32mScraped package consensusDE: 1368/2183\u001b[0m\n",
      "\u001b[32mScraped package copa: 1369/2183\u001b[0m\n",
      "\u001b[32mScraped package cosmiq: 1370/2183\u001b[0m\n",
      "\u001b[32mScraped package cpvSNP: 1371/2183\u001b[0m\n",
      "\u001b[32mScraped package DExMA: 1372/2183\u001b[0m\n",
      "\u001b[32mScraped package enhancerHomologSearch: 1373/2183\u001b[0m\n",
      "\u001b[32mScraped package ExperimentSubset: 1374/2183\u001b[0m\n",
      "\u001b[32mScraped package fCI: 1375/2183\u001b[0m\n",
      "\u001b[32mScraped package flowPlots: 1376/2183\u001b[0m\n",
      "\u001b[32mScraped package FoldGO: 1377/2183\u001b[0m\n",
      "\u001b[32mScraped package genArise: 1378/2183\u001b[0m\n",
      "\u001b[32mScraped package geneRecommender: 1379/2183\u001b[0m\n",
      "\u001b[32mScraped package geneRxCluster: 1380/2183\u001b[0m\n",
      "\u001b[32mScraped package geneXtendeR: 1381/2183\u001b[0m\n",
      "\u001b[32mScraped package globalSeq: 1382/2183\u001b[0m\n",
      "\u001b[32mScraped package gpuMagic: 1383/2183\u001b[0m\n",
      "\u001b[32mScraped package GRENITS: 1384/2183\u001b[0m\n",
      "\u001b[32mScraped package GSRI: 1385/2183\u001b[0m\n",
      "\u001b[32mScraped package hermes: 1386/2183\u001b[0m\n",
      "\u001b[32mScraped package immunoClust: 1387/2183\u001b[0m\n",
      "\u001b[32mScraped package infinityFlow: 1388/2183\u001b[0m\n",
      "\u001b[32mScraped package InPAS: 1389/2183\u001b[0m\n",
      "\u001b[32mScraped package IntramiRExploreR: 1390/2183\u001b[0m\n",
      "\u001b[32mScraped package KinSwingR: 1391/2183\u001b[0m\n",
      "\u001b[32mScraped package macat: 1392/2183\u001b[0m\n",
      "\u001b[32mScraped package MassArray: 1393/2183\u001b[0m\n",
      "\u001b[32mScraped package mdp: 1394/2183\u001b[0m\n",
      "\u001b[32mScraped package mdqc: 1395/2183\u001b[0m\n",
      "\u001b[32mScraped package methInheritSim: 1396/2183\u001b[0m\n",
      "\u001b[32mScraped package midasHLA: 1397/2183\u001b[0m\n",
      "\u001b[32mScraped package motifcounter: 1398/2183\u001b[0m\n",
      "\u001b[32mScraped package MSstatsQC: 1399/2183\u001b[0m\n",
      "\u001b[32mScraped package MultiMed: 1400/2183\u001b[0m\n",
      "\u001b[32mScraped package npGSEA: 1401/2183\u001b[0m\n",
      "\u001b[32mScraped package omicplotR: 1402/2183\u001b[0m\n",
      "\u001b[32mScraped package OncoScore: 1403/2183\u001b[0m\n",
      "\u001b[32mScraped package pepXMLTab: 1404/2183\u001b[0m\n",
      "\u001b[32mScraped package pipeComp: 1405/2183\u001b[0m\n",
      "\u001b[32mScraped package plethy: 1406/2183\u001b[0m\n",
      "\u001b[32mScraped package PrInCE: 1407/2183\u001b[0m\n",
      "\u001b[32mScraped package recountmethylation: 1408/2183\u001b[0m\n",
      "\u001b[32mScraped package restfulSE: 1409/2183\u001b[0m\n",
      "\u001b[32mScraped package slalom: 1410/2183\u001b[0m\n",
      "\u001b[32mScraped package sscore: 1411/2183\u001b[0m\n",
      "\u001b[32mScraped package STdeconvolve: 1412/2183\u001b[0m\n",
      "\u001b[32mScraped package synlet: 1413/2183\u001b[0m\n",
      "\u001b[32mScraped package topdownr: 1414/2183\u001b[0m\n",
      "\u001b[32mScraped package transite: 1415/2183\u001b[0m\n",
      "\u001b[32mScraped package ttgsea: 1416/2183\u001b[0m\n",
      "\u001b[32mScraped package VariantExperiment: 1417/2183\u001b[0m\n",
      "\u001b[32mScraped package VplotR: 1418/2183\u001b[0m\n",
      "\u001b[32mScraped package wavClusteR: 1419/2183\u001b[0m\n",
      "\u001b[32mScraped package Xeva: 1420/2183\u001b[0m\n",
      "\u001b[32mScraped package XINA: 1421/2183\u001b[0m\n",
      "\u001b[32mScraped package airpart: 1422/2183\u001b[0m\n",
      "\u001b[32mScraped package AnVILPublish: 1423/2183\u001b[0m\n",
      "\u001b[32mScraped package ASAFE: 1424/2183\u001b[0m\n",
      "\u001b[32mScraped package ASpediaFI: 1425/2183\u001b[0m\n",
      "\u001b[32mScraped package BEAT: 1426/2183\u001b[0m\n",
      "\u001b[32mScraped package BGmix: 1427/2183\u001b[0m\n",
      "\u001b[32mScraped package BiocDockerManager: 1428/2183\u001b[0m\n",
      "\u001b[32mScraped package biomvRCNS: 1429/2183\u001b[0m\n",
      "\u001b[32mScraped package brainflowprobes: 1430/2183\u001b[0m\n",
      "\u001b[32mScraped package CAFE: 1431/2183\u001b[0m\n",
      "\u001b[32mScraped package CancerInSilico: 1432/2183\u001b[0m\n",
      "\u001b[32mScraped package cbaf: 1433/2183\u001b[0m\n",
      "\u001b[32mScraped package ChIPanalyser: 1434/2183\u001b[0m\n",
      "\u001b[32mScraped package chromswitch: 1435/2183\u001b[0m\n",
      "\u001b[32mScraped package clippda: 1436/2183\u001b[0m\n",
      "\u001b[32mScraped package contiBAIT: 1437/2183\u001b[0m\n",
      "\u001b[32mScraped package crisprseekplus: 1438/2183\u001b[0m\n",
      "\u001b[32mScraped package deco: 1439/2183\u001b[0m\n",
      "\u001b[32mScraped package DEScan2: 1440/2183\u001b[0m\n",
      "\u001b[32mScraped package DrugVsDisease: 1441/2183\u001b[0m\n",
      "\u001b[32mScraped package ecolitk: 1442/2183\u001b[0m\n",
      "\u001b[32mScraped package eegc: 1443/2183\u001b[0m\n",
      "\u001b[32mScraped package epigenomix: 1444/2183\u001b[0m\n",
      "\u001b[32mScraped package epigraHMM: 1445/2183\u001b[0m\n",
      "\u001b[32mScraped package fCCAC: 1446/2183\u001b[0m\n",
      "\u001b[32mScraped package flowcatchR: 1447/2183\u001b[0m\n",
      "\u001b[32mScraped package flowMap: 1448/2183\u001b[0m\n",
      "\u001b[32mScraped package flowMatch: 1449/2183\u001b[0m\n",
      "\u001b[32mScraped package GAprediction: 1450/2183\u001b[0m\n",
      "\u001b[32mScraped package GARS: 1451/2183\u001b[0m\n",
      "\u001b[32mScraped package genoCN: 1452/2183\u001b[0m\n",
      "\u001b[32mScraped package genotypeeval: 1453/2183\u001b[0m\n",
      "\u001b[32mScraped package GEOsubmission: 1454/2183\u001b[0m\n",
      "\u001b[32mScraped package gep2pep: 1455/2183\u001b[0m\n",
      "\u001b[32mScraped package graper: 1456/2183\u001b[0m\n",
      "\u001b[32mScraped package hierGWAS: 1457/2183\u001b[0m\n",
      "\u001b[32mScraped package ILoReg: 1458/2183\u001b[0m\n",
      "\u001b[32mScraped package IMAS: 1459/2183\u001b[0m\n",
      "\u001b[32mScraped package IRISFGM: 1460/2183\u001b[0m\n",
      "\u001b[32mScraped package ITALICS: 1461/2183\u001b[0m\n",
      "\u001b[32mScraped package LiquidAssociation: 1462/2183\u001b[0m\n",
      "\u001b[32mScraped package logitT: 1463/2183\u001b[0m\n",
      "\u001b[32mScraped package maPredictDSC: 1464/2183\u001b[0m\n",
      "\u001b[32mScraped package MBCB: 1465/2183\u001b[0m\n",
      "\u001b[32mScraped package MCbiclust: 1466/2183\u001b[0m\n",
      "\u001b[32mScraped package metabomxtr: 1467/2183\u001b[0m\n",
      "\u001b[32mScraped package MetID: 1468/2183\u001b[0m\n",
      "\u001b[32mScraped package miaSim: 1469/2183\u001b[0m\n",
      "\u001b[32mScraped package mpra: 1470/2183\u001b[0m\n",
      "\u001b[32mScraped package netDx: 1471/2183\u001b[0m\n",
      "\u001b[32mScraped package NetSAM: 1472/2183\u001b[0m\n",
      "\u001b[32mScraped package NPARC: 1473/2183\u001b[0m\n",
      "\u001b[32mScraped package omicsPrint: 1474/2183\u001b[0m\n",
      "\u001b[32mScraped package ompBAM: 1475/2183\u001b[0m\n",
      "\u001b[32mScraped package paircompviz: 1476/2183\u001b[0m\n",
      "\u001b[32mScraped package plotGrouper: 1477/2183\u001b[0m\n",
      "\u001b[32mScraped package QSutils: 1478/2183\u001b[0m\n",
      "\u001b[32mScraped package Rcwl: 1479/2183\u001b[0m\n",
      "\u001b[32mScraped package RiboDiPA: 1480/2183\u001b[0m\n",
      "\u001b[32mScraped package RLassoCox: 1481/2183\u001b[0m\n",
      "\u001b[32mScraped package RNAinteract: 1482/2183\u001b[0m\n",
      "\u001b[32mScraped package RTopper: 1483/2183\u001b[0m\n",
      "\u001b[32mScraped package SANTA: 1484/2183\u001b[0m\n",
      "\u001b[32mScraped package scGPS: 1485/2183\u001b[0m\n",
      "\u001b[32mScraped package scTGIF: 1486/2183\u001b[0m\n",
      "\u001b[32mScraped package seq2pathway: 1487/2183\u001b[0m\n",
      "\u001b[32mScraped package spatialHeatmap: 1488/2183\u001b[0m\n",
      "\u001b[32mScraped package ssviz: 1489/2183\u001b[0m\n",
      "\u001b[32mScraped package SubCellBarCode: 1490/2183\u001b[0m\n",
      "\u001b[32mScraped package swfdr: 1491/2183\u001b[0m\n",
      "\u001b[32mScraped package switchde: 1492/2183\u001b[0m\n",
      "\u001b[32mScraped package TFutils: 1493/2183\u001b[0m\n",
      "\u001b[32mScraped package tigre: 1494/2183\u001b[0m\n",
      "\u001b[32mScraped package TPP2D: 1495/2183\u001b[0m\n",
      "\u001b[32mScraped package UNDO: 1496/2183\u001b[0m\n",
      "\u001b[32mScraped package ASURAT: 1497/2183\u001b[0m\n",
      "\u001b[32mScraped package awst: 1498/2183\u001b[0m\n",
      "\u001b[32mScraped package BBCAnalyzer: 1499/2183\u001b[0m\n",
      "\u001b[32mScraped package cellscape: 1500/2183\u001b[0m\n",
      "\u001b[32mScraped package ChIC: 1501/2183\u001b[0m\n",
      "\u001b[32mScraped package CONFESS: 1502/2183\u001b[0m\n",
      "\u001b[32mScraped package CONSTANd: 1503/2183\u001b[0m\n",
      "\u001b[32mScraped package Cormotif: 1504/2183\u001b[0m\n",
      "\u001b[32mScraped package customCMPdb: 1505/2183\u001b[0m\n",
      "\u001b[32mScraped package DAMEfinder: 1506/2183\u001b[0m\n",
      "\u001b[32mScraped package dcGSA: 1507/2183\u001b[0m\n",
      "\u001b[32mScraped package DEComplexDisease: 1508/2183\u001b[0m\n",
      "\u001b[32mScraped package deconvR: 1509/2183\u001b[0m\n",
      "\u001b[32mScraped package DeMAND: 1510/2183\u001b[0m\n",
      "\u001b[32mScraped package DFP: 1511/2183\u001b[0m\n",
      "\u001b[32mScraped package Dino: 1512/2183\u001b[0m\n",
      "\u001b[32mScraped package eiR: 1513/2183\u001b[0m\n",
      "\u001b[32mScraped package epivizrChart: 1514/2183\u001b[0m\n",
      "\u001b[32mScraped package fgga: 1515/2183\u001b[0m\n",
      "\u001b[32mScraped package FISHalyseR: 1516/2183\u001b[0m\n",
      "\u001b[32mScraped package flagme: 1517/2183\u001b[0m\n",
      "\u001b[32mScraped package flowCL: 1518/2183\u001b[0m\n",
      "\u001b[32mScraped package GeneGeneInteR: 1519/2183\u001b[0m\n",
      "\u001b[32mScraped package GeneRegionScan: 1520/2183\u001b[0m\n",
      "\u001b[32mScraped package GEOfastq: 1521/2183\u001b[0m\n",
      "\u001b[32mScraped package GNET2: 1522/2183\u001b[0m\n",
      "\u001b[32mScraped package GraphAT: 1523/2183\u001b[0m\n",
      "\u001b[32mScraped package GraphPAC: 1524/2183\u001b[0m\n",
      "\u001b[32mScraped package GSReg: 1525/2183\u001b[0m\n",
      "\u001b[32mScraped package Harshlight: 1526/2183\u001b[0m\n",
      "\u001b[32mScraped package iasva: 1527/2183\u001b[0m\n",
      "\u001b[32mScraped package ibh: 1528/2183\u001b[0m\n",
      "\u001b[32mScraped package iGC: 1529/2183\u001b[0m\n",
      "\u001b[32mScraped package iSeq: 1530/2183\u001b[0m\n",
      "\u001b[32mScraped package KCsmart: 1531/2183\u001b[0m\n",
      "\u001b[32mScraped package MANOR: 1532/2183\u001b[0m\n",
      "\u001b[32mScraped package MBQN: 1533/2183\u001b[0m\n",
      "\u001b[32mScraped package MEDME: 1534/2183\u001b[0m\n",
      "\u001b[32mScraped package metahdep: 1535/2183\u001b[0m\n",
      "\u001b[32mScraped package MetCirc: 1536/2183\u001b[0m\n",
      "\u001b[32mScraped package MGFM: 1537/2183\u001b[0m\n",
      "\u001b[32mScraped package MinimumDistance: 1538/2183\u001b[0m\n",
      "\u001b[32mScraped package MiPP: 1539/2183\u001b[0m\n",
      "\u001b[32mScraped package MiRaGE: 1540/2183\u001b[0m\n",
      "\u001b[32mScraped package MOGAMUN: 1541/2183\u001b[0m\n",
      "\u001b[32mScraped package multicrispr: 1542/2183\u001b[0m\n",
      "\u001b[32mScraped package nanotatoR: 1543/2183\u001b[0m\n",
      "\u001b[32mScraped package ncRNAtools: 1544/2183\u001b[0m\n",
      "\u001b[32mScraped package NxtIRFcore: 1545/2183\u001b[0m\n",
      "\u001b[32mScraped package onlineFDR: 1546/2183\u001b[0m\n",
      "\u001b[32mScraped package OpenStats: 1547/2183\u001b[0m\n",
      "\u001b[32mScraped package PING: 1548/2183\u001b[0m\n",
      "\u001b[32mScraped package proFIA: 1549/2183\u001b[0m\n",
      "\u001b[32mScraped package PSEA: 1550/2183\u001b[0m\n",
      "\u001b[32mScraped package ptairMS: 1551/2183\u001b[0m\n",
      "\u001b[32mScraped package recoup: 1552/2183\u001b[0m\n",
      "\u001b[32mScraped package regutools: 1553/2183\u001b[0m\n",
      "\u001b[32mScraped package rgsepd: 1554/2183\u001b[0m\n",
      "\u001b[32mScraped package Rmagpie: 1555/2183\u001b[0m\n",
      "\u001b[32mScraped package rRDP: 1556/2183\u001b[0m\n",
      "\u001b[32mScraped package RTCA: 1557/2183\u001b[0m\n",
      "\u001b[32mScraped package SDAMS: 1558/2183\u001b[0m\n",
      "\u001b[32mScraped package seqCAT: 1559/2183\u001b[0m\n",
      "\u001b[32mScraped package sitePath: 1560/2183\u001b[0m\n",
      "\u001b[32mScraped package sRACIPE: 1561/2183\u001b[0m\n",
      "\u001b[32mScraped package ssrch: 1562/2183\u001b[0m\n",
      "\u001b[32mScraped package SummarizedBenchmark: 1563/2183\u001b[0m\n",
      "\u001b[32mScraped package TnT: 1564/2183\u001b[0m\n",
      "\u001b[32mScraped package TransView: 1565/2183\u001b[0m\n",
      "\u001b[32mScraped package triplex: 1566/2183\u001b[0m\n",
      "\u001b[32mScraped package twoddpcr: 1567/2183\u001b[0m\n",
      "\u001b[32mScraped package uSORT: 1568/2183\u001b[0m\n",
      "\u001b[32mScraped package XNAString: 1569/2183\u001b[0m\n",
      "\u001b[32mScraped package AnVILBilling: 1570/2183\u001b[0m\n",
      "\u001b[32mScraped package BindingSiteFinder: 1571/2183\u001b[0m\n",
      "\u001b[32mScraped package bridge: 1572/2183\u001b[0m\n",
      "\u001b[32mScraped package BUScorrect: 1573/2183\u001b[0m\n",
      "\u001b[32mScraped package calm: 1574/2183\u001b[0m\n",
      "\u001b[32mScraped package ChIPXpress: 1575/2183\u001b[0m\n",
      "\u001b[32mScraped package chromDraw: 1576/2183\u001b[0m\n",
      "\u001b[32mScraped package CNVrd2: 1577/2183\u001b[0m\n",
      "\u001b[32mScraped package CoCiteStats: 1578/2183\u001b[0m\n",
      "\u001b[32mScraped package covEB: 1579/2183\u001b[0m\n",
      "\u001b[32mScraped package covRNA: 1580/2183\u001b[0m\n",
      "\u001b[32mScraped package CTDquerier: 1581/2183\u001b[0m\n",
      "\u001b[32mScraped package DMRforPairs: 1582/2183\u001b[0m\n",
      "\u001b[32mScraped package doseR: 1583/2183\u001b[0m\n",
      "\u001b[32mScraped package dpeak: 1584/2183\u001b[0m\n",
      "\u001b[32mScraped package DTA: 1585/2183\u001b[0m\n",
      "\u001b[32mScraped package evaluomeR: 1586/2183\u001b[0m\n",
      "\u001b[32mScraped package fastLiquidAssociation: 1587/2183\u001b[0m\n",
      "\u001b[32mScraped package fcScan: 1588/2183\u001b[0m\n",
      "\u001b[32mScraped package FindIT2: 1589/2183\u001b[0m\n",
      "\u001b[32mScraped package flowBeads: 1590/2183\u001b[0m\n",
      "\u001b[32mScraped package flowCHIC: 1591/2183\u001b[0m\n",
      "\u001b[32mScraped package fobitools: 1592/2183\u001b[0m\n",
      "\u001b[32mScraped package FunChIP: 1593/2183\u001b[0m\n",
      "\u001b[32mScraped package gemini: 1594/2183\u001b[0m\n",
      "\u001b[32mScraped package GenomicTuples: 1595/2183\u001b[0m\n",
      "\u001b[32mScraped package GWAS.BAYES: 1596/2183\u001b[0m\n",
      "\u001b[32mScraped package HELP: 1597/2183\u001b[0m\n",
      "\u001b[32mScraped package HIPPO: 1598/2183\u001b[0m\n",
      "\u001b[32mScraped package HIREewas: 1599/2183\u001b[0m\n",
      "\u001b[32mScraped package HumanTranscriptomeCompendium: 1600/2183\u001b[0m\n",
      "\u001b[32mScraped package INPower: 1601/2183\u001b[0m\n",
      "\u001b[32mScraped package InTAD: 1602/2183\u001b[0m\n",
      "\u001b[32mScraped package ISoLDE: 1603/2183\u001b[0m\n",
      "\u001b[32mScraped package kissDE: 1604/2183\u001b[0m\n",
      "\u001b[32mScraped package lapmix: 1605/2183\u001b[0m\n",
      "\u001b[32mScraped package maCorrPlot: 1606/2183\u001b[0m\n",
      "\u001b[32mScraped package MAI: 1607/2183\u001b[0m\n",
      "\u001b[32mScraped package mAPKL: 1608/2183\u001b[0m\n",
      "\u001b[32mScraped package maskBAD: 1609/2183\u001b[0m\n",
      "\u001b[32mScraped package MEAT: 1610/2183\u001b[0m\n",
      "\u001b[32mScraped package metabCombiner: 1611/2183\u001b[0m\n",
      "\u001b[32mScraped package MIGSA: 1612/2183\u001b[0m\n",
      "\u001b[32mScraped package miRSM: 1613/2183\u001b[0m\n",
      "\u001b[32mScraped package mirTarRnaSeq: 1614/2183\u001b[0m\n",
      "\u001b[32mScraped package MMDiff2: 1615/2183\u001b[0m\n",
      "\u001b[32mScraped package mosbi: 1616/2183\u001b[0m\n",
      "\u001b[32mScraped package MSA2dist: 1617/2183\u001b[0m\n",
      "\u001b[32mScraped package NADfinder: 1618/2183\u001b[0m\n",
      "\u001b[32mScraped package nnNorm: 1619/2183\u001b[0m\n",
      "\u001b[32mScraped package nucleoSim: 1620/2183\u001b[0m\n",
      "\u001b[32mScraped package OLINgui: 1621/2183\u001b[0m\n",
      "\u001b[32mScraped package omicRexposome: 1622/2183\u001b[0m\n",
      "\u001b[32mScraped package oppar: 1623/2183\u001b[0m\n",
      "\u001b[32mScraped package packFinder: 1624/2183\u001b[0m\n",
      "\u001b[32mScraped package pageRank: 1625/2183\u001b[0m\n",
      "\u001b[32mScraped package PAST: 1626/2183\u001b[0m\n",
      "\u001b[32mScraped package pepStat: 1627/2183\u001b[0m\n",
      "\u001b[32mScraped package PLPE: 1628/2183\u001b[0m\n",
      "\u001b[32mScraped package pvac: 1629/2183\u001b[0m\n",
      "\u001b[32mScraped package R3CPET: 1630/2183\u001b[0m\n",
      "\u001b[32mScraped package rama: 1631/2183\u001b[0m\n",
      "\u001b[32mScraped package RareVariantVis: 1632/2183\u001b[0m\n",
      "\u001b[32mScraped package rfPred: 1633/2183\u001b[0m\n",
      "\u001b[32mScraped package ribor: 1634/2183\u001b[0m\n",
      "\u001b[32mScraped package RLMM: 1635/2183\u001b[0m\n",
      "\u001b[32mScraped package RNAmodR.RiboMethSeq: 1636/2183\u001b[0m\n",
      "\u001b[32mScraped package RPA: 1637/2183\u001b[0m\n",
      "\u001b[32mScraped package seqcombo: 1638/2183\u001b[0m\n",
      "\u001b[32mScraped package sevenC: 1639/2183\u001b[0m\n",
      "\u001b[32mScraped package SNAGEE: 1640/2183\u001b[0m\n",
      "\u001b[32mScraped package SpacePAC: 1641/2183\u001b[0m\n",
      "\u001b[32mScraped package spiky: 1642/2183\u001b[0m\n",
      "\u001b[32mScraped package stepNorm: 1643/2183\u001b[0m\n",
      "\u001b[32mScraped package strandCheckR: 1644/2183\u001b[0m\n",
      "\u001b[32mScraped package TIN: 1645/2183\u001b[0m\n",
      "\u001b[32mScraped package tRanslatome: 1646/2183\u001b[0m\n",
      "\u001b[32mScraped package trena: 1647/2183\u001b[0m\n",
      "\u001b[32mScraped package trigger: 1648/2183\u001b[0m\n",
      "\u001b[32mScraped package TVTB: 1649/2183\u001b[0m\n",
      "\u001b[32mScraped package UMI4Cats: 1650/2183\u001b[0m\n",
      "\u001b[32mScraped package veloviz: 1651/2183\u001b[0m\n",
      "\u001b[32mScraped package ArrayExpressHTS: 1652/2183\u001b[0m\n",
      "\u001b[32mScraped package barcodetrackR: 1653/2183\u001b[0m\n",
      "\u001b[32mScraped package BEARscc: 1654/2183\u001b[0m\n",
      "\u001b[32mScraped package benchdamic: 1655/2183\u001b[0m\n",
      "\u001b[32mScraped package bnem: 1656/2183\u001b[0m\n",
      "\u001b[32mScraped package BrainSABER: 1657/2183\u001b[0m\n",
      "\u001b[32mScraped package cellmigRation: 1658/2183\u001b[0m\n",
      "\u001b[32mScraped package CellScore: 1659/2183\u001b[0m\n",
      "\u001b[32mScraped package clusterSeq: 1660/2183\u001b[0m\n",
      "\u001b[32mScraped package ClusterSignificance: 1661/2183\u001b[0m\n",
      "\u001b[32mScraped package cnvGSA: 1662/2183\u001b[0m\n",
      "\u001b[32mScraped package combi: 1663/2183\u001b[0m\n",
      "\u001b[32mScraped package consensus: 1664/2183\u001b[0m\n",
      "\u001b[32mScraped package csdR: 1665/2183\u001b[0m\n",
      "\u001b[32mScraped package cycle: 1666/2183\u001b[0m\n",
      "\u001b[32mScraped package cytoKernel: 1667/2183\u001b[0m\n",
      "\u001b[32mScraped package DeepPINCS: 1668/2183\u001b[0m\n",
      "\u001b[32mScraped package DMCHMM: 1669/2183\u001b[0m\n",
      "\u001b[32mScraped package easyreporting: 1670/2183\u001b[0m\n",
      "\u001b[32mScraped package EBSEA: 1671/2183\u001b[0m\n",
      "\u001b[32mScraped package epialleleR: 1672/2183\u001b[0m\n",
      "\u001b[32mScraped package FRGEpistasis: 1673/2183\u001b[0m\n",
      "\u001b[32mScraped package funtooNorm: 1674/2183\u001b[0m\n",
      "\u001b[32mScraped package gaggle: 1675/2183\u001b[0m\n",
      "\u001b[32mScraped package geneAttribution: 1676/2183\u001b[0m\n",
      "\u001b[32mScraped package hapFabia: 1677/2183\u001b[0m\n",
      "\u001b[32mScraped package HiLDA: 1678/2183\u001b[0m\n",
      "\u001b[32mScraped package icetea: 1679/2183\u001b[0m\n",
      "\u001b[32mScraped package iterativeBMAsurv: 1680/2183\u001b[0m\n",
      "\u001b[32mScraped package LACE: 1681/2183\u001b[0m\n",
      "\u001b[32mScraped package LineagePulse: 1682/2183\u001b[0m\n",
      "\u001b[32mScraped package LRcell: 1683/2183\u001b[0m\n",
      "\u001b[32mScraped package MADSEQ: 1684/2183\u001b[0m\n",
      "\u001b[32mScraped package MAGAR: 1685/2183\u001b[0m\n",
      "\u001b[32mScraped package mBPCR: 1686/2183\u001b[0m\n",
      "\u001b[32mScraped package MDTS: 1687/2183\u001b[0m\n",
      "\u001b[32mScraped package messina: 1688/2183\u001b[0m\n",
      "\u001b[32mScraped package methylMnM: 1689/2183\u001b[0m\n",
      "\u001b[32mScraped package mimager: 1690/2183\u001b[0m\n",
      "\u001b[32mScraped package mirIntegrator: 1691/2183\u001b[0m\n",
      "\u001b[32mScraped package MOSim: 1692/2183\u001b[0m\n",
      "\u001b[32mScraped package msgbsR: 1693/2183\u001b[0m\n",
      "\u001b[32mScraped package MSstatsQCgui: 1694/2183\u001b[0m\n",
      "\u001b[32mScraped package MSstatsSampleSize: 1695/2183\u001b[0m\n",
      "\u001b[32mScraped package NBSplice: 1696/2183\u001b[0m\n",
      "\u001b[32mScraped package netOmics: 1697/2183\u001b[0m\n",
      "\u001b[32mScraped package OmicsLonDA: 1698/2183\u001b[0m\n",
      "\u001b[32mScraped package OMICsPCA: 1699/2183\u001b[0m\n",
      "\u001b[32mScraped package ORFhunteR: 1700/2183\u001b[0m\n",
      "\u001b[32mScraped package periodicDNA: 1701/2183\u001b[0m\n",
      "\u001b[32mScraped package ReQON: 1702/2183\u001b[0m\n",
      "\u001b[32mScraped package RGraph2js: 1703/2183\u001b[0m\n",
      "\u001b[32mScraped package rmspc: 1704/2183\u001b[0m\n",
      "\u001b[32mScraped package rqt: 1705/2183\u001b[0m\n",
      "\u001b[32mScraped package rScudo: 1706/2183\u001b[0m\n",
      "\u001b[32mScraped package sampleClassifier: 1707/2183\u001b[0m\n",
      "\u001b[32mScraped package scAlign: 1708/2183\u001b[0m\n",
      "\u001b[32mScraped package scanMiR: 1709/2183\u001b[0m\n",
      "\u001b[32mScraped package SCArray: 1710/2183\u001b[0m\n",
      "\u001b[32mScraped package SELEX: 1711/2183\u001b[0m\n",
      "\u001b[32mScraped package SigCheck: 1712/2183\u001b[0m\n",
      "\u001b[32mScraped package SingleMoleculeFootprinting: 1713/2183\u001b[0m\n",
      "\u001b[32mScraped package SISPA: 1714/2183\u001b[0m\n",
      "\u001b[32mScraped package SNPediaR: 1715/2183\u001b[0m\n",
      "\u001b[32mScraped package sparseDOSSA: 1716/2183\u001b[0m\n",
      "\u001b[32mScraped package SQLDataFrame: 1717/2183\u001b[0m\n",
      "\u001b[32mScraped package srnadiff: 1718/2183\u001b[0m\n",
      "\u001b[32mScraped package sscu: 1719/2183\u001b[0m\n",
      "\u001b[32mScraped package StarBioTrek: 1720/2183\u001b[0m\n",
      "\u001b[32mScraped package supersigs: 1721/2183\u001b[0m\n",
      "\u001b[32mScraped package TargetDecoy: 1722/2183\u001b[0m\n",
      "\u001b[32mScraped package TileDBArray: 1723/2183\u001b[0m\n",
      "\u001b[32mScraped package TurboNorm: 1724/2183\u001b[0m\n",
      "\u001b[32mScraped package TypeInfo: 1725/2183\u001b[0m\n",
      "\u001b[32mScraped package Ularcirc: 1726/2183\u001b[0m\n",
      "\u001b[32mScraped package unifiedWMWqPCR: 1727/2183\u001b[0m\n",
      "\u001b[32mScraped package VCFArray: 1728/2183\u001b[0m\n",
      "\u001b[32mScraped package vulcan: 1729/2183\u001b[0m\n",
      "\u001b[32mScraped package APL: 1730/2183\u001b[0m\n",
      "\u001b[32mScraped package banocc: 1731/2183\u001b[0m\n",
      "\u001b[32mScraped package biodbChebi: 1732/2183\u001b[0m\n",
      "\u001b[32mScraped package biodbHmdb: 1733/2183\u001b[0m\n",
      "\u001b[32mScraped package biodbUniprot: 1734/2183\u001b[0m\n",
      "\u001b[32mScraped package CCPROMISE: 1735/2183\u001b[0m\n",
      "\u001b[32mScraped package cfDNAPro: 1736/2183\u001b[0m\n",
      "\u001b[32mScraped package CNViz: 1737/2183\u001b[0m\n",
      "\u001b[32mScraped package cyanoFilter: 1738/2183\u001b[0m\n",
      "\u001b[32mScraped package ddPCRclust: 1739/2183\u001b[0m\n",
      "\u001b[32mScraped package diffUTR: 1740/2183\u001b[0m\n",
      "\u001b[32mScraped package DMCFB: 1741/2183\u001b[0m\n",
      "\u001b[32mScraped package epistack: 1742/2183\u001b[0m\n",
      "\u001b[32mScraped package ERSSA: 1743/2183\u001b[0m\n",
      "\u001b[32mScraped package famat: 1744/2183\u001b[0m\n",
      "\u001b[32mScraped package geneClassifiers: 1745/2183\u001b[0m\n",
      "\u001b[32mScraped package GenomicOZone: 1746/2183\u001b[0m\n",
      "\u001b[32mScraped package geva: 1747/2183\u001b[0m\n",
      "\u001b[32mScraped package ggmanh: 1748/2183\u001b[0m\n",
      "\u001b[32mScraped package GIGSEA: 1749/2183\u001b[0m\n",
      "\u001b[32mScraped package gmoviz: 1750/2183\u001b[0m\n",
      "\u001b[32mScraped package GMRP: 1751/2183\u001b[0m\n",
      "\u001b[32mScraped package GRaNIE: 1752/2183\u001b[0m\n",
      "\u001b[32mScraped package GSgalgoR: 1753/2183\u001b[0m\n",
      "\u001b[32mScraped package iASeq: 1754/2183\u001b[0m\n",
      "\u001b[32mScraped package iChip: 1755/2183\u001b[0m\n",
      "\u001b[32mScraped package Informeasure: 1756/2183\u001b[0m\n",
      "\u001b[32mScraped package ipdDb: 1757/2183\u001b[0m\n",
      "\u001b[32mScraped package ivygapSE: 1758/2183\u001b[0m\n",
      "\u001b[32mScraped package KBoost: 1759/2183\u001b[0m\n",
      "\u001b[32mScraped package MantelCorr: 1760/2183\u001b[0m\n",
      "\u001b[32mScraped package MeasurementError.cor: 1761/2183\u001b[0m\n",
      "\u001b[32mScraped package microbiomeDASim: 1762/2183\u001b[0m\n",
      "\u001b[32mScraped package multiscan: 1763/2183\u001b[0m\n",
      "\u001b[32mScraped package NBAMSeq: 1764/2183\u001b[0m\n",
      "\u001b[32mScraped package nempi: 1765/2183\u001b[0m\n",
      "\u001b[32mScraped package nnSVG: 1766/2183\u001b[0m\n",
      "\u001b[32mScraped package nuCpos: 1767/2183\u001b[0m\n",
      "\u001b[32mScraped package oncomix: 1768/2183\u001b[0m\n",
      "\u001b[32mScraped package OTUbase: 1769/2183\u001b[0m\n",
      "\u001b[32mScraped package PANR: 1770/2183\u001b[0m\n",
      "\u001b[32mScraped package parglms: 1771/2183\u001b[0m\n",
      "\u001b[32mScraped package POWSC: 1772/2183\u001b[0m\n",
      "\u001b[32mScraped package pram: 1773/2183\u001b[0m\n",
      "\u001b[32mScraped package prebs: 1774/2183\u001b[0m\n",
      "\u001b[32mScraped package proBAMr: 1775/2183\u001b[0m\n",
      "\u001b[32mScraped package PROMISE: 1776/2183\u001b[0m\n",
      "\u001b[32mScraped package PROPS: 1777/2183\u001b[0m\n",
      "\u001b[32mScraped package Qtlizer: 1778/2183\u001b[0m\n",
      "\u001b[32mScraped package Rbec: 1779/2183\u001b[0m\n",
      "\u001b[32mScraped package RCSL: 1780/2183\u001b[0m\n",
      "\u001b[32mScraped package RepViz: 1781/2183\u001b[0m\n",
      "\u001b[32mScraped package rfaRm: 1782/2183\u001b[0m\n",
      "\u001b[32mScraped package RJMCMCNucleosomes: 1783/2183\u001b[0m\n",
      "\u001b[32mScraped package RLSeq: 1784/2183\u001b[0m\n",
      "\u001b[32mScraped package Rmmquant: 1785/2183\u001b[0m\n",
      "\u001b[32mScraped package RVS: 1786/2183\u001b[0m\n",
      "\u001b[32mScraped package scMAGeCK: 1787/2183\u001b[0m\n",
      "\u001b[32mScraped package SigsPack: 1788/2183\u001b[0m\n",
      "\u001b[32mScraped package SIMAT: 1789/2183\u001b[0m\n",
      "\u001b[32mScraped package SMAP: 1790/2183\u001b[0m\n",
      "\u001b[32mScraped package SNPhood: 1791/2183\u001b[0m\n",
      "\u001b[32mScraped package target: 1792/2183\u001b[0m\n",
      "\u001b[32mScraped package TimeSeriesExperiment: 1793/2183\u001b[0m\n",
      "\u001b[32mScraped package transformGamPoi: 1794/2183\u001b[0m\n",
      "\u001b[32mScraped package traseR: 1795/2183\u001b[0m\n",
      "\u001b[32mScraped package VegaMC: 1796/2183\u001b[0m\n",
      "\u001b[32mScraped package vtpnet: 1797/2183\u001b[0m\n",
      "\u001b[32mScraped package wppi: 1798/2183\u001b[0m\n",
      "\u001b[32mScraped package BayesKnockdown: 1799/2183\u001b[0m\n",
      "\u001b[32mScraped package CellBarcode: 1800/2183\u001b[0m\n",
      "\u001b[32mScraped package CexoR: 1801/2183\u001b[0m\n",
      "\u001b[32mScraped package CIMICE: 1802/2183\u001b[0m\n",
      "\u001b[32mScraped package Clomial: 1803/2183\u001b[0m\n",
      "\u001b[32mScraped package diffGeneAnalysis: 1804/2183\u001b[0m\n",
      "\u001b[32mScraped package DNABarcodeCompatibility: 1805/2183\u001b[0m\n",
      "\u001b[32mScraped package Dune: 1806/2183\u001b[0m\n",
      "\u001b[32mScraped package EpiCompare: 1807/2183\u001b[0m\n",
      "\u001b[32mScraped package epihet: 1808/2183\u001b[0m\n",
      "\u001b[32mScraped package ExCluster: 1809/2183\u001b[0m\n",
      "\u001b[32mScraped package FilterFFPE: 1810/2183\u001b[0m\n",
      "\u001b[32mScraped package flowCyBar: 1811/2183\u001b[0m\n",
      "\u001b[32mScraped package FScanR: 1812/2183\u001b[0m\n",
      "\u001b[32mScraped package GA4GHshiny: 1813/2183\u001b[0m\n",
      "\u001b[32mScraped package GeneGA: 1814/2183\u001b[0m\n",
      "\u001b[32mScraped package GEWIST: 1815/2183\u001b[0m\n",
      "\u001b[32mScraped package HPAStainR: 1816/2183\u001b[0m\n",
      "\u001b[32mScraped package IMPCdata: 1817/2183\u001b[0m\n",
      "\u001b[32mScraped package IsoCorrectoRGUI: 1818/2183\u001b[0m\n",
      "\u001b[32mScraped package IWTomics: 1819/2183\u001b[0m\n",
      "\u001b[32mScraped package LPEadj: 1820/2183\u001b[0m\n",
      "\u001b[32mScraped package mapscape: 1821/2183\u001b[0m\n",
      "\u001b[32mScraped package Melissa: 1822/2183\u001b[0m\n",
      "\u001b[32mScraped package MethPed: 1823/2183\u001b[0m\n",
      "\u001b[32mScraped package missRows: 1824/2183\u001b[0m\n",
      "\u001b[32mScraped package MsBackendRawFileReader: 1825/2183\u001b[0m\n",
      "\u001b[32mScraped package ncGTW: 1826/2183\u001b[0m\n",
      "\u001b[32mScraped package nearBynding: 1827/2183\u001b[0m\n",
      "\u001b[32mScraped package netprioR: 1828/2183\u001b[0m\n",
      "\u001b[32mScraped package netZooR: 1829/2183\u001b[0m\n",
      "\u001b[32mScraped package Omixer: 1830/2183\u001b[0m\n",
      "\u001b[32mScraped package padma: 1831/2183\u001b[0m\n",
      "\u001b[32mScraped package pcxn: 1832/2183\u001b[0m\n",
      "\u001b[32mScraped package PDATK: 1833/2183\u001b[0m\n",
      "\u001b[32mScraped package pgca: 1834/2183\u001b[0m\n",
      "\u001b[32mScraped package pickgene: 1835/2183\u001b[0m\n",
      "\u001b[32mScraped package pogos: 1836/2183\u001b[0m\n",
      "\u001b[32mScraped package primirTSS: 1837/2183\u001b[0m\n",
      "\u001b[32mScraped package procoil: 1838/2183\u001b[0m\n",
      "\u001b[32mScraped package ProteoDisco: 1839/2183\u001b[0m\n",
      "\u001b[32mScraped package RImmPort: 1840/2183\u001b[0m\n",
      "\u001b[32mScraped package RNAmodR.ML: 1841/2183\u001b[0m\n",
      "\u001b[32mScraped package rprimer: 1842/2183\u001b[0m\n",
      "\u001b[32mScraped package SCATE: 1843/2183\u001b[0m\n",
      "\u001b[32mScraped package scBFA: 1844/2183\u001b[0m\n",
      "\u001b[32mScraped package SCFA: 1845/2183\u001b[0m\n",
      "\u001b[32mScraped package scoreInvHap: 1846/2183\u001b[0m\n",
      "\u001b[32mScraped package SemDist: 1847/2183\u001b[0m\n",
      "\u001b[32mScraped package SigFuge: 1848/2183\u001b[0m\n",
      "\u001b[32mScraped package sigsquared: 1849/2183\u001b[0m\n",
      "\u001b[32mScraped package SimBindProfiles: 1850/2183\u001b[0m\n",
      "\u001b[32mScraped package skewr: 1851/2183\u001b[0m\n",
      "\u001b[32mScraped package spkTools: 1852/2183\u001b[0m\n",
      "\u001b[32mScraped package spqn: 1853/2183\u001b[0m\n",
      "\u001b[32mScraped package STROMA4: 1854/2183\u001b[0m\n",
      "\u001b[32mScraped package SynMut: 1855/2183\u001b[0m\n",
      "\u001b[32mScraped package systemPipeTools: 1856/2183\u001b[0m\n",
      "\u001b[32mScraped package TarSeqQC: 1857/2183\u001b[0m\n",
      "\u001b[32mScraped package TRESS: 1858/2183\u001b[0m\n",
      "\u001b[32mScraped package Uniquorn: 1859/2183\u001b[0m\n",
      "\u001b[32mScraped package VaSP: 1860/2183\u001b[0m\n",
      "\u001b[32mScraped package xmapbridge: 1861/2183\u001b[0m\n",
      "\u001b[32mScraped package BUMHMM: 1862/2183\u001b[0m\n",
      "\u001b[32mScraped package BUSseq: 1863/2183\u001b[0m\n",
      "\u001b[32mScraped package cageminer: 1864/2183\u001b[0m\n",
      "\u001b[32mScraped package cliProfiler: 1865/2183\u001b[0m\n",
      "\u001b[32mScraped package ComPrAn: 1866/2183\u001b[0m\n",
      "\u001b[32mScraped package COTAN: 1867/2183\u001b[0m\n",
      "\u001b[32mScraped package CSSQ: 1868/2183\u001b[0m\n",
      "\u001b[32mScraped package DelayedTensor: 1869/2183\u001b[0m\n",
      "\u001b[32mScraped package deltaGseg: 1870/2183\u001b[0m\n",
      "\u001b[32mScraped package Director: 1871/2183\u001b[0m\n",
      "\u001b[32mScraped package divergence: 1872/2183\u001b[0m\n",
      "\u001b[32mScraped package DominoEffect: 1873/2183\u001b[0m\n",
      "\u001b[32mScraped package Doscheda: 1874/2183\u001b[0m\n",
      "\u001b[32mScraped package eudysbiome: 1875/2183\u001b[0m\n",
      "\u001b[32mScraped package flowGraph: 1876/2183\u001b[0m\n",
      "\u001b[32mScraped package genomicInstability: 1877/2183\u001b[0m\n",
      "\u001b[32mScraped package GladiaTOX: 1878/2183\u001b[0m\n",
      "\u001b[32mScraped package HPiP: 1879/2183\u001b[0m\n",
      "\u001b[32mScraped package hummingbird: 1880/2183\u001b[0m\n",
      "\u001b[32mScraped package IgGeneUsage: 1881/2183\u001b[0m\n",
      "\u001b[32mScraped package immunotation: 1882/2183\u001b[0m\n",
      "\u001b[32mScraped package loci2path: 1883/2183\u001b[0m\n",
      "\u001b[32mScraped package lpNet: 1884/2183\u001b[0m\n",
      "\u001b[32mScraped package matchBox: 1885/2183\u001b[0m\n",
      "\u001b[32mScraped package MatrixRider: 1886/2183\u001b[0m\n",
      "\u001b[32mScraped package MiChip: 1887/2183\u001b[0m\n",
      "\u001b[32mScraped package mina: 1888/2183\u001b[0m\n",
      "\u001b[32mScraped package miRmine: 1889/2183\u001b[0m\n",
      "\u001b[32mScraped package mitoClone2: 1890/2183\u001b[0m\n",
      "\u001b[32mScraped package NeuCA: 1891/2183\u001b[0m\n",
      "\u001b[32mScraped package normalize450K: 1892/2183\u001b[0m\n",
      "\u001b[32mScraped package OPWeight: 1893/2183\u001b[0m\n",
      "\u001b[32mScraped package peco: 1894/2183\u001b[0m\n",
      "\u001b[32mScraped package pmm: 1895/2183\u001b[0m\n",
      "\u001b[32mScraped package preciseTAD: 1896/2183\u001b[0m\n",
      "\u001b[32mScraped package RadioGx: 1897/2183\u001b[0m\n",
      "\u001b[32mScraped package randRotation: 1898/2183\u001b[0m\n",
      "\u001b[32mScraped package REBET: 1899/2183\u001b[0m\n",
      "\u001b[32mScraped package ROSeq: 1900/2183\u001b[0m\n",
      "\u001b[32mScraped package Rtpca: 1901/2183\u001b[0m\n",
      "\u001b[32mScraped package scTHI: 1902/2183\u001b[0m\n",
      "\u001b[32mScraped package sitadela: 1903/2183\u001b[0m\n",
      "\u001b[32mScraped package sojourner: 1904/2183\u001b[0m\n",
      "\u001b[32mScraped package spatzie: 1905/2183\u001b[0m\n",
      "\u001b[32mScraped package SPLINTER: 1906/2183\u001b[0m\n",
      "\u001b[32mScraped package staRank: 1907/2183\u001b[0m\n",
      "\u001b[32mScraped package synapter: 1908/2183\u001b[0m\n",
      "\u001b[32mScraped package ternarynet: 1909/2183\u001b[0m\n",
      "\u001b[32mScraped package tLOH: 1910/2183\u001b[0m\n",
      "\u001b[32mScraped package ToxicoGx: 1911/2183\u001b[0m\n",
      "\u001b[32mScraped package VAExprs: 1912/2183\u001b[0m\n",
      "\u001b[32mScraped package VarCon: 1913/2183\u001b[0m\n",
      "\u001b[32mScraped package weitrix: 1914/2183\u001b[0m\n",
      "\u001b[32mScraped package CAEN: 1915/2183\u001b[0m\n",
      "\u001b[32mScraped package ceRNAnetsim: 1916/2183\u001b[0m\n",
      "\u001b[32mScraped package Cogito: 1917/2183\u001b[0m\n",
      "\u001b[32mScraped package deltaCaptureC: 1918/2183\u001b[0m\n",
      "\u001b[32mScraped package DMRScan: 1919/2183\u001b[0m\n",
      "\u001b[32mScraped package EnMCB: 1920/2183\u001b[0m\n",
      "\u001b[32mScraped package fedup: 1921/2183\u001b[0m\n",
      "\u001b[32mScraped package GBScleanR: 1922/2183\u001b[0m\n",
      "\u001b[32mScraped package GeneAccord: 1923/2183\u001b[0m\n",
      "\u001b[32mScraped package GPA: 1924/2183\u001b[0m\n",
      "\u001b[32mScraped package iCheck: 1925/2183\u001b[0m\n",
      "\u001b[32mScraped package INDEED: 1926/2183\u001b[0m\n",
      "\u001b[32mScraped package LedPred: 1927/2183\u001b[0m\n",
      "\u001b[32mScraped package marr: 1928/2183\u001b[0m\n",
      "\u001b[32mScraped package MBttest: 1929/2183\u001b[0m\n",
      "\u001b[32mScraped package MEB: 1930/2183\u001b[0m\n",
      "\u001b[32mScraped package MethTargetedNGS: 1931/2183\u001b[0m\n",
      "\u001b[32mScraped package miRcomp: 1932/2183\u001b[0m\n",
      "\u001b[32mScraped package MouseFM: 1933/2183\u001b[0m\n",
      "\u001b[32mScraped package NeighborNet: 1934/2183\u001b[0m\n",
      "\u001b[32mScraped package occugene: 1935/2183\u001b[0m\n",
      "\u001b[32mScraped package oppti: 1936/2183\u001b[0m\n",
      "\u001b[32mScraped package optimalFlow: 1937/2183\u001b[0m\n",
      "\u001b[32mScraped package PFP: 1938/2183\u001b[0m\n",
      "\u001b[32mScraped package ppcseq: 1939/2183\u001b[0m\n",
      "\u001b[32mScraped package profileScoreDist: 1940/2183\u001b[0m\n",
      "\u001b[32mScraped package proteinProfiles: 1941/2183\u001b[0m\n",
      "\u001b[32mScraped package RCASPAR: 1942/2183\u001b[0m\n",
      "\u001b[32mScraped package RcwlPipelines: 1943/2183\u001b[0m\n",
      "\u001b[32mScraped package reconsi: 1944/2183\u001b[0m\n",
      "\u001b[32mScraped package regsplice: 1945/2183\u001b[0m\n",
      "\u001b[32mScraped package RIVER: 1946/2183\u001b[0m\n",
      "\u001b[32mScraped package rnaEditr: 1947/2183\u001b[0m\n",
      "\u001b[32mScraped package RNAsense: 1948/2183\u001b[0m\n",
      "\u001b[32mScraped package rSWeeP: 1949/2183\u001b[0m\n",
      "\u001b[32mScraped package rTRMui: 1950/2183\u001b[0m\n",
      "\u001b[32mScraped package sagenhaft: 1951/2183\u001b[0m\n",
      "\u001b[32mScraped package Scale4C: 1952/2183\u001b[0m\n",
      "\u001b[32mScraped package scFeatureFilter: 1953/2183\u001b[0m\n",
      "\u001b[32mScraped package Sconify: 1954/2183\u001b[0m\n",
      "\u001b[32mScraped package scTreeViz: 1955/2183\u001b[0m\n",
      "\u001b[32mScraped package segmenter: 1956/2183\u001b[0m\n",
      "\u001b[32mScraped package SIMD: 1957/2183\u001b[0m\n",
      "\u001b[32mScraped package SimFFPE: 1958/2183\u001b[0m\n",
      "\u001b[32mScraped package TDARACNE: 1959/2183\u001b[0m\n",
      "\u001b[32mScraped package transomics2cytoscape: 1960/2183\u001b[0m\n",
      "\u001b[32mScraped package treekoR: 1961/2183\u001b[0m\n",
      "\u001b[32mScraped package TTMap: 1962/2183\u001b[0m\n",
      "\u001b[32mScraped package dks: 1963/2183\u001b[0m\n",
      "\u001b[32mScraped package epidecodeR: 1964/2183\u001b[0m\n",
      "\u001b[32mScraped package hierinf: 1965/2183\u001b[0m\n",
      "\u001b[32mScraped package iPath: 1966/2183\u001b[0m\n",
      "\u001b[32mScraped package LinkHD: 1967/2183\u001b[0m\n",
      "\u001b[32mScraped package LowMACA: 1968/2183\u001b[0m\n",
      "\u001b[32mScraped package MBAmethyl: 1969/2183\u001b[0m\n",
      "\u001b[32mScraped package metavizr: 1970/2183\u001b[0m\n",
      "\u001b[32mScraped package methylscaper: 1971/2183\u001b[0m\n",
      "\u001b[32mScraped package MPFE: 1972/2183\u001b[0m\n",
      "\u001b[32mScraped package MSstatsLiP: 1973/2183\u001b[0m\n",
      "\u001b[32mScraped package multiSight: 1974/2183\u001b[0m\n",
      "\u001b[32mScraped package netboxr: 1975/2183\u001b[0m\n",
      "\u001b[32mScraped package NTW: 1976/2183\u001b[0m\n",
      "\u001b[32mScraped package oneSENSE: 1977/2183\u001b[0m\n",
      "\u001b[32mScraped package OVESEG: 1978/2183\u001b[0m\n",
      "\u001b[32mScraped package ReactomeGraph4R: 1979/2183\u001b[0m\n",
      "\u001b[32mScraped package RiboCrypt: 1980/2183\u001b[0m\n",
      "\u001b[32mScraped package RIPAT: 1981/2183\u001b[0m\n",
      "\u001b[32mScraped package RNAdecay: 1982/2183\u001b[0m\n",
      "\u001b[32mScraped package scanMiRApp: 1983/2183\u001b[0m\n",
      "\u001b[32mScraped package selectKSigs: 1984/2183\u001b[0m\n",
      "\u001b[32mScraped package semisup: 1985/2183\u001b[0m\n",
      "\u001b[32mScraped package SEPIRA: 1986/2183\u001b[0m\n",
      "\u001b[32mScraped package SMAD: 1987/2183\u001b[0m\n",
      "\u001b[32mScraped package SOMNiBUS: 1988/2183\u001b[0m\n",
      "\u001b[32mScraped package sparsenetgls: 1989/2183\u001b[0m\n",
      "\u001b[32mScraped package spikeLI: 1990/2183\u001b[0m\n",
      "\u001b[32mScraped package SplicingFactory: 1991/2183\u001b[0m\n",
      "\u001b[32mScraped package tenXplore: 1992/2183\u001b[0m\n",
      "\u001b[32mScraped package TFHAZ: 1993/2183\u001b[0m\n",
      "\u001b[32mScraped package traviz: 1994/2183\u001b[0m\n",
      "\u001b[32mScraped package tripr: 1995/2183\u001b[0m\n",
      "\u001b[32mScraped package uncoverappLib: 1996/2183\u001b[0m\n",
      "\u001b[32mScraped package beer: 1997/2183\u001b[0m\n",
      "\u001b[32mScraped package biodbLipidmaps: 1998/2183\u001b[0m\n",
      "\u001b[32mScraped package censcyt: 1999/2183\u001b[0m\n",
      "\u001b[32mScraped package dStruct: 2000/2183\u001b[0m\n",
      "\u001b[32mScraped package fastreeR: 2001/2183\u001b[0m\n",
      "\u001b[32mScraped package levi: 2002/2183\u001b[0m\n",
      "\u001b[32mScraped package m6Aboost: 2003/2183\u001b[0m\n",
      "\u001b[32mScraped package MACSQuantifyR: 2004/2183\u001b[0m\n",
      "\u001b[32mScraped package MQmetrics: 2005/2183\u001b[0m\n",
      "\u001b[32mScraped package MSstatsLOBD: 2006/2183\u001b[0m\n",
      "\u001b[32mScraped package netboost: 2007/2183\u001b[0m\n",
      "\u001b[32mScraped package PhenoGeneRanker: 2008/2183\u001b[0m\n",
      "\u001b[32mScraped package receptLoss: 2009/2183\u001b[0m\n",
      "\u001b[32mScraped package rgoslin: 2010/2183\u001b[0m\n",
      "\u001b[32mScraped package ROCpAI: 2011/2183\u001b[0m\n",
      "\u001b[32mScraped package rsemmed: 2012/2183\u001b[0m\n",
      "\u001b[32mScraped package sarks: 2013/2183\u001b[0m\n",
      "\u001b[32mScraped package scReClassify: 2014/2183\u001b[0m\n",
      "\u001b[32mScraped package scShapes: 2015/2183\u001b[0m\n",
      "\u001b[32mScraped package seqArchR: 2016/2183\u001b[0m\n",
      "\u001b[32mScraped package SICtools: 2017/2183\u001b[0m\n",
      "\u001b[32mScraped package SQUADD: 2018/2183\u001b[0m\n",
      "\u001b[32mScraped package ssPATHS: 2019/2183\u001b[0m\n",
      "\u001b[32mScraped package standR: 2020/2183\u001b[0m\n",
      "\u001b[32mScraped package surfaltr: 2021/2183\u001b[0m\n",
      "\u001b[32mScraped package synapsis: 2022/2183\u001b[0m\n",
      "\u001b[32mScraped package TFARM: 2023/2183\u001b[0m\n",
      "\u001b[32mScraped package txcutr: 2024/2183\u001b[0m\n",
      "\u001b[32mScraped package cogeqc: 2025/2183\u001b[0m\n",
      "\u001b[32mScraped package coMethDMR: 2026/2183\u001b[0m\n",
      "\u001b[32mScraped package conclus: 2027/2183\u001b[0m\n",
      "\u001b[32mScraped package frenchFISH: 2028/2183\u001b[0m\n",
      "\u001b[32mScraped package pairkat: 2029/2183\u001b[0m\n",
      "\u001b[32mScraped package pathVar: 2030/2183\u001b[0m\n",
      "\u001b[32mScraped package pengls: 2031/2183\u001b[0m\n",
      "\u001b[32mScraped package PoDCall: 2032/2183\u001b[0m\n",
      "\u001b[32mScraped package SeqGate: 2033/2183\u001b[0m\n",
      "\u001b[32mScraped package tomoda: 2034/2183\u001b[0m\n",
      "\u001b[32mScraped package VERSO: 2035/2183\u001b[0m\n",
      "\u001b[32mScraped package LinTInd: 2036/2183\u001b[0m\n",
      "\u001b[32mScraped package ModCon: 2037/2183\u001b[0m\n",
      "\u001b[32mScraped package ODER: 2038/2183\u001b[0m\n",
      "\u001b[32mScraped package PanomiR: 2039/2183\u001b[0m\n",
      "\u001b[32mScraped package phosphonormalizer: 2040/2183\u001b[0m\n",
      "\u001b[32mScraped package PoTRA: 2041/2183\u001b[0m\n",
      "\u001b[32mScraped package svaRetro: 2042/2183\u001b[0m\n",
      "\u001b[32mScraped package TraRe: 2043/2183\u001b[0m\n",
      "\u001b[32mScraped package bandle: 2044/2183\u001b[0m\n",
      "\u001b[32mScraped package biodbNcbi: 2045/2183\u001b[0m\n",
      "\u001b[32mScraped package CNVMetrics: 2046/2183\u001b[0m\n",
      "\u001b[32mScraped package comapr: 2047/2183\u001b[0m\n",
      "\u001b[32mScraped package crisprDesign: 2048/2183\u001b[0m\n",
      "\u001b[32mScraped package CyTOFpower: 2049/2183\u001b[0m\n",
      "\u001b[32mScraped package cytoMEM: 2050/2183\u001b[0m\n",
      "\u001b[32mScraped package DifferentialRegulation: 2051/2183\u001b[0m\n",
      "\u001b[32mScraped package eds: 2052/2183\u001b[0m\n",
      "\u001b[32mScraped package epimutacions: 2053/2183\u001b[0m\n",
      "\u001b[32mScraped package extraChIPs: 2054/2183\u001b[0m\n",
      "\u001b[32mScraped package rGenomeTracks: 2055/2183\u001b[0m\n",
      "\u001b[32mScraped package Rnits: 2056/2183\u001b[0m\n",
      "\u001b[32mScraped package sccomp: 2057/2183\u001b[0m\n",
      "\u001b[32mScraped package sSNAPPY: 2058/2183\u001b[0m\n",
      "\u001b[32mScraped package Summix: 2059/2183\u001b[0m\n",
      "\u001b[32mScraped package svaNUMT: 2060/2183\u001b[0m\n",
      "\u001b[32mScraped package TEKRABber: 2061/2183\u001b[0m\n",
      "\u001b[32mScraped package tspair: 2062/2183\u001b[0m\n",
      "\u001b[32mScraped package BOBaFIT: 2063/2183\u001b[0m\n",
      "\u001b[32mScraped package iBMQ: 2064/2183\u001b[0m\n",
      "\u001b[32mScraped package iSEEhex: 2065/2183\u001b[0m\n",
      "\u001b[32mScraped package lineagespot: 2066/2183\u001b[0m\n",
      "\u001b[32mScraped package Motif2Site: 2067/2183\u001b[0m\n",
      "\u001b[32mScraped package MuData: 2068/2183\u001b[0m\n",
      "\u001b[32mScraped package qmtools: 2069/2183\u001b[0m\n",
      "\u001b[32mScraped package CBEA: 2070/2183\u001b[0m\n",
      "\u001b[32mScraped package ctgGEM: 2071/2183\u001b[0m\n",
      "\u001b[32mScraped package GAPGOM: 2072/2183\u001b[0m\n",
      "\u001b[32mScraped package IsoGeneGUI: 2073/2183\u001b[0m\n",
      "\u001b[32mScraped package MACPET: 2074/2183\u001b[0m\n",
      "\u001b[32mScraped package MBECS: 2075/2183\u001b[0m\n",
      "\u001b[32mScraped package MMAPPR2: 2076/2183\u001b[0m\n",
      "\u001b[32mScraped package OGRE: 2077/2183\u001b[0m\n",
      "\u001b[32mScraped package omicsViewer: 2078/2183\u001b[0m\n",
      "\u001b[32mScraped package protGear: 2079/2183\u001b[0m\n",
      "\u001b[32mScraped package qsvaR: 2080/2183\u001b[0m\n",
      "\u001b[32mScraped package RNASeqR: 2081/2183\u001b[0m\n",
      "\u001b[32mScraped package scatterHatch: 2082/2183\u001b[0m\n",
      "\u001b[32mScraped package TrajectoryGeometry: 2083/2183\u001b[0m\n",
      "\u001b[32mScraped package updateObject: 2084/2183\u001b[0m\n",
      "\u001b[32mScraped package xcore: 2085/2183\u001b[0m\n",
      "\u001b[32mScraped package biodbMirbase: 2086/2183\u001b[0m\n",
      "\u001b[32mScraped package GenomicInteractionNodes: 2087/2183\u001b[0m\n",
      "\u001b[32mScraped package HTSeqGenie: 2088/2183\u001b[0m\n",
      "\u001b[32mScraped package inveRsion: 2089/2183\u001b[0m\n",
      "\u001b[32mScraped package pareg: 2090/2183\u001b[0m\n",
      "\u001b[32mScraped package RolDE: 2091/2183\u001b[0m\n",
      "\u001b[32mScraped package syntenet: 2092/2183\u001b[0m\n",
      "\u001b[32mScraped package biodbExpasy: 2093/2183\u001b[0m\n",
      "\u001b[32mScraped package biodbNci: 2094/2183\u001b[0m\n",
      "\u001b[32mScraped package borealis: 2095/2183\u001b[0m\n",
      "\u001b[32mScraped package crisprBwa: 2096/2183\u001b[0m\n",
      "\u001b[32mScraped package DepInfeR: 2097/2183\u001b[0m\n",
      "\u001b[32mScraped package GenProSeq: 2098/2183\u001b[0m\n",
      "\u001b[32mScraped package mbOmic: 2099/2183\u001b[0m\n",
      "\u001b[32mScraped package MobilityTransformR: 2100/2183\u001b[0m\n",
      "\u001b[32mScraped package RGMQL: 2101/2183\u001b[0m\n",
      "\u001b[32mScraped package SpatialFeatureExperiment: 2102/2183\u001b[0m\n",
      "\u001b[32mScraped package TREG: 2103/2183\u001b[0m\n",
      "\u001b[32mScraped package single: 2104/2183\u001b[0m\n",
      "\u001b[32mScraped package terraTCGAdata: 2105/2183\u001b[0m\n",
      "\u001b[32mScraped package Travel: 2106/2183\u001b[0m\n",
      "\u001b[32mScraped package HilbertVisGUI: 2107/2183\u001b[0m\n",
      "\u001b[32mScraped package Macarron: 2108/2183\u001b[0m\n",
      "\u001b[32mScraped package PrecisionTrialDrawer: 2109/2183\u001b[0m\n",
      "\u001b[32mScraped package pulsedSilac: 2110/2183\u001b[0m\n",
      "\u001b[32mScraped package RAREsim: 2111/2183\u001b[0m\n",
      "\u001b[32mScraped package tomoseqr: 2112/2183\u001b[0m\n",
      "\u001b[32mScraped package SpotClean: 2113/2183\u001b[0m\n",
      "\u001b[32mScraped package iteremoval: 2114/2183\u001b[0m\n",
      "\u001b[32mScraped package coexnet: 2115/2183\u001b[0m\n",
      "\u001b[32mScraped package crisprVerse: 2116/2183\u001b[0m\n",
      "\u001b[32mScraped package crisprViz: 2117/2183\u001b[0m\n",
      "\u001b[32mScraped package rifi: 2118/2183\u001b[0m\n",
      "\u001b[32mScraped package SPIAT: 2119/2183\u001b[0m\n",
      "\u001b[32mScraped package CTSV: 2120/2183\u001b[0m\n",
      "\u001b[32mScraped package demuxmix: 2121/2183\u001b[0m\n",
      "\u001b[32mScraped package phemd: 2122/2183\u001b[0m\n",
      "\u001b[32mScraped package MsExperiment: 2123/2183\u001b[0m\n",
      "\u001b[32mScraped package MSstatsShiny: 2124/2183\u001b[0m\n",
      "\u001b[32mScraped package SpliceWiz: 2125/2183\u001b[0m\n",
      "\u001b[32mScraped package katdetectr: 2126/2183\u001b[0m\n",
      "\u001b[32mScraped package MethCP: 2127/2183\u001b[0m\n",
      "\u001b[32mScraped package ATACseqTFEA: 2128/2183\u001b[0m\n",
      "\u001b[32mScraped package cardelino: 2129/2183\u001b[0m\n",
      "\u001b[32mScraped package mslp: 2130/2183\u001b[0m\n",
      "\u001b[32mScraped package proteasy: 2131/2183\u001b[0m\n",
      "\u001b[32mScraped package Voyager: 2132/2183\u001b[0m\n",
      "\u001b[32mScraped package EasyCellType: 2133/2183\u001b[0m\n",
      "\u001b[32mScraped package factR: 2134/2183\u001b[0m\n",
      "\u001b[32mScraped package gemma.R: 2135/2183\u001b[0m\n",
      "\u001b[32mScraped package GenomAutomorphism: 2136/2183\u001b[0m\n",
      "\u001b[32mScraped package ggtreeDendro: 2137/2183\u001b[0m\n",
      "\u001b[32mScraped package HiCDOC: 2138/2183\u001b[0m\n",
      "\u001b[32mScraped package scDDboost: 2139/2183\u001b[0m\n",
      "\u001b[32mScraped package ScreenR: 2140/2183\u001b[0m\n",
      "\u001b[32mScraped package simpleSeg: 2141/2183\u001b[0m\n",
      "\u001b[32mScraped package BASiCStan: 2142/2183\u001b[0m\n",
      "\u001b[32mScraped package FuseSOM: 2143/2183\u001b[0m\n",
      "\u001b[32mScraped package goSorensen: 2144/2183\u001b[0m\n",
      "\u001b[32mScraped package HiContacts: 2145/2183\u001b[0m\n",
      "\u001b[32mScraped package PanViz: 2146/2183\u001b[0m\n",
      "\u001b[32mScraped package RedisParam: 2147/2183\u001b[0m\n",
      "\u001b[32mScraped package spaSim: 2148/2183\u001b[0m\n",
      "\u001b[32mScraped package SUITOR: 2149/2183\u001b[0m\n",
      "\u001b[32mScraped package consICA: 2150/2183\u001b[0m\n",
      "\u001b[32mScraped package DNAfusion: 2151/2183\u001b[0m\n",
      "\u001b[32mScraped package epistasisGA: 2152/2183\u001b[0m\n",
      "\u001b[32mScraped package oncoscanR: 2153/2183\u001b[0m\n",
      "\u001b[32mScraped package phenomis: 2154/2183\u001b[0m\n",
      "\u001b[32mScraped package RgnTX: 2155/2183\u001b[0m\n",
      "\u001b[32mScraped package scMET: 2156/2183\u001b[0m\n",
      "\u001b[32mScraped package SimBu: 2157/2183\u001b[0m\n",
      "\u001b[32mScraped package zenith: 2158/2183\u001b[0m\n",
      "\u001b[32mScraped package BiocFHIR: 2159/2183\u001b[0m\n",
      "\u001b[32mScraped package BioNAR: 2160/2183\u001b[0m\n",
      "\u001b[32mScraped package EpiMix: 2161/2183\u001b[0m\n",
      "\u001b[32mScraped package iSEEhub: 2162/2183\u001b[0m\n",
      "\u001b[32mScraped package ISLET: 2163/2183\u001b[0m\n",
      "\u001b[32mScraped package magrene: 2164/2183\u001b[0m\n",
      "\u001b[32mScraped package NetActivity: 2165/2183\u001b[0m\n",
      "\u001b[32mScraped package octad: 2166/2183\u001b[0m\n",
      "\u001b[32mScraped package regioneReloaded: 2167/2183\u001b[0m\n",
      "\u001b[32mScraped package scBubbletree: 2168/2183\u001b[0m\n",
      "\u001b[32mScraped package signifinder: 2169/2183\u001b[0m\n",
      "\u001b[32mScraped package TENxIO: 2170/2183\u001b[0m\n",
      "\u001b[32mScraped package VDJdive: 2171/2183\u001b[0m\n",
      "\u001b[32mScraped package vsclust: 2172/2183\u001b[0m\n",
      "\u001b[32mScraped package ATACCoGAPS: 2173/2183\u001b[0m\n",
      "\u001b[32mScraped package ccImpute: 2174/2183\u001b[0m\n",
      "\u001b[32mScraped package MetaPhOR: 2175/2183\u001b[0m\n",
      "\u001b[32mScraped package omada: 2176/2183\u001b[0m\n",
      "\u001b[32mScraped package RESOLVE: 2177/2183\u001b[0m\n",
      "\u001b[32mScraped package scifer: 2178/2183\u001b[0m\n",
      "\u001b[32mScraped package Statial: 2179/2183\u001b[0m\n",
      "\u001b[32mScraped package stJoincount: 2180/2183\u001b[0m\n",
      "\u001b[32mScraped package CircSeqAlignTk: 2181/2183\u001b[0m\n",
      "\u001b[32mScraped package metabinR: 2182/2183\u001b[0m\n",
      "\u001b[32mScraped package XCIR: 2183/2183\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "# Scraping the Bioconductor website to get the list of packages\n",
    "bioc_package_list = scraper.build_list(bioc_packages)\n",
    "\n",
    "file_name = \"bioconductor_package_object_list.pkl\"\n",
    "# Delete the file if it already exists\n",
    "if os.path.exists(\"data\" + os.sep + file_name):\n",
    "    os.remove(\"data\" + os.sep + file_name)\n",
    "\n",
    "# Save the list of packages in a pkl file\n",
    "with open(\"data\" + os.sep + file_name, \"wb\") as f:\n",
    "    pickle.dump(bioc_package_list, f)"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "**<span style=\"color:yellow\">Visualize some data</span>**"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Name: BiocGenerics\n",
      "Description:\n",
      "The package defines many S4 generic functions used in Bioconductor.\n",
      "Version: 0.44.0\n",
      "Publication date: None\n",
      "Mantainer: HervÃ© PagÃ¨s <hpages.on.github@gmail.com>\n",
      "Authors:\n",
      "Author: The Bioconductor Dev Team\n",
      "Dependencies:\n",
      "  - R (>= 4.0.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - utils, type: depends\n",
      "  - graphics, type: depends\n",
      "  - stats, type: depends\n",
      "  - methods, type: imports\n",
      "  - utils, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/BiocGenerics.html\n",
      "\n",
      "Name: S4Vectors\n",
      "Description:\n",
      "The S4Vectors package defines the Vector and List virtual classes and a set of generic functions that extend the semantic of ordinary vectors and lists in R. Package developers can easily implement vector-like or list-like objects as concrete subclasses of Vector or List. In addition, a few low-level concrete subclasses of general interest (e.g. DataFrame, Rle, Factor, and Hits) are implemented in the S4Vectors package itself (many more are implemented in the IRanges package and in other Bioconductor infrastructure packages).\n",
      "Version: 0.36.1\n",
      "Publication date: None\n",
      "Mantainer: HervÃ© PagÃ¨s <hpages.on.github@gmail.com>\n",
      "Authors:\n",
      "Author: HervÃ© PagÃ¨s [aut, cre], Michael Lawrence [aut], Patrick Aboyoun [aut], Aaron Lun [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 4.0.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - utils, type: depends\n",
      "  - stats, type: depends\n",
      "  - stats4, type: depends\n",
      "  - BiocGenerics, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/S4Vectors.html\n",
      "\n",
      "Name: BiocVersion\n",
      "Description:\n",
      "This package provides repository information for the appropriate version of Bioconductor.\n",
      "Version: 3.16.0\n",
      "Publication date: None\n",
      "Mantainer: Bioconductor Package Maintainer <maintainer@bioconductor.org>\n",
      "Authors:\n",
      "Author: Martin Morgan [aut], Marcel Ramos [ctb], Bioconductor Package Maintainer [ctb, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.2.0) type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/BiocVersion.html\n",
      "\n",
      "Name: GenomeInfoDb\n",
      "Description:\n",
      "Contains data and functions that define and allow translation between different chromosome sequence naming conventions (e.g., \"chr1\" versus \"1\"), including a function that attempts to place sequence names in their natural, rather than lexicographic, order.\n",
      "Version: 1.34.9\n",
      "Publication date: None\n",
      "Mantainer: HervÃ© PagÃ¨s <hpages.on.github@gmail.com>\n",
      "Authors:\n",
      "Author: Sonali Arora [aut], Martin Morgan [aut], Marc Carlson [aut], HervÃ© PagÃ¨s [aut, cre], Prisca Chidimma Maduka [ctb], Atuhurira Kirabo Kakopo [ctb], Emmanuel Chigozie Elendu [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 4.0.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - BiocGenerics, type: depends\n",
      "  - S4Vectors, type: depends\n",
      "  - IRanges, type: depends\n",
      "  - stats, type: imports\n",
      "  - stats4, type: imports\n",
      "  - utils, type: imports\n",
      "  - RCurl, type: imports\n",
      "  - GenomeInfoDbData, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/GenomeInfoDb.html\n",
      "\n",
      "Name: IRanges\n",
      "Description:\n",
      "Provides efficient low-level and highly reusable S4 classes for storing, manipulating and aggregating over annotated ranges of integers. Implements an algebra of range operations, including efficient algorithms for finding overlaps and nearest neighbors. Defines efficient list-like classes for storing, transforming and aggregating large grouped data, i.e., collections of atomic vectors and DataFrames.\n",
      "Version: 2.32.0\n",
      "Publication date: None\n",
      "Mantainer: HervÃ© PagÃ¨s <hpages.on.github@gmail.com>\n",
      "Authors:\n",
      "Author: HervÃ© PagÃ¨s [aut, cre], Patrick Aboyoun [aut], Michael Lawrence [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.0.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - utils, type: depends\n",
      "  - stats, type: depends\n",
      "  - BiocGenerics, type: depends\n",
      "  - S4Vectors, type: depends\n",
      "  - stats4, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/IRanges.html\n",
      "\n",
      "Name: Biobase\n",
      "Description:\n",
      "Functions that are needed by many other packages or which replace R functions.\n",
      "Version: 2.58.0\n",
      "Publication date: None\n",
      "Mantainer: Bioconductor Package Maintainer <maintainer@bioconductor.org>\n",
      "Authors:\n",
      "Author: R. Gentleman [aut], V. Carey [aut], M. Morgan [aut], S. Falcon [aut], Haleema Khan [ctb] ('esApply' and 'BiobaseDevelopment' vignette translation from Sweave to Rmarkdown / HTML), Bioconductor Package Maintainer [cre]\n",
      "Dependencies:\n",
      "  - R (>= 2.10) type: depends\n",
      "  - BiocGenerics, type: depends\n",
      "  - utils, type: depends\n",
      "  - methods, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/Biobase.html\n",
      "\n",
      "Name: zlibbioc\n",
      "Description:\n",
      "This package uses the source code of zlib-1.2.5 to create libraries for systems that do not have these available via other means (most Linux and Mac users should have system-level access to zlib, and no direct need for this package). See the vignette for instructions on use.\n",
      "Version: 1.44.0\n",
      "Publication date: None\n",
      "Mantainer: Bioconductor Package Maintainer <maintainer@bioconductor.org>\n",
      "Authors:\n",
      "Author: Martin Morgan\n",
      "Dependencies:\n",
      "None\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0 + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/zlibbioc.html\n",
      "\n",
      "Name: XVector\n",
      "Description:\n",
      "Provides memory efficient S4 classes for storing sequences \"externally\" (e.g. behind an R external pointer, or on disk).\n",
      "Version: 0.38.0\n",
      "Publication date: None\n",
      "Mantainer: HervÃ© PagÃ¨s <hpages.on.github@gmail.com>\n",
      "Authors:\n",
      "Author: HervÃ© PagÃ¨s and Patrick Aboyoun\n",
      "Dependencies:\n",
      "  - R (>= 4.0.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - BiocGenerics, type: depends\n",
      "  - S4Vectors, type: depends\n",
      "  - IRanges, type: depends\n",
      "  - methods, type: imports\n",
      "  - utils, type: imports\n",
      "  - tools, type: imports\n",
      "  - zlibbioc, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - IRanges, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/XVector.html\n",
      "\n",
      "Name: Biostrings\n",
      "Description:\n",
      "Memory efficient string containers, string matching algorithms, and other utilities, for fast manipulation of large biological sequences or sets of sequences.\n",
      "Version: 2.66.0\n",
      "Publication date: None\n",
      "Mantainer: H. PagÃ¨s <hpages.on.github@gmail.com>\n",
      "Authors:\n",
      "Author: H. PagÃ¨s, P. Aboyoun, R. Gentleman, and S. DebRoy\n",
      "Dependencies:\n",
      "  - R (>= 4.0.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - BiocGenerics, type: depends\n",
      "  - S4Vectors, type: depends\n",
      "  - IRanges, type: depends\n",
      "  - XVector, type: depends\n",
      "  - GenomeInfoDb, type: depends\n",
      "  - methods, type: imports\n",
      "  - utils, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - crayon, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/Biostrings.html\n",
      "\n",
      "Name: BiocParallel\n",
      "Description:\n",
      "This package provides modified versions and novel implementation of functions for parallel evaluation, tailored to use with Bioconductor objects.\n",
      "Version: 1.32.5\n",
      "Publication date: None\n",
      "Mantainer: Martin Morgan <mtmorgan.bioc@gmail.com>\n",
      "Authors:\n",
      "Author: Martin Morgan [aut, cre], Jiefei Wang [aut], Valerie Obenchain [aut], Michel Lang [aut], Ryan Thompson [aut], Nitesh Turaga [aut], Aaron Lun [ctb], Henrik Bengtsson [ctb], Madelyn Carlson [ctb] (Translated 'Random Numbers' vignette from Sweave to RMarkdown / HTML.), Phylis Atieno [ctb] (Translated 'Introduction to BiocParallel' vignette from Sweave to Rmarkdown / HTML.)\n",
      "Dependencies:\n",
      "  - methods, type: depends\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - futile.logger, type: imports\n",
      "  - parallel, type: imports\n",
      "  - snow, type: imports\n",
      "  - codetools, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2 | GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/BiocParallel.html\n",
      "\n",
      "Name: GenomicRanges\n",
      "Description:\n",
      "The ability to efficiently represent and manipulate genomic annotations and alignments is playing a central role when it comes to analyzing high-throughput sequencing data (a.k.a. NGS data). The GenomicRanges package defines general purpose containers for storing and manipulating genomic intervals and variables defined along a genome. More specialized containers for representing and manipulating short alignments against a reference genome, or a matrix-like summarization of an experiment, are defined in the GenomicAlignments and SummarizedExperiment packages, respectively. Both packages build on top of the GenomicRanges infrastructure.\n",
      "Version: 1.50.2\n",
      "Publication date: None\n",
      "Mantainer: HervÃ© PagÃ¨s <hpages.on.github@gmail.com>\n",
      "Authors:\n",
      "Author: Patrick Aboyoun [aut], HervÃ© PagÃ¨s [aut, cre], Michael Lawrence [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.0.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - stats4, type: depends\n",
      "  - BiocGenerics, type: depends\n",
      "  - S4Vectors, type: depends\n",
      "  - IRanges, type: depends\n",
      "  - GenomeInfoDb, type: depends\n",
      "  - utils, type: imports\n",
      "  - stats, type: imports\n",
      "  - XVector, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/GenomicRanges.html\n",
      "\n",
      "Name: DelayedArray\n",
      "Description:\n",
      "Wrapping an array-like object (typically an on-disk object) in a DelayedArray object allows one to perform common array operations on it without loading the object in memory. In order to reduce memory usage and optimize performance, operations on the object are either delayed or executed using a block processing mechanism. Note that this also works on in-memory array-like objects like DataFrame objects (typically with Rle columns), Matrix objects, ordinary arrays and, data frames.\n",
      "Version: 0.24.0\n",
      "Publication date: None\n",
      "Mantainer: HervÃ© PagÃ¨s <hpages.on.github@gmail.com>\n",
      "Authors:\n",
      "Author: HervÃ© PagÃ¨s <hpages.on.github at gmail.com>, with contributions from Peter Hickey <peter.hickey at gmail.com> and Aaron Lun <infinite.monkeys.with.keyboards at gmail.com>\n",
      "Dependencies:\n",
      "  - R (>= 4.0.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - stats4, type: depends\n",
      "  - Matrix, type: depends\n",
      "  - BiocGenerics, type: depends\n",
      "  - MatrixGenerics, type: depends\n",
      "  - S4Vectors, type: depends\n",
      "  - IRanges, type: depends\n",
      "  - stats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/DelayedArray.html\n",
      "\n",
      "Name: SummarizedExperiment\n",
      "Description:\n",
      "The SummarizedExperiment container contains one or more assays, each represented by a matrix-like object of numeric or other mode. The rows typically represent genomic ranges of interest and the columns represent samples.\n",
      "Version: 1.28.0\n",
      "Publication date: None\n",
      "Mantainer: HervÃ© PagÃ¨s <hpages.on.github@gmail.com>\n",
      "Authors:\n",
      "Author: Martin Morgan [aut], Valerie Obenchain [aut], Jim Hester [aut], HervÃ© PagÃ¨s [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.0.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - MatrixGenerics, type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - Biobase, type: depends\n",
      "  - utils, type: imports\n",
      "  - stats, type: imports\n",
      "  - tools, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - DelayedArray, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/SummarizedExperiment.html\n",
      "\n",
      "Name: MatrixGenerics\n",
      "Description:\n",
      "S4 generic functions modeled after the 'matrixStats' API for alternative matrix implementations. Packages with alternative matrix implementation can depend on this package and implement the generic functions that are defined here for a useful set of row and column summary statistics. Other package developers can import this package and handle a different matrix implementations without worrying about incompatibilities.\n",
      "Version: 1.10.0\n",
      "Publication date: None\n",
      "Mantainer: Peter Hickey <peter.hickey@gmail.com>\n",
      "Authors:\n",
      "Author: Constantin Ahlmann-Eltze [aut] , Peter Hickey [aut, cre] , HervÃ© PagÃ¨s [aut]\n",
      "Dependencies:\n",
      "  - matrixStats (>= 0.60.1) type: depends\n",
      "  - methods, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/MatrixGenerics.html\n",
      "\n",
      "Name: limma\n",
      "Description:\n",
      "Data analysis, linear models and differential expression for microarray data.\n",
      "Version: 3.54.1\n",
      "Publication date: None\n",
      "Mantainer: Gordon Smyth <smyth@wehi.edu.au>\n",
      "Authors:\n",
      "Author: Gordon Smyth [cre,aut], Yifang Hu [ctb], Matthew Ritchie [ctb], Jeremy Silver [ctb], James Wettenhall [ctb], Davis McCarthy [ctb], Di Wu [ctb], Wei Shi [ctb], Belinda Phipson [ctb], Aaron Lun [ctb], Natalie Thorne [ctb], Alicia Oshlack [ctb], Carolyn de Graaf [ctb], Yunshun Chen [ctb], Mette Langaas [ctb], Egil Ferkingstad [ctb], Marcus Davy [ctb], Francois Pepin [ctb], Dongseok Choi [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 3.6.0) type: depends\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - methods, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>=2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/limma.html\n",
      "\n",
      "Name: AnnotationDbi\n",
      "Description:\n",
      "Implements a user-friendly interface for querying SQLite-based annotation data packages.\n",
      "Version: 1.60.0\n",
      "Publication date: None\n",
      "Mantainer: Bioconductor Package Maintainer <maintainer@bioconductor.org>\n",
      "Authors:\n",
      "Author: HervÃ© PagÃ¨s, Marc Carlson, Seth Falcon, Nianhua Li\n",
      "Dependencies:\n",
      "  - R (>= 2.7.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - utils, type: depends\n",
      "  - stats4, type: depends\n",
      "  - BiocGenerics, type: depends\n",
      "  - Biobase, type: depends\n",
      "  - IRanges, type: depends\n",
      "  - DBI, type: imports\n",
      "  - RSQLite, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - stats, type: imports\n",
      "  - KEGGREST, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/AnnotationDbi.html\n",
      "\n",
      "Name: KEGGREST\n",
      "Description:\n",
      "A package that provides a client interface to the Kyoto Encyclopedia of Genes and Genomes (KEGG) REST server. Based on KEGGSOAP by J. Zhang, R. Gentleman, and Marc Carlson, and KEGG (python package) by Aurelien Mazurie.\n",
      "Version: 1.38.0\n",
      "Publication date: None\n",
      "Mantainer: Bioconductor Package Maintainer <maintainer@bioconductor.org>\n",
      "Authors:\n",
      "Author: Dan Tenenbaum [aut], Jeremy Volkening [ctb], Bioconductor Package Maintainer [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - methods, type: imports\n",
      "  - httr, type: imports\n",
      "  - png, type: imports\n",
      "  - Biostrings, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/KEGGREST.html\n",
      "\n",
      "Name: annotate\n",
      "Description:\n",
      "Using R enviroments for annotation.\n",
      "Version: 1.76.0\n",
      "Publication date: None\n",
      "Mantainer: Bioconductor Package Maintainer <maintainer@bioconductor.org>\n",
      "Authors:\n",
      "Author: R. Gentleman\n",
      "Dependencies:\n",
      "  - R (>= 2.10) type: depends\n",
      "  - AnnotationDbi, type: depends\n",
      "  - XML, type: depends\n",
      "  - Biobase, type: imports\n",
      "  - DBI, type: imports\n",
      "  - xtable, type: imports\n",
      "  - graphics, type: imports\n",
      "  - utils, type: imports\n",
      "  - stats, type: imports\n",
      "  - methods, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - httr, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/annotate.html\n",
      "\n",
      "Name: BiocFileCache\n",
      "Description:\n",
      "This package creates a persistent on-disk cache of files that the user can add, update, and retrieve. It is useful for managing resources (such as custom Txdb objects) that are costly or difficult to create, web resources, and data files used across sessions.\n",
      "Version: 2.6.1\n",
      "Publication date: None\n",
      "Mantainer: Lori Shepherd <lori.shepherd@roswellpark.org>\n",
      "Authors:\n",
      "Author: Lori Shepherd [aut, cre], Martin Morgan [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.4.0) type: depends\n",
      "  - dbplyr (>= 1.0.0) type: depends\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - RSQLite, type: imports\n",
      "  - DBI, type: imports\n",
      "  - rappdirs, type: imports\n",
      "  - filelock, type: imports\n",
      "  - curl, type: imports\n",
      "  - httr, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/BiocFileCache.html\n",
      "\n",
      "Name: Rhtslib\n",
      "Description:\n",
      "This package provides version 1.15.1 of the 'HTSlib' C library for high-throughput sequence analysis. The package is primarily useful to developers of other R packages who wish to make use of HTSlib. Motivation and instructions for use of this package are in the vignette, vignette(package=\"Rhtslib\", \"Rhtslib\").\n",
      "Version: 2.0.0\n",
      "Publication date: None\n",
      "Mantainer: HervÃ© PagÃ¨s <hpages.on.github@gmail.com>\n",
      "Authors:\n",
      "Author: Nathaniel Hayden [led, aut], Martin Morgan [aut], HervÃ© PagÃ¨s [aut, cre]\n",
      "Dependencies:\n",
      "  - zlibbioc, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/Rhtslib.html\n",
      "\n",
      "Name: Rsamtools\n",
      "Description:\n",
      "This package provides an interface to the 'samtools', 'bcftools', and 'tabix' utilities for manipulating SAM (Sequence Alignment / Map), FASTA, binary variant call (BCF) and compressed indexed tab-delimited (tabix) files.\n",
      "Version: 2.14.0\n",
      "Publication date: None\n",
      "Mantainer: Bioconductor Package Maintainer <maintainer@bioconductor.org>\n",
      "Authors:\n",
      "Author: Martin Morgan, HervÃ© PagÃ¨s, Valerie Obenchain, Nathaniel Hayden\n",
      "Dependencies:\n",
      "  - methods, type: depends\n",
      "  - GenomeInfoDb, type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - Biostrings, type: depends\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - utils, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - XVector, type: imports\n",
      "  - zlibbioc, type: imports\n",
      "  - bitops, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - stats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0 | file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/Rsamtools.html\n",
      "\n",
      "Name: genefilter\n",
      "Description:\n",
      "Some basic functions for filtering genes.\n",
      "Version: 1.80.3\n",
      "Publication date: None\n",
      "Mantainer: Bioconductor Package Maintainer <maintainer@bioconductor.org>\n",
      "Authors:\n",
      "Author: Robert Gentleman, Vincent J. Carey, Wolfgang Huber, Florian Hahne\n",
      "Dependencies:\n",
      "  - BiocGenerics, type: imports\n",
      "  - AnnotationDbi, type: imports\n",
      "  - annotate, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - graphics, type: imports\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - survival, type: imports\n",
      "  - grDevices, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/genefilter.html\n",
      "\n",
      "Name: edgeR\n",
      "Description:\n",
      "Differential expression analysis of RNA-seq expression profiles with biological replication. Implements a range of statistical methodology based on the negative binomial distributions, including empirical Bayes estimation, exact tests, generalized linear models and quasi-likelihood tests. As well as RNA-seq, it be applied to differential signal analysis of other types of genomic data that produce read counts, including ChIP-seq, ATAC-seq, Bisulfite-seq, SAGE and CAGE.\n",
      "Version: 3.40.2\n",
      "Publication date: None\n",
      "Mantainer: Yunshun Chen <yuchen@wehi.edu.au>, Gordon Smyth <smyth@wehi.edu.au>, Aaron Lun <infinite.monkeys.with.keyboards@gmail.com>, Mark Robinson <mark.robinson@imls.uzh.ch>\n",
      "Authors:\n",
      "Author: Yunshun Chen, Aaron TL Lun, Davis J McCarthy, Matthew E Ritchie, Belinda Phipson, Yifang Hu, Xiaobei Zhou, Mark D Robinson, Gordon K Smyth\n",
      "Dependencies:\n",
      "  - R (>= 3.6.0) type: depends\n",
      "  - limma, type: depends\n",
      "  - methods, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - locfit, type: imports\n",
      "  - Rcpp, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>=2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/edgeR.html\n",
      "\n",
      "Name: GenomicAlignments\n",
      "Description:\n",
      "Provides efficient containers for storing and manipulating short genomic alignments (typically obtained by aligning short reads to a reference genome). This includes read counting, computing the coverage, junction detection, and working with the nucleotide content of the alignments.\n",
      "Version: 1.34.0\n",
      "Publication date: None\n",
      "Mantainer: HervÃ© PagÃ¨s <hpages.on.github@gmail.com>\n",
      "Authors:\n",
      "Author: HervÃ© PagÃ¨s [aut, cre], Valerie Obenchain [aut], Martin Morgan [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.0.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - BiocGenerics, type: depends\n",
      "  - S4Vectors, type: depends\n",
      "  - IRanges, type: depends\n",
      "  - GenomeInfoDb, type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - Biostrings, type: depends\n",
      "  - Rsamtools, type: depends\n",
      "  - methods, type: imports\n",
      "  - utils, type: imports\n",
      "  - stats, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/GenomicAlignments.html\n",
      "\n",
      "Name: biomaRt\n",
      "Description:\n",
      "In recent years a wealth of biological data has become available in public data repositories. Easy access to these valuable data resources and firm integration with data analysis is needed for comprehensive bioinformatics data analysis. biomaRt provides an interface to a growing collection of databases implementing the BioMart software suite (). The package enables retrieval of large amounts of data in a uniform way without the need to know the underlying database schemas or write complex SQL queries. The most prominent examples of BioMart databases are maintain by Ensembl, which provides biomaRt users direct access to a diverse set of data and enables a wide range of powerful online queries from gene annotation to database mining.\n",
      "Version: 2.54.0\n",
      "Publication date: None\n",
      "Mantainer: Mike Smith <grimbough@gmail.com>\n",
      "Authors:\n",
      "Author: Steffen Durinck [aut], Wolfgang Huber [aut], Sean Davis [ctb], Francois Pepin [ctb], Vince S Buffalo [ctb], Mike Smith [ctb, cre]\n",
      "Dependencies:\n",
      "  - methods, type: depends\n",
      "  - utils, type: imports\n",
      "  - XML (>= 3.99-0.7) type: imports\n",
      "  - AnnotationDbi, type: imports\n",
      "  - progress, type: imports\n",
      "  - stringr, type: imports\n",
      "  - httr, type: imports\n",
      "  - digest, type: imports\n",
      "  - BiocFileCache, type: imports\n",
      "  - rappdirs, type: imports\n",
      "  - xml2, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/biomaRt.html\n",
      "\n",
      "Name: rtracklayer\n",
      "Description:\n",
      "Extensible framework for interacting with multiple genome browsers (currently UCSC built-in) and manipulating annotation tracks in various formats (currently GFF, BED, bedGraph, BED15, WIG, BigWig and 2bit built-in). The user may export/import tracks to/from the supported browsers, as well as query and modify the browser state, such as the current viewport.\n",
      "Version: 1.58.0\n",
      "Publication date: None\n",
      "Mantainer: Michael Lawrence <michafla@gene.com>\n",
      "Authors:\n",
      "Author: Michael Lawrence, Vince Carey, Robert Gentleman\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - XML (>= 1.98-0) type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - XVector, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - zlibbioc, type: imports\n",
      "  - RCurl (>= 1.4-2) type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - GenomicAlignments, type: imports\n",
      "  - BiocIO, type: imports\n",
      "  - tools, type: imports\n",
      "  - restfulr (>= 0.0.13) type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0 + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/rtracklayer.html\n",
      "\n",
      "Name: Rhdf5lib\n",
      "Description:\n",
      "Provides C and C++ hdf5 libraries.\n",
      "Version: 1.20.0\n",
      "Publication date: None\n",
      "Mantainer: Mike Smith <grimbough@gmail.com>\n",
      "Authors:\n",
      "Author: Mike Smith [ctb, cre] , The HDF Group [cph]\n",
      "Dependencies:\n",
      "  - R (>= 4.0.0) type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/Rhdf5lib.html\n",
      "\n",
      "Name: graph\n",
      "Description:\n",
      "A package that implements some simple graph handling capabilities.\n",
      "Version: 1.76.0\n",
      "Publication date: None\n",
      "Mantainer: Bioconductor Package Maintainer <maintainer@bioconductor.org>\n",
      "Authors:\n",
      "Author: R Gentleman [aut], Elizabeth Whalen [aut], W Huber [aut], S Falcon [aut], Halimat C. Atanda [ctb] (Converted 'GraphClass' vignette from Sweave to RMarkdown / HTML.), Bioconductor Package Maintainer [cre]\n",
      "Dependencies:\n",
      "  - R (>= 2.10) type: depends\n",
      "  - methods, type: depends\n",
      "  - BiocGenerics, type: depends\n",
      "  - stats, type: imports\n",
      "  - stats4, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/graph.html\n",
      "\n",
      "Name: DESeq2\n",
      "Description:\n",
      "Estimate variance-mean dependence in count data from high-throughput sequencing assays and test for differential expression based on a model using the negative binomial distribution.\n",
      "Version: 1.38.3\n",
      "Publication date: None\n",
      "Mantainer: Michael Love <michaelisaiahlove@gmail.com>\n",
      "Authors:\n",
      "Author: Michael Love [aut, cre], Constantin Ahlmann-Eltze [ctb], Kwame Forbes [ctb], Simon Anders [aut, ctb], Wolfgang Huber [aut, ctb], RADIANT EU FP7 [fnd], NIH NHGRI [fnd], CZI [fnd]\n",
      "Dependencies:\n",
      "  - S4Vectors, type: depends\n",
      "  - IRanges, type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - BiocGenerics, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - methods, type: imports\n",
      "  - stats4, type: imports\n",
      "  - locfit, type: imports\n",
      "  - geneplotter, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - Rcpp (>= 0.11.0) type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL (>= 3)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/DESeq2.html\n",
      "\n",
      "Name: geneplotter\n",
      "Description:\n",
      "Functions for plotting genomic data\n",
      "Version: 1.76.0\n",
      "Publication date: None\n",
      "Mantainer: Bioconductor Package Maintainer <maintainer@bioconductor.org>\n",
      "Authors:\n",
      "Author: R. Gentleman, Biocore\n",
      "Dependencies:\n",
      "  - R (>= 2.10) type: depends\n",
      "  - methods, type: depends\n",
      "  - Biobase, type: depends\n",
      "  - BiocGenerics, type: depends\n",
      "  - lattice, type: depends\n",
      "  - annotate, type: depends\n",
      "  - AnnotationDbi, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - grid, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/geneplotter.html\n",
      "\n",
      "Name: GenomicFeatures\n",
      "Description:\n",
      "A set of tools and methods for making and manipulating transcript centric annotations. With these tools the user can easily download the genomic locations of the transcripts, exons and cds of a given organism, from either the UCSC Genome Browser or a BioMart database (more sources will be supported in the future). This information is then stored in a local database that keeps track of the relationship between transcripts, exons, cds and genes. Flexible methods are provided for extracting the desired features in a convenient format.\n",
      "Version: 1.50.4\n",
      "Publication date: None\n",
      "Mantainer: H. PagÃ¨s <hpages.on.github@gmail.com>\n",
      "Authors:\n",
      "Author: M. Carlson [aut], H. PagÃ¨s [aut, cre], P. Aboyoun [aut], S. Falcon [aut], M. Morgan [aut], D. Sarkar [aut], M. Lawrence [aut], V. Obenchain [aut], S. Arora [ctb], J. MacDonald [ctb], M. Ramos [ctb], S. Saini [ctb], P. Shannon [ctb], L. Shepherd [ctb], D. Tenenbaum [ctb], D. Van Twisk [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - BiocGenerics, type: depends\n",
      "  - S4Vectors, type: depends\n",
      "  - IRanges, type: depends\n",
      "  - GenomeInfoDb, type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - AnnotationDbi, type: depends\n",
      "  - methods, type: imports\n",
      "  - utils, type: imports\n",
      "  - stats, type: imports\n",
      "  - tools, type: imports\n",
      "  - DBI, type: imports\n",
      "  - RSQLite (>= 2.0) type: imports\n",
      "  - RCurl, type: imports\n",
      "  - XVector, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - BiocIO, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - biomaRt, type: imports\n",
      "  - Biobase, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/GenomicFeatures.html\n",
      "\n",
      "Name: rhdf5\n",
      "Description:\n",
      "This package provides an interface between HDF5 and R. HDF5's main features are the ability to store and access very large and/or complex datasets and a wide variety of metadata on mass storage (disk) through a completely portable file format. The rhdf5 package is thus suited for the exchange of large and/or complex datasets between R and other software package, and for letting R applications work on datasets that are larger than the available RAM.\n",
      "Version: 2.42.0\n",
      "Publication date: None\n",
      "Mantainer: Mike Smith <mike.smith@embl.de>\n",
      "Authors:\n",
      "Author: Bernd Fischer [aut], Mike Smith [aut, cre] , Gregoire Pau [aut], Martin Morgan [ctb], Daniel van Twisk [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 4.0.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - Rhdf5lib, type: imports\n",
      "  - rhdf5filters, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/rhdf5.html\n",
      "\n",
      "Name: ggtree\n",
      "Description:\n",
      "'ggtree' extends the 'ggplot2' plotting system which implemented the grammar of graphics. 'ggtree' is designed for visualization and annotation of phylogenetic trees and other tree-like structures with their annotation data.\n",
      "Version: 3.6.2\n",
      "Publication date: None\n",
      "Mantainer: Guangchuang Yu <guangchuangyu@gmail.com>\n",
      "Authors:\n",
      "Author: Guangchuang Yu [aut, cre, cph] , Tommy Tsan-Yuk Lam [aut, ths], Shuangbin Xu [aut] , Lin Li [ctb], Bradley Jones [ctb], Justin Silverman [ctb], Watal M. Iwasaki [ctb], Yonghe Xia [ctb], Ruizhu Huang [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - ape, type: imports\n",
      "  - aplot, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - ggplot2 (> 3.3.6) type: imports\n",
      "  - grid, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - methods, type: imports\n",
      "  - purrr, type: imports\n",
      "  - rlang, type: imports\n",
      "  - ggfun (>= 0.0.6) type: imports\n",
      "  - yulab.utils, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - tidytree (>= 0.3.9) type: imports\n",
      "  - treeio, type: imports\n",
      "  - utils, type: imports\n",
      "  - scales, type: imports\n",
      "  - stats, type: imports\n",
      "  - cli, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ggtree.html\n",
      "\n",
      "Name: rhdf5filters\n",
      "Description:\n",
      "Provides a collection of additional compression filters for HDF5 datasets. The package is intended to provide seemless integration with rhdf5, however the compiled filters can also be used with external applications.\n",
      "Version: 1.10.0\n",
      "Publication date: None\n",
      "Mantainer: Mike Smith <grimbough@gmail.com>\n",
      "Authors:\n",
      "Author: Mike Smith [aut, cre]\n",
      "Dependencies:\n",
      "None\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "BSD_2_clause + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/rhdf5filters.html\n",
      "\n",
      "Name: treeio\n",
      "Description:\n",
      "'treeio' is an R package to make it easier to import and store phylogenetic tree with associated data; and to link external data from different sources to phylogeny. It also supports exporting phylogenetic tree with heterogeneous associated data to a single tree file and can be served as a platform for merging tree with associated data and converting file formats.\n",
      "Version: 1.22.0\n",
      "Publication date: None\n",
      "Mantainer: Guangchuang Yu <guangchuangyu@gmail.com>\n",
      "Authors:\n",
      "Author: Guangchuang Yu [aut, cre] , Tommy Tsan-Yuk Lam [ctb, ths], Shuangbin Xu [ctb] , Bradley Jones [ctb], Casey Dunn [ctb], Tyler Bradley [ctb], Konstantinos Geles [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 3.6.0) type: depends\n",
      "  - ape, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - jsonlite, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - methods, type: imports\n",
      "  - rlang, type: imports\n",
      "  - tibble, type: imports\n",
      "  - tidytree (>= 0.3.9) type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/treeio.html\n",
      "\n",
      "Name: BiocIO\n",
      "Description:\n",
      "The `BiocIO` package contains high-level abstract classes and generics used by developers to build IO funcionality within the Bioconductor suite of packages. Implements `import()` and `export()` standard generics for importing and exporting biological data formats. `import()` supports whole-file as well as chunk-wise iterative import. The `import()` interface optionally provides a standard mechanism for 'lazy' access via `filter()` (on row or element-like components of the file resource), `select()` (on column-like components of the file resource) and `collect()`. The `import()` interface optionally provides transparent access to remote (e.g. via https) as well as local access. Developers can register a file extension, e.g., `.loom` for dispatch from character-based URIs to specific `import()` / `export()` methods based on classes representing file types, e.g., `LoomFile()`.\n",
      "Version: 1.8.0\n",
      "Publication date: None\n",
      "Mantainer: Bioconductor Package Maintainer <maintainer@bioconductor.org>\n",
      "Authors:\n",
      "Author: Martin Morgan [aut], Michael Lawrence [aut], Daniel Van Twisk [aut], Bioconductor Package Maintainer [cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - BiocGenerics, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - methods, type: imports\n",
      "  - tools, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/BiocIO.html\n",
      "\n",
      "Name: qvalue\n",
      "Description:\n",
      "This package takes a list of p-values resulting from the simultaneous testing of many hypotheses and estimates their q-values and local FDR values. The q-value of a test measures the proportion of false positives incurred (called the false discovery rate) when that particular test is called significant. The local FDR measures the posterior probability the null hypothesis is true given the test's p-value. Various plots are automatically generated, allowing one to make sensible significance cut-offs. Several mathematical results have recently been shown on the conservative accuracy of the estimated q-values from this software. The software can be applied to problems in genomics, brain imaging, astrophysics, and data mining.\n",
      "Version: 2.30.0\n",
      "Publication date: None\n",
      "Mantainer: John D. Storey <jstorey@princeton.edu>, Andrew J. Bass <ajbass@princeton.edu>\n",
      "Authors:\n",
      "Author: John D. Storey [aut, cre], Andrew J. Bass [aut], Alan Dabney [aut], David Robinson [aut], Gregory Warnes [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 2.10) type: depends\n",
      "  - splines, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - grid, type: imports\n",
      "  - reshape2, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/qvalue.html\n",
      "\n",
      "Name: preprocessCore\n",
      "Description:\n",
      "A library of core preprocessing routines.\n",
      "Version: 1.60.2\n",
      "Publication date: None\n",
      "Mantainer: Ben Bolstad <bmb@bmbolstad.com>\n",
      "Authors:\n",
      "Author: Ben Bolstad <bmb at bmbolstad.com>\n",
      "Dependencies:\n",
      "  - stats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/preprocessCore.html\n",
      "\n",
      "Name: enrichplot\n",
      "Description:\n",
      "The 'enrichplot' package implements several visualization methods for interpreting functional enrichment results obtained from ORA or GSEA analysis. It is mainly designed to work with the 'clusterProfiler' package suite. All the visualization methods are developed based on 'ggplot2' graphics.\n",
      "Version: 1.18.3\n",
      "Publication date: None\n",
      "Mantainer: Guangchuang Yu <guangchuangyu@gmail.com>\n",
      "Authors:\n",
      "Author: Guangchuang Yu [aut, cre] , Erqiang Hu [ctb] , Chun-Hui Gao [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - aplot (>= 0.1.4) type: imports\n",
      "  - DOSE, type: imports\n",
      "  - ggnewscale, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - ggraph, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grid, type: imports\n",
      "  - igraph, type: imports\n",
      "  - methods, type: imports\n",
      "  - plyr, type: imports\n",
      "  - purrr, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - rlang, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - scatterpie, type: imports\n",
      "  - shadowtext, type: imports\n",
      "  - GOSemSim, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - ggtree, type: imports\n",
      "  - yulab.utils (>= 0.0.4) type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/enrichplot.html\n",
      "\n",
      "Name: fgsea\n",
      "Description:\n",
      "The package implements an algorithm for fast gene set enrichment analysis. Using the fast algorithm allows to make more permutations and get more fine grained p-values, which allows to use accurate stantard approaches to multiple hypothesis correction.\n",
      "Version: 1.24.0\n",
      "Publication date: None\n",
      "Mantainer: Alexey Sergushichev <alsergbox@gmail.com>\n",
      "Authors:\n",
      "Author: Gennady Korotkevich [aut], Vladimir Sukhov [aut], Nikolay Budin [ctb], Alexey Sergushichev [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 3.3) type: depends\n",
      "  - Rcpp, type: imports\n",
      "  - data.table, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - stats, type: imports\n",
      "  - ggplot2 (>= 2.2.0) type: imports\n",
      "  - cowplot, type: imports\n",
      "  - grid, type: imports\n",
      "  - fastmatch, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENCE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/fgsea.html\n",
      "\n",
      "Name: DelayedMatrixStats\n",
      "Description:\n",
      "A port of the 'matrixStats' API for use with DelayedMatrix objects from the 'DelayedArray' package. High-performing functions operating on rows and columns of DelayedMatrix objects, e.g. col / rowMedians(), col / rowRanks(), and col / rowSds(). Functions optimized per data type and for subsetted calculations such that both memory usage and processing time is minimized.\n",
      "Version: 1.20.0\n",
      "Publication date: None\n",
      "Mantainer: Peter Hickey <peter.hickey@gmail.com>\n",
      "Authors:\n",
      "Author: Peter Hickey [aut, cre], HervÃ© PagÃ¨s [ctb], Aaron Lun [ctb]\n",
      "Dependencies:\n",
      "  - MatrixGenerics, type: depends\n",
      "  - DelayedArray, type: depends\n",
      "  - methods, type: imports\n",
      "  - matrixStats (>= 0.60.0) type: imports\n",
      "  - sparseMatrixStats, type: imports\n",
      "  - Matrix (>= 1.5-0) type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - IRanges, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/DelayedMatrixStats.html\n",
      "\n",
      "Name: DOSE\n",
      "Description:\n",
      "This package implements five methods proposed by Resnik, Schlicker, Jiang, Lin and Wang respectively for measuring semantic similarities among DO terms and gene products. Enrichment analyses including hypergeometric model and gene set enrichment analysis are also implemented for discovering disease associations of high-throughput biological data.\n",
      "Version: 3.24.2\n",
      "Publication date: None\n",
      "Mantainer: Guangchuang Yu <guangchuangyu@gmail.com>\n",
      "Authors:\n",
      "Author: Guangchuang Yu [aut, cre], Li-Gen Wang [ctb], Vladislav Petyuk [ctb], Giovanni Dall'Olio [ctb], Erqiang Hu [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - AnnotationDbi, type: imports\n",
      "  - HDO.db, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - fgsea, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - GOSemSim, type: imports\n",
      "  - methods, type: imports\n",
      "  - qvalue, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/DOSE.html\n",
      "\n",
      "Name: clusterProfiler\n",
      "Description:\n",
      "This package supports functional characteristics of both coding and non-coding genomics data for thousands of species with up-to-date gene annotation. It provides a univeral interface for gene functional annotation from a variety of sources and thus can be applied in diverse scenarios. It provides a tidy interface to access, manipulate, and visualize enrichment results to help users achieve efficient data interpretation. Datasets obtained from multiple treatments and time points can be analyzed and compared in a single run, easily revealing functional consensus and differences among distinct conditions.\n",
      "Version: 4.6.0\n",
      "Publication date: None\n",
      "Mantainer: Guangchuang Yu <guangchuangyu@gmail.com>\n",
      "Authors:\n",
      "Author: Guangchuang Yu [aut, cre, cph] , Li-Gen Wang [ctb], Erqiang Hu [ctb], Xiao Luo [ctb], Meijun Chen [ctb], Giovanni Dall'Olio [ctb], Wanqian Wei [ctb], Chun-Hui Gao [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - AnnotationDbi, type: imports\n",
      "  - downloader, type: imports\n",
      "  - DOSE, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - enrichplot, type: imports\n",
      "  - GO.db, type: imports\n",
      "  - GOSemSim, type: imports\n",
      "  - gson (>= 0.0.7) type: imports\n",
      "  - magrittr, type: imports\n",
      "  - methods, type: imports\n",
      "  - plyr, type: imports\n",
      "  - qvalue, type: imports\n",
      "  - rlang, type: imports\n",
      "  - stats, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - utils, type: imports\n",
      "  - yulab.utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/clusterProfiler.html\n",
      "\n",
      "Name: GOSemSim\n",
      "Description:\n",
      "The semantic comparisons of Gene Ontology (GO) annotations provide quantitative ways to compute similarities between genes and gene groups, and have became important basis for many bioinformatics analysis approaches. GOSemSim is an R package for semantic similarity computation among GO terms, sets of GO terms, gene products and gene clusters. GOSemSim implemented five methods proposed by Resnik, Schlicker, Jiang, Lin and Wang respectively.\n",
      "Version: 2.24.0\n",
      "Publication date: None\n",
      "Mantainer: Guangchuang Yu <guangchuangyu@gmail.com>\n",
      "Authors:\n",
      "Author: Guangchuang Yu [aut, cre], Alexey Stukalov [ctb], Pingfan Guo [ctb], Chuanle Xiao [ctb], LluÃ­s Revilla Sancho [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - AnnotationDbi, type: imports\n",
      "  - GO.db, type: imports\n",
      "  - methods, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/GOSemSim.html\n",
      "\n",
      "Name: sparseMatrixStats\n",
      "Description:\n",
      "High performance functions for row and column operations on sparse matrices. For example: col / rowMeans2, col / rowMedians, col / rowVars etc. Currently, the optimizations are limited to data in the column sparse format. This package is inspired by the matrixStats package by Henrik Bengtsson.\n",
      "Version: 1.10.0\n",
      "Publication date: None\n",
      "Mantainer: Constantin Ahlmann-Eltze <artjom31415@googlemail.com>\n",
      "Authors:\n",
      "Author: Constantin Ahlmann-Eltze [aut, cre]\n",
      "Dependencies:\n",
      "  - MatrixGenerics, type: depends\n",
      "  - Rcpp, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - matrixStats (>= 0.60.0) type: imports\n",
      "  - methods, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/sparseMatrixStats.html\n",
      "\n",
      "Name: SingleCellExperiment\n",
      "Description:\n",
      "Defines a S4 class for storing data from single-cell experiments. This includes specialized methods to store and retrieve spike-in information, dimensionality reduction coordinates and size factors for each cell, along with the usual metadata for genes and libraries.\n",
      "Version: 1.20.0\n",
      "Publication date: None\n",
      "Mantainer: Davide Risso <risso.davide@gmail.com>\n",
      "Authors:\n",
      "Author: Aaron Lun [aut, cph], Davide Risso [aut, cre, cph], Keegan Korthauer [ctb], Kevin Rue-Albrecht [ctb]\n",
      "Dependencies:\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - methods, type: imports\n",
      "  - utils, type: imports\n",
      "  - stats, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - DelayedArray, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/SingleCellExperiment.html\n",
      "\n",
      "Name: ComplexHeatmap\n",
      "Description:\n",
      "Complex heatmaps are efficient to visualize associations between different sources of data sets and reveal potential patterns. Here the ComplexHeatmap package provides a highly flexible way to arrange multiple heatmaps and supports various annotation graphics.\n",
      "Version: 2.14.0\n",
      "Publication date: None\n",
      "Mantainer: Zuguang Gu <z.gu@dkfz.de>\n",
      "Authors:\n",
      "Author: Zuguang Gu [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - grid, type: depends\n",
      "  - graphics, type: depends\n",
      "  - stats, type: depends\n",
      "  - grDevices, type: depends\n",
      "  - circlize (>= 0.4.14) type: imports\n",
      "  - GetoptLong, type: imports\n",
      "  - colorspace, type: imports\n",
      "  - clue, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - GlobalOptions (>= 0.1.0) type: imports\n",
      "  - png, type: imports\n",
      "  - digest, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - foreach, type: imports\n",
      "  - doParallel, type: imports\n",
      "  - codetools, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ComplexHeatmap.html\n",
      "\n",
      "Name: beachmat\n",
      "Description:\n",
      "Provides a consistent C++ class interface for reading from and writing data to a variety of commonly used matrix types. Ordinary matrices and several sparse/dense Matrix classes are directly supported, third-party S4 classes may be supported by external linkage, while all other matrices are handled by DelayedArray block processing.\n",
      "Version: 2.14.0\n",
      "Publication date: None\n",
      "Mantainer: Aaron Lun <infinite.monkeys.with.keyboards@gmail.com>\n",
      "Authors:\n",
      "Author: Aaron Lun [aut, cre], HervÃ© PagÃ¨s [aut], Mike Smith [aut]\n",
      "Dependencies:\n",
      "  - methods, type: imports\n",
      "  - DelayedArray, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - Rcpp, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/beachmat.html\n",
      "\n",
      "Name: HDF5Array\n",
      "Description:\n",
      "Implement the HDF5Array, H5SparseMatrix, H5ADMatrix, and TENxMatrix classes, 4 convenient and memory-efficient array-like containers for representing and manipulating either: (1) a conventional (a.k.a. dense) HDF5 dataset, (2) an HDF5 sparse matrix (stored in CSR/CSC/Yale format), (3) the central matrix of an h5ad file (or any matrix in the /layers group), or (4) a 10x Genomics sparse matrix. All these containers are DelayedArray extensions and thus support all operations (delayed or block-processed) supported by DelayedArray objects.\n",
      "Version: 1.26.0\n",
      "Publication date: None\n",
      "Mantainer: HervÃ© PagÃ¨s <hpages.on.github@gmail.com>\n",
      "Authors:\n",
      "Author: HervÃ© PagÃ¨s\n",
      "Dependencies:\n",
      "  - R (>= 3.4) type: depends\n",
      "  - methods, type: depends\n",
      "  - DelayedArray, type: depends\n",
      "  - rhdf5, type: depends\n",
      "  - utils, type: imports\n",
      "  - stats, type: imports\n",
      "  - tools, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - rhdf5filters, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - IRanges, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/HDF5Array.html\n",
      "\n",
      "Name: BSgenome\n",
      "Description:\n",
      "Infrastructure shared by all the Biostrings-based genome data packages.\n",
      "Version: 1.66.3\n",
      "Publication date: None\n",
      "Mantainer: HervÃ© PagÃ¨s <hpages.on.github@gmail.com>\n",
      "Authors:\n",
      "Author: HervÃ© PagÃ¨s [aut, cre], Prisca Chidimma Maduka [ctb] (add 'replace' argument to forgeBSgenomeDataPkg())\n",
      "Dependencies:\n",
      "  - R (>= 2.8.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - BiocGenerics, type: depends\n",
      "  - S4Vectors, type: depends\n",
      "  - IRanges, type: depends\n",
      "  - GenomeInfoDb, type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - Biostrings, type: depends\n",
      "  - rtracklayer, type: depends\n",
      "  - methods, type: imports\n",
      "  - utils, type: imports\n",
      "  - stats, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - XVector, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/BSgenome.html\n",
      "\n",
      "Name: multtest\n",
      "Description:\n",
      "Non-parametric bootstrap and permutation resampling-based multiple testing procedures (including empirical Bayes methods) for controlling the family-wise error rate (FWER), generalized family-wise error rate (gFWER), tail probability of the proportion of false positives (TPPFP), and false discovery rate (FDR). Several choices of bootstrap-based null distribution are implemented (centered, centered and scaled, quantile-transformed). Single-step and step-wise methods are available. Tests based on a variety of t- and F-statistics (including t-statistics based on regression parameters from linear and survival models as well as those based on correlation parameters) are included. When probing hypotheses with t-statistics, users may also select a potentially faster null distribution which is multivariate normal with mean zero and variance covariance matrix derived from the vector influence function. Results are reported in terms of adjusted p-values, confidence regions and test statistic cutoffs. The procedures are directly applicable to identifying differentially expressed genes in DNA microarray experiments.\n",
      "Version: 2.54.0\n",
      "Publication date: None\n",
      "Mantainer: Katherine S. Pollard <katherine.pollard@gladstone.ucsf.edu>\n",
      "Authors:\n",
      "Author: Katherine S. Pollard, Houston N. Gilbert, Yongchao Ge, Sandra Taylor, Sandrine Dudoit\n",
      "Dependencies:\n",
      "  - R (>= 2.10) type: depends\n",
      "  - methods, type: depends\n",
      "  - BiocGenerics, type: depends\n",
      "  - Biobase, type: depends\n",
      "  - survival, type: imports\n",
      "  - MASS, type: imports\n",
      "  - stats4, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/multtest.html\n",
      "\n",
      "Name: GEOquery\n",
      "Description:\n",
      "The NCBI Gene Expression Omnibus (GEO) is a public repository of microarray data. Given the rich and varied nature of this resource, it is only natural to want to apply BioConductor tools to these data. GEOquery is the bridge between GEO and BioConductor.\n",
      "Version: 2.66.0\n",
      "Publication date: None\n",
      "Mantainer: Sean Davis <seandavi@gmail.com>\n",
      "Authors:\n",
      "Author: Sean Davis [aut, cre]\n",
      "Dependencies:\n",
      "  - methods, type: depends\n",
      "  - Biobase, type: depends\n",
      "  - readr (>= 1.3.1) type: imports\n",
      "  - xml2, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - data.table, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - limma, type: imports\n",
      "  - curl, type: imports\n",
      "  - R.utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/GEOquery.html\n",
      "\n",
      "Name: BiocSingular\n",
      "Description:\n",
      "Implements exact and approximate methods for singular value decomposition and principal components analysis, in a framework that allows them to be easily switched within Bioconductor packages or workflows. Where possible, parallelization is achieved using the BiocParallel framework.\n",
      "Version: 1.14.0\n",
      "Publication date: None\n",
      "Mantainer: Aaron Lun <infinite.monkeys.with.keyboards@gmail.com>\n",
      "Authors:\n",
      "Author: Aaron Lun [aut, cre, cph]\n",
      "Dependencies:\n",
      "  - BiocGenerics, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - methods, type: imports\n",
      "  - utils, type: imports\n",
      "  - DelayedArray, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - ScaledMatrix, type: imports\n",
      "  - irlba, type: imports\n",
      "  - rsvd, type: imports\n",
      "  - Rcpp, type: imports\n",
      "  - beachmat, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/BiocSingular.html\n",
      "\n",
      "Name: RBGL\n",
      "Description:\n",
      "A fairly extensive and comprehensive interface to the graph algorithms contained in the BOOST library.\n",
      "Version: 1.74.0\n",
      "Publication date: None\n",
      "Mantainer: Bioconductor Package Maintainer <maintainer@bioconductor.org>\n",
      "Authors:\n",
      "Author: Vince Carey <stvjc at channing.harvard.edu>, Li Long <li.long at isb-sib.ch>, R. Gentleman\n",
      "Dependencies:\n",
      "  - graph, type: depends\n",
      "  - methods, type: depends\n",
      "  - methods, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/RBGL.html\n",
      "\n",
      "Name: Rgraphviz\n",
      "Description:\n",
      "Interfaces R with the AT and T graphviz library for plotting R graph objects from the graph package.\n",
      "Version: 2.42.0\n",
      "Publication date: None\n",
      "Mantainer: Kasper Daniel Hansen <kasperdanielhansen@gmail.com>\n",
      "Authors:\n",
      "Author: Kasper Daniel Hansen [cre, aut], Jeff Gentry [aut], Li Long [aut], Robert Gentleman [aut], Seth Falcon [aut], Florian Hahne [aut], Deepayan Sarkar [aut]\n",
      "Dependencies:\n",
      "  - R (>= 2.6.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - utils, type: depends\n",
      "  - graph, type: depends\n",
      "  - grid, type: depends\n",
      "  - stats4, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "EPL\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/Rgraphviz.html\n",
      "\n",
      "Name: ProtGenerics\n",
      "Description:\n",
      "S4 generic functions and classes needed by Bioconductor proteomics packages.\n",
      "Version: 1.30.0\n",
      "Publication date: None\n",
      "Mantainer: Laurent Gatto <laurent.gatto@uclouvain.be>\n",
      "Authors:\n",
      "Author: Laurent Gatto <laurent.gatto at uclouvain.be>, Johannes Rainer <johannes.rainer at eurac.edu>\n",
      "Dependencies:\n",
      "  - methods, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ProtGenerics.html\n",
      "\n",
      "Name: ScaledMatrix\n",
      "Description:\n",
      "Provides delayed computation of a matrix of scaled and centered values. The result is equivalent to using the scale() function but avoids explicit realization of a dense matrix during block processing. This permits greater efficiency in common operations, most notably matrix multiplication.\n",
      "Version: 1.6.0\n",
      "Publication date: None\n",
      "Mantainer: Aaron Lun <infinite.monkeys.with.keyboards@gmail.com>\n",
      "Authors:\n",
      "Author: Aaron Lun [aut, cre, cph]\n",
      "Dependencies:\n",
      "  - methods, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - DelayedArray, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ScaledMatrix.html\n",
      "\n",
      "Name: impute\n",
      "Description:\n",
      "Imputation for microarray data (currently KNN only)\n",
      "Version: 1.72.3\n",
      "Publication date: None\n",
      "Mantainer: Balasubramanian Narasimhan <naras@stat.Stanford.EDU>\n",
      "Authors:\n",
      "Author: Trevor Hastie, Robert Tibshirani, Balasubramanian Narasimhan, Gilbert Chu\n",
      "Dependencies:\n",
      "  - R (>= 2.10) type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/impute.html\n",
      "\n",
      "Name: AnnotationHub\n",
      "Description:\n",
      "This package provides a client for the Bioconductor AnnotationHub web resource. The AnnotationHub web resource provides a central location where genomic files (e.g., VCF, bed, wig) and other resources from standard locations (e.g., UCSC, Ensembl) can be discovered. The resource includes metadata about each resource, e.g., a textual description, tags, and date of modification. The client creates and manages a local cache of files retrieved by the user, helping with quick and reproducible access.\n",
      "Version: 3.6.0\n",
      "Publication date: None\n",
      "Mantainer: Bioconductor Package Maintainer <maintainer@bioconductor.org>\n",
      "Authors:\n",
      "Author: Bioconductor Package Maintainer [cre], Martin Morgan [aut], Marc Carlson [ctb], Dan Tenenbaum [ctb], Sonali Arora [ctb], Valerie Oberchain [ctb], Kayla Morrell [ctb], Lori Shepherd [aut]\n",
      "Dependencies:\n",
      "  - BiocGenerics, type: depends\n",
      "  - BiocFileCache, type: depends\n",
      "  - utils, type: imports\n",
      "  - methods, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - RSQLite, type: imports\n",
      "  - BiocManager, type: imports\n",
      "  - BiocVersion, type: imports\n",
      "  - curl, type: imports\n",
      "  - rappdirs, type: imports\n",
      "  - AnnotationDbi, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - interactiveDisplayBase, type: imports\n",
      "  - httr, type: imports\n",
      "  - yaml, type: imports\n",
      "  - dplyr, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/AnnotationHub.html\n",
      "\n",
      "Name: ensembldb\n",
      "Description:\n",
      "The package provides functions to create and use transcript centric annotation databases/packages. The annotation for the databases are directly fetched from Ensembl using their Perl API. The functionality and data is similar to that of the TxDb packages from the GenomicFeatures package, but, in addition to retrieve all gene/transcript models and annotations from the database, ensembldb provides a filter framework allowing to retrieve annotations for specific entries like genes encoded on a chromosome region or transcript models of lincRNA genes. EnsDb databases built with ensembldb contain also protein annotations and mappings between proteins and their encoding transcripts. Finally, ensembldb provides functions to map between genomic, transcript and protein coordinates.\n",
      "Version: 2.22.0\n",
      "Publication date: None\n",
      "Mantainer: Johannes Rainer <johannes.rainer@eurac.edu>\n",
      "Authors:\n",
      "Author: Johannes Rainer <johannes.rainer at eurac.edu> with contributions from Tim Triche, Sebastian Gibb, Laurent Gatto and Christian Weichenberger.\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - BiocGenerics, type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - GenomicFeatures, type: depends\n",
      "  - AnnotationFilter, type: depends\n",
      "  - methods, type: imports\n",
      "  - RSQLite (>= 1.1) type: imports\n",
      "  - DBI, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - AnnotationDbi, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - ProtGenerics, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - curl, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ensembldb.html\n",
      "\n",
      "Name: affyio\n",
      "Description:\n",
      "Routines for parsing Affymetrix data files based upon file format information. Primary focus is on accessing the CEL and CDF file formats.\n",
      "Version: 1.68.0\n",
      "Publication date: None\n",
      "Mantainer: Ben Bolstad <bmb@bmbolstad.com>\n",
      "Authors:\n",
      "Author: Ben Bolstad <bmb at bmbolstad.com>\n",
      "Dependencies:\n",
      "  - R (>= 2.6.0) type: depends\n",
      "  - zlibbioc, type: imports\n",
      "  - methods, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/affyio.html\n",
      "\n",
      "Name: affy\n",
      "Description:\n",
      "The package contains functions for exploratory oligonucleotide array analysis. The dependence on tkWidgets only concerns few convenience functions. 'affy' is fully functional without it.\n",
      "Version: 1.76.0\n",
      "Publication date: None\n",
      "Mantainer: Rafael A. Irizarry <rafa@ds.harvard.edu>\n",
      "Authors:\n",
      "Author: Rafael A. Irizarry <rafa at ds.harvard.edu>, Laurent Gautier <lgautier at gmail.com>, Benjamin Milo Bolstad <bmb at bmbolstad.com>, and Crispin Miller <cmiller at picr.man.ac.uk> with contributions from Magnus Astrand <Magnus.Astrand at astrazeneca.com>, Leslie M. Cope <cope at mts.jhu.edu>, Robert Gentleman, Jeff Gentry, Conrad Halling <challing at agilixcorp.com>, Wolfgang Huber, James MacDonald <jmacdon at u.washington.edu>, Benjamin I. P. Rubinstein, Christopher Workman <workman at cbs.dtu.dk>, John Zhang\n",
      "Dependencies:\n",
      "  - R (>= 2.8.0) type: depends\n",
      "  - BiocGenerics, type: depends\n",
      "  - Biobase, type: depends\n",
      "  - affyio, type: imports\n",
      "  - BiocManager, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - methods, type: imports\n",
      "  - preprocessCore, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - zlibbioc, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL (>= 2.0)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/affy.html\n",
      "\n",
      "Name: interactiveDisplayBase\n",
      "Description:\n",
      "The interactiveDisplayBase package contains the the basic methods needed to generate interactive Shiny based display methods for Bioconductor objects.\n",
      "Version: 1.36.0\n",
      "Publication date: None\n",
      "Mantainer: Bioconductor Package Maintainer <maintainer@bioconductor.org>\n",
      "Authors:\n",
      "Author: Bioconductor Package Maintainer [cre], Shawn Balcome [aut], Marc Carlson [ctb], Marcel Ramos [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 2.10) type: depends\n",
      "  - methods, type: depends\n",
      "  - BiocGenerics, type: depends\n",
      "  - shiny, type: imports\n",
      "  - DT, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/interactiveDisplayBase.html\n",
      "\n",
      "Name: AnnotationFilter\n",
      "Description:\n",
      "This package provides class and other infrastructure to implement filters for manipulating Bioconductor annotation resources. The filters will be used by ensembldb, Organism.dplyr, and other packages.\n",
      "Version: 1.22.0\n",
      "Publication date: None\n",
      "Mantainer: Bioconductor Package Maintainer <maintainer@bioconductor.org>\n",
      "Authors:\n",
      "Author: Martin Morgan [aut], Johannes Rainer [aut], Joachim Bargsten [ctb], Daniel Van Twisk [ctb], Bioconductor Package Maintainer [cre]\n",
      "Dependencies:\n",
      "  - R (>= 3.4.0) type: depends\n",
      "  - utils, type: imports\n",
      "  - methods, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - lazyeval, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/AnnotationFilter.html\n",
      "\n",
      "Name: GSEABase\n",
      "Description:\n",
      "This package provides classes and methods to support Gene Set Enrichment Analysis (GSEA).\n",
      "Version: 1.60.0\n",
      "Publication date: None\n",
      "Mantainer: Bioconductor Package Maintainer <maintainer@bioconductor.org>\n",
      "Authors:\n",
      "Author: Martin Morgan, Seth Falcon, Robert Gentleman\n",
      "Dependencies:\n",
      "  - R (>= 2.6.0) type: depends\n",
      "  - BiocGenerics, type: depends\n",
      "  - Biobase, type: depends\n",
      "  - annotate, type: depends\n",
      "  - methods, type: depends\n",
      "  - graph, type: depends\n",
      "  - AnnotationDbi, type: imports\n",
      "  - XML, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/GSEABase.html\n",
      "\n",
      "Name: BiocNeighbors\n",
      "Description:\n",
      "Implements exact and approximate methods for nearest neighbor detection, in a framework that allows them to be easily switched within Bioconductor packages or workflows. Exact searches can be performed using the k-means for k-nearest neighbors algorithm or with vantage point trees. Approximate searches can be performed using the Annoy or HNSW libraries. Searching on either Euclidean or Manhattan distances is supported. Parallelization is achieved for all methods by using BiocParallel. Functions are also provided to search for all neighbors within a given distance.\n",
      "Version: 1.16.0\n",
      "Publication date: None\n",
      "Mantainer: Aaron Lun <infinite.monkeys.with.keyboards@gmail.com>\n",
      "Authors:\n",
      "Author: Aaron Lun [aut, cre, cph]\n",
      "Dependencies:\n",
      "  - Rcpp, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - stats, type: imports\n",
      "  - methods, type: imports\n",
      "  - Matrix, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/BiocNeighbors.html\n",
      "\n",
      "Name: scuttle\n",
      "Description:\n",
      "Provides basic utility functions for performing single-cell analyses, focusing on simple normalization, quality control and data transformations. Also provides some helper functions to assist development of other packages.\n",
      "Version: 1.8.4\n",
      "Publication date: None\n",
      "Mantainer: Aaron Lun <infinite.monkeys.with.keyboards@gmail.com>\n",
      "Authors:\n",
      "Author: Aaron Lun [aut, cre], Davis McCarthy [aut]\n",
      "Dependencies:\n",
      "  - SingleCellExperiment, type: depends\n",
      "  - methods, type: imports\n",
      "  - utils, type: imports\n",
      "  - stats, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - Rcpp, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - DelayedArray, type: imports\n",
      "  - DelayedMatrixStats, type: imports\n",
      "  - beachmat, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/scuttle.html\n",
      "\n",
      "Name: VariantAnnotation\n",
      "Description:\n",
      "Annotate variants, compute amino acid coding changes, predict coding outcomes.\n",
      "Version: 1.44.1\n",
      "Publication date: None\n",
      "Mantainer: Bioconductor Package Maintainer <maintainer@bioconductor.org>\n",
      "Authors:\n",
      "Author: Bioconductor Package Maintainer [aut, cre], Valerie Oberchain [aut], Martin Morgan [aut], Michael Lawrence [aut], Stephanie Gogarten [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 4.0.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - BiocGenerics, type: depends\n",
      "  - MatrixGenerics, type: depends\n",
      "  - GenomeInfoDb, type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - Rsamtools, type: depends\n",
      "  - utils, type: imports\n",
      "  - DBI, type: imports\n",
      "  - zlibbioc, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - XVector, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - AnnotationDbi, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - BSgenome, type: imports\n",
      "  - GenomicFeatures, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/VariantAnnotation.html\n",
      "\n",
      "Name: BiocStyle\n",
      "Description:\n",
      "Provides standard formatting styles for Bioconductor PDF and HTML documents. Package vignettes illustrate use and functionality.\n",
      "Version: 2.26.0\n",
      "Publication date: None\n",
      "Mantainer: Bioconductor Package <maintainer@bioconductor.org>\n",
      "Authors:\n",
      "Author: Andrzej OleÅ [aut] , Mike Smith [ctb] , Martin Morgan [ctb], Wolfgang Huber [ctb], Bioconductor Package [cre]\n",
      "Dependencies:\n",
      "  - bookdown, type: imports\n",
      "  - knitr (>= 1.30) type: imports\n",
      "  - rmarkdown (>= 1.2) type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - yaml, type: imports\n",
      "  - BiocManager, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/BiocStyle.html\n",
      "\n",
      "Name: sva\n",
      "Description:\n",
      "The sva package contains functions for removing batch effects and other unwanted variation in high-throughput experiment. Specifically, the sva package contains functions for the identifying and building surrogate variables for high-dimensional data sets. Surrogate variables are covariates constructed directly from high-dimensional data (like gene expression/RNA sequencing/methylation/brain imaging data) that can be used in subsequent analyses to adjust for unknown, unmodeled, or latent sources of noise. The sva package can be used to remove artifacts in three ways: (1) identifying and estimating surrogate variables for unknown sources of variation in high-throughput experiments (Leek and Storey 2007 PLoS Genetics,2008 PNAS), (2) directly removing known batch effects using ComBat (Johnson et al. 2007 Biostatistics) and (3) removing batch effects with known control probes (Leek 2014 biorXiv). Removing batch effects and using surrogate variables in differential expression analysis have been shown to reduce dependence, stabilize error rate estimates, and improve reproducibility, see (Leek and Storey 2007 PLoS Genetics, 2008 PNAS or Leek et al. 2011 Nat. Reviews Genetics).\n",
      "Version: 3.46.0\n",
      "Publication date: None\n",
      "Mantainer: Jeffrey T. Leek <jtleek@gmail.com>, John D. Storey <jstorey@princeton.edu>, W. Evan Johnson <wej@bu.edu>\n",
      "Authors:\n",
      "Author: Jeffrey T. Leek <jtleek at gmail.com>, W. Evan Johnson <wej at bu.edu>, Hilary S. Parker <hiparker at jhsph.edu>, Elana J. Fertig <ejfertig at jhmi.edu>, Andrew E. Jaffe <ajaffe at jhsph.edu>, Yuqing Zhang <zhangyuqing.pkusms at gmail.com>, John D. Storey <jstorey at princeton.edu>, Leonardo Collado Torres <lcolladotor at gmail.com>\n",
      "Dependencies:\n",
      "  - R (>= 3.2) type: depends\n",
      "  - mgcv, type: depends\n",
      "  - genefilter, type: depends\n",
      "  - BiocParallel, type: depends\n",
      "  - matrixStats, type: imports\n",
      "  - stats, type: imports\n",
      "  - graphics, type: imports\n",
      "  - utils, type: imports\n",
      "  - limma, type: imports\n",
      "  - edgeR, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/sva.html\n",
      "\n",
      "Name: ShortRead\n",
      "Description:\n",
      "This package implements sampling, iteration, and input of FASTQ files. The package includes functions for filtering and trimming reads, and for generating a quality assessment report. Data are represented as DNAStringSet-derived objects, and easily manipulated for a diversity of purposes. The package also contains legacy support for early single-end, ungapped alignment formats.\n",
      "Version: 1.56.1\n",
      "Publication date: None\n",
      "Mantainer: Bioconductor Package Maintainer <maintainer@bioconductor.org>\n",
      "Authors:\n",
      "Author: Bioconductor Package Maintainer [cre], Martin Morgan [aut], Michael Lawrence [ctb], Simon Anders [ctb]\n",
      "Dependencies:\n",
      "  - BiocGenerics, type: depends\n",
      "  - BiocParallel, type: depends\n",
      "  - Biostrings, type: depends\n",
      "  - Rsamtools, type: depends\n",
      "  - GenomicAlignments, type: depends\n",
      "  - Biobase, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - hwriter, type: imports\n",
      "  - methods, type: imports\n",
      "  - zlibbioc, type: imports\n",
      "  - lattice, type: imports\n",
      "  - latticeExtra, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ShortRead.html\n",
      "\n",
      "Name: ExperimentHub\n",
      "Description:\n",
      "This package provides a client for the Bioconductor ExperimentHub web resource. ExperimentHub provides a central location where curated data from experiments, publications or training courses can be accessed. Each resource has associated metadata, tags and date of modification. The client creates and manages a local cache of files retrieved enabling quick and reproducible access.\n",
      "Version: 2.6.0\n",
      "Publication date: None\n",
      "Mantainer: Bioconductor Package Maintainer <maintainer@bioconductor.org>\n",
      "Authors:\n",
      "Author: Bioconductor Package Maintainer [cre], Martin Morgan [aut], Marc Carlson [ctb], Dan Tenenbaum [ctb], Sonali Arora [ctb], Valerie Oberchain [ctb], Kayla Morrell [ctb], Lori Shepherd [aut]\n",
      "Dependencies:\n",
      "  - methods, type: depends\n",
      "  - BiocGenerics, type: depends\n",
      "  - AnnotationHub, type: depends\n",
      "  - BiocFileCache, type: depends\n",
      "  - utils, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - BiocManager, type: imports\n",
      "  - curl, type: imports\n",
      "  - rappdirs, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ExperimentHub.html\n",
      "\n",
      "Name: scater\n",
      "Description:\n",
      "A collection of tools for doing various analyses of single-cell RNA-seq gene expression data, with a focus on quality control and visualization.\n",
      "Version: 1.26.1\n",
      "Publication date: None\n",
      "Mantainer: Alan O'Callaghan <alan.ocallaghan@outlook.com>\n",
      "Authors:\n",
      "Author: Davis McCarthy [aut], Kieran Campbell [aut], Aaron Lun [aut, ctb], Quin Wills [aut], Vladimir Kiselev [ctb], Felix G.M. Ernst [ctb], Alan O'Callaghan [ctb, cre]\n",
      "Dependencies:\n",
      "  - SingleCellExperiment, type: depends\n",
      "  - scuttle, type: depends\n",
      "  - ggplot2, type: depends\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - methods, type: imports\n",
      "  - grid, type: imports\n",
      "  - gridExtra, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - DelayedArray, type: imports\n",
      "  - DelayedMatrixStats, type: imports\n",
      "  - beachmat, type: imports\n",
      "  - BiocNeighbors, type: imports\n",
      "  - BiocSingular, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - rlang, type: imports\n",
      "  - ggbeeswarm, type: imports\n",
      "  - viridis, type: imports\n",
      "  - Rtsne, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - RcppML, type: imports\n",
      "  - uwot, type: imports\n",
      "  - pheatmap, type: imports\n",
      "  - ggrastr, type: imports\n",
      "  - ggrepel, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/scater.html\n",
      "\n",
      "Name: vsn\n",
      "Description:\n",
      "The package implements a method for normalising microarray intensities from single- and multiple-color arrays. It can also be used for data from other technologies, as long as they have similar format. The method uses a robust variant of the maximum-likelihood estimator for an additive-multiplicative error model and affine calibration. The model incorporates data calibration step (a.k.a. normalization), a model for the dependence of the variance on the mean intensity and a variance stabilizing data transformation. Differences between transformed intensities are analogous to \"normalized log-ratios\". However, in contrast to the latter, their variance is independent of the mean, and they are usually more sensitive and specific in detecting differential transcription.\n",
      "Version: 3.66.0\n",
      "Publication date: None\n",
      "Mantainer: Wolfgang Huber <wolfgang.huber@embl.org>\n",
      "Authors:\n",
      "Author: Wolfgang Huber, with contributions from Anja von Heydebreck. Many comments and suggestions by users are acknowledged, among them Dennis Kostka, David Kreil, Hans-Ulrich Klein, Robert Gentleman, Deepayan Sarkar and Gordon Smyth\n",
      "Dependencies:\n",
      "  - R (>= 4.0.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - Biobase, type: depends\n",
      "  - affy, type: imports\n",
      "  - limma, type: imports\n",
      "  - lattice, type: imports\n",
      "  - ggplot2, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/vsn.html\n",
      "\n",
      "Name: KEGGgraph\n",
      "Description:\n",
      "KEGGGraph is an interface between KEGG pathway and graph object as well as a collection of tools to analyze, dissect and visualize these graphs. It parses the regularly updated KGML (KEGG XML) files into graph models maintaining all essential pathway attributes. The package offers functionalities including parsing, graph operation, visualization and etc.\n",
      "Version: 1.58.3\n",
      "Publication date: None\n",
      "Mantainer: Jitao David Zhang <jitao_david.zhang@roche.com>\n",
      "Authors:\n",
      "Author: Jitao David Zhang, with inputs from Paul Shannon and HervÃ© PagÃ¨s\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - methods, type: imports\n",
      "  - XML (>= 2.3-0) type: imports\n",
      "  - graph, type: imports\n",
      "  - utils, type: imports\n",
      "  - RCurl, type: imports\n",
      "  - Rgraphviz, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/KEGGgraph.html\n",
      "\n",
      "Name: biovizBase\n",
      "Description:\n",
      "The biovizBase package is designed to provide a set of utilities, color schemes and conventions for genomic data. It serves as the base for various high-level packages for biological data visualization. This saves development effort and encourages consistency.\n",
      "Version: 1.46.0\n",
      "Publication date: None\n",
      "Mantainer: Michael Lawrence <michafla@gene.com>\n",
      "Authors:\n",
      "Author: Tengfei Yin [aut], Michael Lawrence [aut, ths, cre], Dianne Cook [aut, ths], Johannes Rainer [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - grDevices, type: imports\n",
      "  - stats, type: imports\n",
      "  - scales, type: imports\n",
      "  - Hmisc, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - dichromat, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - GenomicAlignments, type: imports\n",
      "  - GenomicFeatures, type: imports\n",
      "  - AnnotationDbi, type: imports\n",
      "  - VariantAnnotation, type: imports\n",
      "  - ensembldb, type: imports\n",
      "  - AnnotationFilter, type: imports\n",
      "  - rlang, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/biovizBase.html\n",
      "\n",
      "Name: biomformat\n",
      "Description:\n",
      "This is an R package for interfacing with the BIOM format. This package includes basic tools for reading biom-format files, accessing and subsetting data tables from a biom object (which is more complex than a single table), as well as limited support for writing a biom-object back to a biom-format file. The design of this API is intended to match the python API and other tools included with the biom-format project, but with a decidedly \"R flavor\" that should be familiar to R users. This includes S4 classes and methods, as well as extensions of common core functions/methods.\n",
      "Version: 1.26.0\n",
      "Publication date: None\n",
      "Mantainer: Paul J. McMurdie <mcmurdie@stanford.edu>\n",
      "Authors:\n",
      "Author: Paul J. McMurdie <mcmurdie at stanford.edu> and Joseph N Paulson <jpaulson at jimmy.harvard.edu>\n",
      "Dependencies:\n",
      "  - R (>= 3.2) type: depends\n",
      "  - methods, type: depends\n",
      "  - plyr (>= 1.8) type: imports\n",
      "  - jsonlite (>= 0.9.16) type: imports\n",
      "  - Matrix (>= 1.2) type: imports\n",
      "  - rhdf5, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/biomformat.html\n",
      "\n",
      "Name: GSVA\n",
      "Description:\n",
      "Gene Set Variation Analysis (GSVA) is a non-parametric, unsupervised method for estimating variation of gene set enrichment through the samples of a expression data set. GSVA performs a change in coordinate systems, transforming the data from a gene by sample matrix to a gene-set by sample matrix, thereby allowing the evaluation of pathway enrichment for each sample. This new matrix of GSVA enrichment scores facilitates applying standard analytical methods like functional enrichment, survival analysis, clustering, CNV-pathway analysis or cross-tissue pathway analysis, in a pathway-centric manner.\n",
      "Version: 1.46.0\n",
      "Publication date: None\n",
      "Mantainer: Robert Castelo <robert.castelo@upf.edu>\n",
      "Authors:\n",
      "Author: Robert Castelo [aut, cre], Justin Guinney [aut], Alexey Sergushichev [ctb], Pablo Sebastian Rodriguez [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - graphics, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - GSEABase, type: imports\n",
      "  - Matrix (>= 1.5-0) type: imports\n",
      "  - parallel, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - SingleCellExperiment, type: imports\n",
      "  - sparseMatrixStats, type: imports\n",
      "  - DelayedArray, type: imports\n",
      "  - DelayedMatrixStats, type: imports\n",
      "  - HDF5Array, type: imports\n",
      "  - BiocSingular, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/GSVA.html\n",
      "\n",
      "Name: pcaMethods\n",
      "Description:\n",
      "Provides Bayesian PCA, Probabilistic PCA, Nipals PCA, Inverse Non-Linear PCA and the conventional SVD PCA. A cluster based method for missing value estimation is included for comparison. BPCA, PPCA and NipalsPCA may be used to perform PCA on incomplete data as well as for accurate missing value estimation. A set of methods for printing and plotting the results is also provided. All PCA methods make use of the same data structure (pcaRes) to provide a common interface to the PCA results. Initiated at the Max-Planck Institute for Molecular Plant Physiology, Golm, Germany.\n",
      "Version: 1.90.0\n",
      "Publication date: None\n",
      "Mantainer: Henning Redestig <henning.red@gmail.com>\n",
      "Authors:\n",
      "Author: Wolfram Stacklies, Henning Redestig, Kevin Wright\n",
      "Dependencies:\n",
      "  - Biobase, type: depends\n",
      "  - methods, type: depends\n",
      "  - BiocGenerics, type: imports\n",
      "  - Rcpp (>= 0.11.3) type: imports\n",
      "  - MASS, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 3)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/pcaMethods.html\n",
      "\n",
      "Name: phyloseq\n",
      "Description:\n",
      "phyloseq provides a set of classes and tools to facilitate the import, storage, analysis, and graphical display of microbiome census data.\n",
      "Version: 1.42.0\n",
      "Publication date: None\n",
      "Mantainer: Paul J. McMurdie <joey711@gmail.com>\n",
      "Authors:\n",
      "Author: Paul J. McMurdie <joey711 at gmail.com>, Susan Holmes <susan at stat.stanford.edu>, with contributions from Gregory Jordan and Scott Chamberlain\n",
      "Dependencies:\n",
      "  - R (>= 3.3.0) type: depends\n",
      "  - ade4 (>= 1.7-4) type: imports\n",
      "  - ape (>= 5.0) type: imports\n",
      "  - Biobase, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - biomformat, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - cluster (>= 2.0.4) type: imports\n",
      "  - data.table (>= 1.10.4) type: imports\n",
      "  - foreach (>= 1.4.3) type: imports\n",
      "  - ggplot2 (>= 2.1.0) type: imports\n",
      "  - igraph (>= 1.0.1) type: imports\n",
      "  - methods (>= 3.3.0) type: imports\n",
      "  - multtest, type: imports\n",
      "  - plyr (>= 1.8.3) type: imports\n",
      "  - reshape2 (>= 1.4.1) type: imports\n",
      "  - scales (>= 0.4.0) type: imports\n",
      "  - vegan (>= 2.5) type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "AGPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/phyloseq.html\n",
      "\n",
      "Name: pathview\n",
      "Description:\n",
      "Pathview is a tool set for pathway based data integration and visualization. It maps and renders a wide variety of biological data on relevant pathway graphs. All users need is to supply their data and specify the target pathway. Pathview automatically downloads the pathway graph data, parses the data file, maps user data to the pathway, and render pathway graph with the mapped data. In addition, Pathview also seamlessly integrates with pathway and gene set (enrichment) analysis tools for large-scale and fully automated analysis.\n",
      "Version: 1.38.0\n",
      "Publication date: None\n",
      "Mantainer: Weijun Luo <luo_weijun@yahoo.com>\n",
      "Authors:\n",
      "Author: Weijun Luo\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - KEGGgraph, type: imports\n",
      "  - XML, type: imports\n",
      "  - Rgraphviz, type: imports\n",
      "  - graph, type: imports\n",
      "  - png, type: imports\n",
      "  - AnnotationDbi, type: imports\n",
      "  - org.Hs.eg.db, type: imports\n",
      "  - KEGGREST, type: imports\n",
      "  - methods, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>=3.0)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/pathview.html\n",
      "\n",
      "Name: biocViews\n",
      "Description:\n",
      "Infrastructure to support 'views' used to classify Bioconductor packages. 'biocViews' are directed acyclic graphs of terms from a controlled vocabulary. There are three major classifications, corresponding to 'software', 'annotation', and 'experiment data' packages.\n",
      "Version: 1.66.2\n",
      "Publication date: None\n",
      "Mantainer: Bioconductor Package Maintainer <maintainer@bioconductor.org>\n",
      "Authors:\n",
      "Author: VJ Carey <stvjc at channing.harvard.edu>, BJ Harshfield <rebjh at channing.harvard.edu>, S Falcon <sfalcon at fhcrc.org> , Sonali Arora, Lori Shepherd <lori.shepherd at roswellpark.org>\n",
      "Dependencies:\n",
      "  - R (>= 3.6.0) type: depends\n",
      "  - Biobase, type: imports\n",
      "  - graph, type: imports\n",
      "  - methods, type: imports\n",
      "  - RBGL, type: imports\n",
      "  - tools, type: imports\n",
      "  - utils, type: imports\n",
      "  - XML, type: imports\n",
      "  - RCurl, type: imports\n",
      "  - RUnit, type: imports\n",
      "  - BiocManager, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/biocViews.html\n",
      "\n",
      "Name: scran\n",
      "Description:\n",
      "Implements miscellaneous functions for interpretation of single-cell RNA-seq data. Methods are provided for assignment of cell cycle phase, detection of highly variable and significantly correlated genes, identification of marker genes, and other common tasks in routine single-cell analysis workflows.\n",
      "Version: 1.26.2\n",
      "Publication date: None\n",
      "Mantainer: Aaron Lun <infinite.monkeys.with.keyboards@gmail.com>\n",
      "Authors:\n",
      "Author: Aaron Lun [aut, cre], Karsten Bach [aut], Jong Kyoung Kim [ctb], Antonio Scialdone [ctb]\n",
      "Dependencies:\n",
      "  - SingleCellExperiment, type: depends\n",
      "  - scuttle, type: depends\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - Rcpp, type: imports\n",
      "  - stats, type: imports\n",
      "  - methods, type: imports\n",
      "  - utils, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - edgeR, type: imports\n",
      "  - limma, type: imports\n",
      "  - igraph, type: imports\n",
      "  - statmod, type: imports\n",
      "  - DelayedArray, type: imports\n",
      "  - DelayedMatrixStats, type: imports\n",
      "  - BiocSingular, type: imports\n",
      "  - bluster, type: imports\n",
      "  - metapod, type: imports\n",
      "  - dqrng, type: imports\n",
      "  - beachmat, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/scran.html\n",
      "\n",
      "Name: DNAcopy\n",
      "Description:\n",
      "Implements the circular binary segmentation (CBS) algorithm to segment DNA copy number data and identify genomic regions with abnormal copy number.\n",
      "Version: 1.72.3\n",
      "Publication date: None\n",
      "Mantainer: Venkatraman E. Seshan <seshanv@mskcc.org>\n",
      "Authors:\n",
      "Author: Venkatraman E. Seshan, Adam Olshen\n",
      "Dependencies:\n",
      "None\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/DNAcopy.html\n",
      "\n",
      "Name: bluster\n",
      "Description:\n",
      "Wraps common clustering algorithms in an easily extended S4 framework. Backends are implemented for hierarchical, k-means and graph-based clustering. Several utilities are also provided to compare and evaluate clustering results.\n",
      "Version: 1.8.0\n",
      "Publication date: None\n",
      "Mantainer: Aaron Lun <infinite.monkeys.with.keyboards@gmail.com>\n",
      "Authors:\n",
      "Author: Aaron Lun [aut, cre], Stephanie Hicks [ctb]\n",
      "Dependencies:\n",
      "  - stats, type: imports\n",
      "  - methods, type: imports\n",
      "  - utils, type: imports\n",
      "  - cluster, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - Rcpp, type: imports\n",
      "  - igraph, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - BiocNeighbors, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/bluster.html\n",
      "\n",
      "Name: EnhancedVolcano\n",
      "Description:\n",
      "Volcano plots represent a useful way to visualise the results of differential expression analyses. Here, we present a highly-configurable function that produces publication-ready volcano plots. EnhancedVolcano will attempt to fit as many point labels in the plot window as possible, thus avoiding 'clogging' up the plot with labels that could not otherwise have been read. Other functionality allows the user to identify up to 4 different types of attributes in the same plot space via colour, shape, size, and shade parameter configurations.\n",
      "Version: 1.16.0\n",
      "Publication date: None\n",
      "Mantainer: Kevin Blighe <kevin@clinicalbioinformatics.co.uk>\n",
      "Authors:\n",
      "Author: Kevin Blighe [aut, cre], Sharmila Rana [aut], Emir Turkes [ctb], Benjamin Ostendorf [ctb], Andrea Grioni [ctb], Myles Lewis [aut]\n",
      "Dependencies:\n",
      "  - ggplot2, type: depends\n",
      "  - ggrepel, type: depends\n",
      "  - methods, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/EnhancedVolcano.html\n",
      "\n",
      "Name: metapod\n",
      "Description:\n",
      "Implements a variety of methods for combining p-values in differential analyses of genome-scale datasets. Functions can combine p-values across different tests in the same analysis (e.g., genomic windows in ChIP-seq, exons in RNA-seq) or for corresponding tests across separate analyses (e.g., replicated comparisons, effect of different treatment conditions). Support is provided for handling log-transformed input p-values, missing values and weighting where appropriate.\n",
      "Version: 1.6.0\n",
      "Publication date: None\n",
      "Mantainer: Aaron Lun <infinite.monkeys.with.keyboards@gmail.com>\n",
      "Authors:\n",
      "Author: Aaron Lun [aut, cre]\n",
      "Dependencies:\n",
      "  - Rcpp, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/metapod.html\n",
      "\n",
      "Name: Gviz\n",
      "Description:\n",
      "Genomic data analyses requires integrated visualization of known genomic information and new experimental data. Gviz uses the biomaRt and the rtracklayer packages to perform live annotation queries to Ensembl and UCSC and translates this to e.g. gene/transcript structures in viewports of the grid graphics package. This results in genomic information plotted together with your data.\n",
      "Version: 1.42.1\n",
      "Publication date: None\n",
      "Mantainer: Robert Ivanek <robert.ivanek@unibas.ch>\n",
      "Authors:\n",
      "Author: Florian Hahne [aut], Steffen Durinck [aut], Robert Ivanek [aut, cre] , Arne Mueller [aut], Steve Lianoglou [aut], Ge Tan [aut], Lance Parsons [aut], Shraddha Pai [aut], Thomas McCarthy [ctb], Felix Ernst [ctb], Mike Smith [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 4.2) type: depends\n",
      "  - methods, type: depends\n",
      "  - S4Vectors, type: depends\n",
      "  - IRanges, type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - grid, type: depends\n",
      "  - XVector, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - lattice, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - biomaRt, type: imports\n",
      "  - AnnotationDbi, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - GenomicFeatures, type: imports\n",
      "  - ensembldb, type: imports\n",
      "  - BSgenome, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - biovizBase, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - latticeExtra (>= 0.6-26) type: imports\n",
      "  - matrixStats (>= 0.8.14) type: imports\n",
      "  - GenomicAlignments, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - digest (>= 0.6.8) type: imports\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/Gviz.html\n",
      "\n",
      "Name: apeglm\n",
      "Description:\n",
      "apeglm provides Bayesian shrinkage estimators for effect sizes for a variety of GLM models, using approximation of the posterior for individual coefficients.\n",
      "Version: 1.20.0\n",
      "Publication date: None\n",
      "Mantainer: Anqi Zhu <anqizhu@live.unc.edu>\n",
      "Authors:\n",
      "Author: Anqi Zhu [aut, cre], Joshua Zitovsky [ctb], Joseph Ibrahim [aut], Michael Love [aut]\n",
      "Dependencies:\n",
      "  - emdbook, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - Rcpp, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/apeglm.html\n",
      "\n",
      "Name: tximport\n",
      "Description:\n",
      "Imports transcript-level abundance, estimated counts and transcript lengths, and summarizes into matrices for use with downstream gene-level analysis packages. Average transcript length, weighted by sample-specific transcript abundance estimates, is provided as a matrix which can be used as an offset for different expression of gene-level counts.\n",
      "Version: 1.26.1\n",
      "Publication date: None\n",
      "Mantainer: Michael Love <michaelisaiahlove@gmail.com>\n",
      "Authors:\n",
      "Author: Michael Love [cre,aut], Charlotte Soneson [aut], Mark Robinson [aut], Rob Patro [ctb], Andrew Parker Morgan [ctb], Ryan C. Thompson [ctb], Matt Shirley [ctb], Avi Srivastava [ctb]\n",
      "Dependencies:\n",
      "  - utils, type: imports\n",
      "  - stats, type: imports\n",
      "  - methods, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL (>=2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/tximport.html\n",
      "\n",
      "Name: TCGAbiolinks\n",
      "Description:\n",
      "The aim of TCGAbiolinks is : i) facilitate the GDC open-access data retrieval, ii) prepare the data using the appropriate pre-processing strategies, iii) provide the means to carry out different standard analyses and iv) to easily reproduce earlier research results. In more detail, the package provides multiple methods for analysis (e.g., differential expression analysis, identifying differentially methylated regions) and methods for visualization (e.g., survival plots, volcano plots, starburst plots) in order to easily develop complete analysis pipelines.\n",
      "Version: 2.25.3\n",
      "Publication date: None\n",
      "Mantainer: Tiago Chedraoui Silva <tiagochst@gmail.com>, Antonio Colaprico <axc1833@med.miami.edu>\n",
      "Authors:\n",
      "Author: Antonio Colaprico, Tiago Chedraoui Silva, Catharina Olsen, Luciano Garofano, Davide Garolini, Claudia Cava, Thais Sabedot, Tathiane Malta, Stefano M. Pagnotta, Isabella Castiglioni, Michele Ceccarelli, Gianluca Bontempi, Houtan Noushmehr\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - downloader (>= 0.4) type: imports\n",
      "  - grDevices, type: imports\n",
      "  - biomaRt, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - graphics, type: imports\n",
      "  - tibble, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - XML (>= 3.98.0) type: imports\n",
      "  - data.table, type: imports\n",
      "  - jsonlite (>= 1.0.0) type: imports\n",
      "  - plyr, type: imports\n",
      "  - knitr, type: imports\n",
      "  - methods, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - stringr (>= 1.0.0) type: imports\n",
      "  - IRanges, type: imports\n",
      "  - rvest (>= 0.3.0) type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - R.utils, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - TCGAbiolinksGUI.data, type: imports\n",
      "  - readr, type: imports\n",
      "  - tools, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - purrr, type: imports\n",
      "  - xml2, type: imports\n",
      "  - httr (>= 1.2.1) type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 3)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/TCGAbiolinks.html\n",
      "\n",
      "Name: batchelor\n",
      "Description:\n",
      "Implements a variety of methods for batch correction of single-cell (RNA sequencing) data. This includes methods based on detecting mutually nearest neighbors, as well as several efficient variants of linear regression of the log-expression values. Functions are also provided to perform global rescaling to remove differences in depth between batches, and to perform a principal components analysis that is robust to differences in the numbers of cells across batches.\n",
      "Version: 1.14.1\n",
      "Publication date: None\n",
      "Mantainer: Aaron Lun <infinite.monkeys.with.keyboards@gmail.com>\n",
      "Authors:\n",
      "Author: Aaron Lun [aut, cre], Laleh Haghverdi [ctb]\n",
      "Dependencies:\n",
      "  - SingleCellExperiment, type: depends\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - Rcpp, type: imports\n",
      "  - stats, type: imports\n",
      "  - methods, type: imports\n",
      "  - utils, type: imports\n",
      "  - igraph, type: imports\n",
      "  - BiocNeighbors, type: imports\n",
      "  - BiocSingular, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - DelayedArray, type: imports\n",
      "  - DelayedMatrixStats, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - scuttle, type: imports\n",
      "  - ResidualMatrix, type: imports\n",
      "  - ScaledMatrix, type: imports\n",
      "  - beachmat, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/batchelor.html\n",
      "\n",
      "Name: topGO\n",
      "Description:\n",
      "topGO package provides tools for testing GO terms while accounting for the topology of the GO graph. Different test statistics and different methods for eliminating local similarities and dependencies between GO terms can be implemented and applied.\n",
      "Version: 2.50.0\n",
      "Publication date: None\n",
      "Mantainer: Adrian Alexa <adrian.alexa@gmail.com>\n",
      "Authors:\n",
      "Author: Adrian Alexa, Jorg Rahnenfuhrer\n",
      "Dependencies:\n",
      "  - R (>= 2.10.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - BiocGenerics, type: depends\n",
      "  - graph, type: depends\n",
      "  - Biobase, type: depends\n",
      "  - GO.db, type: depends\n",
      "  - AnnotationDbi, type: depends\n",
      "  - SparseM (>= 0.73) type: depends\n",
      "  - lattice, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - DBI, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/topGO.html\n",
      "\n",
      "Name: MSnbase\n",
      "Description:\n",
      "MSnbase provides infrastructure for manipulation, processing and visualisation of mass spectrometry and proteomics data, ranging from raw to quantitative and annotated data.\n",
      "Version: 2.24.2\n",
      "Publication date: None\n",
      "Mantainer: Laurent Gatto <laurent.gatto@uclouvain.be>\n",
      "Authors:\n",
      "Author: Laurent Gatto, Johannes Rainer and Sebastian Gibb with contributions from Guangchuang Yu, Samuel Wieczorek, Vasile-Cosmin Lazar, Vladislav Petyuk, Thomas Naake, Richie Cotton, Arne Smits, Martina Fisher, Ludger Goeminne, Adriaan Sticker and Lieven Clement.\n",
      "Dependencies:\n",
      "  - R (>= 3.5) type: depends\n",
      "  - methods, type: depends\n",
      "  - BiocGenerics, type: depends\n",
      "  - Biobase, type: depends\n",
      "  - mzR, type: depends\n",
      "  - S4Vectors, type: depends\n",
      "  - ProtGenerics, type: depends\n",
      "  - MsCoreUtils, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - plyr, type: imports\n",
      "  - vsn, type: imports\n",
      "  - grid, type: imports\n",
      "  - stats4, type: imports\n",
      "  - affy, type: imports\n",
      "  - impute, type: imports\n",
      "  - pcaMethods, type: imports\n",
      "  - MALDIquant (>= 1.16) type: imports\n",
      "  - mzID, type: imports\n",
      "  - digest, type: imports\n",
      "  - lattice, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - XML, type: imports\n",
      "  - scales, type: imports\n",
      "  - MASS, type: imports\n",
      "  - Rcpp, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/MSnbase.html\n",
      "\n",
      "Name: OrganismDbi\n",
      "Description:\n",
      "The package enables a simple unified interface to several annotation packages each of which has its own schema by taking advantage of the fact that each of these packages implements a select methods.\n",
      "Version: 1.40.0\n",
      "Publication date: None\n",
      "Mantainer: Bioconductor Package Maintainer <maintainer@bioconductor.org>\n",
      "Authors:\n",
      "Author: Marc Carlson, HervÃ© PagÃ¨s, Martin Morgan, Valerie Obenchain\n",
      "Dependencies:\n",
      "  - R (>= 2.14.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - BiocGenerics, type: depends\n",
      "  - AnnotationDbi, type: depends\n",
      "  - GenomicFeatures, type: depends\n",
      "  - Biobase, type: imports\n",
      "  - BiocManager, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - graph, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - RBGL, type: imports\n",
      "  - DBI, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - stats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/OrganismDbi.html\n",
      "\n",
      "Name: ConsensusClusterPlus\n",
      "Description:\n",
      "algorithm for determining cluster count and membership by stability evidence in unsupervised analysis\n",
      "Version: 1.62.0\n",
      "Publication date: None\n",
      "Mantainer: Matt Wilkerson <mdwilkerson@outlook.com>\n",
      "Authors:\n",
      "Author: Matt Wilkerson <mdwilkerson at outlook.com>, Peter Waltman <waltman at soe.ucsc.edu>\n",
      "Dependencies:\n",
      "  - Biobase, type: imports\n",
      "  - ALL, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - cluster, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL version 2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ConsensusClusterPlus.html\n",
      "\n",
      "Name: SingleR\n",
      "Description:\n",
      "Performs unbiased cell type recognition from single-cell RNA sequencing data, by leveraging reference transcriptomic datasets of pure cell types to infer the cell of origin of each single cell independently.\n",
      "Version: 2.0.0\n",
      "Publication date: None\n",
      "Mantainer: Aaron Lun <infinite.monkeys.with.keyboards@gmail.com>\n",
      "Authors:\n",
      "Author: Dvir Aran [aut, cph], Aaron Lun [ctb, cre], Daniel Bunis [ctb], Jared Andrews [ctb], Friederike DÃ¼ndar [ctb]\n",
      "Dependencies:\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - methods, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - DelayedArray, type: imports\n",
      "  - DelayedMatrixStats, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - BiocSingular, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - Rcpp, type: imports\n",
      "  - beachmat, type: imports\n",
      "  - parallel, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3 + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/SingleR.html\n",
      "\n",
      "Name: ResidualMatrix\n",
      "Description:\n",
      "Provides delayed computation of a matrix of residuals after fitting a linear model to each column of an input matrix. Also supports partial computation of residuals where selected factors are to be preserved in the output matrix. Implements a number of efficient methods for operating on the delayed matrix of residuals, most notably matrix multiplication and calculation of row/column sums or means.\n",
      "Version: 1.8.0\n",
      "Publication date: None\n",
      "Mantainer: Aaron Lun <infinite.monkeys.with.keyboards@gmail.com>\n",
      "Authors:\n",
      "Author: Aaron Lun [aut, cre, cph]\n",
      "Dependencies:\n",
      "  - methods, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - DelayedArray, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ResidualMatrix.html\n",
      "\n",
      "Name: mzR\n",
      "Description:\n",
      "mzR provides a unified API to the common file formats and parsers available for mass spectrometry data. It comes with a subset of the proteowizard library for mzXML, mzML and mzIdentML. The netCDF reading code has previously been used in XCMS.\n",
      "Version: 2.32.0\n",
      "Publication date: None\n",
      "Mantainer: Steffen Neumann <sneumann@ipb-halle.de>\n",
      "Authors:\n",
      "Author: Bernd Fischer, Steffen Neumann, Laurent Gatto, Qiang Kou, Johannes Rainer\n",
      "Dependencies:\n",
      "  - R (>= 4.0.0) type: depends\n",
      "  - Rcpp (>= 0.10.1) type: depends\n",
      "  - methods, type: depends\n",
      "  - utils, type: depends\n",
      "  - Biobase, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - ProtGenerics, type: imports\n",
      "  - ncdf4, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/mzR.html\n",
      "\n",
      "Name: MultiAssayExperiment\n",
      "Description:\n",
      "MultiAssayExperiment harmonizes data management of multiple experimental assays performed on an overlapping set of specimens. It provides a familiar Bioconductor user experience by extending concepts from SummarizedExperiment, supporting an open-ended mix of standard data classes for individual assays, and allowing subsetting by genomic ranges or rownames. Facilities are provided for reshaping data into wide and long formats for adaptability to graphing and downstream analysis.\n",
      "Version: 1.24.0\n",
      "Publication date: None\n",
      "Mantainer: Marcel Ramos <marcel.ramos@roswellpark.org>\n",
      "Authors:\n",
      "Author: Marcel Ramos [aut, cre] , Levi Waldron [aut], MultiAssay SIG [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 4.2.0) type: depends\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - methods, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - BiocBaseUtils, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - DelayedArray, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - stats, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/MultiAssayExperiment.html\n",
      "\n",
      "Name: maftools\n",
      "Description:\n",
      "Analyze and visualize Mutation Annotation Format (MAF) files from large scale sequencing studies. This package provides various functions to perform most commonly used analyses in cancer genomics and to create feature rich customizable visualzations with minimal effort.\n",
      "Version: 2.14.0\n",
      "Publication date: None\n",
      "Mantainer: Anand Mayakonda <anand_mt@hotmail.com>\n",
      "Authors:\n",
      "Author: Anand Mayakonda [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 3.3) type: depends\n",
      "  - data.table, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - methods, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - Rhtslib, type: imports\n",
      "  - survival, type: imports\n",
      "  - DNAcopy, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/maftools.html\n",
      "\n",
      "Name: illuminaio\n",
      "Description:\n",
      "Tools for parsing Illumina's microarray output files, including IDAT.\n",
      "Version: 0.40.0\n",
      "Publication date: None\n",
      "Mantainer: Kasper Daniel Hansen <kasperdanielhansen@gmail.com>\n",
      "Authors:\n",
      "Author: Keith Baggerly [aut], Henrik Bengtsson [aut], Kasper Daniel Hansen [aut, cre], Matt Ritchie [aut], Mike L. Smith [aut], Tim Triche Jr. [ctb]\n",
      "Dependencies:\n",
      "  - base64, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/illuminaio.html\n",
      "\n",
      "Name: siggenes\n",
      "Description:\n",
      "Identification of differentially expressed genes and estimation of the False Discovery Rate (FDR) using both the Significance Analysis of Microarrays (SAM) and the Empirical Bayes Analyses of Microarrays (EBAM).\n",
      "Version: 1.72.0\n",
      "Publication date: None\n",
      "Mantainer: Holger Schwender <holger.schw@gmx.de>\n",
      "Authors:\n",
      "Author: Holger Schwender\n",
      "Dependencies:\n",
      "  - Biobase, type: depends\n",
      "  - multtest, type: depends\n",
      "  - splines, type: depends\n",
      "  - methods, type: depends\n",
      "  - stats4, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - scrime (>= 1.2.5) type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/siggenes.html\n",
      "\n",
      "Name: MsCoreUtils\n",
      "Description:\n",
      "MsCoreUtils defines low-level functions for mass spectrometry data and is independent of any high-level data structures. These functions include mass spectra processing functions (noise estimation, smoothing, binning), quantitative aggregation functions (median polish, robust summarisation, ...), missing data imputation, data normalisation (quantiles, vsn, ...) as well as misc helper functions, that are used across high-level data structure within the R for Mass Spectrometry packages.\n",
      "Version: 1.10.0\n",
      "Publication date: None\n",
      "Mantainer: RforMassSpectrometry Package Maintainer <maintainer@rformassspectrometry.org>\n",
      "Authors:\n",
      "Author: RforMassSpectrometry Package Maintainer [cre], Laurent Gatto [aut] , Johannes Rainer [aut] , Sebastian Gibb [aut] , Adriaan Sticker [ctb], Sigurdur Smarason [ctb], Thomas Naake [ctb], Josep Maria Badia Aparicio [ctb] , Michael Witting [ctb] , Samuel Wieczorek [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 3.6.0) type: depends\n",
      "  - methods, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - MASS, type: imports\n",
      "  - stats, type: imports\n",
      "  - clue, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/MsCoreUtils.html\n",
      "\n",
      "Name: mixOmics\n",
      "Description:\n",
      "Multivariate methods are well suited to large omics data sets where the number of variables (e.g. genes, proteins, metabolites) is much larger than the number of samples (patients, cells, mice). They have the appealing properties of reducing the dimension of the data by using instrumental variables (components), which are defined as combinations of all variables. Those components are then used to produce useful graphical outputs that enable better understanding of the relationships and correlation structures between the different data sets that are integrated. mixOmics offers a wide range of multivariate methods for the exploration and integration of biological datasets with a particular focus on variable selection. The package proposes several sparse multivariate models we have developed to identify the key variables that are highly correlated, and/or explain the biological outcome of interest. The data that can be analysed with mixOmics may come from high throughput sequencing technologies, such as omics data (transcriptomics, metabolomics, proteomics, metagenomics etc) but also beyond the realm of omics (e.g. spectral imaging). The methods implemented in mixOmics can also handle missing values without having to delete entire rows with missing data. A non exhaustive list of methods include variants of generalised Canonical Correlation Analysis, sparse Partial Least Squares and sparse Discriminant Analysis. Recently we implemented integrative methods to combine multiple data sets: N-integration with variants of Generalised Canonical Correlation Analysis and P-integration with variants of multi-group Partial Least Squares.\n",
      "Version: 6.22.0\n",
      "Publication date: None\n",
      "Mantainer: Al J Abadi <al.jal.abadi@gmail.com>\n",
      "Authors:\n",
      "Author: Kim-Anh Le Cao [aut], Florian Rohart [aut], Ignacio Gonzalez [aut], Sebastien Dejean [aut], Al J Abadi [ctb, cre], Benoit Gautier [ctb], Francois Bartolo [ctb], Pierre Monget [ctb], Jeff Coquery [ctb], FangZou Yao [ctb], Benoit Liquet [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - MASS, type: depends\n",
      "  - lattice, type: depends\n",
      "  - ggplot2, type: depends\n",
      "  - igraph, type: imports\n",
      "  - ellipse, type: imports\n",
      "  - corpcor, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - parallel, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - methods, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - rARPACK, type: imports\n",
      "  - gridExtra, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - ggrepel, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/mixOmics.html\n",
      "\n",
      "Name: monocle\n",
      "Description:\n",
      "Monocle performs differential expression and time-series analysis for single-cell expression experiments. It orders individual cells according to progress through a biological process, without knowing ahead of time which genes define progress through that process. Monocle also performs differential expression analysis, clustering, visualization, and other useful tasks on single cell expression data. It is designed to work with RNA-Seq and qPCR data, but could be used with other types as well.\n",
      "Version: 2.26.0\n",
      "Publication date: None\n",
      "Mantainer: Cole Trapnell <coletrap@uw.edu>\n",
      "Authors:\n",
      "Author: Cole Trapnell\n",
      "Dependencies:\n",
      "  - R (>= 2.10.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - Matrix (>= 1.2-6) type: depends\n",
      "  - Biobase, type: depends\n",
      "  - ggplot2 (>= 1.0.0) type: depends\n",
      "  - VGAM (>= 1.0-6) type: depends\n",
      "  - DDRTree (>= 0.1.4) type: depends\n",
      "  - parallel, type: imports\n",
      "  - igraph (>= 1.0.1) type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - HSMMSingleCell, type: imports\n",
      "  - plyr, type: imports\n",
      "  - cluster, type: imports\n",
      "  - combinat, type: imports\n",
      "  - fastICA, type: imports\n",
      "  - grid, type: imports\n",
      "  - irlba (>= 2.0.0) type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - Rtsne, type: imports\n",
      "  - MASS, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - leidenbase (>= 0.1.9) type: imports\n",
      "  - limma, type: imports\n",
      "  - tibble, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - qlcMatrix, type: imports\n",
      "  - pheatmap, type: imports\n",
      "  - stringr, type: imports\n",
      "  - proxy, type: imports\n",
      "  - slam, type: imports\n",
      "  - viridis, type: imports\n",
      "  - stats, type: imports\n",
      "  - biocViews, type: imports\n",
      "  - RANN (>= 2.5) type: imports\n",
      "  - Rcpp (>= 0.12.0) type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/monocle.html\n",
      "\n",
      "Name: mzID\n",
      "Description:\n",
      "A parser for mzIdentML files implemented using the XML package. The parser tries to be general and able to handle all types of mzIdentML files with the drawback of having less 'pretty' output than a vendor specific parser. Please contact the maintainer with any problems and supply an mzIdentML file so the problems can be fixed quickly.\n",
      "Version: 1.36.0\n",
      "Publication date: None\n",
      "Mantainer: Laurent Gatto <laurent.gatto@uclouvain.be>\n",
      "Authors:\n",
      "Author: Laurent Gatto [ctb, cre] , Thomas Pedersen [aut] , Vladislav Petyuk [ctb]\n",
      "Dependencies:\n",
      "  - methods, type: depends\n",
      "  - XML, type: imports\n",
      "  - plyr, type: imports\n",
      "  - parallel, type: imports\n",
      "  - doParallel, type: imports\n",
      "  - foreach, type: imports\n",
      "  - iterators, type: imports\n",
      "  - ProtGenerics, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/mzID.html\n",
      "\n",
      "Name: AnnotationForge\n",
      "Description:\n",
      "Provides code for generating Annotation packages and their databases. Packages produced are intended to be used with AnnotationDbi.\n",
      "Version: 1.40.1\n",
      "Publication date: None\n",
      "Mantainer: Bioconductor Package Maintainer <maintainer@bioconductor.org>\n",
      "Authors:\n",
      "Author: Marc Carlson, HervÃ© PagÃ¨s\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - utils, type: depends\n",
      "  - BiocGenerics, type: depends\n",
      "  - Biobase, type: depends\n",
      "  - AnnotationDbi, type: depends\n",
      "  - DBI, type: imports\n",
      "  - RSQLite, type: imports\n",
      "  - XML, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - RCurl, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/AnnotationForge.html\n",
      "\n",
      "Name: cytolib\n",
      "Description:\n",
      "This package provides the core data structure and API to represent and interact with the gated cytometry data.\n",
      "Version: 2.10.1\n",
      "Publication date: None\n",
      "Mantainer: Mike Jiang <mike@ozette.ai>\n",
      "Authors:\n",
      "Author: Mike Jiang\n",
      "Dependencies:\n",
      "  - R (>= 3.4) type: depends\n",
      "  - RProtoBufLib, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "AGPL-3.0-only\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/cytolib.html\n",
      "\n",
      "Name: EBImage\n",
      "Description:\n",
      "EBImage provides general purpose functionality for image processing and analysis. In the context of (high-throughput) microscopy-based cellular assays, EBImage offers tools to segment cells and extract quantitative cellular descriptors. This allows the automation of such tasks using the R programming language and facilitates the use of other tools in the R environment for signal processing, statistical modeling, machine learning and visualization with image data.\n",
      "Version: 4.40.0\n",
      "Publication date: None\n",
      "Mantainer: Andrzej OleÅ <andrzej.oles@gmail.com>\n",
      "Authors:\n",
      "Author: Andrzej OleÅ, Gregoire Pau, Mike Smith, Oleg Sklyar, Wolfgang Huber, with contributions from Joseph Barry and Philip A. Marais\n",
      "Dependencies:\n",
      "  - methods, type: depends\n",
      "  - BiocGenerics, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - stats, type: imports\n",
      "  - abind, type: imports\n",
      "  - tiff, type: imports\n",
      "  - jpeg, type: imports\n",
      "  - png, type: imports\n",
      "  - locfit, type: imports\n",
      "  - fftwtools (>= 0.9-7) type: imports\n",
      "  - utils, type: imports\n",
      "  - htmltools, type: imports\n",
      "  - htmlwidgets, type: imports\n",
      "  - RCurl, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/EBImage.html\n",
      "\n",
      "Name: flowCore\n",
      "Description:\n",
      "Provides S4 data structures and basic functions to deal with flow cytometry data.\n",
      "Version: 2.10.0\n",
      "Publication date: None\n",
      "Mantainer: Mike Jiang <mike@ozette.com>\n",
      "Authors:\n",
      "Author: B Ellis [aut], Perry Haaland [aut], Florian Hahne [aut], Nolwenn Le Meur [aut], Nishant Gopalakrishnan [aut], Josef Spidlen [aut], Mike Jiang [aut, cre], Greg Finak [aut], Samuel Granjeaud [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - Biobase, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - stats4, type: imports\n",
      "  - Rcpp, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - cytolib, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/flowCore.html\n",
      "\n",
      "Name: bumphunter\n",
      "Description:\n",
      "Tools for finding bumps in genomic data\n",
      "Version: 1.40.0\n",
      "Publication date: None\n",
      "Mantainer: Tamilselvi Guharaj <selvi@ds.dfci.harvard.edu>\n",
      "Authors:\n",
      "Author: Rafael A. Irizarry [aut], Martin Aryee [aut], Kasper Daniel Hansen [aut], Hector Corrada Bravo [aut], Shan Andrews [ctb], Andrew E. Jaffe [ctb], Harris Jaffee [ctb], Leonardo Collado-Torres [ctb], Tamilselvi Guharaj [cre]\n",
      "Dependencies:\n",
      "  - R (>= 3.5) type: depends\n",
      "  - S4Vectors, type: depends\n",
      "  - IRanges, type: depends\n",
      "  - GenomeInfoDb, type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - foreach, type: depends\n",
      "  - iterators, type: depends\n",
      "  - methods, type: depends\n",
      "  - parallel, type: depends\n",
      "  - locfit, type: depends\n",
      "  - matrixStats, type: imports\n",
      "  - limma, type: imports\n",
      "  - doRNG, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - utils, type: imports\n",
      "  - GenomicFeatures, type: imports\n",
      "  - AnnotationDbi, type: imports\n",
      "  - stats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/bumphunter.html\n",
      "\n",
      "Name: graphite\n",
      "Description:\n",
      "Graph objects from pathway topology derived from KEGG, Panther, PathBank, PharmGKB, Reactome SMPDB and WikiPathways databases.\n",
      "Version: 1.44.0\n",
      "Publication date: None\n",
      "Mantainer: Gabriele Sales <gabriele.sales@unipd.it>\n",
      "Authors:\n",
      "Author: Gabriele Sales [cre], Enrica Calura [aut], Chiara Romualdi [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.2) type: depends\n",
      "  - methods, type: depends\n",
      "  - AnnotationDbi, type: imports\n",
      "  - graph, type: imports\n",
      "  - httr, type: imports\n",
      "  - rappdirs, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - graphics, type: imports\n",
      "  - rlang, type: imports\n",
      "  - purrr, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "AGPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/graphite.html\n",
      "\n",
      "Name: dada2\n",
      "Description:\n",
      "The dada2 package infers exact amplicon sequence variants (ASVs) from high-throughput amplicon sequencing data, replacing the coarser and less accurate OTU clustering approach. The dada2 pipeline takes as input demultiplexed fastq files, and outputs the sequence variants and their sample-wise abundances after removing substitution and chimera errors. Taxonomic classification is available via a native implementation of the RDP naive Bayesian classifier, and species-level assignment to 16S rRNA gene fragments by exact matching.\n",
      "Version: 1.26.0\n",
      "Publication date: None\n",
      "Mantainer: Benjamin Callahan <benjamin.j.callahan@gmail.com>\n",
      "Authors:\n",
      "Author: Benjamin Callahan <benjamin.j.callahan at gmail.com>, Paul McMurdie, Susan Holmes\n",
      "Dependencies:\n",
      "  - R (>= 3.4.0) type: depends\n",
      "  - Rcpp (>= 0.12.0) type: depends\n",
      "  - methods (>= 3.4.0) type: depends\n",
      "  - Biostrings, type: imports\n",
      "  - ggplot2 (>= 2.1.0) type: imports\n",
      "  - reshape2 (>= 1.4.1) type: imports\n",
      "  - ShortRead, type: imports\n",
      "  - RcppParallel (>= 4.3.0) type: imports\n",
      "  - parallel (>= 3.2.0) type: imports\n",
      "  - IRanges, type: imports\n",
      "  - XVector, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/dada2.html\n",
      "\n",
      "Name: RProtoBufLib\n",
      "Description:\n",
      "This package provides the headers and static library of Protocol buffers for other R packages to compile and link against.\n",
      "Version: 2.10.0\n",
      "Publication date: None\n",
      "Mantainer: Mike Jiang <mike@ozette.ai>\n",
      "Authors:\n",
      "Author: Mike Jiang\n",
      "Dependencies:\n",
      "None\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "BSD_3_clause\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/RProtoBufLib.html\n",
      "\n",
      "Name: Rsubread\n",
      "Description:\n",
      "Alignment, quantification and analysis of RNA sequencing data (including both bulk RNA-seq and scRNA-seq) and DNA sequenicng data (including ATAC-seq, ChIP-seq, WGS, WES etc). Includes functionality for read mapping, read counting, SNP calling, structural variant detection and gene fusion discovery. Can be applied to all major sequencing techologies and to both short and long sequence reads.\n",
      "Version: 2.12.2\n",
      "Publication date: None\n",
      "Mantainer: Wei Shi <wei.shi@onjcri.org.au>, Yang Liao <yang.liao@onjcri.org.au> and Gordon K Smyth <smyth@wehi.edu.au>\n",
      "Authors:\n",
      "Author: Wei Shi, Yang Liao and Gordon K Smyth with contributions from Jenny Dai\n",
      "Dependencies:\n",
      "  - grDevices, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - Matrix, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>=3)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/Rsubread.html\n",
      "\n",
      "Name: DirichletMultinomial\n",
      "Description:\n",
      "Dirichlet-multinomial mixture models can be used to describe variability in microbial metagenomic data. This package is an interface to code originally made available by Holmes, Harris, and Quince, 2012, PLoS ONE 7(2): 1-15, as discussed further in the man page for this package, ?DirichletMultinomial.\n",
      "Version: 1.40.0\n",
      "Publication date: None\n",
      "Mantainer: Martin Morgan <martin.morgan@roswellpark.org>\n",
      "Authors:\n",
      "Author: Martin Morgan <martin.morgan at roswellpark.org>\n",
      "Dependencies:\n",
      "  - S4Vectors, type: depends\n",
      "  - IRanges, type: depends\n",
      "  - stats4, type: imports\n",
      "  - methods, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/DirichletMultinomial.html\n",
      "\n",
      "Name: seqLogo\n",
      "Description:\n",
      "seqLogo takes the position weight matrix of a DNA sequence motif and plots the corresponding sequence logo as introduced by Schneider and Stephens (1990).\n",
      "Version: 1.64.0\n",
      "Publication date: None\n",
      "Mantainer: Robert Ivanek <robert.ivanek@unibas.ch>\n",
      "Authors:\n",
      "Author: Oliver Bembom [aut], Robert Ivanek [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.2) type: depends\n",
      "  - methods, type: depends\n",
      "  - grid, type: depends\n",
      "  - stats4, type: imports\n",
      "  - grDevices, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/seqLogo.html\n",
      "\n",
      "Name: DropletUtils\n",
      "Description:\n",
      "Provides a number of utility functions for handling single-cell (RNA-seq) data from droplet technologies such as 10X Genomics. This includes data loading from count matrices or molecule information files, identification of cells from empty droplets, removal of barcode-swapped pseudo-cells, and downsampling of the count matrix.\n",
      "Version: 1.18.1\n",
      "Publication date: None\n",
      "Mantainer: Jonathan Griffiths <jonathan.griffiths.94@gmail.com>\n",
      "Authors:\n",
      "Author: Aaron Lun [aut], Jonathan Griffiths [ctb, cre], Davis McCarthy [ctb], Dongze He [ctb], Rob Patro [ctb]\n",
      "Dependencies:\n",
      "  - SingleCellExperiment, type: depends\n",
      "  - utils, type: imports\n",
      "  - stats, type: imports\n",
      "  - methods, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - Rcpp, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - DelayedArray, type: imports\n",
      "  - DelayedMatrixStats, type: imports\n",
      "  - HDF5Array, type: imports\n",
      "  - rhdf5, type: imports\n",
      "  - edgeR, type: imports\n",
      "  - R.utils, type: imports\n",
      "  - dqrng, type: imports\n",
      "  - beachmat, type: imports\n",
      "  - scuttle, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/DropletUtils.html\n",
      "\n",
      "Name: Category\n",
      "Description:\n",
      "A collection of tools for performing category (gene set enrichment) analysis.\n",
      "Version: 2.64.0\n",
      "Publication date: None\n",
      "Mantainer: Bioconductor Package Maintainer <maintainer@bioconductor.org>\n",
      "Authors:\n",
      "Author: Robert Gentleman [aut], Seth Falcon [ctb], Deepayan Sarkar [ctb], Robert Castelo [ctb], Bioconductor Package Maintainer [cre]\n",
      "Dependencies:\n",
      "  - methods, type: depends\n",
      "  - stats4, type: depends\n",
      "  - BiocGenerics, type: depends\n",
      "  - AnnotationDbi, type: depends\n",
      "  - Biobase, type: depends\n",
      "  - Matrix, type: depends\n",
      "  - utils, type: imports\n",
      "  - stats, type: imports\n",
      "  - graph, type: imports\n",
      "  - RBGL, type: imports\n",
      "  - GSEABase, type: imports\n",
      "  - genefilter, type: imports\n",
      "  - annotate, type: imports\n",
      "  - DBI, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/Category.html\n",
      "\n",
      "Name: minfi\n",
      "Description:\n",
      "Tools to analyze & visualize Illumina Infinium methylation arrays.\n",
      "Version: 1.44.0\n",
      "Publication date: None\n",
      "Mantainer: Kasper Daniel Hansen <kasperdanielhansen@gmail.com>\n",
      "Authors:\n",
      "Author: Kasper Daniel Hansen [cre, aut], Martin Aryee [aut], Rafael A. Irizarry [aut], Andrew E. Jaffe [ctb], Jovana Maksimovic [ctb], E. Andres Houseman [ctb], Jean-Philippe Fortin [ctb], Tim Triche [ctb], Shan V. Andrews [ctb], Peter F. Hickey [ctb]\n",
      "Dependencies:\n",
      "  - methods, type: depends\n",
      "  - BiocGenerics, type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - Biostrings, type: depends\n",
      "  - bumphunter, type: depends\n",
      "  - S4Vectors, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - beanplot, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - lattice, type: imports\n",
      "  - nor1mix, type: imports\n",
      "  - siggenes, type: imports\n",
      "  - limma, type: imports\n",
      "  - preprocessCore, type: imports\n",
      "  - illuminaio, type: imports\n",
      "  - DelayedMatrixStats, type: imports\n",
      "  - mclust, type: imports\n",
      "  - genefilter, type: imports\n",
      "  - nlme, type: imports\n",
      "  - reshape, type: imports\n",
      "  - MASS, type: imports\n",
      "  - quadprog, type: imports\n",
      "  - data.table, type: imports\n",
      "  - GEOquery, type: imports\n",
      "  - stats, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - utils, type: imports\n",
      "  - DelayedArray, type: imports\n",
      "  - HDF5Array, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/minfi.html\n",
      "\n",
      "Name: ggbio\n",
      "Description:\n",
      "The ggbio package extends and specializes the grammar of graphics for biological data. The graphics are designed to answer common scientific questions, in particular those often asked of high throughput genomics data. All core Bioconductor data structures are supported, where appropriate. The package supports detailed views of particular genomic regions, as well as genome-wide overviews. Supported overviews include ideograms and grand linear views. High-level plots include sequence fragment length, edge-linked interval to data view, mismatch pileup, and several splicing summaries.\n",
      "Version: 1.46.0\n",
      "Publication date: None\n",
      "Mantainer: Michael Lawrence <michafla@gene.com>\n",
      "Authors:\n",
      "Author: Tengfei Yin [aut], Michael Lawrence [aut, ths, cre], Dianne Cook [aut, ths], Johannes Rainer [ctb]\n",
      "Dependencies:\n",
      "  - methods, type: depends\n",
      "  - BiocGenerics, type: depends\n",
      "  - ggplot2 (>= 1.0.0) type: depends\n",
      "  - grid, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - gridExtra, type: imports\n",
      "  - scales, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - gtable, type: imports\n",
      "  - Hmisc, type: imports\n",
      "  - biovizBase, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - GenomicAlignments, type: imports\n",
      "  - BSgenome, type: imports\n",
      "  - VariantAnnotation, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - GenomicFeatures, type: imports\n",
      "  - OrganismDbi, type: imports\n",
      "  - GGally, type: imports\n",
      "  - ensembldb, type: imports\n",
      "  - AnnotationDbi, type: imports\n",
      "  - AnnotationFilter, type: imports\n",
      "  - rlang, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ggbio.html\n",
      "\n",
      "Name: DECIPHER\n",
      "Description:\n",
      "A toolset for deciphering and managing biological sequences.\n",
      "Version: 2.26.0\n",
      "Publication date: None\n",
      "Mantainer: Erik Wright <eswright@pitt.edu>\n",
      "Authors:\n",
      "Author: Erik Wright\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - Biostrings, type: depends\n",
      "  - RSQLite (>= 1.1) type: depends\n",
      "  - stats, type: depends\n",
      "  - parallel, type: depends\n",
      "  - methods, type: imports\n",
      "  - DBI, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - XVector, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/DECIPHER.html\n",
      "\n",
      "Name: regioneR\n",
      "Description:\n",
      "regioneR offers a statistical framework based on customizable permutation tests to assess the association between genomic region sets and other genomic features.\n",
      "Version: 1.30.0\n",
      "Publication date: None\n",
      "Mantainer: Bernat Gel <bgel@imppc.org>\n",
      "Authors:\n",
      "Author: Anna Diez-Villanueva <adiez at iconcologia.net>, Roberto Malinverni <roberto.malinverni at gmail.com> and Bernat Gel <bgel at igtp.cat>\n",
      "Dependencies:\n",
      "  - GenomicRanges, type: depends\n",
      "  - memoise, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - BSgenome, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - parallel, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - methods, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - tools, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/regioneR.html\n",
      "\n",
      "Name: ReactomePA\n",
      "Description:\n",
      "This package provides functions for pathway analysis based on REACTOME pathway database. It implements enrichment analysis, gene set enrichment analysis and several functions for visualization.\n",
      "Version: 1.42.0\n",
      "Publication date: None\n",
      "Mantainer: Guangchuang Yu <guangchuangyu@gmail.com>\n",
      "Authors:\n",
      "Author: Guangchuang Yu [aut, cre], Vladislav Petyuk [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 3.4.0) type: depends\n",
      "  - AnnotationDbi, type: imports\n",
      "  - DOSE, type: imports\n",
      "  - enrichplot, type: imports\n",
      "  - ggplot2 (>= 3.3.5) type: imports\n",
      "  - ggraph, type: imports\n",
      "  - reactome.db, type: imports\n",
      "  - igraph, type: imports\n",
      "  - graphite, type: imports\n",
      "  - gson, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ReactomePA.html\n",
      "\n",
      "Name: GOstats\n",
      "Description:\n",
      "A set of tools for interacting with GO and microarray data. A variety of basic manipulation tools for graphs, hypothesis testing and other simple calculations.\n",
      "Version: 2.64.0\n",
      "Publication date: None\n",
      "Mantainer: Bioconductor Package Maintainer <maintainer@bioconductor.org>\n",
      "Authors:\n",
      "Author: Robert Gentleman [aut], Seth Falcon [ctb], Robert Castelo [ctb], Bioconductor Package Maintainer [cre]\n",
      "Dependencies:\n",
      "  - R (>= 2.10) type: depends\n",
      "  - Biobase, type: depends\n",
      "  - Category, type: depends\n",
      "  - graph, type: depends\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - stats4, type: imports\n",
      "  - AnnotationDbi, type: imports\n",
      "  - GO.db, type: imports\n",
      "  - RBGL, type: imports\n",
      "  - annotate, type: imports\n",
      "  - AnnotationForge, type: imports\n",
      "  - Rgraphviz, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/GOstats.html\n",
      "\n",
      "Name: gdsfmt\n",
      "Description:\n",
      "Provides a high-level R interface to CoreArray Genomic Data Structure (GDS) data files. GDS is portable across platforms with hierarchical structure to store multiple scalable array-oriented data sets with metadata information. It is suited for large-scale datasets, especially for data which are much larger than the available random-access memory. The gdsfmt package offers the efficient operations specifically designed for integers of less than 8 bits, since a diploid genotype, like single-nucleotide polymorphism (SNP), usually occupies fewer bits than a byte. Data compression and decompression are available with relatively efficient random access. It is also allowed to read a GDS file in parallel with multiple R processes supported by the package parallel.\n",
      "Version: 1.34.0\n",
      "Publication date: None\n",
      "Mantainer: Xiuwen Zheng <zhengx@u.washington.edu>\n",
      "Authors:\n",
      "Author: Xiuwen Zheng [aut, cre] , Stephanie Gogarten [ctb], Jean-loup Gailly and Mark Adler [ctb] (for the included zlib sources), Yann Collet [ctb] (for the included LZ4 sources), xz contributors [ctb] (for the included liblzma sources)\n",
      "Dependencies:\n",
      "  - R (>= 2.15.0) type: depends\n",
      "  - methods, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/gdsfmt.html\n",
      "\n",
      "Name: aroma.light\n",
      "Description:\n",
      "Methods for microarray analysis that take basic data types such as matrices and lists of vectors. These methods can be used standalone, be utilized in other packages, or be wrapped up in higher-level classes.\n",
      "Version: 3.28.0\n",
      "Publication date: None\n",
      "Mantainer: Henrik Bengtsson <henrikb@braju.com>\n",
      "Authors:\n",
      "Author: Henrik Bengtsson [aut, cre, cph], Pierre Neuvial [ctb], Aaron Lun [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 2.15.2) type: depends\n",
      "  - stats, type: imports\n",
      "  - R.methodsS3 (>= 1.7.1) type: imports\n",
      "  - R.oo (>= 1.23.0) type: imports\n",
      "  - R.utils (>= 2.9.0) type: imports\n",
      "  - matrixStats (>= 0.55.0) type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/aroma.light.html\n",
      "\n",
      "Name: snpStats\n",
      "Description:\n",
      "Classes and statistical methods for large SNP association studies. This extends the earlier snpMatrix package, allowing for uncertainty in genotypes.\n",
      "Version: 1.48.0\n",
      "Publication date: None\n",
      "Mantainer: David Clayton <dc208@cam.ac.uk>\n",
      "Authors:\n",
      "Author: David Clayton <dc208 at cam.ac.uk>\n",
      "Dependencies:\n",
      "  - R (>= 2.10.0) type: depends\n",
      "  - survival, type: depends\n",
      "  - Matrix, type: depends\n",
      "  - methods, type: depends\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - zlibbioc, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/snpStats.html\n",
      "\n",
      "Name: BiocCheck\n",
      "Description:\n",
      "BiocCheck guides maintainers through Bioconductor best practicies. It runs Bioconductor-specific package checks by searching through package code, examples, and vignettes. Maintainers are required to address all errors, warnings, and most notes produced.\n",
      "Version: 1.34.2\n",
      "Publication date: None\n",
      "Mantainer: Marcel Ramos <marcel.ramos@roswellpark.org>\n",
      "Authors:\n",
      "Author: Bioconductor Package Maintainer [aut], Lori Shepherd [aut], Daniel von Twisk [ctb], Kevin Rue [ctb], Marcel Ramos [aut, cre] , Leonardo Collado-Torres [ctb], Federico Marini [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 4.2.0) type: depends\n",
      "  - biocViews, type: imports\n",
      "  - BiocManager, type: imports\n",
      "  - stringdist, type: imports\n",
      "  - graph, type: imports\n",
      "  - httr, type: imports\n",
      "  - tools, type: imports\n",
      "  - codetools, type: imports\n",
      "  - methods, type: imports\n",
      "  - utils, type: imports\n",
      "  - knitr, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/BiocCheck.html\n",
      "\n",
      "Name: ChIPseeker\n",
      "Description:\n",
      "This package implements functions to retrieve the nearest genes around the peak, annotate genomic region of the peak, statstical methods for estimate the significance of overlap among ChIP peak data sets, and incorporate GEO database for user to compare the own dataset with those deposited in database. The comparison can be used to infer cooperative regulation and thus can be used to generate hypotheses. Several visualization functions are implemented to summarize the coverage of the peak experiment, average profile and heatmap of peaks binding to TSS regions, genomic annotation, distance to TSS, and overlap of peaks or genes.\n",
      "Version: 1.34.1\n",
      "Publication date: None\n",
      "Mantainer: Guangchuang Yu <guangchuangyu@gmail.com>\n",
      "Authors:\n",
      "Author: Guangchuang Yu [aut, cre] , Ming Li [ctb], Qianwen Wang [ctb], Yun Yan [ctb], HervÃ© PagÃ¨s [ctb], Michael Kluge [ctb], Thomas Schwarzl [ctb], Zhougeng Xu [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - AnnotationDbi, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - boot, type: imports\n",
      "  - enrichplot, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - GenomicFeatures, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - gplots, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - gtools, type: imports\n",
      "  - methods, type: imports\n",
      "  - plotrix, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - parallel, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - stats, type: imports\n",
      "  - TxDb.Hsapiens.UCSC.hg19.knownGene, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ChIPseeker.html\n",
      "\n",
      "Name: marray\n",
      "Description:\n",
      "Class definitions for two-color spotted microarray data. Fuctions for data input, diagnostic plots, normalization and quality checking.\n",
      "Version: 1.76.0\n",
      "Publication date: None\n",
      "Mantainer: Yee Hwa (Jean) Yang <jean@biostat.ucsf.edu>\n",
      "Authors:\n",
      "Author: Yee Hwa (Jean) Yang <jeany at maths.usyd.edu.au> with contributions from Agnes Paquet and Sandrine Dudoit.\n",
      "Dependencies:\n",
      "  - R (>= 2.10.0) type: depends\n",
      "  - limma, type: depends\n",
      "  - methods, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/marray.html\n",
      "\n",
      "Name: EDASeq\n",
      "Description:\n",
      "Numerical and graphical summaries of RNA-Seq read data. Within-lane normalization procedures to adjust for GC-content effect (or other gene-level effects) on read counts: loess robust local regression, global-scaling, and full-quantile normalization (Risso et al., 2011). Between-lane normalization procedures to adjust for distributional differences between lanes (e.g., sequencing depth): global-scaling and full-quantile normalization (Bullard et al., 2010).\n",
      "Version: 2.32.0\n",
      "Publication date: None\n",
      "Mantainer: Davide Risso <risso.davide@gmail.com>\n",
      "Authors:\n",
      "Author: Davide Risso [aut, cre, cph], Sandrine Dudoit [aut], Ludwig Geistlinger [ctb]\n",
      "Dependencies:\n",
      "  - Biobase, type: depends\n",
      "  - ShortRead, type: depends\n",
      "  - methods, type: imports\n",
      "  - graphics, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - aroma.light, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - biomaRt, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - AnnotationDbi, type: imports\n",
      "  - GenomicFeatures, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - BiocManager, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/EDASeq.html\n",
      "\n",
      "Name: AUCell\n",
      "Description:\n",
      "AUCell allows to identify cells with active gene sets (e.g. signatures, gene modules...) in single-cell RNA-seq data. AUCell uses the \"Area Under the Curve\" (AUC) to calculate whether a critical subset of the input gene set is enriched within the expressed genes for each cell. The distribution of AUC scores across all the cells allows exploring the relative expression of the signature. Since the scoring method is ranking-based, AUCell is independent of the gene expression units and the normalization procedure. In addition, since the cells are evaluated individually, it can easily be applied to bigger datasets, subsetting the expression matrix if needed.\n",
      "Version: 1.20.2\n",
      "Publication date: None\n",
      "Mantainer: Sara Aibar <sara.aibar@kuleuven.vib.be>\n",
      "Authors:\n",
      "Author: Sara Aibar, Stein Aerts. Laboratory of Computational Biology. VIB-KU Leuven Center for Brain & Disease Research. Leuven, Belgium.\n",
      "Dependencies:\n",
      "  - DelayedArray, type: imports\n",
      "  - DelayedMatrixStats, type: imports\n",
      "  - data.table, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - GSEABase, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - methods, type: imports\n",
      "  - mixtools, type: imports\n",
      "  - R.utils, type: imports\n",
      "  - shiny, type: imports\n",
      "  - stats, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/AUCell.html\n",
      "\n",
      "Name: oligoClasses\n",
      "Description:\n",
      "This package contains class definitions, validity checks, and initialization methods for classes used by the oligo and crlmm packages.\n",
      "Version: 1.60.0\n",
      "Publication date: None\n",
      "Mantainer: Benilton Carvalho <beniltoncarvalho@gmail.com> and Robert Scharpf <rscharpf@jhsph.edu>\n",
      "Authors:\n",
      "Author: Benilton Carvalho and Robert Scharpf\n",
      "Dependencies:\n",
      "  - R (>= 2.14) type: depends\n",
      "  - BiocGenerics, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - methods, type: imports\n",
      "  - graphics, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - affyio, type: imports\n",
      "  - foreach, type: imports\n",
      "  - BiocManager, type: imports\n",
      "  - utils, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - RSQLite, type: imports\n",
      "  - DBI, type: imports\n",
      "  - ff, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/oligoClasses.html\n",
      "\n",
      "Name: MAST\n",
      "Description:\n",
      "Methods and models for handling zero-inflated single cell assay data.\n",
      "Version: 1.24.1\n",
      "Publication date: None\n",
      "Mantainer: Andrew McDavid <Andrew_McDavid@urmc.rochester.edu>\n",
      "Authors:\n",
      "Author: Andrew McDavid [aut, cre], Greg Finak [aut], Masanao Yajima [aut]\n",
      "Dependencies:\n",
      "  - SingleCellExperiment, type: depends\n",
      "  - R (>= 3.5) type: depends\n",
      "  - Biobase, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - data.table, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - plyr, type: imports\n",
      "  - stringr, type: imports\n",
      "  - abind, type: imports\n",
      "  - methods, type: imports\n",
      "  - parallel, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - stats, type: imports\n",
      "  - stats4, type: imports\n",
      "  - graphics, type: imports\n",
      "  - utils, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - progress, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL(>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/MAST.html\n",
      "\n",
      "Name: microbiome\n",
      "Description:\n",
      "Utilities for microbiome analysis.\n",
      "Version: 1.20.0\n",
      "Publication date: None\n",
      "Mantainer: Leo Lahti <leo.lahti@iki.fi>\n",
      "Authors:\n",
      "Author: Leo Lahti [aut, cre] , Sudarshan Shetty [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.6.0) type: depends\n",
      "  - phyloseq, type: depends\n",
      "  - ggplot2, type: depends\n",
      "  - Biostrings, type: imports\n",
      "  - compositions, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - Rtsne, type: imports\n",
      "  - scales, type: imports\n",
      "  - stats, type: imports\n",
      "  - tibble, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - utils, type: imports\n",
      "  - vegan, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "BSD_2_clause + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/microbiome.html\n",
      "\n",
      "Name: TFBSTools\n",
      "Description:\n",
      "TFBSTools is a package for the analysis and manipulation of transcription factor binding sites. It includes matrices conversion between Position Frequency Matirx (PFM), Position Weight Matirx (PWM) and Information Content Matrix (ICM). It can also scan putative TFBS from sequence/alignment, query JASPAR database and provides a wrapper of de novo motif discovery software.\n",
      "Version: 1.36.0\n",
      "Publication date: None\n",
      "Mantainer: Ge Tan <ge_tan@live.com>\n",
      "Authors:\n",
      "Author: Ge Tan [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 3.2.2) type: depends\n",
      "  - Biobase, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - BSgenome, type: imports\n",
      "  - caTools (>= 1.17.1) type: imports\n",
      "  - CNEr, type: imports\n",
      "  - DirichletMultinomial, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - gtools (>= 3.5.0) type: imports\n",
      "  - grid, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - methods, type: imports\n",
      "  - DBI (>= 0.6) type: imports\n",
      "  - RSQLite (>= 1.0.0) type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - seqLogo, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - TFMPvalue (>= 0.0.5) type: imports\n",
      "  - XML (>= 3.98-1.3) type: imports\n",
      "  - XVector, type: imports\n",
      "  - parallel, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/TFBSTools.html\n",
      "\n",
      "Name: CNEr\n",
      "Description:\n",
      "Large-scale identification and advanced visualization of sets of conserved noncoding elements.\n",
      "Version: 1.34.0\n",
      "Publication date: None\n",
      "Mantainer: Ge Tan <ge_tan@live.com>\n",
      "Authors:\n",
      "Author: Ge Tan <ge_tan at live.com>\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - Biostrings, type: imports\n",
      "  - DBI (>= 0.7) type: imports\n",
      "  - RSQLite (>= 0.11.4) type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - XVector, type: imports\n",
      "  - GenomicAlignments, type: imports\n",
      "  - methods, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - readr (>= 0.2.2) type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - tools, type: imports\n",
      "  - parallel, type: imports\n",
      "  - reshape2 (>= 1.4.1) type: imports\n",
      "  - ggplot2 (>= 2.1.0) type: imports\n",
      "  - poweRlaw (>= 0.60.3) type: imports\n",
      "  - annotate, type: imports\n",
      "  - GO.db, type: imports\n",
      "  - R.utils (>= 2.3.0) type: imports\n",
      "  - KEGGREST, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2 | file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/CNEr.html\n",
      "\n",
      "Name: gcrma\n",
      "Description:\n",
      "Background adjustment using sequence information\n",
      "Version: 2.70.0\n",
      "Publication date: None\n",
      "Mantainer: Z. Wu <zwu@stat.brown.edu>\n",
      "Authors:\n",
      "Author: Jean(ZHIJIN) Wu, Rafael Irizarry with contributions from James MacDonald <jmacdon at med.umich.edu> Jeff Gentry\n",
      "Dependencies:\n",
      "  - R (>= 2.6.0) type: depends\n",
      "  - affy, type: depends\n",
      "  - graphics, type: depends\n",
      "  - methods, type: depends\n",
      "  - stats, type: depends\n",
      "  - utils, type: depends\n",
      "  - Biobase, type: imports\n",
      "  - affy, type: imports\n",
      "  - affyio, type: imports\n",
      "  - XVector, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - splines, type: imports\n",
      "  - BiocManager, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/gcrma.html\n",
      "\n",
      "Name: affxparser\n",
      "Description:\n",
      "Package for parsing Affymetrix files (CDF, CEL, CHP, BPMAP, BAR). It provides methods for fast and memory efficient parsing of Affymetrix files using the Affymetrix' Fusion SDK. Both ASCII- and binary-based files are supported. Currently, there are methods for reading chip definition file (CDF) and a cell intensity file (CEL). These files can be read either in full or in part. For example, probe signals from a few probesets can be extracted very quickly from a set of CEL files into a convenient list structure.\n",
      "Version: 1.70.0\n",
      "Publication date: None\n",
      "Mantainer: Kasper Daniel Hansen <kasperdanielhansen@gmail.com>\n",
      "Authors:\n",
      "Author: Henrik Bengtsson [aut], James Bullard [aut], Robert Gentleman [ctb], Kasper Daniel Hansen [aut, cre], Jim Hester [ctb], Martin Morgan [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 2.14.0) type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/affxparser.html\n",
      "\n",
      "Name: oligo\n",
      "Description:\n",
      "A package to analyze oligonucleotide arrays (expression/SNP/tiling/exon) at probe-level. It currently supports Affymetrix (CEL files) and NimbleGen arrays (XYS files).\n",
      "Version: 1.62.2\n",
      "Publication date: None\n",
      "Mantainer: Benilton Carvalho <benilton@unicamp.br>\n",
      "Authors:\n",
      "Author: Benilton Carvalho and Rafael Irizarry\n",
      "Dependencies:\n",
      "  - R (>= 3.2.0) type: depends\n",
      "  - BiocGenerics, type: depends\n",
      "  - oligoClasses, type: depends\n",
      "  - Biobase, type: depends\n",
      "  - Biostrings, type: depends\n",
      "  - affyio, type: imports\n",
      "  - affxparser, type: imports\n",
      "  - DBI (>= 0.3.1) type: imports\n",
      "  - ff, type: imports\n",
      "  - graphics, type: imports\n",
      "  - methods, type: imports\n",
      "  - preprocessCore, type: imports\n",
      "  - RSQLite (>= 1.0.0) type: imports\n",
      "  - splines, type: imports\n",
      "  - stats, type: imports\n",
      "  - stats4, type: imports\n",
      "  - utils, type: imports\n",
      "  - zlibbioc, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/oligo.html\n",
      "\n",
      "Name: glmGamPoi\n",
      "Description:\n",
      "Fit linear models to overdispersed count data. The package can estimate the overdispersion and fit repeated models for matrix input. It is designed to handle large input datasets as they typically occur in single cell RNA-seq experiments.\n",
      "Version: 1.10.2\n",
      "Publication date: None\n",
      "Mantainer: Constantin Ahlmann-Eltze <artjom31415@googlemail.com>\n",
      "Authors:\n",
      "Author: Constantin Ahlmann-Eltze [aut, cre] , Michael Love [ctb]\n",
      "Dependencies:\n",
      "  - Rcpp, type: imports\n",
      "  - DelayedMatrixStats, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - MatrixGenerics, type: imports\n",
      "  - DelayedArray, type: imports\n",
      "  - HDF5Array, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - SingleCellExperiment, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - splines, type: imports\n",
      "  - rlang, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/glmGamPoi.html\n",
      "\n",
      "Name: ncdfFlow\n",
      "Description:\n",
      "Provides HDF5 storage based methods and functions for manipulation of flow cytometry data.\n",
      "Version: 2.44.0\n",
      "Publication date: None\n",
      "Mantainer: Mike Jiang <mike@ozette.com>\n",
      "Authors:\n",
      "Author: Mike Jiang,Greg Finak,N. Gopalakrishnan\n",
      "Dependencies:\n",
      "  - R (>= 2.14.0) type: depends\n",
      "  - flowCore, type: depends\n",
      "  - methods, type: depends\n",
      "  - BH, type: depends\n",
      "  - Biobase, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - flowCore, type: imports\n",
      "  - zlibbioc, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "AGPL-3.0-only\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ncdfFlow.html\n",
      "\n",
      "Name: flowWorkspace\n",
      "Description:\n",
      "This package is designed to facilitate comparison of automated gating methods against manual gating done in flowJo. This package allows you to import basic flowJo workspaces into BioConductor and replicate the gating from flowJo using the flowCore functionality. Gating hierarchies, groups of samples, compensation, and transformation are performed so that the output matches the flowJo analysis.\n",
      "Version: 4.10.1\n",
      "Publication date: None\n",
      "Mantainer: Greg Finak <greg@ozette.com>, Mike Jiang <mike@ozette.com>\n",
      "Authors:\n",
      "Author: Greg Finak, Mike Jiang\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - Biobase, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - cytolib, type: imports\n",
      "  - XML, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - graph, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - stats4, type: imports\n",
      "  - utils, type: imports\n",
      "  - RBGL, type: imports\n",
      "  - tools, type: imports\n",
      "  - Rgraphviz, type: imports\n",
      "  - data.table, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - scales, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - RProtoBufLib, type: imports\n",
      "  - flowCore, type: imports\n",
      "  - ncdfFlow, type: imports\n",
      "  - DelayedArray, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "AGPL-3.0-only\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/flowWorkspace.html\n",
      "\n",
      "Name: MassSpecWavelet\n",
      "Description:\n",
      "Peak Detection in Mass Spectrometry data is one of the important preprocessing steps. The performance of peak detection affects subsequent processes, including protein identification, profile alignment and biomarker identification. Using Continuous Wavelet Transform (CWT), this package provides a reliable algorithm for peak detection that does not require any type of smoothing or previous baseline correction method, providing more consistent results for different spectra. See Author: Pan Du [aut], Warren Kibbe [aut], Simon Lin [aut], Sergio Oller Moreno [aut, cre] Maintainer: Sergio Oller Moreno <sergioller at gmail.com> Citation (from within R,   enter citation(\"MassSpecWavelet\")):InstallationTo install this package, start R (version  \"4.2\") and enter:  if (!require(\"BiocManager\", quietly = TRUE))  install.packages(\"BiocManager\")BiocManager::install(\"MassSpecWavelet\")For older versions of R, please refer to the appropriateBioconductor release.  DocumentationTo view documentation for the version of this package installed  in your system, start R and enter:browseVignettes(\"MassSpecWavelet\") HTMLR ScriptFinding local maximaHTMLR ScriptUsing the MassSpecWavelet packagePDF Reference ManualText NEWSDetailsbiocViewsImmunoOncology, MassSpectrometry, PeakDetection, Proteomics, SoftwareVersion1.64.1In Bioconductor sinceBioC 2.0 (R-2.5) (16 years)LicenseLGPL (>= 2)DependsImportsLinkingToSuggestssignal, waveslim, BiocStyle, knitr, rmarkdown, RUnit, benchSystemRequirementsEnhancesURLhttps://github.com/zeehio/MassSpecWavelet BugReportshttp://github.com/zeehio/MassSpecWavelet/issues Depends On MeImports Mecosmiq, xcmsSuggests MedownlitLinks To MeBuild Report Package ArchivesFollow   Installation instructions to use this  package in your R session.Source PackageMassSpecWavelet_1.64.1.tar.gzWindows BinaryMassSpecWavelet_1.64.1.zipmacOS Binary (x86_64)  MassSpecWavelet_1.64.1.tgzmacOS Binary (arm64)  MassSpecWavelet_1.64.1.tgzSource Repositorygit clone https://git.bioconductor.org/packages/MassSpecWaveletSource Repository (Developer Access)git clone git@git.bioconductor.org:packages/MassSpecWaveletBioc Package Browserhttps://code.bioconductor.org/browse/MassSpecWavelet/Package Short Urlhttps://bioconductor.org/packages/MassSpecWavelet/Package Downloads ReportDownload StatsOld Source Packages for BioC 3.16Source Archive\n",
      "Version: 1.64.1\n",
      "Publication date: None\n",
      "Mantainer: Sergio Oller Moreno <sergioller@gmail.com>\n",
      "Authors:\n",
      "Author: Pan Du [aut], Warren Kibbe [aut], Simon Lin [aut], Sergio Oller Moreno [aut, cre]\n",
      "Dependencies:\n",
      "None\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/MassSpecWavelet.html\n",
      "\n",
      "Name: affyPLM\n",
      "Description:\n",
      "A package that extends and improves the functionality of the base affy package. Routines that make heavy use of compiled code for speed. Central focus is on implementation of methods for fitting probe-level models and tools using these models. PLM based quality assessment tools.\n",
      "Version: 1.74.2\n",
      "Publication date: None\n",
      "Mantainer: Ben Bolstad <bmb@bmbolstad.com>\n",
      "Authors:\n",
      "Author: Ben Bolstad <bmb at bmbolstad.com>\n",
      "Dependencies:\n",
      "  - R (>= 2.6.0) type: depends\n",
      "  - BiocGenerics, type: depends\n",
      "  - affy, type: depends\n",
      "  - Biobase, type: depends\n",
      "  - gcrma, type: depends\n",
      "  - stats, type: depends\n",
      "  - preprocessCore, type: depends\n",
      "  - zlibbioc, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - methods, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/affyPLM.html\n",
      "\n",
      "Name: methylumi\n",
      "Description:\n",
      "This package provides classes for holding and manipulating Illumina methylation data. Based on eSet, it can contain MIAME information, sample information, feature information, and multiple matrices of data. An \"intelligent\" import function, methylumiR can read the Illumina text files and create a MethyLumiSet. methylumIDAT can directly read raw IDAT files from HumanMethylation27 and HumanMethylation450 microarrays. Normalization, background correction, and quality control features for GoldenGate, Infinium, and Infinium HD arrays are also included.\n",
      "Version: 2.44.0\n",
      "Publication date: None\n",
      "Mantainer: Sean Davis <seandavi@gmail.com>\n",
      "Authors:\n",
      "Author: Sean Davis, Pan Du, Sven Bilke, Tim Triche, Jr., Moiz Bootwalla\n",
      "Dependencies:\n",
      "  - Biobase, type: depends\n",
      "  - methods, type: depends\n",
      "  - R (>= 2.13) type: depends\n",
      "  - scales, type: depends\n",
      "  - reshape2, type: depends\n",
      "  - ggplot2, type: depends\n",
      "  - matrixStats, type: depends\n",
      "  - FDb.InfiniumMethylation.hg19, type: depends\n",
      "  - minfi, type: depends\n",
      "  - BiocGenerics, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - graphics, type: imports\n",
      "  - lattice, type: imports\n",
      "  - annotate, type: imports\n",
      "  - genefilter, type: imports\n",
      "  - AnnotationDbi, type: imports\n",
      "  - minfi, type: imports\n",
      "  - stats4, type: imports\n",
      "  - illuminaio, type: imports\n",
      "  - GenomicFeatures, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/methylumi.html\n",
      "\n",
      "Name: globaltest\n",
      "Description:\n",
      "The global test tests groups of covariates (or features) for association with a response variable. This package implements the test with diagnostic plots and multiple testing utilities, along with several functions to facilitate the use of this test for gene set testing of GO and KEGG terms.\n",
      "Version: 5.52.0\n",
      "Publication date: None\n",
      "Mantainer: Jelle Goeman <j.j.goeman@lumc.nl>\n",
      "Authors:\n",
      "Author: Jelle Goeman and Jan Oosting, with contributions by Livio Finos, Aldo Solari, Dominic Edelmann\n",
      "Dependencies:\n",
      "  - methods, type: depends\n",
      "  - survival, type: depends\n",
      "  - Biobase, type: imports\n",
      "  - AnnotationDbi, type: imports\n",
      "  - annotate, type: imports\n",
      "  - graphics, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/globaltest.html\n",
      "\n",
      "Name: flowViz\n",
      "Description:\n",
      "Provides visualization tools for flow cytometry data.\n",
      "Version: 1.62.0\n",
      "Publication date: None\n",
      "Mantainer: Mike Jiang <mike@ozette.com>\n",
      "Authors:\n",
      "Author: B. Ellis, R. Gentleman, F. Hahne, N. Le Meur, D. Sarkar, M. Jiang\n",
      "Dependencies:\n",
      "  - R (>= 2.7.0) type: depends\n",
      "  - flowCore, type: depends\n",
      "  - lattice, type: depends\n",
      "  - stats4, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - flowCore, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - grid, type: imports\n",
      "  - KernSmooth, type: imports\n",
      "  - lattice, type: imports\n",
      "  - latticeExtra, type: imports\n",
      "  - MASS, type: imports\n",
      "  - methods, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - hexbin, type: imports\n",
      "  - IDPmisc, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/flowViz.html\n",
      "\n",
      "Name: SNPRelate\n",
      "Description:\n",
      "Genome-wide association studies (GWAS) are widely used to investigate the genetic basis of diseases and traits, but they pose many computational challenges. We developed an R package SNPRelate to provide a binary format for single-nucleotide polymorphism (SNP) data in GWAS utilizing CoreArray Genomic Data Structure (GDS) data files. The GDS format offers the efficient operations specifically designed for integers with two bits, since a SNP could occupy only two bits. SNPRelate is also designed to accelerate two key computations on SNP data using parallel computing for multi-core symmetric multiprocessing computer architectures: Principal Component Analysis (PCA) and relatedness analysis using Identity-By-Descent measures. The SNP GDS format is also used by the GWASTools package with the support of S4 classes and generic functions. The extended GDS format is implemented in the SeqArray package to support the storage of single nucleotide variations (SNVs), insertion/deletion polymorphism (indel) and structural variation calls in whole-genome and whole-exome variant data.\n",
      "Version: 1.32.2\n",
      "Publication date: None\n",
      "Mantainer: Xiuwen Zheng <zhengx@u.washington.edu>\n",
      "Authors:\n",
      "Author: Xiuwen Zheng [aut, cre, cph] , Stephanie Gogarten [ctb], Cathy Laurie [ctb], Bruce Weir [ctb, ths]\n",
      "Dependencies:\n",
      "  - R (>= 2.15) type: depends\n",
      "  - gdsfmt, type: depends\n",
      "  - methods, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/SNPRelate.html\n",
      "\n",
      "Name: xcms\n",
      "Description:\n",
      "Framework for processing and visualization of chromatographically separated and single-spectra mass spectral data. Imports from AIA/ANDI NetCDF, mzXML, mzData and mzML files. Preprocesses data for high-throughput, untargeted analyte profiling.\n",
      "Version: 3.20.0\n",
      "Publication date: None\n",
      "Mantainer: Steffen Neumann <sneumann@ipb-halle.de>\n",
      "Authors:\n",
      "Author: Colin A. Smith [ctb], Ralf Tautenhahn [ctb], Steffen Neumann [aut, cre] , Paul Benton [ctb], Christopher Conley [ctb], Johannes Rainer [ctb] , Michael Witting [ctb], William Kumler [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 4.0.0) type: depends\n",
      "  - BiocParallel, type: depends\n",
      "  - MSnbase, type: depends\n",
      "  - mzR, type: imports\n",
      "  - methods, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - ProtGenerics, type: imports\n",
      "  - lattice, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - plyr, type: imports\n",
      "  - RANN, type: imports\n",
      "  - MassSpecWavelet, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - robustbase, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - MsCoreUtils, type: imports\n",
      "  - MsFeatures, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2) + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/xcms.html\n",
      "\n",
      "Name: basilisk\n",
      "Description:\n",
      "Installs a self-contained conda instance that is managed by the R/Bioconductor installation machinery. This aims to provide a consistent Python environment that can be used reliably by Bioconductor packages. Functions are also provided to enable smooth interoperability of multiple Python environments in a single R session.\n",
      "Version: 1.10.2\n",
      "Publication date: None\n",
      "Mantainer: Aaron Lun <infinite.monkeys.with.keyboards@gmail.com>\n",
      "Authors:\n",
      "Author: Aaron Lun [aut, cre, cph], Vince Carey [ctb]\n",
      "Dependencies:\n",
      "  - utils, type: imports\n",
      "  - methods, type: imports\n",
      "  - parallel, type: imports\n",
      "  - reticulate, type: imports\n",
      "  - dir.expiry, type: imports\n",
      "  - basilisk.utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/basilisk.html\n",
      "\n",
      "Name: Glimma\n",
      "Description:\n",
      "This package generates interactive visualisations for analysis of RNA-sequencing data using output from limma, edgeR or DESeq2 packages in an HTML page. The interactions are built on top of the popular static representations of analysis results in order to provide additional information.\n",
      "Version: 2.8.0\n",
      "Publication date: None\n",
      "Mantainer: Shian Su <su.s@wehi.edu.au>\n",
      "Authors:\n",
      "Author: Shian Su [aut, cre], Hasaru Kariyawasam [aut], Oliver Voogd [aut], Matthew Ritchie [aut], Charity Law [aut], Stuart Lee [ctb], Isaac Virshup [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 4.0.0) type: depends\n",
      "  - htmlwidgets, type: imports\n",
      "  - edgeR, type: imports\n",
      "  - DESeq2, type: imports\n",
      "  - limma, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - stats, type: imports\n",
      "  - jsonlite, type: imports\n",
      "  - methods, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/Glimma.html\n",
      "\n",
      "Name: goseq\n",
      "Description:\n",
      "Detects Gene Ontology and/or other user defined categories which are over/under represented in RNA-seq data\n",
      "Version: 1.50.0\n",
      "Publication date: None\n",
      "Mantainer: Matthew Young <my4@sanger.ac.uk>, Nadia Davidson <nadia.davidson@mcri.edu.au>\n",
      "Authors:\n",
      "Author: Matthew Young\n",
      "Dependencies:\n",
      "  - R (>= 2.11.0) type: depends\n",
      "  - BiasedUrn, type: depends\n",
      "  - geneLenDataBase, type: depends\n",
      "  - mgcv, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - AnnotationDbi, type: imports\n",
      "  - GO.db, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/goseq.html\n",
      "\n",
      "Name: basilisk.utils\n",
      "Description:\n",
      "Implements utilities for installation of the basilisk package, primarily for creation of the underlying Conda instance. This allows us to avoid re-writing the same R code in both the configure script (for centrally administered R installations) and in the lazy installation mechanism (for distributed package binaries). It is highly unlikely that developers - or, heaven forbid, end-users! - will need to interact with this package directly; they should be using the basilisk package instead.\n",
      "Version: 1.10.0\n",
      "Publication date: None\n",
      "Mantainer: Aaron Lun <infinite.monkeys.with.keyboards@gmail.com>\n",
      "Authors:\n",
      "Author: Aaron Lun [aut, cre, cph]\n",
      "Dependencies:\n",
      "  - utils, type: imports\n",
      "  - methods, type: imports\n",
      "  - tools, type: imports\n",
      "  - dir.expiry, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/basilisk.utils.html\n",
      "\n",
      "Name: ggcyto\n",
      "Description:\n",
      "With the dedicated fortify method implemented for flowSet, ncdfFlowSet and GatingSet classes, both raw and gated flow cytometry data can be plotted directly with ggplot. ggcyto wrapper and some customed layers also make it easy to add gates and population statistics to the plot.\n",
      "Version: 1.26.4\n",
      "Publication date: None\n",
      "Mantainer: Mike Jiang <mike@ozette.com>\n",
      "Authors:\n",
      "Author: Mike Jiang\n",
      "Dependencies:\n",
      "  - methods, type: depends\n",
      "  - ggplot2 (>= 3.3.0) type: depends\n",
      "  - flowCore, type: depends\n",
      "  - ncdfFlow, type: depends\n",
      "  - flowWorkspace, type: depends\n",
      "  - plyr, type: imports\n",
      "  - scales, type: imports\n",
      "  - hexbin, type: imports\n",
      "  - data.table, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - gridExtra, type: imports\n",
      "  - rlang, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ggcyto.html\n",
      "\n",
      "Name: bsseq\n",
      "Description:\n",
      "A collection of tools for analyzing and visualizing bisulfite sequencing data.\n",
      "Version: 1.34.0\n",
      "Publication date: None\n",
      "Mantainer: Kasper Daniel Hansen <kasperdanielhansen@gmail.com>\n",
      "Authors:\n",
      "Author: Kasper Daniel Hansen [aut, cre], Peter Hickey [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - BiocGenerics, type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - IRanges, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - scales, type: imports\n",
      "  - stats, type: imports\n",
      "  - graphics, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - locfit, type: imports\n",
      "  - gtools, type: imports\n",
      "  - data.table (>= 1.11.8) type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - R.utils (>= 2.0.0) type: imports\n",
      "  - DelayedMatrixStats, type: imports\n",
      "  - permute, type: imports\n",
      "  - limma, type: imports\n",
      "  - DelayedArray, type: imports\n",
      "  - Rcpp, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - BSgenome, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - utils, type: imports\n",
      "  - HDF5Array, type: imports\n",
      "  - rhdf5, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/bsseq.html\n",
      "\n",
      "Name: msa\n",
      "Description:\n",
      "The 'msa' package provides a unified R/Bioconductor interface to the multiple sequence alignment algorithms ClustalW, ClustalOmega, and Muscle. All three algorithms are integrated in the package, therefore, they do not depend on any external software tools and are available for all major platforms. The multiple sequence alignment algorithms are complemented by a function for pretty-printing multiple sequence alignments using the LaTeX package TeXshade.\n",
      "Version: 1.30.1\n",
      "Publication date: None\n",
      "Mantainer: Ulrich Bodenhofer <bodenhofer@bioinf.jku.at>\n",
      "Authors:\n",
      "Author: Enrico Bonatesta, Christoph Horejs-Kainrath, Ulrich Bodenhofer\n",
      "Dependencies:\n",
      "  - R (>= 3.3.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - Biostrings, type: depends\n",
      "  - Rcpp (>= 0.11.1) type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - tools, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/msa.html\n",
      "\n",
      "Name: InteractionSet\n",
      "Description:\n",
      "Provides the GInteractions, InteractionSet and ContactMatrix objects and associated methods for storing and manipulating genomic interaction data from Hi-C and ChIA-PET experiments.\n",
      "Version: 1.26.1\n",
      "Publication date: None\n",
      "Mantainer: Aaron Lun <infinite.monkeys.with.keyboards@gmail.com>\n",
      "Authors:\n",
      "Author: Aaron Lun [aut, cre], Malcolm Perry [aut], Elizabeth Ing-Simmons [aut]\n",
      "Dependencies:\n",
      "  - GenomicRanges, type: depends\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - methods, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - Rcpp, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/InteractionSet.html\n",
      "\n",
      "Name: dir.expiry\n",
      "Description:\n",
      "Implements an expiration system for access to versioned directories. Directories that have not been accessed by a registered function within a certain time frame are deleted. This aims to reduce disk usage by eliminating obsolete caches generated by old versions of packages.\n",
      "Version: 1.6.0\n",
      "Publication date: None\n",
      "Mantainer: Aaron Lun <infinite.monkeys.with.keyboards@gmail.com>\n",
      "Authors:\n",
      "Author: Aaron Lun [aut, cre]\n",
      "Dependencies:\n",
      "  - utils, type: imports\n",
      "  - filelock, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/dir.expiry.html\n",
      "\n",
      "Name: systemPipeR\n",
      "Description:\n",
      "R package for building and running automated end-to-end analysis workflows for a wide range of next generation sequence (NGS) applications such as RNA-Seq, ChIP-Seq, VAR-Seq and Ribo-Seq. Important features include a uniform workflow interface across different NGS applications, automated report generation, and support for running both R and command-line software, such as NGS aligners or peak/variant callers, on local computers or compute clusters. Efficient handling of complex sample sets and experimental designs is facilitated by a consistently implemented sample annotation infrastructure. Instructions for using systemPipeR are given in the Overview Vignette (HTML). The remaining Vignettes, linked below, are workflow templates for common NGS use cases.\n",
      "Version: 2.4.0\n",
      "Publication date: None\n",
      "Mantainer: Thomas Girke <thomas.girke@ucr.edu>\n",
      "Authors:\n",
      "Author: Thomas Girke\n",
      "Dependencies:\n",
      "  - Rsamtools, type: depends\n",
      "  - Biostrings, type: depends\n",
      "  - ShortRead, type: depends\n",
      "  - methods, type: depends\n",
      "  - GenomicRanges, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - yaml, type: imports\n",
      "  - stringr, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - crayon, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - htmlwidgets, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/systemPipeR.html\n",
      "\n",
      "Name: TrajectoryUtils\n",
      "Description:\n",
      "Implements low-level utilities for single-cell trajectory analysis, primarily intended for re-use inside higher-level packages. Include a function to create a cluster-level minimum spanning tree and data structures to hold pseudotime inference results.\n",
      "Version: 1.6.0\n",
      "Publication date: None\n",
      "Mantainer: Aaron Lun <infinite.monkeys.with.keyboards@gmail.com>\n",
      "Authors:\n",
      "Author: Aaron Lun [aut, cre], Kelly Street [aut]\n",
      "Dependencies:\n",
      "  - SingleCellExperiment, type: depends\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - igraph, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/TrajectoryUtils.html\n",
      "\n",
      "Name: flowClust\n",
      "Description:\n",
      "Robust model-based clustering using a t-mixture model with Box-Cox transformation. Note: users should have GSL installed. Windows users: 'consult the README file available in the inst directory of the source distribution for necessary configuration instructions'.\n",
      "Version: 3.36.0\n",
      "Publication date: None\n",
      "Mantainer: Greg Finak <greg@ozette.ai>, Mike Jiang <mike@ozette.ai>\n",
      "Authors:\n",
      "Author: Raphael Gottardo, Kenneth Lo <c.lo at stat.ubc.ca>, Greg Finak <greg at ozette.ai>\n",
      "Dependencies:\n",
      "  - R (>= 2.5.0) type: depends\n",
      "  - BiocGenerics, type: imports\n",
      "  - methods, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - graph, type: imports\n",
      "  - flowCore, type: imports\n",
      "  - parallel, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/flowClust.html\n",
      "\n",
      "Name: ROC\n",
      "Description:\n",
      "Provide utilities for ROC, with microarray focus.\n",
      "Version: 1.74.0\n",
      "Publication date: None\n",
      "Mantainer: Vince Carey <stvjc@channing.harvard.edu>\n",
      "Authors:\n",
      "Author: Vince Carey <stvjc at channing.harvard.edu>, Henning Redestig for C++ language enhancements\n",
      "Dependencies:\n",
      "  - R (>= 1.9.0) type: depends\n",
      "  - utils, type: depends\n",
      "  - methods, type: depends\n",
      "  - knitr, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ROC.html\n",
      "\n",
      "Name: lumi\n",
      "Description:\n",
      "The lumi package provides an integrated solution for the Illumina microarray data analysis. It includes functions of Illumina BeadStudio (GenomeStudio) data input, quality control, BeadArray-specific variance stabilization, normalization and gene annotation at the probe level. It also includes the functions of processing Illumina methylation microarrays, especially Illumina Infinium methylation microarrays.\n",
      "Version: 2.50.0\n",
      "Publication date: None\n",
      "Mantainer: Lei Huang <lhuang1998@gmail.com>\n",
      "Authors:\n",
      "Author: Pan Du, Richard Bourgon, Gang Feng, Simon Lin\n",
      "Dependencies:\n",
      "  - R (>= 2.10) type: depends\n",
      "  - Biobase, type: depends\n",
      "  - affy, type: imports\n",
      "  - methylumi, type: imports\n",
      "  - GenomicFeatures, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - annotate, type: imports\n",
      "  - lattice, type: imports\n",
      "  - mgcv (>= 1.4-0) type: imports\n",
      "  - nleqslv, type: imports\n",
      "  - KernSmooth, type: imports\n",
      "  - preprocessCore, type: imports\n",
      "  - RSQLite, type: imports\n",
      "  - DBI, type: imports\n",
      "  - AnnotationDbi, type: imports\n",
      "  - MASS, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - stats4, type: imports\n",
      "  - methods, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/lumi.html\n",
      "\n",
      "Name: openCyto\n",
      "Description:\n",
      "This package is designed to facilitate the automated gating methods in sequential way to mimic the manual gating strategy.\n",
      "Version: 2.10.1\n",
      "Publication date: None\n",
      "Mantainer: Mike Jiang <mike@ozette.com>\n",
      "Authors:\n",
      "Author: Mike Jiang, John Ramey, Greg Finak, Raphael Gottardo\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - methods, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - flowCore, type: imports\n",
      "  - flowViz, type: imports\n",
      "  - ncdfFlow, type: imports\n",
      "  - flowWorkspace, type: imports\n",
      "  - flowClust, type: imports\n",
      "  - RBGL, type: imports\n",
      "  - graph, type: imports\n",
      "  - data.table, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "AGPL-3.0-only\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/openCyto.html\n",
      "\n",
      "Name: M3C\n",
      "Description:\n",
      "M3C is a consensus clustering algorithm that uses a Monte Carlo simulation to eliminate overestimation of K and can reject the null hypothesis K=1.\n",
      "Version: 1.20.0\n",
      "Publication date: None\n",
      "Mantainer: Christopher John <chris.r.john86@gmail.com>\n",
      "Authors:\n",
      "Author: Christopher John, David Watson\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - ggplot2, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - doSNOW, type: imports\n",
      "  - cluster, type: imports\n",
      "  - parallel, type: imports\n",
      "  - foreach, type: imports\n",
      "  - doParallel, type: imports\n",
      "  - matrixcalc, type: imports\n",
      "  - Rtsne, type: imports\n",
      "  - corpcor, type: imports\n",
      "  - umap, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "AGPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/M3C.html\n",
      "\n",
      "Name: flowStats\n",
      "Description:\n",
      "Methods and functionality to analyse flow data that is beyond the basic infrastructure provided by the flowCore package.\n",
      "Version: 4.10.0\n",
      "Publication date: None\n",
      "Mantainer: Greg Finak <greg@ozette.com>, Mike Jiang <mike@ozette.com>\n",
      "Authors:\n",
      "Author: Florian Hahne, Nishant Gopalakrishnan, Alireza Hadj Khodabakhshi, Chao-Jen Wong, Kyongryun Lee\n",
      "Dependencies:\n",
      "  - R (>= 3.0.2) type: depends\n",
      "  - BiocGenerics, type: imports\n",
      "  - MASS, type: imports\n",
      "  - flowCore, type: imports\n",
      "  - flowWorkspace, type: imports\n",
      "  - ncdfFlow, type: imports\n",
      "  - flowViz, type: imports\n",
      "  - fda (>= 2.2.6) type: imports\n",
      "  - Biobase, type: imports\n",
      "  - methods, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - cluster, type: imports\n",
      "  - utils, type: imports\n",
      "  - KernSmooth, type: imports\n",
      "  - lattice, type: imports\n",
      "  - ks, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - rrcov, type: imports\n",
      "  - corpcor, type: imports\n",
      "  - mnormt, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/flowStats.html\n",
      "\n",
      "Name: DiffBind\n",
      "Description:\n",
      "Compute differentially bound sites from multiple ChIP-seq experiments using affinity (quantitative) data. Also enables occupancy (overlap) analysis and plotting functions.\n",
      "Version: 3.8.4\n",
      "Publication date: None\n",
      "Mantainer: Rory Stark <rory.stark@cruk.cam.ac.uk>\n",
      "Authors:\n",
      "Author: Rory Stark [aut, cre], Gord Brown [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - RColorBrewer, type: imports\n",
      "  - amap, type: imports\n",
      "  - gplots, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - limma, type: imports\n",
      "  - GenomicAlignments, type: imports\n",
      "  - locfit, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - lattice, type: imports\n",
      "  - systemPipeR, type: imports\n",
      "  - tools, type: imports\n",
      "  - Rcpp, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - parallel, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - DESeq2, type: imports\n",
      "  - methods, type: imports\n",
      "  - graphics, type: imports\n",
      "  - ggrepel, type: imports\n",
      "  - apeglm, type: imports\n",
      "  - ashr, type: imports\n",
      "  - GreyListChIP, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/DiffBind.html\n",
      "\n",
      "Name: slingshot\n",
      "Description:\n",
      "Provides functions for inferring continuous, branching lineage structures in low-dimensional data. Slingshot was designed to model developmental trajectories in single-cell RNA sequencing data and serve as a component in an analysis pipeline after dimensionality reduction and clustering. It is flexible enough to handle arbitrarily many branching events and allows for the incorporation of prior knowledge through supervised graph construction.\n",
      "Version: 2.6.0\n",
      "Publication date: None\n",
      "Mantainer: Kelly Street <street.kelly@gmail.com>\n",
      "Authors:\n",
      "Author: Kelly Street [aut, cre, cph], Davide Risso [aut], Diya Das [aut], Sandrine Dudoit [ths], Koen Van den Berge [ctb], Robrecht Cannoodt [ctb] (, rcannood)\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - princurve (>= 2.0.4) type: depends\n",
      "  - stats, type: depends\n",
      "  - TrajectoryUtils, type: depends\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - igraph, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - methods, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - SingleCellExperiment, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/slingshot.html\n",
      "\n",
      "Name: MsFeatures\n",
      "Description:\n",
      "The MsFeature package defines functionality for Mass Spectrometry features. This includes functions to group (LC-MS) features based on some of their properties, such as retention time (coeluting features), or correlation of signals across samples. This packge hence allows to group features, and its results can be used as an input for the `QFeatures` package which allows to aggregate abundance levels of features within each group. This package defines concepts and functions for base and common data types, implementations for more specific data types are expected to be implemented in the respective packages (such as e.g. `xcms`). All functionality of this package is implemented in a modular way which allows combination of different grouping approaches and enables its re-use in other R packages.\n",
      "Version: 1.6.0\n",
      "Publication date: None\n",
      "Mantainer: Johannes Rainer <Johannes.Rainer@eurac.edu>\n",
      "Authors:\n",
      "Author: Johannes Rainer [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - methods, type: imports\n",
      "  - ProtGenerics, type: imports\n",
      "  - MsCoreUtils, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - stats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/MsFeatures.html\n",
      "\n",
      "Name: DEXSeq\n",
      "Description:\n",
      "The package is focused on finding differential exon usage using RNA-seq exon counts between samples with different experimental designs. It provides functions that allows the user to make the necessary statistical tests based on a model that uses the negative binomial distribution to estimate the variance between biological replicates and generalized linear models for testing. The package also provides functions for the visualization and exploration of the results.\n",
      "Version: 1.44.0\n",
      "Publication date: None\n",
      "Mantainer: Alejandro Reyes <alejandro.reyes.ds@gmail.com>\n",
      "Authors:\n",
      "Author: Simon Anders <sanders at fs.tum.de> and Alejandro Reyes <alejandro.reyes.ds at gmail.com>\n",
      "Dependencies:\n",
      "  - BiocParallel, type: depends\n",
      "  - Biobase, type: depends\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - IRanges, type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - DESeq2, type: depends\n",
      "  - AnnotationDbi, type: depends\n",
      "  - RColorBrewer, type: depends\n",
      "  - S4Vectors, type: depends\n",
      "  - BiocGenerics, type: imports\n",
      "  - biomaRt, type: imports\n",
      "  - hwriter, type: imports\n",
      "  - methods, type: imports\n",
      "  - stringr, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - statmod, type: imports\n",
      "  - geneplotter, type: imports\n",
      "  - genefilter, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 3)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/DEXSeq.html\n",
      "\n",
      "Name: GenomicDataCommons\n",
      "Description:\n",
      "Programmatically access the NIH / NCI Genomic Data Commons RESTful service.\n",
      "Version: 1.22.1\n",
      "Publication date: None\n",
      "Mantainer: Sean Davis <seandavi@gmail.com>\n",
      "Authors:\n",
      "Author: Martin Morgan [aut], Sean Davis [aut, cre], Marcel Ramos [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 3.4.0) type: depends\n",
      "  - magrittr, type: depends\n",
      "  - stats, type: imports\n",
      "  - httr, type: imports\n",
      "  - xml2, type: imports\n",
      "  - jsonlite, type: imports\n",
      "  - utils, type: imports\n",
      "  - rlang, type: imports\n",
      "  - readr, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - rappdirs, type: imports\n",
      "  - tibble, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/GenomicDataCommons.html\n",
      "\n",
      "Name: lpsymphony\n",
      "Description:\n",
      "This package was derived from Rsymphony_0.1-17 from CRAN. These packages provide an R interface to SYMPHONY, an open-source linear programming solver written in C++. The main difference between this package and Rsymphony is that it includes the solver source code (SYMPHONY version 5.6), while Rsymphony expects to find header and library files on the users' system. Thus the intention of lpsymphony is to provide an easy to install interface to SYMPHONY. For Windows, precompiled DLLs are included in this package.\n",
      "Version: 1.26.3\n",
      "Publication date: None\n",
      "Mantainer: Vladislav Kim <Vladislav.Kim@embl.de>\n",
      "Authors:\n",
      "Author: Vladislav Kim [aut, cre], Ted Ralphs [ctb], Menal Guzelsoy [ctb], Ashutosh Mahajan [ctb], Reinhard Harter [ctb], Kurt Hornik [ctb], Cyrille Szymanski [ctb], Stefan Theussl [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 3.0.0) type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "EPL\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/lpsymphony.html\n",
      "\n",
      "Name: tximeta\n",
      "Description:\n",
      "Transcript quantification import from Salmon and alevin with automatic attachment of transcript ranges and release information, and other associated metadata. De novo transcriptomes can be linked to the appropriate sources with linkedTxomes and shared for computational reproducibility.\n",
      "Version: 1.16.1\n",
      "Publication date: None\n",
      "Mantainer: Michael Love <michaelisaiahlove@gmail.com>\n",
      "Authors:\n",
      "Author: Michael Love [aut, cre], Charlotte Soneson [aut, ctb], Peter Hickey [aut, ctb], Rob Patro [aut, ctb], NIH NHGRI [fnd], CZI [fnd]\n",
      "Dependencies:\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - tximport, type: imports\n",
      "  - jsonlite, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - AnnotationDbi, type: imports\n",
      "  - GenomicFeatures, type: imports\n",
      "  - ensembldb, type: imports\n",
      "  - BiocFileCache, type: imports\n",
      "  - AnnotationHub, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - tibble, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - tools, type: imports\n",
      "  - utils, type: imports\n",
      "  - methods, type: imports\n",
      "  - Matrix, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/tximeta.html\n",
      "\n",
      "Name: DSS\n",
      "Description:\n",
      "DSS is an R library performing differntial analysis for count-based sequencing data. It detectes differentially expressed genes (DEGs) from RNA-seq, and differentially methylated loci or regions (DML/DMRs) from bisulfite sequencing (BS-seq). The core of DSS is a new dispersion shrinkage method for estimating the dispersion parameter from Gamma-Poisson or Beta-Binomial distributions.\n",
      "Version: 2.46.0\n",
      "Publication date: None\n",
      "Mantainer: Hao Wu <hao.wu@emory.edu>, Hao Feng <hxf155@case.edu>\n",
      "Authors:\n",
      "Author: Hao Wu<hao.wu at emory.edu>, Hao Feng<hxf155 at case.edu>\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - Biobase, type: depends\n",
      "  - BiocParallel, type: depends\n",
      "  - bsseq, type: depends\n",
      "  - parallel, type: depends\n",
      "  - utils, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - splines, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/DSS.html\n",
      "\n",
      "Name: CytoML\n",
      "Description:\n",
      "Uses platform-specific implemenations of the GatingML2.0 standard to exchange gated cytometry data with other software platforms.\n",
      "Version: 2.10.0\n",
      "Publication date: None\n",
      "Mantainer: Mike Jiang <mike@ozette.com>\n",
      "Authors:\n",
      "Author: Mike Jiang, Jake Wagner\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - cytolib, type: imports\n",
      "  - flowCore, type: imports\n",
      "  - flowWorkspace, type: imports\n",
      "  - openCyto, type: imports\n",
      "  - XML, type: imports\n",
      "  - data.table, type: imports\n",
      "  - jsonlite, type: imports\n",
      "  - RBGL, type: imports\n",
      "  - Rgraphviz, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - methods, type: imports\n",
      "  - graph, type: imports\n",
      "  - graphics, type: imports\n",
      "  - utils, type: imports\n",
      "  - jsonlite, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - methods, type: imports\n",
      "  - ggcyto, type: imports\n",
      "  - yaml, type: imports\n",
      "  - stats, type: imports\n",
      "  - tibble, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "AGPL-3.0-only\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/CytoML.html\n",
      "\n",
      "Name: PCAtools\n",
      "Description:\n",
      "Principal Component Analysis (PCA) is a very powerful technique that has wide applicability in data science, bioinformatics, and further afield. It was initially developed to analyse large volumes of data in order to tease out the differences/relationships between the logical entities being analysed. It extracts the fundamental structure of the data without the need to build any model to represent it. This 'summary' of the data is arrived at through a process of reduction that can transform the large number of variables into a lesser number that are uncorrelated (i.e. the 'principal components'), while at the same time being capable of easy interpretation on the original data. PCAtools provides functions for data exploration via PCA, and allows the user to generate publication-ready figures. PCA is performed via BiocSingular - users can also identify optimal number of principal components via different metrics, such as elbow method and Horn's parallel analysis, which has relevance for data reduction in single-cell RNA-seq (scRNA-seq) and high dimensional mass cytometry data.\n",
      "Version: 2.10.0\n",
      "Publication date: None\n",
      "Mantainer: Kevin Blighe <kevin@clinicalbioinformatics.co.uk>\n",
      "Authors:\n",
      "Author: Kevin Blighe [aut, cre], Anna-Leigh Brown [ctb], Vincent Carey [ctb], Guido Hooiveld [ctb], Aaron Lun [aut, ctb]\n",
      "Dependencies:\n",
      "  - ggplot2, type: depends\n",
      "  - ggrepel, type: depends\n",
      "  - lattice, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - cowplot, type: imports\n",
      "  - methods, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - stats, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - DelayedMatrixStats, type: imports\n",
      "  - DelayedArray, type: imports\n",
      "  - BiocSingular, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - Rcpp, type: imports\n",
      "  - dqrng, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/PCAtools.html\n",
      "\n",
      "Name: gage\n",
      "Description:\n",
      "GAGE is a published method for gene set (enrichment or GSEA) or pathway analysis. GAGE is generally applicable independent of microarray or RNA-Seq data attributes including sample sizes, experimental designs, assay platforms, and other types of heterogeneity, and consistently achieves superior performance over other frequently used methods. In gage package, we provide functions for basic GAGE analysis, result processing and presentation. We have also built pipeline routines for of multiple GAGE analyses in a batch, comparison between parallel analyses, and combined analysis of heterogeneous data from different sources/studies. In addition, we provide demo microarray data and commonly used gene set data based on KEGG pathways and GO terms. These funtions and data are also useful for gene set analysis using other methods.\n",
      "Version: 2.48.0\n",
      "Publication date: None\n",
      "Mantainer: Weijun Luo <luo_weijun@yahoo.com>\n",
      "Authors:\n",
      "Author: Weijun Luo\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - graph, type: imports\n",
      "  - KEGGREST, type: imports\n",
      "  - AnnotationDbi, type: imports\n",
      "  - GO.db, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>=2.0)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/gage.html\n",
      "\n",
      "Name: widgetTools\n",
      "Description:\n",
      "This packages contains tools to support the construction of tcltk widgets\n",
      "Version: 1.76.0\n",
      "Publication date: None\n",
      "Mantainer: Jianhua Zhang <jzhang@jimmy.harvard.edu>\n",
      "Authors:\n",
      "Author: Jianhua Zhang\n",
      "Dependencies:\n",
      "  - R (>= 2.4.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - utils, type: depends\n",
      "  - tcltk, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/widgetTools.html\n",
      "\n",
      "Name: Wrench\n",
      "Description:\n",
      "Wrench is a package for normalization sparse genomic count data, like that arising from 16s metagenomic surveys.\n",
      "Version: 1.16.0\n",
      "Publication date: None\n",
      "Mantainer: Hector Corrada Bravo <hcorrada@gmail.com>\n",
      "Authors:\n",
      "Author: Senthil Kumar Muthiah [aut], Hector Corrada Bravo [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - limma, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - locfit, type: imports\n",
      "  - stats, type: imports\n",
      "  - graphics, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/Wrench.html\n",
      "\n",
      "Name: DynDoc\n",
      "Description:\n",
      "A set of functions to create and interact with dynamic documents and vignettes.\n",
      "Version: 1.76.0\n",
      "Publication date: None\n",
      "Mantainer: Bioconductor Package Maintainer <maintainer@bioconductor.org>\n",
      "Authors:\n",
      "Author: R. Gentleman, Jeff Gentry\n",
      "Dependencies:\n",
      "  - methods, type: depends\n",
      "  - utils, type: depends\n",
      "  - methods, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/DynDoc.html\n",
      "\n",
      "Name: motifmatchr\n",
      "Description:\n",
      "Quickly find motif matches for many motifs and many sequences. Wraps C++ code from the MOODS motif calling library, which was developed by Pasi Rastas, Janne Korhonen, and Petri MartinmÃ¤ki.\n",
      "Version: 1.20.0\n",
      "Publication date: None\n",
      "Mantainer: Alicia Schep <aschep@gmail.com>\n",
      "Authors:\n",
      "Author: Alicia Schep [aut, cre], Stanford University [cph]\n",
      "Dependencies:\n",
      "  - R (>= 3.3) type: depends\n",
      "  - Matrix, type: imports\n",
      "  - Rcpp, type: imports\n",
      "  - methods, type: imports\n",
      "  - TFBSTools, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - BSgenome, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3 + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/motifmatchr.html\n",
      "\n",
      "Name: metagenomeSeq\n",
      "Description:\n",
      "metagenomeSeq is designed to determine features (be it Operational Taxanomic Unit (OTU), species, etc.) that are differentially abundant between two or more groups of multiple samples. metagenomeSeq is designed to address the effects of both normalization and under-sampling of microbial communities on disease association detection and the testing of feature correlations.\n",
      "Version: 1.40.0\n",
      "Publication date: None\n",
      "Mantainer: Joseph N. Paulson <jpaulson@jimmy.harvard.edu>\n",
      "Authors:\n",
      "Author: Joseph Nathaniel Paulson, Nathan D. Olson, Domenick J. Braccia, Justin Wagner, Hisham Talukder, Mihai Pop, Hector Corrada Bravo\n",
      "Dependencies:\n",
      "  - R (>= 3.0) type: depends\n",
      "  - Biobase, type: depends\n",
      "  - limma, type: depends\n",
      "  - glmnet, type: depends\n",
      "  - methods, type: depends\n",
      "  - RColorBrewer, type: depends\n",
      "  - parallel, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - foreach, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - gplots, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - Wrench, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/metagenomeSeq.html\n",
      "\n",
      "Name: tkWidgets\n",
      "Description:\n",
      "Widgets to provide user interfaces. tcltk should have been installed for the widgets to run.\n",
      "Version: 1.76.0\n",
      "Publication date: None\n",
      "Mantainer: J. Zhang <jzhang@jimmy.harvard.edu>\n",
      "Authors:\n",
      "Author: J. Zhang <jzhang at jimmy.harvard.edu>\n",
      "Dependencies:\n",
      "  - R (>= 2.0.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - widgetTools, type: depends\n",
      "  - DynDoc, type: depends\n",
      "  - tools, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/tkWidgets.html\n",
      "\n",
      "Name: bamsignals\n",
      "Description:\n",
      "This package allows to efficiently obtain count vectors from indexed bam files. It counts the number of reads in given genomic ranges and it computes reads profiles and coverage profiles. It also handles paired-end data.\n",
      "Version: 1.30.0\n",
      "Publication date: None\n",
      "Mantainer: Johannes Helmuth <johannes.helmuth@laborberlin.com>\n",
      "Authors:\n",
      "Author: Alessandro Mammana [aut, cre], Johannes Helmuth [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - methods, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - Rcpp (>= 0.10.6) type: imports\n",
      "  - IRanges, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - zlibbioc, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/bamsignals.html\n",
      "\n",
      "Name: ChIPpeakAnno\n",
      "Description:\n",
      "The package includes functions to retrieve the sequences around the peak, obtain enriched Gene Ontology (GO) terms, find the nearest gene, exon, miRNA or custom features such as most conserved elements and other transcription factor binding sites supplied by users. Starting 2.0.5, new functions have been added for finding the peaks with bi-directional promoters with summary statistics (peaksNearBDP), for summarizing the occurrence of motifs in peaks (summarizePatternInPeaks) and for adding other IDs to annotated peaks or enrichedGO (addGeneIDs). This package leverages the biomaRt, IRanges, Biostrings, BSgenome, GO.db, multtest and stat packages.\n",
      "Version: 3.32.0\n",
      "Publication date: None\n",
      "Mantainer: Jianhong Ou <jianhong.ou@duke.edu>, Lihua Julie Zhu <julie.zhu@umassmed.edu>\n",
      "Authors:\n",
      "Author: Lihua Julie Zhu, Jianhong Ou, Jun Yu, Kai Hu, Haibo Liu, HervÃ© PagÃ¨s, Claude Gazin, Nathan Lawson, Ryan Thompson, Simon Lin, David Lapointe and Michael Green\n",
      "Dependencies:\n",
      "  - R (>= 3.5) type: depends\n",
      "  - methods, type: depends\n",
      "  - IRanges, type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - S4Vectors, type: depends\n",
      "  - AnnotationDbi, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - DBI, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - ensembldb, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - GenomicAlignments, type: imports\n",
      "  - GenomicFeatures, type: imports\n",
      "  - RBGL, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - VennDiagram, type: imports\n",
      "  - biomaRt, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graph, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grid, type: imports\n",
      "  - InteractionSet, type: imports\n",
      "  - KEGGREST, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - multtest, type: imports\n",
      "  - regioneR, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ChIPpeakAnno.html\n",
      "\n",
      "Name: FlowSOM\n",
      "Description:\n",
      "FlowSOM offers visualization options for cytometry data, by using Self-Organizing Map clustering and Minimal Spanning Trees.\n",
      "Version: 2.6.0\n",
      "Publication date: None\n",
      "Mantainer: Sofie Van Gassen <sofie.vangassen@ugent.be>\n",
      "Authors:\n",
      "Author: Sofie Van Gassen [aut, cre], Artuur Couckuyt [aut], Katrien Quintelier [aut], Annelies Emmaneel [aut], Britt Callebaut [aut], Yvan Saeys [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - igraph, type: depends\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - colorRamps, type: imports\n",
      "  - ConsensusClusterPlus, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - flowCore, type: imports\n",
      "  - ggforce, type: imports\n",
      "  - ggnewscale, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - ggpubr, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - methods, type: imports\n",
      "  - rlang, type: imports\n",
      "  - Rtsne, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - XML, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/FlowSOM.html\n",
      "\n",
      "Name: chromVAR\n",
      "Description:\n",
      "Determine variation in chromatin accessibility across sets of annotations or peaks. Designed primarily for single-cell or sparse chromatin accessibility data, e.g. from scATAC-seq or sparse bulk ATAC or DNAse-seq experiments.\n",
      "Version: 1.20.2\n",
      "Publication date: None\n",
      "Mantainer: Alicia Schep <aschep@gmail.com>\n",
      "Authors:\n",
      "Author: Alicia Schep [aut, cre], Jason Buenrostro [ctb], Caleb Lareau [ctb], William Greenleaf [ths], Stanford University [cph]\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - IRanges, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - nabor, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - TFBSTools, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - methods, type: imports\n",
      "  - Rcpp, type: imports\n",
      "  - grid, type: imports\n",
      "  - plotly, type: imports\n",
      "  - shiny, type: imports\n",
      "  - miniUI, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - graphics, type: imports\n",
      "  - DT, type: imports\n",
      "  - Rtsne, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - BSgenome, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/chromVAR.html\n",
      "\n",
      "Name: missMethyl\n",
      "Description:\n",
      "Normalisation, testing for differential variability and differential methylation and gene set testing for data from Illumina's Infinium HumanMethylation arrays. The normalisation procedure is subset-quantile within-array normalisation (SWAN), which allows Infinium I and II type probes on a single array to be normalised together. The test for differential variability is based on an empirical Bayes version of Levene's test. Differential methylation testing is performed using RUV, which can adjust for systematic errors of unknown origin in high-dimensional data by using negative control probes. Gene ontology analysis is performed by taking into account the number of probes per gene on the array, as well as taking into account multi-gene associated probes.\n",
      "Version: 1.32.0\n",
      "Publication date: None\n",
      "Mantainer: Belinda Phipson <belinda.phipson@petermac.org>, Jovana Maksimovic <jovana.maksimovic@petermac.org>, Andrew Lonsdale <andrew.lonsdale@petermac.org>\n",
      "Authors:\n",
      "Author: Belinda Phipson and Jovana Maksimovic\n",
      "Dependencies:\n",
      "  - R (>= 3.6.0) type: depends\n",
      "  - IlluminaHumanMethylation450kanno.ilmn12.hg19, type: depends\n",
      "  - IlluminaHumanMethylationEPICanno.ilm10b4.hg19, type: depends\n",
      "  - AnnotationDbi, type: imports\n",
      "  - BiasedUrn, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - GO.db, type: imports\n",
      "  - IlluminaHumanMethylation450kmanifest, type: imports\n",
      "  - IlluminaHumanMethylationEPICmanifest, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - limma, type: imports\n",
      "  - methods, type: imports\n",
      "  - methylumi, type: imports\n",
      "  - minfi, type: imports\n",
      "  - org.Hs.eg.db, type: imports\n",
      "  - ruv, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - statmod, type: imports\n",
      "  - stringr, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/missMethyl.html\n",
      "\n",
      "Name: RcisTarget\n",
      "Description:\n",
      "RcisTarget identifies transcription factor binding motifs (TFBS) over-represented on a gene list. In a first step, RcisTarget selects DNA motifs that are significantly over-represented in the surroundings of the transcription start site (TSS) of the genes in the gene-set. This is achieved by using a database that contains genome-wide cross-species rankings for each motif. The motifs that are then annotated to TFs and those that have a high Normalized Enrichment Score (NES) are retained. Finally, for each motif and gene-set, RcisTarget predicts the candidate target genes (i.e. genes in the gene-set that are ranked above the leading edge).\n",
      "Version: 1.18.2\n",
      "Publication date: None\n",
      "Mantainer: Sara Aibar <sara.aibar@kuleuven.vib.be>\n",
      "Authors:\n",
      "Author: Sara Aibar, Gert Hulselmans, Stein Aerts. Laboratory of Computational Biology. VIB-KU Leuven Center for Brain & Disease Research. Leuven, Belgium\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - AUCell, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - data.table, type: imports\n",
      "  - graphics, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - arrow (>= 2.0.0) type: imports\n",
      "  - dplyr, type: imports\n",
      "  - tibble, type: imports\n",
      "  - GSEABase, type: imports\n",
      "  - methods, type: imports\n",
      "  - R.utils, type: imports\n",
      "  - stats, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/RcisTarget.html\n",
      "\n",
      "Name: GenomicFiles\n",
      "Description:\n",
      "This package provides infrastructure for parallel computations distributed 'by file' or 'by range'. User defined MAPPER and REDUCER functions provide added flexibility for data combination and manipulation.\n",
      "Version: 1.34.0\n",
      "Publication date: None\n",
      "Mantainer: Bioconductor Package Maintainer <maintainer@bioconductor.org>\n",
      "Authors:\n",
      "Author: Bioconductor Package Maintainer [aut, cre], Valerie Obenchain [aut], Michael Love [aut], Lori Shepherd [aut], Martin Morgan [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.1.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - BiocGenerics, type: depends\n",
      "  - MatrixGenerics, type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - BiocParallel, type: depends\n",
      "  - Rsamtools, type: depends\n",
      "  - rtracklayer, type: depends\n",
      "  - GenomicAlignments, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - VariantAnnotation, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/GenomicFiles.html\n",
      "\n",
      "Name: GENIE3\n",
      "Description:\n",
      "This package implements the GENIE3 algorithm for inferring gene regulatory networks from expression data.\n",
      "Version: 1.20.0\n",
      "Publication date: None\n",
      "Mantainer: Van Anh Huynh-Thu <vahuynh@ulg.ac.be>\n",
      "Authors:\n",
      "Author: Van Anh Huynh-Thu, Sara Aibar, Pierre Geurts\n",
      "Dependencies:\n",
      "  - stats, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - dplyr, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/GENIE3.html\n",
      "\n",
      "Name: plyranges\n",
      "Description:\n",
      "A dplyr-like interface for interacting with the common Bioconductor classes Ranges and GenomicRanges. By providing a grammatical and consistent way of manipulating these classes their accessiblity for new Bioconductor users is hopefully increased.\n",
      "Version: 1.18.0\n",
      "Publication date: None\n",
      "Mantainer: Stuart Lee <stuart.andrew.lee@gmail.com>\n",
      "Authors:\n",
      "Author: Stuart Lee [aut, cre] , Michael Lawrence [aut, ctb], Dianne Cook [aut, ctb], Spencer Nystrom [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 3.5) type: depends\n",
      "  - BiocGenerics, type: depends\n",
      "  - IRanges, type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - methods, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - rlang (>= 0.2.0) type: imports\n",
      "  - magrittr, type: imports\n",
      "  - tidyselect (>= 1.0.0) type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - GenomicAlignments, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/plyranges.html\n",
      "\n",
      "Name: DMRcate\n",
      "Description:\n",
      "De novo identification and extraction of differentially methylated regions (DMRs) from the human genome using Whole Genome Bisulfite Sequencing (WGBS) and Illumina Infinium Array (450K and EPIC) data. Provides functionality for filtering probes possibly confounded by SNPs and cross-hybridisation. Includes GRanges generation and plotting functions.\n",
      "Version: 2.12.0\n",
      "Publication date: None\n",
      "Mantainer: Tim Peters <t.peters@garvan.org.au>\n",
      "Authors:\n",
      "Author: Tim Peters\n",
      "Dependencies:\n",
      "  - R (>= 4.0.0) type: depends\n",
      "  - ExperimentHub, type: imports\n",
      "  - bsseq, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - limma, type: imports\n",
      "  - edgeR, type: imports\n",
      "  - DSS, type: imports\n",
      "  - minfi, type: imports\n",
      "  - missMethyl, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - plyr, type: imports\n",
      "  - Gviz, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - methods, type: imports\n",
      "  - graphics, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/DMRcate.html\n",
      "\n",
      "Name: RaggedExperiment\n",
      "Description:\n",
      "This package provides a flexible representation of copy number, mutation, and other data that fit into the ragged array schema for genomic location data. The basic representation of such data provides a rectangular flat table interface to the user with range information in the rows and samples/specimen in the columns. The RaggedExperiment class derives from a GRangesList representation and provides a semblance of a rectangular dataset.\n",
      "Version: 1.22.0\n",
      "Publication date: None\n",
      "Mantainer: Marcel Ramos <marcel.ramos@roswellpark.org>\n",
      "Authors:\n",
      "Author: Martin Morgan [aut], Marcel Ramos [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.2.0) type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - BiocGenerics, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - MatrixGenerics, type: imports\n",
      "  - methods, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - stats, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/RaggedExperiment.html\n",
      "\n",
      "Name: RUVSeq\n",
      "Description:\n",
      "This package implements the remove unwanted variation (RUV) methods of Risso et al. (2014) for the normalization of RNA-Seq read counts between samples.\n",
      "Version: 1.32.0\n",
      "Publication date: None\n",
      "Mantainer: Davide Risso <risso.davide@gmail.com>\n",
      "Authors:\n",
      "Author: Davide Risso [aut, cre, cph], Sandrine Dudoit [aut], Lorena Pantano [ctb], Kamil Slowikowski [ctb]\n",
      "Dependencies:\n",
      "  - Biobase, type: depends\n",
      "  - EDASeq, type: depends\n",
      "  - edgeR, type: depends\n",
      "  - methods, type: imports\n",
      "  - MASS, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/RUVSeq.html\n",
      "\n",
      "Name: STRINGdb\n",
      "Description:\n",
      "The STRINGdb package provides a R interface to the STRING protein-protein interactions database (https://string-db.org).\n",
      "Version: 2.10.1\n",
      "Publication date: None\n",
      "Mantainer: Damian Szklarczyk <damian.szklarczyk@imls.uzh.ch>\n",
      "Authors:\n",
      "Author: Andrea Franceschini <andrea.franceschini at isb-sib.ch>\n",
      "Dependencies:\n",
      "  - R (>= 2.14.0) type: depends\n",
      "  - png, type: imports\n",
      "  - sqldf, type: imports\n",
      "  - plyr, type: imports\n",
      "  - igraph, type: imports\n",
      "  - RCurl, type: imports\n",
      "  - methods, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - gplots, type: imports\n",
      "  - hash, type: imports\n",
      "  - plotrix, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/STRINGdb.html\n",
      "\n",
      "Name: infercnv\n",
      "Description:\n",
      "Using single-cell RNA-Seq expression to visualize CNV in cells.\n",
      "Version: 1.14.0\n",
      "Publication date: None\n",
      "Mantainer: Christophe Georgescu <cgeorges@broadinstitute.org>\n",
      "Authors:\n",
      "Author: Timothy Tickle [aut], Itay Tirosh [aut], Christophe Georgescu [aut, cre], Maxwell Brown [aut], Brian Haas [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - gplots, type: imports\n",
      "  - futile.logger, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - methods, type: imports\n",
      "  - ape, type: imports\n",
      "  - phyclust, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - fastcluster, type: imports\n",
      "  - parallelDist, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - HiddenMarkov, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - edgeR, type: imports\n",
      "  - coin, type: imports\n",
      "  - caTools, type: imports\n",
      "  - digest, type: imports\n",
      "  - RANN, type: imports\n",
      "  - igraph, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - rjags, type: imports\n",
      "  - fitdistrplus, type: imports\n",
      "  - future, type: imports\n",
      "  - foreach, type: imports\n",
      "  - doParallel, type: imports\n",
      "  - Seurat, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - SingleCellExperiment, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - parallel, type: imports\n",
      "  - coda, type: imports\n",
      "  - gridExtra, type: imports\n",
      "  - argparse, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "BSD_3_clause + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/infercnv.html\n",
      "\n",
      "Name: singscore\n",
      "Description:\n",
      "A simple single-sample gene signature scoring method that uses rank-based statistics to analyze the sample's gene expression profile. It scores the expression activities of gene sets at a single-sample level.\n",
      "Version: 1.18.0\n",
      "Publication date: None\n",
      "Mantainer: Dharmesh D. Bhuva <bhuva.d@wehi.edu.au>\n",
      "Authors:\n",
      "Author: Ruqian Lyu [aut, ctb], Momeneh Foroutan [aut, ctb] , Dharmesh D. Bhuva [aut, cre] , Malvika Kharbanda [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.6) type: depends\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - graphics, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - ggrepel, type: imports\n",
      "  - GSEABase, type: imports\n",
      "  - plotly, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - plyr, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - reshape, type: imports\n",
      "  - edgeR, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/singscore.html\n",
      "\n",
      "Name: decontam\n",
      "Description:\n",
      "Simple statistical identification of contaminating sequence features in marker-gene or metagenomics data. Works on any kind of feature derived from environmental sequencing data (e.g. ASVs, OTUs, taxonomic groups, MAGs,...). Requires DNA quantitation data or sequenced negative control samples.\n",
      "Version: 1.18.0\n",
      "Publication date: None\n",
      "Mantainer: Benjamin Callahan <benjamin.j.callahan@gmail.com>\n",
      "Authors:\n",
      "Author: Benjamin Callahan [aut, cre], Nicole Marie Davis [aut], Felix G.M. Ernst [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 3.4.1) type: depends\n",
      "  - methods (>= 3.4.1) type: depends\n",
      "  - ggplot2 (>= 2.1.0) type: imports\n",
      "  - reshape2 (>= 1.4.1) type: imports\n",
      "  - stats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/decontam.html\n",
      "\n",
      "Name: Mfuzz\n",
      "Description:\n",
      "Package for noise-robust soft clustering of gene expression time-series data (including a graphical user interface)\n",
      "Version: 2.58.0\n",
      "Publication date: None\n",
      "Mantainer: Matthias Futschik <matthias.futschik@sysbiolab.eu>\n",
      "Authors:\n",
      "Author: Matthias Futschik <matthias.futschik at sysbiolab.eu>\n",
      "Dependencies:\n",
      "  - R (>= 2.5.0) type: depends\n",
      "  - Biobase, type: depends\n",
      "  - e1071, type: depends\n",
      "  - tcltk, type: imports\n",
      "  - tkWidgets, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/Mfuzz.html\n",
      "\n",
      "Name: SeqArray\n",
      "Description:\n",
      "Data management of large-scale whole-genome sequencing variant calls with thousands of individuals: genotypic data (e.g., SNVs, indels and structural variation calls) and annotations in SeqArray GDS files are stored in an array-oriented and compressed manner, with efficient data access using the R programming language.\n",
      "Version: 1.38.0\n",
      "Publication date: None\n",
      "Mantainer: Xiuwen Zheng <zhengx@u.washington.edu>\n",
      "Authors:\n",
      "Author: Xiuwen Zheng [aut, cre] , Stephanie Gogarten [aut], David Levine [ctb], Cathy Laurie [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - gdsfmt, type: depends\n",
      "  - methods, type: imports\n",
      "  - parallel, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/SeqArray.html\n",
      "\n",
      "Name: scDblFinder\n",
      "Description:\n",
      "The scDblFinder package gathers various methods for the detection and handling of doublets/multiplets in single-cell sequencing data (i.e. multiple cells captured within the same droplet or reaction volume). It includes methods formerly found in the scran package, the new fast and comprehensive scDblFinder method, and a reimplementation of the Amulet detection method for single-cell ATAC-seq.\n",
      "Version: 1.12.0\n",
      "Publication date: None\n",
      "Mantainer: Pierre-Luc Germain <pierre-luc.germain@hest.ethz.ch>\n",
      "Authors:\n",
      "Author: Pierre-Luc Germain [cre, aut] , Aaron Lun [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - SingleCellExperiment, type: depends\n",
      "  - igraph, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - BiocNeighbors, type: imports\n",
      "  - BiocSingular, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - scran, type: imports\n",
      "  - scater, type: imports\n",
      "  - scuttle, type: imports\n",
      "  - bluster, type: imports\n",
      "  - methods, type: imports\n",
      "  - DelayedArray, type: imports\n",
      "  - xgboost, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - MASS, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3 + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/scDblFinder.html\n",
      "\n",
      "Name: GreyListChIP\n",
      "Description:\n",
      "Identify regions of ChIP experiments with high signal in the input, that lead to spurious peaks during peak calling. Remove reads aligning to these regions prior to peak calling, for cleaner ChIP analysis.\n",
      "Version: 1.30.0\n",
      "Publication date: None\n",
      "Mantainer: Rory Stark <rory.stark@cruk.cam.ac.uk>\n",
      "Authors:\n",
      "Author: Gord Brown <gdbzork at gmail.com>\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - GenomicAlignments, type: imports\n",
      "  - BSgenome, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - MASS, type: imports\n",
      "  - parallel, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/GreyListChIP.html\n",
      "\n",
      "Name: survcomp\n",
      "Description:\n",
      "Assessment and Comparison for Performance of Risk Prediction (Survival) Models.\n",
      "Version: 1.48.0\n",
      "Publication date: None\n",
      "Mantainer: Benjamin Haibe-Kains <benjamin.haibe.kains@utoronto.ca>\n",
      "Authors:\n",
      "Author: Benjamin Haibe-Kains [aut, cre], Markus Schroeder [aut], Catharina Olsen [aut], Christos Sotiriou [aut], Gianluca Bontempi [aut], John Quackenbush [aut], Samuel Branders [aut], Zhaleh Safikhani [aut]\n",
      "Dependencies:\n",
      "  - survival, type: depends\n",
      "  - prodlim, type: depends\n",
      "  - R (>= 3.4) type: depends\n",
      "  - ipred, type: imports\n",
      "  - SuppDists, type: imports\n",
      "  - KernSmooth, type: imports\n",
      "  - survivalROC, type: imports\n",
      "  - bootstrap, type: imports\n",
      "  - grid, type: imports\n",
      "  - rmeta, type: imports\n",
      "  - stats, type: imports\n",
      "  - graphics, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/survcomp.html\n",
      "\n",
      "Name: karyoploteR\n",
      "Description:\n",
      "karyoploteR creates karyotype plots of arbitrary genomes and offers a complete set of functions to plot arbitrary data on them. It mimicks many R base graphics functions coupling them with a coordinate change function automatically mapping the chromosome and data coordinates into the plot coordinates. In addition to the provided data plotting functions, it is easy to add new ones.\n",
      "Version: 1.24.0\n",
      "Publication date: None\n",
      "Mantainer: Bernat Gel <bgel@igtp.cat>\n",
      "Authors:\n",
      "Author: Bernat Gel <bgel at igtp.cat>\n",
      "Dependencies:\n",
      "  - R (>= 3.4) type: depends\n",
      "  - regioneR, type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - methods, type: depends\n",
      "  - regioneR, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - stats, type: imports\n",
      "  - graphics, type: imports\n",
      "  - memoise, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - biovizBase, type: imports\n",
      "  - digest, type: imports\n",
      "  - bezier, type: imports\n",
      "  - GenomicFeatures, type: imports\n",
      "  - bamsignals, type: imports\n",
      "  - AnnotationDbi, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - VariantAnnotation, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/karyoploteR.html\n",
      "\n",
      "Name: ReportingTools\n",
      "Description:\n",
      "The ReportingTools software package enables users to easily display reports of analysis results generated from sources such as microarray and sequencing data. The package allows users to create HTML pages that may be viewed on a web browser such as Safari, or in other formats readable by programs such as Excel. Users can generate tables with sortable and filterable columns, make and display plots, and link table entries to other data sources such as NCBI or larger plots within the HTML page. Using the package, users can also produce a table of contents page to link various reports together for a particular project that can be viewed in a web browser. For more examples, please visit our site: http:// research-pub.gene.com/ReportingTools.\n",
      "Version: 2.38.0\n",
      "Publication date: None\n",
      "Mantainer: Jason A. Hackney <hackney.jason@gene.com>, Gabriel Becker <becker.gabe@gene.com>, Jessica L. Larson <larson.jessica@gmail.com>\n",
      "Authors:\n",
      "Author: Jason A. Hackney, Melanie Huntley, Jessica L. Larson, Christina Chaivorapol, Gabriel Becker, and Josh Kaminker\n",
      "Dependencies:\n",
      "  - methods, type: depends\n",
      "  - knitr, type: depends\n",
      "  - utils, type: depends\n",
      "  - Biobase, type: imports\n",
      "  - hwriter, type: imports\n",
      "  - Category, type: imports\n",
      "  - GOstats, type: imports\n",
      "  - limma, type: imports\n",
      "  - lattice, type: imports\n",
      "  - AnnotationDbi, type: imports\n",
      "  - edgeR, type: imports\n",
      "  - annotate, type: imports\n",
      "  - PFAM.db, type: imports\n",
      "  - GSEABase, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - grid, type: imports\n",
      "  - XML, type: imports\n",
      "  - R.utils, type: imports\n",
      "  - DESeq2, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - ggbio, type: imports\n",
      "  - IRanges, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ReportingTools.html\n",
      "\n",
      "Name: wateRmelon\n",
      "Description:\n",
      "15 flavours of betas and three performance metrics, with methods for objects produced by methylumi and minfi packages.\n",
      "Version: 2.4.0\n",
      "Publication date: None\n",
      "Mantainer: Leo C Schalkwyk <lschal@essex.ac.uk>\n",
      "Authors:\n",
      "Author: Leo C Schalkwyk [cre, aut], Tyler J Gorrie-Stone [aut], Ruth Pidsley [aut], Chloe CY Wong [aut], Nizar Touleimat [ctb], Matthieu Defrance [ctb], Andrew Teschendorff [ctb], Jovana Maksimovic [ctb], Louis Y El Khoury [ctb], Yucheng Wang [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - Biobase, type: depends\n",
      "  - limma, type: depends\n",
      "  - methods, type: depends\n",
      "  - matrixStats, type: depends\n",
      "  - methylumi, type: depends\n",
      "  - lumi, type: depends\n",
      "  - ROC, type: depends\n",
      "  - IlluminaHumanMethylation450kanno.ilmn12.hg19, type: depends\n",
      "  - illuminaio, type: depends\n",
      "  - Biobase, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/wateRmelon.html\n",
      "\n",
      "Name: ropls\n",
      "Description:\n",
      "Latent variable modeling with Principal Component Analysis (PCA) and Partial Least Squares (PLS) are powerful methods for visualization, regression, classification, and feature selection of omics data where the number of variables exceeds the number of samples and with multicollinearity among variables. Orthogonal Partial Least Squares (OPLS) enables to separately model the variation correlated (predictive) to the factor of interest and the uncorrelated (orthogonal) variation. While performing similarly to PLS, OPLS facilitates interpretation. Successful applications of these chemometrics techniques include spectroscopic data such as Raman spectroscopy, nuclear magnetic resonance (NMR), mass spectrometry (MS) in metabolomics and proteomics, but also transcriptomics data. In addition to scores, loadings and weights plots, the package provides metrics and graphics to determine the optimal number of components (e.g. with the R2 and Q2 coefficients), check the validity of the model by permutation testing, detect outliers, and perform feature selection (e.g. with Variable Importance in Projection or regression coefficients). The package can be accessed via a user interface on the Workflow4Metabolomics.org online resource for computational metabolomics (built upon the Galaxy environment).\n",
      "Version: 1.30.0\n",
      "Publication date: None\n",
      "Mantainer: Etienne A. Thevenot <etienne.thevenot@cea.fr>\n",
      "Authors:\n",
      "Author: Etienne A. Thevenot [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - Biobase, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - methods, type: imports\n",
      "  - plotly, type: imports\n",
      "  - stats, type: imports\n",
      "  - MultiAssayExperiment, type: imports\n",
      "  - MultiDataSet, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "CeCILL\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ropls.html\n",
      "\n",
      "Name: MultiDataSet\n",
      "Description:\n",
      "Implementation of the BRGE's (Bioinformatic Research Group in Epidemiology from Center for Research in Environmental Epidemiology) MultiDataSet and ResultSet. MultiDataSet is designed for integrating multi omics data sets and ResultSet is a container for omics results. This package contains base classes for MEAL and rexposome packages.\n",
      "Version: 1.26.0\n",
      "Publication date: None\n",
      "Mantainer: Xavier Escrib<c3><a0> Montagut <xavier.escriba@isglobal.org>\n",
      "Authors:\n",
      "Author: Carlos Ruiz-Arenas [aut, cre], Carles Hernandez-Ferrer [aut], Juan R. Gonzalez [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - Biobase, type: depends\n",
      "  - BiocGenerics, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - methods, type: imports\n",
      "  - utils, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - ggrepel, type: imports\n",
      "  - qqman, type: imports\n",
      "  - limma, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/MultiDataSet.html\n",
      "\n",
      "Name: ALDEx2\n",
      "Description:\n",
      "A differential abundance analysis for the comparison of two or more conditions. Useful for analyzing data from standard RNA-seq or meta-RNA-seq assays as well as selected and unselected values from in-vitro sequence selections. Uses a Dirichlet-multinomial model to infer abundance from counts, optimized for three or more experimental replicates. The method infers biological and sampling variation to calculate the expected false discovery rate, given the variation, based on a Wilcoxon Rank Sum test and Welch's t-test (via aldex.ttest), a Kruskal-Wallis test (via aldex.kw), a generalized linear model (via aldex.glm), or a correlation test (via aldex.corr). All tests report p-values and Benjamini-Hochberg corrected p-values. ALDEx2 also calculates expected standardized effect sizes for paired or unpaired study designs.\n",
      "Version: 1.30.0\n",
      "Publication date: None\n",
      "Mantainer: Greg Gloor <ggloor@uwo.ca>\n",
      "Authors:\n",
      "Author: Greg Gloor, Andrew Fernandes, Jean Macklaim, Arianne Albert, Matt Links, Thomas Quinn, Jia Rong Wu, Ruth Grace Wong, Brandon Lieng\n",
      "Dependencies:\n",
      "  - methods, type: depends\n",
      "  - stats, type: depends\n",
      "  - zCompositions, type: depends\n",
      "  - Rfast, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - multtest, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 3)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ALDEx2.html\n",
      "\n",
      "Name: ANCOMBC\n",
      "Description:\n",
      "ANCOMBC is a package containing differential abundance (DA) and correlation analyses for microbiome data. Specifically, the package includes Analysis of Compositions of Microbiomes with Bias Correction 2 (ANCOM-BC2), Analysis of Compositions of Microbiomes with Bias Correction (ANCOM-BC), and Analysis of Composition of Microbiomes (ANCOM) for DA analysis, and Sparse Estimation of Correlations among Microbiomes (SECOM) for correlation analysis. Microbiome data are typically subject to two sources of biases: unequal sampling fractions (sample-specific biases) and differential sequencing efficiencies (taxon-specific biases). Methodologies included in the ANCOMBC package are designed to correct these biases and construct statistically consistent estimators.\n",
      "Version: 2.0.2\n",
      "Publication date: None\n",
      "Mantainer: Huang Lin <huanglinfrederick@gmail.com>\n",
      "Authors:\n",
      "Author: Huang Lin [cre, aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.2.0) type: depends\n",
      "  - mia, type: imports\n",
      "  - stats, type: imports\n",
      "  - CVXR, type: imports\n",
      "  - DescTools, type: imports\n",
      "  - Hmisc, type: imports\n",
      "  - MASS, type: imports\n",
      "  - Rdpack, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - SingleCellExperiment, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - TreeSummarizedExperiment, type: imports\n",
      "  - doParallel, type: imports\n",
      "  - doRNG, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - emmeans, type: imports\n",
      "  - energy, type: imports\n",
      "  - foreach, type: imports\n",
      "  - lme4, type: imports\n",
      "  - lmerTest, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - nloptr, type: imports\n",
      "  - parallel, type: imports\n",
      "  - rlang, type: imports\n",
      "  - rngtools, type: imports\n",
      "  - tibble, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ANCOMBC.html\n",
      "\n",
      "Name: ggtreeExtra\n",
      "Description:\n",
      "'ggtreeExtra' extends the method for mapping and visualizing associated data on phylogenetic tree using 'ggtree'. These associated data can be presented on the external panels to circular layout, fan layout, or other rectangular layout tree built by 'ggtree' with the grammar of 'ggplot2'.\n",
      "Version: 1.8.1\n",
      "Publication date: None\n",
      "Mantainer: Shuangbin Xu <xshuangbin@163.com>\n",
      "Authors:\n",
      "Author: Shuangbin Xu [aut, cre] , Guangchuang Yu [aut, ctb]\n",
      "Dependencies:\n",
      "  - ggplot2, type: imports\n",
      "  - utils, type: imports\n",
      "  - rlang, type: imports\n",
      "  - ggnewscale, type: imports\n",
      "  - stats, type: imports\n",
      "  - ggtree, type: imports\n",
      "  - tidytree (>= 0.3.9) type: imports\n",
      "  - cli, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 3)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ggtreeExtra.html\n",
      "\n",
      "Name: TCGAutils\n",
      "Description:\n",
      "A suite of helper functions for checking and manipulating TCGA data including data obtained from the curatedTCGAData experiment package. These functions aim to simplify and make working with TCGA data more manageable. Exported functions include those that import data from flat files into Bioconductor objects, convert row annotations, and identifier translation via the GDC API.\n",
      "Version: 1.18.0\n",
      "Publication date: None\n",
      "Mantainer: Marcel Ramos <marcel.ramos@roswellpark.org>\n",
      "Authors:\n",
      "Author: Marcel Ramos [aut, cre] , Lucas Schiffer [aut], Sean Davis [ctb], Levi Waldron [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.2.0) type: depends\n",
      "  - AnnotationDbi, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - GenomicFeatures, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - GenomicDataCommons, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - methods, type: imports\n",
      "  - MultiAssayExperiment, type: imports\n",
      "  - RaggedExperiment, type: imports\n",
      "  - rvest, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - stats, type: imports\n",
      "  - stringr, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - utils, type: imports\n",
      "  - xml2, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/TCGAutils.html\n",
      "\n",
      "Name: RCy3\n",
      "Description:\n",
      "Vizualize, analyze and explore networks using Cytoscape via R. Anything you can do using the graphical user interface of Cytoscape, you can now do with a single RCy3 function.\n",
      "Version: 2.18.0\n",
      "Publication date: None\n",
      "Mantainer: Alex Pico <alex.pico@gladstone.ucsf.edu>\n",
      "Authors:\n",
      "Author: Alex Pico [aut, cre] , Tanja Muetze [aut], Paul Shannon [aut], Ruth Isserlin [ctb], Shraddha Pai [ctb], Julia Gustavsen [ctb], Georgi Kolishovski [ctb], Yihang Xin [ctb]\n",
      "Dependencies:\n",
      "  - httr, type: imports\n",
      "  - methods, type: imports\n",
      "  - RJSONIO, type: imports\n",
      "  - XML, type: imports\n",
      "  - utils, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - stats, type: imports\n",
      "  - graph, type: imports\n",
      "  - fs, type: imports\n",
      "  - uuid, type: imports\n",
      "  - uchardet, type: imports\n",
      "  - glue, type: imports\n",
      "  - RCurl, type: imports\n",
      "  - base64url, type: imports\n",
      "  - base64enc, type: imports\n",
      "  - IRkernel, type: imports\n",
      "  - IRdisplay, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - gplots, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/RCy3.html\n",
      "\n",
      "Name: SpatialExperiment\n",
      "Description:\n",
      "Defines an S4 class for storing data from spatially resolved transcriptomics (ST) experiments. The class extends SingleCellExperiment to support storage and retrieval of additional information from spot-based and molecule-based ST platforms, including spatial coordinates, images, and image metadata. A specialized constructor function is included for data from the 10x Genomics Visium platform.\n",
      "Version: 1.8.0\n",
      "Publication date: None\n",
      "Mantainer: Dario Righelli <dario.righelli@gmail.com>\n",
      "Authors:\n",
      "Author: Dario Righelli [aut, cre], Davide Risso [aut], Helena L. Crowell [aut], Lukas M. Weber [aut]\n",
      "Dependencies:\n",
      "  - methods, type: depends\n",
      "  - SingleCellExperiment, type: depends\n",
      "  - rjson, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - magick, type: imports\n",
      "  - utils, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - DropletUtils, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - BiocFileCache, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/SpatialExperiment.html\n",
      "\n",
      "Name: destiny\n",
      "Description:\n",
      "Create and plot diffusion maps.\n",
      "Version: 3.12.0\n",
      "Publication date: None\n",
      "Mantainer: Philipp Angerer <phil.angerer@gmail.com>\n",
      "Authors:\n",
      "Author: Philipp Angerer [cre, aut] , Laleh Haghverdi [ctb], Maren BÃ¼ttner [ctb] , Fabian Theis [ctb] , Carsten Marr [ctb] , Florian BÃ¼ttner [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 3.4.0) type: depends\n",
      "  - methods, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - grid, type: imports\n",
      "  - utils, type: imports\n",
      "  - stats, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - Rcpp (>= 0.10.3) type: imports\n",
      "  - RcppEigen, type: imports\n",
      "  - RSpectra (>= 0.14-0) type: imports\n",
      "  - irlba, type: imports\n",
      "  - pcaMethods, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - SingleCellExperiment, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - ggplot.multistats, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - tidyselect, type: imports\n",
      "  - ggthemes, type: imports\n",
      "  - VIM, type: imports\n",
      "  - knn.covertree, type: imports\n",
      "  - proxy, type: imports\n",
      "  - RcppHNSW, type: imports\n",
      "  - smoother, type: imports\n",
      "  - scales, type: imports\n",
      "  - scatterplot3d, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/destiny.html\n",
      "\n",
      "Name: Ringo\n",
      "Description:\n",
      "The package Ringo facilitates the primary analysis of ChIP-chip data. The main functionalities of the package are data read-in, quality assessment, data visualisation and identification of genomic regions showing enrichment in ChIP-chip. The package has functions to deal with two-color oligonucleotide microarrays from NimbleGen used in ChIP-chip projects, but also contains more general functions for ChIP-chip data analysis, given that the data is supplied as RGList (raw) or ExpressionSet (pre- processed). The package employs functions from various other packages of the Bioconductor project and provides additional ChIP-chip-specific and NimbleGen-specific functionalities.\n",
      "Version: 1.62.0\n",
      "Publication date: None\n",
      "Mantainer: J. Toedling <jtoedling@yahoo.de>\n",
      "Authors:\n",
      "Author: Joern Toedling, Oleg Sklyar, Tammo Krueger, Matt Ritchie, Wolfgang Huber\n",
      "Dependencies:\n",
      "  - methods, type: depends\n",
      "  - Biobase, type: depends\n",
      "  - RColorBrewer, type: depends\n",
      "  - limma, type: depends\n",
      "  - Matrix, type: depends\n",
      "  - grid, type: depends\n",
      "  - lattice, type: depends\n",
      "  - BiocGenerics, type: imports\n",
      "  - genefilter, type: imports\n",
      "  - limma, type: imports\n",
      "  - vsn, type: imports\n",
      "  - stats4, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/Ringo.html\n",
      "\n",
      "Name: BeadDataPackR\n",
      "Description:\n",
      "Provides functionality for the compression and decompression of raw bead-level data from the Illumina BeadArray platform.\n",
      "Version: 1.50.0\n",
      "Publication date: None\n",
      "Mantainer: Mike Smith <grimbough@gmail.com>\n",
      "Authors:\n",
      "Author: Mike Smith, Andy Lynch\n",
      "Dependencies:\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/BeadDataPackR.html\n",
      "\n",
      "Name: beadarray\n",
      "Description:\n",
      "The package is able to read bead-level data (raw TIFFs and text files) output by BeadScan as well as bead-summary data from BeadStudio. Methods for quality assessment and low-level analysis are provided.\n",
      "Version: 2.48.0\n",
      "Publication date: None\n",
      "Mantainer: Mark Dunning <m.j.dunning@sheffield.ac.uk>\n",
      "Authors:\n",
      "Author: Mark Dunning, Mike Smith, Jonathan Cairns, Andy Lynch, Matt Ritchie\n",
      "Dependencies:\n",
      "  - R (>= 2.13.0) type: depends\n",
      "  - BiocGenerics, type: depends\n",
      "  - Biobase, type: depends\n",
      "  - hexbin, type: depends\n",
      "  - BeadDataPackR, type: imports\n",
      "  - limma, type: imports\n",
      "  - AnnotationDbi, type: imports\n",
      "  - stats4, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - illuminaio, type: imports\n",
      "  - methods, type: imports\n",
      "  - ggplot2, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/beadarray.html\n",
      "\n",
      "Name: Repitools\n",
      "Description:\n",
      "Tools for the analysis of enrichment-based epigenomic data. Features include summarization and visualization of epigenomic data across promoters according to gene expression context, finding regions of differential methylation/binding, BayMeth for quantifying methylation etc.\n",
      "Version: 1.44.0\n",
      "Publication date: None\n",
      "Mantainer: Mark Robinson <mark.robinson@imls.uzh.ch>\n",
      "Authors:\n",
      "Author: Mark Robinson <mark.robinson at imls.uzh.ch>, Dario Strbenac <dario.strbenac at sydney.edu.au>, Aaron Statham <a.statham at garvan.org.au>, Andrea Riebler <andrea.riebler at math.ntnu.no>\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - BiocGenerics, type: depends\n",
      "  - parallel, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - GenomicAlignments, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - BSgenome, type: imports\n",
      "  - gplots, type: imports\n",
      "  - grid, type: imports\n",
      "  - MASS, type: imports\n",
      "  - gsmoothr, type: imports\n",
      "  - edgeR, type: imports\n",
      "  - DNAcopy, type: imports\n",
      "  - Ringo, type: imports\n",
      "  - Rsolnp, type: imports\n",
      "  - cluster, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/Repitools.html\n",
      "\n",
      "Name: dittoSeq\n",
      "Description:\n",
      "A universal, user friendly, single-cell and bulk RNA sequencing visualization toolkit that allows highly customizable creation of color blindness friendly, publication-quality figures. dittoSeq accepts both SingleCellExperiment (SCE) and Seurat objects, as well as the import and usage, via conversion to an SCE, of SummarizedExperiment or DGEList bulk data. Visualizations include dimensionality reduction plots, heatmaps, scatterplots, percent composition or expression across groups, and more. Customizations range from size and title adjustments to automatic generation of annotations for heatmaps, overlay of trajectory analysis onto any dimensionality reduciton plot, hidden data overlay upon cursor hovering via ggplotly conversion, and many more. All with simple, discrete inputs. Color blindness friendliness is powered by legend adjustments (enlarged keys), and by allowing the use of shapes or letter-overlay in addition to the carefully selected dittoColors().\n",
      "Version: 1.10.0\n",
      "Publication date: None\n",
      "Mantainer: Daniel Bunis <daniel.bunis@ucsf.edu>\n",
      "Authors:\n",
      "Author: Daniel Bunis [aut, cre], Jared Andrews [aut, ctb]\n",
      "Dependencies:\n",
      "  - ggplot2, type: depends\n",
      "  - methods, type: imports\n",
      "  - colorspace (>= 1.4) type: imports\n",
      "  - gridExtra, type: imports\n",
      "  - cowplot, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - pheatmap, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - ggrepel, type: imports\n",
      "  - ggridges, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - SingleCellExperiment, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/dittoSeq.html\n",
      "\n",
      "Name: fastseg\n",
      "Description:\n",
      "fastseg implements a very fast and efficient segmentation algorithm. It has similar functionality as DNACopy (Olshen and Venkatraman 2004), but is considerably faster and more flexible. fastseg can segment data from DNA microarrays and data from next generation sequencing for example to detect copy number segments. Further it can segment data from RNA microarrays like tiling arrays to identify transcripts. Most generally, it can segment data given as a matrix or as a vector. Various data formats can be used as input to fastseg like expression set objects for microarrays or GRanges for sequencing data. The segmentation criterion of fastseg is based on a statistical test in a Bayesian framework, namely the cyber t-test (Baldi 2001). The speed-up arises from the facts, that sampling is not necessary in for fastseg and that a dynamic programming approach is used for calculation of the segments' first and higher order moments.\n",
      "Version: 1.44.0\n",
      "Publication date: None\n",
      "Mantainer: Alexander Blume <alex.gos90@gmail.com>\n",
      "Authors:\n",
      "Author: Guenter Klambauer\n",
      "Dependencies:\n",
      "  - R (>= 2.13) type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - Biobase, type: depends\n",
      "  - methods, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - stats, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - IRanges, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL (>= 2.0)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/fastseg.html\n",
      "\n",
      "Name: AnVIL\n",
      "Description:\n",
      "The AnVIL is a cloud computing resource developed in part by the National Human Genome Research Institute. The AnVIL package provides end-user and developer functionality. For the end-user, AnVIL provides fast binary package installation, utitlities for working with Terra / AnVIL table and data resources, and convenient functions for file movement to and from Google cloud storage. For developers, AnVIL provides programatic access to the Terra, Leonardo, Rawls, Dockstore, and Gen3 RESTful programming interface, including helper functions to transform JSON responses to formats more amenable to manipulation in R.\n",
      "Version: 1.10.1\n",
      "Publication date: None\n",
      "Mantainer: Martin Morgan <mtmorgan.bioc@gmail.com>\n",
      "Authors:\n",
      "Author: Martin Morgan [aut, cre] , Kayla Interdonato [aut], Nitesh Turaga [aut], BJ Stubbs [ctb], Vincent Carey [ctb], Marcel Ramos [ctb], Sehyun Oh [ctb], Sweta Gopaulakrishnan [ctb], Valerie Obenchain [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 3.6) type: depends\n",
      "  - dplyr, type: depends\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - methods, type: imports\n",
      "  - futile.logger, type: imports\n",
      "  - jsonlite, type: imports\n",
      "  - httr, type: imports\n",
      "  - rapiclient (>= 0.1.3) type: imports\n",
      "  - tibble, type: imports\n",
      "  - tidyselect, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - rlang, type: imports\n",
      "  - shiny, type: imports\n",
      "  - DT, type: imports\n",
      "  - miniUI, type: imports\n",
      "  - htmltools, type: imports\n",
      "  - BiocManager, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/AnVIL.html\n",
      "\n",
      "Name: zinbwave\n",
      "Description:\n",
      "Implements a general and flexible zero-inflated negative binomial model that can be used to provide a low-dimensional representations of single-cell RNA-seq data. The model accounts for zero inflation (dropouts), over-dispersion, and the count nature of the data. The model also accounts for the difference in library sizes and optionally for batch effects and/or other covariates, avoiding the need for pre-normalize the data.\n",
      "Version: 1.20.0\n",
      "Publication date: None\n",
      "Mantainer: Davide Risso <risso.davide@gmail.com>\n",
      "Authors:\n",
      "Author: Davide Risso [aut, cre, cph], Svetlana Gribkova [aut], Fanny Perraudeau [aut], Jean-Philippe Vert [aut], Clara Bagatin [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.4) type: depends\n",
      "  - methods, type: depends\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - SingleCellExperiment, type: depends\n",
      "  - BiocParallel, type: imports\n",
      "  - softImpute, type: imports\n",
      "  - stats, type: imports\n",
      "  - genefilter, type: imports\n",
      "  - edgeR, type: imports\n",
      "  - Matrix, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/zinbwave.html\n",
      "\n",
      "Name: BiocBaseUtils\n",
      "Description:\n",
      "The package provides utility functions related to package development. These include functions that replace slots, and selectors for show methods. It aims to coalesce the various helper functions often re-used throughout the Bioconductor ecosystem.\n",
      "Version: 1.0.0\n",
      "Publication date: None\n",
      "Mantainer: Marcel Ramos <marcel.ramos@roswellpark.org>\n",
      "Authors:\n",
      "Author: Marcel Ramos [aut, cre] , Martin Morgan [ctb], HervÃ© PagÃ¨s [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 4.2.0) type: depends\n",
      "  - methods, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/BiocBaseUtils.html\n",
      "\n",
      "Name: TreeSummarizedExperiment\n",
      "Description:\n",
      "TreeSummarizedExperiment has extended SingleCellExperiment to include hierarchical information on the rows or columns of the rectangular data.\n",
      "Version: 2.6.0\n",
      "Publication date: None\n",
      "Mantainer: Ruizhu Huang <ruizhuRH@gmail.com>\n",
      "Authors:\n",
      "Author: Ruizhu Huang [aut, cre] , Felix G.M. Ernst [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 3.6.0) type: depends\n",
      "  - SingleCellExperiment, type: depends\n",
      "  - S4Vectors, type: depends\n",
      "  - Biostrings, type: depends\n",
      "  - methods, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - utils, type: imports\n",
      "  - ape, type: imports\n",
      "  - rlang, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - treeio, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>=2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/TreeSummarizedExperiment.html\n",
      "\n",
      "Name: ChAMP\n",
      "Description:\n",
      "The package includes quality control metrics, a selection of normalization methods and novel methods to identify differentially methylated regions and to highlight copy number alterations.\n",
      "Version: 2.28.0\n",
      "Publication date: None\n",
      "Mantainer: Yuan Tian <champ450k@gmail.com>\n",
      "Authors:\n",
      "Author: Yuan Tian [cre,aut], Tiffany Morris [ctb], Lee Stirling [ctb], Andrew Feber [ctb], Andrew Teschendorff [ctb], Ankur Chakravarthy [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 3.3) type: depends\n",
      "  - minfi, type: depends\n",
      "  - ChAMPdata, type: depends\n",
      "  - DMRcate, type: depends\n",
      "  - Illumina450ProbeVariants.db, type: depends\n",
      "  - IlluminaHumanMethylationEPICmanifest, type: depends\n",
      "  - DT, type: depends\n",
      "  - RPMM, type: depends\n",
      "  - prettydoc, type: imports\n",
      "  - Hmisc, type: imports\n",
      "  - globaltest, type: imports\n",
      "  - sva, type: imports\n",
      "  - illuminaio, type: imports\n",
      "  - rmarkdown, type: imports\n",
      "  - IlluminaHumanMethylation450kmanifest, type: imports\n",
      "  - IlluminaHumanMethylationEPICanno.ilm10b4.hg19, type: imports\n",
      "  - limma, type: imports\n",
      "  - DNAcopy, type: imports\n",
      "  - preprocessCore, type: imports\n",
      "  - impute, type: imports\n",
      "  - marray, type: imports\n",
      "  - wateRmelon, type: imports\n",
      "  - plyr, type: imports\n",
      "  - goseq, type: imports\n",
      "  - missMethyl, type: imports\n",
      "  - kpmt, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - qvalue, type: imports\n",
      "  - isva, type: imports\n",
      "  - doParallel, type: imports\n",
      "  - bumphunter, type: imports\n",
      "  - quadprog, type: imports\n",
      "  - shiny, type: imports\n",
      "  - shinythemes, type: imports\n",
      "  - plotly (>= 4.5.6) type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - dendextend, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - combinat, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ChAMP.html\n",
      "\n",
      "Name: Nebulosa\n",
      "Description:\n",
      "This package provides a enhanced visualization of single-cell data based on gene-weighted density estimation. Nebulosa recovers the signal from dropped-out features and allows the inspection of the joint expression from multiple features (e.g. genes). Seurat and SingleCellExperiment objects can be used within Nebulosa.\n",
      "Version: 1.8.0\n",
      "Publication date: None\n",
      "Mantainer: Jose Alquicira-Hernandez <alquicirajose@gmail.com>\n",
      "Authors:\n",
      "Author: Jose Alquicira-Hernandez [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - ggplot2, type: depends\n",
      "  - patchwork, type: depends\n",
      "  - Seurat, type: imports\n",
      "  - SingleCellExperiment, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - ks, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - stats, type: imports\n",
      "  - methods, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/Nebulosa.html\n",
      "\n",
      "Name: GlobalAncova\n",
      "Description:\n",
      "The association between a variable of interest (e.g. two groups) and the global pattern of a group of variables (e.g. a gene set) is tested via a global F-test. We give the following arguments in support of the GlobalAncova approach: After appropriate normalisation, gene-expression-data appear rather symmetrical and outliers are no real problem, so least squares should be rather robust. ANCOVA with interaction yields saturated data modelling e.g. different means per group and gene. Covariate adjustment can help to correct for possible selection bias. Variance homogeneity and uncorrelated residuals cannot be expected. Application of ordinary least squares gives unbiased, but no longer optimal estimates (Gauss-Markov-Aitken). Therefore, using the classical F-test is inappropriate, due to correlation. The test statistic however mirrors deviations from the null hypothesis. In combination with a permutation approach, empirical significance levels can be approximated. Alternatively, an approximation yields asymptotic p-values. The framework is generalized to groups of categorical variables or even mixed data by a likelihood ratio approach. Closed and hierarchical testing procedures are supported. This work was supported by the NGFN grant 01 GR 0459, BMBF, Germany and BMBF grant 01ZX1309B, Germany.\n",
      "Version: 4.16.0\n",
      "Publication date: None\n",
      "Mantainer: Manuela Hummel <manuela.hummel@web.de>\n",
      "Authors:\n",
      "Author: U. Mansmann, R. Meister, M. Hummel, R. Scheufele, with contributions from S. Knueppel\n",
      "Dependencies:\n",
      "  - methods, type: depends\n",
      "  - corpcor, type: depends\n",
      "  - globaltest, type: depends\n",
      "  - annotate, type: imports\n",
      "  - AnnotationDbi, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - dendextend, type: imports\n",
      "  - GSEABase, type: imports\n",
      "  - VGAM, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/GlobalAncova.html\n",
      "\n",
      "Name: viper\n",
      "Description:\n",
      "Inference of protein activity from gene expression data, including the VIPER and msVIPER algorithms\n",
      "Version: 1.32.0\n",
      "Publication date: None\n",
      "Mantainer: Mariano J Alvarez <reef103@gmail.com>\n",
      "Authors:\n",
      "Author: Mariano J Alvarez <reef103 at gmail.com>\n",
      "Dependencies:\n",
      "  - R (>= 2.14.0) type: depends\n",
      "  - Biobase, type: depends\n",
      "  - methods, type: depends\n",
      "  - mixtools, type: imports\n",
      "  - stats, type: imports\n",
      "  - parallel, type: imports\n",
      "  - e1071, type: imports\n",
      "  - KernSmooth, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/viper.html\n",
      "\n",
      "Name: IHW\n",
      "Description:\n",
      "Independent hypothesis weighting (IHW) is a multiple testing procedure that increases power compared to the method of Benjamini and Hochberg by assigning data-driven weights to each hypothesis. The input to IHW is a two-column table of p-values and covariates. The covariate can be any continuous-valued or categorical variable that is thought to be informative on the statistical properties of each hypothesis test, while it is independent of the p-value under the null hypothesis.\n",
      "Version: 1.26.0\n",
      "Publication date: None\n",
      "Mantainer: Nikos Ignatiadis <nikos.ignatiadis01@gmail.com>\n",
      "Authors:\n",
      "Author: Nikos Ignatiadis [aut, cre], Wolfgang Huber [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.3.0) type: depends\n",
      "  - methods, type: imports\n",
      "  - slam, type: imports\n",
      "  - lpsymphony, type: imports\n",
      "  - fdrtool, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/IHW.html\n",
      "\n",
      "Name: mia\n",
      "Description:\n",
      "mia implements tools for microbiome analysis based on the SummarizedExperiment, SingleCellExperiment and TreeSummarizedExperiment infrastructure. Data wrangling and analysis in the context of taxonomic data is the main scope. Additional functions for common task are implemented such as community indices calculation and summarization.\n",
      "Version: 1.6.0\n",
      "Publication date: None\n",
      "Mantainer: Tuomas Borman <tuomas.v.borman@utu.fi>\n",
      "Authors:\n",
      "Author: Felix G.M. Ernst [aut] , Sudarshan A. Shetty [aut] , Tuomas Borman [aut, cre] , Leo Lahti [aut] , Yang Cao [ctb], Nathan D. Olson [ctb], Levi Waldron [ctb], Marcel Ramos [ctb], HÃ©ctor Corrada Bravo [ctb], Jayaram Kancherla [ctb], Domenick Braccia [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - SingleCellExperiment, type: depends\n",
      "  - TreeSummarizedExperiment, type: depends\n",
      "  - MultiAssayExperiment, type: depends\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - MASS, type: imports\n",
      "  - ape, type: imports\n",
      "  - decontam, type: imports\n",
      "  - vegan, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - DECIPHER, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - DelayedArray, type: imports\n",
      "  - DelayedMatrixStats, type: imports\n",
      "  - scuttle, type: imports\n",
      "  - scater, type: imports\n",
      "  - DirichletMultinomial, type: imports\n",
      "  - rlang, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - tibble, type: imports\n",
      "  - tidyr, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0 | file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/mia.html\n",
      "\n",
      "Name: quantsmooth\n",
      "Description:\n",
      "Implements quantile smoothing as introduced in: Quantile smoothing of array CGH data; Eilers PH, de Menezes RX; Bioinformatics. 2005 Apr 1;21(7):1146-53.\n",
      "Version: 1.64.0\n",
      "Publication date: None\n",
      "Mantainer: Jan Oosting <j.oosting@lumc.nl>\n",
      "Authors:\n",
      "Author: Jan Oosting, Paul Eilers, Renee Menezes\n",
      "Dependencies:\n",
      "  - R (>= 2.10.0) type: depends\n",
      "  - quantreg, type: depends\n",
      "  - grid, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/quantsmooth.html\n",
      "\n",
      "Name: motifStack\n",
      "Description:\n",
      "The motifStack package is designed for graphic representation of multiple motifs with different similarity scores. It works with both DNA/RNA sequence motif and amino acid sequence motif. In addition, it provides the flexibility for users to customize the graphic parameters such as the font type and symbol colors.\n",
      "Version: 1.42.0\n",
      "Publication date: None\n",
      "Mantainer: Jianhong Ou <jianhong.ou@duke.edu>\n",
      "Authors:\n",
      "Author: Jianhong Ou, Michael Brodsky, Scot Wolfe and Lihua Julie Zhu\n",
      "Dependencies:\n",
      "  - R (>= 2.15.1) type: depends\n",
      "  - methods, type: depends\n",
      "  - grid, type: depends\n",
      "  - ade4, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - htmlwidgets, type: imports\n",
      "  - stats, type: imports\n",
      "  - stats4, type: imports\n",
      "  - utils, type: imports\n",
      "  - XML, type: imports\n",
      "  - TFBSTools, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/motifStack.html\n",
      "\n",
      "Name: ChemmineR\n",
      "Description:\n",
      "ChemmineR is a cheminformatics package for analyzing drug-like small molecule data in R. Its latest version contains functions for efficient processing of large numbers of molecules, physicochemical/structural property predictions, structural similarity searching, classification and clustering of compound libraries with a wide spectrum of algorithms. In addition, it offers visualization functions for compound clustering results and chemical structures.\n",
      "Version: 3.50.0\n",
      "Publication date: None\n",
      "Mantainer: Thomas Girke <thomas.girke@ucr.edu>\n",
      "Authors:\n",
      "Author: Y. Eddie Cao, Kevin Horan, Tyler Backman, Thomas Girke\n",
      "Dependencies:\n",
      "  - R (>= 2.10.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - rjson, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - RCurl, type: imports\n",
      "  - DBI, type: imports\n",
      "  - digest, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - Rcpp (>= 0.11.0) type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - grid, type: imports\n",
      "  - gridExtra, type: imports\n",
      "  - png, type: imports\n",
      "  - base64enc, type: imports\n",
      "  - DT, type: imports\n",
      "  - rsvg, type: imports\n",
      "  - jsonlite, type: imports\n",
      "  - stringi, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ChemmineR.html\n",
      "\n",
      "Name: GWASTools\n",
      "Description:\n",
      "Classes for storing very large GWAS data sets and annotation, and functions for GWAS data cleaning and analysis.\n",
      "Version: 1.44.0\n",
      "Publication date: None\n",
      "Mantainer: Stephanie M. Gogarten <sdmorris@uw.edu>\n",
      "Authors:\n",
      "Author: Stephanie M. Gogarten, Cathy Laurie, Tushar Bhangale, Matthew P. Conomos, Cecelia Laurie, Michael Lawrence, Caitlin McHugh, Ian Painter, Xiuwen Zheng, Jess Shen, Rohit Swarnkar, Adrienne Stilp, Sarah Nelson, David Levine\n",
      "Dependencies:\n",
      "  - Biobase, type: depends\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - methods, type: imports\n",
      "  - gdsfmt, type: imports\n",
      "  - DBI, type: imports\n",
      "  - RSQLite, type: imports\n",
      "  - GWASExactHW, type: imports\n",
      "  - DNAcopy, type: imports\n",
      "  - survival, type: imports\n",
      "  - sandwich, type: imports\n",
      "  - lmtest, type: imports\n",
      "  - logistf, type: imports\n",
      "  - quantsmooth, type: imports\n",
      "  - data.table, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/GWASTools.html\n",
      "\n",
      "Name: minet\n",
      "Description:\n",
      "This package implements various algorithms for inferring mutual information networks from data.\n",
      "Version: 3.56.0\n",
      "Publication date: None\n",
      "Mantainer: Patrick E. Meyer <software@meyerp.com>\n",
      "Authors:\n",
      "Author: Patrick E. Meyer, Frederic Lafitte, Gianluca Bontempi\n",
      "Dependencies:\n",
      "  - infotheo, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/minet.html\n",
      "\n",
      "Name: baySeq\n",
      "Description:\n",
      "This package identifies differential expression in high-throughput 'count' data, such as that derived from next-generation sequencing machines, calculating estimated posterior likelihoods of differential expression (or more complex hypotheses) via empirical Bayesian methods.\n",
      "Version: 2.31.0\n",
      "Publication date: None\n",
      "Mantainer: Thomas J. Hardcastle <tjh48@cam.ac.uk>\n",
      "Authors:\n",
      "Author: Thomas J. Hardcastle\n",
      "Dependencies:\n",
      "  - R (>= 2.3.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - abind, type: depends\n",
      "  - parallel, type: depends\n",
      "  - edgeR, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/baySeq.html\n",
      "\n",
      "Name: ballgown\n",
      "Description:\n",
      "Tools for statistical analysis of assembled transcriptomes, including flexible differential expression analysis, visualization of transcript structures, and matching of assembled transcripts to annotation.\n",
      "Version: 2.30.0\n",
      "Publication date: None\n",
      "Mantainer: Jack Fu <jmfu@jhsph.edu>\n",
      "Authors:\n",
      "Author: Jack Fu [aut], Alyssa C. Frazee [aut, cre], Leonardo Collado-Torres [aut], Andrew E. Jaffe [aut], Jeffrey T. Leek [aut, ths]\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - GenomicRanges, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - splines, type: imports\n",
      "  - sva, type: imports\n",
      "  - limma, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ballgown.html\n",
      "\n",
      "Name: ArrayExpress\n",
      "Description:\n",
      "Access the ArrayExpress Repository at EBI and build Bioconductor data structures: ExpressionSet, AffyBatch, NChannelSet\n",
      "Version: 1.57.0\n",
      "Publication date: None\n",
      "Mantainer: Suhaib Mohammed <suhaib@ebi.ac.uk>\n",
      "Authors:\n",
      "Author: Audrey Kauffmann, Ibrahim Emam, Michael Schubert\n",
      "Dependencies:\n",
      "  - R (>= 2.9.0) type: depends\n",
      "  - Biobase, type: depends\n",
      "  - XML, type: imports\n",
      "  - oligo, type: imports\n",
      "  - limma, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ArrayExpress.html\n",
      "\n",
      "Name: genomation\n",
      "Description:\n",
      "A package for summary and annotation of genomic intervals. Users can visualize and quantify genomic intervals over pre-defined functional regions, such as promoters, exons, introns, etc. The genomic intervals represent regions with a defined chromosome position, which may be associated with a score, such as aligned reads from HT-seq experiments, TF binding sites, methylation scores, etc. The package can use any tabular genomic feature data as long as it has minimal information on the locations of genomic intervals. In addition, It can use BAM or BigWig files as input.\n",
      "Version: 1.30.0\n",
      "Publication date: None\n",
      "Mantainer: Altuna Akalin <aakalin@gmail.com>, Vedran Franke <vedran.franke@gmail.com>, Katarzyna Wreczycka <katwre@gmail.com>\n",
      "Authors:\n",
      "Author: Altuna Akalin [aut, cre], Vedran Franke [aut, cre], Katarzyna Wreczycka [aut], Alexander Gosdschan [ctb], Liz Ing-Simmons [ctb], Bozena Mika-Gospodorz [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - grid, type: depends\n",
      "  - Biostrings, type: imports\n",
      "  - BSgenome, type: imports\n",
      "  - data.table, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - GenomicAlignments, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - gridBase, type: imports\n",
      "  - impute, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - methods, type: imports\n",
      "  - parallel, type: imports\n",
      "  - plotrix, type: imports\n",
      "  - plyr, type: imports\n",
      "  - readr, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - seqPattern, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - Rcpp (>= 0.12.14) type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/genomation.html\n",
      "\n",
      "Name: seqPattern\n",
      "Description:\n",
      "Visualising oligonucleotide patterns and sequence motifs occurrences across a large set of sequences centred at a common reference point and sorted by a user defined feature.\n",
      "Version: 1.30.0\n",
      "Publication date: None\n",
      "Mantainer: Vanja Haberle <vanja.haberle@gmail.com>\n",
      "Authors:\n",
      "Author: Vanja Haberle <vanja.haberle at gmail.com>\n",
      "Dependencies:\n",
      "  - methods, type: depends\n",
      "  - R (>= 2.15.0) type: depends\n",
      "  - Biostrings, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - KernSmooth, type: imports\n",
      "  - plotrix, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/seqPattern.html\n",
      "\n",
      "Name: NOISeq\n",
      "Description:\n",
      "Analysis of RNA-seq expression data or other similar kind of data. Exploratory plots to evualuate saturation, count distribution, expression per chromosome, type of detected features, features length, etc. Differential expression between two experimental conditions with no parametric assumptions.\n",
      "Version: 2.42.0\n",
      "Publication date: None\n",
      "Mantainer: Sonia Tarazona <sotacam@eio.upv.es>\n",
      "Authors:\n",
      "Author: Sonia Tarazona, Pedro Furio-Tari, Maria Jose Nueda, Alberto Ferrer and Ana Conesa\n",
      "Dependencies:\n",
      "  - R (>= 2.13.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - Biobase, type: depends\n",
      "  - splines (>= 3.0.1) type: depends\n",
      "  - Matrix (>= 1.2) type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/NOISeq.html\n",
      "\n",
      "Name: zellkonverter\n",
      "Description:\n",
      "Provides methods to convert between Python AnnData objects and SingleCellExperiment objects. These are primarily intended for use by downstream Bioconductor packages that wrap Python methods for single-cell data analysis. It also includes functions to read and write H5AD files used for saving AnnData objects to disk.\n",
      "Version: 1.8.0\n",
      "Publication date: None\n",
      "Mantainer: Luke Zappia <luke@lazappi.id.au>\n",
      "Authors:\n",
      "Author: Luke Zappia [aut, cre] , Aaron Lun [aut]\n",
      "Dependencies:\n",
      "  - Matrix, type: imports\n",
      "  - basilisk, type: imports\n",
      "  - reticulate, type: imports\n",
      "  - SingleCellExperiment, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - DelayedArray, type: imports\n",
      "  - methods, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - utils, type: imports\n",
      "  - cli, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/zellkonverter.html\n",
      "\n",
      "Name: RTCGAToolbox\n",
      "Description:\n",
      "Managing data from large scale projects such as The Cancer Genome Atlas (TCGA) for further analysis is an important and time consuming step for research projects. Several efforts, such as Firehose project, make TCGA pre-processed data publicly available via web services and data portals but it requires managing, downloading and preparing the data for following steps. We developed an open source and extensible R based data client for Firehose pre-processed data and demonstrated its use with sample case studies. Results showed that RTCGAToolbox could improve data management for researchers who are interested with TCGA data. In addition, it can be integrated with other analysis pipelines for following data analysis.\n",
      "Version: 2.28.0\n",
      "Publication date: None\n",
      "Mantainer: Marcel Ramos <marcel.ramos@roswellpark.org>\n",
      "Authors:\n",
      "Author: Mehmet Samur [aut], Marcel Ramos [aut, cre], Ludwig Geistlinger [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - BiocGenerics, type: imports\n",
      "  - data.table, type: imports\n",
      "  - DelayedArray, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - httr, type: imports\n",
      "  - limma, type: imports\n",
      "  - methods, type: imports\n",
      "  - RaggedExperiment, type: imports\n",
      "  - RCircos, type: imports\n",
      "  - RCurl, type: imports\n",
      "  - RJSONIO, type: imports\n",
      "  - rvest, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - stats, type: imports\n",
      "  - stringr, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - survival, type: imports\n",
      "  - TCGAutils, type: imports\n",
      "  - XML, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/RTCGAToolbox.html\n",
      "\n",
      "Name: arrayQualityMetrics\n",
      "Description:\n",
      "This package generates microarray quality metrics reports for data in Bioconductor microarray data containers (ExpressionSet, NChannelSet, AffyBatch). One and two color array platforms are supported.\n",
      "Version: 3.54.0\n",
      "Publication date: None\n",
      "Mantainer: Mike Smith <mike.smith@embl.de>\n",
      "Authors:\n",
      "Author: Audrey Kauffmann, Wolfgang Huber\n",
      "Dependencies:\n",
      "  - affy, type: imports\n",
      "  - affyPLM, type: imports\n",
      "  - beadarray, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - genefilter, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - grid, type: imports\n",
      "  - gridSVG (>= 1.4-3) type: imports\n",
      "  - Hmisc, type: imports\n",
      "  - hwriter, type: imports\n",
      "  - lattice, type: imports\n",
      "  - latticeExtra, type: imports\n",
      "  - limma, type: imports\n",
      "  - methods, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - setRNG, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - vsn, type: imports\n",
      "  - XML, type: imports\n",
      "  - svglite, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/arrayQualityMetrics.html\n",
      "\n",
      "Name: chipseq\n",
      "Description:\n",
      "Tools for helping process short read data for chipseq experiments\n",
      "Version: 1.48.0\n",
      "Publication date: None\n",
      "Mantainer: Bioconductor Package Maintainer <maintainer@bioconductor.org>\n",
      "Authors:\n",
      "Author: Deepayan Sarkar, Robert Gentleman, Michael Lawrence, Zizhen Yao\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - BiocGenerics, type: depends\n",
      "  - S4Vectors, type: depends\n",
      "  - IRanges, type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - ShortRead, type: depends\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - lattice, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - ShortRead, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/chipseq.html\n",
      "\n",
      "Name: RTCGA\n",
      "Description:\n",
      "The Cancer Genome Atlas (TCGA) Data Portal provides a platform for researchers to search, download, and analyze data sets generated by TCGA. It contains clinical information, genomic characterization data, and high level sequence analysis of the tumor genomes. The key is to understand genomics to improve cancer care. RTCGA package offers download and integration of the variety and volume of TCGA data using patient barcode key, what enables easier data possession. This may have an benefcial infuence on impact on development of science and improvement of patients' treatment. Furthermore, RTCGA package transforms TCGA data to tidy form which is convenient to use.\n",
      "Version: 1.28.0\n",
      "Publication date: None\n",
      "Mantainer: Marcin Kosinski <m.p.kosinski@gmail.com>\n",
      "Authors:\n",
      "Author: Marcin Kosinski [aut, cre], Przemyslaw Biecek [ctb], Witold Chodor [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 3.3.0) type: depends\n",
      "  - XML, type: imports\n",
      "  - RCurl, type: imports\n",
      "  - assertthat, type: imports\n",
      "  - stringi, type: imports\n",
      "  - rvest, type: imports\n",
      "  - data.table, type: imports\n",
      "  - xml2, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - purrr, type: imports\n",
      "  - survival, type: imports\n",
      "  - survminer, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - ggthemes, type: imports\n",
      "  - viridis, type: imports\n",
      "  - knitr, type: imports\n",
      "  - scales, type: imports\n",
      "  - rmarkdown, type: imports\n",
      "  - htmltools, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/RTCGA.html\n",
      "\n",
      "Name: methylKit\n",
      "Description:\n",
      "methylKit is an R package for DNA methylation analysis and annotation from high-throughput bisulfite sequencing. The package is designed to deal with sequencing data from RRBS and its variants, but also target-capture methods and whole genome bisulfite sequencing. It also has functions to analyze base-pair resolution 5hmC data from experimental protocols such as oxBS-Seq and TAB-Seq. Methylation calling can be performed directly from Bismark aligned BAM files.\n",
      "Version: 1.24.0\n",
      "Publication date: None\n",
      "Mantainer: Altuna Akalin <aakalin@gmail.com>, Alexander Gosdschan <alex.gos90@gmail.com>\n",
      "Authors:\n",
      "Author: Altuna Akalin [aut, cre], Matthias Kormaksson [aut], Sheng Li [aut], Arsene Wabo [ctb], Adrian Bierling [aut], Alexander Gosdschan [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - methods, type: depends\n",
      "  - IRanges, type: imports\n",
      "  - data.table (>= 1.9.6) type: imports\n",
      "  - parallel, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - KernSmooth, type: imports\n",
      "  - qvalue, type: imports\n",
      "  - emdbook, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - gtools, type: imports\n",
      "  - fastseg, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - mclust, type: imports\n",
      "  - mgcv, type: imports\n",
      "  - Rcpp, type: imports\n",
      "  - R.utils, type: imports\n",
      "  - limma, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/methylKit.html\n",
      "\n",
      "Name: scds\n",
      "Description:\n",
      "In single cell RNA sequencing (scRNA-seq) data combinations of cells are sometimes considered a single cell (doublets). The scds package provides methods to annotate doublets in scRNA-seq data computationally.\n",
      "Version: 1.14.0\n",
      "Publication date: None\n",
      "Mantainer: Dennis Kostka <kostka@pitt.edu>\n",
      "Authors:\n",
      "Author: Dennis Kostka [aut, cre], Bais Abha [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.6.0) type: depends\n",
      "  - Matrix, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - SingleCellExperiment, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - xgboost, type: imports\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - pROC, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/scds.html\n",
      "\n",
      "Name: MotifDb\n",
      "Description:\n",
      "More than 9900 annotated position frequency matrices from 14 public sources, for multiple organisms.\n",
      "Version: 1.40.0\n",
      "Publication date: None\n",
      "Mantainer: Paul Shannon <pshannon@systemsbiology.org>\n",
      "Authors:\n",
      "Author: Paul Shannon, Matt Richards\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - BiocGenerics, type: depends\n",
      "  - S4Vectors, type: depends\n",
      "  - IRanges, type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - Biostrings, type: depends\n",
      "  - rtracklayer, type: imports\n",
      "  - splitstackshape, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0 | file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/MotifDb.html\n",
      "\n",
      "Name: DEGreport\n",
      "Description:\n",
      "Creation of a HTML report of differential expression analyses of count data. It integrates some of the code mentioned in DESeq2 and edgeR vignettes, and report a ranked list of genes according to the fold changes mean and variability for each selected gene.\n",
      "Version: 1.34.0\n",
      "Publication date: None\n",
      "Mantainer: Lorena Pantano <lorena.pantano@gmail.com>\n",
      "Authors:\n",
      "Author: Lorena Pantano [aut, cre], John Hutchinson [ctb], Victor Barrera [ctb], Mary Piper [ctb], Radhika Khetani [ctb], Kenneth Daily [ctb], Thanneer Malai Perumal [ctb], Rory Kirchner [ctb], Michael Steinbaugh [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 4.0.0) type: depends\n",
      "  - utils, type: imports\n",
      "  - methods, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - broom, type: imports\n",
      "  - circlize, type: imports\n",
      "  - ComplexHeatmap, type: imports\n",
      "  - cowplot, type: imports\n",
      "  - ConsensusClusterPlus, type: imports\n",
      "  - cluster, type: imports\n",
      "  - DESeq2, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - edgeR, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - ggdendro, type: imports\n",
      "  - grid, type: imports\n",
      "  - ggrepel, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - knitr, type: imports\n",
      "  - logging, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - psych, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - reshape, type: imports\n",
      "  - rlang, type: imports\n",
      "  - scales, type: imports\n",
      "  - stats, type: imports\n",
      "  - stringr, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - tibble, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/DEGreport.html\n",
      "\n",
      "Name: R4RNA\n",
      "Description:\n",
      "A package for RNA basepair analysis, including the visualization of basepairs as arc diagrams for easy comparison and annotation of sequence and structure. Arc diagrams can additionally be projected onto multiple sequence alignments to assess basepair conservation and covariation, with numerical methods for computing statistics for each.\n",
      "Version: 1.26.0\n",
      "Publication date: None\n",
      "Mantainer: Daniel Lai <jujubix@cs.ubc.ca>\n",
      "Authors:\n",
      "Author: Daniel Lai, Irmtraud Meyer <irmtraud.meyer at cantab.net>\n",
      "Dependencies:\n",
      "  - R (>= 3.2.0) type: depends\n",
      "  - Biostrings, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/R4RNA.html\n",
      "\n",
      "Name: variancePartition\n",
      "Description:\n",
      "Quantify and interpret multiple sources of biological and technical variation in gene expression experiments. Uses a linear mixed model to quantify variation in gene expression attributable to individual, tissue, time point, or technical variables. Includes dream differential expression analysis for repeated measures.\n",
      "Version: 1.28.4\n",
      "Publication date: None\n",
      "Mantainer: Gabriel E. Hoffman <gabriel.hoffman@mssm.edu>\n",
      "Authors:\n",
      "Author: Gabriel Hoffman [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.0.0) type: depends\n",
      "  - ggplot2, type: depends\n",
      "  - limma, type: depends\n",
      "  - BiocParallel, type: depends\n",
      "  - MASS, type: imports\n",
      "  - pbkrtest (>= 0.4-4) type: imports\n",
      "  - lmerTest, type: imports\n",
      "  - Matrix (>= 1.4.0) type: imports\n",
      "  - iterators, type: imports\n",
      "  - foreach, type: imports\n",
      "  - doParallel, type: imports\n",
      "  - gplots, type: imports\n",
      "  - RhpcBLASctl, type: imports\n",
      "  - progress, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - remaCor (>= 0.0.11) type: imports\n",
      "  - aod, type: imports\n",
      "  - scales, type: imports\n",
      "  - Rdpack, type: imports\n",
      "  - rlang, type: imports\n",
      "  - lme4 (>= 1.1-10) type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - methods, type: imports\n",
      "  - utils, type: imports\n",
      "  - stats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/variancePartition.html\n",
      "\n",
      "Name: flowAI\n",
      "Description:\n",
      "The package is able to perform an automatic or interactive quality control on FCS data acquired using flow cytometry instruments. By evaluating three different properties: 1) flow rate, 2) signal acquisition, 3) dynamic range, the quality control enables the detection and removal of anomalies.\n",
      "Version: 1.28.0\n",
      "Publication date: None\n",
      "Mantainer: Gianni Monaco <mongianni1@gmail.com>\n",
      "Authors:\n",
      "Author: Gianni Monaco, Hao Chen\n",
      "Dependencies:\n",
      "  - R (>= 3.6) type: depends\n",
      "  - ggplot2, type: imports\n",
      "  - flowCore, type: imports\n",
      "  - plyr, type: imports\n",
      "  - changepoint, type: imports\n",
      "  - knitr, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - scales, type: imports\n",
      "  - methods, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - rmarkdown, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/flowAI.html\n",
      "\n",
      "Name: derfinder\n",
      "Description:\n",
      "This package provides functions for annotation-agnostic differential expression analysis of RNA-seq data. Two implementations of the DER Finder approach are included in this package: (1) single base-level F-statistics and (2) DER identification at the expressed regions-level. The DER Finder approach can also be used to identify differentially bounded ChIP-seq peaks.\n",
      "Version: 1.32.0\n",
      "Publication date: None\n",
      "Mantainer: Leonardo Collado-Torres <lcolladotor@gmail.com>\n",
      "Authors:\n",
      "Author: Leonardo Collado-Torres [aut, cre] , Alyssa C. Frazee [ctb], Andrew E. Jaffe [aut] , Jeffrey T. Leek [aut, ths]\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - BiocGenerics, type: imports\n",
      "  - AnnotationDbi, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - bumphunter, type: imports\n",
      "  - derfinderHelper, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - GenomicAlignments, type: imports\n",
      "  - GenomicFeatures, type: imports\n",
      "  - GenomicFiles, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - Hmisc, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - methods, type: imports\n",
      "  - qvalue, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/derfinder.html\n",
      "\n",
      "Name: derfinderHelper\n",
      "Description:\n",
      "Helper package for speeding up the derfinder package when using multiple cores. This package is particularly useful when using BiocParallel and it helps reduce the time spent loading the full derfinder package when running the F-statistics calculation in parallel.\n",
      "Version: 1.32.0\n",
      "Publication date: None\n",
      "Mantainer: Leonardo Collado-Torres <lcolladotor@gmail.com>\n",
      "Authors:\n",
      "Author: Leonardo Collado-Torres [aut, cre] , Andrew E. Jaffe [aut] , Jeffrey T. Leek [aut, ths]\n",
      "Dependencies:\n",
      "  - R (>= 3.2.2) type: depends\n",
      "  - IRanges, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - methods, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/derfinderHelper.html\n",
      "\n",
      "Name: rGREAT\n",
      "Description:\n",
      "GREAT (Genomic Regions Enrichment of Annotations Tool) is a type of functional enrichment analysis directly performed on genomic regions. This package implements the GREAT algorithm (the local GREAT analysis), also it supports directly interacting with the GREAT web service (the online GREAT analysis). Both analysis can be viewed by a Shiny application. rGREAT by default supports more than 600 organisms and a large number of gene set collections, as well as self-provided gene sets and organisms from users. Additionally, it implements a general method for dealing with background regions.\n",
      "Version: 2.0.2\n",
      "Publication date: None\n",
      "Mantainer: Zuguang Gu <z.gu@dkfz.de>\n",
      "Authors:\n",
      "Author: Zuguang Gu [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 3.6.0) type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - IRanges, type: depends\n",
      "  - methods, type: depends\n",
      "  - graphics, type: imports\n",
      "  - rjson, type: imports\n",
      "  - GetoptLong (>= 0.0.9) type: imports\n",
      "  - RCurl, type: imports\n",
      "  - utils, type: imports\n",
      "  - stats, type: imports\n",
      "  - GlobalOptions, type: imports\n",
      "  - shiny, type: imports\n",
      "  - DT, type: imports\n",
      "  - GenomicFeatures, type: imports\n",
      "  - digest, type: imports\n",
      "  - GO.db, type: imports\n",
      "  - progress, type: imports\n",
      "  - circlize, type: imports\n",
      "  - AnnotationDbi, type: imports\n",
      "  - TxDb.Hsapiens.UCSC.hg19.knownGene, type: imports\n",
      "  - TxDb.Hsapiens.UCSC.hg38.knownGene, type: imports\n",
      "  - org.Hs.eg.db, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - foreach, type: imports\n",
      "  - doParallel, type: imports\n",
      "  - Rcpp, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/rGREAT.html\n",
      "\n",
      "Name: SeqVarTools\n",
      "Description:\n",
      "An interface to the fast-access storage format for VCF data provided in SeqArray, with tools for common operations and analysis.\n",
      "Version: 1.36.0\n",
      "Publication date: None\n",
      "Mantainer: Stephanie M. Gogarten <sdmorris@uw.edu>\n",
      "Authors:\n",
      "Author: Stephanie M. Gogarten, Xiuwen Zheng, Adrienne Stilp\n",
      "Dependencies:\n",
      "  - SeqArray, type: depends\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - methods, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - gdsfmt, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - GWASExactHW, type: imports\n",
      "  - logistf, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - data.table, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/SeqVarTools.html\n",
      "\n",
      "Name: CAMERA\n",
      "Description:\n",
      "Annotation of peaklists generated by xcms, rule based annotation of isotopes and adducts, isotope validation, EIC correlation based tagging of unknown adducts and fragments\n",
      "Version: 1.54.0\n",
      "Publication date: None\n",
      "Mantainer: Steffen Neumann <sneumann@ipb-halle.de>\n",
      "Authors:\n",
      "Author: Carsten Kuhl, Ralf Tautenhahn, Hendrik Treutler, Steffen Neumann {ckuhl|htreutle|sneumann}@ipb-halle.de, rtautenh@scripps.edu\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - Biobase, type: depends\n",
      "  - xcms, type: depends\n",
      "  - methods, type: imports\n",
      "  - xcms, type: imports\n",
      "  - RBGL, type: imports\n",
      "  - graph, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - Hmisc, type: imports\n",
      "  - igraph, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/CAMERA.html\n",
      "\n",
      "Name: Maaslin2\n",
      "Description:\n",
      "MaAsLin2 is comprehensive R package for efficiently determining multivariable association between clinical metadata and microbial meta'omic features. MaAsLin2 relies on general linear models to accommodate most modern epidemiological study designs, including cross-sectional and longitudinal, and offers a variety of data exploration, normalization, and transformation methods. MaAsLin2 is the next generation of MaAsLin.\n",
      "Version: 1.12.0\n",
      "Publication date: None\n",
      "Mantainer: Lauren McIver <lauren.j.mciver@gmail.com>\n",
      "Authors:\n",
      "Author: Himel Mallick [aut], Ali Rahnavard [aut], Lauren McIver [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 3.6) type: depends\n",
      "  - robustbase, type: imports\n",
      "  - biglm, type: imports\n",
      "  - pcaPP, type: imports\n",
      "  - edgeR, type: imports\n",
      "  - metagenomeSeq, type: imports\n",
      "  - lpsymphony, type: imports\n",
      "  - pbapply, type: imports\n",
      "  - car, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - vegan, type: imports\n",
      "  - chemometrics, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - pheatmap, type: imports\n",
      "  - logging, type: imports\n",
      "  - data.table, type: imports\n",
      "  - lmerTest, type: imports\n",
      "  - hash, type: imports\n",
      "  - optparse, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - glmmTMB, type: imports\n",
      "  - MASS, type: imports\n",
      "  - cplm, type: imports\n",
      "  - pscl, type: imports\n",
      "  - lme4, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/Maaslin2.html\n",
      "\n",
      "Name: SPIA\n",
      "Description:\n",
      "This package implements the Signaling Pathway Impact Analysis (SPIA) which uses the information form a list of differentially expressed genes and their log fold changes together with signaling pathways topology, in order to identify the pathways most relevant to the condition under the study.\n",
      "Version: 2.50.0\n",
      "Publication date: None\n",
      "Mantainer: Adi Laurentiu Tarca <atarca@med.wayne.edu>\n",
      "Authors:\n",
      "Author: Adi Laurentiu Tarca <atarca at med.wayne.edu>, Purvesh Kathri <purvesh at cs.wayne.edu> and Sorin Draghici <sorin at wayne.edu>\n",
      "Dependencies:\n",
      "  - R (>= 2.14.0) type: depends\n",
      "  - graphics, type: depends\n",
      "  - KEGGgraph, type: depends\n",
      "  - graphics, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/SPIA.html\n",
      "\n",
      "Name: GenomicScores\n",
      "Description:\n",
      "Provide infrastructure to store and access genomewide position-specific scores within R and Bioconductor.\n",
      "Version: 2.10.0\n",
      "Publication date: None\n",
      "Mantainer: Robert Castelo <robert.castelo@upf.edu>\n",
      "Authors:\n",
      "Author: Robert Castelo [aut, cre], Pau Puigdevall [ctb], Pablo RodrÃ­guez [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 3.5) type: depends\n",
      "  - S4Vectors, type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - methods, type: depends\n",
      "  - BiocGenerics, type: depends\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - XML, type: imports\n",
      "  - httr, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - BiocManager, type: imports\n",
      "  - BiocFileCache, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - AnnotationHub, type: imports\n",
      "  - rhdf5, type: imports\n",
      "  - DelayedArray, type: imports\n",
      "  - HDF5Array, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/GenomicScores.html\n",
      "\n",
      "Name: DepecheR\n",
      "Description:\n",
      "The purpose of this package is to identify traits in a dataset that can separate groups. This is done on two levels. First, clustering is performed, using an implementation of sparse K-means. Secondly, the generated clusters are used to predict outcomes of groups of individuals based on their distribution of observations in the different clusters. As certain clusters with separating information will be identified, and these clusters are defined by a sparse number of variables, this method can reduce the complexity of data, to only emphasize the data that actually matters.\n",
      "Version: 1.14.0\n",
      "Publication date: None\n",
      "Mantainer: Jakob Theorell <jakob.theorell@ki.se>\n",
      "Authors:\n",
      "Author: Jakob Theorell [aut, cre], Axel Theorell [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - ggplot2 (>= 3.1.0) type: imports\n",
      "  - MASS (>= 7.3.51) type: imports\n",
      "  - Rcpp (>= 1.0.0) type: imports\n",
      "  - dplyr (>= 0.7.8) type: imports\n",
      "  - gplots (>= 3.0.1) type: imports\n",
      "  - viridis (>= 0.5.1) type: imports\n",
      "  - foreach (>= 1.4.4) type: imports\n",
      "  - doSNOW (>= 1.0.16) type: imports\n",
      "  - matrixStats (>= 0.54.0) type: imports\n",
      "  - mixOmics, type: imports\n",
      "  - moments (>= 0.14) type: imports\n",
      "  - grDevices (>= 3.5.2) type: imports\n",
      "  - graphics (>= 3.5.2) type: imports\n",
      "  - stats (>= 3.5.2) type: imports\n",
      "  - utils (>= 3.5) type: imports\n",
      "  - methods (>= 3.5) type: imports\n",
      "  - parallel (>= 3.5.2) type: imports\n",
      "  - reshape2 (>= 1.4.3) type: imports\n",
      "  - beanplot (>= 1.2) type: imports\n",
      "  - FNN (>= 1.1.3) type: imports\n",
      "  - robustbase (>= 0.93.5) type: imports\n",
      "  - gmodels (>= 2.18.1) type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/DepecheR.html\n",
      "\n",
      "Name: sesame\n",
      "Description:\n",
      "Tools For analyzing Illumina Infinium DNA methylation arrays.SeSAMe provides utilities to support analyses of multiple generations of Infinium DNA methylation BeadChips, including preprocessing, quality control, visualization and inference. SeSAMe features accurate detection calling, intelligent inference of ethnicity, sex and advanced quality control routines.\n",
      "Version: 1.16.1\n",
      "Publication date: None\n",
      "Mantainer: Wanding Zhou <zhouwanding@gmail.com>\n",
      "Authors:\n",
      "Author: Wanding Zhou [aut, cre], Wubin Ding [ctb], David Goldberg [ctb], Ethan Moyer [ctb], Bret Barnes [ctb], Timothy Triche [ctb], Hui Shen [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - sesameData, type: depends\n",
      "  - graphics, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - utils, type: imports\n",
      "  - methods, type: imports\n",
      "  - stringr, type: imports\n",
      "  - readr, type: imports\n",
      "  - tibble, type: imports\n",
      "  - illuminaio, type: imports\n",
      "  - MASS, type: imports\n",
      "  - wheatmap (>= 0.2.0) type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - grid, type: imports\n",
      "  - preprocessCore, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - BiocFileCache, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - stats, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - reshape2, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/sesame.html\n",
      "\n",
      "Name: DEP\n",
      "Description:\n",
      "This package provides an integrated analysis workflow for robust and reproducible analysis of mass spectrometry proteomics data for differential protein expression or differential enrichment. It requires tabular input (e.g. txt files) as generated by quantitative analysis softwares of raw mass spectrometry data, such as MaxQuant or IsobarQuant. Functions are provided for data preparation, filtering, variance normalization and imputation of missing values, as well as statistical testing of differentially enriched / expressed proteins. It also includes tools to check intermediate steps in the workflow, such as normalization and missing values imputation. Finally, visualization tools are provided to explore the results, including heatmap, volcano plot and barplot representations. For scientists with limited experience in R, the package also contains wrapper functions that entail the complete analysis workflow and generate a report. Even easier to use are the interactive Shiny apps that are provided by the package.\n",
      "Version: 1.20.0\n",
      "Publication date: None\n",
      "Mantainer: Arne Smits <smits.arne@gmail.com>\n",
      "Authors:\n",
      "Author: Arne Smits [cre, aut], Wolfgang Huber [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.5) type: depends\n",
      "  - ggplot2, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - purrr, type: imports\n",
      "  - readr, type: imports\n",
      "  - tibble, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - MSnbase, type: imports\n",
      "  - limma, type: imports\n",
      "  - vsn, type: imports\n",
      "  - fdrtool, type: imports\n",
      "  - ggrepel, type: imports\n",
      "  - ComplexHeatmap, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - circlize, type: imports\n",
      "  - shiny, type: imports\n",
      "  - shinydashboard, type: imports\n",
      "  - DT, type: imports\n",
      "  - rmarkdown, type: imports\n",
      "  - assertthat, type: imports\n",
      "  - gridExtra, type: imports\n",
      "  - grid, type: imports\n",
      "  - stats, type: imports\n",
      "  - imputeLCMD, type: imports\n",
      "  - cluster, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/DEP.html\n",
      "\n",
      "Name: Rdisop\n",
      "Description:\n",
      "Identification of metabolites using high precision mass spectrometry. MS Peaks are used to derive a ranked list of sum formulae, alternatively for a given sum formula the theoretical isotope distribution can be calculated to search in MS peak lists.\n",
      "Version: 1.58.0\n",
      "Publication date: None\n",
      "Mantainer: Steffen Neumann <sneumann@ipb-halle.de>\n",
      "Authors:\n",
      "Author: Anton Pervukhin <apervukh at minet.uni-jena.de>, Steffen Neumann <sneumann at ipb-halle.de>\n",
      "Dependencies:\n",
      "  - R (>= 2.0.0) type: depends\n",
      "  - Rcpp, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/Rdisop.html\n",
      "\n",
      "Name: LEA\n",
      "Description:\n",
      "LEA is an R package dedicated to population genomics, landscape genomics and genotype-environment association tests. LEA can run analyses of population structure and genome-wide tests for local adaptation, and also performs imputation of missing genotypes. The package includes statistical methods for estimating ancestry coefficients from large genotypic matrices and for evaluating the number of ancestral populations (snmf, pca). It performs statistical tests using latent factor mixed models for identifying genetic polymorphisms that exhibit association with environmental gradients or phenotypic traits (lfmm2). In addition, LEA computes values of genetic offset based on new or predicted environments. The package includes factor methods for estimating ancestry coefficients from large genotypic matrices and for evaluating the number of ancestral populations (snmf, pca). LEA is mainly based on optimized programs that can scale with the dimensions of large data sets.\n",
      "Version: 3.10.2\n",
      "Publication date: None\n",
      "Mantainer: Olivier Francois <olivier.francois@grenoble-inp.fr>, Eric Frichot <eric.frichot@gmail.com>\n",
      "Authors:\n",
      "Author: Eric Frichot <eric.frichot at gmail.com>, Olivier Francois <olivier.francois at grenoble-inp.fr>, Clement Gain <clement.gain at univ-grenoble-alpes.fr>\n",
      "Dependencies:\n",
      "  - R (>= 3.3.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - stats, type: depends\n",
      "  - utils, type: depends\n",
      "  - graphics, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/LEA.html\n",
      "\n",
      "Name: EnrichedHeatmap\n",
      "Description:\n",
      "Enriched heatmap is a special type of heatmap which visualizes the enrichment of genomic signals on specific target regions. Here we implement enriched heatmap by ComplexHeatmap package. Since this type of heatmap is just a normal heatmap but with some special settings, with the functionality of ComplexHeatmap, it would be much easier to customize the heatmap as well as concatenating to a list of heatmaps to show correspondance between different data sources.\n",
      "Version: 1.27.2\n",
      "Publication date: None\n",
      "Mantainer: Zuguang Gu <z.gu@dkfz.de>\n",
      "Authors:\n",
      "Author: Zuguang Gu [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 3.6.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - grid, type: depends\n",
      "  - ComplexHeatmap, type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - matrixStats, type: imports\n",
      "  - stats, type: imports\n",
      "  - GetoptLong, type: imports\n",
      "  - Rcpp, type: imports\n",
      "  - utils, type: imports\n",
      "  - locfit, type: imports\n",
      "  - circlize (>= 0.4.5) type: imports\n",
      "  - IRanges, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/EnrichedHeatmap.html\n",
      "\n",
      "Name: safe\n",
      "Description:\n",
      "SAFE is a resampling-based method for testing functional categories in gene expression experiments. SAFE can be applied to 2-sample and multi-class comparisons, or simple linear regressions. Other experimental designs can also be accommodated through user-defined functions.\n",
      "Version: 3.38.0\n",
      "Publication date: None\n",
      "Mantainer: Ludwig Geistlinger <ludwig_geistlinger@hms.harvard.edu>\n",
      "Authors:\n",
      "Author: William T. Barry\n",
      "Dependencies:\n",
      "  - R (>= 2.4.0) type: depends\n",
      "  - AnnotationDbi, type: depends\n",
      "  - Biobase, type: depends\n",
      "  - methods, type: depends\n",
      "  - SparseM, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/safe.html\n",
      "\n",
      "Name: SC3\n",
      "Description:\n",
      "A tool for unsupervised clustering and analysis of single cell RNA-Seq data.\n",
      "Version: 1.26.2\n",
      "Publication date: None\n",
      "Mantainer: Vladimir Kiselev <vladimir.yu.kiselev@gmail.com>\n",
      "Authors:\n",
      "Author: Vladimir Kiselev\n",
      "Dependencies:\n",
      "  - R (>= 3.3) type: depends\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - methods, type: imports\n",
      "  - e1071, type: imports\n",
      "  - parallel, type: imports\n",
      "  - foreach, type: imports\n",
      "  - doParallel, type: imports\n",
      "  - doRNG, type: imports\n",
      "  - shiny, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - pheatmap (>= 1.0.8) type: imports\n",
      "  - ROCR, type: imports\n",
      "  - robustbase, type: imports\n",
      "  - rrcov, type: imports\n",
      "  - cluster, type: imports\n",
      "  - WriteXLS, type: imports\n",
      "  - Rcpp (>= 0.11.1) type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - SingleCellExperiment, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/SC3.html\n",
      "\n",
      "Name: universalmotif\n",
      "Description:\n",
      "Allows for importing most common motif types into R for use by functions provided by other Bioconductor motif-related packages. Motifs can be exported into most major motif formats from various classes as defined by other Bioconductor packages. A suite of motif and sequence manipulation and analysis functions are included, including enrichment, comparison, P-value calculation, shuffling, trimming, higher-order motifs, and others.\n",
      "Version: 1.16.0\n",
      "Publication date: None\n",
      "Mantainer: Benjamin Jean-Marie Tremblay <benjamin.tremblay@uwaterloo.ca>\n",
      "Authors:\n",
      "Author: Benjamin Jean-Marie Tremblay [aut, cre] , Spencer Nystrom [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - MASS, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - yaml, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - Rcpp, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - rlang, type: imports\n",
      "  - grid, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/universalmotif.html\n",
      "\n",
      "Name: annotatr\n",
      "Description:\n",
      "Given a set of genomic sites/regions (e.g. ChIP-seq peaks, CpGs, differentially methylated CpGs or regions, SNPs, etc.) it is often of interest to investigate the intersecting genomic annotations. Such annotations include those relating to gene models (promoters, 5'UTRs, exons, introns, and 3'UTRs), CpGs (CpG islands, CpG shores, CpG shelves), or regulatory sequences such as enhancers. The annotatr package provides an easy way to summarize and visualize the intersection of genomic sites/regions with genomic annotations.\n",
      "Version: 1.24.0\n",
      "Publication date: None\n",
      "Mantainer: Raymond G. Cavalcante <rcavalca@umich.edu>\n",
      "Authors:\n",
      "Author: Raymond G. Cavalcante [aut, cre], Maureen A. Sartor [ths]\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - AnnotationDbi, type: imports\n",
      "  - AnnotationHub, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - GenomicFeatures, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - methods, type: imports\n",
      "  - readr, type: imports\n",
      "  - regioneR, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/annotatr.html\n",
      "\n",
      "Name: GenVisR\n",
      "Description:\n",
      "Produce highly customizable publication quality graphics for genomic data primarily at the cohort level.\n",
      "Version: 1.30.0\n",
      "Publication date: None\n",
      "Mantainer: Zachary Skidmore <zlskidmore@gmail.com>\n",
      "Authors:\n",
      "Author: Zachary Skidmore [aut, cre], Alex Wagner [aut], Robert Lesurf [aut], Katie Campbell [aut], Jason Kunisaki [aut], Obi Griffith [aut], Malachi Griffith [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.3.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - AnnotationDbi, type: imports\n",
      "  - biomaRt, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - DBI, type: imports\n",
      "  - GenomicFeatures, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - ggplot2 (>= 2.1.0) type: imports\n",
      "  - gridExtra (>= 2.0.0) type: imports\n",
      "  - gtable, type: imports\n",
      "  - gtools, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - plyr (>= 1.8.3) type: imports\n",
      "  - reshape2, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - scales, type: imports\n",
      "  - viridis, type: imports\n",
      "  - data.table, type: imports\n",
      "  - BSgenome, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - VariantAnnotation, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3 + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/GenVisR.html\n",
      "\n",
      "Name: Rbowtie\n",
      "Description:\n",
      "This package provides an R wrapper around the popular bowtie short read aligner and around SpliceMap, a de novo splice junction discovery and alignment tool. The package is used by the QuasR bioconductor package. We recommend to use the QuasR package instead of using Rbowtie directly.\n",
      "Version: 1.38.0\n",
      "Publication date: None\n",
      "Mantainer: Michael Stadler <michael.stadler@fmi.ch>\n",
      "Authors:\n",
      "Author: Florian Hahne, Anita Lerch, Michael B Stadler\n",
      "Dependencies:\n",
      "None\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0 | file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/Rbowtie.html\n",
      "\n",
      "Name: gwascat\n",
      "Description:\n",
      "Represent and model data in the EMBL-EBI GWAS catalog.\n",
      "Version: 2.30.0\n",
      "Publication date: None\n",
      "Mantainer: VJ Carey <stvjc@channing.harvard.edu>\n",
      "Authors:\n",
      "Author: VJ Carey <stvjc at channing.harvard.edu>\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - S4Vectors, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - GenomicFeatures, type: imports\n",
      "  - readr, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - AnnotationDbi, type: imports\n",
      "  - BiocFileCache, type: imports\n",
      "  - snpStats, type: imports\n",
      "  - VariantAnnotation, type: imports\n",
      "  - AnnotationHub, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/gwascat.html\n",
      "\n",
      "Name: M3Drop\n",
      "Description:\n",
      "This package fits a Michaelis-Menten model to the pattern of dropouts in single-cell RNASeq data. This model is used as a null to identify significantly variable (i.e. differentially expressed) genes for use in downstream analysis, such as clustering cells.\n",
      "Version: 1.24.0\n",
      "Publication date: None\n",
      "Mantainer: Tallulah Andrews <tallulandrews@gmail.com>\n",
      "Authors:\n",
      "Author: Tallulah Andrews <tallulandrews at gmail.com>\n",
      "Dependencies:\n",
      "  - R (>= 3.4) type: depends\n",
      "  - numDeriv, type: depends\n",
      "  - RColorBrewer, type: imports\n",
      "  - gplots, type: imports\n",
      "  - bbmle, type: imports\n",
      "  - statmod, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - irlba, type: imports\n",
      "  - reldist, type: imports\n",
      "  - Hmisc, type: imports\n",
      "  - methods, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>=2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/M3Drop.html\n",
      "\n",
      "Name: scmap\n",
      "Description:\n",
      "Single-cell RNA-seq (scRNA-seq) is widely used to investigate the composition of complex tissues since the technology allows researchers to define cell-types using unsupervised clustering of the transcriptome. However, due to differences in experimental methods and computational analyses, it is often challenging to directly compare the cells identified in two different experiments. scmap is a method for projecting cells from a scRNA-seq experiment on to the cell-types or individual cells identified in a different experiment.\n",
      "Version: 1.20.2\n",
      "Publication date: None\n",
      "Mantainer: Vladimir Kiselev <vladimir.yu.kiselev@gmail.com>\n",
      "Authors:\n",
      "Author: Vladimir Kiselev\n",
      "Dependencies:\n",
      "  - R (>= 3.4) type: depends\n",
      "  - Biobase, type: imports\n",
      "  - SingleCellExperiment, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - proxy, type: imports\n",
      "  - utils, type: imports\n",
      "  - googleVis, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - e1071, type: imports\n",
      "  - randomForest, type: imports\n",
      "  - Rcpp (>= 0.12.12) type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/scmap.html\n",
      "\n",
      "Name: CATALYST\n",
      "Description:\n",
      "Mass cytometry (CyTOF) uses heavy metal isotopes rather than fluorescent tags as reporters to label antibodies, thereby substantially decreasing spectral overlap and allowing for examination of over 50 parameters at the single cell level. While spectral overlap is significantly less pronounced in CyTOF than flow cytometry, spillover due to detection sensitivity, isotopic impurities, and oxide formation can impede data interpretability. We designed CATALYST (Cytometry dATa anALYSis Tools) to provide a pipeline for preprocessing of cytometry data, including i) normalization using bead standards, ii) single-cell deconvolution, and iii) bead-based compensation.\n",
      "Version: 1.22.0\n",
      "Publication date: None\n",
      "Mantainer: Helena L. Crowell <helena.crowell@uzh.ch>\n",
      "Authors:\n",
      "Author: Helena L. Crowell [aut, cre], Vito R.T. Zanotelli [aut], StÃ©phane Chevrier [aut, dtc], Mark D. Robinson [aut, fnd], Bernd Bodenmiller [fnd]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - SingleCellExperiment, type: depends\n",
      "  - circlize, type: imports\n",
      "  - ComplexHeatmap, type: imports\n",
      "  - ConsensusClusterPlus, type: imports\n",
      "  - cowplot, type: imports\n",
      "  - data.table, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - drc, type: imports\n",
      "  - flowCore, type: imports\n",
      "  - FlowSOM, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - ggrepel, type: imports\n",
      "  - ggridges, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - grid, type: imports\n",
      "  - gridExtra, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - methods, type: imports\n",
      "  - nnls, type: imports\n",
      "  - purrr, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - Rtsne, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - scales, type: imports\n",
      "  - scater, type: imports\n",
      "  - stats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>=2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/CATALYST.html\n",
      "\n",
      "Name: pcaExplorer\n",
      "Description:\n",
      "This package provides functionality for interactive visualization of RNA-seq datasets based on Principal Components Analysis. The methods provided allow for quick information extraction and effective data exploration. A Shiny application encapsulates the whole analysis.\n",
      "Version: 2.24.0\n",
      "Publication date: None\n",
      "Mantainer: Federico Marini <marinif@uni-mainz.de>\n",
      "Authors:\n",
      "Author: Federico Marini [aut, cre]\n",
      "Dependencies:\n",
      "  - DESeq2, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - genefilter, type: imports\n",
      "  - ggplot2 (>= 2.0.0) type: imports\n",
      "  - heatmaply, type: imports\n",
      "  - plotly, type: imports\n",
      "  - scales, type: imports\n",
      "  - NMF, type: imports\n",
      "  - plyr, type: imports\n",
      "  - topGO, type: imports\n",
      "  - limma, type: imports\n",
      "  - GOstats, type: imports\n",
      "  - GO.db, type: imports\n",
      "  - AnnotationDbi, type: imports\n",
      "  - shiny (>= 0.12.0) type: imports\n",
      "  - shinydashboard, type: imports\n",
      "  - shinyBS, type: imports\n",
      "  - ggrepel, type: imports\n",
      "  - DT, type: imports\n",
      "  - shinyAce, type: imports\n",
      "  - threejs, type: imports\n",
      "  - biomaRt, type: imports\n",
      "  - pheatmap, type: imports\n",
      "  - knitr, type: imports\n",
      "  - rmarkdown, type: imports\n",
      "  - base64enc, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - methods, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/pcaExplorer.html\n",
      "\n",
      "Name: copynumber\n",
      "Description:\n",
      "Penalized least squares regression is applied to fit piecewise constant curves to copy number data to locate genomic regions of constant copy number. Procedures are available for individual segmentation of each sample, joint segmentation of several samples and joint segmentation of the two data tracks from SNP-arrays. Several plotting functions are available for visualization of the data and the segmentation results.\n",
      "Version: 1.38.0\n",
      "Publication date: None\n",
      "Mantainer: Gro Nilsen <gronilse@ifi.uio.no>\n",
      "Authors:\n",
      "Author: Gro Nilsen, Knut Liestoel and Ole Christian Lingjaerde.\n",
      "Dependencies:\n",
      "  - R (>= 2.10) type: depends\n",
      "  - BiocGenerics, type: depends\n",
      "  - S4Vectors, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/copynumber.html\n",
      "\n",
      "Name: MutationalPatterns\n",
      "Description:\n",
      "Mutational processes leave characteristic footprints in genomic DNA. This package provides a comprehensive set of flexible functions that allows researchers to easily evaluate and visualize a multitude of mutational patterns in base substitution catalogues of e.g. healthy samples, tumour samples, or DNA-repair deficient cells. The package covers a wide range of patterns including: mutational signatures, transcriptional and replicative strand bias, lesion segregation, genomic distribution and association with genomic features, which are collectively meaningful for studying the activity of mutational processes. The package works with single nucleotide variants (SNVs), insertions and deletions (Indels), double base substitutions (DBSs) and larger multi base substitutions (MBSs). The package provides functionalities for both extracting mutational signatures de novo and determining the contribution of previously identified mutational signatures on a single sample level. MutationalPatterns integrates with common R genomic analysis workflows and allows easy association with (publicly available) annotation data.\n",
      "Version: 3.8.1\n",
      "Publication date: None\n",
      "Mantainer: Mark van Roosmalen <vanBoxtelBioinformatics@prinsesmaximacentrum.nl>\n",
      "Authors:\n",
      "Author: Freek Manders [aut] , Francis Blokzijl [aut] , Roel Janssen [aut] , Jurrian de Kanter [ctb] , Rurika Oka [ctb] , Mark van Roosmalen [cre], Ruben van Boxtel [aut, cph] , Edwin Cuppen [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.2.0) type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - NMF (>= 0.20.6) type: depends\n",
      "  - stats, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - BSgenome, type: imports\n",
      "  - VariantAnnotation, type: imports\n",
      "  - dplyr (>= 0.8.3) type: imports\n",
      "  - tibble (>= 2.1.3) type: imports\n",
      "  - purrr (>= 0.3.2) type: imports\n",
      "  - tidyr (>= 1.0.0) type: imports\n",
      "  - stringr (>= 1.4.0) type: imports\n",
      "  - magrittr (>= 1.5) type: imports\n",
      "  - ggplot2 (>= 2.1.0) type: imports\n",
      "  - pracma (>= 1.8.8) type: imports\n",
      "  - IRanges, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - ggdendro (>= 0.1-20) type: imports\n",
      "  - cowplot (>= 0.9.2) type: imports\n",
      "  - ggalluvial (>= 0.12.2) type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - methods, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/MutationalPatterns.html\n",
      "\n",
      "Name: rGADEM\n",
      "Description:\n",
      "rGADEM is an efficient de novo motif discovery tool for large-scale genomic sequence data. It is an open-source R package, which is based on the GADEM software.\n",
      "Version: 2.46.0\n",
      "Publication date: None\n",
      "Mantainer: Arnaud Droit <arnaud.droit@crchuq.ulaval.ca>\n",
      "Authors:\n",
      "Author: Arnaud Droit, Raphael Gottardo, Gordon Robertson and Leiping Li\n",
      "Dependencies:\n",
      "  - R (>= 2.11.0) type: depends\n",
      "  - Biostrings, type: depends\n",
      "  - IRanges, type: depends\n",
      "  - BSgenome, type: depends\n",
      "  - methods, type: depends\n",
      "  - seqLogo, type: depends\n",
      "  - Biostrings, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - methods, type: imports\n",
      "  - graphics, type: imports\n",
      "  - seqLogo, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/rGADEM.html\n",
      "\n",
      "Name: MSstats\n",
      "Description:\n",
      "A set of tools for statistical relative protein significance analysis in DDA, SRM and DIA experiments.\n",
      "Version: 4.6.3\n",
      "Publication date: None\n",
      "Mantainer: Meena Choi <mnchoi67@gmail.com>\n",
      "Authors:\n",
      "Author: Meena Choi [aut, cre], Mateusz Staniak [aut], Tsung-Heng Tsai [aut], Ting Huang [aut], Olga Vitek [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - MSstatsConvert, type: imports\n",
      "  - data.table, type: imports\n",
      "  - checkmate, type: imports\n",
      "  - MASS, type: imports\n",
      "  - limma, type: imports\n",
      "  - lme4, type: imports\n",
      "  - preprocessCore, type: imports\n",
      "  - survival, type: imports\n",
      "  - utils, type: imports\n",
      "  - Rcpp, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - ggrepel, type: imports\n",
      "  - gplots, type: imports\n",
      "  - marray, type: imports\n",
      "  - stats, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - methods, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/MSstats.html\n",
      "\n",
      "Name: mbkmeans\n",
      "Description:\n",
      "Implements the mini-batch k-means algorithm for large datasets, including support for on-disk data representation.\n",
      "Version: 1.14.0\n",
      "Publication date: None\n",
      "Mantainer: Davide Risso <risso.davide@gmail.com>\n",
      "Authors:\n",
      "Author: Yuwei Ni [aut, cph], Davide Risso [aut, cre, cph], Stephanie Hicks [aut, cph], Elizabeth Purdom [aut, cph]\n",
      "Dependencies:\n",
      "  - R (>= 3.6) type: depends\n",
      "  - methods, type: imports\n",
      "  - DelayedArray, type: imports\n",
      "  - Rcpp, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - SingleCellExperiment, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - ClusterR, type: imports\n",
      "  - benchmarkme, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/mbkmeans.html\n",
      "\n",
      "Name: UCell\n",
      "Description:\n",
      "UCell is a package for evaluating gene signatures in single-cell datasets. UCell signature scores, based on the Mann-Whitney U statistic, are robust to dataset size and heterogeneity, and their calculation demands less computing time and memory than other available methods, enabling the processing of large datasets in a few minutes even on machines with limited computing power. UCell can be applied to any single-cell data matrix, and includes functions to directly interact with SingleCellExperiment and Seurat objects.\n",
      "Version: 2.2.0\n",
      "Publication date: None\n",
      "Mantainer: Massimo Andreatta <massimo.andreatta@unil.ch>\n",
      "Authors:\n",
      "Author: Massimo Andreatta [aut, cre] , Santiago Carmona [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.2.0) type: depends\n",
      "  - methods, type: imports\n",
      "  - data.table (>= 1.13.6) type: imports\n",
      "  - Matrix, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - BiocNeighbors, type: imports\n",
      "  - SingleCellExperiment, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3 + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/UCell.html\n",
      "\n",
      "Name: affycoretools\n",
      "Description:\n",
      "Various wrapper functions that have been written to streamline the more common analyses that a core Biostatistician might see.\n",
      "Version: 1.70.0\n",
      "Publication date: None\n",
      "Mantainer: James W. MacDonald <jmacdon@u.washington.edu>\n",
      "Authors:\n",
      "Author: James W. MacDonald\n",
      "Dependencies:\n",
      "  - Biobase, type: depends\n",
      "  - methods, type: depends\n",
      "  - affy, type: imports\n",
      "  - limma, type: imports\n",
      "  - GOstats, type: imports\n",
      "  - gcrma, type: imports\n",
      "  - splines, type: imports\n",
      "  - xtable, type: imports\n",
      "  - AnnotationDbi, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - gplots, type: imports\n",
      "  - oligoClasses, type: imports\n",
      "  - ReportingTools, type: imports\n",
      "  - hwriter, type: imports\n",
      "  - lattice, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - edgeR, type: imports\n",
      "  - RSQLite, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - DBI, type: imports\n",
      "  - Glimma, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/affycoretools.html\n",
      "\n",
      "Name: Organism.dplyr\n",
      "Description:\n",
      "This package provides an alternative interface to Bioconductor 'annotation' resources, in particular the gene identifier mapping functionality of the 'org' packages (e.g., org.Hs.eg.db) and the genome coordinate functionality of the 'TxDb' packages (e.g., TxDb.Hsapiens.UCSC.hg38.knownGene).\n",
      "Version: 1.26.0\n",
      "Publication date: None\n",
      "Mantainer: Martin Morgan <martin.morgan@roswellpark.org>\n",
      "Authors:\n",
      "Author: Martin Morgan [aut, cre], Daniel van Twisk [ctb], Yubo Cheng [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.4) type: depends\n",
      "  - dplyr (>= 0.7.0) type: depends\n",
      "  - AnnotationFilter, type: depends\n",
      "  - RSQLite, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - GenomicFeatures, type: imports\n",
      "  - AnnotationDbi, type: imports\n",
      "  - rlang, type: imports\n",
      "  - methods, type: imports\n",
      "  - tools, type: imports\n",
      "  - utils, type: imports\n",
      "  - BiocFileCache, type: imports\n",
      "  - DBI, type: imports\n",
      "  - dbplyr, type: imports\n",
      "  - tibble, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/Organism.dplyr.html\n",
      "\n",
      "Name: Spectra\n",
      "Description:\n",
      "The Spectra package defines an efficient infrastructure for storing and handling mass spectrometry spectra and functionality to subset, process, visualize and compare spectra data. It provides different implementations (backends) to store mass spectrometry data. These comprise backends tuned for fast data access and processing and backends for very large data sets ensuring a small memory footprint.\n",
      "Version: 1.8.1\n",
      "Publication date: None\n",
      "Mantainer: RforMassSpectrometry Package Maintainer <maintainer@rformassspectrometry.org>\n",
      "Authors:\n",
      "Author: RforMassSpectrometry Package Maintainer [cre], Laurent Gatto [aut] , Johannes Rainer [aut] , Sebastian Gibb [aut] , Jan Stanstrup [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 4.0.0) type: depends\n",
      "  - S4Vectors, type: depends\n",
      "  - BiocParallel, type: depends\n",
      "  - ProtGenerics, type: depends\n",
      "  - methods, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - MsCoreUtils, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - stats, type: imports\n",
      "  - tools, type: imports\n",
      "  - utils, type: imports\n",
      "  - fs, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/Spectra.html\n",
      "\n",
      "Name: splatter\n",
      "Description:\n",
      "Splatter is a package for the simulation of single-cell RNA sequencing count data. It provides a simple interface for creating complex simulations that are reproducible and well-documented. Parameters can be estimated from real data and functions are provided for comparing real and simulated datasets.\n",
      "Version: 1.22.1\n",
      "Publication date: None\n",
      "Mantainer: Luke Zappia <luke@lazappi.id.au>\n",
      "Authors:\n",
      "Author: Luke Zappia [aut, cre] , Belinda Phipson [aut] , Christina Azodi [ctb] , Alicia Oshlack [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - SingleCellExperiment, type: depends\n",
      "  - BiocGenerics, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - checkmate (>= 2.0.0) type: imports\n",
      "  - edgeR, type: imports\n",
      "  - fitdistrplus, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - locfit, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - methods, type: imports\n",
      "  - scales, type: imports\n",
      "  - scater, type: imports\n",
      "  - stats, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - utils, type: imports\n",
      "  - crayon, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - grDevices, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3 + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/splatter.html\n",
      "\n",
      "Name: recount\n",
      "Description:\n",
      "Explore and download data from the recount project available at https://jhubiostatistics.shinyapps.io/recount/. Using the recount package you can download RangedSummarizedExperiment objects at the gene, exon or exon-exon junctions level, the raw counts, the phenotype metadata used, the urls to the sample coverage bigWig files or the mean coverage bigWig file for a particular study. The RangedSummarizedExperiment objects can be used by different packages for performing differential expression analysis. Using http://bioconductor.org/packages/derfinder you can perform annotation-agnostic differential expression analyses with the data from the recount project as described at http://www.nature.com/nbt/journal/v35/n4/full/nbt.3838.html.\n",
      "Version: 1.24.1\n",
      "Publication date: None\n",
      "Mantainer: Leonardo Collado-Torres <lcolladotor@gmail.com>\n",
      "Authors:\n",
      "Author: Leonardo Collado-Torres [aut, cre] , Abhinav Nellore [ctb], Andrew E. Jaffe [ctb] , Margaret A. Taub [ctb], Kai Kammers [ctb], Shannon E. Ellis [ctb] , Kasper Daniel Hansen [ctb] , Ben Langmead [ctb] , Jeffrey T. Leek [aut, ths]\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - BiocParallel, type: imports\n",
      "  - derfinder, type: imports\n",
      "  - downloader, type: imports\n",
      "  - GEOquery, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - methods, type: imports\n",
      "  - RCurl, type: imports\n",
      "  - rentrez, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/recount.html\n",
      "\n",
      "Name: qusage\n",
      "Description:\n",
      "This package is an implementation the Quantitative Set Analysis for Gene Expression (QuSAGE) method described in (Yaari G. et al, Nucl Acids Res, 2013). This is a novel Gene Set Enrichment-type test, which is designed to provide a faster, more accurate, and easier to understand test for gene expression studies. qusage accounts for inter-gene correlations using the Variance Inflation Factor technique proposed by Wu et al. (Nucleic Acids Res, 2012). In addition, rather than simply evaluating the deviation from a null hypothesis with a single number (a P value), qusage quantifies gene set activity with a complete probability density function (PDF). From this PDF, P values and confidence intervals can be easily extracted. Preserving the PDF also allows for post-hoc analysis (e.g., pair-wise comparisons of gene set activity) while maintaining statistical traceability. Finally, while qusage is compatible with individual gene statistics from existing methods (e.g., LIMMA), a Welch-based method is implemented that is shown to improve specificity. The QuSAGE package also includes a mixed effects model implementation, as described in (Turner JA et al, BMC Bioinformatics, 2015), and a meta-analysis framework as described in (Meng H, et al. PLoS Comput Biol. 2019). For questions, contact Chris Bolen (cbolen1@gmail.com) or Steven Kleinstein (steven.kleinstein@yale.edu)\n",
      "Version: 2.32.0\n",
      "Publication date: None\n",
      "Mantainer: Christopher Bolen <cbolen1@gmail.com>\n",
      "Authors:\n",
      "Author: Christopher Bolen and Gur Yaari, with contributions from Juilee Thakar, Hailong Meng, Jacob Turner, Derek Blankenship, and Steven Kleinstein\n",
      "Dependencies:\n",
      "  - R (>= 2.10) type: depends\n",
      "  - limma, type: depends\n",
      "  - methods, type: depends\n",
      "  - utils, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - nlme, type: imports\n",
      "  - emmeans, type: imports\n",
      "  - fftw, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/qusage.html\n",
      "\n",
      "Name: DRIMSeq\n",
      "Description:\n",
      "The package provides two frameworks. One for the differential transcript usage analysis between different conditions and one for the tuQTL analysis. Both are based on modeling the counts of genomic features (i.e., transcripts) with the Dirichlet-multinomial distribution. The package also makes available functions for visualization and exploration of the data and results.\n",
      "Version: 1.26.0\n",
      "Publication date: None\n",
      "Mantainer: Malgorzata Nowicka <gosia.nowicka.uzh@gmail.com>\n",
      "Authors:\n",
      "Author: Malgorzata Nowicka [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 3.4.0) type: depends\n",
      "  - utils, type: imports\n",
      "  - stats, type: imports\n",
      "  - MASS, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - methods, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - limma, type: imports\n",
      "  - edgeR, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - reshape2, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 3)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/DRIMSeq.html\n",
      "\n",
      "Name: Heatplus\n",
      "Description:\n",
      "Display a rectangular heatmap (intensity plot) of a data matrix. By default, both samples (columns) and features (row) of the matrix are sorted according to a hierarchical clustering, and the corresponding dendrogram is plotted. Optionally, panels with additional information about samples and features can be added to the plot.\n",
      "Version: 3.6.0\n",
      "Publication date: None\n",
      "Mantainer: Alexander Ploner <Alexander.Ploner@ki.se>\n",
      "Authors:\n",
      "Author: Alexander Ploner <Alexander.Ploner at ki.se>\n",
      "Dependencies:\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - stats, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/Heatplus.html\n",
      "\n",
      "Name: EBSeq\n",
      "Description:\n",
      "Differential Expression analysis at both gene and isoform level using RNA-seq data\n",
      "Version: 1.38.0\n",
      "Publication date: None\n",
      "Mantainer: Ning Leng <lengning1@gmail.com>\n",
      "Authors:\n",
      "Author: Ning Leng, Christina Kendziorski\n",
      "Dependencies:\n",
      "  - blockmodeling, type: depends\n",
      "  - gplots, type: depends\n",
      "  - testthat, type: depends\n",
      "  - R (>= 3.0.0) type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/EBSeq.html\n",
      "\n",
      "Name: SRAdb\n",
      "Description:\n",
      "The Sequence Read Archive (SRA) is the largest public repository of sequencing data from the next generation of sequencing platforms including Roche 454 GS System, Illumina Genome Analyzer, Applied Biosystems SOLiD System, Helicos Heliscope, and others. However, finding data of interest can be challenging using current tools. SRAdb is an attempt to make access to the metadata associated with submission, study, sample, experiment and run much more feasible. This is accomplished by parsing all the NCBI SRA metadata into a SQLite database that can be stored and queried locally. Fulltext search in the package make querying metadata very flexible and powerful. fastq and sra files can be downloaded for doing alignment locally. Beside ftp protocol, the SRAdb has funcitons supporting fastp protocol (ascp from Aspera Connect) for faster downloading large data files over long distance. The SQLite database is updated regularly as new data is added to SRA and can be downloaded at will for the most up-to-date metadata.\n",
      "Version: 1.60.0\n",
      "Publication date: None\n",
      "Mantainer: Jack Zhu <zhujack@mail.nih.gov>\n",
      "Authors:\n",
      "Author: Jack Zhu and Sean Davis\n",
      "Dependencies:\n",
      "  - RSQLite, type: depends\n",
      "  - graph, type: depends\n",
      "  - RCurl, type: depends\n",
      "  - GEOquery, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/SRAdb.html\n",
      "\n",
      "Name: AIMS\n",
      "Description:\n",
      "This package contains the AIMS implementation. It contains necessary functions to assign the five intrinsic molecular subtypes (Luminal A, Luminal B, Her2-enriched, Basal-like, Normal-like). Assignments could be done on individual samples as well as on dataset of gene expression data.\n",
      "Version: 1.30.0\n",
      "Publication date: None\n",
      "Mantainer: Eric R Paquet <eric.r.paquet@gmail.com>\n",
      "Authors:\n",
      "Author: Eric R. Paquet, Michael T. Hallett\n",
      "Dependencies:\n",
      "  - R (>= 2.10) type: depends\n",
      "  - e1071, type: depends\n",
      "  - Biobase, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/AIMS.html\n",
      "\n",
      "Name: fishpond\n",
      "Description:\n",
      "Fishpond contains methods for differential transcript and gene expression analysis of RNA-seq data using inferential replicates for uncertainty of abundance quantification, as generated by Gibbs sampling or bootstrap sampling. Also the package contains a number of utilities for working with Salmon and Alevin quantification files.\n",
      "Version: 2.4.1\n",
      "Publication date: None\n",
      "Mantainer: Michael Love <michaelisaiahlove@gmail.com>\n",
      "Authors:\n",
      "Author: Anqi Zhu [aut, ctb], Michael Love [aut, cre], Avi Srivastava [aut, ctb], Rob Patro [aut, ctb], Joseph Ibrahim [aut, ctb], Hirak Sarkar [ctb], Euphy Wu [ctb], Noor Pratap Singh [ctb], Scott Van Buren [ctb], Dongze He [ctb], Steve Lianoglou [ctb], Wes Wilson [ctb], Jeroen Gilis [ctb]\n",
      "Dependencies:\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - methods, type: imports\n",
      "  - abind, type: imports\n",
      "  - gtools, type: imports\n",
      "  - qvalue, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - svMisc, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - SingleCellExperiment, type: imports\n",
      "  - jsonlite, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/fishpond.html\n",
      "\n",
      "Name: tradeSeq\n",
      "Description:\n",
      "tradeSeq provides a flexible method for fitting regression models that can be used to find genes that are differentially expressed along one or multiple lineages in a trajectory. Based on the fitted models, it uses a variety of tests suited to answer different questions of interest, e.g. the discovery of genes for which expression is associated with pseudotime, or which are differentially expressed (in a specific region) along the trajectory. It fits a negative binomial generalized additive model (GAM) for each gene, and performs inference on the parameters of the GAM.\n",
      "Version: 1.12.0\n",
      "Publication date: None\n",
      "Mantainer: Hector Roux de Bezieux <hector.rouxdebezieux@berkeley.edu>\n",
      "Authors:\n",
      "Author: Koen Van den Berge [aut], Hector Roux de Bezieux [aut, cre] , Kelly Street [aut, ctb], Lieven Clement [aut, ctb], Sandrine Dudoit [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 3.6) type: depends\n",
      "  - mgcv, type: imports\n",
      "  - edgeR, type: imports\n",
      "  - SingleCellExperiment, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - slingshot, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - pbapply, type: imports\n",
      "  - igraph, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - princurve, type: imports\n",
      "  - methods, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - tibble, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - TrajectoryUtils, type: imports\n",
      "  - viridis, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - MASS, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/tradeSeq.html\n",
      "\n",
      "Name: ChIPQC\n",
      "Description:\n",
      "Quality metrics for ChIPseq data.\n",
      "Version: 1.34.1\n",
      "Publication date: None\n",
      "Mantainer: Tom Carroll <tc.infomatics@gmail.com>, Rory Stark <rory.stark@cruk.cam.ac.uk>\n",
      "Authors:\n",
      "Author: Tom Carroll, Wei Liu, Ines de Santiago, Rory Stark\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - ggplot2, type: depends\n",
      "  - DiffBind, type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - BiocParallel, type: depends\n",
      "  - BiocGenerics, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - GenomicAlignments, type: imports\n",
      "  - chipseq, type: imports\n",
      "  - gtools, type: imports\n",
      "  - methods, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - Nozzle.R1, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - GenomicFeatures, type: imports\n",
      "  - TxDb.Hsapiens.UCSC.hg19.knownGene, type: imports\n",
      "  - TxDb.Hsapiens.UCSC.hg18.knownGene, type: imports\n",
      "  - TxDb.Mmusculus.UCSC.mm10.knownGene, type: imports\n",
      "  - TxDb.Mmusculus.UCSC.mm9.knownGene, type: imports\n",
      "  - TxDb.Rnorvegicus.UCSC.rn4.ensGene, type: imports\n",
      "  - TxDb.Celegans.UCSC.ce6.ensGene, type: imports\n",
      "  - TxDb.Dmelanogaster.UCSC.dm3.ensGene, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 3)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ChIPQC.html\n",
      "\n",
      "Name: genefu\n",
      "Description:\n",
      "This package contains functions implementing various tasks usually required by gene expression analysis, especially in breast cancer studies: gene mapping between different microarray platforms, identification of molecular subtypes, implementation of published gene signatures, gene selection, and survival analysis.\n",
      "Version: 2.30.0\n",
      "Publication date: None\n",
      "Mantainer: Benjamin Haibe-Kains <benjamin.haibe.kains@utoronto.ca>\n",
      "Authors:\n",
      "Author: Deena M.A. Gendoo [aut], Natchar Ratanasirigulchai [aut], Markus S. Schroeder [aut], Laia Pare [aut], Joel S Parker [aut], Aleix Prat [aut], Nikta Feizi [ctb], Christopher Eeles [ctb], Benjamin Haibe-Kains [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - survcomp, type: depends\n",
      "  - biomaRt, type: depends\n",
      "  - iC10, type: depends\n",
      "  - AIMS, type: depends\n",
      "  - amap, type: imports\n",
      "  - impute, type: imports\n",
      "  - mclust, type: imports\n",
      "  - limma, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/genefu.html\n",
      "\n",
      "Name: lfa\n",
      "Description:\n",
      "LFA is a method for a PCA analogue on Binomial data via estimation of latent structure in the natural parameter.\n",
      "Version: 1.28.2\n",
      "Publication date: None\n",
      "Mantainer: Wei Hao <whao@princeton.edu>, John D. Storey <jstorey@princeton.edu>\n",
      "Authors:\n",
      "Author: Wei Hao, Minsun Song, John D. Storey\n",
      "Dependencies:\n",
      "  - R (>= 3.2) type: depends\n",
      "  - corpcor, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/lfa.html\n",
      "\n",
      "Name: sangerseqR\n",
      "Description:\n",
      "This package contains several tools for analyzing Sanger Sequencing data files in R, including reading .scf and .ab1 files, making basecalls and plotting chromatograms.\n",
      "Version: 1.34.0\n",
      "Publication date: None\n",
      "Mantainer: Jonathon Hill <jhill@byu.edu>\n",
      "Authors:\n",
      "Author: Jonathon T. Hill, Bradley Demarest\n",
      "Dependencies:\n",
      "  - R (>= 3.0.2) type: depends\n",
      "  - Biostrings, type: depends\n",
      "  - methods, type: imports\n",
      "  - shiny, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/sangerseqR.html\n",
      "\n",
      "Name: piano\n",
      "Description:\n",
      "Piano performs gene set analysis using various statistical methods, from different gene level statistics and a wide range of gene-set collections. Furthermore, the Piano package contains functions for combining the results of multiple runs of gene set analyses.\n",
      "Version: 2.14.0\n",
      "Publication date: None\n",
      "Mantainer: Leif Varemo Wigge <piano.rpkg@gmail.com>\n",
      "Authors:\n",
      "Author: Leif Varemo Wigge <piano.rpkg at gmail.com> and Intawat Nookaew <piano.rpkg at gmail.com>\n",
      "Dependencies:\n",
      "  - R (>= 3.5) type: depends\n",
      "  - BiocGenerics, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - gplots, type: imports\n",
      "  - igraph, type: imports\n",
      "  - relations, type: imports\n",
      "  - marray, type: imports\n",
      "  - fgsea, type: imports\n",
      "  - shiny, type: imports\n",
      "  - DT, type: imports\n",
      "  - htmlwidgets, type: imports\n",
      "  - shinyjs, type: imports\n",
      "  - shinydashboard, type: imports\n",
      "  - visNetwork, type: imports\n",
      "  - scales, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - methods, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>=2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/piano.html\n",
      "\n",
      "Name: TSCAN\n",
      "Description:\n",
      "Provides methods to perform trajectory analysis based on a minimum spanning tree constructed from cluster centroids. Computes pseudotemporal cell orderings by mapping cells in each cluster (or new cells) to the closest edge in the tree. Uses linear modelling to identify differentially expressed genes along each path through the tree. Several plotting and interactive visualization functions are also implemented.\n",
      "Version: 1.36.0\n",
      "Publication date: None\n",
      "Mantainer: Zhicheng Ji <zji4@jhu.edu>\n",
      "Authors:\n",
      "Author: Zhicheng Ji [aut, cre], Hongkai Ji [aut], Aaron Lun [ctb]\n",
      "Dependencies:\n",
      "  - SingleCellExperiment, type: depends\n",
      "  - TrajectoryUtils, type: depends\n",
      "  - ggplot2, type: imports\n",
      "  - shiny, type: imports\n",
      "  - plyr, type: imports\n",
      "  - grid, type: imports\n",
      "  - fastICA, type: imports\n",
      "  - igraph, type: imports\n",
      "  - combinat, type: imports\n",
      "  - mgcv, type: imports\n",
      "  - mclust, type: imports\n",
      "  - gplots, type: imports\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - DelayedArray, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL(>=2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/TSCAN.html\n",
      "\n",
      "Name: mygene\n",
      "Description:\n",
      "MyGene.Info_ provides simple-to-use REST web services to query/retrieve gene annotation data. It's designed with simplicity and performance emphasized. *mygene*, is an easy-to-use R wrapper to access MyGene.Info_ services.\n",
      "Version: 1.34.0\n",
      "Publication date: None\n",
      "Mantainer: Adam Mark, Cyrus Afrasiabi, Chunlei Wu <cwu@scripps.edu>\n",
      "Authors:\n",
      "Author: Adam Mark, Ryan Thompson, Cyrus Afrasiabi, Chunlei Wu\n",
      "Dependencies:\n",
      "  - R (>= 3.2.1) type: depends\n",
      "  - GenomicFeatures, type: depends\n",
      "  - httr (>= 0.3) type: imports\n",
      "  - jsonlite (>= 0.9.7) type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - Hmisc, type: imports\n",
      "  - sqldf, type: imports\n",
      "  - plyr, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/mygene.html\n",
      "\n",
      "Name: csaw\n",
      "Description:\n",
      "Detection of differentially bound regions in ChIP-seq data with sliding windows, with methods for normalization and proper FDR control.\n",
      "Version: 1.32.0\n",
      "Publication date: None\n",
      "Mantainer: Aaron Lun <infinite.monkeys.with.keyboards@gmail.com>\n",
      "Authors:\n",
      "Author: Aaron Lun [aut, cre], Gordon Smyth [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - Rcpp, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - edgeR, type: imports\n",
      "  - limma, type: imports\n",
      "  - methods, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - stats, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - metapod, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/csaw.html\n",
      "\n",
      "Name: celda\n",
      "Description:\n",
      "Celda is a suite of Bayesian hierarchical models for clustering single-cell RNA-sequencing (scRNA-seq) data. It is able to perform \"bi-clustering\" and simultaneously cluster genes into gene modules and cells into cell subpopulations. It also contains DecontX, a novel Bayesian method to computationally estimate and remove RNA contamination in individual cells without empty droplet information. A variety of scRNA-seq data visualization functions is also included.\n",
      "Version: 1.14.2\n",
      "Publication date: None\n",
      "Mantainer: Joshua Campbell <camp@bu.edu>\n",
      "Authors:\n",
      "Author: Joshua Campbell [aut, cre], Shiyi Yang [aut], Zhe Wang [aut], Sean Corbett [aut], Yusuke Koga [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - SingleCellExperiment, type: depends\n",
      "  - Matrix, type: depends\n",
      "  - plyr, type: imports\n",
      "  - foreach, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - grid, type: imports\n",
      "  - scales, type: imports\n",
      "  - gtable, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - doParallel, type: imports\n",
      "  - digest, type: imports\n",
      "  - methods, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - data.table, type: imports\n",
      "  - Rcpp, type: imports\n",
      "  - RcppEigen, type: imports\n",
      "  - uwot, type: imports\n",
      "  - enrichR, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - MCMCprecision, type: imports\n",
      "  - ggrepel, type: imports\n",
      "  - Rtsne, type: imports\n",
      "  - withr, type: imports\n",
      "  - scater, type: imports\n",
      "  - scran, type: imports\n",
      "  - dbscan, type: imports\n",
      "  - DelayedArray, type: imports\n",
      "  - stringr, type: imports\n",
      "  - ComplexHeatmap, type: imports\n",
      "  - multipanelfigure, type: imports\n",
      "  - circlize, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/celda.html\n",
      "\n",
      "Name: QFeatures\n",
      "Description:\n",
      "The QFeatures infrastructure enables the management and processing of quantitative features for high-throughput mass spectrometry assays. It provides a familiar Bioconductor user experience to manages quantitative data across different assay levels (such as peptide spectrum matches, peptides and proteins) in a coherent and tractable format.\n",
      "Version: 1.8.0\n",
      "Publication date: None\n",
      "Mantainer: Laurent Gatto <laurent.gatto@uclouvain.be>\n",
      "Authors:\n",
      "Author: Laurent Gatto [aut, cre] , Christophe Vanderaa [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - MultiAssayExperiment, type: depends\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - ProtGenerics, type: imports\n",
      "  - AnnotationFilter, type: imports\n",
      "  - lazyeval, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - MsCoreUtils, type: imports\n",
      "  - igraph, type: imports\n",
      "  - plotly, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/QFeatures.html\n",
      "\n",
      "Name: supraHex\n",
      "Description:\n",
      "A supra-hexagonal map is a giant hexagon on a 2-dimensional grid seamlessly consisting of smaller hexagons. It is supposed to train, analyse and visualise a high-dimensional omics input data. The supraHex is able to carry out gene clustering/meta-clustering and sample correlation, plus intuitive visualisations to facilitate exploratory analysis. More importantly, it allows for overlaying additional data onto the trained map to explore relations between input and additional data. So with supraHex, it is also possible to carry out multilayer omics data comparisons. Newly added utilities are advanced heatmap visualisation and tree-based analysis of sample relationships. Uniquely to this package, users can ultrafastly understand any tabular omics data, both scientifically and artistically, especially in a sample-specific fashion but without loss of information on large genes.\n",
      "Version: 1.36.0\n",
      "Publication date: None\n",
      "Mantainer: Hai Fang <hfang@well.ox.ac.uk>\n",
      "Authors:\n",
      "Author: Hai Fang and Julian Gough\n",
      "Dependencies:\n",
      "  - R (>= 3.6) type: depends\n",
      "  - hexbin, type: depends\n",
      "  - ape, type: imports\n",
      "  - MASS, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - readr, type: imports\n",
      "  - tibble, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - stringr, type: imports\n",
      "  - purrr, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - igraph, type: imports\n",
      "  - methods, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/supraHex.html\n",
      "\n",
      "Name: progeny\n",
      "Description:\n",
      "PROGENy is resource that leverages a large compendium of publicly available signaling perturbation experiments to yield a common core of pathway responsive genes for human and mouse. These, coupled with any statistical method, can be used to infer pathway activities from bulk or single-cell transcriptomics.\n",
      "Version: 1.20.0\n",
      "Publication date: None\n",
      "Mantainer: AurÃ©lien Dugourd <aurelien.dugourd@bioquant.uni-heidelberg.de>\n",
      "Authors:\n",
      "Author: Michael Schubert [aut], Alberto Valdeolivas [ctb] , Christian H. Holland [ctb] , Igor Bulanov [ctb], AurÃ©lien Dugourd [cre, ctb]\n",
      "Dependencies:\n",
      "  - R (>= 3.6.0) type: depends\n",
      "  - Biobase, type: imports\n",
      "  - stats, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - ggrepel, type: imports\n",
      "  - gridExtra, type: imports\n",
      "  - decoupleR, type: imports\n",
      "  - reshape2, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Apache License (== 2.0) | file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/progeny.html\n",
      "\n",
      "Name: OmnipathR\n",
      "Description:\n",
      "A client for the OmniPath web service (https://www.omnipathdb.org) and many other resources. It also includes functions to transform and pretty print some of the downloaded data, functions to access a number of other resources such as BioPlex, ConsensusPathDB, EVEX, Gene Ontology, Guide to Pharmacology (IUPHAR/BPS), Harmonizome, HTRIdb, Human Phenotype Ontology, InWeb InBioMap, KEGG Pathway, Pathway Commons, Ramilowski et al. 2015, RegNetwork, ReMap, TF census, TRRUST and Vinayagam et al. 2011. Furthermore, OmnipathR features a close integration with the NicheNet method for ligand activity prediction from transcriptomics data, and its R implementation `nichenetr` (available only on github).\n",
      "Version: 3.5.25\n",
      "Publication date: None\n",
      "Mantainer: Denes Turei <turei.denes@gmail.com>\n",
      "Authors:\n",
      "Author: Alberto Valdeolivas [aut] , Denes Turei [cre, aut] , Attila Gabor [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - checkmate, type: imports\n",
      "  - crayon, type: imports\n",
      "  - curl, type: imports\n",
      "  - digest, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - httr, type: imports\n",
      "  - igraph, type: imports\n",
      "  - jsonlite, type: imports\n",
      "  - later, type: imports\n",
      "  - logger, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - progress, type: imports\n",
      "  - purrr, type: imports\n",
      "  - rappdirs, type: imports\n",
      "  - readr (>= 2.0.0) type: imports\n",
      "  - readxl, type: imports\n",
      "  - rlang, type: imports\n",
      "  - rmarkdown, type: imports\n",
      "  - rvest, type: imports\n",
      "  - stats, type: imports\n",
      "  - stringr, type: imports\n",
      "  - tibble, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - tidyselect, type: imports\n",
      "  - tools, type: imports\n",
      "  - utils, type: imports\n",
      "  - withr, type: imports\n",
      "  - xml2, type: imports\n",
      "  - yaml, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/OmnipathR.html\n",
      "\n",
      "Name: QuasR\n",
      "Description:\n",
      "This package provides a framework for the quantification and analysis of Short Reads. It covers a complete workflow starting from raw sequence reads, over creation of alignments and quality control plots, to the quantification of genomic regions of interest.\n",
      "Version: 1.38.0\n",
      "Publication date: None\n",
      "Mantainer: Michael Stadler <michael.stadler@fmi.ch>\n",
      "Authors:\n",
      "Author: Anita Lerch [aut], Charlotte Soneson [aut] , Dimos Gaidatzis [aut], Michael Stadler [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - parallel, type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - Rbowtie, type: depends\n",
      "  - methods, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - utils, type: imports\n",
      "  - stats, type: imports\n",
      "  - tools, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - BSgenome, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - GenomicFeatures, type: imports\n",
      "  - ShortRead, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - GenomicFiles, type: imports\n",
      "  - AnnotationDbi, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/QuasR.html\n",
      "\n",
      "Name: CGHbase\n",
      "Description:\n",
      "Contains functions and classes that are needed by arrayCGH packages.\n",
      "Version: 1.58.0\n",
      "Publication date: None\n",
      "Mantainer: Mark van de Wiel <mark.vdwiel@vumc.nl>\n",
      "Authors:\n",
      "Author: Sjoerd Vosse, Mark van de Wiel\n",
      "Dependencies:\n",
      "  - R (>= 2.10) type: depends\n",
      "  - methods, type: depends\n",
      "  - Biobase, type: depends\n",
      "  - marray, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/CGHbase.html\n",
      "\n",
      "Name: MSstatsConvert\n",
      "Description:\n",
      "MSstatsConvert provides tools for importing reports of Mass Spectrometry data processing tools into R format suitable for statistical analysis using the MSstats and MSstatsTMT packages.\n",
      "Version: 1.8.2\n",
      "Publication date: None\n",
      "Mantainer: Mateusz Staniak <mtst@mstaniak.pl>\n",
      "Authors:\n",
      "Author: Mateusz Staniak [aut, cre], Meena Choi [aut], Ting Huang [aut], Olga Vitek [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - data.table, type: imports\n",
      "  - log4r, type: imports\n",
      "  - methods, type: imports\n",
      "  - checkmate, type: imports\n",
      "  - utils, type: imports\n",
      "  - stringi, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/MSstatsConvert.html\n",
      "\n",
      "Name: ATACseqQC\n",
      "Description:\n",
      "ATAC-seq, an assay for Transposase-Accessible Chromatin using sequencing, is a rapid and sensitive method for chromatin accessibility analysis. It was developed as an alternative method to MNase-seq, FAIRE-seq and DNAse-seq. Comparing to the other methods, ATAC-seq requires less amount of the biological samples and time to process. In the process of analyzing several ATAC-seq dataset produced in our labs, we learned some of the unique aspects of the quality assessment for ATAC-seq data.To help users to quickly assess whether their ATAC-seq experiment is successful, we developed ATACseqQC package partially following the guideline published in Nature Method 2013 (Greenleaf et al.), including diagnostic plot of fragment size distribution, proportion of mitochondria reads, nucleosome positioning pattern, and CTCF or other Transcript Factor footprints.\n",
      "Version: 1.22.0\n",
      "Publication date: None\n",
      "Mantainer: Jianhong Ou <jianhong.ou@duke.edu>\n",
      "Authors:\n",
      "Author: Jianhong Ou, Haibo Liu, Feng Yan, Jun Yu, Michelle Kelliher, Lucio Castilla, Nathan Lawson, Lihua Julie Zhu\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - BiocGenerics, type: depends\n",
      "  - S4Vectors, type: depends\n",
      "  - BSgenome, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - ChIPpeakAnno, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - GenomicAlignments, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - GenomicScores, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grid, type: imports\n",
      "  - limma, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - randomForest, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - stats, type: imports\n",
      "  - motifStack, type: imports\n",
      "  - preseqR, type: imports\n",
      "  - utils, type: imports\n",
      "  - KernSmooth, type: imports\n",
      "  - edgeR, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ATACseqQC.html\n",
      "\n",
      "Name: scde\n",
      "Description:\n",
      "The scde package implements a set of statistical methods for analyzing single-cell RNA-seq data. scde fits individual error models for single-cell RNA-seq measurements. These models can then be used for assessment of differential expression between groups of cells, as well as other types of analysis. The scde package also contains the pagoda framework which applies pathway and gene set overdispersion analysis to identify and characterize putative cell subpopulations based on transcriptional signatures. The overall approach to the differential expression analysis is detailed in the following publication: \"Bayesian approach to single-cell differential expression analysis\" (Kharchenko PV, Silberstein L, Scadden DT, Nature Methods, doi: 10.1038/nmeth.2967). The overall approach to subpopulation identification and characterization is detailed in the following pre-print: \"Characterizing transcriptional heterogeneity through pathway and gene set overdispersion analysis\" (Fan J, Salathia N, Liu R, Kaeser G, Yung Y, Herman J, Kaper F, Fan JB, Zhang K, Chun J, and Kharchenko PV, Nature Methods, doi:10.1038/nmeth.3734).\n",
      "Version: 2.26.2\n",
      "Publication date: None\n",
      "Mantainer: Jean Fan <jeanfan@jhu.edu>\n",
      "Authors:\n",
      "Author: Peter Kharchenko [aut, cre], Jean Fan [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.0.0) type: depends\n",
      "  - flexmix, type: depends\n",
      "  - Rcpp (>= 0.10.4) type: imports\n",
      "  - RcppArmadillo (>= 0.5.400.2.0) type: imports\n",
      "  - mgcv, type: imports\n",
      "  - Rook, type: imports\n",
      "  - rjson, type: imports\n",
      "  - MASS, type: imports\n",
      "  - Cairo, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - edgeR, type: imports\n",
      "  - quantreg, type: imports\n",
      "  - methods, type: imports\n",
      "  - nnet, type: imports\n",
      "  - RMTstat, type: imports\n",
      "  - extRemes, type: imports\n",
      "  - pcaMethods, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - parallel, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/scde.html\n",
      "\n",
      "Name: LoomExperiment\n",
      "Description:\n",
      "The LoomExperiment package provide a means to easily convert the Bioconductor \"Experiment\" classes to loom files and vice versa.\n",
      "Version: 1.16.0\n",
      "Publication date: None\n",
      "Mantainer: Bioconductor Package Maintainer <maintainer@bioconductor.org>\n",
      "Authors:\n",
      "Author: Martin Morgan, Daniel Van Twisk\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - S4Vectors, type: depends\n",
      "  - SingleCellExperiment, type: depends\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - methods, type: depends\n",
      "  - rhdf5, type: depends\n",
      "  - BiocIO, type: depends\n",
      "  - DelayedArray, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - HDF5Array, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - stats, type: imports\n",
      "  - stringr, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/LoomExperiment.html\n",
      "\n",
      "Name: MicrobiotaProcess\n",
      "Description:\n",
      "MicrobiotaProcess is an R package for analysis, visualization and biomarker discovery of microbial datasets. It introduces MPSE class, this make it more interoperable with the existing computing ecosystem. Moreover, it introduces a tidy microbiome data structure paradigm and analysis grammar. It provides a wide variety of microbiome data analysis procedures under the unified and common framework (tidy-like framework).\n",
      "Version: 1.10.3\n",
      "Publication date: None\n",
      "Mantainer: Shuangbin Xu <xshuangbin@163.com>\n",
      "Authors:\n",
      "Author: Shuangbin Xu [aut, cre] , Guangchuang Yu [aut, ctb]\n",
      "Dependencies:\n",
      "  - R (>= 4.0.0) type: depends\n",
      "  - ape, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - ggrepel, type: imports\n",
      "  - vegan, type: imports\n",
      "  - zoo, type: imports\n",
      "  - ggtree, type: imports\n",
      "  - tidytree (>= 0.4.2) type: imports\n",
      "  - MASS, type: imports\n",
      "  - methods, type: imports\n",
      "  - rlang, type: imports\n",
      "  - tibble, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - coin, type: imports\n",
      "  - ggsignif, type: imports\n",
      "  - patchwork, type: imports\n",
      "  - ggstar, type: imports\n",
      "  - tidyselect, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - foreach, type: imports\n",
      "  - treeio, type: imports\n",
      "  - pillar, type: imports\n",
      "  - cli, type: imports\n",
      "  - plyr, type: imports\n",
      "  - dtplyr, type: imports\n",
      "  - ggtreeExtra, type: imports\n",
      "  - data.table, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 3.0)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/MicrobiotaProcess.html\n",
      "\n",
      "Name: MOFA2\n",
      "Description:\n",
      "The MOFA2 package contains a collection of tools for training and analysing multi-omic factor analysis (MOFA). MOFA is a probabilistic factor model that aims to identify principal axes of variation from data sets that can comprise multiple omic layers and/or groups of samples. Additional time or space information on the samples can be incorporated using the MEFISTO framework, which is part of MOFA2. Downstream analysis functions to inspect molecular features underlying each factor, vizualisation, imputation etc are available.\n",
      "Version: 1.8.0\n",
      "Publication date: None\n",
      "Mantainer: Ricard Argelaguet <ricard.argelaguet@gmail.com>\n",
      "Authors:\n",
      "Author: Ricard Argelaguet [aut, cre] , Damien Arnol [aut] , Danila Bredikhin [aut] , Britta Velten [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - rhdf5, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - pheatmap, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - methods, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - cowplot, type: imports\n",
      "  - ggrepel, type: imports\n",
      "  - reticulate, type: imports\n",
      "  - HDF5Array, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - stats, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - forcats, type: imports\n",
      "  - utils, type: imports\n",
      "  - corrplot, type: imports\n",
      "  - DelayedArray, type: imports\n",
      "  - Rtsne, type: imports\n",
      "  - uwot, type: imports\n",
      "  - basilisk, type: imports\n",
      "  - stringi, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/MOFA2.html\n",
      "\n",
      "Name: GENESIS\n",
      "Description:\n",
      "The GENESIS package provides methodology for estimating, inferring, and accounting for population and pedigree structure in genetic analyses. The current implementation provides functions to perform PC-AiR (Conomos et al., 2015, Gen Epi) and PC-Relate (Conomos et al., 2016, AJHG). PC-AiR performs a Principal Components Analysis on genome-wide SNP data for the detection of population structure in a sample that may contain known or cryptic relatedness. Unlike standard PCA, PC-AiR accounts for relatedness in the sample to provide accurate ancestry inference that is not confounded by family structure. PC-Relate uses ancestry representative principal components to adjust for population structure/ancestry and accurately estimate measures of recent genetic relatedness such as kinship coefficients, IBD sharing probabilities, and inbreeding coefficients. Additionally, functions are provided to perform efficient variance component estimation and mixed model association testing for both quantitative and binary phenotypes.\n",
      "Version: 2.28.0\n",
      "Publication date: None\n",
      "Mantainer: Stephanie M. Gogarten <sdmorris@uw.edu>\n",
      "Authors:\n",
      "Author: Matthew P. Conomos, Stephanie M. Gogarten, Lisa Brown, Han Chen, Thomas Lumley, Kenneth Rice, Tamar Sofer, Adrienne Stilp, Timothy Thornton, Chaoyu Yu\n",
      "Dependencies:\n",
      "  - Biobase, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - GWASTools, type: imports\n",
      "  - gdsfmt, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - SeqArray, type: imports\n",
      "  - SeqVarTools, type: imports\n",
      "  - SNPRelate, type: imports\n",
      "  - data.table, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - igraph, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - methods, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/GENESIS.html\n",
      "\n",
      "Name: cBioPortalData\n",
      "Description:\n",
      "The cBioPortalData R package accesses study datasets from the cBio Cancer Genomics Portal. It accesses the data either from the pre-packaged zip / tar files or from the API interface that was recently implemented by the cBioPortal Data Team. The package can provide data in either tabular format or with MultiAssayExperiment object that uses familiar Bioconductor data representations.\n",
      "Version: 2.10.3\n",
      "Publication date: None\n",
      "Mantainer: Marcel Ramos <marcel.ramos@roswellpark.org>\n",
      "Authors:\n",
      "Author: Levi Waldron [aut], Marcel Ramos [aut, cre] , Karim Mezhoud [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 4.2.0) type: depends\n",
      "  - AnVIL, type: depends\n",
      "  - MultiAssayExperiment, type: depends\n",
      "  - BiocFileCache, type: imports\n",
      "  - digest, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - httr, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - methods, type: imports\n",
      "  - readr, type: imports\n",
      "  - RaggedExperiment, type: imports\n",
      "  - RTCGAToolbox, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - stats, type: imports\n",
      "  - tibble, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - TCGAutils, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "AGPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/cBioPortalData.html\n",
      "\n",
      "Name: MAGeCKFlute\n",
      "Description:\n",
      "CRISPR (clustered regularly interspaced short palindrome repeats) coupled with nuclease Cas9 (CRISPR/Cas9) screens represent a promising technology to systematically evaluate gene functions. Data analysis for CRISPR/Cas9 screens is a critical process that includes identifying screen hits and exploring biological functions for these hits in downstream analysis. We have previously developed two algorithms, MAGeCK and MAGeCK-VISPR, to analyze CRISPR/Cas9 screen data in various scenarios. These two algorithms allow users to perform quality control, read count generation and normalization, and calculate beta score to evaluate gene selection performance. In downstream analysis, the biological functional analysis is required for understanding biological functions of these identified genes with different screening purposes. Here, We developed MAGeCKFlute for supporting downstream analysis. MAGeCKFlute provides several strategies to remove potential biases within sgRNA-level read counts and gene-level beta scores. The downstream analysis with the package includes identifying essential, non-essential, and target-associated genes, and performing biological functional category analysis, pathway enrichment analysis and protein complex enrichment analysis of these genes. The package also visualizes genes in multiple ways to benefit users exploring screening data. Collectively, MAGeCKFlute enables accurate identification of essential, non-essential, and targeted genes, as well as their related biological functions. This vignette explains the use of the package and demonstrates typical workflows.\n",
      "Version: 2.2.0\n",
      "Publication date: None\n",
      "Mantainer: Wubing Zhang <Watson5bZhang@gmail.com>\n",
      "Authors:\n",
      "Author: Binbin Wang, Wubing Zhang, Feizhen Wu, Wei Li & X. Shirley Liu\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - Biobase, type: imports\n",
      "  - gridExtra, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - ggrepel, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - grid, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - DOSE, type: imports\n",
      "  - clusterProfiler, type: imports\n",
      "  - pathview, type: imports\n",
      "  - enrichplot, type: imports\n",
      "  - msigdbr, type: imports\n",
      "  - depmap, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>=3)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/MAGeCKFlute.html\n",
      "\n",
      "Name: scMerge\n",
      "Description:\n",
      "Like all gene expression data, single-cell RNA-seq (scRNA-Seq) data suffers from batch effects and other unwanted variations that makes accurate biological interpretations difficult. The scMerge method leverages factor analysis, stably expressed genes (SEGs) and (pseudo-) replicates to remove unwanted variations and merge multiple scRNA-Seq data. This package contains all the necessary functions in the scMerge pipeline, including the identification of SEGs, replication-identification methods, and merging of scRNA-Seq data.\n",
      "Version: 1.14.0\n",
      "Publication date: None\n",
      "Mantainer: Yingxin Lin <yingxin.lin@sydney.edu.au>\n",
      "Authors:\n",
      "Author: Yingxin Lin [aut, cre], Kevin Wang [aut], Sydney Bioinformatics and Biometrics Group [fnd]\n",
      "Dependencies:\n",
      "  - R (>= 3.6.0) type: depends\n",
      "  - BiocParallel, type: imports\n",
      "  - BiocSingular, type: imports\n",
      "  - cluster, type: imports\n",
      "  - DelayedArray, type: imports\n",
      "  - DelayedMatrixStats, type: imports\n",
      "  - distr, type: imports\n",
      "  - igraph, type: imports\n",
      "  - M3Drop, type: imports\n",
      "  - parallel, type: imports\n",
      "  - pdist, type: imports\n",
      "  - proxy, type: imports\n",
      "  - ruv, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - SingleCellExperiment, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/scMerge.html\n",
      "\n",
      "Name: ggmsa\n",
      "Description:\n",
      "A visual exploration tool for multiple sequence alignment and associated data. Supports MSA of DNA, RNA, and protein sequences using 'ggplot2'. Multiple sequence alignment can easily be combined with other 'ggplot2' plots, such as phylogenetic tree Visualized by 'ggtree', boxplot, genome map and so on. More features: visualization of sequence logos, sequence bundles, RNA secondary structures and detection of sequence recombinations.\n",
      "Version: 1.4.0\n",
      "Publication date: None\n",
      "Mantainer: Lang Zhou <nyzhoulang@gmail.com>\n",
      "Authors:\n",
      "Author: Lang Zhou [aut, cre], Guangchuang Yu [aut, ths] , Shuangbin Xu [ctb], Huina Huang [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 4.1.0) type: depends\n",
      "  - Biostrings, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - utils, type: imports\n",
      "  - stats, type: imports\n",
      "  - aplot, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - ggalt, type: imports\n",
      "  - ggforce, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - R4RNA, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - seqmagick, type: imports\n",
      "  - grid, type: imports\n",
      "  - methods, type: imports\n",
      "  - statebins, type: imports\n",
      "  - ggtree, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ggmsa.html\n",
      "\n",
      "Name: rols\n",
      "Description:\n",
      "The rols package is an interface to the Ontology Lookup Service (OLS) to access and query hundred of ontolgies directly from R.\n",
      "Version: 2.26.0\n",
      "Publication date: None\n",
      "Mantainer: Laurent Gatto <laurent.gatto@uclouvain.be>\n",
      "Authors:\n",
      "Author: Laurent Gatto [aut, cre], Tiage Chedraoui Silva [ctb], Andrew Clugston [ctb]\n",
      "Dependencies:\n",
      "  - methods, type: depends\n",
      "  - httr, type: imports\n",
      "  - progress, type: imports\n",
      "  - jsonlite, type: imports\n",
      "  - utils, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/rols.html\n",
      "\n",
      "Name: GDCRNATools\n",
      "Description:\n",
      "This is an easy-to-use package for downloading, organizing, and integrative analyzing RNA expression data in GDC with an emphasis on deciphering the lncRNA-mRNA related ceRNA regulatory network in cancer. Three databases of lncRNA-miRNA interactions including spongeScan, starBase, and miRcode, as well as three databases of mRNA-miRNA interactions including miRTarBase, starBase, and miRcode are incorporated into the package for ceRNAs network construction. limma, edgeR, and DESeq2 can be used to identify differentially expressed genes/miRNAs. Functional enrichment analyses including GO, KEGG, and DO can be performed based on the clusterProfiler and DO packages. Both univariate CoxPH and KM survival analyses of multiple genes can be implemented in the package. Besides some routine visualization functions such as volcano plot, bar plot, and KM plot, a few simply shiny apps are developed to facilitate visualization of results on a local webpage.\n",
      "Version: 1.18.0\n",
      "Publication date: None\n",
      "Mantainer: Ruidong Li <rli012@ucr.edu>, Han Qu <hqu002@ucr.edu>\n",
      "Authors:\n",
      "Author: Ruidong Li, Han Qu, Shibo Wang, Julong Wei, Le Zhang, Renyuan Ma, Jianming Lu, Jianguo Zhu, Wei-De Zhong, Zhenyu Jia\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - shiny, type: imports\n",
      "  - jsonlite, type: imports\n",
      "  - rjson, type: imports\n",
      "  - XML, type: imports\n",
      "  - limma, type: imports\n",
      "  - edgeR, type: imports\n",
      "  - DESeq2, type: imports\n",
      "  - clusterProfiler, type: imports\n",
      "  - DOSE, type: imports\n",
      "  - org.Hs.eg.db, type: imports\n",
      "  - biomaRt, type: imports\n",
      "  - survival, type: imports\n",
      "  - survminer, type: imports\n",
      "  - pathview, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - gplots, type: imports\n",
      "  - DT, type: imports\n",
      "  - GenomicDataCommons, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/GDCRNATools.html\n",
      "\n",
      "Name: clusterExperiment\n",
      "Description:\n",
      "Provides functionality for running and comparing many different clusterings of single-cell sequencing data or other large mRNA Expression data sets.\n",
      "Version: 2.18.2\n",
      "Publication date: None\n",
      "Mantainer: Elizabeth Purdom <epurdom@stat.berkeley.edu>\n",
      "Authors:\n",
      "Author: Elizabeth Purdom [aut, cre, cph], Davide Risso [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.6.0) type: depends\n",
      "  - SingleCellExperiment, type: depends\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - BiocGenerics, type: depends\n",
      "  - methods, type: imports\n",
      "  - NMF, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - ape (>= 5.0) type: imports\n",
      "  - cluster, type: imports\n",
      "  - stats, type: imports\n",
      "  - limma, type: imports\n",
      "  - howmany, type: imports\n",
      "  - locfdr, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - graphics, type: imports\n",
      "  - parallel, type: imports\n",
      "  - BiocSingular, type: imports\n",
      "  - kernlab, type: imports\n",
      "  - stringr, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - DelayedArray, type: imports\n",
      "  - HDF5Array, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - Rcpp, type: imports\n",
      "  - edgeR, type: imports\n",
      "  - scales, type: imports\n",
      "  - zinbwave, type: imports\n",
      "  - phylobase, type: imports\n",
      "  - pracma, type: imports\n",
      "  - mbkmeans, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/clusterExperiment.html\n",
      "\n",
      "Name: iClusterPlus\n",
      "Description:\n",
      "Integrative clustering of multiple genomic data using a joint latent variable model.\n",
      "Version: 1.34.3\n",
      "Publication date: None\n",
      "Mantainer: Qianxing Mo <qianxing.mo@moffitt.org>, Ronglai Shen <shenr@mskcc.org>\n",
      "Authors:\n",
      "Author: Qianxing Mo, Ronglai Shen\n",
      "Dependencies:\n",
      "  - R (>= 3.3.0) type: depends\n",
      "  - parallel, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/iClusterPlus.html\n",
      "\n",
      "Name: powerTCR\n",
      "Description:\n",
      "This package provides a model for the clone size distribution of the TCR repertoire. Further, it permits comparative analysis of TCR repertoire libraries based on theoretical model fits.\n",
      "Version: 1.18.0\n",
      "Publication date: None\n",
      "Mantainer: Hillary Koch <hillary.koch01@gmail.com>\n",
      "Authors:\n",
      "Author: Hillary Koch\n",
      "Dependencies:\n",
      "  - cubature, type: imports\n",
      "  - doParallel, type: imports\n",
      "  - evmix, type: imports\n",
      "  - foreach, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - methods, type: imports\n",
      "  - parallel, type: imports\n",
      "  - purrr, type: imports\n",
      "  - stats, type: imports\n",
      "  - truncdist, type: imports\n",
      "  - vegan, type: imports\n",
      "  - VGAM, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/powerTCR.html\n",
      "\n",
      "Name: trackViewer\n",
      "Description:\n",
      "Visualize mapped reads along with annotation as track layers for NGS dataset such as ChIP-seq, RNA-seq, miRNA-seq, DNA-seq, SNPs and methylation data.\n",
      "Version: 1.34.0\n",
      "Publication date: None\n",
      "Mantainer: Jianhong Ou <jianhong.ou@duke.edu>\n",
      "Authors:\n",
      "Author: Jianhong Ou [aut, cre] , Julie Lihua Zhu [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - grDevices, type: depends\n",
      "  - methods, type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - grid, type: depends\n",
      "  - Rcpp, type: depends\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - GenomicAlignments, type: imports\n",
      "  - GenomicFeatures, type: imports\n",
      "  - Gviz, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - scales, type: imports\n",
      "  - tools, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - AnnotationDbi, type: imports\n",
      "  - grImport, type: imports\n",
      "  - htmlwidgets, type: imports\n",
      "  - plotrix, type: imports\n",
      "  - Rgraphviz, type: imports\n",
      "  - InteractionSet, type: imports\n",
      "  - graph, type: imports\n",
      "  - utils, type: imports\n",
      "  - rhdf5, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/trackViewer.html\n",
      "\n",
      "Name: UniProt.ws\n",
      "Description:\n",
      "The Universal Protein Resource (UniProt) is a comprehensive resource for protein sequence and annotation data. This package provides a collection of functions for retrieving, processing, and re-packaging UniProt web services. The package makes use of UniProt's modernized REST API and allows mapping of identifiers accross different databases.\n",
      "Version: 2.38.1\n",
      "Publication date: None\n",
      "Mantainer: Marcel Ramos <marcel.ramos@roswellpark.org>\n",
      "Authors:\n",
      "Author: Marc Carlson [aut], Csaba Ortutay [ctb], Marcel Ramos [aut, cre]\n",
      "Dependencies:\n",
      "  - methods, type: depends\n",
      "  - utils, type: depends\n",
      "  - RSQLite, type: depends\n",
      "  - BiocGenerics, type: depends\n",
      "  - AnnotationDbi, type: imports\n",
      "  - BiocFileCache, type: imports\n",
      "  - BiocBaseUtils, type: imports\n",
      "  - rjsoncons, type: imports\n",
      "  - jsonlite, type: imports\n",
      "  - httr, type: imports\n",
      "  - httpcache, type: imports\n",
      "  - progress, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/UniProt.ws.html\n",
      "\n",
      "Name: CGHcall\n",
      "Description:\n",
      "Calls aberrations for array CGH data using a six state mixture model as well as several biological concepts that are ignored by existing algorithms. Visualization of profiles is also provided.\n",
      "Version: 2.60.0\n",
      "Publication date: None\n",
      "Mantainer: Mark van de Wiel <mark.vdwiel@vumc.nl>\n",
      "Authors:\n",
      "Author: Mark van de Wiel, Sjoerd Vosse\n",
      "Dependencies:\n",
      "  - R (>= 2.0.0) type: depends\n",
      "  - impute, type: depends\n",
      "  - DNAcopy, type: depends\n",
      "  - methods, type: depends\n",
      "  - Biobase, type: depends\n",
      "  - CGHbase, type: depends\n",
      "  - snowfall, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (http://www.gnu.org/copyleft/gpl.html)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/CGHcall.html\n",
      "\n",
      "Name: simplifyEnrichment\n",
      "Description:\n",
      "A new clustering algorithm, \"binary cut\", for clustering similarity matrices of functional terms is implemeted in this package. It also provides functions for visualizing, summarizing and comparing the clusterings.\n",
      "Version: 1.8.0\n",
      "Publication date: None\n",
      "Mantainer: Zuguang Gu <z.gu@dkfz.de>\n",
      "Authors:\n",
      "Author: Zuguang Gu [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 3.6.0) type: depends\n",
      "  - BiocGenerics, type: depends\n",
      "  - grid, type: depends\n",
      "  - GOSemSim, type: imports\n",
      "  - ComplexHeatmap, type: imports\n",
      "  - circlize, type: imports\n",
      "  - GetoptLong, type: imports\n",
      "  - digest, type: imports\n",
      "  - tm, type: imports\n",
      "  - GO.db, type: imports\n",
      "  - org.Hs.eg.db, type: imports\n",
      "  - AnnotationDbi, type: imports\n",
      "  - slam, type: imports\n",
      "  - methods, type: imports\n",
      "  - clue, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - proxyC, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - cluster (>= 1.14.2) type: imports\n",
      "  - colorspace, type: imports\n",
      "  - GlobalOptions (>= 0.1.0) type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/simplifyEnrichment.html\n",
      "\n",
      "Name: rWikiPathways\n",
      "Description:\n",
      "Use this package to interface with the WikiPathways API. It provides programmatic access to WikiPathways content in multiple data and image formats, including official monthly release files and convenient GMT read/write functions.\n",
      "Version: 1.18.0\n",
      "Publication date: None\n",
      "Mantainer: Egon Willighagen <egon.willighagen@gmail.com>\n",
      "Authors:\n",
      "Author: Egon Willighagen [aut, cre] , Alex Pico [aut]\n",
      "Dependencies:\n",
      "  - httr, type: imports\n",
      "  - utils, type: imports\n",
      "  - XML, type: imports\n",
      "  - rjson, type: imports\n",
      "  - data.table, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - RCurl, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/rWikiPathways.html\n",
      "\n",
      "Name: MLInterfaces\n",
      "Description:\n",
      "This package provides uniform interfaces to machine learning code for data in R and Bioconductor containers.\n",
      "Version: 1.78.0\n",
      "Publication date: None\n",
      "Mantainer: V. Carey <stvjc@channing.harvard.edu>\n",
      "Authors:\n",
      "Author: Vince Carey <stvjc at channing.harvard.edu>, Robert Gentleman, Jess Mar, and contributions from Jason Vertrees <jv at cs.dartmouth.edu> and Laurent Gatto <lg390 at cam.ac.uk>\n",
      "Dependencies:\n",
      "  - R (>= 3.5) type: depends\n",
      "  - Rcpp, type: depends\n",
      "  - methods, type: depends\n",
      "  - BiocGenerics, type: depends\n",
      "  - Biobase, type: depends\n",
      "  - annotate, type: depends\n",
      "  - cluster, type: depends\n",
      "  - gdata, type: imports\n",
      "  - pls, type: imports\n",
      "  - sfsmisc, type: imports\n",
      "  - MASS, type: imports\n",
      "  - rpart, type: imports\n",
      "  - genefilter, type: imports\n",
      "  - fpc, type: imports\n",
      "  - ggvis, type: imports\n",
      "  - shiny, type: imports\n",
      "  - gbm, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - hwriter, type: imports\n",
      "  - threejs (>= 0.2.2) type: imports\n",
      "  - mlbench, type: imports\n",
      "  - stats4, type: imports\n",
      "  - tools, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/MLInterfaces.html\n",
      "\n",
      "Name: LBE\n",
      "Description:\n",
      "LBE is an efficient procedure for estimating the proportion of true null hypotheses, the false discovery rate (and so the q-values) in the framework of estimating procedures based on the marginal distribution of the p-values without assumption for the alternative hypothesis.\n",
      "Version: 1.66.0\n",
      "Publication date: None\n",
      "Mantainer: Cyril Dalmasso <dalmasso@vjf.inserm.fr>\n",
      "Authors:\n",
      "Author: Cyril Dalmasso\n",
      "Dependencies:\n",
      "  - stats, type: depends\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/LBE.html\n",
      "\n",
      "Name: rpx\n",
      "Description:\n",
      "The rpx package implements an interface to proteomics data submitted to the ProteomeXchange consortium.\n",
      "Version: 2.6.0\n",
      "Publication date: None\n",
      "Mantainer: Laurent Gatto <laurent.gatto@uclouvain.be>\n",
      "Authors:\n",
      "Author: Laurent Gatto\n",
      "Dependencies:\n",
      "  - methods, type: depends\n",
      "  - BiocFileCache, type: imports\n",
      "  - jsonlite, type: imports\n",
      "  - xml2, type: imports\n",
      "  - RCurl, type: imports\n",
      "  - curl, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/rpx.html\n",
      "\n",
      "Name: BumpyMatrix\n",
      "Description:\n",
      "Implements the BumpyMatrix class and several subclasses for holding non-scalar objects in each entry of the matrix. This is akin to a ragged array but the raggedness is in the third dimension, much like a bumpy surface - hence the name. Of particular interest is the BumpyDataFrameMatrix, where each entry is a Bioconductor data frame. This allows us to naturally represent multivariate data in a format that is compatible with two-dimensional containers like the SummarizedExperiment and MultiAssayExperiment objects.\n",
      "Version: 1.6.0\n",
      "Publication date: None\n",
      "Mantainer: Aaron Lun <infinite.monkeys.with.keyboards@gmail.com>\n",
      "Authors:\n",
      "Author: Aaron Lun [aut, cre], Genentech, Inc. [cph]\n",
      "Dependencies:\n",
      "  - utils, type: imports\n",
      "  - methods, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - IRanges, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/BumpyMatrix.html\n",
      "\n",
      "Name: DEsingle\n",
      "Description:\n",
      "DEsingle is an R package for differential expression (DE) analysis of single-cell RNA-seq (scRNA-seq) data. It defines and detects 3 types of differentially expressed genes between two groups of single cells, with regard to different expression status (DEs), differential expression abundance (DEa), and general differential expression (DEg). DEsingle employs Zero-Inflated Negative Binomial model to estimate the proportion of real and dropout zeros and to define and detect the 3 types of DE genes. Results showed that DEsingle outperforms existing methods for scRNA-seq DE analysis, and can reveal different types of DE genes that are enriched in different biological functions.\n",
      "Version: 1.18.1\n",
      "Publication date: None\n",
      "Mantainer: Zhun Miao <miaoz13@tsinghua.org.cn>\n",
      "Authors:\n",
      "Author: Zhun Miao <miaoz13 at tsinghua.org.cn>\n",
      "Dependencies:\n",
      "  - R (>= 3.4.0) type: depends\n",
      "  - stats, type: imports\n",
      "  - Matrix (>= 1.2-14) type: imports\n",
      "  - MASS (>= 7.3-45) type: imports\n",
      "  - VGAM (>= 1.0-2) type: imports\n",
      "  - bbmle (>= 1.0.18) type: imports\n",
      "  - gamlss (>= 4.4-0) type: imports\n",
      "  - maxLik (>= 1.3-4) type: imports\n",
      "  - pscl (>= 1.4.9) type: imports\n",
      "  - BiocParallel, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/DEsingle.html\n",
      "\n",
      "Name: TCC\n",
      "Description:\n",
      "This package provides a series of functions for performing differential expression analysis from RNA-seq count data using robust normalization strategy (called DEGES). The basic idea of DEGES is that potential differentially expressed genes or transcripts (DEGs) among compared samples should be removed before data normalization to obtain a well-ranked gene list where true DEGs are top-ranked and non-DEGs are bottom ranked. This can be done by performing a multi-step normalization strategy (called DEGES for DEG elimination strategy). A major characteristic of TCC is to provide the robust normalization methods for several kinds of count data (two-group with or without replicates, multi-group/multi-factor, and so on) by virtue of the use of combinations of functions in depended packages.\n",
      "Version: 1.38.0\n",
      "Publication date: None\n",
      "Mantainer: Jianqiang Sun <wukong@bi.a.u-tokyo.ac.jp>, Tomoaki Nishiyama <tomoakin@staff.kanazawa-u.ac.jp>\n",
      "Authors:\n",
      "Author: Jianqiang Sun, Tomoaki Nishiyama, Kentaro Shimizu, and Koji Kadota\n",
      "Dependencies:\n",
      "  - R (>= 3.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - DESeq2, type: depends\n",
      "  - edgeR, type: depends\n",
      "  - baySeq, type: depends\n",
      "  - ROC, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/TCC.html\n",
      "\n",
      "Name: rrvgo\n",
      "Description:\n",
      "Reduce and visualize lists of Gene Ontology terms by identifying redudance based on semantic similarity.\n",
      "Version: 1.10.0\n",
      "Publication date: None\n",
      "Mantainer: Sergi Sayols <sergisayolspuig@gmail.com>\n",
      "Authors:\n",
      "Author: Sergi Sayols [aut, cre]\n",
      "Dependencies:\n",
      "  - GOSemSim, type: imports\n",
      "  - AnnotationDbi, type: imports\n",
      "  - GO.db, type: imports\n",
      "  - pheatmap, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - ggrepel, type: imports\n",
      "  - treemap, type: imports\n",
      "  - tm, type: imports\n",
      "  - wordcloud, type: imports\n",
      "  - shiny, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - grid, type: imports\n",
      "  - stats, type: imports\n",
      "  - methods, type: imports\n",
      "  - umap, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/rrvgo.html\n",
      "\n",
      "Name: EnrichmentBrowser\n",
      "Description:\n",
      "The EnrichmentBrowser package implements essential functionality for the enrichment analysis of gene expression data. The analysis combines the advantages of set-based and network-based enrichment analysis in order to derive high-confidence gene sets and biological pathways that are differentially regulated in the expression data under investigation. Besides, the package facilitates the visualization and exploration of such sets and pathways.\n",
      "Version: 2.28.0\n",
      "Publication date: None\n",
      "Mantainer: Ludwig Geistlinger <Ludwig_Geistlinger@hms.harvard.edu>\n",
      "Authors:\n",
      "Author: Ludwig Geistlinger [aut, cre], Gergely Csaba [aut], Mara Santarelli [ctb], Mirko Signorelli [ctb], Marcel Ramos [ctb], Levi Waldron [ctb], Ralf Zimmer [aut]\n",
      "Dependencies:\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - graph, type: depends\n",
      "  - AnnotationDbi, type: imports\n",
      "  - BiocFileCache, type: imports\n",
      "  - BiocManager, type: imports\n",
      "  - GSEABase, type: imports\n",
      "  - GO.db, type: imports\n",
      "  - KEGGREST, type: imports\n",
      "  - KEGGgraph, type: imports\n",
      "  - Rgraphviz, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - SPIA, type: imports\n",
      "  - edgeR, type: imports\n",
      "  - graphite, type: imports\n",
      "  - hwriter, type: imports\n",
      "  - limma, type: imports\n",
      "  - methods, type: imports\n",
      "  - pathview, type: imports\n",
      "  - safe, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/EnrichmentBrowser.html\n",
      "\n",
      "Name: Icens\n",
      "Description:\n",
      "Many functions for computing the NPMLE for censored and truncated data.\n",
      "Version: 1.70.0\n",
      "Publication date: None\n",
      "Mantainer: Bioconductor Package Maintainer <maintainer@bioconductor.org>\n",
      "Authors:\n",
      "Author: R. Gentleman and Alain Vandal\n",
      "Dependencies:\n",
      "  - survival, type: depends\n",
      "  - graphics, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/Icens.html\n",
      "\n",
      "Name: QDNAseq\n",
      "Description:\n",
      "Quantitative DNA sequencing for chromosomal aberrations. The genome is divided into non-overlapping fixed-sized bins, number of sequence reads in each counted, adjusted with a simultaneous two-dimensional loess correction for sequence mappability and GC content, and filtered to remove spurious regions in the genome. Downstream steps of segmentation and calling are also implemented via packages DNAcopy and CGHcall, respectively.\n",
      "Version: 1.34.0\n",
      "Publication date: None\n",
      "Mantainer: Daoud Sie <d.sie@vumc.nl>\n",
      "Authors:\n",
      "Author: Ilari Scheinin [aut], Daoud Sie [aut, cre], Henrik Bengtsson [aut], Erik van Dijk [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 3.1.0) type: depends\n",
      "  - graphics, type: imports\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - CGHbase, type: imports\n",
      "  - CGHcall, type: imports\n",
      "  - DNAcopy, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - matrixStats (>= 0.60.0) type: imports\n",
      "  - R.utils (>= 2.9.0) type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - future.apply (>= 1.8.1) type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/QDNAseq.html\n",
      "\n",
      "Name: ctc\n",
      "Description:\n",
      "Tools for export and import classification trees and clusters to other programs\n",
      "Version: 1.72.0\n",
      "Publication date: None\n",
      "Mantainer: Antoine Lucas <antoinelucas@gmail.com>\n",
      "Authors:\n",
      "Author: Antoine Lucas <antoinelucas at gmail.com>, Laurent Gautier <laurent at cbs.dtu.dk>\n",
      "Dependencies:\n",
      "  - amap, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ctc.html\n",
      "\n",
      "Name: decoupleR\n",
      "Description:\n",
      "Many methods allow us to extract biological activities from omics data using information from prior knowledge resources, reducing the dimensionality for increased statistical power and better interpretability. Here, we present decoupleR, a Bioconductor package containing different statistical methods to extract these signatures within a unified framework. decoupleR allows the user to flexibly test any method with any resource. It incorporates methods that take into account the sign and weight of network interactions. decoupleR can be used with any omic, as long as its features can be linked to a biological process based on prior knowledge. For example, in transcriptomics gene sets regulated by a transcription factor, or in phospho-proteomics phosphosites that are targeted by a kinase.\n",
      "Version: 2.4.0\n",
      "Publication date: None\n",
      "Mantainer: Pau Badia-i-Mompel <pau.badia@uni-heidelberg.de>\n",
      "Authors:\n",
      "Author: Pau Badia-i-Mompel [aut, cre] , JesÃºs VÃ©lez-Santiago [aut] , Jana Braunger [aut] , Celina Geiss [aut] , Daniel Dimitrov [aut] , Sophia MÃ¼ller-Dott [aut] , Petr Taus [aut] , AurÃ©lien Dugourd [aut] , Christian H. Holland [aut] , Ricardo O. Ramirez Flores [aut] , Julio Saez-Rodriguez [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - broom, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - purrr, type: imports\n",
      "  - rlang, type: imports\n",
      "  - stats, type: imports\n",
      "  - stringr, type: imports\n",
      "  - tibble, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - tidyselect, type: imports\n",
      "  - withr, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3 + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/decoupleR.html\n",
      "\n",
      "Name: Rbowtie2\n",
      "Description:\n",
      "This package provides an R wrapper of the popular bowtie2 sequencing reads aligner and AdapterRemoval, a convenient tool for rapid adapter trimming, identification, and read merging. The package contains wrapper functions that allow for genome indexing and alignment to those indexes. The package also allows for the creation of .bam files via Rsamtools.\n",
      "Version: 2.4.1\n",
      "Publication date: None\n",
      "Mantainer: Zheng Wei <wzweizheng@qq.com>\n",
      "Authors:\n",
      "Author: Zheng Wei [aut, cre], Wei Zhang [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.1.0) type: depends\n",
      "  - magrittr, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 3)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/Rbowtie2.html\n",
      "\n",
      "Name: iSEE\n",
      "Description:\n",
      "Create an interactive Shiny-based graphical user interface for exploring data stored in SummarizedExperiment objects, including row- and column-level metadata. The interface supports transmission of selections between plots and tables, code tracking, interactive tours, interactive or programmatic initialization, preservation of app state, and extensibility to new panel types via S4 classes. Special attention is given to single-cell data in a SingleCellExperiment object with visualization of dimensionality reduction results.\n",
      "Version: 2.10.0\n",
      "Publication date: None\n",
      "Mantainer: Kevin Rue-Albrecht <kevinrue67@gmail.com>\n",
      "Authors:\n",
      "Author: Kevin Rue-Albrecht [aut, cre] , Federico Marini [aut] , Charlotte Soneson [aut] , Aaron Lun [aut]\n",
      "Dependencies:\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - SingleCellExperiment, type: depends\n",
      "  - methods, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - utils, type: imports\n",
      "  - stats, type: imports\n",
      "  - shiny, type: imports\n",
      "  - shinydashboard, type: imports\n",
      "  - shinyAce, type: imports\n",
      "  - shinyjs, type: imports\n",
      "  - DT, type: imports\n",
      "  - rintrojs, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - ggrepel, type: imports\n",
      "  - colourpicker, type: imports\n",
      "  - igraph, type: imports\n",
      "  - vipor, type: imports\n",
      "  - mgcv, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - viridisLite, type: imports\n",
      "  - shinyWidgets, type: imports\n",
      "  - ComplexHeatmap, type: imports\n",
      "  - circlize, type: imports\n",
      "  - grid, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/iSEE.html\n",
      "\n",
      "Name: maSigPro\n",
      "Description:\n",
      "maSigPro is a regression based approach to find genes for which there are significant gene expression profile differences between experimental groups in time course microarray and RNA-Seq experiments.\n",
      "Version: 1.70.0\n",
      "Publication date: None\n",
      "Mantainer: Maria Jose Nueda <mj.nueda@ua.es>\n",
      "Authors:\n",
      "Author: Ana Conesa and Maria Jose Nueda\n",
      "Dependencies:\n",
      "  - R (>= 2.3.1) type: depends\n",
      "  - Biobase, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - venn, type: imports\n",
      "  - mclust, type: imports\n",
      "  - stats, type: imports\n",
      "  - MASS, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/maSigPro.html\n",
      "\n",
      "Name: cmapR\n",
      "Description:\n",
      "The Connectivity Map (CMap) is a massive resource of perturbational gene expression profiles built by researchers at the Broad Institute and funded by the NIH Library of Integrated Network-Based Cellular Signatures (LINCS) program. Please visit https://clue.io for more information. The cmapR package implements methods to parse, manipulate, and write common CMap data objects, such as annotated matrices and collections of gene sets.\n",
      "Version: 1.10.0\n",
      "Publication date: None\n",
      "Mantainer: Ted Natoli <ted.e.natoli@gmail.com>\n",
      "Authors:\n",
      "Author: Ted Natoli [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - methods, type: imports\n",
      "  - rhdf5, type: imports\n",
      "  - data.table, type: imports\n",
      "  - flowCore, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - matrixStats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/cmapR.html\n",
      "\n",
      "Name: SGSeq\n",
      "Description:\n",
      "SGSeq is a software package for analyzing splice events from RNA-seq data. Input data are RNA-seq reads mapped to a reference genome in BAM format. Genes are represented as a splice graph, which can be obtained from existing annotation or predicted from the mapped sequence reads. Splice events are identified from the graph and are quantified locally using structurally compatible reads at the start or end of each splice variant. The software includes functions for splice event prediction, quantification, visualization and interpretation.\n",
      "Version: 1.32.0\n",
      "Publication date: None\n",
      "Mantainer: Leonard Goldstein <ldgoldstein@gmail.com>\n",
      "Authors:\n",
      "Author: Leonard Goldstein [cre, aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - IRanges, type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - Rsamtools, type: depends\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - methods, type: depends\n",
      "  - AnnotationDbi, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - GenomicAlignments, type: imports\n",
      "  - GenomicFeatures, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - RUnit, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - igraph, type: imports\n",
      "  - parallel, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - stats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/SGSeq.html\n",
      "\n",
      "Name: GeneOverlap\n",
      "Description:\n",
      "Test two sets of gene lists and visualize the results.\n",
      "Version: 1.34.0\n",
      "Publication date: None\n",
      "Mantainer: Ant<c3><b3>nio Miguel de Jesus Domingues, Max-Planck Institute for Cell Biology and Genetics <amjdomingues@gmail.com>\n",
      "Authors:\n",
      "Author: Li Shen, Icahn School of Medicine at Mount Sinai <shenli.sam at gmail.com>\n",
      "Dependencies:\n",
      "  - stats, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - gplots, type: imports\n",
      "  - methods, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/GeneOverlap.html\n",
      "\n",
      "Name: RankProd\n",
      "Description:\n",
      "Non-parametric method for identifying differentially expressed (up- or down- regulated) genes based on the estimated percentage of false predictions (pfp). The method can combine data sets from different origins (meta-analysis) to increase the power of the identification.\n",
      "Version: 3.24.0\n",
      "Publication date: None\n",
      "Mantainer: Francesco Del Carratore <francescodc87@gmail.com>\n",
      "Authors:\n",
      "Author: Francesco Del Carratore <francesco.delcarratore at manchester.ac.uk>, Andris Jankevics <andris.jankevics at gmail.com> Fangxin Hong <fxhong at jimmy.harvard.edu>, Ben Wittner <Wittner.Ben at mgh.harvard.edu>, Rainer Breitling <rainer.breitling at manchester.ac.uk>, and Florian Battke <battke at informatik.uni-tuebingen.de>\n",
      "Dependencies:\n",
      "  - R (>= 3.2.1) type: depends\n",
      "  - stats, type: depends\n",
      "  - methods, type: depends\n",
      "  - Rmpfr, type: depends\n",
      "  - gmp, type: depends\n",
      "  - graphics, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/RankProd.html\n",
      "\n",
      "Name: recount3\n",
      "Description:\n",
      "The recount3 package enables access to a large amount of uniformly processed RNA-seq data from human and mouse. You can download RangedSummarizedExperiment objects at the gene, exon or exon-exon junctions level with sample metadata and QC statistics. In addition we provide access to sample coverage BigWig files.\n",
      "Version: 1.8.0\n",
      "Publication date: None\n",
      "Mantainer: Leonardo Collado-Torres <lcolladotor@gmail.com>\n",
      "Authors:\n",
      "Author: Leonardo Collado-Torres [aut, cre]\n",
      "Dependencies:\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - BiocFileCache, type: imports\n",
      "  - methods, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - utils, type: imports\n",
      "  - RCurl, type: imports\n",
      "  - data.table, type: imports\n",
      "  - R.utils, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - sessioninfo, type: imports\n",
      "  - tools, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/recount3.html\n",
      "\n",
      "Name: soGGi\n",
      "Description:\n",
      "The soGGi package provides a toolset to create genomic interval aggregate/summary plots of signal or motif occurence from BAM and bigWig files as well as PWM, rlelist, GRanges and GAlignments Bioconductor objects. soGGi allows for normalisation, transformation and arithmetic operation on and between summary plot objects as well as grouping and subsetting of plots by GRanges objects and user supplied metadata. Plots are created using the GGplot2 libary to allow user defined manipulation of the returned plot object. Coupled together, soGGi features a broad set of methods to visualise genomics data in the context of groups of genomic intervals such as genes, superenhancers and transcription factor binding events.\n",
      "Version: 1.30.0\n",
      "Publication date: None\n",
      "Mantainer: Tom Carroll <tc.infomatics@gmail.com>\n",
      "Authors:\n",
      "Author: Gopuraja Dharmalingam, Doug Barrows, Tom Carroll\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - BiocGenerics, type: depends\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - methods, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - GenomicAlignments, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - preprocessCore, type: imports\n",
      "  - chipseq, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 3)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/soGGi.html\n",
      "\n",
      "Name: scRepertoire\n",
      "Description:\n",
      "scRepertoire was built to process data derived from the 10x Genomics Chromium Immune Profiling for both T-cell receptor (TCR) and immunoglobulin (Ig) enrichment workflows and subsequently interacts with the popular Seurat and SingleCellExperiment R packages. It also allows for general analysis of single-cell clonotype information without the use of expression information. The package functions as a wrapper for Startrac and powerTCR R packages.\n",
      "Version: 1.8.0\n",
      "Publication date: None\n",
      "Mantainer: Nick Borcherding <ncborch@gmail.com>\n",
      "Authors:\n",
      "Author: Nick Borcherding [aut, cre]\n",
      "Dependencies:\n",
      "  - ggplot2, type: depends\n",
      "  - R (>= 4.0) type: depends\n",
      "  - stringdist, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - ggalluvial, type: imports\n",
      "  - stringr, type: imports\n",
      "  - vegan, type: imports\n",
      "  - powerTCR, type: imports\n",
      "  - SingleCellExperiment, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - plyr, type: imports\n",
      "  - parallel, type: imports\n",
      "  - doParallel, type: imports\n",
      "  - methods, type: imports\n",
      "  - utils, type: imports\n",
      "  - rlang, type: imports\n",
      "  - igraph, type: imports\n",
      "  - ggraph, type: imports\n",
      "  - tidygraph, type: imports\n",
      "  - SeuratObject, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/scRepertoire.html\n",
      "\n",
      "Name: cummeRbund\n",
      "Description:\n",
      "Allows for persistent storage, access, exploration, and manipulation of Cufflinks high-throughput sequencing data. In addition, provides numerous plotting functions for commonly used visualizations.\n",
      "Version: 2.40.0\n",
      "Publication date: None\n",
      "Mantainer: Loyal A. Goff <lgoff@csail.mit.edu>\n",
      "Authors:\n",
      "Author: L. Goff, C. Trapnell, D. Kelley\n",
      "Dependencies:\n",
      "  - R (>= 2.7.0) type: depends\n",
      "  - BiocGenerics, type: depends\n",
      "  - RSQLite, type: depends\n",
      "  - ggplot2, type: depends\n",
      "  - reshape2, type: depends\n",
      "  - fastcluster, type: depends\n",
      "  - rtracklayer, type: depends\n",
      "  - Gviz, type: depends\n",
      "  - methods, type: imports\n",
      "  - plyr, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - Biobase, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/cummeRbund.html\n",
      "\n",
      "Name: cqn\n",
      "Description:\n",
      "A normalization tool for RNA-Seq data, implementing the conditional quantile normalization method.\n",
      "Version: 1.44.0\n",
      "Publication date: None\n",
      "Mantainer: Kasper Daniel Hansen <kasperdanielhansen@gmail.com>\n",
      "Authors:\n",
      "Author: Jean (Zhijin) Wu, Kasper Daniel Hansen\n",
      "Dependencies:\n",
      "  - R (>= 2.10.0) type: depends\n",
      "  - mclust, type: depends\n",
      "  - nor1mix, type: depends\n",
      "  - stats, type: depends\n",
      "  - preprocessCore, type: depends\n",
      "  - splines, type: depends\n",
      "  - quantreg, type: depends\n",
      "  - splines, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/cqn.html\n",
      "\n",
      "Name: ELMER\n",
      "Description:\n",
      "ELMER is designed to use DNA methylation and gene expression from a large number of samples to infere regulatory element landscape and transcription factor network in primary tissue.\n",
      "Version: 2.22.0\n",
      "Publication date: None\n",
      "Mantainer: Tiago Chedraoui Silva <tiagochst@usp.br>\n",
      "Authors:\n",
      "Author: Tiago Chedraoui Silva [aut, cre], Lijing Yao [aut], Simon Coetzee [aut], Nicole Gull [ctb], Hui Shen [ctb], Peter Laird [ctb], Peggy Farnham [aut], Dechen Li [ctb], Benjamin Berman [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.4.0) type: depends\n",
      "  - ELMER.data, type: depends\n",
      "  - GenomicRanges, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - reshape, type: imports\n",
      "  - grid, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - methods, type: imports\n",
      "  - parallel, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - GenomicFeatures, type: imports\n",
      "  - TCGAbiolinks, type: imports\n",
      "  - plyr, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - Gviz, type: imports\n",
      "  - ComplexHeatmap, type: imports\n",
      "  - circlize, type: imports\n",
      "  - MultiAssayExperiment, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - biomaRt, type: imports\n",
      "  - doParallel, type: imports\n",
      "  - downloader, type: imports\n",
      "  - ggrepel, type: imports\n",
      "  - lattice, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - readr, type: imports\n",
      "  - scales, type: imports\n",
      "  - rvest, type: imports\n",
      "  - xml2, type: imports\n",
      "  - plotly, type: imports\n",
      "  - gridExtra, type: imports\n",
      "  - rmarkdown, type: imports\n",
      "  - stringr, type: imports\n",
      "  - tibble, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - progress, type: imports\n",
      "  - purrr, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - ggpubr, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - DelayedArray, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ELMER.html\n",
      "\n",
      "Name: BioNet\n",
      "Description:\n",
      "This package provides functions for the integrated analysis of protein-protein interaction networks and the detection of functional modules. Different datasets can be integrated into the network by assigning p-values of statistical tests to the nodes of the network. E.g. p-values obtained from the differential expression of the genes from an Affymetrix array are assigned to the nodes of the network. By fitting a beta-uniform mixture model and calculating scores from the p-values, overall scores of network regions can be calculated and an integer linear programming algorithm identifies the maximum scoring subnetwork.\n",
      "Version: 1.58.0\n",
      "Publication date: None\n",
      "Mantainer: Marcus Dittrich <marcus.dittrich@biozentrum.uni-wuerzburg.de>\n",
      "Authors:\n",
      "Author: Marcus Dittrich and Daniela Beisser\n",
      "Dependencies:\n",
      "  - R (>= 2.10.0) type: depends\n",
      "  - graph, type: depends\n",
      "  - RBGL, type: depends\n",
      "  - igraph (>= 1.0.1) type: imports\n",
      "  - AnnotationDbi, type: imports\n",
      "  - Biobase, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/BioNet.html\n",
      "\n",
      "Name: heatmaps\n",
      "Description:\n",
      "This package provides functions for plotting heatmaps of genome-wide data across genomic intervals, such as ChIP-seq signals at peaks or across promoters. Many functions are also provided for investigating sequence features.\n",
      "Version: 1.22.0\n",
      "Publication date: None\n",
      "Mantainer: Malcolm Perry <mgperry32@gmail.com>\n",
      "Authors:\n",
      "Author: Malcolm Perry <mgperry32 at gmail.com>\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - methods, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - KernSmooth, type: imports\n",
      "  - plotrix, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - EBImage, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/heatmaps.html\n",
      "\n",
      "Name: GLAD\n",
      "Description:\n",
      "Analysis of array CGH data : detection of breakpoints in genomic profiles and assignment of a status (gain, normal or loss) to each chromosomal regions identified.\n",
      "Version: 2.62.0\n",
      "Publication date: None\n",
      "Mantainer: Philippe Hupe <glad@curie.fr>\n",
      "Authors:\n",
      "Author: Philippe Hupe\n",
      "Dependencies:\n",
      "  - R (>= 2.10) type: depends\n",
      "  - aws, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/GLAD.html\n",
      "\n",
      "Name: made4\n",
      "Description:\n",
      "Multivariate data analysis and graphical display of microarray data. Functions include for supervised dimension reduction (between group analysis) and joint dimension reduction of 2 datasets (coinertia analysis). It contains functions that require R package ade4.\n",
      "Version: 1.72.0\n",
      "Publication date: None\n",
      "Mantainer: Aedin Culhane <Aedin@jimmy.harvard.edu>\n",
      "Authors:\n",
      "Author: Aedin Culhane\n",
      "Dependencies:\n",
      "  - RColorBrewer, type: depends\n",
      "  - gplots, type: depends\n",
      "  - scatterplot3d, type: depends\n",
      "  - Biobase, type: depends\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - ade4, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/made4.html\n",
      "\n",
      "Name: pvca\n",
      "Description:\n",
      "This package contains the function to assess the batch sourcs by fitting all \"sources\" as random effects including two-way interaction terms in the Mixed Model(depends on lme4 package) to selected principal components, which were obtained from the original data correlation matrix. This package accompanies the book \"Batch Effects and Noise in Microarray Experiements, chapter 12.\n",
      "Version: 1.38.0\n",
      "Publication date: None\n",
      "Mantainer: Jianying LI <li11@niehs.nih.gov>\n",
      "Authors:\n",
      "Author: Pierre Bushel <bushel at niehs.nih.gov>\n",
      "Dependencies:\n",
      "  - R (>= 2.15.1) type: depends\n",
      "  - Matrix, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - vsn, type: imports\n",
      "  - stats, type: imports\n",
      "  - lme4, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL (>= 2.0)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/pvca.html\n",
      "\n",
      "Name: EpiDISH\n",
      "Description:\n",
      "EpiDISH is a R package to infer the proportions of a priori known cell-types present in a sample representing a mixture of such cell-types. Right now, the package can be used on DNAm data of whole blood, generic epithelial tissue and breast tissue. Besides, the package provides a function that allows the identification of differentially methylated cell-types and their directionality of change in Epigenome-Wide Association Studies.\n",
      "Version: 2.14.1\n",
      "Publication date: None\n",
      "Mantainer: Shijie C. Zheng <shijieczheng@gmail.com>\n",
      "Authors:\n",
      "Author: Andrew E. Teschendorff [aut], Shijie C. Zheng [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - MASS, type: imports\n",
      "  - e1071, type: imports\n",
      "  - quadprog, type: imports\n",
      "  - parallel, type: imports\n",
      "  - stats, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - stringr, type: imports\n",
      "  - locfdr, type: imports\n",
      "  - Matrix, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/EpiDISH.html\n",
      "\n",
      "Name: IsoformSwitchAnalyzeR\n",
      "Description:\n",
      "Analysis of alternative splicing and isoform switches with predicted functional consequences (e.g. gain/loss of protein domains etc.) from quantification of all types of RNASeq by tools such as Kallisto, Salmon, StringTie, Cufflinks/Cuffdiff etc.\n",
      "Version: 1.20.0\n",
      "Publication date: None\n",
      "Mantainer: Kristoffer Vitting-Seerup <k.vitting.seerup@gmail.com>\n",
      "Authors:\n",
      "Author: Kristoffer Vitting-Seerup [cre, aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.6) type: depends\n",
      "  - limma, type: depends\n",
      "  - DEXSeq, type: depends\n",
      "  - ggplot2, type: depends\n",
      "  - methods, type: imports\n",
      "  - BSgenome, type: imports\n",
      "  - plyr, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - gridExtra, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - DRIMSeq, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - VennDiagram, type: imports\n",
      "  - DBI, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - grid, type: imports\n",
      "  - tximport, type: imports\n",
      "  - tximeta, type: imports\n",
      "  - edgeR, type: imports\n",
      "  - futile.logger, type: imports\n",
      "  - stringr, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - readr, type: imports\n",
      "  - tibble, type: imports\n",
      "  - XVector, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - RCurl, type: imports\n",
      "  - Biobase, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/IsoformSwitchAnalyzeR.html\n",
      "\n",
      "Name: annaffy\n",
      "Description:\n",
      "Functions for handling data from Bioconductor Affymetrix annotation data packages. Produces compact HTML and text reports including experimental data and URL links to many online databases. Allows searching biological metadata using various criteria.\n",
      "Version: 1.70.0\n",
      "Publication date: None\n",
      "Mantainer: Colin A. Smith <colin@colinsmith.org>\n",
      "Authors:\n",
      "Author: Colin A. Smith <colin at colinsmith.org>\n",
      "Dependencies:\n",
      "  - R (>= 2.5.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - Biobase, type: depends\n",
      "  - BiocManager, type: depends\n",
      "  - GO.db, type: depends\n",
      "  - AnnotationDbi, type: imports\n",
      "  - DBI, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/annaffy.html\n",
      "\n",
      "Name: diffcyt\n",
      "Description:\n",
      "Statistical methods for differential discovery analyses in high-dimensional cytometry data (including flow cytometry, mass cytometry or CyTOF, and oligonucleotide-tagged cytometry), based on a combination of high-resolution clustering and empirical Bayes moderated tests adapted from transcriptomics.\n",
      "Version: 1.18.0\n",
      "Publication date: None\n",
      "Mantainer: Lukas M. Weber <lukas.weber.edu@gmail.com>\n",
      "Authors:\n",
      "Author: Lukas M. Weber [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 3.4.0) type: depends\n",
      "  - flowCore, type: imports\n",
      "  - FlowSOM, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - limma, type: imports\n",
      "  - edgeR, type: imports\n",
      "  - lme4, type: imports\n",
      "  - multcomp, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - stats, type: imports\n",
      "  - methods, type: imports\n",
      "  - utils, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - ComplexHeatmap, type: imports\n",
      "  - circlize, type: imports\n",
      "  - grid, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/diffcyt.html\n",
      "\n",
      "Name: hpar\n",
      "Description:\n",
      "The hpar package provides a simple R interface to and data from the Human Protein Atlas project.\n",
      "Version: 1.40.0\n",
      "Publication date: None\n",
      "Mantainer: Laurent Gatto <laurent.gatto@uclouvain.be>\n",
      "Authors:\n",
      "Author: Laurent Gatto [cre, aut] , Manon Martin [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/hpar.html\n",
      "\n",
      "Name: HMMcopy\n",
      "Description:\n",
      "Corrects GC and mappability biases for readcounts (i.e. coverage) in non-overlapping windows of fixed length for single whole genome samples, yielding a rough estimate of copy number for furthur analysis. Designed for rapid correction of high coverage whole genome tumour and normal samples.\n",
      "Version: 1.40.0\n",
      "Publication date: None\n",
      "Mantainer: Daniel Lai <dalai@bccrc.ca>\n",
      "Authors:\n",
      "Author: Daniel Lai, Gavin Ha, Sohrab Shah\n",
      "Dependencies:\n",
      "  - R (>= 2.10.0) type: depends\n",
      "  - data.table (>= 1.11.8) type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/HMMcopy.html\n",
      "\n",
      "Name: pRoloc\n",
      "Description:\n",
      "The pRoloc package implements machine learning and visualisation methods for the analysis and interogation of quantitiative mass spectrometry data to reliably infer protein sub-cellular localisation.\n",
      "Version: 1.38.2\n",
      "Publication date: None\n",
      "Mantainer: Laurent Gatto <laurent.gatto@uclouvain.be>\n",
      "Authors:\n",
      "Author: Laurent Gatto, Oliver Crook and Lisa M. Breckels with contributions from Thomas Burger and Samuel Wieczorek\n",
      "Dependencies:\n",
      "  - R (>= 3.5) type: depends\n",
      "  - MSnbase, type: depends\n",
      "  - MLInterfaces, type: depends\n",
      "  - methods, type: depends\n",
      "  - Rcpp (>= 0.10.3) type: depends\n",
      "  - BiocParallel, type: depends\n",
      "  - stats4, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - mclust (>= 4.3) type: imports\n",
      "  - caret, type: imports\n",
      "  - e1071, type: imports\n",
      "  - sampling, type: imports\n",
      "  - class, type: imports\n",
      "  - kernlab, type: imports\n",
      "  - lattice, type: imports\n",
      "  - nnet, type: imports\n",
      "  - randomForest, type: imports\n",
      "  - proxy, type: imports\n",
      "  - FNN, type: imports\n",
      "  - hexbin, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - stats, type: imports\n",
      "  - dendextend, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - scales, type: imports\n",
      "  - MASS, type: imports\n",
      "  - knitr, type: imports\n",
      "  - mvtnorm, type: imports\n",
      "  - LaplacesDemon, type: imports\n",
      "  - coda, type: imports\n",
      "  - mixtools, type: imports\n",
      "  - gtools, type: imports\n",
      "  - plyr, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - biomaRt, type: imports\n",
      "  - utils, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/pRoloc.html\n",
      "\n",
      "Name: ReadqPCR\n",
      "Description:\n",
      "The package provides functions to read raw RT-qPCR data of different platforms.\n",
      "Version: 1.44.0\n",
      "Publication date: None\n",
      "Mantainer: James Perkins <jimrperkins@gmail.com>\n",
      "Authors:\n",
      "Author: James Perkins, Matthias Kohl, Nor Izayu Abdul Rahman\n",
      "Dependencies:\n",
      "  - R (>= 2.14.0) type: depends\n",
      "  - Biobase, type: depends\n",
      "  - methods, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ReadqPCR.html\n",
      "\n",
      "Name: InteractiveComplexHeatmap\n",
      "Description:\n",
      "This package can easily make heatmaps which are produced by the ComplexHeatmap package into interactive applications. It provides two types of interactivities: 1. on the interactive graphics device, and 2. on a Shiny app. It also provides functions for integrating the interactive heatmap widgets for more complex Shiny app development.\n",
      "Version: 1.6.0\n",
      "Publication date: None\n",
      "Mantainer: Zuguang Gu <z.gu@dkfz.de>\n",
      "Authors:\n",
      "Author: Zuguang Gu [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.0.0) type: depends\n",
      "  - ComplexHeatmap, type: depends\n",
      "  - grDevices, type: imports\n",
      "  - stats, type: imports\n",
      "  - shiny, type: imports\n",
      "  - grid, type: imports\n",
      "  - GetoptLong, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - digest, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - kableExtra (>= 1.3.1) type: imports\n",
      "  - utils, type: imports\n",
      "  - svglite, type: imports\n",
      "  - htmltools, type: imports\n",
      "  - clisymbols, type: imports\n",
      "  - jsonlite, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - fontawesome, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/InteractiveComplexHeatmap.html\n",
      "\n",
      "Name: exomeCopy\n",
      "Description:\n",
      "Detection of copy number variants (CNV) from exome sequencing samples, including unpaired samples. The package implements a hidden Markov model which uses positional covariates, such as background read depth and GC-content, to simultaneously normalize and segment the samples into regions of constant copy count.\n",
      "Version: 1.44.0\n",
      "Publication date: None\n",
      "Mantainer: Michael Love <michaelisaiahlove@gmail.com>\n",
      "Authors:\n",
      "Author: Michael Love\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - IRanges, type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - Rsamtools, type: depends\n",
      "  - stats4, type: imports\n",
      "  - methods, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/exomeCopy.html\n",
      "\n",
      "Name: RedeR\n",
      "Description:\n",
      "RedeR is an R-based package combined with a stand-alone Java application for interactive visualization and manipulation of nested networks.\n",
      "Version: 2.2.1\n",
      "Publication date: None\n",
      "Mantainer: Mauro Castro <mauro.a.castro@gmail.com>\n",
      "Authors:\n",
      "Author: Mauro Castro, Xin Wang, Florian Markowetz\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - igraph, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 3)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/RedeR.html\n",
      "\n",
      "Name: escape\n",
      "Description:\n",
      "A bridging R package to facilitate gene set enrichment analysis (GSEA) in the context of single-cell RNA sequencing. Using raw count information, Seurat objects, or SingleCellExperiment format, users can perform and visualize GSEA across individual cells.\n",
      "Version: 1.8.0\n",
      "Publication date: None\n",
      "Mantainer: Nick Borcherding <ncborch@gmail.com>\n",
      "Authors:\n",
      "Author: Nick Borcherding [aut, cre], Jared Andrews [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - grDevices, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - GSEABase, type: imports\n",
      "  - GSVA, type: imports\n",
      "  - SingleCellExperiment, type: imports\n",
      "  - ggridges, type: imports\n",
      "  - msigdbr, type: imports\n",
      "  - stats, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - UCell, type: imports\n",
      "  - broom, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - patchwork, type: imports\n",
      "  - MatrixGenerics, type: imports\n",
      "  - utils, type: imports\n",
      "  - rlang, type: imports\n",
      "  - stringr, type: imports\n",
      "  - data.table, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - methods, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/escape.html\n",
      "\n",
      "Name: AnnotationHubData\n",
      "Description:\n",
      "These recipes convert a wide variety and a growing number of public bioinformatic data sets into easily-used standard Bioconductor data structures.\n",
      "Version: 1.28.0\n",
      "Publication date: None\n",
      "Mantainer: Bioconductor Package Maintainer <maintainer@bioconductor.org>\n",
      "Authors:\n",
      "Author: Martin Morgan [ctb], Marc Carlson [ctb], Dan Tenenbaum [ctb], Sonali Arora [ctb], Paul Shannon [ctb], Lori Shepherd [ctb], Bioconductor Package Maintainer [cre]\n",
      "Dependencies:\n",
      "  - R (>= 3.2.2) type: depends\n",
      "  - methods, type: depends\n",
      "  - utils, type: depends\n",
      "  - S4Vectors, type: depends\n",
      "  - IRanges, type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - AnnotationHub, type: depends\n",
      "  - GenomicFeatures, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - jsonlite, type: imports\n",
      "  - BiocManager, type: imports\n",
      "  - biocViews, type: imports\n",
      "  - BiocCheck, type: imports\n",
      "  - graph, type: imports\n",
      "  - AnnotationDbi, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - DBI, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - OrganismDbi, type: imports\n",
      "  - RSQLite, type: imports\n",
      "  - AnnotationForge, type: imports\n",
      "  - futile.logger (>= 1.3.0) type: imports\n",
      "  - XML, type: imports\n",
      "  - RCurl, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/AnnotationHubData.html\n",
      "\n",
      "Name: DEFormats\n",
      "Description:\n",
      "Convert between different data formats used by differential gene expression analysis tools.\n",
      "Version: 1.26.0\n",
      "Publication date: None\n",
      "Mantainer: Andrzej OleÅ <andrzej.oles@embl.de>\n",
      "Authors:\n",
      "Author: Andrzej OleÅ\n",
      "Dependencies:\n",
      "  - checkmate, type: imports\n",
      "  - data.table, type: imports\n",
      "  - DESeq2, type: imports\n",
      "  - edgeR, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - methods, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - stats, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/DEFormats.html\n",
      "\n",
      "Name: microbiomeMarker\n",
      "Description:\n",
      "To date, a number of methods have been developed for microbiome marker discovery based on metagenomic profiles, e.g. LEfSe. However, all of these methods have its own advantages and disadvantages, and none of them is considered standard or universal. Moreover, different programs or softwares may be development using different programming languages, even in different operating systems. Here, we have developed an all-in-one R package microbiomeMarker that integrates commonly used differential analysis methods as well as three machine learning-based approaches, including Logistic regression, Random forest, and Support vector machine, to facilitate the identification of microbiome markers.\n",
      "Version: 1.4.0\n",
      "Publication date: None\n",
      "Mantainer: Yang Cao <caoyang.name@gmail.com>\n",
      "Authors:\n",
      "Author: Yang Cao [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.1.0) type: depends\n",
      "  - dplyr, type: imports\n",
      "  - phyloseq, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - purrr, type: imports\n",
      "  - MASS, type: imports\n",
      "  - utils, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - tibble, type: imports\n",
      "  - rlang, type: imports\n",
      "  - stats, type: imports\n",
      "  - coin, type: imports\n",
      "  - ggtree, type: imports\n",
      "  - tidytree, type: imports\n",
      "  - methods, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - patchwork, type: imports\n",
      "  - ggsignif, type: imports\n",
      "  - metagenomeSeq, type: imports\n",
      "  - DESeq2, type: imports\n",
      "  - edgeR, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - yaml, type: imports\n",
      "  - biomformat, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - ComplexHeatmap, type: imports\n",
      "  - ANCOMBC, type: imports\n",
      "  - caret, type: imports\n",
      "  - limma, type: imports\n",
      "  - ALDEx2, type: imports\n",
      "  - multtest, type: imports\n",
      "  - plotROC, type: imports\n",
      "  - vegan, type: imports\n",
      "  - pROC, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/microbiomeMarker.html\n",
      "\n",
      "Name: NanoStringNCTools\n",
      "Description:\n",
      "Tools for NanoString Technologies nCounter Technology. Provides support for reading RCC files into an ExpressionSet derived object. Also includes methods for QC and normalizaztion of NanoString data.\n",
      "Version: 1.6.1\n",
      "Publication date: None\n",
      "Mantainer: Nicole Ortogero <nortogero@nanostring.com>\n",
      "Authors:\n",
      "Author: Patrick Aboyoun [aut], Nicole Ortogero [cre], Zhi Yang [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 3.6) type: depends\n",
      "  - Biobase, type: depends\n",
      "  - S4Vectors, type: depends\n",
      "  - ggplot2, type: depends\n",
      "  - BiocGenerics, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - ggbeeswarm, type: imports\n",
      "  - ggiraph, type: imports\n",
      "  - ggthemes, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - methods, type: imports\n",
      "  - pheatmap, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/NanoStringNCTools.html\n",
      "\n",
      "Name: GeomxTools\n",
      "Description:\n",
      "Tools for NanoString Technologies GeoMx Technology. Package provides functions for reading in DCC and PKC files based on an ExpressionSet derived object. Normalization and QC functions are also included.\n",
      "Version: 3.2.0\n",
      "Publication date: None\n",
      "Mantainer: Nicole Ortogero <nortogero@nanostring.com>\n",
      "Authors:\n",
      "Author: Nicole Ortogero [cre, aut], Zhi Yang [aut], Ronalyn Vitancol [aut], Maddy Griswold [aut], David Henderson [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.6) type: depends\n",
      "  - Biobase, type: depends\n",
      "  - NanoStringNCTools, type: depends\n",
      "  - S4Vectors, type: depends\n",
      "  - BiocGenerics, type: imports\n",
      "  - rjson, type: imports\n",
      "  - readxl, type: imports\n",
      "  - EnvStats, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - methods, type: imports\n",
      "  - utils, type: imports\n",
      "  - stats, type: imports\n",
      "  - data.table, type: imports\n",
      "  - lmerTest, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - stringr, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - GGally, type: imports\n",
      "  - rlang, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - SeuratObject, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/GeomxTools.html\n",
      "\n",
      "Name: BUSpaRse\n",
      "Description:\n",
      "The kallisto | bustools pipeline is a fast and modular set of tools to convert single cell RNA-seq reads in fastq files into gene count or transcript compatibility counts (TCC) matrices for downstream analysis. Central to this pipeline is the barcode, UMI, and set (BUS) file format. This package serves the following purposes: First, this package allows users to manipulate BUS format files as data frames in R and then convert them into gene count or TCC matrices. Furthermore, since R and Rcpp code is easier to handle than pure C++ code, users are encouraged to tweak the source code of this package to experiment with new uses of BUS format and different ways to convert the BUS file into gene count matrix. Second, this package can conveniently generate files required to generate gene count matrices for spliced and unspliced transcripts for RNA velocity. Here biotypes can be filtered and scaffolds and haplotypes can be removed, and the filtered transcriptome can be extracted and written to disk. Third, this package implements utility functions to get transcripts and associated genes required to convert BUS files to gene count matrices, to write the transcript to gene information in the format required by bustools, and to read output of bustools into R as sparses matrices.\n",
      "Version: 1.12.2\n",
      "Publication date: None\n",
      "Mantainer: Lambda Moses <dlu2@caltech.edu>\n",
      "Authors:\n",
      "Author: Lambda Moses [aut, cre] , Lior Pachter [aut, ths]\n",
      "Dependencies:\n",
      "  - R (>= 3.6) type: depends\n",
      "  - AnnotationDbi, type: imports\n",
      "  - AnnotationFilter, type: imports\n",
      "  - biomaRt, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - BSgenome, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - ensembldb, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - GenomicFeatures, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - methods, type: imports\n",
      "  - plyranges, type: imports\n",
      "  - Rcpp, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - stats, type: imports\n",
      "  - stringr, type: imports\n",
      "  - tibble, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - utils, type: imports\n",
      "  - zeallot, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "BSD_2_clause + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/BUSpaRse.html\n",
      "\n",
      "Name: multiMiR\n",
      "Description:\n",
      "A collection of microRNAs/targets from external resources, including validated microRNA-target databases (miRecords, miRTarBase and TarBase), predicted microRNA-target databases (DIANA-microT, ElMMo, MicroCosm, miRanda, miRDB, PicTar, PITA and TargetScan) and microRNA-disease/drug databases (miR2Disease, Pharmaco-miR VerSe and PhenomiR).\n",
      "Version: 1.20.0\n",
      "Publication date: None\n",
      "Mantainer: Matt Mulvahill <matt.mulvahill@gmail.com>\n",
      "Authors:\n",
      "Author: Yuanbin Ru [aut], Matt Mulvahill [cre, aut], Spencer Mahaffey [aut], Katerina Kechris [aut, cph, ths]\n",
      "Dependencies:\n",
      "  - R (>= 3.4) type: depends\n",
      "  - stats, type: imports\n",
      "  - XML, type: imports\n",
      "  - RCurl, type: imports\n",
      "  - purrr (>= 0.2.2) type: imports\n",
      "  - tibble (>= 1.2) type: imports\n",
      "  - methods, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - AnnotationDbi, type: imports\n",
      "  - dplyr, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/multiMiR.html\n",
      "\n",
      "Name: NormqPCR\n",
      "Description:\n",
      "Functions for the selection of optimal reference genes and the normalisation of real-time quantitative PCR data.\n",
      "Version: 1.44.0\n",
      "Publication date: None\n",
      "Mantainer: James Perkins <jimrperkins@gmail.com>\n",
      "Authors:\n",
      "Author: Matthias Kohl, James Perkins, Nor Izayu Abdul Rahman\n",
      "Dependencies:\n",
      "  - R (>= 2.14.0) type: depends\n",
      "  - stats, type: depends\n",
      "  - RColorBrewer, type: depends\n",
      "  - Biobase, type: depends\n",
      "  - methods, type: depends\n",
      "  - ReadqPCR, type: depends\n",
      "  - qpcR, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/NormqPCR.html\n",
      "\n",
      "Name: bambu\n",
      "Description:\n",
      "bambu is a R package for multi-sample transcript discovery and quantification using long read RNA-Seq data. You can use bambu after read alignment to obtain expression estimates for known and novel transcripts and genes. The output from bambu can directly be used for visualisation and downstream analysis such as differential gene expression or transcript usage.\n",
      "Version: 3.0.8\n",
      "Publication date: None\n",
      "Mantainer: Ying Chen <chen_ying@gis.a-star.edu.sg>\n",
      "Authors:\n",
      "Author: Ying Chen [cre, aut], Andre Sim [aut], Yuk Kei Wan [aut], Jonathan Goeke [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - S4Vectors, type: depends\n",
      "  - BSgenome, type: depends\n",
      "  - IRanges, type: depends\n",
      "  - BiocGenerics, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - data.table, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - GenomicAlignments, type: imports\n",
      "  - GenomicFeatures, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - stats, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - methods, type: imports\n",
      "  - Rcpp, type: imports\n",
      "  - xgboost, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3 + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/bambu.html\n",
      "\n",
      "Name: bioDist\n",
      "Description:\n",
      "A collection of software tools for calculating distance measures.\n",
      "Version: 1.70.0\n",
      "Publication date: None\n",
      "Mantainer: Bioconductor Package Maintainer <maintainer@bioconductor.org>\n",
      "Authors:\n",
      "Author: B. Ding, R. Gentleman and Vincent Carey\n",
      "Dependencies:\n",
      "  - R (>= 2.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - Biobase, type: depends\n",
      "  - KernSmooth, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/bioDist.html\n",
      "\n",
      "Name: OmicCircos\n",
      "Description:\n",
      "OmicCircos is an R application and package for generating high-quality circular plots for omics data.\n",
      "Version: 1.36.0\n",
      "Publication date: None\n",
      "Mantainer: Ying Hu <yhu@mail.nih.gov>\n",
      "Authors:\n",
      "Author: Ying Hu <yhu at mail.nih.gov> Chunhua Yan <yanch at mail.nih.gov>\n",
      "Dependencies:\n",
      "  - R (>= 2.14.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/OmicCircos.html\n",
      "\n",
      "Name: muscat\n",
      "Description:\n",
      "`muscat` provides various methods and visualization tools for DS analysis in multi-sample, multi-group, multi-(cell-)subpopulation scRNA-seq data, including cell-level mixed models and methods based on aggregated âpseudobulkâ data, as well as a flexible simulation platform that mimics both single and multi-sample scRNA-seq data.\n",
      "Version: 1.12.1\n",
      "Publication date: None\n",
      "Mantainer: Helena L. Crowell <helena@crowell.eu>\n",
      "Authors:\n",
      "Author: Helena L. Crowell [aut, cre], Pierre-Luc Germain [aut], Charlotte Soneson [aut], Anthony Sonrel [aut], Mark D. Robinson [aut, fnd]\n",
      "Dependencies:\n",
      "  - R (>= 4.2) type: depends\n",
      "  - BiocParallel, type: imports\n",
      "  - blme, type: imports\n",
      "  - ComplexHeatmap, type: imports\n",
      "  - data.table, type: imports\n",
      "  - DESeq2, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - edgeR, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - glmmTMB, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - grid, type: imports\n",
      "  - limma, type: imports\n",
      "  - lmerTest, type: imports\n",
      "  - lme4, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - methods, type: imports\n",
      "  - progress, type: imports\n",
      "  - purrr, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - scales, type: imports\n",
      "  - scater, type: imports\n",
      "  - scuttle, type: imports\n",
      "  - sctransform, type: imports\n",
      "  - stats, type: imports\n",
      "  - SingleCellExperiment, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - variancePartition, type: imports\n",
      "  - viridis, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/muscat.html\n",
      "\n",
      "Name: hopach\n",
      "Description:\n",
      "The HOPACH clustering algorithm builds a hierarchical tree of clusters by recursively partitioning a data set, while ordering and possibly collapsing clusters at each level. The algorithm uses the Mean/Median Split Silhouette (MSS) criteria to identify the level of the tree with maximally homogeneous clusters. It also runs the tree down to produce a final ordered list of the elements. The non-parametric bootstrap allows one to estimate the probability that each element belongs to each cluster (fuzzy clustering).\n",
      "Version: 2.58.0\n",
      "Publication date: None\n",
      "Mantainer: Katherine S. Pollard <katherine.pollard@gladstone.ucsf.edu>\n",
      "Authors:\n",
      "Author: Katherine S. Pollard, with Mark J. van der Laan <laan at stat.berkeley.edu> and Greg Wall\n",
      "Dependencies:\n",
      "  - R (>= 2.11.0) type: depends\n",
      "  - cluster, type: depends\n",
      "  - Biobase, type: depends\n",
      "  - methods, type: depends\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/hopach.html\n",
      "\n",
      "Name: CancerSubtypes\n",
      "Description:\n",
      "CancerSubtypes integrates the current common computational biology methods for cancer subtypes identification and provides a standardized framework for cancer subtype analysis based multi-omics data, such as gene expression, miRNA expression, DNA methylation and others.\n",
      "Version: 1.24.0\n",
      "Publication date: None\n",
      "Mantainer: Taosheng Xu <taosheng.x@gmail.com>\n",
      "Authors:\n",
      "Author: Taosheng Xu [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - sigclust, type: depends\n",
      "  - NMF, type: depends\n",
      "  - cluster, type: imports\n",
      "  - impute, type: imports\n",
      "  - limma, type: imports\n",
      "  - ConsensusClusterPlus, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - survival, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/CancerSubtypes.html\n",
      "\n",
      "Name: GenomicInteractions\n",
      "Description:\n",
      "Utilities for handling genomic interaction data such as ChIA-PET or Hi-C, annotating genomic features with interaction information, and producing plots and summary statistics.\n",
      "Version: 1.32.0\n",
      "Publication date: None\n",
      "Mantainer: Liz Ing-Simmons <liz.ingsimmons@gmail.com>\n",
      "Authors:\n",
      "Author: Harmston, N., Ing-Simmons, E., Perry, M., Baresic, A., Lenhard, B.\n",
      "Dependencies:\n",
      "  - R (>= 3.5) type: depends\n",
      "  - InteractionSet, type: depends\n",
      "  - Rsamtools, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - data.table, type: imports\n",
      "  - stringr, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - grid, type: imports\n",
      "  - gridExtra, type: imports\n",
      "  - methods, type: imports\n",
      "  - igraph, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - Gviz, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - grDevices, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/GenomicInteractions.html\n",
      "\n",
      "Name: MungeSumstats\n",
      "Description:\n",
      "The *MungeSumstats* package is designed to facilitate the standardisation of GWAS summary statistics. It reformats inputted summary statisitics to include SNP, CHR, BP and can look up these values if any are missing. It also pefrorms dozens of QC and filtering steps to ensure high data quality and minimise inter-study differences.\n",
      "Version: 1.6.0\n",
      "Publication date: None\n",
      "Mantainer: Alan Murphy <alanmurph94@hotmail.com>\n",
      "Authors:\n",
      "Author: Alan Murphy [aut, cre] , Brian Schilder [aut, ctb] , Nathan Skene [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - magrittr, type: imports\n",
      "  - data.table, type: imports\n",
      "  - utils, type: imports\n",
      "  - R.utils, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - stats, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - BSgenome, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - stringr, type: imports\n",
      "  - VariantAnnotation, type: imports\n",
      "  - googleAuthR, type: imports\n",
      "  - httr, type: imports\n",
      "  - jsonlite, type: imports\n",
      "  - methods, type: imports\n",
      "  - parallel, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - RCurl, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/MungeSumstats.html\n",
      "\n",
      "Name: RnBeads\n",
      "Description:\n",
      "RnBeads facilitates comprehensive analysis of various types of DNA methylation data at the genome scale.\n",
      "Version: 2.16.0\n",
      "Publication date: None\n",
      "Mantainer: Fabian Mueller <team@rnbeads.org>\n",
      "Authors:\n",
      "Author: Yassen Assenov [aut], Christoph Bock [aut], Pavlo Lutsik [aut], Michael Scherer [aut], Fabian Mueller [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 3.0.0) type: depends\n",
      "  - BiocGenerics, type: depends\n",
      "  - S4Vectors, type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - MASS, type: depends\n",
      "  - cluster, type: depends\n",
      "  - ff, type: depends\n",
      "  - fields, type: depends\n",
      "  - ggplot2 (>= 0.9.2) type: depends\n",
      "  - gplots, type: depends\n",
      "  - grid, type: depends\n",
      "  - gridExtra, type: depends\n",
      "  - limma, type: depends\n",
      "  - matrixStats, type: depends\n",
      "  - methods, type: depends\n",
      "  - illuminaio, type: depends\n",
      "  - methylumi, type: depends\n",
      "  - plyr, type: depends\n",
      "  - IRanges, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/RnBeads.html\n",
      "\n",
      "Name: flowClean\n",
      "Description:\n",
      "A quality control tool for flow cytometry data based on compositional data analysis.\n",
      "Version: 1.36.0\n",
      "Publication date: None\n",
      "Mantainer: Kipper Fletez-Brant <cafletezbrant@gmail.com>\n",
      "Authors:\n",
      "Author: Kipper Fletez-Brant\n",
      "Dependencies:\n",
      "  - R (>= 2.15.0) type: depends\n",
      "  - flowCore, type: depends\n",
      "  - bit, type: imports\n",
      "  - changepoint, type: imports\n",
      "  - sfsmisc, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/flowClean.html\n",
      "\n",
      "Name: lefser\n",
      "Description:\n",
      "lefser is an implementation in R of the popular \"LDA Effect Size (LEfSe)\" method for microbiome biomarker discovery. It uses the Kruskal-Wallis test, Wilcoxon-Rank Sum test, and Linear Discriminant Analysis to find biomarkers of groups and sub-groups.\n",
      "Version: 1.8.0\n",
      "Publication date: None\n",
      "Mantainer: Asya Khleborodova <asya.bioconductor@gmail.com>\n",
      "Authors:\n",
      "Author: Asya Khleborodova [cre, aut], Ludwig Geistlinger [ctb], Marcel Ramos [ctb] , Levi Waldron [ctb]\n",
      "Dependencies:\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - R (>= 4.0.0) type: depends\n",
      "  - coin, type: imports\n",
      "  - MASS, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - stats, type: imports\n",
      "  - methods, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/lefser.html\n",
      "\n",
      "Name: ChemmineOB\n",
      "Description:\n",
      "ChemmineOB provides an R interface to a subset of cheminformatics functionalities implemented by the OpelBabel C++ project. OpenBabel is an open source cheminformatics toolbox that includes utilities for structure format interconversions, descriptor calculations, compound similarity searching and more. ChemineOB aims to make a subset of these utilities available from within R. For non-developers, ChemineOB is primarily intended to be used from ChemmineR as an add-on package rather than used directly.\n",
      "Version: 1.36.0\n",
      "Publication date: None\n",
      "Mantainer: Thomas Girke <thomas.girke@ucr.edu>\n",
      "Authors:\n",
      "Author: Kevin Horan, Thomas Girke\n",
      "Dependencies:\n",
      "  - R (>= 2.15.1) type: depends\n",
      "  - methods, type: depends\n",
      "  - BiocGenerics, type: imports\n",
      "  - zlibbioc, type: imports\n",
      "  - Rcpp (>= 0.11.0) type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ChemmineOB.html\n",
      "\n",
      "Name: Rqc\n",
      "Description:\n",
      "Rqc is an optimised tool designed for quality control and assessment of high-throughput sequencing data. It performs parallel processing of entire files and produces a report which contains a set of high-resolution graphics.\n",
      "Version: 1.32.0\n",
      "Publication date: None\n",
      "Mantainer: Welliton Souza <well309@gmail.com>\n",
      "Authors:\n",
      "Author: Welliton Souza, Benilton Carvalho <beniltoncarvalho at gmail.com>\n",
      "Dependencies:\n",
      "  - BiocParallel, type: depends\n",
      "  - ShortRead, type: depends\n",
      "  - ggplot2, type: depends\n",
      "  - BiocGenerics, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - methods, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - knitr (>= 1.7) type: imports\n",
      "  - BiocStyle, type: imports\n",
      "  - plyr, type: imports\n",
      "  - markdown, type: imports\n",
      "  - grid, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - Rcpp (>= 0.11.6) type: imports\n",
      "  - biovizBase, type: imports\n",
      "  - shiny, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - GenomicAlignments, type: imports\n",
      "  - GenomicFiles, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/Rqc.html\n",
      "\n",
      "Name: DEGseq\n",
      "Description:\n",
      "DEGseq is an R package to identify differentially expressed genes from RNA-Seq data.\n",
      "Version: 1.52.0\n",
      "Publication date: None\n",
      "Mantainer: Likun Wang <wanglk@hsc.pku.edu.cn>\n",
      "Authors:\n",
      "Author: Likun Wang <wanglk at hsc.pku.edu.cn> and Xi Wang <wang-xi05 at mails.tsinghua.edu.cn>.\n",
      "Dependencies:\n",
      "  - R (>= 2.8.0) type: depends\n",
      "  - qvalue, type: depends\n",
      "  - methods, type: depends\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL (>=2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/DEGseq.html\n",
      "\n",
      "Name: muscle\n",
      "Description:\n",
      "MUSCLE performs multiple sequence alignments of nucleotide or amino acid sequences.\n",
      "Version: 3.40.0\n",
      "Publication date: None\n",
      "Mantainer: Alex T. Kalinka <alex.t.kalinka@gmail.com>\n",
      "Authors:\n",
      "Author: Algorithm by Robert C. Edgar. R port by Alex T. Kalinka.\n",
      "Dependencies:\n",
      "  - Biostrings, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Unlimited\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/muscle.html\n",
      "\n",
      "Name: DEqMS\n",
      "Description:\n",
      "DEqMS is developped on top of Limma. However, Limma assumes same prior variance for all genes. In proteomics, the accuracy of protein abundance estimates varies by the number of peptides/PSMs quantified in both label-free and labelled data. Proteins quantification by multiple peptides or PSMs are more accurate. DEqMS package is able to estimate different prior variances for proteins quantified by different number of PSMs/peptides, therefore acchieving better accuracy. The package can be applied to analyze both label-free and labelled proteomics data.\n",
      "Version: 1.16.0\n",
      "Publication date: None\n",
      "Mantainer: Yafeng Zhu <yafeng.zhu@outlook.com>\n",
      "Authors:\n",
      "Author: Yafeng Zhu\n",
      "Dependencies:\n",
      "  - R (>= 3.5) type: depends\n",
      "  - graphics, type: depends\n",
      "  - stats, type: depends\n",
      "  - ggplot2, type: depends\n",
      "  - matrixStats, type: depends\n",
      "  - limma, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/DEqMS.html\n",
      "\n",
      "Name: fmcsR\n",
      "Description:\n",
      "The fmcsR package introduces an efficient maximum common substructure (MCS) algorithms combined with a novel matching strategy that allows for atom and/or bond mismatches in the substructures shared among two small molecules. The resulting flexible MCSs (FMCSs) are often larger than strict MCSs, resulting in the identification of more common features in their source structures, as well as a higher sensitivity in finding compounds with weak structural similarities. The fmcsR package provides several utilities to use the FMCS algorithm for pairwise compound comparisons, structure similarity searching and clustering.\n",
      "Version: 1.40.0\n",
      "Publication date: None\n",
      "Mantainer: Thomas Girke <thomas.girke@ucr.edu>\n",
      "Authors:\n",
      "Author: Yan Wang, Tyler Backman, Kevin Horan, Thomas Girke\n",
      "Dependencies:\n",
      "  - R (>= 2.10.0) type: depends\n",
      "  - ChemmineR, type: depends\n",
      "  - methods, type: depends\n",
      "  - RUnit, type: imports\n",
      "  - methods, type: imports\n",
      "  - ChemmineR, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - parallel, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/fmcsR.html\n",
      "\n",
      "Name: MeSHDbi\n",
      "Description:\n",
      "The package is unified implementation of MeSH.db, MeSH.AOR.db, and MeSH.PCR.db and also is interface to construct Gene-MeSH package (MeSH.XXX.eg.db). loadMeSHDbiPkg import sqlite file and generate MeSH.XXX.eg.db.\n",
      "Version: 1.34.0\n",
      "Publication date: None\n",
      "Mantainer: Koki Tsuyuzaki <k.t.the-answer@hotmail.co.jp>\n",
      "Authors:\n",
      "Author: Koki Tsuyuzaki\n",
      "Dependencies:\n",
      "  - R (>= 3.0.1) type: depends\n",
      "  - methods, type: imports\n",
      "  - AnnotationDbi, type: imports\n",
      "  - RSQLite, type: imports\n",
      "  - Biobase, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/MeSHDbi.html\n",
      "\n",
      "Name: ReactomeGSA\n",
      "Description:\n",
      "The ReactomeGSA packages uses Reactome's online analysis service to perform a multi-omics gene set analysis. The main advantage of this package is, that the retrieved results can be visualized using REACTOME's powerful webapplication. Since Reactome's analysis service also uses R to perfrom the actual gene set analysis you will get similar results when using the same packages (such as limma and edgeR) locally. Therefore, if you only require a gene set analysis, different packages are more suited.\n",
      "Version: 1.12.0\n",
      "Publication date: None\n",
      "Mantainer: Johannes Griss <johannes.griss@meduniwien.ac.at>\n",
      "Authors:\n",
      "Author: Johannes Griss [aut, cre]\n",
      "Dependencies:\n",
      "  - jsonlite, type: imports\n",
      "  - httr, type: imports\n",
      "  - progress, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - methods, type: imports\n",
      "  - gplots, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - tidyr, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ReactomeGSA.html\n",
      "\n",
      "Name: GEOmetadb\n",
      "Description:\n",
      "The NCBI Gene Expression Omnibus (GEO) represents the largest public repository of microarray data. However, finding data of interest can be challenging using current tools. GEOmetadb is an attempt to make access to the metadata associated with samples, platforms, and datasets much more feasible. This is accomplished by parsing all the NCBI GEO metadata into a SQLite database that can be stored and queried locally. GEOmetadb is simply a thin wrapper around the SQLite database along with associated documentation. Finally, the SQLite database is updated regularly as new data is added to GEO and can be downloaded at will for the most up-to-date metadata. GEOmetadb paper: http://bioinformatics.oxfordjournals.org/cgi/content/short/24/23/2798 .\n",
      "Version: 1.60.0\n",
      "Publication date: None\n",
      "Mantainer: Jack Zhu <zhujack@mail.nih.gov>\n",
      "Authors:\n",
      "Author: Jack Zhu and Sean Davis\n",
      "Dependencies:\n",
      "  - GEOquery, type: depends\n",
      "  - RSQLite, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/GEOmetadb.html\n",
      "\n",
      "Name: flowDensity\n",
      "Description:\n",
      "This package provides tools for automated sequential gating analogous to the manual gating strategy based on the density of the data.\n",
      "Version: 1.32.0\n",
      "Publication date: None\n",
      "Mantainer: Mehrnoush Malek <mmalekes@bccrc.ca>\n",
      "Authors:\n",
      "Author: Mehrnoush Malek,M. Jafar Taghiyar\n",
      "Dependencies:\n",
      "  - flowCore, type: imports\n",
      "  - graphics, type: imports\n",
      "  - flowViz, type: imports\n",
      "  - car, type: imports\n",
      "  - sp, type: imports\n",
      "  - rgeos, type: imports\n",
      "  - gplots, type: imports\n",
      "  - RFOC, type: imports\n",
      "  - flowWorkspace, type: imports\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - grDevices, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/flowDensity.html\n",
      "\n",
      "Name: MsBackendMgf\n",
      "Description:\n",
      "Mass spectrometry (MS) data backend supporting import and export of MS/MS spectra data from Mascot Generic Format (mgf) files. Objects defined in this package are supposed to be used with the Spectra Bioconductor package. This package thus adds mgf file support to the Spectra package.\n",
      "Version: 1.6.0\n",
      "Publication date: None\n",
      "Mantainer: RforMassSpectrometry Package Maintainer <maintainer@rformassspectrometry.org>\n",
      "Authors:\n",
      "Author: RforMassSpectrometry Package Maintainer [cre], Laurent Gatto [aut] , Johannes Rainer [aut] , Sebastian Gibb [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - Spectra, type: depends\n",
      "  - BiocParallel, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - MsCoreUtils, type: imports\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/MsBackendMgf.html\n",
      "\n",
      "Name: BiocSet\n",
      "Description:\n",
      "BiocSet displays different biological sets in a triple tibble format. These three tibbles are `element`, `set`, and `elementset`. The user has the abilty to activate one of these three tibbles to perform common functions from the dplyr package. Mapping functionality and accessing web references for elements/sets are also available in BiocSet.\n",
      "Version: 1.12.1\n",
      "Publication date: None\n",
      "Mantainer: Kayla Morrell <kayla.morrell@roswellpark.org>\n",
      "Authors:\n",
      "Author: Kayla Morrell [aut, cre], Martin Morgan [aut], Kevin Rue-Albrecht [ctb], LluÃ­s Revilla Sancho [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 3.6) type: depends\n",
      "  - dplyr, type: depends\n",
      "  - methods, type: imports\n",
      "  - tibble, type: imports\n",
      "  - utils, type: imports\n",
      "  - rlang, type: imports\n",
      "  - plyr, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - BiocIO, type: imports\n",
      "  - AnnotationDbi, type: imports\n",
      "  - KEGGREST, type: imports\n",
      "  - ontologyIndex, type: imports\n",
      "  - tidyr, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/BiocSet.html\n",
      "\n",
      "Name: GOfuncR\n",
      "Description:\n",
      "GOfuncR performs a gene ontology enrichment analysis based on the ontology enrichment software FUNC. GO-annotations are obtained from OrganismDb or OrgDb packages ('Homo.sapiens' by default); the GO-graph is included in the package and updated regularly (01-May-2021). GOfuncR provides the standard candidate vs. background enrichment analysis using the hypergeometric test, as well as three additional tests: (i) the Wilcoxon rank-sum test that is used when genes are ranked, (ii) a binomial test that is used when genes are associated with two counts and (iii) a Chi-square or Fisher's exact test that is used in cases when genes are associated with four counts. To correct for multiple testing and interdependency of the tests, family-wise error rates are computed based on random permutations of the gene-associated variables. GOfuncR also provides tools for exploring the ontology graph and the annotations, and options to take gene-length or spatial clustering of genes into account. It is also possible to provide custom gene coordinates, annotations and ontologies.\n",
      "Version: 1.18.0\n",
      "Publication date: None\n",
      "Mantainer: Steffi Grote <grote.steffi@gmail.com>\n",
      "Authors:\n",
      "Author: Steffi Grote\n",
      "Dependencies:\n",
      "  - R (>= 3.4) type: depends\n",
      "  - vioplot (>= 0.2) type: depends\n",
      "  - Rcpp (>= 0.11.5) type: imports\n",
      "  - mapplots (>= 1.5) type: imports\n",
      "  - gtools (>= 3.5.0) type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - AnnotationDbi, type: imports\n",
      "  - utils, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/GOfuncR.html\n",
      "\n",
      "Name: quantro\n",
      "Description:\n",
      "A data-driven test for the assumptions of quantile normalization using raw data such as objects that inherit eSets (e.g. ExpressionSet, MethylSet). Group level information about each sample (such as Tumor / Normal status) must also be provided because the test assesses if there are global differences in the distributions between the user-defined groups.\n",
      "Version: 1.32.0\n",
      "Publication date: None\n",
      "Mantainer: Stephanie Hicks <shicks19@jhu.edu>\n",
      "Authors:\n",
      "Author: Stephanie Hicks [aut, cre] , Rafael Irizarry [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - Biobase, type: imports\n",
      "  - minfi, type: imports\n",
      "  - doParallel, type: imports\n",
      "  - foreach, type: imports\n",
      "  - iterators, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - methods, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>=3)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/quantro.html\n",
      "\n",
      "Name: SIMLR\n",
      "Description:\n",
      "Single-cell RNA-seq technologies enable high throughput gene expression measurement of individual cells, and allow the discovery of heterogeneity within cell populations. Measurement of cell-to-cell gene expression similarity is critical for the identification, visualization and analysis of cell populations. However, single-cell data introduce challenges to conventional measures of gene expression similarity because of the high level of noise, outliers and dropouts. We develop a novel similarity-learning framework, SIMLR (Single-cell Interpretation via Multi-kernel LeaRning), which learns an appropriate distance metric from the data for dimension reduction, clustering and visualization.\n",
      "Version: 1.24.3\n",
      "Publication date: None\n",
      "Mantainer: Luca De Sano <luca.desano@gmail.com>\n",
      "Authors:\n",
      "Author: Daniele Ramazzotti [aut] , Bo Wang [aut], Luca De Sano [cre, aut] , Serafim Batzoglou [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 4.1.0) type: depends\n",
      "  - parallel, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - stats, type: imports\n",
      "  - methods, type: imports\n",
      "  - Rcpp, type: imports\n",
      "  - pracma, type: imports\n",
      "  - RcppAnnoy, type: imports\n",
      "  - RSpectra, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/SIMLR.html\n",
      "\n",
      "Name: philr\n",
      "Description:\n",
      "PhILR is short for Phylogenetic Isometric Log-Ratio Transform. This package provides functions for the analysis of compositional data (e.g., data representing proportions of different variables/parts). Specifically this package allows analysis of compositional data where the parts can be related through a phylogenetic tree (as is common in microbiota survey data) and makes available the Isometric Log Ratio transform built from the phylogenetic tree and utilizing a weighted reference measure.\n",
      "Version: 1.24.0\n",
      "Publication date: None\n",
      "Mantainer: Justin Silverman <jsilve24@gmail.com>\n",
      "Authors:\n",
      "Author: Justin Silverman [aut, cre], Leo Lahti [ctb]\n",
      "Dependencies:\n",
      "  - ape, type: imports\n",
      "  - phangorn, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - ggtree, type: imports\n",
      "  - methods, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/philr.html\n",
      "\n",
      "Name: cicero\n",
      "Description:\n",
      "Cicero computes putative cis-regulatory maps from single-cell chromatin accessibility data. It also extends monocle 2 for use in chromatin accessibility data.\n",
      "Version: 1.16.0\n",
      "Publication date: None\n",
      "Mantainer: Hannah Pliner <hpliner@uw.edu>\n",
      "Authors:\n",
      "Author: Hannah Pliner [aut, cre], Cole Trapnell [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - monocle, type: depends\n",
      "  - Gviz, type: depends\n",
      "  - assertthat (>= 0.2.0) type: imports\n",
      "  - Biobase, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - data.table (>= 1.10.4) type: imports\n",
      "  - dplyr (>= 0.7.4) type: imports\n",
      "  - FNN (>= 1.1) type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - ggplot2 (>= 2.2.1) type: imports\n",
      "  - glasso (>= 1.8) type: imports\n",
      "  - grDevices, type: imports\n",
      "  - igraph (>= 1.1.0) type: imports\n",
      "  - IRanges, type: imports\n",
      "  - Matrix (>= 1.2-12) type: imports\n",
      "  - methods, type: imports\n",
      "  - parallel, type: imports\n",
      "  - plyr (>= 1.8.4) type: imports\n",
      "  - reshape2 (>= 1.4.3) type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - stats, type: imports\n",
      "  - stringi, type: imports\n",
      "  - stringr (>= 1.2.0) type: imports\n",
      "  - tibble (>= 1.4.2) type: imports\n",
      "  - tidyr, type: imports\n",
      "  - VGAM (>= 1.0-5) type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/cicero.html\n",
      "\n",
      "Name: cn.mops\n",
      "Description:\n",
      "cn.mops (Copy Number estimation by a Mixture Of PoissonS) is a data processing pipeline for copy number variations and aberrations (CNVs and CNAs) from next generation sequencing (NGS) data. The package supplies functions to convert BAM files into read count matrices or genomic ranges objects, which are the input objects for cn.mops. cn.mops models the depths of coverage across samples at each genomic position. Therefore, it does not suffer from read count biases along chromosomes. Using a Bayesian approach, cn.mops decomposes read variations across samples into integer copy numbers and noise by its mixture components and Poisson distributions, respectively. cn.mops guarantees a low FDR because wrong detections are indicated by high noise and filtered out. cn.mops is very fast and written in C++.\n",
      "Version: 1.44.0\n",
      "Publication date: None\n",
      "Mantainer: Gundula Povysil <povysil@bioinf.jku.at>\n",
      "Authors:\n",
      "Author: Guenter Klambauer [aut], Gundula Povysil [cre]\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - utils, type: depends\n",
      "  - stats, type: depends\n",
      "  - graphics, type: depends\n",
      "  - parallel, type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - BiocGenerics, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - exomeCopy, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL (>= 2.0)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/cn.mops.html\n",
      "\n",
      "Name: MSstatsTMT\n",
      "Description:\n",
      "The package provides statistical tools for detecting differentially abundant proteins in shotgun mass spectrometry-based proteomic experiments with tandem mass tag (TMT) labeling. It provides multiple functionalities, including aata visualization, protein quantification and normalization, and statistical modeling and inference. Furthermore, it is inter-operable with other data processing tools, such as Proteome Discoverer, MaxQuant, OpenMS and SpectroMine.\n",
      "Version: 2.6.0\n",
      "Publication date: None\n",
      "Mantainer: Ting Huang <thuang0703@gmail.com>\n",
      "Authors:\n",
      "Author: Ting Huang [aut, cre], Meena Choi [aut], Mateusz Staniak [aut], Sicheng Hao [aut], Olga Vitek [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.2) type: depends\n",
      "  - limma, type: imports\n",
      "  - lme4, type: imports\n",
      "  - lmerTest, type: imports\n",
      "  - methods, type: imports\n",
      "  - data.table, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - MSstats, type: imports\n",
      "  - MSstatsConvert, type: imports\n",
      "  - checkmate, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/MSstatsTMT.html\n",
      "\n",
      "Name: PADOG\n",
      "Description:\n",
      "This package implements a general purpose gene set analysis method called PADOG that downplays the importance of genes that apear often accross the sets of genes to be analyzed. The package provides also a benchmark for gene set analysis methods in terms of sensitivity and ranking using 24 public datasets from KEGGdzPathwaysGEO package.\n",
      "Version: 1.40.0\n",
      "Publication date: None\n",
      "Mantainer: Adi L. Tarca <atarca@med.wayne.edu>\n",
      "Authors:\n",
      "Author: Adi Laurentiu Tarca <atarca at med.wayne.edu>; Zhonghui Xu <zhonghui.xu at gmail.com>\n",
      "Dependencies:\n",
      "  - R (>= 3.0.0) type: depends\n",
      "  - KEGGdzPathwaysGEO, type: depends\n",
      "  - methods, type: depends\n",
      "  - Biobase, type: depends\n",
      "  - limma, type: imports\n",
      "  - AnnotationDbi, type: imports\n",
      "  - GSA, type: imports\n",
      "  - foreach, type: imports\n",
      "  - doRNG, type: imports\n",
      "  - hgu133plus2.db, type: imports\n",
      "  - hgu133a.db, type: imports\n",
      "  - KEGGREST, type: imports\n",
      "  - nlme, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/PADOG.html\n",
      "\n",
      "Name: Rhisat2\n",
      "Description:\n",
      "An R interface to the HISAT2 spliced short-read aligner by Kim et al. (2015). The package contains wrapper functions to create a genome index and to perform the read alignment to the generated index.\n",
      "Version: 1.14.0\n",
      "Publication date: None\n",
      "Mantainer: Charlotte Soneson <charlottesoneson@gmail.com>\n",
      "Authors:\n",
      "Author: Charlotte Soneson [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.2) type: depends\n",
      "  - GenomicFeatures, type: imports\n",
      "  - SGSeq, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - methods, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/Rhisat2.html\n",
      "\n",
      "Name: CoreGx\n",
      "Description:\n",
      "A collection of functions and classes which serve as the foundation for our lab's suite of R packages, such as 'PharmacoGx' and 'RadioGx'. This package was created to abstract shared functionality from other lab package releases to increase ease of maintainability and reduce code repetition in current and future 'Gx' suite programs. Major features include a 'CoreSet' class, from which 'RadioSet' and 'PharmacoSet' are derived, along with get and set methods for each respective slot. Additional functions related to fitting and plotting dose response curves, quantifying statistical correlation and calculating area under the curve (AUC) or survival fraction (SF) are included. For more details please see the included documentation, as well as: Smirnov, P., Safikhani, Z., El-Hachem, N., Wang, D., She, A., Olsen, C., Freeman, M., Selby, H., Gendoo, D., Grossman, P., Beck, A., Aerts, H., Lupien, M., Goldenberg, A. (2015) . Manem, V., Labie, M., Smirnov, P., Kofia, V., Freeman, M., Koritzinksy, M., Abazeed, M., Haibe-Kains, B., Bratman, S. (2018) .\n",
      "Version: 2.2.0\n",
      "Publication date: None\n",
      "Mantainer: Benjamin Haibe-Kains <benjamin.haibe.kains@utoronto.ca>\n",
      "Authors:\n",
      "Author: Petr Smirnov [aut], Ian Smith [aut], Christopher Eeles [aut], Feifei Li [aut], Benjamin Haibe-Kains [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - BiocGenerics, type: depends\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - Biobase, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - MultiAssayExperiment, type: imports\n",
      "  - MatrixGenerics, type: imports\n",
      "  - piano, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - parallel, type: imports\n",
      "  - BumpyMatrix, type: imports\n",
      "  - checkmate, type: imports\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - lsa, type: imports\n",
      "  - data.table, type: imports\n",
      "  - crayon, type: imports\n",
      "  - glue, type: imports\n",
      "  - rlang, type: imports\n",
      "  - bench, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 3)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/CoreGx.html\n",
      "\n",
      "Name: genbankr\n",
      "Description:\n",
      "Reads Genbank files.\n",
      "Version: 1.26.0\n",
      "Publication date: None\n",
      "Mantainer: Gabriel Becker <becker.gabriel@gene.com>\n",
      "Authors:\n",
      "Author: Gabriel Becker [aut, cre], Michael Lawrence [aut]\n",
      "Dependencies:\n",
      "  - methods, type: depends\n",
      "  - BiocGenerics, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - GenomicFeatures, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - VariantAnnotation, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - Biobase, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/genbankr.html\n",
      "\n",
      "Name: biobroom\n",
      "Description:\n",
      "This package contains methods for converting standard objects constructed by bioinformatics packages, especially those in Bioconductor, and converting them to tidy data. It thus serves as a complement to the broom package, and follows the same the tidy, augment, glance division of tidying methods. Tidying data makes it easy to recombine, reshape and visualize bioinformatics analyses.\n",
      "Version: 1.30.0\n",
      "Publication date: None\n",
      "Mantainer: John D. Storey <jstorey@princeton.edu> and Andrew J. Bass <ajbass@princeton.edu>\n",
      "Authors:\n",
      "Author: Andrew J. Bass, David G. Robinson, Steve Lianoglou, Emily Nelson, John D. Storey, with contributions from Laurent Gatto\n",
      "Dependencies:\n",
      "  - R (>= 3.0.0) type: depends\n",
      "  - broom, type: depends\n",
      "  - dplyr, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - Biobase, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/biobroom.html\n",
      "\n",
      "Name: ExperimentHubData\n",
      "Description:\n",
      "Functions to add metadata to ExperimentHub db and resource files to AWS S3 buckets.\n",
      "Version: 1.24.0\n",
      "Publication date: None\n",
      "Mantainer: Bioconductor Package Maintainer <maintainer@bioconductor.org>\n",
      "Authors:\n",
      "Author: Bioconductor Maintainer [cre]\n",
      "Dependencies:\n",
      "  - utils, type: depends\n",
      "  - BiocGenerics, type: depends\n",
      "  - S4Vectors, type: depends\n",
      "  - AnnotationHubData, type: depends\n",
      "  - methods, type: imports\n",
      "  - ExperimentHub, type: imports\n",
      "  - BiocManager, type: imports\n",
      "  - DBI, type: imports\n",
      "  - httr, type: imports\n",
      "  - curl, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ExperimentHubData.html\n",
      "\n",
      "Name: HTSFilter\n",
      "Description:\n",
      "This package implements a filtering procedure for replicated transcriptome sequencing data based on a global Jaccard similarity index in order to identify genes with low, constant levels of expression across one or more experimental conditions.\n",
      "Version: 1.38.0\n",
      "Publication date: None\n",
      "Mantainer: Andrea Rau <andrea.rau@inrae.fr>\n",
      "Authors:\n",
      "Author: Andrea Rau [cre, aut] , Melina Gallopin [ctb], Gilles Celeux [ctb], Florence JaffrÃ©zic [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 4.0.0) type: depends\n",
      "  - edgeR, type: imports\n",
      "  - DESeq2, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - utils, type: imports\n",
      "  - stats, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - methods, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/HTSFilter.html\n",
      "\n",
      "Name: DeconRNASeq\n",
      "Description:\n",
      "DeconSeq is an R package for deconvolution of heterogeneous tissues based on mRNA-Seq data. It modeled expression levels from heterogeneous cell populations in mRNA-Seq as the weighted average of expression from different constituting cell types and predicted cell type proportions of single expression profiles.\n",
      "Version: 1.40.0\n",
      "Publication date: None\n",
      "Mantainer: Ting Gong <tinggong@gmail.com>\n",
      "Authors:\n",
      "Author: Ting Gong <tinggong at gmail.com> Joseph D. Szustakowski <joseph.szustakowski at novartis.com>\n",
      "Dependencies:\n",
      "  - R (>= 2.14.0) type: depends\n",
      "  - limSolve, type: depends\n",
      "  - pcaMethods, type: depends\n",
      "  - ggplot2, type: depends\n",
      "  - grid, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/DeconRNASeq.html\n",
      "\n",
      "Name: cleaver\n",
      "Description:\n",
      "In-silico cleavage of polypeptide sequences. The cleavage rules are taken from: http://web.expasy.org/peptide_cutter/peptidecutter_enzymes.html\n",
      "Version: 1.36.0\n",
      "Publication date: None\n",
      "Mantainer: Sebastian Gibb <mail@sebastiangibb.de>\n",
      "Authors:\n",
      "Author: Sebastian Gibb [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 3.0.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - Biostrings, type: depends\n",
      "  - S4Vectors, type: imports\n",
      "  - IRanges, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 3)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/cleaver.html\n",
      "\n",
      "Name: synergyfinder\n",
      "Description:\n",
      "Efficient implementations for analyzing pre-clinical multiple drug combination datasets. It provides efficient implementations for 1.the popular synergy scoring models, including HSA, Loewe, Bliss, and ZIP to quantify the degree of drug combination synergy; 2. higher order drug combination data analysis and synergy landscape visualization for unlimited number of drugs in a combination; 3. statistical analysis of drug combination synergy and sensitivity with confidence intervals and p-values; 4. synergy barometer for harmonizing multiple synergy scoring methods to provide a consensus metric of synergy; 5. evaluation of synergy and sensitivity simultaneously to provide an unbiased interpretation of the clinical potential of the drug combinations. Based on this package, we also provide a web application (https://synergyfinder.org/) for users who prefer graphical user interface.\n",
      "Version: 3.6.3\n",
      "Publication date: None\n",
      "Mantainer: Shuyu Zheng <shuyu.zheng@helsinki.fi>\n",
      "Authors:\n",
      "Author: Shuyu Zheng [aut, cre], Jing Tang [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.0.0) type: depends\n",
      "  - drc (>= 3.0-1) type: imports\n",
      "  - reshape2 (>= 1.4.4) type: imports\n",
      "  - tidyverse (>= 1.3.0) type: imports\n",
      "  - dplyr (>= 1.0.3) type: imports\n",
      "  - tidyr (>= 1.1.2) type: imports\n",
      "  - purrr (>= 0.3.4) type: imports\n",
      "  - furrr (>= 0.2.2) type: imports\n",
      "  - ggplot2 (>= 3.3.3) type: imports\n",
      "  - ggforce (>= 0.3.2) type: imports\n",
      "  - grid (>= 4.0.2) type: imports\n",
      "  - vegan (>= 2.5-7) type: imports\n",
      "  - gstat (>= 2.0-6) type: imports\n",
      "  - sp (>= 1.4-5) type: imports\n",
      "  - methods (>= 4.0.2) type: imports\n",
      "  - SpatialExtremes (>= 2.0-9) type: imports\n",
      "  - ggrepel (>= 0.9.1) type: imports\n",
      "  - kriging (>= 1.1) type: imports\n",
      "  - plotly (>= 4.9.3) type: imports\n",
      "  - stringr (>= 1.4.0) type: imports\n",
      "  - future (>= 1.21.0) type: imports\n",
      "  - mice (>= 3.13.0) type: imports\n",
      "  - lattice (>= 0.20-41) type: imports\n",
      "  - nleqslv (>= 3.3.2) type: imports\n",
      "  - stats (>= 4.0.2) type: imports\n",
      "  - graphics (>= 4.0.2) type: imports\n",
      "  - grDevices (>= 4.0.2) type: imports\n",
      "  - magrittr (>= 2.0.1) type: imports\n",
      "  - pbapply (>= 1.4-3) type: imports\n",
      "  - metR (>= 0.9.1) type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Mozilla Public License 2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/synergyfinder.html\n",
      "\n",
      "Name: EGSEA\n",
      "Description:\n",
      "This package implements the Ensemble of Gene Set Enrichment Analyses (EGSEA) method for gene set testing.\n",
      "Version: 1.26.0\n",
      "Publication date: None\n",
      "Mantainer: Monther Alhamdoosh <m.hamdoosh@gmail.com>\n",
      "Authors:\n",
      "Author: Monther Alhamdoosh, Luyi Tian, Milica Ng and Matthew Ritchie\n",
      "Dependencies:\n",
      "  - R (>= 3.5) type: depends\n",
      "  - Biobase, type: depends\n",
      "  - gage, type: depends\n",
      "  - AnnotationDbi, type: depends\n",
      "  - topGO, type: depends\n",
      "  - pathview, type: depends\n",
      "  - PADOG, type: imports\n",
      "  - GSVA, type: imports\n",
      "  - globaltest, type: imports\n",
      "  - limma, type: imports\n",
      "  - edgeR, type: imports\n",
      "  - HTMLUtils (>= 0.1.5) type: imports\n",
      "  - hwriter (>= 1.2.2) type: imports\n",
      "  - gplots (>= 2.14.2) type: imports\n",
      "  - ggplot2 (>= 1.0.0) type: imports\n",
      "  - safe, type: imports\n",
      "  - stringi (>= 0.5.0) type: imports\n",
      "  - parallel, type: imports\n",
      "  - stats, type: imports\n",
      "  - metap, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - utils, type: imports\n",
      "  - org.Hs.eg.db, type: imports\n",
      "  - org.Mm.eg.db, type: imports\n",
      "  - org.Rn.eg.db, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - methods, type: imports\n",
      "  - EGSEAdata, type: imports\n",
      "  - htmlwidgets, type: imports\n",
      "  - plotly, type: imports\n",
      "  - DT, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/EGSEA.html\n",
      "\n",
      "Name: ENmix\n",
      "Description:\n",
      "Tool kits for quanlity control, analysis and visulization of Illumina DNA methylation arrays.\n",
      "Version: 1.34.0\n",
      "Publication date: None\n",
      "Mantainer: Zongli Xu <xuz@niehs.nih.gov>\n",
      "Authors:\n",
      "Author: Zongli Xu [cre, aut], Liang Niu [aut], Jack Taylor [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - parallel, type: depends\n",
      "  - doParallel, type: depends\n",
      "  - foreach, type: depends\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - stats, type: depends\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - preprocessCore, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - methods, type: imports\n",
      "  - utils, type: imports\n",
      "  - geneplotter, type: imports\n",
      "  - impute, type: imports\n",
      "  - minfi, type: imports\n",
      "  - RPMM, type: imports\n",
      "  - illuminaio, type: imports\n",
      "  - dynamicTreeCut, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - gtools, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - ExperimentHub, type: imports\n",
      "  - AnnotationHub, type: imports\n",
      "  - genefilter, type: imports\n",
      "  - gplots, type: imports\n",
      "  - quadprog, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ENmix.html\n",
      "\n",
      "Name: cytomapper\n",
      "Description:\n",
      "Highly multiplexed imaging acquires the single-cell expression of selected proteins in a spatially-resolved fashion. These measurements can be visualised across multiple length-scales. First, pixel-level intensities represent the spatial distributions of feature expression with highest resolution. Second, after segmentation, expression values or cell-level metadata (e.g. cell-type information) can be visualised on segmented cell areas. This package contains functions for the visualisation of multiplexed read-outs and cell-level information obtained by multiplexed imaging technologies. The main functions of this package allow 1. the visualisation of pixel-level information across multiple channels, 2. the display of cell-level information (expression and/or metadata) on segmentation masks and 3. gating and visualisation of single cells.\n",
      "Version: 1.10.1\n",
      "Publication date: None\n",
      "Mantainer: Nils Eling <nils.eling@dqbm.uzh.ch>\n",
      "Authors:\n",
      "Author: Nils Eling [aut, cre] , Nicolas Damond [aut] , Tobias Hoch [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - EBImage, type: depends\n",
      "  - SingleCellExperiment, type: depends\n",
      "  - methods, type: depends\n",
      "  - SpatialExperiment, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - HDF5Array, type: imports\n",
      "  - DelayedArray, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - viridis, type: imports\n",
      "  - utils, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - tools, type: imports\n",
      "  - graphics, type: imports\n",
      "  - raster, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - stats, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - ggbeeswarm, type: imports\n",
      "  - svgPanZoom, type: imports\n",
      "  - svglite, type: imports\n",
      "  - shiny, type: imports\n",
      "  - shinydashboard, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - rhdf5, type: imports\n",
      "  - nnls, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/cytomapper.html\n",
      "\n",
      "Name: VennDetail\n",
      "Description:\n",
      "A set of functions to generate high-resolution Venn,Vennpie plot,extract and combine details of these subsets with user datasets in data frame is available.\n",
      "Version: 1.14.0\n",
      "Publication date: None\n",
      "Mantainer: Kai Guo <guokai8@gmail.com>\n",
      "Authors:\n",
      "Author: Kai Guo, Brett McGregor\n",
      "Dependencies:\n",
      "  - utils, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - stats, type: imports\n",
      "  - methods, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - purrr, type: imports\n",
      "  - tibble, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - UpSetR, type: imports\n",
      "  - VennDiagram, type: imports\n",
      "  - grid, type: imports\n",
      "  - futile.logger, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/VennDetail.html\n",
      "\n",
      "Name: MetaboCoreUtils\n",
      "Description:\n",
      "MetaboCoreUtils defines metabolomics-related core functionality provided as low-level functions to allow a data structure-independent usage across various R packages. This includes functions to calculate between ion (adduct) and compound mass-to-charge ratios and masses or functions to work with chemical formulas. The package provides also a set of adduct definitions and information on some commercially available internal standard mixes commonly used in MS experiments.\n",
      "Version: 1.6.0\n",
      "Publication date: None\n",
      "Mantainer: Johannes Rainer <Johannes.Rainer@eurac.edu>\n",
      "Authors:\n",
      "Author: Johannes Rainer [aut, cre] , Michael Witting [aut] , Andrea Vicini [aut], Liesa Salzer [ctb] , Sebastian Gibb [ctb] , Michael Stravs [ctb] , Roger Gine [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - utils, type: imports\n",
      "  - MsCoreUtils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/MetaboCoreUtils.html\n",
      "\n",
      "Name: stageR\n",
      "Description:\n",
      "The stageR package allows automated stage-wise analysis of high-throughput gene expression data. The method is published in Genome Biology at https://genomebiology.biomedcentral.com/articles/10.1186/s13059-017-1277-0\n",
      "Version: 1.20.0\n",
      "Publication date: None\n",
      "Mantainer: Koen Van den Berge <koen.vdberge@gmail.com>\n",
      "Authors:\n",
      "Author: Koen Van den Berge and Lieven Clement\n",
      "Dependencies:\n",
      "  - R (>= 3.4) type: depends\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GNU General Public License version 3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/stageR.html\n",
      "\n",
      "Name: pathifier\n",
      "Description:\n",
      "Pathifier is an algorithm that infers pathway deregulation scores for each tumor sample on the basis of expression data. This score is determined, in a context-specific manner, for every particular dataset and type of cancer that is being investigated. The algorithm transforms gene-level information into pathway-level information, generating a compact and biologically relevant representation of each sample.\n",
      "Version: 1.36.0\n",
      "Publication date: None\n",
      "Mantainer: Assif Yitzhaky <assif.yitzhaky@weizmann.ac.il>\n",
      "Authors:\n",
      "Author: Yotam Drier\n",
      "Dependencies:\n",
      "  - R.oo, type: imports\n",
      "  - princurve (>= 2.0.4) type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-1.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/pathifier.html\n",
      "\n",
      "Name: profileplyr\n",
      "Description:\n",
      "Quick and straightforward visualization of read signal over genomic intervals is key for generating hypotheses from sequencing data sets (e.g. ChIP-seq, ATAC-seq, bisulfite/methyl-seq). Many tools both inside and outside of R and Bioconductor are available to explore these types of data, and they typically start with a bigWig or BAM file and end with some representation of the signal (e.g. heatmap). profileplyr leverages many Bioconductor tools to allow for both flexibility and additional functionality in workflows that end with visualization of the read signal.\n",
      "Version: 1.14.1\n",
      "Publication date: None\n",
      "Mantainer: Tom Carroll <tc.infomatics@gmail.com>, Doug Barrows <doug.barrows@gmail.com>\n",
      "Authors:\n",
      "Author: Tom Carroll and Doug Barrows\n",
      "Dependencies:\n",
      "  - R (>= 3.6) type: depends\n",
      "  - BiocGenerics, type: depends\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - GenomicRanges, type: imports\n",
      "  - stats, type: imports\n",
      "  - soGGi, type: imports\n",
      "  - methods, type: imports\n",
      "  - utils, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - R.utils, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - rjson, type: imports\n",
      "  - ChIPseeker, type: imports\n",
      "  - GenomicFeatures, type: imports\n",
      "  - TxDb.Hsapiens.UCSC.hg19.knownGene, type: imports\n",
      "  - TxDb.Hsapiens.UCSC.hg38.knownGene, type: imports\n",
      "  - TxDb.Mmusculus.UCSC.mm10.knownGene, type: imports\n",
      "  - TxDb.Mmusculus.UCSC.mm9.knownGene, type: imports\n",
      "  - org.Hs.eg.db, type: imports\n",
      "  - org.Mm.eg.db, type: imports\n",
      "  - rGREAT, type: imports\n",
      "  - pheatmap, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - EnrichedHeatmap, type: imports\n",
      "  - ComplexHeatmap, type: imports\n",
      "  - grid, type: imports\n",
      "  - circlize, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - rlang, type: imports\n",
      "  - tiff, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 3)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/profileplyr.html\n",
      "\n",
      "Name: BiocWorkflowTools\n",
      "Description:\n",
      "Provides functions to ease the transition between Rmarkdown and LaTeX documents when authoring a Bioconductor Workflow.\n",
      "Version: 1.24.0\n",
      "Publication date: None\n",
      "Mantainer: Mike Smith <grimbough@gmail.com>\n",
      "Authors:\n",
      "Author: Mike Smith [aut, cre], Andrzej OleÅ [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.4) type: depends\n",
      "  - BiocStyle, type: imports\n",
      "  - bookdown, type: imports\n",
      "  - git2r, type: imports\n",
      "  - httr, type: imports\n",
      "  - knitr, type: imports\n",
      "  - rmarkdown, type: imports\n",
      "  - rstudioapi, type: imports\n",
      "  - stringr, type: imports\n",
      "  - tools, type: imports\n",
      "  - utils, type: imports\n",
      "  - usethis, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/BiocWorkflowTools.html\n",
      "\n",
      "Name: densvis\n",
      "Description:\n",
      "Implements the density-preserving modification to t-SNE and UMAP described by Narayan et al. (2020) . The non-linear dimensionality reduction techniques t-SNE and UMAP enable users to summarise complex high-dimensional sequencing data such as single cell RNAseq using lower dimensional representations. These lower dimensional representations enable the visualisation of discrete transcriptional states, as well as continuous trajectory (for example, in early development). However, these methods focus on the local neighbourhood structure of the data. In some cases, this results in misleading visualisations, where the density of cells in the low-dimensional embedding does not represent the transcriptional heterogeneity of data in the original high-dimensional space. den-SNE and densMAP aim to enable more accurate visual interpretation of high-dimensional datasets by producing lower-dimensional embeddings that accurately represent the heterogeneity of the original high-dimensional space, enabling the identification of homogeneous and heterogeneous cell states. This accuracy is accomplished by including in the optimisation process a term which considers the local density of points in the original high-dimensional space. This can help to create visualisations that are more representative of heterogeneity in the original high-dimensional space.\n",
      "Version: 1.8.3\n",
      "Publication date: None\n",
      "Mantainer: Alan O'Callaghan <alan.ocallaghan@outlook.com>\n",
      "Authors:\n",
      "Author: Alan O'Callaghan [aut, cre], Ashwinn Narayan [aut], Hyunghoon Cho [aut]\n",
      "Dependencies:\n",
      "  - Rcpp, type: imports\n",
      "  - basilisk, type: imports\n",
      "  - assertthat, type: imports\n",
      "  - reticulate, type: imports\n",
      "  - irlba, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/densvis.html\n",
      "\n",
      "Name: LOLA\n",
      "Description:\n",
      "Provides functions for testing overlap of sets of genomic regions with public and custom region set (genomic ranges) databases. This makes it possible to do automated enrichment analysis for genomic region sets, thus facilitating interpretation of functional genomics and epigenomics data.\n",
      "Version: 1.28.0\n",
      "Publication date: None\n",
      "Mantainer: Nathan Sheffield <nathan@code.databio.org>\n",
      "Authors:\n",
      "Author: Nathan Sheffield <http://www.databio.org> [aut, cre], Christoph Bock [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - BiocGenerics, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - data.table, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - utils, type: imports\n",
      "  - stats, type: imports\n",
      "  - methods, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/LOLA.html\n",
      "\n",
      "Name: PharmacoGx\n",
      "Description:\n",
      "Contains a set of functions to perform large-scale analysis of pharmaco-genomic data. These include the PharmacoSet object for storing the results of pharmacogenomic experiments, as well as a number of functions for computing common summaries of drug-dose response and correlating them with the molecular features in a cancer cell-line.\n",
      "Version: 3.2.0\n",
      "Publication date: None\n",
      "Mantainer: Benjamin Haibe-Kains <benjamin.haibe.kains@utoronto.ca>\n",
      "Authors:\n",
      "Author: Petr Smirnov [aut], Christopher Eeles [aut], Zhaleh Safikhani [aut], Mark Freeman [aut], Feifei Li [aut], Benjamin Haibe-Kains [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 3.6) type: depends\n",
      "  - CoreGx, type: depends\n",
      "  - BiocGenerics, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - MultiAssayExperiment, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - magicaxis, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - parallel, type: imports\n",
      "  - caTools, type: imports\n",
      "  - methods, type: imports\n",
      "  - downloader, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - jsonlite, type: imports\n",
      "  - data.table, type: imports\n",
      "  - checkmate, type: imports\n",
      "  - boot, type: imports\n",
      "  - coop, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 3)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/PharmacoGx.html\n",
      "\n",
      "Name: msmsEDA\n",
      "Description:\n",
      "Exploratory data analysis to assess the quality of a set of LC-MS/MS experiments, and visualize de influence of the involved factors.\n",
      "Version: 1.36.0\n",
      "Publication date: None\n",
      "Mantainer: Josep Gregori <josep.gregori@gmail.com>\n",
      "Authors:\n",
      "Author: Josep Gregori, Alex Sanchez, and Josep Villanueva\n",
      "Dependencies:\n",
      "  - R (>= 3.0.1) type: depends\n",
      "  - MSnbase, type: depends\n",
      "  - MASS, type: imports\n",
      "  - gplots, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/msmsEDA.html\n",
      "\n",
      "Name: CEMiTool\n",
      "Description:\n",
      "The CEMiTool package unifies the discovery and the analysis of coexpression gene modules in a fully automatic manner, while providing a user-friendly html report with high quality graphs. Our tool evaluates if modules contain genes that are over-represented by specific pathways or that are altered in a specific sample group. Additionally, CEMiTool is able to integrate transcriptomic data with interactome information, identifying the potential hubs on each network.\n",
      "Version: 1.22.0\n",
      "Publication date: None\n",
      "Mantainer: Helder Nakaya <hnakaya@usp.br>\n",
      "Authors:\n",
      "Author: Pedro Russo [aut], Gustavo Ferreira [aut], Matheus BÃ¼rger [aut], Lucas Cardozo [aut], Diogenes Lima [aut], Thiago Hirata [aut], Melissa Lever [aut], Helder Nakaya [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - methods, type: imports\n",
      "  - scales, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - data.table (>= 1.9.4) type: imports\n",
      "  - WGCNA, type: imports\n",
      "  - grid, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - ggpmisc, type: imports\n",
      "  - ggthemes, type: imports\n",
      "  - ggrepel, type: imports\n",
      "  - sna, type: imports\n",
      "  - clusterProfiler, type: imports\n",
      "  - fgsea, type: imports\n",
      "  - stringr, type: imports\n",
      "  - knitr, type: imports\n",
      "  - rmarkdown, type: imports\n",
      "  - igraph, type: imports\n",
      "  - DT, type: imports\n",
      "  - htmltools, type: imports\n",
      "  - pracma, type: imports\n",
      "  - intergraph, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - utils, type: imports\n",
      "  - network, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - ggdendro, type: imports\n",
      "  - gridExtra, type: imports\n",
      "  - gtable, type: imports\n",
      "  - fastcluster, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/CEMiTool.html\n",
      "\n",
      "Name: orthogene\n",
      "Description:\n",
      "`orthogene` is an R package for easy mapping of orthologous genes across hundreds of species. It pulls up-to-date gene ortholog mappings across **700+ organisms**. It also provides various utility functions to aggregate/expand common objects (e.g. data.frames, gene expression matrices, lists) using **1:1**, **many:1**, **1:many** or **many:many** gene mappings, both within- and between-species.\n",
      "Version: 1.4.1\n",
      "Publication date: None\n",
      "Mantainer: Brian Schilder <brian_schilder@alumni.brown.edu>\n",
      "Authors:\n",
      "Author: Brian Schilder [cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - dplyr, type: imports\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - jsonlite, type: imports\n",
      "  - homologene, type: imports\n",
      "  - gprofiler2, type: imports\n",
      "  - babelgene, type: imports\n",
      "  - data.table, type: imports\n",
      "  - parallel, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - ggpubr, type: imports\n",
      "  - patchwork, type: imports\n",
      "  - DelayedArray, type: imports\n",
      "  - grr, type: imports\n",
      "  - repmis, type: imports\n",
      "  - ggtree, type: imports\n",
      "  - tools, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/orthogene.html\n",
      "\n",
      "Name: PureCN\n",
      "Description:\n",
      "This package estimates tumor purity, copy number, and loss of heterozygosity (LOH), and classifies single nucleotide variants (SNVs) by somatic status and clonality. PureCN is designed for targeted short read sequencing data, integrates well with standard somatic variant detection and copy number pipelines, and has support for tumor samples without matching normal samples.\n",
      "Version: 2.4.0\n",
      "Publication date: None\n",
      "Mantainer: Markus Riester <markus.riester@novartis.com>\n",
      "Authors:\n",
      "Author: Markus Riester [aut, cre] , Angad P. Singh [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - DNAcopy, type: depends\n",
      "  - VariantAnnotation, type: depends\n",
      "  - GenomicRanges, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - data.table, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - GenomicFeatures, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - gridExtra, type: imports\n",
      "  - futile.logger, type: imports\n",
      "  - VGAM, type: imports\n",
      "  - tools, type: imports\n",
      "  - methods, type: imports\n",
      "  - mclust, type: imports\n",
      "  - rhdf5, type: imports\n",
      "  - Matrix, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/PureCN.html\n",
      "\n",
      "Name: ROTS\n",
      "Description:\n",
      "Calculates the Reproducibility-Optimized Test Statistic (ROTS) for differential testing in omics data.\n",
      "Version: 1.26.0\n",
      "Publication date: None\n",
      "Mantainer: Tomi Suomi <tomi.suomi@utu.fi>\n",
      "Authors:\n",
      "Author: Fatemeh Seyednasrollah, Tomi Suomi, Laura L. Elo\n",
      "Dependencies:\n",
      "  - R (>= 3.3) type: depends\n",
      "  - Rcpp, type: imports\n",
      "  - stats, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - methods, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ROTS.html\n",
      "\n",
      "Name: singleCellTK\n",
      "Description:\n",
      "The Single Cell Toolkit (SCTK) in the singleCellTK package provides an interface to popular tools for importing, quality control, analysis, and visualization of single cell RNA-seq data. SCTK allows users to seamlessly integrate tools from various packages at different stages of the analysis workflow. A general \"a la carte\" workflow gives users the ability access to multiple methods for data importing, calculation of general QC metrics, doublet detection, ambient RNA estimation and removal, filtering, normalization, batch correction or integration, dimensionality reduction, 2-D embedding, clustering, marker detection, differential expression, cell type labeling, pathway analysis, and data exporting. Curated workflows can be used to run Seurat and Celda. Streamlined quality control can be performed on the command line using the SCTK-QC pipeline. Users can analyze their data using commands in the R console or by using an interactive Shiny Graphical User Interface (GUI). Specific analyses or entire workflows can be summarized and shared with comprehensive HTML reports generated by Rmarkdown. Additional documentation and vignettes can be found at camplab.net/sctk.\n",
      "Version: 2.8.0\n",
      "Publication date: None\n",
      "Mantainer: Yichen Wang <wangych@bu.edu>\n",
      "Authors:\n",
      "Author: Yichen Wang [aut, cre] , Irzam Sarfraz [aut] , Rui Hong [aut], Yusuke Koga [aut], Salam Alabdullatif [aut], David Jenkins [aut] , Vidya Akavoor [aut], Xinyun Cao [aut], Shruthi Bandyadka [aut], Anastasia Leshchyk [aut], Tyler Faits [aut], Mohammed Muzamil Khan [aut], Zhe Wang [aut], W. Evan Johnson [aut] , Joshua David Campbell [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - SingleCellExperiment, type: depends\n",
      "  - DelayedArray, type: depends\n",
      "  - Biobase, type: depends\n",
      "  - ape, type: imports\n",
      "  - AnnotationHub, type: imports\n",
      "  - batchelor, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - celldex, type: imports\n",
      "  - colourpicker, type: imports\n",
      "  - colorspace, type: imports\n",
      "  - cowplot, type: imports\n",
      "  - cluster, type: imports\n",
      "  - ComplexHeatmap, type: imports\n",
      "  - data.table, type: imports\n",
      "  - DelayedMatrixStats, type: imports\n",
      "  - DESeq2, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - DT, type: imports\n",
      "  - ExperimentHub, type: imports\n",
      "  - ensembldb, type: imports\n",
      "  - fields, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - ggplotify, type: imports\n",
      "  - ggrepel, type: imports\n",
      "  - ggtree, type: imports\n",
      "  - gridExtra, type: imports\n",
      "  - GSVA, type: imports\n",
      "  - GSVAdata, type: imports\n",
      "  - igraph, type: imports\n",
      "  - KernSmooth, type: imports\n",
      "  - limma, type: imports\n",
      "  - MAST, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - methods, type: imports\n",
      "  - msigdbr, type: imports\n",
      "  - multtest, type: imports\n",
      "  - plotly, type: imports\n",
      "  - plyr, type: imports\n",
      "  - ROCR, type: imports\n",
      "  - Rtsne, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - scater, type: imports\n",
      "  - scMerge, type: imports\n",
      "  - scran, type: imports\n",
      "  - Seurat (>= 3.1.3) type: imports\n",
      "  - shiny, type: imports\n",
      "  - shinyjs, type: imports\n",
      "  - SingleR, type: imports\n",
      "  - SoupX, type: imports\n",
      "  - sva, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - shinyalert, type: imports\n",
      "  - circlize, type: imports\n",
      "  - enrichR, type: imports\n",
      "  - celda, type: imports\n",
      "  - shinycssloaders, type: imports\n",
      "  - DropletUtils, type: imports\n",
      "  - scds, type: imports\n",
      "  - reticulate (>= 1.14) type: imports\n",
      "  - tools, type: imports\n",
      "  - tximport, type: imports\n",
      "  - fishpond, type: imports\n",
      "  - withr, type: imports\n",
      "  - GSEABase, type: imports\n",
      "  - R.utils, type: imports\n",
      "  - zinbwave, type: imports\n",
      "  - scRNAseq, type: imports\n",
      "  - TENxPBMCData, type: imports\n",
      "  - yaml, type: imports\n",
      "  - rmarkdown, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - scDblFinder, type: imports\n",
      "  - metap, type: imports\n",
      "  - VAM (>= 0.5.3) type: imports\n",
      "  - tibble, type: imports\n",
      "  - rlang, type: imports\n",
      "  - TSCAN, type: imports\n",
      "  - TrajectoryUtils, type: imports\n",
      "  - scuttle, type: imports\n",
      "  - utils, type: imports\n",
      "  - stats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/singleCellTK.html\n",
      "\n",
      "Name: OUTRIDER\n",
      "Description:\n",
      "Identification of aberrant gene expression in RNA-seq data. Read count expectations are modeled by an autoencoder to control for confounders in the data. Given these expectations, the RNA-seq read counts are assumed to follow a negative binomial distribution with a gene-specific dispersion. Outliers are then identified as read counts that significantly deviate from this distribution. Furthermore, OUTRIDER provides useful plotting functions to analyze and visualize the results.\n",
      "Version: 1.16.3\n",
      "Publication date: None\n",
      "Mantainer: Christian Mertes <mertes@in.tum.de>\n",
      "Authors:\n",
      "Author: Felix Brechtmann [aut], Christian Mertes [aut, cre] , Agne Matuseviciute [aut], Michaela Fee MÃ¼ller [ctb], Vicente Yepez [aut] , Julien Gagneur [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.6) type: depends\n",
      "  - BiocParallel, type: depends\n",
      "  - GenomicFeatures, type: depends\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - data.table, type: depends\n",
      "  - methods, type: depends\n",
      "  - BBmisc, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - DESeq2, type: imports\n",
      "  - generics, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - heatmaply, type: imports\n",
      "  - pheatmap, type: imports\n",
      "  - graphics, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - plotly, type: imports\n",
      "  - plyr, type: imports\n",
      "  - pcaMethods, type: imports\n",
      "  - PRROC, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - scales, type: imports\n",
      "  - splines, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/OUTRIDER.html\n",
      "\n",
      "Name: memes\n",
      "Description:\n",
      "A seamless interface to the MEME Suite family of tools for motif analysis. 'memes' provides data aware utilities for using GRanges objects as entrypoints to motif analysis, data structures for examining & editing motif lists, and novel data visualizations. 'memes' functions and data structures are amenable to both base R and tidyverse workflows.\n",
      "Version: 1.6.0\n",
      "Publication date: None\n",
      "Mantainer: Spencer Nystrom <nystromdev@gmail.com>\n",
      "Authors:\n",
      "Author: Spencer Nystrom [aut, cre, cph]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - Biostrings, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - cmdfun (>= 1.0.2) type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - ggseqlogo, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - methods, type: imports\n",
      "  - patchwork, type: imports\n",
      "  - processx, type: imports\n",
      "  - purrr, type: imports\n",
      "  - rlang, type: imports\n",
      "  - readr, type: imports\n",
      "  - stats, type: imports\n",
      "  - tools, type: imports\n",
      "  - tibble, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - utils, type: imports\n",
      "  - usethis, type: imports\n",
      "  - universalmotif, type: imports\n",
      "  - xml2, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/memes.html\n",
      "\n",
      "Name: BayesSpace\n",
      "Description:\n",
      "Tools for clustering and enhancing the resolution of spatial gene expression experiments. BayesSpace clusters a low-dimensional representation of the gene expression matrix, incorporating a spatial prior to encourage neighboring spots to cluster together. The method can enhance the resolution of the low-dimensional representation into \"sub-spots\", for which features such as gene expression or cell type composition can be imputed.\n",
      "Version: 1.8.2\n",
      "Publication date: None\n",
      "Mantainer: Matt Stone <mstone@fredhutch.org>\n",
      "Authors:\n",
      "Author: Edward Zhao [aut], Matt Stone [aut, cre], Xing Ren [ctb], Raphael Gottardo [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 4.0.0) type: depends\n",
      "  - SingleCellExperiment, type: depends\n",
      "  - Rcpp (>= 1.0.4.6) type: imports\n",
      "  - stats, type: imports\n",
      "  - purrr, type: imports\n",
      "  - scater, type: imports\n",
      "  - scran, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - coda, type: imports\n",
      "  - rhdf5, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - assertthat, type: imports\n",
      "  - mclust, type: imports\n",
      "  - RCurl, type: imports\n",
      "  - DirichletReg, type: imports\n",
      "  - xgboost, type: imports\n",
      "  - utils, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - scales, type: imports\n",
      "  - BiocFileCache, type: imports\n",
      "  - BiocSingular, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/BayesSpace.html\n",
      "\n",
      "Name: msmsTests\n",
      "Description:\n",
      "Statistical tests for label-free LC-MS/MS data by spectral counts, to discover differentially expressed proteins between two biological conditions. Three tests are available: Poisson GLM regression, quasi-likelihood GLM regression, and the negative binomial of the edgeR package.The three models admit blocking factors to control for nuissance variables.To assure a good level of reproducibility a post-test filter is available, where we may set the minimum effect size considered biologicaly relevant, and the minimum expression of the most abundant condition.\n",
      "Version: 1.36.0\n",
      "Publication date: None\n",
      "Mantainer: Josep Gregori i Font <josep.gregori@gmail.com>\n",
      "Authors:\n",
      "Author: Josep Gregori, Alex Sanchez, and Josep Villanueva\n",
      "Dependencies:\n",
      "  - R (>= 3.0.1) type: depends\n",
      "  - MSnbase, type: depends\n",
      "  - msmsEDA, type: depends\n",
      "  - edgeR, type: imports\n",
      "  - qvalue, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/msmsTests.html\n",
      "\n",
      "Name: HilbertVis\n",
      "Description:\n",
      "Functions to visualize long vectors of integer data by means of Hilbert curves\n",
      "Version: 1.56.0\n",
      "Publication date: None\n",
      "Mantainer: Simon Anders <sanders@fs.tum.de>\n",
      "Authors:\n",
      "Author: Simon Anders <sanders at fs.tum.de>\n",
      "Dependencies:\n",
      "  - R (>= 2.6.0) type: depends\n",
      "  - grid, type: depends\n",
      "  - lattice, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 3)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/HilbertVis.html\n",
      "\n",
      "Name: proDA\n",
      "Description:\n",
      "Account for missing values in label-free mass spectrometry data without imputation. The package implements a probabilistic dropout model that ensures that the information from observed and missing values are properly combined. It adds empirical Bayesian priors to increase power to detect differentially abundant proteins.\n",
      "Version: 1.12.0\n",
      "Publication date: None\n",
      "Mantainer: Constantin Ahlmann-Eltze <artjom31415@googlemail.com>\n",
      "Authors:\n",
      "Author: Constantin Ahlmann-Eltze [aut, cre] , Simon Anders [ths]\n",
      "Dependencies:\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - methods, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - extraDistr, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/proDA.html\n",
      "\n",
      "Name: AgiMicroRna\n",
      "Description:\n",
      "Processing and Analysis of Agilent microRNA data\n",
      "Version: 2.48.0\n",
      "Publication date: None\n",
      "Mantainer: Pedro Lopez-Romero <plopez@cnic.es>\n",
      "Authors:\n",
      "Author: Pedro Lopez-Romero <plopez at cnic.es>\n",
      "Dependencies:\n",
      "  - R (>= 2.10) type: depends\n",
      "  - methods, type: depends\n",
      "  - Biobase, type: depends\n",
      "  - limma, type: depends\n",
      "  - affy, type: depends\n",
      "  - preprocessCore, type: depends\n",
      "  - affycoretools, type: depends\n",
      "  - Biobase, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/AgiMicroRna.html\n",
      "\n",
      "Name: HiCcompare\n",
      "Description:\n",
      "HiCcompare provides functions for joint normalization and difference detection in multiple Hi-C datasets. HiCcompare operates on processed Hi-C data in the form of chromosome-specific chromatin interaction matrices. It accepts three-column tab-separated text files storing chromatin interaction matrices in a sparse matrix format which are available from several sources. HiCcompare is designed to give the user the ability to perform a comparative analysis on the 3-Dimensional structure of the genomes of cells in different biological states.`HiCcompare` differs from other packages that attempt to compare Hi-C data in that it works on processed data in chromatin interaction matrix format instead of pre-processed sequencing data. In addition, `HiCcompare` provides a non-parametric method for the joint normalization and removal of biases between two Hi-C datasets for the purpose of comparative analysis. `HiCcompare` also provides a simple yet robust method for detecting differences between Hi-C datasets.\n",
      "Version: 1.20.0\n",
      "Publication date: None\n",
      "Mantainer: John Stansfield <stansfieldjc@vcu.edu>, Mikhail Dozmorov <mikhail.dozmorov@vcuhealth.org>\n",
      "Authors:\n",
      "Author: John Stansfield <stansfieldjc at vcu.edu>, Kellen Cresswell <cresswellkg at vcu.edu>, Mikhail Dozmorov <mikhail.dozmorov at vcuhealth.org>\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - dplyr, type: depends\n",
      "  - data.table, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - gridExtra, type: imports\n",
      "  - mgcv, type: imports\n",
      "  - stats, type: imports\n",
      "  - InteractionSet, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - QDNAseq, type: imports\n",
      "  - KernSmooth, type: imports\n",
      "  - methods, type: imports\n",
      "  - utils, type: imports\n",
      "  - graphics, type: imports\n",
      "  - pheatmap, type: imports\n",
      "  - gtools, type: imports\n",
      "  - rhdf5, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/HiCcompare.html\n",
      "\n",
      "Name: HTqPCR\n",
      "Description:\n",
      "Analysis of Ct values from high throughput quantitative real-time PCR (qPCR) assays across multiple conditions or replicates. The input data can be from spatially-defined formats such ABI TaqMan Low Density Arrays or OpenArray; LightCycler from Roche Applied Science; the CFX plates from Bio-Rad Laboratories; conventional 96- or 384-well plates; or microfluidic devices such as the Dynamic Arrays from Fluidigm Corporation. HTqPCR handles data loading, quality assessment, normalization, visualization and parametric or non-parametric testing for statistical significance in Ct values between features (e.g. genes, microRNAs).\n",
      "Version: 1.52.0\n",
      "Publication date: None\n",
      "Mantainer: Heidi Dvinge <hdvinge@fredhutch.org>\n",
      "Authors:\n",
      "Author: Heidi Dvinge, Paul Bertone\n",
      "Dependencies:\n",
      "  - Biobase, type: depends\n",
      "  - RColorBrewer, type: depends\n",
      "  - limma, type: depends\n",
      "  - affy, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - gplots, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - limma, type: imports\n",
      "  - methods, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - stats, type: imports\n",
      "  - stats4, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/HTqPCR.html\n",
      "\n",
      "Name: TOAST\n",
      "Description:\n",
      "This package is devoted to analyzing high-throughput data (e.g. gene expression microarray, DNA methylation microarray, RNA-seq) from complex tissues. Current functionalities include 1. detect cell-type specific or cross-cell type differential signals 2. tree-based differential analysis 3. improve variable selection in reference-free deconvolution 4. partial reference-free deconvolution with prior knowledge.\n",
      "Version: 1.12.0\n",
      "Publication date: None\n",
      "Mantainer: Ziyi Li <zli16@mdanderson.org>\n",
      "Authors:\n",
      "Author: Ziyi Li and Weiwei Zhang and Luxiao Chen and Hao Wu\n",
      "Dependencies:\n",
      "  - R (>= 3.6) type: depends\n",
      "  - EpiDISH, type: depends\n",
      "  - limma, type: depends\n",
      "  - nnls, type: depends\n",
      "  - quadprog, type: depends\n",
      "  - stats, type: imports\n",
      "  - methods, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - corpcor, type: imports\n",
      "  - doParallel, type: imports\n",
      "  - parallel, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - GGally, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/TOAST.html\n",
      "\n",
      "Name: biocthis\n",
      "Description:\n",
      "This package expands the usethis package with the goal of helping automate the process of creating R packages for Bioconductor or making them Bioconductor-friendly.\n",
      "Version: 1.8.1\n",
      "Publication date: None\n",
      "Mantainer: Leonardo Collado-Torres <lcolladotor@gmail.com>\n",
      "Authors:\n",
      "Author: Leonardo Collado-Torres [aut, cre] , Marcel Ramos [ctb]\n",
      "Dependencies:\n",
      "  - BiocManager, type: imports\n",
      "  - fs, type: imports\n",
      "  - glue, type: imports\n",
      "  - rlang, type: imports\n",
      "  - styler, type: imports\n",
      "  - usethis (>= 2.0.1) type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/biocthis.html\n",
      "\n",
      "Name: Linnorm\n",
      "Description:\n",
      "Linnorm is an algorithm for normalizing and transforming RNA-seq, single cell RNA-seq, ChIP-seq count data or any large scale count data. It has been independently reviewed by Tian et al. on Nature Methods (https://doi.org/10.1038/s41592-019-0425-8). Linnorm can work with raw count, CPM, RPKM, FPKM and TPM.\n",
      "Version: 2.22.2\n",
      "Publication date: None\n",
      "Mantainer: Ken Shun Hang Yip <shunyip@bu.edu>\n",
      "Authors:\n",
      "Author: Shun Hang Yip <shunyip at bu.edu>, Panwen Wang <pwwang at pwwang.com>, Jean-Pierre Kocher <Kocher.JeanPierre at mayo.edu>, Pak Chung Sham <pcsham at hku.hk>, Junwen Wang <junwen at uw.edu>\n",
      "Dependencies:\n",
      "  - R (>= 4.1.0) type: depends\n",
      "  - Rcpp (>= 0.12.2) type: imports\n",
      "  - RcppArmadillo (>= 0.8.100.1.0) type: imports\n",
      "  - fpc, type: imports\n",
      "  - vegan, type: imports\n",
      "  - mclust, type: imports\n",
      "  - apcluster, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - ellipse, type: imports\n",
      "  - limma, type: imports\n",
      "  - utils, type: imports\n",
      "  - statmod, type: imports\n",
      "  - MASS, type: imports\n",
      "  - igraph, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - fastcluster, type: imports\n",
      "  - ggdendro, type: imports\n",
      "  - zoo, type: imports\n",
      "  - stats, type: imports\n",
      "  - amap, type: imports\n",
      "  - Rtsne, type: imports\n",
      "  - gmodels, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/Linnorm.html\n",
      "\n",
      "Name: miloR\n",
      "Description:\n",
      "Milo performs single-cell differential abundance testing. Cell states are modelled as representative neighbourhoods on a nearest neighbour graph. Hypothesis testing is performed using a negative bionomial generalized linear model.\n",
      "Version: 1.6.0\n",
      "Publication date: None\n",
      "Mantainer: Mike Morgan <michael.morgan@cruk.cam.ac.uk>\n",
      "Authors:\n",
      "Author: Mike Morgan [aut, cre], Emma Dann [aut, ctb]\n",
      "Dependencies:\n",
      "  - R (>= 4.0.0) type: depends\n",
      "  - edgeR, type: depends\n",
      "  - BiocNeighbors, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - SingleCellExperiment, type: imports\n",
      "  - Matrix (>= 1.3-0) type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - stats, type: imports\n",
      "  - stringr, type: imports\n",
      "  - methods, type: imports\n",
      "  - igraph, type: imports\n",
      "  - irlba, type: imports\n",
      "  - cowplot, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - BiocSingular, type: imports\n",
      "  - limma, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - tibble, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - ggraph, type: imports\n",
      "  - gtools, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - patchwork, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - ggrepel, type: imports\n",
      "  - ggbeeswarm, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - grDevices, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3 + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/miloR.html\n",
      "\n",
      "Name: ORFik\n",
      "Description:\n",
      "R package for analysis of transcript and translation features through manipulation of sequence data and NGS data like Ribo-Seq, RNA-Seq, TCP-Seq and CAGE. It is generalized in the sense that any transcript region can be analysed, as the name hints to it was made with investigation of ribosomal patterns over Open Reading Frames (ORFs) as it's primary use case. ORFik is extremely fast through use of C++, data.table and GenomicRanges. Package allows to reassign starts of the transcripts with the use of CAGE-Seq data, automatic shifting of RiboSeq reads, finding of Open Reading Frames for whole genomes and much more.\n",
      "Version: 1.18.2\n",
      "Publication date: None\n",
      "Mantainer: Haakon Tjeldnes <hauken_heyken@hotmail.com>\n",
      "Authors:\n",
      "Author: Haakon Tjeldnes [aut, cre, dtc], Kornel Labun [aut, cph], Michal Swirski [ctb], Katarzyna Chyzynska [ctb, dtc], Yamila Torres Cleuren [ctb, ths], Evind Valen [ths, fnd]\n",
      "Dependencies:\n",
      "  - R (>= 3.6.0) type: depends\n",
      "  - IRanges, type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - GenomicAlignments, type: depends\n",
      "  - AnnotationDbi, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - biomaRt, type: imports\n",
      "  - biomartr, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - BSgenome, type: imports\n",
      "  - cowplot (>= 1.0.0) type: imports\n",
      "  - data.table (>= 1.11.8) type: imports\n",
      "  - DESeq2, type: imports\n",
      "  - fst (>= 0.9.2) type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - GenomicFeatures, type: imports\n",
      "  - ggplot2 (>= 2.2.1) type: imports\n",
      "  - gridExtra (>= 2.3) type: imports\n",
      "  - httr (>= 1.3.0) type: imports\n",
      "  - jsonlite, type: imports\n",
      "  - methods (>= 3.6.0) type: imports\n",
      "  - R.utils, type: imports\n",
      "  - Rcpp (>= 1.0.0) type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - stats, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - tools, type: imports\n",
      "  - utils, type: imports\n",
      "  - xml2 (>= 1.2.0) type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ORFik.html\n",
      "\n",
      "Name: sigFeature\n",
      "Description:\n",
      "This package provides a novel feature selection algorithm for binary classification using support vector machine recursive feature elimination SVM-RFE and t-statistic. In this feature selection process, the selected features are differentially significant between the two classes and also they are good classifier with higher degree of classification accuracy.\n",
      "Version: 1.16.0\n",
      "Publication date: None\n",
      "Mantainer: Pijush Das Developer <topijush@gmail.com>\n",
      "Authors:\n",
      "Author: Pijush Das Developer [aut, cre], Dr. Susanta Roychudhury User [ctb], Dr. Sucheta Tripathy User [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - biocViews, type: imports\n",
      "  - nlme, type: imports\n",
      "  - e1071, type: imports\n",
      "  - openxlsx, type: imports\n",
      "  - pheatmap, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - SparseM, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - methods, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/sigFeature.html\n",
      "\n",
      "Name: rawrr\n",
      "Description:\n",
      "This package wraps the functionality of the RawFileReader .NET assembly. Within the R environment, spectra and chromatograms are represented by S3 objects (Kockmann T. et al. (2020) ). The package provides basic functions to download and install the required third-party libraries. The package is developed, tested, and used at the Functional Genomics Center Zurich, Switzerland .\n",
      "Version: 1.6.1\n",
      "Publication date: None\n",
      "Mantainer: Christian Panse <cp@fgcz.ethz.ch>\n",
      "Authors:\n",
      "Author: Christian Panse [aut, cre] , Tobias Kockmann [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/rawrr.html\n",
      "\n",
      "Name: SomaticSignatures\n",
      "Description:\n",
      "The SomaticSignatures package identifies mutational signatures of single nucleotide variants (SNVs). It provides a infrastructure related to the methodology described in Nik-Zainal (2012, Cell), with flexibility in the matrix decomposition algorithms.\n",
      "Version: 2.34.0\n",
      "Publication date: None\n",
      "Mantainer: Julian Gehring <jg-bioc@gmx.com>\n",
      "Authors:\n",
      "Author: Julian Gehring\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - VariantAnnotation, type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - NMF, type: depends\n",
      "  - S4Vectors, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - ggbio, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - NMF, type: imports\n",
      "  - pcaMethods, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - methods, type: imports\n",
      "  - proxy, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/SomaticSignatures.html\n",
      "\n",
      "Name: sangeranalyseR\n",
      "Description:\n",
      "This package builds on sangerseqR to allow users to create contigs from collections of Sanger sequencing reads. It provides a wide range of options for a number of commonly-performed actions including read trimming, detecting secondary peaks, and detecting indels using a reference sequence. All parameters can be adjusted interactively either in R or in the associated Shiny applications. There is extensive online documentation, and the package can outputs detailed HTML reports, including chromatograms.\n",
      "Version: 1.8.0\n",
      "Publication date: None\n",
      "Mantainer: Kuan-Hao Chao <ntueeb05howard@gmail.com>\n",
      "Authors:\n",
      "Author: Rob Lanfear <rob.lanfear at gmail.com>, Kuan-Hao Chao <ntueeb05howard at gmail.com>\n",
      "Dependencies:\n",
      "  - R (>= 4.0.0) type: depends\n",
      "  - stringr, type: depends\n",
      "  - ape, type: depends\n",
      "  - Biostrings, type: depends\n",
      "  - DECIPHER, type: depends\n",
      "  - parallel, type: depends\n",
      "  - reshape2, type: depends\n",
      "  - phangorn, type: depends\n",
      "  - sangerseqR, type: depends\n",
      "  - gridExtra, type: depends\n",
      "  - shiny, type: depends\n",
      "  - shinydashboard, type: depends\n",
      "  - shinyjs, type: depends\n",
      "  - data.table, type: depends\n",
      "  - plotly, type: depends\n",
      "  - DT, type: depends\n",
      "  - zeallot, type: depends\n",
      "  - excelR, type: depends\n",
      "  - shinycssloaders, type: depends\n",
      "  - ggdendro, type: depends\n",
      "  - shinyWidgets, type: depends\n",
      "  - openxlsx, type: depends\n",
      "  - tools, type: depends\n",
      "  - rmarkdown (>= 2.9) type: depends\n",
      "  - knitr (>= 1.33) type: depends\n",
      "  - seqinr, type: depends\n",
      "  - BiocStyle, type: depends\n",
      "  - logger, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/sangeranalyseR.html\n",
      "\n",
      "Name: scDD\n",
      "Description:\n",
      "This package implements a method to analyze single-cell RNA- seq Data utilizing flexible Dirichlet Process mixture models. Genes with differential distributions of expression are classified into several interesting patterns of differences between two conditions. The package also includes functions for simulating data with these patterns from negative binomial distributions.\n",
      "Version: 1.22.0\n",
      "Publication date: None\n",
      "Mantainer: Keegan Korthauer <keegan@stat.ubc.ca>\n",
      "Authors:\n",
      "Author: Keegan Korthauer [cre, aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - fields, type: imports\n",
      "  - mclust, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - outliers, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - EBSeq, type: imports\n",
      "  - arm, type: imports\n",
      "  - SingleCellExperiment, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - scran, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/scDD.html\n",
      "\n",
      "Name: drawProteins\n",
      "Description:\n",
      "This package draws protein schematics from Uniprot API output. From the JSON returned by the GET command, it creates a dataframe from the Uniprot Features API. This dataframe can then be used by geoms based on ggplot2 and base R to draw protein schematics.\n",
      "Version: 1.18.0\n",
      "Publication date: None\n",
      "Mantainer: Paul Brennan <brennanpincardiff@gmail.com>\n",
      "Authors:\n",
      "Author: Paul Brennan [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - ggplot2, type: imports\n",
      "  - httr, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - readr, type: imports\n",
      "  - tidyr, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/drawProteins.html\n",
      "\n",
      "Name: MSnID\n",
      "Description:\n",
      "Extracts MS/MS ID data from mzIdentML (leveraging mzID package) or text files. After collating the search results from multiple datasets it assesses their identification quality and optimize filtering criteria to achieve the maximum number of identifications while not exceeding a specified false discovery rate. Also contains a number of utilities to explore the MS/MS results and assess missed and irregular enzymatic cleavages, mass measurement accuracy, etc.\n",
      "Version: 1.32.0\n",
      "Publication date: None\n",
      "Mantainer: Vlad Petyuk <petyuk@gmail.com>\n",
      "Authors:\n",
      "Author: Vlad Petyuk with contributions from Laurent Gatto\n",
      "Dependencies:\n",
      "  - R (>= 2.10) type: depends\n",
      "  - Rcpp, type: depends\n",
      "  - MSnbase, type: imports\n",
      "  - mzID, type: imports\n",
      "  - R.cache, type: imports\n",
      "  - foreach, type: imports\n",
      "  - doParallel, type: imports\n",
      "  - parallel, type: imports\n",
      "  - methods, type: imports\n",
      "  - iterators, type: imports\n",
      "  - data.table, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - ProtGenerics, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - mzR, type: imports\n",
      "  - BiocStyle, type: imports\n",
      "  - msmsTests, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - RUnit, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - purrr, type: imports\n",
      "  - rlang, type: imports\n",
      "  - stringr, type: imports\n",
      "  - tibble, type: imports\n",
      "  - AnnotationHub, type: imports\n",
      "  - AnnotationDbi, type: imports\n",
      "  - xtable, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/MSnID.html\n",
      "\n",
      "Name: rBiopaxParser\n",
      "Description:\n",
      "Parses BioPAX files and represents them in R, at the moment BioPAX level 2 and level 3 are supported.\n",
      "Version: 2.38.0\n",
      "Publication date: None\n",
      "Mantainer: Frank Kramer <frank.kramer@informatik.uni-augsburg.de>\n",
      "Authors:\n",
      "Author: Frank Kramer\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - data.table, type: depends\n",
      "  - XML, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/rBiopaxParser.html\n",
      "\n",
      "Name: sSeq\n",
      "Description:\n",
      "The purpose of this package is to discover the genes that are differentially expressed between two conditions in RNA-seq experiments. Gene expression is measured in counts of transcripts and modeled with the Negative Binomial (NB) distribution using a shrinkage approach for dispersion estimation. The method of moment (MM) estimates for dispersion are shrunk towards an estimated target, which minimizes the average squared difference between the shrinkage estimates and the initial estimates. The exact per-gene probability under the NB model is calculated, and used to test the hypothesis that the expected expression of a gene in two conditions identically follow a NB distribution.\n",
      "Version: 1.36.0\n",
      "Publication date: None\n",
      "Mantainer: Danni Yu <dyu@purdue.edu>\n",
      "Authors:\n",
      "Author: Danni Yu <dyu at purdue.edu>, Wolfgang Huber <whuber at embl.de> and Olga Vitek <ovitek at purdue.edu>\n",
      "Dependencies:\n",
      "  - R (>= 3.0) type: depends\n",
      "  - caTools, type: depends\n",
      "  - RColorBrewer, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 3)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/sSeq.html\n",
      "\n",
      "Name: quantiseqr\n",
      "Description:\n",
      "This package provides a streamlined workflow for the quanTIseq method, developed to perform the quantification of the Tumor Immune contexture from RNA-seq data. The quantification is performed against the TIL10 signature (dissecting the contributions of ten immune cell types), carefully crafted from a collection of human RNA-seq samples. The TIL10 signature has been extensively validated using simulated, flow cytometry, and immunohistochemistry data.\n",
      "Version: 1.6.0\n",
      "Publication date: None\n",
      "Mantainer: Federico Marini <marinif@uni-mainz.de>\n",
      "Authors:\n",
      "Author: Federico Marini [aut, cre] , Francesca Finotello [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.1.0) type: depends\n",
      "  - Biobase, type: imports\n",
      "  - limSolve, type: imports\n",
      "  - MASS, type: imports\n",
      "  - methods, type: imports\n",
      "  - preprocessCore, type: imports\n",
      "  - stats, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - rlang, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/quantiseqr.html\n",
      "\n",
      "Name: plotgardener\n",
      "Description:\n",
      "Coordinate-based genomic visualization package for R. It grants users the ability to programmatically produce complex, multi-paneled figures. Tailored for genomics, plotgardener allows users to visualize large complex genomic datasets and provides exquisite control over how plots are placed and arranged on a page.\n",
      "Version: 1.4.2\n",
      "Publication date: None\n",
      "Mantainer: Nicole Kramer <nekramer@live.unc.edu>\n",
      "Authors:\n",
      "Author: Nicole Kramer [aut, cre] , Eric S. Davis [aut] , Craig Wenger [aut] , Sarah Parker [ctb], Erika Deoudes [art], Michael Love [ctb], Douglas H. Phanstiel [aut, cph]\n",
      "Dependencies:\n",
      "  - R (>= 4.1.0) type: depends\n",
      "  - curl, type: imports\n",
      "  - data.table, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - grid, type: imports\n",
      "  - ggplotify, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - methods, type: imports\n",
      "  - plyranges, type: imports\n",
      "  - purrr, type: imports\n",
      "  - Rcpp, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - rlang, type: imports\n",
      "  - stats, type: imports\n",
      "  - strawr, type: imports\n",
      "  - tools, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/plotgardener.html\n",
      "\n",
      "Name: clustifyr\n",
      "Description:\n",
      "Package designed to aid in classifying cells from single-cell RNA sequencing data using external reference data (e.g., bulk RNA-seq, scRNA-seq, microarray, gene lists). A variety of correlation based methods and gene list enrichment methods are provided to assist cell type assignment.\n",
      "Version: 1.10.0\n",
      "Publication date: None\n",
      "Mantainer: Kent Riemondy <kent.riemondy@cuanschutz.edu>\n",
      "Authors:\n",
      "Author: Rui Fu [aut], Kent Riemondy [cre, aut], Austin Gillen [ctb], Chengzhe Tian [ctb], Jay Hesselberth [ctb], Yue Hao [ctb], Michelle Daya [ctb], Sidhant Puntambekar [ctb], RNA Bioscience Initiative [fnd, cph]\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - cowplot, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - entropy, type: imports\n",
      "  - fgsea, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - readr, type: imports\n",
      "  - rlang, type: imports\n",
      "  - scales, type: imports\n",
      "  - stringr, type: imports\n",
      "  - tibble, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - stats, type: imports\n",
      "  - methods, type: imports\n",
      "  - SingleCellExperiment, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - proxy, type: imports\n",
      "  - httr, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/clustifyr.html\n",
      "\n",
      "Name: Harman\n",
      "Description:\n",
      "Harman is a PCA and constrained optimisation based technique that maximises the removal of batch effects from datasets, with the constraint that the probability of overcorrection (i.e. removing genuine biological signal along with batch noise) is kept to a fraction which is set by the end-user.\n",
      "Version: 1.26.0\n",
      "Publication date: None\n",
      "Mantainer: Jason Ross <jason.ross@csiro.au>\n",
      "Authors:\n",
      "Author: Yalchin Oytam [aut], Josh Bowden [aut], Jason Ross [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 3.6) type: depends\n",
      "  - Rcpp (>= 0.11.2) type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - Ckmeans.1d.dp, type: imports\n",
      "  - parallel, type: imports\n",
      "  - methods, type: imports\n",
      "  - matrixStats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3 + file LICENCE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/Harman.html\n",
      "\n",
      "Name: miaViz\n",
      "Description:\n",
      "The miaViz package implements plotting function to work with TreeSummarizedExperiment and related objects in a context of microbiome analysis. Among others this includes plotting tree, graph and microbiome series data. The package is part of the broader miaverse framework.\n",
      "Version: 1.6.0\n",
      "Publication date: None\n",
      "Mantainer: Tuomas Borman <tuomas.v.borman@utu.fi>\n",
      "Authors:\n",
      "Author: Felix G.M. Ernst [aut] , Tuomas Borman [aut, cre] , Leo Lahti [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - TreeSummarizedExperiment, type: depends\n",
      "  - mia, type: depends\n",
      "  - ggplot2, type: depends\n",
      "  - ggraph (>= 2.0) type: depends\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - DelayedArray, type: imports\n",
      "  - scater, type: imports\n",
      "  - ggtree, type: imports\n",
      "  - ggnewscale, type: imports\n",
      "  - viridis, type: imports\n",
      "  - tibble, type: imports\n",
      "  - tidytree, type: imports\n",
      "  - tidygraph, type: imports\n",
      "  - rlang, type: imports\n",
      "  - purrr, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - ape, type: imports\n",
      "  - DirichletMultinomial, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0 | file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/miaViz.html\n",
      "\n",
      "Name: MethylSeekR\n",
      "Description:\n",
      "This is a package for the discovery of regulatory regions from Bis-seq data\n",
      "Version: 1.38.0\n",
      "Publication date: None\n",
      "Mantainer: Lukas Burger <Lukas.Burger@fmi.ch>\n",
      "Authors:\n",
      "Author: Lukas Burger, Dimos Gaidatzis, Dirk Schubeler and Michael Stadler\n",
      "Dependencies:\n",
      "  - rtracklayer, type: depends\n",
      "  - parallel (>= 2.15.1) type: depends\n",
      "  - mhsmm (>= 0.4.4) type: depends\n",
      "  - IRanges, type: imports\n",
      "  - BSgenome, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - geneplotter, type: imports\n",
      "  - graphics (>= 2.15.2) type: imports\n",
      "  - grDevices (>= 2.15.2) type: imports\n",
      "  - parallel (>= 2.15.2) type: imports\n",
      "  - stats (>= 2.15.2) type: imports\n",
      "  - utils (>= 2.15.2) type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>=2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/MethylSeekR.html\n",
      "\n",
      "Name: snm\n",
      "Description:\n",
      "SNM is a modeling strategy especially designed for normalizing high-throughput genomic data. The underlying premise of our approach is that your data is a function of what we refer to as study-specific variables. These variables are either biological variables that represent the target of the statistical analysis, or adjustment variables that represent factors arising from the experimental or biological setting the data is drawn from. The SNM approach aims to simultaneously model all study-specific variables in order to more accurately characterize the biological or clinical variables of interest.\n",
      "Version: 1.46.0\n",
      "Publication date: None\n",
      "Mantainer: John D. Storey <jstorey@princeton.edu>\n",
      "Authors:\n",
      "Author: Brig Mecham and John D. Storey <jstorey at princeton.edu>\n",
      "Dependencies:\n",
      "  - R (>= 2.12.0) type: depends\n",
      "  - corpcor, type: imports\n",
      "  - lme4 (>= 1.0) type: imports\n",
      "  - splines, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/snm.html\n",
      "\n",
      "Name: StructuralVariantAnnotation\n",
      "Description:\n",
      "StructuralVariantAnnotation provides a framework for analysis of structural variants within the Bioconductor ecosystem. This package contains contains useful helper functions for dealing with structural variants in VCF format. The packages contains functions for parsing VCFs from a number of popular callers as well as functions for dealing with breakpoints involving two separate genomic loci encoded as GRanges objects.\n",
      "Version: 1.13.0\n",
      "Publication date: None\n",
      "Mantainer: Daniel Cameron <daniel.l.cameron@gmail.com>\n",
      "Authors:\n",
      "Author: Daniel Cameron [aut, cre] , Ruining Dong [aut]\n",
      "Dependencies:\n",
      "  - GenomicRanges, type: depends\n",
      "  - rtracklayer, type: depends\n",
      "  - VariantAnnotation, type: depends\n",
      "  - BiocGenerics, type: depends\n",
      "  - R (>= 4.1.0) type: depends\n",
      "  - assertthat, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - stringr, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - methods, type: imports\n",
      "  - rlang, type: imports\n",
      "  - GenomicFeatures, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3 + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/StructuralVariantAnnotation.html\n",
      "\n",
      "Name: flowUtils\n",
      "Description:\n",
      "Bioconductor version: Release (3.16)\n",
      "Version: 1.62.0\n",
      "Publication date: None\n",
      "Mantainer: Author: J. Spidlen., N. Gopalakrishnan, F. Hahne, B. Ellis, R. Gentleman, M. Dalphin, N. Le Meur, B. Purcell, W. Jiang\n",
      "Authors:\n",
      "Provides utilities for flow cytometry data.\n",
      "Dependencies:\n",
      "  - R (>= 2.2.0) type: depends\n",
      "  - Biobase, type: imports\n",
      "  - graph, type: imports\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - corpcor, type: imports\n",
      "  - RUnit, type: imports\n",
      "  - XML, type: imports\n",
      "  - flowCore, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/flowUtils.html\n",
      "\n",
      "Name: RNASeqPower\n",
      "Description:\n",
      "RNA-seq, sample size\n",
      "Version: 1.38.0\n",
      "Publication date: None\n",
      "Mantainer: Terry M Therneau <therneau.terry@mayo.edu>\n",
      "Authors:\n",
      "Author: Terry M Therneau [aut, cre], Hart Stephen [ctb]\n",
      "Dependencies:\n",
      "None\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL (>=2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/RNASeqPower.html\n",
      "\n",
      "Name: SingleCellSignalR\n",
      "Description:\n",
      "Allows single cell RNA seq data analysis, clustering, creates internal network and infers cell-cell interactions.\n",
      "Version: 1.10.0\n",
      "Publication date: None\n",
      "Mantainer: Jacques Colinge <jacques.colinge@inserm.fr>\n",
      "Authors:\n",
      "Author: Simon Cabello-Aguilar [aut], Jacques Colinge [cre, aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - BiocManager, type: imports\n",
      "  - circlize, type: imports\n",
      "  - limma, type: imports\n",
      "  - igraph, type: imports\n",
      "  - gplots, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - edgeR, type: imports\n",
      "  - SIMLR, type: imports\n",
      "  - data.table, type: imports\n",
      "  - pheatmap, type: imports\n",
      "  - stats, type: imports\n",
      "  - Rtsne, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stringr, type: imports\n",
      "  - foreach, type: imports\n",
      "  - multtest, type: imports\n",
      "  - scran, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/SingleCellSignalR.html\n",
      "\n",
      "Name: topconfects\n",
      "Description:\n",
      "Rank results by confident effect sizes, while maintaining False Discovery Rate and False Coverage-statement Rate control. Topconfects is an alternative presentation of TREAT results with improved usability, eliminating p-values and instead providing confidence bounds. The main application is differential gene expression analysis, providing genes ranked in order of confident log2 fold change, but it can be applied to any collection of effect sizes with associated standard errors.\n",
      "Version: 1.14.0\n",
      "Publication date: None\n",
      "Mantainer: Paul Harrison <paul.harrison@monash.edu>\n",
      "Authors:\n",
      "Author: Paul Harrison [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 3.6.0) type: depends\n",
      "  - methods, type: imports\n",
      "  - utils, type: imports\n",
      "  - stats, type: imports\n",
      "  - assertthat, type: imports\n",
      "  - ggplot2, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL-2.1 | file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/topconfects.html\n",
      "\n",
      "Name: CoGAPS\n",
      "Description:\n",
      "Coordinated Gene Activity in Pattern Sets (CoGAPS) implements a Bayesian MCMC matrix factorization algorithm, GAPS, and links it to gene set statistic methods to infer biological process activity. It can be used to perform sparse matrix factorization on any data, and when this data represents biomolecules, to do gene set analysis.\n",
      "Version: 3.18.0\n",
      "Publication date: None\n",
      "Mantainer: Elana J. Fertig <ejfertig@jhmi.edu>, Thomas D. Sherman <tomsherman159@gmail.com>, Jeanette Johnson <jjohn450@jhmi.edu>\n",
      "Authors:\n",
      "Author: Thomas Sherman, Wai-shing Lee, Conor Kelton, Ondrej Maxian, Jacob Carey, Genevieve Stein-O'Brien, Michael Considine, Maggie Wodicka, John Stansfield, Shawn Sivy, Carlo Colantuoni, Alexander Favorov, Mike Ochs, Elana Fertig\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - BiocParallel, type: imports\n",
      "  - cluster, type: imports\n",
      "  - methods, type: imports\n",
      "  - gplots, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - Rcpp, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - SingleCellExperiment, type: imports\n",
      "  - stats, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - tools, type: imports\n",
      "  - utils, type: imports\n",
      "  - rhdf5, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "BSD_3_clause + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/CoGAPS.html\n",
      "\n",
      "Name: genomeIntervals\n",
      "Description:\n",
      "This package defines classes for representing genomic intervals and provides functions and methods for working with these. Note: The package provides the basic infrastructure for and is enhanced by the package 'girafe'.\n",
      "Version: 1.54.0\n",
      "Publication date: None\n",
      "Mantainer: Julien Gagneur <gagneur@in.tum.de>\n",
      "Authors:\n",
      "Author: Julien Gagneur <gagneur at in.tum.de>, Joern Toedling, Richard Bourgon, Nicolas Delhomme\n",
      "Dependencies:\n",
      "  - R (>= 2.15.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - intervals (>= 0.14.0) type: depends\n",
      "  - BiocGenerics, type: depends\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/genomeIntervals.html\n",
      "\n",
      "Name: ViSEAGO\n",
      "Description:\n",
      "The main objective of ViSEAGO package is to carry out a data mining of biological functions and establish links between genes involved in the study. We developed ViSEAGO in R to facilitate functional Gene Ontology (GO) analysis of complex experimental design with multiple comparisons of interest. It allows to study large-scale datasets together and visualize GO profiles to capture biological knowledge. The acronym stands for three major concepts of the analysis: Visualization, Semantic similarity and Enrichment Analysis of Gene Ontology. It provides access to the last current GO annotations, which are retrieved from one of NCBI EntrezGene, Ensembl or Uniprot databases for several species. Using available R packages and novel developments, ViSEAGO extends classical functional GO analysis to focus on functional coherence by aggregating closely related biological themes while studying multiple datasets at once. It provides both a synthetic and detailed view using interactive functionalities respecting the GO graph structure and ensuring functional coherence supplied by semantic similarity. ViSEAGO has been successfully applied on several datasets from different species with a variety of biological questions. Results can be easily shared between bioinformaticians and biologists, enhancing reporting capabilities while maintaining reproducibility.\n",
      "Version: 1.12.0\n",
      "Publication date: None\n",
      "Mantainer: Aurelien Brionne <aurelien.brionne@inrae.fr>\n",
      "Authors:\n",
      "Author: Aurelien Brionne [aut, cre], Amelie Juanchich [aut], Christelle hennequet-antier [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.6) type: depends\n",
      "  - data.table, type: imports\n",
      "  - AnnotationDbi, type: imports\n",
      "  - AnnotationForge, type: imports\n",
      "  - biomaRt, type: imports\n",
      "  - dendextend, type: imports\n",
      "  - DiagrammeR, type: imports\n",
      "  - DT, type: imports\n",
      "  - dynamicTreeCut, type: imports\n",
      "  - fgsea, type: imports\n",
      "  - GOSemSim, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - GO.db, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - heatmaply, type: imports\n",
      "  - htmlwidgets, type: imports\n",
      "  - igraph, type: imports\n",
      "  - methods, type: imports\n",
      "  - plotly, type: imports\n",
      "  - processx, type: imports\n",
      "  - topGO, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - R.utils, type: imports\n",
      "  - scales, type: imports\n",
      "  - stats, type: imports\n",
      "  - UpSetR, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ViSEAGO.html\n",
      "\n",
      "Name: BRGenomics\n",
      "Description:\n",
      "This package provides useful and efficient utilites for the analysis of high-resolution genomic data using standard Bioconductor methods and classes. BRGenomics is feature-rich and simplifies a number of post-alignment processing steps and data handling. Emphasis is on efficient analysis of multiple datasets, with support for normalization and blacklisting. Included are functions for: spike-in normalizing data; generating basepair-resolution readcounts and coverage data (e.g. for heatmaps); importing and processing bam files (e.g. for conversion to bigWig files); generating metaplots/metaprofiles (bootstrapped mean profiles) with confidence intervals; conveniently calling DESeq2 without using sample-blind estimates of genewise dispersion; among other features.\n",
      "Version: 1.10.0\n",
      "Publication date: None\n",
      "Mantainer: Mike DeBerardine <mike.deberardine@gmail.com>\n",
      "Authors:\n",
      "Author: Mike DeBerardine [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - rtracklayer, type: depends\n",
      "  - GenomeInfoDb, type: depends\n",
      "  - S4Vectors, type: depends\n",
      "  - GenomicRanges, type: imports\n",
      "  - parallel, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - stats, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - GenomicAlignments, type: imports\n",
      "  - DESeq2, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - utils, type: imports\n",
      "  - methods, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/BRGenomics.html\n",
      "\n",
      "Name: debrowser\n",
      "Description:\n",
      "Bioinformatics platform containing interactive plots and tables for differential gene and region expression studies. Allows visualizing expression data much more deeply in an interactive and faster way. By changing the parameters, users can easily discover different parts of the data that like never have been done before. Manually creating and looking these plots takes time. With DEBrowser users can prepare plots without writing any code. Differential expression, PCA and clustering analysis are made on site and the results are shown in various plots such as scatter, bar, box, volcano, ma plots and Heatmaps.\n",
      "Version: 1.26.1\n",
      "Publication date: None\n",
      "Mantainer: Alper Kucukural <alper.kucukural@umassmed.edu>\n",
      "Authors:\n",
      "Author: Alper Kucukural <alper.kucukural at umassmed.edu>, Onur Yukselen <onur.yukselen at umassmed.edu>, Manuel Garber <manuel.garber at umassmed.edu>\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - shiny, type: imports\n",
      "  - jsonlite, type: imports\n",
      "  - shinyjs, type: imports\n",
      "  - shinydashboard, type: imports\n",
      "  - shinyBS, type: imports\n",
      "  - gplots, type: imports\n",
      "  - DT, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - annotate, type: imports\n",
      "  - AnnotationDbi, type: imports\n",
      "  - DESeq2, type: imports\n",
      "  - DOSE, type: imports\n",
      "  - igraph, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - stringi, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - org.Hs.eg.db, type: imports\n",
      "  - org.Mm.eg.db, type: imports\n",
      "  - limma, type: imports\n",
      "  - edgeR, type: imports\n",
      "  - clusterProfiler, type: imports\n",
      "  - methods, type: imports\n",
      "  - sva, type: imports\n",
      "  - RCurl, type: imports\n",
      "  - enrichplot, type: imports\n",
      "  - colourpicker, type: imports\n",
      "  - plotly, type: imports\n",
      "  - heatmaply, type: imports\n",
      "  - Harman, type: imports\n",
      "  - pathview, type: imports\n",
      "  - apeglm, type: imports\n",
      "  - ashr, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3 + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/debrowser.html\n",
      "\n",
      "Name: iCOBRA\n",
      "Description:\n",
      "This package provides functions for calculation and visualization of performance metrics for evaluation of ranking and binary classification (assignment) methods. Various types of performance plots can be generated programmatically. The package also contains a shiny application for interactive exploration of results.\n",
      "Version: 1.26.0\n",
      "Publication date: None\n",
      "Mantainer: Charlotte Soneson <charlottesoneson@gmail.com>\n",
      "Authors:\n",
      "Author: Charlotte Soneson [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - shiny (>= 0.9.1.9008) type: imports\n",
      "  - shinydashboard, type: imports\n",
      "  - shinyBS, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - ggplot2 (>= 2.0.0) type: imports\n",
      "  - scales, type: imports\n",
      "  - ROCR, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - DT, type: imports\n",
      "  - limma, type: imports\n",
      "  - methods, type: imports\n",
      "  - UpSetR, type: imports\n",
      "  - markdown, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>=2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/iCOBRA.html\n",
      "\n",
      "Name: normr\n",
      "Description:\n",
      "Robust normalization and difference calling procedures for ChIP-seq and alike data. Read counts are modeled jointly as a binomial mixture model with a user-specified number of components. A fitted background estimate accounts for the effect of enrichment in certain regions and, therefore, represents an appropriate null hypothesis. This robust background is used to identify significantly enriched or depleted regions.\n",
      "Version: 1.24.0\n",
      "Publication date: None\n",
      "Mantainer: Johannes Helmuth <johannes.helmuth@laborberlin.com>\n",
      "Authors:\n",
      "Author: Johannes Helmuth [aut, cre], Ho-Ryun Chung [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.3.0) type: depends\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - parallel, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - Rcpp (>= 0.11) type: imports\n",
      "  - qvalue, type: imports\n",
      "  - bamsignals, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/normr.html\n",
      "\n",
      "Name: plier\n",
      "Description:\n",
      "The PLIER (Probe Logarithmic Error Intensity Estimate) method produces an improved signal by accounting for experimentally observed patterns in probe behavior and handling error at the appropriately at low and high signal values.\n",
      "Version: 1.68.0\n",
      "Publication date: None\n",
      "Mantainer: Crispin Miller <cmiller@picr.man.ac.uk>\n",
      "Authors:\n",
      "Author: Affymetrix Inc., Crispin J Miller, PICR\n",
      "Dependencies:\n",
      "  - R (>= 2.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - affy, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - methods, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/plier.html\n",
      "\n",
      "Name: dmrseq\n",
      "Description:\n",
      "This package implements an approach for scanning the genome to detect and perform accurate inference on differentially methylated regions from Whole Genome Bisulfite Sequencing data. The method is based on comparing detected regions to a pooled null distribution, that can be implemented even when as few as two samples per population are available. Region-level statistics are obtained by fitting a generalized least squares (GLS) regression model with a nested autoregressive correlated error structure for the effect of interest on transformed methylation proportions.\n",
      "Version: 1.18.0\n",
      "Publication date: None\n",
      "Mantainer: Keegan Korthauer <keegan@stat.ubc.ca>\n",
      "Authors:\n",
      "Author: Keegan Korthauer [cre, aut] , Rafael Irizarry [aut] , Yuval Benjamini [aut], Sutirtha Chakraborty [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.5) type: depends\n",
      "  - bsseq, type: depends\n",
      "  - GenomicRanges, type: imports\n",
      "  - nlme, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - bumphunter, type: imports\n",
      "  - DelayedMatrixStats, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - outliers, type: imports\n",
      "  - methods, type: imports\n",
      "  - locfit, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - annotatr, type: imports\n",
      "  - AnnotationHub, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - splines, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/dmrseq.html\n",
      "\n",
      "Name: DNABarcodes\n",
      "Description:\n",
      "The package offers a function to create DNA barcode sets capable of correcting insertion, deletion, and substitution errors. Existing barcodes can be analysed regarding their minimal, maximal and average distances between barcodes. Finally, reads that start with a (possibly mutated) barcode can be demultiplexed, i.e., assigned to their original reference barcode.\n",
      "Version: 1.28.0\n",
      "Publication date: None\n",
      "Mantainer: Tilo Buschmann <tilo.buschmann.ac@gmail.com>\n",
      "Authors:\n",
      "Author: Tilo Buschmann <tilo.buschmann.ac at gmail.com>\n",
      "Dependencies:\n",
      "  - Matrix, type: depends\n",
      "  - parallel, type: depends\n",
      "  - Rcpp (>= 0.11.2) type: imports\n",
      "  - BH, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/DNABarcodes.html\n",
      "\n",
      "Name: EBarrays\n",
      "Description:\n",
      "EBarrays provides tools for the analysis of replicated/unreplicated microarray data.\n",
      "Version: 2.62.0\n",
      "Publication date: None\n",
      "Mantainer: Ming Yuan <myuan@isye.gatech.edu>\n",
      "Authors:\n",
      "Author: Ming Yuan, Michael Newton, Deepayan Sarkar and Christina Kendziorski\n",
      "Dependencies:\n",
      "  - R (>= 1.8.0) type: depends\n",
      "  - Biobase, type: depends\n",
      "  - lattice, type: depends\n",
      "  - methods, type: depends\n",
      "  - Biobase, type: imports\n",
      "  - cluster, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - lattice, type: imports\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/EBarrays.html\n",
      "\n",
      "Name: miRBaseConverter\n",
      "Description:\n",
      "A comprehensive tool for converting and retrieving the miRNA Name, Accession, Sequence, Version, History and Family information in different miRBase versions. It can process a huge number of miRNAs in a short time without other depends.\n",
      "Version: 1.22.0\n",
      "Publication date: None\n",
      "Mantainer: Taosheng Xu<taosheng.x@gmail.com>\n",
      "Authors:\n",
      "Author: Taosheng Xu<taosheng.x at gmail.com>, Thuc Le<Thuc.Le at unisa.edu.au>\n",
      "Dependencies:\n",
      "  - R (>= 3.4) type: depends\n",
      "  - stats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/miRBaseConverter.html\n",
      "\n",
      "Name: MLSeq\n",
      "Description:\n",
      "This package applies several machine learning methods, including SVM, bagSVM, Random Forest and CART to RNA-Seq data.\n",
      "Version: 2.16.0\n",
      "Publication date: None\n",
      "Mantainer: Gokmen Zararsiz <gokmenzararsiz@hotmail.com>\n",
      "Authors:\n",
      "Author: Gokmen Zararsiz [aut, cre], Dincer Goksuluk [aut], Selcuk Korkmaz [aut], Vahap Eldem [aut], Izzet Parug Duru [ctb], Ahmet Ozturk [aut], Ahmet Ergun Karaagaoglu [aut, ths]\n",
      "Dependencies:\n",
      "  - caret, type: depends\n",
      "  - ggplot2, type: depends\n",
      "  - testthat, type: imports\n",
      "  - VennDiagram, type: imports\n",
      "  - pamr, type: imports\n",
      "  - methods, type: imports\n",
      "  - DESeq2, type: imports\n",
      "  - edgeR, type: imports\n",
      "  - limma, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - plyr, type: imports\n",
      "  - foreach, type: imports\n",
      "  - utils, type: imports\n",
      "  - sSeq, type: imports\n",
      "  - xtable, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL(>=2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/MLSeq.html\n",
      "\n",
      "Name: schex\n",
      "Description:\n",
      "Builds hexbin plots for variables and dimension reduction stored in single cell omics data such as SingleCellExperiment and SeuratObject. The ideas used in this package are based on the excellent work of Dan Carr, Nicholas Lewin-Koh, Martin Maechler and Thomas Lumley.\n",
      "Version: 1.12.0\n",
      "Publication date: None\n",
      "Mantainer: Saskia Freytag <saskia.freytag@perkins.uwa.edu.au>\n",
      "Authors:\n",
      "Author: Saskia Freytag\n",
      "Dependencies:\n",
      "  - SingleCellExperiment, type: depends\n",
      "  - Seurat, type: depends\n",
      "  - ggplot2 (>= 3.2.1) type: depends\n",
      "  - shiny, type: depends\n",
      "  - hexbin, type: imports\n",
      "  - stats, type: imports\n",
      "  - methods, type: imports\n",
      "  - cluster, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - entropy, type: imports\n",
      "  - ggforce, type: imports\n",
      "  - scales, type: imports\n",
      "  - grid, type: imports\n",
      "  - concaveman, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/schex.html\n",
      "\n",
      "Name: aCGH\n",
      "Description:\n",
      "Functions for reading aCGH data from image analysis output files and clone information files, creation of aCGH S3 objects for storing these data. Basic methods for accessing/replacing, subsetting, printing and plotting aCGH objects.\n",
      "Version: 1.76.0\n",
      "Publication date: None\n",
      "Mantainer: Peter Dimitrov <dimitrov@stat.Berkeley.EDU>\n",
      "Authors:\n",
      "Author: Jane Fridlyand <jfridlyand at cc.ucsf.edu>, Peter Dimitrov <dimitrov at stat.Berkeley.EDU>\n",
      "Dependencies:\n",
      "  - R (>= 2.10) type: depends\n",
      "  - cluster, type: depends\n",
      "  - survival, type: depends\n",
      "  - multtest, type: depends\n",
      "  - Biobase, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - splines, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/aCGH.html\n",
      "\n",
      "Name: fabia\n",
      "Description:\n",
      "Biclustering by \"Factor Analysis for Bicluster Acquisition\" (FABIA). FABIA is a model-based technique for biclustering, that is clustering rows and columns simultaneously. Biclusters are found by factor analysis where both the factors and the loading matrix are sparse. FABIA is a multiplicative model that extracts linear dependencies between samples and feature patterns. It captures realistic non-Gaussian data distributions with heavy tails as observed in gene expression measurements. FABIA utilizes well understood model selection techniques like the EM algorithm and variational approaches and is embedded into a Bayesian framework. FABIA ranks biclusters according to their information content and separates spurious biclusters from true biclusters. The code is written in C.\n",
      "Version: 2.44.0\n",
      "Publication date: None\n",
      "Mantainer: Andreas Mitterecker <mitterecker@bioinf.jku.at>\n",
      "Authors:\n",
      "Author: Sepp Hochreiter <hochreit at bioinf.jku.at>\n",
      "Dependencies:\n",
      "  - R (>= 3.6.0) type: depends\n",
      "  - Biobase, type: depends\n",
      "  - methods, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL (>= 2.1)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/fabia.html\n",
      "\n",
      "Name: splots\n",
      "Description:\n",
      "This package is provided to support legacy code and reverse dependencies, but it should not be used as a dependency for new code development. It provides a single function, plotScreen, for visualising data in microtitre plate or slide format. As a better alternative for such functionality, please consider the platetools package on CRAN (https://cran.r-project.org/package=platetools and https://github.com/Swarchal/platetools), or generic ggplot2 graphics functionality.\n",
      "Version: 1.64.0\n",
      "Publication date: None\n",
      "Mantainer: Wolfgang Huber <wolfgang.huber@embl.de>\n",
      "Authors:\n",
      "Author: Wolfgang Huber, Oleg Sklyar\n",
      "Dependencies:\n",
      "  - grid, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/splots.html\n",
      "\n",
      "Name: tracktables\n",
      "Description:\n",
      "Methods to create complex IGV genome browser sessions and dynamic IGV reports in HTML pages.\n",
      "Version: 1.32.0\n",
      "Publication date: None\n",
      "Mantainer: Tom Carroll <tc.infomatics@gmail.com>\n",
      "Authors:\n",
      "Author: Tom Carroll, Sanjay Khadayate, Anne Pajon, Ziwei Liang\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - IRanges, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - XVector, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - XML, type: imports\n",
      "  - tractor.base, type: imports\n",
      "  - stringr, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - methods, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 3)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/tracktables.html\n",
      "\n",
      "Name: cola\n",
      "Description:\n",
      "Subgroup classification is a basic task in genomic data analysis, especially for gene expression and DNA methylation data analysis. It can also be used to test the agreement to known clinical annotations, or to test whether there exist significant batch effects. The cola package provides a general framework for subgroup classification by consensus partitioning. It has the following features: 1. It modularizes the consensus partitioning processes that various methods can be easily integrated. 2. It provides rich visualizations for interpreting the results. 3. It allows running multiple methods at the same time and provides functionalities to straightforward compare results. 4. It provides a new method to extract features which are more efficient to separate subgroups. 5. It automatically generates detailed reports for the complete analysis. 6. It allows applying consensus partitioning in a hierarchical manner.\n",
      "Version: 2.4.0\n",
      "Publication date: None\n",
      "Mantainer: Zuguang Gu <z.gu@dkfz.de>\n",
      "Authors:\n",
      "Author: Zuguang Gu [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 3.6.0) type: depends\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grid, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - ComplexHeatmap, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - GetoptLong, type: imports\n",
      "  - circlize (>= 0.4.7) type: imports\n",
      "  - GlobalOptions (>= 0.1.0) type: imports\n",
      "  - clue, type: imports\n",
      "  - parallel, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - cluster, type: imports\n",
      "  - skmeans, type: imports\n",
      "  - png, type: imports\n",
      "  - mclust, type: imports\n",
      "  - crayon, type: imports\n",
      "  - methods, type: imports\n",
      "  - xml2, type: imports\n",
      "  - microbenchmark, type: imports\n",
      "  - httr, type: imports\n",
      "  - knitr, type: imports\n",
      "  - markdown, type: imports\n",
      "  - digest, type: imports\n",
      "  - impute, type: imports\n",
      "  - brew, type: imports\n",
      "  - Rcpp (>= 0.11.0) type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - eulerr, type: imports\n",
      "  - foreach, type: imports\n",
      "  - doParallel, type: imports\n",
      "  - irlba, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/cola.html\n",
      "\n",
      "Name: CMA\n",
      "Description:\n",
      "This package provides a comprehensive collection of various microarray-based classification algorithms both from Machine Learning and Statistics. Variable Selection, Hyperparameter tuning, Evaluation and Comparison can be performed combined or stepwise in a user-friendly environment.\n",
      "Version: 1.56.0\n",
      "Publication date: None\n",
      "Mantainer: Roman Hornung <hornung@ibe.med.uni-muenchen.de>\n",
      "Authors:\n",
      "Author: Martin Slawski <ms at cs.uni-sb.de>, Anne-Laure Boulesteix <boulesteix at ibe.med.uni-muenchen.de>, Christoph Bernau <bernau at ibe.med.uni-muenchen.de>.\n",
      "Dependencies:\n",
      "  - R (>= 2.10) type: depends\n",
      "  - methods, type: depends\n",
      "  - stats, type: depends\n",
      "  - Biobase, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/CMA.html\n",
      "\n",
      "Name: conumee\n",
      "Description:\n",
      "This package contains a set of processing and plotting methods for performing copy-number variation (CNV) analysis using Illumina 450k or EPIC methylation arrays.\n",
      "Version: 1.32.0\n",
      "Publication date: None\n",
      "Mantainer: Volker Hovestadt <conumee@hovestadt.bio>\n",
      "Authors:\n",
      "Author: Volker Hovestadt, Marc Zapatka\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - minfi, type: depends\n",
      "  - IlluminaHumanMethylation450kanno.ilmn12.hg19, type: depends\n",
      "  - IlluminaHumanMethylation450kmanifest, type: depends\n",
      "  - IlluminaHumanMethylationEPICanno.ilm10b2.hg19, type: depends\n",
      "  - IlluminaHumanMethylationEPICmanifest, type: depends\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - DNAcopy, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/conumee.html\n",
      "\n",
      "Name: CompoundDb\n",
      "Description:\n",
      "CompoundDb provides functionality to create and use (chemical) compound annotation databases from a variety of different sources such as LipidMaps, HMDB, ChEBI or MassBank. The database format allows to store in addition MS/MS spectra along with compound information. The package provides also a backend for Bioconductor's Spectra package and allows thus to match experimetal MS/MS spectra against MS/MS spectra in the database. Databases can be stored in SQLite format and are thus portable.\n",
      "Version: 1.2.1\n",
      "Publication date: None\n",
      "Mantainer: Johannes Rainer <johannes.rainer@eurac.edu>\n",
      "Authors:\n",
      "Author: Jan Stanstrup [aut] , Johannes Rainer [aut, cre] , Josep M. Badia [ctb] , Roger Gine [aut] , Andrea Vicini [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - methods, type: depends\n",
      "  - AnnotationFilter, type: depends\n",
      "  - S4Vectors, type: depends\n",
      "  - BiocGenerics, type: imports\n",
      "  - ChemmineR, type: imports\n",
      "  - tibble, type: imports\n",
      "  - jsonlite, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - DBI, type: imports\n",
      "  - dbplyr, type: imports\n",
      "  - RSQLite, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - ProtGenerics, type: imports\n",
      "  - xml2, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - Spectra, type: imports\n",
      "  - MsCoreUtils, type: imports\n",
      "  - MetaboCoreUtils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/CompoundDb.html\n",
      "\n",
      "Name: IPO\n",
      "Description:\n",
      "The outcome of XCMS data processing strongly depends on the parameter settings. IPO (`Isotopologue Parameter Optimization`) is a parameter optimization tool that is applicable for different kinds of samples and liquid chromatography coupled to high resolution mass spectrometry devices, fast and free of labeling steps. IPO uses natural, stable 13C isotopes to calculate a peak picking score. Retention time correction is optimized by minimizing the relative retention time differences within features and grouping parameters are optimized by maximizing the number of features showing exactly one peak from each injection of a pooled sample. The different parameter settings are achieved by design of experiment. The resulting scores are evaluated using response surface models.\n",
      "Version: 1.24.0\n",
      "Publication date: None\n",
      "Mantainer: Thomas Lieb <thomas.r.lieb@gmail.com>\n",
      "Authors:\n",
      "Author: Gunnar Libiseller <Gunnar.Libiseller at joanneum.at>, Christoph Magnes <christoph.magnes at joanneum.at>, Thomas Lieb <thomas.r.lieb at gmail.com>\n",
      "Dependencies:\n",
      "  - xcms, type: depends\n",
      "  - rsm, type: depends\n",
      "  - CAMERA, type: depends\n",
      "  - grDevices, type: depends\n",
      "  - graphics, type: depends\n",
      "  - stats, type: depends\n",
      "  - utils, type: depends\n",
      "  - BiocParallel, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2) + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/IPO.html\n",
      "\n",
      "Name: tidySummarizedExperiment\n",
      "Description:\n",
      "tidySummarizedExperiment is an adapter that abstracts the 'SummarizedExperiment' container in the form of tibble and allows the data manipulation, plotting and nesting using 'tidyverse'\n",
      "Version: 1.8.0\n",
      "Publication date: None\n",
      "Mantainer: Stefano Mangiola <mangiolastefano@gmail.com>\n",
      "Authors:\n",
      "Author: Stefano Mangiola [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.0.0) type: depends\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - tibble (>= 3.0.4) type: imports\n",
      "  - dplyr, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - rlang, type: imports\n",
      "  - purrr, type: imports\n",
      "  - lifecycle, type: imports\n",
      "  - methods, type: imports\n",
      "  - plotly, type: imports\n",
      "  - utils, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - tidyselect, type: imports\n",
      "  - ellipsis, type: imports\n",
      "  - vctrs, type: imports\n",
      "  - pillar, type: imports\n",
      "  - stringr, type: imports\n",
      "  - cli, type: imports\n",
      "  - fansi, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/tidySummarizedExperiment.html\n",
      "\n",
      "Name: convert\n",
      "Description:\n",
      "Define coerce methods for microarray data objects.\n",
      "Version: 1.74.0\n",
      "Publication date: None\n",
      "Mantainer: Yee Hwa (Jean) Yang <jean@biostat.ucsf.edu>\n",
      "Authors:\n",
      "Author: Gordon Smyth <smyth at wehi.edu.au>, James Wettenhall <wettenhall at wehi.edu.au>, Yee Hwa (Jean Yang) <jean at biostat.ucsf.edu>, Martin Morgan <Martin.Morgan at RoswellPark.org>\n",
      "Dependencies:\n",
      "  - R (>= 2.6.0) type: depends\n",
      "  - Biobase, type: depends\n",
      "  - limma, type: depends\n",
      "  - marray, type: depends\n",
      "  - utils, type: depends\n",
      "  - methods, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/convert.html\n",
      "\n",
      "Name: microRNA\n",
      "Description:\n",
      "Different data resources for microRNAs and some functions for manipulating them.\n",
      "Version: 1.56.0\n",
      "Publication date: None\n",
      "Mantainer: \"James F. Reid\" <james.reid@ifom-ieo-campus.it>\n",
      "Authors:\n",
      "Author: R. Gentleman, S. Falcon\n",
      "Dependencies:\n",
      "  - R (>= 2.10) type: depends\n",
      "  - Biostrings, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/microRNA.html\n",
      "\n",
      "Name: qpgraph\n",
      "Description:\n",
      "Estimate gene and eQTL networks from high-throughput expression and genotyping assays.\n",
      "Version: 2.32.2\n",
      "Publication date: None\n",
      "Mantainer: Robert Castelo <robert.castelo@upf.edu>\n",
      "Authors:\n",
      "Author: Robert Castelo [aut, cre], Alberto Roverato [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.5) type: depends\n",
      "  - methods, type: imports\n",
      "  - parallel, type: imports\n",
      "  - Matrix (>= 1.5-0) type: imports\n",
      "  - grid, type: imports\n",
      "  - annotate, type: imports\n",
      "  - graph, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - AnnotationDbi, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - GenomicFeatures, type: imports\n",
      "  - mvtnorm, type: imports\n",
      "  - qtl, type: imports\n",
      "  - Rgraphviz, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/qpgraph.html\n",
      "\n",
      "Name: SpatialDecon\n",
      "Description:\n",
      "Using spatial or bulk gene expression data, estimates abundance of mixed cell types within each observation. Based on \"Advances in mixed cell deconvolution enable quantification of cell types in spatial transcriptomic data\", Danaher (2022). Designed for use with the NanoString GeoMx platform, but applicable to any gene expression data.\n",
      "Version: 1.8.0\n",
      "Publication date: None\n",
      "Mantainer: Maddy Griswold <mgriswold@nanostring.com>\n",
      "Authors:\n",
      "Author: Maddy Griswold [cre, aut], Patrick Danaher [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.0.0) type: depends\n",
      "  - grDevices, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - graphics, type: imports\n",
      "  - SeuratObject, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - GeomxTools, type: imports\n",
      "  - repmis, type: imports\n",
      "  - methods, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - logNormReg (>= 0.4) type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/SpatialDecon.html\n",
      "\n",
      "Name: tidybulk\n",
      "Description:\n",
      "This is a collection of utility functions that allow to perform exploration of and calculations to RNA sequencing data, in a modular, pipe-friendly and tidy fashion.\n",
      "Version: 1.10.0\n",
      "Publication date: None\n",
      "Mantainer: Stefano Mangiola <mangiolastefano@gmail.com>\n",
      "Authors:\n",
      "Author: Stefano Mangiola [aut, cre], Maria Doyle [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 4.1.0) type: depends\n",
      "  - tibble, type: imports\n",
      "  - readr, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - stringi, type: imports\n",
      "  - stringr, type: imports\n",
      "  - rlang, type: imports\n",
      "  - purrr, type: imports\n",
      "  - tidyselect, type: imports\n",
      "  - preprocessCore, type: imports\n",
      "  - stats, type: imports\n",
      "  - parallel, type: imports\n",
      "  - utils, type: imports\n",
      "  - lifecycle, type: imports\n",
      "  - scales, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - methods, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - crayon, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/tidybulk.html\n",
      "\n",
      "Name: meshes\n",
      "Description:\n",
      "MeSH (Medical Subject Headings) is the NLM controlled vocabulary used to manually index articles for MEDLINE/PubMed. MeSH terms were associated by Entrez Gene ID by three methods, gendoo, gene2pubmed and RBBH. This association is fundamental for enrichment and semantic analyses. meshes supports enrichment analysis (over-representation and gene set enrichment analysis) of gene list or whole expression profile. The semantic comparisons of MeSH terms provide quantitative ways to compute similarities between genes and gene groups. meshes implemented five methods proposed by Resnik, Schlicker, Jiang, Lin and Wang respectively and supports more than 70 species.\n",
      "Version: 1.24.0\n",
      "Publication date: None\n",
      "Mantainer: Guangchuang Yu <guangchuangyu@gmail.com>\n",
      "Authors:\n",
      "Author: Guangchuang Yu [aut, cre], Erqiang Hu [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 4.1.0) type: depends\n",
      "  - AnnotationDbi, type: imports\n",
      "  - DOSE, type: imports\n",
      "  - enrichplot, type: imports\n",
      "  - GOSemSim, type: imports\n",
      "  - methods, type: imports\n",
      "  - utils, type: imports\n",
      "  - AnnotationHub, type: imports\n",
      "  - MeSHDbi, type: imports\n",
      "  - yulab.utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/meshes.html\n",
      "\n",
      "Name: CARNIVAL\n",
      "Description:\n",
      "An upgraded causal reasoning tool from Melas et al in R with updated assignments of TFs' weights from PROGENy scores. Optimization parameters can be freely adjusted and multiple solutions can be obtained and aggregated.\n",
      "Version: 2.8.0\n",
      "Publication date: None\n",
      "Mantainer: Attila Gabor <attila.gabor@uni-heidelberg.de>\n",
      "Authors:\n",
      "Author: Enio Gjerga [aut] , Panuwat Trairatphisan [aut], Anika Liu [ctb], Alberto Valdeolivas [ctb], Nikolas Peschke [ctb], Aurelien Dugourd [ctb], Attila Gabor [cre], Olga Ivanova [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - readr, type: imports\n",
      "  - stringr, type: imports\n",
      "  - lpSolve, type: imports\n",
      "  - igraph, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - tibble, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - rjson, type: imports\n",
      "  - rmarkdown, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/CARNIVAL.html\n",
      "\n",
      "Name: ensemblVEP\n",
      "Description:\n",
      "Query the Ensembl Variant Effect Predictor via the perl API.\n",
      "Version: 1.40.0\n",
      "Publication date: None\n",
      "Mantainer: Bioconductor Package Maintainer <maintainer@bioconductor.org>\n",
      "Authors:\n",
      "Author: Valerie Obenchain and Lori Shepherd\n",
      "Dependencies:\n",
      "  - methods, type: depends\n",
      "  - BiocGenerics, type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - VariantAnnotation, type: depends\n",
      "  - S4Vectors, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - stats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ensemblVEP.html\n",
      "\n",
      "Name: pipeFrame\n",
      "Description:\n",
      "pipeFrame is an R package for building a componentized bioinformatics pipeline. Each step in this pipeline is wrapped in the framework, so the connection among steps is created seamlessly and automatically. Users could focus more on fine-tuning arguments rather than spending a lot of time on transforming file format, passing task outputs to task inputs or installing the dependencies. Componentized step elements can be customized into other new pipelines flexibly as well. This pipeline can be split into several important functional steps, so it is much easier for users to understand the complex arguments from each step rather than parameter combination from the whole pipeline. At the same time, componentized pipeline can restart at the breakpoint and avoid rerunning the whole pipeline, which may save a lot of time for users on pipeline tuning or such issues as power off or process other interrupts.\n",
      "Version: 1.14.0\n",
      "Publication date: None\n",
      "Mantainer: Zheng Wei <wzweizheng@qq.com>\n",
      "Authors:\n",
      "Author: Zheng Wei, Shining Ma\n",
      "Dependencies:\n",
      "  - R (>= 4.0.0) type: depends\n",
      "  - BSgenome, type: imports\n",
      "  - digest, type: imports\n",
      "  - visNetwork, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - methods, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - parallel, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - rmarkdown, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/pipeFrame.html\n",
      "\n",
      "Name: wiggleplotr\n",
      "Description:\n",
      "Tools to visualise read coverage from sequencing experiments together with genomic annotations (genes, transcripts, peaks). Introns of long transcripts can be rescaled to a fixed length for better visualisation of exonic read coverage.\n",
      "Version: 1.22.0\n",
      "Publication date: None\n",
      "Mantainer: Kaur Alasoo <kaur.alasoo@gmail.com>\n",
      "Authors:\n",
      "Author: Kaur Alasoo [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 3.6) type: depends\n",
      "  - dplyr, type: imports\n",
      "  - ggplot2 (>= 2.2.0) type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - cowplot, type: imports\n",
      "  - assertthat, type: imports\n",
      "  - purrr, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Apache License 2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/wiggleplotr.html\n",
      "\n",
      "Name: HiTC\n",
      "Description:\n",
      "The HiTC package was developed to explore high-throughput 'C' data such as 5C or Hi-C. Dedicated R classes as well as standard methods for quality controls, normalization, visualization, and further analysis are also provided.\n",
      "Version: 1.42.0\n",
      "Publication date: None\n",
      "Mantainer: Nicolas Servant <nicolas.servant@curie.fr>\n",
      "Authors:\n",
      "Author: Nicolas Servant\n",
      "Dependencies:\n",
      "  - R (>= 2.15.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - IRanges, type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - Biostrings, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - parallel, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/HiTC.html\n",
      "\n",
      "Name: kebabs\n",
      "Description:\n",
      "The package provides functionality for kernel-based analysis of DNA, RNA, and amino acid sequences via SVM-based methods. As core functionality, kebabs implements following sequence kernels: spectrum kernel, mismatch kernel, gappy pair kernel, and motif kernel. Apart from an efficient implementation of standard position-independent functionality, the kernels are extended in a novel way to take the position of patterns into account for the similarity measure. Because of the flexibility of the kernel formulation, other kernels like the weighted degree kernel or the shifted weighted degree kernel with constant weighting of positions are included as special cases. An annotation-specific variant of the kernels uses annotation information placed along the sequence together with the patterns in the sequence. The package allows for the generation of a kernel matrix or an explicit feature representation in dense or sparse format for all available kernels which can be used with methods implemented in other R packages. With focus on SVM-based methods, kebabs provides a framework which simplifies the usage of existing SVM implementations in kernlab, e1071, and LiblineaR. Binary and multi-class classification as well as regression tasks can be used in a unified way without having to deal with the different functions, parameters, and formats of the selected SVM. As support for choosing hyperparameters, the package provides cross validation - including grouped cross validation, grid search and model selection functions. For easier biological interpretation of the results, the package computes feature weights for all SVMs and prediction profiles which show the contribution of individual sequence positions to the prediction result and indicate the relevance of sequence sections for the learning result and the underlying biological functions.\n",
      "Version: 1.32.0\n",
      "Publication date: None\n",
      "Mantainer: Ulrich Bodenhofer <bodenhofer@bioinf.jku.at>\n",
      "Authors:\n",
      "Author: Johannes Palme\n",
      "Dependencies:\n",
      "  - R (>= 3.3.0) type: depends\n",
      "  - Biostrings, type: depends\n",
      "  - kernlab, type: depends\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - Rcpp (>= 0.11.2) type: imports\n",
      "  - Matrix (>= 1.5-0) type: imports\n",
      "  - XVector, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - e1071, type: imports\n",
      "  - LiblineaR, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - utils, type: imports\n",
      "  - apcluster, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2.1)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/kebabs.html\n",
      "\n",
      "Name: cellHTS2\n",
      "Description:\n",
      "This package provides tools for the analysis of high-throughput assays that were performed in microtitre plate formats (including but not limited to 384-well plates). The functionality includes data import and management, normalisation, quality assessment, replicate summarisation and statistical scoring. A webpage that provides a detailed graphical overview over the data and analysis results is produced. In our work, we have applied the package to RNAi screens on fly and human cells, and for screens of yeast libraries. See ?cellHTS2 for a brief introduction.\n",
      "Version: 2.62.0\n",
      "Publication date: None\n",
      "Mantainer: Joseph Barry <joseph.barry@embl.de>\n",
      "Authors:\n",
      "Author: Ligia Bras, Wolfgang Huber <whuber at embl.de>, Michael Boutros <m.boutros at dkfz.de>, Gregoire Pau <gpau at embl.de>, Florian Hahne <florian.hahne at novartis.com>\n",
      "Dependencies:\n",
      "  - R (>= 2.10) type: depends\n",
      "  - RColorBrewer, type: depends\n",
      "  - Biobase, type: depends\n",
      "  - methods, type: depends\n",
      "  - genefilter, type: depends\n",
      "  - splots, type: depends\n",
      "  - vsn, type: depends\n",
      "  - hwriter, type: depends\n",
      "  - locfit, type: depends\n",
      "  - grid, type: depends\n",
      "  - GSEABase, type: imports\n",
      "  - Category, type: imports\n",
      "  - stats4, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/cellHTS2.html\n",
      "\n",
      "Name: coRdon\n",
      "Description:\n",
      "Tool for analysis of codon usage in various unannotated or KEGG/COG annotated DNA sequences. Calculates different measures of CU bias and CU-based predictors of gene expressivity, and performs gene set enrichment analysis for annotated sequences. Implements several methods for visualization of CU and enrichment analysis results.\n",
      "Version: 1.16.0\n",
      "Publication date: None\n",
      "Mantainer: Anamaria Elek <anamariaelek@gmail.com>\n",
      "Authors:\n",
      "Author: Anamaria Elek [cre, aut], Maja Kuzman [aut], Kristian Vlahovicek [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.5) type: depends\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - stringr, type: imports\n",
      "  - purrr, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - data.table, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/coRdon.html\n",
      "\n",
      "Name: gtrellis\n",
      "Description:\n",
      "Genome level Trellis graph visualizes genomic data conditioned by genomic categories (e.g. chromosomes). For each genomic category, multiple dimensional data which are represented as tracks describe different features from different aspects. This package provides high flexibility to arrange genomic categories and to add self-defined graphics in the plot.\n",
      "Version: 1.30.0\n",
      "Publication date: None\n",
      "Mantainer: Zuguang Gu <z.gu@dkfz.de>\n",
      "Authors:\n",
      "Author: Zuguang Gu [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 3.1.2) type: depends\n",
      "  - grid, type: depends\n",
      "  - IRanges, type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - circlize (>= 0.4.8) type: imports\n",
      "  - GetoptLong, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/gtrellis.html\n",
      "\n",
      "Name: SIAMCAT\n",
      "Description:\n",
      "Pipeline for Statistical Inference of Associations between Microbial Communities And host phenoTypes (SIAMCAT). A primary goal of analyzing microbiome data is to determine changes in community composition that are associated with environmental factors. In particular, linking human microbiome composition to host phenotypes such as diseases has become an area of intense research. For this, robust statistical modeling and biomarker extraction toolkits are crucially needed. SIAMCAT provides a full pipeline supporting data preprocessing, statistical association testing, statistical modeling (LASSO logistic regression) including tools for evaluation and interpretation of these models (such as cross validation, parameter selection, ROC analysis and diagnostic model plots).\n",
      "Version: 2.2.0\n",
      "Publication date: None\n",
      "Mantainer: Jakob Wirbel <jakob.wirbel@embl.de>\n",
      "Authors:\n",
      "Author: Konrad Zych [aut] , Jakob Wirbel [aut, cre] , Georg Zeller [aut] , Morgan Essex [ctb], Nicolai Karcher [ctb], Kersten Breuer [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 3.6.0) type: depends\n",
      "  - mlr3, type: depends\n",
      "  - phyloseq, type: depends\n",
      "  - beanplot, type: imports\n",
      "  - glmnet, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - grid, type: imports\n",
      "  - gridBase, type: imports\n",
      "  - gridExtra, type: imports\n",
      "  - LiblineaR, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - methods, type: imports\n",
      "  - pROC, type: imports\n",
      "  - PRROC, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - scales, type: imports\n",
      "  - stats, type: imports\n",
      "  - stringr, type: imports\n",
      "  - utils, type: imports\n",
      "  - infotheo, type: imports\n",
      "  - progress, type: imports\n",
      "  - corrplot, type: imports\n",
      "  - lmerTest, type: imports\n",
      "  - mlr3learners, type: imports\n",
      "  - mlr3tuning, type: imports\n",
      "  - paradox, type: imports\n",
      "  - lgr, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/SIAMCAT.html\n",
      "\n",
      "Name: sigPathway\n",
      "Description:\n",
      "Conducts pathway analysis by calculating the NT_k and NE_k statistics as described in Tian et al. (2005)\n",
      "Version: 1.66.2\n",
      "Publication date: None\n",
      "Mantainer: Weil Lai <wlai@alum.mit.edu>\n",
      "Authors:\n",
      "Author: Weil Lai (optimized R and C code), Lu Tian and Peter Park (algorithm development and initial R code)\n",
      "Dependencies:\n",
      "  - R (>= 2.10) type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/sigPathway.html\n",
      "\n",
      "Name: BrowserViz\n",
      "Description:\n",
      "Interactvive graphics in a web browser from R, using websockets and JSON.\n",
      "Version: 2.20.0\n",
      "Publication date: None\n",
      "Mantainer: Paul Shannon <pshannon@systemsbiology.org>\n",
      "Authors:\n",
      "Author: Paul Shannon\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - jsonlite (>= 1.5) type: depends\n",
      "  - httpuv (>= 1.5.0) type: depends\n",
      "  - methods, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/BrowserViz.html\n",
      "\n",
      "Name: GOSim\n",
      "Description:\n",
      "This package implements several functions useful for computing similarities between GO terms and gene products based on their GO annotation. Moreover it allows for computing a GO enrichment analysis\n",
      "Version: 1.36.0\n",
      "Publication date: None\n",
      "Mantainer: Holger Froehlich <frohlich@bit.uni-bonn.de>\n",
      "Authors:\n",
      "Author: Holger Froehlich <frohlich at bit.uni-bonn.de>\n",
      "Dependencies:\n",
      "  - GO.db, type: depends\n",
      "  - annotate, type: depends\n",
      "  - org.Hs.eg.db, type: imports\n",
      "  - AnnotationDbi, type: imports\n",
      "  - topGO, type: imports\n",
      "  - cluster, type: imports\n",
      "  - flexmix, type: imports\n",
      "  - RBGL, type: imports\n",
      "  - graph, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - corpcor, type: imports\n",
      "  - Rcpp, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/GOSim.html\n",
      "\n",
      "Name: matter\n",
      "Description:\n",
      "Memory-efficient reading, writing, and manipulation of structured binary data as file-based vectors, matrices, arrays, lists, and data frames.\n",
      "Version: 2.0.1\n",
      "Publication date: None\n",
      "Mantainer: Kylie A. Bemis <k.bemis@northeastern.edu>\n",
      "Authors:\n",
      "Author: Kylie A. Bemis <k.bemis at northeastern.edu>\n",
      "Dependencies:\n",
      "  - R (>= 3.5) type: depends\n",
      "  - BiocParallel, type: depends\n",
      "  - Matrix, type: depends\n",
      "  - methods, type: depends\n",
      "  - stats, type: depends\n",
      "  - BiocGenerics, type: imports\n",
      "  - ProtGenerics, type: imports\n",
      "  - digest, type: imports\n",
      "  - irlba, type: imports\n",
      "  - biglm, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/matter.html\n",
      "\n",
      "Name: MethylMix\n",
      "Description:\n",
      "MethylMix is an algorithm implemented to identify hyper and hypomethylated genes for a disease. MethylMix is based on a beta mixture model to identify methylation states and compares them with the normal DNA methylation state. MethylMix uses a novel statistic, the Differential Methylation value or DM-value defined as the difference of a methylation state with the normal methylation state. Finally, matched gene expression data is used to identify, besides differential, functional methylation states by focusing on methylation changes that effect gene expression. References: Gevaert 0. MethylMix: an R package for identifying DNA methylation-driven genes. Bioinformatics (Oxford, England). 2015;31(11):1839-41. doi:10.1093/bioinformatics/btv020. Gevaert O, Tibshirani R, Plevritis SK. Pancancer analysis of DNA methylation-driven genes using MethylMix. Genome Biology. 2015;16(1):17. doi:10.1186/s13059-014-0579-8.\n",
      "Version: 2.28.0\n",
      "Publication date: None\n",
      "Mantainer: Olivier Gevaert <olivier.gevaert@gmail.com>\n",
      "Authors:\n",
      "Author: Olivier Gevaert\n",
      "Dependencies:\n",
      "  - R (>= 3.2.0) type: depends\n",
      "  - foreach, type: imports\n",
      "  - RPMM, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - RCurl, type: imports\n",
      "  - impute, type: imports\n",
      "  - data.table, type: imports\n",
      "  - limma, type: imports\n",
      "  - R.matlab, type: imports\n",
      "  - digest, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/MethylMix.html\n",
      "\n",
      "Name: Rcpi\n",
      "Description:\n",
      "A molecular informatics toolkit with an integration of bioinformatics and chemoinformatics tools for drug discovery.\n",
      "Version: 1.34.0\n",
      "Publication date: None\n",
      "Mantainer: Nan Xiao <me@nanx.me>\n",
      "Authors:\n",
      "Author: Nan Xiao [aut, cre], Dong-Sheng Cao [aut], Qing-Song Xu [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.0.2) type: depends\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - methods, type: imports\n",
      "  - RCurl, type: imports\n",
      "  - rjson, type: imports\n",
      "  - foreach, type: imports\n",
      "  - doParallel, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - GOSemSim, type: imports\n",
      "  - rcdk (>= 3.3.8) type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0 | file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/Rcpi.html\n",
      "\n",
      "Name: TCseq\n",
      "Description:\n",
      "Quantitative and differential analysis of epigenomic and transcriptomic time course sequencing data, clustering analysis and visualization of temporal patterns of time course data.\n",
      "Version: 1.22.6\n",
      "Publication date: None\n",
      "Mantainer: Mengjun Wu <minervajunjun@gmail.com>\n",
      "Authors:\n",
      "Author: Mengjun Wu <minervajunjun at gmail.com>, Lei Gu <leigu at broadinstitute.org>\n",
      "Dependencies:\n",
      "  - R (>= 3.4) type: depends\n",
      "  - edgeR, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - GenomicAlignments, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - e1071, type: imports\n",
      "  - cluster, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - grid, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - methods, type: imports\n",
      "  - locfit, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/TCseq.html\n",
      "\n",
      "Name: polyester\n",
      "Description:\n",
      "This package can be used to simulate RNA-seq reads from differential expression experiments with replicates. The reads can then be aligned and used to perform comparisons of methods for differential expression.\n",
      "Version: 1.34.0\n",
      "Publication date: None\n",
      "Mantainer: Jack Fu <jmfu@jhsph.edu>, Jeff Leek <jtleek@gmail.com>\n",
      "Authors:\n",
      "Author: Alyssa C. Frazee, Andrew E. Jaffe, Rory Kirchner, Jeffrey T. Leek\n",
      "Dependencies:\n",
      "  - R (>= 3.0.0) type: depends\n",
      "  - Biostrings, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - logspline, type: imports\n",
      "  - limma, type: imports\n",
      "  - zlibbioc, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/polyester.html\n",
      "\n",
      "Name: RTN\n",
      "Description:\n",
      "A transcriptional regulatory network (TRN) consists of a collection of transcription factors (TFs) and the regulated target genes. TFs are regulators that recognize specific DNA sequences and guide the expression of the genome, either activating or repressing the expression the target genes. The set of genes controlled by the same TF forms a regulon. This package provides classes and methods for the reconstruction of TRNs and analysis of regulons.\n",
      "Version: 2.22.1\n",
      "Publication date: None\n",
      "Mantainer: Mauro Castro <mauro.a.castro@gmail.com>\n",
      "Authors:\n",
      "Author: Clarice Groeneveld [ctb], Gordon Robertson [ctb], Xin Wang [aut], Michael Fletcher [aut], Florian Markowetz [aut], Kerstin Meyer [aut], and Mauro Castro [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.6.3) type: depends\n",
      "  - methods, type: depends\n",
      "  - RedeR, type: imports\n",
      "  - minet, type: imports\n",
      "  - viper, type: imports\n",
      "  - mixtools, type: imports\n",
      "  - snow, type: imports\n",
      "  - stats, type: imports\n",
      "  - limma, type: imports\n",
      "  - data.table, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - igraph, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - car, type: imports\n",
      "  - pwr, type: imports\n",
      "  - pheatmap, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/RTN.html\n",
      "\n",
      "Name: Cardinal\n",
      "Description:\n",
      "Implements statistical & computational tools for analyzing mass spectrometry imaging datasets, including methods for efficient pre-processing, spatial segmentation, and classification.\n",
      "Version: 3.0.1\n",
      "Publication date: None\n",
      "Mantainer: Kylie A. Bemis <k.bemis@northeastern.edu>\n",
      "Authors:\n",
      "Author: Kylie A. Bemis <k.bemis at northeastern.edu>\n",
      "Dependencies:\n",
      "  - ProtGenerics, type: depends\n",
      "  - BiocGenerics, type: depends\n",
      "  - BiocParallel, type: depends\n",
      "  - EBImage, type: depends\n",
      "  - graphics, type: depends\n",
      "  - methods, type: depends\n",
      "  - S4Vectors, type: depends\n",
      "  - stats, type: depends\n",
      "  - Biobase, type: imports\n",
      "  - irlba, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - matter, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - mclust, type: imports\n",
      "  - nlme, type: imports\n",
      "  - parallel, type: imports\n",
      "  - signal, type: imports\n",
      "  - sp, type: imports\n",
      "  - stats4, type: imports\n",
      "  - utils, type: imports\n",
      "  - viridisLite, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/Cardinal.html\n",
      "\n",
      "Name: MetaboAnnotation\n",
      "Description:\n",
      "High level functions to assist in annotation of (metabolomics) data sets. These include functions to perform simple tentative annotations based on mass matching but also functions to consider m/z and retention times for annotation of LC-MS features given that respective reference values are available. In addition, the function provides high-level functions to simplify matching of LC-MS/MS spectra against spectral libraries and objects and functionality to represent and manage such matched data.\n",
      "Version: 1.2.0\n",
      "Publication date: None\n",
      "Mantainer: Johannes Rainer <Johannes.Rainer@eurac.edu>\n",
      "Authors:\n",
      "Author: Michael Witting [aut] , Johannes Rainer [aut, cre] , Andrea Vicini [aut] , Carolin Huber [aut] , Nir Shachaf [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 4.0.0) type: depends\n",
      "  - BiocGenerics, type: imports\n",
      "  - MsCoreUtils, type: imports\n",
      "  - MetaboCoreUtils, type: imports\n",
      "  - ProtGenerics, type: imports\n",
      "  - methods, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - Spectra, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - QFeatures, type: imports\n",
      "  - graphics, type: imports\n",
      "  - CompoundDb, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/MetaboAnnotation.html\n",
      "\n",
      "Name: miRNAtap\n",
      "Description:\n",
      "The package facilitates implementation of workflows requiring miRNA predictions, it allows to integrate ranked miRNA target predictions from multiple sources available online and aggregate them with various methods which improves quality of predictions above any of the single sources. Currently predictions are available for Homo sapiens, Mus musculus and Rattus norvegicus (the last one through homology translation).\n",
      "Version: 1.32.0\n",
      "Publication date: None\n",
      "Mantainer: T. Ian Simpson <ian.simpson@ed.ac.uk>\n",
      "Authors:\n",
      "Author: Maciej Pajak, T. Ian Simpson\n",
      "Dependencies:\n",
      "  - R (>= 3.3.0) type: depends\n",
      "  - AnnotationDbi, type: depends\n",
      "  - DBI, type: imports\n",
      "  - RSQLite, type: imports\n",
      "  - stringr, type: imports\n",
      "  - sqldf, type: imports\n",
      "  - plyr, type: imports\n",
      "  - methods, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/miRNAtap.html\n",
      "\n",
      "Name: mogsa\n",
      "Description:\n",
      "This package provide a method for doing gene set analysis based on multiple omics data.\n",
      "Version: 1.32.0\n",
      "Publication date: None\n",
      "Mantainer: Chen Meng <mengchen18@gmail.com>\n",
      "Authors:\n",
      "Author: Chen Meng\n",
      "Dependencies:\n",
      "  - R (>= 3.4.0) type: depends\n",
      "  - methods, type: imports\n",
      "  - graphite, type: imports\n",
      "  - genefilter, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - gplots, type: imports\n",
      "  - GSEABase, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - parallel, type: imports\n",
      "  - corpcor, type: imports\n",
      "  - svd, type: imports\n",
      "  - cluster, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/mogsa.html\n",
      "\n",
      "Name: BASiCS\n",
      "Description:\n",
      "Single-cell mRNA sequencing can uncover novel cell-to-cell heterogeneity in gene expression levels in seemingly homogeneous populations of cells. However, these experiments are prone to high levels of technical noise, creating new challenges for identifying genes that show genuine heterogeneous expression within the population of cells under study. BASiCS (Bayesian Analysis of Single-Cell Sequencing data) is an integrated Bayesian hierarchical model to perform statistical analyses of single-cell RNA sequencing datasets in the context of supervised experiments (where the groups of cells of interest are known a priori, e.g. experimental conditions or cell types). BASiCS performs built-in data normalisation (global scaling) and technical noise quantification (based on spike-in genes). BASiCS provides an intuitive detection criterion for highly (or lowly) variable genes within a single group of cells. Additionally, BASiCS can compare gene expression patterns between two or more pre-specified groups of cells. Unlike traditional differential expression tools, BASiCS quantifies changes in expression that lie beyond comparisons of means, also allowing the study of changes in cell-to-cell heterogeneity. The latter can be quantified via a biological over-dispersion parameter that measures the excess of variability that is observed with respect to Poisson sampling noise, after normalisation and technical noise removal. Due to the strong mean/over-dispersion confounding that is typically observed for scRNA-seq datasets, BASiCS also tests for changes in residual over-dispersion, defined by residual values with respect to a global mean/over-dispersion trend.\n",
      "Version: 2.10.5\n",
      "Publication date: None\n",
      "Mantainer: Catalina Vallejos <catalina.vallejos@igmm.ed.ac.uk>\n",
      "Authors:\n",
      "Author: Catalina Vallejos [aut, cre], Nils Eling [aut], Alan O'Callaghan [aut], Sylvia Richardson [ctb], John Marioni [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - SingleCellExperiment, type: depends\n",
      "  - Biobase, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - coda, type: imports\n",
      "  - cowplot, type: imports\n",
      "  - ggExtra, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - MASS, type: imports\n",
      "  - methods, type: imports\n",
      "  - Rcpp (>= 0.11.3) type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - scran, type: imports\n",
      "  - scuttle, type: imports\n",
      "  - stats, type: imports\n",
      "  - stats4, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - viridis, type: imports\n",
      "  - utils, type: imports\n",
      "  - Matrix (>= 1.5.0) type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - assertthat, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - hexbin, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/BASiCS.html\n",
      "\n",
      "Name: CelliD\n",
      "Description:\n",
      "CelliD is a clustering-free multivariate statistical method for the robust extraction of per-cell gene signatures from single-cell RNA-seq. CelliD allows unbiased cell identity recognition across different donors, tissues-of-origin, model organisms and single-cell omics protocols. The package can also be used to explore functional pathways enrichment in single cell data.\n",
      "Version: 1.6.2\n",
      "Publication date: None\n",
      "Mantainer: Akira Cortal <akira.cortal@institutimagine.org>\n",
      "Authors:\n",
      "Author: Akira Cortal [aut, cre], Antonio Rausell [aut, ctb]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - Seurat (>= 4.0.1) type: depends\n",
      "  - SingleCellExperiment, type: depends\n",
      "  - Rcpp, type: imports\n",
      "  - RcppArmadillo, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - tictoc, type: imports\n",
      "  - scater, type: imports\n",
      "  - stringr, type: imports\n",
      "  - irlba, type: imports\n",
      "  - data.table, type: imports\n",
      "  - glue, type: imports\n",
      "  - pbapply, type: imports\n",
      "  - umap, type: imports\n",
      "  - Rtsne, type: imports\n",
      "  - reticulate, type: imports\n",
      "  - fastmatch, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - fgsea, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3 + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/CelliD.html\n",
      "\n",
      "Name: GeneTonic\n",
      "Description:\n",
      "This package provides a Shiny application that aims to combine at different levels the existing pieces of the transcriptome data and results, in a way that makes it easier to generate insightful observations and hypothesis - combining the benefits of interactivity and reproducibility, e.g. by capturing the features and gene sets of interest highlighted during the live session, and creating an HTML report as an artifact where text, code, and output coexist.\n",
      "Version: 2.2.0\n",
      "Publication date: None\n",
      "Mantainer: Federico Marini <marinif@uni-mainz.de>\n",
      "Authors:\n",
      "Author: Federico Marini [aut, cre] , Annekathrin Ludt [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.0.0) type: depends\n",
      "  - AnnotationDbi, type: imports\n",
      "  - backbone, type: imports\n",
      "  - bs4Dash (>= 2.0.0) type: imports\n",
      "  - circlize, type: imports\n",
      "  - colorspace, type: imports\n",
      "  - colourpicker, type: imports\n",
      "  - ComplexHeatmap, type: imports\n",
      "  - ComplexUpset, type: imports\n",
      "  - dendextend, type: imports\n",
      "  - DESeq2, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - DT, type: imports\n",
      "  - dynamicTreeCut, type: imports\n",
      "  - expm, type: imports\n",
      "  - ggforce, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - ggrepel, type: imports\n",
      "  - GO.db, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - grid, type: imports\n",
      "  - igraph, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - methods, type: imports\n",
      "  - plotly, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - rintrojs, type: imports\n",
      "  - rlang, type: imports\n",
      "  - rmarkdown, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - scales, type: imports\n",
      "  - shiny, type: imports\n",
      "  - shinyAce, type: imports\n",
      "  - shinycssloaders, type: imports\n",
      "  - shinyWidgets, type: imports\n",
      "  - stats, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - tippy, type: imports\n",
      "  - tools, type: imports\n",
      "  - utils, type: imports\n",
      "  - viridis, type: imports\n",
      "  - visNetwork, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/GeneTonic.html\n",
      "\n",
      "Name: agilp\n",
      "Description:\n",
      "More about what it does (maybe more than one line)\n",
      "Version: 3.30.0\n",
      "Publication date: None\n",
      "Mantainer: Benny Chain <b.chain@ucl.ac.uk>\n",
      "Authors:\n",
      "Author: Benny Chain <b.chain at ucl.ac.uk>\n",
      "Dependencies:\n",
      "  - R (>= 2.14.0) type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/agilp.html\n",
      "\n",
      "Name: BatchQC\n",
      "Description:\n",
      "Sequencing and microarray samples often are collected or processed in multiple batches or at different times. This often produces technical biases that can lead to incorrect results in the downstream analysis. BatchQC is a software tool that streamlines batch preprocessing and evaluation by providing interactive diagnostics, visualizations, and statistical analyses to explore the extent to which batch variation impacts the data. BatchQC diagnostics help determine whether batch adjustment needs to be done, and how correction should be applied before proceeding with a downstream analysis. Moreover, BatchQC interactively applies multiple common batch effect approaches to the data, and the user can quickly see the benefits of each method. BatchQC is developed as a Shiny App. The output is organized into multiple tabs, and each tab features an important part of the batch effect analysis and visualization of the data. The BatchQC interface has the following analysis groups: Summary, Differential Expression, Median Correlations, Heatmaps, Circular Dendrogram, PCA Analysis, Shape, ComBat and SVA.\n",
      "Version: 1.26.0\n",
      "Publication date: None\n",
      "Mantainer: Solaiappan Manimaran <manimaran_1975@hotmail.com>\n",
      "Authors:\n",
      "Author: Solaiappan Manimaran <manimaran_1975 at hotmail.com>, W. Evan Johnson <wej at bu.edu>, Heather Selby <selbyh at bu.edu>, Claire Ruberman <claireruberman at gmail.com>, Kwame Okrah <kwame.okrah at gmail.com>, Hector Corrada Bravo <hcorrada at gmail.com>\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - utils, type: imports\n",
      "  - rmarkdown, type: imports\n",
      "  - knitr, type: imports\n",
      "  - pander, type: imports\n",
      "  - gplots, type: imports\n",
      "  - MCMCpack, type: imports\n",
      "  - shiny, type: imports\n",
      "  - sva, type: imports\n",
      "  - corpcor, type: imports\n",
      "  - moments, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - ggvis, type: imports\n",
      "  - heatmaply, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - limma, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - methods, type: imports\n",
      "  - Matrix, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/BatchQC.html\n",
      "\n",
      "Name: crlmm\n",
      "Description:\n",
      "Faster implementation of CRLMM specific to SNP 5.0 and 6.0 arrays, as well as a copy number tool specific to 5.0, 6.0, and Illumina platforms.\n",
      "Version: 1.56.0\n",
      "Publication date: None\n",
      "Mantainer: Benilton S Carvalho <benilton@unicamp.br>, Robert Scharpf <rscharpf@jhsph.edu>, Matt Ritchie <mritchie@wehi.EDU.AU>\n",
      "Authors:\n",
      "Author: Benilton S Carvalho, Robert Scharpf, Matt Ritchie, Ingo Ruczinski, Rafael A Irizarry\n",
      "Dependencies:\n",
      "  - R (>= 2.14.0) type: depends\n",
      "  - oligoClasses, type: depends\n",
      "  - preprocessCore, type: depends\n",
      "  - methods, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - affyio, type: imports\n",
      "  - illuminaio, type: imports\n",
      "  - ellipse, type: imports\n",
      "  - mvtnorm, type: imports\n",
      "  - splines, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - lattice, type: imports\n",
      "  - ff, type: imports\n",
      "  - foreach, type: imports\n",
      "  - RcppEigen (>= 0.3.1.2.1) type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - VGAM, type: imports\n",
      "  - parallel, type: imports\n",
      "  - graphics, type: imports\n",
      "  - limma, type: imports\n",
      "  - beanplot, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/crlmm.html\n",
      "\n",
      "Name: FELLA\n",
      "Description:\n",
      "Enrichment of metabolomics data using KEGG entries. Given a set of affected compounds, FELLA suggests affected reactions, enzymes, modules and pathways using label propagation in a knowledge model network. The resulting subnetwork can be visualised and exported.\n",
      "Version: 1.18.0\n",
      "Publication date: None\n",
      "Mantainer: Sergio Picart-Armada <sergi.picart@upc.edu>\n",
      "Authors:\n",
      "Author: Sergio Picart-Armada [aut, cre], Francesc Fernandez-Albert [aut], Alexandre Perera-Lluna [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - methods, type: imports\n",
      "  - igraph, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - KEGGREST, type: imports\n",
      "  - plyr, type: imports\n",
      "  - stats, type: imports\n",
      "  - graphics, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/FELLA.html\n",
      "\n",
      "Name: makecdfenv\n",
      "Description:\n",
      "This package has two functions. One reads a Affymetrix chip description file (CDF) and creates a hash table environment containing the location/probe set membership mapping. The other creates a package that automatically loads that environment.\n",
      "Version: 1.74.0\n",
      "Publication date: None\n",
      "Mantainer: James W. MacDonald <jmacdon@u.washington.edu>\n",
      "Authors:\n",
      "Author: Rafael A. Irizarry <rafa at jhu.edu>, Laurent Gautier <laurent at cbs.dtu.dk>, Wolfgang Huber <w.huber at dkfz-heidelberg.de>, Ben Bolstad <bmb at bmbolstad.com>\n",
      "Dependencies:\n",
      "  - R (>= 2.6.0) type: depends\n",
      "  - affyio, type: depends\n",
      "  - Biobase, type: imports\n",
      "  - affy, type: imports\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - zlibbioc, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/makecdfenv.html\n",
      "\n",
      "Name: a4Preproc\n",
      "Description:\n",
      "Utility functions to pre-process data for the Automated Affymetrix Array Analysis set of packages.\n",
      "Version: 1.46.0\n",
      "Publication date: None\n",
      "Mantainer: Laure Cougnaud <laure.cougnaud@openanalytics.eu>\n",
      "Authors:\n",
      "Author: Willem Talloen [aut], Tobias Verbeke [aut], Laure Cougnaud [cre]\n",
      "Dependencies:\n",
      "  - BiocGenerics, type: imports\n",
      "  - Biobase, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/a4Preproc.html\n",
      "\n",
      "Name: cellxgenedp\n",
      "Description:\n",
      "The cellxgene data portal (https://cellxgene.cziscience.com/) provides a graphical user interface to collections of single-cell sequence data processed in standard ways to 'count matrix' summaries. The cellxgenedp package provides an alternative, R-based inteface, allowind data discovery, viewing, and downloading.\n",
      "Version: 1.2.2\n",
      "Publication date: None\n",
      "Mantainer: Martin Morgan <mtmorgan.bioc@gmail.com>\n",
      "Authors:\n",
      "Author: Martin Morgan [aut, cre] , Kayla Interdonato [aut]\n",
      "Dependencies:\n",
      "  - dplyr, type: depends\n",
      "  - httr, type: imports\n",
      "  - curl, type: imports\n",
      "  - jsonlite, type: imports\n",
      "  - utils, type: imports\n",
      "  - tools, type: imports\n",
      "  - shiny, type: imports\n",
      "  - DT, type: imports\n",
      "  - rjsoncons, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/cellxgenedp.html\n",
      "\n",
      "Name: FRASER\n",
      "Description:\n",
      "Detection of rare aberrant splicing events in transcriptome profiles. Read count ratio expectations are modeled by an autoencoder to control for confounding factors in the data. Given these expectations, the ratios are assumed to follow a beta-binomial distribution with a junction specific dispersion. Outlier events are then identified as read-count ratios that deviate significantly from this distribution. FRASER is able to detect alternative splicing, but also intron retention. The package aims to support diagnostics in the field of rare diseases where RNA-seq is performed to identify aberrant splicing defects.\n",
      "Version: 1.10.2\n",
      "Publication date: None\n",
      "Mantainer: Christian Mertes <mertes@in.tum.de>\n",
      "Authors:\n",
      "Author: Christian Mertes [aut, cre], Ines Scheller [aut], Vicente Yepez [ctb], Julien Gagneur [aut]\n",
      "Dependencies:\n",
      "  - BiocParallel, type: depends\n",
      "  - data.table, type: depends\n",
      "  - Rsamtools, type: depends\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - AnnotationDbi, type: imports\n",
      "  - BBmisc, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - biomaRt, type: imports\n",
      "  - BSgenome, type: imports\n",
      "  - cowplot, type: imports\n",
      "  - DelayedArray, type: imports\n",
      "  - DelayedMatrixStats, type: imports\n",
      "  - extraDistr, type: imports\n",
      "  - generics, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - GenomicAlignments, type: imports\n",
      "  - GenomicFeatures, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - ggrepel, type: imports\n",
      "  - HDF5Array, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - methods, type: imports\n",
      "  - OUTRIDER, type: imports\n",
      "  - pcaMethods, type: imports\n",
      "  - pheatmap, type: imports\n",
      "  - plotly, type: imports\n",
      "  - PRROC, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - rhdf5, type: imports\n",
      "  - Rsubread, type: imports\n",
      "  - R.utils, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - stats, type: imports\n",
      "  - tibble, type: imports\n",
      "  - tools, type: imports\n",
      "  - utils, type: imports\n",
      "  - VGAM, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/FRASER.html\n",
      "\n",
      "Name: lipidr\n",
      "Description:\n",
      "lipidr an easy-to-use R package implementing a complete workflow for downstream analysis of targeted and untargeted lipidomics data. lipidomics results can be imported into lipidr as a numerical matrix or a Skyline export, allowing integration into current analysis frameworks. Data mining of lipidomics datasets is enabled through integration with Metabolomics Workbench API. lipidr allows data inspection, normalization, univariate and multivariate analysis, displaying informative visualizations. lipidr also implements a novel Lipid Set Enrichment Analysis (LSEA), harnessing molecular information such as lipid class, total chain length and unsaturation.\n",
      "Version: 2.12.0\n",
      "Publication date: None\n",
      "Mantainer: Ahmed Mohamed <mohamed@kuicr.kyoto-u.ac.jp>\n",
      "Authors:\n",
      "Author: Ahmed Mohamed [cre] , Ahmed Mohamed [aut], Jeffrey Molendijk [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.6.0) type: depends\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - data.table, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - rlang, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - forcats, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - limma, type: imports\n",
      "  - fgsea, type: imports\n",
      "  - ropls, type: imports\n",
      "  - imputeLCMD, type: imports\n",
      "  - magrittr, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/lipidr.html\n",
      "\n",
      "Name: BHC\n",
      "Description:\n",
      "The method performs bottom-up hierarchical clustering, using a Dirichlet Process (infinite mixture) to model uncertainty in the data and Bayesian model selection to decide at each step which clusters to merge. This avoids several limitations of traditional methods, for example how many clusters there should be and how to choose a principled distance metric. This implementation accepts multinomial (i.e. discrete, with 2+ categories) or time-series data. This version also includes a randomised algorithm which is more efficient for larger data sets.\n",
      "Version: 1.50.0\n",
      "Publication date: None\n",
      "Mantainer: Rich Savage <r.s.savage@warwick.ac.uk>\n",
      "Authors:\n",
      "Author: Rich Savage, Emma Cooke, Robert Darkins, Yang Xu\n",
      "Dependencies:\n",
      "None\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/BHC.html\n",
      "\n",
      "Name: BioQC\n",
      "Description:\n",
      "BioQC performs quality control of high-throughput expression data based on tissue gene signatures. It can detect tissue heterogeneity in gene expression data. The core algorithm is a Wilcoxon-Mann-Whitney test that is optimised for high performance.\n",
      "Version: 1.26.0\n",
      "Publication date: None\n",
      "Mantainer: Jitao David Zhang <jitao_david.zhang@roche.com>\n",
      "Authors:\n",
      "Author: Jitao David Zhang [cre, aut], Laura Badi [aut], Gregor Sturm [aut], Roland Ambs [aut], Iakov Davydov [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - Biobase, type: depends\n",
      "  - edgeR, type: imports\n",
      "  - Rcpp, type: imports\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>=3) + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/BioQC.html\n",
      "\n",
      "Name: flowPeaks\n",
      "Description:\n",
      "A fast and automatic clustering to classify the cells into subpopulations based on finding the peaks from the overall density function generated by K-means.\n",
      "Version: 1.44.0\n",
      "Publication date: None\n",
      "Mantainer: Yongchao Ge<yongchao.ge@gmail.com>\n",
      "Authors:\n",
      "Author: Yongchao Ge<yongchao.ge at gmail.com>\n",
      "Dependencies:\n",
      "  - R (>= 2.12.0) type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-1.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/flowPeaks.html\n",
      "\n",
      "Name: GSAR\n",
      "Description:\n",
      "Gene set analysis using specific alternative hypotheses. Tests for differential expression, scale and net correlation structure.\n",
      "Version: 1.32.1\n",
      "Publication date: None\n",
      "Mantainer: Yasir Rahmatallah <yrahmatallah@uams.edu>, Galina Glazko <gvglazko@uams.edu>\n",
      "Authors:\n",
      "Author: Yasir Rahmatallah <yrahmatallah at uams.edu>, Galina Glazko <gvglazko at uams.edu>\n",
      "Dependencies:\n",
      "  - R (>= 3.0.1) type: depends\n",
      "  - igraph (>= 0.7.1) type: depends\n",
      "  - stats, type: imports\n",
      "  - graphics, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>=2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/GSAR.html\n",
      "\n",
      "Name: CRISPRseek\n",
      "Description:\n",
      "The package includes functions to find potential guide RNAs for the CRISPR editing system including Base Editors and the Prime Editor for input target sequences, optionally filter guide RNAs without restriction enzyme cut site, or without paired guide RNAs, genome-wide search for off-targets, score, rank, fetch flank sequence and indicate whether the target and off-targets are located in exon region or not. Potential guide RNAs are annotated with total score of the top5 and topN off-targets, detailed topN mismatch sites, restriction enzyme cut sites, and paired guide RNAs. The package also output indels and their frequencies for Cas9 targeted sites.\n",
      "Version: 1.38.0\n",
      "Publication date: None\n",
      "Mantainer: Lihua Julie Zhu <julie.zhu@umassmed.edu>\n",
      "Authors:\n",
      "Author: Lihua Julie Zhu, Paul Scemama, Benjamin R. Holmes, HervÃ© PagÃ¨s, Kai Hu, Hui Mao, Michael Lawrence, Isana Veksler-Lublinsky, Victor Ambros, Neil Aronin and Michael Brodsky\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - BiocGenerics, type: depends\n",
      "  - Biostrings, type: depends\n",
      "  - parallel, type: imports\n",
      "  - data.table, type: imports\n",
      "  - seqinr, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - BSgenome, type: imports\n",
      "  - hash, type: imports\n",
      "  - methods, type: imports\n",
      "  - reticulate, type: imports\n",
      "  - rhdf5, type: imports\n",
      "  - XVector, type: imports\n",
      "  - DelayedArray, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - keras, type: imports\n",
      "  - mltools, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/CRISPRseek.html\n",
      "\n",
      "Name: flowMeans\n",
      "Description:\n",
      "Identifies cell populations in Flow Cytometry data using non-parametric clustering and segmented-regression-based change point detection. Note: R 2.11.0 or newer is required.\n",
      "Version: 1.58.0\n",
      "Publication date: None\n",
      "Mantainer: Nima Aghaeepour <naghaeep@gmail.com>\n",
      "Authors:\n",
      "Author: Nima Aghaeepour\n",
      "Dependencies:\n",
      "  - R (>= 2.10.0) type: depends\n",
      "  - Biobase, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - methods, type: imports\n",
      "  - rrcov, type: imports\n",
      "  - stats, type: imports\n",
      "  - feature, type: imports\n",
      "  - flowCore, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/flowMeans.html\n",
      "\n",
      "Name: MEDIPS\n",
      "Description:\n",
      "MEDIPS was developed for analyzing data derived from methylated DNA immunoprecipitation (MeDIP) experiments followed by sequencing (MeDIP-seq). However, MEDIPS provides functionalities for the analysis of any kind of quantitative sequencing data (e.g. ChIP-seq, MBD-seq, CMS-seq and others) including calculation of differential coverage between groups of samples and saturation and correlation analysis.\n",
      "Version: 1.50.0\n",
      "Publication date: None\n",
      "Mantainer: Lukas Chavez <lukaschavez@ucsd.edu>\n",
      "Authors:\n",
      "Author: Lukas Chavez, Matthias Lienhard, Joern Dietrich, Isaac Lopez Moyado\n",
      "Dependencies:\n",
      "  - R (>= 3.0) type: depends\n",
      "  - BSgenome, type: depends\n",
      "  - Rsamtools, type: depends\n",
      "  - GenomicRanges, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - graphics, type: imports\n",
      "  - gtools, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - edgeR, type: imports\n",
      "  - DNAcopy, type: imports\n",
      "  - biomaRt, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - preprocessCore, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>=2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/MEDIPS.html\n",
      "\n",
      "Name: NormalyzerDE\n",
      "Description:\n",
      "NormalyzerDE provides screening of normalization methods for LC-MS based expression data. It calculates a range of normalized matrices using both existing approaches and a novel time-segmented approach, calculates performance measures and generates an evaluation report. Furthermore, it provides an easy utility for Limma- or ANOVA- based differential expression analysis.\n",
      "Version: 1.16.0\n",
      "Publication date: None\n",
      "Mantainer: Jakob Willforss <jakob.willforss@hotmail.com>\n",
      "Authors:\n",
      "Author: Jakob Willforss\n",
      "Dependencies:\n",
      "  - R (>= 3.6) type: depends\n",
      "  - vsn, type: imports\n",
      "  - preprocessCore, type: imports\n",
      "  - limma, type: imports\n",
      "  - MASS, type: imports\n",
      "  - ape, type: imports\n",
      "  - car, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - methods, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - RcmdrMisc, type: imports\n",
      "  - raster, type: imports\n",
      "  - utils, type: imports\n",
      "  - stats, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - ggforce, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/NormalyzerDE.html\n",
      "\n",
      "Name: seqTools\n",
      "Description:\n",
      "Analyze read length, phred scores and alphabet frequency and DNA k-mers on uncompressed and compressed fastq files.\n",
      "Version: 1.32.0\n",
      "Publication date: None\n",
      "Mantainer: Wolfgang Kaisers <kaisers@med.uni-duesseldorf.de>\n",
      "Authors:\n",
      "Author: Wolfgang Kaisers\n",
      "Dependencies:\n",
      "  - methods, type: depends\n",
      "  - utils, type: depends\n",
      "  - zlibbioc, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/seqTools.html\n",
      "\n",
      "Name: DAPAR\n",
      "Description:\n",
      "The package DAPAR is a Bioconductor distributed R package which provides all the necessary functions to analyze quantitative data from label-free proteomics experiments. Contrarily to most other similar R packages, it is endowed with rich and user-friendly graphical interfaces, so that no programming skill is required (see `Prostar` package).\n",
      "Version: 1.30.5\n",
      "Publication date: None\n",
      "Mantainer: Samuel Wieczorek <samuel.wieczorek@cea.fr>\n",
      "Authors:\n",
      "Author: Samuel Wieczorek [aut, cre], Florence Combes [aut], Thomas Burger [aut], Vasile-Cosmin Lazar [ctb], Enora Fremy [ctb], Helene Borges [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 4.2.0) type: depends\n",
      "  - Biobase, type: imports\n",
      "  - MSnbase, type: imports\n",
      "  - DAPARdata, type: imports\n",
      "  - utils, type: imports\n",
      "  - highcharter, type: imports\n",
      "  - foreach, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/DAPAR.html\n",
      "\n",
      "Name: edge\n",
      "Description:\n",
      "The edge package implements methods for carrying out differential expression analyses of genome-wide gene expression studies. Significance testing using the optimal discovery procedure and generalized likelihood ratio tests (equivalent to F-tests and t-tests) are implemented for general study designs. Special functions are available to facilitate the analysis of common study designs, including time course experiments. Other packages such as snm, sva, and qvalue are integrated in edge to provide a wide range of tools for gene expression analysis.\n",
      "Version: 2.30.0\n",
      "Publication date: None\n",
      "Mantainer: John D. Storey <jstorey@princeton.edu>, Andrew J. Bass <ajbass@princeton.edu>\n",
      "Authors:\n",
      "Author: John D. Storey, Jeffrey T. Leek and Andrew J. Bass\n",
      "Dependencies:\n",
      "  - R (>= 3.1.0) type: depends\n",
      "  - Biobase, type: depends\n",
      "  - methods, type: imports\n",
      "  - splines, type: imports\n",
      "  - sva, type: imports\n",
      "  - snm, type: imports\n",
      "  - qvalue, type: imports\n",
      "  - MASS, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/edge.html\n",
      "\n",
      "Name: Guitar\n",
      "Description:\n",
      "The package is designed for visualization of RNA-related genomic features with respect to the landmarks of RNA transcripts, i.e., transcription starting site, start codon, stop codon and transcription ending site.\n",
      "Version: 2.14.0\n",
      "Publication date: None\n",
      "Mantainer: Jia Meng <jia.meng@xjtlu.edu.cn>\n",
      "Authors:\n",
      "Author: Xiao Du, Hui Liu, Lin Zhang, Jia Meng\n",
      "Dependencies:\n",
      "  - GenomicFeatures, type: depends\n",
      "  - rtracklayer, type: depends\n",
      "  - AnnotationDbi, type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - magrittr, type: depends\n",
      "  - ggplot2, type: depends\n",
      "  - methods, type: depends\n",
      "  - stats, type: depends\n",
      "  - utils, type: depends\n",
      "  - knitr, type: depends\n",
      "  - dplyr, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/Guitar.html\n",
      "\n",
      "Name: ontoProc\n",
      "Description:\n",
      "Support harvesting of diverse bioinformatic ontologies, making particular use of the ontologyIndex package on CRAN. We provide snapshots of key ontologies for terms about cells, cell lines, chemical compounds, and anatomy, to help analyze genome-scale experiments, particularly cell x compound screens. Another purpose is to strengthen development of compelling use cases for richer interfaces to emerging ontologies.\n",
      "Version: 1.20.0\n",
      "Publication date: None\n",
      "Mantainer: Vincent Carey <stvjc@channing.harvard.edu>\n",
      "Authors:\n",
      "Author: Vincent Carey [ctb, cre] , Sara Stankiewicz [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - ontologyIndex, type: depends\n",
      "  - Biobase, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - BiocFileCache, type: imports\n",
      "  - shiny, type: imports\n",
      "  - graph, type: imports\n",
      "  - Rgraphviz, type: imports\n",
      "  - ontologyPlot, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - DT, type: imports\n",
      "  - igraph, type: imports\n",
      "  - AnnotationHub, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ontoProc.html\n",
      "\n",
      "Name: SamSPECTRAL\n",
      "Description:\n",
      "Samples large data such that spectral clustering is possible while preserving density information in edge weights. More specifically, given a matrix of coordinates as input, SamSPECTRAL first builds the communities to sample the data points. Then, it builds a graph and after weighting the edges by conductance computation, the graph is passed to a classic spectral clustering algorithm to find the spectral clusters. The last stage of SamSPECTRAL is to combine the spectral clusters. The resulting \"connected components\" estimate biological cell populations in the data. See the vignette for more details on how to use this package, some illustrations, and simple examples.\n",
      "Version: 1.52.0\n",
      "Publication date: None\n",
      "Mantainer: Habil <zare@u.washington.edu>\n",
      "Authors:\n",
      "Author: Habil Zare and Parisa Shooshtari\n",
      "Dependencies:\n",
      "  - R (>= 3.3.3) type: depends\n",
      "  - methods, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/SamSPECTRAL.html\n",
      "\n",
      "Name: tidySingleCellExperiment\n",
      "Description:\n",
      "tidySingleCellExperiment is an adapter that abstracts the 'SingleCellExperiment' container in the form of tibble and allows the data manipulation, plotting and nesting using 'tidyverse'\n",
      "Version: 1.8.0\n",
      "Publication date: None\n",
      "Mantainer: Stefano Mangiola <mangiolastefano@gmail.com>\n",
      "Authors:\n",
      "Author: Stefano Mangiola [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.0.0) type: depends\n",
      "  - ttservice, type: depends\n",
      "  - SingleCellExperiment, type: depends\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - tibble, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - plotly, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - rlang, type: imports\n",
      "  - purrr, type: imports\n",
      "  - lifecycle, type: imports\n",
      "  - methods, type: imports\n",
      "  - utils, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - tidyselect, type: imports\n",
      "  - ellipsis, type: imports\n",
      "  - pillar, type: imports\n",
      "  - stringr, type: imports\n",
      "  - cli, type: imports\n",
      "  - fansi, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/tidySingleCellExperiment.html\n",
      "\n",
      "Name: arrayQuality\n",
      "Description:\n",
      "Functions for performing print-run and array level quality assessment.\n",
      "Version: 1.76.0\n",
      "Publication date: None\n",
      "Mantainer: Agnes Paquet <paquetagnes@yahoo.com>\n",
      "Authors:\n",
      "Author: Agnes Paquet and Jean Yee Hwa Yang <yeehwa at stat.berkeley.edu>\n",
      "Dependencies:\n",
      "  - R (>= 2.2.0) type: depends\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - grid, type: imports\n",
      "  - gridBase, type: imports\n",
      "  - hexbin, type: imports\n",
      "  - limma, type: imports\n",
      "  - marray, type: imports\n",
      "  - methods, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/arrayQuality.html\n",
      "\n",
      "Name: BRAIN\n",
      "Description:\n",
      "Package for calculating aggregated isotopic distribution and exact center-masses for chemical substances (in this version composed of C, H, N, O and S). This is an implementation of the BRAIN algorithm described in the paper by J. Claesen, P. Dittwald, T. Burzykowski and D. Valkenborg.\n",
      "Version: 1.44.0\n",
      "Publication date: None\n",
      "Mantainer: Piotr Dittwald <piotr.dittwald@mimuw.edu.pl>\n",
      "Authors:\n",
      "Author: Piotr Dittwald, with contributions of Dirk Valkenborg and Jurgen Claesen\n",
      "Dependencies:\n",
      "  - R (>= 2.8.1) type: depends\n",
      "  - PolynomF, type: depends\n",
      "  - Biostrings, type: depends\n",
      "  - lattice, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/BRAIN.html\n",
      "\n",
      "Name: ccrepe\n",
      "Description:\n",
      "The CCREPE (Compositionality Corrected by REnormalizaion and PErmutation) package is designed to assess the significance of general similarity measures in compositional datasets. In microbial abundance data, for example, the total abundances of all microbes sum to one; CCREPE is designed to take this constraint into account when assigning p-values to similarity measures between the microbes. The package has two functions: ccrepe: Calculates similarity measures, p-values and q-values for relative abundances of bugs in one or two body sites using bootstrap and permutation matrices of the data. nc.score: Calculates species-level co-variation and co-exclusion patterns based on an extension of the checkerboard score to ordinal data.\n",
      "Version: 1.34.0\n",
      "Publication date: None\n",
      "Mantainer: Emma Schwager <emma.schwager@gmail.com>,Craig Bielski<craig.bielski@gmail.com>, George Weingart<george.weingart@gmail.com>\n",
      "Authors:\n",
      "Author: Emma Schwager <emh146 at mail.harvard.edu>,Craig Bielski<craig.bielski at gmail.com>, George Weingart<george.weingart at gmail.com>\n",
      "Dependencies:\n",
      "  - infotheo (>= 1.1) type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ccrepe.html\n",
      "\n",
      "Name: distinct\n",
      "Description:\n",
      "distinct is a statistical method to perform differential testing between two or more groups of distributions; differential testing is performed via hierarchical non-parametric permutation tests on the cumulative distribution functions (cdfs) of each sample. While most methods for differential expression target differences in the mean abundance between conditions, distinct, by comparing full cdfs, identifies, both, differential patterns involving changes in the mean, as well as more subtle variations that do not involve the mean (e.g., unimodal vs. bi-modal distributions with the same mean). distinct is a general and flexible tool: due to its fully non-parametric nature, which makes no assumptions on how the data was generated, it can be applied to a variety of datasets. It is particularly suitable to perform differential state analyses on single cell data (i.e., differential analyses within sub-populations of cells), such as single cell RNA sequencing (scRNA-seq) and high-dimensional flow or mass cytometry (HDCyto) data. To use distinct one needs data from two or more groups of samples (i.e., experimental conditions), with at least 2 samples (i.e., biological replicates) per group.\n",
      "Version: 1.10.2\n",
      "Publication date: None\n",
      "Mantainer: Simone Tiberi <simone.tiberi@uzh.ch>\n",
      "Authors:\n",
      "Author: Simone Tiberi [aut, cre].\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - Rcpp, type: imports\n",
      "  - Rfast, type: imports\n",
      "  - stats, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - SingleCellExperiment, type: imports\n",
      "  - methods, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - foreach, type: imports\n",
      "  - parallel, type: imports\n",
      "  - doParallel, type: imports\n",
      "  - doRNG, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - limma, type: imports\n",
      "  - scater, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 3)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/distinct.html\n",
      "\n",
      "Name: methylGSA\n",
      "Description:\n",
      "The main functions for methylGSA are methylglm and methylRRA. methylGSA implements logistic regression adjusting number of probes as a covariate. methylRRA adjusts multiple p-values of each gene by Robust Rank Aggregation. For more detailed help information, please see the vignette.\n",
      "Version: 1.16.0\n",
      "Publication date: None\n",
      "Mantainer: Xu Ren <xuren2120@gmail.com>\n",
      "Authors:\n",
      "Author: Xu Ren [aut, cre], Pei Fen Kuan [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.5) type: depends\n",
      "  - RobustRankAggreg, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - stringr, type: imports\n",
      "  - stats, type: imports\n",
      "  - clusterProfiler, type: imports\n",
      "  - missMethyl, type: imports\n",
      "  - org.Hs.eg.db, type: imports\n",
      "  - reactome.db, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - GO.db, type: imports\n",
      "  - AnnotationDbi, type: imports\n",
      "  - shiny, type: imports\n",
      "  - IlluminaHumanMethylation450kanno.ilmn12.hg19, type: imports\n",
      "  - IlluminaHumanMethylationEPICanno.ilm10b4.hg19, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/methylGSA.html\n",
      "\n",
      "Name: TCGAbiolinksGUI\n",
      "Description:\n",
      "\"TCGAbiolinksGUI: A Graphical User Interface to analyze cancer molecular and clinical data. A demo version of GUI is found in https://tcgabiolinksgui.shinyapps.io/tcgabiolinks/\"\n",
      "Version: 1.23.0\n",
      "Publication date: None\n",
      "Mantainer: Tiago C. Silva <tiagochst@gmail.com>\n",
      "Authors:\n",
      "Author: Tiago Chedraoui Silva <tiagochst at gmail.com>, Antonio Colaprico <antonio.colaprico at ulb.ac.be>, Catharina Olsen <colsen at ulb.ac.be>, Michele Ceccarelli, Gianluca Bontempi <gbonte at ulb.ac.be>, Benjamin P. Berman <Benjamin.Berman at cshs.org>, Houtan Noushmehr <houtana at gmail.com>\n",
      "Dependencies:\n",
      "  - R (>= 3.3.1) type: depends\n",
      "  - shinydashboard (>= 0.5.3) type: depends\n",
      "  - TCGAbiolinksGUI.data, type: depends\n",
      "  - shiny (>= 0.14.1) type: imports\n",
      "  - downloader (>= 0.4) type: imports\n",
      "  - grid, type: imports\n",
      "  - DT, type: imports\n",
      "  - plotly, type: imports\n",
      "  - readr, type: imports\n",
      "  - maftools, type: imports\n",
      "  - stringr (>= 1.1.0) type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - ggrepel, type: imports\n",
      "  - data.table, type: imports\n",
      "  - caret, type: imports\n",
      "  - shinyFiles (>= 0.6.2) type: imports\n",
      "  - ggplot2 (>= 2.1.0) type: imports\n",
      "  - pathview, type: imports\n",
      "  - ELMER, type: imports\n",
      "  - clusterProfiler, type: imports\n",
      "  - parallel, type: imports\n",
      "  - TCGAbiolinks, type: imports\n",
      "  - shinyjs (>= 0.7) type: imports\n",
      "  - colourpicker, type: imports\n",
      "  - sesame, type: imports\n",
      "  - shinyBS (>= 0.61) type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 3)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/TCGAbiolinksGUI.html\n",
      "\n",
      "Name: qrqc\n",
      "Description:\n",
      "Quickly scans reads and gathers statistics on base and quality frequencies, read length, k-mers by position, and frequent sequences. Produces graphical output of statistics for use in quality control pipelines, and an optional HTML quality report. S4 SequenceSummary objects allow specific tests and functionality to be written around the data collected.\n",
      "Version: 1.52.0\n",
      "Publication date: None\n",
      "Mantainer: Vince Buffalo <vsbuffalo@ucdavis.edu>\n",
      "Authors:\n",
      "Author: Vince Buffalo\n",
      "Dependencies:\n",
      "  - reshape, type: depends\n",
      "  - ggplot2, type: depends\n",
      "  - Biostrings, type: depends\n",
      "  - biovizBase, type: depends\n",
      "  - brew, type: depends\n",
      "  - xtable, type: depends\n",
      "  - testthat, type: depends\n",
      "  - reshape, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - biovizBase, type: imports\n",
      "  - graphics, type: imports\n",
      "  - methods, type: imports\n",
      "  - plyr, type: imports\n",
      "  - stats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>=2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/qrqc.html\n",
      "\n",
      "Name: Rfastp\n",
      "Description:\n",
      "Rfastp is an R wrapper of fastp developed in c++. fastp performs quality control for fastq files. including low quality bases trimming, polyX trimming, adapter auto-detection and trimming, paired-end reads merging, UMI sequence/id handling. Rfastp can concatenate multiple files into one file (like shell command cat) and accept multiple files as input.\n",
      "Version: 1.8.0\n",
      "Publication date: None\n",
      "Mantainer: Thomas Carroll <tc.infomatics@gmail.com>\n",
      "Authors:\n",
      "Author: Wei Wang [aut] , Ji-Dung Luo [ctb] , Thomas Carroll [cre, aut]\n",
      "Dependencies:\n",
      "  - Rcpp, type: imports\n",
      "  - rjson, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - reshape2, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3 + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/Rfastp.html\n",
      "\n",
      "Name: snifter\n",
      "Description:\n",
      "Provides an R wrapper for the implementation of FI-tSNE from the python package openTNSE. See PoliÄar et al. (2019) and the algorithm described by Linderman et al. (2018) .\n",
      "Version: 1.8.0\n",
      "Publication date: None\n",
      "Mantainer: Alan O'Callaghan <alan.ocallaghan@outlook.com>\n",
      "Authors:\n",
      "Author: Alan O'Callaghan [aut, cre], Aaron Lun [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.0.0) type: depends\n",
      "  - basilisk, type: imports\n",
      "  - reticulate, type: imports\n",
      "  - irlba, type: imports\n",
      "  - stats, type: imports\n",
      "  - assertthat, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/snifter.html\n",
      "\n",
      "Name: CellBench\n",
      "Description:\n",
      "This package contains infrastructure for benchmarking analysis methods and access to single cell mixture benchmarking data. It provides a framework for organising analysis methods and testing combinations of methods in a pipeline without explicitly laying out each combination. It also provides utilities for sampling and filtering SingleCellExperiment objects, constructing lists of functions with varying parameters, and multithreaded evaluation of analysis methods.\n",
      "Version: 1.14.0\n",
      "Publication date: None\n",
      "Mantainer: Shian Su <su.s@wehi.edu.au>\n",
      "Authors:\n",
      "Author: Shian Su [cre, aut], Saskia Freytag [aut], Luyi Tian [aut], Xueyi Dong [aut], Matthew Ritchie [aut], Peter Hickey [ctb], Stuart Lee [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 3.6) type: depends\n",
      "  - SingleCellExperiment, type: depends\n",
      "  - magrittr, type: depends\n",
      "  - methods, type: depends\n",
      "  - stats, type: depends\n",
      "  - tibble, type: depends\n",
      "  - utils, type: depends\n",
      "  - BiocGenerics, type: imports\n",
      "  - BiocFileCache, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - rlang, type: imports\n",
      "  - glue, type: imports\n",
      "  - memoise, type: imports\n",
      "  - purrr (>= 0.3.0) type: imports\n",
      "  - rappdirs, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - tidyselect, type: imports\n",
      "  - lubridate, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/CellBench.html\n",
      "\n",
      "Name: Modstrings\n",
      "Description:\n",
      "Representing nucleotide modifications in a nucleotide sequence is usually done via special characters from a number of sources. This represents a challenge to work with in R and the Biostrings package. The Modstrings package implements this functionallity for RNA and DNA sequences containing modified nucleotides by translating the character internally in order to work with the infrastructure of the Biostrings package. For this the ModRNAString and ModDNAString classes and derivates and functions to construct and modify these objects despite the encoding issues are implemenented. In addition the conversion from sequences to list like location information (and the reverse operation) is implemented as well.\n",
      "Version: 1.14.0\n",
      "Publication date: None\n",
      "Mantainer: Felix G.M. Ernst <felix.gm.ernst@outlook.com>\n",
      "Authors:\n",
      "Author: Felix G.M. Ernst [aut, cre] , Denis L.J. Lafontaine [ctb, fnd]\n",
      "Dependencies:\n",
      "  - R (>= 3.6) type: depends\n",
      "  - Biostrings, type: depends\n",
      "  - methods, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - XVector, type: imports\n",
      "  - stringi, type: imports\n",
      "  - stringr, type: imports\n",
      "  - crayon, type: imports\n",
      "  - grDevices, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/Modstrings.html\n",
      "\n",
      "Name: omicade4\n",
      "Description:\n",
      "This package performes multiple co-inertia analysis of omics datasets.\n",
      "Version: 1.38.0\n",
      "Publication date: None\n",
      "Mantainer: Chen Meng <mengchen18@gmail.com>\n",
      "Authors:\n",
      "Author: Chen Meng, Aedin Culhane, Amin M. Gholami.\n",
      "Dependencies:\n",
      "  - R (>= 3.0.0) type: depends\n",
      "  - ade4, type: depends\n",
      "  - made4, type: imports\n",
      "  - Biobase, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/omicade4.html\n",
      "\n",
      "Name: ExploreModelMatrix\n",
      "Description:\n",
      "Given a sample data table and a design formula, ExploreModelMatrix generates an interactive application for exploration of the resulting design matrix. This can be helpful for interpreting model coefficients and constructing appropriate contrasts in (generalized) linear models. Static visualizations can also be generated.\n",
      "Version: 1.10.0\n",
      "Publication date: None\n",
      "Mantainer: Charlotte Soneson <charlottesoneson@gmail.com>\n",
      "Authors:\n",
      "Author: Charlotte Soneson [aut, cre] , Federico Marini [aut] , Michael Love [aut] , Florian Geier [aut] , Michael Stadler [aut]\n",
      "Dependencies:\n",
      "  - shiny (>= 1.5.0) type: imports\n",
      "  - shinydashboard, type: imports\n",
      "  - DT, type: imports\n",
      "  - cowplot, type: imports\n",
      "  - utils, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - stats, type: imports\n",
      "  - methods, type: imports\n",
      "  - rintrojs, type: imports\n",
      "  - scales, type: imports\n",
      "  - tibble, type: imports\n",
      "  - MASS, type: imports\n",
      "  - limma, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - shinyjs, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ExploreModelMatrix.html\n",
      "\n",
      "Name: imcRtools\n",
      "Description:\n",
      "This R package supports the handling and analysis of imaging mass cytometry and other highly multiplexed imaging data. The main functionality includes reading in single-cell data after image segmentation and measurement, data formatting to perform channel spillover correction and a number of spatial analysis approaches. First, cell-cell interactions are detected via spatial graph construction; these graphs can be visualized with cells representing nodes and interactions representing edges. Furthermore, per cell, its direct neighbours are summarized to allow spatial clustering. Per image/grouping level, interactions between types of cells are counted, averaged and compared against random permutations. In that way, types of cells that interact more (attraction) or less (avoidance) frequently than expected by chance are detected.\n",
      "Version: 1.4.2\n",
      "Publication date: None\n",
      "Mantainer: Nils Eling <nils.eling@dqbm.uzh.ch>\n",
      "Authors:\n",
      "Author: Nils Eling [aut, cre] , Tobias Hoch [ctb], Vito Zanotelli [ctb], Jana Fischer [ctb], Daniel Schulz [ctb], Lasse Meyer [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - SpatialExperiment, type: depends\n",
      "  - S4Vectors, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - methods, type: imports\n",
      "  - pheatmap, type: imports\n",
      "  - scuttle, type: imports\n",
      "  - stringr, type: imports\n",
      "  - readr, type: imports\n",
      "  - EBImage, type: imports\n",
      "  - cytomapper, type: imports\n",
      "  - abind, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - viridis, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - DT, type: imports\n",
      "  - igraph, type: imports\n",
      "  - SingleCellExperiment, type: imports\n",
      "  - vroom, type: imports\n",
      "  - BiocNeighbors, type: imports\n",
      "  - RTriangle, type: imports\n",
      "  - ggraph, type: imports\n",
      "  - tidygraph, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - data.table, type: imports\n",
      "  - sf, type: imports\n",
      "  - concaveman, type: imports\n",
      "  - tidyselect, type: imports\n",
      "  - distances, type: imports\n",
      "  - MatrixGenerics, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/imcRtools.html\n",
      "\n",
      "Name: maser\n",
      "Description:\n",
      "This package provides functionalities for downstream analysis, annotation and visualizaton of alternative splicing events generated by rMATS.\n",
      "Version: 1.16.0\n",
      "Publication date: None\n",
      "Mantainer: Diogo F.T. Veiga <diogof.veiga@gmail.com>\n",
      "Authors:\n",
      "Author: Diogo F.T. Veiga [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - ggplot2, type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - dplyr, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - Gviz, type: imports\n",
      "  - DT, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - methods, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - parallel, type: imports\n",
      "  - data.table, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/maser.html\n",
      "\n",
      "Name: pandaR\n",
      "Description:\n",
      "Runs PANDA, an algorithm for discovering novel network structure by combining information from multiple complementary data sources.\n",
      "Version: 1.30.0\n",
      "Publication date: None\n",
      "Mantainer: Joseph N. Paulson <paulson.joseph@gene.com>, Dan Schlauch <dschlauch@genospace.com>\n",
      "Authors:\n",
      "Author: Dan Schlauch, Joseph N. Paulson, Albert Young, John Quackenbush, Kimberly Glass\n",
      "Dependencies:\n",
      "  - R (>= 3.0.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - Biobase, type: depends\n",
      "  - BiocGenerics, type: depends\n",
      "  - matrixStats, type: imports\n",
      "  - igraph, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - grid, type: imports\n",
      "  - reshape, type: imports\n",
      "  - plyr, type: imports\n",
      "  - RUnit, type: imports\n",
      "  - hexbin, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/pandaR.html\n",
      "\n",
      "Name: pdInfoBuilder\n",
      "Description:\n",
      "Builds platform design information packages. These consist of a SQLite database containing feature-level data such as x, y position on chip and featureSet ID. The database also incorporates featureSet-level annotation data. The products of this packages are used by the oligo pkg.\n",
      "Version: 1.62.0\n",
      "Publication date: None\n",
      "Mantainer: Benilton Carvalho <benilton@unicamp.br>\n",
      "Authors:\n",
      "Author: Seth Falcon, Vince Carey, Matt Settles, Kristof de Beuf, Benilton Carvalho\n",
      "Dependencies:\n",
      "  - R (>= 3.2.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - Biobase, type: depends\n",
      "  - RSQLite (>= 1.0.0) type: depends\n",
      "  - affxparser, type: depends\n",
      "  - oligo, type: depends\n",
      "  - Biostrings, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - DBI (>= 0.3.1) type: imports\n",
      "  - IRanges, type: imports\n",
      "  - oligoClasses, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/pdInfoBuilder.html\n",
      "\n",
      "Name: regionReport\n",
      "Description:\n",
      "Generate HTML or PDF reports to explore a set of regions such as the results from annotation-agnostic expression analysis of RNA-seq data at base-pair resolution performed by derfinder. You can also create reports for DESeq2 or edgeR results.\n",
      "Version: 1.32.0\n",
      "Publication date: None\n",
      "Mantainer: Leonardo Collado-Torres <lcolladotor@gmail.com>\n",
      "Authors:\n",
      "Author: Leonardo Collado-Torres [aut, cre] , Andrew E. Jaffe [aut] , Jeffrey T. Leek [aut, ths]\n",
      "Dependencies:\n",
      "  - R (>= 3.2) type: depends\n",
      "  - BiocStyle, type: imports\n",
      "  - derfinder, type: imports\n",
      "  - DEFormats, type: imports\n",
      "  - DESeq2, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - knitr (>= 1.6) type: imports\n",
      "  - knitrBootstrap (>= 0.9.0) type: imports\n",
      "  - methods, type: imports\n",
      "  - RefManageR, type: imports\n",
      "  - rmarkdown (>= 0.9.5) type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/regionReport.html\n",
      "\n",
      "Name: scone\n",
      "Description:\n",
      "SCONE is an R package for comparing and ranking the performance of different normalization schemes for single-cell RNA-seq and other high-throughput analyses.\n",
      "Version: 1.22.0\n",
      "Publication date: None\n",
      "Mantainer: Davide Risso <risso.davide@gmail.com>\n",
      "Authors:\n",
      "Author: Michael Cole [aut, cph], Davide Risso [aut, cre, cph], Matteo Borella [ctb], Chiara Romualdi [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 3.4) type: depends\n",
      "  - methods, type: depends\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - aroma.light, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - class, type: imports\n",
      "  - cluster, type: imports\n",
      "  - compositions, type: imports\n",
      "  - diptest, type: imports\n",
      "  - edgeR, type: imports\n",
      "  - fpc, type: imports\n",
      "  - gplots, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - hexbin, type: imports\n",
      "  - limma, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - mixtools, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - boot, type: imports\n",
      "  - rhdf5, type: imports\n",
      "  - RUVSeq, type: imports\n",
      "  - rARPACK, type: imports\n",
      "  - MatrixGenerics, type: imports\n",
      "  - SingleCellExperiment, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/scone.html\n",
      "\n",
      "Name: a4Core\n",
      "Description:\n",
      "Utility functions for the Automated Affymetrix Array Analysis set of packages.\n",
      "Version: 1.46.0\n",
      "Publication date: None\n",
      "Mantainer: Laure Cougnaud <laure.cougnaud@openanalytics.eu>\n",
      "Authors:\n",
      "Author: Willem Talloen [aut], Tobias Verbeke [aut], Laure Cougnaud [cre]\n",
      "Dependencies:\n",
      "  - Biobase, type: imports\n",
      "  - glmnet, type: imports\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/a4Core.html\n",
      "\n",
      "Name: exomePeak2\n",
      "Description:\n",
      "exomePeak2 provides peak detection and differential methylation for Methylated RNA Immunoprecipitation Sequencing (MeRIP-Seq) data. MeRIP-Seq is a commonly applied sequencing assay that measures the location and abundance of RNA modification sites under specific cellular conditions. In practice, the technique is sensitive to PCR amplification biases commonly found in NGS data. In addition, the efficiency of immunoprecipitation often varies between different IP samples. exomePeak2 can perform peak calling and differential analysis independent of GC content bias and IP efficiency changes.\n",
      "Version: 1.10.0\n",
      "Publication date: None\n",
      "Mantainer: Zhen Wei <zhen.wei10@icloud.com>\n",
      "Authors:\n",
      "Author: Zhen Wei [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - Rsamtools, type: imports\n",
      "  - GenomicAlignments, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - GenomicFeatures, type: imports\n",
      "  - DESeq2, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - mclust, type: imports\n",
      "  - BSgenome, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - speedglm, type: imports\n",
      "  - splines, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/exomePeak2.html\n",
      "\n",
      "Name: metaMS\n",
      "Description:\n",
      "MS-based metabolomics data processing and compound annotation pipeline.\n",
      "Version: 1.34.0\n",
      "Publication date: None\n",
      "Mantainer: Yann Guitton <yann.guitton@gmail.com>\n",
      "Authors:\n",
      "Author: Ron Wehrens [aut] (author of GC-MS part, Initial Maintainer), Pietro Franceschi [aut] (author of LC-MS part), Nir Shahaf [ctb], Matthias Scholz [ctb], Georg Weingart [ctb] (development of GC-MS approach), Elisabete Carvalho [ctb] (testing and feedback of GC-MS pipeline), Yann Guitton [ctb, cre] , Julien Saint-Vanne [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - CAMERA, type: depends\n",
      "  - xcms, type: depends\n",
      "  - Matrix, type: imports\n",
      "  - tools, type: imports\n",
      "  - robustbase, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/metaMS.html\n",
      "\n",
      "Name: yarn\n",
      "Description:\n",
      "Expedite large RNA-Seq analyses using a combination of previously developed tools. YARN is meant to make it easier for the user in performing basic mis-annotation quality control, filtering, and condition-aware normalization. YARN leverages many Bioconductor tools and statistical techniques to account for the large heterogeneity and sparsity found in very large RNA-seq experiments.\n",
      "Version: 1.24.0\n",
      "Publication date: None\n",
      "Mantainer: Joseph N Paulson <paulson.joseph@gene.com>\n",
      "Authors:\n",
      "Author: Joseph N Paulson [aut, cre], Cho-Yi Chen [aut], Camila Lopes-Ramos [aut], Marieke Kuijjer [aut], John Platig [aut], Abhijeet Sonawane [aut], Maud Fagny [aut], Kimberly Glass [aut], John Quackenbush [aut]\n",
      "Dependencies:\n",
      "  - Biobase, type: depends\n",
      "  - biomaRt, type: imports\n",
      "  - downloader, type: imports\n",
      "  - edgeR, type: imports\n",
      "  - gplots, type: imports\n",
      "  - graphics, type: imports\n",
      "  - limma, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - preprocessCore, type: imports\n",
      "  - readr, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - stats, type: imports\n",
      "  - quantro, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/yarn.html\n",
      "\n",
      "Name: diffHic\n",
      "Description:\n",
      "Detects differential interactions across biological conditions in a Hi-C experiment. Methods are provided for read alignment and data pre-processing into interaction counts. Statistical analysis is based on edgeR and supports normalization and filtering. Several visualization options are also available.\n",
      "Version: 1.30.0\n",
      "Publication date: None\n",
      "Mantainer: Aaron Lun <infinite.monkeys.with.keyboards@gmail.com>\n",
      "Authors:\n",
      "Author: Aaron Lun [aut, cre], Gordon Smyth [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.5) type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - InteractionSet, type: depends\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - Rsamtools, type: imports\n",
      "  - Rhtslib, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - BSgenome, type: imports\n",
      "  - rhdf5, type: imports\n",
      "  - edgeR, type: imports\n",
      "  - limma, type: imports\n",
      "  - csaw, type: imports\n",
      "  - locfit, type: imports\n",
      "  - methods, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - Rcpp, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/diffHic.html\n",
      "\n",
      "Name: frma\n",
      "Description:\n",
      "Preprocessing and analysis for single microarrays and microarray batches.\n",
      "Version: 1.50.0\n",
      "Publication date: None\n",
      "Mantainer: Matthew N. McCall <mccallm@gmail.com>\n",
      "Authors:\n",
      "Author: Matthew N. McCall <mccallm at gmail.com>, Rafael A. Irizarry <rafa at jhu.edu>, with contributions from Terry Therneau\n",
      "Dependencies:\n",
      "  - R (>= 2.10.0) type: depends\n",
      "  - Biobase, type: depends\n",
      "  - Biobase, type: imports\n",
      "  - MASS, type: imports\n",
      "  - DBI, type: imports\n",
      "  - affy, type: imports\n",
      "  - methods, type: imports\n",
      "  - oligo, type: imports\n",
      "  - oligoClasses, type: imports\n",
      "  - preprocessCore, type: imports\n",
      "  - utils, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/frma.html\n",
      "\n",
      "Name: methylclock\n",
      "Description:\n",
      "This package allows to estimate chronological and gestational DNA methylation (DNAm) age as well as biological age using different methylation clocks. Chronological DNAm age (in years) : Horvath's clock, Hannum's clock, BNN, Horvath's skin+blood clock, PedBE clock and Wu's clock. Gestational DNAm age : Knight's clock, Bohlin's clock, Mayne's clock and Lee's clocks. Biological DNAm clocks : Levine's clock and Telomere Length's clock.\n",
      "Version: 1.4.0\n",
      "Publication date: None\n",
      "Mantainer: Dolors Pelegri-Siso <dolors.pelegri@isglobal.org>\n",
      "Authors:\n",
      "Author: Dolors Pelegri-Siso [aut, cre] , Juan R. Gonzalez [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.1.0) type: depends\n",
      "  - methylclockData, type: depends\n",
      "  - devtools, type: depends\n",
      "  - quadprog, type: depends\n",
      "  - Rcpp (>= 1.0.6) type: imports\n",
      "  - ExperimentHub, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - impute, type: imports\n",
      "  - PerformanceAnalytics, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - ggpmisc, type: imports\n",
      "  - tidyverse, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - ggpubr, type: imports\n",
      "  - minfi, type: imports\n",
      "  - tibble, type: imports\n",
      "  - RPMM, type: imports\n",
      "  - stats, type: imports\n",
      "  - graphics, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - gridExtra, type: imports\n",
      "  - preprocessCore, type: imports\n",
      "  - dynamicTreeCut, type: imports\n",
      "  - planet, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/methylclock.html\n",
      "\n",
      "Name: twilight\n",
      "Description:\n",
      "In a typical microarray setting with gene expression data observed under two conditions, the local false discovery rate describes the probability that a gene is not differentially expressed between the two conditions given its corrresponding observed score or p-value level. The resulting curve of p-values versus local false discovery rate offers an insight into the twilight zone between clear differential and clear non-differential gene expression. Package 'twilight' contains two main functions: Function twilight.pval performs a two-condition test on differences in means for a given input matrix or expression set and computes permutation based p-values. Function twilight performs a stochastic downhill search to estimate local false discovery rates and effect size distributions. The package further provides means to filter for permutations that describe the null distribution correctly. Using filtered permutations, the influence of hidden confounders could be diminished.\n",
      "Version: 1.74.0\n",
      "Publication date: None\n",
      "Mantainer: Stefanie Scheid <stefanie.scheid@gmx.de>\n",
      "Authors:\n",
      "Author: Stefanie Scheid <stefanie.scheid at gmx.de>\n",
      "Dependencies:\n",
      "  - R (>= 2.10) type: depends\n",
      "  - splines (>= 2.2.0) type: depends\n",
      "  - stats (>= 2.2.0) type: depends\n",
      "  - Biobase, type: depends\n",
      "  - Biobase, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - stats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/twilight.html\n",
      "\n",
      "Name: zFPKM\n",
      "Description:\n",
      "Perform the zFPKM transform on RNA-seq FPKM data. This algorithm is based on the publication by Hart et al., 2013 (Pubmed ID 24215113). Reference recommends using zFPKM > -3 to select expressed genes. Validated with encode open/closed chromosome data. Works well for gene level data using FPKM or TPM. Does not appear to calibrate well for transcript level data.\n",
      "Version: 1.20.0\n",
      "Publication date: None\n",
      "Mantainer: Ron Ammar <ron.ammar@bms.com>\n",
      "Authors:\n",
      "Author: Ron Ammar [aut, cre], John Thompson [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.4.0) type: depends\n",
      "  - checkmate, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3 | file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/zFPKM.html\n",
      "\n",
      "Name: mosaics\n",
      "Description:\n",
      "This package provides functions for fitting MOSAiCS and MOSAiCS-HMM, a statistical framework to analyze one-sample or two-sample ChIP-seq data of transcription factor binding and histone modification.\n",
      "Version: 2.36.0\n",
      "Publication date: None\n",
      "Mantainer: Dongjun Chung <dongjun.chung@gmail.com>\n",
      "Authors:\n",
      "Author: Dongjun Chung, Pei Fen Kuan, Rene Welch, Sunduz Keles\n",
      "Dependencies:\n",
      "  - R (>= 3.0.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - graphics, type: depends\n",
      "  - Rcpp, type: depends\n",
      "  - MASS, type: imports\n",
      "  - splines, type: imports\n",
      "  - lattice, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - GenomicAlignments, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/mosaics.html\n",
      "\n",
      "Name: paxtoolsr\n",
      "Description:\n",
      "The package provides a set of R functions for interacting with BioPAX OWL files using Paxtools and the querying Pathway Commons (PC) molecular interaction database. Pathway Commons is a project by the Memorial Sloan-Kettering Cancer Center (MSKCC), Dana-Farber Cancer Institute (DFCI), and the University of Toronto. Pathway Commons databases include: BIND, BioGRID, CORUM, CTD, DIP, DrugBank, HPRD, HumanCyc, IntAct, KEGG, MirTarBase, Panther, PhosphoSitePlus, Reactome, RECON, TRANSFAC.\n",
      "Version: 1.32.0\n",
      "Publication date: None\n",
      "Mantainer: Augustin Luna <lunaa@cbio.mskcc.org>\n",
      "Authors:\n",
      "Author: Augustin Luna [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 3.2) type: depends\n",
      "  - rJava (>= 0.9-8) type: depends\n",
      "  - methods, type: depends\n",
      "  - XML, type: depends\n",
      "  - utils, type: imports\n",
      "  - httr, type: imports\n",
      "  - igraph, type: imports\n",
      "  - plyr, type: imports\n",
      "  - rjson, type: imports\n",
      "  - R.utils, type: imports\n",
      "  - jsonlite, type: imports\n",
      "  - readr, type: imports\n",
      "  - rappdirs, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/paxtoolsr.html\n",
      "\n",
      "Name: puma\n",
      "Description:\n",
      "Most analyses of Affymetrix GeneChip data (including tranditional 3' arrays and exon arrays and Human Transcriptome Array 2.0) are based on point estimates of expression levels and ignore the uncertainty of such estimates. By propagating uncertainty to downstream analyses we can improve results from microarray analyses. For the first time, the puma package makes a suite of uncertainty propagation methods available to a general audience. In additon to calculte gene expression from Affymetrix 3' arrays, puma also provides methods to process exon arrays and produces gene and isoform expression for alternative splicing study. puma also offers improvements in terms of scope and speed of execution over previously available uncertainty propagation methods. Included are summarisation, differential expression detection, clustering and PCA methods, together with useful plotting functions.\n",
      "Version: 3.40.0\n",
      "Publication date: None\n",
      "Mantainer: Xuejun Liu <xuejun.liu@nuaa.edu.cn>\n",
      "Authors:\n",
      "Author: Richard D. Pearson, Xuejun Liu, Magnus Rattray, Marta Milo, Neil D. Lawrence, Guido Sanguinetti, Li Zhang\n",
      "Dependencies:\n",
      "  - R (>= 3.2.0) type: depends\n",
      "  - oligo, type: depends\n",
      "  - graphics, type: depends\n",
      "  - grDevices, type: depends\n",
      "  - methods, type: depends\n",
      "  - stats, type: depends\n",
      "  - utils, type: depends\n",
      "  - mclust, type: depends\n",
      "  - oligoClasses, type: depends\n",
      "  - Biobase, type: imports\n",
      "  - affy, type: imports\n",
      "  - affyio, type: imports\n",
      "  - oligoClasses, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/puma.html\n",
      "\n",
      "Name: QUBIC\n",
      "Description:\n",
      "The core function of this R package is to provide the implementation of the well-cited and well-reviewed QUBIC algorithm, aiming to deliver an effective and efficient biclustering capability. This package also includes the following related functions: (i) a qualitative representation of the input gene expression data, through a well-designed discretization way considering the underlying data property, which can be directly used in other biclustering programs; (ii) visualization of identified biclusters using heatmap in support of overall expression pattern analysis; (iii) bicluster-based co-expression network elucidation and visualization, where different correlation coefficient scores between a pair of genes are provided; and (iv) a generalize output format of biclusters and corresponding network can be freely downloaded so that a user can easily do following comprehensive functional enrichment analysis (e.g. DAVID) and advanced network visualization (e.g. Cytoscape).\n",
      "Version: 1.26.0\n",
      "Publication date: None\n",
      "Mantainer: Yu Zhang <zy26@jlu.edu.cn>\n",
      "Authors:\n",
      "Author: Yu Zhang [aut, cre], Qin Ma [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.1) type: depends\n",
      "  - biclust, type: depends\n",
      "  - Rcpp (>= 0.11.0) type: imports\n",
      "  - methods, type: imports\n",
      "  - Matrix, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "CC BY-NC-ND 4.0 + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/QUBIC.html\n",
      "\n",
      "Name: dupRadar\n",
      "Description:\n",
      "Duplication rate quality control for RNA-Seq datasets.\n",
      "Version: 1.28.0\n",
      "Publication date: None\n",
      "Mantainer: Sergi Sayols <sergisayolspuig@gmail.com>, Holger Klein <holger.klein@gmail.com>\n",
      "Authors:\n",
      "Author: Sergi Sayols <sergisayolspuig at gmail.com>, Holger Klein <holger.klein at gmail.com>\n",
      "Dependencies:\n",
      "  - R (>= 3.2.0) type: depends\n",
      "  - Rsubread, type: imports\n",
      "  - KernSmooth, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/dupRadar.html\n",
      "\n",
      "Name: ExpressionAtlas\n",
      "Description:\n",
      "This package is for searching for datasets in EMBL-EBI Expression Atlas, and downloading them into R for further analysis. Each Expression Atlas dataset is represented as a SimpleList object with one element per platform. Sequencing data is contained in a SummarizedExperiment object, while microarray data is contained in an ExpressionSet or MAList object.\n",
      "Version: 1.26.0\n",
      "Publication date: None\n",
      "Mantainer: Pedro Madrigal <pmadrigal@ebi.ac.uk>\n",
      "Authors:\n",
      "Author: Maria Keays [aut] , Pedro Madrigal [cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.2.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - Biobase, type: depends\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - limma, type: depends\n",
      "  - S4Vectors, type: depends\n",
      "  - xml2, type: depends\n",
      "  - RCurl, type: depends\n",
      "  - jsonlite, type: depends\n",
      "  - BiocStyle, type: depends\n",
      "  - utils, type: imports\n",
      "  - XML, type: imports\n",
      "  - httr, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 3)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ExpressionAtlas.html\n",
      "\n",
      "Name: granulator\n",
      "Description:\n",
      "granulator is an R package for the cell type deconvolution of heterogeneous tissues based on bulk RNA-seq data or single cell RNA-seq expression profiles. The package provides a unified testing interface to rapidly run and benchmark multiple state-of-the-art deconvolution methods. Data for the deconvolution of peripheral blood mononuclear cells (PBMCs) into individual immune cell types is provided as well.\n",
      "Version: 1.6.0\n",
      "Publication date: None\n",
      "Mantainer: Sabina Pfister <sabina.pfister@novartis.com>\n",
      "Authors:\n",
      "Author: Sabina Pfister [aut, cre], Vincent Kuettel [aut], Enrico Ferrero [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - cowplot, type: imports\n",
      "  - e1071, type: imports\n",
      "  - epiR, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - dtangle, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - ggplotify, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - limSolve, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - MASS, type: imports\n",
      "  - nnls, type: imports\n",
      "  - parallel, type: imports\n",
      "  - pheatmap, type: imports\n",
      "  - purrr, type: imports\n",
      "  - rlang, type: imports\n",
      "  - stats, type: imports\n",
      "  - tibble, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/granulator.html\n",
      "\n",
      "Name: PeacoQC\n",
      "Description:\n",
      "This is a package that includes pre-processing and quality control functions that can remove margin events, compensate and transform the data and that will use PeacoQCSignalStability for quality control. This last function will first detect peaks in each channel of the flowframe. It will remove anomalies based on the IsolationTree function and the MAD outlier detection method. This package can be used for both flow- and mass cytometry data.\n",
      "Version: 1.8.0\n",
      "Publication date: None\n",
      "Mantainer: Annelies Emmaneel <annelies.emmaneel@hotmail.com>\n",
      "Authors:\n",
      "Author: Annelies Emmaneel [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - circlize, type: imports\n",
      "  - ComplexHeatmap, type: imports\n",
      "  - flowCore, type: imports\n",
      "  - flowWorkspace, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - grid, type: imports\n",
      "  - gridExtra, type: imports\n",
      "  - methods, type: imports\n",
      "  - plyr, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>=3)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/PeacoQC.html\n",
      "\n",
      "Name: r3Cseq\n",
      "Description:\n",
      "This package is used for the analysis of long-range chromatin interactions from 3C-seq assay.\n",
      "Version: 1.44.0\n",
      "Publication date: None\n",
      "Mantainer: Supat Thongjuea <supat.thongjuea@imm.ox.ac.uk> or <supat.thongjuea@gmail.com>\n",
      "Authors:\n",
      "Author: Supat Thongjuea, MRC WIMM Centre for Computational Biology, Weatherall Institute of Molecular Medicine, University of Oxford, UK <supat.thongjuea at imm.ox.ac.uk>\n",
      "Dependencies:\n",
      "  - GenomicRanges, type: depends\n",
      "  - Rsamtools, type: depends\n",
      "  - rtracklayer, type: depends\n",
      "  - VGAM, type: depends\n",
      "  - qvalue, type: depends\n",
      "  - methods, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - data.table, type: imports\n",
      "  - sqldf, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/r3Cseq.html\n",
      "\n",
      "Name: VariantTools\n",
      "Description:\n",
      "Explore, diagnose, and compare variant calls using filters.\n",
      "Version: 1.40.0\n",
      "Publication date: None\n",
      "Mantainer: Michael Lawrence <michafla@gene.com>\n",
      "Authors:\n",
      "Author: Michael Lawrence, Jeremiah Degenhardt, Robert Gentleman\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - S4Vectors, type: depends\n",
      "  - IRanges, type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - VariantAnnotation, type: depends\n",
      "  - methods, type: depends\n",
      "  - Rsamtools, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - parallel, type: imports\n",
      "  - GenomicFeatures, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - BSgenome, type: imports\n",
      "  - Biobase, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/VariantTools.html\n",
      "\n",
      "Name: BicARE\n",
      "Description:\n",
      "Biclustering Analysis and Results Exploration\n",
      "Version: 1.56.0\n",
      "Publication date: None\n",
      "Mantainer: Pierre Gestraud <pierre.gestraud@curie.fr>\n",
      "Authors:\n",
      "Author: Pierre Gestraud\n",
      "Dependencies:\n",
      "  - R (>= 1.8.0) type: depends\n",
      "  - Biobase, type: depends\n",
      "  - multtest, type: depends\n",
      "  - GSEABase, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/BicARE.html\n",
      "\n",
      "Name: CAGEr\n",
      "Description:\n",
      "Preprocessing of CAGE sequencing data, identification and normalization of transcription start sites and downstream analysis of transcription start sites clusters (promoters).\n",
      "Version: 2.4.0\n",
      "Publication date: None\n",
      "Mantainer: Charles Plessy <charles.plessy@oist.jp>\n",
      "Authors:\n",
      "Author: Vanja Haberle [aut], Charles Plessy [cre], Damir Baranasic [ctb], Sarvesh Nikumbh [ctb]\n",
      "Dependencies:\n",
      "  - methods, type: depends\n",
      "  - MultiAssayExperiment, type: depends\n",
      "  - R (>= 4.1.0) type: depends\n",
      "  - BiocGenerics, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - BSgenome, type: imports\n",
      "  - data.table, type: imports\n",
      "  - DelayedArray, type: imports\n",
      "  - DelayedMatrixStats, type: imports\n",
      "  - formula.tools, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - GenomicAlignments, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - ggplot2 (>= 2.2.0) type: imports\n",
      "  - gtools, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - KernSmooth, type: imports\n",
      "  - memoise, type: imports\n",
      "  - plyr, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - som, type: imports\n",
      "  - stringdist, type: imports\n",
      "  - stringi, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - utils, type: imports\n",
      "  - vegan, type: imports\n",
      "  - VGAM, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/CAGEr.html\n",
      "\n",
      "Name: CSAR\n",
      "Description:\n",
      "Statistical tools for ChIP-seq data analysis. The package includes the statistical method described in Kaufmann et al. (2009) PLoS Biology: 7(4):e1000090. Briefly, Taking the average DNA fragment size subjected to sequencing into account, the software calculates genomic single-nucleotide read-enrichment values. After normalization, sample and control are compared using a test based on the Poisson distribution. Test statistic thresholds to control the false discovery rate are obtained through random permutation.\n",
      "Version: 1.50.0\n",
      "Publication date: None\n",
      "Mantainer: Jose M Muino <jose.muino@live.com>\n",
      "Authors:\n",
      "Author: Jose M Muino\n",
      "Dependencies:\n",
      "  - R (>= 2.15.0) type: depends\n",
      "  - S4Vectors, type: depends\n",
      "  - IRanges, type: depends\n",
      "  - GenomeInfoDb, type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/CSAR.html\n",
      "\n",
      "Name: hypeR\n",
      "Description:\n",
      "An R Package for Geneset Enrichment Workflows.\n",
      "Version: 1.14.0\n",
      "Publication date: None\n",
      "Mantainer: Anthony Federico <anfed@bu.edu>\n",
      "Authors:\n",
      "Author: Anthony Federico [aut, cre], Stefano Monti [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.6.0) type: depends\n",
      "  - ggplot2, type: imports\n",
      "  - ggforce, type: imports\n",
      "  - R6, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - purrr, type: imports\n",
      "  - stats, type: imports\n",
      "  - stringr, type: imports\n",
      "  - scales, type: imports\n",
      "  - rlang, type: imports\n",
      "  - httr, type: imports\n",
      "  - openxlsx, type: imports\n",
      "  - htmltools, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - reactable, type: imports\n",
      "  - msigdbr, type: imports\n",
      "  - kableExtra, type: imports\n",
      "  - rmarkdown, type: imports\n",
      "  - igraph, type: imports\n",
      "  - visNetwork, type: imports\n",
      "  - shiny, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3 + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/hypeR.html\n",
      "\n",
      "Name: megadepth\n",
      "Description:\n",
      "This package provides an R interface to Megadepth by Christopher Wilks available at https://github.com/ChristopherWilks/megadepth. It is particularly useful for computing the coverage of a set of genomic regions across bigWig or BAM files. With this package, you can build base-pair coverage matrices for regions or annotations of your choice from BigWig files. Megadepth was used to create the raw files provided by https://bioconductor.org/packages/recount3.\n",
      "Version: 1.8.0\n",
      "Publication date: None\n",
      "Mantainer: David Zhang <david.zhang.12@ucl.ac.uk>\n",
      "Authors:\n",
      "Author: Leonardo Collado-Torres [aut] , David Zhang [aut, cre]\n",
      "Dependencies:\n",
      "  - xfun, type: imports\n",
      "  - utils, type: imports\n",
      "  - fs, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - readr, type: imports\n",
      "  - cmdfun, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - magrittr, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/megadepth.html\n",
      "\n",
      "Name: SCnorm\n",
      "Description:\n",
      "This package implements SCnorm â a method to normalize single-cell RNA-seq data.\n",
      "Version: 1.20.0\n",
      "Publication date: None\n",
      "Mantainer: Rhonda Bacher <rbacher@ufl.edu>\n",
      "Authors:\n",
      "Author: Rhonda Bacher\n",
      "Dependencies:\n",
      "  - R (>= 3.4.0) type: depends\n",
      "  - SingleCellExperiment, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - stats, type: imports\n",
      "  - methods, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - parallel, type: imports\n",
      "  - quantreg, type: imports\n",
      "  - cluster, type: imports\n",
      "  - moments, type: imports\n",
      "  - data.table, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - forcats, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/SCnorm.html\n",
      "\n",
      "Name: scry\n",
      "Description:\n",
      "Many modern biological datasets consist of small counts that are not well fit by standard linear-Gaussian methods such as principal component analysis. This package provides implementations of count-based feature selection and dimension reduction algorithms. These methods can be used to facilitate unsupervised analysis of any high-dimensional data such as single-cell RNA-seq.\n",
      "Version: 1.10.0\n",
      "Publication date: None\n",
      "Mantainer: Kelly Street <street.kelly@gmail.com>\n",
      "Authors:\n",
      "Author: Kelly Street [aut, cre], F. William Townes [aut, cph], Davide Risso [aut], Stephanie Hicks [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - stats, type: depends\n",
      "  - methods, type: depends\n",
      "  - DelayedArray, type: imports\n",
      "  - glmpca (>= 0.2.0) type: imports\n",
      "  - HDF5Array, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - SingleCellExperiment, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - BiocSingular, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/scry.html\n",
      "\n",
      "Name: BiocPkgTools\n",
      "Description:\n",
      "Bioconductor has a rich ecosystem of metadata around packages, usage, and build status. This package is a simple collection of functions to access that metadata from R. The goal is to expose metadata for data mining and value-added functionality such as package searching, text mining, and analytics on packages.\n",
      "Version: 1.16.0\n",
      "Publication date: None\n",
      "Mantainer: Sean Davis <seandavi@gmail.com>\n",
      "Authors:\n",
      "Author: Shian Su [aut, ctb], Lori Shepherd [ctb], Marcel Ramos [ctb], Felix G.M. Ernst [ctb], Jennifer Wokaty [ctb], Charlotte Soneson [ctb], Martin Morgan [ctb], Vince Carey [ctb], Sean Davis [aut, cre]\n",
      "Dependencies:\n",
      "  - htmlwidgets, type: depends\n",
      "  - BiocFileCache, type: imports\n",
      "  - BiocManager, type: imports\n",
      "  - biocViews, type: imports\n",
      "  - tibble, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - methods, type: imports\n",
      "  - rlang, type: imports\n",
      "  - stringr, type: imports\n",
      "  - rvest, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - xml2, type: imports\n",
      "  - readr, type: imports\n",
      "  - httr, type: imports\n",
      "  - htmltools, type: imports\n",
      "  - DT, type: imports\n",
      "  - tools, type: imports\n",
      "  - utils, type: imports\n",
      "  - igraph, type: imports\n",
      "  - jsonlite, type: imports\n",
      "  - gh, type: imports\n",
      "  - RBGL, type: imports\n",
      "  - graph, type: imports\n",
      "  - rorcid, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/BiocPkgTools.html\n",
      "\n",
      "Name: clusterStab\n",
      "Description:\n",
      "This package can be used to estimate the number of clusters in a set of microarray data, as well as test the stability of these clusters.\n",
      "Version: 1.70.0\n",
      "Publication date: None\n",
      "Mantainer: James W. MacDonald <jmacdon@u.washington.edu>\n",
      "Authors:\n",
      "Author: James W. MacDonald, Debashis Ghosh, Mark Smolkin\n",
      "Dependencies:\n",
      "  - Biobase, type: depends\n",
      "  - R (>= 1.9.0) type: depends\n",
      "  - methods, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/clusterStab.html\n",
      "\n",
      "Name: deepSNV\n",
      "Description:\n",
      "This package provides provides quantitative variant callers for detecting subclonal mutations in ultra-deep (>=100x coverage) sequencing experiments. The deepSNV algorithm is used for a comparative setup with a control experiment of the same loci and uses a beta-binomial model and a likelihood ratio test to discriminate sequencing errors and subclonal SNVs. The shearwater algorithm computes a Bayes classifier based on a beta-binomial model for variant calling with multiple samples for precisely estimating model parameters - such as local error rates and dispersion - and prior knowledge, e.g. from variation data bases such as COSMIC.\n",
      "Version: 1.44.0\n",
      "Publication date: None\n",
      "Mantainer: Moritz Gerstung <moritz.gerstung@ebi.ac.uk>\n",
      "Authors:\n",
      "Author: Niko Beerenwinkel [ths], Raul Alcantara [ctb], David Jones [ctb], John Marshall [ctb], Inigo Martincorena [ctb], Moritz Gerstung [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 2.13.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - graphics, type: depends\n",
      "  - parallel, type: depends\n",
      "  - IRanges, type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - Biostrings, type: depends\n",
      "  - VGAM, type: depends\n",
      "  - VariantAnnotation, type: depends\n",
      "  - Rhtslib, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/deepSNV.html\n",
      "\n",
      "Name: discordant\n",
      "Description:\n",
      "Discordant is an R package that identifies pairs of features that correlate differently between phenotypic groups, with application to -omics data sets. Discordant uses a mixture model that âbinsâ molecular feature pairs based on their type of coexpression or coabbundance. Algorithm is explained further in \"Differential Correlation for Sequencing Data\"\" (Siska et al. 2016).\n",
      "Version: 1.22.0\n",
      "Publication date: None\n",
      "Mantainer: McGrath Max <max.mcgrath@ucdenver.edu>\n",
      "Authors:\n",
      "Author: Charlotte Siska [aut], McGrath Max [aut, cre], Katerina Kechris [aut, cph, ths]\n",
      "Dependencies:\n",
      "  - R (>= 4.1.0) type: depends\n",
      "  - Rcpp, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - stats, type: imports\n",
      "  - biwt, type: imports\n",
      "  - gtools, type: imports\n",
      "  - MASS, type: imports\n",
      "  - tools, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - methods, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/discordant.html\n",
      "\n",
      "Name: girafe\n",
      "Description:\n",
      "The package 'girafe' deals with the genome-level representation of aligned reads from next-generation sequencing data. It contains an object class for enabling a detailed description of genome intervals with aligned reads and functions for comparing, visualising, exporting and working with such intervals and the aligned reads. As such, the package interacts with and provides a link between the packages ShortRead, IRanges and genomeIntervals.\n",
      "Version: 1.50.0\n",
      "Publication date: None\n",
      "Mantainer: J. Toedling <jtoedling@yahoo.de>\n",
      "Authors:\n",
      "Author: Joern Toedling, with contributions from Constance Ciaudo, Olivier Voinnet, Edith Heard, Emmanuel Barillot, and Wolfgang Huber\n",
      "Dependencies:\n",
      "  - R (>= 2.10.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - BiocGenerics, type: depends\n",
      "  - S4Vectors, type: depends\n",
      "  - Rsamtools, type: depends\n",
      "  - intervals (>= 0.13.1) type: depends\n",
      "  - ShortRead, type: depends\n",
      "  - genomeIntervals, type: depends\n",
      "  - grid, type: depends\n",
      "  - methods, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - IRanges, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/girafe.html\n",
      "\n",
      "Name: GOexpress\n",
      "Description:\n",
      "The package contains methods to visualise the expression profile of genes from a microarray or RNA-seq experiment, and offers a supervised clustering approach to identify GO terms containing genes with expression levels that best classify two or more predefined groups of samples. Annotations for the genes present in the expression dataset may be obtained from Ensembl through the biomaRt package, if not provided by the user. The default random forest framework is used to evaluate the capacity of each gene to cluster samples according to the factor of interest. Finally, GO terms are scored by averaging the rank (alternatively, score) of their respective gene sets to cluster the samples. P-values may be computed to assess the significance of GO term ranking. Visualisation function include gene expression profile, gene ontology-based heatmaps, and hierarchical clustering of experimental samples using gene expression data.\n",
      "Version: 1.32.0\n",
      "Publication date: None\n",
      "Mantainer: Kevin Rue-Albrecht <kevinrue67@gmail.com>\n",
      "Authors:\n",
      "Author: Kevin Rue-Albrecht [aut, cre], Tharvesh M.L. Ali [ctb], Paul A. McGettigan [ctb], Belinda Hernandez [ctb], David A. Magee [ctb], Nicolas C. Nalpas [ctb], Andrew Parnell [ctb], Stephen V. Gordon [ths], David E. MacHugh [ths]\n",
      "Dependencies:\n",
      "  - R (>= 3.4) type: depends\n",
      "  - grid, type: depends\n",
      "  - stats, type: depends\n",
      "  - graphics, type: depends\n",
      "  - Biobase, type: depends\n",
      "  - biomaRt, type: imports\n",
      "  - stringr (>= 0.6.2) type: imports\n",
      "  - ggplot2 (>= 0.9.0) type: imports\n",
      "  - RColorBrewer (>= 1.0) type: imports\n",
      "  - gplots (>= 2.13.0) type: imports\n",
      "  - randomForest (>= 4.6) type: imports\n",
      "  - RCurl (>= 1.95) type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 3)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/GOexpress.html\n",
      "\n",
      "Name: gpls\n",
      "Description:\n",
      "Classification using generalized partial least squares for two-group and multi-group (more than 2 group) classification.\n",
      "Version: 1.70.0\n",
      "Publication date: None\n",
      "Mantainer: Bioconductor Package Maintainer <maintainer@bioconductor.org>\n",
      "Authors:\n",
      "Author: Beiying Ding\n",
      "Dependencies:\n",
      "  - stats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/gpls.html\n",
      "\n",
      "Name: iCARE\n",
      "Description:\n",
      "An R package to compute Individualized Coherent Absolute Risk Estimators.\n",
      "Version: 1.26.0\n",
      "Publication date: None\n",
      "Mantainer: Bill Wheeler <wheelerb@imsweb.com>\n",
      "Authors:\n",
      "Author: Paige Maas, Parichoy Pal Choudhury, Nilanjan Chatterjee and William Wheeler\n",
      "Dependencies:\n",
      "  - R (>= 3.3.0) type: depends\n",
      "  - plotrix, type: depends\n",
      "  - gtools, type: depends\n",
      "  - Hmisc, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3 + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/iCARE.html\n",
      "\n",
      "Name: mgsa\n",
      "Description:\n",
      "Model-based Gene Set Analysis (MGSA) is a Bayesian modeling approach for gene set enrichment. The package mgsa implements MGSA and tools to use MGSA together with the Gene Ontology.\n",
      "Version: 1.46.0\n",
      "Publication date: None\n",
      "Mantainer: Sebastian Bauer <mail@sebastianbauer.info>\n",
      "Authors:\n",
      "Author: Sebastian Bauer <mail at sebastianbauer.info>, Julien Gagneur <gagneur at genzentrum.lmu.de>\n",
      "Dependencies:\n",
      "  - R (>= 2.14.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - gplots, type: depends\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/mgsa.html\n",
      "\n",
      "Name: MWASTools\n",
      "Description:\n",
      "MWASTools provides a complete pipeline to perform metabolome-wide association studies. Key functionalities of the package include: quality control analysis of metabonomic data; MWAS using different association models (partial correlations; generalized linear models); model validation using non-parametric bootstrapping; visualization of MWAS results; NMR metabolite identification using STOCSY; and biological interpretation of MWAS results.\n",
      "Version: 1.22.0\n",
      "Publication date: None\n",
      "Mantainer: Andrea Rodriguez-Martinez <andrea.rodriguez-martinez13@imperial.ac.uk>, Rafael Ayala <rafael.ayala@oist.jp>\n",
      "Authors:\n",
      "Author: Andrea Rodriguez-Martinez, Joram M. Posma, Rafael Ayala, Ana L. Neves, Maryam Anwar, Jeremy K. Nicholson, Marc-Emmanuel Dumas\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - glm2, type: imports\n",
      "  - ppcor, type: imports\n",
      "  - qvalue, type: imports\n",
      "  - car, type: imports\n",
      "  - boot, type: imports\n",
      "  - grid, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - gridExtra, type: imports\n",
      "  - igraph, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - KEGGgraph, type: imports\n",
      "  - RCurl, type: imports\n",
      "  - KEGGREST, type: imports\n",
      "  - ComplexHeatmap, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "CC BY-NC-ND 4.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/MWASTools.html\n",
      "\n",
      "Name: velociraptor\n",
      "Description:\n",
      "This package provides Bioconductor-friendly wrappers for RNA velocity calculations in single-cell RNA-seq data. We use the basilisk package to manage Conda environments, and the zellkonverter package to convert data structures between SingleCellExperiment (R) and AnnData (Python). The information produced by the velocity methods is stored in the various components of the SingleCellExperiment class.\n",
      "Version: 1.8.0\n",
      "Publication date: None\n",
      "Mantainer: Kevin Rue-Albrecht <kevinrue67@gmail.com>\n",
      "Authors:\n",
      "Author: Kevin Rue-Albrecht [aut, cre] , Aaron Lun [aut] , Charlotte Soneson [aut] , Michael Stadler [aut]\n",
      "Dependencies:\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - reticulate, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - DelayedArray, type: imports\n",
      "  - basilisk, type: imports\n",
      "  - zellkonverter, type: imports\n",
      "  - scuttle, type: imports\n",
      "  - SingleCellExperiment, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - BiocSingular, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/velociraptor.html\n",
      "\n",
      "Name: vidger\n",
      "Description:\n",
      "The aim of vidger is to rapidly generate information-rich visualizations for the interpretation of differential gene expression results from three widely-used tools: Cuffdiff, DESeq2, and edgeR.\n",
      "Version: 1.18.0\n",
      "Publication date: None\n",
      "Mantainer: Brandon Monier <brandon.monier@gmail.com>\n",
      "Authors:\n",
      "Author: Brandon Monier [aut, cre], Adam McDermaid [aut], Jing Zhao [aut], Qin Ma [aut, fnd]\n",
      "Dependencies:\n",
      "  - R (>= 3.5) type: depends\n",
      "  - Biobase, type: imports\n",
      "  - DESeq2, type: imports\n",
      "  - edgeR, type: imports\n",
      "  - GGally, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - ggrepel, type: imports\n",
      "  - knitr, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - rmarkdown, type: imports\n",
      "  - scales, type: imports\n",
      "  - stats, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3 | file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/vidger.html\n",
      "\n",
      "Name: animalcules\n",
      "Description:\n",
      "animalcules is an R package for utilizing up-to-date data analytics, visualization methods, and machine learning models to provide users an easy-to-use interactive microbiome analysis framework. It can be used as a standalone software package or users can explore their data with the accompanying interactive R Shiny application. Traditional microbiome analysis such as alpha/beta diversity and differential abundance analysis are enhanced, while new methods like biomarker identification are introduced by animalcules. Powerful interactive and dynamic figures generated by animalcules enable users to understand their data better and discover new insights.\n",
      "Version: 1.14.0\n",
      "Publication date: None\n",
      "Mantainer: Yue Zhao <yuezh@bu.edu>\n",
      "Authors:\n",
      "Author: Yue Zhao [aut, cre] , Anthony Federico [aut] , W. Evan Johnson [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.0.0) type: depends\n",
      "  - assertthat, type: imports\n",
      "  - shiny, type: imports\n",
      "  - shinyjs, type: imports\n",
      "  - DESeq2, type: imports\n",
      "  - caret, type: imports\n",
      "  - plotly, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - rentrez, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - covr, type: imports\n",
      "  - ape, type: imports\n",
      "  - vegan, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - MultiAssayExperiment, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - XML, type: imports\n",
      "  - forcats, type: imports\n",
      "  - scales, type: imports\n",
      "  - lattice, type: imports\n",
      "  - glmnet, type: imports\n",
      "  - tsne, type: imports\n",
      "  - plotROC, type: imports\n",
      "  - DT, type: imports\n",
      "  - utils, type: imports\n",
      "  - limma, type: imports\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - tibble, type: imports\n",
      "  - biomformat, type: imports\n",
      "  - umap, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - GUniFrac, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/animalcules.html\n",
      "\n",
      "Name: chipenrich\n",
      "Description:\n",
      "ChIP-Enrich and Poly-Enrich perform gene set enrichment testing using peaks called from a ChIP-seq experiment. The method empirically corrects for confounding factors such as the length of genes, and the mappability of the sequence surrounding genes.\n",
      "Version: 2.22.0\n",
      "Publication date: None\n",
      "Mantainer: Raymond G. Cavalcante <rcavalca@umich.edu>\n",
      "Authors:\n",
      "Author: Ryan P. Welch [aut, cph], Chee Lee [aut], Raymond G. Cavalcante [aut], Kai Wang [cre], Chris Lee [aut], Laura J. Scott [ths], Maureen A. Sartor [ths]\n",
      "Dependencies:\n",
      "  - R (>= 3.4.0) type: depends\n",
      "  - AnnotationDbi, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - chipenrich.data, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - grid, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - lattice, type: imports\n",
      "  - latticeExtra, type: imports\n",
      "  - MASS, type: imports\n",
      "  - methods, type: imports\n",
      "  - mgcv, type: imports\n",
      "  - org.Dm.eg.db, type: imports\n",
      "  - org.Dr.eg.db, type: imports\n",
      "  - org.Hs.eg.db, type: imports\n",
      "  - org.Mm.eg.db, type: imports\n",
      "  - org.Rn.eg.db, type: imports\n",
      "  - parallel, type: imports\n",
      "  - plyr, type: imports\n",
      "  - rms, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - stats, type: imports\n",
      "  - stringr, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/chipenrich.html\n",
      "\n",
      "Name: ChIPseqR\n",
      "Description:\n",
      "ChIPseqR identifies protein binding sites from ChIP-seq and nucleosome positioning experiments. The model used to describe binding events was developed to locate nucleosomes but should flexible enough to handle other types of experiments as well.\n",
      "Version: 1.52.0\n",
      "Publication date: None\n",
      "Mantainer: Peter Humburg <peter.humburg@gmail.com>\n",
      "Authors:\n",
      "Author: Peter Humburg\n",
      "Dependencies:\n",
      "  - R (>= 2.10.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - BiocGenerics, type: depends\n",
      "  - S4Vectors, type: depends\n",
      "  - Biostrings, type: imports\n",
      "  - fBasics, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - HilbertVis, type: imports\n",
      "  - ShortRead, type: imports\n",
      "  - stats, type: imports\n",
      "  - timsac, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ChIPseqR.html\n",
      "\n",
      "Name: CNTools\n",
      "Description:\n",
      "This package provides tools to convert the output of segmentation analysis using DNAcopy to a matrix structure with overlapping segments as rows and samples as columns so that other computational analyses can be applied to segmented data\n",
      "Version: 1.54.0\n",
      "Publication date: None\n",
      "Mantainer: J. Zhang <jzhang@jimmy.harvard.edu>\n",
      "Authors:\n",
      "Author: Jianhua Zhang\n",
      "Dependencies:\n",
      "  - R (>= 2.10) type: depends\n",
      "  - methods, type: depends\n",
      "  - tools, type: depends\n",
      "  - stats, type: depends\n",
      "  - genefilter, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/CNTools.html\n",
      "\n",
      "Name: EWCE\n",
      "Description:\n",
      "Bioconductor version: Release (3.16)\n",
      "Version: 1.6.0\n",
      "Publication date: None\n",
      "Mantainer: Author: Alan Murphy [cre] , Brian Schilder [aut] , Nathan Skene [aut]\n",
      "Authors:\n",
      "Used to determine which cell types are enriched within gene lists. The package provides tools for testing enrichments within simple gene lists (such as human disease associated genes) and those resulting from differential expression studies. The package does not depend upon any particular Single Cell Transcriptome dataset and user defined datasets can be loaded in and used in the analyses.\n",
      "Dependencies:\n",
      "  - R (>= 4.2) type: depends\n",
      "  - RNOmni (>= 1.0) type: depends\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - methods, type: imports\n",
      "  - ewceData, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - limma, type: imports\n",
      "  - stringr, type: imports\n",
      "  - HGNChelper, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - parallel, type: imports\n",
      "  - SingleCellExperiment, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - DelayedArray, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - orthogene, type: imports\n",
      "  - data.table, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/EWCE.html\n",
      "\n",
      "Name: MetaboSignal\n",
      "Description:\n",
      "MetaboSignal is an R package that allows merging, analyzing and customizing metabolic and signaling KEGG pathways. It is a network-based approach designed to explore the topological relationship between genes (signaling- or enzymatic-genes) and metabolites, representing a powerful tool to investigate the genetic landscape and regulatory networks of metabolic phenotypes.\n",
      "Version: 1.28.0\n",
      "Publication date: None\n",
      "Mantainer: Andrea Rodriguez-Martinez <andrea.rodriguez-martinez13@imperial.ac.uk>, Rafael Ayala <rafaelayalahernandez@gmail.com>\n",
      "Authors:\n",
      "Author: Andrea Rodriguez-Martinez, Rafael Ayala, Joram M. Posma, Ana L. Neves, Maryam Anwar, Jeremy K. Nicholson, Marc-Emmanuel Dumas\n",
      "Dependencies:\n",
      "  - R (>= 3.3) type: depends\n",
      "  - KEGGgraph, type: imports\n",
      "  - hpar, type: imports\n",
      "  - igraph, type: imports\n",
      "  - RCurl, type: imports\n",
      "  - KEGGREST, type: imports\n",
      "  - EnsDb.Hsapiens.v75, type: imports\n",
      "  - stats, type: imports\n",
      "  - graphics, type: imports\n",
      "  - utils, type: imports\n",
      "  - org.Hs.eg.db, type: imports\n",
      "  - biomaRt, type: imports\n",
      "  - AnnotationDbi, type: imports\n",
      "  - MWASTools, type: imports\n",
      "  - mygene, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/MetaboSignal.html\n",
      "\n",
      "Name: OCplus\n",
      "Description:\n",
      "This package allows to characterize the operating characteristics of a microarray experiment, i.e. the trade-off between false discovery rate and the power to detect truly regulated genes. The package includes tools both for planned experiments (for sample size assessment) and for already collected data (identification of differentially expressed genes).\n",
      "Version: 1.72.0\n",
      "Publication date: None\n",
      "Mantainer: Alexander Ploner <Alexander.Ploner@ki.se>\n",
      "Authors:\n",
      "Author: Yudi Pawitan <Yudi.Pawitan at ki.se> and Alexander Ploner <Alexander.Ploner at ki.se>\n",
      "Dependencies:\n",
      "  - R (>= 2.1.0) type: depends\n",
      "  - multtest, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - stats, type: imports\n",
      "  - interp, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/OCplus.html\n",
      "\n",
      "Name: TreeAndLeaf\n",
      "Description:\n",
      "The TreeAndLeaf package combines unrooted and force-directed graph algorithms in order to layout binary trees, aiming to represent multiple layers of information onto dendrogram leaves.\n",
      "Version: 1.10.0\n",
      "Publication date: None\n",
      "Mantainer: Milena A. Cardoso <milenandreuzo@gmail.com>\n",
      "Authors:\n",
      "Author: Leonardo W. Kume, Luis E. A. Rizzardi, Milena A. Cardoso, Mauro A. A. Castro\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - RedeR, type: imports\n",
      "  - igraph, type: imports\n",
      "  - ape, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/TreeAndLeaf.html\n",
      "\n",
      "Name: biodb\n",
      "Description:\n",
      "The biodb package provides access to standard remote chemical and biological databases (ChEBI, KEGG, HMDB, ...), as well as to in-house local database files (CSV, SQLite), with easy retrieval of entries, access to web services, search of compounds by mass and/or name, and mass spectra matching for LCMS and MSMS. Its architecture as a development framework facilitates the development of new database connectors for local projects or inside separate published packages.\n",
      "Version: 1.6.1\n",
      "Publication date: None\n",
      "Mantainer: Pierrick Roger <pierrick.roger@cea.fr>\n",
      "Authors:\n",
      "Author: Pierrick Roger [aut, cre] , Alexis DelabriÃ¨re [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 4.1.0) type: depends\n",
      "  - BiocFileCache, type: imports\n",
      "  - R6, type: imports\n",
      "  - RCurl, type: imports\n",
      "  - RSQLite, type: imports\n",
      "  - Rcpp, type: imports\n",
      "  - XML, type: imports\n",
      "  - chk, type: imports\n",
      "  - jsonlite, type: imports\n",
      "  - lgr, type: imports\n",
      "  - lifecycle, type: imports\n",
      "  - methods, type: imports\n",
      "  - openssl, type: imports\n",
      "  - plyr, type: imports\n",
      "  - progress, type: imports\n",
      "  - rappdirs, type: imports\n",
      "  - stats, type: imports\n",
      "  - stringr, type: imports\n",
      "  - tools, type: imports\n",
      "  - withr, type: imports\n",
      "  - yaml, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "AGPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/biodb.html\n",
      "\n",
      "Name: CopyNumberPlots\n",
      "Description:\n",
      "CopyNumberPlots have a set of functions extending karyoploteRs functionality to create beautiful, customizable and flexible plots of copy-number related data.\n",
      "Version: 1.14.0\n",
      "Publication date: None\n",
      "Mantainer: Bernat Gel <bgel@igtp.cat>\n",
      "Authors:\n",
      "Author: Bernat Gel <bgel at igtp.cat> and Miriam Magallon <mmagallon at igtp.cat>\n",
      "Dependencies:\n",
      "  - R (>= 3.6) type: depends\n",
      "  - karyoploteR, type: depends\n",
      "  - regioneR, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - VariantAnnotation, type: imports\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - cn.mops, type: imports\n",
      "  - rhdf5, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/CopyNumberPlots.html\n",
      "\n",
      "Name: gaia\n",
      "Description:\n",
      "Bioconductor version: Release (3.16)\n",
      "Version: 2.42.0\n",
      "Publication date: None\n",
      "Mantainer: Author: Sandro Morganella et al.\n",
      "Authors:\n",
      "This package allows to assess the statistical significance of chromosomal aberrations.\n",
      "Dependencies:\n",
      "  - R (>= 2.10) type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/gaia.html\n",
      "\n",
      "Name: NanoStringDiff\n",
      "Description:\n",
      "This Package utilizes a generalized linear model(GLM) of the negative binomial family to characterize count data and allows for multi-factor design. NanoStrongDiff incorporate size factors, calculated from positive controls and housekeeping controls, and background level, obtained from negative controls, in the model framework so that all the normalization information provided by NanoString nCounter Analyzer is fully utilized.\n",
      "Version: 1.28.0\n",
      "Publication date: None\n",
      "Mantainer: tingting zhai <tingting.zhai@uky.edu>,hong wang <hong.wang@uky.edu>\n",
      "Authors:\n",
      "Author: hong wang <hong.wang at uky.edu>, tingting zhai <tingting.zhai at uky.edu>, chi wang <chi.wang at uky.edu>\n",
      "Dependencies:\n",
      "  - Biobase, type: depends\n",
      "  - matrixStats, type: imports\n",
      "  - methods, type: imports\n",
      "  - Rcpp, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/NanoStringDiff.html\n",
      "\n",
      "Name: planet\n",
      "Description:\n",
      "This package contains R functions to infer additional biological variables to supplemental DNA methylation analysis of placental data. This includes inferring ethnicity/ancestry, gestational age, and cell composition from placental DNA methylation array (450k/850k) data. The package comes with an example processed placental dataset.\n",
      "Version: 1.6.0\n",
      "Publication date: None\n",
      "Mantainer: Victor Yuan <victor.2wy@gmail.com>\n",
      "Authors:\n",
      "Author: Victor Yuan [aut, cre], Wendy P. Robinson [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - methods, type: imports\n",
      "  - tibble, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - dplyr, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/planet.html\n",
      "\n",
      "Name: rebook\n",
      "Description:\n",
      "Provides utilities to re-use content across chapters of a Bioconductor book. This is mostly based on functionality developed while writing the OSCA book, but generalized for potential use in other large books with heavy compute. Also contains some functions to assist book deployment.\n",
      "Version: 1.8.0\n",
      "Publication date: None\n",
      "Mantainer: Aaron Lun <infinite.monkeys.with.keyboards@gmail.com>\n",
      "Authors:\n",
      "Author: Aaron Lun [aut, cre, cph]\n",
      "Dependencies:\n",
      "  - utils, type: imports\n",
      "  - methods, type: imports\n",
      "  - knitr (>= 1.32) type: imports\n",
      "  - rmarkdown, type: imports\n",
      "  - CodeDepends, type: imports\n",
      "  - dir.expiry, type: imports\n",
      "  - filelock, type: imports\n",
      "  - BiocStyle, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/rebook.html\n",
      "\n",
      "Name: RnaSeqSampleSize\n",
      "Description:\n",
      "Bioconductor version: Release (3.16)\n",
      "Version: 2.8.0\n",
      "Publication date: None\n",
      "Mantainer: Author: Shilin Zhao Developer [aut, cre], Chung-I Li [aut], Yan Guo [aut], Quanhu Sheng [aut], Yu Shyr [aut]\n",
      "Authors:\n",
      "RnaSeqSampleSize package provides a sample size calculation method based on negative binomial model and the exact test for assessing differential expression analysis of RNA-seq data. It controls FDR for multiple testing and utilizes the average read count and dispersion distributions from real data to estimate a more reliable sample size. It is also equipped with several unique features, including estimation for interested genes or pathway, power curve visualization, and parameter optimization.\n",
      "Dependencies:\n",
      "  - R (>= 4.0.0) type: depends\n",
      "  - ggplot2, type: depends\n",
      "  - RnaSeqSampleSizeData, type: depends\n",
      "  - biomaRt, type: imports\n",
      "  - edgeR, type: imports\n",
      "  - heatmap3, type: imports\n",
      "  - matlab, type: imports\n",
      "  - KEGGREST, type: imports\n",
      "  - methods, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - Rcpp (>= 0.11.2) type: imports\n",
      "  - recount, type: imports\n",
      "  - ggpubr, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - tidyselect, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/RnaSeqSampleSize.html\n",
      "\n",
      "Name: ChIPsim\n",
      "Description:\n",
      "A general framework for the simulation of ChIP-seq data. Although currently focused on nucleosome positioning the package is designed to support different types of experiments.\n",
      "Version: 1.52.0\n",
      "Publication date: None\n",
      "Mantainer: Peter Humburg <Peter.Humburg@gmail.com>\n",
      "Authors:\n",
      "Author: Peter Humburg\n",
      "Dependencies:\n",
      "  - Biostrings, type: depends\n",
      "  - IRanges, type: imports\n",
      "  - XVector, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - ShortRead, type: imports\n",
      "  - graphics, type: imports\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ChIPsim.html\n",
      "\n",
      "Name: coseq\n",
      "Description:\n",
      "Co-expression analysis for expression profiles arising from high-throughput sequencing data. Feature (e.g., gene) profiles are clustered using adapted transformations and mixture models or a K-means algorithm, and model selection criteria (to choose an appropriate number of clusters) are provided.\n",
      "Version: 1.22.0\n",
      "Publication date: None\n",
      "Mantainer: Andrea Rau <andrea.rau@inrae.fr>\n",
      "Authors:\n",
      "Author: Andrea Rau [cre, aut] , Cathy Maugis-Rabusseau [ctb], Antoine Godichon-Baggioni [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 4.0.0) type: depends\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - S4Vectors, type: depends\n",
      "  - edgeR, type: imports\n",
      "  - DESeq2, type: imports\n",
      "  - capushe, type: imports\n",
      "  - Rmixmod, type: imports\n",
      "  - e1071, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - scales, type: imports\n",
      "  - HTSFilter, type: imports\n",
      "  - corrplot, type: imports\n",
      "  - HTSCluster, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - methods, type: imports\n",
      "  - compositions, type: imports\n",
      "  - mvtnorm, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/coseq.html\n",
      "\n",
      "Name: ddCt\n",
      "Description:\n",
      "The Delta-Delta-Ct (ddCt) Algorithm is an approximation method to determine relative gene expression with quantitative real-time PCR (qRT-PCR) experiments. Compared to other approaches, it requires no standard curve for each primer-target pair, therefore reducing the working load and yet returning accurate enough results as long as the assumptions of the amplification efficiency hold. The ddCt package implements a pipeline to collect, analyse and visualize qRT-PCR results, for example those from TaqMan SDM software, mainly using the ddCt method. The pipeline can be either invoked by a script in command-line or through the API consisting of S4-Classes, methods and functions.\n",
      "Version: 1.54.0\n",
      "Publication date: None\n",
      "Mantainer: Jitao David Zhang <jitao_david.zhang@roche.com>\n",
      "Authors:\n",
      "Author: Jitao David Zhang, Rudolf Biczok, and Markus Ruschhaupt\n",
      "Dependencies:\n",
      "  - R (>= 2.3.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - Biobase, type: imports\n",
      "  - RColorBrewer (>= 0.1-3) type: imports\n",
      "  - xtable, type: imports\n",
      "  - lattice, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ddCt.html\n",
      "\n",
      "Name: eisaR\n",
      "Description:\n",
      "Exon-intron split analysis (EISA) uses ordinary RNA-seq data to measure changes in mature RNA and pre-mRNA reads across different experimental conditions to quantify transcriptional and post-transcriptional regulation of gene expression. For details see Gaidatzis et al., Nat Biotechnol 2015. doi: 10.1038/nbt.3269. eisaR implements the major steps of EISA in R.\n",
      "Version: 1.10.0\n",
      "Publication date: None\n",
      "Mantainer: Michael Stadler <michael.stadler@fmi.ch>\n",
      "Authors:\n",
      "Author: Michael Stadler [aut, cre], Dimos Gaidatzis [aut], Lukas Burger [aut], Charlotte Soneson [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - limma, type: imports\n",
      "  - edgeR, type: imports\n",
      "  - methods, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/eisaR.html\n",
      "\n",
      "Name: geNetClassifier\n",
      "Description:\n",
      "Comprehensive package to automatically train and validate a multi-class SVM classifier based on gene expression data. Provides transparent selection of gene markers, their coexpression networks, and an interface to query the classifier.\n",
      "Version: 1.38.0\n",
      "Publication date: None\n",
      "Mantainer: Sara Aibar <saibar@usal.es>\n",
      "Authors:\n",
      "Author: Sara Aibar, Celia Fontanillo and Javier De Las Rivas. Bioinformatics and Functional Genomics Group. Cancer Research Center (CiC-IBMCC, CSIC/USAL). Salamanca. Spain.\n",
      "Dependencies:\n",
      "  - R (>= 2.10.1) type: depends\n",
      "  - Biobase, type: depends\n",
      "  - EBarrays, type: depends\n",
      "  - minet, type: depends\n",
      "  - methods, type: depends\n",
      "  - e1071, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/geNetClassifier.html\n",
      "\n",
      "Name: glmSparseNet\n",
      "Description:\n",
      "glmSparseNet is an R-package that generalizes sparse regression models when the features (e.g. genes) have a graph structure (e.g. protein-protein interactions), by including network-based regularizers. glmSparseNet uses the glmnet R-package, by including centrality measures of the network as penalty weights in the regularization. The current version implements regularization based on node degree, i.e. the strength and/or number of its associated edges, either by promoting hubs in the solution or orphan genes in the solution. All the glmnet distribution families are supported, namely \"gaussian\", \"poisson\", \"binomial\", \"multinomial\", \"cox\", and \"mgaussian\".\n",
      "Version: 1.16.0\n",
      "Publication date: None\n",
      "Mantainer: AndrÃ© VerÃ­ssimo <andre.verissimo@tecnico.ulisboa.pt>\n",
      "Authors:\n",
      "Author: AndrÃ© VerÃ­ssimo [aut, cre], Susana Vinga [aut], Eunice Carrasquinha [ctb], Marta Lopes [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - Matrix, type: depends\n",
      "  - MultiAssayExperiment, type: depends\n",
      "  - glmnet, type: depends\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - biomaRt, type: imports\n",
      "  - futile.logger, type: imports\n",
      "  - futile.options, type: imports\n",
      "  - forcats, type: imports\n",
      "  - utils, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - glue, type: imports\n",
      "  - readr, type: imports\n",
      "  - digest, type: imports\n",
      "  - httr, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - survminer, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - stringr, type: imports\n",
      "  - parallel, type: imports\n",
      "  - methods, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/glmSparseNet.html\n",
      "\n",
      "Name: GWENA\n",
      "Description:\n",
      "The development of high-throughput sequencing led to increased use of co-expression analysis to go beyong single feature (i.e. gene) focus. We propose GWENA (Gene Whole co-Expression Network Analysis) , a tool designed to perform gene co-expression network analysis and explore the results in a single pipeline. It includes functional enrichment of modules of co-expressed genes, phenotypcal association, topological analysis and comparison of networks configuration between conditions.\n",
      "Version: 1.8.0\n",
      "Publication date: None\n",
      "Mantainer: GwenaÃ«lle Lemoine <lemoine.gwenaelle@gmail.com>\n",
      "Authors:\n",
      "Author: GwenaÃ«lle Lemoine [aut, cre] , Marie-Pier Scott-Boyer [ths], Arnaud Droit [fnd]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - WGCNA (>= 1.67) type: imports\n",
      "  - dplyr (>= 0.8.3) type: imports\n",
      "  - dynamicTreeCut (>= 1.63-1) type: imports\n",
      "  - ggplot2 (>= 3.1.1) type: imports\n",
      "  - gprofiler2 (>= 0.1.6) type: imports\n",
      "  - magrittr (>= 1.5) type: imports\n",
      "  - tibble (>= 2.1.1) type: imports\n",
      "  - tidyr (>= 1.0.0) type: imports\n",
      "  - NetRep (>= 1.2.1) type: imports\n",
      "  - igraph (>= 1.2.4.1) type: imports\n",
      "  - RColorBrewer (>= 1.1-2) type: imports\n",
      "  - purrr (>= 0.3.3) type: imports\n",
      "  - rlist (>= 0.4.6.1) type: imports\n",
      "  - matrixStats (>= 0.55.0) type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - stringr (>= 1.4.0) type: imports\n",
      "  - cluster (>= 2.1.0) type: imports\n",
      "  - grDevices (>= 4.0.4) type: imports\n",
      "  - methods, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/GWENA.html\n",
      "\n",
      "Name: miQC\n",
      "Description:\n",
      "Single-cell RNA-sequencing (scRNA-seq) has made it possible to profile gene expression in tissues at high resolution. An important preprocessing step prior to performing downstream analyses is to identify and remove cells with poor or degraded sample quality using quality control (QC) metrics. Two widely used QC metrics to identify a âlow-qualityâ cell are (i) if the cell includes a high proportion of reads that map to mitochondrial DNA encoded genes (mtDNA) and (ii) if a small number of genes are detected. miQC is data-driven QC metric that jointly models both the proportion of reads mapping to mtDNA and the number of detected genes with mixture models in a probabilistic framework to predict the low-quality cells in a given dataset.\n",
      "Version: 1.6.0\n",
      "Publication date: None\n",
      "Mantainer: Ariel Hippen <ariel.hippen@gmail.com>\n",
      "Authors:\n",
      "Author: Ariel Hippen [aut, cre], Stephanie Hicks [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - SingleCellExperiment, type: imports\n",
      "  - flexmix, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - splines, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "BSD_3_clause + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/miQC.html\n",
      "\n",
      "Name: OrderedList\n",
      "Description:\n",
      "Detection of similarities between ordered lists of genes. Thereby, either simple lists can be compared or gene expression data can be used to deduce the lists. Significance of similarities is evaluated by shuffling lists or by resampling in microarray data, respectively.\n",
      "Version: 1.70.0\n",
      "Publication date: None\n",
      "Mantainer: Claudio Lottaz <Claudio.Lottaz@klinik.uni-regensburg.de>\n",
      "Authors:\n",
      "Author: Xinan Yang, Stefanie Scheid, Claudio Lottaz\n",
      "Dependencies:\n",
      "  - R (>= 3.6.1) type: depends\n",
      "  - Biobase, type: depends\n",
      "  - twilight, type: depends\n",
      "  - methods, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/OrderedList.html\n",
      "\n",
      "Name: statTarget\n",
      "Description:\n",
      "A streamlined tool provides a graphical user interface for quality control based signal drift correction (QC-RFSC), integration of data from multi-batch MS-based experiments, and the comprehensive statistical analysis in metabolomics and proteomics.\n",
      "Version: 1.28.0\n",
      "Publication date: None\n",
      "Mantainer: Hemi Luan <hemi.luan@gmail.com>\n",
      "Authors:\n",
      "Author: Hemi Luan\n",
      "Dependencies:\n",
      "  - R (>= 3.6.0) type: depends\n",
      "  - randomForest, type: imports\n",
      "  - plyr, type: imports\n",
      "  - pdist, type: imports\n",
      "  - ROC, type: imports\n",
      "  - utils, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - rrcov, type: imports\n",
      "  - stats, type: imports\n",
      "  - pls, type: imports\n",
      "  - impute, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL (>= 3)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/statTarget.html\n",
      "\n",
      "Name: struct\n",
      "Description:\n",
      "Defines and includes a set of class-based templates for developing and implementing data processing and analysis workflows, with a strong emphasis on statistics and machine learning. The templates can be used and where needed extended to 'wrap' tools and methods from other packages into a common standardised structure to allow for effective and fast integration. Model objects can be combined into sequences, and sequences nested in iterators using overloaded operators to simplify and improve readability of the code. Ontology lookup has been integrated and implemented to provide standardised definitions for methods, inputs and outputs wrapped using the class-based templates.\n",
      "Version: 1.10.0\n",
      "Publication date: None\n",
      "Mantainer: Gavin Rhys Lloyd <g.r.lloyd@bham.ac.uk>\n",
      "Authors:\n",
      "Author: Gavin Rhys Lloyd [aut, cre], Ralf Johannes Maria Weber [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - methods, type: imports\n",
      "  - ontologyIndex, type: imports\n",
      "  - datasets, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - knitr, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - rols, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/struct.html\n",
      "\n",
      "Name: tweeDEseq\n",
      "Description:\n",
      "Differential expression analysis of RNA-seq using the Poisson-Tweedie family of distributions.\n",
      "Version: 1.44.0\n",
      "Publication date: None\n",
      "Mantainer: Juan R Gonzalez <jrgonzalez@creal.cat>\n",
      "Authors:\n",
      "Author: Juan R Gonzalez <jrgonzalez at creal.cat> and Mikel Esnaola <mesnaola at creal.cat> (with contributions from Robert Castelo <robert.castelo at upf.edu>)\n",
      "Dependencies:\n",
      "  - R (>= 2.12.0) type: depends\n",
      "  - MASS, type: imports\n",
      "  - limma, type: imports\n",
      "  - edgeR, type: imports\n",
      "  - parallel, type: imports\n",
      "  - cqn, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/tweeDEseq.html\n",
      "\n",
      "Name: bacon\n",
      "Description:\n",
      "Bacon can be used to remove inflation and bias often observed in epigenome- and transcriptome-wide association studies. To this end bacon constructs an empirical null distribution using a Gibbs Sampling algorithm by fitting a three-component normal mixture on z-scores.\n",
      "Version: 1.26.0\n",
      "Publication date: None\n",
      "Mantainer: Maarten van Iterson <mviterson@gmail.com>\n",
      "Authors:\n",
      "Author: Maarten van Iterson [aut, cre], Erik van Zwet [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 3.3) type: depends\n",
      "  - methods, type: depends\n",
      "  - stats, type: depends\n",
      "  - ggplot2, type: depends\n",
      "  - graphics, type: depends\n",
      "  - BiocParallel, type: depends\n",
      "  - ellipse, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/bacon.html\n",
      "\n",
      "Name: GeneticsPed\n",
      "Description:\n",
      "Classes and methods for handling pedigree data. It also includes functions to calculate genetic relationship measures as relationship and inbreeding coefficients and other utilities. Note that package is not yet stable. Use it with care!\n",
      "Version: 1.60.3\n",
      "Publication date: None\n",
      "Mantainer: David Henderson <DNADavenator@GMail.Com>\n",
      "Authors:\n",
      "Author: Gregor Gorjanc and David A. Henderson <DNADavenator at GMail.Com>, with code contributions by Brian Kinghorn and Andrew Percy (see file COPYING)\n",
      "Dependencies:\n",
      "  - R (>= 2.4.0) type: depends\n",
      "  - MASS, type: depends\n",
      "  - gdata, type: imports\n",
      "  - genetics, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL (>= 2.1) | file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/GeneticsPed.html\n",
      "\n",
      "Name: HIBAG\n",
      "Description:\n",
      "Imputes HLA classical alleles using GWAS SNP data, and it relies on a training set of HLA and SNP genotypes. HIBAG can be used by researchers with published parameter estimates instead of requiring access to large training sample datasets. It combines the concepts of attribute bagging, an ensemble classifier method, with haplotype inference for SNPs and HLA types. Attribute bagging is a technique which improves the accuracy and stability of classifier ensembles using bootstrap aggregating and random variable selection.\n",
      "Version: 1.34.1\n",
      "Publication date: None\n",
      "Mantainer: Xiuwen Zheng <zhengx@u.washington.edu>\n",
      "Authors:\n",
      "Author: Xiuwen Zheng [aut, cre, cph] , Bruce Weir [ctb, ths]\n",
      "Dependencies:\n",
      "  - R (>= 3.2.0) type: depends\n",
      "  - methods, type: imports\n",
      "  - RcppParallel, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/HIBAG.html\n",
      "\n",
      "Name: motifbreakR\n",
      "Description:\n",
      "We introduce motifbreakR, which allows the biologist to judge in the first place whether the sequence surrounding the polymorphism is a good match, and in the second place how much information is gained or lost in one allele of the polymorphism relative to another. MotifbreakR is both flexible and extensible over previous offerings; giving a choice of algorithms for interrogation of genomes with motifs from public sources that users can choose from; these are 1) a weighted-sum probability matrix, 2) log-probabilities, and 3) weighted by relative entropy. MotifbreakR can predict effects for novel or previously described variants in public databases, making it suitable for tasks beyond the scope of its original design. Lastly, it can be used to interrogate any genome curated within Bioconductor (currently there are 32 species, a total of 109 versions).\n",
      "Version: 2.12.0\n",
      "Publication date: None\n",
      "Mantainer: Simon Gert Coetzee <simon.coetzee@cshs.org>\n",
      "Authors:\n",
      "Author: Simon Gert Coetzee [aut, cre], Dennis J. Hazelett [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.1.0) type: depends\n",
      "  - grid, type: depends\n",
      "  - MotifDb, type: depends\n",
      "  - methods, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - stringr, type: imports\n",
      "  - parallel, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - BSgenome, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - VariantAnnotation, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - motifStack, type: imports\n",
      "  - Gviz, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - TFMPvalue, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/motifbreakR.html\n",
      "\n",
      "Name: openPrimeR\n",
      "Description:\n",
      "An implementation of methods for designing, evaluating, and comparing primer sets for multiplex PCR. Primers are designed by solving a set cover problem such that the number of covered template sequences is maximized with the smallest possible set of primers. To guarantee that high-quality primers are generated, only primers fulfilling constraints on their physicochemical properties are selected. A Shiny app providing a user interface for the functionalities of this package is provided by the 'openPrimeRui' package.\n",
      "Version: 1.20.0\n",
      "Publication date: None\n",
      "Mantainer: Matthias DÃ¶ring <matthias-doering@gmx.de>\n",
      "Authors:\n",
      "Author: Matthias DÃ¶ring [aut, cre], Nico Pfeifer [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.0.0) type: depends\n",
      "  - Biostrings, type: imports\n",
      "  - XML (>= 3.98-1.4) type: imports\n",
      "  - scales (>= 0.4.0) type: imports\n",
      "  - reshape2 (>= 1.4.1) type: imports\n",
      "  - seqinr (>= 3.3-3) type: imports\n",
      "  - IRanges, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - ggplot2 (>= 2.1.0) type: imports\n",
      "  - plyr (>= 1.8.4) type: imports\n",
      "  - dplyr (>= 0.5.0) type: imports\n",
      "  - stringdist (>= 0.9.4.1) type: imports\n",
      "  - stringr (>= 1.0.0) type: imports\n",
      "  - RColorBrewer (>= 1.1-2) type: imports\n",
      "  - DECIPHER, type: imports\n",
      "  - lpSolveAPI (>= 5.5.2.0-17) type: imports\n",
      "  - digest (>= 0.6.9) type: imports\n",
      "  - Hmisc (>= 3.17-4) type: imports\n",
      "  - ape (>= 3.5) type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - foreach (>= 1.4.3) type: imports\n",
      "  - magrittr (>= 1.5) type: imports\n",
      "  - distr (>= 2.6) type: imports\n",
      "  - distrEx (>= 2.6) type: imports\n",
      "  - fitdistrplus (>= 1.0-7) type: imports\n",
      "  - uniqtag (>= 1.0) type: imports\n",
      "  - openxlsx (>= 4.0.17) type: imports\n",
      "  - grid (>= 3.1.0) type: imports\n",
      "  - grDevices (>= 3.1.0) type: imports\n",
      "  - stats (>= 3.1.0) type: imports\n",
      "  - utils (>= 3.1.0) type: imports\n",
      "  - methods (>= 3.1.0) type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/openPrimeR.html\n",
      "\n",
      "Name: RbcBook1\n",
      "Description:\n",
      "tools for building book\n",
      "Version: 1.66.0\n",
      "Publication date: None\n",
      "Mantainer: Vince Carey <stvjc@channing.harvard.edu>\n",
      "Authors:\n",
      "Author: Vince Carey <stvjc at channing.harvard.edu> and Wolfgang Huber <huber at ebi.ac.uk>\n",
      "Dependencies:\n",
      "  - R (>= 2.10) type: depends\n",
      "  - Biobase, type: depends\n",
      "  - graph, type: depends\n",
      "  - rpart, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/RbcBook1.html\n",
      "\n",
      "Name: TissueEnrich\n",
      "Description:\n",
      "The TissueEnrich package is used to calculate enrichment of tissue-specific genes in a set of input genes. For example, the user can input the most highly expressed genes from RNA-Seq data, or gene co-expression modules to determine which tissue-specific genes are enriched in those datasets. Tissue-specific genes were defined by processing RNA-Seq data from the Human Protein Atlas (HPA) (UhlÃ©n et al. 2015), GTEx (Ardlie et al. 2015), and mouse ENCODE (Shen et al. 2012) using the algorithm from the HPA (UhlÃ©n et al. 2015).The hypergeometric test is being used to determine if the tissue-specific genes are enriched among the input genes. Along with tissue-specific gene enrichment, the TissueEnrich package can also be used to define tissue-specific genes from expression datasets provided by the user, which can then be used to calculate tissue-specific gene enrichments.\n",
      "Version: 1.18.0\n",
      "Publication date: None\n",
      "Mantainer: Ashish Jain <jain@iastate.edu>\n",
      "Authors:\n",
      "Author: Ashish Jain [aut, cre], Geetu Tuteja [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.5) type: depends\n",
      "  - ensurer (>= 1.1.0) type: depends\n",
      "  - ggplot2 (>= 2.2.1) type: depends\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - GSEABase, type: depends\n",
      "  - dplyr (>= 0.7.3) type: imports\n",
      "  - tidyr (>= 0.8.0) type: imports\n",
      "  - stats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/TissueEnrich.html\n",
      "\n",
      "Name: tricycle\n",
      "Description:\n",
      "The package contains functions to infer and visualize cell cycle process using Single Cell RNASeq data. It exploits the idea of transfer learning, projecting new data to the previous learned biologically interpretable space. We provide a pre-learned cell cycle space, which could be used to infer cell cycle time of human and mouse single cell samples. In addition, we also offer functions to visualize cell cycle time on different embeddings and functions to build new reference.\n",
      "Version: 1.6.0\n",
      "Publication date: None\n",
      "Mantainer: Shijie Zheng <shijieczheng@gmail.com>\n",
      "Authors:\n",
      "Author: Shijie Zheng [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - SingleCellExperiment, type: depends\n",
      "  - methods, type: imports\n",
      "  - circular, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - ggnewscale, type: imports\n",
      "  - AnnotationDbi, type: imports\n",
      "  - scater, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - scattermore, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - stats, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/tricycle.html\n",
      "\n",
      "Name: BgeeDB\n",
      "Description:\n",
      "A package for the annotation and gene expression data download from Bgee database, and TopAnat analysis: GO-like enrichment of anatomical terms, mapped to genes by expression patterns.\n",
      "Version: 2.24.0\n",
      "Publication date: None\n",
      "Mantainer: Julien Wollbrett <julien.wollbrett@unil.ch>, Julien Roux <julien.roux@unibas.ch>, Andrea Komljenovic <andreakomljenovic@gmail.com>, Frederic Bastian <bgee@sib.swiss>\n",
      "Authors:\n",
      "Author: Andrea Komljenovic [aut, cre], Julien Roux [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 3.6.0) type: depends\n",
      "  - topGO, type: depends\n",
      "  - tidyr, type: depends\n",
      "  - R.utils, type: imports\n",
      "  - data.table, type: imports\n",
      "  - curl, type: imports\n",
      "  - RCurl, type: imports\n",
      "  - digest, type: imports\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - RSQLite, type: imports\n",
      "  - graph, type: imports\n",
      "  - Biobase, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3 + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/BgeeDB.html\n",
      "\n",
      "Name: pathwayPCA\n",
      "Description:\n",
      "pathwayPCA is an integrative analysis tool that implements the principal component analysis (PCA) based pathway analysis approaches described in Chen et al. (2008), Chen et al. (2010), and Chen (2011). pathwayPCA allows users to: (1) Test pathway association with binary, continuous, or survival phenotypes. (2) Extract relevant genes in the pathways using the SuperPCA and AES-PCA approaches. (3) Compute principal components (PCs) based on the selected genes. These estimated latent variables represent pathway activities for individual subjects, which can then be used to perform integrative pathway analysis, such as multi-omics analysis. (4) Extract relevant genes that drive pathway significance as well as data corresponding to these relevant genes for additional in-depth analysis. (5) Perform analyses with enhanced computational efficiency with parallel computing and enhanced data safety with S4-class data objects. (6) Analyze studies with complex experimental designs, with multiple covariates, and with interaction effects, e.g., testing whether pathway association with clinical phenotype is different between male and female subjects. Citations: Chen et al. (2008) ; Chen et al. (2010) ; and Chen (2011) .\n",
      "Version: 1.14.0\n",
      "Publication date: None\n",
      "Mantainer: Gabriel Odom <gabriel.odom@med.miami.edu>\n",
      "Authors:\n",
      "Author: Gabriel Odom [aut, cre], James Ban [aut], Lizhong Liu [aut], Lily Wang [aut], Steven Chen [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.1) type: depends\n",
      "  - lars, type: imports\n",
      "  - methods, type: imports\n",
      "  - parallel, type: imports\n",
      "  - stats, type: imports\n",
      "  - survival, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/pathwayPCA.html\n",
      "\n",
      "Name: PICS\n",
      "Description:\n",
      "Probabilistic inference of ChIP-Seq using an empirical Bayes mixture model approach.\n",
      "Version: 2.42.0\n",
      "Publication date: None\n",
      "Mantainer: Renan Sauteraud <renan.sauteraud@gmail.com>\n",
      "Authors:\n",
      "Author: Xuekui Zhang <xzhang at stat.ubc.ca>, Raphael Gottardo <rgottard at fhcrc.org>\n",
      "Dependencies:\n",
      "  - R (>= 3.0.0) type: depends\n",
      "  - utils, type: imports\n",
      "  - stats, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - methods, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - GenomicAlignments, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/PICS.html\n",
      "\n",
      "Name: pmp\n",
      "Description:\n",
      "Methods and tools for (pre-)processing of metabolomics datasets (i.e. peak matrices), including filtering, normalisation, missing value imputation, scaling, and signal drift and batch effect correction methods. Filtering methods are based on: the fraction of missing values (across samples or features); Relative Standard Deviation (RSD) calculated from the Quality Control (QC) samples; the blank samples. Normalisation methods include Probabilistic Quotient Normalisation (PQN) and normalisation to total signal intensity. A unified user interface for several commonly used missing value imputation algorithms is also provided. Supported methods are: k-nearest neighbours (knn), random forests (rf), Bayesian PCA missing value estimator (bpca), mean or median value of the given feature and a constant small value. The generalised logarithm (glog) transformation algorithm is available to stabilise the variance across low and high intensity mass spectral features. Finally, this package provides an implementation of the Quality Control-Robust Spline Correction (QCRSC) algorithm for signal drift and batch effect correction of mass spectrometry-based datasets.\n",
      "Version: 1.10.0\n",
      "Publication date: None\n",
      "Mantainer: Gavin Rhys Lloyd <g.r.lloyd@bham.ac.uk>\n",
      "Authors:\n",
      "Author: Andris Jankevics [aut], Gavin Rhys Lloyd [aut, cre], Ralf Johannes Maria Weber [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - stats, type: imports\n",
      "  - impute, type: imports\n",
      "  - pcaMethods, type: imports\n",
      "  - missForest, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - methods, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/pmp.html\n",
      "\n",
      "Name: a4Base\n",
      "Description:\n",
      "Base utility functions are available for the Automated Affymetrix Array Analysis set of packages.\n",
      "Version: 1.46.0\n",
      "Publication date: None\n",
      "Mantainer: Laure Cougnaud <laure.cougnaud@openanalytics.eu>\n",
      "Authors:\n",
      "Author: Willem Talloen [aut], Tine Casneuf [aut], An De Bondt [aut], Steven Osselaer [aut], Hinrich Goehlmann [aut], Willem Ligtenberg [aut], Tobias Verbeke [aut], Laure Cougnaud [cre]\n",
      "Dependencies:\n",
      "  - a4Preproc, type: depends\n",
      "  - a4Core, type: depends\n",
      "  - methods, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grid, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - annaffy, type: imports\n",
      "  - mpm, type: imports\n",
      "  - genefilter, type: imports\n",
      "  - limma, type: imports\n",
      "  - multtest, type: imports\n",
      "  - glmnet, type: imports\n",
      "  - gplots, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/a4Base.html\n",
      "\n",
      "Name: biocGraph\n",
      "Description:\n",
      "This package provides examples and code that make use of the different graph related packages produced by Bioconductor.\n",
      "Version: 1.60.0\n",
      "Publication date: None\n",
      "Mantainer: Florian Hahne <florian.hahne@novartis.com>\n",
      "Authors:\n",
      "Author: Li Long <li.long at isb-sib.ch>, Robert Gentleman <rgentlem at fhcrc.org>, Seth Falcon <sethf at fhcrc.org> Florian Hahne <fhahne at fhcrc.org>\n",
      "Dependencies:\n",
      "  - Rgraphviz, type: depends\n",
      "  - graph, type: depends\n",
      "  - Rgraphviz, type: imports\n",
      "  - geneplotter, type: imports\n",
      "  - graph, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - methods, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/biocGraph.html\n",
      "\n",
      "Name: HelloRanges\n",
      "Description:\n",
      "Translates bedtools command-line invocations to R code calling functions from the Bioconductor *Ranges infrastructure. This is intended to educate novice Bioconductor users and to compare the syntax and semantics of the two frameworks.\n",
      "Version: 1.24.0\n",
      "Publication date: None\n",
      "Mantainer: Michael Lawrence <michafla@gene.com>\n",
      "Authors:\n",
      "Author: Michael Lawrence\n",
      "Dependencies:\n",
      "  - methods, type: depends\n",
      "  - BiocGenerics, type: depends\n",
      "  - S4Vectors, type: depends\n",
      "  - IRanges, type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - Biostrings, type: depends\n",
      "  - BSgenome, type: depends\n",
      "  - GenomicFeatures, type: depends\n",
      "  - VariantAnnotation, type: depends\n",
      "  - Rsamtools, type: depends\n",
      "  - GenomicAlignments, type: depends\n",
      "  - rtracklayer, type: depends\n",
      "  - GenomeInfoDb, type: depends\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - BiocIO, type: depends\n",
      "  - docopt, type: imports\n",
      "  - stats, type: imports\n",
      "  - tools, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/HelloRanges.html\n",
      "\n",
      "Name: igvR\n",
      "Description:\n",
      "Access to igv.js, the Integrative Genomics Viewer running in a web browser.\n",
      "Version: 1.18.0\n",
      "Publication date: None\n",
      "Mantainer: Paul Shannon <paul.thurmond.shannon@gmail.com>\n",
      "Authors:\n",
      "Author: Paul Shannon\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - GenomicAlignments, type: depends\n",
      "  - BrowserViz, type: depends\n",
      "  - methods, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - httpuv, type: imports\n",
      "  - utils, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - VariantAnnotation, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - httr, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/igvR.html\n",
      "\n",
      "Name: proBatch\n",
      "Description:\n",
      "These tools facilitate batch effects analysis and correction in high-throughput experiments. It was developed primarily for mass-spectrometry proteomics (DIA/SWATH), but could also be applicable to most omic data with minor adaptations. The package contains functions for diagnostics (proteome/genome-wide and feature-level), correction (normalization and batch effects correction) and quality control. Non-linear fitting based approaches were also included to deal with complex, mass spectrometry-specific signal drifts.\n",
      "Version: 1.13.0\n",
      "Publication date: None\n",
      "Mantainer: Chloe H. Lee <chloe.h.lee94@gmail.com>\n",
      "Authors:\n",
      "Author: Jelena Cuklina <chuklina.jelena at gmail.com>, Chloe H. Lee <chloe.h.lee94 at gmail.com>, Patrick Pedrioli <pedrioli at gmail.com>\n",
      "Dependencies:\n",
      "  - R (>= 3.6) type: depends\n",
      "  - Biobase, type: imports\n",
      "  - corrplot, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - data.table, type: imports\n",
      "  - ggfortify, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - lazyeval, type: imports\n",
      "  - lubridate, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - pheatmap, type: imports\n",
      "  - preprocessCore, type: imports\n",
      "  - purrr, type: imports\n",
      "  - pvca, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - rlang, type: imports\n",
      "  - scales, type: imports\n",
      "  - stats, type: imports\n",
      "  - sva, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - tibble, type: imports\n",
      "  - tools, type: imports\n",
      "  - utils, type: imports\n",
      "  - viridis, type: imports\n",
      "  - wesanderson, type: imports\n",
      "  - WGCNA, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/proBatch.html\n",
      "\n",
      "Name: PWMEnrich\n",
      "Description:\n",
      "A toolkit of high-level functions for DNA motif scanning and enrichment analysis built upon Biostrings. The main functionality is PWM enrichment analysis of already known PWMs (e.g. from databases such as MotifDb), but the package also implements high-level functions for PWM scanning and visualisation. The package does not perform \"de novo\" motif discovery, but is instead focused on using motifs that are either experimentally derived or computationally constructed by other tools.\n",
      "Version: 4.34.0\n",
      "Publication date: None\n",
      "Mantainer: Diego Diez <diego10ruiz@gmail.com>\n",
      "Authors:\n",
      "Author: Robert Stojnic, Diego Diez\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - BiocGenerics, type: depends\n",
      "  - Biostrings, type: depends\n",
      "  - grid, type: imports\n",
      "  - seqLogo, type: imports\n",
      "  - gdata, type: imports\n",
      "  - evd, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/PWMEnrich.html\n",
      "\n",
      "Name: REDseq\n",
      "Description:\n",
      "The package includes functions to build restriction enzyme cut site (RECS) map, distribute mapped sequences on the map with five different approaches, find enriched/depleted RECSs for a sample, and identify differentially enriched/depleted RECSs between samples.\n",
      "Version: 1.44.0\n",
      "Publication date: None\n",
      "Mantainer: Lihua Julie Zhu <julie.zhu@umassmed.edu>\n",
      "Authors:\n",
      "Author: Lihua Julie Zhu, Junhui Li and Thomas Fazzio\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - BiocGenerics, type: depends\n",
      "  - BSgenome.Celegans.UCSC.ce2, type: depends\n",
      "  - multtest, type: depends\n",
      "  - Biostrings, type: depends\n",
      "  - BSgenome, type: depends\n",
      "  - ChIPpeakAnno, type: depends\n",
      "  - AnnotationDbi, type: imports\n",
      "  - graphics, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>=2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/REDseq.html\n",
      "\n",
      "Name: cogena\n",
      "Description:\n",
      "cogena is a workflow for co-expressed gene-set enrichment analysis. It aims to discovery smaller scale, but highly correlated cellular events that may be of great biological relevance. A novel pipeline for drug discovery and drug repositioning based on the cogena workflow is proposed. Particularly, candidate drugs can be predicted based on the gene expression of disease-related data, or other similar drugs can be identified based on the gene expression of drug-related data. Moreover, the drug mode of action can be disclosed by the associated pathway analysis. In summary, cogena is a flexible workflow for various gene set enrichment analysis for co-expressed genes, with a focus on pathway/GO analysis and drug repositioning.\n",
      "Version: 1.32.0\n",
      "Publication date: None\n",
      "Mantainer: Zhilong Jia <zhilongjia@gmail.com>\n",
      "Authors:\n",
      "Author: Zhilong Jia [aut, cre], Michael Barnes [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.6) type: depends\n",
      "  - cluster, type: depends\n",
      "  - ggplot2, type: depends\n",
      "  - kohonen, type: depends\n",
      "  - methods, type: imports\n",
      "  - class, type: imports\n",
      "  - gplots, type: imports\n",
      "  - mclust, type: imports\n",
      "  - amap, type: imports\n",
      "  - apcluster, type: imports\n",
      "  - foreach, type: imports\n",
      "  - parallel, type: imports\n",
      "  - doParallel, type: imports\n",
      "  - fastcluster, type: imports\n",
      "  - corrplot, type: imports\n",
      "  - biwt, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - stringr, type: imports\n",
      "  - tibble, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - devtools, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/cogena.html\n",
      "\n",
      "Name: CrispRVariants\n",
      "Description:\n",
      "CrispRVariants provides tools for analysing the results of a CRISPR-Cas9 mutagenesis sequencing experiment, or other sequencing experiments where variants within a given region are of interest. These tools allow users to localize variant allele combinations with respect to any genomic location (e.g. the Cas9 cut site), plot allele combinations and calculate mutation rates with flexible filtering of unrelated variants.\n",
      "Version: 1.26.0\n",
      "Publication date: None\n",
      "Mantainer: Helen Lindsay <helen.lindsay@chuv.ch>\n",
      "Authors:\n",
      "Author: Helen Lindsay [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 3.5) type: depends\n",
      "  - ggplot2 (>= 2.2.0) type: depends\n",
      "  - AnnotationDbi, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - methods, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - GenomicAlignments, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - grid, type: imports\n",
      "  - gridExtra, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/CrispRVariants.html\n",
      "\n",
      "Name: isobar\n",
      "Description:\n",
      "isobar provides methods for preprocessing, normalization, and report generation for the analysis of quantitative mass spectrometry proteomics data labeled with isobaric tags, such as iTRAQ and TMT. Features modules for integrating and validating PTM-centric datasets (isobar-PTM). More information on http://www.ms-isobar.org.\n",
      "Version: 1.44.0\n",
      "Publication date: None\n",
      "Mantainer: Florian P Breitwieser <florian.bw@gmail.com>\n",
      "Authors:\n",
      "Author: Florian P Breitwieser <florian.bw at gmail.com> and Jacques Colinge <jacques.colinge at inserm.fr>, with contributions from Alexey Stukalov <stukalov at biochem.mpg.de>, Xavier Robin <xavier.robin at unige.ch> and Florent Gluck <florent.gluck at unige.ch>\n",
      "Dependencies:\n",
      "  - R (>= 2.10.0) type: depends\n",
      "  - Biobase, type: depends\n",
      "  - stats, type: depends\n",
      "  - methods, type: depends\n",
      "  - distr, type: imports\n",
      "  - plyr, type: imports\n",
      "  - biomaRt, type: imports\n",
      "  - ggplot2, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/isobar.html\n",
      "\n",
      "Name: phenopath\n",
      "Description:\n",
      "PhenoPath infers genomic trajectories (pseudotimes) in the presence of heterogeneous genetic and environmental backgrounds and tests for interactions between them.\n",
      "Version: 1.22.0\n",
      "Publication date: None\n",
      "Mantainer: Kieran Campbell <kieranrcampbell@gmail.com>\n",
      "Authors:\n",
      "Author: Kieran Campbell\n",
      "Dependencies:\n",
      "  - Rcpp (>= 0.12.8) type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - tibble, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - tidyr, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Apache License (== 2.0)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/phenopath.html\n",
      "\n",
      "Name: PROcess\n",
      "Description:\n",
      "A package for processing protein mass spectrometry data.\n",
      "Version: 1.74.0\n",
      "Publication date: None\n",
      "Mantainer: Xiaochun Li <xiaochun@jimmy.harvard.edu>\n",
      "Authors:\n",
      "Author: Xiaochun Li\n",
      "Dependencies:\n",
      "  - Icens, type: depends\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - Icens, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/PROcess.html\n",
      "\n",
      "Name: RDRToolbox\n",
      "Description:\n",
      "A package for nonlinear dimension reduction using the Isomap and LLE algorithm. It also includes a routine for computing the Davis-Bouldin-Index for cluster validation, a plotting tool and a data generator for microarray gene expression data and for the Swiss Roll dataset.\n",
      "Version: 1.48.0\n",
      "Publication date: None\n",
      "Mantainer: Christoph Bartenhagen <c.bartenhagen@uni-koeln.de>\n",
      "Authors:\n",
      "Author: Christoph Bartenhagen\n",
      "Dependencies:\n",
      "  - R (>= 2.9.0) type: depends\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - MASS, type: imports\n",
      "  - rgl, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/RDRToolbox.html\n",
      "\n",
      "Name: segmentSeq\n",
      "Description:\n",
      "High-throughput sequencing technologies allow the production of large volumes of short sequences, which can be aligned to the genome to create a set of matches to the genome. By looking for regions of the genome which to which there are high densities of matches, we can infer a segmentation of the genome into regions of biological significance. The methods in this package allow the simultaneous segmentation of data from multiple samples, taking into account replicate data, in order to create a consensus segmentation. This has obvious applications in a number of classes of sequencing experiments, particularly in the discovery of small RNA loci and novel mRNA transcriptome discovery.\n",
      "Version: 2.32.0\n",
      "Publication date: None\n",
      "Mantainer: Thomas J. Hardcastle <tjh48@cam.ac.uk>\n",
      "Authors:\n",
      "Author: Thomas J. Hardcastle\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - baySeq, type: depends\n",
      "  - S4Vectors, type: depends\n",
      "  - parallel, type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - ShortRead, type: depends\n",
      "  - stats, type: depends\n",
      "  - Rsamtools, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - utils, type: imports\n",
      "  - abind, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/segmentSeq.html\n",
      "\n",
      "Name: CellNOptR\n",
      "Description:\n",
      "This package does optimisation of boolean logic networks of signalling pathways based on a previous knowledge network and a set of data upon perturbation of the nodes in the network.\n",
      "Version: 1.44.0\n",
      "Publication date: None\n",
      "Mantainer: Attila Gabor <attila.gabor@uni-heidelberg.de>\n",
      "Authors:\n",
      "Author: Thomas Cokelaer [aut], Federica Eduati [aut], Aidan MacNamara [aut], S Schrier [ctb], Camille Terfve [aut], Enio Gjerga [ctb], Attila Gabor [cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.0.0) type: depends\n",
      "  - RBGL, type: depends\n",
      "  - graph, type: depends\n",
      "  - methods, type: depends\n",
      "  - RCurl, type: depends\n",
      "  - Rgraphviz, type: depends\n",
      "  - XML, type: depends\n",
      "  - ggplot2, type: depends\n",
      "  - rmarkdown, type: depends\n",
      "  - igraph, type: imports\n",
      "  - stringi, type: imports\n",
      "  - stringr, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/CellNOptR.html\n",
      "\n",
      "Name: cydar\n",
      "Description:\n",
      "Identifies differentially abundant populations between samples and groups in mass cytometry data. Provides methods for counting cells into hyperspheres, controlling the spatial false discovery rate, and visualizing changes in abundance in the high-dimensional marker space.\n",
      "Version: 1.22.0\n",
      "Publication date: None\n",
      "Mantainer: Aaron Lun <infinite.monkeys.with.keyboards@gmail.com>\n",
      "Authors:\n",
      "Author: Aaron Lun [aut, cre]\n",
      "Dependencies:\n",
      "  - SingleCellExperiment, type: depends\n",
      "  - viridis, type: imports\n",
      "  - methods, type: imports\n",
      "  - shiny, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - utils, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - flowCore, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - Rcpp, type: imports\n",
      "  - BiocNeighbors, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/cydar.html\n",
      "\n",
      "Name: Mulcom\n",
      "Description:\n",
      "Identification of differentially expressed genes and false discovery rate (FDR) calculation by Multiple Comparison test.\n",
      "Version: 1.48.0\n",
      "Publication date: None\n",
      "Mantainer: Claudio Isella <claudio.isella@ircc.it>\n",
      "Authors:\n",
      "Author: Claudio Isella\n",
      "Dependencies:\n",
      "  - R (>= 2.10) type: depends\n",
      "  - Biobase, type: depends\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - stats, type: imports\n",
      "  - methods, type: imports\n",
      "  - fields, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/Mulcom.html\n",
      "\n",
      "Name: RandomWalkRestartMH\n",
      "Description:\n",
      "This package performs Random Walk with Restart on multiplex and heterogeneous networks. It is described in the following article: \"Random Walk With Restart On Multiplex And Heterogeneous Biological Networks\" .\n",
      "Version: 1.18.0\n",
      "Publication date: None\n",
      "Mantainer: Alberto Valdeolivas <alvaldeolivas@gmail.com>\n",
      "Authors:\n",
      "Author: Alberto Valdeolivas [cre, aut, ctb]\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - igraph, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - dnet, type: imports\n",
      "  - methods, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/RandomWalkRestartMH.html\n",
      "\n",
      "Name: ROntoTools\n",
      "Description:\n",
      "Suite of tools for functional analysis.\n",
      "Version: 2.26.0\n",
      "Publication date: None\n",
      "Mantainer: Calin Voichita <calin@wayne.edu>\n",
      "Authors:\n",
      "Author: Calin Voichita <calin at wayne.edu> and Sahar Ansari <saharansari at wayne.edu> and Sorin Draghici <sorin at wayne.edu>\n",
      "Dependencies:\n",
      "  - methods, type: depends\n",
      "  - graph, type: depends\n",
      "  - boot, type: depends\n",
      "  - KEGGREST, type: depends\n",
      "  - KEGGgraph, type: depends\n",
      "  - Rgraphviz, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "CC BY-NC-ND 4.0 + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ROntoTools.html\n",
      "\n",
      "Name: alpine\n",
      "Description:\n",
      "Fragment sequence bias modeling and correction for RNA-seq transcript abundance estimation.\n",
      "Version: 1.24.0\n",
      "Publication date: None\n",
      "Mantainer: Michael Love <michaelisaiahlove@gmail.com>\n",
      "Authors:\n",
      "Author: Michael Love, Rafael Irizarry\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - Biostrings, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - GenomicAlignments, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - GenomicFeatures, type: imports\n",
      "  - speedglm, type: imports\n",
      "  - splines, type: imports\n",
      "  - graph, type: imports\n",
      "  - RBGL, type: imports\n",
      "  - stringr, type: imports\n",
      "  - stats, type: imports\n",
      "  - methods, type: imports\n",
      "  - graphics, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>=2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/alpine.html\n",
      "\n",
      "Name: BioNERO\n",
      "Description:\n",
      "BioNERO aims to integrate all aspects of biological network inference in a single package, including data preprocessing, exploratory analyses, network inference, and analyses for biological interpretations. BioNERO can be used to infer gene coexpression networks (GCNs) and gene regulatory networks (GRNs) from gene expression data. Additionally, it can be used to explore topological properties of protein-protein interaction (PPI) networks. GCN inference relies on the popular WGCNA algorithm. GRN inference is based on the \"wisdom of the crowds\" principle, which consists in inferring GRNs with multiple algorithms (here, CLR, GENIE3 and ARACNE) and calculating the average rank for each interaction pair. As all steps of network analyses are included in this package, BioNERO makes users avoid having to learn the syntaxes of several packages and how to communicate between them. Finally, users can also identify consensus modules across independent expression sets and calculate intra and interspecies module preservation statistics between different networks.\n",
      "Version: 1.6.0\n",
      "Publication date: None\n",
      "Mantainer: Fabricio Almeida-Silva <fabricio_almeidasilva@hotmail.com>\n",
      "Authors:\n",
      "Author: Fabricio Almeida-Silva [cre, aut] , Thiago Venancio [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - WGCNA, type: imports\n",
      "  - dynamicTreeCut, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - sva, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - ComplexHeatmap, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - ggrepel, type: imports\n",
      "  - patchwork, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - igraph, type: imports\n",
      "  - ggnetwork, type: imports\n",
      "  - intergraph, type: imports\n",
      "  - networkD3, type: imports\n",
      "  - ggnewscale, type: imports\n",
      "  - NetRep, type: imports\n",
      "  - stats, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - utils, type: imports\n",
      "  - methods, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - minet, type: imports\n",
      "  - GENIE3, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/BioNERO.html\n",
      "\n",
      "Name: esATAC\n",
      "Description:\n",
      "This package provides a framework and complete preset pipeline for quantification and analysis of ATAC-seq Reads. It covers raw sequencing reads preprocessing (FASTQ files), reads alignment (Rbowtie2), aligned reads file operations (SAM, BAM, and BED files), peak calling (F-seq), genome annotations (Motif, GO, SNP analysis) and quality control report. The package is managed by dataflow graph. It is easy for user to pass variables seamlessly between processes and understand the workflow. Users can process FASTQ files through end-to-end preset pipeline which produces a pretty HTML report for quality control and preliminary statistical results, or customize workflow starting from any intermediate stages with esATAC functions easily and flexibly.\n",
      "Version: 1.20.0\n",
      "Publication date: None\n",
      "Mantainer: Zheng Wei <wzweizheng@qq.com>\n",
      "Authors:\n",
      "Author: Zheng Wei, Wei Zhang\n",
      "Dependencies:\n",
      "  - R (>= 4.0.0) type: depends\n",
      "  - Rsamtools, type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - ShortRead, type: depends\n",
      "  - pipeFrame, type: depends\n",
      "  - Rcpp (>= 0.12.11) type: imports\n",
      "  - methods, type: imports\n",
      "  - knitr, type: imports\n",
      "  - Rbowtie2, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - ChIPseeker, type: imports\n",
      "  - clusterProfiler, type: imports\n",
      "  - igraph, type: imports\n",
      "  - rJava, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - digest, type: imports\n",
      "  - BSgenome, type: imports\n",
      "  - AnnotationDbi, type: imports\n",
      "  - GenomicAlignments, type: imports\n",
      "  - GenomicFeatures, type: imports\n",
      "  - R.utils, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - rmarkdown, type: imports\n",
      "  - tools, type: imports\n",
      "  - VennDiagram, type: imports\n",
      "  - grid, type: imports\n",
      "  - JASPAR2018, type: imports\n",
      "  - TFBSTools, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - parallel, type: imports\n",
      "  - corrplot, type: imports\n",
      "  - BiocManager, type: imports\n",
      "  - motifmatchr, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3 | file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/esATAC.html\n",
      "\n",
      "Name: FastqCleaner\n",
      "Description:\n",
      "An interactive web application for quality control, filtering and trimming of FASTQ files. This user-friendly tool combines a pipeline for data processing based on Biostrings and ShortRead infrastructure, with a cutting-edge visual environment. Single-Read and Paired-End files can be locally processed. Diagnostic interactive plots (CG content, per-base sequence quality, etc.) are provided for both the input and output files.\n",
      "Version: 1.16.1\n",
      "Publication date: None\n",
      "Mantainer: Leandro Roser <learoser@gmail.com>\n",
      "Authors:\n",
      "Author: Leandro Roser [aut, cre], FernÃ¡n AgÃ¼ero [aut], Daniel SÃ¡nchez [aut]\n",
      "Dependencies:\n",
      "  - methods, type: imports\n",
      "  - shiny, type: imports\n",
      "  - stats, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - ShortRead, type: imports\n",
      "  - DT, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - graphics, type: imports\n",
      "  - htmltools, type: imports\n",
      "  - shinyBS, type: imports\n",
      "  - Rcpp (>= 0.12.12) type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/FastqCleaner.html\n",
      "\n",
      "Name: iBBiG\n",
      "Description:\n",
      "iBBiG is a bi-clustering algorithm which is optimizes for binary data analysis. We apply it to meta-gene set analysis of large numbers of gene expression datasets. The iterative algorithm extracts groups of phenotypes from multiple studies that are associated with similar gene sets. iBBiG does not require prior knowledge of the number or scale of clusters and allows discovery of clusters with diverse sizes\n",
      "Version: 1.42.0\n",
      "Publication date: None\n",
      "Mantainer: Aedin Culhane <aedin@jimmy.harvard.edu>\n",
      "Authors:\n",
      "Author: Daniel Gusenleitner, Aedin Culhane\n",
      "Dependencies:\n",
      "  - biclust, type: depends\n",
      "  - stats4, type: imports\n",
      "  - xtable, type: imports\n",
      "  - ade4, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/iBBiG.html\n",
      "\n",
      "Name: ideal\n",
      "Description:\n",
      "This package provides functions for an Interactive Differential Expression AnaLysis of RNA-sequencing datasets, to extract quickly and effectively information downstream the step of differential expression. A Shiny application encapsulates the whole package.\n",
      "Version: 1.22.0\n",
      "Publication date: None\n",
      "Mantainer: Federico Marini <marinif@uni-mainz.de>\n",
      "Authors:\n",
      "Author: Federico Marini [aut, cre]\n",
      "Dependencies:\n",
      "  - topGO, type: depends\n",
      "  - DESeq2, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - ggplot2 (>= 2.0.0) type: imports\n",
      "  - heatmaply, type: imports\n",
      "  - plotly, type: imports\n",
      "  - pheatmap, type: imports\n",
      "  - pcaExplorer, type: imports\n",
      "  - IHW, type: imports\n",
      "  - gplots, type: imports\n",
      "  - UpSetR, type: imports\n",
      "  - goseq, type: imports\n",
      "  - stringr, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - limma, type: imports\n",
      "  - GOstats, type: imports\n",
      "  - GO.db, type: imports\n",
      "  - AnnotationDbi, type: imports\n",
      "  - shiny (>= 0.12.0) type: imports\n",
      "  - shinydashboard, type: imports\n",
      "  - shinyBS, type: imports\n",
      "  - DT, type: imports\n",
      "  - rentrez, type: imports\n",
      "  - rintrojs, type: imports\n",
      "  - rlang, type: imports\n",
      "  - ggrepel, type: imports\n",
      "  - knitr, type: imports\n",
      "  - rmarkdown, type: imports\n",
      "  - shinyAce, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - base64enc, type: imports\n",
      "  - methods, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ideal.html\n",
      "\n",
      "Name: SplicingGraphs\n",
      "Description:\n",
      "This package allows the user to create, manipulate, and visualize splicing graphs and their bubbles based on a gene model for a given organism. Additionally it allows the user to assign RNA-seq reads to the edges of a set of splicing graphs, and to summarize them in different ways.\n",
      "Version: 1.38.0\n",
      "Publication date: None\n",
      "Mantainer: H. PagÃ¨s <hpages.on.github@gmail.com>\n",
      "Authors:\n",
      "Author: D. Bindreither, M. Carlson, M. Morgan, H. PagÃ¨s\n",
      "Dependencies:\n",
      "  - GenomicFeatures, type: depends\n",
      "  - GenomicAlignments, type: depends\n",
      "  - Rgraphviz, type: depends\n",
      "  - methods, type: imports\n",
      "  - utils, type: imports\n",
      "  - graphics, type: imports\n",
      "  - igraph, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - GenomicFeatures, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - GenomicAlignments, type: imports\n",
      "  - graph, type: imports\n",
      "  - Rgraphviz, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/SplicingGraphs.html\n",
      "\n",
      "Name: switchBox\n",
      "Description:\n",
      "The package offer different classifiers based on comparisons of pair of features (TSP), using various decision rules (e.g., majority wins principle).\n",
      "Version: 1.34.0\n",
      "Publication date: None\n",
      "Mantainer: Bahman Afsari <bahman@jhu.edu>, Luigi Marchionni <marchion@jhu.edu>, Wikum Dinalankara <wdinala1@jhmi.edu>\n",
      "Authors:\n",
      "Author: Bahman Afsari <bahman at jhu.edu>, Luigi Marchionni <marchion at jhu.edu>, Wikum Dinalankara <wdinala1 at jhmi.edu>\n",
      "Dependencies:\n",
      "  - R (>= 2.13.1) type: depends\n",
      "  - pROC, type: depends\n",
      "  - gplots, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/switchBox.html\n",
      "\n",
      "Name: tRNA\n",
      "Description:\n",
      "The tRNA package allows tRNA sequences and structures to be accessed and used for subsetting. In addition, it provides visualization tools to compare feature parameters of multiple tRNA sets and correlate them to additional data. The tRNA package uses GRanges objects as inputs requiring only few additional column data sets.\n",
      "Version: 1.16.1\n",
      "Publication date: None\n",
      "Mantainer: Felix GM Ernst <felix.gm.ernst@outlook.com>\n",
      "Authors:\n",
      "Author: Felix GM Ernst [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 3.5) type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - Structstrings, type: depends\n",
      "  - stringr, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - methods, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - XVector, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - Modstrings, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - scales, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3 + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/tRNA.html\n",
      "\n",
      "Name: ABSSeq\n",
      "Description:\n",
      "Inferring differential expression genes by absolute counts difference between two groups, utilizing Negative binomial distribution and moderating fold-change according to heterogeneity of dispersion across expression level.\n",
      "Version: 1.52.0\n",
      "Publication date: None\n",
      "Mantainer: Wentao Yang <wyang@zoologie.uni-kiel.de>\n",
      "Authors:\n",
      "Author: Wentao Yang\n",
      "Dependencies:\n",
      "  - R (>= 2.10) type: depends\n",
      "  - methods, type: depends\n",
      "  - locfit, type: imports\n",
      "  - limma, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 3)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ABSSeq.html\n",
      "\n",
      "Name: derfinderPlot\n",
      "Description:\n",
      "This package provides plotting functions for results from the derfinder package. This helps separate the graphical dependencies required for making these plots from the core functionality of derfinder.\n",
      "Version: 1.32.0\n",
      "Publication date: None\n",
      "Mantainer: Leonardo Collado-Torres <lcolladotor@gmail.com>\n",
      "Authors:\n",
      "Author: Leonardo Collado-Torres [aut, cre] , Andrew E. Jaffe [aut] , Jeffrey T. Leek [aut, ths]\n",
      "Dependencies:\n",
      "  - R (>= 3.2) type: depends\n",
      "  - derfinder, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - GenomicFeatures, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - ggbio, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - limma, type: imports\n",
      "  - methods, type: imports\n",
      "  - plyr, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - scales, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/derfinderPlot.html\n",
      "\n",
      "Name: HPAanalyze\n",
      "Description:\n",
      "Provide functions for retrieving, exploratory analyzing and visualizing the Human Protein Atlas data.\n",
      "Version: 1.16.0\n",
      "Publication date: None\n",
      "Mantainer: Anh Nhat Tran <trannhatanh89@gmail.com>\n",
      "Authors:\n",
      "Author: Anh Nhat Tran [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - dplyr, type: imports\n",
      "  - openxlsx, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - tibble, type: imports\n",
      "  - xml2, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - gridExtra, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3 + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/HPAanalyze.html\n",
      "\n",
      "Name: MMUPHin\n",
      "Description:\n",
      "MMUPHin is an R package for meta-analysis tasks of microbiome cohorts. It has function interfaces for: a) covariate-controlled batch- and cohort effect adjustment, b) meta-analysis differential abundance testing, c) meta-analysis unsupervised discrete structure (clustering) discovery, and d) meta-analysis unsupervised continuous structure discovery.\n",
      "Version: 1.12.1\n",
      "Publication date: None\n",
      "Mantainer: Siyuan MA <syma.research@gmail.com>\n",
      "Authors:\n",
      "Author: Siyuan Ma\n",
      "Dependencies:\n",
      "  - R (>= 3.6) type: depends\n",
      "  - Maaslin2, type: imports\n",
      "  - metafor, type: imports\n",
      "  - fpc, type: imports\n",
      "  - igraph, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - stringr, type: imports\n",
      "  - cowplot, type: imports\n",
      "  - utils, type: imports\n",
      "  - stats, type: imports\n",
      "  - grDevices, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/MMUPHin.html\n",
      "\n",
      "Name: Structstrings\n",
      "Description:\n",
      "The Structstrings package implements the widely used dot bracket annotation for storing base pairing information in structured RNA. Structstrings uses the infrastructure provided by the Biostrings package and derives the DotBracketString and related classes from the BString class. From these, base pair tables can be produced for in depth analysis. In addition, the loop indices of the base pairs can be retrieved as well. For better efficiency, information conversion is implemented in C, inspired to a large extend by the ViennaRNA package.\n",
      "Version: 1.14.0\n",
      "Publication date: None\n",
      "Mantainer: Felix G.M. Ernst <felix.gm.ernst@outlook.com>\n",
      "Authors:\n",
      "Author: Felix G.M. Ernst [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - S4Vectors, type: depends\n",
      "  - IRanges, type: depends\n",
      "  - Biostrings, type: depends\n",
      "  - methods, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - XVector, type: imports\n",
      "  - stringr, type: imports\n",
      "  - stringi, type: imports\n",
      "  - crayon, type: imports\n",
      "  - grDevices, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/Structstrings.html\n",
      "\n",
      "Name: ccfindR\n",
      "Description:\n",
      "A collection of tools for cancer genomic data clustering analyses, including those for single cell RNA-seq. Cell clustering and feature gene selection analysis employ Bayesian (and maximum likelihood) non-negative matrix factorization (NMF) algorithm. Input data set consists of RNA count matrix, gene, and cell bar code annotations. Analysis outputs are factor matrices for multiple ranks and marginal likelihood values for each rank. The package includes utilities for downstream analyses, including meta-gene identification, visualization, and construction of rank-based trees for clusters.\n",
      "Version: 1.18.0\n",
      "Publication date: None\n",
      "Mantainer: Jun Woo <jwoo@umn.edu>\n",
      "Authors:\n",
      "Author: Jun Woo [aut, cre], Jinhua Wang [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.6.0) type: depends\n",
      "  - stats, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - utils, type: imports\n",
      "  - methods, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - SingleCellExperiment, type: imports\n",
      "  - Rtsne, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - gtools, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - ape, type: imports\n",
      "  - Rmpi, type: imports\n",
      "  - irlba, type: imports\n",
      "  - Rcpp, type: imports\n",
      "  - Rdpack (>= 0.7) type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ccfindR.html\n",
      "\n",
      "Name: CiteFuse\n",
      "Description:\n",
      "CiteFuse pacakage implements a suite of methods and tools for CITE-seq data from pre-processing to integrative analytics, including doublet detection, network-based modality integration, cell type clustering, differential RNA and protein expression analysis, ADT evaluation, ligand-receptor interaction analysis, and interactive web-based visualisation of the analyses.\n",
      "Version: 1.10.0\n",
      "Publication date: None\n",
      "Mantainer: Yingxin Lin <yingxin.lin@sydney.edu.au>\n",
      "Authors:\n",
      "Author: Yingxin Lin [aut, cre], Hani Kim [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - SingleCellExperiment, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - mixtools, type: imports\n",
      "  - cowplot, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - gridExtra, type: imports\n",
      "  - grid, type: imports\n",
      "  - dbscan, type: imports\n",
      "  - propr, type: imports\n",
      "  - uwot, type: imports\n",
      "  - Rtsne, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - igraph, type: imports\n",
      "  - scales, type: imports\n",
      "  - scran, type: imports\n",
      "  - graphics, type: imports\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - ggridges, type: imports\n",
      "  - randomForest, type: imports\n",
      "  - pheatmap, type: imports\n",
      "  - ggraph, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - rhdf5, type: imports\n",
      "  - rlang, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/CiteFuse.html\n",
      "\n",
      "Name: flowVS\n",
      "Description:\n",
      "Per-channel variance stabilization from a collection of flow cytometry samples by Bertlett test for homogeneity of variances. The approach is applicable to microarrays data as well.\n",
      "Version: 1.30.0\n",
      "Publication date: None\n",
      "Mantainer: Ariful Azad <azad@iu.edu>\n",
      "Authors:\n",
      "Author: Ariful Azad\n",
      "Dependencies:\n",
      "  - R (>= 3.2) type: depends\n",
      "  - methods, type: depends\n",
      "  - flowCore, type: depends\n",
      "  - flowViz, type: depends\n",
      "  - flowStats, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/flowVS.html\n",
      "\n",
      "Name: msqrob2\n",
      "Description:\n",
      "msqrob2 provides a robust linear mixed model framework for assessing differential abundance in MS-based Quantitative proteomics experiments. Our workflows can start from raw peptide intensities or summarised protein expression values. The model parameter estimates can be stabilized by ridge regression, empirical Bayes variance estimation and robust M-estimation. msqrob2's hurde workflow can handle missing data without having to rely on hard-to-verify imputation assumptions, and, outcompetes state-of-the-art methods with and without imputation for both high and low missingness. It builds on QFeature infrastructure for quantitative mass spectrometry data to store the model results together with the raw data and preprocessed data.\n",
      "Version: 1.6.1\n",
      "Publication date: None\n",
      "Mantainer: Lieven Clement <lieven.clement@ugent.be>\n",
      "Authors:\n",
      "Author: Lieven Clement [aut, cre] , Laurent Gatto [aut] , Oliver M. Crook [aut] , Adriaan Sticker [ctb], Ludger Goeminne [ctb], Milan Malfait [ctb] , Stijn Vandenbulcke [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - QFeatures, type: depends\n",
      "  - stats, type: imports\n",
      "  - methods, type: imports\n",
      "  - lme4, type: imports\n",
      "  - purrr, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - MASS, type: imports\n",
      "  - limma, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - MultiAssayExperiment, type: imports\n",
      "  - codetools, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/msqrob2.html\n",
      "\n",
      "Name: sechm\n",
      "Description:\n",
      "sechm provides a simple interface between SummarizedExperiment objects and the ComplexHeatmap package. It enables plotting annotated heatmaps from SE objects, with easy access to rowData and colData columns, and implements a number of features to make the generation of heatmaps easier and more flexible. These functionalities used to be part of the SEtools package.\n",
      "Version: 1.6.0\n",
      "Publication date: None\n",
      "Mantainer: Pierre-Luc Germain <pierre-luc.germain@hest.ethz.ch>\n",
      "Authors:\n",
      "Author: Pierre-Luc Germain [cre, aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - S4Vectors, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - seriation, type: imports\n",
      "  - ComplexHeatmap, type: imports\n",
      "  - circlize, type: imports\n",
      "  - methods, type: imports\n",
      "  - randomcoloR, type: imports\n",
      "  - stats, type: imports\n",
      "  - grid, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - matrixStats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/sechm.html\n",
      "\n",
      "Name: SpeCond\n",
      "Description:\n",
      "This package performs a gene expression data analysis to detect condition-specific genes. Such genes are significantly up- or down-regulated in a small number of conditions. It does so by fitting a mixture of normal distributions to the expression values. Conditions can be environmental conditions, different tissues, organs or any other sources that you wish to compare in terms of gene expression.\n",
      "Version: 1.52.0\n",
      "Publication date: None\n",
      "Mantainer: Florence Cavalli <florence@ebi.ac.uk>\n",
      "Authors:\n",
      "Author: Florence Cavalli\n",
      "Dependencies:\n",
      "  - R (>= 2.10.0) type: depends\n",
      "  - mclust (>= 3.3.1) type: depends\n",
      "  - Biobase, type: depends\n",
      "  - fields, type: depends\n",
      "  - hwriter (>= 1.1) type: depends\n",
      "  - RColorBrewer, type: depends\n",
      "  - methods, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL (>=2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/SpeCond.html\n",
      "\n",
      "Name: SPOTlight\n",
      "Description:\n",
      "`SPOTlight`provides a method to deconvolute spatial transcriptomics spots using a seeded NMF approach along with visualization tools to assess the results. Spatially resolved gene expression profiles are key to understand tissue organization and function. However, novel spatial transcriptomics (ST) profiling techniques lack single-cell resolution and require a combination with single-cell RNA sequencing (scRNA-seq) information to deconvolute the spatially indexed datasets. Leveraging the strengths of both data types, we developed SPOTlight, a computational tool that enables the integration of ST with scRNA-seq data to infer the location of cell types and states within a complex tissue. SPOTlight is centered around a seeded non-negative matrix factorization (NMF) regression, initialized using cell-type marker genes and non-negative least squares (NNLS) to subsequently deconvolute ST capture locations (spots).\n",
      "Version: 1.2.0\n",
      "Publication date: None\n",
      "Mantainer: Marc Elosua-Bayes <elosua.marc@gmail.com>\n",
      "Authors:\n",
      "Author: Marc Elosua-Bayes [aut, cre], Helena L. Crowell [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - ggplot2, type: imports\n",
      "  - NMF, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - nnls, type: imports\n",
      "  - SingleCellExperiment, type: imports\n",
      "  - stats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/SPOTlight.html\n",
      "\n",
      "Name: chromPlot\n",
      "Description:\n",
      "Package designed to visualize genomic data along the chromosomes, where the vertical chromosomes are sorted by number, with sex chromosomes at the end.\n",
      "Version: 1.26.0\n",
      "Publication date: None\n",
      "Mantainer: Karen Y. Orostica <korostica09@gmail.com>\n",
      "Authors:\n",
      "Author: Ricardo A. Verdugo and Karen Y. Orostica\n",
      "Dependencies:\n",
      "  - stats, type: depends\n",
      "  - utils, type: depends\n",
      "  - graphics, type: depends\n",
      "  - grDevices, type: depends\n",
      "  - datasets, type: depends\n",
      "  - base, type: depends\n",
      "  - biomaRt, type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - R (>= 3.1.0) type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/chromPlot.html\n",
      "\n",
      "Name: GDSArray\n",
      "Description:\n",
      "GDS files are widely used to represent genotyping or sequence data. The GDSArray package implements the `GDSArray` class to represent nodes in GDS files in a matrix-like representation that allows easy manipulation (e.g., subsetting, mathematical transformation) in _R_. The data remains on disk until needed, so that very large files can be processed.\n",
      "Version: 1.18.0\n",
      "Publication date: None\n",
      "Mantainer: Qian Liu <qliu7@buffalo.edu>\n",
      "Authors:\n",
      "Author: Qian Liu [aut, cre], Martin Morgan [aut], HervÃ© PagÃ¨s [aut], Xiuwen Zheng [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.5) type: depends\n",
      "  - gdsfmt, type: depends\n",
      "  - methods, type: depends\n",
      "  - BiocGenerics, type: depends\n",
      "  - DelayedArray, type: depends\n",
      "  - tools, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - SNPRelate, type: imports\n",
      "  - SeqArray, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/GDSArray.html\n",
      "\n",
      "Name: hypergraph\n",
      "Description:\n",
      "A package that implements some simple capabilities for representing and manipulating hypergraphs.\n",
      "Version: 1.70.0\n",
      "Publication date: None\n",
      "Mantainer: Bioconductor Package Maintainer <maintainer@bioconductor.org>\n",
      "Authors:\n",
      "Author: Seth Falcon, Robert Gentleman\n",
      "Dependencies:\n",
      "  - R (>= 2.1.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - utils, type: depends\n",
      "  - graph, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/hypergraph.html\n",
      "\n",
      "Name: meshr\n",
      "Description:\n",
      "A set of annotation maps describing the entire MeSH assembled using data from MeSH.\n",
      "Version: 2.4.0\n",
      "Publication date: None\n",
      "Mantainer: Koki Tsuyuzaki <k.t.the-answer@hotmail.co.jp>\n",
      "Authors:\n",
      "Author: Koki Tsuyuzaki, Itoshi Nikaido, Gota Morota\n",
      "Dependencies:\n",
      "  - R (>= 4.1.0) type: depends\n",
      "  - markdown, type: imports\n",
      "  - rmarkdown, type: imports\n",
      "  - BiocStyle, type: imports\n",
      "  - knitr, type: imports\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - fdrtool, type: imports\n",
      "  - MeSHDbi, type: imports\n",
      "  - Category, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - RSQLite, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/meshr.html\n",
      "\n",
      "Name: OmaDB\n",
      "Description:\n",
      "A package for the orthology prediction data download from OMA database.\n",
      "Version: 2.14.0\n",
      "Publication date: None\n",
      "Mantainer: Klara Kaleb <klara.kaleb18@ic.ac.uk>, Adrian Altenhoff <adrian.altenhoff@inf.ethz.ch>\n",
      "Authors:\n",
      "Author: Klara Kaleb\n",
      "Dependencies:\n",
      "  - R (>= 3.5) type: depends\n",
      "  - httr (>= 1.2.1) type: depends\n",
      "  - plyr (>= 1.8.4) type: depends\n",
      "  - utils, type: imports\n",
      "  - ape, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - methods, type: imports\n",
      "  - topGO, type: imports\n",
      "  - jsonlite, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/OmaDB.html\n",
      "\n",
      "Name: signatureSearch\n",
      "Description:\n",
      "This package implements algorithms and data structures for performing gene expression signature (GES) searches, and subsequently interpreting the results functionally with specialized enrichment methods.\n",
      "Version: 1.12.0\n",
      "Publication date: None\n",
      "Mantainer: Brendan Gongol <bgong001@ucr.edu>\n",
      "Authors:\n",
      "Author: Yuzhu Duan [aut], Brendan Gongol [cre, aut], Thomas Girke [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.6.0) type: depends\n",
      "  - Rcpp, type: depends\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - org.Hs.eg.db, type: depends\n",
      "  - AnnotationDbi, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - data.table, type: imports\n",
      "  - ExperimentHub, type: imports\n",
      "  - HDF5Array, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - RSQLite, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - fgsea, type: imports\n",
      "  - scales, type: imports\n",
      "  - methods, type: imports\n",
      "  - qvalue, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - visNetwork, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - fastmatch, type: imports\n",
      "  - reactome.db, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - clusterProfiler, type: imports\n",
      "  - readr, type: imports\n",
      "  - DOSE, type: imports\n",
      "  - rhdf5, type: imports\n",
      "  - GSEABase, type: imports\n",
      "  - DelayedArray, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - tibble, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/signatureSearch.html\n",
      "\n",
      "Name: AneuFinder\n",
      "Description:\n",
      "AneuFinder implements functions for copy-number detection, breakpoint detection, and karyotype and heterogeneity analysis in single-cell whole genome sequencing and strand-seq data.\n",
      "Version: 1.26.0\n",
      "Publication date: None\n",
      "Mantainer: Aaron Taudt <aaron.taudt@gmail.com>\n",
      "Authors:\n",
      "Author: Aaron Taudt, Bjorn Bakker, David Porubsky\n",
      "Dependencies:\n",
      "  - R (>= 3.5) type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - ggplot2, type: depends\n",
      "  - cowplot, type: depends\n",
      "  - AneuFinderData, type: depends\n",
      "  - methods, type: imports\n",
      "  - utils, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - foreach, type: imports\n",
      "  - doParallel, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - bamsignals, type: imports\n",
      "  - DNAcopy, type: imports\n",
      "  - ecp, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - GenomicAlignments, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - ggdendro, type: imports\n",
      "  - ggrepel, type: imports\n",
      "  - ReorderCluster, type: imports\n",
      "  - mclust, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/AneuFinder.html\n",
      "\n",
      "Name: BiRewire\n",
      "Description:\n",
      "Fast functions for bipartite network rewiring through N consecutive switching steps (See References) and for the computation of the minimal number of switching steps to be performed in order to maximise the dissimilarity with respect to the original network. Includes functions for the analysis of the introduced randomness across the switching steps and several other routines to analyse the resulting networks and their natural projections. Extension to undirected networks and directed signed networks is also provided. Starting from version 1.9.7 a more precise bound (especially for small network) has been implemented. Starting from version 2.2.0 the analysis routine is more complete and a visual montioring of the underlying Markov Chain has been implemented. Starting from 3.6.0 the library can handle also matrices with NA (not for the directed signed graphs). Since version 3.27.1 it is possible to add a constraint for dsg generation: usually positive and negative arc between two nodes could be not accepted.\n",
      "Version: 3.30.0\n",
      "Publication date: None\n",
      "Mantainer: Andrea Gobbi <gobbi.andrea@mail.com>\n",
      "Authors:\n",
      "Author: Andrea Gobbi [aut], Francesco Iorio [aut], Giuseppe Jurman [cbt], Davide Albanese [cbt], Julio Saez-Rodriguez [cbt].\n",
      "Dependencies:\n",
      "  - igraph, type: depends\n",
      "  - slam, type: depends\n",
      "  - Rtsne, type: depends\n",
      "  - Matrix, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/BiRewire.html\n",
      "\n",
      "Name: clipper\n",
      "Description:\n",
      "Implements topological gene set analysis using a two-step empirical approach. It exploits graph decomposition theory to create a junction tree and reconstruct the most relevant signal path. In the first step clipper selects significant pathways according to statistical tests on the means and the concentration matrices of the graphs derived from pathway topologies. Then, it \"clips\" the whole pathway identifying the signal paths having the greatest association with a specific phenotype.\n",
      "Version: 1.38.0\n",
      "Publication date: None\n",
      "Mantainer: Paolo Martini <paolo.cavei@gmail.com>\n",
      "Authors:\n",
      "Author: Paolo Martini <paolo.cavei at gmail.com>, Gabriele Sales <gabriele.sales at unipd.it>, Chiara Romualdi <chiara.romualdi at unipd.it>\n",
      "Dependencies:\n",
      "  - R (>= 2.15.0) type: depends\n",
      "  - Matrix, type: depends\n",
      "  - graph, type: depends\n",
      "  - methods, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - Rcpp, type: imports\n",
      "  - igraph, type: imports\n",
      "  - gRbase (>= 1.6.6) type: imports\n",
      "  - qpgraph, type: imports\n",
      "  - KEGGgraph, type: imports\n",
      "  - corpcor, type: imports\n",
      "  - RBGL, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "AGPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/clipper.html\n",
      "\n",
      "Name: mfa\n",
      "Description:\n",
      "MFA models genomic bifurcations using a Bayesian hierarchical mixture of factor analysers.\n",
      "Version: 1.20.0\n",
      "Publication date: None\n",
      "Mantainer: Kieran Campbell <kieranrcampbell@gmail.com>\n",
      "Authors:\n",
      "Author: Kieran Campbell [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 3.4.0) type: depends\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - Rcpp, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - ggmcmc, type: imports\n",
      "  - MCMCpack, type: imports\n",
      "  - MCMCglmm, type: imports\n",
      "  - coda, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - tibble, type: imports\n",
      "  - Biobase, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/mfa.html\n",
      "\n",
      "Name: MSstatsPTM\n",
      "Description:\n",
      "MSstatsPTM provides general statistical methods for quantitative characterization of post-translational modifications (PTMs). Supports DDA, DIA, SRM, and tandem mass tag (TMT) labeling. Typically, the analysis involves the quantification of PTM sites (i.e., modified residues) and their corresponding proteins, as well as the integration of the quantification results. MSstatsPTM provides functions for summarization, estimation of PTM site abundance, and detection of changes in PTMs across experimental conditions.\n",
      "Version: 2.0.2\n",
      "Publication date: None\n",
      "Mantainer: Devon Kohler <kohler.d@northeastern.edu>\n",
      "Authors:\n",
      "Author: Devon Kohler [aut, cre], Tsung-Heng Tsai [aut], Ting Huang [aut], Mateusz Staniak [aut], Meena Choi [aut], Olga Vitek [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.2) type: depends\n",
      "  - dplyr, type: imports\n",
      "  - gridExtra, type: imports\n",
      "  - stringr, type: imports\n",
      "  - stats, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - MSstatsTMT, type: imports\n",
      "  - MSstatsConvert, type: imports\n",
      "  - MSstats, type: imports\n",
      "  - data.table, type: imports\n",
      "  - Rcpp, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - checkmate, type: imports\n",
      "  - ggrepel, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/MSstatsPTM.html\n",
      "\n",
      "Name: NanoStringQCPro\n",
      "Description:\n",
      "NanoStringQCPro provides a set of quality metrics that can be used to assess the quality of NanoString mRNA gene expression data -- i.e. to identify outlier probes and outlier samples. It also provides different background subtraction and normalization approaches for this data. It outputs suggestions for flagging samples/probes and an easily sharable html quality control output.\n",
      "Version: 1.30.0\n",
      "Publication date: None\n",
      "Mantainer: Robert Ziman <ziman.robert@gene.com>\n",
      "Authors:\n",
      "Author: Dorothee Nickles <nickles.dorothee at gene.com>, Thomas Sandmann <sandmann.thomas at gene.com>, Robert Ziman <ziman.robert at gene.com>, Richard Bourgon <bourgon.richard at gene.com>\n",
      "Dependencies:\n",
      "  - R (>= 3.2) type: depends\n",
      "  - methods, type: depends\n",
      "  - AnnotationDbi, type: imports\n",
      "  - org.Hs.eg.db, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - knitr (>= 1.12) type: imports\n",
      "  - NMF (>= 0.20.5) type: imports\n",
      "  - RColorBrewer (>= 1.0-5) type: imports\n",
      "  - png (>= 0.1-7) type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/NanoStringQCPro.html\n",
      "\n",
      "Name: rsbml\n",
      "Description:\n",
      "Links R to libsbml for SBML parsing, validating output, provides an S4 SBML DOM, converts SBML to R graph objects. Optionally links to the SBML ODE Solver Library (SOSLib) for simulating models.\n",
      "Version: 2.56.0\n",
      "Publication date: None\n",
      "Mantainer: Michael Lawrence <michafla@gene.com>\n",
      "Authors:\n",
      "Author: Michael Lawrence <michafla at gene.com>\n",
      "Dependencies:\n",
      "  - R (>= 2.6.0) type: depends\n",
      "  - BiocGenerics, type: depends\n",
      "  - methods, type: depends\n",
      "  - utils, type: depends\n",
      "  - BiocGenerics, type: imports\n",
      "  - graph, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/rsbml.html\n",
      "\n",
      "Name: trio\n",
      "Description:\n",
      "Testing SNPs and SNP interactions with a genotypic TDT. This package furthermore contains functions for computing pairwise values of LD measures and for identifying LD blocks, as well as functions for setting up matched case pseudo-control genotype data for case-parent trios in order to run trio logic regression, for imputing missing genotypes in trios, for simulating case-parent trios with disease risk dependent on SNP interaction, and for power and sample size calculation in trio data.\n",
      "Version: 3.36.0\n",
      "Publication date: None\n",
      "Mantainer: Holger Schwender <holger.schw@gmx.de>\n",
      "Authors:\n",
      "Author: Holger Schwender, Qing Li, Philipp Berger, Christoph Neumann, Margaret Taub, Ingo Ruczinski\n",
      "Dependencies:\n",
      "  - R (>= 3.0.1) type: depends\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - survival, type: imports\n",
      "  - utils, type: imports\n",
      "  - siggenes, type: imports\n",
      "  - LogicReg (>= 1.6.1) type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/trio.html\n",
      "\n",
      "Name: annotationTools\n",
      "Description:\n",
      "Functions to annotate microarrays, find orthologs, and integrate heterogeneous gene expression profiles using annotation and other molecular biology information available as flat file database (plain text files).\n",
      "Version: 1.72.0\n",
      "Publication date: None\n",
      "Mantainer: Alexandre Kuhn <alexandre.m.kuhn@gmail.com>\n",
      "Authors:\n",
      "Author: Alexandre Kuhn <alexandre.m.kuhn at gmail.com>\n",
      "Dependencies:\n",
      "  - Biobase, type: imports\n",
      "  - stats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/annotationTools.html\n",
      "\n",
      "Name: chopsticks\n",
      "Description:\n",
      "Implements classes and methods for large-scale SNP association studies\n",
      "Version: 1.64.0\n",
      "Publication date: None\n",
      "Mantainer: Hin-Tak Leung <htl10@users.sourceforge.net>\n",
      "Authors:\n",
      "Author: Hin-Tak Leung <htl10 at users.sourceforge.net>\n",
      "Dependencies:\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - methods, type: imports\n",
      "  - survival, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/chopsticks.html\n",
      "\n",
      "Name: FEAST\n",
      "Description:\n",
      "Cell clustering is one of the most important and commonly performed tasks in single-cell RNA sequencing (scRNA-seq) data analysis. An important step in cell clustering is to select a subset of genes (referred to as âfeaturesâ), whose expression patterns will then be used for downstream clustering. A good set of features should include the ones that distinguish different cell types, and the quality of such set could have significant impact on the clustering accuracy. FEAST is an R library for selecting most representative features before performing the core of scRNA-seq clustering. It can be used as a plug-in for the etablished clustering algorithms such as SC3, TSCAN, SHARP, SIMLR, and Seurat. The core of FEAST algorithm includes three steps: 1. consensus clustering; 2. gene-level significance inference; 3. validation of an optimized feature set.\n",
      "Version: 1.6.0\n",
      "Publication date: None\n",
      "Mantainer: Kenong Su <kenong.su@emory.edu>\n",
      "Authors:\n",
      "Author: Kenong Su [aut, cre], Hao Wu [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - mclust, type: depends\n",
      "  - BiocParallel, type: depends\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - SingleCellExperiment, type: imports\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - irlba, type: imports\n",
      "  - TSCAN, type: imports\n",
      "  - SC3, type: imports\n",
      "  - matrixStats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/FEAST.html\n",
      "\n",
      "Name: keggorthology\n",
      "Description:\n",
      "graphical representation of the Feb 2010 KEGG Orthology. The KEGG orthology is a set of pathway IDs that are not to be confused with the KEGG ortholog IDs.\n",
      "Version: 2.50.0\n",
      "Publication date: None\n",
      "Mantainer: VJ Carey <stvjc@channing.harvard.edu>\n",
      "Authors:\n",
      "Author: VJ Carey <stvjc at channing.harvard.edu>\n",
      "Dependencies:\n",
      "  - R (>= 2.5.0) type: depends\n",
      "  - stats, type: depends\n",
      "  - graph, type: depends\n",
      "  - hgu95av2.db, type: depends\n",
      "  - AnnotationDbi, type: imports\n",
      "  - graph, type: imports\n",
      "  - DBI, type: imports\n",
      "  - graph, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - tools, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/keggorthology.html\n",
      "\n",
      "Name: PhosR\n",
      "Description:\n",
      "PhosR is a package for the comprenhensive analysis of phosphoproteomic data. There are two major components to PhosR: processing and downstream analysis. PhosR consists of various processing tools for phosphoproteomics data including filtering, imputation, normalisation, and functional analysis for inferring active kinases and signalling pathways.\n",
      "Version: 1.8.0\n",
      "Publication date: None\n",
      "Mantainer: Taiyun Kim <taiyun.kim91@gmail.com>\n",
      "Authors:\n",
      "Author: Pengyi Yang [aut], Taiyun Kim [aut, cre], Hani Jieun Kim [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.1.0) type: depends\n",
      "  - ruv, type: imports\n",
      "  - e1071, type: imports\n",
      "  - dendextend, type: imports\n",
      "  - limma, type: imports\n",
      "  - pcaMethods, type: imports\n",
      "  - stats, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - circlize, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - igraph, type: imports\n",
      "  - pheatmap, type: imports\n",
      "  - preprocessCore, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - rlang, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - utils, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - methods, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - GGally, type: imports\n",
      "  - ggdendro, type: imports\n",
      "  - ggpubr, type: imports\n",
      "  - network, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - ggtext, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3 + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/PhosR.html\n",
      "\n",
      "Name: PROPER\n",
      "Description:\n",
      "This package provide simulation based methods for evaluating the statistical power in differential expression analysis from RNA-seq data.\n",
      "Version: 1.30.0\n",
      "Publication date: None\n",
      "Mantainer: Hao Wu <hao.wu@emory.edu>\n",
      "Authors:\n",
      "Author: Hao Wu\n",
      "Dependencies:\n",
      "  - R (>= 3.3) type: depends\n",
      "  - edgeR, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/PROPER.html\n",
      "\n",
      "Name: shinyMethyl\n",
      "Description:\n",
      "Interactive tool for visualizing Illumina methylation array data. Both the 450k and EPIC array are supported.\n",
      "Version: 1.34.0\n",
      "Publication date: None\n",
      "Mantainer: Jean-Philippe Fortin <jfortin@jhsph.edu>\n",
      "Authors:\n",
      "Author: Jean-Philippe Fortin [cre, aut], Kasper Daniel Hansen [aut]\n",
      "Dependencies:\n",
      "  - methods, type: depends\n",
      "  - BiocGenerics, type: depends\n",
      "  - shiny (>= 0.13.2) type: depends\n",
      "  - minfi, type: depends\n",
      "  - IlluminaHumanMethylation450kmanifest, type: depends\n",
      "  - matrixStats, type: depends\n",
      "  - R (>= 3.0.0) type: depends\n",
      "  - RColorBrewer, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/shinyMethyl.html\n",
      "\n",
      "Name: spicyR\n",
      "Description:\n",
      "spicyR provides a series of functions to aid in the analysis of both immunofluorescence and mass cytometry imaging data as well as other assays that can deeply phenotype individual cells and their spatial location.\n",
      "Version: 1.10.7\n",
      "Publication date: None\n",
      "Mantainer: Ellis Patrick <ellis.patrick@sydney.edu.au>\n",
      "Authors:\n",
      "Author: Nicolas Canete [aut], Ellis Patrick [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - ggplot2, type: imports\n",
      "  - concaveman, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - spatstat.explore, type: imports\n",
      "  - spatstat.geom, type: imports\n",
      "  - lmerTest, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - lme4, type: imports\n",
      "  - methods, type: imports\n",
      "  - mgcv, type: imports\n",
      "  - pheatmap, type: imports\n",
      "  - rlang, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - stats, type: imports\n",
      "  - data.table, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - scam, type: imports\n",
      "  - SingleCellExperiment, type: imports\n",
      "  - SpatialExperiment, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - ggforce, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>=2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/spicyR.html\n",
      "\n",
      "Name: YAPSA\n",
      "Description:\n",
      "This package provides functions and routines for supervised analyses of mutational signatures (i.e., the signatures have to be known, cf. L. Alexandrov et al., Nature 2013 and L. Alexandrov et al., Bioaxiv 2018). In particular, the family of functions LCD (LCD = linear combination decomposition) can use optimal signature-specific cutoffs which takes care of different detectability of the different signatures. Moreover, the package provides different sets of mutational signatures, including the COSMIC and PCAWG SNV signatures and the PCAWG Indel signatures; the latter infering that with YAPSA, the concept of supervised analysis of mutational signatures is extended to Indel signatures. YAPSA also provides confidence intervals as computed by profile likelihoods and can perform signature analysis on a stratified mutational catalogue (SMC = stratify mutational catalogue) in order to analyze enrichment and depletion patterns for the signatures in different strata.\n",
      "Version: 1.24.0\n",
      "Publication date: None\n",
      "Mantainer: Daniel Huebschmann <huebschmann.daniel@googlemail.com>\n",
      "Authors:\n",
      "Author: Daniel Huebschmann [aut, cre], Lea Jopp-Saile [aut], Carolin Andresen [aut], Zuguang Gu [aut], Matthias Schlesner [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.6.0) type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - ggplot2, type: depends\n",
      "  - grid, type: depends\n",
      "  - limSolve, type: imports\n",
      "  - SomaticSignatures, type: imports\n",
      "  - VariantAnnotation, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - gridExtra, type: imports\n",
      "  - corrplot, type: imports\n",
      "  - dendextend, type: imports\n",
      "  - GetoptLong, type: imports\n",
      "  - circlize, type: imports\n",
      "  - gtrellis, type: imports\n",
      "  - doParallel, type: imports\n",
      "  - PMCMRplus, type: imports\n",
      "  - ggbeeswarm, type: imports\n",
      "  - ComplexHeatmap, type: imports\n",
      "  - KEGGREST, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - BSgenome.Hsapiens.UCSC.hg19, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - pracma, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/YAPSA.html\n",
      "\n",
      "Name: Chicago\n",
      "Description:\n",
      "A pipeline for analysing Capture Hi-C data.\n",
      "Version: 1.26.0\n",
      "Publication date: None\n",
      "Mantainer: Mikhail Spivakov <mikhail.spivakov@lms.mrc.ac.uk>\n",
      "Authors:\n",
      "Author: Jonathan Cairns, Paula Freire Pritchett, Steven Wingett, Mikhail Spivakov\n",
      "Dependencies:\n",
      "  - R (>= 3.3.1) type: depends\n",
      "  - data.table, type: depends\n",
      "  - matrixStats, type: imports\n",
      "  - MASS, type: imports\n",
      "  - Hmisc, type: imports\n",
      "  - Delaporte, type: imports\n",
      "  - methods, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/Chicago.html\n",
      "\n",
      "Name: chromstaR\n",
      "Description:\n",
      "This package implements functions for combinatorial and differential analysis of ChIP-seq data. It includes uni- and multivariate peak-calling, export to genome browser viewable files, and functions for enrichment analyses.\n",
      "Version: 1.24.0\n",
      "Publication date: None\n",
      "Mantainer: Aaron Taudt <aaron.taudt@gmail.com>\n",
      "Authors:\n",
      "Author: Aaron Taudt, Maria Colome Tatche, Matthias Heinig, Minh Anh Nguyen\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - ggplot2, type: depends\n",
      "  - chromstaRData, type: depends\n",
      "  - methods, type: imports\n",
      "  - utils, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - foreach, type: imports\n",
      "  - doParallel, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - GenomicAlignments, type: imports\n",
      "  - bamsignals, type: imports\n",
      "  - mvtnorm, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/chromstaR.html\n",
      "\n",
      "Name: GRmetrics\n",
      "Description:\n",
      "Functions for calculating and visualizing growth-rate inhibition (GR) metrics.\n",
      "Version: 1.24.0\n",
      "Publication date: None\n",
      "Mantainer: Nicholas Clark <nicholas.clark00@gmail.com>, Mario Medvedovic <medvedm@ucmail.uc.edu>\n",
      "Authors:\n",
      "Author: Nicholas Clark\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - drc, type: imports\n",
      "  - plotly, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - stats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/GRmetrics.html\n",
      "\n",
      "Name: HubPub\n",
      "Description:\n",
      "HubPub provides users with functionality to help with the Bioconductor Hub structures. The package provides the ability to create a skeleton of a Hub style package that the user can then populate with the necessary information. There are also functions to help add resources to the Hub package metadata files as well as publish data to the Bioconductor S3 bucket.\n",
      "Version: 1.6.0\n",
      "Publication date: None\n",
      "Mantainer: Kayla Interdonato <kayla.morrell@roswellpark.org>\n",
      "Authors:\n",
      "Author: Kayla Interdonato [aut, cre], Martin Morgan [aut]\n",
      "Dependencies:\n",
      "  - available, type: imports\n",
      "  - usethis, type: imports\n",
      "  - biocthis, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - aws.s3, type: imports\n",
      "  - fs, type: imports\n",
      "  - BiocManager, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/HubPub.html\n",
      "\n",
      "Name: RRHO\n",
      "Description:\n",
      "The package is aimed at inference on the amount of agreement in two sorted lists using the Rank-Rank Hypergeometric Overlap test.\n",
      "Version: 1.38.0\n",
      "Publication date: None\n",
      "Mantainer: Jonathan Rosenblatt <john.ros.work@gmail.com>\n",
      "Authors:\n",
      "Author: Jonathan Rosenblatt and Jason Stein\n",
      "Dependencies:\n",
      "  - R (>= 2.10) type: depends\n",
      "  - grid, type: depends\n",
      "  - VennDiagram, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/RRHO.html\n",
      "\n",
      "Name: scPipe\n",
      "Description:\n",
      "A preprocessing pipeline for single cell RNA-seq/ATAC-seq data that starts from the fastq files and produces a feature count matrix with associated quality control information. It can process fastq data generated by CEL-seq, MARS-seq, Drop-seq, Chromium 10x and SMART-seq protocols.\n",
      "Version: 1.20.4\n",
      "Publication date: None\n",
      "Mantainer: Shian Su <su.s@wehi.edu.au>\n",
      "Authors:\n",
      "Author: Luyi Tian [aut], Shian Su [aut, cre], Shalin Naik [ctb], Shani Amarasinghe [aut], Oliver Voogd [aut], Phil Yang [aut], Matthew Ritchie [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 4.2.0) type: depends\n",
      "  - SingleCellExperiment, type: depends\n",
      "  - AnnotationDbi, type: imports\n",
      "  - basilisk, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - biomaRt, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - data.table, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - DropletUtils, type: imports\n",
      "  - flexmix, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - GenomicAlignments, type: imports\n",
      "  - GGally, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - glue (>= 1.3.0) type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - hash, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - MASS, type: imports\n",
      "  - Matrix (>= 1.5.0) type: imports\n",
      "  - mclust, type: imports\n",
      "  - methods, type: imports\n",
      "  - MultiAssayExperiment, type: imports\n",
      "  - org.Hs.eg.db, type: imports\n",
      "  - org.Mm.eg.db, type: imports\n",
      "  - purrr, type: imports\n",
      "  - Rcpp (>= 0.11.3) type: imports\n",
      "  - reshape, type: imports\n",
      "  - reticulate, type: imports\n",
      "  - Rhtslib, type: imports\n",
      "  - rlang, type: imports\n",
      "  - robustbase, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - Rsubread, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - scales, type: imports\n",
      "  - stats, type: imports\n",
      "  - stringr, type: imports\n",
      "  - tibble, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - tools, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/scPipe.html\n",
      "\n",
      "Name: chimeraviz\n",
      "Description:\n",
      "chimeraviz manages data from fusion gene finders and provides useful visualization tools.\n",
      "Version: 1.24.0\n",
      "Publication date: None\n",
      "Mantainer: Stian LÃ¥gstad <stianlagstad@gmail.com>\n",
      "Authors:\n",
      "Author: Stian LÃ¥gstad [aut, cre], Sen Zhao [ctb], Andreas M. Hoff [ctb], Bjarne Johannessen [ctb], Ole Christian LingjÃ¦rde [ctb], Rolf Skotheim [ctb]\n",
      "Dependencies:\n",
      "  - Biostrings, type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - IRanges, type: depends\n",
      "  - Gviz, type: depends\n",
      "  - S4Vectors, type: depends\n",
      "  - ensembldb, type: depends\n",
      "  - AnnotationFilter, type: depends\n",
      "  - data.table, type: depends\n",
      "  - methods, type: imports\n",
      "  - grid, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - GenomicAlignments, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - graphics, type: imports\n",
      "  - AnnotationDbi, type: imports\n",
      "  - RCircos, type: imports\n",
      "  - org.Hs.eg.db, type: imports\n",
      "  - org.Mm.eg.db, type: imports\n",
      "  - rmarkdown, type: imports\n",
      "  - graph, type: imports\n",
      "  - Rgraphviz, type: imports\n",
      "  - DT, type: imports\n",
      "  - plyr, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - BiocStyle, type: imports\n",
      "  - checkmate, type: imports\n",
      "  - gtools, type: imports\n",
      "  - magick, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/chimeraviz.html\n",
      "\n",
      "Name: goProfiles\n",
      "Description:\n",
      "The package implements methods to compare lists of genes based on comparing the corresponding 'functional profiles'.\n",
      "Version: 1.60.0\n",
      "Publication date: None\n",
      "Mantainer: Alex Sanchez <asanchez@ub.edu>\n",
      "Authors:\n",
      "Author: Alex Sanchez, Jordi Ocana and Miquel Salicru\n",
      "Dependencies:\n",
      "  - Biobase, type: depends\n",
      "  - AnnotationDbi, type: depends\n",
      "  - GO.db, type: depends\n",
      "  - CompQuadForm, type: depends\n",
      "  - stringr, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/goProfiles.html\n",
      "\n",
      "Name: interactiveDisplay\n",
      "Description:\n",
      "The interactiveDisplay package contains the methods needed to generate interactive Shiny based display methods for Bioconductor objects.\n",
      "Version: 1.36.0\n",
      "Publication date: None\n",
      "Mantainer: Bioconductor Package Maintainer <maintainer@bioconductor.org>\n",
      "Authors:\n",
      "Author: Bioconductor Package Maintainer [cre], Shawn Balcome [aut], Marc Carlson [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - BiocGenerics, type: depends\n",
      "  - grid, type: depends\n",
      "  - interactiveDisplayBase, type: imports\n",
      "  - shiny, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - plyr, type: imports\n",
      "  - gridSVG, type: imports\n",
      "  - XML, type: imports\n",
      "  - Category, type: imports\n",
      "  - AnnotationDbi, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/interactiveDisplay.html\n",
      "\n",
      "Name: SEtools\n",
      "Description:\n",
      "This includes a set of tools for working with the SummarizedExperiment class, including merging, melting, aggregation functions. Plotting functions historically in this package have been moved to the sechm package.\n",
      "Version: 1.12.0\n",
      "Publication date: None\n",
      "Mantainer: Pierre-Luc Germain <pierre-luc.germain@hest.ethz.ch>\n",
      "Authors:\n",
      "Author: Pierre-Luc Germain [cre, aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - sechm, type: depends\n",
      "  - BiocParallel, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - DESeq2, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - data.table, type: imports\n",
      "  - edgeR, type: imports\n",
      "  - openxlsx, type: imports\n",
      "  - pheatmap, type: imports\n",
      "  - stats, type: imports\n",
      "  - circlize, type: imports\n",
      "  - methods, type: imports\n",
      "  - sva, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/SEtools.html\n",
      "\n",
      "Name: Spaniel\n",
      "Description:\n",
      "Spaniel includes a series of tools to aid the quality control and analysis of Spatial Transcriptomics data. Spaniel can import data from either the original Spatial Transcriptomics system or 10X Visium technology. The package contains functions to create a SingleCellExperiment Seurat object and provides a method of loading a histologial image into R. The spanielPlot function allows visualisation of metrics contained within the S4 object overlaid onto the image of the tissue.\n",
      "Version: 1.12.0\n",
      "Publication date: None\n",
      "Mantainer: Rachel Queen <rachel.queen@newcastle.ac.uk>\n",
      "Authors:\n",
      "Author: Rachel Queen [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - Seurat, type: imports\n",
      "  - SingleCellExperiment, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - methods, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - scater, type: imports\n",
      "  - scran, type: imports\n",
      "  - igraph, type: imports\n",
      "  - shiny, type: imports\n",
      "  - jpeg, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - utils, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - DropletUtils, type: imports\n",
      "  - jsonlite, type: imports\n",
      "  - png, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/Spaniel.html\n",
      "\n",
      "Name: sparrow\n",
      "Description:\n",
      "Provides a unified interface to a variety of GSEA techniques from different bioconductor packages. Results are harmonized into a single object and can be interrogated uniformly for quick exploration and interpretation of results. Interactive exploration of GSEA results is enabled through a shiny app provided by a sparrow.shiny sibling package.\n",
      "Version: 1.4.0\n",
      "Publication date: None\n",
      "Mantainer: Steve Lianoglou <slianoglou@gmail.com>\n",
      "Authors:\n",
      "Author: Steve Lianoglou [aut, cre] , Arkadiusz Gladki [ctb], Denali Therapeutics [fnd] (2018+), Genentech [fnd] (2014 - 2017)\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - babelgene (>= 21.4) type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - BiocSet, type: imports\n",
      "  - checkmate, type: imports\n",
      "  - circlize, type: imports\n",
      "  - ComplexHeatmap, type: imports\n",
      "  - data.table (>= 1.10.4) type: imports\n",
      "  - DelayedMatrixStats, type: imports\n",
      "  - edgeR, type: imports\n",
      "  - ggplot2 (>= 2.2.0) type: imports\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - GSEABase, type: imports\n",
      "  - irlba, type: imports\n",
      "  - limma, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - methods, type: imports\n",
      "  - plotly (>= 4.9.0) type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - viridis, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/sparrow.html\n",
      "\n",
      "Name: alevinQC\n",
      "Description:\n",
      "Generate QC reports summarizing the output from an alevin run. Reports can be generated as html or pdf files, or as shiny applications.\n",
      "Version: 1.14.0\n",
      "Publication date: None\n",
      "Mantainer: Charlotte Soneson <charlottesoneson@gmail.com>\n",
      "Authors:\n",
      "Author: Charlotte Soneson [aut, cre] , Avi Srivastava [aut], Rob Patro [aut], Dongze He [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - rmarkdown (>= 2.5) type: imports\n",
      "  - tools, type: imports\n",
      "  - methods, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - GGally, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - rjson, type: imports\n",
      "  - shiny, type: imports\n",
      "  - shinydashboard, type: imports\n",
      "  - DT, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - tximport, type: imports\n",
      "  - cowplot, type: imports\n",
      "  - rlang, type: imports\n",
      "  - Rcpp, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/alevinQC.html\n",
      "\n",
      "Name: ASpli\n",
      "Description:\n",
      "Integrative pipeline for the analysis of alternative splicing using RNAseq.\n",
      "Version: 2.8.0\n",
      "Publication date: None\n",
      "Mantainer: Estefania Mancini <emancini@leloir.org.ar>\n",
      "Authors:\n",
      "Author: Estefania Mancini, Andres Rabinovich, Javier Iserte, Marcelo Yanovsky and Ariel Chernomoretz\n",
      "Dependencies:\n",
      "  - methods, type: depends\n",
      "  - grDevices, type: depends\n",
      "  - stats, type: depends\n",
      "  - utils, type: depends\n",
      "  - parallel, type: depends\n",
      "  - edgeR, type: depends\n",
      "  - limma, type: depends\n",
      "  - AnnotationDbi, type: depends\n",
      "  - GenomicRanges, type: imports\n",
      "  - GenomicFeatures, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - GenomicAlignments, type: imports\n",
      "  - Gviz, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - BiocStyle, type: imports\n",
      "  - igraph, type: imports\n",
      "  - htmltools, type: imports\n",
      "  - data.table, type: imports\n",
      "  - UpSetR, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - DT, type: imports\n",
      "  - MASS, type: imports\n",
      "  - grid, type: imports\n",
      "  - graphics, type: imports\n",
      "  - pbmcapply, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ASpli.html\n",
      "\n",
      "Name: bigPint\n",
      "Description:\n",
      "Methods for visualizing large multivariate datasets using static and interactive scatterplot matrices, parallel coordinate plots, volcano plots, and litre plots. Includes examples for visualizing RNA-sequencing datasets and differentially expressed genes.\n",
      "Version: 1.14.0\n",
      "Publication date: None\n",
      "Mantainer: Lindsay Rutter <lindsayannerutter@gmail.com>\n",
      "Authors:\n",
      "Author: Lindsay Rutter [aut, cre], Dianne Cook [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.6.0) type: depends\n",
      "  - DelayedArray, type: imports\n",
      "  - dplyr (>= 0.7.2) type: imports\n",
      "  - GGally (>= 1.3.2) type: imports\n",
      "  - ggplot2 (>= 2.2.1) type: imports\n",
      "  - graphics (>= 3.5.0) type: imports\n",
      "  - grDevices (>= 3.5.0) type: imports\n",
      "  - grid (>= 3.5.0) type: imports\n",
      "  - gridExtra (>= 2.3) type: imports\n",
      "  - hexbin (>= 1.27.1) type: imports\n",
      "  - Hmisc (>= 4.0.3) type: imports\n",
      "  - htmlwidgets (>= 0.9) type: imports\n",
      "  - methods (>= 3.5.2) type: imports\n",
      "  - plotly (>= 4.7.1) type: imports\n",
      "  - plyr (>= 1.8.4) type: imports\n",
      "  - RColorBrewer (>= 1.1.2) type: imports\n",
      "  - reshape (>= 0.8.7) type: imports\n",
      "  - shiny (>= 1.0.5) type: imports\n",
      "  - shinycssloaders (>= 0.2.0) type: imports\n",
      "  - shinydashboard (>= 0.6.1) type: imports\n",
      "  - stats (>= 3.5.0) type: imports\n",
      "  - stringr (>= 1.3.1) type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - tidyr (>= 0.7.0) type: imports\n",
      "  - utils (>= 3.5.0) type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/bigPint.html\n",
      "\n",
      "Name: dearseq\n",
      "Description:\n",
      "Differential Expression Analysis RNA-seq data with variance component score test accounting for data heteroscedasticity through precision weights. Perform both gene-wise and gene set analyses, and can deal with repeated or longitudinal data. Methods are detailed in: i) Agniel D & Hejblum BP (2017) Variance component score test for time-course gene set analysis of longitudinal RNA-seq data, Biostatistics, 18(4):589-604 ; and ii) Gauthier M, Agniel D, ThiÃ©baut R & Hejblum BP (2020) dearseq: a variance component score test for RNA-Seq differential analysis that effectively controls the false discovery rate, NAR Genomics and Bioinformatics, 2(4):lqaa093.\n",
      "Version: 1.10.0\n",
      "Publication date: None\n",
      "Mantainer: Boris P. Hejblum <boris.hejblum@u-bordeaux.fr>\n",
      "Authors:\n",
      "Author: Denis Agniel [aut], Boris P. Hejblum [aut, cre], Marine Gauthier [aut], MÃ©lanie Huchon [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 3.6.0) type: depends\n",
      "  - CompQuadForm, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - KernSmooth, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - methods, type: imports\n",
      "  - patchwork, type: imports\n",
      "  - parallel, type: imports\n",
      "  - pbapply, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - rlang, type: imports\n",
      "  - scattermore, type: imports\n",
      "  - stats, type: imports\n",
      "  - statmod, type: imports\n",
      "  - survey, type: imports\n",
      "  - tibble, type: imports\n",
      "  - viridisLite, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2 | file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/dearseq.html\n",
      "\n",
      "Name: FCBF\n",
      "Description:\n",
      "This package provides a simple R implementation for the Fast Correlation Based Filter described in Yu, L. and Liu, H.; Feature Selection for High-Dimensional Data: A Fast Correlation Based Filter Solution,Proc. 20th Intl. Conf. Mach. Learn. (ICML-2003), Washington DC, 2003 The current package is an intent to make easier for bioinformaticians to use FCBF for feature selection, especially regarding transcriptomic data.This implies discretizing expression (function discretize_exprs) before calculating the features that explain the class, but are not predictable by other features. The functions are implemented based on the algorithm of Yu and Liu, 2003 and Rajarshi Guha's implementation from 13/05/2005 available (as of 26/08/2018) at http://www.rguha.net/code/R/fcbf.R .\n",
      "Version: 2.6.0\n",
      "Publication date: None\n",
      "Mantainer: Tiago Lubiana <tiago.lubiana.alves@usp.br>\n",
      "Authors:\n",
      "Author: Tiago Lubiana [aut, cre], Helder Nakaya [aut, ths]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - ggplot2, type: imports\n",
      "  - gridExtra, type: imports\n",
      "  - pbapply, type: imports\n",
      "  - parallel, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - stats, type: imports\n",
      "  - mclust, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/FCBF.html\n",
      "\n",
      "Name: GenomicDistributions\n",
      "Description:\n",
      "If you have a set of genomic ranges, this package can help you with visualization and comparison. It produces several kinds of plots, for example: Chromosome distribution plots, which visualize how your regions are distributed over chromosomes; feature distance distribution plots, which visualizes how your regions are distributed relative to a feature of interest, like Transcription Start Sites (TSSs); genomic partition plots, which visualize how your regions overlap given genomic features such as promoters, introns, exons, or intergenic regions. It also makes it easy to compare one set of ranges to another.\n",
      "Version: 1.6.0\n",
      "Publication date: None\n",
      "Mantainer: Kristyna Kupkova <kristynakupkova@gmail.com>\n",
      "Authors:\n",
      "Author: Kristyna Kupkova [aut, cre], Jose Verdezoto [aut], Tessa Danehy [aut], John Lawson [aut], Jose Verdezoto [aut], Michal Stolarczyk [aut], Jason Smith [aut], Bingjie Xue [aut], Sophia Rogers [aut], John Stubbs [aut], Nathan C. Sheffield [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - IRanges, type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - data.table, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - methods, type: imports\n",
      "  - utils, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - plyr, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - scales, type: imports\n",
      "  - broom, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - stats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "BSD_2_clause + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/GenomicDistributions.html\n",
      "\n",
      "Name: gwasurvivr\n",
      "Description:\n",
      "gwasurvivr is a package to perform survival analysis using Cox proportional hazard models on imputed genetic data.\n",
      "Version: 1.16.0\n",
      "Publication date: None\n",
      "Mantainer: Abbas Rizvi <aarizv@gmail.com>\n",
      "Authors:\n",
      "Author: Abbas Rizvi, Ezgi Karaesmen, Martin Morgan, Lara Sucheston-Campbell\n",
      "Dependencies:\n",
      "  - R (>= 3.4.0) type: depends\n",
      "  - GWASTools, type: imports\n",
      "  - survival, type: imports\n",
      "  - VariantAnnotation, type: imports\n",
      "  - parallel, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - SNPRelate, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/gwasurvivr.html\n",
      "\n",
      "Name: Pi\n",
      "Description:\n",
      "Priority index or Pi is developed as a genomic-led target prioritisation system. It integrates functional genomic predictors, knowledge of network connectivity and immune ontologies to prioritise potential drug targets at the gene and pathway level.\n",
      "Version: 2.10.0\n",
      "Publication date: None\n",
      "Mantainer: Hai Fang <hfang.shanghai@gmail.com>\n",
      "Authors:\n",
      "Author: Hai Fang, the ULTRA-DD Consortium, Julian C Knight\n",
      "Dependencies:\n",
      "  - igraph, type: depends\n",
      "  - dnet, type: depends\n",
      "  - ggplot2, type: depends\n",
      "  - graphics, type: depends\n",
      "  - Matrix, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - supraHex, type: imports\n",
      "  - scales, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - ggrepel, type: imports\n",
      "  - ROCR, type: imports\n",
      "  - randomForest, type: imports\n",
      "  - glmnet, type: imports\n",
      "  - lattice, type: imports\n",
      "  - caret, type: imports\n",
      "  - plot3D, type: imports\n",
      "  - stats, type: imports\n",
      "  - methods, type: imports\n",
      "  - MASS, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - ggnetwork, type: imports\n",
      "  - osfr, type: imports\n",
      "  - RCircos, type: imports\n",
      "  - purrr, type: imports\n",
      "  - readr, type: imports\n",
      "  - tibble, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/Pi.html\n",
      "\n",
      "Name: psichomics\n",
      "Description:\n",
      "Interactive R package with an intuitive Shiny-based graphical interface for alternative splicing quantification and integrative analyses of alternative splicing and gene expression based on The Cancer Genome Atlas (TCGA), the Genotype-Tissue Expression project (GTEx), Sequence Read Archive (SRA) and user-provided data. The tool interactively performs survival, dimensionality reduction and median- and variance-based differential splicing and gene expression analyses that benefit from the incorporation of clinical and molecular sample-associated features (such as tumour stage or survival). Interactive visual access to genomic mapping and functional annotation of selected alternative splicing events is also included.\n",
      "Version: 1.24.0\n",
      "Publication date: None\n",
      "Mantainer: Nuno Saraiva-Agostinho <nunodanielagostinho@gmail.com>\n",
      "Authors:\n",
      "Author: Nuno Saraiva-Agostinho [aut, cre] , Nuno LuÃ­s Barbosa-Morais [aut, led, ths] , AndrÃ© FalcÃ£o [ths], Lina Gallego Paez [ctb], Marie Bordone [ctb], Teresa Maia [ctb], Mariana Ferreira [ctb], Ana Carolina Leote [ctb], Bernardo de Almeida [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - shiny (>= 1.7.0) type: depends\n",
      "  - shinyBS, type: depends\n",
      "  - AnnotationDbi, type: imports\n",
      "  - AnnotationHub, type: imports\n",
      "  - BiocFileCache, type: imports\n",
      "  - cluster, type: imports\n",
      "  - colourpicker, type: imports\n",
      "  - data.table, type: imports\n",
      "  - digest, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - DT (>= 0.2) type: imports\n",
      "  - edgeR, type: imports\n",
      "  - fastICA, type: imports\n",
      "  - fastmatch, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - ggrepel, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - highcharter (>= 0.5.0) type: imports\n",
      "  - htmltools, type: imports\n",
      "  - httr, type: imports\n",
      "  - jsonlite, type: imports\n",
      "  - limma, type: imports\n",
      "  - pairsD3, type: imports\n",
      "  - plyr, type: imports\n",
      "  - purrr, type: imports\n",
      "  - Rcpp (>= 0.12.14) type: imports\n",
      "  - recount, type: imports\n",
      "  - Rfast, type: imports\n",
      "  - R.utils, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - shinyjs, type: imports\n",
      "  - stringr, type: imports\n",
      "  - stats, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - survival, type: imports\n",
      "  - tools, type: imports\n",
      "  - utils, type: imports\n",
      "  - XML, type: imports\n",
      "  - xtable, type: imports\n",
      "  - methods, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/psichomics.html\n",
      "\n",
      "Name: SBGNview\n",
      "Description:\n",
      "SBGNview is a tool set for pathway based data visalization, integration and analysis. SBGNview is similar and complementary to the widely used Pathview, with the following key features: 1. Pathway definition by the widely adopted Systems Biology Graphical Notation (SBGN); 2. Supports multiple major pathway databases beyond KEGG (Reactome, MetaCyc, SMPDB, PANTHER, METACROP) and user defined pathways; 3. Covers 5,200 reference pathways and over 3,000 species by default; 4. Extensive graphics controls, including glyph and edge attributes, graph layout and sub-pathway highlight; 5. SBGN pathway data manipulation, processing, extraction and analysis.\n",
      "Version: 1.12.0\n",
      "Publication date: None\n",
      "Mantainer: Weijun Luo <luo_weijun@yahoo.com>\n",
      "Authors:\n",
      "Author: Xiaoxi Dong*, Kovidh Vegesna*, Weijun Luo\n",
      "Dependencies:\n",
      "  - R (>= 3.6) type: depends\n",
      "  - pathview, type: depends\n",
      "  - SBGNview.data, type: depends\n",
      "  - Rdpack, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - xml2, type: imports\n",
      "  - rsvg, type: imports\n",
      "  - igraph, type: imports\n",
      "  - rmarkdown, type: imports\n",
      "  - knitr, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - AnnotationDbi, type: imports\n",
      "  - httr, type: imports\n",
      "  - KEGGREST, type: imports\n",
      "  - bookdown, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "AGPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/SBGNview.html\n",
      "\n",
      "Name: bayNorm\n",
      "Description:\n",
      "bayNorm is used for normalizing single-cell RNA-seq data.\n",
      "Version: 1.16.2\n",
      "Publication date: None\n",
      "Mantainer: Wenhao Tang <wt215@ic.ac.uk>\n",
      "Authors:\n",
      "Author: Wenhao Tang [aut, cre], Fran<U+00E7>ois Bertaux [aut], Philipp Thomas [aut], Claire Stefanelli [aut], Malika Saint [aut], Samuel Marguerat [aut], Vahid Shahrezaei [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.5) type: depends\n",
      "  - Rcpp (>= 0.12.12) type: imports\n",
      "  - BB, type: imports\n",
      "  - foreach, type: imports\n",
      "  - iterators, type: imports\n",
      "  - doSNOW, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - parallel, type: imports\n",
      "  - MASS, type: imports\n",
      "  - locfit, type: imports\n",
      "  - fitdistrplus, type: imports\n",
      "  - stats, type: imports\n",
      "  - methods, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - SingleCellExperiment, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/bayNorm.html\n",
      "\n",
      "Name: bugsigdbr\n",
      "Description:\n",
      "The bugsigdbr package implements convenient access to bugsigdb.org from within R/Bioconductor. The goal of the package is to facilitate import of BugSigDB data into R/Bioconductor, provide utilities for extracting microbe signatures, and enable export of the extracted signatures to plain text files in standard file formats such as GMT.\n",
      "Version: 1.4.3\n",
      "Publication date: None\n",
      "Mantainer: Ludwig Geistlinger <ludwig_geistlinger@hms.harvard.edu>\n",
      "Authors:\n",
      "Author: Ludwig Geistlinger [aut, cre], Jennifer Wokaty [aut], Levi Waldron [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - BiocFileCache, type: imports\n",
      "  - methods, type: imports\n",
      "  - vroom, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/bugsigdbr.html\n",
      "\n",
      "Name: CNVRanger\n",
      "Description:\n",
      "The CNVRanger package implements a comprehensive tool suite for CNV analysis. This includes functionality for summarizing individual CNV calls across a population, assessing overlap with functional genomic regions, and association analysis with gene expression and quantitative phenotypes.\n",
      "Version: 1.14.0\n",
      "Publication date: None\n",
      "Mantainer: Ludwig Geistlinger <ludwig_geistlinger@hms.harvard.edu>\n",
      "Authors:\n",
      "Author: Ludwig Geistlinger [aut, cre], Vinicius Henrique da Silva [aut], Marcel Ramos [ctb], Levi Waldron [ctb]\n",
      "Dependencies:\n",
      "  - GenomicRanges, type: depends\n",
      "  - RaggedExperiment, type: depends\n",
      "  - BiocGenerics, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - GDSArray, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - SNPRelate, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - data.table, type: imports\n",
      "  - edgeR, type: imports\n",
      "  - gdsfmt, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - lattice, type: imports\n",
      "  - limma, type: imports\n",
      "  - methods, type: imports\n",
      "  - plyr, type: imports\n",
      "  - qqman, type: imports\n",
      "  - rappdirs, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/CNVRanger.html\n",
      "\n",
      "Name: consensusSeekeR\n",
      "Description:\n",
      "This package compares genomic positions and genomic ranges from multiple experiments to extract common regions. The size of the analyzed region is adjustable as well as the number of experiences in which a feature must be present in a potential region to tag this region as a consensus region.\n",
      "Version: 1.26.0\n",
      "Publication date: None\n",
      "Mantainer: Astrid Deschenes <adeschen@hotmail.com>\n",
      "Authors:\n",
      "Author: Astrid Deschenes [cre, aut], Fabien Claude Lamaze [ctb], Pascal Belleau [aut], Arnaud Droit [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - BiocGenerics, type: depends\n",
      "  - IRanges, type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - BiocParallel, type: depends\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - stringr, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - methods, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/consensusSeekeR.html\n",
      "\n",
      "Name: decompTumor2Sig\n",
      "Description:\n",
      "Uses quadratic programming for signature refitting, i.e., to decompose the mutation catalog from an individual tumor sample into a set of given mutational signatures (either Alexandrov-model signatures or Shiraishi-model signatures), computing weights that reflect the contributions of the signatures to the mutation load of the tumor.\n",
      "Version: 2.14.0\n",
      "Publication date: None\n",
      "Mantainer: Rosario M. Piro <rmpiro@gmail.com>\n",
      "Authors:\n",
      "Author: Rosario M. Piro [aut, cre], Sandra Krueger [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - ggplot2, type: depends\n",
      "  - methods, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - quadprog (>= 1.5-5) type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - stats, type: imports\n",
      "  - GenomicFeatures, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - plyr, type: imports\n",
      "  - utils, type: imports\n",
      "  - graphics, type: imports\n",
      "  - BSgenome.Hsapiens.UCSC.hg19, type: imports\n",
      "  - TxDb.Hsapiens.UCSC.hg19.knownGene, type: imports\n",
      "  - VariantAnnotation, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - ggseqlogo, type: imports\n",
      "  - gridExtra, type: imports\n",
      "  - data.table, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - readxl, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/decompTumor2Sig.html\n",
      "\n",
      "Name: HybridMTest\n",
      "Description:\n",
      "Performs hybrid multiple testing that incorporates method selection and assumption evaluations into the analysis using empirical Bayes probability (EBP) estimates obtained by Grenander density estimation. For instance, for 3-group comparison analysis, Hybrid Multiple testing considers EBPs as weighted EBPs between F-test and H-test with EBPs from Shapiro Wilk test of normality as weigth. Instead of just using EBPs from F-test only or using H-test only, this methodology combines both types of EBPs through EBPs from Shapiro Wilk test of normality. This methodology uses then the law of total EBPs.\n",
      "Version: 1.42.0\n",
      "Publication date: None\n",
      "Mantainer: Demba Fofana <demba.fofana@stjude.org>\n",
      "Authors:\n",
      "Author: Stan Pounds <stanley.pounds at stjude.org>, Demba Fofana <demba.fofana at stjude.org>\n",
      "Dependencies:\n",
      "  - R (>= 2.9.0) type: depends\n",
      "  - Biobase, type: depends\n",
      "  - fdrtool, type: depends\n",
      "  - MASS, type: depends\n",
      "  - survival, type: depends\n",
      "  - stats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL Version 2 or later\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/HybridMTest.html\n",
      "\n",
      "Name: iSEEu\n",
      "Description:\n",
      "iSEEu (the iSEE universe) contains diverse functionality to extend the usage of the iSEE package, including additional classes for the panels, or modes allowing easy configuration of iSEE applications.\n",
      "Version: 1.10.0\n",
      "Publication date: None\n",
      "Mantainer: Kevin Rue-Albrecht <kevinrue67@gmail.com>\n",
      "Authors:\n",
      "Author: Kevin Rue-Albrecht [aut, cre] , Charlotte Soneson [aut] , Federico Marini [aut] , Aaron Lun [aut] , Michael Stadler [ctb]\n",
      "Dependencies:\n",
      "  - iSEE, type: depends\n",
      "  - iSEEhex, type: depends\n",
      "  - methods, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - shiny, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - SingleCellExperiment, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - DT, type: imports\n",
      "  - stats, type: imports\n",
      "  - colourpicker, type: imports\n",
      "  - shinyAce, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/iSEEu.html\n",
      "\n",
      "Name: multiGSEA\n",
      "Description:\n",
      "Extracted features from pathways derived from 8 different databases (KEGG, Reactome, Biocarta, etc.) can be used on transcriptomic, proteomic, and/or metabolomic level to calculate a combined GSEA-based enrichment score.\n",
      "Version: 1.8.1\n",
      "Publication date: None\n",
      "Mantainer: Sebastian Canzler <sebastian.canzler@ufz.de>\n",
      "Authors:\n",
      "Author: Sebastian Canzler [aut, cre] , JÃ¶rg HackermÃ¼ller [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.0.0) type: depends\n",
      "  - magrittr, type: imports\n",
      "  - graphite, type: imports\n",
      "  - AnnotationDbi, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - fgsea, type: imports\n",
      "  - metap, type: imports\n",
      "  - rappdirs, type: imports\n",
      "  - rlang, type: imports\n",
      "  - methods, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/multiGSEA.html\n",
      "\n",
      "Name: multiHiCcompare\n",
      "Description:\n",
      "multiHiCcompare provides functions for joint normalization and difference detection in multiple Hi-C datasets. This extension of the original HiCcompare package now allows for Hi-C experiments with more than 2 groups and multiple samples per group. multiHiCcompare operates on processed Hi-C data in the form of sparse upper triangular matrices. It accepts four column (chromosome, region1, region2, IF) tab-separated text files storing chromatin interaction matrices. multiHiCcompare provides cyclic loess and fast loess (fastlo) methods adapted to jointly normalizing Hi-C data. Additionally, it provides a general linear model (GLM) framework adapting the edgeR package to detect differences in Hi-C data in a distance dependent manner.\n",
      "Version: 1.16.0\n",
      "Publication date: None\n",
      "Mantainer: John Stansfield <stansfieldjc@vcu.edu>, Mikhail Dozmorov <mikhail.dozmorov@vcuhealth.org>\n",
      "Authors:\n",
      "Author: John Stansfield <stansfieldjc at vcu.edu>, Mikhail Dozmorov <mikhail.dozmorov at vcuhealth.org>\n",
      "Dependencies:\n",
      "  - R (>= 4.0.0) type: depends\n",
      "  - data.table, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - HiCcompare, type: imports\n",
      "  - edgeR, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - qqman, type: imports\n",
      "  - pheatmap, type: imports\n",
      "  - methods, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - pbapply, type: imports\n",
      "  - GenomeInfoDbData, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - aggregation, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/multiHiCcompare.html\n",
      "\n",
      "Name: rain\n",
      "Description:\n",
      "This package uses non-parametric methods to detect rhythms in time series. It deals with outliers, missing values and is optimized for time series comprising 10-100 measurements. As it does not assume expect any distinct waveform it is optimal or detecting oscillating behavior (e.g. circadian or cell cycle) in e.g. genome- or proteome-wide biological measurements such as: micro arrays, proteome mass spectrometry, or metabolome measurements.\n",
      "Version: 1.32.0\n",
      "Publication date: None\n",
      "Mantainer: Paul F. Thaben <paul.thaben@charite.de>\n",
      "Authors:\n",
      "Author: Paul F. Thaben, PÃ¥l O. Westermark\n",
      "Dependencies:\n",
      "  - R (>= 2.10) type: depends\n",
      "  - gmp, type: depends\n",
      "  - multtest, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/rain.html\n",
      "\n",
      "Name: DNAshapeR\n",
      "Description:\n",
      "DNAhapeR is an R/BioConductor package for ultra-fast, high-throughput predictions of DNA shape features. The package allows to predict, visualize and encode DNA shape features for statistical learning.\n",
      "Version: 1.26.0\n",
      "Publication date: None\n",
      "Mantainer: Tsu-Pei Chiu <tsupeich@usc.edu>\n",
      "Authors:\n",
      "Author: Tsu-Pei Chiu and Federico Comoglio\n",
      "Dependencies:\n",
      "  - R (>= 3.4) type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - Rcpp (>= 0.12.1) type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - fields, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/DNAshapeR.html\n",
      "\n",
      "Name: MetaVolcanoR\n",
      "Description:\n",
      "MetaVolcanoR combines differential gene expression results. It implements three strategies to summarize differential gene expression from different studies. i) Random Effects Model (REM) approach, ii) a p-value combining-approach, and iii) a vote-counting approach. In all cases, MetaVolcano exploits the Volcano plot reasoning to visualize the gene expression meta-analysis results.\n",
      "Version: 1.12.0\n",
      "Publication date: None\n",
      "Mantainer: Cesar Prada <cesar.prada.medina@gmail.com>\n",
      "Authors:\n",
      "Author: Cesar Prada [aut, cre], Diogenes Lima [aut], Helder Nakaya [aut, ths]\n",
      "Dependencies:\n",
      "  - R (>= 4.1.1) type: depends\n",
      "  - methods, type: imports\n",
      "  - data.table, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - plotly, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - cowplot, type: imports\n",
      "  - parallel, type: imports\n",
      "  - metafor, type: imports\n",
      "  - metap, type: imports\n",
      "  - rlang, type: imports\n",
      "  - topconfects, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - htmlwidgets, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/MetaVolcanoR.html\n",
      "\n",
      "Name: nondetects\n",
      "Description:\n",
      "Methods to model and impute non-detects in the results of qPCR experiments.\n",
      "Version: 2.28.0\n",
      "Publication date: None\n",
      "Mantainer: Valeriia Sherina <valery.sherina@gmail.com>\n",
      "Authors:\n",
      "Author: Matthew N. McCall <mccallm at gmail.com>, Valeriia Sherina <valery.sherina at gmail.com>\n",
      "Dependencies:\n",
      "  - R (>= 3.2) type: depends\n",
      "  - Biobase, type: depends\n",
      "  - limma, type: imports\n",
      "  - mvtnorm, type: imports\n",
      "  - utils, type: imports\n",
      "  - methods, type: imports\n",
      "  - arm, type: imports\n",
      "  - HTqPCR, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/nondetects.html\n",
      "\n",
      "Name: phenoTest\n",
      "Description:\n",
      "Tools to test correlation between gene expression and phenotype in a way that is efficient, structured, fast and scalable. GSEA is also provided.\n",
      "Version: 1.46.0\n",
      "Publication date: None\n",
      "Mantainer: Evarist Planet <evarist.planet@epfl.ch>\n",
      "Authors:\n",
      "Author: Evarist Planet\n",
      "Dependencies:\n",
      "  - R (>= 3.6.0) type: depends\n",
      "  - Biobase, type: depends\n",
      "  - methods, type: depends\n",
      "  - annotate, type: depends\n",
      "  - Heatplus, type: depends\n",
      "  - BMA, type: depends\n",
      "  - ggplot2, type: depends\n",
      "  - Hmisc, type: depends\n",
      "  - survival, type: imports\n",
      "  - limma, type: imports\n",
      "  - gplots, type: imports\n",
      "  - Category, type: imports\n",
      "  - AnnotationDbi, type: imports\n",
      "  - hopach, type: imports\n",
      "  - biomaRt, type: imports\n",
      "  - GSEABase, type: imports\n",
      "  - genefilter, type: imports\n",
      "  - xtable, type: imports\n",
      "  - annotate, type: imports\n",
      "  - mgcv, type: imports\n",
      "  - hgu133a.db, type: imports\n",
      "  - ellipse, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>=2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/phenoTest.html\n",
      "\n",
      "Name: Pigengene\n",
      "Description:\n",
      "Pigengene package provides an efficient way to infer biological signatures from gene expression profiles. The signatures are independent from the underlying platform, e.g., the input can be microarray or RNA Seq data. It can even infer the signatures using data from one platform, and evaluate them on the other. Pigengene identifies the modules (clusters) of highly coexpressed genes using coexpression network analysis, summarizes the biological information of each module in an eigengene, learns a Bayesian network that models the probabilistic dependencies between modules, and builds a decision tree based on the expression of eigengenes.\n",
      "Version: 1.24.0\n",
      "Publication date: None\n",
      "Mantainer: Habil Zare <zare@u.washington.edu>\n",
      "Authors:\n",
      "Author: Habil Zare, Amir Foroushani, Rupesh Agrahari, Meghan Short, Isha Mehta, and Neda Emami\n",
      "Dependencies:\n",
      "  - R (>= 4.0.3) type: depends\n",
      "  - graph, type: depends\n",
      "  - BiocStyle, type: depends\n",
      "  - bnlearn (>= 4.7) type: imports\n",
      "  - C50 (>= 0.1.2) type: imports\n",
      "  - MASS, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - partykit, type: imports\n",
      "  - Rgraphviz, type: imports\n",
      "  - WGCNA, type: imports\n",
      "  - GO.db, type: imports\n",
      "  - impute, type: imports\n",
      "  - preprocessCore, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - parallel, type: imports\n",
      "  - pheatmap (>= 1.0.8) type: imports\n",
      "  - dplyr, type: imports\n",
      "  - gdata, type: imports\n",
      "  - clusterProfiler, type: imports\n",
      "  - ReactomePA, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - openxlsx, type: imports\n",
      "  - DBI, type: imports\n",
      "  - DOSE, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>=2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/Pigengene.html\n",
      "\n",
      "Name: Prostar\n",
      "Description:\n",
      "This package provides a GUI interface for the DAPAR package. The package Prostar (Proteomics statistical analysis with R) is a Bioconductor distributed R package which provides all the necessary functions to analyze quantitative data from label-free proteomics experiments. Contrarily to most other similar R packages, it is endowed with rich and user-friendly graphical interfaces, so that no programming skill is required.\n",
      "Version: 1.30.6\n",
      "Publication date: None\n",
      "Mantainer: Samuel Wieczorek <samuel.wieczorek@cea.fr>\n",
      "Authors:\n",
      "Author: Thomas Burger [aut], Florence Combes [aut], Samuel Wieczorek [cre, aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.2.0) type: depends\n",
      "  - DAPAR, type: imports\n",
      "  - DAPARdata, type: imports\n",
      "  - rhandsontable, type: imports\n",
      "  - data.table, type: imports\n",
      "  - shiny, type: imports\n",
      "  - shinyBS, type: imports\n",
      "  - shinyAce, type: imports\n",
      "  - highcharter, type: imports\n",
      "  - htmlwidgets, type: imports\n",
      "  - webshot, type: imports\n",
      "  - shinythemes, type: imports\n",
      "  - later, type: imports\n",
      "  - shinycssloaders, type: imports\n",
      "  - future, type: imports\n",
      "  - promises, type: imports\n",
      "  - shinyjqui, type: imports\n",
      "  - tibble, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - gplots, type: imports\n",
      "  - shinyjs, type: imports\n",
      "  - vioplot, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/Prostar.html\n",
      "\n",
      "Name: RMassBank\n",
      "Description:\n",
      "Workflow to process tandem MS files and build MassBank records. Functions include automated extraction of tandem MS spectra, formula assignment to tandem MS fragments, recalibration of tandem MS spectra with assigned fragments, spectrum cleanup, automated retrieval of compound information from Internet databases, and export to MassBank records.\n",
      "Version: 3.8.0\n",
      "Publication date: None\n",
      "Mantainer: RMassBank@Eawag <massbank@eawag.ch>\n",
      "Authors:\n",
      "Author: Michael Stravs, Emma Schymanski, Steffen Neumann, Erik Mueller, with contributions from Tobias Schulze\n",
      "Dependencies:\n",
      "  - Rcpp, type: depends\n",
      "  - XML, type: imports\n",
      "  - rjson, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - digest, type: imports\n",
      "  - rcdk, type: imports\n",
      "  - yaml, type: imports\n",
      "  - mzR, type: imports\n",
      "  - methods, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - MSnbase, type: imports\n",
      "  - httr, type: imports\n",
      "  - enviPat, type: imports\n",
      "  - assertthat, type: imports\n",
      "  - logger, type: imports\n",
      "  - RCurl, type: imports\n",
      "  - readJDX, type: imports\n",
      "  - webchem, type: imports\n",
      "  - ChemmineR, type: imports\n",
      "  - ChemmineOB, type: imports\n",
      "  - R.utils, type: imports\n",
      "  - data.table, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/RMassBank.html\n",
      "\n",
      "Name: CAGEfightR\n",
      "Description:\n",
      "CAGE is a widely used high throughput assay for measuring transcription start site (TSS) activity. CAGEfightR is an R/Bioconductor package for performing a wide range of common data analysis tasks for CAGE and 5'-end data in general. Core functionality includes: import of CAGE TSSs (CTSSs), tag (or unidirectional) clustering for TSS identification, bidirectional clustering for enhancer identification, annotation with transcript and gene models, correlation of TSS and enhancer expression, calculation of TSS shapes, quantification of CAGE expression as expression matrices and genome brower visualization.\n",
      "Version: 1.18.0\n",
      "Publication date: None\n",
      "Mantainer: Malte Thodberg <maltethodberg@gmail.com>\n",
      "Authors:\n",
      "Author: Malte Thodberg\n",
      "Dependencies:\n",
      "  - R (>= 3.5) type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - rtracklayer, type: depends\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - pryr (>= 0.1.3) type: imports\n",
      "  - assertthat (>= 0.2.0) type: imports\n",
      "  - methods (>= 3.6.3) type: imports\n",
      "  - Matrix (>= 1.2-12) type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - GenomicFeatures, type: imports\n",
      "  - GenomicAlignments, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - GenomicFiles, type: imports\n",
      "  - Gviz, type: imports\n",
      "  - InteractionSet, type: imports\n",
      "  - GenomicInteractions, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3 + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/CAGEfightR.html\n",
      "\n",
      "Name: CODEX\n",
      "Description:\n",
      "A normalization and copy number variation calling procedure for whole exome DNA sequencing data. CODEX relies on the availability of multiple samples processed using the same sequencing pipeline for normalization, and does not require matched controls. The normalization model in CODEX includes terms that specifically remove biases due to GC content, exon length and targeting and amplification efficiency, and latent systemic artifacts. CODEX also includes a Poisson likelihood-based recursive segmentation procedure that explicitly models the count-based exome sequencing data.\n",
      "Version: 1.30.0\n",
      "Publication date: None\n",
      "Mantainer: Yuchao Jiang <yuchaoj@wharton.upenn.edu>\n",
      "Authors:\n",
      "Author: Yuchao Jiang, Nancy R. Zhang\n",
      "Dependencies:\n",
      "  - R (>= 3.2.3) type: depends\n",
      "  - Rsamtools, type: depends\n",
      "  - GenomeInfoDb, type: depends\n",
      "  - BSgenome.Hsapiens.UCSC.hg19, type: depends\n",
      "  - IRanges, type: depends\n",
      "  - Biostrings, type: depends\n",
      "  - S4Vectors, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/CODEX.html\n",
      "\n",
      "Name: compcodeR\n",
      "Description:\n",
      "This package provides extensive functionality for comparing results obtained by different methods for differential expression analysis of RNAseq data. It also contains functions for simulating count data. Finally, it provides convenient interfaces to several packages for performing the differential expression analysis. These can also be used as templates for setting up and running a user-defined differential analysis workflow within the framework of the package.\n",
      "Version: 1.34.0\n",
      "Publication date: None\n",
      "Mantainer: Charlotte Soneson <charlottesoneson@gmail.com>\n",
      "Authors:\n",
      "Author: Charlotte Soneson [aut, cre] , Paul Bastide [aut] , MÃ©lina Gallopin [aut] (0000-0002-2431-7825 )\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - sm, type: depends\n",
      "  - tcltk, type: imports\n",
      "  - knitr (>= 1.2) type: imports\n",
      "  - markdown, type: imports\n",
      "  - ROCR, type: imports\n",
      "  - lattice (>= 0.16) type: imports\n",
      "  - gplots, type: imports\n",
      "  - gtools, type: imports\n",
      "  - caTools, type: imports\n",
      "  - grid, type: imports\n",
      "  - KernSmooth, type: imports\n",
      "  - MASS, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - stringr, type: imports\n",
      "  - modeest, type: imports\n",
      "  - edgeR, type: imports\n",
      "  - limma, type: imports\n",
      "  - vioplot, type: imports\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - ape, type: imports\n",
      "  - phylolm, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/compcodeR.html\n",
      "\n",
      "Name: FamAgg\n",
      "Description:\n",
      "Framework providing basic pedigree analysis and plotting utilities as well as a variety of methods to evaluate familial aggregation of traits in large pedigrees.\n",
      "Version: 1.26.0\n",
      "Publication date: None\n",
      "Mantainer: Johannes Rainer <johannes.rainer@eurac.edu>\n",
      "Authors:\n",
      "Author: J. Rainer, D. Taliun, C.X. Weichenberger\n",
      "Dependencies:\n",
      "  - methods, type: depends\n",
      "  - kinship2, type: depends\n",
      "  - igraph, type: depends\n",
      "  - gap (>= 1.1-17) type: imports\n",
      "  - Matrix, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - utils, type: imports\n",
      "  - survey, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/FamAgg.html\n",
      "\n",
      "Name: gmapR\n",
      "Description:\n",
      "GSNAP and GMAP are a pair of tools to align short-read data written by Tom Wu. This package provides convenience methods to work with GMAP and GSNAP from within R. In addition, it provides methods to tally alignment results on a per-nucleotide basis using the bam_tally tool.\n",
      "Version: 1.40.0\n",
      "Publication date: None\n",
      "Mantainer: Michael Lawrence <michafla@gene.com>\n",
      "Authors:\n",
      "Author: Cory Barr, Thomas Wu, Michael Lawrence\n",
      "Dependencies:\n",
      "  - R (>= 2.15.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - GenomeInfoDb, type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - Rsamtools, type: depends\n",
      "  - S4Vectors, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - GenomicFeatures, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - VariantAnnotation, type: imports\n",
      "  - tools, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - BSgenome, type: imports\n",
      "  - GenomicAlignments, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/gmapR.html\n",
      "\n",
      "Name: logicFS\n",
      "Description:\n",
      "Identification of interactions between binary variables using Logic Regression. Can, e.g., be used to find interesting SNP interactions. Contains also a bagging version of logic regression for classification.\n",
      "Version: 2.18.0\n",
      "Publication date: None\n",
      "Mantainer: Holger Schwender <holger.schw@gmx.de>\n",
      "Authors:\n",
      "Author: Holger Schwender, Tobias Tietz\n",
      "Dependencies:\n",
      "  - LogicReg, type: depends\n",
      "  - mcbiopi, type: depends\n",
      "  - survival, type: depends\n",
      "  - graphics, type: imports\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/logicFS.html\n",
      "\n",
      "Name: mnem\n",
      "Description:\n",
      "Mixture Nested Effects Models (mnem) is an extension of Nested Effects Models and allows for the analysis of single cell perturbation data provided by methods like Perturb-Seq (Dixit et al., 2016) or Crop-Seq (Datlinger et al., 2017). In those experiments each of many cells is perturbed by a knock-down of a specific gene, i.e. several cells are perturbed by a knock-down of gene A, several by a knock-down of gene B, ... and so forth. The observed read-out has to be multi-trait and in the case of the Perturb-/Crop-Seq gene are expression profiles for each cell. mnem uses a mixture model to simultaneously cluster the cell population into k clusters and and infer k networks causally linking the perturbed genes for each cluster. The mixture components are inferred via an expectation maximization algorithm.\n",
      "Version: 1.14.0\n",
      "Publication date: None\n",
      "Mantainer: Martin Pirkl <martinpirkl@yahoo.de>\n",
      "Authors:\n",
      "Author: Martin Pirkl [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - cluster, type: imports\n",
      "  - graph, type: imports\n",
      "  - Rgraphviz, type: imports\n",
      "  - flexclust, type: imports\n",
      "  - lattice, type: imports\n",
      "  - naturalsort, type: imports\n",
      "  - snowfall, type: imports\n",
      "  - stats4, type: imports\n",
      "  - tsne, type: imports\n",
      "  - methods, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - Linnorm, type: imports\n",
      "  - data.table, type: imports\n",
      "  - Rcpp, type: imports\n",
      "  - RcppEigen, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - e1071, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - wesanderson, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/mnem.html\n",
      "\n",
      "Name: monaLisa\n",
      "Description:\n",
      "Useful functions to work with sequence motifs in the analysis of genomics data. These include methods to annotate genomic regions or sequences with predicted motif hits and to identify motifs that drive observed changes in accessibility or expression. Functions to produce informative visualizations of the obtained results are also provided.\n",
      "Version: 1.4.0\n",
      "Publication date: None\n",
      "Mantainer: Michael Stadler <michael.stadler@fmi.ch>\n",
      "Authors:\n",
      "Author: Dania Machlab [aut] , Lukas Burger [aut], Charlotte Soneson [aut] , Michael Stadler [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - TFBSTools, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - stabs, type: imports\n",
      "  - BSgenome, type: imports\n",
      "  - glmnet, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - grid, type: imports\n",
      "  - circlize, type: imports\n",
      "  - ComplexHeatmap, type: imports\n",
      "  - XVector, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - tools, type: imports\n",
      "  - vioplot, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 3)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/monaLisa.html\n",
      "\n",
      "Name: ngsReports\n",
      "Description:\n",
      "This package provides methods and object classes for parsing FastQC reports and output summaries from other NGS tools into R. As well as parsing files, multiple plotting methods have been implemented for visualising the parsed data. Plots can be generated as static ggplot objects or interactive plotly objects.\n",
      "Version: 2.0.3\n",
      "Publication date: None\n",
      "Mantainer: Stephen Pederson <stephen.pederson.au@gmail.com>\n",
      "Authors:\n",
      "Author: Stephen Pederson [aut, cre], Christopher Ward [aut], Thu-Hien To [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.2.0) type: depends\n",
      "  - BiocGenerics, type: depends\n",
      "  - ggplot2 (>= 3.4.0) type: depends\n",
      "  - patchwork (>= 1.1.1) type: depends\n",
      "  - tibble (>= 1.3.1) type: depends\n",
      "  - Biostrings, type: imports\n",
      "  - checkmate, type: imports\n",
      "  - dplyr (>= 1.0.0) type: imports\n",
      "  - DT, type: imports\n",
      "  - forcats, type: imports\n",
      "  - ggdendro, type: imports\n",
      "  - grDevices (>= 3.6.0) type: imports\n",
      "  - grid, type: imports\n",
      "  - lifecycle, type: imports\n",
      "  - lubridate, type: imports\n",
      "  - methods, type: imports\n",
      "  - pander, type: imports\n",
      "  - plotly (>= 4.9.4) type: imports\n",
      "  - readr, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - rlang, type: imports\n",
      "  - rmarkdown, type: imports\n",
      "  - scales, type: imports\n",
      "  - stats, type: imports\n",
      "  - stringr, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - tidyselect (>= 0.2.3) type: imports\n",
      "  - utils, type: imports\n",
      "  - zoo, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ngsReports.html\n",
      "\n",
      "Name: PREDA\n",
      "Description:\n",
      "Package for the position related analysis of quantitative functional genomics data.\n",
      "Version: 1.44.0\n",
      "Publication date: None\n",
      "Mantainer: Francesco Ferrari <francesco.ferrari@ifom.eu>\n",
      "Authors:\n",
      "Author: Francesco Ferrari <francesco.ferrari at ifom.eu>\n",
      "Dependencies:\n",
      "  - R (>= 2.9.0) type: depends\n",
      "  - Biobase, type: depends\n",
      "  - lokern (>= 1.0.9) type: depends\n",
      "  - multtest, type: depends\n",
      "  - stats, type: depends\n",
      "  - methods, type: depends\n",
      "  - annotate, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/PREDA.html\n",
      "\n",
      "Name: ACME\n",
      "Description:\n",
      "ACME (Algorithms for Calculating Microarray Enrichment) is a set of tools for analysing tiling array ChIP/chip, DNAse hypersensitivity, or other experiments that result in regions of the genome showing \"enrichment\". It does not rely on a specific array technology (although the array should be a \"tiling\" array), is very general (can be applied in experiments resulting in regions of enrichment), and is very insensitive to array noise or normalization methods. It is also very fast and can be applied on whole-genome tiling array experiments quite easily with enough memory.\n",
      "Version: 2.54.0\n",
      "Publication date: None\n",
      "Mantainer: Sean Davis <sdavis2@mail.nih.gov>\n",
      "Authors:\n",
      "Author: Sean Davis <sdavis2 at mail.nih.gov>\n",
      "Dependencies:\n",
      "  - R (>= 2.10) type: depends\n",
      "  - Biobase, type: depends\n",
      "  - methods, type: depends\n",
      "  - BiocGenerics, type: depends\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ACME.html\n",
      "\n",
      "Name: biosigner\n",
      "Description:\n",
      "Feature selection is critical in omics data analysis to extract restricted and meaningful molecular signatures from complex and high-dimension data, and to build robust classifiers. This package implements a new method to assess the relevance of the variables for the prediction performances of the classifier. The approach can be run in parallel with the PLS-DA, Random Forest, and SVM binary classifiers. The signatures and the corresponding 'restricted' models are returned, enabling future predictions on new datasets. A Galaxy implementation of the package is available within the Workflow4metabolomics.org online infrastructure for computational metabolomics.\n",
      "Version: 1.26.0\n",
      "Publication date: None\n",
      "Mantainer: Etienne A. Thevenot <etienne.thevenot@cea.fr>\n",
      "Authors:\n",
      "Author: Philippe Rinaudo [aut], Etienne A. Thevenot [aut, cre]\n",
      "Dependencies:\n",
      "  - Biobase, type: imports\n",
      "  - methods, type: imports\n",
      "  - e1071, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - MultiAssayExperiment, type: imports\n",
      "  - MultiDataSet, type: imports\n",
      "  - randomForest, type: imports\n",
      "  - ropls, type: imports\n",
      "  - stats, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "CeCILL\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/biosigner.html\n",
      "\n",
      "Name: CytoTree\n",
      "Description:\n",
      "Bioconductor version: Release (3.16)\n",
      "Version: 1.8.0\n",
      "Publication date: None\n",
      "Mantainer: Author: Yuting Dai [aut, cre]\n",
      "Authors:\n",
      "A trajectory inference toolkit for flow and mass cytometry data. CytoTree is a valuable tool to build a tree-shaped trajectory using flow and mass cytometry data. The application of CytoTree ranges from clustering and dimensionality reduction to trajectory reconstruction and pseudotime estimation. It offers complete analyzing workflow for flow and mass cytometry data.\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - igraph, type: depends\n",
      "  - FlowSOM, type: imports\n",
      "  - Rtsne, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - destiny, type: imports\n",
      "  - gmodels, type: imports\n",
      "  - flowUtils, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - flowCore, type: imports\n",
      "  - sva, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - methods, type: imports\n",
      "  - mclust, type: imports\n",
      "  - prettydoc, type: imports\n",
      "  - RANN (>= 2.5) type: imports\n",
      "  - Rcpp (>= 0.12.0) type: imports\n",
      "  - BiocNeighbors, type: imports\n",
      "  - cluster, type: imports\n",
      "  - pheatmap, type: imports\n",
      "  - scatterpie, type: imports\n",
      "  - umap, type: imports\n",
      "  - scatterplot3d, type: imports\n",
      "  - limma, type: imports\n",
      "  - stringr, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - grid, type: imports\n",
      "  - stats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/CytoTree.html\n",
      "\n",
      "Name: diffcoexp\n",
      "Description:\n",
      "A tool for the identification of differentially coexpressed links (DCLs) and differentially coexpressed genes (DCGs). DCLs are gene pairs with significantly different correlation coefficients under two conditions. DCGs are genes with significantly more DCLs than by chance.\n",
      "Version: 1.18.0\n",
      "Publication date: None\n",
      "Mantainer: Wenbin Wei <wenbin.wei2@durham.ac.uk>\n",
      "Authors:\n",
      "Author: Wenbin Wei, Sandeep Amberkar, Winston Hide\n",
      "Dependencies:\n",
      "  - R (>= 3.5) type: depends\n",
      "  - WGCNA, type: depends\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - stats, type: imports\n",
      "  - DiffCorr, type: imports\n",
      "  - psych, type: imports\n",
      "  - igraph, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/diffcoexp.html\n",
      "\n",
      "Name: DMRcaller\n",
      "Description:\n",
      "Uses Bisulfite sequencing data in two conditions and identifies differentially methylated regions between the conditions in CG and non-CG context. The input is the CX report files produced by Bismark and the output is a list of DMRs stored as GRanges objects.\n",
      "Version: 1.30.0\n",
      "Publication date: None\n",
      "Mantainer: Nicolae Radu Zabet <n.r.zabet@gen.cam.ac.uk>\n",
      "Authors:\n",
      "Author: Nicolae Radu Zabet <n.r.zabet at gen.cam.ac.uk>, Jonathan Michael Foonlan Tsang <jmft2 at cam.ac.uk>, Alessandro Pio Greco <apgrec at essex.ac.uk> and Ryan Merritt <rmerri at essex.ac.uk>\n",
      "Dependencies:\n",
      "  - R (>= 3.5) type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - IRanges, type: depends\n",
      "  - S4Vectors, type: depends\n",
      "  - parallel, type: imports\n",
      "  - Rcpp, type: imports\n",
      "  - RcppRoll, type: imports\n",
      "  - betareg, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/DMRcaller.html\n",
      "\n",
      "Name: erma\n",
      "Description:\n",
      "Software and data to support epigenomic road map adventures.\n",
      "Version: 1.14.0\n",
      "Publication date: None\n",
      "Mantainer: VJ Carey <stvjc@channing.harvard.edu>\n",
      "Authors:\n",
      "Author: VJ Carey <stvjc at channing.harvard.edu>\n",
      "Dependencies:\n",
      "  - R (>= 3.1) type: depends\n",
      "  - methods, type: depends\n",
      "  - Homo.sapiens, type: depends\n",
      "  - GenomicFiles, type: depends\n",
      "  - rtracklayer, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - shiny, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - AnnotationDbi, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/erma.html\n",
      "\n",
      "Name: flowCut\n",
      "Description:\n",
      "Common techinical complications such as clogging can result in spurious events and fluorescence intensity shifting, flowCut is designed to detect and remove technical artifacts from your data by removing segments that show statistical differences from other segments.\n",
      "Version: 1.8.0\n",
      "Publication date: None\n",
      "Mantainer: Justin Meskas <justinmeskas@gmail.com>\n",
      "Authors:\n",
      "Author: Justin Meskas [cre, aut], Sherrie Wang [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.4) type: depends\n",
      "  - flowCore, type: depends\n",
      "  - flowDensity, type: imports\n",
      "  - Cairo, type: imports\n",
      "  - e1071, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - methods, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/flowCut.html\n",
      "\n",
      "Name: flowFP\n",
      "Description:\n",
      "Fingerprint generation of flow cytometry data, used to facilitate the application of machine learning and datamining tools for flow cytometry.\n",
      "Version: 1.56.3\n",
      "Publication date: None\n",
      "Mantainer: Herb Holyst <holyst@pennmedicine.upenn.edu>, Wade Rogers <wade.rogers@spcytomics.com>\n",
      "Authors:\n",
      "Author: Herb Holyst <holyst at pennmedicine.upenn.edu>, Wade Rogers <wade.rogers at spcytomics.com>\n",
      "Dependencies:\n",
      "  - R (>= 2.10) type: depends\n",
      "  - flowCore, type: depends\n",
      "  - flowViz, type: depends\n",
      "  - Biobase, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - stats4, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/flowFP.html\n",
      "\n",
      "Name: HilbertCurve\n",
      "Description:\n",
      "Hilbert curve is a type of space-filling curves that fold one dimensional axis into a two dimensional space, but with still preserves the locality. This package aims to provide an easy and flexible way to visualize data through Hilbert curve.\n",
      "Version: 1.28.0\n",
      "Publication date: None\n",
      "Mantainer: Zuguang Gu <z.gu@dkfz.de>\n",
      "Authors:\n",
      "Author: Zuguang Gu [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 3.6.0) type: depends\n",
      "  - grid, type: depends\n",
      "  - methods, type: imports\n",
      "  - utils, type: imports\n",
      "  - HilbertVis, type: imports\n",
      "  - png, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - circlize (>= 0.3.3) type: imports\n",
      "  - IRanges, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - polylabelr, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/HilbertCurve.html\n",
      "\n",
      "Name: SCAN.UPC\n",
      "Description:\n",
      "SCAN is a microarray normalization method to facilitate personalized-medicine workflows. Rather than processing microarray samples as groups, which can introduce biases and present logistical challenges, SCAN normalizes each sample individually by modeling and removing probe- and array-specific background noise using only data from within each array. SCAN can be applied to one-channel (e.g., Affymetrix) or two-channel (e.g., Agilent) microarrays. The Universal exPression Codes (UPC) method is an extension of SCAN that estimates whether a given gene/transcript is active above background levels in a given sample. The UPC method can be applied to one-channel or two-channel microarrays as well as to RNA-Seq read counts. Because UPC values are represented on the same scale and have an identical interpretation for each platform, they can be used for cross-platform data integration.\n",
      "Version: 2.40.0\n",
      "Publication date: None\n",
      "Mantainer: Stephen R. Piccolo <stephen_piccolo@byu.edu>\n",
      "Authors:\n",
      "Author: Stephen R. Piccolo and Andrea H. Bild and W. Evan Johnson\n",
      "Dependencies:\n",
      "  - R (>= 2.14.0) type: depends\n",
      "  - Biobase, type: depends\n",
      "  - oligo, type: depends\n",
      "  - Biostrings, type: depends\n",
      "  - GEOquery, type: depends\n",
      "  - affy, type: depends\n",
      "  - affyio, type: depends\n",
      "  - foreach, type: depends\n",
      "  - sva, type: depends\n",
      "  - utils, type: imports\n",
      "  - methods, type: imports\n",
      "  - MASS, type: imports\n",
      "  - tools, type: imports\n",
      "  - IRanges, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/SCAN.UPC.html\n",
      "\n",
      "Name: SpatialCPie\n",
      "Description:\n",
      "SpatialCPie is an R package designed to facilitate cluster evaluation for spatial transcriptomics data by providing intuitive visualizations that display the relationships between clusters in order to guide the user during cluster identification and other downstream applications. The package is built around a shiny \"gadget\" to allow the exploration of the data with multiple plots in parallel and an interactive UI. The user can easily toggle between different cluster resolutions in order to choose the most appropriate visual cues.\n",
      "Version: 1.14.0\n",
      "Publication date: None\n",
      "Mantainer: Joseph Bergenstraahle <joseph.bergenstrahle@gmail.com>\n",
      "Authors:\n",
      "Author: Joseph Bergenstraahle [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 3.6) type: depends\n",
      "  - colorspace (>= 1.3-2) type: imports\n",
      "  - data.table (>= 1.12.2) type: imports\n",
      "  - digest (>= 0.6.21) type: imports\n",
      "  - dplyr (>= 0.7.6) type: imports\n",
      "  - ggforce (>= 0.3.0) type: imports\n",
      "  - ggiraph (>= 0.5.0) type: imports\n",
      "  - ggplot2 (>= 3.0.0) type: imports\n",
      "  - ggrepel (>= 0.8.0) type: imports\n",
      "  - grid (>= 3.5.1) type: imports\n",
      "  - igraph (>= 1.2.2) type: imports\n",
      "  - lpSolve (>= 5.6.13) type: imports\n",
      "  - methods (>= 3.5.0) type: imports\n",
      "  - purrr (>= 0.2.5) type: imports\n",
      "  - readr (>= 1.1.1) type: imports\n",
      "  - rlang (>= 0.2.2) type: imports\n",
      "  - shiny (>= 1.1.0) type: imports\n",
      "  - shinycssloaders (>= 0.2.0) type: imports\n",
      "  - shinyjs (>= 1.0) type: imports\n",
      "  - shinyWidgets (>= 0.4.8) type: imports\n",
      "  - stats (>= 3.6.0) type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - tibble (>= 1.4.2) type: imports\n",
      "  - tidyr (>= 0.8.1) type: imports\n",
      "  - tidyselect (>= 0.2.4) type: imports\n",
      "  - tools (>= 3.6.0) type: imports\n",
      "  - utils (>= 3.5.0) type: imports\n",
      "  - zeallot (>= 0.1.0) type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/SpatialCPie.html\n",
      "\n",
      "Name: tilingArray\n",
      "Description:\n",
      "The package provides functionality that can be useful for the analysis of high-density tiling microarray data (such as from Affymetrix genechips) for measuring transcript abundance and architecture. The main functionalities of the package are: 1. the class 'segmentation' for representing partitionings of a linear series of data; 2. the function 'segment' for fitting piecewise constant models using a dynamic programming algorithm that is both fast and exact; 3. the function 'confint' for calculating confidence intervals using the strucchange package; 4. the function 'plotAlongChrom' for generating pretty plots; 5. the function 'normalizeByReference' for probe-sequence dependent response adjustment from a (set of) reference hybridizations.\n",
      "Version: 1.76.0\n",
      "Publication date: None\n",
      "Mantainer: Zhenyu Xu <zxu@embl.de>\n",
      "Authors:\n",
      "Author: Wolfgang Huber, Zhenyu Xu, Joern Toedling with contributions from Matt Ritchie\n",
      "Dependencies:\n",
      "  - R (>= 2.11.0) type: depends\n",
      "  - Biobase, type: depends\n",
      "  - methods, type: depends\n",
      "  - pixmap, type: depends\n",
      "  - strucchange, type: imports\n",
      "  - affy, type: imports\n",
      "  - vsn, type: imports\n",
      "  - genefilter, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - grid, type: imports\n",
      "  - stats4, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/tilingArray.html\n",
      "\n",
      "Name: TRONCO\n",
      "Description:\n",
      "The TRONCO (TRanslational ONCOlogy) R package collects algorithms to infer progression models via the approach of Suppes-Bayes Causal Network, both from an ensemble of tumors (cross-sectional samples) and within an individual patient (multi-region or single-cell samples). The package provides parallel implementation of algorithms that process binary matrices where each row represents a tumor sample and each column a single-nucleotide or a structural variant driving the progression; a 0/1 value models the absence/presence of that alteration in the sample. The tool can import data from plain, MAF or GISTIC format files, and can fetch it from the cBioPortal for cancer genomics. Functions for data manipulation and visualization are provided, as well as functions to import/export such data to other bioinformatics tools for, e.g, clustering or detection of mutually exclusive alterations. Inferred models can be visualized and tested for their confidence via bootstrap and cross-validation. TRONCO is used for the implementation of the Pipeline for Cancer Inference (PICNIC).\n",
      "Version: 2.30.0\n",
      "Publication date: None\n",
      "Mantainer: Luca De Sano <luca.desano@gmail.com>\n",
      "Authors:\n",
      "Author: Marco Antoniotti [ctb], Giulio Caravagna [aut, cre], Luca De Sano [aut] , Alex Graudenzi [aut], Giancarlo Mauri [ctb], Bud Mishra [ctb], Daniele Ramazzotti [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.1.0) type: depends\n",
      "  - bnlearn, type: imports\n",
      "  - Rgraphviz, type: imports\n",
      "  - gtools, type: imports\n",
      "  - parallel, type: imports\n",
      "  - foreach, type: imports\n",
      "  - doParallel, type: imports\n",
      "  - iterators, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - circlize, type: imports\n",
      "  - igraph, type: imports\n",
      "  - grid, type: imports\n",
      "  - gridExtra, type: imports\n",
      "  - xtable, type: imports\n",
      "  - gtable, type: imports\n",
      "  - scales, type: imports\n",
      "  - R.matlab, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - methods, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/TRONCO.html\n",
      "\n",
      "Name: ACE\n",
      "Description:\n",
      "Uses segmented copy number data to estimate tumor cell percentage and produce copy number plots displaying absolute copy numbers.\n",
      "Version: 1.16.0\n",
      "Publication date: None\n",
      "Mantainer: Jos B Poell <j.poell@amsterdamumc.nl>\n",
      "Authors:\n",
      "Author: Jos B Poell\n",
      "Dependencies:\n",
      "  - R (>= 3.4) type: depends\n",
      "  - Biobase, type: imports\n",
      "  - QDNAseq, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - grid, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - methods, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ACE.html\n",
      "\n",
      "Name: ASSIGN\n",
      "Description:\n",
      "ASSIGN is a computational tool to evaluate the pathway deregulation/activation status in individual patient samples. ASSIGN employs a flexible Bayesian factor analysis approach that adapts predetermined pathway signatures derived either from knowledge-based literature or from perturbation experiments to the cell-/tissue-specific pathway signatures. The deregulation/activation level of each context-specific pathway is quantified to a score, which represents the extent to which a patient sample encompasses the pathway deregulation/activation signature.\n",
      "Version: 1.34.0\n",
      "Publication date: None\n",
      "Mantainer: Ying Shen <yshen3@bu.edu>, W. Evan Johnson <wej@bu.edu>, David Jenkins <dfj@bu.edu>, Mumtehena Rahman <moom.rahman@utah.edu>\n",
      "Authors:\n",
      "Author: Ying Shen, Andrea H. Bild, W. Evan Johnson, and Mumtehena Rahman\n",
      "Dependencies:\n",
      "  - R (>= 3.4) type: depends\n",
      "  - gplots, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - msm, type: imports\n",
      "  - Rlab, type: imports\n",
      "  - stats, type: imports\n",
      "  - sva, type: imports\n",
      "  - utils, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - yaml, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ASSIGN.html\n",
      "\n",
      "Name: ccmap\n",
      "Description:\n",
      "Finds drugs and drug combinations that are predicted to reverse or mimic gene expression signatures. These drugs might reverse diseases or mimic healthy lifestyles.\n",
      "Version: 1.24.0\n",
      "Publication date: None\n",
      "Mantainer: Alex Pickering <alexvpickering@gmail.com>\n",
      "Authors:\n",
      "Author: Alex Pickering\n",
      "Dependencies:\n",
      "  - AnnotationDbi, type: imports\n",
      "  - BiocManager (>= 1.30.4) type: imports\n",
      "  - ccdata, type: imports\n",
      "  - doParallel (>= 1.0.10) type: imports\n",
      "  - data.table (>= 1.10.4) type: imports\n",
      "  - foreach (>= 1.4.3) type: imports\n",
      "  - parallel (>= 3.3.3) type: imports\n",
      "  - xgboost (>= 0.6.4) type: imports\n",
      "  - lsa (>= 0.73.1) type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ccmap.html\n",
      "\n",
      "Name: celaref\n",
      "Description:\n",
      "After the clustering step of a single-cell RNAseq experiment, this package aims to suggest labels/cell types for the clusters, on the basis of similarity to a reference dataset. It requires a table of read counts per cell per gene, and a list of the cells belonging to each of the clusters, (for both test and reference data).\n",
      "Version: 1.16.0\n",
      "Publication date: None\n",
      "Mantainer: Sarah Williams <sarah.williams1@monash.edu>\n",
      "Authors:\n",
      "Author: Sarah Williams [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - MAST, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - rlang, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - readr, type: imports\n",
      "  - tibble, type: imports\n",
      "  - DelayedArray, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/celaref.html\n",
      "\n",
      "Name: countsimQC\n",
      "Description:\n",
      "countsimQC provides functionality to create a comprehensive report comparing a broad range of characteristics across a collection of count matrices. One important use case is the comparison of one or more synthetic count matrices to a real count matrix, possibly the one underlying the simulations. However, any collection of count matrices can be compared.\n",
      "Version: 1.16.1\n",
      "Publication date: None\n",
      "Mantainer: Charlotte Soneson <charlottesoneson@gmail.com>\n",
      "Authors:\n",
      "Author: Charlotte Soneson [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 3.5) type: depends\n",
      "  - rmarkdown (>= 2.5) type: imports\n",
      "  - edgeR, type: imports\n",
      "  - DESeq2, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - tools, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - genefilter, type: imports\n",
      "  - DT, type: imports\n",
      "  - GenomeInfoDbData, type: imports\n",
      "  - caTools, type: imports\n",
      "  - randtests, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - methods, type: imports\n",
      "  - ragg, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>=2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/countsimQC.html\n",
      "\n",
      "Name: EmpiricalBrownsMethod\n",
      "Description:\n",
      "Combining P-values from multiple statistical tests is common in bioinformatics. However, this procedure is non-trivial for dependent P-values. This package implements an empirical adaptation of Brownâs Method (an extension of Fisherâs Method) for combining dependent P-values which is appropriate for highly correlated data sets found in high-throughput biological experiments.\n",
      "Version: 1.26.0\n",
      "Publication date: None\n",
      "Mantainer: David Gibbs <dgibbs@systemsbiology.org>\n",
      "Authors:\n",
      "Author: William Poole\n",
      "Dependencies:\n",
      "  - R (>= 3.2.0) type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/EmpiricalBrownsMethod.html\n",
      "\n",
      "Name: GSEAlm\n",
      "Description:\n",
      "Models and methods for fitting linear models to gene expression data, together with tools for computing and using various regression diagnostics.\n",
      "Version: 1.58.0\n",
      "Publication date: None\n",
      "Mantainer: Assaf Oron <assaf@uw.edu>\n",
      "Authors:\n",
      "Author: Assaf Oron, Robert Gentleman (with contributions from S. Falcon and Z. Jiang)\n",
      "Dependencies:\n",
      "  - Biobase, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/GSEAlm.html\n",
      "\n",
      "Name: lisaClust\n",
      "Description:\n",
      "lisaClust provides a series of functions to identify and visualise regions of tissue where spatial associations between cell-types is similar. This package can be used to provide a high-level summary of cell-type colocalization in multiplexed imaging data that has been segmented at a single-cell resolution.\n",
      "Version: 1.6.3\n",
      "Publication date: None\n",
      "Mantainer: Ellis Patrick <ellis.patrick@sydney.edu.au>\n",
      "Authors:\n",
      "Author: Ellis Patrick [aut, cre], Nicolas Canete [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - ggplot2, type: imports\n",
      "  - class, type: imports\n",
      "  - concaveman, type: imports\n",
      "  - grid, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - spatstat.explore, type: imports\n",
      "  - spatstat.geom, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - methods, type: imports\n",
      "  - spicyR, type: imports\n",
      "  - purrr, type: imports\n",
      "  - stats, type: imports\n",
      "  - data.table, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - SingleCellExperiment, type: imports\n",
      "  - SpatialExperiment, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - pheatmap, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>=2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/lisaClust.html\n",
      "\n",
      "Name: PAIRADISE\n",
      "Description:\n",
      "This package implements the PAIRADISE procedure for detecting differential isoform expression between matched replicates in paired RNA-Seq data.\n",
      "Version: 1.14.0\n",
      "Publication date: None\n",
      "Mantainer: Qiang Hu <Qiang.Hu@roswellpark.org>, Levon Demirdjian <levondem@ucla.edu>\n",
      "Authors:\n",
      "Author: Levon Demirdjian, Ying Nian Wu, Yi Xing\n",
      "Dependencies:\n",
      "  - R (>= 3.6) type: depends\n",
      "  - nloptr, type: depends\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - stats, type: imports\n",
      "  - methods, type: imports\n",
      "  - abind, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/PAIRADISE.html\n",
      "\n",
      "Name: phantasus\n",
      "Description:\n",
      "Phantasus is a web-application for visual and interactive gene expression analysis. Phantasus is based on Morpheus â a web-based software for heatmap visualisation and analysis, which was integrated with an R environment via OpenCPU API. Aside from basic visualization and filtering methods, R-based methods such as k-means clustering, principal component analysis or differential expression analysis with limma package are supported.\n",
      "Version: 1.18.4\n",
      "Publication date: None\n",
      "Mantainer: Alexey Sergushichev <alsergbox@gmail.com>\n",
      "Authors:\n",
      "Author: Daria Zenkova [aut], Vladislav Kamenev [aut], Rita Sablina [ctb], Maxim Kleverov [ctb], Maxim Artyomov [aut], Alexey Sergushichev [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 3.5) type: depends\n",
      "  - ggplot2, type: imports\n",
      "  - protolite, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - GEOquery, type: imports\n",
      "  - Rook, type: imports\n",
      "  - htmltools, type: imports\n",
      "  - httpuv, type: imports\n",
      "  - jsonlite, type: imports\n",
      "  - limma, type: imports\n",
      "  - opencpu, type: imports\n",
      "  - assertthat, type: imports\n",
      "  - methods, type: imports\n",
      "  - httr, type: imports\n",
      "  - rhdf5, type: imports\n",
      "  - utils, type: imports\n",
      "  - parallel, type: imports\n",
      "  - stringr, type: imports\n",
      "  - fgsea, type: imports\n",
      "  - svglite, type: imports\n",
      "  - gtable, type: imports\n",
      "  - stats, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - pheatmap, type: imports\n",
      "  - scales, type: imports\n",
      "  - ccaPP, type: imports\n",
      "  - grid, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - AnnotationDbi, type: imports\n",
      "  - DESeq2, type: imports\n",
      "  - data.table, type: imports\n",
      "  - curl, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/phantasus.html\n",
      "\n",
      "Name: rCGH\n",
      "Description:\n",
      "A comprehensive pipeline for analyzing and interactively visualizing genomic profiles generated through commercial or custom aCGH arrays. As inputs, rCGH supports Agilent dual-color Feature Extraction files (.txt), from 44 to 400K, Affymetrix SNP6.0 and cytoScanHD probeset.txt, cychp.txt, and cnchp.txt files exported from ChAS or Affymetrix Power Tools. rCGH also supports custom arrays, provided data complies with the expected format. This package takes over all the steps required for individual genomic profiles analysis, from reading files to profiles segmentation and gene annotations. This package also provides several visualization functions (static or interactive) which facilitate individual profiles interpretation. Input files can be in compressed format, e.g. .bz2 or .gz.\n",
      "Version: 1.28.0\n",
      "Publication date: None\n",
      "Mantainer: Frederic Commo <fredcommo@gmail.com>\n",
      "Authors:\n",
      "Author: Frederic Commo [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 3.4) type: depends\n",
      "  - methods, type: depends\n",
      "  - stats, type: depends\n",
      "  - utils, type: depends\n",
      "  - graphics, type: depends\n",
      "  - plyr, type: imports\n",
      "  - DNAcopy, type: imports\n",
      "  - lattice, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - grid, type: imports\n",
      "  - shiny (>= 0.11.1) type: imports\n",
      "  - limma, type: imports\n",
      "  - affy, type: imports\n",
      "  - mclust, type: imports\n",
      "  - TxDb.Hsapiens.UCSC.hg18.knownGene, type: imports\n",
      "  - TxDb.Hsapiens.UCSC.hg19.knownGene, type: imports\n",
      "  - TxDb.Hsapiens.UCSC.hg38.knownGene, type: imports\n",
      "  - org.Hs.eg.db, type: imports\n",
      "  - GenomicFeatures, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - AnnotationDbi, type: imports\n",
      "  - parallel, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - aCGH, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/rCGH.html\n",
      "\n",
      "Name: RefPlus\n",
      "Description:\n",
      "The package contains functions for pre-processing Affymetrix data using the RMA+ and the RMA++ methods.\n",
      "Version: 1.68.0\n",
      "Publication date: None\n",
      "Mantainer: Kai-Ming Chang <kaiming@gmail.com>\n",
      "Authors:\n",
      "Author: Kai-Ming Chang <kaiming at gmail.com>, Chris Harbron <Chris.Harbron at astrazeneca.com>, Marie C South <Marie.C.South at astrazeneca.com>\n",
      "Dependencies:\n",
      "  - R (>= 2.8.0) type: depends\n",
      "  - Biobase, type: depends\n",
      "  - affy, type: depends\n",
      "  - affyPLM, type: depends\n",
      "  - preprocessCore, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/RefPlus.html\n",
      "\n",
      "Name: SPONGE\n",
      "Description:\n",
      "This package provides methods to efficiently detect competitive endogeneous RNA interactions between two genes. Such interactions are mediated by one or several miRNAs such that both gene and miRNA expression data for a larger number of samples is needed as input. The SPONGE package now also includes spongEffects: ceRNA modules offer patient-specific insights into the miRNA regulatory landscape.\n",
      "Version: 1.20.0\n",
      "Publication date: None\n",
      "Mantainer: Markus List <markus.list@tum.de>\n",
      "Authors:\n",
      "Author: Markus List [aut, cre] , Markus Hoffmann [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.6) type: depends\n",
      "  - Biobase, type: imports\n",
      "  - stats, type: imports\n",
      "  - ppcor, type: imports\n",
      "  - logging, type: imports\n",
      "  - foreach, type: imports\n",
      "  - doRNG, type: imports\n",
      "  - data.table, type: imports\n",
      "  - MASS, type: imports\n",
      "  - expm, type: imports\n",
      "  - gRbase, type: imports\n",
      "  - glmnet, type: imports\n",
      "  - igraph, type: imports\n",
      "  - iterators, type: imports\n",
      "  - tidyverse, type: imports\n",
      "  - caret, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - biomaRt, type: imports\n",
      "  - randomForest, type: imports\n",
      "  - ggridges, type: imports\n",
      "  - cvms, type: imports\n",
      "  - miRBaseConverter, type: imports\n",
      "  - ComplexHeatmap, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - MetBrewer, type: imports\n",
      "  - rlang, type: imports\n",
      "  - tnet, type: imports\n",
      "  - ggpubr, type: imports\n",
      "  - stringr, type: imports\n",
      "  - tidyr, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>=3)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/SPONGE.html\n",
      "\n",
      "Name: BEclear\n",
      "Description:\n",
      "Provides functions to detect and correct for batch effects in DNA methylation data. The core function is based on latent factor models and can also be used to predict missing values in any other matrix containing real numbers.\n",
      "Version: 2.14.0\n",
      "Publication date: None\n",
      "Mantainer: David Rasp <david.j.rasp@gmail.com>\n",
      "Authors:\n",
      "Author: David Rasp [aut, cre] , Markus Merl [aut], Ruslan Akulenko [aut]\n",
      "Dependencies:\n",
      "  - BiocParallel, type: depends\n",
      "  - futile.logger, type: imports\n",
      "  - Rdpack, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - data.table (>= 1.11.8) type: imports\n",
      "  - Rcpp, type: imports\n",
      "  - abind, type: imports\n",
      "  - stats, type: imports\n",
      "  - graphics, type: imports\n",
      "  - utils, type: imports\n",
      "  - methods, type: imports\n",
      "  - dixonTest, type: imports\n",
      "  - ids, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/BEclear.html\n",
      "\n",
      "Name: BiSeq\n",
      "Description:\n",
      "The BiSeq package provides useful classes and functions to handle and analyze targeted bisulfite sequencing (BS) data such as reduced-representation bisulfite sequencing (RRBS) data. In particular, it implements an algorithm to detect differentially methylated regions (DMRs). The package takes already aligned BS data from one or multiple samples.\n",
      "Version: 1.38.0\n",
      "Publication date: None\n",
      "Mantainer: Katja Hebestreit <katja.hebestreit@gmail.com>\n",
      "Authors:\n",
      "Author: Katja Hebestreit, Hans-Ulrich Klein\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - S4Vectors, type: depends\n",
      "  - IRanges, type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - Formula, type: depends\n",
      "  - methods, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - parallel, type: imports\n",
      "  - betareg, type: imports\n",
      "  - lokern, type: imports\n",
      "  - Formula, type: imports\n",
      "  - globaltest, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/BiSeq.html\n",
      "\n",
      "Name: cliqueMS\n",
      "Description:\n",
      "Annotates data from liquid chromatography coupled to mass spectrometry (LC/MS) metabolomics experiments. Based on a network algorithm (O.Senan, A. Aguilar- Mogas, M. Navarro, O. Yanes, R.GuimerÃ  and M. Sales-Pardo, Bioinformatics, 35(20), 2019), 'CliqueMS' builds a weighted similarity network where nodes are features and edges are weighted according to the similarity of this features. Then it searches for the most plausible division of the similarity network into cliques (fully connected components). Finally it annotates metabolites within each clique, obtaining for each annotated metabolite the neutral mass and their features, corresponding to isotopes, ionization adducts and fragmentation adducts of that metabolite.\n",
      "Version: 1.12.2\n",
      "Publication date: None\n",
      "Mantainer: Oriol Senan Campos <oriol.senan@praenoscere.com>\n",
      "Authors:\n",
      "Author: Oriol Senan Campos [aut, cre], Antoni Aguilar-Mogas [aut], Jordi Capellades [aut], Miriam Navarro [aut], Oscar Yanes [aut], Roger Guimera [aut], Marta Sales-Pardo [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.6.0) type: depends\n",
      "  - Rcpp (>= 0.12.15) type: imports\n",
      "  - xcms, type: imports\n",
      "  - MSnbase, type: imports\n",
      "  - igraph, type: imports\n",
      "  - qlcMatrix, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - methods, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/cliqueMS.html\n",
      "\n",
      "Name: DaMiRseq\n",
      "Description:\n",
      "The DaMiRseq package offers a tidy pipeline of data mining procedures to identify transcriptional biomarkers and exploit them for both binary and multi-class classification purposes. The package accepts any kind of data presented as a table of raw counts and allows including both continous and factorial variables that occur with the experimental setting. A series of functions enable the user to clean up the data by filtering genomic features and samples, to adjust data by identifying and removing the unwanted source of variation (i.e. batches and confounding factors) and to select the best predictors for modeling. Finally, a \"stacking\" ensemble learning technique is applied to build a robust classification model. Every step includes a checkpoint that the user may exploit to assess the effects of data management by looking at diagnostic plots, such as clustering and heatmaps, RLE boxplots, MDS or correlation plot.\n",
      "Version: 2.10.0\n",
      "Publication date: None\n",
      "Mantainer: Mattia Chiesa <mattia.chiesa@cardiologicomonzino.it>\n",
      "Authors:\n",
      "Author: Mattia Chiesa <mattia.chiesa at cardiologicomonzino.it>, Luca Piacentini <luca.piacentini at cardiologicomonzino.it>\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - ggplot2, type: depends\n",
      "  - DESeq2, type: imports\n",
      "  - limma, type: imports\n",
      "  - EDASeq, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - sva, type: imports\n",
      "  - Hmisc, type: imports\n",
      "  - pheatmap, type: imports\n",
      "  - FactoMineR, type: imports\n",
      "  - corrplot, type: imports\n",
      "  - randomForest, type: imports\n",
      "  - e1071, type: imports\n",
      "  - caret, type: imports\n",
      "  - MASS, type: imports\n",
      "  - lubridate, type: imports\n",
      "  - plsVarSel, type: imports\n",
      "  - kknn, type: imports\n",
      "  - FSelector, type: imports\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - ineq, type: imports\n",
      "  - arm, type: imports\n",
      "  - pls, type: imports\n",
      "  - RSNNS, type: imports\n",
      "  - edgeR, type: imports\n",
      "  - plyr, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/DaMiRseq.html\n",
      "\n",
      "Name: epivizrServer\n",
      "Description:\n",
      "This package provides objects to manage WebSocket connections to epiviz apps. Other epivizr package use this infrastructure.\n",
      "Version: 1.26.0\n",
      "Publication date: None\n",
      "Mantainer: Hector Corrada Bravo <hcorrada@gmail.com>\n",
      "Authors:\n",
      "Author: Hector Corrada Bravo [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 3.2.3) type: depends\n",
      "  - methods, type: depends\n",
      "  - httpuv (>= 1.3.0) type: imports\n",
      "  - R6 (>= 2.0.0) type: imports\n",
      "  - rjson, type: imports\n",
      "  - mime (>= 0.2) type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/epivizrServer.html\n",
      "\n",
      "Name: FGNet\n",
      "Description:\n",
      "Build and visualize functional gene and term networks from clustering of enrichment analyses in multiple annotation spaces. The package includes a graphical user interface (GUI) and functions to perform the functional enrichment analysis through DAVID, GeneTerm Linker, gage (GSEA) and topGO.\n",
      "Version: 3.32.0\n",
      "Publication date: None\n",
      "Mantainer: Sara Aibar <saibar@usal.es>\n",
      "Authors:\n",
      "Author: Sara Aibar, Celia Fontanillo, Conrad Droste and Javier De Las Rivas.\n",
      "Dependencies:\n",
      "  - R (>= 4.2.0) type: depends\n",
      "  - igraph (>= 0.6) type: imports\n",
      "  - hwriter, type: imports\n",
      "  - R.utils, type: imports\n",
      "  - XML, type: imports\n",
      "  - plotrix, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - png, type: imports\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/FGNet.html\n",
      "\n",
      "Name: InterCellar\n",
      "Description:\n",
      "InterCellar is implemented as an R/Bioconductor Package containing a Shiny app that allows users to interactively analyze cell-cell communication from scRNA-seq data. Starting from precomputed ligand-receptor interactions, InterCellar provides filtering options, annotations and multiple visualizations to explore clusters, genes and functions. Finally, based on functional annotation from Gene Ontology and pathway databases, InterCellar implements data-driven analyses to investigate cell-cell communication in one or multiple conditions.\n",
      "Version: 2.4.0\n",
      "Publication date: None\n",
      "Mantainer: Marta Interlandi <marta.interlandi01@gmail.com>\n",
      "Authors:\n",
      "Author: Marta Interlandi [cre, aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - config, type: imports\n",
      "  - golem, type: imports\n",
      "  - shiny, type: imports\n",
      "  - DT, type: imports\n",
      "  - shinydashboard, type: imports\n",
      "  - shinyFiles, type: imports\n",
      "  - shinycssloaders, type: imports\n",
      "  - data.table, type: imports\n",
      "  - fs, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - circlize, type: imports\n",
      "  - colourpicker, type: imports\n",
      "  - dendextend, type: imports\n",
      "  - factoextra, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - plotly, type: imports\n",
      "  - plyr, type: imports\n",
      "  - shinyFeedback, type: imports\n",
      "  - shinyalert, type: imports\n",
      "  - tibble, type: imports\n",
      "  - umap, type: imports\n",
      "  - visNetwork, type: imports\n",
      "  - wordcloud2, type: imports\n",
      "  - readxl, type: imports\n",
      "  - htmlwidgets, type: imports\n",
      "  - colorspace, type: imports\n",
      "  - signal, type: imports\n",
      "  - scales, type: imports\n",
      "  - htmltools, type: imports\n",
      "  - ComplexHeatmap, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - stats, type: imports\n",
      "  - tools, type: imports\n",
      "  - utils, type: imports\n",
      "  - biomaRt, type: imports\n",
      "  - rlang, type: imports\n",
      "  - fmsb, type: imports\n",
      "  - igraph, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/InterCellar.html\n",
      "\n",
      "Name: myvariant\n",
      "Description:\n",
      "MyVariant.info is a comprehensive aggregation of variant annotation resources. myvariant is a wrapper for querying MyVariant.info services\n",
      "Version: 1.28.0\n",
      "Publication date: None\n",
      "Mantainer: Adam Mark, Chunlei Wu <cwu@scripps.edu>\n",
      "Authors:\n",
      "Author: Adam Mark\n",
      "Dependencies:\n",
      "  - R (>= 3.2.1) type: depends\n",
      "  - VariantAnnotation, type: depends\n",
      "  - httr, type: imports\n",
      "  - jsonlite, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - Hmisc, type: imports\n",
      "  - plyr, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/myvariant.html\n",
      "\n",
      "Name: pRolocGUI\n",
      "Description:\n",
      "The package pRolocGUI comprises functions to interactively visualise spatial proteomics data on the basis of pRoloc, pRolocdata and shiny.\n",
      "Version: 2.8.0\n",
      "Publication date: None\n",
      "Mantainer: Lisa Breckels <lms79@cam.ac.uk>\n",
      "Authors:\n",
      "Author: Lisa Breckels [aut, cre] , Thomas Naake [aut], Laurent Gatto [aut]\n",
      "Dependencies:\n",
      "  - methods, type: depends\n",
      "  - R (>= 3.1.0) type: depends\n",
      "  - pRoloc, type: depends\n",
      "  - Biobase, type: depends\n",
      "  - MSnbase, type: depends\n",
      "  - shiny (>= 0.9.1) type: imports\n",
      "  - scales, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - DT (>= 0.1.40) type: imports\n",
      "  - graphics, type: imports\n",
      "  - utils, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - shinydashboardPlus (>= 2.0.0) type: imports\n",
      "  - colourpicker, type: imports\n",
      "  - shinyhelper, type: imports\n",
      "  - shinyWidgets, type: imports\n",
      "  - shinyjs, type: imports\n",
      "  - colorspace, type: imports\n",
      "  - stats, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - grid, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - shinydashboard, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/pRolocGUI.html\n",
      "\n",
      "Name: RGSEA\n",
      "Description:\n",
      "Combining bootstrap aggregating and Gene set enrichment analysis (GSEA), RGSEA is a classfication algorithm with high robustness and no over-fitting problem. It performs well especially for the data generated from different exprements.\n",
      "Version: 1.32.0\n",
      "Publication date: None\n",
      "Mantainer: Chengcheng Ma <ccma@sibs.ac.cn>\n",
      "Authors:\n",
      "Author: Chengcheng Ma\n",
      "Dependencies:\n",
      "  - R (>= 2.10.0) type: depends\n",
      "  - BiocGenerics, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL(>=3)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/RGSEA.html\n",
      "\n",
      "Name: SPsimSeq\n",
      "Description:\n",
      "SPsimSeq uses a specially designed exponential family for density estimation to constructs the distribution of gene expression levels from a given real RNA sequencing data (single-cell or bulk), and subsequently simulates a new dataset from the estimated marginal distributions using Gaussian-copulas to retain the dependence between genes. It allows simulation of multiple groups and batches with any required sample size and library size.\n",
      "Version: 1.8.0\n",
      "Publication date: None\n",
      "Mantainer: Joris Meys <Joris.Meys@ugent.be>\n",
      "Authors:\n",
      "Author: Alemu Takele Assefa [aut], Olivier Thas [ths], Joris Meys [cre], Stijn Hawinkel [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - stats, type: imports\n",
      "  - methods, type: imports\n",
      "  - SingleCellExperiment, type: imports\n",
      "  - fitdistrplus, type: imports\n",
      "  - graphics, type: imports\n",
      "  - edgeR, type: imports\n",
      "  - Hmisc, type: imports\n",
      "  - WGCNA, type: imports\n",
      "  - limma, type: imports\n",
      "  - mvtnorm, type: imports\n",
      "  - phyloseq, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/SPsimSeq.html\n",
      "\n",
      "Name: TPP\n",
      "Description:\n",
      "Analyze thermal proteome profiling (TPP) experiments with varying temperatures (TR) or compound concentrations (CCR).\n",
      "Version: 3.26.1\n",
      "Publication date: None\n",
      "Mantainer: Dorothee Childs <d.childs@posteo.de>\n",
      "Authors:\n",
      "Author: Dorothee Childs, Nils Kurzawa, Holger Franken, Carola Doce, Mikhail Savitski and Wolfgang Huber\n",
      "Dependencies:\n",
      "  - R (>= 3.4) type: depends\n",
      "  - Biobase, type: depends\n",
      "  - dplyr, type: depends\n",
      "  - magrittr, type: depends\n",
      "  - tidyr, type: depends\n",
      "  - biobroom, type: imports\n",
      "  - broom, type: imports\n",
      "  - data.table, type: imports\n",
      "  - doParallel, type: imports\n",
      "  - foreach, type: imports\n",
      "  - futile.logger, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - gridExtra, type: imports\n",
      "  - grid, type: imports\n",
      "  - knitr, type: imports\n",
      "  - limma, type: imports\n",
      "  - MASS, type: imports\n",
      "  - mefa, type: imports\n",
      "  - nls2, type: imports\n",
      "  - openxlsx (>= 2.4.0) type: imports\n",
      "  - parallel, type: imports\n",
      "  - plyr, type: imports\n",
      "  - purrr, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - RCurl, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - rmarkdown, type: imports\n",
      "  - splines, type: imports\n",
      "  - stats, type: imports\n",
      "  - stringr, type: imports\n",
      "  - tibble, type: imports\n",
      "  - utils, type: imports\n",
      "  - VennDiagram, type: imports\n",
      "  - VGAM, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/TPP.html\n",
      "\n",
      "Name: acde\n",
      "Description:\n",
      "This package provides a multivariate inferential analysis method for detecting differentially expressed genes in gene expression data. It uses artificial components, close to the data's principal components but with an exact interpretation in terms of differential genetic expression, to identify differentially expressed genes while controlling the false discovery rate (FDR). The methods on this package are described in the vignette or in the article 'Multivariate Method for Inferential Identification of Differentially Expressed Genes in Gene Expression Experiments' by J. P. Acosta, L. Lopez-Kleine and S. Restrepo (2015, pending publication).\n",
      "Version: 1.28.0\n",
      "Publication date: None\n",
      "Mantainer: Juan Pablo Acosta <jpacostar@unal.edu.co>\n",
      "Authors:\n",
      "Author: Juan Pablo Acosta, Liliana Lopez-Kleine\n",
      "Dependencies:\n",
      "  - R (>= 3.3) type: depends\n",
      "  - boot (>= 1.3) type: depends\n",
      "  - stats, type: imports\n",
      "  - graphics, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/acde.html\n",
      "\n",
      "Name: AllelicImbalance\n",
      "Description:\n",
      "Provides a framework for allelic specific expression investigation using RNA-seq data.\n",
      "Version: 1.36.0\n",
      "Publication date: None\n",
      "Mantainer: Jesper R Gadin <j.r.gadin@gmail.com>\n",
      "Authors:\n",
      "Author: Jesper R Gadin, Lasse Folkersen\n",
      "Dependencies:\n",
      "  - R (>= 4.0.0) type: depends\n",
      "  - grid, type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - GenomicAlignments, type: depends\n",
      "  - methods, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - AnnotationDbi, type: imports\n",
      "  - BSgenome, type: imports\n",
      "  - VariantAnnotation, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - GenomicFeatures, type: imports\n",
      "  - Gviz, type: imports\n",
      "  - lattice, type: imports\n",
      "  - latticeExtra, type: imports\n",
      "  - gridExtra, type: imports\n",
      "  - seqinr, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - nlme, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/AllelicImbalance.html\n",
      "\n",
      "Name: artMS\n",
      "Description:\n",
      "artMS provides a set of tools for the analysis of proteomics label-free datasets. It takes as input the MaxQuant search result output (evidence.txt file) and performs quality control, relative quantification using MSstats, downstream analysis and integration. artMS also provides a set of functions to re-format and make it compatible with other analytical tools, including, SAINTq, SAINTexpress, Phosfate, and PHOTON. Check [http://artms.org](http://artms.org) for details.\n",
      "Version: 1.16.0\n",
      "Publication date: None\n",
      "Mantainer: David Jimenez-Morales <biodavidjm@gmail.com>\n",
      "Authors:\n",
      "Author: David Jimenez-Morales [aut, cre] , Alexandre Rosa Campos [aut, ctb] , John Von Dollen [aut], Nevan Krogan [aut] , Danielle Swaney [aut, ctb]\n",
      "Dependencies:\n",
      "  - R (>= 4.1.0) type: depends\n",
      "  - AnnotationDbi, type: imports\n",
      "  - bit64, type: imports\n",
      "  - circlize, type: imports\n",
      "  - cluster, type: imports\n",
      "  - corrplot, type: imports\n",
      "  - data.table, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - getopt, type: imports\n",
      "  - ggdendro, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - gplots, type: imports\n",
      "  - ggrepel, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - grid, type: imports\n",
      "  - limma, type: imports\n",
      "  - MSstats, type: imports\n",
      "  - openxlsx, type: imports\n",
      "  - org.Hs.eg.db, type: imports\n",
      "  - pheatmap, type: imports\n",
      "  - plotly, type: imports\n",
      "  - plyr, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - scales, type: imports\n",
      "  - seqinr, type: imports\n",
      "  - stats, type: imports\n",
      "  - stringr, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - UpSetR, type: imports\n",
      "  - utils, type: imports\n",
      "  - VennDiagram, type: imports\n",
      "  - yaml, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 3) + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/artMS.html\n",
      "\n",
      "Name: ASSET\n",
      "Description:\n",
      "An R package for subset-based analysis of heterogeneous traits and disease subtypes. The package allows the user to search through all possible subsets of z-scores to identify the subset of traits giving the best meta-analyzed z-score. Further, it returns a p-value adjusting for the multiple-testing involved in the search. It also allows for searching for the best combination of disease subtypes associated with each variant.\n",
      "Version: 2.16.0\n",
      "Publication date: None\n",
      "Mantainer: Samsiddhi Bhattacharjee <sb1@nibmg.ac.in>\n",
      "Authors:\n",
      "Author: Samsiddhi Bhattacharjee [aut, cre], Guanghao Qi [aut], Nilanjan Chatterjee [aut], William Wheeler [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - stats, type: depends\n",
      "  - graphics, type: depends\n",
      "  - MASS, type: imports\n",
      "  - msm, type: imports\n",
      "  - rmeta, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2 + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ASSET.html\n",
      "\n",
      "Name: crossmeta\n",
      "Description:\n",
      "Implements cross-platform and cross-species meta-analyses of Affymentrix, Illumina, and Agilent microarray data. This package automates common tasks such as downloading, normalizing, and annotating raw GEO data. The user then selects control and treatment samples in order to perform differential expression analyses for all comparisons. After analysing each contrast seperately, the user can select tissue sources for each contrast and specify any tissue sources that should be grouped for the subsequent meta-analyses.\n",
      "Version: 1.24.0\n",
      "Publication date: None\n",
      "Mantainer: Alex Pickering <alexvpickering@gmail.com>\n",
      "Authors:\n",
      "Author: Alex Pickering\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - affy, type: imports\n",
      "  - affxparser, type: imports\n",
      "  - AnnotationDbi, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - BiocManager (>= 1.30.4) type: imports\n",
      "  - DT (>= 0.2) type: imports\n",
      "  - DBI (>= 1.0.0) type: imports\n",
      "  - data.table (>= 1.10.4) type: imports\n",
      "  - edgeR, type: imports\n",
      "  - fdrtool (>= 1.2.15) type: imports\n",
      "  - GEOquery, type: imports\n",
      "  - limma, type: imports\n",
      "  - matrixStats (>= 0.51.0) type: imports\n",
      "  - metaMA (>= 3.1.2) type: imports\n",
      "  - miniUI (>= 0.1.1) type: imports\n",
      "  - methods, type: imports\n",
      "  - oligo, type: imports\n",
      "  - reader (>= 1.0.6) type: imports\n",
      "  - RCurl (>= 1.95.4.11) type: imports\n",
      "  - RSQLite (>= 2.1.1) type: imports\n",
      "  - stringr (>= 1.2.0) type: imports\n",
      "  - sva, type: imports\n",
      "  - shiny (>= 1.0.0) type: imports\n",
      "  - shinyjs (>= 2.0.0) type: imports\n",
      "  - shinyBS (>= 0.61) type: imports\n",
      "  - shinyWidgets (>= 0.5.3) type: imports\n",
      "  - shinypanel (>= 0.1.0) type: imports\n",
      "  - tibble, type: imports\n",
      "  - XML (>= 3.98.1.17) type: imports\n",
      "  - readxl (>= 1.3.1) type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/crossmeta.html\n",
      "\n",
      "Name: esetVis\n",
      "Description:\n",
      "Utility functions for visualization of expressionSet (or SummarizedExperiment) Bioconductor object, including spectral map, tsne and linear discriminant analysis. Static plot via the ggplot2 package or interactive via the ggvis or rbokeh packages are available.\n",
      "Version: 1.24.0\n",
      "Publication date: None\n",
      "Mantainer: Laure Cougnaud <laure.cougnaud@openanalytics.eu>\n",
      "Authors:\n",
      "Author: Laure Cougnaud <laure.cougnaud at openanalytics.eu>\n",
      "Dependencies:\n",
      "  - mpm, type: imports\n",
      "  - hexbin, type: imports\n",
      "  - Rtsne, type: imports\n",
      "  - MLP, type: imports\n",
      "  - grid, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - MASS, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - methods, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/esetVis.html\n",
      "\n",
      "Name: Herper\n",
      "Description:\n",
      "Many tools for data analysis are not available in R, but are present in public repositories like conda. The Herper package provides a comprehensive set of functions to interact with the conda package managament system. With Herper users can install, manage and run conda packages from the comfort of their R session. Herper also provides an ad-hoc approach to handling external system requirements for R packages. For people developing packages with python conda dependencies we recommend using basilisk (https://bioconductor.org/packages/release/bioc/html/basilisk.html) to internally support these system requirments pre-hoc.\n",
      "Version: 1.8.1\n",
      "Publication date: None\n",
      "Mantainer: Thomas Carroll <tc.infomatics@gmail.com>\n",
      "Authors:\n",
      "Author: Matt Paul [aut] , Thomas Carroll [aut, cre] , Doug Barrows [aut], Kathryn Rozen-Gagnon [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - reticulate, type: depends\n",
      "  - utils, type: imports\n",
      "  - rjson, type: imports\n",
      "  - withr, type: imports\n",
      "  - stats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/Herper.html\n",
      "\n",
      "Name: HiCDCPlus\n",
      "Description:\n",
      "Systematic 3D interaction calls and differential analysis for Hi-C and HiChIP. The HiC-DC+ (Hi-C/HiChIP direct caller plus) package enables principled statistical analysis of Hi-C and HiChIP data sets â including calling significant interactions within a single experiment and performing differential analysis between conditions given replicate experiments â to facilitate global integrative studies. HiC-DC+ estimates significant interactions in a Hi-C or HiChIP experiment directly from the raw contact matrix for each chromosome up to a specified genomic distance, binned by uniform genomic intervals or restriction enzyme fragments, by training a background model to account for random polymer ligation and systematic sources of read count variation.\n",
      "Version: 1.6.0\n",
      "Publication date: None\n",
      "Mantainer: Merve Sahin <merve.sahn@gmail.com>\n",
      "Authors:\n",
      "Author: Merve Sahin [cre, aut]\n",
      "Dependencies:\n",
      "  - Rcpp, type: imports\n",
      "  - InteractionSet, type: imports\n",
      "  - GenomicInteractions, type: imports\n",
      "  - bbmle, type: imports\n",
      "  - pscl, type: imports\n",
      "  - BSgenome, type: imports\n",
      "  - data.table, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - rlang, type: imports\n",
      "  - splines, type: imports\n",
      "  - MASS, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - tibble, type: imports\n",
      "  - R.utils, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - methods, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/HiCDCPlus.html\n",
      "\n",
      "Name: hmdbQuery\n",
      "Description:\n",
      "Define utilities for exploration of human metabolome database, including functions to retrieve specific metabolite entries and data snapshots with pairwise associations (metabolite-gene,-protein,-disease).\n",
      "Version: 1.18.0\n",
      "Publication date: None\n",
      "Mantainer: VJ Carey <stvjc@channing.harvard.edu>\n",
      "Authors:\n",
      "Author: Vince Carey <stvjc at channing.harvard.edu>\n",
      "Dependencies:\n",
      "  - R (>= 3.5) type: depends\n",
      "  - XML, type: depends\n",
      "  - S4Vectors, type: imports\n",
      "  - methods, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/hmdbQuery.html\n",
      "\n",
      "Name: MesKit\n",
      "Description:\n",
      "MesKit provides commonly used analysis and visualization modules based on mutational data generated by multi-region sequencing (MRS). This package allows to depict mutational profiles, measure heterogeneity within or between tumors from the same patient, track evolutionary dynamics, as well as characterize mutational patterns on different levels. Shiny application was also developed for a need of GUI-based analysis. As a handy tool, MesKit can facilitate the interpretation of tumor heterogeneity and the understanding of evolutionary relationship between regions in MRS study.\n",
      "Version: 1.8.0\n",
      "Publication date: None\n",
      "Mantainer: Mengni Liu <niinleslie@gmail.com>\n",
      "Authors:\n",
      "Author: Mengni Liu [aut, cre] , Jianyu Chen [aut, ctb] , Xin Wang [aut, ctb]\n",
      "Dependencies:\n",
      "  - R (>= 4.0.0) type: depends\n",
      "  - methods, type: imports\n",
      "  - data.table, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - tidyr (>= 1.0.0) type: imports\n",
      "  - ape (>= 5.4.1) type: imports\n",
      "  - ggrepel, type: imports\n",
      "  - pracma, type: imports\n",
      "  - ggridges, type: imports\n",
      "  - AnnotationDbi, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - circlize, type: imports\n",
      "  - cowplot, type: imports\n",
      "  - mclust, type: imports\n",
      "  - phangorn, type: imports\n",
      "  - ComplexHeatmap, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/MesKit.html\n",
      "\n",
      "Name: MLP\n",
      "Description:\n",
      "Pathway analysis based on p-values associated to genes from a genes expression analysis of interest. Utility functions enable to extract pathways from the Gene Ontology Biological Process (GOBP), Molecular Function (GOMF) and Cellular Component (GOCC), Kyoto Encyclopedia of Genes of Genomes (KEGG) and Reactome databases. Methodology, and helper functions to display the results as a table, barplot of pathway significance, Gene Ontology graph and pathway significance are available.\n",
      "Version: 1.46.0\n",
      "Publication date: None\n",
      "Mantainer: Tobias Verbeke <tobias.verbeke@openanalytics.eu>\n",
      "Authors:\n",
      "Author: Nandini Raghavan [aut], Tobias Verbeke [aut], An De Bondt [aut], Javier Cabrera [ctb], Dhammika Amaratunga [ctb], Tine Casneuf [ctb], Willem Ligtenberg [ctb], Laure Cougnaud [cre], Katarzyna Gorczak [ctb]\n",
      "Dependencies:\n",
      "  - AnnotationDbi, type: imports\n",
      "  - gplots, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/MLP.html\n",
      "\n",
      "Name: msPurity\n",
      "Description:\n",
      "msPurity R package was developed to: 1) Assess the spectral quality of fragmentation spectra by evaluating the \"precursor ion purity\". 2) Process fragmentation spectra. 3) Perform spectral matching. What is precursor ion purity? -What we call \"Precursor ion purity\" is a measure of the contribution of a selected precursor peak in an isolation window used for fragmentation. The simple calculation involves dividing the intensity of the selected precursor peak by the total intensity of the isolation window. When assessing MS/MS spectra this calculation is done before and after the MS/MS scan of interest and the purity is interpolated at the recorded time of the MS/MS acquisition. Additionally, isotopic peaks can be removed, low abundance peaks are removed that are thought to have limited contribution to the resulting MS/MS spectra and the isolation efficiency of the mass spectrometer can be used to normalise the intensities used for the calculation.\n",
      "Version: 1.24.0\n",
      "Publication date: None\n",
      "Mantainer: Thomas N. Lawson <thomas.nigel.lawson@gmail.com>\n",
      "Authors:\n",
      "Author: Thomas N. Lawson [aut, cre] , Ralf Weber [ctb], Martin Jones [ctb], Julien Saint-Vanne [ctb], Andris Jankevics [ctb], Mark Viant [ths], Warwick Dunn [ths]\n",
      "Dependencies:\n",
      "  - Rcpp, type: depends\n",
      "  - plyr, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - dbplyr, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - foreach, type: imports\n",
      "  - parallel, type: imports\n",
      "  - doSNOW, type: imports\n",
      "  - stringr, type: imports\n",
      "  - mzR, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - fastcluster, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - DBI, type: imports\n",
      "  - RSQLite, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3 + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/msPurity.html\n",
      "\n",
      "Name: ramwas\n",
      "Description:\n",
      "A complete toolset for methylome-wide association studies (MWAS). It is specifically designed for data from enrichment based methylation assays, but can be applied to other data as well. The analysis pipeline includes seven steps: (1) scanning aligned reads from BAM files, (2) calculation of quality control measures, (3) creation of methylation score (coverage) matrix, (4) principal component analysis for capturing batch effects and detection of outliers, (5) association analysis with respect to phenotypes of interest while correcting for top PCs and known covariates, (6) annotation of significant findings, and (7) multi-marker analysis (methylation risk score) using elastic net. Additionally, RaMWAS include tools for joint analysis of methlyation and genotype data. This work is published in Bioinformatics, Shabalin et al. (2018) .\n",
      "Version: 1.22.0\n",
      "Publication date: None\n",
      "Mantainer: Andrey A Shabalin <andrey.shabalin@gmail.com>\n",
      "Authors:\n",
      "Author: Andrey A Shabalin [aut, cre] , Shaunna L Clark [aut], Mohammad W Hattab [aut], Karolina A Aberg [aut], Edwin J C G van den Oord [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.3.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - filematrix, type: depends\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - digest, type: imports\n",
      "  - glmnet, type: imports\n",
      "  - KernSmooth, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - GenomicAlignments, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - parallel, type: imports\n",
      "  - biomaRt, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ramwas.html\n",
      "\n",
      "Name: scPCA\n",
      "Description:\n",
      "A toolbox for sparse contrastive principal component analysis (scPCA) of high-dimensional biological data. scPCA combines the stability and interpretability of sparse PCA with contrastive PCA's ability to disentangle biological signal from unwanted variation through the use of control data. Also implements and extends cPCA.\n",
      "Version: 1.12.0\n",
      "Publication date: None\n",
      "Mantainer: Philippe Boileau <philippe_boileau@berkeley.edu>\n",
      "Authors:\n",
      "Author: Philippe Boileau [aut, cre, cph] , Nima Hejazi [aut] , Sandrine Dudoit [ctb, ths]\n",
      "Dependencies:\n",
      "  - R (>= 4.0.0) type: depends\n",
      "  - stats, type: imports\n",
      "  - methods, type: imports\n",
      "  - assertthat, type: imports\n",
      "  - tibble, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - purrr, type: imports\n",
      "  - stringr, type: imports\n",
      "  - Rdpack, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - elasticnet, type: imports\n",
      "  - sparsepca, type: imports\n",
      "  - cluster, type: imports\n",
      "  - kernlab, type: imports\n",
      "  - origami, type: imports\n",
      "  - RSpectra, type: imports\n",
      "  - coop, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - DelayedArray, type: imports\n",
      "  - ScaledMatrix, type: imports\n",
      "  - MatrixGenerics, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/scPCA.html\n",
      "\n",
      "Name: ssize\n",
      "Description:\n",
      "Functions for computing and displaying sample size information for gene expression arrays.\n",
      "Version: 1.72.0\n",
      "Publication date: None\n",
      "Mantainer: Gregory R. Warnes <greg@random-technologies-llc.com>\n",
      "Authors:\n",
      "Author: Gregory R. Warnes, Peng Liu, and Fasheng Li\n",
      "Dependencies:\n",
      "  - gdata, type: depends\n",
      "  - xtable, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ssize.html\n",
      "\n",
      "Name: dcanr\n",
      "Description:\n",
      "This package implements methods and an evaluation framework to infer differential co-expression/association networks. Various methods are implemented and can be evaluated using simulated datasets. Inference of differential co-expression networks can allow identification of networks that are altered between two conditions (e.g., health and disease).\n",
      "Version: 1.14.0\n",
      "Publication date: None\n",
      "Mantainer: Dharmesh D. Bhuva <bhuva.d@wehi.edu.au>\n",
      "Authors:\n",
      "Author: Dharmesh D. Bhuva [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 3.6.0) type: depends\n",
      "  - igraph, type: imports\n",
      "  - foreach, type: imports\n",
      "  - plyr, type: imports\n",
      "  - stringr, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - methods, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - circlize, type: imports\n",
      "  - doRNG, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/dcanr.html\n",
      "\n",
      "Name: easier\n",
      "Description:\n",
      "This package provides a workflow for the use of EaSIeR tool, developed to assess patients' likelihood to respond to ICB therapies providing just the patients' RNA-seq data as input. We integrate RNA-seq data with different types of prior knowledge to extract quantitative descriptors of the tumor microenvironment from several points of view, including composition of the immune repertoire, and activity of intra- and extra-cellular communications. Then, we use multi-task machine learning trained in TCGA data to identify how these descriptors can simultaneously predict several state-of-the-art hallmarks of anti-cancer immune response. In this way we derive cancer-specific models and identify cancer-specific systems biomarkers of immune response. These biomarkers have been experimentally validated in the literature and the performance of EaSIeR predictions has been validated using independent datasets form four different cancer types with patients treated with anti-PD1 or anti-PDL1 therapy.\n",
      "Version: 1.4.0\n",
      "Publication date: None\n",
      "Mantainer: Oscar Lapuente-Santana <o.lapuente.santana@tue.nl>\n",
      "Authors:\n",
      "Author: Oscar Lapuente-Santana [aut, cre] , Federico Marini [aut] , Arsenij Ustjanzew [aut] , Francesca Finotello [aut] , Federica Eduati [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.2.0) type: depends\n",
      "  - progeny, type: imports\n",
      "  - easierData, type: imports\n",
      "  - dorothea, type: imports\n",
      "  - quantiseqr, type: imports\n",
      "  - ROCR, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - stats, type: imports\n",
      "  - graphics, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - ggpubr, type: imports\n",
      "  - DESeq2, type: imports\n",
      "  - utils, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - rlang, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - rstatix, type: imports\n",
      "  - ggrepel, type: imports\n",
      "  - coin, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/easier.html\n",
      "\n",
      "Name: epiNEM\n",
      "Description:\n",
      "epiNEM is an extension of the original Nested Effects Models (NEM). EpiNEM is able to take into account double knockouts and infer more complex network signalling pathways. It is tailored towards large scale double knock-out screens.\n",
      "Version: 1.22.0\n",
      "Publication date: None\n",
      "Mantainer: Martin Pirkl <martinpirkl@yahoo.de>\n",
      "Authors:\n",
      "Author: Madeline Diekmann & Martin Pirkl\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - BoolNet, type: imports\n",
      "  - e1071, type: imports\n",
      "  - gtools, type: imports\n",
      "  - stats, type: imports\n",
      "  - igraph, type: imports\n",
      "  - utils, type: imports\n",
      "  - lattice, type: imports\n",
      "  - latticeExtra, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - pcalg, type: imports\n",
      "  - minet, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graph, type: imports\n",
      "  - mnem, type: imports\n",
      "  - latex2exp, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/epiNEM.html\n",
      "\n",
      "Name: MEAL\n",
      "Description:\n",
      "Package to integrate methylation and expression data. It can also perform methylation or expression analysis alone. Several plotting functionalities are included as well as a new region analysis based on redundancy analysis. Effect of SNPs on a region can also be estimated.\n",
      "Version: 1.28.0\n",
      "Publication date: None\n",
      "Mantainer: Xavier EscribÃ  Montagut <xavier.escriba@isglobal.org>\n",
      "Authors:\n",
      "Author: Carlos Ruiz-Arenas [aut, cre], Juan R. Gonzalez [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.6.0) type: depends\n",
      "  - Biobase, type: depends\n",
      "  - MultiDataSet, type: depends\n",
      "  - GenomicRanges, type: imports\n",
      "  - limma, type: imports\n",
      "  - vegan, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - minfi, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - methods, type: imports\n",
      "  - parallel, type: imports\n",
      "  - ggplot2 (>= 2.0.0) type: imports\n",
      "  - permute, type: imports\n",
      "  - Gviz, type: imports\n",
      "  - missMethyl, type: imports\n",
      "  - isva, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - SmartSVA, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - matrixStats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/MEAL.html\n",
      "\n",
      "Name: PepsNMR\n",
      "Description:\n",
      "This package provides R functions for common pre-procssing steps that are applied on 1H-NMR data. It also provides a function to read the FID signals directly in the Bruker format.\n",
      "Version: 1.16.0\n",
      "Publication date: None\n",
      "Mantainer: Manon Martin <manon.martin@uclouvain.be>\n",
      "Authors:\n",
      "Author: Manon Martin [aut, cre], Bernadette Govaerts [aut, ths], BenoÃ®t Legat [aut], Paul H.C. Eilers [aut], Pascal de Tullio [dtc], Bruno Boulanger [ctb], Julien Vanwinsberghe [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 3.6) type: depends\n",
      "  - Matrix, type: imports\n",
      "  - ptw, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - gridExtra, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - methods, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2 | file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/PepsNMR.html\n",
      "\n",
      "Name: PPInfer\n",
      "Description:\n",
      "Interactions between proteins occur in many, if not most, biological processes. Most proteins perform their functions in networks associated with other proteins and other biomolecules. This fact has motivated the development of a variety of experimental methods for the identification of protein interactions. This variety has in turn ushered in the development of numerous different computational approaches for modeling and predicting protein interactions. Sometimes an experiment is aimed at identifying proteins closely related to some interesting proteins. A network based statistical learning method is used to infer the putative functions of proteins from the known functions of its neighboring proteins on a PPI network. This package identifies such proteins often involved in the same or similar biological functions.\n",
      "Version: 1.24.2\n",
      "Publication date: None\n",
      "Mantainer: Dongmin Jung <dmdmjung@gmail.com>\n",
      "Authors:\n",
      "Author: Dongmin Jung, Xijin Ge\n",
      "Dependencies:\n",
      "  - biomaRt, type: depends\n",
      "  - fgsea, type: depends\n",
      "  - kernlab, type: depends\n",
      "  - ggplot2, type: depends\n",
      "  - igraph, type: depends\n",
      "  - STRINGdb, type: depends\n",
      "  - yeastExpData, type: depends\n",
      "  - httr, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/PPInfer.html\n",
      "\n",
      "Name: pqsfinder\n",
      "Description:\n",
      "Pqsfinder detects DNA and RNA sequence patterns that are likely to fold into an intramolecular G-quadruplex (G4). Unlike many other approaches, pqsfinder is able to detect G4s folded from imperfect G-runs containing bulges or mismatches or G4s having long loops. Pqsfinder also assigns an integer score to each hit that was fitted on G4 sequencing data and corresponds to expected stability of the folded G4.\n",
      "Version: 2.14.1\n",
      "Publication date: None\n",
      "Mantainer: Jiri Hon <jiri.hon@gmail.com>\n",
      "Authors:\n",
      "Author: Jiri Hon, Dominika Labudova, Matej Lexa and Tomas Martinek\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - Biostrings, type: depends\n",
      "  - Rcpp (>= 0.12.3) type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - methods, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "BSD_2_clause + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/pqsfinder.html\n",
      "\n",
      "Name: rcellminer\n",
      "Description:\n",
      "The NCI-60 cancer cell line panel has been used over the course of several decades as an anti-cancer drug screen. This panel was developed as part of the Developmental Therapeutics Program (DTP, http://dtp.nci.nih.gov/) of the U.S. National Cancer Institute (NCI). Thousands of compounds have been tested on the NCI-60, which have been extensively characterized by many platforms for gene and protein expression, copy number, mutation, and others (Reinhold, et al., 2012). The purpose of the CellMiner project (http://discover.nci.nih.gov/ cellminer) has been to integrate data from multiple platforms used to analyze the NCI-60 and to provide a powerful suite of tools for exploration of NCI-60 data.\n",
      "Version: 2.20.0\n",
      "Publication date: None\n",
      "Mantainer: Augustin Luna <lunaa@cbio.mskcc.org>, Vinodh Rajapakse <vinodh.rajapakse@nih.gov>, Fathi Elloumi <fathi.elloumi@nih.gov>\n",
      "Authors:\n",
      "Author: Augustin Luna, Vinodh Rajapakse, Fabricio Sousa\n",
      "Dependencies:\n",
      "  - R (>= 3.2) type: depends\n",
      "  - Biobase, type: depends\n",
      "  - rcellminerData, type: depends\n",
      "  - stringr, type: imports\n",
      "  - gplots, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - shiny, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL-3 + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/rcellminer.html\n",
      "\n",
      "Name: rexposome\n",
      "Description:\n",
      "Package that allows to explore the exposome and to perform association analyses between exposures and health outcomes.\n",
      "Version: 1.20.1\n",
      "Publication date: None\n",
      "Mantainer: Xavier EscribÃ  Montagut <xavier.escriba@isglobal.org>\n",
      "Authors:\n",
      "Author: Carles Hernandez-Ferrer [aut, cre], Juan R. Gonzalez [aut], Xavier EscribÃ -Montagut [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.5) type: depends\n",
      "  - Biobase, type: depends\n",
      "  - methods, type: imports\n",
      "  - utils, type: imports\n",
      "  - stats, type: imports\n",
      "  - lsr, type: imports\n",
      "  - FactoMineR, type: imports\n",
      "  - stringr, type: imports\n",
      "  - circlize, type: imports\n",
      "  - corrplot, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - pryr, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - imputeLCMD, type: imports\n",
      "  - scatterplot3d, type: imports\n",
      "  - glmnet, type: imports\n",
      "  - gridExtra, type: imports\n",
      "  - grid, type: imports\n",
      "  - Hmisc, type: imports\n",
      "  - gplots, type: imports\n",
      "  - gtools, type: imports\n",
      "  - scales, type: imports\n",
      "  - lme4, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - ggrepel, type: imports\n",
      "  - mice, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/rexposome.html\n",
      "\n",
      "Name: structToolbox\n",
      "Description:\n",
      "An extensive set of data (pre-)processing and analysis methods and tools for metabolomics and other omics, with a strong emphasis on statistics and machine learning. This toolbox allows the user to build extensive and standardised workflows for data analysis. The methods and tools have been implemented using class-based templates provided by the struct (Statistics in R Using Class-based Templates) package. The toolbox includes pre-processing methods (e.g. signal drift and batch correction, normalisation, missing value imputation and scaling), univariate (e.g. ttest, various forms of ANOVA, KruskalâWallis test and more) and multivariate statistical methods (e.g. PCA and PLS, including cross-validation and permutation testing) as well as machine learning methods (e.g. Support Vector Machines). The STATistics Ontology (STATO) has been integrated and implemented to provide standardised definitions for the different methods, inputs and outputs.\n",
      "Version: 1.10.1\n",
      "Publication date: None\n",
      "Mantainer: Gavin Rhys Lloyd <g.r.lloyd@bham.ac.uk>\n",
      "Authors:\n",
      "Author: Gavin Rhys Lloyd [aut, cre] , Ralf Johannes Maria Weber [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - struct, type: depends\n",
      "  - ggplot2, type: imports\n",
      "  - ggthemes, type: imports\n",
      "  - grid, type: imports\n",
      "  - gridExtra, type: imports\n",
      "  - methods, type: imports\n",
      "  - scales, type: imports\n",
      "  - sp, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/structToolbox.html\n",
      "\n",
      "Name: a4Reporting\n",
      "Description:\n",
      "Utility functions to facilitate the reporting of the Automated Affymetrix Array Analysis Reporting set of packages.\n",
      "Version: 1.46.0\n",
      "Publication date: None\n",
      "Mantainer: Laure Cougnaud <laure.cougnaud@openanalytics.eu>\n",
      "Authors:\n",
      "Author: Tobias Verbeke [aut], Laure Cougnaud [cre]\n",
      "Dependencies:\n",
      "  - methods, type: imports\n",
      "  - xtable, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/a4Reporting.html\n",
      "\n",
      "Name: affycomp\n",
      "Description:\n",
      "The package contains functions that can be used to compare expression measures for Affymetrix Oligonucleotide Arrays.\n",
      "Version: 1.74.0\n",
      "Publication date: None\n",
      "Mantainer: Rafael A. Irizarry <rafa@jhu.edu>\n",
      "Authors:\n",
      "Author: Rafael A. Irizarry <rafa at jhu.edu> and Zhijin Wu <zwu at stat.brown.edu> with contributions from Simon Cawley <simon_cawley at affymetrix.com>\n",
      "Dependencies:\n",
      "  - R (>= 2.13.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - Biobase, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/affycomp.html\n",
      "\n",
      "Name: amplican\n",
      "Description:\n",
      "`amplican` performs alignment of the amplicon reads, normalizes gathered data, calculates multiple statistics (e.g. cut rates, frameshifts) and presents results in form of aggregated reports. Data and statistics can be broken down by experiments, barcodes, user defined groups, guides and amplicons allowing for quick identification of potential problems.\n",
      "Version: 1.20.0\n",
      "Publication date: None\n",
      "Mantainer: Eivind Valen <eivind.valen@gmail.com>\n",
      "Authors:\n",
      "Author: Kornel Labun [aut], Eivind Valen [cph, cre]\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - BiocGenerics, type: depends\n",
      "  - Biostrings, type: depends\n",
      "  - data.table (>= 1.10.4-3) type: depends\n",
      "  - Rcpp, type: imports\n",
      "  - utils (>= 3.4.1) type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - ShortRead, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - gtable (>= 0.2.0) type: imports\n",
      "  - gridExtra (>= 2.2.1) type: imports\n",
      "  - ggplot2 (>= 2.2.0) type: imports\n",
      "  - ggthemes (>= 3.4.0) type: imports\n",
      "  - waffle (>= 0.7.0) type: imports\n",
      "  - stringr (>= 1.2.0) type: imports\n",
      "  - stats (>= 3.4.1) type: imports\n",
      "  - matrixStats (>= 0.52.2) type: imports\n",
      "  - Matrix (>= 1.2-10) type: imports\n",
      "  - dplyr (>= 0.7.2) type: imports\n",
      "  - rmarkdown (>= 1.6) type: imports\n",
      "  - knitr (>= 1.16) type: imports\n",
      "  - clusterCrit (>= 1.2.7) type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/amplican.html\n",
      "\n",
      "Name: CNORode\n",
      "Description:\n",
      "Logic based ordinary differential equation (ODE) add-on to CellNOptR.\n",
      "Version: 1.40.0\n",
      "Publication date: None\n",
      "Mantainer: Attila Gabor <attila.gabor@uni-heidelberg.de>\n",
      "Authors:\n",
      "Author: David Henriques, Thomas Cokelaer, Attila Gabor, Federica Eduati, Enio Gjerga\n",
      "Dependencies:\n",
      "  - CellNOptR, type: depends\n",
      "  - genalg, type: depends\n",
      "  - knitr, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/CNORode.html\n",
      "\n",
      "Name: coMET\n",
      "Description:\n",
      "Visualisation of EWAS results in a genomic region. In addition to phenotype-association P-values, coMET also generates plots of co-methylation patterns and provides a series of annotation tracks. It can be used to other omic-wide association scans as lon:g as the data can be translated to genomic level and for any species.\n",
      "Version: 1.30.0\n",
      "Publication date: None\n",
      "Mantainer: Tiphaine Martin <tiphaine.martin@mssm.edu>\n",
      "Authors:\n",
      "Author: Tiphaine C. Martin [aut,cre], Thomas Hardiman [aut], Idil Yet [aut], Pei-Chien Tsai [aut], Jordana T. Bell [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.1.0) type: depends\n",
      "  - grid, type: depends\n",
      "  - utils, type: depends\n",
      "  - biomaRt, type: depends\n",
      "  - Gviz, type: depends\n",
      "  - psych, type: depends\n",
      "  - hash, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - gridExtra, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - stats, type: imports\n",
      "  - corrplot, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/coMET.html\n",
      "\n",
      "Name: customProDB\n",
      "Description:\n",
      "Database search is the most widely used approach for peptide and protein identification in mass spectrometry-based proteomics studies. Our previous study showed that sample-specific protein databases derived from RNA-Seq data can better approximate the real protein pools in the samples and thus improve protein identification. More importantly, single nucleotide variations, short insertion and deletions and novel junctions identified from RNA-Seq data make protein database more complete and sample-specific. Here, we report an R package customProDB that enables the easy generation of customized databases from RNA-Seq data for proteomics search. This work bridges genomics and proteomics studies and facilitates cross-omics data integration.\n",
      "Version: 1.38.0\n",
      "Publication date: None\n",
      "Mantainer: Xiaojing Wang <xwang.research@gmail.com> Bo Wen <wenbostar@gmail.com>\n",
      "Authors:\n",
      "Author: Xiaojing Wang\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - IRanges, type: depends\n",
      "  - AnnotationDbi, type: depends\n",
      "  - biomaRt, type: depends\n",
      "  - S4Vectors, type: imports\n",
      "  - DBI, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - GenomicAlignments, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - GenomicFeatures, type: imports\n",
      "  - stringr, type: imports\n",
      "  - RCurl, type: imports\n",
      "  - plyr, type: imports\n",
      "  - VariantAnnotation, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - RSQLite, type: imports\n",
      "  - AhoCorasickTrie, type: imports\n",
      "  - methods, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/customProDB.html\n",
      "\n",
      "Name: DiscoRhythm\n",
      "Description:\n",
      "Set of functions for estimation of cyclical characteristics, such as period, phase, amplitude, and statistical significance in large temporal datasets. Supporting functions are available for quality control, dimensionality reduction, spectral analysis, and analysis of experimental replicates. Contains a R Shiny web interface to execute all workflow steps.\n",
      "Version: 1.14.0\n",
      "Publication date: None\n",
      "Mantainer: Matthew Carlucci <Matthew.Carlucci@camh.ca>\n",
      "Authors:\n",
      "Author: Matthew Carlucci [aut, cre], Algimantas KriÅ¡ÄiÅ«nas [aut], Haohan Li [aut], Povilas Gibas [aut], Karolis KonceviÄius [aut], Art Petronis [aut], Gabriel Oh [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.6.0) type: depends\n",
      "  - matrixTests, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - MetaCycle (>= 1.2.0) type: imports\n",
      "  - data.table, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - ggExtra, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - broom, type: imports\n",
      "  - shiny, type: imports\n",
      "  - shinyBS, type: imports\n",
      "  - shinycssloaders, type: imports\n",
      "  - shinydashboard, type: imports\n",
      "  - shinyjs, type: imports\n",
      "  - BiocStyle, type: imports\n",
      "  - rmarkdown, type: imports\n",
      "  - knitr, type: imports\n",
      "  - kableExtra, type: imports\n",
      "  - magick, type: imports\n",
      "  - VennDiagram, type: imports\n",
      "  - UpSetR, type: imports\n",
      "  - heatmaply, type: imports\n",
      "  - viridis, type: imports\n",
      "  - plotly, type: imports\n",
      "  - DT, type: imports\n",
      "  - gridExtra, type: imports\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - zip, type: imports\n",
      "  - reshape2, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/DiscoRhythm.html\n",
      "\n",
      "Name: enrichTF\n",
      "Description:\n",
      "As transcription factors (TFs) play a crucial role in regulating the transcription process through binding on the genome alone or in a combinatorial manner, TF enrichment analysis is an efficient and important procedure to locate the candidate functional TFs from a set of experimentally defined regulatory regions. While it is commonly accepted that structurally related TFs may have similar binding preference to sequences (i.e. motifs) and one TF may have multiple motifs, TF enrichment analysis is much more challenging than motif enrichment analysis. Here we present a R package for TF enrichment analysis which combine motif enrichment with the PECA model.\n",
      "Version: 1.14.0\n",
      "Publication date: None\n",
      "Mantainer: Zheng Wei <wzwz@stanford.edu>\n",
      "Authors:\n",
      "Author: Zheng Wei, Zhana Duren, Shining Ma\n",
      "Dependencies:\n",
      "  - pipeFrame, type: depends\n",
      "  - BSgenome, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - motifmatchr, type: imports\n",
      "  - TFBSTools, type: imports\n",
      "  - R.utils, type: imports\n",
      "  - methods, type: imports\n",
      "  - JASPAR2018, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - utils, type: imports\n",
      "  - parallel, type: imports\n",
      "  - stats, type: imports\n",
      "  - ggpubr, type: imports\n",
      "  - heatmap3, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - clusterProfiler, type: imports\n",
      "  - rmarkdown, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - magrittr, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/enrichTF.html\n",
      "\n",
      "Name: scClassify\n",
      "Description:\n",
      "scClassify is a multiscale classification framework for single-cell RNA-seq data based on ensemble learning and cell type hierarchies, enabling sample size estimation required for accurate cell type classification and joint classification of cells using multiple references.\n",
      "Version: 1.10.0\n",
      "Publication date: None\n",
      "Mantainer: Yingxin Lin <yingxin.lin@sydney.edu.au>\n",
      "Authors:\n",
      "Author: Yingxin Lin\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - S4Vectors, type: imports\n",
      "  - limma, type: imports\n",
      "  - ggraph, type: imports\n",
      "  - igraph, type: imports\n",
      "  - methods, type: imports\n",
      "  - cluster, type: imports\n",
      "  - minpack.lm, type: imports\n",
      "  - mixtools, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - proxy, type: imports\n",
      "  - proxyC, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - hopach, type: imports\n",
      "  - diptest, type: imports\n",
      "  - mgcv, type: imports\n",
      "  - stats, type: imports\n",
      "  - graphics, type: imports\n",
      "  - statmod, type: imports\n",
      "  - Cepo, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/scClassify.html\n",
      "\n",
      "Name: a4\n",
      "Description:\n",
      "Umbrella package is available for the entire Automated Affymetrix Array Analysis suite of package.\n",
      "Version: 1.46.0\n",
      "Publication date: None\n",
      "Mantainer: Laure Cougnaud <laure.cougnaud@openanalytics.eu>\n",
      "Authors:\n",
      "Author: Willem Talloen [aut], Tobias Verbeke [aut], Laure Cougnaud [cre]\n",
      "Dependencies:\n",
      "  - a4Base, type: depends\n",
      "  - a4Preproc, type: depends\n",
      "  - a4Classif, type: depends\n",
      "  - a4Core, type: depends\n",
      "  - a4Reporting, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/a4.html\n",
      "\n",
      "Name: AffyCompatible\n",
      "Description:\n",
      "Bioconductor version: Release (3.16)\n",
      "Version: 1.58.0\n",
      "Publication date: None\n",
      "Mantainer: Author: Martin Morgan, Robert Gentleman\n",
      "Authors:\n",
      "This package provides an interface to Affymetrix chip annotation and sample attribute files. The package allows an easy way for users to download and manage local data bases of Affynmetrix NetAffx annotation files. The package also provides access to GeneChip Operating System (GCOS) and GeneChip Command Console (AGCC)-compatible sample annotation files.\n",
      "Dependencies:\n",
      "  - R (>= 2.7.0) type: depends\n",
      "  - XML (>= 2.8-1) type: depends\n",
      "  - RCurl (>= 0.8-1) type: depends\n",
      "  - methods, type: depends\n",
      "  - Biostrings, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/AffyCompatible.html\n",
      "\n",
      "Name: ASICS\n",
      "Description:\n",
      "With a set of pure metabolite reference spectra, ASICS quantifies concentration of metabolites in a complex spectrum. The identification of metabolites is performed by fitting a mixture model to the spectra of the library with a sparse penalty. The method and its statistical properties are described in Tardivel et al. (2017) .\n",
      "Version: 2.14.0\n",
      "Publication date: None\n",
      "Mantainer: GaÃ«lle Lefort <gaelle.lefort@inrae.fr>\n",
      "Authors:\n",
      "Author: GaÃ«lle Lefort [aut, cre], RÃ©mi Servien [aut], Patrick Tardivel [aut], Nathalie Vialaneix [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.5) type: depends\n",
      "  - BiocParallel, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - glmnet, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - gridExtra, type: imports\n",
      "  - methods, type: imports\n",
      "  - mvtnorm, type: imports\n",
      "  - PepsNMR, type: imports\n",
      "  - plyr, type: imports\n",
      "  - quadprog, type: imports\n",
      "  - ropls, type: imports\n",
      "  - stats, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - utils, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - zoo, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ASICS.html\n",
      "\n",
      "Name: BiocSklearn\n",
      "Description:\n",
      "This package provides interfaces to selected sklearn elements, and demonstrates fault tolerant use of python modules requiring extensive iteration.\n",
      "Version: 1.20.1\n",
      "Publication date: None\n",
      "Mantainer: Vince Carey <stvjc@channing.harvard.edu>\n",
      "Authors:\n",
      "Author: Vince Carey [cre, aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - reticulate, type: depends\n",
      "  - methods, type: depends\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - basilisk, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/BiocSklearn.html\n",
      "\n",
      "Name: Cepo\n",
      "Description:\n",
      "Defining the identity of a cell is fundamental to understand the heterogeneity of cells to various environmental signals and perturbations. We present Cepo, a new method to explore cell identities from single-cell RNA-sequencing data using differential stability as a new metric to define cell identity genes. Cepo computes cell-type specific gene statistics pertaining to differential stable gene expression.\n",
      "Version: 1.4.0\n",
      "Publication date: None\n",
      "Mantainer: Hani Jieun Kim <hani.kim127@gmail.com>\n",
      "Authors:\n",
      "Author: Hani Jieun Kim [aut, cre] , Kevin Wang [aut]\n",
      "Dependencies:\n",
      "  - GSEABase, type: depends\n",
      "  - R (>= 4.1) type: depends\n",
      "  - DelayedMatrixStats, type: imports\n",
      "  - DelayedArray, type: imports\n",
      "  - HDF5Array, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - methods, type: imports\n",
      "  - SingleCellExperiment, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - rlang, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - patchwork, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - stats, type: imports\n",
      "  - dplyr, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/Cepo.html\n",
      "\n",
      "Name: GOsummaries\n",
      "Description:\n",
      "A package to visualise Gene Ontology (GO) enrichment analysis results on gene lists arising from different analyses such clustering or PCA. The significant GO categories are visualised as word clouds that can be combined with different plots summarising the underlying data.\n",
      "Version: 2.34.0\n",
      "Publication date: None\n",
      "Mantainer: Raivo Kolde <raivo.kolde@eesti.ee>\n",
      "Authors:\n",
      "Author: Raivo Kolde <raivo.kolde at eesti.ee>\n",
      "Dependencies:\n",
      "  - R (>= 2.15) type: depends\n",
      "  - Rcpp, type: depends\n",
      "  - plyr, type: imports\n",
      "  - grid, type: imports\n",
      "  - gProfileR, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - limma, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - gtable, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/GOsummaries.html\n",
      "\n",
      "Name: h5vc\n",
      "Description:\n",
      "This package contains functions to interact with tally data from NGS experiments that is stored in HDF5 files.\n",
      "Version: 2.32.0\n",
      "Publication date: None\n",
      "Mantainer: Paul Theodor Pyl <paul.theodor.pyl@gmail.com>\n",
      "Authors:\n",
      "Author: Paul Theodor Pyl\n",
      "Dependencies:\n",
      "  - grid, type: depends\n",
      "  - gridExtra, type: depends\n",
      "  - ggplot2, type: depends\n",
      "  - rhdf5, type: imports\n",
      "  - reshape, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - methods, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - abind, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - BatchJobs, type: imports\n",
      "  - h5vcData, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 3)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/h5vc.html\n",
      "\n",
      "Name: MAIT\n",
      "Description:\n",
      "The MAIT package contains functions to perform end-to-end statistical analysis of LC/MS Metabolomic Data. Special emphasis is put on peak annotation and in modular function design of the functions.\n",
      "Version: 1.32.0\n",
      "Publication date: None\n",
      "Mantainer: Pol Sola-Santos <pol.soladelossantos@gmail.com>\n",
      "Authors:\n",
      "Author: Francesc Fernandez-Albert, Rafael Llorach, Cristina Andres-LaCueva, Alexandre Perera\n",
      "Dependencies:\n",
      "  - R (>= 2.10) type: depends\n",
      "  - CAMERA, type: depends\n",
      "  - Rcpp, type: depends\n",
      "  - pls, type: depends\n",
      "  - gplots, type: imports\n",
      "  - e1071, type: imports\n",
      "  - class, type: imports\n",
      "  - MASS, type: imports\n",
      "  - plsgenomics, type: imports\n",
      "  - agricolae, type: imports\n",
      "  - xcms, type: imports\n",
      "  - methods, type: imports\n",
      "  - caret, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/MAIT.html\n",
      "\n",
      "Name: MetaNeighbor\n",
      "Description:\n",
      "MetaNeighbor allows users to quantify cell type replicability across datasets using neighbor voting.\n",
      "Version: 1.18.0\n",
      "Publication date: None\n",
      "Mantainer: Stephan Fischer <fischer@cshl.edu>\n",
      "Authors:\n",
      "Author: Megan Crow [aut, cre], Sara Ballouz [ctb], Manthan Shah [ctb], Stephan Fischer [ctb], Jesse Gillis [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.5) type: depends\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - methods, type: imports\n",
      "  - stats (>= 3.4) type: imports\n",
      "  - utils (>= 3.4) type: imports\n",
      "  - Matrix (>= 1.2) type: imports\n",
      "  - matrixStats (>= 0.54) type: imports\n",
      "  - beanplot (>= 1.2) type: imports\n",
      "  - gplots (>= 3.0.1) type: imports\n",
      "  - RColorBrewer (>= 1.1.2) type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - SingleCellExperiment, type: imports\n",
      "  - igraph, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - tibble, type: imports\n",
      "  - ggplot2, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/MetaNeighbor.html\n",
      "\n",
      "Name: ReactomeContentService4R\n",
      "Description:\n",
      "Reactome is a free, open-source, open access, curated and peer-reviewed knowledgebase of bio-molecular pathways. This package is to interact with the Reactome Content Service API. Pre-built functions would allow users to retrieve data and images that consist of proteins, pathways, and other molecules related to a specific gene or entity in Reactome.\n",
      "Version: 1.6.0\n",
      "Publication date: None\n",
      "Mantainer: Chi-Lam Poon <clpoon807@gmail.com>\n",
      "Authors:\n",
      "Author: Chi-Lam Poon [aut, cre] , Reactome [cph]\n",
      "Dependencies:\n",
      "  - httr, type: imports\n",
      "  - jsonlite, type: imports\n",
      "  - utils, type: imports\n",
      "  - magick (>= 2.5.1) type: imports\n",
      "  - data.table, type: imports\n",
      "  - doParallel, type: imports\n",
      "  - foreach, type: imports\n",
      "  - parallel, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Apache License (>= 2.0) | file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ReactomeContentService4R.html\n",
      "\n",
      "Name: RNAAgeCalc\n",
      "Description:\n",
      "It has been shown that both DNA methylation and RNA transcription are linked to chronological age and age related diseases. Several estimators have been developed to predict human aging from DNA level and RNA level. Most of the human transcriptional age predictor are based on microarray data and limited to only a few tissues. To date, transcriptional studies on aging using RNASeq data from different human tissues is limited. The aim of this package is to provide a tool for across-tissue and tissue-specific transcriptional age calculation based on GTEx RNASeq data.\n",
      "Version: 1.10.0\n",
      "Publication date: None\n",
      "Mantainer: Xu Ren <xuren2120@gmail.com>\n",
      "Authors:\n",
      "Author: Xu Ren [aut, cre], Pei Fen Kuan [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.6) type: depends\n",
      "  - ggplot2, type: imports\n",
      "  - recount, type: imports\n",
      "  - impute, type: imports\n",
      "  - AnnotationDbi, type: imports\n",
      "  - org.Hs.eg.db, type: imports\n",
      "  - stats, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - methods, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/RNAAgeCalc.html\n",
      "\n",
      "Name: RNAmodR\n",
      "Description:\n",
      "RNAmodR provides classes and workflows for loading/aggregation data from high througput sequencing aimed at detecting post-transcriptional modifications through analysis of specific patterns. In addition, utilities are provided to validate and visualize the results. The RNAmodR package provides a core functionality from which specific analysis strategies can be easily implemented as a seperate package.\n",
      "Version: 1.12.0\n",
      "Publication date: None\n",
      "Mantainer: Felix G.M. Ernst <felix.gm.ernst@outlook.com>\n",
      "Authors:\n",
      "Author: Felix G.M. Ernst [aut, cre] , Denis L.J. Lafontaine [ctb, fnd]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - S4Vectors, type: depends\n",
      "  - IRanges, type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - Modstrings, type: depends\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - GenomicFeatures, type: imports\n",
      "  - GenomicAlignments, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - BSgenome, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - colorRamps, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - Gviz, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - graphics, type: imports\n",
      "  - ROCR, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/RNAmodR.html\n",
      "\n",
      "Name: satuRn\n",
      "Description:\n",
      "satuRn provides a higly performant and scalable framework for performing differential transcript usage analyses. The package consists of three main functions. The first function, fitDTU, fits quasi-binomial generalized linear models that model transcript usage in different groups of interest. The second function, testDTU, tests for differential usage of transcripts between groups of interest. Finally, plotDTU visualizes the usage profiles of transcripts in groups of interest.\n",
      "Version: 1.6.0\n",
      "Publication date: None\n",
      "Mantainer: Jeroen Gilis <jeroen.gilis@ugent.be>\n",
      "Authors:\n",
      "Author: Jeroen Gilis [aut, cre], Kristoffer Vitting-Seerup [ctb], Koen Van den Berge [ctb], Lieven Clement [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - locfdr, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - limma, type: imports\n",
      "  - pbapply, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - boot, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - stats, type: imports\n",
      "  - methods, type: imports\n",
      "  - graphics, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/satuRn.html\n",
      "\n",
      "Name: TitanCNA\n",
      "Description:\n",
      "Hidden Markov model to segment and predict regions of subclonal copy number alterations (CNA) and loss of heterozygosity (LOH), and estimate cellular prevalence of clonal clusters in tumour whole genome sequencing data.\n",
      "Version: 1.36.0\n",
      "Publication date: None\n",
      "Mantainer: Gavin Ha <gha@fredhutch.org>\n",
      "Authors:\n",
      "Author: Gavin Ha\n",
      "Dependencies:\n",
      "  - R (>= 3.5.1) type: depends\n",
      "  - BiocGenerics, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - VariantAnnotation, type: imports\n",
      "  - foreach (>= 1.4.3) type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - data.table (>= 1.10.4) type: imports\n",
      "  - dplyr (>= 0.5.0) type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/TitanCNA.html\n",
      "\n",
      "Name: a4Classif\n",
      "Description:\n",
      "Functionalities for classification of Affymetrix microarray data, integrating within the Automated Affymetrix Array Analysis set of packages.\n",
      "Version: 1.46.0\n",
      "Publication date: None\n",
      "Mantainer: Laure Cougnaud <laure.cougnaud@openanalytics.eu>\n",
      "Authors:\n",
      "Author: Willem Talloen [aut], Tobias Verbeke [aut], Laure Cougnaud [cre]\n",
      "Dependencies:\n",
      "  - a4Core, type: depends\n",
      "  - a4Preproc, type: depends\n",
      "  - methods, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - ROCR, type: imports\n",
      "  - pamr, type: imports\n",
      "  - glmnet, type: imports\n",
      "  - varSelRF, type: imports\n",
      "  - utils, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/a4Classif.html\n",
      "\n",
      "Name: AMOUNTAIN\n",
      "Description:\n",
      "A pure data-driven gene network, weighted gene co-expression network (WGCN) could be constructed only from expression profile. Different layers in such networks may represent different time points, multiple conditions or various species. AMOUNTAIN aims to search active modules in multi-layer WGCN using a continuous optimization approach.\n",
      "Version: 1.24.0\n",
      "Publication date: None\n",
      "Mantainer: Dong Li <dxl466@cs.bham.ac.uk>\n",
      "Authors:\n",
      "Author: Dong Li, Shan He, Zhisong Pan and Guyu Hu\n",
      "Dependencies:\n",
      "  - R (>= 3.3.0) type: depends\n",
      "  - stats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/AMOUNTAIN.html\n",
      "\n",
      "Name: ClassifyR\n",
      "Description:\n",
      "The software formalises a framework for classification in R. There are four stages; Data transformation, feature selection, classifier training, and prediction. The requirements of variable types and names are fixed, but specialised variables for functions can also be provided. The classification framework is wrapped in a driver loop, that reproducibly carries out a number of cross-validation schemes. Functions for differential expression, differential variability, and differential distribution are included. Additional functions may be developed by the user, by creating an interface to the framework.\n",
      "Version: 3.2.7\n",
      "Publication date: None\n",
      "Mantainer: Dario Strbenac <dario.strbenac@sydney.edu.au>\n",
      "Authors:\n",
      "Author: Dario Strbenac, Ellis Patrick, Sourish Iyengar, Harry Robertson, Andy Tran, John Ormerod, Graham Mann, Jean Yang\n",
      "Dependencies:\n",
      "  - R (>= 4.1.0) type: depends\n",
      "  - generics, type: depends\n",
      "  - methods, type: depends\n",
      "  - S4Vectors, type: depends\n",
      "  - MultiAssayExperiment, type: depends\n",
      "  - BiocParallel, type: depends\n",
      "  - survival, type: depends\n",
      "  - grid, type: imports\n",
      "  - genefilter, type: imports\n",
      "  - utils, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - rlang, type: imports\n",
      "  - ranger, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ClassifyR.html\n",
      "\n",
      "Name: CopywriteR\n",
      "Description:\n",
      "CopywriteR extracts DNA copy number information from targeted sequencing by utiizing off-target reads. It allows for extracting uniformly distributed copy number information, can be used without reference, and can be applied to sequencing data obtained from various techniques including chromatin immunoprecipitation and target enrichment on small gene panels. Thereby, CopywriteR constitutes a widely applicable alternative to available copy number detection tools.\n",
      "Version: 2.29.0\n",
      "Publication date: None\n",
      "Mantainer: Oscar Krijgsman <o.krijgsman@nki.nl>\n",
      "Authors:\n",
      "Author: Thomas Kuilman\n",
      "Dependencies:\n",
      "  - R (>= 3.2) type: depends\n",
      "  - BiocParallel, type: depends\n",
      "  - matrixStats, type: imports\n",
      "  - gtools, type: imports\n",
      "  - data.table, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - chipseq, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - DNAcopy, type: imports\n",
      "  - GenomicAlignments, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - CopyhelpeR, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - futile.logger, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/CopywriteR.html\n",
      "\n",
      "Name: CoverageView\n",
      "Description:\n",
      "This package provides a framework for the visualization of genome coverage profiles. It can be used for ChIP-seq experiments, but it can be also used for genome-wide nucleosome positioning experiments or other experiment types where it is important to have a framework in order to inspect how the coverage distributed across the genome\n",
      "Version: 1.36.0\n",
      "Publication date: None\n",
      "Mantainer: Ernesto Lowy <ernestolowy@gmail.com>\n",
      "Authors:\n",
      "Author: Ernesto Lowy\n",
      "Dependencies:\n",
      "  - R (>= 2.10) type: depends\n",
      "  - methods, type: depends\n",
      "  - Rsamtools, type: depends\n",
      "  - rtracklayer, type: depends\n",
      "  - S4Vectors, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - GenomicAlignments, type: imports\n",
      "  - parallel, type: imports\n",
      "  - tools, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/CoverageView.html\n",
      "\n",
      "Name: CRImage\n",
      "Description:\n",
      "CRImage provides functionality to process and analyze images, in particular to classify cells in biological images. Furthermore, in the context of tumor images, it provides functionality to calculate tumour cellularity.\n",
      "Version: 1.46.0\n",
      "Publication date: None\n",
      "Mantainer: Henrik Failmezger <failmezger@mpipz.mpg.de>, Yinyin Yuan <Yinyin.Yuan@cancer.org.uk>\n",
      "Authors:\n",
      "Author: Henrik Failmezger <failmezger at mpipz.mpg.de>, Yinyin Yuan <Yinyin.Yuan at cancer.org.uk>, Oscar Rueda <oscar.rueda at cancer.org.uk>, Florian Markowetz <Florian.Markowetz at cancer.org.uk>\n",
      "Dependencies:\n",
      "  - EBImage, type: depends\n",
      "  - DNAcopy, type: depends\n",
      "  - aCGH, type: depends\n",
      "  - MASS, type: imports\n",
      "  - e1071, type: imports\n",
      "  - foreach, type: imports\n",
      "  - sgeostat, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/CRImage.html\n",
      "\n",
      "Name: farms\n",
      "Description:\n",
      "The package provides the summarization algorithm called Factor Analysis for Robust Microarray Summarization (FARMS) and a novel unsupervised feature selection criterion called \"I/NI-calls\"\n",
      "Version: 1.50.0\n",
      "Publication date: None\n",
      "Mantainer: Djork-Arne Clevert <okko@clevert.de>\n",
      "Authors:\n",
      "Author: Djork-Arne Clevert <okko at clevert.de>\n",
      "Dependencies:\n",
      "  - R (>= 2.8) type: depends\n",
      "  - affy, type: depends\n",
      "  - MASS, type: depends\n",
      "  - methods, type: depends\n",
      "  - affy, type: imports\n",
      "  - MASS, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - methods, type: imports\n",
      "  - graphics, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL (>= 2.1)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/farms.html\n",
      "\n",
      "Name: ffpe\n",
      "Description:\n",
      "Identify low-quality data using metrics developed for expression data derived from Formalin-Fixed, Paraffin-Embedded (FFPE) data. Also a function for making Concordance at the Top plots (CAT-plots).\n",
      "Version: 1.42.0\n",
      "Publication date: None\n",
      "Mantainer: Levi Waldron <lwaldron.research@gmail.com>\n",
      "Authors:\n",
      "Author: Levi Waldron\n",
      "Dependencies:\n",
      "  - R (>= 2.10.0) type: depends\n",
      "  - TTR, type: depends\n",
      "  - methods, type: depends\n",
      "  - Biobase, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - affy, type: imports\n",
      "  - lumi, type: imports\n",
      "  - methylumi, type: imports\n",
      "  - sfsmisc, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ffpe.html\n",
      "\n",
      "Name: GeoDiff\n",
      "Description:\n",
      "A series of statistical models using count generating distributions for background modelling, feature and sample QC, normalization and differential expression analysis on GeoMx RNA data. The application of these methods are demonstrated by example data analysis vignette.\n",
      "Version: 1.4.2\n",
      "Publication date: None\n",
      "Mantainer: Nicole Ortogero <nortogero@nanostring.com>\n",
      "Authors:\n",
      "Author: Nicole Ortogero [cre], Lei Yang [aut], Zhi Yang [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.1.0) type: depends\n",
      "  - Biobase, type: depends\n",
      "  - Matrix, type: imports\n",
      "  - robust, type: imports\n",
      "  - plyr, type: imports\n",
      "  - lme4, type: imports\n",
      "  - Rcpp (>= 1.0.4.6) type: imports\n",
      "  - withr, type: imports\n",
      "  - methods, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - testthat, type: imports\n",
      "  - GeomxTools, type: imports\n",
      "  - NanoStringNCTools, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/GeoDiff.html\n",
      "\n",
      "Name: gespeR\n",
      "Description:\n",
      "Estimates gene-specific phenotypes from off-target confounded RNAi screens. The phenotype of each siRNA is modeled based on on-targeted and off-targeted genes, using a regularized linear regression model.\n",
      "Version: 1.30.0\n",
      "Publication date: None\n",
      "Mantainer: Fabian Schmich <fabian.schmich@bsse.ethz.ch>\n",
      "Authors:\n",
      "Author: Fabian Schmich\n",
      "Dependencies:\n",
      "  - methods, type: depends\n",
      "  - graphics, type: depends\n",
      "  - ggplot2, type: depends\n",
      "  - R (>= 2.10) type: depends\n",
      "  - Matrix, type: imports\n",
      "  - glmnet, type: imports\n",
      "  - cellHTS2, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - biomaRt, type: imports\n",
      "  - doParallel, type: imports\n",
      "  - parallel, type: imports\n",
      "  - foreach, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - dplyr, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/gespeR.html\n",
      "\n",
      "Name: ggspavis\n",
      "Description:\n",
      "Visualization functions for spatially resolved transcriptomics datasets stored in SpatialExperiment format. Includes functions to create several types of plots for data from from spot-based (e.g. 10x Genomics Visium) and molecule-based (e.g. seqFISH) technological platforms.\n",
      "Version: 1.4.0\n",
      "Publication date: None\n",
      "Mantainer: Lukas M. Weber <lukas.weber.edu@gmail.com>\n",
      "Authors:\n",
      "Author: Lukas M. Weber [aut, cre] , Helena L. Crowell [aut]\n",
      "Dependencies:\n",
      "  - ggplot2, type: depends\n",
      "  - SpatialExperiment, type: imports\n",
      "  - SingleCellExperiment, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - ggside, type: imports\n",
      "  - grid, type: imports\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ggspavis.html\n",
      "\n",
      "Name: hiAnnotator\n",
      "Description:\n",
      "hiAnnotator contains set of functions which allow users to annotate a GRanges object with custom set of annotations. The basic philosophy of this package is to take two GRanges objects (query & subject) with common set of seqnames (i.e. chromosomes) and return associated annotation per seqnames and rows from the query matching seqnames and rows from the subject (i.e. genes or cpg islands). The package comes with three types of annotation functions which calculates if a position from query is: within a feature, near a feature, or count features in defined window sizes. Moreover, each function is equipped with parallel backend to utilize the foreach package. In addition, the package is equipped with wrapper functions, which finds appropriate columns needed to make a GRanges object from a common data frame.\n",
      "Version: 1.32.0\n",
      "Publication date: None\n",
      "Mantainer: Nirav V Malani <malnirav@gmail.com>\n",
      "Authors:\n",
      "Author: Nirav V Malani <malnirav at gmail.com>\n",
      "Dependencies:\n",
      "  - GenomicRanges, type: depends\n",
      "  - R (>= 2.10) type: depends\n",
      "  - foreach, type: imports\n",
      "  - iterators, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - BSgenome, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - scales, type: imports\n",
      "  - methods, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/hiAnnotator.html\n",
      "\n",
      "Name: LymphoSeq\n",
      "Description:\n",
      "This R package analyzes high-throughput sequencing of T and B cell receptor complementarity determining region 3 (CDR3) sequences generated by Adaptive Biotechnologies' ImmunoSEQ assay. Its input comes from tab-separated value (.tsv) files exported from the ImmunoSEQ analyzer.\n",
      "Version: 1.26.0\n",
      "Publication date: None\n",
      "Mantainer: David Coffey <dcoffey@fredhutch.org>\n",
      "Authors:\n",
      "Author: David Coffey <dcoffey at fredhutch.org>\n",
      "Dependencies:\n",
      "  - R (>= 3.3) type: depends\n",
      "  - LymphoSeqDB, type: depends\n",
      "  - data.table, type: imports\n",
      "  - plyr, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - reshape, type: imports\n",
      "  - VennDiagram, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - ineq, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - circlize, type: imports\n",
      "  - grid, type: imports\n",
      "  - utils, type: imports\n",
      "  - stats, type: imports\n",
      "  - ggtree, type: imports\n",
      "  - msa, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - phangorn, type: imports\n",
      "  - stringdist, type: imports\n",
      "  - UpSetR, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/LymphoSeq.html\n",
      "\n",
      "Name: MACSr\n",
      "Description:\n",
      "The Model-based Analysis of ChIP-Seq (MACS) is a widely used toolkit for identifying transcript factor binding sites. This package is an R wrapper of the lastest MACS3.\n",
      "Version: 1.6.0\n",
      "Publication date: None\n",
      "Mantainer: Qiang Hu <Qiang.Hu@roswellpark.org>\n",
      "Authors:\n",
      "Author: Qiang Hu [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.1.0) type: depends\n",
      "  - utils, type: imports\n",
      "  - reticulate, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - methods, type: imports\n",
      "  - basilisk, type: imports\n",
      "  - ExperimentHub, type: imports\n",
      "  - AnnotationHub, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "BSD_3_clause + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/MACSr.html\n",
      "\n",
      "Name: methrix\n",
      "Description:\n",
      "Bedgraph files generated by Bisulfite pipelines often come in various flavors. Critical downstream step requires summarization of these files into methylation/coverage matrices. This step of data aggregation is done by Methrix, including many other useful downstream functions.\n",
      "Version: 1.12.0\n",
      "Publication date: None\n",
      "Mantainer: Anand Mayakonda <anand_mt@hotmail.com>\n",
      "Authors:\n",
      "Author: Anand Mayakonda [aut, cre] , Reka Toth [aut] , Rajbir Batra [ctb], Clarissa Feuerstein-AkgÃ¶z [ctb], Joschka Hey [ctb], Maximilian SchÃ¶nung [ctb], Pavlo Lutsik [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 3.6) type: depends\n",
      "  - data.table (>= 1.12.4) type: depends\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - rtracklayer, type: imports\n",
      "  - DelayedArray, type: imports\n",
      "  - HDF5Array, type: imports\n",
      "  - BSgenome, type: imports\n",
      "  - DelayedMatrixStats, type: imports\n",
      "  - parallel, type: imports\n",
      "  - methods, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - IRanges, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/methrix.html\n",
      "\n",
      "Name: MsBackendMassbank\n",
      "Description:\n",
      "Mass spectrometry (MS) data backend supporting import and export of MS/MS library spectra from MassBank record files. Different backends are available that allow handling of data in plain MassBank text file format or allow also to interact directly with MassBank SQL databases. Objects from this package are supposed to be used with the Spectra Bioconductor package. This package thus adds MassBank support to the Spectra package.\n",
      "Version: 1.6.1\n",
      "Publication date: None\n",
      "Mantainer: RforMassSpectrometry Package Maintainer <maintainer@rformassspectrometry.org>\n",
      "Authors:\n",
      "Author: RforMassSpectrometry Package Maintainer [cre], Michael Witting [aut] , Johannes Rainer [aut] , Michael Stravs [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - Spectra, type: depends\n",
      "  - BiocParallel, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - methods, type: imports\n",
      "  - ProtGenerics, type: imports\n",
      "  - MsCoreUtils, type: imports\n",
      "  - DBI, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/MsBackendMassbank.html\n",
      "\n",
      "Name: ndexr\n",
      "Description:\n",
      "This package offers an interface to NDEx servers, e.g. the public server at http://ndexbio.org/. It can retrieve and save networks via the API. Networks are offered as RCX object and as igraph representation.\n",
      "Version: 1.20.1\n",
      "Publication date: None\n",
      "Mantainer: Florian Auer <florian.auer@informatik.uni-augsburg.de>\n",
      "Authors:\n",
      "Author: Florian Auer <florian.auer at informatik.uni-augsburg.de>, Frank Kramer <frank.kramer at informatik.uni-augsburg.de>, Alex Ishkin <aleksandr.ishkin at thomsonreuters.com>, Dexter Pratt <depratt at ucsc.edu>\n",
      "Dependencies:\n",
      "  - RCX, type: depends\n",
      "  - httr, type: imports\n",
      "  - jsonlite, type: imports\n",
      "  - plyr, type: imports\n",
      "  - tidyr, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "BSD_3_clause\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ndexr.html\n",
      "\n",
      "Name: oposSOM\n",
      "Description:\n",
      "This package translates microarray expression data into metadata of reduced dimension. It provides various sample-centered and group-centered visualizations, sample similarity analyses and functional enrichment analyses. The underlying SOM algorithm combines feature clustering, multidimensional scaling and dimension reduction, along with strong visualization capabilities. It enables extraction and description of functional expression modules inherent in the data.\n",
      "Version: 2.16.0\n",
      "Publication date: None\n",
      "Mantainer: Henry Loeffler-Wirth <wirth@izbi.uni-leipzig.de>\n",
      "Authors:\n",
      "Author: Henry Loeffler-Wirth <wirth at izbi.uni-leipzig.de>, Hoang Thanh Le <le at izbi.uni-leipzig.de> and Martin Kalcher <mkalcher at porkbox.net>\n",
      "Dependencies:\n",
      "  - R (>= 4.0.0) type: depends\n",
      "  - igraph (>= 1.0.0) type: depends\n",
      "  - fastICA, type: imports\n",
      "  - tsne, type: imports\n",
      "  - scatterplot3d, type: imports\n",
      "  - pixmap, type: imports\n",
      "  - fdrtool, type: imports\n",
      "  - ape, type: imports\n",
      "  - biomaRt, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - RcppParallel, type: imports\n",
      "  - Rcpp, type: imports\n",
      "  - methods, type: imports\n",
      "  - graph, type: imports\n",
      "  - XML, type: imports\n",
      "  - png, type: imports\n",
      "  - RCurl, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>=2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/oposSOM.html\n",
      "\n",
      "Name: rDGIdb\n",
      "Description:\n",
      "The rDGIdb package provides a wrapper for the Drug Gene Interaction Database (DGIdb). For simplicity, the wrapper query function and output resembles the user interface and results format provided on the DGIdb website (https://www.dgidb.org/).\n",
      "Version: 1.24.0\n",
      "Publication date: None\n",
      "Mantainer: Lars Bosshard <bosshard@nexus.ethz.ch>\n",
      "Authors:\n",
      "Author: Thomas Thurnherr, Franziska Singer, Daniel J. Stekhoven, and Niko Beerenwinkel\n",
      "Dependencies:\n",
      "  - jsonlite, type: imports\n",
      "  - httr, type: imports\n",
      "  - methods, type: imports\n",
      "  - graphics, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/rDGIdb.html\n",
      "\n",
      "Name: scp\n",
      "Description:\n",
      "Utility functions for manipulating, processing, and analyzing mass spectrometry-based single-cell proteomics data. The package is an extension to the 'QFeatures' package and relies on 'SingleCellExpirement' to enable single-cell proteomics analyses. The package offers the user the functionality to process quantitative table (as generated by MaxQuant, Proteome Discoverer, and more) into data tables ready for downstream analysis and data visualization.\n",
      "Version: 1.8.0\n",
      "Publication date: None\n",
      "Mantainer: Christophe Vanderaa <christophe.vanderaa@uclouvain.be>\n",
      "Authors:\n",
      "Author: Christophe Vanderaa [aut, cre] , Laurent Gatto [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.2.0) type: depends\n",
      "  - QFeatures, type: depends\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - SingleCellExperiment, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - MultiAssayExperiment, type: imports\n",
      "  - MsCoreUtils, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - magrittr, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/scp.html\n",
      "\n",
      "Name: signeR\n",
      "Description:\n",
      "The signeR package provides an empirical Bayesian approach to mutational signature discovery. It is designed to analyze single nucleotide variaton (SNV) counts in cancer genomes, but can also be applied to other features as well. Functionalities to characterize signatures or genome samples according to exposure patterns are also provided.\n",
      "Version: 2.0.2\n",
      "Publication date: None\n",
      "Mantainer: Renan Valieris <renan.valieris@accamargo.org.br>\n",
      "Authors:\n",
      "Author: Rafael Rosales, Rodrigo Drummond, Renan Valieris, Israel Tojal da Silva\n",
      "Dependencies:\n",
      "  - R (>= 3.0.2) type: depends\n",
      "  - VariantAnnotation, type: depends\n",
      "  - NMF, type: depends\n",
      "  - BiocGenerics, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - class, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - nloptr, type: imports\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - PMCMRplus, type: imports\n",
      "  - parallel, type: imports\n",
      "  - pvclust, type: imports\n",
      "  - ppclust, type: imports\n",
      "  - clue, type: imports\n",
      "  - survival, type: imports\n",
      "  - maxstat, type: imports\n",
      "  - survivalAnalysis, type: imports\n",
      "  - future, type: imports\n",
      "  - VGAM, type: imports\n",
      "  - MASS, type: imports\n",
      "  - kknn, type: imports\n",
      "  - glmnet, type: imports\n",
      "  - e1071, type: imports\n",
      "  - randomForest, type: imports\n",
      "  - ada, type: imports\n",
      "  - future.apply, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - pROC, type: imports\n",
      "  - pheatmap, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - listenv, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - scales, type: imports\n",
      "  - survminer, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - ggpubr, type: imports\n",
      "  - cowplot, type: imports\n",
      "  - tibble, type: imports\n",
      "  - readr, type: imports\n",
      "  - shiny, type: imports\n",
      "  - shinydashboard, type: imports\n",
      "  - shinycssloaders, type: imports\n",
      "  - shinyWidgets, type: imports\n",
      "  - bsplus, type: imports\n",
      "  - DT, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - BiocFileCache, type: imports\n",
      "  - proxy, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - BSgenome, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/signeR.html\n",
      "\n",
      "Name: ADImpute\n",
      "Description:\n",
      "Single-cell RNA sequencing (scRNA-seq) methods are typically unable to quantify the expression levels of all genes in a cell, creating a need for the computational prediction of missing values (âdropout imputationâ). Most existing dropout imputation methods are limited in the sense that they exclusively use the scRNA-seq dataset at hand and do not exploit external gene-gene relationship information. Here we propose two novel methods: a gene regulatory network-based approach using gene-gene relationships learnt from external data and a baseline approach corresponding to a sample-wide average. ADImpute can implement these novel methods and also combine them with existing imputation methods (currently supported: DrImpute, SAVER). ADImpute can learn the best performing method per gene and combine the results from different methods into an ensemble.\n",
      "Version: 1.8.0\n",
      "Publication date: None\n",
      "Mantainer: Ana Carolina Leote <anacarolinaleote@gmail.com>\n",
      "Authors:\n",
      "Author: Ana Carolina Leote [cre, aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - checkmate, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - data.table, type: imports\n",
      "  - DrImpute, type: imports\n",
      "  - kernlab, type: imports\n",
      "  - MASS, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - methods, type: imports\n",
      "  - rsvd, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - SAVER, type: imports\n",
      "  - SingleCellExperiment, type: imports\n",
      "  - stats, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3 + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ADImpute.html\n",
      "\n",
      "Name: CeTF\n",
      "Description:\n",
      "This package provides the necessary functions for performing the Partial Correlation coefficient with Information Theory (PCIT) (Reverter and Chan 2008) and Regulatory Impact Factors (RIF) (Reverter et al. 2010) algorithm. The PCIT algorithm identifies meaningful correlations to define edges in a weighted network and can be applied to any correlation-based network including but not limited to gene co-expression networks, while the RIF algorithm identify critical Transcription Factors (TF) from gene expression data. These two algorithms when combined provide a very relevant layer of information for gene expression studies (Microarray, RNA-seq and single-cell RNA-seq data).\n",
      "Version: 1.10.2\n",
      "Publication date: None\n",
      "Mantainer: Carlos Alberto Oliveira de Biagi Junior <cbiagijr@gmail.com>\n",
      "Authors:\n",
      "Author: Carlos Alberto Oliveira de Biagi Junior [aut, cre], Ricardo Perecin Nociti [aut], Breno Osvaldo Funicheli [aut], JoÃ£o Paulo Bianchi Ximenez [ctb], PatrÃ­cia de CÃ¡ssia Ruy [ctb], Marcelo Gomes de Paula [ctb], Rafael dos Santos Bezerra [ctb], Wilson AraÃºjo da Silva Junior [aut, ths]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - circlize, type: imports\n",
      "  - ComplexHeatmap, type: imports\n",
      "  - clusterProfiler, type: imports\n",
      "  - DESeq2, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - GenomicTools.fileHandler, type: imports\n",
      "  - GGally, type: imports\n",
      "  - ggnetwork, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - ggpubr, type: imports\n",
      "  - ggrepel, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grid, type: imports\n",
      "  - igraph, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - network, type: imports\n",
      "  - Rcpp, type: imports\n",
      "  - RCy3, type: imports\n",
      "  - stats, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - utils, type: imports\n",
      "  - methods, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/CeTF.html\n",
      "\n",
      "Name: CHETAH\n",
      "Description:\n",
      "CHETAH (CHaracterization of cEll Types Aided by Hierarchical classification) is an accurate, selective and fast scRNA-seq classifier. Classification is guided by a reference dataset, preferentially also a scRNA-seq dataset. By hierarchical clustering of the reference data, CHETAH creates a classification tree that enables a step-wise, top-to-bottom classification. Using a novel stopping rule, CHETAH classifies the input cells to the cell types of the references and to \"intermediate types\": more general classifications that ended in an intermediate node of the tree.\n",
      "Version: 1.14.0\n",
      "Publication date: None\n",
      "Mantainer: Jurrian de Kanter <jurriandekanter@gmail.com>\n",
      "Authors:\n",
      "Author: Jurrian de Kanter [aut, cre], Philip Lijnzaad [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.2) type: depends\n",
      "  - ggplot2, type: depends\n",
      "  - SingleCellExperiment, type: depends\n",
      "  - shiny, type: imports\n",
      "  - plotly, type: imports\n",
      "  - pheatmap, type: imports\n",
      "  - bioDist, type: imports\n",
      "  - dendextend, type: imports\n",
      "  - cowplot, type: imports\n",
      "  - corrplot, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - stats, type: imports\n",
      "  - graphics, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/CHETAH.html\n",
      "\n",
      "Name: COMPASS\n",
      "Description:\n",
      "COMPASS is a statistical framework that enables unbiased analysis of antigen-specific T-cell subsets. COMPASS uses a Bayesian hierarchical framework to model all observed cell-subsets and select the most likely to be antigen-specific while regularizing the small cell counts that often arise in multi-parameter space. The model provides a posterior probability of specificity for each cell subset and each sample, which can be used to profile a subject's immune response to external stimuli such as infection or vaccination.\n",
      "Version: 1.36.2\n",
      "Publication date: None\n",
      "Mantainer: Greg Finak <gfinak@fhcrc.org>\n",
      "Authors:\n",
      "Author: Lynn Lin, Kevin Ushey, Greg Finak, Ravio Kolde (pheatmap)\n",
      "Dependencies:\n",
      "  - R (>= 3.0.3) type: depends\n",
      "  - methods, type: imports\n",
      "  - Rcpp, type: imports\n",
      "  - data.table, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - scales, type: imports\n",
      "  - grid, type: imports\n",
      "  - plyr, type: imports\n",
      "  - knitr, type: imports\n",
      "  - abind, type: imports\n",
      "  - clue, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - utils, type: imports\n",
      "  - pdist, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - rlang, type: imports\n",
      "  - BiocStyle, type: imports\n",
      "  - rmarkdown, type: imports\n",
      "  - foreach, type: imports\n",
      "  - coda, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/COMPASS.html\n",
      "\n",
      "Name: condiments\n",
      "Description:\n",
      "This package encapsulate many functions to conduct a differential topology analysis. It focuses on analyzing an 'omic dataset with multiple conditions. While the package is mostly geared toward scRNASeq, it does not place any restriction on the actual input format.\n",
      "Version: 1.6.0\n",
      "Publication date: None\n",
      "Mantainer: Hector Roux de Bezieux <hector.rouxdebezieux@berkeley.edu>\n",
      "Authors:\n",
      "Author: Hector Roux de Bezieux [aut, cre] , Koen Van den Berge [aut, ctb], Kelly Street [aut, ctb]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - slingshot, type: imports\n",
      "  - mgcv, type: imports\n",
      "  - RANN, type: imports\n",
      "  - stats, type: imports\n",
      "  - SingleCellExperiment, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - utils, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - dplyr (>= 1.0) type: imports\n",
      "  - Ecume (>= 0.9.1) type: imports\n",
      "  - methods, type: imports\n",
      "  - pbapply, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - TrajectoryUtils, type: imports\n",
      "  - igraph, type: imports\n",
      "  - distinct, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/condiments.html\n",
      "\n",
      "Name: corral\n",
      "Description:\n",
      "Correspondence analysis (CA) is a matrix factorization method, and is similar to principal components analysis (PCA). Whereas PCA is designed for application to continuous, approximately normally distributed data, CA is appropriate for non-negative, count-based data that are in the same additive scale. The corral package implements CA for dimensionality reduction of a single matrix of single-cell data, as well as a multi-table adaptation of CA that leverages data-optimized scaling to align data generated from different sequencing platforms by projecting into a shared latent space. corral utilizes sparse matrices and a fast implementation of SVD, and can be called directly on Bioconductor objects (e.g., SingleCellExperiment) for easy pipeline integration. The package also includes additional options, including variations of CA to address overdispersion in count data, as well as the option to apply CA-style processing to continuous data (e.g., proteomic TOF intensities) with the Hellinger distance adaptation of CA.\n",
      "Version: 1.8.0\n",
      "Publication date: None\n",
      "Mantainer: Lauren Hsu <lrnshoe@gmail.com>\n",
      "Authors:\n",
      "Author: Lauren Hsu [aut, cre] , Aedin Culhane [aut]\n",
      "Dependencies:\n",
      "  - ggplot2, type: imports\n",
      "  - ggthemes, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - gridExtra, type: imports\n",
      "  - irlba, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - methods, type: imports\n",
      "  - MultiAssayExperiment, type: imports\n",
      "  - pals, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - SingleCellExperiment, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - transport, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/corral.html\n",
      "\n",
      "Name: DeMixT\n",
      "Description:\n",
      "DeMixT is a software package that performs deconvolution on transcriptome data from a mixture of two or three components.\n",
      "Version: 1.14.0\n",
      "Publication date: None\n",
      "Mantainer: Shuai Guo <shuai.guo@uth.tmc.edu>\n",
      "Authors:\n",
      "Author: Zeya Wang <zw17.rice at gmail.com>, Shaolong Cao<scao at mdanderson.org>, Wenyi Wang <wwang7 at at mdanderson.org>\n",
      "Dependencies:\n",
      "  - R (>= 3.6.0) type: depends\n",
      "  - parallel, type: depends\n",
      "  - Rcpp (>= 1.0.0) type: depends\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - knitr, type: depends\n",
      "  - KernSmooth, type: depends\n",
      "  - matrixcalc, type: depends\n",
      "  - rmarkdown, type: depends\n",
      "  - DSS, type: depends\n",
      "  - dendextend, type: depends\n",
      "  - psych, type: depends\n",
      "  - sva, type: depends\n",
      "  - matrixStats, type: imports\n",
      "  - stats, type: imports\n",
      "  - truncdist, type: imports\n",
      "  - base64enc, type: imports\n",
      "  - ggplot2, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/DeMixT.html\n",
      "\n",
      "Name: easyRNASeq\n",
      "Description:\n",
      "Calculates the coverage of high-throughput short-reads against a genome of reference and summarizes it per feature of interest (e.g. exon, gene, transcript). The data can be normalized as 'RPKM' or by the 'DESeq' or 'edgeR' package.\n",
      "Version: 2.34.0\n",
      "Publication date: None\n",
      "Mantainer: Nicolas Delhomme <nicolas.delhomme@umu.se>\n",
      "Authors:\n",
      "Author: Nicolas Delhomme, Ismael Padioleau, Bastian Schiffthaler, Niklas Maehler\n",
      "Dependencies:\n",
      "  - Biobase, type: imports\n",
      "  - BiocFileCache, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - biomaRt, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - edgeR, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - genomeIntervals, type: imports\n",
      "  - GenomicAlignments, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - graphics, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - LSD (>= 4.1-0) type: imports\n",
      "  - locfit, type: imports\n",
      "  - methods, type: imports\n",
      "  - parallel, type: imports\n",
      "  - rappdirs (>= 0.3.1) type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - ShortRead, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/easyRNASeq.html\n",
      "\n",
      "Name: GeneStructureTools\n",
      "Description:\n",
      "GeneStructureTools can be used to create in silico alternative splicing events, and analyse potential effects this has on functional gene products.\n",
      "Version: 1.18.0\n",
      "Publication date: None\n",
      "Mantainer: Beth Signal <b.signal@garvan.org.au>\n",
      "Authors:\n",
      "Author: Beth Signal\n",
      "Dependencies:\n",
      "  - Biostrings, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - data.table, type: imports\n",
      "  - plyr, type: imports\n",
      "  - stringdist, type: imports\n",
      "  - stringr, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - BSgenome.Mmusculus.UCSC.mm10, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - Gviz, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - methods, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "BSD_3_clause + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/GeneStructureTools.html\n",
      "\n",
      "Name: mCSEA\n",
      "Description:\n",
      "Identification of diferentially methylated regions (DMRs) in predefined regions (promoters, CpG islands...) from the human genome using Illumina's 450K or EPIC microarray data. Provides methods to rank CpG probes based on linear models and includes plotting functions.\n",
      "Version: 1.18.0\n",
      "Publication date: None\n",
      "Mantainer: Jordi Martorell-MarugÃ¡n <jmartorellm@gmail.com>\n",
      "Authors:\n",
      "Author: Jordi Martorell-MarugÃ¡n and Pedro Carmona-SÃ¡ez\n",
      "Dependencies:\n",
      "  - R (>= 3.5) type: depends\n",
      "  - mCSEAdata, type: depends\n",
      "  - Homo.sapiens, type: depends\n",
      "  - biomaRt, type: imports\n",
      "  - fgsea, type: imports\n",
      "  - GenomicFeatures, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - Gviz, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - limma, type: imports\n",
      "  - methods, type: imports\n",
      "  - parallel, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - stats, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/mCSEA.html\n",
      "\n",
      "Name: metagene\n",
      "Description:\n",
      "This package produces metagene plots to compare the behavior of DNA-interacting proteins at selected groups of genes/features. Bam files are used to increase the resolution. Multiple combination of group of bam files and/or group of genomic regions can be compared in a single analysis. Bootstraping analysis is used to compare the groups and locate regions with statistically different enrichment profiles.\n",
      "Version: 2.30.0\n",
      "Publication date: None\n",
      "Mantainer: Charles Joly Beauparlant <charles.joly-beauparlant@crchul.ulaval.ca>\n",
      "Authors:\n",
      "Author: Charles Joly Beauparlant <charles.joly-beauparlant at crchul.ulaval.ca>, Fabien Claude Lamaze <fabien.lamaze.1 at ulaval.ca>, Rawane Samb <rawane.samb.1 at ulaval.ca>, Cedric Lippens <lippens.cedric at protonmail>, Astrid Louise Deschenes <Astrid-Louise.Deschenes at crchudequebec.ulaval.ca> and Arnaud Droit <arnaud.droit at crchuq.ulaval.ca>.\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - R6 (>= 2.0) type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - BiocParallel, type: depends\n",
      "  - rtracklayer, type: imports\n",
      "  - gplots, type: imports\n",
      "  - tools, type: imports\n",
      "  - GenomicAlignments, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - GenomicFeatures, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - purrr, type: imports\n",
      "  - data.table, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - methods, type: imports\n",
      "  - utils, type: imports\n",
      "  - ensembldb, type: imports\n",
      "  - EnsDb.Hsapiens.v86, type: imports\n",
      "  - stringr, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0 | file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/metagene.html\n",
      "\n",
      "Name: multiClust\n",
      "Description:\n",
      "Clustering is carried out to identify patterns in transcriptomics profiles to determine clinically relevant subgroups of patients. Feature (gene) selection is a critical and an integral part of the process. Currently, there are many feature selection and clustering methods to identify the relevant genes and perform clustering of samples. However, choosing an appropriate methodology is difficult. In addition, extensive feature selection methods have not been supported by the available packages. Hence, we developed an integrative R-package called multiClust that allows researchers to experiment with the choice of combination of methods for gene selection and clustering with ease. Using multiClust, we identified the best performing clustering methodology in the context of clinical outcome. Our observations demonstrate that simple methods such as variance-based ranking perform well on the majority of data sets, provided that the appropriate number of genes is selected. However, different gene ranking and selection methods remain relevant as no methodology works for all studies.\n",
      "Version: 1.28.0\n",
      "Publication date: None\n",
      "Mantainer: Nathan Lawlor <nathan.lawlor03@gmail.com>\n",
      "Authors:\n",
      "Author: Nathan Lawlor [aut, cre], Peiyong Guan [aut], Alec Fabbri [aut], Krish Karuturi [aut], Joshy George [aut]\n",
      "Dependencies:\n",
      "  - mclust, type: imports\n",
      "  - ctc, type: imports\n",
      "  - survival, type: imports\n",
      "  - cluster, type: imports\n",
      "  - dendextend, type: imports\n",
      "  - amap, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/multiClust.html\n",
      "\n",
      "Name: panelcn.mops\n",
      "Description:\n",
      "CNV detection tool for targeted NGS panel data. Extension of the cn.mops package.\n",
      "Version: 1.20.0\n",
      "Publication date: None\n",
      "Mantainer: Gundula Povysil <povysil@bioinf.jku.at>\n",
      "Authors:\n",
      "Author: Verena Haunschmid [aut], Gundula Povysil [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - cn.mops, type: depends\n",
      "  - methods, type: depends\n",
      "  - utils, type: depends\n",
      "  - stats, type: depends\n",
      "  - graphics, type: depends\n",
      "  - GenomicRanges, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - grDevices, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL (>= 2.0)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/panelcn.mops.html\n",
      "\n",
      "Name: plgem\n",
      "Description:\n",
      "The Power Law Global Error Model (PLGEM) has been shown to faithfully model the variance-versus-mean dependence that exists in a variety of genome-wide datasets, including microarray and proteomics data. The use of PLGEM has been shown to improve the detection of differentially expressed genes or proteins in these datasets.\n",
      "Version: 1.70.0\n",
      "Publication date: None\n",
      "Mantainer: Norman Pavelka <normanpavelka@gmail.com>\n",
      "Authors:\n",
      "Author: Mattia Pelizzola <mattia.pelizzola at gmail.com> and Norman Pavelka <normanpavelka at gmail.com>\n",
      "Dependencies:\n",
      "  - R (>= 2.10) type: depends\n",
      "  - utils, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - MASS, type: imports\n",
      "  - methods, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/plgem.html\n",
      "\n",
      "Name: scAnnotatR\n",
      "Description:\n",
      "The package comprises a set of pretrained machine learning models to predict basic immune cell types. This enables all users to quickly get a first annotation of the cell types present in their dataset without requiring prior knowledge. scAnnotatR also allows users to train their own models to predict new cell types based on specific research needs.\n",
      "Version: 1.4.0\n",
      "Publication date: None\n",
      "Mantainer: Johannes Griss <johannes.griss@meduniwien.ac.at>\n",
      "Authors:\n",
      "Author: Vy Nguyen [aut] , Johannes Griss [cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - Seurat, type: depends\n",
      "  - SingleCellExperiment, type: depends\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - dplyr, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - caret, type: imports\n",
      "  - ROCR, type: imports\n",
      "  - pROC, type: imports\n",
      "  - data.tree, type: imports\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - e1071, type: imports\n",
      "  - ape, type: imports\n",
      "  - kernlab, type: imports\n",
      "  - AnnotationHub, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/scAnnotatR.html\n",
      "\n",
      "Name: timescape\n",
      "Description:\n",
      "TimeScape is an automated tool for navigating temporal clonal evolution data. The key attributes of this implementation involve the enumeration of clones, their evolutionary relationships and their shifting dynamics over time. TimeScape requires two inputs: (i) the clonal phylogeny and (ii) the clonal prevalences. Optionally, TimeScape accepts a data table of targeted mutations observed in each clone and their allele prevalences over time. The output is the TimeScape plot showing clonal prevalence vertically, time horizontally, and the plot height optionally encoding tumour volume during tumour-shrinking events. At each sampling time point (denoted by a faint white line), the height of each clone accurately reflects its proportionate prevalence. These prevalences form the anchors for bezier curves that visually represent the dynamic transitions between time points.\n",
      "Version: 1.22.0\n",
      "Publication date: None\n",
      "Mantainer: Maia Smith <maiaannesmith@gmail.com>\n",
      "Authors:\n",
      "Author: Maia Smith [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 3.3) type: depends\n",
      "  - htmlwidgets (>= 0.5) type: imports\n",
      "  - jsonlite (>= 0.9.19) type: imports\n",
      "  - stringr (>= 1.0.0) type: imports\n",
      "  - dplyr (>= 0.4.3) type: imports\n",
      "  - gtools (>= 3.5.0) type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/timescape.html\n",
      "\n",
      "Name: affylmGUI\n",
      "Description:\n",
      "A Graphical User Interface (GUI) for analysis of Affymetrix microarray gene expression data using the affy and limma packages.\n",
      "Version: 1.72.0\n",
      "Publication date: None\n",
      "Mantainer: Gordon Smyth <smyth@wehi.edu.au>\n",
      "Authors:\n",
      "Author: James Wettenhall [cre,aut], Gordon Smyth [aut], Ken Simpson [aut], Keith Satterley [ctb]\n",
      "Dependencies:\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - tcltk, type: imports\n",
      "  - tkrplot, type: imports\n",
      "  - limma, type: imports\n",
      "  - affy, type: imports\n",
      "  - affyio, type: imports\n",
      "  - affyPLM, type: imports\n",
      "  - gcrma, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - AnnotationDbi, type: imports\n",
      "  - BiocManager, type: imports\n",
      "  - R2HTML, type: imports\n",
      "  - xtable, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>=2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/affylmGUI.html\n",
      "\n",
      "Name: AlpsNMR\n",
      "Description:\n",
      "Reads Bruker NMR data directories both zipped and unzipped. It provides automated and efficient signal processing for untargeted NMR metabolomics. It is able to interpolate the samples, detect outliers, exclude regions, normalize, detect peaks, align the spectra, integrate peaks, manage metadata and visualize the spectra. After spectra proccessing, it can apply multivariate analysis on extracted data. Efficient plotting with 1-D data is also available. Basic reading of 1D ACD/Labs exported JDX samples is also available.\n",
      "Version: 4.0.4\n",
      "Publication date: None\n",
      "Mantainer: Sergio Oller Moreno <sergioller@gmail.com>\n",
      "Authors:\n",
      "Author: Ivan Montoliu Roura [aut], Sergio Oller Moreno [aut, cre] , Francisco Madrid Gambin [aut] , Luis Fernandez [aut] , Laura LÃ³pez SÃ¡nchez [ctb], HÃ©ctor Gracia Cabrera [aut], Santiago Marco ColÃ¡s [aut] , NestlÃ© Institute of Health Sciences [cph], Institute for Bioengineering of Catalonia [cph]\n",
      "Dependencies:\n",
      "  - R (>= 4.2) type: depends\n",
      "  - future (>= 1.10.0) type: depends\n",
      "  - utils, type: imports\n",
      "  - generics, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - cli, type: imports\n",
      "  - magrittr (>= 1.5) type: imports\n",
      "  - dplyr (>= 1.1.0) type: imports\n",
      "  - signal (>= 0.7-6) type: imports\n",
      "  - rlang (>= 0.3.0.1) type: imports\n",
      "  - scales (>= 1.2.0) type: imports\n",
      "  - stringr (>= 1.3.1) type: imports\n",
      "  - tibble (>= 1.3.4) type: imports\n",
      "  - tidyr (>= 1.0.0) type: imports\n",
      "  - tidyselect, type: imports\n",
      "  - readxl (>= 1.1.0) type: imports\n",
      "  - purrr (>= 0.2.5) type: imports\n",
      "  - glue (>= 1.2.0) type: imports\n",
      "  - reshape2 (>= 1.4.3) type: imports\n",
      "  - mixOmics, type: imports\n",
      "  - matrixStats (>= 0.54.0) type: imports\n",
      "  - fs (>= 1.2.6) type: imports\n",
      "  - rmarkdown (>= 1.10) type: imports\n",
      "  - speaq (>= 2.4.0) type: imports\n",
      "  - htmltools (>= 0.3.6) type: imports\n",
      "  - pcaPP (>= 1.9-73) type: imports\n",
      "  - ggplot2 (>= 3.1.0) type: imports\n",
      "  - baseline (>= 1.2-1) type: imports\n",
      "  - vctrs (>= 0.3.0) type: imports\n",
      "  - BiocParallel, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/AlpsNMR.html\n",
      "\n",
      "Name: bioassayR\n",
      "Description:\n",
      "bioassayR is a computational tool that enables simultaneous analysis of thousands of bioassay experiments performed over a diverse set of compounds and biological targets. Unique features include support for large-scale cross-target analyses of both public and custom bioassays, generation of high throughput screening fingerprints (HTSFPs), and an optional preloaded database that provides access to a substantial portion of publicly available bioactivity data.\n",
      "Version: 1.36.0\n",
      "Publication date: None\n",
      "Mantainer: Daniela Cassol <danicassol@gmail.com>\n",
      "Authors:\n",
      "Author: Tyler Backman, Ronly Schlenk, Thomas Girke\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - DBI (>= 0.3.1) type: depends\n",
      "  - RSQLite (>= 1.0.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - Matrix, type: depends\n",
      "  - rjson, type: depends\n",
      "  - BiocGenerics, type: depends\n",
      "  - XML, type: imports\n",
      "  - ChemmineR, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/bioassayR.html\n",
      "\n",
      "Name: CellMixS\n",
      "Description:\n",
      "CellMixS provides metrics and functions to evaluate batch effects, data integration and batch effect correction in single cell trancriptome data with single cell resolution. Results can be visualized and summarised on different levels, e.g. on cell, celltype or dataset level.\n",
      "Version: 1.14.0\n",
      "Publication date: None\n",
      "Mantainer: Almut LÃ¼tge <almut.luetge@uzh.ch>\n",
      "Authors:\n",
      "Author: Almut LÃ¼tge [aut, cre]\n",
      "Dependencies:\n",
      "  - kSamples, type: depends\n",
      "  - R (>= 4.0) type: depends\n",
      "  - BiocNeighbors, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - scater, type: imports\n",
      "  - viridis, type: imports\n",
      "  - cowplot, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - SingleCellExperiment, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - ggridges, type: imports\n",
      "  - stats, type: imports\n",
      "  - purrr, type: imports\n",
      "  - methods, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>=2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/CellMixS.html\n",
      "\n",
      "Name: DiffLogo\n",
      "Description:\n",
      "DiffLogo is an easy-to-use tool to visualize motif differences.\n",
      "Version: 2.22.0\n",
      "Publication date: None\n",
      "Mantainer: Hendrik Treutler<hendrik.treutler@gmail.com>\n",
      "Authors:\n",
      "Author: c( person(\"Martin\", \"Nettling\", role = c(\"aut\", \"cre\"), email = \"martin.nettling@informatik.uni-halle.de\"), person(\"Hendrik\", \"Treutler\", role = c(\"aut\", \"cre\"), email = \"hendrik.treutler@ipb-halle.de\"), person(\"Jan\", \"Grau\", role = c(\"aut\", \"ctb\"), email = \"grau@informatik.uni-halle.de\"), person(\"Andrey\", \"Lando\", role = c(\"aut\", \"ctb\"), email = \"dronte@autosome.ru\"), person(\"Jens\", \"Keilwagen\", role = c(\"aut\", \"ctb\"), email = \"jens.keilwagen@julius-kuehn.de\"), person(\"Stefan\", \"Posch\", role = \"aut\", email = \"posch@informatik.uni-halle.de\"), person(\"Ivo\", \"Grosse\", role = \"aut\", email = \"grosse@informatik.uni-halle.de\"))\n",
      "Dependencies:\n",
      "  - R (>= 3.4) type: depends\n",
      "  - stats, type: depends\n",
      "  - cba, type: depends\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - utils, type: imports\n",
      "  - tools, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/DiffLogo.html\n",
      "\n",
      "Name: EGAD\n",
      "Description:\n",
      "The package implements a series of highly efficient tools to calculate functional properties of networks based on guilt by association methods.\n",
      "Version: 1.26.0\n",
      "Publication date: None\n",
      "Mantainer: Sara Ballouz <sarahballouz@gmail.com>\n",
      "Authors:\n",
      "Author: Sara Ballouz [aut, cre], Melanie Weber [aut, ctb], Paul Pavlidis [aut], Jesse Gillis [aut, ctb]\n",
      "Dependencies:\n",
      "  - R (>= 3.5) type: depends\n",
      "  - gplots, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - GEOquery, type: imports\n",
      "  - limma, type: imports\n",
      "  - impute, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - zoo, type: imports\n",
      "  - igraph, type: imports\n",
      "  - plyr, type: imports\n",
      "  - MASS, type: imports\n",
      "  - RCurl, type: imports\n",
      "  - methods, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/EGAD.html\n",
      "\n",
      "Name: GeneMeta\n",
      "Description:\n",
      "A collection of meta-analysis tools for analysing high throughput experimental data\n",
      "Version: 1.70.0\n",
      "Publication date: None\n",
      "Mantainer: Bioconductor Package Maintainer <maintainer@bioconductor.org>\n",
      "Authors:\n",
      "Author: Lara Lusa <lusa at ifom-firc.it>, R. Gentleman, M. Ruschhaupt\n",
      "Dependencies:\n",
      "  - R (>= 2.10) type: depends\n",
      "  - methods, type: depends\n",
      "  - Biobase, type: depends\n",
      "  - genefilter, type: depends\n",
      "  - methods, type: imports\n",
      "  - Biobase, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/GeneMeta.html\n",
      "\n",
      "Name: geneplast\n",
      "Description:\n",
      "Geneplast is designed for evolutionary and plasticity analysis based on orthologous groups distribution in a given species tree. It uses Shannon information theory and orthologs abundance to estimate the Evolutionary Plasticity Index. Additionally, it implements the Bridge algorithm to determine the evolutionary root of a given gene based on its orthologs distribution.\n",
      "Version: 1.24.0\n",
      "Publication date: None\n",
      "Mantainer: Mauro Castro <mauro.a.castro@gmail.com>\n",
      "Authors:\n",
      "Author: Rodrigo Dalmolin, Mauro Castro\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - igraph, type: imports\n",
      "  - snow, type: imports\n",
      "  - ape, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - data.table, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/geneplast.html\n",
      "\n",
      "Name: hipathia\n",
      "Description:\n",
      "Hipathia is a method for the computation of signal transduction along signaling pathways from transcriptomic data. The method is based on an iterative algorithm which is able to compute the signal intensity passing through the nodes of a network by taking into account the level of expression of each gene and the intensity of the signal arriving to it. It also provides a new approach to functional analysis allowing to compute the signal arriving to the functions annotated to each pathway.\n",
      "Version: 2.14.0\n",
      "Publication date: None\n",
      "Mantainer: Marta R. Hidalgo <marta.hidalgo@outlook.es>\n",
      "Authors:\n",
      "Author: Marta R. Hidalgo [aut, cre], JosÃ© Carbonell-Caballero [ctb], Francisco Salavert [ctb], Alicia Amadoz [ctb], Ãankut Cubuk [ctb], Joaquin Dopazo [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 3.6) type: depends\n",
      "  - igraph (>= 1.0.1) type: depends\n",
      "  - AnnotationHub, type: depends\n",
      "  - MultiAssayExperiment, type: depends\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - coin, type: imports\n",
      "  - stats, type: imports\n",
      "  - limma, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - utils, type: imports\n",
      "  - graphics, type: imports\n",
      "  - preprocessCore, type: imports\n",
      "  - servr, type: imports\n",
      "  - DelayedArray, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - methods, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/hipathia.html\n",
      "\n",
      "Name: MicrobiomeProfiler\n",
      "Description:\n",
      "This is an R/shiny package to perform functional enrichment analysis for microbiome data. This package was based on clusterProfiler. Moreover, MicrobiomeProfiler support KEGG enrichment analysis, COG enrichment analysis, Microbe-Disease association enrichment analysis, Metabo-Pathway analysis.\n",
      "Version: 1.4.0\n",
      "Publication date: None\n",
      "Mantainer: Meijun Chen <mjchen1996@outlook.com>\n",
      "Authors:\n",
      "Author: Meijun Chen [aut, cre] , Guangchuang Yu [aut, ths]\n",
      "Dependencies:\n",
      "  - R (>= 4.1.0) type: depends\n",
      "  - clusterProfiler, type: imports\n",
      "  - config, type: imports\n",
      "  - DT, type: imports\n",
      "  - enrichplot, type: imports\n",
      "  - golem, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - shiny (>= 1.6.0) type: imports\n",
      "  - shinyWidgets, type: imports\n",
      "  - shinycustomloader, type: imports\n",
      "  - htmltools, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - graphics, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/MicrobiomeProfiler.html\n",
      "\n",
      "Name: RCAS\n",
      "Description:\n",
      "RCAS is an R/Bioconductor package designed as a generic reporting tool for the functional analysis of transcriptome-wide regions of interest detected by high-throughput experiments. Such transcriptomic regions could be, for instance, signal peaks detected by CLIP-Seq analysis for protein-RNA interaction sites, RNA modification sites (alias the epitranscriptome), CAGE-tag locations, or any other collection of query regions at the level of the transcriptome. RCAS produces in-depth annotation summaries and coverage profiles based on the distribution of the query regions with respect to transcript features (exons, introns, 5'/3' UTR regions, exon-intron boundaries, promoter regions). Moreover, RCAS can carry out functional enrichment analyses and discriminative motif discovery.\n",
      "Version: 1.24.0\n",
      "Publication date: None\n",
      "Mantainer: Bora Uyar <bora.uyar@mdc-berlin.de>\n",
      "Authors:\n",
      "Author: Bora Uyar [aut, cre], Dilmurat Yusuf [aut], Ricardo Wurmus [aut], Altuna Akalin [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - plotly (>= 4.5.2) type: depends\n",
      "  - DT (>= 0.2) type: depends\n",
      "  - data.table, type: depends\n",
      "  - GenomicRanges, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - BSgenome, type: imports\n",
      "  - BSgenome.Hsapiens.UCSC.hg19, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - GenomicFeatures, type: imports\n",
      "  - rmarkdown (>= 0.9.5) type: imports\n",
      "  - genomation, type: imports\n",
      "  - knitr (>= 1.12.3) type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - plotrix, type: imports\n",
      "  - pbapply, type: imports\n",
      "  - RSQLite, type: imports\n",
      "  - proxy, type: imports\n",
      "  - pheatmap, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - cowplot, type: imports\n",
      "  - ggseqlogo, type: imports\n",
      "  - utils, type: imports\n",
      "  - ranger, type: imports\n",
      "  - gprofiler2, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/RCAS.html\n",
      "\n",
      "Name: scDataviz\n",
      "Description:\n",
      "In the single cell World, which includes flow cytometry, mass cytometry, single-cell RNA-seq (scRNA-seq), and others, there is a need to improve data visualisation and to bring analysis capabilities to researchers even from non-technical backgrounds. scDataviz attempts to fit into this space, while also catering for advanced users. Additonally, due to the way that scDataviz is designed, which is based on SingleCellExperiment, it has a 'plug and play' feel, and immediately lends itself as flexibile and compatibile with studies that go beyond scDataviz. Finally, the graphics in scDataviz are generated via the ggplot engine, which means that users can 'add on' features to these with ease.\n",
      "Version: 1.8.0\n",
      "Publication date: None\n",
      "Mantainer: Kevin Blighe <kevin@clinicalbioinformatics.co.uk>\n",
      "Authors:\n",
      "Author: Kevin Blighe [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - S4Vectors, type: depends\n",
      "  - SingleCellExperiment, type: depends\n",
      "  - ggplot2, type: imports\n",
      "  - ggrepel, type: imports\n",
      "  - flowCore, type: imports\n",
      "  - umap, type: imports\n",
      "  - Seurat, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - scales, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - corrplot, type: imports\n",
      "  - stats, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - utils, type: imports\n",
      "  - MASS, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - methods, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/scDataviz.html\n",
      "\n",
      "Name: SeqGSEA\n",
      "Description:\n",
      "The package generally provides methods for gene set enrichment analysis of high-throughput RNA-Seq data by integrating differential expression and splicing. It uses negative binomial distribution to model read count data, which accounts for sequencing biases and biological variation. Based on permutation tests, statistical significance can also be achieved regarding each gene's differential expression and splicing, respectively.\n",
      "Version: 1.38.0\n",
      "Publication date: None\n",
      "Mantainer: Xi Wang <Xi.Wang@dkfz.de>\n",
      "Authors:\n",
      "Author: Xi Wang <Xi.Wang at newcastle.edu.au>\n",
      "Dependencies:\n",
      "  - Biobase, type: depends\n",
      "  - doParallel, type: depends\n",
      "  - DESeq2, type: depends\n",
      "  - methods, type: imports\n",
      "  - biomaRt, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 3)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/SeqGSEA.html\n",
      "\n",
      "Name: SpidermiR\n",
      "Description:\n",
      "The aims of SpidermiR are : i) facilitate the network open-access data retrieval from GeneMania data, ii) prepare the data using the appropriate gene nomenclature, iii) integration of miRNA data in a specific network, iv) provide different standard analyses and v) allow the user to visualize the results. In more detail, the package provides multiple methods for query, prepare and download network data (GeneMania), and the integration with validated and predicted miRNA data (mirWalk, miRTarBase, miRandola, Miranda, PicTar and TargetScan). Furthermore, we also present a statistical test to identify pharmaco-mir relationships using the gene-drug interactions derived by DGIdb and MATADOR database.\n",
      "Version: 1.28.0\n",
      "Publication date: None\n",
      "Mantainer: Claudia Cava <claudia.cava@ibfm.cnr.it>\n",
      "Authors:\n",
      "Author: Claudia Cava, Antonio Colaprico, Alex Graudenzi, Gloria Bertoli, Tiago C. Silva, Catharina Olsen, Houtan Noushmehr, Gianluca Bontempi, Giancarlo Mauri, Isabella Castiglioni\n",
      "Dependencies:\n",
      "  - R (>= 3.0.0) type: depends\n",
      "  - httr, type: imports\n",
      "  - igraph, type: imports\n",
      "  - utils, type: imports\n",
      "  - stats, type: imports\n",
      "  - miRNAtap, type: imports\n",
      "  - miRNAtap.db, type: imports\n",
      "  - AnnotationDbi, type: imports\n",
      "  - org.Hs.eg.db, type: imports\n",
      "  - gdata, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 3)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/SpidermiR.html\n",
      "\n",
      "Name: VanillaICE\n",
      "Description:\n",
      "Hidden Markov Models for characterizing chromosomal alteration in high throughput SNP arrays.\n",
      "Version: 1.60.0\n",
      "Publication date: None\n",
      "Mantainer: Robert Scharpf <rscharpf@jhu.edu>\n",
      "Authors:\n",
      "Author: Robert Scharpf [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - BiocGenerics, type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - MatrixGenerics, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - oligoClasses, type: imports\n",
      "  - foreach, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - data.table, type: imports\n",
      "  - grid, type: imports\n",
      "  - lattice, type: imports\n",
      "  - methods, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - crlmm, type: imports\n",
      "  - tools, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - BSgenome.Hsapiens.UCSC.hg18, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/VanillaICE.html\n",
      "\n",
      "Name: branchpointer\n",
      "Description:\n",
      "Predicts branchpoint probability for sites in intronic branchpoint windows. Queries can be supplied as intronic regions; or to evaluate the effects of mutations, SNPs.\n",
      "Version: 1.24.0\n",
      "Publication date: None\n",
      "Mantainer: Beth Signal <b.signal@garvan.org.au>\n",
      "Authors:\n",
      "Author: Beth Signal\n",
      "Dependencies:\n",
      "  - caret, type: depends\n",
      "  - R (>= 3.4) type: depends\n",
      "  - plyr, type: imports\n",
      "  - kernlab, type: imports\n",
      "  - gbm, type: imports\n",
      "  - stringr, type: imports\n",
      "  - cowplot, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - biomaRt, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - parallel, type: imports\n",
      "  - utils, type: imports\n",
      "  - stats, type: imports\n",
      "  - BSgenome.Hsapiens.UCSC.hg38, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - data.table, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "BSD_3_clause + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/branchpointer.html\n",
      "\n",
      "Name: cellbaseR\n",
      "Description:\n",
      "This R package makes use of the exhaustive RESTful Web service API that has been implemented for the Cellabase database. It enable researchers to query and obtain a wealth of biological information from a single database saving a lot of time. Another benefit is that researchers can easily make queries about different biological topics and link all this information together as all information is integrated.\n",
      "Version: 1.22.0\n",
      "Publication date: None\n",
      "Mantainer: Mohammed OE Abdallah <melsiddieg@gmail.com>\n",
      "Authors:\n",
      "Author: Mohammed OE Abdallah\n",
      "Dependencies:\n",
      "  - R (>= 3.4) type: depends\n",
      "  - methods, type: imports\n",
      "  - jsonlite, type: imports\n",
      "  - httr, type: imports\n",
      "  - data.table, type: imports\n",
      "  - pbapply, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - R.utils, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - foreach, type: imports\n",
      "  - utils, type: imports\n",
      "  - parallel, type: imports\n",
      "  - doParallel, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Apache License (== 2.0)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/cellbaseR.html\n",
      "\n",
      "Name: EBSeqHMM\n",
      "Description:\n",
      "The EBSeqHMM package implements an auto-regressive hidden Markov model for statistical analysis in ordered RNA-seq experiments (e.g. time course or spatial course data). The EBSeqHMM package provides functions to identify genes and isoforms that have non-constant expression profile over the time points/positions, and cluster them into expression paths.\n",
      "Version: 1.32.0\n",
      "Publication date: None\n",
      "Mantainer: Ning Leng <lengning1@gmail.com>\n",
      "Authors:\n",
      "Author: Ning Leng, Christina Kendziorski\n",
      "Dependencies:\n",
      "  - EBSeq, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/EBSeqHMM.html\n",
      "\n",
      "Name: erccdashboard\n",
      "Description:\n",
      "Technical performance metrics for differential gene expression experiments using External RNA Controls Consortium (ERCC) spike-in ratio mixtures.\n",
      "Version: 1.32.0\n",
      "Publication date: None\n",
      "Mantainer: Sarah Munro <sarah.munro@gmail.com>\n",
      "Authors:\n",
      "Author: Sarah Munro, Steve Lund\n",
      "Dependencies:\n",
      "  - R (>= 3.2) type: depends\n",
      "  - ggplot2 (>= 2.1.0) type: depends\n",
      "  - gridExtra (>= 2.0.0) type: depends\n",
      "  - edgeR, type: imports\n",
      "  - gplots, type: imports\n",
      "  - grid, type: imports\n",
      "  - gtools, type: imports\n",
      "  - limma, type: imports\n",
      "  - locfit, type: imports\n",
      "  - MASS, type: imports\n",
      "  - plyr, type: imports\n",
      "  - qvalue, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - ROCR, type: imports\n",
      "  - scales, type: imports\n",
      "  - stringr, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>=2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/erccdashboard.html\n",
      "\n",
      "Name: fcoex\n",
      "Description:\n",
      "The fcoex package implements an easy-to use interface to co-expression analysis based on the FCBF (Fast Correlation-Based Filter) algorithm. it was implemented especifically to deal with single-cell data. The modules found can be used to redefine cell populations, unrevel novel gene associations and predict gene function by guilt-by-association. The package structure is adapted from the CEMiTool package, relying on visualizations and code designed and written by CEMiTool's authors.\n",
      "Version: 1.12.0\n",
      "Publication date: None\n",
      "Mantainer: Tiago Lubiana <tiago.lubiana.alves@usp.br>\n",
      "Authors:\n",
      "Author: Tiago Lubiana [aut, cre], Helder Nakaya [aut, ths]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - FCBF, type: imports\n",
      "  - parallel, type: imports\n",
      "  - progress, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - ggrepel, type: imports\n",
      "  - igraph, type: imports\n",
      "  - grid, type: imports\n",
      "  - intergraph, type: imports\n",
      "  - stringr, type: imports\n",
      "  - clusterProfiler, type: imports\n",
      "  - data.table, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - methods, type: imports\n",
      "  - network, type: imports\n",
      "  - scales, type: imports\n",
      "  - sna, type: imports\n",
      "  - utils, type: imports\n",
      "  - stats, type: imports\n",
      "  - SingleCellExperiment, type: imports\n",
      "  - pathwayPCA, type: imports\n",
      "  - Matrix, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/fcoex.html\n",
      "\n",
      "Name: groHMM\n",
      "Description:\n",
      "A pipeline for the analysis of GRO-seq data.\n",
      "Version: 1.32.0\n",
      "Publication date: None\n",
      "Mantainer: Tulip Nandu <tulip.nandu@utsouthwestern.edu>, W. Lee Kraus <lee.kraus@utsouthwestern.edu>\n",
      "Authors:\n",
      "Author: Charles G. Danko, Minho Chae, Andre Martins, W. Lee Kraus\n",
      "Dependencies:\n",
      "  - R (>= 3.0.2) type: depends\n",
      "  - MASS, type: depends\n",
      "  - parallel, type: depends\n",
      "  - S4Vectors, type: depends\n",
      "  - IRanges, type: depends\n",
      "  - GenomeInfoDb, type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - GenomicAlignments, type: depends\n",
      "  - rtracklayer, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/groHMM.html\n",
      "\n",
      "Name: hca\n",
      "Description:\n",
      "This package provides users with the ability to query the Human Cell Atlas data repository for single-cell experiment data. The `projects()`, `files()`, `samples()` and `bundles()` functions retrieve summary information on each of these indexes; corresponding `*_details()` are available for individual entries of each index. File-based resources can be downloaded using `files_download()`. Advanced use of the package allows the user to page through large result sets, and to flexibly query the 'list-of-lists' structure representing query responses.\n",
      "Version: 1.6.0\n",
      "Publication date: None\n",
      "Mantainer: Martin Morgan <martin.morgan@roswellpark.org>\n",
      "Authors:\n",
      "Author: Maya McDaniel [aut], Martin Morgan [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - httr, type: imports\n",
      "  - jsonlite, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - tibble, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - readr, type: imports\n",
      "  - BiocFileCache, type: imports\n",
      "  - tools, type: imports\n",
      "  - utils, type: imports\n",
      "  - digest, type: imports\n",
      "  - shiny, type: imports\n",
      "  - miniUI, type: imports\n",
      "  - DT, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/hca.html\n",
      "\n",
      "Name: IdeoViz\n",
      "Description:\n",
      "Plots data associated with arbitrary genomic intervals along chromosomal ideogram.\n",
      "Version: 1.34.0\n",
      "Publication date: None\n",
      "Mantainer: Shraddha Pai <shraddha.pai@utoronto.ca>\n",
      "Authors:\n",
      "Author: Shraddha Pai <shraddha.pai at utoronto.ca>, Jingliang Ren\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - Biobase, type: depends\n",
      "  - IRanges, type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - RColorBrewer, type: depends\n",
      "  - rtracklayer, type: depends\n",
      "  - graphics, type: depends\n",
      "  - GenomeInfoDb, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/IdeoViz.html\n",
      "\n",
      "Name: maanova\n",
      "Description:\n",
      "Analysis of N-dye Micro Array experiment using mixed model effect. Containing analysis of variance, permutation and bootstrap, cluster and consensus tree.\n",
      "Version: 1.68.2\n",
      "Publication date: None\n",
      "Mantainer: Keith Sheppard <keith.sheppard@jax.org>\n",
      "Authors:\n",
      "Author: Hao Wu, modified by Hyuna Yang and Keith Sheppard with ideas from Gary Churchill, Katie Kerr and Xiangqin Cui.\n",
      "Dependencies:\n",
      "  - R (>= 2.10) type: depends\n",
      "  - Biobase, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/maanova.html\n",
      "\n",
      "Name: MEIGOR\n",
      "Description:\n",
      "Global Optimization\n",
      "Version: 1.31.0\n",
      "Publication date: None\n",
      "Mantainer: Jose A. Egea <josea.egea@gmail.com>\n",
      "Authors:\n",
      "Author: Jose A. Egea, David Henriques, Alexandre Fdez. Villaverde, Thomas Cokelaer\n",
      "Dependencies:\n",
      "  - Rsolnp, type: depends\n",
      "  - snowfall, type: depends\n",
      "  - CNORode, type: depends\n",
      "  - deSolve, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/MEIGOR.html\n",
      "\n",
      "Name: metaseqR2\n",
      "Description:\n",
      "Provides an interface to several normalization and statistical testing packages for RNA-Seq gene expression data. Additionally, it creates several diagnostic plots, performs meta-analysis by combinining the results of several statistical tests and reports the results in an interactive way.\n",
      "Version: 1.10.0\n",
      "Publication date: None\n",
      "Mantainer: Panagiotis Moulos <moulos@fleming.gr>\n",
      "Authors:\n",
      "Author: Panagiotis Moulos [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.0.0) type: depends\n",
      "  - DESeq2, type: depends\n",
      "  - limma, type: depends\n",
      "  - locfit, type: depends\n",
      "  - splines, type: depends\n",
      "  - ABSSeq, type: imports\n",
      "  - baySeq, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - biomaRt, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - corrplot, type: imports\n",
      "  - DSS, type: imports\n",
      "  - DT, type: imports\n",
      "  - EDASeq, type: imports\n",
      "  - edgeR, type: imports\n",
      "  - genefilter, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - GenomicAlignments, type: imports\n",
      "  - GenomicFeatures, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - gplots, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - harmonicmeanp, type: imports\n",
      "  - heatmaply, type: imports\n",
      "  - htmltools, type: imports\n",
      "  - httr, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - jsonlite, type: imports\n",
      "  - lattice, type: imports\n",
      "  - log4r, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - MASS, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - methods, type: imports\n",
      "  - NBPSeq, type: imports\n",
      "  - pander, type: imports\n",
      "  - parallel, type: imports\n",
      "  - qvalue, type: imports\n",
      "  - rmarkdown, type: imports\n",
      "  - rmdformats, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - RSQLite, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - stats, type: imports\n",
      "  - stringr, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - survcomp, type: imports\n",
      "  - utils, type: imports\n",
      "  - VennDiagram, type: imports\n",
      "  - vsn, type: imports\n",
      "  - yaml, type: imports\n",
      "  - zoo, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 3)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/metaseqR2.html\n",
      "\n",
      "Name: microbiomeExplorer\n",
      "Description:\n",
      "The MicrobiomeExplorer R package is designed to facilitate the analysis and visualization of marker-gene survey feature data. It allows a user to perform and visualize typical microbiome analytical workflows either through the command line or an interactive Shiny application included with the package. In addition to applying common analytical workflows the application enables automated analysis report generation.\n",
      "Version: 1.8.0\n",
      "Publication date: None\n",
      "Mantainer: Janina Reeder <reederj1@gene.com>\n",
      "Authors:\n",
      "Author: Joseph Paulson [aut], Janina Reeder [aut, cre], Mo Huang [aut], Genentech [cph, fnd]\n",
      "Dependencies:\n",
      "  - shiny, type: depends\n",
      "  - magrittr, type: depends\n",
      "  - metagenomeSeq, type: depends\n",
      "  - Biobase, type: depends\n",
      "  - shinyjs (>= 2.0.0) type: imports\n",
      "  - shinydashboard, type: imports\n",
      "  - shinycssloaders, type: imports\n",
      "  - shinyWidgets, type: imports\n",
      "  - rmarkdown (>= 1.9.0) type: imports\n",
      "  - DESeq2, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - purrr, type: imports\n",
      "  - rlang, type: imports\n",
      "  - knitr, type: imports\n",
      "  - readr, type: imports\n",
      "  - DT (>= 0.12.0) type: imports\n",
      "  - biomformat, type: imports\n",
      "  - tools, type: imports\n",
      "  - stringr, type: imports\n",
      "  - vegan, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - heatmaply, type: imports\n",
      "  - car, type: imports\n",
      "  - broom, type: imports\n",
      "  - limma, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - tibble, type: imports\n",
      "  - forcats, type: imports\n",
      "  - lubridate, type: imports\n",
      "  - methods, type: imports\n",
      "  - plotly (>= 4.9.1) type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/microbiomeExplorer.html\n",
      "\n",
      "Name: NanoMethViz\n",
      "Description:\n",
      "NanoMethViz is a toolkit for visualising methylation data from Oxford Nanopore sequencing. It can be used to explore methylation patterns from reads derived from Oxford Nanopore direct DNA sequencing with methylation called by callers including nanopolish, f5c and megalodon. The plots in this package allow the visualisation of methylation profiles aggregated over experimental groups and across classes of genomic features.\n",
      "Version: 2.4.2\n",
      "Publication date: None\n",
      "Mantainer: Shian Su <su.s@wehi.edu.au>\n",
      "Authors:\n",
      "Author: Shian Su [cre, aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.0.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - ggplot2, type: depends\n",
      "  - cpp11 (>= 0.2.5) type: imports\n",
      "  - readr, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - BiocSingular, type: imports\n",
      "  - bsseq, type: imports\n",
      "  - forcats, type: imports\n",
      "  - assertthat, type: imports\n",
      "  - AnnotationDbi, type: imports\n",
      "  - Rcpp, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - data.table, type: imports\n",
      "  - e1071, type: imports\n",
      "  - fs, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - ggrastr, type: imports\n",
      "  - glue, type: imports\n",
      "  - graphics, type: imports\n",
      "  - limma, type: imports\n",
      "  - patchwork, type: imports\n",
      "  - purrr, type: imports\n",
      "  - rlang, type: imports\n",
      "  - R.utils, type: imports\n",
      "  - RSQLite, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - scales (>= 1.2.0) type: imports\n",
      "  - scico, type: imports\n",
      "  - stats, type: imports\n",
      "  - stringr, type: imports\n",
      "  - tibble, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - utils, type: imports\n",
      "  - withr, type: imports\n",
      "  - zlibbioc, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Apache License (>= 2.0)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/NanoMethViz.html\n",
      "\n",
      "Name: netbiov\n",
      "Description:\n",
      "A package that provides an effective visualization of large biological networks\n",
      "Version: 1.31.0\n",
      "Publication date: None\n",
      "Mantainer: Shailesh tripathi <shailesh.tripathy@gmail.com>\n",
      "Authors:\n",
      "Author: Shailesh tripathi and Frank Emmert-Streib\n",
      "Dependencies:\n",
      "  - R (>= 3.1.0) type: depends\n",
      "  - igraph (>= 0.7.1) type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/netbiov.html\n",
      "\n",
      "Name: POMA\n",
      "Description:\n",
      "A reproducible and easy-to-use toolkit for visualization, pre-processing, exploration, and statistical analysis of omics datasets. The main aim of POMA is to enable a flexible data cleaning and statistical analysis processes in one comprehensible and user-friendly R package. This package has a Shiny app version called POMAShiny that implements all POMA functions. See https://github.com/pcastellanoescuder/POMAShiny. See Castellano-Escuder P, GonzÃ¡lez-DomÃ­nguez R, Carmona-Pontaque F, et al. (2021) for more details.\n",
      "Version: 1.8.0\n",
      "Publication date: None\n",
      "Mantainer: Pol Castellano-Escuder <polcaes@gmail.com>\n",
      "Authors:\n",
      "Author: Pol Castellano-Escuder [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - broom, type: imports\n",
      "  - caret, type: imports\n",
      "  - ComplexHeatmap, type: imports\n",
      "  - dbscan, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - DESeq2, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - ggrepel, type: imports\n",
      "  - glasso (>= 1.11) type: imports\n",
      "  - glmnet, type: imports\n",
      "  - impute, type: imports\n",
      "  - limma, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - mixOmics, type: imports\n",
      "  - randomForest, type: imports\n",
      "  - RankProd, type: imports\n",
      "  - rmarkdown, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - tibble, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - uwot, type: imports\n",
      "  - vegan, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/POMA.html\n",
      "\n",
      "Name: projectR\n",
      "Description:\n",
      "Functions for the projection of data into the spaces defined by PCA, CoGAPS, NMF, correlation, and clustering.\n",
      "Version: 1.14.0\n",
      "Publication date: None\n",
      "Mantainer: Genevieve Stein-O'Brien <gsteinobrien@gmail.com>\n",
      "Authors:\n",
      "Author: Gaurav Sharma, Genevieve Stein-O'Brien\n",
      "Dependencies:\n",
      "  - methods, type: imports\n",
      "  - cluster, type: imports\n",
      "  - stats, type: imports\n",
      "  - limma, type: imports\n",
      "  - CoGAPS, type: imports\n",
      "  - NMF, type: imports\n",
      "  - ROCR, type: imports\n",
      "  - ggalluvial, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - viridis, type: imports\n",
      "  - scales, type: imports\n",
      "  - ggplot2, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (==2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/projectR.html\n",
      "\n",
      "Name: QuaternaryProd\n",
      "Description:\n",
      "QuaternaryProd is an R package that performs causal reasoning on biological networks, including publicly available networks such as STRINGdb. QuaternaryProd is an open-source alternative to commercial products such as Inginuity Pathway Analysis. For a given a set of differentially expressed genes, QuaternaryProd computes the significance of upstream regulators in the network by performing causal reasoning using the Quaternary Dot Product Scoring Statistic (Quaternary Statistic), Ternary Dot product Scoring Statistic (Ternary Statistic) and Fisher's exact test (Enrichment test). The Quaternary Statistic handles signed, unsigned and ambiguous edges in the network. Ambiguity arises when the direction of causality is unknown, or when the source node (e.g., a protein) has edges with conflicting signs for the same target gene. On the other hand, the Ternary Statistic provides causal reasoning using the signed and unambiguous edges only. The Vignette provides more details on the Quaternary Statistic and illustrates an example of how to perform causal reasoning using STRINGdb.\n",
      "Version: 1.32.0\n",
      "Publication date: None\n",
      "Mantainer: Carl Tony Fakhry <cfakhry@cs.umb.edu>\n",
      "Authors:\n",
      "Author: Carl Tony Fakhry [cre, aut], Ping Chen [ths], Kourosh Zarringhalam [aut, ths]\n",
      "Dependencies:\n",
      "  - R (>= 3.2.0) type: depends\n",
      "  - Rcpp (>= 0.11.3) type: depends\n",
      "  - dplyr, type: depends\n",
      "  - yaml (>= 2.1.18) type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>=3)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/QuaternaryProd.html\n",
      "\n",
      "Name: sevenbridges\n",
      "Description:\n",
      "R client and utilities for Seven Bridges platform API, from Cancer Genomics Cloud to other Seven Bridges supported platforms.\n",
      "Version: 1.28.0\n",
      "Publication date: None\n",
      "Mantainer: Soner Koc <soner.koc@sevenbridges.com>\n",
      "Authors:\n",
      "Author: Soner Koc [aut, cre], Nan Xiao [aut], Tengfei Yin [aut], Dusan Randjelovic [ctb], Emile Young [ctb], Seven Bridges Genomics [cph, fnd]\n",
      "Dependencies:\n",
      "  - methods, type: depends\n",
      "  - utils, type: depends\n",
      "  - stats, type: depends\n",
      "  - httr, type: imports\n",
      "  - jsonlite, type: imports\n",
      "  - yaml, type: imports\n",
      "  - objectProperties, type: imports\n",
      "  - stringr, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - docopt, type: imports\n",
      "  - curl, type: imports\n",
      "  - uuid, type: imports\n",
      "  - data.table, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Apache License 2.0 | file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/sevenbridges.html\n",
      "\n",
      "Name: SharedObject\n",
      "Description:\n",
      "This package is developed for facilitating parallel computing in R. It is capable to create an R object in the shared memory space and share the data across multiple R processes. It avoids the overhead of memory dulplication and data transfer, which make sharing big data object across many clusters possible.\n",
      "Version: 1.12.0\n",
      "Publication date: None\n",
      "Mantainer: Jiefei Wang <szwjf08@gmail.com>\n",
      "Authors:\n",
      "Author: Jiefei Wang [aut, cre], Martin Morgan [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.6.0) type: depends\n",
      "  - Rcpp, type: imports\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/SharedObject.html\n",
      "\n",
      "Name: anota\n",
      "Description:\n",
      "Genome wide studies of translational control is emerging as a tool to study verious biological conditions. The output from such analysis is both the mRNA level (e.g. cytosolic mRNA level) and the levl of mRNA actively involved in translation (the actively translating mRNA level) for each mRNA. The standard analysis of such data strives towards identifying differential translational between two or more sample classes - i.e. differences in actively translated mRNA levels that are independent of underlying differences in cytosolic mRNA levels. This package allows for such analysis using partial variances and the random variance model. As 10s of thousands of mRNAs are analyzed in parallell the library performs a number of tests to assure that the data set is suitable for such analysis.\n",
      "Version: 1.46.0\n",
      "Publication date: None\n",
      "Mantainer: Ola Larsson <ola.larsson@ki.se>\n",
      "Authors:\n",
      "Author: Ola Larsson <ola.larsson at ki.se>, Nahum Sonenberg <nahum.sonenberg at mcgill.ca>, Robert Nadon <robert.nadon at mcgill.ca>\n",
      "Dependencies:\n",
      "  - qvalue, type: depends\n",
      "  - multtest, type: imports\n",
      "  - qvalue, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/anota.html\n",
      "\n",
      "Name: APAlyzer\n",
      "Description:\n",
      "Perform 3'UTR APA, Intronic APA and gene expression analysis using RNA-seq data.\n",
      "Version: 1.12.0\n",
      "Publication date: None\n",
      "Mantainer: Ruijia Wang <rjwang.bioinfo@gmail.com>\n",
      "Authors:\n",
      "Author: Ruijia Wang [cre, aut] , Bin Tian [aut], Wei-Chun Chen [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - GenomicRanges, type: imports\n",
      "  - GenomicFeatures, type: imports\n",
      "  - GenomicAlignments, type: imports\n",
      "  - DESeq2, type: imports\n",
      "  - ggrepel, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - Rsubread, type: imports\n",
      "  - stats, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - methods, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - VariantAnnotation, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - repmis, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - HybridMTest, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/APAlyzer.html\n",
      "\n",
      "Name: BioNetStat\n",
      "Description:\n",
      "A package to perform differential network analysis, differential node analysis (differential coexpression analysis), network and metabolic pathways view.\n",
      "Version: 1.18.0\n",
      "Publication date: None\n",
      "Mantainer: Vinicius Jardim <viniciusjc@gmail.com>\n",
      "Authors:\n",
      "Author: VinÃ­cius Jardim, Suzana Santos, AndrÃ© Fujita, and Marcos Buckeridge\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - shiny, type: depends\n",
      "  - igraph, type: depends\n",
      "  - shinyBS, type: depends\n",
      "  - pathview, type: depends\n",
      "  - DT, type: depends\n",
      "  - BiocParallel, type: imports\n",
      "  - RJSONIO, type: imports\n",
      "  - whisker, type: imports\n",
      "  - yaml, type: imports\n",
      "  - pheatmap, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - plyr, type: imports\n",
      "  - utils, type: imports\n",
      "  - stats, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - Hmisc, type: imports\n",
      "  - psych, type: imports\n",
      "  - knitr, type: imports\n",
      "  - rmarkdown, type: imports\n",
      "  - markdown, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 3)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/BioNetStat.html\n",
      "\n",
      "Name: BridgeDbR\n",
      "Description:\n",
      "Use BridgeDb functions and load identifier mapping databases in R. It uses GitHub, Zenodo, and Figshare if you use this package to download identifier mappings files.\n",
      "Version: 2.8.0\n",
      "Publication date: None\n",
      "Mantainer: Egon Willighagen <egon.willighagen@gmail.com>\n",
      "Authors:\n",
      "Author: Christ Leemans <christleemans at gmail.com>, Egon Willighagen <egon.willighagen at gmail.com>, Denise Slenter, Anwesha Bohler <anweshabohler at gmail.com>, Lars Eijssen <l.eijssen at maastrichtuniversity.nl>, Tooba Abbassi-Daloii\n",
      "Dependencies:\n",
      "  - R (>= 3.3.0) type: depends\n",
      "  - rJava, type: depends\n",
      "  - curl, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "AGPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/BridgeDbR.html\n",
      "\n",
      "Name: CGEN\n",
      "Description:\n",
      "This is a package for analysis of case-control data in genetic epidemiology. It provides a set of statistical methods for evaluating gene-environment (or gene-genes) interactions under multiplicative and additive risk models, with or without assuming gene-environment (or gene-gene) independence in the underlying population.\n",
      "Version: 3.34.3\n",
      "Publication date: None\n",
      "Mantainer: Justin Lee <jhylee@stanford.edu>\n",
      "Authors:\n",
      "Author: Samsiddhi Bhattacharjee [aut], Nilanjan Chatterjee [aut], Summer Han [aut], Minsun Song [aut], William Wheeler [aut], Matthieu de Rochemonteix [aut], Nilotpal Sanyal [aut], Justin Lee [cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - survival, type: depends\n",
      "  - mvtnorm, type: depends\n",
      "  - stats, type: imports\n",
      "  - graphics, type: imports\n",
      "  - utils, type: imports\n",
      "  - grDevices, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2 + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/CGEN.html\n",
      "\n",
      "Name: CluMSID\n",
      "Description:\n",
      "CluMSID is a tool that aids the identification of features in untargeted LC-MS/MS analysis by the use of MS2 spectra similarity and unsupervised statistical methods. It offers functions for a complete and customisable workflow from raw data to visualisations and is interfaceable with the xmcs family of preprocessing packages.\n",
      "Version: 1.14.0\n",
      "Publication date: None\n",
      "Mantainer: Tobias Depke <depke@mailbox.org>\n",
      "Authors:\n",
      "Author: Tobias Depke [aut, cre], Raimo Franke [ctb], Mark Broenstrup [ths]\n",
      "Dependencies:\n",
      "  - R (>= 3.6) type: depends\n",
      "  - mzR, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - dbscan, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - ape, type: imports\n",
      "  - network, type: imports\n",
      "  - GGally, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - plotly, type: imports\n",
      "  - methods, type: imports\n",
      "  - utils, type: imports\n",
      "  - stats, type: imports\n",
      "  - sna, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - gplots, type: imports\n",
      "  - MSnbase, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/CluMSID.html\n",
      "\n",
      "Name: crisprBowtie\n",
      "Description:\n",
      "Provides a user-friendly interface to map on-targets and off-targets of CRISPR gRNA spacer sequences using bowtie. The alignment is fast, and can be performed using either commonly-used or custom CRISPR nucleases. The alignment can work with any reference or custom genomes. Both DNA- and RNA-targeting nucleases are supported.\n",
      "Version: 1.2.0\n",
      "Publication date: None\n",
      "Mantainer: Jean-Philippe Fortin <fortin946@gmail.com>\n",
      "Authors:\n",
      "Author: Jean-Philippe Fortin [aut, cre]\n",
      "Dependencies:\n",
      "  - methods, type: depends\n",
      "  - BiocGenerics, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - BSgenome, type: imports\n",
      "  - crisprBase, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - Rbowtie, type: imports\n",
      "  - readr, type: imports\n",
      "  - stats, type: imports\n",
      "  - stringr, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/crisprBowtie.html\n",
      "\n",
      "Name: crisprScore\n",
      "Description:\n",
      "Provides R wrappers of several on-target and off-target scoring methods for CRISPR guide RNAs (gRNAs). The following nucleases are supported: SpCas9, AsCas12a, enAsCas12a, and RfxCas13d (CasRx). The available on-target cutting efficiency scoring methods are RuleSet1, Azimuth, DeepHF, DeepCpf1, enPAM+GB, and CRISPRscan. Both the CFD and MIT scoring methods are available for off-target specificity prediction. The package also provides a Lindel-derived score to predict the probability of a gRNA to produce indels inducing a frameshift for the Cas9 nuclease. Note that DeepHF, DeepCpf1 and enPAM+GB are not available on Windows machines.\n",
      "Version: 1.2.0\n",
      "Publication date: None\n",
      "Mantainer: Jean-Philippe Fortin <fortin946@gmail.com>\n",
      "Authors:\n",
      "Author: Jean-Philippe Fortin [aut, cre, cph], Aaron Lun [aut], Luke Hoberecht [ctb], Pirunthan Perampalam [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - crisprScoreData, type: depends\n",
      "  - basilisk, type: imports\n",
      "  - basilisk.utils, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - methods, type: imports\n",
      "  - randomForest, type: imports\n",
      "  - reticulate, type: imports\n",
      "  - stringr, type: imports\n",
      "  - utils, type: imports\n",
      "  - XVector, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/crisprScore.html\n",
      "\n",
      "Name: diffuStats\n",
      "Description:\n",
      "Label propagation approaches are a widely used procedure in computational biology for giving context to molecular entities using network data. Node labels, which can derive from gene expression, genome-wide association studies, protein domains or metabolomics profiling, are propagated to their neighbours in the network, effectively smoothing the scores through prior annotated knowledge and prioritising novel candidates. The R package diffuStats contains a collection of diffusion kernels and scoring approaches that facilitates their computation, characterisation and benchmarking.\n",
      "Version: 1.18.0\n",
      "Publication date: None\n",
      "Mantainer: Sergio Picart-Armada <sergi.picart@upc.edu>\n",
      "Authors:\n",
      "Author: Sergio Picart-Armada [aut, cre], Alexandre Perera-Lluna [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.4) type: depends\n",
      "  - grDevices, type: imports\n",
      "  - stats, type: imports\n",
      "  - methods, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - MASS, type: imports\n",
      "  - checkmate, type: imports\n",
      "  - expm, type: imports\n",
      "  - igraph, type: imports\n",
      "  - Rcpp, type: imports\n",
      "  - RcppArmadillo, type: imports\n",
      "  - RcppParallel, type: imports\n",
      "  - plyr, type: imports\n",
      "  - precrec, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/diffuStats.html\n",
      "\n",
      "Name: epivizrData\n",
      "Description:\n",
      "Serve data from Bioconductor Objects through a WebSocket connection.\n",
      "Version: 1.26.0\n",
      "Publication date: None\n",
      "Mantainer: Hector Corrada Bravo <hcorrada@gmail.com>\n",
      "Authors:\n",
      "Author: Hector Corrada Bravo [aut, cre], Florin Chelaru [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.4) type: depends\n",
      "  - methods, type: depends\n",
      "  - epivizrServer, type: depends\n",
      "  - Biobase, type: depends\n",
      "  - S4Vectors, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - OrganismDbi, type: imports\n",
      "  - GenomicFeatures, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - ensembldb, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/epivizrData.html\n",
      "\n",
      "Name: ImmuneSpaceR\n",
      "Description:\n",
      "Provides a convenient API for accessing data sets within ImmuneSpace Data and Tools Portal (datatools.immunespace.org), the data repository and analysis platform of the Human Immunology Project Consortium (HIPC).\n",
      "Version: 1.26.1\n",
      "Publication date: None\n",
      "Mantainer: ImmuneSpace Package Maintainer <immunespace@gmail.com>\n",
      "Authors:\n",
      "Author: Greg Finak [aut], Renan Sauteraud [aut], Mike Jiang [aut], Gil Guday [aut], Leo Dashevskiy [aut], Evan Henrich [aut], Ju Yeong Kim [aut], Lauren Wolfe [aut], Helen Miller [aut], Raphael Gottardo [aut], ImmuneSpace Package Maintainer [cre, cph]\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - utils, type: imports\n",
      "  - R6, type: imports\n",
      "  - data.table, type: imports\n",
      "  - curl, type: imports\n",
      "  - httr, type: imports\n",
      "  - Rlabkey (>= 2.3.1) type: imports\n",
      "  - Biobase, type: imports\n",
      "  - pheatmap, type: imports\n",
      "  - ggplot2 (>= 3.2.0) type: imports\n",
      "  - scales, type: imports\n",
      "  - stats, type: imports\n",
      "  - gplots, type: imports\n",
      "  - plotly, type: imports\n",
      "  - heatmaply (>= 0.7.0) type: imports\n",
      "  - jsonlite, type: imports\n",
      "  - rmarkdown, type: imports\n",
      "  - preprocessCore, type: imports\n",
      "  - flowCore, type: imports\n",
      "  - flowWorkspace, type: imports\n",
      "  - digest, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ImmuneSpaceR.html\n",
      "\n",
      "Name: IONiseR\n",
      "Description:\n",
      "IONiseR provides tools for the quality assessment of Oxford Nanopore MinION data. It extracts summary statistics from a set of fast5 files and can be used either before or after base calling. In addition to standard summaries of the read-types produced, it provides a number of plots for visualising metrics relative to experiment run time or spatially over the surface of a flowcell.\n",
      "Version: 2.22.0\n",
      "Publication date: None\n",
      "Mantainer: Mike Smith <grimbough@gmail.com>\n",
      "Authors:\n",
      "Author: Mike Smith [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 3.4) type: depends\n",
      "  - rhdf5, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - ShortRead, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - methods, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - XVector, type: imports\n",
      "  - tibble, type: imports\n",
      "  - stats, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - bit64, type: imports\n",
      "  - stringr, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/IONiseR.html\n",
      "\n",
      "Name: PSMatch\n",
      "Description:\n",
      "The PSMatch package helps proteomics practitioners to load, handle and manage Peptide Spectrum Matches. It provides functions to model peptide-protein relations as adjacency matrices and connected components, visualise these as graphs and make informed decision about shared peptide filtering. The package also provides functions to calculate and visualise MS2 fragment ions.\n",
      "Version: 1.2.0\n",
      "Publication date: None\n",
      "Mantainer: Laurent Gatto <laurent.gatto@uclouvain.be>\n",
      "Authors:\n",
      "Author: Laurent Gatto [aut, cre] , Johannes Rainer [aut] , Sebastian Gibb [aut] , Samuel Wieczorek [ctb], Thomas Burger [ctb]\n",
      "Dependencies:\n",
      "  - S4Vectors, type: depends\n",
      "  - utils, type: imports\n",
      "  - stats, type: imports\n",
      "  - igraph, type: imports\n",
      "  - methods, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - ProtGenerics, type: imports\n",
      "  - QFeatures, type: imports\n",
      "  - MsCoreUtils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/PSMatch.html\n",
      "\n",
      "Name: qsea\n",
      "Description:\n",
      "qsea (quantitative sequencing enrichment analysis) was developed as the successor of the MEDIPS package for analyzing data derived from methylated DNA immunoprecipitation (MeDIP) experiments followed by sequencing (MeDIP-seq). However, qsea provides several functionalities for the analysis of other kinds of quantitative sequencing data (e.g. ChIP-seq, MBD-seq, CMS-seq and others) including calculation of differential enrichment between groups of samples.\n",
      "Version: 1.24.0\n",
      "Publication date: None\n",
      "Mantainer: Matthias Lienhard <lienhard@molgen.mpg.de>\n",
      "Authors:\n",
      "Author: Matthias Lienhard, Lukas Chavez, Ralf Herwig\n",
      "Dependencies:\n",
      "  - R (>= 3.5) type: depends\n",
      "  - Biostrings, type: imports\n",
      "  - graphics, type: imports\n",
      "  - gtools, type: imports\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - HMMcopy, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - BSgenome, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - limma, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - zoo, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>=2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/qsea.html\n",
      "\n",
      "Name: RiboProfiling\n",
      "Description:\n",
      "Starting with a BAM file, this package provides the necessary functions for quality assessment, read start position recalibration, the counting of reads on CDS, 3'UTR, and 5'UTR, plotting of count data: pairs, log fold-change, codon frequency and coverage assessment, principal component analysis on codon coverage.\n",
      "Version: 1.28.0\n",
      "Publication date: None\n",
      "Mantainer: A. Popa <alexandra.mariela.popa@gmail.com>\n",
      "Authors:\n",
      "Author: Alexandra Popa\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - Biostrings, type: depends\n",
      "  - BiocGenerics, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - GenomicFeatures, type: imports\n",
      "  - grid, type: imports\n",
      "  - plyr, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - GenomicAlignments, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - ggbio, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - data.table, type: imports\n",
      "  - sqldf, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/RiboProfiling.html\n",
      "\n",
      "Name: ribosomeProfilingQC\n",
      "Description:\n",
      "Ribo-Seq (also named ribosome profiling or footprinting) measures translatome (unlike RNA-Seq, which sequences the transcriptome) by direct quantification of the ribosome-protected fragments (RPFs). This package provides the tools for quality assessment of ribosome profiling. In addition, it can preprocess Ribo-Seq data for subsequent differential analysis.\n",
      "Version: 1.10.0\n",
      "Publication date: None\n",
      "Mantainer: Jianhong Ou <jianhong.ou@duke.edu>\n",
      "Authors:\n",
      "Author: Jianhong Ou [aut, cre] , Mariah Hoye [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - AnnotationDbi, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - BSgenome, type: imports\n",
      "  - EDASeq, type: imports\n",
      "  - GenomicAlignments, type: imports\n",
      "  - GenomicFeatures, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - methods, type: imports\n",
      "  - motifStack, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - RUVSeq, type: imports\n",
      "  - Rsubread, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - XVector, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - ggfittext, type: imports\n",
      "  - scales, type: imports\n",
      "  - ggrepel, type: imports\n",
      "  - utils, type: imports\n",
      "  - cluster, type: imports\n",
      "  - stats, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grid, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>=3) + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ribosomeProfilingQC.html\n",
      "\n",
      "Name: rqubic\n",
      "Description:\n",
      "This package implements the QUBIC algorithm introduced by Li et al. for the qualitative biclustering with gene expression data.\n",
      "Version: 1.44.0\n",
      "Publication date: None\n",
      "Mantainer: Jitao David Zhang <jitao_david.zhang@roche.com>\n",
      "Authors:\n",
      "Author: Jitao David Zhang [aut, cre, ctb]\n",
      "Dependencies:\n",
      "  - methods, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - biclust, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/rqubic.html\n",
      "\n",
      "Name: RSeqAn\n",
      "Description:\n",
      "Headers and some wrapper functions from the SeqAn C++ library for ease of usage in R.\n",
      "Version: 1.18.0\n",
      "Publication date: None\n",
      "Mantainer: August Guang <august.guang@gmail.com>\n",
      "Authors:\n",
      "Author: August Guang [aut, cre]\n",
      "Dependencies:\n",
      "  - Rcpp, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "BSD_3_clause + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/RSeqAn.html\n",
      "\n",
      "Name: runibic\n",
      "Description:\n",
      "This package implements UbiBic algorithm in R. This biclustering algorithm for analysis of gene expression data was introduced by Zhenjia Wang et al. in 2016. It is currently considered the most promising biclustering method for identification of meaningful structures in complex and noisy data.\n",
      "Version: 1.20.0\n",
      "Publication date: None\n",
      "Mantainer: Patryk Orzechowski <patryk.orzechowski@gmail.com>\n",
      "Authors:\n",
      "Author: Patryk Orzechowski, Artur PaÅszczyk\n",
      "Dependencies:\n",
      "  - R (>= 3.4.0) type: depends\n",
      "  - biclust, type: depends\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - Rcpp (>= 0.12.12) type: imports\n",
      "  - testthat, type: imports\n",
      "  - methods, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/runibic.html\n",
      "\n",
      "Name: scHOT\n",
      "Description:\n",
      "Single cell Higher Order Testing (scHOT) is an R package that facilitates testing changes in higher order structure of gene expression along either a developmental trajectory or across space. scHOT is general and modular in nature, can be run in multiple data contexts such as along a continuous trajectory, between discrete groups, and over spatial orientations; as well as accommodate any higher order measurement such as variability or correlation. scHOT meaningfully adds to first order effect testing, such as differential expression, and provides a framework for interrogating higher order interactions from single cell data.\n",
      "Version: 1.10.0\n",
      "Publication date: None\n",
      "Mantainer: Shila Ghazanfar <shazanfar@gmail.com>\n",
      "Authors:\n",
      "Author: Shila Ghazanfar [aut, cre], Yingxin Lin [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - S4Vectors, type: imports\n",
      "  - SingleCellExperiment, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - reshape, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - igraph, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - ggforce, type: imports\n",
      "  - graphics, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/scHOT.html\n",
      "\n",
      "Name: subSeq\n",
      "Description:\n",
      "Subsampling of high throughput sequencing count data for use in experiment design and analysis.\n",
      "Version: 1.28.0\n",
      "Publication date: None\n",
      "Mantainer: Andrew J. Bass <ajbass@princeton.edu>, John D. Storey <jstorey@princeton.edu>\n",
      "Authors:\n",
      "Author: David Robinson, John D. Storey, with contributions from Andrew J. Bass\n",
      "Dependencies:\n",
      "  - R (>= 3.2) type: depends\n",
      "  - data.table, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - qvalue, type: imports\n",
      "  - digest, type: imports\n",
      "  - Biobase, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/subSeq.html\n",
      "\n",
      "Name: BADER\n",
      "Description:\n",
      "For RNA sequencing count data, BADER fits a Bayesian hierarchical model. The algorithm returns the posterior probability of differential expression for each gene between two groups A and B. The joint posterior distribution of the variables in the model can be returned in the form of posterior samples, which can be used for further down-stream analyses such as gene set enrichment.\n",
      "Version: 1.36.0\n",
      "Publication date: None\n",
      "Mantainer: Andreas Neudecker <a.neudecker@arcor.de>\n",
      "Authors:\n",
      "Author: Andreas Neudecker, Matthias Katzfuss\n",
      "Dependencies:\n",
      "None\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/BADER.html\n",
      "\n",
      "Name: BCRANK\n",
      "Description:\n",
      "Functions and classes for de novo prediction of transcription factor binding consensus by heuristic search\n",
      "Version: 1.60.0\n",
      "Publication date: None\n",
      "Mantainer: Adam Ameur <Adam.Ameur@genpat.uu.se>\n",
      "Authors:\n",
      "Author: Adam Ameur <Adam.Ameur at genpat.uu.se>\n",
      "Dependencies:\n",
      "  - methods, type: depends\n",
      "  - Biostrings, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/BCRANK.html\n",
      "\n",
      "Name: cancerclass\n",
      "Description:\n",
      "The classification protocol starts with a feature selection step and continues with nearest-centroid classification. The accurarcy of the predictor can be evaluated using training and test set validation, leave-one-out cross-validation or in a multiple random validation protocol. Methods for calculation and visualization of continuous prediction scores allow to balance sensitivity and specificity and define a cutoff value according to clinical requirements.\n",
      "Version: 1.42.0\n",
      "Publication date: None\n",
      "Mantainer: Daniel Kosztyla <danielkossi@hotmail.com>\n",
      "Authors:\n",
      "Author: Jan Budczies, Daniel Kosztyla\n",
      "Dependencies:\n",
      "  - R (>= 2.14.0) type: depends\n",
      "  - Biobase, type: depends\n",
      "  - binom, type: depends\n",
      "  - methods, type: depends\n",
      "  - stats, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL 3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/cancerclass.html\n",
      "\n",
      "Name: CoRegNet\n",
      "Description:\n",
      "This package provides methods to identify active transcriptional programs. Methods and classes are provided to import or infer large scale co-regulatory network from transcriptomic data. The specificity of the encoded networks is to model Transcription Factor cooperation. External regulation evidences (TFBS, ChIP,...) can be integrated to assess the inferred network and refine it if necessary. Transcriptional activity of the regulators in the network can be estimated using an measure of their influence in a given sample. Finally, an interactive UI can be used to navigate through the network of cooperative regulators and to visualize their activity in a specific sample or subgroup sample. The proposed visualization tool can be used to integrate gene expression, transcriptional activity, copy number status, sample classification and a transcriptional network including co-regulation information.\n",
      "Version: 1.36.0\n",
      "Publication date: None\n",
      "Mantainer: Remy Nicolle <remy.c.nicolle@gmail.com>\n",
      "Authors:\n",
      "Author: Remy Nicolle, Thibault Venzac and Mohamed Elati\n",
      "Dependencies:\n",
      "  - R (>= 2.14) type: depends\n",
      "  - igraph, type: depends\n",
      "  - shiny, type: depends\n",
      "  - arules, type: depends\n",
      "  - methods, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/CoRegNet.html\n",
      "\n",
      "Name: cosmosR\n",
      "Description:\n",
      "COSMOS (Causal Oriented Search of Multi-Omic Space) is a method that integrates phosphoproteomics, transcriptomics, and metabolomics data sets based on prior knowledge of signaling, metabolic, and gene regulatory networks. It estimated the activities of transcrption factors and kinases and finds a network-level causal reasoning. Thereby, COSMOS provides mechanistic hypotheses for experimental observations across mulit-omics datasets.\n",
      "Version: 1.6.0\n",
      "Publication date: None\n",
      "Mantainer: Attila Gabor <attila.gabor@uni-heidelberg.de>\n",
      "Authors:\n",
      "Author: AurÃ©lien Dugourd [aut] , Attila Gabor [cre] , Katharina Zirngibl [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - CARNIVAL, type: imports\n",
      "  - dorothea, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - GSEABase, type: imports\n",
      "  - igraph, type: imports\n",
      "  - progress, type: imports\n",
      "  - purrr, type: imports\n",
      "  - rlang, type: imports\n",
      "  - stringr, type: imports\n",
      "  - utils, type: imports\n",
      "  - visNetwork, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/cosmosR.html\n",
      "\n",
      "Name: EBcoexpress\n",
      "Description:\n",
      "An Empirical Bayesian Approach to Differential Co-Expression Analysis at the Gene-Pair Level\n",
      "Version: 1.42.0\n",
      "Publication date: None\n",
      "Mantainer: John A. Dawson <jadawson@wisc.edu>\n",
      "Authors:\n",
      "Author: John A. Dawson\n",
      "Dependencies:\n",
      "  - EBarrays, type: depends\n",
      "  - mclust, type: depends\n",
      "  - minqa, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/EBcoexpress.html\n",
      "\n",
      "Name: flowMerge\n",
      "Description:\n",
      "Merging of mixture components for model-based automated gating of flow cytometry data using the flowClust framework. Note: users should have a working copy of flowClust 2.0 installed.\n",
      "Version: 2.46.0\n",
      "Publication date: None\n",
      "Mantainer: Greg Finak <gfinak@fhcrc.org>\n",
      "Authors:\n",
      "Author: Greg Finak <gfinak at fhcrc.org>, Raphael Gottardo <rgottard at fhcrc.org>\n",
      "Dependencies:\n",
      "  - graph, type: depends\n",
      "  - feature, type: depends\n",
      "  - flowClust, type: depends\n",
      "  - Rgraphviz, type: depends\n",
      "  - foreach, type: depends\n",
      "  - snow, type: depends\n",
      "  - rrcov, type: imports\n",
      "  - flowCore, type: imports\n",
      "  - graphics, type: imports\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/flowMerge.html\n",
      "\n",
      "Name: flowTrans\n",
      "Description:\n",
      "Profile maximum likelihood estimation of parameters for flow cytometry data transformations.\n",
      "Version: 1.50.0\n",
      "Publication date: None\n",
      "Mantainer: Greg Finak <gfinak@fredhutch.org>\n",
      "Authors:\n",
      "Author: Greg Finak <gfinak at fredhutch.org>, Juan Manuel-Perez <jperez at ircm.qc.ca>, Raphael Gottardo <rgottard at fredhutch.org>\n",
      "Dependencies:\n",
      "  - R (>= 2.11.0) type: depends\n",
      "  - flowCore, type: depends\n",
      "  - flowViz, type: depends\n",
      "  - flowClust, type: depends\n",
      "  - flowCore, type: imports\n",
      "  - methods, type: imports\n",
      "  - flowViz, type: imports\n",
      "  - stats, type: imports\n",
      "  - flowClust, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/flowTrans.html\n",
      "\n",
      "Name: goTools\n",
      "Description:\n",
      "Wraper functions for description/comparison of oligo ID list using Gene Ontology database\n",
      "Version: 1.72.0\n",
      "Publication date: None\n",
      "Mantainer: Agnes Paquet <paquetagnes@yahoo.com>\n",
      "Authors:\n",
      "Author: Yee Hwa (Jean) Yang <jean at biostat.ucsf.edu>, Agnes Paquet <paquetagnes at yahoo.com>\n",
      "Dependencies:\n",
      "  - GO.db, type: depends\n",
      "  - AnnotationDbi, type: imports\n",
      "  - GO.db, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/goTools.html\n",
      "\n",
      "Name: GSEABenchmarkeR\n",
      "Description:\n",
      "The GSEABenchmarkeR package implements an extendable framework for reproducible evaluation of set- and network-based methods for enrichment analysis of gene expression data. This includes support for the efficient execution of these methods on comprehensive real data compendia (microarray and RNA-seq) using parallel computation on standard workstations and institutional computer grids. Methods can then be assessed with respect to runtime, statistical significance, and relevance of the results for the phenotypes investigated.\n",
      "Version: 1.18.0\n",
      "Publication date: None\n",
      "Mantainer: Ludwig Geistlinger <Ludwig_Geistlinger@hms.harvard.edu>\n",
      "Authors:\n",
      "Author: Ludwig Geistlinger [aut, cre], Gergely Csaba [aut], Mara Santarelli [ctb], Lucas Schiffer [ctb], Marcel Ramos [ctb], Ralf Zimmer [aut], Levi Waldron [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - Biobase, type: depends\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - AnnotationDbi, type: imports\n",
      "  - AnnotationHub, type: imports\n",
      "  - BiocFileCache, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - edgeR, type: imports\n",
      "  - EnrichmentBrowser, type: imports\n",
      "  - ExperimentHub, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - KEGGandMetacoreDzPathwaysGEO, type: imports\n",
      "  - KEGGdzPathwaysGEO, type: imports\n",
      "  - methods, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/GSEABenchmarkeR.html\n",
      "\n",
      "Name: GUIDEseq\n",
      "Description:\n",
      "The package implements GUIDE-seq and PEtag-seq analysis workflow including functions for filtering UMI and reads with low coverage, obtaining unique insertion sites (proxy of cleavage sites), estimating the locations of the insertion sites, aka, peaks, merging estimated insertion sites from plus and minus strand, and performing off target search of the extended regions around insertion sites.\n",
      "Version: 1.28.0\n",
      "Publication date: None\n",
      "Mantainer: Lihua Julie Zhu <julie.zhu@umassmed.edu>\n",
      "Authors:\n",
      "Author: Lihua Julie Zhu, Michael Lawrence, Ankit Gupta, HervÃ© PagÃ¨s , Alper Kucukural, Manuel Garber, Scot A. Wolfe\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - BiocGenerics, type: depends\n",
      "  - Biostrings, type: imports\n",
      "  - CRISPRseek, type: imports\n",
      "  - ChIPpeakAnno, type: imports\n",
      "  - data.table, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - BSgenome, type: imports\n",
      "  - parallel, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - stringr, type: imports\n",
      "  - multtest, type: imports\n",
      "  - GenomicAlignments, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - hash, type: imports\n",
      "  - limma, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - GenomicFeatures, type: imports\n",
      "  - rio, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - tools, type: imports\n",
      "  - methods, type: imports\n",
      "  - purrr, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - openxlsx, type: imports\n",
      "  - patchwork, type: imports\n",
      "  - rlang, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/GUIDEseq.html\n",
      "\n",
      "Name: methylSig\n",
      "Description:\n",
      "MethylSig is a package for testing for differentially methylated cytosines (DMCs) or regions (DMRs) in whole-genome bisulfite sequencing (WGBS) or reduced representation bisulfite sequencing (RRBS) experiments. MethylSig uses a beta binomial model to test for significant differences between groups of samples. Several options exist for either site-specific or sliding window tests, and variance estimation.\n",
      "Version: 1.10.0\n",
      "Publication date: None\n",
      "Mantainer: Raymond G. Cavalcante <rcavalca@umich.edu>\n",
      "Authors:\n",
      "Author: Yongseok Park [aut], Raymond G. Cavalcante [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 3.6) type: depends\n",
      "  - bsseq, type: imports\n",
      "  - DelayedArray, type: imports\n",
      "  - DelayedMatrixStats, type: imports\n",
      "  - DSS, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - methods, type: imports\n",
      "  - parallel, type: imports\n",
      "  - stats, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/methylSig.html\n",
      "\n",
      "Name: netresponse\n",
      "Description:\n",
      "Algorithms for functional network analysis. Includes an implementation of a variational Dirichlet process Gaussian mixture model for nonparametric mixture modeling.\n",
      "Version: 1.58.0\n",
      "Publication date: None\n",
      "Mantainer: Leo Lahti <leo.lahti@iki.fi>\n",
      "Authors:\n",
      "Author: Leo Lahti, Olli-Pekka Huovilainen, Antonio Gusmao and Juuso Parkkinen\n",
      "Dependencies:\n",
      "  - R (>= 2.15.1) type: depends\n",
      "  - BiocStyle, type: depends\n",
      "  - Rgraphviz, type: depends\n",
      "  - rmarkdown, type: depends\n",
      "  - methods, type: depends\n",
      "  - minet, type: depends\n",
      "  - mclust, type: depends\n",
      "  - reshape2, type: depends\n",
      "  - ggplot2, type: imports\n",
      "  - graph, type: imports\n",
      "  - igraph, type: imports\n",
      "  - parallel, type: imports\n",
      "  - plyr, type: imports\n",
      "  - qvalue, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>=2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/netresponse.html\n",
      "\n",
      "Name: nucleR\n",
      "Description:\n",
      "Nucleosome positioning for Tiling Arrays and NGS experiments.\n",
      "Version: 2.30.0\n",
      "Publication date: None\n",
      "Mantainer: Alba Sala <alba.sala@irbbarcelona.org>\n",
      "Authors:\n",
      "Author: Oscar Flores, Ricard Illa\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - Biobase, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - ShortRead, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - parallel, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - grDevices, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL (>= 3)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/nucleR.html\n",
      "\n",
      "Name: PhyloProfile\n",
      "Description:\n",
      "PhyloProfile is a tool for exploring complex phylogenetic profiles. Phylogenetic profiles, presence/absence patterns of genes over a set of species, are commonly used to trace the functional and evolutionary history of genes across species and time. With PhyloProfile we can enrich regular phylogenetic profiles with further data like sequence/structure similarity, to make phylogenetic profiling more meaningful. Besides the interactive visualisation powered by R-Shiny, the package offers a set of further analysis features to gain insights like the gene age estimation or core gene identification.\n",
      "Version: 1.12.5\n",
      "Publication date: None\n",
      "Mantainer: Vinh Tran <tran@bio.uni-frankfurt.de>\n",
      "Authors:\n",
      "Author: Vinh Tran [aut, cre] , Bastian Greshake Tzovaras [aut], Ingo Ebersberger [aut], Carla MÃ¶lbert [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 4.2.0) type: depends\n",
      "  - ape, type: imports\n",
      "  - bioDist, type: imports\n",
      "  - BiocStyle, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - colourpicker, type: imports\n",
      "  - data.table, type: imports\n",
      "  - DT, type: imports\n",
      "  - energy, type: imports\n",
      "  - ExperimentHub, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - gridExtra, type: imports\n",
      "  - pbapply, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - RCurl, type: imports\n",
      "  - shiny, type: imports\n",
      "  - shinyBS, type: imports\n",
      "  - shinyFiles, type: imports\n",
      "  - shinyjs, type: imports\n",
      "  - OmaDB, type: imports\n",
      "  - plyr, type: imports\n",
      "  - xml2, type: imports\n",
      "  - zoo, type: imports\n",
      "  - yaml, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/PhyloProfile.html\n",
      "\n",
      "Name: ProteoMM\n",
      "Description:\n",
      "ProteoMM is a statistical method to perform model-based peptide-level differential expression analysis of single or multiple datasets. For multiple datasets ProteoMM produces a single fold change and p-value for each protein across multiple datasets. ProteoMM provides functionality for normalization, missing value imputation and differential expression. Model-based peptide-level imputation and differential expression analysis component of package follows the analysis described in âA statistical framework for protein quantitation in bottom-up MS based proteomics\" (Karpievitch et al. Bioinformatics 2009). EigenMS normalisation is implemented as described in \"Normalization of peak intensities in bottom-up MS-based proteomics using singular value decomposition.\" (Karpievitch et al. Bioinformatics 2009).\n",
      "Version: 1.16.0\n",
      "Publication date: None\n",
      "Mantainer: Yuliya V Karpievitch <yuliya.k@gmail.com>\n",
      "Authors:\n",
      "Author: Yuliya V Karpievitch, Tim Stuart and Sufyaan Mohamed\n",
      "Dependencies:\n",
      "  - R (>= 3.5) type: depends\n",
      "  - gdata, type: imports\n",
      "  - biomaRt, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - ggrepel, type: imports\n",
      "  - gtools, type: imports\n",
      "  - stats, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - graphics, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ProteoMM.html\n",
      "\n",
      "Name: Rcade\n",
      "Description:\n",
      "Bioconductor version: Release (3.16)\n",
      "Version: 1.39.1\n",
      "Publication date: None\n",
      "Mantainer: Author: Jonathan Cairns\n",
      "Authors:\n",
      "Rcade (which stands for \"R-based analysis of ChIP-seq And Differential Expression\") is a tool for integrating ChIP-seq data with differential expression summary data, through a Bayesian framework. A key application is in identifing the genes targeted by a transcription factor of interest - that is, we collect genes that are associated with a ChIP-seq peak, and differential expression under some perturbation related to that TF.\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - Rsamtools, type: depends\n",
      "  - baySeq, type: depends\n",
      "  - utils, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - stats, type: imports\n",
      "  - graphics, type: imports\n",
      "  - rgl, type: imports\n",
      "  - plotrix, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - GenomicAlignments, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/Rcade.html\n",
      "\n",
      "Name: RegEnrich\n",
      "Description:\n",
      "This package is a pipeline to identify the key gene regulators in a biological process, for example in cell differentiation and in cell development after stimulation. There are four major steps in this pipeline: (1) differential expression analysis; (2) regulator-target network inference; (3) enrichment analysis; and (4) regulators scoring and ranking.\n",
      "Version: 1.8.0\n",
      "Publication date: None\n",
      "Mantainer: Weiyang Tao <weiyangtao1513@gmail.com>\n",
      "Authors:\n",
      "Author: Weiyang Tao [cre, aut], Aridaman Pandit [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.0.0) type: depends\n",
      "  - S4Vectors, type: depends\n",
      "  - dplyr, type: depends\n",
      "  - tibble, type: depends\n",
      "  - BiocSet, type: depends\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - randomForest, type: imports\n",
      "  - fgsea, type: imports\n",
      "  - DOSE, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - DESeq2, type: imports\n",
      "  - limma, type: imports\n",
      "  - WGCNA, type: imports\n",
      "  - ggplot2 (>= 2.2.0) type: imports\n",
      "  - methods, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - magrittr, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/RegEnrich.html\n",
      "\n",
      "Name: seqCNA\n",
      "Description:\n",
      "Copy number analysis of high-throughput sequencing cancer data with fast summarization, extensive filtering and improved normalization\n",
      "Version: 1.44.0\n",
      "Publication date: None\n",
      "Mantainer: David Mosen-Ansorena <dmosen.gn@cicbiogune.es>\n",
      "Authors:\n",
      "Author: David Mosen-Ansorena\n",
      "Dependencies:\n",
      "  - R (>= 3.0) type: depends\n",
      "  - GLAD, type: depends\n",
      "  - doSNOW (>= 1.0.5) type: depends\n",
      "  - adehabitatLT (>= 0.3.4) type: depends\n",
      "  - seqCNA.annot, type: depends\n",
      "  - methods, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/seqCNA.html\n",
      "\n",
      "Name: SPEM\n",
      "Description:\n",
      "This package can optimize the parameter in S-system models given time series data\n",
      "Version: 1.38.0\n",
      "Publication date: None\n",
      "Mantainer: Xinyi YANG <yangxinyi@picb.ac.cn>\n",
      "Authors:\n",
      "Author: Xinyi YANG Developer, Jennifer E. DENT Developer and Christine NARDINI Supervisor\n",
      "Dependencies:\n",
      "  - R (>= 2.15.1) type: depends\n",
      "  - Rsolnp, type: depends\n",
      "  - Biobase, type: depends\n",
      "  - methods, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/SPEM.html\n",
      "\n",
      "Name: TMixClust\n",
      "Description:\n",
      "Implementation of a clustering method for time series gene expression data based on mixed-effects models with Gaussian variables and non-parametric cubic splines estimation. The method can robustly account for the high levels of noise present in typical gene expression time series datasets.\n",
      "Version: 1.20.0\n",
      "Publication date: None\n",
      "Mantainer: Monica Golumbeanu <golumbeanu.monica@gmail.com>\n",
      "Authors:\n",
      "Author: Monica Golumbeanu <golumbeanu.monica at gmail.com>\n",
      "Dependencies:\n",
      "  - R (>= 3.4) type: depends\n",
      "  - gss, type: imports\n",
      "  - mvtnorm, type: imports\n",
      "  - stats, type: imports\n",
      "  - zoo, type: imports\n",
      "  - cluster, type: imports\n",
      "  - utils, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - flexclust, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - SPEM, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>=2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/TMixClust.html\n",
      "\n",
      "Name: tRNAscanImport\n",
      "Description:\n",
      "The package imports the result of tRNAscan-SE as a GRanges object.\n",
      "Version: 1.18.0\n",
      "Publication date: None\n",
      "Mantainer: Felix G.M. Ernst <felix.gm.ernst@outlook.com>\n",
      "Authors:\n",
      "Author: Felix G.M. Ernst [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 3.5) type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - tRNA, type: depends\n",
      "  - methods, type: imports\n",
      "  - stringr, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - Structstrings, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - XVector, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - BSgenome, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3 + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/tRNAscanImport.html\n",
      "\n",
      "Name: altcdfenvs\n",
      "Description:\n",
      "Convenience data structures and functions to handle cdfenvs\n",
      "Version: 2.60.0\n",
      "Publication date: None\n",
      "Mantainer: Laurent Gautier <lgautier@gmail.com>\n",
      "Authors:\n",
      "Author: Laurent Gautier <lgautier at gmail.com>\n",
      "Dependencies:\n",
      "  - R (>= 2.7) type: depends\n",
      "  - methods, type: depends\n",
      "  - BiocGenerics, type: depends\n",
      "  - S4Vectors, type: depends\n",
      "  - Biobase, type: depends\n",
      "  - affy, type: depends\n",
      "  - makecdfenv, type: depends\n",
      "  - Biostrings, type: depends\n",
      "  - hypergraph, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/altcdfenvs.html\n",
      "\n",
      "Name: anota2seq\n",
      "Description:\n",
      "anota2seq provides analysis of translational efficiency and differential expression analysis for polysome-profiling and ribosome-profiling studies (two or more sample classes) quantified by RNA sequencing or DNA-microarray. Polysome-profiling and ribosome-profiling typically generate data for two RNA sources; translated mRNA and total mRNA. Analysis of differential expression is used to estimate changes within each RNA source (i.e. translated mRNA or total mRNA). Analysis of translational efficiency aims to identify changes in translation efficiency leading to altered protein levels that are independent of total mRNA levels (i.e. changes in translated mRNA that are independent of levels of total mRNA) or buffering, a mechanism regulating translational efficiency so that protein levels remain constant despite fluctuating total mRNA levels (i.e. changes in total mRNA that are independent of levels of translated mRNA). anota2seq applies analysis of partial variance and the random variance model to fulfill these tasks.\n",
      "Version: 1.20.0\n",
      "Publication date: None\n",
      "Mantainer: Christian Oertlin <christian.oertlin@ki.se>, Julie Lorent <julie.lorent@ki.se>\n",
      "Authors:\n",
      "Author: Christian Oertlin <christian.oertlin at ki.se>, Julie Lorent <julie.lorent at ki.se>, Ola Larsson <ola.larsson at ki.se>\n",
      "Dependencies:\n",
      "  - R (>= 3.4.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - multtest, type: imports\n",
      "  - qvalue, type: imports\n",
      "  - limma, type: imports\n",
      "  - DESeq2, type: imports\n",
      "  - edgeR, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/anota2seq.html\n",
      "\n",
      "Name: BANDITS\n",
      "Description:\n",
      "BANDITS is a Bayesian hierarchical model for detecting differential splicing of genes and transcripts, via differential transcript usage (DTU), between two or more conditions. The method uses a Bayesian hierarchical framework, which allows for sample specific proportions in a Dirichlet-Multinomial model, and samples the allocation of fragments to the transcripts. Parameters are inferred via Markov chain Monte Carlo (MCMC) techniques and a DTU test is performed via a multivariate Wald test on the posterior densities for the average relative abundance of transcripts.\n",
      "Version: 1.14.2\n",
      "Publication date: None\n",
      "Mantainer: Simone Tiberi <simone.tiberi@uzh.ch>\n",
      "Authors:\n",
      "Author: Simone Tiberi [aut, cre].\n",
      "Dependencies:\n",
      "  - R (>= 4.0.0) type: depends\n",
      "  - Rcpp, type: imports\n",
      "  - doRNG, type: imports\n",
      "  - MASS, type: imports\n",
      "  - data.table, type: imports\n",
      "  - R.utils, type: imports\n",
      "  - doParallel, type: imports\n",
      "  - parallel, type: imports\n",
      "  - foreach, type: imports\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - graphics, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - DRIMSeq, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 3)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/BANDITS.html\n",
      "\n",
      "Name: bioCancer\n",
      "Description:\n",
      "bioCancer is a Shiny App to visualize and analyse interactively Multi-Assays of Cancer Genomic Data.\n",
      "Version: 1.26.04\n",
      "Publication date: None\n",
      "Mantainer: Karim Mezhoud <kmezhoud@gmail.com>\n",
      "Authors:\n",
      "Author: Karim Mezhoud [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 3.6.0) type: depends\n",
      "  - radiant.data (>= 0.9.1) type: depends\n",
      "  - XML (>= 3.98) type: depends\n",
      "  - R.oo, type: imports\n",
      "  - httr, type: imports\n",
      "  - DT (>= 0.3) type: imports\n",
      "  - dplyr (>= 0.7.2) type: imports\n",
      "  - shiny (>= 1.0.5) type: imports\n",
      "  - AnnotationDbi, type: imports\n",
      "  - methods, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - geNetClassifier, type: imports\n",
      "  - org.Hs.eg.db, type: imports\n",
      "  - DOSE, type: imports\n",
      "  - clusterProfiler, type: imports\n",
      "  - reactome.db, type: imports\n",
      "  - ReactomePA, type: imports\n",
      "  - DiagrammeR (<= 1.01) type: imports\n",
      "  - visNetwork, type: imports\n",
      "  - htmlwidgets, type: imports\n",
      "  - plyr, type: imports\n",
      "  - tibble, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "AGPL-3 | file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/bioCancer.html\n",
      "\n",
      "Name: BiocOncoTK\n",
      "Description:\n",
      "Provide a central interface to various tools for genome-scale analysis of cancer studies.\n",
      "Version: 1.18.0\n",
      "Publication date: None\n",
      "Mantainer: VJ Carey <stvjc@channing.harvard.edu>\n",
      "Authors:\n",
      "Author: Vince Carey\n",
      "Dependencies:\n",
      "  - R (>= 3.6.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - utils, type: depends\n",
      "  - ComplexHeatmap, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - bigrquery, type: imports\n",
      "  - shiny, type: imports\n",
      "  - stats, type: imports\n",
      "  - httr, type: imports\n",
      "  - rjson, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - grid, type: imports\n",
      "  - DT, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - DBI, type: imports\n",
      "  - GenomicFeatures, type: imports\n",
      "  - curatedTCGAData, type: imports\n",
      "  - scales, type: imports\n",
      "  - ggpubr, type: imports\n",
      "  - plyr, type: imports\n",
      "  - car, type: imports\n",
      "  - graph, type: imports\n",
      "  - Rgraphviz, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/BiocOncoTK.html\n",
      "\n",
      "Name: BioCor\n",
      "Description:\n",
      "Calculates functional similarities based on the pathways described on KEGG and REACTOME or in gene sets. These similarities can be calculated for pathways or gene sets, genes, or clusters and combined with other similarities. They can be used to improve networks, gene selection, testing relationships...\n",
      "Version: 1.22.0\n",
      "Publication date: None\n",
      "Mantainer: LluÃ­s Revilla Sancho <lluis.revilla@gmail.com>\n",
      "Authors:\n",
      "Author: LluÃ­s Revilla Sancho [aut, cre] , Pau Sancho-Bru [ths] , Juan JosÃ© Salvatella Lozano [ths]\n",
      "Dependencies:\n",
      "  - R (>= 3.4.0) type: depends\n",
      "  - BiocParallel, type: imports\n",
      "  - GSEABase, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - methods, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/BioCor.html\n",
      "\n",
      "Name: EMDomics\n",
      "Description:\n",
      "The EMDomics algorithm is used to perform a supervised multi-class analysis to measure the magnitude and statistical significance of observed continuous genomics data between groups. Usually the data will be gene expression values from array-based or sequence-based experiments, but data from other types of experiments can also be analyzed (e.g. copy number variation). Traditional methods like Significance Analysis of Microarrays (SAM) and Linear Models for Microarray Data (LIMMA) use significance tests based on summary statistics (mean and standard deviation) of the distributions. This approach lacks power to identify expression differences between groups that show high levels of intra-group heterogeneity. The Earth Mover's Distance (EMD) algorithm instead computes the \"work\" needed to transform one distribution into another, thus providing a metric of the overall difference in shape between two distributions. Permutation of sample labels is used to generate q-values for the observed EMD scores. This package also incorporates the Komolgorov-Smirnov (K-S) test and the Cramer von Mises test (CVM), which are both common distribution comparison tests.\n",
      "Version: 2.28.0\n",
      "Publication date: None\n",
      "Mantainer: Sadhika Malladi <contact@sadhikamalladi.com> and Daniel Schmolze <emd@schmolze.com>\n",
      "Authors:\n",
      "Author: Sadhika Malladi [aut, cre], Daniel Schmolze [aut, cre], Andrew Beck [aut], Sheida Nabavi [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.2.1) type: depends\n",
      "  - emdist, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - CDFt, type: imports\n",
      "  - preprocessCore, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/EMDomics.html\n",
      "\n",
      "Name: epivizr\n",
      "Description:\n",
      "This package provides connections to the epiviz web app (http://epiviz.cbcb.umd.edu) for interactive visualization of genomic data. Objects in R/bioc interactive sessions can be displayed in genome browser tracks or plots to be explored by navigation through genomic regions. Fundamental Bioconductor data structures are supported (e.g., GenomicRanges and RangedSummarizedExperiment objects), while providing an easy mechanism to support other data structures (through package epivizrData). Visualizations (using d3.js) can be easily added to the web app as well.\n",
      "Version: 2.28.0\n",
      "Publication date: None\n",
      "Mantainer: Hector Corrada Bravo <hcorrada@gmail.com>\n",
      "Authors:\n",
      "Author: Hector Corrada Bravo, Florin Chelaru, Llewellyn Smith, Naomi Goldstein, Jayaram Kancherla, Morgan Walter, Brian Gottfried\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - epivizrServer, type: imports\n",
      "  - epivizrData, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - bumphunter, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/epivizr.html\n",
      "\n",
      "Name: hyperdraw\n",
      "Description:\n",
      "Functions for visualizing hypergraphs.\n",
      "Version: 1.50.0\n",
      "Publication date: None\n",
      "Mantainer: Paul Murrell <p.murrell@auckland.ac.nz>\n",
      "Authors:\n",
      "Author: Paul Murrell\n",
      "Dependencies:\n",
      "  - R (>= 2.9.0) type: depends\n",
      "  - methods, type: imports\n",
      "  - grid, type: imports\n",
      "  - graph, type: imports\n",
      "  - hypergraph, type: imports\n",
      "  - Rgraphviz, type: imports\n",
      "  - stats4, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/hyperdraw.html\n",
      "\n",
      "Name: metagene2\n",
      "Description:\n",
      "This package produces metagene plots to compare coverages of sequencing experiments at selected groups of genomic regions. It can be used for such analyses as assessing the binding of DNA-interacting proteins at promoter regions or surveying antisense transcription over the length of a gene. The metagene2 package can manage all aspects of the analysis, from normalization of coverages to plot facetting according to experimental metadata. Bootstraping analysis is used to provide confidence intervals of per-sample mean coverages.\n",
      "Version: 1.14.0\n",
      "Publication date: None\n",
      "Mantainer: Eric Fournier <ericfournier2@yahoo.ca>\n",
      "Authors:\n",
      "Author: Eric Fournier [cre, aut], Charles Joly Beauparlant [aut], Cedric Lippens [aut], Arnaud Droit [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - R6 (>= 2.0) type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - BiocParallel, type: depends\n",
      "  - rtracklayer, type: imports\n",
      "  - tools, type: imports\n",
      "  - GenomicAlignments, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - purrr, type: imports\n",
      "  - data.table, type: imports\n",
      "  - methods, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - reshape2, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/metagene2.html\n",
      "\n",
      "Name: metaSeq\n",
      "Description:\n",
      "The probabilities by one-sided NOISeq are combined by Fisher's method or Stouffer's method\n",
      "Version: 1.38.0\n",
      "Publication date: None\n",
      "Mantainer: Koki Tsuyuzaki <k.t.the-answer@hotmail.co.jp>\n",
      "Authors:\n",
      "Author: Koki Tsuyuzaki, Itoshi Nikaido\n",
      "Dependencies:\n",
      "  - R (>= 2.13.0) type: depends\n",
      "  - NOISeq, type: depends\n",
      "  - snow, type: depends\n",
      "  - Rcpp, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/metaSeq.html\n",
      "\n",
      "Name: methylCC\n",
      "Description:\n",
      "A tool to estimate the cell composition of DNA methylation whole blood sample measured on any platform technology (microarray and sequencing).\n",
      "Version: 1.12.0\n",
      "Publication date: None\n",
      "Mantainer: Stephanie C. Hicks <shicks19@jhu.edu>\n",
      "Authors:\n",
      "Author: Stephanie C. Hicks [aut, cre] , Rafael Irizarry [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.6) type: depends\n",
      "  - FlowSorted.Blood.450k, type: depends\n",
      "  - Biobase, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - minfi, type: imports\n",
      "  - bsseq, type: imports\n",
      "  - quadprog, type: imports\n",
      "  - plyranges, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - bumphunter, type: imports\n",
      "  - genefilter, type: imports\n",
      "  - methods, type: imports\n",
      "  - IlluminaHumanMethylation450kmanifest, type: imports\n",
      "  - IlluminaHumanMethylation450kanno.ilmn12.hg19, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "CC BY 4.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/methylCC.html\n",
      "\n",
      "Name: MineICA\n",
      "Description:\n",
      "The goal of MineICA is to perform Independent Component Analysis (ICA) on multiple transcriptome datasets, integrating additional data (e.g molecular, clinical and pathological). This Integrative ICA helps the biological interpretation of the components by studying their association with variables (e.g sample annotations) and gene sets, and enables the comparison of components from different datasets using correlation-based graph.\n",
      "Version: 1.38.0\n",
      "Publication date: None\n",
      "Mantainer: Anne Biton <anne.biton@gmail.com>\n",
      "Authors:\n",
      "Author: Anne Biton\n",
      "Dependencies:\n",
      "  - R (>= 2.10) type: depends\n",
      "  - methods, type: depends\n",
      "  - BiocGenerics, type: depends\n",
      "  - Biobase, type: depends\n",
      "  - plyr, type: depends\n",
      "  - ggplot2, type: depends\n",
      "  - scales, type: depends\n",
      "  - foreach, type: depends\n",
      "  - xtable, type: depends\n",
      "  - biomaRt, type: depends\n",
      "  - gtools, type: depends\n",
      "  - GOstats, type: depends\n",
      "  - cluster, type: depends\n",
      "  - marray, type: depends\n",
      "  - mclust, type: depends\n",
      "  - RColorBrewer, type: depends\n",
      "  - colorspace, type: depends\n",
      "  - igraph, type: depends\n",
      "  - Rgraphviz, type: depends\n",
      "  - graph, type: depends\n",
      "  - annotate, type: depends\n",
      "  - Hmisc, type: depends\n",
      "  - fastICA, type: depends\n",
      "  - JADE, type: depends\n",
      "  - AnnotationDbi, type: imports\n",
      "  - lumi, type: imports\n",
      "  - fpc, type: imports\n",
      "  - lumiHumanAll.db, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/MineICA.html\n",
      "\n",
      "Name: odseq\n",
      "Description:\n",
      "Performs outlier detection of sequences in a multiple sequence alignment using bootstrap of predefined distance metrics. Outlier sequences can make downstream analyses unreliable or make the alignments less accurate while they are being constructed. This package implements the OD-seq algorithm proposed by Jehl et al (doi 10.1186/s12859-015-0702-1) for aligned sequences and a variant using string kernels for unaligned sequences.\n",
      "Version: 1.26.0\n",
      "Publication date: None\n",
      "Mantainer: JosÃ© JimÃ©nez <jose@jimenezluna.com>\n",
      "Authors:\n",
      "Author: JosÃ© JimÃ©nez\n",
      "Dependencies:\n",
      "  - R (>= 3.2.3) type: depends\n",
      "  - msa, type: imports\n",
      "  - kebabs, type: imports\n",
      "  - mclust (>= 5.1) type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/odseq.html\n",
      "\n",
      "Name: qckitfastq\n",
      "Description:\n",
      "Assessment of FASTQ file format with multiple metrics including quality score, sequence content, overrepresented sequence and Kmers.\n",
      "Version: 1.14.0\n",
      "Publication date: None\n",
      "Mantainer: August Guang <august.guang@gmail.com>\n",
      "Authors:\n",
      "Author: Wenyue Xing [aut], August Guang [aut, cre]\n",
      "Dependencies:\n",
      "  - magrittr, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - seqTools, type: imports\n",
      "  - zlibbioc, type: imports\n",
      "  - data.table, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - Rcpp, type: imports\n",
      "  - rlang, type: imports\n",
      "  - RSeqAn, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/qckitfastq.html\n",
      "\n",
      "Name: SpectralTAD\n",
      "Description:\n",
      "SpectralTAD is an R package designed to identify Topologically Associated Domains (TADs) from Hi-C contact matrices. It uses a modified version of spectral clustering that uses a sliding window to quickly detect TADs. The function works on a range of different formats of contact matrices and returns a bed file of TAD coordinates. The method does not require users to adjust any parameters to work and gives them control over the number of hierarchical levels to be returned.\n",
      "Version: 1.14.0\n",
      "Publication date: None\n",
      "Mantainer: Kellen Cresswell <cresswellkg@vcu.edu>\n",
      "Authors:\n",
      "Author: Kellen Cresswell <cresswellkg at vcu.edu>, John Stansfield <stansfieldjc at vcu.edu>, Mikhail Dozmorov <mikhail.dozmorov at vcuhealth.org>\n",
      "Dependencies:\n",
      "  - R (>= 3.6) type: depends\n",
      "  - dplyr, type: imports\n",
      "  - PRIMME, type: imports\n",
      "  - cluster, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - parallel, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - HiCcompare, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/SpectralTAD.html\n",
      "\n",
      "Name: splineTimeR\n",
      "Description:\n",
      "This package provides functions for differential gene expression analysis of gene expression time-course data. Natural cubic spline regression models are used. Identified genes may further be used for pathway enrichment analysis and/or the reconstruction of time dependent gene regulatory association networks.\n",
      "Version: 1.26.0\n",
      "Publication date: None\n",
      "Mantainer: Herbert Braselmann <hbraselmann@online.de>, Martin Selmansberger <martin.selmansberger@helmholtz-muenchen.de>\n",
      "Authors:\n",
      "Author: Agata Michna\n",
      "Dependencies:\n",
      "  - R (>= 3.3) type: depends\n",
      "  - Biobase, type: depends\n",
      "  - igraph, type: depends\n",
      "  - limma, type: depends\n",
      "  - GSEABase, type: depends\n",
      "  - gtools, type: depends\n",
      "  - splines, type: depends\n",
      "  - GeneNet (>= 1.2.13) type: depends\n",
      "  - longitudinal (>= 1.1.12) type: depends\n",
      "  - FIs, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/splineTimeR.html\n",
      "\n",
      "Name: TFEA.ChIP\n",
      "Description:\n",
      "Package to analize transcription factor enrichment in a gene set using data from ChIP-Seq experiments.\n",
      "Version: 1.18.0\n",
      "Publication date: None\n",
      "Mantainer: Laura Puente SantamarÃ­a <lpsantamaria@iib.uam.es>\n",
      "Authors:\n",
      "Author: Laura Puente SantamarÃ­a, Luis del Peso\n",
      "Dependencies:\n",
      "  - R (>= 3.5) type: depends\n",
      "  - GenomicRanges, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - biomaRt, type: imports\n",
      "  - GenomicFeatures, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - R.utils, type: imports\n",
      "  - methods, type: imports\n",
      "  - org.Hs.eg.db, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/TFEA.ChIP.html\n",
      "\n",
      "Name: timecourse\n",
      "Description:\n",
      "Functions for data analysis and graphical displays for developmental microarray time course data.\n",
      "Version: 1.70.0\n",
      "Publication date: None\n",
      "Mantainer: Yu Chuan Tai <yuchuan@stat.berkeley.edu>\n",
      "Authors:\n",
      "Author: Yu Chuan Tai\n",
      "Dependencies:\n",
      "  - R (>= 2.1.1) type: depends\n",
      "  - MASS, type: depends\n",
      "  - methods, type: depends\n",
      "  - Biobase, type: imports\n",
      "  - graphics, type: imports\n",
      "  - limma, type: imports\n",
      "  - MASS, type: imports\n",
      "  - marray, type: imports\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/timecourse.html\n",
      "\n",
      "Name: vissE\n",
      "Description:\n",
      "This package enables the interpretation and analysis of results from a gene set enrichment analysis using network-based and text-mining approaches. Most enrichment analyses result in large lists of significant gene sets that are difficult to interpret. Tools in this package help build a similarity-based network of significant gene sets from a gene set enrichment analysis that can then be investigated for their biological function using text-mining approaches.\n",
      "Version: 1.6.0\n",
      "Publication date: None\n",
      "Mantainer: Dharmesh D. Bhuva <bhuva.d@wehi.edu.au>\n",
      "Authors:\n",
      "Author: Dharmesh D. Bhuva [aut, cre] , Ahmed Mohamed [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - igraph, type: imports\n",
      "  - methods, type: imports\n",
      "  - plyr, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - scico, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - tm, type: imports\n",
      "  - ggwordcloud, type: imports\n",
      "  - GSEABase, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - ggforce, type: imports\n",
      "  - msigdb, type: imports\n",
      "  - ggrepel, type: imports\n",
      "  - textstem, type: imports\n",
      "  - tidygraph, type: imports\n",
      "  - stats, type: imports\n",
      "  - scales, type: imports\n",
      "  - ggraph, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/vissE.html\n",
      "\n",
      "Name: ABarray\n",
      "Description:\n",
      "Automated pipline to perform gene expression analysis for Applied Biosystems Genome Survey Microarray (AB1700) data format. Functions include data preprocessing, filtering, control probe analysis, statistical analysis in one single function. A GUI interface is also provided. The raw data, processed data, graphics output and statistical results are organized into folders according to the analysis settings used.\n",
      "Version: 1.66.0\n",
      "Publication date: None\n",
      "Mantainer: Yongming Andrew Sun <sunya@appliedbiosystems.com>\n",
      "Authors:\n",
      "Author: Yongming Andrew Sun\n",
      "Dependencies:\n",
      "  - Biobase, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - methods, type: imports\n",
      "  - multtest, type: imports\n",
      "  - stats, type: imports\n",
      "  - tcltk, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ABarray.html\n",
      "\n",
      "Name: adSplit\n",
      "Description:\n",
      "This package implements clustering of microarray gene expression profiles according to functional annotations. For each term genes are annotated to, splits into two subclasses are computed and a significance of the supporting gene set is determined.\n",
      "Version: 1.68.0\n",
      "Publication date: None\n",
      "Mantainer: Claudio Lottaz <Claudio.Lottaz@klinik.uni-regensburg.de>\n",
      "Authors:\n",
      "Author: Claudio Lottaz, Joern Toedling\n",
      "Dependencies:\n",
      "  - R (>= 2.1.0) type: depends\n",
      "  - methods (>= 2.1.0) type: depends\n",
      "  - AnnotationDbi, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - cluster (>= 1.9.1) type: imports\n",
      "  - GO.db, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - KEGGREST, type: imports\n",
      "  - multtest, type: imports\n",
      "  - stats (>= 2.1.0) type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/adSplit.html\n",
      "\n",
      "Name: CGHnormaliter\n",
      "Description:\n",
      "Normalization and centralization of array comparative genomic hybridization (aCGH) data. The algorithm uses an iterative procedure that effectively eliminates the influence of imbalanced copy numbers. This leads to a more reliable assessment of copy number alterations (CNAs).\n",
      "Version: 1.52.0\n",
      "Publication date: None\n",
      "Mantainer: Bart P.P. van Houte <bvhoute@few.vu.nl>\n",
      "Authors:\n",
      "Author: Bart P.P. van Houte, Thomas W. Binsl, Hannes Hettling\n",
      "Dependencies:\n",
      "  - CGHcall, type: depends\n",
      "  - CGHbase, type: depends\n",
      "  - Biobase, type: imports\n",
      "  - CGHbase, type: imports\n",
      "  - CGHcall, type: imports\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 3)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/CGHnormaliter.html\n",
      "\n",
      "Name: CGHregions\n",
      "Description:\n",
      "Dimension Reduction for Array CGH Data with Minimal Information Loss\n",
      "Version: 1.56.0\n",
      "Publication date: None\n",
      "Mantainer: Sjoerd Vosse <info@vossewebdevelopment.nl>\n",
      "Authors:\n",
      "Author: Sjoerd Vosse & Mark van de Wiel\n",
      "Dependencies:\n",
      "  - R (>= 2.0.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - Biobase, type: depends\n",
      "  - CGHbase, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (http://www.gnu.org/copyleft/gpl.html)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/CGHregions.html\n",
      "\n",
      "Name: dagLogo\n",
      "Description:\n",
      "Visualize significant conserved amino acid sequence pattern in groups based on probability theory.\n",
      "Version: 1.36.0\n",
      "Publication date: None\n",
      "Mantainer: Jianhong Ou <jianhong.ou@duke.edu>\n",
      "Authors:\n",
      "Author: Jianhong Ou, Haibo Liu, Alexey Stukalov, Niraj Nirala, Usha Acharya, Lihua Julie Zhu\n",
      "Dependencies:\n",
      "  - R (>= 3.0.1) type: depends\n",
      "  - methods, type: depends\n",
      "  - grid, type: depends\n",
      "  - pheatmap, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - UniProt.ws, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - utils, type: imports\n",
      "  - biomaRt, type: imports\n",
      "  - motifStack, type: imports\n",
      "  - httr, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>=2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/dagLogo.html\n",
      "\n",
      "Name: gaga\n",
      "Description:\n",
      "Implements the GaGa model for high-throughput data analysis, including differential expression analysis, supervised gene clustering and classification. Additionally, it performs sequential sample size calculations using the GaGa and LNNGV models (the latter from EBarrays package).\n",
      "Version: 2.44.0\n",
      "Publication date: None\n",
      "Mantainer: David Rossell <rosselldavid@gmail.com>\n",
      "Authors:\n",
      "Author: David Rossell <rosselldavid at gmail.com>.\n",
      "Dependencies:\n",
      "  - R (>= 2.8.0) type: depends\n",
      "  - Biobase, type: depends\n",
      "  - coda, type: depends\n",
      "  - EBarrays, type: depends\n",
      "  - mgcv, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/gaga.html\n",
      "\n",
      "Name: GeneNetworkBuilder\n",
      "Description:\n",
      "Appliation for discovering direct or indirect targets of transcription factors using ChIP-chip or ChIP-seq, and microarray or RNA-seq gene expression data. Inputting a list of genes of potential targets of one TF from ChIP-chip or ChIP-seq, and the gene expression results, GeneNetworkBuilder generates a regulatory network of the TF.\n",
      "Version: 1.40.0\n",
      "Publication date: None\n",
      "Mantainer: Jianhong Ou <jianhong.ou@duke.edu>\n",
      "Authors:\n",
      "Author: Jianhong Ou, Haibo Liu, Heidi A Tissenbaum and Lihua Julie Zhu\n",
      "Dependencies:\n",
      "  - R (>= 2.15.1) type: depends\n",
      "  - Rcpp (>= 0.9.13) type: depends\n",
      "  - plyr, type: imports\n",
      "  - graph, type: imports\n",
      "  - htmlwidgets, type: imports\n",
      "  - Rgraphviz, type: imports\n",
      "  - rjson, type: imports\n",
      "  - XML, type: imports\n",
      "  - methods, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - stats, type: imports\n",
      "  - graphics, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/GeneNetworkBuilder.html\n",
      "\n",
      "Name: InterMineR\n",
      "Description:\n",
      "Databases based on the InterMine platform such as FlyMine, modMine (modENCODE), RatMine, YeastMine, HumanMine and TargetMine are integrated databases of genomic, expression and protein data for various organisms. Integrating data makes it possible to run sophisticated data mining queries that span domains of biological knowledge. This R package provides interfaces with these databases through webservices. It makes most from the correspondence of the data frame object in R and the table object in databases, while hiding the details of data exchange through XML or JSON.\n",
      "Version: 1.20.0\n",
      "Publication date: None\n",
      "Mantainer: InterMine Team <r.lyne@gen.cam.ac.uk>\n",
      "Authors:\n",
      "Author: Bing Wang, Julie Sullivan, Rachel Lyne, Konstantinos Kyritsis, Celia Sanchez\n",
      "Dependencies:\n",
      "  - R (>= 3.4.1) type: depends\n",
      "  - Biostrings, type: imports\n",
      "  - RCurl, type: imports\n",
      "  - XML, type: imports\n",
      "  - xml2, type: imports\n",
      "  - RJSONIO, type: imports\n",
      "  - sqldf, type: imports\n",
      "  - igraph, type: imports\n",
      "  - httr, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - methods, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/InterMineR.html\n",
      "\n",
      "Name: limmaGUI\n",
      "Description:\n",
      "A Graphical User Interface for differential expression analysis of two-color microarray data using the limma package.\n",
      "Version: 1.74.0\n",
      "Publication date: None\n",
      "Mantainer: Gordon Smyth <smyth@wehi.edu.au>\n",
      "Authors:\n",
      "Author: James Wettenhall [aut], Gordon Smyth [aut], Keith Satterley [ctb]\n",
      "Dependencies:\n",
      "  - methods, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - limma, type: imports\n",
      "  - R2HTML, type: imports\n",
      "  - tcltk, type: imports\n",
      "  - tkrplot, type: imports\n",
      "  - xtable, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>=2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/limmaGUI.html\n",
      "\n",
      "Name: lmdme\n",
      "Description:\n",
      "linear ANOVA decomposition of Multivariate Designed Experiments implementation based on limma lmFit. Features: i)Flexible formula type interface, ii) Fast limma based implementation, iii) p-values for each estimated coefficient levels in each factor, iv) F values for factor effects and v) plotting functions for PCA and PLS.\n",
      "Version: 1.40.0\n",
      "Publication date: None\n",
      "Mantainer: Cristobal Fresno <cfresno@bdmg.com.ar>\n",
      "Authors:\n",
      "Author: Cristobal Fresno and Elmer A. Fernandez\n",
      "Dependencies:\n",
      "  - R (>= 2.14.1) type: depends\n",
      "  - pls, type: depends\n",
      "  - stemHypoxia, type: depends\n",
      "  - stats, type: imports\n",
      "  - methods, type: imports\n",
      "  - limma, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>=2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/lmdme.html\n",
      "\n",
      "Name: metabolomicsWorkbenchR\n",
      "Description:\n",
      "This package provides functions for interfacing with the Metabolomics Workbench RESTful API. Study, compound, protein and gene information can be searched for using the API. Methods to obtain study data in common Bioconductor formats such as SummarizedExperiment and MultiAssayExperiment are also included.\n",
      "Version: 1.8.0\n",
      "Publication date: None\n",
      "Mantainer: Gavin Rhys Lloyd <g.r.lloyd@bham.ac.uk>\n",
      "Authors:\n",
      "Author: Gavin Rhys Lloyd [aut, cre], Ralf Johannes Maria Weber [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - data.table, type: imports\n",
      "  - httr, type: imports\n",
      "  - jsonlite, type: imports\n",
      "  - methods, type: imports\n",
      "  - MultiAssayExperiment, type: imports\n",
      "  - struct, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/metabolomicsWorkbenchR.html\n",
      "\n",
      "Name: methimpute\n",
      "Description:\n",
      "This package implements functions for calling methylation for all cytosines in the genome.\n",
      "Version: 1.20.0\n",
      "Publication date: None\n",
      "Mantainer: Aaron Taudt <aaron.taudt@gmail.com>\n",
      "Authors:\n",
      "Author: Aaron Taudt\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - ggplot2, type: depends\n",
      "  - Rcpp (>= 0.12.4.5) type: imports\n",
      "  - methods, type: imports\n",
      "  - utils, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - stats, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - minpack.lm, type: imports\n",
      "  - data.table, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/methimpute.html\n",
      "\n",
      "Name: MIRA\n",
      "Description:\n",
      "DNA methylation contains information about the regulatory state of the cell. MIRA aggregates genome-scale DNA methylation data into a DNA methylation profile for a given region set with shared biological annotation. Using this profile, MIRA infers and scores the collective regulatory activity for the region set. MIRA facilitates regulatory analysis in situations where classical regulatory assays would be difficult and allows public sources of region sets to be leveraged for novel insight into the regulatory state of DNA methylation datasets.\n",
      "Version: 1.20.0\n",
      "Publication date: None\n",
      "Mantainer: John Lawson <jtl2hk@virginia.edu>\n",
      "Authors:\n",
      "Author: Nathan Sheffield <http://www.databio.org> [aut], Christoph Bock [ctb], John Lawson [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 3.5) type: depends\n",
      "  - BiocGenerics, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - data.table, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - stats, type: imports\n",
      "  - bsseq, type: imports\n",
      "  - methods, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/MIRA.html\n",
      "\n",
      "Name: msImpute\n",
      "Description:\n",
      "MsImpute is a package for imputation of peptide intensity in proteomics experiments. It additionally contains tools for MAR/MNAR diagnosis and assessment of distortions to the probability distribution of the data post imputation. The missing values are imputed by low-rank approximation of the underlying data matrix if they are MAR (method = \"v2\"), by Barycenter approach if missingness is MNAR (\"v2-mnar\"), or by Peptide Identity Propagation (PIP).\n",
      "Version: 1.8.0\n",
      "Publication date: None\n",
      "Mantainer: Soroor Hediyeh-zadeh <hediyehzadeh.s@wehi.edu.au>\n",
      "Authors:\n",
      "Author: Soroor Hediyeh-zadeh [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - softImpute, type: imports\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - graphics, type: imports\n",
      "  - pdist, type: imports\n",
      "  - reticulate, type: imports\n",
      "  - scran, type: imports\n",
      "  - data.table, type: imports\n",
      "  - FNN, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - limma, type: imports\n",
      "  - mvtnorm, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - dplyr, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>=2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/msImpute.html\n",
      "\n",
      "Name: PERFect\n",
      "Description:\n",
      "PERFect is a novel permutation filtering approach designed to address two unsolved problems in microbiome data processing: (i) define and quantify loss due to filtering by implementing thresholds, and (ii) introduce and evaluate a permutation test for filtering loss to provide a measure of excessive filtering.\n",
      "Version: 1.12.0\n",
      "Publication date: None\n",
      "Mantainer: Quy Cao <quy.cao@umontana.edu>\n",
      "Authors:\n",
      "Author: Ekaterina Smirnova <ekaterina.smirnova at vcuhealth.org>, Quy Cao <quy.cao at umontana.edu>\n",
      "Dependencies:\n",
      "  - R (>= 3.6.0) type: depends\n",
      "  - sn (>= 1.5.2) type: depends\n",
      "  - ggplot2 (>= 3.0.0) type: imports\n",
      "  - phyloseq, type: imports\n",
      "  - zoo (>= 1.8.3) type: imports\n",
      "  - psych (>= 1.8.4) type: imports\n",
      "  - stats (>= 3.6.0) type: imports\n",
      "  - Matrix (>= 1.2.14) type: imports\n",
      "  - fitdistrplus (>= 1.0.12) type: imports\n",
      "  - parallel (>= 3.6.0) type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/PERFect.html\n",
      "\n",
      "Name: PloGO2\n",
      "Description:\n",
      "Functions for enrichment analysis and plotting gene ontology or KEGG pathway information for multiple data subsets at the same time. It also enables encorporating multiple conditions and abundance data.\n",
      "Version: 1.10.0\n",
      "Publication date: None\n",
      "Mantainer: Jemma Wu <jemma.wu@mq.edu.au>, Dana Pascovici <dana.pascovici@mq.edu.au>\n",
      "Authors:\n",
      "Author: Dana Pascovici, Jemma Wu\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - GO.db, type: depends\n",
      "  - GOstats, type: depends\n",
      "  - lattice, type: imports\n",
      "  - httr, type: imports\n",
      "  - openxlsx, type: imports\n",
      "  - xtable, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/PloGO2.html\n",
      "\n",
      "Name: SAIGEgds\n",
      "Description:\n",
      "Scalable implementation of generalized mixed models with highly optimized C++ implementation and integration with Genomic Data Structure (GDS) files. It is designed for single variant tests in large-scale phenome-wide association studies (PheWAS) with millions of variants and samples, controlling for sample structure and case-control imbalance. The implementation is based on the original SAIGE R package (v0.29.4.4 for single variant tests, Zhou et al. 2018). SAIGEgds also implements some of the SPAtest functions in C to speed up the calculation of Saddlepoint approximation. Benchmarks show that SAIGEgds is 5 to 6 times faster than the original SAIGE R package.\n",
      "Version: 1.12.4\n",
      "Publication date: None\n",
      "Mantainer: Xiuwen Zheng <xiuwen.zheng@abbvie.com>\n",
      "Authors:\n",
      "Author: Xiuwen Zheng [aut, cre] , Wei Zhou [ctb] (the original author of the SAIGE R package), J. Wade Davis [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - gdsfmt, type: depends\n",
      "  - SeqArray, type: depends\n",
      "  - Rcpp, type: depends\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - RcppParallel, type: imports\n",
      "  - SPAtest (>= 3.0.0) type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/SAIGEgds.html\n",
      "\n",
      "Name: TEQC\n",
      "Description:\n",
      "Target capture experiments combine hybridization-based (in solution or on microarrays) capture and enrichment of genomic regions of interest (e.g. the exome) with high throughput sequencing of the captured DNA fragments. This package provides functionalities for assessing and visualizing the quality of the target enrichment process, like specificity and sensitivity of the capture, per-target read coverage and so on.\n",
      "Version: 4.20.0\n",
      "Publication date: None\n",
      "Mantainer: Sarah Bonnin <Sarah.Bonnin@crg.eu>\n",
      "Authors:\n",
      "Author: M. Hummel, S. Bonnin, E. Lowy, G. Roma\n",
      "Dependencies:\n",
      "  - methods, type: depends\n",
      "  - BiocGenerics, type: depends\n",
      "  - IRanges, type: depends\n",
      "  - Rsamtools, type: depends\n",
      "  - hwriter, type: depends\n",
      "  - Biobase, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/TEQC.html\n",
      "\n",
      "Name: vbmp\n",
      "Description:\n",
      "Variational Bayesian Multinomial Probit Regression with Gaussian Process Priors. It estimates class membership posterior probability employing variational and sparse approximation to the full posterior. This software also incorporates feature weighting by means of Automatic Relevance Determination.\n",
      "Version: 1.66.0\n",
      "Publication date: None\n",
      "Mantainer: Nicola Lama <nicola.lama@unina2.it>\n",
      "Authors:\n",
      "Author: Nicola Lama <nicola.lama at unina2.it>, Mark Girolami <girolami at dcs.gla.ac.uk>\n",
      "Dependencies:\n",
      "  - R (>= 2.10) type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/vbmp.html\n",
      "\n",
      "Name: affyContam\n",
      "Description:\n",
      "structured corruption of cel file data to demonstrate QA effectiveness\n",
      "Version: 1.56.0\n",
      "Publication date: None\n",
      "Mantainer: V. Carey <stvjc@channing.harvard.edu>\n",
      "Authors:\n",
      "Author: V. Carey\n",
      "Dependencies:\n",
      "  - R (>= 2.7.0) type: depends\n",
      "  - tools, type: depends\n",
      "  - methods, type: depends\n",
      "  - utils, type: depends\n",
      "  - Biobase, type: depends\n",
      "  - affy, type: depends\n",
      "  - affydata, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/affyContam.html\n",
      "\n",
      "Name: beadarraySNP\n",
      "Description:\n",
      "Importing data from Illumina SNP experiments and performing copy number calculations and reports.\n",
      "Version: 1.64.0\n",
      "Publication date: None\n",
      "Mantainer: Jan Oosting <j.oosting@lumc.nl>\n",
      "Authors:\n",
      "Author: Jan Oosting\n",
      "Dependencies:\n",
      "  - methods, type: depends\n",
      "  - Biobase, type: depends\n",
      "  - quantsmooth, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/beadarraySNP.html\n",
      "\n",
      "Name: bigmelon\n",
      "Description:\n",
      "Methods for working with Illumina arrays using gdsfmt.\n",
      "Version: 1.24.0\n",
      "Publication date: None\n",
      "Mantainer: Tyler J. Gorrie-Stone <tyler.gorrie-stone@diamond.ac.uk>\n",
      "Authors:\n",
      "Author: Tyler J. Gorrie-Stone [cre, aut], Ayden Saffari [aut], Karim Malki [aut], Leonard C. Schalkwyk [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.3) type: depends\n",
      "  - wateRmelon, type: depends\n",
      "  - gdsfmt, type: depends\n",
      "  - methods, type: depends\n",
      "  - minfi, type: depends\n",
      "  - Biobase, type: depends\n",
      "  - methylumi, type: depends\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - GEOquery, type: imports\n",
      "  - graphics, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - illuminaio, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/bigmelon.html\n",
      "\n",
      "Name: biscuiteer\n",
      "Description:\n",
      "A test harness for bsseq loading of Biscuit output, summarization of WGBS data over defined regions and in mappable samples, with or without imputation, dropping of mostly-NA rows, age estimates, etc.\n",
      "Version: 1.12.0\n",
      "Publication date: None\n",
      "Mantainer: Jacob Morrison <jacob.morrison@vai.org>\n",
      "Authors:\n",
      "Author: Tim Triche [aut], Wanding Zhou [aut], Benjamin Johnson [aut], Jacob Morrison [aut, cre], Lyong Heo [aut], James Eapen [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.1.0) type: depends\n",
      "  - biscuiteerData, type: depends\n",
      "  - bsseq, type: depends\n",
      "  - readr, type: imports\n",
      "  - qualV, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - impute, type: imports\n",
      "  - HDF5Array, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - data.table, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - VariantAnnotation, type: imports\n",
      "  - DelayedMatrixStats, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - Mus.musculus, type: imports\n",
      "  - Homo.sapiens, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - QDNAseq, type: imports\n",
      "  - dmrseq, type: imports\n",
      "  - methods, type: imports\n",
      "  - utils, type: imports\n",
      "  - R.utils, type: imports\n",
      "  - gtools, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/biscuiteer.html\n",
      "\n",
      "Name: BitSeq\n",
      "Description:\n",
      "The BitSeq package is targeted for transcript expression analysis and differential expression analysis of RNA-seq data in two stage process. In the first stage it uses Bayesian inference methodology to infer expression of individual transcripts from individual RNA-seq experiments. The second stage of BitSeq embraces the differential expression analysis of transcript expression. Providing expression estimates from replicates of multiple conditions, Log-Normal model of the estimates is used for inferring the condition mean transcript expression and ranking the transcripts based on the likelihood of differential expression.\n",
      "Version: 1.41.0\n",
      "Publication date: None\n",
      "Mantainer: Antti Honkela <antti.honkela@helsinki.fi>, Panagiotis Papastamoulis <papastamoulis@aueb.gr>\n",
      "Authors:\n",
      "Author: Peter Glaus, Antti Honkela and Magnus Rattray\n",
      "Dependencies:\n",
      "  - Rsamtools, type: depends\n",
      "  - S4Vectors, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - methods, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0 + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/BitSeq.html\n",
      "\n",
      "Name: cghMCR\n",
      "Description:\n",
      "This package provides functions to identify genomic regions of interests based on segmented copy number data from multiple samples.\n",
      "Version: 1.56.0\n",
      "Publication date: None\n",
      "Mantainer: J. Zhang <jzhang@jimmy.harvard.edu>\n",
      "Authors:\n",
      "Author: J. Zhang and B. Feng\n",
      "Dependencies:\n",
      "  - methods, type: depends\n",
      "  - DNAcopy, type: depends\n",
      "  - CNTools, type: depends\n",
      "  - limma, type: depends\n",
      "  - BiocGenerics, type: imports\n",
      "  - stats4, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/cghMCR.html\n",
      "\n",
      "Name: ChromHeatMap\n",
      "Description:\n",
      "The ChromHeatMap package can be used to plot genome-wide data (e.g. expression, CGH, SNP) along each strand of a given chromosome as a heat map. The generated heat map can be used to interactively identify probes and genes of interest.\n",
      "Version: 1.52.0\n",
      "Publication date: None\n",
      "Mantainer: Tim F. Rayner <tfrayner@gmail.com>\n",
      "Authors:\n",
      "Author: Tim F. Rayner\n",
      "Dependencies:\n",
      "  - R (>= 2.9.0) type: depends\n",
      "  - BiocGenerics, type: depends\n",
      "  - annotate, type: depends\n",
      "  - AnnotationDbi, type: depends\n",
      "  - Biobase, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ChromHeatMap.html\n",
      "\n",
      "Name: CHRONOS\n",
      "Description:\n",
      "A package used for efficient unraveling of the inherent dynamic properties of pathways. MicroRNA-mediated subpathway topologies are extracted and evaluated by exploiting the temporal transition and the fold change activity of the linked genes/microRNAs.\n",
      "Version: 1.26.0\n",
      "Publication date: None\n",
      "Mantainer: Panos Balomenos <balomenos@upatras.gr>\n",
      "Authors:\n",
      "Author: Aristidis G. Vrahatis, Konstantina Dimitrakopoulou, Panos Balomenos\n",
      "Dependencies:\n",
      "  - R (>= 3.5) type: depends\n",
      "  - XML, type: imports\n",
      "  - RCurl, type: imports\n",
      "  - RBGL, type: imports\n",
      "  - parallel, type: imports\n",
      "  - foreach, type: imports\n",
      "  - doParallel, type: imports\n",
      "  - openxlsx, type: imports\n",
      "  - igraph, type: imports\n",
      "  - circlize, type: imports\n",
      "  - graph, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - methods, type: imports\n",
      "  - biomaRt, type: imports\n",
      "  - rJava, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/CHRONOS.html\n",
      "\n",
      "Name: debCAM\n",
      "Description:\n",
      "An R package for fully unsupervised deconvolution of complex tissues. It provides basic functions to perform unsupervised deconvolution on mixture expression profiles by Convex Analysis of Mixtures (CAM) and some auxiliary functions to help understand the subpopulation-specific results. It also implements functions to perform supervised deconvolution based on prior knowledge of molecular markers, S matrix or A matrix. Combining molecular markers from CAM and from prior knowledge can achieve semi-supervised deconvolution of mixtures.\n",
      "Version: 1.16.0\n",
      "Publication date: None\n",
      "Mantainer: Lulu Chen <luluchen@vt.edu>\n",
      "Authors:\n",
      "Author: Lulu Chen <luluchen at vt.edu>\n",
      "Dependencies:\n",
      "  - R (>= 3.5) type: depends\n",
      "  - methods, type: imports\n",
      "  - rJava, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - stats, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - corpcor, type: imports\n",
      "  - geometry, type: imports\n",
      "  - NMF, type: imports\n",
      "  - nnls, type: imports\n",
      "  - DMwR2, type: imports\n",
      "  - pcaPP, type: imports\n",
      "  - apcluster, type: imports\n",
      "  - graphics, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/debCAM.html\n",
      "\n",
      "Name: EventPointer\n",
      "Description:\n",
      "EventPointer is an R package to identify alternative splicing events that involve either simple (case-control experiment) or complex experimental designs such as time course experiments and studies including paired-samples. The algorithm can be used to analyze data from either junction arrays (Affymetrix Arrays) or sequencing data (RNA-Seq). The software returns a data.frame with the detected alternative splicing events: gene name, type of event (cassette, alternative 3',...,etc), genomic position, statistical significance and increment of the percent spliced in (Delta PSI) for all the events. The algorithm can generate a series of files to visualize the detected alternative splicing events in IGV. This eases the interpretation of results and the design of primers for standard PCR validation.\n",
      "Version: 3.6.0\n",
      "Publication date: None\n",
      "Mantainer: Juan A. Ferrer-Bonsoms <jafhernandez@tecnun.es>\n",
      "Authors:\n",
      "Author: Juan Pablo Romero [aut], Juan A. Ferrer-Bonsoms [aut, cre], Pablo Sacristan [aut], Ander Muniategui [aut], Fernando Carazo [aut], Ander Aramburu [aut], Angel Rubio [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - SGSeq, type: depends\n",
      "  - Matrix, type: depends\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - GenomicFeatures, type: imports\n",
      "  - stringr, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - igraph, type: imports\n",
      "  - MASS, type: imports\n",
      "  - nnls, type: imports\n",
      "  - limma, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - RBGL, type: imports\n",
      "  - prodlim, type: imports\n",
      "  - graph, type: imports\n",
      "  - methods, type: imports\n",
      "  - utils, type: imports\n",
      "  - stats, type: imports\n",
      "  - doParallel, type: imports\n",
      "  - foreach, type: imports\n",
      "  - affxparser, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - qvalue, type: imports\n",
      "  - cobs, type: imports\n",
      "  - rhdf5, type: imports\n",
      "  - BSgenome, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - glmnet, type: imports\n",
      "  - abind, type: imports\n",
      "  - iterators, type: imports\n",
      "  - lpSolve, type: imports\n",
      "  - poibin, type: imports\n",
      "  - speedglm, type: imports\n",
      "  - tximport, type: imports\n",
      "  - fgsea, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/EventPointer.html\n",
      "\n",
      "Name: GEOexplorer\n",
      "Description:\n",
      "GEOexplorer is a Shiny app that enables exploratory data analysis and differential gene expression of gene expression analysis on microarray gene expression datasets held on the GEO database. The outputs are interactive graphs that enable users to explore the results of the analysis. The development of GEOexplorer was made possible because of the excellent code provided by GEO2R (https: //www.ncbi.nlm.nih.gov/geo/geo2r/).\n",
      "Version: 1.4.0\n",
      "Publication date: None\n",
      "Mantainer: Guy Hunt <guypwhunt1992@gmail.com>\n",
      "Authors:\n",
      "Author: Guy Hunt [aut, cre] , Rafael Henkin [ctb, ths] , Fabrizio Smeraldi [ctb, ths] , Michael Barnes [ctb, ths]\n",
      "Dependencies:\n",
      "  - shiny, type: depends\n",
      "  - limma, type: depends\n",
      "  - Biobase, type: depends\n",
      "  - plotly, type: depends\n",
      "  - shinyBS, type: depends\n",
      "  - DT, type: imports\n",
      "  - htmltools, type: imports\n",
      "  - factoextra, type: imports\n",
      "  - heatmaply, type: imports\n",
      "  - maptools, type: imports\n",
      "  - pheatmap, type: imports\n",
      "  - scales, type: imports\n",
      "  - shinyHeatmaply, type: imports\n",
      "  - shinybusy, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - stringr, type: imports\n",
      "  - umap, type: imports\n",
      "  - GEOquery, type: imports\n",
      "  - impute, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - stats, type: imports\n",
      "  - graphics, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/GEOexplorer.html\n",
      "\n",
      "Name: MethReg\n",
      "Description:\n",
      "Epigenome-wide association studies (EWAS) detects a large number of DNA methylation differences, often hundreds of differentially methylated regions and thousands of CpGs, that are significantly associated with a disease, many are located in non-coding regions. Therefore, there is a critical need to better understand the functional impact of these CpG methylations and to further prioritize the significant changes. MethReg is an R package for integrative modeling of DNA methylation, target gene expression and transcription factor binding sites data, to systematically identify and rank functional CpG methylations. MethReg evaluates, prioritizes and annotates CpG sites with high regulatory potential using matched methylation and gene expression data, along with external TF-target interaction databases based on manually curation, ChIP-seq experiments or gene regulatory network analysis.\n",
      "Version: 1.8.0\n",
      "Publication date: None\n",
      "Mantainer: Tiago Silva <tiagochst@gmail.com>\n",
      "Authors:\n",
      "Author: Tiago Silva [aut, cre] , Lily Wang [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - dplyr, type: imports\n",
      "  - plyr, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - DelayedArray, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - ggpubr, type: imports\n",
      "  - tibble, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - sesameData, type: imports\n",
      "  - sesame, type: imports\n",
      "  - AnnotationHub, type: imports\n",
      "  - ExperimentHub, type: imports\n",
      "  - stringr, type: imports\n",
      "  - readr, type: imports\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - MASS, type: imports\n",
      "  - rlang, type: imports\n",
      "  - pscl, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - sfsmisc, type: imports\n",
      "  - progress, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/MethReg.html\n",
      "\n",
      "Name: methylPipe\n",
      "Description:\n",
      "Memory efficient analysis of base resolution DNA methylation data in both the CpG and non-CpG sequence context. Integration of DNA methylation data derived from any methodology providing base- or low-resolution data.\n",
      "Version: 1.32.0\n",
      "Publication date: None\n",
      "Mantainer: Kamal Kishore <kamal.fartiyal84@gmail.com>\n",
      "Authors:\n",
      "Author: Kamal Kishore\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - grDevices, type: depends\n",
      "  - graphics, type: depends\n",
      "  - stats, type: depends\n",
      "  - utils, type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - Rsamtools, type: depends\n",
      "  - marray, type: imports\n",
      "  - gplots, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - Gviz, type: imports\n",
      "  - GenomicAlignments, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - parallel, type: imports\n",
      "  - data.table, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL(>=2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/methylPipe.html\n",
      "\n",
      "Name: MetNet\n",
      "Description:\n",
      "MetNet contains functionality to infer metabolic network topologies from quantitative data and high-resolution mass/charge information. Using statistical models (including correlation, mutual information, regression and Bayes statistics) and quantitative data (intensity values of features) adjacency matrices are inferred that can be combined to a consensus matrix. Mass differences calculated between mass/charge values of features will be matched against a data frame of supplied mass/charge differences referring to transformations of enzymatic activities. In a third step, the two levels of information are combined to form a adjacency matrix inferred from both quantitative and structure information.\n",
      "Version: 1.16.0\n",
      "Publication date: None\n",
      "Mantainer: Thomas Naake <thomasnaake@googlemail.com>\n",
      "Authors:\n",
      "Author: Thomas Naake [aut, cre], Liesa Salzer [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - S4Vectors, type: depends\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - bnlearn (>= 4.3) type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - corpcor (>= 1.6.10) type: imports\n",
      "  - dplyr (>= 1.0.3) type: imports\n",
      "  - ggplot2 (>= 3.3.3) type: imports\n",
      "  - GeneNet (>= 1.2.15) type: imports\n",
      "  - GENIE3, type: imports\n",
      "  - methods (>= 3.5) type: imports\n",
      "  - parmigene (>= 1.0.2) type: imports\n",
      "  - psych (>= 2.1.6) type: imports\n",
      "  - rlang (>= 0.4.10) type: imports\n",
      "  - stabs (>= 0.6) type: imports\n",
      "  - stats (>= 3.6) type: imports\n",
      "  - tibble (>= 3.0.5) type: imports\n",
      "  - tidyr (>= 1.1.2) type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 3)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/MetNet.html\n",
      "\n",
      "Name: miRspongeR\n",
      "Description:\n",
      "This package provides several functions to explore miRNA sponge (also called ceRNA or miRNA decoy) regulation from putative miRNA-target interactions or/and transcriptomics data (including bulk, single-cell and spatial gene expression data). It provides eight popular methods for identifying miRNA sponge interactions, and an integrative method to integrate miRNA sponge interactions from different methods, as well as the functions to validate miRNA sponge interactions, and infer miRNA sponge modules, conduct enrichment analysis of miRNA sponge modules, and conduct survival analysis of miRNA sponge modules. By using a sample control variable strategy, it provides a function to infer sample-specific miRNA sponge interactions. In terms of sample-specific miRNA sponge interactions, it implements three similarity methods to construct sample-sample correlation network.\n",
      "Version: 2.2.0\n",
      "Publication date: None\n",
      "Mantainer: Junpeng Zhang <zhangjunpeng411@gmail.com>\n",
      "Authors:\n",
      "Author: Junpeng Zhang\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - corpcor, type: imports\n",
      "  - SPONGE, type: imports\n",
      "  - parallel, type: imports\n",
      "  - igraph, type: imports\n",
      "  - MCL, type: imports\n",
      "  - clusterProfiler, type: imports\n",
      "  - ReactomePA, type: imports\n",
      "  - DOSE, type: imports\n",
      "  - survival, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - linkcomm, type: imports\n",
      "  - utils, type: imports\n",
      "  - Rcpp, type: imports\n",
      "  - org.Hs.eg.db, type: imports\n",
      "  - foreach, type: imports\n",
      "  - doParallel, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/miRspongeR.html\n",
      "\n",
      "Name: moanin\n",
      "Description:\n",
      "Simple and efficient workflow for time-course gene expression data, built on publictly available open-source projects hosted on CRAN and bioconductor. moanin provides helper functions for all the steps required for analysing time-course data using functional data analysis: (1) functional modeling of the timecourse data; (2) differential expression analysis; (3) clustering; (4) downstream analysis.\n",
      "Version: 1.6.0\n",
      "Publication date: None\n",
      "Mantainer: Nelle Varoquaux <nelle.varoquaux@gmail.com>\n",
      "Authors:\n",
      "Author: Elizabeth Purdom [aut] , Nelle Varoquaux [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - topGO, type: depends\n",
      "  - stats, type: depends\n",
      "  - S4Vectors, type: imports\n",
      "  - MASS (>= 1.0.0) type: imports\n",
      "  - limma, type: imports\n",
      "  - viridis, type: imports\n",
      "  - edgeR, type: imports\n",
      "  - graphics, type: imports\n",
      "  - methods, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - NMI, type: imports\n",
      "  - zoo, type: imports\n",
      "  - ClusterR, type: imports\n",
      "  - splines, type: imports\n",
      "  - matrixStats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "BSD 3-clause License + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/moanin.html\n",
      "\n",
      "Name: OncoSimulR\n",
      "Description:\n",
      "Functions for forward population genetic simulation in asexual populations, with special focus on cancer progression. Fitness can be an arbitrary function of genetic interactions between multiple genes or modules of genes, including epistasis, order restrictions in mutation accumulation, and order effects. Fitness (including just birth, just death, or both birth and death) can also be a function of the relative and absolute frequencies of other genotypes (i.e., frequency-dependent fitness). Mutation rates can differ between genes, and we can include mutator/antimutator genes (to model mutator phenotypes). Simulating multi-species scenarios and therapeutic interventions, including adaptive therapy, is also possible. Simulations use continuous-time models and can include driver and passenger genes and modules. Also included are functions for: simulating random DAGs of the type found in Oncogenetic Trees, Conjunctive Bayesian Networks, and other cancer progression models; plotting and sampling from single or multiple realizations of the simulations, including single-cell sampling; plotting the parent-child relationships of the clones; generating random fitness landscapes (Rough Mount Fuji, House of Cards, additive, NK, Ising, and Eggbox models) and plotting them.\n",
      "Version: 4.0.2\n",
      "Publication date: None\n",
      "Mantainer: Ramon Diaz-Uriarte <rdiaz02@gmail.com>\n",
      "Authors:\n",
      "Author: Ramon Diaz-Uriarte [aut, cre], Sergio Sanchez-Carrillo [aut], Juan Antonio Miguel Gonzalez [aut], Alberto Gonzalez Klein [aut], Javier Mu\\~noz Haro [aut], Javier Lopez Cano [aut], Niklas Endres [ctb], Mark Taylor [ctb], Arash Partow [ctb], Sophie Brouillet [ctb], Sebastian Matuszewski [ctb], Harry Annoni [ctb], Luca Ferretti [ctb], Guillaume Achaz [ctb], Tymoteusz Wolodzko [ctb], Guillermo Gorines Cordero [ctb], Ivan Lorca Alonso [ctb], Francisco Mu\\~noz Lopez [ctb], David Roncero Moro\\~no [ctb], Alvaro Quevedo [ctb], Pablo Perez [ctb], Cristina Devesa [ctb], Alejandro Herrador [ctb], Holger Froehlich [ctb], Florian Markowetz [ctb], Achim Tresch [ctb], Theresa Niederberger [ctb], Christian Bender [ctb], Matthias Maneck [ctb], Claudio Lottaz [ctb], Tim Beissbarth [ctb], Sara Dorado Alfaro [ctb], Miguel Hernandez del Valle [ctb], Alvaro Huertas Garcia [ctb], Diego Ma\\~nanes Cayero [ctb], Alejandro Martin Mu\\~noz [ctb], Marta Couce Iglesias [ctb], Silvia Garcia Cobos [ctb], Carlos Madariaga Aramendi [ctb], Ana Rodriguez Ronchel [ctb], Lucia Sanchez Garcia [ctb], Yolanda Benitez Quesada [ctb], Asier Fernandez Pato [ctb], Esperanza Lopez Lopez [ctb], Alberto Manuel Parra Perez [ctb], Jorge Garcia Calleja [ctb], Ana del Ramo Galian [ctb], Alejandro de los Reyes Benitez [ctb], Guillermo Garcia Hoyos [ctb], Rosalia Palomino Cabrera [ctb], Rafael Barrero Rodriguez [ctb], Silvia Talavera Marcos [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - Rcpp (>= 0.12.4) type: imports\n",
      "  - parallel, type: imports\n",
      "  - data.table, type: imports\n",
      "  - graph, type: imports\n",
      "  - Rgraphviz, type: imports\n",
      "  - gtools, type: imports\n",
      "  - igraph, type: imports\n",
      "  - methods, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - car, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - smatr, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - ggrepel, type: imports\n",
      "  - stringr, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 3)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/OncoSimulR.html\n",
      "\n",
      "Name: pwrEWAS\n",
      "Description:\n",
      "pwrEWAS is a user-friendly tool to assists researchers in the design and planning of EWAS to help circumvent under- and overpowered studies.\n",
      "Version: 1.12.0\n",
      "Publication date: None\n",
      "Mantainer: Stefan Graw <shgraw@uams.edu>\n",
      "Authors:\n",
      "Author: Stefan Graw\n",
      "Dependencies:\n",
      "  - shinyBS, type: depends\n",
      "  - foreach, type: depends\n",
      "  - doParallel, type: imports\n",
      "  - abind, type: imports\n",
      "  - truncnorm, type: imports\n",
      "  - CpGassoc, type: imports\n",
      "  - shiny, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - parallel, type: imports\n",
      "  - shinyWidgets, type: imports\n",
      "  - BiocManager, type: imports\n",
      "  - doSNOW, type: imports\n",
      "  - limma, type: imports\n",
      "  - genefilter, type: imports\n",
      "  - stats, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - methods, type: imports\n",
      "  - utils, type: imports\n",
      "  - graphics, type: imports\n",
      "  - pwrEWAS.data, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/pwrEWAS.html\n",
      "\n",
      "Name: qsmooth\n",
      "Description:\n",
      "Smooth quantile normalization is a generalization of quantile normalization, which is average of the two types of assumptions about the data generation process: quantile normalization and quantile normalization between groups.\n",
      "Version: 1.14.0\n",
      "Publication date: None\n",
      "Mantainer: Stephanie C. Hicks <shicks19@jhu.edu>\n",
      "Authors:\n",
      "Author: Stephanie C. Hicks [aut, cre] , Kwame Okrah [aut], Koen Van den Berge [ctb], Hector Corrada Bravo [aut] , Rafael Irizarry [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - utils, type: imports\n",
      "  - sva, type: imports\n",
      "  - stats, type: imports\n",
      "  - methods, type: imports\n",
      "  - graphics, type: imports\n",
      "  - Hmisc, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "CC BY 4.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/qsmooth.html\n",
      "\n",
      "Name: RBioinf\n",
      "Description:\n",
      "Functions and datasets and examples to accompany the monograph R For Bioinformatics.\n",
      "Version: 1.58.0\n",
      "Publication date: None\n",
      "Mantainer: Robert Gentleman <rgentlem@gmail.com>\n",
      "Authors:\n",
      "Author: Robert Gentleman\n",
      "Dependencies:\n",
      "  - graph, type: depends\n",
      "  - methods, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/RBioinf.html\n",
      "\n",
      "Name: rbsurv\n",
      "Description:\n",
      "This package selects genes associated with survival.\n",
      "Version: 2.56.0\n",
      "Publication date: None\n",
      "Mantainer: Soo-heang Eo <hanansh@korea.ac.kr>\n",
      "Authors:\n",
      "Author: HyungJun Cho <hj4cho at korea.ac.kr>, Sukwoo Kim <s4kim at korea.ac.kr>, Soo-heang Eo <hanansh at korea.ac.kr>, Jaewoo Kang <kangj at korea.ac.kr>\n",
      "Dependencies:\n",
      "  - R (>= 2.5.0) type: depends\n",
      "  - Biobase, type: depends\n",
      "  - survival, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/rbsurv.html\n",
      "\n",
      "Name: REMP\n",
      "Description:\n",
      "Machine learning-based tools to predict DNA methylation of locus-specific repetitive elements (RE) by learning surrounding genetic and epigenetic information. These tools provide genomewide and single-base resolution of DNA methylation prediction on RE that are difficult to measure using array-based or sequencing-based platforms, which enables epigenome-wide association study (EWAS) and differentially methylated region (DMR) analysis on RE.\n",
      "Version: 1.22.0\n",
      "Publication date: None\n",
      "Mantainer: Yinan Zheng <y-zheng@northwestern.edu>\n",
      "Authors:\n",
      "Author: Yinan Zheng [aut, cre], Lei Liu [aut], Wei Zhang [aut], Warren Kibbe [aut], Lifang Hou [aut, cph]\n",
      "Dependencies:\n",
      "  - R (>= 3.6) type: depends\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - minfi, type: depends\n",
      "  - readr, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - methods, type: imports\n",
      "  - settings, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - doParallel, type: imports\n",
      "  - parallel, type: imports\n",
      "  - foreach, type: imports\n",
      "  - caret, type: imports\n",
      "  - kernlab, type: imports\n",
      "  - ranger, type: imports\n",
      "  - BSgenome, type: imports\n",
      "  - AnnotationHub, type: imports\n",
      "  - org.Hs.eg.db, type: imports\n",
      "  - impute, type: imports\n",
      "  - iterators, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/REMP.html\n",
      "\n",
      "Name: snapCGH\n",
      "Description:\n",
      "Methods for segmenting, normalising and processing aCGH data; including plotting functions for visualising raw and segmented data for individual and multiple arrays.\n",
      "Version: 1.68.0\n",
      "Publication date: None\n",
      "Mantainer: John Marioni <marioni@ebi.ac.uk>\n",
      "Authors:\n",
      "Author: Mike L. Smith, John C. Marioni, Steven McKinney, Thomas Hardcastle, Natalie P. Thorne\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - aCGH, type: imports\n",
      "  - cluster, type: imports\n",
      "  - DNAcopy, type: imports\n",
      "  - GLAD, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - limma, type: imports\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - tilingArray, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/snapCGH.html\n",
      "\n",
      "Name: spatialDE\n",
      "Description:\n",
      "SpatialDE is a method to find spatially variable genes (SVG) from spatial transcriptomics data. This package provides wrappers to use the Python SpatialDE library in R, using reticulate and basilisk.\n",
      "Version: 1.4.3\n",
      "Publication date: None\n",
      "Mantainer: Gabriele Sales <gabriele.sales@unipd.it>\n",
      "Authors:\n",
      "Author: Davide Corso [aut] , Milan Malfait [aut] , Lambda Moses [aut] , Gabriele Sales [cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - reticulate, type: imports\n",
      "  - basilisk, type: imports\n",
      "  - checkmate, type: imports\n",
      "  - stats, type: imports\n",
      "  - SpatialExperiment, type: imports\n",
      "  - methods, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - ggrepel, type: imports\n",
      "  - scales, type: imports\n",
      "  - gridExtra, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/spatialDE.html\n",
      "\n",
      "Name: TADCompare\n",
      "Description:\n",
      "TADCompare is an R package designed to identify and characterize differential Topologically Associated Domains (TADs) between multiple Hi-C contact matrices. It contains functions for finding differential TADs between two datasets, finding differential TADs over time and identifying consensus TADs across multiple matrices. It takes all of the main types of HiC input and returns simple, comprehensive, easy to analyze results.\n",
      "Version: 1.8.0\n",
      "Publication date: None\n",
      "Mantainer: Kellen Cresswell <cresswellkg@vcu.edu>\n",
      "Authors:\n",
      "Author: Kellen Cresswell <cresswellkg at vcu.edu>, Mikhail Dozmorov <mikhail.dozmorov at vcuhealth.org>\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - dplyr, type: imports\n",
      "  - PRIMME, type: imports\n",
      "  - cluster, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - HiCcompare, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - ggpubr, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - cowplot, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/TADCompare.html\n",
      "\n",
      "Name: timeOmics\n",
      "Description:\n",
      "timeOmics is a generic data-driven framework to integrate multi-Omics longitudinal data measured on the same biological samples and select key temporal features with strong associations within the same sample group. The main steps of timeOmics are: 1. Plaform and time-specific normalization and filtering steps; 2. Modelling each biological into one time expression profile; 3. Clustering features with the same expression profile over time; 4. Post-hoc validation step.\n",
      "Version: 1.10.0\n",
      "Publication date: None\n",
      "Mantainer: Antoine Bodein <antoine.bodein.1@ulaval.ca>\n",
      "Authors:\n",
      "Author: Antoine Bodein [aut, cre], Olivier Chapleur [aut], Kim-Anh Le Cao [aut], Arnaud Droit [aut]\n",
      "Dependencies:\n",
      "  - mixOmics, type: depends\n",
      "  - R (>= 4.0) type: depends\n",
      "  - dplyr, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - tibble, type: imports\n",
      "  - purrr, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - stringr, type: imports\n",
      "  - ggrepel, type: imports\n",
      "  - propr, type: imports\n",
      "  - lmtest, type: imports\n",
      "  - plyr, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/timeOmics.html\n",
      "\n",
      "Name: atSNP\n",
      "Description:\n",
      "atSNP performs affinity tests of motif matches with the SNP or the reference genomes and SNP-led changes in motif matches.\n",
      "Version: 1.14.0\n",
      "Publication date: None\n",
      "Mantainer: Sunyoung Shin <sunyoung.shin@utdallas.edu>\n",
      "Authors:\n",
      "Author: Chandler Zuo [aut], Sunyoung Shin [aut, cre], Sunduz Keles [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.6) type: depends\n",
      "  - BSgenome, type: imports\n",
      "  - BiocFileCache, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - Rcpp, type: imports\n",
      "  - data.table, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grid, type: imports\n",
      "  - motifStack, type: imports\n",
      "  - rappdirs, type: imports\n",
      "  - stats, type: imports\n",
      "  - testthat, type: imports\n",
      "  - utils, type: imports\n",
      "  - lifecycle, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/atSNP.html\n",
      "\n",
      "Name: BufferedMatrix\n",
      "Description:\n",
      "A tabular style data object where most data is stored outside main memory. A buffer is used to speed up access to data.\n",
      "Version: 1.62.0\n",
      "Publication date: None\n",
      "Mantainer: Ben Bolstad <bmb@bmbolstad.com>\n",
      "Authors:\n",
      "Author: Ben Bolstad <bmb at bmbolstad.com>\n",
      "Dependencies:\n",
      "  - R (>= 2.6.0) type: depends\n",
      "  - methods, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/BufferedMatrix.html\n",
      "\n",
      "Name: circRNAprofiler\n",
      "Description:\n",
      "R-based computational framework for a comprehensive in silico analysis of circRNAs. This computational framework allows to combine and analyze circRNAs previously detected by multiple publicly available annotation-based circRNA detection tools. It covers different aspects of circRNAs analysis from differential expression analysis, evolutionary conservation, biogenesis to functional analysis.\n",
      "Version: 1.12.2\n",
      "Publication date: None\n",
      "Mantainer: Simona Aufiero <simo.aufiero@gmail.com>\n",
      "Authors:\n",
      "Author: Simona Aufiero\n",
      "Dependencies:\n",
      "  - R (>= 4.2.0) type: depends\n",
      "  - dplyr, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - readr, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - stringr, type: imports\n",
      "  - stringi, type: imports\n",
      "  - DESeq2, type: imports\n",
      "  - edgeR, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - seqinr, type: imports\n",
      "  - R.utils, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - utils, type: imports\n",
      "  - rlang, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - stats, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - universalmotif, type: imports\n",
      "  - AnnotationHub, type: imports\n",
      "  - BSgenome.Hsapiens.UCSC.hg19, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - gwascat, type: imports\n",
      "  - BSgenome, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/circRNAprofiler.html\n",
      "\n",
      "Name: codelink\n",
      "Description:\n",
      "This package facilitates reading, preprocessing and manipulating Codelink microarray data. The raw data must be exported as text file using the Codelink software.\n",
      "Version: 1.66.0\n",
      "Publication date: None\n",
      "Mantainer: Diego Diez <diego10ruiz@gmail.com>\n",
      "Authors:\n",
      "Author: Diego Diez\n",
      "Dependencies:\n",
      "  - R (>= 2.10) type: depends\n",
      "  - BiocGenerics, type: depends\n",
      "  - methods, type: depends\n",
      "  - Biobase, type: depends\n",
      "  - limma, type: depends\n",
      "  - annotate, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/codelink.html\n",
      "\n",
      "Name: isomiRs\n",
      "Description:\n",
      "Characterization of miRNAs and isomiRs, clustering and differential expression.\n",
      "Version: 1.26.0\n",
      "Publication date: None\n",
      "Mantainer: Lorena Pantano <lorena.pantano@gmail.com>\n",
      "Authors:\n",
      "Author: Lorena Pantano [aut, cre], Georgia Escaramis [aut] (CIBERESP - CIBER Epidemiologia y Salud Publica)\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - AnnotationDbi, type: imports\n",
      "  - assertive.sets, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - broom, type: imports\n",
      "  - cluster, type: imports\n",
      "  - cowplot, type: imports\n",
      "  - DEGreport, type: imports\n",
      "  - DESeq2, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - gplots, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - gtools, type: imports\n",
      "  - gridExtra, type: imports\n",
      "  - grid, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - GGally, type: imports\n",
      "  - limma, type: imports\n",
      "  - methods, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - readr, type: imports\n",
      "  - reshape, type: imports\n",
      "  - rlang, type: imports\n",
      "  - stats, type: imports\n",
      "  - stringr, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - tibble, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/isomiRs.html\n",
      "\n",
      "Name: IVAS\n",
      "Description:\n",
      "Identification of genetic variants affecting alternative splicing.\n",
      "Version: 2.18.0\n",
      "Publication date: None\n",
      "Mantainer: Seonggyun Han <hangost@ssu.ac.kr>\n",
      "Authors:\n",
      "Author: Seonggyun Han, Sangsoo Kim\n",
      "Dependencies:\n",
      "  - R (> 3.0.0) type: depends\n",
      "  - GenomicFeatures, type: depends\n",
      "  - ggplot2, type: depends\n",
      "  - Biobase, type: depends\n",
      "  - doParallel, type: imports\n",
      "  - lme4, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - foreach, type: imports\n",
      "  - AnnotationDbi, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - ggfortify, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - methods, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - utils, type: imports\n",
      "  - stats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/IVAS.html\n",
      "\n",
      "Name: metaCCA\n",
      "Description:\n",
      "metaCCA performs multivariate analysis of a single or multiple GWAS based on univariate regression coefficients. It allows multivariate representation of both phenotype and genotype. metaCCA extends the statistical technique of canonical correlation analysis to the setting where original individual-level records are not available, and employs a covariance shrinkage algorithm to achieve robustness.\n",
      "Version: 1.26.0\n",
      "Publication date: None\n",
      "Mantainer: Anna Cichonska <anna.cichonska@gmail.com>\n",
      "Authors:\n",
      "Author: Anna Cichonska <anna.cichonska at gmail.com>\n",
      "Dependencies:\n",
      "None\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/metaCCA.html\n",
      "\n",
      "Name: MethylAid\n",
      "Description:\n",
      "A visual and interactive web application using RStudio's shiny package. Bad quality samples are detected using sample-dependent and sample-independent controls present on the array and user adjustable thresholds. In depth exploration of bad quality samples can be performed using several interactive diagnostic plots of the quality control probes present on the array. Furthermore, the impact of any batch effect provided by the user can be explored.\n",
      "Version: 1.32.0\n",
      "Publication date: None\n",
      "Mantainer: L.J.Sinke <L.J.Sinke@lumc.nl>\n",
      "Authors:\n",
      "Author: Maarten van Iterson [aut, cre], Elmar Tobi[ctb], Roderick Slieker[ctb], Wouter den Hollander[ctb], Rene Luijk[ctb] and Bas Heijmans[ctb]\n",
      "Dependencies:\n",
      "  - R (>= 3.4) type: depends\n",
      "  - Biobase, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - grid, type: imports\n",
      "  - gridBase, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - hexbin, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - minfi, type: imports\n",
      "  - methods, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - shiny, type: imports\n",
      "  - stats, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/MethylAid.html\n",
      "\n",
      "Name: mistyR\n",
      "Description:\n",
      "mistyR is an implementation of the Multiview Intercellular SpaTialmodeling framework (MISTy). MISTy is an explainable machine learning framework for knowledge extraction and analysis of single-cell, highly multiplexed, spatially resolved data. MISTy facilitates an in-depth understanding of marker interactions by profiling the intra- and intercellular relationships. MISTy is a flexible framework able to process a custom number of views. Each of these views can describe a different spatial context, i.e., define a relationship among the observed expressions of the markers, such as intracellular regulation or paracrine regulation, but also, the views can also capture cell-type specific relationships, capture relations between functional footprints or focus on relations between different anatomical regions. Each MISTy view is considered as a potential source of variability in the measured marker expressions. Each MISTy view is then analyzed for its contribution to the total expression of each marker and is explained in terms of the interactions with other measurements that led to the observed contribution.\n",
      "Version: 1.6.1\n",
      "Publication date: None\n",
      "Mantainer: Jovan Tanevski <jovan.tanevski@uni-heidelberg.de>\n",
      "Authors:\n",
      "Author: Jovan Tanevski [cre, aut] , Ricardo Omar Ramirez Flores [ctb] , Philipp SchÃ¤fer [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - assertthat, type: imports\n",
      "  - caret, type: imports\n",
      "  - deldir, type: imports\n",
      "  - digest, type: imports\n",
      "  - distances, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - filelock, type: imports\n",
      "  - furrr (>= 0.2.0) type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - methods, type: imports\n",
      "  - purrr, type: imports\n",
      "  - ranger, type: imports\n",
      "  - readr (>= 2.0.0) type: imports\n",
      "  - ridge, type: imports\n",
      "  - rlang, type: imports\n",
      "  - rlist, type: imports\n",
      "  - R.utils, type: imports\n",
      "  - stats, type: imports\n",
      "  - stringr, type: imports\n",
      "  - tibble, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - tidyselect (>= 1.2.0) type: imports\n",
      "  - utils, type: imports\n",
      "  - withr, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/mistyR.html\n",
      "\n",
      "Name: mitch\n",
      "Description:\n",
      "mitch is an R package for multi-contrast enrichment analysis. At itâs heart, it uses a rank-MANOVA based statistical approach to detect sets of genes that exhibit enrichment in the multidimensional space as compared to the background. The rank-MANOVA concept dates to work by Cox and Mann (https://doi.org/10.1186/1471-2105-13-S16-S12). mitch is useful for pathway analysis of profiling studies with one, two or more contrasts, or in studies with multiple omics profiling, for example proteomic, transcriptomic, epigenomic analysis of the same samples. mitch is perfectly suited for pathway level differential analysis of scRNA-seq data. The main strengths of mitch are that it can import datasets easily from many upstream tools and has advanced plotting features to visualise these enrichments.\n",
      "Version: 1.10.0\n",
      "Publication date: None\n",
      "Mantainer: Mark Ziemann <mark.ziemann@gmail.com>\n",
      "Authors:\n",
      "Author: Mark Ziemann [aut, cre, cph], Antony Kaspi [aut, cph]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - stats, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - utils, type: imports\n",
      "  - MASS, type: imports\n",
      "  - plyr, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - parallel, type: imports\n",
      "  - GGally, type: imports\n",
      "  - grid, type: imports\n",
      "  - gridExtra, type: imports\n",
      "  - knitr, type: imports\n",
      "  - rmarkdown, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - gplots, type: imports\n",
      "  - beeswarm, type: imports\n",
      "  - echarts4r, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "CC BY-SA 4.0 + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/mitch.html\n",
      "\n",
      "Name: musicatk\n",
      "Description:\n",
      "Mutational signatures are carcinogenic exposures or aberrant cellular processes that can cause alterations to the genome. We created musicatk (MUtational SIgnature Comprehensive Analysis ToolKit) to address shortcomings in versatility and ease of use in other pre-existing computational tools. Although many different types of mutational data have been generated, current software packages do not have a flexible framework to allow users to mix and match different types of mutations in the mutational signature inference process. Musicatk enables users to count and combine multiple mutation types, including SBS, DBS, and indels. Musicatk calculates replication strand, transcription strand and combinations of these features along with discovery from unique and proprietary genomic feature associated with any mutation type. Musicatk also implements several methods for discovery of new signatures as well as methods to infer exposure given an existing set of signatures. Musicatk provides functions for visualization and downstream exploratory analysis including the ability to compare signatures between cohorts and find matching signatures in COSMIC V2 or COSMIC V3.\n",
      "Version: 1.8.0\n",
      "Publication date: None\n",
      "Mantainer: Aaron Chevalier <atgc@bu.edu>\n",
      "Authors:\n",
      "Author: Aaron Chevalier [cre] (0000-0002-3968-9250), Joshua D. Campbell [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.0.0) type: depends\n",
      "  - NMF, type: depends\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - VariantAnnotation, type: imports\n",
      "  - cowplot, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - base, type: imports\n",
      "  - methods, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - tibble, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - gtools, type: imports\n",
      "  - gridExtra, type: imports\n",
      "  - MCMCprecision, type: imports\n",
      "  - MASS, type: imports\n",
      "  - matrixTests, type: imports\n",
      "  - data.table, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - rlang, type: imports\n",
      "  - BSgenome, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - GenomicFeatures, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - uwot, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - stringr, type: imports\n",
      "  - TxDb.Hsapiens.UCSC.hg19.knownGene, type: imports\n",
      "  - TxDb.Hsapiens.UCSC.hg38.knownGene, type: imports\n",
      "  - BSgenome.Hsapiens.UCSC.hg19, type: imports\n",
      "  - BSgenome.Hsapiens.UCSC.hg38, type: imports\n",
      "  - BSgenome.Mmusculus.UCSC.mm9, type: imports\n",
      "  - BSgenome.Mmusculus.UCSC.mm10, type: imports\n",
      "  - deconstructSigs, type: imports\n",
      "  - decompTumor2Sig, type: imports\n",
      "  - topicmodels, type: imports\n",
      "  - ggrepel, type: imports\n",
      "  - withr, type: imports\n",
      "  - plotly, type: imports\n",
      "  - utils, type: imports\n",
      "  - factoextra, type: imports\n",
      "  - cluster, type: imports\n",
      "  - ComplexHeatmap, type: imports\n",
      "  - philentropy, type: imports\n",
      "  - shinydashboard, type: imports\n",
      "  - sortable, type: imports\n",
      "  - maftools, type: imports\n",
      "  - shiny, type: imports\n",
      "  - shinyjs, type: imports\n",
      "  - shinyalert, type: imports\n",
      "  - shinybusy, type: imports\n",
      "  - shinyBS, type: imports\n",
      "  - TCGAbiolinks, type: imports\n",
      "  - shinyjqui, type: imports\n",
      "  - stringi, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/musicatk.html\n",
      "\n",
      "Name: NetPathMiner\n",
      "Description:\n",
      "NetPathMiner is a general framework for network path mining using genome-scale networks. It constructs networks from KGML, SBML and BioPAX files, providing three network representations, metabolic, reaction and gene representations. NetPathMiner finds active paths and applies machine learning methods to summarize found paths for easy interpretation. It also provides static and interactive visualizations of networks and paths to aid manual investigation.\n",
      "Version: 1.34.2\n",
      "Publication date: None\n",
      "Mantainer: Ahmed Mohamed <mohamed@kuicr.kyoto-u.ac.jp>\n",
      "Authors:\n",
      "Author: Ahmed Mohamed <mohamed at kuicr.kyoto-u.ac.jp>, Tim Hancock <timothy.hancock at kuicr.kyoto-u.ac.jp>, Ichigaku Takigawa <takigawa at kuicr.kyoto-u.ac.jp>, Nicolas Wicker <nicolas.wicker at unistra.fr>\n",
      "Dependencies:\n",
      "  - R (>= 3.0.2) type: depends\n",
      "  - igraph (>= 1.0) type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/NetPathMiner.html\n",
      "\n",
      "Name: pkgDepTools\n",
      "Description:\n",
      "This package provides tools for computing and analyzing dependency relationships among R packages. It provides tools for building a graph-based representation of the dependencies among all packages in a list of CRAN-style package repositories. There are also utilities for computing installation order of a given package. If the RCurl package is available, an estimate of the download size required to install a given package and its dependencies can be obtained.\n",
      "Version: 1.63.0\n",
      "Publication date: None\n",
      "Mantainer: Bioconductor Package Maintainer <maintainer@bioconductor.org>\n",
      "Authors:\n",
      "Author: Seth Falcon [aut], Bioconductor Package Maintainer [cre]\n",
      "Dependencies:\n",
      "  - methods, type: depends\n",
      "  - graph, type: depends\n",
      "  - RBGL, type: depends\n",
      "  - graph, type: imports\n",
      "  - RBGL, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/pkgDepTools.html\n",
      "\n",
      "Name: rmelting\n",
      "Description:\n",
      "R interface to the MELTING 5 program (https://www.ebi.ac.uk/biomodels/tools/melting/) to compute melting temperatures of nucleic acid duplexes along with other thermodynamic parameters.\n",
      "Version: 1.14.0\n",
      "Publication date: None\n",
      "Mantainer: J. Aravind <j.aravind@icar.gov.in>\n",
      "Authors:\n",
      "Author: J. Aravind [aut, cre] , G. K. Krishna [aut], Bob Rudis [ctb] (melting5jars), Nicolas Le NovÃ¨re [ctb] (MELTING 5 Java Library), Marine Dumousseau [ctb] (MELTING 5 Java Library), William John Gowers [ctb] (MELTING 5 Java Library)\n",
      "Dependencies:\n",
      "  - R (>= 3.6) type: depends\n",
      "  - Rdpack, type: imports\n",
      "  - rJava (>= 0.9-8) type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2 | GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/rmelting.html\n",
      "\n",
      "Name: RTNsurvival\n",
      "Description:\n",
      "RTNsurvival is a tool for integrating regulons generated by the RTN package with survival information. For a given regulon, the 2-tailed GSEA approach computes a differential Enrichment Score (dES) for each individual sample, and the dES distribution of all samples is then used to assess the survival statistics for the cohort. There are two main survival analysis workflows: a Cox Proportional Hazards approach used to model regulons as predictors of survival time, and a Kaplan-Meier analysis assessing the stratification of a cohort based on the regulon activity. All plots can be fine-tuned to the user's specifications.\n",
      "Version: 1.22.0\n",
      "Publication date: None\n",
      "Mantainer: Clarice Groeneveld <clari.groeneveld@gmail.com>, Mauro A. A. Castro <mauro.a.castro@gmail.com>\n",
      "Authors:\n",
      "Author: Clarice S. Groeneveld, Vinicius S. Chagas, Mauro A. A. Castro\n",
      "Dependencies:\n",
      "  - R (>= 3.6.3) type: depends\n",
      "  - RTN, type: depends\n",
      "  - RTNduals, type: depends\n",
      "  - methods, type: depends\n",
      "  - survival, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - scales, type: imports\n",
      "  - data.table, type: imports\n",
      "  - egg, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - pheatmap, type: imports\n",
      "  - dunn.test, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/RTNsurvival.html\n",
      "\n",
      "Name: rTRM\n",
      "Description:\n",
      "rTRM identifies transcriptional regulatory modules (TRMs) from protein-protein interaction networks.\n",
      "Version: 1.36.0\n",
      "Publication date: None\n",
      "Mantainer: Diego Diez <diego10ruiz@gmail.com>\n",
      "Authors:\n",
      "Author: Diego Diez\n",
      "Dependencies:\n",
      "  - R (>= 2.10) type: depends\n",
      "  - igraph (>= 1.0) type: depends\n",
      "  - methods, type: imports\n",
      "  - AnnotationDbi, type: imports\n",
      "  - DBI, type: imports\n",
      "  - RSQLite, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/rTRM.html\n",
      "\n",
      "Name: savR\n",
      "Description:\n",
      "Parse Illumina Sequence Analysis Viewer (SAV) files, access data, and generate QC plots.\n",
      "Version: 1.36.0\n",
      "Publication date: None\n",
      "Mantainer: R. Brent Calder <brent.calder@einstein.yu.edu>\n",
      "Authors:\n",
      "Author: R. Brent Calder\n",
      "Dependencies:\n",
      "  - ggplot2, type: depends\n",
      "  - methods, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - scales, type: imports\n",
      "  - gridExtra, type: imports\n",
      "  - XML, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "AGPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/savR.html\n",
      "\n",
      "Name: scruff\n",
      "Description:\n",
      "A pipeline which processes single cell RNA-seq (scRNA-seq) reads from CEL-seq and CEL-seq2 protocols. Demultiplex scRNA-seq FASTQ files, align reads to reference genome using Rsubread, and generate UMI filtered count matrix. Also provide visualizations of read alignments and pre- and post-alignment QC metrics.\n",
      "Version: 1.16.0\n",
      "Publication date: None\n",
      "Mantainer: Zhe Wang <zhe@bu.edu>\n",
      "Authors:\n",
      "Author: Zhe Wang [aut, cre], Junming Hu [aut], Joshua Campbell [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - data.table, type: imports\n",
      "  - GenomicAlignments, type: imports\n",
      "  - GenomicFeatures, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - ShortRead, type: imports\n",
      "  - parallel, type: imports\n",
      "  - plyr, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - AnnotationDbi, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - methods, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - ggthemes, type: imports\n",
      "  - scales, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - stringdist, type: imports\n",
      "  - ggbio, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - SingleCellExperiment, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - Rsubread, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/scruff.html\n",
      "\n",
      "Name: seqsetvis\n",
      "Description:\n",
      "seqsetvis enables the visualization and analysis of sets of genomic sites in next gen sequencing data. Although seqsetvis was designed for the comparison of mulitple ChIP-seq samples, this package is domain-agnostic and allows the processing of multiple genomic coordinate files (bed-like files) and signal files (bigwig files pileups from bam file).\n",
      "Version: 1.18.0\n",
      "Publication date: None\n",
      "Mantainer: Joseph R Boyd <jrboyd@uvm.edu>\n",
      "Authors:\n",
      "Author: Joseph R Boyd [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 3.6) type: depends\n",
      "  - ggplot2, type: depends\n",
      "  - cowplot, type: imports\n",
      "  - data.table, type: imports\n",
      "  - eulerr, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - GenomicAlignments, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - ggplotify, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - grid, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - limma, type: imports\n",
      "  - methods, type: imports\n",
      "  - pbapply, type: imports\n",
      "  - pbmcapply, type: imports\n",
      "  - png, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - stats, type: imports\n",
      "  - UpSetR, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/seqsetvis.html\n",
      "\n",
      "Name: abseqR\n",
      "Description:\n",
      "AbSeq is a comprehensive bioinformatic pipeline for the analysis of sequencing datasets generated from antibody libraries and abseqR is one of its packages. abseqR empowers the users of abseqPy (https://github.com/malhamdoosh/abseqPy) with plotting and reporting capabilities and allows them to generate interactive HTML reports for the convenience of viewing and sharing with other researchers. Additionally, abseqR extends abseqPy to compare multiple repertoire analyses and perform further downstream analysis on its output.\n",
      "Version: 1.16.0\n",
      "Publication date: None\n",
      "Mantainer: JiaHong Fong <jiahfong@gmail.com>\n",
      "Authors:\n",
      "Author: JiaHong Fong [cre, aut], Monther Alhamdoosh [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - ggplot2, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - circlize, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - VennDiagram, type: imports\n",
      "  - plyr, type: imports\n",
      "  - flexdashboard, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - png, type: imports\n",
      "  - grid, type: imports\n",
      "  - gridExtra, type: imports\n",
      "  - rmarkdown, type: imports\n",
      "  - knitr, type: imports\n",
      "  - vegan, type: imports\n",
      "  - ggcorrplot, type: imports\n",
      "  - ggdendro, type: imports\n",
      "  - plotly, type: imports\n",
      "  - BiocStyle, type: imports\n",
      "  - stringr, type: imports\n",
      "  - utils, type: imports\n",
      "  - methods, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - stats, type: imports\n",
      "  - tools, type: imports\n",
      "  - graphics, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3 | file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/abseqR.html\n",
      "\n",
      "Name: ADAM\n",
      "Description:\n",
      "ADAM is a GSEA R package created to group a set of genes from comparative samples (control versus experiment) belonging to different species according to their respective functions (Gene Ontology and KEGG pathways as default) and show their significance by calculating p-values referring togene diversity and activity. Each group of genes is called GFAG (Group of Functionally Associated Genes).\n",
      "Version: 1.14.0\n",
      "Publication date: None\n",
      "Mantainer: Jose Luiz Rybarczyk Filho <jose.luiz@unesp.br>\n",
      "Authors:\n",
      "Author: AndrÃ© Luiz Molan [aut], Giordano Bruno Sanches Seco [ctb], Agnes Takeda [ctb], Jose Rybarczyk Filho [ctb, cre, ths]\n",
      "Dependencies:\n",
      "  - R (>= 3.5) type: depends\n",
      "  - stats, type: depends\n",
      "  - utils, type: depends\n",
      "  - methods, type: depends\n",
      "  - Rcpp (>= 0.12.18) type: imports\n",
      "  - GO.db, type: imports\n",
      "  - KEGGREST, type: imports\n",
      "  - knitr, type: imports\n",
      "  - pbapply (>= 1.3-4) type: imports\n",
      "  - dplyr (>= 0.7.6) type: imports\n",
      "  - DT (>= 0.4) type: imports\n",
      "  - stringr (>= 1.3.1) type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ADAM.html\n",
      "\n",
      "Name: Anaquin\n",
      "Description:\n",
      "The project is intended to support the use of sequins (synthetic sequencing spike-in controls) owned and made available by the Garvan Institute of Medical Research. The goal is to provide a standard open source library for quantitative analysis, modelling and visualization of spike-in controls.\n",
      "Version: 2.22.0\n",
      "Publication date: None\n",
      "Mantainer: Ted Wong <t.wong@garvan.org.au>\n",
      "Authors:\n",
      "Author: Ted Wong\n",
      "Dependencies:\n",
      "  - R (>= 3.3) type: depends\n",
      "  - ggplot2 (>= 2.2.0) type: depends\n",
      "  - ggplot2, type: imports\n",
      "  - ROCR, type: imports\n",
      "  - knitr, type: imports\n",
      "  - qvalue, type: imports\n",
      "  - locfit, type: imports\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - plyr, type: imports\n",
      "  - DESeq2, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "BSD_3_clause + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/Anaquin.html\n",
      "\n",
      "Name: BaseSpaceR\n",
      "Description:\n",
      "A rich R interface to Illumina's BaseSpace cloud computing environment, enabling the fast development of data analysis and visualisation tools.\n",
      "Version: 1.42.0\n",
      "Publication date: None\n",
      "Mantainer: Jared O'Connell <joconnell@illumina.com>\n",
      "Authors:\n",
      "Author: Adrian Alexa\n",
      "Dependencies:\n",
      "  - R (>= 2.15.0) type: depends\n",
      "  - RCurl, type: depends\n",
      "  - RJSONIO, type: depends\n",
      "  - methods, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Apache License 2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/BaseSpaceR.html\n",
      "\n",
      "Name: BloodGen3Module\n",
      "Description:\n",
      "The BloodGen3Module package provides functions for R user performing module repertoire analyses and generating fingerprint representations. Functions can perform group comparison or individual sample analysis and visualization by fingerprint grid plot or fingerprint heatmap. Module repertoire analyses typically involve determining the percentage of the constitutive genes for each module that are significantly increased or decreased. As we describe in details;https://www.biorxiv.org/content/10.1101/525709v2 and https://pubmed.ncbi.nlm.nih.gov/33624743/, the results of module repertoire analyses can be represented in a fingerprint format, where red and blue spots indicate increases or decreases in module activity. These spots are subsequently represented either on a grid, with each position being assigned to a given module, or in a heatmap where the samples are arranged in columns and the modules in rows.\n",
      "Version: 1.6.0\n",
      "Publication date: None\n",
      "Mantainer: Darawan Rinchai <drinchai@gmail.com>\n",
      "Authors:\n",
      "Author: Darawan Rinchai [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - ExperimentHub, type: imports\n",
      "  - methods, type: imports\n",
      "  - grid, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - circlize, type: imports\n",
      "  - testthat, type: imports\n",
      "  - ComplexHeatmap, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - gtools, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - preprocessCore, type: imports\n",
      "  - randomcoloR, type: imports\n",
      "  - V8, type: imports\n",
      "  - limma, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/BloodGen3Module.html\n",
      "\n",
      "Name: CausalR\n",
      "Description:\n",
      "Causal network analysis methods for regulator prediction and network reconstruction from genome scale data.\n",
      "Version: 1.30.0\n",
      "Publication date: None\n",
      "Mantainer: Glyn Bradley <glyn.x.bradley@gsk.com>, Steven Barrett <steven.j.barrett@gsk.com>\n",
      "Authors:\n",
      "Author: Glyn Bradley, Steven Barrett, Chirag Mistry, Mark Pipe, David Wille, David Riley, Bhushan Bonde, Peter Woollard\n",
      "Dependencies:\n",
      "  - R (>= 3.2.0) type: depends\n",
      "  - igraph, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/CausalR.html\n",
      "\n",
      "Name: CFAssay\n",
      "Description:\n",
      "The package provides functions for calculation of linear-quadratic cell survival curves and for ANOVA of experimental 2-way designs along with the colony formation assay.\n",
      "Version: 1.32.0\n",
      "Publication date: None\n",
      "Mantainer: Herbert Braselmann <hbraselmann@online.de>\n",
      "Authors:\n",
      "Author: Herbert Braselmann\n",
      "Dependencies:\n",
      "  - R (>= 2.10.0) type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/CFAssay.html\n",
      "\n",
      "Name: CINdex\n",
      "Description:\n",
      "The CINdex package addresses important area of high-throughput genomic analysis. It allows the automated processing and analysis of the experimental DNA copy number data generated by Affymetrix SNP 6.0 arrays or similar high throughput technologies. It calculates the chromosome instability (CIN) index that allows to quantitatively characterize genome-wide DNA copy number alterations as a measure of chromosomal instability. This package calculates not only overall genomic instability, but also instability in terms of copy number gains and losses separately at the chromosome and cytoband level.\n",
      "Version: 1.26.0\n",
      "Publication date: None\n",
      "Mantainer: Yuriy Gusev <yg63@georgetown.edu>\n",
      "Authors:\n",
      "Author: Lei Song [aut] (Innovation Center for Biomedical Informatics, Georgetown University Medical Center), Krithika Bhuvaneshwar [aut] (Innovation Center for Biomedical Informatics, Georgetown University Medical Center), Yue Wang [aut, ths] (Virginia Polytechnic Institute and State University), Yuanjian Feng [aut] (Virginia Polytechnic Institute and State University), Ie-Ming Shih [aut] (Johns Hopkins University School of Medicine), Subha Madhavan [aut] (Innovation Center for Biomedical Informatics, Georgetown University Medical Center), Yuriy Gusev [aut, cre] (Innovation Center for Biomedical Informatics, Georgetown University Medical Center)\n",
      "Dependencies:\n",
      "  - R (>= 3.3) type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - bitops, type: imports\n",
      "  - gplots, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - som, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - gridExtra, type: imports\n",
      "  - png, type: imports\n",
      "  - stringr, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/CINdex.html\n",
      "\n",
      "Name: consensusOV\n",
      "Description:\n",
      "This package implements four major subtype classifiers for high-grade serous (HGS) ovarian cancer as described by Helland et al. (PLoS One, 2011), Bentink et al. (PLoS One, 2012), Verhaak et al. (J Clin Invest, 2013), and Konecny et al. (J Natl Cancer Inst, 2014). In addition, the package implements a consensus classifier, which consolidates and improves on the robustness of the proposed subtype classifiers, thereby providing reliable stratification of patients with HGS ovarian tumors of clearly defined subtype.\n",
      "Version: 1.20.0\n",
      "Publication date: None\n",
      "Mantainer: Benjamin Haibe-Kains <benjamin.haibe.kains@utoronto.ca>\n",
      "Authors:\n",
      "Author: Gregory M Chen, Lavanya Kannan, Ludwig Geistlinger, Victor Kofia, Levi Waldron, Benjamin Haibe-Kains\n",
      "Dependencies:\n",
      "  - R (>= 3.6) type: depends\n",
      "  - Biobase, type: imports\n",
      "  - GSVA, type: imports\n",
      "  - gdata, type: imports\n",
      "  - genefu, type: imports\n",
      "  - limma, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - randomForest, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - methods, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/consensusOV.html\n",
      "\n",
      "Name: DART\n",
      "Description:\n",
      "Denoising Algorithm based on Relevance network Topology (DART) is an algorithm designed to evaluate the consistency of prior information molecular signatures (e.g in-vitro perturbation expression signatures) in independent molecular data (e.g gene expression data sets). If consistent, a pruning network strategy is then used to infer the activation status of the molecular signature in individual samples.\n",
      "Version: 1.46.0\n",
      "Publication date: None\n",
      "Mantainer: Charles Shijie Zheng <charles_zheng@live.com>\n",
      "Authors:\n",
      "Author: Yan Jiao, Katherine Lawler, Andrew E Teschendorff, Charles Shijie Zheng\n",
      "Dependencies:\n",
      "  - R (>= 2.10.0) type: depends\n",
      "  - igraph (>= 0.6.0) type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/DART.html\n",
      "\n",
      "Name: FitHiC\n",
      "Description:\n",
      "Fit-Hi-C is a tool for assigning statistical confidence estimates to intra-chromosomal contact maps produced by genome-wide genome architecture assays such as Hi-C.\n",
      "Version: 1.24.0\n",
      "Publication date: None\n",
      "Mantainer: Ruyu Tan <rut003@ucsd.edu>\n",
      "Authors:\n",
      "Author: Ferhat Ay [aut] (Python original, https://noble.gs.washington.edu/proj/fit-hi-c/), Timothy L. Bailey [aut], William S. Noble [aut], Ruyu Tan [aut, cre, trl] (R port)\n",
      "Dependencies:\n",
      "  - data.table, type: imports\n",
      "  - fdrtool, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - Rcpp, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/FitHiC.html\n",
      "\n",
      "Name: fmrs\n",
      "Description:\n",
      "The package obtains parameter estimation, i.e., maximum likelihood estimators (MLE), via the Expectation-Maximization (EM) algorithm for the Finite Mixture of Regression (FMR) models with Normal distribution, and MLE for the Finite Mixture of Accelerated Failure Time Regression (FMAFTR) subject to right censoring with Log-Normal and Weibull distributions via the EM algorithm and the Newton-Raphson algorithm (for Weibull distribution). More importantly, the package obtains the maximum penalized likelihood (MPLE) for both FMR and FMAFTR models (collectively called FMRs). A component-wise tuning parameter selection based on a component-wise BIC is implemented in the package. Furthermore, this package provides Ridge Regression and Elastic Net.\n",
      "Version: 1.8.0\n",
      "Publication date: None\n",
      "Mantainer: Farhad Shokoohi <shokoohi@icloud.com>\n",
      "Authors:\n",
      "Author: Farhad Shokoohi [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.1.0) type: depends\n",
      "  - methods, type: imports\n",
      "  - survival, type: imports\n",
      "  - stats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/fmrs.html\n",
      "\n",
      "Name: intansv\n",
      "Description:\n",
      "This package provides efficient tools to read and integrate structural variations predicted by popular softwares. Annotation and visulation of structural variations are also implemented in the package.\n",
      "Version: 1.38.0\n",
      "Publication date: None\n",
      "Mantainer: Wen Yao <ywhzau@gmail.com>\n",
      "Authors:\n",
      "Author: Wen Yao <ywhzau at gmail.com>\n",
      "Dependencies:\n",
      "  - R (>= 2.14.0) type: depends\n",
      "  - plyr, type: depends\n",
      "  - ggbio, type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - BiocGenerics, type: imports\n",
      "  - IRanges, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/intansv.html\n",
      "\n",
      "Name: interacCircos\n",
      "Description:\n",
      "Implement in an efficient approach to display the genomic data, relationship, information in an interactive circular genome(Circos) plot. 'interacCircos' are inspired by 'circosJS', 'BioCircos.js' and 'NG-Circos' and we integrate the modules of 'circosJS', 'BioCircos.js' and 'NG-Circos' into this R package, based on 'htmlwidgets' framework.\n",
      "Version: 1.8.0\n",
      "Publication date: None\n",
      "Mantainer: Zhe Cui <mrcuizhe@gmail.com>\n",
      "Authors:\n",
      "Author: Zhe Cui [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - RColorBrewer, type: imports\n",
      "  - htmlwidgets, type: imports\n",
      "  - plyr, type: imports\n",
      "  - methods, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/interacCircos.html\n",
      "\n",
      "Name: LPE\n",
      "Description:\n",
      "This LPE library is used to do significance analysis of microarray data with small number of replicates. It uses resampling based FDR adjustment, and gives less conservative results than traditional 'BH' or 'BY' procedures. Data accepted is raw data in txt format from MAS4, MAS5 or dChip. Data can also be supplied after normalization. LPE library is primarily used for analyzing data between two conditions. To use it for paired data, see LPEP library. For using LPE in multiple conditions, use HEM library.\n",
      "Version: 1.72.0\n",
      "Publication date: None\n",
      "Mantainer: Nitin Jain <emailnitinjain@gmail.com>\n",
      "Authors:\n",
      "Author: Nitin Jain <emailnitinjain at gmail.com>, Michael O'Connell <michaelo at warath.com>, Jae K. Lee <jaeklee at virginia.edu>. Includes R source code contributed by HyungJun Cho <hcho at virginia.edu>\n",
      "Dependencies:\n",
      "  - R (>= 2.10) type: depends\n",
      "  - stats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/LPE.html\n",
      "\n",
      "Name: MatrixQCvis\n",
      "Description:\n",
      "Data quality assessment is an integral part of preparatory data analysis to ensure sound biological information retrieval. We present here the MatrixQCvis package, which provides shiny-based interactive visualization of data quality metrics at the per-sample and per-feature level. It is broadly applicable to quantitative omics data types that come in matrix-like format (features x samples). It enables the detection of low-quality samples, drifts, outliers and batch effects in data sets. Visualizations include amongst others bar- and violin plots of the (count/intensity) values, mean vs standard deviation plots, MA plots, empirical cumulative distribution function (ECDF) plots, visualizations of the distances between samples, and multiple types of dimension reduction plots. Furthermore, MatrixQCvis allows for differential expression analysis based on the limma (moderated t-tests) and proDA (Wald tests) packages. MatrixQCvis builds upon the popular Bioconductor SummarizedExperiment S4 class and enables thus the facile integration into existing workflows. The package is especially tailored towards metabolomics and proteomics mass spectrometry data, but also allows to assess the data quality of other data types that can be represented in a SummarizedExperiment object.\n",
      "Version: 1.6.1\n",
      "Publication date: None\n",
      "Mantainer: Thomas Naake <thomasnaake@googlemail.com>\n",
      "Authors:\n",
      "Author: Thomas Naake [aut, cre] , Wolfgang Huber [aut]\n",
      "Dependencies:\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - plotly (>= 4.9.3) type: depends\n",
      "  - shiny (>= 1.6.0) type: depends\n",
      "  - ComplexHeatmap, type: imports\n",
      "  - dplyr (>= 1.0.5) type: imports\n",
      "  - ggplot2 (>= 3.3.3) type: imports\n",
      "  - grDevices (>= 4.1.0) type: imports\n",
      "  - Hmisc (>= 4.5-0) type: imports\n",
      "  - htmlwidgets (>= 1.5.3) type: imports\n",
      "  - impute, type: imports\n",
      "  - imputeLCMD (>= 2.0) type: imports\n",
      "  - limma, type: imports\n",
      "  - MASS (>= 7.3-58.1) type: imports\n",
      "  - methods (>= 4.1.0) type: imports\n",
      "  - pcaMethods, type: imports\n",
      "  - proDA, type: imports\n",
      "  - rlang (>= 0.4.10) type: imports\n",
      "  - rmarkdown (>= 2.7) type: imports\n",
      "  - Rtsne (>= 0.15) type: imports\n",
      "  - shinydashboard (>= 0.7.1) type: imports\n",
      "  - shinyhelper (>= 0.3.2) type: imports\n",
      "  - shinyjs (>= 2.0.0) type: imports\n",
      "  - stats (>= 4.1.0) type: imports\n",
      "  - tibble (>= 3.1.1) type: imports\n",
      "  - tidyr (>= 1.1.3) type: imports\n",
      "  - umap (>= 0.2.7.0) type: imports\n",
      "  - UpSetR (>= 1.4.0) type: imports\n",
      "  - vsn, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 3)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/MatrixQCvis.html\n",
      "\n",
      "Name: MetaCyto\n",
      "Description:\n",
      "This package provides functions for preprocessing, automated gating and meta-analysis of cytometry data. It also provides functions that facilitate the collection of cytometry data from the ImmPort database.\n",
      "Version: 1.20.0\n",
      "Publication date: None\n",
      "Mantainer: Zicheng Hu <zicheng.hu@ucsf.edu>\n",
      "Authors:\n",
      "Author: Zicheng Hu, Chethan Jujjavarapu, Sanchita Bhattacharya, Atul J. Butte\n",
      "Dependencies:\n",
      "  - R (>= 3.4) type: depends\n",
      "  - flowCore, type: imports\n",
      "  - tidyr (>= 0.7) type: imports\n",
      "  - fastcluster, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - metafor, type: imports\n",
      "  - cluster, type: imports\n",
      "  - FlowSOM, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/MetaCyto.html\n",
      "\n",
      "Name: MOMA\n",
      "Description:\n",
      "This package implements the inference of candidate master regulator proteins from multi-omics' data (MOMA) algorithm, as well as ancillary analysis and visualization functions.\n",
      "Version: 1.10.0\n",
      "Publication date: None\n",
      "Mantainer: Sunny Jones <sunnyjjones@gmail.com>\n",
      "Authors:\n",
      "Author: Evan Paull [aut], Sunny Jones [aut, cre], Mariano Alvarez [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - circlize, type: imports\n",
      "  - cluster, type: imports\n",
      "  - ComplexHeatmap, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grid, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - methods, type: imports\n",
      "  - MKmisc, type: imports\n",
      "  - MultiAssayExperiment, type: imports\n",
      "  - parallel, type: imports\n",
      "  - qvalue, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - readr, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - rlang, type: imports\n",
      "  - stats, type: imports\n",
      "  - stringr, type: imports\n",
      "  - tibble, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/MOMA.html\n",
      "\n",
      "Name: MoonlightR\n",
      "Description:\n",
      "Motivation: The understanding of cancer mechanism requires the identification of genes playing a role in the development of the pathology and the characterization of their role (notably oncogenes and tumor suppressors). Results: We present an R/bioconductor package called MoonlightR which returns a list of candidate driver genes for specific cancer types on the basis of TCGA expression data. The method first infers gene regulatory networks and then carries out a functional enrichment analysis (FEA) (implementing an upstream regulator analysis, URA) to score the importance of well-known biological processes with respect to the studied cancer type. Eventually, by means of random forests, MoonlightR predicts two specific roles for the candidate driver genes: i) tumor suppressor genes (TSGs) and ii) oncogenes (OCGs). As a consequence, this methodology does not only identify genes playing a dual role (e.g. TSG in one cancer type and OCG in another) but also helps in elucidating the biological processes underlying their specific roles. In particular, MoonlightR can be used to discover OCGs and TSGs in the same cancer type. This may help in answering the question whether some genes change role between early stages (I, II) and late stages (III, IV) in breast cancer. In the future, this analysis could be useful to determine the causes of different resistances to chemotherapeutic treatments.\n",
      "Version: 1.24.0\n",
      "Publication date: None\n",
      "Mantainer: Matteo Tiberti <tiberti@cancer.dk>\n",
      "Authors:\n",
      "Author: Antonio Colaprico [aut], Catharina Olsen [aut], Matthew H. Bailey [aut], Gabriel J. Odom [aut], Thilde Terkelsen [aut], Mona Nourbakhsh [aut], Astrid Saksager [aut], Tiago C. Silva [aut], AndrÃ© V. Olsen [aut], Laura Cantini [aut], Andrei Zinovyev [aut], Emmanuel Barillot [aut], Houtan Noushmehr [aut], Gloria Bertoli [aut], Isabella Castiglioni [aut], Claudia Cava [aut], Gianluca Bontempi [aut], Xi Steven Chen [aut], Elena Papaleo [aut], Matteo Tiberti [cre, aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.5) type: depends\n",
      "  - doParallel, type: depends\n",
      "  - foreach, type: depends\n",
      "  - parmigene, type: imports\n",
      "  - randomForest, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - gplots, type: imports\n",
      "  - circlize, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - HiveR, type: imports\n",
      "  - clusterProfiler, type: imports\n",
      "  - DOSE, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - limma, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - TCGAbiolinks, type: imports\n",
      "  - GEOquery, type: imports\n",
      "  - stats, type: imports\n",
      "  - RISmed, type: imports\n",
      "  - grid, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 3)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/MoonlightR.html\n",
      "\n",
      "Name: mumosa\n",
      "Description:\n",
      "Assorted utilities for multi-modal analyses of single-cell datasets. Includes functions to combine multiple modalities for downstream analysis, perform MNN-based batch correction across multiple modalities, and to compute correlations between assay values for different modalities.\n",
      "Version: 1.6.0\n",
      "Publication date: None\n",
      "Mantainer: Aaron Lun <infinite.monkeys.with.keyboards@gmail.com>\n",
      "Authors:\n",
      "Author: Aaron Lun [aut, cre]\n",
      "Dependencies:\n",
      "  - SingleCellExperiment, type: depends\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - methods, type: imports\n",
      "  - igraph, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - DelayedArray, type: imports\n",
      "  - DelayedMatrixStats, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - BiocNeighbors, type: imports\n",
      "  - BiocSingular, type: imports\n",
      "  - ScaledMatrix, type: imports\n",
      "  - beachmat, type: imports\n",
      "  - scuttle, type: imports\n",
      "  - metapod, type: imports\n",
      "  - scran, type: imports\n",
      "  - batchelor, type: imports\n",
      "  - uwot, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/mumosa.html\n",
      "\n",
      "Name: Oscope\n",
      "Description:\n",
      "Oscope is a statistical pipeline developed to identifying and recovering the base cycle profiles of oscillating genes in an unsynchronized single cell RNA-seq experiment. The Oscope pipeline includes three modules: a sine model module to search for candidate oscillator pairs; a K-medoids clustering module to cluster candidate oscillators into groups; and an extended nearest insertion module to recover the base cycle order for each oscillator group.\n",
      "Version: 1.28.0\n",
      "Publication date: None\n",
      "Mantainer: Ning Leng <lengning1@gmail.com>\n",
      "Authors:\n",
      "Author: Ning Leng\n",
      "Dependencies:\n",
      "  - EBSeq, type: depends\n",
      "  - cluster, type: depends\n",
      "  - testthat, type: depends\n",
      "  - BiocParallel, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/Oscope.html\n",
      "\n",
      "Name: PAA\n",
      "Description:\n",
      "PAA imports single color (protein) microarray data that has been saved in gpr file format - esp. ProtoArray data. After preprocessing (background correction, batch filtering, normalization) univariate feature preselection is performed (e.g., using the \"minimum M statistic\" approach - hereinafter referred to as \"mMs\"). Subsequently, a multivariate feature selection is conducted to discover biomarker candidates. Therefore, either a frequency-based backwards elimination aproach or ensemble feature selection can be used. PAA provides a complete toolbox of analysis tools including several different plots for results examination and evaluation.\n",
      "Version: 1.32.0\n",
      "Publication date: None\n",
      "Mantainer: Michael Turewicz <michael.turewicz@rub.de>, Martin Eisenacher <martin.eisenacher@rub.de>\n",
      "Authors:\n",
      "Author: Michael Turewicz [aut, cre], Martin Eisenacher [ctb, cre]\n",
      "Dependencies:\n",
      "  - R (>= 3.2.0) type: depends\n",
      "  - Rcpp (>= 0.11.6) type: depends\n",
      "  - e1071, type: imports\n",
      "  - gplots, type: imports\n",
      "  - gtools, type: imports\n",
      "  - limma, type: imports\n",
      "  - MASS, type: imports\n",
      "  - mRMRe, type: imports\n",
      "  - randomForest, type: imports\n",
      "  - ROCR, type: imports\n",
      "  - sva, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "BSD_3_clause + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/PAA.html\n",
      "\n",
      "Name: qcmetrics\n",
      "Description:\n",
      "The package provides a framework for generic quality control of data. It permits to create, manage and visualise individual or sets of quality control metrics and generate quality control reports in various formats.\n",
      "Version: 1.36.0\n",
      "Publication date: None\n",
      "Mantainer: Laurent Gatto <laurent.gatto@uclouvain.be>\n",
      "Authors:\n",
      "Author: Laurent Gatto [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 3.3) type: depends\n",
      "  - Biobase, type: imports\n",
      "  - methods, type: imports\n",
      "  - knitr, type: imports\n",
      "  - tools, type: imports\n",
      "  - xtable, type: imports\n",
      "  - pander, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/qcmetrics.html\n",
      "\n",
      "Name: RCX\n",
      "Description:\n",
      "Create, handle, validate, visualize and convert networks in the Cytoscape exchange (CX) format to standard data types and objects. The package also provides conversion to and from objects of iGraph and graphNEL. The CX format is also used by the NDEx platform, a online commons for biological networks, and the network visualization software Cytocape.\n",
      "Version: 1.2.2\n",
      "Publication date: None\n",
      "Mantainer: Florian Auer <florian.auer@informatik.uni-augsburg.de>\n",
      "Authors:\n",
      "Author: Florian Auer [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.2.0) type: depends\n",
      "  - jsonlite, type: imports\n",
      "  - plyr, type: imports\n",
      "  - igraph, type: imports\n",
      "  - methods, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/RCX.html\n",
      "\n",
      "Name: rhdf5client\n",
      "Description:\n",
      "Provides functionality for reading data from h5serv server from within R.\n",
      "Version: 1.20.0\n",
      "Publication date: None\n",
      "Mantainer: Vincent Carey <stvjc@channing.harvard.edu>\n",
      "Authors:\n",
      "Author: Samuela Pollack [aut], Shweta Gopaulakrishnan [aut], BJ Stubbs [aut], Vincent Carey [cre, aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.6) type: depends\n",
      "  - methods, type: depends\n",
      "  - DelayedArray, type: depends\n",
      "  - S4Vectors, type: imports\n",
      "  - httr, type: imports\n",
      "  - R6, type: imports\n",
      "  - rjson, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/rhdf5client.html\n",
      "\n",
      "Name: riboSeqR\n",
      "Description:\n",
      "Plotting functions, frameshift detection and parsing of sequencing data from ribosome profiling experiments.\n",
      "Version: 1.32.0\n",
      "Publication date: None\n",
      "Mantainer: Thomas J. Hardcastle <tjh48@cam.ac.uk>\n",
      "Authors:\n",
      "Author: Thomas J. Hardcastle\n",
      "Dependencies:\n",
      "  - R (>= 3.0.2) type: depends\n",
      "  - methods, type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - abind, type: depends\n",
      "  - Rsamtools, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - baySeq, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - seqLogo, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/riboSeqR.html\n",
      "\n",
      "Name: scmeth\n",
      "Description:\n",
      "Functions to analyze methylation data can be found here. Some functions are relevant for single cell methylation data but most other functions can be used for any methylation data. Highlight of this workflow is the comprehensive quality control report.\n",
      "Version: 1.18.0\n",
      "Publication date: None\n",
      "Mantainer: Divy Kangeyan <divyswar01@g.harvard.edu>\n",
      "Authors:\n",
      "Author: Divy Kangeyan <divyswar01 at g.harvard.edu>\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - knitr, type: imports\n",
      "  - rmarkdown, type: imports\n",
      "  - bsseq, type: imports\n",
      "  - AnnotationHub, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - BSgenome, type: imports\n",
      "  - DelayedArray, type: imports\n",
      "  - annotatr, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - DT, type: imports\n",
      "  - HDF5Array, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/scmeth.html\n",
      "\n",
      "Name: TargetSearch\n",
      "Description:\n",
      "This packages provides a flexible, fast and accurate method for targeted pre-processing of GC-MS data. The user provides a (often very large) set of GC chromatograms and a metabolite library of targets. The package will automatically search those targets in the chromatograms resulting in a data matrix that can be used for further data analysis.\n",
      "Version: 2.0.0\n",
      "Publication date: None\n",
      "Mantainer: Alvaro Cuadros-Inostroza <acuadros+bioc@gmail.com>\n",
      "Authors:\n",
      "Author: Alvaro Cuadros-Inostroza [aut, cre], Jan Lisec [aut], Henning Redestig [aut], Matt Hannah [aut]\n",
      "Dependencies:\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - methods, type: imports\n",
      "  - ncdf4, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - assertthat, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/TargetSearch.html\n",
      "\n",
      "Name: AffyRNADegradation\n",
      "Description:\n",
      "The package helps with the assessment and correction of RNA degradation effects in Affymetrix 3' expression arrays. The parameter d gives a robust and accurate measure of RNA integrity. The correction removes the probe positional bias, and thus improves comparability of samples that are affected by RNA degradation.\n",
      "Version: 1.44.0\n",
      "Publication date: None\n",
      "Mantainer: Mario Fasold <fasold@izbi.uni-leipzig.de>\n",
      "Authors:\n",
      "Author: Mario Fasold\n",
      "Dependencies:\n",
      "  - R (>= 2.9.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - affy, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/AffyRNADegradation.html\n",
      "\n",
      "Name: aggregateBioVar\n",
      "Description:\n",
      "For single cell RNA-seq data collected from more than one subject (e.g. biological sample or technical replicates), this package contains tools to summarize single cell gene expression profiles at the level of subject. A SingleCellExperiment object is taken as input and converted to a list of SummarizedExperiment objects, where each list element corresponds to an assigned cell type. The SummarizedExperiment objects contain aggregate gene-by-subject count matrices and inter-subject column metadata for individual subjects that can be processed using downstream bulk RNA-seq tools.\n",
      "Version: 1.8.0\n",
      "Publication date: None\n",
      "Mantainer: Jason Ratcliff <jason-ratcliff@uiowa.edu>\n",
      "Authors:\n",
      "Author: Jason Ratcliff [aut, cre] , Andrew Thurman [aut], Michael Chimenti [ctb], Alejandro Pezzulo [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - stats, type: imports\n",
      "  - methods, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - SingleCellExperiment, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - tibble, type: imports\n",
      "  - rlang, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/aggregateBioVar.html\n",
      "\n",
      "Name: ANF\n",
      "Description:\n",
      "This package is used for complex patient clustering by integrating multi-omic data through affinity network fusion.\n",
      "Version: 1.20.0\n",
      "Publication date: None\n",
      "Mantainer: Tianle Ma <tianlema@buffalo.edu>\n",
      "Authors:\n",
      "Author: Tianle Ma, Aidong Zhang\n",
      "Dependencies:\n",
      "  - igraph, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - survival, type: imports\n",
      "  - MASS, type: imports\n",
      "  - stats, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ANF.html\n",
      "\n",
      "Name: apComplex\n",
      "Description:\n",
      "Functions to estimate a bipartite graph of protein complex membership using AP-MS data.\n",
      "Version: 2.64.0\n",
      "Publication date: None\n",
      "Mantainer: Denise Scholtens <dscholtens@northwestern.edu>\n",
      "Authors:\n",
      "Author: Denise Scholtens <dscholtens at northwestern.edu>\n",
      "Dependencies:\n",
      "  - R (>= 2.10) type: depends\n",
      "  - graph, type: depends\n",
      "  - RBGL, type: depends\n",
      "  - Rgraphviz, type: imports\n",
      "  - stats, type: imports\n",
      "  - org.Sc.sgd.db, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/apComplex.html\n",
      "\n",
      "Name: basecallQC\n",
      "Description:\n",
      "The basecallQC package provides tools to work with Illumina bcl2Fastq (versions >= 2.1.7) software.Prior to basecalling and demultiplexing using the bcl2Fastq software, basecallQC functions allow the user to update Illumina sample sheets from versions <= 1.8.9 to >= 2.1.7 standards, clean sample sheets of common problems such as invalid sample names and IDs, create read and index basemasks and the bcl2Fastq command. Following the generation of basecalled and demultiplexed data, the basecallQC packages allows the user to generate HTML tables, plots and a self contained report of summary metrics from Illumina XML output files.\n",
      "Version: 1.22.0\n",
      "Publication date: None\n",
      "Mantainer: Thomas Carroll <tc.infomatics@gmail.com>\n",
      "Authors:\n",
      "Author: Thomas Carroll and Marian Dore\n",
      "Dependencies:\n",
      "  - R (>= 3.4) type: depends\n",
      "  - stats, type: depends\n",
      "  - utils, type: depends\n",
      "  - methods, type: depends\n",
      "  - rmarkdown, type: depends\n",
      "  - knitr, type: depends\n",
      "  - prettydoc, type: depends\n",
      "  - yaml, type: depends\n",
      "  - ggplot2, type: imports\n",
      "  - stringr, type: imports\n",
      "  - XML, type: imports\n",
      "  - raster, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - data.table, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - DT, type: imports\n",
      "  - lazyeval, type: imports\n",
      "  - ShortRead, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 3)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/basecallQC.html\n",
      "\n",
      "Name: bgx\n",
      "Description:\n",
      "Bayesian integrated analysis of Affymetrix GeneChips\n",
      "Version: 1.64.0\n",
      "Publication date: None\n",
      "Mantainer: Ernest Turro <et341@cam.ac.uk>\n",
      "Authors:\n",
      "Author: Ernest Turro, Graeme Ambler, Anne-Mette K Hein\n",
      "Dependencies:\n",
      "  - R (>= 2.0.1) type: depends\n",
      "  - Biobase, type: depends\n",
      "  - affy, type: depends\n",
      "  - gcrma, type: depends\n",
      "  - Rcpp (>= 0.11.0) type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/bgx.html\n",
      "\n",
      "Name: BPRMeth\n",
      "Description:\n",
      "The BPRMeth package is a probabilistic method to quantify explicit features of methylation profiles, in a way that would make it easier to formally use such profiles in downstream modelling efforts, such as predicting gene expression levels or clustering genomic regions or cells according to their methylation profiles.\n",
      "Version: 1.24.2\n",
      "Publication date: None\n",
      "Mantainer: Chantriolnt-Andreas Kapourani <kapouranis.andreas@gmail.com>\n",
      "Authors:\n",
      "Author: Chantriolnt-Andreas Kapourani [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - assertthat, type: imports\n",
      "  - methods, type: imports\n",
      "  - MASS, type: imports\n",
      "  - doParallel, type: imports\n",
      "  - parallel, type: imports\n",
      "  - e1071, type: imports\n",
      "  - earth, type: imports\n",
      "  - foreach, type: imports\n",
      "  - randomForest, type: imports\n",
      "  - stats, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - data.table, type: imports\n",
      "  - graphics, type: imports\n",
      "  - truncnorm, type: imports\n",
      "  - mvtnorm, type: imports\n",
      "  - Rcpp (>= 0.12.14) type: imports\n",
      "  - matrixcalc, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - kernlab, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - cowplot, type: imports\n",
      "  - BiocStyle, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3 | file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/BPRMeth.html\n",
      "\n",
      "Name: canceR\n",
      "Description:\n",
      "The package is user friendly interface based on the cgdsr and other modeling packages to explore, compare, and analyse all available Cancer Data (Clinical data, Gene Mutation, Gene Methylation, Gene Expression, Protein Phosphorylation, Copy Number Alteration) hosted by the Computational Biology Center at Memorial-Sloan-Kettering Cancer Center (MSKCC).\n",
      "Version: 1.32.0\n",
      "Publication date: None\n",
      "Mantainer: Karim Mezhoud <kmezhoud@gmail.com>\n",
      "Authors:\n",
      "Author: Karim Mezhoud. Nuclear Safety & Security Department. Nuclear Science Center of Tunisia.\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - tcltk, type: depends\n",
      "  - GSEABase, type: imports\n",
      "  - tkrplot, type: imports\n",
      "  - geNetClassifier, type: imports\n",
      "  - RUnit, type: imports\n",
      "  - Formula, type: imports\n",
      "  - rpart, type: imports\n",
      "  - survival, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - phenoTest, type: imports\n",
      "  - circlize, type: imports\n",
      "  - plyr, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - R.oo, type: imports\n",
      "  - R.methodsS3, type: imports\n",
      "  - httr, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/canceR.html\n",
      "\n",
      "Name: Clonality\n",
      "Description:\n",
      "Statistical tests for clonality versus independence of tumors from the same patient based on their LOH or genomewide copy number profiles\n",
      "Version: 1.46.0\n",
      "Publication date: None\n",
      "Mantainer: Irina Ostrovnaya <ostrovni@mskcc.org>\n",
      "Authors:\n",
      "Author: Irina Ostrovnaya\n",
      "Dependencies:\n",
      "  - R (>= 2.12.2) type: depends\n",
      "  - DNAcopy, type: depends\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/Clonality.html\n",
      "\n",
      "Name: clst\n",
      "Description:\n",
      "Package for modified nearest-neighbor classification based on calculation of a similarity threshold distinguishing within-group from between-group comparisons.\n",
      "Version: 1.46.0\n",
      "Publication date: None\n",
      "Mantainer: Noah Hoffman <ngh2@uw.edu>\n",
      "Authors:\n",
      "Author: Noah Hoffman\n",
      "Dependencies:\n",
      "  - R (>= 2.10) type: depends\n",
      "  - ROC, type: imports\n",
      "  - lattice, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/clst.html\n",
      "\n",
      "Name: CNVPanelizer\n",
      "Description:\n",
      "A method that allows for the use of a collection of non-matched normal tissue samples. Our approach uses a non-parametric bootstrap subsampling of the available reference samples to estimate the distribution of read counts from targeted sequencing. As inspired by random forest, this is combined with a procedure that subsamples the amplicons associated with each of the targeted genes. The obtained information allows us to reliably classify the copy number aberrations on the gene level.\n",
      "Version: 1.30.0\n",
      "Publication date: None\n",
      "Mantainer: Thomas Wolf <thomas_wolf71@gmx.de>\n",
      "Authors:\n",
      "Author: Cristiano Oliveira [aut], Thomas Wolf [aut, cre], Albrecht Stenzinger [ctb], Volker Endris [ctb], Nicole Pfarr [ctb], Benedikt Brors [ths], Wilko Weichert [ths]\n",
      "Dependencies:\n",
      "  - R (>= 3.2.0) type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - BiocGenerics, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - NOISeq, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - exomeCopy, type: imports\n",
      "  - foreach, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - plyr, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - gplots, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - stringr, type: imports\n",
      "  - testthat, type: imports\n",
      "  - graphics, type: imports\n",
      "  - methods, type: imports\n",
      "  - shiny, type: imports\n",
      "  - shinyFiles, type: imports\n",
      "  - shinyjs, type: imports\n",
      "  - grid, type: imports\n",
      "  - openxlsx, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/CNVPanelizer.html\n",
      "\n",
      "Name: compEpiTools\n",
      "Description:\n",
      "Tools for computational epigenomics developed for the analysis, integration and simultaneous visualization of various (epi)genomics data types across multiple genomic regions in multiple samples.\n",
      "Version: 1.32.0\n",
      "Publication date: None\n",
      "Mantainer: Kamal Kishore <kamal.fartiyal84@gmail.com>\n",
      "Authors:\n",
      "Author: Mattia Pelizzola [aut], Kamal Kishore [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - topGO, type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - AnnotationDbi, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - parallel, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - gplots, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - GenomicFeatures, type: imports\n",
      "  - XVector, type: imports\n",
      "  - methylPipe, type: imports\n",
      "  - GO.db, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/compEpiTools.html\n",
      "\n",
      "Name: doppelgangR\n",
      "Description:\n",
      "The main function is doppelgangR(), which takes as minimal input a list of ExpressionSet object, and searches all list pairs for duplicated samples. The search is based on the genomic data (exprs(eset)), phenotype/clinical data (pData(eset)), and \"smoking guns\" - supposedly unique identifiers found in pData(eset).\n",
      "Version: 1.26.0\n",
      "Publication date: None\n",
      "Mantainer: Levi Waldron <lwaldron.research@gmail.com>\n",
      "Authors:\n",
      "Author: Levi Waldron [aut, cre], Markus Reister [aut, ctb], Marcel Ramos [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - Biobase, type: depends\n",
      "  - BiocParallel, type: depends\n",
      "  - sva, type: imports\n",
      "  - impute, type: imports\n",
      "  - digest, type: imports\n",
      "  - mnormt, type: imports\n",
      "  - methods, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>=2.0)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/doppelgangR.html\n",
      "\n",
      "Name: GeneExpressionSignature\n",
      "Description:\n",
      "This package gives the implementations of the gene expression signature and its distance to each. Gene expression signature is represented as a list of genes whose expression is correlated with a biological state of interest. And its distance is defined using a nonparametric, rank-based pattern-matching strategy based on the Kolmogorov-Smirnov statistic. Gene expression signature and its distance can be used to detect similarities among the signatures of drugs, diseases, and biological states of interest.\n",
      "Version: 1.44.0\n",
      "Publication date: None\n",
      "Mantainer: Yang Cao <yiluheihei@gmail.com>\n",
      "Authors:\n",
      "Author: Yang Cao [aut, cre], Fei Li [ctb], Lu Han [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - Biobase, type: imports\n",
      "  - stats, type: imports\n",
      "  - methods, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/GeneExpressionSignature.html\n",
      "\n",
      "Name: gscreend\n",
      "Description:\n",
      "Package for the analysis of pooled genetic screens (e.g. CRISPR-KO). The analysis of such screens is based on the comparison of gRNA abundances before and after a cell proliferation phase. The gscreend packages takes gRNA counts as input and allows detection of genes whose knockout decreases or increases cell proliferation.\n",
      "Version: 1.12.0\n",
      "Publication date: None\n",
      "Mantainer: Katharina Imkeller <k.imkeller@dkfz.de>\n",
      "Authors:\n",
      "Author: Katharina Imkeller [cre, aut], Wolfgang Huber [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.6) type: depends\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - nloptr, type: imports\n",
      "  - fGarch, type: imports\n",
      "  - methods, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - graphics, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/gscreend.html\n",
      "\n",
      "Name: HGC\n",
      "Description:\n",
      "HGC (short for Hierarchical Graph-based Clustering) is an R package for conducting hierarchical clustering on large-scale single-cell RNA-seq (scRNA-seq) data. The key idea is to construct a dendrogram of cells on their shared nearest neighbor (SNN) graph. HGC provides functions for building graphs and for conducting hierarchical clustering on the graph. The users with old R version could visit https://github.com/XuegongLab/HGC/tree/HGC4oldRVersion to get HGC package built for R 3.6.\n",
      "Version: 1.6.0\n",
      "Publication date: None\n",
      "Mantainer: XGlab <xglab@mail.tsinghua.edu.cn>\n",
      "Authors:\n",
      "Author: Zou Ziheng [aut], Hua Kui [aut], XGlab [cre, cph]\n",
      "Dependencies:\n",
      "  - R (>= 4.1.0) type: depends\n",
      "  - Rcpp (>= 1.0.0) type: imports\n",
      "  - RcppEigen (>= 0.3.2.0) type: imports\n",
      "  - Matrix, type: imports\n",
      "  - RANN, type: imports\n",
      "  - ape, type: imports\n",
      "  - dendextend, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - mclust, type: imports\n",
      "  - patchwork, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/HGC.html\n",
      "\n",
      "Name: IsoCorrectoR\n",
      "Description:\n",
      "IsoCorrectoR performs the correction of mass spectrometry data from stable isotope labeling/tracing metabolomics experiments with regard to natural isotope abundance and tracer impurity. Data from both MS and MS/MS measurements can be corrected (with any tracer isotope: 13C, 15N, 18O...), as well as ultra-high resolution MS data from multiple-tracer experiments (e.g. 13C and 15N used simultaneously). See the Bioconductor package IsoCorrectoRGUI for a graphical user interface to IsoCorrectoR. NOTE: With R version 4.0.0, writing correction results to Excel files may currently not work on Windows. However, writing results to csv works as before.\n",
      "Version: 1.16.0\n",
      "Publication date: None\n",
      "Mantainer: Christian Kohler <christian.kohler@ur.de>\n",
      "Authors:\n",
      "Author: Christian Kohler [cre, aut], Paul Heinrich [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.5) type: depends\n",
      "  - dplyr, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - methods, type: imports\n",
      "  - quadprog, type: imports\n",
      "  - readr, type: imports\n",
      "  - readxl, type: imports\n",
      "  - stringr, type: imports\n",
      "  - tibble, type: imports\n",
      "  - tools, type: imports\n",
      "  - utils, type: imports\n",
      "  - pracma, type: imports\n",
      "  - WriteXLS, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/IsoCorrectoR.html\n",
      "\n",
      "Name: LOBSTAHS\n",
      "Description:\n",
      "LOBSTAHS is a multifunction package for screening, annotation, and putative identification of mass spectral features in large, HPLC-MS lipid datasets. In silico data for a wide range of lipids, oxidized lipids, and oxylipins can be generated from user-supplied structural criteria with a database generation function. LOBSTAHS then applies these databases to assign putative compound identities to features in any high-mass accuracy dataset that has been processed using xcms and CAMERA. Users can then apply a series of orthogonal screening criteria based on adduct ion formation patterns, chromatographic retention time, and other properties, to evaluate and assign confidence scores to this list of preliminary assignments. During the screening routine, LOBSTAHS rejects assignments that do not meet the specified criteria, identifies potential isomers and isobars, and assigns a variety of annotation codes to assist the user in evaluating the accuracy of each assignment.\n",
      "Version: 1.24.0\n",
      "Publication date: None\n",
      "Mantainer: Henry Holm <hholm@whoi.edu>, Daniel Lowenstein <dlowenstein@whoi.edu>, James Collins <james.r.collins@aya.yale.edu>\n",
      "Authors:\n",
      "Author: James Collins [aut, cre], Helen Fredricks [aut], Bethanie Edwards [aut], Henry Holm [aut], Benjamin Van Mooy [aut], Daniel Lowenstein [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.4) type: depends\n",
      "  - xcms, type: depends\n",
      "  - CAMERA, type: depends\n",
      "  - methods, type: depends\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 3) + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/LOBSTAHS.html\n",
      "\n",
      "Name: Mergeomics\n",
      "Description:\n",
      "The Mergeomics pipeline serves as a flexible framework for integrating multidimensional omics-disease associations, functional genomics, canonical pathways and gene-gene interaction networks to generate mechanistic hypotheses. It includes two main parts, 1) Marker set enrichment analysis (MSEA); 2) Weighted Key Driver Analysis (wKDA).\n",
      "Version: 1.26.0\n",
      "Publication date: None\n",
      "Mantainer: Zeyneb Kurt <zeynebkurt@gmail.com>\n",
      "Authors:\n",
      "Author: Ville-Petteri Makinen, Le Shu, Yuqi Zhao, Zeyneb Kurt, Bin Zhang, Xia Yang\n",
      "Dependencies:\n",
      "  - R (>= 3.0.1) type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/Mergeomics.html\n",
      "\n",
      "Name: OSAT\n",
      "Description:\n",
      "A sizable genomics study such as microarray often involves the use of multiple batches (groups) of experiment due to practical complication. To minimize batch effects, a careful experiment design should ensure the even distribution of biological groups and confounding factors across batches. OSAT (Optimal Sample Assignment Tool) is developed to facilitate the allocation of collected samples to different batches. With minimum steps, it produces setup that optimizes the even distribution of samples in groups of biological interest into different batches, reducing the confounding or correlation between batches and the biological variables of interest. It can also optimize the even distribution of confounding factors across batches. Our tool can handle challenging instances where incomplete and unbalanced sample collections are involved as well as ideal balanced RCBD. OSAT provides a number of predefined layout for some of the most commonly used genomics platform. Related paper can be find at http://www.biomedcentral.com/1471-2164/13/689 .\n",
      "Version: 1.46.0\n",
      "Publication date: None\n",
      "Mantainer: Li Yan <li.yan@roswellpark.org>\n",
      "Authors:\n",
      "Author: Li Yan\n",
      "Dependencies:\n",
      "  - methods, type: depends\n",
      "  - stats, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/OSAT.html\n",
      "\n",
      "Name: PECA\n",
      "Description:\n",
      "Calculates Probe-level Expression Change Averages (PECA) to identify differential expression in Affymetrix gene expression microarray studies or in proteomic studies using peptide-level mesurements respectively.\n",
      "Version: 1.34.0\n",
      "Publication date: None\n",
      "Mantainer: Tomi Suomi <tomi.suomi@utu.fi>\n",
      "Authors:\n",
      "Author: Tomi Suomi, Jukka Hiissa, Laura L. Elo\n",
      "Dependencies:\n",
      "  - R (>= 3.3) type: depends\n",
      "  - ROTS, type: imports\n",
      "  - limma, type: imports\n",
      "  - affy, type: imports\n",
      "  - genefilter, type: imports\n",
      "  - preprocessCore, type: imports\n",
      "  - aroma.affymetrix, type: imports\n",
      "  - aroma.core, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/PECA.html\n",
      "\n",
      "Name: Pviz\n",
      "Description:\n",
      "Pviz adapts the Gviz package for protein sequences and data.\n",
      "Version: 1.32.0\n",
      "Publication date: None\n",
      "Mantainer: Renan Sauteraud <rsautera@fhcrc.org>\n",
      "Authors:\n",
      "Author: Renan Sauteraud, Mike Jiang, Raphael Gottardo\n",
      "Dependencies:\n",
      "  - R (>= 3.0.0) type: depends\n",
      "  - Gviz, type: depends\n",
      "  - biovizBase, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - data.table, type: imports\n",
      "  - methods, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/Pviz.html\n",
      "\n",
      "Name: SCANVIS\n",
      "Description:\n",
      "SCANVIS is a set of annotation-dependent tools for analyzing splice junctions and their read support as predetermined by an alignment tool of choice (for example, STAR aligner). SCANVIS assesses each junction's relative read support (RRS) by relating to the context of local split reads aligning to annotated transcripts. SCANVIS also annotates each splice junction by indicating whether the junction is supported by annotation or not, and if not, what type of junction it is (e.g. exon skipping, alternative 5' or 3' events, Novel Exons). Unannotated junctions are also futher annotated by indicating whether it induces a frame shift or not. SCANVIS includes a visualization function to generate static sashimi-style plots depicting relative read support and number of split reads using arc thickness and arc heights, making it easy for users to spot well-supported junctions. These plots also clearly delineate unannotated junctions from annotated ones using designated color schemes, and users can also highlight splice junctions of choice. Variants and/or a read profile are also incoroporated into the plot if the user supplies variants in bed format and/or the BAM file. One further feature of the visualization function is that users can submit multiple samples of a certain disease or cohort to generate a single plot - this occurs via a \"merge\" function wherein junction details over multiple samples are merged to generate a single sashimi plot, which is useful when contrasting cohorots (eg. disease vs control).\n",
      "Version: 1.12.0\n",
      "Publication date: None\n",
      "Mantainer: Phaedra Agius <pagius@nygenome.org>\n",
      "Authors:\n",
      "Author: Phaedra Agius <pagius at nygenome.org>\n",
      "Dependencies:\n",
      "  - R (>= 3.6) type: depends\n",
      "  - IRanges, type: imports\n",
      "  - plotrix, type: imports\n",
      "  - RCurl, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/SCANVIS.html\n",
      "\n",
      "Name: scTensor\n",
      "Description:\n",
      "The algorithm is based on the non-negative tucker decomposition (NTD2) of nnTensor.\n",
      "Version: 2.8.0\n",
      "Publication date: None\n",
      "Mantainer: Koki Tsuyuzaki <k.t.the-answer@hotmail.co.jp>\n",
      "Authors:\n",
      "Author: Koki Tsuyuzaki [aut, cre], Kozo Nishida [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.1.0) type: depends\n",
      "  - methods, type: imports\n",
      "  - RSQLite, type: imports\n",
      "  - igraph, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - plotly, type: imports\n",
      "  - reactome.db, type: imports\n",
      "  - AnnotationDbi, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - SingleCellExperiment, type: imports\n",
      "  - nnTensor (>= 1.1.5) type: imports\n",
      "  - ccTensor (>= 1.0.2) type: imports\n",
      "  - rTensor (>= 1.4.8) type: imports\n",
      "  - abind, type: imports\n",
      "  - plotrix, type: imports\n",
      "  - heatmaply, type: imports\n",
      "  - tagcloud, type: imports\n",
      "  - rmarkdown, type: imports\n",
      "  - BiocStyle, type: imports\n",
      "  - knitr, type: imports\n",
      "  - AnnotationHub, type: imports\n",
      "  - MeSHDbi, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - outliers, type: imports\n",
      "  - Category, type: imports\n",
      "  - meshr, type: imports\n",
      "  - GOstats, type: imports\n",
      "  - ReactomePA, type: imports\n",
      "  - DOSE, type: imports\n",
      "  - crayon, type: imports\n",
      "  - checkmate, type: imports\n",
      "  - BiocManager, type: imports\n",
      "  - visNetwork, type: imports\n",
      "  - schex, type: imports\n",
      "  - ggplot2, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/scTensor.html\n",
      "\n",
      "Name: seqbias\n",
      "Description:\n",
      "This package implements a model of per-position sequencing bias in high-throughput sequencing data using a simple Bayesian network, the structure and parameters of which are trained on a set of aligned reads and a reference genome sequence.\n",
      "Version: 1.46.0\n",
      "Publication date: None\n",
      "Mantainer: Daniel Jones <djones3@fredhutch.org>\n",
      "Authors:\n",
      "Author: Daniel Jones <djones3 at fredhutch.org>\n",
      "Dependencies:\n",
      "  - R (>= 3.0.2) type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - Biostrings, type: depends\n",
      "  - methods, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/seqbias.html\n",
      "\n",
      "Name: snapcount\n",
      "Description:\n",
      "snapcount is a client interface to the Snaptron webservices which support querying by gene name or genomic region. Results include raw expression counts derived from alignment of RNA-seq samples and/or various summarized measures of expression across one or more regions/genes per-sample (e.g. percent spliced in).\n",
      "Version: 1.10.0\n",
      "Publication date: None\n",
      "Mantainer: Rone Charles <rcharle8@jh.edu>\n",
      "Authors:\n",
      "Author: Rone Charles [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.0.0) type: depends\n",
      "  - R6, type: imports\n",
      "  - httr, type: imports\n",
      "  - rlang, type: imports\n",
      "  - purrr, type: imports\n",
      "  - jsonlite, type: imports\n",
      "  - assertthat, type: imports\n",
      "  - data.table, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - methods, type: imports\n",
      "  - stringr, type: imports\n",
      "  - stats, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/snapcount.html\n",
      "\n",
      "Name: systemPipeShiny\n",
      "Description:\n",
      "systemPipeShiny (SPS) extends the widely used systemPipeR (SPR) workflow environment with a versatile graphical user interface provided by a Shiny App. This allows non-R users, such as experimentalists, to run many systemPipeRâs workflow designs, control, and visualization functionalities interactively without requiring knowledge of R. Most importantly, SPS has been designed as a general purpose framework for interacting with other R packages in an intuitive manner. Like most Shiny Apps, SPS can be used on both local computers as well as centralized server-based deployments that can be accessed remotely as a public web service for using SPRâs functionalities with community and/or private data. The framework can integrate many core packages from the R/Bioconductor ecosystem. Examples of SPSâ current functionalities include: (a) interactive creation of experimental designs and metadata using an easy to use tabular editor or file uploader; (b) visualization of workflow topologies combined with auto-generation of R Markdown preview for interactively designed workflows; (d) access to a wide range of data processing routines; (e) and an extendable set of visualization functionalities. Complex visual results can be managed on a 'Canvas Workbenchâ allowing users to organize and to compare plots in an efficient manner combined with a session snapshot feature to continue work at a later time. The present suite of pre-configured visualization examples. The modular design of SPR makes it easy to design custom functions without any knowledge of Shiny, as well as extending the environment in the future with contributions from the community.\n",
      "Version: 1.8.1\n",
      "Publication date: None\n",
      "Mantainer: Le Zhang <le.zhang001@email.ucr.edu>\n",
      "Authors:\n",
      "Author: Le Zhang [aut, cre], Daniela Cassol [aut], Ponmathi Ramasamy [aut], Jianhai Zhang [aut], Gordon Mosher [aut], Thomas Girke [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.0.0) type: depends\n",
      "  - shiny (>= 1.6.0) type: depends\n",
      "  - spsUtil (>= 0.2.2) type: depends\n",
      "  - spsComps (>= 0.3.2) type: depends\n",
      "  - drawer (>= 0.2) type: depends\n",
      "  - DT, type: imports\n",
      "  - assertthat, type: imports\n",
      "  - bsplus, type: imports\n",
      "  - crayon, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - htmltools, type: imports\n",
      "  - glue, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - methods, type: imports\n",
      "  - plotly, type: imports\n",
      "  - rlang, type: imports\n",
      "  - rstudioapi, type: imports\n",
      "  - shinyAce, type: imports\n",
      "  - shinyFiles, type: imports\n",
      "  - shinyWidgets, type: imports\n",
      "  - shinydashboard, type: imports\n",
      "  - shinydashboardPlus (>= 2.0.0) type: imports\n",
      "  - shinyjqui, type: imports\n",
      "  - shinyjs, type: imports\n",
      "  - shinytoastr, type: imports\n",
      "  - stringr, type: imports\n",
      "  - stats, type: imports\n",
      "  - styler, type: imports\n",
      "  - tibble, type: imports\n",
      "  - utils, type: imports\n",
      "  - vroom (>= 1.3.1) type: imports\n",
      "  - yaml, type: imports\n",
      "  - R6, type: imports\n",
      "  - RSQLite, type: imports\n",
      "  - openssl, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 3)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/systemPipeShiny.html\n",
      "\n",
      "Name: waddR\n",
      "Description:\n",
      "The package offers statistical tests based on the 2-Wasserstein distance for detecting and characterizing differences between two distributions given in the form of samples. Functions for calculating the 2-Wasserstein distance and testing for differential distributions are provided, as well as a specifically tailored test for differential expression in single-cell RNA sequencing data.\n",
      "Version: 1.12.0\n",
      "Publication date: None\n",
      "Mantainer: Julian Flesch <julianflesch@gmail.com>\n",
      "Authors:\n",
      "Author: Roman Schefzik [aut], Julian Flesch [cre]\n",
      "Dependencies:\n",
      "  - R (>= 3.6.0) type: depends\n",
      "  - Rcpp (>= 1.0.1) type: imports\n",
      "  - arm (>= 1.10-1) type: imports\n",
      "  - eva, type: imports\n",
      "  - BiocFileCache, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - SingleCellExperiment, type: imports\n",
      "  - parallel, type: imports\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/waddR.html\n",
      "\n",
      "Name: AMARETTO\n",
      "Description:\n",
      "Integrating an increasing number of available multi-omics cancer data remains one of the main challenges to improve our understanding of cancer. One of the main challenges is using multi-omics data for identifying novel cancer driver genes. We have developed an algorithm, called AMARETTO, that integrates copy number, DNA methylation and gene expression data to identify a set of driver genes by analyzing cancer samples and connects them to clusters of co-expressed genes, which we define as modules. We applied AMARETTO in a pancancer setting to identify cancer driver genes and their modules on multiple cancer sites. AMARETTO captures modules enriched in angiogenesis, cell cycle and EMT, and modules that accurately predict survival and molecular subtypes. This allows AMARETTO to identify novel cancer driver genes directing canonical cancer pathways.\n",
      "Version: 1.13.0\n",
      "Publication date: None\n",
      "Mantainer: Olivier Gevaert <olivier.gevaert@gmail.com>\n",
      "Authors:\n",
      "Author: Jayendra Shinde, Celine Everaert, Shaimaa Bakr, Mohsen Nabian, Jishu Xu, Vincent Carey, Nathalie Pochet and Olivier Gevaert\n",
      "Dependencies:\n",
      "  - R (>= 3.6) type: depends\n",
      "  - impute, type: depends\n",
      "  - doParallel, type: depends\n",
      "  - grDevices, type: depends\n",
      "  - dplyr, type: depends\n",
      "  - methods, type: depends\n",
      "  - ComplexHeatmap, type: depends\n",
      "  - callr (>= 3.0.0.9001) type: imports\n",
      "  - Matrix, type: imports\n",
      "  - Rcpp, type: imports\n",
      "  - BiocFileCache, type: imports\n",
      "  - DT, type: imports\n",
      "  - MultiAssayExperiment, type: imports\n",
      "  - circlize, type: imports\n",
      "  - curatedTCGAData, type: imports\n",
      "  - foreach, type: imports\n",
      "  - glmnet, type: imports\n",
      "  - httr, type: imports\n",
      "  - limma, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - readr, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - tibble, type: imports\n",
      "  - rmarkdown, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grid, type: imports\n",
      "  - parallel, type: imports\n",
      "  - stats, type: imports\n",
      "  - knitr, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - gridExtra, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Apache License (== 2.0) + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/AMARETTO.html\n",
      "\n",
      "Name: antiProfiles\n",
      "Description:\n",
      "Implements gene expression anti-profiles as described in Corrada Bravo et al., BMC Bioinformatics 2012, 13:272 doi:10.1186/1471-2105-13-272.\n",
      "Version: 1.38.0\n",
      "Publication date: None\n",
      "Mantainer: Hector Corrada Bravo <hcorrada@gmail.com>\n",
      "Authors:\n",
      "Author: Hector Corrada Bravo, Rafael A. Irizarry and Jeffrey T. Leek\n",
      "Dependencies:\n",
      "  - R (>= 3.0) type: depends\n",
      "  - matrixStats (>= 0.50.0) type: depends\n",
      "  - methods (>= 2.14) type: depends\n",
      "  - locfit (>= 1.5) type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/antiProfiles.html\n",
      "\n",
      "Name: ASEB\n",
      "Description:\n",
      "ASEB is an R package to predict lysine sites that can be acetylated by a specific KAT-family.\n",
      "Version: 1.42.0\n",
      "Publication date: None\n",
      "Mantainer: Likun Wang <wanglk@hsc.pku.edu.cn>\n",
      "Authors:\n",
      "Author: Likun Wang <wanglk at hsc.pku.edu.cn> and Tingting Li <litt at hsc.pku.edu.cn>.\n",
      "Dependencies:\n",
      "  - R (>= 2.8.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - graphics, type: imports\n",
      "  - methods, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 3)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ASEB.html\n",
      "\n",
      "Name: BaalChIP\n",
      "Description:\n",
      "The package offers functions to process multiple ChIP-seq BAM files and detect allele-specific events. Computes allele counts at individual variants (SNPs/SNVs), implements extensive QC steps to remove problematic variants, and utilizes a bayesian framework to identify statistically significant allele- specific events. BaalChIP is able to account for copy number differences between the two alleles, a known phenotypical feature of cancer samples.\n",
      "Version: 1.24.0\n",
      "Publication date: None\n",
      "Mantainer: Ines de Santiago <inesdesantiago@gmail.com>\n",
      "Authors:\n",
      "Author: Ines de Santiago, Wei Liu, Ke Yuan, Martin O'Reilly, Chandra SR Chilamakuri, Bruce Ponder, Kerstin Meyer, Florian Markowetz\n",
      "Dependencies:\n",
      "  - R (>= 3.3.1) type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - IRanges, type: depends\n",
      "  - Rsamtools, type: depends\n",
      "  - GenomicAlignments, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - doParallel, type: imports\n",
      "  - parallel, type: imports\n",
      "  - doBy, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - scales, type: imports\n",
      "  - coda, type: imports\n",
      "  - foreach, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - methods, type: imports\n",
      "  - utils, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/BaalChIP.html\n",
      "\n",
      "Name: Basic4Cseq\n",
      "Description:\n",
      "Basic4Cseq is an R/Bioconductor package for basic filtering, analysis and subsequent visualization of 4C-seq data. Virtual fragment libraries can be created for any BSGenome package, and filter functions for both reads and fragments and basic quality controls are included. Fragment data in the vicinity of the experiment's viewpoint can be visualized as a coverage plot based on a running median approach and a multi-scale contact profile.\n",
      "Version: 1.34.0\n",
      "Publication date: None\n",
      "Mantainer: Carolin Walter <carolin.walter@uni-muenster.de>\n",
      "Authors:\n",
      "Author: Carolin Walter\n",
      "Dependencies:\n",
      "  - R (>= 3.4) type: depends\n",
      "  - Biostrings, type: depends\n",
      "  - GenomicAlignments, type: depends\n",
      "  - caTools, type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - grDevices, type: depends\n",
      "  - graphics, type: depends\n",
      "  - stats, type: depends\n",
      "  - utils, type: depends\n",
      "  - methods, type: imports\n",
      "  - RCircos, type: imports\n",
      "  - BSgenome.Ecoli.NCBI.20080805, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/Basic4Cseq.html\n",
      "\n",
      "Name: biodbKegg\n",
      "Description:\n",
      "The biodbKegg library is an extension of the biodb framework package that provides access to the KEGG databases Compound, Enzyme, Genes, Module, Orthology and Reaction. It allows to retrieve entries by their accession numbers. Web services like \"find\", \"list\" and \"findExactMass\" are also available. Some functions for navigating along the pathways have also been implemented.\n",
      "Version: 1.4.0\n",
      "Publication date: None\n",
      "Mantainer: Pierrick Roger <pierrick.roger@cea.fr>\n",
      "Authors:\n",
      "Author: Pierrick Roger [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - R6, type: imports\n",
      "  - biodb, type: imports\n",
      "  - chk, type: imports\n",
      "  - lifecycle, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "AGPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/biodbKegg.html\n",
      "\n",
      "Name: casper\n",
      "Description:\n",
      "Infer alternative splicing from paired-end RNA-seq data. The model is based on counting paths across exons, rather than pairwise exon connections, and estimates the fragment size and start distributions non-parametrically, which improves estimation precision.\n",
      "Version: 2.32.0\n",
      "Publication date: None\n",
      "Mantainer: David Rossell <rosselldavid@gmail.com>\n",
      "Authors:\n",
      "Author: David Rossell, Camille Stephan-Otto, Manuel Kroiss, Miranda Stobbe, Victor Pena\n",
      "Dependencies:\n",
      "  - R (>= 3.6.0) type: depends\n",
      "  - Biobase, type: depends\n",
      "  - IRanges, type: depends\n",
      "  - methods, type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - BiocGenerics, type: imports\n",
      "  - coda, type: imports\n",
      "  - EBarrays, type: imports\n",
      "  - gaga, type: imports\n",
      "  - gtools, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - GenomicFeatures, type: imports\n",
      "  - limma, type: imports\n",
      "  - mgcv, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - sqldf, type: imports\n",
      "  - survival, type: imports\n",
      "  - VGAM, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>=2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/casper.html\n",
      "\n",
      "Name: CellaRepertorium\n",
      "Description:\n",
      "Methods to cluster and analyze high-throughput single cell immune cell repertoires, especially from the 10X Genomics VDJ solution. Contains an R interface to CD-HIT (Li and Godzik 2006). Methods to visualize and analyze paired heavy-light chain data. Tests for specific expansion, as well as omnibus oligoclonality under hypergeometric models.\n",
      "Version: 1.8.0\n",
      "Publication date: None\n",
      "Mantainer: Andrew McDavid <Andrew_McDavid@urmc.rochester.edu>\n",
      "Authors:\n",
      "Author: Andrew McDavid [aut, cre], Yu Gu [aut], Erik VonKaenel [aut], Aaron Wagner [aut], Thomas Lin Pedersen [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - dplyr, type: imports\n",
      "  - tibble, type: imports\n",
      "  - stringr, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - Rcpp, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - methods, type: imports\n",
      "  - rlang (>= 0.3) type: imports\n",
      "  - purrr, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - forcats, type: imports\n",
      "  - progress, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - generics, type: imports\n",
      "  - glue, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/CellaRepertorium.html\n",
      "\n",
      "Name: ChromSCape\n",
      "Description:\n",
      "ChromSCape - Chromatin landscape profiling for Single Cells - is a ready-to-launch user-friendly Shiny Application for the analysis of single-cell epigenomics datasets (scChIP-seq, scATAC-seq, scCUT&Tag, ...) from aligned data to differential analysis & gene set enrichment analysis. It is highly interactive, enables users to save their analysis and covers a wide range of analytical steps: QC, preprocessing, filtering, batch correction, dimensionality reduction, vizualisation, clustering, differential analysis and gene set analysis.\n",
      "Version: 1.8.0\n",
      "Publication date: None\n",
      "Mantainer: Pacome Prompsy <pacome.prompsy@curie.fr>\n",
      "Authors:\n",
      "Author: Pacome Prompsy [aut, cre] , Celine Vallot [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - shiny, type: imports\n",
      "  - colourpicker, type: imports\n",
      "  - shinyjs, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - shinyFiles, type: imports\n",
      "  - shinyhelper, type: imports\n",
      "  - shinyWidgets, type: imports\n",
      "  - shinydashboardPlus, type: imports\n",
      "  - shinycssloaders, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - plotly, type: imports\n",
      "  - shinydashboard, type: imports\n",
      "  - colorRamps, type: imports\n",
      "  - kableExtra, type: imports\n",
      "  - viridis, type: imports\n",
      "  - batchelor, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - parallel, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - ggrepel, type: imports\n",
      "  - gggenes, type: imports\n",
      "  - gridExtra, type: imports\n",
      "  - qualV, type: imports\n",
      "  - stringdist, type: imports\n",
      "  - stringr, type: imports\n",
      "  - fs, type: imports\n",
      "  - qs, type: imports\n",
      "  - DT, type: imports\n",
      "  - scran, type: imports\n",
      "  - scater, type: imports\n",
      "  - ConsensusClusterPlus, type: imports\n",
      "  - Rtsne, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - irlba, type: imports\n",
      "  - rlist, type: imports\n",
      "  - umap, type: imports\n",
      "  - tibble, type: imports\n",
      "  - methods, type: imports\n",
      "  - jsonlite, type: imports\n",
      "  - edgeR, type: imports\n",
      "  - stats, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - utils, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - SingleCellExperiment, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - msigdbr, type: imports\n",
      "  - forcats, type: imports\n",
      "  - Rcpp, type: imports\n",
      "  - coop, type: imports\n",
      "  - matrixTests, type: imports\n",
      "  - DelayedArray, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ChromSCape.html\n",
      "\n",
      "Name: CNVfilteR\n",
      "Description:\n",
      "CNVfilteR identifies those CNVs that can be discarded by using the single nucleotide variant (SNV) calls that are usually obtained in common NGS pipelines.\n",
      "Version: 1.12.1\n",
      "Publication date: None\n",
      "Mantainer: Jose Marcos Moreno-Cabrera <jpuntomarcos@gmail.com>\n",
      "Authors:\n",
      "Author: Jose Marcos Moreno-Cabrera [aut, cre] , Bernat Gel [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - IRanges, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - pracma, type: imports\n",
      "  - regioneR, type: imports\n",
      "  - assertthat, type: imports\n",
      "  - karyoploteR, type: imports\n",
      "  - CopyNumberPlots, type: imports\n",
      "  - graphics, type: imports\n",
      "  - utils, type: imports\n",
      "  - VariantAnnotation, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - methods, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/CNVfilteR.html\n",
      "\n",
      "Name: CORREP\n",
      "Description:\n",
      "Multivariate correlation estimation and statistical inference. See package vignette.\n",
      "Version: 1.64.0\n",
      "Publication date: None\n",
      "Mantainer: Dongxiao Zhu <doz@stowers-institute.org>\n",
      "Authors:\n",
      "Author: Dongxiao Zhu and Youjuan Li\n",
      "Dependencies:\n",
      "  - e1071, type: imports\n",
      "  - stats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/CORREP.html\n",
      "\n",
      "Name: cTRAP\n",
      "Description:\n",
      "Compare differential gene expression results with those from known cellular perturbations (such as gene knock-down, overexpression or small molecules) derived from the Connectivity Map. Such analyses allow not only to infer the molecular causes of the observed difference in gene expression but also to identify small molecules that could drive or revert specific transcriptomic alterations.\n",
      "Version: 1.16.0\n",
      "Publication date: None\n",
      "Mantainer: Nuno Saraiva-Agostinho <nunodanielagostinho@gmail.com>\n",
      "Authors:\n",
      "Author: Bernardo P. de Almeida [aut], Nuno Saraiva-Agostinho [aut, cre], Nuno L. Barbosa-Morais [aut, led]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - AnnotationDbi, type: imports\n",
      "  - AnnotationHub, type: imports\n",
      "  - binr, type: imports\n",
      "  - cowplot, type: imports\n",
      "  - data.table, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - DT, type: imports\n",
      "  - fastmatch, type: imports\n",
      "  - fgsea, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - ggrepel, type: imports\n",
      "  - graphics, type: imports\n",
      "  - highcharter, type: imports\n",
      "  - htmltools, type: imports\n",
      "  - httr, type: imports\n",
      "  - limma, type: imports\n",
      "  - methods, type: imports\n",
      "  - parallel, type: imports\n",
      "  - pbapply, type: imports\n",
      "  - purrr, type: imports\n",
      "  - qs, type: imports\n",
      "  - R.utils, type: imports\n",
      "  - readxl, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - rhdf5, type: imports\n",
      "  - rlang, type: imports\n",
      "  - scales, type: imports\n",
      "  - shiny (>= 1.7.0) type: imports\n",
      "  - shinycssloaders, type: imports\n",
      "  - stats, type: imports\n",
      "  - tibble, type: imports\n",
      "  - tools, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/cTRAP.html\n",
      "\n",
      "Name: CytoGLMM\n",
      "Description:\n",
      "The CytoGLMM R package implements two multiple regression strategies: A bootstrapped generalized linear model (GLM) and a generalized linear mixed model (GLMM). Most current data analysis tools compare expressions across many computationally discovered cell types. CytoGLMM focuses on just one cell type. Our narrower field of application allows us to define a more specific statistical model with easier to control statistical guarantees. As a result, CytoGLMM finds differential proteins in flow and mass cytometry data while reducing biases arising from marker correlations and safeguarding against false discoveries induced by patient heterogeneity.\n",
      "Version: 1.6.0\n",
      "Publication date: None\n",
      "Mantainer: Christof Seiler <christof.seiler@maastrichtuniversity.nl>\n",
      "Authors:\n",
      "Author: Christof Seiler [aut, cre]\n",
      "Dependencies:\n",
      "  - stats, type: imports\n",
      "  - methods, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - cowplot, type: imports\n",
      "  - doParallel, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - factoextra, type: imports\n",
      "  - flexmix, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - mbest, type: imports\n",
      "  - pheatmap, type: imports\n",
      "  - speedglm, type: imports\n",
      "  - stringr, type: imports\n",
      "  - strucchange, type: imports\n",
      "  - tibble, type: imports\n",
      "  - ggrepel, type: imports\n",
      "  - MASS, type: imports\n",
      "  - logging, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - caret, type: imports\n",
      "  - rlang, type: imports\n",
      "  - grDevices, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/CytoGLMM.html\n",
      "\n",
      "Name: DeepBlueR\n",
      "Description:\n",
      "Accessing the DeepBlue Epigenetics Data Server through R.\n",
      "Version: 1.24.1\n",
      "Publication date: None\n",
      "Mantainer: Felipe Albrecht <felipe.albrecht@mpi-inf.mpg.de>, Markus List <markus.list@wzw.tum.de>, Quirin Manz <quirin.manz@tum.de>\n",
      "Authors:\n",
      "Author: Felipe Albrecht, Markus List\n",
      "Dependencies:\n",
      "  - R (>= 3.3) type: depends\n",
      "  - XML, type: depends\n",
      "  - RCurl, type: depends\n",
      "  - GenomicRanges, type: imports\n",
      "  - data.table, type: imports\n",
      "  - stringr, type: imports\n",
      "  - diffr, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - methods, type: imports\n",
      "  - rjson, type: imports\n",
      "  - utils, type: imports\n",
      "  - R.utils, type: imports\n",
      "  - foreach, type: imports\n",
      "  - withr, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - settings, type: imports\n",
      "  - filehash, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>=2.0)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/DeepBlueR.html\n",
      "\n",
      "Name: DelayedDataFrame\n",
      "Description:\n",
      "Based on the standard DataFrame metaphor, we are trying to implement the feature of delayed operation on the DelayedDataFrame, with a slot of lazyIndex, which saves the mapping indexes for each column of DelayedDataFrame. Methods like show, validity check, [/[[ subsetting, rbind/cbind are implemented for DelayedDataFrame to be operated around lazyIndex. The listData slot stays untouched until a realization call e.g., DataFrame constructor OR as.list() is invoked.\n",
      "Version: 1.14.0\n",
      "Publication date: None\n",
      "Mantainer: Qian Liu <Qian.Liu@roswellpark.org>\n",
      "Authors:\n",
      "Author: Qian Liu [aut, cre], HervÃ© PagÃ¨s [aut], Martin Morgan [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.6) type: depends\n",
      "  - S4Vectors, type: depends\n",
      "  - DelayedArray, type: depends\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/DelayedDataFrame.html\n",
      "\n",
      "Name: flowSpecs\n",
      "Description:\n",
      "This package is intended to fill the role of conventional cytometry pre-processing software, for spectral decomposition, transformation, visualization and cleanup, and to aid further downstream analyses, such as with DepecheR, by enabling transformation of flowFrames and flowSets to dataframes. Functions for flowCore-compliant automatic 1D-gating/filtering are in the pipe line. The package name has been chosen both as it will deal with spectral cytometry and as it will hopefully give the user a nice pair of spectacles through which to view their data.\n",
      "Version: 1.12.0\n",
      "Publication date: None\n",
      "Mantainer: Jakob Theorell <jakob.theorell@ki.se>\n",
      "Authors:\n",
      "Author: Jakob Theorell [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - ggplot2 (>= 3.1.0) type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - reshape2 (>= 1.4.3) type: imports\n",
      "  - flowCore, type: imports\n",
      "  - zoo (>= 1.8.6) type: imports\n",
      "  - stats (>= 3.6.0) type: imports\n",
      "  - methods (>= 3.6.0) type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/flowSpecs.html\n",
      "\n",
      "Name: GeneBreak\n",
      "Description:\n",
      "Recurrent breakpoint gene detection on copy number aberration profiles.\n",
      "Version: 1.28.0\n",
      "Publication date: None\n",
      "Mantainer: Evert van den Broek <vandenbroek.evert@gmail.com>\n",
      "Authors:\n",
      "Author: Evert van den Broek, Stef van Lieshout\n",
      "Dependencies:\n",
      "  - R (>= 3.2) type: depends\n",
      "  - QDNAseq, type: depends\n",
      "  - CGHcall, type: depends\n",
      "  - CGHbase, type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - graphics, type: imports\n",
      "  - methods, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/GeneBreak.html\n",
      "\n",
      "Name: GOTHiC\n",
      "Description:\n",
      "This is a Hi-C analysis package using a cumulative binomial test to detect interactions between distal genomic loci that have significantly more reads than expected by chance in Hi-C experiments. It takes mapped paired NGS reads as input and gives back the list of significant interactions for a given bin size in the genome.\n",
      "Version: 1.34.0\n",
      "Publication date: None\n",
      "Mantainer: Borbala Mifsud <b.mifsud@qmul.ac.uk>\n",
      "Authors:\n",
      "Author: Borbala Mifsud and Robert Sugar\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - Biostrings, type: depends\n",
      "  - BSgenome, type: depends\n",
      "  - data.table, type: depends\n",
      "  - BiocGenerics, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - ShortRead, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - BiocManager, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - utils, type: imports\n",
      "  - stats, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/GOTHiC.html\n",
      "\n",
      "Name: GraphAlignment\n",
      "Description:\n",
      "Graph alignment is an extension package for the R programming environment which provides functions for finding an alignment between two networks based on link and node similarity scores. (J. Berg and M. Laessig, \"Cross-species analysis of biological networks by Bayesian alignment\", PNAS 103 (29), 10967-10972 (2006))\n",
      "Version: 1.62.0\n",
      "Publication date: None\n",
      "Mantainer: Joern P. Meier <mail@ionflux.org>\n",
      "Authors:\n",
      "Author: Joern P. Meier <mail at ionflux.org>, Michal Kolar, Ville Mustonen, Michael Laessig, and Johannes Berg.\n",
      "Dependencies:\n",
      "None\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/GraphAlignment.html\n",
      "\n",
      "Name: GSALightning\n",
      "Description:\n",
      "GSALightning provides a fast implementation of permutation-based gene set analysis for two-sample problem. This package is particularly useful when testing simultaneously a large number of gene sets, or when a large number of permutations is necessary for more accurate p-values estimation.\n",
      "Version: 1.26.0\n",
      "Publication date: None\n",
      "Mantainer: Billy Heung Wing Chang <billyheungwing@gmail.com>\n",
      "Authors:\n",
      "Author: Billy Heung Wing Chang\n",
      "Dependencies:\n",
      "  - R (>= 3.3.0) type: depends\n",
      "  - Matrix, type: imports\n",
      "  - data.table, type: imports\n",
      "  - stats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>=2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/GSALightning.html\n",
      "\n",
      "Name: HiCBricks\n",
      "Description:\n",
      "HiCBricks is a library designed for handling large high-resolution Hi-C datasets. Over the years, the Hi-C field has experienced a rapid increase in the size and complexity of datasets. HiCBricks is meant to overcome the challenges related to the analysis of such large datasets within the R environment. HiCBricks offers user-friendly and efficient solutions for handling large high-resolution Hi-C datasets. The package provides an R/Bioconductor framework with the bricks to build more complex data analysis pipelines and algorithms. HiCBricks already incorporates example algorithms for calling domain boundaries and functions for high quality data visualization.\n",
      "Version: 1.16.0\n",
      "Publication date: None\n",
      "Mantainer: Koustav Pal <koustav.pal@ifom.eu>\n",
      "Authors:\n",
      "Author: Koustav Pal [aut, cre], Carmen Livi [ctb], Ilario Tagliaferri [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 3.6) type: depends\n",
      "  - utils, type: depends\n",
      "  - curl, type: depends\n",
      "  - rhdf5, type: depends\n",
      "  - R6, type: depends\n",
      "  - grid, type: depends\n",
      "  - ggplot2, type: imports\n",
      "  - viridis, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - scales, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - stringr, type: imports\n",
      "  - data.table, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - stats, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - digest, type: imports\n",
      "  - tibble, type: imports\n",
      "  - jsonlite, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - R.utils, type: imports\n",
      "  - readr, type: imports\n",
      "  - methods, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/HiCBricks.html\n",
      "\n",
      "Name: hiReadsProcessor\n",
      "Description:\n",
      "hiReadsProcessor contains set of functions which allow users to process LM-PCR products sequenced using any platform. Given an excel/txt file containing parameters for demultiplexing and sample metadata, the functions automate trimming of adaptors and identification of the genomic product. Genomic products are further processed for QC and abundance quantification.\n",
      "Version: 1.34.0\n",
      "Publication date: None\n",
      "Mantainer: Nirav V Malani <malnirav@gmail.com>\n",
      "Authors:\n",
      "Author: Nirav V Malani <malnirav at gmail.com>\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - Biostrings, type: depends\n",
      "  - GenomicAlignments, type: depends\n",
      "  - BiocParallel, type: depends\n",
      "  - hiAnnotator, type: depends\n",
      "  - sonicLength, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - readxl, type: imports\n",
      "  - methods, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/hiReadsProcessor.html\n",
      "\n",
      "Name: idpr\n",
      "Description:\n",
      "âidprâ aims to integrate tools for the computational analysis of intrinsically disordered proteins (IDPs) within R. This package is used to identify known characteristics of IDPs for a sequence of interest with easily reported and dynamic results. Additionally, this package includes tools for IDP-based sequence analysis to be used in conjunction with other R packages.\n",
      "Version: 1.8.0\n",
      "Publication date: None\n",
      "Mantainer: William M. McFadden <wmm27@pitt.edu>\n",
      "Authors:\n",
      "Author: William M. McFadden [cre, aut], Judith L. Yanowitz [aut, fnd], Michael Buszczak [ctb, fnd]\n",
      "Dependencies:\n",
      "  - R (>= 4.1.0) type: depends\n",
      "  - ggplot2 (>= 3.3.0) type: imports\n",
      "  - magrittr (>= 1.5) type: imports\n",
      "  - dplyr (>= 0.8.5) type: imports\n",
      "  - plyr (>= 1.8.6) type: imports\n",
      "  - jsonlite (>= 1.6.1) type: imports\n",
      "  - rlang (>= 0.4.6) type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - methods (>= 4.0.0) type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL (>= 3)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/idpr.html\n",
      "\n",
      "Name: INSPEcT\n",
      "Description:\n",
      "INSPEcT (INference of Synthesis, Processing and dEgradation rates from Transcriptomic data) RNA-seq data in time-course experiments or steady-state conditions, with or without the support of nascent RNA data.\n",
      "Version: 1.28.0\n",
      "Publication date: None\n",
      "Mantainer: Stefano de Pretis <ste.depo@gmail.com>, Mattia Furlan <Mattia.Furlan@iit.it>\n",
      "Authors:\n",
      "Author: Stefano de Pretis\n",
      "Dependencies:\n",
      "  - R (>= 3.6) type: depends\n",
      "  - methods, type: depends\n",
      "  - Biobase, type: depends\n",
      "  - BiocParallel, type: depends\n",
      "  - pROC, type: imports\n",
      "  - deSolve, type: imports\n",
      "  - rootSolve, type: imports\n",
      "  - KernSmooth, type: imports\n",
      "  - gdata, type: imports\n",
      "  - GenomicFeatures, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - GenomicAlignments, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - DESeq2, type: imports\n",
      "  - plgem, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - TxDb.Mmusculus.UCSC.mm9.knownGene, type: imports\n",
      "  - shiny, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/INSPEcT.html\n",
      "\n",
      "Name: ISAnalytics\n",
      "Description:\n",
      "In gene therapy, stem cells are modified using viral vectors to deliver the therapeutic transgene and replace functional properties since the genetic modification is stable and inherited in all cell progeny. The retrieval and mapping of the sequences flanking the virus-host DNA junctions allows the identification of insertion sites (IS), essential for monitoring the evolution of genetically modified cells in vivo. A comprehensive toolkit for the analysis of IS is required to foster clonal trackign studies and supporting the assessment of safety and long term efficacy in vivo. This package is aimed at (1) supporting automation of IS workflow, (2) performing base and advance analysis for IS tracking (clonal abundance, clonal expansions and statistics for insertional mutagenesis, etc.), (3) providing basic biology insights of transduced stem cells in vivo.\n",
      "Version: 1.8.1\n",
      "Publication date: None\n",
      "Mantainer: Andrea Calabria <calabria.andrea@hsr.it>\n",
      "Authors:\n",
      "Author: Andrea Calabria [aut, cre], Giulio Spinozzi [aut], Giulia Pais [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.2) type: depends\n",
      "  - magrittr, type: depends\n",
      "  - utils, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - readr, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - purrr, type: imports\n",
      "  - rlang, type: imports\n",
      "  - tibble, type: imports\n",
      "  - stringr, type: imports\n",
      "  - fs, type: imports\n",
      "  - lubridate, type: imports\n",
      "  - lifecycle, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - ggrepel, type: imports\n",
      "  - stats, type: imports\n",
      "  - psych, type: imports\n",
      "  - data.table, type: imports\n",
      "  - readxl, type: imports\n",
      "  - tools, type: imports\n",
      "  - Rcapture, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - forcats, type: imports\n",
      "  - glue, type: imports\n",
      "  - shiny, type: imports\n",
      "  - shinyWidgets, type: imports\n",
      "  - datamods, type: imports\n",
      "  - bslib, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "CC BY 4.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ISAnalytics.html\n",
      "\n",
      "Name: ldblock\n",
      "Description:\n",
      "Define data structures for linkage disequilibrium measures in populations.\n",
      "Version: 1.28.0\n",
      "Publication date: None\n",
      "Mantainer: VJ Carey <stvjc@channing.harvard.edu>\n",
      "Authors:\n",
      "Author: VJ Carey <stvjc at channing.harvard.edu>\n",
      "Dependencies:\n",
      "  - R (>= 3.5) type: depends\n",
      "  - methods, type: depends\n",
      "  - rlang, type: depends\n",
      "  - BiocGenerics, type: imports\n",
      "  - httr, type: imports\n",
      "  - Matrix, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ldblock.html\n",
      "\n",
      "Name: lionessR\n",
      "Description:\n",
      "LIONESS, or Linear Interpolation to Obtain Network Estimates for Single Samples, can be used to reconstruct single-sample networks (https://arxiv.org/abs/1505.06440). This code implements the LIONESS equation in the lioness function in R to reconstruct single-sample networks. The default network reconstruction method we use is based on Pearson correlation. However, lionessR can run on any network reconstruction algorithms that returns a complete, weighted adjacency matrix. lionessR works for both unipartite and bipartite networks.\n",
      "Version: 1.12.0\n",
      "Publication date: None\n",
      "Mantainer: Ping-Han Hsieh <dn070017@gmail.com>\n",
      "Authors:\n",
      "Author: Marieke Lydia Kuijjer [aut] , Ping-Han Hsieh [cre]\n",
      "Dependencies:\n",
      "  - R (>= 3.6.0) type: depends\n",
      "  - stats, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/lionessR.html\n",
      "\n",
      "Name: methylInheritance\n",
      "Description:\n",
      "Permutation analysis, based on Monte Carlo sampling, for testing the hypothesis that the number of conserved differentially methylated elements, between several generations, is associated to an effect inherited from a treatment and that stochastic effect can be dismissed.\n",
      "Version: 1.22.0\n",
      "Publication date: None\n",
      "Mantainer: Astrid DeschÃªnes <adeschen@hotmail.com>\n",
      "Authors:\n",
      "Author: Astrid DeschÃªnes [cre, aut] , Pascal Belleau [aut] , Arnaud Droit [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.5) type: depends\n",
      "  - methylKit, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - methods, type: imports\n",
      "  - parallel, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - gridExtra, type: imports\n",
      "  - rebus, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/methylInheritance.html\n",
      "\n",
      "Name: MSPrep\n",
      "Description:\n",
      "Package performs summarization of replicates, filtering by frequency, several different options for imputing missing data, and a variety of options for transforming, batch correcting, and normalizing data.\n",
      "Version: 1.8.0\n",
      "Publication date: None\n",
      "Mantainer: Max McGrath <max.mcgrath@ucdenver.edu>\n",
      "Authors:\n",
      "Author: Max McGrath [aut, cre], Matt Mulvahill [aut], Grant Hughes [aut], Sean Jacobson [aut], Harrison Pielke-lombardo [aut], Katerina Kechris [aut, cph, ths]\n",
      "Dependencies:\n",
      "  - R (>= 4.1.0) type: depends\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - pcaMethods, type: imports\n",
      "  - crmn, type: imports\n",
      "  - preprocessCore, type: imports\n",
      "  - dplyr (>= 0.7) type: imports\n",
      "  - tidyr, type: imports\n",
      "  - tibble (>= 1.2) type: imports\n",
      "  - magrittr, type: imports\n",
      "  - rlang, type: imports\n",
      "  - stats, type: imports\n",
      "  - stringr, type: imports\n",
      "  - methods, type: imports\n",
      "  - missForest, type: imports\n",
      "  - sva, type: imports\n",
      "  - VIM, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/MSPrep.html\n",
      "\n",
      "Name: MultiBaC\n",
      "Description:\n",
      "MultiBaC is a strategy to correct batch effects from multiomic datasets distributed across different labs or data acquisition events. MultiBaC is the first Batch effect correction algorithm that dealing with batch effect correction in multiomics datasets. MultiBaC is able to remove batch effects across different omics generated within separate batches provided that at least one common omic data type is included in all the batches considered.\n",
      "Version: 1.8.0\n",
      "Publication date: None\n",
      "Mantainer: The package maintainer <manuelugidos@gmail.com>\n",
      "Authors:\n",
      "Author: person(\"Manuel\", \"Ugidos\", email = \"manuelugidos@gmail.com\"), person(\"Sonia\", \"Tarazona\", email = \"sotacam@gmail.com\"), person(\"MarÃ­a JosÃ©\", \"Nueda\", email = \"mjnueda@ua.es\")\n",
      "Dependencies:\n",
      "  - Matrix, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - MultiAssayExperiment, type: imports\n",
      "  - ropls, type: imports\n",
      "  - graphics, type: imports\n",
      "  - methods, type: imports\n",
      "  - plotrix, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - pcaMethods, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/MultiBaC.html\n",
      "\n",
      "Name: multiOmicsViz\n",
      "Description:\n",
      "Calculate the spearman correlation between the source omics data and other target omics data, identify the significant correlations and plot the significant correlations on the heat map in which the x-axis and y-axis are ordered by the chromosomal location.\n",
      "Version: 1.22.0\n",
      "Publication date: None\n",
      "Mantainer: Jing Wang <jingwang.uestc@gmail.com>\n",
      "Authors:\n",
      "Author: Jing Wang <jingwang.uestc at gmail.com>\n",
      "Dependencies:\n",
      "  - R (>= 3.3.2) type: depends\n",
      "  - methods, type: imports\n",
      "  - parallel, type: imports\n",
      "  - doParallel, type: imports\n",
      "  - foreach, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - utils, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - stats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/multiOmicsViz.html\n",
      "\n",
      "Name: NanoTube\n",
      "Description:\n",
      "NanoTube includes functions for the processing, quality control, analysis, and visualization of NanoString nCounter data. Analysis functions include differential analysis and gene set analysis methods, as well as postprocessing steps to help understand the results. Additional functions are included to enable interoperability with other Bioconductor NanoString data analysis packages.\n",
      "Version: 1.4.0\n",
      "Publication date: None\n",
      "Mantainer: Caleb Class <cclass@butler.edu>\n",
      "Authors:\n",
      "Author: Caleb Class [cre, aut] , Caiden Lukan [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - Biobase, type: depends\n",
      "  - ggplot2, type: depends\n",
      "  - limma, type: depends\n",
      "  - fgsea, type: imports\n",
      "  - methods, type: imports\n",
      "  - reshape, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3 + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/NanoTube.html\n",
      "\n",
      "Name: openPrimeRui\n",
      "Description:\n",
      "A Shiny application providing methods for designing, evaluating, and comparing primer sets for multiplex polymerase chain reaction. Primers are designed by solving a set cover problem such that the number of covered template sequences is maximized with the smallest possible set of primers. To guarantee that high-quality primers are generated, only primers fulfilling constraints on their physicochemical properties are selected.\n",
      "Version: 1.20.0\n",
      "Publication date: None\n",
      "Mantainer: Matthias DÃ¶ring <matthias-doering@gmx.de>\n",
      "Authors:\n",
      "Author: Matthias DÃ¶ring [aut, cre], Nico Pfeifer [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.0.0) type: depends\n",
      "  - openPrimeR, type: depends\n",
      "  - shiny (>= 1.0.2) type: imports\n",
      "  - shinyjs (>= 0.9) type: imports\n",
      "  - shinyBS (>= 0.61) type: imports\n",
      "  - DT (>= 0.2) type: imports\n",
      "  - rmarkdown (>= 1.0) type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/openPrimeRui.html\n",
      "\n",
      "Name: PCAN\n",
      "Description:\n",
      "Phenotypes comparison based on a pathway consensus approach. Assess the relationship between candidate genes and a set of phenotypes based on additional genes related to the candidate (e.g. Pathways or network neighbors).\n",
      "Version: 1.26.0\n",
      "Publication date: None\n",
      "Mantainer: Matthew Page <matthew.page@ucb.com> and Patrice Godard <patrice.godard@gmail.com>\n",
      "Authors:\n",
      "Author: Matthew Page and Patrice Godard\n",
      "Dependencies:\n",
      "  - R (>= 3.3) type: depends\n",
      "  - BiocParallel, type: depends\n",
      "  - grDevices, type: imports\n",
      "  - stats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "CC BY-NC-ND 4.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/PCAN.html\n",
      "\n",
      "Name: peakPantheR\n",
      "Description:\n",
      "An automated pipeline for the detection, integration and reporting of predefined features across a large number of mass spectrometry data files. It enables the real time annotation of multiple compounds in a single file, or the parallel annotation of multiple compounds in multiple files. A graphical user interface as well as command line functions will assist in assessing the quality of annotation and update fitting parameters until a satisfactory result is obtained.\n",
      "Version: 1.12.2\n",
      "Publication date: None\n",
      "Mantainer: Arnaud Wolfer <adwolfer@gmail.com>\n",
      "Authors:\n",
      "Author: Arnaud Wolfer [aut, cre] , Goncalo Correia [aut] , Jake Pearce [ctb], Caroline Sands [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 4.2) type: depends\n",
      "  - foreach (>= 1.4.4) type: imports\n",
      "  - doParallel (>= 1.0.11) type: imports\n",
      "  - ggplot2 (>= 3.3.0) type: imports\n",
      "  - gridExtra (>= 2.3) type: imports\n",
      "  - MSnbase, type: imports\n",
      "  - mzR, type: imports\n",
      "  - stringr (>= 1.2.0) type: imports\n",
      "  - methods (>= 3.4.0) type: imports\n",
      "  - XML (>= 3.98.1.10) type: imports\n",
      "  - minpack.lm (>= 1.2.1) type: imports\n",
      "  - scales (>= 0.5.0) type: imports\n",
      "  - shiny (>= 1.0.5) type: imports\n",
      "  - bslib, type: imports\n",
      "  - shinycssloaders (>= 1.0.0) type: imports\n",
      "  - DT (>= 0.15) type: imports\n",
      "  - pracma (>= 2.2.3) type: imports\n",
      "  - utils, type: imports\n",
      "  - lubridate, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/peakPantheR.html\n",
      "\n",
      "Name: qPLEXanalyzer\n",
      "Description:\n",
      "Tools for TMT based quantitative proteomics data analysis.\n",
      "Version: 1.16.0\n",
      "Publication date: None\n",
      "Mantainer: Ashley Sawle <ads2202cu@gmail.com>\n",
      "Authors:\n",
      "Author: Matthew Eldridge [aut], Kamal Kishore [aut], Ashley Sawle [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - Biobase, type: depends\n",
      "  - MSnbase, type: depends\n",
      "  - assertthat, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - dplyr (>= 1.0.0) type: imports\n",
      "  - ggdendro, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - limma, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - preprocessCore, type: imports\n",
      "  - purrr, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - readr, type: imports\n",
      "  - rlang, type: imports\n",
      "  - scales, type: imports\n",
      "  - stats, type: imports\n",
      "  - stringr, type: imports\n",
      "  - tibble, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - tidyselect, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/qPLEXanalyzer.html\n",
      "\n",
      "Name: R453Plus1Toolbox\n",
      "Description:\n",
      "The R453Plus1 Toolbox comprises useful functions for the analysis of data generated by Roche's 454 sequencing platform. It adds functions for quality assurance as well as for annotation and visualization of detected variants, complementing the software tools shipped by Roche with their product. Further, a pipeline for the detection of structural variants is provided.\n",
      "Version: 1.48.0\n",
      "Publication date: None\n",
      "Mantainer: Hans-Ulrich Klein <h.klein@uni-muenster.de>\n",
      "Authors:\n",
      "Author: Hans-Ulrich Klein, Christoph Bartenhagen, Christian Ruckert\n",
      "Dependencies:\n",
      "  - R (>= 2.12.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - VariantAnnotation, type: depends\n",
      "  - Biostrings, type: depends\n",
      "  - Biobase, type: depends\n",
      "  - utils, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - tools, type: imports\n",
      "  - xtable, type: imports\n",
      "  - R2HTML, type: imports\n",
      "  - TeachingDemos, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - XVector, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - biomaRt, type: imports\n",
      "  - BSgenome, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - ShortRead, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/R453Plus1Toolbox.html\n",
      "\n",
      "Name: RCyjs\n",
      "Description:\n",
      "Interactive viewing and exploration of graphs, connecting R to Cytoscape.js, using websockets.\n",
      "Version: 2.20.0\n",
      "Publication date: None\n",
      "Mantainer: Paul Shannon <paul.thurmond.shannon@gmail.com>\n",
      "Authors:\n",
      "Author: Paul Shannon\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - BrowserViz, type: depends\n",
      "  - graph, type: depends\n",
      "  - methods, type: imports\n",
      "  - httpuv (>= 1.5.0) type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - base64enc, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/RCyjs.html\n",
      "\n",
      "Name: Risa\n",
      "Description:\n",
      "The Investigation / Study / Assay (ISA) tab-delimited format is a general purpose framework with which to collect and communicate complex metadata (i.e. sample characteristics, technologies used, type of measurements made) from experiments employing a combination of technologies, spanning from traditional approaches to high-throughput techniques. Risa allows to access metadata/data in ISA-Tab format and build Bioconductor data structures. Currently, data generated from microarray, flow cytometry and metabolomics-based (i.e. mass spectrometry) assays are supported. The package is extendable and efforts are undergoing to support metadata associated to proteomics assays.\n",
      "Version: 1.40.0\n",
      "Publication date: None\n",
      "Mantainer: Alejandra Gonzalez-Beltran <alejandra.gonzalez.beltran@gmail.com>\n",
      "Authors:\n",
      "Author: Alejandra Gonzalez-Beltran, Audrey Kauffmann, Steffen Neumann, Gabriella Rustici, ISA Team\n",
      "Dependencies:\n",
      "  - R (>= 2.0.9) type: depends\n",
      "  - Biobase, type: depends\n",
      "  - methods, type: depends\n",
      "  - Rcpp (>= 0.9.13) type: depends\n",
      "  - biocViews, type: depends\n",
      "  - affy, type: depends\n",
      "  - xcms, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/Risa.html\n",
      "\n",
      "Name: roar\n",
      "Description:\n",
      "Identify preferential usage of APA sites, comparing two biological conditions, starting from known alternative sites and alignments obtained from standard RNA-seq experiments.\n",
      "Version: 1.34.0\n",
      "Publication date: None\n",
      "Mantainer: Elena Grassi <grassi.e@gmail.com>\n",
      "Authors:\n",
      "Author: Elena Grassi\n",
      "Dependencies:\n",
      "  - R (>= 3.0.1) type: depends\n",
      "  - methods, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - GenomicAlignments, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/roar.html\n",
      "\n",
      "Name: RTNduals\n",
      "Description:\n",
      "RTNduals is a tool that searches for possible co-regulatory loops between regulon pairs generated by the RTN package. It compares the shared targets in order to infer 'dual regulons', a new concept that tests whether regulators can co-operate or compete in influencing targets.\n",
      "Version: 1.22.0\n",
      "Publication date: None\n",
      "Mantainer: Mauro Castro <mauro.a.castro@gmail.com>, Clarice Groeneveld <clari.groeneveld@gmail.com>\n",
      "Authors:\n",
      "Author: Vinicius S. Chagas, Clarice S. Groeneveld, Gordon Robertson, Kerstin B. Meyer, Mauro A. A. Castro\n",
      "Dependencies:\n",
      "  - R (>= 3.6.3) type: depends\n",
      "  - RTN, type: depends\n",
      "  - methods, type: depends\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/RTNduals.html\n",
      "\n",
      "Name: SBMLR\n",
      "Description:\n",
      "This package contains a systems biology markup language (SBML) interface to R.\n",
      "Version: 1.94.0\n",
      "Publication date: None\n",
      "Mantainer: Tomas Radivoyevitch <radivot@gmail.com>\n",
      "Authors:\n",
      "Author: Tomas Radivoyevitch, Vishak Venkateswaran\n",
      "Dependencies:\n",
      "  - XML, type: depends\n",
      "  - deSolve, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/SBMLR.html\n",
      "\n",
      "Name: SCBN\n",
      "Description:\n",
      "This package provides a scale based normalization (SCBN) method to identify genes with differential expression between different species. It takes into account the available knowledge of conserved orthologous genes and the hypothesis testing framework to detect differentially expressed orthologous genes. The method on this package are described in the article 'A statistical normalization method and differential expression analysis for RNA-seq data between different species' by Yan Zhou, Jiadi Zhu, Tiejun Tong, Junhui Wang, Bingqing Lin, Jun Zhang (2018, pending publication).\n",
      "Version: 1.16.0\n",
      "Publication date: None\n",
      "Mantainer: Yan Zhou <2160090406@email.szu.edu.cn>\n",
      "Authors:\n",
      "Author: Yan Zhou\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - stats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/SCBN.html\n",
      "\n",
      "Name: similaRpeak\n",
      "Description:\n",
      "This package calculates metrics which quantify the level of similarity between ChIP-Seq profiles. More specifically, the package implements six pseudometrics specialized in pattern similarity detection in ChIP-Seq profiles.\n",
      "Version: 1.30.0\n",
      "Publication date: None\n",
      "Mantainer: Astrid DeschÃªnes <adeschen@hotmail.com>\n",
      "Authors:\n",
      "Author: Astrid DeschÃªnes [cre, aut], Elsa Bernatchez [aut], Charles Joly Beauparlant [aut], Fabien Claude Lamaze [aut], Rawane Samb [aut], Pascal Belleau [aut], Arnaud Droit [aut]\n",
      "Dependencies:\n",
      "  - R6 (>= 2.0) type: depends\n",
      "  - stats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/similaRpeak.html\n",
      "\n",
      "Name: SparseSignatures\n",
      "Description:\n",
      "Point mutations occurring in a genome can be divided into 96 categories based on the base being mutated, the base it is mutated into and its two flanking bases. Therefore, for any patient, it is possible to represent all the point mutations occurring in that patient's tumor as a vector of length 96, where each element represents the count of mutations for a given category in the patient. A mutational signature represents the pattern of mutations produced by a mutagen or mutagenic process inside the cell. Each signature can also be represented by a vector of length 96, where each element represents the probability that this particular mutagenic process generates a mutation of the 96 above mentioned categories. In this R package, we provide a set of functions to extract and visualize the mutational signatures that best explain the mutation counts of a large number of patients.\n",
      "Version: 2.8.0\n",
      "Publication date: None\n",
      "Mantainer: Luca De Sano <luca.desano@gmail.com>\n",
      "Authors:\n",
      "Author: Daniele Ramazzotti [cre, aut] , Avantika Lal [aut], Keli Liu [ctb], Luca De Sano [aut] , Robert Tibshirani [ctb], Arend Sidow [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.1.0) type: depends\n",
      "  - NMF, type: depends\n",
      "  - nnlasso, type: imports\n",
      "  - nnls, type: imports\n",
      "  - parallel, type: imports\n",
      "  - data.table, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - BSgenome, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - gridExtra, type: imports\n",
      "  - reshape2, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/SparseSignatures.html\n",
      "\n",
      "Name: transcriptogramer\n",
      "Description:\n",
      "R package for transcriptional analysis based on transcriptograms, a method to analyze transcriptomes that projects expression values on a set of ordered proteins, arranged such that the probability that gene products participate in the same metabolic pathway exponentially decreases with the increase of the distance between two proteins of the ordering. Transcriptograms are, hence, genome wide gene expression profiles that provide a global view for the cellular metabolism, while indicating gene sets whose expressions are altered.\n",
      "Version: 1.20.0\n",
      "Publication date: None\n",
      "Mantainer: Diego Morais <vinx@ufrn.edu.br>\n",
      "Authors:\n",
      "Author: Diego Morais [aut, cre], Rodrigo Dalmolin [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.4) type: depends\n",
      "  - methods, type: depends\n",
      "  - biomaRt, type: imports\n",
      "  - data.table, type: imports\n",
      "  - doSNOW, type: imports\n",
      "  - foreach, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - igraph, type: imports\n",
      "  - limma, type: imports\n",
      "  - parallel, type: imports\n",
      "  - progress, type: imports\n",
      "  - RedeR, type: imports\n",
      "  - snow, type: imports\n",
      "  - stats, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - topGO, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/transcriptogramer.html\n",
      "\n",
      "Name: Trendy\n",
      "Description:\n",
      "Trendy implements segmented (or breakpoint) regression models to estimate breakpoints which represent changes in expression for each feature/gene in high throughput data with ordered conditions.\n",
      "Version: 1.20.0\n",
      "Publication date: None\n",
      "Mantainer: Rhonda Bacher <rbacher@ufl.edu>\n",
      "Authors:\n",
      "Author: Rhonda Bacher and Ning Leng\n",
      "Dependencies:\n",
      "  - R (>= 3.4) type: depends\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - segmented, type: imports\n",
      "  - gplots, type: imports\n",
      "  - parallel, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - DT, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - methods, type: imports\n",
      "  - shiny, type: imports\n",
      "  - shinyFiles, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/Trendy.html\n",
      "\n",
      "Name: ADAMgui\n",
      "Description:\n",
      "ADAMgui is a Graphical User Interface for the ADAM package. The ADAMgui package provides 2 shiny-based applications that allows the user to study the output of the ADAM package files through different plots. It's possible, for example, to choose a specific GFAG and observe the gene expression behavior with the plots created with the GFAGtargetUi function. Features such as differential expression and foldchange can be easily seen with aid of the plots made with GFAGpathUi function.\n",
      "Version: 1.14.0\n",
      "Publication date: None\n",
      "Mantainer: Jose Luiz Rybarczyk Filho <jose.luiz@unesp.br>\n",
      "Authors:\n",
      "Author: Giordano Bruno Sanches Seco [aut], AndrÃ© Luiz Molan [ctb], Agnes Takeda [ctb], Jose Rybarczyk Filho [ctb, cre, ths]\n",
      "Dependencies:\n",
      "  - R (>= 3.6) type: depends\n",
      "  - stats, type: depends\n",
      "  - utils, type: depends\n",
      "  - methods, type: depends\n",
      "  - ADAM, type: depends\n",
      "  - GO.db, type: imports\n",
      "  - dplyr (>= 0.7.6) type: imports\n",
      "  - shiny (>= 1.1.0) type: imports\n",
      "  - stringr (>= 1.3.1) type: imports\n",
      "  - stringi (>= 1.2.4) type: imports\n",
      "  - varhandle (>= 2.0.3) type: imports\n",
      "  - ggplot2 (>= 3.0.0) type: imports\n",
      "  - ggrepel (>= 0.8.0) type: imports\n",
      "  - ggpubr (>= 0.1.8) type: imports\n",
      "  - ggsignif (>= 0.4.0) type: imports\n",
      "  - reshape2 (>= 1.4.3) type: imports\n",
      "  - RColorBrewer (>= 1.1-2) type: imports\n",
      "  - colorRamps (>= 2.3) type: imports\n",
      "  - DT (>= 0.4) type: imports\n",
      "  - data.table (>= 1.11.4) type: imports\n",
      "  - gridExtra (>= 2.3) type: imports\n",
      "  - shinyjs (>= 1.0) type: imports\n",
      "  - knitr, type: imports\n",
      "  - testthat, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ADAMgui.html\n",
      "\n",
      "Name: adductomicsR\n",
      "Description:\n",
      "Processes MS2 data to identify potentially adducted peptides from spectra that has been corrected for mass drift and retention time drift and quantifies MS1 level mass spectral peaks.\n",
      "Version: 1.14.0\n",
      "Publication date: None\n",
      "Mantainer: Josie Hayes <jlhayes1982@gmail.com>\n",
      "Authors:\n",
      "Author: Josie Hayes <jlhayes1982 at gmail.com>\n",
      "Dependencies:\n",
      "  - R (>= 3.6) type: depends\n",
      "  - adductData, type: depends\n",
      "  - ExperimentHub, type: depends\n",
      "  - AnnotationHub, type: depends\n",
      "  - parallel (>= 3.3.2) type: imports\n",
      "  - data.table (>= 1.10.4) type: imports\n",
      "  - OrgMassSpecR (>= 0.4.6) type: imports\n",
      "  - foreach (>= 1.4.3) type: imports\n",
      "  - mzR, type: imports\n",
      "  - ade4 (>= 1.7.6) type: imports\n",
      "  - rvest (>= 0.3.2) type: imports\n",
      "  - pastecs (>= 1.3.18) type: imports\n",
      "  - reshape2 (>= 1.4.2) type: imports\n",
      "  - pracma (>= 2.0.4) type: imports\n",
      "  - DT (>= 0.2) type: imports\n",
      "  - fpc (>= 2.1.10) type: imports\n",
      "  - doSNOW (>= 1.0.14) type: imports\n",
      "  - fastcluster (>= 1.1.22) type: imports\n",
      "  - RcppEigen (>= 0.3.3.3.0) type: imports\n",
      "  - bootstrap (>= 2017.2) type: imports\n",
      "  - smoother (>= 1.1) type: imports\n",
      "  - dplyr (>= 0.7.5) type: imports\n",
      "  - zoo (>= 1.8) type: imports\n",
      "  - stats (>= 3.5.0) type: imports\n",
      "  - utils (>= 3.5.0) type: imports\n",
      "  - graphics (>= 3.5.0) type: imports\n",
      "  - grDevices (>= 3.5.0) type: imports\n",
      "  - methods (>= 3.5.0) type: imports\n",
      "  - datasets (>= 3.5.0) type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/adductomicsR.html\n",
      "\n",
      "Name: AGDEX\n",
      "Description:\n",
      "A tool to evaluate agreement of differential expression for cross-species genomics\n",
      "Version: 1.46.0\n",
      "Publication date: None\n",
      "Mantainer: Cuilan lani Gao <cuilan.gao@stjude.org>\n",
      "Authors:\n",
      "Author: Stan Pounds <stanley.pounds at stjude.org>; Cuilan Lani Gao <cuilan.gao at stjude.org>\n",
      "Dependencies:\n",
      "  - R (>= 2.10) type: depends\n",
      "  - Biobase, type: depends\n",
      "  - GSEABase, type: depends\n",
      "  - stats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL Version 2 or later\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/AGDEX.html\n",
      "\n",
      "Name: ASGSCA\n",
      "Description:\n",
      "The package provides tools to model and test the association between multiple genotypes and multiple traits, taking into account the prior biological knowledge. Genes, and clinical pathways are incorporated in the model as latent variables. The method is based on Generalized Structured Component Analysis (GSCA).\n",
      "Version: 1.32.0\n",
      "Publication date: None\n",
      "Mantainer: Hela Romdhani <hela.romdhani@mcgill.ca>\n",
      "Authors:\n",
      "Author: Hela Romdhani, Stepan Grinek , Heungsun Hwang and Aurelie Labbe.\n",
      "Dependencies:\n",
      "  - Matrix, type: imports\n",
      "  - MASS, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ASGSCA.html\n",
      "\n",
      "Name: atena\n",
      "Description:\n",
      "Quantify expression of transposable elements (TEs) from RNA-seq data through different methods, including ERVmap, TEtranscripts and Telescope. A common interface is provided to use each of these methods, which consists of building a parameter object, calling the quantification function with this object and getting a SummarizedExperiment object as output container of the quantified expression profiles. The implementation allows one to quantify TEs and gene transcripts in an integrated manner.\n",
      "Version: 1.4.1\n",
      "Publication date: None\n",
      "Mantainer: Beatriz Calvo-Serra <beatriz.calvo@upf.edu>\n",
      "Authors:\n",
      "Author: Beatriz Calvo-Serra [aut, cre], Robert Castelo [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - GenomicAlignments, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - SQUAREM, type: imports\n",
      "  - sparseMatrixStats, type: imports\n",
      "  - AnnotationHub, type: imports\n",
      "  - scales, type: imports\n",
      "  - matrixStats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/atena.html\n",
      "\n",
      "Name: attract\n",
      "Description:\n",
      "This package contains the functions to find the gene expression modules that represent the drivers of Kauffman's attractor landscape. The modules are the core attractor pathways that discriminate between different cell types of groups of interest. Each pathway has a set of synexpression groups, which show transcriptionally-coordinated changes in gene expression.\n",
      "Version: 1.50.0\n",
      "Publication date: None\n",
      "Mantainer: Samuel Zimmerman <samuel.e.zimmerman@gmail.com>\n",
      "Authors:\n",
      "Author: Jessica Mar\n",
      "Dependencies:\n",
      "  - R (>= 3.4.0) type: depends\n",
      "  - AnnotationDbi, type: depends\n",
      "  - Biobase, type: imports\n",
      "  - limma, type: imports\n",
      "  - cluster, type: imports\n",
      "  - GOstats, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - reactome.db, type: imports\n",
      "  - KEGGREST, type: imports\n",
      "  - org.Hs.eg.db, type: imports\n",
      "  - utils, type: imports\n",
      "  - methods, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL (>= 2.0)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/attract.html\n",
      "\n",
      "Name: AWFisher\n",
      "Description:\n",
      "Implementation of the adaptively weighted fisher's method, including fast p-value computing, variability index, and meta-pattern.\n",
      "Version: 1.12.0\n",
      "Publication date: None\n",
      "Mantainer: Zhiguang Huo <zhuo@ufl.edu>\n",
      "Authors:\n",
      "Author: Zhiguang Huo\n",
      "Dependencies:\n",
      "  - R (>= 3.6) type: depends\n",
      "  - edgeR, type: imports\n",
      "  - limma, type: imports\n",
      "  - stats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/AWFisher.html\n",
      "\n",
      "Name: BAC\n",
      "Description:\n",
      "Bioconductor version: Release (3.16)\n",
      "Version: 1.58.0\n",
      "Publication date: None\n",
      "Mantainer: Author: Raphael Gottardo\n",
      "Authors:\n",
      "This package uses a Bayesian hierarchical model to detect enriched regions from ChIP-chip experiments\n",
      "Dependencies:\n",
      "  - R (>= 2.10) type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/BAC.html\n",
      "\n",
      "Name: BDMMAcorrect\n",
      "Description:\n",
      "Metagenomic sequencing techniques enable quantitative analyses of the microbiome. However, combining the microbial data from these experiments is challenging due to the variations between experiments. The existing methods for correcting batch effects do not consider the interactions between variablesâmicrobial taxa in microbial studiesâand the overdispersion of the microbiome data. Therefore, they are not applicable to microbiome data. We develop a new method, Bayesian Dirichlet-multinomial regression meta-analysis (BDMMA), to simultaneously model the batch effects and detect the microbial taxa associated with phenotypes. BDMMA automatically models the dependence among microbial taxa and is robust to the high dimensionality of the microbiome and their association sparsity.\n",
      "Version: 1.16.2\n",
      "Publication date: None\n",
      "Mantainer: ZHENWEI DAI <daizwhao@gmail.com>\n",
      "Authors:\n",
      "Author: ZHENWEI DAI <daizwhao at gmail.com>\n",
      "Dependencies:\n",
      "  - R (>= 3.5) type: depends\n",
      "  - vegan, type: depends\n",
      "  - ellipse, type: depends\n",
      "  - ggplot2, type: depends\n",
      "  - ape, type: depends\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - Rcpp (>= 0.12.12) type: imports\n",
      "  - RcppArmadillo, type: imports\n",
      "  - RcppEigen, type: imports\n",
      "  - stats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/BDMMAcorrect.html\n",
      "\n",
      "Name: CellTrails\n",
      "Description:\n",
      "CellTrails is an unsupervised algorithm for the de novo chronological ordering, visualization and analysis of single-cell expression data. CellTrails makes use of a geometrically motivated concept of lower-dimensional manifold learning, which exhibits a multitude of virtues that counteract intrinsic noise of single cell data caused by drop-outs, technical variance, and redundancy of predictive variables. CellTrails enables the reconstruction of branching trajectories and provides an intuitive graphical representation of expression patterns along all branches simultaneously. It allows the user to define and infer the expression dynamics of individual and multiple pathways towards distinct phenotypes.\n",
      "Version: 1.16.0\n",
      "Publication date: None\n",
      "Mantainer: Daniel Ellwanger <dc.ellwanger.dev@gmail.com>\n",
      "Authors:\n",
      "Author: Daniel Ellwanger [aut, cre, cph]\n",
      "Dependencies:\n",
      "  - R (>= 3.5) type: depends\n",
      "  - SingleCellExperiment, type: depends\n",
      "  - BiocGenerics, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - cba, type: imports\n",
      "  - dendextend, type: imports\n",
      "  - dtw, type: imports\n",
      "  - EnvStats, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - ggrepel, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - igraph, type: imports\n",
      "  - maptree, type: imports\n",
      "  - methods, type: imports\n",
      "  - mgcv, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - Rtsne, type: imports\n",
      "  - stats, type: imports\n",
      "  - splines, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/CellTrails.html\n",
      "\n",
      "Name: clstutils\n",
      "Description:\n",
      "Tools for performing taxonomic assignment based on phylogeny using pplacer and clst.\n",
      "Version: 1.46.0\n",
      "Publication date: None\n",
      "Mantainer: Noah Hoffman <ngh2@uw.edu>\n",
      "Authors:\n",
      "Author: Noah Hoffman\n",
      "Dependencies:\n",
      "  - R (>= 2.10) type: depends\n",
      "  - clst, type: depends\n",
      "  - rjson, type: depends\n",
      "  - ape, type: depends\n",
      "  - lattice, type: imports\n",
      "  - RSQLite, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/clstutils.html\n",
      "\n",
      "Name: CNAnorm\n",
      "Description:\n",
      "Performs ratio, GC content correction and normalization of data obtained using low coverage (one read every 100-10,000 bp) high troughput sequencing. It performs a \"discrete\" normalization looking for the ploidy of the genome. It will also provide tumour content if at least two ploidy states can be found.\n",
      "Version: 1.44.3\n",
      "Publication date: None\n",
      "Mantainer: Stefano Berri <sberri@illumina.com>\n",
      "Authors:\n",
      "Author: Stefano Berri <sberri at illumina.com>, Henry M. Wood <H.M.Wood at leeds.ac.uk>, Arief Gusnanto <a.gusnanto at leeds.ac.uk>\n",
      "Dependencies:\n",
      "  - R (>= 2.10.1) type: depends\n",
      "  - methods, type: depends\n",
      "  - DNAcopy, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/CNAnorm.html\n",
      "\n",
      "Name: CNORfeeder\n",
      "Description:\n",
      "This package integrates literature-constrained and data-driven methods to infer signalling networks from perturbation experiments. It permits to extends a given network with links derived from the data via various inference methods and uses information on physical interactions of proteins to guide and validate the integration of links.\n",
      "Version: 1.38.0\n",
      "Publication date: None\n",
      "Mantainer: Attila Gabor <attila.gabor@uni-heidelberg.de>\n",
      "Authors:\n",
      "Author: Federica Eduati [aut], Enio Gjerga [ctb], Attila Gabor [cre]\n",
      "Dependencies:\n",
      "  - R (>= 3.6.0) type: depends\n",
      "  - CellNOptR, type: depends\n",
      "  - graph, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/CNORfeeder.html\n",
      "\n",
      "Name: crisprBase\n",
      "Description:\n",
      "Provides S4 classes for general nucleases, CRISPR nucleases, CRISPR nickases, and base editors.Several CRISPR-specific genome arithmetic functions are implemented to help extract genomic coordinates of spacer and protospacer sequences. Commonly-used CRISPR nuclease objects are provided that can be readily used in other packages. Both DNA- and RNA-targeting nucleases are supported.\n",
      "Version: 1.2.0\n",
      "Publication date: None\n",
      "Mantainer: Jean-Philippe Fortin <fortin946@gmail.com>\n",
      "Authors:\n",
      "Author: Jean-Philippe Fortin [aut, cre]\n",
      "Dependencies:\n",
      "  - utils, type: depends\n",
      "  - methods, type: depends\n",
      "  - R (>= 4.1) type: depends\n",
      "  - BiocGenerics, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - graphics, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - stringr, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/crisprBase.html\n",
      "\n",
      "Name: DEWSeq\n",
      "Description:\n",
      "DEWSeq is a sliding window approach for the analysis of differentially enriched binding regions eCLIP or iCLIP next generation sequencing data.\n",
      "Version: 1.12.0\n",
      "Publication date: None\n",
      "Mantainer: bioinformatics team Hentze <biohentze@embl.de>\n",
      "Authors:\n",
      "Author: Sudeep Sahadevan [aut], Thomas Schwarzl [aut], bioinformatics team Hentze [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.0.0) type: depends\n",
      "  - R.utils, type: depends\n",
      "  - DESeq2, type: depends\n",
      "  - BiocParallel, type: depends\n",
      "  - BiocGenerics, type: imports\n",
      "  - data.table (>= 1.11.8) type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - methods, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL (>= 3)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/DEWSeq.html\n",
      "\n",
      "Name: diggit\n",
      "Description:\n",
      "Inference of Genetic Variants Driving Cellullar Phenotypes by the DIGGIT algorithm\n",
      "Version: 1.30.0\n",
      "Publication date: None\n",
      "Mantainer: Mariano J Alvarez <reef103@gmail.com>\n",
      "Authors:\n",
      "Author: Mariano J Alvarez <reef103 at gmail.com>\n",
      "Dependencies:\n",
      "  - R (>= 3.0.2) type: depends\n",
      "  - Biobase, type: depends\n",
      "  - methods, type: depends\n",
      "  - ks, type: imports\n",
      "  - viper, type: imports\n",
      "  - parallel, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/diggit.html\n",
      "\n",
      "Name: drugTargetInteractions\n",
      "Description:\n",
      "Provides utilities for identifying drug-target interactions for sets of small molecule or gene/protein identifiers. The required drug-target interaction information is obained from a local SQLite instance of the ChEMBL database. ChEMBL has been chosen for this purpose, because it provides one of the most comprehensive and best annotatated knowledge resources for drug-target information available in the public domain.\n",
      "Version: 1.6.0\n",
      "Publication date: None\n",
      "Mantainer: Thomas Girke <thomas.girke@ucr.edu>\n",
      "Authors:\n",
      "Author: Thomas Girke [cre, aut]\n",
      "Dependencies:\n",
      "  - methods, type: depends\n",
      "  - R (>= 4.1) type: depends\n",
      "  - utils, type: imports\n",
      "  - RSQLite, type: imports\n",
      "  - UniProt.ws, type: imports\n",
      "  - biomaRt, type: imports\n",
      "  - ensembldb, type: imports\n",
      "  - BiocFileCache, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - rappdirs, type: imports\n",
      "  - AnnotationFilter, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/drugTargetInteractions.html\n",
      "\n",
      "Name: epivizrStandalone\n",
      "Description:\n",
      "This package imports the epiviz visualization JavaScript app for genomic data interactive visualization. The 'epivizrServer' package is used to provide a web server running completely within R. This standalone version allows to browse arbitrary genomes through genome annotations provided by Bioconductor packages.\n",
      "Version: 1.26.0\n",
      "Publication date: None\n",
      "Mantainer: Hector Corrada Bravo <hcorrada@gmail.com>\n",
      "Authors:\n",
      "Author: Hector Corrada Bravo, Jayaram Kancherla\n",
      "Dependencies:\n",
      "  - R (>= 3.2.3) type: depends\n",
      "  - epivizr, type: depends\n",
      "  - methods, type: depends\n",
      "  - git2r, type: imports\n",
      "  - epivizrServer, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - GenomicFeatures, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/epivizrStandalone.html\n",
      "\n",
      "Name: gcapc\n",
      "Description:\n",
      "Peak calling for ChIP-seq data with consideration of potential GC bias in sequencing reads. GC bias is first estimated with generalized linear mixture models using effective GC strategy, then applied into peak significance estimation.\n",
      "Version: 1.22.0\n",
      "Publication date: None\n",
      "Mantainer: Mingxiang Teng <tengmx@gmail.com>\n",
      "Authors:\n",
      "Author: Mingxiang Teng and Rafael A. Irizarry\n",
      "Dependencies:\n",
      "  - R (>= 3.4) type: depends\n",
      "  - BiocGenerics, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - BSgenome, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - GenomicAlignments, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - MASS, type: imports\n",
      "  - splines, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - methods, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/gcapc.html\n",
      "\n",
      "Name: gcatest\n",
      "Description:\n",
      "GCAT is an association test for genome wide association studies that controls for population structure under a general class of trait. models.\n",
      "Version: 1.28.2\n",
      "Publication date: None\n",
      "Mantainer: Wei Hao <whao@princeton.edu>, John D. Storey <jstorey@princeton.edu>\n",
      "Authors:\n",
      "Author: Wei Hao, Minsun Song, John D. Storey\n",
      "Dependencies:\n",
      "  - R (>= 3.2) type: depends\n",
      "  - lfa, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/gcatest.html\n",
      "\n",
      "Name: GeneSelectMMD\n",
      "Description:\n",
      "Gene selection based on a mixture of marginal distributions.\n",
      "Version: 2.42.3\n",
      "Publication date: None\n",
      "Mantainer: Weiliang Qiu <weiliang.qiu@gmail.com>\n",
      "Authors:\n",
      "Author: Jarrett Morrow <remdj at channing.harvard.edu>, Weiliang Qiu <weiliang.qiu at gmail.com>, Wenqing He <whe at stats.uwo.ca>, Xiaogang Wang <stevenw at mathstat.yorku.ca>, Ross Lazarus <ross.lazarus at channing.harvard.edu>.\n",
      "Dependencies:\n",
      "  - R (>= 2.13.2) type: depends\n",
      "  - Biobase, type: depends\n",
      "  - MASS, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - limma, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/GeneSelectMMD.html\n",
      "\n",
      "Name: genomes\n",
      "Description:\n",
      "Download genome and assembly reports from NCBI\n",
      "Version: 3.28.0\n",
      "Publication date: None\n",
      "Mantainer: Chris Stubben <stubben@lanl.gov>\n",
      "Authors:\n",
      "Author: Chris Stubben\n",
      "Dependencies:\n",
      "  - readr, type: depends\n",
      "  - curl, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/genomes.html\n",
      "\n",
      "Name: goSTAG\n",
      "Description:\n",
      "Gene lists derived from the results of genomic analyses are rich in biological information. For instance, differentially expressed genes (DEGs) from a microarray or RNA-Seq analysis are related functionally in terms of their response to a treatment or condition. Gene lists can vary in size, up to several thousand genes, depending on the robustness of the perturbations or how widely different the conditions are biologically. Having a way to associate biological relatedness between hundreds and thousands of genes systematically is impractical by manually curating the annotation and function of each gene. Over-representation analysis (ORA) of genes was developed to identify biological themes. Given a Gene Ontology (GO) and an annotation of genes that indicate the categories each one fits into, significance of the over-representation of the genes within the ontological categories is determined by a Fisher's exact test or modeling according to a hypergeometric distribution. Comparing a small number of enriched biological categories for a few samples is manageable using Venn diagrams or other means for assessing overlaps. However, with hundreds of enriched categories and many samples, the comparisons are laborious. Furthermore, if there are enriched categories that are shared between samples, trying to represent a common theme across them is highly subjective. goSTAG uses GO subtrees to tag and annotate genes within a set. goSTAG visualizes the similarities between the over-representation of DEGs by clustering the p-values from the enrichment statistical tests and labels clusters with the GO term that has the most paths to the root within the subtree generated from all the GO terms in the cluster.\n",
      "Version: 1.22.0\n",
      "Publication date: None\n",
      "Mantainer: Brian D. Bennett <brian.bennett@nih.gov>\n",
      "Authors:\n",
      "Author: Brian D. Bennett and Pierre R. Bushel\n",
      "Dependencies:\n",
      "  - R (>= 3.4) type: depends\n",
      "  - AnnotationDbi, type: imports\n",
      "  - biomaRt, type: imports\n",
      "  - GO.db, type: imports\n",
      "  - graphics, type: imports\n",
      "  - memoise, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/goSTAG.html\n",
      "\n",
      "Name: gpart\n",
      "Description:\n",
      "Bioconductor version: Release (3.16)\n",
      "Version: 1.16.0\n",
      "Publication date: None\n",
      "Mantainer: Author: Sun Ah Kim [aut, cre, cph], Yun Joo Yoo [aut, cph]\n",
      "Authors:\n",
      "we provide a new SNP sequence partitioning method which partitions the whole SNP sequence based on not only LD block structures but also gene location information. The LD block construction for GPART is performed using Big-LD algorithm, with additional improvement from previous version reported in Kim et al.(2017). We also add a visualization tool to show the LD heatmap with the information of LD block boundaries and gene locations in the package.\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - grid, type: depends\n",
      "  - Homo.sapiens, type: depends\n",
      "  - TxDb.Hsapiens.UCSC.hg38.knownGene, type: depends\n",
      "  - igraph, type: imports\n",
      "  - biomaRt, type: imports\n",
      "  - Rcpp, type: imports\n",
      "  - data.table, type: imports\n",
      "  - OrganismDbi, type: imports\n",
      "  - AnnotationDbi, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - IRanges, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/gpart.html\n",
      "\n",
      "Name: GSEAmining\n",
      "Description:\n",
      "Gene Set Enrichment Analysis is a very powerful and interesting computational method that allows an easy correlation between differential expressed genes and biological processes. Unfortunately, although it was designed to help researchers to interpret gene expression data it can generate huge amounts of results whose biological meaning can be difficult to interpret. Many available tools rely on the hierarchically structured Gene Ontology (GO) classification to reduce reundandcy in the results. However, due to the popularity of GSEA many more gene set collections, such as those in the Molecular Signatures Database are emerging. Since these collections are not organized as those in GO, their usage for GSEA do not always give a straightforward answer or, in other words, getting all the meaninful information can be challenging with the currently available tools. For these reasons, GSEAmining was born to be an easy tool to create reproducible reports to help researchers make biological sense of GSEA outputs. Given the results of GSEA, GSEAmining clusters the different gene sets collections based on the presence of the same genes in the leadind edge (core) subset. Leading edge subsets are those genes that contribute most to the enrichment score of each collection of genes or gene sets. For this reason, gene sets that participate in similar biological processes should share genes in common and in turn cluster together. After that, GSEAmining is able to identify and represent for each cluster: - The most enriched terms in the names of gene sets (as wordclouds) - The most enriched genes in the leading edge subsets (as bar plots). In each case, positive and negative enrichments are shown in different colors so it is easy to distinguish biological processes or genes that may be of interest in that particular study.\n",
      "Version: 1.8.0\n",
      "Publication date: None\n",
      "Mantainer: Oriol ArquÃ©s <oriol.arques@gmail.com>\n",
      "Authors:\n",
      "Author: Oriol ArquÃ©s [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - dplyr, type: imports\n",
      "  - tidytext, type: imports\n",
      "  - dendextend, type: imports\n",
      "  - tibble, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - ggwordcloud, type: imports\n",
      "  - stringr, type: imports\n",
      "  - gridExtra, type: imports\n",
      "  - rlang, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - methods, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3 | file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/GSEAmining.html\n",
      "\n",
      "Name: iterativeBMA\n",
      "Description:\n",
      "The iterative Bayesian Model Averaging (BMA) algorithm is a variable selection and classification algorithm with an application of classifying 2-class microarray samples, as described in Yeung, Bumgarner and Raftery (Bioinformatics 2005, 21: 2394-2402).\n",
      "Version: 1.56.0\n",
      "Publication date: None\n",
      "Mantainer: Ka Yee Yeung <kayee@u.washington.edu>\n",
      "Authors:\n",
      "Author: Ka Yee Yeung, University of Washington, Seattle, WA, with contributions from Adrian Raftery and Ian Painter\n",
      "Dependencies:\n",
      "  - BMA, type: depends\n",
      "  - leaps, type: depends\n",
      "  - Biobase, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/iterativeBMA.html\n",
      "\n",
      "Name: KEGGlincs\n",
      "Description:\n",
      "See what is going on 'under the hood' of KEGG pathways by explicitly re-creating the pathway maps from information obtained from KGML files.\n",
      "Version: 1.24.0\n",
      "Publication date: None\n",
      "Mantainer: Shana White <vandersm@mail.uc.edu>, Mario Medvedovic <medvedm@ucmail.uc.edu>\n",
      "Authors:\n",
      "Author: Shana White\n",
      "Dependencies:\n",
      "  - R (>= 3.3) type: depends\n",
      "  - KOdata, type: depends\n",
      "  - hgu133a.db, type: depends\n",
      "  - org.Hs.eg.db, type: depends\n",
      "  - AnnotationDbi, type: imports\n",
      "  - KEGGgraph, type: imports\n",
      "  - igraph, type: imports\n",
      "  - plyr, type: imports\n",
      "  - gtools, type: imports\n",
      "  - httr, type: imports\n",
      "  - RJSONIO, type: imports\n",
      "  - KEGGREST, type: imports\n",
      "  - methods, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - XML, type: imports\n",
      "  - grDevices, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/KEGGlincs.html\n",
      "\n",
      "Name: KnowSeq\n",
      "Description:\n",
      "KnowSeq proposes a novel methodology that comprises the most relevant steps in the Transcriptomic gene expression analysis. KnowSeq expects to serve as an integrative tool that allows to process and extract relevant biomarkers, as well as to assess them through a Machine Learning approaches. Finally, the last objective of KnowSeq is the biological knowledge extraction from the biomarkers (Gene Ontology enrichment, Pathway listing and Visualization and Evidences related to the addressed disease). Although the package allows analyzing all the data manually, the main strenght of KnowSeq is the possibilty of carrying out an automatic and intelligent HTML report that collect all the involved steps in one document. It is important to highligh that the pipeline is totally modular and flexible, hence it can be started from whichever of the different steps. KnowSeq expects to serve as a novel tool to help to the experts in the field to acquire robust knowledge and conclusions for the data and diseases to study.\n",
      "Version: 1.12.0\n",
      "Publication date: None\n",
      "Mantainer: Daniel Castillo-Secilla <cased@ugr.es>\n",
      "Authors:\n",
      "Author: Daniel Castillo-Secilla [aut, cre], Juan Manuel Galvez [ctb], Francisco Carrillo-Perez [ctb], Marta Verona-Almeida [ctb], Daniel Redondo-Sanchez [ctb], Francisco Manuel Ortuno [ctb], Luis Javier Herrera [ctb], Ignacio Rojas [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - cqn, type: depends\n",
      "  - stringr, type: imports\n",
      "  - methods, type: imports\n",
      "  - ggplot2 (>= 3.3.0) type: imports\n",
      "  - jsonlite, type: imports\n",
      "  - kernlab, type: imports\n",
      "  - rlist, type: imports\n",
      "  - rmarkdown, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - e1071, type: imports\n",
      "  - randomForest, type: imports\n",
      "  - caret, type: imports\n",
      "  - XML, type: imports\n",
      "  - praznik, type: imports\n",
      "  - R.utils, type: imports\n",
      "  - httr, type: imports\n",
      "  - sva, type: imports\n",
      "  - edgeR, type: imports\n",
      "  - limma, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - Hmisc (>= 4.4.0) type: imports\n",
      "  - gridExtra, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>=2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/KnowSeq.html\n",
      "\n",
      "Name: LRBaseDbi\n",
      "Description:\n",
      "Interface to construct LRBase package (LRBase.XXX.eg.db).\n",
      "Version: 2.8.0\n",
      "Publication date: None\n",
      "Mantainer: Koki Tsuyuzaki <k.t.the-answer@hotmail.co.jp>\n",
      "Authors:\n",
      "Author: Koki Tsuyuzaki\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - AnnotationDbi, type: imports\n",
      "  - RSQLite, type: imports\n",
      "  - DBI, type: imports\n",
      "  - Biobase, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/LRBaseDbi.html\n",
      "\n",
      "Name: maigesPack\n",
      "Description:\n",
      "This package uses functions of various other packages together with other functions in a coordinated way to handle and analyse cDNA microarray data\n",
      "Version: 1.62.0\n",
      "Publication date: None\n",
      "Mantainer: Gustavo H. Esteves <gesteves@gmail.com>\n",
      "Authors:\n",
      "Author: Gustavo H. Esteves <gesteves at gmail.com>, with contributions from Roberto Hirata Jr <hirata at ime.usp.br>, E. Jordao Neves <neves at ime.usp.br>, Elier B. Cristo <elier at ime.usp.br>, Ana C. Simoes <anakqui at ime.usp.br> and Lucas Fahham <fahham at linux.ime.usp.br>\n",
      "Dependencies:\n",
      "  - R (>= 2.10) type: depends\n",
      "  - convert, type: depends\n",
      "  - graph, type: depends\n",
      "  - limma, type: depends\n",
      "  - marray, type: depends\n",
      "  - methods, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/maigesPack.html\n",
      "\n",
      "Name: miRLAB\n",
      "Description:\n",
      "Provide tools exploring miRNA-mRNA relationships, including popular miRNA target prediction methods, ensemble methods that integrate individual methods, functions to get data from online resources, functions to validate the results, and functions to conduct enrichment analyses.\n",
      "Version: 1.28.0\n",
      "Publication date: None\n",
      "Mantainer: Thuc Duy Le <Thuc.Le@unisa.edu.au>\n",
      "Authors:\n",
      "Author: Thuc Duy Le, Junpeng Zhang, Mo Chen, Vu Viet Hoang Pham\n",
      "Dependencies:\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - RCurl, type: imports\n",
      "  - httr, type: imports\n",
      "  - stringr, type: imports\n",
      "  - Hmisc, type: imports\n",
      "  - energy, type: imports\n",
      "  - entropy, type: imports\n",
      "  - gplots, type: imports\n",
      "  - glmnet, type: imports\n",
      "  - impute, type: imports\n",
      "  - limma, type: imports\n",
      "  - pcalg, type: imports\n",
      "  - TCGAbiolinks, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - ctc, type: imports\n",
      "  - InvariantCausalPrediction, type: imports\n",
      "  - Category, type: imports\n",
      "  - GOstats, type: imports\n",
      "  - org.Hs.eg.db, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>=2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/miRLAB.html\n",
      "\n",
      "Name: miRNAmeConverter\n",
      "Description:\n",
      "Translating mature miRNA names to different miRBase versions, sequence retrieval, checking names for validity and detecting miRBase version of a given set of names (data from http://www.mirbase.org/).\n",
      "Version: 1.26.0\n",
      "Publication date: None\n",
      "Mantainer: Stefan J. Haunsberger <stefan.haunsberger@gmail.com>\n",
      "Authors:\n",
      "Author: Stefan Haunsberger [aut, cre]\n",
      "Dependencies:\n",
      "  - miRBaseVersions.db, type: depends\n",
      "  - DBI, type: imports\n",
      "  - AnnotationDbi, type: imports\n",
      "  - reshape2, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/miRNAmeConverter.html\n",
      "\n",
      "Name: miRNApath\n",
      "Description:\n",
      "This package provides pathway enrichment techniques for miRNA expression data. Specifically, the set of methods handles the many-to-many relationship between miRNAs and the multiple genes they are predicted to target (and thus affect.) It also handles the gene-to-pathway relationships separately. Both steps are designed to preserve the additive effects of miRNAs on genes, many miRNAs affecting one gene, one miRNA affecting multiple genes, or many miRNAs affecting many genes.\n",
      "Version: 1.58.0\n",
      "Publication date: None\n",
      "Mantainer: James M. Ward <jmw86069@gmail.com>\n",
      "Authors:\n",
      "Author: James M. Ward <jmw86069 at gmail.com> with contributions from Yunling Shi, Cindy Richards, John P. Cogswell\n",
      "Dependencies:\n",
      "  - methods, type: depends\n",
      "  - R (>= 2.7.0) type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL-2.1\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/miRNApath.html\n",
      "\n",
      "Name: MODA\n",
      "Description:\n",
      "MODA can be used to estimate and construct condition-specific gene co-expression networks, and identify differentially expressed subnetworks as conserved or condition specific modules which are potentially associated with relevant biological processes.\n",
      "Version: 1.24.0\n",
      "Publication date: None\n",
      "Mantainer: Dong Li <dxl466@cs.bham.ac.uk>\n",
      "Authors:\n",
      "Author: Dong Li, James B. Brown, Luisa Orsini, Zhisong Pan, Guyu Hu and Shan He\n",
      "Dependencies:\n",
      "  - R (>= 3.3) type: depends\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - WGCNA, type: imports\n",
      "  - dynamicTreeCut, type: imports\n",
      "  - igraph, type: imports\n",
      "  - cluster, type: imports\n",
      "  - AMOUNTAIN, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/MODA.html\n",
      "\n",
      "Name: nullranges\n",
      "Description:\n",
      "Modular package for generation of sets of ranges representing the null hypothesis. These can take the form of bootstrap samples of ranges (using the block bootstrap framework of Bickel et al 2010), or sets of control ranges that are matched across one or more covariates. nullranges is designed to be inter-operable with other packages for analysis of genomic overlap enrichment, including the plyranges Bioconductor package.\n",
      "Version: 1.4.0\n",
      "Publication date: None\n",
      "Mantainer: Michael Love <michaelisaiahlove@gmail.com>\n",
      "Authors:\n",
      "Author: Michael Love [aut, cre] , Wancen Mu [aut] , Eric Davis [aut] , Douglas Phanstiel [aut] , Stuart Lee [aut] , Mikhail Dozmorov [ctb], Tim Triche [ctb], CZI [fnd]\n",
      "Dependencies:\n",
      "  - stats, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - methods, type: imports\n",
      "  - rlang, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - scales, type: imports\n",
      "  - InteractionSet, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - plyranges, type: imports\n",
      "  - ks, type: imports\n",
      "  - speedglm, type: imports\n",
      "  - data.table, type: imports\n",
      "  - progress, type: imports\n",
      "  - ggridges, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/nullranges.html\n",
      "\n",
      "Name: OLIN\n",
      "Description:\n",
      "Functions for normalisation of two-color microarrays by optimised local regression and for detection of artefacts in microarray data\n",
      "Version: 1.76.0\n",
      "Publication date: None\n",
      "Mantainer: Matthias Futschik <mfutschik@ualg.pt>\n",
      "Authors:\n",
      "Author: Matthias Futschik <mfutschik at ualg.pt>\n",
      "Dependencies:\n",
      "  - R (>= 2.10) type: depends\n",
      "  - methods, type: depends\n",
      "  - locfit, type: depends\n",
      "  - marray, type: depends\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - limma, type: imports\n",
      "  - marray, type: imports\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/OLIN.html\n",
      "\n",
      "Name: parody\n",
      "Description:\n",
      "Provide routines for univariate and multivariate outlier detection with a focus on parametric methods, but support for some methods based on resistant statistics.\n",
      "Version: 1.56.0\n",
      "Publication date: None\n",
      "Mantainer: Vince Carey <stvjc@channing.harvard.edu>\n",
      "Authors:\n",
      "Author: Vince Carey [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - tools, type: depends\n",
      "  - utils, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/parody.html\n",
      "\n",
      "Name: proActiv\n",
      "Description:\n",
      "Most human genes have multiple promoters that control the expression of different isoforms. The use of these alternative promoters enables the regulation of isoform expression pre-transcriptionally. Alternative promoters have been found to be important in a wide number of cell types and diseases. proActiv is an R package that enables the analysis of promoters from RNA-seq data. proActiv uses aligned reads as input, and generates counts and normalized promoter activity estimates for each annotated promoter. In particular, proActiv accepts junction files from TopHat2 or STAR or BAM files as inputs. These estimates can then be used to identify which promoter is active, which promoter is inactive, and which promoters change their activity across conditions. proActiv also allows visualization of promoter activity across conditions.\n",
      "Version: 1.8.0\n",
      "Publication date: None\n",
      "Mantainer: Joseph Lee <joseph.lee@u.nus.edu>\n",
      "Authors:\n",
      "Author: Deniz Demircioglu [aut] , Jonathan GÃ¶ke [aut], Joseph Lee [cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.0.0) type: depends\n",
      "  - AnnotationDbi, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - data.table, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - DESeq2, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - GenomicFeatures, type: imports\n",
      "  - GenomicAlignments, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - gplots, type: imports\n",
      "  - graphics, type: imports\n",
      "  - methods, type: imports\n",
      "  - rlang, type: imports\n",
      "  - scales, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - stats, type: imports\n",
      "  - tibble, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/proActiv.html\n",
      "\n",
      "Name: RUVcorr\n",
      "Description:\n",
      "RUVcorr allows to apply global removal of unwanted variation (ridged version of RUV) to real and simulated gene expression data.\n",
      "Version: 1.30.0\n",
      "Publication date: None\n",
      "Mantainer: Saskia Freytag <saskia.freytag@perkins.uwa.edu.au>\n",
      "Authors:\n",
      "Author: Saskia Freytag\n",
      "Dependencies:\n",
      "  - corrplot, type: imports\n",
      "  - MASS, type: imports\n",
      "  - stats, type: imports\n",
      "  - lattice, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - gridExtra, type: imports\n",
      "  - snowfall, type: imports\n",
      "  - psych, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - grid, type: imports\n",
      "  - bladderbatch, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - graphics, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/RUVcorr.html\n",
      "\n",
      "Name: shinyepico\n",
      "Description:\n",
      "ShinyÃPICo is a graphical pipeline to analyze Illumina DNA methylation arrays (450k or EPIC). It allows to calculate differentially methylated positions and differentially methylated regions in a user-friendly interface. Moreover, it includes several options to export the results and obtain files to perform downstream analysis.\n",
      "Version: 1.6.0\n",
      "Publication date: None\n",
      "Mantainer: Octavio Morante-Palacios <octaviompa@gmail.com>\n",
      "Authors:\n",
      "Author: Octavio Morante-Palacios [cre, aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.2.0) type: depends\n",
      "  - DT (>= 0.15.0) type: imports\n",
      "  - data.table (>= 1.13.0) type: imports\n",
      "  - doParallel (>= 1.0.0) type: imports\n",
      "  - dplyr (>= 1.0.9) type: imports\n",
      "  - foreach (>= 1.5.0) type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - ggplot2 (>= 3.3.0) type: imports\n",
      "  - gplots (>= 3.0.0) type: imports\n",
      "  - heatmaply (>= 1.1.0) type: imports\n",
      "  - limma, type: imports\n",
      "  - minfi, type: imports\n",
      "  - plotly (>= 4.9.2) type: imports\n",
      "  - reshape2 (>= 1.4.0) type: imports\n",
      "  - rlang (>= 1.0.2) type: imports\n",
      "  - rmarkdown (>= 2.3.0) type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - shiny (>= 1.5.0) type: imports\n",
      "  - shinyWidgets (>= 0.5.0) type: imports\n",
      "  - shinycssloaders (>= 0.3.0) type: imports\n",
      "  - shinyjs (>= 1.1.0) type: imports\n",
      "  - shinythemes (>= 1.1.0) type: imports\n",
      "  - statmod (>= 1.4.0) type: imports\n",
      "  - tidyr (>= 1.2.0) type: imports\n",
      "  - zip (>= 2.1.0) type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "AGPL-3 + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/shinyepico.html\n",
      "\n",
      "Name: survtype\n",
      "Description:\n",
      "Subtypes are defined as groups of samples that have distinct molecular and clinical features. Genomic data can be analyzed for discovering patient subtypes, associated with clinical data, especially for survival information. This package is aimed to identify subtypes that are both clinically relevant and biologically meaningful.\n",
      "Version: 1.14.0\n",
      "Publication date: None\n",
      "Mantainer: Dongmin Jung <dmdmjung@gmail.com>\n",
      "Authors:\n",
      "Author: Dongmin Jung\n",
      "Dependencies:\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - pheatmap, type: depends\n",
      "  - survival, type: depends\n",
      "  - survminer, type: depends\n",
      "  - clustvarsel, type: depends\n",
      "  - stats, type: depends\n",
      "  - utils, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/survtype.html\n",
      "\n",
      "Name: TAPseq\n",
      "Description:\n",
      "Design primers for targeted single-cell RNA-seq used by TAP-seq. Create sequence templates for target gene panels and design gene-specific primers using Primer3. Potential off-targets can be estimated with BLAST. Requires working installations of Primer3 and BLASTn.\n",
      "Version: 1.10.0\n",
      "Publication date: None\n",
      "Mantainer: Andreas R. Gschwind <andreas.gschwind@stanford.edu>\n",
      "Authors:\n",
      "Author: Andreas R. Gschwind [aut, cre] , Lars Velten [aut] , Lars M. Steinmetz [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.0.0) type: depends\n",
      "  - methods, type: imports\n",
      "  - GenomicAlignments, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - BSgenome, type: imports\n",
      "  - GenomicFeatures, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/TAPseq.html\n",
      "\n",
      "Name: tRNAdbImport\n",
      "Description:\n",
      "tRNAdbImport imports the entries of the tRNAdb and mtRNAdb (http://trna.bioinf.uni-leipzig.de) as GRanges object.\n",
      "Version: 1.16.0\n",
      "Publication date: None\n",
      "Mantainer: Felix G.M. Ernst <felix.gm.ernst@outlook.com>\n",
      "Authors:\n",
      "Author: Felix G.M. Ernst [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 3.5) type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - Modstrings, type: depends\n",
      "  - Structstrings, type: depends\n",
      "  - tRNA, type: depends\n",
      "  - Biostrings, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - stringr, type: imports\n",
      "  - xml2, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - methods, type: imports\n",
      "  - httr, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3 + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/tRNAdbImport.html\n",
      "\n",
      "Name: VariantFiltering\n",
      "Description:\n",
      "Filter genetic variants using different criteria such as inheritance model, amino acid change consequence, minor allele frequencies across human populations, splice site strength, conservation, etc.\n",
      "Version: 1.34.0\n",
      "Publication date: None\n",
      "Mantainer: Robert Castelo <robert.castelo@upf.edu>\n",
      "Authors:\n",
      "Author: Robert Castelo [aut, cre], Dei Martinez Elurbe [ctb], Pau Puigdevall [ctb], Joan Fernandez [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - BiocGenerics, type: depends\n",
      "  - VariantAnnotation, type: depends\n",
      "  - utils, type: imports\n",
      "  - stats, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - RBGL, type: imports\n",
      "  - graph, type: imports\n",
      "  - AnnotationDbi, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - GenomicFeatures, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - BSgenome, type: imports\n",
      "  - GenomicScores, type: imports\n",
      "  - Gviz, type: imports\n",
      "  - shiny, type: imports\n",
      "  - shinythemes, type: imports\n",
      "  - shinyjs, type: imports\n",
      "  - DT, type: imports\n",
      "  - shinyTree, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/VariantFiltering.html\n",
      "\n",
      "Name: ADaCGH2\n",
      "Description:\n",
      "Analysis and plotting of array CGH data. Allows usage of Circular Binary Segementation, wavelet-based smoothing (both as in Liu et al., and HaarSeg as in Ben-Yaacov and Eldar), HMM, BioHMM, GLAD, CGHseg. Most computations are parallelized (either via forking or with clusters, including MPI and sockets clusters) and use ff for storing data.\n",
      "Version: 2.38.0\n",
      "Publication date: None\n",
      "Mantainer: Ramon Diaz-Uriarte <rdiaz02@gmail.com>\n",
      "Authors:\n",
      "Author: Ramon Diaz-Uriarte <rdiaz02 at gmail.com> and Oscar M. Rueda <rueda.om at gmail.com>. Wavelet-based aCGH smoothing code from Li Hsu <lih at fhcrc.org> and Douglas Grove <dgrove at fhcrc.org>. Imagemap code from Barry Rowlingson <B.Rowlingson at lancaster.ac.uk>. HaarSeg code from Erez Ben-Yaacov; downloaded from <http://www.ee.technion.ac.il/people/YoninaEldar/Info/software/HaarSeg.htm>. Code from ffbase <https://github.com/edwindj/ffbase> by Edwin de Jonge <edwindjonge at gmail.com>, Jan Wijffels, Jan van der Laan.\n",
      "Dependencies:\n",
      "  - R (>= 3.2.0) type: depends\n",
      "  - parallel, type: depends\n",
      "  - ff, type: depends\n",
      "  - GLAD, type: depends\n",
      "  - bit, type: imports\n",
      "  - DNAcopy, type: imports\n",
      "  - tilingArray, type: imports\n",
      "  - waveslim, type: imports\n",
      "  - cluster, type: imports\n",
      "  - aCGH, type: imports\n",
      "  - snapCGH, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 3)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ADaCGH2.html\n",
      "\n",
      "Name: BadRegionFinder\n",
      "Description:\n",
      "BadRegionFinder is a package for identifying regions with a bad, acceptable and good coverage in sequence alignment data available as bam files. The whole genome may be considered as well as a set of target regions. Various visual and textual types of output are available.\n",
      "Version: 1.26.0\n",
      "Publication date: None\n",
      "Mantainer: Sarah Sandmann <sarah.sandmann@uni-muenster.de>\n",
      "Authors:\n",
      "Author: Sarah Sandmann\n",
      "Dependencies:\n",
      "  - VariantAnnotation, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - biomaRt, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - utils, type: imports\n",
      "  - stats, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/BadRegionFinder.html\n",
      "\n",
      "Name: BAGS\n",
      "Description:\n",
      "R package providing functions to perform geneset significance analysis over simple cross-sectional data between 2 and 5 phenotypes of interest.\n",
      "Version: 2.38.0\n",
      "Publication date: None\n",
      "Mantainer: Alejandro Quiroz-Zarate <aquiroz@jimmy.harvard.edu>\n",
      "Authors:\n",
      "Author: Alejandro Quiroz-Zarate\n",
      "Dependencies:\n",
      "  - R (>= 2.10) type: depends\n",
      "  - breastCancerVDX, type: depends\n",
      "  - Biobase, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/BAGS.html\n",
      "\n",
      "Name: BasicSTARRseq\n",
      "Description:\n",
      "Basic peak calling on STARR-seq data based on a method introduced in \"Genome-Wide Quantitative Enhancer Activity Maps Identified by STARR-seq\" Arnold et al. Science. 2013 Mar 1;339(6123):1074-7. doi: 10.1126/science. 1232542. Epub 2013 Jan 17.\n",
      "Version: 1.26.0\n",
      "Publication date: None\n",
      "Mantainer: Annika Buerger <annika.buerger@ukmuenster.de>\n",
      "Authors:\n",
      "Author: Annika Buerger\n",
      "Dependencies:\n",
      "  - GenomicRanges, type: depends\n",
      "  - GenomicAlignments, type: depends\n",
      "  - S4Vectors, type: imports\n",
      "  - methods, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - stats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/BasicSTARRseq.html\n",
      "\n",
      "Name: blacksheepr\n",
      "Description:\n",
      "Blacksheep is a tool designed for outlier analysis in the context of pairwise comparisons in an effort to find distinguishing characteristics from two groups. This tool was designed to be applied for biological applications such as phosphoproteomics or transcriptomics, but it can be used for any data that can be represented by a 2D table, and has two sub populations within the table to compare.\n",
      "Version: 1.12.0\n",
      "Publication date: None\n",
      "Mantainer: RugglesLab <ruggleslab@gmail.com>\n",
      "Authors:\n",
      "Author: MacIntosh Cornwell [aut], RugglesLab [cre]\n",
      "Dependencies:\n",
      "  - R (>= 3.6) type: depends\n",
      "  - grid, type: imports\n",
      "  - stats, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - utils, type: imports\n",
      "  - circlize, type: imports\n",
      "  - viridis, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - ComplexHeatmap, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - pasilla, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/blacksheepr.html\n",
      "\n",
      "Name: blima\n",
      "Description:\n",
      "Package blima includes several algorithms for the preprocessing of Illumina microarray data. It focuses to the bead level analysis and provides novel approach to the quantile normalization of the vectors of unequal lengths. It provides variety of the methods for background correction including background subtraction, RMA like convolution and background outlier removal. It also implements variance stabilizing transformation on the bead level. There are also implemented methods for data summarization. It also provides the methods for performing T-tests on the detector (bead) level and on the probe level for differential expression testing.\n",
      "Version: 1.32.0\n",
      "Publication date: None\n",
      "Mantainer: VojtÄch Kulvait <kulvait@gmail.com>\n",
      "Authors:\n",
      "Author: VojtÄch Kulvait\n",
      "Dependencies:\n",
      "  - R (>= 3.3) type: depends\n",
      "  - beadarray, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - Rcpp (>= 0.12.8) type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - stats, type: imports\n",
      "  - graphics, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/blima.html\n",
      "\n",
      "Name: BLMA\n",
      "Description:\n",
      "Suit of tools for bi-level meta-analysis. The package can be used in a wide range of applications, including general hypothesis testings, differential expression analysis, functional analysis, and pathway analysis.\n",
      "Version: 1.22.0\n",
      "Publication date: None\n",
      "Mantainer: Hung Nguyen <hungnp@nevada.unr.edu>\n",
      "Authors:\n",
      "Author: Tin Nguyen <tinn at unr.edu>, Hung Nguyen <hungnp at nevada.unr.edu>, and Sorin Draghici <sorin at wayne.edu>\n",
      "Dependencies:\n",
      "  - ROntoTools, type: depends\n",
      "  - GSA, type: depends\n",
      "  - PADOG, type: depends\n",
      "  - limma, type: depends\n",
      "  - graph, type: depends\n",
      "  - stats, type: depends\n",
      "  - utils, type: depends\n",
      "  - parallel, type: depends\n",
      "  - Biobase, type: depends\n",
      "  - metafor, type: depends\n",
      "  - methods, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>=2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/BLMA.html\n",
      "\n",
      "Name: bnbc\n",
      "Description:\n",
      "Tools to normalize (several) Hi-C data from replicates.\n",
      "Version: 1.20.0\n",
      "Publication date: None\n",
      "Mantainer: Kipper Fletez-Brant <cafletezbrant@gmail.com>\n",
      "Authors:\n",
      "Author: Kipper Fletez-Brant [cre, aut], Kasper Daniel Hansen [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - BiocGenerics, type: depends\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - Rcpp (>= 0.12.12) type: imports\n",
      "  - IRanges, type: imports\n",
      "  - rhdf5, type: imports\n",
      "  - data.table, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - preprocessCore, type: imports\n",
      "  - sva, type: imports\n",
      "  - parallel, type: imports\n",
      "  - EBImage, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/bnbc.html\n",
      "\n",
      "Name: BubbleTree\n",
      "Description:\n",
      "CNV analysis in groups of tumor samples.\n",
      "Version: 2.28.0\n",
      "Publication date: None\n",
      "Mantainer: Todd Creasy <creasyt@medimmune.com>, Wei Zhu <weizhu365@gmail.com>\n",
      "Authors:\n",
      "Author: Wei Zhu <zhuw at medimmune.com>, Michael Kuziora <kuzioram at medimmune.com>, Todd Creasy <creasyt at medimmune.com>, Brandon Higgs <higgsb at medimmune.com>\n",
      "Dependencies:\n",
      "  - R (>= 3.5) type: depends\n",
      "  - IRanges, type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - plyr, type: depends\n",
      "  - dplyr, type: depends\n",
      "  - magrittr, type: depends\n",
      "  - BiocGenerics, type: imports\n",
      "  - BiocStyle, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - WriteXLS, type: imports\n",
      "  - gtools, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - limma, type: imports\n",
      "  - grid, type: imports\n",
      "  - gtable, type: imports\n",
      "  - gridExtra, type: imports\n",
      "  - biovizBase, type: imports\n",
      "  - e1071, type: imports\n",
      "  - methods, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL (>= 3)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/BubbleTree.html\n",
      "\n",
      "Name: cellity\n",
      "Description:\n",
      "A support vector machine approach to identifying and filtering low quality cells from single-cell RNA-seq datasets.\n",
      "Version: 1.26.0\n",
      "Publication date: None\n",
      "Mantainer: Tomislav Ilicic <ti243@cam.ac.uk>\n",
      "Authors:\n",
      "Author: Tomislav Illicic, Davis McCarthy\n",
      "Dependencies:\n",
      "  - R (>= 3.3) type: depends\n",
      "  - AnnotationDbi, type: imports\n",
      "  - e1071, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - grid, type: imports\n",
      "  - mvoutlier, type: imports\n",
      "  - org.Hs.eg.db, type: imports\n",
      "  - org.Mm.eg.db, type: imports\n",
      "  - robustbase, type: imports\n",
      "  - stats, type: imports\n",
      "  - topGO, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/cellity.html\n",
      "\n",
      "Name: ChIPComp\n",
      "Description:\n",
      "ChIPComp detects differentially bound sharp binding sites across multiple conditions considering matching control.\n",
      "Version: 1.28.0\n",
      "Publication date: None\n",
      "Mantainer: Li Chen <li.chen@emory.edu>\n",
      "Authors:\n",
      "Author: Hao Wu, Li Chen, Zhaohui S.Qin, Chi Wang\n",
      "Dependencies:\n",
      "  - R (>= 3.2.0) type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - IRanges, type: depends\n",
      "  - rtracklayer, type: depends\n",
      "  - GenomeInfoDb, type: depends\n",
      "  - S4Vectors, type: depends\n",
      "  - Rsamtools, type: imports\n",
      "  - limma, type: imports\n",
      "  - BSgenome.Hsapiens.UCSC.hg19, type: imports\n",
      "  - BSgenome.Mmusculus.UCSC.mm9, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ChIPComp.html\n",
      "\n",
      "Name: coGPS\n",
      "Description:\n",
      "Gene Set Enrichment Analysis of P-value based statistics for outlier gene detection in dataset merged from multiple studies\n",
      "Version: 1.42.0\n",
      "Publication date: None\n",
      "Mantainer: Yingying Wei <ywei@jhsph.edu>\n",
      "Authors:\n",
      "Author: Yingying Wei, Michael Ochs\n",
      "Dependencies:\n",
      "  - R (>= 2.13.0) type: depends\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/coGPS.html\n",
      "\n",
      "Name: COHCAP\n",
      "Description:\n",
      "COHCAP (pronounced \"co-cap\") provides a pipeline to analyze single-nucleotide resolution methylation data (Illumina 450k/EPIC methylation array, targeted BS-Seq, etc.). It provides differential methylation for CpG Sites, differential methylation for CpG Islands, integration with gene expression data, with visualizaton options. Discussion Group: https://sourceforge.net/p/cohcap/discussion/bioconductor/\n",
      "Version: 1.44.0\n",
      "Publication date: None\n",
      "Mantainer: Charles Warden <cwarden@coh.org>\n",
      "Authors:\n",
      "Author: Charles Warden <cwarden at coh.org>, Yate-Ching Yuan <yyuan at coh.org>, Xiwei Wu <xwu at coh.org>\n",
      "Dependencies:\n",
      "  - WriteXLS, type: depends\n",
      "  - COHCAPanno, type: depends\n",
      "  - RColorBrewer, type: depends\n",
      "  - gplots, type: depends\n",
      "  - Rcpp, type: imports\n",
      "  - RcppArmadillo, type: imports\n",
      "  - BH, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/COHCAP.html\n",
      "\n",
      "Name: ctsGE\n",
      "Description:\n",
      "Methodology for supervised clustering of potentially many predictor variables, such as genes etc., in time series datasets Provides functions that help the user assigning genes to predefined set of model profiles.\n",
      "Version: 1.24.0\n",
      "Publication date: None\n",
      "Mantainer: Michal Sharabi-Schwager <michalsharabi@gmail.com>\n",
      "Authors:\n",
      "Author: Michal Sharabi-Schwager [aut, cre], Ron Ophir [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.2) type: depends\n",
      "  - ccaPP, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - limma, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - shiny, type: imports\n",
      "  - stats, type: imports\n",
      "  - stringr, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ctsGE.html\n",
      "\n",
      "Name: CytoDx\n",
      "Description:\n",
      "This package provides functions that predict clinical outcomes using single cell data (such as flow cytometry data, RNA single cell sequencing data) without the requirement of cell gating or clustering.\n",
      "Version: 1.18.0\n",
      "Publication date: None\n",
      "Mantainer: Zicheng Hu <zicheng.hu@ucsf.edu>\n",
      "Authors:\n",
      "Author: Zicheng Hu\n",
      "Dependencies:\n",
      "  - R (>= 3.5) type: depends\n",
      "  - doParallel, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - glmnet, type: imports\n",
      "  - rpart, type: imports\n",
      "  - rpart.plot, type: imports\n",
      "  - stats, type: imports\n",
      "  - flowCore, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/CytoDx.html\n",
      "\n",
      "Name: daMA\n",
      "Description:\n",
      "This package contains functions for the efficient design of factorial two-colour microarray experiments and for the statistical analysis of factorial microarray data. Statistical details are described in Bretz et al. (2003, submitted)\n",
      "Version: 1.70.0\n",
      "Publication date: None\n",
      "Mantainer: Jobst Landgrebe <jlandgr1@gwdg.de>\n",
      "Authors:\n",
      "Author: Jobst Landgrebe <jlandgr1 at gwdg.de> and Frank Bretz <bretz at bioinf.uni-hannover.de>\n",
      "Dependencies:\n",
      "  - MASS, type: imports\n",
      "  - stats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/daMA.html\n",
      "\n",
      "Name: dasper\n",
      "Description:\n",
      "The aim of dasper is to detect aberrant splicing events from RNA-seq data. dasper will use as input both junction and coverage data from RNA-seq to calculate the deviation of each splicing event in a patient from a set of user-defined controls. dasper uses an unsupervised outlier detection algorithm to score each splicing event in the patient with an outlier score representing the degree to which that splicing event looks abnormal.\n",
      "Version: 1.7.0\n",
      "Publication date: None\n",
      "Mantainer: David Zhang <david.zhang.12@ucl.ac.uk>\n",
      "Authors:\n",
      "Author: David Zhang [aut, cre] , Leonardo Collado-Torres [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - basilisk, type: imports\n",
      "  - BiocFileCache, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - data.table, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - GenomicFeatures, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - ggpubr, type: imports\n",
      "  - ggrepel, type: imports\n",
      "  - grid, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - megadepth, type: imports\n",
      "  - methods, type: imports\n",
      "  - plyranges, type: imports\n",
      "  - readr, type: imports\n",
      "  - reticulate, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - stringr, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - tidyr, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/dasper.html\n",
      "\n",
      "Name: DEGraph\n",
      "Description:\n",
      "DEGraph implements recent hypothesis testing methods which directly assess whether a particular gene network is differentially expressed between two conditions. This is to be contrasted with the more classical two-step approaches which first test individual genes, then test gene sets for enrichment in differentially expressed genes. These recent methods take into account the topology of the network to yield more powerful detection procedures. DEGraph provides methods to easily test all KEGG pathways for differential expression on any gene expression data set and tools to visualize the results.\n",
      "Version: 1.50.0\n",
      "Publication date: None\n",
      "Mantainer: Laurent Jacob <laurent.jacob@gmail.com>\n",
      "Authors:\n",
      "Author: Laurent Jacob, Pierre Neuvial and Sandrine Dudoit\n",
      "Dependencies:\n",
      "  - R (>= 2.10.0) type: depends\n",
      "  - R.utils, type: depends\n",
      "  - graph, type: imports\n",
      "  - KEGGgraph, type: imports\n",
      "  - lattice, type: imports\n",
      "  - mvtnorm, type: imports\n",
      "  - R.methodsS3, type: imports\n",
      "  - RBGL, type: imports\n",
      "  - Rgraphviz, type: imports\n",
      "  - rrcov, type: imports\n",
      "  - NCIgraph, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/DEGraph.html\n",
      "\n",
      "Name: DriverNet\n",
      "Description:\n",
      "DriverNet is a package to predict functional important driver genes in cancer by integrating genome data (mutation and copy number variation data) and transcriptome data (gene expression data). The different kinds of data are combined by an influence graph, which is a gene-gene interaction network deduced from pathway data. A greedy algorithm is used to find the possible driver genes, which may mutated in a larger number of patients and these mutations will push the gene expression values of the connected genes to some extreme values.\n",
      "Version: 1.38.0\n",
      "Publication date: None\n",
      "Mantainer: Jiarui Ding <jiaruid@cs.ubc.ca>\n",
      "Authors:\n",
      "Author: Ali Bashashati, Reza Haffari, Jiarui Ding, Gavin Ha, Kenneth Liu, Jamie Rosner and Sohrab Shah\n",
      "Dependencies:\n",
      "  - R (>= 2.10) type: depends\n",
      "  - methods, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/DriverNet.html\n",
      "\n",
      "Name: EpiTxDb\n",
      "Description:\n",
      "EpiTxDb facilitates the storage of epitranscriptomic information. More specifically, it can keep track of modification identity, position, the enzyme for introducing it on the RNA, a specifier which determines the position on the RNA to be modified and the literature references each modification is associated with.\n",
      "Version: 1.10.0\n",
      "Publication date: None\n",
      "Mantainer: Felix G.M. Ernst <felix.gm.ernst@outlook.com>\n",
      "Authors:\n",
      "Author: Felix G.M. Ernst [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - AnnotationDbi, type: depends\n",
      "  - Modstrings, type: depends\n",
      "  - methods, type: imports\n",
      "  - utils, type: imports\n",
      "  - httr, type: imports\n",
      "  - xml2, type: imports\n",
      "  - curl, type: imports\n",
      "  - GenomicFeatures, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - BiocFileCache, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - RSQLite, type: imports\n",
      "  - DBI, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - tRNAdbImport, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/EpiTxDb.html\n",
      "\n",
      "Name: garfield\n",
      "Description:\n",
      "GARFIELD is a non-parametric functional enrichment analysis approach described in the paper GARFIELD: GWAS analysis of regulatory or functional information enrichment with LD correction. Briefly, it is a method that leverages GWAS findings with regulatory or functional annotations (primarily from ENCODE and Roadmap epigenomics data) to find features relevant to a phenotype of interest. It performs greedy pruning of GWAS SNPs (LD r2 > 0.1) and then annotates them based on functional information overlap. Next, it quantifies Fold Enrichment (FE) at various GWAS significance cutoffs and assesses them by permutation testing, while matching for minor allele frequency, distance to nearest transcription start site and number of LD proxies (r2 > 0.8).\n",
      "Version: 1.26.0\n",
      "Publication date: None\n",
      "Mantainer: Valentina Iotchkova <vi1@sanger.ac.uk>\n",
      "Authors:\n",
      "Author: Sandro Morganella <sm22 at sanger.ac.uk>\n",
      "Dependencies:\n",
      "None\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/garfield.html\n",
      "\n",
      "Name: gCrisprTools\n",
      "Description:\n",
      "Set of tools for evaluating pooled high-throughput screening experiments, typically employing CRISPR/Cas9 or shRNA expression cassettes. Contains methods for interrogating library and cassette behavior within an experiment, identifying differentially abundant cassettes, aggregating signals to identify candidate targets for empirical validation, hypothesis testing, and comprehensive reporting. Version 2.0 extends these applications to include a variety of tools for contextualizing and integrating signals across many experiments, incorporates extended signal enrichment methodologies via the \"sparrow\" package, and streamlines many formal requirements to aid in interpretablity.\n",
      "Version: 2.4.0\n",
      "Publication date: None\n",
      "Mantainer: Russell Bainer <russ.bainer@gmail.com>\n",
      "Authors:\n",
      "Author: Russell Bainer, Dariusz Ratman, Steve Lianoglou, Peter Haverty\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - Biobase, type: imports\n",
      "  - limma, type: imports\n",
      "  - RobustRankAggreg, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - grid, type: imports\n",
      "  - rmarkdown, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - methods, type: imports\n",
      "  - ComplexHeatmap, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - parallel, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/gCrisprTools.html\n",
      "\n",
      "Name: GenomicSuperSignature\n",
      "Description:\n",
      "This package provides a novel method for interpreting new transcriptomic datasets through near-instantaneous comparison to public archives without high-performance computing requirements. Through the pre-computed index, users can identify public resources associated with their dataset such as gene sets, MeSH term, and publication. Functions to identify interpretable annotations and intuitive visualization options are implemented in this package.\n",
      "Version: 1.6.0\n",
      "Publication date: None\n",
      "Mantainer: Sehyun Oh <shbrief@gmail.com>\n",
      "Authors:\n",
      "Author: Sehyun Oh [aut, cre], Levi Waldron [aut], Sean Davis [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.1.0) type: depends\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - ComplexHeatmap, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - methods, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - ggpubr, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - plotly, type: imports\n",
      "  - BiocFileCache, type: imports\n",
      "  - grid, type: imports\n",
      "  - flextable, type: imports\n",
      "  - irlba, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/GenomicSuperSignature.html\n",
      "\n",
      "Name: getDEE2\n",
      "Description:\n",
      "Digital Expression Explorer 2 (or DEE2 for short) is a repository of processed RNA-seq data in the form of counts. It was designed so that researchers could undertake re-analysis and meta-analysis of published RNA-seq studies quickly and easily. As of April 2020, over 1 million SRA datasets have been processed. This package provides an R interface to access these expression data. More information about the DEE2 project can be found at the project homepage (http://dee2.io) and main publication (https://doi.org/10.1093/gigascience/giz022).\n",
      "Version: 1.8.0\n",
      "Publication date: None\n",
      "Mantainer: Mark Ziemann <mark.ziemann@gmail.com>\n",
      "Authors:\n",
      "Author: Mark Ziemann [aut, cre], Antony Kaspi [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - htm2txt, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/getDEE2.html\n",
      "\n",
      "Name: GGPA\n",
      "Description:\n",
      "Genome-wide association studies (GWAS) is a widely used tool for identification of genetic variants associated with phenotypes and diseases, though complex diseases featuring many genetic variants with small effects present difficulties for traditional these studies. By leveraging pleiotropy, the statistical power of a single GWAS can be increased. This package provides functions for fitting graph-GPA, a statistical framework to prioritize GWAS results by integrating pleiotropy. 'GGPA' package provides user-friendly interface to fit graph-GPA models, implement association mapping, and generate a phenotype graph.\n",
      "Version: 1.10.2\n",
      "Publication date: None\n",
      "Mantainer: Dongjun Chung <dongjun.chung@gmail.com>\n",
      "Authors:\n",
      "Author: Dongjun Chung, Hang J. Kim, Carter Allen\n",
      "Dependencies:\n",
      "  - R (>= 4.0.0) type: depends\n",
      "  - stats, type: depends\n",
      "  - methods, type: depends\n",
      "  - graphics, type: depends\n",
      "  - GGally, type: depends\n",
      "  - network, type: depends\n",
      "  - sna, type: depends\n",
      "  - scales, type: depends\n",
      "  - matrixStats, type: depends\n",
      "  - Rcpp (>= 0.11.3) type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/GGPA.html\n",
      "\n",
      "Name: GmicR\n",
      "Description:\n",
      "This package uses bayesian network learning to detect relationships between Gene Modules detected by WGCNA and immune cell signatures defined by xCell. It is a hypothesis generating tool.\n",
      "Version: 1.12.0\n",
      "Publication date: None\n",
      "Mantainer: Richard Virgen-Slane <RVS.BioTools@gmail.com>\n",
      "Authors:\n",
      "Author: Richard Virgen-Slane\n",
      "Dependencies:\n",
      "  - AnnotationDbi, type: imports\n",
      "  - ape, type: imports\n",
      "  - bnlearn, type: imports\n",
      "  - Category, type: imports\n",
      "  - DT, type: imports\n",
      "  - doParallel, type: imports\n",
      "  - foreach, type: imports\n",
      "  - gRbase, type: imports\n",
      "  - GSEABase, type: imports\n",
      "  - gRain, type: imports\n",
      "  - GOstats, type: imports\n",
      "  - org.Hs.eg.db, type: imports\n",
      "  - org.Mm.eg.db, type: imports\n",
      "  - shiny, type: imports\n",
      "  - WGCNA, type: imports\n",
      "  - data.table, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2 + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/GmicR.html\n",
      "\n",
      "Name: GOpro\n",
      "Description:\n",
      "Find the most characteristic gene ontology terms for groups of human genes. This package was created as a part of the thesis which was developed under the auspices of MI^2 Group (http://mi2.mini.pw.edu.pl/, https://github.com/geneticsMiNIng).\n",
      "Version: 1.24.0\n",
      "Publication date: None\n",
      "Mantainer: Lidia Chrabaszcz <chrabaszcz.lidia@gmail.com>\n",
      "Authors:\n",
      "Author: Lidia Chrabaszcz\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - AnnotationDbi, type: imports\n",
      "  - dendextend, type: imports\n",
      "  - doParallel, type: imports\n",
      "  - foreach, type: imports\n",
      "  - parallel, type: imports\n",
      "  - org.Hs.eg.db, type: imports\n",
      "  - GO.db, type: imports\n",
      "  - Rcpp, type: imports\n",
      "  - stats, type: imports\n",
      "  - graphics, type: imports\n",
      "  - MultiAssayExperiment, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/GOpro.html\n",
      "\n",
      "Name: idiogram\n",
      "Description:\n",
      "A package for plotting genomic data by chromosomal location\n",
      "Version: 1.74.0\n",
      "Publication date: None\n",
      "Mantainer: Karl J. Dykema <karl.dykema@vai.org>\n",
      "Authors:\n",
      "Author: Karl J. Dykema <karl.dykema at vai.org>\n",
      "Dependencies:\n",
      "  - R (>= 2.10) type: depends\n",
      "  - methods, type: depends\n",
      "  - Biobase, type: depends\n",
      "  - annotate, type: depends\n",
      "  - plotrix, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/idiogram.html\n",
      "\n",
      "Name: idr2d\n",
      "Description:\n",
      "A tool to measure reproducibility between genomic experiments that produce two-dimensional peaks (interactions between peaks), such as ChIA-PET, HiChIP, and HiC. idr2d is an extension of the original idr package, which is intended for (one-dimensional) ChIP-seq peaks.\n",
      "Version: 1.12.0\n",
      "Publication date: None\n",
      "Mantainer: Konstantin Krismer <krismer@mit.edu>\n",
      "Authors:\n",
      "Author: Konstantin Krismer [aut, cre, cph] , David Gifford [ths, cph]\n",
      "Dependencies:\n",
      "  - R (>= 3.6) type: depends\n",
      "  - dplyr (>= 0.7.6) type: imports\n",
      "  - futile.logger (>= 1.4.3) type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - ggplot2 (>= 3.1.1) type: imports\n",
      "  - grDevices, type: imports\n",
      "  - grid, type: imports\n",
      "  - idr (>= 1.2) type: imports\n",
      "  - IRanges, type: imports\n",
      "  - magrittr (>= 1.5) type: imports\n",
      "  - methods, type: imports\n",
      "  - reticulate (>= 1.13) type: imports\n",
      "  - scales (>= 1.0.0) type: imports\n",
      "  - stats, type: imports\n",
      "  - stringr (>= 1.3.1) type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/idr2d.html\n",
      "\n",
      "Name: IntEREst\n",
      "Description:\n",
      "This package performs Intron-Exon Retention analysis on RNA-seq data (.bam files).\n",
      "Version: 1.22.2\n",
      "Publication date: None\n",
      "Mantainer: Ali Oghabian <Ali.Oghabian@Helsinki.Fi>, Mikko Frilander <Mikko.Frilander@helsinki.fi>\n",
      "Authors:\n",
      "Author: Ali Oghabian <Ali.Oghabian at Helsinki.Fi>, Dario Greco <dario.greco at helsinki.fi>, Mikko Frilander <Mikko.Frilander at helsinki.fi>\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - Rsamtools, type: depends\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - edgeR, type: depends\n",
      "  - S4Vectors, type: depends\n",
      "  - seqLogo, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - GenomicFeatures, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - seqinr, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - grid, type: imports\n",
      "  - methods, type: imports\n",
      "  - DBI, type: imports\n",
      "  - RMySQL, type: imports\n",
      "  - GenomicAlignments, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - DEXSeq, type: imports\n",
      "  - DESeq2, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/IntEREst.html\n",
      "\n",
      "Name: MVCClass\n",
      "Description:\n",
      "Creates classes used in model-view-controller (MVC) design\n",
      "Version: 1.72.0\n",
      "Publication date: None\n",
      "Mantainer: Elizabeth Whalen <ewhalen@hsph.harvard.edu>\n",
      "Authors:\n",
      "Author: Elizabeth Whalen\n",
      "Dependencies:\n",
      "  - R (>= 2.1.0) type: depends\n",
      "  - methods, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/MVCClass.html\n",
      "\n",
      "Name: NCIgraph\n",
      "Description:\n",
      "Provides various methods to load the pathways from the NCI Pathways Database in R graph objects and to re-format them.\n",
      "Version: 1.46.0\n",
      "Publication date: None\n",
      "Mantainer: Laurent Jacob <laurent.jacob@gmail.com>\n",
      "Authors:\n",
      "Author: Laurent Jacob\n",
      "Dependencies:\n",
      "  - R (>= 2.10.0) type: depends\n",
      "  - graph, type: imports\n",
      "  - KEGGgraph, type: imports\n",
      "  - methods, type: imports\n",
      "  - RBGL, type: imports\n",
      "  - RCy3, type: imports\n",
      "  - R.methodsS3, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/NCIgraph.html\n",
      "\n",
      "Name: nethet\n",
      "Description:\n",
      "Package nethet is an implementation of statistical solid methodology enabling the analysis of network heterogeneity from high-dimensional data. It combines several implementations of recent statistical innovations useful for estimation and comparison of networks in a heterogeneous, high-dimensional setting. In particular, we provide code for formal two-sample testing in Gaussian graphical models (differential network and GGM-GSA; Stadler and Mukherjee, 2013, 2014) and make a novel network-based clustering algorithm available (mixed graphical lasso, Stadler and Mukherjee, 2013).\n",
      "Version: 1.30.0\n",
      "Publication date: None\n",
      "Mantainer: Nicolas Staedler <staedler.n@gmail.com>, Frank Dondelinger <fdondelinger.work@gmail.com>\n",
      "Authors:\n",
      "Author: Nicolas Staedler, Frank Dondelinger\n",
      "Dependencies:\n",
      "  - glasso, type: imports\n",
      "  - mvtnorm, type: imports\n",
      "  - GeneNet, type: imports\n",
      "  - huge, type: imports\n",
      "  - CompQuadForm, type: imports\n",
      "  - ggm, type: imports\n",
      "  - mclust, type: imports\n",
      "  - parallel, type: imports\n",
      "  - GSA, type: imports\n",
      "  - limma, type: imports\n",
      "  - multtest, type: imports\n",
      "  - ICSNP, type: imports\n",
      "  - glmnet, type: imports\n",
      "  - network, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/nethet.html\n",
      "\n",
      "Name: podkat\n",
      "Description:\n",
      "This package provides an association test that is capable of dealing with very rare and even private variants. This is accomplished by a kernel-based approach that takes the positions of the variants into account. The test can be used for pre-processed matrix data, but also directly for variant data stored in VCF files. Association testing can be performed whole-genome, whole-exome, or restricted to pre-defined regions of interest. The test is complemented by tools for analyzing and visualizing the results.\n",
      "Version: 1.30.0\n",
      "Publication date: None\n",
      "Mantainer: Ulrich Bodenhofer <bodenhofer@bioinf.jku.at>\n",
      "Authors:\n",
      "Author: Ulrich Bodenhofer\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - Rsamtools, type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - Rcpp (>= 0.11.1) type: imports\n",
      "  - parallel, type: imports\n",
      "  - stats, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - utils, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - BSgenome, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/podkat.html\n",
      "\n",
      "Name: pwOmics\n",
      "Description:\n",
      "pwOmics performs pathway-based level-specific data comparison of matching omics data sets based on pre-analysed user-specified lists of differential genes/transcripts and phosphoproteins. A separate downstream analysis of phosphoproteomic data including pathway identification, transcription factor identification and target gene identification is opposed to the upstream analysis starting with gene or transcript information as basis for identification of upstream transcription factors and potential proteomic regulators. The cross-platform comparative analysis allows for comprehensive analysis of single time point experiments and time-series experiments by providing static and dynamic analysis tools for data integration. In addition, it provides functions to identify individual signaling axes based on data integration.\n",
      "Version: 1.30.0\n",
      "Publication date: None\n",
      "Mantainer: Maren Sitte <Maren.Sitte@med.uni-goettingen.de>\n",
      "Authors:\n",
      "Author: Astrid Wachter <Astrid.Wachter at med.uni-goettingen.de>\n",
      "Dependencies:\n",
      "  - R (>= 3.2) type: depends\n",
      "  - data.table, type: imports\n",
      "  - rBiopaxParser, type: imports\n",
      "  - igraph, type: imports\n",
      "  - STRINGdb, type: imports\n",
      "  - graphics, type: imports\n",
      "  - gplots, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - AnnotationDbi, type: imports\n",
      "  - biomaRt, type: imports\n",
      "  - AnnotationHub, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - graph, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/pwOmics.html\n",
      "\n",
      "Name: RBM\n",
      "Description:\n",
      "Use A Resampling-Based Empirical Bayes Approach to Assess Differential Expression in Two-Color Microarrays and RNA-Seq data sets.\n",
      "Version: 1.30.0\n",
      "Publication date: None\n",
      "Mantainer: Dongmei Li <Dongmei_Li@urmc.rochester.edu>\n",
      "Authors:\n",
      "Author: Dongmei Li and Chin-Yuan Liang\n",
      "Dependencies:\n",
      "  - R (>= 3.2.0) type: depends\n",
      "  - limma, type: depends\n",
      "  - marray, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/RBM.html\n",
      "\n",
      "Name: RSVSim\n",
      "Description:\n",
      "RSVSim is a package for the simulation of deletions, insertions, inversion, tandem-duplications and translocations of various sizes in any genome available as FASTA-file or BSgenome data package. SV breakpoints can be placed uniformly accross the whole genome, with a bias towards repeat regions and regions of high homology (for hg19) or at user-supplied coordinates.\n",
      "Version: 1.38.0\n",
      "Publication date: None\n",
      "Mantainer: Christoph Bartenhagen <c.bartenhagen@uni-koeln.de>\n",
      "Authors:\n",
      "Author: Christoph Bartenhagen\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - Biostrings, type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - methods, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - ShortRead, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/RSVSim.html\n",
      "\n",
      "Name: RUVnormalize\n",
      "Description:\n",
      "RUVnormalize is meant to remove unwanted variation from gene expression data when the factor of interest is not defined, e.g., to clean up a dataset for general use or to do any kind of unsupervised analysis.\n",
      "Version: 1.32.0\n",
      "Publication date: None\n",
      "Mantainer: Laurent Jacob <laurent.jacob@univ-lyon1.fr>\n",
      "Authors:\n",
      "Author: Laurent Jacob\n",
      "Dependencies:\n",
      "  - R (>= 2.10.0) type: depends\n",
      "  - RUVnormalizeData, type: imports\n",
      "  - Biobase, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/RUVnormalize.html\n",
      "\n",
      "Name: sights\n",
      "Description:\n",
      "SIGHTS is a suite of normalization methods, statistical tests, and diagnostic graphical tools for high throughput screening (HTS) assays. HTS assays use microtitre plates to screen large libraries of compounds for their biological, chemical, or biochemical activity.\n",
      "Version: 1.24.0\n",
      "Publication date: None\n",
      "Mantainer: Elika Garg <elika.garg@mail.mcgill.ca>\n",
      "Authors:\n",
      "Author: Elika Garg [aut, cre], Carl Murie [aut], Heydar Ensha [ctb], Robert Nadon [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.3) type: depends\n",
      "  - MASS (>= 7.3) type: imports\n",
      "  - qvalue, type: imports\n",
      "  - ggplot2 (>= 2.0) type: imports\n",
      "  - reshape2 (>= 1.4) type: imports\n",
      "  - lattice (>= 0.2) type: imports\n",
      "  - stats (>= 3.3) type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3 | file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/sights.html\n",
      "\n",
      "Name: sizepower\n",
      "Description:\n",
      "This package has been prepared to assist users in computing either a sample size or power value for a microarray experimental study. The user is referred to the cited references for technical background on the methodology underpinning these calculations. This package provides support for five types of sample size and power calculations. These five types can be adapted in various ways to encompass many of the standard designs encountered in practice.\n",
      "Version: 1.68.0\n",
      "Publication date: None\n",
      "Mantainer: Weiliang Qiu <weiliang.qiu@gmail.com>\n",
      "Authors:\n",
      "Author: Weiliang Qiu <weiliang.qiu at gmail.com> and Mei-Ling Ting Lee <meilinglee at sph.osu.edu> and George Alex Whitmore <george.whitmore at mcgill.ca>\n",
      "Dependencies:\n",
      "  - stats, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/sizepower.html\n",
      "\n",
      "Name: SLqPCR\n",
      "Description:\n",
      "Functions for analysis of real-time quantitative PCR data at SIRS-Lab GmbH\n",
      "Version: 1.64.0\n",
      "Publication date: None\n",
      "Mantainer: Matthias Kohl <kohl@sirs-lab.com>\n",
      "Authors:\n",
      "Author: Matthias Kohl\n",
      "Dependencies:\n",
      "  - R (>= 2.4.0) type: depends\n",
      "  - stats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/SLqPCR.html\n",
      "\n",
      "Name: STATegRa\n",
      "Description:\n",
      "Classes and tools for multi-omics data integration.\n",
      "Version: 1.34.0\n",
      "Publication date: None\n",
      "Mantainer: David Gomez-Cabrero <david.gomezcabrero@ki.se>, NÃºria Planell <nuria.planell.picola@navarra.es>\n",
      "Authors:\n",
      "Author: STATegra Consortia\n",
      "Dependencies:\n",
      "  - R (>= 2.10) type: depends\n",
      "  - Biobase, type: imports\n",
      "  - gridExtra, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - grid, type: imports\n",
      "  - MASS, type: imports\n",
      "  - calibrate, type: imports\n",
      "  - gplots, type: imports\n",
      "  - edgeR, type: imports\n",
      "  - limma, type: imports\n",
      "  - foreach, type: imports\n",
      "  - affy, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/STATegRa.html\n",
      "\n",
      "Name: Streamer\n",
      "Description:\n",
      "Large data files can be difficult to work with in R, where data generally resides in memory. This package encourages a style of programming where data is 'streamed' from disk into R via a `producer' and through a series of `consumers' that, typically reduce the original data to a manageable size. The package provides useful Producer and Consumer stream components for operations such as data input, sampling, indexing, and transformation; see package?Streamer for details.\n",
      "Version: 1.44.0\n",
      "Publication date: None\n",
      "Mantainer: Martin Morgan <martin.morgan@roswellpark.org>\n",
      "Authors:\n",
      "Author: Martin Morgan, Nishant Gopalakrishnan\n",
      "Dependencies:\n",
      "  - methods, type: imports\n",
      "  - graph, type: imports\n",
      "  - RBGL, type: imports\n",
      "  - parallel, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/Streamer.html\n",
      "\n",
      "Name: SWATH2stats\n",
      "Description:\n",
      "This package is intended to transform SWATH data from the OpenSWATH software into a format readable by other statistics packages while performing filtering, annotation and FDR estimation.\n",
      "Version: 1.28.0\n",
      "Publication date: None\n",
      "Mantainer: Peter Blattmann <blattmann@imsb.biol.ethz.ch>\n",
      "Authors:\n",
      "Author: Peter Blattmann [aut, cre] Moritz Heusel [aut] Ruedi Aebersold [aut]\n",
      "Dependencies:\n",
      "  - R (>= 2.10.0) type: depends\n",
      "  - data.table, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - stats, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - utils, type: imports\n",
      "  - biomaRt, type: imports\n",
      "  - methods, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/SWATH2stats.html\n",
      "\n",
      "Name: SwathXtend\n",
      "Description:\n",
      "Contains utility functions for integrating spectral libraries for SWATH and statistical data analysis for SWATH generated data.\n",
      "Version: 2.20.0\n",
      "Publication date: None\n",
      "Mantainer: Jemma Wu <jwu@proteome.org.au>\n",
      "Authors:\n",
      "Author: J WU and D Pascovici\n",
      "Dependencies:\n",
      "  - e1071, type: depends\n",
      "  - openxlsx, type: depends\n",
      "  - VennDiagram, type: depends\n",
      "  - lattice, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/SwathXtend.html\n",
      "\n",
      "Name: transcriptR\n",
      "Description:\n",
      "The differences in the RNA types being sequenced have an impact on the resulting sequencing profiles. mRNA-seq data is enriched with reads derived from exons, while GRO-, nucRNA- and chrRNA-seq demonstrate a substantial broader coverage of both exonic and intronic regions. The presence of intronic reads in GRO-seq type of data makes it possible to use it to computationally identify and quantify all de novo continuous regions of transcription distributed across the genome. This type of data, however, is more challenging to interpret and less common practice compared to mRNA-seq. One of the challenges for primary transcript detection concerns the simultaneous transcription of closely spaced genes, which needs to be properly divided into individually transcribed units. The R package transcriptR combines RNA-seq data with ChIP-seq data of histone modifications that mark active Transcription Start Sites (TSSs), such as, H3K4me3 or H3K9/14Ac to overcome this challenge. The advantage of this approach over the use of, for example, gene annotations is that this approach is data driven and therefore able to deal also with novel and case specific events. Furthermore, the integration of ChIP- and RNA-seq data allows the identification all known and novel active transcription start sites within a given sample.\n",
      "Version: 1.26.0\n",
      "Publication date: None\n",
      "Mantainer: Armen R. Karapetyan <armen.karapetyan87@gmail.com>\n",
      "Authors:\n",
      "Author: Armen R. Karapetyan <armen.karapetyan87 at gmail.com>\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - BiocGenerics, type: imports\n",
      "  - caret, type: imports\n",
      "  - chipseq, type: imports\n",
      "  - e1071, type: imports\n",
      "  - GenomicAlignments, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - GenomicFeatures, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - pROC, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/transcriptR.html\n",
      "\n",
      "Name: tscR\n",
      "Description:\n",
      "Clustering for time series data using slope distance and/or shape distance.\n",
      "Version: 1.10.0\n",
      "Publication date: None\n",
      "Mantainer: Fernando PÃ©rez-Sanz <fernando.perez8@um.es>\n",
      "Authors:\n",
      "Author: Fernando PÃ©rez-Sanz [aut, cre], Miriam Riquelme-PÃ©rez [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.1.0) type: depends\n",
      "  - dplyr, type: depends\n",
      "  - gridExtra, type: imports\n",
      "  - methods, type: imports\n",
      "  - dtw, type: imports\n",
      "  - class, type: imports\n",
      "  - kmlShape, type: imports\n",
      "  - graphics, type: imports\n",
      "  - cluster, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - knitr, type: imports\n",
      "  - rmarkdown, type: imports\n",
      "  - prettydoc, type: imports\n",
      "  - grid, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - latex2exp, type: imports\n",
      "  - stats, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/tscR.html\n",
      "\n",
      "Name: XDE\n",
      "Description:\n",
      "Multi-level model for cross-study detection of differential gene expression.\n",
      "Version: 2.44.0\n",
      "Publication date: None\n",
      "Mantainer: Robert Scharpf <rscharpf@jhsph.edu>\n",
      "Authors:\n",
      "Author: R.B. Scharpf, G. Parmigiani, A.B. Nobel, and H. Tjelmeland\n",
      "Dependencies:\n",
      "  - R (>= 2.10.0) type: depends\n",
      "  - Biobase, type: depends\n",
      "  - BiocGenerics, type: imports\n",
      "  - genefilter, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - gtools, type: imports\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - mvtnorm, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - GeneMeta, type: imports\n",
      "  - siggenes, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/XDE.html\n",
      "\n",
      "Name: affyILM\n",
      "Description:\n",
      "affyILM is a preprocessing tool which estimates gene expression levels for Affymetrix Gene Chips. Input from physical chemistry is employed to first background subtract intensities before calculating concentrations on behalf of the Langmuir model.\n",
      "Version: 1.50.0\n",
      "Publication date: None\n",
      "Mantainer: Myriam Kroll and Fabrice Berger <fabrice.berger@gmail.com>\n",
      "Authors:\n",
      "Author: K. Myriam Kroll, Fabrice Berger, Gerard Barkema, Enrico Carlon\n",
      "Dependencies:\n",
      "  - R (>= 2.10.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - gcrma, type: depends\n",
      "  - affxparser, type: imports\n",
      "  - affy, type: imports\n",
      "  - graphics, type: imports\n",
      "  - Biobase, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/affyILM.html\n",
      "\n",
      "Name: autonomics\n",
      "Description:\n",
      "This package offers a generic and intuitive solution for cross-platform omics data analysis. It has functions for import, preprocessing, exploration, contrast analysis and visualization of omics data. It follows a tidy, functional programming paradigm.\n",
      "Version: 1.6.0\n",
      "Publication date: None\n",
      "Mantainer: Aditya Bhagwat <aditya.bhagwat@uni-marburg.de>\n",
      "Authors:\n",
      "Author: Aditya Bhagwat [aut, cre], Shahina Hayat [aut], Anna Halama [ctb], Richard Cotton [ctb], Laure Cougnaud [ctb], Rudolf Engelke [ctb], Hinrich Goehlmann [sad], Karsten Suhre [sad], Johannes Graumann [aut, sad, rth]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - abind, type: imports\n",
      "  - assertive, type: imports\n",
      "  - BiocFileCache, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - colorspace, type: imports\n",
      "  - data.table, type: imports\n",
      "  - edgeR, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - ggrepel, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - grid, type: imports\n",
      "  - gridExtra, type: imports\n",
      "  - limma, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - methods, type: imports\n",
      "  - MultiAssayExperiment, type: imports\n",
      "  - parallel, type: imports\n",
      "  - pcaMethods, type: imports\n",
      "  - rappdirs, type: imports\n",
      "  - rlang, type: imports\n",
      "  - R.utils, type: imports\n",
      "  - readxl, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - scales, type: imports\n",
      "  - stats, type: imports\n",
      "  - stringi, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - tools, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/autonomics.html\n",
      "\n",
      "Name: bcSeq\n",
      "Description:\n",
      "This Rcpp-based package implements a highly efficient data structure and algorithm for performing alignment of short reads from CRISPR or shRNA screens to reference barcode library. Sequencing error are considered and matching qualities are evaluated based on Phred scores. A Bayes' classifier is employed to predict the originating barcode of a read. The package supports provision of user-defined probability models for evaluating matching qualities. The package also supports multi-threading.\n",
      "Version: 1.20.0\n",
      "Publication date: None\n",
      "Mantainer: Jiaxing Lin <jiaxing.lin@duke.edu>\n",
      "Authors:\n",
      "Author: Jiaxing Lin [aut, cre], Jeremy Gresham [aut], Jichun Xie [aut], Kouros Owzar [aut], Tongrong Wang [ctb], So Young Kim [ctb], James Alvarez [ctb], Jeffrey S. Damrauer [ctb], Scott Floyd [ctb], Joshua Granek [ctb], Andrew Allen [ctb], Cliburn Chan [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 3.4.0) type: depends\n",
      "  - Rcpp (>= 0.12.12) type: imports\n",
      "  - Matrix, type: imports\n",
      "  - Biostrings, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/bcSeq.html\n",
      "\n",
      "Name: BgeeCall\n",
      "Description:\n",
      "BgeeCall allows to generate present/absent gene expression calls without using an arbitrary cutoff like TPM<1. Calls are generated based on reference intergenic sequences. These sequences are generated based on expression of all RNA-Seq libraries of each species integrated in Bgee (https://bgee.org).\n",
      "Version: 1.14.0\n",
      "Publication date: None\n",
      "Mantainer: Julien Wollbrett <julien.wollbrett@unil.ch>\n",
      "Authors:\n",
      "Author: Julien Wollbrett [aut, cre], Sara Fonseca Costa [aut], Julien Roux [aut], Marc Robinson Rechavi [ctb], Frederic Bastian [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.6) type: depends\n",
      "  - GenomicFeatures, type: imports\n",
      "  - tximport, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - biomaRt, type: imports\n",
      "  - jsonlite, type: imports\n",
      "  - methods, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - data.table, type: imports\n",
      "  - sjmisc, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - rslurm, type: imports\n",
      "  - rhdf5, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3 + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/BgeeCall.html\n",
      "\n",
      "Name: BiGGR\n",
      "Description:\n",
      "This package provides an interface to simulate metabolic reconstruction from the BiGG database(http://bigg.ucsd.edu/) and other metabolic reconstruction databases. The package facilitates flux balance analysis (FBA) and the sampling of feasible flux distributions. Metabolic networks and estimated fluxes can be visualized with hypergraphs.\n",
      "Version: 1.34.0\n",
      "Publication date: None\n",
      "Mantainer: Anand K. Gavai <anand.gavai@bioinformatics.nl>, Hannes Hettling <hannes.hettling@naturalis.nl>\n",
      "Authors:\n",
      "Author: Anand K. Gavai, Hannes Hettling\n",
      "Dependencies:\n",
      "  - R (>= 2.14.0) type: depends\n",
      "  - rsbml, type: depends\n",
      "  - hyperdraw, type: depends\n",
      "  - LIM, type: depends\n",
      "  - stringr, type: depends\n",
      "  - hypergraph, type: imports\n",
      "  - limSolve, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/BiGGR.html\n",
      "\n",
      "Name: biobtreeR\n",
      "Description:\n",
      "The biobtreeR package provides an interface to [biobtree](https://github.com/tamerh/biobtree) tool which covers large set of bioinformatics datasets and allows search and chain mappings functionalities.\n",
      "Version: 1.10.0\n",
      "Publication date: None\n",
      "Mantainer: Tamer Gur <tgur@ebi.ac.uk>\n",
      "Authors:\n",
      "Author: Tamer Gur\n",
      "Dependencies:\n",
      "  - httr, type: imports\n",
      "  - httpuv, type: imports\n",
      "  - stringi, type: imports\n",
      "  - jsonlite, type: imports\n",
      "  - methods, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/biobtreeR.html\n",
      "\n",
      "Name: BioMVCClass\n",
      "Description:\n",
      "Creates classes used in model-view-controller (MVC) design\n",
      "Version: 1.66.0\n",
      "Publication date: None\n",
      "Mantainer: Elizabeth Whalen <ewhalen@hsph.harvard.edu>\n",
      "Authors:\n",
      "Author: Elizabeth Whalen\n",
      "Dependencies:\n",
      "  - R (>= 2.1.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - MVCClass, type: depends\n",
      "  - Biobase, type: depends\n",
      "  - graph, type: depends\n",
      "  - Rgraphviz, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/BioMVCClass.html\n",
      "\n",
      "Name: breakpointR\n",
      "Description:\n",
      "This package implements functions for finding breakpoints, plotting and export of Strand-seq data.\n",
      "Version: 1.16.0\n",
      "Publication date: None\n",
      "Mantainer: David Porubsky <david.porubsky@gmail.com>\n",
      "Authors:\n",
      "Author: David Porubsky, Ashley Sanders, Aaron Taudt\n",
      "Dependencies:\n",
      "  - R (>= 3.5) type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - cowplot, type: depends\n",
      "  - breakpointRdata, type: depends\n",
      "  - methods, type: imports\n",
      "  - utils, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - stats, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - GenomicAlignments, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - gtools, type: imports\n",
      "  - doParallel, type: imports\n",
      "  - foreach, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/breakpointR.html\n",
      "\n",
      "Name: brendaDb\n",
      "Description:\n",
      "R interface for importing and analyzing enzyme information from the BRENDA database.\n",
      "Version: 1.12.0\n",
      "Publication date: None\n",
      "Mantainer: Yi Zhou <yi.zhou@uga.edu>\n",
      "Authors:\n",
      "Author: Yi Zhou [aut, cre]\n",
      "Dependencies:\n",
      "  - dplyr, type: imports\n",
      "  - Rcpp, type: imports\n",
      "  - tibble, type: imports\n",
      "  - stringr, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - purrr, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - crayon, type: imports\n",
      "  - utils, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - rlang, type: imports\n",
      "  - BiocFileCache, type: imports\n",
      "  - rappdirs, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/brendaDb.html\n",
      "\n",
      "Name: BufferedMatrixMethods\n",
      "Description:\n",
      "Microarray analysis methods that use BufferedMatrix objects\n",
      "Version: 1.62.0\n",
      "Publication date: None\n",
      "Mantainer: Ben Bolstad <bmb@bmbolstad.com>\n",
      "Authors:\n",
      "Author: Ben Bolstad <bmb at bmbolstad.com>\n",
      "Dependencies:\n",
      "  - R (>= 2.6.0) type: depends\n",
      "  - BufferedMatrix, type: depends\n",
      "  - methods, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/BufferedMatrixMethods.html\n",
      "\n",
      "Name: BUS\n",
      "Description:\n",
      "This package can be used to compute associations among genes (gene-networks) or between genes and some external traits (i.e. clinical).\n",
      "Version: 1.54.0\n",
      "Publication date: None\n",
      "Mantainer: Yuanhua Liu <liuyuanhua@picb.ac.cn>\n",
      "Authors:\n",
      "Author: Yin Jin, Hesen Peng, Lei Wang, Raffaele Fronza, Yuanhua Liu and Christine Nardini\n",
      "Dependencies:\n",
      "  - R (>= 2.3.0) type: depends\n",
      "  - minet, type: depends\n",
      "  - stats, type: imports\n",
      "  - infotheo, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/BUS.html\n",
      "\n",
      "Name: cn.farms\n",
      "Description:\n",
      "This package implements the cn.FARMS algorithm for copy number variation (CNV) analysis. cn.FARMS allows to analyze the most common Affymetrix (250K-SNP6.0) array types, supports high-performance computing using snow and ff.\n",
      "Version: 1.46.0\n",
      "Publication date: None\n",
      "Mantainer: Andreas Mitterecker <mitterecker@ml.jku.at>\n",
      "Authors:\n",
      "Author: Andreas Mitterecker, Djork-Arne Clevert\n",
      "Dependencies:\n",
      "  - R (>= 3.0) type: depends\n",
      "  - Biobase, type: depends\n",
      "  - methods, type: depends\n",
      "  - ff, type: depends\n",
      "  - oligoClasses, type: depends\n",
      "  - snow, type: depends\n",
      "  - DBI, type: imports\n",
      "  - affxparser, type: imports\n",
      "  - oligo, type: imports\n",
      "  - DNAcopy, type: imports\n",
      "  - preprocessCore, type: imports\n",
      "  - lattice, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL (>= 2.0)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/cn.farms.html\n",
      "\n",
      "Name: CNORdt\n",
      "Description:\n",
      "This add-on to the package CellNOptR handles time-course data, as opposed to steady state data in CellNOptR. It scales the simulation step to allow comparison and model fitting for time-course data. Future versions will optimize delays and strengths for each edge.\n",
      "Version: 1.40.0\n",
      "Publication date: None\n",
      "Mantainer: A. MacNamara <aidan.macnamara@ebi.ac.uk>\n",
      "Authors:\n",
      "Author: A. MacNamara\n",
      "Dependencies:\n",
      "  - R (>= 1.8.0) type: depends\n",
      "  - CellNOptR, type: depends\n",
      "  - abind, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/CNORdt.html\n",
      "\n",
      "Name: CNORfuzzy\n",
      "Description:\n",
      "This package is an extension to CellNOptR. It contains additional functionality needed to simulate and train a prior knowledge network to experimental data using constrained fuzzy logic (cFL, rather than Boolean logic as is the case in CellNOptR). Additionally, this package will contain functions to use for the compilation of multiple optimization results (either Boolean or cFL).\n",
      "Version: 1.40.0\n",
      "Publication date: None\n",
      "Mantainer: T. Cokelaer <cokelaer@ebi.ac.uk>\n",
      "Authors:\n",
      "Author: M. Morris, T. Cokelaer\n",
      "Dependencies:\n",
      "  - R (>= 2.15.0) type: depends\n",
      "  - CellNOptR, type: depends\n",
      "  - nloptr (>= 0.8.5) type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/CNORfuzzy.html\n",
      "\n",
      "Name: COCOA\n",
      "Description:\n",
      "COCOA is a method for understanding epigenetic variation among samples. COCOA can be used with epigenetic data that includes genomic coordinates and an epigenetic signal, such as DNA methylation and chromatin accessibility data. To describe the method on a high level, COCOA quantifies inter-sample variation with either a supervised or unsupervised technique then uses a database of \"region sets\" to annotate the variation among samples. A region set is a set of genomic regions that share a biological annotation, for instance transcription factor (TF) binding regions, histone modification regions, or open chromatin regions. COCOA can identify region sets that are associated with epigenetic variation between samples and increase understanding of variation in your data.\n",
      "Version: 2.12.0\n",
      "Publication date: None\n",
      "Mantainer: John Lawson <jtl2hk@virginia.edu>\n",
      "Authors:\n",
      "Author: John Lawson [aut, cre], Nathan Sheffield [aut] (http://www.databio.org), Jason Smith [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 3.5) type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - BiocGenerics, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - data.table, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - stats, type: imports\n",
      "  - methods, type: imports\n",
      "  - ComplexHeatmap, type: imports\n",
      "  - MIRA, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - grid, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - simpleCache, type: imports\n",
      "  - fitdistrplus, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/COCOA.html\n",
      "\n",
      "Name: COSNet\n",
      "Description:\n",
      "Package that implements the COSNet classification algorithm. The algorithm predicts node labels in partially labeled graphs where few positives are available for the class being predicted.\n",
      "Version: 1.32.0\n",
      "Publication date: None\n",
      "Mantainer: Marco Frasca<frasca@di.unimi.it>\n",
      "Authors:\n",
      "Author: Marco Frasca and Giorgio Valentini -- Universita' degli Studi di Milano\n",
      "Dependencies:\n",
      "None\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/COSNet.html\n",
      "\n",
      "Name: DIAlignR\n",
      "Description:\n",
      "To obtain unbiased proteome coverage from a biological sample, mass-spectrometer is operated in Data Independent Acquisition (DIA) mode. Alignment of these DIA runs establishes consistency and less missing values in complete data-matrix. This package implements dynamic programming with affine gap penalty based approach for pair-wise alignment of analytes. A hybrid approach of global alignment (through MS2 features) and local alignment (with MS2 chromatograms) is implemented in this tool.\n",
      "Version: 2.6.0\n",
      "Publication date: None\n",
      "Mantainer: Shubham Gupta <shubham.1637@gmail.com>\n",
      "Authors:\n",
      "Author: Shubham Gupta [aut, cre] , Hannes Rost [aut] , Justin Sing [aut]\n",
      "Dependencies:\n",
      "  - methods, type: depends\n",
      "  - stats, type: depends\n",
      "  - R (>= 4.0) type: depends\n",
      "  - zoo (>= 1.8-3) type: imports\n",
      "  - data.table, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - rlang, type: imports\n",
      "  - mzR, type: imports\n",
      "  - signal, type: imports\n",
      "  - bit64, type: imports\n",
      "  - reticulate, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - RSQLite, type: imports\n",
      "  - DBI, type: imports\n",
      "  - ape, type: imports\n",
      "  - phangorn, type: imports\n",
      "  - pracma, type: imports\n",
      "  - RMSNumpress, type: imports\n",
      "  - Rcpp, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/DIAlignR.html\n",
      "\n",
      "Name: fdrame\n",
      "Description:\n",
      "This package contains two main functions. The first is fdr.ma which takes normalized expression data array, experimental design and computes adjusted p-values It returns the fdr adjusted p-values and plots, according to the methods described in (Reiner, Yekutieli and Benjamini 2002). The second, is fdr.gui() which creates a simple graphic user interface to access fdr.ma\n",
      "Version: 1.70.0\n",
      "Publication date: None\n",
      "Mantainer: Effi Kenigsberg <effiken.fdrame@gmail.com>\n",
      "Authors:\n",
      "Author: Yoav Benjamini, Effi Kenigsberg, Anat Reiner, Daniel Yekutieli\n",
      "Dependencies:\n",
      "  - tcltk, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/fdrame.html\n",
      "\n",
      "Name: flowPloidy\n",
      "Description:\n",
      "Determine sample ploidy via flow cytometry histogram analysis. Reads Flow Cytometry Standard (FCS) files via the flowCore bioconductor package, and provides functions for determining the DNA ploidy of samples based on internal standards.\n",
      "Version: 1.24.0\n",
      "Publication date: None\n",
      "Mantainer: Tyler Smith <tyler@plantarum.ca>\n",
      "Authors:\n",
      "Author: Tyler Smith <tyler at plantarum.ca>\n",
      "Dependencies:\n",
      "  - flowCore, type: imports\n",
      "  - car, type: imports\n",
      "  - caTools, type: imports\n",
      "  - knitr, type: imports\n",
      "  - rmarkdown, type: imports\n",
      "  - minpack.lm, type: imports\n",
      "  - shiny, type: imports\n",
      "  - methods, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/flowPloidy.html\n",
      "\n",
      "Name: GA4GHclient\n",
      "Description:\n",
      "GA4GHclient provides an easy way to access public data servers through Global Alliance for Genomics and Health (GA4GH) genomics API. It provides low-level access to GA4GH API and translates response data into Bioconductor-based class objects.\n",
      "Version: 1.22.0\n",
      "Publication date: None\n",
      "Mantainer: Welliton Souza <well309@gmail.com>\n",
      "Authors:\n",
      "Author: Welliton Souza [aut, cre], Benilton Carvalho [ctb], Cristiane Rocha [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - S4Vectors, type: depends\n",
      "  - BiocGenerics, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - httr, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - jsonlite, type: imports\n",
      "  - methods, type: imports\n",
      "  - VariantAnnotation, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/GA4GHclient.html\n",
      "\n",
      "Name: GCSscore\n",
      "Description:\n",
      "For differential expression analysis of 3'IVT and WT-style microarrays from Affymetrix/Thermo-Fisher. Based on S-score algorithm originally described by Zhang et al 2002.\n",
      "Version: 1.12.0\n",
      "Publication date: None\n",
      "Mantainer: Guy M. Harris <harrisgm@vcu.edu>\n",
      "Authors:\n",
      "Author: Guy M. Harris & Shahroze Abbas & Michael F. Miles\n",
      "Dependencies:\n",
      "  - R (>= 3.6) type: depends\n",
      "  - BiocManager, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - utils, type: imports\n",
      "  - methods, type: imports\n",
      "  - RSQLite, type: imports\n",
      "  - devtools, type: imports\n",
      "  - dplR, type: imports\n",
      "  - stringr, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - affxparser, type: imports\n",
      "  - data.table, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>=3)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/GCSscore.html\n",
      "\n",
      "Name: GEM\n",
      "Description:\n",
      "Tools for analyzing EWAS, methQTL and GxE genome widely.\n",
      "Version: 1.24.0\n",
      "Publication date: None\n",
      "Mantainer: Hong Pan <pan_hong@sics.a-star.edu.sg>\n",
      "Authors:\n",
      "Author: Hong Pan, Joanna D Holbrook, Neerja Karnani, Chee-Keong Kwoh\n",
      "Dependencies:\n",
      "  - R (>= 3.3) type: depends\n",
      "  - tcltk, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/GEM.html\n",
      "\n",
      "Name: GRridge\n",
      "Description:\n",
      "This package allows the use of multiple sources of co-data (e.g. external p-values, gene lists, annotation) to improve prediction of binary, continuous and survival response using (logistic, linear or Cox) group-regularized ridge regression. It also facilitates post-hoc variable selection and prediction diagnostics by cross-validation using ROC curves and AUC.\n",
      "Version: 1.22.0\n",
      "Publication date: None\n",
      "Mantainer: Mark A. van de Wiel <mark.vdwiel@vumc.nl>\n",
      "Authors:\n",
      "Author: Mark A. van de Wiel <mark.vdwiel at vumc.nl>, Putri W. Novianti <p.novianti at vumc.nl>\n",
      "Dependencies:\n",
      "  - R (>= 3.2) type: depends\n",
      "  - penalized, type: depends\n",
      "  - Iso, type: depends\n",
      "  - survival, type: depends\n",
      "  - methods, type: depends\n",
      "  - graph, type: depends\n",
      "  - stats, type: depends\n",
      "  - glmnet, type: depends\n",
      "  - mvtnorm, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/GRridge.html\n",
      "\n",
      "Name: HEM\n",
      "Description:\n",
      "This package fits heterogeneous error models for analysis of microarray data\n",
      "Version: 1.70.0\n",
      "Publication date: None\n",
      "Mantainer: HyungJun Cho <hcho@virginia.edu>\n",
      "Authors:\n",
      "Author: HyungJun Cho <hcho at virginia.edu> and Jae K. Lee <jaeklee at virginia.edu>\n",
      "Dependencies:\n",
      "  - R (>= 2.1.0) type: depends\n",
      "  - Biobase, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/HEM.html\n",
      "\n",
      "Name: imageHTS\n",
      "Description:\n",
      "imageHTS is an R package dedicated to the analysis of high-throughput microscopy-based screens. The package provides a modular and extensible framework to segment cells, extract quantitative cell features, predict cell types and browse screen data through web interfaces. Designed to operate in distributed environments, imageHTS provides a standardized access to remote data and facilitates the dissemination of high-throughput microscopy-based datasets.\n",
      "Version: 1.48.0\n",
      "Publication date: None\n",
      "Mantainer: Joseph Barry <joseph.barry@embl.de>\n",
      "Authors:\n",
      "Author: Gregoire Pau, Xian Zhang, Michael Boutros, Wolfgang Huber\n",
      "Dependencies:\n",
      "  - R (>= 2.9.0) type: depends\n",
      "  - EBImage, type: depends\n",
      "  - cellHTS2, type: depends\n",
      "  - tools, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - hwriter, type: imports\n",
      "  - methods, type: imports\n",
      "  - vsn, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - e1071, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL-2.1\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/imageHTS.html\n",
      "\n",
      "Name: IMMAN\n",
      "Description:\n",
      "Reconstructing Interlog Protein Network (IPN) integrated from several Protein protein Interaction Networks (PPINs). Using this package, overlaying different PPINs to mine conserved common networks between diverse species will be applicable.\n",
      "Version: 1.18.0\n",
      "Publication date: None\n",
      "Mantainer: Minoo Ashtiani <ashtiani.minoo@gmail.com>\n",
      "Authors:\n",
      "Author: Minoo Ashtiani, Payman Nickchi, Abdollah Safari, Mehdi Mirzaie, Mohieddin Jafari\n",
      "Dependencies:\n",
      "  - STRINGdb, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - igraph, type: imports\n",
      "  - graphics, type: imports\n",
      "  - utils, type: imports\n",
      "  - seqinr, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/IMMAN.html\n",
      "\n",
      "Name: martini\n",
      "Description:\n",
      "martini deals with the low power inherent to GWAS studies by using prior knowledge represented as a network. SNPs are the vertices of the network, and the edges represent biological relationships between them (genomic adjacency, belonging to the same gene, physical interaction between protein products). The network is scanned using SConES, which looks for groups of SNPs maximally associated with the phenotype, that form a close subnetwork.\n",
      "Version: 1.18.1\n",
      "Publication date: None\n",
      "Mantainer: Hector Climente-Gonzalez <hector.climente@riken.jp>\n",
      "Authors:\n",
      "Author: Hector Climente-Gonzalez [aut, cre] , Chloe-Agathe Azencott [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - igraph (>= 1.0.1) type: imports\n",
      "  - Matrix, type: imports\n",
      "  - memoise (>= 2.0.0) type: imports\n",
      "  - methods (>= 3.3.2) type: imports\n",
      "  - Rcpp (>= 0.12.8) type: imports\n",
      "  - snpStats, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/martini.html\n",
      "\n",
      "Name: MBASED\n",
      "Description:\n",
      "The package implements MBASED algorithm for detecting allele-specific gene expression from RNA count data, where allele counts at individual loci (SNVs) are integrated into a gene-specific measure of ASE, and utilizes simulations to appropriately assess the statistical significance of observed ASE.\n",
      "Version: 1.32.0\n",
      "Publication date: None\n",
      "Mantainer: Oleg Mayba <mayba.oleg@gene.com>\n",
      "Authors:\n",
      "Author: Oleg Mayba, Houston Gilbert\n",
      "Dependencies:\n",
      "  - RUnit, type: depends\n",
      "  - BiocGenerics, type: depends\n",
      "  - BiocParallel, type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - SummarizedExperiment, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/MBASED.html\n",
      "\n",
      "Name: Metab\n",
      "Description:\n",
      "Metab is an R package for high-throughput processing of metabolomics data analysed by the Automated Mass Spectral Deconvolution and Identification System (AMDIS) (http://chemdata.nist.gov/mass-spc/amdis/downloads/). In addition, it performs statistical hypothesis test (t-test) and analysis of variance (ANOVA). Doing so, Metab considerably speed up the data mining process in metabolomics and produces better quality results. Metab was developed using interactive features, allowing users with lack of R knowledge to appreciate its functionalities.\n",
      "Version: 1.32.0\n",
      "Publication date: None\n",
      "Mantainer: Raphael Aggio <ragg005@aucklanduni.ac.nz>\n",
      "Authors:\n",
      "Author: Raphael Aggio <ragg005 at aucklanduni.ac.nz>\n",
      "Dependencies:\n",
      "  - xcms, type: depends\n",
      "  - R (>= 3.0.1) type: depends\n",
      "  - svDialogs, type: depends\n",
      "  - pander, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>=2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/Metab.html\n",
      "\n",
      "Name: MGFR\n",
      "Description:\n",
      "The package is designed to detect marker genes from RNA-seq data.\n",
      "Version: 1.24.0\n",
      "Publication date: None\n",
      "Mantainer: Khadija El Amrani <a.khadija@gmx.de>\n",
      "Authors:\n",
      "Author: Khadija El Amrani\n",
      "Dependencies:\n",
      "  - R (>= 3.5) type: depends\n",
      "  - biomaRt, type: imports\n",
      "  - annotate, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/MGFR.html\n",
      "\n",
      "Name: MIMOSA\n",
      "Description:\n",
      "Modeling count data using Dirichlet-multinomial and beta-binomial mixtures with applications to single-cell assays.\n",
      "Version: 1.36.0\n",
      "Publication date: None\n",
      "Mantainer: Greg Finak <gfinak@fhcrc.org>\n",
      "Authors:\n",
      "Author: Greg Finak <gfinak at fhcrc.org>\n",
      "Dependencies:\n",
      "  - R (>= 3.0.2) type: depends\n",
      "  - MASS, type: depends\n",
      "  - plyr, type: depends\n",
      "  - reshape, type: depends\n",
      "  - Biobase, type: depends\n",
      "  - ggplot2, type: depends\n",
      "  - methods, type: imports\n",
      "  - Formula, type: imports\n",
      "  - data.table, type: imports\n",
      "  - pracma, type: imports\n",
      "  - MCMCpack, type: imports\n",
      "  - coda, type: imports\n",
      "  - modeest, type: imports\n",
      "  - testthat, type: imports\n",
      "  - Rcpp, type: imports\n",
      "  - scales, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - rlang, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/MIMOSA.html\n",
      "\n",
      "Name: MPRAnalyze\n",
      "Description:\n",
      "MPRAnalyze provides statistical framework for the analysis of data generated by Massively Parallel Reporter Assays (MPRAs), used to directly measure enhancer activity. MPRAnalyze can be used for quantification of enhancer activity, classification of active enhancers and comparative analyses of enhancer activity between conditions. MPRAnalyze construct a nested pair of generalized linear models (GLMs) to relate the DNA and RNA observations, easily adjustable to various experimental designs and conditions, and provides a set of rigorous statistical testig schemes.\n",
      "Version: 1.16.0\n",
      "Publication date: None\n",
      "Mantainer: Tal Ashuach <tal_ashuach@berkeley.edu>\n",
      "Authors:\n",
      "Author: Tal Ashuach [aut, cre], David S Fischer [aut], Anat Kriemer [ctb], Fabian J Theis [ctb], Nir Yosef [ctb],\n",
      "Dependencies:\n",
      "  - BiocParallel, type: imports\n",
      "  - methods, type: imports\n",
      "  - progress, type: imports\n",
      "  - stats, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/MPRAnalyze.html\n",
      "\n",
      "Name: MsBackendMsp\n",
      "Description:\n",
      "Mass spectrometry (MS) data backend supporting import and handling of MS/MS spectra from NIST MSP Format (msp) files. Import of data from files with different MSP *flavours* is supported. Objects from this package add support for MSP files to Bioconductor's Spectra package. This package is thus not supposed to be used without the Spectra package that provides a complete infrastructure for MS data handling.\n",
      "Version: 1.2.0\n",
      "Publication date: None\n",
      "Mantainer: Johannes Rainer <Johannes.Rainer@eurac.edu>\n",
      "Authors:\n",
      "Author: Neumann Steffen [aut] , Johannes Rainer [aut, cre] , Michael Witting [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 4.1.0) type: depends\n",
      "  - Spectra, type: depends\n",
      "  - BiocParallel, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - MsCoreUtils, type: imports\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/MsBackendMsp.html\n",
      "\n",
      "Name: netSmooth\n",
      "Description:\n",
      "netSmooth is an R package for network smoothing of single cell RNA sequencing data. Using bio networks such as protein-protein interactions as priors for gene co-expression, netsmooth improves cell type identification from noisy, sparse scRNAseq data.\n",
      "Version: 1.18.0\n",
      "Publication date: None\n",
      "Mantainer: Jonathan Ronen <yablee@gmail.com>\n",
      "Authors:\n",
      "Author: Jonathan Ronen [aut, cre], Altuna Akalin [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.5) type: depends\n",
      "  - scater, type: depends\n",
      "  - clusterExperiment, type: depends\n",
      "  - entropy, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - SingleCellExperiment, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - cluster, type: imports\n",
      "  - data.table, type: imports\n",
      "  - stats, type: imports\n",
      "  - methods, type: imports\n",
      "  - DelayedArray, type: imports\n",
      "  - HDF5Array, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/netSmooth.html\n",
      "\n",
      "Name: NewWave\n",
      "Description:\n",
      "A model designed for dimensionality reduction and batch effect removal for scRNA-seq data. It is designed to be massively parallelizable using shared objects that prevent memory duplication, and it can be used with different mini-batch approaches in order to reduce time consumption. It assumes a negative binomial distribution for the data with a dispersion parameter that can be both commonwise across gene both genewise.\n",
      "Version: 1.8.0\n",
      "Publication date: None\n",
      "Mantainer: Federico Agostinis <federico.agostinis@outlook.com>\n",
      "Authors:\n",
      "Author: Federico Agostinis [aut, cre], Chiara Romualdi [aut], Gabriele Sales [aut], Davide Risso [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - methods, type: imports\n",
      "  - SingleCellExperiment, type: imports\n",
      "  - parallel, type: imports\n",
      "  - irlba, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - DelayedArray, type: imports\n",
      "  - BiocSingular, type: imports\n",
      "  - SharedObject, type: imports\n",
      "  - stats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/NewWave.html\n",
      "\n",
      "Name: NuPoP\n",
      "Description:\n",
      "NuPoP is an R package for Nucleosome Positioning Prediction.This package is built upon a duration hidden Markov model proposed in Xi et al, 2010; Wang et al, 2008. The core of the package was written in Fotran. In addition to the R package, a stand-alone Fortran software tool is also available at https://github.com/jipingw. The Fortran codes have complete functonality as the R package. Note: NuPoP has two separate functions for prediction of nucleosome positioning, one for MNase-map trained models and the other for chemical map-trained models. The latter was implemented for four species including yeast, S.pombe, mouse and human, trained based on our recent publications. We noticed there is another package nuCpos by another group for prediction of nucleosome positioning trained with chemicals. A report to compare recent versions of NuPoP with nuCpos can be found at https://github.com/jiping/NuPoP_doc. Some more information can be found and will be posted at https://github.com/jipingw/NuPoP.\n",
      "Version: 2.6.3\n",
      "Publication date: None\n",
      "Mantainer: Ji-Ping Wang<jzwang@northwestern.edu>\n",
      "Authors:\n",
      "Author: Ji-Ping Wang <jzwang at northwestern.edu>; Liqun Xi <liqunxi02 at gmail.com>; Oscar Zarate <zarate at u.northwestern.edu>\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - graphics, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/NuPoP.html\n",
      "\n",
      "Name: PathNet\n",
      "Description:\n",
      "PathNet uses topological information present in pathways and differential expression levels of genes (obtained from microarray experiment) to identify pathways that are 1) significantly enriched and 2) associated with each other in the context of differential expression. The algorithm is described in: PathNet: A tool for pathway analysis using topological information. Dutta B, Wallqvist A, and Reifman J. Source Code for Biology and Medicine 2012 Sep 24;7(1):10.\n",
      "Version: 1.38.0\n",
      "Publication date: None\n",
      "Mantainer: Ludwig Geistlinger <Ludwig_Geistlinger@hms.harvard.edu>\n",
      "Authors:\n",
      "Author: Bhaskar Dutta <bhaskar.dutta at gmail.com>, Anders Wallqvist <awallqvist at bhsai.org>, and Jaques Reifman <jreifman at bhsai.org>\n",
      "Dependencies:\n",
      "None\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/PathNet.html\n",
      "\n",
      "Name: PathoStat\n",
      "Description:\n",
      "The purpose of this package is to perform Statistical Microbiome Analysis on metagenomics results from sequencing data samples. In particular, it supports analyses on the PathoScope generated report files. PathoStat provides various functionalities including Relative Abundance charts, Diversity estimates and plots, tests of Differential Abundance, Time Series visualization, and Core OTU analysis.\n",
      "Version: 1.24.0\n",
      "Publication date: None\n",
      "Mantainer: Solaiappan Manimaran <manimaran_1975@hotmail.com>, Yue Zhao <jasonzhao0307@gmail.com>\n",
      "Authors:\n",
      "Author: Solaiappan Manimaran <manimaran_1975 at hotmail.com>, Matthew Bendall <bendall at gwmail.gwu.edu>, Sandro Valenzuela Diaz <sandrolvalenzuelad at gmail.com>, Eduardo Castro <castronallar at gmail.com>, Tyler Faits <tfaits at gmail.com>, Yue Zhao <jasonzhao0307 at gmail.com>, Anthony Nicholas Federico <anfed at bu.edu>, W. Evan Johnson <wej at bu.edu>\n",
      "Dependencies:\n",
      "  - R (>= 3.5) type: depends\n",
      "  - limma, type: imports\n",
      "  - corpcor, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - scales, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - rentrez, type: imports\n",
      "  - DT, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - plyr, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - phyloseq, type: imports\n",
      "  - shiny, type: imports\n",
      "  - stats, type: imports\n",
      "  - methods, type: imports\n",
      "  - XML, type: imports\n",
      "  - graphics, type: imports\n",
      "  - utils, type: imports\n",
      "  - BiocStyle, type: imports\n",
      "  - edgeR, type: imports\n",
      "  - DESeq2, type: imports\n",
      "  - ComplexHeatmap, type: imports\n",
      "  - plotly, type: imports\n",
      "  - webshot, type: imports\n",
      "  - vegan, type: imports\n",
      "  - shinyjs, type: imports\n",
      "  - glmnet, type: imports\n",
      "  - gmodels, type: imports\n",
      "  - ROCR, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - knitr, type: imports\n",
      "  - devtools, type: imports\n",
      "  - ape, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/PathoStat.html\n",
      "\n",
      "Name: PhIPData\n",
      "Description:\n",
      "PhIPData defines an S4 class for phage-immunoprecipitation sequencing (PhIP-seq) experiments. Buliding upon the RangedSummarizedExperiment class, PhIPData enables users to coordinate metadata with experimental data in analyses. Additionally, PhIPData provides specialized methods to subset and identify beads-only samples, subset objects using virus aliases, and use existing peptide libraries to populate object parameters.\n",
      "Version: 1.6.0\n",
      "Publication date: None\n",
      "Mantainer: Athena Chen <achen70@jhu.edu>\n",
      "Authors:\n",
      "Author: Athena Chen [aut, cre] , Rob Scharpf [aut], Ingo Ruczinski [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.1.0) type: depends\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - BiocFileCache, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - methods, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - edgeR, type: imports\n",
      "  - cli, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/PhIPData.html\n",
      "\n",
      "Name: psygenet2r\n",
      "Description:\n",
      "Package to retrieve data from PsyGeNET database (www.psygenet.org) and to perform comorbidity studies with PsyGeNET's and user's data.\n",
      "Version: 1.30.0\n",
      "Publication date: None\n",
      "Mantainer: Alba Gutierrez-Sacristan <a.gutierrez.sacristan@gmail.com>\n",
      "Authors:\n",
      "Author: Alba Gutierrez-Sacristan [aut, cre], Carles Hernandez-Ferrer [aut], Jaun R. Gonzalez [aut], Laura I. Furlong [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.4) type: depends\n",
      "  - stringr, type: imports\n",
      "  - RCurl, type: imports\n",
      "  - igraph, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - grid, type: imports\n",
      "  - parallel, type: imports\n",
      "  - biomaRt, type: imports\n",
      "  - BgeeDB, type: imports\n",
      "  - topGO, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - labeling, type: imports\n",
      "  - GO.db, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/psygenet2r.html\n",
      "\n",
      "Name: QuartPAC\n",
      "Description:\n",
      "Identifies clustering of somatic mutations in proteins over the entire quaternary structure.\n",
      "Version: 1.30.0\n",
      "Publication date: None\n",
      "Mantainer: Gregory Ryslik <gregory.ryslik@yale.edu>\n",
      "Authors:\n",
      "Author: Gregory Ryslik, Yuwei Cheng, Hongyu Zhao\n",
      "Dependencies:\n",
      "  - iPAC, type: depends\n",
      "  - GraphPAC, type: depends\n",
      "  - SpacePAC, type: depends\n",
      "  - data.table, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/QuartPAC.html\n",
      "\n",
      "Name: RITAN\n",
      "Description:\n",
      "Tools for comprehensive gene set enrichment and extraction of multi-resource high confidence subnetworks. RITAN facilitates bioinformatic tasks for enabling network biology research.\n",
      "Version: 1.22.0\n",
      "Publication date: None\n",
      "Mantainer: Michael Zimmermann <mtzimmermann@mcw.edu>\n",
      "Authors:\n",
      "Author: Michael Zimmermann [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - graphics, type: imports\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - grid, type: imports\n",
      "  - gridExtra, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - gplots, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - plotrix, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - STRINGdb, type: imports\n",
      "  - MCL, type: imports\n",
      "  - linkcomm, type: imports\n",
      "  - dynamicTreeCut, type: imports\n",
      "  - gsubfn, type: imports\n",
      "  - hash, type: imports\n",
      "  - png, type: imports\n",
      "  - sqldf, type: imports\n",
      "  - igraph, type: imports\n",
      "  - BgeeDB, type: imports\n",
      "  - knitr, type: imports\n",
      "  - RITANdata, type: imports\n",
      "  - GenomicFeatures, type: imports\n",
      "  - ensembldb, type: imports\n",
      "  - AnnotationFilter, type: imports\n",
      "  - EnsDb.Hsapiens.v86, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/RITAN.html\n",
      "\n",
      "Name: rnaseqcomp\n",
      "Description:\n",
      "Several quantitative and visualized benchmarks for RNA-seq quantification pipelines. Two-condition quantifications for genes, transcripts, junctions or exons by each pipeline with necessary meta information should be organized into numeric matrices in order to proceed the evaluation.\n",
      "Version: 1.28.0\n",
      "Publication date: None\n",
      "Mantainer: Mingxiang Teng <tengmx@gmail.com>\n",
      "Authors:\n",
      "Author: Mingxiang Teng and Rafael A. Irizarry\n",
      "Dependencies:\n",
      "  - R (>= 3.2.0) type: depends\n",
      "  - RColorBrewer, type: imports\n",
      "  - methods, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/rnaseqcomp.html\n",
      "\n",
      "Name: Rtreemix\n",
      "Description:\n",
      "Rtreemix is a package that offers an environment for estimating the mutagenetic trees mixture models from cross-sectional data and using them for various predictions. It includes functions for fitting the trees mixture models, likelihood computations, model comparisons, waiting time estimations, stability analysis, etc.\n",
      "Version: 1.60.0\n",
      "Publication date: None\n",
      "Mantainer: Jasmina Bogojeska <jasmina.bogojeska@gmail.com>\n",
      "Authors:\n",
      "Author: Jasmina Bogojeska\n",
      "Dependencies:\n",
      "  - R (>= 2.5.0) type: depends\n",
      "  - methods, type: imports\n",
      "  - graph, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - Hmisc, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/Rtreemix.html\n",
      "\n",
      "Name: sincell\n",
      "Description:\n",
      "Cell differentiation processes are achieved through a continuum of hierarchical intermediate cell-states that might be captured by single-cell RNA seq. Existing computational approaches for the assessment of cell-state hierarchies from single-cell data might be formalized under a general workflow composed of i) a metric to assess cell-to-cell similarities (combined or not with a dimensionality reduction step), and ii) a graph-building algorithm (optionally making use of a cells-clustering step). Sincell R package implements a methodological toolbox allowing flexible workflows under such framework. Furthermore, Sincell contributes new algorithms to provide cell-state hierarchies with statistical support while accounting for stochastic factors in single-cell RNA seq. Graphical representations and functional association tests are provided to interpret hierarchies.\n",
      "Version: 1.30.0\n",
      "Publication date: None\n",
      "Mantainer: Miguel Julia <migueljuliamolina@gmail.com>, Antonio Rausell<antonio.rausell@isb-sib.ch>\n",
      "Authors:\n",
      "Author: Miguel Julia <migueljuliamolina at gmail.com>, Amalio Telenti <atelenti at jcvi.org>, Antonio Rausell <antonio.rausell at isb-sib.ch>\n",
      "Dependencies:\n",
      "  - R (>= 3.0.2) type: depends\n",
      "  - igraph, type: depends\n",
      "  - Rcpp (>= 0.11.2) type: imports\n",
      "  - entropy, type: imports\n",
      "  - scatterplot3d, type: imports\n",
      "  - MASS, type: imports\n",
      "  - TSP, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - fields, type: imports\n",
      "  - proxy, type: imports\n",
      "  - parallel, type: imports\n",
      "  - Rtsne, type: imports\n",
      "  - fastICA, type: imports\n",
      "  - cluster, type: imports\n",
      "  - statmod, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/sincell.html\n",
      "\n",
      "Name: specL\n",
      "Description:\n",
      "provides a functions for generating spectra libraries that can be used for MRM SRM MS workflows in proteomics. The package provides a BiblioSpec reader, a function which can add the protein information using a FASTA formatted amino acid file, and an export method for using the created library in the Spectronaut software. The package is developed, tested and used at the Functional Genomics Center Zurich .\n",
      "Version: 1.32.1\n",
      "Publication date: None\n",
      "Mantainer: Christian Panse <cp@fgcz.ethz.ch>\n",
      "Authors:\n",
      "Author: Christian Panse [aut, cre] , Jonas Grossmann [aut] , Christian Trachsel [aut], Witold E. Wolski [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 3.6) type: depends\n",
      "  - DBI (>= 0.5) type: depends\n",
      "  - methods (>= 3.3) type: depends\n",
      "  - protViz (>= 0.5) type: depends\n",
      "  - RSQLite (>= 1.1) type: depends\n",
      "  - seqinr (>= 3.3) type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/specL.html\n",
      "\n",
      "Name: SynExtend\n",
      "Description:\n",
      "Shared order between genomic sequences provide a great deal of information. Synteny objects produced by the R package DECIPHER provides quantitative information about that shared order. SynExtend provides tools for extracting information from Synteny objects.\n",
      "Version: 1.10.2\n",
      "Publication date: None\n",
      "Mantainer: Nicholas Cooley <npc19@pitt.edu>\n",
      "Authors:\n",
      "Author: Nicholas Cooley [aut, cre] , Aidan Lakshman [aut, ctb] , Adelle Fernando [ctb], Erik Wright [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.2.0) type: depends\n",
      "  - DECIPHER, type: depends\n",
      "  - methods, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - utils, type: imports\n",
      "  - stats, type: imports\n",
      "  - parallel, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/SynExtend.html\n",
      "\n",
      "Name: tanggle\n",
      "Description:\n",
      "Offers functions for plotting split (or implicit) networks (unrooted, undirected) and explicit networks (rooted, directed) with reticulations extending. 'ggtree' and using functions from 'ape' and 'phangorn'. It extends the 'ggtree' package [@Yu2017] to allow the visualization of phylogenetic networks using the 'ggplot2' syntax. It offers an alternative to the plot functions already available in 'ape' Paradis and Schliep (2019) and 'phangorn' Schliep (2011) .\n",
      "Version: 1.4.0\n",
      "Publication date: None\n",
      "Mantainer: Klaus Schliep <klaus.schliep@gmail.com>\n",
      "Authors:\n",
      "Author: Klaus Schliep [aut, cre] , Marta Vidal-Garcia [aut], Claudia Solis-Lemus [aut] , Leann Biancani [aut], Eren Ada [aut], L. Francisco Henao Diaz [aut], Guangchuang Yu [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - ggplot2 (>= 2.2.0) type: depends\n",
      "  - ggtree, type: depends\n",
      "  - ape (>= 5.0) type: imports\n",
      "  - phangorn (>= 2.5) type: imports\n",
      "  - utils, type: imports\n",
      "  - methods, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/tanggle.html\n",
      "\n",
      "Name: TBSignatureProfiler\n",
      "Description:\n",
      "Gene signatures of TB progression, TB disease, and other TB disease states have been validated and published previously. This package aggregates known signatures and provides computational tools to enlist their usage on other datasets. The TBSignatureProfiler makes it easy to profile RNA-Seq data using these signatures and includes common signature profiling tools including ASSIGN, GSVA, and ssGSEA. Original models for some gene signatures are also available. A shiny app provides some functionality alongside for detailed command line accessibility.\n",
      "Version: 1.10.0\n",
      "Publication date: None\n",
      "Mantainer: Aubrey R. Odom-Mabey <aodom@bu.edu>\n",
      "Authors:\n",
      "Author: Aubrey R. Odom-Mabey [aut, cre, dtm] , David Jenkins [aut, org] , Xutao Wang [aut], Yue Zhao [ctb] , Christian Love [ctb], W. Evan Johnson [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.2.0) type: depends\n",
      "  - ASSIGN, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - ComplexHeatmap, type: imports\n",
      "  - DESeq2, type: imports\n",
      "  - DT, type: imports\n",
      "  - edgeR, type: imports\n",
      "  - gdata, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - GSVA, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - methods, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - rlang, type: imports\n",
      "  - ROCit, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - singscore, type: imports\n",
      "  - stats, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/TBSignatureProfiler.html\n",
      "\n",
      "Name: TNBC.CMS\n",
      "Description:\n",
      "This package implements a machine learning-based classifier for the assignment of consensus molecular subtypes to TNBC samples. It also provides functions to summarize genomic and clinical characteristics.\n",
      "Version: 1.14.0\n",
      "Publication date: None\n",
      "Mantainer: Doyeong Yu <parklab.bi@gmail.com>\n",
      "Authors:\n",
      "Author: Doyeong Yu, Jihyun Kim, In Hae Park, Charny Park\n",
      "Dependencies:\n",
      "  - R (>= 3.6.0) type: depends\n",
      "  - e1071, type: depends\n",
      "  - quadprog, type: depends\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - GSVA, type: imports\n",
      "  - pheatmap, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - pracma, type: imports\n",
      "  - GGally, type: imports\n",
      "  - R.utils, type: imports\n",
      "  - forestplot, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - ggpubr, type: imports\n",
      "  - survival, type: imports\n",
      "  - grid, type: imports\n",
      "  - stats, type: imports\n",
      "  - methods, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/TNBC.CMS.html\n",
      "\n",
      "Name: AffiXcan\n",
      "Description:\n",
      "Impute a GReX (Genetically Regulated Expression) for a set of genes in a sample of individuals, using a method based on the Total Binding Affinity (TBA). Statistical models to impute GReX can be trained with a training dataset where the real total expression values are known.\n",
      "Version: 1.16.0\n",
      "Publication date: None\n",
      "Mantainer: Alessandro Lussana <alessandro.lussana@protonmail.com>\n",
      "Authors:\n",
      "Author: Alessandro Lussana [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 3.6) type: depends\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - MultiAssayExperiment, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - crayon, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/AffiXcan.html\n",
      "\n",
      "Name: appreci8R\n",
      "Description:\n",
      "The appreci8R is an R version of our appreci8-algorithm - A Pipeline for PREcise variant Calling Integrating 8 tools. Variant calling results of our standard appreci8-tools (GATK, Platypus, VarScan, FreeBayes, LoFreq, SNVer, samtools and VarDict), as well as up to 5 additional tools is combined, evaluated and filtered.\n",
      "Version: 1.16.0\n",
      "Publication date: None\n",
      "Mantainer: Sarah Sandmann <sarah.sandmann@uni-muenster.de>\n",
      "Authors:\n",
      "Author: Sarah Sandmann\n",
      "Dependencies:\n",
      "  - shiny, type: imports\n",
      "  - shinyjs, type: imports\n",
      "  - DT, type: imports\n",
      "  - VariantAnnotation, type: imports\n",
      "  - BSgenome, type: imports\n",
      "  - BSgenome.Hsapiens.UCSC.hg19, type: imports\n",
      "  - TxDb.Hsapiens.UCSC.hg19.knownGene, type: imports\n",
      "  - Homo.sapiens, type: imports\n",
      "  - SNPlocs.Hsapiens.dbSNP144.GRCh37, type: imports\n",
      "  - XtraSNPlocs.Hsapiens.dbSNP144.GRCh37, type: imports\n",
      "  - rsnps, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - MafDb.1Kgenomes.phase3.hs37d5, type: imports\n",
      "  - MafDb.ExAC.r1.0.hs37d5, type: imports\n",
      "  - MafDb.gnomADex.r2.1.hs37d5, type: imports\n",
      "  - COSMIC.67, type: imports\n",
      "  - rentrez, type: imports\n",
      "  - PolyPhen.Hsapiens.dbSNP131, type: imports\n",
      "  - SIFT.Hsapiens.dbSNP137, type: imports\n",
      "  - seqinr, type: imports\n",
      "  - openxlsx, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - stringr, type: imports\n",
      "  - utils, type: imports\n",
      "  - stats, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - GenomicFeatures, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - GenomicScores, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/appreci8R.html\n",
      "\n",
      "Name: ARRmNormalization\n",
      "Description:\n",
      "Perform the Adaptive Robust Regression method (ARRm) for the normalization of methylation data from the Illumina Infinium HumanMethylation 450k assay.\n",
      "Version: 1.38.0\n",
      "Publication date: None\n",
      "Mantainer: Jean-Philippe Fortin <jfortin@jhsph.edu>\n",
      "Authors:\n",
      "Author: Jean-Philippe Fortin, Celia M.T. Greenwood, Aurelie Labbe.\n",
      "Dependencies:\n",
      "  - R (>= 2.15.1) type: depends\n",
      "  - ARRmData, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ARRmNormalization.html\n",
      "\n",
      "Name: AssessORF\n",
      "Description:\n",
      "In order to assess the quality of a set of predicted genes for a genome, evidence must first be mapped to that genome. Next, each gene must be categorized based on how strong the evidence is for or against that gene. The AssessORF package provides the functions and class structures necessary for accomplishing those tasks, using proteomic hits and evolutionarily conserved start codons as the forms of evidence.\n",
      "Version: 1.16.0\n",
      "Publication date: None\n",
      "Mantainer: Deepank Korandla <dkorandl@alumni.cmu.edu>\n",
      "Authors:\n",
      "Author: Deepank Korandla [aut, cre], Erik Wright [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - DECIPHER, type: depends\n",
      "  - Biostrings, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/AssessORF.html\n",
      "\n",
      "Name: BiFET\n",
      "Description:\n",
      "BiFET identifies TFs whose footprints are over-represented in target regions compared to background regions after correcting for the bias arising from the imbalance in read counts and GC contents between the target and background regions. For a given TF k, BiFET tests the null hypothesis that the target regions have the same probability of having footprints for the TF k as the background regions while correcting for the read count and GC content bias. For this, we use the number of target regions with footprints for TF k, t_k as a test statistic and calculate the p-value as the probability of observing t_k or more target regions with footprints under the null hypothesis.\n",
      "Version: 1.18.0\n",
      "Publication date: None\n",
      "Mantainer: Ahrim Youn <Ahrim.Youn@jax.org>\n",
      "Authors:\n",
      "Author: Ahrim Youn [aut, cre], Eladio Marquez [aut], Nathan Lawlor [aut], Michael Stitzel [aut], Duygu Ucar [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - stats, type: imports\n",
      "  - poibin, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/BiFET.html\n",
      "\n",
      "Name: BioMM\n",
      "Description:\n",
      "The identification of reproducible biological patterns from high-dimensional omics data is a key factor in understanding the biology of complex disease or traits. Incorporating prior biological knowledge into machine learning is an important step in advancing such research. We have proposed a biologically informed multi-stage machine learing framework termed BioMM specifically for phenotype prediction based on omics-scale data where we can evaluate different machine learning models with prior biological meta information.\n",
      "Version: 1.14.0\n",
      "Publication date: None\n",
      "Mantainer: Junfang Chen <junfang.chen33@gmail.com>\n",
      "Authors:\n",
      "Author: Junfang Chen and Emanuel Schwarz\n",
      "Dependencies:\n",
      "  - R (>= 3.6) type: depends\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - lattice, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - glmnet, type: imports\n",
      "  - rms, type: imports\n",
      "  - precrec, type: imports\n",
      "  - nsprcomp, type: imports\n",
      "  - ranger, type: imports\n",
      "  - e1071, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - vioplot, type: imports\n",
      "  - CMplot, type: imports\n",
      "  - imager, type: imports\n",
      "  - topGO, type: imports\n",
      "  - xlsx, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/BioMM.html\n",
      "\n",
      "Name: BioTIP\n",
      "Description:\n",
      "Adopting tipping-point theory to transcriptome profiles to unravel disease regulatory trajectory.\n",
      "Version: 1.12.0\n",
      "Publication date: None\n",
      "Mantainer: Yuxi (Jennifer) Sun <ysun11@uchicago.edu>, Zhezhen Wang <zhezhen@uchicago.edu>, and X Holly Yang <xyang2@uchicago.edu>\n",
      "Authors:\n",
      "Author: Zhezhen Wang, Andrew Goldstein, Yuxi Sun, Biniam Feleke, Qier An, Antonio Feliciano, Xinan Yang\n",
      "Dependencies:\n",
      "  - R (>= 3.6) type: depends\n",
      "  - igraph, type: imports\n",
      "  - cluster, type: imports\n",
      "  - psych, type: imports\n",
      "  - stringr, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - MASS, type: imports\n",
      "  - scran, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/BioTIP.html\n",
      "\n",
      "Name: biotmle\n",
      "Description:\n",
      "Tools for differential expression biomarker discovery based on microarray and next-generation sequencing data that leverage efficient semiparametric estimators of the average treatment effect for variable importance analysis. Estimation and inference of the (marginal) average treatment effects of potential biomarkers are computed by targeted minimum loss-based estimation, with joint, stable inference constructed across all biomarkers using a generalization of moderated statistics for use with the estimated efficient influence function. The procedure accommodates the use of ensemble machine learning for the estimation of nuisance functions.\n",
      "Version: 1.22.0\n",
      "Publication date: None\n",
      "Mantainer: Nima Hejazi <nh@nimahejazi.org>\n",
      "Authors:\n",
      "Author: Nima Hejazi [aut, cre, cph] , Alan Hubbard [aut, ths] , Mark van der Laan [aut, ths] , Weixin Cai [ctb] , Philippe Boileau [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - stats, type: imports\n",
      "  - methods, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - tibble, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - ggsci, type: imports\n",
      "  - superheat, type: imports\n",
      "  - assertthat, type: imports\n",
      "  - drtmle (>= 1.0.4) type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - limma, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/biotmle.html\n",
      "\n",
      "Name: categoryCompare\n",
      "Description:\n",
      "Calculates significant annotations (categories) in each of two (or more) feature (i.e. gene) lists, determines the overlap between the annotations, and returns graphical and tabular data about the significant annotations and which combinations of feature lists the annotations were found to be significant. Interactive exploration is facilitated through the use of RCytoscape (heavily suggested).\n",
      "Version: 1.42.0\n",
      "Publication date: None\n",
      "Mantainer: Robert M. Flight <rflight79@gmail.com>\n",
      "Authors:\n",
      "Author: Robert M. Flight <rflight79 at gmail.com>\n",
      "Dependencies:\n",
      "  - R (>= 2.10) type: depends\n",
      "  - Biobase, type: depends\n",
      "  - BiocGenerics, type: depends\n",
      "  - AnnotationDbi, type: imports\n",
      "  - hwriter, type: imports\n",
      "  - GSEABase, type: imports\n",
      "  - Category, type: imports\n",
      "  - GOstats, type: imports\n",
      "  - annotate, type: imports\n",
      "  - colorspace, type: imports\n",
      "  - graph, type: imports\n",
      "  - RCy3, type: imports\n",
      "  - methods, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/categoryCompare.html\n",
      "\n",
      "Name: CellMapper\n",
      "Description:\n",
      "Infers cell type-specific expression based on co-expression similarity with known cell type marker genes. Can make accurate predictions using publicly available expression data, even when a cell type has not been isolated before.\n",
      "Version: 1.24.0\n",
      "Publication date: None\n",
      "Mantainer: Brad Nelms <bnelms.research@gmail.com>\n",
      "Authors:\n",
      "Author: Brad Nelms\n",
      "Dependencies:\n",
      "  - S4Vectors, type: depends\n",
      "  - methods, type: depends\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/CellMapper.html\n",
      "\n",
      "Name: ChIPexoQual\n",
      "Description:\n",
      "Package with a quality control pipeline for ChIP-exo/nexus data.\n",
      "Version: 1.22.0\n",
      "Publication date: None\n",
      "Mantainer: Rene Welch <welch@stat.wisc.edu>\n",
      "Authors:\n",
      "Author: Rene Welch, Dongjun Chung, Sunduz Keles\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - GenomicAlignments, type: depends\n",
      "  - methods, type: imports\n",
      "  - utils, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - stats, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - ggplot2 (>= 1.0) type: imports\n",
      "  - data.table (>= 1.9.6) type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - biovizBase, type: imports\n",
      "  - broom (>= 0.4) type: imports\n",
      "  - RColorBrewer (>= 1.1) type: imports\n",
      "  - dplyr (>= 0.5) type: imports\n",
      "  - scales (>= 0.4.0) type: imports\n",
      "  - viridis (>= 0.3) type: imports\n",
      "  - hexbin (>= 1.27) type: imports\n",
      "  - rmarkdown, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>=2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ChIPexoQual.html\n",
      "\n",
      "Name: cleanUpdTSeq\n",
      "Description:\n",
      "This package implements a Naive Bayes classifier for accurately differentiating true polyadenylation sites (pA sites) from oligo(dT)-mediated 3' end sequencing such as PAS-Seq, PolyA-Seq and RNA-Seq by filtering out false polyadenylation sites, mainly due to oligo(dT)-mediated internal priming during reverse transcription. The classifer is highly accurate and outperforms other heuristic methods.\n",
      "Version: 1.36.0\n",
      "Publication date: None\n",
      "Mantainer: Jianhong Ou <Jianhong.Ou@duke.edu>; Lihua Julie Zhu <Julie.Zhu@umassmed.edu>\n",
      "Authors:\n",
      "Author: Sarah Sheppard, Haibo Liu, Jianhong Ou, Nathan Lawson, Lihua Julie Zhu\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - BSgenome.Drerio.UCSC.danRer7, type: depends\n",
      "  - methods, type: depends\n",
      "  - BSgenome, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - seqinr, type: imports\n",
      "  - e1071, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - utils, type: imports\n",
      "  - stringr, type: imports\n",
      "  - stats, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/cleanUpdTSeq.html\n",
      "\n",
      "Name: compartmap\n",
      "Description:\n",
      "Compartmap performs direct inference of higher-order chromatin from scRNA-seq and scATAC-seq. This package implements a James-Stein estimator for computing single-cell level higher-order chromatin domains. Further, we utilize random matrix theory as a method to de-noise correlation matrices to achieve a similar \"plaid-like\" patterning as observed in Hi-C and scHi-C data.\n",
      "Version: 1.16.0\n",
      "Publication date: None\n",
      "Mantainer: Benjamin Johnson <ben.johnson@vai.org>\n",
      "Authors:\n",
      "Author: Benjamin Johnson [aut, cre], Tim Triche [aut], Hui Shen [aut], Kasper Hansen [aut], Jean-Philippe Fortin [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.1.0) type: depends\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - RaggedExperiment, type: depends\n",
      "  - BiocSingular, type: depends\n",
      "  - HDF5Array, type: depends\n",
      "  - GenomicRanges, type: imports\n",
      "  - parallel, type: imports\n",
      "  - grid, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - scales, type: imports\n",
      "  - DelayedArray, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - DelayedMatrixStats, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - RMTstat, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3 + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/compartmap.html\n",
      "\n",
      "Name: CSSP\n",
      "Description:\n",
      "Power computation for ChIP-Seq data based on Bayesian estimation for local poisson counting process.\n",
      "Version: 1.36.0\n",
      "Publication date: None\n",
      "Mantainer: Chandler Zuo<zuo@stat.wisc.edu>\n",
      "Authors:\n",
      "Author: Chandler Zuo, Sunduz Keles\n",
      "Dependencies:\n",
      "  - methods, type: imports\n",
      "  - splines, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/CSSP.html\n",
      "\n",
      "Name: dce\n",
      "Description:\n",
      "Compute differential causal effects (dce) on (biological) networks. Given observational samples from a control experiment and non-control (e.g., cancer) for two genes A and B, we can compute differential causal effects with a (generalized) linear regression. If the causal effect of gene A on gene B in the control samples is different from the causal effect in the non-control samples the dce will differ from zero. We regularize the dce computation by the inclusion of prior network information from pathway databases such as KEGG.\n",
      "Version: 1.6.0\n",
      "Publication date: None\n",
      "Mantainer: Kim Philipp Jablonski <kim.philipp.jablonski@gmail.com>\n",
      "Authors:\n",
      "Author: Kim Philipp Jablonski [aut, cre] , Martin Pirkl [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - stats, type: imports\n",
      "  - methods, type: imports\n",
      "  - assertthat, type: imports\n",
      "  - graph, type: imports\n",
      "  - pcalg, type: imports\n",
      "  - purrr, type: imports\n",
      "  - tidyverse, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - ggraph, type: imports\n",
      "  - tidygraph, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - rlang, type: imports\n",
      "  - expm, type: imports\n",
      "  - MASS, type: imports\n",
      "  - edgeR, type: imports\n",
      "  - epiNEM, type: imports\n",
      "  - igraph, type: imports\n",
      "  - metap, type: imports\n",
      "  - mnem, type: imports\n",
      "  - naturalsort, type: imports\n",
      "  - ppcor, type: imports\n",
      "  - glm2, type: imports\n",
      "  - graphite, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - glue, type: imports\n",
      "  - Rgraphviz, type: imports\n",
      "  - harmonicmeanp, type: imports\n",
      "  - org.Hs.eg.db, type: imports\n",
      "  - logger, type: imports\n",
      "  - shadowtext, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/dce.html\n",
      "\n",
      "Name: DegNorm\n",
      "Description:\n",
      "This package performs degradation normalization in bulk RNA-seq data to improve differential expression analysis accuracy.\n",
      "Version: 1.8.2\n",
      "Publication date: None\n",
      "Mantainer: Ji-Ping Wang <jzwang@northwestern.edu>\n",
      "Authors:\n",
      "Author: Bin Xiong and Ji-Ping Wang\n",
      "Dependencies:\n",
      "  - R (>= 4.0.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - Rcpp (>= 1.0.2) type: imports\n",
      "  - GenomicFeatures, type: imports\n",
      "  - parallel, type: imports\n",
      "  - foreach, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - doParallel, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - GenomicAlignments, type: imports\n",
      "  - heatmaply, type: imports\n",
      "  - data.table, type: imports\n",
      "  - stats, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - plyr, type: imports\n",
      "  - plotly, type: imports\n",
      "  - utils, type: imports\n",
      "  - viridis, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL (>= 3)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/DegNorm.html\n",
      "\n",
      "Name: DelayedRandomArray\n",
      "Description:\n",
      "Implements a DelayedArray of random values where the realization of the sampled values is delayed until they are needed. Reproducible sampling within any subarray is achieved by chunking where each chunk is initialized with a different random seed and stream. The usual distributions in the stats package are supported, along with scalar, vector and arrays for the parameters.\n",
      "Version: 1.6.0\n",
      "Publication date: None\n",
      "Mantainer: Aaron Lun <infinite.monkeys.with.keyboards@gmail.com>\n",
      "Authors:\n",
      "Author: Aaron Lun [aut, cre]\n",
      "Dependencies:\n",
      "  - DelayedArray, type: depends\n",
      "  - methods, type: imports\n",
      "  - dqrng, type: imports\n",
      "  - Rcpp, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/DelayedRandomArray.html\n",
      "\n",
      "Name: DEsubs\n",
      "Description:\n",
      "DEsubs is a network-based systems biology package that extracts disease-perturbed subpathways within a pathway network as recorded by RNA-seq experiments. It contains an extensive and customizable framework covering a broad range of operation modes at all stages of the subpathway analysis, enabling a case-specific approach. The operation modes refer to the pathway network construction and processing, the subpathway extraction, visualization and enrichment analysis with regard to various biological and pharmacological features. Its capabilities render it a tool-guide for both the modeler and experimentalist for the identification of more robust systems-level biomarkers for complex diseases.\n",
      "Version: 1.24.0\n",
      "Publication date: None\n",
      "Mantainer: Aristidis G. Vrahatis <agvrahatis@upatras.gr>, Panos Balomenos <balomenos@upatras.gr>\n",
      "Authors:\n",
      "Author: Aristidis G. Vrahatis and Panos Balomenos\n",
      "Dependencies:\n",
      "  - R (>= 3.3) type: depends\n",
      "  - locfit, type: depends\n",
      "  - graph, type: imports\n",
      "  - igraph, type: imports\n",
      "  - RBGL, type: imports\n",
      "  - circlize, type: imports\n",
      "  - limma, type: imports\n",
      "  - edgeR, type: imports\n",
      "  - EBSeq, type: imports\n",
      "  - NBPSeq, type: imports\n",
      "  - stats, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - pheatmap, type: imports\n",
      "  - utils, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - jsonlite, type: imports\n",
      "  - tools, type: imports\n",
      "  - DESeq2, type: imports\n",
      "  - methods, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/DEsubs.html\n",
      "\n",
      "Name: dyebias\n",
      "Description:\n",
      "Many two-colour hybridizations suffer from a dye bias that is both gene-specific and slide-specific. The former depends on the content of the nucleotide used for labeling; the latter depends on the labeling percentage. The slide-dependency was hitherto not recognized, and made addressing the artefact impossible. Given a reasonable number of dye-swapped pairs of hybridizations, or of same vs. same hybridizations, both the gene- and slide-biases can be estimated and corrected using the GASSCO method (Margaritis et al., Mol. Sys. Biol. 5:266 (2009), doi:10.1038/msb.2009.21)\n",
      "Version: 1.58.0\n",
      "Publication date: None\n",
      "Mantainer: Philip Lijnzaad <plijnzaad@gmail.com>\n",
      "Authors:\n",
      "Author: Philip Lijnzaad and Thanasis Margaritis\n",
      "Dependencies:\n",
      "  - R (>= 1.4.1) type: depends\n",
      "  - marray, type: depends\n",
      "  - Biobase, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/dyebias.html\n",
      "\n",
      "Name: ExiMiR\n",
      "Description:\n",
      "This package contains functions for reading raw data in ImaGene TXT format obtained from Exiqon miRCURY LNA arrays, annotating them with appropriate GAL files, and normalizing them using a spike-in probe-based method. Other platforms and data formats are also supported.\n",
      "Version: 2.40.0\n",
      "Publication date: None\n",
      "Mantainer: Sylvain Gubian <DL.RSupport@pmi.com>\n",
      "Authors:\n",
      "Author: Sylvain Gubian <DL.RSupport at pmi.com>, Alain Sewer <DL.RSupport at pmi.com>, PMP SA\n",
      "Dependencies:\n",
      "  - R (>= 2.10) type: depends\n",
      "  - Biobase, type: depends\n",
      "  - affy, type: depends\n",
      "  - limma, type: depends\n",
      "  - affyio, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - preprocessCore, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ExiMiR.html\n",
      "\n",
      "Name: factDesign\n",
      "Description:\n",
      "This package provides a set of tools for analyzing data from a factorial designed microarray experiment, or any microarray experiment for which a linear model is appropriate. The functions can be used to evaluate tests of contrast of biological interest and perform single outlier detection.\n",
      "Version: 1.74.0\n",
      "Publication date: None\n",
      "Mantainer: Denise Scholtens <dscholtens@northwestern.edu>\n",
      "Authors:\n",
      "Author: Denise Scholtens\n",
      "Dependencies:\n",
      "  - Biobase, type: depends\n",
      "  - stats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/factDesign.html\n",
      "\n",
      "Name: FLAMES\n",
      "Description:\n",
      "Semi-supervised isoform detection and annotation from both bulk and single-cell long read RNA-seq data. Flames provides automated pipelines for analysing isoforms, as well as intermediate functions for manual execution.\n",
      "Version: 1.4.3\n",
      "Publication date: None\n",
      "Mantainer: Voogd Oliver <voogd.o@wehi.edu.au>\n",
      "Authors:\n",
      "Author: Tian Luyi [aut], Voogd Oliver [aut, cre], Schuster Jakob [aut], Wang Changqing [aut], Su Shian [aut], Ritchie Matthew [ctb]\n",
      "Dependencies:\n",
      "  - basilisk, type: imports\n",
      "  - bambu, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - circlize, type: imports\n",
      "  - ComplexHeatmap, type: imports\n",
      "  - cowplot, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - DropletUtils, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - GenomicFeatures, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - ggbio, type: imports\n",
      "  - grid, type: imports\n",
      "  - gridExtra, type: imports\n",
      "  - igraph, type: imports\n",
      "  - jsonlite, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - parallel, type: imports\n",
      "  - reticulate, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - SingleCellExperiment, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - scater, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - scuttle, type: imports\n",
      "  - stats, type: imports\n",
      "  - scran, type: imports\n",
      "  - stringr, type: imports\n",
      "  - MultiAssayExperiment, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - utils, type: imports\n",
      "  - withr, type: imports\n",
      "  - zlibbioc, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/FLAMES.html\n",
      "\n",
      "Name: flowBin\n",
      "Description:\n",
      "Software to combine flow cytometry data that has been multiplexed into multiple tubes with common markers between them, by establishing common bins across tubes in terms of the common markers, then determining expression within each tube for each bin in terms of the tube-specific markers.\n",
      "Version: 1.34.0\n",
      "Publication date: None\n",
      "Mantainer: Kieran O'Neill <koneill@bccrc.ca>\n",
      "Authors:\n",
      "Author: Kieran O'Neill\n",
      "Dependencies:\n",
      "  - methods, type: depends\n",
      "  - flowCore, type: depends\n",
      "  - flowFP, type: depends\n",
      "  - R (>= 2.10) type: depends\n",
      "  - class, type: imports\n",
      "  - limma, type: imports\n",
      "  - snow, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/flowBin.html\n",
      "\n",
      "Name: flowTime\n",
      "Description:\n",
      "This package facilitates analysis of both timecourse and steady state flow cytometry experiments. This package was originially developed for quantifying the function of gene regulatory networks in yeast (strain W303) expressing fluorescent reporter proteins using BD Accuri C6 and SORP cytometers. However, the functions are for the most part general and may be adapted for analysis of other organisms using other flow cytometers. Functions in this package facilitate the annotation of flow cytometry data with experimental metadata, as often required for publication and general ease-of-reuse. Functions for creating, saving and loading gate sets are also included. In the past, we have typically generated summary statistics for each flowset for each timepoint and then annotated and analyzed these summary statistics. This method loses a great deal of the power that comes from the large amounts of individual cell data generated in flow cytometry, by essentially collapsing this data into a bulk measurement after subsetting. In addition to these summary functions, this package also contains functions to facilitate annotation and analysis of steady-state or time-lapse data utilizing all of the data collected from the thousands of individual cells in each sample.\n",
      "Version: 1.22.2\n",
      "Publication date: None\n",
      "Mantainer: R. Clay Wright <wright.clay@gmail.com>\n",
      "Authors:\n",
      "Author: R. Clay Wright [aut, cre], Nick Bolten [aut], Edith Pierre-Jerome [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.4) type: depends\n",
      "  - flowCore, type: depends\n",
      "  - utils, type: imports\n",
      "  - dplyr (>= 1.0.0) type: imports\n",
      "  - tibble, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - plyr, type: imports\n",
      "  - rlang, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/flowTime.html\n",
      "\n",
      "Name: frmaTools\n",
      "Description:\n",
      "Tools for advanced use of the frma package.\n",
      "Version: 1.50.0\n",
      "Publication date: None\n",
      "Mantainer: Matthew N. McCall <mccallm@gmail.com>\n",
      "Authors:\n",
      "Author: Matthew N. McCall <mccallm at gmail.com>, Rafael A. Irizarry <rafa at jhu.edu>\n",
      "Dependencies:\n",
      "  - R (>= 2.10.0) type: depends\n",
      "  - affy, type: depends\n",
      "  - Biobase, type: imports\n",
      "  - DBI, type: imports\n",
      "  - methods, type: imports\n",
      "  - preprocessCore, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/frmaTools.html\n",
      "\n",
      "Name: GateFinder\n",
      "Description:\n",
      "Given a vector of cluster memberships for a cell population, identifies a sequence of gates (polygon filters on 2D scatter plots) for isolation of that cell type.\n",
      "Version: 1.18.0\n",
      "Publication date: None\n",
      "Mantainer: Nima Aghaeepour <naghaeep@gmail.com>\n",
      "Authors:\n",
      "Author: Nima Aghaeepour <naghaeep at gmail.com>, Erin F. Simonds <erin.simonds at gmail.com>\n",
      "Dependencies:\n",
      "  - splancs, type: imports\n",
      "  - mvoutlier, type: imports\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - diptest, type: imports\n",
      "  - flowCore, type: imports\n",
      "  - flowFP, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/GateFinder.html\n",
      "\n",
      "Name: GISPA\n",
      "Description:\n",
      "GISPA is a method intended for the researchers who are interested in defining gene sets with similar, a priori specified molecular profile. GISPA method has been previously published in Nucleic Acid Research (Kowalski et al., 2016; PMID: 26826710).\n",
      "Version: 1.22.0\n",
      "Publication date: None\n",
      "Mantainer: Bhakti Dwivedi <bhakti.dwivedi@emory.edu>\n",
      "Authors:\n",
      "Author: Bhakti Dwivedi and Jeanne Kowalski\n",
      "Dependencies:\n",
      "  - R (>= 3.5) type: depends\n",
      "  - Biobase, type: imports\n",
      "  - changepoint, type: imports\n",
      "  - data.table, type: imports\n",
      "  - genefilter, type: imports\n",
      "  - graphics, type: imports\n",
      "  - GSEABase, type: imports\n",
      "  - HH, type: imports\n",
      "  - lattice, type: imports\n",
      "  - latticeExtra, type: imports\n",
      "  - plyr, type: imports\n",
      "  - scatterplot3d, type: imports\n",
      "  - stats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/GISPA.html\n",
      "\n",
      "Name: GSCA\n",
      "Description:\n",
      "GSCA takes as input several lists of activated and repressed genes. GSCA then searches through a compendium of publicly available gene expression profiles for biological contexts that are enriched with a specified pattern of gene expression. GSCA provides both traditional R functions and interactive, user-friendly user interface.\n",
      "Version: 2.28.0\n",
      "Publication date: None\n",
      "Mantainer: Zhicheng Ji <zji4@jhu.edu>\n",
      "Authors:\n",
      "Author: Zhicheng Ji, Hongkai Ji\n",
      "Dependencies:\n",
      "  - shiny, type: depends\n",
      "  - sp, type: depends\n",
      "  - gplots, type: depends\n",
      "  - ggplot2, type: depends\n",
      "  - reshape2, type: depends\n",
      "  - RColorBrewer, type: depends\n",
      "  - rhdf5, type: depends\n",
      "  - R (>= 2.10.0) type: depends\n",
      "  - graphics, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL(>=2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/GSCA.html\n",
      "\n",
      "Name: gsean\n",
      "Description:\n",
      "Biological molecules in a living organism seldom work individually. They usually interact each other in a cooperative way. Biological process is too complicated to understand without considering such interactions. Thus, network-based procedures can be seen as powerful methods for studying complex process. However, many methods are devised for analyzing individual genes. It is said that techniques based on biological networks such as gene co-expression are more precise ways to represent information than those using lists of genes only. This package is aimed to integrate the gene expression and biological network. A biological network is constructed from gene expression data and it is used for Gene Set Enrichment Analysis.\n",
      "Version: 1.18.0\n",
      "Publication date: None\n",
      "Mantainer: Dongmin Jung <dmdmjung@gmail.com>\n",
      "Authors:\n",
      "Author: Dongmin Jung\n",
      "Dependencies:\n",
      "  - R (>= 3.5) type: depends\n",
      "  - fgsea, type: depends\n",
      "  - PPInfer, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/gsean.html\n",
      "\n",
      "Name: HDTD\n",
      "Description:\n",
      "Characterization of intra-individual variability using physiologically relevant measurements provides important insights into fundamental biological questions ranging from cell type identity to tumor development. For each individual, the data measurements can be written as a matrix with the different subsamples of the individual recorded in the columns and the different phenotypic units recorded in the rows. Datasets of this type are called high-dimensional transposable data. The HDTD package provides functions for conducting statistical inference for the mean relationship between the row and column variables and for the covariance structure within and between the row and column variables.\n",
      "Version: 1.32.2\n",
      "Publication date: None\n",
      "Mantainer: Anestis Touloumis <A.Touloumis@brighton.ac.uk>\n",
      "Authors:\n",
      "Author: Anestis Touloumis [cre, aut] , John C. Marioni [aut] , Simon Tavar\\'{e} [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - stats, type: imports\n",
      "  - Rcpp (>= 1.0.1) type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/HDTD.html\n",
      "\n",
      "Name: iCNV\n",
      "Description:\n",
      "Integrative copy number variation (CNV) detection from multiple platform and experimental design.\n",
      "Version: 1.18.0\n",
      "Publication date: None\n",
      "Mantainer: Zilu Zhou <zhouzilu@pennmedicine.upenn.edu>\n",
      "Authors:\n",
      "Author: Zilu Zhou, Nancy Zhang\n",
      "Dependencies:\n",
      "  - R (>= 3.3.1) type: depends\n",
      "  - CODEX, type: depends\n",
      "  - fields, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - truncnorm, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - data.table, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - rlang, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/iCNV.html\n",
      "\n",
      "Name: iPAC\n",
      "Description:\n",
      "iPAC is a novel tool to identify somatic amino acid mutation clustering within proteins while taking into account protein structure.\n",
      "Version: 1.42.0\n",
      "Publication date: None\n",
      "Mantainer: Gregory Ryslik <gregory.ryslik@yale.edu>\n",
      "Authors:\n",
      "Author: Gregory Ryslik, Hongyu Zhao\n",
      "Dependencies:\n",
      "  - R (>= 2.15) type: depends\n",
      "  - gdata, type: depends\n",
      "  - scatterplot3d, type: depends\n",
      "  - Biostrings, type: depends\n",
      "  - multtest, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/iPAC.html\n",
      "\n",
      "Name: iterClust\n",
      "Description:\n",
      "A framework for performing clustering analysis iteratively.\n",
      "Version: 1.20.0\n",
      "Publication date: None\n",
      "Mantainer: Hongxu Ding <hd2326@columbia.edu>\n",
      "Authors:\n",
      "Author: Hongxu Ding and Andrea Califano\n",
      "Dependencies:\n",
      "  - R (>= 3.4.1) type: depends\n",
      "  - Biobase, type: imports\n",
      "  - cluster, type: imports\n",
      "  - stats, type: imports\n",
      "  - methods, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/iterClust.html\n",
      "\n",
      "Name: les\n",
      "Description:\n",
      "The 'les' package estimates Loci of Enhanced Significance (LES) in tiling microarray data. These are regions of regulation such as found in differential transcription, CHiP-chip, or DNA modification analysis. The package provides a universal framework suitable for identifying differential effects in tiling microarray data sets, and is independent of the underlying statistics at the level of single probes.\n",
      "Version: 1.48.0\n",
      "Publication date: None\n",
      "Mantainer: Julian Gehring <jg-bioc@gmx.com>\n",
      "Authors:\n",
      "Author: Julian Gehring, Clemens Kreutz, Jens Timmer\n",
      "Dependencies:\n",
      "  - R (>= 2.13.2) type: depends\n",
      "  - methods, type: depends\n",
      "  - graphics, type: depends\n",
      "  - fdrtool, type: depends\n",
      "  - boot, type: imports\n",
      "  - gplots, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/les.html\n",
      "\n",
      "Name: massiR\n",
      "Description:\n",
      "Predicts the sex of samples in gene expression microarray datasets\n",
      "Version: 1.34.0\n",
      "Publication date: None\n",
      "Mantainer: Sam Buckberry <sam.buckberry@adelaide.edu.au>\n",
      "Authors:\n",
      "Author: Sam Buckberry\n",
      "Dependencies:\n",
      "  - cluster, type: depends\n",
      "  - gplots, type: depends\n",
      "  - diptest, type: depends\n",
      "  - Biobase, type: depends\n",
      "  - R (>= 3.0.2) type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/massiR.html\n",
      "\n",
      "Name: metapone\n",
      "Description:\n",
      "The package conducts pathway testing from untargetted metabolomics data. It requires the user to supply feature-level test results, from case-control testing, regression, or other suitable feature-level tests for the study design. Weights are given to metabolic features based on how many metabolites they could potentially match to. The package can combine positive and negative mode results in pathway tests.\n",
      "Version: 1.4.0\n",
      "Publication date: None\n",
      "Mantainer: Tianwei Yu <yutianwei@cuhk.edu.cn>\n",
      "Authors:\n",
      "Author: Leqi Tian [aut], Tianwei Yu [aut], Tianwei Yu [cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.1.0) type: depends\n",
      "  - BiocParallel, type: depends\n",
      "  - fields, type: depends\n",
      "  - markdown, type: depends\n",
      "  - fdrtool, type: depends\n",
      "  - fgsea, type: depends\n",
      "  - ggplot2, type: depends\n",
      "  - ggrepel, type: depends\n",
      "  - methods, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/metapone.html\n",
      "\n",
      "Name: NoRCE\n",
      "Description:\n",
      "While some non-coding RNAs (ncRNAs) are assigned critical regulatory roles, most remain functionally uncharacterized. This presents a challenge whenever an interesting set of ncRNAs needs to be analyzed in a functional context. Transcripts located close-by on the genome are often regulated together. This genomic proximity on the sequence can hint to a functional association. We present a tool, NoRCE, that performs cis enrichment analysis for a given set of ncRNAs. Enrichment is carried out using the functional annotations of the coding genes located proximal to the input ncRNAs. Other biologically relevant information such as topologically associating domain (TAD) boundaries, co-expression patterns, and miRNA target prediction information can be incorporated to conduct a richer enrichment analysis. To this end, NoRCE includes several relevant datasets as part of its data repository, including cell-line specific TAD boundaries, functional gene sets, and expression data for coding & ncRNAs specific to cancer. Additionally, the users can utilize custom data files in their investigation. Enrichment results can be retrieved in a tabular format or visualized in several different ways. NoRCE is currently available for the following species: human, mouse, rat, zebrafish, fruit fly, worm, and yeast.\n",
      "Version: 1.10.0\n",
      "Publication date: None\n",
      "Mantainer: Gulden Olgun <gulden@cs.bilkent.edu.tr>\n",
      "Authors:\n",
      "Author: Gulden Olgun [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.2.0) type: depends\n",
      "  - KEGGREST, type: imports\n",
      "  - png, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - graphics, type: imports\n",
      "  - RSQLite, type: imports\n",
      "  - DBI, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - stringr, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - reactome.db, type: imports\n",
      "  - rWikiPathways, type: imports\n",
      "  - RCurl, type: imports\n",
      "  - dbplyr, type: imports\n",
      "  - utils, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - igraph, type: imports\n",
      "  - stats, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - readr, type: imports\n",
      "  - GO.db, type: imports\n",
      "  - zlibbioc, type: imports\n",
      "  - biomaRt, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - GenomicFeatures, type: imports\n",
      "  - AnnotationDbi, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/NoRCE.html\n",
      "\n",
      "Name: panp\n",
      "Description:\n",
      "A function to make gene presence/absence calls based on distance from negative strand matching probesets (NSMP) which are derived from Affymetrix annotation. PANP is applied after gene expression values are created, and therefore can be used after any preprocessing method such as MAS5 or GCRMA, or PM-only methods like RMA. NSMP sets have been established for the HGU133A and HGU133-Plus-2.0 chipsets to date.\n",
      "Version: 1.68.0\n",
      "Publication date: None\n",
      "Mantainer: Peter Warren <peter.warren@verizon.net>\n",
      "Authors:\n",
      "Author: Peter Warren\n",
      "Dependencies:\n",
      "  - R (>= 2.10) type: depends\n",
      "  - affy, type: depends\n",
      "  - Biobase, type: depends\n",
      "  - Biobase, type: imports\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/panp.html\n",
      "\n",
      "Name: Path2PPI\n",
      "Description:\n",
      "Package to predict protein-protein interaction (PPI) networks in target organisms for which only a view information about PPIs is available. Path2PPI predicts PPI networks based on sets of proteins which can belong to a certain pathway from well-established model organisms. It helps to combine and transfer information of a certain pathway or biological process from several reference organisms to one target organism. Path2PPI only depends on the sequence similarity of the involved proteins.\n",
      "Version: 1.28.0\n",
      "Publication date: None\n",
      "Mantainer: Oliver Philipp <contact@oliverphilipp.info>\n",
      "Authors:\n",
      "Author: Oliver Philipp [aut, cre], Ina Koch [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 3.2.1) type: depends\n",
      "  - igraph (>= 1.0.1) type: depends\n",
      "  - methods, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/Path2PPI.html\n",
      "\n",
      "Name: pathRender\n",
      "Description:\n",
      "build graphs from pathway databases, render them by Rgraphviz.\n",
      "Version: 1.66.0\n",
      "Publication date: None\n",
      "Mantainer: Vince Carey <stvjc@channing.harvard.edu>\n",
      "Authors:\n",
      "Author: Li Long <lilong at isb-sib.ch>\n",
      "Dependencies:\n",
      "  - graph, type: depends\n",
      "  - Rgraphviz, type: depends\n",
      "  - RColorBrewer, type: depends\n",
      "  - cMAP, type: depends\n",
      "  - AnnotationDbi, type: depends\n",
      "  - methods, type: depends\n",
      "  - stats4, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/pathRender.html\n",
      "\n",
      "Name: PhenStat\n",
      "Description:\n",
      "Package contains methods for statistical analysis of phenotypic data.\n",
      "Version: 2.34.0\n",
      "Publication date: None\n",
      "Mantainer: Hamed Haselimashhadi <hamedhm@ebi.ac.uk>\n",
      "Authors:\n",
      "Author: Natalja Kurbatova, Natasha Karp, Jeremy Mason, Hamed Haselimashhadi\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - SmoothWin, type: imports\n",
      "  - methods, type: imports\n",
      "  - car, type: imports\n",
      "  - nlme, type: imports\n",
      "  - nortest, type: imports\n",
      "  - MASS, type: imports\n",
      "  - msgps, type: imports\n",
      "  - logistf, type: imports\n",
      "  - knitr, type: imports\n",
      "  - tools, type: imports\n",
      "  - pingr, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - reshape, type: imports\n",
      "  - corrplot, type: imports\n",
      "  - graph, type: imports\n",
      "  - lme4, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - utils, type: imports\n",
      "  - stats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/PhenStat.html\n",
      "\n",
      "Name: qpcrNorm\n",
      "Description:\n",
      "The package contains functions to perform normalization of high-throughput qPCR data. Basic functions for processing raw Ct data plus functions to generate diagnostic plots are also available.\n",
      "Version: 1.56.0\n",
      "Publication date: None\n",
      "Mantainer: Jessica Mar <jess@jimmy.harvard.edu>\n",
      "Authors:\n",
      "Author: Jessica Mar\n",
      "Dependencies:\n",
      "  - methods, type: depends\n",
      "  - Biobase, type: depends\n",
      "  - limma, type: depends\n",
      "  - affy, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/qpcrNorm.html\n",
      "\n",
      "Name: ramr\n",
      "Description:\n",
      "ramr is an R package for detection of low-frequency aberrant methylation events in large data sets obtained by methylation profiling using array or high-throughput bisulfite sequencing. In addition, package provides functions to visualize found aberrantly methylated regions (AMRs), to generate sets of all possible regions to be used as reference sets for enrichment analysis, and to generate biologically relevant test data sets for performance evaluation of AMR/DMR search algorithms.\n",
      "Version: 1.6.0\n",
      "Publication date: None\n",
      "Mantainer: Oleksii Nikolaienko <oleksii.nikolaienko@gmail.com>\n",
      "Authors:\n",
      "Author: Oleksii Nikolaienko [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - parallel, type: depends\n",
      "  - doParallel, type: depends\n",
      "  - foreach, type: depends\n",
      "  - doRNG, type: depends\n",
      "  - methods, type: depends\n",
      "  - IRanges, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - EnvStats, type: imports\n",
      "  - ExtDist, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ramr.html\n",
      "\n",
      "Name: randPack\n",
      "Description:\n",
      "A suite of classes and functions for randomizing patients in clinical trials.\n",
      "Version: 1.44.0\n",
      "Publication date: None\n",
      "Mantainer: Robert Gentleman <rgentlem@gmail.com>\n",
      "Authors:\n",
      "Author: Vincent Carey <stvjc at channing.harvard.edu> and Robert Gentleman\n",
      "Dependencies:\n",
      "  - methods, type: depends\n",
      "  - Biobase, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic 2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/randPack.html\n",
      "\n",
      "Name: Rbwa\n",
      "Description:\n",
      "Provides an R wrapper for BWA alignment algorithms. Both BWA-backtrack and BWA-MEM are available. Convenience function to build a BWA index from a reference genome is also provided. Currently not supported for Windows machines.\n",
      "Version: 1.2.0\n",
      "Publication date: None\n",
      "Mantainer: Jean-Philippe Fortin <fortin946@gmail.com>\n",
      "Authors:\n",
      "Author: Jean-Philippe Fortin [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/Rbwa.html\n",
      "\n",
      "Name: RCM\n",
      "Description:\n",
      "Combine ideas of log-linear analysis of contingency table, flexible response function estimation and empirical Bayes dispersion estimation for explorative visualization of microbiome datasets. The package includes unconstrained as well as constrained analysis. In addition, diagnostic plot to detect lack of fit are available.\n",
      "Version: 1.14.0\n",
      "Publication date: None\n",
      "Mantainer: Stijn Hawinkel <stijn.hawinkel@psb.ugent.be>\n",
      "Authors:\n",
      "Author: Stijn Hawinkel [cre, aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - DBI, type: depends\n",
      "  - RColorBrewer, type: imports\n",
      "  - alabama, type: imports\n",
      "  - edgeR, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - tseries, type: imports\n",
      "  - stats, type: imports\n",
      "  - VGAM, type: imports\n",
      "  - ggplot2 (>= 2.2.1.9000) type: imports\n",
      "  - nleqslv, type: imports\n",
      "  - phyloseq, type: imports\n",
      "  - tensor, type: imports\n",
      "  - MASS, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - methods, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/RCM.html\n",
      "\n",
      "Name: RNAmodR.AlkAnilineSeq\n",
      "Description:\n",
      "RNAmodR.AlkAnilineSeq implements the detection of m7G, m3C and D modifications on RNA from experimental data generated with the AlkAnilineSeq protocol. The package builds on the core functionality of the RNAmodR package to detect specific patterns of the modifications in high throughput sequencing data.\n",
      "Version: 1.12.0\n",
      "Publication date: None\n",
      "Mantainer: Felix G.M. Ernst <felix.gm.ernst@outlook.com>\n",
      "Authors:\n",
      "Author: Felix G.M. Ernst [aut, cre] , Denis L.J. Lafontaine [ctb, fnd]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - RNAmodR, type: depends\n",
      "  - methods, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - Gviz, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/RNAmodR.AlkAnilineSeq.html\n",
      "\n",
      "Name: scCB2\n",
      "Description:\n",
      "scCB2 is an R package implementing CB2 for distinguishing real cells from empty droplets in droplet-based single cell RNA-seq experiments (especially for 10x Chromium). It is based on clustering similar barcodes and calculating Monte-Carlo p-value for each cluster to test against background distribution. This cluster-level test outperforms single-barcode-level tests in dealing with low count barcodes and homogeneous sequencing library, while keeping FDR well controlled.\n",
      "Version: 1.8.0\n",
      "Publication date: None\n",
      "Mantainer: Zijian Ni <zni25@wisc.edu>\n",
      "Authors:\n",
      "Author: Zijian Ni [aut, cre], Shuyang Chen [ctb], Christina Kendziorski [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 3.6.0) type: depends\n",
      "  - SingleCellExperiment, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - methods, type: imports\n",
      "  - utils, type: imports\n",
      "  - stats, type: imports\n",
      "  - edgeR, type: imports\n",
      "  - rhdf5, type: imports\n",
      "  - parallel, type: imports\n",
      "  - DropletUtils, type: imports\n",
      "  - doParallel, type: imports\n",
      "  - iterators, type: imports\n",
      "  - foreach, type: imports\n",
      "  - Seurat, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/scCB2.html\n",
      "\n",
      "Name: SCOPE\n",
      "Description:\n",
      "Whole genome single-cell DNA sequencing (scDNA-seq) enables characterization of copy number profiles at the cellular level. This circumvents the averaging effects associated with bulk-tissue sequencing and has increased resolution yet decreased ambiguity in deconvolving cancer subclones and elucidating cancer evolutionary history. ScDNA-seq data is, however, sparse, noisy, and highly variable even within a homogeneous cell population, due to the biases and artifacts that are introduced during the library preparation and sequencing procedure. Here, we propose SCOPE, a normalization and copy number estimation method for scDNA-seq data. The distinguishing features of SCOPE include: (i) utilization of cell-specific Gini coefficients for quality controls and for identification of normal/diploid cells, which are further used as negative control samples in a Poisson latent factor model for normalization; (ii) modeling of GC content bias using an expectation-maximization algorithm embedded in the Poisson generalized linear models, which accounts for the different copy number states along the genome; (iii) a cross-sample iterative segmentation procedure to identify breakpoints that are shared across cells from the same genetic background.\n",
      "Version: 1.10.0\n",
      "Publication date: None\n",
      "Mantainer: Rujin Wang <rujin@email.unc.edu>\n",
      "Authors:\n",
      "Author: Rujin Wang, Danyu Lin, Yuchao Jiang\n",
      "Dependencies:\n",
      "  - R (>= 3.6.0) type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - IRanges, type: depends\n",
      "  - Rsamtools, type: depends\n",
      "  - GenomeInfoDb, type: depends\n",
      "  - BSgenome.Hsapiens.UCSC.hg19, type: depends\n",
      "  - stats, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - utils, type: imports\n",
      "  - DescTools, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - gplots, type: imports\n",
      "  - foreach, type: imports\n",
      "  - parallel, type: imports\n",
      "  - doParallel, type: imports\n",
      "  - DNAcopy, type: imports\n",
      "  - BSgenome, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/SCOPE.html\n",
      "\n",
      "Name: scRecover\n",
      "Description:\n",
      "scRecover is an R package for imputation of single-cell RNA-seq (scRNA-seq) data. It will detect and impute dropout values in a scRNA-seq raw read counts matrix while keeping the real zeros unchanged, since there are both dropout zeros and real zeros in scRNA-seq data. By combination with scImpute, SAVER and MAGIC, scRecover not only detects dropout and real zeros at higher accuracy, but also improve the downstream clustering and visualization results.\n",
      "Version: 1.14.1\n",
      "Publication date: None\n",
      "Mantainer: Zhun Miao <miaoz13@tsinghua.org.cn>\n",
      "Authors:\n",
      "Author: Zhun Miao, Xuegong Zhang <zhangxg at tsinghua.edu.cn>\n",
      "Dependencies:\n",
      "  - R (>= 3.4.0) type: depends\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - methods, type: imports\n",
      "  - graphics, type: imports\n",
      "  - doParallel, type: imports\n",
      "  - foreach, type: imports\n",
      "  - parallel, type: imports\n",
      "  - penalized, type: imports\n",
      "  - kernlab, type: imports\n",
      "  - rsvd, type: imports\n",
      "  - Matrix (>= 1.2-14) type: imports\n",
      "  - MASS (>= 7.3-45) type: imports\n",
      "  - pscl (>= 1.4.9) type: imports\n",
      "  - bbmle (>= 1.0.18) type: imports\n",
      "  - gamlss (>= 4.4-0) type: imports\n",
      "  - preseqR (>= 4.0.0) type: imports\n",
      "  - SAVER (>= 1.1.1) type: imports\n",
      "  - BiocParallel, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/scRecover.html\n",
      "\n",
      "Name: SeqSQC\n",
      "Description:\n",
      "The SeqSQC is designed to identify problematic samples in NGS data, including samples with gender mismatch, contamination, cryptic relatedness, and population outlier.\n",
      "Version: 1.20.0\n",
      "Publication date: None\n",
      "Mantainer: Qian Liu <qliu7@buffalo.edu>\n",
      "Authors:\n",
      "Author: Qian Liu [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 3.4) type: depends\n",
      "  - ExperimentHub, type: depends\n",
      "  - SNPRelate, type: depends\n",
      "  - e1071, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - gdsfmt, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - GGally, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - methods, type: imports\n",
      "  - rbokeh, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - rmarkdown, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/SeqSQC.html\n",
      "\n",
      "Name: SIM\n",
      "Description:\n",
      "Finds associations between two human genomic datasets.\n",
      "Version: 1.68.0\n",
      "Publication date: None\n",
      "Mantainer: Renee X. de Menezes <r.menezes@nki.nl>\n",
      "Authors:\n",
      "Author: Renee X. de Menezes and Judith M. Boer\n",
      "Dependencies:\n",
      "  - R (>= 3.5) type: depends\n",
      "  - quantreg, type: depends\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - globaltest, type: imports\n",
      "  - quantsmooth, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/SIM.html\n",
      "\n",
      "Name: SMITE\n",
      "Description:\n",
      "This package builds on the Epimods framework which facilitates finding weighted subnetworks (\"modules\") on Illumina Infinium 27k arrays using the SpinGlass algorithm, as implemented in the iGraph package. We have created a class of gene centric annotations associated with p-values and effect sizes and scores from any researchers prior statistical results to find functional modules.\n",
      "Version: 1.26.0\n",
      "Publication date: None\n",
      "Mantainer: Neil Ari Wijetunga <nawijet@gmail.com>, Andrew Damon Johnston <Andrew.Johnston@med.einstein.yu.edu>\n",
      "Authors:\n",
      "Author: Neil Ari Wijetunga, Andrew Damon Johnston, John Murray Greally\n",
      "Dependencies:\n",
      "  - R (>= 3.5) type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - scales, type: imports\n",
      "  - plyr, type: imports\n",
      "  - Hmisc, type: imports\n",
      "  - AnnotationDbi, type: imports\n",
      "  - org.Hs.eg.db, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - reactome.db, type: imports\n",
      "  - KEGGREST, type: imports\n",
      "  - BioNet, type: imports\n",
      "  - goseq, type: imports\n",
      "  - methods, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - igraph, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - tools, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - geneLenDataBase, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>=2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/SMITE.html\n",
      "\n",
      "Name: STAN\n",
      "Description:\n",
      "Genome segmentation with hidden Markov models has become a useful tool to annotate genomic elements, such as promoters and enhancers. STAN (genomic STate ANnotation) implements (bidirectional) hidden Markov models (HMMs) using a variety of different probability distributions, which can model a wide range of current genomic data (e.g. continuous, discrete, binary). STAN de novo learns and annotates the genome into a given number of 'genomic states'. The 'genomic states' may for instance reflect distinct genome-associated protein complexes (e.g. 'transcription states') or describe recurring patterns of chromatin features (referred to as 'chromatin states'). Unlike other tools, STAN also allows for the integration of strand-specific (e.g. RNA) and non-strand-specific data (e.g. ChIP).\n",
      "Version: 2.26.2\n",
      "Publication date: None\n",
      "Mantainer: Rafael Campos-Martin <campos@mpipz.mpg.de>\n",
      "Authors:\n",
      "Author: Benedikt Zacher, Julia Ertl, Rafael Campos-Martin, Julien Gagneur, Achim Tresch\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - poilog, type: depends\n",
      "  - parallel, type: depends\n",
      "  - GenomicRanges, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - Gviz, type: imports\n",
      "  - Rsolnp, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/STAN.html\n",
      "\n",
      "Name: TargetScore\n",
      "Description:\n",
      "Infer the posterior distributions of microRNA targets by probabilistically modelling the likelihood microRNA-overexpression fold-changes and sequence-based scores. Variaitonal Bayesian Gaussian mixture model (VB-GMM) is applied to log fold-changes and sequence scores to obtain the posteriors of latent variable being the miRNA targets. The final targetScore is computed as the sigmoid-transformed fold-change weighted by the averaged posteriors of target components over all of the features.\n",
      "Version: 1.36.0\n",
      "Publication date: None\n",
      "Mantainer: Yue Li <yueli@cs.toronto.edu>\n",
      "Authors:\n",
      "Author: Yue Li\n",
      "Dependencies:\n",
      "  - pracma, type: depends\n",
      "  - Matrix, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/TargetScore.html\n",
      "\n",
      "Name: TimiRGeN\n",
      "Description:\n",
      "TimiRGeN (Time Incorporated miR-mRNA Generation of Networks) is a novel R package which functionally analyses and integrates time course miRNA-mRNA differential expression data. This tool can generate small networks within R or export results into cytoscape or pathvisio for more detailed network construction and hypothesis generation. This tool is created for researchers that wish to dive deep into time series multi-omic datasets. TimiRGeN goes further than many other tools in terms of data reduction. Here, potentially hundreds-of-thousands of potential miRNA-mRNA interactions can be whittled down into a handful of high confidence miRNA-mRNA interactions affecting a signalling pathway, across a time course.\n",
      "Version: 1.8.0\n",
      "Publication date: None\n",
      "Mantainer: Krutik Patel <krutik.patel@newcastle.ac.uk>\n",
      "Authors:\n",
      "Author: Krutik Patel [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - Mfuzz, type: depends\n",
      "  - MultiAssayExperiment, type: depends\n",
      "  - biomaRt, type: imports\n",
      "  - clusterProfiler, type: imports\n",
      "  - dplyr (>= 0.8.4) type: imports\n",
      "  - FreqProf, type: imports\n",
      "  - gtools (>= 3.8.1) type: imports\n",
      "  - gplots, type: imports\n",
      "  - ggdendro, type: imports\n",
      "  - gghighlight, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - igraph (>= 1.2.4.2) type: imports\n",
      "  - RCy3, type: imports\n",
      "  - readxl, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - rWikiPathways, type: imports\n",
      "  - scales, type: imports\n",
      "  - stats, type: imports\n",
      "  - tidyr (>= 1.0.2) type: imports\n",
      "  - stringr (>= 1.4.0) type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/TimiRGeN.html\n",
      "\n",
      "Name: weaver\n",
      "Description:\n",
      "This package provides enhancements on the Sweave() function in the base package. In particular a facility for caching code chunk results is included.\n",
      "Version: 1.64.0\n",
      "Publication date: None\n",
      "Mantainer: Seth Falcon <seth@userprimary.net>\n",
      "Authors:\n",
      "Author: Seth Falcon\n",
      "Dependencies:\n",
      "  - R (>= 2.5.0) type: depends\n",
      "  - digest, type: depends\n",
      "  - tools, type: depends\n",
      "  - utils, type: depends\n",
      "  - codetools, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/weaver.html\n",
      "\n",
      "Name: webbioc\n",
      "Description:\n",
      "An integrated web interface for doing microarray analysis using several of the Bioconductor packages. It is intended to be deployed as a centralized bioinformatics resource for use by many users. (Currently only Affymetrix oligonucleotide analysis is supported.)\n",
      "Version: 1.70.0\n",
      "Publication date: None\n",
      "Mantainer: Colin A. Smith <colin@colinsmith.org>\n",
      "Authors:\n",
      "Author: Colin A. Smith <colin at colinsmith.org>\n",
      "Dependencies:\n",
      "  - R (>= 1.8.0) type: depends\n",
      "  - Biobase, type: depends\n",
      "  - affy, type: depends\n",
      "  - multtest, type: depends\n",
      "  - annaffy, type: depends\n",
      "  - vsn, type: depends\n",
      "  - gcrma, type: depends\n",
      "  - qvalue, type: depends\n",
      "  - multtest, type: imports\n",
      "  - qvalue, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - BiocManager, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/webbioc.html\n",
      "\n",
      "Name: wpm\n",
      "Description:\n",
      "The Well-Plate Maker (WPM) is a shiny application deployed as an R package. Functions for a command-line/script use are also available. The WPM allows users to generate well plate maps to carry out their experiments while improving the handling of batch effects. In particular, it helps controlling the \"plate effect\" thanks to its ability to randomize samples over multiple well plates. The algorithm for placing the samples is inspired by the backtracking algorithm: the samples are placed at random while respecting specific spatial constraints.\n",
      "Version: 1.8.0\n",
      "Publication date: None\n",
      "Mantainer: Helene Borges <borges.helene.sophie@gmail.com>\n",
      "Authors:\n",
      "Author: Helene Borges [aut, cre], Thomas Burger [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.1.0) type: depends\n",
      "  - utils, type: imports\n",
      "  - methods, type: imports\n",
      "  - cli, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - config, type: imports\n",
      "  - golem, type: imports\n",
      "  - shiny, type: imports\n",
      "  - DT, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - rlang, type: imports\n",
      "  - stringr, type: imports\n",
      "  - shinydashboard, type: imports\n",
      "  - shinyWidgets, type: imports\n",
      "  - shinycustomloader, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - logging, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/wpm.html\n",
      "\n",
      "Name: yamss\n",
      "Description:\n",
      "Tools to analyze and visualize high-throughput metabolomics data aquired using chromatography-mass spectrometry. These tools preprocess data in a way that enables reliable and powerful differential analysis.\n",
      "Version: 1.24.0\n",
      "Publication date: None\n",
      "Mantainer: Leslie Myint <leslie.myint@gmail.com>\n",
      "Authors:\n",
      "Author: Leslie Myint [cre, aut], Kasper Daniel Hansen [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - BiocGenerics, type: depends\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - IRanges, type: imports\n",
      "  - stats, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - EBImage, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - mzR, type: imports\n",
      "  - data.table, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - limma, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/yamss.html\n",
      "\n",
      "Name: AlphaBeta\n",
      "Description:\n",
      "AlphaBeta is a computational method for estimating epimutation rates and spectra from high-throughput DNA methylation data in plants. The method has been specifically designed to: 1. analyze 'germline' epimutations in the context of multi-generational mutation accumulation lines (MA-lines). 2. analyze 'somatic' epimutations in the context of plant development and aging.\n",
      "Version: 1.12.0\n",
      "Publication date: None\n",
      "Mantainer: Yadollah Shahryary Dizaji <shahryary@gmail.com>\n",
      "Authors:\n",
      "Author: Yadollah Shahryary Dizaji [cre, aut], Frank Johannes [aut], Rashmi Hazarika [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.6.0) type: depends\n",
      "  - dplyr (>= 0.7) type: imports\n",
      "  - data.table (>= 1.10) type: imports\n",
      "  - stringr (>= 1.3) type: imports\n",
      "  - utils (>= 3.6.0) type: imports\n",
      "  - gtools (>= 3.8.0) type: imports\n",
      "  - optimx (>= 2018-7.10) type: imports\n",
      "  - expm (>= 0.999-4) type: imports\n",
      "  - stats (>= 3.6) type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - igraph (>= 1.2.4) type: imports\n",
      "  - graphics (>= 3.6) type: imports\n",
      "  - ggplot2 (>= 3.2) type: imports\n",
      "  - grDevices (>= 3.6) type: imports\n",
      "  - plotly (>= 4.9) type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/AlphaBeta.html\n",
      "\n",
      "Name: annmap\n",
      "Description:\n",
      "annmap provides annotation mappings for Affymetrix exon arrays and coordinate based queries to support deep sequencing data analysis. Database access is hidden behind the API which provides a set of functions such as genesInRange(), geneToExon(), exonDetails(), etc. Functions to plot gene architecture and BAM file data are also provided. Underlying data are from Ensembl. The annmap database can be downloaded from: https://figshare.manchester.ac.uk/account/articles/16685071\n",
      "Version: 1.40.0\n",
      "Publication date: None\n",
      "Mantainer: Chris Wirth <Christopher.Wirth@cruk.manchester.ac.uk>\n",
      "Authors:\n",
      "Author: Tim Yates <Tim.Yates at cruk.manchester.ac.uk>\n",
      "Dependencies:\n",
      "  - R (>= 2.15.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - DBI, type: imports\n",
      "  - RMySQL (>= 0.6-0) type: imports\n",
      "  - digest, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - grid, type: imports\n",
      "  - lattice, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - genefilter, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/annmap.html\n",
      "\n",
      "Name: arrayMvout\n",
      "Description:\n",
      "This package supports the application of diverse quality metrics to AffyBatch instances, summarizing these metrics via PCA, and then performing parametric outlier detection on the PCs to identify aberrant arrays with a fixed Type I error rate\n",
      "Version: 1.56.0\n",
      "Publication date: None\n",
      "Mantainer: V. Carey <stvjc@channing.harvard.edu>\n",
      "Authors:\n",
      "Author: Z. Gao, A. Asare, R. Wang, V. Carey\n",
      "Dependencies:\n",
      "  - R (>= 2.6.0) type: depends\n",
      "  - tools, type: depends\n",
      "  - methods, type: depends\n",
      "  - utils, type: depends\n",
      "  - parody, type: depends\n",
      "  - Biobase, type: depends\n",
      "  - affy, type: depends\n",
      "  - mdqc, type: imports\n",
      "  - affyContam, type: imports\n",
      "  - lumi, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/arrayMvout.html\n",
      "\n",
      "Name: cbpManager\n",
      "Description:\n",
      "This R package provides an R Shiny application that enables the user to generate, manage, and edit data and metadata files suitable for the import in cBioPortal for Cancer Genomics. Create cancer studies and edit its metadata. Upload mutation data of a patient that will be concatenated to the data_mutation_extended.txt file of the study. Create and edit clinical patient data, sample data, and timeline data. Create custom timeline tracks for patients.\n",
      "Version: 1.6.0\n",
      "Publication date: None\n",
      "Mantainer: Arsenij Ustjanzew <arsenij.ustjanzew@gmail.com>\n",
      "Authors:\n",
      "Author: Arsenij Ustjanzew [aut, cre, cph] , Federico Marini [aut]\n",
      "Dependencies:\n",
      "  - shiny, type: depends\n",
      "  - shinydashboard, type: depends\n",
      "  - utils, type: imports\n",
      "  - DT, type: imports\n",
      "  - htmltools, type: imports\n",
      "  - vroom, type: imports\n",
      "  - plyr, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - jsonlite, type: imports\n",
      "  - rapportools, type: imports\n",
      "  - basilisk, type: imports\n",
      "  - reticulate, type: imports\n",
      "  - shinyBS, type: imports\n",
      "  - shinycssloaders, type: imports\n",
      "  - rintrojs, type: imports\n",
      "  - markdown, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "AGPL-3 + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/cbpManager.html\n",
      "\n",
      "Name: cellTree\n",
      "Description:\n",
      "This packages computes a Latent Dirichlet Allocation (LDA) model of single-cell RNA-seq data and builds a compact tree modelling the relationship between individual cells over time or space.\n",
      "Version: 1.27.0\n",
      "Publication date: None\n",
      "Mantainer: David duVerle <dave@cb.k.u-tokyo.ac.jp>\n",
      "Authors:\n",
      "Author: David duVerle [aut, cre], Koji Tsuda [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.3) type: depends\n",
      "  - topGO, type: depends\n",
      "  - topicmodels, type: imports\n",
      "  - slam, type: imports\n",
      "  - maptpx, type: imports\n",
      "  - igraph, type: imports\n",
      "  - xtable, type: imports\n",
      "  - gplots, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/cellTree.html\n",
      "\n",
      "Name: cisPath\n",
      "Description:\n",
      "cisPath is an R package that uses web browsers to visualize and manage protein-protein interaction networks.\n",
      "Version: 1.38.0\n",
      "Publication date: None\n",
      "Mantainer: Likun Wang <wanglk@hsc.pku.edu.cn>\n",
      "Authors:\n",
      "Author: Likun Wang <wanglk at hsc.pku.edu.cn>\n",
      "Dependencies:\n",
      "  - R (>= 2.10.0) type: depends\n",
      "  - methods, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 3)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/cisPath.html\n",
      "\n",
      "Name: clustComp\n",
      "Description:\n",
      "clustComp is a package that implements several techniques for the comparison and visualisation of relationships between different clustering results, either flat versus flat or hierarchical versus flat. These relationships among clusters are displayed using a weighted bi-graph, in which the nodes represent the clusters and the edges connect pairs of nodes with non-empty intersection; the weight of each edge is the number of elements in that intersection and is displayed through the edge thickness. The best layout of the bi-graph is provided by the barycentre algorithm, which minimises the weighted number of crossings. In the case of comparing a hierarchical and a non-hierarchical clustering, the dendrogram is pruned at different heights, selected by exploring the tree by depth-first search, starting at the root. Branches are decided to be split according to the value of a scoring function, that can be based either on the aesthetics of the bi-graph or on the mutual information between the hierarchical and the flat clusterings. A mapping between groups of clusters from each side is constructed with a greedy algorithm, and can be additionally visualised.\n",
      "Version: 1.26.0\n",
      "Publication date: None\n",
      "Mantainer: Aurora Torrente <aurora@ebi.ac.uk>\n",
      "Authors:\n",
      "Author: Aurora Torrente and Alvis Brazma.\n",
      "Dependencies:\n",
      "  - R (>= 3.3) type: depends\n",
      "  - sm, type: imports\n",
      "  - stats, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/clustComp.html\n",
      "\n",
      "Name: ClusterJudge\n",
      "Description:\n",
      "ClusterJudge implements the functions, examples and other software published as an algorithm by Gibbons, FD and Roth FP. The article is called \"Judging the Quality of Gene Expression-Based Clustering Methods Using Gene Annotation\" and it appeared in Genome Research, vol. 12, pp1574-1581 (2002). See package?ClusterJudge for an overview.\n",
      "Version: 1.20.0\n",
      "Publication date: None\n",
      "Mantainer: Adrian Pasculescu <a.pasculescu@gmail.com>\n",
      "Authors:\n",
      "Author: Adrian Pasculescu\n",
      "Dependencies:\n",
      "  - R (>= 3.6) type: depends\n",
      "  - stats, type: depends\n",
      "  - utils, type: depends\n",
      "  - graphics, type: depends\n",
      "  - infotheo, type: depends\n",
      "  - lattice, type: depends\n",
      "  - latticeExtra, type: depends\n",
      "  - httr, type: depends\n",
      "  - jsonlite, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ClusterJudge.html\n",
      "\n",
      "Name: CNVgears\n",
      "Description:\n",
      "This package contains a set of functions to perform several type of processing and analysis on CNVs calling pipelines/algorithms results in an integrated manner and regardless of the raw data type (SNPs array or NGS). It provides functions to combine multiple CNV calling results into a single object, filter them, compute CNVRs (CNV Regions) and inheritance patterns, detect genic load, and more. The package is best suited for studies in human family-based cohorts.\n",
      "Version: 1.6.0\n",
      "Publication date: None\n",
      "Mantainer: Simone Montalbano <simone.montalbano@protonmail.com>\n",
      "Authors:\n",
      "Author: Simone Montalbano [cre, aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - data.table, type: depends\n",
      "  - ggplot2, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/CNVgears.html\n",
      "\n",
      "Name: consensusDE\n",
      "Description:\n",
      "This package allows users to perform DE analysis using multiple algorithms. It seeks consensus from multiple methods. Currently it supports \"Voom\", \"EdgeR\" and \"DESeq\". It uses RUV-seq (optional) to remove unwanted sources of variation.\n",
      "Version: 1.16.0\n",
      "Publication date: None\n",
      "Mantainer: Ashley J. Waardenberg <a.waardenberg@gmail.com>\n",
      "Authors:\n",
      "Author: Ashley J. Waardenberg [aut, cre], Martha M. Cooper [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 3.5) type: depends\n",
      "  - BiocGenerics, type: depends\n",
      "  - airway, type: imports\n",
      "  - AnnotationDbi, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - data.table, type: imports\n",
      "  - dendextend, type: imports\n",
      "  - DESeq2, type: imports\n",
      "  - EDASeq, type: imports\n",
      "  - ensembldb, type: imports\n",
      "  - edgeR, type: imports\n",
      "  - EnsDb.Hsapiens.v86, type: imports\n",
      "  - GenomicAlignments, type: imports\n",
      "  - GenomicFeatures, type: imports\n",
      "  - limma, type: imports\n",
      "  - org.Hs.eg.db, type: imports\n",
      "  - pcaMethods, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - RUVSeq, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - stats, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - TxDb.Dmelanogaster.UCSC.dm3.ensGene, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/consensusDE.html\n",
      "\n",
      "Name: copa\n",
      "Description:\n",
      "COPA is a method to find genes that undergo recurrent fusion in a given cancer type by finding pairs of genes that have mutually exclusive outlier profiles.\n",
      "Version: 1.66.0\n",
      "Publication date: None\n",
      "Mantainer: James W. MacDonald <jmacdon@u.washington.edu>\n",
      "Authors:\n",
      "Author: James W. MacDonald\n",
      "Dependencies:\n",
      "  - Biobase, type: depends\n",
      "  - methods, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/copa.html\n",
      "\n",
      "Name: cosmiq\n",
      "Description:\n",
      "cosmiq is a tool for the preprocessing of liquid- or gas - chromatography mass spectrometry (LCMS/GCMS) data with a focus on metabolomics or lipidomics applications. To improve the detection of low abundant signals, cosmiq generates master maps of the mZ/RT space from all acquired runs before a peak detection algorithm is applied. The result is a more robust identification and quantification of low-intensity MS signals compared to conventional approaches where peak picking is performed in each LCMS/GCMS file separately. The cosmiq package builds on the xcmsSet object structure and can be therefore integrated well with the package xcms as an alternative preprocessing step.\n",
      "Version: 1.32.0\n",
      "Publication date: None\n",
      "Mantainer: David Fischer <dajofischer@googlemail.com>\n",
      "Authors:\n",
      "Author: David Fischer [aut, cre], Christian Panse [aut] , Endre Laczko [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 3.6) type: depends\n",
      "  - Rcpp, type: depends\n",
      "  - pracma, type: imports\n",
      "  - xcms, type: imports\n",
      "  - MassSpecWavelet, type: imports\n",
      "  - faahKO, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/cosmiq.html\n",
      "\n",
      "Name: cpvSNP\n",
      "Description:\n",
      "Gene set analysis methods exist to combine SNP-level association p-values into gene sets, calculating a single association p-value for each gene set. This package implements two such methods that require only the calculated SNP p-values, the gene set(s) of interest, and a correlation matrix (if desired). One method (GLOSSI) requires independent SNPs and the other (VEGAS) can take into account correlation (LD) among the SNPs. Built-in plotting functions are available to help users visualize results.\n",
      "Version: 1.30.0\n",
      "Publication date: None\n",
      "Mantainer: Caitlin McHugh <mchughc@uw.edu>\n",
      "Authors:\n",
      "Author: Caitlin McHugh, Jessica Larson, and Jason Hackney\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - GenomicFeatures, type: depends\n",
      "  - GSEABase, type: depends\n",
      "  - methods, type: imports\n",
      "  - corpcor, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - plyr, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/cpvSNP.html\n",
      "\n",
      "Name: DExMA\n",
      "Description:\n",
      "performing all the steps of gene expression meta-analysis considering the possible existence of missing genes. It provides the necessary functions to be able to perform the different methods of gene expression meta-analysis. In addition, it contains functions to apply quality controls, download GEO datasets and show graphical representations of the results.\n",
      "Version: 1.6.0\n",
      "Publication date: None\n",
      "Mantainer: Juan Antonio Villatoro-GarcÃ­a <juanantoniovillatorogarcia@gmail.com>\n",
      "Authors:\n",
      "Author: Juan Antonio Villatoro-GarcÃ­a [aut, cre], Pedro Carmona-SÃ¡ez [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - DExMAdata, type: depends\n",
      "  - Biobase, type: imports\n",
      "  - GEOquery, type: imports\n",
      "  - impute, type: imports\n",
      "  - limma, type: imports\n",
      "  - pheatmap, type: imports\n",
      "  - plyr, type: imports\n",
      "  - scales, type: imports\n",
      "  - snpStats, type: imports\n",
      "  - sva, type: imports\n",
      "  - swamp, type: imports\n",
      "  - stats, type: imports\n",
      "  - methods, type: imports\n",
      "  - utils, type: imports\n",
      "  - bnstruct, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - grDevices, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/DExMA.html\n",
      "\n",
      "Name: enhancerHomologSearch\n",
      "Description:\n",
      "Get ENCODE data of enhancer region via H3K4me1 peaks and search homolog regions for given sequences. The candidates of enhancer homolog regions can be filtered by distance to target TSS. The top candidates from human and mouse will be aligned to each other and then exported as multiple alignments with given enhancer.\n",
      "Version: 1.4.2\n",
      "Publication date: None\n",
      "Mantainer: Jianhong Ou <jianhong.ou@duke.edu>\n",
      "Authors:\n",
      "Author: Jianhong Ou [aut, cre] , Valentina Cigliola [dtc], Kenneth Poss [fnd]\n",
      "Dependencies:\n",
      "  - R (>= 4.1.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - BiocGenerics, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - BSgenome, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - BiocFileCache, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - httr, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - jsonlite, type: imports\n",
      "  - motifmatchr, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - Rcpp, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/enhancerHomologSearch.html\n",
      "\n",
      "Name: ExperimentSubset\n",
      "Description:\n",
      "Experiment objects such as the SummarizedExperiment or SingleCellExperiment are data containers for one or more matrix-like assays along with the associated row and column data. Often only a subset of the original data is needed for down-stream analysis. For example, filtering out poor quality samples will require excluding some columns before analysis. The ExperimentSubset object is a container to efficiently manage different subsets of the same data without having to make separate objects for each new subset.\n",
      "Version: 1.8.0\n",
      "Publication date: None\n",
      "Mantainer: Irzam Sarfraz <irzam9095@gmail.com>\n",
      "Authors:\n",
      "Author: Irzam Sarfraz [aut, cre] , Muhammad Asif [aut, ths] , Joshua D. Campbell [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.0.0) type: depends\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - SingleCellExperiment, type: depends\n",
      "  - SpatialExperiment, type: depends\n",
      "  - TreeSummarizedExperiment, type: depends\n",
      "  - methods, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ExperimentSubset.html\n",
      "\n",
      "Name: fCI\n",
      "Description:\n",
      "(f-divergence Cutoff Index), is to find DEGs in the transcriptomic & proteomic data, and identify DEGs by computing the difference between the distribution of fold-changes for the control-control and remaining (non-differential) case-control gene expression ratio data. fCI provides several advantages compared to existing methods.\n",
      "Version: 1.28.0\n",
      "Publication date: None\n",
      "Mantainer: Shaojun Tang <tangshao2008@gmail.com>\n",
      "Authors:\n",
      "Author: Shaojun Tang\n",
      "Dependencies:\n",
      "  - R (>= 3.1) type: depends\n",
      "  - FNN, type: depends\n",
      "  - psych, type: depends\n",
      "  - gtools, type: depends\n",
      "  - zoo, type: depends\n",
      "  - rgl, type: depends\n",
      "  - grid, type: depends\n",
      "  - VennDiagram, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/fCI.html\n",
      "\n",
      "Name: flowPlots\n",
      "Description:\n",
      "Graphical displays with embedded statistical tests for gated ICS flow cytometry data, and a data class which stores \"stacked\" data and has methods for computing summary measures on stacked data, such as marginal and polyfunctional degree data.\n",
      "Version: 1.46.0\n",
      "Publication date: None\n",
      "Mantainer: N. Hawkins <hawkins@fhcrc.org>\n",
      "Authors:\n",
      "Author: N. Hawkins, S. Self\n",
      "Dependencies:\n",
      "  - R (>= 2.13.0) type: depends\n",
      "  - methods, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/flowPlots.html\n",
      "\n",
      "Name: FoldGO\n",
      "Description:\n",
      "FoldGO is a package designed to annotate gene sets derived from expression experiments and identify fold-change-specific GO terms.\n",
      "Version: 1.16.0\n",
      "Publication date: None\n",
      "Mantainer: Daniil Wiebe <daniil.wiebe@gmail.com>\n",
      "Authors:\n",
      "Author: Daniil Wiebe <daniil.wiebe at gmail.com> [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - topGO, type: imports\n",
      "  - ggplot2 (>= 2.2.1) type: imports\n",
      "  - tidyr (>= 0.8.0) type: imports\n",
      "  - stats, type: imports\n",
      "  - methods, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/FoldGO.html\n",
      "\n",
      "Name: genArise\n",
      "Description:\n",
      "genArise is an easy to use tool for dual color microarray data. Its GUI-Tk based environment let any non-experienced user performs a basic, but not simple, data analysis just following a wizard. In addition it provides some tools for the developer.\n",
      "Version: 1.74.0\n",
      "Publication date: None\n",
      "Mantainer: IFC Development Team <info-genarise@ifc.unam.mx>\n",
      "Authors:\n",
      "Author: Ana Patricia Gomez Mayen <pgomez at ifc.unam.mx>,\\\\ Gustavo Corral Guille <gcorral at ifc.unam.mx>, \\\\ Lina Riego Ruiz <lriego at ifc.unam.mx>,\\\\ Gerardo Coello Coutino <gcoello at ifc.unam.mx>\n",
      "Dependencies:\n",
      "  - R (>= 1.7.1) type: depends\n",
      "  - locfit, type: depends\n",
      "  - tkrplot, type: depends\n",
      "  - methods, type: depends\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - tcltk, type: imports\n",
      "  - utils, type: imports\n",
      "  - xtable, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/genArise.html\n",
      "\n",
      "Name: geneRecommender\n",
      "Description:\n",
      "This package contains a targeted clustering algorithm for the analysis of microarray data. The algorithm can aid in the discovery of new genes with similar functions to a given list of genes already known to have closely related functions.\n",
      "Version: 1.70.0\n",
      "Publication date: None\n",
      "Mantainer: Greg Hather <ghather@stat.berkeley.edu>\n",
      "Authors:\n",
      "Author: Gregory J. Hather <ghather at stat.berkeley.edu>, with contributions from Art B. Owen <art at stat.stanford.edu> and Terence P. Speed <terry at stat.berkeley.edu>\n",
      "Dependencies:\n",
      "  - R (>= 1.8.0) type: depends\n",
      "  - Biobase, type: depends\n",
      "  - methods, type: depends\n",
      "  - Biobase, type: imports\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/geneRecommender.html\n",
      "\n",
      "Name: geneRxCluster\n",
      "Description:\n",
      "Detect Differential Clustering of Genomic Sites such as gene therapy integrations. The package provides some functions for exploring genomic insertion sites originating from two different sources. Possibly, the two sources are two different gene therapy vectors. Vectors are preferred that target sensitive regions less frequently, motivating the search for localized clusters of insertions and comparison of the clusters formed by integration of different vectors. Scan statistics allow the discovery of spatial differences in clustering and calculation of False Discovery Rates (FDRs) providing statistical methods for comparing retroviral vectors. A scan statistic for comparing two vectors using multiple window widths to detect clustering differentials and compute FDRs is implemented here.\n",
      "Version: 1.34.0\n",
      "Publication date: None\n",
      "Mantainer: Charles Berry <ccberry@ucsd.edu>\n",
      "Authors:\n",
      "Author: Charles Berry\n",
      "Dependencies:\n",
      "  - GenomicRanges, type: depends\n",
      "  - IRanges, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/geneRxCluster.html\n",
      "\n",
      "Name: geneXtendeR\n",
      "Description:\n",
      "geneXtendeR optimizes the functional annotation of ChIP-seq peaks by exploring relative differences in annotating ChIP-seq peak sets to variable-length gene bodies. In contrast to prior techniques, geneXtendeR considers peak annotations beyond just the closest gene, allowing users to see peak summary statistics for the first-closest gene, second-closest gene, ..., n-closest gene whilst ranking the output according to biologically relevant events and iteratively comparing the fidelity of peak-to-gene overlap across a user-defined range of upstream and downstream extensions on the original boundaries of each gene's coordinates. Since different ChIP-seq peak callers produce different differentially enriched peaks with a large variance in peak length distribution and total peak count, annotating peak lists with their nearest genes can often be a noisy process. As such, the goal of geneXtendeR is to robustly link differentially enriched peaks with their respective genes, thereby aiding experimental follow-up and validation in designing primers for a set of prospective gene candidates during qPCR.\n",
      "Version: 1.24.0\n",
      "Publication date: None\n",
      "Mantainer: Bohdan Khomtchouk <khomtchoukmed@gmail.com>\n",
      "Authors:\n",
      "Author: Bohdan Khomtchouk [aut, cre], William Koehler [aut]\n",
      "Dependencies:\n",
      "  - rtracklayer, type: depends\n",
      "  - GO.db, type: depends\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - data.table, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - graphics, type: imports\n",
      "  - networkD3, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - SnowballC, type: imports\n",
      "  - tm, type: imports\n",
      "  - utils, type: imports\n",
      "  - wordcloud, type: imports\n",
      "  - AnnotationDbi, type: imports\n",
      "  - BiocStyle, type: imports\n",
      "  - org.Rn.eg.db, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 3)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/geneXtendeR.html\n",
      "\n",
      "Name: globalSeq\n",
      "Description:\n",
      "The method may be conceptualised as a test of overall significance in regression analysis, where the response variable is overdispersed and the number of explanatory variables exceeds the sample size. Useful for testing for association between RNA-Seq and high-dimensional data.\n",
      "Version: 1.26.0\n",
      "Publication date: None\n",
      "Mantainer: Armin Rauschenberger <armin.rauschenberger@uni.lu>\n",
      "Authors:\n",
      "Author: Armin Rauschenberger [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 3.0.0) type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/globalSeq.html\n",
      "\n",
      "Name: gpuMagic\n",
      "Description:\n",
      "The package aims to help users write openCL code with little or no effort. It is able to compile an user-defined R function and run it on a device such as a CPU or a GPU. The user can also write and run their openCL code directly by calling .kernel function.\n",
      "Version: 1.14.0\n",
      "Publication date: None\n",
      "Mantainer: Jiefei Wang <szwjf08@gmail.com>\n",
      "Authors:\n",
      "Author: Jiefei Wang [aut, cre], Martin Morgan [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.6.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - utils, type: depends\n",
      "  - Deriv, type: imports\n",
      "  - DescTools, type: imports\n",
      "  - digest, type: imports\n",
      "  - pryr, type: imports\n",
      "  - stringr, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/gpuMagic.html\n",
      "\n",
      "Name: GRENITS\n",
      "Description:\n",
      "The package offers four network inference statistical models using Dynamic Bayesian Networks and Gibbs Variable Selection: a linear interaction model, two linear interaction models with added experimental noise (Gaussian and Student distributed) for the case where replicates are available and a non-linear interaction model.\n",
      "Version: 1.50.2\n",
      "Publication date: None\n",
      "Mantainer: Edward Morrissey <edward.morrissey@gmail.com>\n",
      "Authors:\n",
      "Author: Edward Morrissey\n",
      "Dependencies:\n",
      "  - R (>= 2.12.0) type: depends\n",
      "  - Rcpp (>= 0.8.6) type: depends\n",
      "  - RcppArmadillo (>= 0.2.8) type: depends\n",
      "  - ggplot2 (>= 0.9.0) type: depends\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/GRENITS.html\n",
      "\n",
      "Name: GSRI\n",
      "Description:\n",
      "The GSRI package estimates the number of differentially expressed genes in gene sets, utilizing the concept of the Gene Set Regulation Index (GSRI).\n",
      "Version: 2.46.0\n",
      "Publication date: None\n",
      "Mantainer: Julian Gehring <jg-bioc@gmx.com>\n",
      "Authors:\n",
      "Author: Julian Gehring, Kilian Bartholome, Clemens Kreutz, Jens Timmer\n",
      "Dependencies:\n",
      "  - R (>= 2.14.2) type: depends\n",
      "  - fdrtool, type: depends\n",
      "  - methods, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - genefilter, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - GSEABase, type: imports\n",
      "  - les, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/GSRI.html\n",
      "\n",
      "Name: hermes\n",
      "Description:\n",
      "Provides classes and functions for quality control, filtering, normalization and differential expression analysis of pre-processed RNA-seq data. Data can be imported from `SummarizedExperiment` as well as `matrix` objects and can be annotated from BioMart. Filtering for genes without too low expression or containing required annotations, as well as filtering for samples with sufficient correlation to other samples or total number of reads is supported. The standard normalization methods including `cpm`, `rpkm` and `tpm` can be used, and `DESeq2` as well as `voom` differential expression analyses are available.\n",
      "Version: 1.2.0\n",
      "Publication date: None\n",
      "Mantainer: Daniel SabanÃ©s BovÃ© <daniel.sabanes_bove@roche.com>\n",
      "Authors:\n",
      "Author: Daniel SabanÃ©s BovÃ© [aut, cre], Namrata Bhatia [aut], Stefanie Bienert [aut], Benoit Falquet [aut], Haocheng Li [aut], Jeff Luong [aut], Lyndsee Midori Zhang [aut], Simona Rossomanno [aut], Tim Treis [aut], Mark Yan [aut], Naomi Chang [aut], Chendi Liao [aut], Carolyn Zhang [aut], Joseph N. Paulson [aut]\n",
      "Dependencies:\n",
      "  - ggfortify, type: depends\n",
      "  - R (>= 4.1) type: depends\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - assertthat, type: imports\n",
      "  - biomaRt, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - checkmate (>= 2.1) type: imports\n",
      "  - circlize, type: imports\n",
      "  - ComplexHeatmap, type: imports\n",
      "  - DESeq2, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - edgeR, type: imports\n",
      "  - EnvStats, type: imports\n",
      "  - forcats, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - ggrepel (>= 0.9) type: imports\n",
      "  - IRanges, type: imports\n",
      "  - lifecycle, type: imports\n",
      "  - limma, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - methods, type: imports\n",
      "  - MultiAssayExperiment, type: imports\n",
      "  - purrr, type: imports\n",
      "  - R6, type: imports\n",
      "  - Rdpack, type: imports\n",
      "  - rlang, type: imports\n",
      "  - stats, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Apache License 2.0 | file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/hermes.html\n",
      "\n",
      "Name: immunoClust\n",
      "Description:\n",
      "immunoClust is a model based clustering approach for Flow Cytometry samples. The cell-events of single Flow Cytometry samples are modelled by a mixture of multinominal normal- or t-distributions. The cell-event clusters of several samples are modelled by a mixture of multinominal normal-distributions aiming stable co-clusters across these samples.\n",
      "Version: 1.30.0\n",
      "Publication date: None\n",
      "Mantainer: Till Soerensen <till.soerensen@bioretis.com>\n",
      "Authors:\n",
      "Author: Till Soerensen [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 3.6) type: depends\n",
      "  - flowCore, type: depends\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grid, type: imports\n",
      "  - lattice, type: imports\n",
      "  - grDevices, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/immunoClust.html\n",
      "\n",
      "Name: infinityFlow\n",
      "Description:\n",
      "Pipeline to analyze and merge data files produced by BioLegend's LEGENDScreen or BD Human Cell Surface Marker Screening Panel (BD Lyoplates).\n",
      "Version: 1.8.0\n",
      "Publication date: None\n",
      "Mantainer: Etienne Becht <etienne.becht@protonmail.com>\n",
      "Authors:\n",
      "Author: Etienne Becht [cre, aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.0.0) type: depends\n",
      "  - flowCore, type: depends\n",
      "  - stats, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - utils, type: imports\n",
      "  - graphics, type: imports\n",
      "  - pbapply, type: imports\n",
      "  - matlab, type: imports\n",
      "  - png, type: imports\n",
      "  - raster, type: imports\n",
      "  - grid, type: imports\n",
      "  - uwot, type: imports\n",
      "  - gtools, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - generics, type: imports\n",
      "  - parallel, type: imports\n",
      "  - methods, type: imports\n",
      "  - xgboost, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/infinityFlow.html\n",
      "\n",
      "Name: InPAS\n",
      "Description:\n",
      "Alternative polyadenylation (APA) is one of the important post- transcriptional regulation mechanisms which occurs in most human genes. InPAS facilitates the discovery of novel APA sites and the differential usage of APA sites from RNA-Seq data. It leverages cleanUpdTSeq to fine tune identified APA sites by removing false sites.\n",
      "Version: 2.6.0\n",
      "Publication date: None\n",
      "Mantainer: Jianhong Ou <jianhong.ou@duke.edu>\n",
      "Authors:\n",
      "Author: Jianhong Ou [aut, cre], Haibo Liu [aut], Lihua Julie Zhu [aut], Sungmi M. Park [aut], Michael R. Green [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - AnnotationDbi, type: imports\n",
      "  - batchtools, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - BSgenome, type: imports\n",
      "  - cleanUpdTSeq, type: imports\n",
      "  - depmixS4, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - flock, type: imports\n",
      "  - future, type: imports\n",
      "  - future.apply, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - GenomicFeatures, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - limma, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - methods, type: imports\n",
      "  - parallelly, type: imports\n",
      "  - plyranges, type: imports\n",
      "  - preprocessCore, type: imports\n",
      "  - readr, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - RSQLite, type: imports\n",
      "  - stats, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/InPAS.html\n",
      "\n",
      "Name: IntramiRExploreR\n",
      "Description:\n",
      "Intra-miR-ExploreR, an integrative miRNA target prediction bioinformatics tool, identifies targets combining expression and biophysical interactions of a given microRNA (miR). Using the tool, we have identified targets for 92 intragenic miRs in D. melanogaster, using available microarray expression data, from Affymetrix 1 and Affymetrix2 microarray array platforms, providing a global perspective of intragenic miR targets in Drosophila. Predicted targets are grouped according to biological functions using the DAVID Gene Ontology tool and are ranked based on a biologically relevant scoring system, enabling the user to identify functionally relevant targets for a given miR.\n",
      "Version: 1.20.0\n",
      "Publication date: None\n",
      "Mantainer: Surajit Bhattacharya <sbhattach2@childrensnational.org>\n",
      "Authors:\n",
      "Author: Surajit Bhattacharya and Daniel Cox\n",
      "Dependencies:\n",
      "  - R (>= 3.4) type: depends\n",
      "  - igraph (>= 1.0.1) type: imports\n",
      "  - FGNet, type: imports\n",
      "  - knitr (>= 1.12.3) type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/IntramiRExploreR.html\n",
      "\n",
      "Name: KinSwingR\n",
      "Description:\n",
      "KinSwingR integrates phosphosite data derived from mass-spectrometry data and kinase-substrate predictions to predict kinase activity. Several functions allow the user to build PWM models of kinase-subtrates, statistically infer PWM:substrate matches, and integrate these data to infer kinase activity.\n",
      "Version: 1.16.0\n",
      "Publication date: None\n",
      "Mantainer: Ashley J. Waardenberg <a.waardenberg@gmail.com>\n",
      "Authors:\n",
      "Author: Ashley J. Waardenberg [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 3.5) type: depends\n",
      "  - data.table, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - sqldf, type: imports\n",
      "  - stats, type: imports\n",
      "  - grid, type: imports\n",
      "  - grDevices, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/KinSwingR.html\n",
      "\n",
      "Name: macat\n",
      "Description:\n",
      "This library contains functions to investigate links between differential gene expression and the chromosomal localization of the genes. MACAT is motivated by the common observation of phenomena involving large chromosomal regions in tumor cells. MACAT is the implementation of a statistical approach for identifying significantly differentially expressed chromosome regions. The functions have been tested on a publicly available data set about acute lymphoblastic leukemia (Yeoh et al.Cancer Cell 2002), which is provided in the library 'stjudem'.\n",
      "Version: 1.72.0\n",
      "Publication date: None\n",
      "Mantainer: Joern Toedling <jtoedling@yahoo.de>\n",
      "Authors:\n",
      "Author: Benjamin Georgi, Matthias Heinig, Stefan Roepcke, Sebastian Schmeier, Joern Toedling\n",
      "Dependencies:\n",
      "  - Biobase, type: depends\n",
      "  - annotate, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/macat.html\n",
      "\n",
      "Name: MassArray\n",
      "Description:\n",
      "This package is designed for the import, quality control, analysis, and visualization of methylation data generated using Sequenom's MassArray platform. The tools herein contain a highly detailed amplicon prediction for optimal assay design. Also included are quality control measures of data, such as primer dimer and bisulfite conversion efficiency estimation. Methylation data are calculated using the same algorithms contained in the EpiTyper software package. Additionally, automatic SNP-detection can be used to flag potentially confounded data from specific CG sites. Visualization includes barplots of methylation data as well as UCSC Genome Browser-compatible BED tracks. Multiple assays can be positionally combined for integrated analysis.\n",
      "Version: 1.50.0\n",
      "Publication date: None\n",
      "Mantainer: Reid F. Thompson <reid.thompson@gmail.com>\n",
      "Authors:\n",
      "Author: Reid F. Thompson <reid.thompson at gmail.com>, John M. Greally <john.greally at einstein.yu.edu>\n",
      "Dependencies:\n",
      "  - R (>= 2.10.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>=2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/MassArray.html\n",
      "\n",
      "Name: mdp\n",
      "Description:\n",
      "The Molecular Degree of Perturbation webtool quantifies the heterogeneity of samples. It takes a data.frame of omic data that contains at least two classes (control and test) and assigns a score to all samples based on how perturbed they are compared to the controls. It is based on the Molecular Distance to Health (Pankla et al. 2009), and expands on this algorithm by adding the options to calculate the z-score using the modified z-score (using median absolute deviation), change the z-score zeroing threshold, and look at genes that are most perturbed in the test versus control classes.\n",
      "Version: 1.18.0\n",
      "Publication date: None\n",
      "Mantainer: Helder Nakaya <hnakaya@usp.br>\n",
      "Authors:\n",
      "Author: Melissa Lever [aut], Pedro Russo [aut], Helder Nakaya [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - ggplot2, type: imports\n",
      "  - gridExtra, type: imports\n",
      "  - grid, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/mdp.html\n",
      "\n",
      "Name: mdqc\n",
      "Description:\n",
      "MDQC is a multivariate quality assessment method for microarrays based on quality control (QC) reports. The Mahalanobis distance of an array's quality attributes is used to measure the similarity of the quality of that array against the quality of the other arrays. Then, arrays with unusually high distances can be flagged as potentially low-quality.\n",
      "Version: 1.60.0\n",
      "Publication date: None\n",
      "Mantainer: Gabriela Cohen-Freue <gcohen@mrl.ubc.ca>\n",
      "Authors:\n",
      "Author: Justin Harrington\n",
      "Dependencies:\n",
      "  - R (>= 2.2.1) type: depends\n",
      "  - cluster, type: depends\n",
      "  - MASS, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/mdqc.html\n",
      "\n",
      "Name: methInheritSim\n",
      "Description:\n",
      "Simulate a multigeneration methylation case versus control experiment with inheritance relation using a real control dataset.\n",
      "Version: 1.20.0\n",
      "Publication date: None\n",
      "Mantainer: Pascal Belleau <pascal_belleau@hotmail.com>\n",
      "Authors:\n",
      "Author: Pascal Belleau, Astrid DeschÃªnes and Arnaud Droit\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - methylKit, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - parallel, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - msm, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/methInheritSim.html\n",
      "\n",
      "Name: midasHLA\n",
      "Description:\n",
      "MiDAS is a R package for immunogenetics data transformation and statistical analysis. MiDAS accepts input data in the form of HLA alleles and KIR types, and can transform it into biologically meaningful variables, enabling HLA amino acid fine mapping, analyses of HLA evolutionary divergence, KIR gene presence, as well as validated HLA-KIR interactions. Further, it allows comprehensive statistical association analysis workflows with phenotypes of diverse measurement scales. MiDAS closes a gap between the inference of immunogenetic variation and its efficient utilization to make relevant discoveries related to T cell, Natural Killer cell, and disease biology.\n",
      "Version: 1.6.0\n",
      "Publication date: None\n",
      "Mantainer: Maciej MigdaÅ <mcjmigdal@gmail.com>\n",
      "Authors:\n",
      "Author: Christian Hammer [aut], Maciej MigdaÅ [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - MultiAssayExperiment, type: depends\n",
      "  - assertthat (>= 0.2.0) type: imports\n",
      "  - broom (>= 0.5.1) type: imports\n",
      "  - dplyr (>= 0.8.0.1) type: imports\n",
      "  - formattable (>= 0.2.0.1) type: imports\n",
      "  - HardyWeinberg (>= 1.6.3) type: imports\n",
      "  - kableExtra (>= 1.1.0) type: imports\n",
      "  - knitr (>= 1.21) type: imports\n",
      "  - magrittr (>= 1.5) type: imports\n",
      "  - methods, type: imports\n",
      "  - stringi (>= 1.2.4) type: imports\n",
      "  - rlang (>= 0.3.1) type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - stats, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - tibble (>= 2.0.1) type: imports\n",
      "  - utils, type: imports\n",
      "  - qdapTools (>= 1.3.3) type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENCE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/midasHLA.html\n",
      "\n",
      "Name: motifcounter\n",
      "Description:\n",
      "'motifcounter' provides motif matching, motif counting and motif enrichment functionality based on position frequency matrices. The main features of the packages include the utilization of higher-order background models and accounting for self-overlapping motif matches when determining motif enrichment. The background model allows to capture dinucleotide (or higher-order nucleotide) composition adequately which may reduced model biases and misleading results compared to using simple GC background models. When conducting a motif enrichment analysis based on the motif match count, the package relies on a compound Poisson distribution or alternatively a combinatorial model. These distribution account for self-overlapping motif structures as exemplified by repeat-like or palindromic motifs, and allow to determine the p-value and fold-enrichment for a set of observed motif matches.\n",
      "Version: 1.22.0\n",
      "Publication date: None\n",
      "Mantainer: Wolfgang Kopp <wolfgang.kopp@mdc-berlin.de>\n",
      "Authors:\n",
      "Author: Wolfgang Kopp [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 3.0) type: depends\n",
      "  - Biostrings, type: imports\n",
      "  - methods, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/motifcounter.html\n",
      "\n",
      "Name: MSstatsQC\n",
      "Description:\n",
      "MSstatsQC is an R package which provides longitudinal system suitability monitoring and quality control tools for proteomic experiments.\n",
      "Version: 2.16.0\n",
      "Publication date: None\n",
      "Mantainer: Eralp Dogu <eralp.dogu@gmail.com>\n",
      "Authors:\n",
      "Author: Eralp Dogu [aut, cre], Sara Taheri [aut], Olga Vitek [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - dplyr, type: imports\n",
      "  - plotly, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - ggExtra, type: imports\n",
      "  - stats, type: imports\n",
      "  - grid, type: imports\n",
      "  - MSnbase, type: imports\n",
      "  - qcmetrics, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic License 2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/MSstatsQC.html\n",
      "\n",
      "Name: MultiMed\n",
      "Description:\n",
      "Implements methods for testing multiple mediators\n",
      "Version: 2.20.0\n",
      "Publication date: None\n",
      "Mantainer: Simina M. Boca <smb310@georgetown.edu>\n",
      "Authors:\n",
      "Author: Simina M. Boca, Ruth Heller, Joshua N. Sampson\n",
      "Dependencies:\n",
      "  - R (>= 3.1.0) type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2) + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/MultiMed.html\n",
      "\n",
      "Name: npGSEA\n",
      "Description:\n",
      "Current gene set enrichment methods rely upon permutations for inference. These approaches are computationally expensive and have minimum achievable p-values based on the number of permutations, not on the actual observed statistics. We have derived three parametric approximations to the permutation distributions of two gene set enrichment test statistics. We are able to reduce the computational burden and granularity issues of permutation testing with our method, which is implemented in this package. npGSEA calculates gene set enrichment statistics and p-values without the computational cost of permutations. It is applicable in settings where one or many gene sets are of interest. There are also built-in plotting functions to help users visualize results.\n",
      "Version: 1.34.0\n",
      "Publication date: None\n",
      "Mantainer: Jessica Larson <larson.jess@gmail.com>\n",
      "Authors:\n",
      "Author: Jessica Larson and Art Owen\n",
      "Dependencies:\n",
      "  - GSEABase, type: depends\n",
      "  - Biobase, type: imports\n",
      "  - methods, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/npGSEA.html\n",
      "\n",
      "Name: omicplotR\n",
      "Description:\n",
      "A Shiny app for visual exploration of omic datasets as compositions, and differential abundance analysis using ALDEx2. Useful for exploring RNA-seq, meta-RNA-seq, 16s rRNA gene sequencing with visualizations such as principal component analysis biplots (coloured using metadata for visualizing each variable), dendrograms and stacked bar plots, and effect plots (ALDEx2). Input is a table of counts and metadata file (if metadata exists), with options to filter data by count or by metadata to remove low counts, or to visualize select samples according to selected metadata.\n",
      "Version: 1.18.0\n",
      "Publication date: None\n",
      "Mantainer: Daniel Giguere <dgiguer@uwo.ca>\n",
      "Authors:\n",
      "Author: Daniel Giguere [aut, cre], Jean Macklaim [aut], Greg Gloor [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.6) type: depends\n",
      "  - ALDEx2, type: depends\n",
      "  - compositions, type: imports\n",
      "  - DT, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - knitr, type: imports\n",
      "  - jsonlite, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - rmarkdown, type: imports\n",
      "  - shiny, type: imports\n",
      "  - stats, type: imports\n",
      "  - vegan, type: imports\n",
      "  - zCompositions, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/omicplotR.html\n",
      "\n",
      "Name: OncoScore\n",
      "Description:\n",
      "OncoScore is a tool to measure the association of genes to cancer based on citation frequencies in biomedical literature. The score is evaluated from PubMed literature by dynamically updatable web queries.\n",
      "Version: 1.26.0\n",
      "Publication date: None\n",
      "Mantainer: Luca De Sano <luca.desano@gmail.com>\n",
      "Authors:\n",
      "Author: Luca De Sano [aut] , Carlo Gambacorti Passerini [ctb], Rocco Piazza [ctb], Daniele Ramazzotti [cre, aut] , Roberta Spinelli [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 4.1.0) type: depends\n",
      "  - biomaRt, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - utils, type: imports\n",
      "  - methods, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/OncoScore.html\n",
      "\n",
      "Name: pepXMLTab\n",
      "Description:\n",
      "Parsing pepXML files based one XML package. The package tries to handle pepXML files generated from different softwares. The output will be a peptide-spectrum-matching tabular file. The package also provide function to filter the PSMs based on FDR.\n",
      "Version: 1.32.0\n",
      "Publication date: None\n",
      "Mantainer: Xiaojing Wang <xiaojing.wang@vanderbilt.edu>\n",
      "Authors:\n",
      "Author: Xiaojing Wang\n",
      "Dependencies:\n",
      "  - R (>= 3.0.1) type: depends\n",
      "  - XML (>= 3.98-1.1) type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/pepXMLTab.html\n",
      "\n",
      "Name: pipeComp\n",
      "Description:\n",
      "A simple framework to facilitate the comparison of pipelines involving various steps and parameters. The `pipelineDefinition` class represents pipelines as, minimally, a set of functions consecutively executed on the output of the previous one, and optionally accompanied by step-wise evaluation and aggregation functions. Given such an object, a set of alternative parameters/methods, and benchmark datasets, the `runPipeline` function then proceeds through all combinations arguments, avoiding recomputing the same step twice and compiling evaluations on the fly to avoid storing potentially large intermediate data.\n",
      "Version: 1.8.0\n",
      "Publication date: None\n",
      "Mantainer: Pierre-Luc Germain <pierre-luc.germain@hest.ethz.ch>\n",
      "Authors:\n",
      "Author: Pierre-Luc Germain [cre, aut] , Anthony Sonrel [aut] , Mark D. Robinson [aut, fnd]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - BiocParallel, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - ComplexHeatmap, type: imports\n",
      "  - SingleCellExperiment, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - Seurat, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - cluster, type: imports\n",
      "  - aricode, type: imports\n",
      "  - methods, type: imports\n",
      "  - utils, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - grid, type: imports\n",
      "  - scales, type: imports\n",
      "  - scran, type: imports\n",
      "  - viridisLite, type: imports\n",
      "  - clue, type: imports\n",
      "  - randomcoloR, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - cowplot, type: imports\n",
      "  - intrinsicDimension, type: imports\n",
      "  - scater, type: imports\n",
      "  - knitr, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - stats, type: imports\n",
      "  - Rtsne, type: imports\n",
      "  - uwot, type: imports\n",
      "  - circlize, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/pipeComp.html\n",
      "\n",
      "Name: plethy\n",
      "Description:\n",
      "This package provides the infrastructure and tools to import, query and perform basic analysis of whole body plethysmography and metabolism data. Currently support is limited to data derived from Buxco respirometry instruments as exported by their FinePointe software.\n",
      "Version: 1.36.0\n",
      "Publication date: None\n",
      "Mantainer: Daniel Bottomly <bottomly@ohsu.edu>\n",
      "Authors:\n",
      "Author: Daniel Bottomly [aut, cre], Marty Ferris [ctb], Beth Wilmot [aut], Shannon McWeeney [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.1.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - DBI (>= 0.5-1) type: depends\n",
      "  - RSQLite (>= 1.1) type: depends\n",
      "  - BiocGenerics, type: depends\n",
      "  - S4Vectors, type: depends\n",
      "  - Streamer, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - plyr, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - Biobase, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/plethy.html\n",
      "\n",
      "Name: PrInCE\n",
      "Description:\n",
      "PrInCE (Predicting Interactomes from Co-Elution) uses a naive Bayes classifier trained on dataset-derived features to recover protein-protein interactions from co-elution chromatogram profiles. This package contains the R implementation of PrInCE.\n",
      "Version: 1.14.0\n",
      "Publication date: None\n",
      "Mantainer: Michael Skinnider <michael.skinnider@msl.ubc.ca>\n",
      "Authors:\n",
      "Author: Michael Skinnider [aut, trl, cre], R. Greg Stacey [ctb], Nichollas Scott [ctb], Anders Kristensen [ctb], Leonard Foster [aut, led]\n",
      "Dependencies:\n",
      "  - R (>= 3.6.0) type: depends\n",
      "  - purrr (>= 0.2.4) type: imports\n",
      "  - dplyr (>= 0.7.4) type: imports\n",
      "  - tidyr (>= 0.8.99) type: imports\n",
      "  - forecast (>= 8.2) type: imports\n",
      "  - progress (>= 1.1.2) type: imports\n",
      "  - Hmisc (>= 4.0) type: imports\n",
      "  - naivebayes (>= 0.9.1) type: imports\n",
      "  - robustbase (>= 0.92-7) type: imports\n",
      "  - ranger (>= 0.8.0) type: imports\n",
      "  - LiblineaR (>= 2.10-8) type: imports\n",
      "  - speedglm (>= 0.3-2) type: imports\n",
      "  - tester (>= 0.1.7) type: imports\n",
      "  - magrittr (>= 1.5) type: imports\n",
      "  - Biobase, type: imports\n",
      "  - MSnbase, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - methods, type: imports\n",
      "  - Rdpack (>= 0.7) type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3 + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/PrInCE.html\n",
      "\n",
      "Name: recountmethylation\n",
      "Description:\n",
      "Resources for cross-study analyses of public DNAm array data from NCBI GEO repo, produced using Illumina's Infinium HumanMethylation450K (HM450K) and MethylationEPIC (EPIC) platforms. Provided functions enable download, summary, and filtering of large compilation files. Vignettes detail background about file formats, example analyses, and more. Note the disclaimer on package load and consult the main manuscripts for further info.\n",
      "Version: 1.8.3\n",
      "Publication date: None\n",
      "Mantainer: Sean K Maden <maden@ohsu.edu>\n",
      "Authors:\n",
      "Author: Sean K Maden [cre, aut] , Brian Walsh [aut] , Kyle Ellrott [aut] , Kasper D Hansen [aut] , Reid F Thompson [aut] , Abhinav Nellore [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - minfi, type: imports\n",
      "  - HDF5Array, type: imports\n",
      "  - rhdf5, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - utils, type: imports\n",
      "  - methods, type: imports\n",
      "  - RCurl, type: imports\n",
      "  - R.utils, type: imports\n",
      "  - BiocFileCache, type: imports\n",
      "  - IlluminaHumanMethylation450kmanifest, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/recountmethylation.html\n",
      "\n",
      "Name: restfulSE\n",
      "Description:\n",
      "This package provides functions and classes to interface with remote data stores by operating on SummarizedExperiment-like objects.\n",
      "Version: 1.20.0\n",
      "Publication date: None\n",
      "Mantainer: Shweta Gopaulakrishnan <shwetagopaul92@gmail.com>\n",
      "Authors:\n",
      "Author: Vincent Carey [aut], Shweta Gopaulakrishnan [cre, aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.6) type: depends\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - DelayedArray, type: depends\n",
      "  - utils, type: imports\n",
      "  - stats, type: imports\n",
      "  - methods, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - AnnotationDbi, type: imports\n",
      "  - DBI, type: imports\n",
      "  - GO.db, type: imports\n",
      "  - rhdf5client, type: imports\n",
      "  - dplyr (>= 0.7.1) type: imports\n",
      "  - magrittr, type: imports\n",
      "  - bigrquery, type: imports\n",
      "  - ExperimentHub, type: imports\n",
      "  - AnnotationHub, type: imports\n",
      "  - rlang, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/restfulSE.html\n",
      "\n",
      "Name: slalom\n",
      "Description:\n",
      "slalom is a scalable modelling framework for single-cell RNA-seq data that uses gene set annotations to dissect single-cell transcriptome heterogeneity, thereby allowing to identify biological drivers of cell-to-cell variability and model confounding factors. The method uses Bayesian factor analysis with a latent variable model to identify active pathways (selected by the user, e.g. KEGG pathways) that explain variation in a single-cell RNA-seq dataset. This an R/C++ implementation of the f-scLVM Python package. See the publication describing the method at https://doi.org/10.1186/s13059-017-1334-8.\n",
      "Version: 1.20.2\n",
      "Publication date: None\n",
      "Mantainer: Davis McCarthy <davis@ebi.ac.uk>\n",
      "Authors:\n",
      "Author: Florian Buettner [aut], Naruemon Pratanwanich [aut], Davis McCarthy [aut, cre], John Marioni [aut], Oliver Stegle [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - Rcpp (>= 0.12.8) type: imports\n",
      "  - RcppArmadillo, type: imports\n",
      "  - BH, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - grid, type: imports\n",
      "  - GSEABase, type: imports\n",
      "  - methods, type: imports\n",
      "  - rsvd, type: imports\n",
      "  - SingleCellExperiment, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - stats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/slalom.html\n",
      "\n",
      "Name: sscore\n",
      "Description:\n",
      "This package contains an implementation of the S-Score algorithm as described by Zhang et al (2002).\n",
      "Version: 1.70.0\n",
      "Publication date: None\n",
      "Mantainer: Richard Kennedy <rkennedy@ms.soph.uab.edu>\n",
      "Authors:\n",
      "Author: Richard Kennedy <rkennedy at ms.soph.uab.edu>, based on C++ code from Li Zhang <zhangli at odin.mdacc.tmc.edu> and Borland Delphi code from Robnet Kerns <rtkerns at vcu.edu>.\n",
      "Dependencies:\n",
      "  - R (>= 1.8.0) type: depends\n",
      "  - affy, type: depends\n",
      "  - affyio, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/sscore.html\n",
      "\n",
      "Name: STdeconvolve\n",
      "Description:\n",
      "STdeconvolve as an unsupervised, reference-free approach to infer latent cell-type proportions and transcriptional profiles within multi-cellular spatially-resolved pixels from spatial transcriptomics (ST) datasets. STdeconvolve builds on latent Dirichlet allocation (LDA), a generative statistical model commonly used in natural language processing for discovering latent topics in collections of documents. In the context of natural language processing, given a count matrix of words in documents, LDA infers the distribution of words for each topic and the distribution of topics in each document. In the context of ST data, given a count matrix of gene expression in multi-cellular ST pixels, STdeconvolve applies LDA to infer the putative transcriptional profile for each cell-type and the proportional representation of each cell-type in each multi-cellular ST pixel.\n",
      "Version: 1.2.0\n",
      "Publication date: None\n",
      "Mantainer: Brendan Miller <bmille79@jh.edu>\n",
      "Authors:\n",
      "Author: Brendan Miller [aut, cre] , Jean Fan [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - topicmodels, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - methods, type: imports\n",
      "  - mgcv, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - scatterpie, type: imports\n",
      "  - viridis, type: imports\n",
      "  - slam, type: imports\n",
      "  - stats, type: imports\n",
      "  - clue, type: imports\n",
      "  - liger, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/STdeconvolve.html\n",
      "\n",
      "Name: synlet\n",
      "Description:\n",
      "Select hits from synthetic lethal RNAi screen data. For example, there are two identical celllines except one gene is knocked-down in one cellline. The interest is to find genes that lead to stronger lethal effect when they are knocked-down further by siRNA. Quality control and various visualisation tools are implemented. Four different algorithms could be used to pick up the interesting hits. This package is designed based on 384 wells plates, but may apply to other platforms with proper configuration.\n",
      "Version: 1.28.0\n",
      "Publication date: None\n",
      "Mantainer: Chunxuan Shao <c.shao@dkfz-heidelberg.de>\n",
      "Authors:\n",
      "Author: Chunxuan Shao <c.shao at dkfz-heidelberg.de>\n",
      "Dependencies:\n",
      "  - R (>= 3.2.0) type: depends\n",
      "  - ggplot2, type: depends\n",
      "  - doBy, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - grid, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - RankProd, type: imports\n",
      "  - reshape2, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/synlet.html\n",
      "\n",
      "Name: topdownr\n",
      "Description:\n",
      "The topdownr package allows automatic and systemic investigation of fragment conditions. It creates Thermo Orbitrap Fusion Lumos method files to test hundreds of fragmentation conditions. Additionally it provides functions to analyse and process the generated MS data and determine the best conditions to maximise overall fragment coverage.\n",
      "Version: 1.20.0\n",
      "Publication date: None\n",
      "Mantainer: Sebastian Gibb <mail@sebastiangibb.de>\n",
      "Authors:\n",
      "Author: Sebastian Gibb [aut, cre] , Pavel Shliaha [aut] , Ole NÃ¸rregaard Jensen [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.5) type: depends\n",
      "  - methods, type: depends\n",
      "  - BiocGenerics, type: depends\n",
      "  - ProtGenerics, type: depends\n",
      "  - Biostrings, type: depends\n",
      "  - S4Vectors, type: depends\n",
      "  - grDevices, type: imports\n",
      "  - stats, type: imports\n",
      "  - tools, type: imports\n",
      "  - utils, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - Matrix (>= 1.4-2) type: imports\n",
      "  - MSnbase, type: imports\n",
      "  - ggplot2 (>= 2.2.1) type: imports\n",
      "  - mzR, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 3)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/topdownr.html\n",
      "\n",
      "Name: transite\n",
      "Description:\n",
      "transite is a computational method that allows comprehensive analysis of the regulatory role of RNA-binding proteins in various cellular processes by leveraging preexisting gene expression data and current knowledge of binding preferences of RNA-binding proteins.\n",
      "Version: 1.16.0\n",
      "Publication date: None\n",
      "Mantainer: Konstantin Krismer <krismer@mit.edu>\n",
      "Authors:\n",
      "Author: Konstantin Krismer [aut, cre, cph] , Anna Gattinger [aut] , Michael Yaffe [ths, cph] , Ian Cannell [ths]\n",
      "Dependencies:\n",
      "  - R (>= 3.5) type: depends\n",
      "  - BiocGenerics, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - dplyr (>= 0.7.6) type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - ggplot2 (>= 3.0.0) type: imports\n",
      "  - ggseqlogo (>= 0.1) type: imports\n",
      "  - grDevices, type: imports\n",
      "  - gridExtra (>= 2.3) type: imports\n",
      "  - methods, type: imports\n",
      "  - parallel, type: imports\n",
      "  - Rcpp (>= 1.0.4.8) type: imports\n",
      "  - scales (>= 1.0.0) type: imports\n",
      "  - stats, type: imports\n",
      "  - TFMPvalue (>= 0.0.8) type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/transite.html\n",
      "\n",
      "Name: ttgsea\n",
      "Description:\n",
      "Functional enrichment analysis methods such as gene set enrichment analysis (GSEA) have been widely used for analyzing gene expression data. GSEA is a powerful method to infer results of gene expression data at a level of gene sets by calculating enrichment scores for predefined sets of genes. GSEA depends on the availability and accuracy of gene sets. There are overlaps between terms of gene sets or categories because multiple terms may exist for a single biological process, and it can thus lead to redundancy within enriched terms. In other words, the sets of related terms are overlapping. Using deep learning, this pakage is aimed to predict enrichment scores for unique tokens or words from text in names of gene sets to resolve this overlapping set issue. Furthermore, we can coin a new term by combining tokens and find its enrichment score by predicting such a combined tokens.\n",
      "Version: 1.6.3\n",
      "Publication date: None\n",
      "Mantainer: Dongmin Jung <dmdmjung@gmail.com>\n",
      "Authors:\n",
      "Author: Dongmin Jung [cre, aut]\n",
      "Dependencies:\n",
      "  - keras, type: depends\n",
      "  - tm, type: imports\n",
      "  - text2vec, type: imports\n",
      "  - tokenizers, type: imports\n",
      "  - textstem, type: imports\n",
      "  - stopwords, type: imports\n",
      "  - data.table, type: imports\n",
      "  - purrr, type: imports\n",
      "  - DiagrammeR, type: imports\n",
      "  - stats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ttgsea.html\n",
      "\n",
      "Name: VariantExperiment\n",
      "Description:\n",
      "VariantExperiment is a Bioconductor package for saving data in VCF/GDS format into RangedSummarizedExperiment object. The high-throughput genetic/genomic data are saved in GDSArray objects. The annotation data for features/samples are saved in DelayedDataFrame format with mono-dimensional GDSArray in each column. The on-disk representation of both assay data and annotation data achieves on-disk reading and processing and saves memory space significantly. The interface of RangedSummarizedExperiment data format enables easy and common manipulations for high-throughput genetic/genomic data with common SummarizedExperiment metaphor in R and Bioconductor.\n",
      "Version: 1.12.0\n",
      "Publication date: None\n",
      "Mantainer: Qian Liu <Qian.Liu@roswellpark.org>\n",
      "Authors:\n",
      "Author: Qian Liu [aut, cre], HervÃ© PagÃ¨s [aut], Martin Morgan [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.6.0) type: depends\n",
      "  - S4Vectors, type: depends\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - GDSArray, type: imports\n",
      "  - DelayedDataFrame, type: imports\n",
      "  - tools, type: imports\n",
      "  - utils, type: imports\n",
      "  - stats, type: imports\n",
      "  - methods, type: imports\n",
      "  - gdsfmt, type: imports\n",
      "  - SNPRelate, type: imports\n",
      "  - SeqArray, type: imports\n",
      "  - SeqVarTools, type: imports\n",
      "  - DelayedArray, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - IRanges, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/VariantExperiment.html\n",
      "\n",
      "Name: VplotR\n",
      "Description:\n",
      "The pattern of digestion and protection from DNA nucleases such as DNAse I, micrococcal nuclease, and Tn5 transposase can be used to infer the location of associated proteins. This package contains useful functions to analyze patterns of paired-end sequencing fragment density. VplotR facilitates the generation of V-plots and footprint profiles over single or aggregated genomic loci of interest.\n",
      "Version: 1.8.0\n",
      "Publication date: None\n",
      "Mantainer: Jacques Serizay <jacquesserizay@gmail.com>\n",
      "Authors:\n",
      "Author: Jacques Serizay [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - IRanges, type: depends\n",
      "  - ggplot2, type: depends\n",
      "  - cowplot, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - GenomicAlignments, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - zoo, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - parallel, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - methods, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/VplotR.html\n",
      "\n",
      "Name: wavClusteR\n",
      "Description:\n",
      "The package provides an integrated pipeline for the analysis of PAR-CLIP data. PAR-CLIP-induced transitions are first discriminated from sequencing errors, SNPs and additional non-experimental sources by a non- parametric mixture model. The protein binding sites (clusters) are then resolved at high resolution and cluster statistics are estimated using a rigorous Bayesian framework. Post-processing of the results, data export for UCSC genome browser visualization and motif search analysis are provided. In addition, the package allows to integrate RNA-Seq data to estimate the False Discovery Rate of cluster detection. Key functions support parallel multicore computing. Note: while wavClusteR was designed for PAR-CLIP data analysis, it can be applied to the analysis of other NGS data obtained from experimental procedures that induce nucleotide substitutions (e.g. BisSeq).\n",
      "Version: 2.32.0\n",
      "Publication date: None\n",
      "Mantainer: Federico Comoglio <federico.comoglio@gmail.com>\n",
      "Authors:\n",
      "Author: Federico Comoglio and Cem Sievers\n",
      "Dependencies:\n",
      "  - R (>= 3.2) type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - Rsamtools, type: depends\n",
      "  - methods, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - foreach, type: imports\n",
      "  - GenomicFeatures, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - Hmisc, type: imports\n",
      "  - mclust, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - seqinr, type: imports\n",
      "  - stringr, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/wavClusteR.html\n",
      "\n",
      "Name: Xeva\n",
      "Description:\n",
      "Contains set of functions to perform analysis of patient-derived xenograft (PDX) data.\n",
      "Version: 1.14.0\n",
      "Publication date: None\n",
      "Mantainer: Benjamin Haibe-Kains <benjamin.haibe.kains@utoronto.ca>\n",
      "Authors:\n",
      "Author: Arvind Mer, Benjamin Haibe-Kains\n",
      "Dependencies:\n",
      "  - R (>= 3.6) type: depends\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - BBmisc, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - scales, type: imports\n",
      "  - ComplexHeatmap, type: imports\n",
      "  - parallel, type: imports\n",
      "  - doParallel, type: imports\n",
      "  - Rmisc, type: imports\n",
      "  - grid, type: imports\n",
      "  - nlme, type: imports\n",
      "  - PharmacoGx, type: imports\n",
      "  - downloader, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/Xeva.html\n",
      "\n",
      "Name: XINA\n",
      "Description:\n",
      "The aim of XINA is to determine which proteins exhibit similar patterns within and across experimental conditions, since proteins with co-abundance patterns may have common molecular functions. XINA imports multiple datasets, tags dataset in silico, and combines the data for subsequent subgrouping into multiple clusters. The result is a single output depicting the variation across all conditions. XINA, not only extracts coabundance profiles within and across experiments, but also incorporates protein-protein interaction databases and integrative resources such as KEGG to infer interactors and molecular functions, respectively, and produces intuitive graphical outputs.\n",
      "Version: 1.16.0\n",
      "Publication date: None\n",
      "Mantainer: Lang Ho Lee <lhlee@bwh.harvard.edu> and Sasha A. Singh <sasingh@bwh.harvard.edu>\n",
      "Authors:\n",
      "Author: Lang Ho Lee <lhlee at bwh.harvard.edu> and Sasha A. Singh <sasingh at bwh.harvard.edu>\n",
      "Dependencies:\n",
      "  - R (>= 3.5) type: depends\n",
      "  - mclust, type: imports\n",
      "  - plyr, type: imports\n",
      "  - alluvial, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - igraph, type: imports\n",
      "  - gridExtra, type: imports\n",
      "  - tools, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - utils, type: imports\n",
      "  - STRINGdb, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/XINA.html\n",
      "\n",
      "Name: airpart\n",
      "Description:\n",
      "Airpart identifies sets of genes displaying differential cell-type-specific allelic imbalance across cell types or states, utilizing single-cell allelic counts. It makes use of a generalized fused lasso with binomial observations of allelic counts to partition cell types by their allelic imbalance. Alternatively, a nonparametric method for partitioning cell types is offered. The package includes a number of visualizations and quality control functions for examining single cell allelic imbalance datasets.\n",
      "Version: 1.6.0\n",
      "Publication date: None\n",
      "Mantainer: Wancen Mu <wancen@live.unc.edu>\n",
      "Authors:\n",
      "Author: Wancen Mu [aut, cre] , Michael Love [aut, ctb]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - SingleCellExperiment, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - scater, type: imports\n",
      "  - stats, type: imports\n",
      "  - smurf, type: imports\n",
      "  - apeglm, type: imports\n",
      "  - emdbook, type: imports\n",
      "  - mclust, type: imports\n",
      "  - clue, type: imports\n",
      "  - dynamicTreeCut, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - plyr, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - ComplexHeatmap, type: imports\n",
      "  - forestplot, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - rlang, type: imports\n",
      "  - lpSolve, type: imports\n",
      "  - grid, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - utils, type: imports\n",
      "  - pbapply, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/airpart.html\n",
      "\n",
      "Name: AnVILPublish\n",
      "Description:\n",
      "Use this package to create or update AnVIL workspaces from resources such as R / Bioconductor packages. The metadata about the package (e.g., select information from the package DESCRIPTION file and from vignette YAML headings) are used to populate the 'DASHBOARD'. Vignettes are translated to python notebooks ready for evaluation in AnVIL.\n",
      "Version: 1.8.0\n",
      "Publication date: None\n",
      "Mantainer: Martin Morgan <mtmorgan.bioc@gmail.com>\n",
      "Authors:\n",
      "Author: Martin Morgan [aut, cre] , Kayla Interdonato [aut], Vincent Carey [ctb]\n",
      "Dependencies:\n",
      "  - AnVIL, type: imports\n",
      "  - httr, type: imports\n",
      "  - jsonlite, type: imports\n",
      "  - rmarkdown, type: imports\n",
      "  - yaml, type: imports\n",
      "  - readr, type: imports\n",
      "  - whisker, type: imports\n",
      "  - tools, type: imports\n",
      "  - utils, type: imports\n",
      "  - stats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/AnVILPublish.html\n",
      "\n",
      "Name: ASAFE\n",
      "Description:\n",
      "Given admixed individuals' bi-allelic SNP genotypes and ancestry pairs (where each ancestry can take one of three values) for multiple SNPs, perform an EM algorithm to deal with the fact that SNP genotypes are unphased with respect to ancestry pairs, in order to estimate ancestry-specific allele frequencies for all SNPs.\n",
      "Version: 1.24.0\n",
      "Publication date: None\n",
      "Mantainer: Qian Zhang <qszhang@uw.edu>\n",
      "Authors:\n",
      "Author: Qian Zhang <qszhang at uw.edu>\n",
      "Dependencies:\n",
      "  - R (>= 3.2) type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ASAFE.html\n",
      "\n",
      "Name: ASpediaFI\n",
      "Description:\n",
      "This package provides functionalities for a systematic and integrative analysis of alternative splicing events and their functional interactions.\n",
      "Version: 1.11.0\n",
      "Publication date: None\n",
      "Mantainer: Doyeong Yu <parklab.bi@gmail.com>\n",
      "Authors:\n",
      "Author: Doyeong Yu, Kyubin Lee, Daejin Hyung, Soo Young Cho, Charny Park\n",
      "Dependencies:\n",
      "  - R (>= 3.6.0) type: depends\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - ROCR, type: depends\n",
      "  - BiocParallel, type: imports\n",
      "  - GenomicAlignments, type: imports\n",
      "  - GenomicFeatures, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - IVAS, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - biomaRt, type: imports\n",
      "  - limma, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - stats, type: imports\n",
      "  - DRaWR, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - Gviz, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - fgsea, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - igraph, type: imports\n",
      "  - graphics, type: imports\n",
      "  - e1071, type: imports\n",
      "  - methods, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - scales, type: imports\n",
      "  - grid, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - mGSZ, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ASpediaFI.html\n",
      "\n",
      "Name: BEAT\n",
      "Description:\n",
      "Model-based analysis of single-cell methylation data\n",
      "Version: 1.36.0\n",
      "Publication date: None\n",
      "Mantainer: Kemal Akman <akman@mpipz.mpg.de>\n",
      "Authors:\n",
      "Author: Kemal Akman <akman at mpipz.mpg.de>\n",
      "Dependencies:\n",
      "  - R (>= 2.13.0) type: depends\n",
      "  - GenomicRanges, type: imports\n",
      "  - ShortRead, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - BSgenome, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL (>= 3.0)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/BEAT.html\n",
      "\n",
      "Name: BGmix\n",
      "Description:\n",
      "Fully Bayesian mixture models for differential gene expression\n",
      "Version: 1.58.0\n",
      "Publication date: None\n",
      "Mantainer: Alex Lewin <a.m.lewin@imperial.ac.uk>\n",
      "Authors:\n",
      "Author: Alex Lewin, Natalia Bochkina\n",
      "Dependencies:\n",
      "  - R (>= 2.3.1) type: depends\n",
      "  - KernSmooth, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/BGmix.html\n",
      "\n",
      "Name: BiocDockerManager\n",
      "Description:\n",
      "Package works analogous to BiocManager but for docker images. Use the BiocDockerManager package to install and manage docker images provided by the Bioconductor project. A convenient package to install images, update images and find which Bioconductor based docker images are available.\n",
      "Version: 1.10.0\n",
      "Publication date: None\n",
      "Mantainer: Bioconductor Package Maintainer <maintainer@bioconductor.org>\n",
      "Authors:\n",
      "Author: Bioconductor Package Maintainer [cre], Nitesh Turaga [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - httr, type: imports\n",
      "  - whisker, type: imports\n",
      "  - readr, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - utils, type: imports\n",
      "  - methods, type: imports\n",
      "  - memoise, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/BiocDockerManager.html\n",
      "\n",
      "Name: biomvRCNS\n",
      "Description:\n",
      "In this package, a Hidden Semi Markov Model (HSMM) and one homogeneous segmentation model are designed and implemented for segmentation genomic data, with the aim of assisting in transcripts detection using high throughput technology like RNA-seq or tiling array, and copy number analysis using aCGH or sequencing.\n",
      "Version: 1.38.0\n",
      "Publication date: None\n",
      "Mantainer: Yang Du <tooyoung@gmail.com>\n",
      "Authors:\n",
      "Author: Yang Du\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - IRanges, type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - Gviz, type: depends\n",
      "  - methods, type: imports\n",
      "  - mvtnorm, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/biomvRCNS.html\n",
      "\n",
      "Name: brainflowprobes\n",
      "Description:\n",
      "Use these functions to characterize genomic regions for BrainFlow target probe design.\n",
      "Version: 1.12.0\n",
      "Publication date: None\n",
      "Mantainer: Amanda Price <amanda.joy.price@gmail.com>\n",
      "Authors:\n",
      "Author: Amanda Price [aut, cre] , Leonardo Collado-Torres [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 3.6.0) type: depends\n",
      "  - Biostrings, type: imports\n",
      "  - BSgenome.Hsapiens.UCSC.hg19, type: imports\n",
      "  - bumphunter, type: imports\n",
      "  - cowplot (>= 1.0.0) type: imports\n",
      "  - derfinder, type: imports\n",
      "  - derfinderPlot, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - ggplot2 (>= 3.1.1) type: imports\n",
      "  - RColorBrewer (>= 1.1) type: imports\n",
      "  - utils, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - GenomicState, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/brainflowprobes.html\n",
      "\n",
      "Name: CAFE\n",
      "Description:\n",
      "Detection and visualizations of gross chromosomal aberrations using Affymetrix expression microarrays as input\n",
      "Version: 1.34.0\n",
      "Publication date: None\n",
      "Mantainer: Sander Bollen <sander.h.bollen@gmail.com>\n",
      "Authors:\n",
      "Author: Sander Bollen\n",
      "Dependencies:\n",
      "  - R (>= 2.10) type: depends\n",
      "  - biovizBase, type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - IRanges, type: depends\n",
      "  - ggbio, type: depends\n",
      "  - affy, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - annotate, type: imports\n",
      "  - grid, type: imports\n",
      "  - gridExtra, type: imports\n",
      "  - tcltk, type: imports\n",
      "  - Biobase, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/CAFE.html\n",
      "\n",
      "Name: CancerInSilico\n",
      "Description:\n",
      "The CancerInSilico package provides an R interface for running mathematical models of tumor progresson and generating gene expression data from the results. This package has the underlying models implemented in C++ and the output and analysis features implemented in R.\n",
      "Version: 2.18.0\n",
      "Publication date: None\n",
      "Mantainer: Thomas D. Sherman <tomsherman159@gmail.com>, Elana J. Fertig <ejfertig@jhmi.edu>\n",
      "Authors:\n",
      "Author: Thomas D. Sherman, Raymond Cheng, Elana J. Fertig\n",
      "Dependencies:\n",
      "  - R (>= 3.4) type: depends\n",
      "  - Rcpp, type: depends\n",
      "  - methods, type: imports\n",
      "  - utils, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/CancerInSilico.html\n",
      "\n",
      "Name: cbaf\n",
      "Description:\n",
      "This package contains functions that allow analysing and comparing omic data across various cancers/cancer subgroups easily. So far, it is compatible with RNA-seq, microRNA-seq, microarray and methylation datasets that are stored on cbioportal.org.\n",
      "Version: 1.20.1\n",
      "Publication date: None\n",
      "Mantainer: Arman Shahrisa <shahrisa.arman@hotmail.com>\n",
      "Authors:\n",
      "Author: Arman Shahrisa [aut, cre, cph], Maryam Tahmasebi Birgani [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - BiocFileCache, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - cBioPortalData, type: imports\n",
      "  - genefilter, type: imports\n",
      "  - gplots, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - openxlsx, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/cbaf.html\n",
      "\n",
      "Name: ChIPanalyser\n",
      "Description:\n",
      "ChIPanalyser is a package to predict and understand TF binding by utilizing a statistical thermodynamic model. The model incorporates 4 main factors thought to drive TF binding: Chromatin State, Binding energy, Number of bound molecules and a scaling factor modulating TF binding affinity. Taken together, ChIPanalyser produces ChIP-like profiles that closely mimic the patterns seens in real ChIP-seq data.\n",
      "Version: 1.20.0\n",
      "Publication date: None\n",
      "Mantainer: Patrick C.N. Martin <pcnmartin@gmail.com>\n",
      "Authors:\n",
      "Author: Patrick C.N.Martin & Nicolae Radu Zabet\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - Biostrings, type: depends\n",
      "  - BSgenome, type: depends\n",
      "  - RcppRoll, type: depends\n",
      "  - parallel, type: depends\n",
      "  - methods, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - ROCR, type: imports\n",
      "  - BiocManager, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ChIPanalyser.html\n",
      "\n",
      "Name: chromswitch\n",
      "Description:\n",
      "Chromswitch implements a flexible method to detect chromatin state switches between samples in two biological conditions in a specific genomic region of interest given peaks or chromatin state calls from ChIP-seq data.\n",
      "Version: 1.20.0\n",
      "Publication date: None\n",
      "Mantainer: Selin Jessa <selinjessa@gmail.com>\n",
      "Authors:\n",
      "Author: Selin Jessa [aut, cre], Claudia L. Kleinman [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - cluster (>= 2.0.6) type: imports\n",
      "  - Biobase, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - dplyr (>= 0.5.0) type: imports\n",
      "  - gplots (>= 3.0.1) type: imports\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - lazyeval (>= 0.2.0) type: imports\n",
      "  - matrixStats (>= 0.52) type: imports\n",
      "  - magrittr (>= 1.5) type: imports\n",
      "  - methods, type: imports\n",
      "  - NMF (>= 0.20.6) type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - stats, type: imports\n",
      "  - tidyr (>= 0.6.3) type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/chromswitch.html\n",
      "\n",
      "Name: clippda\n",
      "Description:\n",
      "Methods for the nalysis of data from clinical proteomic profiling studies. The focus is on the studies of human subjects, which are often observational case-control by design and have technical replicates. A method for sample size determination for planning these studies is proposed. It incorporates routines for adjusting for the expected heterogeneities and imbalances in the data and the within-sample replicate correlations.\n",
      "Version: 1.48.0\n",
      "Publication date: None\n",
      "Mantainer: Stephen Nyangoma <s.o.nyangoma@bham.ac.uk>\n",
      "Authors:\n",
      "Author: Stephen Nyangoma\n",
      "Dependencies:\n",
      "  - R (>= 2.13.1) type: depends\n",
      "  - limma, type: depends\n",
      "  - statmod, type: depends\n",
      "  - rgl, type: depends\n",
      "  - lattice, type: depends\n",
      "  - scatterplot3d, type: depends\n",
      "  - graphics, type: depends\n",
      "  - grDevices, type: depends\n",
      "  - stats, type: depends\n",
      "  - utils, type: depends\n",
      "  - Biobase, type: depends\n",
      "  - tools, type: depends\n",
      "  - methods, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>=2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/clippda.html\n",
      "\n",
      "Name: contiBAIT\n",
      "Description:\n",
      "Using strand inheritance data from multiple single cells from the organism whose genome is to be assembled, contiBAIT can cluster unbridged contigs together into putative chromosomes, and order the contigs within those chromosomes.\n",
      "Version: 1.26.0\n",
      "Publication date: None\n",
      "Mantainer: Kieran O'Neill <koneill@bccrc.ca>\n",
      "Authors:\n",
      "Author: Kieran O'Neill, Mark Hills, Mike Gottlieb\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - BH (>= 1.51.0.3) type: depends\n",
      "  - Rsamtools, type: depends\n",
      "  - data.table, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - clue, type: imports\n",
      "  - cluster, type: imports\n",
      "  - gplots, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - Rcpp, type: imports\n",
      "  - TSP, type: imports\n",
      "  - GenomicFiles, type: imports\n",
      "  - gtools, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - DNAcopy, type: imports\n",
      "  - colorspace, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - methods, type: imports\n",
      "  - exomeCopy, type: imports\n",
      "  - GenomicAlignments, type: imports\n",
      "  - diagram, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "BSD_2_clause + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/contiBAIT.html\n",
      "\n",
      "Name: crisprseekplus\n",
      "Description:\n",
      "Bioinformatics platform containing interface to work with offTargetAnalysis and compare2Sequences in the CRISPRseek package, and GUIDEseqAnalysis.\n",
      "Version: 1.24.0\n",
      "Publication date: None\n",
      "Mantainer: Alper Kucukural <alper.kucukural@umassmed.edu>\n",
      "Authors:\n",
      "Author: Sophie Wigmore <Sophie.Wigmore at umassmed.edu>, Alper Kucukural <alper.kucukural at umassmed.edu>, Lihua Julie Zhu <julie.zhu at umassmed.edu>, Michael Brodsky <Michael.Brodsky at umassmed.edu>, Manuel Garber <Manuel.Garber at umassmed.edu>\n",
      "Dependencies:\n",
      "  - R (>= 3.3.0) type: depends\n",
      "  - shiny, type: depends\n",
      "  - shinyjs, type: depends\n",
      "  - CRISPRseek, type: depends\n",
      "  - DT, type: imports\n",
      "  - utils, type: imports\n",
      "  - GUIDEseq, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - GenomicFeatures, type: imports\n",
      "  - BiocManager, type: imports\n",
      "  - BSgenome, type: imports\n",
      "  - AnnotationDbi, type: imports\n",
      "  - hash, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3 + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/crisprseekplus.html\n",
      "\n",
      "Name: deco\n",
      "Description:\n",
      "This package discovers differential features in hetero- and homogeneous omic data by a two-step method including subsampling LIMMA and NSCA. DECO reveals feature associations to hidden subclasses not exclusively related to higher deregulation levels.\n",
      "Version: 1.13.0\n",
      "Publication date: None\n",
      "Mantainer: Francisco Jose Campos Laborie <fjcamlab@gmail.com>\n",
      "Authors:\n",
      "Author: Francisco Jose Campos-Laborie, Jose Manuel Sanchez-Santos and Javier De Las Rivas. Bioinformatics and Functional Genomics Group. Cancer Research Center (CiC-IBMCC, CSIC/USAL). Salamanca. Spain.\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - AnnotationDbi, type: depends\n",
      "  - BiocParallel, type: depends\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - limma, type: depends\n",
      "  - stats, type: imports\n",
      "  - methods, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - foreign, type: imports\n",
      "  - graphics, type: imports\n",
      "  - BiocStyle, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - cluster, type: imports\n",
      "  - gplots, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - locfit, type: imports\n",
      "  - made4, type: imports\n",
      "  - ade4, type: imports\n",
      "  - sfsmisc, type: imports\n",
      "  - scatterplot3d, type: imports\n",
      "  - gdata, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - utils, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - gridExtra, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>=3)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/deco.html\n",
      "\n",
      "Name: DEScan2\n",
      "Description:\n",
      "Integrated peak and differential caller, specifically designed for broad epigenomic signals.\n",
      "Version: 1.18.2\n",
      "Publication date: None\n",
      "Mantainer: Dario Righelli <dario.righelli@gmail.com>\n",
      "Authors:\n",
      "Author: Dario Righelli [aut, cre], John Koberstein [aut], Bruce Gomes [aut], Nancy Zhang [aut], Claudia Angelini [aut], Lucia Peixoto [aut], Davide Risso [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.5) type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - BiocParallel, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - ChIPpeakAnno, type: imports\n",
      "  - data.table, type: imports\n",
      "  - DelayedArray, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - GenomicAlignments, type: imports\n",
      "  - glue, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - plyr, type: imports\n",
      "  - Rcpp (>= 0.12.13) type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - tools, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/DEScan2.html\n",
      "\n",
      "Name: DrugVsDisease\n",
      "Description:\n",
      "This package generates ranked lists of differential gene expression for either disease or drug profiles. Input data can be downloaded from Array Express or GEO, or from local CEL files. Ranked lists of differential expression and associated p-values are calculated using Limma. Enrichment scores (Subramanian et al. PNAS 2005) are calculated to a reference set of default drug or disease profiles, or a set of custom data supplied by the user. Network visualisation of significant scores are output in Cytoscape format.\n",
      "Version: 2.40.0\n",
      "Publication date: None\n",
      "Mantainer: j. Saez-Rodriguez <saezrodriguez@ebi.ac.uk>\n",
      "Authors:\n",
      "Author: C. Pacini\n",
      "Dependencies:\n",
      "  - R (>= 2.10) type: depends\n",
      "  - affy, type: depends\n",
      "  - limma, type: depends\n",
      "  - biomaRt, type: depends\n",
      "  - ArrayExpress, type: depends\n",
      "  - GEOquery, type: depends\n",
      "  - DrugVsDiseasedata, type: depends\n",
      "  - cMap2data, type: depends\n",
      "  - qvalue, type: depends\n",
      "  - annotate, type: imports\n",
      "  - hgu133a.db, type: imports\n",
      "  - hgu133a2.db, type: imports\n",
      "  - hgu133plus2.db, type: imports\n",
      "  - RUnit, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - xtable, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/DrugVsDisease.html\n",
      "\n",
      "Name: ecolitk\n",
      "Description:\n",
      "Meta-data and tools to work with E. coli. The tools are mostly plotting functions to work with circular genomes. They can used with other genomes/plasmids.\n",
      "Version: 1.70.0\n",
      "Publication date: None\n",
      "Mantainer: Laurent Gautier <lgautier@gmail.com>\n",
      "Authors:\n",
      "Author: Laurent Gautier\n",
      "Dependencies:\n",
      "  - R (>= 2.10) type: depends\n",
      "  - Biobase, type: imports\n",
      "  - graphics, type: imports\n",
      "  - methods, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ecolitk.html\n",
      "\n",
      "Name: eegc\n",
      "Description:\n",
      "This package has been developed to evaluate cellular engineering processes for direct differentiation of stem cells or conversion (transdifferentiation) of somatic cells to primary cells based on high throughput gene expression data screened either by DNA microarray or RNA sequencing. The package takes gene expression profiles as inputs from three types of samples: (i) somatic or stem cells to be (trans)differentiated (input of the engineering process), (ii) induced cells to be evaluated (output of the engineering process) and (iii) target primary cells (reference for the output). The package performs differential gene expression analysis for each pair-wise sample comparison to identify and evaluate the transcriptional differences among the 3 types of samples (input, output, reference). The ideal goal is to have induced and primary reference cell showing overlapping profiles, both very different from the original cells.\n",
      "Version: 1.24.0\n",
      "Publication date: None\n",
      "Mantainer: Xiaoyuan Zhou <zhouxiaoyuan@picb.ac.cn>\n",
      "Authors:\n",
      "Author: Xiaoyuan Zhou, Guofeng Meng, Christine Nardini, Hongkang Mei\n",
      "Dependencies:\n",
      "  - R (>= 3.4.0) type: depends\n",
      "  - R.utils, type: imports\n",
      "  - gplots, type: imports\n",
      "  - sna, type: imports\n",
      "  - wordcloud, type: imports\n",
      "  - igraph, type: imports\n",
      "  - pheatmap, type: imports\n",
      "  - edgeR, type: imports\n",
      "  - DESeq2, type: imports\n",
      "  - clusterProfiler, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - org.Hs.eg.db, type: imports\n",
      "  - org.Mm.eg.db, type: imports\n",
      "  - limma, type: imports\n",
      "  - DOSE, type: imports\n",
      "  - AnnotationDbi, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/eegc.html\n",
      "\n",
      "Name: epigenomix\n",
      "Description:\n",
      "A package for the integrative analysis of RNA-seq or microarray based gene transcription and histone modification data obtained by ChIP-seq. The package provides methods for data preprocessing and matching as well as methods for fitting bayesian mixture models in order to detect genes with differences in both data types.\n",
      "Version: 1.38.0\n",
      "Publication date: None\n",
      "Mantainer: Hans-Ulrich Klein <h.klein@uni-muenster.de>\n",
      "Authors:\n",
      "Author: Hans-Ulrich Klein, Martin Schaefer\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - Biobase, type: depends\n",
      "  - S4Vectors, type: depends\n",
      "  - IRanges, type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - BiocGenerics, type: imports\n",
      "  - MCMCpack, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - parallel, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - beadarray, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/epigenomix.html\n",
      "\n",
      "Name: epigraHMM\n",
      "Description:\n",
      "epigraHMM provides a set of tools for the analysis of epigenomic data based on hidden Markov Models. It contains two separate peak callers, one for consensus peaks from biological or technical replicates, and one for differential peaks from multi-replicate multi-condition experiments. In differential peak calling, epigraHMM provides window-specific posterior probabilities associated with every possible combinatorial pattern of read enrichment across conditions.\n",
      "Version: 1.6.2\n",
      "Publication date: None\n",
      "Mantainer: Pedro Baldoni <pedrobaldoni@gmail.com>\n",
      "Authors:\n",
      "Author: Pedro Baldoni [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - Rcpp, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - data.table, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - methods, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - bamsignals, type: imports\n",
      "  - csaw, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - limma, type: imports\n",
      "  - stats, type: imports\n",
      "  - Rhdf5lib, type: imports\n",
      "  - rhdf5, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - MASS, type: imports\n",
      "  - scales, type: imports\n",
      "  - ggpubr, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - GreyListChIP, type: imports\n",
      "  - pheatmap, type: imports\n",
      "  - grDevices, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/epigraHMM.html\n",
      "\n",
      "Name: fCCAC\n",
      "Description:\n",
      "Computational evaluation of variability across DNA or RNA sequencing datasets is a crucial step in genomics, as it allows both to evaluate reproducibility of replicates, and to compare different datasets to identify potential correlations. fCCAC applies functional Canonical Correlation Analysis to allow the assessment of: (i) reproducibility of biological or technical replicates, analyzing their shared covariance in higher order components; and (ii) the associations between different datasets. fCCAC represents a more sophisticated approach that complements Pearson correlation of genomic coverage.\n",
      "Version: 1.24.0\n",
      "Publication date: None\n",
      "Mantainer: Pedro Madrigal <pmadrigal@ebi.ac.uk>\n",
      "Authors:\n",
      "Author: Pedro Madrigal [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.2.0) type: depends\n",
      "  - S4Vectors, type: depends\n",
      "  - IRanges, type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - grid, type: depends\n",
      "  - fda, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - genomation, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - ComplexHeatmap, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/fCCAC.html\n",
      "\n",
      "Name: flowcatchR\n",
      "Description:\n",
      "flowcatchR is a set of tools to analyze in vivo microscopy imaging data, focused on tracking flowing blood cells. It guides the steps from segmentation to calculation of features, filtering out particles not of interest, providing also a set of utilities to help checking the quality of the performed operations (e.g. how good the segmentation was). It allows investigating the issue of tracking flowing cells such as in blood vessels, to categorize the particles in flowing, rolling and adherent. This classification is applied in the study of phenomena such as hemostasis and study of thrombosis development. Moreover, flowcatchR presents an integrated workflow solution, based on the integration with a Shiny App and Jupyter notebooks, which is delivered alongside the package, and can enable fully reproducible bioimage analysis in the R environment.\n",
      "Version: 1.32.0\n",
      "Publication date: None\n",
      "Mantainer: Federico Marini <marinif@uni-mainz.de>\n",
      "Authors:\n",
      "Author: Federico Marini [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 2.10) type: depends\n",
      "  - methods, type: depends\n",
      "  - EBImage, type: depends\n",
      "  - colorRamps, type: imports\n",
      "  - abind, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - plotly, type: imports\n",
      "  - shiny, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "BSD_3_clause + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/flowcatchR.html\n",
      "\n",
      "Name: flowMap\n",
      "Description:\n",
      "flowMap quantifies the similarity of cell populations across multiple flow cytometry samples using a nonparametric multivariate statistical test. The method is able to map cell populations of different size, shape, and proportion across multiple flow cytometry samples. The algorithm can be incorporate in any flow cytometry work flow that requires accurat quantification of similarity between cell populations.\n",
      "Version: 1.36.0\n",
      "Publication date: None\n",
      "Mantainer: Chiaowen Joyce Hsiao <joyce.hsiao1@gmail.com>\n",
      "Authors:\n",
      "Author: Chiaowen Joyce Hsiao, Yu Qian, and Richard H. Scheuermann\n",
      "Dependencies:\n",
      "  - R (>= 3.0.1) type: depends\n",
      "  - ade4 (>= 1.5-2) type: depends\n",
      "  - doParallel (>= 1.0.3) type: depends\n",
      "  - abind (>= 1.4.0) type: depends\n",
      "  - reshape2 (>= 1.2.2) type: depends\n",
      "  - scales (>= 0.2.3) type: depends\n",
      "  - Matrix (>= 1.1-4) type: depends\n",
      "  - methods (>= 2.14) type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>=2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/flowMap.html\n",
      "\n",
      "Name: flowMatch\n",
      "Description:\n",
      "Matching cell populations and building meta-clusters and templates from a collection of FC samples.\n",
      "Version: 1.34.0\n",
      "Publication date: None\n",
      "Mantainer: Ariful Azad <azad@lbl.gov>\n",
      "Authors:\n",
      "Author: Ariful Azad\n",
      "Dependencies:\n",
      "  - R (>= 3.0.0) type: depends\n",
      "  - Rcpp (>= 0.11.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - flowCore, type: depends\n",
      "  - Biobase, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/flowMatch.html\n",
      "\n",
      "Name: GAprediction\n",
      "Description:\n",
      "[GAprediction] predicts gestational age using Illumina HumanMethylation450 CpG data.\n",
      "Version: 1.24.0\n",
      "Publication date: None\n",
      "Mantainer: Jon Bohlin <jon.bohlin@gmail.com>\n",
      "Authors:\n",
      "Author: Jon Bohlin\n",
      "Dependencies:\n",
      "  - R (>= 3.3) type: depends\n",
      "  - glmnet, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - Matrix, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>=2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/GAprediction.html\n",
      "\n",
      "Name: GARS\n",
      "Description:\n",
      "Feature selection aims to identify and remove redundant, irrelevant and noisy variables from high-dimensional datasets. Selecting informative features affects the subsequent classification and regression analyses by improving their overall performances. Several methods have been proposed to perform feature selection: most of them relies on univariate statistics, correlation, entropy measurements or the usage of backward/forward regressions. Herein, we propose an efficient, robust and fast method that adopts stochastic optimization approaches for high-dimensional. GARS is an innovative implementation of a genetic algorithm that selects robust features in high-dimensional and challenging datasets.\n",
      "Version: 1.18.0\n",
      "Publication date: None\n",
      "Mantainer: Mattia Chiesa <mattia.chiesa@hotmail.it>\n",
      "Authors:\n",
      "Author: Mattia Chiesa <mattia.chiesa at hotmail.it>, Luca Piacentini <luca.piacentini at cardiologicomonzino.it>\n",
      "Dependencies:\n",
      "  - R (>= 3.5) type: depends\n",
      "  - ggplot2, type: depends\n",
      "  - cluster, type: depends\n",
      "  - DaMiRseq, type: imports\n",
      "  - MLSeq, type: imports\n",
      "  - stats, type: imports\n",
      "  - methods, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/GARS.html\n",
      "\n",
      "Name: genoCN\n",
      "Description:\n",
      "Simultaneous identification of copy number states and genotype calls for regions of either copy number variations or copy number aberrations\n",
      "Version: 1.50.0\n",
      "Publication date: None\n",
      "Mantainer: Wei Sun <wsun@bios.unc.edu>\n",
      "Authors:\n",
      "Author: Wei Sun and ZhengZheng Tang\n",
      "Dependencies:\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>=2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/genoCN.html\n",
      "\n",
      "Name: genotypeeval\n",
      "Description:\n",
      "Takes in a gVCF or VCF and reports metrics to assess quality of calls.\n",
      "Version: 1.30.0\n",
      "Publication date: None\n",
      "Mantainer: Jennifer Tom <tom.jennifer@gene.com>\n",
      "Authors:\n",
      "Author: Jennifer Tom [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 3.4.0) type: depends\n",
      "  - VariantAnnotation, type: depends\n",
      "  - ggplot2, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - methods, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/genotypeeval.html\n",
      "\n",
      "Name: GEOsubmission\n",
      "Description:\n",
      "Helps to easily submit a microarray dataset and the associated sample information to GEO by preparing a single file for upload (direct deposit).\n",
      "Version: 1.50.0\n",
      "Publication date: None\n",
      "Mantainer: Alexandre Kuhn <alexandre.m.kuhn@gmail.com>\n",
      "Authors:\n",
      "Author: Alexandre Kuhn <alexandre.m.kuhn at gmail.com>\n",
      "Dependencies:\n",
      "  - affy, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/GEOsubmission.html\n",
      "\n",
      "Name: gep2pep\n",
      "Description:\n",
      "Pathway Expression Profiles (PEPs) are based on the expression of pathways (defined as sets of genes) as opposed to individual genes. This package converts gene expression profiles to PEPs and performs enrichment analysis of both pathways and experimental conditions, such as \"drug set enrichment analysis\" and \"gene2drug\" drug discovery analysis respectively.\n",
      "Version: 1.18.0\n",
      "Publication date: None\n",
      "Mantainer: Francesco Napolitano <franapoli@gmail.com>\n",
      "Authors:\n",
      "Author: Francesco Napolitano <franapoli at gmail.com>\n",
      "Dependencies:\n",
      "  - repo (>= 2.1.1) type: imports\n",
      "  - foreach, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - GSEABase, type: imports\n",
      "  - methods, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - XML, type: imports\n",
      "  - rhdf5, type: imports\n",
      "  - digest, type: imports\n",
      "  - iterators, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/gep2pep.html\n",
      "\n",
      "Name: graper\n",
      "Description:\n",
      "This package enables regression and classification on high-dimensional data with different relative strengths of penalization for different feature groups, such as different assays or omic types. The optimal relative strengths are chosen adaptively. Optimisation is performed using a variational Bayes approach.\n",
      "Version: 1.14.2\n",
      "Publication date: None\n",
      "Mantainer: Britta Velten <britta.velten@gmail.com>\n",
      "Authors:\n",
      "Author: Britta Velten [aut, cre], Wolfgang Huber [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.6) type: depends\n",
      "  - Matrix, type: imports\n",
      "  - Rcpp, type: imports\n",
      "  - stats, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - methods, type: imports\n",
      "  - cowplot, type: imports\n",
      "  - matrixStats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/graper.html\n",
      "\n",
      "Name: hierGWAS\n",
      "Description:\n",
      "Testing individual SNPs, as well as arbitrarily large groups of SNPs in GWA studies, using a joint model of all SNPs. The method controls the FWER, and provides an automatic, data-driven refinement of the SNP clusters to smaller groups or single markers.\n",
      "Version: 1.28.0\n",
      "Publication date: None\n",
      "Mantainer: Laura Buzdugan <buzdugan@stat.math.ethz.ch>\n",
      "Authors:\n",
      "Author: Laura Buzdugan\n",
      "Dependencies:\n",
      "  - R (>= 3.2.0) type: depends\n",
      "  - fastcluster, type: imports\n",
      "  - glmnet, type: imports\n",
      "  - fmsb, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/hierGWAS.html\n",
      "\n",
      "Name: ILoReg\n",
      "Description:\n",
      "ILoReg is a tool for identification of cell populations from scRNA-seq data. In particular, ILoReg is useful for finding cell populations with subtle transcriptomic differences. The method utilizes a self-supervised learning method, called Iteratitive Clustering Projection (ICP), to find cluster probabilities, which are used in noise reduction prior to PCA and the subsequent hierarchical clustering and t-SNE steps. Additionally, functions for differential expression analysis to find gene markers for the populations and gene expression visualization are provided.\n",
      "Version: 1.8.0\n",
      "Publication date: None\n",
      "Mantainer: Johannes Smolander <johannes.smolander@gmail.com>\n",
      "Authors:\n",
      "Author: Johannes Smolander [cre, aut], Sini Junttila [aut], Mikko S VenÃ¤lÃ¤inen [aut], Laura L Elo [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.0.0) type: depends\n",
      "  - Matrix, type: imports\n",
      "  - parallel, type: imports\n",
      "  - foreach, type: imports\n",
      "  - aricode, type: imports\n",
      "  - LiblineaR, type: imports\n",
      "  - SparseM, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - cowplot, type: imports\n",
      "  - RSpectra, type: imports\n",
      "  - umap, type: imports\n",
      "  - Rtsne, type: imports\n",
      "  - fastcluster, type: imports\n",
      "  - parallelDist, type: imports\n",
      "  - cluster, type: imports\n",
      "  - dendextend, type: imports\n",
      "  - DescTools, type: imports\n",
      "  - plyr, type: imports\n",
      "  - scales, type: imports\n",
      "  - pheatmap, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - doRNG, type: imports\n",
      "  - SingleCellExperiment, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - doSNOW, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ILoReg.html\n",
      "\n",
      "Name: IMAS\n",
      "Description:\n",
      "Integrative analysis of Multi-omics data for Alternative splicing.\n",
      "Version: 1.22.0\n",
      "Publication date: None\n",
      "Mantainer: Seonggyun Han <hangost@ssu.ac.kr>\n",
      "Authors:\n",
      "Author: Seonggyun Han, Younghee Lee\n",
      "Dependencies:\n",
      "  - R (> 3.0.0) type: depends\n",
      "  - GenomicFeatures, type: depends\n",
      "  - ggplot2, type: depends\n",
      "  - IVAS, type: depends\n",
      "  - doParallel, type: imports\n",
      "  - lme4, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - foreach, type: imports\n",
      "  - AnnotationDbi, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - stats, type: imports\n",
      "  - ggfortify, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - methods, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - utils, type: imports\n",
      "  - graphics, type: imports\n",
      "  - gridExtra, type: imports\n",
      "  - grid, type: imports\n",
      "  - lattice, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - survival, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - GenomicAlignments, type: imports\n",
      "  - parallel, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/IMAS.html\n",
      "\n",
      "Name: IRISFGM\n",
      "Description:\n",
      "Single-cell RNA-Seq data is useful in discovering cell heterogeneity and signature genes in specific cell populations in cancer and other complex diseases. Specifically, the investigation of functional gene modules (FGM) can help to understand gene interactive networks and complex biological processes. QUBIC2 is recognized as one of the most efficient and effective tools for FGM identification from scRNA-Seq data. However, its availability is limited to a C implementation, and its applicative power is affected by only a few downstream analyses functionalities. We developed an R package named IRIS-FGM (integrative scRNA-Seq interpretation system for functional gene module analysis) to support the investigation of FGMs and cell clustering using scRNA-Seq data. Empowered by QUBIC2, IRIS-FGM can identify co-expressed and co-regulated FGMs, predict types/clusters, identify differentially expressed genes, and perform functional enrichment analysis. It is noteworthy that IRIS-FGM also applies Seurat objects that can be easily used in the Seurat vignettes.\n",
      "Version: 1.6.0\n",
      "Publication date: None\n",
      "Mantainer: Yuzhou Chang <yuzhou.chang@osumc.edu>\n",
      "Authors:\n",
      "Author: Yuzhou Chang [aut, cre], Qin Ma [aut], Carter Allen [aut], Dongjun Chung [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - Rcpp (>= 1.0.0) type: imports\n",
      "  - MCL, type: imports\n",
      "  - anocva, type: imports\n",
      "  - Polychrome, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - colorspace, type: imports\n",
      "  - AnnotationDbi, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - org.Hs.eg.db, type: imports\n",
      "  - org.Mm.eg.db, type: imports\n",
      "  - pheatmap, type: imports\n",
      "  - AdaptGauss, type: imports\n",
      "  - DEsingle, type: imports\n",
      "  - DrImpute, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - Seurat, type: imports\n",
      "  - SingleCellExperiment, type: imports\n",
      "  - clusterProfiler, type: imports\n",
      "  - ggpubr, type: imports\n",
      "  - ggraph, type: imports\n",
      "  - igraph, type: imports\n",
      "  - mixtools, type: imports\n",
      "  - scater, type: imports\n",
      "  - scran, type: imports\n",
      "  - stats, type: imports\n",
      "  - methods, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - utils, type: imports\n",
      "  - knitr, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/IRISFGM.html\n",
      "\n",
      "Name: ITALICS\n",
      "Description:\n",
      "A Method to normalize of Affymetrix GeneChip Human Mapping 100K and 500K set\n",
      "Version: 2.58.0\n",
      "Publication date: None\n",
      "Mantainer: Guillem Rigaill <italics@curie.fr>\n",
      "Authors:\n",
      "Author: Guillem Rigaill, Philippe Hupe\n",
      "Dependencies:\n",
      "  - R (>= 2.0.0) type: depends\n",
      "  - GLAD, type: depends\n",
      "  - ITALICSData, type: depends\n",
      "  - oligo, type: depends\n",
      "  - affxparser, type: depends\n",
      "  - pd.mapping50k.xba240, type: depends\n",
      "  - affxparser, type: imports\n",
      "  - DBI, type: imports\n",
      "  - GLAD, type: imports\n",
      "  - oligo, type: imports\n",
      "  - oligoClasses, type: imports\n",
      "  - stats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ITALICS.html\n",
      "\n",
      "Name: LiquidAssociation\n",
      "Description:\n",
      "The package contains functions for calculate direct and model-based estimators for liquid association. It also provides functions for testing the existence of liquid association given a gene triplet data.\n",
      "Version: 1.52.0\n",
      "Publication date: None\n",
      "Mantainer: Yen-Yi Ho <yho@jhsph.edu>\n",
      "Authors:\n",
      "Author: Yen-Yi Ho <yho at jhsph.edu>\n",
      "Dependencies:\n",
      "  - geepack, type: depends\n",
      "  - methods, type: depends\n",
      "  - yeastCC, type: depends\n",
      "  - org.Sc.sgd.db, type: depends\n",
      "  - Biobase, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>=3)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/LiquidAssociation.html\n",
      "\n",
      "Name: logitT\n",
      "Description:\n",
      "The logitT library implements the Logit-t algorithm introduced in --A high performance test of differential gene expression for oligonucleotide arrays-- by William J Lemon, Sandya Liyanarachchi and Ming You for use with Affymetrix data stored in an AffyBatch object in R.\n",
      "Version: 1.56.0\n",
      "Publication date: None\n",
      "Mantainer: Tobias Guennel <tguennel@vcu.edu>\n",
      "Authors:\n",
      "Author: Tobias Guennel <tguennel at vcu.edu>\n",
      "Dependencies:\n",
      "  - affy, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/logitT.html\n",
      "\n",
      "Name: maPredictDSC\n",
      "Description:\n",
      "This package implements the classification pipeline of the best overall team (Team221) in the IMPROVER Diagnostic Signature Challenge. Additional functionality is added to compare 27 combinations of data preprocessing, feature selection and classifier types.\n",
      "Version: 1.36.0\n",
      "Publication date: None\n",
      "Mantainer: Adi Laurentiu Tarca <atarca@med.wayne.edu>\n",
      "Authors:\n",
      "Author: Adi Laurentiu Tarca <atarca at med.wayne.edu>\n",
      "Dependencies:\n",
      "  - R (>= 2.15.0) type: depends\n",
      "  - MASS, type: depends\n",
      "  - affy, type: depends\n",
      "  - limma, type: depends\n",
      "  - gcrma, type: depends\n",
      "  - ROC, type: depends\n",
      "  - class, type: depends\n",
      "  - e1071, type: depends\n",
      "  - caret, type: depends\n",
      "  - hgu133plus2.db, type: depends\n",
      "  - ROCR, type: depends\n",
      "  - AnnotationDbi, type: depends\n",
      "  - LungCancerACvsSCCGEO, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/maPredictDSC.html\n",
      "\n",
      "Name: MBCB\n",
      "Description:\n",
      "This package provides a model-based background correction method, which incorporates the negative control beads to pre-process Illumina BeadArray data.\n",
      "Version: 1.52.0\n",
      "Publication date: None\n",
      "Mantainer: Jeff Allen <Jeffrey.Allen@UTSouthwestern.edu>\n",
      "Authors:\n",
      "Author: Yang Xie <Yang.Xie at UTSouthwestern.edu>\n",
      "Dependencies:\n",
      "  - R (>= 2.9.0) type: depends\n",
      "  - tcltk, type: depends\n",
      "  - tcltk2, type: depends\n",
      "  - preprocessCore, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/MBCB.html\n",
      "\n",
      "Name: MCbiclust\n",
      "Description:\n",
      "Custom made algorithm and associated methods for finding, visualising and analysing biclusters in large gene expression data sets. Algorithm is based on with a supplied gene set of size n, finding the maximum strength correlation matrix containing m samples from the data set.\n",
      "Version: 1.22.0\n",
      "Publication date: None\n",
      "Mantainer: Robert Bentham <robert.bentham.11@ucl.ac.uk>\n",
      "Authors:\n",
      "Author: Robert Bentham\n",
      "Dependencies:\n",
      "  - R (>= 3.4) type: depends\n",
      "  - BiocParallel, type: imports\n",
      "  - graphics, type: imports\n",
      "  - utils, type: imports\n",
      "  - stats, type: imports\n",
      "  - AnnotationDbi, type: imports\n",
      "  - GO.db, type: imports\n",
      "  - org.Hs.eg.db, type: imports\n",
      "  - GGally, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - scales, type: imports\n",
      "  - cluster, type: imports\n",
      "  - WGCNA, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/MCbiclust.html\n",
      "\n",
      "Name: metabomxtr\n",
      "Description:\n",
      "The functions in this package return optimized parameter estimates and log likelihoods for mixture models of truncated data with normal or lognormal distributions.\n",
      "Version: 1.32.0\n",
      "Publication date: None\n",
      "Mantainer: Michael Nodzenski <michael.nodzenski@gmail.com>\n",
      "Authors:\n",
      "Author: Michael Nodzenski, Anna Reisetter, Denise Scholtens\n",
      "Dependencies:\n",
      "  - methods, type: depends\n",
      "  - Biobase, type: depends\n",
      "  - optimx, type: imports\n",
      "  - Formula, type: imports\n",
      "  - plyr, type: imports\n",
      "  - multtest, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - ggplot2, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/metabomxtr.html\n",
      "\n",
      "Name: MetID\n",
      "Description:\n",
      "This package uses an innovative network-based approach that will enhance our ability to determine the identities of significant ions detected by LC-MS.\n",
      "Version: 1.16.0\n",
      "Publication date: None\n",
      "Mantainer: Zhenzhi Li <zzrickli@gmail.com>\n",
      "Authors:\n",
      "Author: Zhenzhi Li <zzrickli at gmail.com>\n",
      "Dependencies:\n",
      "  - R (>= 3.5) type: depends\n",
      "  - utils (>= 3.3.1) type: imports\n",
      "  - stats (>= 3.4.2) type: imports\n",
      "  - devtools (>= 1.13.0) type: imports\n",
      "  - stringr (>= 1.3.0) type: imports\n",
      "  - Matrix (>= 1.2-12) type: imports\n",
      "  - igraph (>= 1.2.1) type: imports\n",
      "  - ChemmineR, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/MetID.html\n",
      "\n",
      "Name: miaSim\n",
      "Description:\n",
      "Microbiome time series simulation with generalized Lotka-Volterra model, Self-Organized Instability (SOI), and other models. Hubbell's Neutral model is used to determine the abundance matrix. The resulting abundance matrix is applied to SummarizedExperiment or TreeSummarizedExperiment objects.\n",
      "Version: 1.4.0\n",
      "Publication date: None\n",
      "Mantainer: Yagmur Simsek <yagmur.simsek@hsrw.org>\n",
      "Authors:\n",
      "Author: Karoline Faust [aut], Yu Gao [aut], Emma Gheysen [aut], Daniel Rios Garza [aut], Yagmur Simsek [cre, aut], Leo Lahti [aut]\n",
      "Dependencies:\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - TreeSummarizedExperiment, type: depends\n",
      "  - deSolve, type: imports\n",
      "  - stats, type: imports\n",
      "  - poweRlaw, type: imports\n",
      "  - gtools, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - MatrixGenerics, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0 | file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/miaSim.html\n",
      "\n",
      "Name: mpra\n",
      "Description:\n",
      "Tools for data management, count preprocessing, and differential analysis in massively parallel report assays (MPRA).\n",
      "Version: 1.20.0\n",
      "Publication date: None\n",
      "Mantainer: Leslie Myint <leslie.myint@gmail.com>\n",
      "Authors:\n",
      "Author: Leslie Myint [cre, aut], Kasper D. Hansen [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - BiocGenerics, type: depends\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - limma, type: depends\n",
      "  - S4Vectors, type: imports\n",
      "  - scales, type: imports\n",
      "  - stats, type: imports\n",
      "  - graphics, type: imports\n",
      "  - statmod, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/mpra.html\n",
      "\n",
      "Name: netDx\n",
      "Description:\n",
      "netDx is a general-purpose algorithm to build a patient classifier from heterogenous patient data. The method converts data into patient similarity networks at the level of features. Feature selection identifies features of predictive value to each class. Methods are provided for versatile predictor design and performance evaluation using standard measures. netDx natively groups molecular data into pathway-level features and connects with Cytoscape for network visualization of pathway themes. For method details see: Pai et al. (2019). netDx: interpretable patient classification using integrated patient similarity networks. Molecular Systems Biology. 15, e8497\n",
      "Version: 1.9.0\n",
      "Publication date: None\n",
      "Mantainer: Shraddha Pai <shraddha.pai@utoronto.ca>\n",
      "Authors:\n",
      "Author: Shraddha Pai [aut, cre] , Philipp Weber [aut], Ahmad Shah [aut], Luca Giudice [aut], Shirley Hui [aut], Anne NÃ¸hr [ctb], Indy Ng [ctb], Ruth Isserlin [aut], Hussam Kaka [aut], Gary Bader [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.6) type: depends\n",
      "  - ROCR, type: imports\n",
      "  - pracma, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - glmnet, type: imports\n",
      "  - igraph, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - parallel, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - MultiAssayExperiment, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - methods, type: imports\n",
      "  - BiocFileCache, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - bigmemory, type: imports\n",
      "  - doParallel, type: imports\n",
      "  - foreach, type: imports\n",
      "  - combinat, type: imports\n",
      "  - rappdirs, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - Rtsne, type: imports\n",
      "  - httr, type: imports\n",
      "  - plotrix, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/netDx.html\n",
      "\n",
      "Name: NetSAM\n",
      "Description:\n",
      "The NetSAM (Network Seriation and Modularization) package takes an edge-list representation of a weighted or unweighted network as an input, performs network seriation and modularization analysis, and generates as files that can be used as an input for the one-dimensional network visualization tool NetGestalt (http://www.netgestalt.org) or other network analysis. The NetSAM package can also generate correlation network (e.g. co-expression network) based on the input matrix data, perform seriation and modularization analysis for the correlation network and calculate the associations between the sample features and modules or identify the associated GO terms for the modules.\n",
      "Version: 1.38.0\n",
      "Publication date: None\n",
      "Mantainer: Zhiao Shi <zhiao.shi@gmail.com>\n",
      "Authors:\n",
      "Author: Jing Wang <jing.wang at bcm.edu>\n",
      "Dependencies:\n",
      "  - R (>= 3.0.0) type: depends\n",
      "  - seriation (>= 1.0-6) type: depends\n",
      "  - igraph (>= 0.6-1) type: depends\n",
      "  - tools (>= 3.0.0) type: depends\n",
      "  - WGCNA (>= 1.34.0) type: depends\n",
      "  - biomaRt, type: depends\n",
      "  - methods, type: imports\n",
      "  - AnnotationDbi, type: imports\n",
      "  - doParallel (>= 1.0.10) type: imports\n",
      "  - foreach (>= 1.4.0) type: imports\n",
      "  - survival (>= 2.37-7) type: imports\n",
      "  - GO.db, type: imports\n",
      "  - R2HTML (>= 2.2.0) type: imports\n",
      "  - DBI (>= 0.5-1) type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/NetSAM.html\n",
      "\n",
      "Name: NPARC\n",
      "Description:\n",
      "Perform non-parametric analysis of response curves as described by Childs, Bach, Franken et al. (2019): Non-parametric analysis of thermal proteome profiles reveals novel drug-binding proteins.\n",
      "Version: 1.10.0\n",
      "Publication date: None\n",
      "Mantainer: Nils Kurzawa <nilskurzawa@gmail.com>\n",
      "Authors:\n",
      "Author: Dorothee Childs, Nils Kurzawa\n",
      "Dependencies:\n",
      "  - R (>= 4.0.0) type: depends\n",
      "  - dplyr, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - broom, type: imports\n",
      "  - MASS, type: imports\n",
      "  - rlang, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - stats, type: imports\n",
      "  - methods, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/NPARC.html\n",
      "\n",
      "Name: omicsPrint\n",
      "Description:\n",
      "omicsPrint provides functionality for cross omic genetic fingerprinting, for example, to verify sample relationships between multiple omics data types, i.e. genomic, transcriptomic and epigenetic (DNA methylation).\n",
      "Version: 1.18.0\n",
      "Publication date: None\n",
      "Mantainer: Davy Cats <davycats.dc@gmail.com>\n",
      "Authors:\n",
      "Author: Maarten van Iterson [aut], Davy Cats [cre]\n",
      "Dependencies:\n",
      "  - R (>= 3.5) type: depends\n",
      "  - MASS, type: depends\n",
      "  - methods, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - MultiAssayExperiment, type: imports\n",
      "  - RaggedExperiment, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/omicsPrint.html\n",
      "\n",
      "Name: ompBAM\n",
      "Description:\n",
      "This packages provides C++ header files for developers wishing to create R packages that processes BAM files. ompBAM automates file access, memory management, and handling of multiple threads 'behind the scenes', so developers can focus on creating domain-specific functionality. The included vignette contains detailed documentation of this API, including quick-start instructions to create a new ompBAM-based package, and step-by-step explanation of the functionality behind the example packaged included within ompBAM.\n",
      "Version: 1.2.0\n",
      "Publication date: None\n",
      "Mantainer: Alex Chit Hei Wong <a.wong@centenary.org.au>\n",
      "Authors:\n",
      "Author: Alex Chit Hei Wong [aut, cre, cph]\n",
      "Dependencies:\n",
      "  - utils, type: imports\n",
      "  - Rcpp, type: imports\n",
      "  - zlibbioc, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ompBAM.html\n",
      "\n",
      "Name: paircompviz\n",
      "Description:\n",
      "This package provides visualization of the results from the multiple (i.e. pairwise) comparison tests such as pairwise.t.test, pairwise.prop.test or pairwise.wilcox.test. The groups being compared are visualized as nodes in Hasse diagram. Such approach enables very clear and vivid depiction of which group is significantly greater than which others, especially if comparing a large number of groups.\n",
      "Version: 1.36.0\n",
      "Publication date: None\n",
      "Mantainer: Michal Burda <michal.burda@osu.cz>\n",
      "Authors:\n",
      "Author: Michal Burda\n",
      "Dependencies:\n",
      "  - R (>= 2.10) type: depends\n",
      "  - Rgraphviz, type: depends\n",
      "  - Rgraphviz, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>=3.0)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/paircompviz.html\n",
      "\n",
      "Name: plotGrouper\n",
      "Description:\n",
      "A shiny app-based GUI wrapper for ggplot with built-in statistical analysis. Import data from file and use dropdown menus and checkboxes to specify the plotting variables, graph type, and look of your plots. Once created, plots can be saved independently or stored in a report that can be saved as a pdf. If new data are added to the file, the report can be refreshed to include new data. Statistical tests can be selected and added to the graphs. Analysis of flow cytometry data is especially integrated with plotGrouper. Count data can be transformed to return the absolute number of cells in a sample (this feature requires inclusion of the number of beads per sample and information about any dilution performed).\n",
      "Version: 1.16.0\n",
      "Publication date: None\n",
      "Mantainer: John D. Gagnon <john.gagnon.42@gmail.com>\n",
      "Authors:\n",
      "Author: John D. Gagnon [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 3.5) type: depends\n",
      "  - ggplot2 (>= 3.0.0) type: imports\n",
      "  - dplyr (>= 0.7.6) type: imports\n",
      "  - tidyr (>= 0.2.0) type: imports\n",
      "  - tibble (>= 1.4.2) type: imports\n",
      "  - stringr (>= 1.3.1) type: imports\n",
      "  - readr (>= 1.1.1) type: imports\n",
      "  - readxl (>= 1.1.0) type: imports\n",
      "  - scales (>= 1.0.0) type: imports\n",
      "  - stats, type: imports\n",
      "  - grid, type: imports\n",
      "  - gridExtra (>= 2.3) type: imports\n",
      "  - egg (>= 0.4.0) type: imports\n",
      "  - gtable (>= 0.2.0) type: imports\n",
      "  - ggpubr (>= 0.1.8) type: imports\n",
      "  - shiny (>= 1.1.0) type: imports\n",
      "  - shinythemes (>= 1.1.1) type: imports\n",
      "  - colourpicker (>= 1.0) type: imports\n",
      "  - magrittr (>= 1.5) type: imports\n",
      "  - Hmisc (>= 4.1.1) type: imports\n",
      "  - rlang (>= 0.2.2) type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/plotGrouper.html\n",
      "\n",
      "Name: QSutils\n",
      "Description:\n",
      "Set of utility functions for viral quasispecies analysis with NGS data. Most functions are equally useful for metagenomic studies. There are three main types: (1) data manipulation and explorationâfunctions useful for converting reads to haplotypes and frequencies, repairing reads, intersecting strand haplotypes, and visualizing haplotype alignments. (2) diversity indicesâfunctions to compute diversity and entropy, in which incidence, abundance, and functional indices are considered. (3) data simulationâfunctions useful for generating random viral quasispecies data.\n",
      "Version: 1.16.0\n",
      "Publication date: None\n",
      "Mantainer: Mercedes Guerrero-Murillo <mergumu@gmail.com>\n",
      "Authors:\n",
      "Author: Mercedes Guerrero-Murillo [cre, aut] , Josep Gregori i Font [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.5) type: depends\n",
      "  - Biostrings, type: depends\n",
      "  - BiocGenerics, type: depends\n",
      "  - methods, type: depends\n",
      "  - ape, type: imports\n",
      "  - stats, type: imports\n",
      "  - psych, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/QSutils.html\n",
      "\n",
      "Name: Rcwl\n",
      "Description:\n",
      "The Common Workflow Language (CWL) is an open standard for development of data analysis workflows that is portable and scalable across different tools and working environments. Rcwl provides a simple way to wrap command line tools and build CWL data analysis pipelines programmatically within R. It increases the ease of usage, development, and maintenance of CWL pipelines.\n",
      "Version: 1.14.0\n",
      "Publication date: None\n",
      "Mantainer: Qiang Hu <qiang.hu@roswellpark.org>\n",
      "Authors:\n",
      "Author: Qiang Hu [aut, cre], Qian Liu [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.6) type: depends\n",
      "  - yaml, type: depends\n",
      "  - methods, type: depends\n",
      "  - S4Vectors, type: depends\n",
      "  - utils, type: imports\n",
      "  - stats, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - batchtools, type: imports\n",
      "  - DiagrammeR, type: imports\n",
      "  - shiny, type: imports\n",
      "  - R.utils, type: imports\n",
      "  - codetools, type: imports\n",
      "  - basilisk, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2 | file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/Rcwl.html\n",
      "\n",
      "Name: RiboDiPA\n",
      "Description:\n",
      "This package performs differential pattern analysis for Ribo-seq data. It identifies genes with significantly different patterns in the ribosome footprint between two conditions. RiboDiPA contains five major components including bam file processing, P-site mapping, data binning, differential pattern analysis and footprint visualization.\n",
      "Version: 1.6.0\n",
      "Publication date: None\n",
      "Mantainer: Ji-Ping Wang <jzwang@northwestern.edu>\n",
      "Authors:\n",
      "Author: Keren Li [aut], Matt Hope [aut], Xiaozhong Wang [aut], Ji-Ping Wang [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - Rsamtools, type: depends\n",
      "  - GenomicFeatures, type: depends\n",
      "  - GenomicAlignments, type: depends\n",
      "  - Rcpp (>= 1.0.2) type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - data.table, type: imports\n",
      "  - elitism, type: imports\n",
      "  - methods, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - reldist, type: imports\n",
      "  - doParallel, type: imports\n",
      "  - foreach, type: imports\n",
      "  - parallel, type: imports\n",
      "  - qvalue, type: imports\n",
      "  - DESeq2, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - BiocFileCache, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL (>= 3)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/RiboDiPA.html\n",
      "\n",
      "Name: RLassoCox\n",
      "Description:\n",
      "RLassoCox is a package that implements the RLasso-Cox model proposed by Wei Liu. The RLasso-Cox model integrates gene interaction information into the Lasso-Cox model for accurate survival prediction and survival biomarker discovery. It is based on the hypothesis that topologically important genes in the gene interaction network tend to have stable expression changes. The RLasso-Cox model uses random walk to evaluate the topological weight of genes, and then highlights topologically important genes to improve the generalization ability of the Lasso-Cox model. The RLasso-Cox model has the advantage of identifying small gene sets with high prognostic performance on independent datasets, which may play an important role in identifying robust survival biomarkers for various cancer types.\n",
      "Version: 1.6.0\n",
      "Publication date: None\n",
      "Mantainer: Wei Liu <freelw@qq.com>\n",
      "Authors:\n",
      "Author: Wei Liu [cre, aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - glmnet, type: depends\n",
      "  - Matrix, type: imports\n",
      "  - igraph, type: imports\n",
      "  - survival, type: imports\n",
      "  - stats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/RLassoCox.html\n",
      "\n",
      "Name: RNAinteract\n",
      "Description:\n",
      "RNAinteract estimates genetic interactions from multi-dimensional read-outs like features extracted from images. The screen is assumed to be performed in multi-well plates or similar designs. Starting from a list of features (e.g. cell number, area, fluorescence intensity) per well, genetic interactions are estimated. The packages provides functions for reporting interacting gene pairs, plotting heatmaps and double RNAi plots. An HTML report can be written for quality control and analysis.\n",
      "Version: 1.46.0\n",
      "Publication date: None\n",
      "Mantainer: Mike Smith <mike.smith@embl.de>\n",
      "Authors:\n",
      "Author: Bernd Fischer [aut], Wolfgang Huber [ctb], Mike Smith [cre]\n",
      "Dependencies:\n",
      "  - R (>= 2.12.0) type: depends\n",
      "  - RColorBrewer, type: imports\n",
      "  - ICS, type: imports\n",
      "  - ICSNP, type: imports\n",
      "  - cellHTS2, type: imports\n",
      "  - geneplotter, type: imports\n",
      "  - gplots, type: imports\n",
      "  - grid, type: imports\n",
      "  - hwriter, type: imports\n",
      "  - lattice, type: imports\n",
      "  - latticeExtra, type: imports\n",
      "  - limma, type: imports\n",
      "  - methods, type: imports\n",
      "  - splots, type: imports\n",
      "  - abind, type: imports\n",
      "  - locfit, type: imports\n",
      "  - Biobase, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/RNAinteract.html\n",
      "\n",
      "Name: RTopper\n",
      "Description:\n",
      "the RTopper package is designed to perform and integrate gene set enrichment results across multiple genomic platforms.\n",
      "Version: 1.44.1\n",
      "Publication date: None\n",
      "Mantainer: Luigi Marchionni <marchion@jhu.edu>\n",
      "Authors:\n",
      "Author: Luigi Marchionni <marchion at jhu.edu>, Svitlana Tyekucheva <svitlana at jimmy.harvard.edu>\n",
      "Dependencies:\n",
      "  - R (>= 2.12.0) type: depends\n",
      "  - Biobase, type: depends\n",
      "  - limma, type: imports\n",
      "  - multtest, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 3) + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/RTopper.html\n",
      "\n",
      "Name: SANTA\n",
      "Description:\n",
      "Bioconductor version: Release (3.16)\n",
      "Version: 2.34.0\n",
      "Publication date: None\n",
      "Mantainer: Author: Alex Cornish [cre, aut]\n",
      "Authors:\n",
      "This package provides methods for measuring the strength of association between a network and a phenotype. It does this by measuring clustering of the phenotype across the network (Knet). Vertices can also be individually ranked by their strength of association with high-weight vertices (Knode).\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - igraph, type: depends\n",
      "  - graphics, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/SANTA.html\n",
      "\n",
      "Name: scGPS\n",
      "Description:\n",
      "The package implements two main algorithms to answer two key questions: a SCORE (Stable Clustering at Optimal REsolution) to find subpopulations, followed by scGPS to investigate the relationships between subpopulations.\n",
      "Version: 1.12.2\n",
      "Publication date: None\n",
      "Mantainer: Quan Nguyen <quan.nguyen@uq.edu.au>\n",
      "Authors:\n",
      "Author: Quan Nguyen [aut, cre], Michael Thompson [aut], Anne Senabouth [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.6) type: depends\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - dynamicTreeCut, type: depends\n",
      "  - SingleCellExperiment, type: depends\n",
      "  - glmnet (> 2.0) type: imports\n",
      "  - caret (>= 6.0) type: imports\n",
      "  - ggplot2 (>= 2.2.1) type: imports\n",
      "  - fastcluster, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - Rcpp, type: imports\n",
      "  - RcppArmadillo, type: imports\n",
      "  - RcppParallel, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - DESeq2, type: imports\n",
      "  - locfit, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/scGPS.html\n",
      "\n",
      "Name: scTGIF\n",
      "Description:\n",
      "scTGIF connects the cells and the related gene functions without cell type label.\n",
      "Version: 1.12.0\n",
      "Publication date: None\n",
      "Mantainer: Koki Tsuyuzaki <k.t.the-answer@hotmail.co.jp>\n",
      "Authors:\n",
      "Author: Koki Tsuyuzaki [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 3.6.0) type: depends\n",
      "  - GSEABase, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - SingleCellExperiment, type: imports\n",
      "  - BiocStyle, type: imports\n",
      "  - plotly, type: imports\n",
      "  - tagcloud, type: imports\n",
      "  - rmarkdown, type: imports\n",
      "  - Rcpp, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - utils, type: imports\n",
      "  - knitr, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - nnTensor, type: imports\n",
      "  - methods, type: imports\n",
      "  - scales, type: imports\n",
      "  - msigdbr, type: imports\n",
      "  - schex, type: imports\n",
      "  - tibble, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - igraph, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/scTGIF.html\n",
      "\n",
      "Name: seq2pathway\n",
      "Description:\n",
      "Seq2pathway is a novel tool for functional gene-set (or termed as pathway) analysis of next-generation sequencing data, consisting of \"seq2gene\" and \"gene2path\" components. The seq2gene links sequence-level measurements of genomic regions (including SNPs or point mutation coordinates) to gene-level scores, and the gene2pathway summarizes gene scores to pathway-scores for each sample. The seq2gene has the feasibility to assign both coding and non-exon regions to a broader range of neighboring genes than only the nearest one, thus facilitating the study of functional non-coding regions. The gene2pathway takes into account the quantity of significance for gene members within a pathway compared those outside a pathway. The output of seq2pathway is a general structure of quantitative pathway-level scores, thus allowing one to functional interpret such datasets as RNA-seq, ChIP-seq, GWAS, and derived from other next generational sequencing experiments.\n",
      "Version: 1.30.0\n",
      "Publication date: None\n",
      "Mantainer: Arjun Kinstlick <akinstlick@uchicago.edu>\n",
      "Authors:\n",
      "Author: Xinan Yang <xyang2 at uchicago.edu>; Bin Wang <binw at uchicago.edu>\n",
      "Dependencies:\n",
      "  - R (>= 3.6.2) type: depends\n",
      "  - nnet, type: imports\n",
      "  - WGCNA, type: imports\n",
      "  - GSA, type: imports\n",
      "  - biomaRt, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - seq2pathway.data, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/seq2pathway.html\n",
      "\n",
      "Name: spatialHeatmap\n",
      "Description:\n",
      "The spatialHeatmap package provides functionalities for visualizing cell-, tissue- and organ-specific data of biological assays by coloring the corresponding spatial features defined in anatomical images according to a numeric color key.\n",
      "Version: 2.4.0\n",
      "Publication date: None\n",
      "Mantainer: Jianhai Zhang <jzhan067@ucr.edu>\n",
      "Authors:\n",
      "Author: Jianhai Zhang [aut, trl, cre], Jordan Hayes [aut], Le Zhang [aut], Bing Yang [aut], Wolf Frommer [aut], Julia Bailey-Serres [aut], Thomas Girke [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - BiocParallel, type: imports\n",
      "  - BiocFileCache, type: imports\n",
      "  - data.table, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - edgeR, type: imports\n",
      "  - WGCNA, type: imports\n",
      "  - flashClust, type: imports\n",
      "  - htmlwidgets, type: imports\n",
      "  - genefilter, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - ggdendro, type: imports\n",
      "  - grImport, type: imports\n",
      "  - grid, type: imports\n",
      "  - gridExtra, type: imports\n",
      "  - gplots, type: imports\n",
      "  - igraph, type: imports\n",
      "  - limma, type: imports\n",
      "  - methods, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - rsvg, type: imports\n",
      "  - shiny, type: imports\n",
      "  - dynamicTreeCut, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - ggplotify, type: imports\n",
      "  - parallel, type: imports\n",
      "  - plotly, type: imports\n",
      "  - rappdirs, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - scater, type: imports\n",
      "  - scuttle, type: imports\n",
      "  - scran, type: imports\n",
      "  - stats, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - SingleCellExperiment, type: imports\n",
      "  - shinydashboard, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - tibble, type: imports\n",
      "  - utils, type: imports\n",
      "  - visNetwork, type: imports\n",
      "  - UpSetR, type: imports\n",
      "  - xml2, type: imports\n",
      "  - yaml, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/spatialHeatmap.html\n",
      "\n",
      "Name: ssviz\n",
      "Description:\n",
      "Small RNA sequencing viewer\n",
      "Version: 1.32.0\n",
      "Publication date: None\n",
      "Mantainer: Diana Low <lowdiana@gmail.com>\n",
      "Authors:\n",
      "Author: Diana Low\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - Rsamtools, type: depends\n",
      "  - Biostrings, type: depends\n",
      "  - reshape, type: depends\n",
      "  - ggplot2, type: depends\n",
      "  - RColorBrewer, type: depends\n",
      "  - stats, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ssviz.html\n",
      "\n",
      "Name: SubCellBarCode\n",
      "Description:\n",
      "Mass-Spectrometry based spatial proteomics have enabled the proteome-wide mapping of protein subcellular localization (Orre et al. 2019, Molecular Cell). SubCellBarCode R package robustly classifies proteins into corresponding subcellular localization.\n",
      "Version: 1.14.0\n",
      "Publication date: None\n",
      "Mantainer: Taner Arslan <taner.arslan@ki.se>\n",
      "Authors:\n",
      "Author: Taner Arslan\n",
      "Dependencies:\n",
      "  - R (>= 3.6) type: depends\n",
      "  - Rtsne, type: imports\n",
      "  - scatterplot3d, type: imports\n",
      "  - caret, type: imports\n",
      "  - e1071, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - gridExtra, type: imports\n",
      "  - networkD3, type: imports\n",
      "  - ggrepel, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - org.Hs.eg.db, type: imports\n",
      "  - AnnotationDbi, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/SubCellBarCode.html\n",
      "\n",
      "Name: swfdr\n",
      "Description:\n",
      "This package allows users to estimate the science-wise false discovery rate from Jager and Leek, \"Empirical estimates suggest most published medical research is true,\" 2013, Biostatistics, using an EM approach due to the presence of rounding and censoring. It also allows users to estimate the false discovery rate conditional on covariates, using a regression framework, as per Boca and Leek, \"A direct approach to estimating false discovery rates conditional on covariates,\" 2018, PeerJ.\n",
      "Version: 1.24.0\n",
      "Publication date: None\n",
      "Mantainer: Simina M. Boca <smb310@georgetown.edu>, Jeffrey T. Leek <jtleek@gmail.com>\n",
      "Authors:\n",
      "Author: Jeffrey T. Leek, Leah Jager, Simina M. Boca, Tomasz Konopka\n",
      "Dependencies:\n",
      "  - R (>= 3.4) type: depends\n",
      "  - methods, type: imports\n",
      "  - splines, type: imports\n",
      "  - stats4, type: imports\n",
      "  - stats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 3)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/swfdr.html\n",
      "\n",
      "Name: switchde\n",
      "Description:\n",
      "Inference and detection of switch-like differential expression across single-cell RNA-seq trajectories.\n",
      "Version: 1.24.0\n",
      "Publication date: None\n",
      "Mantainer: Kieran Campbell <kieranrcampbell@gmail.com>\n",
      "Authors:\n",
      "Author: Kieran Campbell [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 3.4) type: depends\n",
      "  - SingleCellExperiment, type: depends\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/switchde.html\n",
      "\n",
      "Name: TFutils\n",
      "Description:\n",
      "This package helps users to work with TF metadata from various sources. Significant catalogs of TFs and classifications thereof are made available. Tools for working with motif scans are also provided.\n",
      "Version: 1.18.0\n",
      "Publication date: None\n",
      "Mantainer: Vincent Carey <stvjc@channing.harvard.edu>\n",
      "Authors:\n",
      "Author: Vincent Carey [aut, cre], Shweta Gopaulakrishnan [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.1.0) type: depends\n",
      "  - methods, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - miniUI, type: imports\n",
      "  - shiny, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - GSEABase, type: imports\n",
      "  - rjson, type: imports\n",
      "  - BiocFileCache, type: imports\n",
      "  - DT, type: imports\n",
      "  - httr, type: imports\n",
      "  - readxl, type: imports\n",
      "  - AnnotationDbi, type: imports\n",
      "  - org.Hs.eg.db, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/TFutils.html\n",
      "\n",
      "Name: tigre\n",
      "Description:\n",
      "The tigre package implements our methodology of Gaussian process differential equation models for analysis of gene expression time series from single input motif networks. The package can be used for inferring unobserved transcription factor (TF) protein concentrations from expression measurements of known target genes, or for ranking candidate targets of a TF.\n",
      "Version: 1.52.0\n",
      "Publication date: None\n",
      "Mantainer: Antti Honkela <antti.honkela@helsinki.fi>\n",
      "Authors:\n",
      "Author: Antti Honkela, Pei Gao, Jonatan Ropponen, Miika-Petteri Matikainen, Magnus Rattray, Neil D. Lawrence\n",
      "Dependencies:\n",
      "  - R (>= 2.11.0) type: depends\n",
      "  - BiocGenerics, type: depends\n",
      "  - Biobase, type: depends\n",
      "  - methods, type: imports\n",
      "  - AnnotationDbi, type: imports\n",
      "  - gplots, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - annotate, type: imports\n",
      "  - DBI, type: imports\n",
      "  - RSQLite, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "AGPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/tigre.html\n",
      "\n",
      "Name: TPP2D\n",
      "Description:\n",
      "Detection of ligand-protein interactions from 2D thermal profiles (DLPTP), Performs an FDR-controlled analysis of 2D-TPP experiments by functional analysis of dose-response curves across temperatures.\n",
      "Version: 1.14.0\n",
      "Publication date: None\n",
      "Mantainer: Nils Kurzawa <nilskurzawa@gmail.com>\n",
      "Authors:\n",
      "Author: Nils Kurzawa [aut, cre], Holger Franken [aut], Simon Anders [aut], Wolfgang Huber [aut], Mikhail M. Savitski [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.6.0) type: depends\n",
      "  - stats, type: depends\n",
      "  - utils, type: depends\n",
      "  - dplyr, type: depends\n",
      "  - methods, type: depends\n",
      "  - ggplot2, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - foreach, type: imports\n",
      "  - doParallel, type: imports\n",
      "  - openxlsx, type: imports\n",
      "  - stringr, type: imports\n",
      "  - RCurl, type: imports\n",
      "  - parallel, type: imports\n",
      "  - MASS, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - limma, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/TPP2D.html\n",
      "\n",
      "Name: UNDO\n",
      "Description:\n",
      "UNDO is an R package for unsupervised deconvolution of tumor and stromal mixed expression data. It detects marker genes and deconvolutes the mixing expression data without any prior knowledge.\n",
      "Version: 1.40.0\n",
      "Publication date: None\n",
      "Mantainer: Niya Wang <wangny@vt.edu>\n",
      "Authors:\n",
      "Author: Niya Wang <wangny at vt.edu>\n",
      "Dependencies:\n",
      "  - R (>= 2.15.2) type: depends\n",
      "  - methods, type: depends\n",
      "  - BiocGenerics, type: depends\n",
      "  - Biobase, type: depends\n",
      "  - MASS, type: imports\n",
      "  - boot, type: imports\n",
      "  - nnls, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/UNDO.html\n",
      "\n",
      "Name: ASURAT\n",
      "Description:\n",
      "ASURAT is a software for single-cell data analysis. Using ASURAT, one can simultaneously perform unsupervised clustering and biological interpretation in terms of cell type, disease, biological process, and signaling pathway activity. Inputting a single-cell RNA-seq data and knowledge-based databases, such as Cell Ontology, Gene Ontology, KEGG, etc., ASURAT transforms gene expression tables into original multivariate tables, termed sign-by-sample matrices (SSMs).\n",
      "Version: 1.2.0\n",
      "Publication date: None\n",
      "Mantainer: Keita Iida <kiida@protein.osaka-u.ac.jp>\n",
      "Authors:\n",
      "Author: Keita Iida [aut, cre] , Johannes Nicolaus Wibisana [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 4.0.0) type: depends\n",
      "  - SingleCellExperiment, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - Rcpp (>= 1.0.7) type: imports\n",
      "  - cluster, type: imports\n",
      "  - utils, type: imports\n",
      "  - plot3D, type: imports\n",
      "  - ComplexHeatmap, type: imports\n",
      "  - circlize, type: imports\n",
      "  - grid, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3 + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ASURAT.html\n",
      "\n",
      "Name: awst\n",
      "Description:\n",
      "We propose an Asymmetric Within-Sample Transformation (AWST) to regularize RNA-seq read counts and reduce the effect of noise on the classification of samples. AWST comprises two main steps: standardization and smoothing. These steps transform gene expression data to reduce the noise of the lowly expressed features, which suffer from background effects and low signal-to-noise ratio, and the influence of the highly expressed features, which may be the result of amplification bias and other experimental artifacts.\n",
      "Version: 1.6.0\n",
      "Publication date: None\n",
      "Mantainer: Davide Risso <risso.davide@gmail.com>\n",
      "Authors:\n",
      "Author: Davide Risso [aut, cre, cph] , Stefano Pagnotta [aut, cph]\n",
      "Dependencies:\n",
      "  - stats, type: imports\n",
      "  - methods, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/awst.html\n",
      "\n",
      "Name: BBCAnalyzer\n",
      "Description:\n",
      "BBCAnalyzer is a package for visualizing the relative or absolute number of bases, deletions and insertions at defined positions in sequence alignment data available as bam files in comparison to the reference bases. Markers for the relative base frequencies, the mean quality of the detected bases, known mutations or polymorphisms and variants called in the data may additionally be included in the plots.\n",
      "Version: 1.28.0\n",
      "Publication date: None\n",
      "Mantainer: Sarah Sandmann <sarah.sandmann@uni-muenster.de>\n",
      "Authors:\n",
      "Author: Sarah Sandmann\n",
      "Dependencies:\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - VariantAnnotation, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - Biostrings, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/BBCAnalyzer.html\n",
      "\n",
      "Name: cellscape\n",
      "Description:\n",
      "CellScape facilitates interactive browsing of single cell clonal evolution datasets. The tool requires two main inputs: (i) the genomic content of each single cell in the form of either copy number segments or targeted mutation values, and (ii) a single cell phylogeny. Phylogenetic formats can vary from dendrogram-like phylogenies with leaf nodes to evolutionary model-derived phylogenies with observed or latent internal nodes. The CellScape phylogeny is flexibly input as a table of source-target edges to support arbitrary representations, where each node may or may not have associated genomic data. The output of CellScape is an interactive interface displaying a single cell phylogeny and a cell-by-locus genomic heatmap representing the mutation status in each cell for each locus.\n",
      "Version: 1.22.0\n",
      "Publication date: None\n",
      "Mantainer: Maia Smith <maiaannesmith@gmail.com>\n",
      "Authors:\n",
      "Author: Maia Smith [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 3.3) type: depends\n",
      "  - htmlwidgets (>= 0.5) type: imports\n",
      "  - jsonlite (>= 0.9.19) type: imports\n",
      "  - reshape2 (>= 1.4.1) type: imports\n",
      "  - stringr (>= 1.0.0) type: imports\n",
      "  - plyr (>= 1.8.3) type: imports\n",
      "  - dplyr (>= 0.4.3) type: imports\n",
      "  - gtools (>= 3.5.0) type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/cellscape.html\n",
      "\n",
      "Name: ChIC\n",
      "Description:\n",
      "Quality control (QC) pipeline for ChIP-seq data using a comprehensive set of QC metrics, including previously proposed metrics as well as novel ones, based on local characteristics of the enrichment profile. The package provides functions to calculate a set of QC metrics, a compendium with reference values and machine learning models to score sample quality.\n",
      "Version: 1.18.0\n",
      "Publication date: None\n",
      "Mantainer: Carmen Maria Livi <carmen.livi@external.ifom.eu>\n",
      "Authors:\n",
      "Author: Carmen Maria Livi\n",
      "Dependencies:\n",
      "  - spp, type: depends\n",
      "  - R (>= 3.6) type: depends\n",
      "  - ChIC.data, type: imports\n",
      "  - caTools, type: imports\n",
      "  - methods, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - parallel, type: imports\n",
      "  - progress, type: imports\n",
      "  - randomForest, type: imports\n",
      "  - caret, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - graphics, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - genomeIntervals, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ChIC.html\n",
      "\n",
      "Name: CONFESS\n",
      "Description:\n",
      "Single Cell Fluidigm Spot Detector.\n",
      "Version: 1.26.0\n",
      "Publication date: None\n",
      "Mantainer: Diana LOW <lowdiana@gmail.com>\n",
      "Authors:\n",
      "Author: Diana LOW and Efthimios MOTAKIS\n",
      "Dependencies:\n",
      "  - R (>= 3.3) type: depends\n",
      "  - grDevices, type: depends\n",
      "  - utils, type: depends\n",
      "  - stats, type: depends\n",
      "  - graphics, type: depends\n",
      "  - methods, type: imports\n",
      "  - changepoint, type: imports\n",
      "  - cluster, type: imports\n",
      "  - contrast, type: imports\n",
      "  - data.table (>= 1.9.7) type: imports\n",
      "  - ecp, type: imports\n",
      "  - EBImage, type: imports\n",
      "  - flexmix, type: imports\n",
      "  - flowCore, type: imports\n",
      "  - flowClust, type: imports\n",
      "  - flowMeans, type: imports\n",
      "  - flowMerge, type: imports\n",
      "  - flowPeaks, type: imports\n",
      "  - foreach, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - grid, type: imports\n",
      "  - limma, type: imports\n",
      "  - MASS, type: imports\n",
      "  - moments, type: imports\n",
      "  - outliers, type: imports\n",
      "  - parallel, type: imports\n",
      "  - plotrix, type: imports\n",
      "  - raster, type: imports\n",
      "  - readbitmap, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - SamSPECTRAL, type: imports\n",
      "  - waveslim, type: imports\n",
      "  - wavethresh, type: imports\n",
      "  - zoo, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/CONFESS.html\n",
      "\n",
      "Name: CONSTANd\n",
      "Description:\n",
      "Normalizes a data matrix `data` by raking (using the RAS method by Bacharach, see references) the Nrows by Ncols matrix such that the row means and column means equal 1. The result is a normalized data matrix `K=RAS`, a product of row mulipliers `R` and column multipliers `S` with the original matrix `A`. Missing information needs to be presented as `NA` values and not as zero values, because CONSTANd is able to ignore missing values when calculating the mean. Using CONSTANd normalization allows for the direct comparison of values between samples within the same and even across different CONSTANd-normalized data matrices.\n",
      "Version: 1.6.0\n",
      "Publication date: None\n",
      "Mantainer: Dirk Valkenborg <dirk.valkenborg@uhasselt.be>\n",
      "Authors:\n",
      "Author: Joris Van Houtven [aut, trl], Geert Jan Bex [trl], Dirk Valkenborg [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/CONSTANd.html\n",
      "\n",
      "Name: Cormotif\n",
      "Description:\n",
      "It fits correlation motif model to multiple studies to detect study specific differential expression patterns.\n",
      "Version: 1.44.0\n",
      "Publication date: None\n",
      "Mantainer: Yingying Wei <ywei@jhsph.edu>\n",
      "Authors:\n",
      "Author: Hongkai Ji, Yingying Wei\n",
      "Dependencies:\n",
      "  - R (>= 2.12.0) type: depends\n",
      "  - affy, type: depends\n",
      "  - limma, type: depends\n",
      "  - affy, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/Cormotif.html\n",
      "\n",
      "Name: customCMPdb\n",
      "Description:\n",
      "This package serves as a query interface for important community collections of small molecules, while also allowing users to include custom compound collections.\n",
      "Version: 1.8.0\n",
      "Publication date: None\n",
      "Mantainer: Yuzhu Duan <yduan004@ucr.edu>\n",
      "Authors:\n",
      "Author: Yuzhu Duan [aut, cre], Thomas Girke [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - AnnotationHub, type: imports\n",
      "  - RSQLite, type: imports\n",
      "  - XML, type: imports\n",
      "  - utils, type: imports\n",
      "  - ChemmineR, type: imports\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - rappdirs, type: imports\n",
      "  - BiocFileCache, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/customCMPdb.html\n",
      "\n",
      "Name: DAMEfinder\n",
      "Description:\n",
      "'DAMEfinder' offers functionality for taking methtuple or bismark outputs to calculate ASM scores and compute DAMEs. It also offers nice visualization of methyl-circle plots.\n",
      "Version: 1.10.1\n",
      "Publication date: None\n",
      "Mantainer: Stephany Orjuela <sorjuelal@gmail.com>\n",
      "Authors:\n",
      "Author: Stephany Orjuela [aut, cre] , Dania Machlab [aut], Mark Robinson [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - stats, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - readr, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - GenomicAlignments, type: imports\n",
      "  - stringr, type: imports\n",
      "  - plyr, type: imports\n",
      "  - VariantAnnotation, type: imports\n",
      "  - parallel, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - methods, type: imports\n",
      "  - limma, type: imports\n",
      "  - bumphunter, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - cowplot, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/DAMEfinder.html\n",
      "\n",
      "Name: dcGSA\n",
      "Description:\n",
      "Distance-correlation based Gene Set Analysis for longitudinal gene expression profiles. In longitudinal studies, the gene expression profiles were collected at each visit from each subject and hence there are multiple measurements of the gene expression profiles for each subject. The dcGSA package could be used to assess the associations between gene sets and clinical outcomes of interest by fully taking advantage of the longitudinal nature of both the gene expression profiles and clinical outcomes.\n",
      "Version: 1.26.0\n",
      "Publication date: None\n",
      "Mantainer: Jiehuan sun <jiehuan.sun@yale.edu>\n",
      "Authors:\n",
      "Author: Jiehuan Sun [aut, cre], Jose Herazo-Maya [aut], Xiu Huang [aut], Naftali Kaminski [aut], and Hongyu Zhao [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.3) type: depends\n",
      "  - Matrix, type: depends\n",
      "  - BiocParallel, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/dcGSA.html\n",
      "\n",
      "Name: DEComplexDisease\n",
      "Description:\n",
      "Bioconductor version: Release (3.16)\n",
      "Version: 1.18.0\n",
      "Publication date: None\n",
      "Mantainer: Author: Guofeng Meng\n",
      "Authors:\n",
      "It is designed to find the differential expressed genes (DEGs) for complex disease, which is characterized by the heterogeneous genomic expression profiles. Different from the established DEG analysis tools, it does not assume the patients of complex diseases to share the common DEGs. By applying a bi-clustering algorithm, DECD finds the DEGs shared by as many patients. In this way, DECD describes the DEGs of complex disease in a novel syntax, e.g. a gene list composed of 200 genes are differentially expressed in 30% percent of studied complex disease. Applying the DECD analysis results, users are possible to find the patients affected by the same mechanism based on the shared signatures.\n",
      "Dependencies:\n",
      "  - R (>= 3.3.3) type: depends\n",
      "  - Rcpp (>= 0.12.7) type: imports\n",
      "  - DESeq2, type: imports\n",
      "  - edgeR, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - ComplexHeatmap, type: imports\n",
      "  - grid, type: imports\n",
      "  - parallel, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - methods, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/DEComplexDisease.html\n",
      "\n",
      "Name: deconvR\n",
      "Description:\n",
      "This package provides a collection of functions designed for analyzing deconvolution of the bulk sample(s) using an atlas of reference omic signature profiles and a user-selected model. Users are given the option to create or extend a reference atlas and,also simulate the desired size of the bulk signature profile of the reference cell types.The package includes the cell-type-specific methylation atlas and, Illumina Epic B5 probe ids that can be used in deconvolution. Additionally,we included BSmeth2Probe, to make mapping WGBS data to their probe IDs easier.\n",
      "Version: 1.4.3\n",
      "Publication date: None\n",
      "Mantainer: Irem B. GÃ¼ndÃ¼z <irembgunduz@gmail.com>\n",
      "Authors:\n",
      "Author: Irem B. GÃ¼ndÃ¼z [aut, cre] , Veronika Ebenal [aut] , Altuna Akalin [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - data.table (>= 1.14.0) type: depends\n",
      "  - S4Vectors, type: imports\n",
      "  - methylKit, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - stats, type: imports\n",
      "  - methods, type: imports\n",
      "  - foreach (>= 1.5.1) type: imports\n",
      "  - magrittr (>= 2.0.1) type: imports\n",
      "  - matrixStats (>= 0.61.0) type: imports\n",
      "  - e1071 (>= 1.7.9) type: imports\n",
      "  - quadprog (>= 1.5.8) type: imports\n",
      "  - nnls (>= 1.4) type: imports\n",
      "  - rsq (>= 2.2) type: imports\n",
      "  - MASS, type: imports\n",
      "  - utils, type: imports\n",
      "  - dplyr (>= 1.0.7) type: imports\n",
      "  - tidyr (>= 1.1.3) type: imports\n",
      "  - assertthat, type: imports\n",
      "  - minfi, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/deconvR.html\n",
      "\n",
      "Name: DeMAND\n",
      "Description:\n",
      "DEMAND predicts Drug MoA by interrogating a cell context specific regulatory network with a small number (N >= 6) of compound-induced gene expression signatures, to elucidate specific proteins whose interactions in the network is dysregulated by the compound.\n",
      "Version: 1.28.0\n",
      "Publication date: None\n",
      "Mantainer: Jung Hoon Woo <jw2853@columbia.edu>, Mariano Alvarez <reef103@gmail.com>\n",
      "Authors:\n",
      "Author: Jung Hoon Woo <jw2853 at columbia.edu>, Yishai Shimoni <ys2559 at columbia.edu>\n",
      "Dependencies:\n",
      "  - R (>= 2.14.0) type: depends\n",
      "  - KernSmooth, type: depends\n",
      "  - methods, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/DeMAND.html\n",
      "\n",
      "Name: DFP\n",
      "Description:\n",
      "This package provides a supervised technique able to identify differentially expressed genes, based on the construction of \\emph{Fuzzy Patterns} (FPs). The Fuzzy Patterns are built by means of applying 3 Membership Functions to discretized gene expression values.\n",
      "Version: 1.56.0\n",
      "Publication date: None\n",
      "Mantainer: Rodrigo Alvarez-Glez <rodrigo.djv@uvigo.es>\n",
      "Authors:\n",
      "Author: R. Alvarez-Gonzalez, D. Glez-Pena, F. Diaz, F. Fdez-Riverola\n",
      "Dependencies:\n",
      "  - methods, type: depends\n",
      "  - Biobase, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/DFP.html\n",
      "\n",
      "Name: Dino\n",
      "Description:\n",
      "Dino normalizes single-cell, mRNA sequencing data to correct for technical variation, particularly sequencing depth, prior to downstream analysis. The approach produces a matrix of corrected expression for which the dependency between sequencing depth and the full distribution of normalized expression; many existing methods aim to remove only the dependency between sequencing depth and the mean of the normalized expression. This is particuarly useful in the context of highly sparse datasets such as those produced by 10X genomics and other uninque molecular identifier (UMI) based microfluidics protocols for which the depth-dependent proportion of zeros in the raw expression data can otherwise present a challenge.\n",
      "Version: 1.4.0\n",
      "Publication date: None\n",
      "Mantainer: Jared Brown <brownj@ds.dfci.harvard.edu>\n",
      "Authors:\n",
      "Author: Jared Brown [aut, cre] , Christina Kendziorski [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 4.0.0) type: depends\n",
      "  - BiocParallel, type: imports\n",
      "  - BiocSingular, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - SingleCellExperiment, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - Seurat, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - parallel, type: imports\n",
      "  - scran, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - stats, type: imports\n",
      "  - methods, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/Dino.html\n",
      "\n",
      "Name: eiR\n",
      "Description:\n",
      "The eiR package provides utilities for accelerated structure similarity searching of very large small molecule data sets using an embedding and indexing approach.\n",
      "Version: 1.38.0\n",
      "Publication date: None\n",
      "Mantainer: Thomas Girke <thomas.girke@ucr.edu>\n",
      "Authors:\n",
      "Author: Kevin Horan, Yiqun Cao and Tyler Backman\n",
      "Dependencies:\n",
      "  - R (>= 2.10.0) type: depends\n",
      "  - ChemmineR, type: depends\n",
      "  - methods, type: depends\n",
      "  - DBI, type: depends\n",
      "  - snow, type: imports\n",
      "  - tools, type: imports\n",
      "  - snowfall, type: imports\n",
      "  - RUnit, type: imports\n",
      "  - methods, type: imports\n",
      "  - ChemmineR, type: imports\n",
      "  - RCurl, type: imports\n",
      "  - digest, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - RcppAnnoy (>= 0.0.9) type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/eiR.html\n",
      "\n",
      "Name: epivizrChart\n",
      "Description:\n",
      "This package provides an API for interactive visualization of genomic data using epiviz web components. Objects in R/BioConductor can be used to generate interactive R markdown/notebook documents or can be visualized in the R Studio's default viewer.\n",
      "Version: 1.20.0\n",
      "Publication date: None\n",
      "Mantainer: Hector Corrada Bravo <hcorrada@gmail.com>\n",
      "Authors:\n",
      "Author: Brian Gottfried [aut], Jayaram Kancherla [aut], Hector Corrada Bravo [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - epivizrData, type: imports\n",
      "  - epivizrServer, type: imports\n",
      "  - htmltools, type: imports\n",
      "  - rjson, type: imports\n",
      "  - methods, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/epivizrChart.html\n",
      "\n",
      "Name: fgga\n",
      "Description:\n",
      "Package that implements the FGGA algorithm. This package provides a hierarchical ensemble method based ob factor graphs for the consistent cross-ontology annotation of protein coding genes. FGGA embodies elements of predicate logic, communication theory, supervised learning and inference in graphical models.\n",
      "Version: 1.6.0\n",
      "Publication date: None\n",
      "Mantainer: Flavio Spetale <spetale@cifasis-conicet.gov.ar>\n",
      "Authors:\n",
      "Author: Flavio Spetale [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.2) type: depends\n",
      "  - RBGL, type: depends\n",
      "  - graph, type: imports\n",
      "  - stats, type: imports\n",
      "  - e1071, type: imports\n",
      "  - methods, type: imports\n",
      "  - gRbase, type: imports\n",
      "  - jsonlite, type: imports\n",
      "  - BiocFileCache, type: imports\n",
      "  - curl, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/fgga.html\n",
      "\n",
      "Name: FISHalyseR\n",
      "Description:\n",
      "FISHalyseR provides functionality to process and analyse digital cell culture images, in particular to quantify FISH probes within nuclei. Furthermore, it extract the spatial location of each nucleus as well as each probe enabling spatial co-localisation analysis.\n",
      "Version: 1.32.0\n",
      "Publication date: None\n",
      "Mantainer: Karesh Arunakirinathan <akaresh88@gmail.com>, Andreas Heindl <andreas.heindl@icr.ac.uk>\n",
      "Authors:\n",
      "Author: Karesh Arunakirinathan <akaresh88 at gmail.com>, Andreas Heindl <andreas.heindl at icr.ac.uk>\n",
      "Dependencies:\n",
      "  - EBImage, type: depends\n",
      "  - abind, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/FISHalyseR.html\n",
      "\n",
      "Name: flagme\n",
      "Description:\n",
      "Fragment-level analysis of gas chromatography-massspectrometry metabolomics data.\n",
      "Version: 1.54.0\n",
      "Publication date: None\n",
      "Mantainer: Mark Robinson <mark.robinson@imls.uzh.ch>, Riccardo Romoli <riccardo.romoli@unifi.it>\n",
      "Authors:\n",
      "Author: Mark Robinson <mark.robinson at imls.uzh.ch>, Riccardo Romoli <riccardo.romoli at unifi.it>\n",
      "Dependencies:\n",
      "  - gcspikelite, type: depends\n",
      "  - xcms, type: depends\n",
      "  - CAMERA, type: depends\n",
      "  - gplots, type: imports\n",
      "  - graphics, type: imports\n",
      "  - MASS, type: imports\n",
      "  - methods, type: imports\n",
      "  - SparseM, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/flagme.html\n",
      "\n",
      "Name: flowCL\n",
      "Description:\n",
      "Semantic labelling of flow cytometric cell populations.\n",
      "Version: 1.35.0\n",
      "Publication date: None\n",
      "Mantainer: Justin Meskas <justinmeskas@gmail.com>\n",
      "Authors:\n",
      "Author: Justin Meskas, Radina Droumeva\n",
      "Dependencies:\n",
      "  - R (>= 3.4) type: depends\n",
      "  - Rgraphviz, type: depends\n",
      "  - SPARQL, type: depends\n",
      "  - methods, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - utils, type: imports\n",
      "  - graph, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/flowCL.html\n",
      "\n",
      "Name: GeneGeneInteR\n",
      "Description:\n",
      "The aim of this package is to propose several methods for testing gene-gene interaction in case-control association studies. Such a test can be done by aggregating SNP-SNP interaction tests performed at the SNP level (SSI) or by using gene-gene multidimensionnal methods (GGI) methods. The package also proposes tools for a graphic display of the results. .\n",
      "Version: 1.24.0\n",
      "Publication date: None\n",
      "Mantainer: Mathieu Emily <mathieu.emily@agrocampus-ouest.fr>\n",
      "Authors:\n",
      "Author: Mathieu Emily [aut, cre], Nicolas Sounac [ctb], Florian Kroell [ctb], Magalie Houee-Bigot [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - snpStats, type: imports\n",
      "  - mvtnorm, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - igraph, type: imports\n",
      "  - kernlab, type: imports\n",
      "  - FactoMineR, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - data.table, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - methods, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/GeneGeneInteR.html\n",
      "\n",
      "Name: GeneRegionScan\n",
      "Description:\n",
      "A package with focus on analysis of discrete regions of the genome. This package is useful for investigation of one or a few genes using Affymetrix data, since it will extract probe level data using the Affymetrix Power Tools application and wrap these data into a ProbeLevelSet. A ProbeLevelSet directly extends the expressionSet, but includes additional information about the sequence of each probe and the probe set it is derived from. The package includes a number of functions used for plotting these probe level data as a function of location along sequences of mRNA-strands. This can be used for analysis of variable splicing, and is especially well suited for use with exon-array data.\n",
      "Version: 1.54.0\n",
      "Publication date: None\n",
      "Mantainer: Lasse Folkersen <lasfol@cbs.dtu.dk>\n",
      "Authors:\n",
      "Author: Lasse Folkersen, Diego Diez\n",
      "Dependencies:\n",
      "  - methods, type: depends\n",
      "  - Biobase, type: depends\n",
      "  - Biostrings, type: depends\n",
      "  - S4Vectors, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - affxparser, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - Biostrings, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/GeneRegionScan.html\n",
      "\n",
      "Name: GEOfastq\n",
      "Description:\n",
      "GEOfastq is used to download fastq files from the European Nucleotide Archive (ENA) starting with an accession from the Gene Expression Omnibus (GEO). To do this, sample metadata is retrieved from GEO and the Sequence Read Archive (SRA). SRA run accessions are then used to construct FTP and aspera download links for fastq files generated by the ENA.\n",
      "Version: 1.6.0\n",
      "Publication date: None\n",
      "Mantainer: Alex Pickering <alexvpickering@gmail.com>\n",
      "Authors:\n",
      "Author: Alex Pickering [cre, aut]\n",
      "Dependencies:\n",
      "  - xml2, type: imports\n",
      "  - rvest, type: imports\n",
      "  - stringr, type: imports\n",
      "  - RCurl, type: imports\n",
      "  - doParallel, type: imports\n",
      "  - foreach, type: imports\n",
      "  - plyr, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/GEOfastq.html\n",
      "\n",
      "Name: GNET2\n",
      "Description:\n",
      "Cluster genes to functional groups with E-M process. Iteratively perform TF assigning and Gene assigning, until the assignment of genes did not change, or max number of iterations is reached.\n",
      "Version: 1.14.0\n",
      "Publication date: None\n",
      "Mantainer: Chen Chen <ccm3x@mail.missouri.edu>\n",
      "Authors:\n",
      "Author: Chen Chen, Jie Hou and Jianlin Cheng\n",
      "Dependencies:\n",
      "  - R (>= 3.6) type: depends\n",
      "  - ggplot2, type: imports\n",
      "  - xgboost, type: imports\n",
      "  - Rcpp, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - grid, type: imports\n",
      "  - DiagrammeR, type: imports\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - graphics, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - igraph, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Apache License 2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/GNET2.html\n",
      "\n",
      "Name: GraphAT\n",
      "Description:\n",
      "Functions and data used in Balasubramanian, et al. (2004)\n",
      "Version: 1.70.0\n",
      "Publication date: None\n",
      "Mantainer: Thomas LaFramboise <tlaframb@hsph.harvard.edu>\n",
      "Authors:\n",
      "Author: R. Balasubramanian, T. LaFramboise, D. Scholtens\n",
      "Dependencies:\n",
      "  - R (>= 2.10) type: depends\n",
      "  - graph, type: depends\n",
      "  - methods, type: depends\n",
      "  - graph, type: imports\n",
      "  - MCMCpack, type: imports\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/GraphAT.html\n",
      "\n",
      "Name: GraphPAC\n",
      "Description:\n",
      "Identifies mutational clusters of amino acids in a protein while utilizing the proteins tertiary structure via a graph theoretical model.\n",
      "Version: 1.40.0\n",
      "Publication date: None\n",
      "Mantainer: Gregory Ryslik <gregory.ryslik@yale.edu>\n",
      "Authors:\n",
      "Author: Gregory Ryslik, Hongyu Zhao\n",
      "Dependencies:\n",
      "  - R (>= 2.15) type: depends\n",
      "  - iPAC, type: depends\n",
      "  - igraph, type: depends\n",
      "  - TSP, type: depends\n",
      "  - RMallow, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/GraphPAC.html\n",
      "\n",
      "Name: GSReg\n",
      "Description:\n",
      "A package for gene set analysis based on the variability of expressions as well as a method to detect Alternative Splicing Events . It implements DIfferential RAnk Conservation (DIRAC) and gene set Expression Variation Analysis (EVA) methods. For detecting Differentially Spliced genes, it provides an implementation of the Spliced-EVA (SEVA).\n",
      "Version: 1.32.0\n",
      "Publication date: None\n",
      "Mantainer: Bahman Afsari <bahman@jhu.edu>, Elana J. Fertig <ejfertig@jhmi.edu>\n",
      "Authors:\n",
      "Author: Bahman Afsari <bahman at jhu.edu>, Elana J. Fertig <ejfertig at jhmi.edu>\n",
      "Dependencies:\n",
      "  - R (>= 2.13.1) type: depends\n",
      "  - Homo.sapiens, type: depends\n",
      "  - org.Hs.eg.db, type: depends\n",
      "  - GenomicFeatures, type: depends\n",
      "  - AnnotationDbi, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/GSReg.html\n",
      "\n",
      "Name: Harshlight\n",
      "Description:\n",
      "The package is used to detect extended, diffuse and compact blemishes on microarray chips. Harshlight automatically marks the areas in a collection of chips (affybatch objects) and a corrected AffyBatch object is returned, in which the defected areas are substituted with NAs or the median of the values of the same probe in the other chips in the collection. The new version handle the substitute value as whole matrix to solve the memory problem.\n",
      "Version: 1.70.0\n",
      "Publication date: None\n",
      "Mantainer: Maurizio Pellegrino <mpellegri@berkeley.edu>\n",
      "Authors:\n",
      "Author: Mayte Suarez-Farinas, Maurizio Pellegrino, Knut M. Wittkowski, Marcelo O. Magnasco\n",
      "Dependencies:\n",
      "  - R (>= 2.10) type: depends\n",
      "  - affy, type: imports\n",
      "  - altcdfenvs, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/Harshlight.html\n",
      "\n",
      "Name: iasva\n",
      "Description:\n",
      "Iteratively Adjusted Surrogate Variable Analysis (IA-SVA) is a statistical framework to uncover hidden sources of variation even when these sources are correlated. IA-SVA provides a flexible methodology to i) identify a hidden factor for unwanted heterogeneity while adjusting for all known factors; ii) test the significance of the putative hidden factor for explaining the unmodeled variation in the data; and iii), if significant, use the estimated factor as an additional known factor in the next iteration to uncover further hidden factors.\n",
      "Version: 1.16.0\n",
      "Publication date: None\n",
      "Mantainer: Donghyung Lee <Donghyung.Lee@jax.org>, Anthony Cheng <Anthony.Cheng@jax.org>\n",
      "Authors:\n",
      "Author: Donghyung Lee [aut, cre], Anthony Cheng [aut], Nathan Lawlor [aut], Duygu Ucar [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.5) type: depends\n",
      "  - irlba, type: imports\n",
      "  - stats, type: imports\n",
      "  - cluster, type: imports\n",
      "  - graphics, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/iasva.html\n",
      "\n",
      "Name: ibh\n",
      "Description:\n",
      "This package contains methods for calculating Interaction Based Homogeneity to evaluate fitness of gene lists to an interaction network which is useful for evaluation of clustering results and gene list analysis. BioGRID interactions are used in the calculation. The user can also provide their own interactions.\n",
      "Version: 1.46.0\n",
      "Publication date: None\n",
      "Mantainer: Kircicegi Korkmaz <e102771@ceng.metu.edu.tr>\n",
      "Authors:\n",
      "Author: Kircicegi Korkmaz, Volkan Atalay, Rengul Cetin Atalay.\n",
      "Dependencies:\n",
      "  - simpIntLists, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ibh.html\n",
      "\n",
      "Name: iGC\n",
      "Description:\n",
      "This package is intended to identify differentially expressed genes driven by Copy Number Alterations from samples with both gene expression and CNA data.\n",
      "Version: 1.28.0\n",
      "Publication date: None\n",
      "Mantainer: Liang-Bo Wang <r02945054@ntu.edu.tw>\n",
      "Authors:\n",
      "Author: Yi-Pin Lai [aut], Liang-Bo Wang [aut, cre], Tzu-Pin Lu [aut], Eric Y. Chuang [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.2.0) type: depends\n",
      "  - plyr, type: imports\n",
      "  - data.table, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/iGC.html\n",
      "\n",
      "Name: iSeq\n",
      "Description:\n",
      "Bayesian hidden Ising models are implemented to identify IP-enriched genomic regions from ChIP-seq data. They can be used to analyze ChIP-seq data with and without controls and replicates.\n",
      "Version: 1.50.0\n",
      "Publication date: None\n",
      "Mantainer: Qianxing Mo <qianxing.mo@moffitt.org>\n",
      "Authors:\n",
      "Author: Qianxing Mo\n",
      "Dependencies:\n",
      "  - R (>= 2.10.0) type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/iSeq.html\n",
      "\n",
      "Name: KCsmart\n",
      "Description:\n",
      "Multi sample aCGH analysis package using kernel convolution\n",
      "Version: 2.56.0\n",
      "Publication date: None\n",
      "Mantainer: Jorma de Ronde <j.d.ronde@nki.nl>\n",
      "Authors:\n",
      "Author: Jorma de Ronde, Christiaan Klijn, Arno Velds\n",
      "Dependencies:\n",
      "  - siggenes, type: depends\n",
      "  - multtest, type: depends\n",
      "  - KernSmooth, type: depends\n",
      "  - methods, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/KCsmart.html\n",
      "\n",
      "Name: MANOR\n",
      "Description:\n",
      "Importation, normalization, visualization, and quality control functions to correct identified sources of variability in array-CGH experiments.\n",
      "Version: 1.70.0\n",
      "Publication date: None\n",
      "Mantainer: Pierre Neuvial <pierre.neuvial@math.cnrs.fr>\n",
      "Authors:\n",
      "Author: Pierre Neuvial <pierre.neuvial at math.cnrs.fr>, Philippe HupÃ© <philippe.hupe at curie.fr>\n",
      "Dependencies:\n",
      "  - R (>= 2.10) type: depends\n",
      "  - GLAD, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/MANOR.html\n",
      "\n",
      "Name: MBQN\n",
      "Description:\n",
      "Modified quantile normalization for omics or other matrix-like data distorted in location and scale.\n",
      "Version: 2.10.0\n",
      "Publication date: None\n",
      "Mantainer: Eva Brombacher <brombach@imbi.uni-freiburg.de>\n",
      "Authors:\n",
      "Author: Eva Brombacher [aut, cre] , Clemens Kreutz [aut, ctb] , Ariane Schad [aut, ctb]\n",
      "Dependencies:\n",
      "  - R (>= 3.6) type: depends\n",
      "  - stats, type: imports\n",
      "  - graphics, type: imports\n",
      "  - utils, type: imports\n",
      "  - limma, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - preprocessCore, type: imports\n",
      "  - BiocFileCache, type: imports\n",
      "  - rappdirs, type: imports\n",
      "  - xml2, type: imports\n",
      "  - RCurl, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - PairedData, type: imports\n",
      "  - rmarkdown, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3 + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/MBQN.html\n",
      "\n",
      "Name: MEDME\n",
      "Description:\n",
      "MEDME allows the prediction of absolute and relative methylation levels based on measures obtained by MeDIP-microarray experiments\n",
      "Version: 1.58.0\n",
      "Publication date: None\n",
      "Mantainer: Mattia Pelizzola <mattia.pelizzola@gmail.com>\n",
      "Authors:\n",
      "Author: Mattia Pelizzola and Annette Molinaro\n",
      "Dependencies:\n",
      "  - R (>= 2.15) type: depends\n",
      "  - grDevices, type: depends\n",
      "  - graphics, type: depends\n",
      "  - methods, type: depends\n",
      "  - stats, type: depends\n",
      "  - utils, type: depends\n",
      "  - Biostrings, type: imports\n",
      "  - MASS, type: imports\n",
      "  - drc, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/MEDME.html\n",
      "\n",
      "Name: metahdep\n",
      "Description:\n",
      "Tools for meta-analysis in the presence of hierarchical (and/or sampling) dependence, including with gene expression studies\n",
      "Version: 1.56.0\n",
      "Publication date: None\n",
      "Mantainer: John R. Stevens <john.r.stevens@usu.edu>\n",
      "Authors:\n",
      "Author: John R. Stevens, Gabriel Nicholas\n",
      "Dependencies:\n",
      "  - R (>= 2.10) type: depends\n",
      "  - methods, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/metahdep.html\n",
      "\n",
      "Name: MetCirc\n",
      "Description:\n",
      "MetCirc comprises a workflow to interactively explore high-resolution MS/MS metabolomics data. MetCirc uses the Spectra object infrastructure defined in the package Spectra that stores MS/MS spectra. MetCirc offers functionality to calculate similarity between precursors based on the normalised dot product, neutral losses or user-defined functions and visualise similarities in a circular layout. Within the interactive framework the user can annotate MS/MS features based on their similarity to (known) related MS/MS features.\n",
      "Version: 1.28.0\n",
      "Publication date: None\n",
      "Mantainer: Thomas Naake <thomasnaake@googlemail.com>\n",
      "Authors:\n",
      "Author: Thomas Naake <thomasnaake at googlemail.com>, Johannes Rainer <johannes.rainer at eurac.edu> and Emmanuel Gaquerel <emmanuel.gaquerel at ibmp-cnrs.unistra.fr>\n",
      "Dependencies:\n",
      "  - R (>= 3.5) type: depends\n",
      "  - amap (>= 0.8) type: depends\n",
      "  - circlize (>= 0.3.9) type: depends\n",
      "  - scales (>= 0.3.0) type: depends\n",
      "  - shiny (>= 1.0.0) type: depends\n",
      "  - Spectra, type: depends\n",
      "  - ggplot2 (>= 3.2.1) type: imports\n",
      "  - MsCoreUtils, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 3)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/MetCirc.html\n",
      "\n",
      "Name: MGFM\n",
      "Description:\n",
      "The package is designed to detect marker genes from Microarray gene expression data sets\n",
      "Version: 1.32.0\n",
      "Publication date: None\n",
      "Mantainer: Khadija El Amrani <khadija.el-amrani@charite.de>\n",
      "Authors:\n",
      "Author: Khadija El Amrani\n",
      "Dependencies:\n",
      "  - AnnotationDbi, type: depends\n",
      "  - annotate, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/MGFM.html\n",
      "\n",
      "Name: MinimumDistance\n",
      "Description:\n",
      "Analysis of de novo copy number variants in trios from high-dimensional genotyping platforms.\n",
      "Version: 1.42.0\n",
      "Publication date: None\n",
      "Mantainer: Robert Scharpf <rscharpf@jhu.edu>\n",
      "Authors:\n",
      "Author: Robert B Scharpf and Ingo Ruczinski\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - VanillaICE, type: depends\n",
      "  - methods, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - MatrixGenerics, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - oligoClasses, type: imports\n",
      "  - DNAcopy, type: imports\n",
      "  - ff, type: imports\n",
      "  - foreach, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - lattice, type: imports\n",
      "  - data.table, type: imports\n",
      "  - grid, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/MinimumDistance.html\n",
      "\n",
      "Name: MiPP\n",
      "Description:\n",
      "This package finds optimal sets of genes that seperate samples into two or more classes.\n",
      "Version: 1.70.0\n",
      "Publication date: None\n",
      "Mantainer: Sukwoo Kim <s4kim@korea.ac.kr>\n",
      "Authors:\n",
      "Author: HyungJun Cho <hj4cho at korea.ac.kr>, Sukwoo Kim <s4kim at korea.ac.kr>, Mat Soukup <soukup at fda.gov>, and Jae K. Lee <jaeklee at virginia.edu>\n",
      "Dependencies:\n",
      "  - R (>= 2.4) type: depends\n",
      "  - Biobase, type: imports\n",
      "  - e1071, type: imports\n",
      "  - MASS, type: imports\n",
      "  - stats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/MiPP.html\n",
      "\n",
      "Name: MiRaGE\n",
      "Description:\n",
      "The package contains functions for inferece of target gene regulation by miRNA, based on only target gene expression profile.\n",
      "Version: 1.40.0\n",
      "Publication date: None\n",
      "Mantainer: Y-h. Taguchi <tag@granular.com>\n",
      "Authors:\n",
      "Author: Y-h. Taguchi <tag at granular.com>\n",
      "Dependencies:\n",
      "  - R (>= 3.1.0) type: depends\n",
      "  - Biobase, type: depends\n",
      "  - BiocGenerics, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - AnnotationDbi, type: imports\n",
      "  - BiocManager, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/MiRaGE.html\n",
      "\n",
      "Name: MOGAMUN\n",
      "Description:\n",
      "MOGAMUN is a multi-objective genetic algorithm that identifies active modules in a multiplex biological network. This allows analyzing different biological networks at the same time. MOGAMUN is based on NSGA-II (Non-Dominated Sorting Genetic Algorithm, version II), which we adapted to work on networks.\n",
      "Version: 1.8.0\n",
      "Publication date: None\n",
      "Mantainer: Elva-MarÃ­a Novoa-del-Toro <elvanov@hotmail.com>\n",
      "Authors:\n",
      "Author: Elva-MarÃ­a Novoa-del-Toro [aut, cre]\n",
      "Dependencies:\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - RCy3, type: imports\n",
      "  - stringr, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - RUnit, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - igraph, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3 + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/MOGAMUN.html\n",
      "\n",
      "Name: multicrispr\n",
      "Description:\n",
      "This package is for designing Crispr/Cas9 and Prime Editing experiments. It contains functions to (1) define and transform genomic targets, (2) find spacers (4) count offtarget (mis)matches, and (5) compute Doench2016/2014 targeting efficiency. Care has been taken for multicrispr to scale well towards large target sets, enabling the design of large Crispr/Cas9 libraries.\n",
      "Version: 1.8.0\n",
      "Publication date: None\n",
      "Mantainer: Aditya Bhagwat <aditya.bhagwat@mpi-bn.mpg.de>\n",
      "Authors:\n",
      "Author: Aditya Bhagwat [aut, cre], Rene Wiegandt [ctb], Mette Bentsen [ctb], Jens Preussner [ctb], Michael Lawrence [ctb], HervÃ© PagÃ¨s [ctb], Johannes Graumann [sad], Mario Looso [sad, rth]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - assertive, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - BSgenome, type: imports\n",
      "  - CRISPRseek, type: imports\n",
      "  - data.table, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - GenomicFeatures, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - grid, type: imports\n",
      "  - karyoploteR, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - methods, type: imports\n",
      "  - parallel, type: imports\n",
      "  - plyranges, type: imports\n",
      "  - Rbowtie, type: imports\n",
      "  - reticulate, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - stats, type: imports\n",
      "  - stringi, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - tidyselect, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/multicrispr.html\n",
      "\n",
      "Name: nanotatoR\n",
      "Description:\n",
      "Whole genome sequencing (WGS) has successfully been used to identify single-nucleotide variants (SNV), small insertions and deletions (INDELs) and, more recently, small copy number variants (CNVs). However, due to utilization of short reads, it is not well suited for identification of structural variants (SV). Optical mapping (OM) from Bionano Genomics, utilizes long fluorescently labeled megabase size DNA molecules for de novo genome assembly and identification of SVs with a much higher sensitivity than WGS. Nevertheless, currently available SV annotation tools have limited number of functions. NanotatoR is an R package written to provide a set of annotations for SVs identified by OM. It uses Database of Genomic Variants (DGV), Database of Chromosomal Imbalance and Phenotype in Humans Using Ensembl Resources (DECIPHER) as well as a subset (154 samples) of 1000 Genome Project to calculate the population frequencies of the SVs (an optional internal cohort SV frequency calculation is also available). NanotatoR creates a primary gene list (PG) from NCBI databases based on probandâs phenotype specific keywords and compares the list to the set of genes overlapping/near SVs. The output is given in an Excel file format, which is subdivided into multiple sheets based on SV type (e.g., INDELs, Inversions, Translocations). Users then have a choice to filter SVs using the provided annotations for de novo (if parental samples are available) or inherited rare variants.\n",
      "Version: 1.14.0\n",
      "Publication date: None\n",
      "Mantainer: Surajit Bhattacharya <sbhattach2@childrensnational.org>\n",
      "Authors:\n",
      "Author: Surajit Bhattacharya, Hayk Barsheghyan, Emmanuele C Delot and Eric Vilain\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - hash (>= 2.2.6) type: imports\n",
      "  - openxlsx (>= 4.0.17) type: imports\n",
      "  - rentrez (>= 1.1.0) type: imports\n",
      "  - stats, type: imports\n",
      "  - rlang, type: imports\n",
      "  - stringr, type: imports\n",
      "  - knitr, type: imports\n",
      "  - testthat, type: imports\n",
      "  - utils, type: imports\n",
      "  - AnnotationDbi, type: imports\n",
      "  - httr, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - tidyverse, type: imports\n",
      "  - VarfromPDB, type: imports\n",
      "  - org.Hs.eg.db, type: imports\n",
      "  - curl, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - XML, type: imports\n",
      "  - XML2R, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/nanotatoR.html\n",
      "\n",
      "Name: ncRNAtools\n",
      "Description:\n",
      "ncRNAtools provides a set of basic tools for handling and analyzing non-coding RNAs. These include tools to access the RNAcentral database and to predict and visualize the secondary structure of non-coding RNAs. The package also provides tools to read, write and interconvert the file formats most commonly used for representing such secondary structures.\n",
      "Version: 1.8.0\n",
      "Publication date: None\n",
      "Mantainer: Lara Selles Vidal <lara.selles@oist.jp>\n",
      "Authors:\n",
      "Author: Lara Selles Vidal [cre, aut] , Rafael Ayala [aut] , Guy-Bart Stan [aut] , Rodrigo Ledesma-Amaro [aut]\n",
      "Dependencies:\n",
      "  - httr, type: imports\n",
      "  - xml2, type: imports\n",
      "  - utils, type: imports\n",
      "  - methods, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ncRNAtools.html\n",
      "\n",
      "Name: NxtIRFcore\n",
      "Description:\n",
      "Interactively analyses Intron Retention and Alternative Splicing Events (ASE) in RNA-seq data. NxtIRF quantifies ASE events in BAM files aligned to the genome using a splice-aware aligner such as STAR. The core quantitation algorithm relies on the IRFinder/C++ engine ported via Rcpp for multi-platform compatibility. In addition, NxtIRF provides convenient pipelines for downstream analysis and publication-ready visualisation tools. Note that NxtIRFcore is now replaced by SpliceWiz in Bioconductor 3.16 onwards.\n",
      "Version: 1.4.0\n",
      "Publication date: None\n",
      "Mantainer: Alex Chit Hei Wong <a.wong@centenary.org.au>\n",
      "Authors:\n",
      "Author: Alex Chit Hei Wong [aut, cre, cph], William Ritchie [cph], Ulf Schmitz [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - NxtIRFdata, type: depends\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - tools, type: imports\n",
      "  - parallel, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - Rcpp (>= 1.0.5) type: imports\n",
      "  - data.table, type: imports\n",
      "  - fst, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - AnnotationHub, type: imports\n",
      "  - BiocFileCache, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - BSgenome, type: imports\n",
      "  - DelayedArray, type: imports\n",
      "  - DelayedMatrixStats, type: imports\n",
      "  - genefilter, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - HDF5Array, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - plotly, type: imports\n",
      "  - R.utils, type: imports\n",
      "  - rhdf5, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/NxtIRFcore.html\n",
      "\n",
      "Name: onlineFDR\n",
      "Description:\n",
      "This package allows users to control the false discovery rate (FDR) or familywise error rate (FWER) for online multiple hypothesis testing, where hypotheses arrive in a stream. In this framework, a null hypothesis is rejected based on the evidence against it and on the previous rejection decisions.\n",
      "Version: 2.6.0\n",
      "Publication date: None\n",
      "Mantainer: David S. Robertson <david.robertson@mrc-bsu.cam.ac.uk>\n",
      "Authors:\n",
      "Author: David S. Robertson [aut, cre], Lathan Liou [aut], Aaditya Ramdas [aut], Adel Javanmard [ctb], Andrea Montanari [ctb], Jinjin Tian [ctb], Tijana Zrnic [ctb], Natasha A. Karp [aut]\n",
      "Dependencies:\n",
      "  - stats, type: imports\n",
      "  - Rcpp, type: imports\n",
      "  - progress, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/onlineFDR.html\n",
      "\n",
      "Name: OpenStats\n",
      "Description:\n",
      "Package contains several methods for statistical analysis of genotype to phenotype association in high-throughput screening pipelines.\n",
      "Version: 1.10.0\n",
      "Publication date: None\n",
      "Mantainer: Hamed Haseli Mashhadi <hamedhm@ebi.ac.uk>\n",
      "Authors:\n",
      "Author: Hamed Haseli Mashhadi\n",
      "Dependencies:\n",
      "  - nlme, type: depends\n",
      "  - MASS, type: imports\n",
      "  - jsonlite, type: imports\n",
      "  - Hmisc, type: imports\n",
      "  - methods, type: imports\n",
      "  - knitr, type: imports\n",
      "  - AICcmodavg, type: imports\n",
      "  - car, type: imports\n",
      "  - rlist, type: imports\n",
      "  - summarytools, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/OpenStats.html\n",
      "\n",
      "Name: PING\n",
      "Description:\n",
      "Probabilistic inference of ChIP-Seq using an empirical Bayes mixture model approach.\n",
      "Version: 2.42.0\n",
      "Publication date: None\n",
      "Mantainer: Renan Sauteraud <renan.sauteraud@gmail.com>\n",
      "Authors:\n",
      "Author: Xuekui Zhang <xuezhang at jhsph.edu>, Raphael Gottardo <rgottard at fredhutch.org>, Sangsoon Woo <swoo at fhcrc.org>\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - methods, type: imports\n",
      "  - PICS, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - stats, type: imports\n",
      "  - Gviz, type: imports\n",
      "  - fda, type: imports\n",
      "  - BSgenome, type: imports\n",
      "  - stats4, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/PING.html\n",
      "\n",
      "Name: proFIA\n",
      "Description:\n",
      "Flow Injection Analysis coupled to High-Resolution Mass Spectrometry is a promising approach for high-throughput metabolomics. FIA- HRMS data, however, cannot be pre-processed with current software tools which rely on liquid chromatography separation, or handle low resolution data only. Here we present the proFIA package, which implements a new methodology to pre-process FIA-HRMS raw data (netCDF, mzData, mzXML, and mzML) including noise modelling and injection peak reconstruction, and generate the peak table. The workflow includes noise modelling, band detection and filtering then signal matching and missing value imputation. The peak table can then be exported as a .tsv file for further analysis. Visualisations to assess the quality of the data and of the signal made are easely produced.\n",
      "Version: 1.23.0\n",
      "Publication date: None\n",
      "Mantainer: Alexis Delabriere <alexis.delabriere@outlook.fr>\n",
      "Authors:\n",
      "Author: Alexis Delabriere and Etienne Thevenot.\n",
      "Dependencies:\n",
      "  - R (>= 2.5.0) type: depends\n",
      "  - xcms, type: depends\n",
      "  - stats, type: imports\n",
      "  - graphics, type: imports\n",
      "  - utils, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - methods, type: imports\n",
      "  - pracma, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - minpack.lm, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - missForest, type: imports\n",
      "  - ropls, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "CeCILL\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/proFIA.html\n",
      "\n",
      "Name: PSEA\n",
      "Description:\n",
      "Deconvolution of gene expression data by Population-Specific Expression Analysis (PSEA).\n",
      "Version: 1.32.0\n",
      "Publication date: None\n",
      "Mantainer: Alexandre Kuhn <alexandre.m.kuhn@gmail.com>\n",
      "Authors:\n",
      "Author: Alexandre Kuhn <alexandre.m.kuhn at gmail.com>\n",
      "Dependencies:\n",
      "  - Biobase, type: imports\n",
      "  - MASS, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/PSEA.html\n",
      "\n",
      "Name: ptairMS\n",
      "Description:\n",
      "This package implements a suite of methods to preprocess data from PTR-TOF-MS instruments (HDF5 format) and generates the 'sample by features' table of peak intensities in addition to the sample and feature metadata (as a single ExpressionSet object for subsequent statistical analysis). This package also permit usefull tools for cohorts management as analyzing data progressively, visualization tools and quality control. The steps include calibration, expiration detection, peak detection and quantification, feature alignment, missing value imputation and feature annotation. Applications to exhaled air and cell culture in headspace are described in the vignettes and examples. This package was used for data analysis of Gassin Delyle study on adults undergoing invasive mechanical ventilation in the intensive care unit due to severe COVID-19 or non-COVID-19 acute respiratory distress syndrome (ARDS), and permit to identfy four potentiel biomarquers of the infection.\n",
      "Version: 1.6.0\n",
      "Publication date: None\n",
      "Mantainer: camille Roquencourt <camille.roquencourt@hotmail.fr>\n",
      "Authors:\n",
      "Author: camille Roquencourt [aut, cre]\n",
      "Dependencies:\n",
      "  - Biobase, type: imports\n",
      "  - bit64, type: imports\n",
      "  - chron, type: imports\n",
      "  - data.table, type: imports\n",
      "  - doParallel, type: imports\n",
      "  - DT, type: imports\n",
      "  - enviPat, type: imports\n",
      "  - foreach, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - ggpubr, type: imports\n",
      "  - gridExtra, type: imports\n",
      "  - Hmisc, type: imports\n",
      "  - methods, type: imports\n",
      "  - minpack.lm, type: imports\n",
      "  - MSnbase, type: imports\n",
      "  - parallel, type: imports\n",
      "  - plotly, type: imports\n",
      "  - rhdf5, type: imports\n",
      "  - rlang, type: imports\n",
      "  - Rcpp, type: imports\n",
      "  - shiny, type: imports\n",
      "  - shinyscreenshot, type: imports\n",
      "  - signal, type: imports\n",
      "  - scales, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ptairMS.html\n",
      "\n",
      "Name: recoup\n",
      "Description:\n",
      "recoup calculates and plots signal profiles created from short sequence reads derived from Next Generation Sequencing technologies. The profiles provided are either sumarized curve profiles or heatmap profiles. Currently, recoup supports genomic profile plots for reads derived from ChIP-Seq and RNA-Seq experiments. The package uses ggplot2 and ComplexHeatmap graphics facilities for curve and heatmap coverage profiles respectively.\n",
      "Version: 1.26.0\n",
      "Publication date: None\n",
      "Mantainer: Panagiotis Moulos <moulos@fleming.gr>\n",
      "Authors:\n",
      "Author: Panagiotis Moulos <moulos at fleming.gr>\n",
      "Dependencies:\n",
      "  - R (>= 4.0.0) type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - GenomicAlignments, type: depends\n",
      "  - ggplot2, type: depends\n",
      "  - ComplexHeatmap, type: depends\n",
      "  - BiocGenerics, type: imports\n",
      "  - biomaRt, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - circlize, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - GenomicFeatures, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - httr, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - methods, type: imports\n",
      "  - parallel, type: imports\n",
      "  - RSQLite, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - stats, type: imports\n",
      "  - stringr, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 3)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/recoup.html\n",
      "\n",
      "Name: regutools\n",
      "Description:\n",
      "RegulonDB has collected, harmonized and centralized data from hundreds of experiments for nearly two decades and is considered a point of reference for transcriptional regulation in Escherichia coli K12. Here, we present the regutools R package to facilitate programmatic access to RegulonDB data in computational biology. regutools provides researchers with the possibility of writing reproducible workflows with automated queries to RegulonDB. The regutools package serves as a bridge between RegulonDB data and the Bioconductor ecosystem by reusing the data structures and statistical methods powered by other Bioconductor packages. We demonstrate the integration of regutools with Bioconductor by analyzing transcription factor DNA binding sites and transcriptional regulatory networks from RegulonDB. We anticipate that regutools will serve as a useful building block in our progress to further our understanding of gene regulatory networks.\n",
      "Version: 1.10.0\n",
      "Publication date: None\n",
      "Mantainer: Joselyn Chavez <joselynchavezf@gmail.com>\n",
      "Authors:\n",
      "Author: Joselyn Chavez [aut, cre] , Carmina Barberena-Jonas [aut] , Jesus E. Sotelo-Fonseca [aut] , Jose Alquicira-Hernandez [ctb] , Heladia Salgado [ctb] , Leonardo Collado-Torres [aut] , Alejandro Reyes [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - AnnotationDbi, type: imports\n",
      "  - AnnotationHub, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - DBI, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - Gviz, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - RCy3, type: imports\n",
      "  - RSQLite, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - BiocFileCache, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/regutools.html\n",
      "\n",
      "Name: rgsepd\n",
      "Description:\n",
      "R/GSEPD is a bioinformatics package for R to help disambiguate transcriptome samples (a matrix of RNA-Seq counts at transcript IDs) by automating differential expression (with DESeq2), then gene set enrichment (with GOSeq), and finally a N-dimensional projection to quantify in which ways each sample is like either treatment group.\n",
      "Version: 1.30.0\n",
      "Publication date: None\n",
      "Mantainer: Karl Stamm <karl.stamm@gmail.com>\n",
      "Authors:\n",
      "Author: Karl Stamm\n",
      "Dependencies:\n",
      "  - R (>= 4.2.0) type: depends\n",
      "  - DESeq2, type: depends\n",
      "  - goseq, type: depends\n",
      "  - gplots, type: imports\n",
      "  - biomaRt, type: imports\n",
      "  - org.Hs.eg.db, type: imports\n",
      "  - GO.db, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - AnnotationDbi, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/rgsepd.html\n",
      "\n",
      "Name: Rmagpie\n",
      "Description:\n",
      "Microarray Classification is designed for both biologists and statisticians. It offers the ability to train a classifier on a labelled microarray dataset and to then use that classifier to predict the class of new observations. A range of modern classifiers are available, including support vector machines (SVMs), nearest shrunken centroids (NSCs)... Advanced methods are provided to estimate the predictive error rate and to report the subset of genes which appear essential in discriminating between classes.\n",
      "Version: 1.54.0\n",
      "Publication date: None\n",
      "Mantainer: Camille Maumet <Rmagpie@gmail.com>\n",
      "Authors:\n",
      "Author: Camille Maumet <Rmagpie at gmail.com>, with contributions from C. Ambroise J. Zhu\n",
      "Dependencies:\n",
      "  - R (>= 2.6.1) type: depends\n",
      "  - Biobase, type: depends\n",
      "  - Biobase, type: imports\n",
      "  - e1071, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - kernlab, type: imports\n",
      "  - methods, type: imports\n",
      "  - pamr, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 3)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/Rmagpie.html\n",
      "\n",
      "Name: rRDP\n",
      "Description:\n",
      "Seamlessly interfaces RDP classifier (version 2.9).\n",
      "Version: 1.32.0\n",
      "Publication date: None\n",
      "Mantainer: Michael Hahsler <mhahsler@lyle.smu.edu>\n",
      "Authors:\n",
      "Author: Michael Hahsler, Anurag Nagar\n",
      "Dependencies:\n",
      "  - Biostrings, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2 | file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/rRDP.html\n",
      "\n",
      "Name: RTCA\n",
      "Description:\n",
      "Import, analyze and visualize data from Roche(R) xCELLigence RTCA systems. The package imports real-time cell electrical impedance data into R. As an alternative to commercial software shipped along the system, the Bioconductor package RTCA provides several unique transformation (normalization) strategies and various visualization tools.\n",
      "Version: 1.50.0\n",
      "Publication date: None\n",
      "Mantainer: Jitao David Zhang <davidvonpku@gmail.com>\n",
      "Authors:\n",
      "Author: Jitao David Zhang\n",
      "Dependencies:\n",
      "  - methods, type: depends\n",
      "  - stats, type: depends\n",
      "  - graphics, type: depends\n",
      "  - Biobase, type: depends\n",
      "  - RColorBrewer, type: depends\n",
      "  - gtools, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/RTCA.html\n",
      "\n",
      "Name: SDAMS\n",
      "Description:\n",
      "This Package utilizes a Semi-parametric Differential Abundance/expression analysis (SDA) method for metabolomics and proteomics data from mass spectrometry as well as single-cell RNA sequencing data. SDA is able to robustly handle non-normally distributed data and provides a clear quantification of the effect size.\n",
      "Version: 1.18.0\n",
      "Publication date: None\n",
      "Mantainer: Yuntong Li <yuntong.li@uky.edu>\n",
      "Authors:\n",
      "Author: Yuntong Li <yuntong.li at uky.edu>, Chi Wang <chi.wang at uky.edu>, Li Chen <lichenuky at uky.edu>\n",
      "Dependencies:\n",
      "  - R (>= 3.5) type: depends\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - trust, type: imports\n",
      "  - qvalue, type: imports\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/SDAMS.html\n",
      "\n",
      "Name: seqCAT\n",
      "Description:\n",
      "The seqCAT package uses variant calling data (in the form of VCF files) from high throughput sequencing technologies to authenticate and validate the source, function and characteristics of biological samples used in scientific endeavours.\n",
      "Version: 1.20.0\n",
      "Publication date: None\n",
      "Mantainer: Erik Fasterius <erik.fasterius@outlook.com>\n",
      "Authors:\n",
      "Author: Erik Fasterius [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 3.6) type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - VariantAnnotation, type: depends\n",
      "  - dplyr (>= 0.5.0) type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - ggplot2 (>= 2.2.1) type: imports\n",
      "  - grid (>= 3.5.0) type: imports\n",
      "  - IRanges, type: imports\n",
      "  - methods, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - rlang, type: imports\n",
      "  - scales (>= 0.4.1) type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - stats, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - tidyr (>= 0.6.1) type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENCE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/seqCAT.html\n",
      "\n",
      "Name: sitePath\n",
      "Description:\n",
      "Using site polymorphism is one of the ways to cluster DNA/protein sequences but it is possible for the sequences with the same polymorphism on a single site to be genetically distant. This package is aimed at clustering sequences using site polymorphism and their corresponding phylogenetic trees. By considering their location on the tree, only the structurally adjacent sequences will be clustered. However, the adjacent sequences may not necessarily have the same polymorphism. So a branch-and-bound like algorithm is used to minimize the entropy representing the purity of site polymorphism of each cluster.\n",
      "Version: 1.14.0\n",
      "Publication date: None\n",
      "Mantainer: Chengyang Ji <chengyang.ji12@alumni.xjtlu.edu.cn>\n",
      "Authors:\n",
      "Author: Chengyang Ji [aut, cre, cph] , Hangyu Zhou [ths], Aiping Wu [ths]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - RColorBrewer, type: imports\n",
      "  - Rcpp, type: imports\n",
      "  - ape, type: imports\n",
      "  - aplot, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - ggrepel, type: imports\n",
      "  - ggtree, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - gridExtra, type: imports\n",
      "  - methods, type: imports\n",
      "  - parallel, type: imports\n",
      "  - seqinr, type: imports\n",
      "  - stats, type: imports\n",
      "  - tidytree, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/sitePath.html\n",
      "\n",
      "Name: sRACIPE\n",
      "Description:\n",
      "sRACIPE implements a randomization-based method for gene circuit modeling. It allows us to study the effect of both the gene expression noise and the parametric variation on any gene regulatory circuit (GRC) using only its topology, and simulates an ensemble of models with random kinetic parameters at multiple noise levels. Statistical analysis of the generated gene expressions reveals the basin of attraction and stability of various phenotypic states and their changes associated with intrinsic and extrinsic noises. sRACIPE provides a holistic picture to evaluate the effects of both the stochastic nature of cellular processes and the parametric variation.\n",
      "Version: 1.14.0\n",
      "Publication date: None\n",
      "Mantainer: Vivek Kohar <vivek.kohar@gmail.com>\n",
      "Authors:\n",
      "Author: Vivek Kohar [aut, cre] , Mingyang Lu [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.6.0) type: depends\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - methods, type: depends\n",
      "  - Rcpp, type: depends\n",
      "  - ggplot2, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - MASS, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - gridExtra, type: imports\n",
      "  - visNetwork, type: imports\n",
      "  - gplots, type: imports\n",
      "  - umap, type: imports\n",
      "  - htmlwidgets, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - graphics, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/sRACIPE.html\n",
      "\n",
      "Name: ssrch\n",
      "Description:\n",
      "Demonstrate tokenization and a search gadget for collections of CSV files.\n",
      "Version: 1.14.0\n",
      "Publication date: None\n",
      "Mantainer: VJ Carey <stvjc@channing.harvard.edu>\n",
      "Authors:\n",
      "Author: Vince Carey\n",
      "Dependencies:\n",
      "  - R (>= 3.6) type: depends\n",
      "  - methods, type: depends\n",
      "  - shiny, type: imports\n",
      "  - DT, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ssrch.html\n",
      "\n",
      "Name: SummarizedBenchmark\n",
      "Description:\n",
      "This package defines the BenchDesign and SummarizedBenchmark classes for building, executing, and evaluating benchmark experiments of computational methods. The SummarizedBenchmark class extends the RangedSummarizedExperiment object, and is designed to provide infrastructure to store and compare the results of applying different methods to a shared data set. This class provides an integrated interface to store metadata such as method parameters and software versions as well as ground truths (when these are available) and evaluation metrics.\n",
      "Version: 2.16.0\n",
      "Publication date: None\n",
      "Mantainer: Patrick Kimes <patrick.kimes@gmail.com>\n",
      "Authors:\n",
      "Author: Alejandro Reyes [aut] , Patrick Kimes [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 3.6) type: depends\n",
      "  - tidyr, type: depends\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - S4Vectors, type: depends\n",
      "  - BiocGenerics, type: depends\n",
      "  - methods, type: depends\n",
      "  - UpSetR, type: depends\n",
      "  - rlang, type: depends\n",
      "  - stringr, type: depends\n",
      "  - utils, type: depends\n",
      "  - BiocParallel, type: depends\n",
      "  - ggplot2, type: depends\n",
      "  - mclust, type: depends\n",
      "  - dplyr, type: depends\n",
      "  - digest, type: depends\n",
      "  - sessioninfo, type: depends\n",
      "  - crayon, type: depends\n",
      "  - tibble, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 3)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/SummarizedBenchmark.html\n",
      "\n",
      "Name: TnT\n",
      "Description:\n",
      "A R interface to the TnT javascript library (https://github.com/ tntvis) to provide interactive and flexible visualization of track-based genomic data.\n",
      "Version: 1.20.0\n",
      "Publication date: None\n",
      "Mantainer: Jialin Ma <marlin-@gmx.cn>\n",
      "Authors:\n",
      "Author: Jialin Ma [cre, aut], Miguel Pignatelli [aut], Toby Hocking [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.4) type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - htmlwidgets, type: imports\n",
      "  - jsonlite, type: imports\n",
      "  - data.table, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - knitr, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "AGPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/TnT.html\n",
      "\n",
      "Name: TransView\n",
      "Description:\n",
      "This package provides efficient tools to generate, access and display read densities of sequencing based data sets such as from RNA-Seq and ChIP-Seq.\n",
      "Version: 1.42.0\n",
      "Publication date: None\n",
      "Mantainer: Julius Muller <ju-mu@alumni.ethz.ch>\n",
      "Authors:\n",
      "Author: Julius Muller\n",
      "Dependencies:\n",
      "  - methods, type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - BiocGenerics, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - zlibbioc, type: imports\n",
      "  - gplots, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/TransView.html\n",
      "\n",
      "Name: triplex\n",
      "Description:\n",
      "This package provides functions for identification and visualization of potential intramolecular triplex patterns in DNA sequence. The main functionality is to detect the positions of subsequences capable of folding into an intramolecular triplex (H-DNA) in a much larger sequence. The potential H-DNA (triplexes) should be made of as many cannonical nucleotide triplets as possible. The package includes visualization showing the exact base-pairing in 1D, 2D or 3D.\n",
      "Version: 1.38.0\n",
      "Publication date: None\n",
      "Mantainer: Jiri Hon <jiri.hon@gmail.com>\n",
      "Authors:\n",
      "Author: Jiri Hon, Matej Lexa, Tomas Martinek and Kamil Rajdl with contributions from Daniel Kopecek\n",
      "Dependencies:\n",
      "  - R (>= 2.15.0) type: depends\n",
      "  - S4Vectors, type: depends\n",
      "  - IRanges, type: depends\n",
      "  - XVector, type: depends\n",
      "  - Biostrings, type: depends\n",
      "  - methods, type: imports\n",
      "  - grid, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "BSD_2_clause + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/triplex.html\n",
      "\n",
      "Name: twoddpcr\n",
      "Description:\n",
      "The twoddpcr package takes Droplet Digital PCR (ddPCR) droplet amplitude data from Bio-Rad's QuantaSoft and can classify the droplets. A summary of the positive/negative droplet counts can be generated, which can then be used to estimate the number of molecules using the Poisson distribution. This is the first open source package that facilitates the automatic classification of general two channel ddPCR data. Previous work includes 'definetherain' (Jones et al., 2014) and 'ddpcRquant' (Trypsteen et al., 2015) which both handle one channel ddPCR experiments only. The 'ddpcr' package available on CRAN (Attali et al., 2016) supports automatic gating of a specific class of two channel ddPCR experiments only.\n",
      "Version: 1.22.0\n",
      "Publication date: None\n",
      "Mantainer: Anthony Chiu <anthony@achiu.me>\n",
      "Authors:\n",
      "Author: Anthony Chiu [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 3.4) type: depends\n",
      "  - class, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - hexbin, type: imports\n",
      "  - methods, type: imports\n",
      "  - scales, type: imports\n",
      "  - shiny, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/twoddpcr.html\n",
      "\n",
      "Name: uSORT\n",
      "Description:\n",
      "This package is designed to uncover the intrinsic cell progression path from single-cell RNA-seq data. It incorporates data pre-processing, preliminary PCA gene selection, preliminary cell ordering, feature selection, refined cell ordering, and post-analysis interpretation and visualization.\n",
      "Version: 1.24.0\n",
      "Publication date: None\n",
      "Mantainer: Hao Chen <chen_hao@immunol.a-star.edu.sg>\n",
      "Authors:\n",
      "Author: Mai Chan Lau, Hao Chen, Jinmiao Chen\n",
      "Dependencies:\n",
      "  - R (>= 3.3.0) type: depends\n",
      "  - tcltk, type: depends\n",
      "  - igraph, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - RANN, type: imports\n",
      "  - RSpectra, type: imports\n",
      "  - VGAM, type: imports\n",
      "  - gplots, type: imports\n",
      "  - parallel, type: imports\n",
      "  - plyr, type: imports\n",
      "  - methods, type: imports\n",
      "  - cluster, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - fpc, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - monocle, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/uSORT.html\n",
      "\n",
      "Name: XNAString\n",
      "Description:\n",
      "The XNAString package allows for description of base sequences and associated chemical modifications in a single object. XNAString is able to capture single stranded, as well as double stranded molecules. Chemical modifications are represented as independent strings associated with different features of the molecules (base sequence, sugar sequence, backbone sequence, modifications) and can be read or written to a HELM notation. It also enables secondary structure prediction using RNAfold from ViennaRNA. XNAString is designed to be efficient representation of nucleic-acid based therapeutics, therefore it stores information about target sequences and provides interface for matching and alignment functions from Biostrings package.\n",
      "Version: 1.6.0\n",
      "Publication date: None\n",
      "Mantainer: Marianna Plucinska <marianna.plucinska@roche.com>\n",
      "Authors:\n",
      "Author: Anna GÃ³rska [aut], Marianna Plucinska [aut, cre], Lykke Pedersen [aut], Lukasz Kielpinski [aut], Disa Tehler [aut], Peter H. Hagedorn [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - utils, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - BSgenome, type: imports\n",
      "  - data.table, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - methods, type: imports\n",
      "  - Rcpp, type: imports\n",
      "  - stringi, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - future.apply, type: imports\n",
      "  - stringr, type: imports\n",
      "  - formattable, type: imports\n",
      "  - stats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/XNAString.html\n",
      "\n",
      "Name: AnVILBilling\n",
      "Description:\n",
      "AnVILBilling helps monitor AnVIL-related costs in R, using queries to a BigQuery table to which costs are exported daily. Functions are defined to help categorize tasks and associated expenditures, and to visualize and explore expense profiles over time. This package will be expanded to help users estimate costs for specific task sets.\n",
      "Version: 1.8.0\n",
      "Publication date: None\n",
      "Mantainer: Vince Carey <stvjc@channing.harvard.edu>\n",
      "Authors:\n",
      "Author: BJ Stubbs [aut], Vince Carey [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - methods, type: imports\n",
      "  - DT, type: imports\n",
      "  - shiny, type: imports\n",
      "  - bigrquery, type: imports\n",
      "  - shinytoastr, type: imports\n",
      "  - DBI, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - lubridate, type: imports\n",
      "  - plotly, type: imports\n",
      "  - ggplot2, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/AnVILBilling.html\n",
      "\n",
      "Name: BindingSiteFinder\n",
      "Description:\n",
      "Precise knowledge on the binding sites of an RNA-binding protein (RBP) is key to understand (post-) transcriptional regulatory processes. Here we present a workflow that describes how exact binding sites can be defined from iCLIP data. The package provides functions for binding site definition and result visualization. For details please see the vignette.\n",
      "Version: 1.4.0\n",
      "Publication date: None\n",
      "Mantainer: Mirko BrÃ¼ggemann <mirko.brueggemann@mail.de>\n",
      "Authors:\n",
      "Author: Mirko BrÃ¼ggemann [aut, cre] , Kathi Zarnack [aut]\n",
      "Dependencies:\n",
      "  - GenomicRanges, type: depends\n",
      "  - R (>= 4.2) type: depends\n",
      "  - tidyr, type: imports\n",
      "  - tibble, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - plyr, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - stats, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - methods, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - ggforce, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - Gviz, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/BindingSiteFinder.html\n",
      "\n",
      "Name: bridge\n",
      "Description:\n",
      "Bioconductor version: Release (3.16)\n",
      "Version: 1.62.0\n",
      "Publication date: None\n",
      "Mantainer: Author: Raphael Gottardo\n",
      "Authors:\n",
      "Test for differentially expressed genes with microarray data. This package can be used with both cDNA microarrays or Affymetrix chip. The packge fits a robust Bayesian hierarchical model for testing for differential expression. Outliers are modeled explicitly using a $t$-distribution. The model includes an exchangeable prior for the variances which allow different variances for the genes but still shrink extreme empirical variances. Our model can be used for testing for differentially expressed genes among multiple samples, and can distinguish between the different possible patterns of differential expression when there are three or more samples. Parameter estimation is carried out using a novel version of Markov Chain Monte Carlo that is appropriate when the model puts mass on subspaces of the full parameter space.\n",
      "Dependencies:\n",
      "  - R (>= 1.9.0) type: depends\n",
      "  - rama, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/bridge.html\n",
      "\n",
      "Name: BUScorrect\n",
      "Description:\n",
      "High-throughput experimental data are accumulating exponentially in public databases. However, mining valid scientific discoveries from these abundant resources is hampered by technical artifacts and inherent biological heterogeneity. The former are usually termed \"batch effects,\" and the latter is often modelled by \"subtypes.\" The R package BUScorrect fits a Bayesian hierarchical model, the Batch-effects-correction-with-Unknown-Subtypes model (BUS), to correct batch effects in the presence of unknown subtypes. BUS is capable of (a) correcting batch effects explicitly, (b) grouping samples that share similar characteristics into subtypes, (c) identifying features that distinguish subtypes, and (d) enjoying a linear-order computation complexity.\n",
      "Version: 1.16.0\n",
      "Publication date: None\n",
      "Mantainer: Xiangyu Luo <xyluo1991@gmail.com>\n",
      "Authors:\n",
      "Author: Xiangyu Luo <xyluo1991 at gmail.com>, Yingying Wei <yweicuhk at gmail.com>\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - gplots, type: imports\n",
      "  - methods, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - stats, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/BUScorrect.html\n",
      "\n",
      "Name: calm\n",
      "Description:\n",
      "Statistical methods for multiple testing with covariate information. Traditional multiple testing methods only consider a list of test statistics, such as p-values. Our methods incorporate the auxiliary information, such as the lengths of gene coding regions or the minor allele frequencies of SNPs, to improve power.\n",
      "Version: 1.12.0\n",
      "Publication date: None\n",
      "Mantainer: Kun Liang <kun.liang@uwaterloo.ca>\n",
      "Authors:\n",
      "Author: Kun Liang [aut, cre]\n",
      "Dependencies:\n",
      "  - mgcv, type: imports\n",
      "  - stats, type: imports\n",
      "  - graphics, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>=2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/calm.html\n",
      "\n",
      "Name: ChIPXpress\n",
      "Description:\n",
      "ChIPXpress takes as input predicted TF bound genes from ChIPx data and uses a corresponding database of gene expression profiles downloaded from NCBI GEO to rank the TF bound targets in order of which gene is most likely to be functional TF target.\n",
      "Version: 1.42.0\n",
      "Publication date: None\n",
      "Mantainer: George Wu <georgetwu@gmail.com>\n",
      "Authors:\n",
      "Author: George Wu\n",
      "Dependencies:\n",
      "  - R (>= 2.10) type: depends\n",
      "  - ChIPXpressData, type: depends\n",
      "  - Biobase, type: imports\n",
      "  - GEOquery, type: imports\n",
      "  - frma, type: imports\n",
      "  - affy, type: imports\n",
      "  - bigmemory, type: imports\n",
      "  - biganalytics, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL(>=2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ChIPXpress.html\n",
      "\n",
      "Name: chromDraw\n",
      "Description:\n",
      "ChromDraw is a R package for drawing the schemes of karyotype(s) in the linear and circular fashion. It is possible to visualized cytogenetic marsk on the chromosomes. This tool has own input data format. Input data can be imported from the GenomicRanges data structure. This package can visualized the data in the BED file format. Here is requirement on to the first nine fields of the BED format. Output files format are *.eps and *.svg.\n",
      "Version: 2.28.0\n",
      "Publication date: None\n",
      "Mantainer: Jan Janecka <jan.janecka@ceitec.muni.cz>\n",
      "Authors:\n",
      "Author: Jan Janecka, Ing., Mgr. CEITEC Masaryk University\n",
      "Dependencies:\n",
      "  - R (>= 3.0.0) type: depends\n",
      "  - Rcpp (>= 0.11.1) type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/chromDraw.html\n",
      "\n",
      "Name: CNVrd2\n",
      "Description:\n",
      "CNVrd2 uses next-generation sequencing data to measure human gene copy number for multiple samples, indentify SNPs tagging copy number variants and detect copy number polymorphic genomic regions.\n",
      "Version: 1.36.0\n",
      "Publication date: None\n",
      "Mantainer: Hoang Tan Nguyen <hoangtannguyenvn@gmail.com>\n",
      "Authors:\n",
      "Author: Hoang Tan Nguyen, Tony R Merriman and Mik Black\n",
      "Dependencies:\n",
      "  - R (>= 3.0.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - VariantAnnotation, type: depends\n",
      "  - parallel, type: depends\n",
      "  - rjags, type: depends\n",
      "  - ggplot2, type: depends\n",
      "  - gridExtra, type: depends\n",
      "  - DNAcopy, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/CNVrd2.html\n",
      "\n",
      "Name: CoCiteStats\n",
      "Description:\n",
      "A collection of software tools for dealing with co-citation data.\n",
      "Version: 1.70.0\n",
      "Publication date: None\n",
      "Mantainer: Bioconductor Package Maintainer <maintainer@bioconductor.org>\n",
      "Authors:\n",
      "Author: B. Ding and R. Gentleman\n",
      "Dependencies:\n",
      "  - R (>= 2.0) type: depends\n",
      "  - org.Hs.eg.db, type: depends\n",
      "  - AnnotationDbi, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "CPL\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/CoCiteStats.html\n",
      "\n",
      "Name: covEB\n",
      "Description:\n",
      "Using bayesian methods to estimate correlation matrices assuming that they can be written and estimated as block diagonal matrices. These block diagonal matrices are determined using shrinkage parameters that values below this parameter to zero.\n",
      "Version: 1.24.0\n",
      "Publication date: None\n",
      "Mantainer: C. Pacini <clarepacini@gmail.com>\n",
      "Authors:\n",
      "Author: C. Pacini\n",
      "Dependencies:\n",
      "  - R (>= 3.3) type: depends\n",
      "  - mvtnorm, type: depends\n",
      "  - igraph, type: depends\n",
      "  - gsl, type: depends\n",
      "  - Biobase, type: depends\n",
      "  - stats, type: depends\n",
      "  - LaplacesDemon, type: depends\n",
      "  - Matrix, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/covEB.html\n",
      "\n",
      "Name: covRNA\n",
      "Description:\n",
      "This package provides the analysis methods fourthcorner and RLQ analysis for large-scale transcriptomic data.\n",
      "Version: 1.24.0\n",
      "Publication date: None\n",
      "Mantainer: Lara Urban <lara.h.urban@ebi.ac.uk>\n",
      "Authors:\n",
      "Author: Lara Urban <lara.h.urban at ebi.ac.uk>\n",
      "Dependencies:\n",
      "  - ade4, type: depends\n",
      "  - Biobase, type: depends\n",
      "  - parallel, type: imports\n",
      "  - genefilter, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - stats, type: imports\n",
      "  - graphics, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/covRNA.html\n",
      "\n",
      "Name: CTDquerier\n",
      "Description:\n",
      "Bioconductor version: Release (3.16)\n",
      "Version: 2.6.0\n",
      "Publication date: None\n",
      "Mantainer: Author: Carles Hernandez-Ferrer [aut], Juan R. Gonzalez [aut], Xavier EscribÃ -Montagut [cre]\n",
      "Authors:\n",
      "Package to retrieve and visualize data from the Comparative Toxicogenomics Database (http://ctdbase.org/). The downloaded data is formated as DataFrames for further downstream analyses.\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - RCurl, type: imports\n",
      "  - stringr, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - stringdist, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - igraph, type: imports\n",
      "  - utils, type: imports\n",
      "  - grid, type: imports\n",
      "  - gridExtra, type: imports\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - BiocFileCache, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/CTDquerier.html\n",
      "\n",
      "Name: DMRforPairs\n",
      "Description:\n",
      "DMRforPairs (formerly DMR2+) allows researchers to compare n>=2 unique samples with regard to their methylation profile. The (pairwise) comparison of n unique single samples distinguishes DMRforPairs from other existing pipelines as these often compare groups of samples in either single CpG locus or region based analysis. DMRforPairs defines regions of interest as genomic ranges with sufficient probes located in close proximity to each other. Probes in one region are optionally annotated to the same functional class(es). Differential methylation is evaluated by comparing the methylation values within each region between individual samples and (if the difference is sufficiently large), testing this difference formally for statistical significance.\n",
      "Version: 1.34.0\n",
      "Publication date: None\n",
      "Mantainer: Martin Rijlaarsdam <m.a.rijlaarsdam@gmail.com>\n",
      "Authors:\n",
      "Author: Martin Rijlaarsdam [aut, cre], Yvonne vd Zwan [aut], Lambert Dorssers [aut], Leendert Looijenga [aut]\n",
      "Dependencies:\n",
      "  - R (>= 2.15.2) type: depends\n",
      "  - Gviz, type: depends\n",
      "  - R2HTML (>= 2.2.1) type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - parallel, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/DMRforPairs.html\n",
      "\n",
      "Name: doseR\n",
      "Description:\n",
      "doseR package is a next generation sequencing package for sex chromosome dosage compensation which can be applied broadly to detect shifts in gene expression among an arbitrary number of pre-defined groups of loci. doseR is a differential gene expression package for count data, that detects directional shifts in expression for multiple, specific subsets of genes, broad utility in systems biology research. doseR has been prepared to manage the nature of the data and the desired set of inferences. doseR uses S4 classes to store count data from sequencing experiment. It contains functions to normalize and filter count data, as well as to plot and calculate statistics of count data. It contains a framework for linear modeling of count data. The package has been tested using real and simulated data.\n",
      "Version: 1.14.0\n",
      "Publication date: None\n",
      "Mantainer: ake.vastermark <ake.vastermark@gmail.com>\n",
      "Authors:\n",
      "Author: AJ Vaestermark, JR Walters.\n",
      "Dependencies:\n",
      "  - R (>= 3.6) type: depends\n",
      "  - edgeR, type: imports\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - graphics, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - mclust, type: imports\n",
      "  - lme4, type: imports\n",
      "  - RUnit, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - digest, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/doseR.html\n",
      "\n",
      "Name: dpeak\n",
      "Description:\n",
      "dPeak is a statistical framework for the high resolution identification of protein-DNA interaction sites using PET and SET ChIP-Seq and ChIP-exo data. It provides computationally efficient and user friendly interface to process ChIP-seq and ChIP-exo data, implement exploratory analysis, fit dPeak model, and export list of predicted binding sites for downstream analysis.\n",
      "Version: 1.10.0\n",
      "Publication date: None\n",
      "Mantainer: Dongjun Chung <dongjun.chung@gmail.com>\n",
      "Authors:\n",
      "Author: Dongjun Chung, Carter Allen\n",
      "Dependencies:\n",
      "  - R (>= 4.0.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - stats, type: depends\n",
      "  - utils, type: depends\n",
      "  - graphics, type: depends\n",
      "  - Rcpp, type: depends\n",
      "  - MASS, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - BSgenome, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - parallel, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/dpeak.html\n",
      "\n",
      "Name: DTA\n",
      "Description:\n",
      "Dynamic Transcriptome Analysis (DTA) can monitor the cellular response to perturbations with higher sensitivity and temporal resolution than standard transcriptomics. The package implements the underlying kinetic modeling approach capable of the precise determination of synthesis- and decay rates from individual microarray or RNAseq measurements.\n",
      "Version: 2.44.0\n",
      "Publication date: None\n",
      "Mantainer: Bjoern Schwalb <schwalb@lmb.uni-muenchen.de>\n",
      "Authors:\n",
      "Author: Bjoern Schwalb, Benedikt Zacher, Sebastian Duemcke, Achim Tresch\n",
      "Dependencies:\n",
      "  - R (>= 2.10) type: depends\n",
      "  - LSD, type: depends\n",
      "  - scatterplot3d, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/DTA.html\n",
      "\n",
      "Name: evaluomeR\n",
      "Description:\n",
      "Evaluating the reliability of your own metrics and the measurements done on your own datasets by analysing the stability and goodness of the classifications of such metrics.\n",
      "Version: 1.14.0\n",
      "Publication date: None\n",
      "Mantainer: JosÃ© Antonio BernabÃ©-DÃ­az <joseantonio.bernabe1@um.es>\n",
      "Authors:\n",
      "Author: JosÃ© Antonio BernabÃ©-DÃ­az [aut, cre], Manuel Franco [aut], Juana-MarÃ­a Vivo [aut], Manuel Quesada-MartÃ­nez [aut], Astrid Duque-Ramos [aut], Jesualdo TomÃ¡s FernÃ¡ndez-Breis [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.6) type: depends\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - MultiAssayExperiment, type: depends\n",
      "  - cluster (>= 2.0.9) type: depends\n",
      "  - fpc (>= 2.2-3) type: depends\n",
      "  - randomForest (>= 4.6.14) type: depends\n",
      "  - flexmix (>= 2.3.15) type: depends\n",
      "  - corrplot (>= 0.84) type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - ggdendro, type: imports\n",
      "  - plotrix, type: imports\n",
      "  - stats, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - Rdpack, type: imports\n",
      "  - MASS, type: imports\n",
      "  - class, type: imports\n",
      "  - prabclus, type: imports\n",
      "  - mclust, type: imports\n",
      "  - kableExtra, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/evaluomeR.html\n",
      "\n",
      "Name: fastLiquidAssociation\n",
      "Description:\n",
      "This package extends the function of the LiquidAssociation package for genome-wide application. It integrates a screening method into the LA analysis to reduce the number of triplets to be examined for a high LA value and provides code for use in subsequent significance analyses.\n",
      "Version: 1.34.0\n",
      "Publication date: None\n",
      "Mantainer: Tina Gunderson <gunderson.tina@gmail.com>\n",
      "Authors:\n",
      "Author: Tina Gunderson\n",
      "Dependencies:\n",
      "  - methods, type: depends\n",
      "  - LiquidAssociation, type: depends\n",
      "  - parallel, type: depends\n",
      "  - doParallel, type: depends\n",
      "  - stats, type: depends\n",
      "  - Hmisc, type: depends\n",
      "  - utils, type: depends\n",
      "  - WGCNA, type: imports\n",
      "  - impute, type: imports\n",
      "  - preprocessCore, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/fastLiquidAssociation.html\n",
      "\n",
      "Name: fcScan\n",
      "Description:\n",
      "This package is used to detect combination of genomic coordinates falling within a user defined window size along with user defined overlap between identified neighboring clusters. It can be used for genomic data where the clusters are built on a specific chromosome or specific strand. Clustering can be performed with a \"greedy\" option allowing thus the presence of additional sites within the allowed window size.\n",
      "Version: 1.12.0\n",
      "Publication date: None\n",
      "Mantainer: Pierre Khoueiry <pk17@aub.edu.lb> Abdullah El-Kurdi <ak161@aub.edu.lb>\n",
      "Authors:\n",
      "Author: Abdullah El-Kurdi [aut], Ghiwa khalil [aut], Georges Khazen [ctb], Pierre Khoueiry [aut, cre]\n",
      "Dependencies:\n",
      "  - stats, type: imports\n",
      "  - plyr, type: imports\n",
      "  - VariantAnnotation, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - methods, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - foreach, type: imports\n",
      "  - doParallel, type: imports\n",
      "  - parallel, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/fcScan.html\n",
      "\n",
      "Name: FindIT2\n",
      "Description:\n",
      "This package implements functions to find influential TF and target based on different input type. It have five module: Multi-peak multi-gene annotaion(mmPeakAnno module), Calculate regulation potential(calcRP module), Find influential Target based on ChIP-Seq and RNA-Seq data(Find influential Target module), Find influential TF based on different input(Find influential TF module), Calculate peak-gene or peak-peak correlation(peakGeneCor module). And there are also some other useful function like integrate different source information, calculate jaccard similarity for your TF.\n",
      "Version: 1.4.0\n",
      "Publication date: None\n",
      "Mantainer: Guandong Shang <shangguandong1996@163.com>\n",
      "Authors:\n",
      "Author: Guandong Shang [aut, cre]\n",
      "Dependencies:\n",
      "  - GenomicRanges, type: depends\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - withr, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - GenomicFeatures, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - rlang, type: imports\n",
      "  - patchwork, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - qvalue, type: imports\n",
      "  - stringr, type: imports\n",
      "  - utils, type: imports\n",
      "  - stats, type: imports\n",
      "  - ggrepel, type: imports\n",
      "  - tibble, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - MultiAssayExperiment, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - progress, type: imports\n",
      "  - purrr, type: imports\n",
      "  - glmnet, type: imports\n",
      "  - methods, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/FindIT2.html\n",
      "\n",
      "Name: flowBeads\n",
      "Description:\n",
      "This package extends flowCore to provide functionality specific to bead data. One of the goals of this package is to automate analysis of bead data for the purpose of normalisation.\n",
      "Version: 1.36.0\n",
      "Publication date: None\n",
      "Mantainer: Nikolas Pontikos <n.pontikos@gmail.com>\n",
      "Authors:\n",
      "Author: Nikolas Pontikos\n",
      "Dependencies:\n",
      "  - R (>= 2.15.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - Biobase, type: depends\n",
      "  - rrcov, type: depends\n",
      "  - flowCore, type: depends\n",
      "  - flowCore, type: imports\n",
      "  - rrcov, type: imports\n",
      "  - knitr, type: imports\n",
      "  - xtable, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/flowBeads.html\n",
      "\n",
      "Name: flowCHIC\n",
      "Description:\n",
      "A package to analyze flow cytometric data of complex microbial communities based on histogram images\n",
      "Version: 1.32.0\n",
      "Publication date: None\n",
      "Mantainer: Author: Joachim Schumann <joachim.schumann@ufz.de>\n",
      "Authors:\n",
      "Author: Joachim Schumann <joachim.schumann at ufz.de>, Christin Koch <christin.koch at ufz.de>, Ingo Fetzer <info.fetzer at stockholmresilience.su.se>, Susann MÃ¼ller <susann.mueller at ufz.de>\n",
      "Dependencies:\n",
      "  - R (>= 3.1.0) type: depends\n",
      "  - methods, type: imports\n",
      "  - flowCore, type: imports\n",
      "  - EBImage, type: imports\n",
      "  - vegan, type: imports\n",
      "  - hexbin, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - grid, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/flowCHIC.html\n",
      "\n",
      "Name: fobitools\n",
      "Description:\n",
      "A set of tools for interacting with Food-Biomarker Ontology (FOBI). A collection of basic manipulation tools for biological significance analysis, graphs, and text mining strategies for annotating nutritional data.\n",
      "Version: 1.6.0\n",
      "Publication date: None\n",
      "Mantainer: Pol Castellano-Escuder <polcaes@gmail.com>\n",
      "Authors:\n",
      "Author: Pol Castellano-Escuder [aut, cre] , Cristina AndrÃ©s-Lacueva [aut] , Alex SÃ¡nchez-Pla [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - clisymbols, type: imports\n",
      "  - crayon, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - fgsea, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - ggraph, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - ontologyIndex, type: imports\n",
      "  - purrr, type: imports\n",
      "  - RecordLinkage, type: imports\n",
      "  - stringr, type: imports\n",
      "  - textclean, type: imports\n",
      "  - tictoc, type: imports\n",
      "  - tidygraph, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - vroom, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/fobitools.html\n",
      "\n",
      "Name: FunChIP\n",
      "Description:\n",
      "Preprocessing and smoothing of ChIP-Seq peaks and efficient implementation of the k-mean alignment algorithm to classify them.\n",
      "Version: 1.24.0\n",
      "Publication date: None\n",
      "Mantainer: Alice Parodi <alicecarla.parodi@polimi.it>\n",
      "Authors:\n",
      "Author: Alice Parodi [aut, cre], Marco Morelli [aut, cre], Laura M. Sangalli [aut], Piercesare Secchi [aut], Simone Vantini [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - shiny, type: imports\n",
      "  - fda, type: imports\n",
      "  - doParallel, type: imports\n",
      "  - GenomicAlignments, type: imports\n",
      "  - Rcpp, type: imports\n",
      "  - methods, type: imports\n",
      "  - foreach, type: imports\n",
      "  - parallel, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/FunChIP.html\n",
      "\n",
      "Name: gemini\n",
      "Description:\n",
      "GEMINI uses log-fold changes to model sample-dependent and independent effects, and uses a variational Bayes approach to infer these effects. The inferred effects are used to score and identify genetic interactions, such as lethality and recovery. More details can be found in Zamanighomi et al. 2019 (in press).\n",
      "Version: 1.12.0\n",
      "Publication date: None\n",
      "Mantainer: Sidharth Jain <sidharthsjain@gmail.com>\n",
      "Authors:\n",
      "Author: Mahdi Zamanighomi [aut], Sidharth Jain [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.1.0) type: depends\n",
      "  - dplyr, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - mixtools, type: imports\n",
      "  - scales, type: imports\n",
      "  - pbmcapply, type: imports\n",
      "  - parallel, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "BSD_3_clause + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/gemini.html\n",
      "\n",
      "Name: GenomicTuples\n",
      "Description:\n",
      "GenomicTuples defines general purpose containers for storing genomic tuples. It aims to provide functionality for tuples of genomic co-ordinates that are analogous to those available for genomic ranges in the GenomicRanges Bioconductor package.\n",
      "Version: 1.32.0\n",
      "Publication date: None\n",
      "Mantainer: Peter Hickey <peter.hickey@gmail.com>\n",
      "Authors:\n",
      "Author: Peter Hickey [aut, cre], Marcin Cieslik [ctb], HervÃ© PagÃ¨s [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - GenomeInfoDb, type: depends\n",
      "  - S4Vectors, type: depends\n",
      "  - methods, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - Rcpp (>= 0.11.2) type: imports\n",
      "  - IRanges, type: imports\n",
      "  - data.table, type: imports\n",
      "  - stats4, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/GenomicTuples.html\n",
      "\n",
      "Name: GWAS.BAYES\n",
      "Description:\n",
      "This package is built to perform GWAS analysis for selfing species. The research related to this package was supported in part by National Science Foundation Award 1853549.\n",
      "Version: 1.8.0\n",
      "Publication date: None\n",
      "Mantainer: Jake Williams <jwilliams@vt.edu>\n",
      "Authors:\n",
      "Author: Jake Williams [aut, cre], Marco Ferreira [aut], Tieming Ji [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - Rcpp (>= 1.0.3) type: depends\n",
      "  - RcppEigen (>= 0.3.3.7.0) type: depends\n",
      "  - GA (>= 3.2) type: depends\n",
      "  - caret (>= 6.0-86) type: depends\n",
      "  - ggplot2 (>= 3.3.0) type: depends\n",
      "  - doParallel (>= 1.0.15) type: depends\n",
      "  - memoise (>= 1.1.0) type: depends\n",
      "  - reshape2 (>= 1.4.4) type: depends\n",
      "  - Matrix (>= 1.2-18) type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2 | GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/GWAS.BAYES.html\n",
      "\n",
      "Name: HELP\n",
      "Description:\n",
      "The package contains a modular pipeline for analysis of HELP microarray data, and includes graphical and mathematical tools with more general applications.\n",
      "Version: 1.56.0\n",
      "Publication date: None\n",
      "Mantainer: Reid F. Thompson <reid.thompson@gmail.com>\n",
      "Authors:\n",
      "Author: Reid F. Thompson <reid.thompson at gmail.com>, John M. Greally <john.greally at einstein.yu.edu>, with contributions from Mark Reimers <mreimers at vcu.edu>\n",
      "Dependencies:\n",
      "  - R (>= 2.8.0) type: depends\n",
      "  - stats, type: depends\n",
      "  - graphics, type: depends\n",
      "  - grDevices, type: depends\n",
      "  - Biobase, type: depends\n",
      "  - methods, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/HELP.html\n",
      "\n",
      "Name: HIPPO\n",
      "Description:\n",
      "For scRNA-seq data, it selects features and clusters the cells simultaneously for single-cell UMI data. It has a novel feature selection method using the zero inflation instead of gene variance, and computationally faster than other existing methods since it only relies on PCA+Kmeans rather than graph-clustering or consensus clustering.\n",
      "Version: 1.10.0\n",
      "Publication date: None\n",
      "Mantainer: Tae Kim <tk382@uchicago.edu>\n",
      "Authors:\n",
      "Author: Tae Kim [aut, cre], Mengjie Chen [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.6.0) type: depends\n",
      "  - ggplot2, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - gridExtra, type: imports\n",
      "  - Rtsne, type: imports\n",
      "  - umap, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - rlang, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - irlba, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - SingleCellExperiment, type: imports\n",
      "  - ggrepel, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>=2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/HIPPO.html\n",
      "\n",
      "Name: HIREewas\n",
      "Description:\n",
      "In epigenome-wide association studies, the measured signals for each sample are a mixture of methylation profiles from different cell types. The current approaches to the association detection only claim whether a cytosine-phosphate-guanine (CpG) site is associated with the phenotype or not, but they cannot determine the cell type in which the risk-CpG site is affected by the phenotype. We propose a solid statistical method, HIgh REsolution (HIRE), which not only substantially improves the power of association detection at the aggregated level as compared to the existing methods but also enables the detection of risk-CpG sites for individual cell types. The \"HIREewas\" R package is to implement HIRE model in R.\n",
      "Version: 1.16.0\n",
      "Publication date: None\n",
      "Mantainer: Xiangyu Luo <xyluo1991@gmail.com>\n",
      "Authors:\n",
      "Author: Xiangyu Luo <xyluo1991 at gmail.com>, Can Yang <macyang at ust.hk>, Yingying Wei <yweicuhk at gmail.com>\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - quadprog, type: imports\n",
      "  - gplots, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - stats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/HIREewas.html\n",
      "\n",
      "Name: HumanTranscriptomeCompendium\n",
      "Description:\n",
      "Provide tools for working with a compendium of human transcriptome sequences (originally htxcomp).\n",
      "Version: 1.14.0\n",
      "Publication date: None\n",
      "Mantainer: VJ Carey <stvjc@channing.harvard.edu>\n",
      "Authors:\n",
      "Author: Sean Davis, Vince Carey\n",
      "Dependencies:\n",
      "  - R (>= 3.6) type: depends\n",
      "  - shiny, type: imports\n",
      "  - ssrch, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/HumanTranscriptomeCompendium.html\n",
      "\n",
      "Name: INPower\n",
      "Description:\n",
      "An R package for computing the number of susceptibility SNPs and power of future studies\n",
      "Version: 1.34.0\n",
      "Publication date: None\n",
      "Mantainer: Bill Wheeler <wheelerb@imsweb.com>\n",
      "Authors:\n",
      "Author: Ju-Hyun Park\n",
      "Dependencies:\n",
      "  - R (>= 3.1.0) type: depends\n",
      "  - mvtnorm, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2 + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/INPower.html\n",
      "\n",
      "Name: InTAD\n",
      "Description:\n",
      "The package is focused on the detection of correlation between expressed genes and selected epigenomic signals (i.e. enhancers obtained from ChIP-seq data) either within topologically associated domains (TADs) or between chromatin contact loop anchors. Various parameters can be controlled to investigate the influence of external factors and visualization plots are available for each analysis step.\n",
      "Version: 1.18.0\n",
      "Publication date: None\n",
      "Mantainer: Konstantin Okonechnikov <k.okonechnikov@gmail.com>\n",
      "Authors:\n",
      "Author: Konstantin Okonechnikov, Serap Erkek, Lukas Chavez\n",
      "Dependencies:\n",
      "  - R (>= 3.5) type: depends\n",
      "  - methods, type: depends\n",
      "  - S4Vectors, type: depends\n",
      "  - IRanges, type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - MultiAssayExperiment, type: depends\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - stats, type: depends\n",
      "  - BiocGenerics, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - parallel, type: imports\n",
      "  - graphics, type: imports\n",
      "  - mclust, type: imports\n",
      "  - qvalue, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - utils, type: imports\n",
      "  - ggpubr, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>=2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/InTAD.html\n",
      "\n",
      "Name: ISoLDE\n",
      "Description:\n",
      "This package provides ISoLDE a new method for identifying imprinted genes. This method is dedicated to data arising from RNA sequencing technologies. The ISoLDE package implements original statistical methodology described in the publication below.\n",
      "Version: 1.26.0\n",
      "Publication date: None\n",
      "Mantainer: Christelle ReynÃ¨s <christelle.reynes@igf.cnrs.fr>\n",
      "Authors:\n",
      "Author: Christelle ReynÃ¨s [aut, cre], Marine Rohmer [aut], Guilhem Kister [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.3.0) type: depends\n",
      "  - graphics, type: depends\n",
      "  - grDevices, type: depends\n",
      "  - stats, type: depends\n",
      "  - utils, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2.0)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ISoLDE.html\n",
      "\n",
      "Name: kissDE\n",
      "Description:\n",
      "Retrieves condition-specific variants in RNA-seq data (SNVs, alternative-splicings, indels). It has been developed as a post-treatment of 'KisSplice' but can also be used with user's own data.\n",
      "Version: 1.18.0\n",
      "Publication date: None\n",
      "Mantainer: AurÃ©lie Siberchicot <aurelie.siberchicot@univ-lyon1.fr>\n",
      "Authors:\n",
      "Author: Clara Benoit-Pilven [aut], Camille Marchet [aut], Janice Kielbassa [aut], Lilia Brinza [aut], Audric Cologne [aut], AurÃ©lie Siberchicot [aut, cre], Vincent Lacroix [aut], Frank Picard [ctb], Laurent Jacob [ctb], Vincent Miele [ctb]\n",
      "Dependencies:\n",
      "  - aods3, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - DESeq2, type: imports\n",
      "  - DSS, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - gplots, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - foreach, type: imports\n",
      "  - doParallel, type: imports\n",
      "  - parallel, type: imports\n",
      "  - shiny, type: imports\n",
      "  - shinycssloaders, type: imports\n",
      "  - ade4, type: imports\n",
      "  - factoextra, type: imports\n",
      "  - DT, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/kissDE.html\n",
      "\n",
      "Name: lapmix\n",
      "Description:\n",
      "Laplace mixture modelling of microarray experiments. A hierarchical Bayesian approach is used, and the hyperparameters are estimated using empirical Bayes. The main purpose is to identify differentially expressed genes.\n",
      "Version: 1.64.0\n",
      "Publication date: None\n",
      "Mantainer: Yann Ruffieux <yann.ruffieux@epfl.ch>\n",
      "Authors:\n",
      "Author: Yann Ruffieux, contributions from Debjani Bhowmick, Anthony C. Davison, and Darlene R. Goldstein\n",
      "Dependencies:\n",
      "  - R (>= 2.6.0) type: depends\n",
      "  - stats, type: depends\n",
      "  - Biobase, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - tools, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/lapmix.html\n",
      "\n",
      "Name: maCorrPlot\n",
      "Description:\n",
      "Graphically displays correlation in microarray data that is due to insufficient normalization\n",
      "Version: 1.68.0\n",
      "Publication date: None\n",
      "Mantainer: Alexander Ploner <Alexander.Ploner@ki.se>\n",
      "Authors:\n",
      "Author: Alexander Ploner <Alexander.Ploner at ki.se>\n",
      "Dependencies:\n",
      "  - lattice, type: depends\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - lattice, type: imports\n",
      "  - stats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/maCorrPlot.html\n",
      "\n",
      "Name: MAI\n",
      "Description:\n",
      "A two-step approach to imputing missing data in metabolomics. Step 1 uses a random forest classifier to classify missing values as either Missing Completely at Random/Missing At Random (MCAR/MAR) or Missing Not At Random (MNAR). MCAR/MAR are combined because it is often difficult to distinguish these two missing types in metabolomics data. Step 2 imputes the missing values based on the classified missing mechanisms, using the appropriate imputation algorithms. Imputation algorithms tested and available for MCAR/MAR include Bayesian Principal Component Analysis (BPCA), Multiple Imputation No-Skip K-Nearest Neighbors (Multi_nsKNN), and Random Forest. Imputation algorithms tested and available for MNAR include nsKNN and a single imputation approach for imputation of metabolites where left-censoring is present.\n",
      "Version: 1.4.0\n",
      "Publication date: None\n",
      "Mantainer: Jonathan Dekermanjian <Jonathan.Dekermanjian@CUAnschutz.edu>\n",
      "Authors:\n",
      "Author: Jonathan Dekermanjian [aut, cre], Elin Shaddox [aut], Debmalya Nandy [aut], Debashis Ghosh [aut], Katerina Kechris [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - caret, type: imports\n",
      "  - parallel, type: imports\n",
      "  - doParallel, type: imports\n",
      "  - foreach, type: imports\n",
      "  - e1071, type: imports\n",
      "  - future.apply, type: imports\n",
      "  - future, type: imports\n",
      "  - missForest, type: imports\n",
      "  - pcaMethods, type: imports\n",
      "  - tidyverse, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - methods, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/MAI.html\n",
      "\n",
      "Name: mAPKL\n",
      "Description:\n",
      "We propose a hybrid FS method (mAP-KL), which combines multiple hypothesis testing and affinity propagation (AP)-clustering algorithm along with the Krzanowski & Lai cluster quality index, to select a small yet informative subset of genes.\n",
      "Version: 1.28.0\n",
      "Publication date: None\n",
      "Mantainer: Argiris Sakellariou <argisake@gmail.com>\n",
      "Authors:\n",
      "Author: Argiris Sakellariou\n",
      "Dependencies:\n",
      "  - R (>= 3.6.0) type: depends\n",
      "  - Biobase, type: depends\n",
      "  - multtest, type: imports\n",
      "  - clusterSim, type: imports\n",
      "  - apcluster, type: imports\n",
      "  - limma, type: imports\n",
      "  - e1071, type: imports\n",
      "  - AnnotationDbi, type: imports\n",
      "  - methods, type: imports\n",
      "  - parmigene, type: imports\n",
      "  - igraph, type: imports\n",
      "  - reactome.db, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/mAPKL.html\n",
      "\n",
      "Name: maskBAD\n",
      "Description:\n",
      "Package includes functions to analyze and mask microarray expression data.\n",
      "Version: 1.42.0\n",
      "Publication date: None\n",
      "Mantainer: Michael Dannemann <michael_dannemann@eva.mpg.de>\n",
      "Authors:\n",
      "Author: Michael Dannemann <michael_dannemann at eva.mpg.de>\n",
      "Dependencies:\n",
      "  - R (>= 2.10) type: depends\n",
      "  - gcrma, type: depends\n",
      "  - affy, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/maskBAD.html\n",
      "\n",
      "Name: MEAT\n",
      "Description:\n",
      "This package estimates epigenetic age in skeletal muscle, using DNA methylation data generated with the Illumina Infinium technology (HM27, HM450 and HMEPIC).\n",
      "Version: 1.10.0\n",
      "Publication date: None\n",
      "Mantainer: Sarah Voisin <sarah.voisin.aeris@gmail.com>\n",
      "Authors:\n",
      "Author: Sarah Voisin [aut, cre] , Steve Horvath [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - impute, type: imports\n",
      "  - dynamicTreeCut (>= 1.63) type: imports\n",
      "  - glmnet (>= 2.0) type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - stringr, type: imports\n",
      "  - tibble, type: imports\n",
      "  - RPMM (>= 1.25) type: imports\n",
      "  - minfi, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - wateRmelon, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/MEAT.html\n",
      "\n",
      "Name: metabCombiner\n",
      "Description:\n",
      "This package aligns LC-HRMS metabolomics datasets acquired from biologically similar specimens analyzed under similar, but not necessarily identical, conditions. Peak-picked and simply aligned metabolomics feature tables (consisting of m/z, rt, and per-sample abundance measurements, plus optional identifiers & adduct annotations) are accepted as input. The package outputs a combined table of feature pair alignments, organized into groups of similar m/z, and ranked by a similarity score. Input tables are assumed to be acquired using similar (but not necessarily identical) analytical methods.\n",
      "Version: 1.8.0\n",
      "Publication date: None\n",
      "Mantainer: Hani Habra <hhabra1@gmail.com>\n",
      "Authors:\n",
      "Author: Hani Habra [aut, cre], Alla Karnovsky [ths]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - dplyr (>= 1.0) type: depends\n",
      "  - methods, type: imports\n",
      "  - mgcv, type: imports\n",
      "  - caret, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - rlang, type: imports\n",
      "  - graphics, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - tidyr, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/metabCombiner.html\n",
      "\n",
      "Name: MIGSA\n",
      "Description:\n",
      "Massive and Integrative Gene Set Analysis. The MIGSA package allows to perform a massive and integrative gene set analysis over several expression and gene sets simultaneously. It provides a common gene expression analytic framework that grants a comprehensive and coherent analysis. Only a minimal user parameter setting is required to perform both singular and gene set enrichment analyses in an integrative manner by means of the best available methods, i.e. dEnricher and mGSZ respectively. The greatest strengths of this big omics data tool are the availability of several functions to explore, analyze and visualize its results in order to facilitate the data mining task over huge information sources. MIGSA package also provides several functions that allow to easily load the most updated gene sets from several repositories.\n",
      "Version: 1.21.0\n",
      "Publication date: None\n",
      "Mantainer: Juan C. Rodriguez <jcrodriguez@unc.edu.ar>\n",
      "Authors:\n",
      "Author: Juan C. Rodriguez, Cristobal Fresno, Andrea S. Llera and Elmer A. Fernandez\n",
      "Dependencies:\n",
      "  - R (>= 3.4) type: depends\n",
      "  - methods, type: depends\n",
      "  - BiocGenerics, type: depends\n",
      "  - AnnotationDbi, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - compiler, type: imports\n",
      "  - data.table, type: imports\n",
      "  - edgeR, type: imports\n",
      "  - futile.logger, type: imports\n",
      "  - ggdendro, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - GO.db, type: imports\n",
      "  - GOstats, type: imports\n",
      "  - graph, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - grid, type: imports\n",
      "  - GSEABase, type: imports\n",
      "  - ismev, type: imports\n",
      "  - jsonlite, type: imports\n",
      "  - limma, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - org.Hs.eg.db, type: imports\n",
      "  - RBGL, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - Rgraphviz, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - vegan, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/MIGSA.html\n",
      "\n",
      "Name: miRSM\n",
      "Description:\n",
      "The package aims to identify miRNA sponge modules in heterogeneous data. It provides several functions to study miRNA sponge modules, including popular methods for inferring gene modules (candidate miRNA sponge modules), and a function to identify miRNA sponge modules, as well as several functions to conduct modular analysis of miRNA sponge modules.\n",
      "Version: 1.16.0\n",
      "Publication date: None\n",
      "Mantainer: Junpeng Zhang <zhangjunpeng411@gmail.com>\n",
      "Authors:\n",
      "Author: Junpeng Zhang [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - WGCNA, type: imports\n",
      "  - flashClust, type: imports\n",
      "  - dynamicTreeCut, type: imports\n",
      "  - GFA, type: imports\n",
      "  - igraph, type: imports\n",
      "  - linkcomm, type: imports\n",
      "  - MCL, type: imports\n",
      "  - NMF, type: imports\n",
      "  - biclust, type: imports\n",
      "  - iBBiG, type: imports\n",
      "  - fabia, type: imports\n",
      "  - BicARE, type: imports\n",
      "  - isa2, type: imports\n",
      "  - s4vd, type: imports\n",
      "  - BiBitR, type: imports\n",
      "  - rqubic, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - PMA, type: imports\n",
      "  - stats, type: imports\n",
      "  - dbscan, type: imports\n",
      "  - subspace, type: imports\n",
      "  - mclust, type: imports\n",
      "  - SOMbrero, type: imports\n",
      "  - ppclust, type: imports\n",
      "  - miRspongeR, type: imports\n",
      "  - Rcpp, type: imports\n",
      "  - utils, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - GSEABase, type: imports\n",
      "  - org.Hs.eg.db, type: imports\n",
      "  - MatrixCorrelation, type: imports\n",
      "  - energy, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/miRSM.html\n",
      "\n",
      "Name: mirTarRnaSeq\n",
      "Description:\n",
      "mirTarRnaSeq R package can be used for interactive mRNA miRNA sequencing statistical analysis. This package utilizes expression or differential expression mRNA and miRNA sequencing results and performs interactive correlation and various GLMs (Regular GLM, Multivariate GLM, and Interaction GLMs ) analysis between mRNA and miRNA expriments. These experiments can be time point experiments, and or condition expriments.\n",
      "Version: 1.6.0\n",
      "Publication date: None\n",
      "Mantainer: Mercedeh Movassagh <mercedeh@ds.dfci.harvard.edu>\n",
      "Authors:\n",
      "Author: Mercedeh Movassagh [aut, cre] , Sarah Morton [aut], Rafael Irizarry [aut], Jeffrey Bailey [aut], Joseph N Paulson [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.1.0) type: depends\n",
      "  - ggplot2, type: depends\n",
      "  - purrr, type: imports\n",
      "  - MASS, type: imports\n",
      "  - pscl, type: imports\n",
      "  - assertthat, type: imports\n",
      "  - caTools, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - pheatmap, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - corrplot, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - data.table, type: imports\n",
      "  - R.utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/mirTarRnaSeq.html\n",
      "\n",
      "Name: MMDiff2\n",
      "Description:\n",
      "This package detects statistically significant differences between read enrichment profiles in different ChIP-Seq samples. To take advantage of shape differences it uses Kernel methods (Maximum Mean Discrepancy, MMD).\n",
      "Version: 1.26.0\n",
      "Publication date: None\n",
      "Mantainer: Gabriele Schweikert <gschweik@staffmail.ed.ac.uk>\n",
      "Authors:\n",
      "Author: Gabriele Schweikert [cre, aut], David Kuo [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - Rsamtools, type: depends\n",
      "  - Biobase, type: depends\n",
      "  - GenomicRanges, type: imports\n",
      "  - locfit, type: imports\n",
      "  - BSgenome, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - shiny, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - parallel, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - methods, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/MMDiff2.html\n",
      "\n",
      "Name: mosbi\n",
      "Description:\n",
      "This package is a implementation of biclustering ensemble method MoSBi (Molecular signature Identification from Biclustering). MoSBi provides standardized interfaces for biclustering results and can combine their results with a multi-algorithm ensemble approach to compute robust ensemble biclusters on molecular omics data. This is done by computing similarity networks of biclusters and filtering for overlaps using a custom error model. After that, the louvain modularity it used to extract bicluster communities from the similarity network, which can then be converted to ensemble biclusters. Additionally, MoSBi includes several network visualization methods to give an intuitive and scalable overview of the results. MoSBi comes with several biclustering algorithms, but can be easily extended to new biclustering algorithms.\n",
      "Version: 1.4.0\n",
      "Publication date: None\n",
      "Mantainer: Tim Daniel Rose <tim.rose@wzw.tum.de>\n",
      "Authors:\n",
      "Author: Tim Daniel Rose [cre, aut], Josch Konstantin Pauling [aut], Nikolai Koehler [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - Rcpp, type: imports\n",
      "  - BH, type: imports\n",
      "  - xml2, type: imports\n",
      "  - methods, type: imports\n",
      "  - igraph, type: imports\n",
      "  - fabia, type: imports\n",
      "  - RcppParallel, type: imports\n",
      "  - biclust, type: imports\n",
      "  - isa2, type: imports\n",
      "  - QUBIC, type: imports\n",
      "  - akmbiclust, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "AGPL-3 + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/mosbi.html\n",
      "\n",
      "Name: MSA2dist\n",
      "Description:\n",
      "MSA2dist calculates pairwise distances between all sequences of a DNAStringSet or a AAStringSet using a custom score matrix and conducts codon based analysis. It uses scoring matrices to be used in these pairwise distance calcualtions which can be adapted to any scoring for DNA or AA characters. E.g. by using literal distances MSA2dist calcualtes pairwise IUPAC distances.\n",
      "Version: 1.2.0\n",
      "Publication date: None\n",
      "Mantainer: Kristian K Ullrich <ullrich@evolbio.mpg.de>\n",
      "Authors:\n",
      "Author: Kristian K Ullrich [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.2.0) type: depends\n",
      "  - Rcpp, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - ape, type: imports\n",
      "  - doParallel, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - foreach, type: imports\n",
      "  - methods, type: imports\n",
      "  - parallel, type: imports\n",
      "  - rlang, type: imports\n",
      "  - seqinr, type: imports\n",
      "  - stringr, type: imports\n",
      "  - tibble, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - stats, type: imports\n",
      "  - stringi, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3 + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/MSA2dist.html\n",
      "\n",
      "Name: NADfinder\n",
      "Description:\n",
      "Nucleolus is an important structure inside the nucleus in eukaryotic cells. It is the site for transcribing rDNA into rRNA and for assembling ribosomes, aka ribosome biogenesis. In addition, nucleoli are dynamic hubs through which numerous proteins shuttle and contact specific non-rDNA genomic loci. Deep sequencing analyses of DNA associated with isolated nucleoli (NAD- seq) have shown that specific loci, termed nucleolus- associated domains (NADs) form frequent three- dimensional associations with nucleoli. NAD-seq has been used to study the biological functions of NAD and the dynamics of NAD distribution during embryonic stem cell (ESC) differentiation. Here, we developed a Bioconductor package NADfinder for bioinformatic analysis of the NAD-seq data, including baseline correction, smoothing, normalization, peak calling, and annotation.\n",
      "Version: 1.22.0\n",
      "Publication date: None\n",
      "Mantainer: Jianhong Ou <jianhong.ou@duke.edu>, Lihua Julie Zhu <julie.zhu@umassmed.edu>\n",
      "Authors:\n",
      "Author: Jianhong Ou, Haibo Liu, Jun Yu, HervÃ© PagÃ¨s, Paul Kaufman, Lihua Julie Zhu\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - BiocGenerics, type: depends\n",
      "  - IRanges, type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - S4Vectors, type: depends\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - graphics, type: imports\n",
      "  - methods, type: imports\n",
      "  - baseline, type: imports\n",
      "  - signal, type: imports\n",
      "  - GenomicAlignments, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - limma, type: imports\n",
      "  - trackViewer, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - metap, type: imports\n",
      "  - EmpiricalBrownsMethod, type: imports\n",
      "  - ATACseqQC, type: imports\n",
      "  - corrplot, type: imports\n",
      "  - csaw, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/NADfinder.html\n",
      "\n",
      "Name: nnNorm\n",
      "Description:\n",
      "This package allows to detect and correct for spatial and intensity biases with two-channel microarray data. The normalization method implemented in this package is based on robust neural networks fitting.\n",
      "Version: 2.62.0\n",
      "Publication date: None\n",
      "Mantainer: Adi Laurentiu Tarca <atarca@med.wayne.edu>\n",
      "Authors:\n",
      "Author: Adi Laurentiu Tarca <atarca at med.wayne.edu>\n",
      "Dependencies:\n",
      "  - R (>= 2.2.0) type: depends\n",
      "  - marray, type: depends\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - marray, type: imports\n",
      "  - methods, type: imports\n",
      "  - nnet, type: imports\n",
      "  - stats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/nnNorm.html\n",
      "\n",
      "Name: nucleoSim\n",
      "Description:\n",
      "This package can generate a synthetic map with reads covering the nucleosome regions as well as a synthetic map with forward and reverse reads emulating next-generation sequencing. The synthetic hybridization data of âTiling Arraysâ can also be generated. The user has choice between three different distributions for the read positioning: Normal, Student and Uniform. In addition, a visualization tool is provided to explore the synthetic nucleosome maps.\n",
      "Version: 1.26.0\n",
      "Publication date: None\n",
      "Mantainer: Astrid DeschÃªnes <adeschen@hotmail.com>\n",
      "Authors:\n",
      "Author: Rawane Samb [aut], Astrid DeschÃªnes [cre, aut] , Pascal Belleau [aut] , Arnaud Droit [aut]\n",
      "Dependencies:\n",
      "  - stats, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - graphics, type: imports\n",
      "  - methods, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/nucleoSim.html\n",
      "\n",
      "Name: OLINgui\n",
      "Description:\n",
      "Graphical user interface for the OLIN package\n",
      "Version: 1.72.0\n",
      "Publication date: None\n",
      "Mantainer: Matthias Futschik <mfutschik@ualg.pt>\n",
      "Authors:\n",
      "Author: Matthias Futschik <mfutschik at ualg.pt>\n",
      "Dependencies:\n",
      "  - R (>= 2.0.0) type: depends\n",
      "  - OLIN, type: depends\n",
      "  - graphics, type: imports\n",
      "  - marray, type: imports\n",
      "  - OLIN, type: imports\n",
      "  - tcltk, type: imports\n",
      "  - tkWidgets, type: imports\n",
      "  - widgetTools, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/OLINgui.html\n",
      "\n",
      "Name: omicRexposome\n",
      "Description:\n",
      "omicRexposome systematizes the association evaluation between exposures and omic data, taking advantage of MultiDataSet for coordinated data management, rexposome for exposome data definition and limma for association testing. Also to perform data integration mixing exposome and omic data using multi co-inherent analysis (omicade4) and multi-canonical correlation analysis (PMA).\n",
      "Version: 1.20.0\n",
      "Publication date: None\n",
      "Mantainer: Xavier EscribÃ  Montagut <xavier.escriba@isglobal.org>\n",
      "Authors:\n",
      "Author: Carles Hernandez-Ferrer [aut, cre], Juan R. GonzÃ¡lez [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - Biobase, type: depends\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - methods, type: imports\n",
      "  - rexposome, type: imports\n",
      "  - limma, type: imports\n",
      "  - sva, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - ggrepel, type: imports\n",
      "  - PMA, type: imports\n",
      "  - omicade4, type: imports\n",
      "  - gridExtra, type: imports\n",
      "  - MultiDataSet, type: imports\n",
      "  - SmartSVA, type: imports\n",
      "  - isva, type: imports\n",
      "  - parallel, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - stringr, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/omicRexposome.html\n",
      "\n",
      "Name: oppar\n",
      "Description:\n",
      "The R implementation of mCOPA package published by Wang et al. (2012). Oppar provides methods for Cancer Outlier profile Analysis. Although initially developed to detect outlier genes in cancer studies, methods presented in oppar can be used for outlier profile analysis in general. In addition, tools are provided for gene set enrichment and pathway analysis.\n",
      "Version: 1.26.0\n",
      "Publication date: None\n",
      "Mantainer: Soroor Hediyeh zadeh <hediyehzadeh.s@wehi.edu.au>\n",
      "Authors:\n",
      "Author: Chenwei Wang [aut], Alperen Taciroglu [aut], Stefan R Maetschke [aut], Colleen C Nelson [aut], Mark Ragan [aut], Melissa Davis [aut], Soroor Hediyeh zadeh [cre], Momeneh Foroutan [ctr]\n",
      "Dependencies:\n",
      "  - R (>= 3.3) type: depends\n",
      "  - Biobase, type: imports\n",
      "  - methods, type: imports\n",
      "  - GSEABase, type: imports\n",
      "  - GSVA, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/oppar.html\n",
      "\n",
      "Name: packFinder\n",
      "Description:\n",
      "Algorithm and tools for in silico pack-TYPE transposon discovery. Filters a given genome for properties unique to DNA transposons and provides tools for the investigation of returned matches. Sequences are input in DNAString format, and ranges are returned as a dataframe (in the format returned by as.dataframe(GRanges)).\n",
      "Version: 1.10.0\n",
      "Publication date: None\n",
      "Mantainer: Jack Gisby <jackgisby@gmail.com>\n",
      "Authors:\n",
      "Author: Jack Gisby [aut, cre] , Marco Catoni [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.1.0) type: depends\n",
      "  - Biostrings, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - kmer, type: imports\n",
      "  - ape, type: imports\n",
      "  - methods, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/packFinder.html\n",
      "\n",
      "Name: pageRank\n",
      "Description:\n",
      "Implemented temporal PageRank analysis as defined by Rozenshtein and Gionis. Implemented multiplex PageRank as defined by Halu et al. Applied temporal and multiplex PageRank in gene regulatory network analysis.\n",
      "Version: 1.8.0\n",
      "Publication date: None\n",
      "Mantainer: Hongxu Ding <hd2326@columbia.edu>\n",
      "Authors:\n",
      "Author: Hongxu Ding [aut, cre, ctb, cph]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - GenomicRanges, type: imports\n",
      "  - igraph, type: imports\n",
      "  - motifmatchr, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/pageRank.html\n",
      "\n",
      "Name: PAST\n",
      "Description:\n",
      "PAST takes GWAS output and assigns SNPs to genes, uses those genes to find pathways associated with the genes, and plots pathways based on significance. Implements methods for reading GWAS input data, finding genes associated with SNPs, calculating enrichment score and significance of pathways, and plotting pathways.\n",
      "Version: 1.14.0\n",
      "Publication date: None\n",
      "Mantainer: Thrash Adam <thrash@igbb.msstate.edu>\n",
      "Authors:\n",
      "Author: Thrash Adam [cre, aut], DeOrnellis Mason [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - rlang, type: imports\n",
      "  - iterators, type: imports\n",
      "  - parallel, type: imports\n",
      "  - foreach, type: imports\n",
      "  - doParallel, type: imports\n",
      "  - qvalue, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>=3) + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/PAST.html\n",
      "\n",
      "Name: pepStat\n",
      "Description:\n",
      "Statistical analysis of peptide microarrays\n",
      "Version: 1.32.0\n",
      "Publication date: None\n",
      "Mantainer: Gregory C Imholte <gimholte@uw.edu>\n",
      "Authors:\n",
      "Author: Raphael Gottardo, Gregory C Imholte, Renan Sauteraud, Mike Jiang\n",
      "Dependencies:\n",
      "  - R (>= 3.0.0) type: depends\n",
      "  - Biobase, type: depends\n",
      "  - IRanges, type: depends\n",
      "  - limma, type: imports\n",
      "  - fields, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - plyr, type: imports\n",
      "  - tools, type: imports\n",
      "  - methods, type: imports\n",
      "  - data.table, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/pepStat.html\n",
      "\n",
      "Name: PLPE\n",
      "Description:\n",
      "This package performs tests for paired high-throughput data.\n",
      "Version: 1.58.0\n",
      "Publication date: None\n",
      "Mantainer: Soo-heang Eo <hanansh@korea.ac.kr>\n",
      "Authors:\n",
      "Author: HyungJun Cho <hj4cho at korea.ac.kr> and Jae K. Lee <jaeklee at virginia.edu>\n",
      "Dependencies:\n",
      "  - R (>= 2.6.2) type: depends\n",
      "  - Biobase, type: depends\n",
      "  - LPE, type: depends\n",
      "  - MASS, type: depends\n",
      "  - methods, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/PLPE.html\n",
      "\n",
      "Name: pvac\n",
      "Description:\n",
      "The package contains the function for filtering genes by the proportion of variation accounted for by the first principal component (PVAC).\n",
      "Version: 1.46.0\n",
      "Publication date: None\n",
      "Mantainer: Jun Lu <jlu276@gmail.com>, Pierre R. Bushel <bushel@niehs.nih.gov>\n",
      "Authors:\n",
      "Author: Jun Lu and Pierre R. Bushel\n",
      "Dependencies:\n",
      "  - R (>= 2.8.0) type: depends\n",
      "  - affy, type: imports\n",
      "  - stats, type: imports\n",
      "  - Biobase, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL (>= 2.0)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/pvac.html\n",
      "\n",
      "Name: R3CPET\n",
      "Description:\n",
      "The package provides a method to infer the set of proteins that are more probably to work together to maintain chormatin interaction given a ChIA-PET experiment results.\n",
      "Version: 1.30.0\n",
      "Publication date: None\n",
      "Mantainer: Mohamed Nadhir Djekidel <djek.nad@gmail.com>\n",
      "Authors:\n",
      "Author: Djekidel MN, Yang Chen et al.\n",
      "Dependencies:\n",
      "  - R (>= 3.2) type: depends\n",
      "  - Rcpp (>= 0.10.4) type: depends\n",
      "  - methods, type: depends\n",
      "  - methods, type: imports\n",
      "  - parallel, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - pheatmap, type: imports\n",
      "  - clValid, type: imports\n",
      "  - igraph, type: imports\n",
      "  - data.table, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - Hmisc, type: imports\n",
      "  - RCurl, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - ggbio, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>=2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/R3CPET.html\n",
      "\n",
      "Name: rama\n",
      "Description:\n",
      "Bioconductor version: Release (3.16)\n",
      "Version: 1.72.0\n",
      "Publication date: None\n",
      "Mantainer: Author: Raphael Gottardo\n",
      "Authors:\n",
      "Robust estimation of cDNA microarray intensities with replicates. The package uses a Bayesian hierarchical model for the robust estimation. Outliers are modeled explicitly using a t-distribution, and the model also addresses classical issues such as design effects, normalization, transformation, and nonconstant variance.\n",
      "Dependencies:\n",
      "  - R (>= 2.5.0) type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/rama.html\n",
      "\n",
      "Name: RareVariantVis\n",
      "Description:\n",
      "Second version of RareVariantVis package aims to provide comprehensive information about rare variants for your genome data. It annotates, filters and presents genomic variants (especially rare ones) in a global, per chromosome way. For discovered rare variants CRISPR guide RNAs are designed, so the user can plan further functional studies. Large structural variants, including copy number variants are also supported. Package accepts variants directly from variant caller - for example GATK or Speedseq. Output of package are lists of variants together with adequate visualization. Visualization of variants is performed in two ways - standard that outputs png figures and interactive that uses JavaScript d3 package. Interactive visualization allows to analyze trio/family data, for example in search for causative variants in rare Mendelian diseases, in point-and-click interface. The package includes homozygous region caller and allows to analyse whole human genomes in less than 30 minutes on a desktop computer. RareVariantVis disclosed novel causes of several rare monogenic disorders, including one with non-coding causative variant - keratolythic winter erythema.\n",
      "Version: 2.26.0\n",
      "Publication date: None\n",
      "Mantainer: Tomasz Stokowy <tomasz.stokowy@k2.uib.no>\n",
      "Authors:\n",
      "Author: Adam Gudys and Tomasz Stokowy\n",
      "Dependencies:\n",
      "  - BiocGenerics, type: depends\n",
      "  - VariantAnnotation, type: depends\n",
      "  - googleVis, type: depends\n",
      "  - GenomicFeatures, type: depends\n",
      "  - S4Vectors, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - gtools, type: imports\n",
      "  - BSgenome, type: imports\n",
      "  - BSgenome.Hsapiens.UCSC.hg19, type: imports\n",
      "  - TxDb.Hsapiens.UCSC.hg19.knownGene, type: imports\n",
      "  - phastCons100way.UCSC.hg19, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - GenomicScores, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/RareVariantVis.html\n",
      "\n",
      "Name: rfPred\n",
      "Description:\n",
      "Based on external numerous data files where rfPred scores are pre-calculated on all genomic positions of the human exome, the package gives rfPred scores to missense variants identified by the chromosome, the position (hg19 version), the referent and alternative nucleotids and the uniprot identifier of the protein. Note that for using the package, the user has to download the TabixFile and index (approximately 3.3 Go).\n",
      "Version: 1.36.0\n",
      "Publication date: None\n",
      "Mantainer: Hugo Varet <varethugo@gmail.com>\n",
      "Authors:\n",
      "Author: Fabienne Jabot-Hanin, Hugo Varet and Jean-Philippe Jais\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - utils, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - data.table, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - parallel, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>=2 )\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/rfPred.html\n",
      "\n",
      "Name: ribor\n",
      "Description:\n",
      "The ribor package provides an R Interface for .ribo files. It provides functionality to read the .ribo file, which is of HDF5 format, and performs common analyses on its contents.\n",
      "Version: 1.10.0\n",
      "Publication date: None\n",
      "Mantainer: Michael Geng <michaelgeng@utexas.edu>\n",
      "Authors:\n",
      "Author: Michael Geng [cre, aut], Hakan Ozadam [aut], Can Cenik [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.6.0) type: depends\n",
      "  - dplyr, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - hash, type: imports\n",
      "  - methods, type: imports\n",
      "  - rhdf5, type: imports\n",
      "  - rlang, type: imports\n",
      "  - stats, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - tools, type: imports\n",
      "  - yaml, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ribor.html\n",
      "\n",
      "Name: RLMM\n",
      "Description:\n",
      "A classification algorithm, based on a multi-chip, multi-SNP approach for Affymetrix SNP arrays. Using a large training sample where the genotype labels are known, this aglorithm will obtain more accurate classification results on new data. RLMM is based on a robust, linear model and uses the Mahalanobis distance for classification. The chip-to-chip non-biological variation is removed through normalization. This model-based algorithm captures the similarities across genotype groups and probes, as well as thousands other SNPs for accurate classification. NOTE: 100K-Xba only at for now.\n",
      "Version: 1.60.0\n",
      "Publication date: None\n",
      "Mantainer: Nusrat Rabbee <nrabbee@post.harvard.edu>\n",
      "Authors:\n",
      "Author: Nusrat Rabbee <nrabbee at post.harvard.edu>, Gary Wong <wongg62 at berkeley.edu>\n",
      "Dependencies:\n",
      "  - R (>= 2.1.0) type: depends\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - MASS, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/RLMM.html\n",
      "\n",
      "Name: RNAmodR.RiboMethSeq\n",
      "Description:\n",
      "RNAmodR.RiboMethSeq implements the detection of 2'-O methylations on RNA from experimental data generated with the RiboMethSeq protocol. The package builds on the core functionality of the RNAmodR package to detect specific patterns of the modifications in high throughput sequencing data.\n",
      "Version: 1.12.0\n",
      "Publication date: None\n",
      "Mantainer: Felix G.M. Ernst <felix.gm.ernst@outlook.com>\n",
      "Authors:\n",
      "Author: Felix G.M. Ernst [aut, cre] , Denis L.J. Lafontaine [ctb, fnd]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - RNAmodR, type: depends\n",
      "  - methods, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - Gviz, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/RNAmodR.RiboMethSeq.html\n",
      "\n",
      "Name: RPA\n",
      "Description:\n",
      "Probabilistic analysis of probe reliability and differential gene expression on short oligonucleotide arrays.\n",
      "Version: 1.54.0\n",
      "Publication date: None\n",
      "Mantainer: Leo Lahti <leo.lahti@iki.fi>\n",
      "Authors:\n",
      "Author: Leo Lahti [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 3.1.1) type: depends\n",
      "  - affy, type: depends\n",
      "  - BiocGenerics, type: depends\n",
      "  - BiocStyle, type: depends\n",
      "  - methods, type: depends\n",
      "  - rmarkdown, type: depends\n",
      "  - phyloseq, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "BSD_2_clause + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/RPA.html\n",
      "\n",
      "Name: seqcombo\n",
      "Description:\n",
      "Provides useful functions for visualizing virus reassortment events.\n",
      "Version: 1.20.0\n",
      "Publication date: None\n",
      "Mantainer: Guangchuang Yu <guangchuangyu@gmail.com>\n",
      "Authors:\n",
      "Author: Guangchuang Yu [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 3.4.0) type: depends\n",
      "  - ggplot2, type: imports\n",
      "  - grid, type: imports\n",
      "  - igraph, type: imports\n",
      "  - utils, type: imports\n",
      "  - yulab.utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/seqcombo.html\n",
      "\n",
      "Name: sevenC\n",
      "Description:\n",
      "Chromatin looping is an essential feature of eukaryotic genomes and can bring regulatory sequences, such as enhancers or transcription factor binding sites, in the close physical proximity of regulated target genes. Here, we provide sevenC, an R package that uses protein binding signals from ChIP-seq and sequence motif information to predict chromatin looping events. Cross-linking of proteins that bind close to loop anchors result in ChIP-seq signals at both anchor loci. These signals are used at CTCF motif pairs together with their distance and orientation to each other to predict whether they interact or not. The resulting chromatin loops might be used to associate enhancers or transcription factor binding sites (e.g., ChIP-seq peaks) to regulated target genes.\n",
      "Version: 1.18.0\n",
      "Publication date: None\n",
      "Mantainer: Jonas Ibn-Salem <jonas.ibn-salem@tron-mainz.de>\n",
      "Authors:\n",
      "Author: Jonas Ibn-Salem [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 3.5) type: depends\n",
      "  - InteractionSet, type: depends\n",
      "  - rtracklayer, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - readr (>= 1.1.0) type: imports\n",
      "  - purrr (>= 0.2.2) type: imports\n",
      "  - data.table (>= 1.10.4) type: imports\n",
      "  - boot (>= 1.3-20) type: imports\n",
      "  - methods (>= 3.4.1) type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/sevenC.html\n",
      "\n",
      "Name: SNAGEE\n",
      "Description:\n",
      "Signal-to-Noise applied to Gene Expression Experiments. Signal-to-noise ratios can be used as a proxy for quality of gene expression studies and samples. The SNRs can be calculated on any gene expression data set as long as gene IDs are available, no access to the raw data files is necessary. This allows to flag problematic studies and samples in any public data set.\n",
      "Version: 1.38.0\n",
      "Publication date: None\n",
      "Mantainer: David Venet <davenet@ulb.ac.be>\n",
      "Authors:\n",
      "Author: David Venet <davenet at ulb.ac.be>\n",
      "Dependencies:\n",
      "  - R (>= 2.6.0) type: depends\n",
      "  - SNAGEEdata, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/SNAGEE.html\n",
      "\n",
      "Name: SpacePAC\n",
      "Description:\n",
      "Identifies clustering of somatic mutations in proteins via a simulation approach while considering the protein's tertiary structure.\n",
      "Version: 1.36.0\n",
      "Publication date: None\n",
      "Mantainer: Gregory Ryslik <gregory.ryslik@yale.edu>\n",
      "Authors:\n",
      "Author: Gregory Ryslik, Hongyu Zhao\n",
      "Dependencies:\n",
      "  - R (>= 2.15) type: depends\n",
      "  - iPAC, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/SpacePAC.html\n",
      "\n",
      "Name: spiky\n",
      "Description:\n",
      "spiky implements methods and model generation for cfMeDIP (cell-free methylated DNA immunoprecipitation) with spike-in controls. CfMeDIP is an enrichment protocol which avoids destructive conversion of scarce template, making it ideal as a \"liquid biopsy,\" but creating certain challenges in comparing results across specimens, subjects, and experiments. The use of synthetic spike-in standard oligos allows diagnostics performed with cfMeDIP to quantitatively compare samples across subjects, experiments, and time points in both relative and absolute terms.\n",
      "Version: 1.4.0\n",
      "Publication date: None\n",
      "Mantainer: Tim Triche <trichelab@gmail.com>\n",
      "Authors:\n",
      "Author: Samantha Wilson [aut], Lauren Harmon [aut], Tim Triche [aut, cre]\n",
      "Dependencies:\n",
      "  - Rsamtools, type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - R (>= 3.6.0) type: depends\n",
      "  - stats, type: imports\n",
      "  - scales, type: imports\n",
      "  - bamlss, type: imports\n",
      "  - methods, type: imports\n",
      "  - tools, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - GenomicAlignments, type: imports\n",
      "  - BlandAltmanLeh, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - BSgenome, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - graphics, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/spiky.html\n",
      "\n",
      "Name: stepNorm\n",
      "Description:\n",
      "Stepwise normalization functions for cDNA microarray data.\n",
      "Version: 1.70.0\n",
      "Publication date: None\n",
      "Mantainer: Yuanyuan Xiao <yxiao@itsa.ucsf.edu>\n",
      "Authors:\n",
      "Author: Yuanyuan Xiao <yxiao at itsa.ucsf.edu>, Yee Hwa (Jean) Yang <jean at biostat.ucsf.edu>\n",
      "Dependencies:\n",
      "  - R (>= 1.8.0) type: depends\n",
      "  - marray, type: depends\n",
      "  - methods, type: depends\n",
      "  - marray, type: imports\n",
      "  - MASS, type: imports\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/stepNorm.html\n",
      "\n",
      "Name: strandCheckR\n",
      "Description:\n",
      "This package aims to quantify and remove putative double strand DNA from a strand-specific RNA sample. There are also options and methods to plot the positive/negative proportions of all sliding windows, which allow users to have an idea of how much the sample was contaminated and the appropriate threshold to be used for filtering.\n",
      "Version: 1.16.0\n",
      "Publication date: None\n",
      "Mantainer: Thu-Hien To <tothuhien@gmail.com>\n",
      "Authors:\n",
      "Author: Thu-Hien To [aut, cre], Steve Pederson [aut]\n",
      "Dependencies:\n",
      "  - dplyr, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - GenomicAlignments, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - grid, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - stats, type: imports\n",
      "  - gridExtra, type: imports\n",
      "  - TxDb.Hsapiens.UCSC.hg38.knownGene, type: imports\n",
      "  - methods, type: imports\n",
      "  - stringr, type: imports\n",
      "  - rmarkdown, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/strandCheckR.html\n",
      "\n",
      "Name: TIN\n",
      "Description:\n",
      "The TIN package implements a set of tools for transcriptome instability analysis based on exon expression profiles. Deviating exon usage is studied in the context of splicing factors to analyse to what degree transcriptome instability is correlated to splicing factor expression. In the transcriptome instability correlation analysis, the data is compared to both random permutations of alternative splicing scores and expression of random gene sets.\n",
      "Version: 1.30.0\n",
      "Publication date: None\n",
      "Mantainer: Bjarne Johannessen <bjajoh@rr-research.no>\n",
      "Authors:\n",
      "Author: Bjarne Johannessen, Anita Sveen and Rolf I. Skotheim\n",
      "Dependencies:\n",
      "  - R (>= 2.12.0) type: depends\n",
      "  - data.table, type: depends\n",
      "  - impute, type: depends\n",
      "  - aroma.affymetrix, type: depends\n",
      "  - WGCNA, type: imports\n",
      "  - squash, type: imports\n",
      "  - stringr, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/TIN.html\n",
      "\n",
      "Name: tRanslatome\n",
      "Description:\n",
      "Detection of differentially expressed genes (DEGs) from the comparison of two biological conditions (treated vs. untreated, diseased vs. normal, mutant vs. wild-type) among different levels of gene expression (transcriptome ,translatome, proteome), using several statistical methods: Rank Product, Translational Efficiency, t-test, Limma, ANOTA, DESeq, edgeR. Possibility to plot the results with scatterplots, histograms, MA plots, standard deviation (SD) plots, coefficient of variation (CV) plots. Detection of significantly enriched post-transcriptional regulatory factors (RBPs, miRNAs, etc) and Gene Ontology terms in the lists of DEGs previously identified for the two expression levels. Comparison of GO terms enriched only in one of the levels or in both. Calculation of the semantic similarity score between the lists of enriched GO terms coming from the two expression levels. Visual examination and comparison of the enriched terms with heatmaps, radar plots and barplots.\n",
      "Version: 1.36.0\n",
      "Publication date: None\n",
      "Mantainer: Toma Tebaldi <tebaldi@science.unitn.it>, Erik Dassi <erik.dassi@unitn.it>\n",
      "Authors:\n",
      "Author: Toma Tebaldi, Erik Dassi, Galena Kostoska\n",
      "Dependencies:\n",
      "  - R (>= 2.15.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - limma, type: depends\n",
      "  - sigPathway, type: depends\n",
      "  - anota, type: depends\n",
      "  - DESeq2, type: depends\n",
      "  - edgeR, type: depends\n",
      "  - RankProd, type: depends\n",
      "  - topGO, type: depends\n",
      "  - org.Hs.eg.db, type: depends\n",
      "  - GOSemSim, type: depends\n",
      "  - Heatplus, type: depends\n",
      "  - gplots, type: depends\n",
      "  - plotrix, type: depends\n",
      "  - Biobase, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/tRanslatome.html\n",
      "\n",
      "Name: trena\n",
      "Description:\n",
      "Methods for reconstructing transcriptional regulatory networks, especially in species for which genome-wide TF binding site information is available.\n",
      "Version: 1.20.0\n",
      "Publication date: None\n",
      "Mantainer: Paul Shannon <paul.thurmond.shannon@gmail.com>\n",
      "Authors:\n",
      "Author: Seth Ament <seth.ament at systemsbiology.org>, Paul Shannon <pshannon at systemsbioloyg.org>, Matthew Richards <mrichard at systemsbiology.org>\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - utils, type: depends\n",
      "  - glmnet (>= 2.0.3) type: depends\n",
      "  - MotifDb, type: depends\n",
      "  - RSQLite, type: imports\n",
      "  - RMySQL, type: imports\n",
      "  - lassopv, type: imports\n",
      "  - randomForest, type: imports\n",
      "  - xgboost, type: imports\n",
      "  - RPostgreSQL, type: imports\n",
      "  - methods, type: imports\n",
      "  - DBI, type: imports\n",
      "  - BSgenome, type: imports\n",
      "  - BSgenome.Hsapiens.UCSC.hg38, type: imports\n",
      "  - BSgenome.Hsapiens.UCSC.hg19, type: imports\n",
      "  - BSgenome.Mmusculus.UCSC.mm10, type: imports\n",
      "  - SNPlocs.Hsapiens.dbSNP150.GRCh38, type: imports\n",
      "  - org.Hs.eg.db, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - biomaRt, type: imports\n",
      "  - AnnotationDbi, type: imports\n",
      "  - WGCNA, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/trena.html\n",
      "\n",
      "Name: trigger\n",
      "Description:\n",
      "This R package provides tools for the statistical analysis of integrative genomic data that involve some combination of: genotypes, high-dimensional intermediate traits (e.g., gene expression, protein abundance), and higher-order traits (phenotypes). The package includes functions to: (1) construct global linkage maps between genetic markers and gene expression; (2) analyze multiple-locus linkage (epistasis) for gene expression; (3) quantify the proportion of genome-wide variation explained by each locus and identify eQTL hotspots; (4) estimate pair-wise causal gene regulatory probabilities and construct gene regulatory networks; and (5) identify causal genes for a quantitative trait of interest.\n",
      "Version: 1.44.0\n",
      "Publication date: None\n",
      "Mantainer: John D. Storey <jstorey@princeton.edu>\n",
      "Authors:\n",
      "Author: Lin S. Chen <lchen at health.bsd.uchicago.edu>, Dipen P. Sangurdekar <dps at genomics.princeton.edu> and John D. Storey <jstorey at princeton.edu>\n",
      "Dependencies:\n",
      "  - R (>= 2.14.0) type: depends\n",
      "  - corpcor, type: depends\n",
      "  - qtl, type: depends\n",
      "  - qvalue, type: imports\n",
      "  - methods, type: imports\n",
      "  - graphics, type: imports\n",
      "  - sva, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/trigger.html\n",
      "\n",
      "Name: TVTB\n",
      "Description:\n",
      "The package provides S4 classes and methods to filter, summarise and visualise genetic variation data stored in VCF files. In particular, the package extends the FilterRules class (S4Vectors package) to define news classes of filter rules applicable to the various slots of VCF objects. Functionalities are integrated and demonstrated in a Shiny web-application, the Shiny Variant Explorer (tSVE).\n",
      "Version: 1.24.0\n",
      "Publication date: None\n",
      "Mantainer: Kevin Rue-Albrecht <kevinrue67@gmail.com>\n",
      "Authors:\n",
      "Author: Kevin Rue-Albrecht [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 3.4) type: depends\n",
      "  - methods, type: depends\n",
      "  - utils, type: depends\n",
      "  - stats, type: depends\n",
      "  - AnnotationFilter, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - ensembldb, type: imports\n",
      "  - ensemblVEP, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - GGally, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - Gviz, type: imports\n",
      "  - limma, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - VariantAnnotation, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/TVTB.html\n",
      "\n",
      "Name: UMI4Cats\n",
      "Description:\n",
      "UMI-4C is a technique that allows characterization of 3D chromatin interactions with a bait of interest, taking advantage of a sonication step to produce unique molecular identifiers (UMIs) that help remove duplication bias, thus allowing a better differential comparsion of chromatin interactions between conditions. This package allows processing of UMI-4C data, starting from FastQ files provided by the sequencing facility. It provides two statistical methods for detecting differential contacts and includes a visualization function to plot integrated information from a UMI-4C assay.\n",
      "Version: 1.8.1\n",
      "Publication date: None\n",
      "Mantainer: Mireia Ramos-Rodriguez <mireiarr9@gmail.com>\n",
      "Authors:\n",
      "Author: Mireia Ramos-Rodriguez [aut, cre] , Marc Subirana-Granes [aut], Lorenzo Pasquali [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.0.0) type: depends\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - magick, type: imports\n",
      "  - cowplot, type: imports\n",
      "  - scales, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - ShortRead, type: imports\n",
      "  - zoo, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - regioneR, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - BSgenome, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - DESeq2, type: imports\n",
      "  - R.utils, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - stringr, type: imports\n",
      "  - Rbowtie2, type: imports\n",
      "  - methods, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - GenomicAlignments, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - utils, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - stats, type: imports\n",
      "  - org.Hs.eg.db, type: imports\n",
      "  - annotate, type: imports\n",
      "  - TxDb.Hsapiens.UCSC.hg19.knownGene, type: imports\n",
      "  - rlang, type: imports\n",
      "  - GenomicFeatures, type: imports\n",
      "  - BiocFileCache, type: imports\n",
      "  - rappdirs, type: imports\n",
      "  - fda, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/UMI4Cats.html\n",
      "\n",
      "Name: veloviz\n",
      "Description:\n",
      "VeloViz uses each cellâs current observed and predicted future transcriptional states inferred from RNA velocity analysis to build a nearest neighbor graph between cells in the population. Edges are then pruned based on a cosine correlation threshold and/or a distance threshold and the resulting graph is visualized using a force-directed graph layout algorithm. VeloViz can help ensure that relationships between cell states are reflected in the 2D embedding, allowing for more reliable representation of underlying cellular trajectories.\n",
      "Version: 1.4.0\n",
      "Publication date: None\n",
      "Mantainer: Lyla Atta <lylaatta@jhmi.edu>\n",
      "Authors:\n",
      "Author: Lyla Atta [aut, cre] , Jean Fan [aut] , Arpan Sahoo [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - Rcpp, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - igraph, type: imports\n",
      "  - mgcv, type: imports\n",
      "  - RSpectra, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/veloviz.html\n",
      "\n",
      "Name: ArrayExpressHTS\n",
      "Description:\n",
      "RNA-Seq processing pipeline for public ArrayExpress experiments or local datasets\n",
      "Version: 1.47.0\n",
      "Publication date: None\n",
      "Mantainer: Angela Goncalves <filimon@ebi.ac.uk>, Andrew Tikhonov <andrew@ebi.ac.uk>\n",
      "Authors:\n",
      "Author: Angela Goncalves, Andrew Tikhonov\n",
      "Dependencies:\n",
      "  - sampling, type: depends\n",
      "  - Rsamtools, type: depends\n",
      "  - snow, type: depends\n",
      "  - Biobase, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - Hmisc, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - R2HTML, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - ShortRead, type: imports\n",
      "  - XML, type: imports\n",
      "  - biomaRt, type: imports\n",
      "  - edgeR, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - methods, type: imports\n",
      "  - rJava, type: imports\n",
      "  - stats, type: imports\n",
      "  - svMisc, type: imports\n",
      "  - utils, type: imports\n",
      "  - sendmailR, type: imports\n",
      "  - bitops, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic License 2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ArrayExpressHTS.html\n",
      "\n",
      "Name: barcodetrackR\n",
      "Description:\n",
      "barcodetrackR is an R package developed for the analysis and visualization of clonal tracking data. Data required is samples and tag abundances in matrix form. Usually from cellular barcoding experiments, integration site retrieval analyses, or similar technologies.\n",
      "Version: 1.6.0\n",
      "Publication date: None\n",
      "Mantainer: Diego Alexander Espinoza <diego.espinoza@pennmedicine.upenn.edu>\n",
      "Authors:\n",
      "Author: Diego Alexander Espinoza [aut, cre], Ryland Mortlock [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - cowplot, type: imports\n",
      "  - circlize, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - ggdendro, type: imports\n",
      "  - ggridges, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - plyr, type: imports\n",
      "  - proxy, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - rlang, type: imports\n",
      "  - scales, type: imports\n",
      "  - shiny, type: imports\n",
      "  - stats, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - tibble, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - vegan, type: imports\n",
      "  - viridis, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/barcodetrackR.html\n",
      "\n",
      "Name: BEARscc\n",
      "Description:\n",
      "BEARscc is a noise estimation and injection tool that is designed to assess putative single-cell RNA-seq clusters in the context of experimental noise estimated by ERCC spike-in controls.\n",
      "Version: 1.18.0\n",
      "Publication date: None\n",
      "Mantainer: Benjamin Schuster-Boeckler <benjamin.schuster-boeckler@ludwig.ox.ac.uk>\n",
      "Authors:\n",
      "Author: David T. Severson <david_severson at hms.harvard.edu>\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - ggplot2, type: imports\n",
      "  - SingleCellExperiment, type: imports\n",
      "  - data.table, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - graphics, type: imports\n",
      "  - compiler, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/BEARscc.html\n",
      "\n",
      "Name: benchdamic\n",
      "Description:\n",
      "Starting from a microbiome dataset (16S or WMS with absolute count values) it is possible to perform several analysis to assess the performances of many differential abundance detection methods. A basic and standardized version of the main differential abundance analysis methods is supplied but the user can also add his method to the benchmark. The analyses focus on 4 main aspects: i) the goodness of fit of each method's distributional assumptions on the observed count data, ii) the ability to control the false discovery rate, iii) the within and between method concordances, iv) the truthfulness of the findings if any apriori knowledge is given. Several graphical functions are available for result visualization.\n",
      "Version: 1.4.0\n",
      "Publication date: None\n",
      "Mantainer: Matteo Calgaro <mcalgaro93@gmail.com>\n",
      "Authors:\n",
      "Author: Matteo Calgaro [aut, cre], Chiara Romualdi [aut], Davide Risso [aut], Nicola Vitulo [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.2.0) type: depends\n",
      "  - stats, type: imports\n",
      "  - stats4, type: imports\n",
      "  - utils, type: imports\n",
      "  - methods, type: imports\n",
      "  - phyloseq, type: imports\n",
      "  - TreeSummarizedExperiment, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - zinbwave, type: imports\n",
      "  - edgeR, type: imports\n",
      "  - DESeq2, type: imports\n",
      "  - limma, type: imports\n",
      "  - ALDEx2, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - MAST, type: imports\n",
      "  - Seurat, type: imports\n",
      "  - ANCOMBC, type: imports\n",
      "  - NOISeq, type: imports\n",
      "  - dearseq, type: imports\n",
      "  - metagenomeSeq, type: imports\n",
      "  - corncob, type: imports\n",
      "  - MGLM, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - plyr, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - ggdendro, type: imports\n",
      "  - ggridges, type: imports\n",
      "  - graphics, type: imports\n",
      "  - cowplot, type: imports\n",
      "  - tidytext, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/benchdamic.html\n",
      "\n",
      "Name: bnem\n",
      "Description:\n",
      "bnem combines the use of indirect measurements of Nested Effects Models (package mnem) with the Boolean networks of CellNOptR. Perturbation experiments of signalling nodes in cells are analysed for their effect on the global gene expression profile. Those profiles give evidence for the Boolean regulation of down-stream nodes in the network, e.g., whether two parents activate their child independently (OR-gate) or jointly (AND-gate).\n",
      "Version: 1.6.0\n",
      "Publication date: None\n",
      "Mantainer: Martin Pirkl <martinpirkl@yahoo.de>\n",
      "Authors:\n",
      "Author: Martin Pirkl [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - CellNOptR, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - snowfall, type: imports\n",
      "  - Rgraphviz, type: imports\n",
      "  - cluster, type: imports\n",
      "  - flexclust, type: imports\n",
      "  - stats, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - epiNEM, type: imports\n",
      "  - mnem, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - methods, type: imports\n",
      "  - utils, type: imports\n",
      "  - graphics, type: imports\n",
      "  - graph, type: imports\n",
      "  - affy, type: imports\n",
      "  - binom, type: imports\n",
      "  - limma, type: imports\n",
      "  - sva, type: imports\n",
      "  - vsn, type: imports\n",
      "  - rmarkdown, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/bnem.html\n",
      "\n",
      "Name: BrainSABER\n",
      "Description:\n",
      "Bioconductor version: Release (3.16)\n",
      "Version: 1.8.0\n",
      "Publication date: None\n",
      "Mantainer: Author: Carrie Minette [aut], Evgeni Radichev [aut], USD Biomedical Engineering [aut, cre]\n",
      "Authors:\n",
      "The Allen Institute for Brain Science provides an RNA sequencing (RNA-Seq) data resource for studying transcriptional mechanisms involved in human brain development known as BrainSpan. BrainSABER is an R package that facilitates comparison of user data with the various developmental stages and brain structures found in the BrainSpan atlas by generating dynamic similarity heatmaps for the two data sets. It also provides a self-validating container for user data.\n",
      "Dependencies:\n",
      "  - R (>= 4.1.0) type: depends\n",
      "  - biomaRt, type: depends\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - data.table, type: imports\n",
      "  - lsa, type: imports\n",
      "  - methods, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - utils, type: imports\n",
      "  - BiocFileCache, type: imports\n",
      "  - shiny, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/BrainSABER.html\n",
      "\n",
      "Name: cellmigRation\n",
      "Description:\n",
      "Import TIFF images of fluorescently labeled cells, and track cell movements over time. Parallelization is supported for image processing and for fast computation of cell trajectories. In-depth analysis of cell trajectories is enabled by 15 trajectory analysis functions.\n",
      "Version: 1.6.0\n",
      "Publication date: None\n",
      "Mantainer: Waldir Leoncio <w.l.netto@medisin.uio.no>\n",
      "Authors:\n",
      "Author: Salim Ghannoum [aut, cph], Damiano Fantini [aut, cph], Waldir Leoncio [cre, aut], Ãystein SÃ¸rensen [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - methods, type: depends\n",
      "  - foreach, type: depends\n",
      "  - tiff, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - parallel, type: imports\n",
      "  - doParallel, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - FME, type: imports\n",
      "  - SpatialTools, type: imports\n",
      "  - sp, type: imports\n",
      "  - vioplot, type: imports\n",
      "  - FactoMineR, type: imports\n",
      "  - Hmisc, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/cellmigRation.html\n",
      "\n",
      "Name: CellScore\n",
      "Description:\n",
      "The CellScore package contains functions to evaluate the cell identity of a test sample, given a cell transition defined with a starting (donor) cell type and a desired target cell type. The evaluation is based upon a scoring system, which uses a set of standard samples of known cell types, as the reference set. The functions have been carried out on a large set of microarray data from one platform (Affymetrix Human Genome U133 Plus 2.0). In principle, the method could be applied to any expression dataset, provided that there are a sufficient number of standard samples and that the data are normalized.\n",
      "Version: 1.18.0\n",
      "Publication date: None\n",
      "Mantainer: Nancy Mah <nancy.l.mah@googlemail.com>\n",
      "Authors:\n",
      "Author: Nancy Mah, Katerina Taskova\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - Biobase, type: imports\n",
      "  - graphics (>= 3.5.0) type: imports\n",
      "  - grDevices (>= 3.5.0) type: imports\n",
      "  - gplots (>= 3.0.1) type: imports\n",
      "  - lsa (>= 0.73.1) type: imports\n",
      "  - methods (>= 3.5.0) type: imports\n",
      "  - RColorBrewer (>= 1.1-2) type: imports\n",
      "  - squash (>= 1.0.8) type: imports\n",
      "  - stats (>= 3.5.0) type: imports\n",
      "  - utils (>= 3.5.0) type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/CellScore.html\n",
      "\n",
      "Name: clusterSeq\n",
      "Description:\n",
      "Identification of clusters of co-expressed genes based on their expression across multiple (replicated) biological samples.\n",
      "Version: 1.22.0\n",
      "Publication date: None\n",
      "Mantainer: Thomas J. Hardcastle <tjh48@cam.ac.uk>\n",
      "Authors:\n",
      "Author: Thomas J. Hardcastle & Irene Papatheodorou\n",
      "Dependencies:\n",
      "  - R (>= 3.0.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - BiocParallel, type: depends\n",
      "  - baySeq, type: depends\n",
      "  - graphics, type: depends\n",
      "  - stats, type: depends\n",
      "  - utils, type: depends\n",
      "  - BiocGenerics, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/clusterSeq.html\n",
      "\n",
      "Name: ClusterSignificance\n",
      "Description:\n",
      "The ClusterSignificance package provides tools to assess if class clusters in dimensionality reduced data representations have a separation different from permuted data. The term class clusters here refers to, clusters of points representing known classes in the data. This is particularly useful to determine if a subset of the variables, e.g. genes in a specific pathway, alone can separate samples into these established classes. ClusterSignificance accomplishes this by, projecting all points onto a one dimensional line. Cluster separations are then scored and the probability of the seen separation being due to chance is evaluated using a permutation method.\n",
      "Version: 1.26.0\n",
      "Publication date: None\n",
      "Mantainer: Jason T Serviss <jason.serviss@ki.se>\n",
      "Authors:\n",
      "Author: Jason T. Serviss [aut, cre], Jesper R. Gadin [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.3.0) type: depends\n",
      "  - methods, type: imports\n",
      "  - pracma, type: imports\n",
      "  - princurve (>= 2.0.5) type: imports\n",
      "  - scatterplot3d, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - utils, type: imports\n",
      "  - stats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ClusterSignificance.html\n",
      "\n",
      "Name: cnvGSA\n",
      "Description:\n",
      "This package is intended to facilitate gene-set association with rare CNVs in case-control studies.\n",
      "Version: 1.42.0\n",
      "Publication date: None\n",
      "Mantainer: Joseph Lugo <joseph.r.lugo@gmail.com>\n",
      "Authors:\n",
      "Author: Daniele Merico <daniele.merico at gmail.com>, Robert Ziman <rziman at gmail.com>; packaged by Joseph Lugo <joseph.r.lugo at gmail.com>\n",
      "Dependencies:\n",
      "  - brglm, type: depends\n",
      "  - doParallel, type: depends\n",
      "  - foreach, type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - methods, type: depends\n",
      "  - splitstackshape, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/cnvGSA.html\n",
      "\n",
      "Name: combi\n",
      "Description:\n",
      "This explorative ordination method combines quasi-likelihood estimation, compositional regression models and latent variable models for integrative visualization of several omics datasets. Both unconstrained and constrained integration are available. The results are shown as interpretable, compositional multiplots.\n",
      "Version: 1.10.0\n",
      "Publication date: None\n",
      "Mantainer: Stijn Hawinkel <stijn.hawinkel@psb.ugent.be>\n",
      "Authors:\n",
      "Author: Stijn Hawinkel [cre, aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - DBI, type: depends\n",
      "  - ggplot2, type: imports\n",
      "  - nleqslv, type: imports\n",
      "  - phyloseq, type: imports\n",
      "  - tensor, type: imports\n",
      "  - stats, type: imports\n",
      "  - limma, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - BB, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - alabama, type: imports\n",
      "  - cobs, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - vegan, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - methods, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/combi.html\n",
      "\n",
      "Name: consensus\n",
      "Description:\n",
      "An implementation of the American Society for Testing and Materials (ASTM) Standard E691 for interlaboratory testing procedures, designed for cross-platform genomic measurements. Given three (3) or more genomic platforms or laboratory protocols, this package provides interlaboratory testing procedures giving per-locus comparisons for sensitivity and precision between platforms.\n",
      "Version: 1.16.0\n",
      "Publication date: None\n",
      "Mantainer: Tim Peters <t.peters@garvan.org.au>\n",
      "Authors:\n",
      "Author: Tim Peters\n",
      "Dependencies:\n",
      "  - R (>= 3.5) type: depends\n",
      "  - RColorBrewer, type: depends\n",
      "  - matrixStats, type: imports\n",
      "  - gplots, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - methods, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "BSD_3_clause + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/consensus.html\n",
      "\n",
      "Name: csdR\n",
      "Description:\n",
      "This package contains functionality to run differential gene co-expression across two different conditions. The algorithm is inspired by Voigt et al. 2017 and finds Conserved, Specific and Differentiated genes (hence the name CSD). This package include efficient and variance calculation by bootstrapping and Welford's algorithm.\n",
      "Version: 1.4.0\n",
      "Publication date: None\n",
      "Mantainer: Jakob Peder Pettersen <jakobpeder.pettersen@gmail.com>\n",
      "Authors:\n",
      "Author: Jakob Peder Pettersen [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.1.0) type: depends\n",
      "  - WGCNA, type: imports\n",
      "  - glue, type: imports\n",
      "  - RhpcBLASctl, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - Rcpp, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/csdR.html\n",
      "\n",
      "Name: cycle\n",
      "Description:\n",
      "Package for assessing the statistical significance of periodic expression based on Fourier analysis and comparison with data generated by different background models\n",
      "Version: 1.52.0\n",
      "Publication date: None\n",
      "Mantainer: Matthias Futschik <mfutschik@ualg.pt>\n",
      "Authors:\n",
      "Author: Matthias Futschik <mfutschik at ualg.pt>\n",
      "Dependencies:\n",
      "  - R (>= 2.10.0) type: depends\n",
      "  - Mfuzz, type: depends\n",
      "  - Biobase, type: imports\n",
      "  - stats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/cycle.html\n",
      "\n",
      "Name: cytoKernel\n",
      "Description:\n",
      "cytoKernel implements a kernel-based score test to identify differentially expressed features in high-dimensional biological experiments. This approach can be applied across many different high-dimensional biological data including gene expression data and dimensionally reduced cytometry-based marker expression data. In this R package, we implement functions that compute the feature-wise p values and their corresponding adjusted p values. Additionally, it also computes the feature-wise shrunk effect sizes and their corresponding shrunken effect size. Further, it calculates the percent of differentially expressed features and plots user-friendly heatmap of the top differentially expressed features on the rows and samples on the columns.\n",
      "Version: 1.4.0\n",
      "Publication date: None\n",
      "Mantainer: Tusharkanti Ghosh <tusharkantighosh30@gmail.com>\n",
      "Authors:\n",
      "Author: Tusharkanti Ghosh [aut, cre], Victor Lui [aut], Pratyaydipta Rudra [aut], Souvik Seal [aut], Thao Vu [aut], Elena Hsieh [aut], Debashis Ghosh [aut, cph]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - Rcpp, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - utils, type: imports\n",
      "  - methods, type: imports\n",
      "  - ComplexHeatmap, type: imports\n",
      "  - circlize, type: imports\n",
      "  - ashr, type: imports\n",
      "  - data.table, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - stats, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - rlang, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/cytoKernel.html\n",
      "\n",
      "Name: DeepPINCS\n",
      "Description:\n",
      "The identification of novel compound-protein interaction (CPI) is important in drug discovery. Revealing unknown compound-protein interactions is useful to design a new drug for a target protein by screening candidate compounds. The accurate CPI prediction assists in effective drug discovery process. To identify potential CPI effectively, prediction methods based on machine learning and deep learning have been developed. Data for sequences are provided as discrete symbolic data. In the data, compounds are represented as SMILES (simplified molecular-input line-entry system) strings and proteins are sequences in which the characters are amino acids. The outcome is defined as a variable that indicates how strong two molecules interact with each other or whether there is an interaction between them. In this package, a deep-learning based model that takes only sequence information of both compounds and proteins as input and the outcome as output is used to predict CPI. The model is implemented by using compound and protein encoders with useful features. The CPI model also supports other modeling tasks, including protein-protein interaction (PPI), chemical-chemical interaction (CCI), or single compounds and proteins. Although the model is designed for proteins, DNA and RNA can be used if they are represented as sequences.\n",
      "Version: 1.6.0\n",
      "Publication date: None\n",
      "Mantainer: Dongmin Jung <dmdmjung@gmail.com>\n",
      "Authors:\n",
      "Author: Dongmin Jung [cre, aut]\n",
      "Dependencies:\n",
      "  - keras, type: depends\n",
      "  - R (>= 4.1) type: depends\n",
      "  - tensorflow, type: imports\n",
      "  - CatEncoders, type: imports\n",
      "  - matlab, type: imports\n",
      "  - rcdk, type: imports\n",
      "  - stringdist, type: imports\n",
      "  - tokenizers, type: imports\n",
      "  - webchem, type: imports\n",
      "  - purrr, type: imports\n",
      "  - ttgsea, type: imports\n",
      "  - PRROC, type: imports\n",
      "  - reticulate, type: imports\n",
      "  - stats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/DeepPINCS.html\n",
      "\n",
      "Name: DMCHMM\n",
      "Description:\n",
      "A pipeline for identifying differentially methylated CpG sites using Hidden Markov Model in bisulfite sequencing data. DNA methylation studies have enabled researchers to understand methylation patterns and their regulatory roles in biological processes and disease. However, only a limited number of statistical approaches have been developed to provide formal quantitative analysis. Specifically, a few available methods do identify differentially methylated CpG (DMC) sites or regions (DMR), but they suffer from limitations that arise mostly due to challenges inherent in bisulfite sequencing data. These challenges include: (1) that read-depths vary considerably among genomic positions and are often low; (2) both methylation and autocorrelation patterns change as regions change; and (3) CpG sites are distributed unevenly. Furthermore, there are several methodological limitations: almost none of these tools is capable of comparing multiple groups and/or working with missing values, and only a few allow continuous or multiple covariates. The last of these is of great interest among researchers, as the goal is often to find which regions of the genome are associated with several exposures and traits. To tackle these issues, we have developed an efficient DMC identification method based on Hidden Markov Models (HMMs) called âDMCHMMâ which is a three-step approach (model selection, prediction, testing) aiming to address the aforementioned drawbacks.\n",
      "Version: 1.20.0\n",
      "Publication date: None\n",
      "Mantainer: Farhad Shokoohi <shokoohi@icloud.com>\n",
      "Authors:\n",
      "Author: Farhad Shokoohi\n",
      "Dependencies:\n",
      "  - R (>= 4.1.0) type: depends\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - methods, type: depends\n",
      "  - S4Vectors, type: depends\n",
      "  - BiocParallel, type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - IRanges, type: depends\n",
      "  - fdrtool, type: depends\n",
      "  - utils, type: imports\n",
      "  - stats, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - multcomp, type: imports\n",
      "  - calibrate, type: imports\n",
      "  - graphics, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/DMCHMM.html\n",
      "\n",
      "Name: easyreporting\n",
      "Description:\n",
      "An S4 class for facilitating the automated creation of rmarkdown files inside other packages/software even without knowing rmarkdown language. Best if implemented in functions as \"recursive\" style programming.\n",
      "Version: 1.10.0\n",
      "Publication date: None\n",
      "Mantainer: Dario Righelli <dario.righelli@gmail.com>\n",
      "Authors:\n",
      "Author: Dario Righelli [cre, aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - rmarkdown, type: imports\n",
      "  - methods, type: imports\n",
      "  - tools, type: imports\n",
      "  - shiny, type: imports\n",
      "  - rlang, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/easyreporting.html\n",
      "\n",
      "Name: EBSEA\n",
      "Description:\n",
      "Calculates differential expression of genes based on exon counts of genes obtained from RNA-seq sequencing data.\n",
      "Version: 1.26.0\n",
      "Publication date: None\n",
      "Mantainer: Arfa Mehmood <arfa.mehmood@utu.fi>\n",
      "Authors:\n",
      "Author: Arfa Mehmood, Asta Laiho, Laura L. Elo\n",
      "Dependencies:\n",
      "  - R (>= 4.0.0) type: depends\n",
      "  - DESeq2, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - EmpiricalBrownsMethod, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/EBSEA.html\n",
      "\n",
      "Name: epialleleR\n",
      "Description:\n",
      "Epialleles are specific DNA methylation patterns that are mitotically and/or meiotically inherited. This package calls hypermethylated epiallele frequencies at the level of genomic regions or individual cytosines in next-generation sequencing data using binary alignment map (BAM) files as an input. Other functionality includes extracting methylation patterns, computing the empirical cumulative distribution function for per-read beta values, and testing the significance of the association between epiallele methylation status and base frequencies at particular genomic positions (SNPs).\n",
      "Version: 1.6.1\n",
      "Publication date: None\n",
      "Mantainer: Oleksii Nikolaienko <oleksii.nikolaienko@gmail.com>\n",
      "Authors:\n",
      "Author: Oleksii Nikolaienko [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - stats, type: imports\n",
      "  - methods, type: imports\n",
      "  - utils, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - VariantAnnotation, type: imports\n",
      "  - stringi, type: imports\n",
      "  - data.table, type: imports\n",
      "  - Rcpp, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/epialleleR.html\n",
      "\n",
      "Name: FRGEpistasis\n",
      "Description:\n",
      "A Tool for Epistasis Analysis Based on Functional Regression Model\n",
      "Version: 1.34.0\n",
      "Publication date: None\n",
      "Mantainer: Futao Zhang <futoaz@gmail.com>\n",
      "Authors:\n",
      "Author: Futao Zhang\n",
      "Dependencies:\n",
      "  - R (>= 2.15) type: depends\n",
      "  - MASS, type: depends\n",
      "  - fda, type: depends\n",
      "  - methods, type: depends\n",
      "  - stats, type: depends\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/FRGEpistasis.html\n",
      "\n",
      "Name: funtooNorm\n",
      "Description:\n",
      "Provides a function to normalize Illumina Infinium Human Methylation 450 BeadChip (Illumina 450K), correcting for tissue and/or cell type.\n",
      "Version: 1.22.0\n",
      "Publication date: None\n",
      "Mantainer: Kathleen Klein <kathleen.klein@mail.mcgill.ca>\n",
      "Authors:\n",
      "Author: Celia Greenwood <celia.greenwood at mcgill.ca>,Stepan Grinek <stepan.grinek at ladydavis.ca>, Maxime Turgeon <maxime.turgeon at mail.mcgill.ca>, Kathleen Klein <kathleen.klein at mail.mcgill.ca>\n",
      "Dependencies:\n",
      "  - R (>= 3.4) type: depends\n",
      "  - pls, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - minfi, type: imports\n",
      "  - methods, type: imports\n",
      "  - IlluminaHumanMethylation450kmanifest, type: imports\n",
      "  - IlluminaHumanMethylation450kanno.ilmn12.hg19, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/funtooNorm.html\n",
      "\n",
      "Name: gaggle\n",
      "Description:\n",
      "This package contains functions enabling data exchange between R and Gaggle enabled bioinformatics software, including Cytoscape, Firegoose and Gaggle Genome Browser.\n",
      "Version: 1.66.0\n",
      "Publication date: None\n",
      "Mantainer: Christopher Bare <cbare@systemsbiology.org>\n",
      "Authors:\n",
      "Author: Paul Shannon <pshannon at systemsbiology.org>\n",
      "Dependencies:\n",
      "  - R (>= 2.3.0) type: depends\n",
      "  - rJava (>= 0.4) type: depends\n",
      "  - graph, type: depends\n",
      "  - RUnit (>= 0.4.17) type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL version 2 or newer\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/gaggle.html\n",
      "\n",
      "Name: geneAttribution\n",
      "Description:\n",
      "Identification of the most likely gene or genes through which variation at a given genomic locus in the human genome acts. The most basic functionality assumes that the closer gene is to the input locus, the more likely the gene is to be causative. Additionally, any empirical data that links genomic regions to genes (e.g. eQTL or genome conformation data) can be used if it is supplied in the UCSC .BED file format.\n",
      "Version: 1.24.0\n",
      "Publication date: None\n",
      "Mantainer: Arthur Wuster <wustera@gene.com>\n",
      "Authors:\n",
      "Author: Arthur Wuster\n",
      "Dependencies:\n",
      "  - utils, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - org.Hs.eg.db, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - GenomicFeatures, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/geneAttribution.html\n",
      "\n",
      "Name: hapFabia\n",
      "Description:\n",
      "A package to identify very short IBD segments in large sequencing data by FABIA biclustering. Two haplotypes are identical by descent (IBD) if they share a segment that both inherited from a common ancestor. Current IBD methods reliably detect long IBD segments because many minor alleles in the segment are concordant between the two haplotypes. However, many cohort studies contain unrelated individuals which share only short IBD segments. This package provides software to identify short IBD segments in sequencing data. Knowledge of short IBD segments are relevant for phasing of genotyping data, association studies, and for population genetics, where they shed light on the evolutionary history of humans. The package supports VCF formats, is based on sparse matrix operations, and provides visualization of haplotype clusters in different formats.\n",
      "Version: 1.40.0\n",
      "Publication date: None\n",
      "Mantainer: Andreas Mitterecker <mitterecker@bioinf.jku.at>\n",
      "Authors:\n",
      "Author: Sepp Hochreiter <hochreit at bioinf.jku.at>\n",
      "Dependencies:\n",
      "  - R (>= 3.6.0) type: depends\n",
      "  - Biobase, type: depends\n",
      "  - fabia, type: depends\n",
      "  - methods, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL (>= 2.1)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/hapFabia.html\n",
      "\n",
      "Name: HiLDA\n",
      "Description:\n",
      "A package built under the Bayesian framework of applying hierarchical latent Dirichlet allocation. It statistically tests whether the mutational exposures of mutational signatures (Shiraishi-model signatures) are different between two groups. The package also provides inference and visualization.\n",
      "Version: 1.12.0\n",
      "Publication date: None\n",
      "Mantainer: Zhi Yang <zyang895@gmail.com>\n",
      "Authors:\n",
      "Author: Zhi Yang [aut, cre], Yuichi Shiraishi [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - ggplot2, type: depends\n",
      "  - R2jags, type: imports\n",
      "  - abind, type: imports\n",
      "  - cowplot, type: imports\n",
      "  - grid, type: imports\n",
      "  - forcats, type: imports\n",
      "  - stringr, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - XVector, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - GenomicFeatures, type: imports\n",
      "  - BSgenome.Hsapiens.UCSC.hg19, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - stats, type: imports\n",
      "  - TxDb.Hsapiens.UCSC.hg19.knownGene, type: imports\n",
      "  - utils, type: imports\n",
      "  - methods, type: imports\n",
      "  - Rcpp, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/HiLDA.html\n",
      "\n",
      "Name: icetea\n",
      "Description:\n",
      "icetea (Integrating Cap Enrichment with Transcript Expression Analysis) provides functions for end-to-end analysis of multiple 5'-profiling methods such as CAGE, RAMPAGE and MAPCap, beginning from raw reads to detection of transcription start sites using replicates. It also allows performing differential TSS detection between group of samples, therefore, integrating the mRNA cap enrichment information with transcript expression analysis.\n",
      "Version: 1.16.0\n",
      "Publication date: None\n",
      "Mantainer: Vivek Bhardwaj <v.bhardwaj@hubrecht.eu>\n",
      "Authors:\n",
      "Author: Vivek Bhardwaj [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - methods, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - GenomicFeatures, type: imports\n",
      "  - ShortRead, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - GenomicAlignments, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - VariantAnnotation, type: imports\n",
      "  - limma, type: imports\n",
      "  - edgeR, type: imports\n",
      "  - csaw, type: imports\n",
      "  - DESeq2, type: imports\n",
      "  - TxDb.Dmelanogaster.UCSC.dm6.ensGene, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3 + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/icetea.html\n",
      "\n",
      "Name: iterativeBMAsurv\n",
      "Description:\n",
      "The iterative Bayesian Model Averaging (BMA) algorithm for survival analysis is a variable selection method for applying survival analysis to microarray data.\n",
      "Version: 1.56.0\n",
      "Publication date: None\n",
      "Mantainer: Ka Yee Yeung <kayee@u.washington.edu>\n",
      "Authors:\n",
      "Author: Amalia Annest, University of Washington, Tacoma, WA Ka Yee Yeung, University of Washington, Seattle, WA\n",
      "Dependencies:\n",
      "  - BMA, type: depends\n",
      "  - leaps, type: depends\n",
      "  - survival, type: depends\n",
      "  - splines, type: depends\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - stats, type: imports\n",
      "  - survival, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/iterativeBMAsurv.html\n",
      "\n",
      "Name: LACE\n",
      "Description:\n",
      "LACE is an algorithmic framework that processes single-cell somatic mutation profiles from cancer samples collected at different time points and in distinct experimental settings, to produce longitudinal models of cancer evolution. The approach solves a Boolean Matrix Factorization problem with phylogenetic constraints, by maximizing a weighed likelihood function computed on multiple time points.\n",
      "Version: 2.2.0\n",
      "Publication date: None\n",
      "Mantainer: Davide Maspero <d.maspero@campus.unimib.it>\n",
      "Authors:\n",
      "Author: Daniele Ramazzotti [aut] , Fabrizio Angaroni [aut], Davide Maspero [cre, aut], Alex Graudenzi [aut], Luca De Sano [aut] , Gianluca Ascolani [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.1.0) type: depends\n",
      "  - curl, type: imports\n",
      "  - igraph, type: imports\n",
      "  - foreach, type: imports\n",
      "  - doParallel, type: imports\n",
      "  - sortable, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - data.tree, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - parallel, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - Rfast, type: imports\n",
      "  - stats, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - utils, type: imports\n",
      "  - purrr, type: imports\n",
      "  - stringi, type: imports\n",
      "  - stringr, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - jsonlite, type: imports\n",
      "  - readr, type: imports\n",
      "  - configr, type: imports\n",
      "  - DT, type: imports\n",
      "  - tools, type: imports\n",
      "  - fs, type: imports\n",
      "  - data.table, type: imports\n",
      "  - htmltools, type: imports\n",
      "  - htmlwidgets, type: imports\n",
      "  - bsplus, type: imports\n",
      "  - shinyvalidate, type: imports\n",
      "  - shiny, type: imports\n",
      "  - shinythemes, type: imports\n",
      "  - shinyFiles, type: imports\n",
      "  - shinyjs, type: imports\n",
      "  - shinyBS, type: imports\n",
      "  - shinydashboard, type: imports\n",
      "  - biomaRt, type: imports\n",
      "  - callr, type: imports\n",
      "  - logr, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/LACE.html\n",
      "\n",
      "Name: LineagePulse\n",
      "Description:\n",
      "LineagePulse is a differential expression and expression model fitting package tailored to single-cell RNA-seq data (scRNA-seq). LineagePulse accounts for batch effects, drop-out and variable sequencing depth. One can use LineagePulse to perform longitudinal differential expression analysis across pseudotime as a continuous coordinate or between discrete groups of cells (e.g. pre-defined clusters or experimental conditions). Expression model fits can be directly extracted from LineagePulse.\n",
      "Version: 1.18.0\n",
      "Publication date: None\n",
      "Mantainer: David S Fischer <david.fischer@helmholtz-muenchen.de>\n",
      "Authors:\n",
      "Author: David S Fischer [aut, cre], Fabian Theis [ctb], Nir Yosef [ctb]\n",
      "Dependencies:\n",
      "  - BiocParallel, type: imports\n",
      "  - circlize, type: imports\n",
      "  - compiler, type: imports\n",
      "  - ComplexHeatmap, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - gplots, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - grid, type: imports\n",
      "  - knitr, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - methods, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - SingleCellExperiment, type: imports\n",
      "  - splines, type: imports\n",
      "  - stats, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/LineagePulse.html\n",
      "\n",
      "Name: LRcell\n",
      "Description:\n",
      "The goal of LRcell is to identify specific sub-cell types that drives the changes observed in a bulk RNA-seq differential gene expression experiment. To achieve this, LRcell utilizes sets of cell marker genes acquired from single-cell RNA-sequencing (scRNA-seq) as indicators for various cell types in the tissue of interest. Next, for each cell type, using its marker genes as indicators, we apply Logistic Regression on the complete set of genes with differential expression p-values to calculate a cell-type significance p-value. Finally, these p-values are compared to predict which one(s) are likely to be responsible for the differential gene expression pattern observed in the bulk RNA-seq experiments. LRcell is inspired by the LRpath[@sartor2009lrpath] algorithm developed by Sartor et al., originally designed for pathway/gene set enrichment analysis. LRcell contains three major components: LRcell analysis, plot generation and marker gene selection. All modules in this package are written in R. This package also provides marker genes in the Prefrontal Cortex (pFC) human brain region, human PBMC and nine mouse brain regions (Frontal Cortex, Cerebellum, Globus Pallidus, Hippocampus, Entopeduncular, Posterior Cortex, Striatum, Substantia Nigra and Thalamus).\n",
      "Version: 1.6.0\n",
      "Publication date: None\n",
      "Mantainer: Wenjing Ma <wenjing.ma@emory.edu>\n",
      "Authors:\n",
      "Author: Wenjing Ma [cre, aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - ExperimentHub, type: depends\n",
      "  - AnnotationHub, type: depends\n",
      "  - BiocParallel, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - ggrepel, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/LRcell.html\n",
      "\n",
      "Name: MADSEQ\n",
      "Description:\n",
      "The MADSEQ package provides a group of hierarchical Bayeisan models for the detection of mosaic aneuploidy, the inference of the type of aneuploidy and also for the quantification of the fraction of aneuploid cells in the sample.\n",
      "Version: 1.24.0\n",
      "Publication date: None\n",
      "Mantainer: Yu Kong <yu.kong@phd.einstein.yu.edu>\n",
      "Authors:\n",
      "Author: Yu Kong, Adam Auton, John Murray Greally\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - rjags (>= 4.6) type: depends\n",
      "  - VGAM, type: imports\n",
      "  - coda, type: imports\n",
      "  - BSgenome, type: imports\n",
      "  - BSgenome.Hsapiens.UCSC.hg19, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - methods, type: imports\n",
      "  - preprocessCore, type: imports\n",
      "  - GenomicAlignments, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - VariantAnnotation, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - utils, type: imports\n",
      "  - zlibbioc, type: imports\n",
      "  - vcfR, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL(>=2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/MADSEQ.html\n",
      "\n",
      "Name: MAGAR\n",
      "Description:\n",
      "\"Methylation-Aware Genotype Association in R\" (MAGAR) computes methQTL from DNA methylation and genotyping data from matched samples. MAGAR uses a linear modeling stragety to call CpGs/SNPs that are methQTLs. MAGAR accounts for the local correlation structure of CpGs.\n",
      "Version: 1.6.0\n",
      "Publication date: None\n",
      "Mantainer: Michael Scherer <mscherer@mpi-inf.mpg.de>\n",
      "Authors:\n",
      "Author: Michael Scherer [cre, aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - HDF5Array, type: depends\n",
      "  - RnBeads, type: depends\n",
      "  - snpStats, type: depends\n",
      "  - crlmm, type: depends\n",
      "  - doParallel, type: imports\n",
      "  - igraph, type: imports\n",
      "  - bigstatsr, type: imports\n",
      "  - rjson, type: imports\n",
      "  - plyr, type: imports\n",
      "  - data.table, type: imports\n",
      "  - UpSetR, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - jsonlite, type: imports\n",
      "  - methods, type: imports\n",
      "  - ff, type: imports\n",
      "  - argparse, type: imports\n",
      "  - impute, type: imports\n",
      "  - RnBeads.hg19, type: imports\n",
      "  - utils, type: imports\n",
      "  - stats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/MAGAR.html\n",
      "\n",
      "Name: mBPCR\n",
      "Description:\n",
      "It contains functions for estimating the DNA copy number profile using mBPCR with the aim of detecting regions with copy number changes.\n",
      "Version: 1.52.0\n",
      "Publication date: None\n",
      "Mantainer: P.M.V. Rancoita <rancoita.paola@gmail.com>\n",
      "Authors:\n",
      "Author: P.M.V. Rancoita <rancoita.paola at gmail.com>, with contributions from M. Hutter <marcus.hutter at anu.edu.au>\n",
      "Dependencies:\n",
      "  - oligoClasses, type: depends\n",
      "  - GWASTools, type: depends\n",
      "  - Biobase, type: imports\n",
      "  - graphics, type: imports\n",
      "  - methods, type: imports\n",
      "  - utils, type: imports\n",
      "  - grDevices, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/mBPCR.html\n",
      "\n",
      "Name: MDTS\n",
      "Description:\n",
      "A package for the detection of de novo copy number deletions in targeted sequencing of trios with high sensitivity and positive predictive value.\n",
      "Version: 1.18.0\n",
      "Publication date: None\n",
      "Mantainer: Jack M.. Fu <jmfu@jhsph.edu>\n",
      "Authors:\n",
      "Author: Jack M.. Fu [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - GenomicAlignments, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - DNAcopy, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - parallel, type: imports\n",
      "  - stringr, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/MDTS.html\n",
      "\n",
      "Name: messina\n",
      "Description:\n",
      "Messina is a collection of algorithms for constructing optimally robust single-gene classifiers, and for identifying differential expression in the presence of outliers or unknown sample subgroups. The methods have application in identifying lead features to develop into clinical tests (both diagnostic and prognostic), and in identifying differential expression when a fraction of samples show unusual patterns of expression.\n",
      "Version: 1.34.0\n",
      "Publication date: None\n",
      "Mantainer: Mark Pinese <mpinese@ccia.org.au>\n",
      "Authors:\n",
      "Author: Mark Pinese [aut], Mark Pinese [cre], Mark Pinese [cph]\n",
      "Dependencies:\n",
      "  - R (>= 3.1.0) type: depends\n",
      "  - survival (>= 2.37-4) type: depends\n",
      "  - methods, type: depends\n",
      "  - Rcpp (>= 0.11.1) type: imports\n",
      "  - plyr (>= 1.8) type: imports\n",
      "  - ggplot2 (>= 0.9.3.1) type: imports\n",
      "  - grid (>= 3.1.0) type: imports\n",
      "  - foreach (>= 1.4.1) type: imports\n",
      "  - graphics, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "EPL (>= 1.0)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/messina.html\n",
      "\n",
      "Name: methylMnM\n",
      "Description:\n",
      "To give the exactly p-value and q-value of MeDIP-seq and MRE-seq data for different samples comparation.\n",
      "Version: 1.36.0\n",
      "Publication date: None\n",
      "Mantainer: Yan Zhou<zhouy1016@163.com>\n",
      "Authors:\n",
      "Author: Yan Zhou, Bo Zhang, Nan Lin, BaoXue Zhang and Ting Wang\n",
      "Dependencies:\n",
      "  - R (>= 2.12.1) type: depends\n",
      "  - edgeR, type: depends\n",
      "  - statmod, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/methylMnM.html\n",
      "\n",
      "Name: mimager\n",
      "Description:\n",
      "Easily visualize and inspect microarrays for spatial artifacts.\n",
      "Version: 1.22.0\n",
      "Publication date: None\n",
      "Mantainer: Aaron Wolen <aaron@wolen.com>\n",
      "Authors:\n",
      "Author: Aaron Wolen [aut, cre, cph]\n",
      "Dependencies:\n",
      "  - Biobase, type: depends\n",
      "  - BiocGenerics, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - preprocessCore, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - methods, type: imports\n",
      "  - grid, type: imports\n",
      "  - gtable, type: imports\n",
      "  - scales, type: imports\n",
      "  - DBI, type: imports\n",
      "  - affy, type: imports\n",
      "  - affyPLM, type: imports\n",
      "  - oligo, type: imports\n",
      "  - oligoClasses, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/mimager.html\n",
      "\n",
      "Name: mirIntegrator\n",
      "Description:\n",
      "Tools for augmenting signaling pathways to perform pathway analysis of microRNA and mRNA expression levels.\n",
      "Version: 1.28.0\n",
      "Publication date: None\n",
      "Mantainer: Diana Diaz <dmd@wayne.edu>\n",
      "Authors:\n",
      "Author: Diana Diaz <dmd at wayne dot edu>\n",
      "Dependencies:\n",
      "  - R (>= 3.3) type: depends\n",
      "  - graph, type: imports\n",
      "  - ROntoTools, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - org.Hs.eg.db, type: imports\n",
      "  - AnnotationDbi, type: imports\n",
      "  - Rgraphviz, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>=3)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/mirIntegrator.html\n",
      "\n",
      "Name: MOSim\n",
      "Description:\n",
      "MOSim package simulates multi-omic experiments that mimic regulatory mechanisms within the cell, allowing flexible experimental design including time course and multiple groups.\n",
      "Version: 1.12.0\n",
      "Publication date: None\n",
      "Mantainer: Carlos MartÃ­nez <cmarmir@gmail.com>\n",
      "Authors:\n",
      "Author: Carlos MartÃ­nez [cre, aut], Sonia Tarazona [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.6) type: depends\n",
      "  - HiddenMarkov, type: imports\n",
      "  - zoo, type: imports\n",
      "  - methods, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - stringi, type: imports\n",
      "  - lazyeval, type: imports\n",
      "  - rlang, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - purrr, type: imports\n",
      "  - scales, type: imports\n",
      "  - stringr, type: imports\n",
      "  - tibble, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/MOSim.html\n",
      "\n",
      "Name: msgbsR\n",
      "Description:\n",
      "Pipeline for the anaysis of a MS-GBS experiment.\n",
      "Version: 1.22.0\n",
      "Publication date: None\n",
      "Mantainer: Benjamin Mayne <benjamin.mayne@adelaide.edu.au>\n",
      "Authors:\n",
      "Author: Benjamin Mayne\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - methods, type: depends\n",
      "  - BSgenome, type: imports\n",
      "  - easyRNASeq, type: imports\n",
      "  - edgeR, type: imports\n",
      "  - GenomicAlignments, type: imports\n",
      "  - GenomicFeatures, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - ggbio, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - parallel, type: imports\n",
      "  - plyr, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - R.utils, type: imports\n",
      "  - stats, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/msgbsR.html\n",
      "\n",
      "Name: MSstatsQCgui\n",
      "Description:\n",
      "MSstatsQCgui is a Shiny app which provides longitudinal system suitability monitoring and quality control tools for proteomic experiments.\n",
      "Version: 1.18.0\n",
      "Publication date: None\n",
      "Mantainer: Eralp Dogu <eralp.dogu@gmail.com>\n",
      "Authors:\n",
      "Author: Eralp Dogu [aut, cre], Sara Taheri [aut], Olga Vitek [aut]\n",
      "Dependencies:\n",
      "  - shiny, type: imports\n",
      "  - MSstatsQC, type: imports\n",
      "  - ggExtra, type: imports\n",
      "  - gridExtra, type: imports\n",
      "  - plotly, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - grid, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic License 2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/MSstatsQCgui.html\n",
      "\n",
      "Name: MSstatsSampleSize\n",
      "Description:\n",
      "The packages estimates the variance in the input protein abundance data and simulates data with predefined number of biological replicates based on the variance estimation. It reports the mean predictive accuracy of the classifier and mean protein importance over multiple iterations of the simulation.\n",
      "Version: 1.12.0\n",
      "Publication date: None\n",
      "Mantainer: Ting Huang <thuang0703@gmail.com>\n",
      "Authors:\n",
      "Author: Ting Huang [aut, cre], Meena Choi [aut], Olga Vitek [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.6) type: depends\n",
      "  - ggplot2, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - caret, type: imports\n",
      "  - gridExtra, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - MSstats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/MSstatsSampleSize.html\n",
      "\n",
      "Name: NBSplice\n",
      "Description:\n",
      "The package proposes a differential splicing evaluation method based on isoform quantification. It applies generalized linear models with negative binomial distribution to infer changes in isoform relative expression.\n",
      "Version: 1.15.0\n",
      "Publication date: None\n",
      "Mantainer: Gabriela Merino <merino.gabriela33@gmail.com>\n",
      "Authors:\n",
      "Author: Gabriela A. Merino and Elmer A. Fernandez\n",
      "Dependencies:\n",
      "  - R (>= 3.5) type: depends\n",
      "  - methods, type: depends\n",
      "  - edgeR, type: imports\n",
      "  - stats, type: imports\n",
      "  - MASS, type: imports\n",
      "  - car, type: imports\n",
      "  - mppa, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - reshape2, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>=2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/NBSplice.html\n",
      "\n",
      "Name: netOmics\n",
      "Description:\n",
      "netOmics is a multi-omics networks builder and explorer. It uses a combination of network inference algorithms and and knowledge-based graphs to build multi-layered networks. The package can be combined with timeOmics to incorporate time-course expression data and build sub-networks from multi-omics kinetic clusters. Finally, from the generated multi-omics networks, propagation analyses allow the identification of missing biological functions (1), multi-omics mechanisms (2) and molecules between kinetic clusters (3). This helps to resolve complex regulatory mechanisms.\n",
      "Version: 1.4.0\n",
      "Publication date: None\n",
      "Mantainer: Antoine Bodein <antoine.bodein.1@ulaval.ca>\n",
      "Authors:\n",
      "Author: Antoine Bodein [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - dplyr, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - igraph, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - minet, type: imports\n",
      "  - purrr, type: imports\n",
      "  - tibble, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - AnnotationDbi, type: imports\n",
      "  - GO.db, type: imports\n",
      "  - RandomWalkRestartMH, type: imports\n",
      "  - gprofiler2, type: imports\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/netOmics.html\n",
      "\n",
      "Name: OmicsLonDA\n",
      "Description:\n",
      "Statistical method that provides robust identification of time intervals where omics features (such as proteomics, lipidomics, metabolomics, transcriptomics, microbiome, as well as physiological parameters captured by wearable sensors such as heart rhythm, body temperature, and activity level) are significantly different between groups.\n",
      "Version: 1.14.0\n",
      "Publication date: None\n",
      "Mantainer: Ahmed A. Metwally <ametwall@stanford.edu>\n",
      "Authors:\n",
      "Author: Ahmed A. Metwally, Tom Zhang, Michael Snyder\n",
      "Dependencies:\n",
      "  - R (>= 3.6) type: depends\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - gss, type: imports\n",
      "  - plyr, type: imports\n",
      "  - zoo, type: imports\n",
      "  - pracma, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - parallel, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - methods, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/OmicsLonDA.html\n",
      "\n",
      "Name: OMICsPCA\n",
      "Description:\n",
      "OMICsPCA is an analysis pipeline designed to integrate multi OMICs experiments done on various subjects (e.g. Cell lines, individuals), treatments (e.g. disease/control) or time points and to analyse such integrated data from various various angles and perspectives. In it's core OMICsPCA uses Principal Component Analysis (PCA) to integrate multiomics experiments from various sources and thus has ability to over data insufficiency issues by using the ingegrated data as representatives. OMICsPCA can be used in various application including analysis of overall distribution of OMICs assays across various samples /individuals /time points; grouping assays by user-defined conditions; identification of source of variation, similarity/dissimilarity between assays, variables or individuals.\n",
      "Version: 1.16.0\n",
      "Publication date: None\n",
      "Mantainer: Subhadeep Das <subhadeep1024@gmail.com>\n",
      "Authors:\n",
      "Author: Subhadeep Das [aut, cre], Dr. Sucheta Tripathy [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - OMICsPCAdata, type: depends\n",
      "  - HelloRanges, type: imports\n",
      "  - fpc, type: imports\n",
      "  - stats, type: imports\n",
      "  - MultiAssayExperiment, type: imports\n",
      "  - pdftools, type: imports\n",
      "  - methods, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - utils, type: imports\n",
      "  - clValid, type: imports\n",
      "  - NbClust, type: imports\n",
      "  - cowplot, type: imports\n",
      "  - rmarkdown, type: imports\n",
      "  - kableExtra, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - factoextra, type: imports\n",
      "  - rgl, type: imports\n",
      "  - corrplot, type: imports\n",
      "  - MASS, type: imports\n",
      "  - graphics, type: imports\n",
      "  - FactoMineR, type: imports\n",
      "  - PerformanceAnalytics, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - data.table, type: imports\n",
      "  - cluster, type: imports\n",
      "  - magick, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/OMICsPCA.html\n",
      "\n",
      "Name: ORFhunteR\n",
      "Description:\n",
      "The ORFhunteR package is a R and C++ library for an automatic determination and annotation of open reading frames (ORF) in a large set of RNA molecules. It efficiently implements the machine learning model based on vectorization of nucleotide sequences and the random forest classification algorithm. The ORFhunteR package consists of a set of functions written in the R language in conjunction with C++. The efficiency of the package was confirmed by the examples of the analysis of RNA molecules from the NCBI RefSeq and Ensembl databases. The package can be used in basic and applied biomedical research related to the study of the transcriptome of normal as well as altered (for example, cancer) human cells.\n",
      "Version: 1.6.0\n",
      "Publication date: None\n",
      "Mantainer: Vasily V. Grinev <grinev_vv@bsu.by>\n",
      "Authors:\n",
      "Author: Vasily V. Grinev [aut, cre] , Mikalai M. Yatskou [aut], Victor V. Skakun [aut], Maryna Chepeleva [aut], Petr V. Nazarov [aut]\n",
      "Dependencies:\n",
      "  - Biostrings, type: depends\n",
      "  - rtracklayer, type: depends\n",
      "  - Peptides, type: depends\n",
      "  - Rcpp (>= 1.0.3) type: imports\n",
      "  - BSgenome.Hsapiens.UCSC.hg38, type: imports\n",
      "  - data.table, type: imports\n",
      "  - stringr, type: imports\n",
      "  - randomForest, type: imports\n",
      "  - xfun, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - parallel, type: imports\n",
      "  - graphics, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ORFhunteR.html\n",
      "\n",
      "Name: periodicDNA\n",
      "Description:\n",
      "This R package helps the user identify k-mers (e.g. di- or tri-nucleotides) present periodically in a set of genomic loci (typically regulatory elements). The functions of this package provide a straightforward approach to find periodic occurrences of k-mers in DNA sequences, such as regulatory elements. It is not aimed at identifying motifs separated by a conserved distance; for this type of analysis, please visit MEME website.\n",
      "Version: 1.8.0\n",
      "Publication date: None\n",
      "Mantainer: Jacques Serizay <jacquesserizay@gmail.com>\n",
      "Authors:\n",
      "Author: Jacques Serizay [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - Biostrings, type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - IRanges, type: depends\n",
      "  - BSgenome, type: depends\n",
      "  - BiocParallel, type: depends\n",
      "  - S4Vectors, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - stats, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - zoo, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - methods, type: imports\n",
      "  - parallel, type: imports\n",
      "  - cowplot, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3 + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/periodicDNA.html\n",
      "\n",
      "Name: ReQON\n",
      "Description:\n",
      "Algorithm for recalibrating the base quality scores for aligned sequencing data in BAM format.\n",
      "Version: 1.44.0\n",
      "Publication date: None\n",
      "Mantainer: Christopher Cabanski <cabanskc@gmail.com>\n",
      "Authors:\n",
      "Author: Christopher Cabanski, Keary Cavin, Chris Bizon\n",
      "Dependencies:\n",
      "  - R (>= 3.0.2) type: depends\n",
      "  - Rsamtools, type: depends\n",
      "  - seqbias, type: depends\n",
      "  - rJava, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - grDevices, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ReQON.html\n",
      "\n",
      "Name: RGraph2js\n",
      "Description:\n",
      "Generator of web pages which display interactive network/graph visualizations with D3js, jQuery and Raphael.\n",
      "Version: 1.26.0\n",
      "Publication date: None\n",
      "Mantainer: Stephane Cano <DL.RSupport@pmi.com>\n",
      "Authors:\n",
      "Author: Stephane Cano [aut, cre], Sylvain Gubian [aut], Florian Martin [aut]\n",
      "Dependencies:\n",
      "  - utils, type: imports\n",
      "  - whisker, type: imports\n",
      "  - rjson, type: imports\n",
      "  - digest, type: imports\n",
      "  - graph, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/RGraph2js.html\n",
      "\n",
      "Name: rmspc\n",
      "Description:\n",
      "The rmspc package runs MSPC (Multiple Sample Peak Calling) software using R. The analysis of ChIP-seq samples outputs a number of enriched regions (commonly known as \"peaks\"), each indicating a protein-DNA interaction or a specific chromatin modification. When replicate samples are analyzed, overlapping peaks are expected. This repeated evidence can therefore be used to locally lower the minimum significance required to accept a peak. MSPC uses combined evidence from replicated experiments to evaluate peak calling output, rescuing peaks, and reduce false positives. It takes any number of replicates as input and improves sensitivity and specificity of peak calling on each, and identifies consensus regions between the input samples.\n",
      "Version: 1.4.0\n",
      "Publication date: None\n",
      "Mantainer: Meriem Bahda <meriembahda@gmail.com>\n",
      "Authors:\n",
      "Author: Vahid Jalili [aut], Marzia Angela Cremona [aut], Fernando Palluzzi [aut], Meriem Bahda [aut, cre]\n",
      "Dependencies:\n",
      "  - processx, type: imports\n",
      "  - BiocManager, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - stats, type: imports\n",
      "  - tools, type: imports\n",
      "  - methods, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - stringr, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/rmspc.html\n",
      "\n",
      "Name: rqt\n",
      "Description:\n",
      "Despite the recent advances of modern GWAS methods, it still remains an important problem of addressing calculation an effect size and corresponding p-value for the whole gene rather than for single variant. The R- package rqt offers gene-level GWAS meta-analysis. For more information, see: \"Gene-set association tests for next-generation sequencing data\" by Lee et al (2016), Bioinformatics, 32(17), i611-i619, .\n",
      "Version: 1.24.0\n",
      "Publication date: None\n",
      "Mantainer: Ilya Y. Zhbannikov <ilya.zhbannikov@duke.edu>\n",
      "Authors:\n",
      "Author: I. Y. Zhbannikov, K. G. Arbeev, A. I. Yashin.\n",
      "Dependencies:\n",
      "  - R (>= 3.4) type: depends\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - stats, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - ropls, type: imports\n",
      "  - methods, type: imports\n",
      "  - car, type: imports\n",
      "  - RUnit, type: imports\n",
      "  - metap, type: imports\n",
      "  - CompQuadForm, type: imports\n",
      "  - glmnet, type: imports\n",
      "  - utils, type: imports\n",
      "  - pls, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/rqt.html\n",
      "\n",
      "Name: rScudo\n",
      "Description:\n",
      "SCUDO (Signature-based Clustering for Diagnostic Purposes) is a rank-based method for the analysis of gene expression profiles for diagnostic and classification purposes. It is based on the identification of sample-specific gene signatures composed of the most up- and down-regulated genes for that sample. Starting from gene expression data, functions in this package identify sample-specific gene signatures and use them to build a graph of samples. In this graph samples are joined by edges if they have a similar expression profile, according to a pre-computed similarity matrix. The similarity between the expression profiles of two samples is computed using a method similar to GSEA. The graph of samples can then be used to perform community clustering or to perform supervised classification of samples in a testing set.\n",
      "Version: 1.14.0\n",
      "Publication date: None\n",
      "Mantainer: Matteo Ciciani <matteo.ciciani@gmail.com>\n",
      "Authors:\n",
      "Author: Matteo Ciciani [aut, cre], Thomas Cantore [aut], Enrica Colasurdo [ctb], Mario Lauria [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 3.6) type: depends\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - igraph, type: imports\n",
      "  - stringr, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/rScudo.html\n",
      "\n",
      "Name: sampleClassifier\n",
      "Description:\n",
      "The package is designed to classify microarray RNA-seq gene expression profiles.\n",
      "Version: 1.22.0\n",
      "Publication date: None\n",
      "Mantainer: Khadija El Amrani <a.khadija@gmx.de>\n",
      "Authors:\n",
      "Author: Khadija El Amrani [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - MGFM, type: depends\n",
      "  - MGFR, type: depends\n",
      "  - annotate, type: depends\n",
      "  - e1071, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/sampleClassifier.html\n",
      "\n",
      "Name: scAlign\n",
      "Description:\n",
      "An unsupervised deep learning method for data alignment, integration and estimation of per-cell differences in -omic data (e.g. gene expression) across datasets (conditions, tissues, species). See Johansen and Quon (2019) for more details.\n",
      "Version: 1.11.0\n",
      "Publication date: None\n",
      "Mantainer: Nelson Johansen <njjohansen@ucdavis.edu>\n",
      "Authors:\n",
      "Author: Nelson Johansen [aut, cre], Gerald Quon [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.6) type: depends\n",
      "  - SingleCellExperiment, type: depends\n",
      "  - Seurat (>= 2.3.4) type: depends\n",
      "  - tensorflow, type: depends\n",
      "  - purrr, type: depends\n",
      "  - irlba, type: depends\n",
      "  - Rtsne, type: depends\n",
      "  - ggplot2, type: depends\n",
      "  - methods, type: depends\n",
      "  - utils, type: depends\n",
      "  - FNN, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/scAlign.html\n",
      "\n",
      "Name: scanMiR\n",
      "Description:\n",
      "A set of tools for working with miRNA affinity models (KdModels), efficiently scanning for miRNA binding sites, and predicting target repression. It supports scanning using miRNA seeds, full miRNA sequences (enabling 3' alignment) and KdModels, and includes the prediction of slicing and TDMD sites. Finally, it includes utility and plotting functions (e.g. for the visual representation of miRNA-target alignment).\n",
      "Version: 1.4.0\n",
      "Publication date: None\n",
      "Mantainer: Fridolin Gross <fridoling@hotmail.com>\n",
      "Authors:\n",
      "Author: Pierre-Luc Germain [aut] , Michael Soutschek [aut], Fridolin Gross [cre, aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - Biostrings, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - data.table, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - methods, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - stats, type: imports\n",
      "  - stringi, type: imports\n",
      "  - utils, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grid, type: imports\n",
      "  - ggseqlogo, type: imports\n",
      "  - cowplot, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/scanMiR.html\n",
      "\n",
      "Name: SCArray\n",
      "Description:\n",
      "Provides large-scale single-cell RNA-seq data manipulation using Genomic Data Structure (GDS) files. It combines dense and sparse matrices stored in GDS files and the Bioconductor infrastructure framework (SingleCellExperiment and DelayedArray) to provide out-of-memory data storage and large-scale manipulation using the R programming language.\n",
      "Version: 1.6.0\n",
      "Publication date: None\n",
      "Mantainer: Xiuwen Zheng <xiuwen.zheng@abbvie.com>\n",
      "Authors:\n",
      "Author: Xiuwen Zheng [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - gdsfmt, type: depends\n",
      "  - methods, type: depends\n",
      "  - DelayedArray, type: depends\n",
      "  - BiocGenerics, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - utils, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - SingleCellExperiment, type: imports\n",
      "  - DelayedMatrixStats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/SCArray.html\n",
      "\n",
      "Name: SELEX\n",
      "Description:\n",
      "Tools for quantifying DNA binding specificities based on SELEX-seq data.\n",
      "Version: 1.30.0\n",
      "Publication date: None\n",
      "Mantainer: Harmen J. Bussemaker <hjb2004@columbia.edu>\n",
      "Authors:\n",
      "Author: Chaitanya Rastogi, Dahong Liu, Lucas Melo, and Harmen J. Bussemaker\n",
      "Dependencies:\n",
      "  - rJava (>= 0.5-0) type: depends\n",
      "  - Biostrings, type: depends\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>=2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/SELEX.html\n",
      "\n",
      "Name: SigCheck\n",
      "Description:\n",
      "While gene signatures are frequently used to predict phenotypes (e.g. predict prognosis of cancer patients), it it not always clear how optimal or meaningful they are (cf David Venet, Jacques E. Dumont, and Vincent Detours' paper \"Most Random Gene Expression Signatures Are Significantly Associated with Breast Cancer Outcome\"). Based on suggestions in that paper, SigCheck accepts a data set (as an ExpressionSet) and a gene signature, and compares its performance on survival and/or classification tasks against a) random gene signatures of the same length; b) known, related and unrelated gene signatures; and c) permuted data and/or metadata.\n",
      "Version: 2.30.0\n",
      "Publication date: None\n",
      "Mantainer: Rory Stark <rory.stark@cruk.cam.ac.uk>\n",
      "Authors:\n",
      "Author: Rory Stark <rory.stark at cruk.cam.ac.uk> and Justin Norden\n",
      "Dependencies:\n",
      "  - R (>= 3.2.0) type: depends\n",
      "  - MLInterfaces, type: depends\n",
      "  - Biobase, type: depends\n",
      "  - e1071, type: depends\n",
      "  - BiocParallel, type: depends\n",
      "  - survival, type: depends\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - methods, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/SigCheck.html\n",
      "\n",
      "Name: SingleMoleculeFootprinting\n",
      "Description:\n",
      "SingleMoleculeFootprinting is an R package providing functions to analyze Single Molecule Footprinting (SMF) data. Following the workflow exemplified in its vignette, the user will be able to perform basic data analysis of SMF data with minimal coding effort. Starting from an aligned bam file, we show how to perform quality controls over sequencing libraries, extract methylation information at the single molecule level accounting for the two possible kind of SMF experiments (single enzyme or double enzyme), classify single molecules based on their patterns of molecular occupancy, plot SMF information at a given genomic location\n",
      "Version: 1.6.0\n",
      "Publication date: None\n",
      "Mantainer: Guido Barzaghi <guido.barzaghi@embl.de>\n",
      "Authors:\n",
      "Author: Guido Barzaghi [aut, cre] , Arnaud Krebs [aut] , Mike Smith [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 4.1.0) type: depends\n",
      "  - BiocGenerics, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - BSgenome, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - data.table, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - plyr, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - stats, type: imports\n",
      "  - QuasR, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/SingleMoleculeFootprinting.html\n",
      "\n",
      "Name: SISPA\n",
      "Description:\n",
      "Sample Integrated Set Profile Analysis (SISPA) is a method designed to define sample groups with similar gene set enrichment profiles.\n",
      "Version: 1.28.0\n",
      "Publication date: None\n",
      "Mantainer: Bhakti Dwivedi <bhakti.dwivedi@emory.edu>\n",
      "Authors:\n",
      "Author: Bhakti Dwivedi and Jeanne Kowalski\n",
      "Dependencies:\n",
      "  - R (>= 3.5) type: depends\n",
      "  - genefilter, type: depends\n",
      "  - GSVA, type: depends\n",
      "  - changepoint, type: depends\n",
      "  - data.table, type: imports\n",
      "  - plyr, type: imports\n",
      "  - ggplot2, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/SISPA.html\n",
      "\n",
      "Name: SNPediaR\n",
      "Description:\n",
      "SNPediaR provides some tools for downloading and parsing data from the SNPedia web site . The implemented functions allow users to import the wiki text available in SNPedia pages and to extract the most relevant information out of them. If some information in the downloaded pages is not automatically processed by the library functions, users can easily implement their own parsers to access it in an efficient way.\n",
      "Version: 1.24.0\n",
      "Publication date: None\n",
      "Mantainer: David Montaner <david.montaner@gmail.com>\n",
      "Authors:\n",
      "Author: David Montaner [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 3.0.0) type: depends\n",
      "  - RCurl, type: imports\n",
      "  - jsonlite, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/SNPediaR.html\n",
      "\n",
      "Name: sparseDOSSA\n",
      "Description:\n",
      "The package is to provide a model based Bayesian method to characterize and simulate microbiome data. sparseDOSSA's model captures the marginal distribution of each microbial feature as a truncated, zero-inflated log-normal distribution, with parameters distributed as a parent log-normal distribution. The model can be effectively fit to reference microbial datasets in order to parameterize their microbes and communities, or to simulate synthetic datasets of similar population structure. Most importantly, it allows users to include both known feature-feature and feature-metadata correlation structures and thus provides a gold standard to enable benchmarking of statistical methods for metagenomic data analysis.\n",
      "Version: 1.22.0\n",
      "Publication date: None\n",
      "Mantainer: Boyu Ren<bor158@mail.harvard.edu>, Emma Schwager <emma.schwager@gmail.com>, George Weingart<george.weingart@gmail.com>\n",
      "Authors:\n",
      "Author: Boyu Ren<bor158 at mail.harvard.edu>, Emma Schwager<emma.schwager at gmail.com>, Timothy Tickle<ttickle at hsph.harvard.edu>, Curtis Huttenhower <chuttenh at hsph.harvard.edu>\n",
      "Dependencies:\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - optparse, type: imports\n",
      "  - MASS, type: imports\n",
      "  - tmvtnorm (>= 1.4.10) type: imports\n",
      "  - MCMCpack, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/sparseDOSSA.html\n",
      "\n",
      "Name: SQLDataFrame\n",
      "Description:\n",
      "SQLDataFrame is developed to lazily represent and efficiently analyze SQL-based tables in _R_. SQLDataFrame supports common and familiar 'DataFrame' operations such as '[' subsetting, rbind, cbind, etc.. The internal implementation is based on the widely adopted dplyr grammar and SQL commands. In-memory datasets or plain text files (.txt, .csv, etc.) could also be easily converted into SQLDataFrames objects (which generates a new database on-disk).\n",
      "Version: 1.12.0\n",
      "Publication date: None\n",
      "Mantainer: Qian Liu <qian.liu@roswellpark.org>\n",
      "Authors:\n",
      "Author: Qian Liu [aut, cre] , Martin Morgan [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.6) type: depends\n",
      "  - dplyr (>= 0.8.0.1) type: depends\n",
      "  - dbplyr (>= 1.4.0) type: depends\n",
      "  - S4Vectors, type: depends\n",
      "  - DBI, type: imports\n",
      "  - lazyeval, type: imports\n",
      "  - methods, type: imports\n",
      "  - tools, type: imports\n",
      "  - stats, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - RSQLite, type: imports\n",
      "  - tibble, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/SQLDataFrame.html\n",
      "\n",
      "Name: srnadiff\n",
      "Description:\n",
      "srnadiff is a package that finds differently expressed regions from RNA-seq data at base-resolution level without relying on existing annotation. To do so, the package implements the identify-then-annotate methodology that builds on the idea of combining two pipelines approachs differential expressed regions detection and differential expression quantification. It reads BAM files as input, and outputs a list differentially regions, together with the adjusted p-values.\n",
      "Version: 1.18.0\n",
      "Publication date: None\n",
      "Mantainer: Zytnicki Matthias <matthias.zytnicki@inra.fr>\n",
      "Authors:\n",
      "Author: Zytnicki Matthias [aut, cre], Gonzalez Ignacio [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.6) type: depends\n",
      "  - Rcpp (>= 0.12.8) type: imports\n",
      "  - methods, type: imports\n",
      "  - devtools, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - DESeq2, type: imports\n",
      "  - edgeR, type: imports\n",
      "  - baySeq, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - GenomicFeatures, type: imports\n",
      "  - GenomicAlignments, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - Gviz, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - BiocStyle, type: imports\n",
      "  - BiocManager, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/srnadiff.html\n",
      "\n",
      "Name: sscu\n",
      "Description:\n",
      "The package calculates the indexes for selective stength in codon usage in bacteria species. (1) The package can calculate the strength of selected codon usage bias (sscu, also named as s_index) based on Paul Sharp's method. The method take into account of background mutation rate, and focus only on four pairs of codons with universal translational advantages in all bacterial species. Thus the sscu index is comparable among different species. (2) The package can detect the strength of translational accuracy selection by Akashi's test. The test tabulating all codons into four categories with the feature as conserved/variable amino acids and optimal/non-optimal codons. (3) Optimal codon lists (selected codons) can be calculated by either op_highly function (by using the highly expressed genes compared with all genes to identify optimal codons), or op_corre_CodonW/op_corre_NCprime function (by correlative method developed by Hershberg & Petrov). Users will have a list of optimal codons for further analysis, such as input to the Akashi's test. (4) The detailed codon usage information, such as RSCU value, number of optimal codons in the highly/all gene set, as well as the genomic gc3 value, can be calculate by the optimal_codon_statistics and genomic_gc3 function. (5) Furthermore, we added one test function low_frequency_op in the package. The function try to find the low frequency optimal codons, among all the optimal codons identified by the op_highly function.\n",
      "Version: 2.28.0\n",
      "Publication date: None\n",
      "Mantainer: Yu Sun <sunyu1357@gmail.com>\n",
      "Authors:\n",
      "Author: Yu Sun\n",
      "Dependencies:\n",
      "  - R (>= 3.3) type: depends\n",
      "  - Biostrings, type: imports\n",
      "  - seqinr (>= 3.1-3) type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/sscu.html\n",
      "\n",
      "Name: StarBioTrek\n",
      "Description:\n",
      "This tool StarBioTrek presents some methodologies to measure pathway activity and cross-talk among pathways integrating also the information of network data.\n",
      "Version: 1.24.0\n",
      "Publication date: None\n",
      "Mantainer: Claudia Cava <claudia.cava@ibfm.cnr.it>\n",
      "Authors:\n",
      "Author: Claudia Cava, Isabella Castiglioni\n",
      "Dependencies:\n",
      "  - R (>= 3.3) type: depends\n",
      "  - SpidermiR, type: imports\n",
      "  - graphite, type: imports\n",
      "  - AnnotationDbi, type: imports\n",
      "  - e1071, type: imports\n",
      "  - ROCR, type: imports\n",
      "  - MLmetrics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - igraph, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - ggplot2, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 3)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/StarBioTrek.html\n",
      "\n",
      "Name: supersigs\n",
      "Description:\n",
      "Generate SuperSigs (supervised mutational signatures) from single nucleotide variants in the cancer genome. Functions included in the package allow the user to learn supervised mutational signatures from their data and apply them to new data. The methodology is based on the one described in Afsari (2021, ELife).\n",
      "Version: 1.6.0\n",
      "Publication date: None\n",
      "Mantainer: Albert Kuo <albertkuo@jhu.edu>\n",
      "Authors:\n",
      "Author: Albert Kuo [aut, cre] , Yifan Zhang [aut], Bahman Afsari [aut], Cristian Tomasetti [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - assertthat, type: imports\n",
      "  - caret, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - rsample, type: imports\n",
      "  - methods, type: imports\n",
      "  - rlang, type: imports\n",
      "  - utils, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - stats, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/supersigs.html\n",
      "\n",
      "Name: TargetDecoy\n",
      "Description:\n",
      "A first step in the data analysis of Mass Spectrometry (MS) based proteomics data is to identify peptides and proteins. With this respect the huge number of experimental mass spectra typically have to be assigned to theoretical peptides derived from a sequence database. Search engines are used for this purpose. These tools compare each of the observed spectra to all candidate theoretical spectra derived from the sequence data base and calculate a score for each comparison. The observed spectrum is then assigned to the theoretical peptide with the best score, which is also referred to as the peptide to spectrum match (PSM). It is of course crucial for the downstream analysis to evaluate the quality of these matches. Therefore False Discovery Rate (FDR) control is used to return a reliable list PSMs. The FDR, however, requires a good characterisation of the score distribution of PSMs that are matched to the wrong peptide (bad target hits). In proteomics, the target decoy approach (TDA) is typically used for this purpose. The TDA method matches the spectra to a database of real (targets) and nonsense peptides (decoys). A popular approach to generate these decoys is to reverse the target database. Hence, all the PSMs that match to a decoy are known to be bad hits and the distribution of their scores are used to estimate the distribution of the bad scoring target PSMs. A crucial assumption of the TDA is that the decoy PSM hits have similar properties as bad target hits so that the decoy PSM scores are a good simulation of the target PSM scores. Users, however, typically do not evaluate these assumptions. To this end we developed TargetDecoy to generate diagnostic plots to evaluate the quality of the target decoy method.\n",
      "Version: 1.4.0\n",
      "Publication date: None\n",
      "Mantainer: Elke Debrie <elkedebrie@gmail.com>\n",
      "Authors:\n",
      "Author: Elke Debrie [aut, cre], Lieven Clement [aut] , Milan Malfait [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - ggplot2, type: imports\n",
      "  - ggpubr, type: imports\n",
      "  - methods, type: imports\n",
      "  - miniUI, type: imports\n",
      "  - mzID, type: imports\n",
      "  - mzR, type: imports\n",
      "  - shiny, type: imports\n",
      "  - stats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/TargetDecoy.html\n",
      "\n",
      "Name: TileDBArray\n",
      "Description:\n",
      "Implements a DelayedArray backend for reading and writing dense or sparse arrays in the TileDB format. The resulting TileDBArrays are compatible with all Bioconductor pipelines that can accept DelayedArray instances.\n",
      "Version: 1.8.0\n",
      "Publication date: None\n",
      "Mantainer: Aaron Lun <infinite.monkeys.with.keyboards@gmail.com>\n",
      "Authors:\n",
      "Author: Aaron Lun [aut, cre], Genentech, Inc. [cph]\n",
      "Dependencies:\n",
      "  - DelayedArray, type: depends\n",
      "  - methods, type: imports\n",
      "  - Rcpp, type: imports\n",
      "  - tiledb, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/TileDBArray.html\n",
      "\n",
      "Name: TurboNorm\n",
      "Description:\n",
      "A fast scatterplot smoother based on B-splines with second-order difference penalty. Functions for microarray normalization of single-colour data i.e. Affymetrix/Illumina and two-colour data supplied as marray MarrayRaw-objects or limma RGList-objects are available.\n",
      "Version: 1.46.0\n",
      "Publication date: None\n",
      "Mantainer: Maarten van Iterson <mviterson@gmail.com>\n",
      "Authors:\n",
      "Author: Maarten van Iterson and Chantal van Leeuwen\n",
      "Dependencies:\n",
      "  - R (>= 2.12.0) type: depends\n",
      "  - convert, type: depends\n",
      "  - limma, type: depends\n",
      "  - marray, type: depends\n",
      "  - stats, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - affy, type: imports\n",
      "  - lattice, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/TurboNorm.html\n",
      "\n",
      "Name: TypeInfo\n",
      "Description:\n",
      "A prototype for a mechanism for specifying the types of parameters and the return value for an R function. This is meta-information that can be used to generate stubs for servers and various interfaces to these functions. Additionally, the arguments in a call to a typed function can be validated using the type specifications. We allow types to be specified as either i) by class name using either inheritance - is(x, className), or strict instance of - class(x) %in% className, or ii) a dynamic test given as an R expression which is evaluated at run-time. More precise information and interesting tests can be done via ii), but it is harder to use this information as meta-data as it requires more effort to interpret it and it is of course run-time information. It is typically more meaningful.\n",
      "Version: 1.64.0\n",
      "Publication date: None\n",
      "Mantainer: Duncan Temple Lang <duncan@wald.ucdavis.edu>\n",
      "Authors:\n",
      "Author: Duncan Temple Lang Robert Gentleman (<rgentlem at fhcrc.org>)\n",
      "Dependencies:\n",
      "  - methods, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "BSD_2_clause\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/TypeInfo.html\n",
      "\n",
      "Name: Ularcirc\n",
      "Description:\n",
      "Ularcirc reads in STAR aligned splice junction files and provides visualisation and analysis tools for splicing analysis. Users can assess backsplice junctions and forward canonical junctions.\n",
      "Version: 1.16.0\n",
      "Publication date: None\n",
      "Mantainer: David Humphreys <d.humphreys@victorchang.edu.au>\n",
      "Authors:\n",
      "Author: David Humphreys [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 3.4.0) type: depends\n",
      "  - AnnotationHub, type: imports\n",
      "  - AnnotationDbi, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - BSgenome, type: imports\n",
      "  - data.table (>= 1.9.4) type: imports\n",
      "  - DT, type: imports\n",
      "  - GenomicFeatures, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - GenomeInfoDbData, type: imports\n",
      "  - GenomicAlignments, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - ggrepel, type: imports\n",
      "  - gsubfn, type: imports\n",
      "  - mirbase.db, type: imports\n",
      "  - moments, type: imports\n",
      "  - Organism.dplyr, type: imports\n",
      "  - plotgardener, type: imports\n",
      "  - R.utils, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - shiny, type: imports\n",
      "  - shinydashboard, type: imports\n",
      "  - shinyFiles, type: imports\n",
      "  - shinyjs, type: imports\n",
      "  - yaml, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/Ularcirc.html\n",
      "\n",
      "Name: unifiedWMWqPCR\n",
      "Description:\n",
      "This packages implements the unified Wilcoxon-Mann-Whitney Test for qPCR data. This modified test allows for testing differential expression in qPCR data.\n",
      "Version: 1.34.0\n",
      "Publication date: None\n",
      "Mantainer: Joris Meys <Joris.Meys@UGent.be>\n",
      "Authors:\n",
      "Author: Jan R. De Neve & Joris Meys\n",
      "Dependencies:\n",
      "  - methods, type: depends\n",
      "  - BiocGenerics, type: imports\n",
      "  - stats, type: imports\n",
      "  - graphics, type: imports\n",
      "  - HTqPCR, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>=2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/unifiedWMWqPCR.html\n",
      "\n",
      "Name: VCFArray\n",
      "Description:\n",
      "VCFArray extends the DelayedArray to represent VCF data entries as array-like objects with on-disk / remote VCF file as backend. Data entries from VCF files, including info fields, FORMAT fields, and the fixed columns (REF, ALT, QUAL, FILTER) could be converted into VCFArray instances with different dimensions.\n",
      "Version: 1.14.0\n",
      "Publication date: None\n",
      "Mantainer: Qian Liu <qliu7@buffalo.edu>\n",
      "Authors:\n",
      "Author: Qian Liu [aut, cre], Martin Morgan [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.6) type: depends\n",
      "  - methods, type: depends\n",
      "  - BiocGenerics, type: depends\n",
      "  - DelayedArray, type: depends\n",
      "  - tools, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - VariantAnnotation, type: imports\n",
      "  - GenomicFiles, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/VCFArray.html\n",
      "\n",
      "Name: vulcan\n",
      "Description:\n",
      "Vulcan (VirtUaL ChIP-Seq Analysis through Networks) is a package that interrogates gene regulatory networks to infer cofactors significantly enriched in a differential binding signature coming from ChIP-Seq data. In order to do so, our package combines strategies from different BioConductor packages: DESeq for data normalization, ChIPpeakAnno and DiffBind for annotation and definition of ChIP-Seq genomic peaks, csaw to define optimal peak width and viper for applying a regulatory network over a differential binding signature.\n",
      "Version: 1.20.0\n",
      "Publication date: None\n",
      "Mantainer: Federico M. Giorgi <federico.giorgi@gmail.com>\n",
      "Authors:\n",
      "Author: Federico M. Giorgi, Andrew N. Holding, Florian Markowetz\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - ChIPpeakAnno, type: depends\n",
      "  - TxDb.Hsapiens.UCSC.hg19.knownGene, type: depends\n",
      "  - zoo, type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - S4Vectors, type: depends\n",
      "  - viper, type: depends\n",
      "  - DiffBind, type: depends\n",
      "  - locfit, type: depends\n",
      "  - wordcloud, type: imports\n",
      "  - csaw, type: imports\n",
      "  - gplots, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - caTools, type: imports\n",
      "  - graphics, type: imports\n",
      "  - DESeq2, type: imports\n",
      "  - Biobase, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/vulcan.html\n",
      "\n",
      "Name: APL\n",
      "Description:\n",
      "APL is a package developed for computation of Association Plots (AP), a method for visualization and analysis of single cell transcriptomics data. The main focus of APL is the identification of genes characteristic for individual clusters of cells from input data. The package performs correspondence analysis (CA) and allows to identify cluster-specific genes using Association Plots. Additionally, APL computes the cluster-specificity scores for all genes which allows to rank the genes by their specificity for a selected cell cluster of interest.\n",
      "Version: 1.2.0\n",
      "Publication date: None\n",
      "Mantainer: Elzbieta Gralinska <gralinska@molgen.mpg.de>\n",
      "Authors:\n",
      "Author: Elzbieta Gralinska [cre, aut], Clemens Kohl [aut], Martin Vingron [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.2) type: depends\n",
      "  - reticulate, type: imports\n",
      "  - ggrepel, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - viridisLite, type: imports\n",
      "  - plotly, type: imports\n",
      "  - Seurat, type: imports\n",
      "  - SingleCellExperiment, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - topGO, type: imports\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - org.Hs.eg.db, type: imports\n",
      "  - org.Mm.eg.db, type: imports\n",
      "  - rlang, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 3)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/APL.html\n",
      "\n",
      "Name: banocc\n",
      "Description:\n",
      "BAnOCC is a package designed for compositional data, where each sample sums to one. It infers the approximate covariance of the unconstrained data using a Bayesian model coded with `rstan`. It provides as output the `stanfit` object as well as posterior median and credible interval estimates for each correlation element.\n",
      "Version: 1.22.0\n",
      "Publication date: None\n",
      "Mantainer: George Weingart <george.weingart@gmail.com>, Curtis Huttenhower <chuttenh@hsph.harvard.edu>\n",
      "Authors:\n",
      "Author: Emma Schwager [aut, cre], Curtis Huttenhower [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.5.1) type: depends\n",
      "  - rstan (>= 2.17.4) type: depends\n",
      "  - coda (>= 0.18.1) type: imports\n",
      "  - mvtnorm, type: imports\n",
      "  - stringr, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/banocc.html\n",
      "\n",
      "Name: biodbChebi\n",
      "Description:\n",
      "The biodbChebi library provides access to the ChEBI Database, using biodb package framework. It allows to retrieve entries by their accession number. Web services can be accessed for searching the database by name, mass or other fields.\n",
      "Version: 1.4.0\n",
      "Publication date: None\n",
      "Mantainer: Pierrick Roger <pierrick.roger@cea.fr>\n",
      "Authors:\n",
      "Author: Pierrick Roger [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - R6, type: imports\n",
      "  - biodb, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "AGPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/biodbChebi.html\n",
      "\n",
      "Name: biodbHmdb\n",
      "Description:\n",
      "The biodbHmdb library is an extension of the biodb framework package that provides access to the HMDB Metabolites database. It allows to download the whole HMDB Metabolites database locally, access entries and search for entries by name or description. A future version of this package will also include a search by mass and mass spectra annotation.\n",
      "Version: 1.4.0\n",
      "Publication date: None\n",
      "Mantainer: Pierrick Roger <pierrick.roger@cea.fr>\n",
      "Authors:\n",
      "Author: Pierrick Roger [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - R6, type: imports\n",
      "  - biodb, type: imports\n",
      "  - Rcpp, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "AGPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/biodbHmdb.html\n",
      "\n",
      "Name: biodbUniprot\n",
      "Description:\n",
      "The biodbUniprot library is an extension of the biodb framework package. It provides access to the UniProt database. It allows to retrieve entries by their accession number, and run web service queries for searching for entries.\n",
      "Version: 1.4.0\n",
      "Publication date: None\n",
      "Mantainer: Pierrick Roger <pierrick.roger@cea.fr>\n",
      "Authors:\n",
      "Author: Pierrick Roger [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.1.0) type: depends\n",
      "  - R6, type: imports\n",
      "  - biodb, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "AGPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/biodbUniprot.html\n",
      "\n",
      "Name: CCPROMISE\n",
      "Description:\n",
      "Perform Canonical correlation between two forms of high demensional genetic data, and associate the first compoent of each form of data with a specific biologically interesting pattern of associations with multiple endpoints. A probe level analysis is also implemented.\n",
      "Version: 1.24.0\n",
      "Publication date: None\n",
      "Mantainer: Xueyuan Cao <xueyuan.cao@stjude.org>\n",
      "Authors:\n",
      "Author: Xueyuan Cao <xueyuan.cao at stjude.org> and Stanley.pounds <stanley.pounds at stjude.org>\n",
      "Dependencies:\n",
      "  - R (>= 3.3.0) type: depends\n",
      "  - stats, type: depends\n",
      "  - methods, type: depends\n",
      "  - CCP, type: depends\n",
      "  - PROMISE, type: depends\n",
      "  - Biobase, type: depends\n",
      "  - GSEABase, type: depends\n",
      "  - utils, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/CCPROMISE.html\n",
      "\n",
      "Name: cfDNAPro\n",
      "Description:\n",
      "cfDNA fragments carry important features for building cancer sample classification ML models, such as fragment size, and fragment end motif etc. Analyzing and visualizing fragment size metrics, as well as other biological features in a curated, standardized, scalable, well-documented, and reproducible way might be time intensive. This package intends to resolve these problems and simplify the process. It offers two sets of functions for cfDNA feature characterization and visualization.\n",
      "Version: 1.4.0\n",
      "Publication date: None\n",
      "Mantainer: Haichao Wang <hw538@cam.ac.uk>\n",
      "Authors:\n",
      "Author: Haichao Wang [aut, cre], Hui Zhao [ctb], Elkie Chan [ctb], Christopher Smith [ctb], Tomer Kaplan [ctb], Florian Markowetz [ctb], Nitzan Rosenfeld [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 4.1.0) type: depends\n",
      "  - magrittr (>= 1.5.0) type: depends\n",
      "  - tibble, type: imports\n",
      "  - GenomicAlignments, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - plyranges, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - dplyr (>= 0.8.3) type: imports\n",
      "  - stringr (>= 1.4.0) type: imports\n",
      "  - quantmod (>= 0.4) type: imports\n",
      "  - ggplot2 (>= 3.2.1) type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - rlang (>= 0.4.0) type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/cfDNAPro.html\n",
      "\n",
      "Name: CNViz\n",
      "Description:\n",
      "CNViz takes probe, gene, and segment-level log2 copy number ratios and launches a Shiny app to visualize your sample's copy number profile. You can also integrate loss of heterozygosity (LOH) and single nucleotide variant (SNV) data.\n",
      "Version: 1.6.0\n",
      "Publication date: None\n",
      "Mantainer: Rebecca Greenblatt <rebecca.greenblatt@gmail.com>\n",
      "Authors:\n",
      "Author: Rebecca Greenblatt [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - shiny (>= 1.5.0) type: depends\n",
      "  - dplyr, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - plotly, type: imports\n",
      "  - karyoploteR, type: imports\n",
      "  - CopyNumberPlots, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - DT, type: imports\n",
      "  - scales, type: imports\n",
      "  - graphics, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/CNViz.html\n",
      "\n",
      "Name: cyanoFilter\n",
      "Description:\n",
      "An approach to filter out and/or identify phytoplankton cells from all particles measured via flow cytometry pigment and cell complexity information. It does this using a sequence of one-dimensional gates on pre-defined channels measuring certain pigmentation and complexity. The package is especially tuned for cyanobacteria, but will work fine for phytoplankton communities where there is at least one cell characteristic that differentiates every phytoplankton in the community.\n",
      "Version: 1.6.0\n",
      "Publication date: None\n",
      "Mantainer: Oluwafemi Olusoji <oluwafemi.olusoji@uhasselt.be>\n",
      "Authors:\n",
      "Author: Oluwafemi Olusoji [cre, aut], Aerts Marc [ctb], Delaender Frederik [ctb], Neyens Thomas [ctb], Spaak jurg [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.1.0) type: depends\n",
      "  - Biobase, type: imports\n",
      "  - flowCore, type: imports\n",
      "  - flowDensity, type: imports\n",
      "  - flowClust, type: imports\n",
      "  - cytometree, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - GGally, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - methods, type: imports\n",
      "  - mrfDepth, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/cyanoFilter.html\n",
      "\n",
      "Name: ddPCRclust\n",
      "Description:\n",
      "The ddPCRclust algorithm can automatically quantify the CPDs of non-orthogonal ddPCR reactions with up to four targets. In order to determine the correct droplet count for each target, it is crucial to both identify all clusters and label them correctly based on their position. For more information on what data can be analyzed and how a template needs to be formatted, please check the vignette.\n",
      "Version: 1.18.0\n",
      "Publication date: None\n",
      "Mantainer: Benedikt G. Brink <bbrink@cebitec.uni-bielefeld.de>\n",
      "Authors:\n",
      "Author: Benedikt G. Brink [aut, cre], Justin Meskas [ctb], Ryan R. Brinkman [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 3.5) type: depends\n",
      "  - plotrix, type: imports\n",
      "  - clue, type: imports\n",
      "  - parallel, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - openxlsx, type: imports\n",
      "  - R.utils, type: imports\n",
      "  - flowCore, type: imports\n",
      "  - flowDensity, type: imports\n",
      "  - SamSPECTRAL, type: imports\n",
      "  - flowPeaks, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ddPCRclust.html\n",
      "\n",
      "Name: diffUTR\n",
      "Description:\n",
      "The diffUTR package provides a uniform interface and plotting functions for limma/edgeR/DEXSeq -powered differential bin/exon usage. It includes in addition an improved version of the limma::diffSplice method. Most importantly, diffUTR further extends the application of these frameworks to differential UTR usage analysis using poly-A site databases.\n",
      "Version: 1.6.0\n",
      "Publication date: None\n",
      "Mantainer: Pierre-Luc Germain <pierre-luc.germain@hest.ethz.ch>\n",
      "Authors:\n",
      "Author: Pierre-Luc Germain [cre, aut] , Stefan Gerber [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - S4Vectors, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - limma, type: imports\n",
      "  - edgeR, type: imports\n",
      "  - DEXSeq, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - Rsubread, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - ComplexHeatmap, type: imports\n",
      "  - ggrepel, type: imports\n",
      "  - stringi, type: imports\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - ensembldb, type: imports\n",
      "  - viridisLite, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/diffUTR.html\n",
      "\n",
      "Name: DMCFB\n",
      "Description:\n",
      "DMCFB is a pipeline for identifying differentially methylated cytosines using a Bayesian functional regression model in bisulfite sequencing data. By using a functional regression data model, it tries to capture position-specific, group-specific and other covariates-specific methylation patterns as well as spatial correlation patterns and unknown underlying models of methylation data. It is robust and flexible with respect to the true underlying models and inclusion of any covariates, and the missing values are imputed using spatial correlation between positions and samples. A Bayesian approach is adopted for estimation and inference in the proposed method.\n",
      "Version: 1.12.0\n",
      "Publication date: None\n",
      "Mantainer: Farhad Shokoohi <shokoohi@icloud.com>\n",
      "Authors:\n",
      "Author: Farhad Shokoohi [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.0.0) type: depends\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - methods, type: depends\n",
      "  - S4Vectors, type: depends\n",
      "  - BiocParallel, type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - IRanges, type: depends\n",
      "  - utils, type: imports\n",
      "  - stats, type: imports\n",
      "  - speedglm, type: imports\n",
      "  - MASS, type: imports\n",
      "  - data.table, type: imports\n",
      "  - splines, type: imports\n",
      "  - arm, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - benchmarkme, type: imports\n",
      "  - tibble, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - fastDummies, type: imports\n",
      "  - graphics, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/DMCFB.html\n",
      "\n",
      "Name: epistack\n",
      "Description:\n",
      "The epistack package main objective is the visualizations of stacks of genomic tracks (such as, but not restricted to, ChIP-seq, ATAC-seq, DNA methyation or genomic conservation data) centered at genomic regions of interest.\n",
      "Version: 1.4.0\n",
      "Publication date: None\n",
      "Mantainer: DEVAILLY Guillaume <gdevailly@hotmail.com>\n",
      "Authors:\n",
      "Author: SACI Safia [aut], DEVAILLY Guillaume [cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - GenomicRanges, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - viridisLite, type: imports\n",
      "  - graphics, type: imports\n",
      "  - plotrix, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - stats, type: imports\n",
      "  - methods, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/epistack.html\n",
      "\n",
      "Name: ERSSA\n",
      "Description:\n",
      "The ERSSA package takes user supplied RNA-seq differential expression dataset and calculates the number of differentially expressed genes at varying biological replicate levels. This allows the user to determine, without relying on any a priori assumptions, whether sufficient differential detection has been acheived with their RNA-seq dataset.\n",
      "Version: 1.16.0\n",
      "Publication date: None\n",
      "Mantainer: Zixuan Shao <Zixuanshao.zach@gmail.com>\n",
      "Authors:\n",
      "Author: Zixuan Shao [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.0.0) type: depends\n",
      "  - edgeR, type: imports\n",
      "  - DESeq2, type: imports\n",
      "  - ggplot2 (>= 3.0.0) type: imports\n",
      "  - RColorBrewer (>= 1.1-2) type: imports\n",
      "  - plyr (>= 1.8.4) type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3 | file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ERSSA.html\n",
      "\n",
      "Name: famat\n",
      "Description:\n",
      "Famat is made to collect data about lists of genes and metabolites provided by user, and to visualize it through a Shiny app. Information collected is: - Pathways containing some of the user's genes and metabolites (obtained using a pathway enrichment analysis). - Direct interactions between user's elements inside pathways. - Information about elements (their identifiers and descriptions). - Go terms enrichment analysis performed on user's genes. The Shiny app is composed of: - information about genes, metabolites, and direct interactions between them inside pathways. - an heatmap showing which elements from the list are in pathways (pathways are structured in hierarchies). - hierarchies of enriched go terms using Molecular Function and Biological Process.\n",
      "Version: 1.8.0\n",
      "Publication date: None\n",
      "Mantainer: Mathieu Charles <mathieu.charles@inrae.fr>\n",
      "Authors:\n",
      "Author: Mathieu Charles [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - KEGGREST, type: imports\n",
      "  - mgcv, type: imports\n",
      "  - stats, type: imports\n",
      "  - BiasedUrn, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - gprofiler2, type: imports\n",
      "  - rWikiPathways, type: imports\n",
      "  - reactome.db, type: imports\n",
      "  - stringr, type: imports\n",
      "  - GO.db, type: imports\n",
      "  - ontologyIndex, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - shiny, type: imports\n",
      "  - shinydashboard, type: imports\n",
      "  - shinyBS, type: imports\n",
      "  - plotly, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - DT, type: imports\n",
      "  - clusterProfiler, type: imports\n",
      "  - org.Hs.eg.db, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/famat.html\n",
      "\n",
      "Name: geneClassifiers\n",
      "Description:\n",
      "This packages aims for easy accessible application of classifiers which have been published in literature using an ExpressionSet as input.\n",
      "Version: 1.22.0\n",
      "Publication date: None\n",
      "Mantainer: R Kuiper <r.kuiper.emc@gmail.com>\n",
      "Authors:\n",
      "Author: R Kuiper [cre, aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.6.0) type: depends\n",
      "  - utils, type: imports\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/geneClassifiers.html\n",
      "\n",
      "Name: GenomicOZone\n",
      "Description:\n",
      "The package clusters gene activity along chromosome into zones, detects differential zones as outstanding, and visualizes maps of outstanding zones across the genome. It enables characterization of effects on multiple genes within adaptive genomic neighborhoods, which could arise from genome reorganization, structural variation, or epigenome alteration. It guarantees cluster optimality, linear runtime to sample size, and reproducibility. One can apply it on genome-wide activity measurements such as copy number, transcriptomic, proteomic, and methylation data.\n",
      "Version: 1.12.0\n",
      "Publication date: None\n",
      "Mantainer: Hua Zhong<zh9118@gmail.com>, Mingzhou Song <joemsong@cs.nmsu.edu>\n",
      "Authors:\n",
      "Author: Hua Zhong, Mingzhou Song\n",
      "Dependencies:\n",
      "  - R (>= 4.0.0) type: depends\n",
      "  - Ckmeans.1d.dp (>= 4.3.0) type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - biomaRt, type: depends\n",
      "  - ggplot2, type: depends\n",
      "  - grDevices, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - plyr, type: imports\n",
      "  - gridExtra, type: imports\n",
      "  - lsr, type: imports\n",
      "  - parallel, type: imports\n",
      "  - ggbio, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - Rdpack, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL (>=3)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/GenomicOZone.html\n",
      "\n",
      "Name: geva\n",
      "Description:\n",
      "Statistic methods to evaluate variations of differential expression (DE) between multiple biological conditions. It takes into account the fold-changes and p-values from previous differential expression (DE) results that use large-scale data (*e.g.*, microarray and RNA-seq) and evaluates which genes would react in response to the distinct experiments. This evaluation involves an unique pipeline of statistical methods, including weighted summarization, quantile detection, cluster analysis, and ANOVA tests, in order to classify a subset of relevant genes whose DE is similar or dependent to certain biological factors.\n",
      "Version: 1.6.0\n",
      "Publication date: None\n",
      "Mantainer: Itamar JosÃ© GuimarÃ£es Nunes <nunesijg@gmail.com>\n",
      "Authors:\n",
      "Author: Itamar JosÃ© GuimarÃ£es Nunes [aut, cre] , Murilo Zanini David [ctb], Bruno CÃ©sar Feltes [ctb] , Marcio Dorn [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - dbscan, type: imports\n",
      "  - fastcluster, type: imports\n",
      "  - matrixStats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/geva.html\n",
      "\n",
      "Name: ggmanh\n",
      "Description:\n",
      "Manhattan plot and QQ Plot are commonly used to visualize the end result of Genome Wide Association Study. The \"ggmanh\" package aims to keep the generation of these plots simple while maintaining customizability. Main functions include manhattan_plot, qqunif, and thinPoints.\n",
      "Version: 1.2.0\n",
      "Publication date: None\n",
      "Mantainer: John Lee <john.lee@abbvie.com>\n",
      "Authors:\n",
      "Author: John Lee [aut, cre], Xiuwen Zheng [ctb, dtc]\n",
      "Dependencies:\n",
      "  - methods, type: depends\n",
      "  - ggplot2, type: depends\n",
      "  - gdsfmt, type: imports\n",
      "  - ggrepel, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - rlang, type: imports\n",
      "  - scales, type: imports\n",
      "  - SeqArray, type: imports\n",
      "  - stats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ggmanh.html\n",
      "\n",
      "Name: GIGSEA\n",
      "Description:\n",
      "We presented the Genotype-imputed Gene Set Enrichment Analysis (GIGSEA), a novel method that uses GWAS-and-eQTL-imputed trait-associated differential gene expression to interrogate gene set enrichment for the trait-associated SNPs. By incorporating eQTL from large gene expression studies, e.g. GTEx, GIGSEA appropriately addresses such challenges for SNP enrichment as gene size, gene boundary, SNP distal regulation, and multiple-marker regulation. The weighted linear regression model, taking as weights both imputation accuracy and model completeness, was used to perform the enrichment test, properly adjusting the bias due to redundancy in different gene sets. The permutation test, furthermore, is used to evaluate the significance of enrichment, whose efficiency can be largely elevated by expressing the computational intensive part in terms of large matrix operation. We have shown the appropriate type I error rates for GIGSEA (<5%), and the preliminary results also demonstrate its good performance to uncover the real signal.\n",
      "Version: 1.16.0\n",
      "Publication date: None\n",
      "Mantainer: Shijia Zhu <shijia.zhu@mssm.edu>\n",
      "Authors:\n",
      "Author: Shijia Zhu\n",
      "Dependencies:\n",
      "  - R (>= 3.5) type: depends\n",
      "  - Matrix, type: depends\n",
      "  - MASS, type: depends\n",
      "  - locfdr, type: depends\n",
      "  - stats, type: depends\n",
      "  - utils, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/GIGSEA.html\n",
      "\n",
      "Name: gmoviz\n",
      "Description:\n",
      "Genetically modified organisms (GMOs) and cell lines are widely used models in all kinds of biological research. As part of characterising these models, DNA sequencing technology and bioinformatics analyses are used systematically to study their genomes. Therefore, large volumes of data are generated and various algorithms are applied to analyse this data, which introduces a challenge on representing all findings in an informative and concise manner. `gmoviz` provides users with an easy way to visualise and facilitate the explanation of complex genomic editing events on a larger, biologically-relevant scale.\n",
      "Version: 1.10.0\n",
      "Publication date: None\n",
      "Mantainer: Kathleen Zeglinski <kathleen.zeglinski@csl.com.au>\n",
      "Authors:\n",
      "Author: Kathleen Zeglinski [cre, aut], Arthur Hsu [aut], Monther Alhamdoosh [aut] , Constantinos Koutsakis [aut]\n",
      "Dependencies:\n",
      "  - circlize, type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - graphics, type: depends\n",
      "  - R (>= 4.0) type: depends\n",
      "  - grid, type: imports\n",
      "  - gridBase, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - ComplexHeatmap, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - methods, type: imports\n",
      "  - GenomicAlignments, type: imports\n",
      "  - GenomicFeatures, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - pracma, type: imports\n",
      "  - colorspace, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/gmoviz.html\n",
      "\n",
      "Name: GMRP\n",
      "Description:\n",
      "Perform Mendelian randomization analysis of multiple SNPs to determine risk factors causing disease of study and to exclude confounding variabels and perform path analysis to construct path of risk factors to the disease.\n",
      "Version: 1.26.0\n",
      "Publication date: None\n",
      "Mantainer: Yuan-De Tan <tanyuande@gmail.com>\n",
      "Authors:\n",
      "Author: Yuan-De Tan\n",
      "Dependencies:\n",
      "  - R (>= 3.3.0) type: depends\n",
      "  - stats, type: depends\n",
      "  - utils, type: depends\n",
      "  - graphics, type: depends\n",
      "  - grDevices, type: depends\n",
      "  - diagram, type: depends\n",
      "  - plotrix, type: depends\n",
      "  - base, type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/GMRP.html\n",
      "\n",
      "Name: GRaNIE\n",
      "Description:\n",
      "Genetic variants associated with diseases often affect non-coding regions, thus likely having a regulatory role. To understand the effects of genetic variants in these regulatory regions, identifying genes that are modulated by specific regulatory elements (REs) is crucial. The effect of gene regulatory elements, such as enhancers, is often cell-type specific, likely because the combinations of transcription factors (TFs) that are regulating a given enhancer have celltype specific activity. This TF activity can be quantified with existing tools such as diffTF and captures differences in binding of a TF in open chromatin regions. Collectively, this forms a gene regulatory network (GRN) with cell-type and data-specific TF-RE and RE-gene links. Here, we reconstruct such a GRN using bulk RNAseq and open chromatin (e.g., using ATACseq or ChIPseq for open chromatin marks) and optionally TF activity data. Our network contains different types of links, connecting TFs to regulatory elements, the latter of which is connected to genes in the vicinity or within the same chromatin domain (TAD). We use a statistical framework to assign empirical FDRs and weights to all links using a permutation-based approach.\n",
      "Version: 1.2.6\n",
      "Publication date: None\n",
      "Mantainer: Christian Arnold <chrarnold@web.de>\n",
      "Authors:\n",
      "Author: Christian Arnold [cre, aut], Judith Zaugg [aut], Rim Moussa [aut], Armando Reyes-Palomares [ctb], Giovanni Palla [ctb], Maksim Kholmatov [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 4.2.0) type: depends\n",
      "  - futile.logger, type: imports\n",
      "  - checkmate, type: imports\n",
      "  - patchwork, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - data.table, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - ComplexHeatmap, type: imports\n",
      "  - DESeq2, type: imports\n",
      "  - circlize, type: imports\n",
      "  - progress, type: imports\n",
      "  - utils, type: imports\n",
      "  - methods, type: imports\n",
      "  - stringr, type: imports\n",
      "  - scales, type: imports\n",
      "  - igraph, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - rlang, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - forcats, type: imports\n",
      "  - gridExtra, type: imports\n",
      "  - limma, type: imports\n",
      "  - tidyselect, type: imports\n",
      "  - readr, type: imports\n",
      "  - grid, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - stats, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - tibble, type: imports\n",
      "  - viridis, type: imports\n",
      "  - colorspace, type: imports\n",
      "  - biomaRt, type: imports\n",
      "  - topGO, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/GRaNIE.html\n",
      "\n",
      "Name: GSgalgoR\n",
      "Description:\n",
      "A multi-objective optimization algorithm for disease sub-type discovery based on a non-dominated sorting genetic algorithm. The 'Galgo' framework combines the advantages of clustering algorithms for grouping heterogeneous 'omics' data and the searching properties of genetic algorithms for feature selection. The algorithm search for the optimal number of clusters determination considering the features that maximize the survival difference between sub-types while keeping cluster consistency high.\n",
      "Version: 1.8.0\n",
      "Publication date: None\n",
      "Mantainer: Carlos Catania <harpomaxx@gmail.com>\n",
      "Authors:\n",
      "Author: Martin Guerrero [aut], Carlos Catania [cre]\n",
      "Dependencies:\n",
      "  - cluster, type: imports\n",
      "  - doParallel, type: imports\n",
      "  - foreach, type: imports\n",
      "  - matchingR, type: imports\n",
      "  - nsga2R, type: imports\n",
      "  - survival, type: imports\n",
      "  - proxy, type: imports\n",
      "  - stats, type: imports\n",
      "  - methods, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/GSgalgoR.html\n",
      "\n",
      "Name: iASeq\n",
      "Description:\n",
      "It fits correlation motif model to multiple RNAseq or ChIPseq studies to improve detection of allele-specific events and describe correlation patterns across studies.\n",
      "Version: 1.42.0\n",
      "Publication date: None\n",
      "Mantainer: Yingying Wei <ywei@jhsph.edu>\n",
      "Authors:\n",
      "Author: Yingying Wei, Hongkai Ji\n",
      "Dependencies:\n",
      "  - R (>= 2.14.1) type: depends\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/iASeq.html\n",
      "\n",
      "Name: iChip\n",
      "Description:\n",
      "Hidden Ising models are implemented to identify enriched genomic regions in ChIP-chip data. They can be used to analyze the data from multiple platforms (e.g., Affymetrix, Agilent, and NimbleGen), and the data with single to multiple replicates.\n",
      "Version: 1.52.0\n",
      "Publication date: None\n",
      "Mantainer: Qianxing Mo <qianxing.mo@moffitt.org>\n",
      "Authors:\n",
      "Author: Qianxing Mo\n",
      "Dependencies:\n",
      "  - R (>= 2.10.0) type: depends\n",
      "  - limma, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/iChip.html\n",
      "\n",
      "Name: Informeasure\n",
      "Description:\n",
      "This package compiles most of the information measures currently available: mutual information, conditional mutual information, interaction information, partial information decomposition and part mutual information. All of these estimators can be used to quantify nonlinear dependence between variables in (biological regulatory) network inference. The first estimator is used to infer bivariate networks while the last four estimators are dedicated to analysis of trivariate networks.\n",
      "Version: 1.8.0\n",
      "Publication date: None\n",
      "Mantainer: Chu Pan <chu.pan@hnu.edu.cn>\n",
      "Authors:\n",
      "Author: Chu Pan [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - entropy, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/Informeasure.html\n",
      "\n",
      "Name: ipdDb\n",
      "Description:\n",
      "All alleles from the IPD IMGT/HLA and IPD KIR database for Homo sapiens. Reference: Robinson J, Maccari G, Marsh SGE, Walter L, Blokhuis J, Bimber B, Parham P, De Groot NG, Bontrop RE, Guethlein LA, and Hammond JA KIR Nomenclature in non-human species Immunogenetics (2018), in preparation.\n",
      "Version: 1.16.0\n",
      "Publication date: None\n",
      "Mantainer: Steffen Klasberg <klasberg@dkms-lab.de>\n",
      "Authors:\n",
      "Author: Steffen Klasberg\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - AnnotationDbi, type: depends\n",
      "  - AnnotationHub, type: depends\n",
      "  - Biostrings, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - RSQLite, type: imports\n",
      "  - DBI, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - stats, type: imports\n",
      "  - assertthat, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ipdDb.html\n",
      "\n",
      "Name: ivygapSE\n",
      "Description:\n",
      "Define a SummarizedExperiment and exploratory app for Ivy-GAP glioblastoma image, expression, and clinical data.\n",
      "Version: 1.20.0\n",
      "Publication date: None\n",
      "Mantainer: VJ Carey <stvjc@channing.harvard.edu>\n",
      "Authors:\n",
      "Author: Vince Carey\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - shiny, type: imports\n",
      "  - survival, type: imports\n",
      "  - survminer, type: imports\n",
      "  - hwriter, type: imports\n",
      "  - plotly, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - UpSetR, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ivygapSE.html\n",
      "\n",
      "Name: KBoost\n",
      "Description:\n",
      "Reconstructing gene regulatory networks and transcription factor activity is crucial to understand biological processes and holds potential for developing personalized treatment. Yet, it is still an open problem as state-of-art algorithm are often not able to handle large amounts of data. Furthermore, many of the present methods predict numerous false positives and are unable to integrate other sources of information such as previously known interactions. Here we introduce KBoost, an algorithm that uses kernel PCA regression, boosting and Bayesian model averaging for fast and accurate reconstruction of gene regulatory networks. KBoost can also use a prior network built on previously known transcription factor targets. We have benchmarked KBoost using three different datasets against other high performing algorithms. The results show that our method compares favourably to other methods across datasets.\n",
      "Version: 1.6.0\n",
      "Publication date: None\n",
      "Mantainer: Luis F. Iglesias-Martinez <luis.iglesiasmartinez@ucd.ie>\n",
      "Authors:\n",
      "Author: Luis F. Iglesias-Martinez [aut, cre] , Barbara de Kegel [aut], Walter Kolch [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - stats, type: depends\n",
      "  - utils, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2 | GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/KBoost.html\n",
      "\n",
      "Name: MantelCorr\n",
      "Description:\n",
      "Computes Mantel cluster correlations from a (p x n) numeric data matrix (e.g. microarray gene-expression data).\n",
      "Version: 1.68.0\n",
      "Publication date: None\n",
      "Mantainer: Brian Steinmeyer <steinmeb@ilya.wustl.edu>\n",
      "Authors:\n",
      "Author: Brian Steinmeyer and William Shannon\n",
      "Dependencies:\n",
      "  - R (>= 2.10) type: depends\n",
      "  - stats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/MantelCorr.html\n",
      "\n",
      "Name: MeasurementError.cor\n",
      "Description:\n",
      "Two-stage measurement error model for correlation estimation with smaller bias than the usual sample correlation\n",
      "Version: 1.70.0\n",
      "Publication date: None\n",
      "Mantainer: Beiying Ding <bding@amgen.com>\n",
      "Authors:\n",
      "Author: Beiying Ding\n",
      "Dependencies:\n",
      "None\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/MeasurementError.cor.html\n",
      "\n",
      "Name: microbiomeDASim\n",
      "Description:\n",
      "A toolkit for simulating differential microbiome data designed for longitudinal analyses. Several functional forms may be specified for the mean trend. Observations are drawn from a multivariate normal model. The objective of this package is to be able to simulate data in order to accurately compare different longitudinal methods for differential abundance.\n",
      "Version: 1.12.0\n",
      "Publication date: None\n",
      "Mantainer: Justin Williams <williazo@ucla.edu>\n",
      "Authors:\n",
      "Author: Justin Williams, Hector Corrada Bravo, Jennifer Tom, Joseph Nathaniel Paulson\n",
      "Dependencies:\n",
      "  - R (>= 3.6.0) type: depends\n",
      "  - graphics, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - MASS, type: imports\n",
      "  - tmvtnorm, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - mvtnorm, type: imports\n",
      "  - pbapply, type: imports\n",
      "  - stats, type: imports\n",
      "  - phyloseq, type: imports\n",
      "  - metagenomeSeq, type: imports\n",
      "  - Biobase, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/microbiomeDASim.html\n",
      "\n",
      "Name: multiscan\n",
      "Description:\n",
      "Estimates gene expressions from several laser scans of the same microarray\n",
      "Version: 1.58.0\n",
      "Publication date: None\n",
      "Mantainer: Mizanur Khondoker <mizanur.khondoker@ed.ac.uk>\n",
      "Authors:\n",
      "Author: Mizanur Khondoker <mizanur.khondoker at ed.ac.uk>, Chris Glasbey, Bruce Worton.\n",
      "Dependencies:\n",
      "  - R (>= 2.3.0) type: depends\n",
      "  - Biobase, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/multiscan.html\n",
      "\n",
      "Name: NBAMSeq\n",
      "Description:\n",
      "High-throughput sequencing experiments followed by differential expression analysis is a widely used approach to detect genomic biomarkers. A fundamental step in differential expression analysis is to model the association between gene counts and covariates of interest. NBAMSeq a flexible statistical model based on the generalized additive model and allows for information sharing across genes in variance estimation.\n",
      "Version: 1.14.0\n",
      "Publication date: None\n",
      "Mantainer: Xu Ren <xuren2120@gmail.com>\n",
      "Authors:\n",
      "Author: Xu Ren [aut, cre], Pei Fen Kuan [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.6) type: depends\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - S4Vectors, type: depends\n",
      "  - DESeq2, type: imports\n",
      "  - mgcv (>= 1.8-24) type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - genefilter, type: imports\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/NBAMSeq.html\n",
      "\n",
      "Name: nempi\n",
      "Description:\n",
      "Takes as input an incomplete perturbation profile and differential gene expression in log odds and infers unobserved perturbations and augments observed ones. The inference is done by iteratively inferring a network from the perturbations and inferring perturbations from the network. The network inference is done by Nested Effects Models.\n",
      "Version: 1.6.0\n",
      "Publication date: None\n",
      "Mantainer: Martin Pirkl <martinpirkl@yahoo.de>\n",
      "Authors:\n",
      "Author: Martin Pirkl [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - mnem, type: depends\n",
      "  - e1071, type: imports\n",
      "  - nnet, type: imports\n",
      "  - randomForest, type: imports\n",
      "  - naturalsort, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - epiNEM, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/nempi.html\n",
      "\n",
      "Name: nnSVG\n",
      "Description:\n",
      "Method for scalable identification of spatially variable genes (SVGs) in spatially-resolved transcriptomics data. The method is based on nearest-neighbor Gaussian processes and uses the BRISC algorithm for model fitting and parameter estimation. Allows identification and ranking of SVGs with flexible length scales across a tissue slide or within spatial domains defined by covariates. Scales linearly with the number of spatial locations and can be applied to datasets containing thousands or more spatial locations.\n",
      "Version: 1.2.0\n",
      "Publication date: None\n",
      "Mantainer: Lukas M. Weber <lukas.weber.edu@gmail.com>\n",
      "Authors:\n",
      "Author: Lukas M. Weber [aut, cre] , Stephanie C. Hicks [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.2) type: depends\n",
      "  - SpatialExperiment, type: imports\n",
      "  - SingleCellExperiment, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - BRISC, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - stats, type: imports\n",
      "  - methods, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/nnSVG.html\n",
      "\n",
      "Name: nuCpos\n",
      "Description:\n",
      "nuCpos, a derivative of NuPoP, is an R package for prediction of nucleosome positions. In nuCpos, a duration hidden Markov model is trained with a chemical map of nucleosomes either from budding yeast, fission yeast, or mouse embryonic stem cells. nuCpos outputs the Viterbi (most probable) path of nucleosome-linker states, predicted nucleosome occupancy scores and histone binding affinity (HBA) scores as NuPoP does. nuCpos can also calculate local and whole nucleosomal HBA scores for a given 147-bp sequence. Note: This package was designed to demonstrate the use of chemical maps in prediction. As the parental package NuPoP now provide chemical-map-based prediction, users are strongly encouraged to use it for dHMM-based prediction.\n",
      "Version: 1.16.3\n",
      "Publication date: None\n",
      "Mantainer: Hiroaki Kato <hkato@med.shimane-u.ac.jp>\n",
      "Authors:\n",
      "Author: Hiroaki Kato, Takeshi Urano\n",
      "Dependencies:\n",
      "  - R (>= 4.2.0) type: depends\n",
      "  - graphics, type: imports\n",
      "  - methods, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/nuCpos.html\n",
      "\n",
      "Name: oncomix\n",
      "Description:\n",
      "This package helps identify mRNAs that are overexpressed in subsets of tumors relative to normal tissue. Ideal inputs would be paired tumor-normal data from the same tissue from many patients (>15 pairs). This unsupervised approach relies on the observation that oncogenes are characteristically overexpressed in only a subset of tumors in the population, and may help identify oncogene candidates purely based on differences in mRNA expression between previously unknown subtypes.\n",
      "Version: 1.20.0\n",
      "Publication date: None\n",
      "Mantainer: Daniel Pique <daniel.pique@med.einstein.yu.edu>\n",
      "Authors:\n",
      "Author: Daniel Pique, John Greally, Jessica Mar\n",
      "Dependencies:\n",
      "  - R (>= 3.4.0) type: depends\n",
      "  - ggplot2, type: imports\n",
      "  - ggrepel, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - mclust, type: imports\n",
      "  - stats, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/oncomix.html\n",
      "\n",
      "Name: OTUbase\n",
      "Description:\n",
      "Provides a platform for Operational Taxonomic Unit based analysis\n",
      "Version: 1.48.0\n",
      "Publication date: None\n",
      "Mantainer: Daniel Beck <danlbek@gmail.com>\n",
      "Authors:\n",
      "Author: Daniel Beck, Matt Settles, and James A. Foster\n",
      "Dependencies:\n",
      "  - R (>= 2.9.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - S4Vectors, type: depends\n",
      "  - IRanges, type: depends\n",
      "  - ShortRead, type: depends\n",
      "  - Biobase, type: depends\n",
      "  - vegan, type: depends\n",
      "  - Biostrings, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/OTUbase.html\n",
      "\n",
      "Name: PANR\n",
      "Description:\n",
      "This package provides S4 classes and methods for inferring functional gene networks with edges encoding posterior beliefs of gene association types and nodes encoding perturbation effects.\n",
      "Version: 1.44.0\n",
      "Publication date: None\n",
      "Mantainer: Xin Wang <xin_wang@hms.harvard.edu>\n",
      "Authors:\n",
      "Author: Xin Wang <xin_wang at hms.harvard.edu>\n",
      "Dependencies:\n",
      "  - R (>= 2.14) type: depends\n",
      "  - igraph, type: depends\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - MASS, type: imports\n",
      "  - methods, type: imports\n",
      "  - pvclust, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - RedeR, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/PANR.html\n",
      "\n",
      "Name: parglms\n",
      "Description:\n",
      "This package provides support for parallelized estimation of GLMs/GEEs, catering for dispersed data.\n",
      "Version: 1.30.0\n",
      "Publication date: None\n",
      "Mantainer: VJ Carey <stvjc@channing.harvard.edu>\n",
      "Authors:\n",
      "Author: VJ Carey <stvjc at channing.harvard.edu>\n",
      "Dependencies:\n",
      "  - methods, type: depends\n",
      "  - BiocGenerics, type: imports\n",
      "  - BatchJobs, type: imports\n",
      "  - foreach, type: imports\n",
      "  - doParallel, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/parglms.html\n",
      "\n",
      "Name: POWSC\n",
      "Description:\n",
      "Determining the sample size for adequate power to detect statistical significance is a crucial step at the design stage for high-throughput experiments. Even though a number of methods and tools are available for sample size calculation for microarray and RNA-seq in the context of differential expression (DE), this topic in the field of single-cell RNA sequencing is understudied. Moreover, the unique data characteristics present in scRNA-seq such as sparsity and heterogeneity increase the challenge. We propose POWSC, a simulation-based method, to provide power evaluation and sample size recommendation for single-cell RNA sequencing DE analysis. POWSC consists of a data simulator that creates realistic expression data, and a power assessor that provides a comprehensive evaluation and visualization of the power and sample size relationship.\n",
      "Version: 1.6.0\n",
      "Publication date: None\n",
      "Mantainer: Kenong Su <kenong.su@emory.edu>\n",
      "Authors:\n",
      "Author: Kenong Su [aut, cre], Hao Wu [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - Biobase, type: depends\n",
      "  - SingleCellExperiment, type: depends\n",
      "  - MAST, type: depends\n",
      "  - pheatmap, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - limma, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/POWSC.html\n",
      "\n",
      "Name: pram\n",
      "Description:\n",
      "Publicly available RNA-seq data is routinely used for retrospective analysis to elucidate new biology. Novel transcript discovery enabled by large collections of RNA-seq datasets has emerged as one of such analysis. To increase the power of transcript discovery from large collections of RNA-seq datasets, we developed a new R package named Pooling RNA-seq and Assembling Models (PRAM), which builds transcript models in intergenic regions from pooled RNA-seq datasets. This package includes functions for defining intergenic regions, extracting and pooling related RNA-seq alignments, predicting, selected, and evaluating transcript models.\n",
      "Version: 1.14.0\n",
      "Publication date: None\n",
      "Mantainer: Peng Liu <pliu55.wisc@gmail.com>\n",
      "Authors:\n",
      "Author: Peng Liu [aut, cre], Colin N. Dewey [aut], SÃ¼ndÃ¼z KeleÅ [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.6) type: depends\n",
      "  - methods, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - tools, type: imports\n",
      "  - utils, type: imports\n",
      "  - data.table (>= 1.11.8) type: imports\n",
      "  - GenomicAlignments, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 3)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/pram.html\n",
      "\n",
      "Name: prebs\n",
      "Description:\n",
      "The prebs package aims at making RNA-sequencing (RNA-seq) data more comparable to microarray data. The comparability is achieved by summarizing sequencing-based expressions of probe regions using a modified version of RMA algorithm. The pipeline takes mapped reads in BAM format as an input and produces either gene expressions or original microarray probe set expressions as an output.\n",
      "Version: 1.38.0\n",
      "Publication date: None\n",
      "Mantainer: Karolis Uziela <karolis.uziela@scilifelab.se>\n",
      "Authors:\n",
      "Author: Karolis Uziela and Antti Honkela\n",
      "Dependencies:\n",
      "  - R (>= 2.14.0) type: depends\n",
      "  - GenomicAlignments, type: depends\n",
      "  - affy, type: depends\n",
      "  - RPA, type: depends\n",
      "  - parallel, type: imports\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/prebs.html\n",
      "\n",
      "Name: proBAMr\n",
      "Description:\n",
      "Mapping PSMs back to genome. The package builds SAM file from shotgun proteomics data The package also provides function to prepare annotation from GTF file.\n",
      "Version: 1.32.0\n",
      "Publication date: None\n",
      "Mantainer: Xiaojing Wang <xiaojing.wang@vanderbilt.edu>\n",
      "Authors:\n",
      "Author: Xiaojing Wang\n",
      "Dependencies:\n",
      "  - R (>= 3.0.1) type: depends\n",
      "  - IRanges, type: depends\n",
      "  - AnnotationDbi, type: depends\n",
      "  - GenomicRanges, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - GenomicFeatures, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/proBAMr.html\n",
      "\n",
      "Name: PROMISE\n",
      "Description:\n",
      "A general tool to identify genomic features with a specific biologically interesting pattern of associations with multiple endpoint variables as described in Pounds et. al. (2009) Bioinformatics 25: 2013-2019\n",
      "Version: 1.50.0\n",
      "Publication date: None\n",
      "Mantainer: Stan Pounds <stanley.pounds@stjude.org>, Xueyuan Cao <xueyuan.cao@stjude.org>\n",
      "Authors:\n",
      "Author: Stan Pounds <stanley.pounds at stjude.org>, Xueyuan Cao <xueyuan.cao at stjude.org>\n",
      "Dependencies:\n",
      "  - R (>= 3.1.0) type: depends\n",
      "  - Biobase, type: depends\n",
      "  - GSEABase, type: depends\n",
      "  - Biobase, type: imports\n",
      "  - GSEABase, type: imports\n",
      "  - stats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/PROMISE.html\n",
      "\n",
      "Name: PROPS\n",
      "Description:\n",
      "This package calculates probabilistic pathway scores using gene expression data. Gene expression values are aggregated into pathway-based scores using Bayesian network representations of biological pathways.\n",
      "Version: 1.20.0\n",
      "Publication date: None\n",
      "Mantainer: Lichy Han <lhan2@stanford.edu>\n",
      "Authors:\n",
      "Author: Lichy Han\n",
      "Dependencies:\n",
      "  - bnlearn, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - sva, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - Biobase, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/PROPS.html\n",
      "\n",
      "Name: Qtlizer\n",
      "Description:\n",
      "This R package provides access to the Qtlizer web server. Qtlizer annotates lists of common small variants (mainly SNPs) and genes in humans with associated changes in gene expression using the most comprehensive database of published quantitative trait loci (QTLs).\n",
      "Version: 1.12.0\n",
      "Publication date: None\n",
      "Mantainer: Matthias Munz <matthias.munz@gmx.de>\n",
      "Authors:\n",
      "Author: Matthias Munz [aut, cre] , Julia Remes [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.6.0) type: depends\n",
      "  - httr, type: imports\n",
      "  - curl, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - stringi, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/Qtlizer.html\n",
      "\n",
      "Name: Rbec\n",
      "Description:\n",
      "Rbec is a adapted version of DADA2 for analyzing amplicon sequencing data from synthetic communities (SynComs), where the reference sequences for each strain exists. Rbec can not only accurately profile the microbial compositions in SynComs, but also predict the contaminants in SynCom samples.\n",
      "Version: 1.6.0\n",
      "Publication date: None\n",
      "Mantainer: Pengfan Zhang <pzhang@mpipz.mpg.de>\n",
      "Authors:\n",
      "Author: Pengfan Zhang [aut, cre]\n",
      "Dependencies:\n",
      "  - Rcpp (>= 1.0.6) type: imports\n",
      "  - dada2, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - readr, type: imports\n",
      "  - doParallel, type: imports\n",
      "  - foreach, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/Rbec.html\n",
      "\n",
      "Name: RCSL\n",
      "Description:\n",
      "A novel clustering algorithm and toolkit RCSL (Rank Constrained Similarity Learning) to accurately identify various cell types using scRNA-seq data from a complex tissue. RCSL considers both lo-cal similarity and global similarity among the cells to discern the subtle differences among cells of the same type as well as larger differences among cells of different types. RCSL uses Spearmanâs rank correlations of a cellâs expression vector with those of other cells to measure its global similar-ity, and adaptively learns neighbour representation of a cell as its local similarity. The overall similar-ity of a cell to other cells is a linear combination of its global similarity and local similarity.\n",
      "Version: 1.6.0\n",
      "Publication date: None\n",
      "Mantainer: Qinglin Mei <meiqinglinkf@163.com>\n",
      "Authors:\n",
      "Author: Qinglin Mei [cre, aut], Guojun Li [fnd], Zhengchang Su [fnd]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - RcppAnnoy, type: imports\n",
      "  - igraph, type: imports\n",
      "  - NbClust, type: imports\n",
      "  - Rtsne, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - methods, type: imports\n",
      "  - pracma, type: imports\n",
      "  - umap, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/RCSL.html\n",
      "\n",
      "Name: RepViz\n",
      "Description:\n",
      "RepViz enables the view of a genomic region in a simple and efficient way. RepViz allows simultaneous viewing of both intra- and intergroup variation in sequencing counts of the studied conditions, as well as their comparison to the output features (e.g. identified peaks) from user selected data analysis methods.The RepViz tool is primarily designed for chromatin data such as ChIP-seq and ATAC-seq, but can also be used with other sequencing data such as RNA-seq, or combinations of different types of genomic data.\n",
      "Version: 1.14.0\n",
      "Publication date: None\n",
      "Mantainer: Thomas Faux, Asta Laiho <faux.thomas1@gmail.com> <asta.laiho@utu.fi>\n",
      "Authors:\n",
      "Author: Thomas Faux, Kalle RytkÃ¶nen, Asta Laiho, Laura L. Elo\n",
      "Dependencies:\n",
      "  - R (>= 3.5.1) type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - Rsamtools, type: depends\n",
      "  - IRanges, type: depends\n",
      "  - biomaRt, type: depends\n",
      "  - S4Vectors, type: depends\n",
      "  - graphics, type: depends\n",
      "  - grDevices, type: depends\n",
      "  - utils, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/RepViz.html\n",
      "\n",
      "Name: rfaRm\n",
      "Description:\n",
      "rfaRm provides a client interface to the Rfam database of RNA families. Data that can be retrieved include RNA families, secondary structure images, covariance models, sequences within each family, alignments leading to the identification of a family and secondary structures in the dot-bracket format.\n",
      "Version: 1.10.2\n",
      "Publication date: None\n",
      "Mantainer: Lara Selles Vidal <lara.selles@oist.jp>, Rafael Ayala <rafael.ayala@oist.jp>\n",
      "Authors:\n",
      "Author: Lara Selles Vidal, Rafael Ayala, Guy-Bart Stan, Rodrigo Ledesma-Amaro\n",
      "Dependencies:\n",
      "  - httr, type: imports\n",
      "  - stringi, type: imports\n",
      "  - rsvg, type: imports\n",
      "  - magick, type: imports\n",
      "  - data.table, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - utils, type: imports\n",
      "  - rvest, type: imports\n",
      "  - xml2, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/rfaRm.html\n",
      "\n",
      "Name: RJMCMCNucleosomes\n",
      "Description:\n",
      "This package does nucleosome positioning using informative Multinomial-Dirichlet prior in a t-mixture with reversible jump estimation of nucleosome positions for genome-wide profiling.\n",
      "Version: 1.22.0\n",
      "Publication date: None\n",
      "Mantainer: Astrid DeschÃªnes <adeschen@hotmail.com>\n",
      "Authors:\n",
      "Author: Pascal Belleau [aut], Rawane Samb [aut], Astrid DeschÃªnes [cre, aut], Khader Khadraoui [aut], Lajmi Lakhal-Chaieb [aut], Arnaud Droit [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - IRanges, type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - Rcpp (>= 0.12.5) type: imports\n",
      "  - consensusSeekeR, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - stats, type: imports\n",
      "  - graphics, type: imports\n",
      "  - methods, type: imports\n",
      "  - grDevices, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/RJMCMCNucleosomes.html\n",
      "\n",
      "Name: RLSeq\n",
      "Description:\n",
      "RLSeq is a toolkit for analyzing and evaluating R-loop mapping datasets. RLSeq serves two primary purposes: (1) to facilitate the evaluation of dataset quality, and (2) to enable R-loop analysis in the context of publicly-available data sets from RLBase. The package is intended to provide a simple pipeline, called with the `RLSeq()` function, which performs all main analyses. Individual functions are also accessible and provide custom analysis capabilities. Finally an HTML report is generated with `report()`.\n",
      "Version: 1.4.1\n",
      "Publication date: None\n",
      "Mantainer: Henry Miller <millerh1@uthscsa.edu>\n",
      "Authors:\n",
      "Author: Henry Miller [aut, cre, cph] , Daniel Montemayor [ctb] , Simon Levy [ctb] , Anna Vines [ctb] , Alexander Bishop [ths, cph]\n",
      "Dependencies:\n",
      "  - R (>= 4.2.0) type: depends\n",
      "  - dplyr, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - grid, type: imports\n",
      "  - regioneR, type: imports\n",
      "  - valr, type: imports\n",
      "  - caretEnsemble, type: imports\n",
      "  - GenomicFeatures, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - ComplexHeatmap, type: imports\n",
      "  - AnnotationHub, type: imports\n",
      "  - VennDiagram, type: imports\n",
      "  - callr, type: imports\n",
      "  - circlize, type: imports\n",
      "  - ggplotify, type: imports\n",
      "  - ggprism, type: imports\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - RLHub, type: imports\n",
      "  - aws.s3, type: imports\n",
      "  - pheatmap, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/RLSeq.html\n",
      "\n",
      "Name: Rmmquant\n",
      "Description:\n",
      "RNA-Seq is currently used routinely, and it provides accurate information on gene transcription. However, the method cannot accurately estimate duplicated genes expression. Several strategies have been previously used, but all of them provide biased results. With Rmmquant, if a read maps at different positions, the tool detects that the corresponding genes are duplicated; it merges the genes and creates a merged gene. The counts of ambiguous reads is then based on the input genes and the merged genes. Rmmquant is a drop-in replacement of the widely used tools findOverlaps and featureCounts that handles multi-mapping reads in an unabiased way.\n",
      "Version: 1.16.0\n",
      "Publication date: None\n",
      "Mantainer: Zytnicki Matthias <matthias.zytnicki@inra.fr>\n",
      "Authors:\n",
      "Author: Zytnicki Matthias [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 3.6) type: depends\n",
      "  - Rcpp (>= 0.12.8) type: imports\n",
      "  - methods, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - devtools, type: imports\n",
      "  - TBX20BamSubset, type: imports\n",
      "  - TxDb.Mmusculus.UCSC.mm9.knownGene, type: imports\n",
      "  - org.Mm.eg.db, type: imports\n",
      "  - DESeq2, type: imports\n",
      "  - BiocStyle, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/Rmmquant.html\n",
      "\n",
      "Name: RVS\n",
      "Description:\n",
      "Rare Variant Sharing (RVS) implements tests of association and linkage between rare genetic variant genotypes and a dichotomous phenotype, e.g. a disease status, in family samples. The tests are based on probabilities of rare variant sharing by relatives under the null hypothesis of absence of linkage and association between the rare variants and the phenotype and apply to single variants or multiple variants in a region (e.g. gene-based test).\n",
      "Version: 1.20.0\n",
      "Publication date: None\n",
      "Mantainer: Thomas Sherman <tomsherman159@gmail.com>\n",
      "Authors:\n",
      "Author: Alexandre Bureau, Ingo Ruczinski, Samuel Younkin, Thomas Sherman\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - GENLIB, type: imports\n",
      "  - gRain, type: imports\n",
      "  - snpStats, type: imports\n",
      "  - kinship2, type: imports\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/RVS.html\n",
      "\n",
      "Name: scMAGeCK\n",
      "Description:\n",
      "scMAGeCK is a computational model to identify genes associated with multiple expression phenotypes from CRISPR screening coupled with single-cell RNA sequencing data (CROP-seq)\n",
      "Version: 1.9.1\n",
      "Publication date: None\n",
      "Mantainer: Xiaolong Cheng <xiaolongcheng1120@gmail.com>\n",
      "Authors:\n",
      "Author: Wei Li, Xiaolong Cheng, Lin Yang\n",
      "Dependencies:\n",
      "  - Seurat, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "BSD_2_clause\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/scMAGeCK.html\n",
      "\n",
      "Name: SigsPack\n",
      "Description:\n",
      "Single sample estimation of exposure to mutational signatures. Exposures to known mutational signatures are estimated for single samples, based on quadratic programming algorithms. Bootstrapping the input mutational catalogues provides estimations on the stability of these exposures. The effect of the sequence composition of mutational context can be taken into account by normalising the catalogues.\n",
      "Version: 1.12.0\n",
      "Publication date: None\n",
      "Mantainer: Franziska Schumann <franziska.schumann@student.hpi.de>\n",
      "Authors:\n",
      "Author: Franziska Schumann <franziska.schumann at student.hpi.de>\n",
      "Dependencies:\n",
      "  - R (>= 3.6) type: depends\n",
      "  - quadprog (>= 1.5-5) type: imports\n",
      "  - methods, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - BSgenome, type: imports\n",
      "  - VariantAnnotation, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/SigsPack.html\n",
      "\n",
      "Name: SIMAT\n",
      "Description:\n",
      "This package provides a pipeline for analysis of GC-MS data acquired in selected ion monitoring (SIM) mode. The tool also provides a guidance in choosing appropriate fragments for the targets of interest by using an optimization algorithm. This is done by considering overlapping peaks from a provided library by the user.\n",
      "Version: 1.30.0\n",
      "Publication date: None\n",
      "Mantainer: M. R. Nezami Ranjbar <nranjbar@vt.edu>\n",
      "Authors:\n",
      "Author: M. R. Nezami Ranjbar <nranjbar at vt.edu>\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - Rcpp (>= 0.11.3) type: depends\n",
      "  - mzR, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - grid, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/SIMAT.html\n",
      "\n",
      "Name: SMAP\n",
      "Description:\n",
      "Functions and classes for DNA copy number profiling of array-CGH data\n",
      "Version: 1.62.0\n",
      "Publication date: None\n",
      "Mantainer: Robin Andersson <robin.andersson@lcb.uu.se>\n",
      "Authors:\n",
      "Author: Robin Andersson <robin.andersson at lcb.uu.se>\n",
      "Dependencies:\n",
      "  - R (>= 2.10) type: depends\n",
      "  - methods, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/SMAP.html\n",
      "\n",
      "Name: SNPhood\n",
      "Description:\n",
      "To date, thousands of single nucleotide polymorphisms (SNPs) have been found to be associated with complex traits and diseases. However, the vast majority of these disease-associated SNPs lie in the non-coding part of the genome, and are likely to affect regulatory elements, such as enhancers and promoters, rather than function of a protein. Thus, to understand the molecular mechanisms underlying genetic traits and diseases, it becomes increasingly important to study the effect of a SNP on nearby molecular traits such as chromatin environment or transcription factor (TF) binding. Towards this aim, we developed SNPhood, a user-friendly *Bioconductor* R package to investigate and visualize the local neighborhood of a set of SNPs of interest for NGS data such as chromatin marks or transcription factor binding sites from ChIP-Seq or RNA- Seq experiments. SNPhood comprises a set of easy-to-use functions to extract, normalize and summarize reads for a genomic region, perform various data quality checks, normalize read counts using additional input files, and to cluster and visualize the regions according to the binding pattern. The regions around each SNP can be binned in a user-defined fashion to allow for analysis of very broad patterns as well as a detailed investigation of specific binding shapes. Furthermore, SNPhood supports the integration with genotype information to investigate and visualize genotype-specific binding patterns. Finally, SNPhood can be employed for determining, investigating, and visualizing allele-specific binding patterns around the SNPs of interest.\n",
      "Version: 1.28.0\n",
      "Publication date: None\n",
      "Mantainer: Christian Arnold <christian.arnold@embl.de>\n",
      "Authors:\n",
      "Author: Christian Arnold [aut, cre], Pooja Bhat [aut], Judith Zaugg [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - Rsamtools, type: depends\n",
      "  - data.table, type: depends\n",
      "  - checkmate, type: depends\n",
      "  - DESeq2, type: imports\n",
      "  - cluster, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - lattice, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - VariantAnnotation, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - methods, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - gridExtra, type: imports\n",
      "  - stats, type: imports\n",
      "  - grid, type: imports\n",
      "  - utils, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - scales, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL (>= 3)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/SNPhood.html\n",
      "\n",
      "Name: target\n",
      "Description:\n",
      "Implement the BETA algorithm for infering direct target genes from DNA-binding and perturbation expression data Wang et al. (2013) . Extend the algorithm to predict the combined function of two DNA-binding elements from comprable binding and expression data.\n",
      "Version: 1.12.0\n",
      "Publication date: None\n",
      "Mantainer: Mahmoud Ahmed <mahmoud.s.fahmy@students.kasralainy.edu.eg>\n",
      "Authors:\n",
      "Author: Mahmoud Ahmed [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 3.6) type: depends\n",
      "  - BiocGenerics, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - graphics, type: imports\n",
      "  - shiny, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/target.html\n",
      "\n",
      "Name: TimeSeriesExperiment\n",
      "Description:\n",
      "Bioconductor version: Release (3.16)\n",
      "Version: 1.16.0\n",
      "Publication date: None\n",
      "Mantainer: Author: Lan Huong Nguyen [cre, aut]\n",
      "Authors:\n",
      "TimeSeriesExperiment is a visualization and analysis toolbox for short time course data. The package includes dimensionality reduction, clustering, two-sample differential expression testing and gene ranking techniques. Additionally, it also provides methods for retrieving enriched pathways.\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - S4Vectors, type: depends\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - dynamicTreeCut, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - edgeR, type: imports\n",
      "  - DESeq2, type: imports\n",
      "  - ggplot2 (>= 3.0.0) type: imports\n",
      "  - graphics, type: imports\n",
      "  - Hmisc, type: imports\n",
      "  - limma, type: imports\n",
      "  - methods, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - proxy, type: imports\n",
      "  - stats, type: imports\n",
      "  - tibble, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - vegan, type: imports\n",
      "  - viridis, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/TimeSeriesExperiment.html\n",
      "\n",
      "Name: transformGamPoi\n",
      "Description:\n",
      "Variance-stabilizing transformations help with the analysis of heteroskedastic data (i.e., data where the variance is not constant, like count data). This package provide two types of variance stabilizing transformations: (1) methods based on the delta method (e.g., 'acosh', 'log(x+1)'), (2) model residual based (Pearson and randomized quantile residuals).\n",
      "Version: 1.4.0\n",
      "Publication date: None\n",
      "Mantainer: Constantin Ahlmann-Eltze <artjom31415@googlemail.com>\n",
      "Authors:\n",
      "Author: Constantin Ahlmann-Eltze [aut, cre]\n",
      "Dependencies:\n",
      "  - glmGamPoi, type: imports\n",
      "  - DelayedArray, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - MatrixGenerics, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - HDF5Array, type: imports\n",
      "  - methods, type: imports\n",
      "  - utils, type: imports\n",
      "  - Rcpp, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/transformGamPoi.html\n",
      "\n",
      "Name: traseR\n",
      "Description:\n",
      "traseR performs GWAS trait-associated SNP enrichment analyses in genomic intervals using different hypothesis testing approaches, also provides various functionalities to explore and visualize the results.\n",
      "Version: 1.28.0\n",
      "Publication date: None\n",
      "Mantainer: li chen<li.chen@emory.edu>\n",
      "Authors:\n",
      "Author: Li Chen, Zhaohui S.Qin\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - IRanges, type: depends\n",
      "  - BSgenome.Hsapiens.UCSC.hg19, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/traseR.html\n",
      "\n",
      "Name: VegaMC\n",
      "Description:\n",
      "This package enables the detection of driver chromosomal imbalances including loss of heterozygosity (LOH) from array comparative genomic hybridization (aCGH) data. VegaMC performs a joint segmentation of a dataset and uses a statistical framework to distinguish between driver and passenger mutation. VegaMC has been implemented so that it can be immediately integrated with the output produced by PennCNV tool. In addition, VegaMC produces in output two web pages that allows a rapid navigation between both the detected regions and the altered genes. In the web page that summarizes the altered genes, the link to the respective Ensembl gene web page is reported.\n",
      "Version: 3.36.0\n",
      "Publication date: None\n",
      "Mantainer: Sandro Morganella <morganellaalx@gmail.com>\n",
      "Authors:\n",
      "Author: S. Morganella and M. Ceccarelli\n",
      "Dependencies:\n",
      "  - R (>= 2.10.0) type: depends\n",
      "  - biomaRt, type: depends\n",
      "  - Biobase, type: depends\n",
      "  - methods, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/VegaMC.html\n",
      "\n",
      "Name: vtpnet\n",
      "Description:\n",
      "variant-transcription factor-phenotype networks, inspired by Maurano et al., Science (2012), PMID 22955828\n",
      "Version: 0.38.0\n",
      "Publication date: None\n",
      "Mantainer: VJ Carey <stvjc@channing.harvard.edu>\n",
      "Authors:\n",
      "Author: VJ Carey <stvjc at channing.harvard.edu>\n",
      "Dependencies:\n",
      "  - R (>= 3.0.0) type: depends\n",
      "  - graph, type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - gwascat, type: depends\n",
      "  - doParallel, type: depends\n",
      "  - foreach, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/vtpnet.html\n",
      "\n",
      "Name: wppi\n",
      "Description:\n",
      "Protein-protein interaction data is essential for omics data analysis and modeling. Database knowledge is general, not specific for cell type, physiological condition or any other context determining which connections are functional and contribute to the signaling. Functional annotations such as Gene Ontology and Human Phenotype Ontology might help to evaluate the relevance of interactions. This package predicts functional relevance of protein-protein interactions based on functional annotations such as Human Protein Ontology and Gene Ontology, and prioritizes genes based on network topology, functional scores and a path search algorithm.\n",
      "Version: 1.6.0\n",
      "Publication date: None\n",
      "Mantainer: Ana Galhoz <ana.galhoz@helmholtz-muenchen.de>\n",
      "Authors:\n",
      "Author: Ana Galhoz [cre, aut] , Denes Turei [aut] , Michael P. Menden [aut] , Albert Krewinkel [ctb, cph] (pagebreak Lua filter)\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - dplyr, type: imports\n",
      "  - igraph, type: imports\n",
      "  - logger, type: imports\n",
      "  - methods, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - OmnipathR, type: imports\n",
      "  - progress, type: imports\n",
      "  - purrr, type: imports\n",
      "  - rlang, type: imports\n",
      "  - RCurl, type: imports\n",
      "  - stats, type: imports\n",
      "  - tibble, type: imports\n",
      "  - tidyr, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/wppi.html\n",
      "\n",
      "Name: BayesKnockdown\n",
      "Description:\n",
      "A simple, fast Bayesian method for computing posterior probabilities for relationships between a single predictor variable and multiple potential outcome variables, incorporating prior probabilities of relationships. In the context of knockdown experiments, the predictor variable is the knocked-down gene, while the other genes are potential targets. Can also be used for differential expression/2-class data.\n",
      "Version: 1.24.0\n",
      "Publication date: None\n",
      "Mantainer: William Chad Young <wmchad@uw.edu>\n",
      "Authors:\n",
      "Author: William Chad Young\n",
      "Dependencies:\n",
      "  - R (>= 3.3) type: depends\n",
      "  - stats, type: imports\n",
      "  - Biobase, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/BayesKnockdown.html\n",
      "\n",
      "Name: CellBarcode\n",
      "Description:\n",
      "This package performs Cellular DNA Barcode (genetic lineage tracing) analysis. The package can handle all kinds of DNA barcodes, as long as the barcode within a single sequencing read and has a pattern which can be matched by a regular expression. This package can handle barcode with flexible length, with or without UMI (unique molecular identifier). This tool also can be used for pre-processing of some amplicon sequencing such as CRISPR gRNA screening, immune repertoire sequencing and meta genome data.\n",
      "Version: 1.4.0\n",
      "Publication date: None\n",
      "Mantainer: Wenjie Sun <sunwjie@gmail.com>\n",
      "Authors:\n",
      "Author: Wenjie Sun [cre], Anne-Marie Lyne [aut], Leila Perie [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.1.0) type: depends\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - Rcpp (>= 1.0.5) type: imports\n",
      "  - data.table (>= 1.12.6) type: imports\n",
      "  - plyr, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - stringr, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - ShortRead, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - egg, type: imports\n",
      "  - Ckmeans.1d.dp, type: imports\n",
      "  - utils, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - seqinr, type: imports\n",
      "  - zlibbioc, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/CellBarcode.html\n",
      "\n",
      "Name: CexoR\n",
      "Description:\n",
      "Strand specific peak-pair calling in ChIP-exo replicates. The cumulative Skellam distribution function is used to detect significant normalised count differences of opposed sign at each DNA strand (peak-pairs). Then, irreproducible discovery rate for overlapping peak-pairs across biological replicates is computed.\n",
      "Version: 1.36.0\n",
      "Publication date: None\n",
      "Mantainer: Pedro Madrigal <pmadrigal@ebi.ac.uk>\n",
      "Authors:\n",
      "Author: Pedro Madrigal [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.2.0) type: depends\n",
      "  - S4Vectors, type: depends\n",
      "  - IRanges, type: depends\n",
      "  - Rsamtools, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - idr, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - genomation, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0 | GPL-2 + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/CexoR.html\n",
      "\n",
      "Name: CIMICE\n",
      "Description:\n",
      "CIMICE is a tool in the field of tumor phylogenetics and its goal is to build a Markov Chain (called Cancer Progression Markov Chain, CPMC) in order to model tumor subtypes evolution. The input of CIMICE is a Mutational Matrix, so a boolean matrix representing altered genes in a collection of samples. These samples are assumed to be obtained with single-cell DNA analysis techniques and the tool is specifically written to use the peculiarities of this data for the CMPC construction.\n",
      "Version: 1.6.0\n",
      "Publication date: None\n",
      "Mantainer: NicolÃ² Rossi <olocin.issor@gmail.com>\n",
      "Authors:\n",
      "Author: NicolÃ² Rossi [aut, cre] (Lab. of Computational Biology and Bioinformatics, Department of Mathematics, Computer Science and Physics, University of Udine, )\n",
      "Dependencies:\n",
      "  - dplyr, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - glue, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - igraph, type: imports\n",
      "  - networkD3, type: imports\n",
      "  - visNetwork, type: imports\n",
      "  - ggcorrplot, type: imports\n",
      "  - purrr, type: imports\n",
      "  - ggraph, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - maftools, type: imports\n",
      "  - assertthat, type: imports\n",
      "  - tidygraph, type: imports\n",
      "  - expm, type: imports\n",
      "  - Matrix, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/CIMICE.html\n",
      "\n",
      "Name: Clomial\n",
      "Description:\n",
      "Clomial fits binomial distributions to counts obtained from Next Gen Sequencing data of multiple samples of the same tumor. The trained parameters can be interpreted to infer the clonal structure of the tumor.\n",
      "Version: 1.34.0\n",
      "Publication date: None\n",
      "Mantainer: Habil Zare <zare@u.washington.edu>\n",
      "Authors:\n",
      "Author: Habil Zare and Alex Hu\n",
      "Dependencies:\n",
      "  - R (>= 2.10) type: depends\n",
      "  - matrixStats, type: depends\n",
      "  - methods, type: imports\n",
      "  - permute, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/Clomial.html\n",
      "\n",
      "Name: diffGeneAnalysis\n",
      "Description:\n",
      "Analyze microarray data\n",
      "Version: 1.80.0\n",
      "Publication date: None\n",
      "Mantainer: Choudary Jagarlamudi <choudary.jagar@swosu.edu>\n",
      "Authors:\n",
      "Author: Choudary Jagarlamudi\n",
      "Dependencies:\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - minpack.lm (>= 1.0-4) type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/diffGeneAnalysis.html\n",
      "\n",
      "Name: DNABarcodeCompatibility\n",
      "Description:\n",
      "The package allows one to obtain optimised combinations of DNA barcodes to be used for multiplex sequencing. In each barcode combination, barcodes are pooled with respect to Illumina chemistry constraints. Combinations can be filtered to keep those that are robust against substitution and insertion/deletion errors thereby facilitating the demultiplexing step. In addition, the package provides an optimiser function to further favor the selection of barcode combinations with least heterogeneity in barcode usage.\n",
      "Version: 1.14.0\n",
      "Publication date: None\n",
      "Mantainer: CÃ©line TrÃ©beau <ctrebeau@pasteur.fr>\n",
      "Authors:\n",
      "Author: CÃ©line TrÃ©beau [cre] , Jacques Boutet de Monvel [aut] , Fabienne Wong Jun Tai [ctb], RaphaÃ«l Etournay [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.6.0) type: depends\n",
      "  - dplyr, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - numbers, type: imports\n",
      "  - purrr, type: imports\n",
      "  - stringr, type: imports\n",
      "  - DNABarcodes, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - methods, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/DNABarcodeCompatibility.html\n",
      "\n",
      "Name: Dune\n",
      "Description:\n",
      "Given a set of clustering labels, Dune merges pairs of clusters to increase mean ARI between labels, improving replicability.\n",
      "Version: 1.10.0\n",
      "Publication date: None\n",
      "Mantainer: Hector Roux de Bezieux <hector.rouxdebezieux@berkeley.edu>\n",
      "Authors:\n",
      "Author: Hector Roux de Bezieux [aut, cre] , Kelly Street [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.6) type: depends\n",
      "  - BiocParallel, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - utils, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - gganimate, type: imports\n",
      "  - purrr, type: imports\n",
      "  - aricode, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/Dune.html\n",
      "\n",
      "Name: EpiCompare\n",
      "Description:\n",
      "EpiCompare is used to compare and analyse epigenetic datasets for quality control and benchmarking purposes. The package outputs an HTML report consisting of three sections: (1. General metrics) Metrics on peaks (percentage of blacklisted and non-standard peaks, and peak widths) and fragments (duplication rate) of samples, (2. Peak overlap) Percentage and statistical significance of overlapping and non-overlapping peaks. Also includes upset plot and (3. Functional annotation) functional annotation (ChromHMM, ChIPseeker and enrichment analysis) of peaks. Also includes peak enrichment around TSS.\n",
      "Version: 1.2.0\n",
      "Publication date: None\n",
      "Mantainer: Sera Choi <serachoi1230@gmail.com>\n",
      "Authors:\n",
      "Author: Sera Choi [aut, cre] , Brian Schilder [aut] , Leyla Abbasova [aut], Alan Murphy [aut] , Nathan Skene [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.1.0) type: depends\n",
      "  - AnnotationHub, type: imports\n",
      "  - BRGenomics, type: imports\n",
      "  - ChIPseeker, type: imports\n",
      "  - data.table, type: imports\n",
      "  - genomation, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - htmltools, type: imports\n",
      "  - methods, type: imports\n",
      "  - plotly, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - rmarkdown, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - stats, type: imports\n",
      "  - stringr, type: imports\n",
      "  - utils, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/EpiCompare.html\n",
      "\n",
      "Name: epihet\n",
      "Description:\n",
      "epihet is an R-package that calculates the epigenetic heterogeneity between cancer cells and/or normal cells. The functions establish a pipeline that take in bisulfite sequencing data from multiple samples and use the data to track similarities and differences in epipolymorphism,proportion of discordantly methylated sequencing reads (PDR),and Shannon entropy values at epialleles that are shared between the samples.epihet can be used to perform analysis on the data by creating pheatmaps, box plots, PCA plots, and t-SNE plots. MA plots can also be created by calculating the differential heterogeneity of the samples. And we construct co-epihet network and perform network analysis.\n",
      "Version: 1.13.0\n",
      "Publication date: None\n",
      "Mantainer: Xiaowen Chen <Xiaowen.Chen@jax.org>\n",
      "Authors:\n",
      "Author: Xiaowen Chen [aut, cre], Haitham Ashoor [aut], Ryan Musich [aut], Mingsheng Zhang [aut], Jiahui Wang [aut], Sheng Li [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.6) type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - IRanges, type: depends\n",
      "  - S4Vectors, type: depends\n",
      "  - ggplot2, type: depends\n",
      "  - foreach, type: depends\n",
      "  - Rtsne, type: depends\n",
      "  - igraph, type: depends\n",
      "  - data.table, type: imports\n",
      "  - doParallel, type: imports\n",
      "  - EntropyExplorer, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - pheatmap, type: imports\n",
      "  - utils, type: imports\n",
      "  - qvalue, type: imports\n",
      "  - WGCNA, type: imports\n",
      "  - ReactomePA, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/epihet.html\n",
      "\n",
      "Name: ExCluster\n",
      "Description:\n",
      "ExCluster flattens Ensembl and GENCODE GTF files into GFF files, which are used to count reads per non-overlapping exon bin from BAM files. This read counting is done using the function featureCounts from the package Rsubread. Library sizes are normalized across all biological replicates, and ExCluster then compares two different conditions to detect signifcantly differentially spliced genes. This process requires at least two independent biological repliates per condition, and ExCluster accepts only exactly two conditions at a time. ExCluster ultimately produces false discovery rates (FDRs) per gene, which are used to detect significance. Exon log2 fold change (log2FC) means and variances may be plotted for each significantly differentially spliced gene, which helps scientists develop hypothesis and target differential splicing events for RT-qPCR validation in the wet lab.\n",
      "Version: 1.16.0\n",
      "Publication date: None\n",
      "Mantainer: R. Matthew Tanner <rtann038@uottawa.ca>\n",
      "Authors:\n",
      "Author: R. Matthew Tanner, William L. Stanford, and Theodore J. Perkins\n",
      "Dependencies:\n",
      "  - Rsubread, type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - rtracklayer, type: depends\n",
      "  - matrixStats, type: depends\n",
      "  - IRanges, type: depends\n",
      "  - stats, type: imports\n",
      "  - methods, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ExCluster.html\n",
      "\n",
      "Name: FilterFFPE\n",
      "Description:\n",
      "This package finds and filters artificial chimeric reads specifically generated in next-generation sequencing (NGS) process of formalin-fixed paraffin-embedded (FFPE) tissues. These artificial chimeric reads can lead to a large number of false positive structural variation (SV) calls. The required input is an indexed BAM file of a FFPE sample.\n",
      "Version: 1.8.0\n",
      "Publication date: None\n",
      "Mantainer: Lanying Wei <lanying.wei@uni-muenster.de>\n",
      "Authors:\n",
      "Author: Lanying Wei [aut, cre]\n",
      "Dependencies:\n",
      "  - foreach, type: imports\n",
      "  - doParallel, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - parallel, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/FilterFFPE.html\n",
      "\n",
      "Name: flowCyBar\n",
      "Description:\n",
      "A package to analyze flow cytometric data using gate information to follow population/community dynamics\n",
      "Version: 1.34.0\n",
      "Publication date: None\n",
      "Mantainer: Joachim Schumann <joachim.schumann@ufz.de>\n",
      "Authors:\n",
      "Author: Joachim Schumann <joachim.schumann at ufz.de>, Christin Koch <christin.koch at ufz.de>, Susanne GÃ¼nther <susanne.guenther at ufz.de>, Ingo Fetzer <ingo.fetzer at stockholmresilience.su.se>, Susann MÃ¼ller <susann.mueller at ufz.de>\n",
      "Dependencies:\n",
      "  - R (>= 3.0.0) type: depends\n",
      "  - gplots, type: imports\n",
      "  - vegan, type: imports\n",
      "  - methods, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/flowCyBar.html\n",
      "\n",
      "Name: FScanR\n",
      "Description:\n",
      "'FScanR' identifies Programmed Ribosomal Frameshifting (PRF) events from BLASTX homolog sequence alignment between targeted genomic/cDNA/mRNA sequences against the peptide library of the same species or a close relative. The output by BLASTX or diamond BLASTX will be used as input of 'FScanR' and should be in a tabular format with 14 columns. For BLASTX, the output parameter should be: -outfmt '6 qseqid sseqid pident length mismatch gapopen qstart qend sstart send evalue bitscore qframe sframe'. For diamond BLASTX, the output parameter should be: -outfmt 6 qseqid sseqid pident length mismatch gapopen qstart qend sstart send evalue bitscore qframe qframe.\n",
      "Version: 1.8.0\n",
      "Publication date: None\n",
      "Mantainer: Xiao Chen <seanchen607@gmail.com>\n",
      "Authors:\n",
      "Author: Xiao Chen [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - stats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/FScanR.html\n",
      "\n",
      "Name: GA4GHshiny\n",
      "Description:\n",
      "GA4GHshiny package provides an easy way to interact with data servers based on Global Alliance for Genomics and Health (GA4GH) genomics API through a Shiny application. It also integrates with Beacon Network.\n",
      "Version: 1.20.0\n",
      "Publication date: None\n",
      "Mantainer: Welliton Souza <well309@gmail.com>\n",
      "Authors:\n",
      "Author: Welliton Souza [aut, cre], Benilton Carvalho [ctb], Cristiane Rocha [ctb], Elizabeth Borgognoni [ctb]\n",
      "Dependencies:\n",
      "  - GA4GHclient, type: depends\n",
      "  - AnnotationDbi, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - DT, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - openxlsx, type: imports\n",
      "  - GenomicFeatures, type: imports\n",
      "  - methods, type: imports\n",
      "  - purrr, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - shiny, type: imports\n",
      "  - shinyjs, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - shinythemes, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/GA4GHshiny.html\n",
      "\n",
      "Name: GeneGA\n",
      "Description:\n",
      "R based Genetic algorithm for gene expression optimization by considering both mRNA secondary structure and codon usage bias, GeneGA includes the information of highly expressed genes of almost 200 genomes. Meanwhile, Vienna RNA Package is needed to ensure GeneGA to function properly.\n",
      "Version: 1.48.0\n",
      "Publication date: None\n",
      "Mantainer: Zhenpeng Li <zpli21@gmail.com>\n",
      "Authors:\n",
      "Author: Zhenpeng Li and Haixiu Huang\n",
      "Dependencies:\n",
      "  - seqinr, type: depends\n",
      "  - hash, type: depends\n",
      "  - methods, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL version 2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/GeneGA.html\n",
      "\n",
      "Name: GEWIST\n",
      "Description:\n",
      "This 'GEWIST' package provides statistical tools to efficiently optimize SNP prioritization for gene-gene and gene-environment interactions.\n",
      "Version: 1.42.0\n",
      "Publication date: None\n",
      "Mantainer: Wei Q. Deng <dengwq@mcmaster.ca>\n",
      "Authors:\n",
      "Author: Wei Q. Deng, Guillaume Pare\n",
      "Dependencies:\n",
      "  - R (>= 2.10) type: depends\n",
      "  - car, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/GEWIST.html\n",
      "\n",
      "Name: HPAStainR\n",
      "Description:\n",
      "This package is built around the HPAStainR function. The purpose of the HPAStainR function is to query the visual staining data in the Human Protein Atlas to return a table of staining ranked cell types. The function also has multiple arguments to personalize to output as well to include cancer data, csv readable names, modify the confidence levels of the results and more. The other functions exist exclusively to easily acquire the data required to run HPAStainR.\n",
      "Version: 1.8.0\n",
      "Publication date: None\n",
      "Mantainer: Tim O. Nieuwenhuis <tnieuwe1@jhmi.edu>\n",
      "Authors:\n",
      "Author: Tim O. Nieuwenhuis [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.1.0) type: depends\n",
      "  - dplyr, type: depends\n",
      "  - tidyr, type: depends\n",
      "  - utils, type: imports\n",
      "  - stats, type: imports\n",
      "  - scales, type: imports\n",
      "  - stringr, type: imports\n",
      "  - tibble, type: imports\n",
      "  - shiny, type: imports\n",
      "  - data.table, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/HPAStainR.html\n",
      "\n",
      "Name: IMPCdata\n",
      "Description:\n",
      "Package contains methods for data retrieval from IMPC Database.\n",
      "Version: 1.34.0\n",
      "Publication date: None\n",
      "Mantainer: Jeremy Mason <jmason@ebi.ac.uk>\n",
      "Authors:\n",
      "Author: Natalja Kurbatova, Jeremy Mason\n",
      "Dependencies:\n",
      "  - R (>= 2.3.0) type: depends\n",
      "  - rjson, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/IMPCdata.html\n",
      "\n",
      "Name: IsoCorrectoRGUI\n",
      "Description:\n",
      "IsoCorrectoRGUI is a Graphical User Interface for the IsoCorrectoR package. IsoCorrectoR performs the correction of mass spectrometry data from stable isotope labeling/tracing metabolomics experiments with regard to natural isotope abundance and tracer impurity. Data from both MS and MS/MS measurements can be corrected (with any tracer isotope: 13C, 15N, 18O...), as well as high resolution MS data from multiple-tracer experiments (e.g. 13C and 15N used simultaneously).\n",
      "Version: 1.14.0\n",
      "Publication date: None\n",
      "Mantainer: Christian Kohler <christian.kohler@ur.de>\n",
      "Authors:\n",
      "Author: Christian Kohler [cre, aut], Paul Kuerner [aut], Paul Heinrich [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.6) type: depends\n",
      "  - IsoCorrectoR, type: imports\n",
      "  - readxl, type: imports\n",
      "  - tcltk2, type: imports\n",
      "  - tcltk, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/IsoCorrectoRGUI.html\n",
      "\n",
      "Name: IWTomics\n",
      "Description:\n",
      "Implementation of the Interval-Wise Testing (IWT) for omics data. This inferential procedure tests for differences in \"Omics\" data between two groups of genomic regions (or between a group of genomic regions and a reference center of symmetry), and does not require fixing location and scale at the outset.\n",
      "Version: 1.22.0\n",
      "Publication date: None\n",
      "Mantainer: Marzia A Cremona <mac78@psu.edu>\n",
      "Authors:\n",
      "Author: Marzia A Cremona, Alessia Pini, Francesca Chiaromonte, Simone Vantini\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - parallel, type: imports\n",
      "  - gtable, type: imports\n",
      "  - grid, type: imports\n",
      "  - graphics, type: imports\n",
      "  - methods, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - KernSmooth, type: imports\n",
      "  - fda, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - tools, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>=2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/IWTomics.html\n",
      "\n",
      "Name: LPEadj\n",
      "Description:\n",
      "Two options are added to the LPE algorithm. The original LPE method sets all variances below the max variance in the ordered distribution of variances to the maximum variance. in LPEadj this option is turned off by default. The second option is to use a variance adjustment based on sample size rather than pi/2. By default the LPEadj uses the sample size based variance adjustment.\n",
      "Version: 1.58.0\n",
      "Publication date: None\n",
      "Mantainer: Carl Murie <carl.murie@mcgill.ca>\n",
      "Authors:\n",
      "Author: Carl Murie <carl.murie at mcgill.ca>, Robert Nadon <robert.nadon at mcgill.ca>\n",
      "Dependencies:\n",
      "  - LPE, type: depends\n",
      "  - LPE, type: imports\n",
      "  - stats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/LPEadj.html\n",
      "\n",
      "Name: mapscape\n",
      "Description:\n",
      "MapScape integrates clonal prevalence, clonal hierarchy, anatomic and mutational information to provide interactive visualization of spatial clonal evolution. There are four inputs to MapScape: (i) the clonal phylogeny, (ii) clonal prevalences, (iii) an image reference, which may be a medical image or drawing and (iv) pixel locations for each sample on the referenced image. Optionally, MapScape can accept a data table of mutations for each clone and their variant allele frequencies in each sample. The output of MapScape consists of a cropped anatomical image surrounded by two representations of each tumour sample. The first, a cellular aggregate, visually displays the prevalence of each clone. The second shows a skeleton of the clonal phylogeny while highlighting only those clones present in the sample. Together, these representations enable the analyst to visualize the distribution of clones throughout anatomic space.\n",
      "Version: 1.22.0\n",
      "Publication date: None\n",
      "Mantainer: Maia Smith <maiaannesmith@gmail.com>\n",
      "Authors:\n",
      "Author: Maia Smith [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 3.3) type: depends\n",
      "  - htmlwidgets (>= 0.5) type: imports\n",
      "  - jsonlite (>= 0.9.19) type: imports\n",
      "  - base64enc (>= 0.1-3) type: imports\n",
      "  - stringr (>= 1.0.0) type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/mapscape.html\n",
      "\n",
      "Name: Melissa\n",
      "Description:\n",
      "Melissa is a Baysian probabilistic model for jointly clustering and imputing single cell methylomes. This is done by taking into account local correlations via a Generalised Linear Model approach and global similarities using a mixture modelling approach.\n",
      "Version: 1.14.0\n",
      "Publication date: None\n",
      "Mantainer: C. A. Kapourani <kapouranis.andreas@gmail.com>\n",
      "Authors:\n",
      "Author: C. A. Kapourani [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - BPRMeth, type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - data.table, type: imports\n",
      "  - parallel, type: imports\n",
      "  - ROCR, type: imports\n",
      "  - matrixcalc, type: imports\n",
      "  - mclust, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - doParallel, type: imports\n",
      "  - foreach, type: imports\n",
      "  - MCMCpack, type: imports\n",
      "  - cowplot, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - mvtnorm, type: imports\n",
      "  - truncnorm, type: imports\n",
      "  - assertthat, type: imports\n",
      "  - BiocStyle, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3 | file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/Melissa.html\n",
      "\n",
      "Name: MethPed\n",
      "Description:\n",
      "Classification of pediatric tumors into biologically defined subtypes is challenging and multifaceted approaches are needed. For this aim, we developed a diagnostic classifier based on DNA methylation profiles. We offer MethPed as an easy-to-use toolbox that allows researchers and clinical diagnosticians to test single samples as well as large cohorts for subclass prediction of pediatric brain tumors. The current version of MethPed can classify the following tumor diagnoses/subgroups: Diffuse Intrinsic Pontine Glioma (DIPG), Ependymoma, Embryonal tumors with multilayered rosettes (ETMR), Glioblastoma (GBM), Medulloblastoma (MB) - Group 3 (MB_Gr3), Group 4 (MB_Gr3), Group WNT (MB_WNT), Group SHH (MB_SHH) and Pilocytic Astrocytoma (PiloAstro).\n",
      "Version: 1.26.0\n",
      "Publication date: None\n",
      "Mantainer: Helena CarÃ©n <helenacarenlab@gmail.com>\n",
      "Authors:\n",
      "Author: Mohammad Tanvir Ahamed [aut, trl], Anna Danielsson [aut], SzilÃ¡rd Nemes [aut, trl], Helena CarÃ©n [aut, cre, cph]\n",
      "Dependencies:\n",
      "  - R (>= 3.0.0) type: depends\n",
      "  - Biobase, type: depends\n",
      "  - randomForest, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/MethPed.html\n",
      "\n",
      "Name: missRows\n",
      "Description:\n",
      "The missRows package implements the MI-MFA method to deal with missing individuals ('biological units') in multi-omics data integration. The MI-MFA method generates multiple imputed datasets from a Multiple Factor Analysis model, then the yield results are combined in a single consensus solution. The package provides functions for estimating coordinates of individuals and variables, imputing missing individuals, and various diagnostic plots to inspect the pattern of missingness and visualize the uncertainty due to missing values.\n",
      "Version: 1.18.0\n",
      "Publication date: None\n",
      "Mantainer: Gonzalez Ignacio <ignacio.gonzalez@bbox.fr>\n",
      "Authors:\n",
      "Author: Ignacio Gonzalez and Valentin Voillet\n",
      "Dependencies:\n",
      "  - R (>= 3.5) type: depends\n",
      "  - methods, type: depends\n",
      "  - ggplot2, type: depends\n",
      "  - grDevices, type: depends\n",
      "  - MultiAssayExperiment, type: depends\n",
      "  - plyr, type: imports\n",
      "  - stats, type: imports\n",
      "  - gtools, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/missRows.html\n",
      "\n",
      "Name: MsBackendRawFileReader\n",
      "Description:\n",
      "implements a MsBackend for the Spectra package using Thermo Fisher Scientific's NewRawFileReader .Net libraries. The package is generalizing the functionality introduced by the rawrr package (Kockmann T. et al. (2020) ) Methods defined in this package are supposed to extend the Spectra Bioconductor package.\n",
      "Version: 1.4.0\n",
      "Publication date: None\n",
      "Mantainer: Christian Panse <cp@fgcz.ethz.ch>\n",
      "Authors:\n",
      "Author: Christian Panse [aut, cre] , Tobias Kockmann [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - methods, type: depends\n",
      "  - Spectra, type: depends\n",
      "  - MsCoreUtils, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - rawrr, type: imports\n",
      "  - utils, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/MsBackendRawFileReader.html\n",
      "\n",
      "Name: ncGTW\n",
      "Description:\n",
      "The purpose of ncGTW is to help XCMS for LC-MS data alignment. Currently, ncGTW can detect the misaligned feature groups by XCMS, and the user can choose to realign these feature groups by ncGTW or not.\n",
      "Version: 1.12.0\n",
      "Publication date: None\n",
      "Mantainer: Chiung-Ting Wu <ctwu@vt.edu>\n",
      "Authors:\n",
      "Author: Chiung-Ting Wu <ctwu at vt.edu>\n",
      "Dependencies:\n",
      "  - methods, type: depends\n",
      "  - BiocParallel, type: depends\n",
      "  - xcms, type: depends\n",
      "  - Rcpp, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ncGTW.html\n",
      "\n",
      "Name: nearBynding\n",
      "Description:\n",
      "Provides a pipeline to discern RNA structure at and proximal to the site of protein binding within regions of the transcriptome defined by the user. CLIP protein-binding data can be input as either aligned BAM or peak-called bedGraph files. RNA structure can either be predicted internally from sequence or users have the option to input their own RNA structure data. RNA structure binding profiles can be visually and quantitatively compared across multiple formats.\n",
      "Version: 1.8.0\n",
      "Publication date: None\n",
      "Mantainer: Veronica Busa <vbusa1@jhmi.edu>\n",
      "Authors:\n",
      "Author: Veronica Busa [cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - R.utils, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - plyranges, type: imports\n",
      "  - transport, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - methods, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - utils, type: imports\n",
      "  - stats, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - TxDb.Hsapiens.UCSC.hg19.knownGene, type: imports\n",
      "  - TxDb.Hsapiens.UCSC.hg38.knownGene, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - gplots, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - rlang, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/nearBynding.html\n",
      "\n",
      "Name: netprioR\n",
      "Description:\n",
      "A model for semi-supervised prioritisation of genes integrating network data, phenotypes and additional prior knowledge about TP and TN gene labels from the literature or experts.\n",
      "Version: 1.24.0\n",
      "Publication date: None\n",
      "Mantainer: Fabian Schmich <fabian.schmich@bsse.ethz.ch>\n",
      "Authors:\n",
      "Author: Fabian Schmich\n",
      "Dependencies:\n",
      "  - methods, type: depends\n",
      "  - graphics, type: depends\n",
      "  - R (>= 3.3) type: depends\n",
      "  - stats, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - doParallel, type: imports\n",
      "  - foreach, type: imports\n",
      "  - parallel, type: imports\n",
      "  - sparseMVN, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - gridExtra, type: imports\n",
      "  - pROC, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/netprioR.html\n",
      "\n",
      "Name: netZooR\n",
      "Description:\n",
      "netZooR unifies the implementations of several Network Zoo methods (netzoo, netzoo.github.io) into a single package by creating interfaces between network inference and network analysis methods. Currently, the package has 3 methods for network inference including PANDA and its optimized implementation OTTER (network reconstruction using mutliple lines of biological evidence), LIONESS (single-sample network inference), and EGRET (genotype-specific networks). Network analysis methods include CONDOR (community detection), ALPACA (differential community detection), CRANE (significance estimation of differential modules), MONSTER (estimation of network transition states). In addition, YARN allows to process gene expresssion data for tissue-specific analyses and SAMBAR infers missing mutation data based on pathway information.\n",
      "Version: 1.2.0\n",
      "Publication date: None\n",
      "Mantainer: Marouen Ben Guebila <benguebila@hsph.harvard.edu>\n",
      "Authors:\n",
      "Author: Marouen Ben Guebila [aut, cre] , Tian Wang [aut] , John Platig [aut], Marieke Kuijjer [aut] , Megha Padi [aut] , Rebekka Burkholz [aut], Des Weighill [aut] , Kate Shutta [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 4.2.0) type: depends\n",
      "  - igraph, type: depends\n",
      "  - reticulate, type: depends\n",
      "  - pandaR, type: depends\n",
      "  - yarn, type: depends\n",
      "  - RCy3, type: imports\n",
      "  - viridisLite, type: imports\n",
      "  - STRINGdb, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - GOstats, type: imports\n",
      "  - AnnotationDbi, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - GO.db, type: imports\n",
      "  - org.Hs.eg.db, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - gplots, type: imports\n",
      "  - nnet, type: imports\n",
      "  - data.table, type: imports\n",
      "  - vegan, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - reshape, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - penalized, type: imports\n",
      "  - parallel, type: imports\n",
      "  - doParallel, type: imports\n",
      "  - foreach, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - ggdendro, type: imports\n",
      "  - grid, type: imports\n",
      "  - MASS, type: imports\n",
      "  - assertthat, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - methods, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - graphics, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/netZooR.html\n",
      "\n",
      "Name: Omixer\n",
      "Description:\n",
      "Omixer - an Bioconductor package for multivariate and reproducible sample randomization, which ensures optimal sample distribution across batches with well-documented methods. It outputs lab-friendly sample layouts, reducing the risk of sample mixups when manually pipetting randomized samples.\n",
      "Version: 1.8.0\n",
      "Publication date: None\n",
      "Mantainer: Lucy Sinke <l.j.sinke@lumc.nl>\n",
      "Authors:\n",
      "Author: Lucy Sinke [cre, aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.0.0) type: depends\n",
      "  - dplyr, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - forcats, type: imports\n",
      "  - tibble, type: imports\n",
      "  - gridExtra, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - readr, type: imports\n",
      "  - tidyselect, type: imports\n",
      "  - grid, type: imports\n",
      "  - stats, type: imports\n",
      "  - stringr, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/Omixer.html\n",
      "\n",
      "Name: padma\n",
      "Description:\n",
      "Use multiple factor analysis to calculate individualized pathway-centric scores of deviation with respect to the sampled population based on multi-omic assays (e.g., RNA-seq, copy number alterations, methylation, etc). Graphical and numerical outputs are provided to identify highly aberrant individuals for a particular pathway of interest, as well as the gene and omics drivers of aberrant multi-omic profiles.\n",
      "Version: 1.8.0\n",
      "Publication date: None\n",
      "Mantainer: Andrea Rau <andrea.rau@inrae.fr>\n",
      "Authors:\n",
      "Author: Andrea Rau [cre, aut] , Regina Manansala [aut], Florence JaffrÃ©zic [ctb], Denis LaloÃ« [aut], Paul Auer [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.1.0) type: depends\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - S4Vectors, type: depends\n",
      "  - FactoMineR, type: imports\n",
      "  - MultiAssayExperiment, type: imports\n",
      "  - methods, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>=3)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/padma.html\n",
      "\n",
      "Name: pcxn\n",
      "Description:\n",
      "Discover the correlated pathways/gene sets of a single pathway/gene set or discover correlation relationships among multiple pathways/gene sets. Draw a heatmap or create a network of your query and extract members of each pathway/gene set found in the available collections (MSigDB H hallmark, MSigDB C2 Canonical pathways, MSigDB C5 GO BP and Pathprint).\n",
      "Version: 2.20.0\n",
      "Publication date: None\n",
      "Mantainer: Sokratis Kariotis <s.kariotis@sheffield.ac.uk>\n",
      "Authors:\n",
      "Author: Sokratis Kariotis, Yered Pita-Juarez, Winston Hide, Wenbin Wei\n",
      "Dependencies:\n",
      "  - R (>= 3.4) type: depends\n",
      "  - pcxnData, type: depends\n",
      "  - methods, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - utils, type: imports\n",
      "  - pheatmap, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/pcxn.html\n",
      "\n",
      "Name: PDATK\n",
      "Description:\n",
      "Pancreatic ductal adenocarcinoma (PDA) has a relatively poor prognosis and is one of the most lethal cancers. Molecular classification of gene expression profiles holds the potential to identify meaningful subtypes which can inform therapeutic strategy in the clinical setting. The Pancreatic Cancer Adenocarcinoma Tool-Kit (PDATK) provides an S4 class-based interface for performing unsupervised subtype discovery, cross-cohort meta-clustering, gene-expression-based classification, and subsequent survival analysis to identify prognostically useful subtypes in pancreatic cancer and beyond. Two novel methods, Consensus Subtypes in Pancreatic Cancer (CSPC) and Pancreatic Cancer Overall Survival Predictor (PCOSP) are included for consensus-based meta-clustering and overall-survival prediction, respectively. Additionally, four published subtype classifiers and three published prognostic gene signatures are included to allow users to easily recreate published results, apply existing classifiers to new data, and benchmark the relative performance of new methods. The use of existing Bioconductor classes as input to all PDATK classes and methods enables integration with existing Bioconductor datasets, including the 21 pancreatic cancer patient cohorts available in the MetaGxPancreas data package. PDATK has been used to replicate results from Sandhu et al (2019) [https://doi.org/10.1200/cci.18.00102] and an additional paper is in the works using CSPC to validate subtypes from the included published classifiers, both of which use the data available in MetaGxPancreas. The inclusion of subtype centroids and prognostic gene signatures from these and other publications will enable researchers and clinicians to classify novel patient gene expression data, allowing the direct clinical application of the classifiers included in PDATK. Overall, PDATK provides a rich set of tools to identify and validate useful prognostic and molecular subtypes based on gene-expression data, benchmark new classifiers against existing ones, and apply discovered classifiers on novel patient data to inform clinical decision making.\n",
      "Version: 1.6.0\n",
      "Publication date: None\n",
      "Mantainer: Benjamin Haibe-Kains <benjamin.haibe.kains@utoronto.ca>\n",
      "Authors:\n",
      "Author: Vandana Sandhu [aut], Heewon Seo [aut], Christopher Eeles [aut], Neha Rohatgi [ctb], Benjamin Haibe-Kains [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - data.table, type: imports\n",
      "  - MultiAssayExperiment, type: imports\n",
      "  - ConsensusClusterPlus, type: imports\n",
      "  - igraph, type: imports\n",
      "  - ggplotify, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - clusterRepro, type: imports\n",
      "  - CoreGx, type: imports\n",
      "  - caret, type: imports\n",
      "  - survminer, type: imports\n",
      "  - methods, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - survival, type: imports\n",
      "  - stats, type: imports\n",
      "  - plyr, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - MatrixGenerics, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - rlang, type: imports\n",
      "  - piano, type: imports\n",
      "  - scales, type: imports\n",
      "  - survcomp, type: imports\n",
      "  - genefu, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - switchBox, type: imports\n",
      "  - reportROC, type: imports\n",
      "  - pROC, type: imports\n",
      "  - verification, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/PDATK.html\n",
      "\n",
      "Name: pgca\n",
      "Description:\n",
      "Protein Group Code Algorithm (PGCA) is a computationally inexpensive algorithm to merge protein summaries from multiple experimental quantitative proteomics data. The algorithm connects two or more groups with overlapping accession numbers. In some cases, pairwise groups are mutually exclusive but they may still be connected by another group (or set of groups) with overlapping accession numbers. Thus, groups created by PGCA from multiple experimental runs (i.e., global groups) are called \"connected\" groups. These identified global protein groups enable the analysis of quantitative data available for protein groups instead of unique protein identifiers.\n",
      "Version: 1.22.0\n",
      "Publication date: None\n",
      "Mantainer: Gabriela Cohen-Freue <gcohen@stat.ubc.ca>\n",
      "Authors:\n",
      "Author: Gabriela Cohen-Freue <gcohen at stat.ubc.ca>\n",
      "Dependencies:\n",
      "  - utils, type: imports\n",
      "  - stats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/pgca.html\n",
      "\n",
      "Name: pickgene\n",
      "Description:\n",
      "Functions to Analyze Microarray (Gene Expression) Data.\n",
      "Version: 1.70.0\n",
      "Publication date: None\n",
      "Mantainer: Brian S. Yandell <yandell@stat.wisc.edu>\n",
      "Authors:\n",
      "Author: Brian S. Yandell <yandell at stat.wisc.edu>\n",
      "Dependencies:\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - MASS, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/pickgene.html\n",
      "\n",
      "Name: pogos\n",
      "Description:\n",
      "Provide simple utilities for querying bhklab PharmacoDB, modeling API outputs, and integrating to cell and compound ontologies.\n",
      "Version: 1.18.0\n",
      "Publication date: None\n",
      "Mantainer: VJ Carey <stvjc@channing.harvard.edu>\n",
      "Authors:\n",
      "Author: Vince Carey <stvjc at channing.harvard.edu>\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - rjson (>= 0.2.15) type: depends\n",
      "  - httr (>= 1.3.1) type: depends\n",
      "  - methods, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - utils, type: imports\n",
      "  - shiny, type: imports\n",
      "  - ontoProc, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - graphics, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/pogos.html\n",
      "\n",
      "Name: primirTSS\n",
      "Description:\n",
      "A fast, convenient tool to identify the TSSs of miRNAs by integrating the data of H3K4me3 and Pol II as well as combining the conservation level and sequence feature, provided within both command-line and graphical interfaces, which achieves a better performance than the previous non-cell-specific methods on miRNA TSSs.\n",
      "Version: 1.16.0\n",
      "Publication date: None\n",
      "Mantainer: Pumin Li <ipumin@163.com>\n",
      "Authors:\n",
      "Author: Pumin Li [aut, cre], Qi Xu [aut], Jie Li [aut], Jin Wang [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - GenomicRanges, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - dplyr (>= 0.7.6) type: imports\n",
      "  - stringr (>= 1.3.1) type: imports\n",
      "  - tidyr (>= 0.8.1) type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - purrr (>= 0.2.5) type: imports\n",
      "  - BSgenome.Hsapiens.UCSC.hg38, type: imports\n",
      "  - phastCons100way.UCSC.hg38, type: imports\n",
      "  - GenomicScores, type: imports\n",
      "  - shiny (>= 1.0.5) type: imports\n",
      "  - Gviz, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - TFBSTools, type: imports\n",
      "  - JASPAR2018, type: imports\n",
      "  - tibble (>= 1.4.2) type: imports\n",
      "  - R.utils (>= 2.6.0) type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/primirTSS.html\n",
      "\n",
      "Name: procoil\n",
      "Description:\n",
      "The package allows for predicting whether a coiled coil sequence (amino acid sequence plus heptad register) is more likely to form a dimer or more likely to form a trimer. Additionally to the prediction itself, a prediction profile is computed which allows for determining the strengths to which the individual residues are indicative for either class. Prediction profiles can also be visualized as curves or heatmaps.\n",
      "Version: 2.26.0\n",
      "Publication date: None\n",
      "Mantainer: Ulrich Bodenhofer <bodenhofer@bioinf.jku.at>\n",
      "Authors:\n",
      "Author: Ulrich Bodenhofer\n",
      "Dependencies:\n",
      "  - R (>= 3.3.0) type: depends\n",
      "  - kebabs, type: depends\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - graphics, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/procoil.html\n",
      "\n",
      "Name: ProteoDisco\n",
      "Description:\n",
      "ProteoDisco is an R package to facilitate proteogenomics studies. It houses functions to create customized (variant) protein databases based on user-submitted genomic variants, splice-junctions, fusion genes and manual transcript sequences. The flexible workflow can be adopted to suit a myriad of research and experimental settings.\n",
      "Version: 1.4.0\n",
      "Publication date: None\n",
      "Mantainer: Job van Riet <jobvriet@gmail.com>\n",
      "Authors:\n",
      "Author: Job van Riet [cre], Wesley van de Geer [aut], Harmen van de Werken [ths]\n",
      "Dependencies:\n",
      "  - R (>= 4.1.0) type: depends\n",
      "  - BiocGenerics, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - checkmate (>= 2.0.0) type: imports\n",
      "  - cleaver, type: imports\n",
      "  - dplyr (>= 1.0.6) type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - GenomicFeatures, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - methods (>= 4.1.0) type: imports\n",
      "  - ParallelLogger (>= 2.0.1) type: imports\n",
      "  - plyr (>= 1.8.6) type: imports\n",
      "  - rlang (>= 0.4.11) type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - tibble (>= 3.1.2) type: imports\n",
      "  - tidyr (>= 1.1.3) type: imports\n",
      "  - VariantAnnotation, type: imports\n",
      "  - XVector, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ProteoDisco.html\n",
      "\n",
      "Name: RImmPort\n",
      "Description:\n",
      "The RImmPort package simplifies access to ImmPort data for analysis in the R environment. It provides a standards-based interface to the ImmPort study data that is in a proprietary format.\n",
      "Version: 1.26.0\n",
      "Publication date: None\n",
      "Mantainer: Zicheng Hu <Zicheng.Hu@ucsf.edu>, Ravi Shankar <rshankar@stanford.edu>\n",
      "Authors:\n",
      "Author: Ravi Shankar <rshankar at stanford.edu>\n",
      "Dependencies:\n",
      "  - plyr, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - DBI, type: imports\n",
      "  - data.table, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - methods, type: imports\n",
      "  - sqldf, type: imports\n",
      "  - tools, type: imports\n",
      "  - utils, type: imports\n",
      "  - RSQLite, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/RImmPort.html\n",
      "\n",
      "Name: RNAmodR.ML\n",
      "Description:\n",
      "RNAmodR.ML extend the functionality of the RNAmodR package and classical detection strategies towards detection through machine learning models. RNAmodR.ML provides classes, functions and an example workflow to establish a detection stratedy, which can be packaged.\n",
      "Version: 1.12.0\n",
      "Publication date: None\n",
      "Mantainer: Felix G.M. Ernst <felix.gm.ernst@outlook.com>\n",
      "Authors:\n",
      "Author: Felix G.M. Ernst [aut, cre] , Denis L.J. Lafontaine [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 3.6) type: depends\n",
      "  - RNAmodR, type: depends\n",
      "  - methods, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - stats, type: imports\n",
      "  - ranger, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/RNAmodR.ML.html\n",
      "\n",
      "Name: rprimer\n",
      "Description:\n",
      "Functions, workflow, and a Shiny application for visualizing sequence conservation and designing degenerate primers, probes, and (RT)-(q/d)PCR assays from a multiple DNA sequence alignment. The results can be presented in data frame format and visualized as dashboard-like plots. For more information, please see the package vignette.\n",
      "Version: 1.2.0\n",
      "Publication date: None\n",
      "Mantainer: Sofia Persson <sofiapersson27@gmail.com>\n",
      "Authors:\n",
      "Author: Sofia Persson [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - Biostrings, type: imports\n",
      "  - bslib, type: imports\n",
      "  - DT, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - mathjaxr, type: imports\n",
      "  - methods, type: imports\n",
      "  - patchwork, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - shiny, type: imports\n",
      "  - shinycssloaders, type: imports\n",
      "  - shinyFeedback, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/rprimer.html\n",
      "\n",
      "Name: SCATE\n",
      "Description:\n",
      "SCATE is a software tool for extracting and enhancing the sparse and discrete Single-cell ATAC-seq Signal. Single-cell sequencing assay for transposase-accessible chromatin (scATAC-seq) is the state-of-the-art technology for analyzing genome-wide regulatory landscapes in single cells. Single-cell ATAC-seq data are sparse and noisy, and analyzing such data is challenging. Existing computational methods cannot accurately reconstruct activities of individual cis-regulatory elements (CREs) in individual cells or rare cell subpopulations. SCATE was developed to adaptively integrate information from co-activated CREs, similar cells, and publicly available regulome data and substantially increase the accuracy for estimating activities of individual CREs. We demonstrate that SCATE can be used to better reconstruct the regulatory landscape of a heterogeneous sample.\n",
      "Version: 1.8.0\n",
      "Publication date: None\n",
      "Mantainer: Wenpin Hou <wp.hou3@gmail.com>\n",
      "Authors:\n",
      "Author: Zhicheng Ji [aut], Weiqiang Zhou [aut], Wenpin Hou [cre, aut] , Hongkai Ji [aut]\n",
      "Dependencies:\n",
      "  - parallel, type: depends\n",
      "  - preprocessCore, type: depends\n",
      "  - splines, type: depends\n",
      "  - splines2, type: depends\n",
      "  - xgboost, type: depends\n",
      "  - SCATEData, type: depends\n",
      "  - Rtsne, type: depends\n",
      "  - mclust, type: depends\n",
      "  - utils, type: imports\n",
      "  - stats, type: imports\n",
      "  - GenomicAlignments, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/SCATE.html\n",
      "\n",
      "Name: scBFA\n",
      "Description:\n",
      "This package is designed to model gene detection pattern of scRNA-seq through a binary factor analysis model. This model allows user to pass into a cell level covariate matrix X and gene level covariate matrix Q to account for nuisance variance(e.g batch effect), and it will output a low dimensional embedding matrix for downstream analysis.\n",
      "Version: 1.12.0\n",
      "Publication date: None\n",
      "Mantainer: Ruoxin Li <uskli@ucdavis.edu>\n",
      "Authors:\n",
      "Author: Ruoxin Li [aut, cre], Gerald Quon [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.6) type: depends\n",
      "  - SingleCellExperiment, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - Seurat, type: imports\n",
      "  - MASS, type: imports\n",
      "  - zinbwave, type: imports\n",
      "  - stats, type: imports\n",
      "  - copula, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - DESeq2, type: imports\n",
      "  - utils, type: imports\n",
      "  - grid, type: imports\n",
      "  - methods, type: imports\n",
      "  - Matrix, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3 + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/scBFA.html\n",
      "\n",
      "Name: SCFA\n",
      "Description:\n",
      "Subtyping via Consensus Factor Analysis (SCFA) can efficiently remove noisy signals from consistent molecular patterns in multi-omics data. SCFA first uses an autoencoder to select only important features and then repeatedly performs factor analysis to represent the data with different numbers of factors. Using these representations, it can reliably identify cancer subtypes and accurately predict risk scores of patients.\n",
      "Version: 1.8.0\n",
      "Publication date: None\n",
      "Mantainer: Duc Tran <duct@nevada.unr.edu>\n",
      "Authors:\n",
      "Author: Duc Tran [aut, cre], Hung Nguyen [aut], Tin Nguyen [fnd]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - matrixStats, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - torch (>= 0.3.0) type: imports\n",
      "  - coro, type: imports\n",
      "  - igraph, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - cluster, type: imports\n",
      "  - clusterCrit, type: imports\n",
      "  - psych, type: imports\n",
      "  - glmnet, type: imports\n",
      "  - RhpcBLASctl, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - methods, type: imports\n",
      "  - survival, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/SCFA.html\n",
      "\n",
      "Name: scoreInvHap\n",
      "Description:\n",
      "scoreInvHap can get the samples' inversion status of known inversions. scoreInvHap uses SNP data as input and requires the following information about the inversion: genotype frequencies in the different haplotypes, R2 between the region SNPs and inversion status and heterozygote genotypes in the reference. The package include this data for 21 inversions.\n",
      "Version: 1.20.0\n",
      "Publication date: None\n",
      "Mantainer: Dolors Pelegri-Siso <dolors.pelegri@isglobal.org>\n",
      "Authors:\n",
      "Author: Carlos Ruiz [aut], Dolors PelegrÃ­ [aut], Juan R. Gonzalez [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 3.6.0) type: depends\n",
      "  - Biostrings, type: imports\n",
      "  - methods, type: imports\n",
      "  - snpStats, type: imports\n",
      "  - VariantAnnotation, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - graphics, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/scoreInvHap.html\n",
      "\n",
      "Name: SemDist\n",
      "Description:\n",
      "This package implements methods to calculate information accretion for a given version of the gene ontology and uses this data to calculate remaining uncertainty, misinformation, and semantic similarity for given sets of predicted annotations and true annotations from a protein function predictor.\n",
      "Version: 1.32.0\n",
      "Publication date: None\n",
      "Mantainer: Ian Gonzalez <gonzalez.isv@gmail.com>\n",
      "Authors:\n",
      "Author: Ian Gonzalez and Wyatt Clark\n",
      "Dependencies:\n",
      "  - R (>= 3.1) type: depends\n",
      "  - AnnotationDbi, type: depends\n",
      "  - GO.db, type: depends\n",
      "  - annotate, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/SemDist.html\n",
      "\n",
      "Name: SigFuge\n",
      "Description:\n",
      "Algorithm for testing significance of clustering in RNA-seq data.\n",
      "Version: 1.36.0\n",
      "Publication date: None\n",
      "Mantainer: Patrick Kimes <patrick.kimes@gmail.com>\n",
      "Authors:\n",
      "Author: Patrick Kimes, Christopher Cabanski\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - ggplot2, type: imports\n",
      "  - matlab, type: imports\n",
      "  - reshape, type: imports\n",
      "  - sigclust, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/SigFuge.html\n",
      "\n",
      "Name: sigsquared\n",
      "Description:\n",
      "By leveraging statistical properties (log-rank test for survival) of patient cohorts defined by binary thresholds, poor-prognosis patients are identified by the sigsquared package via optimization over a cost function reducing type I and II error.\n",
      "Version: 1.30.0\n",
      "Publication date: None\n",
      "Mantainer: UnJin Lee <unjin@uchicago.edu>\n",
      "Authors:\n",
      "Author: UnJin Lee\n",
      "Dependencies:\n",
      "  - R (>= 3.2.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - Biobase, type: imports\n",
      "  - survival, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL version 3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/sigsquared.html\n",
      "\n",
      "Name: SimBindProfiles\n",
      "Description:\n",
      "SimBindProfiles identifies common and unique binding regions in genome tiling array data. This package does not rely on peak calling, but directly compares binding profiles processed on the same array platform. It implements a simple threshold approach, thus allowing retrieval of commonly and differentially bound regions between datasets as well as events of compensation and increased binding.\n",
      "Version: 1.36.0\n",
      "Publication date: None\n",
      "Mantainer: Bettina Fischer <bef22@cam.ac.uk>\n",
      "Authors:\n",
      "Author: Bettina Fischer, Enrico Ferrero, Robert Stojnic, Steve Russell\n",
      "Dependencies:\n",
      "  - R (>= 2.10) type: depends\n",
      "  - methods, type: depends\n",
      "  - Ringo, type: depends\n",
      "  - limma, type: imports\n",
      "  - mclust, type: imports\n",
      "  - Biobase, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/SimBindProfiles.html\n",
      "\n",
      "Name: skewr\n",
      "Description:\n",
      "The skewr package is a tool for visualizing the output of the Illumina Human Methylation 450k BeadChip to aid in quality control. It creates a panel of nine plots. Six of the plots represent the density of either the methylated intensity or the unmethylated intensity given by one of three subsets of the 485,577 total probes. These subsets include Type I-red, Type I-green, and Type II.The remaining three distributions give the density of the Beta-values for these same three subsets. Each of the nine plots optionally displays the distributions of the \"rs\" SNP probes and the probes associated with imprinted genes as series of 'tick' marks located above the x-axis.\n",
      "Version: 1.30.0\n",
      "Publication date: None\n",
      "Mantainer: Ryan Putney <ryanputney@gmail.com>\n",
      "Authors:\n",
      "Author: Ryan Putney [cre, aut], Steven Eschrich [aut], Anders Berglund [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.1.1) type: depends\n",
      "  - methylumi, type: depends\n",
      "  - wateRmelon, type: depends\n",
      "  - mixsmsn, type: depends\n",
      "  - IlluminaHumanMethylation450kmanifest, type: depends\n",
      "  - minfi, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/skewr.html\n",
      "\n",
      "Name: spkTools\n",
      "Description:\n",
      "The package contains functions that can be used to compare expression measures on different array platforms.\n",
      "Version: 1.54.0\n",
      "Publication date: None\n",
      "Mantainer: Matthew N McCall <mccallm@gmail.com>\n",
      "Authors:\n",
      "Author: Matthew N McCall <mccallm at gmail.com>, Rafael A Irizarry <rafa at jhu.edu>\n",
      "Dependencies:\n",
      "  - R (>= 2.7.0) type: depends\n",
      "  - Biobase, type: depends\n",
      "  - Biobase, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - gtools, type: imports\n",
      "  - methods, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/spkTools.html\n",
      "\n",
      "Name: spqn\n",
      "Description:\n",
      "The spqn package implements spatial quantile normalization (SpQN). This method was developed to remove a mean-correlation relationship in correlation matrices built from gene expression data. It can serve as pre-processing step prior to a co-expression analysis.\n",
      "Version: 1.10.0\n",
      "Publication date: None\n",
      "Mantainer: Yi Wang <yiwangthu5@gmail.com>\n",
      "Authors:\n",
      "Author: Yi Wang [cre, aut], Kasper Daniel Hansen [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - ggplot2, type: depends\n",
      "  - ggridges, type: depends\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - BiocGenerics, type: depends\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - matrixStats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/spqn.html\n",
      "\n",
      "Name: STROMA4\n",
      "Description:\n",
      "This package estimates four stromal properties identified in TNBC patients in each patient of a gene expression datasets. These stromal property assignments can be combined to subtype patients. These four stromal properties were identified in Triple negative breast cancer (TNBC) patients and represent the presence of different cells in the stroma: T-cells (T), B-cells (B), stromal infiltrating epithelial cells (E), and desmoplasia (D). Additionally this package can also be used to estimate generative properties for the Lehmann subtypes, an alternative TNBC subtyping scheme (PMID: 21633166).\n",
      "Version: 1.22.0\n",
      "Publication date: None\n",
      "Mantainer: Sadiq Saleh <sadiq.mehdiismailsaleh@mail.mcgill.ca>\n",
      "Authors:\n",
      "Author: Sadiq Saleh [aut, cre], Michael Hallett [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.4) type: depends\n",
      "  - Biobase, type: depends\n",
      "  - BiocParallel, type: depends\n",
      "  - cluster, type: depends\n",
      "  - matrixStats, type: depends\n",
      "  - stats, type: depends\n",
      "  - graphics, type: depends\n",
      "  - utils, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/STROMA4.html\n",
      "\n",
      "Name: SynMut\n",
      "Description:\n",
      "There are increasing demands on designing virus mutants with specific dinucleotide or codon composition. This tool can take both dinucleotide preference and/or codon usage bias into account while designing mutants. It is a powerful tool for in silico designs of DNA sequence mutants.\n",
      "Version: 1.14.0\n",
      "Publication date: None\n",
      "Mantainer: Haogao Gu <hggu@connect.hku.hk>\n",
      "Authors:\n",
      "Author: Haogao Gu [aut, cre], Leo L.M. Poon [led]\n",
      "Dependencies:\n",
      "  - seqinr, type: imports\n",
      "  - methods, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - stringr, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/SynMut.html\n",
      "\n",
      "Name: systemPipeTools\n",
      "Description:\n",
      "systemPipeTools package extends the widely used systemPipeR (SPR) workflow environment with an enhanced toolkit for data visualization, including utilities to automate the data visualizaton for analysis of differentially expressed genes (DEGs). systemPipeTools provides data transformation and data exploration functions via scatterplots, hierarchical clustering heatMaps, principal component analysis, multidimensional scaling, generalized principal components, t-Distributed Stochastic Neighbor embedding (t-SNE), and MA and volcano plots. All these utilities can be integrated with the modular design of the systemPipeR environment that allows users to easily substitute any of these features and/or custom with alternatives.\n",
      "Version: 1.6.0\n",
      "Publication date: None\n",
      "Mantainer: Daniela Cassol <danicassol@gmail.com>\n",
      "Authors:\n",
      "Author: Daniela Cassol [aut, cre], Ponmathi Ramasamy [aut], Le Zhang [aut], Thomas Girke [aut]\n",
      "Dependencies:\n",
      "  - DESeq2, type: imports\n",
      "  - GGally, type: imports\n",
      "  - Rtsne, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - ape, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - ggrepel, type: imports\n",
      "  - ggtree, type: imports\n",
      "  - glmpca, type: imports\n",
      "  - pheatmap, type: imports\n",
      "  - plotly, type: imports\n",
      "  - tibble, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - DT, type: imports\n",
      "  - stats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/systemPipeTools.html\n",
      "\n",
      "Name: TarSeqQC\n",
      "Description:\n",
      "The package allows the representation of targeted experiment in R. This is based on current packages and incorporates functions to do a quality control over this kind of experiments and a fast exploration of the sequenced regions. An xlsx file is generated as output.\n",
      "Version: 1.27.0\n",
      "Publication date: None\n",
      "Mantainer: Gabriela Merino <merino.gabriela33@gmail.com>\n",
      "Authors:\n",
      "Author: Gabriela A. Merino, Cristobal Fresno, Yanina Murua, Andrea S. Llera and Elmer A. Fernandez\n",
      "Dependencies:\n",
      "  - R (>= 3.5.1) type: depends\n",
      "  - methods, type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - Rsamtools, type: depends\n",
      "  - ggplot2, type: depends\n",
      "  - plyr, type: depends\n",
      "  - openxlsx, type: depends\n",
      "  - grDevices, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - cowplot, type: imports\n",
      "  - graphics, type: imports\n",
      "  - GenomicAlignments, type: imports\n",
      "  - Hmisc, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>=2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/TarSeqQC.html\n",
      "\n",
      "Name: TRESS\n",
      "Description:\n",
      "This package is devoted to analyzing MeRIP-seq data. Current functionalities include 1. detection of transcriptome wide m6A methylation regions 2. detection of transcriptome wide differential m6A methylation regions.\n",
      "Version: 1.4.0\n",
      "Publication date: None\n",
      "Mantainer: Zhenxing Guo <zhenxing.guo@emory.edu>\n",
      "Authors:\n",
      "Author: Zhenxing Guo [aut, cre], Hao Wu [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 4.1.0) type: depends\n",
      "  - parallel, type: depends\n",
      "  - S4Vectors, type: depends\n",
      "  - utils, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - stats, type: imports\n",
      "  - methods, type: imports\n",
      "  - graphics, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - GenomicFeatures, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - AnnotationDbi, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3 + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/TRESS.html\n",
      "\n",
      "Name: Uniquorn\n",
      "Description:\n",
      "'Uniquorn' enables users to identify cancer cell lines. Cancer cell line misidentification and cross-contamination reprents a significant challenge for cancer researchers. The identification is vital and in the frame of this package based on the locations/ loci of somatic and germline mutations/ variations. The input format is vcf/ vcf.gz and the files have to contain a single cancer cell line sample (i.e. a single member/genotype/gt column in the vcf file).\n",
      "Version: 2.18.0\n",
      "Publication date: None\n",
      "Mantainer: 'Raik Otto' <raik.otto@hu-berlin.de>\n",
      "Authors:\n",
      "Author: Raik Otto\n",
      "Dependencies:\n",
      "  - R (>= 3.5) type: depends\n",
      "  - stringr, type: imports\n",
      "  - R.utils, type: imports\n",
      "  - WriteXLS, type: imports\n",
      "  - stats, type: imports\n",
      "  - doParallel, type: imports\n",
      "  - foreach, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - VariantAnnotation, type: imports\n",
      "  - data.table, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/Uniquorn.html\n",
      "\n",
      "Name: VaSP\n",
      "Description:\n",
      "Discovery of genome-wide variable alternative splicing events from short-read RNA-seq data and visualizations of gene splicing information for publication-quality multi-panel figures in a population. (Warning: The visualizing function is removed due to the dependent package Sushi deprecated. If you want to use it, please change back to an older version.)\n",
      "Version: 1.10.0\n",
      "Publication date: None\n",
      "Mantainer: Huihui Yu <yuhuihui2011@foxmail.com>\n",
      "Authors:\n",
      "Author: Huihui Yu [aut, cre] , Qian Du [aut] , Chi Zhang [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - ballgown, type: depends\n",
      "  - IRanges, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - parallel, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - GenomicAlignments, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - cluster, type: imports\n",
      "  - stats, type: imports\n",
      "  - graphics, type: imports\n",
      "  - methods, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2.0)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/VaSP.html\n",
      "\n",
      "Name: xmapbridge\n",
      "Description:\n",
      "xmapBridge can plot graphs in the X:Map genome browser. This package exports plotting files in a suitable format.\n",
      "Version: 1.56.0\n",
      "Publication date: None\n",
      "Mantainer: Chris Wirth <Christopher.Wirth@cruk.manchester.ac.uk>\n",
      "Authors:\n",
      "Author: Tim Yates <Tim.Yates at cruk.manchester.ac.uk> and Crispin J Miller <Crispin.Miller at cruk.manchester.ac.uk>\n",
      "Dependencies:\n",
      "  - R (>= 2.0) type: depends\n",
      "  - methods, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/xmapbridge.html\n",
      "\n",
      "Name: BUMHMM\n",
      "Description:\n",
      "This is a probabilistic modelling pipeline for computing per- nucleotide posterior probabilities of modification from the data collected in structure probing experiments. The model supports multiple experimental replicates and empirically corrects coverage- and sequence-dependent biases. The model utilises the measure of a \"drop-off rate\" for each nucleotide, which is compared between replicates through a log-ratio (LDR). The LDRs between control replicates define a null distribution of variability in drop-off rate observed by chance and LDRs between treatment and control replicates gets compared to this distribution. Resulting empirical p-values (probability of being \"drawn\" from the null distribution) are used as observations in a Hidden Markov Model with a Beta-Uniform Mixture model used as an emission model. The resulting posterior probabilities indicate the probability of a nucleotide of having being modified in a structure probing experiment.\n",
      "Version: 1.22.0\n",
      "Publication date: None\n",
      "Mantainer: Alina Selega <alina.selega@gmail.com>\n",
      "Authors:\n",
      "Author: Alina Selega (alina.selega@gmail.com), Sander Granneman, Guido Sanguinetti\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - devtools, type: imports\n",
      "  - stringi, type: imports\n",
      "  - gtools, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - IRanges, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/BUMHMM.html\n",
      "\n",
      "Name: BUSseq\n",
      "Description:\n",
      "BUSseq R package fits an interpretable Bayesian hierarchical model---the Batch Effects Correction with Unknown Subtypes for scRNA seq Data (BUSseq)---to correct batch effects in the presence of unknown cell types. BUSseq is able to simultaneously correct batch effects, clusters cell types, and takes care of the count data nature, the overdispersion, the dropout events, and the cell-specific sequencing depth of scRNA-seq data. After correcting the batch effects with BUSseq, the corrected value can be used for downstream analysis as if all cells were sequenced in a single batch. BUSseq can integrate read count matrices obtained from different scRNA-seq platforms and allow cell types to be measured in some but not all of the batches as long as the experimental design fulfills the conditions listed in our manuscript.\n",
      "Version: 1.4.2\n",
      "Publication date: None\n",
      "Mantainer: Fangda Song <sfd1994895@gmail.com>\n",
      "Authors:\n",
      "Author: Fangda Song [aut, cre] , Ga Ming Chan [aut], Yingying Wei [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.6) type: depends\n",
      "  - SingleCellExperiment, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - gplots, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/BUSseq.html\n",
      "\n",
      "Name: cageminer\n",
      "Description:\n",
      "This package aims to integrate GWAS-derived SNPs and coexpression networks to mine candidate genes associated with a particular phenotype. For that, users must define a set of guide genes, which are known genes involved in the studied phenotype. Additionally, the mined candidates can be given a score that favor candidates that are hubs and/or transcription factors. The scores can then be used to rank and select the top n most promising genes for downstream experiments.\n",
      "Version: 1.4.0\n",
      "Publication date: None\n",
      "Mantainer: FabrÃ­cio Almeida-Silva <fabricio_almeidasilva@hotmail.com>\n",
      "Authors:\n",
      "Author: FabrÃ­cio Almeida-Silva [aut, cre] , Thiago Venancio [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - ggplot2, type: imports\n",
      "  - ggbio, type: imports\n",
      "  - ggtext, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - methods, type: imports\n",
      "  - BioNERO, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/cageminer.html\n",
      "\n",
      "Name: cliProfiler\n",
      "Description:\n",
      "An easy and fast way to visualize and profile the high-throughput IP data. This package generates the meta gene profile and other profiles. These profiles could provide valuable information for understanding the IP experiment results.\n",
      "Version: 1.4.0\n",
      "Publication date: None\n",
      "Mantainer: You Zhou <youzhoulearning@gmail.com>\n",
      "Authors:\n",
      "Author: You Zhou [aut, cre] , Kathi Zarnack [aut]\n",
      "Dependencies:\n",
      "  - S4Vectors, type: depends\n",
      "  - methods, type: depends\n",
      "  - R (>= 4.1) type: depends\n",
      "  - dplyr, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - BSgenome, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/cliProfiler.html\n",
      "\n",
      "Name: ComPrAn\n",
      "Description:\n",
      "This package is for analysis of SILAC labeled complexome profiling data. It uses peptide table in tab-delimited format as an input and produces ready-to-use tables and plots.\n",
      "Version: 1.6.0\n",
      "Publication date: None\n",
      "Mantainer: Petra Palenikova <palenikova3@gmail.com>\n",
      "Authors:\n",
      "Author: Rick Scavetta [aut], Petra Palenikova [aut, cre]\n",
      "Dependencies:\n",
      "  - data.table, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - forcats, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - purrr, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - rlang, type: imports\n",
      "  - stringr, type: imports\n",
      "  - shiny, type: imports\n",
      "  - DT, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - VennDiagram, type: imports\n",
      "  - rio, type: imports\n",
      "  - scales, type: imports\n",
      "  - shinydashboard, type: imports\n",
      "  - shinyjs, type: imports\n",
      "  - stats, type: imports\n",
      "  - tibble, type: imports\n",
      "  - grid, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ComPrAn.html\n",
      "\n",
      "Name: COTAN\n",
      "Description:\n",
      "Statistical and computational method to analyze the co-expression of gene pairs at single cell level. It provides the foundation for single-cell gene interactome analysis. The basic idea is studying the zero UMI counts' distribution instead of focusing on positive counts; this is done with a generalized contingency tables framework. COTAN can effectively assess the correlated or anti-correlated expression of gene pairs. It provides a numerical index related to the correlation and an approximate p-value for the associated independence test. COTAN can also evaluate whether single genes are differentially expressed, scoring them with a newly defined global differentiation index. Moreover, this approach provides ways to plot and cluster genes according to their co-expression pattern with other genes, effectively helping the study of gene interactions and becoming a new tool to identify cell-identity marker genes.\n",
      "Version: 1.2.0\n",
      "Publication date: None\n",
      "Mantainer: GalfrÃ¨ Silvia Giulia <silvia.galfre@uniroma1.it>\n",
      "Authors:\n",
      "Author: GalfrÃ¨ Silvia Giulia [aut, cre] , Morandin Francesco [aut] , Pietrosanto Marco [aut] , Cremisi Federico [aut] , Helmer-Citterich Manuela [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - dplyr, type: imports\n",
      "  - methods, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - ggrepel, type: imports\n",
      "  - stats, type: imports\n",
      "  - parallel, type: imports\n",
      "  - tibble, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - irlba, type: imports\n",
      "  - ComplexHeatmap, type: imports\n",
      "  - circlize, type: imports\n",
      "  - grid, type: imports\n",
      "  - scales, type: imports\n",
      "  - utils, type: imports\n",
      "  - rlang, type: imports\n",
      "  - Rfast, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/COTAN.html\n",
      "\n",
      "Name: CSSQ\n",
      "Description:\n",
      "This package is desgined to perform statistical analysis to identify statistically significant differentially bound regions between multiple groups of ChIP-seq dataset.\n",
      "Version: 1.10.0\n",
      "Publication date: None\n",
      "Mantainer: Fan Lab@Georgia Institute of Technology <yuhong.fan@biology.gatech.edu>\n",
      "Authors:\n",
      "Author: Ashwath Kumar [aut], Michael Y Hu [aut], Yajun Mei [aut], Yuhong Fan [aut]\n",
      "Dependencies:\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - IRanges, type: depends\n",
      "  - S4Vectors, type: depends\n",
      "  - rtracklayer, type: depends\n",
      "  - GenomicAlignments, type: imports\n",
      "  - GenomicFeatures, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/CSSQ.html\n",
      "\n",
      "Name: DelayedTensor\n",
      "Description:\n",
      "DelayedTensor operates Tensor arithmetic directly on DelayedArray object. DelayedTensor provides some generic function related to Tensor arithmetic/decompotision and dispatches it on the DelayedArray class. DelayedTensor also suppors Tensor contraction by einsum function, which is inspired by numpy einsum.\n",
      "Version: 1.4.0\n",
      "Publication date: None\n",
      "Mantainer: Koki Tsuyuzaki <k.t.the-answer@hotmail.co.jp>\n",
      "Authors:\n",
      "Author: Koki Tsuyuzaki [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.1.0) type: depends\n",
      "  - methods, type: imports\n",
      "  - utils, type: imports\n",
      "  - DelayedArray, type: imports\n",
      "  - HDF5Array, type: imports\n",
      "  - BiocSingular, type: imports\n",
      "  - rTensor, type: imports\n",
      "  - DelayedRandomArray, type: imports\n",
      "  - irlba, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - einsum, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/DelayedTensor.html\n",
      "\n",
      "Name: deltaGseg\n",
      "Description:\n",
      "Identifying distinct subpopulations through multiscale time series analysis\n",
      "Version: 1.38.0\n",
      "Publication date: None\n",
      "Mantainer: Diana Low <lowdiana@gmail.com>\n",
      "Authors:\n",
      "Author: Diana Low, Efthymios Motakis\n",
      "Dependencies:\n",
      "  - R (>= 2.15.1) type: depends\n",
      "  - methods, type: depends\n",
      "  - ggplot2, type: depends\n",
      "  - changepoint, type: depends\n",
      "  - wavethresh, type: depends\n",
      "  - tseries, type: depends\n",
      "  - pvclust, type: depends\n",
      "  - fBasics, type: depends\n",
      "  - grid, type: depends\n",
      "  - reshape, type: depends\n",
      "  - scales, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/deltaGseg.html\n",
      "\n",
      "Name: Director\n",
      "Description:\n",
      "Director is an R package designed to streamline the visualization of molecular effects in regulatory cascades. It utilizes the R package htmltools and a modified Sankey plugin of the JavaScript library D3 to provide a fast and easy, browser-enabled solution to discovering potentially interesting downstream effects of regulatory and/or co-expressed molecules. The diagrams are robust, interactive, and packaged as highly-portable HTML files that eliminate the need for third-party software to view. This enables a straightforward approach for scientists to interpret the data produced, and bioinformatics developers an alternative means to present relevant data.\n",
      "Version: 1.24.0\n",
      "Publication date: None\n",
      "Mantainer: Katherine Icay <kat.icay@gmail.com>\n",
      "Authors:\n",
      "Author: Katherine Icay [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - htmltools, type: imports\n",
      "  - utils, type: imports\n",
      "  - grDevices, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3 + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/Director.html\n",
      "\n",
      "Name: divergence\n",
      "Description:\n",
      "This package provides functionality for performing divergence analysis as presented in Dinalankara et al, \"Digitizing omics profiles by divergence from a baseline\", PANS 2018. This allows the user to simplify high dimensional omics data into a binary or ternary format which encapsulates how the data is divergent from a specified baseline group with the same univariate or multivariate features.\n",
      "Version: 1.14.0\n",
      "Publication date: None\n",
      "Mantainer: Wikum Dinalankara <wdd4001@med.cornell.edu>, Luigi Marchionni <marchion@jhu.edu>\n",
      "Authors:\n",
      "Author: Wikum Dinalankara <wdd4001 at med.cornell.edu>, Luigi Marchionni <marchion at jhu.edu>, Qian Ke <qke1 at jhu.edu>\n",
      "Dependencies:\n",
      "  - R (>= 3.6) type: depends\n",
      "  - SummarizedExperiment, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/divergence.html\n",
      "\n",
      "Name: DominoEffect\n",
      "Description:\n",
      "The functions support identification and annotation of hotspot residues in proteins. These are individual amino acids that accumulate mutations at a much higher rate than their surrounding regions.\n",
      "Version: 1.18.0\n",
      "Publication date: None\n",
      "Mantainer: Marija Buljan <marija.buljan.2@gmail.com>, Peter Blattmann <blattmann@imsb.biol.ethz.ch>\n",
      "Authors:\n",
      "Author: Marija Buljan and Peter Blattmann\n",
      "Dependencies:\n",
      "  - R (>= 3.5) type: depends\n",
      "  - biomaRt, type: imports\n",
      "  - data.table, type: imports\n",
      "  - utils, type: imports\n",
      "  - stats, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - VariantAnnotation, type: imports\n",
      "  - AnnotationDbi, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - methods, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 3)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/DominoEffect.html\n",
      "\n",
      "Name: Doscheda\n",
      "Description:\n",
      "Doscheda focuses on quantitative chemoproteomics used to determine protein interaction profiles of small molecules from whole cell or tissue lysates using Mass Spectrometry data. The package provides a shiny application to run the pipeline, several visualisations and a downloadable report of an experiment.\n",
      "Version: 1.20.0\n",
      "Publication date: None\n",
      "Mantainer: Bruno Contrino <br1contrino@yahoo.co.uk>\n",
      "Authors:\n",
      "Author: Bruno Contrino, Piero Ricchiuto\n",
      "Dependencies:\n",
      "  - R (>= 3.4) type: depends\n",
      "  - methods, type: imports\n",
      "  - drc, type: imports\n",
      "  - stats, type: imports\n",
      "  - httr, type: imports\n",
      "  - jsonlite, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - vsn, type: imports\n",
      "  - affy, type: imports\n",
      "  - limma, type: imports\n",
      "  - stringr, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - calibrate, type: imports\n",
      "  - corrgram, type: imports\n",
      "  - gridExtra, type: imports\n",
      "  - DT, type: imports\n",
      "  - shiny, type: imports\n",
      "  - shinydashboard, type: imports\n",
      "  - readxl, type: imports\n",
      "  - prodlim, type: imports\n",
      "  - matrixStats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/Doscheda.html\n",
      "\n",
      "Name: eudysbiome\n",
      "Description:\n",
      "eudysbiome a package that permits to annotate the differential genera as harmful/harmless based on their ability to contribute to host diseases (as indicated in literature) or unknown based on their ambiguous genus classification. Further, the package statistically measures the eubiotic (harmless genera increase or harmful genera decrease) or dysbiotic(harmless genera decrease or harmful genera increase) impact of a given treatment or environmental change on the (gut-intestinal, GI) microbiome in comparison to the microbiome of the reference condition.\n",
      "Version: 1.28.0\n",
      "Publication date: None\n",
      "Mantainer: Xiaoyuan Zhou <zhouxiaoyuan@picb.ac.cn>\n",
      "Authors:\n",
      "Author: Xiaoyuan Zhou, Christine Nardini\n",
      "Dependencies:\n",
      "  - R (>= 3.1.0) type: depends\n",
      "  - plyr, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - R.utils, type: imports\n",
      "  - Biostrings, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/eudysbiome.html\n",
      "\n",
      "Name: flowGraph\n",
      "Description:\n",
      "Identifies maximal differential cell populations in flow cytometry data taking into account dependencies between cell populations; flowGraph calculates and plots SpecEnr abundance scores given cell population cell counts.\n",
      "Version: 1.6.0\n",
      "Publication date: None\n",
      "Mantainer: Alice Yue <aya43@sfu.ca>\n",
      "Authors:\n",
      "Author: Alice Yue [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - effsize, type: imports\n",
      "  - furrr, type: imports\n",
      "  - future, type: imports\n",
      "  - purrr, type: imports\n",
      "  - ggiraph, type: imports\n",
      "  - ggrepel, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - igraph, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - visNetwork, type: imports\n",
      "  - htmlwidgets, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - methods, type: imports\n",
      "  - stringr, type: imports\n",
      "  - stringi, type: imports\n",
      "  - Rdpack, type: imports\n",
      "  - data.table (>= 1.9.5) type: imports\n",
      "  - gridExtra, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/flowGraph.html\n",
      "\n",
      "Name: genomicInstability\n",
      "Description:\n",
      "This package contain functions to run genomic instability analysis (GIA) from scRNA-Seq data. GIA estimates the association between gene expression and genomic location of the coding genes. It uses the aREA algorithm to quantify the enrichment of sets of contiguous genes (loci-blocks) on the gene expression profiles and estimates the Genomic Instability Score (GIS) for each analyzed cell.\n",
      "Version: 1.4.0\n",
      "Publication date: None\n",
      "Mantainer: Mariano Alvarez <reef103@gmail.com>\n",
      "Authors:\n",
      "Author: Mariano Alvarez [aut, cre], Pasquale Laise [aut], DarwinHealth [cph]\n",
      "Dependencies:\n",
      "  - R (>= 4.1.0) type: depends\n",
      "  - checkmate, type: depends\n",
      "  - mixtools, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/genomicInstability.html\n",
      "\n",
      "Name: GladiaTOX\n",
      "Description:\n",
      "GladiaTOX R package is an open-source, flexible solution to high-content screening data processing and reporting in biomedical research. GladiaTOX takes advantage of the tcpl core functionalities and provides a number of extensions: it provides a web-service solution to fetch raw data; it computes severity scores and exports ToxPi formatted files; furthermore it contains a suite of functionalities to generate pdf reports for quality control and data processing.\n",
      "Version: 1.14.0\n",
      "Publication date: None\n",
      "Mantainer: PMP S.A. R Support <DL.RSupport@pmi.com>\n",
      "Authors:\n",
      "Author: Vincenzo Belcastro [aut, cre], Dayne L Filer [aut], Stephane Cano [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.6.0) type: depends\n",
      "  - data.table (>= 1.9.4) type: depends\n",
      "  - DBI, type: imports\n",
      "  - RMySQL, type: imports\n",
      "  - RSQLite, type: imports\n",
      "  - numDeriv, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - parallel, type: imports\n",
      "  - stats, type: imports\n",
      "  - methods, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - xtable, type: imports\n",
      "  - tools, type: imports\n",
      "  - brew, type: imports\n",
      "  - stringr, type: imports\n",
      "  - RJSONIO, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - ggrepel, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - utils, type: imports\n",
      "  - RCurl, type: imports\n",
      "  - XML, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/GladiaTOX.html\n",
      "\n",
      "Name: HPiP\n",
      "Description:\n",
      "HPiP (Host-Pathogen Interaction Prediction) uses an ensemble learning algorithm for prediction of host-pathogen protein-protein interactions (HP-PPIs) using structural and physicochemical descriptors computed from amino acid-composition of host and pathogen proteins.The proposed package can effectively address data shortages and data unavailability for HP-PPI network reconstructions. Moreover, establishing computational frameworks in that regard will reveal mechanistic insights into infectious diseases and suggest potential HP-PPI targets, thus narrowing down the range of possible candidates for subsequent wet-lab experimental validations.\n",
      "Version: 1.4.0\n",
      "Publication date: None\n",
      "Mantainer: Matineh Rahmatbakhsh <matinerb.94@gmail.com>\n",
      "Authors:\n",
      "Author: Matineh Rahmatbakhsh [aut, trl, cre], Mohan Babu [led]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - dplyr (>= 1.0.6) type: imports\n",
      "  - httr (>= 1.4.2) type: imports\n",
      "  - readr, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - tibble, type: imports\n",
      "  - utils, type: imports\n",
      "  - stringr, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - caret, type: imports\n",
      "  - corrplot, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - pROC, type: imports\n",
      "  - PRROC, type: imports\n",
      "  - igraph, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - purrr, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - protr, type: imports\n",
      "  - MCL, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/HPiP.html\n",
      "\n",
      "Name: hummingbird\n",
      "Description:\n",
      "A package for detecting differential methylation. It exploits a Bayesian hidden Markov model that incorporates location dependence among genomic loci, unlike most existing methods that assume independence among observations. Bayesian priors are applied to permit information sharing across an entire chromosome for improved power of detection. The direct output of our software package is the best sequence of methylation states, eliminating the use of a subjective, and most of the time an arbitrary, threshold of p-value for determining significance. At last, our methodology does not require replication in either or both of the two comparison groups.\n",
      "Version: 1.8.0\n",
      "Publication date: None\n",
      "Mantainer: Eleni Adam <eadam002@odu.edu>\n",
      "Authors:\n",
      "Author: Eleni Adam [aut, cre], Tieming Ji [aut], Desh Ranjan [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - Rcpp, type: imports\n",
      "  - graphics, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - IRanges, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>=2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/hummingbird.html\n",
      "\n",
      "Name: IgGeneUsage\n",
      "Description:\n",
      "Detection of biases in the usage of immunoglobulin (Ig) genes is an important task in immune repertoire profiling. IgGeneUsage detects aberrant Ig gene usage between biological conditions using a probabilistic model which is analyzed computationally by Bayes inference. With this IgGeneUsage also avoids some common problems related to the current practice of null-hypothesis significance testing.\n",
      "Version: 1.12.0\n",
      "Publication date: None\n",
      "Mantainer: Simo Kitanovski <simo.kitanovski@uni-due.de>\n",
      "Authors:\n",
      "Author: Simo Kitanovski [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.2.0) type: depends\n",
      "  - methods, type: imports\n",
      "  - reshape2 (>= 1.4.3) type: imports\n",
      "  - Rcpp (>= 0.12.0) type: imports\n",
      "  - RcppParallel (>= 5.0.1) type: imports\n",
      "  - rstan (>= 2.18.1) type: imports\n",
      "  - rstantools (>= 2.2.0) type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/IgGeneUsage.html\n",
      "\n",
      "Name: immunotation\n",
      "Description:\n",
      "MHC (major histocompatibility complex) molecules are cell surface complexes that present antigens to T cells. The repertoire of antigens presented in a given genetic background largely depends on the sequence of the encoded MHC molecules, and thus, in humans, on the highly variable HLA (human leukocyte antigen) genes of the hyperpolymorphic HLA locus. More than 28,000 different HLA alleles have been reported, with significant differences in allele frequencies between human populations worldwide. Reproducible and consistent annotation of HLA alleles in large-scale bioinformatics workflows remains challenging, because the available reference databases and software tools often use different HLA naming schemes. The package immunotation provides tools for consistent annotation of HLA genes in typical immunoinformatics workflows such as for example the prediction of MHC-presented peptides in different human donors. Converter functions that provide mappings between different HLA naming schemes are based on the MHC restriction ontology (MRO). The package also provides automated access to HLA alleles frequencies in worldwide human reference populations stored in the Allele Frequency Net Database.\n",
      "Version: 1.5.0\n",
      "Publication date: None\n",
      "Mantainer: Katharina Imkeller <k.imkeller@dkfz.de>\n",
      "Authors:\n",
      "Author: Katharina Imkeller [cre, aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - stringr, type: imports\n",
      "  - ontologyIndex, type: imports\n",
      "  - curl, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - readr, type: imports\n",
      "  - rvest, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - xml2, type: imports\n",
      "  - maps, type: imports\n",
      "  - rlang, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/immunotation.html\n",
      "\n",
      "Name: loci2path\n",
      "Description:\n",
      "loci2path performs statistics-rigorous enrichment analysis of eQTLs in genomic regions of interest. Using eQTL collections provided by the Genotype-Tissue Expression (GTEx) project and pathway collections from MSigDB.\n",
      "Version: 1.18.0\n",
      "Publication date: None\n",
      "Mantainer: Tianlei Xu <tianlei.xu@emory.edu>\n",
      "Authors:\n",
      "Author: Tianlei Xu\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - pheatmap, type: imports\n",
      "  - wordcloud, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - data.table, type: imports\n",
      "  - methods, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - stats, type: imports\n",
      "  - graphics, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/loci2path.html\n",
      "\n",
      "Name: lpNet\n",
      "Description:\n",
      "lpNet aims at infering biological networks, in particular signaling and gene networks. For that it takes perturbation data, either steady-state or time-series, as input and generates an LP model which allows the inference of signaling networks. For parameter identification either leave-one-out cross-validation or stratified n-fold cross-validation can be used.\n",
      "Version: 2.30.0\n",
      "Publication date: None\n",
      "Mantainer: Lars Kaderali <lars.kaderali@uni-greifswald.de>\n",
      "Authors:\n",
      "Author: Bettina Knapp, Marta R. A. Matos, Johanna Mazur, Lars Kaderali\n",
      "Dependencies:\n",
      "  - lpSolve, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic License 2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/lpNet.html\n",
      "\n",
      "Name: matchBox\n",
      "Description:\n",
      "The matchBox package enables comparing ranked vectors of features, merging multiple datasets, removing redundant features, using CAT-plots and Venn diagrams, and computing statistical significance.\n",
      "Version: 1.40.0\n",
      "Publication date: None\n",
      "Mantainer: Luigi Marchionni <marchion@jhu.edu>, Anuj Gupta <agupta52@jhu.edu>\n",
      "Authors:\n",
      "Author: Luigi Marchionni <marchion at jhu.edu>, Anuj Gupta <agupta52 at jhu.edu>\n",
      "Dependencies:\n",
      "  - R (>= 2.8.0) type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/matchBox.html\n",
      "\n",
      "Name: MatrixRider\n",
      "Description:\n",
      "Calculates a single number for a whole sequence that reflects the propensity of a DNA binding protein to interact with it. The DNA binding protein has to be described with a PFM matrix, for example gotten from Jaspar.\n",
      "Version: 1.30.0\n",
      "Publication date: None\n",
      "Mantainer: Elena Grassi <grassi.e@gmail.com>\n",
      "Authors:\n",
      "Author: Elena Grassi\n",
      "Dependencies:\n",
      "  - R (>= 3.1.2) type: depends\n",
      "  - methods, type: imports\n",
      "  - TFBSTools, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - XVector, type: imports\n",
      "  - Biostrings, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/MatrixRider.html\n",
      "\n",
      "Name: MiChip\n",
      "Description:\n",
      "This package takes the MiChip miRNA microarray .grp scanner output files and parses these out, providing summary and plotting functions to analyse MiChip hybridizations. A set of hybridizations is packaged into an ExpressionSet allowing it to be used by other BioConductor packages.\n",
      "Version: 1.52.0\n",
      "Publication date: None\n",
      "Mantainer: Jonathon Blake <blake@embl.de>\n",
      "Authors:\n",
      "Author: Jonathon Blake <blake at embl.de>\n",
      "Dependencies:\n",
      "  - R (>= 2.3.0) type: depends\n",
      "  - Biobase, type: depends\n",
      "  - Biobase, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/MiChip.html\n",
      "\n",
      "Name: mina\n",
      "Description:\n",
      "An increasing number of microbiome datasets have been generated and analyzed with the help of rapidly developing sequencing technologies. At present, analysis of taxonomic profiling data is mainly conducted using composition-based methods, which ignores interactions between community members. Besides this, a lack of efficient ways to compare microbial interaction networks limited the study of community dynamics. To better understand how community diversity is affected by complex interactions between its members, we developed a framework (Microbial community dIversity and Network Analysis, mina), a comprehensive framework for microbial community diversity analysis and network comparison. By defining and integrating network-derived community features, we greatly reduce noise-to-signal ratio for diversity analyses. A bootstrap and permutation-based method was implemented to assess community network dissimilarities and extract discriminative features in a statistically principled way.\n",
      "Version: 1.6.0\n",
      "Publication date: None\n",
      "Mantainer: Rui Guan <guan@mpipz.mpg.de>\n",
      "Authors:\n",
      "Author: Rui Guan [aut, cre], Ruben Garrido-Oter [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 4.0.0) type: depends\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - Rcpp, type: imports\n",
      "  - MCL, type: imports\n",
      "  - RSpectra, type: imports\n",
      "  - apcluster, type: imports\n",
      "  - bigmemory, type: imports\n",
      "  - foreach, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - parallel, type: imports\n",
      "  - parallelDist, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - plyr, type: imports\n",
      "  - biganalytics, type: imports\n",
      "  - stringr, type: imports\n",
      "  - Hmisc, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/mina.html\n",
      "\n",
      "Name: miRmine\n",
      "Description:\n",
      "miRmine database is a collection of expression profiles from different publicly available miRNA-seq datasets, Panwar et al (2017) miRmine: A Database of Human miRNA Expression, prepared with this data package as RangedSummarizedExperiment.\n",
      "Version: 1.20.0\n",
      "Publication date: None\n",
      "Mantainer: Dusan Randjelovic <dusan.randjelovic@sbgenomics.com>\n",
      "Authors:\n",
      "Author: Dusan Randjelovic [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - SummarizedExperiment, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 3)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/miRmine.html\n",
      "\n",
      "Name: mitoClone2\n",
      "Description:\n",
      "This package primarily identifies variants in mitochondrial genomes from BAM alignment files. It filters these variants to remove RNA editing events then estimates their evolutionary relationship (i.e. their phylogenetic tree) and groups single cells into clones. It also visualizes the mutations and providing additional genomic context.\n",
      "Version: 1.4.0\n",
      "Publication date: None\n",
      "Mantainer: Benjamin Story <story.benjamin@gmail.com>\n",
      "Authors:\n",
      "Author: Benjamin Story [aut, cre], Lars Velten [aut], Gregor MÃ¶nke [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.1.0) type: depends\n",
      "  - reshape2, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - pheatmap, type: imports\n",
      "  - deepSNV, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - Rhtslib, type: imports\n",
      "  - parallel, type: imports\n",
      "  - methods, type: imports\n",
      "  - ggplot2, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/mitoClone2.html\n",
      "\n",
      "Name: NeuCA\n",
      "Description:\n",
      "NeuCA is is a neural-network based method for scRNA-seq data annotation. It can automatically adjust its classification strategy depending on cell type correlations, to accurately annotate cell. NeuCA can automatically utilize the structure information of the cell types through a hierarchical tree to improve the annotation accuracy. It is especially helpful when the data contain closely correlated cell types.\n",
      "Version: 1.4.0\n",
      "Publication date: None\n",
      "Mantainer: Hao Feng <hxf155@case.edu>\n",
      "Authors:\n",
      "Author: Ziyi Li [aut], Hao Feng [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - keras, type: depends\n",
      "  - limma, type: depends\n",
      "  - e1071, type: depends\n",
      "  - SingleCellExperiment, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/NeuCA.html\n",
      "\n",
      "Name: normalize450K\n",
      "Description:\n",
      "Precise measurements are important for epigenome-wide studies investigating DNA methylation in whole blood samples, where effect sizes are expected to be small in magnitude. The 450K platform is often affected by batch effects and proper preprocessing is recommended. This package provides functions to read and normalize 450K '.idat' files. The normalization corrects for dye bias and biases related to signal intensity and methylation of probes using local regression. No adjustment for probe type bias is performed to avoid the trade-off of precision for accuracy of beta-values.\n",
      "Version: 1.26.0\n",
      "Publication date: None\n",
      "Mantainer: Jonathan Alexander Heiss <jonathan.heiss@posteo.de>\n",
      "Authors:\n",
      "Author: Jonathan Alexander Heiss\n",
      "Dependencies:\n",
      "  - R (>= 3.3) type: depends\n",
      "  - Biobase, type: depends\n",
      "  - illuminaio, type: depends\n",
      "  - quadprog, type: depends\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "BSD_2_clause + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/normalize450K.html\n",
      "\n",
      "Name: OPWeight\n",
      "Description:\n",
      "This package perform weighted-pvalue based multiple hypothesis test and provides corresponding information such as ranking probability, weight, significant tests, etc . To conduct this testing procedure, the testing method apply a probabilistic relationship between the test rank and the corresponding test effect size.\n",
      "Version: 1.20.0\n",
      "Publication date: None\n",
      "Mantainer: Mohamad Hasan <shakilmohamad7@gmail.com>\n",
      "Authors:\n",
      "Author: Mohamad Hasan [aut, cre], Paul Schliekelman [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.4.0) type: depends\n",
      "  - graphics, type: imports\n",
      "  - qvalue, type: imports\n",
      "  - MASS, type: imports\n",
      "  - tibble, type: imports\n",
      "  - stats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/OPWeight.html\n",
      "\n",
      "Name: peco\n",
      "Description:\n",
      "Our approach provides a way to assign continuous cell cycle phase using scRNA-seq data, and consequently, allows to identify cyclic trend of gene expression levels along the cell cycle. This package provides method and training data, which includes scRNA-seq data collected from 6 individual cell lines of induced pluripotent stem cells (iPSCs), and also continuous cell cycle phase derived from FUCCI fluorescence imaging data.\n",
      "Version: 1.10.0\n",
      "Publication date: None\n",
      "Mantainer: Chiaowen Joyce Hsiao <joyce.hsiao1@gmail.com>\n",
      "Authors:\n",
      "Author: Chiaowen Joyce Hsiao [aut, cre], Matthew Stephens [aut], John Blischak [ctb], Peter Carbonetto [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - assertthat, type: imports\n",
      "  - circular, type: imports\n",
      "  - conicfit, type: imports\n",
      "  - doParallel, type: imports\n",
      "  - foreach, type: imports\n",
      "  - genlasso (>= 1.4) type: imports\n",
      "  - graphics, type: imports\n",
      "  - methods, type: imports\n",
      "  - parallel, type: imports\n",
      "  - scater, type: imports\n",
      "  - SingleCellExperiment, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 3)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/peco.html\n",
      "\n",
      "Name: pmm\n",
      "Description:\n",
      "The Parallel Mixed Model (PMM) approach is suitable for hit selection and cross-comparison of RNAi screens generated in experiments that are performed in parallel under several conditions. For example, we could think of the measurements or readouts from cells under RNAi knock-down, which are infected with several pathogens or which are grown from different cell lines.\n",
      "Version: 1.30.0\n",
      "Publication date: None\n",
      "Mantainer: Anna Drewek <adrewek@stat.math.ethz.ch>\n",
      "Authors:\n",
      "Author: Anna Drewek\n",
      "Dependencies:\n",
      "  - R (>= 2.10) type: depends\n",
      "  - lme4, type: imports\n",
      "  - splines, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/pmm.html\n",
      "\n",
      "Name: preciseTAD\n",
      "Description:\n",
      "preciseTAD provides functions to predict the location of boundaries of topologically associated domains (TADs) and chromatin loops at base-level resolution. As an input, it takes BED-formatted genomic coordinates of domain boundaries detected from low-resolution Hi-C data, and coordinates of high-resolution genomic annotations from ENCODE or other consortia. preciseTAD employs several feature engineering strategies and resampling techniques to address class imbalance, and trains an optimized random forest model for predicting low-resolution domain boundaries. Translated on a base-level, preciseTAD predicts the probability for each base to be a boundary. Density-based clustering and scalable partitioning techniques are used to detect precise boundary regions and summit points. Compared with low-resolution boundaries, preciseTAD boundaries are highly enriched for CTCF, RAD21, SMC3, and ZNF143 signal and more conserved across cell lines. The pre-trained model can accurately predict boundaries in another cell line using CTCF, RAD21, SMC3, and ZNF143 annotation data for this cell line.\n",
      "Version: 1.8.0\n",
      "Publication date: None\n",
      "Mantainer: Mikhail Dozmorov <mikhail.dozmorov@gmail.com>\n",
      "Authors:\n",
      "Author: Spiro Stilianoudakis [aut], Mikhail Dozmorov [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - S4Vectors, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - randomForest, type: imports\n",
      "  - ModelMetrics, type: imports\n",
      "  - e1071, type: imports\n",
      "  - PRROC, type: imports\n",
      "  - pROC, type: imports\n",
      "  - caret, type: imports\n",
      "  - utils, type: imports\n",
      "  - cluster, type: imports\n",
      "  - dbscan, type: imports\n",
      "  - doSNOW, type: imports\n",
      "  - foreach, type: imports\n",
      "  - pbapply, type: imports\n",
      "  - stats, type: imports\n",
      "  - parallel, type: imports\n",
      "  - gtools, type: imports\n",
      "  - rCGH, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/preciseTAD.html\n",
      "\n",
      "Name: RadioGx\n",
      "Description:\n",
      "Computational tool box for radio-genomic analysis which integrates radio-response data, radio-biological modelling and comprehensive cell line annotations for hundreds of cancer cell lines. The 'RadioSet' class enables creation and manipulation of standardized datasets including information about cancer cells lines, radio-response assays and dose-response indicators. Included methods allow fitting and plotting dose-response data using established radio-biological models along with quality control to validate results. Additional functions related to fitting and plotting dose response curves, quantifying statistical correlation and calculating area under the curve (AUC) or survival fraction (SF) are included. For more details please see the included documentation, references, as well as: Manem, V. et al (2018) .\n",
      "Version: 2.2.0\n",
      "Publication date: None\n",
      "Mantainer: Benjamin Haibe-Kains <benjamin.haibe.kains@utoronto.ca>\n",
      "Authors:\n",
      "Author: Venkata Manem [aut], Petr Smirnov [aut], Ian Smith [aut], Meghan Lambie [aut], Christopher Eeles [aut], Scott Bratman [aut], Benjamin Haibe-Kains [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - CoreGx, type: depends\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - data.table, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - parallel, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - caTools, type: imports\n",
      "  - magicaxis, type: imports\n",
      "  - methods, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - scales, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - assertthat, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - downloader, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/RadioGx.html\n",
      "\n",
      "Name: randRotation\n",
      "Description:\n",
      "A collection of methods for performing random rotations on high-dimensional, normally distributed data (e.g. microarray or RNA-seq data) with batch structure. The random rotation approach allows exact testing of dependent test statistics with linear models following arbitrary batch effect correction methods.\n",
      "Version: 1.10.0\n",
      "Publication date: None\n",
      "Mantainer: Peter Hettegger <p.hettegger@gmail.com>\n",
      "Authors:\n",
      "Author: Peter Hettegger [aut, cre]\n",
      "Dependencies:\n",
      "  - methods, type: imports\n",
      "  - graphics, type: imports\n",
      "  - utils, type: imports\n",
      "  - stats, type: imports\n",
      "  - Rdpack (>= 0.7) type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/randRotation.html\n",
      "\n",
      "Name: REBET\n",
      "Description:\n",
      "There is an increasing focus to investigate the association between rare variants and diseases. The REBET package implements the subREgion-based BurdEn Test which is a powerful burden test that simultaneously identifies susceptibility loci and sub-regions.\n",
      "Version: 1.16.0\n",
      "Publication date: None\n",
      "Mantainer: Bill Wheeler <wheelerb@imsweb.com>\n",
      "Authors:\n",
      "Author: Bill Wheeler [cre], Bin Zhu [aut], Lisa Mirabello [ctb], Nilanjan Chatterjee [ctb]\n",
      "Dependencies:\n",
      "  - ASSET, type: depends\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/REBET.html\n",
      "\n",
      "Name: ROSeq\n",
      "Description:\n",
      "ROSeq - A rank based approach to modeling gene expression with filtered and normalized read count matrix. ROSeq takes filtered and normalized read matrix and cell-annotation/condition as input and determines the differentially expressed genes between the contrasting groups of single cells. One of the input parameters is the number of cores to be used.\n",
      "Version: 1.10.0\n",
      "Publication date: None\n",
      "Mantainer: Krishan Gupta <krishang@iiitd.ac.in>\n",
      "Authors:\n",
      "Author: Krishan Gupta [aut, cre], Manan Lalit [aut], Aditya Biswas [aut], Abhik Ghosh [aut], Debarka Sengupta [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - pbmcapply, type: imports\n",
      "  - edgeR, type: imports\n",
      "  - limma, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ROSeq.html\n",
      "\n",
      "Name: Rtpca\n",
      "Description:\n",
      "R package for performing thermal proximity co-aggregation analysis with thermal proteome profiling datasets to analyse protein complex assembly and (differential) protein-protein interactions across conditions.\n",
      "Version: 1.8.0\n",
      "Publication date: None\n",
      "Mantainer: Nils Kurzawa <nilskurzawa@gmail.com>\n",
      "Authors:\n",
      "Author: Nils Kurzawa [aut, cre], AndrÃ© Mateus [aut], Mikhail M. Savitski [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.0.0) type: depends\n",
      "  - stats, type: depends\n",
      "  - dplyr, type: depends\n",
      "  - tidyr, type: depends\n",
      "  - Biobase, type: imports\n",
      "  - methods, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - pROC, type: imports\n",
      "  - fdrtool, type: imports\n",
      "  - splines, type: imports\n",
      "  - utils, type: imports\n",
      "  - tibble, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/Rtpca.html\n",
      "\n",
      "Name: scTHI\n",
      "Description:\n",
      "scTHI is an R package to identify active pairs of ligand-receptors from single cells in order to study,among others, tumor-host interactions. scTHI contains a set of signatures to classify cells from the tumor microenvironment.\n",
      "Version: 1.10.0\n",
      "Publication date: None\n",
      "Mantainer: Michele Ceccarelli <m.ceccarelli@gmail.com>\n",
      "Authors:\n",
      "Author: Francesca Pia Caruso [aut], Michele Ceccarelli [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - BiocParallel, type: imports\n",
      "  - Rtsne, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/scTHI.html\n",
      "\n",
      "Name: sitadela\n",
      "Description:\n",
      "Provides an interface to build a unified database of genomic annotations and their coordinates (gene, transcript and exon levels). It is aimed to be used when simple tab-delimited annotations (or simple GRanges objects) are required instead of the more complex annotation Bioconductor packages. Also useful when combinatorial annotation elements are reuired, such as RefSeq coordinates with Ensembl biotypes. Finally, it can download, construct and handle annotations with versioned genes and transcripts (where available, e.g. RefSeq and latest Ensembl). This is particularly useful in precision medicine applications where the latter must be reported.\n",
      "Version: 1.6.0\n",
      "Publication date: None\n",
      "Mantainer: Panagiotis Moulos <moulos@fleming.gr>\n",
      "Authors:\n",
      "Author: Panagiotis Moulos [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.1.0) type: depends\n",
      "  - Biobase, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - biomaRt, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - GenomicFeatures, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - methods, type: imports\n",
      "  - parallel, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - RSQLite, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - tools, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/sitadela.html\n",
      "\n",
      "Name: sojourner\n",
      "Description:\n",
      "Single molecule tracking has evolved as a novel new approach complementing genomic sequencing, it reports live biophysical properties of molecules being investigated besides properties relating their coding sequence; here we provided \"sojourner\" package, to address statistical and bioinformatic needs related to the analysis and comprehension of high throughput single molecule tracking data.\n",
      "Version: 1.11.0\n",
      "Publication date: None\n",
      "Mantainer: Sojourner Developer <sojourner.developer@outlook.com>\n",
      "Authors:\n",
      "Author: Sheng Liu [aut], Sun Jay Yoo [aut], Xiao Na Tang [aut], Young Soo Sung [aut], Carl Wu [aut], Anand Ranjan [ctb], Vu Nguyen [ctb], Sojourner Developer [cre]\n",
      "Dependencies:\n",
      "  - ggplot2, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - gridExtra, type: imports\n",
      "  - EBImage, type: imports\n",
      "  - MASS, type: imports\n",
      "  - R.matlab, type: imports\n",
      "  - Rcpp, type: imports\n",
      "  - fitdistrplus, type: imports\n",
      "  - mclust, type: imports\n",
      "  - minpack.lm, type: imports\n",
      "  - mixtools, type: imports\n",
      "  - mltools, type: imports\n",
      "  - nls2, type: imports\n",
      "  - plyr, type: imports\n",
      "  - sampSurf, type: imports\n",
      "  - scales, type: imports\n",
      "  - shiny, type: imports\n",
      "  - shinyjs, type: imports\n",
      "  - sp, type: imports\n",
      "  - truncnorm, type: imports\n",
      "  - utils, type: imports\n",
      "  - stats, type: imports\n",
      "  - pixmap, type: imports\n",
      "  - rlang, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - grid, type: imports\n",
      "  - compiler, type: imports\n",
      "  - lattice, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/sojourner.html\n",
      "\n",
      "Name: spatzie\n",
      "Description:\n",
      "Identifies motifs that are significantly co-enriched from enhancer-promoter interaction data. While enhancer-promoter annotation is commonly used to define groups of interaction anchors, spatzie also supports co-enrichment analysis between preprocessed interaction anchors. Supports BEDPE interaction data derived from genome-wide assays such as HiC, ChIA-PET, and HiChIP. Can also be used to look for differentially enriched motif pairs between two interaction experiments.\n",
      "Version: 1.4.0\n",
      "Publication date: None\n",
      "Mantainer: Jennifer Hammelman <jhammelm@mit.edu>\n",
      "Authors:\n",
      "Author: Jennifer Hammelman [aut, cre, cph] , Konstantin Krismer [aut] , David Gifford [ths, cph]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - BiocGenerics, type: imports\n",
      "  - BSgenome, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - GenomicFeatures, type: imports\n",
      "  - GenomicInteractions, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - motifmatchr, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - stats, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - TFBSTools, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/spatzie.html\n",
      "\n",
      "Name: SPLINTER\n",
      "Description:\n",
      "Provides tools to analyze alternative splicing sites, interpret outcomes based on sequence information, select and design primers for site validiation and give visual representation of the event to guide downstream experiments.\n",
      "Version: 1.24.0\n",
      "Publication date: None\n",
      "Mantainer: Diana Low <lowdiana@gmail.com>\n",
      "Authors:\n",
      "Author: Diana Low [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 3.6.0) type: depends\n",
      "  - grDevices, type: depends\n",
      "  - stats, type: depends\n",
      "  - graphics, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - seqLogo, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - biomaRt, type: imports\n",
      "  - GenomicAlignments, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - GenomicFeatures, type: imports\n",
      "  - Gviz, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - utils, type: imports\n",
      "  - plyr, type: imports\n",
      "  - stringr, type: imports\n",
      "  - methods, type: imports\n",
      "  - BSgenome.Mmusculus.UCSC.mm9, type: imports\n",
      "  - googleVis, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/SPLINTER.html\n",
      "\n",
      "Name: staRank\n",
      "Description:\n",
      "Detecting all relevant variables from a data set is challenging, especially when only few samples are available and data is noisy. Stability ranking provides improved variable rankings of increased robustness using resampling or subsampling.\n",
      "Version: 1.40.0\n",
      "Publication date: None\n",
      "Mantainer: Juliane Siebourg <juliane.siebourg@bsse.ethz.ch>\n",
      "Authors:\n",
      "Author: Juliane Siebourg, Niko Beerenwinkel\n",
      "Dependencies:\n",
      "  - methods, type: depends\n",
      "  - cellHTS2, type: depends\n",
      "  - R (>= 2.10) type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/staRank.html\n",
      "\n",
      "Name: synapter\n",
      "Description:\n",
      "The synapter package provides functionality to reanalyse label-free proteomics data acquired on a Synapt G2 mass spectrometer. One or several runs, possibly processed with additional ion mobility separation to increase identification accuracy can be combined to other quantitation files to maximise identification and quantitation accuracy.\n",
      "Version: 2.22.0\n",
      "Publication date: None\n",
      "Mantainer: Laurent Gatto <laurent.gatto@uclouvain.be> Sebastian Gibb <mail@sebastiangibb.de>\n",
      "Authors:\n",
      "Author: Laurent Gatto, Nick J. Bond, Pavel V. Shliaha and Sebastian Gibb.\n",
      "Dependencies:\n",
      "  - R (>= 3.1.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - MSnbase, type: depends\n",
      "  - RColorBrewer, type: imports\n",
      "  - lattice, type: imports\n",
      "  - qvalue, type: imports\n",
      "  - multtest, type: imports\n",
      "  - utils, type: imports\n",
      "  - tools, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - cleaver, type: imports\n",
      "  - readr (>= 0.2) type: imports\n",
      "  - rmarkdown (>= 1.0) type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/synapter.html\n",
      "\n",
      "Name: ternarynet\n",
      "Description:\n",
      "Gene-regulatory network (GRN) modeling seeks to infer dependencies between genes and thereby provide insight into the regulatory relationships that exist within a cell. This package provides a computational Bayesian approach to GRN estimation from perturbation experiments using a ternary network model, in which gene expression is discretized into one of 3 states: up, unchanged, or down). The ternarynet package includes a parallel implementation of the replica exchange Monte Carlo algorithm for fitting network models, using MPI.\n",
      "Version: 1.42.0\n",
      "Publication date: None\n",
      "Mantainer: McCall N. Matthew <mccallm@gmail.com>\n",
      "Authors:\n",
      "Author: Matthew N. McCall <mccallm at gmail.com>, Anthony Almudevar <Anthony_Alumudevar at urmc.rochester.edu>, David Burton <David_Burton at urmc.rochester.edu>, Harry Stern <harry.stern at rochester.edu>\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - utils, type: imports\n",
      "  - igraph, type: imports\n",
      "  - methods, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ternarynet.html\n",
      "\n",
      "Name: tLOH\n",
      "Description:\n",
      "tLOH, or transcriptomicsLOH, assesses evidence for loss of heterozygosity (LOH) in pre-processed spatial transcriptomics data. This tool requires spatial transcriptomics cluster and allele count information at likely heterozygous single-nucleotide polymorphism (SNP) positions in VCF format. Bayes factors are calculated at each SNP to determine likelihood of potential loss of heterozygosity event. Two plotting functions are included to visualize allele fraction and aggregated Bayes factor per chromosome. Data generated with the 10X Genomics Visium Spatial Gene Expression platform must be pre-processed to obtain an individual sample VCF with columns for each cluster. Required fields are allele depth (AD) with counts for reference/alternative alleles and read depth (DP).\n",
      "Version: 1.6.0\n",
      "Publication date: None\n",
      "Mantainer: Michelle Webb <michelgw@usc.edu>\n",
      "Authors:\n",
      "Author: Michelle Webb [cre, aut], David Craig [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.2) type: depends\n",
      "  - scales, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - data.table, type: imports\n",
      "  - purrr, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - VariantAnnotation, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - MatrixGenerics, type: imports\n",
      "  - bestNormalize, type: imports\n",
      "  - depmixS4, type: imports\n",
      "  - naniar, type: imports\n",
      "  - stringr, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/tLOH.html\n",
      "\n",
      "Name: ToxicoGx\n",
      "Description:\n",
      "Contains a set of functions to perform large-scale analysis of toxicogenomic data, providing a standardized data structure to hold information relevant to annotation, visualization and statistical analysis of toxicogenomic data.\n",
      "Version: 2.2.0\n",
      "Publication date: None\n",
      "Mantainer: Benjamin Haibe-Kains <benjamin.haibe.kains@utoronto.ca>\n",
      "Authors:\n",
      "Author: Sisira Nair [aut], Esther Yoo [aut], Christopher Eeles [aut], Amy Tang [aut], Nehme El-Hachem [aut], Petr Smirnov [aut], Benjamin Haibe-Kains [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - CoreGx, type: depends\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - tibble, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - caTools, type: imports\n",
      "  - downloader, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - methods, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - data.table, type: imports\n",
      "  - assertthat, type: imports\n",
      "  - scales, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - parallel, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - limma, type: imports\n",
      "  - jsonlite, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ToxicoGx.html\n",
      "\n",
      "Name: VAExprs\n",
      "Description:\n",
      "A fundamental problem in biomedical research is the low number of observations, mostly due to a lack of available biosamples, prohibitive costs, or ethical reasons. By augmenting a few real observations with artificially generated samples, their analysis could lead to more robust and higher reproducible. One possible solution to the problem is the use of generative models, which are statistical models of data that attempt to capture the entire probability distribution from the observations. Using the variational autoencoder (VAE), a well-known deep generative model, this package is aimed to generate samples with gene expression data, especially for single-cell RNA-seq data. Furthermore, the VAE can use conditioning to produce specific cell types or subpopulations. The conditional VAE (CVAE) allows us to create targeted samples rather than completely random ones.\n",
      "Version: 1.4.0\n",
      "Publication date: None\n",
      "Mantainer: Dongmin Jung <dmdmjung@gmail.com>\n",
      "Authors:\n",
      "Author: Dongmin Jung [cre, aut]\n",
      "Dependencies:\n",
      "  - keras, type: depends\n",
      "  - mclust, type: depends\n",
      "  - SingleCellExperiment, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - tensorflow, type: imports\n",
      "  - scater, type: imports\n",
      "  - CatEncoders, type: imports\n",
      "  - DeepPINCS, type: imports\n",
      "  - purrr, type: imports\n",
      "  - DiagrammeR, type: imports\n",
      "  - stats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/VAExprs.html\n",
      "\n",
      "Name: VarCon\n",
      "Description:\n",
      "VarCon is an R package which converts the positional information from the annotation of an single nucleotide variation (SNV) (either referring to the coding sequence or the reference genomic sequence). It retrieves the genomic reference sequence around the position of the single nucleotide variation. To asses, whether the SNV could potentially influence binding of splicing regulatory proteins VarCon calcualtes the HEXplorer score as an estimation. Besides, VarCon additionally reports splice site strengths of splice sites within the retrieved genomic sequence and any changes due to the SNV.\n",
      "Version: 1.6.0\n",
      "Publication date: None\n",
      "Mantainer: Johannes Ptok <Johannes.Ptok@posteo.de>\n",
      "Authors:\n",
      "Author: Johannes Ptok [aut, cre]\n",
      "Dependencies:\n",
      "  - Biostrings, type: depends\n",
      "  - BSgenome, type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - R (>= 4.1) type: depends\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - shiny, type: imports\n",
      "  - shinycssloaders, type: imports\n",
      "  - shinyFiles, type: imports\n",
      "  - ggplot2, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/VarCon.html\n",
      "\n",
      "Name: weitrix\n",
      "Description:\n",
      "Data type and tools for working with matrices having precision weights and missing data. This package provides a common representation and tools that can be used with many types of high-throughput data. The meaning of the weights is compatible with usage in the base R function \"lm\" and the package \"limma\". Calibrate weights to account for known predictors of precision. Find rows with excess variability. Perform differential testing and find rows with the largest confident differences. Find PCA-like components of variation even with many missing values, rotated so that individual components may be meaningfully interpreted. DelayedArray matrices and BiocParallel are supported.\n",
      "Version: 1.10.0\n",
      "Publication date: None\n",
      "Mantainer: Paul Harrison <paul.harrison@monash.edu>\n",
      "Authors:\n",
      "Author: Paul Harrison [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 3.6) type: depends\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - methods, type: imports\n",
      "  - utils, type: imports\n",
      "  - stats, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - assertthat, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - DelayedArray, type: imports\n",
      "  - DelayedMatrixStats, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - limma, type: imports\n",
      "  - topconfects, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - purrr, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - rlang, type: imports\n",
      "  - scales, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - splines, type: imports\n",
      "  - Ckmeans.1d.dp, type: imports\n",
      "  - glm2, type: imports\n",
      "  - RhpcBLASctl, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL-2.1 | file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/weitrix.html\n",
      "\n",
      "Name: CAEN\n",
      "Description:\n",
      "With the development of high-throughput techniques, more and more gene expression analysis tend to replace hybridization-based microarrays with the revolutionary technology.The novel method encodes the category again by employing the rank of samples for each gene in each class. We then consider the correlation coefficient of gene and class with rank of sample and new rank of category. The highest correlation coefficient genes are considered as the feature genes which are most effective to classify the samples.\n",
      "Version: 1.6.0\n",
      "Publication date: None\n",
      "Mantainer: Zhou Yan <2160090406@email.szu.edu.cn>\n",
      "Authors:\n",
      "Author: Zhou Yan [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - stats, type: imports\n",
      "  - PoiClaClu, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - methods, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/CAEN.html\n",
      "\n",
      "Name: ceRNAnetsim\n",
      "Description:\n",
      "This package simulates regulations of ceRNA (Competing Endogenous) expression levels after a expression level change in one or more miRNA/mRNAs. The methodolgy adopted by the package has potential to incorparate any ceRNA (circRNA, lincRNA, etc.) into miRNA:target interaction network. The package basically distributes miRNA expression over available ceRNAs where each ceRNA attracks miRNAs proportional to its amount. But, the package can utilize multiple parameters that modify miRNA effect on its target (seed type, binding energy, binding location, etc.). The functions handle the given dataset as graph object and the processes progress via edge and node variables.\n",
      "Version: 1.10.0\n",
      "Publication date: None\n",
      "Mantainer: Selcen Ari Yuka <selcenarii@gmail.com>\n",
      "Authors:\n",
      "Author: Selcen Ari Yuka [aut, cre] , Alper Yilmaz [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.0.0) type: depends\n",
      "  - dplyr, type: depends\n",
      "  - tidygraph, type: depends\n",
      "  - furrr, type: imports\n",
      "  - rlang, type: imports\n",
      "  - tibble, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - ggraph, type: imports\n",
      "  - igraph, type: imports\n",
      "  - purrr, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - future, type: imports\n",
      "  - stats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 3.0)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ceRNAnetsim.html\n",
      "\n",
      "Name: Cogito\n",
      "Description:\n",
      "Biological studies often consist of multiple conditions which are examined with different laboratory set ups like RNA-sequencing or ChIP-sequencing. To get an overview about the whole resulting data set, Cogito provides an automated, complete, reproducible and clear report about all samples and basic comparisons between all different samples. This report can be used as documentation about the data set or as starting point for further custom analysis.\n",
      "Version: 1.4.0\n",
      "Publication date: None\n",
      "Mantainer: Annika BÃ¼rger <a.buerger@uni-muenster.de>\n",
      "Authors:\n",
      "Author: Annika BÃ¼rger [cre, aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - jsonlite, type: depends\n",
      "  - GenomicFeatures, type: depends\n",
      "  - entropy, type: depends\n",
      "  - BiocManager, type: imports\n",
      "  - rmarkdown, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - AnnotationDbi, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - methods, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - TxDb.Mmusculus.UCSC.mm9.knownGene, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/Cogito.html\n",
      "\n",
      "Name: deltaCaptureC\n",
      "Description:\n",
      "This package discovers meso-scale chromatin remodelling from 3C data. 3C data is local in nature. It givens interaction counts between restriction enzyme digestion fragments and a preferred 'viewpoint' region. By binning this data and using permutation testing, this package can test whether there are statistically significant changes in the interaction counts between the data from two cell types or two treatments.\n",
      "Version: 1.12.0\n",
      "Publication date: None\n",
      "Mantainer: Michael Shapiro <michael.shapiro@crick.ac.uk>\n",
      "Authors:\n",
      "Author: Michael Shapiro [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 3.6) type: depends\n",
      "  - IRanges, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - DESeq2, type: imports\n",
      "  - tictoc, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/deltaCaptureC.html\n",
      "\n",
      "Name: DMRScan\n",
      "Description:\n",
      "This package detects significant differentially methylated regions (for both qualitative and quantitative traits), using a scan statistic with underlying Poisson heuristics. The scan statistic will depend on a sequence of window sizes (# of CpGs within each window) and on a threshold for each window size. This threshold can be calculated by three different means: i) analytically using Siegmund et.al (2012) solution (preferred), ii) an important sampling as suggested by Zhang (2008), and a iii) full MCMC modeling of the data, choosing between a number of different options for modeling the dependency between each CpG.\n",
      "Version: 1.20.0\n",
      "Publication date: None\n",
      "Mantainer: Christian M Page <page.ntnu@gmail.com>\n",
      "Authors:\n",
      "Author: Christian M Page [aut, cre], Linda Vos [aut], Trine B Rounge [ctb, dtc], Hanne F Harbo [ths], Bettina K Andreassen [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.6.0) type: depends\n",
      "  - Matrix, type: imports\n",
      "  - MASS, type: imports\n",
      "  - RcppRoll, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - methods, type: imports\n",
      "  - mvtnorm, type: imports\n",
      "  - stats, type: imports\n",
      "  - parallel, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/DMRScan.html\n",
      "\n",
      "Name: EnMCB\n",
      "Description:\n",
      "Creation of the correlated blocks using DNA methylation profiles. Machine learning models can be constructed to predict differentially methylated blocks and disease progression.\n",
      "Version: 1.10.0\n",
      "Publication date: None\n",
      "Mantainer: Xin Yu <whirlsyu@gmail.com>\n",
      "Authors:\n",
      "Author: Xin Yu\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - survivalROC, type: imports\n",
      "  - glmnet, type: imports\n",
      "  - rms, type: imports\n",
      "  - mboost, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - igraph, type: imports\n",
      "  - survivalsvm, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - BiocFileCache, type: imports\n",
      "  - boot, type: imports\n",
      "  - e1071, type: imports\n",
      "  - survival, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/EnMCB.html\n",
      "\n",
      "Name: fedup\n",
      "Description:\n",
      "An R package that tests for enrichment and depletion of user-defined pathways using a Fisher's exact test. The method is designed for versatile pathway annotation formats (eg. gmt, txt, xlsx) to allow the user to run pathway analysis on custom annotations. This package is also integrated with Cytoscape to provide network-based pathway visualization that enhances the interpretability of the results.\n",
      "Version: 1.6.0\n",
      "Publication date: None\n",
      "Mantainer: Catherine Ross <catherinem.ross@mail.utoronto.ca>\n",
      "Authors:\n",
      "Author: Catherine Ross [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - openxlsx, type: imports\n",
      "  - tibble, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - data.table, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - ggthemes, type: imports\n",
      "  - forcats, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - RCy3, type: imports\n",
      "  - utils, type: imports\n",
      "  - stats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/fedup.html\n",
      "\n",
      "Name: GBScleanR\n",
      "Description:\n",
      "GBScleanR is a package for quality check, filtering, and error correction of genotype data derived from next generation sequcener (NGS) based genotyping platforms. GBScleanR takes Variant Call Format (VCF) file as input. The main function of this package is `estGeno()` which estimates the true genotypes of samples from given read counts for genotype markers using a hidden Markov model with incorporating uneven observation ratio of allelic reads. This implementation gives robust genotype estimation even in noisy genotype data usually observed in Genotyping-By-Sequnencing (GBS) and similar methods, e.g. RADseq. The current implementation accepts genotype data of a diploid population at any generation of multi-parental cross, e.g. biparental F2 from inbred parents, biparental F2 from outbred parents, and 8-way recombinant inbred lines (8-way RILs) which can be refered to as MAGIC population.\n",
      "Version: 1.2.9\n",
      "Publication date: None\n",
      "Mantainer: Tomoyuki Furuta <f.tomoyuki@okayama-u.ac.jp>\n",
      "Authors:\n",
      "Author: Tomoyuki Furuta [aut, cre]\n",
      "Dependencies:\n",
      "  - SeqArray, type: depends\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - methods, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - expm, type: imports\n",
      "  - Rcpp, type: imports\n",
      "  - RcppParallel, type: imports\n",
      "  - gdsfmt, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3 + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/GBScleanR.html\n",
      "\n",
      "Name: GeneAccord\n",
      "Description:\n",
      "A statistical framework to examine the combinations of clones that co-exist in tumors. More precisely, the algorithm finds pairs of genes that are mutated in the same tumor but in different clones, i.e. their subclonal mutation profiles are mutually exclusive. We refer to this as clonally exclusive. It means that the mutations occurred in different branches of the tumor phylogeny, indicating parallel evolution of the clones. Our statistical framework assesses whether a pattern of clonal exclusivity occurs more often than expected by chance alone across a cohort of patients. The required input data are the mutated gene-to-clone assignments from a cohort of cancer patients, which were obtained by running phylogenetic tree inference methods. Reconstructing the evolutionary history of a tumor and detecting the clones is challenging. For nondeterministic algorithms, repeated tree inference runs may lead to slightly different mutation-to-clone assignments. Therefore, our algorithm was designed to allow the input of multiple gene-to-clone assignments per patient. They may have been generated by repeatedly performing the tree inference, or by sampling from the posterior distribution of trees. The tree inference methods designate the mutations to individual clones. The mutations can then be mapped to genes or pathways. Hence our statistical framework can be applied on the gene level, or on the pathway level to detect clonally exclusive pairs of pathways. If a pair is significantly clonally exclusive, it points towards the fact that this specific clone configuration confers a selective advantage, possibly through synergies between the clones with these mutations.\n",
      "Version: 1.15.0\n",
      "Publication date: None\n",
      "Mantainer: Ariane L. Moore <ariane.moore@bsse.ethz.ch>\n",
      "Authors:\n",
      "Author: Ariane L. Moore, Jack Kuipers and Niko Beerenwinkel\n",
      "Dependencies:\n",
      "  - R (>= 3.5) type: depends\n",
      "  - biomaRt, type: imports\n",
      "  - caTools, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - gtools, type: imports\n",
      "  - ggpubr, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - maxLik, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - stats, type: imports\n",
      "  - tibble, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/GeneAccord.html\n",
      "\n",
      "Name: GPA\n",
      "Description:\n",
      "This package provides functions for fitting GPA, a statistical framework to prioritize GWAS results by integrating pleiotropy information and annotation data. In addition, it also includes ShinyGPA, an interactive visualization toolkit to investigate pleiotropic architecture.\n",
      "Version: 1.10.0\n",
      "Publication date: None\n",
      "Mantainer: Dongjun Chung <dongjun.chung@gmail.com>\n",
      "Authors:\n",
      "Author: Dongjun Chung, Emma Kortemeier, Carter Allen\n",
      "Dependencies:\n",
      "  - R (>= 4.0.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - graphics, type: depends\n",
      "  - Rcpp, type: depends\n",
      "  - parallel, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - ggrepel, type: imports\n",
      "  - plyr, type: imports\n",
      "  - vegan, type: imports\n",
      "  - DT, type: imports\n",
      "  - shiny, type: imports\n",
      "  - shinyBS, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - grDevices, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/GPA.html\n",
      "\n",
      "Name: iCheck\n",
      "Description:\n",
      "QC pipeline and data analysis tools for high-dimensional Illumina mRNA expression data.\n",
      "Version: 1.28.0\n",
      "Publication date: None\n",
      "Mantainer: Weiliang Qiu <stwxq@channing.harvard.edu>\n",
      "Authors:\n",
      "Author: Weiliang Qiu [aut, cre], Brandon Guo [aut, ctb], Christopher Anderson [aut, ctb], Barbara Klanderman [aut, ctb], Vincent Carey [aut, ctb], Benjamin Raby [aut, ctb]\n",
      "Dependencies:\n",
      "  - R (>= 3.2.0) type: depends\n",
      "  - Biobase, type: depends\n",
      "  - lumi, type: depends\n",
      "  - gplots, type: depends\n",
      "  - stats, type: imports\n",
      "  - graphics, type: imports\n",
      "  - preprocessCore, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - randomForest, type: imports\n",
      "  - affy, type: imports\n",
      "  - limma, type: imports\n",
      "  - parallel, type: imports\n",
      "  - GeneSelectMMD, type: imports\n",
      "  - rgl, type: imports\n",
      "  - MASS, type: imports\n",
      "  - lmtest, type: imports\n",
      "  - scatterplot3d, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/iCheck.html\n",
      "\n",
      "Name: INDEED\n",
      "Description:\n",
      "An R package for integrated differential expression and differential network analysis based on omic data for cancer biomarker discovery. Both correlation and partial correlation can be used to generate differential network to aid the traditional differential expression analysis to identify changes between biomolecules on both their expression and pairwise association levels. A detailed description of the methodology has been published in Methods journal (PMID: 27592383). An interactive visualization feature allows for the exploration and selection of candidate biomarkers.\n",
      "Version: 2.12.0\n",
      "Publication date: None\n",
      "Mantainer: Ressom group <hwr@georgetown.edu>, Yiming Zuo <yimingzuo@gmail.com>\n",
      "Authors:\n",
      "Author: Yiming Zuo <yimingzuo at gmail.com>, Kian Ghaffari <kg.ghaffari at gmail.com>, Zhenzhi Li <zzrickli at gmail.com>\n",
      "Dependencies:\n",
      "  - glasso (>= 1.8) type: depends\n",
      "  - R (>= 3.5) type: depends\n",
      "  - devtools (>= 1.13.0) type: imports\n",
      "  - graphics (>= 3.3.1) type: imports\n",
      "  - stats (>= 3.3.1) type: imports\n",
      "  - utils (>= 3.3.1) type: imports\n",
      "  - igraph (>= 1.2.4) type: imports\n",
      "  - visNetwork (>= 2.0.6) type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/INDEED.html\n",
      "\n",
      "Name: LedPred\n",
      "Description:\n",
      "This package aims at creating a predictive model of regulatory sequences used to score unknown sequences based on the content of DNA motifs, next-generation sequencing (NGS) peaks and signals and other numerical scores of the sequences using supervised classification. The package contains a workflow based on the support vector machine (SVM) algorithm that maps features to sequences, optimize SVM parameters and feature number and creates a model that can be stored and used to score the regulatory potential of unknown sequences.\n",
      "Version: 1.32.0\n",
      "Publication date: None\n",
      "Mantainer: Aitor Gonzalez <aitor.gonzalez@univ-amu.fr>\n",
      "Authors:\n",
      "Author: Elodie Darbo, Denis Seyres, Aitor Gonzalez\n",
      "Dependencies:\n",
      "  - R (>= 3.2.0) type: depends\n",
      "  - e1071 (>= 1.6) type: depends\n",
      "  - akima, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - irr, type: imports\n",
      "  - jsonlite, type: imports\n",
      "  - parallel, type: imports\n",
      "  - plot3D, type: imports\n",
      "  - plyr, type: imports\n",
      "  - RCurl, type: imports\n",
      "  - ROCR, type: imports\n",
      "  - testthat, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT | file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/LedPred.html\n",
      "\n",
      "Name: marr\n",
      "Description:\n",
      "marr (Maximum Rank Reproducibility) is a nonparametric approach that detects reproducible signals using a maximal rank statistic for high-dimensional biological data. In this R package, we implement functions that measures the reproducibility of features per sample pair and sample pairs per feature in high-dimensional biological replicate experiments. The user-friendly plot functions in this package also plot histograms of the reproducibility of features per sample pair and sample pairs per feature. Furthermore, our approach also allows the users to select optimal filtering threshold values for the identification of reproducible features and sample pairs based on output visualization checks (histograms). This package also provides the subset of data filtered by reproducible features and/or sample pairs.\n",
      "Version: 1.8.0\n",
      "Publication date: None\n",
      "Mantainer: Tusharkanti Ghosh <tusharkantighosh30@gmail.com>\n",
      "Authors:\n",
      "Author: Tusharkanti Ghosh [aut, cre], Max McGrath [aut], Daisy Philtron [aut], Katerina Kechris [aut], Debashis Ghosh [aut, cph]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - Rcpp, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - utils, type: imports\n",
      "  - methods, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - rlang, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 3)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/marr.html\n",
      "\n",
      "Name: MBttest\n",
      "Description:\n",
      "MBttest method was developed from beta t-test method of Baggerly et al(2003). Compared to baySeq (Hard castle and Kelly 2010), DESeq (Anders and Huber 2010) and exact test (Robinson and Smyth 2007, 2008) and the GLM of McCarthy et al(2012), MBttest is of high work efficiency,that is, it has high power, high conservativeness of FDR estimation and high stability. MBttest is suit- able to transcriptomic data, tag data, SAGE data (count data) from small samples or a few replicate libraries. It can be used to identify genes, mRNA isoforms or tags differentially expressed between two conditions.\n",
      "Version: 1.26.0\n",
      "Publication date: None\n",
      "Mantainer: Yuan-De Tan <tanyuande@gmail.com>\n",
      "Authors:\n",
      "Author: Yuan-De Tan\n",
      "Dependencies:\n",
      "  - R (>= 3.3.0) type: depends\n",
      "  - stats, type: depends\n",
      "  - gplots, type: depends\n",
      "  - gtools, type: depends\n",
      "  - graphics, type: depends\n",
      "  - base, type: depends\n",
      "  - utils, type: depends\n",
      "  - grDevices, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/MBttest.html\n",
      "\n",
      "Name: MEB\n",
      "Description:\n",
      "Identifying differentially expressed genes between the same or different species is an urgent demand for biological and medical research. For RNA-seq data, systematic technical effects and different sequencing depths are usually encountered when conducting experiments. Normalization is regarded as an essential step in the discovery of biologically important changes in expression. The present methods usually involve normalization of the data with a scaling factor, followed by detection of significant genes. However, more than one scaling factor may exist because of the complexity of real data. Consequently, methods that normalize data by a single scaling factor may deliver suboptimal performance or may not even work. The development of modern machine learning techniques has provided a new perspective regarding discrimination between differentially expressed (DE) and non-DE genes. However, in reality, the non-DE genes comprise only a small set and may contain housekeeping genes (in same species) or conserved orthologous genes (in different species). Therefore, the process of detecting DE genes can be formulated as a one-class classification problem, where only non-DE genes are observed, while DE genes are completely absent from the training data. We transform the problem to an outlier detection problem by treating DE genes as outliers, and we propose a normalization-invariant minimum enclosing ball (NIMEB) method to construct a smallest possible ball to contain the known non-DE genes in a feature space. The genes outside the minimum enclosing ball can then be naturally considered to be DE genes. Compared with the existing methods, the proposed NIMEB method does not require data normalization, which is particularly attractive when the RNA-seq data include more than one scaling factor. Furthermore, the NIMEB method could be easily extended to different species without normalization.\n",
      "Version: 1.12.0\n",
      "Publication date: None\n",
      "Mantainer: Jiadi Zhu <2160090406@email.szu.edu.cn>, Yan Zhou <zhouy1016@szu.edu.cn>\n",
      "Authors:\n",
      "Author: Yan Zhou, Jiadi Zhu\n",
      "Dependencies:\n",
      "  - R (>= 3.6.0) type: depends\n",
      "  - e1071, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/MEB.html\n",
      "\n",
      "Name: MethTargetedNGS\n",
      "Description:\n",
      "Perform step by step methylation analysis of Next Generation Sequencing data.\n",
      "Version: 1.30.0\n",
      "Publication date: None\n",
      "Mantainer: Muhammad Ahmer Jamil <engr.ahmerjamil@gmail.com>\n",
      "Authors:\n",
      "Author: Muhammad Ahmer Jamil with Contribution of Prof. Holger Frohlich and Priv.-Doz. Dr. Osman El-Maarri\n",
      "Dependencies:\n",
      "  - R (>= 3.1.2) type: depends\n",
      "  - stringr, type: depends\n",
      "  - seqinr, type: depends\n",
      "  - gplots, type: depends\n",
      "  - Biostrings, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/MethTargetedNGS.html\n",
      "\n",
      "Name: miRcomp\n",
      "Description:\n",
      "Based on a large miRNA dilution study, this package provides tools to read in the raw amplification data and use these data to assess the performance of methods that estimate expression from the amplification curves.\n",
      "Version: 1.28.0\n",
      "Publication date: None\n",
      "Mantainer: Matthew N. McCall <mccallm@gmail.com>\n",
      "Authors:\n",
      "Author: Matthew N. McCall <mccallm at gmail.com>, Lauren Kemperman <lkemperm at u.rochester.edu>\n",
      "Dependencies:\n",
      "  - R (>= 3.2) type: depends\n",
      "  - Biobase, type: depends\n",
      "  - miRcompData, type: depends\n",
      "  - utils, type: imports\n",
      "  - methods, type: imports\n",
      "  - graphics, type: imports\n",
      "  - KernSmooth, type: imports\n",
      "  - stats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3 | file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/miRcomp.html\n",
      "\n",
      "Name: MouseFM\n",
      "Description:\n",
      "This package provides methods for genetic finemapping in inbred mice by taking advantage of their very high homozygosity rate (>95%).\n",
      "Version: 1.8.0\n",
      "Publication date: None\n",
      "Mantainer: Matthias Munz <matthias.munz@gmx.de>\n",
      "Authors:\n",
      "Author: Matthias Munz [aut, cre] , Inken Wohlers [aut] , Hauke Busch [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.0.0) type: depends\n",
      "  - httr, type: imports\n",
      "  - curl, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - scales, type: imports\n",
      "  - gtools, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - data.table, type: imports\n",
      "  - jsonlite, type: imports\n",
      "  - rlist, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - methods, type: imports\n",
      "  - biomaRt, type: imports\n",
      "  - stats, type: imports\n",
      "  - IRanges, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/MouseFM.html\n",
      "\n",
      "Name: NeighborNet\n",
      "Description:\n",
      "Identify the putative mechanism explaining the active interactions between genes in the investigated phenotype.\n",
      "Version: 1.16.0\n",
      "Publication date: None\n",
      "Mantainer: Sahar Ansari <saharansari@wayne.edu>\n",
      "Authors:\n",
      "Author: Sahar Ansari <saharansari at wayne.edu> and Sorin Draghici <sorin at wayne.edu>\n",
      "Dependencies:\n",
      "  - methods, type: depends\n",
      "  - graph, type: imports\n",
      "  - stats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "CC BY-NC-ND 4.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/NeighborNet.html\n",
      "\n",
      "Name: occugene\n",
      "Description:\n",
      "Statistical tools for building random mutagenesis libraries for prokaryotes. The package has functions for handling the occupancy distribution for a multinomial and for estimating the number of essential genes in random transposon mutagenesis libraries.\n",
      "Version: 1.58.0\n",
      "Publication date: None\n",
      "Mantainer: Oliver Will <oliverrreader@gmail.com>\n",
      "Authors:\n",
      "Author: Oliver Will <oliverrreader at gmail.com>\n",
      "Dependencies:\n",
      "  - R (>= 2.0.0) type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/occugene.html\n",
      "\n",
      "Name: oppti\n",
      "Description:\n",
      "The aim of oppti is to analyze protein (and phosphosite) expressions to find outlying markers for each sample in the given cohort(s) for the discovery of personalized actionable targets.\n",
      "Version: 1.12.0\n",
      "Publication date: None\n",
      "Mantainer: Abdulkadir Elmas <abdulkadir.elmas@mssm.edu>\n",
      "Authors:\n",
      "Author: Abdulkadir Elmas\n",
      "Dependencies:\n",
      "  - R (>= 3.5) type: depends\n",
      "  - limma, type: imports\n",
      "  - stats, type: imports\n",
      "  - reshape, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - pheatmap, type: imports\n",
      "  - knitr, type: imports\n",
      "  - methods, type: imports\n",
      "  - devtools, type: imports\n",
      "  - parallelDist, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/oppti.html\n",
      "\n",
      "Name: optimalFlow\n",
      "Description:\n",
      "Optimal-transport techniques applied to supervised flow cytometry gating.\n",
      "Version: 1.10.0\n",
      "Publication date: None\n",
      "Mantainer: Hristo Inouzhe <hristo.inouzhe@gmail.com>\n",
      "Authors:\n",
      "Author: Hristo Inouzhe <hristo.inouzhe at gmail.com>\n",
      "Dependencies:\n",
      "  - dplyr, type: depends\n",
      "  - optimalFlowData, type: depends\n",
      "  - rlang (>= 0.4.0) type: depends\n",
      "  - transport, type: imports\n",
      "  - parallel, type: imports\n",
      "  - Rfast, type: imports\n",
      "  - robustbase, type: imports\n",
      "  - dbscan, type: imports\n",
      "  - randomForest, type: imports\n",
      "  - foreach, type: imports\n",
      "  - graphics, type: imports\n",
      "  - doParallel, type: imports\n",
      "  - stats, type: imports\n",
      "  - flowMeans, type: imports\n",
      "  - rgl, type: imports\n",
      "  - ellipse, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/optimalFlow.html\n",
      "\n",
      "Name: PFP\n",
      "Description:\n",
      "An implementation of the pathway fingerprint framework that introduced in paper \"Pathway Fingerprint: a novel pathway knowledge and topology based method for biomarker discovery and characterization\". This method provides a systematic comparisons between a gene set (such as a list of differentially expressed genes) and well-studied \"basic pathway networks\" (KEGG pathways), measuring the importance of pathways and genes for the gene set. The package is helpful for researchers to find the biomarkers and its function.\n",
      "Version: 1.6.0\n",
      "Publication date: None\n",
      "Mantainer: XC Zhang <kunghero@163.com>\n",
      "Authors:\n",
      "Author: XC Zhang [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - graph, type: imports\n",
      "  - igraph, type: imports\n",
      "  - KEGGgraph, type: imports\n",
      "  - clusterProfiler, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - plyr, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - stats, type: imports\n",
      "  - methods, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/PFP.html\n",
      "\n",
      "Name: ppcseq\n",
      "Description:\n",
      "Relative transcript abundance has proven to be a valuable tool for understanding the function of genes in biological systems. For the differential analysis of transcript abundance using RNA sequencing data, the negative binomial model is by far the most frequently adopted. However, common methods that are based on a negative binomial model are not robust to extreme outliers, which we found to be abundant in public datasets. So far, no rigorous and probabilistic methods for detection of outliers have been developed for RNA sequencing data, leaving the identification mostly to visual inspection. Recent advances in Bayesian computation allow large-scale comparison of observed data against its theoretical distribution given in a statistical model. Here we propose ppcseq, a key quality-control tool for identifying transcripts that include outlier data points in differential expression analysis, which do not follow a negative binomial distribution. Applying ppcseq to analyse several publicly available datasets using popular tools, we show that from 3 to 10 percent of differentially abundant transcripts across algorithms and datasets had statistics inflated by the presence of outliers.\n",
      "Version: 1.6.0\n",
      "Publication date: None\n",
      "Mantainer: Stefano Mangiola <mangiolastefano@gmail.com>\n",
      "Authors:\n",
      "Author: Stefano Mangiola [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.1.0) type: depends\n",
      "  - methods, type: imports\n",
      "  - Rcpp (>= 0.12.0) type: imports\n",
      "  - rstan (>= 2.18.1) type: imports\n",
      "  - rstantools (>= 2.0.0) type: imports\n",
      "  - tibble, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - purrr, type: imports\n",
      "  - future, type: imports\n",
      "  - furrr, type: imports\n",
      "  - tidyr (>= 0.8.3.9000) type: imports\n",
      "  - lifecycle, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - foreach, type: imports\n",
      "  - tidybayes, type: imports\n",
      "  - edgeR, type: imports\n",
      "  - benchmarkme, type: imports\n",
      "  - parallel, type: imports\n",
      "  - rlang, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - graphics, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ppcseq.html\n",
      "\n",
      "Name: profileScoreDist\n",
      "Description:\n",
      "Regularization and score distributions for position count matrices.\n",
      "Version: 1.26.0\n",
      "Publication date: None\n",
      "Mantainer: Paal O. Westermark <pal-olof.westermark@charite.de>\n",
      "Authors:\n",
      "Author: Paal O. Westermark\n",
      "Dependencies:\n",
      "  - R (>= 3.3) type: depends\n",
      "  - Rcpp, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - methods, type: imports\n",
      "  - graphics, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/profileScoreDist.html\n",
      "\n",
      "Name: proteinProfiles\n",
      "Description:\n",
      "Significance assessment for distance measures of time-course protein profiles\n",
      "Version: 1.38.0\n",
      "Publication date: None\n",
      "Mantainer: Julian Gehring <jg-bioc@gmx.com>\n",
      "Authors:\n",
      "Author: Julian Gehring\n",
      "Dependencies:\n",
      "  - R (>= 2.15.2) type: depends\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/proteinProfiles.html\n",
      "\n",
      "Name: RCASPAR\n",
      "Description:\n",
      "The package is the R-version of the C-based software \\bold{CASPAR} (Kaderali,2006: \\url{http://bioinformatics.oxfordjournals.org/content/22/12/1495}). It is meant to help predict survival times in the presence of high-dimensional explanatory covariates. The model is a piecewise baseline hazard Cox regression model with an Lq-norm based prior that selects for the most important regression coefficients, and in turn the most relevant covariates for survival analysis. It was primarily tried on gene expression and aCGH data, but can be used on any other type of high-dimensional data and in disciplines other than biology and medicine.\n",
      "Version: 1.44.0\n",
      "Publication date: None\n",
      "Mantainer: Douaa Mugahid <douaa.mugahid@gmail.com>, Lars Kaderali <lars.kaderali@uni-greifswald.de>\n",
      "Authors:\n",
      "Author: Douaa Mugahid, Lars Kaderali\n",
      "Dependencies:\n",
      "None\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>=3)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/RCASPAR.html\n",
      "\n",
      "Name: RcwlPipelines\n",
      "Description:\n",
      "A collection of Bioinformatics tools and pipelines based on R and the Common Workflow Language.\n",
      "Version: 1.14.0\n",
      "Publication date: None\n",
      "Mantainer: Qiang Hu <qiang.hu@roswellpark.org>\n",
      "Authors:\n",
      "Author: Qiang Hu [aut, cre], Qian Liu [aut], Shuang Gao [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.6) type: depends\n",
      "  - Rcwl, type: depends\n",
      "  - BiocFileCache, type: depends\n",
      "  - rappdirs, type: imports\n",
      "  - methods, type: imports\n",
      "  - utils, type: imports\n",
      "  - git2r, type: imports\n",
      "  - httr, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/RcwlPipelines.html\n",
      "\n",
      "Name: reconsi\n",
      "Description:\n",
      "Improves simultaneous inference under dependence of tests by estimating a collapsed null distribution through resampling. Accounting for the dependence between tests increases the power while reducing the variability of the false discovery proportion. This dependence is common in genomics applications, e.g. when combining flow cytometry measurements with microbiome sequence counts.\n",
      "Version: 1.10.0\n",
      "Publication date: None\n",
      "Mantainer: Stijn Hawinkel <stijn.hawinkel@psb.ugent.be>\n",
      "Authors:\n",
      "Author: Stijn Hawinkel [cre, aut]\n",
      "Dependencies:\n",
      "  - phyloseq, type: imports\n",
      "  - ks, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - stats, type: imports\n",
      "  - methods, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - Matrix, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/reconsi.html\n",
      "\n",
      "Name: regsplice\n",
      "Description:\n",
      "Statistical methods for detection of differential splicing (differential exon usage) in RNA-seq and exon microarray data, using L1-regularization (lasso) to improve power.\n",
      "Version: 1.24.0\n",
      "Publication date: None\n",
      "Mantainer: Lukas M. Weber <lukas.weber.edu@gmail.com>\n",
      "Authors:\n",
      "Author: Lukas M. Weber [aut, cre]\n",
      "Dependencies:\n",
      "  - glmnet, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - limma, type: imports\n",
      "  - edgeR, type: imports\n",
      "  - stats, type: imports\n",
      "  - pbapply, type: imports\n",
      "  - utils, type: imports\n",
      "  - methods, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/regsplice.html\n",
      "\n",
      "Name: RIVER\n",
      "Description:\n",
      "An implementation of a probabilistic modeling framework that jointly analyzes personal genome and transcriptome data to estimate the probability that a variant has regulatory impact in that individual. It is based on a generative model that assumes that genomic annotations, such as the location of a variant with respect to regulatory elements, determine the prior probability that variant is a functional regulatory variant, which is an unobserved variable. The functional regulatory variant status then influences whether nearby genes are likely to display outlier levels of gene expression in that person. See the RIVER website for more information, documentation and examples.\n",
      "Version: 1.22.0\n",
      "Publication date: None\n",
      "Mantainer: Yungil Kim <ipw012@gmail.com>\n",
      "Authors:\n",
      "Author: Yungil Kim [aut, cre], Alexis Battle [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.3.2) type: depends\n",
      "  - glmnet, type: imports\n",
      "  - pROC, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - methods, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/RIVER.html\n",
      "\n",
      "Name: rnaEditr\n",
      "Description:\n",
      "RNAeditr analyzes site-specific RNA editing events, as well as hyper-editing regions. The editing frequencies can be tested against binary, continuous or survival outcomes. Multiple covariate variables as well as interaction effects can also be incorporated in the statistical models.\n",
      "Version: 1.8.0\n",
      "Publication date: None\n",
      "Mantainer: Lanyu Zhang <jennyzly2016@gmail.com>\n",
      "Authors:\n",
      "Author: Lanyu Zhang [aut, cre], Gabriel Odom [aut], Tiago Silva [aut], Lissette Gomez [aut], Lily Wang [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - GenomicRanges, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - bumphunter, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - stats, type: imports\n",
      "  - survival, type: imports\n",
      "  - logistf, type: imports\n",
      "  - plyr, type: imports\n",
      "  - corrplot, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/rnaEditr.html\n",
      "\n",
      "Name: RNAsense\n",
      "Description:\n",
      "RNA-sense tool compares RNA-seq time curves in two experimental conditions, i.e. wild-type and mutant, and works in three steps. At Step 1, it builds expression profile for each transcript in one condition (i.e. wild-type) and tests if the transcript abundance grows or decays significantly. Dynamic transcripts are then sorted to non-overlapping groups (time profiles) by the time point of switch up or down. At Step 2, RNA-sense outputs the groups of differentially expressed transcripts, which are up- or downregulated in the mutant compared to the wild-type at each time point. At Step 3, Correlations (Fisher's exact test) between the outputs of Step 1 (switch up- and switch down- time profile groups) and the outputs of Step2 (differentially expressed transcript groups) are calculated. The results of the correlation analysis are printed as two-dimensional color plot, with time profiles and differential expression groups at y- and x-axis, respectively, and facilitates the biological interpretation of the data.\n",
      "Version: 1.12.0\n",
      "Publication date: None\n",
      "Mantainer: Marcus Rosenblatt <marcus.rosenblatt@gmail.com>\n",
      "Authors:\n",
      "Author: Marcus Rosenblatt [cre], Gao Meijang [aut], Helge Hass [aut], Daria Onichtchouk [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.6) type: depends\n",
      "  - ggplot2, type: imports\n",
      "  - parallel, type: imports\n",
      "  - NBPSeq, type: imports\n",
      "  - qvalue, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - methods, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/RNAsense.html\n",
      "\n",
      "Name: rSWeeP\n",
      "Description:\n",
      "The SWeeP method was developed to favor the analizes between amino acids sequences and to assist alignment free phylogenetic studies. This method is based on the concept of sparse words, which is applied in the scan of biological sequences and its the conversion in a matrix of ocurrences. Aiming the generation of low dimensional matrices of Amino Acid Sequence occurrences.\n",
      "Version: 1.10.0\n",
      "Publication date: None\n",
      "Mantainer: Danrley R. Fernandes <DanrleyRF@gmail.com>\n",
      "Authors:\n",
      "Author: Danrley R. Fernandes [com, cre, aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - pracma, type: imports\n",
      "  - stats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/rSWeeP.html\n",
      "\n",
      "Name: rTRMui\n",
      "Description:\n",
      "This package provides a web interface to compute transcriptional regulatory modules with rTRM.\n",
      "Version: 1.36.0\n",
      "Publication date: None\n",
      "Mantainer: Diego Diez <diego10ruiz@gmail.com>\n",
      "Authors:\n",
      "Author: Diego Diez\n",
      "Dependencies:\n",
      "  - shiny (>= 0.9) type: imports\n",
      "  - rTRM, type: imports\n",
      "  - MotifDb, type: imports\n",
      "  - org.Hs.eg.db, type: imports\n",
      "  - org.Mm.eg.db, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/rTRMui.html\n",
      "\n",
      "Name: sagenhaft\n",
      "Description:\n",
      "This package implements several functions useful for analysis of gene expression data by sequencing tags as done in SAGE (Serial Analysis of Gene Expressen) data, i.e. extraction of a SAGE library from sequence files, sequence error correction, library comparison. Sequencing error correction is implementing using an Expectation Maximization Algorithm based on a Mixture Model of tag counts.\n",
      "Version: 1.68.0\n",
      "Publication date: None\n",
      "Mantainer: Tim Beissbarth <tim.beissbarth@bioinf.med.uni-goettingen.de>\n",
      "Authors:\n",
      "Author: Tim Beissbarth <tim.beissbarth at bioinf.med.uni-goettingen.de>, with contributions from Gordon Smyth <smyth at wehi.edu.au>\n",
      "Dependencies:\n",
      "  - R (>= 2.10) type: depends\n",
      "  - SparseM (>= 0.73) type: depends\n",
      "  - methods, type: depends\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/sagenhaft.html\n",
      "\n",
      "Name: Scale4C\n",
      "Description:\n",
      "Scale4C is an R/Bioconductor package for scale-space transformation and visualization of 4C-seq data. The scale-space transformation is a multi-scale visualization technique to transform a 2D signal (e.g. 4C-seq reads on a genomic interval of choice) into a tesselation in the scale space (2D, genomic position x scale factor) by applying different smoothing kernels (Gauss, with increasing sigma). This transformation allows for explorative analysis and comparisons of the data's structure with other samples.\n",
      "Version: 1.20.0\n",
      "Publication date: None\n",
      "Mantainer: Carolin Walter <carolin.walter@uni-muenster.de>\n",
      "Authors:\n",
      "Author: Carolin Walter\n",
      "Dependencies:\n",
      "  - R (>= 3.4) type: depends\n",
      "  - smoothie, type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - IRanges, type: depends\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - methods, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/Scale4C.html\n",
      "\n",
      "Name: scFeatureFilter\n",
      "Description:\n",
      "An R implementation of the correlation-based method developed in the Joshi laboratory to analyse and filter processed single-cell RNAseq data. It returns a filtered version of the data containing only genes expression values unaffected by systematic noise.\n",
      "Version: 1.18.0\n",
      "Publication date: None\n",
      "Mantainer: Guillaume Devailly <gdevailly@hotmail.com>\n",
      "Authors:\n",
      "Author: Angeles Arzalluz-Luque [aut], Guillaume Devailly [aut, cre], Anagha Joshi [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.6) type: depends\n",
      "  - dplyr (>= 0.7.3) type: imports\n",
      "  - ggplot2 (>= 2.1.0) type: imports\n",
      "  - magrittr (>= 1.5) type: imports\n",
      "  - rlang (>= 0.1.2) type: imports\n",
      "  - tibble (>= 1.3.4) type: imports\n",
      "  - stats, type: imports\n",
      "  - methods, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/scFeatureFilter.html\n",
      "\n",
      "Name: Sconify\n",
      "Description:\n",
      "This package does k-nearest neighbor based statistics and visualizations with flow and mass cytometery data. This gives tSNE maps\"fold change\" functionality and provides a data quality metric by assessing manifold overlap between fcs files expected to be the same. Other applications using this package include imputation, marker redundancy, and testing the relative information loss of lower dimension embeddings compared to the original manifold.\n",
      "Version: 1.18.0\n",
      "Publication date: None\n",
      "Mantainer: Tyler J Burns <burns.tyler@gmail.com>\n",
      "Authors:\n",
      "Author: Tyler J Burns\n",
      "Dependencies:\n",
      "  - R (>= 3.5) type: depends\n",
      "  - tibble, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - FNN, type: imports\n",
      "  - flowCore, type: imports\n",
      "  - Rtsne, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - utils, type: imports\n",
      "  - stats, type: imports\n",
      "  - readr, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/Sconify.html\n",
      "\n",
      "Name: scTreeViz\n",
      "Description:\n",
      "scTreeViz provides classes to support interactive data aggregation and visualization of single cell RNA-seq datasets with hierarchies for e.g. cell clusters at different resolutions. The `TreeIndex` class provides methods to manage hierarchy and split the tree at a given resolution or across resolutions. The `TreeViz` class extends `SummarizedExperiment` and can performs quick aggregations on the count matrix defined by clusters.\n",
      "Version: 1.4.0\n",
      "Publication date: None\n",
      "Mantainer: Jayaram Kancherla <jayaram.kancherla@gmail.com>\n",
      "Authors:\n",
      "Author: Jayaram Kancherla [aut, cre], Hector Corrada Bravo [aut], Kazi Tasnim Zinat [aut], Stephanie Hicks [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - epivizr, type: depends\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - data.table, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - digest, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - Rtsne, type: imports\n",
      "  - httr, type: imports\n",
      "  - igraph, type: imports\n",
      "  - clustree, type: imports\n",
      "  - scran, type: imports\n",
      "  - sys, type: imports\n",
      "  - epivizrData, type: imports\n",
      "  - epivizrServer, type: imports\n",
      "  - ggraph, type: imports\n",
      "  - scater, type: imports\n",
      "  - Seurat, type: imports\n",
      "  - SingleCellExperiment, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/scTreeViz.html\n",
      "\n",
      "Name: segmenter\n",
      "Description:\n",
      "Chromatin segmentation analysis transforms ChIP-seq data into signals over the genome. The latter represents the observed states in a multivariate Markov model to predict the chromatin's underlying states. ChromHMM, written in Java, integrates histone modification datasets to learn the chromatin states de-novo. The goal of this package is to call chromHMM from within R, capture the output files in an S4 object and interface to other relevant Bioconductor analysis tools. In addition, segmenter provides functions to test, select and visualize the output of the segmentation.\n",
      "Version: 1.4.0\n",
      "Publication date: None\n",
      "Mantainer: Mahmoud Ahmed <mahmoud.s.fahmy@students.kasralainy.edu.eg>\n",
      "Authors:\n",
      "Author: Mahmoud Ahmed [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - ChIPseeker, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - bamsignals, type: imports\n",
      "  - ComplexHeatmap, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - methods, type: imports\n",
      "  - chromhmmData, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/segmenter.html\n",
      "\n",
      "Name: SIMD\n",
      "Description:\n",
      "This package provides a inferential analysis method for detecting differentially expressed CpG sites in MeDIP-seq data. It uses statistical framework and EM algorithm, to identify differentially expressed CpG sites. The methods on this package are described in the article 'Methylation-level Inferences and Detection of Differential Methylation with Medip-seq Data' by Yan Zhou, Jiadi Zhu, Mingtao Zhao, Baoxue Zhang, Chunfu Jiang and Xiyan Yang (2018, pending publication).\n",
      "Version: 1.16.0\n",
      "Publication date: None\n",
      "Mantainer: Jiadi Zhu <2160090406@email.szu.edu.cn>\n",
      "Authors:\n",
      "Author: Yan Zhou\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - edgeR, type: imports\n",
      "  - statmod, type: imports\n",
      "  - methylMnM, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/SIMD.html\n",
      "\n",
      "Name: SimFFPE\n",
      "Description:\n",
      "The NGS (Next-Generation Sequencing) reads from FFPE (Formalin-Fixed Paraffin-Embedded) samples contain numerous artifact chimeric reads (ACRS), which can lead to false positive structural variant calls. These ACRs are derived from the combination of two single-stranded DNA (ss-DNA) fragments with short reverse complementary regions (SRCRs). This package simulates these artifact chimeric reads as well as normal reads for FFPE samples on the whole genome / several chromosomes / large regions.\n",
      "Version: 1.10.0\n",
      "Publication date: None\n",
      "Mantainer: Lanying Wei <lanying.wei@uni-muenster.de>\n",
      "Authors:\n",
      "Author: Lanying Wei [aut, cre]\n",
      "Dependencies:\n",
      "  - Biostrings, type: depends\n",
      "  - dplyr, type: imports\n",
      "  - foreach, type: imports\n",
      "  - doParallel, type: imports\n",
      "  - truncnorm, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - parallel, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - methods, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/SimFFPE.html\n",
      "\n",
      "Name: TDARACNE\n",
      "Description:\n",
      "To infer gene networks from time-series measurements is a current challenge into bioinformatics research area. In order to detect dependencies between genes at different time delays, we propose an approach to infer gene regulatory networks from time-series measurements starting from a well known algorithm based on information theory. The proposed algorithm is expected to be useful in reconstruction of small biological directed networks from time course data.\n",
      "Version: 1.47.0\n",
      "Publication date: None\n",
      "Mantainer: Zoppoli Pietro <zoppoli.pietro@gmail.com>\n",
      "Authors:\n",
      "Author: Zoppoli P.,Morganella S., Ceccarelli M.\n",
      "Dependencies:\n",
      "  - GenKern, type: depends\n",
      "  - Rgraphviz, type: depends\n",
      "  - Biobase, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/TDARACNE.html\n",
      "\n",
      "Name: transomics2cytoscape\n",
      "Description:\n",
      "transomics2cytoscape generates a file for 3D transomics visualization by providing input that specifies the IDs of multiple KEGG pathway layers, their corresponding Z-axis heights, and an input that represents the edges between the pathway layers. The edges are used, for example, to describe the relationships between kinase on a pathway and enzyme on another pathway. This package automates creation of a transomics network as shown in the figure in Yugi.2014 (https://doi.org/10.1016/j.celrep.2014.07.021) using Cytoscape automation (https://doi.org/10.1186/s13059-019-1758-4).\n",
      "Version: 1.8.0\n",
      "Publication date: None\n",
      "Mantainer: Kozo Nishida <kozo.nishida@gmail.com>\n",
      "Authors:\n",
      "Author: Kozo Nishida [aut, cre] , Katsuyuki Yugi [aut]\n",
      "Dependencies:\n",
      "  - RCy3, type: imports\n",
      "  - KEGGREST, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - purrr, type: imports\n",
      "  - tibble, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/transomics2cytoscape.html\n",
      "\n",
      "Name: treekoR\n",
      "Description:\n",
      "treekoR is a novel framework that aims to utilise the hierarchical nature of single cell cytometry data to find robust and interpretable associations between cell subsets and patient clinical end points. These associations are aimed to recapitulate the nested proportions prevalent in workflows inovlving manual gating, which are often overlooked in workflows using automatic clustering to identify cell populations. We developed treekoR to: Derive a hierarchical tree structure of cell clusters; quantify a cell types as a proportion relative to all cells in a sample (%total), and, as the proportion relative to a parent population (%parent); perform significance testing using the calculated proportions; and provide an interactive html visualisation to help highlight key results.\n",
      "Version: 1.6.1\n",
      "Publication date: None\n",
      "Mantainer: Adam Chan <adam.s.chan@sydney.edu.au>\n",
      "Authors:\n",
      "Author: Adam Chan [aut, cre], Ellis Patrick [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - data.table, type: imports\n",
      "  - ggiraph, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - hopach, type: imports\n",
      "  - ape, type: imports\n",
      "  - ggtree, type: imports\n",
      "  - patchwork, type: imports\n",
      "  - SingleCellExperiment, type: imports\n",
      "  - diffcyt, type: imports\n",
      "  - edgeR, type: imports\n",
      "  - lme4, type: imports\n",
      "  - multcomp, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/treekoR.html\n",
      "\n",
      "Name: TTMap\n",
      "Description:\n",
      "TTMap is a clustering method that groups together samples with the same deviation in comparison to a control group. It is specially useful when the data is small. It is parameter free.\n",
      "Version: 1.20.0\n",
      "Publication date: None\n",
      "Mantainer: Rachel Jeitziner <rachel.jeitziner@epfl.ch>\n",
      "Authors:\n",
      "Author: Rachel Jeitziner\n",
      "Dependencies:\n",
      "  - rgl, type: depends\n",
      "  - colorRamps, type: depends\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - methods, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - Biobase, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/TTMap.html\n",
      "\n",
      "Name: dks\n",
      "Description:\n",
      "The dks package consists of a set of diagnostic functions for multiple testing methods. The functions can be used to determine if the p-values produced by a multiple testing procedure are correct. These functions are designed to be applied to simulated data. The functions require the entire set of p-values from multiple simulated studies, so that the joint distribution can be evaluated.\n",
      "Version: 1.44.0\n",
      "Publication date: None\n",
      "Mantainer: Jeffrey T. Leek <jleek@jhsph.edu>\n",
      "Authors:\n",
      "Author: Jeffrey T. Leek <jleek at jhsph.edu>\n",
      "Dependencies:\n",
      "  - R (>= 2.8) type: depends\n",
      "  - cubature, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/dks.html\n",
      "\n",
      "Name: epidecodeR\n",
      "Description:\n",
      "epidecodeR is a package capable of analysing impact of degree of DNA/RNA epigenetic chemical modifications on dysregulation of genes or proteins. This package integrates chemical modification data generated from a host of epigenomic or epitranscriptomic techniques such as ChIP-seq, ATAC-seq, m6A-seq, etc. and dysregulated gene lists in the form of differential gene expression, ribosome occupancy or differential protein translation and identify impact of dysregulation of genes caused due to varying degrees of chemical modifications associated with the genes. epidecodeR generates cumulative distribution function (CDF) plots showing shifts in trend of overall log2FC between genes divided into groups based on the degree of modification associated with the genes. The tool also tests for significance of difference in log2FC between groups of genes.\n",
      "Version: 1.6.0\n",
      "Publication date: None\n",
      "Mantainer: Kandarp Joshi <kandarpbioinfo@gmail.com>\n",
      "Authors:\n",
      "Author: Kandarp Joshi [aut, cre], Dan Ohtan Wang [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.1.0) type: depends\n",
      "  - EnvStats, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - rstatix, type: imports\n",
      "  - ggpubr, type: imports\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - dplyr, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/epidecodeR.html\n",
      "\n",
      "Name: hierinf\n",
      "Description:\n",
      "Tools to perform hierarchical inference for one or multiple studies / data sets based on high-dimensional multivariate (generalised) linear models. A possible application is to perform hierarchical inference for GWA studies to find significant groups or single SNPs (if the signal is strong) in a data-driven and automated procedure. The method is based on an efficient hierarchical multiple testing correction and controls the FWER. The functions can easily be run in parallel.\n",
      "Version: 1.16.0\n",
      "Publication date: None\n",
      "Mantainer: Claude Renaux <renaux@stat.math.ethz.ch>\n",
      "Authors:\n",
      "Author: Claude Renaux [aut, cre], Laura Buzdugan [aut], Markus Kalisch [aut], Peter BÃ¼hlmann [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.6.0) type: depends\n",
      "  - fmsb, type: imports\n",
      "  - glmnet, type: imports\n",
      "  - methods, type: imports\n",
      "  - parallel, type: imports\n",
      "  - stats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3 | file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/hierinf.html\n",
      "\n",
      "Name: iPath\n",
      "Description:\n",
      "iPath is the Bioconductor package used for calculating personalized pathway score and test the association with survival outcomes. Abundant single-gene biomarkers have been identified and used in the clinics. However, hundreds of oncogenes or tumor-suppressor genes are involved during the process of tumorigenesis. We believe individual-level expression patterns of pre-defined pathways or gene sets are better biomarkers than single genes. In this study, we devised a computational method named iPath to identify prognostic biomarker pathways, one sample at a time. To test its utility, we conducted a pan-cancer analysis across 14 cancer types from The Cancer Genome Atlas and demonstrated that iPath is capable of identifying highly predictive biomarkers for clinical outcomes, including overall survival, tumor subtypes, and tumor stage classifications. We found that pathway-based biomarkers are more robust and effective than single genes.\n",
      "Version: 1.4.2\n",
      "Publication date: None\n",
      "Mantainer: Kenong Su <kenong.su@emory.edu>\n",
      "Authors:\n",
      "Author: Kenong Su [aut, cre], Zhaohui Qin [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - mclust, type: depends\n",
      "  - BiocParallel, type: depends\n",
      "  - survival, type: depends\n",
      "  - Rcpp (>= 1.0.5) type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - ggpubr, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - survminer, type: imports\n",
      "  - stats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/iPath.html\n",
      "\n",
      "Name: LinkHD\n",
      "Description:\n",
      "Here we present Link-HD, an approach to integrate heterogeneous datasets, as a generalization of STATIS-ACT (âStructuration des Tableaux A Trois Indices de la StatistiqueâAnalyse Conjointe de Tableauxâ), a family of methods to join and compare information from multiple subspaces. However, STATIS-ACT has some drawbacks since it only allows continuous data and it is unable to establish relationships between samples and features. In order to tackle these constraints, we incorporate multiple distance options and a linear regression based Biplot model in order to stablish relationships between observations and variable and perform variable selection.\n",
      "Version: 1.12.0\n",
      "Publication date: None\n",
      "Mantainer: \"Laura M Zingaretti\" <m.lau.zingaretti@gmail.com>\n",
      "Authors:\n",
      "Author: Laura M. Zingaretti [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 3.6.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - ggplot2, type: depends\n",
      "  - stats, type: depends\n",
      "  - scales, type: imports\n",
      "  - cluster, type: imports\n",
      "  - graphics, type: imports\n",
      "  - ggpubr, type: imports\n",
      "  - gridExtra, type: imports\n",
      "  - vegan, type: imports\n",
      "  - rio, type: imports\n",
      "  - MultiAssayExperiment, type: imports\n",
      "  - emmeans, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - data.table, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/LinkHD.html\n",
      "\n",
      "Name: LowMACA\n",
      "Description:\n",
      "The LowMACA package is a simple suite of tools to investigate and analyze the mutation profile of several proteins or pfam domains via consensus alignment. You can conduct an hypothesis driven exploratory analysis using our package simply providing a set of genes or pfam domains of your interest.\n",
      "Version: 1.28.0\n",
      "Publication date: None\n",
      "Mantainer: Giorgio Melloni <melloni.giorgio@gmail.com> , Stefano de Pretis <ste.depo@gmail.com>\n",
      "Authors:\n",
      "Author: Giorgio Melloni , Stefano de Pretis\n",
      "Dependencies:\n",
      "  - R (>= 2.10) type: depends\n",
      "  - cBioPortalData, type: imports\n",
      "  - parallel, type: imports\n",
      "  - stringr, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - data.table, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - methods, type: imports\n",
      "  - LowMACAAnnotation, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - motifStack, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - httr, type: imports\n",
      "  - grid, type: imports\n",
      "  - gridBase, type: imports\n",
      "  - plyr, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/LowMACA.html\n",
      "\n",
      "Name: MBAmethyl\n",
      "Description:\n",
      "This package provides a function for reconstructing DNA methylation values from raw measurements. It iteratively implements the group fused lars to smooth related-by-location methylation values and the constrained least squares to remove probe affinity effect across multiple sequences.\n",
      "Version: 1.32.0\n",
      "Publication date: None\n",
      "Mantainer: Tao Wang <tao.wang.tw376@yale.edu>\n",
      "Authors:\n",
      "Author: Tao Wang, Mengjie Chen\n",
      "Dependencies:\n",
      "  - R (>= 2.15) type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/MBAmethyl.html\n",
      "\n",
      "Name: metavizr\n",
      "Description:\n",
      "This package provides Websocket communication to the metaviz web app (http://metaviz.cbcb.umd.edu) for interactive visualization of metagenomics data. Objects in R/bioc interactive sessions can be displayed in plots and data can be explored using a facetzoom visualization. Fundamental Bioconductor data structures are supported (e.g., MRexperiment objects), while providing an easy mechanism to support other data structures. Visualizations (using d3.js) can be easily added to the web app as well.\n",
      "Version: 1.21.0\n",
      "Publication date: None\n",
      "Mantainer: Hector Corrada Bravo <hcorrada@gmail.com>\n",
      "Authors:\n",
      "Author: Hector Corrada Bravo [cre, aut], Florin Chelaru [aut], Justin Wagner [aut], Jayaram Kancherla [aut], Joseph Paulson [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.4) type: depends\n",
      "  - metagenomeSeq, type: depends\n",
      "  - methods, type: depends\n",
      "  - data.table, type: depends\n",
      "  - Biobase, type: depends\n",
      "  - digest, type: depends\n",
      "  - epivizr, type: imports\n",
      "  - epivizrData, type: imports\n",
      "  - epivizrServer, type: imports\n",
      "  - epivizrStandalone, type: imports\n",
      "  - vegan, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - phyloseq, type: imports\n",
      "  - httr, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/metavizr.html\n",
      "\n",
      "Name: methylscaper\n",
      "Description:\n",
      "methylscaper is an R package for processing and visualizing data jointly profiling methylation and chromatin accessibility (MAPit, NOMe-seq, scNMT-seq, nanoNOMe, etc.). The package supports both single-cell and single-molecule data, and a common interface for jointly visualizing both data types through the generation of ordered representational methylation-state matrices. The Shiny app allows for an interactive seriation process of refinement and re-weighting that optimally orders the cells or DNA molecules to discover methylation patterns and nucleosome positioning.\n",
      "Version: 1.6.0\n",
      "Publication date: None\n",
      "Mantainer: Bacher Rhonda <rbacher@ufl.edu>\n",
      "Authors:\n",
      "Author: Bacher Rhonda [aut, cre], Parker Knight [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.1.0) type: depends\n",
      "  - shiny, type: imports\n",
      "  - shinyjs, type: imports\n",
      "  - seriation, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - seqinr, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - Rfast, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - tools, type: imports\n",
      "  - methods, type: imports\n",
      "  - shinyFiles, type: imports\n",
      "  - data.table, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/methylscaper.html\n",
      "\n",
      "Name: MPFE\n",
      "Description:\n",
      "Estimate distribution of methylation patterns from a table of counts from a bisulphite sequencing experiment given a non-conversion rate and read error rate.\n",
      "Version: 1.34.0\n",
      "Publication date: None\n",
      "Mantainer: Conrad Burden <conrad.burden@anu.edu.au>\n",
      "Authors:\n",
      "Author: Peijie Lin, Sylvain Foret, Conrad Burden\n",
      "Dependencies:\n",
      "None\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 3)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/MPFE.html\n",
      "\n",
      "Name: MSstatsLiP\n",
      "Description:\n",
      "Tools for LiP peptide and protein significance analysis. Provides functions for summarization, estimation of LiP peptide abundance, and detection of changes across conditions. Utilizes functionality across the MSstats family of packages.\n",
      "Version: 1.4.1\n",
      "Publication date: None\n",
      "Mantainer: Devon Kohler <kohler.d@northeastern.edu>\n",
      "Authors:\n",
      "Author: Devon Kohler [aut, cre], Tsung-Heng Tsai [aut], Ting Huang [aut], Mateusz Staniak [aut], Meena Choi [aut], Valentina Cappelletti [aut], Liliana Malinovska [aut], Olga Vitek [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - dplyr, type: imports\n",
      "  - gridExtra, type: imports\n",
      "  - stringr, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - MSstats, type: imports\n",
      "  - MSstatsConvert, type: imports\n",
      "  - data.table, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - MSstatsPTM, type: imports\n",
      "  - Rcpp, type: imports\n",
      "  - checkmate, type: imports\n",
      "  - factoextra, type: imports\n",
      "  - ggpubr, type: imports\n",
      "  - purrr, type: imports\n",
      "  - tibble, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - tidyverse, type: imports\n",
      "  - scales, type: imports\n",
      "  - stats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/MSstatsLiP.html\n",
      "\n",
      "Name: multiSight\n",
      "Description:\n",
      "multiSight is an R package providing functions to analyze your omic datasets in a multi-omics manner based on Stouffer's p-value pooling and multi-block statistical methods. For each omic dataset you furnish, multiSight provides classification models with feature selection you can use as biosignature: (i) To forecast phenotypes (e.g. to diagnostic tasks, histological subtyping), (ii) To design Pathways and gene ontology enrichments (Over Representation Analysis), (iii) To build Network inference linked to PubMed querying to make assumptions easier and data-driven. Main analysis are embedded in an user-friendly graphical interface.\n",
      "Version: 1.6.0\n",
      "Publication date: None\n",
      "Mantainer: Florian Jeanneret <florian.jeanneret@cea.fr>\n",
      "Authors:\n",
      "Author: Florian Jeanneret [cre, aut] , Stephane Gazut [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - golem, type: imports\n",
      "  - config, type: imports\n",
      "  - R6, type: imports\n",
      "  - shiny, type: imports\n",
      "  - shinydashboard, type: imports\n",
      "  - DT, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - stringr, type: imports\n",
      "  - anyLib, type: imports\n",
      "  - caret, type: imports\n",
      "  - biosigner, type: imports\n",
      "  - mixOmics, type: imports\n",
      "  - stats, type: imports\n",
      "  - DESeq2, type: imports\n",
      "  - clusterProfiler, type: imports\n",
      "  - rWikiPathways, type: imports\n",
      "  - ReactomePA, type: imports\n",
      "  - enrichplot, type: imports\n",
      "  - ppcor, type: imports\n",
      "  - metap, type: imports\n",
      "  - infotheo, type: imports\n",
      "  - igraph, type: imports\n",
      "  - networkD3, type: imports\n",
      "  - easyPubMed, type: imports\n",
      "  - utils, type: imports\n",
      "  - htmltools, type: imports\n",
      "  - rmarkdown, type: imports\n",
      "  - ggnewscale, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "CeCILL + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/multiSight.html\n",
      "\n",
      "Name: netboxr\n",
      "Description:\n",
      "NetBox is a network-based approach that combines prior knowledge with a network clustering algorithm. The algorithm allows for the identification of functional modules and allows for combining multiple data types, such as mutations and copy number alterations. NetBox performs network analysis on human interaction networks, and comes pre-loaded with a Human Interaction Network (HIN) derived from four literature curated data sources, including the Human Protein Reference Database (HPRD), Reactome, NCI-Nature Pathway Interaction (PID) Database, and the MSKCC Cancer Cell Map.\n",
      "Version: 1.9.0\n",
      "Publication date: None\n",
      "Mantainer: Eirc Minwei Liu <emliu.research@gmail.com>\n",
      "Authors:\n",
      "Author: Eric Minwei Liu [aut,cre], Augustin Luna [aut], Ethan Cerami [aut], Chris Sander [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.0.0) type: depends\n",
      "  - igraph (>= 1.2.4.1) type: depends\n",
      "  - parallel, type: depends\n",
      "  - RColorBrewer, type: imports\n",
      "  - DT, type: imports\n",
      "  - stats, type: imports\n",
      "  - clusterProfiler, type: imports\n",
      "  - data.table, type: imports\n",
      "  - gplots, type: imports\n",
      "  - jsonlite, type: imports\n",
      "  - plyr, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL-3 + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/netboxr.html\n",
      "\n",
      "Name: NTW\n",
      "Description:\n",
      "This package predicts the gene-gene interaction network and identifies the direct transcriptional targets of the perturbation using an ODE (Ordinary Differential Equation) based method.\n",
      "Version: 1.48.0\n",
      "Publication date: None\n",
      "Mantainer: Yuanhua Liu <liuyuanhua@picb.ac.cn>\n",
      "Authors:\n",
      "Author: Wei Xiao, Yin Jin, Darong Lai, Xinyi Yang, Yuanhua Liu, Christine Nardini\n",
      "Dependencies:\n",
      "  - R (>= 2.3.0) type: depends\n",
      "  - mvtnorm, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/NTW.html\n",
      "\n",
      "Name: oneSENSE\n",
      "Description:\n",
      "A graphical user interface that facilitates the dimensional reduction method based on the t-distributed stochastic neighbor embedding (t-SNE) algorithm, for categorical analysis of mass cytometry data. With One-SENSE, measured parameters are grouped into predefined categories, and cells are projected onto a space composed of one dimension for each category. Each dimension is informative and can be annotated through the use of heatplots aligned in parallel to each axis, allowing for simultaneous visualization of two catergories across a two-dimensional plot. The cellular occupancy of the resulting plots alllows for direct assessment of the relationships between the categories.\n",
      "Version: 1.20.0\n",
      "Publication date: None\n",
      "Mantainer: Yong Kee Tan <yongkee.t@gmail.com>\n",
      "Authors:\n",
      "Author: Cheng Yang, Evan Newell, Yong Kee Tan\n",
      "Dependencies:\n",
      "  - R (>= 3.4) type: depends\n",
      "  - webshot, type: depends\n",
      "  - shiny, type: depends\n",
      "  - shinyFiles, type: depends\n",
      "  - scatterplot3d, type: depends\n",
      "  - Rtsne, type: imports\n",
      "  - plotly, type: imports\n",
      "  - gplots, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - methods, type: imports\n",
      "  - flowCore, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>=3)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/oneSENSE.html\n",
      "\n",
      "Name: OVESEG\n",
      "Description:\n",
      "An R package for multiple-group comparison to detect tissue/cell-specific marker genes among subtypes. It provides functions to compute OVESEG-test statistics, derive component weights in the mixture null distribution model and estimate p-values from weightedly aggregated permutations. Obtained posterior probabilities of component null hypotheses can also portrait all kinds of upregulation patterns among subtypes.\n",
      "Version: 1.14.0\n",
      "Publication date: None\n",
      "Mantainer: Lulu Chen <luluchen@vt.edu>\n",
      "Authors:\n",
      "Author: Lulu Chen <luluchen at vt.edu>\n",
      "Dependencies:\n",
      "  - R (>= 3.6) type: depends\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - methods, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - limma, type: imports\n",
      "  - fdrtool, type: imports\n",
      "  - Rcpp, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/OVESEG.html\n",
      "\n",
      "Name: ReactomeGraph4R\n",
      "Description:\n",
      "Pathways, reactions, and biological entities in Reactome knowledge are systematically represented as an ordered network. Instances are represented as nodes and relationships between instances as edges; they are all stored in the Reactome Graph Database. This package serves as an interface to query the interconnected data from a local Neo4j database, with the aim of minimizing the usage of Neo4j Cypher queries.\n",
      "Version: 1.6.0\n",
      "Publication date: None\n",
      "Mantainer: Chi-Lam Poon <clpoon807@gmail.com>\n",
      "Authors:\n",
      "Author: Chi-Lam Poon [aut, cre] , Reactome [cph]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - neo4r, type: imports\n",
      "  - utils, type: imports\n",
      "  - getPass, type: imports\n",
      "  - jsonlite, type: imports\n",
      "  - purrr, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - data.table, type: imports\n",
      "  - rlang, type: imports\n",
      "  - ReactomeContentService4R, type: imports\n",
      "  - doParallel, type: imports\n",
      "  - parallel, type: imports\n",
      "  - foreach, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Apache License (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ReactomeGraph4R.html\n",
      "\n",
      "Name: RiboCrypt\n",
      "Description:\n",
      "R Package for interactive visualization and browsing NGS data. It contains a browser for both transcript and genomic coordinate view. In addition a QC and general metaplots are included, among others differential translation plots and gene expression plots. The package is still under development.\n",
      "Version: 1.4.0\n",
      "Publication date: None\n",
      "Mantainer: Michal Swirski <michal.swirski@uw.edu.pl>\n",
      "Authors:\n",
      "Author: Michal Swirski [aut, cre], Haakon Tjeldnes [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 3.6.0) type: depends\n",
      "  - ORFik, type: depends\n",
      "  - BiocGenerics, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - data.table, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - GenomicFeatures, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - plotly, type: imports\n",
      "  - rlang, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/RiboCrypt.html\n",
      "\n",
      "Name: RIPAT\n",
      "Description:\n",
      "RIPAT is developed as an R package for retroviral integration sites annotation and distribution analysis. RIPAT needs local alignment results from BLAST and BLAT. Specific input format is depicted in RIPAT manual. RIPAT provides RV integration pattern analysis result as forms of R objects, excel file with multiple sheets and plots.\n",
      "Version: 1.8.0\n",
      "Publication date: None\n",
      "Mantainer: Min-Jeong Baek <mjbaek16@korea.ac.kr>\n",
      "Authors:\n",
      "Author: Min-Jeong Baek [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - biomaRt, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - ggplot2 (>= 3.1.0) type: imports\n",
      "  - grDevices (>= 3.5.3) type: imports\n",
      "  - IRanges, type: imports\n",
      "  - karyoploteR, type: imports\n",
      "  - openxlsx (>= 4.1.4) type: imports\n",
      "  - plyr (>= 1.8.4) type: imports\n",
      "  - regioneR, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - stats (>= 3.5.3) type: imports\n",
      "  - stringr (>= 1.3.1) type: imports\n",
      "  - utils (>= 3.5.3) type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/RIPAT.html\n",
      "\n",
      "Name: RNAdecay\n",
      "Description:\n",
      "RNA degradation is monitored through measurement of RNA abundance after inhibiting RNA synthesis. This package has functions and example scripts to facilitate (1) data normalization, (2) data modeling using constant decay rate or time-dependent decay rate models, (3) the evaluation of treatment or genotype effects, and (4) plotting of the data and models. Data Normalization: functions and scripts make easy the normalization to the initial (T0) RNA abundance, as well as a method to correct for artificial inflation of Reads per Million (RPM) abundance in global assessments as the total size of the RNA pool decreases. Modeling: Normalized data is then modeled using maximum likelihood to fit parameters. For making treatment or genotype comparisons (up to four), the modeling step models all possible treatment effects on each gene by repeating the modeling with constraints on the model parameters (i.e., the decay rate of treatments A and B are modeled once with them being equal and again allowing them to both vary independently). Model Selection: The AICc value is calculated for each model, and the model with the lowest AICc is chosen. Modeling results of selected models are then compiled into a single data frame. Graphical Plotting: functions are provided to easily visualize decay data model, or half-life distributions using ggplot2 package functions.\n",
      "Version: 1.18.0\n",
      "Publication date: None\n",
      "Mantainer: Reed Sorenson <reedssorenson@gmail.com>\n",
      "Authors:\n",
      "Author: Reed Sorenson [aut, cre], Katrina Johnson [aut], Frederick Adler [aut], Leslie Sieburth [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.5) type: depends\n",
      "  - stats, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - grid, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - gplots, type: imports\n",
      "  - utils, type: imports\n",
      "  - TMB, type: imports\n",
      "  - nloptr, type: imports\n",
      "  - scales, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/RNAdecay.html\n",
      "\n",
      "Name: scanMiRApp\n",
      "Description:\n",
      "A shiny interface to the scanMiR package. The application enables the scanning of transcripts and custom sequences for miRNA binding sites, the visualization of KdModels and binding results, as well as browsing predicted repression data. In addition contains the IndexedFst class for fast indexed reading of large GenomicRanges or data.frames, and some utilities for facilitating scans and identifying enriched miRNA-target pairs.\n",
      "Version: 1.4.0\n",
      "Publication date: None\n",
      "Mantainer: Pierre-Luc Germain <pierre-luc.germain@hest.ethz.ch>\n",
      "Authors:\n",
      "Author: Pierre-Luc Germain [cre, aut] , Michael Soutschek [aut], Fridolin Gross [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - AnnotationDbi, type: imports\n",
      "  - AnnotationFilter, type: imports\n",
      "  - AnnotationHub, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - data.table, type: imports\n",
      "  - digest, type: imports\n",
      "  - DT, type: imports\n",
      "  - ensembldb, type: imports\n",
      "  - fst, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - GenomicFeatures, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - htmlwidgets, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - methods, type: imports\n",
      "  - plotly, type: imports\n",
      "  - rintrojs, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - scanMiR, type: imports\n",
      "  - scanMiRData, type: imports\n",
      "  - shiny, type: imports\n",
      "  - shinycssloaders, type: imports\n",
      "  - shinydashboard, type: imports\n",
      "  - shinyjqui, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - waiter, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/scanMiRApp.html\n",
      "\n",
      "Name: selectKSigs\n",
      "Description:\n",
      "A package to suggest the number of mutational signatures in a collection of somatic mutations using calculating the cross-validated perplexity score.\n",
      "Version: 1.10.0\n",
      "Publication date: None\n",
      "Mantainer: Zhi Yang <zyang895@gmail.com>\n",
      "Authors:\n",
      "Author: Zhi Yang [aut, cre], Yuichi Shiraishi [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 3.6) type: depends\n",
      "  - HiLDA, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - gtools, type: imports\n",
      "  - methods, type: imports\n",
      "  - Rcpp, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/selectKSigs.html\n",
      "\n",
      "Name: semisup\n",
      "Description:\n",
      "Implements a parametric semi-supervised mixture model. The permutation test detects markers with main or interactive effects, without distinguishing them. Possible applications include genome-wide association analysis and differential expression analysis.\n",
      "Version: 1.22.0\n",
      "Publication date: None\n",
      "Mantainer: Armin Rauschenberger <armin.rauschenberger@uni.lu>\n",
      "Authors:\n",
      "Author: Armin Rauschenberger [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 3.0.0) type: depends\n",
      "  - VGAM, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/semisup.html\n",
      "\n",
      "Name: SEPIRA\n",
      "Description:\n",
      "SEPIRA (Systems EPigenomics Inference of Regulatory Activity) is an algorithm that infers sample-specific transcription factor activity from the genome-wide expression or DNA methylation profile of the sample.\n",
      "Version: 1.18.0\n",
      "Publication date: None\n",
      "Mantainer: Yuting Chen <cytwarmmay@hotmail.com>\n",
      "Authors:\n",
      "Author: Yuting Chen [aut, cre], Andrew Teschendorff [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - limma, type: imports\n",
      "  - corpcor (>= 1.6.9) type: imports\n",
      "  - parallel (>= 3.3.1) type: imports\n",
      "  - stats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/SEPIRA.html\n",
      "\n",
      "Name: SMAD\n",
      "Description:\n",
      "Assigning probability scores to prey proteins captured in affinity purification mass spectrometry (AP-MS) expriments to infer protein-protein interactions. The output would facilitate non-specific background removal as contaminants are commonly found in AP-MS data.\n",
      "Version: 1.14.1\n",
      "Publication date: None\n",
      "Mantainer: Qingzhou Zhang <zqzneptune@hotmail.com>\n",
      "Authors:\n",
      "Author: Qingzhou Zhang [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 3.6.0) type: depends\n",
      "  - RcppAlgos, type: depends\n",
      "  - magrittr (>= 1.5) type: imports\n",
      "  - dplyr, type: imports\n",
      "  - stats, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - utils, type: imports\n",
      "  - Rcpp (>= 1.0.0) type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/SMAD.html\n",
      "\n",
      "Name: SOMNiBUS\n",
      "Description:\n",
      "This package aims to analyse count-based methylation data on predefined genomic regions, such as those obtained by targeted sequencing, and thus to identify differentially methylated regions (DMRs) that are associated with phenotypes or traits. The method is built a rich flexible model that allows for the effects, on the methylation levels, of multiple covariates to vary smoothly along genomic regions. At the same time, this method also allows for sequencing errors and can adjust for variability in cell type mixture.\n",
      "Version: 1.6.0\n",
      "Publication date: None\n",
      "Mantainer: Kathleen Klein <kathleen.klein@mail.mcgill.ca>\n",
      "Authors:\n",
      "Author: Kaiqiong Zhao [aut], Kathleen Klein [cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.1.0) type: depends\n",
      "  - graphics, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - mgcv, type: imports\n",
      "  - stats, type: imports\n",
      "  - VGAM, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/SOMNiBUS.html\n",
      "\n",
      "Name: sparsenetgls\n",
      "Description:\n",
      "The package provides methods of combining the graph structure learning and generalized least squares regression to improve the regression estimation. The main function sparsenetgls() provides solutions for multivariate regression with Gaussian distributed dependant variables and explanatory variables utlizing multiple well-known graph structure learning approaches to estimating the precision matrix, and uses a penalized variance covariance matrix with a distance tuning parameter of the graph structure in deriving the sandwich estimators in generalized least squares (gls) regression. This package also provides functions for assessing a Gaussian graphical model which uses the penalized approach. It uses Receiver Operative Characteristics curve as a visualization tool in the assessment.\n",
      "Version: 1.16.0\n",
      "Publication date: None\n",
      "Mantainer: Irene Zeng <szen003@aucklanduni.ac.nz>\n",
      "Authors:\n",
      "Author: Irene Zeng [aut, cre], Thomas Lumley [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 4.0.0) type: depends\n",
      "  - Matrix, type: depends\n",
      "  - MASS, type: depends\n",
      "  - methods, type: imports\n",
      "  - glmnet, type: imports\n",
      "  - huge, type: imports\n",
      "  - stats, type: imports\n",
      "  - graphics, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/sparsenetgls.html\n",
      "\n",
      "Name: spikeLI\n",
      "Description:\n",
      "SpikeLI is a package that performs the analysis of the Affymetrix spike-in data using the Langmuir Isotherm. The aim of this package is to show the advantages of a physical-chemistry based analysis of the Affymetrix microarray data compared to the traditional methods. The spike-in (or Latin square) data for the HGU95 and HGU133 chipsets have been downloaded from the Affymetrix web site. The model used in the spikeLI package is described in details in E. Carlon and T. Heim, Physica A 362, 433 (2006).\n",
      "Version: 2.58.0\n",
      "Publication date: None\n",
      "Mantainer: Enrico Carlon <enrico.carlon@fys.kuleuven.be>\n",
      "Authors:\n",
      "Author: Delphine Baillon, Paul Leclercq <paulleclercq at hotmail.com>, Sarah Ternisien, Thomas Heim, Enrico Carlon <enrico.carlon at fys.kuleuven.be>\n",
      "Dependencies:\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/spikeLI.html\n",
      "\n",
      "Name: SplicingFactory\n",
      "Description:\n",
      "The SplicingFactory R package uses transcript-level expression values to analyze splicing diversity based on various statistical measures, like Shannon entropy or the Gini index. These measures can quantify transcript isoform diversity within samples or between conditions. Additionally, the package analyzes the isoform diversity data, looking for significant changes between conditions.\n",
      "Version: 1.6.0\n",
      "Publication date: None\n",
      "Mantainer: Endre Sebestyen <endre.sebestyen@gmail.com>\n",
      "Authors:\n",
      "Author: Peter A. Szikora [aut], Tamas Por [aut], Endre Sebestyen [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3 + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/SplicingFactory.html\n",
      "\n",
      "Name: tenXplore\n",
      "Description:\n",
      "Perform ontological exploration of scRNA-seq of 1.3 million mouse neurons from 10x genomics.\n",
      "Version: 1.20.0\n",
      "Publication date: None\n",
      "Mantainer: VJ Carey <stvjc@channing.harvard.edu>\n",
      "Authors:\n",
      "Author: Vince Carey\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - shiny, type: depends\n",
      "  - restfulSE, type: depends\n",
      "  - methods, type: imports\n",
      "  - ontoProc, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - AnnotationDbi, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - org.Mm.eg.db, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/tenXplore.html\n",
      "\n",
      "Name: TFHAZ\n",
      "Description:\n",
      "It finds trascription factor (TF) high accumulation DNA zones, i.e., regions along the genome where there is a high presence of different transcription factors. Starting from a dataset containing the genomic positions of TF binding regions, for each base of the selected chromosome the accumulation of TFs is computed. Three different types of accumulation (TF, region and base accumulation) are available, together with the possibility of considering, in the single base accumulation computing, the TFs present not only in that single base, but also in its neighborhood, within a window of a given width. Two different methods for the search of TF high accumulation DNA zones, called \"binding regions\" and \"overlaps\", are available. In addition, some functions are provided in order to analyze, visualize and compare results obtained with different input parameters.\n",
      "Version: 1.20.0\n",
      "Publication date: None\n",
      "Mantainer: Gaia Ceddia <gaia.ceddia@polimi.it>\n",
      "Authors:\n",
      "Author: Alberto Marchesi, Silvia Cascianelli, Marco Masseroli\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - GenomicRanges, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - methods, type: imports\n",
      "  - ORFik, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/TFHAZ.html\n",
      "\n",
      "Name: traviz\n",
      "Description:\n",
      "traviz provides a suite of functions to plot trajectory related objects from Bioconductor packages. It allows plotting trajectories in reduced dimension, as well as averge gene expression smoothers as a function of pseudotime. Asides from general utility functions, traviz also allows plotting trajectories estimated by Slingshot, as well as smoothers estimated by tradeSeq. Furthermore, it allows for visualization of Slingshot trajectories using ggplot2.\n",
      "Version: 1.4.0\n",
      "Publication date: None\n",
      "Mantainer: Koen Van den Berge <koen.vdberge@gmail.com>\n",
      "Authors:\n",
      "Author: Hector Roux de Bezieux [aut, ctb], Kelly Street [aut, ctb], Koen Van den Berge [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - ggplot2, type: imports\n",
      "  - viridis, type: imports\n",
      "  - mgcv, type: imports\n",
      "  - SingleCellExperiment, type: imports\n",
      "  - slingshot, type: imports\n",
      "  - princurve, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - methods, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - rgl, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/traviz.html\n",
      "\n",
      "Name: tripr\n",
      "Description:\n",
      "TRIP is a software framework that provides analytics services on antigen receptor (B cell receptor immunoglobulin, BcR IG | T cell receptor, TR) gene sequence data. It is a web application written in R Shiny. It takes as input the output files of the IMGT/HighV-Quest tool. Users can select to analyze the data from each of the input samples separately, or the combined data files from all samples and visualize the results accordingly.\n",
      "Version: 1.4.0\n",
      "Publication date: None\n",
      "Mantainer: Nikolaos Pechlivanis <inab.bioinformatics@lists.certh.gr>\n",
      "Authors:\n",
      "Author: Maria Th. Kotouza [aut], Katerina Gemenetzi [aut], Chrysi Galigalidou [aut], Elisavet Vlachonikola [aut], Nikolaos Pechlivanis [cre], Andreas Agathangelidis [aut], Raphael Sandaltzopoulos [aut], Pericles A. Mitkas [aut], Kostas Stamatopoulos [aut], Anastasia Chatzidimitriou [aut], Fotis E. Psomopoulos [aut], Iason Ofeidis [aut]\n",
      "Dependencies:\n",
      "  - shiny (>= 1.6.0) type: depends\n",
      "  - shinyBS, type: depends\n",
      "  - shinyjs, type: imports\n",
      "  - shinyFiles, type: imports\n",
      "  - plyr, type: imports\n",
      "  - data.table, type: imports\n",
      "  - DT, type: imports\n",
      "  - stringr, type: imports\n",
      "  - stringdist, type: imports\n",
      "  - plot3D, type: imports\n",
      "  - gridExtra, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - plotly, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - pryr, type: imports\n",
      "  - config (>= 0.3.1) type: imports\n",
      "  - golem (>= 0.3.1) type: imports\n",
      "  - methods, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/tripr.html\n",
      "\n",
      "Name: uncoverappLib\n",
      "Description:\n",
      "a Shiny application containing a suite of graphical and statistical tools to support clinical assessment of low coverage regions.It displays three web pages each providing a different analysis module: Coverage analysis, calculate AF by allele frequency app and binomial distribution. uncoverAPP provides a statisticl summary of coverage given target file or genes name.\n",
      "Version: 1.7.0\n",
      "Publication date: None\n",
      "Mantainer: Emanuela Iovino <emanuela.iovino@unibo.it>\n",
      "Authors:\n",
      "Author: Emanuela Iovino [cre, aut], Tommaso Pippucci [aut]\n",
      "Dependencies:\n",
      "  - markdown, type: imports\n",
      "  - shiny, type: imports\n",
      "  - shinyjs, type: imports\n",
      "  - shinyBS, type: imports\n",
      "  - shinyWidgets, type: imports\n",
      "  - shinycssloaders, type: imports\n",
      "  - DT, type: imports\n",
      "  - Gviz, type: imports\n",
      "  - Homo.sapiens, type: imports\n",
      "  - openxlsx, type: imports\n",
      "  - condformat, type: imports\n",
      "  - stringr, type: imports\n",
      "  - org.Hs.eg.db, type: imports\n",
      "  - TxDb.Hsapiens.UCSC.hg38.knownGene, type: imports\n",
      "  - BiocFileCache, type: imports\n",
      "  - rappdirs, type: imports\n",
      "  - TxDb.Hsapiens.UCSC.hg19.knownGene, type: imports\n",
      "  - rlist, type: imports\n",
      "  - utils, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - EnsDb.Hsapiens.v75, type: imports\n",
      "  - EnsDb.Hsapiens.v86, type: imports\n",
      "  - OrganismDbi, type: imports\n",
      "  - BSgenome.Hsapiens.UCSC.hg19, type: imports\n",
      "  - BSgenome.Hsapiens.UCSC.hg38, type: imports\n",
      "  - processx, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/uncoverappLib.html\n",
      "\n",
      "Name: beer\n",
      "Description:\n",
      "BEER implements a Bayesian model for analyzing phage-immunoprecipitation sequencing (PhIP-seq) data. Given a PhIPData object, BEER returns posterior probabilities of enriched antibody responses, point estimates for the relative fold-change in comparison to negative control samples, and more. Additionally, BEER provides a convenient implementation for using edgeR to identify enriched antibody responses.\n",
      "Version: 1.2.0\n",
      "Publication date: None\n",
      "Mantainer: Athena Chen <achen70@jhu.edu>\n",
      "Authors:\n",
      "Author: Athena Chen [aut, cre] , Rob Scharpf [aut], Ingo Ruczinski [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.2.0) type: depends\n",
      "  - PhIPData, type: depends\n",
      "  - rjags, type: depends\n",
      "  - cli, type: imports\n",
      "  - edgeR, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - methods, type: imports\n",
      "  - progressr, type: imports\n",
      "  - stats, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/beer.html\n",
      "\n",
      "Name: biodbLipidmaps\n",
      "Description:\n",
      "The biodbLipidmaps library provides access to the Lipidmaps Structure Database, using biodb package framework. It allows to retrieve entries by their accession number, and run web the services lmsdSearch and lmsdRecord.\n",
      "Version: 1.4.1\n",
      "Publication date: None\n",
      "Mantainer: Pierrick Roger <pierrick.roger@cea.fr>\n",
      "Authors:\n",
      "Author: Pierrick Roger [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - biodb, type: imports\n",
      "  - lifecycle, type: imports\n",
      "  - R6, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "AGPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/biodbLipidmaps.html\n",
      "\n",
      "Name: censcyt\n",
      "Description:\n",
      "Methods for differential abundance analysis in high-dimensional cytometry data when a covariate is subject to right censoring (e.g. survival time) based on multiple imputation and generalized linear mixed models.\n",
      "Version: 1.6.1\n",
      "Publication date: None\n",
      "Mantainer: Reto Gerber <gerberreto@pm.me>\n",
      "Authors:\n",
      "Author: Reto Gerber [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - diffcyt, type: depends\n",
      "  - BiocParallel, type: imports\n",
      "  - broom.mixed, type: imports\n",
      "  - dirmult, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - edgeR, type: imports\n",
      "  - fitdistrplus, type: imports\n",
      "  - lme4, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - MASS, type: imports\n",
      "  - methods, type: imports\n",
      "  - mice, type: imports\n",
      "  - multcomp, type: imports\n",
      "  - purrr, type: imports\n",
      "  - rlang, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - stats, type: imports\n",
      "  - stringr, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - survival, type: imports\n",
      "  - tibble, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/censcyt.html\n",
      "\n",
      "Name: dStruct\n",
      "Description:\n",
      "dStruct identifies differentially reactive regions from RNA structurome profiling data. dStruct is compatible with a broad range of structurome profiling technologies, e.g., SHAPE-MaP, DMS-MaPseq, Structure-Seq, SHAPE-Seq, etc. See Choudhary et al, Genome Biology, 2019 for the underlying method.\n",
      "Version: 1.4.0\n",
      "Publication date: None\n",
      "Mantainer: Krishna Choudhary <kchoudhary@ucdavis.edu>\n",
      "Authors:\n",
      "Author: Krishna Choudhary [aut, cre] , Sharon Aviran [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - zoo, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - purrr, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - parallel, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - rlang, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/dStruct.html\n",
      "\n",
      "Name: fastreeR\n",
      "Description:\n",
      "Calculate distances, build phylogenetic trees or perform hierarchical clustering between the samples of a VCF or FASTA file. Functions are implemented in Java and called via rJava. Parallel implementation that operates directly on the VCF or FASTA file for fast execution.\n",
      "Version: 1.2.0\n",
      "Publication date: None\n",
      "Mantainer: Anestis Gkanogiannis <anestis@gkanogiannis.com>\n",
      "Authors:\n",
      "Author: Anestis Gkanogiannis [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.2) type: depends\n",
      "  - ape, type: imports\n",
      "  - data.table, type: imports\n",
      "  - dynamicTreeCut, type: imports\n",
      "  - methods, type: imports\n",
      "  - R.utils, type: imports\n",
      "  - rJava, type: imports\n",
      "  - stats, type: imports\n",
      "  - stringr, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/fastreeR.html\n",
      "\n",
      "Name: levi\n",
      "Description:\n",
      "The tool integrates data from biological networks with transcriptomes, displaying a heatmap with surface curves to evidence the altered regions.\n",
      "Version: 1.16.0\n",
      "Publication date: None\n",
      "Mantainer: Jose Luiz Rybarczyk Filho <jose.luiz@unesp.br>\n",
      "Authors:\n",
      "Author: Rafael Pilan [aut], Isabelle Silva [ctb], Agnes Takeda [ctb], Jose Rybarczyk Filho [ctb, cre, ths]\n",
      "Dependencies:\n",
      "  - DT (>= 0.4) type: imports\n",
      "  - RColorBrewer (>= 1.1-2) type: imports\n",
      "  - colorspace (>= 1.3-2) type: imports\n",
      "  - dplyr (>= 0.7.4) type: imports\n",
      "  - ggplot2 (>= 2.2.1) type: imports\n",
      "  - httr (>= 1.3.1) type: imports\n",
      "  - igraph (>= 1.2.1) type: imports\n",
      "  - reshape2 (>= 1.4.3) type: imports\n",
      "  - shiny (>= 1.0.5) type: imports\n",
      "  - shinydashboard (>= 0.7.0) type: imports\n",
      "  - shinyjs (>= 1.0) type: imports\n",
      "  - xml2 (>= 1.2.0) type: imports\n",
      "  - knitr, type: imports\n",
      "  - Rcpp (>= 0.12.18) type: imports\n",
      "  - grid, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - testthat, type: imports\n",
      "  - methods, type: imports\n",
      "  - rmarkdown, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/levi.html\n",
      "\n",
      "Name: m6Aboost\n",
      "Description:\n",
      "This package can help user to run the m6Aboost model on their own miCLIP2 data. The package includes functions to assign the read counts and get the features to run the m6Aboost model. The miCLIP2 data should be stored in a GRanges object. More details can be found in the vignette.\n",
      "Version: 1.4.0\n",
      "Publication date: None\n",
      "Mantainer: You Zhou <youzhoulearning@gmail.com>\n",
      "Authors:\n",
      "Author: You Zhou [aut, cre] , Kathi Zarnack [aut]\n",
      "Dependencies:\n",
      "  - S4Vectors, type: depends\n",
      "  - adabag, type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - R (>= 4.1) type: depends\n",
      "  - dplyr, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - BSgenome, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - utils, type: imports\n",
      "  - methods, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - ExperimentHub, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/m6Aboost.html\n",
      "\n",
      "Name: MACSQuantifyR\n",
      "Description:\n",
      "Automatically process the metadata of MACSQuantify FACS sorter. It runs multiple modules: i) imports of raw file and graphical selection of duplicates in well plate, ii) computes statistics on data and iii) can compute combination index.\n",
      "Version: 1.12.0\n",
      "Publication date: None\n",
      "Mantainer: RaphaÃ«l Bonnet <raphael.bonnet@univ-cotedazur.fr>\n",
      "Authors:\n",
      "Author: RaphaÃ«l Bonnet [aut, cre], Marielle Nebout [dtc],Giulia Biondani [dtc], Jean-FranÃ§ois Peyron[aut,ths], Inserm [fnd]\n",
      "Dependencies:\n",
      "  - readxl, type: imports\n",
      "  - graphics, type: imports\n",
      "  - tools, type: imports\n",
      "  - utils, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - ggrepel, type: imports\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - latticeExtra, type: imports\n",
      "  - lattice, type: imports\n",
      "  - rmarkdown, type: imports\n",
      "  - png, type: imports\n",
      "  - grid, type: imports\n",
      "  - gridExtra, type: imports\n",
      "  - prettydoc, type: imports\n",
      "  - rvest, type: imports\n",
      "  - xml2, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/MACSQuantifyR.html\n",
      "\n",
      "Name: MQmetrics\n",
      "Description:\n",
      "The package MQmetrics (MaxQuant metrics) provides a workflow to analyze the quality and reproducibility of your proteomics mass spectrometry analysis from MaxQuant.Input data are extracted from several MaxQuant output tables and produces a pdf report. It includes several visualization tools to check numerous parameters regarding the quality of the runs. It also includes two functions to visualize the iRT peptides from Biognosys in case they were spiked in the samples.\n",
      "Version: 1.6.0\n",
      "Publication date: None\n",
      "Mantainer: Alvaro Sanchez-Villalba <sanchezvillalba.alvaro@gmail.com>\n",
      "Authors:\n",
      "Author: Alvaro Sanchez-Villalba [aut, cre], Thomas Stehrer [aut], Marek Vrbacky [aut]\n",
      "Dependencies:\n",
      "  - ggplot2, type: imports\n",
      "  - readr, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - purrr, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - gridExtra, type: imports\n",
      "  - utils, type: imports\n",
      "  - stringr, type: imports\n",
      "  - ggpubr, type: imports\n",
      "  - stats, type: imports\n",
      "  - cowplot, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - scales, type: imports\n",
      "  - grid, type: imports\n",
      "  - rlang, type: imports\n",
      "  - ggforce, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - gtable, type: imports\n",
      "  - plyr, type: imports\n",
      "  - knitr, type: imports\n",
      "  - rmarkdown, type: imports\n",
      "  - ggrepel, type: imports\n",
      "  - gghalves, type: imports\n",
      "  - tools, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/MQmetrics.html\n",
      "\n",
      "Name: MSstatsLOBD\n",
      "Description:\n",
      "The MSstatsLOBD package allows calculation and visualization of limit of blac (LOB) and limit of detection (LOD). We define the LOB as the highest apparent concentration of a peptide expected when replicates of a blank sample containing no peptides are measured. The LOD is defined as the measured concentration value for which the probability of falsely claiming the absence of a peptide in the sample is 0.05, given a probability 0.05 of falsely claiming its presence. These functionalities were previously a part of the MSstats package. The methodology is described in Galitzine (2018) .\n",
      "Version: 1.6.0\n",
      "Publication date: None\n",
      "Mantainer: Devon Kohler <kohler.d@northeastern.edu>\n",
      "Authors:\n",
      "Author: Devon Kohler [aut, cre], Mateusz Staniak [aut], Cyril Galitzine [aut], Meena Choi [aut], Olga Vitek [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - minpack.lm, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - utils, type: imports\n",
      "  - stats, type: imports\n",
      "  - grDevices, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/MSstatsLOBD.html\n",
      "\n",
      "Name: netboost\n",
      "Description:\n",
      "Boosting supported network analysis for high-dimensional omics applications. This package comes bundled with the MC-UPGMA clustering package by Yaniv Loewenstein.\n",
      "Version: 2.6.1\n",
      "Publication date: None\n",
      "Mantainer: Pascal Schlosser <pascal.schlosser@uniklinik-freiburg.de>\n",
      "Authors:\n",
      "Author: Pascal Schlosser [aut, cre], Jochen Knaus [aut, ctb], Yaniv Loewenstein [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.0.0) type: depends\n",
      "  - Rcpp, type: imports\n",
      "  - RcppParallel, type: imports\n",
      "  - parallel, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - dynamicTreeCut, type: imports\n",
      "  - WGCNA, type: imports\n",
      "  - impute, type: imports\n",
      "  - colorspace, type: imports\n",
      "  - methods, type: imports\n",
      "  - R.utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/netboost.html\n",
      "\n",
      "Name: PhenoGeneRanker\n",
      "Description:\n",
      "This package is a gene/phenotype prioritization tool that utilizes multiplex heterogeneous gene phenotype network. PhenoGeneRanker allows multi-layer gene and phenotype networks. It also calculates empirical p-values of gene/phenotype ranking using random stratified sampling of genes/phenotypes based on their connectivity degree in the network. https://dl.acm.org/doi/10.1145/3307339.3342155.\n",
      "Version: 1.6.0\n",
      "Publication date: None\n",
      "Mantainer: Cagatay Dursun <cagataydursun@gmail.com>\n",
      "Authors:\n",
      "Author: Cagatay Dursun [aut, cre]\n",
      "Dependencies:\n",
      "  - igraph, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - foreach, type: imports\n",
      "  - doParallel, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - parallel, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Creative Commons Attribution 4.0 International License\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/PhenoGeneRanker.html\n",
      "\n",
      "Name: receptLoss\n",
      "Description:\n",
      "receptLoss identifies genes whose expression is lost in subsets of tumors relative to normal tissue. It is particularly well-suited in cases where the number of normal tissue samples is small, as the distribution of gene expression in normal tissue samples is approximated by a Gaussian. Originally designed for identifying nuclear hormone receptor expression loss but can be applied transcriptome wide as well.\n",
      "Version: 1.10.0\n",
      "Publication date: None\n",
      "Mantainer: Daniel Pique <daniel.pique@med.einstein.yu.edu>\n",
      "Authors:\n",
      "Author: Daniel Pique, John Greally, Jessica Mar\n",
      "Dependencies:\n",
      "  - R (>= 3.6.0) type: depends\n",
      "  - dplyr, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3 + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/receptLoss.html\n",
      "\n",
      "Name: rgoslin\n",
      "Description:\n",
      "The R implementation for the Grammar of Succint Lipid Nomenclature parses different short hand notation dialects for lipid names. It normalizes them to a standard name. It further provides calculated monoisotopic masses and sum formulas for each successfully parsed lipid name and supplements it with LIPID MAPS Category and Class information. Also, the structural level and further structural details about the head group, fatty acyls and functional groups are returned, where applicable.\n",
      "Version: 1.2.0\n",
      "Publication date: None\n",
      "Mantainer: Nils Hoffmann <nils.hoffmann@cebitec.uni-bielefeld.de>\n",
      "Authors:\n",
      "Author: Nils Hoffmann [aut, cre] , Dominik Kopczynski [aut]\n",
      "Dependencies:\n",
      "  - Rcpp (>= 1.0.3) type: imports\n",
      "  - dplyr, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/rgoslin.html\n",
      "\n",
      "Name: ROCpAI\n",
      "Description:\n",
      "The package analyzes the Curve ROC, identificates it among different types of Curve ROC and calculates the area under de curve through the method that is most accuracy. This package is able to standarizate proper and improper pAUC.\n",
      "Version: 1.10.0\n",
      "Publication date: None\n",
      "Mantainer: Juan-Pedro Garcia <juanpedro.garcia4@um.es>\n",
      "Authors:\n",
      "Author: Juan-Pedro Garcia [aut, cre], Manuel Franco [aut], Juana-MarÃ­a Vivo [aut]\n",
      "Dependencies:\n",
      "  - boot, type: depends\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - fission, type: depends\n",
      "  - knitr, type: depends\n",
      "  - methods, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ROCpAI.html\n",
      "\n",
      "Name: rsemmed\n",
      "Description:\n",
      "A programmatic interface to the Semantic MEDLINE database. It provides functions for searching the database for concepts and finding paths between concepts. Path searching can also be tailored to user specifications, such as placing restrictions on concept types and the type of link between concepts. It also provides functions for summarizing and visualizing those paths.\n",
      "Version: 1.8.0\n",
      "Publication date: None\n",
      "Mantainer: Leslie Myint <leslie.myint@gmail.com>\n",
      "Authors:\n",
      "Author: Leslie Myint [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - igraph, type: depends\n",
      "  - methods, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - stringr, type: imports\n",
      "  - dplyr, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/rsemmed.html\n",
      "\n",
      "Name: sarks\n",
      "Description:\n",
      "Suffix Array Kernel Smoothing (see https://academic.oup.com/bioinformatics/article-abstract/35/20/3944/5418797), or SArKS, identifies sequence motifs whose presence correlates with numeric scores (such as differential expression statistics) assigned to the sequences (such as gene promoters). SArKS smooths over sequence similarity, quantified by location within a suffix array based on the full set of input sequences. A second round of smoothing over spatial proximity within sequences reveals multi-motif domains. Discovered motifs can then be merged or extended based on adjacency within MMDs. False positive rates are estimated and controlled by permutation testing.\n",
      "Version: 1.10.0\n",
      "Publication date: None\n",
      "Mantainer: Dennis Wylie <denniscwylie@gmail.com>\n",
      "Authors:\n",
      "Author: Dennis Wylie [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - rJava, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - utils, type: imports\n",
      "  - stats, type: imports\n",
      "  - cluster, type: imports\n",
      "  - binom, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "BSD_3_clause + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/sarks.html\n",
      "\n",
      "Name: scReClassify\n",
      "Description:\n",
      "A post hoc cell type classification tool to fine-tune cell type annotations generated by any cell type classification procedure with semi-supervised learning algorithm AdaSampling technique. The current version of scReClassify supports Support Vector Machine and Random Forest as a base classifier.\n",
      "Version: 1.4.0\n",
      "Publication date: None\n",
      "Mantainer: Taiyun Kim <taiyun.kim91@gmail.com>\n",
      "Authors:\n",
      "Author: Pengyi Yang [aut] , Taiyun Kim [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - randomForest, type: imports\n",
      "  - e1071, type: imports\n",
      "  - stats, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - SingleCellExperiment, type: imports\n",
      "  - methods, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3 + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/scReClassify.html\n",
      "\n",
      "Name: scShapes\n",
      "Description:\n",
      "We present a novel statistical framework for identifying differential distributions in single-cell RNA-sequencing (scRNA-seq) data between treatment conditions by modeling gene expression read counts using generalized linear models (GLMs). We model each gene independently under each treatment condition using error distributions Poisson (P), Negative Binomial (NB), Zero-inflated Poisson (ZIP) and Zero-inflated Negative Binomial (ZINB) with log link function and model based normalization for differences in sequencing depth. Since all four distributions considered in our framework belong to the same family of distributions, we first perform a Kolmogorov-Smirnov (KS) test to select genes belonging to the family of ZINB distributions. Genes passing the KS test will be then modeled using GLMs. Model selection is done by calculating the Bayesian Information Criterion (BIC) and likelihood ratio test (LRT) statistic.\n",
      "Version: 1.4.0\n",
      "Publication date: None\n",
      "Mantainer: Malindrie Dharmaratne <malindrie@gmail.com>\n",
      "Authors:\n",
      "Author: Malindrie Dharmaratne [cre, aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - Matrix, type: imports\n",
      "  - stats, type: imports\n",
      "  - methods, type: imports\n",
      "  - pscl, type: imports\n",
      "  - VGAM, type: imports\n",
      "  - dgof, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - MASS, type: imports\n",
      "  - emdbook, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/scShapes.html\n",
      "\n",
      "Name: seqArchR\n",
      "Description:\n",
      "seqArchR enables unsupervised discovery of _de novo_ clusters with characteristic sequence architectures characterized by position-specific motifs or composition of stretches of nucleotides, e.g., CG-richness. seqArchR does _not_ require any specifications w.r.t. the number of clusters, the length of any individual motifs, or the distance between motifs if and when they occur in pairs/groups; it directly detects them from the data. seqArchR uses non-negative matrix factorization (NMF) as its backbone, and employs a chunking-based iterative procedure that enables processing of large sequence collections efficiently. Wrapper functions are provided for visualizing cluster architectures as sequence logos.\n",
      "Version: 1.2.0\n",
      "Publication date: None\n",
      "Mantainer: Sarvesh Nikumbh <sarvesh.nikumbh@gmail.com>\n",
      "Authors:\n",
      "Author: Sarvesh Nikumbh [aut, cre, cph]\n",
      "Dependencies:\n",
      "  - R (>= 4.2.0) type: depends\n",
      "  - utils, type: imports\n",
      "  - graphics, type: imports\n",
      "  - cvTools (>= 0.3.2) type: imports\n",
      "  - MASS, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - cluster, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - fpc, type: imports\n",
      "  - cli, type: imports\n",
      "  - prettyunits, type: imports\n",
      "  - reshape2 (>= 1.4.3) type: imports\n",
      "  - reticulate (>= 1.22) type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - ggplot2 (>= 3.1.1) type: imports\n",
      "  - ggseqlogo (>= 0.1) type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3 | file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/seqArchR.html\n",
      "\n",
      "Name: SICtools\n",
      "Description:\n",
      "This package is to find SNV/Indel differences between two bam files with near relationship in a way of pairwise comparison thourgh each base position across the genome region of interest. The difference is inferred by fisher test and euclidean distance, the input of which is the base count (A,T,G,C) in a given position and read counts for indels that span no less than 2bp on both sides of indel region.\n",
      "Version: 1.28.0\n",
      "Publication date: None\n",
      "Mantainer: Xiaobin Xing <xiaobinxing0316@gmail.com>\n",
      "Authors:\n",
      "Author: Xiaobin Xing, Wu Wei\n",
      "Dependencies:\n",
      "  - R (>= 3.0.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - Rsamtools, type: depends\n",
      "  - doParallel (>= 1.0.8) type: depends\n",
      "  - Biostrings, type: depends\n",
      "  - stringr (>= 0.6.2) type: depends\n",
      "  - matrixStats (>= 0.10.0) type: depends\n",
      "  - plyr (>= 1.8.3) type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - IRanges, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>=2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/SICtools.html\n",
      "\n",
      "Name: SQUADD\n",
      "Description:\n",
      "This package SQUADD is a SQUAD add-on. It permits to generate SQUAD simulation matrix, prediction Heat-Map and Correlation Circle from PCA analysis.\n",
      "Version: 1.48.0\n",
      "Publication date: None\n",
      "Mantainer: Martial Sankar <martial.sankar@sib.swiss>\n",
      "Authors:\n",
      "Author: Martial Sankar, supervised by Christian Hardtke and Ioannis Xenarios\n",
      "Dependencies:\n",
      "  - R (>= 2.11.0) type: depends\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - methods, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>=2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/SQUADD.html\n",
      "\n",
      "Name: ssPATHS\n",
      "Description:\n",
      "This package generates pathway scores from expression data for single samples after training on a reference cohort. The score is generated by taking the expression of a gene set (pathway) from a reference cohort and performing linear discriminant analysis to distinguish samples in the cohort that have the pathway augmented and not. The separating hyperplane is then used to score new samples.\n",
      "Version: 1.12.0\n",
      "Publication date: None\n",
      "Mantainer: Natalie R. Davidson <natalie.davidson@inf.ethz.ch>\n",
      "Authors:\n",
      "Author: Natalie R. Davidson\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - ROCR, type: imports\n",
      "  - dml, type: imports\n",
      "  - MESS, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ssPATHS.html\n",
      "\n",
      "Name: standR\n",
      "Description:\n",
      "standR is an user-friendly R package providing functions to assist conducting good-practice analysis of Nanostring's GeoMX DSP data. All functions in the package are built based on the SpatialExperiment object, allowing integration into various spatial transcriptomics-related packages from Bioconductor. standR allows data inspection, quality control, normalization, batch correction and evaluation with informative visualizations.\n",
      "Version: 1.2.2\n",
      "Publication date: None\n",
      "Mantainer: Ning Liu <liu.n@wehi.edu.au>\n",
      "Authors:\n",
      "Author: Ning Liu [aut, cre] , Dharmesh D Bhuva [aut] , Ahmed Mohamed [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - dplyr, type: imports\n",
      "  - SpatialExperiment, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - SingleCellExperiment, type: imports\n",
      "  - edgeR, type: imports\n",
      "  - rlang, type: imports\n",
      "  - readr, type: imports\n",
      "  - tibble, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - ruv, type: imports\n",
      "  - limma, type: imports\n",
      "  - patchwork, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - stats, type: imports\n",
      "  - methods, type: imports\n",
      "  - ggalluvial, type: imports\n",
      "  - mclustcomp, type: imports\n",
      "  - RUVSeq, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/standR.html\n",
      "\n",
      "Name: surfaltr\n",
      "Description:\n",
      "Cell surface proteins form a major fraction of the druggable proteome and can be used for tissue-specific delivery of oligonucleotide/cell-based therapeutics. Alternatively spliced surface protein isoforms have been shown to differ in their subcellular localization and/or their transmembrane (TM) topology. Surface proteins are hydrophobic and remain difficult to study thereby necessitating the use of TM topology prediction methods such as TMHMM and Phobius. However, there exists a need for bioinformatic approaches to streamline batch processing of isoforms for comparing and visualizing topologies. To address this gap, we have developed an R package, surfaltr. It pairs inputted isoforms, either known alternatively spliced or novel, with their APPRIS annotated principal counterparts, predicts their TM topologies using TMHMM or Phobius, and generates a customizable graphical output. Further, surfaltr facilitates the prioritization of biologically diverse isoform pairs through the incorporation of three different ranking metrics and through protein alignment functions. Citations for programs mentioned here can be found in the vignette.\n",
      "Version: 1.4.0\n",
      "Publication date: None\n",
      "Mantainer: Pooja Gangras <gangras_pooja@lilly.com>\n",
      "Authors:\n",
      "Author: Pooja Gangras [aut, cre] , Aditi Merchant [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - dplyr (>= 1.0.6) type: imports\n",
      "  - biomaRt, type: imports\n",
      "  - protr (>= 1.6-2) type: imports\n",
      "  - seqinr (>= 4.2-5) type: imports\n",
      "  - ggplot2 (>= 3.3.2) type: imports\n",
      "  - utils (>= 2.10.1) type: imports\n",
      "  - stringr (>= 1.4.0) type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - readr (>= 1.4.0) type: imports\n",
      "  - httr (>= 1.4.2) type: imports\n",
      "  - testthat (>= 3.0.0) type: imports\n",
      "  - xml2 (>= 1.3.2) type: imports\n",
      "  - msa, type: imports\n",
      "  - methods (>= 4.0.3) type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/surfaltr.html\n",
      "\n",
      "Name: synapsis\n",
      "Description:\n",
      "Synapsis is a Bioconductor software package for automated (unbiased and reproducible) analysis of meiotic immunofluorescence datasets. The primary functions of the software can i) identify cells in meiotic prophase that are labelled by a synaptonemal complex axis or central element protein, ii) isolate individual synaptonemal complexes and measure their physical length, iii) quantify foci and co-localise them with synaptonemal complexes, iv) measure interference between synaptonemal complex-associated foci. The software has applications that extend to multiple species and to the analysis of other proteins that label meiotic prophase chromosomes. The software converts meiotic immunofluorescence images into R data frames that are compatible with machine learning methods. Given a set of microscopy images of meiotic spread slides, synapsis crops images around individual single cells, counts colocalising foci on strands on a per cell basis, and measures the distance between foci on any given strand.\n",
      "Version: 1.4.0\n",
      "Publication date: None\n",
      "Mantainer: Lucy McNeill <luc.mcneill@gmail.com>\n",
      "Authors:\n",
      "Author: Lucy McNeill [aut, cre, cph] , Wayne Crismani [rev, ctb]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - EBImage, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - graphics, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/synapsis.html\n",
      "\n",
      "Name: TFARM\n",
      "Description:\n",
      "It searches for relevant associations of transcription factors with a transcription factor target, in specific genomic regions. It also allows to evaluate the Importance Index distribution of transcription factors (and combinations of transcription factors) in association rules.\n",
      "Version: 1.20.0\n",
      "Publication date: None\n",
      "Mantainer: Liuba Nausicaa Martino <liuban.martino@gmail.com>\n",
      "Authors:\n",
      "Author: Liuba Nausicaa Martino, Alice Parodi, Gaia Ceddia, Piercesare Secchi, Stefano Campaner, Marco Masseroli\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - arules, type: imports\n",
      "  - fields, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stringr, type: imports\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - gplots, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/TFARM.html\n",
      "\n",
      "Name: txcutr\n",
      "Description:\n",
      "Various mRNA sequencing library preparation methods generate sequencing reads specifically from the transcript ends. Analyses that focus on quantification of isoform usage from such data can be aided by using truncated versions of transcriptome annotations, both at the alignment or pseudo-alignment stage, as well as in downstream analysis. This package implements some convenience methods for readily generating such truncated annotations and their corresponding sequences.\n",
      "Version: 1.4.0\n",
      "Publication date: None\n",
      "Mantainer: Mervin Fansler <mef3005@med.cornell.edu>\n",
      "Authors:\n",
      "Author: Mervin Fansler [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.1.0) type: depends\n",
      "  - AnnotationDbi, type: imports\n",
      "  - GenomicFeatures, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - stats, type: imports\n",
      "  - methods, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/txcutr.html\n",
      "\n",
      "Name: cogeqc\n",
      "Description:\n",
      "cogeqc aims to facilitate systematic quality checks on standard comparative genomics analyses to help researchers detect issues and select the most suitable parameters for each data set. cogeqc can be used to asses: i. genome assembly quality with BUSCOs; ii. orthogroup inference using a protein domain-based approach and; iii. synteny detection using synteny network properties. There are also data visualization functions to explore QC summary statistics.\n",
      "Version: 1.2.1\n",
      "Publication date: None\n",
      "Mantainer: FabrÃ­cio Almeida-Silva <fabricio_almeidasilva@hotmail.com>\n",
      "Authors:\n",
      "Author: FabrÃ­cio Almeida-Silva [aut, cre] , Yves Van de Peer [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.2.0) type: depends\n",
      "  - utils, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - methods, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - ggtree, type: imports\n",
      "  - patchwork, type: imports\n",
      "  - igraph, type: imports\n",
      "  - Biostrings, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/cogeqc.html\n",
      "\n",
      "Name: coMethDMR\n",
      "Description:\n",
      "coMethDMR identifies genomic regions associated with continuous phenotypes by optimally leverages covariations among CpGs within predefined genomic regions. Instead of testing all CpGs within a genomic region, coMethDMR carries out an additional step that selects co-methylated sub-regions first without using any outcome information. Next, coMethDMR tests association between methylation within the sub-region and continuous phenotype using a random coefficient mixed effects model, which models both variations between CpG sites within the region and differential methylation simultaneously.\n",
      "Version: 1.2.0\n",
      "Publication date: None\n",
      "Mantainer: Fernanda Veitzman <fveit001@fiu.edu>\n",
      "Authors:\n",
      "Author: Fernanda Veitzman [cre], Lissette Gomez [aut], Tiago Silva [aut], Ning Lijiao [ctb], Boissel Mathilde [ctb], Lily Wang [aut], Gabriel Odom [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - AnnotationHub, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - bumphunter, type: imports\n",
      "  - ExperimentHub, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - lmerTest, type: imports\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/coMethDMR.html\n",
      "\n",
      "Name: conclus\n",
      "Description:\n",
      "CONCLUS is a tool for robust clustering and positive marker features selection of single-cell RNA-seq (sc-RNA-seq) datasets. It takes advantage of a consensus clustering approach that greatly simplify sc-RNA-seq data analysis for the user. Of note, CONCLUS does not cover the preprocessing steps of sequencing files obtained following next-generation sequencing. CONCLUS is organized into the following steps: Generation of multiple t-SNE plots with a range of parameters including different selection of genes extracted from PCA. Use the Density-based spatial clustering of applications with noise (DBSCAN) algorithm for idenfication of clusters in each generated t-SNE plot. All DBSCAN results are combined into a cell similarity matrix. The cell similarity matrix is used to define \"CONSENSUS\" clusters conserved accross the previously defined clustering solutions. Identify marker genes for each concensus cluster.\n",
      "Version: 1.5.0\n",
      "Publication date: None\n",
      "Mantainer: Ilyess Rachedi <rachedi.ilyess@gmail.com>\n",
      "Authors:\n",
      "Author: Ilyess Rachedi [cre], Nicolas Descostes [aut], Polina Pavlovich [aut], Christophe Lancrin [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - org.Hs.eg.db, type: imports\n",
      "  - org.Mm.eg.db, type: imports\n",
      "  - dbscan, type: imports\n",
      "  - fpc, type: imports\n",
      "  - factoextra, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - BiocFileCache, type: imports\n",
      "  - parallel, type: imports\n",
      "  - doParallel, type: imports\n",
      "  - foreach, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - biomaRt, type: imports\n",
      "  - AnnotationDbi, type: imports\n",
      "  - methods, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - scran, type: imports\n",
      "  - scater, type: imports\n",
      "  - pheatmap, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - gridExtra, type: imports\n",
      "  - SingleCellExperiment, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - scales, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - Rtsne, type: imports\n",
      "  - GEOquery, type: imports\n",
      "  - clusterProfiler, type: imports\n",
      "  - stringr, type: imports\n",
      "  - tools, type: imports\n",
      "  - rlang, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/conclus.html\n",
      "\n",
      "Name: frenchFISH\n",
      "Description:\n",
      "FrenchFISH comprises a nuclear volume correction method coupled with two types of Poisson models: either a Poisson model for improved manual spot counting without the need for control probes; or a homogenous Poisson Point Process model for automated spot counting.\n",
      "Version: 1.10.0\n",
      "Publication date: None\n",
      "Mantainer: Adam Berman <agb61@cam.ac.uk>\n",
      "Authors:\n",
      "Author: Adam Berman, Geoff Macintyre\n",
      "Dependencies:\n",
      "  - utils, type: imports\n",
      "  - MCMCpack, type: imports\n",
      "  - NHPoisson, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/frenchFISH.html\n",
      "\n",
      "Name: pairkat\n",
      "Description:\n",
      "PaIRKAT is model framework for assessing statistical relationships between networks of metabolites (pathways) and an outcome of interest (phenotype). PaIRKAT queries the KEGG database to determine interactions between metabolites from which network connectivity is constructed. This model framework improves testing power on high dimensional data by including graph topography in the kernel machine regression setting. Studies on high dimensional data can struggle to include the complex relationships between variables. The semi-parametric kernel machine regression model is a powerful tool for capturing these types of relationships. They provide a framework for testing for relationships between outcomes of interest and high dimensional data such as metabolomic, genomic, or proteomic pathways. PaIRKAT uses known biological connections between high dimensional variables by representing them as edges of âgraphsâ or ânetworks.â It is common for nodes (e.g. metabolites) to be disconnected from all others within the graph, which leads to meaningful decreases in testing power whether or not the graph information is included. We include a graph regularization or âsmoothingâ approach for managing this issue.\n",
      "Version: 1.4.0\n",
      "Publication date: None\n",
      "Mantainer: Max McGrath <max.mcgrath@ucdenver.edu>\n",
      "Authors:\n",
      "Author: Charlie Carpenter [aut], Cameron Severn [aut], Max McGrath [cre, aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - KEGGREST, type: imports\n",
      "  - igraph, type: imports\n",
      "  - data.table, type: imports\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - CompQuadForm, type: imports\n",
      "  - tibble, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/pairkat.html\n",
      "\n",
      "Name: pathVar\n",
      "Description:\n",
      "This package contains the functions to find the pathways that have significantly different variability than a reference gene set. It also finds the categories from this pathway that are significant where each category is a cluster of genes. The genes are separated into clusters by their level of variability.\n",
      "Version: 1.28.0\n",
      "Publication date: None\n",
      "Mantainer: Samuel Zimmerman <samuel.e.zimmerman@gmail.com>\n",
      "Authors:\n",
      "Author: Laurence de Torrente, Samuel Zimmerman, Jessica Mar\n",
      "Dependencies:\n",
      "  - R (>= 3.3.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - ggplot2, type: depends\n",
      "  - gridExtra, type: depends\n",
      "  - EMT, type: imports\n",
      "  - mclust, type: imports\n",
      "  - Matching, type: imports\n",
      "  - data.table, type: imports\n",
      "  - stats, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL (>= 2.0)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/pathVar.html\n",
      "\n",
      "Name: pengls\n",
      "Description:\n",
      "Combine generalised least squares methodology from the nlme package for dealing with autocorrelation with penalised least squares methods from the glmnet package to deal with high dimensionality. This pengls packages glues them together through an iterative loop. The resulting method is applicable to high dimensional datasets that exhibit autocorrelation, such as spatial or temporal data.\n",
      "Version: 1.4.0\n",
      "Publication date: None\n",
      "Mantainer: Stijn Hawinkel <stijn.hawinkel@psb.ugent.be>\n",
      "Authors:\n",
      "Author: Stijn Hawinkel [cre, aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.2.0) type: depends\n",
      "  - glmnet, type: imports\n",
      "  - nlme, type: imports\n",
      "  - stats, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/pengls.html\n",
      "\n",
      "Name: PoDCall\n",
      "Description:\n",
      "Reads files exported from 'QuantaSoft' containing amplitude values from a run of ddPCR (96 well plate) and robustly sets thresholds to determine positive droplets for each channel of each individual well. Concentration and normalized concentration in addition to other metrics is then calculated for each well. Results are returned as a table, optionally written to file, as well as optional plots (scatterplot and histogram) for both channels per well written to file. The package includes a shiny application which provides an interactive and user-friendly interface to the full functionality of PoDCall.\n",
      "Version: 1.6.0\n",
      "Publication date: None\n",
      "Mantainer: Hans Petter Brodal <hans.petter.brodal@rr-research.no>\n",
      "Authors:\n",
      "Author: Hans Petter Brodal [aut, cre], Marine Jeanmougin [aut], Guro Elisabeth Lind [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.2) type: depends\n",
      "  - ggplot2, type: imports\n",
      "  - gridExtra, type: imports\n",
      "  - mclust, type: imports\n",
      "  - diptest, type: imports\n",
      "  - rlist, type: imports\n",
      "  - shiny, type: imports\n",
      "  - DT, type: imports\n",
      "  - LaplacesDemon, type: imports\n",
      "  - purrr, type: imports\n",
      "  - shinyjs, type: imports\n",
      "  - readr, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/PoDCall.html\n",
      "\n",
      "Name: SeqGate\n",
      "Description:\n",
      "Filtering of lowly expressed features (e.g. genes) is a common step before performing statistical analysis, but an arbitrary threshold is generally chosen. SeqGate implements a method that rationalize this step by the analysis of the distibution of counts in replicate samples. The gate is the threshold above which sequenced features can be considered as confidently quantified.\n",
      "Version: 1.8.0\n",
      "Publication date: None\n",
      "Mantainer: StÃ©phanie Rialle <stephanie.rialle@mgx.cnrs.fr>\n",
      "Authors:\n",
      "Author: Christelle ReynÃ¨s [aut], StÃ©phanie Rialle [aut, cre]\n",
      "Dependencies:\n",
      "  - S4Vectors, type: depends\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - stats, type: imports\n",
      "  - methods, type: imports\n",
      "  - BiocManager, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2.0)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/SeqGate.html\n",
      "\n",
      "Name: tomoda\n",
      "Description:\n",
      "This package provides many easy-to-use methods to analyze and visualize tomo-seq data. The tomo-seq technique is based on cryosectioning of tissue and performing RNA-seq on consecutive sections. (Reference: Kruse F, Junker JP, van Oudenaarden A, Bakkers J. Tomo-seq: A method to obtain genome-wide expression data with spatial resolution. Methods Cell Biol. 2016;135:299-307. doi:10.1016/bs.mcb.2016.01.006) The main purpose of the package is to find zones with similar transcriptional profiles and spatially expressed genes in a tomo-seq sample. Several visulization functions are available to create easy-to-modify plots.\n",
      "Version: 1.8.0\n",
      "Publication date: None\n",
      "Mantainer: Wendao Liu <liuwd15@tsinghua.org.cn>\n",
      "Authors:\n",
      "Author: Wendao Liu [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.0.0) type: depends\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - Rtsne, type: imports\n",
      "  - umap, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - ggrepel, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/tomoda.html\n",
      "\n",
      "Name: VERSO\n",
      "Description:\n",
      "Mutations that rapidly accumulate in viral genomes during a pandemic can be used to track the evolution of the virus and, accordingly, unravel the viral infection network. To this extent, sequencing samples of the virus can be employed to estimate models from genomic epidemiology and may serve, for instance, to estimate the proportion of undetected infected people by uncovering cryptic transmissions, as well as to predict likely trends in the number of infected, hospitalized, dead and recovered people. VERSO is an algorithmic framework that processes variants profiles from viral samples to produce phylogenetic models of viral evolution. The approach solves a Boolean Matrix Factorization problem with phylogenetic constraints, by maximizing a log-likelihood function. VERSO includes two separate and subsequent steps; in this package we provide an R implementation of VERSO STEP 1.\n",
      "Version: 1.8.0\n",
      "Publication date: None\n",
      "Mantainer: Davide Maspero <d.maspero@campus.unimib.it>\n",
      "Authors:\n",
      "Author: Daniele Ramazzotti [aut] , Fabrizio Angaroni [aut], Davide Maspero [cre, aut], Alex Graudenzi [aut], Luca De Sano [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.1.0) type: depends\n",
      "  - ape, type: imports\n",
      "  - parallel, type: imports\n",
      "  - Rfast, type: imports\n",
      "  - stats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/VERSO.html\n",
      "\n",
      "Name: LinTInd\n",
      "Description:\n",
      "When we combine gene-editing technology and sequencing technology, we need to reconstruct a lineage tree from alleles generated and calculate the similarity between each pair of groups. FindIndel() and IndelForm() function will help you align each read to reference sequence and generate scar form strings respectively. IndelIdents() function will help you to define a scar form for each cell or read. IndelPlot() function will help you to visualize the distribution of deletion and insertion. TagProcess() function will help you to extract indels for each cell or read. TagDist() function will help you to calculate the similarity between each pair of groups across the indwells they contain. BuildTree() function will help you to reconstruct a tree. PlotTree() function will help you to visualize the tree.\n",
      "Version: 1.2.0\n",
      "Publication date: None\n",
      "Mantainer: Luyue Wang <wly1995310@gmail.com>\n",
      "Authors:\n",
      "Author: Luyue Wang [aut, cre], Bin Xiang [ctb], Hengxin Liu [ctb], Wu Wei [ths]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - ggplot2, type: depends\n",
      "  - parallel, type: depends\n",
      "  - stats, type: depends\n",
      "  - S4Vectors, type: depends\n",
      "  - data.tree, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - networkD3, type: imports\n",
      "  - stringdist, type: imports\n",
      "  - purrr, type: imports\n",
      "  - ape, type: imports\n",
      "  - cowplot, type: imports\n",
      "  - ggnewscale, type: imports\n",
      "  - stringr, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - rlist, type: imports\n",
      "  - pheatmap, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - ggtree, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/LinTInd.html\n",
      "\n",
      "Name: ModCon\n",
      "Description:\n",
      "Collection of functions to calculate a nucleotide sequence surrounding for splice donors sites to either activate or repress donor usage. The proposed alternative nucleotide sequence encodes the same amino acid and could be applied e.g. in reporter systems to silence or activate cryptic splice donor sites.\n",
      "Version: 1.6.0\n",
      "Publication date: None\n",
      "Mantainer: Johannes Ptok <Johannes.Ptok@posteo.de>\n",
      "Authors:\n",
      "Author: Johannes Ptok [aut, cre]\n",
      "Dependencies:\n",
      "  - data.table, type: depends\n",
      "  - parallel, type: depends\n",
      "  - utils, type: depends\n",
      "  - stats, type: depends\n",
      "  - R (>= 4.1) type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3 + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ModCon.html\n",
      "\n",
      "Name: ODER\n",
      "Description:\n",
      "The aim of ODER is to identify previously unannotated expressed regions (ERs) using RNA-sequencing data. For this purpose, ODER defines and optimises the definition of ERs, then connected these ERs to genes using junction data. In this way, ODER improves gene annotation. Gene annotation is a staple input of many bioinformatic pipelines and a more complete gene annotation can enable more accurate interpretation of disease associated variants.\n",
      "Version: 1.4.0\n",
      "Publication date: None\n",
      "Mantainer: David Zhang <david.zhang.12@ucl.ac.uk>\n",
      "Authors:\n",
      "Author: Emmanuel Olagbaju [aut], David Zhang [aut, cre] , Sebastian Guelfi [ctb], Siddharth Sethi [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - BiocGenerics, type: imports\n",
      "  - BiocFileCache, type: imports\n",
      "  - dasper, type: imports\n",
      "  - derfinder, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - ggpubr, type: imports\n",
      "  - ggrepel, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - stringr, type: imports\n",
      "  - data.table, type: imports\n",
      "  - megadepth, type: imports\n",
      "  - methods, type: imports\n",
      "  - plyr, type: imports\n",
      "  - purrr, type: imports\n",
      "  - tibble, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ODER.html\n",
      "\n",
      "Name: PanomiR\n",
      "Description:\n",
      "PanomiR is a package to detect miRNAs that target groups of pathways from gene expression data. This package provides functionality for generating pathway activity profiles, determining differentially activated pathways between user-specified conditions, determining clusters of pathways via the PCxN package, and generating miRNAs targeting clusters of pathways. These function can be used separately or sequentially to analyze RNA-Seq data.\n",
      "Version: 1.2.0\n",
      "Publication date: None\n",
      "Mantainer: Pourya Naderi <pouryany@gmail.com>\n",
      "Authors:\n",
      "Author: Pourya Naderi [aut, cre], Yue Yang (Alan) Teo [aut], Ilya Sytchev [aut], Winston Hide [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.2.0) type: depends\n",
      "  - clusterProfiler, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - forcats, type: imports\n",
      "  - GSEABase, type: imports\n",
      "  - igraph, type: imports\n",
      "  - limma, type: imports\n",
      "  - metap, type: imports\n",
      "  - org.Hs.eg.db, type: imports\n",
      "  - parallel, type: imports\n",
      "  - preprocessCore, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - rlang, type: imports\n",
      "  - tibble, type: imports\n",
      "  - withr, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/PanomiR.html\n",
      "\n",
      "Name: phosphonormalizer\n",
      "Description:\n",
      "It uses the overlap between enriched and non-enriched datasets to compensate for the bias introduced in global phosphorylation after applying median normalization.\n",
      "Version: 1.22.0\n",
      "Publication date: None\n",
      "Mantainer: Sohrab Saraei <sohrab.saraei@blueprintgenetics.com>\n",
      "Authors:\n",
      "Author: Sohrab Saraei [aut, cre], Tomi Suomi [ctb], Otto Kauko [ctb], Laura Elo [ths]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - plyr, type: imports\n",
      "  - stats, type: imports\n",
      "  - graphics, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - methods, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/phosphonormalizer.html\n",
      "\n",
      "Name: PoTRA\n",
      "Description:\n",
      "The PoTRA analysis is based on topological ranks of genes in biological pathways. PoTRA can be used to detect pathways involved in disease (Li, Liu & Dinu, 2018). We use PageRank to measure the relative topological ranks of genes in each biological pathway, then select hub genes for each pathway, and use Fishers Exact test to determine if the number of hub genes in each pathway is altered from normal to cancer (Li, Liu & Dinu, 2018). Alternatively, if the distribution of topological ranks of gene in a pathway is altered between normal and cancer, this pathway might also be involved in cancer (Li, Liu & Dinu, 2018). Hence, we use the KolmogorovâSmirnov test to detect pathways that have an altered distribution of topological ranks of genes between two phenotypes (Li, Liu & Dinu, 2018). PoTRA can be used with the KEGG, Reactome, SMPDB and PharmGKB, Panther, PathBank, etc databases from the devel graphite library.\n",
      "Version: 1.13.0\n",
      "Publication date: None\n",
      "Mantainer: Margaret Linan <mlinan@asu.edu>\n",
      "Authors:\n",
      "Author: Chaoxing Li, Li Liu and Valentin Dinu\n",
      "Dependencies:\n",
      "  - R (>= 3.6.0) type: depends\n",
      "  - stats, type: depends\n",
      "  - BiocGenerics, type: depends\n",
      "  - org.Hs.eg.db, type: depends\n",
      "  - igraph, type: depends\n",
      "  - graph, type: depends\n",
      "  - graphite, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/PoTRA.html\n",
      "\n",
      "Name: svaRetro\n",
      "Description:\n",
      "svaRetro contains functions for detecting retrotransposed transcripts (RTs) from structural variant calls. It takes structural variant calls in GRanges of breakend notation and identifies RTs by exon-exon junctions and insertion sites. The candidate RTs are reported by events and annotated with information of the inserted transcripts.\n",
      "Version: 1.4.0\n",
      "Publication date: None\n",
      "Mantainer: Ruining Dong <lnyidrn@gmail.com>\n",
      "Authors:\n",
      "Author: Ruining Dong [aut, cre]\n",
      "Dependencies:\n",
      "  - GenomicRanges, type: depends\n",
      "  - rtracklayer, type: depends\n",
      "  - BiocGenerics, type: depends\n",
      "  - StructuralVariantAnnotation, type: depends\n",
      "  - R (>= 4.0) type: depends\n",
      "  - VariantAnnotation, type: imports\n",
      "  - assertthat, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - stringr, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - methods, type: imports\n",
      "  - rlang, type: imports\n",
      "  - GenomicFeatures, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3 + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/svaRetro.html\n",
      "\n",
      "Name: TraRe\n",
      "Description:\n",
      "TraRe (Transcriptional Rewiring) is an R package which contains the necessary tools to carry out several functions. Identification of module-based gene regulatory networks (GRN); score-based classification of these modules via a rewiring test; visualization of rewired modules to analyze condition-based GRN deregulation and drop out genes recovering via cliques methodology. For each tool, an html report can be generated containing useful information about the generated GRN and statistical data about the performed tests. These tools have been developed considering sequenced data (RNA-Seq).\n",
      "Version: 1.5.0\n",
      "Publication date: None\n",
      "Mantainer: Jesus De La Fuente CedeÃ±o <jdelafuentec@unav.es>\n",
      "Authors:\n",
      "Author: Jesus De La Fuente CedeÃ±o [aut, cre, cph] , Mikel Hernaez [aut, cph, ths] , Charles Blatti [aut, cph]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - hash, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - stats, type: imports\n",
      "  - methods, type: imports\n",
      "  - igraph, type: imports\n",
      "  - utils, type: imports\n",
      "  - glmnet, type: imports\n",
      "  - vbsr, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - gplots, type: imports\n",
      "  - gtools, type: imports\n",
      "  - pvclust, type: imports\n",
      "  - R.utils, type: imports\n",
      "  - dqrng, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - matrixStats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/TraRe.html\n",
      "\n",
      "Name: bandle\n",
      "Description:\n",
      "The Bandle package enables the analysis and visualisation of differential localisation experiments using mass-spectrometry data. Experimental methods supported include dynamic LOPIT-DC, hyperLOPIT, Dynamic Organellar Maps, Dynamic PCP. It provides Bioconductor infrastructure to analyse these data.\n",
      "Version: 1.2.2\n",
      "Publication date: None\n",
      "Mantainer: Oliver M. Crook <oliver.crook@stats.ox.ac.uk>\n",
      "Authors:\n",
      "Author: Oliver M. Crook [aut, cre] , Lisa Breckels [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - S4Vectors, type: depends\n",
      "  - Biobase, type: depends\n",
      "  - MSnbase, type: depends\n",
      "  - pRoloc, type: depends\n",
      "  - Rcpp (>= 1.0.4.6) type: imports\n",
      "  - pRolocdata, type: imports\n",
      "  - lbfgs, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - plyr, type: imports\n",
      "  - knitr, type: imports\n",
      "  - methods, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - robustbase, type: imports\n",
      "  - BiocStyle, type: imports\n",
      "  - ggalluvial, type: imports\n",
      "  - ggrepel, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - circlize, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - rlang, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/bandle.html\n",
      "\n",
      "Name: biodbNcbi\n",
      "Description:\n",
      "The biodbNcbi library provides access to the NCBI databases CCDS, Gene, Pubchem Comp and Pubchem Subst, using biodb package framework. It allows to retrieve entries by their accession number. Web services can be accessed for searching the database by name or mass.\n",
      "Version: 1.2.0\n",
      "Publication date: None\n",
      "Mantainer: Pierrick Roger <pierrick.roger@cea.fr>\n",
      "Authors:\n",
      "Author: Pierrick Roger [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - biodb, type: imports\n",
      "  - R6, type: imports\n",
      "  - XML, type: imports\n",
      "  - chk, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "AGPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/biodbNcbi.html\n",
      "\n",
      "Name: CNVMetrics\n",
      "Description:\n",
      "The CNVMetrics package calculates similarity metrics to facilitate copy number variant comparison among samples and/or methods. Similarity metrics can be employed to compare CNV profiles of genetically unrelated samples as well as those with a common genetic background. Some metrics are based on the shared amplified/deleted regions while other metrics rely on the level of amplification/deletion. The data type used as input is a plain text file containing the genomic position of the copy number variations, as well as the status and/or the log2 ratio values. Finally, a visualization tool is provided to explore resulting metrics.\n",
      "Version: 1.2.0\n",
      "Publication date: None\n",
      "Mantainer: Astrid DeschÃªnes <adeschen@hotmail.com>\n",
      "Authors:\n",
      "Author: Astrid DeschÃªnes [aut, cre] , Pascal Belleau [aut] , David A. Tuveson [aut], Alexander Krasnitz [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - GenomicRanges, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - methods, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - stats, type: imports\n",
      "  - pheatmap, type: imports\n",
      "  - gridExtra, type: imports\n",
      "  - grDevices, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/CNVMetrics.html\n",
      "\n",
      "Name: comapr\n",
      "Description:\n",
      "comapr detects crossover intervals for single gametes from their haplotype states sequences and stores the crossovers in GRanges object. The genetic distances can then be calculated via the mapping functions using estimated crossover rates for maker intervals. Visualisation functions for plotting interval-based genetic map or cumulative genetic distances are implemented, which help reveal the variation of crossovers landscapes across the genome and across individuals.\n",
      "Version: 1.2.0\n",
      "Publication date: None\n",
      "Mantainer: Ruqian Lyu <xiaoru.best@gmail.com>\n",
      "Authors:\n",
      "Author: Ruqian Lyu [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.1.0) type: depends\n",
      "  - methods, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - gridExtra, type: imports\n",
      "  - plotly, type: imports\n",
      "  - circlize, type: imports\n",
      "  - rlang, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - foreach, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - scales, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - utils, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - grid, type: imports\n",
      "  - stats, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - plyr, type: imports\n",
      "  - Gviz, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/comapr.html\n",
      "\n",
      "Name: crisprDesign\n",
      "Description:\n",
      "Provides a comprehensive suite of functions to design and annotate CRISPR guide RNA (gRNAs) sequences. This includes on- and off-target search, on-target efficiency scoring, off-target scoring, full gene and TSS contextual annotations, and SNP annotation (human only). It currently support five types of CRISPR modalities (modes of perturbations): CRISPR knockout, CRISPR activation, CRISPR inhibition, CRISPR base editing, and CRISPR knockdown. All types of CRISPR nucleases are supported, including DNA- and RNA-target nucleases such as Cas9, Cas12a, and Cas13d. All types of base editors are also supported. gRNA design can be performed on reference genomes, transcriptomes, and custom DNA and RNA sequences. Both unpaired and paired gRNA designs are enabled.\n",
      "Version: 1.0.0\n",
      "Publication date: None\n",
      "Mantainer: Jean-Philippe Fortin <fortin946@gmail.com>\n",
      "Authors:\n",
      "Author: Jean-Philippe Fortin [aut, cre], Luke Hoberecht [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.2.0) type: depends\n",
      "  - crisprBase, type: depends\n",
      "  - AnnotationDbi, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - BSgenome, type: imports\n",
      "  - crisprBowtie, type: imports\n",
      "  - crisprScore, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - GenomicFeatures, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - MatrixGenerics, type: imports\n",
      "  - methods, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - VariantAnnotation, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/crisprDesign.html\n",
      "\n",
      "Name: CyTOFpower\n",
      "Description:\n",
      "This package is a tool to predict the power of CyTOF experiments in the context of differential state analyses. The package provides a shiny app with two options to predict the power of an experiment: i. generation of in-sicilico CyTOF data, using users input ii. browsing in a grid of parameters for which the power was already precomputed.\n",
      "Version: 1.4.0\n",
      "Publication date: None\n",
      "Mantainer: Anne-Maud Ferreira <anne-maud.ferreira@stanford.edu>\n",
      "Authors:\n",
      "Author: Anne-Maud Ferreira [cre, aut] , Catherine Blish [aut], Susan Holmes [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - CytoGLMM, type: imports\n",
      "  - diffcyt, type: imports\n",
      "  - DT, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - methods, type: imports\n",
      "  - rlang, type: imports\n",
      "  - stats, type: imports\n",
      "  - shiny, type: imports\n",
      "  - shinyFeedback, type: imports\n",
      "  - shinyjs, type: imports\n",
      "  - shinyMatrix, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - tibble, type: imports\n",
      "  - tidyr, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/CyTOFpower.html\n",
      "\n",
      "Name: cytoMEM\n",
      "Description:\n",
      "MEM, Marker Enrichment Modeling, automatically generates and displays quantitative labels for cell populations that have been identified from single-cell data. The input for MEM is a dataset that has pre-clustered or pre-gated populations with cells in rows and features in columns. Labels convey a list of measured features and the features' levels of relative enrichment on each population. MEM can be applied to a wide variety of data types and can compare between MEM labels from flow cytometry, mass cytometry, single cell RNA-seq, and spectral flow cytometry using RMSD.\n",
      "Version: 1.2.0\n",
      "Publication date: None\n",
      "Mantainer: Jonathan Irish <jonathan.irish@vanderbilt.edu>\n",
      "Authors:\n",
      "Author: Sierra Lima [aut] , Kirsten Diggins [aut] , Jonathan Irish [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.2.0) type: depends\n",
      "  - gplots, type: imports\n",
      "  - tools, type: imports\n",
      "  - flowCore, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - methods, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/cytoMEM.html\n",
      "\n",
      "Name: DifferentialRegulation\n",
      "Description:\n",
      "DifferentialRegulation is a method for detecting differentially regulated genes between two groups of samples (e.g., healthy vs. disease, or treated vs. untreated samples), by targeting differences in the balance of spliced and unspliced mRNA abundances, obtained from single-cell RNA-sequencing (scRNA-seq) data. DifferentialRegulation accounts for the sample-to-sample variability, and embeds multiple samples in a Bayesian hierarchical model. In particular, when reads are compatible with multiple genes or multiple splicing versions of a gene (unspliced spliced or ambiguous), the method allocates these multi-mapping reads to the gene of origin and their splicing version. Parameters are inferred via Markov chain Monte Carlo (MCMC) techniques (Metropolis-within-Gibbs).\n",
      "Version: 1.2.2\n",
      "Publication date: None\n",
      "Mantainer: Simone Tiberi <simone.tiberi@uzh.ch>\n",
      "Authors:\n",
      "Author: Simone Tiberi [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.2.0) type: depends\n",
      "  - methods, type: imports\n",
      "  - Rcpp, type: imports\n",
      "  - doRNG, type: imports\n",
      "  - MASS, type: imports\n",
      "  - data.table, type: imports\n",
      "  - doParallel, type: imports\n",
      "  - parallel, type: imports\n",
      "  - foreach, type: imports\n",
      "  - stats, type: imports\n",
      "  - BANDITS, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - SingleCellExperiment, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - ggplot2, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/DifferentialRegulation.html\n",
      "\n",
      "Name: eds\n",
      "Description:\n",
      "This packages provides a single function, readEDS. This is a low-level utility for reading in Alevin EDS format into R. This function is not designed for end-users but instead the package is predominantly for simplifying package dependency graph for other Bioconductor packages.\n",
      "Version: 1.0.0\n",
      "Publication date: None\n",
      "Mantainer: Avi Srivastava <asrivastava@cs.stonybrook.edu>\n",
      "Authors:\n",
      "Author: Avi Srivastava [aut, cre], Michael Love [aut, ctb]\n",
      "Dependencies:\n",
      "  - Matrix, type: depends\n",
      "  - Rcpp, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/eds.html\n",
      "\n",
      "Name: epimutacions\n",
      "Description:\n",
      "The package includes some statistical outlier detection methods for epimutations detection in DNA methylation data. The methods included in the package are MANOVA, Multivariate linear models, isolation forest, robust mahalanobis distance, quantile and beta. The methods compare a case sample with a suspected disease against a reference panel (composed of healthy individuals) to identify epimutations in the given case sample. It also contains functions to annotate and visualize the identified epimutations.\n",
      "Version: 1.2.0\n",
      "Publication date: None\n",
      "Mantainer: Dolors Pelegri-Siso <dolors.pelegri@isglobal.org>\n",
      "Authors:\n",
      "Author: Dolors Pelegri-Siso [aut, cre] , Juan R. Gonzalez [aut] , Carlos Ruiz-Arenas [aut] , Carles Hernandez-Ferrer [aut] , Leire Abarrategui [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.2.0) type: depends\n",
      "  - epimutacionsData, type: depends\n",
      "  - minfi, type: imports\n",
      "  - bumphunter, type: imports\n",
      "  - isotree, type: imports\n",
      "  - robustbase, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - GenomicFeatures, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - stats, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - utils, type: imports\n",
      "  - biomaRt, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - Homo.sapiens, type: imports\n",
      "  - purrr, type: imports\n",
      "  - tibble, type: imports\n",
      "  - Gviz, type: imports\n",
      "  - TxDb.Hsapiens.UCSC.hg19.knownGene, type: imports\n",
      "  - TxDb.Hsapiens.UCSC.hg18.knownGene, type: imports\n",
      "  - TxDb.Hsapiens.UCSC.hg38.knownGene, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - AnnotationDbi, type: imports\n",
      "  - AnnotationHub, type: imports\n",
      "  - ExperimentHub, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - grid, type: imports\n",
      "  - ensembldb, type: imports\n",
      "  - gridExtra, type: imports\n",
      "  - IlluminaHumanMethylation450kmanifest, type: imports\n",
      "  - IlluminaHumanMethylationEPICmanifest, type: imports\n",
      "  - IlluminaHumanMethylation450kanno.ilmn12.hg19, type: imports\n",
      "  - IlluminaHumanMethylationEPICanno.ilm10b2.hg19, type: imports\n",
      "  - ggrepel, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/epimutacions.html\n",
      "\n",
      "Name: extraChIPs\n",
      "Description:\n",
      "This package builds on existing tools and adds some simple but extremely useful capabilities for working with ChIP-Seq data. The focus is on detecting differential binding windows/regions. One set of functions focusses on set-operations retaining mcols for GRanges objects, whilst another group of functions are to aid visualisation of results. Coercion to tibble objects is also included.\n",
      "Version: 1.2.4\n",
      "Publication date: None\n",
      "Mantainer: Stephen Pederson <stephen.pederson.au@gmail.com>\n",
      "Authors:\n",
      "Author: Stephen Pederson [aut, cre]\n",
      "Dependencies:\n",
      "  - BiocParallel, type: depends\n",
      "  - R (>= 4.2.0) type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - ggplot2 (>= 3.4.0) type: depends\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - tibble, type: depends\n",
      "  - BiocIO, type: imports\n",
      "  - broom, type: imports\n",
      "  - ComplexUpset, type: imports\n",
      "  - csaw, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - edgeR, type: imports\n",
      "  - EnrichedHeatmap, type: imports\n",
      "  - forcats, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - GenomicInteractions, type: imports\n",
      "  - ggforce, type: imports\n",
      "  - ggrepel, type: imports\n",
      "  - ggside, type: imports\n",
      "  - glue, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - grid, type: imports\n",
      "  - Gviz, type: imports\n",
      "  - InteractionSet, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - limma, type: imports\n",
      "  - methods, type: imports\n",
      "  - patchwork, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - rlang, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - scales, type: imports\n",
      "  - stats, type: imports\n",
      "  - stringr, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - tidyselect, type: imports\n",
      "  - utils, type: imports\n",
      "  - vctrs, type: imports\n",
      "  - VennDiagram, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/extraChIPs.html\n",
      "\n",
      "Name: rGenomeTracks\n",
      "Description:\n",
      "rGenomeTracks package leverages the power of pyGenomeTracks software with the interactivity of R. pyGenomeTracks is a python software that offers robust method for visualizing epigenetic data files like narrowPeak, Hic matrix, TADs and arcs, however though, here is no way currently to use it within R interactive session. rGenomeTracks wrapped the whole functionality of pyGenomeTracks with additional utilites to make to more pleasant for R users.\n",
      "Version: 1.4.0\n",
      "Publication date: None\n",
      "Mantainer: Omar Elashkar <omar.i.elashkar@gmail.com>\n",
      "Authors:\n",
      "Author: Omar Elashkar [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.1.0) type: depends\n",
      "  - imager, type: imports\n",
      "  - reticulate, type: imports\n",
      "  - methods, type: imports\n",
      "  - rGenomeTracksData, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/rGenomeTracks.html\n",
      "\n",
      "Name: Rnits\n",
      "Description:\n",
      "R/Bioconductor package for normalization, curve registration and inference in time course gene expression data.\n",
      "Version: 1.32.1\n",
      "Publication date: None\n",
      "Mantainer: Dipen P. Sangurdekar <dipen.sangurdekar@gmail.com>\n",
      "Authors:\n",
      "Author: Dipen P. Sangurdekar <dipen.sangurdekar at gmail.com>\n",
      "Dependencies:\n",
      "  - R (>= 3.6.0) type: depends\n",
      "  - Biobase, type: depends\n",
      "  - ggplot2, type: depends\n",
      "  - limma, type: depends\n",
      "  - methods, type: depends\n",
      "  - affy, type: imports\n",
      "  - boot, type: imports\n",
      "  - impute, type: imports\n",
      "  - splines, type: imports\n",
      "  - graphics, type: imports\n",
      "  - qvalue, type: imports\n",
      "  - reshape2, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/Rnits.html\n",
      "\n",
      "Name: sccomp\n",
      "Description:\n",
      "A robust and outlier-aware method for testing differential tissue composition from single-cell data. This model can infer changes in tissue composition and heterogeneity, and can produce realistic data simulations based on any existing dataset. This model can also transfer knowledge from a large set of integrated datasets to increase accuracy further.\n",
      "Version: 1.2.1\n",
      "Publication date: None\n",
      "Mantainer: Stefano Mangiola <mangiolastefano@gmail.com>\n",
      "Authors:\n",
      "Author: Stefano Mangiola [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.2.0) type: depends\n",
      "  - methods, type: imports\n",
      "  - Rcpp (>= 0.12.0) type: imports\n",
      "  - RcppParallel (>= 5.0.1) type: imports\n",
      "  - rstantools (>= 2.1.1) type: imports\n",
      "  - rstan (>= 2.18.1) type: imports\n",
      "  - SeuratObject, type: imports\n",
      "  - SingleCellExperiment, type: imports\n",
      "  - parallel, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - purrr, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - rlang, type: imports\n",
      "  - tibble, type: imports\n",
      "  - boot, type: imports\n",
      "  - lifecycle, type: imports\n",
      "  - stats, type: imports\n",
      "  - tidyselect, type: imports\n",
      "  - utils, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - ggrepel, type: imports\n",
      "  - patchwork, type: imports\n",
      "  - forcats, type: imports\n",
      "  - readr, type: imports\n",
      "  - scales, type: imports\n",
      "  - stringr, type: imports\n",
      "  - glue, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/sccomp.html\n",
      "\n",
      "Name: sSNAPPY\n",
      "Description:\n",
      "A single sample pathway perturbation testing method for RNA-seq data. The method propagates changes in gene expression down gene-set topologies to compute single-sample directional pathway perturbation scores that reflect potential directions of changes.Perturbation scores can be used to test significance of pathway perturbation at both individual-sample and treatment levels.\n",
      "Version: 1.2.4\n",
      "Publication date: None\n",
      "Mantainer: Wenjun Liu <wenjun.liu@adelaide.edu.au>\n",
      "Authors:\n",
      "Author: Wenjun Liu [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.2.0) type: depends\n",
      "  - dplyr, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - rlang, type: imports\n",
      "  - stats, type: imports\n",
      "  - purrr, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - graphite, type: imports\n",
      "  - Rcpp, type: imports\n",
      "  - tibble, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - ggraph, type: imports\n",
      "  - igraph, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - org.Hs.eg.db, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - edgeR, type: imports\n",
      "  - methods, type: imports\n",
      "  - ggforce, type: imports\n",
      "  - ggnewscale, type: imports\n",
      "  - pheatmap, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/sSNAPPY.html\n",
      "\n",
      "Name: Summix\n",
      "Description:\n",
      "This package contains the Summix method for estimating and adjusting for ancestry in genetic summary allele frequency data. The function summix estimates reference ancestry proportions using a mixture model. The adjAF function produces ancestry adjusted allele frequencies for an observed sample with ancestry proportions matching a target person or sample.\n",
      "Version: 2.4.0\n",
      "Publication date: None\n",
      "Mantainer: Audrey Hendricks <audrey.hendricks@ucdenver.edu>\n",
      "Authors:\n",
      "Author: Audrey Hendricks [cre], Stoneman Haley [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - nloptr, type: imports\n",
      "  - methods, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/Summix.html\n",
      "\n",
      "Name: svaNUMT\n",
      "Description:\n",
      "svaNUMT contains functions for detecting NUMT events from structural variant calls. It takes structural variant calls in GRanges of breakend notation and identifies NUMTs by nuclear-mitochondrial breakend junctions. The main function reports candidate NUMTs if there is a pair of valid insertion sites found on the nuclear genome within a certain distance threshold. The candidate NUMTs are reported by events.\n",
      "Version: 1.4.0\n",
      "Publication date: None\n",
      "Mantainer: Ruining Dong <lnyidrn@gmail.com>\n",
      "Authors:\n",
      "Author: Ruining Dong [aut, cre]\n",
      "Dependencies:\n",
      "  - GenomicRanges, type: depends\n",
      "  - rtracklayer, type: depends\n",
      "  - VariantAnnotation, type: depends\n",
      "  - StructuralVariantAnnotation, type: depends\n",
      "  - BiocGenerics, type: depends\n",
      "  - Biostrings, type: depends\n",
      "  - R (>= 4.0) type: depends\n",
      "  - assertthat, type: imports\n",
      "  - stringr, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - methods, type: imports\n",
      "  - rlang, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - GenomicFeatures, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3 + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/svaNUMT.html\n",
      "\n",
      "Name: TEKRABber\n",
      "Description:\n",
      "TEKRABber is made to provide a user-friendly pipeline for comparing orthologs and transposable elements (TEs) between two species. It considers the orthology confidence between two species from BioMart to normalize expression counts and detect differentially expressed orthologs/TEs. Then it provides one to one correlation analysis for desired orthologs and TEs. There is also an app function to have a first insight on the result. Users can prepare orthologs/TEs RNA-seq expression data by their own preference to run TEKRABber following the data structure mentioned in the vignettes.\n",
      "Version: 1.2.0\n",
      "Publication date: None\n",
      "Mantainer: Yao-Chung Chen <yao-chung.chen@fu-berlin.de>\n",
      "Authors:\n",
      "Author: Yao-Chung Chen [aut, cre] , Katja Nowick [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - apeglm, type: imports\n",
      "  - biomaRt, type: imports\n",
      "  - DESeq2, type: imports\n",
      "  - Rcpp (>= 1.0.7) type: imports\n",
      "  - SCBN, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/TEKRABber.html\n",
      "\n",
      "Name: tspair\n",
      "Description:\n",
      "Bioconductor version: Release (3.16)\n",
      "Version: 1.56.0\n",
      "Publication date: None\n",
      "Mantainer: Author: Jeffrey T. Leek <jtleek@jhu.edu>\n",
      "Authors:\n",
      "These functions calculate the pair of genes that show the maximum difference in ranking between two user specified groups. This \"top scoring pair\" maximizes the average of sensitivity and specificity over all rank based classifiers using a pair of genes in the data set. The advantage of classifying samples based on only the relative rank of a pair of genes is (a) the classifiers are much simpler and often more interpretable than more complicated classification schemes and (b) if arrays can be classified using only a pair of genes, PCR based tests could be used for classification of samples. See the references for the tspcalc() function for references regarding TSP classifiers.\n",
      "Dependencies:\n",
      "  - R (>= 2.10) type: depends\n",
      "  - Biobase, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/tspair.html\n",
      "\n",
      "Name: BOBaFIT\n",
      "Description:\n",
      "This package provides a method to refit and correct the diploid region in copy number profiles. It uses a clustering algorithm to identify pathology-specific normal (diploid) chromosomes and then use their copy number signal to refit the whole profile. The package is composed by three functions: DRrefit (the main function), ComputeNormalChromosome and PlotCluster.\n",
      "Version: 1.2.0\n",
      "Publication date: None\n",
      "Mantainer: Gaia Mazzocchetti <bioinformatic.seragnoli@gmail.com>\n",
      "Authors:\n",
      "Author: Andrea Poletti [aut], Gaia Mazzocchetti [aut, cre], Vincenza Solli [aut]\n",
      "Dependencies:\n",
      "  - R (>= 2.10) type: depends\n",
      "  - dplyr, type: imports\n",
      "  - NbClust, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - ggbio, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - stats, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - ggforce, type: imports\n",
      "  - stringr, type: imports\n",
      "  - plyranges, type: imports\n",
      "  - methods, type: imports\n",
      "  - utils, type: imports\n",
      "  - magrittr, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 3)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/BOBaFIT.html\n",
      "\n",
      "Name: iBMQ\n",
      "Description:\n",
      "integrated Bayesian Modeling of eQTL data\n",
      "Version: 1.38.0\n",
      "Publication date: None\n",
      "Mantainer: Greg Imholte <gimholte@uw.edu>\n",
      "Authors:\n",
      "Author: Marie-Pier Scott-Boyer and Greg Imholte\n",
      "Dependencies:\n",
      "  - R (>= 2.15.0) type: depends\n",
      "  - Biobase, type: depends\n",
      "  - ggplot2 (>= 0.9.2) type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/iBMQ.html\n",
      "\n",
      "Name: iSEEhex\n",
      "Description:\n",
      "This package provides panels summarising data points in hexagonal bins for `iSEE`. It is part of `iSEEu`, the iSEE universe of panels that extend the `iSEE` package.\n",
      "Version: 1.0.0\n",
      "Publication date: None\n",
      "Mantainer: Kevin Rue-Albrecht <kevinrue67@gmail.com>\n",
      "Authors:\n",
      "Author: Kevin Rue-Albrecht [aut, cre] , Charlotte Soneson [aut] , Federico Marini [aut] , Aaron Lun [aut]\n",
      "Dependencies:\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - iSEE, type: depends\n",
      "  - ggplot2, type: imports\n",
      "  - hexbin, type: imports\n",
      "  - methods, type: imports\n",
      "  - shiny, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/iSEEhex.html\n",
      "\n",
      "Name: lineagespot\n",
      "Description:\n",
      "Lineagespot is a framework written in R, and aims to identify SARS-CoV-2 related mutations based on a single (or a list) of variant(s) file(s) (i.e., variant calling format). The method can facilitate the detection of SARS-CoV-2 lineages in wastewater samples using next generation sequencing, and attempts to infer the potential distribution of the SARS-CoV-2 lineages.\n",
      "Version: 1.2.0\n",
      "Publication date: None\n",
      "Mantainer: Nikolaos Pechlivanis <inab.bioinformatics@lists.certh.gr>\n",
      "Authors:\n",
      "Author: Nikolaos Pechlivanis [aut, cre] , Maria Tsagiopoulou [aut], Maria Christina Maniou [aut], Anastasis Togkousidis [aut], Evangelia Mouchtaropoulou [aut], Taxiarchis Chassalevris [aut], Serafeim Chaintoutis [aut], Chrysostomos Dovas [aut], Maria Petala [aut], Margaritis Kostoglou [aut], Thodoris Karapantsios [aut], Stamatia Laidou [aut], Elisavet Vlachonikola [aut], Aspasia Orfanou [aut], Styliani-Christina Fragkouli [aut], Sofoklis Keisaris [aut], Anastasia Chatzidimitriou [aut], Agis Papadopoulos [aut], Nikolaos Papaioannou [aut], Anagnostis Argiriou [aut], Fotis E. Psomopoulos [aut]\n",
      "Dependencies:\n",
      "  - VariantAnnotation, type: imports\n",
      "  - MatrixGenerics, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - data.table, type: imports\n",
      "  - stringr, type: imports\n",
      "  - httr, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/lineagespot.html\n",
      "\n",
      "Name: Motif2Site\n",
      "Description:\n",
      "Detect binding sites using motifs IUPAC sequence or bed coordinates and ChIP-seq experiments in bed or bam format. Combine/compare binding sites across experiments, tissues, or conditions. All normalization and differential steps are done using TMM-GLM method. Signal decomposition is done by setting motifs as the centers of the mixture of normal distribution curves.\n",
      "Version: 1.2.0\n",
      "Publication date: None\n",
      "Mantainer: Peyman Zarrineh <peyman.zarrineh@manchester.ac.uk>\n",
      "Authors:\n",
      "Author: Peyman Zarrineh [cre, aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - S4Vectors, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - methods, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - BSgenome, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - MASS, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - GenomicAlignments, type: imports\n",
      "  - edgeR, type: imports\n",
      "  - mixtools, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/Motif2Site.html\n",
      "\n",
      "Name: MuData\n",
      "Description:\n",
      "Save MultiAssayExperiments to h5mu files supported by muon and mudata. Muon is a Python framework for multimodal omics data analysis. It uses an HDF5-based format for data storage.\n",
      "Version: 1.2.0\n",
      "Publication date: None\n",
      "Mantainer: Danila Bredikhin <danila.bredikhin@embl.de>\n",
      "Authors:\n",
      "Author: Danila Bredikhin [aut, cre] , Ilia Kats [aut]\n",
      "Dependencies:\n",
      "  - Matrix, type: depends\n",
      "  - S4Vectors, type: depends\n",
      "  - rhdf5, type: depends\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - MultiAssayExperiment, type: imports\n",
      "  - SingleCellExperiment, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - DelayedArray, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/MuData.html\n",
      "\n",
      "Name: qmtools\n",
      "Description:\n",
      "The qmtools (quantitative metabolomics tools) package provides basic tools for processing quantitative metabolomics data with the standard SummarizedExperiment class. This includes functions for imputation, normalization, feature filtering, feature clustering, dimension-reduction, and visualization to help users prepare data for statistical analysis. Several functions in this package could also be used in other types of omics data.\n",
      "Version: 1.2.0\n",
      "Publication date: None\n",
      "Mantainer: Jaehyun Joo <jaehyunjoo@outlook.com>\n",
      "Authors:\n",
      "Author: Jaehyun Joo [aut, cre], Blanca Himes [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.2.0) type: depends\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - rlang, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - patchwork, type: imports\n",
      "  - heatmaply, type: imports\n",
      "  - methods, type: imports\n",
      "  - MsCoreUtils, type: imports\n",
      "  - stats, type: imports\n",
      "  - igraph, type: imports\n",
      "  - VIM, type: imports\n",
      "  - scales, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/qmtools.html\n",
      "\n",
      "Name: CBEA\n",
      "Description:\n",
      "This package implements CBEA, a method to perform set-based analysis for microbiome relative abundance data. This approach constructs a competitive balance between taxa within the set and remainder taxa per sample. More details can be found in the Nguyen et al. 2021+ manuscript. Additionally, this package adds support functions to help users perform taxa-set enrichment analyses using existing gene set analysis methods. In the future we hope to also provide curated knowledge driven taxa sets.\n",
      "Version: 1.2.0\n",
      "Publication date: None\n",
      "Mantainer: Quang Nguyen <quangpmnguyen@gmail.com>\n",
      "Authors:\n",
      "Author: Quang Nguyen [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.2.0) type: depends\n",
      "  - BiocParallel, type: imports\n",
      "  - BiocSet, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - lmom, type: imports\n",
      "  - fitdistrplus, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - methods, type: imports\n",
      "  - mixtools, type: imports\n",
      "  - Rcpp (>= 1.0.7) type: imports\n",
      "  - stats, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - tibble, type: imports\n",
      "  - TreeSummarizedExperiment, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - glue, type: imports\n",
      "  - generics, type: imports\n",
      "  - rlang, type: imports\n",
      "  - goftest, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/CBEA.html\n",
      "\n",
      "Name: ctgGEM\n",
      "Description:\n",
      "Bioconductor version: Release (3.16)\n",
      "Version: 1.10.0\n",
      "Publication date: None\n",
      "Mantainer: Author: Mark Block [aut], Carrie Minette [aut], Evgeni Radichev [aut], Etienne Gnimpieba [aut], Mariah Hoffman [aut], USD Biomedical Engineering [aut, cre]\n",
      "Authors:\n",
      "Cell Tree Generator for Gene Expression Matrices (ctgGEM) streamlines the building of cell-state hierarchies from single-cell gene expression data across multiple existing tools for improved comparability and reproducibility. It supports pseudotemporal ordering algorithms and visualization tools from monocle, cellTree, TSCAN, sincell, and destiny, and provides a unified output format for integration with downstream data analysis workflows and Cytoscape.\n",
      "Dependencies:\n",
      "  - monocle, type: depends\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - Biobase, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - igraph, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - methods, type: imports\n",
      "  - utils, type: imports\n",
      "  - sincell, type: imports\n",
      "  - TSCAN, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL(>=2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ctgGEM.html\n",
      "\n",
      "Name: GAPGOM\n",
      "Description:\n",
      "Collection of various measures and tools for lncRNA annotation prediction put inside a redistributable R package. The package contains two main algorithms; lncRNA2GOA and TopoICSim. lncRNA2GOA tries to annotate novel genes (in this specific case lncRNAs) by using various correlation/geometric scoring methods on correlated expression data. After correlating/scoring, the results are annotated and enriched. TopoICSim is a topologically based method, that compares gene similarity based on the topology of the GO DAG by information content (IC) between GO terms.\n",
      "Version: 1.14.0\n",
      "Publication date: None\n",
      "Mantainer: Rezvan Ehsani <rezvanehsani74@gmail.com>\n",
      "Authors:\n",
      "Author: Rezvan Ehsani [aut, cre], Casper van Mourik [aut], Finn DrablÃ¸s [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - methods, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - fastmatch, type: imports\n",
      "  - plyr, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - data.table, type: imports\n",
      "  - igraph, type: imports\n",
      "  - graph, type: imports\n",
      "  - RBGL, type: imports\n",
      "  - GO.db, type: imports\n",
      "  - org.Hs.eg.db, type: imports\n",
      "  - org.Mm.eg.db, type: imports\n",
      "  - GOSemSim, type: imports\n",
      "  - GEOquery, type: imports\n",
      "  - AnnotationDbi, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - BiocFileCache, type: imports\n",
      "  - matrixStats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/GAPGOM.html\n",
      "\n",
      "Name: IsoGeneGUI\n",
      "Description:\n",
      "Bioconductor version: Release (3.16)\n",
      "Version: 2.34.0\n",
      "Publication date: None\n",
      "Mantainer: Author: Setia Pramana, Dan Lin, Philippe Haldermans, Tobias Verbeke, Martin Otava\n",
      "Authors:\n",
      "The IsoGene Graphical User Interface (IsoGene-GUI) is a user friendly interface of the IsoGene package which is aimed to identify for genes with a monotonic trend in the expression levels with respect to the increasing doses. Additionally, GUI extension of original package contains various tools to perform clustering of dose-response profiles. Testing is addressed through several test statistics: global likelihood ratio test (E2), Bartholomew 1961, Barlow et al. 1972 and Robertson et al. 1988), Williams (1971, 1972), Marcus (1976), the M (Hu et al. 2005) and the modified M (Lin et al. 2007). The p-values of the global likelihood ratio test (E2) are obtained using the exact distribution and permutations. The other four test statistics are obtained using permutations. Several p-values adjustment are provided: Bonferroni, Holm (1979), Hochberg (1988), and Sidak procedures for controlling the family-wise Type I error rate (FWER), and BH (Benjamini and Hochberg 1995) and BY (Benjamini and Yekutieli 2001) procedures are used for controlling the FDR. The inference is based on resampling methods, which control the False Discovery Rate (FDR), for both permutations (Ge et al., 2003) and the Significance Analysis of Microarrays (SAM, Tusher et al., 2001). Clustering methods are outsourced from CRAN packages ORCME, ORIClust. The package ORCME is based on delta-clustering method (Cheng and Church, 2000) and ORIClust on Order Restricted Information Criterion (Liu et al., 2009), both perform same task but from different perspective and their outputs are clusters of genes. Additionally, profile selection for given gene based on Generalized ORIC (Kuiper et al., 2014) from package goric and permutation test for E2 based on package orQA are included in IsoGene-GUI. None of these four packages has GUI.\n",
      "Dependencies:\n",
      "  - tcltk, type: depends\n",
      "  - xlsx, type: depends\n",
      "  - Rcpp, type: imports\n",
      "  - tkrplot, type: imports\n",
      "  - multtest, type: imports\n",
      "  - relimp, type: imports\n",
      "  - geneplotter, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - Iso, type: imports\n",
      "  - IsoGene, type: imports\n",
      "  - ORCME, type: imports\n",
      "  - ORIClust, type: imports\n",
      "  - orQA, type: imports\n",
      "  - goric, type: imports\n",
      "  - ff, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - jpeg, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/IsoGeneGUI.html\n",
      "\n",
      "Name: MACPET\n",
      "Description:\n",
      "Bioconductor version: Release (3.16)\n",
      "Version: 1.18.0\n",
      "Publication date: None\n",
      "Mantainer: Author: Ioannis Vardaxis\n",
      "Authors:\n",
      "The MACPET package can be used for complete interaction analysis for ChIA-PET data. MACPET reads ChIA-PET data in BAM or SAM format and separates the data into Self-ligated, Intra- and Inter-chromosomal PETs. Furthermore, MACPET breaks the genome into regions and applies 2D mixture models for identifying candidate peaks/binding sites using skewed generalized students-t distributions (SGT). It then uses a local poisson model for finding significant binding sites. Finally it runs an additive interaction-analysis model for calling for significant interactions between those peaks. MACPET is mainly written in C++, and it also supports the BiocParallel package.\n",
      "Dependencies:\n",
      "  - R (>= 3.6.1) type: depends\n",
      "  - InteractionSet, type: depends\n",
      "  - bigmemory (>= 4.5.33) type: depends\n",
      "  - BH (>= 1.66.0.1) type: depends\n",
      "  - Rcpp (>= 1.0.1) type: depends\n",
      "  - intervals (>= 0.15.1) type: imports\n",
      "  - plyr (>= 1.8.4) type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - stats (>= 3.6.1) type: imports\n",
      "  - utils (>= 3.6.1) type: imports\n",
      "  - methods (>= 3.6.1) type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - gtools (>= 3.8.1) type: imports\n",
      "  - GenomicAlignments, type: imports\n",
      "  - knitr (>= 1.23) type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - Rbowtie, type: imports\n",
      "  - GEOquery, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - ShortRead, type: imports\n",
      "  - futile.logger (>= 1.4.3) type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/MACPET.html\n",
      "\n",
      "Name: MBECS\n",
      "Description:\n",
      "The Microbiome Batch Effect Correction Suite (MBECS) provides a set of functions to evaluate and mitigate unwated noise due to processing in batches. To that end it incorporates a host of batch correcting algorithms (BECA) from various packages. In addition it offers a correction and reporting pipeline that provides a preliminary look at the characteristics of a data-set before and after correcting for batch effects.\n",
      "Version: 1.2.0\n",
      "Publication date: None\n",
      "Mantainer: Michael Olbrich <M.Olbrich@protonmail.com>\n",
      "Authors:\n",
      "Author: Michael Olbrich [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - methods, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - phyloseq, type: imports\n",
      "  - limma, type: imports\n",
      "  - lme4, type: imports\n",
      "  - lmerTest, type: imports\n",
      "  - pheatmap, type: imports\n",
      "  - rmarkdown, type: imports\n",
      "  - cluster, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - gridExtra, type: imports\n",
      "  - ruv, type: imports\n",
      "  - sva, type: imports\n",
      "  - tibble, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - vegan, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - Matrix, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/MBECS.html\n",
      "\n",
      "Name: MMAPPR2\n",
      "Description:\n",
      "MMAPPR2 maps mutations resulting from pooled RNA-seq data from the F2 cross of forward genetic screens. Its predecessor is described in a paper published in Genome Research (Hill et al. 2013). MMAPPR2 accepts aligned BAM files as well as a reference genome as input, identifies loci of high sequence disparity between the control and mutant RNA sequences, predicts variant effects using Ensembl's Variant Effect Predictor, and outputs a ranked list of candidate mutations.\n",
      "Version: 1.12.0\n",
      "Publication date: None\n",
      "Mantainer: Jonathon Hill <jhill@byu.edu>\n",
      "Authors:\n",
      "Author: Kyle Johnsen [aut], Nathaniel Jenkins [aut], Jonathon Hill [cre]\n",
      "Dependencies:\n",
      "  - R (>= 3.6.0) type: depends\n",
      "  - ensemblVEP, type: imports\n",
      "  - gmapR, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - VariantAnnotation, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - VariantTools, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - methods, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - stringr, type: imports\n",
      "  - data.table, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/MMAPPR2.html\n",
      "\n",
      "Name: OGRE\n",
      "Description:\n",
      "OGRE calculates overlap between user defined genomic region datasets. Any regions can be supplied i.e. genes, SNPs, or reads from sequencing experiments. Key numbers help analyse the extend of overlaps which can also be visualized at a genomic level.\n",
      "Version: 1.2.0\n",
      "Publication date: None\n",
      "Mantainer: Sven Berres <svenbioinf@gmail.com>\n",
      "Authors:\n",
      "Author: Sven Berres [aut, cre], JÃ¶rg Gromoll [ctb], Marius WÃ¶ste [ctb], Sarah Sandmann [ctb], Sandra Laurentino [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 4.1.0) type: depends\n",
      "  - S4Vectors, type: depends\n",
      "  - GenomicRanges, type: imports\n",
      "  - methods, type: imports\n",
      "  - data.table, type: imports\n",
      "  - assertthat, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - Gviz, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - AnnotationHub, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - stats, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - shiny, type: imports\n",
      "  - shinyFiles, type: imports\n",
      "  - DT, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - shinydashboard, type: imports\n",
      "  - shinyBS, type: imports\n",
      "  - tidyr, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/OGRE.html\n",
      "\n",
      "Name: omicsViewer\n",
      "Description:\n",
      "omicsViewer visualizes ExpressionSet (or SummarizedExperiment) in an interactive way. The omicsViewer has a separate back- and front-end. In the back-end, users need to prepare an ExpressionSet that contains all the necessary information for the downstream data interpretation. Some extra requirements on the headers of phenotype data or feature data are imposed so that the provided information can be clearly recognized by the front-end, at the same time, keep a minimum modification on the existing ExpressionSet object. The pure dependency on R/Bioconductor guarantees maximum flexibility in the statistical analysis in the back-end. Once the ExpressionSet is prepared, it can be visualized using the front-end, implemented by shiny and plotly. Both features and samples could be selected from (data) tables or graphs (scatter plot/heatmap). Different types of analyses, such as enrichment analysis (using Bioconductor package fgsea or fisher's exact test) and STRING network analysis, will be performed on the fly and the results are visualized simultaneously. When a subset of samples and a phenotype variable is selected, a significance test on means (t-test or ranked based test; when phenotype variable is quantitative) or test of independence (chi-square or fisherâs exact test; when phenotype data is categorical) will be performed to test the association between the phenotype of interest with the selected samples. Additionally, other analyses can be easily added as extra shiny modules. Therefore, omicsViewer will greatly facilitate data exploration, many different hypotheses can be explored in a short time without the need for knowledge of R. In addition, the resulting data could be easily shared using a shiny server. Otherwise, a standalone version of omicsViewer together with designated omics data could be easily created by integrating it with portable R, which can be shared with collaborators or submitted as supplementary data together with a manuscript.\n",
      "Version: 1.2.0\n",
      "Publication date: None\n",
      "Mantainer: Chen Meng <mengchen18@gmail.com>\n",
      "Authors:\n",
      "Author: Chen Meng [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.2) type: depends\n",
      "  - survminer, type: imports\n",
      "  - survival, type: imports\n",
      "  - fastmatch, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - stringr, type: imports\n",
      "  - beeswarm, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - DT, type: imports\n",
      "  - shiny, type: imports\n",
      "  - shinythemes, type: imports\n",
      "  - shinyWidgets, type: imports\n",
      "  - plotly, type: imports\n",
      "  - networkD3, type: imports\n",
      "  - httr, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - fgsea, type: imports\n",
      "  - openxlsx, type: imports\n",
      "  - psych, type: imports\n",
      "  - shinybusy, type: imports\n",
      "  - ggseqlogo, type: imports\n",
      "  - htmlwidgets, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grid, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - methods, type: imports\n",
      "  - shinyjs, type: imports\n",
      "  - curl, type: imports\n",
      "  - flatxml, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - RSQLite, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - shinycssloaders, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/omicsViewer.html\n",
      "\n",
      "Name: protGear\n",
      "Description:\n",
      "A generic three-step pre-processing package for protein microarray data. This package contains different data pre-processing procedures to allow comparison of their performance.These steps are background correction, the coefficient of variation (CV) based filtering, batch correction and normalization.\n",
      "Version: 1.2.0\n",
      "Publication date: None\n",
      "Mantainer: Kennedy Mwai <keniajin@gmail.com>\n",
      "Authors:\n",
      "Author: Kennedy Mwai [cre, aut], James Mburu [aut], Jacqueline Waeni [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 4.2) type: depends\n",
      "  - dplyr (>= 0.8.0) type: depends\n",
      "  - limma, type: depends\n",
      "  - vsn, type: depends\n",
      "  - magrittr (>= 1.5) type: imports\n",
      "  - stats (>= 3.6) type: imports\n",
      "  - ggplot2 (>= 3.3.0) type: imports\n",
      "  - tidyr (>= 1.1.3) type: imports\n",
      "  - data.table (>= 1.14.0) type: imports\n",
      "  - ggpubr (>= 0.4.0) type: imports\n",
      "  - gtools (>= 3.8.2) type: imports\n",
      "  - tibble (>= 3.1.0) type: imports\n",
      "  - rmarkdown (>= 2.9) type: imports\n",
      "  - knitr (>= 1.33) type: imports\n",
      "  - utils (>= 3.6) type: imports\n",
      "  - genefilter, type: imports\n",
      "  - readr (>= 2.0.1) type: imports\n",
      "  - Biobase, type: imports\n",
      "  - plyr (>= 1.8.6) type: imports\n",
      "  - Kendall (>= 2.2) type: imports\n",
      "  - shiny (>= 1.0.0) type: imports\n",
      "  - purrr (>= 0.3.4) type: imports\n",
      "  - plotly (>= 4.9.0) type: imports\n",
      "  - MASS (>= 7.3) type: imports\n",
      "  - htmltools (>= 0.4.0) type: imports\n",
      "  - flexdashboard (>= 0.5.2) type: imports\n",
      "  - shinydashboard (>= 0.7.1) type: imports\n",
      "  - kableExtra (>= 1.3.4) type: imports\n",
      "  - GGally (>= 2.1.2) type: imports\n",
      "  - pheatmap (>= 1.0.12) type: imports\n",
      "  - grid (>= 4.1.1) type: imports\n",
      "  - styler (>= 1.6.1) type: imports\n",
      "  - factoextra (>= 1.0.7) type: imports\n",
      "  - FactoMineR (>= 2.4) type: imports\n",
      "  - rlang (>= 0.4.11) type: imports\n",
      "  - remotes (>= 2.4.0) type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/protGear.html\n",
      "\n",
      "Name: qsvaR\n",
      "Description:\n",
      "The qsvaR package contains functions for removing the effect of degration in rna-seq data from postmortem brain tissue. The package is equipped to help users generate principal components associated with degradation. The components can be used in differential expression analysis to remove the effects of degradation.\n",
      "Version: 1.2.0\n",
      "Publication date: None\n",
      "Mantainer: Joshua Stolz <josh.stolz@libd.org>\n",
      "Authors:\n",
      "Author: Joshua Stolz [aut, cre] , Leonardo Collado-Torres [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 4.2) type: depends\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - sva, type: imports\n",
      "  - stats, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - methods, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/qsvaR.html\n",
      "\n",
      "Name: RNASeqR\n",
      "Description:\n",
      "This R package is designed for case-control RNA-Seq analysis (two-group). There are six steps: \"RNASeqRParam S4 Object Creation\", \"Environment Setup\", \"Quality Assessment\", \"Reads Alignment & Quantification\", \"Gene-level Differential Analyses\" and \"Functional Analyses\". Each step corresponds to a function in this package. After running functions in order, a basic RNASeq analysis would be done easily.\n",
      "Version: 1.15.1\n",
      "Publication date: None\n",
      "Mantainer: Kuan-Hao Chao <ntueeb05howard@gmail.com>\n",
      "Authors:\n",
      "Author: Kuan-Hao Chao\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - ggplot2, type: depends\n",
      "  - pathview, type: depends\n",
      "  - edgeR, type: depends\n",
      "  - methods, type: depends\n",
      "  - Rsamtools, type: imports\n",
      "  - tools, type: imports\n",
      "  - reticulate, type: imports\n",
      "  - ballgown, type: imports\n",
      "  - gridExtra, type: imports\n",
      "  - rafalib, type: imports\n",
      "  - FactoMineR, type: imports\n",
      "  - factoextra, type: imports\n",
      "  - corrplot, type: imports\n",
      "  - PerformanceAnalytics, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - DESeq2, type: imports\n",
      "  - systemPipeR, type: imports\n",
      "  - systemPipeRdata, type: imports\n",
      "  - clusterProfiler, type: imports\n",
      "  - org.Hs.eg.db, type: imports\n",
      "  - org.Sc.sgd.db, type: imports\n",
      "  - stringr, type: imports\n",
      "  - pheatmap, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - DOSE, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - parallel, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/RNASeqR.html\n",
      "\n",
      "Name: scatterHatch\n",
      "Description:\n",
      "The objective of this package is to efficiently create scatterplots where groups can be distinguished by color and texture. Visualizations in computational biology tend to have many groups making it difficult to distinguish between groups solely on color. Thus, this package is useful for increasing the accessibility of scatterplot visualizations to those with visual impairments such as color blindness.\n",
      "Version: 1.4.0\n",
      "Publication date: None\n",
      "Mantainer: Atul Deshpande <adeshpande@jhu.edu>\n",
      "Authors:\n",
      "Author: Atul Deshpande [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - grid, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - plyr, type: imports\n",
      "  - spatstat.geom, type: imports\n",
      "  - stats, type: imports\n",
      "  - grDevices, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/scatterHatch.html\n",
      "\n",
      "Name: TrajectoryGeometry\n",
      "Description:\n",
      "Given a time series or pseudo-times series of gene expression data, we might wish to know: Do the changes in gene expression in these data exhibit directionality? Are there turning points in this directionality. Do different subsets of the data move in different directions? This package uses spherical geometry to probe these sorts of questions. In particular, if we are looking at (say) the first n dimensions of the PCA of gene expression, directionality can be detected as the clustering of points on the (n-1)-dimensional sphere.\n",
      "Version: 1.6.0\n",
      "Publication date: None\n",
      "Mantainer: Michael Shapiro <michael.shapiro@crick.ac.uk>\n",
      "Authors:\n",
      "Author: Michael Shapiro [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - pracma, type: imports\n",
      "  - rgl, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - stats, type: imports\n",
      "  - methods, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/TrajectoryGeometry.html\n",
      "\n",
      "Name: updateObject\n",
      "Description:\n",
      "A set of tools built around updateObject() to work with old serialized S4 instances. The package is primarily useful to package maintainers who want to update the serialized S4 instances included in their package. This is still work-in-progress.\n",
      "Version: 1.2.0\n",
      "Publication date: None\n",
      "Mantainer: HervÃ© PagÃ¨s <hpages.on.github@gmail.com>\n",
      "Authors:\n",
      "Author: HervÃ© PagÃ¨s [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.2.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - BiocGenerics, type: depends\n",
      "  - S4Vectors, type: depends\n",
      "  - utils, type: imports\n",
      "  - digest, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/updateObject.html\n",
      "\n",
      "Name: xcore\n",
      "Description:\n",
      "xcore is an R package for transcription factor activity modeling based on known molecular signatures and user's gene expression data. Accompanying xcoredata package provides a collection of molecular signatures, constructed from publicly available ChiP-seq experiments. xcore use ridge regression to model changes in expression as a linear combination of molecular signatures and find their unknown activities. Obtained, estimates can be further tested for significance to select molecular signatures with the highest predicted effect on the observed expression changes.\n",
      "Version: 1.2.0\n",
      "Publication date: None\n",
      "Mantainer: Maciej MigdaÅ <mcjmigdal@gmail.com>\n",
      "Authors:\n",
      "Author: Maciej MigdaÅ [aut, cre] , BogumiÅ Kaczkowski [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.2) type: depends\n",
      "  - DelayedArray, type: imports\n",
      "  - edgeR, type: imports\n",
      "  - foreach (>= 1.5.1) type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - glmnet (>= 4.1.2) type: imports\n",
      "  - IRanges, type: imports\n",
      "  - iterators (>= 1.0.13) type: imports\n",
      "  - magrittr (>= 2.0.1) type: imports\n",
      "  - Matrix (>= 1.3.4) type: imports\n",
      "  - methods (>= 4.1.1) type: imports\n",
      "  - MultiAssayExperiment, type: imports\n",
      "  - stats, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/xcore.html\n",
      "\n",
      "Name: biodbMirbase\n",
      "Description:\n",
      "The biodbMirbase library is an extension of the biodb framework package, that provides access to miRBase mature database. It allows to retrieve entries by their accession number, and run specific web services. Description: The biodbMirbase library provides access to the miRBase Database, using biodb package framework.\n",
      "Version: 1.2.2\n",
      "Publication date: None\n",
      "Mantainer: Pierrick Roger <pierrick.roger@cea.fr>\n",
      "Authors:\n",
      "Author: Pierrick Roger [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - biodb, type: imports\n",
      "  - R6, type: imports\n",
      "  - stringr, type: imports\n",
      "  - chk, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "AGPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/biodbMirbase.html\n",
      "\n",
      "Name: GenomicInteractionNodes\n",
      "Description:\n",
      "The GenomicInteractionNodes package can import interactions from bedpe file and define the interaction nodes, the genomic interaction sites with multiple interaction loops. The interaction nodes is a binding platform regulates one or multiple genes. The detected interaction nodes will be annotated for downstream validation.\n",
      "Version: 1.2.0\n",
      "Publication date: None\n",
      "Mantainer: Jianhong Ou <jianhong.ou@duke.edu>\n",
      "Authors:\n",
      "Author: Jianhong Ou [aut, cre], Yarui Diao [fnd]\n",
      "Dependencies:\n",
      "  - R (>= 4.2.0) type: depends\n",
      "  - stats, type: depends\n",
      "  - AnnotationDbi, type: imports\n",
      "  - graph, type: imports\n",
      "  - GO.db, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - GenomicFeatures, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - methods, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - RBGL, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/GenomicInteractionNodes.html\n",
      "\n",
      "Name: HTSeqGenie\n",
      "Description:\n",
      "Libraries to perform NGS analysis.\n",
      "Version: 4.28.1\n",
      "Publication date: None\n",
      "Mantainer: Jens Reeder <reeder.jens@gene.com>\n",
      "Authors:\n",
      "Author: Gregoire Pau, Jens Reeder\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - gmapR, type: depends\n",
      "  - ShortRead, type: depends\n",
      "  - VariantAnnotation, type: depends\n",
      "  - BiocGenerics, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - chipseq, type: imports\n",
      "  - hwriter (>= 1.3.0) type: imports\n",
      "  - Cairo (>= 1.5.5) type: imports\n",
      "  - GenomicFeatures, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - parallel, type: imports\n",
      "  - tools, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - GenomicAlignments, type: imports\n",
      "  - VariantTools, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - methods, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/HTSeqGenie.html\n",
      "\n",
      "Name: inveRsion\n",
      "Description:\n",
      "Bioconductor version: Release (3.16)\n",
      "Version: 1.46.2\n",
      "Publication date: None\n",
      "Mantainer: Author: Alejandro Caceres\n",
      "Authors:\n",
      "Package to find genetic inversions in genotype (SNP array) data.\n",
      "Dependencies:\n",
      "  - methods, type: depends\n",
      "  - haplo.stats, type: depends\n",
      "  - graphics, type: imports\n",
      "  - methods, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/inveRsion.html\n",
      "\n",
      "Name: pareg\n",
      "Description:\n",
      "Compute pathway enrichment scores while accounting for term-term relations. This package uses a regularized multiple linear regression to regress differential expression p-values obtained from multi-condition experiments on a pathway membership matrix. By doing so, it is able to incorporate additional biological knowledge into the enrichment analysis and to estimate pathway enrichment scores more robustly.\n",
      "Version: 1.2.0\n",
      "Publication date: None\n",
      "Mantainer: Kim Philipp Jablonski <kim.philipp.jablonski@gmail.com>\n",
      "Authors:\n",
      "Author: Kim Philipp Jablonski [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.2) type: depends\n",
      "  - tensorflow (>= 2.2.0) type: depends\n",
      "  - tfprobability (>= 0.10.0) type: depends\n",
      "  - stats, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - purrr, type: imports\n",
      "  - furrr, type: imports\n",
      "  - tibble, type: imports\n",
      "  - glue, type: imports\n",
      "  - tidygraph, type: imports\n",
      "  - igraph, type: imports\n",
      "  - proxy, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - ggraph, type: imports\n",
      "  - rlang, type: imports\n",
      "  - progress, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - matrixLaplacian, type: imports\n",
      "  - keras, type: imports\n",
      "  - nloptr, type: imports\n",
      "  - shadowtext, type: imports\n",
      "  - methods, type: imports\n",
      "  - DOSE, type: imports\n",
      "  - stringr, type: imports\n",
      "  - reticulate, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/pareg.html\n",
      "\n",
      "Name: RolDE\n",
      "Description:\n",
      "RolDE detects longitudinal differential expression between two conditions in noisy high-troughput data. Suitable even for data with a moderate amount of missing values.RolDE is a composite method, consisting of three independent modules with different approaches to detecting longitudinal differential expression. The combination of these diverse modules allows RolDE to robustly detect varying differences in longitudinal trends and expression levels in diverse data types and experimental settings.\n",
      "Version: 1.2.0\n",
      "Publication date: None\n",
      "Mantainer: Tommi Valikangas <tsvali@utu.fi>\n",
      "Authors:\n",
      "Author: Tommi Valikangas [cre, aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.2.0) type: depends\n",
      "  - stats, type: imports\n",
      "  - methods, type: imports\n",
      "  - ROTS, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - foreach, type: imports\n",
      "  - parallel, type: imports\n",
      "  - doParallel, type: imports\n",
      "  - doRNG, type: imports\n",
      "  - rngtools, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - nlme, type: imports\n",
      "  - qvalue, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/RolDE.html\n",
      "\n",
      "Name: syntenet\n",
      "Description:\n",
      "syntenet can be used to infer synteny networks from whole-genome protein sequences and analyze them. Anchor pairs are detected with the MCScanX algorithm, which was ported to this package with the Rcpp framework for R and C++ integration. Anchor pairs from synteny analyses are treated as an undirected unweighted graph (i.e., a synteny network), and users can perform: i. network clustering; ii. phylogenomic profiling (by identifying which species contain which clusters) and; iii. microsynteny-based phylogeny reconstruction with maximum likelihood.\n",
      "Version: 1.0.4\n",
      "Publication date: None\n",
      "Mantainer: FabrÃ­cio Almeida-Silva <fabricio_almeidasilva@hotmail.com>\n",
      "Authors:\n",
      "Author: FabrÃ­cio Almeida-Silva [aut, cre] , Tao Zhao [aut] , Kristian K Ullrich [aut] , Yves Van de Peer [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.2) type: depends\n",
      "  - Rcpp (>= 1.0.8) type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - rlang, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - utils, type: imports\n",
      "  - methods, type: imports\n",
      "  - igraph, type: imports\n",
      "  - stats, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - pheatmap, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - ggnetwork, type: imports\n",
      "  - intergraph, type: imports\n",
      "  - networkD3, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/syntenet.html\n",
      "\n",
      "Name: biodbExpasy\n",
      "Description:\n",
      "The biodbExpasy library provides access to Expasy ENZYME database, using biodb package framework. It allows to retrieve entries by their accession number. Web services can be accessed for searching the database by name or comments.\n",
      "Version: 1.2.0\n",
      "Publication date: None\n",
      "Mantainer: Pierrick Roger <pierrick.roger@cea.fr>\n",
      "Authors:\n",
      "Author: Pierrick Roger [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - biodb, type: imports\n",
      "  - R6, type: imports\n",
      "  - stringr, type: imports\n",
      "  - chk, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "AGPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/biodbExpasy.html\n",
      "\n",
      "Name: biodbNci\n",
      "Description:\n",
      "The biodbNci library is an extension of the biodb framework package. It provides access to biodbNci, a library for connecting to the National Cancer Institute (USA) CACTUS Database. It allows to retrieve entries by their accession number, and run specific web services.\n",
      "Version: 1.2.0\n",
      "Publication date: None\n",
      "Mantainer: Pierrick Roger <pierrick.roger@cea.fr>\n",
      "Authors:\n",
      "Author: Pierrick Roger [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - biodb, type: imports\n",
      "  - R6, type: imports\n",
      "  - Rcpp, type: imports\n",
      "  - chk, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "AGPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/biodbNci.html\n",
      "\n",
      "Name: borealis\n",
      "Description:\n",
      "Borealis is an R library performing outlier analysis for count-based bisulfite sequencing data. It detectes outlier methylated CpG sites from bisulfite sequencing (BS-seq). The core of Borealis is modeling Beta-Binomial distributions. This can be useful for rare disease diagnoses.\n",
      "Version: 1.2.2\n",
      "Publication date: None\n",
      "Mantainer: Garrett Jenkinson <gargar934@gmail.com>\n",
      "Authors:\n",
      "Author: Garrett Jenkinson [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.2.0) type: depends\n",
      "  - Biobase, type: depends\n",
      "  - doParallel, type: imports\n",
      "  - snow, type: imports\n",
      "  - purrr, type: imports\n",
      "  - plyr, type: imports\n",
      "  - foreach, type: imports\n",
      "  - gamlss, type: imports\n",
      "  - gamlss.dist, type: imports\n",
      "  - bsseq, type: imports\n",
      "  - methods, type: imports\n",
      "  - DSS, type: imports\n",
      "  - R.utils, type: imports\n",
      "  - utils, type: imports\n",
      "  - stats, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - cowplot, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - rlang, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/borealis.html\n",
      "\n",
      "Name: crisprBwa\n",
      "Description:\n",
      "Provides a user-friendly interface to map on-targets and off-targets of CRISPR gRNA spacer sequences using bwa. The alignment is fast, and can be performed using either commonly-used or custom CRISPR nucleases. The alignment can work with any reference or custom genomes. Currently not supported on Windows machines.\n",
      "Version: 1.2.0\n",
      "Publication date: None\n",
      "Mantainer: Jean-Philippe Fortin <fortin946@gmail.com>\n",
      "Authors:\n",
      "Author: Jean-Philippe Fortin [aut, cre]\n",
      "Dependencies:\n",
      "  - methods, type: depends\n",
      "  - BiocGenerics, type: imports\n",
      "  - BSgenome, type: imports\n",
      "  - crisprBase, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - Rbwa, type: imports\n",
      "  - readr, type: imports\n",
      "  - stats, type: imports\n",
      "  - stringr, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/crisprBwa.html\n",
      "\n",
      "Name: DepInfeR\n",
      "Description:\n",
      "DepInfeR integrates two experimentally accessible input data matrices: the drug sensitivity profiles of cancer cell lines or primary tumors ex-vivo (X), and the drug affinities of a set of proteins (Y), to infer a matrix of molecular protein dependencies of the cancers (Ã). DepInfeR deconvolutes the protein inhibition effect on the viability phenotype by using regularized multivariate linear regression. It assigns a âdependence coefficientâ to each protein and each sample, and therefore could be used to gain a causal and accurate understanding of functional consequences of genomic aberrations in a heterogeneous disease, as well as to guide the choice of pharmacological intervention for a specific cancer type, sub-type, or an individual patient. For more information, please read out preprint on bioRxiv: https://doi.org/10.1101/2022.01.11.475864.\n",
      "Version: 1.2.0\n",
      "Publication date: None\n",
      "Mantainer: Junyan Lu <jylu1118@gmail.com>\n",
      "Authors:\n",
      "Author: Junyan Lu [aut, cre] , Alina Batzilla [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.2.0) type: depends\n",
      "  - matrixStats, type: imports\n",
      "  - glmnet, type: imports\n",
      "  - stats, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/DepInfeR.html\n",
      "\n",
      "Name: GenProSeq\n",
      "Description:\n",
      "Generative modeling for protein engineering is key to solving fundamental problems in synthetic biology, medicine, and material science. Machine learning has enabled us to generate useful protein sequences on a variety of scales. Generative models are machine learning methods which seek to model the distribution underlying the data, allowing for the generation of novel samples with similar properties to those on which the model was trained. Generative models of proteins can learn biologically meaningful representations helpful for a variety of downstream tasks. Furthermore, they can learn to generate protein sequences that have not been observed before and to assign higher probability to protein sequences that satisfy desired criteria. In this package, common deep generative models for protein sequences, such as variational autoencoder (VAE), generative adversarial networks (GAN), and autoregressive models are available. In the VAE and GAN, the Word2vec is used for embedding. The transformer encoder is applied to protein sequences for the autoregressive model.\n",
      "Version: 1.2.0\n",
      "Publication date: None\n",
      "Mantainer: Dongmin Jung <dmdmjung@gmail.com>\n",
      "Authors:\n",
      "Author: Dongmin Jung [cre, aut]\n",
      "Dependencies:\n",
      "  - keras, type: depends\n",
      "  - mclust, type: depends\n",
      "  - R (>= 4.2) type: depends\n",
      "  - tensorflow, type: imports\n",
      "  - word2vec, type: imports\n",
      "  - DeepPINCS, type: imports\n",
      "  - ttgsea, type: imports\n",
      "  - CatEncoders, type: imports\n",
      "  - reticulate, type: imports\n",
      "  - stats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/GenProSeq.html\n",
      "\n",
      "Name: mbOmic\n",
      "Description:\n",
      "The mbOmic package contains a set of analysis functions for microbiomics and metabolomics data, designed to analyze the inter-omic correlation between microbiology and metabolites. Integrative analysis of the microbiome and metabolome is the aim of mbOmic. Additionally, the identification of enterotype using the gut microbiota abundance is preliminaryimplemented.\n",
      "Version: 1.2.0\n",
      "Publication date: None\n",
      "Mantainer: Congcong Gong <congconggong33@gmail.com>\n",
      "Authors:\n",
      "Author: Congcong Gong [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.1.0) type: depends\n",
      "  - parallel, type: imports\n",
      "  - doParallel, type: imports\n",
      "  - psych, type: imports\n",
      "  - WGCNA, type: imports\n",
      "  - data.table, type: imports\n",
      "  - igraph, type: imports\n",
      "  - visNetwork, type: imports\n",
      "  - cluster, type: imports\n",
      "  - clusterSim, type: imports\n",
      "  - methods, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/mbOmic.html\n",
      "\n",
      "Name: MobilityTransformR\n",
      "Description:\n",
      "MobilityTransformR collects a tool set for effective mobility scale transformation of CE-MS/MS data in order to increase reproducibility. It provides functionality to determine the migration times from mobility markers that have been added to the analysis and performs the transformation based on these markers. MobilityTransformR supports the conversion of numeric vectors, Spectra-objects, and MSnOnDiskExp.\n",
      "Version: 1.2.0\n",
      "Publication date: None\n",
      "Mantainer: Liesa Salzer <liesa.salzer@helmholtz-muenchen.de>\n",
      "Authors:\n",
      "Author: Liesa Salzer [cre, aut]\n",
      "Dependencies:\n",
      "  - MSnbase, type: depends\n",
      "  - R (>= 4.2) type: depends\n",
      "  - xcms, type: imports\n",
      "  - MetaboCoreUtils, type: imports\n",
      "  - Spectra, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/MobilityTransformR.html\n",
      "\n",
      "Name: RGMQL\n",
      "Description:\n",
      "This package brings the GenoMetric Query Language (GMQL) functionalities into the R environment. GMQL is a high-level, declarative language to manage heterogeneous genomic datasets for biomedical purposes, using simple queries to process genomic regions and their metadata and properties. GMQL adopts algorithms efficiently designed for big data using cloud-computing technologies (like Apache Hadoop and Spark) allowing GMQL to run on modern infrastructures, in order to achieve scalability and high performance. It allows to create, manipulate and extract genomic data from different data sources both locally and remotely. Our RGMQL functions allow complex queries and processing leveraging on the R idiomatic paradigm. The RGMQL package also provides a rich set of ancillary classes that allow sophisticated input/output management and sorting, such as: ASC, DESC, BAG, MIN, MAX, SUM, AVG, MEDIAN, STD, Q1, Q2, Q3 (and many others). Note that many RGMQL functions are not directly executed in R environment, but are deferred until real execution is issued.\n",
      "Version: 1.18.0\n",
      "Publication date: None\n",
      "Mantainer: Simone Pallotta <simonepallotta@hotmail.com>\n",
      "Authors:\n",
      "Author: Simone Pallotta [aut, cre], Marco Masseroli [aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.4.2) type: depends\n",
      "  - RGMQLlib, type: depends\n",
      "  - httr, type: imports\n",
      "  - rJava, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - data.table, type: imports\n",
      "  - utils, type: imports\n",
      "  - plyr, type: imports\n",
      "  - xml2, type: imports\n",
      "  - methods, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - stats, type: imports\n",
      "  - glue, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/RGMQL.html\n",
      "\n",
      "Name: SpatialFeatureExperiment\n",
      "Description:\n",
      "A new S4 class integrating Simple Features with the R package sf to bring geospatial data analysis methods based on vector data to spatial transcriptomics. Also implements management of spatial neighborhood graphs and geometric operations. This pakage builds upon SpatialExperiment and SingleCellExperiment, hence methods for these parent classes can still be used.\n",
      "Version: 1.0.3\n",
      "Publication date: None\n",
      "Mantainer: Lambda Moses <dlu2@caltech.edu>\n",
      "Authors:\n",
      "Author: Lambda Moses [aut, cre] , Lior Pachter [aut, ths]\n",
      "Dependencies:\n",
      "  - R (>= 4.2.0) type: depends\n",
      "  - BiocGenerics, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - methods, type: imports\n",
      "  - rjson, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - sf, type: imports\n",
      "  - SingleCellExperiment, type: imports\n",
      "  - SpatialExperiment, type: imports\n",
      "  - spdep (>= 1.1-7) type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/SpatialFeatureExperiment.html\n",
      "\n",
      "Name: TREG\n",
      "Description:\n",
      "RNA abundance and cell size parameters could improve RNA-seq deconvolution algorithms to more accurately estimate cell type proportions given the different cell type transcription activity levels. A Total RNA Expression Gene (TREG) can facilitate estimating total RNA content using single molecule fluorescent in situ hybridization (smFISH). We developed a data-driven approach using a measure of expression invariance to find candidate TREGs in postmortem human brain single nucleus RNA-seq. This R package implements the method for identifying candidate TREGs from snRNA-seq data.\n",
      "Version: 1.2.0\n",
      "Publication date: None\n",
      "Mantainer: Louise Huuki-Myers <lahuuki@gmail.com>\n",
      "Authors:\n",
      "Author: Louise Huuki-Myers [aut, cre] , Leonardo Collado-Torres [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 4.2) type: depends\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - Matrix, type: imports\n",
      "  - purrr, type: imports\n",
      "  - rafalib, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/TREG.html\n",
      "\n",
      "Name: single\n",
      "Description:\n",
      "Accurate consensus sequence from nanopore reads of a DNA gene library. SINGLe corrects for systematic errors in nanopore sequencing reads of gene libraries and it retrieves true consensus sequences of variants identified by a barcode, needing only a few reads per variant. More information in preprint doi: https://doi.org/10.1101/2020.03.25.007146.\n",
      "Version: 1.2.0\n",
      "Publication date: None\n",
      "Mantainer: Rocio Espada <rocio.espada@espci.fr>\n",
      "Authors:\n",
      "Author: Rocio Espada [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.1) type: depends\n",
      "  - Biostrings, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - GenomicAlignments, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - methods, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - rlang, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - stats, type: imports\n",
      "  - stringr, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/single.html\n",
      "\n",
      "Name: terraTCGAdata\n",
      "Description:\n",
      "Leverage the existing open access TCGA data on Terra with well-established Bioconductor infrastructure. Make use of the Terra data model without learning its complexities. With a few functions, you can copy / download and generate a MultiAssayExperiment from the TCGA example workspaces provided by Terra.\n",
      "Version: 1.2.0\n",
      "Publication date: None\n",
      "Mantainer: Marcel Ramos <marcel.ramosperez@roswellpark.org>\n",
      "Authors:\n",
      "Author: Marcel Ramos [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.2.0) type: depends\n",
      "  - AnVIL, type: depends\n",
      "  - MultiAssayExperiment, type: depends\n",
      "  - BiocFileCache, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - methods, type: imports\n",
      "  - RaggedExperiment, type: imports\n",
      "  - readr, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - stats, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - TCGAutils, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/terraTCGAdata.html\n",
      "\n",
      "Name: Travel\n",
      "Description:\n",
      "Creates a virtual pointer for R's ALTREP object which does not have the data allocates in memory. The pointer is made by the file mapping of a virtual file so it behaves exactly the same as a regular pointer. All the requests to access the pointer will be sent to the underlying file system and eventually handled by a customized data-reading function. The main purpose of the package is to reduce the memory consumption when using R's vector to represent a large data. The use cases of the package include on-disk data representation, compressed vector(e.g. RLE) and etc.\n",
      "Version: 1.6.0\n",
      "Publication date: None\n",
      "Mantainer: Jiefei Wang <szwjf08@gmail.com>\n",
      "Authors:\n",
      "Author: Jiefei Wang [aut, cre], Martin Morgan [aut]\n",
      "Dependencies:\n",
      "  - Rcpp, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/Travel.html\n",
      "\n",
      "Name: HilbertVisGUI\n",
      "Description:\n",
      "An interactive tool to visualize long vectors of integer data by means of Hilbert curves\n",
      "Version: 1.56.0\n",
      "Publication date: None\n",
      "Mantainer: Simon Anders <sanders@fs.tum.de>\n",
      "Authors:\n",
      "Author: Simon Anders <sanders at fs.tum.de>\n",
      "Dependencies:\n",
      "  - R (>= 2.6.0) type: depends\n",
      "  - HilbertVis, type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 3)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/HilbertVisGUI.html\n",
      "\n",
      "Name: Macarron\n",
      "Description:\n",
      "Macarron is a workflow for the prioritization of potentially bioactive metabolites from metabolomics experiments. Prioritization integrates strengths of evidences of bioactivity such as covariation with a known metabolite, abundance relative to a known metabolite and association with an environmental or phenotypic indicator of bioactivity. Broadly, the workflow consists of stratified clustering of metabolic spectral features which co-vary in abundance in a condition, transfer of functional annotations, estimation of relative abundance and differential abundance analysis to identify associations between features and phenotype/condition.\n",
      "Version: 1.2.0\n",
      "Publication date: None\n",
      "Mantainer: Sagun Maharjan <sagunmaharjann@gmail.com>\n",
      "Authors:\n",
      "Author: Amrisha Bhosle [aut], Ludwig Geistlinger [aut], Sagun Maharjan [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.2.0) type: depends\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - BiocParallel, type: imports\n",
      "  - DelayedArray, type: imports\n",
      "  - WGCNA, type: imports\n",
      "  - ff, type: imports\n",
      "  - data.table, type: imports\n",
      "  - dynamicTreeCut, type: imports\n",
      "  - Maaslin2, type: imports\n",
      "  - plyr, type: imports\n",
      "  - stats, type: imports\n",
      "  - psych, type: imports\n",
      "  - xml2, type: imports\n",
      "  - RCurl, type: imports\n",
      "  - RJSONIO, type: imports\n",
      "  - logging, type: imports\n",
      "  - methods, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/Macarron.html\n",
      "\n",
      "Name: PrecisionTrialDrawer\n",
      "Description:\n",
      "A suite of methods to design umbrella and basket trials for precision oncology.\n",
      "Version: 1.14.0\n",
      "Publication date: None\n",
      "Mantainer: Giorgio Melloni <melloni.giorgio@gmail.com>\n",
      "Authors:\n",
      "Author: Giorgio Melloni, Alessandro Guida, Luca Mazzarella\n",
      "Dependencies:\n",
      "  - R (>= 3.6) type: depends\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - methods, type: imports\n",
      "  - cBioPortalData, type: imports\n",
      "  - parallel, type: imports\n",
      "  - stringr, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - data.table, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - biomaRt, type: imports\n",
      "  - XML, type: imports\n",
      "  - httr, type: imports\n",
      "  - jsonlite, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - ggrepel, type: imports\n",
      "  - grid, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - LowMACAAnnotation, type: imports\n",
      "  - googleVis, type: imports\n",
      "  - shiny, type: imports\n",
      "  - shinyBS, type: imports\n",
      "  - DT, type: imports\n",
      "  - brglm, type: imports\n",
      "  - matrixStats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/PrecisionTrialDrawer.html\n",
      "\n",
      "Name: pulsedSilac\n",
      "Description:\n",
      "This package provides several tools for pulsed-SILAC data analysis. Functions are provided to organize the data, calculate isotope ratios, isotope fractions, model protein turnover, compare turnover models, estimate cell growth and estimate isotope recycling. Several visualization tools are also included to do basic data exploration, quality control, condition comparison, individual model inspection and model comparison.\n",
      "Version: 1.12.0\n",
      "Publication date: None\n",
      "Mantainer: Marc PagÃ¨s-Gallego <M.PagesGallego@umcutrecht.nl>\n",
      "Authors:\n",
      "Author: Marc PagÃ¨s-Gallego, Tobias B. Dansen\n",
      "Dependencies:\n",
      "  - R (>= 3.6.0) type: depends\n",
      "  - robustbase, type: imports\n",
      "  - methods, type: imports\n",
      "  - R.utils, type: imports\n",
      "  - taRifx, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - ggridges, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - UpSetR, type: imports\n",
      "  - cowplot, type: imports\n",
      "  - grid, type: imports\n",
      "  - MuMIn, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/pulsedSilac.html\n",
      "\n",
      "Name: RAREsim\n",
      "Description:\n",
      "Haplotype simulations of rare variant genetic data that emulates real data can be performed with RAREsim. RAREsim uses the expected number of variants in MAC bins - either as provided by default parameters or estimated from target data - and an abundance of rare variants as simulated HAPGEN2 to probabilistically prune variants. RAREsim produces haplotypes that emulate real sequencing data with respect to the total number of variants, allele frequency spectrum, haplotype structure, and variant annotation.\n",
      "Version: 1.2.0\n",
      "Publication date: None\n",
      "Mantainer: Ryan Barnard <rbarnard1107@gmail.com>\n",
      "Authors:\n",
      "Author: Megan Null [aut], Ryan Barnard [cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.1.0) type: depends\n",
      "  - nloptr, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/RAREsim.html\n",
      "\n",
      "Name: tomoseqr\n",
      "Description:\n",
      "`tomoseqr` is an R package for analyzing Tomo-seq data. Tomo-seq is a genome-wide RNA tomography method that combines combining high-throughput RNA sequencing with cryosectioning for spatially resolved transcriptomics. `tomoseqr` reconstructs 3D expression patterns from tomo-seq data and visualizes the reconstructed 3D expression patterns.\n",
      "Version: 1.2.0\n",
      "Publication date: None\n",
      "Mantainer: Ryosuke Matsuzawa <kinakomochi.work@gmail.com>\n",
      "Authors:\n",
      "Author: Ryosuke Matsuzawa [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.2) type: depends\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - animation, type: imports\n",
      "  - tibble, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - stringr, type: imports\n",
      "  - purrr, type: imports\n",
      "  - methods, type: imports\n",
      "  - shiny, type: imports\n",
      "  - BiocFileCache, type: imports\n",
      "  - readr, type: imports\n",
      "  - tools, type: imports\n",
      "  - plotly, type: imports\n",
      "  - ggplot2, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/tomoseqr.html\n",
      "\n",
      "Name: SpotClean\n",
      "Description:\n",
      "SpotClean is a computational method to adjust for spot swapping in spatial transcriptomics data. Recent spatial transcriptomics experiments utilize slides containing thousands of spots with spot-specific barcodes that bind mRNA. Ideally, unique molecular identifiers at a spot measure spot-specific expression, but this is often not the case due to bleed from nearby spots, an artifact we refer to as spot swapping. SpotClean is able to estimate the contamination rate in observed data and decontaminate the spot swapping effect, thus increase the sensitivity and precision of downstream analyses.\n",
      "Version: 1.0.1\n",
      "Publication date: None\n",
      "Mantainer: Zijian Ni <zni25@wisc.edu>\n",
      "Authors:\n",
      "Author: Zijian Ni [aut, cre] , Christina Kendziorski [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 4.2.0) type: depends\n",
      "  - stats, type: imports\n",
      "  - methods, type: imports\n",
      "  - utils, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - SpatialExperiment, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - rhdf5, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - grid, type: imports\n",
      "  - readbitmap, type: imports\n",
      "  - rjson, type: imports\n",
      "  - tibble, type: imports\n",
      "  - viridis, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - Seurat, type: imports\n",
      "  - rlang, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/SpotClean.html\n",
      "\n",
      "Name: iteremoval\n",
      "Description:\n",
      "Bioconductor version: Release (3.16)\n",
      "Version: 1.18.0\n",
      "Publication date: None\n",
      "Mantainer: Author: Jiacheng Chuan [aut, cre]\n",
      "Authors:\n",
      "The package provides a flexible algorithm to screen features of two distinct groups in consideration of overfitting and overall performance. It was originally tailored for methylation locus screening of NGS data, and it can also be used as a generic method for feature selection. Each step of the algorithm provides a default method for simple implemention, and the method can be replaced by a user defined function.\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - ggplot2 (>= 2.2.1) type: depends\n",
      "  - magrittr, type: imports\n",
      "  - graphics, type: imports\n",
      "  - utils, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/iteremoval.html\n",
      "\n",
      "Name: coexnet\n",
      "Description:\n",
      "Bioconductor version: Release (3.16)\n",
      "Version: 1.19.1\n",
      "Publication date: None\n",
      "Mantainer: Author: Juan David Henao [aut,cre], Liliana Lopez-Kleine [aut], Andres Pinzon-Velasco [aut]\n",
      "Authors:\n",
      "Extracts the gene expression matrix from GEO DataSets (.CEL files) as a AffyBatch object. Additionally, can make the normalization process using two different methods (vsn and rma). The summarization (pass from multi-probe to one gene) uses two different criteria (Maximum value and Median of the samples expression data) and the process of gene differentially expressed analisys using two methods (sam and acde). The construction of the co-expression network can be conduced using two different methods, Pearson Correlation Coefficient (PCC) or Mutual Information (MI) and choosing a threshold value using a graph theory approach.\n",
      "Dependencies:\n",
      "  - R (>= 3.6) type: depends\n",
      "  - affy, type: imports\n",
      "  - siggenes, type: imports\n",
      "  - GEOquery, type: imports\n",
      "  - vsn, type: imports\n",
      "  - igraph, type: imports\n",
      "  - acde, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - limma, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - STRINGdb, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - minet, type: imports\n",
      "  - rmarkdown, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/coexnet.html\n",
      "\n",
      "Name: crisprVerse\n",
      "Description:\n",
      "The crisprVerse is a modular ecosystem of R packages developed for the design and manipulation of CRISPR guide RNAs (gRNAs). All packages share a common language and design principles. This package is designed to make it easy to install and load the crisprVerse packages in a single step. To learn more about the crisprVerse, visit .\n",
      "Version: 1.0.0\n",
      "Publication date: None\n",
      "Mantainer: Jean-Philippe Fortin <fortin946@gmail.com>\n",
      "Authors:\n",
      "Author: Jean-Philippe Fortin [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.2.0) type: depends\n",
      "  - BiocManager, type: imports\n",
      "  - cli, type: imports\n",
      "  - crisprBase, type: imports\n",
      "  - crisprBowtie, type: imports\n",
      "  - crisprScore, type: imports\n",
      "  - crisprDesign, type: imports\n",
      "  - crisprViz, type: imports\n",
      "  - rlang, type: imports\n",
      "  - tools, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/crisprVerse.html\n",
      "\n",
      "Name: crisprViz\n",
      "Description:\n",
      "Provides functionalities to visualize and contextualize CRISPR guide RNAs (gRNAs) on genomic tracks across nucleases and applications. Works in conjunction with the crisprBase and crisprDesign Bioconductor packages. Plots are produced using the Gviz framework.\n",
      "Version: 1.0.0\n",
      "Publication date: None\n",
      "Mantainer: Jean-Philippe Fortin <fortin946@gmail.com>\n",
      "Authors:\n",
      "Author: Jean-Philippe Fortin [aut, cre], Luke Hoberecht [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.2.0) type: depends\n",
      "  - crisprBase, type: depends\n",
      "  - crisprDesign, type: depends\n",
      "  - BiocGenerics, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - BSgenome, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - GenomicFeatures, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - Gviz, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - methods, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/crisprViz.html\n",
      "\n",
      "Name: rifi\n",
      "Description:\n",
      "'rifi' analyses data from rifampicin time series created by microarray or RNAseq. 'rifi' is a transcriptome data analysis tool for the holistic identification of transcription and decay associated processes. The decay constants and the delay of the onset of decay is fitted for each probe/bin. Subsequently, probes/bins of equal properties are combined into segments by dynamic programming, independent of a existing genome annotation. This allows to detect transcript segments of different stability or transcriptional events within one annotated gene. In addition to the classic decay constant/half-life analysis, 'rifi' detects processing sites, transcription pausing sites, internal transcription start sites in operons, sites of partial transcription termination in operons, identifies areas of likely transcriptional interference by the collision mechanism and gives an estimate of the transcription velocity. All data are integrated to give an estimate of continous transcriptional units, i.e. operons. Comprehensive output tables and visualizations of the full genome result and the individual fits for all probes/bins are produced.\n",
      "Version: 1.2.0\n",
      "Publication date: None\n",
      "Mantainer: Jens Georg <jens.georg@biologie.uni-freiburg.de>\n",
      "Authors:\n",
      "Author: Loubna Youssar [aut, ctb], Walja Wanney [aut, ctb], Jens Georg [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.2) type: depends\n",
      "  - car, type: imports\n",
      "  - cowplot, type: imports\n",
      "  - doMC, type: imports\n",
      "  - parallel, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - egg, type: imports\n",
      "  - foreach, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - grid, type: imports\n",
      "  - methods, type: imports\n",
      "  - nls2, type: imports\n",
      "  - nnet, type: imports\n",
      "  - rlang, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - scales, type: imports\n",
      "  - stats, type: imports\n",
      "  - stringr, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - tibble, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3 + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/rifi.html\n",
      "\n",
      "Name: SPIAT\n",
      "Description:\n",
      "SPIAT (**Sp**atial **I**mage **A**nalysis of **T**issues) is an R package with a suite of data processing, quality control, visualization and data analysis tools. SPIAT is compatible with data generated from single-cell spatial proteomics platforms (e.g. OPAL, CODEX, MIBI, cellprofiler). SPIAT reads spatial data in the form of X and Y coordinates of cells, marker intensities and cell phenotypes. SPIAT includes six analysis modules that allow visualization, calculation of cell colocalization, categorization of the immune microenvironment relative to tumor areas, analysis of cellular neighborhoods, and the quantification of spatial heterogeneity, providing a comprehensive toolkit for spatial data analysis.\n",
      "Version: 1.0.4\n",
      "Publication date: None\n",
      "Mantainer: Yuzhou Feng <yuzhou.feng@petermac.org>\n",
      "Authors:\n",
      "Author: Anna Trigos [aut], Yuzhou Feng [aut, cre], Tianpei Yang [aut], Mabel Li [aut], John Zhu [aut], Volkan Ozcoban [aut], Maria Doyle [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.2.0) type: depends\n",
      "  - SpatialExperiment, type: depends\n",
      "  - apcluster (>= 1.4.7) type: imports\n",
      "  - ggplot2 (>= 3.2.1) type: imports\n",
      "  - gridExtra (>= 2.3) type: imports\n",
      "  - gtools (>= 3.8.1) type: imports\n",
      "  - reshape2 (>= 1.4.3) type: imports\n",
      "  - dplyr (>= 0.8.3) type: imports\n",
      "  - plotly (>= 4.9.0) type: imports\n",
      "  - RANN (>= 2.6.1) type: imports\n",
      "  - pracma (>= 2.2.5) type: imports\n",
      "  - dbscan (>= 1.1-5) type: imports\n",
      "  - mmand (>= 1.5.4) type: imports\n",
      "  - tibble (>= 2.1.3) type: imports\n",
      "  - grDevices, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - vroom, type: imports\n",
      "  - ComplexHeatmap, type: imports\n",
      "  - dittoSeq, type: imports\n",
      "  - spatstat.geom, type: imports\n",
      "  - alphahull, type: imports\n",
      "  - methods, type: imports\n",
      "  - spatstat.explore, type: imports\n",
      "  - raster, type: imports\n",
      "  - sp, type: imports\n",
      "  - elsa, type: imports\n",
      "  - Rtsne, type: imports\n",
      "  - umap, type: imports\n",
      "  - rlang, type: imports\n",
      "  - graphics, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/SPIAT.html\n",
      "\n",
      "Name: CTSV\n",
      "Description:\n",
      "The R package CTSV implements the CTSV approach developed by Jinge Yu and Xiangyu Luo that detects cell-type-specific spatially variable genes accounting for excess zeros. CTSV directly models sparse raw count data through a zero-inflated negative binomial regression model, incorporates cell-type proportions, and performs hypothesis testing based on R package pscl. The package outputs p-values and q-values for genes in each cell type, and CTSV is scalable to datasets with tens of thousands of genes measured on hundreds of spots. CTSV can be installed in Windows, Linux, and Mac OS.\n",
      "Version: 1.0.0\n",
      "Publication date: None\n",
      "Mantainer: Jinge Yu Developer <yjgruc@ruc.edu.cn>\n",
      "Authors:\n",
      "Author: Jinge Yu Developer [aut, cre], Xiangyu Luo Developer [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.2) type: depends\n",
      "  - stats, type: imports\n",
      "  - pscl, type: imports\n",
      "  - qvalue, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - methods, type: imports\n",
      "  - knitr, type: imports\n",
      "  - SpatialExperiment, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/CTSV.html\n",
      "\n",
      "Name: demuxmix\n",
      "Description:\n",
      "A package for demultiplexing single-cell sequencing experiments of pooled cells labeled with barcode oligonucleotides. The package implements methods to fit regression mixture models for a probabilistic classification of cells, including multiplet detection. Demultiplexing error rates can be estimated, and methods for quality control are provided.\n",
      "Version: 1.0.0\n",
      "Publication date: None\n",
      "Mantainer: Hans-Ulrich Klein <hansulrich.klein@gmail.com>\n",
      "Authors:\n",
      "Author: Hans-Ulrich Klein [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.0.0) type: depends\n",
      "  - stats, type: imports\n",
      "  - MASS, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - gridExtra, type: imports\n",
      "  - methods, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/demuxmix.html\n",
      "\n",
      "Name: phemd\n",
      "Description:\n",
      "Package for comparing and generating a low-dimensional embedding of multiple single-cell samples.\n",
      "Version: 1.14.0\n",
      "Publication date: None\n",
      "Mantainer: William S Chen <wil.yum.chen@gmail.com>\n",
      "Authors:\n",
      "Author: William S Chen\n",
      "Dependencies:\n",
      "  - R (>= 3.5) type: depends\n",
      "  - monocle, type: depends\n",
      "  - Seurat, type: depends\n",
      "  - SingleCellExperiment, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - igraph, type: imports\n",
      "  - transport, type: imports\n",
      "  - pracma, type: imports\n",
      "  - cluster, type: imports\n",
      "  - Rtsne, type: imports\n",
      "  - destiny, type: imports\n",
      "  - RANN, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - maptree, type: imports\n",
      "  - pheatmap, type: imports\n",
      "  - scatterplot3d, type: imports\n",
      "  - VGAM, type: imports\n",
      "  - methods, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - cowplot, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - phateR, type: imports\n",
      "  - reticulate, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/phemd.html\n",
      "\n",
      "Name: MsExperiment\n",
      "Description:\n",
      "Infrastructure to store and manage all aspects related to a complete proteomics or metabolomics mass spectrometry (MS) experiment. The MsExperiment package provides light-weight and flexible containers for MS experiments building on the new MS infrastructure provided by the Spectra, QFeatures and related packages. Along with raw data representations, links to original data files and sample annotations, additional metadata or annotations can also be stored within the MsExperiment container. To guarantee maximum flexibility only minimal constraints are put on the type and content of the data within the containers.\n",
      "Version: 1.0.0\n",
      "Publication date: None\n",
      "Mantainer: Laurent Gatto <laurent.gatto@uclouvain.be>\n",
      "Authors:\n",
      "Author: Laurent Gatto [aut, cre] , Johannes Rainer [aut] , Sebastian Gibb [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.2) type: depends\n",
      "  - ProtGenerics, type: depends\n",
      "  - methods, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - Spectra, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - QFeatures, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/MsExperiment.html\n",
      "\n",
      "Name: MSstatsShiny\n",
      "Description:\n",
      "MSstatsShiny is an R-Shiny graphical user interface (GUI) integrated with the R packages MSstats, MSstatsTMT, and MSstatsPTM. It provides a point and click end-to-end analysis pipeline applicable to a wide variety of experimental designs. These include data-dependedent acquisitions (DDA) which are label-free or tandem mass tag (TMT)-based, as well as DIA, SRM, and PRM acquisitions and those targeting post-translational modifications (PTMs). The application automatically saves users selections and builds an R script that recreates their analysis, supporting reproducible data analysis.\n",
      "Version: 1.0.7\n",
      "Publication date: None\n",
      "Mantainer: Devon Kohler <kohler.d@northeastern.edu>\n",
      "Authors:\n",
      "Author: Devon Kohler [aut, cre], Maanasa Kaza [aut], Cristina Pasi [aut], Ting Huang [aut], Mateusz Staniak [aut], Dhaval Mohandas [aut], Eduard Sabido [aut], Meena Choi [aut], Olga Vitek [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.2) type: depends\n",
      "  - shiny, type: imports\n",
      "  - shinyBS, type: imports\n",
      "  - shinyjs, type: imports\n",
      "  - shinybusy, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - data.table, type: imports\n",
      "  - Hmisc, type: imports\n",
      "  - MSstats, type: imports\n",
      "  - MSstatsTMT, type: imports\n",
      "  - MSstatsPTM, type: imports\n",
      "  - MSstatsConvert, type: imports\n",
      "  - gplots, type: imports\n",
      "  - marray, type: imports\n",
      "  - DT, type: imports\n",
      "  - ggrepel, type: imports\n",
      "  - uuid, type: imports\n",
      "  - utils, type: imports\n",
      "  - stats, type: imports\n",
      "  - htmltools, type: imports\n",
      "  - methods, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - graphics, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/MSstatsShiny.html\n",
      "\n",
      "Name: SpliceWiz\n",
      "Description:\n",
      "Reads and fragments aligned to splice junctions can be used to quantify alternative splicing events (ASE). However, overlapping ASEs can confound their quantification. SpliceWiz quantifies ASEs, calculating percent-spliced-in (PSI) using junction reads, and intron retention using IRFinder-based quantitation. Novel filters identify ASEs that are relatively less confounded by overlapping events, whereby PSIs can be calculated with higher confidence. SpliceWiz is ultra-fast, using multi-threaded processing of BAM files. It can be run using a graphical user or command line interfaces. GUI-based interactive visualization of differential ASEs, including novel group-based RNA-seq coverage visualization, simplifies short-read RNA-seq analysis in R.\n",
      "Version: 1.0.2\n",
      "Publication date: None\n",
      "Mantainer: Alex Chit Hei Wong <a.wong@centenary.org.au>\n",
      "Authors:\n",
      "Author: Alex Chit Hei Wong [aut, cre, cph], Ulf Schmitz [ctb], William Ritchie [cph]\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - NxtIRFdata, type: depends\n",
      "  - ompBAM, type: imports\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - tools, type: imports\n",
      "  - parallel, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - Rcpp (>= 1.0.5) type: imports\n",
      "  - data.table, type: imports\n",
      "  - fst, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - AnnotationHub, type: imports\n",
      "  - BiocFileCache, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - BSgenome, type: imports\n",
      "  - DelayedArray, type: imports\n",
      "  - DelayedMatrixStats, type: imports\n",
      "  - genefilter, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - HDF5Array, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - progress, type: imports\n",
      "  - plotly, type: imports\n",
      "  - R.utils, type: imports\n",
      "  - rhdf5, type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - shiny, type: imports\n",
      "  - shinyFiles, type: imports\n",
      "  - shinyWidgets, type: imports\n",
      "  - shinydashboard, type: imports\n",
      "  - rhandsontable, type: imports\n",
      "  - DT, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - heatmaply, type: imports\n",
      "  - pheatmap, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - XML, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/SpliceWiz.html\n",
      "\n",
      "Name: katdetectr\n",
      "Description:\n",
      "Kataegis refers to the occurrence of regional hypermutation and is a phenomenon observed in a wide range of malignancies. Using changepoint detection katdetectr aims to identify putative kataegis foci from common data-formats housing genomic variants. Katdetectr has shown to be a robust package for the detection, characterization and visualization of kataegis.\n",
      "Version: 1.0.0\n",
      "Publication date: None\n",
      "Mantainer: Daan Hazelaar <daanhazelaar@gmail.com>\n",
      "Authors:\n",
      "Author: Daan Hazelaar [aut, cre] , Job van Riet [aut] , Harmen van de Werken [ths]\n",
      "Dependencies:\n",
      "  - R (>= 4.2) type: depends\n",
      "  - BiocParallel, type: imports\n",
      "  - changepoint (>= 2.2.3) type: imports\n",
      "  - changepoint.np (>= 1.0.3) type: imports\n",
      "  - checkmate (>= 2.0.0) type: imports\n",
      "  - dplyr (>= 1.0.8) type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - maftools, type: imports\n",
      "  - methods (>= 4.1.3) type: imports\n",
      "  - rlang (>= 1.0.2) type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - tibble (>= 3.1.6) type: imports\n",
      "  - VariantAnnotation, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - Rdpack (>= 2.3.1) type: imports\n",
      "  - ggplot2 (>= 3.3.5) type: imports\n",
      "  - tidyr (>= 1.2.0) type: imports\n",
      "  - BSgenome, type: imports\n",
      "  - ggtext (>= 0.1.1) type: imports\n",
      "  - BSgenome.Hsapiens.UCSC.hg19, type: imports\n",
      "  - BSgenome.Hsapiens.UCSC.hg38, type: imports\n",
      "  - plyranges, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3 + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/katdetectr.html\n",
      "\n",
      "Name: MethCP\n",
      "Description:\n",
      "MethCP is a differentially methylated region (DMR) detecting method for whole-genome bisulfite sequencing (WGBS) data, which is applicable for a wide range of experimental designs beyond the two-group comparisons, such as time-course data. MethCP identifies DMRs based on change point detection, which naturally segments the genome and provides region-level differential analysis.\n",
      "Version: 1.11.0\n",
      "Publication date: None\n",
      "Mantainer: Boying Gong <jorothy_gong@berkeley.edu>\n",
      "Authors:\n",
      "Author: Boying Gong [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 3.6.0) type: depends\n",
      "  - methods, type: imports\n",
      "  - utils, type: imports\n",
      "  - stats, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - bsseq, type: imports\n",
      "  - DSS, type: imports\n",
      "  - methylKit, type: imports\n",
      "  - DNAcopy, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/MethCP.html\n",
      "\n",
      "Name: ATACseqTFEA\n",
      "Description:\n",
      "Assay for Transpose-Accessible Chromatin using sequencing (ATAC-seq) is a technique to assess genome-wide chromatin accessibility by probing open chromatin with hyperactive mutant Tn5 Transposase that inserts sequencing adapters into open regions of the genome. ATACseqTFEA is an improvement of the current computational method that detects differential activity of transcription factors (TFs). ATACseqTFEA not only uses the difference of open region information, but also (or emphasizes) the difference of TFs footprints (cutting sites or insertion sites). ATACseqTFEA provides an easy, rigorous way to broadly assess TF activity changes between two conditions.\n",
      "Version: 1.0.1\n",
      "Publication date: None\n",
      "Mantainer: Jianhong Ou <jianhong.ou@duke.edu>\n",
      "Authors:\n",
      "Author: Jianhong Ou [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.2) type: depends\n",
      "  - BiocGenerics, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - GenomicAlignments, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - motifmatchr, type: imports\n",
      "  - TFBSTools, type: imports\n",
      "  - stats, type: imports\n",
      "  - pracma, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - ggrepel, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - limma, type: imports\n",
      "  - methods, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ATACseqTFEA.html\n",
      "\n",
      "Name: cardelino\n",
      "Description:\n",
      "Methods to infer clonal tree configuration for a population of cells using single-cell RNA-seq data (scRNA-seq), and possibly other data modalities. Methods are also provided to assign cells to inferred clones and explore differences in gene expression between clones. These methods can flexibly integrate information from imperfect clonal trees inferred based on bulk exome-seq data, and sparse variant alleles expressed in scRNA-seq data. A flexible beta-binomial error model that accounts for stochastic dropout events as well as systematic allelic imbalance is used.\n",
      "Version: 1.0.0\n",
      "Publication date: None\n",
      "Mantainer: Jeffrey Pullin <jpullin@svi.edu.au>\n",
      "Authors:\n",
      "Author: Jeffrey Pullin [aut, cre], Yuanhua Huang [aut], Davis McCarthy [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.2) type: depends\n",
      "  - stats, type: depends\n",
      "  - combinat, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - ggtree, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - methods, type: imports\n",
      "  - pheatmap, type: imports\n",
      "  - snpStats, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - utils, type: imports\n",
      "  - VariantAnnotation, type: imports\n",
      "  - vcfR, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/cardelino.html\n",
      "\n",
      "Name: mslp\n",
      "Description:\n",
      "An integrated pipeline to predict the potential synthetic lethality partners (SLPs) of tumour mutations, based on gene expression, mutation profiling and cell line genetic screens data. It has builtd-in support for data from cBioPortal. The primary SLPs correlating with muations in WT and compensating for the loss of function of mutations are predicted by random forest based methods (GENIE3) and Rank Products, respectively. Genetic screens are employed to identfy consensus SLPs leads to reduced cell viability when perturbed.\n",
      "Version: 1.0.1\n",
      "Publication date: None\n",
      "Mantainer: Chunxuan Shao <chunxuan@outlook.com>\n",
      "Authors:\n",
      "Author: Chunxuan Shao [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.2.0) type: depends\n",
      "  - data.table (>= 1.13.0) type: depends\n",
      "  - doRNG, type: imports\n",
      "  - fmsb, type: imports\n",
      "  - foreach, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - org.Hs.eg.db, type: imports\n",
      "  - pROC, type: imports\n",
      "  - randomForest, type: imports\n",
      "  - RankProd, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/mslp.html\n",
      "\n",
      "Name: proteasy\n",
      "Description:\n",
      "Retrieval of experimentally derived protease- and cleavage data derived from the MEROPS database. Proteasy contains functions for mapping peptide termini to known sites where a protease cleaves. This package also makes it possible to quickly look up known substrates based on a list of (potential) proteases, or vice versa - look up proteases based on a list of substrates.\n",
      "Version: 1.0.0\n",
      "Publication date: None\n",
      "Mantainer: Martin RydÃ©n <martin.ryden@med.lu.se>\n",
      "Authors:\n",
      "Author: Martin RydÃ©n [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.2.0) type: depends\n",
      "  - data.table, type: imports\n",
      "  - stringr, type: imports\n",
      "  - ensembldb, type: imports\n",
      "  - AnnotationFilter, type: imports\n",
      "  - EnsDb.Hsapiens.v86, type: imports\n",
      "  - EnsDb.Mmusculus.v79, type: imports\n",
      "  - EnsDb.Rnorvegicus.v79, type: imports\n",
      "  - Rcpi, type: imports\n",
      "  - methods, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/proteasy.html\n",
      "\n",
      "Name: Voyager\n",
      "Description:\n",
      "SpatialFeatureExperiment (SFE) is a new S4 class for working with spatial single-cell genomics data. The voyager package implements basic exploratory spatial data analysis (ESDA) methods for SFE. This first version supports univariate global spatial ESDA methods such as Moran's I, permutation testing for Moran's I, and correlograms. The Voyager package also implements plotting functions to plot SFE data and ESDA results. Multivariate ESDA and univariate local metrics will be added in later versions.\n",
      "Version: 1.0.9\n",
      "Publication date: None\n",
      "Mantainer: Lambda Moses <dlu2@caltech.edu>\n",
      "Authors:\n",
      "Author: Lambda Moses [aut, cre] , Kayla Jackson [aut] , Lior Pachter [aut, rev]\n",
      "Dependencies:\n",
      "  - R (>= 4.2.0) type: depends\n",
      "  - BiocParallel, type: imports\n",
      "  - bluster, type: imports\n",
      "  - ggnewscale, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - methods, type: imports\n",
      "  - patchwork, type: imports\n",
      "  - rlang, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - scales, type: imports\n",
      "  - scico, type: imports\n",
      "  - sf, type: imports\n",
      "  - SingleCellExperiment, type: imports\n",
      "  - SpatialExperiment, type: imports\n",
      "  - SpatialFeatureExperiment, type: imports\n",
      "  - spdep, type: imports\n",
      "  - stats, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/Voyager.html\n",
      "\n",
      "Name: EasyCellType\n",
      "Description:\n",
      "We developed EasyCellType which can automatically examine the input marker lists obtained from existing software such as Seurat over the cell markerdatabases. Two quantification approaches to annotate cell types are provided: Gene set enrichment analysis (GSEA) and a modified versio of Fisher's exact test. The function presents annotation recommendations in graphical outcomes: bar plots for each cluster showing candidate cell types, as well as a dot plot summarizing the top 5 significant annotations for each cluster.\n",
      "Version: 1.0.0\n",
      "Publication date: None\n",
      "Mantainer: Ruoxing Li <ruoxingli@outlook.com>\n",
      "Authors:\n",
      "Author: Ruoxing Li [aut, cre, ctb], Ziyi Li [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 4.2.0) type: depends\n",
      "  - clusterProfiler, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - forcats, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - rlang, type: imports\n",
      "  - stats, type: imports\n",
      "  - org.Hs.eg.db, type: imports\n",
      "  - org.Mm.eg.db, type: imports\n",
      "  - AnnotationDbi, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/EasyCellType.html\n",
      "\n",
      "Name: factR\n",
      "Description:\n",
      "factR contain tools to process and interact with custom-assembled transcriptomes (GTF). At its core, factR constructs CDS information on custom transcripts and subsequently predicts its functional output. In addition, factR has tools capable of plotting transcripts, correcting chromosome and gene information and shortlisting new transcripts.\n",
      "Version: 1.0.0\n",
      "Publication date: None\n",
      "Mantainer: Fursham Hamid <fursham.h@gmail.com>\n",
      "Authors:\n",
      "Author: Fursham Hamid [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.2) type: depends\n",
      "  - BiocGenerics, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - dplyr (>= 1.0) type: imports\n",
      "  - GenomicFeatures, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - purrr (>= 0.3) type: imports\n",
      "  - rtracklayer, type: imports\n",
      "  - tidyr (>= 1.1) type: imports\n",
      "  - methods (>= 4.0) type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - data.table (>= 1.14) type: imports\n",
      "  - rlang (>= 1.0) type: imports\n",
      "  - tibble (>= 3.1) type: imports\n",
      "  - wiggleplotr, type: imports\n",
      "  - RCurl (>= 1.98) type: imports\n",
      "  - XML (>= 3.99) type: imports\n",
      "  - drawProteins, type: imports\n",
      "  - ggplot2 (>= 3.3) type: imports\n",
      "  - stringr (>= 1.4) type: imports\n",
      "  - pbapply (>= 1.5) type: imports\n",
      "  - stats (>= 4.1) type: imports\n",
      "  - utils (>= 4.1) type: imports\n",
      "  - graphics (>= 4.1) type: imports\n",
      "  - crayon, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/factR.html\n",
      "\n",
      "Name: gemma.R\n",
      "Description:\n",
      "Low- and high-level wrappers for Gemma's RESTful API. They enable access to curated expression and differential expression data from over 10,000 published studies. Gemma is a web site, database and a set of tools for the meta-analysis, re-use and sharing of genomics data, currently primarily targeted at the analysis of gene expression profiles.\n",
      "Version: 1.0.1\n",
      "Publication date: None\n",
      "Mantainer: Ogan Mancarci <ogan.mancarci@gmail.com>\n",
      "Authors:\n",
      "Author: Javier Castillo-Arnemann [aut] , Jordan Sicherman [aut] , Ogan Mancarci [cre, aut] , Guillaume Poirier-Morency [aut]\n",
      "Dependencies:\n",
      "  - magrittr, type: imports\n",
      "  - glue, type: imports\n",
      "  - memoise, type: imports\n",
      "  - jsonlite, type: imports\n",
      "  - data.table, type: imports\n",
      "  - rlang, type: imports\n",
      "  - lubridate, type: imports\n",
      "  - utils, type: imports\n",
      "  - stringr, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - tibble, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - httr, type: imports\n",
      "  - rappdirs, type: imports\n",
      "  - bit64, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Apache License (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/gemma.R.html\n",
      "\n",
      "Name: GenomAutomorphism\n",
      "Description:\n",
      "This is a R package to compute the automorphisms between pairwise aligned DNA sequences represented as elements from a Genomic Abelian group. In a general scenario, from genomic regions till the whole genomes from a given population (from any species or close related species) can be algebraically represented as a direct sum of cyclic groups or more specifically Abelian p-groups. Basically, we propose the representation of multiple sequence alignments of length N bp as element of a finite Abelian group created by the direct sum of homocyclic Abelian group of prime-power order.\n",
      "Version: 1.0.2\n",
      "Publication date: None\n",
      "Mantainer: Robersy Sanchez <genomicmath@gmail.com>\n",
      "Authors:\n",
      "Author: Robersy Sanchez [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.2) type: depends\n",
      "  - Biostrings, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - data.table, type: imports\n",
      "  - parallel, type: imports\n",
      "  - doParallel, type: imports\n",
      "  - foreach, type: imports\n",
      "  - methods, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - stats, type: imports\n",
      "  - numbers, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/GenomAutomorphism.html\n",
      "\n",
      "Name: ggtreeDendro\n",
      "Description:\n",
      "Offers a set of 'autoplot' methods to visualize tree-like structures (e.g., hierarchical clustering and classification/regression trees) using 'ggtree'. You can adjust graphical parameters using grammar of graphic syntax and integrate external data to the tree.\n",
      "Version: 1.0.0\n",
      "Publication date: None\n",
      "Mantainer: Guangchuang Yu <guangchuangyu@gmail.com>\n",
      "Authors:\n",
      "Author: Guangchuang Yu [aut, cre, cph] , Shuangbin Xu [ctb] , Chuanjie Zhang [ctb]\n",
      "Dependencies:\n",
      "  - ggtree, type: depends\n",
      "  - ggplot2, type: imports\n",
      "  - stats, type: imports\n",
      "  - tidytree, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ggtreeDendro.html\n",
      "\n",
      "Name: HiCDOC\n",
      "Description:\n",
      "HiCDOC normalizes intrachromosomal Hi-C matrices, uses unsupervised learning to predict A/B compartments from multiple replicates, and detects significant compartment changes between experiment conditions. It provides a collection of functions assembled into a pipeline to filter and normalize the data, predict the compartments and visualize the results. It accepts several type of data: tabular `.tsv` files, Cooler `.cool` or `.mcool` files, Juicer `.hic` files or HiC-Pro `.matrix` and `.bed` files.\n",
      "Version: 1.0.0\n",
      "Publication date: None\n",
      "Mantainer: MaignÃ© Ãlise <elise.maigne@inrae.fr>\n",
      "Authors:\n",
      "Author: Kurylo Cyril [aut], Zytnicki Matthias [aut], Foissac Sylvain [aut], MaignÃ© Ãlise [aut, cre]\n",
      "Dependencies:\n",
      "  - InteractionSet, type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - R (>= 4.1.0) type: depends\n",
      "  - methods, type: imports\n",
      "  - zlibbioc, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - Rcpp (>= 0.12.8) type: imports\n",
      "  - stats, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - gtools, type: imports\n",
      "  - pbapply, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - rhdf5, type: imports\n",
      "  - grid, type: imports\n",
      "  - ggpubr, type: imports\n",
      "  - gridExtra, type: imports\n",
      "  - ggExtra, type: imports\n",
      "  - data.table, type: imports\n",
      "  - multiHiCcompare, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "LGPL (>= 3)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/HiCDOC.html\n",
      "\n",
      "Name: scDDboost\n",
      "Description:\n",
      "scDDboost is an R package to analyze changes in the distribution of single-cell expression data between two experimental conditions. Compared to other methods that assess differential expression, scDDboost benefits uniquely from information conveyed by the clustering of cells into cellular subtypes. Through a novel empirical Bayesian formulation it calculates gene-specific posterior probabilities that the marginal expression distribution is the same (or different) between the two conditions. The implementation in scDDboost treats gene-level expression data within each condition as a mixture of negative binomial distributions.\n",
      "Version: 1.0.0\n",
      "Publication date: None\n",
      "Mantainer: Xiuyu Ma <watsonforfun@gmail.com>\n",
      "Authors:\n",
      "Author: Xiuyu Ma [cre, aut], Michael A. Newton [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 4.2) type: depends\n",
      "  - ggplot2, type: depends\n",
      "  - Rcpp (>= 0.12.11) type: imports\n",
      "  - RcppEigen (>= 0.3.2.9.0) type: imports\n",
      "  - EBSeq, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - mclust, type: imports\n",
      "  - SingleCellExperiment, type: imports\n",
      "  - cluster, type: imports\n",
      "  - Oscope, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - stats, type: imports\n",
      "  - methods, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL (>= 2)\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/scDDboost.html\n",
      "\n",
      "Name: ScreenR\n",
      "Description:\n",
      "ScreenR is a package suitable to perform hit identification in loss of function High Throughput Biological Screenings performed using barcoded shRNA-based libraries. ScreenR combines the computing power of software such as edgeR with the simplicity of use of the Tidyverse metapackage. ScreenR executes a pipeline able to find candidate hits from barcode counts, and integrates a wide range of visualization modes for each step of the analysis.\n",
      "Version: 1.0.0\n",
      "Publication date: None\n",
      "Mantainer: Emanuel Michele Soda <emanuelsoda@gmail.com>\n",
      "Authors:\n",
      "Author: Emanuel Michele Soda [aut, cre] (0000-0002-2301-6465), Elena Ceccacci [aut] (0000-0002-2285-8994), Saverio Minucci [fnd, ths] (0000-0001-5678-536X)\n",
      "Dependencies:\n",
      "  - R (>= 4.2) type: depends\n",
      "  - methods (>= 4.0) type: imports\n",
      "  - rlang (>= 0.4) type: imports\n",
      "  - stringr (>= 1.4) type: imports\n",
      "  - limma, type: imports\n",
      "  - patchwork (>= 1.1) type: imports\n",
      "  - tibble (>= 3.1.6) type: imports\n",
      "  - scales (>= 1.1.1) type: imports\n",
      "  - ggvenn (>= 0.1.9) type: imports\n",
      "  - purrr (>= 0.3.4) type: imports\n",
      "  - ggplot2 (>= 3.3) type: imports\n",
      "  - stats, type: imports\n",
      "  - tidyr (>= 1.2) type: imports\n",
      "  - magrittr (>= 1.0) type: imports\n",
      "  - dplyr (>= 1.0) type: imports\n",
      "  - edgeR, type: imports\n",
      "  - tidyselect (>= 1.1.2) type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ScreenR.html\n",
      "\n",
      "Name: simpleSeg\n",
      "Description:\n",
      "Image segmentation is the process of identifying the borders of individual objects (in this case cells) within an image. This allows for the features of cells such as marker expression and morphology to be extracted, stored and analysed. simpleSeg provides functionality for user friendly, watershed based segmentation on multiplexed cellular images in R based on the intensity of user specified protein marker channels. simpleSeg can also be used for the normalization of single cell data obtained from multiple images.\n",
      "Version: 1.0.2\n",
      "Publication date: None\n",
      "Mantainer: Ellis Patrick <ellis.patrick@sydney.edu.au>\n",
      "Authors:\n",
      "Author: Nicolas Canete [aut], Alexander Nicholls [aut], Ellis Patrick [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - BiocParallel, type: imports\n",
      "  - EBImage, type: imports\n",
      "  - terra, type: imports\n",
      "  - stats, type: imports\n",
      "  - spatstat.geom, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - methods, type: imports\n",
      "  - cytomapper, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/simpleSeg.html\n",
      "\n",
      "Name: BASiCStan\n",
      "Description:\n",
      "Provides an interface to infer the parameters of BASiCS using the variational inference (ADVI), Markov chain Monte Carlo (NUTS), and maximum a posteriori (BFGS) inference engines in the Stan programming language. BASiCS is a Bayesian hierarchical model that uses an adaptive Metropolis within Gibbs sampling scheme. Alternative inference methods provided by Stan may be preferable in some situations, for example for particularly large data or posterior distributions with difficult geometries.\n",
      "Version: 1.0.0\n",
      "Publication date: None\n",
      "Mantainer: Alan O'Callaghan <alan.ocallaghan@outlook.com>\n",
      "Authors:\n",
      "Author: Alan O'Callaghan [aut, cre], Catalina Vallejos [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.2) type: depends\n",
      "  - BASiCS, type: depends\n",
      "  - rstan (>= 2.18.1) type: depends\n",
      "  - methods, type: imports\n",
      "  - glmGamPoi, type: imports\n",
      "  - scran, type: imports\n",
      "  - scuttle, type: imports\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - SingleCellExperiment, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - Rcpp (>= 0.12.0) type: imports\n",
      "  - RcppParallel (>= 5.0.1) type: imports\n",
      "  - rstantools (>= 2.1.1) type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/BASiCStan.html\n",
      "\n",
      "Name: FuseSOM\n",
      "Description:\n",
      "A correlation-based multiview self-organizing map for the characterization of cell types in highly multiplexed in situ imaging cytometry assays (`FuseSOM`) is a tool for unsupervised clustering. `FuseSOM` is robust and achieves high accuracy by combining a `Self Organizing Map` architecture and a `Multiview` integration of correlation based metrics. This allows FuseSOM to cluster highly multiplexed in situ imaging cytometry assays.\n",
      "Version: 1.0.1\n",
      "Publication date: None\n",
      "Mantainer: Elijah Willie <ewil3501@uni.sydney.edu.au>\n",
      "Authors:\n",
      "Author: Elijah Willie [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.2.0) type: depends\n",
      "  - psych, type: imports\n",
      "  - FCPS, type: imports\n",
      "  - analogue, type: imports\n",
      "  - coop, type: imports\n",
      "  - pheatmap, type: imports\n",
      "  - ggplotify, type: imports\n",
      "  - fastcluster, type: imports\n",
      "  - fpc, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - stringr, type: imports\n",
      "  - ggpubr, type: imports\n",
      "  - proxy, type: imports\n",
      "  - cluster, type: imports\n",
      "  - diptest, type: imports\n",
      "  - methods, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - stats, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/FuseSOM.html\n",
      "\n",
      "Name: goSorensen\n",
      "Description:\n",
      "This package implements inferential methods to compare gene lists (in this first release, to prove equivalence) in terms of their biological meaning as expressed in the GO. The compared gene lists are characterized by cross-tabulation frequency tables of enriched GO items. Dissimilarity between gene lists is evaluated using the Sorensen-Dice index. The fundamental guiding principle is that two gene lists are taken as similar if they share a great proportion of common enriched GO items.\n",
      "Version: 1.0.0\n",
      "Publication date: None\n",
      "Mantainer: Pablo Flores <p_flores@espoch.edu.ec>\n",
      "Authors:\n",
      "Author: Pablo Flores [aut, cre] , Jordi Ocana [aut, ctb] (0000-0002-4736-699), Alexandre Sanchez-Pla [ctb] , Miquel Salicru [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 4.2.0) type: depends\n",
      "  - GO.db, type: imports\n",
      "  - org.Hs.eg.db, type: imports\n",
      "  - goProfiles, type: imports\n",
      "  - stats, type: imports\n",
      "  - clusterProfiler, type: imports\n",
      "  - parallel, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/goSorensen.html\n",
      "\n",
      "Name: HiContacts\n",
      "Description:\n",
      "HiContacts: R interface to (m)cool files and other Hi-C processed file formats. HiContacts provides a collection of tools to analyse and visualize Hi-C datasets. It can import data from pairs or (m)cool files.\n",
      "Version: 1.0.0\n",
      "Publication date: None\n",
      "Mantainer: Jacques Serizay <jacquesserizay@gmail.com>\n",
      "Authors:\n",
      "Author: Jacques Serizay [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.2) type: depends\n",
      "  - HiContactsData, type: imports\n",
      "  - InteractionSet, type: imports\n",
      "  - GenomicInteractions, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - methods, type: imports\n",
      "  - rhdf5, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - vroom, type: imports\n",
      "  - tibble, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - glue, type: imports\n",
      "  - stringr, type: imports\n",
      "  - reticulate, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - ggrastr, type: imports\n",
      "  - scales, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/HiContacts.html\n",
      "\n",
      "Name: PanViz\n",
      "Description:\n",
      "This pacakge integrates data from the Kyoto Encyclopedia of Genes and Genomes (KEGG) with summary-level genome-wide association (GWAS) data, such as that provided by the GWAS Catalog or GWAS Central databases, or a user's own study or dataset, in order to produce biological networks, termed IMONs (Integrated Multi-Omic Networks). IMONs can be used to analyse trait-specific polymorphic data within the context of biochemical and metabolic reaction networks, providing greater biological interpretability for GWAS data.\n",
      "Version: 1.0.0\n",
      "Publication date: None\n",
      "Mantainer: Luca Anholt <la1317@ic.ac.uk>\n",
      "Authors:\n",
      "Author: Luca Anholt [cre, aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.2.0) type: depends\n",
      "  - tidyr, type: imports\n",
      "  - stringr, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - tibble, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - futile.logger, type: imports\n",
      "  - utils, type: imports\n",
      "  - easycsv, type: imports\n",
      "  - rentrez, type: imports\n",
      "  - igraph, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - data.table, type: imports\n",
      "  - colorspace, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - rlang, type: imports\n",
      "  - methods, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/PanViz.html\n",
      "\n",
      "Name: RedisParam\n",
      "Description:\n",
      "This package provides a Redis-based back-end for BiocParallel, enabling an alternative mechanism for distributed computation. The The 'manager' distributes tasks to a 'worker' pool through a central Redis server, rather than directly to workers as with other BiocParallel implementations. This means that the worker pool can change dynamically during job evaluation. All features of BiocParallel are supported, including reproducible random number streams, logging to the manager, and alternative 'load balancing' task distributions.\n",
      "Version: 1.0.0\n",
      "Publication date: None\n",
      "Mantainer: Martin Morgan <mtmorgan.bioc@gmail.com>\n",
      "Authors:\n",
      "Author: Martin Morgan [aut, cre] , Jiefei Wang [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.2.0) type: depends\n",
      "  - BiocParallel, type: depends\n",
      "  - methods, type: imports\n",
      "  - redux, type: imports\n",
      "  - withr, type: imports\n",
      "  - futile.logger, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/RedisParam.html\n",
      "\n",
      "Name: spaSim\n",
      "Description:\n",
      "A suite of functions for simulating spatial patterns of cells in tissue images. Output images are multitype point data in SingleCellExperiment format. Each point represents a cell, with its 2D locations and cell type. Potential cell patterns include background cells, tumour/immune cell clusters, immune rings, and blood/lymphatic vessels.\n",
      "Version: 1.0.2\n",
      "Publication date: None\n",
      "Mantainer: Yuzhou Feng <yuzhou.feng@petermac.org>\n",
      "Authors:\n",
      "Author: Yuzhou Feng [aut, cre] , Anna Trigos [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.2.0) type: depends\n",
      "  - ggplot2, type: imports\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - spatstat.geom, type: imports\n",
      "  - spatstat.random, type: imports\n",
      "  - SpatialExperiment, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - RANN, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/spaSim.html\n",
      "\n",
      "Name: SUITOR\n",
      "Description:\n",
      "An unsupervised cross-validation method to select the optimal number of mutational signatures. A data set of mutational counts is split into training and validation data.Signatures are estimated in the training data and then used to predict the mutations in the validation data.\n",
      "Version: 1.0.0\n",
      "Publication date: None\n",
      "Mantainer: Bill Wheeler <wheelerb@imsweb.com>\n",
      "Authors:\n",
      "Author: DongHyuk Lee [aut], Bin Zhu [aut], Bill Wheeler [cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.2.0) type: depends\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - graphics, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/SUITOR.html\n",
      "\n",
      "Name: consICA\n",
      "Description:\n",
      "consICA implements a data-driven deconvolution method â consensus independent component analysis (ICA) to decompose heterogeneous omics data and extract features suitable for patient diagnostics and prognostics. The method separates biologically relevant transcriptional signals from technical effects and provides information about the cellular composition and biological processes. The implementation of parallel computing in the package ensures efficient analysis of modern multicore systems.\n",
      "Version: 1.0.0\n",
      "Publication date: None\n",
      "Mantainer: Petr V. Nazarov <petr.nazarov@lih.lu>\n",
      "Authors:\n",
      "Author: Petr V. Nazarov [aut, cre] , Tony Kaoma [aut] , Maryna Chepeleva [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.2.0) type: depends\n",
      "  - fastICA (>= 1.2.1) type: imports\n",
      "  - sm, type: imports\n",
      "  - org.Hs.eg.db, type: imports\n",
      "  - GO.db, type: imports\n",
      "  - stats, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - graph, type: imports\n",
      "  - methods, type: imports\n",
      "  - pheatmap, type: imports\n",
      "  - survival, type: imports\n",
      "  - topGO, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/consICA.html\n",
      "\n",
      "Name: DNAfusion\n",
      "Description:\n",
      "Paired-end sequencing of cfDNA generated BAM files can be used as input to discover EML4-ALK variants. This package was developed using position deduplicated BAM files generated with the AVENIO Oncology Analysis Software. These files are made using the AVENIO ctDNA surveillance kit and Illumina Nextseq 500 sequencing. This is a targeted hybridization NGS approach and includes ALK-specific but not EML4-specific probes.\n",
      "Version: 1.0.0\n",
      "Publication date: None\n",
      "Mantainer: Christoffer Trier Maansson <ctm@clin.au.dk>\n",
      "Authors:\n",
      "Author: Christoffer Trier Maansson [aut, cre] , Emma Roger Andersen [ctb, rev], Maiken Parm Ulhoi [dtc], Peter Meldgaard [dtc], Boe Sandahl Sorensen [rev, fnd]\n",
      "Dependencies:\n",
      "  - R (>= 4.2.0) type: depends\n",
      "  - bamsignals, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - GenomicAlignments, type: imports\n",
      "  - BiocBaseUtils, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/DNAfusion.html\n",
      "\n",
      "Name: epistasisGA\n",
      "Description:\n",
      "This package runs the GADGETS method to identify epistatic effects in nuclear family studies. It also provides functions for permutation-based inference and graphical visualization of the results.\n",
      "Version: 1.0.2\n",
      "Publication date: None\n",
      "Mantainer: Michael Nodzenski <michael.nodzenski@gmail.com>\n",
      "Authors:\n",
      "Author: Michael Nodzenski [aut, cre], Juno Krahn [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 4.0) type: depends\n",
      "  - BiocParallel, type: imports\n",
      "  - data.table, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - stats, type: imports\n",
      "  - survival, type: imports\n",
      "  - igraph, type: imports\n",
      "  - batchtools, type: imports\n",
      "  - qgraph, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - parallel, type: imports\n",
      "  - Rcpp (>= 0.11.0) type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - grid, type: imports\n",
      "  - graphics, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/epistasisGA.html\n",
      "\n",
      "Name: oncoscanR\n",
      "Description:\n",
      "The software uses the copy number segments from a text file and identifies all chromosome arms that are globally altered and computes various genome-wide scores. The following HRD scores (characteristic of BRCA-mutated cancers) are included: LST, HR-LOH, nLST and gLOH. the package is tailored for the ThermoFisher Oncoscan assay analyzed with their Chromosome Alteration Suite (ChAS) but can be adapted to any input.\n",
      "Version: 1.0.0\n",
      "Publication date: None\n",
      "Mantainer: Yann Christinat <yann.christinat@hcuge.ch>\n",
      "Authors:\n",
      "Author: Yann Christinat [aut, cre], Geneva University Hospitals [aut, cph]\n",
      "Dependencies:\n",
      "  - R (>= 4.2) type: depends\n",
      "  - IRanges, type: depends\n",
      "  - GenomicRanges, type: depends\n",
      "  - magrittr, type: depends\n",
      "  - readr, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - methods, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/oncoscanR.html\n",
      "\n",
      "Name: phenomis\n",
      "Description:\n",
      "The 'phenomis' package provides methods to perform post-processing (i.e. quality control and normalization) as well as univariate statistical analysis of single and multi-omics data sets. These methods include quality control metrics, signal drift and batch effect correction, intensity transformation, univariate hypothesis testing, but also clustering (as well as annotation of metabolomics data). The data are handled in the standard Bioconductor formats (i.e. SummarizedExperiment and MultiAssayExperiment for single and multi-omics datasets, respectively; the alternative ExpressionSet and MultiDataSet formats are also supported for convenience). As a result, all methods can be readily chained as workflows. The pipeline can be further enriched by multivariate analysis and feature selection, by using the 'ropls' and 'biosigner' packages, which support the same formats. Data can be conveniently imported from and exported to text files. Although the methods were initially targeted to metabolomics data, most of the methods can be applied to other types of omics data (e.g., transcriptomics, proteomics).\n",
      "Version: 1.0.2\n",
      "Publication date: None\n",
      "Mantainer: Etienne A. Thevenot <etienne.thevenot@cea.fr>\n",
      "Authors:\n",
      "Author: Etienne A. Thevenot [aut, cre] , Natacha Lenuzza [ctb], Marie Tremblay-Franco [ctb], Alyssa Imbert [ctb], Pierrick Roger [ctb], Eric Venot [ctb], Sylvain Dechaumet [ctb]\n",
      "Dependencies:\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - Biobase, type: imports\n",
      "  - biodb, type: imports\n",
      "  - biodbChebi, type: imports\n",
      "  - data.table, type: imports\n",
      "  - futile.logger, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - ggrepel, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - grid, type: imports\n",
      "  - htmlwidgets, type: imports\n",
      "  - igraph, type: imports\n",
      "  - limma, type: imports\n",
      "  - methods, type: imports\n",
      "  - MultiAssayExperiment, type: imports\n",
      "  - MultiDataSet, type: imports\n",
      "  - PMCMRplus, type: imports\n",
      "  - plotly, type: imports\n",
      "  - ranger, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - ropls, type: imports\n",
      "  - stats, type: imports\n",
      "  - tibble, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - utils, type: imports\n",
      "  - VennDiagram, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "CeCILL\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/phenomis.html\n",
      "\n",
      "Name: RgnTX\n",
      "Description:\n",
      "RgnTX allows the integration of transcriptome annotations so as to model the complex alternative splicing patterns. It supports the testing of transcriptome elements without clear isoform association, which is often the real scenario due to technical limitations. It involves functions that do permutaion test for evaluating association between features and transcriptome regions.\n",
      "Version: 1.0.0\n",
      "Publication date: None\n",
      "Mantainer: Yue Wang <yue.wang19@student.xjtlu.edu.cn>\n",
      "Authors:\n",
      "Author: Yue Wang [aut, cre], Jia Meng [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.2.0) type: depends\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - GenomicFeatures, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - graphics, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - methods, type: imports\n",
      "  - regioneR, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - stats, type: imports\n",
      "  - TxDb.Hsapiens.UCSC.hg19.knownGene, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/RgnTX.html\n",
      "\n",
      "Name: scMET\n",
      "Description:\n",
      "High-throughput single-cell measurements of DNA methylomes can quantify methylation heterogeneity and uncover its role in gene regulation. However, technical limitations and sparse coverage can preclude this task. scMET is a hierarchical Bayesian model which overcomes sparsity, sharing information across cells and genomic features to robustly quantify genuine biological heterogeneity. scMET can identify highly variable features that drive epigenetic heterogeneity, and perform differential methylation and variability analyses. We illustrate how scMET facilitates the characterization of epigenetically distinct cell populations and how it enables the formulation of novel hypotheses on the epigenetic regulation of gene expression.\n",
      "Version: 1.0.0\n",
      "Publication date: None\n",
      "Mantainer: Andreas C. Kapourani <kapouranis.andreas@gmail.com>\n",
      "Authors:\n",
      "Author: Andreas C. Kapourani [aut, cre] , John Riddell [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 4.2.0) type: depends\n",
      "  - methods, type: imports\n",
      "  - Rcpp (>= 1.0.0) type: imports\n",
      "  - RcppParallel (>= 5.0.1) type: imports\n",
      "  - rstan (>= 2.21.3) type: imports\n",
      "  - rstantools (>= 2.1.0) type: imports\n",
      "  - VGAM, type: imports\n",
      "  - data.table, type: imports\n",
      "  - MASS, type: imports\n",
      "  - logitnorm, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - assertthat, type: imports\n",
      "  - viridis, type: imports\n",
      "  - coda, type: imports\n",
      "  - BiocStyle, type: imports\n",
      "  - cowplot, type: imports\n",
      "  - stats, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - SingleCellExperiment, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/scMET.html\n",
      "\n",
      "Name: SimBu\n",
      "Description:\n",
      "SimBu can be used to simulate bulk RNA-seq datasets with known cell type fractions. You can either use your own single-cell study for the simulation or the sfaira database. Different pre-defined simulation scenarios exist, as are options to run custom simulations. Additionally, expression values can be adapted by adding an mRNA bias, which produces more biologically relevant simulations.\n",
      "Version: 1.0.0\n",
      "Publication date: None\n",
      "Mantainer: Alexander Dietrich <alex.dietrich@tum.de>\n",
      "Authors:\n",
      "Author: Alexander Dietrich [aut, cre]\n",
      "Dependencies:\n",
      "  - basilisk, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - data.table, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - tools, type: imports\n",
      "  - Matrix (>= 1.3.3) type: imports\n",
      "  - methods, type: imports\n",
      "  - phyloseq, type: imports\n",
      "  - proxyC, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - RCurl, type: imports\n",
      "  - reticulate, type: imports\n",
      "  - sparseMatrixStats, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - tidyr, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3 + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/SimBu.html\n",
      "\n",
      "Name: zenith\n",
      "Description:\n",
      "Zenith performs gene set analysis on the result of differential expression using linear (mixed) modeling with dream by considering the correlation between gene expression traits. This package implements the camera method from the limma package proposed by Wu and Smyth (2012). Zenith is a simple extension of camera to be compatible with linear mixed models implemented in variancePartition::dream().\n",
      "Version: 1.0.1\n",
      "Publication date: None\n",
      "Mantainer: Gabriel Hoffman <gabriel.hoffman@mssm.edu>\n",
      "Authors:\n",
      "Author: Gabriel Hoffman [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.2.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - variancePartition, type: imports\n",
      "  - limma, type: imports\n",
      "  - EnrichmentBrowser, type: imports\n",
      "  - GSEABase, type: imports\n",
      "  - msigdbr (>= 7.5.1) type: imports\n",
      "  - Rfast, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - progress, type: imports\n",
      "  - utils, type: imports\n",
      "  - Rdpack, type: imports\n",
      "  - stats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/zenith.html\n",
      "\n",
      "Name: BiocFHIR\n",
      "Description:\n",
      "FHIR R4 bundles in JSON format are derived from https://synthea.mitre.org/downloads. Transformation inspired by a kaggle notebook published by Dr Alexander Scarlat, https://www.kaggle.com/code/drscarlat/fhir-starter-parse-healthcare-bundles-into-tables. This is a very limited illustration of some basic parsing and reorganization processes. Additional tooling will be required to move beyond the Synthea data illustrations.\n",
      "Version: 1.0.0\n",
      "Publication date: None\n",
      "Mantainer: Vincent Carey <stvjc@channing.harvard.edu>\n",
      "Authors:\n",
      "Author: Vincent Carey [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.2) type: depends\n",
      "  - DT, type: imports\n",
      "  - shiny, type: imports\n",
      "  - jsonlite, type: imports\n",
      "  - graph, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - visNetwork, type: imports\n",
      "  - igraph, type: imports\n",
      "  - utils, type: imports\n",
      "  - methods, type: imports\n",
      "  - BiocBaseUtils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/BiocFHIR.html\n",
      "\n",
      "Name: BioNAR\n",
      "Description:\n",
      "the R package BioNAR, developed to step by step analysis of PPI network. The aim is to quantify and rank each proteinâs simultaneous impact into multiple complexes based on network topology and clustering. Package also enables estimating of co-occurrence of diseases across the network and specific clusters pointing towards shared/common mechanisms.\n",
      "Version: 1.0.0\n",
      "Publication date: None\n",
      "Mantainer: Anatoly Sorokin <lptolik@gmail.com>\n",
      "Authors:\n",
      "Author: Colin Mclean [aut], Anatoly Sorokin [aut, cre], Oksana Sorokina [aut], J. Douglas Armstrong [aut, fnd], T. Ian Simpson [ctb, fnd]\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - igraph, type: depends\n",
      "  - poweRlaw, type: depends\n",
      "  - latex2exp, type: depends\n",
      "  - RSpectra, type: depends\n",
      "  - Rdpack, type: depends\n",
      "  - stringr, type: imports\n",
      "  - synaptome.db, type: imports\n",
      "  - clusterCons, type: imports\n",
      "  - fgsea, type: imports\n",
      "  - grid, type: imports\n",
      "  - methods, type: imports\n",
      "  - scales, type: imports\n",
      "  - AnnotationDbi, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - GO.db, type: imports\n",
      "  - org.Hs.eg.db, type: imports\n",
      "  - rSpectral, type: imports\n",
      "  - WGCNA, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - ggrepel, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/BioNAR.html\n",
      "\n",
      "Name: EpiMix\n",
      "Description:\n",
      "EpiMix is a comprehensive tool for the integrative analysis of high-throughput DNA methylation data and gene expression data. EpiMix enables automated data downloading (from TCGA or GEO), preprocessing, methylation modeling, interactive visualization and functional annotation.To identify hypo- or hypermethylated CpG sites across physiological or pathological conditions, EpiMix uses a beta mixture modeling to identify the methylation states of each CpG probe and compares the methylation of the experimental group to the control group.The output from EpiMix is the functional DNA methylation that is predictive of gene expression. EpiMix incorporates specialized algorithms to identify functional DNA methylation at various genetic elements, including proximal cis-regulatory elements of protein-coding genes, distal enhancers, and genes encoding microRNAs and lncRNAs.\n",
      "Version: 1.0.1\n",
      "Publication date: None\n",
      "Mantainer: Yuanning Zheng <eric2021@stanford.edu>\n",
      "Authors:\n",
      "Author: Yuanning Zheng [aut, cre], John Jun [aut], Olivier Gevaert [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.2.0) type: depends\n",
      "  - EpiMix.data, type: depends\n",
      "  - AnnotationHub, type: imports\n",
      "  - AnnotationDbi, type: imports\n",
      "  - Biobase, type: imports\n",
      "  - biomaRt, type: imports\n",
      "  - data.table, type: imports\n",
      "  - doParallel, type: imports\n",
      "  - doSNOW, type: imports\n",
      "  - downloader, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - ELMER.data, type: imports\n",
      "  - ExperimentHub, type: imports\n",
      "  - foreach, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - GenomicFeatures, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - GEOquery, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - graphics, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - impute, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - limma, type: imports\n",
      "  - methods, type: imports\n",
      "  - parallel, type: imports\n",
      "  - plyr, type: imports\n",
      "  - progress, type: imports\n",
      "  - R.matlab, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - RCurl, type: imports\n",
      "  - rlang, type: imports\n",
      "  - RPMM, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - stats, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - tibble, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/EpiMix.html\n",
      "\n",
      "Name: iSEEhub\n",
      "Description:\n",
      "This package defines a custom landing page for an iSEE app interfacing with the Bioconductor ExperimentHub. The landing page allows users to browse the ExperimentHub, select a data set, download and cache it, and import it directly into a Bioconductor iSEE app.\n",
      "Version: 1.0.0\n",
      "Publication date: None\n",
      "Mantainer: Kevin Rue-Albrecht <kevinrue67@gmail.com>\n",
      "Authors:\n",
      "Author: Kevin Rue-Albrecht [aut, cre]\n",
      "Dependencies:\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - SingleCellExperiment, type: depends\n",
      "  - ExperimentHub, type: depends\n",
      "  - AnnotationHub, type: imports\n",
      "  - BiocManager, type: imports\n",
      "  - DT, type: imports\n",
      "  - iSEE, type: imports\n",
      "  - methods, type: imports\n",
      "  - rintrojs, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - shiny, type: imports\n",
      "  - shinydashboard, type: imports\n",
      "  - shinyjs, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/iSEEhub.html\n",
      "\n",
      "Name: ISLET\n",
      "Description:\n",
      "ISLET is a method to conduct signal deconvolution for general -omics data. It can estimate the individual-specific and cell-type-specific reference panels, when there are multiple samples observed from each subject. It takes the input of the observed mixture data (feature by sample matrix), and the cell type mixture proportions (sample by cell type matrix), and the sample-to-subject information. It can solve for the reference panel on the individual-basis. It can also conduct test to identify cell-type-specific differential expression (csDE) genes.\n",
      "Version: 1.0.0\n",
      "Publication date: None\n",
      "Mantainer: Hao Feng <hxf155@case.edu>\n",
      "Authors:\n",
      "Author: Hao Feng [aut, cre], Qian Li [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.1.0) type: depends\n",
      "  - Matrix, type: depends\n",
      "  - parallel, type: depends\n",
      "  - BiocParallel, type: depends\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - BiocGenerics, type: depends\n",
      "  - stats, type: imports\n",
      "  - methods, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ISLET.html\n",
      "\n",
      "Name: magrene\n",
      "Description:\n",
      "magrene allows the identification and analysis of graph motifs in (duplicated) gene regulatory networks (GRNs), including lambda, V, PPI V, delta, and bifan motifs. GRNs can be tested for motif enrichment by comparing motif frequencies to a null distribution generated from degree-preserving simulated GRNs. Motif frequencies can be analyzed in the context of gene duplications to explore the impact of small-scale and whole-genome duplications on gene regulatory networks. Finally, users can calculate interaction similarity for gene pairs based on the Sorensen-Dice similarity index.\n",
      "Version: 1.0.0\n",
      "Publication date: None\n",
      "Mantainer: FabrÃ­cio Almeida-Silva <fabricio_almeidasilva@hotmail.com>\n",
      "Authors:\n",
      "Author: FabrÃ­cio Almeida-Silva [aut, cre] , Yves Van de Peer [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.2.0) type: depends\n",
      "  - utils, type: imports\n",
      "  - stats, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/magrene.html\n",
      "\n",
      "Name: NetActivity\n",
      "Description:\n",
      "#' NetActivity enables to compute gene set scores from previously trained sparsely-connected autoencoders. The package contains a function to prepare the data (`prepareSummarizedExperiment`) and a function to compute the gene set scores (`computeGeneSetScores`). The package `NetActivityData` contains different pre-trained models to be directly applied to the data. Alternatively, the users might use the package to compute gene set scores using custom models.\n",
      "Version: 1.0.0\n",
      "Publication date: None\n",
      "Mantainer: Carlos Ruiz-Arenas <carlos.ruiza@upf.edu>\n",
      "Authors:\n",
      "Author: Carlos Ruiz-Arenas [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.1.0) type: depends\n",
      "  - airway, type: imports\n",
      "  - DelayedArray, type: imports\n",
      "  - DelayedMatrixStats, type: imports\n",
      "  - DESeq2, type: imports\n",
      "  - methods, type: imports\n",
      "  - methods, type: imports\n",
      "  - NetActivityData, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/NetActivity.html\n",
      "\n",
      "Name: octad\n",
      "Description:\n",
      "OCTAD provides a platform for virtually screening compounds targeting precise cancer patient groups. The essential idea is to identify drugs that reverse the gene expression signature of disease by tamping down over-expressed genes and stimulating weakly expressed ones. The package offers deep-learning based reference tissue selection, disease gene expression signature creation, pathway enrichment analysis, drug reversal potency scoring, cancer cell line selection, drug enrichment analysis and in silico hit validation. It currently covers ~20,000 patient tissue samples covering 50 cancer types, and expression profiles for ~12,000 distinct compounds.\n",
      "Version: 1.0.0\n",
      "Publication date: None\n",
      "Mantainer: E. Chekalin <eygen.chekalin@gmail.com>\n",
      "Authors:\n",
      "Author: E. Chekalin [aut, cre], S. Paithankar [aut], B. Zeng [aut], B. Glicksberg [ctb], P. Newbury [ctb], J. Xing [ctb], K. Liu [ctb], A. Wen [ctb], D. Joseph [ctb], B. Chen [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.2.0) type: depends\n",
      "  - magrittr, type: depends\n",
      "  - dplyr, type: depends\n",
      "  - ggplot2, type: depends\n",
      "  - edgeR, type: depends\n",
      "  - RUVSeq, type: depends\n",
      "  - DESeq2, type: depends\n",
      "  - limma, type: depends\n",
      "  - rhdf5, type: depends\n",
      "  - foreach, type: depends\n",
      "  - Rfast, type: depends\n",
      "  - octad.db, type: depends\n",
      "  - stats, type: depends\n",
      "  - httr, type: depends\n",
      "  - ExperimentHub, type: depends\n",
      "  - AnnotationHub, type: depends\n",
      "  - Biobase, type: depends\n",
      "  - S4Vectors, type: depends\n",
      "  - EDASeq, type: imports\n",
      "  - GSVA, type: imports\n",
      "  - data.table, type: imports\n",
      "  - htmlwidgets, type: imports\n",
      "  - plotly, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/octad.html\n",
      "\n",
      "Name: regioneReloaded\n",
      "Description:\n",
      "RegioneReloaded is a package that allows simultaneous analysis of associations between genomic region sets, enabling clustering of data and the creation of ready-to-publish graphs. It takes over and expands on all the features of its predecessor regioneR. It also incorporates a strategy to improve p-value calculations and normalize z-scores coming from multiple analysis to allow for their direct comparison. RegioneReloaded builds upon regioneR by adding new plotting functions for obtaining publication-ready graphs.\n",
      "Version: 1.0.0\n",
      "Publication date: None\n",
      "Mantainer: Roberto Malinverni <roberto.malinverni@gmail.com>\n",
      "Authors:\n",
      "Author: Roberto Malinverni [aut, cre] , David Corujo [aut], Bernat Gel [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.2) type: depends\n",
      "  - regioneR, type: depends\n",
      "  - stats, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - Rtsne, type: imports\n",
      "  - umap, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - ggrepel, type: imports\n",
      "  - reshape2, type: imports\n",
      "  - methods, type: imports\n",
      "  - scales, type: imports\n",
      "  - cluster, type: imports\n",
      "  - grid, type: imports\n",
      "  - grDevices, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/regioneReloaded.html\n",
      "\n",
      "Name: scBubbletree\n",
      "Description:\n",
      "scBubbletree is a quantitative method for visual exploration of scRNA-seq data. It preserves biologically meaningful properties of scRNA-seq data, such as local and global cell distances, as well as the density distribution of cells across the sample. scBubbletree is scalable and avoids the overplotting problem, and is able to visualize diverse cell attributes derived from multiomic single-cell experiments. Importantly, Importantly, scBubbletree is easy to use and to integrate with popular approaches for scRNA-seq data analysis.\n",
      "Version: 1.0.0\n",
      "Publication date: None\n",
      "Mantainer: Simo Kitanovski <simokitanovski@gmail.com>\n",
      "Authors:\n",
      "Author: Simo Kitanovski [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.2.0) type: depends\n",
      "  - reshape2, type: imports\n",
      "  - future, type: imports\n",
      "  - future.apply, type: imports\n",
      "  - ape, type: imports\n",
      "  - scales, type: imports\n",
      "  - Seurat, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - ggtree, type: imports\n",
      "  - patchwork, type: imports\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "  - base, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3 + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/scBubbletree.html\n",
      "\n",
      "Name: signifinder\n",
      "Description:\n",
      "signifinder is an R package for computing and exploring a compendium of tumor signatures. It allows computing signatures scores providing the only gene expression values and returns a single-sample score. Currently, signifinder contains 46 distinct signatures collected from the literature.\n",
      "Version: 1.0.0\n",
      "Publication date: None\n",
      "Mantainer: Stefania Pirrotta <stefania.pirrotta@phd.unipd.it>\n",
      "Authors:\n",
      "Author: Stefania Pirrotta [cre, aut], Enrica Calura [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.2.0) type: depends\n",
      "  - ggplot2, type: imports\n",
      "  - org.Hs.eg.db, type: imports\n",
      "  - patchwork, type: imports\n",
      "  - AnnotationDbi, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - ComplexHeatmap, type: imports\n",
      "  - DGEobj.utils, type: imports\n",
      "  - GSVA, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - consensusOV, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - ensembldb, type: imports\n",
      "  - ggridges, type: imports\n",
      "  - grid, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - maxstat, type: imports\n",
      "  - methods, type: imports\n",
      "  - openair, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - TxDb.Hsapiens.UCSC.hg19.knownGene, type: imports\n",
      "  - TxDb.Hsapiens.UCSC.hg38.knownGene, type: imports\n",
      "  - stats, type: imports\n",
      "  - survival, type: imports\n",
      "  - survminer, type: imports\n",
      "  - viridis, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "AGPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/signifinder.html\n",
      "\n",
      "Name: TENxIO\n",
      "Description:\n",
      "Provides a structured S4 approach to importing data files from the 10X pipelines. It mainly supports Single Cell Multiome ATAC + Gene Expression data among other data types. The main Bioconductor data representations used are SingleCellExperiment and RaggedExperiment.\n",
      "Version: 1.0.1\n",
      "Publication date: None\n",
      "Mantainer: Marcel Ramos <marcel.ramos@roswellpark.org>\n",
      "Authors:\n",
      "Author: Marcel Ramos [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.2.0) type: depends\n",
      "  - SingleCellExperiment, type: depends\n",
      "  - SummarizedExperiment, type: depends\n",
      "  - BiocBaseUtils, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - BiocIO, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - MatrixGenerics, type: imports\n",
      "  - methods, type: imports\n",
      "  - RCurl, type: imports\n",
      "  - readr, type: imports\n",
      "  - R.utils, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/TENxIO.html\n",
      "\n",
      "Name: VDJdive\n",
      "Description:\n",
      "This package provides functions for handling and analyzing immune receptor repertoire data, such as produced by the CellRanger V(D)J pipeline. This includes reading the data into R, merging it with paired single-cell data, quantifying clonotype abundances, calculating diversity metrics, and producing common plots. It implements the E-M Algorithm for clonotype assignment, along with other methods, which makes use of ambiguous cells for improved quantification.\n",
      "Version: 1.0.0\n",
      "Publication date: None\n",
      "Mantainer: Kelly Street <street.kelly@gmail.com>\n",
      "Authors:\n",
      "Author: Kelly Street [aut, cre] , Mercedeh Movassagh [aut] , Jill Lundell [aut] , Jared Brown [ctb], Linglin Huang [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 4.2) type: depends\n",
      "  - basilisk, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "  - cowplot, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - gridExtra, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - Matrix, type: imports\n",
      "  - methods, type: imports\n",
      "  - RColorBrewer, type: imports\n",
      "  - reticulate, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - SingleCellExperiment, type: imports\n",
      "  - stats, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "  - utils, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/VDJdive.html\n",
      "\n",
      "Name: vsclust\n",
      "Description:\n",
      "Feature-based variance-sensitive clustering of omics data. Optimizes cluster assignment by taking into account individual feature variance. Includes several modules for statistical testing, clustering and enrichment analysis.\n",
      "Version: 1.0.0\n",
      "Publication date: None\n",
      "Mantainer: Veit Schwaemmle <veits@bmb.sdu.dk>\n",
      "Authors:\n",
      "Author: Veit Schwaemmle [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.2.0) type: depends\n",
      "  - matrixStats, type: imports\n",
      "  - limma, type: imports\n",
      "  - parallel, type: imports\n",
      "  - shiny, type: imports\n",
      "  - qvalue, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - stats, type: imports\n",
      "  - MultiAssayExperiment, type: imports\n",
      "  - graphics, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/vsclust.html\n",
      "\n",
      "Name: ATACCoGAPS\n",
      "Description:\n",
      "Provides tools for running the CoGAPS algorithm (Fertig et al, 2010) on single-cell ATAC sequencing data and analysis of the results. Can be used to perform analyses at the level of genes, motifs, TFs, or pathways. Additionally provides tools for transfer learning and data integration with single-cell RNA sequencing data.\n",
      "Version: 1.0.0\n",
      "Publication date: None\n",
      "Mantainer: Rossin Erbe <rerbe1@jhmi.edu>\n",
      "Authors:\n",
      "Author: Rossin Erbe [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.2.0) type: depends\n",
      "  - CoGAPS, type: depends\n",
      "  - gtools, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - projectR, type: imports\n",
      "  - TFBSTools, type: imports\n",
      "  - GeneOverlap, type: imports\n",
      "  - msigdbr, type: imports\n",
      "  - tidyverse, type: imports\n",
      "  - gplots, type: imports\n",
      "  - motifmatchr, type: imports\n",
      "  - chromVAR, type: imports\n",
      "  - GenomicFeatures, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - fgsea, type: imports\n",
      "  - rGREAT, type: imports\n",
      "  - JASPAR2016, type: imports\n",
      "  - Homo.sapiens, type: imports\n",
      "  - Mus.musculus, type: imports\n",
      "  - BSgenome.Hsapiens.UCSC.hg19, type: imports\n",
      "  - BSgenome.Mmusculus.UCSC.mm10, type: imports\n",
      "  - stringr, type: imports\n",
      "  - dplyr, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ATACCoGAPS.html\n",
      "\n",
      "Name: ccImpute\n",
      "Description:\n",
      "Dropout events make the lowly expressed genes indistinguishable from true zero expression and different than the low expression present in cells of the same type. This issue makes any subsequent downstream analysis difficult. ccImpute is an imputation algorithm that uses cell similarity established by consensus clustering to impute the most probable dropout events in the scRNA-seq datasets. ccImpute demonstrated performance which exceeds the performance of existing imputation approaches while introducing the least amount of new noise as measured by clustering performance characteristics on datasets with known cell identities.\n",
      "Version: 1.0.2\n",
      "Publication date: None\n",
      "Mantainer: Marcin Malec <mamalec@iu.edu>\n",
      "Authors:\n",
      "Author: Marcin Malec [cre, aut]\n",
      "Dependencies:\n",
      "  - Rcpp, type: imports\n",
      "  - matrixStats, type: imports\n",
      "  - stats, type: imports\n",
      "  - SIMLR, type: imports\n",
      "  - BiocParallel, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/ccImpute.html\n",
      "\n",
      "Name: MetaPhOR\n",
      "Description:\n",
      "MetaPhOR was developed to enable users to assess metabolic dysregulation using transcriptomic-level data (RNA-sequencing and Microarray data) and produce publication-quality figures. A list of differentially expressed genes (DEGs), which includes fold change and p value, from DESeq2 or limma, can be used as input, with sample size for MetaPhOR, and will produce a data frame of scores for each KEGG pathway. These scores represent the magnitude and direction of transcriptional change within the pathway, along with estimated p-values.MetaPhOR then uses these scores to visualize metabolic profiles within and between samples through a variety of mechanisms, including: bubble plots, heatmaps, and pathway models.\n",
      "Version: 1.0.0\n",
      "Publication date: None\n",
      "Mantainer: Emily Isenhart <emily.isenhart@roswellpark.org>\n",
      "Authors:\n",
      "Author: Emily Isenhart [aut, cre], Spencer Rosario [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.2.0) type: depends\n",
      "  - utils, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - ggrepel, type: imports\n",
      "  - stringr, type: imports\n",
      "  - pheatmap, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - stats, type: imports\n",
      "  - clusterProfiler, type: imports\n",
      "  - RecordLinkage, type: imports\n",
      "  - RCy3, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/MetaPhOR.html\n",
      "\n",
      "Name: omada\n",
      "Description:\n",
      "Symptomatic heterogeneity in complex diseases reveals differences in molecular states that need to be investigated. However, selecting the numerous parameters of an exploratory clustering analysis in RNA profiling studies requires deep understanding of machine learning and extensive computational experimentation. Tools that assist with such decisions without prior field knowledge are nonexistent and further gene association analyses need to be performed independently. We have developed a suite of tools to automate these processes and make robust unsupervised clustering of transcriptomic data more accessible through automated machine learning based functions. The efficiency of each tool was tested with four datasets characterised by different expression signal strengths. Our toolkitâs decisions reflected the real number of stable partitions in datasets where the subgroups are discernible. Even in datasets with less clear biological distinctions, stable subgroups with different expression profiles and clinical associations were found.\n",
      "Version: 1.0.0\n",
      "Publication date: None\n",
      "Mantainer: Sokratis Kariotis <sokratiskariotis@gmail.com>\n",
      "Authors:\n",
      "Author: Sokratis Kariotis [aut, cre]\n",
      "Dependencies:\n",
      "  - pdfCluster (>= 1.0-3) type: depends\n",
      "  - kernlab (>= 0.9-29) type: depends\n",
      "  - R (>= 4.2) type: depends\n",
      "  - fpc (>= 2.2-9) type: depends\n",
      "  - Rcpp (>= 1.0.7) type: depends\n",
      "  - diceR (>= 0.6.0) type: depends\n",
      "  - ggplot2 (>= 3.3.5) type: depends\n",
      "  - reshape (>= 0.8.8) type: depends\n",
      "  - clusterCrit (>= 1.2.8) type: depends\n",
      "  - clValid (>= 0.7) type: depends\n",
      "  - glmnet (>= 4.1.3) type: depends\n",
      "  - dplyr (>= 1.0.7) type: depends\n",
      "  - stats (>= 4.1.2) type: depends\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/omada.html\n",
      "\n",
      "Name: RESOLVE\n",
      "Description:\n",
      "Cancer is a genetic disease caused by somatic mutations in genes controlling key biological functions such as cellular growth and division. Such mutations may arise both through cell-intrinsic and exogenous processes, generating characteristic mutational patterns over the genome named mutational signatures. The study of mutational signatures have become a standard component of modern genomics studies, since it can reveal which (environmental and endogenous) mutagenic processes are active in a tumor, and may highlight markers for therapeutic response. Mutational signatures computational analysis presents many pitfalls. First, the task of determining the number of signatures is very complex and depends on heuristics. Second, several signatures have no clear etiology, casting doubt on them being computational artifacts rather than due to mutagenic processes. Last, approaches for signatures assignment are greatly influenced by the set of signatures used for the analysis. To overcome these limitations, we developed RESOLVE (Robust EStimation Of mutationaL signatures Via rEgularization), a framework that allows the efficient extraction and assignment of mutational signatures. RESOLVE implements a novel algorithm that enables (i) the efficient extraction, (ii) exposure estimation, and (iii) confidence assessment during the computational inference of mutational signatures.\n",
      "Version: 1.0.0\n",
      "Publication date: None\n",
      "Mantainer: Luca De Sano <luca.desano@gmail.com>\n",
      "Authors:\n",
      "Author: Daniele Ramazzotti [aut] , Luca De Sano [cre, aut]\n",
      "Dependencies:\n",
      "  - R (>= 3.5.0) type: depends\n",
      "  - NMF, type: depends\n",
      "  - Biostrings, type: imports\n",
      "  - BSgenome, type: imports\n",
      "  - BSgenome.Hsapiens.1000genomes.hs37d5, type: imports\n",
      "  - cluster, type: imports\n",
      "  - data.table, type: imports\n",
      "  - GenomeInfoDb, type: imports\n",
      "  - GenomicRanges, type: imports\n",
      "  - glmnet, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - gridExtra, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - lsa, type: imports\n",
      "  - nnls, type: imports\n",
      "  - parallel, type: imports\n",
      "  - reshape2, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/RESOLVE.html\n",
      "\n",
      "Name: scifer\n",
      "Description:\n",
      "Have you ever index sorted cells in a 96 or 384-well plate and then sequenced using Sanger sequencing? If so, you probably had some struggles to either check the electropherogram of each cell sequenced manually, or when you tried to identify which cell was sorted where after sequencing the plate. Scifer was developed to solve this issue by performing basic quality control of Sanger sequences and merging flow cytometry data from probed single-cell sorted B cells with sequencing data. scifer can export summary tables, 'fasta' files, electropherograms for visual inspection, and generate reports.\n",
      "Version: 1.0.0\n",
      "Publication date: None\n",
      "Mantainer: Rodrigo Arcoverde Cerveira <rodrigo.arcoverdi@gmail.com>\n",
      "Authors:\n",
      "Author: Rodrigo Arcoverde Cerveira [aut, cre, cph] , Sebastian Ols [aut, dtc] , Karin LorÃ© [dtc, ths]\n",
      "Dependencies:\n",
      "  - dplyr, type: imports\n",
      "  - rmarkdown, type: imports\n",
      "  - data.table, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - parallel, type: imports\n",
      "  - stats, type: imports\n",
      "  - plyr, type: imports\n",
      "  - knitr, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - gridExtra, type: imports\n",
      "  - DECIPHER, type: imports\n",
      "  - stringr, type: imports\n",
      "  - sangerseqR, type: imports\n",
      "  - kableExtra, type: imports\n",
      "  - tibble, type: imports\n",
      "  - scales, type: imports\n",
      "  - rlang, type: imports\n",
      "  - flowCore, type: imports\n",
      "  - methods, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/scifer.html\n",
      "\n",
      "Name: Statial\n",
      "Description:\n",
      "Statial is a suite of functions for identifying changes in cell state. The functionality provided by Statial provides robust quantification of cell type localisation which are invariant to changes in tissue structure. In addition to this Statial uncovers changes in marker expression associated with varying levels of localisation. These features can be used to explore how the structure and function of different cell types may be altered by the agents they are surrounded with.\n",
      "Version: 1.0.1\n",
      "Publication date: None\n",
      "Mantainer: Farhan Ameen <fame2827@uni.sydney.edu.au>\n",
      "Authors:\n",
      "Author: Farhan Ameen [aut, cre], Sourish Iyengar [aut], Shila Ghazanfar [aut], Ellis Patrick [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.1.0) type: depends\n",
      "  - BiocParallel, type: imports\n",
      "  - spatstat.geom, type: imports\n",
      "  - concaveman, type: imports\n",
      "  - tidyverse, type: imports\n",
      "  - data.table, type: imports\n",
      "  - spatstat.explore, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - SingleCellExperiment, type: imports\n",
      "  - tibble, type: imports\n",
      "  - stringr, type: imports\n",
      "  - tidyselect, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - methods, type: imports\n",
      "  - stats, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/Statial.html\n",
      "\n",
      "Name: stJoincount\n",
      "Description:\n",
      "stJoincount, the application of join count analysis to the spatial transcriptomics dataset. This tool converts the spatial map into a raster object (a two-dimensional image as a rectangular matrix or grid of square pixels), where clusters labelled spots are converted to adjacent pixels with a calculated resolution. A neighbors' list was created based on the rasterized sample, which identifies adjacent and diagonal neighbors for each pixel. After adding binary spatial weights to the neighbors' list, a multi-categorical join count analysis is then performed, allowing all possible combinations of cluster pairings to be tabulated. The function returns the observed join counts, the expected count under conditions of spatial randomness, and the variance of observed to expected calculated under non-free sampling. The z-score is then calculated as the difference between observed and expected counts, divided by the square root of the variance.\n",
      "Version: 1.0.0\n",
      "Publication date: None\n",
      "Mantainer: Jiarong Song <songjiar@usc.edu>\n",
      "Authors:\n",
      "Author: Jiarong Song [cre, aut] , Rania Bassiouni [aut], David Craig [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.2.0) type: depends\n",
      "  - graphics, type: imports\n",
      "  - stats, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - sp, type: imports\n",
      "  - raster, type: imports\n",
      "  - spdep, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - pheatmap, type: imports\n",
      "  - grDevices, type: imports\n",
      "  - Seurat, type: imports\n",
      "  - SpatialExperiment, type: imports\n",
      "  - SummarizedExperiment, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/stJoincount.html\n",
      "\n",
      "Name: CircSeqAlignTk\n",
      "Description:\n",
      "CircSeqAlignTk is designed for end-to-end RNA-Seq data analysis of circular genome sequences, from alignment to visualization. It mainly targets viroids which are composed of 246-401 nt circular RNAs. In addition, CircSeqAlignTk implements a tidy interface to generate synthetic sequencing data that mimic real RNA-Seq data, allowing developers to evaluate the performance of alignment tools and workflows.\n",
      "Version: 1.0.0\n",
      "Publication date: None\n",
      "Mantainer: Jianqiang Sun <sun@biunit.dev>\n",
      "Authors:\n",
      "Author: Jianqiang Sun [aut, cre] , Xi Fu [aut], Wei Cao [aut]\n",
      "Dependencies:\n",
      "  - R (>= 4.2) type: depends\n",
      "  - stats, type: imports\n",
      "  - tools, type: imports\n",
      "  - utils, type: imports\n",
      "  - methods, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "  - rlang, type: imports\n",
      "  - magrittr, type: imports\n",
      "  - dplyr, type: imports\n",
      "  - tidyr, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - BiocGenerics, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - ShortRead, type: imports\n",
      "  - Rsamtools, type: imports\n",
      "  - Rbowtie2, type: imports\n",
      "  - Rhisat2, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "MIT + file LICENSE\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/CircSeqAlignTk.html\n",
      "\n",
      "Name: metabinR\n",
      "Description:\n",
      "Provide functions for performing abundance and compositional based binning on metagenomic samples, directly from FASTA or FASTQ files. Functions are implemented in Java and called via rJava. Parallel implementation that operates directly on input FASTA/FASTQ files for fast execution.\n",
      "Version: 1.0.0\n",
      "Publication date: None\n",
      "Mantainer: Anestis Gkanogiannis <anestis@gkanogiannis.com>\n",
      "Authors:\n",
      "Author: Anestis Gkanogiannis [aut, cre]\n",
      "Dependencies:\n",
      "  - R (>= 4.2) type: depends\n",
      "  - methods, type: imports\n",
      "  - rJava, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-3\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/metabinR.html\n",
      "\n",
      "Name: XCIR\n",
      "Description:\n",
      "Bioconductor version: Release (3.16)\n",
      "Version: 1.12.0\n",
      "Publication date: None\n",
      "Mantainer: Author: Renan Sauteraud, Dajiang Liu\n",
      "Authors:\n",
      "Models and tools for subject level analysis of X chromosome inactivation (XCI) and XCI-escape inference.\n",
      "Dependencies:\n",
      "  - methods, type: depends\n",
      "  - stats, type: imports\n",
      "  - utils, type: imports\n",
      "  - tools, type: imports\n",
      "  - data.table, type: imports\n",
      "  - Biostrings, type: imports\n",
      "  - IRanges, type: imports\n",
      "  - VariantAnnotation, type: imports\n",
      "  - seqminer, type: imports\n",
      "  - ggplot2, type: imports\n",
      "  - biomaRt, type: imports\n",
      "  - readxl, type: imports\n",
      "  - S4Vectors, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "GPL-2\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/XCIR.html\n",
      "\n"
     ]
    }
   ],
   "source": [
    "# Dump the list of packages\n",
    "for p in bioc_package_list:\n",
    "    p.dump()"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## **2.3 - Data export**"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "In this section the issue of export of the data obtained is going to be discussed.\n",
    "\n",
    "As a format, a CSV file has been chosen with the same structure used by ***Libraries.io*** and in turn ***Olivia***"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "**We remember the structure of the CSV:**"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "The CSV to generate will have a structure similar to the CSV of dependencies to try to make the maximum with Olivia."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>ID</th>\n",
       "      <th>Platform</th>\n",
       "      <th>Project Name</th>\n",
       "      <th>Project ID</th>\n",
       "      <th>Version Number</th>\n",
       "      <th>Version ID</th>\n",
       "      <th>Dependency Name</th>\n",
       "      <th>Dependency Platform</th>\n",
       "      <th>Dependency Kind</th>\n",
       "      <th>Optional Dependency</th>\n",
       "      <th>Dependency Requirements</th>\n",
       "      <th>Dependency Project ID</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>66065047</td>\n",
       "      <td>CRAN</td>\n",
       "      <td>nmfem</td>\n",
       "      <td>2524222</td>\n",
       "      <td>1.0.0</td>\n",
       "      <td>11273453</td>\n",
       "      <td>R</td>\n",
       "      <td>CRAN</td>\n",
       "      <td>depends</td>\n",
       "      <td>False</td>\n",
       "      <td>*</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>66065056</td>\n",
       "      <td>CRAN</td>\n",
       "      <td>nmfem</td>\n",
       "      <td>2524222</td>\n",
       "      <td>1.0.0</td>\n",
       "      <td>11273453</td>\n",
       "      <td>plyr</td>\n",
       "      <td>CRAN</td>\n",
       "      <td>imports</td>\n",
       "      <td>False</td>\n",
       "      <td>*</td>\n",
       "      <td>691054.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>66065066</td>\n",
       "      <td>CRAN</td>\n",
       "      <td>nmfem</td>\n",
       "      <td>2524222</td>\n",
       "      <td>1.0.0</td>\n",
       "      <td>11273453</td>\n",
       "      <td>dplyr</td>\n",
       "      <td>CRAN</td>\n",
       "      <td>imports</td>\n",
       "      <td>False</td>\n",
       "      <td>*</td>\n",
       "      <td>688151.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>66065076</td>\n",
       "      <td>CRAN</td>\n",
       "      <td>nmfem</td>\n",
       "      <td>2524222</td>\n",
       "      <td>1.0.0</td>\n",
       "      <td>11273453</td>\n",
       "      <td>d3heatmap</td>\n",
       "      <td>CRAN</td>\n",
       "      <td>imports</td>\n",
       "      <td>False</td>\n",
       "      <td>*</td>\n",
       "      <td>957160.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>66065086</td>\n",
       "      <td>CRAN</td>\n",
       "      <td>nmfem</td>\n",
       "      <td>2524222</td>\n",
       "      <td>1.0.0</td>\n",
       "      <td>11273453</td>\n",
       "      <td>mixtools</td>\n",
       "      <td>CRAN</td>\n",
       "      <td>imports</td>\n",
       "      <td>False</td>\n",
       "      <td>*</td>\n",
       "      <td>690560.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "         ID Platform Project Name  Project ID Version Number  Version ID  \\\n",
       "0  66065047     CRAN        nmfem     2524222          1.0.0    11273453   \n",
       "1  66065056     CRAN        nmfem     2524222          1.0.0    11273453   \n",
       "2  66065066     CRAN        nmfem     2524222          1.0.0    11273453   \n",
       "3  66065076     CRAN        nmfem     2524222          1.0.0    11273453   \n",
       "4  66065086     CRAN        nmfem     2524222          1.0.0    11273453   \n",
       "\n",
       "  Dependency Name Dependency Platform Dependency Kind  Optional Dependency  \\\n",
       "0               R                CRAN         depends                False   \n",
       "1            plyr                CRAN         imports                False   \n",
       "2           dplyr                CRAN         imports                False   \n",
       "3       d3heatmap                CRAN         imports                False   \n",
       "4        mixtools                CRAN         imports                False   \n",
       "\n",
       "  Dependency Requirements  Dependency Project ID  \n",
       "0                       *                    NaN  \n",
       "1                       *               691054.0  \n",
       "2                       *               688151.0  \n",
       "3                       *               957160.0  \n",
       "4                       *               690560.0  "
      ]
     },
     "execution_count": 2,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "dependencies_df = pd.read_csv('data/dependencies-1.6.0-2020-01-12-cran.csv')\n",
    "\n",
    "# We show the first 5 rows of the dataframe\n",
    "dependencies_df.head()"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "**To generate these CSV we use the data obtained through scraping**"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### **2.3.1 - Building CRAN csv**"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "**<span style=\"color:yellow\">Load the package objects</span>**\n",
    "\n",
    "First we load the list of objects generated above"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Number of packages:  19248\n"
     ]
    }
   ],
   "source": [
    "# Read the file with the list of bioconductor packages\n",
    "with open(\"data\" + os.sep + \"CRAN_package_object_list.pkl\", \"rb\") as f:\n",
    "    CRAN_package_list = pickle.load(f)\n",
    "\n",
    "print(\"Number of packages: \", len(CRAN_package_list))"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "**<span style=\"color:yellow\">Build the csv</span>**\n",
    "\n",
    "\n",
    "We prepare the CSV structure"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "metadata": {},
   "outputs": [],
   "source": [
    "# ID,Platform,Project Name,Project ID,Version Number,Version ID,Dependency Name,Dependency Platform,Dependency Kind,Optional Dependency,Dependency Requirements,Dependency Project ID\n",
    "columns = [\n",
    "    \"ID\",\n",
    "    \"Platform\",\n",
    "    \"Project Name\",\n",
    "    \"Project ID\",\n",
    "    \"Version Number\", \n",
    "    \"Version ID\", \n",
    "    \"Dependency Name\", \n",
    "    \"Dependency Platform\", \n",
    "    \"Dependency Kind\", \n",
    "    \"Optional Dependency\", \n",
    "    \"Dependency Requirements\", \n",
    "    \"Dependency Project ID\"\n",
    "]\n",
    "\n",
    "# We create a dataframe with the data from the Bioconductor packages\n",
    "CRAN_df = pd.DataFrame(columns=columns)"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "We generate the CSV from the data"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "100%|██████████| 19248/19248 [15:29<00:00, 20.71it/s]\n"
     ]
    }
   ],
   "source": [
    "# For each Cran package, we store the dependencies in the dataframe\n",
    "import warnings\n",
    "warnings.simplefilter(action='ignore', category=FutureWarning)\n",
    "\n",
    "for p in tqdm(CRAN_package_list):\n",
    "    for d in p.dependencies:\n",
    "\n",
    "        # Build the row\n",
    "        d = {\n",
    "            \"ID\": None,\n",
    "            \"Platform\": \"CRAN\",\n",
    "            \"Project Name\": p.name,\n",
    "            \"Project ID\": None,\n",
    "            \"Version Number\": p.version,\n",
    "            \"Version ID\": None,\n",
    "            \"Dependency Name\": d.name,\n",
    "            \"Dependency Platform\": \"CRAN\",\n",
    "            \"Dependency Kind\": d.type,\n",
    "            \"Optional Dependency\": None,\n",
    "            \"Dependency Requirements\": None,\n",
    "            \"Dependency Project ID\": None,\n",
    "        }\n",
    "        \n",
    "        # Add the row to the dataframe\n",
    "        CRAN_df = CRAN_df.append(d, ignore_index=True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Save the dataframe in a csv file\n",
    "CRAN_df.to_csv(\"data\" + os.sep + \"CRAN_scraped_dependencies.csv\", index=False)\n"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### **2.3.2 - Building Bioconductor csv**"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "**<span style=\"color:yellow\">Load package object data</span>**\n",
    "\n",
    "\n",
    "The process is the same as we have done in the previous section for CRAN"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Read the file with the list of bioconductor packages\n",
    "with open(\"data\" + os.sep + \"bioconductor_package_object_list.pkl\", \"rb\") as f:\n",
    "    bioc_package_list = pickle.load(f)"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "**<span style=\"color:yellow\">Build csv file</span>**\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [],
   "source": [
    "# ID,Platform,Project Name,Project ID,Version Number,Version ID,Dependency Name,Dependency Platform,Dependency Kind,Optional Dependency,Dependency Requirements,Dependency Project ID\n",
    "columns = [\n",
    "    \"ID\",\n",
    "    \"Platform\",\n",
    "    \"Project Name\",\n",
    "    \"Project ID\",\n",
    "    \"Version Number\", \n",
    "    \"Version ID\", \n",
    "    \"Dependency Name\", \n",
    "    \"Dependency Platform\", \n",
    "    \"Dependency Kind\", \n",
    "    \"Optional Dependency\", \n",
    "    \"Dependency Requirements\", \n",
    "    \"Dependency Project ID\"\n",
    "]\n",
    "\n",
    "# We create a dataframe with the data from the Bioconductor packages\n",
    "bioc_df = pd.DataFrame(columns=columns)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "100%|██████████| 2183/2183 [01:30<00:00, 24.16it/s]\n"
     ]
    }
   ],
   "source": [
    "# For each Bioconductor package, we store the dependencies in the dataframe\n",
    "import warnings\n",
    "warnings.simplefilter(action='ignore', category=FutureWarning)\n",
    "\n",
    "for p in tqdm(bioc_package_list):\n",
    "    for d in p.dependencies:\n",
    "\n",
    "        # Build the row\n",
    "        d = {\n",
    "            \"ID\": None,\n",
    "            \"Platform\": \"Bioconductor\",\n",
    "            \"Project Name\": p.name,\n",
    "            \"Project ID\": None,\n",
    "            \"Version Number\": p.version,\n",
    "            \"Version ID\": None,\n",
    "            \"Dependency Name\": d.name,\n",
    "            \"Dependency Platform\": \"Bioconductor\",\n",
    "            \"Dependency Kind\": d.type,\n",
    "            \"Optional Dependency\": None,\n",
    "            \"Dependency Requirements\": None,\n",
    "            \"Dependency Project ID\": None,\n",
    "        }\n",
    "        \n",
    "        # Add the row to the dataframe\n",
    "        bioc_df = bioc_df.append(d, ignore_index=True)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Save the dataframe in a csv file\n",
    "bioc_df.to_csv(\"data\" + os.sep + \"bioconductor_scraped_dependencies.csv\", index=False)"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "**<span style=\"color:yellow\">Visualize some data</span>**"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Load the dataframe from the csv file\n",
    "bioc_df = pd.read_csv(\"data\" + os.sep + \"bioconductor_scraped_dependencies.csv\")\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>ID</th>\n",
       "      <th>Platform</th>\n",
       "      <th>Project Name</th>\n",
       "      <th>Project ID</th>\n",
       "      <th>Version Number</th>\n",
       "      <th>Version ID</th>\n",
       "      <th>Dependency Name</th>\n",
       "      <th>Dependency Platform</th>\n",
       "      <th>Dependency Kind</th>\n",
       "      <th>Optional Dependency</th>\n",
       "      <th>Dependency Requirements</th>\n",
       "      <th>Dependency Project ID</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>27769</th>\n",
       "      <td>NaN</td>\n",
       "      <td>Bioconductor</td>\n",
       "      <td>XCIR</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1.12.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>tools</td>\n",
       "      <td>Bioconductor</td>\n",
       "      <td>imports</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>27770</th>\n",
       "      <td>NaN</td>\n",
       "      <td>Bioconductor</td>\n",
       "      <td>XCIR</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1.12.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>data.table</td>\n",
       "      <td>Bioconductor</td>\n",
       "      <td>imports</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>27771</th>\n",
       "      <td>NaN</td>\n",
       "      <td>Bioconductor</td>\n",
       "      <td>XCIR</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1.12.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Biostrings</td>\n",
       "      <td>Bioconductor</td>\n",
       "      <td>imports</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>27772</th>\n",
       "      <td>NaN</td>\n",
       "      <td>Bioconductor</td>\n",
       "      <td>XCIR</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1.12.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>IRanges</td>\n",
       "      <td>Bioconductor</td>\n",
       "      <td>imports</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>27773</th>\n",
       "      <td>NaN</td>\n",
       "      <td>Bioconductor</td>\n",
       "      <td>XCIR</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1.12.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>VariantAnnotation</td>\n",
       "      <td>Bioconductor</td>\n",
       "      <td>imports</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>27774</th>\n",
       "      <td>NaN</td>\n",
       "      <td>Bioconductor</td>\n",
       "      <td>XCIR</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1.12.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>seqminer</td>\n",
       "      <td>Bioconductor</td>\n",
       "      <td>imports</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>27775</th>\n",
       "      <td>NaN</td>\n",
       "      <td>Bioconductor</td>\n",
       "      <td>XCIR</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1.12.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>ggplot2</td>\n",
       "      <td>Bioconductor</td>\n",
       "      <td>imports</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>27776</th>\n",
       "      <td>NaN</td>\n",
       "      <td>Bioconductor</td>\n",
       "      <td>XCIR</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1.12.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>biomaRt</td>\n",
       "      <td>Bioconductor</td>\n",
       "      <td>imports</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>27777</th>\n",
       "      <td>NaN</td>\n",
       "      <td>Bioconductor</td>\n",
       "      <td>XCIR</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1.12.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>readxl</td>\n",
       "      <td>Bioconductor</td>\n",
       "      <td>imports</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>27778</th>\n",
       "      <td>NaN</td>\n",
       "      <td>Bioconductor</td>\n",
       "      <td>XCIR</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1.12.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>S4Vectors</td>\n",
       "      <td>Bioconductor</td>\n",
       "      <td>imports</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "       ID      Platform Project Name  Project ID Version Number  Version ID  \\\n",
       "27769 NaN  Bioconductor         XCIR         NaN         1.12.0         NaN   \n",
       "27770 NaN  Bioconductor         XCIR         NaN         1.12.0         NaN   \n",
       "27771 NaN  Bioconductor         XCIR         NaN         1.12.0         NaN   \n",
       "27772 NaN  Bioconductor         XCIR         NaN         1.12.0         NaN   \n",
       "27773 NaN  Bioconductor         XCIR         NaN         1.12.0         NaN   \n",
       "27774 NaN  Bioconductor         XCIR         NaN         1.12.0         NaN   \n",
       "27775 NaN  Bioconductor         XCIR         NaN         1.12.0         NaN   \n",
       "27776 NaN  Bioconductor         XCIR         NaN         1.12.0         NaN   \n",
       "27777 NaN  Bioconductor         XCIR         NaN         1.12.0         NaN   \n",
       "27778 NaN  Bioconductor         XCIR         NaN         1.12.0         NaN   \n",
       "\n",
       "         Dependency Name Dependency Platform Dependency Kind  \\\n",
       "27769              tools        Bioconductor         imports   \n",
       "27770         data.table        Bioconductor         imports   \n",
       "27771         Biostrings        Bioconductor         imports   \n",
       "27772            IRanges        Bioconductor         imports   \n",
       "27773  VariantAnnotation        Bioconductor         imports   \n",
       "27774           seqminer        Bioconductor         imports   \n",
       "27775            ggplot2        Bioconductor         imports   \n",
       "27776            biomaRt        Bioconductor         imports   \n",
       "27777             readxl        Bioconductor         imports   \n",
       "27778          S4Vectors        Bioconductor         imports   \n",
       "\n",
       "       Optional Dependency  Dependency Requirements  Dependency Project ID  \n",
       "27769                  NaN                      NaN                    NaN  \n",
       "27770                  NaN                      NaN                    NaN  \n",
       "27771                  NaN                      NaN                    NaN  \n",
       "27772                  NaN                      NaN                    NaN  \n",
       "27773                  NaN                      NaN                    NaN  \n",
       "27774                  NaN                      NaN                    NaN  \n",
       "27775                  NaN                      NaN                    NaN  \n",
       "27776                  NaN                      NaN                    NaN  \n",
       "27777                  NaN                      NaN                    NaN  \n",
       "27778                  NaN                      NaN                    NaN  "
      ]
     },
     "execution_count": 5,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "bioc_df.tail(10)"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "**<span style=\"color:yellow\">Compare object integrity vs registration in the CSV</span>**\n",
    "\n",
    "As can be seen below, the information is consistent and identical in both structures"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>ID</th>\n",
       "      <th>Platform</th>\n",
       "      <th>Project Name</th>\n",
       "      <th>Project ID</th>\n",
       "      <th>Version Number</th>\n",
       "      <th>Version ID</th>\n",
       "      <th>Dependency Name</th>\n",
       "      <th>Dependency Platform</th>\n",
       "      <th>Dependency Kind</th>\n",
       "      <th>Optional Dependency</th>\n",
       "      <th>Dependency Requirements</th>\n",
       "      <th>Dependency Project ID</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>16</th>\n",
       "      <td>NaN</td>\n",
       "      <td>Bioconductor</td>\n",
       "      <td>GenomeInfoDb</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1.34.9</td>\n",
       "      <td>NaN</td>\n",
       "      <td>R</td>\n",
       "      <td>Bioconductor</td>\n",
       "      <td>depends</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>17</th>\n",
       "      <td>NaN</td>\n",
       "      <td>Bioconductor</td>\n",
       "      <td>GenomeInfoDb</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1.34.9</td>\n",
       "      <td>NaN</td>\n",
       "      <td>methods</td>\n",
       "      <td>Bioconductor</td>\n",
       "      <td>depends</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18</th>\n",
       "      <td>NaN</td>\n",
       "      <td>Bioconductor</td>\n",
       "      <td>GenomeInfoDb</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1.34.9</td>\n",
       "      <td>NaN</td>\n",
       "      <td>BiocGenerics</td>\n",
       "      <td>Bioconductor</td>\n",
       "      <td>depends</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19</th>\n",
       "      <td>NaN</td>\n",
       "      <td>Bioconductor</td>\n",
       "      <td>GenomeInfoDb</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1.34.9</td>\n",
       "      <td>NaN</td>\n",
       "      <td>S4Vectors</td>\n",
       "      <td>Bioconductor</td>\n",
       "      <td>depends</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>20</th>\n",
       "      <td>NaN</td>\n",
       "      <td>Bioconductor</td>\n",
       "      <td>GenomeInfoDb</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1.34.9</td>\n",
       "      <td>NaN</td>\n",
       "      <td>IRanges</td>\n",
       "      <td>Bioconductor</td>\n",
       "      <td>depends</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>21</th>\n",
       "      <td>NaN</td>\n",
       "      <td>Bioconductor</td>\n",
       "      <td>GenomeInfoDb</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1.34.9</td>\n",
       "      <td>NaN</td>\n",
       "      <td>stats</td>\n",
       "      <td>Bioconductor</td>\n",
       "      <td>imports</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>22</th>\n",
       "      <td>NaN</td>\n",
       "      <td>Bioconductor</td>\n",
       "      <td>GenomeInfoDb</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1.34.9</td>\n",
       "      <td>NaN</td>\n",
       "      <td>stats4</td>\n",
       "      <td>Bioconductor</td>\n",
       "      <td>imports</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>23</th>\n",
       "      <td>NaN</td>\n",
       "      <td>Bioconductor</td>\n",
       "      <td>GenomeInfoDb</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1.34.9</td>\n",
       "      <td>NaN</td>\n",
       "      <td>utils</td>\n",
       "      <td>Bioconductor</td>\n",
       "      <td>imports</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>24</th>\n",
       "      <td>NaN</td>\n",
       "      <td>Bioconductor</td>\n",
       "      <td>GenomeInfoDb</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1.34.9</td>\n",
       "      <td>NaN</td>\n",
       "      <td>RCurl</td>\n",
       "      <td>Bioconductor</td>\n",
       "      <td>imports</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>25</th>\n",
       "      <td>NaN</td>\n",
       "      <td>Bioconductor</td>\n",
       "      <td>GenomeInfoDb</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1.34.9</td>\n",
       "      <td>NaN</td>\n",
       "      <td>GenomeInfoDbData</td>\n",
       "      <td>Bioconductor</td>\n",
       "      <td>imports</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "    ID      Platform  Project Name  Project ID Version Number  Version ID  \\\n",
       "16 NaN  Bioconductor  GenomeInfoDb         NaN         1.34.9         NaN   \n",
       "17 NaN  Bioconductor  GenomeInfoDb         NaN         1.34.9         NaN   \n",
       "18 NaN  Bioconductor  GenomeInfoDb         NaN         1.34.9         NaN   \n",
       "19 NaN  Bioconductor  GenomeInfoDb         NaN         1.34.9         NaN   \n",
       "20 NaN  Bioconductor  GenomeInfoDb         NaN         1.34.9         NaN   \n",
       "21 NaN  Bioconductor  GenomeInfoDb         NaN         1.34.9         NaN   \n",
       "22 NaN  Bioconductor  GenomeInfoDb         NaN         1.34.9         NaN   \n",
       "23 NaN  Bioconductor  GenomeInfoDb         NaN         1.34.9         NaN   \n",
       "24 NaN  Bioconductor  GenomeInfoDb         NaN         1.34.9         NaN   \n",
       "25 NaN  Bioconductor  GenomeInfoDb         NaN         1.34.9         NaN   \n",
       "\n",
       "     Dependency Name Dependency Platform Dependency Kind  Optional Dependency  \\\n",
       "16                 R        Bioconductor         depends                  NaN   \n",
       "17           methods        Bioconductor         depends                  NaN   \n",
       "18      BiocGenerics        Bioconductor         depends                  NaN   \n",
       "19         S4Vectors        Bioconductor         depends                  NaN   \n",
       "20           IRanges        Bioconductor         depends                  NaN   \n",
       "21             stats        Bioconductor         imports                  NaN   \n",
       "22            stats4        Bioconductor         imports                  NaN   \n",
       "23             utils        Bioconductor         imports                  NaN   \n",
       "24             RCurl        Bioconductor         imports                  NaN   \n",
       "25  GenomeInfoDbData        Bioconductor         imports                  NaN   \n",
       "\n",
       "    Dependency Requirements  Dependency Project ID  \n",
       "16                      NaN                    NaN  \n",
       "17                      NaN                    NaN  \n",
       "18                      NaN                    NaN  \n",
       "19                      NaN                    NaN  \n",
       "20                      NaN                    NaN  \n",
       "21                      NaN                    NaN  \n",
       "22                      NaN                    NaN  \n",
       "23                      NaN                    NaN  \n",
       "24                      NaN                    NaN  \n",
       "25                      NaN                    NaN  "
      ]
     },
     "execution_count": 6,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# Make a lil check\n",
    "bioc_df[bioc_df[\"Project Name\"] == \"GenomeInfoDb\"]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Name: GenomeInfoDb\n",
      "Description:\n",
      "Contains data and functions that define and allow translation between different chromosome sequence naming conventions (e.g., \"chr1\" versus \"1\"), including a function that attempts to place sequence names in their natural, rather than lexicographic, order.\n",
      "Version: 1.34.9\n",
      "Publication date: None\n",
      "Mantainer: HervÃ© PagÃ¨s <hpages.on.github@gmail.com>\n",
      "Authors:\n",
      "Author: Sonali Arora [aut], Martin Morgan [aut], Marc Carlson [aut], HervÃ© PagÃ¨s [aut, cre], Prisca Chidimma Maduka [ctb], Atuhurira Kirabo Kakopo [ctb], Emmanuel Chigozie Elendu [ctb]\n",
      "Dependencies:\n",
      "  - R (>= 4.0.0) type: depends\n",
      "  - methods, type: depends\n",
      "  - BiocGenerics, type: depends\n",
      "  - S4Vectors, type: depends\n",
      "  - IRanges, type: depends\n",
      "  - stats, type: imports\n",
      "  - stats4, type: imports\n",
      "  - utils, type: imports\n",
      "  - RCurl, type: imports\n",
      "  - GenomeInfoDbData, type: imports\n",
      "Requires compilation: None\n",
      "In CRAN: None\n",
      "In Bioconductor: True\n",
      "Licenses:\n",
      "Artistic-2.0\n",
      "Links:\n",
      "https://www.bioconductor.org/packages/release/bioc/html/GenomeInfoDb.html\n",
      "\n"
     ]
    }
   ],
   "source": [
    "bioc_package_list[3].dump()"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# <ins>**3 - Libraries.io web API**</ins>"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Libraries.io offers us an API through which we can obtain the data referred to the beginning of this notebook.\n",
    "\n",
    "Unfortunately, if we have no data of the packages in the repository, we cannot obtain any data, since the API requires specifying the name of the package for which we want to obtain information.\n",
    "\n",
    "We can get more information about the API at:\n",
    "\n",
    "- https://libraries.io/api"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "**Basic example of use of the API:**\n",
    "\n",
    "We get a list of 30 repositories available in *Libraries.io*"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "200\n",
      "[{\"name\":\"NPM\",\"project_count\":2881884,\"homepage\":\"https://www.npmjs.com\",\"color\":\"#f1e05a\",\"default_language\":\"JavaScript\"},{\"name\":\"Pypi\",\"project_count\":530299,\"homepage\":\"https://pypi.org/\",\"color\":\"#3572A5\",\"default_language\":\"Python\"},{\"name\":\"Maven\",\"project_count\":525139,\"homepage\":\"http://maven.org\",\"color\":\"#b07219\",\"default_language\":\"Java\"},{\"name\":\"Go\",\"project_count\":439688,\"homepage\":\"https://pkg.go.dev\",\"color\":\"#375eab\",\"default_language\":null},{\"name\":\"NuGet\",\"project_count\":419199,\"homepage\":\"https://www.nuget.org\",\"color\":\"#178600\",\"default_language\":\"C#\"},{\"name\":\"Packagist\",\"project_count\":373571,\"homepage\":\"https://packagist.org\",\"color\":\"#4F5D95\",\"default_language\":\"PHP\"},{\"name\":\"Rubygems\",\"project_count\":182775,\"homepage\":\"https://rubygems.org\",\"color\":\"#701516\",\"default_language\":\"Ruby\"},{\"name\":\"Cargo\",\"project_count\":106988,\"homepage\":\"https://crates.io\",\"color\":\"#dea584\",\"default_language\":\"Rust\"},{\"name\":\"CocoaPods\",\"project_count\":92180,\"homepage\":\"http://cocoapods.org/\",\"color\":\"#438eff\",\"default_language\":\"Objective-C\"},{\"name\":\"Bower\",\"project_count\":69218,\"homepage\":\"http://bower.io\",\"color\":\"#563d7c\",\"default_language\":\"CSS\"},{\"name\":\"CPAN\",\"project_count\":39881,\"homepage\":\"https://metacpan.org\",\"color\":\"#0298c3\",\"default_language\":\"Perl\"},{\"name\":\"Pub\",\"project_count\":39462,\"homepage\":\"https://pub.dartlang.org\",\"color\":\"#00B4AB\",\"default_language\":\"Dart\"},{\"name\":\"Clojars\",\"project_count\":24268,\"homepage\":\"https://clojars.org\",\"color\":\"#db5855\",\"default_language\":\"Clojure\"},{\"name\":\"CRAN\",\"project_count\":23758,\"homepage\":\"https://cran.r-project.org/\",\"color\":\"#198CE7\",\"default_language\":\"R\"},{\"name\":\"Conda\",\"project_count\":18701,\"homepage\":\"https://anaconda.org\",\"color\":\"#fff\",\"default_language\":null},{\"name\":\"Hackage\",\"project_count\":16985,\"homepage\":\"http://hackage.haskell.org\",\"color\":\"#29b544\",\"default_language\":null},{\"name\":\"Hex\",\"project_count\":14152,\"homepage\":\"https://hex.pm\",\"color\":\"#6e4a7e\",\"default_language\":\"Elixir\"},{\"name\":\"Meteor\",\"project_count\":13338,\"homepage\":\"https://atmospherejs.com\",\"color\":\"#f1e05a\",\"default_language\":\"JavaScript\"},{\"name\":\"Homebrew\",\"project_count\":7523,\"homepage\":\"http://brew.sh/\",\"color\":\"#555555\",\"default_language\":\"C\"},{\"name\":\"Puppet\",\"project_count\":6920,\"homepage\":\"https://forge.puppet.com\",\"color\":\"#302B6D\",\"default_language\":\"Puppet\"},{\"name\":\"Carthage\",\"project_count\":4617,\"homepage\":\"https://github.com/Carthage/Carthage\",\"color\":\"#ffac45\",\"default_language\":\"Swift\"},{\"name\":\"SwiftPM\",\"project_count\":4201,\"homepage\":\"https://developer.apple.com/swift/\",\"color\":\"#ffac45\",\"default_language\":\"Swift\"},{\"name\":\"Julia\",\"project_count\":3041,\"homepage\":\"http://pkg.julialang.org/\",\"color\":\"#a270ba\",\"default_language\":\"Julia\"},{\"name\":\"Elm\",\"project_count\":2773,\"homepage\":\"http://package.elm-lang.org/\",\"color\":\"#60B5CC\",\"default_language\":\"Elm\"},{\"name\":\"Dub\",\"project_count\":2561,\"homepage\":\"http://code.dlang.org\",\"color\":\"#ba595e\",\"default_language\":\"D\"},{\"name\":\"Racket\",\"project_count\":2413,\"homepage\":\"http://pkgs.racket-lang.org/\",\"color\":\"#375eab\",\"default_language\":null},{\"name\":\"Nimble\",\"project_count\":2132,\"homepage\":\"https://github.com/nim-lang/nimble\",\"color\":\"#37775b\",\"default_language\":\"Nim\"},{\"name\":\"Haxelib\",\"project_count\":1694,\"homepage\":\"https://lib.haxe.org\",\"color\":\"#df7900\",\"default_language\":\"Haxe\"},{\"name\":\"PureScript\",\"project_count\":685,\"homepage\":\"https://github.com/purescript/psc-package\",\"color\":\"#1D222D\",\"default_language\":\"PureScript\"},{\"name\":\"Alcatraz\",\"project_count\":462,\"homepage\":\"http://alcatraz.io\",\"color\":\"#438eff\",\"default_language\":\"Objective-C\"},{\"name\":\"Inqlude\",\"project_count\":225,\"homepage\":\"https://inqlude.org/\",\"color\":\"#f34b7d\",\"default_language\":\"C++\"}]\n"
     ]
    }
   ],
   "source": [
    "api_key = \"dcc8a237b15dac0e6334df54bd9573fb\"\n",
    "\n",
    "# make a request to the API\n",
    "response = requests.get(\"https://libraries.io/api/platforms?api_key=\" + api_key)\n",
    "print(response.status_code)\n",
    "print(response.text)"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "There is a Python wrapper for the API.\n",
    "\n",
    "- https://pypi.org/project/pybraries/\n",
    "- https://pybraries.readthedocs.io/en/latest/stubs/pybraries.search.Search.html"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Set up the environment variables using python\n",
    "import os\n",
    "os.environ['LIBRARIES_API_KEY'] = 'dcc8a237b15dac0e6334df54bd9573fb'\n",
    "\n",
    "# Import the libraries api wrapper\n",
    "from pybraries import Search   "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Repositories: \n",
      "NPM; Pypi; Maven; Go; NuGet; Packagist; Rubygems; Cargo; CocoaPods; Bower; CPAN; Pub; Clojars; CRAN; Conda; Hackage; Hex; Meteor; Homebrew; Puppet; Carthage; SwiftPM; Julia; Elm; Dub; Racket; Nimble; Haxelib; PureScript; Alcatraz; Inqlude; \n",
      "\n",
      "CRAN's A3 package data at libraries.io:\n",
      "dependent_repos_count: 0\n",
      "dependents_count: 0\n",
      "deprecation_reason: None\n",
      "description: Accurate, Adaptable, and Accessible Error Metrics for Predictive\n",
      "Models\n",
      "forks: 0\n",
      "homepage: None\n",
      "keywords: []\n",
      "language: None\n",
      "latest_download_url: https://cran.r-project.org/src/contrib/A3_1.0.0.tar.gz\n",
      "latest_release_number: 1.0.0\n",
      "latest_release_published_at: 2015-08-16T00:00:00.000Z\n",
      "latest_stable_release_number: 1.0.0\n",
      "latest_stable_release_published_at: 2015-08-16T00:00:00.000Z\n",
      "license_normalized: False\n",
      "licenses: GPL-2 | GPL-3 [expanded from: GPL (≥ 2)]\n",
      "name: A3\n",
      "normalized_licenses: ['CNRI-Python-GPL-Compatible', 'CNRI-Python-GPL-Compatible']\n",
      "package_manager_url: https://cran.r-project.org/package=A3\n",
      "platform: CRAN\n",
      "rank: 5\n",
      "repository_license: None\n",
      "repository_url: None\n",
      "stars: 0\n",
      "status: None\n",
      "versions: [{'number': '0.9.1', 'published_at': '2013-02-07T10:00:00.000Z', 'spdx_expression': None, 'original_license': None, 'researched_at': None, 'repository_sources': ['CRAN']}, {'number': '0.9.2', 'published_at': '2013-03-26T00:00:00.000Z', 'spdx_expression': None, 'original_license': None, 'researched_at': None, 'repository_sources': ['CRAN']}, {'number': '1.0.0', 'published_at': '2015-08-16T00:00:00.000Z', 'spdx_expression': None, 'original_license': None, 'researched_at': None, 'repository_sources': ['CRAN']}]\n"
     ]
    }
   ],
   "source": [
    "search = Search()\n",
    "repo_list = search.platforms() \n",
    "print(\"Repositories: \")\n",
    "for repo in repo_list:\n",
    "    print(repo['name'] + \"; \", end='')\n",
    "\n",
    "print(\"\\n\\nCRAN's A3 package data at libraries.io:\")\n",
    "A3_api = search.project(\"CRAN\", \"A3\")\n",
    "for key, value in A3_api.items():\n",
    "    print(key + \": \" + str(value))\n"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.10.6"
  },
  "orig_nbformat": 4,
  "vscode": {
   "interpreter": {
    "hash": "916dbcbb3f70747c44a77c7bcd40155683ae19c65e1c03b4aa3499c5328201f1"
   }
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
