{
 "cells": [
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Olivia-Finder introduction"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 0 - Previous requirements"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "**We make sure to have the necessary units installed**"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "%pip install -r ../requirements.txt"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "**Add the Library Route to Path**"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Add the path to the olivia_finder package\n",
    "import sys\n",
    "sys.path.append('../')"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 1 - DataSource"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "The class **DataSource** provides an interface for the obtaining of data from the different existing Packs managers.\n",
    "\n",
    "A datasource can implement classes:\n",
    "\n",
    "- **Scraper**, to obtain the data directly from the website of the package manager\n",
    "- **CSVNetwork**, to obtain the data from a CSV file"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "In this example we will instantize the desired implementation of data source class\n",
    "\n",
    "Below are some of its most relevant features"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Data obtaining via web scarping"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "In the first place we import the implementation of the data source we want, for this example we will use the Bioconductor Scraper"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "2023-03-20 21:10:54 [   DEBUG] Added SSLProxies to proxy builders (logger.py:140)\n",
      "2023-03-20 21:10:54 [   DEBUG] Added FreeProxyList to proxy builders (logger.py:140)\n",
      "2023-03-20 21:10:54 [   DEBUG] Added GeonodeProxy to proxy builders (logger.py:140)\n",
      "2023-03-20 21:10:54 [   DEBUG] Starting new HTTPS connection (1): www.sslproxies.org:443 (connectionpool.py:973)\n",
      "2023-03-20 21:10:54 [   DEBUG] https://www.sslproxies.org:443 \"GET / HTTP/1.1\" 200 None (connectionpool.py:452)\n",
      "2023-03-20 21:10:54 [   DEBUG] Found 100 proxies from SSLProxies (logger.py:140)\n",
      "2023-03-20 21:10:54 [   DEBUG] Starting new HTTPS connection (1): free-proxy-list.net:443 (connectionpool.py:973)\n",
      "2023-03-20 21:10:54 [   DEBUG] https://free-proxy-list.net:443 \"GET /anonymous-proxy.html HTTP/1.1\" 200 None (connectionpool.py:452)\n",
      "2023-03-20 21:10:54 [   DEBUG] Found 100 proxies from FreeProxyList (logger.py:140)\n",
      "2023-03-20 21:10:54 [   DEBUG] Starting new HTTPS connection (1): proxylist.geonode.com:443 (connectionpool.py:973)\n",
      "2023-03-20 21:11:00 [   DEBUG] https://proxylist.geonode.com:443 \"GET /api/proxy-list?limit=500&page=1&sort_by=lastChecked&sort_type=desc HTTP/1.1\" 200 None (connectionpool.py:452)\n",
      "2023-03-20 21:11:00 [   DEBUG] Found 500 proxies from GeonodeProxy (logger.py:140)\n",
      "2023-03-20 21:11:00 [   DEBUG] Proxies len: 685 (logger.py:140)\n",
      "2023-03-20 21:11:00 [   DEBUG] Proxy Handler initialized with 685 proxies (logger.py:140)\n",
      "2023-03-20 21:11:00 [   DEBUG] Useragents loaded from file: /home/dnllns/Documentos/repositorios/olivia-finder/olivia_finder/olivia_finder/myrequests/data/useragents.txt (logger.py:140)\n"
     ]
    }
   ],
   "source": [
    "from olivia_finder.data_source.scrapers.bioconductor import BiocScraper\n",
    "bioc_scraper_ds = BiocScraper()"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Show relevant information about the data source"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "{'name': 'Bioconductor', 'description': 'Scraper class implementation for the Bioconductor package network'}\n"
     ]
    }
   ],
   "source": [
    "print(bioc_scraper_ds.get_info())"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Get a list with the name of the packages obtained from this source\n",
    "\n",
    "Specifically, the Biic Scraper class gets the list of packages from the url:\n",
    "\n",
    "-   https://bioconductor.org/packages/release/BiocViews.html#___Software"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Each specific implementation of a Scraper must manage this process on its own."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "2023-03-20 21:11:44 [   DEBUG] Started executable: `geckodriver` in a child process with pid: 75041 (service.py:204)\n",
      "2023-03-20 21:11:44 [   DEBUG] POST http://localhost:46743/session {\"capabilities\": {\"firstMatch\": [{}], \"alwaysMatch\": {\"browserName\": \"firefox\", \"acceptInsecureCerts\": true, \"moz:debuggerAddress\": true, \"pageLoadStrategy\": \"normal\", \"moz:firefoxOptions\": {\"args\": [\"-headless\"]}}}} (remote_connection.py:303)\n",
      "2023-03-20 21:11:44 [   DEBUG] Starting new HTTP connection (1): localhost:46743 (connectionpool.py:227)\n",
      "2023-03-20 21:11:45 [   DEBUG] http://localhost:46743 \"POST /session HTTP/1.1\" 200 764 (connectionpool.py:452)\n",
      "2023-03-20 21:11:45 [   DEBUG] Remote response: status=200 | data={\"value\":{\"sessionId\":\"a5e86af6-7734-4cab-a830-816a0e7a134b\",\"capabilities\":{\"acceptInsecureCerts\":true,\"browserName\":\"firefox\",\"browserVersion\":\"111.0\",\"moz:accessibilityChecks\":false,\"moz:buildID\":\"20230310033340\",\"moz:debuggerAddress\":\"127.0.0.1:53717\",\"moz:geckodriverVersion\":\"0.32.2\",\"moz:headless\":true,\"moz:platformVersion\":\"5.19.0-35-generic\",\"moz:processID\":75163,\"moz:profile\":\"/tmp/rust_mozprofilece6gqO\",\"moz:shutdownTimeout\":60000,\"moz:useNonSpecCompliantPointerOrigin\":false,\"moz:webdriverClick\":true,\"moz:windowless\":false,\"pageLoadStrategy\":\"normal\",\"platformName\":\"linux\",\"proxy\":{},\"setWindowRect\":true,\"strictFileInteractability\":false,\"timeouts\":{\"implicit\":0,\"pageLoad\":300000,\"script\":30000},\"unhandledPromptBehavior\":\"dismiss and notify\"}}} | headers=HTTPHeaderDict({'content-type': 'application/json; charset=utf-8', 'cache-control': 'no-cache', 'content-length': '764', 'date': 'Mon, 20 Mar 2023 20:11:44 GMT'}) (remote_connection.py:319)\n",
      "2023-03-20 21:11:45 [   DEBUG] Finished Request (remote_connection.py:346)\n",
      "2023-03-20 21:11:45 [   DEBUG] POST http://localhost:46743/session/a5e86af6-7734-4cab-a830-816a0e7a134b/url {\"url\": \"https://www.bioconductor.org/packages/release/BiocViews.html#___Software\"} (remote_connection.py:303)\n",
      "2023-03-20 21:11:47 [   DEBUG] http://localhost:46743 \"POST /session/a5e86af6-7734-4cab-a830-816a0e7a134b/url HTTP/1.1\" 200 14 (connectionpool.py:452)\n",
      "2023-03-20 21:11:47 [   DEBUG] Remote response: status=200 | data={\"value\":null} | headers=HTTPHeaderDict({'content-type': 'application/json; charset=utf-8', 'cache-control': 'no-cache', 'content-length': '14', 'date': 'Mon, 20 Mar 2023 20:11:45 GMT'}) (remote_connection.py:319)\n",
      "2023-03-20 21:11:47 [   DEBUG] Finished Request (remote_connection.py:346)\n",
      "2023-03-20 21:11:47 [   DEBUG] POST http://localhost:46743/session/a5e86af6-7734-4cab-a830-816a0e7a134b/element {\"using\": \"css selector\", \"value\": \"[id=\\\"biocViews_package_table\\\"]\"} (remote_connection.py:303)\n",
      "2023-03-20 21:11:47 [   DEBUG] http://localhost:46743 \"POST /session/a5e86af6-7734-4cab-a830-816a0e7a134b/element HTTP/1.1\" 200 88 (connectionpool.py:452)\n",
      "2023-03-20 21:11:47 [   DEBUG] Remote response: status=200 | data={\"value\":{\"element-6066-11e4-a52e-4f735466cecf\":\"5534fd34-47c3-4380-b01d-a18941502dfa\"}} | headers=HTTPHeaderDict({'content-type': 'application/json; charset=utf-8', 'cache-control': 'no-cache', 'content-length': '88', 'date': 'Mon, 20 Mar 2023 20:11:47 GMT'}) (remote_connection.py:319)\n",
      "2023-03-20 21:11:47 [   DEBUG] Finished Request (remote_connection.py:346)\n",
      "2023-03-20 21:11:47 [   DEBUG] POST http://localhost:46743/session/a5e86af6-7734-4cab-a830-816a0e7a134b/execute/sync {\"script\": \"/* getAttribute */return (function(){return (function(){var h=this||self;function aa(a){return\\\"string\\\"==typeof a}function ba(a,b){a=a.split(\\\".\\\");var c=h;a[0]in c||\\\"undefined\\\"==typeof c.execScript||c.execScript(\\\"var \\\"+a[0]);for(var d;a.length&&(d=a.shift());)a.length||void 0===b?c[d]&&c[d]!==Object.prototype[d]?c=c[d]:c=c[d]={}:c[d]=b}\\nfunction ca(a){var b=typeof a;if(\\\"object\\\"==b)if(a){if(a instanceof Array)return\\\"array\\\";if(a instanceof Object)return b;var c=Object.prototype.toString.call(a);if(\\\"[object Window]\\\"==c)return\\\"object\\\";if(\\\"[object Array]\\\"==c||\\\"number\\\"==typeof a.length&&\\\"undefined\\\"!=typeof a.splice&&\\\"undefined\\\"!=typeof a.propertyIsEnumerable&&!a.propertyIsEnumerable(\\\"splice\\\"))return\\\"array\\\";if(\\\"[object Function]\\\"==c||\\\"undefined\\\"!=typeof a.call&&\\\"undefined\\\"!=typeof a.propertyIsEnumerable&&!a.propertyIsEnumerable(\\\"call\\\"))return\\\"function\\\"}else return\\\"null\\\";\\nelse if(\\\"function\\\"==b&&\\\"undefined\\\"==typeof a.call)return\\\"object\\\";return b}function da(a){var b=typeof a;return\\\"object\\\"==b&&null!=a||\\\"function\\\"==b}function ea(a,b,c){return a.call.apply(a.bind,arguments)}function fa(a,b,c){if(!a)throw Error();if(2<arguments.length){var d=Array.prototype.slice.call(arguments,2);return function(){var e=Array.prototype.slice.call(arguments);Array.prototype.unshift.apply(e,d);return a.apply(b,e)}}return function(){return a.apply(b,arguments)}}\\nfunction ha(a,b,c){Function.prototype.bind&&-1!=Function.prototype.bind.toString().indexOf(\\\"native code\\\")?ha=ea:ha=fa;return ha.apply(null,arguments)}function ia(a,b){var c=Array.prototype.slice.call(arguments,1);return function(){var d=c.slice();d.push.apply(d,arguments);return a.apply(this,d)}}function m(a,b){function c(){}c.prototype=b.prototype;a.H=b.prototype;a.prototype=new c;a.prototype.constructor=a}\\nfunction ja(a,b,c){var d=arguments.callee.caller;if(\\\"undefined\\\"!==typeof d.H){for(var e=Array(arguments.length-1),f=1;f<arguments.length;f++)e[f-1]=arguments[f];d.H.constructor.apply(a,e)}else{if(\\\"string\\\"!=typeof b&&\\\"symbol\\\"!=typeof b)throw Error(\\\"method names provided to goog.base must be a string or a symbol\\\");e=Array(arguments.length-2);for(f=2;f<arguments.length;f++)e[f-2]=arguments[f];f=!1;for(var g=a.constructor.prototype;g;g=Object.getPrototypeOf(g))if(g[b]===d)f=!0;else if(f){g[b].apply(a,\\ne);return}if(a[b]===d)a.constructor.prototype[b].apply(a,e);else throw Error(\\\"goog.base called from a method of one name to a method of a different name\\\");}};/*\\n\\n The MIT License\\n\\n Copyright (c) 2007 Cybozu Labs, Inc.\\n Copyright (c) 2012 Google Inc.\\n\\n Permission is hereby granted, free of charge, to any person obtaining a copy\\n of this software and associated documentation files (the \\\"Software\\\"), to\\n deal in the Software without restriction, including without limitation the\\n rights to use, copy, modify, merge, publish, distribute, sublicense, and/or\\n sell copies of the Software, and to permit persons to whom the Software is\\n furnished to do so, subject to the following conditions:\\n\\n The above copyright notice and this permission notice shall be included in\\n all copies or substantial portions of the Software.\\n\\n THE SOFTWARE IS PROVIDED \\\"AS IS\\\", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR\\n IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,\\n FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE\\n AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER\\n LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING\\n FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS\\n IN THE SOFTWARE.\\n*/\\nfunction ka(a,b,c){this.a=a;this.b=b||1;this.f=c||1};var la;var ma=Array.prototype.indexOf?function(a,b){return Array.prototype.indexOf.call(a,b,void 0)}:function(a,b){if(\\\"string\\\"===typeof a)return\\\"string\\\"!==typeof b||1!=b.length?-1:a.indexOf(b,0);for(var c=0;c<a.length;c++)if(c in a&&a[c]===b)return c;return-1},n=Array.prototype.forEach?function(a,b){Array.prototype.forEach.call(a,b,void 0)}:function(a,b){for(var c=a.length,d=\\\"string\\\"===typeof a?a.split(\\\"\\\"):a,e=0;e<c;e++)e in d&&b.call(void 0,d[e],e,a)},na=Array.prototype.filter?function(a,b){return Array.prototype.filter.call(a,\\nb,void 0)}:function(a,b){for(var c=a.length,d=[],e=0,f=\\\"string\\\"===typeof a?a.split(\\\"\\\"):a,g=0;g<c;g++)if(g in f){var k=f[g];b.call(void 0,k,g,a)&&(d[e++]=k)}return d},oa=Array.prototype.map?function(a,b){return Array.prototype.map.call(a,b,void 0)}:function(a,b){for(var c=a.length,d=Array(c),e=\\\"string\\\"===typeof a?a.split(\\\"\\\"):a,f=0;f<c;f++)f in e&&(d[f]=b.call(void 0,e[f],f,a));return d},pa=Array.prototype.reduce?function(a,b,c){return Array.prototype.reduce.call(a,b,c)}:function(a,b,c){var d=c;n(a,\\nfunction(e,f){d=b.call(void 0,d,e,f,a)});return d},qa=Array.prototype.some?function(a,b){return Array.prototype.some.call(a,b,void 0)}:function(a,b){for(var c=a.length,d=\\\"string\\\"===typeof a?a.split(\\\"\\\"):a,e=0;e<c;e++)if(e in d&&b.call(void 0,d[e],e,a))return!0;return!1};function ra(a,b){a:{for(var c=a.length,d=\\\"string\\\"===typeof a?a.split(\\\"\\\"):a,e=0;e<c;e++)if(e in d&&b.call(void 0,d[e],e,a)){b=e;break a}b=-1}return 0>b?null:\\\"string\\\"===typeof a?a.charAt(b):a[b]}\\nfunction sa(a){return Array.prototype.concat.apply([],arguments)}function ta(a,b,c){return 2>=arguments.length?Array.prototype.slice.call(a,b):Array.prototype.slice.call(a,b,c)};var ua=String.prototype.trim?function(a){return a.trim()}:function(a){return/^[\\\\s\\\\xa0]*([\\\\s\\\\S]*?)[\\\\s\\\\xa0]*$/.exec(a)[1]};\\nfunction va(a,b){var c=0;a=ua(String(a)).split(\\\".\\\");b=ua(String(b)).split(\\\".\\\");for(var d=Math.max(a.length,b.length),e=0;0==c&&e<d;e++){var f=a[e]||\\\"\\\",g=b[e]||\\\"\\\";do{f=/(\\\\d*)(\\\\D*)(.*)/.exec(f)||[\\\"\\\",\\\"\\\",\\\"\\\",\\\"\\\"];g=/(\\\\d*)(\\\\D*)(.*)/.exec(g)||[\\\"\\\",\\\"\\\",\\\"\\\",\\\"\\\"];if(0==f[0].length&&0==g[0].length)break;c=wa(0==f[1].length?0:parseInt(f[1],10),0==g[1].length?0:parseInt(g[1],10))||wa(0==f[2].length,0==g[2].length)||wa(f[2],g[2]);f=f[3];g=g[3]}while(0==c)}return c}function wa(a,b){return a<b?-1:a>b?1:0};var p;a:{var xa=h.navigator;if(xa){var ya=xa.userAgent;if(ya){p=ya;break a}}p=\\\"\\\"}function q(a){return-1!=p.indexOf(a)};function za(){return q(\\\"Firefox\\\")||q(\\\"FxiOS\\\")}function Aa(){return(q(\\\"Chrome\\\")||q(\\\"CriOS\\\"))&&!q(\\\"Edge\\\")};function Ba(){return q(\\\"iPhone\\\")&&!q(\\\"iPod\\\")&&!q(\\\"iPad\\\")};function Ca(a,b){var c=Da;return Object.prototype.hasOwnProperty.call(c,a)?c[a]:c[a]=b(a)};var Ea=q(\\\"Opera\\\"),r=q(\\\"Trident\\\")||q(\\\"MSIE\\\"),Fa=q(\\\"Edge\\\"),t=q(\\\"Gecko\\\")&&!(-1!=p.toLowerCase().indexOf(\\\"webkit\\\")&&!q(\\\"Edge\\\"))&&!(q(\\\"Trident\\\")||q(\\\"MSIE\\\"))&&!q(\\\"Edge\\\"),Ga=-1!=p.toLowerCase().indexOf(\\\"webkit\\\")&&!q(\\\"Edge\\\"),Ha=q(\\\"Macintosh\\\"),Ia=q(\\\"Windows\\\");function Ja(){var a=h.document;return a?a.documentMode:void 0}var Ka;\\na:{var La=\\\"\\\",Ma=function(){var a=p;if(t)return/rv:([^\\\\);]+)(\\\\)|;)/.exec(a);if(Fa)return/Edge\\\\/([\\\\d\\\\.]+)/.exec(a);if(r)return/\\\\b(?:MSIE|rv)[: ]([^\\\\);]+)(\\\\)|;)/.exec(a);if(Ga)return/WebKit\\\\/(\\\\S+)/.exec(a);if(Ea)return/(?:Version)[ \\\\/]?(\\\\S+)/.exec(a)}();Ma&&(La=Ma?Ma[1]:\\\"\\\");if(r){var Na=Ja();if(null!=Na&&Na>parseFloat(La)){Ka=String(Na);break a}}Ka=La}var Da={};function Oa(a){return Ca(a,function(){return 0<=va(Ka,a)})}var w;w=h.document&&r?Ja():void 0;var x=r&&!(9<=Number(w)),Pa=r&&!(8<=Number(w));function Qa(a,b,c,d){this.a=a;this.nodeName=c;this.nodeValue=d;this.nodeType=2;this.parentNode=this.ownerElement=b}function Sa(a,b){var c=Pa&&\\\"href\\\"==b.nodeName?a.getAttribute(b.nodeName,2):b.nodeValue;return new Qa(b,a,b.nodeName,c)};function Ta(a){this.b=a;this.a=0}function Ua(a){a=a.match(Va);for(var b=0;b<a.length;b++)Wa.test(a[b])&&a.splice(b,1);return new Ta(a)}var Va=/\\\\$?(?:(?![0-9-\\\\.])(?:\\\\*|[\\\\w-\\\\.]+):)?(?![0-9-\\\\.])(?:\\\\*|[\\\\w-\\\\.]+)|\\\\/\\\\/|\\\\.\\\\.|::|\\\\d+(?:\\\\.\\\\d*)?|\\\\.\\\\d+|\\\"[^\\\"]*\\\"|'[^']*'|[!<>]=|\\\\s+|./g,Wa=/^\\\\s/;function y(a,b){return a.b[a.a+(b||0)]}function z(a){return a.b[a.a++]}function Xa(a){return a.b.length<=a.a};function Ya(a){return a.scrollingElement?a.scrollingElement:Ga||\\\"CSS1Compat\\\"!=a.compatMode?a.body||a.documentElement:a.documentElement}function Za(a,b){if(!a||!b)return!1;if(a.contains&&1==b.nodeType)return a==b||a.contains(b);if(\\\"undefined\\\"!=typeof a.compareDocumentPosition)return a==b||!!(a.compareDocumentPosition(b)&16);for(;b&&a!=b;)b=b.parentNode;return b==a}\\nfunction $a(a,b){if(a==b)return 0;if(a.compareDocumentPosition)return a.compareDocumentPosition(b)&2?1:-1;if(r&&!(9<=Number(w))){if(9==a.nodeType)return-1;if(9==b.nodeType)return 1}if(\\\"sourceIndex\\\"in a||a.parentNode&&\\\"sourceIndex\\\"in a.parentNode){var c=1==a.nodeType,d=1==b.nodeType;if(c&&d)return a.sourceIndex-b.sourceIndex;var e=a.parentNode,f=b.parentNode;return e==f?ab(a,b):!c&&Za(e,b)?-1*bb(a,b):!d&&Za(f,a)?bb(b,a):(c?a.sourceIndex:e.sourceIndex)-(d?b.sourceIndex:f.sourceIndex)}d=A(a);c=d.createRange();\\nc.selectNode(a);c.collapse(!0);a=d.createRange();a.selectNode(b);a.collapse(!0);return c.compareBoundaryPoints(h.Range.START_TO_END,a)}function bb(a,b){var c=a.parentNode;if(c==b)return-1;for(;b.parentNode!=c;)b=b.parentNode;return ab(b,a)}function ab(a,b){for(;b=b.previousSibling;)if(b==a)return-1;return 1}function A(a){return 9==a.nodeType?a:a.ownerDocument||a.document}var cb={SCRIPT:1,STYLE:1,HEAD:1,IFRAME:1,OBJECT:1},db={IMG:\\\" \\\",BR:\\\"\\\\n\\\"};\\nfunction eb(a,b,c){if(!(a.nodeName in cb))if(3==a.nodeType)c?b.push(String(a.nodeValue).replace(/(\\\\r\\\\n|\\\\r|\\\\n)/g,\\\"\\\")):b.push(a.nodeValue);else if(a.nodeName in db)b.push(db[a.nodeName]);else for(a=a.firstChild;a;)eb(a,b,c),a=a.nextSibling}function fb(a){this.a=a||h.document||document}fb.prototype.getElementsByTagName=function(a,b){return(b||this.a).getElementsByTagName(String(a))};function B(a){var b=null,c=a.nodeType;1==c&&(b=a.textContent,b=void 0==b||null==b?a.innerText:b,b=void 0==b||null==b?\\\"\\\":b);if(\\\"string\\\"!=typeof b)if(x&&\\\"title\\\"==a.nodeName.toLowerCase()&&1==c)b=a.text;else if(9==c||1==c){a=9==c?a.documentElement:a.firstChild;c=0;var d=[];for(b=\\\"\\\";a;){do 1!=a.nodeType&&(b+=a.nodeValue),x&&\\\"title\\\"==a.nodeName.toLowerCase()&&(b+=a.text),d[c++]=a;while(a=a.firstChild);for(;c&&!(a=d[--c].nextSibling););}}else b=a.nodeValue;return b}\\nfunction C(a,b,c){if(null===b)return!0;try{if(!a.getAttribute)return!1}catch(d){return!1}Pa&&\\\"class\\\"==b&&(b=\\\"className\\\");return null==c?!!a.getAttribute(b):a.getAttribute(b,2)==c}function gb(a,b,c,d,e){return(x?hb:ib).call(null,a,b,aa(c)?c:null,aa(d)?d:null,e||new D)}\\nfunction hb(a,b,c,d,e){if(a instanceof jb||8==a.b||c&&null===a.b){var f=b.all;if(!f)return e;a=kb(a);if(\\\"*\\\"!=a&&(f=b.getElementsByTagName(a),!f))return e;if(c){for(var g=[],k=0;b=f[k++];)C(b,c,d)&&g.push(b);f=g}for(k=0;b=f[k++];)\\\"*\\\"==a&&\\\"!\\\"==b.tagName||e.add(b);return e}lb(a,b,c,d,e);return e}\\nfunction ib(a,b,c,d,e){b.getElementsByName&&d&&\\\"name\\\"==c&&!r?(b=b.getElementsByName(d),n(b,function(f){a.a(f)&&e.add(f)})):b.getElementsByClassName&&d&&\\\"class\\\"==c?(b=b.getElementsByClassName(d),n(b,function(f){f.className==d&&a.a(f)&&e.add(f)})):a instanceof E?lb(a,b,c,d,e):b.getElementsByTagName&&(b=b.getElementsByTagName(a.f()),n(b,function(f){C(f,c,d)&&e.add(f)}));return e}\\nfunction mb(a,b,c,d,e){var f;if((a instanceof jb||8==a.b||c&&null===a.b)&&(f=b.childNodes)){var g=kb(a);if(\\\"*\\\"!=g&&(f=na(f,function(k){return k.tagName&&k.tagName.toLowerCase()==g}),!f))return e;c&&(f=na(f,function(k){return C(k,c,d)}));n(f,function(k){\\\"*\\\"==g&&(\\\"!\\\"==k.tagName||\\\"*\\\"==g&&1!=k.nodeType)||e.add(k)});return e}return nb(a,b,c,d,e)}function nb(a,b,c,d,e){for(b=b.firstChild;b;b=b.nextSibling)C(b,c,d)&&a.a(b)&&e.add(b);return e}\\nfunction lb(a,b,c,d,e){for(b=b.firstChild;b;b=b.nextSibling)C(b,c,d)&&a.a(b)&&e.add(b),lb(a,b,c,d,e)}function kb(a){if(a instanceof E){if(8==a.b)return\\\"!\\\";if(null===a.b)return\\\"*\\\"}return a.f()};function D(){this.b=this.a=null;this.o=0}function ob(a){this.f=a;this.a=this.b=null}function pb(a,b){if(!a.a)return b;if(!b.a)return a;var c=a.a;b=b.a;for(var d=null,e,f=0;c&&b;){e=c.f;var g=b.f;e==g||e instanceof Qa&&g instanceof Qa&&e.a==g.a?(e=c,c=c.a,b=b.a):0<$a(c.f,b.f)?(e=b,b=b.a):(e=c,c=c.a);(e.b=d)?d.a=e:a.a=e;d=e;f++}for(e=c||b;e;)e.b=d,d=d.a=e,f++,e=e.a;a.b=d;a.o=f;return a}function qb(a,b){b=new ob(b);b.a=a.a;a.b?a.a.b=b:a.a=a.b=b;a.a=b;a.o++}\\nD.prototype.add=function(a){a=new ob(a);a.b=this.b;this.a?this.b.a=a:this.a=this.b=a;this.b=a;this.o++};function rb(a){return(a=a.a)?a.f:null}function sb(a){return(a=rb(a))?B(a):\\\"\\\"}function F(a,b){return new tb(a,!!b)}function tb(a,b){this.f=a;this.b=(this.v=b)?a.b:a.a;this.a=null}function G(a){var b=a.b;if(null==b)return null;var c=a.a=b;a.b=a.v?b.b:b.a;return c.f};function H(a){this.l=a;this.b=this.i=!1;this.f=null}function I(a){return\\\"\\\\n  \\\"+a.toString().split(\\\"\\\\n\\\").join(\\\"\\\\n  \\\")}function ub(a,b){a.i=b}function vb(a,b){a.b=b}function J(a,b){a=a.a(b);return a instanceof D?+sb(a):+a}function L(a,b){a=a.a(b);return a instanceof D?sb(a):\\\"\\\"+a}function wb(a,b){a=a.a(b);return a instanceof D?!!a.o:!!a};function xb(a,b,c){H.call(this,a.l);this.c=a;this.j=b;this.u=c;this.i=b.i||c.i;this.b=b.b||c.b;this.c==yb&&(c.b||c.i||4==c.l||0==c.l||!b.f?b.b||b.i||4==b.l||0==b.l||!c.f||(this.f={name:c.f.name,A:b}):this.f={name:b.f.name,A:c})}m(xb,H);\\nfunction zb(a,b,c,d,e){b=b.a(d);c=c.a(d);var f;if(b instanceof D&&c instanceof D){b=F(b);for(d=G(b);d;d=G(b))for(e=F(c),f=G(e);f;f=G(e))if(a(B(d),B(f)))return!0;return!1}if(b instanceof D||c instanceof D){b instanceof D?(e=b,d=c):(e=c,d=b);f=F(e);for(var g=typeof d,k=G(f);k;k=G(f)){switch(g){case \\\"number\\\":k=+B(k);break;case \\\"boolean\\\":k=!!B(k);break;case \\\"string\\\":k=B(k);break;default:throw Error(\\\"Illegal primitive type for comparison.\\\");}if(e==b&&a(k,d)||e==c&&a(d,k))return!0}return!1}return e?\\\"boolean\\\"==\\ntypeof b||\\\"boolean\\\"==typeof c?a(!!b,!!c):\\\"number\\\"==typeof b||\\\"number\\\"==typeof c?a(+b,+c):a(b,c):a(+b,+c)}xb.prototype.a=function(a){return this.c.s(this.j,this.u,a)};xb.prototype.toString=function(){var a=\\\"Binary Expression: \\\"+this.c;a+=I(this.j);return a+=I(this.u)};function Ab(a,b,c,d){this.L=a;this.G=b;this.l=c;this.s=d}Ab.prototype.toString=function(){return this.L};var Bb={};\\nfunction M(a,b,c,d){if(Bb.hasOwnProperty(a))throw Error(\\\"Binary operator already created: \\\"+a);a=new Ab(a,b,c,d);return Bb[a.toString()]=a}M(\\\"div\\\",6,1,function(a,b,c){return J(a,c)/J(b,c)});M(\\\"mod\\\",6,1,function(a,b,c){return J(a,c)%J(b,c)});M(\\\"*\\\",6,1,function(a,b,c){return J(a,c)*J(b,c)});M(\\\"+\\\",5,1,function(a,b,c){return J(a,c)+J(b,c)});M(\\\"-\\\",5,1,function(a,b,c){return J(a,c)-J(b,c)});M(\\\"<\\\",4,2,function(a,b,c){return zb(function(d,e){return d<e},a,b,c)});\\nM(\\\">\\\",4,2,function(a,b,c){return zb(function(d,e){return d>e},a,b,c)});M(\\\"<=\\\",4,2,function(a,b,c){return zb(function(d,e){return d<=e},a,b,c)});M(\\\">=\\\",4,2,function(a,b,c){return zb(function(d,e){return d>=e},a,b,c)});var yb=M(\\\"=\\\",3,2,function(a,b,c){return zb(function(d,e){return d==e},a,b,c,!0)});M(\\\"!=\\\",3,2,function(a,b,c){return zb(function(d,e){return d!=e},a,b,c,!0)});M(\\\"and\\\",2,2,function(a,b,c){return wb(a,c)&&wb(b,c)});M(\\\"or\\\",1,2,function(a,b,c){return wb(a,c)||wb(b,c)});function Cb(a,b){if(b.a.length&&4!=a.l)throw Error(\\\"Primary expression must evaluate to nodeset if filter has predicate(s).\\\");H.call(this,a.l);this.c=a;this.j=b;this.i=a.i;this.b=a.b}m(Cb,H);Cb.prototype.a=function(a){a=this.c.a(a);return Db(this.j,a)};Cb.prototype.toString=function(){var a=\\\"Filter:\\\"+I(this.c);return a+=I(this.j)};function Eb(a,b){if(b.length<a.F)throw Error(\\\"Function \\\"+a.m+\\\" expects at least\\\"+a.F+\\\" arguments, \\\"+b.length+\\\" given\\\");if(null!==a.D&&b.length>a.D)throw Error(\\\"Function \\\"+a.m+\\\" expects at most \\\"+a.D+\\\" arguments, \\\"+b.length+\\\" given\\\");a.K&&n(b,function(c,d){if(4!=c.l)throw Error(\\\"Argument \\\"+d+\\\" to function \\\"+a.m+\\\" is not of type Nodeset: \\\"+c);});H.call(this,a.l);this.B=a;this.c=b;ub(this,a.i||qa(b,function(c){return c.i}));vb(this,a.J&&!b.length||a.I&&!!b.length||qa(b,function(c){return c.b}))}\\nm(Eb,H);Eb.prototype.a=function(a){return this.B.s.apply(null,sa(a,this.c))};Eb.prototype.toString=function(){var a=\\\"Function: \\\"+this.B;if(this.c.length){var b=pa(this.c,function(c,d){return c+I(d)},\\\"Arguments:\\\");a+=I(b)}return a};function Fb(a,b,c,d,e,f,g,k){this.m=a;this.l=b;this.i=c;this.J=d;this.I=!1;this.s=e;this.F=f;this.D=void 0!==g?g:f;this.K=!!k}Fb.prototype.toString=function(){return this.m};var Gb={};\\nfunction N(a,b,c,d,e,f,g,k){if(Gb.hasOwnProperty(a))throw Error(\\\"Function already created: \\\"+a+\\\".\\\");Gb[a]=new Fb(a,b,c,d,e,f,g,k)}N(\\\"boolean\\\",2,!1,!1,function(a,b){return wb(b,a)},1);N(\\\"ceiling\\\",1,!1,!1,function(a,b){return Math.ceil(J(b,a))},1);N(\\\"concat\\\",3,!1,!1,function(a,b){return pa(ta(arguments,1),function(c,d){return c+L(d,a)},\\\"\\\")},2,null);N(\\\"contains\\\",2,!1,!1,function(a,b,c){b=L(b,a);a=L(c,a);return-1!=b.indexOf(a)},2);N(\\\"count\\\",1,!1,!1,function(a,b){return b.a(a).o},1,1,!0);\\nN(\\\"false\\\",2,!1,!1,function(){return!1},0);N(\\\"floor\\\",1,!1,!1,function(a,b){return Math.floor(J(b,a))},1);N(\\\"id\\\",4,!1,!1,function(a,b){function c(k){if(x){var l=e.all[k];if(l){if(l.nodeType&&k==l.id)return l;if(l.length)return ra(l,function(u){return k==u.id})}return null}return e.getElementById(k)}var d=a.a,e=9==d.nodeType?d:d.ownerDocument;a=L(b,a).split(/\\\\s+/);var f=[];n(a,function(k){k=c(k);!k||0<=ma(f,k)||f.push(k)});f.sort($a);var g=new D;n(f,function(k){g.add(k)});return g},1);\\nN(\\\"lang\\\",2,!1,!1,function(){return!1},1);N(\\\"last\\\",1,!0,!1,function(a){if(1!=arguments.length)throw Error(\\\"Function last expects ()\\\");return a.f},0);N(\\\"local-name\\\",3,!1,!0,function(a,b){return(a=b?rb(b.a(a)):a.a)?a.localName||a.nodeName.toLowerCase():\\\"\\\"},0,1,!0);N(\\\"name\\\",3,!1,!0,function(a,b){return(a=b?rb(b.a(a)):a.a)?a.nodeName.toLowerCase():\\\"\\\"},0,1,!0);N(\\\"namespace-uri\\\",3,!0,!1,function(){return\\\"\\\"},0,1,!0);\\nN(\\\"normalize-space\\\",3,!1,!0,function(a,b){return(b?L(b,a):B(a.a)).replace(/[\\\\s\\\\xa0]+/g,\\\" \\\").replace(/^\\\\s+|\\\\s+$/g,\\\"\\\")},0,1);N(\\\"not\\\",2,!1,!1,function(a,b){return!wb(b,a)},1);N(\\\"number\\\",1,!1,!0,function(a,b){return b?J(b,a):+B(a.a)},0,1);N(\\\"position\\\",1,!0,!1,function(a){return a.b},0);N(\\\"round\\\",1,!1,!1,function(a,b){return Math.round(J(b,a))},1);N(\\\"starts-with\\\",2,!1,!1,function(a,b,c){b=L(b,a);a=L(c,a);return 0==b.lastIndexOf(a,0)},2);N(\\\"string\\\",3,!1,!0,function(a,b){return b?L(b,a):B(a.a)},0,1);\\nN(\\\"string-length\\\",1,!1,!0,function(a,b){return(b?L(b,a):B(a.a)).length},0,1);N(\\\"substring\\\",3,!1,!1,function(a,b,c,d){c=J(c,a);if(isNaN(c)||Infinity==c||-Infinity==c)return\\\"\\\";d=d?J(d,a):Infinity;if(isNaN(d)||-Infinity===d)return\\\"\\\";c=Math.round(c)-1;var e=Math.max(c,0);a=L(b,a);return Infinity==d?a.substring(e):a.substring(e,c+Math.round(d))},2,3);N(\\\"substring-after\\\",3,!1,!1,function(a,b,c){b=L(b,a);a=L(c,a);c=b.indexOf(a);return-1==c?\\\"\\\":b.substring(c+a.length)},2);\\nN(\\\"substring-before\\\",3,!1,!1,function(a,b,c){b=L(b,a);a=L(c,a);a=b.indexOf(a);return-1==a?\\\"\\\":b.substring(0,a)},2);N(\\\"sum\\\",1,!1,!1,function(a,b){a=F(b.a(a));b=0;for(var c=G(a);c;c=G(a))b+=+B(c);return b},1,1,!0);N(\\\"translate\\\",3,!1,!1,function(a,b,c,d){b=L(b,a);c=L(c,a);var e=L(d,a);a={};for(d=0;d<c.length;d++){var f=c.charAt(d);f in a||(a[f]=e.charAt(d))}c=\\\"\\\";for(d=0;d<b.length;d++)f=b.charAt(d),c+=f in a?a[f]:f;return c},3);N(\\\"true\\\",2,!1,!1,function(){return!0},0);function E(a,b){this.j=a;this.c=void 0!==b?b:null;this.b=null;switch(a){case \\\"comment\\\":this.b=8;break;case \\\"text\\\":this.b=3;break;case \\\"processing-instruction\\\":this.b=7;break;case \\\"node\\\":break;default:throw Error(\\\"Unexpected argument\\\");}}function Hb(a){return\\\"comment\\\"==a||\\\"text\\\"==a||\\\"processing-instruction\\\"==a||\\\"node\\\"==a}E.prototype.a=function(a){return null===this.b||this.b==a.nodeType};E.prototype.f=function(){return this.j};\\nE.prototype.toString=function(){var a=\\\"Kind Test: \\\"+this.j;null===this.c||(a+=I(this.c));return a};function Ib(a){H.call(this,3);this.c=a.substring(1,a.length-1)}m(Ib,H);Ib.prototype.a=function(){return this.c};Ib.prototype.toString=function(){return\\\"Literal: \\\"+this.c};function jb(a,b){this.m=a.toLowerCase();a=\\\"*\\\"==this.m?\\\"*\\\":\\\"http://www.w3.org/1999/xhtml\\\";this.c=b?b.toLowerCase():a}jb.prototype.a=function(a){var b=a.nodeType;if(1!=b&&2!=b)return!1;b=void 0!==a.localName?a.localName:a.nodeName;return\\\"*\\\"!=this.m&&this.m!=b.toLowerCase()?!1:\\\"*\\\"==this.c?!0:this.c==(a.namespaceURI?a.namespaceURI.toLowerCase():\\\"http://www.w3.org/1999/xhtml\\\")};jb.prototype.f=function(){return this.m};\\njb.prototype.toString=function(){return\\\"Name Test: \\\"+(\\\"http://www.w3.org/1999/xhtml\\\"==this.c?\\\"\\\":this.c+\\\":\\\")+this.m};function Jb(a){H.call(this,1);this.c=a}m(Jb,H);Jb.prototype.a=function(){return this.c};Jb.prototype.toString=function(){return\\\"Number: \\\"+this.c};function Kb(a,b){H.call(this,a.l);this.j=a;this.c=b;this.i=a.i;this.b=a.b;1==this.c.length&&(a=this.c[0],a.C||a.c!=Lb||(a=a.u,\\\"*\\\"!=a.f()&&(this.f={name:a.f(),A:null})))}m(Kb,H);function Mb(){H.call(this,4)}m(Mb,H);Mb.prototype.a=function(a){var b=new D;a=a.a;9==a.nodeType?b.add(a):b.add(a.ownerDocument);return b};Mb.prototype.toString=function(){return\\\"Root Helper Expression\\\"};function Nb(){H.call(this,4)}m(Nb,H);Nb.prototype.a=function(a){var b=new D;b.add(a.a);return b};Nb.prototype.toString=function(){return\\\"Context Helper Expression\\\"};\\nfunction Ob(a){return\\\"/\\\"==a||\\\"//\\\"==a}Kb.prototype.a=function(a){var b=this.j.a(a);if(!(b instanceof D))throw Error(\\\"Filter expression must evaluate to nodeset.\\\");a=this.c;for(var c=0,d=a.length;c<d&&b.o;c++){var e=a[c],f=F(b,e.c.v);if(e.i||e.c!=Pb)if(e.i||e.c!=Qb){var g=G(f);for(b=e.a(new ka(g));null!=(g=G(f));)g=e.a(new ka(g)),b=pb(b,g)}else g=G(f),b=e.a(new ka(g));else{for(g=G(f);(b=G(f))&&(!g.contains||g.contains(b))&&b.compareDocumentPosition(g)&8;g=b);b=e.a(new ka(g))}}return b};\\nKb.prototype.toString=function(){var a=\\\"Path Expression:\\\"+I(this.j);if(this.c.length){var b=pa(this.c,function(c,d){return c+I(d)},\\\"Steps:\\\");a+=I(b)}return a};function Rb(a,b){this.a=a;this.v=!!b}\\nfunction Db(a,b,c){for(c=c||0;c<a.a.length;c++)for(var d=a.a[c],e=F(b),f=b.o,g,k=0;g=G(e);k++){var l=a.v?f-k:k+1;g=d.a(new ka(g,l,f));if(\\\"number\\\"==typeof g)l=l==g;else if(\\\"string\\\"==typeof g||\\\"boolean\\\"==typeof g)l=!!g;else if(g instanceof D)l=0<g.o;else throw Error(\\\"Predicate.evaluate returned an unexpected type.\\\");if(!l){l=e;g=l.f;var u=l.a;if(!u)throw Error(\\\"Next must be called at least once before remove.\\\");var K=u.b;u=u.a;K?K.a=u:g.a=u;u?u.b=K:g.b=K;g.o--;l.a=null}}return b}\\nRb.prototype.toString=function(){return pa(this.a,function(a,b){return a+I(b)},\\\"Predicates:\\\")};function O(a,b,c,d){H.call(this,4);this.c=a;this.u=b;this.j=c||new Rb([]);this.C=!!d;b=this.j;b=0<b.a.length?b.a[0].f:null;a.M&&b&&(a=b.name,a=x?a.toLowerCase():a,this.f={name:a,A:b.A});a:{a=this.j;for(b=0;b<a.a.length;b++)if(c=a.a[b],c.i||1==c.l||0==c.l){a=!0;break a}a=!1}this.i=a}m(O,H);\\nO.prototype.a=function(a){var b=a.a,c=this.f,d=null,e=null,f=0;c&&(d=c.name,e=c.A?L(c.A,a):null,f=1);if(this.C)if(this.i||this.c!=Sb)if(b=F((new O(Tb,new E(\\\"node\\\"))).a(a)),c=G(b))for(a=this.s(c,d,e,f);null!=(c=G(b));)a=pb(a,this.s(c,d,e,f));else a=new D;else a=gb(this.u,b,d,e),a=Db(this.j,a,f);else a=this.s(a.a,d,e,f);return a};O.prototype.s=function(a,b,c,d){a=this.c.B(this.u,a,b,c);return a=Db(this.j,a,d)};\\nO.prototype.toString=function(){var a=\\\"Step:\\\"+I(\\\"Operator: \\\"+(this.C?\\\"//\\\":\\\"/\\\"));this.c.m&&(a+=I(\\\"Axis: \\\"+this.c));a+=I(this.u);if(this.j.a.length){var b=pa(this.j.a,function(c,d){return c+I(d)},\\\"Predicates:\\\");a+=I(b)}return a};function Ub(a,b,c,d){this.m=a;this.B=b;this.v=c;this.M=d}Ub.prototype.toString=function(){return this.m};var Vb={};function P(a,b,c,d){if(Vb.hasOwnProperty(a))throw Error(\\\"Axis already created: \\\"+a);b=new Ub(a,b,c,!!d);return Vb[a]=b}\\nP(\\\"ancestor\\\",function(a,b){for(var c=new D;b=b.parentNode;)a.a(b)&&qb(c,b);return c},!0);P(\\\"ancestor-or-self\\\",function(a,b){var c=new D;do a.a(b)&&qb(c,b);while(b=b.parentNode);return c},!0);\\nvar Lb=P(\\\"attribute\\\",function(a,b){var c=new D,d=a.f();if(\\\"style\\\"==d&&x&&b.style)return c.add(new Qa(b.style,b,\\\"style\\\",b.style.cssText)),c;var e=b.attributes;if(e)if(a instanceof E&&null===a.b||\\\"*\\\"==d)for(a=0;d=e[a];a++)x?d.nodeValue&&c.add(Sa(b,d)):c.add(d);else(d=e.getNamedItem(d))&&(x?d.nodeValue&&c.add(Sa(b,d)):c.add(d));return c},!1),Sb=P(\\\"child\\\",function(a,b,c,d,e){return(x?mb:nb).call(null,a,b,aa(c)?c:null,aa(d)?d:null,e||new D)},!1,!0);P(\\\"descendant\\\",gb,!1,!0);\\nvar Tb=P(\\\"descendant-or-self\\\",function(a,b,c,d){var e=new D;C(b,c,d)&&a.a(b)&&e.add(b);return gb(a,b,c,d,e)},!1,!0),Pb=P(\\\"following\\\",function(a,b,c,d){var e=new D;do for(var f=b;f=f.nextSibling;)C(f,c,d)&&a.a(f)&&e.add(f),e=gb(a,f,c,d,e);while(b=b.parentNode);return e},!1,!0);P(\\\"following-sibling\\\",function(a,b){for(var c=new D;b=b.nextSibling;)a.a(b)&&c.add(b);return c},!1);P(\\\"namespace\\\",function(){return new D},!1);\\nvar Wb=P(\\\"parent\\\",function(a,b){var c=new D;if(9==b.nodeType)return c;if(2==b.nodeType)return c.add(b.ownerElement),c;b=b.parentNode;a.a(b)&&c.add(b);return c},!1),Qb=P(\\\"preceding\\\",function(a,b,c,d){var e=new D,f=[];do f.unshift(b);while(b=b.parentNode);for(var g=1,k=f.length;g<k;g++){var l=[];for(b=f[g];b=b.previousSibling;)l.unshift(b);for(var u=0,K=l.length;u<K;u++)b=l[u],C(b,c,d)&&a.a(b)&&e.add(b),e=gb(a,b,c,d,e)}return e},!0,!0);\\nP(\\\"preceding-sibling\\\",function(a,b){for(var c=new D;b=b.previousSibling;)a.a(b)&&qb(c,b);return c},!0);var Xb=P(\\\"self\\\",function(a,b){var c=new D;a.a(b)&&c.add(b);return c},!1);function Yb(a){H.call(this,1);this.c=a;this.i=a.i;this.b=a.b}m(Yb,H);Yb.prototype.a=function(a){return-J(this.c,a)};Yb.prototype.toString=function(){return\\\"Unary Expression: -\\\"+I(this.c)};function Zb(a){H.call(this,4);this.c=a;ub(this,qa(this.c,function(b){return b.i}));vb(this,qa(this.c,function(b){return b.b}))}m(Zb,H);Zb.prototype.a=function(a){var b=new D;n(this.c,function(c){c=c.a(a);if(!(c instanceof D))throw Error(\\\"Path expression must evaluate to NodeSet.\\\");b=pb(b,c)});return b};Zb.prototype.toString=function(){return pa(this.c,function(a,b){return a+I(b)},\\\"Union Expression:\\\")};function $b(a,b){this.a=a;this.b=b}function ac(a){for(var b,c=[];;){Q(a,\\\"Missing right hand side of binary expression.\\\");b=bc(a);var d=z(a.a);if(!d)break;var e=(d=Bb[d]||null)&&d.G;if(!e){a.a.a--;break}for(;c.length&&e<=c[c.length-1].G;)b=new xb(c.pop(),c.pop(),b);c.push(b,d)}for(;c.length;)b=new xb(c.pop(),c.pop(),b);return b}function Q(a,b){if(Xa(a.a))throw Error(b);}function cc(a,b){a=z(a.a);if(a!=b)throw Error(\\\"Bad token, expected: \\\"+b+\\\" got: \\\"+a);}\\nfunction dc(a){a=z(a.a);if(\\\")\\\"!=a)throw Error(\\\"Bad token: \\\"+a);}function ec(a){a=z(a.a);if(2>a.length)throw Error(\\\"Unclosed literal string\\\");return new Ib(a)}\\nfunction fc(a){var b=[];if(Ob(y(a.a))){var c=z(a.a);var d=y(a.a);if(\\\"/\\\"==c&&(Xa(a.a)||\\\".\\\"!=d&&\\\"..\\\"!=d&&\\\"@\\\"!=d&&\\\"*\\\"!=d&&!/(?![0-9])[\\\\w]/.test(d)))return new Mb;d=new Mb;Q(a,\\\"Missing next location step.\\\");c=gc(a,c);b.push(c)}else{a:{c=y(a.a);d=c.charAt(0);switch(d){case \\\"$\\\":throw Error(\\\"Variable reference not allowed in HTML XPath\\\");case \\\"(\\\":z(a.a);c=ac(a);Q(a,'unclosed \\\"(\\\"');cc(a,\\\")\\\");break;case '\\\"':case \\\"'\\\":c=ec(a);break;default:if(isNaN(+c))if(!Hb(c)&&/(?![0-9])[\\\\w]/.test(d)&&\\\"(\\\"==y(a.a,1)){c=z(a.a);\\nc=Gb[c]||null;z(a.a);for(d=[];\\\")\\\"!=y(a.a);){Q(a,\\\"Missing function argument list.\\\");d.push(ac(a));if(\\\",\\\"!=y(a.a))break;z(a.a)}Q(a,\\\"Unclosed function argument list.\\\");dc(a);c=new Eb(c,d)}else{c=null;break a}else c=new Jb(+z(a.a))}\\\"[\\\"==y(a.a)&&(d=new Rb(hc(a)),c=new Cb(c,d))}if(c)if(Ob(y(a.a)))d=c;else return c;else c=gc(a,\\\"/\\\"),d=new Nb,b.push(c)}for(;Ob(y(a.a));)c=z(a.a),Q(a,\\\"Missing next location step.\\\"),c=gc(a,c),b.push(c);return new Kb(d,b)}\\nfunction gc(a,b){if(\\\"/\\\"!=b&&\\\"//\\\"!=b)throw Error('Step op should be \\\"/\\\" or \\\"//\\\"');if(\\\".\\\"==y(a.a)){var c=new O(Xb,new E(\\\"node\\\"));z(a.a);return c}if(\\\"..\\\"==y(a.a))return c=new O(Wb,new E(\\\"node\\\")),z(a.a),c;if(\\\"@\\\"==y(a.a)){var d=Lb;z(a.a);Q(a,\\\"Missing attribute name\\\")}else if(\\\"::\\\"==y(a.a,1)){if(!/(?![0-9])[\\\\w]/.test(y(a.a).charAt(0)))throw Error(\\\"Bad token: \\\"+z(a.a));var e=z(a.a);d=Vb[e]||null;if(!d)throw Error(\\\"No axis with name: \\\"+e);z(a.a);Q(a,\\\"Missing node name\\\")}else d=Sb;e=y(a.a);if(/(?![0-9])[\\\\w\\\\*]/.test(e.charAt(0)))if(\\\"(\\\"==\\ny(a.a,1)){if(!Hb(e))throw Error(\\\"Invalid node type: \\\"+e);e=z(a.a);if(!Hb(e))throw Error(\\\"Invalid type name: \\\"+e);cc(a,\\\"(\\\");Q(a,\\\"Bad nodetype\\\");var f=y(a.a).charAt(0),g=null;if('\\\"'==f||\\\"'\\\"==f)g=ec(a);Q(a,\\\"Bad nodetype\\\");dc(a);e=new E(e,g)}else if(e=z(a.a),f=e.indexOf(\\\":\\\"),-1==f)e=new jb(e);else{g=e.substring(0,f);if(\\\"*\\\"==g)var k=\\\"*\\\";else if(k=a.b(g),!k)throw Error(\\\"Namespace prefix not declared: \\\"+g);e=e.substr(f+1);e=new jb(e,k)}else throw Error(\\\"Bad token: \\\"+z(a.a));a=new Rb(hc(a),d.v);return c||\\nnew O(d,e,a,\\\"//\\\"==b)}function hc(a){for(var b=[];\\\"[\\\"==y(a.a);){z(a.a);Q(a,\\\"Missing predicate expression.\\\");var c=ac(a);b.push(c);Q(a,\\\"Unclosed predicate expression.\\\");cc(a,\\\"]\\\")}return b}function bc(a){if(\\\"-\\\"==y(a.a))return z(a.a),new Yb(bc(a));var b=fc(a);if(\\\"|\\\"!=y(a.a))a=b;else{for(b=[b];\\\"|\\\"==z(a.a);)Q(a,\\\"Missing next union location path.\\\"),b.push(fc(a));a.a.a--;a=new Zb(b)}return a};function ic(a){switch(a.nodeType){case 1:return ia(jc,a);case 9:return ic(a.documentElement);case 11:case 10:case 6:case 12:return kc;default:return a.parentNode?ic(a.parentNode):kc}}function kc(){return null}function jc(a,b){if(a.prefix==b)return a.namespaceURI||\\\"http://www.w3.org/1999/xhtml\\\";var c=a.getAttributeNode(\\\"xmlns:\\\"+b);return c&&c.specified?c.value||null:a.parentNode&&9!=a.parentNode.nodeType?jc(a.parentNode,b):null};function lc(a,b){if(!a.length)throw Error(\\\"Empty XPath expression.\\\");a=Ua(a);if(Xa(a))throw Error(\\\"Invalid XPath expression.\\\");b?\\\"function\\\"==ca(b)||(b=ha(b.lookupNamespaceURI,b)):b=function(){return null};var c=ac(new $b(a,b));if(!Xa(a))throw Error(\\\"Bad token: \\\"+z(a));this.evaluate=function(d,e){d=c.a(new ka(d));return new T(d,e)}}\\nfunction T(a,b){if(0==b)if(a instanceof D)b=4;else if(\\\"string\\\"==typeof a)b=2;else if(\\\"number\\\"==typeof a)b=1;else if(\\\"boolean\\\"==typeof a)b=3;else throw Error(\\\"Unexpected evaluation result.\\\");if(2!=b&&1!=b&&3!=b&&!(a instanceof D))throw Error(\\\"value could not be converted to the specified type\\\");this.resultType=b;switch(b){case 2:this.stringValue=a instanceof D?sb(a):\\\"\\\"+a;break;case 1:this.numberValue=a instanceof D?+sb(a):+a;break;case 3:this.booleanValue=a instanceof D?0<a.o:!!a;break;case 4:case 5:case 6:case 7:var c=\\nF(a);var d=[];for(var e=G(c);e;e=G(c))d.push(e instanceof Qa?e.a:e);this.snapshotLength=a.o;this.invalidIteratorState=!1;break;case 8:case 9:a=rb(a);this.singleNodeValue=a instanceof Qa?a.a:a;break;default:throw Error(\\\"Unknown XPathResult type.\\\");}var f=0;this.iterateNext=function(){if(4!=b&&5!=b)throw Error(\\\"iterateNext called with wrong result type\\\");return f>=d.length?null:d[f++]};this.snapshotItem=function(g){if(6!=b&&7!=b)throw Error(\\\"snapshotItem called with wrong result type\\\");return g>=d.length||\\n0>g?null:d[g]}}T.ANY_TYPE=0;T.NUMBER_TYPE=1;T.STRING_TYPE=2;T.BOOLEAN_TYPE=3;T.UNORDERED_NODE_ITERATOR_TYPE=4;T.ORDERED_NODE_ITERATOR_TYPE=5;T.UNORDERED_NODE_SNAPSHOT_TYPE=6;T.ORDERED_NODE_SNAPSHOT_TYPE=7;T.ANY_UNORDERED_NODE_TYPE=8;T.FIRST_ORDERED_NODE_TYPE=9;function mc(a){this.lookupNamespaceURI=ic(a)}\\nfunction nc(a,b){a=a||h;var c=a.Document&&a.Document.prototype||a.document;if(!c.evaluate||b)a.XPathResult=T,c.evaluate=function(d,e,f,g){return(new lc(d,f)).evaluate(e,g)},c.createExpression=function(d,e){return new lc(d,e)},c.createNSResolver=function(d){return new mc(d)}}ba(\\\"wgxpath.install\\\",nc);ba(\\\"wgxpath.install\\\",nc);var oc=window;function U(a,b){this.code=a;this.a=V[a]||pc;this.message=b||\\\"\\\";a=this.a.replace(/((?:^|\\\\s+)[a-z])/g,function(c){return c.toUpperCase().replace(/^[\\\\s\\\\xa0]+/g,\\\"\\\")});b=a.length-5;if(0>b||a.indexOf(\\\"Error\\\",b)!=b)a+=\\\"Error\\\";this.name=a;a=Error(this.message);a.name=this.name;this.stack=a.stack||\\\"\\\"}m(U,Error);var pc=\\\"unknown error\\\",V={15:\\\"element not selectable\\\",11:\\\"element not visible\\\"};V[31]=pc;V[30]=pc;V[24]=\\\"invalid cookie domain\\\";V[29]=\\\"invalid element coordinates\\\";V[12]=\\\"invalid element state\\\";\\nV[32]=\\\"invalid selector\\\";V[51]=\\\"invalid selector\\\";V[52]=\\\"invalid selector\\\";V[17]=\\\"javascript error\\\";V[405]=\\\"unsupported operation\\\";V[34]=\\\"move target out of bounds\\\";V[27]=\\\"no such alert\\\";V[7]=\\\"no such element\\\";V[8]=\\\"no such frame\\\";V[23]=\\\"no such window\\\";V[28]=\\\"script timeout\\\";V[33]=\\\"session not created\\\";V[10]=\\\"stale element reference\\\";V[21]=\\\"timeout\\\";V[25]=\\\"unable to set cookie\\\";V[26]=\\\"unexpected alert open\\\";V[13]=pc;V[9]=\\\"unknown command\\\";var qc=za(),rc=Ba()||q(\\\"iPod\\\"),sc=q(\\\"iPad\\\"),tc=q(\\\"Android\\\")&&!(Aa()||za()||q(\\\"Opera\\\")||q(\\\"Silk\\\")),uc=Aa(),vc=q(\\\"Safari\\\")&&!(Aa()||q(\\\"Coast\\\")||q(\\\"Opera\\\")||q(\\\"Edge\\\")||q(\\\"Edg/\\\")||q(\\\"OPR\\\")||za()||q(\\\"Silk\\\")||q(\\\"Android\\\"))&&!(Ba()||q(\\\"iPad\\\")||q(\\\"iPod\\\"));function wc(a){return(a=a.exec(p))?a[1]:\\\"\\\"}var xc=function(){if(qc)return wc(/Firefox\\\\/([0-9.]+)/);if(r||Fa||Ea)return Ka;if(uc)return Ba()||q(\\\"iPad\\\")||q(\\\"iPod\\\")?wc(/CriOS\\\\/([0-9.]+)/):wc(/Chrome\\\\/([0-9.]+)/);if(vc&&!(Ba()||q(\\\"iPad\\\")||q(\\\"iPod\\\")))return wc(/Version\\\\/([0-9.]+)/);if(rc||sc){var a=/Version\\\\/(\\\\S+).*Mobile\\\\/(\\\\S+)/.exec(p);if(a)return a[1]+\\\".\\\"+a[2]}else if(tc)return(a=wc(/Android\\\\s+([0-9.]+)/))?a:wc(/Version\\\\/([0-9.]+)/);return\\\"\\\"}();var yc=sc||rc,zc;if(tc){var Ac=/Android\\\\s+([0-9\\\\.]+)/.exec(p);zc=Ac?Ac[1]:\\\"0\\\"}else zc=\\\"0\\\";var Bc=zc,Cc=r&&!(8<=Number(w)),Dc=r&&!(9<=Number(w));function Ec(a,b){b=b.toLowerCase();return\\\"style\\\"==b?Fc(a.style.cssText):Cc&&\\\"value\\\"==b&&Gc(a,\\\"INPUT\\\")?a.value:Dc&&!0===a[b]?String(a.getAttribute(b)):(a=a.getAttributeNode(b))&&a.specified?a.value:null}var Hc=/[;]+(?=(?:(?:[^\\\"]*\\\"){2})*[^\\\"]*$)(?=(?:(?:[^']*'){2})*[^']*$)(?=(?:[^()]*\\\\([^()]*\\\\))*[^()]*$)/;\\nfunction Fc(a){var b=[];n(a.split(Hc),function(c){var d=c.indexOf(\\\":\\\");0<d&&(c=[c.slice(0,d),c.slice(d+1)],2==c.length&&b.push(c[0].toLowerCase(),\\\":\\\",c[1],\\\";\\\"))});b=b.join(\\\"\\\");return b=\\\";\\\"==b.charAt(b.length-1)?b:b+\\\";\\\"}function Ic(a,b){Cc&&\\\"value\\\"==b&&Gc(a,\\\"OPTION\\\")&&null===Ec(a,\\\"value\\\")?(b=[],eb(a,b,!1),a=b.join(\\\"\\\")):a=a[b];return a}function Gc(a,b){b&&\\\"string\\\"!==typeof b&&(b=b.toString());return!!a&&1==a.nodeType&&(!b||a.tagName.toUpperCase()==b)}\\nfunction Jc(a){return Gc(a,\\\"OPTION\\\")?!0:Gc(a,\\\"INPUT\\\")?(a=a.type.toLowerCase(),\\\"checkbox\\\"==a||\\\"radio\\\"==a):!1};function Kc(a){a=a?A(a):document;return!r||9<=Number(w)||\\\"CSS1Compat\\\"==(a?new fb(A(a)):la||(la=new fb)).a.compatMode?a.documentElement:a.body};var Lc=!(r&&!(r?0<=va(w,10):Oa(10))),Mc=tc?!(tc?0<=va(Bc,4):0<=va(xc,4)):!yc,Nc=r&&oc.navigator.msPointerEnabled;function Oc(a,b,c){this.a=a;this.b=b;this.f=c}Oc.prototype.create=function(a){a=A(a).createEvent(\\\"HTMLEvents\\\");a.initEvent(this.a,this.b,this.f);return a};Oc.prototype.toString=function(){return this.a};function W(a,b,c){ja(this,a,b,c)}m(W,Oc);\\nW.prototype.create=function(a,b){if(!t&&this==Pc)throw new U(9,\\\"Browser does not support a mouse pixel scroll event.\\\");var c=A(a),d=c?c.parentWindow||c.defaultView:window;var e=c.createEvent(\\\"MouseEvents\\\");var f=1;this==Qc&&(t||(e.wheelDelta=b.wheelDelta),t&&(f=b.wheelDelta/-40));t&&this==Pc&&(f=b.wheelDelta);e.initMouseEvent(this.a,this.b,this.f,d,f,b.clientX,b.clientY,b.clientX,b.clientY,b.ctrlKey,b.altKey,b.shiftKey,b.metaKey,b.button,b.relatedTarget);if(r&&0===e.pageX&&0===e.pageY&&Object.defineProperty){a=\\nYa((a?new fb(A(a)):la||(la=new fb)).a);c=Kc(c);var g=b.clientX+a.scrollLeft-c.clientLeft,k=b.clientY+a.scrollTop-c.clientTop;Object.defineProperty(e,\\\"pageX\\\",{get:function(){return g}});Object.defineProperty(e,\\\"pageY\\\",{get:function(){return k}})}return e};function Rc(a,b,c){ja(this,a,b,c)}m(Rc,Oc);\\nRc.prototype.create=function(a,b){var c=A(a);if(t&&!(r?0<=va(w,93):Oa(93))){a=c?c.parentWindow||c.defaultView:window;var d=b.charCode?0:b.keyCode;c=c.createEvent(\\\"KeyboardEvent\\\");c.initKeyEvent(this.a,this.b,this.f,a,b.ctrlKey,b.altKey,b.shiftKey,b.metaKey,d,b.charCode);this.a==Sc&&b.preventDefault&&c.preventDefault()}else if(c=c.createEvent(\\\"Events\\\"),c.initEvent(this.a,this.b,this.f),c.altKey=b.altKey,c.ctrlKey=b.ctrlKey,c.metaKey=b.metaKey,c.shiftKey=b.shiftKey,c.keyCode=b.charCode||b.keyCode,Ga||\\nFa)c.charCode=this==Sc?c.keyCode:0;return c};function Tc(a,b,c){ja(this,a,b,c)}m(Tc,Oc);\\nTc.prototype.create=function(a,b){function c(R){R=oa(R,function(v){return g.createTouch(k,a,v.identifier,v.pageX,v.pageY,v.screenX,v.screenY)});return g.createTouchList.apply(g,R)}function d(R){var v=oa(R,function(S){return{identifier:S.identifier,screenX:S.screenX,screenY:S.screenY,clientX:S.clientX,clientY:S.clientY,pageX:S.pageX,pageY:S.pageY,target:a}});v.item=function(S){return v[S]};return v}function e(R){return oa(R,function(v){return new Touch({identifier:v.identifier,screenX:v.screenX,screenY:v.screenY,\\nclientX:v.clientX,clientY:v.clientY,pageX:v.pageX,pageY:v.pageY,target:a})})}function f(R,v){switch(R){case 1:return d(v);case 2:return c(v);case 3:return e(v)}return null}if(!Lc)throw new U(9,\\\"Browser does not support firing touch events.\\\");var g=A(a),k=g?g.parentWindow||g.defaultView:window;if(Mc)var l=1;else if(TouchEvent.prototype.initTouchEvent)l=2;else if(TouchEvent&&0<TouchEvent.length)l=3;else throw new U(9,\\\"Not able to create touch events in this browser\\\");var u=f(l,b.changedTouches),K=b.touches==\\nb.changedTouches?u:f(l,b.touches),Ra=b.targetTouches==b.changedTouches?u:f(l,b.targetTouches);if(1==l)l=g.createEvent(\\\"MouseEvents\\\"),l.initMouseEvent(this.a,this.b,this.f,k,1,0,0,b.clientX,b.clientY,b.ctrlKey,b.altKey,b.shiftKey,b.metaKey,0,b.relatedTarget),l.touches=K,l.targetTouches=Ra,l.changedTouches=u,l.scale=b.scale,l.rotation=b.rotation;else if(2==l)l=g.createEvent(\\\"TouchEvent\\\"),0==l.initTouchEvent.length?l.initTouchEvent(K,Ra,u,this.a,k,0,0,b.clientX,b.clientY,b.ctrlKey,b.altKey,b.shiftKey,\\nb.metaKey):l.initTouchEvent(this.a,this.b,this.f,k,1,0,0,b.clientX,b.clientY,b.ctrlKey,b.altKey,b.shiftKey,b.metaKey,K,Ra,u,b.scale,b.rotation),l.relatedTarget=b.relatedTarget;else if(3==l)l=new TouchEvent(this.a,{touches:K,targetTouches:Ra,changedTouches:u,bubbles:this.b,cancelable:this.f,ctrlKey:b.ctrlKey,shiftKey:b.shiftKey,altKey:b.altKey,metaKey:b.metaKey});else throw new U(9,\\\"Illegal TouchEventStrategy_ value (this is a bug)\\\");return l};function X(a,b,c){ja(this,a,b,c)}m(X,Oc);\\nX.prototype.create=function(a,b){if(!Nc)throw new U(9,\\\"Browser does not support MSGesture events.\\\");var c=A(a);a=c?c.parentWindow||c.defaultView:window;c=c.createEvent(\\\"MSGestureEvent\\\");c.initGestureEvent(this.a,this.b,this.f,a,1,0,0,b.clientX,b.clientY,0,0,b.translationX,b.translationY,b.scale,b.expansion,b.rotation,b.velocityX,b.velocityY,b.velocityExpansion,b.velocityAngular,(new Date).getTime(),b.relatedTarget);return c};function Y(a,b,c){ja(this,a,b,c)}m(Y,Oc);\\nY.prototype.create=function(a,b){if(!Nc)throw new U(9,\\\"Browser does not support MSPointer events.\\\");var c=A(a);a=c?c.parentWindow||c.defaultView:window;c=c.createEvent(\\\"MSPointerEvent\\\");c.initPointerEvent(this.a,this.b,this.f,a,0,0,0,b.clientX,b.clientY,b.ctrlKey,b.altKey,b.shiftKey,b.metaKey,b.button,b.relatedTarget,0,0,b.width,b.height,b.pressure,b.rotation,b.tiltX,b.tiltY,b.pointerId,b.pointerType,0,b.isPrimary);return c};new W(\\\"click\\\",!0,!0);new W(\\\"contextmenu\\\",!0,!0);new W(\\\"dblclick\\\",!0,!0);\\nnew W(\\\"mousedown\\\",!0,!0);new W(\\\"mousemove\\\",!0,!1);new W(\\\"mouseout\\\",!0,!0);new W(\\\"mouseover\\\",!0,!0);new W(\\\"mouseup\\\",!0,!0);var Qc=new W(t?\\\"DOMMouseScroll\\\":\\\"mousewheel\\\",!0,!0),Pc=new W(\\\"MozMousePixelScroll\\\",!0,!0);new Rc(\\\"keydown\\\",!0,!0);var Sc=new Rc(\\\"keypress\\\",!0,!0);new Rc(\\\"keyup\\\",!0,!0);new Tc(\\\"touchend\\\",!0,!0);new Tc(\\\"touchmove\\\",!0,!0);new Tc(\\\"touchstart\\\",!0,!0);new X(\\\"MSGestureChange\\\",!0,!0);new X(\\\"MSGestureEnd\\\",!0,!0);new X(\\\"MSGestureHold\\\",!0,!0);new X(\\\"MSGestureStart\\\",!0,!0);\\nnew X(\\\"MSGestureTap\\\",!0,!0);new X(\\\"MSInertiaStart\\\",!0,!0);new Y(\\\"MSGotPointerCapture\\\",!0,!1);new Y(\\\"MSLostPointerCapture\\\",!0,!1);new Y(\\\"MSPointerCancel\\\",!0,!0);new Y(\\\"MSPointerDown\\\",!0,!0);new Y(\\\"MSPointerMove\\\",!0,!0);new Y(\\\"MSPointerOver\\\",!0,!0);new Y(\\\"MSPointerOut\\\",!0,!0);new Y(\\\"MSPointerUp\\\",!0,!0);function Uc(a,b){this.b={};this.a=[];this.f=0;var c=arguments.length;if(1<c){if(c%2)throw Error(\\\"Uneven number of arguments\\\");for(var d=0;d<c;d+=2)this.set(arguments[d],arguments[d+1])}else if(a)if(a instanceof Uc)for(c=Vc(a),d=0;d<c.length;d++)this.set(c[d],a.get(c[d]));else for(d in a)this.set(d,a[d])}\\nfunction Vc(a){if(a.f!=a.a.length){for(var b=0,c=0;b<a.a.length;){var d=a.a[b];Object.prototype.hasOwnProperty.call(a.b,d)&&(a.a[c++]=d);b++}a.a.length=c}if(a.f!=a.a.length){var e={};for(c=b=0;b<a.a.length;)d=a.a[b],Object.prototype.hasOwnProperty.call(e,d)||(a.a[c++]=d,e[d]=1),b++;a.a.length=c}return a.a.concat()}Uc.prototype.get=function(a,b){return Object.prototype.hasOwnProperty.call(this.b,a)?this.b[a]:b};\\nUc.prototype.set=function(a,b){Object.prototype.hasOwnProperty.call(this.b,a)||(this.f++,this.a.push(a));this.b[a]=b};var Wc={};function Z(a,b,c){da(a)&&(a=t?a.g:a.h);a=new Xc(a);!b||b in Wc&&!c||(Wc[b]={key:a,shift:!1},c&&(Wc[c]={key:a,shift:!0}));return a}function Xc(a){this.code=a}Z(8);Z(9);Z(13);var Yc=Z(16),Zc=Z(17),$c=Z(18);Z(19);Z(20);Z(27);Z(32,\\\" \\\");Z(33);Z(34);Z(35);Z(36);Z(37);Z(38);Z(39);Z(40);Z(44);Z(45);Z(46);Z(48,\\\"0\\\",\\\")\\\");Z(49,\\\"1\\\",\\\"!\\\");Z(50,\\\"2\\\",\\\"@\\\");Z(51,\\\"3\\\",\\\"#\\\");Z(52,\\\"4\\\",\\\"$\\\");Z(53,\\\"5\\\",\\\"%\\\");Z(54,\\\"6\\\",\\\"^\\\");Z(55,\\\"7\\\",\\\"&\\\");Z(56,\\\"8\\\",\\\"*\\\");Z(57,\\\"9\\\",\\\"(\\\");Z(65,\\\"a\\\",\\\"A\\\");Z(66,\\\"b\\\",\\\"B\\\");Z(67,\\\"c\\\",\\\"C\\\");Z(68,\\\"d\\\",\\\"D\\\");\\nZ(69,\\\"e\\\",\\\"E\\\");Z(70,\\\"f\\\",\\\"F\\\");Z(71,\\\"g\\\",\\\"G\\\");Z(72,\\\"h\\\",\\\"H\\\");Z(73,\\\"i\\\",\\\"I\\\");Z(74,\\\"j\\\",\\\"J\\\");Z(75,\\\"k\\\",\\\"K\\\");Z(76,\\\"l\\\",\\\"L\\\");Z(77,\\\"m\\\",\\\"M\\\");Z(78,\\\"n\\\",\\\"N\\\");Z(79,\\\"o\\\",\\\"O\\\");Z(80,\\\"p\\\",\\\"P\\\");Z(81,\\\"q\\\",\\\"Q\\\");Z(82,\\\"r\\\",\\\"R\\\");Z(83,\\\"s\\\",\\\"S\\\");Z(84,\\\"t\\\",\\\"T\\\");Z(85,\\\"u\\\",\\\"U\\\");Z(86,\\\"v\\\",\\\"V\\\");Z(87,\\\"w\\\",\\\"W\\\");Z(88,\\\"x\\\",\\\"X\\\");Z(89,\\\"y\\\",\\\"Y\\\");Z(90,\\\"z\\\",\\\"Z\\\");var ad=Z(Ia?{g:91,h:91}:Ha?{g:224,h:91}:{g:0,h:91});Z(Ia?{g:92,h:92}:Ha?{g:224,h:93}:{g:0,h:92});Z(Ia?{g:93,h:93}:Ha?{g:0,h:0}:{g:93,h:null});Z({g:96,h:96},\\\"0\\\");Z({g:97,h:97},\\\"1\\\");\\nZ({g:98,h:98},\\\"2\\\");Z({g:99,h:99},\\\"3\\\");Z({g:100,h:100},\\\"4\\\");Z({g:101,h:101},\\\"5\\\");Z({g:102,h:102},\\\"6\\\");Z({g:103,h:103},\\\"7\\\");Z({g:104,h:104},\\\"8\\\");Z({g:105,h:105},\\\"9\\\");Z({g:106,h:106},\\\"*\\\");Z({g:107,h:107},\\\"+\\\");Z({g:109,h:109},\\\"-\\\");Z({g:110,h:110},\\\".\\\");Z({g:111,h:111},\\\"/\\\");Z(144);Z(112);Z(113);Z(114);Z(115);Z(116);Z(117);Z(118);Z(119);Z(120);Z(121);Z(122);Z(123);Z({g:107,h:187},\\\"=\\\",\\\"+\\\");Z(108,\\\",\\\");Z({g:109,h:189},\\\"-\\\",\\\"_\\\");Z(188,\\\",\\\",\\\"<\\\");Z(190,\\\".\\\",\\\">\\\");Z(191,\\\"/\\\",\\\"?\\\");Z(192,\\\"`\\\",\\\"~\\\");Z(219,\\\"[\\\",\\\"{\\\");\\nZ(220,\\\"\\\\\\\\\\\",\\\"|\\\");Z(221,\\\"]\\\",\\\"}\\\");Z({g:59,h:186},\\\";\\\",\\\":\\\");Z(222,\\\"'\\\",'\\\"');var bd=new Uc;bd.set(1,Yc);bd.set(2,Zc);bd.set(4,$c);bd.set(8,ad);(function(a){var b=new Uc;n(Vc(a),function(c){b.set(a.get(c).code,c)});return b})(bd);var cd={\\\"class\\\":\\\"className\\\",readonly:\\\"readOnly\\\"},dd=\\\"allowfullscreen allowpaymentrequest allowusermedia async autofocus autoplay checked compact complete controls declare default defaultchecked defaultselected defer disabled ended formnovalidate hidden indeterminate iscontenteditable ismap itemscope loop multiple muted nohref nomodule noresize noshade novalidate nowrap open paused playsinline pubdate readonly required reversed scoped seamless seeking selected truespeed typemustmatch willvalidate\\\".split(\\\" \\\");ba(\\\"_\\\",function(a,b){var c=null,d=b.toLowerCase();if(\\\"style\\\"==d)return(c=a.style)&&!aa(c)&&(c=c.cssText),c;if((\\\"selected\\\"==d||\\\"checked\\\"==d)&&Jc(a)){if(!Jc(a))throw new U(15,\\\"Element is not selectable\\\");b=\\\"selected\\\";c=a.type&&a.type.toLowerCase();if(\\\"checkbox\\\"==c||\\\"radio\\\"==c)b=\\\"checked\\\";return Ic(a,b)?\\\"true\\\":null}var e=Gc(a,\\\"A\\\");if(Gc(a,\\\"IMG\\\")&&\\\"src\\\"==d||e&&\\\"href\\\"==d)return(c=Ec(a,d))&&(c=Ic(a,d)),c;if(\\\"spellcheck\\\"==d){c=Ec(a,d);if(null!==c){if(\\\"false\\\"==c.toLowerCase())return\\\"false\\\";if(\\\"true\\\"==c.toLowerCase())return\\\"true\\\"}return Ic(a,\\nd)+\\\"\\\"}e=cd[b]||b;if(0<=ma(dd,d))return(c=null!==Ec(a,b)||Ic(a,e))?\\\"true\\\":null;try{var f=Ic(a,e)}catch(g){}null==f||da(f)?c=Ec(a,b):c=f;return null!=c?c.toString():null});; return this._.apply(null,arguments);}).apply({navigator:typeof window!='undefined'?window.navigator:null,document:typeof window!='undefined'?window.document:null}, arguments);}\\n).apply(null, arguments);\", \"args\": [{\"element-6066-11e4-a52e-4f735466cecf\": \"5534fd34-47c3-4380-b01d-a18941502dfa\"}, \"innerHTML\"]} (remote_connection.py:303)\n",
      "2023-03-20 21:11:47 [   DEBUG] http://localhost:46743 \"POST /session/a5e86af6-7734-4cab-a830-816a0e7a134b/execute/sync HTTP/1.1\" 200 724764 (connectionpool.py:452)\n",
      "2023-03-20 21:11:47 [   DEBUG] Remote response: status=200 | data={\"value\":\"<thead><tr role=\\\"row\\\" style=\\\"height: 0px;\\\"><th class=\\\"sorting\\\" role=\\\"columnheader\\\" tabindex=\\\"0\\\" aria-controls=\\\"biocViews_package_table\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\" style=\\\"width: 169px; padding-top: 0px; padding-bottom: 0px; border-top-width: 0px; border-bottom-width: 0px; height: 0px;\\\" aria-label=\\\"Package: activate to sort column ascending\\\"></th><th class=\\\"sorting\\\" role=\\\"columnheader\\\" tabindex=\\\"0\\\" aria-controls=\\\"biocViews_package_table\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\" style=\\\"width: 109px; padding-top: 0px; padding-bottom: 0px; border-top-width: 0px; border-bottom-width: 0px; height: 0px;\\\" aria-label=\\\"Maintainer: activate to sort column ascending\\\"></th><th class=\\\"sorting\\\" role=\\\"columnheader\\\" tabindex=\\\"0\\\" aria-controls=\\\"biocViews_package_table\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\" style=\\\"width: 166px; padding-top: 0px; padding-bottom: 0px; border-top-width: 0px; border-bottom-width: 0px; height: 0px;\\\" aria-label=\\\"Title: activate to sort column ascending\\\"></th><th class=\\\"sorting_asc\\\" role=\\\"columnheader\\\" tabindex=\\\"0\\\" aria-controls=\\\"biocViews_package_table\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\" style=\\\"width: 31px; padding-top: 0px; padding-bottom: 0px; border-top-width: 0px; border-bottom-width: 0px; height: 0px;\\\" aria-sort=\\\"ascending\\\" aria-label=\\\"Rank: activate to sort column descending\\\"></th></tr></thead><tbody role=\\\"alert\\\" aria-live=\\\"polite\\\" aria-relevant=\\\"all\\\"><tr id=\\\"pkg_XCIR\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/XCIR.html\\\">XCIR</a></td>\\n\\t<td class=\\\" \\\">Renan Sauteraud</td>\\n\\t<td class=\\\" \\\">XCI-inference</td>\\n\\t<td class=\\\"  sorting_1\\\">1</td>\\n</tr><tr id=\\\"pkg_metabinR\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/metabinR.html\\\">metabinR</a></td>\\n\\t<td class=\\\" \\\">Anestis Gkanogiannis</td>\\n\\t<td class=\\\" \\\">Abundance and Compositional Based Binning of Metagenomes</td>\\n\\t<td class=\\\"  sorting_1\\\">2</td>\\n</tr><tr id=\\\"pkg_ATACCoGAPS\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ATACCoGAPS.html\\\">ATACCoGAPS</a></td>\\n\\t<td class=\\\" \\\">Rossin Erbe</td>\\n\\t<td class=\\\" \\\">Analysis Tools for scATACseq Data with CoGAPS</td>\\n\\t<td class=\\\"  sorting_1\\\">3</td>\\n</tr><tr id=\\\"pkg_ccImpute\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ccImpute.html\\\">ccImpute</a></td>\\n\\t<td class=\\\" \\\">Marcin Malec</td>\\n\\t<td class=\\\" \\\">ccImpute: an accurate and scalable consensus clustering based approach to impute dropout events in the single-cell RNA-seq data (https://doi.org/10.1186/s12859-022-04814-8)</td>\\n\\t<td class=\\\"  sorting_1\\\">3</td>\\n</tr><tr id=\\\"pkg_CircSeqAlignTk\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/CircSeqAlignTk.html\\\">CircSeqAlignTk</a></td>\\n\\t<td class=\\\" \\\">Jianqiang Sun</td>\\n\\t<td class=\\\" \\\">A toolkit for end-to-end analysis of RNA-seq data for circular genomes</td>\\n\\t<td class=\\\"  sorting_1\\\">3</td>\\n</tr><tr id=\\\"pkg_omada\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/omada.html\\\">omada</a></td>\\n\\t<td class=\\\" \\\">Sokratis Kariotis</td>\\n\\t<td class=\\\" \\\">Machine learning tools for automated transcriptome clustering analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">3</td>\\n</tr><tr id=\\\"pkg_RESOLVE\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/RESOLVE.html\\\">RESOLVE</a></td>\\n\\t<td class=\\\" \\\">Luca De Sano</td>\\n\\t<td class=\\\" \\\">RESOLVE: An R package for the efficient analysis of mutational signatures from cancer genomes</td>\\n\\t<td class=\\\"  sorting_1\\\">3</td>\\n</tr><tr id=\\\"pkg_scifer\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/scifer.html\\\">scifer</a></td>\\n\\t<td class=\\\" \\\">Rodrigo Arcoverde Cerveira</td>\\n\\t<td class=\\\" \\\">Scifer: Single-Cell Immunoglobulin Filtering of Sanger Sequences</td>\\n\\t<td class=\\\"  sorting_1\\\">3</td>\\n</tr><tr id=\\\"pkg_Statial\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/Statial.html\\\">Statial</a></td>\\n\\t<td class=\\\" \\\">Farhan Ameen</td>\\n\\t<td class=\\\" \\\">A package to identify changes in cell state relative to spatial associations</td>\\n\\t<td class=\\\"  sorting_1\\\">3</td>\\n</tr><tr id=\\\"pkg_stJoincount\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/stJoincount.html\\\">stJoincount</a></td>\\n\\t<td class=\\\" \\\">Jiarong Song</td>\\n\\t<td class=\\\" \\\">stJoincount - Join count statistic for quantifying spatial correlation between clusters</td>\\n\\t<td class=\\\"  sorting_1\\\">3</td>\\n</tr><tr id=\\\"pkg_BiocFHIR\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/BiocFHIR.html\\\">BiocFHIR</a></td>\\n\\t<td class=\\\" \\\">Vincent Carey</td>\\n\\t<td class=\\\" \\\">Illustration of FHIR ingestion and transformation using R</td>\\n\\t<td class=\\\"  sorting_1\\\">11</td>\\n</tr><tr id=\\\"pkg_EpiMix\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/EpiMix.html\\\">EpiMix</a></td>\\n\\t<td class=\\\" \\\">Yuanning Zheng</td>\\n\\t<td class=\\\" \\\">EpiMix: an integrative tool for the population-level analysis of DNA methylation</td>\\n\\t<td class=\\\"  sorting_1\\\">11</td>\\n</tr><tr id=\\\"pkg_iSEEhub\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/iSEEhub.html\\\">iSEEhub</a></td>\\n\\t<td class=\\\" \\\">Kevin Rue-Albrecht</td>\\n\\t<td class=\\\" \\\">iSEE for the Bioconductor ExperimentHub</td>\\n\\t<td class=\\\"  sorting_1\\\">11</td>\\n</tr><tr id=\\\"pkg_MetaPhOR\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/MetaPhOR.html\\\">MetaPhOR</a></td>\\n\\t<td class=\\\" \\\">Emily Isenhart</td>\\n\\t<td class=\\\" \\\">Metabolic Pathway Analysis of RNA</td>\\n\\t<td class=\\\"  sorting_1\\\">11</td>\\n</tr><tr id=\\\"pkg_regioneReloaded\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/regioneReloaded.html\\\">regioneReloaded</a></td>\\n\\t<td class=\\\" \\\">Roberto Malinverni</td>\\n\\t<td class=\\\" \\\">RegioneReloaded: Multiple Association for Genomic Region Sets</td>\\n\\t<td class=\\\"  sorting_1\\\">11</td>\\n</tr><tr id=\\\"pkg_signifinder\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/signifinder.html\\\">signifinder</a></td>\\n\\t<td class=\\\" \\\">Stefania Pirrotta</td>\\n\\t<td class=\\\" \\\">Implementations of transcriptional cancer signatures</td>\\n\\t<td class=\\\"  sorting_1\\\">11</td>\\n</tr><tr id=\\\"pkg_vsclust\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/vsclust.html\\\">vsclust</a></td>\\n\\t<td class=\\\" \\\">Veit Schwaemmle</td>\\n\\t<td class=\\\" \\\">Feature-based variance-sensitive quantitative clustering</td>\\n\\t<td class=\\\"  sorting_1\\\">11</td>\\n</tr><tr id=\\\"pkg_BioNAR\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/BioNAR.html\\\">BioNAR</a></td>\\n\\t<td class=\\\" \\\">Anatoly Sorokin</td>\\n\\t<td class=\\\" \\\">Biological Network Analysis in R</td>\\n\\t<td class=\\\"  sorting_1\\\">18</td>\\n</tr><tr id=\\\"pkg_consICA\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/consICA.html\\\">consICA</a></td>\\n\\t<td class=\\\" \\\">Petr V. Nazarov</td>\\n\\t<td class=\\\" \\\">consensus Independent Component Analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">18</td>\\n</tr><tr id=\\\"pkg_DNAfusion\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/DNAfusion.html\\\">DNAfusion</a></td>\\n\\t<td class=\\\" \\\">Christoffer Trier Maansson</td>\\n\\t<td class=\\\" \\\">Identification of gene fusions using paired-end sequencing</td>\\n\\t<td class=\\\"  sorting_1\\\">18</td>\\n</tr><tr id=\\\"pkg_epistasisGA\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/epistasisGA.html\\\">epistasisGA</a></td>\\n\\t<td class=\\\" \\\">Michael Nodzenski</td>\\n\\t<td class=\\\" \\\">An R package to identify multi-snp effects in nuclear family studies using the GADGETS method</td>\\n\\t<td class=\\\"  sorting_1\\\">18</td>\\n</tr><tr id=\\\"pkg_ISLET\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ISLET.html\\\">ISLET</a></td>\\n\\t<td class=\\\" \\\">Hao Feng</td>\\n\\t<td class=\\\" \\\">Individual-Specific ceLl typE referencing Tool</td>\\n\\t<td class=\\\"  sorting_1\\\">18</td>\\n</tr><tr id=\\\"pkg_magrene\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/magrene.html\\\">magrene</a></td>\\n\\t<td class=\\\" \\\">Fabrcio Almeida-Silva</td>\\n\\t<td class=\\\" \\\">Motif Analysis In Gene Regulatory Networks</td>\\n\\t<td class=\\\"  sorting_1\\\">18</td>\\n</tr><tr id=\\\"pkg_NetActivity\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/NetActivity.html\\\">NetActivity</a></td>\\n\\t<td class=\\\" \\\">Carlos Ruiz-Arenas</td>\\n\\t<td class=\\\" \\\">Compute gene set scores from a deep learning framework</td>\\n\\t<td class=\\\"  sorting_1\\\">18</td>\\n</tr><tr id=\\\"pkg_octad\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/octad.html\\\">octad</a></td>\\n\\t<td class=\\\" \\\">E. Chekalin</td>\\n\\t<td class=\\\" \\\">Open Cancer TherApeutic Discovery (OCTAD)</td>\\n\\t<td class=\\\"  sorting_1\\\">18</td>\\n</tr><tr id=\\\"pkg_RgnTX\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/RgnTX.html\\\">RgnTX</a></td>\\n\\t<td class=\\\" \\\">Yue Wang</td>\\n\\t<td class=\\\" \\\">Colocalization analysis of transcriptome elements in the presence of isoform heterogeneity and ambiguity</td>\\n\\t<td class=\\\"  sorting_1\\\">18</td>\\n</tr><tr id=\\\"pkg_scBubbletree\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/scBubbletree.html\\\">scBubbletree</a></td>\\n\\t<td class=\\\" \\\">Simo Kitanovski</td>\\n\\t<td class=\\\" \\\">Quantitative visual exploration of scRNA-seq data</td>\\n\\t<td class=\\\"  sorting_1\\\">18</td>\\n</tr><tr id=\\\"pkg_TENxIO\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/TENxIO.html\\\">TENxIO</a></td>\\n\\t<td class=\\\" \\\">Marcel Ramos</td>\\n\\t<td class=\\\" \\\">Import methods for 10X Genomics files</td>\\n\\t<td class=\\\"  sorting_1\\\">18</td>\\n</tr><tr id=\\\"pkg_VDJdive\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/VDJdive.html\\\">VDJdive</a></td>\\n\\t<td class=\\\" \\\">Kelly Street</td>\\n\\t<td class=\\\" \\\">Analysis Tools for 10X V(D)J Data</td>\\n\\t<td class=\\\"  sorting_1\\\">18</td>\\n</tr><tr id=\\\"pkg_zenith\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/zenith.html\\\">zenith</a></td>\\n\\t<td class=\\\" \\\">Gabriel Hoffman</td>\\n\\t<td class=\\\" \\\">Gene set analysis following differential expression using linear (mixed) modeling with dream</td>\\n\\t<td class=\\\"  sorting_1\\\">18</td>\\n</tr><tr id=\\\"pkg_oncoscanR\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/oncoscanR.html\\\">oncoscanR</a></td>\\n\\t<td class=\\\" \\\">Yann Christinat</td>\\n\\t<td class=\\\" \\\">Secondary analyses of CNV data (HRD and more)</td>\\n\\t<td class=\\\"  sorting_1\\\">31</td>\\n</tr><tr id=\\\"pkg_PanViz\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/PanViz.html\\\">PanViz</a></td>\\n\\t<td class=\\\" \\\">Luca Anholt</td>\\n\\t<td class=\\\" \\\">Integrating Multi-Omic Network Data With Summay-Level GWAS Data</td>\\n\\t<td class=\\\"  sorting_1\\\">31</td>\\n</tr><tr id=\\\"pkg_phenomis\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/phenomis.html\\\">phenomis</a></td>\\n\\t<td class=\\\" \\\">Etienne A. Thevenot</td>\\n\\t<td class=\\\" \\\">Postprocessing and univariate analysis of omics data</td>\\n\\t<td class=\\\"  sorting_1\\\">31</td>\\n</tr><tr id=\\\"pkg_RedisParam\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/RedisParam.html\\\">RedisParam</a></td>\\n\\t<td class=\\\" \\\">Martin Morgan</td>\\n\\t<td class=\\\" \\\">Provide a 'redis' back-end for BiocParallel</td>\\n\\t<td class=\\\"  sorting_1\\\">31</td>\\n</tr><tr id=\\\"pkg_scMET\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/scMET.html\\\">scMET</a></td>\\n\\t<td class=\\\" \\\">Andreas C. Kapourani</td>\\n\\t<td class=\\\" \\\">Bayesian modelling of cell-to-cell DNA methylation heterogeneity</td>\\n\\t<td class=\\\"  sorting_1\\\">31</td>\\n</tr><tr id=\\\"pkg_SimBu\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/SimBu.html\\\">SimBu</a></td>\\n\\t<td class=\\\" \\\">Alexander Dietrich</td>\\n\\t<td class=\\\" \\\">Simulate Bulk RNA-seq Datasets from Single-Cell Datasets</td>\\n\\t<td class=\\\"  sorting_1\\\">31</td>\\n</tr><tr id=\\\"pkg_spaSim\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/spaSim.html\\\">spaSim</a></td>\\n\\t<td class=\\\" \\\">Yuzhou Feng</td>\\n\\t<td class=\\\" \\\">Spatial point data simulator for tissue images</td>\\n\\t<td class=\\\"  sorting_1\\\">31</td>\\n</tr><tr id=\\\"pkg_SUITOR\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/SUITOR.html\\\">SUITOR</a></td>\\n\\t<td class=\\\" \\\">Bill Wheeler</td>\\n\\t<td class=\\\" \\\">Selecting the number of mutational signatures through cross-validation</td>\\n\\t<td class=\\\"  sorting_1\\\">31</td>\\n</tr><tr id=\\\"pkg_BASiCStan\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/BASiCStan.html\\\">BASiCStan</a></td>\\n\\t<td class=\\\" \\\">Alan O'Callaghan</td>\\n\\t<td class=\\\" \\\">Stan implementation of BASiCS</td>\\n\\t<td class=\\\"  sorting_1\\\">39</td>\\n</tr><tr id=\\\"pkg_EasyCellType\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/EasyCellType.html\\\">EasyCellType</a></td>\\n\\t<td class=\\\" \\\">Ruoxing Li</td>\\n\\t<td class=\\\" \\\">Annotate cell types for scRNA-seq data</td>\\n\\t<td class=\\\"  sorting_1\\\">39</td>\\n</tr><tr id=\\\"pkg_factR\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/factR.html\\\">factR</a></td>\\n\\t<td class=\\\" \\\">Fursham Hamid</td>\\n\\t<td class=\\\" \\\">Functional Annotation of Custom Transcriptomes</td>\\n\\t<td class=\\\"  sorting_1\\\">39</td>\\n</tr><tr id=\\\"pkg_FuseSOM\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/FuseSOM.html\\\">FuseSOM</a></td>\\n\\t<td class=\\\" \\\">Elijah Willie</td>\\n\\t<td class=\\\" \\\">A Correlation Based Multiview Self Organizing Maps Clustering For IMC Datasets</td>\\n\\t<td class=\\\"  sorting_1\\\">39</td>\\n</tr><tr id=\\\"pkg_goSorensen\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/goSorensen.html\\\">goSorensen</a></td>\\n\\t<td class=\\\" \\\">Pablo Flores</td>\\n\\t<td class=\\\" \\\">Statistical inference based on the Sorensen-Dice dissimilarity and the Gene Ontology (GO)</td>\\n\\t<td class=\\\"  sorting_1\\\">39</td>\\n</tr><tr id=\\\"pkg_MethCP\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/MethCP.html\\\">MethCP</a></td>\\n\\t<td class=\\\" \\\">Boying Gong</td>\\n\\t<td class=\\\" \\\">Differential methylation anlsysis for bisulfite sequencing data</td>\\n\\t<td class=\\\"  sorting_1\\\">39</td>\\n</tr><tr id=\\\"pkg_phemd\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/phemd.html\\\">phemd</a></td>\\n\\t<td class=\\\" \\\">William S Chen</td>\\n\\t<td class=\\\" \\\">Phenotypic EMD for comparison of single-cell samples</td>\\n\\t<td class=\\\"  sorting_1\\\">39</td>\\n</tr><tr id=\\\"pkg_scDDboost\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/scDDboost.html\\\">scDDboost</a></td>\\n\\t<td class=\\\" \\\">Xiuyu Ma</td>\\n\\t<td class=\\\" \\\">A compositional model to assess expression changes from single-cell rna-seq data</td>\\n\\t<td class=\\\"  sorting_1\\\">39</td>\\n</tr><tr id=\\\"pkg_ScreenR\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ScreenR.html\\\">ScreenR</a></td>\\n\\t<td class=\\\" \\\">Emanuel Michele Soda</td>\\n\\t<td class=\\\" \\\">Package to Perform High Throughput Biological Screening</td>\\n\\t<td class=\\\"  sorting_1\\\">39</td>\\n</tr><tr id=\\\"pkg_ATACseqTFEA\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ATACseqTFEA.html\\\">ATACseqTFEA</a></td>\\n\\t<td class=\\\" \\\">Jianhong Ou</td>\\n\\t<td class=\\\" \\\">Transcription Factor Enrichment Analysis for ATAC-seq</td>\\n\\t<td class=\\\"  sorting_1\\\">48</td>\\n</tr><tr id=\\\"pkg_GenomAutomorphism\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/GenomAutomorphism.html\\\">GenomAutomorphism</a></td>\\n\\t<td class=\\\" \\\">Robersy Sanchez</td>\\n\\t<td class=\\\" \\\">Compute the automorphisms between DNA's Abelian group representations</td>\\n\\t<td class=\\\"  sorting_1\\\">48</td>\\n</tr><tr id=\\\"pkg_HiCDOC\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/HiCDOC.html\\\">HiCDOC</a></td>\\n\\t<td class=\\\" \\\">Maign lise</td>\\n\\t<td class=\\\" \\\">A/B compartment detection and differential analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">48</td>\\n</tr><tr id=\\\"pkg_mslp\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/mslp.html\\\">mslp</a></td>\\n\\t<td class=\\\" \\\">Chunxuan Shao</td>\\n\\t<td class=\\\" \\\">Predict synthetic lethal partners of tumour mutations</td>\\n\\t<td class=\\\"  sorting_1\\\">48</td>\\n</tr><tr id=\\\"pkg_proteasy\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/proteasy.html\\\">proteasy</a></td>\\n\\t<td class=\\\" \\\">Martin Rydn</td>\\n\\t<td class=\\\" \\\">Protease Mapping</td>\\n\\t<td class=\\\"  sorting_1\\\">48</td>\\n</tr><tr id=\\\"pkg_ggtreeDendro\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ggtreeDendro.html\\\">ggtreeDendro</a></td>\\n\\t<td class=\\\" \\\">Guangchuang Yu</td>\\n\\t<td class=\\\" \\\">Drawing 'dendrogram' using 'ggtree'</td>\\n\\t<td class=\\\"  sorting_1\\\">53</td>\\n</tr><tr id=\\\"pkg_cardelino\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/cardelino.html\\\">cardelino</a></td>\\n\\t<td class=\\\" \\\">Jeffrey Pullin</td>\\n\\t<td class=\\\" \\\">Clone Identification from Single Cell Data</td>\\n\\t<td class=\\\"  sorting_1\\\">54</td>\\n</tr><tr id=\\\"pkg_coexnet\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/coexnet.html\\\">coexnet</a></td>\\n\\t<td class=\\\" \\\">Juan David Henao</td>\\n\\t<td class=\\\" \\\">coexnet: An R package to build CO-EXpression NETworks from Microarray Data</td>\\n\\t<td class=\\\"  sorting_1\\\">54</td>\\n</tr><tr id=\\\"pkg_gemma.R\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/gemma.R.html\\\">gemma.R</a></td>\\n\\t<td class=\\\" \\\">Ogan Mancarci</td>\\n\\t<td class=\\\" \\\">A wrapper for Gemma's Restful API to access curated gene expression data and differential expression analyses</td>\\n\\t<td class=\\\"  sorting_1\\\">54</td>\\n</tr><tr id=\\\"pkg_HiContacts\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/HiContacts.html\\\">HiContacts</a></td>\\n\\t<td class=\\\" \\\">Jacques Serizay</td>\\n\\t<td class=\\\" \\\">HiContacts: R interface to cool files</td>\\n\\t<td class=\\\"  sorting_1\\\">54</td>\\n</tr><tr id=\\\"pkg_iteremoval\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/iteremoval.html\\\">iteremoval</a></td>\\n\\t<td class=\\\" \\\">Jiacheng Chuan</td>\\n\\t<td class=\\\" \\\">Iteration removal method for feature selection</td>\\n\\t<td class=\\\"  sorting_1\\\">54</td>\\n</tr><tr id=\\\"pkg_SpliceWiz\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/SpliceWiz.html\\\">SpliceWiz</a></td>\\n\\t<td class=\\\" \\\">Alex Chit Hei Wong</td>\\n\\t<td class=\\\" \\\">Easy, optimized, and accurate alternative splicing analysis in R</td>\\n\\t<td class=\\\"  sorting_1\\\">54</td>\\n</tr><tr id=\\\"pkg_Voyager\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/Voyager.html\\\">Voyager</a></td>\\n\\t<td class=\\\" \\\">Lambda Moses</td>\\n\\t<td class=\\\" \\\">From geospatial to spatial omics</td>\\n\\t<td class=\\\"  sorting_1\\\">54</td>\\n</tr><tr id=\\\"pkg_CTSV\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/CTSV.html\\\">CTSV</a></td>\\n\\t<td class=\\\" \\\">Jinge Yu Developer</td>\\n\\t<td class=\\\" \\\">Identification of cell-type-specific spatially variable genes accounting for excess zeros</td>\\n\\t<td class=\\\"  sorting_1\\\">61</td>\\n</tr><tr id=\\\"pkg_simpleSeg\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/simpleSeg.html\\\">simpleSeg</a></td>\\n\\t<td class=\\\" \\\">Ellis Patrick</td>\\n\\t<td class=\\\" \\\">A package to perform simple cell segmentation</td>\\n\\t<td class=\\\"  sorting_1\\\">61</td>\\n</tr><tr id=\\\"pkg_katdetectr\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/katdetectr.html\\\">katdetectr</a></td>\\n\\t<td class=\\\" \\\">Daan Hazelaar</td>\\n\\t<td class=\\\" \\\">Detection, Characterization and Visualization of Kataegis in Sequencing Data</td>\\n\\t<td class=\\\"  sorting_1\\\">63</td>\\n</tr><tr id=\\\"pkg_MSstatsShiny\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/MSstatsShiny.html\\\">MSstatsShiny</a></td>\\n\\t<td class=\\\" \\\">Devon Kohler</td>\\n\\t<td class=\\\" \\\">MSstats GUI for Statistical Anaylsis of Proteomics Experiments</td>\\n\\t<td class=\\\"  sorting_1\\\">63</td>\\n</tr><tr id=\\\"pkg_PrecisionTrialDrawer\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/PrecisionTrialDrawer.html\\\">PrecisionTrialDrawer</a></td>\\n\\t<td class=\\\" \\\">Giorgio Melloni</td>\\n\\t<td class=\\\" \\\">A Tool to Analyze and Design NGS Based Custom Gene Panels</td>\\n\\t<td class=\\\"  sorting_1\\\">63</td>\\n</tr><tr id=\\\"pkg_pulsedSilac\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/pulsedSilac.html\\\">pulsedSilac</a></td>\\n\\t<td class=\\\" \\\">Marc Pags-Gallego</td>\\n\\t<td class=\\\" \\\">Analysis of pulsed-SILAC quantitative proteomics data</td>\\n\\t<td class=\\\"  sorting_1\\\">63</td>\\n</tr><tr id=\\\"pkg_rifi\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/rifi.html\\\">rifi</a></td>\\n\\t<td class=\\\" \\\">Jens Georg</td>\\n\\t<td class=\\\" \\\">'rifi' analyses data from rifampicin time series created by microarray or RNAseq</td>\\n\\t<td class=\\\"  sorting_1\\\">67</td>\\n</tr><tr id=\\\"pkg_SPIAT\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/SPIAT.html\\\">SPIAT</a></td>\\n\\t<td class=\\\" \\\">Yuzhou Feng</td>\\n\\t<td class=\\\" \\\">Spatial Image Analysis of Tissues</td>\\n\\t<td class=\\\"  sorting_1\\\">67</td>\\n</tr><tr id=\\\"pkg_HilbertVisGUI\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/HilbertVisGUI.html\\\">HilbertVisGUI</a></td>\\n\\t<td class=\\\" \\\">Simon Anders</td>\\n\\t<td class=\\\" \\\">HilbertVisGUI</td>\\n\\t<td class=\\\"  sorting_1\\\">69</td>\\n</tr><tr id=\\\"pkg_MsExperiment\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/MsExperiment.html\\\">MsExperiment</a></td>\\n\\t<td class=\\\" \\\">Laurent Gatto</td>\\n\\t<td class=\\\" \\\">Infrastructure for Mass Spectrometry Experiments</td>\\n\\t<td class=\\\"  sorting_1\\\">69</td>\\n</tr><tr id=\\\"pkg_Travel\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/Travel.html\\\">Travel</a></td>\\n\\t<td class=\\\" \\\">Jiefei Wang</td>\\n\\t<td class=\\\" \\\">An utility to create an ALTREP object with a virtual pointer</td>\\n\\t<td class=\\\"  sorting_1\\\">69</td>\\n</tr><tr id=\\\"pkg_ctgGEM\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ctgGEM.html\\\">ctgGEM</a></td>\\n\\t<td class=\\\" \\\">USD Biomedical Engineering</td>\\n\\t<td class=\\\" \\\">Generating Tree Hierarchy Visualizations from Gene Expression Data</td>\\n\\t<td class=\\\"  sorting_1\\\">72</td>\\n</tr><tr id=\\\"pkg_inveRsion\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/inveRsion.html\\\">inveRsion</a></td>\\n\\t<td class=\\\" \\\">Alejandro Caceres</td>\\n\\t<td class=\\\" \\\">Inversions in genotype data</td>\\n\\t<td class=\\\"  sorting_1\\\">72</td>\\n</tr><tr id=\\\"pkg_IsoGeneGUI\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/IsoGeneGUI.html\\\">IsoGeneGUI</a></td>\\n\\t<td class=\\\" \\\">Setia Pramana</td>\\n\\t<td class=\\\" \\\">A graphical user interface to conduct a dose-response analysis of microarray data</td>\\n\\t<td class=\\\"  sorting_1\\\">72</td>\\n</tr><tr id=\\\"pkg_MACPET\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/MACPET.html\\\">MACPET</a></td>\\n\\t<td class=\\\" \\\">Ioannis Vardaxis</td>\\n\\t<td class=\\\" \\\">Model based analysis for paired-end data</td>\\n\\t<td class=\\\"  sorting_1\\\">72</td>\\n</tr><tr id=\\\"pkg_RGMQL\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/RGMQL.html\\\">RGMQL</a></td>\\n\\t<td class=\\\" \\\">Simone Pallotta</td>\\n\\t<td class=\\\" \\\">GenoMetric Query Language for R/Bioconductor</td>\\n\\t<td class=\\\"  sorting_1\\\">72</td>\\n</tr><tr id=\\\"pkg_crisprVerse\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/crisprVerse.html\\\">crisprVerse</a></td>\\n\\t<td class=\\\" \\\">Jean-Philippe Fortin</td>\\n\\t<td class=\\\" \\\">Easily install and load the crisprVerse ecosystem for CRISPR gRNA design</td>\\n\\t<td class=\\\"  sorting_1\\\">77</td>\\n</tr><tr id=\\\"pkg_crisprViz\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/crisprViz.html\\\">crisprViz</a></td>\\n\\t<td class=\\\" \\\">Jean-Philippe Fortin</td>\\n\\t<td class=\\\" \\\">Visualization Functions for CRISPR gRNAs</td>\\n\\t<td class=\\\"  sorting_1\\\">77</td>\\n</tr><tr id=\\\"pkg_GAPGOM\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/GAPGOM.html\\\">GAPGOM</a></td>\\n\\t<td class=\\\" \\\">Rezvan Ehsani</td>\\n\\t<td class=\\\" \\\">GAPGOM (novel Gene Annotation Prediction and other GO Metrics)</td>\\n\\t<td class=\\\"  sorting_1\\\">77</td>\\n</tr><tr id=\\\"pkg_MMAPPR2\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/MMAPPR2.html\\\">MMAPPR2</a></td>\\n\\t<td class=\\\" \\\">Jonathon Hill</td>\\n\\t<td class=\\\" \\\">Mutation Mapping Analysis Pipeline for Pooled RNA-Seq</td>\\n\\t<td class=\\\"  sorting_1\\\">77</td>\\n</tr><tr id=\\\"pkg_RAREsim\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/RAREsim.html\\\">RAREsim</a></td>\\n\\t<td class=\\\" \\\">Ryan Barnard</td>\\n\\t<td class=\\\" \\\">Simulation of Rare Variant Genetic Data</td>\\n\\t<td class=\\\"  sorting_1\\\">77</td>\\n</tr><tr id=\\\"pkg_tomoseqr\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/tomoseqr.html\\\">tomoseqr</a></td>\\n\\t<td class=\\\" \\\">Ryosuke Matsuzawa</td>\\n\\t<td class=\\\" \\\">R Package for Analyzing Tomo-seq Data</td>\\n\\t<td class=\\\"  sorting_1\\\">77</td>\\n</tr><tr id=\\\"pkg_HTSeqGenie\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/HTSeqGenie.html\\\">HTSeqGenie</a></td>\\n\\t<td class=\\\" \\\">Jens Reeder</td>\\n\\t<td class=\\\" \\\">A NGS analysis pipeline.</td>\\n\\t<td class=\\\"  sorting_1\\\">83</td>\\n</tr><tr id=\\\"pkg_Macarron\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/Macarron.html\\\">Macarron</a></td>\\n\\t<td class=\\\" \\\">Sagun Maharjan</td>\\n\\t<td class=\\\" \\\">Prioritization of potentially bioactive metabolic features from epidemiological and environmental metabolomics datasets</td>\\n\\t<td class=\\\"  sorting_1\\\">83</td>\\n</tr><tr id=\\\"pkg_RNASeqR\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/RNASeqR.html\\\">RNASeqR</a></td>\\n\\t<td class=\\\" \\\">Kuan-Hao Chao</td>\\n\\t<td class=\\\" \\\">RNASeqR: an R package for automated two-group RNA-Seq analysis workflow</td>\\n\\t<td class=\\\"  sorting_1\\\">83</td>\\n</tr><tr id=\\\"pkg_single\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/single.html\\\">single</a></td>\\n\\t<td class=\\\" \\\">Rocio Espada</td>\\n\\t<td class=\\\" \\\">Accurate consensus sequence from nanopore reads of a gene library</td>\\n\\t<td class=\\\"  sorting_1\\\">83</td>\\n</tr><tr id=\\\"pkg_SpotClean\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/SpotClean.html\\\">SpotClean</a></td>\\n\\t<td class=\\\" \\\">Zijian Ni</td>\\n\\t<td class=\\\" \\\">SpotClean adjusts for spot swapping in spatial transcriptomics data</td>\\n\\t<td class=\\\"  sorting_1\\\">83</td>\\n</tr><tr id=\\\"pkg_biodbExpasy\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/biodbExpasy.html\\\">biodbExpasy</a></td>\\n\\t<td class=\\\" \\\">Pierrick Roger</td>\\n\\t<td class=\\\" \\\">biodbExpasy, a library for connecting to Expasy ENZYME database.</td>\\n\\t<td class=\\\"  sorting_1\\\">88</td>\\n</tr><tr id=\\\"pkg_biodbNci\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/biodbNci.html\\\">biodbNci</a></td>\\n\\t<td class=\\\" \\\">Pierrick Roger</td>\\n\\t<td class=\\\" \\\">biodbNci, a library for connecting to biodbNci, a library for connecting to the National Cancer Institute (USA) CACTUS Database</td>\\n\\t<td class=\\\"  sorting_1\\\">88</td>\\n</tr><tr id=\\\"pkg_borealis\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/borealis.html\\\">borealis</a></td>\\n\\t<td class=\\\" \\\">Garrett Jenkinson</td>\\n\\t<td class=\\\" \\\">Bisulfite-seq OutlieR mEthylation At singLe-sIte reSolution</td>\\n\\t<td class=\\\"  sorting_1\\\">88</td>\\n</tr><tr id=\\\"pkg_DepInfeR\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/DepInfeR.html\\\">DepInfeR</a></td>\\n\\t<td class=\\\" \\\">Junyan Lu</td>\\n\\t<td class=\\\" \\\">Inferring tumor-specific cancer dependencies through integrating ex-vivo drug response assays and drug-protein profiling</td>\\n\\t<td class=\\\"  sorting_1\\\">88</td>\\n</tr><tr id=\\\"pkg_GenProSeq\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/GenProSeq.html\\\">GenProSeq</a></td>\\n\\t<td class=\\\" \\\">Dongmin Jung</td>\\n\\t<td class=\\\" \\\">Generating Protein Sequences with Deep Generative Models</td>\\n\\t<td class=\\\"  sorting_1\\\">88</td>\\n</tr><tr id=\\\"pkg_mbOmic\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/mbOmic.html\\\">mbOmic</a></td>\\n\\t<td class=\\\" \\\">Congcong Gong</td>\\n\\t<td class=\\\" \\\">Integrative analysis of the microbiome and metabolome</td>\\n\\t<td class=\\\"  sorting_1\\\">88</td>\\n</tr><tr id=\\\"pkg_pareg\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/pareg.html\\\">pareg</a></td>\\n\\t<td class=\\\" \\\">Kim Philipp Jablonski</td>\\n\\t<td class=\\\" \\\">Pathway enrichment using a regularized regression approach</td>\\n\\t<td class=\\\"  sorting_1\\\">88</td>\\n</tr><tr id=\\\"pkg_PoTRA\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/PoTRA.html\\\">PoTRA</a></td>\\n\\t<td class=\\\" \\\">Margaret Linan</td>\\n\\t<td class=\\\" \\\">PoTRA: Pathways of Topological Rank Analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">88</td>\\n</tr><tr id=\\\"pkg_Rnits\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/Rnits.html\\\">Rnits</a></td>\\n\\t<td class=\\\" \\\">Dipen P. Sangurdekar</td>\\n\\t<td class=\\\" \\\">R Normalization and Inference of Time Series data</td>\\n\\t<td class=\\\"  sorting_1\\\">88</td>\\n</tr><tr id=\\\"pkg_scatterHatch\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/scatterHatch.html\\\">scatterHatch</a></td>\\n\\t<td class=\\\" \\\">Atul Deshpande</td>\\n\\t<td class=\\\" \\\">Creates hatched patterns for scatterplots</td>\\n\\t<td class=\\\"  sorting_1\\\">88</td>\\n</tr><tr id=\\\"pkg_terraTCGAdata\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/terraTCGAdata.html\\\">terraTCGAdata</a></td>\\n\\t<td class=\\\" \\\">Marcel Ramos</td>\\n\\t<td class=\\\" \\\">OpenAccess TCGA Data on Terra as MultiAssayExperiment</td>\\n\\t<td class=\\\"  sorting_1\\\">88</td>\\n</tr><tr id=\\\"pkg_TREG\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/TREG.html\\\">TREG</a></td>\\n\\t<td class=\\\" \\\">Louise Huuki-Myers</td>\\n\\t<td class=\\\" \\\">Tools for finding Total RNA Expression Genes in single nucleus RNA-seq data</td>\\n\\t<td class=\\\"  sorting_1\\\">88</td>\\n</tr><tr id=\\\"pkg_tspair\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/tspair.html\\\">tspair</a></td>\\n\\t<td class=\\\" \\\">Jeffrey T. Leek</td>\\n\\t<td class=\\\" \\\">Top Scoring Pairs for Microarray Classification</td>\\n\\t<td class=\\\"  sorting_1\\\">88</td>\\n</tr><tr id=\\\"pkg_biodbMirbase\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/biodbMirbase.html\\\">biodbMirbase</a></td>\\n\\t<td class=\\\" \\\">Pierrick Roger</td>\\n\\t<td class=\\\" \\\">biodbMirbase, a library for connecting to miRBase mature database</td>\\n\\t<td class=\\\"  sorting_1\\\">101</td>\\n</tr><tr id=\\\"pkg_iBMQ\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/iBMQ.html\\\">iBMQ</a></td>\\n\\t<td class=\\\" \\\">Greg Imholte</td>\\n\\t<td class=\\\" \\\">integrated Bayesian Modeling of eQTL data</td>\\n\\t<td class=\\\"  sorting_1\\\">101</td>\\n</tr><tr id=\\\"pkg_MobilityTransformR\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/MobilityTransformR.html\\\">MobilityTransformR</a></td>\\n\\t<td class=\\\" \\\">Liesa Salzer</td>\\n\\t<td class=\\\" \\\">Effective mobility scale transformation of CE-MS(/MS) data</td>\\n\\t<td class=\\\"  sorting_1\\\">101</td>\\n</tr><tr id=\\\"pkg_OGRE\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/OGRE.html\\\">OGRE</a></td>\\n\\t<td class=\\\" \\\">Sven Berres</td>\\n\\t<td class=\\\" \\\">Calculate, visualize and analyse overlap between genomic regions</td>\\n\\t<td class=\\\"  sorting_1\\\">101</td>\\n</tr><tr id=\\\"pkg_RolDE\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/RolDE.html\\\">RolDE</a></td>\\n\\t<td class=\\\" \\\">Tommi Valikangas</td>\\n\\t<td class=\\\" \\\">RolDE: Robust longitudinal Differential Expression</td>\\n\\t<td class=\\\"  sorting_1\\\">101</td>\\n</tr><tr id=\\\"pkg_TrajectoryGeometry\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/TrajectoryGeometry.html\\\">TrajectoryGeometry</a></td>\\n\\t<td class=\\\" \\\">Michael Shapiro</td>\\n\\t<td class=\\\" \\\">This Package Discovers Directionality in Time and Pseudo-times Series of Gene Expression Patterns</td>\\n\\t<td class=\\\"  sorting_1\\\">101</td>\\n</tr><tr id=\\\"pkg_BOBaFIT\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/BOBaFIT.html\\\">BOBaFIT</a></td>\\n\\t<td class=\\\" \\\">Gaia Mazzocchetti</td>\\n\\t<td class=\\\" \\\">Refitting diploid region profiles using a clustering procedure</td>\\n\\t<td class=\\\"  sorting_1\\\">107</td>\\n</tr><tr id=\\\"pkg_CBEA\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/CBEA.html\\\">CBEA</a></td>\\n\\t<td class=\\\" \\\">Quang Nguyen</td>\\n\\t<td class=\\\" \\\">Competitive Balances for Taxonomic Enrichment Analysis in R</td>\\n\\t<td class=\\\"  sorting_1\\\">107</td>\\n</tr><tr id=\\\"pkg_comapr\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/comapr.html\\\">comapr</a></td>\\n\\t<td class=\\\" \\\">Ruqian Lyu</td>\\n\\t<td class=\\\" \\\">Crossover analysis and genetic map construction</td>\\n\\t<td class=\\\"  sorting_1\\\">107</td>\\n</tr><tr id=\\\"pkg_crisprBwa\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/crisprBwa.html\\\">crisprBwa</a></td>\\n\\t<td class=\\\" \\\">Jean-Philippe Fortin</td>\\n\\t<td class=\\\" \\\">BWA-based alignment of CRISPR gRNA spacer sequences</td>\\n\\t<td class=\\\"  sorting_1\\\">107</td>\\n</tr><tr id=\\\"pkg_GenomicInteractionNodes\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/GenomicInteractionNodes.html\\\">GenomicInteractionNodes</a></td>\\n\\t<td class=\\\" \\\">Jianhong Ou</td>\\n\\t<td class=\\\" \\\">A R/Bioconductor package to detect the interaction nodes from HiC/HiChIP/HiCAR data</td>\\n\\t<td class=\\\"  sorting_1\\\">107</td>\\n</tr><tr id=\\\"pkg_MBECS\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/MBECS.html\\\">MBECS</a></td>\\n\\t<td class=\\\" \\\">Michael Olbrich</td>\\n\\t<td class=\\\" \\\">Evaluation and correction of batch effects in microbiome data-sets</td>\\n\\t<td class=\\\"  sorting_1\\\">107</td>\\n</tr><tr id=\\\"pkg_ModCon\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ModCon.html\\\">ModCon</a></td>\\n\\t<td class=\\\" \\\">Johannes Ptok</td>\\n\\t<td class=\\\" \\\">Modifying splice site usage by changing the mRNP code, while maintaining the genetic code</td>\\n\\t<td class=\\\"  sorting_1\\\">107</td>\\n</tr><tr id=\\\"pkg_Motif2Site\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/Motif2Site.html\\\">Motif2Site</a></td>\\n\\t<td class=\\\" \\\">Peyman Zarrineh</td>\\n\\t<td class=\\\" \\\">Detect binding sites from motifs and ChIP-seq experiments, and compare binding sites across conditions</td>\\n\\t<td class=\\\"  sorting_1\\\">107</td>\\n</tr><tr id=\\\"pkg_ODER\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ODER.html\\\">ODER</a></td>\\n\\t<td class=\\\" \\\">David Zhang</td>\\n\\t<td class=\\\" \\\">Optimising the Definition of Expressed Regions</td>\\n\\t<td class=\\\"  sorting_1\\\">107</td>\\n</tr><tr id=\\\"pkg_protGear\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/protGear.html\\\">protGear</a></td>\\n\\t<td class=\\\" \\\">Kennedy Mwai</td>\\n\\t<td class=\\\" \\\">Protein Micro Array Data Management and Interactive Visualization</td>\\n\\t<td class=\\\"  sorting_1\\\">107</td>\\n</tr><tr id=\\\"pkg_qsvaR\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/qsvaR.html\\\">qsvaR</a></td>\\n\\t<td class=\\\" \\\">Joshua Stolz</td>\\n\\t<td class=\\\" \\\">Generate Quality Surrogate Variable Analysis for Degradation Correction</td>\\n\\t<td class=\\\"  sorting_1\\\">107</td>\\n</tr><tr id=\\\"pkg_rGenomeTracks\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/rGenomeTracks.html\\\">rGenomeTracks</a></td>\\n\\t<td class=\\\" \\\">Omar Elashkar</td>\\n\\t<td class=\\\" \\\">Integerated visualization of epigenomic data</td>\\n\\t<td class=\\\"  sorting_1\\\">107</td>\\n</tr><tr id=\\\"pkg_Summix\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/Summix.html\\\">Summix</a></td>\\n\\t<td class=\\\" \\\">Audrey Hendricks</td>\\n\\t<td class=\\\" \\\">Summix: A method to estimate and adjust for population structure in genetic summary data</td>\\n\\t<td class=\\\"  sorting_1\\\">107</td>\\n</tr><tr id=\\\"pkg_svaNUMT\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/svaNUMT.html\\\">svaNUMT</a></td>\\n\\t<td class=\\\" \\\">Ruining Dong</td>\\n\\t<td class=\\\" \\\">NUMT detection from structural variant calls</td>\\n\\t<td class=\\\"  sorting_1\\\">107</td>\\n</tr><tr id=\\\"pkg_TraRe\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/TraRe.html\\\">TraRe</a></td>\\n\\t<td class=\\\" \\\">Jesus De La Fuente Cedeo</td>\\n\\t<td class=\\\" \\\">Transcriptional Rewiring</td>\\n\\t<td class=\\\"  sorting_1\\\">107</td>\\n</tr><tr id=\\\"pkg_updateObject\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/updateObject.html\\\">updateObject</a></td>\\n\\t<td class=\\\" \\\">Herv Pags</td>\\n\\t<td class=\\\" \\\">Find/fix old serialized S4 instances</td>\\n\\t<td class=\\\"  sorting_1\\\">107</td>\\n</tr><tr id=\\\"pkg_xcore\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/xcore.html\\\">xcore</a></td>\\n\\t<td class=\\\" \\\">Maciej Migda</td>\\n\\t<td class=\\\" \\\">xcore expression regulators inference</td>\\n\\t<td class=\\\"  sorting_1\\\">107</td>\\n</tr><tr id=\\\"pkg_CNVMetrics\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/CNVMetrics.html\\\">CNVMetrics</a></td>\\n\\t<td class=\\\" \\\">Astrid Deschnes</td>\\n\\t<td class=\\\" \\\">Copy Number Variant Metrics</td>\\n\\t<td class=\\\"  sorting_1\\\">124</td>\\n</tr><tr id=\\\"pkg_CyTOFpower\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/CyTOFpower.html\\\">CyTOFpower</a></td>\\n\\t<td class=\\\" \\\">Anne-Maud Ferreira</td>\\n\\t<td class=\\\" \\\">Power analysis for CyTOF experiments</td>\\n\\t<td class=\\\"  sorting_1\\\">124</td>\\n</tr><tr id=\\\"pkg_demuxmix\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/demuxmix.html\\\">demuxmix</a></td>\\n\\t<td class=\\\" \\\">Hans-Ulrich Klein</td>\\n\\t<td class=\\\" \\\">Demultiplexing oligo-barcoded scRNA-seq data using regression mixture models</td>\\n\\t<td class=\\\"  sorting_1\\\">124</td>\\n</tr><tr id=\\\"pkg_omicsViewer\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/omicsViewer.html\\\">omicsViewer</a></td>\\n\\t<td class=\\\" \\\">Chen Meng</td>\\n\\t<td class=\\\" \\\">Interactive and explorative visualization of SummarizedExperssionSet or ExpressionSet using omicsViewer</td>\\n\\t<td class=\\\"  sorting_1\\\">124</td>\\n</tr><tr id=\\\"pkg_phosphonormalizer\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/phosphonormalizer.html\\\">phosphonormalizer</a></td>\\n\\t<td class=\\\" \\\">Sohrab Saraei</td>\\n\\t<td class=\\\" \\\">Compensates for the bias introduced by median normalization in</td>\\n\\t<td class=\\\"  sorting_1\\\">124</td>\\n</tr><tr id=\\\"pkg_PoDCall\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/PoDCall.html\\\">PoDCall</a></td>\\n\\t<td class=\\\" \\\">Hans Petter Brodal</td>\\n\\t<td class=\\\" \\\">Positive Droplet Calling for DNA Methylation Droplet Digital PCR</td>\\n\\t<td class=\\\"  sorting_1\\\">124</td>\\n</tr><tr id=\\\"pkg_qmtools\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/qmtools.html\\\">qmtools</a></td>\\n\\t<td class=\\\" \\\">Jaehyun Joo</td>\\n\\t<td class=\\\" \\\">Quantitative Metabolomics Data Processing Tools</td>\\n\\t<td class=\\\"  sorting_1\\\">124</td>\\n</tr><tr id=\\\"pkg_sccomp\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/sccomp.html\\\">sccomp</a></td>\\n\\t<td class=\\\" \\\">Stefano Mangiola</td>\\n\\t<td class=\\\" \\\">Robust Outlier-aware Estimation of Composition and Heterogeneity for Single-cell Data</td>\\n\\t<td class=\\\"  sorting_1\\\">124</td>\\n</tr><tr id=\\\"pkg_svaRetro\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/svaRetro.html\\\">svaRetro</a></td>\\n\\t<td class=\\\" \\\">Ruining Dong</td>\\n\\t<td class=\\\" \\\">Retrotransposed transcript detection from structural variants</td>\\n\\t<td class=\\\"  sorting_1\\\">124</td>\\n</tr><tr id=\\\"pkg_TEKRABber\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/TEKRABber.html\\\">TEKRABber</a></td>\\n\\t<td class=\\\" \\\">Yao-Chung Chen</td>\\n\\t<td class=\\\" \\\">An R package estimates the correlations of orthologs and transposable elements between two species</td>\\n\\t<td class=\\\"  sorting_1\\\">124</td>\\n</tr><tr id=\\\"pkg_tomoda\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/tomoda.html\\\">tomoda</a></td>\\n\\t<td class=\\\" \\\">Wendao Liu</td>\\n\\t<td class=\\\" \\\">Tomo-seq data analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">124</td>\\n</tr><tr id=\\\"pkg_biodbNcbi\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/biodbNcbi.html\\\">biodbNcbi</a></td>\\n\\t<td class=\\\" \\\">Pierrick Roger</td>\\n\\t<td class=\\\" \\\">biodbNcbi, a library for connecting to NCBI Databases.</td>\\n\\t<td class=\\\"  sorting_1\\\">135</td>\\n</tr><tr id=\\\"pkg_conclus\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/conclus.html\\\">conclus</a></td>\\n\\t<td class=\\\" \\\">Ilyess Rachedi</td>\\n\\t<td class=\\\" \\\">ScRNA-seq Workflow CONCLUS - From CONsensus CLUSters To A Meaningful CONCLUSion</td>\\n\\t<td class=\\\"  sorting_1\\\">135</td>\\n</tr><tr id=\\\"pkg_dStruct\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/dStruct.html\\\">dStruct</a></td>\\n\\t<td class=\\\" \\\">Krishna Choudhary</td>\\n\\t<td class=\\\" \\\">Identifying differentially reactive regions from RNA structurome profiling data</td>\\n\\t<td class=\\\"  sorting_1\\\">135</td>\\n</tr><tr id=\\\"pkg_levi\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/levi.html\\\">levi</a></td>\\n\\t<td class=\\\" \\\">Jose Luiz Rybarczyk Filho</td>\\n\\t<td class=\\\" \\\">Landscape Expression Visualization Interface</td>\\n\\t<td class=\\\"  sorting_1\\\">135</td>\\n</tr><tr id=\\\"pkg_lineagespot\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/lineagespot.html\\\">lineagespot</a></td>\\n\\t<td class=\\\" \\\">Nikolaos Pechlivanis</td>\\n\\t<td class=\\\" \\\">Detection of SARS-CoV-2 lineages in wastewater samples using next-generation sequencing</td>\\n\\t<td class=\\\"  sorting_1\\\">135</td>\\n</tr><tr id=\\\"pkg_m6Aboost\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/m6Aboost.html\\\">m6Aboost</a></td>\\n\\t<td class=\\\" \\\">You Zhou</td>\\n\\t<td class=\\\" \\\">m6Aboost</td>\\n\\t<td class=\\\"  sorting_1\\\">135</td>\\n</tr><tr id=\\\"pkg_MQmetrics\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/MQmetrics.html\\\">MQmetrics</a></td>\\n\\t<td class=\\\" \\\">Alvaro Sanchez-Villalba</td>\\n\\t<td class=\\\" \\\">Quality Control of Protemics Data</td>\\n\\t<td class=\\\"  sorting_1\\\">135</td>\\n</tr><tr id=\\\"pkg_MuData\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/MuData.html\\\">MuData</a></td>\\n\\t<td class=\\\" \\\">Danila Bredikhin</td>\\n\\t<td class=\\\" \\\">Serialization for MultiAssayExperiment Objects</td>\\n\\t<td class=\\\"  sorting_1\\\">135</td>\\n</tr><tr id=\\\"pkg_pairkat\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/pairkat.html\\\">pairkat</a></td>\\n\\t<td class=\\\" \\\">Max McGrath</td>\\n\\t<td class=\\\" \\\">PaIRKAT</td>\\n\\t<td class=\\\"  sorting_1\\\">135</td>\\n</tr><tr id=\\\"pkg_PanomiR\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/PanomiR.html\\\">PanomiR</a></td>\\n\\t<td class=\\\" \\\">Pourya Naderi</td>\\n\\t<td class=\\\" \\\">Detection of miRNAs that regulate interacting groups of pathways</td>\\n\\t<td class=\\\"  sorting_1\\\">135</td>\\n</tr><tr id=\\\"pkg_pathVar\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/pathVar.html\\\">pathVar</a></td>\\n\\t<td class=\\\" \\\">Samuel Zimmerman</td>\\n\\t<td class=\\\" \\\">Methods to Find Pathways with Significantly Different Variability</td>\\n\\t<td class=\\\"  sorting_1\\\">135</td>\\n</tr><tr id=\\\"pkg_pengls\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/pengls.html\\\">pengls</a></td>\\n\\t<td class=\\\" \\\">Stijn Hawinkel</td>\\n\\t<td class=\\\" \\\">Fit Penalised Generalised Least Squares models</td>\\n\\t<td class=\\\"  sorting_1\\\">135</td>\\n</tr><tr id=\\\"pkg_scShapes\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/scShapes.html\\\">scShapes</a></td>\\n\\t<td class=\\\" \\\">Malindrie Dharmaratne</td>\\n\\t<td class=\\\" \\\">A Statistical Framework for Modeling and Identifying Differential Distributions in Single-cell RNA-sequencing Data</td>\\n\\t<td class=\\\"  sorting_1\\\">135</td>\\n</tr><tr id=\\\"pkg_SeqGate\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/SeqGate.html\\\">SeqGate</a></td>\\n\\t<td class=\\\" \\\">Stphanie Rialle</td>\\n\\t<td class=\\\" \\\">Filtering of Lowly Expressed Features</td>\\n\\t<td class=\\\"  sorting_1\\\">135</td>\\n</tr><tr id=\\\"pkg_SICtools\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/SICtools.html\\\">SICtools</a></td>\\n\\t<td class=\\\" \\\">Xiaobin Xing</td>\\n\\t<td class=\\\" \\\">Find SNV/Indel differences between two bam files with near relationship</td>\\n\\t<td class=\\\"  sorting_1\\\">135</td>\\n</tr><tr id=\\\"pkg_sSNAPPY\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/sSNAPPY.html\\\">sSNAPPY</a></td>\\n\\t<td class=\\\" \\\">Wenjun Liu</td>\\n\\t<td class=\\\" \\\">Single Sample directioNAl Pathway Perturbation analYsis</td>\\n\\t<td class=\\\"  sorting_1\\\">135</td>\\n</tr><tr id=\\\"pkg_TFARM\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/TFARM.html\\\">TFARM</a></td>\\n\\t<td class=\\\" \\\">Liuba Nausicaa Martino</td>\\n\\t<td class=\\\" \\\">Transcription Factors Association Rules Miner</td>\\n\\t<td class=\\\"  sorting_1\\\">135</td>\\n</tr><tr id=\\\"pkg_VERSO\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/VERSO.html\\\">VERSO</a></td>\\n\\t<td class=\\\" \\\">Davide Maspero</td>\\n\\t<td class=\\\" \\\">Viral Evolution ReconStructiOn (VERSO)</td>\\n\\t<td class=\\\"  sorting_1\\\">135</td>\\n</tr><tr id=\\\"pkg_bandle\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/bandle.html\\\">bandle</a></td>\\n\\t<td class=\\\" \\\">Oliver M. Crook</td>\\n\\t<td class=\\\" \\\">An R package for the Bayesian analysis of differential subcellular localisation experiments</td>\\n\\t<td class=\\\"  sorting_1\\\">153</td>\\n</tr><tr id=\\\"pkg_biodbLipidmaps\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/biodbLipidmaps.html\\\">biodbLipidmaps</a></td>\\n\\t<td class=\\\" \\\">Pierrick Roger</td>\\n\\t<td class=\\\" \\\">biodbLipidmaps, a library for connecting to the Lipidmaps Structure database</td>\\n\\t<td class=\\\"  sorting_1\\\">153</td>\\n</tr><tr id=\\\"pkg_censcyt\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/censcyt.html\\\">censcyt</a></td>\\n\\t<td class=\\\" \\\">Reto Gerber</td>\\n\\t<td class=\\\" \\\">Differential abundance analysis with a right censored covariate in high-dimensional cytometry</td>\\n\\t<td class=\\\"  sorting_1\\\">153</td>\\n</tr><tr id=\\\"pkg_cytoMEM\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/cytoMEM.html\\\">cytoMEM</a></td>\\n\\t<td class=\\\" \\\">Jonathan Irish</td>\\n\\t<td class=\\\" \\\">Marker Enrichment Modeling (MEM)</td>\\n\\t<td class=\\\"  sorting_1\\\">153</td>\\n</tr><tr id=\\\"pkg_DifferentialRegulation\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/DifferentialRegulation.html\\\">DifferentialRegulation</a></td>\\n\\t<td class=\\\" \\\">Simone Tiberi</td>\\n\\t<td class=\\\" \\\">Differentially regulated genes from scRNA-seq data</td>\\n\\t<td class=\\\"  sorting_1\\\">153</td>\\n</tr><tr id=\\\"pkg_epimutacions\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/epimutacions.html\\\">epimutacions</a></td>\\n\\t<td class=\\\" \\\">Dolors Pelegri-Siso</td>\\n\\t<td class=\\\" \\\">Robust outlier identification for DNA methylation data</td>\\n\\t<td class=\\\"  sorting_1\\\">153</td>\\n</tr><tr id=\\\"pkg_extraChIPs\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/extraChIPs.html\\\">extraChIPs</a></td>\\n\\t<td class=\\\" \\\">Stephen Pederson</td>\\n\\t<td class=\\\" \\\">Additional functions for working with ChIP-Seq data</td>\\n\\t<td class=\\\"  sorting_1\\\">153</td>\\n</tr><tr id=\\\"pkg_frenchFISH\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/frenchFISH.html\\\">frenchFISH</a></td>\\n\\t<td class=\\\" \\\">Adam Berman</td>\\n\\t<td class=\\\" \\\">Poisson Models for Quantifying DNA Copy-number from FISH Images of Tissue Sections</td>\\n\\t<td class=\\\"  sorting_1\\\">153</td>\\n</tr><tr id=\\\"pkg_LinTInd\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/LinTInd.html\\\">LinTInd</a></td>\\n\\t<td class=\\\" \\\">Luyue Wang</td>\\n\\t<td class=\\\" \\\">Lineage tracing by indels</td>\\n\\t<td class=\\\"  sorting_1\\\">153</td>\\n</tr><tr id=\\\"pkg_MACSQuantifyR\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/MACSQuantifyR.html\\\">MACSQuantifyR</a></td>\\n\\t<td class=\\\" \\\">Raphal Bonnet</td>\\n\\t<td class=\\\" \\\">Fast treatment of MACSQuantify FACS data</td>\\n\\t<td class=\\\"  sorting_1\\\">153</td>\\n</tr><tr id=\\\"pkg_metavizr\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/metavizr.html\\\">metavizr</a></td>\\n\\t<td class=\\\" \\\">Hector Corrada Bravo</td>\\n\\t<td class=\\\" \\\">R Interface to the metaviz web app for interactive metagenomics data analysis and visualization</td>\\n\\t<td class=\\\"  sorting_1\\\">153</td>\\n</tr><tr id=\\\"pkg_methylscaper\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/methylscaper.html\\\">methylscaper</a></td>\\n\\t<td class=\\\" \\\">Bacher Rhonda</td>\\n\\t<td class=\\\" \\\">Visualization of Methylation Data</td>\\n\\t<td class=\\\"  sorting_1\\\">153</td>\\n</tr><tr id=\\\"pkg_MSstatsLOBD\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/MSstatsLOBD.html\\\">MSstatsLOBD</a></td>\\n\\t<td class=\\\" \\\">Devon Kohler</td>\\n\\t<td class=\\\" \\\">Assay characterization: estimation of limit of blanc(LoB) and limit of detection(LOD)</td>\\n\\t<td class=\\\"  sorting_1\\\">153</td>\\n</tr><tr id=\\\"pkg_multiSight\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/multiSight.html\\\">multiSight</a></td>\\n\\t<td class=\\\" \\\">Florian Jeanneret</td>\\n\\t<td class=\\\" \\\">Multi-omics Classification, Functional Enrichment and Network Inference analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">153</td>\\n</tr><tr id=\\\"pkg_netboost\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/netboost.html\\\">netboost</a></td>\\n\\t<td class=\\\" \\\">Pascal Schlosser</td>\\n\\t<td class=\\\" \\\">Network Analysis Supported by Boosting</td>\\n\\t<td class=\\\"  sorting_1\\\">153</td>\\n</tr><tr id=\\\"pkg_netboxr\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/netboxr.html\\\">netboxr</a></td>\\n\\t<td class=\\\" \\\">Eirc Minwei Liu</td>\\n\\t<td class=\\\" \\\">netboxr</td>\\n\\t<td class=\\\"  sorting_1\\\">153</td>\\n</tr><tr id=\\\"pkg_oneSENSE\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/oneSENSE.html\\\">oneSENSE</a></td>\\n\\t<td class=\\\" \\\">Yong Kee Tan</td>\\n\\t<td class=\\\" \\\">One-Dimensional Soli-Expression by Nonlinear Stochastic Embedding (OneSENSE)</td>\\n\\t<td class=\\\"  sorting_1\\\">153</td>\\n</tr><tr id=\\\"pkg_optimalFlow\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/optimalFlow.html\\\">optimalFlow</a></td>\\n\\t<td class=\\\" \\\">Hristo Inouzhe</td>\\n\\t<td class=\\\" \\\">optimalFlow</td>\\n\\t<td class=\\\"  sorting_1\\\">153</td>\\n</tr><tr id=\\\"pkg_PhenoGeneRanker\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/PhenoGeneRanker.html\\\">PhenoGeneRanker</a></td>\\n\\t<td class=\\\" \\\">Cagatay Dursun</td>\\n\\t<td class=\\\" \\\">PhenoGeneRanker: A gene and phenotype prioritization tool</td>\\n\\t<td class=\\\"  sorting_1\\\">153</td>\\n</tr><tr id=\\\"pkg_receptLoss\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/receptLoss.html\\\">receptLoss</a></td>\\n\\t<td class=\\\" \\\">Daniel Pique</td>\\n\\t<td class=\\\" \\\">Unsupervised Identification of Genes with Expression Loss in Subsets of Tumors</td>\\n\\t<td class=\\\"  sorting_1\\\">153</td>\\n</tr><tr id=\\\"pkg_RNAsense\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/RNAsense.html\\\">RNAsense</a></td>\\n\\t<td class=\\\" \\\">Marcus Rosenblatt</td>\\n\\t<td class=\\\" \\\">Analysis of Time-Resolved RNA-Seq Data</td>\\n\\t<td class=\\\"  sorting_1\\\">153</td>\\n</tr><tr id=\\\"pkg_ROCpAI\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ROCpAI.html\\\">ROCpAI</a></td>\\n\\t<td class=\\\" \\\">Juan-Pedro Garcia</td>\\n\\t<td class=\\\" \\\">Receiver Operating Characteristic Partial Area Indexes for evaluating classifiers</td>\\n\\t<td class=\\\"  sorting_1\\\">153</td>\\n</tr><tr id=\\\"pkg_rsemmed\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/rsemmed.html\\\">rsemmed</a></td>\\n\\t<td class=\\\" \\\">Leslie Myint</td>\\n\\t<td class=\\\" \\\">An interface to the Semantic MEDLINE database</td>\\n\\t<td class=\\\"  sorting_1\\\">153</td>\\n</tr><tr id=\\\"pkg_sarks\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/sarks.html\\\">sarks</a></td>\\n\\t<td class=\\\" \\\">Dennis Wylie</td>\\n\\t<td class=\\\" \\\">Suffix Array Kernel Smoothing for discovery of correlative sequence motifs and multi-motif domains</td>\\n\\t<td class=\\\"  sorting_1\\\">153</td>\\n</tr><tr id=\\\"pkg_scanMiRApp\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/scanMiRApp.html\\\">scanMiRApp</a></td>\\n\\t<td class=\\\" \\\">Pierre-Luc Germain</td>\\n\\t<td class=\\\" \\\">scanMiR shiny application</td>\\n\\t<td class=\\\"  sorting_1\\\">153</td>\\n</tr><tr id=\\\"pkg_selectKSigs\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/selectKSigs.html\\\">selectKSigs</a></td>\\n\\t<td class=\\\" \\\">Zhi Yang</td>\\n\\t<td class=\\\" \\\">Selecting the number of mutational signatures using a perplexity-based measure and cross-validation</td>\\n\\t<td class=\\\"  sorting_1\\\">153</td>\\n</tr><tr id=\\\"pkg_SEPIRA\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/SEPIRA.html\\\">SEPIRA</a></td>\\n\\t<td class=\\\" \\\">Yuting Chen</td>\\n\\t<td class=\\\" \\\">Systems EPigenomics Inference of Regulatory Activity</td>\\n\\t<td class=\\\"  sorting_1\\\">153</td>\\n</tr><tr id=\\\"pkg_SMAD\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/SMAD.html\\\">SMAD</a></td>\\n\\t<td class=\\\" \\\">Qingzhou Zhang</td>\\n\\t<td class=\\\" \\\">Statistical Modelling of AP-MS Data (SMAD)</td>\\n\\t<td class=\\\"  sorting_1\\\">153</td>\\n</tr><tr id=\\\"pkg_sojourner\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/sojourner.html\\\">sojourner</a></td>\\n\\t<td class=\\\" \\\">Sojourner Developer</td>\\n\\t<td class=\\\" \\\">Statistical analysis of single molecule trajectories</td>\\n\\t<td class=\\\"  sorting_1\\\">153</td>\\n</tr><tr id=\\\"pkg_SOMNiBUS\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/SOMNiBUS.html\\\">SOMNiBUS</a></td>\\n\\t<td class=\\\" \\\">Kathleen Klein</td>\\n\\t<td class=\\\" \\\">Smooth modeling of bisulfite sequencing</td>\\n\\t<td class=\\\"  sorting_1\\\">153</td>\\n</tr><tr id=\\\"pkg_sparsenetgls\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/sparsenetgls.html\\\">sparsenetgls</a></td>\\n\\t<td class=\\\" \\\">Irene Zeng</td>\\n\\t<td class=\\\" \\\">Using Gaussian graphical structue learning estimation in generalized least squared regression for multivariate normal regression</td>\\n\\t<td class=\\\"  sorting_1\\\">153</td>\\n</tr><tr id=\\\"pkg_SpatialFeatureExperiment\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/SpatialFeatureExperiment.html\\\">SpatialFeatureExperiment</a></td>\\n\\t<td class=\\\" \\\">Lambda Moses</td>\\n\\t<td class=\\\" \\\">Integrating SpatialExperiment with Simple Features in sf</td>\\n\\t<td class=\\\"  sorting_1\\\">153</td>\\n</tr><tr id=\\\"pkg_SQUADD\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/SQUADD.html\\\">SQUADD</a></td>\\n\\t<td class=\\\" \\\">Martial Sankar</td>\\n\\t<td class=\\\" \\\">Add-on of the SQUAD Software</td>\\n\\t<td class=\\\"  sorting_1\\\">153</td>\\n</tr><tr id=\\\"pkg_ssPATHS\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ssPATHS.html\\\">ssPATHS</a></td>\\n\\t<td class=\\\" \\\">Natalie R. Davidson</td>\\n\\t<td class=\\\" \\\">ssPATHS: Single Sample PATHway Score</td>\\n\\t<td class=\\\"  sorting_1\\\">153</td>\\n</tr><tr id=\\\"pkg_synapsis\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/synapsis.html\\\">synapsis</a></td>\\n\\t<td class=\\\" \\\">Lucy McNeill</td>\\n\\t<td class=\\\" \\\">An R package to automate the analysis of double-strand break repair during meiosis</td>\\n\\t<td class=\\\"  sorting_1\\\">153</td>\\n</tr><tr id=\\\"pkg_syntenet\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/syntenet.html\\\">syntenet</a></td>\\n\\t<td class=\\\" \\\">Fabrcio Almeida-Silva</td>\\n\\t<td class=\\\" \\\">Inference And Analysis Of Synteny Networks</td>\\n\\t<td class=\\\"  sorting_1\\\">153</td>\\n</tr><tr id=\\\"pkg_tenXplore\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/tenXplore.html\\\">tenXplore</a></td>\\n\\t<td class=\\\" \\\">VJ Carey</td>\\n\\t<td class=\\\" \\\">ontological exploration of scRNA-seq of 1.3 million mouse neurons from 10x genomics</td>\\n\\t<td class=\\\"  sorting_1\\\">153</td>\\n</tr><tr id=\\\"pkg_txcutr\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/txcutr.html\\\">txcutr</a></td>\\n\\t<td class=\\\" \\\">Mervin Fansler</td>\\n\\t<td class=\\\" \\\">Transcriptome CUTteR</td>\\n\\t<td class=\\\"  sorting_1\\\">153</td>\\n</tr><tr id=\\\"pkg_uncoverappLib\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/uncoverappLib.html\\\">uncoverappLib</a></td>\\n\\t<td class=\\\" \\\">Emanuela Iovino</td>\\n\\t<td class=\\\" \\\">Interactive graphical application for clinical assessment of sequence coverage at the base-pair level</td>\\n\\t<td class=\\\"  sorting_1\\\">153</td>\\n</tr><tr id=\\\"pkg_coMethDMR\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/coMethDMR.html\\\">coMethDMR</a></td>\\n\\t<td class=\\\" \\\">Fernanda Veitzman</td>\\n\\t<td class=\\\" \\\">Accurate identification of co-methylated and differentially methylated regions in epigenome-wide association studies</td>\\n\\t<td class=\\\"  sorting_1\\\">192</td>\\n</tr><tr id=\\\"pkg_dks\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/dks.html\\\">dks</a></td>\\n\\t<td class=\\\" \\\">Jeffrey T. Leek</td>\\n\\t<td class=\\\" \\\">The double Kolmogorov-Smirnov package for evaluating multiple testing procedures.</td>\\n\\t<td class=\\\"  sorting_1\\\">192</td>\\n</tr><tr id=\\\"pkg_EnMCB\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/EnMCB.html\\\">EnMCB</a></td>\\n\\t<td class=\\\" \\\">Xin Yu</td>\\n\\t<td class=\\\" \\\">Predicting Disease Progression Based on Methylation Correlated Blocks using Ensemble Models</td>\\n\\t<td class=\\\"  sorting_1\\\">192</td>\\n</tr><tr id=\\\"pkg_epidecodeR\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/epidecodeR.html\\\">epidecodeR</a></td>\\n\\t<td class=\\\" \\\">Kandarp Joshi</td>\\n\\t<td class=\\\" \\\">epidecodeR: a functional exploration tool for epigenetic and epitranscriptomic regulation</td>\\n\\t<td class=\\\"  sorting_1\\\">192</td>\\n</tr><tr id=\\\"pkg_GeneAccord\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/GeneAccord.html\\\">GeneAccord</a></td>\\n\\t<td class=\\\" \\\">Ariane L. Moore</td>\\n\\t<td class=\\\" \\\">Detection of clonally exclusive gene or pathway pairs in a cohort of cancer patients</td>\\n\\t<td class=\\\"  sorting_1\\\">192</td>\\n</tr><tr id=\\\"pkg_hierinf\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/hierinf.html\\\">hierinf</a></td>\\n\\t<td class=\\\" \\\">Claude Renaux</td>\\n\\t<td class=\\\" \\\">Hierarchical Inference</td>\\n\\t<td class=\\\"  sorting_1\\\">192</td>\\n</tr><tr id=\\\"pkg_iSEEhex\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/iSEEhex.html\\\">iSEEhex</a></td>\\n\\t<td class=\\\" \\\">Kevin Rue-Albrecht</td>\\n\\t<td class=\\\" \\\">iSEE extension for summarising data points in hexagonal bins</td>\\n\\t<td class=\\\"  sorting_1\\\">192</td>\\n</tr><tr id=\\\"pkg_LinkHD\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/LinkHD.html\\\">LinkHD</a></td>\\n\\t<td class=\\\" \\\">\\\"Laura M Zingaretti\\\"</td>\\n\\t<td class=\\\" \\\">LinkHD: a versatile framework to explore and integrate heterogeneous data</td>\\n\\t<td class=\\\"  sorting_1\\\">192</td>\\n</tr><tr id=\\\"pkg_LowMACA\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/LowMACA.html\\\">LowMACA</a></td>\\n\\t<td class=\\\" \\\">Giorgio Melloni , Stefano de Pretis</td>\\n\\t<td class=\\\" \\\">LowMACA - Low frequency Mutation Analysis via Consensus Alignment</td>\\n\\t<td class=\\\"  sorting_1\\\">192</td>\\n</tr><tr id=\\\"pkg_marr\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/marr.html\\\">marr</a></td>\\n\\t<td class=\\\" \\\">Tusharkanti Ghosh</td>\\n\\t<td class=\\\" \\\">Maximum rank reproducibility</td>\\n\\t<td class=\\\"  sorting_1\\\">192</td>\\n</tr><tr id=\\\"pkg_MBttest\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/MBttest.html\\\">MBttest</a></td>\\n\\t<td class=\\\" \\\">Yuan-De Tan</td>\\n\\t<td class=\\\" \\\">Multiple Beta t-Tests</td>\\n\\t<td class=\\\"  sorting_1\\\">192</td>\\n</tr><tr id=\\\"pkg_MEB\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/MEB.html\\\">MEB</a></td>\\n\\t<td class=\\\" \\\">Jiadi Zhu, Yan Zhou</td>\\n\\t<td class=\\\" \\\">A normalization-invariant minimum enclosing ball method to detect differentially expressed genes for RNA-seq data</td>\\n\\t<td class=\\\"  sorting_1\\\">192</td>\\n</tr><tr id=\\\"pkg_MouseFM\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/MouseFM.html\\\">MouseFM</a></td>\\n\\t<td class=\\\" \\\">Matthias Munz</td>\\n\\t<td class=\\\" \\\">In-silico methods for genetic finemapping in inbred mice</td>\\n\\t<td class=\\\"  sorting_1\\\">192</td>\\n</tr><tr id=\\\"pkg_occugene\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/occugene.html\\\">occugene</a></td>\\n\\t<td class=\\\" \\\">Oliver Will</td>\\n\\t<td class=\\\" \\\">Functions for Multinomial Occupancy Distribution</td>\\n\\t<td class=\\\"  sorting_1\\\">192</td>\\n</tr><tr id=\\\"pkg_OVESEG\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/OVESEG.html\\\">OVESEG</a></td>\\n\\t<td class=\\\" \\\">Lulu Chen</td>\\n\\t<td class=\\\" \\\">OVESEG-test to detect tissue/cell-specific markers</td>\\n\\t<td class=\\\"  sorting_1\\\">192</td>\\n</tr><tr id=\\\"pkg_ppcseq\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ppcseq.html\\\">ppcseq</a></td>\\n\\t<td class=\\\" \\\">Stefano Mangiola</td>\\n\\t<td class=\\\" \\\">Probabilistic Outlier Identification for RNA Sequencing Generalized Linear Models</td>\\n\\t<td class=\\\"  sorting_1\\\">192</td>\\n</tr><tr id=\\\"pkg_profileScoreDist\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/profileScoreDist.html\\\">profileScoreDist</a></td>\\n\\t<td class=\\\" \\\">Paal O. Westermark</td>\\n\\t<td class=\\\" \\\">Profile score distributions</td>\\n\\t<td class=\\\"  sorting_1\\\">192</td>\\n</tr><tr id=\\\"pkg_proteinProfiles\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/proteinProfiles.html\\\">proteinProfiles</a></td>\\n\\t<td class=\\\" \\\">Julian Gehring</td>\\n\\t<td class=\\\" \\\">Protein Profiling</td>\\n\\t<td class=\\\"  sorting_1\\\">192</td>\\n</tr><tr id=\\\"pkg_ReactomeGraph4R\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ReactomeGraph4R.html\\\">ReactomeGraph4R</a></td>\\n\\t<td class=\\\" \\\">Chi-Lam Poon</td>\\n\\t<td class=\\\" \\\">Interface for the Reactome Graph Database</td>\\n\\t<td class=\\\"  sorting_1\\\">192</td>\\n</tr><tr id=\\\"pkg_regsplice\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/regsplice.html\\\">regsplice</a></td>\\n\\t<td class=\\\" \\\">Lukas M. Weber</td>\\n\\t<td class=\\\" \\\">L1-regularization based methods for detection of differential splicing</td>\\n\\t<td class=\\\"  sorting_1\\\">192</td>\\n</tr><tr id=\\\"pkg_RiboCrypt\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/RiboCrypt.html\\\">RiboCrypt</a></td>\\n\\t<td class=\\\" \\\">Michal Swirski</td>\\n\\t<td class=\\\" \\\">Interactive visualization in genomics</td>\\n\\t<td class=\\\"  sorting_1\\\">192</td>\\n</tr><tr id=\\\"pkg_RIPAT\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/RIPAT.html\\\">RIPAT</a></td>\\n\\t<td class=\\\" \\\">Min-Jeong Baek</td>\\n\\t<td class=\\\" \\\">Retroviral Integration Pattern Analysis Tool (RIPAT)</td>\\n\\t<td class=\\\"  sorting_1\\\">192</td>\\n</tr><tr id=\\\"pkg_RNAdecay\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/RNAdecay.html\\\">RNAdecay</a></td>\\n\\t<td class=\\\" \\\">Reed Sorenson</td>\\n\\t<td class=\\\" \\\">Maximum Likelihood Decay Modeling of RNA Degradation Data</td>\\n\\t<td class=\\\"  sorting_1\\\">192</td>\\n</tr><tr id=\\\"pkg_rnaEditr\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/rnaEditr.html\\\">rnaEditr</a></td>\\n\\t<td class=\\\" \\\">Lanyu Zhang</td>\\n\\t<td class=\\\" \\\">Statistical analysis of RNA editing sites and hyper-editing regions</td>\\n\\t<td class=\\\"  sorting_1\\\">192</td>\\n</tr><tr id=\\\"pkg_Sconify\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/Sconify.html\\\">Sconify</a></td>\\n\\t<td class=\\\" \\\">Tyler J Burns</td>\\n\\t<td class=\\\" \\\">A toolkit for performing KNN-based statistics for flow and mass cytometry data</td>\\n\\t<td class=\\\"  sorting_1\\\">192</td>\\n</tr><tr id=\\\"pkg_scReClassify\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/scReClassify.html\\\">scReClassify</a></td>\\n\\t<td class=\\\" \\\">Taiyun Kim</td>\\n\\t<td class=\\\" \\\">scReClassify: post hoc cell type classification of single-cell RNA-seq data</td>\\n\\t<td class=\\\"  sorting_1\\\">192</td>\\n</tr><tr id=\\\"pkg_segmenter\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/segmenter.html\\\">segmenter</a></td>\\n\\t<td class=\\\" \\\">Mahmoud Ahmed</td>\\n\\t<td class=\\\" \\\">Perform Chromatin Segmentation Analysis in R by Calling ChromHMM</td>\\n\\t<td class=\\\"  sorting_1\\\">192</td>\\n</tr><tr id=\\\"pkg_semisup\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/semisup.html\\\">semisup</a></td>\\n\\t<td class=\\\" \\\">Armin Rauschenberger</td>\\n\\t<td class=\\\" \\\">Semi-Supervised Mixture Model</td>\\n\\t<td class=\\\"  sorting_1\\\">192</td>\\n</tr><tr id=\\\"pkg_spatzie\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/spatzie.html\\\">spatzie</a></td>\\n\\t<td class=\\\" \\\">Jennifer Hammelman</td>\\n\\t<td class=\\\" \\\">Identification of enriched motif pairs from chromatin interaction data</td>\\n\\t<td class=\\\"  sorting_1\\\">192</td>\\n</tr><tr id=\\\"pkg_spikeLI\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/spikeLI.html\\\">spikeLI</a></td>\\n\\t<td class=\\\" \\\">Enrico Carlon</td>\\n\\t<td class=\\\" \\\">Affymetrix Spike-in Langmuir Isotherm Data Analysis Tool</td>\\n\\t<td class=\\\"  sorting_1\\\">192</td>\\n</tr><tr id=\\\"pkg_SplicingFactory\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/SplicingFactory.html\\\">SplicingFactory</a></td>\\n\\t<td class=\\\" \\\">Endre Sebestyen</td>\\n\\t<td class=\\\" \\\">Splicing Diversity Analysis for Transcriptome Data</td>\\n\\t<td class=\\\"  sorting_1\\\">192</td>\\n</tr><tr id=\\\"pkg_surfaltr\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/surfaltr.html\\\">surfaltr</a></td>\\n\\t<td class=\\\" \\\">Pooja Gangras</td>\\n\\t<td class=\\\" \\\">Rapid Comparison of Surface Protein Isoform Membrane Topologies Through surfaltr</td>\\n\\t<td class=\\\"  sorting_1\\\">192</td>\\n</tr><tr id=\\\"pkg_TDARACNE\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/TDARACNE.html\\\">TDARACNE</a></td>\\n\\t<td class=\\\" \\\">Zoppoli Pietro</td>\\n\\t<td class=\\\" \\\">Network reverse engineering from time course data.</td>\\n\\t<td class=\\\"  sorting_1\\\">192</td>\\n</tr><tr id=\\\"pkg_TFHAZ\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/TFHAZ.html\\\">TFHAZ</a></td>\\n\\t<td class=\\\" \\\">Gaia Ceddia</td>\\n\\t<td class=\\\" \\\">Transcription Factor High Accumulation Zones</td>\\n\\t<td class=\\\"  sorting_1\\\">192</td>\\n</tr><tr id=\\\"pkg_TimeSeriesExperiment\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/TimeSeriesExperiment.html\\\">TimeSeriesExperiment</a></td>\\n\\t<td class=\\\" \\\">Lan Huong Nguyen</td>\\n\\t<td class=\\\" \\\">Analysis for short time-series data</td>\\n\\t<td class=\\\"  sorting_1\\\">192</td>\\n</tr><tr id=\\\"pkg_transomics2cytoscape\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/transomics2cytoscape.html\\\">transomics2cytoscape</a></td>\\n\\t<td class=\\\" \\\">Kozo Nishida</td>\\n\\t<td class=\\\" \\\">A tool set for 3D Trans-Omic network visualization with Cytoscape</td>\\n\\t<td class=\\\"  sorting_1\\\">192</td>\\n</tr><tr id=\\\"pkg_traviz\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/traviz.html\\\">traviz</a></td>\\n\\t<td class=\\\" \\\">Koen Van den Berge</td>\\n\\t<td class=\\\" \\\">Trajectory functions for visualization and interpretation.</td>\\n\\t<td class=\\\"  sorting_1\\\">192</td>\\n</tr><tr id=\\\"pkg_TTMap\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/TTMap.html\\\">TTMap</a></td>\\n\\t<td class=\\\" \\\">Rachel Jeitziner</td>\\n\\t<td class=\\\" \\\">Two-Tier Mapper: a clustering tool based on topological data analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">192</td>\\n</tr><tr id=\\\"pkg_beer\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/beer.html\\\">beer</a></td>\\n\\t<td class=\\\" \\\">Athena Chen</td>\\n\\t<td class=\\\" \\\">Bayesian Enrichment Estimation in R</td>\\n\\t<td class=\\\"  sorting_1\\\">230</td>\\n</tr><tr id=\\\"pkg_CAEN\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/CAEN.html\\\">CAEN</a></td>\\n\\t<td class=\\\" \\\">Zhou Yan</td>\\n\\t<td class=\\\" \\\">Category encoding method for selecting feature genes for the classification of single-cell RNA-seq</td>\\n\\t<td class=\\\"  sorting_1\\\">230</td>\\n</tr><tr id=\\\"pkg_ceRNAnetsim\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ceRNAnetsim.html\\\">ceRNAnetsim</a></td>\\n\\t<td class=\\\" \\\">Selcen Ari Yuka</td>\\n\\t<td class=\\\" \\\">Regulation Simulator of Interaction between miRNA and Competing RNAs (ceRNA)</td>\\n\\t<td class=\\\"  sorting_1\\\">230</td>\\n</tr><tr id=\\\"pkg_crisprDesign\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/crisprDesign.html\\\">crisprDesign</a></td>\\n\\t<td class=\\\" \\\">Jean-Philippe Fortin</td>\\n\\t<td class=\\\" \\\">Comprehensive design of CRISPR gRNAs for nucleases and base editors</td>\\n\\t<td class=\\\"  sorting_1\\\">230</td>\\n</tr><tr id=\\\"pkg_deltaCaptureC\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/deltaCaptureC.html\\\">deltaCaptureC</a></td>\\n\\t<td class=\\\" \\\">Michael Shapiro</td>\\n\\t<td class=\\\" \\\">This Package Discovers Meso-scale Chromatin Remodeling from 3C Data</td>\\n\\t<td class=\\\"  sorting_1\\\">230</td>\\n</tr><tr id=\\\"pkg_iCheck\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/iCheck.html\\\">iCheck</a></td>\\n\\t<td class=\\\" \\\">Weiliang Qiu</td>\\n\\t<td class=\\\" \\\">QC Pipeline and Data Analysis Tools for High-Dimensional Illumina mRNA Expression Data</td>\\n\\t<td class=\\\"  sorting_1\\\">230</td>\\n</tr><tr id=\\\"pkg_INDEED\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/INDEED.html\\\">INDEED</a></td>\\n\\t<td class=\\\" \\\">Ressom group, Yiming Zuo</td>\\n\\t<td class=\\\" \\\">Interactive Visualization of Integrated Differential Expression and Differential Network Analysis for Biomarker Candidate Selection Package</td>\\n\\t<td class=\\\"  sorting_1\\\">230</td>\\n</tr><tr id=\\\"pkg_iPath\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/iPath.html\\\">iPath</a></td>\\n\\t<td class=\\\" \\\">Kenong Su</td>\\n\\t<td class=\\\" \\\">iPath pipeline for detecting perturbed pathways at individual level</td>\\n\\t<td class=\\\"  sorting_1\\\">230</td>\\n</tr><tr id=\\\"pkg_LedPred\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/LedPred.html\\\">LedPred</a></td>\\n\\t<td class=\\\" \\\">Aitor Gonzalez</td>\\n\\t<td class=\\\" \\\">Learning from DNA to Predict Enhancers</td>\\n\\t<td class=\\\"  sorting_1\\\">230</td>\\n</tr><tr id=\\\"pkg_loci2path\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/loci2path.html\\\">loci2path</a></td>\\n\\t<td class=\\\" \\\">Tianlei Xu</td>\\n\\t<td class=\\\" \\\">Loci2path: regulatory annotation of genomic intervals based on tissue-specific expression QTLs</td>\\n\\t<td class=\\\"  sorting_1\\\">230</td>\\n</tr><tr id=\\\"pkg_mapscape\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/mapscape.html\\\">mapscape</a></td>\\n\\t<td class=\\\" \\\">Maia Smith</td>\\n\\t<td class=\\\" \\\">mapscape</td>\\n\\t<td class=\\\"  sorting_1\\\">230</td>\\n</tr><tr id=\\\"pkg_MBAmethyl\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/MBAmethyl.html\\\">MBAmethyl</a></td>\\n\\t<td class=\\\" \\\">Tao Wang</td>\\n\\t<td class=\\\" \\\">Model-based analysis of DNA methylation data</td>\\n\\t<td class=\\\"  sorting_1\\\">230</td>\\n</tr><tr id=\\\"pkg_MiChip\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/MiChip.html\\\">MiChip</a></td>\\n\\t<td class=\\\" \\\">Jonathon Blake</td>\\n\\t<td class=\\\" \\\">MiChip Parsing and Summarizing Functions</td>\\n\\t<td class=\\\"  sorting_1\\\">230</td>\\n</tr><tr id=\\\"pkg_miRcomp\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/miRcomp.html\\\">miRcomp</a></td>\\n\\t<td class=\\\" \\\">Matthew N. McCall</td>\\n\\t<td class=\\\" \\\">Tools to assess and compare miRNA expression estimatation methods</td>\\n\\t<td class=\\\"  sorting_1\\\">230</td>\\n</tr><tr id=\\\"pkg_miRmine\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/miRmine.html\\\">miRmine</a></td>\\n\\t<td class=\\\" \\\">Dusan Randjelovic</td>\\n\\t<td class=\\\" \\\">Data package with miRNA-seq datasets from miRmine database as RangedSummarizedExperiment</td>\\n\\t<td class=\\\"  sorting_1\\\">230</td>\\n</tr><tr id=\\\"pkg_mitoClone2\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/mitoClone2.html\\\">mitoClone2</a></td>\\n\\t<td class=\\\" \\\">Benjamin Story</td>\\n\\t<td class=\\\" \\\">Clonal Population Identification in Single-Cell RNA-Seq Data using Mitochondrial and Somatic Mutations</td>\\n\\t<td class=\\\"  sorting_1\\\">230</td>\\n</tr><tr id=\\\"pkg_MPFE\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/MPFE.html\\\">MPFE</a></td>\\n\\t<td class=\\\" \\\">Conrad Burden</td>\\n\\t<td class=\\\" \\\">Estimation of the amplicon methylation pattern distribution from bisulphite sequencing data</td>\\n\\t<td class=\\\"  sorting_1\\\">230</td>\\n</tr><tr id=\\\"pkg_NeighborNet\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/NeighborNet.html\\\">NeighborNet</a></td>\\n\\t<td class=\\\" \\\">Sahar Ansari</td>\\n\\t<td class=\\\" \\\">Neighbor_net analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">230</td>\\n</tr><tr id=\\\"pkg_NeuCA\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/NeuCA.html\\\">NeuCA</a></td>\\n\\t<td class=\\\" \\\">Hao Feng</td>\\n\\t<td class=\\\" \\\">NEUral network-based single-Cell Annotation tool</td>\\n\\t<td class=\\\"  sorting_1\\\">230</td>\\n</tr><tr id=\\\"pkg_NTW\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/NTW.html\\\">NTW</a></td>\\n\\t<td class=\\\" \\\">Yuanhua Liu</td>\\n\\t<td class=\\\" \\\">Predict gene network using an Ordinary Differential Equation (ODE) based method</td>\\n\\t<td class=\\\"  sorting_1\\\">230</td>\\n</tr><tr id=\\\"pkg_oppti\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/oppti.html\\\">oppti</a></td>\\n\\t<td class=\\\" \\\">Abdulkadir Elmas</td>\\n\\t<td class=\\\" \\\">Outlier Protein and Phosphosite Target Identifier</td>\\n\\t<td class=\\\"  sorting_1\\\">230</td>\\n</tr><tr id=\\\"pkg_peco\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/peco.html\\\">peco</a></td>\\n\\t<td class=\\\" \\\">Chiaowen Joyce Hsiao</td>\\n\\t<td class=\\\" \\\">A Supervised Approach for **P**r**e**dicting **c**ell Cycle Pr**o**gression using scRNA-seq data</td>\\n\\t<td class=\\\"  sorting_1\\\">230</td>\\n</tr><tr id=\\\"pkg_pmm\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/pmm.html\\\">pmm</a></td>\\n\\t<td class=\\\" \\\">Anna Drewek</td>\\n\\t<td class=\\\" \\\">Parallel Mixed Model</td>\\n\\t<td class=\\\"  sorting_1\\\">230</td>\\n</tr><tr id=\\\"pkg_preciseTAD\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/preciseTAD.html\\\">preciseTAD</a></td>\\n\\t<td class=\\\" \\\">Mikhail Dozmorov</td>\\n\\t<td class=\\\" \\\">preciseTAD: A machine learning framework for precise TAD boundary prediction</td>\\n\\t<td class=\\\"  sorting_1\\\">230</td>\\n</tr><tr id=\\\"pkg_RCASPAR\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/RCASPAR.html\\\">RCASPAR</a></td>\\n\\t<td class=\\\" \\\">Douaa Mugahid, Lars Kaderali</td>\\n\\t<td class=\\\" \\\">A package for survival time prediction based on a piecewise baseline hazard Cox regression model.</td>\\n\\t<td class=\\\"  sorting_1\\\">230</td>\\n</tr><tr id=\\\"pkg_reconsi\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/reconsi.html\\\">reconsi</a></td>\\n\\t<td class=\\\" \\\">Stijn Hawinkel</td>\\n\\t<td class=\\\" \\\">Resampling Collapsed Null Distributions for Simultaneous Inference</td>\\n\\t<td class=\\\"  sorting_1\\\">230</td>\\n</tr><tr id=\\\"pkg_rgoslin\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/rgoslin.html\\\">rgoslin</a></td>\\n\\t<td class=\\\" \\\">Nils Hoffmann</td>\\n\\t<td class=\\\" \\\">Lipid Shorthand Name Parsing and Normalization</td>\\n\\t<td class=\\\"  sorting_1\\\">230</td>\\n</tr><tr id=\\\"pkg_RIVER\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/RIVER.html\\\">RIVER</a></td>\\n\\t<td class=\\\" \\\">Yungil Kim</td>\\n\\t<td class=\\\" \\\">R package for RIVER (RNA-Informed Variant Effect on Regulation)</td>\\n\\t<td class=\\\"  sorting_1\\\">230</td>\\n</tr><tr id=\\\"pkg_rSWeeP\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/rSWeeP.html\\\">rSWeeP</a></td>\\n\\t<td class=\\\" \\\">Danrley R. Fernandes</td>\\n\\t<td class=\\\" \\\">Functions to creation of low dimensional comparative matrices of Amino Acid Sequence occurrences</td>\\n\\t<td class=\\\"  sorting_1\\\">230</td>\\n</tr><tr id=\\\"pkg_Rtpca\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/Rtpca.html\\\">Rtpca</a></td>\\n\\t<td class=\\\" \\\">Nils Kurzawa</td>\\n\\t<td class=\\\" \\\">Thermal proximity co-aggregation with R</td>\\n\\t<td class=\\\"  sorting_1\\\">230</td>\\n</tr><tr id=\\\"pkg_rTRMui\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/rTRMui.html\\\">rTRMui</a></td>\\n\\t<td class=\\\" \\\">Diego Diez</td>\\n\\t<td class=\\\" \\\">A shiny user interface for rTRM</td>\\n\\t<td class=\\\"  sorting_1\\\">230</td>\\n</tr><tr id=\\\"pkg_sagenhaft\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/sagenhaft.html\\\">sagenhaft</a></td>\\n\\t<td class=\\\" \\\">Tim Beissbarth</td>\\n\\t<td class=\\\" \\\">Collection of functions for reading and comparing SAGE libraries</td>\\n\\t<td class=\\\"  sorting_1\\\">230</td>\\n</tr><tr id=\\\"pkg_Scale4C\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/Scale4C.html\\\">Scale4C</a></td>\\n\\t<td class=\\\" \\\">Carolin Walter</td>\\n\\t<td class=\\\" \\\">Scale4C: an R/Bioconductor package for scale-space transformation of 4C-seq data</td>\\n\\t<td class=\\\"  sorting_1\\\">230</td>\\n</tr><tr id=\\\"pkg_SCFA\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/SCFA.html\\\">SCFA</a></td>\\n\\t<td class=\\\" \\\">Duc Tran</td>\\n\\t<td class=\\\" \\\">SCFA: Subtyping via Consensus Factor Analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">230</td>\\n</tr><tr id=\\\"pkg_scFeatureFilter\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/scFeatureFilter.html\\\">scFeatureFilter</a></td>\\n\\t<td class=\\\" \\\">Guillaume Devailly</td>\\n\\t<td class=\\\" \\\">A correlation-based method for quality filtering of single-cell RNAseq data</td>\\n\\t<td class=\\\"  sorting_1\\\">230</td>\\n</tr><tr id=\\\"pkg_scTreeViz\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/scTreeViz.html\\\">scTreeViz</a></td>\\n\\t<td class=\\\" \\\">Jayaram Kancherla</td>\\n\\t<td class=\\\" \\\">R/Bioconductor package to interactively explore and visualize single cell RNA-seq datasets with hierarhical annotations</td>\\n\\t<td class=\\\"  sorting_1\\\">230</td>\\n</tr><tr id=\\\"pkg_SIMD\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/SIMD.html\\\">SIMD</a></td>\\n\\t<td class=\\\" \\\">Jiadi Zhu</td>\\n\\t<td class=\\\" \\\">Statistical Inferences with MeDIP-seq Data (SIMD) to infer the methylation level for each CpG site</td>\\n\\t<td class=\\\"  sorting_1\\\">230</td>\\n</tr><tr id=\\\"pkg_SimFFPE\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/SimFFPE.html\\\">SimFFPE</a></td>\\n\\t<td class=\\\" \\\">Lanying Wei</td>\\n\\t<td class=\\\" \\\">NGS Read Simulator for FFPE Tissue</td>\\n\\t<td class=\\\"  sorting_1\\\">230</td>\\n</tr><tr id=\\\"pkg_sitadela\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/sitadela.html\\\">sitadela</a></td>\\n\\t<td class=\\\" \\\">Panagiotis Moulos</td>\\n\\t<td class=\\\" \\\">An R package for the easy provision of simple but complete tab-delimited genomic annotation from a variety of sources and organisms</td>\\n\\t<td class=\\\"  sorting_1\\\">230</td>\\n</tr><tr id=\\\"pkg_SPLINTER\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/SPLINTER.html\\\">SPLINTER</a></td>\\n\\t<td class=\\\" \\\">Diana Low</td>\\n\\t<td class=\\\" \\\">Splice Interpreter of Transcripts</td>\\n\\t<td class=\\\"  sorting_1\\\">230</td>\\n</tr><tr id=\\\"pkg_ToxicoGx\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ToxicoGx.html\\\">ToxicoGx</a></td>\\n\\t<td class=\\\" \\\">Benjamin Haibe-Kains</td>\\n\\t<td class=\\\" \\\">Analysis of Large-Scale Toxico-Genomic Data</td>\\n\\t<td class=\\\"  sorting_1\\\">230</td>\\n</tr><tr id=\\\"pkg_tripr\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/tripr.html\\\">tripr</a></td>\\n\\t<td class=\\\" \\\">Nikolaos Pechlivanis</td>\\n\\t<td class=\\\" \\\">T-cell Receptor/Immunoglobulin Profiler (TRIP)</td>\\n\\t<td class=\\\"  sorting_1\\\">230</td>\\n</tr><tr id=\\\"pkg_BUMHMM\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/BUMHMM.html\\\">BUMHMM</a></td>\\n\\t<td class=\\\" \\\">Alina Selega</td>\\n\\t<td class=\\\" \\\">Computational pipeline for computing probability of modification from structure probing experiment data</td>\\n\\t<td class=\\\"  sorting_1\\\">272</td>\\n</tr><tr id=\\\"pkg_Cogito\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/Cogito.html\\\">Cogito</a></td>\\n\\t<td class=\\\" \\\">Annika Brger</td>\\n\\t<td class=\\\" \\\">Compare genomic intervals tool - Automated, complete, reproducible and clear report about genomic and epigenomic data sets</td>\\n\\t<td class=\\\"  sorting_1\\\">272</td>\\n</tr><tr id=\\\"pkg_CSSQ\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/CSSQ.html\\\">CSSQ</a></td>\\n\\t<td class=\\\" \\\">Fan Lab at Georgia Institute of Technology</td>\\n\\t<td class=\\\" \\\">Chip-seq Signal Quantifier Pipeline</td>\\n\\t<td class=\\\"  sorting_1\\\">272</td>\\n</tr><tr id=\\\"pkg_DelayedTensor\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/DelayedTensor.html\\\">DelayedTensor</a></td>\\n\\t<td class=\\\" \\\">Koki Tsuyuzaki</td>\\n\\t<td class=\\\" \\\">R package for sparse and out-of-core arithmetic and decomposition of Tensor</td>\\n\\t<td class=\\\"  sorting_1\\\">272</td>\\n</tr><tr id=\\\"pkg_Director\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/Director.html\\\">Director</a></td>\\n\\t<td class=\\\" \\\">Katherine Icay</td>\\n\\t<td class=\\\" \\\">A dynamic visualization tool of multi-level data</td>\\n\\t<td class=\\\"  sorting_1\\\">272</td>\\n</tr><tr id=\\\"pkg_divergence\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/divergence.html\\\">divergence</a></td>\\n\\t<td class=\\\" \\\">Wikum Dinalankara, Luigi Marchionni</td>\\n\\t<td class=\\\" \\\">Divergence: Functionality for assessing omics data by divergence with respect to a baseline</td>\\n\\t<td class=\\\"  sorting_1\\\">272</td>\\n</tr><tr id=\\\"pkg_DMRScan\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/DMRScan.html\\\">DMRScan</a></td>\\n\\t<td class=\\\" \\\">Christian M Page</td>\\n\\t<td class=\\\" \\\">Detection of Differentially Methylated Regions</td>\\n\\t<td class=\\\"  sorting_1\\\">272</td>\\n</tr><tr id=\\\"pkg_Doscheda\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/Doscheda.html\\\">Doscheda</a></td>\\n\\t<td class=\\\" \\\">Bruno Contrino</td>\\n\\t<td class=\\\" \\\">A DownStream Chemo-Proteomics Analysis Pipeline</td>\\n\\t<td class=\\\"  sorting_1\\\">272</td>\\n</tr><tr id=\\\"pkg_epihet\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/epihet.html\\\">epihet</a></td>\\n\\t<td class=\\\" \\\">Xiaowen Chen</td>\\n\\t<td class=\\\" \\\">Determining Epigenetic Heterogeneity from Bisulfite Sequencing Data</td>\\n\\t<td class=\\\"  sorting_1\\\">272</td>\\n</tr><tr id=\\\"pkg_fedup\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/fedup.html\\\">fedup</a></td>\\n\\t<td class=\\\" \\\">Catherine Ross</td>\\n\\t<td class=\\\" \\\">Fisher's Test for Enrichment and Depletion of User-Defined Pathways</td>\\n\\t<td class=\\\"  sorting_1\\\">272</td>\\n</tr><tr id=\\\"pkg_flowGraph\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/flowGraph.html\\\">flowGraph</a></td>\\n\\t<td class=\\\" \\\">Alice Yue</td>\\n\\t<td class=\\\" \\\">Identifying differential cell populations in flow cytometry data accounting for marker frequency</td>\\n\\t<td class=\\\"  sorting_1\\\">272</td>\\n</tr><tr id=\\\"pkg_genomicInstability\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/genomicInstability.html\\\">genomicInstability</a></td>\\n\\t<td class=\\\" \\\">Mariano Alvarez</td>\\n\\t<td class=\\\" \\\">Genomic Instability estimation for scRNA-Seq</td>\\n\\t<td class=\\\"  sorting_1\\\">272</td>\\n</tr><tr id=\\\"pkg_HPiP\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/HPiP.html\\\">HPiP</a></td>\\n\\t<td class=\\\" \\\">Matineh Rahmatbakhsh</td>\\n\\t<td class=\\\" \\\">Host-Pathogen Interaction Prediction</td>\\n\\t<td class=\\\"  sorting_1\\\">272</td>\\n</tr><tr id=\\\"pkg_hummingbird\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/hummingbird.html\\\">hummingbird</a></td>\\n\\t<td class=\\\" \\\">Eleni Adam</td>\\n\\t<td class=\\\" \\\">Bayesian Hidden Markov Model for the detection of differentially methylated regions</td>\\n\\t<td class=\\\"  sorting_1\\\">272</td>\\n</tr><tr id=\\\"pkg_IgGeneUsage\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/IgGeneUsage.html\\\">IgGeneUsage</a></td>\\n\\t<td class=\\\" \\\">Simo Kitanovski</td>\\n\\t<td class=\\\" \\\">Differential gene usage in immune repertoires</td>\\n\\t<td class=\\\"  sorting_1\\\">272</td>\\n</tr><tr id=\\\"pkg_IsoCorrectoRGUI\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/IsoCorrectoRGUI.html\\\">IsoCorrectoRGUI</a></td>\\n\\t<td class=\\\" \\\">Christian Kohler</td>\\n\\t<td class=\\\" \\\">Graphical User Interface for IsoCorrectoR</td>\\n\\t<td class=\\\"  sorting_1\\\">272</td>\\n</tr><tr id=\\\"pkg_lpNet\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/lpNet.html\\\">lpNet</a></td>\\n\\t<td class=\\\" \\\">Lars Kaderali</td>\\n\\t<td class=\\\" \\\">Linear Programming Model for Network Inference</td>\\n\\t<td class=\\\"  sorting_1\\\">272</td>\\n</tr><tr id=\\\"pkg_matchBox\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/matchBox.html\\\">matchBox</a></td>\\n\\t<td class=\\\" \\\">Luigi Marchionni, Anuj Gupta</td>\\n\\t<td class=\\\" \\\">Utilities to compute, compare, and plot the agreement between ordered vectors of features (ie. distinct genomic experiments). The package includes Correspondence-At-the-TOP (CAT) analysis.</td>\\n\\t<td class=\\\"  sorting_1\\\">272</td>\\n</tr><tr id=\\\"pkg_MatrixRider\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/MatrixRider.html\\\">MatrixRider</a></td>\\n\\t<td class=\\\" \\\">Elena Grassi</td>\\n\\t<td class=\\\" \\\">Obtain total affinity and occupancies for binding site matrices on a given sequence</td>\\n\\t<td class=\\\"  sorting_1\\\">272</td>\\n</tr><tr id=\\\"pkg_MethPed\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/MethPed.html\\\">MethPed</a></td>\\n\\t<td class=\\\" \\\">Helena Carn</td>\\n\\t<td class=\\\" \\\">A DNA methylation classifier tool for the identification of pediatric brain tumor subtypes</td>\\n\\t<td class=\\\"  sorting_1\\\">272</td>\\n</tr><tr id=\\\"pkg_MethTargetedNGS\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/MethTargetedNGS.html\\\">MethTargetedNGS</a></td>\\n\\t<td class=\\\" \\\">Muhammad Ahmer Jamil</td>\\n\\t<td class=\\\" \\\">Perform Methylation Analysis on Next Generation Sequencing Data</td>\\n\\t<td class=\\\"  sorting_1\\\">272</td>\\n</tr><tr id=\\\"pkg_mina\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/mina.html\\\">mina</a></td>\\n\\t<td class=\\\" \\\">Rui Guan</td>\\n\\t<td class=\\\" \\\">Microbial community dIversity and Network Analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">272</td>\\n</tr><tr id=\\\"pkg_MSstatsLiP\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/MSstatsLiP.html\\\">MSstatsLiP</a></td>\\n\\t<td class=\\\" \\\">Devon Kohler</td>\\n\\t<td class=\\\" \\\">LiP Significance Analysis in shotgun mass spectrometry-based proteomic experiments</td>\\n\\t<td class=\\\"  sorting_1\\\">272</td>\\n</tr><tr id=\\\"pkg_nearBynding\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/nearBynding.html\\\">nearBynding</a></td>\\n\\t<td class=\\\" \\\">Veronica Busa</td>\\n\\t<td class=\\\" \\\">Discern RNA structure proximal to protein binding</td>\\n\\t<td class=\\\"  sorting_1\\\">272</td>\\n</tr><tr id=\\\"pkg_netprioR\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/netprioR.html\\\">netprioR</a></td>\\n\\t<td class=\\\" \\\">Fabian Schmich</td>\\n\\t<td class=\\\" \\\">A model for network-based prioritisation of genes</td>\\n\\t<td class=\\\"  sorting_1\\\">272</td>\\n</tr><tr id=\\\"pkg_normalize450K\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/normalize450K.html\\\">normalize450K</a></td>\\n\\t<td class=\\\" \\\">Jonathan Alexander Heiss</td>\\n\\t<td class=\\\" \\\">Preprocessing of Illumina Infinium 450K data</td>\\n\\t<td class=\\\"  sorting_1\\\">272</td>\\n</tr><tr id=\\\"pkg_Omixer\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/Omixer.html\\\">Omixer</a></td>\\n\\t<td class=\\\" \\\">Lucy Sinke</td>\\n\\t<td class=\\\" \\\">Omixer: multivariate and reproducible sample randomization to proactively counter batch effects in omics studies</td>\\n\\t<td class=\\\"  sorting_1\\\">272</td>\\n</tr><tr id=\\\"pkg_OPWeight\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/OPWeight.html\\\">OPWeight</a></td>\\n\\t<td class=\\\" \\\">Mohamad Hasan</td>\\n\\t<td class=\\\" \\\">Optimal p-value weighting with independent information</td>\\n\\t<td class=\\\"  sorting_1\\\">272</td>\\n</tr><tr id=\\\"pkg_padma\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/padma.html\\\">padma</a></td>\\n\\t<td class=\\\" \\\">Andrea Rau</td>\\n\\t<td class=\\\" \\\">Individualized Multi-Omic Pathway Deviation Scores Using Multiple Factor Analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">272</td>\\n</tr><tr id=\\\"pkg_pcxn\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/pcxn.html\\\">pcxn</a></td>\\n\\t<td class=\\\" \\\">Sokratis Kariotis</td>\\n\\t<td class=\\\" \\\">Exploring, analyzing and visualizing functions utilizing the pcxnData package</td>\\n\\t<td class=\\\"  sorting_1\\\">272</td>\\n</tr><tr id=\\\"pkg_PDATK\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/PDATK.html\\\">PDATK</a></td>\\n\\t<td class=\\\" \\\">Benjamin Haibe-Kains</td>\\n\\t<td class=\\\" \\\">Pancreatic Ductal Adenocarcinoma Tool-Kit</td>\\n\\t<td class=\\\"  sorting_1\\\">272</td>\\n</tr><tr id=\\\"pkg_PFP\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/PFP.html\\\">PFP</a></td>\\n\\t<td class=\\\" \\\">XC Zhang</td>\\n\\t<td class=\\\" \\\">Pathway Fingerprint Framework in R</td>\\n\\t<td class=\\\"  sorting_1\\\">272</td>\\n</tr><tr id=\\\"pkg_pogos\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/pogos.html\\\">pogos</a></td>\\n\\t<td class=\\\" \\\">VJ Carey</td>\\n\\t<td class=\\\" \\\">PharmacOGenomics Ontology Support</td>\\n\\t<td class=\\\"  sorting_1\\\">272</td>\\n</tr><tr id=\\\"pkg_RadioGx\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/RadioGx.html\\\">RadioGx</a></td>\\n\\t<td class=\\\" \\\">Benjamin Haibe-Kains</td>\\n\\t<td class=\\\" \\\">Analysis of Large-Scale Radio-Genomic Data</td>\\n\\t<td class=\\\"  sorting_1\\\">272</td>\\n</tr><tr id=\\\"pkg_randRotation\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/randRotation.html\\\">randRotation</a></td>\\n\\t<td class=\\\" \\\">Peter Hettegger</td>\\n\\t<td class=\\\" \\\">Random Rotation Methods for High Dimensional Data with Batch Structure</td>\\n\\t<td class=\\\"  sorting_1\\\">272</td>\\n</tr><tr id=\\\"pkg_REBET\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/REBET.html\\\">REBET</a></td>\\n\\t<td class=\\\" \\\">Bill Wheeler</td>\\n\\t<td class=\\\" \\\">The subREgion-based BurdEn Test (REBET)</td>\\n\\t<td class=\\\"  sorting_1\\\">272</td>\\n</tr><tr id=\\\"pkg_RNAmodR.ML\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/RNAmodR.ML.html\\\">RNAmodR.ML</a></td>\\n\\t<td class=\\\" \\\">Felix G.M. Ernst</td>\\n\\t<td class=\\\" \\\">Detecting patterns of post-transcriptional modifications using machine learning</td>\\n\\t<td class=\\\"  sorting_1\\\">272</td>\\n</tr><tr id=\\\"pkg_ROSeq\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ROSeq.html\\\">ROSeq</a></td>\\n\\t<td class=\\\" \\\">Krishan Gupta</td>\\n\\t<td class=\\\" \\\">Modeling expression ranks for noise-tolerant differential expression analysis of scRNA-Seq data</td>\\n\\t<td class=\\\"  sorting_1\\\">272</td>\\n</tr><tr id=\\\"pkg_scBFA\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/scBFA.html\\\">scBFA</a></td>\\n\\t<td class=\\\" \\\">Ruoxin Li</td>\\n\\t<td class=\\\" \\\">A dimensionality reduction tool using gene detection pattern to mitigate noisy expression profile of scRNA-seq</td>\\n\\t<td class=\\\"  sorting_1\\\">272</td>\\n</tr><tr id=\\\"pkg_scTHI\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/scTHI.html\\\">scTHI</a></td>\\n\\t<td class=\\\" \\\">Michele Ceccarelli</td>\\n\\t<td class=\\\" \\\">Indentification of significantly activated ligand-receptor interactions across clusters of cells from single-cell RNA sequencing data</td>\\n\\t<td class=\\\"  sorting_1\\\">272</td>\\n</tr><tr id=\\\"pkg_seqArchR\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/seqArchR.html\\\">seqArchR</a></td>\\n\\t<td class=\\\" \\\">Sarvesh Nikumbh</td>\\n\\t<td class=\\\" \\\">Identify Different Architectures of Sequence Elements</td>\\n\\t<td class=\\\"  sorting_1\\\">272</td>\\n</tr><tr id=\\\"pkg_staRank\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/staRank.html\\\">staRank</a></td>\\n\\t<td class=\\\" \\\">Juliane Siebourg</td>\\n\\t<td class=\\\" \\\">Stability Ranking</td>\\n\\t<td class=\\\"  sorting_1\\\">272</td>\\n</tr><tr id=\\\"pkg_STROMA4\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/STROMA4.html\\\">STROMA4</a></td>\\n\\t<td class=\\\" \\\">Sadiq Saleh</td>\\n\\t<td class=\\\" \\\">Assign Properties to TNBC Patients</td>\\n\\t<td class=\\\"  sorting_1\\\">272</td>\\n</tr><tr id=\\\"pkg_TarSeqQC\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/TarSeqQC.html\\\">TarSeqQC</a></td>\\n\\t<td class=\\\" \\\">Gabriela Merino</td>\\n\\t<td class=\\\" \\\">TARgeted SEQuencing Experiment Quality Control</td>\\n\\t<td class=\\\"  sorting_1\\\">272</td>\\n</tr><tr id=\\\"pkg_tLOH\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/tLOH.html\\\">tLOH</a></td>\\n\\t<td class=\\\" \\\">Michelle Webb</td>\\n\\t<td class=\\\" \\\">Assessment of evidence for LOH in spatial transcriptomics pre-processed data using Bayes factor calculations</td>\\n\\t<td class=\\\"  sorting_1\\\">272</td>\\n</tr><tr id=\\\"pkg_treekoR\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/treekoR.html\\\">treekoR</a></td>\\n\\t<td class=\\\" \\\">Adam Chan</td>\\n\\t<td class=\\\" \\\">Cytometry Cluster Hierarchy and Cellular-to-phenotype Associations</td>\\n\\t<td class=\\\"  sorting_1\\\">272</td>\\n</tr><tr id=\\\"pkg_TRESS\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/TRESS.html\\\">TRESS</a></td>\\n\\t<td class=\\\" \\\">Zhenxing Guo</td>\\n\\t<td class=\\\" \\\">Toolbox for mRNA epigenetics sequencing analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">272</td>\\n</tr><tr id=\\\"pkg_VarCon\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/VarCon.html\\\">VarCon</a></td>\\n\\t<td class=\\\" \\\">Johannes Ptok</td>\\n\\t<td class=\\\" \\\">VarCon: an R package for retrieving neighboring nucleotides of an SNV</td>\\n\\t<td class=\\\"  sorting_1\\\">272</td>\\n</tr><tr id=\\\"pkg_weitrix\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/weitrix.html\\\">weitrix</a></td>\\n\\t<td class=\\\" \\\">Paul Harrison</td>\\n\\t<td class=\\\" \\\">Tools for matrices with precision weights, test and explore weighted or sparse data</td>\\n\\t<td class=\\\"  sorting_1\\\">272</td>\\n</tr><tr id=\\\"pkg_ArrayExpressHTS\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ArrayExpressHTS.html\\\">ArrayExpressHTS</a></td>\\n\\t<td class=\\\" \\\">Angela Goncalves, Andrew Tikhonov</td>\\n\\t<td class=\\\" \\\">ArrayExpress High Throughput Sequencing Processing Pipeline</td>\\n\\t<td class=\\\"  sorting_1\\\">321</td>\\n</tr><tr id=\\\"pkg_BayesKnockdown\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/BayesKnockdown.html\\\">BayesKnockdown</a></td>\\n\\t<td class=\\\" \\\">William Chad Young</td>\\n\\t<td class=\\\" \\\">BayesKnockdown: Posterior Probabilities for Edges from Knockdown Data</td>\\n\\t<td class=\\\"  sorting_1\\\">321</td>\\n</tr><tr id=\\\"pkg_BUSseq\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/BUSseq.html\\\">BUSseq</a></td>\\n\\t<td class=\\\" \\\">Fangda Song</td>\\n\\t<td class=\\\" \\\">Batch Effect Correction with Unknow Subtypes for scRNA-seq data</td>\\n\\t<td class=\\\"  sorting_1\\\">321</td>\\n</tr><tr id=\\\"pkg_cageminer\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/cageminer.html\\\">cageminer</a></td>\\n\\t<td class=\\\" \\\">Fabrcio Almeida-Silva</td>\\n\\t<td class=\\\" \\\">Candidate Gene Miner</td>\\n\\t<td class=\\\"  sorting_1\\\">321</td>\\n</tr><tr id=\\\"pkg_CexoR\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/CexoR.html\\\">CexoR</a></td>\\n\\t<td class=\\\" \\\">Pedro Madrigal</td>\\n\\t<td class=\\\" \\\">An R package to uncover high-resolution protein-DNA interactions in ChIP-exo replicates</td>\\n\\t<td class=\\\"  sorting_1\\\">321</td>\\n</tr><tr id=\\\"pkg_CIMICE\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/CIMICE.html\\\">CIMICE</a></td>\\n\\t<td class=\\\" \\\">Nicol Rossi</td>\\n\\t<td class=\\\" \\\">CIMICE-R: (Markov) Chain Method to Inferr Cancer Evolution</td>\\n\\t<td class=\\\"  sorting_1\\\">321</td>\\n</tr><tr id=\\\"pkg_Clomial\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/Clomial.html\\\">Clomial</a></td>\\n\\t<td class=\\\" \\\">Habil Zare</td>\\n\\t<td class=\\\" \\\">Infers clonal composition of a tumor</td>\\n\\t<td class=\\\"  sorting_1\\\">321</td>\\n</tr><tr id=\\\"pkg_cogeqc\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/cogeqc.html\\\">cogeqc</a></td>\\n\\t<td class=\\\" \\\">Fabrcio Almeida-Silva</td>\\n\\t<td class=\\\" \\\">Systematic quality checks on comparative genomics analyses</td>\\n\\t<td class=\\\"  sorting_1\\\">321</td>\\n</tr><tr id=\\\"pkg_COTAN\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/COTAN.html\\\">COTAN</a></td>\\n\\t<td class=\\\" \\\">Galfr Silvia Giulia</td>\\n\\t<td class=\\\" \\\">COexpression Tables ANalysis</td>\\n\\t<td class=\\\"  sorting_1\\\">321</td>\\n</tr><tr id=\\\"pkg_cyanoFilter\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/cyanoFilter.html\\\">cyanoFilter</a></td>\\n\\t<td class=\\\" \\\">Oluwafemi Olusoji</td>\\n\\t<td class=\\\" \\\">Phytoplankton Population Identification using Cell Pigmentation and/or Complexity</td>\\n\\t<td class=\\\"  sorting_1\\\">321</td>\\n</tr><tr id=\\\"pkg_deltaGseg\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/deltaGseg.html\\\">deltaGseg</a></td>\\n\\t<td class=\\\" \\\">Diana Low</td>\\n\\t<td class=\\\" \\\">deltaGseg</td>\\n\\t<td class=\\\"  sorting_1\\\">321</td>\\n</tr><tr id=\\\"pkg_DNABarcodeCompatibility\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/DNABarcodeCompatibility.html\\\">DNABarcodeCompatibility</a></td>\\n\\t<td class=\\\" \\\">Cline Trbeau</td>\\n\\t<td class=\\\" \\\">A Tool for Optimizing Combinations of DNA Barcodes Used in Multiplexed Experiments on Next Generation Sequencing Platforms</td>\\n\\t<td class=\\\"  sorting_1\\\">321</td>\\n</tr><tr id=\\\"pkg_DominoEffect\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/DominoEffect.html\\\">DominoEffect</a></td>\\n\\t<td class=\\\" \\\">Marija Buljan, Peter Blattmann</td>\\n\\t<td class=\\\" \\\">Identification and Annotation of Protein Hotspot Residues</td>\\n\\t<td class=\\\"  sorting_1\\\">321</td>\\n</tr><tr id=\\\"pkg_Dune\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/Dune.html\\\">Dune</a></td>\\n\\t<td class=\\\" \\\">Hector Roux de Bezieux</td>\\n\\t<td class=\\\" \\\">Improving replicability in single-cell RNA-Seq cell type discovery</td>\\n\\t<td class=\\\"  sorting_1\\\">321</td>\\n</tr><tr id=\\\"pkg_ERSSA\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ERSSA.html\\\">ERSSA</a></td>\\n\\t<td class=\\\" \\\">Zixuan Shao</td>\\n\\t<td class=\\\" \\\">Empirical RNA-seq Sample Size Analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">321</td>\\n</tr><tr id=\\\"pkg_eudysbiome\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/eudysbiome.html\\\">eudysbiome</a></td>\\n\\t<td class=\\\" \\\">Xiaoyuan Zhou</td>\\n\\t<td class=\\\" \\\">Cartesian plot and contingency test on 16S Microbial data</td>\\n\\t<td class=\\\"  sorting_1\\\">321</td>\\n</tr><tr id=\\\"pkg_ExCluster\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ExCluster.html\\\">ExCluster</a></td>\\n\\t<td class=\\\" \\\">R. Matthew Tanner</td>\\n\\t<td class=\\\" \\\">ExCluster robustly detects differentially expressed exons between two conditions of RNA-seq data, requiring at least two independent biological replicates per condition</td>\\n\\t<td class=\\\"  sorting_1\\\">321</td>\\n</tr><tr id=\\\"pkg_famat\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/famat.html\\\">famat</a></td>\\n\\t<td class=\\\" \\\">Mathieu Charles</td>\\n\\t<td class=\\\" \\\">Functional analysis of metabolic and transcriptomic data</td>\\n\\t<td class=\\\"  sorting_1\\\">321</td>\\n</tr><tr id=\\\"pkg_fastreeR\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/fastreeR.html\\\">fastreeR</a></td>\\n\\t<td class=\\\" \\\">Anestis Gkanogiannis</td>\\n\\t<td class=\\\" \\\">Phylogenetic, Distance and Other Calculations on VCF and Fasta Files</td>\\n\\t<td class=\\\"  sorting_1\\\">321</td>\\n</tr><tr id=\\\"pkg_FilterFFPE\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/FilterFFPE.html\\\">FilterFFPE</a></td>\\n\\t<td class=\\\" \\\">Lanying Wei</td>\\n\\t<td class=\\\" \\\">FFPE Artificial Chimeric Read Filter for NGS data</td>\\n\\t<td class=\\\"  sorting_1\\\">321</td>\\n</tr><tr id=\\\"pkg_FScanR\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/FScanR.html\\\">FScanR</a></td>\\n\\t<td class=\\\" \\\">Xiao Chen</td>\\n\\t<td class=\\\" \\\">Detect Programmed Ribosomal Frameshifting Events from mRNA/cDNA BLASTX Output</td>\\n\\t<td class=\\\"  sorting_1\\\">321</td>\\n</tr><tr id=\\\"pkg_GA4GHshiny\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/GA4GHshiny.html\\\">GA4GHshiny</a></td>\\n\\t<td class=\\\" \\\">Welliton Souza</td>\\n\\t<td class=\\\" \\\">Shiny application for interacting with GA4GH-based data servers</td>\\n\\t<td class=\\\"  sorting_1\\\">321</td>\\n</tr><tr id=\\\"pkg_GBScleanR\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/GBScleanR.html\\\">GBScleanR</a></td>\\n\\t<td class=\\\" \\\">Tomoyuki Furuta</td>\\n\\t<td class=\\\" \\\">Error correction tool for noisy genotyping by sequencing (GBS) data</td>\\n\\t<td class=\\\"  sorting_1\\\">321</td>\\n</tr><tr id=\\\"pkg_GeneGA\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/GeneGA.html\\\">GeneGA</a></td>\\n\\t<td class=\\\" \\\">Zhenpeng Li</td>\\n\\t<td class=\\\" \\\">Design gene based on both mRNA secondary structure and codon usage bias using Genetic algorithm</td>\\n\\t<td class=\\\"  sorting_1\\\">321</td>\\n</tr><tr id=\\\"pkg_geva\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/geva.html\\\">geva</a></td>\\n\\t<td class=\\\" \\\">Itamar Jos Guimares Nunes</td>\\n\\t<td class=\\\" \\\">Gene Expression Variation Analysis (GEVA)</td>\\n\\t<td class=\\\"  sorting_1\\\">321</td>\\n</tr><tr id=\\\"pkg_GladiaTOX\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/GladiaTOX.html\\\">GladiaTOX</a></td>\\n\\t<td class=\\\" \\\">PMP S.A. R Support</td>\\n\\t<td class=\\\" \\\">R Package for Processing High Content Screening data</td>\\n\\t<td class=\\\"  sorting_1\\\">321</td>\\n</tr><tr id=\\\"pkg_GPA\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/GPA.html\\\">GPA</a></td>\\n\\t<td class=\\\" \\\">Dongjun Chung</td>\\n\\t<td class=\\\" \\\">GPA (Genetic analysis incorporating Pleiotropy and Annotation)</td>\\n\\t<td class=\\\"  sorting_1\\\">321</td>\\n</tr><tr id=\\\"pkg_GSgalgoR\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/GSgalgoR.html\\\">GSgalgoR</a></td>\\n\\t<td class=\\\" \\\">Carlos Catania</td>\\n\\t<td class=\\\" \\\">An Evolutionary Framework for the Identification and Study of Prognostic Gene Expression Signatures in Cancer</td>\\n\\t<td class=\\\"  sorting_1\\\">321</td>\\n</tr><tr id=\\\"pkg_ivygapSE\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ivygapSE.html\\\">ivygapSE</a></td>\\n\\t<td class=\\\" \\\">VJ Carey</td>\\n\\t<td class=\\\" \\\">A SummarizedExperiment for Ivy-GAP data</td>\\n\\t<td class=\\\"  sorting_1\\\">321</td>\\n</tr><tr id=\\\"pkg_IWTomics\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/IWTomics.html\\\">IWTomics</a></td>\\n\\t<td class=\\\" \\\">Marzia A Cremona</td>\\n\\t<td class=\\\" \\\">Interval-Wise Testing for Omics Data</td>\\n\\t<td class=\\\"  sorting_1\\\">321</td>\\n</tr><tr id=\\\"pkg_KBoost\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/KBoost.html\\\">KBoost</a></td>\\n\\t<td class=\\\" \\\">Luis F. Iglesias-Martinez</td>\\n\\t<td class=\\\" \\\">Inference of gene regulatory networks from gene expression data</td>\\n\\t<td class=\\\"  sorting_1\\\">321</td>\\n</tr><tr id=\\\"pkg_missRows\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/missRows.html\\\">missRows</a></td>\\n\\t<td class=\\\" \\\">Gonzalez Ignacio</td>\\n\\t<td class=\\\" \\\">Handling Missing Individuals in Multi-Omics Data Integration</td>\\n\\t<td class=\\\"  sorting_1\\\">321</td>\\n</tr><tr id=\\\"pkg_MsBackendRawFileReader\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/MsBackendRawFileReader.html\\\">MsBackendRawFileReader</a></td>\\n\\t<td class=\\\" \\\">Christian Panse</td>\\n\\t<td class=\\\" \\\">Mass Spectrometry Backend for Reading Thermo Fisher Scientific raw Files</td>\\n\\t<td class=\\\"  sorting_1\\\">321</td>\\n</tr><tr id=\\\"pkg_NBAMSeq\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/NBAMSeq.html\\\">NBAMSeq</a></td>\\n\\t<td class=\\\" \\\">Xu Ren</td>\\n\\t<td class=\\\" \\\">Negative Binomial Additive Model for RNA-Seq Data</td>\\n\\t<td class=\\\"  sorting_1\\\">321</td>\\n</tr><tr id=\\\"pkg_ncGTW\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ncGTW.html\\\">ncGTW</a></td>\\n\\t<td class=\\\" \\\">Chiung-Ting Wu</td>\\n\\t<td class=\\\" \\\">Alignment of LC-MS Profiles by Neighbor-wise Compound-specific Graphical Time Warping with Misalignment Detection</td>\\n\\t<td class=\\\"  sorting_1\\\">321</td>\\n</tr><tr id=\\\"pkg_nempi\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/nempi.html\\\">nempi</a></td>\\n\\t<td class=\\\" \\\">Martin Pirkl</td>\\n\\t<td class=\\\" \\\">Inferring unobserved perturbations from gene expression data</td>\\n\\t<td class=\\\"  sorting_1\\\">321</td>\\n</tr><tr id=\\\"pkg_oncomix\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/oncomix.html\\\">oncomix</a></td>\\n\\t<td class=\\\" \\\">Daniel Pique</td>\\n\\t<td class=\\\" \\\">Identifying Genes Overexpressed in Subsets of Tumors from Tumor-Normal mRNA Expression Data</td>\\n\\t<td class=\\\"  sorting_1\\\">321</td>\\n</tr><tr id=\\\"pkg_parglms\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/parglms.html\\\">parglms</a></td>\\n\\t<td class=\\\" \\\">VJ Carey</td>\\n\\t<td class=\\\" \\\">support for parallelized estimation of GLMs/GEEs</td>\\n\\t<td class=\\\"  sorting_1\\\">321</td>\\n</tr><tr id=\\\"pkg_pgca\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/pgca.html\\\">pgca</a></td>\\n\\t<td class=\\\" \\\">Gabriela Cohen-Freue</td>\\n\\t<td class=\\\" \\\">PGCA: An Algorithm to Link Protein Groups Created from MS/MS Data</td>\\n\\t<td class=\\\"  sorting_1\\\">321</td>\\n</tr><tr id=\\\"pkg_pickgene\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/pickgene.html\\\">pickgene</a></td>\\n\\t<td class=\\\" \\\">Brian S. Yandell</td>\\n\\t<td class=\\\" \\\">Adaptive Gene Picking for Microarray Expression Data Analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">321</td>\\n</tr><tr id=\\\"pkg_pram\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/pram.html\\\">pram</a></td>\\n\\t<td class=\\\" \\\">Peng Liu</td>\\n\\t<td class=\\\" \\\">Pooling RNA-seq datasets for assembling transcript models</td>\\n\\t<td class=\\\"  sorting_1\\\">321</td>\\n</tr><tr id=\\\"pkg_prebs\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/prebs.html\\\">prebs</a></td>\\n\\t<td class=\\\" \\\">Karolis Uziela</td>\\n\\t<td class=\\\" \\\">Probe region expression estimation for RNA-seq data for improved microarray comparability</td>\\n\\t<td class=\\\"  sorting_1\\\">321</td>\\n</tr><tr id=\\\"pkg_primirTSS\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/primirTSS.html\\\">primirTSS</a></td>\\n\\t<td class=\\\" \\\">Pumin Li</td>\\n\\t<td class=\\\" \\\">Prediction of pri-miRNA Transcription Start Site</td>\\n\\t<td class=\\\"  sorting_1\\\">321</td>\\n</tr><tr id=\\\"pkg_PROPS\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/PROPS.html\\\">PROPS</a></td>\\n\\t<td class=\\\" \\\">Lichy Han</td>\\n\\t<td class=\\\" \\\">PRObabilistic Pathway Score (PROPS)</td>\\n\\t<td class=\\\"  sorting_1\\\">321</td>\\n</tr><tr id=\\\"pkg_RcwlPipelines\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/RcwlPipelines.html\\\">RcwlPipelines</a></td>\\n\\t<td class=\\\" \\\">Qiang Hu</td>\\n\\t<td class=\\\" \\\">Bioinformatics pipelines based on Rcwl</td>\\n\\t<td class=\\\"  sorting_1\\\">321</td>\\n</tr><tr id=\\\"pkg_RImmPort\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/RImmPort.html\\\">RImmPort</a></td>\\n\\t<td class=\\\" \\\">Zicheng Hu, Ravi Shankar</td>\\n\\t<td class=\\\" \\\">RImmPort: Enabling Ready-for-analysis Immunology Research Data</td>\\n\\t<td class=\\\"  sorting_1\\\">321</td>\\n</tr><tr id=\\\"pkg_RJMCMCNucleosomes\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/RJMCMCNucleosomes.html\\\">RJMCMCNucleosomes</a></td>\\n\\t<td class=\\\" \\\">Astrid Deschnes</td>\\n\\t<td class=\\\" \\\">Bayesian hierarchical model for genome-wide nucleosome positioning with high-throughput short-read data (MNase-Seq)</td>\\n\\t<td class=\\\"  sorting_1\\\">321</td>\\n</tr><tr id=\\\"pkg_RLSeq\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/RLSeq.html\\\">RLSeq</a></td>\\n\\t<td class=\\\" \\\">Henry Miller</td>\\n\\t<td class=\\\" \\\">RLSeq: An analysis package for R-loop mapping data</td>\\n\\t<td class=\\\"  sorting_1\\\">321</td>\\n</tr><tr id=\\\"pkg_Rmmquant\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/Rmmquant.html\\\">Rmmquant</a></td>\\n\\t<td class=\\\" \\\">Zytnicki Matthias</td>\\n\\t<td class=\\\" \\\">RNA-Seq multi-mapping Reads Quantification Tool</td>\\n\\t<td class=\\\"  sorting_1\\\">321</td>\\n</tr><tr id=\\\"pkg_rScudo\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/rScudo.html\\\">rScudo</a></td>\\n\\t<td class=\\\" \\\">Matteo Ciciani</td>\\n\\t<td class=\\\" \\\">Signature-based Clustering for Diagnostic Purposes</td>\\n\\t<td class=\\\"  sorting_1\\\">321</td>\\n</tr><tr id=\\\"pkg_SCArray\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/SCArray.html\\\">SCArray</a></td>\\n\\t<td class=\\\" \\\">Xiuwen Zheng</td>\\n\\t<td class=\\\" \\\">Large-scale single-cell RNA-seq data manipulation with GDS files</td>\\n\\t<td class=\\\"  sorting_1\\\">321</td>\\n</tr><tr id=\\\"pkg_SCATE\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/SCATE.html\\\">SCATE</a></td>\\n\\t<td class=\\\" \\\">Wenpin Hou</td>\\n\\t<td class=\\\" \\\">SCATE: Single-cell ATAC-seq Signal Extraction and Enhancement</td>\\n\\t<td class=\\\"  sorting_1\\\">321</td>\\n</tr><tr id=\\\"pkg_scMAGeCK\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/scMAGeCK.html\\\">scMAGeCK</a></td>\\n\\t<td class=\\\" \\\">Xiaolong Cheng</td>\\n\\t<td class=\\\" \\\">Identify genes associated with multiple expression phenotypes in single-cell CRISPR screening data</td>\\n\\t<td class=\\\"  sorting_1\\\">321</td>\\n</tr><tr id=\\\"pkg_scoreInvHap\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/scoreInvHap.html\\\">scoreInvHap</a></td>\\n\\t<td class=\\\" \\\">Dolors Pelegri-Siso</td>\\n\\t<td class=\\\" \\\">Get inversion status in predefined regions</td>\\n\\t<td class=\\\"  sorting_1\\\">321</td>\\n</tr><tr id=\\\"pkg_SemDist\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/SemDist.html\\\">SemDist</a></td>\\n\\t<td class=\\\" \\\">Ian Gonzalez</td>\\n\\t<td class=\\\" \\\">Information Accretion-based Function Predictor Evaluation</td>\\n\\t<td class=\\\"  sorting_1\\\">321</td>\\n</tr><tr id=\\\"pkg_SigFuge\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/SigFuge.html\\\">SigFuge</a></td>\\n\\t<td class=\\\" \\\">Patrick Kimes</td>\\n\\t<td class=\\\" \\\">SigFuge</td>\\n\\t<td class=\\\"  sorting_1\\\">321</td>\\n</tr><tr id=\\\"pkg_SigsPack\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/SigsPack.html\\\">SigsPack</a></td>\\n\\t<td class=\\\" \\\">Franziska Schumann</td>\\n\\t<td class=\\\" \\\">Mutational Signature Estimation for Single Samples</td>\\n\\t<td class=\\\"  sorting_1\\\">321</td>\\n</tr><tr id=\\\"pkg_SIMAT\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/SIMAT.html\\\">SIMAT</a></td>\\n\\t<td class=\\\" \\\">M. R. Nezami Ranjbar</td>\\n\\t<td class=\\\" \\\">GC-SIM-MS data processing and alaysis tool</td>\\n\\t<td class=\\\"  sorting_1\\\">321</td>\\n</tr><tr id=\\\"pkg_SimBindProfiles\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/SimBindProfiles.html\\\">SimBindProfiles</a></td>\\n\\t<td class=\\\" \\\">Bettina Fischer</td>\\n\\t<td class=\\\" \\\">Similar Binding Profiles</td>\\n\\t<td class=\\\"  sorting_1\\\">321</td>\\n</tr><tr id=\\\"pkg_skewr\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/skewr.html\\\">skewr</a></td>\\n\\t<td class=\\\" \\\">Ryan Putney</td>\\n\\t<td class=\\\" \\\">Visualize Intensities Produced by Illumina's Human Methylation 450k BeadChip</td>\\n\\t<td class=\\\"  sorting_1\\\">321</td>\\n</tr><tr id=\\\"pkg_SNPediaR\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/SNPediaR.html\\\">SNPediaR</a></td>\\n\\t<td class=\\\" \\\">David Montaner</td>\\n\\t<td class=\\\" \\\">Query data from SNPedia</td>\\n\\t<td class=\\\"  sorting_1\\\">321</td>\\n</tr><tr id=\\\"pkg_SNPhood\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/SNPhood.html\\\">SNPhood</a></td>\\n\\t<td class=\\\" \\\">Christian Arnold</td>\\n\\t<td class=\\\" \\\">SNPhood: Investigate, quantify and visualise the epigenomic neighbourhood of SNPs using NGS data</td>\\n\\t<td class=\\\"  sorting_1\\\">321</td>\\n</tr><tr id=\\\"pkg_spqn\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/spqn.html\\\">spqn</a></td>\\n\\t<td class=\\\" \\\">Yi Wang</td>\\n\\t<td class=\\\" \\\">Spatial quantile normalization</td>\\n\\t<td class=\\\"  sorting_1\\\">321</td>\\n</tr><tr id=\\\"pkg_SynMut\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/SynMut.html\\\">SynMut</a></td>\\n\\t<td class=\\\" \\\">Haogao Gu</td>\\n\\t<td class=\\\" \\\">SynMut: Designing Synonymously Mutated Sequences with Different Genomic Signatures</td>\\n\\t<td class=\\\"  sorting_1\\\">321</td>\\n</tr><tr id=\\\"pkg_systemPipeTools\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/systemPipeTools.html\\\">systemPipeTools</a></td>\\n\\t<td class=\\\" \\\">Daniela Cassol</td>\\n\\t<td class=\\\" \\\">Tools for data visualization</td>\\n\\t<td class=\\\"  sorting_1\\\">321</td>\\n</tr><tr id=\\\"pkg_ternarynet\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ternarynet.html\\\">ternarynet</a></td>\\n\\t<td class=\\\" \\\">McCall N. Matthew</td>\\n\\t<td class=\\\" \\\">Ternary Network Estimation</td>\\n\\t<td class=\\\"  sorting_1\\\">321</td>\\n</tr><tr id=\\\"pkg_transformGamPoi\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/transformGamPoi.html\\\">transformGamPoi</a></td>\\n\\t<td class=\\\" \\\">Constantin Ahlmann-Eltze</td>\\n\\t<td class=\\\" \\\">Variance Stabilizing Transformation for Gamma-Poisson Models</td>\\n\\t<td class=\\\"  sorting_1\\\">321</td>\\n</tr><tr id=\\\"pkg_VAExprs\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/VAExprs.html\\\">VAExprs</a></td>\\n\\t<td class=\\\" \\\">Dongmin Jung</td>\\n\\t<td class=\\\" \\\">Generating Samples of Gene Expression Data with Variational Autoencoders</td>\\n\\t<td class=\\\"  sorting_1\\\">321</td>\\n</tr><tr id=\\\"pkg_VaSP\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/VaSP.html\\\">VaSP</a></td>\\n\\t<td class=\\\" \\\">Huihui Yu</td>\\n\\t<td class=\\\" \\\">Quantification and Visualization of Variations of Splicing in Population</td>\\n\\t<td class=\\\"  sorting_1\\\">321</td>\\n</tr><tr id=\\\"pkg_vulcan\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/vulcan.html\\\">vulcan</a></td>\\n\\t<td class=\\\" \\\">Federico M. Giorgi</td>\\n\\t<td class=\\\" \\\">VirtUaL ChIP-Seq data Analysis using Networks</td>\\n\\t<td class=\\\"  sorting_1\\\">321</td>\\n</tr><tr id=\\\"pkg_xmapbridge\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/xmapbridge.html\\\">xmapbridge</a></td>\\n\\t<td class=\\\" \\\">Chris Wirth</td>\\n\\t<td class=\\\" \\\">Export plotting files to the xmapBridge for visualisation in X:Map</td>\\n\\t<td class=\\\"  sorting_1\\\">321</td>\\n</tr><tr id=\\\"pkg_biodbHmdb\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/biodbHmdb.html\\\">biodbHmdb</a></td>\\n\\t<td class=\\\" \\\">Pierrick Roger</td>\\n\\t<td class=\\\" \\\">biodbHmdb, a library for connecting to the HMDB Database</td>\\n\\t<td class=\\\"  sorting_1\\\">392</td>\\n</tr><tr id=\\\"pkg_bnem\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/bnem.html\\\">bnem</a></td>\\n\\t<td class=\\\" \\\">Martin Pirkl</td>\\n\\t<td class=\\\" \\\">Training of logical models from indirect measurements of perturbation experiments</td>\\n\\t<td class=\\\"  sorting_1\\\">392</td>\\n</tr><tr id=\\\"pkg_BrainSABER\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/BrainSABER.html\\\">BrainSABER</a></td>\\n\\t<td class=\\\" \\\">USD Biomedical Engineering</td>\\n\\t<td class=\\\" \\\">Brain Span Atlas in Biobase Expressionset R toolset</td>\\n\\t<td class=\\\"  sorting_1\\\">392</td>\\n</tr><tr id=\\\"pkg_CCPROMISE\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/CCPROMISE.html\\\">CCPROMISE</a></td>\\n\\t<td class=\\\" \\\">Xueyuan Cao</td>\\n\\t<td class=\\\" \\\">PROMISE analysis with Canonical Correlation for Two Forms of High Dimensional Genetic Data</td>\\n\\t<td class=\\\"  sorting_1\\\">392</td>\\n</tr><tr id=\\\"pkg_CellBarcode\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/CellBarcode.html\\\">CellBarcode</a></td>\\n\\t<td class=\\\" \\\">Wenjie Sun</td>\\n\\t<td class=\\\" \\\">Cellular DNA Barcode Analysis toolkit</td>\\n\\t<td class=\\\"  sorting_1\\\">392</td>\\n</tr><tr id=\\\"pkg_cellmigRation\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/cellmigRation.html\\\">cellmigRation</a></td>\\n\\t<td class=\\\" \\\">Waldir Leoncio</td>\\n\\t<td class=\\\" \\\">Track Cells, Analyze Cell Trajectories and Compute Migration Statistics</td>\\n\\t<td class=\\\"  sorting_1\\\">392</td>\\n</tr><tr id=\\\"pkg_cliProfiler\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/cliProfiler.html\\\">cliProfiler</a></td>\\n\\t<td class=\\\" \\\">You Zhou</td>\\n\\t<td class=\\\" \\\">A package for the CLIP data visualization</td>\\n\\t<td class=\\\"  sorting_1\\\">392</td>\\n</tr><tr id=\\\"pkg_ComPrAn\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ComPrAn.html\\\">ComPrAn</a></td>\\n\\t<td class=\\\" \\\">Petra Palenikova</td>\\n\\t<td class=\\\" \\\">Complexome Profiling Analysis package</td>\\n\\t<td class=\\\"  sorting_1\\\">392</td>\\n</tr><tr id=\\\"pkg_ddPCRclust\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ddPCRclust.html\\\">ddPCRclust</a></td>\\n\\t<td class=\\\" \\\">Benedikt G. Brink</td>\\n\\t<td class=\\\" \\\">Clustering algorithm for ddPCR data</td>\\n\\t<td class=\\\"  sorting_1\\\">392</td>\\n</tr><tr id=\\\"pkg_diffGeneAnalysis\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/diffGeneAnalysis.html\\\">diffGeneAnalysis</a></td>\\n\\t<td class=\\\" \\\">Choudary Jagarlamudi</td>\\n\\t<td class=\\\" \\\">Performs differential gene expression Analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">392</td>\\n</tr><tr id=\\\"pkg_DMCFB\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/DMCFB.html\\\">DMCFB</a></td>\\n\\t<td class=\\\" \\\">Farhad Shokoohi</td>\\n\\t<td class=\\\" \\\">Differentially Methylated Cytosines via a Bayesian Functional Approach</td>\\n\\t<td class=\\\"  sorting_1\\\">392</td>\\n</tr><tr id=\\\"pkg_EBSEA\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/EBSEA.html\\\">EBSEA</a></td>\\n\\t<td class=\\\" \\\">Arfa Mehmood</td>\\n\\t<td class=\\\" \\\">Exon Based Strategy for Expression Analysis of genes</td>\\n\\t<td class=\\\"  sorting_1\\\">392</td>\\n</tr><tr id=\\\"pkg_flowCyBar\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/flowCyBar.html\\\">flowCyBar</a></td>\\n\\t<td class=\\\" \\\">Joachim Schumann</td>\\n\\t<td class=\\\" \\\">Analyze flow cytometric data using gate information</td>\\n\\t<td class=\\\"  sorting_1\\\">392</td>\\n</tr><tr id=\\\"pkg_funtooNorm\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/funtooNorm.html\\\">funtooNorm</a></td>\\n\\t<td class=\\\" \\\">Kathleen Klein</td>\\n\\t<td class=\\\" \\\">Normalization Procedure for Infinium HumanMethylation450 BeadChip Kit</td>\\n\\t<td class=\\\"  sorting_1\\\">392</td>\\n</tr><tr id=\\\"pkg_gaggle\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/gaggle.html\\\">gaggle</a></td>\\n\\t<td class=\\\" \\\">Christopher Bare</td>\\n\\t<td class=\\\" \\\">Broadcast data between R and Gaggle</td>\\n\\t<td class=\\\"  sorting_1\\\">392</td>\\n</tr><tr id=\\\"pkg_GenomicOZone\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/GenomicOZone.html\\\">GenomicOZone</a></td>\\n\\t<td class=\\\" \\\">Hua Zhong, Mingzhou Song</td>\\n\\t<td class=\\\" \\\">Delineate outstanding genomic zones of differential gene activity</td>\\n\\t<td class=\\\"  sorting_1\\\">392</td>\\n</tr><tr id=\\\"pkg_GEWIST\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/GEWIST.html\\\">GEWIST</a></td>\\n\\t<td class=\\\" \\\">Wei Q. Deng</td>\\n\\t<td class=\\\" \\\">Gene Environment Wide Interaction Search Threshold</td>\\n\\t<td class=\\\"  sorting_1\\\">392</td>\\n</tr><tr id=\\\"pkg_GIGSEA\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/GIGSEA.html\\\">GIGSEA</a></td>\\n\\t<td class=\\\" \\\">Shijia Zhu</td>\\n\\t<td class=\\\" \\\">Genotype Imputed Gene Set Enrichment Analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">392</td>\\n</tr><tr id=\\\"pkg_gmoviz\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/gmoviz.html\\\">gmoviz</a></td>\\n\\t<td class=\\\" \\\">Kathleen Zeglinski</td>\\n\\t<td class=\\\" \\\">Seamless visualization of complex genomic variations in GMOs and edited cell lines</td>\\n\\t<td class=\\\"  sorting_1\\\">392</td>\\n</tr><tr id=\\\"pkg_GMRP\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/GMRP.html\\\">GMRP</a></td>\\n\\t<td class=\\\" \\\">Yuan-De Tan</td>\\n\\t<td class=\\\" \\\">GWAS-based Mendelian Randomization and Path Analyses</td>\\n\\t<td class=\\\"  sorting_1\\\">392</td>\\n</tr><tr id=\\\"pkg_HPAStainR\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/HPAStainR.html\\\">HPAStainR</a></td>\\n\\t<td class=\\\" \\\">Tim O. Nieuwenhuis</td>\\n\\t<td class=\\\" \\\">Queries the Human Protein Atlas Staining Data for Multiple Proteins and Genes</td>\\n\\t<td class=\\\"  sorting_1\\\">392</td>\\n</tr><tr id=\\\"pkg_iASeq\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/iASeq.html\\\">iASeq</a></td>\\n\\t<td class=\\\" \\\">Yingying Wei</td>\\n\\t<td class=\\\" \\\">iASeq: integrating multiple sequencing datasets for detecting allele-specific events</td>\\n\\t<td class=\\\"  sorting_1\\\">392</td>\\n</tr><tr id=\\\"pkg_icetea\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/icetea.html\\\">icetea</a></td>\\n\\t<td class=\\\" \\\">Vivek Bhardwaj</td>\\n\\t<td class=\\\" \\\">Integrating Cap Enrichment with Transcript Expression Analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">392</td>\\n</tr><tr id=\\\"pkg_iChip\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/iChip.html\\\">iChip</a></td>\\n\\t<td class=\\\" \\\">Qianxing Mo</td>\\n\\t<td class=\\\" \\\">Bayesian Modeling of ChIP-chip Data Through Hidden Ising Models</td>\\n\\t<td class=\\\"  sorting_1\\\">392</td>\\n</tr><tr id=\\\"pkg_immunotation\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/immunotation.html\\\">immunotation</a></td>\\n\\t<td class=\\\" \\\">Katharina Imkeller</td>\\n\\t<td class=\\\" \\\">Tools for working with diverse immune genes</td>\\n\\t<td class=\\\"  sorting_1\\\">392</td>\\n</tr><tr id=\\\"pkg_IMPCdata\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/IMPCdata.html\\\">IMPCdata</a></td>\\n\\t<td class=\\\" \\\">Jeremy Mason</td>\\n\\t<td class=\\\" \\\">Retrieves data from IMPC database</td>\\n\\t<td class=\\\"  sorting_1\\\">392</td>\\n</tr><tr id=\\\"pkg_Informeasure\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/Informeasure.html\\\">Informeasure</a></td>\\n\\t<td class=\\\" \\\">Chu Pan</td>\\n\\t<td class=\\\" \\\">R implementation of Information measures</td>\\n\\t<td class=\\\"  sorting_1\\\">392</td>\\n</tr><tr id=\\\"pkg_ipdDb\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ipdDb.html\\\">ipdDb</a></td>\\n\\t<td class=\\\" \\\">Steffen Klasberg</td>\\n\\t<td class=\\\" \\\">IPD IMGT/HLA and IPD KIR database for Homo sapiens</td>\\n\\t<td class=\\\"  sorting_1\\\">392</td>\\n</tr><tr id=\\\"pkg_LACE\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/LACE.html\\\">LACE</a></td>\\n\\t<td class=\\\" \\\">Davide Maspero</td>\\n\\t<td class=\\\" \\\">Longitudinal Analysis of Cancer Evolution (LACE)</td>\\n\\t<td class=\\\"  sorting_1\\\">392</td>\\n</tr><tr id=\\\"pkg_LPEadj\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/LPEadj.html\\\">LPEadj</a></td>\\n\\t<td class=\\\" \\\">Carl Murie</td>\\n\\t<td class=\\\" \\\">A correction of the local pooled error (LPE) method to replace the asymptotic variance adjustment with an unbiased adjustment based on sample size.</td>\\n\\t<td class=\\\"  sorting_1\\\">392</td>\\n</tr><tr id=\\\"pkg_MADSEQ\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/MADSEQ.html\\\">MADSEQ</a></td>\\n\\t<td class=\\\" \\\">Yu Kong</td>\\n\\t<td class=\\\" \\\">Mosaic Aneuploidy Detection and Quantification using Massive Parallel Sequencing Data</td>\\n\\t<td class=\\\"  sorting_1\\\">392</td>\\n</tr><tr id=\\\"pkg_MantelCorr\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/MantelCorr.html\\\">MantelCorr</a></td>\\n\\t<td class=\\\" \\\">Brian Steinmeyer</td>\\n\\t<td class=\\\" \\\">Compute Mantel Cluster Correlations</td>\\n\\t<td class=\\\"  sorting_1\\\">392</td>\\n</tr><tr id=\\\"pkg_MDTS\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/MDTS.html\\\">MDTS</a></td>\\n\\t<td class=\\\" \\\">Jack M.. Fu</td>\\n\\t<td class=\\\" \\\">Detection of de novo deletion in targeted sequencing trios</td>\\n\\t<td class=\\\"  sorting_1\\\">392</td>\\n</tr><tr id=\\\"pkg_MeasurementError.cor\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/MeasurementError.cor.html\\\">MeasurementError.cor</a></td>\\n\\t<td class=\\\" \\\">Beiying Ding</td>\\n\\t<td class=\\\" \\\">Measurement Error model estimate for correlation coefficient</td>\\n\\t<td class=\\\"  sorting_1\\\">392</td>\\n</tr><tr id=\\\"pkg_Melissa\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/Melissa.html\\\">Melissa</a></td>\\n\\t<td class=\\\" \\\">C. A. Kapourani</td>\\n\\t<td class=\\\" \\\">Bayesian clustering and imputationa of single cell methylomes</td>\\n\\t<td class=\\\"  sorting_1\\\">392</td>\\n</tr><tr id=\\\"pkg_MIGSA\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/MIGSA.html\\\">MIGSA</a></td>\\n\\t<td class=\\\" \\\">Juan C. Rodriguez</td>\\n\\t<td class=\\\" \\\">Massive and Integrative Gene Set Analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">392</td>\\n</tr><tr id=\\\"pkg_multiscan\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/multiscan.html\\\">multiscan</a></td>\\n\\t<td class=\\\" \\\">Mizanur Khondoker</td>\\n\\t<td class=\\\" \\\">R package for combining multiple scans</td>\\n\\t<td class=\\\"  sorting_1\\\">392</td>\\n</tr><tr id=\\\"pkg_NBSplice\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/NBSplice.html\\\">NBSplice</a></td>\\n\\t<td class=\\\" \\\">Gabriela Merino</td>\\n\\t<td class=\\\" \\\">Negative Binomial Models to detect Differential Splicing</td>\\n\\t<td class=\\\"  sorting_1\\\">392</td>\\n</tr><tr id=\\\"pkg_OTUbase\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/OTUbase.html\\\">OTUbase</a></td>\\n\\t<td class=\\\" \\\">Daniel Beck</td>\\n\\t<td class=\\\" \\\">Provides structure and functions for the analysis of OTU data</td>\\n\\t<td class=\\\"  sorting_1\\\">392</td>\\n</tr><tr id=\\\"pkg_POWSC\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/POWSC.html\\\">POWSC</a></td>\\n\\t<td class=\\\" \\\">Kenong Su</td>\\n\\t<td class=\\\" \\\">Simulation, power evaluation, and sample size recommendation for single cell RNA-seq</td>\\n\\t<td class=\\\"  sorting_1\\\">392</td>\\n</tr><tr id=\\\"pkg_proBAMr\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/proBAMr.html\\\">proBAMr</a></td>\\n\\t<td class=\\\" \\\">Xiaojing Wang</td>\\n\\t<td class=\\\" \\\">Generating SAM file for PSMs in shotgun proteomics data</td>\\n\\t<td class=\\\"  sorting_1\\\">392</td>\\n</tr><tr id=\\\"pkg_procoil\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/procoil.html\\\">procoil</a></td>\\n\\t<td class=\\\" \\\">Ulrich Bodenhofer</td>\\n\\t<td class=\\\" \\\">Prediction of Oligomerization of Coiled Coil Proteins</td>\\n\\t<td class=\\\"  sorting_1\\\">392</td>\\n</tr><tr id=\\\"pkg_ProteoDisco\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ProteoDisco.html\\\">ProteoDisco</a></td>\\n\\t<td class=\\\" \\\">Job van Riet</td>\\n\\t<td class=\\\" \\\">Generation of customized protein variant databases from genomic variants, splice-junctions and manual sequences</td>\\n\\t<td class=\\\"  sorting_1\\\">392</td>\\n</tr><tr id=\\\"pkg_Rbec\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/Rbec.html\\\">Rbec</a></td>\\n\\t<td class=\\\" \\\">Pengfan Zhang</td>\\n\\t<td class=\\\" \\\">Rbec: a tool for analysis of amplicon sequencing data from synthetic microbial communities</td>\\n\\t<td class=\\\"  sorting_1\\\">392</td>\\n</tr><tr id=\\\"pkg_RCSL\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/RCSL.html\\\">RCSL</a></td>\\n\\t<td class=\\\" \\\">Qinglin Mei</td>\\n\\t<td class=\\\" \\\">Rank Constrained Similarity Learning for single cell RNA sequencing data</td>\\n\\t<td class=\\\"  sorting_1\\\">392</td>\\n</tr><tr id=\\\"pkg_RepViz\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/RepViz.html\\\">RepViz</a></td>\\n\\t<td class=\\\" \\\">Thomas Faux, Asta Laiho</td>\\n\\t<td class=\\\" \\\">Replicate oriented Visualization of a genomic region</td>\\n\\t<td class=\\\"  sorting_1\\\">392</td>\\n</tr><tr id=\\\"pkg_RGraph2js\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/RGraph2js.html\\\">RGraph2js</a></td>\\n\\t<td class=\\\" \\\">Stephane Cano</td>\\n\\t<td class=\\\" \\\">Convert a Graph into a D3js Script</td>\\n\\t<td class=\\\"  sorting_1\\\">392</td>\\n</tr><tr id=\\\"pkg_rqt\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/rqt.html\\\">rqt</a></td>\\n\\t<td class=\\\" \\\">Ilya Y. Zhbannikov</td>\\n\\t<td class=\\\" \\\">rqt: utilities for gene-level meta-analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">392</td>\\n</tr><tr id=\\\"pkg_RVS\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/RVS.html\\\">RVS</a></td>\\n\\t<td class=\\\" \\\">Thomas Sherman</td>\\n\\t<td class=\\\" \\\">Computes estimates of the probability of related individuals sharing a rare variant</td>\\n\\t<td class=\\\"  sorting_1\\\">392</td>\\n</tr><tr id=\\\"pkg_sigsquared\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/sigsquared.html\\\">sigsquared</a></td>\\n\\t<td class=\\\" \\\">UnJin Lee</td>\\n\\t<td class=\\\" \\\">Gene signature generation for functionally validated signaling pathways</td>\\n\\t<td class=\\\"  sorting_1\\\">392</td>\\n</tr><tr id=\\\"pkg_sitePath\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/sitePath.html\\\">sitePath</a></td>\\n\\t<td class=\\\" \\\">Chengyang Ji</td>\\n\\t<td class=\\\" \\\">Phylogeny-based sequence clustering with site polymorphism</td>\\n\\t<td class=\\\"  sorting_1\\\">392</td>\\n</tr><tr id=\\\"pkg_spkTools\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/spkTools.html\\\">spkTools</a></td>\\n\\t<td class=\\\" \\\">Matthew N McCall</td>\\n\\t<td class=\\\" \\\">Methods for Spike-in Arrays</td>\\n\\t<td class=\\\"  sorting_1\\\">392</td>\\n</tr><tr id=\\\"pkg_srnadiff\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/srnadiff.html\\\">srnadiff</a></td>\\n\\t<td class=\\\" \\\">Zytnicki Matthias</td>\\n\\t<td class=\\\" \\\">Finding differentially expressed unannotated genomic regions from RNA-seq data</td>\\n\\t<td class=\\\"  sorting_1\\\">392</td>\\n</tr><tr id=\\\"pkg_sscu\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/sscu.html\\\">sscu</a></td>\\n\\t<td class=\\\" \\\">Yu Sun</td>\\n\\t<td class=\\\" \\\">Strength of Selected Codon Usage</td>\\n\\t<td class=\\\"  sorting_1\\\">392</td>\\n</tr><tr id=\\\"pkg_StarBioTrek\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/StarBioTrek.html\\\">StarBioTrek</a></td>\\n\\t<td class=\\\" \\\">Claudia Cava</td>\\n\\t<td class=\\\" \\\">StarBioTrek</td>\\n\\t<td class=\\\"  sorting_1\\\">392</td>\\n</tr><tr id=\\\"pkg_target\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/target.html\\\">target</a></td>\\n\\t<td class=\\\" \\\">Mahmoud Ahmed</td>\\n\\t<td class=\\\" \\\">Predict Combined Function of Transcription Factors</td>\\n\\t<td class=\\\"  sorting_1\\\">392</td>\\n</tr><tr id=\\\"pkg_traseR\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/traseR.html\\\">traseR</a></td>\\n\\t<td class=\\\" \\\">li chen</td>\\n\\t<td class=\\\" \\\">GWAS trait-associated SNP enrichment analyses in genomic intervals</td>\\n\\t<td class=\\\"  sorting_1\\\">392</td>\\n</tr><tr id=\\\"pkg_TurboNorm\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/TurboNorm.html\\\">TurboNorm</a></td>\\n\\t<td class=\\\" \\\">Maarten van Iterson</td>\\n\\t<td class=\\\" \\\">A fast scatterplot smoother suitable for microarray normalization</td>\\n\\t<td class=\\\"  sorting_1\\\">392</td>\\n</tr><tr id=\\\"pkg_TVTB\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/TVTB.html\\\">TVTB</a></td>\\n\\t<td class=\\\" \\\">Kevin Rue-Albrecht</td>\\n\\t<td class=\\\" \\\">TVTB: The VCF Tool Box</td>\\n\\t<td class=\\\"  sorting_1\\\">392</td>\\n</tr><tr id=\\\"pkg_Uniquorn\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/Uniquorn.html\\\">Uniquorn</a></td>\\n\\t<td class=\\\" \\\">'Raik Otto'</td>\\n\\t<td class=\\\" \\\">Identification of cancer cell lines based on their weighted mutational/ variational fingerprint</td>\\n\\t<td class=\\\"  sorting_1\\\">392</td>\\n</tr><tr id=\\\"pkg_VegaMC\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/VegaMC.html\\\">VegaMC</a></td>\\n\\t<td class=\\\" \\\">Sandro Morganella</td>\\n\\t<td class=\\\" \\\">VegaMC: A Package Implementing a Variational Piecewise Smooth Model for Identification of Driver Chromosomal Imbalances in Cancer</td>\\n\\t<td class=\\\"  sorting_1\\\">392</td>\\n</tr><tr id=\\\"pkg_vtpnet\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/vtpnet.html\\\">vtpnet</a></td>\\n\\t<td class=\\\" \\\">VJ Carey</td>\\n\\t<td class=\\\" \\\">variant-transcription factor-phenotype networks</td>\\n\\t<td class=\\\"  sorting_1\\\">392</td>\\n</tr><tr id=\\\"pkg_wppi\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/wppi.html\\\">wppi</a></td>\\n\\t<td class=\\\" \\\">Ana Galhoz</td>\\n\\t<td class=\\\" \\\">Weighting protein-protein interactions</td>\\n\\t<td class=\\\"  sorting_1\\\">392</td>\\n</tr><tr id=\\\"pkg_banocc\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/banocc.html\\\">banocc</a></td>\\n\\t<td class=\\\" \\\">George Weingart, Curtis Huttenhower</td>\\n\\t<td class=\\\" \\\">Bayesian ANalysis Of Compositional Covariance</td>\\n\\t<td class=\\\"  sorting_1\\\">455</td>\\n</tr><tr id=\\\"pkg_BEARscc\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/BEARscc.html\\\">BEARscc</a></td>\\n\\t<td class=\\\" \\\">Benjamin Schuster-Boeckler</td>\\n\\t<td class=\\\" \\\">BEARscc (Bayesian ERCC Assesstment of Robustness of Single Cell Clusters)</td>\\n\\t<td class=\\\"  sorting_1\\\">455</td>\\n</tr><tr id=\\\"pkg_benchdamic\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/benchdamic.html\\\">benchdamic</a></td>\\n\\t<td class=\\\" \\\">Matteo Calgaro</td>\\n\\t<td class=\\\" \\\">Benchmark of differential abundance methods on microbiome data</td>\\n\\t<td class=\\\"  sorting_1\\\">455</td>\\n</tr><tr id=\\\"pkg_biodbChebi\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/biodbChebi.html\\\">biodbChebi</a></td>\\n\\t<td class=\\\" \\\">Pierrick Roger</td>\\n\\t<td class=\\\" \\\">biodbChebi, a library for connecting to the ChEBI Database</td>\\n\\t<td class=\\\"  sorting_1\\\">455</td>\\n</tr><tr id=\\\"pkg_biodbUniprot\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/biodbUniprot.html\\\">biodbUniprot</a></td>\\n\\t<td class=\\\" \\\">Pierrick Roger</td>\\n\\t<td class=\\\" \\\">biodbUniprot, a library for connecting to the Uniprot Database</td>\\n\\t<td class=\\\"  sorting_1\\\">455</td>\\n</tr><tr id=\\\"pkg_bridge\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/bridge.html\\\">bridge</a></td>\\n\\t<td class=\\\" \\\">Raphael Gottardo</td>\\n\\t<td class=\\\" \\\">Bayesian Robust Inference for Differential Gene Expression</td>\\n\\t<td class=\\\"  sorting_1\\\">455</td>\\n</tr><tr id=\\\"pkg_CellScore\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/CellScore.html\\\">CellScore</a></td>\\n\\t<td class=\\\" \\\">Nancy Mah</td>\\n\\t<td class=\\\" \\\">Tool for Evaluation of Cell Identity from Transcription Profiles</td>\\n\\t<td class=\\\"  sorting_1\\\">455</td>\\n</tr><tr id=\\\"pkg_cfDNAPro\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/cfDNAPro.html\\\">cfDNAPro</a></td>\\n\\t<td class=\\\" \\\">Haichao Wang</td>\\n\\t<td class=\\\" \\\">cfDNAPro extracts and Visualises biological features from whole genome sequencing data of cell-free DNA</td>\\n\\t<td class=\\\"  sorting_1\\\">455</td>\\n</tr><tr id=\\\"pkg_ChIPXpress\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ChIPXpress.html\\\">ChIPXpress</a></td>\\n\\t<td class=\\\" \\\">George Wu</td>\\n\\t<td class=\\\" \\\">ChIPXpress: enhanced transcription factor target gene identification from ChIP-seq and ChIP-chip data using publicly available gene expression profiles</td>\\n\\t<td class=\\\"  sorting_1\\\">455</td>\\n</tr><tr id=\\\"pkg_clusterSeq\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/clusterSeq.html\\\">clusterSeq</a></td>\\n\\t<td class=\\\" \\\">Thomas J. Hardcastle</td>\\n\\t<td class=\\\" \\\">Clustering of high-throughput sequencing data by identifying co-expression patterns</td>\\n\\t<td class=\\\"  sorting_1\\\">455</td>\\n</tr><tr id=\\\"pkg_ClusterSignificance\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ClusterSignificance.html\\\">ClusterSignificance</a></td>\\n\\t<td class=\\\" \\\">Jason T Serviss</td>\\n\\t<td class=\\\" \\\">The ClusterSignificance package provides tools to assess if class clusters in dimensionality reduced data representations have a separation different from permuted data</td>\\n\\t<td class=\\\"  sorting_1\\\">455</td>\\n</tr><tr id=\\\"pkg_combi\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/combi.html\\\">combi</a></td>\\n\\t<td class=\\\" \\\">Stijn Hawinkel</td>\\n\\t<td class=\\\" \\\">Compositional omics model based visual integration</td>\\n\\t<td class=\\\"  sorting_1\\\">455</td>\\n</tr><tr id=\\\"pkg_consensus\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/consensus.html\\\">consensus</a></td>\\n\\t<td class=\\\" \\\">Tim Peters</td>\\n\\t<td class=\\\" \\\">Cross-platform consensus analysis of genomic measurements via interlaboratory testing method</td>\\n\\t<td class=\\\"  sorting_1\\\">455</td>\\n</tr><tr id=\\\"pkg_covEB\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/covEB.html\\\">covEB</a></td>\\n\\t<td class=\\\" \\\">C. Pacini</td>\\n\\t<td class=\\\" \\\">Empirical Bayes estimate of block diagonal covariance matrices</td>\\n\\t<td class=\\\"  sorting_1\\\">455</td>\\n</tr><tr id=\\\"pkg_csdR\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/csdR.html\\\">csdR</a></td>\\n\\t<td class=\\\" \\\">Jakob Peder Pettersen</td>\\n\\t<td class=\\\" \\\">Differential gene co-expression</td>\\n\\t<td class=\\\"  sorting_1\\\">455</td>\\n</tr><tr id=\\\"pkg_cycle\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/cycle.html\\\">cycle</a></td>\\n\\t<td class=\\\" \\\">Matthias Futschik</td>\\n\\t<td class=\\\" \\\">Significance of periodic expression pattern in time-series data</td>\\n\\t<td class=\\\"  sorting_1\\\">455</td>\\n</tr><tr id=\\\"pkg_cytoKernel\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/cytoKernel.html\\\">cytoKernel</a></td>\\n\\t<td class=\\\" \\\">Tusharkanti Ghosh</td>\\n\\t<td class=\\\" \\\">Differential expression using kernel-based score test</td>\\n\\t<td class=\\\"  sorting_1\\\">455</td>\\n</tr><tr id=\\\"pkg_DEComplexDisease\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/DEComplexDisease.html\\\">DEComplexDisease</a></td>\\n\\t<td class=\\\" \\\">Guofeng Meng</td>\\n\\t<td class=\\\" \\\">A tool for differential expression analysis and DEGs based investigation to complex diseases by bi-clustering analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">455</td>\\n</tr><tr id=\\\"pkg_DeepPINCS\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/DeepPINCS.html\\\">DeepPINCS</a></td>\\n\\t<td class=\\\" \\\">Dongmin Jung</td>\\n\\t<td class=\\\" \\\">Protein Interactions and Networks with Compounds based on Sequences using Deep Learning</td>\\n\\t<td class=\\\"  sorting_1\\\">455</td>\\n</tr><tr id=\\\"pkg_diffUTR\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/diffUTR.html\\\">diffUTR</a></td>\\n\\t<td class=\\\" \\\">Pierre-Luc Germain</td>\\n\\t<td class=\\\" \\\">diffUTR: Streamlining differential exon and 3' UTR usage</td>\\n\\t<td class=\\\"  sorting_1\\\">455</td>\\n</tr><tr id=\\\"pkg_DMCHMM\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/DMCHMM.html\\\">DMCHMM</a></td>\\n\\t<td class=\\\" \\\">Farhad Shokoohi</td>\\n\\t<td class=\\\" \\\">Differentially Methylated CpG using Hidden Markov Model</td>\\n\\t<td class=\\\"  sorting_1\\\">455</td>\\n</tr><tr id=\\\"pkg_easyreporting\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/easyreporting.html\\\">easyreporting</a></td>\\n\\t<td class=\\\" \\\">Dario Righelli</td>\\n\\t<td class=\\\" \\\">Helps creating report for improving Reproducible Computational Research</td>\\n\\t<td class=\\\"  sorting_1\\\">455</td>\\n</tr><tr id=\\\"pkg_epialleleR\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/epialleleR.html\\\">epialleleR</a></td>\\n\\t<td class=\\\" \\\">Oleksii Nikolaienko</td>\\n\\t<td class=\\\" \\\">Fast, Epiallele-Aware Methylation Reporter</td>\\n\\t<td class=\\\"  sorting_1\\\">455</td>\\n</tr><tr id=\\\"pkg_epistack\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/epistack.html\\\">epistack</a></td>\\n\\t<td class=\\\" \\\">DEVAILLY Guillaume</td>\\n\\t<td class=\\\" \\\">Heatmaps of Stack Profiles from Epigenetic Signals</td>\\n\\t<td class=\\\"  sorting_1\\\">455</td>\\n</tr><tr id=\\\"pkg_evaluomeR\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/evaluomeR.html\\\">evaluomeR</a></td>\\n\\t<td class=\\\" \\\">Jos Antonio Bernab-Daz</td>\\n\\t<td class=\\\" \\\">Evaluation of Bioinformatics Metrics</td>\\n\\t<td class=\\\"  sorting_1\\\">455</td>\\n</tr><tr id=\\\"pkg_fcScan\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/fcScan.html\\\">fcScan</a></td>\\n\\t<td class=\\\" \\\">Pierre Khoueiry Abdullah El-Kurdi</td>\\n\\t<td class=\\\" \\\">fcScan for detecting clusters of coordinates with user defined options</td>\\n\\t<td class=\\\"  sorting_1\\\">455</td>\\n</tr><tr id=\\\"pkg_flowCHIC\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/flowCHIC.html\\\">flowCHIC</a></td>\\n\\t<td class=\\\" \\\">Author: Joachim Schumann</td>\\n\\t<td class=\\\" \\\">Analyze flow cytometric data using histogram information</td>\\n\\t<td class=\\\"  sorting_1\\\">455</td>\\n</tr><tr id=\\\"pkg_flowCL\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/flowCL.html\\\">flowCL</a></td>\\n\\t<td class=\\\" \\\">Justin Meskas</td>\\n\\t<td class=\\\" \\\">Semantic labelling of flow cytometric cell populations</td>\\n\\t<td class=\\\"  sorting_1\\\">455</td>\\n</tr><tr id=\\\"pkg_FRGEpistasis\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/FRGEpistasis.html\\\">FRGEpistasis</a></td>\\n\\t<td class=\\\" \\\">Futao Zhang</td>\\n\\t<td class=\\\" \\\">Epistasis Analysis for Quantitative Traits by Functional Regression Model</td>\\n\\t<td class=\\\"  sorting_1\\\">455</td>\\n</tr><tr id=\\\"pkg_FunChIP\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/FunChIP.html\\\">FunChIP</a></td>\\n\\t<td class=\\\" \\\">Alice Parodi</td>\\n\\t<td class=\\\" \\\">Clustering and Alignment of ChIP-Seq peaks based on their shapes</td>\\n\\t<td class=\\\"  sorting_1\\\">455</td>\\n</tr><tr id=\\\"pkg_geneAttribution\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/geneAttribution.html\\\">geneAttribution</a></td>\\n\\t<td class=\\\" \\\">Arthur Wuster</td>\\n\\t<td class=\\\" \\\">Identification of candidate genes associated with genetic variation</td>\\n\\t<td class=\\\"  sorting_1\\\">455</td>\\n</tr><tr id=\\\"pkg_geneClassifiers\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/geneClassifiers.html\\\">geneClassifiers</a></td>\\n\\t<td class=\\\" \\\">R Kuiper</td>\\n\\t<td class=\\\" \\\">Application of gene classifiers</td>\\n\\t<td class=\\\"  sorting_1\\\">455</td>\\n</tr><tr id=\\\"pkg_hapFabia\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/hapFabia.html\\\">hapFabia</a></td>\\n\\t<td class=\\\" \\\">Andreas Mitterecker</td>\\n\\t<td class=\\\" \\\">hapFabia: Identification of very short segments of identity by descent (IBD) characterized by rare variants in large sequencing data</td>\\n\\t<td class=\\\"  sorting_1\\\">455</td>\\n</tr><tr id=\\\"pkg_HiLDA\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/HiLDA.html\\\">HiLDA</a></td>\\n\\t<td class=\\\" \\\">Zhi Yang</td>\\n\\t<td class=\\\" \\\">Conducting statistical inference on comparing the mutational exposures of mutational signatures by using hierarchical latent Dirichlet allocation</td>\\n\\t<td class=\\\"  sorting_1\\\">455</td>\\n</tr><tr id=\\\"pkg_HIPPO\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/HIPPO.html\\\">HIPPO</a></td>\\n\\t<td class=\\\" \\\">Tae Kim</td>\\n\\t<td class=\\\" \\\">Heterogeneity-Induced Pre-Processing tOol</td>\\n\\t<td class=\\\"  sorting_1\\\">455</td>\\n</tr><tr id=\\\"pkg_InTAD\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/InTAD.html\\\">InTAD</a></td>\\n\\t<td class=\\\" \\\">Konstantin Okonechnikov</td>\\n\\t<td class=\\\" \\\">Search for correlation between epigenetic signals and gene expression in TADs</td>\\n\\t<td class=\\\"  sorting_1\\\">455</td>\\n</tr><tr id=\\\"pkg_iterativeBMAsurv\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/iterativeBMAsurv.html\\\">iterativeBMAsurv</a></td>\\n\\t<td class=\\\" \\\">Ka Yee Yeung</td>\\n\\t<td class=\\\" \\\">The Iterative Bayesian Model Averaging (BMA) Algorithm For Survival Analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">455</td>\\n</tr><tr id=\\\"pkg_kissDE\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/kissDE.html\\\">kissDE</a></td>\\n\\t<td class=\\\" \\\">Aurlie Siberchicot</td>\\n\\t<td class=\\\" \\\">Retrieves Condition-Specific Variants in RNA-Seq Data</td>\\n\\t<td class=\\\"  sorting_1\\\">455</td>\\n</tr><tr id=\\\"pkg_LRcell\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/LRcell.html\\\">LRcell</a></td>\\n\\t<td class=\\\" \\\">Wenjing Ma</td>\\n\\t<td class=\\\" \\\">Differential cell type change analysis using Logistic/linear Regression</td>\\n\\t<td class=\\\"  sorting_1\\\">455</td>\\n</tr><tr id=\\\"pkg_MAGAR\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/MAGAR.html\\\">MAGAR</a></td>\\n\\t<td class=\\\" \\\">Michael Scherer</td>\\n\\t<td class=\\\" \\\">MAGAR: R-package to compute methylation Quantitative Trait Loci (methQTL) from DNA methylation and genotyping data</td>\\n\\t<td class=\\\"  sorting_1\\\">455</td>\\n</tr><tr id=\\\"pkg_mBPCR\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/mBPCR.html\\\">mBPCR</a></td>\\n\\t<td class=\\\" \\\">P.M.V. Rancoita</td>\\n\\t<td class=\\\" \\\">Bayesian Piecewise Constant Regression for DNA copy number estimation</td>\\n\\t<td class=\\\"  sorting_1\\\">455</td>\\n</tr><tr id=\\\"pkg_MEAT\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/MEAT.html\\\">MEAT</a></td>\\n\\t<td class=\\\" \\\">Sarah Voisin</td>\\n\\t<td class=\\\" \\\">Muscle Epigenetic Age Test</td>\\n\\t<td class=\\\"  sorting_1\\\">455</td>\\n</tr><tr id=\\\"pkg_messina\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/messina.html\\\">messina</a></td>\\n\\t<td class=\\\" \\\">Mark Pinese</td>\\n\\t<td class=\\\" \\\">Single-gene classifiers and outlier-resistant detection of differential expression for two-group and survival problems</td>\\n\\t<td class=\\\"  sorting_1\\\">455</td>\\n</tr><tr id=\\\"pkg_microbiomeDASim\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/microbiomeDASim.html\\\">microbiomeDASim</a></td>\\n\\t<td class=\\\" \\\">Justin Williams</td>\\n\\t<td class=\\\" \\\">Microbiome Differential Abundance Simulation</td>\\n\\t<td class=\\\"  sorting_1\\\">455</td>\\n</tr><tr id=\\\"pkg_mimager\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/mimager.html\\\">mimager</a></td>\\n\\t<td class=\\\" \\\">Aaron Wolen</td>\\n\\t<td class=\\\" \\\">mimager: The Microarray Imager</td>\\n\\t<td class=\\\"  sorting_1\\\">455</td>\\n</tr><tr id=\\\"pkg_mirIntegrator\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/mirIntegrator.html\\\">mirIntegrator</a></td>\\n\\t<td class=\\\" \\\">Diana Diaz</td>\\n\\t<td class=\\\" \\\">Integrating microRNA expression into signaling pathways for pathway analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">455</td>\\n</tr><tr id=\\\"pkg_mosbi\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/mosbi.html\\\">mosbi</a></td>\\n\\t<td class=\\\" \\\">Tim Daniel Rose</td>\\n\\t<td class=\\\" \\\">Molecular Signature identification using Biclustering</td>\\n\\t<td class=\\\"  sorting_1\\\">455</td>\\n</tr><tr id=\\\"pkg_MOSim\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/MOSim.html\\\">MOSim</a></td>\\n\\t<td class=\\\" \\\">Carlos Martnez</td>\\n\\t<td class=\\\" \\\">Multi-Omics Simulation (MOSim)</td>\\n\\t<td class=\\\"  sorting_1\\\">455</td>\\n</tr><tr id=\\\"pkg_MSstatsQCgui\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/MSstatsQCgui.html\\\">MSstatsQCgui</a></td>\\n\\t<td class=\\\" \\\">Eralp Dogu</td>\\n\\t<td class=\\\" \\\">A graphical user interface for MSstatsQC package</td>\\n\\t<td class=\\\"  sorting_1\\\">455</td>\\n</tr><tr id=\\\"pkg_MSstatsSampleSize\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/MSstatsSampleSize.html\\\">MSstatsSampleSize</a></td>\\n\\t<td class=\\\" \\\">Ting Huang</td>\\n\\t<td class=\\\" \\\">Simulation tool for optimal design of high-dimensional MS-based proteomics experiment</td>\\n\\t<td class=\\\"  sorting_1\\\">455</td>\\n</tr><tr id=\\\"pkg_NADfinder\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/NADfinder.html\\\">NADfinder</a></td>\\n\\t<td class=\\\" \\\">Jianhong Ou, Lihua Julie Zhu</td>\\n\\t<td class=\\\" \\\">Call wide peaks for sequencing data</td>\\n\\t<td class=\\\"  sorting_1\\\">455</td>\\n</tr><tr id=\\\"pkg_netOmics\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/netOmics.html\\\">netOmics</a></td>\\n\\t<td class=\\\" \\\">Antoine Bodein</td>\\n\\t<td class=\\\" \\\">Multi-Omics (time-course) network-based integration and interpretation</td>\\n\\t<td class=\\\"  sorting_1\\\">455</td>\\n</tr><tr id=\\\"pkg_netZooR\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/netZooR.html\\\">netZooR</a></td>\\n\\t<td class=\\\" \\\">Marouen Ben Guebila</td>\\n\\t<td class=\\\" \\\">Unified methods for the inference and analysis of gene regulatory networks</td>\\n\\t<td class=\\\"  sorting_1\\\">455</td>\\n</tr><tr id=\\\"pkg_nuCpos\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/nuCpos.html\\\">nuCpos</a></td>\\n\\t<td class=\\\" \\\">Hiroaki Kato</td>\\n\\t<td class=\\\" \\\">An R package for prediction of nucleosome positions</td>\\n\\t<td class=\\\"  sorting_1\\\">455</td>\\n</tr><tr id=\\\"pkg_OmicsLonDA\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/OmicsLonDA.html\\\">OmicsLonDA</a></td>\\n\\t<td class=\\\" \\\">Ahmed A. Metwally</td>\\n\\t<td class=\\\" \\\">Omics Longitudinal Differential Analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">455</td>\\n</tr><tr id=\\\"pkg_OMICsPCA\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/OMICsPCA.html\\\">OMICsPCA</a></td>\\n\\t<td class=\\\" \\\">Subhadeep Das</td>\\n\\t<td class=\\\" \\\">An R package for quantitative integration and analysis of multiple omics assays from heterogeneous samples</td>\\n\\t<td class=\\\"  sorting_1\\\">455</td>\\n</tr><tr id=\\\"pkg_ORFhunteR\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ORFhunteR.html\\\">ORFhunteR</a></td>\\n\\t<td class=\\\" \\\">Vasily V. Grinev</td>\\n\\t<td class=\\\" \\\">Predict open reading frames in nucleotide sequences</td>\\n\\t<td class=\\\"  sorting_1\\\">455</td>\\n</tr><tr id=\\\"pkg_packFinder\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/packFinder.html\\\">packFinder</a></td>\\n\\t<td class=\\\" \\\">Jack Gisby</td>\\n\\t<td class=\\\" \\\">de novo Annotation of Pack-TYPE Transposable Elements</td>\\n\\t<td class=\\\"  sorting_1\\\">455</td>\\n</tr><tr id=\\\"pkg_pageRank\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/pageRank.html\\\">pageRank</a></td>\\n\\t<td class=\\\" \\\">Hongxu Ding</td>\\n\\t<td class=\\\" \\\">Temporal and Multiplex PageRank for Gene Regulatory Network Analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">455</td>\\n</tr><tr id=\\\"pkg_PANR\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/PANR.html\\\">PANR</a></td>\\n\\t<td class=\\\" \\\">Xin Wang</td>\\n\\t<td class=\\\" \\\">Posterior association networks and functional modules inferred from rich phenotypes of gene perturbations</td>\\n\\t<td class=\\\"  sorting_1\\\">455</td>\\n</tr><tr id=\\\"pkg_PAST\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/PAST.html\\\">PAST</a></td>\\n\\t<td class=\\\" \\\">Thrash Adam</td>\\n\\t<td class=\\\" \\\">Pathway Association Study Tool (PAST)</td>\\n\\t<td class=\\\"  sorting_1\\\">455</td>\\n</tr><tr id=\\\"pkg_periodicDNA\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/periodicDNA.html\\\">periodicDNA</a></td>\\n\\t<td class=\\\" \\\">Jacques Serizay</td>\\n\\t<td class=\\\" \\\">Set of tools to identify periodic occurrences of k-mers in DNA sequences</td>\\n\\t<td class=\\\"  sorting_1\\\">455</td>\\n</tr><tr id=\\\"pkg_PING\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/PING.html\\\">PING</a></td>\\n\\t<td class=\\\" \\\">Renan Sauteraud</td>\\n\\t<td class=\\\" \\\">Probabilistic inference for Nucleosome Positioning with MNase-based or Sonicated Short-read Data</td>\\n\\t<td class=\\\"  sorting_1\\\">455</td>\\n</tr><tr id=\\\"pkg_PROMISE\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/PROMISE.html\\\">PROMISE</a></td>\\n\\t<td class=\\\" \\\">Stan Pounds, Xueyuan Cao</td>\\n\\t<td class=\\\" \\\">PRojection Onto the Most Interesting Statistical Evidence</td>\\n\\t<td class=\\\"  sorting_1\\\">455</td>\\n</tr><tr id=\\\"pkg_Qtlizer\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/Qtlizer.html\\\">Qtlizer</a></td>\\n\\t<td class=\\\" \\\">Matthias Munz</td>\\n\\t<td class=\\\" \\\">Comprehensive QTL annotation of GWAS results</td>\\n\\t<td class=\\\"  sorting_1\\\">455</td>\\n</tr><tr id=\\\"pkg_ReQON\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ReQON.html\\\">ReQON</a></td>\\n\\t<td class=\\\" \\\">Christopher Cabanski</td>\\n\\t<td class=\\\" \\\">Recalibrating Quality Of Nucleotides</td>\\n\\t<td class=\\\"  sorting_1\\\">455</td>\\n</tr><tr id=\\\"pkg_rfaRm\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/rfaRm.html\\\">rfaRm</a></td>\\n\\t<td class=\\\" \\\">Lara Selles Vidal, Rafael Ayala</td>\\n\\t<td class=\\\" \\\">An R interface to the Rfam database</td>\\n\\t<td class=\\\"  sorting_1\\\">455</td>\\n</tr><tr id=\\\"pkg_rfPred\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/rfPred.html\\\">rfPred</a></td>\\n\\t<td class=\\\" \\\">Hugo Varet</td>\\n\\t<td class=\\\" \\\">Assign rfPred functional prediction scores to a missense variants list</td>\\n\\t<td class=\\\"  sorting_1\\\">455</td>\\n</tr><tr id=\\\"pkg_RNAmodR.RiboMethSeq\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/RNAmodR.RiboMethSeq.html\\\">RNAmodR.RiboMethSeq</a></td>\\n\\t<td class=\\\" \\\">Felix G.M. Ernst</td>\\n\\t<td class=\\\" \\\">Detection of 2'-O methylations by RiboMethSeq</td>\\n\\t<td class=\\\"  sorting_1\\\">455</td>\\n</tr><tr id=\\\"pkg_sampleClassifier\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/sampleClassifier.html\\\">sampleClassifier</a></td>\\n\\t<td class=\\\" \\\">Khadija El Amrani</td>\\n\\t<td class=\\\" \\\">Sample Classifier</td>\\n\\t<td class=\\\"  sorting_1\\\">455</td>\\n</tr><tr id=\\\"pkg_scAlign\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/scAlign.html\\\">scAlign</a></td>\\n\\t<td class=\\\" \\\">Nelson Johansen</td>\\n\\t<td class=\\\" \\\">An alignment and integration method for single cell genomics</td>\\n\\t<td class=\\\"  sorting_1\\\">455</td>\\n</tr><tr id=\\\"pkg_scanMiR\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/scanMiR.html\\\">scanMiR</a></td>\\n\\t<td class=\\\" \\\">Fridolin Gross</td>\\n\\t<td class=\\\" \\\">scanMiR</td>\\n\\t<td class=\\\"  sorting_1\\\">455</td>\\n</tr><tr id=\\\"pkg_SDAMS\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/SDAMS.html\\\">SDAMS</a></td>\\n\\t<td class=\\\" \\\">Yuntong Li</td>\\n\\t<td class=\\\" \\\">Differential Abundant/Expression Analysis for Metabolomics, Proteomics and single-cell RNA sequencing Data</td>\\n\\t<td class=\\\"  sorting_1\\\">455</td>\\n</tr><tr id=\\\"pkg_SELEX\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/SELEX.html\\\">SELEX</a></td>\\n\\t<td class=\\\" \\\">Harmen J. Bussemaker</td>\\n\\t<td class=\\\" \\\">Functions for analyzing SELEX-seq data</td>\\n\\t<td class=\\\"  sorting_1\\\">455</td>\\n</tr><tr id=\\\"pkg_seqcombo\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/seqcombo.html\\\">seqcombo</a></td>\\n\\t<td class=\\\" \\\">Guangchuang Yu</td>\\n\\t<td class=\\\" \\\">Visualization Tool for Genetic Reassortment</td>\\n\\t<td class=\\\"  sorting_1\\\">455</td>\\n</tr><tr id=\\\"pkg_sevenC\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/sevenC.html\\\">sevenC</a></td>\\n\\t<td class=\\\" \\\">Jonas Ibn-Salem</td>\\n\\t<td class=\\\" \\\">Computational Chromosome Conformation Capture by Correlation of ChIP-seq at CTCF motifs</td>\\n\\t<td class=\\\"  sorting_1\\\">455</td>\\n</tr><tr id=\\\"pkg_SigCheck\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/SigCheck.html\\\">SigCheck</a></td>\\n\\t<td class=\\\" \\\">Rory Stark</td>\\n\\t<td class=\\\" \\\">Check a gene signature's prognostic performance against random signatures, known signatures, and permuted data/metadata</td>\\n\\t<td class=\\\"  sorting_1\\\">455</td>\\n</tr><tr id=\\\"pkg_SingleMoleculeFootprinting\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/SingleMoleculeFootprinting.html\\\">SingleMoleculeFootprinting</a></td>\\n\\t<td class=\\\" \\\">Guido Barzaghi</td>\\n\\t<td class=\\\" \\\">Analysis tools for Single Molecule Footprinting (SMF) data</td>\\n\\t<td class=\\\"  sorting_1\\\">455</td>\\n</tr><tr id=\\\"pkg_SMAP\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/SMAP.html\\\">SMAP</a></td>\\n\\t<td class=\\\" \\\">Robin Andersson</td>\\n\\t<td class=\\\" \\\">A Segmental Maximum A Posteriori Approach to Array-CGH Copy Number Profiling</td>\\n\\t<td class=\\\"  sorting_1\\\">455</td>\\n</tr><tr id=\\\"pkg_sparseDOSSA\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/sparseDOSSA.html\\\">sparseDOSSA</a></td>\\n\\t<td class=\\\" \\\">Boyu Ren, Emma Schwager, George Weingart</td>\\n\\t<td class=\\\" \\\">Sparse Data Observations for Simulating Synthetic Abundance</td>\\n\\t<td class=\\\"  sorting_1\\\">455</td>\\n</tr><tr id=\\\"pkg_spiky\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/spiky.html\\\">spiky</a></td>\\n\\t<td class=\\\" \\\">Tim Triche</td>\\n\\t<td class=\\\" \\\">Spike-in calibration for cell-free MeDIP</td>\\n\\t<td class=\\\"  sorting_1\\\">455</td>\\n</tr><tr id=\\\"pkg_standR\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/standR.html\\\">standR</a></td>\\n\\t<td class=\\\" \\\">Ning Liu</td>\\n\\t<td class=\\\" \\\">Spatial transcriptome analyses of Nanostring's DSP data in R</td>\\n\\t<td class=\\\"  sorting_1\\\">455</td>\\n</tr><tr id=\\\"pkg_supersigs\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/supersigs.html\\\">supersigs</a></td>\\n\\t<td class=\\\" \\\">Albert Kuo</td>\\n\\t<td class=\\\" \\\">Supervised mutational signatures</td>\\n\\t<td class=\\\"  sorting_1\\\">455</td>\\n</tr><tr id=\\\"pkg_synapter\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/synapter.html\\\">synapter</a></td>\\n\\t<td class=\\\" \\\">Laurent Gatto Sebastian Gibb</td>\\n\\t<td class=\\\" \\\">Label-free data analysis pipeline for optimal identification and quantitation</td>\\n\\t<td class=\\\"  sorting_1\\\">455</td>\\n</tr><tr id=\\\"pkg_TargetDecoy\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/TargetDecoy.html\\\">TargetDecoy</a></td>\\n\\t<td class=\\\" \\\">Elke Debrie</td>\\n\\t<td class=\\\" \\\">Diagnostic Plots to Evaluate the Target Decoy Approach</td>\\n\\t<td class=\\\"  sorting_1\\\">455</td>\\n</tr><tr id=\\\"pkg_TFutils\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/TFutils.html\\\">TFutils</a></td>\\n\\t<td class=\\\" \\\">Vincent Carey</td>\\n\\t<td class=\\\" \\\">TFutils</td>\\n\\t<td class=\\\"  sorting_1\\\">455</td>\\n</tr><tr id=\\\"pkg_TileDBArray\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/TileDBArray.html\\\">TileDBArray</a></td>\\n\\t<td class=\\\" \\\">Aaron Lun</td>\\n\\t<td class=\\\" \\\">Using TileDB as a DelayedArray Backend</td>\\n\\t<td class=\\\"  sorting_1\\\">455</td>\\n</tr><tr id=\\\"pkg_TIN\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/TIN.html\\\">TIN</a></td>\\n\\t<td class=\\\" \\\">Bjarne Johannessen</td>\\n\\t<td class=\\\" \\\">Transcriptome instability analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">455</td>\\n</tr><tr id=\\\"pkg_tRanslatome\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/tRanslatome.html\\\">tRanslatome</a></td>\\n\\t<td class=\\\" \\\">Toma Tebaldi, Erik Dassi</td>\\n\\t<td class=\\\" \\\">Comparison between multiple levels of gene expression</td>\\n\\t<td class=\\\"  sorting_1\\\">455</td>\\n</tr><tr id=\\\"pkg_trena\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/trena.html\\\">trena</a></td>\\n\\t<td class=\\\" \\\">Paul Shannon</td>\\n\\t<td class=\\\" \\\">Fit transcriptional regulatory networks using gene expression, priors, machine learning</td>\\n\\t<td class=\\\"  sorting_1\\\">455</td>\\n</tr><tr id=\\\"pkg_trigger\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/trigger.html\\\">trigger</a></td>\\n\\t<td class=\\\" \\\">John D. Storey</td>\\n\\t<td class=\\\" \\\">Transcriptional Regulatory Inference from Genetics of Gene ExpRession</td>\\n\\t<td class=\\\"  sorting_1\\\">455</td>\\n</tr><tr id=\\\"pkg_TypeInfo\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/TypeInfo.html\\\">TypeInfo</a></td>\\n\\t<td class=\\\" \\\">Duncan Temple Lang</td>\\n\\t<td class=\\\" \\\">Optional Type Specification Prototype</td>\\n\\t<td class=\\\"  sorting_1\\\">455</td>\\n</tr><tr id=\\\"pkg_Ularcirc\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/Ularcirc.html\\\">Ularcirc</a></td>\\n\\t<td class=\\\" \\\">David Humphreys</td>\\n\\t<td class=\\\" \\\">Shiny app for canonical and back splicing analysis (i.e. circular and mRNA analysis)</td>\\n\\t<td class=\\\"  sorting_1\\\">455</td>\\n</tr><tr id=\\\"pkg_UMI4Cats\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/UMI4Cats.html\\\">UMI4Cats</a></td>\\n\\t<td class=\\\" \\\">Mireia Ramos-Rodriguez</td>\\n\\t<td class=\\\" \\\">UMI4Cats: Processing, analysis and visualization of UMI-4C chromatin contact data</td>\\n\\t<td class=\\\"  sorting_1\\\">455</td>\\n</tr><tr id=\\\"pkg_unifiedWMWqPCR\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/unifiedWMWqPCR.html\\\">unifiedWMWqPCR</a></td>\\n\\t<td class=\\\" \\\">Joris Meys</td>\\n\\t<td class=\\\" \\\">Unified Wilcoxon-Mann Whitney Test for testing differential expression in qPCR data</td>\\n\\t<td class=\\\"  sorting_1\\\">455</td>\\n</tr><tr id=\\\"pkg_VCFArray\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/VCFArray.html\\\">VCFArray</a></td>\\n\\t<td class=\\\" \\\">Qian Liu</td>\\n\\t<td class=\\\" \\\">Representing on-disk / remote VCF files as array-like objects</td>\\n\\t<td class=\\\"  sorting_1\\\">455</td>\\n</tr><tr id=\\\"pkg_airpart\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/airpart.html\\\">airpart</a></td>\\n\\t<td class=\\\" \\\">Wancen Mu</td>\\n\\t<td class=\\\" \\\">Differential cell-type-specific allelic imbalance</td>\\n\\t<td class=\\\"  sorting_1\\\">551</td>\\n</tr><tr id=\\\"pkg_AnVILBilling\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/AnVILBilling.html\\\">AnVILBilling</a></td>\\n\\t<td class=\\\" \\\">Vince Carey</td>\\n\\t<td class=\\\" \\\">Provide functions to retrieve and report on usage expenses in NHGRI AnVIL (anvilproject.org).</td>\\n\\t<td class=\\\"  sorting_1\\\">551</td>\\n</tr><tr id=\\\"pkg_APL\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/APL.html\\\">APL</a></td>\\n\\t<td class=\\\" \\\">Elzbieta Gralinska</td>\\n\\t<td class=\\\" \\\">Association Plots</td>\\n\\t<td class=\\\"  sorting_1\\\">551</td>\\n</tr><tr id=\\\"pkg_ASpediaFI\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ASpediaFI.html\\\">ASpediaFI</a></td>\\n\\t<td class=\\\" \\\">Doyeong Yu</td>\\n\\t<td class=\\\" \\\">ASpedia-FI: Functional Interaction Analysis of Alternative Splicing Events</td>\\n\\t<td class=\\\"  sorting_1\\\">551</td>\\n</tr><tr id=\\\"pkg_barcodetrackR\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/barcodetrackR.html\\\">barcodetrackR</a></td>\\n\\t<td class=\\\" \\\">Diego Alexander Espinoza</td>\\n\\t<td class=\\\" \\\">Functions for Analyzing Cellular Barcoding Data</td>\\n\\t<td class=\\\"  sorting_1\\\">551</td>\\n</tr><tr id=\\\"pkg_BUScorrect\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/BUScorrect.html\\\">BUScorrect</a></td>\\n\\t<td class=\\\" \\\">Xiangyu Luo</td>\\n\\t<td class=\\\" \\\">Batch Effects Correction with Unknown Subtypes</td>\\n\\t<td class=\\\"  sorting_1\\\">551</td>\\n</tr><tr id=\\\"pkg_calm\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/calm.html\\\">calm</a></td>\\n\\t<td class=\\\" \\\">Kun Liang</td>\\n\\t<td class=\\\" \\\">Covariate Assisted Large-scale Multiple testing</td>\\n\\t<td class=\\\"  sorting_1\\\">551</td>\\n</tr><tr id=\\\"pkg_chromDraw\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/chromDraw.html\\\">chromDraw</a></td>\\n\\t<td class=\\\" \\\">Jan Janecka</td>\\n\\t<td class=\\\" \\\">chromDraw is a R package for drawing the schemes of karyotypes in the linear and circular fashion.</td>\\n\\t<td class=\\\"  sorting_1\\\">551</td>\\n</tr><tr id=\\\"pkg_CNViz\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/CNViz.html\\\">CNViz</a></td>\\n\\t<td class=\\\" \\\">Rebecca Greenblatt</td>\\n\\t<td class=\\\" \\\">Copy Number Visualization</td>\\n\\t<td class=\\\"  sorting_1\\\">551</td>\\n</tr><tr id=\\\"pkg_CNVrd2\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/CNVrd2.html\\\">CNVrd2</a></td>\\n\\t<td class=\\\" \\\">Hoang Tan Nguyen</td>\\n\\t<td class=\\\" \\\">CNVrd2: a read depth-based method to detect and genotype complex common copy number variants from next generation sequencing data.</td>\\n\\t<td class=\\\"  sorting_1\\\">551</td>\\n</tr><tr id=\\\"pkg_CoCiteStats\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/CoCiteStats.html\\\">CoCiteStats</a></td>\\n\\t<td class=\\\" \\\">Bioconductor Package Maintainer</td>\\n\\t<td class=\\\" \\\">Different test statistics based on co-citation.</td>\\n\\t<td class=\\\"  sorting_1\\\">551</td>\\n</tr><tr id=\\\"pkg_CONFESS\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/CONFESS.html\\\">CONFESS</a></td>\\n\\t<td class=\\\" \\\">Diana LOW</td>\\n\\t<td class=\\\" \\\">Cell OrderiNg by FluorEScence Signal</td>\\n\\t<td class=\\\"  sorting_1\\\">551</td>\\n</tr><tr id=\\\"pkg_dcGSA\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/dcGSA.html\\\">dcGSA</a></td>\\n\\t<td class=\\\" \\\">Jiehuan sun</td>\\n\\t<td class=\\\" \\\">Distance-correlation based Gene Set Analysis for longitudinal gene expression profiles</td>\\n\\t<td class=\\\"  sorting_1\\\">551</td>\\n</tr><tr id=\\\"pkg_deco\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/deco.html\\\">deco</a></td>\\n\\t<td class=\\\" \\\">Francisco Jose Campos Laborie</td>\\n\\t<td class=\\\" \\\">Decomposing Heterogeneous Cohorts using Omic Data Profiling</td>\\n\\t<td class=\\\"  sorting_1\\\">551</td>\\n</tr><tr id=\\\"pkg_Dino\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/Dino.html\\\">Dino</a></td>\\n\\t<td class=\\\" \\\">Jared Brown</td>\\n\\t<td class=\\\" \\\">Normalization of Single-Cell mRNA Sequencing Data</td>\\n\\t<td class=\\\"  sorting_1\\\">551</td>\\n</tr><tr id=\\\"pkg_DMRforPairs\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/DMRforPairs.html\\\">DMRforPairs</a></td>\\n\\t<td class=\\\" \\\">Martin Rijlaarsdam</td>\\n\\t<td class=\\\" \\\">DMRforPairs: identifying Differentially Methylated Regions between unique samples using array based methylation profiles</td>\\n\\t<td class=\\\"  sorting_1\\\">551</td>\\n</tr><tr id=\\\"pkg_doseR\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/doseR.html\\\">doseR</a></td>\\n\\t<td class=\\\" \\\">ake.vastermark</td>\\n\\t<td class=\\\" \\\">doseR</td>\\n\\t<td class=\\\"  sorting_1\\\">551</td>\\n</tr><tr id=\\\"pkg_dpeak\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/dpeak.html\\\">dpeak</a></td>\\n\\t<td class=\\\" \\\">Dongjun Chung</td>\\n\\t<td class=\\\" \\\">dPeak (Deconvolution of Peaks in ChIP-seq Analysis)</td>\\n\\t<td class=\\\"  sorting_1\\\">551</td>\\n</tr><tr id=\\\"pkg_eiR\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/eiR.html\\\">eiR</a></td>\\n\\t<td class=\\\" \\\">Thomas Girke</td>\\n\\t<td class=\\\" \\\">Accelerated similarity searching of small molecules</td>\\n\\t<td class=\\\"  sorting_1\\\">551</td>\\n</tr><tr id=\\\"pkg_EpiCompare\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/EpiCompare.html\\\">EpiCompare</a></td>\\n\\t<td class=\\\" \\\">Sera Choi</td>\\n\\t<td class=\\\" \\\">Comparison, Benchmarking &amp; QC of Epigenomic Datasets</td>\\n\\t<td class=\\\"  sorting_1\\\">551</td>\\n</tr><tr id=\\\"pkg_FindIT2\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/FindIT2.html\\\">FindIT2</a></td>\\n\\t<td class=\\\" \\\">Guandong Shang</td>\\n\\t<td class=\\\" \\\">find influential TF and Target based on multi-omics data</td>\\n\\t<td class=\\\"  sorting_1\\\">551</td>\\n</tr><tr id=\\\"pkg_flowBeads\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/flowBeads.html\\\">flowBeads</a></td>\\n\\t<td class=\\\" \\\">Nikolas Pontikos</td>\\n\\t<td class=\\\" \\\">flowBeads: Analysis of flow bead data</td>\\n\\t<td class=\\\"  sorting_1\\\">551</td>\\n</tr><tr id=\\\"pkg_fobitools\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/fobitools.html\\\">fobitools</a></td>\\n\\t<td class=\\\" \\\">Pol Castellano-Escuder</td>\\n\\t<td class=\\\" \\\">Tools For Manipulating FOBI Ontology</td>\\n\\t<td class=\\\"  sorting_1\\\">551</td>\\n</tr><tr id=\\\"pkg_GARS\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/GARS.html\\\">GARS</a></td>\\n\\t<td class=\\\" \\\">Mattia Chiesa</td>\\n\\t<td class=\\\" \\\">GARS: Genetic Algorithm for the identification of Robust Subsets of variables in high-dimensional and challenging datasets</td>\\n\\t<td class=\\\"  sorting_1\\\">551</td>\\n</tr><tr id=\\\"pkg_gemini\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/gemini.html\\\">gemini</a></td>\\n\\t<td class=\\\" \\\">Sidharth Jain</td>\\n\\t<td class=\\\" \\\">GEMINI: Variational inference approach to infer genetic interactions from pairwise CRISPR screens</td>\\n\\t<td class=\\\"  sorting_1\\\">551</td>\\n</tr><tr id=\\\"pkg_GWAS.BAYES\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/GWAS.BAYES.html\\\">GWAS.BAYES</a></td>\\n\\t<td class=\\\" \\\">Jake Williams</td>\\n\\t<td class=\\\" \\\">GWAS for Selfing Species</td>\\n\\t<td class=\\\"  sorting_1\\\">551</td>\\n</tr><tr id=\\\"pkg_HIREewas\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/HIREewas.html\\\">HIREewas</a></td>\\n\\t<td class=\\\" \\\">Xiangyu Luo</td>\\n\\t<td class=\\\" \\\">Detection of cell-type-specific risk-CpG sites in epigenome-wide association studies</td>\\n\\t<td class=\\\"  sorting_1\\\">551</td>\\n</tr><tr id=\\\"pkg_iasva\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/iasva.html\\\">iasva</a></td>\\n\\t<td class=\\\" \\\">Donghyung Lee, Anthony Cheng</td>\\n\\t<td class=\\\" \\\">Iteratively Adjusted Surrogate Variable Analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">551</td>\\n</tr><tr id=\\\"pkg_iGC\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/iGC.html\\\">iGC</a></td>\\n\\t<td class=\\\" \\\">Liang-Bo Wang</td>\\n\\t<td class=\\\" \\\">An integrated analysis package of Gene expression and Copy number alteration</td>\\n\\t<td class=\\\"  sorting_1\\\">551</td>\\n</tr><tr id=\\\"pkg_INPower\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/INPower.html\\\">INPower</a></td>\\n\\t<td class=\\\" \\\">Bill Wheeler</td>\\n\\t<td class=\\\" \\\">An R package for computing the number of susceptibility SNPs</td>\\n\\t<td class=\\\"  sorting_1\\\">551</td>\\n</tr><tr id=\\\"pkg_ISoLDE\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ISoLDE.html\\\">ISoLDE</a></td>\\n\\t<td class=\\\" \\\">Christelle Reyns</td>\\n\\t<td class=\\\" \\\">Integrative Statistics of alleLe Dependent Expression</td>\\n\\t<td class=\\\"  sorting_1\\\">551</td>\\n</tr><tr id=\\\"pkg_lapmix\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/lapmix.html\\\">lapmix</a></td>\\n\\t<td class=\\\" \\\">Yann Ruffieux</td>\\n\\t<td class=\\\" \\\">Laplace Mixture Model in Microarray Experiments</td>\\n\\t<td class=\\\"  sorting_1\\\">551</td>\\n</tr><tr id=\\\"pkg_LineagePulse\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/LineagePulse.html\\\">LineagePulse</a></td>\\n\\t<td class=\\\" \\\">David S Fischer</td>\\n\\t<td class=\\\" \\\">Differential expression analysis and model fitting for single-cell RNA-seq data</td>\\n\\t<td class=\\\"  sorting_1\\\">551</td>\\n</tr><tr id=\\\"pkg_MAI\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/MAI.html\\\">MAI</a></td>\\n\\t<td class=\\\" \\\">Jonathan Dekermanjian</td>\\n\\t<td class=\\\" \\\">Mechanism-Aware Imputation</td>\\n\\t<td class=\\\"  sorting_1\\\">551</td>\\n</tr><tr id=\\\"pkg_mAPKL\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/mAPKL.html\\\">mAPKL</a></td>\\n\\t<td class=\\\" \\\">Argiris Sakellariou</td>\\n\\t<td class=\\\" \\\">A Hybrid Feature Selection method for gene expression data</td>\\n\\t<td class=\\\"  sorting_1\\\">551</td>\\n</tr><tr id=\\\"pkg_maskBAD\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/maskBAD.html\\\">maskBAD</a></td>\\n\\t<td class=\\\" \\\">Michael Dannemann</td>\\n\\t<td class=\\\" \\\">Masking probes with binding affinity differences</td>\\n\\t<td class=\\\"  sorting_1\\\">551</td>\\n</tr><tr id=\\\"pkg_metabCombiner\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/metabCombiner.html\\\">metabCombiner</a></td>\\n\\t<td class=\\\" \\\">Hani Habra</td>\\n\\t<td class=\\\" \\\">Method for Combining LC-MS Metabolomics Feature Measurements</td>\\n\\t<td class=\\\"  sorting_1\\\">551</td>\\n</tr><tr id=\\\"pkg_metahdep\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/metahdep.html\\\">metahdep</a></td>\\n\\t<td class=\\\" \\\">John R. Stevens</td>\\n\\t<td class=\\\" \\\">Hierarchical Dependence in Meta-Analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">551</td>\\n</tr><tr id=\\\"pkg_methylMnM\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/methylMnM.html\\\">methylMnM</a></td>\\n\\t<td class=\\\" \\\">Yan Zhou</td>\\n\\t<td class=\\\" \\\">detect different methylation level (DMR)</td>\\n\\t<td class=\\\"  sorting_1\\\">551</td>\\n</tr><tr id=\\\"pkg_MinimumDistance\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/MinimumDistance.html\\\">MinimumDistance</a></td>\\n\\t<td class=\\\" \\\">Robert Scharpf</td>\\n\\t<td class=\\\" \\\">A Package for De Novo CNV Detection in Case-Parent Trios</td>\\n\\t<td class=\\\"  sorting_1\\\">551</td>\\n</tr><tr id=\\\"pkg_miRSM\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/miRSM.html\\\">miRSM</a></td>\\n\\t<td class=\\\" \\\">Junpeng Zhang</td>\\n\\t<td class=\\\" \\\">Inferring miRNA sponge modules in heterogeneous data</td>\\n\\t<td class=\\\"  sorting_1\\\">551</td>\\n</tr><tr id=\\\"pkg_MMDiff2\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/MMDiff2.html\\\">MMDiff2</a></td>\\n\\t<td class=\\\" \\\">Gabriele Schweikert</td>\\n\\t<td class=\\\" \\\">Statistical Testing for ChIP-Seq data sets</td>\\n\\t<td class=\\\"  sorting_1\\\">551</td>\\n</tr><tr id=\\\"pkg_MOGAMUN\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/MOGAMUN.html\\\">MOGAMUN</a></td>\\n\\t<td class=\\\" \\\">Elva-Mara Novoa-del-Toro</td>\\n\\t<td class=\\\" \\\">MOGAMUN: A Multi-Objective Genetic Algorithm to Find Active Modules in Multiplex Biological Networks</td>\\n\\t<td class=\\\"  sorting_1\\\">551</td>\\n</tr><tr id=\\\"pkg_msgbsR\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/msgbsR.html\\\">msgbsR</a></td>\\n\\t<td class=\\\" \\\">Benjamin Mayne</td>\\n\\t<td class=\\\" \\\">msgbsR: methylation sensitive genotyping by sequencing (MS-GBS) R functions</td>\\n\\t<td class=\\\"  sorting_1\\\">551</td>\\n</tr><tr id=\\\"pkg_multicrispr\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/multicrispr.html\\\">multicrispr</a></td>\\n\\t<td class=\\\" \\\">Aditya Bhagwat</td>\\n\\t<td class=\\\" \\\">Multi-locus multi-purpose Crispr/Cas design</td>\\n\\t<td class=\\\"  sorting_1\\\">551</td>\\n</tr><tr id=\\\"pkg_netDx\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/netDx.html\\\">netDx</a></td>\\n\\t<td class=\\\" \\\">Shraddha Pai</td>\\n\\t<td class=\\\" \\\">Network-based patient classifier</td>\\n\\t<td class=\\\"  sorting_1\\\">551</td>\\n</tr><tr id=\\\"pkg_nnNorm\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/nnNorm.html\\\">nnNorm</a></td>\\n\\t<td class=\\\" \\\">Adi Laurentiu Tarca</td>\\n\\t<td class=\\\" \\\">Spatial and intensity based normalization of cDNA microarray data based on robust neural nets</td>\\n\\t<td class=\\\"  sorting_1\\\">551</td>\\n</tr><tr id=\\\"pkg_nucleoSim\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/nucleoSim.html\\\">nucleoSim</a></td>\\n\\t<td class=\\\" \\\">Astrid Deschnes</td>\\n\\t<td class=\\\" \\\">Generate synthetic nucleosome maps</td>\\n\\t<td class=\\\"  sorting_1\\\">551</td>\\n</tr><tr id=\\\"pkg_NxtIRFcore\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/NxtIRFcore.html\\\">NxtIRFcore</a></td>\\n\\t<td class=\\\" \\\">Alex Chit Hei Wong</td>\\n\\t<td class=\\\" \\\">Core Engine for NxtIRF: a User-Friendly Intron Retention and Alternative Splicing Analysis using the IRFinder Engine</td>\\n\\t<td class=\\\"  sorting_1\\\">551</td>\\n</tr><tr id=\\\"pkg_OLINgui\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/OLINgui.html\\\">OLINgui</a></td>\\n\\t<td class=\\\" \\\">Matthias Futschik</td>\\n\\t<td class=\\\" \\\">Graphical user interface for OLIN</td>\\n\\t<td class=\\\"  sorting_1\\\">551</td>\\n</tr><tr id=\\\"pkg_omicRexposome\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/omicRexposome.html\\\">omicRexposome</a></td>\\n\\t<td class=\\\" \\\">Xavier Escrib Montagut</td>\\n\\t<td class=\\\" \\\">Exposome and omic data associatin and integration analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">551</td>\\n</tr><tr id=\\\"pkg_onlineFDR\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/onlineFDR.html\\\">onlineFDR</a></td>\\n\\t<td class=\\\" \\\">David S. Robertson</td>\\n\\t<td class=\\\" \\\">Online error rate control</td>\\n\\t<td class=\\\"  sorting_1\\\">551</td>\\n</tr><tr id=\\\"pkg_OpenStats\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/OpenStats.html\\\">OpenStats</a></td>\\n\\t<td class=\\\" \\\">Hamed Haseli Mashhadi</td>\\n\\t<td class=\\\" \\\">A Robust and Scalable Software Package for Reproducible Analysis of High-Throughput genotype-phenotype association</td>\\n\\t<td class=\\\"  sorting_1\\\">551</td>\\n</tr><tr id=\\\"pkg_oppar\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/oppar.html\\\">oppar</a></td>\\n\\t<td class=\\\" \\\">Soroor Hediyeh zadeh</td>\\n\\t<td class=\\\" \\\">Outlier profile and pathway analysis in R</td>\\n\\t<td class=\\\"  sorting_1\\\">551</td>\\n</tr><tr id=\\\"pkg_pepStat\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/pepStat.html\\\">pepStat</a></td>\\n\\t<td class=\\\" \\\">Gregory C Imholte</td>\\n\\t<td class=\\\" \\\">Statistical analysis of peptide microarrays</td>\\n\\t<td class=\\\"  sorting_1\\\">551</td>\\n</tr><tr id=\\\"pkg_plotGrouper\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/plotGrouper.html\\\">plotGrouper</a></td>\\n\\t<td class=\\\" \\\">John D. Gagnon</td>\\n\\t<td class=\\\" \\\">Shiny app GUI wrapper for ggplot with built-in statistical analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">551</td>\\n</tr><tr id=\\\"pkg_PLPE\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/PLPE.html\\\">PLPE</a></td>\\n\\t<td class=\\\" \\\">Soo-heang Eo</td>\\n\\t<td class=\\\" \\\">Local Pooled Error Test for Differential Expression with Paired High-throughput Data</td>\\n\\t<td class=\\\"  sorting_1\\\">551</td>\\n</tr><tr id=\\\"pkg_proFIA\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/proFIA.html\\\">proFIA</a></td>\\n\\t<td class=\\\" \\\">Alexis Delabriere</td>\\n\\t<td class=\\\" \\\">Preprocessing of FIA-HRMS data</td>\\n\\t<td class=\\\"  sorting_1\\\">551</td>\\n</tr><tr id=\\\"pkg_PSEA\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/PSEA.html\\\">PSEA</a></td>\\n\\t<td class=\\\" \\\">Alexandre Kuhn</td>\\n\\t<td class=\\\" \\\">Population-Specific Expression Analysis.</td>\\n\\t<td class=\\\"  sorting_1\\\">551</td>\\n</tr><tr id=\\\"pkg_pvac\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/pvac.html\\\">pvac</a></td>\\n\\t<td class=\\\" \\\">Jun Lu, Pierre R. Bushel</td>\\n\\t<td class=\\\" \\\">PCA-based gene filtering for Affymetrix arrays</td>\\n\\t<td class=\\\"  sorting_1\\\">551</td>\\n</tr><tr id=\\\"pkg_R3CPET\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/R3CPET.html\\\">R3CPET</a></td>\\n\\t<td class=\\\" \\\">Mohamed Nadhir Djekidel</td>\\n\\t<td class=\\\" \\\">3CPET: Finding Co-factor Complexes in Chia-PET experiment using a Hierarchical Dirichlet Process</td>\\n\\t<td class=\\\"  sorting_1\\\">551</td>\\n</tr><tr id=\\\"pkg_RareVariantVis\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/RareVariantVis.html\\\">RareVariantVis</a></td>\\n\\t<td class=\\\" \\\">Tomasz Stokowy</td>\\n\\t<td class=\\\" \\\">A suite for analysis of rare genomic variants in whole genome sequencing data</td>\\n\\t<td class=\\\"  sorting_1\\\">551</td>\\n</tr><tr id=\\\"pkg_regutools\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/regutools.html\\\">regutools</a></td>\\n\\t<td class=\\\" \\\">Joselyn Chavez</td>\\n\\t<td class=\\\" \\\">regutools: an R package for data extraction from RegulonDB</td>\\n\\t<td class=\\\"  sorting_1\\\">551</td>\\n</tr><tr id=\\\"pkg_rgsepd\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/rgsepd.html\\\">rgsepd</a></td>\\n\\t<td class=\\\" \\\">Karl Stamm</td>\\n\\t<td class=\\\" \\\">Gene Set Enrichment / Projection Displays</td>\\n\\t<td class=\\\"  sorting_1\\\">551</td>\\n</tr><tr id=\\\"pkg_ribor\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ribor.html\\\">ribor</a></td>\\n\\t<td class=\\\" \\\">Michael Geng</td>\\n\\t<td class=\\\" \\\">An R Interface for Ribo Files</td>\\n\\t<td class=\\\"  sorting_1\\\">551</td>\\n</tr><tr id=\\\"pkg_RLMM\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/RLMM.html\\\">RLMM</a></td>\\n\\t<td class=\\\" \\\">Nusrat Rabbee</td>\\n\\t<td class=\\\" \\\">A Genotype Calling Algorithm for Affymetrix SNP Arrays</td>\\n\\t<td class=\\\"  sorting_1\\\">551</td>\\n</tr><tr id=\\\"pkg_rmspc\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/rmspc.html\\\">rmspc</a></td>\\n\\t<td class=\\\" \\\">Meriem Bahda</td>\\n\\t<td class=\\\" \\\">Multiple Sample Peak Calling</td>\\n\\t<td class=\\\"  sorting_1\\\">551</td>\\n</tr><tr id=\\\"pkg_RPA\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/RPA.html\\\">RPA</a></td>\\n\\t<td class=\\\" \\\">Leo Lahti</td>\\n\\t<td class=\\\" \\\">RPA: Robust Probabilistic Averaging for probe-level analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">551</td>\\n</tr><tr id=\\\"pkg_rprimer\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/rprimer.html\\\">rprimer</a></td>\\n\\t<td class=\\\" \\\">Sofia Persson</td>\\n\\t<td class=\\\" \\\">Design Degenerate Oligos from a Multiple DNA Sequence Alignment</td>\\n\\t<td class=\\\"  sorting_1\\\">551</td>\\n</tr><tr id=\\\"pkg_RTCA\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/RTCA.html\\\">RTCA</a></td>\\n\\t<td class=\\\" \\\">Jitao David Zhang</td>\\n\\t<td class=\\\" \\\">Open-source toolkit to analyse data from xCELLigence System (RTCA)</td>\\n\\t<td class=\\\"  sorting_1\\\">551</td>\\n</tr><tr id=\\\"pkg_RTopper\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/RTopper.html\\\">RTopper</a></td>\\n\\t<td class=\\\" \\\">Luigi Marchionni</td>\\n\\t<td class=\\\" \\\">This package is designed to perform Gene Set Analysis across multiple genomic platforms</td>\\n\\t<td class=\\\"  sorting_1\\\">551</td>\\n</tr><tr id=\\\"pkg_seqCAT\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/seqCAT.html\\\">seqCAT</a></td>\\n\\t<td class=\\\" \\\">Erik Fasterius</td>\\n\\t<td class=\\\" \\\">High Throughput Sequencing Cell Authentication Toolkit</td>\\n\\t<td class=\\\"  sorting_1\\\">551</td>\\n</tr><tr id=\\\"pkg_SISPA\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/SISPA.html\\\">SISPA</a></td>\\n\\t<td class=\\\" \\\">Bhakti Dwivedi</td>\\n\\t<td class=\\\" \\\">SISPA: Method for Sample Integrated Set Profile Analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">551</td>\\n</tr><tr id=\\\"pkg_SNAGEE\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/SNAGEE.html\\\">SNAGEE</a></td>\\n\\t<td class=\\\" \\\">David Venet</td>\\n\\t<td class=\\\" \\\">Signal-to-Noise applied to Gene Expression Experiments</td>\\n\\t<td class=\\\"  sorting_1\\\">551</td>\\n</tr><tr id=\\\"pkg_SpacePAC\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/SpacePAC.html\\\">SpacePAC</a></td>\\n\\t<td class=\\\" \\\">Gregory Ryslik</td>\\n\\t<td class=\\\" \\\">Identification of Mutational Clusters in 3D Protein Space via Simulation.</td>\\n\\t<td class=\\\"  sorting_1\\\">551</td>\\n</tr><tr id=\\\"pkg_SQLDataFrame\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/SQLDataFrame.html\\\">SQLDataFrame</a></td>\\n\\t<td class=\\\" \\\">Qian Liu</td>\\n\\t<td class=\\\" \\\">Representation of SQL database in DataFrame metaphor</td>\\n\\t<td class=\\\"  sorting_1\\\">551</td>\\n</tr><tr id=\\\"pkg_stepNorm\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/stepNorm.html\\\">stepNorm</a></td>\\n\\t<td class=\\\" \\\">Yuanyuan Xiao</td>\\n\\t<td class=\\\" \\\">Stepwise normalization functions for cDNA microarrays</td>\\n\\t<td class=\\\"  sorting_1\\\">551</td>\\n</tr><tr id=\\\"pkg_strandCheckR\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/strandCheckR.html\\\">strandCheckR</a></td>\\n\\t<td class=\\\" \\\">Thu-Hien To</td>\\n\\t<td class=\\\" \\\">Calculate strandness information of a bam file</td>\\n\\t<td class=\\\"  sorting_1\\\">551</td>\\n</tr><tr id=\\\"pkg_swfdr\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/swfdr.html\\\">swfdr</a></td>\\n\\t<td class=\\\" \\\">Simina M. Boca, Jeffrey T. Leek</td>\\n\\t<td class=\\\" \\\">Estimation of the science-wise false discovery rate and the false discovery rate conditional on covariates</td>\\n\\t<td class=\\\"  sorting_1\\\">551</td>\\n</tr><tr id=\\\"pkg_TnT\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/TnT.html\\\">TnT</a></td>\\n\\t<td class=\\\" \\\">Jialin Ma</td>\\n\\t<td class=\\\" \\\">Interactive Visualization for Genomic Features</td>\\n\\t<td class=\\\"  sorting_1\\\">551</td>\\n</tr><tr id=\\\"pkg_TransView\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/TransView.html\\\">TransView</a></td>\\n\\t<td class=\\\" \\\">Julius Muller</td>\\n\\t<td class=\\\" \\\">Read density map construction and accession. Visualization of ChIPSeq and RNASeq data sets</td>\\n\\t<td class=\\\"  sorting_1\\\">551</td>\\n</tr><tr id=\\\"pkg_uSORT\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/uSORT.html\\\">uSORT</a></td>\\n\\t<td class=\\\" \\\">Hao Chen</td>\\n\\t<td class=\\\" \\\">uSORT: A self-refining ordering pipeline for gene selection</td>\\n\\t<td class=\\\"  sorting_1\\\">551</td>\\n</tr><tr id=\\\"pkg_XNAString\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/XNAString.html\\\">XNAString</a></td>\\n\\t<td class=\\\" \\\">Marianna Plucinska</td>\\n\\t<td class=\\\" \\\">Efficient Manipulation of Modified Oligonucleotide Sequences</td>\\n\\t<td class=\\\"  sorting_1\\\">551</td>\\n</tr><tr id=\\\"pkg_awst\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/awst.html\\\">awst</a></td>\\n\\t<td class=\\\" \\\">Davide Risso</td>\\n\\t<td class=\\\" \\\">Asymmetric Within-Sample Transformation</td>\\n\\t<td class=\\\"  sorting_1\\\">634</td>\\n</tr><tr id=\\\"pkg_BBCAnalyzer\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/BBCAnalyzer.html\\\">BBCAnalyzer</a></td>\\n\\t<td class=\\\" \\\">Sarah Sandmann</td>\\n\\t<td class=\\\" \\\">BBCAnalyzer: an R/Bioconductor package for visualizing base counts</td>\\n\\t<td class=\\\"  sorting_1\\\">634</td>\\n</tr><tr id=\\\"pkg_BindingSiteFinder\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/BindingSiteFinder.html\\\">BindingSiteFinder</a></td>\\n\\t<td class=\\\" \\\">Mirko Brggemann</td>\\n\\t<td class=\\\" \\\">Binding site defintion based on iCLIP data</td>\\n\\t<td class=\\\"  sorting_1\\\">634</td>\\n</tr><tr id=\\\"pkg_BiocDockerManager\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/BiocDockerManager.html\\\">BiocDockerManager</a></td>\\n\\t<td class=\\\" \\\">Bioconductor Package Maintainer</td>\\n\\t<td class=\\\" \\\">Access Bioconductor docker images</td>\\n\\t<td class=\\\"  sorting_1\\\">634</td>\\n</tr><tr id=\\\"pkg_biomvRCNS\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/biomvRCNS.html\\\">biomvRCNS</a></td>\\n\\t<td class=\\\" \\\">Yang Du</td>\\n\\t<td class=\\\" \\\">Copy Number study and Segmentation for multivariate biological data</td>\\n\\t<td class=\\\"  sorting_1\\\">634</td>\\n</tr><tr id=\\\"pkg_CancerInSilico\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/CancerInSilico.html\\\">CancerInSilico</a></td>\\n\\t<td class=\\\" \\\">Thomas D. Sherman, Elana J. Fertig</td>\\n\\t<td class=\\\" \\\">An R interface for computational modeling of tumor progression</td>\\n\\t<td class=\\\"  sorting_1\\\">634</td>\\n</tr><tr id=\\\"pkg_cellscape\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/cellscape.html\\\">cellscape</a></td>\\n\\t<td class=\\\" \\\">Maia Smith</td>\\n\\t<td class=\\\" \\\">Explores single cell copy number profiles in the context of a single cell tree</td>\\n\\t<td class=\\\"  sorting_1\\\">634</td>\\n</tr><tr id=\\\"pkg_ChIC\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ChIC.html\\\">ChIC</a></td>\\n\\t<td class=\\\" \\\">Carmen Maria Livi</td>\\n\\t<td class=\\\" \\\">Quality Control Pipeline for ChIP-Seq Data</td>\\n\\t<td class=\\\"  sorting_1\\\">634</td>\\n</tr><tr id=\\\"pkg_cnvGSA\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/cnvGSA.html\\\">cnvGSA</a></td>\\n\\t<td class=\\\" \\\">Joseph Lugo</td>\\n\\t<td class=\\\" \\\">Gene Set Analysis of (Rare) Copy Number Variants</td>\\n\\t<td class=\\\"  sorting_1\\\">634</td>\\n</tr><tr id=\\\"pkg_CONSTANd\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/CONSTANd.html\\\">CONSTANd</a></td>\\n\\t<td class=\\\" \\\">Dirk Valkenborg</td>\\n\\t<td class=\\\" \\\">Data normalization by matrix raking</td>\\n\\t<td class=\\\"  sorting_1\\\">634</td>\\n</tr><tr id=\\\"pkg_Cormotif\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/Cormotif.html\\\">Cormotif</a></td>\\n\\t<td class=\\\" \\\">Yingying Wei</td>\\n\\t<td class=\\\" \\\">Correlation Motif Fit</td>\\n\\t<td class=\\\"  sorting_1\\\">634</td>\\n</tr><tr id=\\\"pkg_covRNA\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/covRNA.html\\\">covRNA</a></td>\\n\\t<td class=\\\" \\\">Lara Urban</td>\\n\\t<td class=\\\" \\\">Multivariate Analysis of Transcriptomic Data</td>\\n\\t<td class=\\\"  sorting_1\\\">634</td>\\n</tr><tr id=\\\"pkg_crisprseekplus\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/crisprseekplus.html\\\">crisprseekplus</a></td>\\n\\t<td class=\\\" \\\">Alper Kucukural</td>\\n\\t<td class=\\\" \\\">crisprseekplus</td>\\n\\t<td class=\\\"  sorting_1\\\">634</td>\\n</tr><tr id=\\\"pkg_CTDquerier\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/CTDquerier.html\\\">CTDquerier</a></td>\\n\\t<td class=\\\" \\\">Xavier Escrib-Montagut</td>\\n\\t<td class=\\\" \\\">Package for CTDbase data query, visualization and downstream analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">634</td>\\n</tr><tr id=\\\"pkg_customCMPdb\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/customCMPdb.html\\\">customCMPdb</a></td>\\n\\t<td class=\\\" \\\">Yuzhu Duan</td>\\n\\t<td class=\\\" \\\">Customize and Query Compound Annotation Database</td>\\n\\t<td class=\\\"  sorting_1\\\">634</td>\\n</tr><tr id=\\\"pkg_DAMEfinder\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/DAMEfinder.html\\\">DAMEfinder</a></td>\\n\\t<td class=\\\" \\\">Stephany Orjuela</td>\\n\\t<td class=\\\" \\\">Finds DAMEs - Differential Allelicly MEthylated regions</td>\\n\\t<td class=\\\"  sorting_1\\\">634</td>\\n</tr><tr id=\\\"pkg_DFP\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/DFP.html\\\">DFP</a></td>\\n\\t<td class=\\\" \\\">Rodrigo Alvarez-Glez</td>\\n\\t<td class=\\\" \\\">Gene Selection</td>\\n\\t<td class=\\\"  sorting_1\\\">634</td>\\n</tr><tr id=\\\"pkg_DrugVsDisease\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/DrugVsDisease.html\\\">DrugVsDisease</a></td>\\n\\t<td class=\\\" \\\">j. Saez-Rodriguez</td>\\n\\t<td class=\\\" \\\">Comparison of disease and drug profiles using Gene set Enrichment Analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">634</td>\\n</tr><tr id=\\\"pkg_eds\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/eds.html\\\">eds</a></td>\\n\\t<td class=\\\" \\\">Avi Srivastava</td>\\n\\t<td class=\\\" \\\">eds: Low-level reader for Alevin EDS format</td>\\n\\t<td class=\\\"  sorting_1\\\">634</td>\\n</tr><tr id=\\\"pkg_eegc\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/eegc.html\\\">eegc</a></td>\\n\\t<td class=\\\" \\\">Xiaoyuan Zhou</td>\\n\\t<td class=\\\" \\\">Engineering Evaluation by Gene Categorization (eegc)</td>\\n\\t<td class=\\\"  sorting_1\\\">634</td>\\n</tr><tr id=\\\"pkg_epigraHMM\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/epigraHMM.html\\\">epigraHMM</a></td>\\n\\t<td class=\\\" \\\">Pedro Baldoni</td>\\n\\t<td class=\\\" \\\">Epigenomic R-based analysis with hidden Markov models</td>\\n\\t<td class=\\\"  sorting_1\\\">634</td>\\n</tr><tr id=\\\"pkg_epivizrChart\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/epivizrChart.html\\\">epivizrChart</a></td>\\n\\t<td class=\\\" \\\">Hector Corrada Bravo</td>\\n\\t<td class=\\\" \\\">R interface to epiviz web components</td>\\n\\t<td class=\\\"  sorting_1\\\">634</td>\\n</tr><tr id=\\\"pkg_fastLiquidAssociation\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/fastLiquidAssociation.html\\\">fastLiquidAssociation</a></td>\\n\\t<td class=\\\" \\\">Tina Gunderson</td>\\n\\t<td class=\\\" \\\">functions for genome-wide application of Liquid Association</td>\\n\\t<td class=\\\"  sorting_1\\\">634</td>\\n</tr><tr id=\\\"pkg_fCCAC\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/fCCAC.html\\\">fCCAC</a></td>\\n\\t<td class=\\\" \\\">Pedro Madrigal</td>\\n\\t<td class=\\\" \\\">functional Canonical Correlation Analysis to evaluate Covariance between nucleic acid sequencing datasets</td>\\n\\t<td class=\\\"  sorting_1\\\">634</td>\\n</tr><tr id=\\\"pkg_fgga\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/fgga.html\\\">fgga</a></td>\\n\\t<td class=\\\" \\\">Flavio Spetale</td>\\n\\t<td class=\\\" \\\">Hierarchical ensemble method based on factor graph</td>\\n\\t<td class=\\\"  sorting_1\\\">634</td>\\n</tr><tr id=\\\"pkg_FISHalyseR\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/FISHalyseR.html\\\">FISHalyseR</a></td>\\n\\t<td class=\\\" \\\">Karesh Arunakirinathan, Andreas Heindl</td>\\n\\t<td class=\\\" \\\">FISHalyseR a package for automated FISH quantification</td>\\n\\t<td class=\\\"  sorting_1\\\">634</td>\\n</tr><tr id=\\\"pkg_flagme\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/flagme.html\\\">flagme</a></td>\\n\\t<td class=\\\" \\\">Mark Robinson, Riccardo Romoli</td>\\n\\t<td class=\\\" \\\">Analysis of Metabolomics GC/MS Data</td>\\n\\t<td class=\\\"  sorting_1\\\">634</td>\\n</tr><tr id=\\\"pkg_flowcatchR\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/flowcatchR.html\\\">flowcatchR</a></td>\\n\\t<td class=\\\" \\\">Federico Marini</td>\\n\\t<td class=\\\" \\\">Tools to analyze in vivo microscopy imaging data focused on tracking flowing blood cells</td>\\n\\t<td class=\\\"  sorting_1\\\">634</td>\\n</tr><tr id=\\\"pkg_flowMap\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/flowMap.html\\\">flowMap</a></td>\\n\\t<td class=\\\" \\\">Chiaowen Joyce Hsiao</td>\\n\\t<td class=\\\" \\\">Mapping cell populations in flow cytometry data for cross-sample comparisons using the Friedman-Rafsky Test</td>\\n\\t<td class=\\\"  sorting_1\\\">634</td>\\n</tr><tr id=\\\"pkg_GenomicTuples\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/GenomicTuples.html\\\">GenomicTuples</a></td>\\n\\t<td class=\\\" \\\">Peter Hickey</td>\\n\\t<td class=\\\" \\\">Representation and Manipulation of Genomic Tuples</td>\\n\\t<td class=\\\"  sorting_1\\\">634</td>\\n</tr><tr id=\\\"pkg_genotypeeval\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/genotypeeval.html\\\">genotypeeval</a></td>\\n\\t<td class=\\\" \\\">Jennifer Tom</td>\\n\\t<td class=\\\" \\\">QA/QC of a gVCF or VCF file</td>\\n\\t<td class=\\\"  sorting_1\\\">634</td>\\n</tr><tr id=\\\"pkg_GEOsubmission\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/GEOsubmission.html\\\">GEOsubmission</a></td>\\n\\t<td class=\\\" \\\">Alexandre Kuhn</td>\\n\\t<td class=\\\" \\\">Prepares microarray data for submission to GEO</td>\\n\\t<td class=\\\"  sorting_1\\\">634</td>\\n</tr><tr id=\\\"pkg_gep2pep\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/gep2pep.html\\\">gep2pep</a></td>\\n\\t<td class=\\\" \\\">Francesco Napolitano</td>\\n\\t<td class=\\\" \\\">Creation and Analysis of Pathway Expression Profiles (PEPs)</td>\\n\\t<td class=\\\"  sorting_1\\\">634</td>\\n</tr><tr id=\\\"pkg_GNET2\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/GNET2.html\\\">GNET2</a></td>\\n\\t<td class=\\\" \\\">Chen Chen</td>\\n\\t<td class=\\\" \\\">Constructing gene regulatory networks from expression data through functional module inference</td>\\n\\t<td class=\\\"  sorting_1\\\">634</td>\\n</tr><tr id=\\\"pkg_GraphAT\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/GraphAT.html\\\">GraphAT</a></td>\\n\\t<td class=\\\" \\\">Thomas LaFramboise</td>\\n\\t<td class=\\\" \\\">Graph Theoretic Association Tests</td>\\n\\t<td class=\\\"  sorting_1\\\">634</td>\\n</tr><tr id=\\\"pkg_GSReg\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/GSReg.html\\\">GSReg</a></td>\\n\\t<td class=\\\" \\\">Bahman Afsari, Elana J. Fertig</td>\\n\\t<td class=\\\" \\\">Gene Set Regulation (GS-Reg)</td>\\n\\t<td class=\\\"  sorting_1\\\">634</td>\\n</tr><tr id=\\\"pkg_Harshlight\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/Harshlight.html\\\">Harshlight</a></td>\\n\\t<td class=\\\" \\\">Maurizio Pellegrino</td>\\n\\t<td class=\\\" \\\">A \\\"corrective make-up\\\" program for microarray chips</td>\\n\\t<td class=\\\"  sorting_1\\\">634</td>\\n</tr><tr id=\\\"pkg_HELP\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/HELP.html\\\">HELP</a></td>\\n\\t<td class=\\\" \\\">Reid F. Thompson</td>\\n\\t<td class=\\\" \\\">Tools for HELP data analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">634</td>\\n</tr><tr id=\\\"pkg_HumanTranscriptomeCompendium\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/HumanTranscriptomeCompendium.html\\\">HumanTranscriptomeCompendium</a></td>\\n\\t<td class=\\\" \\\">VJ Carey</td>\\n\\t<td class=\\\" \\\">Tools to work with a Compendium of 181000 human transcriptome sequencing studies</td>\\n\\t<td class=\\\"  sorting_1\\\">634</td>\\n</tr><tr id=\\\"pkg_ibh\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ibh.html\\\">ibh</a></td>\\n\\t<td class=\\\" \\\">Kircicegi Korkmaz</td>\\n\\t<td class=\\\" \\\">Interaction Based Homogeneity for Evaluating Gene Lists</td>\\n\\t<td class=\\\"  sorting_1\\\">634</td>\\n</tr><tr id=\\\"pkg_ILoReg\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ILoReg.html\\\">ILoReg</a></td>\\n\\t<td class=\\\" \\\">Johannes Smolander</td>\\n\\t<td class=\\\" \\\">ILoReg: a tool for high-resolution cell population identification from scRNA-Seq data</td>\\n\\t<td class=\\\"  sorting_1\\\">634</td>\\n</tr><tr id=\\\"pkg_IRISFGM\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/IRISFGM.html\\\">IRISFGM</a></td>\\n\\t<td class=\\\" \\\">Yuzhou Chang</td>\\n\\t<td class=\\\" \\\">Comprehensive Analysis of Gene Interactivity Networks Based on Single-Cell RNA-Seq</td>\\n\\t<td class=\\\"  sorting_1\\\">634</td>\\n</tr><tr id=\\\"pkg_iSeq\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/iSeq.html\\\">iSeq</a></td>\\n\\t<td class=\\\" \\\">Qianxing Mo</td>\\n\\t<td class=\\\" \\\">Bayesian Hierarchical Modeling of ChIP-seq Data Through Hidden Ising Models</td>\\n\\t<td class=\\\"  sorting_1\\\">634</td>\\n</tr><tr id=\\\"pkg_KCsmart\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/KCsmart.html\\\">KCsmart</a></td>\\n\\t<td class=\\\" \\\">Jorma de Ronde</td>\\n\\t<td class=\\\" \\\">Multi sample aCGH analysis package using kernel convolution</td>\\n\\t<td class=\\\"  sorting_1\\\">634</td>\\n</tr><tr id=\\\"pkg_maCorrPlot\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/maCorrPlot.html\\\">maCorrPlot</a></td>\\n\\t<td class=\\\" \\\">Alexander Ploner</td>\\n\\t<td class=\\\" \\\">Visualize artificial correlation in microarray data</td>\\n\\t<td class=\\\"  sorting_1\\\">634</td>\\n</tr><tr id=\\\"pkg_maPredictDSC\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/maPredictDSC.html\\\">maPredictDSC</a></td>\\n\\t<td class=\\\" \\\">Adi Laurentiu Tarca</td>\\n\\t<td class=\\\" \\\">Phenotype prediction using microarray data: approach of the best overall team in the IMPROVER Diagnostic Signature Challenge</td>\\n\\t<td class=\\\"  sorting_1\\\">634</td>\\n</tr><tr id=\\\"pkg_MBQN\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/MBQN.html\\\">MBQN</a></td>\\n\\t<td class=\\\" \\\">Eva Brombacher</td>\\n\\t<td class=\\\" \\\">Mean/Median-balanced quantile normalization</td>\\n\\t<td class=\\\"  sorting_1\\\">634</td>\\n</tr><tr id=\\\"pkg_MCbiclust\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/MCbiclust.html\\\">MCbiclust</a></td>\\n\\t<td class=\\\" \\\">Robert Bentham</td>\\n\\t<td class=\\\" \\\">Massive correlating biclusters for gene expression data and associated methods</td>\\n\\t<td class=\\\"  sorting_1\\\">634</td>\\n</tr><tr id=\\\"pkg_MEDME\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/MEDME.html\\\">MEDME</a></td>\\n\\t<td class=\\\" \\\">Mattia Pelizzola</td>\\n\\t<td class=\\\" \\\">Modelling Experimental Data from MeDIP Enrichment</td>\\n\\t<td class=\\\"  sorting_1\\\">634</td>\\n</tr><tr id=\\\"pkg_metabomxtr\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/metabomxtr.html\\\">metabomxtr</a></td>\\n\\t<td class=\\\" \\\">Michael Nodzenski</td>\\n\\t<td class=\\\" \\\">A package to run mixture models for truncated metabolomics data with normal or lognormal distributions</td>\\n\\t<td class=\\\"  sorting_1\\\">634</td>\\n</tr><tr id=\\\"pkg_MetCirc\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/MetCirc.html\\\">MetCirc</a></td>\\n\\t<td class=\\\" \\\">Thomas Naake</td>\\n\\t<td class=\\\" \\\">Navigating mass spectral similarity in high-resolution MS/MS metabolomics data</td>\\n\\t<td class=\\\"  sorting_1\\\">634</td>\\n</tr><tr id=\\\"pkg_MiPP\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/MiPP.html\\\">MiPP</a></td>\\n\\t<td class=\\\" \\\">Sukwoo Kim</td>\\n\\t<td class=\\\" \\\">Misclassification Penalized Posterior Classification</td>\\n\\t<td class=\\\"  sorting_1\\\">634</td>\\n</tr><tr id=\\\"pkg_MiRaGE\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/MiRaGE.html\\\">MiRaGE</a></td>\\n\\t<td class=\\\" \\\">Y-h. Taguchi</td>\\n\\t<td class=\\\" \\\">MiRNA Ranking by Gene Expression</td>\\n\\t<td class=\\\"  sorting_1\\\">634</td>\\n</tr><tr id=\\\"pkg_mirTarRnaSeq\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/mirTarRnaSeq.html\\\">mirTarRnaSeq</a></td>\\n\\t<td class=\\\" \\\">Mercedeh Movassagh</td>\\n\\t<td class=\\\" \\\">mirTarRnaSeq</td>\\n\\t<td class=\\\"  sorting_1\\\">634</td>\\n</tr><tr id=\\\"pkg_nanotatoR\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/nanotatoR.html\\\">nanotatoR</a></td>\\n\\t<td class=\\\" \\\">Surajit Bhattacharya</td>\\n\\t<td class=\\\" \\\">Next generation structural variant annotation and classification</td>\\n\\t<td class=\\\"  sorting_1\\\">634</td>\\n</tr><tr id=\\\"pkg_ncRNAtools\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ncRNAtools.html\\\">ncRNAtools</a></td>\\n\\t<td class=\\\" \\\">Lara Selles Vidal</td>\\n\\t<td class=\\\" \\\">An R toolkit for non-coding RNA</td>\\n\\t<td class=\\\"  sorting_1\\\">634</td>\\n</tr><tr id=\\\"pkg_nnSVG\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/nnSVG.html\\\">nnSVG</a></td>\\n\\t<td class=\\\" \\\">Lukas M. Weber</td>\\n\\t<td class=\\\" \\\">Scalable identification of spatially variable genes in spatially-resolved transcriptomics data</td>\\n\\t<td class=\\\"  sorting_1\\\">634</td>\\n</tr><tr id=\\\"pkg_omicsPrint\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/omicsPrint.html\\\">omicsPrint</a></td>\\n\\t<td class=\\\" \\\">Davy Cats</td>\\n\\t<td class=\\\" \\\">Cross omic genetic fingerprinting</td>\\n\\t<td class=\\\"  sorting_1\\\">634</td>\\n</tr><tr id=\\\"pkg_pipeComp\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/pipeComp.html\\\">pipeComp</a></td>\\n\\t<td class=\\\" \\\">Pierre-Luc Germain</td>\\n\\t<td class=\\\" \\\">pipeComp pipeline benchmarking framework</td>\\n\\t<td class=\\\"  sorting_1\\\">634</td>\\n</tr><tr id=\\\"pkg_plethy\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/plethy.html\\\">plethy</a></td>\\n\\t<td class=\\\" \\\">Daniel Bottomly</td>\\n\\t<td class=\\\" \\\">R framework for exploration and analysis of respirometry data</td>\\n\\t<td class=\\\"  sorting_1\\\">634</td>\\n</tr><tr id=\\\"pkg_ptairMS\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ptairMS.html\\\">ptairMS</a></td>\\n\\t<td class=\\\" \\\">camille Roquencourt</td>\\n\\t<td class=\\\" \\\">Pre-processing PTR-TOF-MS Data</td>\\n\\t<td class=\\\"  sorting_1\\\">634</td>\\n</tr><tr id=\\\"pkg_QSutils\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/QSutils.html\\\">QSutils</a></td>\\n\\t<td class=\\\" \\\">Mercedes Guerrero-Murillo</td>\\n\\t<td class=\\\" \\\">Quasispecies Diversity</td>\\n\\t<td class=\\\"  sorting_1\\\">634</td>\\n</tr><tr id=\\\"pkg_rama\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/rama.html\\\">rama</a></td>\\n\\t<td class=\\\" \\\">Raphael Gottardo</td>\\n\\t<td class=\\\" \\\">Robust Analysis of MicroArrays</td>\\n\\t<td class=\\\"  sorting_1\\\">634</td>\\n</tr><tr id=\\\"pkg_recoup\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/recoup.html\\\">recoup</a></td>\\n\\t<td class=\\\" \\\">Panagiotis Moulos</td>\\n\\t<td class=\\\" \\\">An R package for the creation of complex genomic profile plots</td>\\n\\t<td class=\\\"  sorting_1\\\">634</td>\\n</tr><tr id=\\\"pkg_RiboDiPA\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/RiboDiPA.html\\\">RiboDiPA</a></td>\\n\\t<td class=\\\" \\\">Ji-Ping Wang</td>\\n\\t<td class=\\\" \\\">Differential pattern analysis for Ribo-seq data</td>\\n\\t<td class=\\\"  sorting_1\\\">634</td>\\n</tr><tr id=\\\"pkg_Rmagpie\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/Rmagpie.html\\\">Rmagpie</a></td>\\n\\t<td class=\\\" \\\">Camille Maumet</td>\\n\\t<td class=\\\" \\\">MicroArray Gene-expression-based Program In Error rate estimation</td>\\n\\t<td class=\\\"  sorting_1\\\">634</td>\\n</tr><tr id=\\\"pkg_SANTA\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/SANTA.html\\\">SANTA</a></td>\\n\\t<td class=\\\" \\\">Alex Cornish</td>\\n\\t<td class=\\\" \\\">Spatial Analysis of Network Associations</td>\\n\\t<td class=\\\"  sorting_1\\\">634</td>\\n</tr><tr id=\\\"pkg_scGPS\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/scGPS.html\\\">scGPS</a></td>\\n\\t<td class=\\\" \\\">Quan Nguyen</td>\\n\\t<td class=\\\" \\\">A complete analysis of single cell subpopulations, from identifying subpopulations to analysing their relationship (scGPS = single cell Global Predictions of Subpopulation)</td>\\n\\t<td class=\\\"  sorting_1\\\">634</td>\\n</tr><tr id=\\\"pkg_scTGIF\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/scTGIF.html\\\">scTGIF</a></td>\\n\\t<td class=\\\" \\\">Koki Tsuyuzaki</td>\\n\\t<td class=\\\" \\\">Cell type annotation for unannotated single-cell RNA-Seq data</td>\\n\\t<td class=\\\"  sorting_1\\\">634</td>\\n</tr><tr id=\\\"pkg_ssrch\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ssrch.html\\\">ssrch</a></td>\\n\\t<td class=\\\" \\\">VJ Carey</td>\\n\\t<td class=\\\" \\\">a simple search engine</td>\\n\\t<td class=\\\"  sorting_1\\\">634</td>\\n</tr><tr id=\\\"pkg_SummarizedBenchmark\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/SummarizedBenchmark.html\\\">SummarizedBenchmark</a></td>\\n\\t<td class=\\\" \\\">Patrick Kimes</td>\\n\\t<td class=\\\" \\\">Classes and methods for performing benchmark comparisons</td>\\n\\t<td class=\\\"  sorting_1\\\">634</td>\\n</tr><tr id=\\\"pkg_tigre\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/tigre.html\\\">tigre</a></td>\\n\\t<td class=\\\" \\\">Antti Honkela</td>\\n\\t<td class=\\\" \\\">Transcription factor Inference through Gaussian process Reconstruction of Expression</td>\\n\\t<td class=\\\"  sorting_1\\\">634</td>\\n</tr><tr id=\\\"pkg_triplex\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/triplex.html\\\">triplex</a></td>\\n\\t<td class=\\\" \\\">Jiri Hon</td>\\n\\t<td class=\\\" \\\">Search and visualize intramolecular triplex-forming sequences in DNA</td>\\n\\t<td class=\\\"  sorting_1\\\">634</td>\\n</tr><tr id=\\\"pkg_twoddpcr\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/twoddpcr.html\\\">twoddpcr</a></td>\\n\\t<td class=\\\" \\\">Anthony Chiu</td>\\n\\t<td class=\\\" \\\">Classify 2-d Droplet Digital PCR (ddPCR) data and quantify the number of starting molecules</td>\\n\\t<td class=\\\"  sorting_1\\\">634</td>\\n</tr><tr id=\\\"pkg_UNDO\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/UNDO.html\\\">UNDO</a></td>\\n\\t<td class=\\\" \\\">Niya Wang</td>\\n\\t<td class=\\\" \\\">Unsupervised Deconvolution of Tumor-Stromal Mixed Expressions</td>\\n\\t<td class=\\\"  sorting_1\\\">634</td>\\n</tr><tr id=\\\"pkg_veloviz\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/veloviz.html\\\">veloviz</a></td>\\n\\t<td class=\\\" \\\">Lyla Atta</td>\\n\\t<td class=\\\" \\\">VeloViz: RNA-velocity informed 2D embeddings for visualizing cell state trajectories</td>\\n\\t<td class=\\\"  sorting_1\\\">634</td>\\n</tr><tr id=\\\"pkg_Xeva\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/Xeva.html\\\">Xeva</a></td>\\n\\t<td class=\\\" \\\">Benjamin Haibe-Kains</td>\\n\\t<td class=\\\" \\\">Analysis of patient-derived xenograft (PDX) data</td>\\n\\t<td class=\\\"  sorting_1\\\">634</td>\\n</tr><tr id=\\\"pkg_XINA\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/XINA.html\\\">XINA</a></td>\\n\\t<td class=\\\" \\\">Lang Ho Lee and Sasha A. Singh</td>\\n\\t<td class=\\\" \\\">Multiplexes Isobaric Mass Tagged-based Kinetics Data for Network Analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">634</td>\\n</tr><tr id=\\\"pkg_annmap\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/annmap.html\\\">annmap</a></td>\\n\\t<td class=\\\" \\\">Chris Wirth</td>\\n\\t<td class=\\\" \\\">Genome annotation and visualisation package pertaining to Affymetrix arrays and NGS analysis.</td>\\n\\t<td class=\\\"  sorting_1\\\">712</td>\\n</tr><tr id=\\\"pkg_AnVILPublish\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/AnVILPublish.html\\\">AnVILPublish</a></td>\\n\\t<td class=\\\" \\\">Martin Morgan</td>\\n\\t<td class=\\\" \\\">Publish Packages and Other Resources to AnVIL Workspaces</td>\\n\\t<td class=\\\"  sorting_1\\\">712</td>\\n</tr><tr id=\\\"pkg_ASAFE\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ASAFE.html\\\">ASAFE</a></td>\\n\\t<td class=\\\" \\\">Qian Zhang</td>\\n\\t<td class=\\\" \\\">Ancestry Specific Allele Frequency Estimation</td>\\n\\t<td class=\\\"  sorting_1\\\">712</td>\\n</tr><tr id=\\\"pkg_BGmix\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/BGmix.html\\\">BGmix</a></td>\\n\\t<td class=\\\" \\\">Alex Lewin</td>\\n\\t<td class=\\\" \\\">Bayesian models for differential gene expression</td>\\n\\t<td class=\\\"  sorting_1\\\">712</td>\\n</tr><tr id=\\\"pkg_BioMM\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/BioMM.html\\\">BioMM</a></td>\\n\\t<td class=\\\" \\\">Junfang Chen</td>\\n\\t<td class=\\\" \\\">BioMM: Biological-informed Multi-stage Machine learning framework for phenotype prediction using omics data</td>\\n\\t<td class=\\\"  sorting_1\\\">712</td>\\n</tr><tr id=\\\"pkg_brainflowprobes\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/brainflowprobes.html\\\">brainflowprobes</a></td>\\n\\t<td class=\\\" \\\">Amanda Price</td>\\n\\t<td class=\\\" \\\">Plots and annotation for choosing BrainFlow target probe sequence</td>\\n\\t<td class=\\\"  sorting_1\\\">712</td>\\n</tr><tr id=\\\"pkg_CAFE\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/CAFE.html\\\">CAFE</a></td>\\n\\t<td class=\\\" \\\">Sander Bollen</td>\\n\\t<td class=\\\" \\\">Chromosmal Aberrations Finder in Expression data</td>\\n\\t<td class=\\\"  sorting_1\\\">712</td>\\n</tr><tr id=\\\"pkg_cbaf\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/cbaf.html\\\">cbaf</a></td>\\n\\t<td class=\\\" \\\">Arman Shahrisa</td>\\n\\t<td class=\\\" \\\">Automated functions for comparing various omic data from cbioportal.org</td>\\n\\t<td class=\\\"  sorting_1\\\">712</td>\\n</tr><tr id=\\\"pkg_cellTree\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/cellTree.html\\\">cellTree</a></td>\\n\\t<td class=\\\" \\\">David duVerle</td>\\n\\t<td class=\\\" \\\">Inference and visualisation of Single-Cell RNA-seq data as a hierarchical tree structure</td>\\n\\t<td class=\\\"  sorting_1\\\">712</td>\\n</tr><tr id=\\\"pkg_ChIPanalyser\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ChIPanalyser.html\\\">ChIPanalyser</a></td>\\n\\t<td class=\\\" \\\">Patrick C.N. Martin</td>\\n\\t<td class=\\\" \\\">ChIPanalyser: Predicting Transcription Factor Binding Sites</td>\\n\\t<td class=\\\"  sorting_1\\\">712</td>\\n</tr><tr id=\\\"pkg_chromswitch\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/chromswitch.html\\\">chromswitch</a></td>\\n\\t<td class=\\\" \\\">Selin Jessa</td>\\n\\t<td class=\\\" \\\">An R package to detect chromatin state switches from epigenomic data</td>\\n\\t<td class=\\\"  sorting_1\\\">712</td>\\n</tr><tr id=\\\"pkg_clippda\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/clippda.html\\\">clippda</a></td>\\n\\t<td class=\\\" \\\">Stephen Nyangoma</td>\\n\\t<td class=\\\" \\\">A package for the clinical proteomic profiling data analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">712</td>\\n</tr><tr id=\\\"pkg_contiBAIT\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/contiBAIT.html\\\">contiBAIT</a></td>\\n\\t<td class=\\\" \\\">Kieran O'Neill</td>\\n\\t<td class=\\\" \\\">Improves Early Build Genome Assemblies using Strand-Seq Data</td>\\n\\t<td class=\\\"  sorting_1\\\">712</td>\\n</tr><tr id=\\\"pkg_DeMAND\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/DeMAND.html\\\">DeMAND</a></td>\\n\\t<td class=\\\" \\\">Jung Hoon Woo, Mariano Alvarez</td>\\n\\t<td class=\\\" \\\">DeMAND</td>\\n\\t<td class=\\\"  sorting_1\\\">712</td>\\n</tr><tr id=\\\"pkg_DEScan2\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/DEScan2.html\\\">DEScan2</a></td>\\n\\t<td class=\\\" \\\">Dario Righelli</td>\\n\\t<td class=\\\" \\\">Differential Enrichment Scan 2</td>\\n\\t<td class=\\\"  sorting_1\\\">712</td>\\n</tr><tr id=\\\"pkg_ecolitk\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ecolitk.html\\\">ecolitk</a></td>\\n\\t<td class=\\\" \\\">Laurent Gautier</td>\\n\\t<td class=\\\" \\\">Meta-data and tools for E. coli</td>\\n\\t<td class=\\\"  sorting_1\\\">712</td>\\n</tr><tr id=\\\"pkg_epigenomix\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/epigenomix.html\\\">epigenomix</a></td>\\n\\t<td class=\\\" \\\">Hans-Ulrich Klein</td>\\n\\t<td class=\\\" \\\">Epigenetic and gene transcription data normalization and integration with mixture models</td>\\n\\t<td class=\\\"  sorting_1\\\">712</td>\\n</tr><tr id=\\\"pkg_flowMatch\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/flowMatch.html\\\">flowMatch</a></td>\\n\\t<td class=\\\" \\\">Ariful Azad</td>\\n\\t<td class=\\\" \\\">Matching and meta-clustering in flow cytometry</td>\\n\\t<td class=\\\"  sorting_1\\\">712</td>\\n</tr><tr id=\\\"pkg_genArise\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/genArise.html\\\">genArise</a></td>\\n\\t<td class=\\\" \\\">IFC Development Team</td>\\n\\t<td class=\\\" \\\">Microarray Analysis tool</td>\\n\\t<td class=\\\"  sorting_1\\\">712</td>\\n</tr><tr id=\\\"pkg_GeneGeneInteR\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/GeneGeneInteR.html\\\">GeneGeneInteR</a></td>\\n\\t<td class=\\\" \\\">Mathieu Emily</td>\\n\\t<td class=\\\" \\\">Tools for Testing Gene-Gene Interaction at the Gene Level</td>\\n\\t<td class=\\\"  sorting_1\\\">712</td>\\n</tr><tr id=\\\"pkg_GeneRegionScan\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/GeneRegionScan.html\\\">GeneRegionScan</a></td>\\n\\t<td class=\\\" \\\">Lasse Folkersen</td>\\n\\t<td class=\\\" \\\">GeneRegionScan</td>\\n\\t<td class=\\\"  sorting_1\\\">712</td>\\n</tr><tr id=\\\"pkg_geneXtendeR\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/geneXtendeR.html\\\">geneXtendeR</a></td>\\n\\t<td class=\\\" \\\">Bohdan Khomtchouk</td>\\n\\t<td class=\\\" \\\">Optimized Functional Annotation Of ChIP-seq Data</td>\\n\\t<td class=\\\"  sorting_1\\\">712</td>\\n</tr><tr id=\\\"pkg_GEOfastq\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/GEOfastq.html\\\">GEOfastq</a></td>\\n\\t<td class=\\\" \\\">Alex Pickering</td>\\n\\t<td class=\\\" \\\">Downloads ENA Fastqs With GEO Accessions</td>\\n\\t<td class=\\\"  sorting_1\\\">712</td>\\n</tr><tr id=\\\"pkg_ggmanh\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ggmanh.html\\\">ggmanh</a></td>\\n\\t<td class=\\\" \\\">John Lee</td>\\n\\t<td class=\\\" \\\">Visualization Tool for GWAS Result</td>\\n\\t<td class=\\\"  sorting_1\\\">712</td>\\n</tr><tr id=\\\"pkg_GRaNIE\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/GRaNIE.html\\\">GRaNIE</a></td>\\n\\t<td class=\\\" \\\">Christian Arnold</td>\\n\\t<td class=\\\" \\\">GRaNIE: Reconstruction cell type specific gene regulatory networks including enhancers using chromatin accessibility and RNA-seq data</td>\\n\\t<td class=\\\"  sorting_1\\\">712</td>\\n</tr><tr id=\\\"pkg_graper\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/graper.html\\\">graper</a></td>\\n\\t<td class=\\\" \\\">Britta Velten</td>\\n\\t<td class=\\\" \\\">Adaptive penalization in high-dimensional regression and classification with external covariates using variational Bayes</td>\\n\\t<td class=\\\"  sorting_1\\\">712</td>\\n</tr><tr id=\\\"pkg_GraphPAC\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/GraphPAC.html\\\">GraphPAC</a></td>\\n\\t<td class=\\\" \\\">Gregory Ryslik</td>\\n\\t<td class=\\\" \\\">Identification of Mutational Clusters in Proteins via a Graph Theoretical Approach.</td>\\n\\t<td class=\\\"  sorting_1\\\">712</td>\\n</tr><tr id=\\\"pkg_iCNV\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/iCNV.html\\\">iCNV</a></td>\\n\\t<td class=\\\" \\\">Zilu Zhou</td>\\n\\t<td class=\\\" \\\">Integrated Copy Number Variation detection</td>\\n\\t<td class=\\\"  sorting_1\\\">712</td>\\n</tr><tr id=\\\"pkg_IMAS\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/IMAS.html\\\">IMAS</a></td>\\n\\t<td class=\\\" \\\">Seonggyun Han</td>\\n\\t<td class=\\\" \\\">Integrative analysis of Multi-omics data for Alternative Splicing</td>\\n\\t<td class=\\\"  sorting_1\\\">712</td>\\n</tr><tr id=\\\"pkg_InPAS\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/InPAS.html\\\">InPAS</a></td>\\n\\t<td class=\\\" \\\">Jianhong Ou</td>\\n\\t<td class=\\\" \\\">Identify Novel Alternative PolyAdenylation Sites (PAS) from RNA-seq data</td>\\n\\t<td class=\\\"  sorting_1\\\">712</td>\\n</tr><tr id=\\\"pkg_logitT\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/logitT.html\\\">logitT</a></td>\\n\\t<td class=\\\" \\\">Tobias Guennel</td>\\n\\t<td class=\\\" \\\">logit-t Package</td>\\n\\t<td class=\\\"  sorting_1\\\">712</td>\\n</tr><tr id=\\\"pkg_macat\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/macat.html\\\">macat</a></td>\\n\\t<td class=\\\" \\\">Joern Toedling</td>\\n\\t<td class=\\\" \\\">MicroArray Chromosome Analysis Tool</td>\\n\\t<td class=\\\"  sorting_1\\\">712</td>\\n</tr><tr id=\\\"pkg_MBCB\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/MBCB.html\\\">MBCB</a></td>\\n\\t<td class=\\\" \\\">Jeff Allen</td>\\n\\t<td class=\\\" \\\">MBCB (Model-based Background Correction for Beadarray)</td>\\n\\t<td class=\\\"  sorting_1\\\">712</td>\\n</tr><tr id=\\\"pkg_mdp\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/mdp.html\\\">mdp</a></td>\\n\\t<td class=\\\" \\\">Helder Nakaya</td>\\n\\t<td class=\\\" \\\">Molecular Degree of Perturbation calculates scores for transcriptome data samples based on their perturbation from controls</td>\\n\\t<td class=\\\"  sorting_1\\\">712</td>\\n</tr><tr id=\\\"pkg_methInheritSim\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/methInheritSim.html\\\">methInheritSim</a></td>\\n\\t<td class=\\\" \\\">Pascal Belleau</td>\\n\\t<td class=\\\" \\\">Simulating Whole-Genome Inherited Bisulphite Sequencing Data</td>\\n\\t<td class=\\\"  sorting_1\\\">712</td>\\n</tr><tr id=\\\"pkg_MetID\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/MetID.html\\\">MetID</a></td>\\n\\t<td class=\\\" \\\">Zhenzhi Li</td>\\n\\t<td class=\\\" \\\">Network-based prioritization of putative metabolite IDs</td>\\n\\t<td class=\\\"  sorting_1\\\">712</td>\\n</tr><tr id=\\\"pkg_MGFM\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/MGFM.html\\\">MGFM</a></td>\\n\\t<td class=\\\" \\\">Khadija El Amrani</td>\\n\\t<td class=\\\" \\\">Marker Gene Finder in Microarray gene expression data</td>\\n\\t<td class=\\\"  sorting_1\\\">712</td>\\n</tr><tr id=\\\"pkg_midasHLA\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/midasHLA.html\\\">midasHLA</a></td>\\n\\t<td class=\\\" \\\">Maciej Migda</td>\\n\\t<td class=\\\" \\\">R package for immunogenomics data handling and association analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">712</td>\\n</tr><tr id=\\\"pkg_motifcounter\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/motifcounter.html\\\">motifcounter</a></td>\\n\\t<td class=\\\" \\\">Wolfgang Kopp</td>\\n\\t<td class=\\\" \\\">R package for analysing TFBSs in DNA sequences</td>\\n\\t<td class=\\\"  sorting_1\\\">712</td>\\n</tr><tr id=\\\"pkg_mpra\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/mpra.html\\\">mpra</a></td>\\n\\t<td class=\\\" \\\">Leslie Myint</td>\\n\\t<td class=\\\" \\\">Analyze massively parallel reporter assays</td>\\n\\t<td class=\\\"  sorting_1\\\">712</td>\\n</tr><tr id=\\\"pkg_MultiMed\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/MultiMed.html\\\">MultiMed</a></td>\\n\\t<td class=\\\" \\\">Simina M. Boca</td>\\n\\t<td class=\\\" \\\">Testing multiple biological mediators simultaneously</td>\\n\\t<td class=\\\"  sorting_1\\\">712</td>\\n</tr><tr id=\\\"pkg_NetSAM\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/NetSAM.html\\\">NetSAM</a></td>\\n\\t<td class=\\\" \\\">Zhiao Shi</td>\\n\\t<td class=\\\" \\\">Network Seriation And Modularization</td>\\n\\t<td class=\\\"  sorting_1\\\">712</td>\\n</tr><tr id=\\\"pkg_NPARC\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/NPARC.html\\\">NPARC</a></td>\\n\\t<td class=\\\" \\\">Nils Kurzawa</td>\\n\\t<td class=\\\" \\\">Non-parametric analysis of response curves for thermal proteome profiling experiments</td>\\n\\t<td class=\\\"  sorting_1\\\">712</td>\\n</tr><tr id=\\\"pkg_omicplotR\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/omicplotR.html\\\">omicplotR</a></td>\\n\\t<td class=\\\" \\\">Daniel Giguere</td>\\n\\t<td class=\\\" \\\">Visual Exploration of Omic Datasets Using a Shiny App</td>\\n\\t<td class=\\\"  sorting_1\\\">712</td>\\n</tr><tr id=\\\"pkg_OncoScore\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/OncoScore.html\\\">OncoScore</a></td>\\n\\t<td class=\\\" \\\">Luca De Sano</td>\\n\\t<td class=\\\" \\\">A tool to identify potentially oncogenic genes</td>\\n\\t<td class=\\\"  sorting_1\\\">712</td>\\n</tr><tr id=\\\"pkg_paircompviz\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/paircompviz.html\\\">paircompviz</a></td>\\n\\t<td class=\\\" \\\">Michal Burda</td>\\n\\t<td class=\\\" \\\">Multiple comparison test visualization</td>\\n\\t<td class=\\\"  sorting_1\\\">712</td>\\n</tr><tr id=\\\"pkg_PrInCE\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/PrInCE.html\\\">PrInCE</a></td>\\n\\t<td class=\\\" \\\">Michael Skinnider</td>\\n\\t<td class=\\\" \\\">Predicting Interactomes from Co-Elution</td>\\n\\t<td class=\\\"  sorting_1\\\">712</td>\\n</tr><tr id=\\\"pkg_recountmethylation\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/recountmethylation.html\\\">recountmethylation</a></td>\\n\\t<td class=\\\" \\\">Sean K Maden</td>\\n\\t<td class=\\\" \\\">Access and analyze public DNA methylation array data compilations</td>\\n\\t<td class=\\\"  sorting_1\\\">712</td>\\n</tr><tr id=\\\"pkg_restfulSE\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/restfulSE.html\\\">restfulSE</a></td>\\n\\t<td class=\\\" \\\">Shweta Gopaulakrishnan</td>\\n\\t<td class=\\\" \\\">Access matrix-like HDF5 server content or BigQuery content through a SummarizedExperiment interface</td>\\n\\t<td class=\\\"  sorting_1\\\">712</td>\\n</tr><tr id=\\\"pkg_RNAmodR.AlkAnilineSeq\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/RNAmodR.AlkAnilineSeq.html\\\">RNAmodR.AlkAnilineSeq</a></td>\\n\\t<td class=\\\" \\\">Felix G.M. Ernst</td>\\n\\t<td class=\\\" \\\">Detection of m7G, m3C and D modification by AlkAnilineSeq</td>\\n\\t<td class=\\\"  sorting_1\\\">712</td>\\n</tr><tr id=\\\"pkg_seq2pathway\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/seq2pathway.html\\\">seq2pathway</a></td>\\n\\t<td class=\\\" \\\">Arjun Kinstlick</td>\\n\\t<td class=\\\" \\\">a novel tool for functional gene-set (or termed as pathway) analysis of next-generation sequencing data</td>\\n\\t<td class=\\\"  sorting_1\\\">712</td>\\n</tr><tr id=\\\"pkg_slalom\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/slalom.html\\\">slalom</a></td>\\n\\t<td class=\\\" \\\">Davis McCarthy</td>\\n\\t<td class=\\\" \\\">Factorial Latent Variable Modeling of Single-Cell RNA-Seq Data</td>\\n\\t<td class=\\\"  sorting_1\\\">712</td>\\n</tr><tr id=\\\"pkg_sRACIPE\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/sRACIPE.html\\\">sRACIPE</a></td>\\n\\t<td class=\\\" \\\">Vivek Kohar</td>\\n\\t<td class=\\\" \\\">Systems biology tool to simulate gene regulatory circuits</td>\\n\\t<td class=\\\"  sorting_1\\\">712</td>\\n</tr><tr id=\\\"pkg_ssviz\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ssviz.html\\\">ssviz</a></td>\\n\\t<td class=\\\" \\\">Diana Low</td>\\n\\t<td class=\\\" \\\">A small RNA-seq visualizer and analysis toolkit</td>\\n\\t<td class=\\\"  sorting_1\\\">712</td>\\n</tr><tr id=\\\"pkg_STAN\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/STAN.html\\\">STAN</a></td>\\n\\t<td class=\\\" \\\">Rafael Campos-Martin</td>\\n\\t<td class=\\\" \\\">The Genomic STate ANnotation Package</td>\\n\\t<td class=\\\"  sorting_1\\\">712</td>\\n</tr><tr id=\\\"pkg_SubCellBarCode\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/SubCellBarCode.html\\\">SubCellBarCode</a></td>\\n\\t<td class=\\\" \\\">Taner Arslan</td>\\n\\t<td class=\\\" \\\">SubCellBarCode: Integrated workflow for robust mapping and visualizing whole human spatial proteome</td>\\n\\t<td class=\\\"  sorting_1\\\">712</td>\\n</tr><tr id=\\\"pkg_switchde\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/switchde.html\\\">switchde</a></td>\\n\\t<td class=\\\" \\\">Kieran Campbell</td>\\n\\t<td class=\\\" \\\">Switch-like differential expression across single-cell trajectories</td>\\n\\t<td class=\\\"  sorting_1\\\">712</td>\\n</tr><tr id=\\\"pkg_TPP2D\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/TPP2D.html\\\">TPP2D</a></td>\\n\\t<td class=\\\" \\\">Nils Kurzawa</td>\\n\\t<td class=\\\" \\\">Detection of ligand-protein interactions from 2D thermal profiles (DLPTP)</td>\\n\\t<td class=\\\"  sorting_1\\\">712</td>\\n</tr><tr id=\\\"pkg_ttgsea\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ttgsea.html\\\">ttgsea</a></td>\\n\\t<td class=\\\" \\\">Dongmin Jung</td>\\n\\t<td class=\\\" \\\">Tokenizing Text of Gene Set Enrichment Analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">712</td>\\n</tr><tr id=\\\"pkg_VariantExperiment\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/VariantExperiment.html\\\">VariantExperiment</a></td>\\n\\t<td class=\\\" \\\">Qian Liu</td>\\n\\t<td class=\\\" \\\">A RangedSummarizedExperiment Container for VCF/GDS Data with GDS Backend</td>\\n\\t<td class=\\\"  sorting_1\\\">712</td>\\n</tr><tr id=\\\"pkg_AlphaBeta\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/AlphaBeta.html\\\">AlphaBeta</a></td>\\n\\t<td class=\\\" \\\">Yadollah Shahryary Dizaji</td>\\n\\t<td class=\\\" \\\">Computational inference of epimutation rates and spectra from high-throughput DNA methylation data in plants</td>\\n\\t<td class=\\\"  sorting_1\\\">772</td>\\n</tr><tr id=\\\"pkg_appreci8R\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/appreci8R.html\\\">appreci8R</a></td>\\n\\t<td class=\\\" \\\">Sarah Sandmann</td>\\n\\t<td class=\\\" \\\">appreci8R: an R/Bioconductor package for filtering SNVs and short indels with high sensitivity and high PPV</td>\\n\\t<td class=\\\"  sorting_1\\\">772</td>\\n</tr><tr id=\\\"pkg_BEAT\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/BEAT.html\\\">BEAT</a></td>\\n\\t<td class=\\\" \\\">Kemal Akman</td>\\n\\t<td class=\\\" \\\">BEAT - BS-Seq Epimutation Analysis Toolkit</td>\\n\\t<td class=\\\"  sorting_1\\\">772</td>\\n</tr><tr id=\\\"pkg_BiFET\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/BiFET.html\\\">BiFET</a></td>\\n\\t<td class=\\\" \\\">Ahrim Youn</td>\\n\\t<td class=\\\" \\\">Bias-free Footprint Enrichment Test</td>\\n\\t<td class=\\\"  sorting_1\\\">772</td>\\n</tr><tr id=\\\"pkg_biotmle\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/biotmle.html\\\">biotmle</a></td>\\n\\t<td class=\\\" \\\">Nima Hejazi</td>\\n\\t<td class=\\\" \\\">Targeted Learning with Moderated Statistics for Biomarker Discovery</td>\\n\\t<td class=\\\"  sorting_1\\\">772</td>\\n</tr><tr id=\\\"pkg_cbpManager\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/cbpManager.html\\\">cbpManager</a></td>\\n\\t<td class=\\\" \\\">Arsenij Ustjanzew</td>\\n\\t<td class=\\\" \\\">Generate, manage, and edit data and metadata files suitable for the import in cBioPortal for Cancer Genomics</td>\\n\\t<td class=\\\"  sorting_1\\\">772</td>\\n</tr><tr id=\\\"pkg_CellMapper\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/CellMapper.html\\\">CellMapper</a></td>\\n\\t<td class=\\\" \\\">Brad Nelms</td>\\n\\t<td class=\\\" \\\">Predict genes expressed selectively in specific cell types</td>\\n\\t<td class=\\\"  sorting_1\\\">772</td>\\n</tr><tr id=\\\"pkg_cisPath\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/cisPath.html\\\">cisPath</a></td>\\n\\t<td class=\\\" \\\">Likun Wang</td>\\n\\t<td class=\\\" \\\">Visualization and management of the protein-protein interaction networks.</td>\\n\\t<td class=\\\"  sorting_1\\\">772</td>\\n</tr><tr id=\\\"pkg_cleanUpdTSeq\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/cleanUpdTSeq.html\\\">cleanUpdTSeq</a></td>\\n\\t<td class=\\\" \\\">Jianhong Ou; Lihua Julie Zhu</td>\\n\\t<td class=\\\" \\\">cleanUpdTSeq cleans up artifacts from polyadenylation sites from oligo(dT)-mediated 3' end RNA sequending data</td>\\n\\t<td class=\\\"  sorting_1\\\">772</td>\\n</tr><tr id=\\\"pkg_clustComp\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/clustComp.html\\\">clustComp</a></td>\\n\\t<td class=\\\" \\\">Aurora Torrente</td>\\n\\t<td class=\\\" \\\">Clustering Comparison Package</td>\\n\\t<td class=\\\"  sorting_1\\\">772</td>\\n</tr><tr id=\\\"pkg_ClusterJudge\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ClusterJudge.html\\\">ClusterJudge</a></td>\\n\\t<td class=\\\" \\\">Adrian Pasculescu</td>\\n\\t<td class=\\\" \\\">Judging Quality of Clustering Methods using Mutual Information</td>\\n\\t<td class=\\\"  sorting_1\\\">772</td>\\n</tr><tr id=\\\"pkg_consensusDE\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/consensusDE.html\\\">consensusDE</a></td>\\n\\t<td class=\\\" \\\">Ashley J. Waardenberg</td>\\n\\t<td class=\\\" \\\">RNA-seq analysis using multiple algorithms</td>\\n\\t<td class=\\\"  sorting_1\\\">772</td>\\n</tr><tr id=\\\"pkg_cosmiq\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/cosmiq.html\\\">cosmiq</a></td>\\n\\t<td class=\\\" \\\">David Fischer</td>\\n\\t<td class=\\\" \\\">cosmiq - COmbining Single Masses Into Quantities</td>\\n\\t<td class=\\\"  sorting_1\\\">772</td>\\n</tr><tr id=\\\"pkg_cpvSNP\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/cpvSNP.html\\\">cpvSNP</a></td>\\n\\t<td class=\\\" \\\">Caitlin McHugh</td>\\n\\t<td class=\\\" \\\">Gene set analysis methods for SNP association p-values that lie in genes in given gene sets</td>\\n\\t<td class=\\\"  sorting_1\\\">772</td>\\n</tr><tr id=\\\"pkg_deconvR\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/deconvR.html\\\">deconvR</a></td>\\n\\t<td class=\\\" \\\">Irem B. Gndz</td>\\n\\t<td class=\\\" \\\">Simulation and Deconvolution of Omic Profiles</td>\\n\\t<td class=\\\"  sorting_1\\\">772</td>\\n</tr><tr id=\\\"pkg_DExMA\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/DExMA.html\\\">DExMA</a></td>\\n\\t<td class=\\\" \\\">Juan Antonio Villatoro-Garca</td>\\n\\t<td class=\\\" \\\">Differential Expression Meta-Analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">772</td>\\n</tr><tr id=\\\"pkg_DTA\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/DTA.html\\\">DTA</a></td>\\n\\t<td class=\\\" \\\">Bjoern Schwalb</td>\\n\\t<td class=\\\" \\\">Dynamic Transcriptome Analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">772</td>\\n</tr><tr id=\\\"pkg_dyebias\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/dyebias.html\\\">dyebias</a></td>\\n\\t<td class=\\\" \\\">Philip Lijnzaad</td>\\n\\t<td class=\\\" \\\">The GASSCO method for correcting for slide-dependent gene-specific dye bias</td>\\n\\t<td class=\\\"  sorting_1\\\">772</td>\\n</tr><tr id=\\\"pkg_enhancerHomologSearch\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/enhancerHomologSearch.html\\\">enhancerHomologSearch</a></td>\\n\\t<td class=\\\" \\\">Jianhong Ou</td>\\n\\t<td class=\\\" \\\">Identification of putative mammalian orthologs to given enhancer</td>\\n\\t<td class=\\\"  sorting_1\\\">772</td>\\n</tr><tr id=\\\"pkg_ExperimentSubset\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ExperimentSubset.html\\\">ExperimentSubset</a></td>\\n\\t<td class=\\\" \\\">Irzam Sarfraz</td>\\n\\t<td class=\\\" \\\">Manages subsets of data with Bioconductor Experiment objects</td>\\n\\t<td class=\\\"  sorting_1\\\">772</td>\\n</tr><tr id=\\\"pkg_factDesign\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/factDesign.html\\\">factDesign</a></td>\\n\\t<td class=\\\" \\\">Denise Scholtens</td>\\n\\t<td class=\\\" \\\">Factorial designed microarray experiment analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">772</td>\\n</tr><tr id=\\\"pkg_fCI\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/fCI.html\\\">fCI</a></td>\\n\\t<td class=\\\" \\\">Shaojun Tang</td>\\n\\t<td class=\\\" \\\">f-divergence Cutoff Index for Differential Expression Analysis in Transcriptomics and Proteomics</td>\\n\\t<td class=\\\"  sorting_1\\\">772</td>\\n</tr><tr id=\\\"pkg_FLAMES\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/FLAMES.html\\\">FLAMES</a></td>\\n\\t<td class=\\\" \\\">Voogd Oliver</td>\\n\\t<td class=\\\" \\\">FLAMES: Full Length Analysis of Mutations and Splicing in long read RNA-seq data</td>\\n\\t<td class=\\\"  sorting_1\\\">772</td>\\n</tr><tr id=\\\"pkg_flowBin\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/flowBin.html\\\">flowBin</a></td>\\n\\t<td class=\\\" \\\">Kieran O'Neill</td>\\n\\t<td class=\\\" \\\">Combining multitube flow cytometry data by binning</td>\\n\\t<td class=\\\"  sorting_1\\\">772</td>\\n</tr><tr id=\\\"pkg_flowPlots\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/flowPlots.html\\\">flowPlots</a></td>\\n\\t<td class=\\\" \\\">N. Hawkins</td>\\n\\t<td class=\\\" \\\">flowPlots: analysis plots and data class for gated flow cytometry data</td>\\n\\t<td class=\\\"  sorting_1\\\">772</td>\\n</tr><tr id=\\\"pkg_flowTime\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/flowTime.html\\\">flowTime</a></td>\\n\\t<td class=\\\" \\\">R. Clay Wright</td>\\n\\t<td class=\\\" \\\">Annotation and analysis of biological dynamical systems using flow cytometry</td>\\n\\t<td class=\\\"  sorting_1\\\">772</td>\\n</tr><tr id=\\\"pkg_FoldGO\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/FoldGO.html\\\">FoldGO</a></td>\\n\\t<td class=\\\" \\\">Daniil Wiebe</td>\\n\\t<td class=\\\" \\\">Package for Fold-specific GO Terms Recognition</td>\\n\\t<td class=\\\"  sorting_1\\\">772</td>\\n</tr><tr id=\\\"pkg_GAprediction\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/GAprediction.html\\\">GAprediction</a></td>\\n\\t<td class=\\\" \\\">Jon Bohlin</td>\\n\\t<td class=\\\" \\\">Prediction of gestational age with Illumina HumanMethylation450 data</td>\\n\\t<td class=\\\"  sorting_1\\\">772</td>\\n</tr><tr id=\\\"pkg_geneRecommender\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/geneRecommender.html\\\">geneRecommender</a></td>\\n\\t<td class=\\\" \\\">Greg Hather</td>\\n\\t<td class=\\\" \\\">A gene recommender algorithm to identify genes coexpressed with a query set of genes</td>\\n\\t<td class=\\\"  sorting_1\\\">772</td>\\n</tr><tr id=\\\"pkg_geneRxCluster\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/geneRxCluster.html\\\">geneRxCluster</a></td>\\n\\t<td class=\\\" \\\">Charles Berry</td>\\n\\t<td class=\\\" \\\">gRx Differential Clustering</td>\\n\\t<td class=\\\"  sorting_1\\\">772</td>\\n</tr><tr id=\\\"pkg_genoCN\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/genoCN.html\\\">genoCN</a></td>\\n\\t<td class=\\\" \\\">Wei Sun</td>\\n\\t<td class=\\\" \\\">genotyping and copy number study tools</td>\\n\\t<td class=\\\"  sorting_1\\\">772</td>\\n</tr><tr id=\\\"pkg_GISPA\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/GISPA.html\\\">GISPA</a></td>\\n\\t<td class=\\\" \\\">Bhakti Dwivedi</td>\\n\\t<td class=\\\" \\\">GISPA: Method for Gene Integrated Set Profile Analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">772</td>\\n</tr><tr id=\\\"pkg_globalSeq\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/globalSeq.html\\\">globalSeq</a></td>\\n\\t<td class=\\\" \\\">Armin Rauschenberger</td>\\n\\t<td class=\\\" \\\">Global Test for Counts</td>\\n\\t<td class=\\\"  sorting_1\\\">772</td>\\n</tr><tr id=\\\"pkg_gpart\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/gpart.html\\\">gpart</a></td>\\n\\t<td class=\\\" \\\">Sun Ah Kim</td>\\n\\t<td class=\\\" \\\">Human genome partitioning of dense sequencing data by identifying haplotype blocks</td>\\n\\t<td class=\\\"  sorting_1\\\">772</td>\\n</tr><tr id=\\\"pkg_GRENITS\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/GRENITS.html\\\">GRENITS</a></td>\\n\\t<td class=\\\" \\\">Edward Morrissey</td>\\n\\t<td class=\\\" \\\">Gene Regulatory Network Inference Using Time Series</td>\\n\\t<td class=\\\"  sorting_1\\\">772</td>\\n</tr><tr id=\\\"pkg_gsean\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/gsean.html\\\">gsean</a></td>\\n\\t<td class=\\\" \\\">Dongmin Jung</td>\\n\\t<td class=\\\" \\\">Gene Set Enrichment Analysis with Networks</td>\\n\\t<td class=\\\"  sorting_1\\\">772</td>\\n</tr><tr id=\\\"pkg_hierGWAS\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/hierGWAS.html\\\">hierGWAS</a></td>\\n\\t<td class=\\\" \\\">Laura Buzdugan</td>\\n\\t<td class=\\\" \\\">Asessing statistical significance in predictive GWA studies</td>\\n\\t<td class=\\\"  sorting_1\\\">772</td>\\n</tr><tr id=\\\"pkg_immunoClust\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/immunoClust.html\\\">immunoClust</a></td>\\n\\t<td class=\\\" \\\">Till Soerensen</td>\\n\\t<td class=\\\" \\\">immunoClust - Automated Pipeline for Population Detection in Flow Cytometry</td>\\n\\t<td class=\\\"  sorting_1\\\">772</td>\\n</tr><tr id=\\\"pkg_infinityFlow\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/infinityFlow.html\\\">infinityFlow</a></td>\\n\\t<td class=\\\" \\\">Etienne Becht</td>\\n\\t<td class=\\\" \\\">Augmenting Massively Parallel Cytometry Experiments Using Multivariate Non-Linear Regressions</td>\\n\\t<td class=\\\"  sorting_1\\\">772</td>\\n</tr><tr id=\\\"pkg_IntramiRExploreR\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/IntramiRExploreR.html\\\">IntramiRExploreR</a></td>\\n\\t<td class=\\\" \\\">Surajit Bhattacharya</td>\\n\\t<td class=\\\" \\\">Predicting Targets for Drosophila Intragenic miRNAs</td>\\n\\t<td class=\\\"  sorting_1\\\">772</td>\\n</tr><tr id=\\\"pkg_ITALICS\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ITALICS.html\\\">ITALICS</a></td>\\n\\t<td class=\\\" \\\">Guillem Rigaill</td>\\n\\t<td class=\\\" \\\">ITALICS</td>\\n\\t<td class=\\\"  sorting_1\\\">772</td>\\n</tr><tr id=\\\"pkg_KinSwingR\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/KinSwingR.html\\\">KinSwingR</a></td>\\n\\t<td class=\\\" \\\">Ashley J. Waardenberg</td>\\n\\t<td class=\\\" \\\">KinSwingR: network-based kinase activity prediction</td>\\n\\t<td class=\\\"  sorting_1\\\">772</td>\\n</tr><tr id=\\\"pkg_LiquidAssociation\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/LiquidAssociation.html\\\">LiquidAssociation</a></td>\\n\\t<td class=\\\" \\\">Yen-Yi Ho</td>\\n\\t<td class=\\\" \\\">LiquidAssociation</td>\\n\\t<td class=\\\"  sorting_1\\\">772</td>\\n</tr><tr id=\\\"pkg_MANOR\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/MANOR.html\\\">MANOR</a></td>\\n\\t<td class=\\\" \\\">Pierre Neuvial</td>\\n\\t<td class=\\\" \\\">CGH Micro-Array NORmalization</td>\\n\\t<td class=\\\"  sorting_1\\\">772</td>\\n</tr><tr id=\\\"pkg_miaSim\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/miaSim.html\\\">miaSim</a></td>\\n\\t<td class=\\\" \\\">Yagmur Simsek</td>\\n\\t<td class=\\\" \\\">Microbiome Data Simulation</td>\\n\\t<td class=\\\"  sorting_1\\\">772</td>\\n</tr><tr id=\\\"pkg_MSstatsQC\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/MSstatsQC.html\\\">MSstatsQC</a></td>\\n\\t<td class=\\\" \\\">Eralp Dogu</td>\\n\\t<td class=\\\" \\\">Longitudinal system suitability monitoring and quality control for proteomic experiments</td>\\n\\t<td class=\\\"  sorting_1\\\">772</td>\\n</tr><tr id=\\\"pkg_npGSEA\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/npGSEA.html\\\">npGSEA</a></td>\\n\\t<td class=\\\" \\\">Jessica Larson</td>\\n\\t<td class=\\\" \\\">Permutation approximation methods for gene set enrichment analysis (non-permutation GSEA)</td>\\n\\t<td class=\\\"  sorting_1\\\">772</td>\\n</tr><tr id=\\\"pkg_panp\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/panp.html\\\">panp</a></td>\\n\\t<td class=\\\" \\\">Peter Warren</td>\\n\\t<td class=\\\" \\\">Presence-Absence Calls from Negative Strand Matching Probesets</td>\\n\\t<td class=\\\"  sorting_1\\\">772</td>\\n</tr><tr id=\\\"pkg_PathoStat\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/PathoStat.html\\\">PathoStat</a></td>\\n\\t<td class=\\\" \\\">Solaiappan Manimaran, Yue Zhao</td>\\n\\t<td class=\\\" \\\">PathoStat Statistical Microbiome Analysis Package</td>\\n\\t<td class=\\\"  sorting_1\\\">772</td>\\n</tr><tr id=\\\"pkg_pepXMLTab\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/pepXMLTab.html\\\">pepXMLTab</a></td>\\n\\t<td class=\\\" \\\">Xiaojing Wang</td>\\n\\t<td class=\\\" \\\">Parsing pepXML files and filter based on peptide FDR.</td>\\n\\t<td class=\\\"  sorting_1\\\">772</td>\\n</tr><tr id=\\\"pkg_PhenStat\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/PhenStat.html\\\">PhenStat</a></td>\\n\\t<td class=\\\" \\\">Hamed Haselimashhadi</td>\\n\\t<td class=\\\" \\\">Statistical analysis of phenotypic data</td>\\n\\t<td class=\\\"  sorting_1\\\">772</td>\\n</tr><tr id=\\\"pkg_RCM\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/RCM.html\\\">RCM</a></td>\\n\\t<td class=\\\" \\\">Stijn Hawinkel</td>\\n\\t<td class=\\\" \\\">Fit row-column association models with the negative binomial distribution for the microbiome</td>\\n\\t<td class=\\\"  sorting_1\\\">772</td>\\n</tr><tr id=\\\"pkg_Rcwl\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/Rcwl.html\\\">Rcwl</a></td>\\n\\t<td class=\\\" \\\">Qiang Hu</td>\\n\\t<td class=\\\" \\\">An R interface to the Common Workflow Language</td>\\n\\t<td class=\\\"  sorting_1\\\">772</td>\\n</tr><tr id=\\\"pkg_RLassoCox\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/RLassoCox.html\\\">RLassoCox</a></td>\\n\\t<td class=\\\" \\\">Wei Liu</td>\\n\\t<td class=\\\" \\\">A reweighted Lasso-Cox by integrating gene interaction information</td>\\n\\t<td class=\\\"  sorting_1\\\">772</td>\\n</tr><tr id=\\\"pkg_RNAinteract\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/RNAinteract.html\\\">RNAinteract</a></td>\\n\\t<td class=\\\" \\\">Mike Smith</td>\\n\\t<td class=\\\" \\\">Estimate Pairwise Interactions from multidimensional features</td>\\n\\t<td class=\\\"  sorting_1\\\">772</td>\\n</tr><tr id=\\\"pkg_rRDP\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/rRDP.html\\\">rRDP</a></td>\\n\\t<td class=\\\" \\\">Michael Hahsler</td>\\n\\t<td class=\\\" \\\">Interface to the RDP Classifier</td>\\n\\t<td class=\\\"  sorting_1\\\">772</td>\\n</tr><tr id=\\\"pkg_scCB2\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/scCB2.html\\\">scCB2</a></td>\\n\\t<td class=\\\" \\\">Zijian Ni</td>\\n\\t<td class=\\\" \\\">CB2 improves power of cell detection in droplet-based single-cell RNA sequencing data</td>\\n\\t<td class=\\\"  sorting_1\\\">772</td>\\n</tr><tr id=\\\"pkg_SCOPE\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/SCOPE.html\\\">SCOPE</a></td>\\n\\t<td class=\\\" \\\">Rujin Wang</td>\\n\\t<td class=\\\" \\\">A normalization and copy number estimation method for single-cell DNA sequencing</td>\\n\\t<td class=\\\"  sorting_1\\\">772</td>\\n</tr><tr id=\\\"pkg_scRecover\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/scRecover.html\\\">scRecover</a></td>\\n\\t<td class=\\\" \\\">Zhun Miao</td>\\n\\t<td class=\\\" \\\">scRecover for imputation of single-cell RNA-seq data</td>\\n\\t<td class=\\\"  sorting_1\\\">772</td>\\n</tr><tr id=\\\"pkg_SeqSQC\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/SeqSQC.html\\\">SeqSQC</a></td>\\n\\t<td class=\\\" \\\">Qian Liu</td>\\n\\t<td class=\\\" \\\">A bioconductor package for sample quality check with next generation sequencing data</td>\\n\\t<td class=\\\"  sorting_1\\\">772</td>\\n</tr><tr id=\\\"pkg_SIM\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/SIM.html\\\">SIM</a></td>\\n\\t<td class=\\\" \\\">Renee X. de Menezes</td>\\n\\t<td class=\\\" \\\">Integrated Analysis on two human genomic datasets</td>\\n\\t<td class=\\\"  sorting_1\\\">772</td>\\n</tr><tr id=\\\"pkg_spatialHeatmap\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/spatialHeatmap.html\\\">spatialHeatmap</a></td>\\n\\t<td class=\\\" \\\">Jianhai Zhang</td>\\n\\t<td class=\\\" \\\">spatialHeatmap</td>\\n\\t<td class=\\\"  sorting_1\\\">772</td>\\n</tr><tr id=\\\"pkg_synlet\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/synlet.html\\\">synlet</a></td>\\n\\t<td class=\\\" \\\">Chunxuan Shao</td>\\n\\t<td class=\\\" \\\">Hits Selection for Synthetic Lethal RNAi Screen Data</td>\\n\\t<td class=\\\"  sorting_1\\\">772</td>\\n</tr><tr id=\\\"pkg_TargetScore\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/TargetScore.html\\\">TargetScore</a></td>\\n\\t<td class=\\\" \\\">Yue Li</td>\\n\\t<td class=\\\" \\\">TargetScore: Infer microRNA targets using microRNA-overexpression data and sequence information</td>\\n\\t<td class=\\\"  sorting_1\\\">772</td>\\n</tr><tr id=\\\"pkg_TimiRGeN\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/TimiRGeN.html\\\">TimiRGeN</a></td>\\n\\t<td class=\\\" \\\">Krutik Patel</td>\\n\\t<td class=\\\" \\\">Time sensitive microRNA-mRNA integration, analysis and network generation tool</td>\\n\\t<td class=\\\"  sorting_1\\\">772</td>\\n</tr><tr id=\\\"pkg_transite\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/transite.html\\\">transite</a></td>\\n\\t<td class=\\\" \\\">Konstantin Krismer</td>\\n\\t<td class=\\\" \\\">RNA-binding protein motif analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">772</td>\\n</tr><tr id=\\\"pkg_VplotR\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/VplotR.html\\\">VplotR</a></td>\\n\\t<td class=\\\" \\\">Jacques Serizay</td>\\n\\t<td class=\\\" \\\">Set of tools to make V-plots and compute footprint profiles</td>\\n\\t<td class=\\\"  sorting_1\\\">772</td>\\n</tr><tr id=\\\"pkg_wavClusteR\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/wavClusteR.html\\\">wavClusteR</a></td>\\n\\t<td class=\\\" \\\">Federico Comoglio</td>\\n\\t<td class=\\\" \\\">Sensitive and highly resolved identification of RNA-protein interaction sites in PAR-CLIP data</td>\\n\\t<td class=\\\"  sorting_1\\\">772</td>\\n</tr><tr id=\\\"pkg_yamss\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/yamss.html\\\">yamss</a></td>\\n\\t<td class=\\\" \\\">Leslie Myint</td>\\n\\t<td class=\\\" \\\">Tools for high-throughput metabolomics</td>\\n\\t<td class=\\\"  sorting_1\\\">772</td>\\n</tr><tr id=\\\"pkg_AffiXcan\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/AffiXcan.html\\\">AffiXcan</a></td>\\n\\t<td class=\\\" \\\">Alessandro Lussana</td>\\n\\t<td class=\\\" \\\">A Functional Approach To Impute Genetically Regulated Expression</td>\\n\\t<td class=\\\"  sorting_1\\\">841</td>\\n</tr><tr id=\\\"pkg_arrayMvout\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/arrayMvout.html\\\">arrayMvout</a></td>\\n\\t<td class=\\\" \\\">V. Carey</td>\\n\\t<td class=\\\" \\\">multivariate outlier detection for expression array QA</td>\\n\\t<td class=\\\"  sorting_1\\\">841</td>\\n</tr><tr id=\\\"pkg_ARRmNormalization\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ARRmNormalization.html\\\">ARRmNormalization</a></td>\\n\\t<td class=\\\" \\\">Jean-Philippe Fortin</td>\\n\\t<td class=\\\" \\\">Adaptive Robust Regression normalization for Illumina methylation data</td>\\n\\t<td class=\\\"  sorting_1\\\">841</td>\\n</tr><tr id=\\\"pkg_AssessORF\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/AssessORF.html\\\">AssessORF</a></td>\\n\\t<td class=\\\" \\\">Deepank Korandla</td>\\n\\t<td class=\\\" \\\">Assess Gene Predictions Using Proteomics and Evolutionary Conservation</td>\\n\\t<td class=\\\"  sorting_1\\\">841</td>\\n</tr><tr id=\\\"pkg_ASURAT\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ASURAT.html\\\">ASURAT</a></td>\\n\\t<td class=\\\" \\\">Keita Iida</td>\\n\\t<td class=\\\" \\\">Functional annotation-driven unsupervised clustering for single-cell data</td>\\n\\t<td class=\\\"  sorting_1\\\">841</td>\\n</tr><tr id=\\\"pkg_BioTIP\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/BioTIP.html\\\">BioTIP</a></td>\\n\\t<td class=\\\" \\\">Yuxi (Jennifer) Sun, Zhezhen Wang, and X Holly Yang</td>\\n\\t<td class=\\\" \\\">BioTIP: An R package for characterization of Biological Tipping-Point</td>\\n\\t<td class=\\\"  sorting_1\\\">841</td>\\n</tr><tr id=\\\"pkg_ChIPexoQual\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ChIPexoQual.html\\\">ChIPexoQual</a></td>\\n\\t<td class=\\\" \\\">Rene Welch</td>\\n\\t<td class=\\\" \\\">ChIPexoQual</td>\\n\\t<td class=\\\"  sorting_1\\\">841</td>\\n</tr><tr id=\\\"pkg_CNVgears\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/CNVgears.html\\\">CNVgears</a></td>\\n\\t<td class=\\\" \\\">Simone Montalbano</td>\\n\\t<td class=\\\" \\\">A Framework of Functions to Combine, Analize and Interpret CNVs Calling Results</td>\\n\\t<td class=\\\"  sorting_1\\\">841</td>\\n</tr><tr id=\\\"pkg_compartmap\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/compartmap.html\\\">compartmap</a></td>\\n\\t<td class=\\\" \\\">Benjamin Johnson</td>\\n\\t<td class=\\\" \\\">Higher-order chromatin domain inference in single cells from scRNA-seq and scATAC-seq</td>\\n\\t<td class=\\\"  sorting_1\\\">841</td>\\n</tr><tr id=\\\"pkg_copa\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/copa.html\\\">copa</a></td>\\n\\t<td class=\\\" \\\">James W. MacDonald</td>\\n\\t<td class=\\\" \\\">Functions to perform cancer outlier profile analysis.</td>\\n\\t<td class=\\\"  sorting_1\\\">841</td>\\n</tr><tr id=\\\"pkg_COSNet\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/COSNet.html\\\">COSNet</a></td>\\n\\t<td class=\\\" \\\">Marco Frasca</td>\\n\\t<td class=\\\" \\\">Cost Sensitive Network for node label prediction on graphs with highly unbalanced labelings</td>\\n\\t<td class=\\\"  sorting_1\\\">841</td>\\n</tr><tr id=\\\"pkg_CytoDx\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/CytoDx.html\\\">CytoDx</a></td>\\n\\t<td class=\\\" \\\">Zicheng Hu</td>\\n\\t<td class=\\\" \\\">Robust prediction of clinical outcomes using cytometry data without cell gating</td>\\n\\t<td class=\\\"  sorting_1\\\">841</td>\\n</tr><tr id=\\\"pkg_dasper\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/dasper.html\\\">dasper</a></td>\\n\\t<td class=\\\" \\\">David Zhang</td>\\n\\t<td class=\\\" \\\">Detecting abberant splicing events from RNA-sequencing data</td>\\n\\t<td class=\\\"  sorting_1\\\">841</td>\\n</tr><tr id=\\\"pkg_dce\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/dce.html\\\">dce</a></td>\\n\\t<td class=\\\" \\\">Kim Philipp Jablonski</td>\\n\\t<td class=\\\" \\\">Pathway Enrichment Based on Differential Causal Effects</td>\\n\\t<td class=\\\"  sorting_1\\\">841</td>\\n</tr><tr id=\\\"pkg_DEGraph\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/DEGraph.html\\\">DEGraph</a></td>\\n\\t<td class=\\\" \\\">Laurent Jacob</td>\\n\\t<td class=\\\" \\\">Two-sample tests on a graph</td>\\n\\t<td class=\\\"  sorting_1\\\">841</td>\\n</tr><tr id=\\\"pkg_ExiMiR\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ExiMiR.html\\\">ExiMiR</a></td>\\n\\t<td class=\\\" \\\">Sylvain Gubian</td>\\n\\t<td class=\\\" \\\">R functions for the normalization of Exiqon miRNA array data</td>\\n\\t<td class=\\\"  sorting_1\\\">841</td>\\n</tr><tr id=\\\"pkg_flowPloidy\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/flowPloidy.html\\\">flowPloidy</a></td>\\n\\t<td class=\\\" \\\">Tyler Smith</td>\\n\\t<td class=\\\" \\\">Analyze flow cytometer data to determine sample ploidy</td>\\n\\t<td class=\\\"  sorting_1\\\">841</td>\\n</tr><tr id=\\\"pkg_GateFinder\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/GateFinder.html\\\">GateFinder</a></td>\\n\\t<td class=\\\" \\\">Nima Aghaeepour</td>\\n\\t<td class=\\\" \\\">Projection-based Gating Strategy Optimization for Flow and Mass Cytometry</td>\\n\\t<td class=\\\"  sorting_1\\\">841</td>\\n</tr><tr id=\\\"pkg_GEM\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/GEM.html\\\">GEM</a></td>\\n\\t<td class=\\\" \\\">Hong Pan</td>\\n\\t<td class=\\\" \\\">GEM: fast association study for the interplay of Gene, Environment and Methylation</td>\\n\\t<td class=\\\"  sorting_1\\\">841</td>\\n</tr><tr id=\\\"pkg_gpuMagic\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/gpuMagic.html\\\">gpuMagic</a></td>\\n\\t<td class=\\\" \\\">Jiefei Wang</td>\\n\\t<td class=\\\" \\\">An openCL compiler with the capacity to compile R functions and run the code on GPU</td>\\n\\t<td class=\\\"  sorting_1\\\">841</td>\\n</tr><tr id=\\\"pkg_GSCA\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/GSCA.html\\\">GSCA</a></td>\\n\\t<td class=\\\" \\\">Zhicheng Ji</td>\\n\\t<td class=\\\" \\\">GSCA: Gene Set Context Analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">841</td>\\n</tr><tr id=\\\"pkg_GSRI\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/GSRI.html\\\">GSRI</a></td>\\n\\t<td class=\\\" \\\">Julian Gehring</td>\\n\\t<td class=\\\" \\\">Gene Set Regulation Index</td>\\n\\t<td class=\\\"  sorting_1\\\">841</td>\\n</tr><tr id=\\\"pkg_HDTD\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/HDTD.html\\\">HDTD</a></td>\\n\\t<td class=\\\" \\\">Anestis Touloumis</td>\\n\\t<td class=\\\" \\\">Statistical Inference about the Mean Matrix and the Covariance Matrices in High-Dimensional Transposable Data (HDTD)</td>\\n\\t<td class=\\\"  sorting_1\\\">841</td>\\n</tr><tr id=\\\"pkg_imageHTS\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/imageHTS.html\\\">imageHTS</a></td>\\n\\t<td class=\\\" \\\">Joseph Barry</td>\\n\\t<td class=\\\" \\\">Analysis of high-throughput microscopy-based screens</td>\\n\\t<td class=\\\"  sorting_1\\\">841</td>\\n</tr><tr id=\\\"pkg_iterClust\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/iterClust.html\\\">iterClust</a></td>\\n\\t<td class=\\\" \\\">Hongxu Ding</td>\\n\\t<td class=\\\" \\\">Iterative Clustering</td>\\n\\t<td class=\\\"  sorting_1\\\">841</td>\\n</tr><tr id=\\\"pkg_les\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/les.html\\\">les</a></td>\\n\\t<td class=\\\" \\\">Julian Gehring</td>\\n\\t<td class=\\\" \\\">Identifying Differential Effects in Tiling Microarray Data</td>\\n\\t<td class=\\\"  sorting_1\\\">841</td>\\n</tr><tr id=\\\"pkg_MassArray\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/MassArray.html\\\">MassArray</a></td>\\n\\t<td class=\\\" \\\">Reid F. Thompson</td>\\n\\t<td class=\\\" \\\">Analytical Tools for MassArray Data</td>\\n\\t<td class=\\\"  sorting_1\\\">841</td>\\n</tr><tr id=\\\"pkg_massiR\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/massiR.html\\\">massiR</a></td>\\n\\t<td class=\\\" \\\">Sam Buckberry</td>\\n\\t<td class=\\\" \\\">massiR: MicroArray Sample Sex Identifier</td>\\n\\t<td class=\\\"  sorting_1\\\">841</td>\\n</tr><tr id=\\\"pkg_MBASED\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/MBASED.html\\\">MBASED</a></td>\\n\\t<td class=\\\" \\\">Oleg Mayba</td>\\n\\t<td class=\\\" \\\">Package containing functions for ASE analysis using Meta-analysis Based Allele-Specific Expression Detection</td>\\n\\t<td class=\\\"  sorting_1\\\">841</td>\\n</tr><tr id=\\\"pkg_MSA2dist\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/MSA2dist.html\\\">MSA2dist</a></td>\\n\\t<td class=\\\" \\\">Kristian K Ullrich</td>\\n\\t<td class=\\\" \\\">MSA2dist calculates pairwise distances between all sequences of a DNAStringSet or a AAStringSet using a custom score matrix and conducts codon based analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">841</td>\\n</tr><tr id=\\\"pkg_netSmooth\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/netSmooth.html\\\">netSmooth</a></td>\\n\\t<td class=\\\" \\\">Jonathan Ronen</td>\\n\\t<td class=\\\" \\\">Network smoothing for scRNAseq</td>\\n\\t<td class=\\\"  sorting_1\\\">841</td>\\n</tr><tr id=\\\"pkg_NewWave\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/NewWave.html\\\">NewWave</a></td>\\n\\t<td class=\\\" \\\">Federico Agostinis</td>\\n\\t<td class=\\\" \\\">Negative binomial model for scRNA-seq</td>\\n\\t<td class=\\\"  sorting_1\\\">841</td>\\n</tr><tr id=\\\"pkg_Path2PPI\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/Path2PPI.html\\\">Path2PPI</a></td>\\n\\t<td class=\\\" \\\">Oliver Philipp</td>\\n\\t<td class=\\\" \\\">Prediction of pathway-related protein-protein interaction networks</td>\\n\\t<td class=\\\"  sorting_1\\\">841</td>\\n</tr><tr id=\\\"pkg_pathRender\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/pathRender.html\\\">pathRender</a></td>\\n\\t<td class=\\\" \\\">Vince Carey</td>\\n\\t<td class=\\\" \\\">Render molecular pathways</td>\\n\\t<td class=\\\"  sorting_1\\\">841</td>\\n</tr><tr id=\\\"pkg_psygenet2r\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/psygenet2r.html\\\">psygenet2r</a></td>\\n\\t<td class=\\\" \\\">Alba Gutierrez-Sacristan</td>\\n\\t<td class=\\\" \\\">psygenet2r - An R package for querying PsyGeNET and to perform comorbidity studies in psychiatric disorders</td>\\n\\t<td class=\\\"  sorting_1\\\">841</td>\\n</tr><tr id=\\\"pkg_qpcrNorm\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/qpcrNorm.html\\\">qpcrNorm</a></td>\\n\\t<td class=\\\" \\\">Jessica Mar</td>\\n\\t<td class=\\\" \\\">Data-driven normalization strategies for high-throughput qPCR data.</td>\\n\\t<td class=\\\"  sorting_1\\\">841</td>\\n</tr><tr id=\\\"pkg_ramr\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ramr.html\\\">ramr</a></td>\\n\\t<td class=\\\" \\\">Oleksii Nikolaienko</td>\\n\\t<td class=\\\" \\\">Detection of Rare Aberrantly Methylated Regions in Array and NGS Data</td>\\n\\t<td class=\\\"  sorting_1\\\">841</td>\\n</tr><tr id=\\\"pkg_randPack\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/randPack.html\\\">randPack</a></td>\\n\\t<td class=\\\" \\\">Robert Gentleman</td>\\n\\t<td class=\\\" \\\">Randomization routines for Clinical Trials</td>\\n\\t<td class=\\\"  sorting_1\\\">841</td>\\n</tr><tr id=\\\"pkg_sincell\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/sincell.html\\\">sincell</a></td>\\n\\t<td class=\\\" \\\">Miguel Julia, Antonio Rausell</td>\\n\\t<td class=\\\" \\\">R package for the statistical assessment of cell state hierarchies from single-cell RNA-seq data</td>\\n\\t<td class=\\\"  sorting_1\\\">841</td>\\n</tr><tr id=\\\"pkg_SMITE\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/SMITE.html\\\">SMITE</a></td>\\n\\t<td class=\\\" \\\">Neil Ari Wijetunga, Andrew Damon Johnston</td>\\n\\t<td class=\\\" \\\">Significance-based Modules Integrating the Transcriptome and Epigenome</td>\\n\\t<td class=\\\"  sorting_1\\\">841</td>\\n</tr><tr id=\\\"pkg_specL\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/specL.html\\\">specL</a></td>\\n\\t<td class=\\\" \\\">Christian Panse</td>\\n\\t<td class=\\\" \\\">specL - Prepare Peptide Spectrum Matches for Use in Targeted Proteomics</td>\\n\\t<td class=\\\"  sorting_1\\\">841</td>\\n</tr><tr id=\\\"pkg_sscore\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/sscore.html\\\">sscore</a></td>\\n\\t<td class=\\\" \\\">Richard Kennedy</td>\\n\\t<td class=\\\" \\\">S-Score Algorithm for Affymetrix Oligonucleotide Microarrays</td>\\n\\t<td class=\\\"  sorting_1\\\">841</td>\\n</tr><tr id=\\\"pkg_topdownr\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/topdownr.html\\\">topdownr</a></td>\\n\\t<td class=\\\" \\\">Sebastian Gibb</td>\\n\\t<td class=\\\" \\\">Investigation of Fragmentation Conditions in Top-Down Proteomics</td>\\n\\t<td class=\\\"  sorting_1\\\">841</td>\\n</tr><tr id=\\\"pkg_weaver\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/weaver.html\\\">weaver</a></td>\\n\\t<td class=\\\" \\\">Seth Falcon</td>\\n\\t<td class=\\\" \\\">Tools and extensions for processing Sweave documents</td>\\n\\t<td class=\\\"  sorting_1\\\">841</td>\\n</tr><tr id=\\\"pkg_webbioc\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/webbioc.html\\\">webbioc</a></td>\\n\\t<td class=\\\" \\\">Colin A. Smith</td>\\n\\t<td class=\\\" \\\">Bioconductor Web Interface</td>\\n\\t<td class=\\\"  sorting_1\\\">841</td>\\n</tr><tr id=\\\"pkg_affyILM\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/affyILM.html\\\">affyILM</a></td>\\n\\t<td class=\\\" \\\">Myriam Kroll and Fabrice Berger</td>\\n\\t<td class=\\\" \\\">Linear Model of background subtraction and the Langmuir isotherm</td>\\n\\t<td class=\\\"  sorting_1\\\">886</td>\\n</tr><tr id=\\\"pkg_autonomics\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/autonomics.html\\\">autonomics</a></td>\\n\\t<td class=\\\" \\\">Aditya Bhagwat</td>\\n\\t<td class=\\\" \\\">Generifying and intuifying cross-platform omics analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">886</td>\\n</tr><tr id=\\\"pkg_bcSeq\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/bcSeq.html\\\">bcSeq</a></td>\\n\\t<td class=\\\" \\\">Jiaxing Lin</td>\\n\\t<td class=\\\" \\\">Fast Sequence Mapping in High-Throughput shRNA and CRISPR Screens</td>\\n\\t<td class=\\\"  sorting_1\\\">886</td>\\n</tr><tr id=\\\"pkg_biobtreeR\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/biobtreeR.html\\\">biobtreeR</a></td>\\n\\t<td class=\\\" \\\">Tamer Gur</td>\\n\\t<td class=\\\" \\\">Using biobtree tool from R</td>\\n\\t<td class=\\\"  sorting_1\\\">886</td>\\n</tr><tr id=\\\"pkg_BioMVCClass\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/BioMVCClass.html\\\">BioMVCClass</a></td>\\n\\t<td class=\\\" \\\">Elizabeth Whalen</td>\\n\\t<td class=\\\" \\\">Model-View-Controller (MVC) Classes That Use Biobase</td>\\n\\t<td class=\\\"  sorting_1\\\">886</td>\\n</tr><tr id=\\\"pkg_blima\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/blima.html\\\">blima</a></td>\\n\\t<td class=\\\" \\\">Vojtch Kulvait</td>\\n\\t<td class=\\\" \\\">Tools for the preprocessing and analysis of the Illumina microarrays on the detector (bead) level</td>\\n\\t<td class=\\\"  sorting_1\\\">886</td>\\n</tr><tr id=\\\"pkg_bnbc\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/bnbc.html\\\">bnbc</a></td>\\n\\t<td class=\\\" \\\">Kipper Fletez-Brant</td>\\n\\t<td class=\\\" \\\">Bandwise normalization and batch correction of Hi-C data</td>\\n\\t<td class=\\\"  sorting_1\\\">886</td>\\n</tr><tr id=\\\"pkg_breakpointR\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/breakpointR.html\\\">breakpointR</a></td>\\n\\t<td class=\\\" \\\">David Porubsky</td>\\n\\t<td class=\\\" \\\">Find breakpoints in Strand-seq data</td>\\n\\t<td class=\\\"  sorting_1\\\">886</td>\\n</tr><tr id=\\\"pkg_brendaDb\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/brendaDb.html\\\">brendaDb</a></td>\\n\\t<td class=\\\" \\\">Yi Zhou</td>\\n\\t<td class=\\\" \\\">The BRENDA Enzyme Database</td>\\n\\t<td class=\\\"  sorting_1\\\">886</td>\\n</tr><tr id=\\\"pkg_BufferedMatrixMethods\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/BufferedMatrixMethods.html\\\">BufferedMatrixMethods</a></td>\\n\\t<td class=\\\" \\\">Ben Bolstad</td>\\n\\t<td class=\\\" \\\">Microarray Data related methods that utlize BufferedMatrix objects</td>\\n\\t<td class=\\\"  sorting_1\\\">886</td>\\n</tr><tr id=\\\"pkg_categoryCompare\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/categoryCompare.html\\\">categoryCompare</a></td>\\n\\t<td class=\\\" \\\">Robert M. Flight</td>\\n\\t<td class=\\\" \\\">Meta-analysis of high-throughput experiments using feature annotations</td>\\n\\t<td class=\\\"  sorting_1\\\">886</td>\\n</tr><tr id=\\\"pkg_cn.farms\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/cn.farms.html\\\">cn.farms</a></td>\\n\\t<td class=\\\" \\\">Andreas Mitterecker</td>\\n\\t<td class=\\\" \\\">cn.FARMS - factor analysis for copy number estimation</td>\\n\\t<td class=\\\"  sorting_1\\\">886</td>\\n</tr><tr id=\\\"pkg_CNORdt\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/CNORdt.html\\\">CNORdt</a></td>\\n\\t<td class=\\\" \\\">A. MacNamara</td>\\n\\t<td class=\\\" \\\">Add-on to CellNOptR: Discretized time treatments</td>\\n\\t<td class=\\\"  sorting_1\\\">886</td>\\n</tr><tr id=\\\"pkg_CNORfuzzy\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/CNORfuzzy.html\\\">CNORfuzzy</a></td>\\n\\t<td class=\\\" \\\">T. Cokelaer</td>\\n\\t<td class=\\\" \\\">Addon to CellNOptR: Fuzzy Logic</td>\\n\\t<td class=\\\"  sorting_1\\\">886</td>\\n</tr><tr id=\\\"pkg_COCOA\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/COCOA.html\\\">COCOA</a></td>\\n\\t<td class=\\\" \\\">John Lawson</td>\\n\\t<td class=\\\" \\\">Coordinate Covariation Analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">886</td>\\n</tr><tr id=\\\"pkg_CSSP\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/CSSP.html\\\">CSSP</a></td>\\n\\t<td class=\\\" \\\">Chandler Zuo</td>\\n\\t<td class=\\\" \\\">ChIP-Seq Statistical Power</td>\\n\\t<td class=\\\"  sorting_1\\\">886</td>\\n</tr><tr id=\\\"pkg_DegNorm\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/DegNorm.html\\\">DegNorm</a></td>\\n\\t<td class=\\\" \\\">Ji-Ping Wang</td>\\n\\t<td class=\\\" \\\">DegNorm: degradation normalization for RNA-seq data</td>\\n\\t<td class=\\\"  sorting_1\\\">886</td>\\n</tr><tr id=\\\"pkg_DelayedRandomArray\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/DelayedRandomArray.html\\\">DelayedRandomArray</a></td>\\n\\t<td class=\\\" \\\">Aaron Lun</td>\\n\\t<td class=\\\" \\\">Delayed Arrays of Random Values</td>\\n\\t<td class=\\\"  sorting_1\\\">886</td>\\n</tr><tr id=\\\"pkg_DEsubs\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/DEsubs.html\\\">DEsubs</a></td>\\n\\t<td class=\\\" \\\">Aristidis G. Vrahatis, Panos Balomenos</td>\\n\\t<td class=\\\" \\\">DEsubs: an R package for flexible identification of differentially expressed subpathways using RNA-seq expression experiments</td>\\n\\t<td class=\\\"  sorting_1\\\">886</td>\\n</tr><tr id=\\\"pkg_DIAlignR\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/DIAlignR.html\\\">DIAlignR</a></td>\\n\\t<td class=\\\" \\\">Shubham Gupta</td>\\n\\t<td class=\\\" \\\">Dynamic Programming Based Alignment of MS2 Chromatograms</td>\\n\\t<td class=\\\"  sorting_1\\\">886</td>\\n</tr><tr id=\\\"pkg_fdrame\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/fdrame.html\\\">fdrame</a></td>\\n\\t<td class=\\\" \\\">Effi Kenigsberg</td>\\n\\t<td class=\\\" \\\">FDR adjustments of Microarray Experiments (FDR-AME)</td>\\n\\t<td class=\\\"  sorting_1\\\">886</td>\\n</tr><tr id=\\\"pkg_frmaTools\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/frmaTools.html\\\">frmaTools</a></td>\\n\\t<td class=\\\" \\\">Matthew N. McCall</td>\\n\\t<td class=\\\" \\\">Frozen RMA Tools</td>\\n\\t<td class=\\\"  sorting_1\\\">886</td>\\n</tr><tr id=\\\"pkg_GA4GHclient\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/GA4GHclient.html\\\">GA4GHclient</a></td>\\n\\t<td class=\\\" \\\">Welliton Souza</td>\\n\\t<td class=\\\" \\\">A Bioconductor package for accessing GA4GH API data servers</td>\\n\\t<td class=\\\"  sorting_1\\\">886</td>\\n</tr><tr id=\\\"pkg_GCSscore\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/GCSscore.html\\\">GCSscore</a></td>\\n\\t<td class=\\\" \\\">Guy M. Harris</td>\\n\\t<td class=\\\" \\\">GCSscore: an R package for microarray analysis for Affymetrix/Thermo Fisher arrays</td>\\n\\t<td class=\\\"  sorting_1\\\">886</td>\\n</tr><tr id=\\\"pkg_GGPA\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/GGPA.html\\\">GGPA</a></td>\\n\\t<td class=\\\" \\\">Dongjun Chung</td>\\n\\t<td class=\\\" \\\">graph-GPA: A graphical model for prioritizing GWAS results and investigating pleiotropic architecture</td>\\n\\t<td class=\\\"  sorting_1\\\">886</td>\\n</tr><tr id=\\\"pkg_GRridge\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/GRridge.html\\\">GRridge</a></td>\\n\\t<td class=\\\" \\\">Mark A. van de Wiel</td>\\n\\t<td class=\\\" \\\">Better prediction by use of co-data: Adaptive group-regularized ridge regression</td>\\n\\t<td class=\\\"  sorting_1\\\">886</td>\\n</tr><tr id=\\\"pkg_gscreend\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/gscreend.html\\\">gscreend</a></td>\\n\\t<td class=\\\" \\\">Katharina Imkeller</td>\\n\\t<td class=\\\" \\\">Analysis of pooled genetic screens</td>\\n\\t<td class=\\\"  sorting_1\\\">886</td>\\n</tr><tr id=\\\"pkg_HEM\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/HEM.html\\\">HEM</a></td>\\n\\t<td class=\\\" \\\">HyungJun Cho</td>\\n\\t<td class=\\\" \\\">Heterogeneous error model for identification of differentially expressed genes under multiple conditions</td>\\n\\t<td class=\\\"  sorting_1\\\">886</td>\\n</tr><tr id=\\\"pkg_idiogram\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/idiogram.html\\\">idiogram</a></td>\\n\\t<td class=\\\" \\\">Karl J. Dykema</td>\\n\\t<td class=\\\" \\\">idiogram</td>\\n\\t<td class=\\\"  sorting_1\\\">886</td>\\n</tr><tr id=\\\"pkg_IntEREst\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/IntEREst.html\\\">IntEREst</a></td>\\n\\t<td class=\\\" \\\">Ali Oghabian, Mikko Frilander</td>\\n\\t<td class=\\\" \\\">Intron-Exon Retention Estimator</td>\\n\\t<td class=\\\"  sorting_1\\\">886</td>\\n</tr><tr id=\\\"pkg_iPAC\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/iPAC.html\\\">iPAC</a></td>\\n\\t<td class=\\\" \\\">Gregory Ryslik</td>\\n\\t<td class=\\\" \\\">Identification of Protein Amino acid Clustering</td>\\n\\t<td class=\\\"  sorting_1\\\">886</td>\\n</tr><tr id=\\\"pkg_martini\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/martini.html\\\">martini</a></td>\\n\\t<td class=\\\" \\\">Hector Climente-Gonzalez</td>\\n\\t<td class=\\\" \\\">GWAS Incorporating Networks</td>\\n\\t<td class=\\\"  sorting_1\\\">886</td>\\n</tr><tr id=\\\"pkg_mdqc\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/mdqc.html\\\">mdqc</a></td>\\n\\t<td class=\\\" \\\">Gabriela Cohen-Freue</td>\\n\\t<td class=\\\" \\\">Mahalanobis Distance Quality Control for microarrays</td>\\n\\t<td class=\\\"  sorting_1\\\">886</td>\\n</tr><tr id=\\\"pkg_Metab\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/Metab.html\\\">Metab</a></td>\\n\\t<td class=\\\" \\\">Raphael Aggio</td>\\n\\t<td class=\\\" \\\">Metab: An R Package for a High-Throughput Analysis of Metabolomics Data Generated by GC-MS.</td>\\n\\t<td class=\\\"  sorting_1\\\">886</td>\\n</tr><tr id=\\\"pkg_MGFR\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/MGFR.html\\\">MGFR</a></td>\\n\\t<td class=\\\" \\\">Khadija El Amrani</td>\\n\\t<td class=\\\" \\\">Marker Gene Finder in RNA-seq data</td>\\n\\t<td class=\\\"  sorting_1\\\">886</td>\\n</tr><tr id=\\\"pkg_MIMOSA\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/MIMOSA.html\\\">MIMOSA</a></td>\\n\\t<td class=\\\" \\\">Greg Finak</td>\\n\\t<td class=\\\" \\\">Mixture Models for Single-Cell Assays</td>\\n\\t<td class=\\\"  sorting_1\\\">886</td>\\n</tr><tr id=\\\"pkg_MPRAnalyze\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/MPRAnalyze.html\\\">MPRAnalyze</a></td>\\n\\t<td class=\\\" \\\">Tal Ashuach</td>\\n\\t<td class=\\\" \\\">Statistical Analysis of MPRA data</td>\\n\\t<td class=\\\"  sorting_1\\\">886</td>\\n</tr><tr id=\\\"pkg_nethet\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/nethet.html\\\">nethet</a></td>\\n\\t<td class=\\\" \\\">Nicolas Staedler, Frank Dondelinger</td>\\n\\t<td class=\\\" \\\">A bioconductor package for high-dimensional exploration of biological network heterogeneity</td>\\n\\t<td class=\\\"  sorting_1\\\">886</td>\\n</tr><tr id=\\\"pkg_NoRCE\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/NoRCE.html\\\">NoRCE</a></td>\\n\\t<td class=\\\" \\\">Gulden Olgun</td>\\n\\t<td class=\\\" \\\">NoRCE: Noncoding RNA Sets Cis Annotation and Enrichment</td>\\n\\t<td class=\\\"  sorting_1\\\">886</td>\\n</tr><tr id=\\\"pkg_NuPoP\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/NuPoP.html\\\">NuPoP</a></td>\\n\\t<td class=\\\" \\\">Ji-Ping Wang</td>\\n\\t<td class=\\\" \\\">An R package for nucleosome positioning prediction</td>\\n\\t<td class=\\\"  sorting_1\\\">886</td>\\n</tr><tr id=\\\"pkg_PathNet\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/PathNet.html\\\">PathNet</a></td>\\n\\t<td class=\\\" \\\">Ludwig Geistlinger</td>\\n\\t<td class=\\\" \\\">An R package for pathway analysis using topological information</td>\\n\\t<td class=\\\"  sorting_1\\\">886</td>\\n</tr><tr id=\\\"pkg_PhIPData\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/PhIPData.html\\\">PhIPData</a></td>\\n\\t<td class=\\\" \\\">Athena Chen</td>\\n\\t<td class=\\\" \\\">Container for PhIP-Seq Experiments</td>\\n\\t<td class=\\\"  sorting_1\\\">886</td>\\n</tr><tr id=\\\"pkg_pwOmics\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/pwOmics.html\\\">pwOmics</a></td>\\n\\t<td class=\\\" \\\">Maren Sitte</td>\\n\\t<td class=\\\" \\\">Pathway-based data integration of omics data</td>\\n\\t<td class=\\\"  sorting_1\\\">886</td>\\n</tr><tr id=\\\"pkg_QuartPAC\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/QuartPAC.html\\\">QuartPAC</a></td>\\n\\t<td class=\\\" \\\">Gregory Ryslik</td>\\n\\t<td class=\\\" \\\">Identification of mutational clusters in protein quaternary structures.</td>\\n\\t<td class=\\\"  sorting_1\\\">886</td>\\n</tr><tr id=\\\"pkg_RITAN\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/RITAN.html\\\">RITAN</a></td>\\n\\t<td class=\\\" \\\">Michael Zimmermann</td>\\n\\t<td class=\\\" \\\">Rapid Integration of Term Annotation and Network resources</td>\\n\\t<td class=\\\"  sorting_1\\\">886</td>\\n</tr><tr id=\\\"pkg_Rtreemix\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/Rtreemix.html\\\">Rtreemix</a></td>\\n\\t<td class=\\\" \\\">Jasmina Bogojeska</td>\\n\\t<td class=\\\" \\\">Rtreemix: Mutagenetic trees mixture models.</td>\\n\\t<td class=\\\"  sorting_1\\\">886</td>\\n</tr><tr id=\\\"pkg_SLqPCR\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/SLqPCR.html\\\">SLqPCR</a></td>\\n\\t<td class=\\\" \\\">Matthias Kohl</td>\\n\\t<td class=\\\" \\\">Functions for analysis of real-time quantitative PCR data at SIRS-Lab GmbH</td>\\n\\t<td class=\\\"  sorting_1\\\">886</td>\\n</tr><tr id=\\\"pkg_SynExtend\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/SynExtend.html\\\">SynExtend</a></td>\\n\\t<td class=\\\" \\\">Nicholas Cooley</td>\\n\\t<td class=\\\" \\\">Tools for Working With Synteny Objects</td>\\n\\t<td class=\\\"  sorting_1\\\">886</td>\\n</tr><tr id=\\\"pkg_tanggle\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/tanggle.html\\\">tanggle</a></td>\\n\\t<td class=\\\" \\\">Klaus Schliep</td>\\n\\t<td class=\\\" \\\">Visualization of Phylogenetic Networks</td>\\n\\t<td class=\\\"  sorting_1\\\">886</td>\\n</tr><tr id=\\\"pkg_TNBC.CMS\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/TNBC.CMS.html\\\">TNBC.CMS</a></td>\\n\\t<td class=\\\" \\\">Doyeong Yu</td>\\n\\t<td class=\\\" \\\">TNBC.CMS: Prediction of TNBC Consensus Molecular Subtypes</td>\\n\\t<td class=\\\"  sorting_1\\\">886</td>\\n</tr><tr id=\\\"pkg_transcriptR\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/transcriptR.html\\\">transcriptR</a></td>\\n\\t<td class=\\\" \\\">Armen R. Karapetyan</td>\\n\\t<td class=\\\" \\\">An Integrative Tool for ChIP- And RNA-Seq Based Primary Transcripts Detection and Quantification</td>\\n\\t<td class=\\\"  sorting_1\\\">886</td>\\n</tr><tr id=\\\"pkg_tRNAdbImport\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/tRNAdbImport.html\\\">tRNAdbImport</a></td>\\n\\t<td class=\\\" \\\">Felix G.M. Ernst</td>\\n\\t<td class=\\\" \\\">Importing from tRNAdb and mitotRNAdb as GRanges objects</td>\\n\\t<td class=\\\"  sorting_1\\\">886</td>\\n</tr><tr id=\\\"pkg_wpm\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/wpm.html\\\">wpm</a></td>\\n\\t<td class=\\\" \\\">Helene Borges</td>\\n\\t<td class=\\\" \\\">Well Plate Maker</td>\\n\\t<td class=\\\"  sorting_1\\\">886</td>\\n</tr><tr id=\\\"pkg_XDE\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/XDE.html\\\">XDE</a></td>\\n\\t<td class=\\\" \\\">Robert Scharpf</td>\\n\\t<td class=\\\" \\\">XDE: a Bayesian hierarchical model for cross-study analysis of differential gene expression</td>\\n\\t<td class=\\\"  sorting_1\\\">886</td>\\n</tr><tr id=\\\"pkg_adductomicsR\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/adductomicsR.html\\\">adductomicsR</a></td>\\n\\t<td class=\\\" \\\">Josie Hayes</td>\\n\\t<td class=\\\" \\\">Processing of adductomic mass spectral datasets</td>\\n\\t<td class=\\\"  sorting_1\\\">940</td>\\n</tr><tr id=\\\"pkg_attract\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/attract.html\\\">attract</a></td>\\n\\t<td class=\\\" \\\">Samuel Zimmerman</td>\\n\\t<td class=\\\" \\\">Methods to Find the Gene Expression Modules that Represent the Drivers of Kauffman's Attractor Landscape</td>\\n\\t<td class=\\\"  sorting_1\\\">940</td>\\n</tr><tr id=\\\"pkg_BadRegionFinder\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/BadRegionFinder.html\\\">BadRegionFinder</a></td>\\n\\t<td class=\\\" \\\">Sarah Sandmann</td>\\n\\t<td class=\\\" \\\">BadRegionFinder: an R/Bioconductor package for identifying regions with bad coverage</td>\\n\\t<td class=\\\"  sorting_1\\\">940</td>\\n</tr><tr id=\\\"pkg_BAGS\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/BAGS.html\\\">BAGS</a></td>\\n\\t<td class=\\\" \\\">Alejandro Quiroz-Zarate</td>\\n\\t<td class=\\\" \\\">A Bayesian Approach for Geneset Selection</td>\\n\\t<td class=\\\"  sorting_1\\\">940</td>\\n</tr><tr id=\\\"pkg_BgeeCall\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/BgeeCall.html\\\">BgeeCall</a></td>\\n\\t<td class=\\\" \\\">Julien Wollbrett</td>\\n\\t<td class=\\\" \\\">Automatic RNA-Seq present/absent gene expression calls generation</td>\\n\\t<td class=\\\"  sorting_1\\\">940</td>\\n</tr><tr id=\\\"pkg_BiGGR\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/BiGGR.html\\\">BiGGR</a></td>\\n\\t<td class=\\\" \\\">Anand K. Gavai, Hannes Hettling</td>\\n\\t<td class=\\\" \\\">Constraint based modeling in R using metabolic reconstruction databases</td>\\n\\t<td class=\\\"  sorting_1\\\">940</td>\\n</tr><tr id=\\\"pkg_blacksheepr\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/blacksheepr.html\\\">blacksheepr</a></td>\\n\\t<td class=\\\" \\\">RugglesLab</td>\\n\\t<td class=\\\" \\\">Outlier Analysis for pairwise differential comparison</td>\\n\\t<td class=\\\"  sorting_1\\\">940</td>\\n</tr><tr id=\\\"pkg_BLMA\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/BLMA.html\\\">BLMA</a></td>\\n\\t<td class=\\\" \\\">Hung Nguyen</td>\\n\\t<td class=\\\" \\\">BLMA: A package for bi-level meta-analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">940</td>\\n</tr><tr id=\\\"pkg_BubbleTree\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/BubbleTree.html\\\">BubbleTree</a></td>\\n\\t<td class=\\\" \\\">Todd Creasy, Wei Zhu</td>\\n\\t<td class=\\\" \\\">BubbleTree: an intuitive visualization to elucidate tumoral aneuploidy and clonality in somatic mosaicism using next generation sequencing data</td>\\n\\t<td class=\\\"  sorting_1\\\">940</td>\\n</tr><tr id=\\\"pkg_BUS\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/BUS.html\\\">BUS</a></td>\\n\\t<td class=\\\" \\\">Yuanhua Liu</td>\\n\\t<td class=\\\" \\\">Gene network reconstruction</td>\\n\\t<td class=\\\"  sorting_1\\\">940</td>\\n</tr><tr id=\\\"pkg_cellity\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/cellity.html\\\">cellity</a></td>\\n\\t<td class=\\\" \\\">Tomislav Ilicic</td>\\n\\t<td class=\\\" \\\">Quality Control for Single-Cell RNA-seq Data</td>\\n\\t<td class=\\\"  sorting_1\\\">940</td>\\n</tr><tr id=\\\"pkg_ChIPComp\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ChIPComp.html\\\">ChIPComp</a></td>\\n\\t<td class=\\\" \\\">Li Chen</td>\\n\\t<td class=\\\" \\\">Quantitative comparison of multiple ChIP-seq datasets</td>\\n\\t<td class=\\\"  sorting_1\\\">940</td>\\n</tr><tr id=\\\"pkg_coGPS\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/coGPS.html\\\">coGPS</a></td>\\n\\t<td class=\\\" \\\">Yingying Wei</td>\\n\\t<td class=\\\" \\\">cancer outlier Gene Profile Sets</td>\\n\\t<td class=\\\"  sorting_1\\\">940</td>\\n</tr><tr id=\\\"pkg_COHCAP\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/COHCAP.html\\\">COHCAP</a></td>\\n\\t<td class=\\\" \\\">Charles Warden</td>\\n\\t<td class=\\\" \\\">CpG Island Analysis Pipeline for Illumina Methylation Array and Targeted BS-Seq Data</td>\\n\\t<td class=\\\"  sorting_1\\\">940</td>\\n</tr><tr id=\\\"pkg_ctsGE\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ctsGE.html\\\">ctsGE</a></td>\\n\\t<td class=\\\" \\\">Michal Sharabi-Schwager</td>\\n\\t<td class=\\\" \\\">Clustering of Time Series Gene Expression data</td>\\n\\t<td class=\\\"  sorting_1\\\">940</td>\\n</tr><tr id=\\\"pkg_EpiTxDb\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/EpiTxDb.html\\\">EpiTxDb</a></td>\\n\\t<td class=\\\" \\\">Felix G.M. Ernst</td>\\n\\t<td class=\\\" \\\">Storing and accessing epitranscriptomic information using the AnnotationDbi interface</td>\\n\\t<td class=\\\"  sorting_1\\\">940</td>\\n</tr><tr id=\\\"pkg_gcapc\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/gcapc.html\\\">gcapc</a></td>\\n\\t<td class=\\\" \\\">Mingxiang Teng</td>\\n\\t<td class=\\\" \\\">GC Aware Peak Caller</td>\\n\\t<td class=\\\"  sorting_1\\\">940</td>\\n</tr><tr id=\\\"pkg_GenomicSuperSignature\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/GenomicSuperSignature.html\\\">GenomicSuperSignature</a></td>\\n\\t<td class=\\\" \\\">Sehyun Oh</td>\\n\\t<td class=\\\" \\\">Interpretation of RNA-seq experiments through robust, efficient comparison to public databases</td>\\n\\t<td class=\\\"  sorting_1\\\">940</td>\\n</tr><tr id=\\\"pkg_GmicR\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/GmicR.html\\\">GmicR</a></td>\\n\\t<td class=\\\" \\\">Richard Virgen-Slane</td>\\n\\t<td class=\\\" \\\">Combines WGCNA and xCell readouts with bayesian network learrning to generate a Gene-Module Immune-Cell network (GMIC)</td>\\n\\t<td class=\\\"  sorting_1\\\">940</td>\\n</tr><tr id=\\\"pkg_GOpro\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/GOpro.html\\\">GOpro</a></td>\\n\\t<td class=\\\" \\\">Lidia Chrabaszcz</td>\\n\\t<td class=\\\" \\\">Find the most characteristic gene ontology terms for groups of human genes</td>\\n\\t<td class=\\\"  sorting_1\\\">940</td>\\n</tr><tr id=\\\"pkg_goSTAG\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/goSTAG.html\\\">goSTAG</a></td>\\n\\t<td class=\\\" \\\">Brian D. Bennett</td>\\n\\t<td class=\\\" \\\">A tool to use GO Subtrees to Tag and Annotate Genes within a set</td>\\n\\t<td class=\\\"  sorting_1\\\">940</td>\\n</tr><tr id=\\\"pkg_idr2d\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/idr2d.html\\\">idr2d</a></td>\\n\\t<td class=\\\" \\\">Konstantin Krismer</td>\\n\\t<td class=\\\" \\\">Irreproducible Discovery Rate for Genomic Interactions Data</td>\\n\\t<td class=\\\"  sorting_1\\\">940</td>\\n</tr><tr id=\\\"pkg_lmdme\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/lmdme.html\\\">lmdme</a></td>\\n\\t<td class=\\\" \\\">Cristobal Fresno</td>\\n\\t<td class=\\\" \\\">Linear Model decomposition for Designed Multivariate Experiments</td>\\n\\t<td class=\\\"  sorting_1\\\">940</td>\\n</tr><tr id=\\\"pkg_LRBaseDbi\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/LRBaseDbi.html\\\">LRBaseDbi</a></td>\\n\\t<td class=\\\" \\\">Koki Tsuyuzaki</td>\\n\\t<td class=\\\" \\\">DBI to construct LRBase-related package</td>\\n\\t<td class=\\\"  sorting_1\\\">940</td>\\n</tr><tr id=\\\"pkg_metapone\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/metapone.html\\\">metapone</a></td>\\n\\t<td class=\\\" \\\">Tianwei Yu</td>\\n\\t<td class=\\\" \\\">Conducts pathway test of metabolomics data using a weighted permutation test</td>\\n\\t<td class=\\\"  sorting_1\\\">940</td>\\n</tr><tr id=\\\"pkg_methylInheritance\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/methylInheritance.html\\\">methylInheritance</a></td>\\n\\t<td class=\\\" \\\">Astrid Deschnes</td>\\n\\t<td class=\\\" \\\">Permutation-Based Analysis associating Conserved Differentially Methylated Elements Across Multiple Generations to a Treatment Effect</td>\\n\\t<td class=\\\"  sorting_1\\\">940</td>\\n</tr><tr id=\\\"pkg_miRLAB\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/miRLAB.html\\\">miRLAB</a></td>\\n\\t<td class=\\\" \\\">Thuc Duy Le</td>\\n\\t<td class=\\\" \\\">Dry lab for exploring miRNA-mRNA relationships</td>\\n\\t<td class=\\\"  sorting_1\\\">940</td>\\n</tr><tr id=\\\"pkg_MSPrep\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/MSPrep.html\\\">MSPrep</a></td>\\n\\t<td class=\\\" \\\">Max McGrath</td>\\n\\t<td class=\\\" \\\">Package for Summarizing, Filtering, Imputing, and Normalizing Metabolomics Data</td>\\n\\t<td class=\\\"  sorting_1\\\">940</td>\\n</tr><tr id=\\\"pkg_MVCClass\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/MVCClass.html\\\">MVCClass</a></td>\\n\\t<td class=\\\" \\\">Elizabeth Whalen</td>\\n\\t<td class=\\\" \\\">Model-View-Controller (MVC) Classes</td>\\n\\t<td class=\\\"  sorting_1\\\">940</td>\\n</tr><tr id=\\\"pkg_NCIgraph\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/NCIgraph.html\\\">NCIgraph</a></td>\\n\\t<td class=\\\" \\\">Laurent Jacob</td>\\n\\t<td class=\\\" \\\">Pathways from the NCI Pathways Database</td>\\n\\t<td class=\\\"  sorting_1\\\">940</td>\\n</tr><tr id=\\\"pkg_OLIN\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/OLIN.html\\\">OLIN</a></td>\\n\\t<td class=\\\" \\\">Matthias Futschik</td>\\n\\t<td class=\\\" \\\">Optimized local intensity-dependent normalisation of two-color microarrays</td>\\n\\t<td class=\\\"  sorting_1\\\">940</td>\\n</tr><tr id=\\\"pkg_PCAN\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/PCAN.html\\\">PCAN</a></td>\\n\\t<td class=\\\" \\\">Matthew Page and Patrice Godard</td>\\n\\t<td class=\\\" \\\">Phenotype Consensus ANalysis (PCAN)</td>\\n\\t<td class=\\\"  sorting_1\\\">940</td>\\n</tr><tr id=\\\"pkg_podkat\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/podkat.html\\\">podkat</a></td>\\n\\t<td class=\\\" \\\">Ulrich Bodenhofer</td>\\n\\t<td class=\\\" \\\">Position-Dependent Kernel Association Test</td>\\n\\t<td class=\\\"  sorting_1\\\">940</td>\\n</tr><tr id=\\\"pkg_R453Plus1Toolbox\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/R453Plus1Toolbox.html\\\">R453Plus1Toolbox</a></td>\\n\\t<td class=\\\" \\\">Hans-Ulrich Klein</td>\\n\\t<td class=\\\" \\\">A package for importing and analyzing data from Roche's Genome Sequencer System</td>\\n\\t<td class=\\\"  sorting_1\\\">940</td>\\n</tr><tr id=\\\"pkg_RBM\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/RBM.html\\\">RBM</a></td>\\n\\t<td class=\\\" \\\">Dongmei Li</td>\\n\\t<td class=\\\" \\\">RBM: a R package for microarray and RNA-Seq data analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">940</td>\\n</tr><tr id=\\\"pkg_RCyjs\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/RCyjs.html\\\">RCyjs</a></td>\\n\\t<td class=\\\" \\\">Paul Shannon</td>\\n\\t<td class=\\\" \\\">Display and manipulate graphs in cytoscape.js</td>\\n\\t<td class=\\\"  sorting_1\\\">940</td>\\n</tr><tr id=\\\"pkg_rnaseqcomp\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/rnaseqcomp.html\\\">rnaseqcomp</a></td>\\n\\t<td class=\\\" \\\">Mingxiang Teng</td>\\n\\t<td class=\\\" \\\">Benchmarks for RNA-seq Quantification Pipelines</td>\\n\\t<td class=\\\"  sorting_1\\\">940</td>\\n</tr><tr id=\\\"pkg_RSVSim\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/RSVSim.html\\\">RSVSim</a></td>\\n\\t<td class=\\\" \\\">Christoph Bartenhagen</td>\\n\\t<td class=\\\" \\\">RSVSim: an R/Bioconductor package for the simulation of structural variations</td>\\n\\t<td class=\\\"  sorting_1\\\">940</td>\\n</tr><tr id=\\\"pkg_RUVnormalize\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/RUVnormalize.html\\\">RUVnormalize</a></td>\\n\\t<td class=\\\" \\\">Laurent Jacob</td>\\n\\t<td class=\\\" \\\">RUV for normalization of expression array data</td>\\n\\t<td class=\\\"  sorting_1\\\">940</td>\\n</tr><tr id=\\\"pkg_sights\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/sights.html\\\">sights</a></td>\\n\\t<td class=\\\" \\\">Elika Garg</td>\\n\\t<td class=\\\" \\\">Statistics and dIagnostic Graphs for HTS</td>\\n\\t<td class=\\\"  sorting_1\\\">940</td>\\n</tr><tr id=\\\"pkg_STATegRa\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/STATegRa.html\\\">STATegRa</a></td>\\n\\t<td class=\\\" \\\">David Gomez-Cabrero, Nria Planell</td>\\n\\t<td class=\\\" \\\">Classes and methods for multi-omics data integration</td>\\n\\t<td class=\\\"  sorting_1\\\">940</td>\\n</tr><tr id=\\\"pkg_Streamer\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/Streamer.html\\\">Streamer</a></td>\\n\\t<td class=\\\" \\\">Martin Morgan</td>\\n\\t<td class=\\\" \\\">Enabling stream processing of large files</td>\\n\\t<td class=\\\"  sorting_1\\\">940</td>\\n</tr><tr id=\\\"pkg_SWATH2stats\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/SWATH2stats.html\\\">SWATH2stats</a></td>\\n\\t<td class=\\\" \\\">Peter Blattmann</td>\\n\\t<td class=\\\" \\\">Transform and Filter SWATH Data for Statistical Packages</td>\\n\\t<td class=\\\"  sorting_1\\\">940</td>\\n</tr><tr id=\\\"pkg_SwathXtend\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/SwathXtend.html\\\">SwathXtend</a></td>\\n\\t<td class=\\\" \\\">Jemma Wu</td>\\n\\t<td class=\\\" \\\">SWATH extended library generation and statistical data analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">940</td>\\n</tr><tr id=\\\"pkg_TBSignatureProfiler\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/TBSignatureProfiler.html\\\">TBSignatureProfiler</a></td>\\n\\t<td class=\\\" \\\">Aubrey R. Odom-Mabey</td>\\n\\t<td class=\\\" \\\">Profile RNA-Seq Data Using TB Pathway Signatures</td>\\n\\t<td class=\\\"  sorting_1\\\">940</td>\\n</tr><tr id=\\\"pkg_Trendy\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/Trendy.html\\\">Trendy</a></td>\\n\\t<td class=\\\" \\\">Rhonda Bacher</td>\\n\\t<td class=\\\" \\\">Breakpoint analysis of time-course expression data</td>\\n\\t<td class=\\\"  sorting_1\\\">940</td>\\n</tr><tr id=\\\"pkg_VariantFiltering\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/VariantFiltering.html\\\">VariantFiltering</a></td>\\n\\t<td class=\\\" \\\">Robert Castelo</td>\\n\\t<td class=\\\" \\\">Filtering of coding and non-coding genetic variants</td>\\n\\t<td class=\\\"  sorting_1\\\">940</td>\\n</tr><tr id=\\\"pkg_waddR\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/waddR.html\\\">waddR</a></td>\\n\\t<td class=\\\" \\\">Julian Flesch</td>\\n\\t<td class=\\\" \\\">Statistical tests for detecting differential distributions based on the 2-Wasserstein distance</td>\\n\\t<td class=\\\"  sorting_1\\\">940</td>\\n</tr><tr id=\\\"pkg_ADaCGH2\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ADaCGH2.html\\\">ADaCGH2</a></td>\\n\\t<td class=\\\" \\\">Ramon Diaz-Uriarte</td>\\n\\t<td class=\\\" \\\">Analysis of big data from aCGH experiments using parallel computing and ff objects</td>\\n\\t<td class=\\\"  sorting_1\\\">988</td>\\n</tr><tr id=\\\"pkg_ADAMgui\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ADAMgui.html\\\">ADAMgui</a></td>\\n\\t<td class=\\\" \\\">Jose Luiz Rybarczyk Filho</td>\\n\\t<td class=\\\" \\\">Activity and Diversity Analysis Module Graphical User Interface</td>\\n\\t<td class=\\\"  sorting_1\\\">988</td>\\n</tr><tr id=\\\"pkg_AGDEX\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/AGDEX.html\\\">AGDEX</a></td>\\n\\t<td class=\\\" \\\">Cuilan lani Gao</td>\\n\\t<td class=\\\" \\\">Agreement of Differential Expression Analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">988</td>\\n</tr><tr id=\\\"pkg_atena\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/atena.html\\\">atena</a></td>\\n\\t<td class=\\\" \\\">Beatriz Calvo-Serra</td>\\n\\t<td class=\\\" \\\">Analysis of Transposable Elements</td>\\n\\t<td class=\\\"  sorting_1\\\">988</td>\\n</tr><tr id=\\\"pkg_AWFisher\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/AWFisher.html\\\">AWFisher</a></td>\\n\\t<td class=\\\" \\\">Zhiguang Huo</td>\\n\\t<td class=\\\" \\\">An R package for fast computing for adaptively weighted fisher's method</td>\\n\\t<td class=\\\"  sorting_1\\\">988</td>\\n</tr><tr id=\\\"pkg_BaalChIP\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/BaalChIP.html\\\">BaalChIP</a></td>\\n\\t<td class=\\\" \\\">Ines de Santiago</td>\\n\\t<td class=\\\" \\\">BaalChIP: Bayesian analysis of allele-specific transcription factor binding in cancer genomes</td>\\n\\t<td class=\\\"  sorting_1\\\">988</td>\\n</tr><tr id=\\\"pkg_BAC\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/BAC.html\\\">BAC</a></td>\\n\\t<td class=\\\" \\\">Raphael Gottardo</td>\\n\\t<td class=\\\" \\\">Bayesian Analysis of Chip-chip experiment</td>\\n\\t<td class=\\\"  sorting_1\\\">988</td>\\n</tr><tr id=\\\"pkg_BasicSTARRseq\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/BasicSTARRseq.html\\\">BasicSTARRseq</a></td>\\n\\t<td class=\\\" \\\">Annika Buerger</td>\\n\\t<td class=\\\" \\\">Basic peak calling on STARR-seq data</td>\\n\\t<td class=\\\"  sorting_1\\\">988</td>\\n</tr><tr id=\\\"pkg_CellTrails\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/CellTrails.html\\\">CellTrails</a></td>\\n\\t<td class=\\\" \\\">Daniel Ellwanger</td>\\n\\t<td class=\\\" \\\">Reconstruction, visualization and analysis of branching trajectories</td>\\n\\t<td class=\\\"  sorting_1\\\">988</td>\\n</tr><tr id=\\\"pkg_ChromSCape\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ChromSCape.html\\\">ChromSCape</a></td>\\n\\t<td class=\\\" \\\">Pacome Prompsy</td>\\n\\t<td class=\\\" \\\">Analysis of single-cell epigenomics datasets with a Shiny App</td>\\n\\t<td class=\\\"  sorting_1\\\">988</td>\\n</tr><tr id=\\\"pkg_clstutils\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/clstutils.html\\\">clstutils</a></td>\\n\\t<td class=\\\" \\\">Noah Hoffman</td>\\n\\t<td class=\\\" \\\">Tools for performing taxonomic assignment</td>\\n\\t<td class=\\\"  sorting_1\\\">988</td>\\n</tr><tr id=\\\"pkg_daMA\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/daMA.html\\\">daMA</a></td>\\n\\t<td class=\\\" \\\">Jobst Landgrebe</td>\\n\\t<td class=\\\" \\\">Efficient design and analysis of factorial two-colour microarray data</td>\\n\\t<td class=\\\"  sorting_1\\\">988</td>\\n</tr><tr id=\\\"pkg_diggit\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/diggit.html\\\">diggit</a></td>\\n\\t<td class=\\\" \\\">Mariano J Alvarez</td>\\n\\t<td class=\\\" \\\">Inference of Genetic Variants Driving Cellular Phenotypes</td>\\n\\t<td class=\\\"  sorting_1\\\">988</td>\\n</tr><tr id=\\\"pkg_DriverNet\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/DriverNet.html\\\">DriverNet</a></td>\\n\\t<td class=\\\" \\\">Jiarui Ding</td>\\n\\t<td class=\\\" \\\">Drivernet: uncovering somatic driver mutations modulating transcriptional networks in cancer</td>\\n\\t<td class=\\\"  sorting_1\\\">988</td>\\n</tr><tr id=\\\"pkg_drugTargetInteractions\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/drugTargetInteractions.html\\\">drugTargetInteractions</a></td>\\n\\t<td class=\\\" \\\">Thomas Girke</td>\\n\\t<td class=\\\" \\\">Drug-Target Interactions</td>\\n\\t<td class=\\\"  sorting_1\\\">988</td>\\n</tr><tr id=\\\"pkg_flowSpecs\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/flowSpecs.html\\\">flowSpecs</a></td>\\n\\t<td class=\\\" \\\">Jakob Theorell</td>\\n\\t<td class=\\\" \\\">Tools for processing of high-dimensional cytometry data</td>\\n\\t<td class=\\\"  sorting_1\\\">988</td>\\n</tr><tr id=\\\"pkg_garfield\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/garfield.html\\\">garfield</a></td>\\n\\t<td class=\\\" \\\">Valentina Iotchkova</td>\\n\\t<td class=\\\" \\\">GWAS Analysis of Regulatory or Functional Information Enrichment with LD correction</td>\\n\\t<td class=\\\"  sorting_1\\\">988</td>\\n</tr><tr id=\\\"pkg_gCrisprTools\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/gCrisprTools.html\\\">gCrisprTools</a></td>\\n\\t<td class=\\\" \\\">Russell Bainer</td>\\n\\t<td class=\\\" \\\">Suite of Functions for Pooled Crispr Screen QC and Analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">988</td>\\n</tr><tr id=\\\"pkg_GeneSelectMMD\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/GeneSelectMMD.html\\\">GeneSelectMMD</a></td>\\n\\t<td class=\\\" \\\">Weiliang Qiu</td>\\n\\t<td class=\\\" \\\">Gene selection based on the marginal distributions of gene profiles that characterized by a mixture of three-component multivariate distributions</td>\\n\\t<td class=\\\"  sorting_1\\\">988</td>\\n</tr><tr id=\\\"pkg_genomes\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/genomes.html\\\">genomes</a></td>\\n\\t<td class=\\\" \\\">Chris Stubben</td>\\n\\t<td class=\\\" \\\">Genome sequencing project metadata</td>\\n\\t<td class=\\\"  sorting_1\\\">988</td>\\n</tr><tr id=\\\"pkg_GSEAmining\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/GSEAmining.html\\\">GSEAmining</a></td>\\n\\t<td class=\\\" \\\">Oriol Arqus</td>\\n\\t<td class=\\\" \\\">Make Biological Sense of Gene Set Enrichment Analysis Outputs</td>\\n\\t<td class=\\\"  sorting_1\\\">988</td>\\n</tr><tr id=\\\"pkg_IMMAN\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/IMMAN.html\\\">IMMAN</a></td>\\n\\t<td class=\\\" \\\">Minoo Ashtiani</td>\\n\\t<td class=\\\" \\\">Interlog protein network reconstruction by Mapping and Mining ANalysis</td>\\n\\t<td class=\\\"  sorting_1\\\">988</td>\\n</tr><tr id=\\\"pkg_INSPEcT\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/INSPEcT.html\\\">INSPEcT</a></td>\\n\\t<td class=\\\" \\\">Stefano de Pretis, Mattia Furlan</td>\\n\\t<td class=\\\" \\\">Modeling RNA synthesis, processing and degradation with RNA-seq data</td>\\n\\t<td class=\\\"  sorting_1\\\">988</td>\\n</tr><tr id=\\\"pkg_ISAnalytics\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ISAnalytics.html\\\">ISAnalytics</a></td>\\n\\t<td class=\\\" \\\">Andrea Calabria</td>\\n\\t<td class=\\\" \\\">Analyze gene therapy vector insertion sites data identified from genomics next generation sequencing reads for clonal tracking studies</td>\\n\\t<td class=\\\"  sorting_1\\\">988</td>\\n</tr><tr id=\\\"pkg_iterativeBMA\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/iterativeBMA.html\\\">iterativeBMA</a></td>\\n\\t<td class=\\\" \\\">Ka Yee Yeung</td>\\n\\t<td class=\\\" \\\">The Iterative Bayesian Model Averaging (BMA) algorithm</td>\\n\\t<td class=\\\"  sorting_1\\\">988</td>\\n</tr><tr id=\\\"pkg_KEGGlincs\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/KEGGlincs.html\\\">KEGGlincs</a></td>\\n\\t<td class=\\\" \\\">Shana White, Mario Medvedovic</td>\\n\\t<td class=\\\" \\\">Visualize all edges within a KEGG pathway and overlay LINCS data</td>\\n\\t<td class=\\\"  sorting_1\\\">988</td>\\n</tr><tr id=\\\"pkg_ldblock\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ldblock.html\\\">ldblock</a></td>\\n\\t<td class=\\\" \\\">VJ Carey</td>\\n\\t<td class=\\\" \\\">data structures for linkage disequilibrium measures in populations</td>\\n\\t<td class=\\\"  sorting_1\\\">988</td>\\n</tr><tr id=\\\"pkg_lionessR\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/lionessR.html\\\">lionessR</a></td>\\n\\t<td class=\\\" \\\">Ping-Han Hsieh</td>\\n\\t<td class=\\\" \\\">Modeling networks for individual samples using LIONESS</td>\\n\\t<td class=\\\"  sorting_1\\\">988</td>\\n</tr><tr id=\\\"pkg_maigesPack\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/maigesPack.html\\\">maigesPack</a></td>\\n\\t<td class=\\\" \\\">Gustavo H. Esteves</td>\\n\\t<td class=\\\" \\\">Functions to handle cDNA microarray data, including several methods of data analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">988</td>\\n</tr><tr id=\\\"pkg_miRNApath\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/miRNApath.html\\\">miRNApath</a></td>\\n\\t<td class=\\\" \\\">James M. Ward</td>\\n\\t<td class=\\\" \\\">miRNApath: Pathway Enrichment for miRNA Expression Data</td>\\n\\t<td class=\\\"  sorting_1\\\">988</td>\\n</tr><tr id=\\\"pkg_MODA\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/MODA.html\\\">MODA</a></td>\\n\\t<td class=\\\" \\\">Dong Li</td>\\n\\t<td class=\\\" \\\">MODA: MOdule Differential Analysis for weighted gene co-expression network</td>\\n\\t<td class=\\\"  sorting_1\\\">988</td>\\n</tr><tr id=\\\"pkg_ompBAM\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ompBAM.html\\\">ompBAM</a></td>\\n\\t<td class=\\\" \\\">Alex Chit Hei Wong</td>\\n\\t<td class=\\\" \\\">C++ Library for OpenMP-based multi-threaded sequential profiling of Binary Alignment Map (BAM) files</td>\\n\\t<td class=\\\"  sorting_1\\\">988</td>\\n</tr><tr id=\\\"pkg_openPrimeRui\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/openPrimeRui.html\\\">openPrimeRui</a></td>\\n\\t<td class=\\\" \\\">Matthias Dring</td>\\n\\t<td class=\\\" \\\">Shiny Application for Multiplex PCR Primer Design and Analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">988</td>\\n</tr><tr id=\\\"pkg_proActiv\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/proActiv.html\\\">proActiv</a></td>\\n\\t<td class=\\\" \\\">Joseph Lee</td>\\n\\t<td class=\\\" \\\">Estimate Promoter Activity from RNA-Seq data</td>\\n\\t<td class=\\\"  sorting_1\\\">988</td>\\n</tr><tr id=\\\"pkg_Risa\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/Risa.html\\\">Risa</a></td>\\n\\t<td class=\\\" \\\">Alejandra Gonzalez-Beltran</td>\\n\\t<td class=\\\" \\\">Converting experimental metadata from ISA-tab into Bioconductor data structures</td>\\n\\t<td class=\\\"  sorting_1\\\">988</td>\\n</tr><tr id=\\\"pkg_RTNduals\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/RTNduals.html\\\">RTNduals</a></td>\\n\\t<td class=\\\" \\\">Mauro Castro, Clarice Groeneveld</td>\\n\\t<td class=\\\" \\\">Analysis of co-regulation and inference of 'dual regulons'</td>\\n\\t<td class=\\\"  sorting_1\\\">988</td>\\n</tr><tr id=\\\"pkg_RUVcorr\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/RUVcorr.html\\\">RUVcorr</a></td>\\n\\t<td class=\\\" \\\">Saskia Freytag</td>\\n\\t<td class=\\\" \\\">Removal of unwanted variation for gene-gene correlations and related analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">988</td>\\n</tr><tr id=\\\"pkg_SCBN\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/SCBN.html\\\">SCBN</a></td>\\n\\t<td class=\\\" \\\">Yan Zhou</td>\\n\\t<td class=\\\" \\\">A statistical normalization method and differential expression analysis for RNA-seq data between different species</td>\\n\\t<td class=\\\"  sorting_1\\\">988</td>\\n</tr><tr id=\\\"pkg_similaRpeak\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/similaRpeak.html\\\">similaRpeak</a></td>\\n\\t<td class=\\\" \\\">Astrid Deschnes</td>\\n\\t<td class=\\\" \\\">Metrics to estimate a level of similarity between two ChIP-Seq profiles</td>\\n\\t<td class=\\\"  sorting_1\\\">988</td>\\n</tr><tr id=\\\"pkg_sizepower\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/sizepower.html\\\">sizepower</a></td>\\n\\t<td class=\\\" \\\">Weiliang Qiu</td>\\n\\t<td class=\\\" \\\">Sample Size and Power Calculation in Micorarray Studies</td>\\n\\t<td class=\\\"  sorting_1\\\">988</td>\\n</tr><tr id=\\\"pkg_SparseSignatures\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/SparseSignatures.html\\\">SparseSignatures</a></td>\\n\\t<td class=\\\" \\\">Luca De Sano</td>\\n\\t<td class=\\\" \\\">SparseSignatures</td>\\n\\t<td class=\\\"  sorting_1\\\">988</td>\\n</tr><tr id=\\\"pkg_STdeconvolve\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/STdeconvolve.html\\\">STdeconvolve</a></td>\\n\\t<td class=\\\" \\\">Brendan Miller</td>\\n\\t<td class=\\\" \\\">Reference-free Cell-Type Deconvolution of Multi-Cellular Spatially Resolved Transcriptomics Data</td>\\n\\t<td class=\\\"  sorting_1\\\">988</td>\\n</tr><tr id=\\\"pkg_survtype\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/survtype.html\\\">survtype</a></td>\\n\\t<td class=\\\" \\\">Dongmin Jung</td>\\n\\t<td class=\\\" \\\">Subtype Identification with Survival Data</td>\\n\\t<td class=\\\"  sorting_1\\\">988</td>\\n</tr><tr id=\\\"pkg_TAPseq\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/TAPseq.html\\\">TAPseq</a></td>\\n\\t<td class=\\\" \\\">Andreas R. Gschwind</td>\\n\\t<td class=\\\" \\\">Targeted scRNA-seq primer design for TAP-seq</td>\\n\\t<td class=\\\"  sorting_1\\\">988</td>\\n</tr><tr id=\\\"pkg_TargetSearch\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/TargetSearch.html\\\">TargetSearch</a></td>\\n\\t<td class=\\\" \\\">Alvaro Cuadros-Inostroza</td>\\n\\t<td class=\\\" \\\">A package for the analysis of GC-MS metabolite profiling data</td>\\n\\t<td class=\\\"  sorting_1\\\">988</td>\\n</tr><tr id=\\\"pkg_transcriptogramer\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/transcriptogramer.html\\\">transcriptogramer</a></td>\\n\\t<td class=\\\" \\\">Diego Morais</td>\\n\\t<td class=\\\" \\\">Transcriptional analysis based on transcriptograms</td>\\n\\t<td class=\\\"  sorting_1\\\">988</td>\\n</tr><tr id=\\\"pkg_AMARETTO\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/AMARETTO.html\\\">AMARETTO</a></td>\\n\\t<td class=\\\" \\\">Olivier Gevaert</td>\\n\\t<td class=\\\" \\\">Regulatory Network Inference and Driver Gene Evaluation using Integrative Multi-Omics Analysis and Penalized Regression</td>\\n\\t<td class=\\\"  sorting_1\\\">1034</td>\\n</tr><tr id=\\\"pkg_ANF\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ANF.html\\\">ANF</a></td>\\n\\t<td class=\\\" \\\">Tianle Ma</td>\\n\\t<td class=\\\" \\\">Affinity Network Fusion for Complex Patient Clustering</td>\\n\\t<td class=\\\"  sorting_1\\\">1034</td>\\n</tr><tr id=\\\"pkg_antiProfiles\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/antiProfiles.html\\\">antiProfiles</a></td>\\n\\t<td class=\\\" \\\">Hector Corrada Bravo</td>\\n\\t<td class=\\\" \\\">Implementation of gene expression anti-profiles</td>\\n\\t<td class=\\\"  sorting_1\\\">1034</td>\\n</tr><tr id=\\\"pkg_ASEB\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ASEB.html\\\">ASEB</a></td>\\n\\t<td class=\\\" \\\">Likun Wang</td>\\n\\t<td class=\\\" \\\">Predict Acetylated Lysine Sites</td>\\n\\t<td class=\\\"  sorting_1\\\">1034</td>\\n</tr><tr id=\\\"pkg_ASGSCA\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ASGSCA.html\\\">ASGSCA</a></td>\\n\\t<td class=\\\" \\\">Hela Romdhani</td>\\n\\t<td class=\\\" \\\">Association Studies for multiple SNPs and multiple traits using Generalized Structured Equation Models</td>\\n\\t<td class=\\\"  sorting_1\\\">1034</td>\\n</tr><tr id=\\\"pkg_Basic4Cseq\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/Basic4Cseq.html\\\">Basic4Cseq</a></td>\\n\\t<td class=\\\" \\\">Carolin Walter</td>\\n\\t<td class=\\\" \\\">Basic4Cseq: an R/Bioconductor package for analyzing 4C-seq data</td>\\n\\t<td class=\\\"  sorting_1\\\">1034</td>\\n</tr><tr id=\\\"pkg_BDMMAcorrect\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/BDMMAcorrect.html\\\">BDMMAcorrect</a></td>\\n\\t<td class=\\\" \\\">ZHENWEI DAI</td>\\n\\t<td class=\\\" \\\">Meta-analysis for the metagenomic read counts data from different cohorts</td>\\n\\t<td class=\\\"  sorting_1\\\">1034</td>\\n</tr><tr id=\\\"pkg_canceR\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/canceR.html\\\">canceR</a></td>\\n\\t<td class=\\\" \\\">Karim Mezhoud</td>\\n\\t<td class=\\\" \\\">A Graphical User Interface for accessing and modeling the Cancer Genomics Data of MSKCC</td>\\n\\t<td class=\\\"  sorting_1\\\">1034</td>\\n</tr><tr id=\\\"pkg_casper\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/casper.html\\\">casper</a></td>\\n\\t<td class=\\\" \\\">David Rossell</td>\\n\\t<td class=\\\" \\\">Characterization of Alternative Splicing based on Paired-End Reads</td>\\n\\t<td class=\\\"  sorting_1\\\">1034</td>\\n</tr><tr id=\\\"pkg_CellaRepertorium\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/CellaRepertorium.html\\\">CellaRepertorium</a></td>\\n\\t<td class=\\\" \\\">Andrew McDavid</td>\\n\\t<td class=\\\" \\\">Data structures, clustering and testing for single cell immune receptor repertoires (scRNAseq RepSeq/AIRR-seq)</td>\\n\\t<td class=\\\"  sorting_1\\\">1034</td>\\n</tr><tr id=\\\"pkg_CNORfeeder\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/CNORfeeder.html\\\">CNORfeeder</a></td>\\n\\t<td class=\\\" \\\">Attila Gabor</td>\\n\\t<td class=\\\" \\\">Integration of CellNOptR to add missing links</td>\\n\\t<td class=\\\"  sorting_1\\\">1034</td>\\n</tr><tr id=\\\"pkg_CNVfilteR\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/CNVfilteR.html\\\">CNVfilteR</a></td>\\n\\t<td class=\\\" \\\">Jose Marcos Moreno-Cabrera</td>\\n\\t<td class=\\\" \\\">Identifies false positives of CNV calling tools by using SNV calls</td>\\n\\t<td class=\\\"  sorting_1\\\">1034</td>\\n</tr><tr id=\\\"pkg_CORREP\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/CORREP.html\\\">CORREP</a></td>\\n\\t<td class=\\\" \\\">Dongxiao Zhu</td>\\n\\t<td class=\\\" \\\">Multivariate Correlation Estimator and Statistical Inference Procedures.</td>\\n\\t<td class=\\\"  sorting_1\\\">1034</td>\\n</tr><tr id=\\\"pkg_cTRAP\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/cTRAP.html\\\">cTRAP</a></td>\\n\\t<td class=\\\" \\\">Nuno Saraiva-Agostinho</td>\\n\\t<td class=\\\" \\\">Identification of candidate causal perturbations from differential gene expression data</td>\\n\\t<td class=\\\"  sorting_1\\\">1034</td>\\n</tr><tr id=\\\"pkg_CytoGLMM\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/CytoGLMM.html\\\">CytoGLMM</a></td>\\n\\t<td class=\\\" \\\">Christof Seiler</td>\\n\\t<td class=\\\" \\\">Conditional Differential Analysis for Flow and Mass Cytometry Experiments</td>\\n\\t<td class=\\\"  sorting_1\\\">1034</td>\\n</tr><tr id=\\\"pkg_DeepBlueR\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/DeepBlueR.html\\\">DeepBlueR</a></td>\\n\\t<td class=\\\" \\\">Felipe Albrecht, Markus List, Quirin Manz</td>\\n\\t<td class=\\\" \\\">DeepBlueR</td>\\n\\t<td class=\\\"  sorting_1\\\">1034</td>\\n</tr><tr id=\\\"pkg_DelayedDataFrame\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/DelayedDataFrame.html\\\">DelayedDataFrame</a></td>\\n\\t<td class=\\\" \\\">Qian Liu</td>\\n\\t<td class=\\\" \\\">Delayed operation on DataFrame using standard DataFrame metaphor</td>\\n\\t<td class=\\\"  sorting_1\\\">1034</td>\\n</tr><tr id=\\\"pkg_DEWSeq\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/DEWSeq.html\\\">DEWSeq</a></td>\\n\\t<td class=\\\" \\\">bioinformatics team Hentze</td>\\n\\t<td class=\\\" \\\">Differential Expressed Windows Based on Negative Binomial Distribution</td>\\n\\t<td class=\\\"  sorting_1\\\">1034</td>\\n</tr><tr id=\\\"pkg_epivizrStandalone\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/epivizrStandalone.html\\\">epivizrStandalone</a></td>\\n\\t<td class=\\\" \\\">Hector Corrada Bravo</td>\\n\\t<td class=\\\" \\\">Run Epiviz Interactive Genomic Data Visualization App within R</td>\\n\\t<td class=\\\"  sorting_1\\\">1034</td>\\n</tr><tr id=\\\"pkg_gcatest\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/gcatest.html\\\">gcatest</a></td>\\n\\t<td class=\\\" \\\">Wei Hao, John D. Storey</td>\\n\\t<td class=\\\" \\\">Genotype Conditional Association TEST</td>\\n\\t<td class=\\\"  sorting_1\\\">1034</td>\\n</tr><tr id=\\\"pkg_GeneExpressionSignature\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/GeneExpressionSignature.html\\\">GeneExpressionSignature</a></td>\\n\\t<td class=\\\" \\\">Yang Cao</td>\\n\\t<td class=\\\" \\\">Gene Expression Signature based Similarity Metric</td>\\n\\t<td class=\\\"  sorting_1\\\">1034</td>\\n</tr><tr id=\\\"pkg_getDEE2\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/getDEE2.html\\\">getDEE2</a></td>\\n\\t<td class=\\\" \\\">Mark Ziemann</td>\\n\\t<td class=\\\" \\\">Programmatic access to the DEE2 RNA expression dataset</td>\\n\\t<td class=\\\"  sorting_1\\\">1034</td>\\n</tr><tr id=\\\"pkg_GOTHiC\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/GOTHiC.html\\\">GOTHiC</a></td>\\n\\t<td class=\\\" \\\">Borbala Mifsud</td>\\n\\t<td class=\\\" \\\">Binomial test for Hi-C data analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">1034</td>\\n</tr><tr id=\\\"pkg_hermes\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/hermes.html\\\">hermes</a></td>\\n\\t<td class=\\\" \\\">Daniel Sabans Bov</td>\\n\\t<td class=\\\" \\\">Preprocessing, analyzing, and reporting of RNA-seq data</td>\\n\\t<td class=\\\"  sorting_1\\\">1034</td>\\n</tr><tr id=\\\"pkg_HGC\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/HGC.html\\\">HGC</a></td>\\n\\t<td class=\\\" \\\">XGlab</td>\\n\\t<td class=\\\" \\\">A fast hierarchical graph-based clustering method</td>\\n\\t<td class=\\\"  sorting_1\\\">1034</td>\\n</tr><tr id=\\\"pkg_hiReadsProcessor\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/hiReadsProcessor.html\\\">hiReadsProcessor</a></td>\\n\\t<td class=\\\" \\\">Nirav V Malani</td>\\n\\t<td class=\\\" \\\">Functions to process LM-PCR reads from 454/Illumina data</td>\\n\\t<td class=\\\"  sorting_1\\\">1034</td>\\n</tr><tr id=\\\"pkg_idpr\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/idpr.html\\\">idpr</a></td>\\n\\t<td class=\\\" \\\">William M. McFadden</td>\\n\\t<td class=\\\" \\\">Profiling and Analyzing Intrinsically Disordered Proteins in R</td>\\n\\t<td class=\\\"  sorting_1\\\">1034</td>\\n</tr><tr id=\\\"pkg_intansv\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/intansv.html\\\">intansv</a></td>\\n\\t<td class=\\\" \\\">Wen Yao</td>\\n\\t<td class=\\\" \\\">Integrative analysis of structural variations</td>\\n\\t<td class=\\\"  sorting_1\\\">1034</td>\\n</tr><tr id=\\\"pkg_KnowSeq\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/KnowSeq.html\\\">KnowSeq</a></td>\\n\\t<td class=\\\" \\\">Daniel Castillo-Secilla</td>\\n\\t<td class=\\\" \\\">KnowSeq R/Bioc package: The Smart Transcriptomic Pipeline</td>\\n\\t<td class=\\\"  sorting_1\\\">1034</td>\\n</tr><tr id=\\\"pkg_MetNet\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/MetNet.html\\\">MetNet</a></td>\\n\\t<td class=\\\" \\\">Thomas Naake</td>\\n\\t<td class=\\\" \\\">Inferring metabolic networks from untargeted high-resolution mass spectrometry data</td>\\n\\t<td class=\\\"  sorting_1\\\">1034</td>\\n</tr><tr id=\\\"pkg_miRNAmeConverter\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/miRNAmeConverter.html\\\">miRNAmeConverter</a></td>\\n\\t<td class=\\\" \\\">Stefan J. Haunsberger</td>\\n\\t<td class=\\\" \\\">Convert miRNA Names to Different miRBase Versions</td>\\n\\t<td class=\\\"  sorting_1\\\">1034</td>\\n</tr><tr id=\\\"pkg_MoonlightR\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/MoonlightR.html\\\">MoonlightR</a></td>\\n\\t<td class=\\\" \\\">Matteo Tiberti</td>\\n\\t<td class=\\\" \\\">Identify oncogenes and tumor suppressor genes from omics data</td>\\n\\t<td class=\\\"  sorting_1\\\">1034</td>\\n</tr><tr id=\\\"pkg_MultiBaC\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/MultiBaC.html\\\">MultiBaC</a></td>\\n\\t<td class=\\\" \\\">The package maintainer</td>\\n\\t<td class=\\\" \\\">Multiomic Batch effect Correction</td>\\n\\t<td class=\\\"  sorting_1\\\">1034</td>\\n</tr><tr id=\\\"pkg_nullranges\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/nullranges.html\\\">nullranges</a></td>\\n\\t<td class=\\\" \\\">Michael Love</td>\\n\\t<td class=\\\" \\\">Generation of null ranges via bootstrapping or covariate matching</td>\\n\\t<td class=\\\"  sorting_1\\\">1034</td>\\n</tr><tr id=\\\"pkg_OSAT\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/OSAT.html\\\">OSAT</a></td>\\n\\t<td class=\\\" \\\">Li Yan</td>\\n\\t<td class=\\\" \\\">OSAT: Optimal Sample Assignment Tool</td>\\n\\t<td class=\\\"  sorting_1\\\">1034</td>\\n</tr><tr id=\\\"pkg_parody\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/parody.html\\\">parody</a></td>\\n\\t<td class=\\\" \\\">Vince Carey</td>\\n\\t<td class=\\\" \\\">Parametric And Resistant Outlier DYtection</td>\\n\\t<td class=\\\"  sorting_1\\\">1034</td>\\n</tr><tr id=\\\"pkg_peakPantheR\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/peakPantheR.html\\\">peakPantheR</a></td>\\n\\t<td class=\\\" \\\">Arnaud Wolfer</td>\\n\\t<td class=\\\" \\\">Peak Picking and Annotation of High Resolution Experiments</td>\\n\\t<td class=\\\"  sorting_1\\\">1034</td>\\n</tr><tr id=\\\"pkg_Pviz\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/Pviz.html\\\">Pviz</a></td>\\n\\t<td class=\\\" \\\">Renan Sauteraud</td>\\n\\t<td class=\\\" \\\">Peptide Annotation and Data Visualization using Gviz</td>\\n\\t<td class=\\\"  sorting_1\\\">1034</td>\\n</tr><tr id=\\\"pkg_qPLEXanalyzer\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/qPLEXanalyzer.html\\\">qPLEXanalyzer</a></td>\\n\\t<td class=\\\" \\\">Ashley Sawle</td>\\n\\t<td class=\\\" \\\">Tools for quantitative proteomics data analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">1034</td>\\n</tr><tr id=\\\"pkg_roar\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/roar.html\\\">roar</a></td>\\n\\t<td class=\\\" \\\">Elena Grassi</td>\\n\\t<td class=\\\" \\\">Identify differential APA usage from RNA-seq alignments</td>\\n\\t<td class=\\\"  sorting_1\\\">1034</td>\\n</tr><tr id=\\\"pkg_SBMLR\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/SBMLR.html\\\">SBMLR</a></td>\\n\\t<td class=\\\" \\\">Tomas Radivoyevitch</td>\\n\\t<td class=\\\" \\\">SBML-R Interface and Analysis Tools</td>\\n\\t<td class=\\\"  sorting_1\\\">1034</td>\\n</tr><tr id=\\\"pkg_scTensor\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/scTensor.html\\\">scTensor</a></td>\\n\\t<td class=\\\" \\\">Koki Tsuyuzaki</td>\\n\\t<td class=\\\" \\\">Detection of cell-cell interaction from single-cell RNA-seq dataset by tensor decomposition</td>\\n\\t<td class=\\\"  sorting_1\\\">1034</td>\\n</tr><tr id=\\\"pkg_shinyepico\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/shinyepico.html\\\">shinyepico</a></td>\\n\\t<td class=\\\" \\\">Octavio Morante-Palacios</td>\\n\\t<td class=\\\" \\\">ShinyPICo</td>\\n\\t<td class=\\\"  sorting_1\\\">1034</td>\\n</tr><tr id=\\\"pkg_systemPipeShiny\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/systemPipeShiny.html\\\">systemPipeShiny</a></td>\\n\\t<td class=\\\" \\\">Le Zhang</td>\\n\\t<td class=\\\" \\\">systemPipeShiny: An Interactive Framework for Workflow Management and Visualization</td>\\n\\t<td class=\\\"  sorting_1\\\">1034</td>\\n</tr><tr id=\\\"pkg_aggregateBioVar\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/aggregateBioVar.html\\\">aggregateBioVar</a></td>\\n\\t<td class=\\\" \\\">Jason Ratcliff</td>\\n\\t<td class=\\\" \\\">Differential Gene Expression Analysis for Multi-subject scRNA-seq</td>\\n\\t<td class=\\\"  sorting_1\\\">1078</td>\\n</tr><tr id=\\\"pkg_Anaquin\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/Anaquin.html\\\">Anaquin</a></td>\\n\\t<td class=\\\" \\\">Ted Wong</td>\\n\\t<td class=\\\" \\\">Statistical analysis of sequins</td>\\n\\t<td class=\\\"  sorting_1\\\">1078</td>\\n</tr><tr id=\\\"pkg_BaseSpaceR\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/BaseSpaceR.html\\\">BaseSpaceR</a></td>\\n\\t<td class=\\\" \\\">Jared O'Connell</td>\\n\\t<td class=\\\" \\\">R SDK for BaseSpace RESTful API</td>\\n\\t<td class=\\\"  sorting_1\\\">1078</td>\\n</tr><tr id=\\\"pkg_BPRMeth\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/BPRMeth.html\\\">BPRMeth</a></td>\\n\\t<td class=\\\" \\\">Chantriolnt-Andreas Kapourani</td>\\n\\t<td class=\\\" \\\">Model higher-order methylation profiles</td>\\n\\t<td class=\\\"  sorting_1\\\">1078</td>\\n</tr><tr id=\\\"pkg_CausalR\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/CausalR.html\\\">CausalR</a></td>\\n\\t<td class=\\\" \\\">Glyn Bradley, Steven Barrett</td>\\n\\t<td class=\\\" \\\">Causal network analysis methods</td>\\n\\t<td class=\\\"  sorting_1\\\">1078</td>\\n</tr><tr id=\\\"pkg_CINdex\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/CINdex.html\\\">CINdex</a></td>\\n\\t<td class=\\\" \\\">Yuriy Gusev</td>\\n\\t<td class=\\\" \\\">Chromosome Instability Index</td>\\n\\t<td class=\\\"  sorting_1\\\">1078</td>\\n</tr><tr id=\\\"pkg_clst\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/clst.html\\\">clst</a></td>\\n\\t<td class=\\\" \\\">Noah Hoffman</td>\\n\\t<td class=\\\" \\\">Classification by local similarity threshold</td>\\n\\t<td class=\\\"  sorting_1\\\">1078</td>\\n</tr><tr id=\\\"pkg_CNAnorm\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/CNAnorm.html\\\">CNAnorm</a></td>\\n\\t<td class=\\\" \\\">Stefano Berri</td>\\n\\t<td class=\\\" \\\">A normalization method for Copy Number Aberration in cancer samples</td>\\n\\t<td class=\\\"  sorting_1\\\">1078</td>\\n</tr><tr id=\\\"pkg_CNVPanelizer\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/CNVPanelizer.html\\\">CNVPanelizer</a></td>\\n\\t<td class=\\\" \\\">Thomas Wolf</td>\\n\\t<td class=\\\" \\\">Reliable CNV detection in targeted sequencing applications</td>\\n\\t<td class=\\\"  sorting_1\\\">1078</td>\\n</tr><tr id=\\\"pkg_compEpiTools\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/compEpiTools.html\\\">compEpiTools</a></td>\\n\\t<td class=\\\" \\\">Kamal Kishore</td>\\n\\t<td class=\\\" \\\">Tools for computational epigenomics</td>\\n\\t<td class=\\\"  sorting_1\\\">1078</td>\\n</tr><tr id=\\\"pkg_doppelgangR\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/doppelgangR.html\\\">doppelgangR</a></td>\\n\\t<td class=\\\" \\\">Levi Waldron</td>\\n\\t<td class=\\\" \\\">Identify likely duplicate samples from genomic or meta-data</td>\\n\\t<td class=\\\"  sorting_1\\\">1078</td>\\n</tr><tr id=\\\"pkg_GeneBreak\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/GeneBreak.html\\\">GeneBreak</a></td>\\n\\t<td class=\\\" \\\">Evert van den Broek</td>\\n\\t<td class=\\\" \\\">Gene Break Detection</td>\\n\\t<td class=\\\"  sorting_1\\\">1078</td>\\n</tr><tr id=\\\"pkg_GraphAlignment\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/GraphAlignment.html\\\">GraphAlignment</a></td>\\n\\t<td class=\\\" \\\">Joern P. Meier</td>\\n\\t<td class=\\\" \\\">GraphAlignment</td>\\n\\t<td class=\\\"  sorting_1\\\">1078</td>\\n</tr><tr id=\\\"pkg_GSALightning\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/GSALightning.html\\\">GSALightning</a></td>\\n\\t<td class=\\\" \\\">Billy Heung Wing Chang</td>\\n\\t<td class=\\\" \\\">Fast Permutation-based Gene Set Analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">1078</td>\\n</tr><tr id=\\\"pkg_IsoCorrectoR\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/IsoCorrectoR.html\\\">IsoCorrectoR</a></td>\\n\\t<td class=\\\" \\\">Christian Kohler</td>\\n\\t<td class=\\\" \\\">Correction for natural isotope abundance and tracer purity in MS and MS/MS data from stable isotope labeling experiments</td>\\n\\t<td class=\\\"  sorting_1\\\">1078</td>\\n</tr><tr id=\\\"pkg_Mergeomics\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/Mergeomics.html\\\">Mergeomics</a></td>\\n\\t<td class=\\\" \\\">Zeyneb Kurt</td>\\n\\t<td class=\\\" \\\">Integrative network analysis of omics data</td>\\n\\t<td class=\\\"  sorting_1\\\">1078</td>\\n</tr><tr id=\\\"pkg_MetaCyto\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/MetaCyto.html\\\">MetaCyto</a></td>\\n\\t<td class=\\\" \\\">Zicheng Hu</td>\\n\\t<td class=\\\" \\\">MetaCyto: A package for meta-analysis of cytometry data</td>\\n\\t<td class=\\\"  sorting_1\\\">1078</td>\\n</tr><tr id=\\\"pkg_MsBackendMsp\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/MsBackendMsp.html\\\">MsBackendMsp</a></td>\\n\\t<td class=\\\" \\\">Johannes Rainer</td>\\n\\t<td class=\\\" \\\">Mass Spectrometry Data Backend for NIST msp Files</td>\\n\\t<td class=\\\"  sorting_1\\\">1078</td>\\n</tr><tr id=\\\"pkg_mumosa\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/mumosa.html\\\">mumosa</a></td>\\n\\t<td class=\\\" \\\">Aaron Lun</td>\\n\\t<td class=\\\" \\\">Multi-Modal Single-Cell Analysis Methods</td>\\n\\t<td class=\\\"  sorting_1\\\">1078</td>\\n</tr><tr id=\\\"pkg_musicatk\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/musicatk.html\\\">musicatk</a></td>\\n\\t<td class=\\\" \\\">Aaron Chevalier</td>\\n\\t<td class=\\\" \\\">Mutational Signature Comprehensive Analysis Toolkit</td>\\n\\t<td class=\\\"  sorting_1\\\">1078</td>\\n</tr><tr id=\\\"pkg_PAA\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/PAA.html\\\">PAA</a></td>\\n\\t<td class=\\\" \\\">Michael Turewicz, Martin Eisenacher</td>\\n\\t<td class=\\\" \\\">PAA (Protein Array Analyzer)</td>\\n\\t<td class=\\\"  sorting_1\\\">1078</td>\\n</tr><tr id=\\\"pkg_PECA\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/PECA.html\\\">PECA</a></td>\\n\\t<td class=\\\" \\\">Tomi Suomi</td>\\n\\t<td class=\\\" \\\">Probe-level Expression Change Averaging</td>\\n\\t<td class=\\\"  sorting_1\\\">1078</td>\\n</tr><tr id=\\\"pkg_qcmetrics\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/qcmetrics.html\\\">qcmetrics</a></td>\\n\\t<td class=\\\" \\\">Laurent Gatto</td>\\n\\t<td class=\\\" \\\">A Framework for Quality Control</td>\\n\\t<td class=\\\"  sorting_1\\\">1078</td>\\n</tr><tr id=\\\"pkg_rhdf5client\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/rhdf5client.html\\\">rhdf5client</a></td>\\n\\t<td class=\\\" \\\">Vincent Carey</td>\\n\\t<td class=\\\" \\\">Access HDF5 content from h5serv</td>\\n\\t<td class=\\\"  sorting_1\\\">1078</td>\\n</tr><tr id=\\\"pkg_SCANVIS\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/SCANVIS.html\\\">SCANVIS</a></td>\\n\\t<td class=\\\" \\\">Phaedra Agius</td>\\n\\t<td class=\\\" \\\">SCANVIS - a tool for SCoring, ANnotating and VISualizing splice junctions</td>\\n\\t<td class=\\\"  sorting_1\\\">1078</td>\\n</tr><tr id=\\\"pkg_scruff\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/scruff.html\\\">scruff</a></td>\\n\\t<td class=\\\" \\\">Zhe Wang</td>\\n\\t<td class=\\\" \\\">Single Cell RNA-Seq UMI Filtering Facilitator (scruff)</td>\\n\\t<td class=\\\"  sorting_1\\\">1078</td>\\n</tr><tr id=\\\"pkg_seqbias\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/seqbias.html\\\">seqbias</a></td>\\n\\t<td class=\\\" \\\">Daniel Jones</td>\\n\\t<td class=\\\" \\\">Estimation of per-position bias in high-throughput sequencing data</td>\\n\\t<td class=\\\"  sorting_1\\\">1078</td>\\n</tr><tr id=\\\"pkg_snapcount\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/snapcount.html\\\">snapcount</a></td>\\n\\t<td class=\\\" \\\">Rone Charles</td>\\n\\t<td class=\\\" \\\">R/Bioconductor Package for interfacing with Snaptron for rapid querying of expression counts</td>\\n\\t<td class=\\\"  sorting_1\\\">1078</td>\\n</tr><tr id=\\\"pkg_tscR\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/tscR.html\\\">tscR</a></td>\\n\\t<td class=\\\" \\\">Fernando Prez-Sanz</td>\\n\\t<td class=\\\" \\\">A time series clustering package combining slope and Frechet distances</td>\\n\\t<td class=\\\"  sorting_1\\\">1078</td>\\n</tr><tr id=\\\"pkg_abseqR\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/abseqR.html\\\">abseqR</a></td>\\n\\t<td class=\\\" \\\">JiaHong Fong</td>\\n\\t<td class=\\\" \\\">Reporting and data analysis functionalities for Rep-Seq datasets of antibody libraries</td>\\n\\t<td class=\\\"  sorting_1\\\">1107</td>\\n</tr><tr id=\\\"pkg_ADAM\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ADAM.html\\\">ADAM</a></td>\\n\\t<td class=\\\" \\\">Jose Luiz Rybarczyk Filho</td>\\n\\t<td class=\\\" \\\">ADAM: Activity and Diversity Analysis Module</td>\\n\\t<td class=\\\"  sorting_1\\\">1107</td>\\n</tr><tr id=\\\"pkg_AffyRNADegradation\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/AffyRNADegradation.html\\\">AffyRNADegradation</a></td>\\n\\t<td class=\\\" \\\">Mario Fasold</td>\\n\\t<td class=\\\" \\\">Analyze and correct probe positional bias in microarray data due to RNA degradation</td>\\n\\t<td class=\\\"  sorting_1\\\">1107</td>\\n</tr><tr id=\\\"pkg_apComplex\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/apComplex.html\\\">apComplex</a></td>\\n\\t<td class=\\\" \\\">Denise Scholtens</td>\\n\\t<td class=\\\" \\\">Estimate protein complex membership using AP-MS protein data</td>\\n\\t<td class=\\\"  sorting_1\\\">1107</td>\\n</tr><tr id=\\\"pkg_basecallQC\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/basecallQC.html\\\">basecallQC</a></td>\\n\\t<td class=\\\" \\\">Thomas Carroll</td>\\n\\t<td class=\\\" \\\">Working with Illumina Basecalling and Demultiplexing input and output files</td>\\n\\t<td class=\\\"  sorting_1\\\">1107</td>\\n</tr><tr id=\\\"pkg_bgx\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/bgx.html\\\">bgx</a></td>\\n\\t<td class=\\\" \\\">Ernest Turro</td>\\n\\t<td class=\\\" \\\">Bayesian Gene eXpression</td>\\n\\t<td class=\\\"  sorting_1\\\">1107</td>\\n</tr><tr id=\\\"pkg_biscuiteer\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/biscuiteer.html\\\">biscuiteer</a></td>\\n\\t<td class=\\\" \\\">Jacob Morrison</td>\\n\\t<td class=\\\" \\\">Convenience Functions for Biscuit</td>\\n\\t<td class=\\\"  sorting_1\\\">1107</td>\\n</tr><tr id=\\\"pkg_Clonality\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/Clonality.html\\\">Clonality</a></td>\\n\\t<td class=\\\" \\\">Irina Ostrovnaya</td>\\n\\t<td class=\\\" \\\">Clonality testing</td>\\n\\t<td class=\\\"  sorting_1\\\">1107</td>\\n</tr><tr id=\\\"pkg_consensusOV\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/consensusOV.html\\\">consensusOV</a></td>\\n\\t<td class=\\\" \\\">Benjamin Haibe-Kains</td>\\n\\t<td class=\\\" \\\">Gene expression-based subtype classification for high-grade serous ovarian cancer</td>\\n\\t<td class=\\\"  sorting_1\\\">1107</td>\\n</tr><tr id=\\\"pkg_DART\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/DART.html\\\">DART</a></td>\\n\\t<td class=\\\" \\\">Charles Shijie Zheng</td>\\n\\t<td class=\\\" \\\">Denoising Algorithm based on Relevance network Topology</td>\\n\\t<td class=\\\"  sorting_1\\\">1107</td>\\n</tr><tr id=\\\"pkg_HiCBricks\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/HiCBricks.html\\\">HiCBricks</a></td>\\n\\t<td class=\\\" \\\">Koustav Pal</td>\\n\\t<td class=\\\" \\\">Framework for Storing and Accessing Hi-C Data Through HDF Files</td>\\n\\t<td class=\\\"  sorting_1\\\">1107</td>\\n</tr><tr id=\\\"pkg_LPE\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/LPE.html\\\">LPE</a></td>\\n\\t<td class=\\\" \\\">Nitin Jain</td>\\n\\t<td class=\\\" \\\">Methods for analyzing microarray data using Local Pooled Error (LPE) method</td>\\n\\t<td class=\\\"  sorting_1\\\">1107</td>\\n</tr><tr id=\\\"pkg_MOMA\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/MOMA.html\\\">MOMA</a></td>\\n\\t<td class=\\\" \\\">Sunny Jones</td>\\n\\t<td class=\\\" \\\">Multi Omic Master Regulator Analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">1107</td>\\n</tr><tr id=\\\"pkg_multiOmicsViz\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/multiOmicsViz.html\\\">multiOmicsViz</a></td>\\n\\t<td class=\\\" \\\">Jing Wang</td>\\n\\t<td class=\\\" \\\">Plot the effect of one omics data on other omics data along the chromosome</td>\\n\\t<td class=\\\"  sorting_1\\\">1107</td>\\n</tr><tr id=\\\"pkg_NanoTube\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/NanoTube.html\\\">NanoTube</a></td>\\n\\t<td class=\\\" \\\">Caleb Class</td>\\n\\t<td class=\\\" \\\">An Easy Pipeline for NanoString nCounter Data Analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">1107</td>\\n</tr><tr id=\\\"pkg_pkgDepTools\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/pkgDepTools.html\\\">pkgDepTools</a></td>\\n\\t<td class=\\\" \\\">Bioconductor Package Maintainer</td>\\n\\t<td class=\\\" \\\">Package Dependency Tools</td>\\n\\t<td class=\\\"  sorting_1\\\">1107</td>\\n</tr><tr id=\\\"pkg_pwrEWAS\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/pwrEWAS.html\\\">pwrEWAS</a></td>\\n\\t<td class=\\\" \\\">Stefan Graw</td>\\n\\t<td class=\\\" \\\">A user-friendly tool for comprehensive power estimation for epigenome wide association studies (EWAS)</td>\\n\\t<td class=\\\"  sorting_1\\\">1107</td>\\n</tr><tr id=\\\"pkg_Rbwa\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/Rbwa.html\\\">Rbwa</a></td>\\n\\t<td class=\\\" \\\">Jean-Philippe Fortin</td>\\n\\t<td class=\\\" \\\">R wrapper for BWA-backtrack and BWA-MEM aligners</td>\\n\\t<td class=\\\"  sorting_1\\\">1107</td>\\n</tr><tr id=\\\"pkg_riboSeqR\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/riboSeqR.html\\\">riboSeqR</a></td>\\n\\t<td class=\\\" \\\">Thomas J. Hardcastle</td>\\n\\t<td class=\\\" \\\">Analysis of sequencing data from ribosome profiling experiments</td>\\n\\t<td class=\\\"  sorting_1\\\">1107</td>\\n</tr><tr id=\\\"pkg_rTRM\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/rTRM.html\\\">rTRM</a></td>\\n\\t<td class=\\\" \\\">Diego Diez</td>\\n\\t<td class=\\\" \\\">Identification of Transcriptional Regulatory Modules from Protein-Protein Interaction Networks</td>\\n\\t<td class=\\\"  sorting_1\\\">1107</td>\\n</tr><tr id=\\\"pkg_scmeth\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/scmeth.html\\\">scmeth</a></td>\\n\\t<td class=\\\" \\\">Divy Kangeyan</td>\\n\\t<td class=\\\" \\\">Functions to conduct quality control analysis in methylation data</td>\\n\\t<td class=\\\"  sorting_1\\\">1107</td>\\n</tr><tr id=\\\"pkg_atSNP\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/atSNP.html\\\">atSNP</a></td>\\n\\t<td class=\\\" \\\">Sunyoung Shin</td>\\n\\t<td class=\\\" \\\">Affinity test for identifying regulatory SNPs</td>\\n\\t<td class=\\\"  sorting_1\\\">1128</td>\\n</tr><tr id=\\\"pkg_biodbKegg\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/biodbKegg.html\\\">biodbKegg</a></td>\\n\\t<td class=\\\" \\\">Pierrick Roger</td>\\n\\t<td class=\\\" \\\">biodbKegg, a library for connecting to the KEGG Database</td>\\n\\t<td class=\\\"  sorting_1\\\">1128</td>\\n</tr><tr id=\\\"pkg_BloodGen3Module\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/BloodGen3Module.html\\\">BloodGen3Module</a></td>\\n\\t<td class=\\\" \\\">Darawan Rinchai</td>\\n\\t<td class=\\\" \\\">This R package for performing module repertoire analyses and generating fingerprint representations</td>\\n\\t<td class=\\\"  sorting_1\\\">1128</td>\\n</tr><tr id=\\\"pkg_CFAssay\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/CFAssay.html\\\">CFAssay</a></td>\\n\\t<td class=\\\" \\\">Herbert Braselmann</td>\\n\\t<td class=\\\" \\\">Statistical analysis for the Colony Formation Assay</td>\\n\\t<td class=\\\"  sorting_1\\\">1128</td>\\n</tr><tr id=\\\"pkg_codelink\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/codelink.html\\\">codelink</a></td>\\n\\t<td class=\\\" \\\">Diego Diez</td>\\n\\t<td class=\\\" \\\">Manipulation of Codelink microarray data</td>\\n\\t<td class=\\\"  sorting_1\\\">1128</td>\\n</tr><tr id=\\\"pkg_debCAM\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/debCAM.html\\\">debCAM</a></td>\\n\\t<td class=\\\" \\\">Lulu Chen</td>\\n\\t<td class=\\\" \\\">Deconvolution by Convex Analysis of Mixtures</td>\\n\\t<td class=\\\"  sorting_1\\\">1128</td>\\n</tr><tr id=\\\"pkg_EventPointer\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/EventPointer.html\\\">EventPointer</a></td>\\n\\t<td class=\\\" \\\">Juan A. Ferrer-Bonsoms</td>\\n\\t<td class=\\\" \\\">An effective identification of alternative splicing events using junction arrays and RNA-Seq data</td>\\n\\t<td class=\\\"  sorting_1\\\">1128</td>\\n</tr><tr id=\\\"pkg_FitHiC\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/FitHiC.html\\\">FitHiC</a></td>\\n\\t<td class=\\\" \\\">Ruyu Tan</td>\\n\\t<td class=\\\" \\\">Confidence estimation for intra-chromosomal contact maps</td>\\n\\t<td class=\\\"  sorting_1\\\">1128</td>\\n</tr><tr id=\\\"pkg_fmrs\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/fmrs.html\\\">fmrs</a></td>\\n\\t<td class=\\\" \\\">Farhad Shokoohi</td>\\n\\t<td class=\\\" \\\">Variable Selection in Finite Mixture of AFT Regression and FMR Models</td>\\n\\t<td class=\\\"  sorting_1\\\">1128</td>\\n</tr><tr id=\\\"pkg_interacCircos\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/interacCircos.html\\\">interacCircos</a></td>\\n\\t<td class=\\\" \\\">Zhe Cui</td>\\n\\t<td class=\\\" \\\">The Generation of Interactive Circos Plot</td>\\n\\t<td class=\\\"  sorting_1\\\">1128</td>\\n</tr><tr id=\\\"pkg_InterMineR\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/InterMineR.html\\\">InterMineR</a></td>\\n\\t<td class=\\\" \\\">InterMine Team</td>\\n\\t<td class=\\\" \\\">R Interface with InterMine-Powered Databases</td>\\n\\t<td class=\\\"  sorting_1\\\">1128</td>\\n</tr><tr id=\\\"pkg_isomiRs\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/isomiRs.html\\\">isomiRs</a></td>\\n\\t<td class=\\\" \\\">Lorena Pantano</td>\\n\\t<td class=\\\" \\\">Analyze isomiRs and miRNAs from small RNA-seq</td>\\n\\t<td class=\\\"  sorting_1\\\">1128</td>\\n</tr><tr id=\\\"pkg_IVAS\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/IVAS.html\\\">IVAS</a></td>\\n\\t<td class=\\\" \\\">Seonggyun Han</td>\\n\\t<td class=\\\" \\\">Identification of genetic Variants affecting Alternative Splicing</td>\\n\\t<td class=\\\"  sorting_1\\\">1128</td>\\n</tr><tr id=\\\"pkg_LOBSTAHS\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/LOBSTAHS.html\\\">LOBSTAHS</a></td>\\n\\t<td class=\\\" \\\">Henry Holm, Daniel Lowenstein, James Collins</td>\\n\\t<td class=\\\" \\\">Lipid and Oxylipin Biomarker Screening through Adduct Hierarchy Sequences</td>\\n\\t<td class=\\\"  sorting_1\\\">1128</td>\\n</tr><tr id=\\\"pkg_MatrixQCvis\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/MatrixQCvis.html\\\">MatrixQCvis</a></td>\\n\\t<td class=\\\" \\\">Thomas Naake</td>\\n\\t<td class=\\\" \\\">Shiny-based interactive data-quality exploration for omics data</td>\\n\\t<td class=\\\"  sorting_1\\\">1128</td>\\n</tr><tr id=\\\"pkg_MethylAid\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/MethylAid.html\\\">MethylAid</a></td>\\n\\t<td class=\\\" \\\">L.J.Sinke</td>\\n\\t<td class=\\\" \\\">Visual and interactive quality control of large Illumina DNA Methylation array data sets</td>\\n\\t<td class=\\\"  sorting_1\\\">1128</td>\\n</tr><tr id=\\\"pkg_methylPipe\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/methylPipe.html\\\">methylPipe</a></td>\\n\\t<td class=\\\" \\\">Kamal Kishore</td>\\n\\t<td class=\\\" \\\">Base resolution DNA methylation data analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">1128</td>\\n</tr><tr id=\\\"pkg_miRspongeR\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/miRspongeR.html\\\">miRspongeR</a></td>\\n\\t<td class=\\\" \\\">Junpeng Zhang</td>\\n\\t<td class=\\\" \\\">Identification and analysis of miRNA sponge regulation</td>\\n\\t<td class=\\\"  sorting_1\\\">1128</td>\\n</tr><tr id=\\\"pkg_mistyR\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/mistyR.html\\\">mistyR</a></td>\\n\\t<td class=\\\" \\\">Jovan Tanevski</td>\\n\\t<td class=\\\" \\\">Multiview Intercellular SpaTial modeling framework</td>\\n\\t<td class=\\\"  sorting_1\\\">1128</td>\\n</tr><tr id=\\\"pkg_mitch\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/mitch.html\\\">mitch</a></td>\\n\\t<td class=\\\" \\\">Mark Ziemann</td>\\n\\t<td class=\\\" \\\">Multi-Contrast Gene Set Enrichment Analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">1128</td>\\n</tr><tr id=\\\"pkg_moanin\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/moanin.html\\\">moanin</a></td>\\n\\t<td class=\\\" \\\">Nelle Varoquaux</td>\\n\\t<td class=\\\" \\\">An R Package for Time Course RNASeq Data Analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">1128</td>\\n</tr><tr id=\\\"pkg_OncoSimulR\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/OncoSimulR.html\\\">OncoSimulR</a></td>\\n\\t<td class=\\\" \\\">Ramon Diaz-Uriarte</td>\\n\\t<td class=\\\" \\\">Forward Genetic Simulation of Cancer Progression with Epistasis</td>\\n\\t<td class=\\\"  sorting_1\\\">1128</td>\\n</tr><tr id=\\\"pkg_Oscope\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/Oscope.html\\\">Oscope</a></td>\\n\\t<td class=\\\" \\\">Ning Leng</td>\\n\\t<td class=\\\" \\\">Oscope - A statistical pipeline for identifying oscillatory genes in unsynchronized single cell RNA-seq</td>\\n\\t<td class=\\\"  sorting_1\\\">1128</td>\\n</tr><tr id=\\\"pkg_rmelting\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/rmelting.html\\\">rmelting</a></td>\\n\\t<td class=\\\" \\\">J. Aravind</td>\\n\\t<td class=\\\" \\\">R Interface to MELTING 5</td>\\n\\t<td class=\\\"  sorting_1\\\">1128</td>\\n</tr><tr id=\\\"pkg_RTNsurvival\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/RTNsurvival.html\\\">RTNsurvival</a></td>\\n\\t<td class=\\\" \\\">Clarice Groeneveld, Mauro A. A. Castro</td>\\n\\t<td class=\\\" \\\">Survival analysis using transcriptional networks inferred by the RTN package</td>\\n\\t<td class=\\\"  sorting_1\\\">1128</td>\\n</tr><tr id=\\\"pkg_savR\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/savR.html\\\">savR</a></td>\\n\\t<td class=\\\" \\\">R. Brent Calder</td>\\n\\t<td class=\\\" \\\">Parse and analyze Illumina SAV files</td>\\n\\t<td class=\\\"  sorting_1\\\">1128</td>\\n</tr><tr id=\\\"pkg_seqsetvis\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/seqsetvis.html\\\">seqsetvis</a></td>\\n\\t<td class=\\\" \\\">Joseph R Boyd</td>\\n\\t<td class=\\\" \\\">Set Based Visualizations for Next-Gen Sequencing Data</td>\\n\\t<td class=\\\"  sorting_1\\\">1128</td>\\n</tr><tr id=\\\"pkg_vbmp\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/vbmp.html\\\">vbmp</a></td>\\n\\t<td class=\\\" \\\">Nicola Lama</td>\\n\\t<td class=\\\" \\\">Variational Bayesian Multinomial Probit Regression</td>\\n\\t<td class=\\\"  sorting_1\\\">1128</td>\\n</tr><tr id=\\\"pkg_adSplit\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/adSplit.html\\\">adSplit</a></td>\\n\\t<td class=\\\" \\\">Claudio Lottaz</td>\\n\\t<td class=\\\" \\\">Annotation-Driven Clustering</td>\\n\\t<td class=\\\"  sorting_1\\\">1156</td>\\n</tr><tr id=\\\"pkg_affyContam\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/affyContam.html\\\">affyContam</a></td>\\n\\t<td class=\\\" \\\">V. Carey</td>\\n\\t<td class=\\\" \\\">structured corruption of affymetrix cel file data</td>\\n\\t<td class=\\\"  sorting_1\\\">1156</td>\\n</tr><tr id=\\\"pkg_bigmelon\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/bigmelon.html\\\">bigmelon</a></td>\\n\\t<td class=\\\" \\\">Tyler J. Gorrie-Stone</td>\\n\\t<td class=\\\" \\\">Illumina methylation array analysis for large experiments</td>\\n\\t<td class=\\\"  sorting_1\\\">1156</td>\\n</tr><tr id=\\\"pkg_BitSeq\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/BitSeq.html\\\">BitSeq</a></td>\\n\\t<td class=\\\" \\\">Antti Honkela, Panagiotis Papastamoulis</td>\\n\\t<td class=\\\" \\\">Transcript expression inference and differential expression analysis for RNA-seq data</td>\\n\\t<td class=\\\"  sorting_1\\\">1156</td>\\n</tr><tr id=\\\"pkg_ChromHeatMap\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ChromHeatMap.html\\\">ChromHeatMap</a></td>\\n\\t<td class=\\\" \\\">Tim F. Rayner</td>\\n\\t<td class=\\\" \\\">Heat map plotting by genome coordinate</td>\\n\\t<td class=\\\"  sorting_1\\\">1156</td>\\n</tr><tr id=\\\"pkg_CHRONOS\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/CHRONOS.html\\\">CHRONOS</a></td>\\n\\t<td class=\\\" \\\">Panos Balomenos</td>\\n\\t<td class=\\\" \\\">CHRONOS: A time-varying method for microRNA-mediated sub-pathway enrichment analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">1156</td>\\n</tr><tr id=\\\"pkg_GeneNetworkBuilder\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/GeneNetworkBuilder.html\\\">GeneNetworkBuilder</a></td>\\n\\t<td class=\\\" \\\">Jianhong Ou</td>\\n\\t<td class=\\\" \\\">GeneNetworkBuilder: a bioconductor package for building regulatory network using ChIP-chip/ChIP-seq data and Gene Expression Data</td>\\n\\t<td class=\\\"  sorting_1\\\">1156</td>\\n</tr><tr id=\\\"pkg_methimpute\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/methimpute.html\\\">methimpute</a></td>\\n\\t<td class=\\\" \\\">Aaron Taudt</td>\\n\\t<td class=\\\" \\\">Imputation-guided re-construction of complete methylomes from WGBS data</td>\\n\\t<td class=\\\"  sorting_1\\\">1156</td>\\n</tr><tr id=\\\"pkg_MineICA\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/MineICA.html\\\">MineICA</a></td>\\n\\t<td class=\\\" \\\">Anne Biton</td>\\n\\t<td class=\\\" \\\">Analysis of an ICA decomposition obtained on genomics data</td>\\n\\t<td class=\\\"  sorting_1\\\">1156</td>\\n</tr><tr id=\\\"pkg_msImpute\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/msImpute.html\\\">msImpute</a></td>\\n\\t<td class=\\\" \\\">Soroor Hediyeh-zadeh</td>\\n\\t<td class=\\\" \\\">Imputation of label-free mass spectrometry peptides</td>\\n\\t<td class=\\\"  sorting_1\\\">1156</td>\\n</tr><tr id=\\\"pkg_NetPathMiner\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/NetPathMiner.html\\\">NetPathMiner</a></td>\\n\\t<td class=\\\" \\\">Ahmed Mohamed</td>\\n\\t<td class=\\\" \\\">NetPathMiner for Biological Network Construction, Path Mining and Visualization</td>\\n\\t<td class=\\\"  sorting_1\\\">1156</td>\\n</tr><tr id=\\\"pkg_qsmooth\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/qsmooth.html\\\">qsmooth</a></td>\\n\\t<td class=\\\" \\\">Stephanie C. Hicks</td>\\n\\t<td class=\\\" \\\">Smooth quantile normalization</td>\\n\\t<td class=\\\"  sorting_1\\\">1156</td>\\n</tr><tr id=\\\"pkg_RBioinf\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/RBioinf.html\\\">RBioinf</a></td>\\n\\t<td class=\\\" \\\">Robert Gentleman</td>\\n\\t<td class=\\\" \\\">RBioinf</td>\\n\\t<td class=\\\"  sorting_1\\\">1156</td>\\n</tr><tr id=\\\"pkg_rbsurv\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/rbsurv.html\\\">rbsurv</a></td>\\n\\t<td class=\\\" \\\">Soo-heang Eo</td>\\n\\t<td class=\\\" \\\">Robust likelihood-based survival modeling with microarray data</td>\\n\\t<td class=\\\"  sorting_1\\\">1156</td>\\n</tr><tr id=\\\"pkg_REMP\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/REMP.html\\\">REMP</a></td>\\n\\t<td class=\\\" \\\">Yinan Zheng</td>\\n\\t<td class=\\\" \\\">Repetitive Element Methylation Prediction</td>\\n\\t<td class=\\\"  sorting_1\\\">1156</td>\\n</tr><tr id=\\\"pkg_SAIGEgds\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/SAIGEgds.html\\\">SAIGEgds</a></td>\\n\\t<td class=\\\" \\\">Xiuwen Zheng</td>\\n\\t<td class=\\\" \\\">Scalable Implementation of Generalized mixed models using GDS files in Phenome-Wide Association Studies</td>\\n\\t<td class=\\\"  sorting_1\\\">1156</td>\\n</tr><tr id=\\\"pkg_TADCompare\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/TADCompare.html\\\">TADCompare</a></td>\\n\\t<td class=\\\" \\\">Kellen Cresswell</td>\\n\\t<td class=\\\" \\\">TADCompare: Identification and characterization of differential TADs</td>\\n\\t<td class=\\\"  sorting_1\\\">1156</td>\\n</tr><tr id=\\\"pkg_TEQC\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/TEQC.html\\\">TEQC</a></td>\\n\\t<td class=\\\" \\\">Sarah Bonnin</td>\\n\\t<td class=\\\" \\\">Quality control for target capture experiments</td>\\n\\t<td class=\\\"  sorting_1\\\">1156</td>\\n</tr><tr id=\\\"pkg_tRNAscanImport\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/tRNAscanImport.html\\\">tRNAscanImport</a></td>\\n\\t<td class=\\\" \\\">Felix G.M. Ernst</td>\\n\\t<td class=\\\" \\\">Importing a tRNAscan-SE result file as GRanges object</td>\\n\\t<td class=\\\"  sorting_1\\\">1156</td>\\n</tr><tr id=\\\"pkg_anota2seq\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/anota2seq.html\\\">anota2seq</a></td>\\n\\t<td class=\\\" \\\">Christian Oertlin, Julie Lorent</td>\\n\\t<td class=\\\" \\\">Generally applicable transcriptome-wide analysis of translational efficiency using anota2seq</td>\\n\\t<td class=\\\"  sorting_1\\\">1175</td>\\n</tr><tr id=\\\"pkg_beadarraySNP\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/beadarraySNP.html\\\">beadarraySNP</a></td>\\n\\t<td class=\\\" \\\">Jan Oosting</td>\\n\\t<td class=\\\" \\\">Normalization and reporting of Illumina SNP bead arrays</td>\\n\\t<td class=\\\"  sorting_1\\\">1175</td>\\n</tr><tr id=\\\"pkg_bioCancer\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/bioCancer.html\\\">bioCancer</a></td>\\n\\t<td class=\\\" \\\">Karim Mezhoud</td>\\n\\t<td class=\\\" \\\">Interactive Multi-Omics Cancers Data Visualization and Analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">1175</td>\\n</tr><tr id=\\\"pkg_BufferedMatrix\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/BufferedMatrix.html\\\">BufferedMatrix</a></td>\\n\\t<td class=\\\" \\\">Ben Bolstad</td>\\n\\t<td class=\\\" \\\">A matrix data storage object held in temporary files</td>\\n\\t<td class=\\\"  sorting_1\\\">1175</td>\\n</tr><tr id=\\\"pkg_cghMCR\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/cghMCR.html\\\">cghMCR</a></td>\\n\\t<td class=\\\" \\\">J. Zhang</td>\\n\\t<td class=\\\" \\\">Find chromosome regions showing common gains/losses</td>\\n\\t<td class=\\\"  sorting_1\\\">1175</td>\\n</tr><tr id=\\\"pkg_crisprBase\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/crisprBase.html\\\">crisprBase</a></td>\\n\\t<td class=\\\" \\\">Jean-Philippe Fortin</td>\\n\\t<td class=\\\" \\\">Base functions and classes for CRISPR gRNA design</td>\\n\\t<td class=\\\"  sorting_1\\\">1175</td>\\n</tr><tr id=\\\"pkg_dagLogo\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/dagLogo.html\\\">dagLogo</a></td>\\n\\t<td class=\\\" \\\">Jianhong Ou</td>\\n\\t<td class=\\\" \\\">dagLogo: a Bioconductor package for visualizing conserved amino acid sequence pattern in groups based on probability theory</td>\\n\\t<td class=\\\"  sorting_1\\\">1175</td>\\n</tr><tr id=\\\"pkg_limmaGUI\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/limmaGUI.html\\\">limmaGUI</a></td>\\n\\t<td class=\\\" \\\">Gordon Smyth</td>\\n\\t<td class=\\\" \\\">GUI for limma Package With Two Color Microarrays</td>\\n\\t<td class=\\\"  sorting_1\\\">1175</td>\\n</tr><tr id=\\\"pkg_MethReg\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/MethReg.html\\\">MethReg</a></td>\\n\\t<td class=\\\" \\\">Tiago Silva</td>\\n\\t<td class=\\\" \\\">Assessing the regulatory potential of DNA methylation regions or sites on gene transcription</td>\\n\\t<td class=\\\"  sorting_1\\\">1175</td>\\n</tr><tr id=\\\"pkg_PERFect\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/PERFect.html\\\">PERFect</a></td>\\n\\t<td class=\\\" \\\">Quy Cao</td>\\n\\t<td class=\\\" \\\">Permutation filtration for microbiome data</td>\\n\\t<td class=\\\"  sorting_1\\\">1175</td>\\n</tr><tr id=\\\"pkg_PloGO2\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/PloGO2.html\\\">PloGO2</a></td>\\n\\t<td class=\\\" \\\">Jemma Wu, Dana Pascovici</td>\\n\\t<td class=\\\" \\\">Plot Gene Ontology and KEGG pathway Annotation and Abundance</td>\\n\\t<td class=\\\"  sorting_1\\\">1175</td>\\n</tr><tr id=\\\"pkg_qckitfastq\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/qckitfastq.html\\\">qckitfastq</a></td>\\n\\t<td class=\\\" \\\">August Guang</td>\\n\\t<td class=\\\" \\\">FASTQ Quality Control</td>\\n\\t<td class=\\\"  sorting_1\\\">1175</td>\\n</tr><tr id=\\\"pkg_RCX\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/RCX.html\\\">RCX</a></td>\\n\\t<td class=\\\" \\\">Florian Auer</td>\\n\\t<td class=\\\" \\\">R package implementing the Cytoscape Exchange (CX) format</td>\\n\\t<td class=\\\"  sorting_1\\\">1175</td>\\n</tr><tr id=\\\"pkg_snapCGH\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/snapCGH.html\\\">snapCGH</a></td>\\n\\t<td class=\\\" \\\">John Marioni</td>\\n\\t<td class=\\\" \\\">Segmentation, normalisation and processing of aCGH data</td>\\n\\t<td class=\\\"  sorting_1\\\">1175</td>\\n</tr><tr id=\\\"pkg_SPEM\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/SPEM.html\\\">SPEM</a></td>\\n\\t<td class=\\\" \\\">Xinyi YANG</td>\\n\\t<td class=\\\" \\\">S-system parameter estimation method</td>\\n\\t<td class=\\\"  sorting_1\\\">1175</td>\\n</tr><tr id=\\\"pkg_splineTimeR\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/splineTimeR.html\\\">splineTimeR</a></td>\\n\\t<td class=\\\" \\\">Herbert Braselmann, Martin Selmansberger</td>\\n\\t<td class=\\\" \\\">Time-course differential gene expression data analysis using spline regression models followed by gene association network reconstruction</td>\\n\\t<td class=\\\"  sorting_1\\\">1175</td>\\n</tr><tr id=\\\"pkg_TFEA.ChIP\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/TFEA.ChIP.html\\\">TFEA.ChIP</a></td>\\n\\t<td class=\\\" \\\">Laura Puente Santamara</td>\\n\\t<td class=\\\" \\\">Analyze Transcription Factor Enrichment</td>\\n\\t<td class=\\\"  sorting_1\\\">1175</td>\\n</tr><tr id=\\\"pkg_timeOmics\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/timeOmics.html\\\">timeOmics</a></td>\\n\\t<td class=\\\" \\\">Antoine Bodein</td>\\n\\t<td class=\\\" \\\">Time-Course Multi-Omics data integration</td>\\n\\t<td class=\\\"  sorting_1\\\">1175</td>\\n</tr><tr id=\\\"pkg_ABarray\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ABarray.html\\\">ABarray</a></td>\\n\\t<td class=\\\" \\\">Yongming Andrew Sun</td>\\n\\t<td class=\\\" \\\">Microarray QA and statistical data analysis for Applied Biosystems Genome Survey Microrarray (AB1700) gene expression data.</td>\\n\\t<td class=\\\"  sorting_1\\\">1193</td>\\n</tr><tr id=\\\"pkg_altcdfenvs\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/altcdfenvs.html\\\">altcdfenvs</a></td>\\n\\t<td class=\\\" \\\">Laurent Gautier</td>\\n\\t<td class=\\\" \\\">alternative CDF environments (aka probeset mappings)</td>\\n\\t<td class=\\\"  sorting_1\\\">1193</td>\\n</tr><tr id=\\\"pkg_BiocOncoTK\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/BiocOncoTK.html\\\">BiocOncoTK</a></td>\\n\\t<td class=\\\" \\\">VJ Carey</td>\\n\\t<td class=\\\" \\\">Bioconductor components for general cancer genomics</td>\\n\\t<td class=\\\"  sorting_1\\\">1193</td>\\n</tr><tr id=\\\"pkg_BioCor\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/BioCor.html\\\">BioCor</a></td>\\n\\t<td class=\\\" \\\">Llus Revilla Sancho</td>\\n\\t<td class=\\\" \\\">Functional similarities</td>\\n\\t<td class=\\\"  sorting_1\\\">1193</td>\\n</tr><tr id=\\\"pkg_CGHnormaliter\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/CGHnormaliter.html\\\">CGHnormaliter</a></td>\\n\\t<td class=\\\" \\\">Bart P.P. van Houte</td>\\n\\t<td class=\\\" \\\">Normalization of array CGH data with imbalanced aberrations.</td>\\n\\t<td class=\\\"  sorting_1\\\">1193</td>\\n</tr><tr id=\\\"pkg_gaga\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/gaga.html\\\">gaga</a></td>\\n\\t<td class=\\\" \\\">David Rossell</td>\\n\\t<td class=\\\" \\\">GaGa hierarchical model for high-throughput data analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">1193</td>\\n</tr><tr id=\\\"pkg_GUIDEseq\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/GUIDEseq.html\\\">GUIDEseq</a></td>\\n\\t<td class=\\\" \\\">Lihua Julie Zhu</td>\\n\\t<td class=\\\" \\\">GUIDE-seq and PEtag-seq analysis pipeline</td>\\n\\t<td class=\\\"  sorting_1\\\">1193</td>\\n</tr><tr id=\\\"pkg_metabolomicsWorkbenchR\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/metabolomicsWorkbenchR.html\\\">metabolomicsWorkbenchR</a></td>\\n\\t<td class=\\\" \\\">Gavin Rhys Lloyd</td>\\n\\t<td class=\\\" \\\">Metabolomics Workbench in R</td>\\n\\t<td class=\\\"  sorting_1\\\">1193</td>\\n</tr><tr id=\\\"pkg_metaCCA\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/metaCCA.html\\\">metaCCA</a></td>\\n\\t<td class=\\\" \\\">Anna Cichonska</td>\\n\\t<td class=\\\" \\\">Summary Statistics-Based Multivariate Meta-Analysis of Genome-Wide Association Studies Using Canonical Correlation Analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">1193</td>\\n</tr><tr id=\\\"pkg_metagene2\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/metagene2.html\\\">metagene2</a></td>\\n\\t<td class=\\\" \\\">Eric Fournier</td>\\n\\t<td class=\\\" \\\">A package to produce metagene plots</td>\\n\\t<td class=\\\"  sorting_1\\\">1193</td>\\n</tr><tr id=\\\"pkg_metaSeq\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/metaSeq.html\\\">metaSeq</a></td>\\n\\t<td class=\\\" \\\">Koki Tsuyuzaki</td>\\n\\t<td class=\\\" \\\">Meta-analysis of RNA-Seq count data in multiple studies</td>\\n\\t<td class=\\\"  sorting_1\\\">1193</td>\\n</tr><tr id=\\\"pkg_MIRA\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/MIRA.html\\\">MIRA</a></td>\\n\\t<td class=\\\" \\\">John Lawson</td>\\n\\t<td class=\\\" \\\">Methylation-Based Inference of Regulatory Activity</td>\\n\\t<td class=\\\"  sorting_1\\\">1193</td>\\n</tr><tr id=\\\"pkg_netresponse\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/netresponse.html\\\">netresponse</a></td>\\n\\t<td class=\\\" \\\">Leo Lahti</td>\\n\\t<td class=\\\" \\\">Functional Network Analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">1193</td>\\n</tr><tr id=\\\"pkg_nucleR\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/nucleR.html\\\">nucleR</a></td>\\n\\t<td class=\\\" \\\">Alba Sala</td>\\n\\t<td class=\\\" \\\">Nucleosome positioning package for R</td>\\n\\t<td class=\\\"  sorting_1\\\">1193</td>\\n</tr><tr id=\\\"pkg_PhyloProfile\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/PhyloProfile.html\\\">PhyloProfile</a></td>\\n\\t<td class=\\\" \\\">Vinh Tran</td>\\n\\t<td class=\\\" \\\">PhyloProfile</td>\\n\\t<td class=\\\"  sorting_1\\\">1193</td>\\n</tr><tr id=\\\"pkg_Rcade\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/Rcade.html\\\">Rcade</a></td>\\n\\t<td class=\\\" \\\">Jonathan Cairns</td>\\n\\t<td class=\\\" \\\">R-based analysis of ChIP-seq And Differential Expression - a tool for integrating a count-based ChIP-seq analysis with differential expression summary data</td>\\n\\t<td class=\\\"  sorting_1\\\">1193</td>\\n</tr><tr id=\\\"pkg_RegEnrich\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/RegEnrich.html\\\">RegEnrich</a></td>\\n\\t<td class=\\\" \\\">Weiyang Tao</td>\\n\\t<td class=\\\" \\\">Gene regulator enrichment analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">1193</td>\\n</tr><tr id=\\\"pkg_spatialDE\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/spatialDE.html\\\">spatialDE</a></td>\\n\\t<td class=\\\" \\\">Gabriele Sales</td>\\n\\t<td class=\\\" \\\">R wrapper for SpatialDE</td>\\n\\t<td class=\\\"  sorting_1\\\">1193</td>\\n</tr><tr id=\\\"pkg_timecourse\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/timecourse.html\\\">timecourse</a></td>\\n\\t<td class=\\\" \\\">Yu Chuan Tai</td>\\n\\t<td class=\\\" \\\">Statistical Analysis for Developmental Microarray Time Course Data</td>\\n\\t<td class=\\\"  sorting_1\\\">1193</td>\\n</tr><tr id=\\\"pkg_TMixClust\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/TMixClust.html\\\">TMixClust</a></td>\\n\\t<td class=\\\" \\\">Monica Golumbeanu</td>\\n\\t<td class=\\\" \\\">Time Series Clustering of Gene Expression with Gaussian Mixed-Effects Models and Smoothing Splines</td>\\n\\t<td class=\\\"  sorting_1\\\">1193</td>\\n</tr><tr id=\\\"pkg_anota\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/anota.html\\\">anota</a></td>\\n\\t<td class=\\\" \\\">Ola Larsson</td>\\n\\t<td class=\\\" \\\">ANalysis Of Translational Activity (ANOTA).</td>\\n\\t<td class=\\\"  sorting_1\\\">1213</td>\\n</tr><tr id=\\\"pkg_APAlyzer\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/APAlyzer.html\\\">APAlyzer</a></td>\\n\\t<td class=\\\" \\\">Ruijia Wang</td>\\n\\t<td class=\\\" \\\">A toolkit for APA analysis using RNA-seq data</td>\\n\\t<td class=\\\"  sorting_1\\\">1213</td>\\n</tr><tr id=\\\"pkg_BADER\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/BADER.html\\\">BADER</a></td>\\n\\t<td class=\\\" \\\">Andreas Neudecker</td>\\n\\t<td class=\\\" \\\">Bayesian Analysis of Differential Expression in RNA Sequencing Data</td>\\n\\t<td class=\\\"  sorting_1\\\">1213</td>\\n</tr><tr id=\\\"pkg_BANDITS\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/BANDITS.html\\\">BANDITS</a></td>\\n\\t<td class=\\\" \\\">Simone Tiberi</td>\\n\\t<td class=\\\" \\\">BANDITS: Bayesian ANalysis of DIfferenTial Splicing</td>\\n\\t<td class=\\\"  sorting_1\\\">1213</td>\\n</tr><tr id=\\\"pkg_CGHregions\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/CGHregions.html\\\">CGHregions</a></td>\\n\\t<td class=\\\" \\\">Sjoerd Vosse</td>\\n\\t<td class=\\\" \\\">Dimension Reduction for Array CGH Data with Minimal Information Loss.</td>\\n\\t<td class=\\\"  sorting_1\\\">1213</td>\\n</tr><tr id=\\\"pkg_circRNAprofiler\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/circRNAprofiler.html\\\">circRNAprofiler</a></td>\\n\\t<td class=\\\" \\\">Simona Aufiero</td>\\n\\t<td class=\\\" \\\">circRNAprofiler: An R-Based Computational Framework for the Downstream Analysis of Circular RNAs</td>\\n\\t<td class=\\\"  sorting_1\\\">1213</td>\\n</tr><tr id=\\\"pkg_CoRegNet\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/CoRegNet.html\\\">CoRegNet</a></td>\\n\\t<td class=\\\" \\\">Remy Nicolle</td>\\n\\t<td class=\\\" \\\">CoRegNet : reconstruction and integrated analysis of co-regulatory networks</td>\\n\\t<td class=\\\"  sorting_1\\\">1213</td>\\n</tr><tr id=\\\"pkg_cosmosR\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/cosmosR.html\\\">cosmosR</a></td>\\n\\t<td class=\\\" \\\">Attila Gabor</td>\\n\\t<td class=\\\" \\\">COSMOS (Causal Oriented Search of Multi-Omic Space)</td>\\n\\t<td class=\\\"  sorting_1\\\">1213</td>\\n</tr><tr id=\\\"pkg_EMDomics\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/EMDomics.html\\\">EMDomics</a></td>\\n\\t<td class=\\\" \\\">Sadhika Malladi and Daniel Schmolze</td>\\n\\t<td class=\\\" \\\">Earth Mover's Distance for Differential Analysis of Genomics Data</td>\\n\\t<td class=\\\"  sorting_1\\\">1213</td>\\n</tr><tr id=\\\"pkg_epivizr\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/epivizr.html\\\">epivizr</a></td>\\n\\t<td class=\\\" \\\">Hector Corrada Bravo</td>\\n\\t<td class=\\\" \\\">R Interface to epiviz web app</td>\\n\\t<td class=\\\"  sorting_1\\\">1213</td>\\n</tr><tr id=\\\"pkg_flowMerge\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/flowMerge.html\\\">flowMerge</a></td>\\n\\t<td class=\\\" \\\">Greg Finak</td>\\n\\t<td class=\\\" \\\">Cluster Merging for Flow Cytometry Data</td>\\n\\t<td class=\\\"  sorting_1\\\">1213</td>\\n</tr><tr id=\\\"pkg_GEOexplorer\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/GEOexplorer.html\\\">GEOexplorer</a></td>\\n\\t<td class=\\\" \\\">Guy Hunt</td>\\n\\t<td class=\\\" \\\">GEOexplorer: an R/Bioconductor package for gene expression analysis and visualisation</td>\\n\\t<td class=\\\"  sorting_1\\\">1213</td>\\n</tr><tr id=\\\"pkg_goTools\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/goTools.html\\\">goTools</a></td>\\n\\t<td class=\\\" \\\">Agnes Paquet</td>\\n\\t<td class=\\\" \\\">Functions for Gene Ontology database</td>\\n\\t<td class=\\\"  sorting_1\\\">1213</td>\\n</tr><tr id=\\\"pkg_groHMM\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/groHMM.html\\\">groHMM</a></td>\\n\\t<td class=\\\" \\\">Tulip Nandu, W. Lee Kraus</td>\\n\\t<td class=\\\" \\\">GRO-seq Analysis Pipeline</td>\\n\\t<td class=\\\"  sorting_1\\\">1213</td>\\n</tr><tr id=\\\"pkg_IONiseR\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/IONiseR.html\\\">IONiseR</a></td>\\n\\t<td class=\\\" \\\">Mike Smith</td>\\n\\t<td class=\\\" \\\">Quality Assessment Tools for Oxford Nanopore MinION data</td>\\n\\t<td class=\\\"  sorting_1\\\">1213</td>\\n</tr><tr id=\\\"pkg_methylCC\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/methylCC.html\\\">methylCC</a></td>\\n\\t<td class=\\\" \\\">Stephanie C. Hicks</td>\\n\\t<td class=\\\" \\\">Estimate the cell composition of whole blood in DNA methylation samples</td>\\n\\t<td class=\\\"  sorting_1\\\">1213</td>\\n</tr><tr id=\\\"pkg_odseq\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/odseq.html\\\">odseq</a></td>\\n\\t<td class=\\\" \\\">Jos Jimnez</td>\\n\\t<td class=\\\" \\\">Outlier detection in multiple sequence alignments</td>\\n\\t<td class=\\\"  sorting_1\\\">1213</td>\\n</tr><tr id=\\\"pkg_qsea\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/qsea.html\\\">qsea</a></td>\\n\\t<td class=\\\" \\\">Matthias Lienhard</td>\\n\\t<td class=\\\" \\\">IP-seq data analysis and vizualization</td>\\n\\t<td class=\\\"  sorting_1\\\">1213</td>\\n</tr><tr id=\\\"pkg_QuaternaryProd\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/QuaternaryProd.html\\\">QuaternaryProd</a></td>\\n\\t<td class=\\\" \\\">Carl Tony Fakhry</td>\\n\\t<td class=\\\" \\\">Computes the Quaternary Dot Product Scoring Statistic for Signed and Unsigned Causal Graphs</td>\\n\\t<td class=\\\"  sorting_1\\\">1213</td>\\n</tr><tr id=\\\"pkg_RiboProfiling\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/RiboProfiling.html\\\">RiboProfiling</a></td>\\n\\t<td class=\\\" \\\">A. Popa</td>\\n\\t<td class=\\\" \\\">Ribosome Profiling Data Analysis: from BAM to Data Representation and Interpretation</td>\\n\\t<td class=\\\"  sorting_1\\\">1213</td>\\n</tr><tr id=\\\"pkg_RSeqAn\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/RSeqAn.html\\\">RSeqAn</a></td>\\n\\t<td class=\\\" \\\">August Guang</td>\\n\\t<td class=\\\" \\\">R SeqAn</td>\\n\\t<td class=\\\"  sorting_1\\\">1213</td>\\n</tr><tr id=\\\"pkg_runibic\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/runibic.html\\\">runibic</a></td>\\n\\t<td class=\\\" \\\">Patryk Orzechowski</td>\\n\\t<td class=\\\" \\\">runibic: row-based biclustering algorithm for analysis of gene expression data in R</td>\\n\\t<td class=\\\"  sorting_1\\\">1213</td>\\n</tr><tr id=\\\"pkg_scHOT\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/scHOT.html\\\">scHOT</a></td>\\n\\t<td class=\\\" \\\">Shila Ghazanfar</td>\\n\\t<td class=\\\" \\\">single-cell higher order testing</td>\\n\\t<td class=\\\"  sorting_1\\\">1213</td>\\n</tr><tr id=\\\"pkg_SpectralTAD\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/SpectralTAD.html\\\">SpectralTAD</a></td>\\n\\t<td class=\\\" \\\">Mikhail Dozmorov</td>\\n\\t<td class=\\\" \\\">SpectralTAD: Hierarchical TAD detection using spectral clustering</td>\\n\\t<td class=\\\"  sorting_1\\\">1213</td>\\n</tr><tr id=\\\"pkg_subSeq\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/subSeq.html\\\">subSeq</a></td>\\n\\t<td class=\\\" \\\">Andrew J. Bass, John D. Storey</td>\\n\\t<td class=\\\" \\\">Subsampling of high-throughput sequencing count data</td>\\n\\t<td class=\\\"  sorting_1\\\">1213</td>\\n</tr><tr id=\\\"pkg_BCRANK\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/BCRANK.html\\\">BCRANK</a></td>\\n\\t<td class=\\\" \\\">Adam Ameur</td>\\n\\t<td class=\\\" \\\">Predicting binding site consensus from ranked DNA sequences</td>\\n\\t<td class=\\\"  sorting_1\\\">1238</td>\\n</tr><tr id=\\\"pkg_BioNetStat\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/BioNetStat.html\\\">BioNetStat</a></td>\\n\\t<td class=\\\" \\\">Vinicius Jardim</td>\\n\\t<td class=\\\" \\\">Biological Network Analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">1238</td>\\n</tr><tr id=\\\"pkg_cancerclass\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/cancerclass.html\\\">cancerclass</a></td>\\n\\t<td class=\\\" \\\">Daniel Kosztyla</td>\\n\\t<td class=\\\" \\\">Development and validation of diagnostic tests from high-dimensional molecular data</td>\\n\\t<td class=\\\"  sorting_1\\\">1238</td>\\n</tr><tr id=\\\"pkg_CHETAH\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/CHETAH.html\\\">CHETAH</a></td>\\n\\t<td class=\\\" \\\">Jurrian de Kanter</td>\\n\\t<td class=\\\" \\\">Fast and accurate scRNA-seq cell type identification</td>\\n\\t<td class=\\\"  sorting_1\\\">1238</td>\\n</tr><tr id=\\\"pkg_CluMSID\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/CluMSID.html\\\">CluMSID</a></td>\\n\\t<td class=\\\" \\\">Tobias Depke</td>\\n\\t<td class=\\\" \\\">Clustering of MS2 Spectra for Metabolite Identification</td>\\n\\t<td class=\\\"  sorting_1\\\">1238</td>\\n</tr><tr id=\\\"pkg_diffuStats\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/diffuStats.html\\\">diffuStats</a></td>\\n\\t<td class=\\\" \\\">Sergio Picart-Armada</td>\\n\\t<td class=\\\" \\\">Diffusion scores on biological networks</td>\\n\\t<td class=\\\"  sorting_1\\\">1238</td>\\n</tr><tr id=\\\"pkg_EBcoexpress\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/EBcoexpress.html\\\">EBcoexpress</a></td>\\n\\t<td class=\\\" \\\">John A. Dawson</td>\\n\\t<td class=\\\" \\\">EBcoexpress for Differential Co-Expression Analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">1238</td>\\n</tr><tr id=\\\"pkg_flowTrans\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/flowTrans.html\\\">flowTrans</a></td>\\n\\t<td class=\\\" \\\">Greg Finak</td>\\n\\t<td class=\\\" \\\">Parameter Optimization for Flow Cytometry Data Transformation</td>\\n\\t<td class=\\\"  sorting_1\\\">1238</td>\\n</tr><tr id=\\\"pkg_GSEABenchmarkeR\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/GSEABenchmarkeR.html\\\">GSEABenchmarkeR</a></td>\\n\\t<td class=\\\" \\\">Ludwig Geistlinger</td>\\n\\t<td class=\\\" \\\">Reproducible GSEA Benchmarking</td>\\n\\t<td class=\\\"  sorting_1\\\">1238</td>\\n</tr><tr id=\\\"pkg_hca\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/hca.html\\\">hca</a></td>\\n\\t<td class=\\\" \\\">Martin Morgan</td>\\n\\t<td class=\\\" \\\">Exploring the Human Cell Atlas Data Coordinating Platform</td>\\n\\t<td class=\\\"  sorting_1\\\">1238</td>\\n</tr><tr id=\\\"pkg_hyperdraw\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/hyperdraw.html\\\">hyperdraw</a></td>\\n\\t<td class=\\\" \\\">Paul Murrell</td>\\n\\t<td class=\\\" \\\">Visualizing Hypergaphs</td>\\n\\t<td class=\\\"  sorting_1\\\">1238</td>\\n</tr><tr id=\\\"pkg_ImmuneSpaceR\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ImmuneSpaceR.html\\\">ImmuneSpaceR</a></td>\\n\\t<td class=\\\" \\\">ImmuneSpace Package Maintainer</td>\\n\\t<td class=\\\" \\\">A Thin Wrapper around the ImmuneSpace Data and Tools Portal</td>\\n\\t<td class=\\\"  sorting_1\\\">1238</td>\\n</tr><tr id=\\\"pkg_metaseqR2\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/metaseqR2.html\\\">metaseqR2</a></td>\\n\\t<td class=\\\" \\\">Panagiotis Moulos</td>\\n\\t<td class=\\\" \\\">An R package for the analysis and result reporting of RNA-Seq data by combining multiple statistical algorithms</td>\\n\\t<td class=\\\"  sorting_1\\\">1238</td>\\n</tr><tr id=\\\"pkg_methylSig\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/methylSig.html\\\">methylSig</a></td>\\n\\t<td class=\\\" \\\">Raymond G. Cavalcante</td>\\n\\t<td class=\\\" \\\">MethylSig: Differential Methylation Testing for WGBS and RRBS Data</td>\\n\\t<td class=\\\"  sorting_1\\\">1238</td>\\n</tr><tr id=\\\"pkg_POMA\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/POMA.html\\\">POMA</a></td>\\n\\t<td class=\\\" \\\">Pol Castellano-Escuder</td>\\n\\t<td class=\\\" \\\">Tools for Omics Data Analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">1238</td>\\n</tr><tr id=\\\"pkg_projectR\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/projectR.html\\\">projectR</a></td>\\n\\t<td class=\\\" \\\">Genevieve Stein-O'Brien</td>\\n\\t<td class=\\\" \\\">Functions for the projection of weights from PCA, CoGAPS, NMF, correlation, and clustering</td>\\n\\t<td class=\\\"  sorting_1\\\">1238</td>\\n</tr><tr id=\\\"pkg_ProteoMM\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ProteoMM.html\\\">ProteoMM</a></td>\\n\\t<td class=\\\" \\\">Yuliya V Karpievitch</td>\\n\\t<td class=\\\" \\\">Multi-Dataset Model-based Differential Expression Proteomics Analysis Platform</td>\\n\\t<td class=\\\"  sorting_1\\\">1238</td>\\n</tr><tr id=\\\"pkg_ribosomeProfilingQC\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ribosomeProfilingQC.html\\\">ribosomeProfilingQC</a></td>\\n\\t<td class=\\\" \\\">Jianhong Ou</td>\\n\\t<td class=\\\" \\\">Ribosome Profiling Quality Control</td>\\n\\t<td class=\\\"  sorting_1\\\">1238</td>\\n</tr><tr id=\\\"pkg_rqubic\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/rqubic.html\\\">rqubic</a></td>\\n\\t<td class=\\\" \\\">Jitao David Zhang</td>\\n\\t<td class=\\\" \\\">Qualitative biclustering algorithm for expression data analysis in R</td>\\n\\t<td class=\\\"  sorting_1\\\">1238</td>\\n</tr><tr id=\\\"pkg_seqCNA\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/seqCNA.html\\\">seqCNA</a></td>\\n\\t<td class=\\\" \\\">David Mosen-Ansorena</td>\\n\\t<td class=\\\" \\\">Copy number analysis of high-throughput sequencing cancer data</td>\\n\\t<td class=\\\"  sorting_1\\\">1238</td>\\n</tr><tr id=\\\"pkg_SharedObject\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/SharedObject.html\\\">SharedObject</a></td>\\n\\t<td class=\\\" \\\">Jiefei Wang</td>\\n\\t<td class=\\\" \\\">Sharing R objects across multiple R processes without memory duplication</td>\\n\\t<td class=\\\"  sorting_1\\\">1238</td>\\n</tr><tr id=\\\"pkg_VanillaICE\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/VanillaICE.html\\\">VanillaICE</a></td>\\n\\t<td class=\\\" \\\">Robert Scharpf</td>\\n\\t<td class=\\\" \\\">A Hidden Markov Model for high throughput genotyping arrays</td>\\n\\t<td class=\\\"  sorting_1\\\">1238</td>\\n</tr><tr id=\\\"pkg_vissE\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/vissE.html\\\">vissE</a></td>\\n\\t<td class=\\\" \\\">Dharmesh D. Bhuva</td>\\n\\t<td class=\\\" \\\">Visualising Set Enrichment Analysis Results</td>\\n\\t<td class=\\\"  sorting_1\\\">1238</td>\\n</tr><tr id=\\\"pkg_bioassayR\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/bioassayR.html\\\">bioassayR</a></td>\\n\\t<td class=\\\" \\\">Daniela Cassol</td>\\n\\t<td class=\\\" \\\">Cross-target analysis of small molecule bioactivity</td>\\n\\t<td class=\\\"  sorting_1\\\">1261</td>\\n</tr><tr id=\\\"pkg_branchpointer\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/branchpointer.html\\\">branchpointer</a></td>\\n\\t<td class=\\\" \\\">Beth Signal</td>\\n\\t<td class=\\\" \\\">Prediction of intronic splicing branchpoints</td>\\n\\t<td class=\\\"  sorting_1\\\">1261</td>\\n</tr><tr id=\\\"pkg_BridgeDbR\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/BridgeDbR.html\\\">BridgeDbR</a></td>\\n\\t<td class=\\\" \\\">Egon Willighagen</td>\\n\\t<td class=\\\" \\\">Code for using BridgeDb identifier mapping framework from within R</td>\\n\\t<td class=\\\"  sorting_1\\\">1261</td>\\n</tr><tr id=\\\"pkg_cellbaseR\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/cellbaseR.html\\\">cellbaseR</a></td>\\n\\t<td class=\\\" \\\">Mohammed OE Abdallah</td>\\n\\t<td class=\\\" \\\">Querying annotation data from the high performance Cellbase web</td>\\n\\t<td class=\\\"  sorting_1\\\">1261</td>\\n</tr><tr id=\\\"pkg_CellMixS\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/CellMixS.html\\\">CellMixS</a></td>\\n\\t<td class=\\\" \\\">Almut Ltge</td>\\n\\t<td class=\\\" \\\">Evaluate Cellspecific Mixing</td>\\n\\t<td class=\\\"  sorting_1\\\">1261</td>\\n</tr><tr id=\\\"pkg_EBSeqHMM\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/EBSeqHMM.html\\\">EBSeqHMM</a></td>\\n\\t<td class=\\\" \\\">Ning Leng</td>\\n\\t<td class=\\\" \\\">Bayesian analysis for identifying gene or isoform expression changes in ordered RNA-seq experiments</td>\\n\\t<td class=\\\"  sorting_1\\\">1261</td>\\n</tr><tr id=\\\"pkg_EGAD\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/EGAD.html\\\">EGAD</a></td>\\n\\t<td class=\\\" \\\">Sara Ballouz</td>\\n\\t<td class=\\\" \\\">Extending guilt by association by degree</td>\\n\\t<td class=\\\"  sorting_1\\\">1261</td>\\n</tr><tr id=\\\"pkg_epivizrData\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/epivizrData.html\\\">epivizrData</a></td>\\n\\t<td class=\\\" \\\">Hector Corrada Bravo</td>\\n\\t<td class=\\\" \\\">Data Management API for epiviz interactive visualization app</td>\\n\\t<td class=\\\"  sorting_1\\\">1261</td>\\n</tr><tr id=\\\"pkg_erccdashboard\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/erccdashboard.html\\\">erccdashboard</a></td>\\n\\t<td class=\\\" \\\">Sarah Munro</td>\\n\\t<td class=\\\" \\\">Assess Differential Gene Expression Experiments with ERCC Controls</td>\\n\\t<td class=\\\"  sorting_1\\\">1261</td>\\n</tr><tr id=\\\"pkg_geneplast\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/geneplast.html\\\">geneplast</a></td>\\n\\t<td class=\\\" \\\">Mauro Castro</td>\\n\\t<td class=\\\" \\\">Evolutionary and plasticity analysis of orthologous groups</td>\\n\\t<td class=\\\"  sorting_1\\\">1261</td>\\n</tr><tr id=\\\"pkg_hipathia\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/hipathia.html\\\">hipathia</a></td>\\n\\t<td class=\\\" \\\">Marta R. Hidalgo</td>\\n\\t<td class=\\\" \\\">HiPathia: High-throughput Pathway Analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">1261</td>\\n</tr><tr id=\\\"pkg_IdeoViz\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/IdeoViz.html\\\">IdeoViz</a></td>\\n\\t<td class=\\\" \\\">Shraddha Pai</td>\\n\\t<td class=\\\" \\\">Plots data (continuous/discrete) along chromosomal ideogram</td>\\n\\t<td class=\\\"  sorting_1\\\">1261</td>\\n</tr><tr id=\\\"pkg_maanova\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/maanova.html\\\">maanova</a></td>\\n\\t<td class=\\\" \\\">Keith Sheppard</td>\\n\\t<td class=\\\" \\\">Tools for analyzing Micro Array experiments</td>\\n\\t<td class=\\\"  sorting_1\\\">1261</td>\\n</tr><tr id=\\\"pkg_MEIGOR\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/MEIGOR.html\\\">MEIGOR</a></td>\\n\\t<td class=\\\" \\\">Jose A. Egea</td>\\n\\t<td class=\\\" \\\">MEIGO - MEtaheuristics for bIoinformatics Global Optimization</td>\\n\\t<td class=\\\"  sorting_1\\\">1261</td>\\n</tr><tr id=\\\"pkg_metagene\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/metagene.html\\\">metagene</a></td>\\n\\t<td class=\\\" \\\">Charles Joly Beauparlant</td>\\n\\t<td class=\\\" \\\">A package to produce metagene plots</td>\\n\\t<td class=\\\"  sorting_1\\\">1261</td>\\n</tr><tr id=\\\"pkg_NanoMethViz\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/NanoMethViz.html\\\">NanoMethViz</a></td>\\n\\t<td class=\\\" \\\">Shian Su</td>\\n\\t<td class=\\\" \\\">Visualise methlation data from Oxford Nanopore sequencing</td>\\n\\t<td class=\\\"  sorting_1\\\">1261</td>\\n</tr><tr id=\\\"pkg_netbiov\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/netbiov.html\\\">netbiov</a></td>\\n\\t<td class=\\\" \\\">Shailesh tripathi</td>\\n\\t<td class=\\\" \\\">A package for visualizing complex biological network</td>\\n\\t<td class=\\\"  sorting_1\\\">1261</td>\\n</tr><tr id=\\\"pkg_RCAS\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/RCAS.html\\\">RCAS</a></td>\\n\\t<td class=\\\" \\\">Bora Uyar</td>\\n\\t<td class=\\\" \\\">RNA Centric Annotation System</td>\\n\\t<td class=\\\"  sorting_1\\\">1261</td>\\n</tr><tr id=\\\"pkg_scDataviz\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/scDataviz.html\\\">scDataviz</a></td>\\n\\t<td class=\\\" \\\">Kevin Blighe</td>\\n\\t<td class=\\\" \\\">scDataviz: single cell dataviz and downstream analyses</td>\\n\\t<td class=\\\"  sorting_1\\\">1261</td>\\n</tr><tr id=\\\"pkg_SeqGSEA\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/SeqGSEA.html\\\">SeqGSEA</a></td>\\n\\t<td class=\\\" \\\">Xi Wang</td>\\n\\t<td class=\\\" \\\">Gene Set Enrichment Analysis (GSEA) of RNA-Seq Data: integrating differential expression and splicing</td>\\n\\t<td class=\\\"  sorting_1\\\">1261</td>\\n</tr><tr id=\\\"pkg_SpidermiR\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/SpidermiR.html\\\">SpidermiR</a></td>\\n\\t<td class=\\\" \\\">Claudia Cava</td>\\n\\t<td class=\\\" \\\">SpidermiR: An R/Bioconductor package for integrative network analysis with miRNA data</td>\\n\\t<td class=\\\"  sorting_1\\\">1261</td>\\n</tr><tr id=\\\"pkg_affylmGUI\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/affylmGUI.html\\\">affylmGUI</a></td>\\n\\t<td class=\\\" \\\">Gordon Smyth</td>\\n\\t<td class=\\\" \\\">GUI for limma Package with Affymetrix Microarrays</td>\\n\\t<td class=\\\"  sorting_1\\\">1282</td>\\n</tr><tr id=\\\"pkg_AMOUNTAIN\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/AMOUNTAIN.html\\\">AMOUNTAIN</a></td>\\n\\t<td class=\\\" \\\">Dong Li</td>\\n\\t<td class=\\\" \\\">Active modules for multilayer weighted gene co-expression networks: a continuous optimization approach</td>\\n\\t<td class=\\\"  sorting_1\\\">1282</td>\\n</tr><tr id=\\\"pkg_CGEN\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/CGEN.html\\\">CGEN</a></td>\\n\\t<td class=\\\" \\\">Justin Lee</td>\\n\\t<td class=\\\" \\\">An R package for analysis of case-control studies in genetic epidemiology</td>\\n\\t<td class=\\\"  sorting_1\\\">1282</td>\\n</tr><tr id=\\\"pkg_condiments\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/condiments.html\\\">condiments</a></td>\\n\\t<td class=\\\" \\\">Hector Roux de Bezieux</td>\\n\\t<td class=\\\" \\\">Differential Topology, Progression and Differentiation</td>\\n\\t<td class=\\\"  sorting_1\\\">1282</td>\\n</tr><tr id=\\\"pkg_DeMixT\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/DeMixT.html\\\">DeMixT</a></td>\\n\\t<td class=\\\" \\\">Shuai Guo</td>\\n\\t<td class=\\\" \\\">Cell type-specific deconvolution of heterogeneous tumor samples with two or three components using expression data from RNAseq or microarray platforms</td>\\n\\t<td class=\\\"  sorting_1\\\">1282</td>\\n</tr><tr id=\\\"pkg_DiffLogo\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/DiffLogo.html\\\">DiffLogo</a></td>\\n\\t<td class=\\\" \\\">Hendrik Treutler</td>\\n\\t<td class=\\\" \\\">DiffLogo: A comparative visualisation of biooligomer motifs</td>\\n\\t<td class=\\\"  sorting_1\\\">1282</td>\\n</tr><tr id=\\\"pkg_fcoex\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/fcoex.html\\\">fcoex</a></td>\\n\\t<td class=\\\" \\\">Tiago Lubiana</td>\\n\\t<td class=\\\" \\\">FCBF-based Co-Expression Networks for Single Cells</td>\\n\\t<td class=\\\"  sorting_1\\\">1282</td>\\n</tr><tr id=\\\"pkg_GeneStructureTools\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/GeneStructureTools.html\\\">GeneStructureTools</a></td>\\n\\t<td class=\\\" \\\">Beth Signal</td>\\n\\t<td class=\\\" \\\">Tools for spliced gene structure manipulation and analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">1282</td>\\n</tr><tr id=\\\"pkg_microbiomeExplorer\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/microbiomeExplorer.html\\\">microbiomeExplorer</a></td>\\n\\t<td class=\\\" \\\">Janina Reeder</td>\\n\\t<td class=\\\" \\\">Microbiome Exploration App</td>\\n\\t<td class=\\\"  sorting_1\\\">1282</td>\\n</tr><tr id=\\\"pkg_multiClust\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/multiClust.html\\\">multiClust</a></td>\\n\\t<td class=\\\" \\\">Nathan Lawlor</td>\\n\\t<td class=\\\" \\\">multiClust: An R-package for Identifying Biologically Relevant Clusters in Cancer Transcriptome Profiles</td>\\n\\t<td class=\\\"  sorting_1\\\">1282</td>\\n</tr><tr id=\\\"pkg_sevenbridges\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/sevenbridges.html\\\">sevenbridges</a></td>\\n\\t<td class=\\\" \\\">Soner Koc</td>\\n\\t<td class=\\\" \\\">Seven Bridges Platform API Client and Common Workflow Language Tool Builder in R</td>\\n\\t<td class=\\\"  sorting_1\\\">1282</td>\\n</tr><tr id=\\\"pkg_a4Classif\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/a4Classif.html\\\">a4Classif</a></td>\\n\\t<td class=\\\" \\\">Laure Cougnaud</td>\\n\\t<td class=\\\" \\\">Automated Affymetrix Array Analysis Classification Package</td>\\n\\t<td class=\\\"  sorting_1\\\">1293</td>\\n</tr><tr id=\\\"pkg_ADImpute\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ADImpute.html\\\">ADImpute</a></td>\\n\\t<td class=\\\" \\\">Ana Carolina Leote</td>\\n\\t<td class=\\\" \\\">Adaptive Dropout Imputer (ADImpute)</td>\\n\\t<td class=\\\"  sorting_1\\\">1293</td>\\n</tr><tr id=\\\"pkg_CeTF\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/CeTF.html\\\">CeTF</a></td>\\n\\t<td class=\\\" \\\">Carlos Alberto Oliveira de Biagi Junior</td>\\n\\t<td class=\\\" \\\">Coexpression for Transcription Factors using Regulatory Impact Factors and Partial Correlation and Information Theory analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">1293</td>\\n</tr><tr id=\\\"pkg_COMPASS\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/COMPASS.html\\\">COMPASS</a></td>\\n\\t<td class=\\\" \\\">Greg Finak</td>\\n\\t<td class=\\\" \\\">Combinatorial Polyfunctionality Analysis of Single Cells</td>\\n\\t<td class=\\\"  sorting_1\\\">1293</td>\\n</tr><tr id=\\\"pkg_CopywriteR\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/CopywriteR.html\\\">CopywriteR</a></td>\\n\\t<td class=\\\" \\\">Oscar Krijgsman</td>\\n\\t<td class=\\\" \\\">Copy number information from targeted sequencing using off-target reads</td>\\n\\t<td class=\\\"  sorting_1\\\">1293</td>\\n</tr><tr id=\\\"pkg_corral\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/corral.html\\\">corral</a></td>\\n\\t<td class=\\\" \\\">Lauren Hsu</td>\\n\\t<td class=\\\" \\\">Correspondence Analysis for Single Cell Data</td>\\n\\t<td class=\\\"  sorting_1\\\">1293</td>\\n</tr><tr id=\\\"pkg_ffpe\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ffpe.html\\\">ffpe</a></td>\\n\\t<td class=\\\" \\\">Levi Waldron</td>\\n\\t<td class=\\\" \\\">Quality assessment and control for FFPE microarray expression data</td>\\n\\t<td class=\\\"  sorting_1\\\">1293</td>\\n</tr><tr id=\\\"pkg_GOsummaries\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/GOsummaries.html\\\">GOsummaries</a></td>\\n\\t<td class=\\\" \\\">Raivo Kolde</td>\\n\\t<td class=\\\" \\\">Word cloud summaries of GO enrichment analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">1293</td>\\n</tr><tr id=\\\"pkg_oposSOM\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/oposSOM.html\\\">oposSOM</a></td>\\n\\t<td class=\\\" \\\">Henry Loeffler-Wirth</td>\\n\\t<td class=\\\" \\\">Comprehensive analysis of transcriptome data</td>\\n\\t<td class=\\\"  sorting_1\\\">1293</td>\\n</tr><tr id=\\\"pkg_plgem\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/plgem.html\\\">plgem</a></td>\\n\\t<td class=\\\" \\\">Norman Pavelka</td>\\n\\t<td class=\\\" \\\">Detect differential expression in microarray and proteomics datasets with the Power Law Global Error Model (PLGEM)</td>\\n\\t<td class=\\\"  sorting_1\\\">1293</td>\\n</tr><tr id=\\\"pkg_rDGIdb\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/rDGIdb.html\\\">rDGIdb</a></td>\\n\\t<td class=\\\" \\\">Lars Bosshard</td>\\n\\t<td class=\\\" \\\">R Wrapper for DGIdb</td>\\n\\t<td class=\\\"  sorting_1\\\">1293</td>\\n</tr><tr id=\\\"pkg_RNAmodR\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/RNAmodR.html\\\">RNAmodR</a></td>\\n\\t<td class=\\\" \\\">Felix G.M. Ernst</td>\\n\\t<td class=\\\" \\\">Detection of post-transcriptional modifications in high throughput sequencing data</td>\\n\\t<td class=\\\"  sorting_1\\\">1293</td>\\n</tr><tr id=\\\"pkg_ASICS\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ASICS.html\\\">ASICS</a></td>\\n\\t<td class=\\\" \\\">Galle Lefort</td>\\n\\t<td class=\\\" \\\">Automatic Statistical Identification in Complex Spectra</td>\\n\\t<td class=\\\"  sorting_1\\\">1305</td>\\n</tr><tr id=\\\"pkg_CoverageView\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/CoverageView.html\\\">CoverageView</a></td>\\n\\t<td class=\\\" \\\">Ernesto Lowy</td>\\n\\t<td class=\\\" \\\">Coverage visualization package for R</td>\\n\\t<td class=\\\"  sorting_1\\\">1305</td>\\n</tr><tr id=\\\"pkg_CRImage\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/CRImage.html\\\">CRImage</a></td>\\n\\t<td class=\\\" \\\">Henrik Failmezger, Yinyin Yuan</td>\\n\\t<td class=\\\" \\\">CRImage a package to classify cells and calculate tumour cellularity</td>\\n\\t<td class=\\\"  sorting_1\\\">1305</td>\\n</tr><tr id=\\\"pkg_easyRNASeq\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/easyRNASeq.html\\\">easyRNASeq</a></td>\\n\\t<td class=\\\" \\\">Nicolas Delhomme</td>\\n\\t<td class=\\\" \\\">Count summarization and normalization for RNA-Seq data</td>\\n\\t<td class=\\\"  sorting_1\\\">1305</td>\\n</tr><tr id=\\\"pkg_GeneMeta\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/GeneMeta.html\\\">GeneMeta</a></td>\\n\\t<td class=\\\" \\\">Bioconductor Package Maintainer</td>\\n\\t<td class=\\\" \\\">MetaAnalysis for High Throughput Experiments</td>\\n\\t<td class=\\\"  sorting_1\\\">1305</td>\\n</tr><tr id=\\\"pkg_gespeR\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/gespeR.html\\\">gespeR</a></td>\\n\\t<td class=\\\" \\\">Fabian Schmich</td>\\n\\t<td class=\\\" \\\">Gene-Specific Phenotype EstimatoR</td>\\n\\t<td class=\\\"  sorting_1\\\">1305</td>\\n</tr><tr id=\\\"pkg_LymphoSeq\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/LymphoSeq.html\\\">LymphoSeq</a></td>\\n\\t<td class=\\\" \\\">David Coffey</td>\\n\\t<td class=\\\" \\\">Analyze high-throughput sequencing of T and B cell receptors</td>\\n\\t<td class=\\\"  sorting_1\\\">1305</td>\\n</tr><tr id=\\\"pkg_MAIT\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/MAIT.html\\\">MAIT</a></td>\\n\\t<td class=\\\" \\\">Pol Sola-Santos</td>\\n\\t<td class=\\\" \\\">Statistical Analysis of Metabolomic Data</td>\\n\\t<td class=\\\"  sorting_1\\\">1305</td>\\n</tr><tr id=\\\"pkg_methrix\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/methrix.html\\\">methrix</a></td>\\n\\t<td class=\\\" \\\">Anand Mayakonda</td>\\n\\t<td class=\\\" \\\">Fast and efficient summarization of generic bedGraph files from Bisufite sequencing</td>\\n\\t<td class=\\\"  sorting_1\\\">1305</td>\\n</tr><tr id=\\\"pkg_panelcn.mops\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/panelcn.mops.html\\\">panelcn.mops</a></td>\\n\\t<td class=\\\" \\\">Gundula Povysil</td>\\n\\t<td class=\\\" \\\">CNV detection tool for targeted NGS panel data</td>\\n\\t<td class=\\\"  sorting_1\\\">1305</td>\\n</tr><tr id=\\\"pkg_RNAAgeCalc\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/RNAAgeCalc.html\\\">RNAAgeCalc</a></td>\\n\\t<td class=\\\" \\\">Xu Ren</td>\\n\\t<td class=\\\" \\\">A multi-tissue transcriptional age calculator</td>\\n\\t<td class=\\\"  sorting_1\\\">1305</td>\\n</tr><tr id=\\\"pkg_timescape\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/timescape.html\\\">timescape</a></td>\\n\\t<td class=\\\" \\\">Maia Smith</td>\\n\\t<td class=\\\" \\\">Patient Clonal Timescapes</td>\\n\\t<td class=\\\"  sorting_1\\\">1305</td>\\n</tr><tr id=\\\"pkg_TitanCNA\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/TitanCNA.html\\\">TitanCNA</a></td>\\n\\t<td class=\\\" \\\">Gavin Ha</td>\\n\\t<td class=\\\" \\\">Subclonal copy number and LOH prediction from whole genome sequencing of tumours</td>\\n\\t<td class=\\\"  sorting_1\\\">1305</td>\\n</tr><tr id=\\\"pkg_a4\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/a4.html\\\">a4</a></td>\\n\\t<td class=\\\" \\\">Laure Cougnaud</td>\\n\\t<td class=\\\" \\\">Automated Affymetrix Array Analysis Umbrella Package</td>\\n\\t<td class=\\\"  sorting_1\\\">1318</td>\\n</tr><tr id=\\\"pkg_AllelicImbalance\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/AllelicImbalance.html\\\">AllelicImbalance</a></td>\\n\\t<td class=\\\" \\\">Jesper R Gadin</td>\\n\\t<td class=\\\" \\\">Investigates Allele Specific Expression</td>\\n\\t<td class=\\\"  sorting_1\\\">1318</td>\\n</tr><tr id=\\\"pkg_amplican\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/amplican.html\\\">amplican</a></td>\\n\\t<td class=\\\" \\\">Eivind Valen</td>\\n\\t<td class=\\\" \\\">Automated analysis of CRISPR experiments</td>\\n\\t<td class=\\\"  sorting_1\\\">1318</td>\\n</tr><tr id=\\\"pkg_BiocSklearn\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/BiocSklearn.html\\\">BiocSklearn</a></td>\\n\\t<td class=\\\" \\\">Vince Carey</td>\\n\\t<td class=\\\" \\\">interface to python sklearn via Rstudio reticulate</td>\\n\\t<td class=\\\"  sorting_1\\\">1318</td>\\n</tr><tr id=\\\"pkg_farms\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/farms.html\\\">farms</a></td>\\n\\t<td class=\\\" \\\">Djork-Arne Clevert</td>\\n\\t<td class=\\\" \\\">FARMS - Factor Analysis for Robust Microarray Summarization</td>\\n\\t<td class=\\\"  sorting_1\\\">1318</td>\\n</tr><tr id=\\\"pkg_ggspavis\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ggspavis.html\\\">ggspavis</a></td>\\n\\t<td class=\\\" \\\">Lukas M. Weber</td>\\n\\t<td class=\\\" \\\">Visualization functions for spatially resolved transcriptomics data</td>\\n\\t<td class=\\\"  sorting_1\\\">1318</td>\\n</tr><tr id=\\\"pkg_h5vc\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/h5vc.html\\\">h5vc</a></td>\\n\\t<td class=\\\" \\\">Paul Theodor Pyl</td>\\n\\t<td class=\\\" \\\">Managing alignment tallies using a hdf5 backend</td>\\n\\t<td class=\\\"  sorting_1\\\">1318</td>\\n</tr><tr id=\\\"pkg_mCSEA\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/mCSEA.html\\\">mCSEA</a></td>\\n\\t<td class=\\\" \\\">Jordi Martorell-Marugn</td>\\n\\t<td class=\\\" \\\">Methylated CpGs Set Enrichment Analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">1318</td>\\n</tr><tr id=\\\"pkg_MEAL\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/MEAL.html\\\">MEAL</a></td>\\n\\t<td class=\\\" \\\">Xavier Escrib Montagut</td>\\n\\t<td class=\\\" \\\">Perform methylation analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">1318</td>\\n</tr><tr id=\\\"pkg_MetaNeighbor\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/MetaNeighbor.html\\\">MetaNeighbor</a></td>\\n\\t<td class=\\\" \\\">Stephan Fischer</td>\\n\\t<td class=\\\" \\\">Single cell replicability analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">1318</td>\\n</tr><tr id=\\\"pkg_MicrobiomeProfiler\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/MicrobiomeProfiler.html\\\">MicrobiomeProfiler</a></td>\\n\\t<td class=\\\" \\\">Meijun Chen</td>\\n\\t<td class=\\\" \\\">An R/shiny package for microbiome functional enrichment analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">1318</td>\\n</tr><tr id=\\\"pkg_MsBackendMassbank\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/MsBackendMassbank.html\\\">MsBackendMassbank</a></td>\\n\\t<td class=\\\" \\\">RforMassSpectrometry Package Maintainer</td>\\n\\t<td class=\\\" \\\">Mass Spectrometry Data Backend for MassBank record Files</td>\\n\\t<td class=\\\"  sorting_1\\\">1318</td>\\n</tr><tr id=\\\"pkg_msPurity\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/msPurity.html\\\">msPurity</a></td>\\n\\t<td class=\\\" \\\">Thomas N. Lawson</td>\\n\\t<td class=\\\" \\\">Automated Evaluation of Precursor Ion Purity for Mass Spectrometry Based Fragmentation in Metabolomics</td>\\n\\t<td class=\\\"  sorting_1\\\">1318</td>\\n</tr><tr id=\\\"pkg_ndexr\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ndexr.html\\\">ndexr</a></td>\\n\\t<td class=\\\" \\\">Florian Auer</td>\\n\\t<td class=\\\" \\\">NDEx R client library</td>\\n\\t<td class=\\\"  sorting_1\\\">1318</td>\\n</tr><tr id=\\\"pkg_scAnnotatR\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/scAnnotatR.html\\\">scAnnotatR</a></td>\\n\\t<td class=\\\" \\\">Johannes Griss</td>\\n\\t<td class=\\\" \\\">Pretrained learning models for cell type prediction on single cell RNA-sequencing data</td>\\n\\t<td class=\\\"  sorting_1\\\">1318</td>\\n</tr><tr id=\\\"pkg_scClassify\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/scClassify.html\\\">scClassify</a></td>\\n\\t<td class=\\\" \\\">Yingxin Lin</td>\\n\\t<td class=\\\" \\\">scClassify: single-cell Hierarchical Classification</td>\\n\\t<td class=\\\"  sorting_1\\\">1318</td>\\n</tr><tr id=\\\"pkg_affycomp\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/affycomp.html\\\">affycomp</a></td>\\n\\t<td class=\\\" \\\">Rafael A. Irizarry</td>\\n\\t<td class=\\\" \\\">Graphics Toolbox for Assessment of Affymetrix Expression Measures</td>\\n\\t<td class=\\\"  sorting_1\\\">1334</td>\\n</tr><tr id=\\\"pkg_AffyCompatible\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/AffyCompatible.html\\\">AffyCompatible</a></td>\\n\\t<td class=\\\" \\\">Martin Morgan</td>\\n\\t<td class=\\\" \\\">Affymetrix GeneChip software compatibility</td>\\n\\t<td class=\\\"  sorting_1\\\">1334</td>\\n</tr><tr id=\\\"pkg_Cepo\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/Cepo.html\\\">Cepo</a></td>\\n\\t<td class=\\\" \\\">Hani Jieun Kim</td>\\n\\t<td class=\\\" \\\">Cepo for the identification of differentially stable genes</td>\\n\\t<td class=\\\"  sorting_1\\\">1334</td>\\n</tr><tr id=\\\"pkg_ClassifyR\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ClassifyR.html\\\">ClassifyR</a></td>\\n\\t<td class=\\\" \\\">Dario Strbenac</td>\\n\\t<td class=\\\" \\\">A framework for cross-validated classification problems, with applications to differential variability and differential distribution testing</td>\\n\\t<td class=\\\"  sorting_1\\\">1334</td>\\n</tr><tr id=\\\"pkg_CNORode\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/CNORode.html\\\">CNORode</a></td>\\n\\t<td class=\\\" \\\">Attila Gabor</td>\\n\\t<td class=\\\" \\\">ODE add-on to CellNOptR</td>\\n\\t<td class=\\\"  sorting_1\\\">1334</td>\\n</tr><tr id=\\\"pkg_DiscoRhythm\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/DiscoRhythm.html\\\">DiscoRhythm</a></td>\\n\\t<td class=\\\" \\\">Matthew Carlucci</td>\\n\\t<td class=\\\" \\\">Interactive Workflow for Discovering Rhythmicity in Biological Data</td>\\n\\t<td class=\\\"  sorting_1\\\">1334</td>\\n</tr><tr id=\\\"pkg_enrichTF\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/enrichTF.html\\\">enrichTF</a></td>\\n\\t<td class=\\\" \\\">Zheng Wei</td>\\n\\t<td class=\\\" \\\">Transcription Factors Enrichment Analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">1334</td>\\n</tr><tr id=\\\"pkg_esetVis\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/esetVis.html\\\">esetVis</a></td>\\n\\t<td class=\\\" \\\">Laure Cougnaud</td>\\n\\t<td class=\\\" \\\">Visualizations of expressionSet Bioconductor object</td>\\n\\t<td class=\\\"  sorting_1\\\">1334</td>\\n</tr><tr id=\\\"pkg_GeoDiff\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/GeoDiff.html\\\">GeoDiff</a></td>\\n\\t<td class=\\\" \\\">Nicole Ortogero</td>\\n\\t<td class=\\\" \\\">Count model based differential expression and normalization on GeoMx RNA data</td>\\n\\t<td class=\\\"  sorting_1\\\">1334</td>\\n</tr><tr id=\\\"pkg_hiAnnotator\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/hiAnnotator.html\\\">hiAnnotator</a></td>\\n\\t<td class=\\\" \\\">Nirav V Malani</td>\\n\\t<td class=\\\" \\\">Functions for annotating GRanges objects</td>\\n\\t<td class=\\\"  sorting_1\\\">1334</td>\\n</tr><tr id=\\\"pkg_PepsNMR\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/PepsNMR.html\\\">PepsNMR</a></td>\\n\\t<td class=\\\" \\\">Manon Martin</td>\\n\\t<td class=\\\" \\\">Pre-process 1H-NMR FID signals</td>\\n\\t<td class=\\\"  sorting_1\\\">1334</td>\\n</tr><tr id=\\\"pkg_pqsfinder\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/pqsfinder.html\\\">pqsfinder</a></td>\\n\\t<td class=\\\" \\\">Jiri Hon</td>\\n\\t<td class=\\\" \\\">Identification of potential quadruplex forming sequences</td>\\n\\t<td class=\\\"  sorting_1\\\">1334</td>\\n</tr><tr id=\\\"pkg_rexposome\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/rexposome.html\\\">rexposome</a></td>\\n\\t<td class=\\\" \\\">Xavier Escrib Montagut</td>\\n\\t<td class=\\\" \\\">Exposome exploration and outcome data analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">1334</td>\\n</tr><tr id=\\\"pkg_signeR\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/signeR.html\\\">signeR</a></td>\\n\\t<td class=\\\" \\\">Renan Valieris</td>\\n\\t<td class=\\\" \\\">Empirical Bayesian approach to mutational signature discovery</td>\\n\\t<td class=\\\"  sorting_1\\\">1334</td>\\n</tr><tr id=\\\"pkg_structToolbox\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/structToolbox.html\\\">structToolbox</a></td>\\n\\t<td class=\\\" \\\">Gavin Rhys Lloyd</td>\\n\\t<td class=\\\" \\\">Data processing &amp; analysis tools for Metabolomics and other omics</td>\\n\\t<td class=\\\"  sorting_1\\\">1334</td>\\n</tr><tr id=\\\"pkg_a4Reporting\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/a4Reporting.html\\\">a4Reporting</a></td>\\n\\t<td class=\\\" \\\">Laure Cougnaud</td>\\n\\t<td class=\\\" \\\">Automated Affymetrix Array Analysis Reporting Package</td>\\n\\t<td class=\\\"  sorting_1\\\">1349</td>\\n</tr><tr id=\\\"pkg_acde\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/acde.html\\\">acde</a></td>\\n\\t<td class=\\\" \\\">Juan Pablo Acosta</td>\\n\\t<td class=\\\" \\\">Artificial Components Detection of Differentially Expressed Genes</td>\\n\\t<td class=\\\"  sorting_1\\\">1349</td>\\n</tr><tr id=\\\"pkg_AlpsNMR\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/AlpsNMR.html\\\">AlpsNMR</a></td>\\n\\t<td class=\\\" \\\">Sergio Oller Moreno</td>\\n\\t<td class=\\\" \\\">Automated spectraL Processing System for NMR</td>\\n\\t<td class=\\\"  sorting_1\\\">1349</td>\\n</tr><tr id=\\\"pkg_coMET\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/coMET.html\\\">coMET</a></td>\\n\\t<td class=\\\" \\\">Tiphaine Martin</td>\\n\\t<td class=\\\" \\\">coMET: visualisation of regional epigenome-wide association scan (EWAS) results and DNA co-methylation patterns</td>\\n\\t<td class=\\\"  sorting_1\\\">1349</td>\\n</tr><tr id=\\\"pkg_crisprBowtie\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/crisprBowtie.html\\\">crisprBowtie</a></td>\\n\\t<td class=\\\" \\\">Jean-Philippe Fortin</td>\\n\\t<td class=\\\" \\\">Bowtie-based alignment of CRISPR gRNA spacer sequences</td>\\n\\t<td class=\\\"  sorting_1\\\">1349</td>\\n</tr><tr id=\\\"pkg_crisprScore\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/crisprScore.html\\\">crisprScore</a></td>\\n\\t<td class=\\\" \\\">Jean-Philippe Fortin</td>\\n\\t<td class=\\\" \\\">On-Target and Off-Target Scoring Algorithms for CRISPR gRNAs</td>\\n\\t<td class=\\\"  sorting_1\\\">1349</td>\\n</tr><tr id=\\\"pkg_customProDB\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/customProDB.html\\\">customProDB</a></td>\\n\\t<td class=\\\" \\\">Xiaojing Wang Bo Wen</td>\\n\\t<td class=\\\" \\\">Generate customized protein database from NGS data, with a focus on RNA-Seq data, for proteomics search</td>\\n\\t<td class=\\\"  sorting_1\\\">1349</td>\\n</tr><tr id=\\\"pkg_CytoTree\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/CytoTree.html\\\">CytoTree</a></td>\\n\\t<td class=\\\" \\\">Yuting Dai</td>\\n\\t<td class=\\\" \\\">A Toolkit for Flow And Mass Cytometry Data</td>\\n\\t<td class=\\\"  sorting_1\\\">1349</td>\\n</tr><tr id=\\\"pkg_DaMiRseq\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/DaMiRseq.html\\\">DaMiRseq</a></td>\\n\\t<td class=\\\" \\\">Mattia Chiesa</td>\\n\\t<td class=\\\" \\\">Data Mining for RNA-seq data: normalization, feature selection and classification</td>\\n\\t<td class=\\\"  sorting_1\\\">1349</td>\\n</tr><tr id=\\\"pkg_HiCDCPlus\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/HiCDCPlus.html\\\">HiCDCPlus</a></td>\\n\\t<td class=\\\" \\\">Merve Sahin</td>\\n\\t<td class=\\\" \\\">Hi-C Direct Caller Plus</td>\\n\\t<td class=\\\"  sorting_1\\\">1349</td>\\n</tr><tr id=\\\"pkg_hmdbQuery\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/hmdbQuery.html\\\">hmdbQuery</a></td>\\n\\t<td class=\\\" \\\">VJ Carey</td>\\n\\t<td class=\\\" \\\">utilities for exploration of human metabolome database</td>\\n\\t<td class=\\\"  sorting_1\\\">1349</td>\\n</tr><tr id=\\\"pkg_MesKit\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/MesKit.html\\\">MesKit</a></td>\\n\\t<td class=\\\" \\\">Mengni Liu</td>\\n\\t<td class=\\\" \\\">A tool kit for dissecting cancer evolution from multi-region derived tumor biopsies via somatic alterations</td>\\n\\t<td class=\\\"  sorting_1\\\">1349</td>\\n</tr><tr id=\\\"pkg_PPInfer\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/PPInfer.html\\\">PPInfer</a></td>\\n\\t<td class=\\\" \\\">Dongmin Jung</td>\\n\\t<td class=\\\" \\\">Inferring functionally related proteins using protein interaction networks</td>\\n\\t<td class=\\\"  sorting_1\\\">1349</td>\\n</tr><tr id=\\\"pkg_PSMatch\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/PSMatch.html\\\">PSMatch</a></td>\\n\\t<td class=\\\" \\\">Laurent Gatto</td>\\n\\t<td class=\\\" \\\">Handling and Managing Peptide Spectrum Matches</td>\\n\\t<td class=\\\"  sorting_1\\\">1349</td>\\n</tr><tr id=\\\"pkg_rcellminer\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/rcellminer.html\\\">rcellminer</a></td>\\n\\t<td class=\\\" \\\">Augustin Luna, Vinodh Rajapakse, Fathi Elloumi</td>\\n\\t<td class=\\\" \\\">rcellminer: Molecular Profiles, Drug Response, and Chemical Structures for the NCI-60 Cell Lines</td>\\n\\t<td class=\\\"  sorting_1\\\">1349</td>\\n</tr><tr id=\\\"pkg_ReactomeContentService4R\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ReactomeContentService4R.html\\\">ReactomeContentService4R</a></td>\\n\\t<td class=\\\" \\\">Chi-Lam Poon</td>\\n\\t<td class=\\\" \\\">Interface for the Reactome Content Service</td>\\n\\t<td class=\\\"  sorting_1\\\">1349</td>\\n</tr><tr id=\\\"pkg_RGSEA\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/RGSEA.html\\\">RGSEA</a></td>\\n\\t<td class=\\\" \\\">Chengcheng Ma</td>\\n\\t<td class=\\\" \\\">Random Gene Set Enrichment Analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">1349</td>\\n</tr><tr id=\\\"pkg_scp\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/scp.html\\\">scp</a></td>\\n\\t<td class=\\\" \\\">Christophe Vanderaa</td>\\n\\t<td class=\\\" \\\">Mass Spectrometry-Based Single-Cell Proteomics Data Analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">1349</td>\\n</tr><tr id=\\\"pkg_scPCA\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/scPCA.html\\\">scPCA</a></td>\\n\\t<td class=\\\" \\\">Philippe Boileau</td>\\n\\t<td class=\\\" \\\">Sparse Contrastive Principal Component Analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">1349</td>\\n</tr><tr id=\\\"pkg_artMS\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/artMS.html\\\">artMS</a></td>\\n\\t<td class=\\\" \\\">David Jimenez-Morales</td>\\n\\t<td class=\\\" \\\">Analytical R tools for Mass Spectrometry</td>\\n\\t<td class=\\\"  sorting_1\\\">1368</td>\\n</tr><tr id=\\\"pkg_BEclear\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/BEclear.html\\\">BEclear</a></td>\\n\\t<td class=\\\" \\\">David Rasp</td>\\n\\t<td class=\\\" \\\">Correction of batch effects in DNA methylation data</td>\\n\\t<td class=\\\"  sorting_1\\\">1368</td>\\n</tr><tr id=\\\"pkg_ccmap\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ccmap.html\\\">ccmap</a></td>\\n\\t<td class=\\\" \\\">Alex Pickering</td>\\n\\t<td class=\\\" \\\">Combination Connectivity Mapping</td>\\n\\t<td class=\\\"  sorting_1\\\">1368</td>\\n</tr><tr id=\\\"pkg_crossmeta\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/crossmeta.html\\\">crossmeta</a></td>\\n\\t<td class=\\\" \\\">Alex Pickering</td>\\n\\t<td class=\\\" \\\">Cross Platform Meta-Analysis of Microarray Data</td>\\n\\t<td class=\\\"  sorting_1\\\">1368</td>\\n</tr><tr id=\\\"pkg_dcanr\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/dcanr.html\\\">dcanr</a></td>\\n\\t<td class=\\\" \\\">Dharmesh D. Bhuva</td>\\n\\t<td class=\\\" \\\">Differential co-expression/association network analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">1368</td>\\n</tr><tr id=\\\"pkg_epiNEM\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/epiNEM.html\\\">epiNEM</a></td>\\n\\t<td class=\\\" \\\">Martin Pirkl</td>\\n\\t<td class=\\\" \\\">epiNEM</td>\\n\\t<td class=\\\"  sorting_1\\\">1368</td>\\n</tr><tr id=\\\"pkg_MACSr\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/MACSr.html\\\">MACSr</a></td>\\n\\t<td class=\\\" \\\">Qiang Hu</td>\\n\\t<td class=\\\" \\\">MACS: Model-based Analysis for ChIP-Seq</td>\\n\\t<td class=\\\"  sorting_1\\\">1368</td>\\n</tr><tr id=\\\"pkg_MLP\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/MLP.html\\\">MLP</a></td>\\n\\t<td class=\\\" \\\">Tobias Verbeke</td>\\n\\t<td class=\\\" \\\">Mean Log P Analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">1368</td>\\n</tr><tr id=\\\"pkg_myvariant\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/myvariant.html\\\">myvariant</a></td>\\n\\t<td class=\\\" \\\">Adam Mark, Chunlei Wu</td>\\n\\t<td class=\\\" \\\">Accesses MyVariant.info variant query and annotation services</td>\\n\\t<td class=\\\"  sorting_1\\\">1368</td>\\n</tr><tr id=\\\"pkg_ramwas\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ramwas.html\\\">ramwas</a></td>\\n\\t<td class=\\\" \\\">Andrey A Shabalin</td>\\n\\t<td class=\\\" \\\">Fast Methylome-Wide Association Study Pipeline for Enrichment Platforms</td>\\n\\t<td class=\\\"  sorting_1\\\">1368</td>\\n</tr><tr id=\\\"pkg_RefPlus\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/RefPlus.html\\\">RefPlus</a></td>\\n\\t<td class=\\\" \\\">Kai-Ming Chang</td>\\n\\t<td class=\\\" \\\">A function set for the Extrapolation Strategy (RMA+) and Extrapolation Averaging (RMA++) methods.</td>\\n\\t<td class=\\\"  sorting_1\\\">1368</td>\\n</tr><tr id=\\\"pkg_ssize\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ssize.html\\\">ssize</a></td>\\n\\t<td class=\\\" \\\">Gregory R. Warnes</td>\\n\\t<td class=\\\" \\\">Estimate Microarray Sample Size</td>\\n\\t<td class=\\\"  sorting_1\\\">1368</td>\\n</tr><tr id=\\\"pkg_BiSeq\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/BiSeq.html\\\">BiSeq</a></td>\\n\\t<td class=\\\" \\\">Katja Hebestreit</td>\\n\\t<td class=\\\" \\\">Processing and analyzing bisulfite sequencing data</td>\\n\\t<td class=\\\"  sorting_1\\\">1380</td>\\n</tr><tr id=\\\"pkg_celaref\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/celaref.html\\\">celaref</a></td>\\n\\t<td class=\\\" \\\">Sarah Williams</td>\\n\\t<td class=\\\" \\\">Single-cell RNAseq cell cluster labelling by reference</td>\\n\\t<td class=\\\"  sorting_1\\\">1380</td>\\n</tr><tr id=\\\"pkg_EmpiricalBrownsMethod\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/EmpiricalBrownsMethod.html\\\">EmpiricalBrownsMethod</a></td>\\n\\t<td class=\\\" \\\">David Gibbs</td>\\n\\t<td class=\\\" \\\">Uses Brown's method to combine p-values from dependent tests</td>\\n\\t<td class=\\\"  sorting_1\\\">1380</td>\\n</tr><tr id=\\\"pkg_FGNet\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/FGNet.html\\\">FGNet</a></td>\\n\\t<td class=\\\" \\\">Sara Aibar</td>\\n\\t<td class=\\\" \\\">Functional Gene Networks derived from biological enrichment analyses</td>\\n\\t<td class=\\\"  sorting_1\\\">1380</td>\\n</tr><tr id=\\\"pkg_GSEAlm\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/GSEAlm.html\\\">GSEAlm</a></td>\\n\\t<td class=\\\" \\\">Assaf Oron</td>\\n\\t<td class=\\\" \\\">Linear Model Toolset for Gene Set Enrichment Analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">1380</td>\\n</tr><tr id=\\\"pkg_InterCellar\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/InterCellar.html\\\">InterCellar</a></td>\\n\\t<td class=\\\" \\\">Marta Interlandi</td>\\n\\t<td class=\\\" \\\">InterCellar: an R-Shiny app for interactive analysis and exploration of cell-cell communication in single-cell transcriptomics</td>\\n\\t<td class=\\\"  sorting_1\\\">1380</td>\\n</tr><tr id=\\\"pkg_phantasus\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/phantasus.html\\\">phantasus</a></td>\\n\\t<td class=\\\" \\\">Alexey Sergushichev</td>\\n\\t<td class=\\\" \\\">Visual and interactive gene expression analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">1380</td>\\n</tr><tr id=\\\"pkg_pRolocGUI\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/pRolocGUI.html\\\">pRolocGUI</a></td>\\n\\t<td class=\\\" \\\">Lisa Breckels</td>\\n\\t<td class=\\\" \\\">Interactive visualisation of spatial proteomics data</td>\\n\\t<td class=\\\"  sorting_1\\\">1380</td>\\n</tr><tr id=\\\"pkg_rCGH\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/rCGH.html\\\">rCGH</a></td>\\n\\t<td class=\\\" \\\">Frederic Commo</td>\\n\\t<td class=\\\" \\\">Comprehensive Pipeline for Analyzing and Visualizing Array-Based CGH Data</td>\\n\\t<td class=\\\"  sorting_1\\\">1380</td>\\n</tr><tr id=\\\"pkg_SPsimSeq\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/SPsimSeq.html\\\">SPsimSeq</a></td>\\n\\t<td class=\\\" \\\">Joris Meys</td>\\n\\t<td class=\\\" \\\">Semi-parametric simulation tool for bulk and single-cell RNA sequencing data</td>\\n\\t<td class=\\\"  sorting_1\\\">1380</td>\\n</tr><tr id=\\\"pkg_TRONCO\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/TRONCO.html\\\">TRONCO</a></td>\\n\\t<td class=\\\" \\\">Luca De Sano</td>\\n\\t<td class=\\\" \\\">TRONCO, an R package for TRanslational ONCOlogy</td>\\n\\t<td class=\\\"  sorting_1\\\">1380</td>\\n</tr><tr id=\\\"pkg_ACE\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ACE.html\\\">ACE</a></td>\\n\\t<td class=\\\" \\\">Jos B Poell</td>\\n\\t<td class=\\\" \\\">Absolute Copy Number Estimation from Low-coverage Whole Genome Sequencing</td>\\n\\t<td class=\\\"  sorting_1\\\">1391</td>\\n</tr><tr id=\\\"pkg_biosigner\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/biosigner.html\\\">biosigner</a></td>\\n\\t<td class=\\\" \\\">Etienne A. Thevenot</td>\\n\\t<td class=\\\" \\\">Signature discovery from omics data</td>\\n\\t<td class=\\\"  sorting_1\\\">1391</td>\\n</tr><tr id=\\\"pkg_DMRcaller\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/DMRcaller.html\\\">DMRcaller</a></td>\\n\\t<td class=\\\" \\\">Nicolae Radu Zabet</td>\\n\\t<td class=\\\" \\\">Differentially Methylated Regions caller</td>\\n\\t<td class=\\\"  sorting_1\\\">1391</td>\\n</tr><tr id=\\\"pkg_erma\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/erma.html\\\">erma</a></td>\\n\\t<td class=\\\" \\\">VJ Carey</td>\\n\\t<td class=\\\" \\\">epigenomic road map adventures</td>\\n\\t<td class=\\\"  sorting_1\\\">1391</td>\\n</tr><tr id=\\\"pkg_FamAgg\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/FamAgg.html\\\">FamAgg</a></td>\\n\\t<td class=\\\" \\\">Johannes Rainer</td>\\n\\t<td class=\\\" \\\">Pedigree Analysis and Familial Aggregation</td>\\n\\t<td class=\\\"  sorting_1\\\">1391</td>\\n</tr><tr id=\\\"pkg_Herper\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/Herper.html\\\">Herper</a></td>\\n\\t<td class=\\\" \\\">Thomas Carroll</td>\\n\\t<td class=\\\" \\\">The Herper package is a simple toolset to install and manage conda packages and environments from R</td>\\n\\t<td class=\\\"  sorting_1\\\">1391</td>\\n</tr><tr id=\\\"pkg_Pigengene\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/Pigengene.html\\\">Pigengene</a></td>\\n\\t<td class=\\\" \\\">Habil Zare</td>\\n\\t<td class=\\\" \\\">Infers biological signatures from gene expression data</td>\\n\\t<td class=\\\"  sorting_1\\\">1391</td>\\n</tr><tr id=\\\"pkg_SpatialCPie\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/SpatialCPie.html\\\">SpatialCPie</a></td>\\n\\t<td class=\\\" \\\">Joseph Bergenstraahle</td>\\n\\t<td class=\\\" \\\">Cluster analysis of Spatial Transcriptomics data</td>\\n\\t<td class=\\\"  sorting_1\\\">1391</td>\\n</tr><tr id=\\\"pkg_TPP\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/TPP.html\\\">TPP</a></td>\\n\\t<td class=\\\" \\\">Dorothee Childs</td>\\n\\t<td class=\\\" \\\">Analyze thermal proteome profiling (TPP) experiments</td>\\n\\t<td class=\\\"  sorting_1\\\">1391</td>\\n</tr><tr id=\\\"pkg_ACME\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ACME.html\\\">ACME</a></td>\\n\\t<td class=\\\" \\\">Sean Davis</td>\\n\\t<td class=\\\" \\\">Algorithms for Calculating Microarray Enrichment (ACME)</td>\\n\\t<td class=\\\"  sorting_1\\\">1400</td>\\n</tr><tr id=\\\"pkg_ASSIGN\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ASSIGN.html\\\">ASSIGN</a></td>\\n\\t<td class=\\\" \\\">Ying Shen, W. Evan Johnson, David Jenkins, Mumtehena Rahman</td>\\n\\t<td class=\\\" \\\">Adaptive Signature Selection and InteGratioN (ASSIGN)</td>\\n\\t<td class=\\\"  sorting_1\\\">1400</td>\\n</tr><tr id=\\\"pkg_cliqueMS\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/cliqueMS.html\\\">cliqueMS</a></td>\\n\\t<td class=\\\" \\\">Oriol Senan Campos</td>\\n\\t<td class=\\\" \\\">Annotation of Isotopes, Adducts and Fragmentation Adducts for in-Source LC/MS Metabolomics Data</td>\\n\\t<td class=\\\"  sorting_1\\\">1400</td>\\n</tr><tr id=\\\"pkg_diffcoexp\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/diffcoexp.html\\\">diffcoexp</a></td>\\n\\t<td class=\\\" \\\">Wenbin Wei</td>\\n\\t<td class=\\\" \\\">Differential Co-expression Analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">1400</td>\\n</tr><tr id=\\\"pkg_easier\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/easier.html\\\">easier</a></td>\\n\\t<td class=\\\" \\\">Oscar Lapuente-Santana</td>\\n\\t<td class=\\\" \\\">Estimate Systems Immune Response from RNA-seq data</td>\\n\\t<td class=\\\"  sorting_1\\\">1400</td>\\n</tr><tr id=\\\"pkg_epivizrServer\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/epivizrServer.html\\\">epivizrServer</a></td>\\n\\t<td class=\\\" \\\">Hector Corrada Bravo</td>\\n\\t<td class=\\\" \\\">WebSocket server infrastructure for epivizr apps and packages</td>\\n\\t<td class=\\\"  sorting_1\\\">1400</td>\\n</tr><tr id=\\\"pkg_flowCut\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/flowCut.html\\\">flowCut</a></td>\\n\\t<td class=\\\" \\\">Justin Meskas</td>\\n\\t<td class=\\\" \\\">Automated Removal of Outlier Events and Flagging of Files Based on Time Versus Fluorescence Analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">1400</td>\\n</tr><tr id=\\\"pkg_HilbertCurve\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/HilbertCurve.html\\\">HilbertCurve</a></td>\\n\\t<td class=\\\" \\\">Zuguang Gu</td>\\n\\t<td class=\\\" \\\">Making 2D Hilbert Curve</td>\\n\\t<td class=\\\"  sorting_1\\\">1400</td>\\n</tr><tr id=\\\"pkg_MetaVolcanoR\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/MetaVolcanoR.html\\\">MetaVolcanoR</a></td>\\n\\t<td class=\\\" \\\">Cesar Prada</td>\\n\\t<td class=\\\" \\\">Gene Expression Meta-analysis Visualization Tool</td>\\n\\t<td class=\\\"  sorting_1\\\">1400</td>\\n</tr><tr id=\\\"pkg_PAIRADISE\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/PAIRADISE.html\\\">PAIRADISE</a></td>\\n\\t<td class=\\\" \\\">Qiang Hu, Levon Demirdjian</td>\\n\\t<td class=\\\" \\\">PAIRADISE: Paired analysis of differential isoform expression</td>\\n\\t<td class=\\\"  sorting_1\\\">1400</td>\\n</tr><tr id=\\\"pkg_SPONGE\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/SPONGE.html\\\">SPONGE</a></td>\\n\\t<td class=\\\" \\\">Markus List</td>\\n\\t<td class=\\\" \\\">Sparse Partial Correlations On Gene Expression</td>\\n\\t<td class=\\\"  sorting_1\\\">1400</td>\\n</tr><tr id=\\\"pkg_ASSET\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ASSET.html\\\">ASSET</a></td>\\n\\t<td class=\\\" \\\">Samsiddhi Bhattacharjee</td>\\n\\t<td class=\\\" \\\">An R package for subset-based association analysis of heterogeneous traits and subtypes</td>\\n\\t<td class=\\\"  sorting_1\\\">1411</td>\\n</tr><tr id=\\\"pkg_CAGEfightR\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/CAGEfightR.html\\\">CAGEfightR</a></td>\\n\\t<td class=\\\" \\\">Malte Thodberg</td>\\n\\t<td class=\\\" \\\">Analysis of Cap Analysis of Gene Expression (CAGE) data using Bioconductor</td>\\n\\t<td class=\\\"  sorting_1\\\">1411</td>\\n</tr><tr id=\\\"pkg_logicFS\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/logicFS.html\\\">logicFS</a></td>\\n\\t<td class=\\\" \\\">Holger Schwender</td>\\n\\t<td class=\\\" \\\">Identification of SNP Interactions</td>\\n\\t<td class=\\\"  sorting_1\\\">1411</td>\\n</tr><tr id=\\\"pkg_phenoTest\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/phenoTest.html\\\">phenoTest</a></td>\\n\\t<td class=\\\" \\\">Evarist Planet</td>\\n\\t<td class=\\\" \\\">Tools to test association between gene expression and phenotype in a way that is efficient, structured, fast and scalable. We also provide tools to do GSEA (Gene set enrichment analysis) and copy number variation.</td>\\n\\t<td class=\\\"  sorting_1\\\">1411</td>\\n</tr><tr id=\\\"pkg_PREDA\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/PREDA.html\\\">PREDA</a></td>\\n\\t<td class=\\\" \\\">Francesco Ferrari</td>\\n\\t<td class=\\\" \\\">Position Related Data Analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">1411</td>\\n</tr><tr id=\\\"pkg_Prostar\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/Prostar.html\\\">Prostar</a></td>\\n\\t<td class=\\\" \\\">Samuel Wieczorek</td>\\n\\t<td class=\\\" \\\">Provides a GUI for DAPAR</td>\\n\\t<td class=\\\"  sorting_1\\\">1411</td>\\n</tr><tr id=\\\"pkg_psichomics\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/psichomics.html\\\">psichomics</a></td>\\n\\t<td class=\\\" \\\">Nuno Saraiva-Agostinho</td>\\n\\t<td class=\\\" \\\">Graphical Interface for Alternative Splicing Quantification, Analysis and Visualisation</td>\\n\\t<td class=\\\"  sorting_1\\\">1411</td>\\n</tr><tr id=\\\"pkg_CNVRanger\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/CNVRanger.html\\\">CNVRanger</a></td>\\n\\t<td class=\\\" \\\">Ludwig Geistlinger</td>\\n\\t<td class=\\\" \\\">Summarization and expression/phenotype association of CNV ranges</td>\\n\\t<td class=\\\"  sorting_1\\\">1418</td>\\n</tr><tr id=\\\"pkg_CODEX\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/CODEX.html\\\">CODEX</a></td>\\n\\t<td class=\\\" \\\">Yuchao Jiang</td>\\n\\t<td class=\\\" \\\">A Normalization and Copy Number Variation Detection Method for Whole Exome Sequencing</td>\\n\\t<td class=\\\"  sorting_1\\\">1418</td>\\n</tr><tr id=\\\"pkg_compcodeR\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/compcodeR.html\\\">compcodeR</a></td>\\n\\t<td class=\\\" \\\">Charlotte Soneson</td>\\n\\t<td class=\\\" \\\">RNAseq data simulation, differential expression analysis and performance comparison of differential expression methods</td>\\n\\t<td class=\\\"  sorting_1\\\">1418</td>\\n</tr><tr id=\\\"pkg_decompTumor2Sig\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/decompTumor2Sig.html\\\">decompTumor2Sig</a></td>\\n\\t<td class=\\\" \\\">Rosario M. Piro</td>\\n\\t<td class=\\\" \\\">Decomposition of individual tumors into mutational signatures by signature refitting</td>\\n\\t<td class=\\\"  sorting_1\\\">1418</td>\\n</tr><tr id=\\\"pkg_DNAshapeR\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/DNAshapeR.html\\\">DNAshapeR</a></td>\\n\\t<td class=\\\" \\\">Tsu-Pei Chiu</td>\\n\\t<td class=\\\" \\\">High-throughput prediction of DNA shape features</td>\\n\\t<td class=\\\"  sorting_1\\\">1418</td>\\n</tr><tr id=\\\"pkg_gmapR\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/gmapR.html\\\">gmapR</a></td>\\n\\t<td class=\\\" \\\">Michael Lawrence</td>\\n\\t<td class=\\\" \\\">An R interface to the GMAP/GSNAP/GSTRUCT suite</td>\\n\\t<td class=\\\"  sorting_1\\\">1418</td>\\n</tr><tr id=\\\"pkg_mnem\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/mnem.html\\\">mnem</a></td>\\n\\t<td class=\\\" \\\">Martin Pirkl</td>\\n\\t<td class=\\\" \\\">Mixture Nested Effects Models</td>\\n\\t<td class=\\\"  sorting_1\\\">1418</td>\\n</tr><tr id=\\\"pkg_ngsReports\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ngsReports.html\\\">ngsReports</a></td>\\n\\t<td class=\\\" \\\">Stephen Pederson</td>\\n\\t<td class=\\\" \\\">Load FastqQC reports and other NGS related files</td>\\n\\t<td class=\\\"  sorting_1\\\">1418</td>\\n</tr><tr id=\\\"pkg_nondetects\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/nondetects.html\\\">nondetects</a></td>\\n\\t<td class=\\\" \\\">Valeriia Sherina</td>\\n\\t<td class=\\\" \\\">Non-detects in qPCR data</td>\\n\\t<td class=\\\"  sorting_1\\\">1418</td>\\n</tr><tr id=\\\"pkg_RMassBank\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/RMassBank.html\\\">RMassBank</a></td>\\n\\t<td class=\\\" \\\">RMassBank at Eawag</td>\\n\\t<td class=\\\" \\\">Workflow to process tandem MS files and build MassBank records</td>\\n\\t<td class=\\\"  sorting_1\\\">1418</td>\\n</tr><tr id=\\\"pkg_satuRn\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/satuRn.html\\\">satuRn</a></td>\\n\\t<td class=\\\" \\\">Jeroen Gilis</td>\\n\\t<td class=\\\" \\\">Scalable Analysis of Differential Transcript Usage for Bulk and Single-Cell RNA-sequencing Applications</td>\\n\\t<td class=\\\"  sorting_1\\\">1418</td>\\n</tr><tr id=\\\"pkg_SCAN.UPC\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/SCAN.UPC.html\\\">SCAN.UPC</a></td>\\n\\t<td class=\\\" \\\">Stephen R. Piccolo</td>\\n\\t<td class=\\\" \\\">Single-channel array normalization (SCAN) and Universal exPression Codes (UPC)</td>\\n\\t<td class=\\\"  sorting_1\\\">1418</td>\\n</tr><tr id=\\\"pkg_tilingArray\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/tilingArray.html\\\">tilingArray</a></td>\\n\\t<td class=\\\" \\\">Zhenyu Xu</td>\\n\\t<td class=\\\" \\\">Transcript mapping with high-density oligonucleotide tiling arrays</td>\\n\\t<td class=\\\"  sorting_1\\\">1418</td>\\n</tr><tr id=\\\"pkg_bayNorm\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/bayNorm.html\\\">bayNorm</a></td>\\n\\t<td class=\\\" \\\">Wenhao Tang</td>\\n\\t<td class=\\\" \\\">Single-cell RNA sequencing data normalization</td>\\n\\t<td class=\\\"  sorting_1\\\">1431</td>\\n</tr><tr id=\\\"pkg_FCBF\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/FCBF.html\\\">FCBF</a></td>\\n\\t<td class=\\\" \\\">Tiago Lubiana</td>\\n\\t<td class=\\\" \\\">Fast Correlation Based Filter for Feature Selection</td>\\n\\t<td class=\\\"  sorting_1\\\">1431</td>\\n</tr><tr id=\\\"pkg_gwasurvivr\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/gwasurvivr.html\\\">gwasurvivr</a></td>\\n\\t<td class=\\\" \\\">Abbas Rizvi</td>\\n\\t<td class=\\\" \\\">gwasurvivr: an R package for genome wide survival analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">1431</td>\\n</tr><tr id=\\\"pkg_rain\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/rain.html\\\">rain</a></td>\\n\\t<td class=\\\" \\\">Paul F. Thaben</td>\\n\\t<td class=\\\" \\\">Rhythmicity Analysis Incorporating Non-parametric Methods</td>\\n\\t<td class=\\\"  sorting_1\\\">1431</td>\\n</tr><tr id=\\\"pkg_SBGNview\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/SBGNview.html\\\">SBGNview</a></td>\\n\\t<td class=\\\" \\\">Weijun Luo</td>\\n\\t<td class=\\\" \\\">\\\"SBGNview: Data Analysis, Integration and Visualization on SBGN Pathways\\\"</td>\\n\\t<td class=\\\"  sorting_1\\\">1431</td>\\n</tr><tr id=\\\"pkg_alevinQC\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/alevinQC.html\\\">alevinQC</a></td>\\n\\t<td class=\\\" \\\">Charlotte Soneson</td>\\n\\t<td class=\\\" \\\">Generate QC Reports For Alevin Output</td>\\n\\t<td class=\\\"  sorting_1\\\">1436</td>\\n</tr><tr id=\\\"pkg_ASpli\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ASpli.html\\\">ASpli</a></td>\\n\\t<td class=\\\" \\\">Estefania Mancini</td>\\n\\t<td class=\\\" \\\">Analysis of Alternative Splicing Using RNA-Seq</td>\\n\\t<td class=\\\"  sorting_1\\\">1436</td>\\n</tr><tr id=\\\"pkg_bigPint\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/bigPint.html\\\">bigPint</a></td>\\n\\t<td class=\\\" \\\">Lindsay Rutter</td>\\n\\t<td class=\\\" \\\">Big multivariate data plotted interactively</td>\\n\\t<td class=\\\"  sorting_1\\\">1436</td>\\n</tr><tr id=\\\"pkg_chimeraviz\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/chimeraviz.html\\\">chimeraviz</a></td>\\n\\t<td class=\\\" \\\">Stian Lgstad</td>\\n\\t<td class=\\\" \\\">Visualization tools for gene fusions</td>\\n\\t<td class=\\\"  sorting_1\\\">1436</td>\\n</tr><tr id=\\\"pkg_chromstaR\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/chromstaR.html\\\">chromstaR</a></td>\\n\\t<td class=\\\" \\\">Aaron Taudt</td>\\n\\t<td class=\\\" \\\">Combinatorial and Differential Chromatin State Analysis for ChIP-Seq Data</td>\\n\\t<td class=\\\"  sorting_1\\\">1436</td>\\n</tr><tr id=\\\"pkg_countsimQC\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/countsimQC.html\\\">countsimQC</a></td>\\n\\t<td class=\\\" \\\">Charlotte Soneson</td>\\n\\t<td class=\\\" \\\">Compare Characteristic Features of Count Data Sets</td>\\n\\t<td class=\\\"  sorting_1\\\">1436</td>\\n</tr><tr id=\\\"pkg_flowFP\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/flowFP.html\\\">flowFP</a></td>\\n\\t<td class=\\\" \\\">Herb Holyst, Wade Rogers</td>\\n\\t<td class=\\\" \\\">Fingerprinting for Flow Cytometry</td>\\n\\t<td class=\\\"  sorting_1\\\">1436</td>\\n</tr><tr id=\\\"pkg_HybridMTest\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/HybridMTest.html\\\">HybridMTest</a></td>\\n\\t<td class=\\\" \\\">Demba Fofana</td>\\n\\t<td class=\\\" \\\">Hybrid Multiple Testing</td>\\n\\t<td class=\\\"  sorting_1\\\">1436</td>\\n</tr><tr id=\\\"pkg_iSEEu\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/iSEEu.html\\\">iSEEu</a></td>\\n\\t<td class=\\\" \\\">Kevin Rue-Albrecht</td>\\n\\t<td class=\\\" \\\">iSEE Universe</td>\\n\\t<td class=\\\"  sorting_1\\\">1436</td>\\n</tr><tr id=\\\"pkg_monaLisa\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/monaLisa.html\\\">monaLisa</a></td>\\n\\t<td class=\\\" \\\">Michael Stadler</td>\\n\\t<td class=\\\" \\\">Binned Motif Enrichment Analysis and Visualization</td>\\n\\t<td class=\\\"  sorting_1\\\">1436</td>\\n</tr><tr id=\\\"pkg_Pi\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/Pi.html\\\">Pi</a></td>\\n\\t<td class=\\\" \\\">Hai Fang</td>\\n\\t<td class=\\\" \\\">Leveraging Genetic Evidence to Prioritise Drug Targets at the Gene and Pathway Level</td>\\n\\t<td class=\\\"  sorting_1\\\">1436</td>\\n</tr><tr id=\\\"pkg_Spaniel\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/Spaniel.html\\\">Spaniel</a></td>\\n\\t<td class=\\\" \\\">Rachel Queen</td>\\n\\t<td class=\\\" \\\">Spatial Transcriptomics Analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">1436</td>\\n</tr><tr id=\\\"pkg_YAPSA\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/YAPSA.html\\\">YAPSA</a></td>\\n\\t<td class=\\\" \\\">Daniel Huebschmann</td>\\n\\t<td class=\\\" \\\">Yet Another Package for Signature Analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">1436</td>\\n</tr><tr id=\\\"pkg_AneuFinder\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/AneuFinder.html\\\">AneuFinder</a></td>\\n\\t<td class=\\\" \\\">Aaron Taudt</td>\\n\\t<td class=\\\" \\\">Analysis of Copy Number Variation in Single-Cell-Sequencing Data</td>\\n\\t<td class=\\\"  sorting_1\\\">1449</td>\\n</tr><tr id=\\\"pkg_dearseq\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/dearseq.html\\\">dearseq</a></td>\\n\\t<td class=\\\" \\\">Boris P. Hejblum</td>\\n\\t<td class=\\\" \\\">Differential Expression Analysis for RNA-seq data through a robust variance component test</td>\\n\\t<td class=\\\"  sorting_1\\\">1449</td>\\n</tr><tr id=\\\"pkg_lisaClust\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/lisaClust.html\\\">lisaClust</a></td>\\n\\t<td class=\\\" \\\">Ellis Patrick</td>\\n\\t<td class=\\\" \\\">lisaClust: Clustering of Local Indicators of Spatial Association</td>\\n\\t<td class=\\\"  sorting_1\\\">1449</td>\\n</tr><tr id=\\\"pkg_multiHiCcompare\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/multiHiCcompare.html\\\">multiHiCcompare</a></td>\\n\\t<td class=\\\" \\\">John Stansfield, Mikhail Dozmorov</td>\\n\\t<td class=\\\" \\\">Normalize and detect differences between Hi-C datasets when replicates of each experimental condition are available</td>\\n\\t<td class=\\\"  sorting_1\\\">1449</td>\\n</tr><tr id=\\\"pkg_scPipe\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/scPipe.html\\\">scPipe</a></td>\\n\\t<td class=\\\" \\\">Shian Su</td>\\n\\t<td class=\\\" \\\">Pipeline for single cell multi-omic data pre-processing</td>\\n\\t<td class=\\\"  sorting_1\\\">1449</td>\\n</tr><tr id=\\\"pkg_goProfiles\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/goProfiles.html\\\">goProfiles</a></td>\\n\\t<td class=\\\" \\\">Alex Sanchez</td>\\n\\t<td class=\\\" \\\">goProfiles: an R package for the statistical analysis of functional profiles</td>\\n\\t<td class=\\\"  sorting_1\\\">1454</td>\\n</tr><tr id=\\\"pkg_GRmetrics\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/GRmetrics.html\\\">GRmetrics</a></td>\\n\\t<td class=\\\" \\\">Nicholas Clark, Mario Medvedovic</td>\\n\\t<td class=\\\" \\\">Calculate growth-rate inhibition (GR) metrics</td>\\n\\t<td class=\\\"  sorting_1\\\">1454</td>\\n</tr><tr id=\\\"pkg_interactiveDisplay\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/interactiveDisplay.html\\\">interactiveDisplay</a></td>\\n\\t<td class=\\\" \\\">Bioconductor Package Maintainer</td>\\n\\t<td class=\\\" \\\">Package for enabling powerful shiny web displays of Bioconductor objects</td>\\n\\t<td class=\\\"  sorting_1\\\">1454</td>\\n</tr><tr id=\\\"pkg_annotationTools\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/annotationTools.html\\\">annotationTools</a></td>\\n\\t<td class=\\\" \\\">Alexandre Kuhn</td>\\n\\t<td class=\\\" \\\">Annotate microarrays and perform cross-species gene expression analyses using flat file databases</td>\\n\\t<td class=\\\"  sorting_1\\\">1457</td>\\n</tr><tr id=\\\"pkg_chopsticks\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/chopsticks.html\\\">chopsticks</a></td>\\n\\t<td class=\\\" \\\">Hin-Tak Leung</td>\\n\\t<td class=\\\" \\\">The 'snp.matrix' and 'X.snp.matrix' Classes</td>\\n\\t<td class=\\\"  sorting_1\\\">1457</td>\\n</tr><tr id=\\\"pkg_chromPlot\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/chromPlot.html\\\">chromPlot</a></td>\\n\\t<td class=\\\" \\\">Karen Y. Orostica</td>\\n\\t<td class=\\\" \\\">Global visualization tool of genomic data</td>\\n\\t<td class=\\\"  sorting_1\\\">1457</td>\\n</tr><tr id=\\\"pkg_mfa\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/mfa.html\\\">mfa</a></td>\\n\\t<td class=\\\" \\\">Kieran Campbell</td>\\n\\t<td class=\\\" \\\">Bayesian hierarchical mixture of factor analyzers for modelling genomic bifurcations</td>\\n\\t<td class=\\\"  sorting_1\\\">1457</td>\\n</tr><tr id=\\\"pkg_multiGSEA\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/multiGSEA.html\\\">multiGSEA</a></td>\\n\\t<td class=\\\" \\\">Sebastian Canzler</td>\\n\\t<td class=\\\" \\\">Combining GSEA-based pathway enrichment with multi omics data integration</td>\\n\\t<td class=\\\"  sorting_1\\\">1457</td>\\n</tr><tr id=\\\"pkg_NanoStringQCPro\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/NanoStringQCPro.html\\\">NanoStringQCPro</a></td>\\n\\t<td class=\\\" \\\">Robert Ziman</td>\\n\\t<td class=\\\" \\\">Quality metrics and data processing methods for NanoString mRNA gene expression data</td>\\n\\t<td class=\\\"  sorting_1\\\">1457</td>\\n</tr><tr id=\\\"pkg_RRHO\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/RRHO.html\\\">RRHO</a></td>\\n\\t<td class=\\\" \\\">Jonathan Rosenblatt</td>\\n\\t<td class=\\\" \\\">Inference on agreement between ordered lists</td>\\n\\t<td class=\\\"  sorting_1\\\">1457</td>\\n</tr><tr id=\\\"pkg_SEtools\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/SEtools.html\\\">SEtools</a></td>\\n\\t<td class=\\\" \\\">Pierre-Luc Germain</td>\\n\\t<td class=\\\" \\\">SEtools: tools for working with SummarizedExperiment</td>\\n\\t<td class=\\\"  sorting_1\\\">1457</td>\\n</tr><tr id=\\\"pkg_shinyMethyl\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/shinyMethyl.html\\\">shinyMethyl</a></td>\\n\\t<td class=\\\" \\\">Jean-Philippe Fortin</td>\\n\\t<td class=\\\" \\\">Interactive visualization for Illumina methylation arrays</td>\\n\\t<td class=\\\"  sorting_1\\\">1457</td>\\n</tr><tr id=\\\"pkg_sparrow\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/sparrow.html\\\">sparrow</a></td>\\n\\t<td class=\\\" \\\">Steve Lianoglou</td>\\n\\t<td class=\\\" \\\">Take command of set enrichment analyses through a unified interface</td>\\n\\t<td class=\\\"  sorting_1\\\">1457</td>\\n</tr><tr id=\\\"pkg_trio\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/trio.html\\\">trio</a></td>\\n\\t<td class=\\\" \\\">Holger Schwender</td>\\n\\t<td class=\\\" \\\">Testing of SNPs and SNP Interactions in Case-Parent Trio Studies</td>\\n\\t<td class=\\\"  sorting_1\\\">1457</td>\\n</tr><tr id=\\\"pkg_tRNA\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/tRNA.html\\\">tRNA</a></td>\\n\\t<td class=\\\" \\\">Felix GM Ernst</td>\\n\\t<td class=\\\" \\\">Analyzing tRNA sequences and structures</td>\\n\\t<td class=\\\"  sorting_1\\\">1457</td>\\n</tr><tr id=\\\"pkg_Chicago\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/Chicago.html\\\">Chicago</a></td>\\n\\t<td class=\\\" \\\">Mikhail Spivakov</td>\\n\\t<td class=\\\" \\\">CHiCAGO: Capture Hi-C Analysis of Genomic Organization</td>\\n\\t<td class=\\\"  sorting_1\\\">1469</td>\\n</tr><tr id=\\\"pkg_clipper\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/clipper.html\\\">clipper</a></td>\\n\\t<td class=\\\" \\\">Paolo Martini</td>\\n\\t<td class=\\\" \\\">Gene Set Analysis Exploiting Pathway Topology</td>\\n\\t<td class=\\\"  sorting_1\\\">1469</td>\\n</tr><tr id=\\\"pkg_consensusSeekeR\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/consensusSeekeR.html\\\">consensusSeekeR</a></td>\\n\\t<td class=\\\" \\\">Astrid Deschenes</td>\\n\\t<td class=\\\" \\\">Detection of consensus regions inside a group of experiences using genomic positions and genomic ranges</td>\\n\\t<td class=\\\"  sorting_1\\\">1469</td>\\n</tr><tr id=\\\"pkg_GDSArray\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/GDSArray.html\\\">GDSArray</a></td>\\n\\t<td class=\\\" \\\">Qian Liu</td>\\n\\t<td class=\\\" \\\">Representing GDS files as array-like objects</td>\\n\\t<td class=\\\"  sorting_1\\\">1469</td>\\n</tr><tr id=\\\"pkg_GenomicDistributions\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/GenomicDistributions.html\\\">GenomicDistributions</a></td>\\n\\t<td class=\\\" \\\">Kristyna Kupkova</td>\\n\\t<td class=\\\" \\\">GenomicDistributions: fast analysis of genomic intervals with Bioconductor</td>\\n\\t<td class=\\\"  sorting_1\\\">1469</td>\\n</tr><tr id=\\\"pkg_keggorthology\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/keggorthology.html\\\">keggorthology</a></td>\\n\\t<td class=\\\" \\\">VJ Carey</td>\\n\\t<td class=\\\" \\\">graph support for KO, KEGG Orthology</td>\\n\\t<td class=\\\"  sorting_1\\\">1469</td>\\n</tr><tr id=\\\"pkg_PhosR\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/PhosR.html\\\">PhosR</a></td>\\n\\t<td class=\\\" \\\">Taiyun Kim</td>\\n\\t<td class=\\\" \\\">A set of methods and tools for comprehensive analysis of phosphoproteomics data</td>\\n\\t<td class=\\\"  sorting_1\\\">1469</td>\\n</tr><tr id=\\\"pkg_PROPER\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/PROPER.html\\\">PROPER</a></td>\\n\\t<td class=\\\" \\\">Hao Wu</td>\\n\\t<td class=\\\" \\\">PROspective Power Evaluation for RNAseq</td>\\n\\t<td class=\\\"  sorting_1\\\">1469</td>\\n</tr><tr id=\\\"pkg_signatureSearch\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/signatureSearch.html\\\">signatureSearch</a></td>\\n\\t<td class=\\\" \\\">Brendan Gongol</td>\\n\\t<td class=\\\" \\\">Environment for Gene Expression Searching Combined with Functional Enrichment Analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">1469</td>\\n</tr><tr id=\\\"pkg_bugsigdbr\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/bugsigdbr.html\\\">bugsigdbr</a></td>\\n\\t<td class=\\\" \\\">Ludwig Geistlinger</td>\\n\\t<td class=\\\" \\\">R-side access to published microbial signatures from BugSigDB</td>\\n\\t<td class=\\\"  sorting_1\\\">1478</td>\\n</tr><tr id=\\\"pkg_FEAST\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/FEAST.html\\\">FEAST</a></td>\\n\\t<td class=\\\" \\\">Kenong Su</td>\\n\\t<td class=\\\" \\\">FEAture SelcTion (FEAST) for Single-cell clustering</td>\\n\\t<td class=\\\"  sorting_1\\\">1478</td>\\n</tr><tr id=\\\"pkg_OmaDB\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/OmaDB.html\\\">OmaDB</a></td>\\n\\t<td class=\\\" \\\">Klara Kaleb, Adrian Altenhoff</td>\\n\\t<td class=\\\" \\\">R wrapper for the OMA REST API</td>\\n\\t<td class=\\\"  sorting_1\\\">1478</td>\\n</tr><tr id=\\\"pkg_rsbml\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/rsbml.html\\\">rsbml</a></td>\\n\\t<td class=\\\" \\\">Michael Lawrence</td>\\n\\t<td class=\\\" \\\">R support for SBML, using libsbml</td>\\n\\t<td class=\\\"  sorting_1\\\">1478</td>\\n</tr><tr id=\\\"pkg_SpeCond\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/SpeCond.html\\\">SpeCond</a></td>\\n\\t<td class=\\\" \\\">Florence Cavalli</td>\\n\\t<td class=\\\" \\\">Condition specific detection from expression data</td>\\n\\t<td class=\\\"  sorting_1\\\">1478</td>\\n</tr><tr id=\\\"pkg_BiRewire\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/BiRewire.html\\\">BiRewire</a></td>\\n\\t<td class=\\\" \\\">Andrea Gobbi</td>\\n\\t<td class=\\\" \\\">High-performing routines for the randomization of a bipartite graph (or a binary event matrix), undirected and directed signed graph preserving degree distribution (or marginal totals)</td>\\n\\t<td class=\\\"  sorting_1\\\">1483</td>\\n</tr><tr id=\\\"pkg_CiteFuse\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/CiteFuse.html\\\">CiteFuse</a></td>\\n\\t<td class=\\\" \\\">Yingxin Lin</td>\\n\\t<td class=\\\" \\\">CiteFuse: multi-modal analysis of CITE-seq data</td>\\n\\t<td class=\\\"  sorting_1\\\">1483</td>\\n</tr><tr id=\\\"pkg_flowVS\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/flowVS.html\\\">flowVS</a></td>\\n\\t<td class=\\\" \\\">Ariful Azad</td>\\n\\t<td class=\\\" \\\">Variance stabilization in flow cytometry (and microarrays)</td>\\n\\t<td class=\\\"  sorting_1\\\">1483</td>\\n</tr><tr id=\\\"pkg_HubPub\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/HubPub.html\\\">HubPub</a></td>\\n\\t<td class=\\\" \\\">Kayla Interdonato</td>\\n\\t<td class=\\\" \\\">Utilities to create and use Bioconductor Hubs</td>\\n\\t<td class=\\\"  sorting_1\\\">1483</td>\\n</tr><tr id=\\\"pkg_hypergraph\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/hypergraph.html\\\">hypergraph</a></td>\\n\\t<td class=\\\" \\\">Bioconductor Package Maintainer</td>\\n\\t<td class=\\\" \\\">A package providing hypergraph data structures</td>\\n\\t<td class=\\\"  sorting_1\\\">1483</td>\\n</tr><tr id=\\\"pkg_meshr\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/meshr.html\\\">meshr</a></td>\\n\\t<td class=\\\" \\\">Koki Tsuyuzaki</td>\\n\\t<td class=\\\" \\\">Tools for conducting enrichment analysis of MeSH</td>\\n\\t<td class=\\\"  sorting_1\\\">1483</td>\\n</tr><tr id=\\\"pkg_ABSSeq\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ABSSeq.html\\\">ABSSeq</a></td>\\n\\t<td class=\\\" \\\">Wentao Yang</td>\\n\\t<td class=\\\" \\\">ABSSeq: a new RNA-Seq analysis method based on modelling absolute expression differences</td>\\n\\t<td class=\\\"  sorting_1\\\">1489</td>\\n</tr><tr id=\\\"pkg_ideal\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ideal.html\\\">ideal</a></td>\\n\\t<td class=\\\" \\\">Federico Marini</td>\\n\\t<td class=\\\" \\\">Interactive Differential Expression AnaLysis</td>\\n\\t<td class=\\\"  sorting_1\\\">1489</td>\\n</tr><tr id=\\\"pkg_Mulcom\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/Mulcom.html\\\">Mulcom</a></td>\\n\\t<td class=\\\" \\\">Claudio Isella</td>\\n\\t<td class=\\\" \\\">Calculates Mulcom test</td>\\n\\t<td class=\\\"  sorting_1\\\">1489</td>\\n</tr><tr id=\\\"pkg_segmentSeq\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/segmentSeq.html\\\">segmentSeq</a></td>\\n\\t<td class=\\\" \\\">Thomas J. Hardcastle</td>\\n\\t<td class=\\\" \\\">Methods for identifying small RNA loci from high-throughput sequencing data</td>\\n\\t<td class=\\\"  sorting_1\\\">1489</td>\\n</tr><tr id=\\\"pkg_SplicingGraphs\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/SplicingGraphs.html\\\">SplicingGraphs</a></td>\\n\\t<td class=\\\" \\\">H. Pags</td>\\n\\t<td class=\\\" \\\">Create, manipulate, visualize splicing graphs, and assign RNA-seq reads to them</td>\\n\\t<td class=\\\"  sorting_1\\\">1489</td>\\n</tr><tr id=\\\"pkg_Structstrings\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/Structstrings.html\\\">Structstrings</a></td>\\n\\t<td class=\\\" \\\">Felix G.M. Ernst</td>\\n\\t<td class=\\\" \\\">Implementation of the dot bracket annotations with Biostrings</td>\\n\\t<td class=\\\"  sorting_1\\\">1489</td>\\n</tr><tr id=\\\"pkg_alpine\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/alpine.html\\\">alpine</a></td>\\n\\t<td class=\\\" \\\">Michael Love</td>\\n\\t<td class=\\\" \\\">alpine</td>\\n\\t<td class=\\\"  sorting_1\\\">1495</td>\\n</tr><tr id=\\\"pkg_ccfindR\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ccfindR.html\\\">ccfindR</a></td>\\n\\t<td class=\\\" \\\">Jun Woo</td>\\n\\t<td class=\\\" \\\">Cancer Clone Finder</td>\\n\\t<td class=\\\"  sorting_1\\\">1495</td>\\n</tr><tr id=\\\"pkg_CrispRVariants\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/CrispRVariants.html\\\">CrispRVariants</a></td>\\n\\t<td class=\\\" \\\">Helen Lindsay</td>\\n\\t<td class=\\\" \\\">Tools for counting and visualising mutations in a target location</td>\\n\\t<td class=\\\"  sorting_1\\\">1495</td>\\n</tr><tr id=\\\"pkg_cydar\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/cydar.html\\\">cydar</a></td>\\n\\t<td class=\\\" \\\">Aaron Lun</td>\\n\\t<td class=\\\" \\\">Using Mass Cytometry for Differential Abundance Analyses</td>\\n\\t<td class=\\\"  sorting_1\\\">1495</td>\\n</tr><tr id=\\\"pkg_FastqCleaner\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/FastqCleaner.html\\\">FastqCleaner</a></td>\\n\\t<td class=\\\" \\\">Leandro Roser</td>\\n\\t<td class=\\\" \\\">A Shiny Application for Quality Control, Filtering and Trimming of FASTQ Files</td>\\n\\t<td class=\\\"  sorting_1\\\">1495</td>\\n</tr><tr id=\\\"pkg_gaia\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/gaia.html\\\">gaia</a></td>\\n\\t<td class=\\\" \\\">S. Morganella</td>\\n\\t<td class=\\\" \\\">GAIA: An R package for genomic analysis of significant chromosomal aberrations.</td>\\n\\t<td class=\\\"  sorting_1\\\">1495</td>\\n</tr><tr id=\\\"pkg_isobar\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/isobar.html\\\">isobar</a></td>\\n\\t<td class=\\\" \\\">Florian P Breitwieser</td>\\n\\t<td class=\\\" \\\">Analysis and quantitation of isobarically tagged MSMS proteomics data</td>\\n\\t<td class=\\\"  sorting_1\\\">1495</td>\\n</tr><tr id=\\\"pkg_MSstatsPTM\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/MSstatsPTM.html\\\">MSstatsPTM</a></td>\\n\\t<td class=\\\" \\\">Devon Kohler</td>\\n\\t<td class=\\\" \\\">Statistical Characterization of Post-translational Modifications</td>\\n\\t<td class=\\\"  sorting_1\\\">1495</td>\\n</tr><tr id=\\\"pkg_spicyR\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/spicyR.html\\\">spicyR</a></td>\\n\\t<td class=\\\" \\\">Ellis Patrick</td>\\n\\t<td class=\\\" \\\">Spatial analysis of in situ cytometry data</td>\\n\\t<td class=\\\"  sorting_1\\\">1495</td>\\n</tr><tr id=\\\"pkg_biocGraph\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/biocGraph.html\\\">biocGraph</a></td>\\n\\t<td class=\\\" \\\">Florian Hahne</td>\\n\\t<td class=\\\" \\\">Graph examples and use cases in Bioinformatics</td>\\n\\t<td class=\\\"  sorting_1\\\">1504</td>\\n</tr><tr id=\\\"pkg_CellNOptR\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/CellNOptR.html\\\">CellNOptR</a></td>\\n\\t<td class=\\\" \\\">Attila Gabor</td>\\n\\t<td class=\\\" \\\">Training of boolean logic models of signalling networks using prior knowledge networks and perturbation data</td>\\n\\t<td class=\\\"  sorting_1\\\">1504</td>\\n</tr><tr id=\\\"pkg_cogena\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/cogena.html\\\">cogena</a></td>\\n\\t<td class=\\\" \\\">Zhilong Jia</td>\\n\\t<td class=\\\" \\\">co-expressed gene-set enrichment analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">1504</td>\\n</tr><tr id=\\\"pkg_derfinderPlot\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/derfinderPlot.html\\\">derfinderPlot</a></td>\\n\\t<td class=\\\" \\\">Leonardo Collado-Torres</td>\\n\\t<td class=\\\" \\\">Plotting functions for derfinder</td>\\n\\t<td class=\\\"  sorting_1\\\">1504</td>\\n</tr><tr id=\\\"pkg_iBBiG\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/iBBiG.html\\\">iBBiG</a></td>\\n\\t<td class=\\\" \\\">Aedin Culhane</td>\\n\\t<td class=\\\" \\\">Iterative Binary Biclustering of Genesets</td>\\n\\t<td class=\\\"  sorting_1\\\">1504</td>\\n</tr><tr id=\\\"pkg_MMUPHin\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/MMUPHin.html\\\">MMUPHin</a></td>\\n\\t<td class=\\\" \\\">Siyuan MA</td>\\n\\t<td class=\\\" \\\">Meta-analysis Methods with Uniform Pipeline for Heterogeneity in Microbiome Studies</td>\\n\\t<td class=\\\"  sorting_1\\\">1504</td>\\n</tr><tr id=\\\"pkg_REDseq\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/REDseq.html\\\">REDseq</a></td>\\n\\t<td class=\\\" \\\">Lihua Julie Zhu</td>\\n\\t<td class=\\\" \\\">Analysis of high-throughput sequencing data processed by restriction enzyme digestion</td>\\n\\t<td class=\\\"  sorting_1\\\">1504</td>\\n</tr><tr id=\\\"pkg_ROntoTools\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ROntoTools.html\\\">ROntoTools</a></td>\\n\\t<td class=\\\" \\\">Calin Voichita</td>\\n\\t<td class=\\\" \\\">R Onto-Tools suite</td>\\n\\t<td class=\\\"  sorting_1\\\">1504</td>\\n</tr><tr id=\\\"pkg_sechm\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/sechm.html\\\">sechm</a></td>\\n\\t<td class=\\\" \\\">Pierre-Luc Germain</td>\\n\\t<td class=\\\" \\\">sechm: Complex Heatmaps from a SummarizedExperiment</td>\\n\\t<td class=\\\"  sorting_1\\\">1504</td>\\n</tr><tr id=\\\"pkg_switchBox\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/switchBox.html\\\">switchBox</a></td>\\n\\t<td class=\\\" \\\">Bahman Afsari, Luigi Marchionni, Wikum Dinalankara</td>\\n\\t<td class=\\\" \\\">Utilities to train and validate classifiers based on pair switching using the K-Top-Scoring-Pair (KTSP) algorithm</td>\\n\\t<td class=\\\"  sorting_1\\\">1504</td>\\n</tr><tr id=\\\"pkg_a4Base\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/a4Base.html\\\">a4Base</a></td>\\n\\t<td class=\\\" \\\">Laure Cougnaud</td>\\n\\t<td class=\\\" \\\">Automated Affymetrix Array Analysis Base Package</td>\\n\\t<td class=\\\"  sorting_1\\\">1514</td>\\n</tr><tr id=\\\"pkg_BioNERO\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/BioNERO.html\\\">BioNERO</a></td>\\n\\t<td class=\\\" \\\">Fabricio Almeida-Silva</td>\\n\\t<td class=\\\" \\\">Biological Network Reconstruction Omnibus</td>\\n\\t<td class=\\\"  sorting_1\\\">1514</td>\\n</tr><tr id=\\\"pkg_esATAC\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/esATAC.html\\\">esATAC</a></td>\\n\\t<td class=\\\" \\\">Zheng Wei</td>\\n\\t<td class=\\\" \\\">An Easy-to-use Systematic pipeline for ATACseq data analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">1514</td>\\n</tr><tr id=\\\"pkg_msqrob2\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/msqrob2.html\\\">msqrob2</a></td>\\n\\t<td class=\\\" \\\">Lieven Clement</td>\\n\\t<td class=\\\" \\\">Robust statistical inference for quantitative LC-MS proteomics</td>\\n\\t<td class=\\\"  sorting_1\\\">1514</td>\\n</tr><tr id=\\\"pkg_phenopath\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/phenopath.html\\\">phenopath</a></td>\\n\\t<td class=\\\" \\\">Kieran Campbell</td>\\n\\t<td class=\\\" \\\">Genomic trajectories with heterogeneous genetic and environmental backgrounds</td>\\n\\t<td class=\\\"  sorting_1\\\">1514</td>\\n</tr><tr id=\\\"pkg_PICS\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/PICS.html\\\">PICS</a></td>\\n\\t<td class=\\\" \\\">Renan Sauteraud</td>\\n\\t<td class=\\\" \\\">Probabilistic inference of ChIP-seq</td>\\n\\t<td class=\\\"  sorting_1\\\">1514</td>\\n</tr><tr id=\\\"pkg_PROcess\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/PROcess.html\\\">PROcess</a></td>\\n\\t<td class=\\\" \\\">Xiaochun Li</td>\\n\\t<td class=\\\" \\\">Ciphergen SELDI-TOF Processing</td>\\n\\t<td class=\\\"  sorting_1\\\">1514</td>\\n</tr><tr id=\\\"pkg_RandomWalkRestartMH\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/RandomWalkRestartMH.html\\\">RandomWalkRestartMH</a></td>\\n\\t<td class=\\\" \\\">Alberto Valdeolivas</td>\\n\\t<td class=\\\" \\\">Random walk with restart on multiplex and heterogeneous Networks</td>\\n\\t<td class=\\\"  sorting_1\\\">1514</td>\\n</tr><tr id=\\\"pkg_ddCt\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ddCt.html\\\">ddCt</a></td>\\n\\t<td class=\\\" \\\">Jitao David Zhang</td>\\n\\t<td class=\\\" \\\">The ddCt Algorithm for the Analysis of Quantitative Real-Time PCR (qRT-PCR)</td>\\n\\t<td class=\\\"  sorting_1\\\">1522</td>\\n</tr><tr id=\\\"pkg_HelloRanges\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/HelloRanges.html\\\">HelloRanges</a></td>\\n\\t<td class=\\\" \\\">Michael Lawrence</td>\\n\\t<td class=\\\" \\\">Introduce *Ranges to bedtools users</td>\\n\\t<td class=\\\"  sorting_1\\\">1522</td>\\n</tr><tr id=\\\"pkg_HPAanalyze\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/HPAanalyze.html\\\">HPAanalyze</a></td>\\n\\t<td class=\\\" \\\">Anh Nhat Tran</td>\\n\\t<td class=\\\" \\\">Retrieve and analyze data from the Human Protein Atlas</td>\\n\\t<td class=\\\"  sorting_1\\\">1522</td>\\n</tr><tr id=\\\"pkg_proBatch\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/proBatch.html\\\">proBatch</a></td>\\n\\t<td class=\\\" \\\">Chloe H. Lee</td>\\n\\t<td class=\\\" \\\">Tools for Diagnostics and Corrections of Batch Effects in Proteomics</td>\\n\\t<td class=\\\"  sorting_1\\\">1522</td>\\n</tr><tr id=\\\"pkg_RDRToolbox\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/RDRToolbox.html\\\">RDRToolbox</a></td>\\n\\t<td class=\\\" \\\">Christoph Bartenhagen</td>\\n\\t<td class=\\\" \\\">A package for nonlinear dimension reduction with Isomap and LLE.</td>\\n\\t<td class=\\\"  sorting_1\\\">1522</td>\\n</tr><tr id=\\\"pkg_BgeeDB\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/BgeeDB.html\\\">BgeeDB</a></td>\\n\\t<td class=\\\" \\\">Julien Wollbrett, Julien Roux, Andrea Komljenovic, Frederic Bastian</td>\\n\\t<td class=\\\" \\\">Annotation and gene expression data retrieval from Bgee database. TopAnat, an anatomical entities Enrichment Analysis tool for UBERON ontology</td>\\n\\t<td class=\\\"  sorting_1\\\">1527</td>\\n</tr><tr id=\\\"pkg_HIBAG\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/HIBAG.html\\\">HIBAG</a></td>\\n\\t<td class=\\\" \\\">Xiuwen Zheng</td>\\n\\t<td class=\\\" \\\">HLA Genotype Imputation with Attribute Bagging</td>\\n\\t<td class=\\\"  sorting_1\\\">1527</td>\\n</tr><tr id=\\\"pkg_openPrimeR\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/openPrimeR.html\\\">openPrimeR</a></td>\\n\\t<td class=\\\" \\\">Matthias Dring</td>\\n\\t<td class=\\\" \\\">Multiplex PCR Primer Design and Analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">1527</td>\\n</tr><tr id=\\\"pkg_PWMEnrich\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/PWMEnrich.html\\\">PWMEnrich</a></td>\\n\\t<td class=\\\" \\\">Diego Diez</td>\\n\\t<td class=\\\" \\\">PWM enrichment analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">1527</td>\\n</tr><tr id=\\\"pkg_ChIPsim\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ChIPsim.html\\\">ChIPsim</a></td>\\n\\t<td class=\\\" \\\">Peter Humburg</td>\\n\\t<td class=\\\" \\\">Simulation of ChIP-seq experiments</td>\\n\\t<td class=\\\"  sorting_1\\\">1531</td>\\n</tr><tr id=\\\"pkg_eisaR\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/eisaR.html\\\">eisaR</a></td>\\n\\t<td class=\\\" \\\">Michael Stadler</td>\\n\\t<td class=\\\" \\\">Exon-Intron Split Analysis (EISA) in R</td>\\n\\t<td class=\\\"  sorting_1\\\">1531</td>\\n</tr><tr id=\\\"pkg_glmSparseNet\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/glmSparseNet.html\\\">glmSparseNet</a></td>\\n\\t<td class=\\\" \\\">Andr Verssimo</td>\\n\\t<td class=\\\" \\\">Network Centrality Metrics for Elastic-Net Regularized Models</td>\\n\\t<td class=\\\"  sorting_1\\\">1531</td>\\n</tr><tr id=\\\"pkg_igvR\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/igvR.html\\\">igvR</a></td>\\n\\t<td class=\\\" \\\">Paul Shannon</td>\\n\\t<td class=\\\" \\\">igvR: integrative genomics viewer</td>\\n\\t<td class=\\\"  sorting_1\\\">1531</td>\\n</tr><tr id=\\\"pkg_OrderedList\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/OrderedList.html\\\">OrderedList</a></td>\\n\\t<td class=\\\" \\\">Claudio Lottaz</td>\\n\\t<td class=\\\" \\\">Similarities of Ordered Gene Lists</td>\\n\\t<td class=\\\"  sorting_1\\\">1531</td>\\n</tr><tr id=\\\"pkg_pathwayPCA\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/pathwayPCA.html\\\">pathwayPCA</a></td>\\n\\t<td class=\\\" \\\">Gabriel Odom</td>\\n\\t<td class=\\\" \\\">Integrative Pathway Analysis with Modern PCA Methodology and Gene Selection</td>\\n\\t<td class=\\\"  sorting_1\\\">1531</td>\\n</tr><tr id=\\\"pkg_pmp\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/pmp.html\\\">pmp</a></td>\\n\\t<td class=\\\" \\\">Gavin Rhys Lloyd</td>\\n\\t<td class=\\\" \\\">Peak Matrix Processing and signal batch correction for metabolomics datasets</td>\\n\\t<td class=\\\"  sorting_1\\\">1531</td>\\n</tr><tr id=\\\"pkg_RbcBook1\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/RbcBook1.html\\\">RbcBook1</a></td>\\n\\t<td class=\\\" \\\">Vince Carey</td>\\n\\t<td class=\\\" \\\">Support for Springer monograph on Bioconductor</td>\\n\\t<td class=\\\"  sorting_1\\\">1531</td>\\n</tr><tr id=\\\"pkg_TissueEnrich\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/TissueEnrich.html\\\">TissueEnrich</a></td>\\n\\t<td class=\\\" \\\">Ashish Jain</td>\\n\\t<td class=\\\" \\\">Tissue-specific gene enrichment analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">1531</td>\\n</tr><tr id=\\\"pkg_tricycle\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/tricycle.html\\\">tricycle</a></td>\\n\\t<td class=\\\" \\\">Shijie Zheng</td>\\n\\t<td class=\\\" \\\">tricycle: Transferable Representation and Inference of cell cycle</td>\\n\\t<td class=\\\"  sorting_1\\\">1531</td>\\n</tr><tr id=\\\"pkg_bacon\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/bacon.html\\\">bacon</a></td>\\n\\t<td class=\\\" \\\">Maarten van Iterson</td>\\n\\t<td class=\\\" \\\">Controlling bias and inflation in association studies using the empirical null distribution</td>\\n\\t<td class=\\\"  sorting_1\\\">1541</td>\\n</tr><tr id=\\\"pkg_discordant\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/discordant.html\\\">discordant</a></td>\\n\\t<td class=\\\" \\\">McGrath Max</td>\\n\\t<td class=\\\" \\\">The Discordant Method: A Novel Approach for Differential Correlation</td>\\n\\t<td class=\\\"  sorting_1\\\">1541</td>\\n</tr><tr id=\\\"pkg_motifbreakR\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/motifbreakR.html\\\">motifbreakR</a></td>\\n\\t<td class=\\\" \\\">Simon Gert Coetzee</td>\\n\\t<td class=\\\" \\\">A Package For Predicting The Disruptiveness Of Single Nucleotide Polymorphisms On Transcription Factor Binding Sites</td>\\n\\t<td class=\\\"  sorting_1\\\">1541</td>\\n</tr><tr id=\\\"pkg_NanoStringDiff\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/NanoStringDiff.html\\\">NanoStringDiff</a></td>\\n\\t<td class=\\\" \\\">tingting zhai,hong wang</td>\\n\\t<td class=\\\" \\\">Differential Expression Analysis of NanoString nCounter Data</td>\\n\\t<td class=\\\"  sorting_1\\\">1541</td>\\n</tr><tr id=\\\"pkg_RnaSeqSampleSize\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/RnaSeqSampleSize.html\\\">RnaSeqSampleSize</a></td>\\n\\t<td class=\\\" \\\">Shilin Zhao Developer</td>\\n\\t<td class=\\\" \\\">RnaSeqSampleSize</td>\\n\\t<td class=\\\"  sorting_1\\\">1541</td>\\n</tr><tr id=\\\"pkg_statTarget\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/statTarget.html\\\">statTarget</a></td>\\n\\t<td class=\\\" \\\">Hemi Luan</td>\\n\\t<td class=\\\" \\\">Statistical Analysis of Molecular Profiles</td>\\n\\t<td class=\\\"  sorting_1\\\">1541</td>\\n</tr><tr id=\\\"pkg_struct\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/struct.html\\\">struct</a></td>\\n\\t<td class=\\\" \\\">Gavin Rhys Lloyd</td>\\n\\t<td class=\\\" \\\">Statistics in R Using Class-based Templates</td>\\n\\t<td class=\\\"  sorting_1\\\">1541</td>\\n</tr><tr id=\\\"pkg_tweeDEseq\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/tweeDEseq.html\\\">tweeDEseq</a></td>\\n\\t<td class=\\\" \\\">Juan R Gonzalez</td>\\n\\t<td class=\\\" \\\">RNA-seq data analysis using the Poisson-Tweedie family of distributions</td>\\n\\t<td class=\\\"  sorting_1\\\">1541</td>\\n</tr><tr id=\\\"pkg_ChIPseqR\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ChIPseqR.html\\\">ChIPseqR</a></td>\\n\\t<td class=\\\" \\\">Peter Humburg</td>\\n\\t<td class=\\\" \\\">Identifying Protein Binding Sites in High-Throughput Sequencing Data</td>\\n\\t<td class=\\\"  sorting_1\\\">1549</td>\\n</tr><tr id=\\\"pkg_coseq\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/coseq.html\\\">coseq</a></td>\\n\\t<td class=\\\" \\\">Andrea Rau</td>\\n\\t<td class=\\\" \\\">Co-Expression Analysis of Sequencing Data</td>\\n\\t<td class=\\\"  sorting_1\\\">1549</td>\\n</tr><tr id=\\\"pkg_geNetClassifier\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/geNetClassifier.html\\\">geNetClassifier</a></td>\\n\\t<td class=\\\" \\\">Sara Aibar</td>\\n\\t<td class=\\\" \\\">Classify diseases and build associated gene networks using gene expression profiles</td>\\n\\t<td class=\\\"  sorting_1\\\">1549</td>\\n</tr><tr id=\\\"pkg_GeneticsPed\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/GeneticsPed.html\\\">GeneticsPed</a></td>\\n\\t<td class=\\\" \\\">David Henderson</td>\\n\\t<td class=\\\" \\\">Pedigree and genetic relationship functions</td>\\n\\t<td class=\\\"  sorting_1\\\">1549</td>\\n</tr><tr id=\\\"pkg_GOexpress\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/GOexpress.html\\\">GOexpress</a></td>\\n\\t<td class=\\\" \\\">Kevin Rue-Albrecht</td>\\n\\t<td class=\\\" \\\">Visualise microarray and RNAseq data using gene ontology annotations</td>\\n\\t<td class=\\\"  sorting_1\\\">1549</td>\\n</tr><tr id=\\\"pkg_miQC\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/miQC.html\\\">miQC</a></td>\\n\\t<td class=\\\" \\\">Ariel Hippen</td>\\n\\t<td class=\\\" \\\">Flexible, probabilistic metrics for quality control of scRNA-seq data</td>\\n\\t<td class=\\\"  sorting_1\\\">1549</td>\\n</tr><tr id=\\\"pkg_OCplus\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/OCplus.html\\\">OCplus</a></td>\\n\\t<td class=\\\" \\\">Alexander Ploner</td>\\n\\t<td class=\\\" \\\">Operating characteristics plus sample size and local fdr for microarray experiments</td>\\n\\t<td class=\\\"  sorting_1\\\">1549</td>\\n</tr><tr id=\\\"pkg_vidger\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/vidger.html\\\">vidger</a></td>\\n\\t<td class=\\\" \\\">Brandon Monier</td>\\n\\t<td class=\\\" \\\">Create rapid visualizations of RNAseq data in R</td>\\n\\t<td class=\\\"  sorting_1\\\">1549</td>\\n</tr><tr id=\\\"pkg_animalcules\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/animalcules.html\\\">animalcules</a></td>\\n\\t<td class=\\\" \\\">Yue Zhao</td>\\n\\t<td class=\\\" \\\">Interactive microbiome analysis toolkit</td>\\n\\t<td class=\\\"  sorting_1\\\">1557</td>\\n</tr><tr id=\\\"pkg_chipenrich\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/chipenrich.html\\\">chipenrich</a></td>\\n\\t<td class=\\\" \\\">Raymond G. Cavalcante</td>\\n\\t<td class=\\\" \\\">Gene Set Enrichment For ChIP-seq Peak Data</td>\\n\\t<td class=\\\"  sorting_1\\\">1557</td>\\n</tr><tr id=\\\"pkg_CNTools\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/CNTools.html\\\">CNTools</a></td>\\n\\t<td class=\\\" \\\">J. Zhang</td>\\n\\t<td class=\\\" \\\">Convert segment data into a region by sample matrix to allow for other high level computational analyses.</td>\\n\\t<td class=\\\"  sorting_1\\\">1557</td>\\n</tr><tr id=\\\"pkg_CopyNumberPlots\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/CopyNumberPlots.html\\\">CopyNumberPlots</a></td>\\n\\t<td class=\\\" \\\">Bernat Gel</td>\\n\\t<td class=\\\" \\\">Create Copy-Number Plots using karyoploteR functionality</td>\\n\\t<td class=\\\"  sorting_1\\\">1557</td>\\n</tr><tr id=\\\"pkg_CSAR\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/CSAR.html\\\">CSAR</a></td>\\n\\t<td class=\\\" \\\">Jose M Muino</td>\\n\\t<td class=\\\" \\\">Statistical tools for the analysis of ChIP-seq data</td>\\n\\t<td class=\\\"  sorting_1\\\">1557</td>\\n</tr><tr id=\\\"pkg_girafe\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/girafe.html\\\">girafe</a></td>\\n\\t<td class=\\\" \\\">J. Toedling</td>\\n\\t<td class=\\\" \\\">Genome Intervals and Read Alignments for Functional Exploration</td>\\n\\t<td class=\\\"  sorting_1\\\">1557</td>\\n</tr><tr id=\\\"pkg_GWENA\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/GWENA.html\\\">GWENA</a></td>\\n\\t<td class=\\\" \\\">Gwenalle Lemoine</td>\\n\\t<td class=\\\" \\\">Pipeline for augmented co-expression analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">1557</td>\\n</tr><tr id=\\\"pkg_MetaboSignal\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/MetaboSignal.html\\\">MetaboSignal</a></td>\\n\\t<td class=\\\" \\\">Andrea Rodriguez-Martinez, Rafael Ayala</td>\\n\\t<td class=\\\" \\\">MetaboSignal: a network-based approach to overlay and explore metabolic and signaling KEGG pathways</td>\\n\\t<td class=\\\"  sorting_1\\\">1557</td>\\n</tr><tr id=\\\"pkg_mgsa\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/mgsa.html\\\">mgsa</a></td>\\n\\t<td class=\\\" \\\">Sebastian Bauer</td>\\n\\t<td class=\\\" \\\">Model-based gene set analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">1557</td>\\n</tr><tr id=\\\"pkg_MWASTools\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/MWASTools.html\\\">MWASTools</a></td>\\n\\t<td class=\\\" \\\">Andrea Rodriguez-Martinez, Rafael Ayala</td>\\n\\t<td class=\\\" \\\">MWASTools: an integrated pipeline to perform metabolome-wide association studies</td>\\n\\t<td class=\\\"  sorting_1\\\">1557</td>\\n</tr><tr id=\\\"pkg_r3Cseq\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/r3Cseq.html\\\">r3Cseq</a></td>\\n\\t<td class=\\\" \\\">Supat Thongjuea or</td>\\n\\t<td class=\\\" \\\">Analysis of Chromosome Conformation Capture and Next-generation Sequencing (3C-seq)</td>\\n\\t<td class=\\\"  sorting_1\\\">1557</td>\\n</tr><tr id=\\\"pkg_clusterStab\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/clusterStab.html\\\">clusterStab</a></td>\\n\\t<td class=\\\" \\\">James W. MacDonald</td>\\n\\t<td class=\\\" \\\">Compute cluster stability scores for microarray data</td>\\n\\t<td class=\\\"  sorting_1\\\">1568</td>\\n</tr><tr id=\\\"pkg_dupRadar\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/dupRadar.html\\\">dupRadar</a></td>\\n\\t<td class=\\\" \\\">Sergi Sayols, Holger Klein</td>\\n\\t<td class=\\\" \\\">Assessment of duplication rates in RNA-Seq datasets</td>\\n\\t<td class=\\\"  sorting_1\\\">1568</td>\\n</tr><tr id=\\\"pkg_SPOTlight\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/SPOTlight.html\\\">SPOTlight</a></td>\\n\\t<td class=\\\" \\\">Marc Elosua-Bayes</td>\\n\\t<td class=\\\" \\\">`SPOTlight`: Spatial Transcriptomics Deconvolution</td>\\n\\t<td class=\\\"  sorting_1\\\">1568</td>\\n</tr><tr id=\\\"pkg_BicARE\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/BicARE.html\\\">BicARE</a></td>\\n\\t<td class=\\\" \\\">Pierre Gestraud</td>\\n\\t<td class=\\\" \\\">Biclustering Analysis and Results Exploration</td>\\n\\t<td class=\\\"  sorting_1\\\">1571</td>\\n</tr><tr id=\\\"pkg_biodb\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/biodb.html\\\">biodb</a></td>\\n\\t<td class=\\\" \\\">Pierrick Roger</td>\\n\\t<td class=\\\" \\\">biodb, a library and a development framework for connecting to chemical and biological databases</td>\\n\\t<td class=\\\"  sorting_1\\\">1571</td>\\n</tr><tr id=\\\"pkg_deepSNV\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/deepSNV.html\\\">deepSNV</a></td>\\n\\t<td class=\\\" \\\">Moritz Gerstung</td>\\n\\t<td class=\\\" \\\">Detection of subclonal SNVs in deep sequencing data.</td>\\n\\t<td class=\\\"  sorting_1\\\">1571</td>\\n</tr><tr id=\\\"pkg_iCARE\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/iCARE.html\\\">iCARE</a></td>\\n\\t<td class=\\\" \\\">Bill Wheeler</td>\\n\\t<td class=\\\" \\\">A Tool for Individualized Coherent Absolute Risk Estimation (iCARE)</td>\\n\\t<td class=\\\"  sorting_1\\\">1571</td>\\n</tr><tr id=\\\"pkg_SCnorm\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/SCnorm.html\\\">SCnorm</a></td>\\n\\t<td class=\\\" \\\">Rhonda Bacher</td>\\n\\t<td class=\\\" \\\">Normalization of single cell RNA-seq data</td>\\n\\t<td class=\\\"  sorting_1\\\">1571</td>\\n</tr><tr id=\\\"pkg_TreeAndLeaf\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/TreeAndLeaf.html\\\">TreeAndLeaf</a></td>\\n\\t<td class=\\\" \\\">Milena A. Cardoso</td>\\n\\t<td class=\\\" \\\">Displaying binary trees with focus on dendrogram leaves</td>\\n\\t<td class=\\\"  sorting_1\\\">1571</td>\\n</tr><tr id=\\\"pkg_velociraptor\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/velociraptor.html\\\">velociraptor</a></td>\\n\\t<td class=\\\" \\\">Kevin Rue-Albrecht</td>\\n\\t<td class=\\\" \\\">Toolkit for Single-Cell Velocity</td>\\n\\t<td class=\\\"  sorting_1\\\">1571</td>\\n</tr><tr id=\\\"pkg_BiocPkgTools\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/BiocPkgTools.html\\\">BiocPkgTools</a></td>\\n\\t<td class=\\\" \\\">Sean Davis</td>\\n\\t<td class=\\\" \\\">Collection of simple tools for learning about Bioconductor Packages</td>\\n\\t<td class=\\\"  sorting_1\\\">1578</td>\\n</tr><tr id=\\\"pkg_CAGEr\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/CAGEr.html\\\">CAGEr</a></td>\\n\\t<td class=\\\" \\\">Charles Plessy</td>\\n\\t<td class=\\\" \\\">Analysis of CAGE (Cap Analysis of Gene Expression) sequencing data for precise mapping of transcription start sites and promoterome mining</td>\\n\\t<td class=\\\"  sorting_1\\\">1578</td>\\n</tr><tr id=\\\"pkg_gpls\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/gpls.html\\\">gpls</a></td>\\n\\t<td class=\\\" \\\">Bioconductor Package Maintainer</td>\\n\\t<td class=\\\" \\\">Classification using generalized partial least squares</td>\\n\\t<td class=\\\"  sorting_1\\\">1578</td>\\n</tr><tr id=\\\"pkg_hypeR\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/hypeR.html\\\">hypeR</a></td>\\n\\t<td class=\\\" \\\">Anthony Federico</td>\\n\\t<td class=\\\" \\\">An R Package For Geneset Enrichment Workflows</td>\\n\\t<td class=\\\"  sorting_1\\\">1578</td>\\n</tr><tr id=\\\"pkg_megadepth\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/megadepth.html\\\">megadepth</a></td>\\n\\t<td class=\\\" \\\">David Zhang</td>\\n\\t<td class=\\\" \\\">megadepth: BigWig and BAM related utilities</td>\\n\\t<td class=\\\"  sorting_1\\\">1578</td>\\n</tr><tr id=\\\"pkg_puma\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/puma.html\\\">puma</a></td>\\n\\t<td class=\\\" \\\">Xuejun Liu</td>\\n\\t<td class=\\\" \\\">Propagating Uncertainty in Microarray Analysis(including Affymetrix tranditional 3' arrays and exon arrays and Human Transcriptome Array 2.0)</td>\\n\\t<td class=\\\"  sorting_1\\\">1578</td>\\n</tr><tr id=\\\"pkg_a4Core\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/a4Core.html\\\">a4Core</a></td>\\n\\t<td class=\\\" \\\">Laure Cougnaud</td>\\n\\t<td class=\\\" \\\">Automated Affymetrix Array Analysis Core Package</td>\\n\\t<td class=\\\"  sorting_1\\\">1584</td>\\n</tr><tr id=\\\"pkg_diffHic\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/diffHic.html\\\">diffHic</a></td>\\n\\t<td class=\\\" \\\">Aaron Lun</td>\\n\\t<td class=\\\" \\\">Differential Analyis of Hi-C Data</td>\\n\\t<td class=\\\"  sorting_1\\\">1584</td>\\n</tr><tr id=\\\"pkg_EWCE\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/EWCE.html\\\">EWCE</a></td>\\n\\t<td class=\\\" \\\">Alan Murphy</td>\\n\\t<td class=\\\" \\\">Expression Weighted Celltype Enrichment</td>\\n\\t<td class=\\\"  sorting_1\\\">1584</td>\\n</tr><tr id=\\\"pkg_frma\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/frma.html\\\">frma</a></td>\\n\\t<td class=\\\" \\\">Matthew N. McCall</td>\\n\\t<td class=\\\" \\\">Frozen RMA and Barcode</td>\\n\\t<td class=\\\"  sorting_1\\\">1584</td>\\n</tr><tr id=\\\"pkg_planet\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/planet.html\\\">planet</a></td>\\n\\t<td class=\\\" \\\">Victor Yuan</td>\\n\\t<td class=\\\" \\\">Placental DNA methylation analysis tools</td>\\n\\t<td class=\\\"  sorting_1\\\">1584</td>\\n</tr><tr id=\\\"pkg_SamSPECTRAL\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/SamSPECTRAL.html\\\">SamSPECTRAL</a></td>\\n\\t<td class=\\\" \\\">Habil</td>\\n\\t<td class=\\\" \\\">Identifies cell population in flow cytometry data</td>\\n\\t<td class=\\\"  sorting_1\\\">1584</td>\\n</tr><tr id=\\\"pkg_twilight\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/twilight.html\\\">twilight</a></td>\\n\\t<td class=\\\" \\\">Stefanie Scheid</td>\\n\\t<td class=\\\" \\\">Estimation of local false discovery rate</td>\\n\\t<td class=\\\"  sorting_1\\\">1584</td>\\n</tr><tr id=\\\"pkg_zFPKM\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/zFPKM.html\\\">zFPKM</a></td>\\n\\t<td class=\\\" \\\">Ron Ammar</td>\\n\\t<td class=\\\" \\\">A suite of functions to facilitate zFPKM transformations</td>\\n\\t<td class=\\\"  sorting_1\\\">1584</td>\\n</tr><tr id=\\\"pkg_ExpressionAtlas\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ExpressionAtlas.html\\\">ExpressionAtlas</a></td>\\n\\t<td class=\\\" \\\">Pedro Madrigal</td>\\n\\t<td class=\\\" \\\">Download datasets from EMBL-EBI Expression Atlas</td>\\n\\t<td class=\\\"  sorting_1\\\">1592</td>\\n</tr><tr id=\\\"pkg_granulator\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/granulator.html\\\">granulator</a></td>\\n\\t<td class=\\\" \\\">Sabina Pfister</td>\\n\\t<td class=\\\" \\\">Rapid benchmarking of methods for *in silico* deconvolution of bulk RNA-seq data</td>\\n\\t<td class=\\\"  sorting_1\\\">1592</td>\\n</tr><tr id=\\\"pkg_metaMS\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/metaMS.html\\\">metaMS</a></td>\\n\\t<td class=\\\" \\\">Yann Guitton</td>\\n\\t<td class=\\\" \\\">MS-based metabolomics annotation pipeline</td>\\n\\t<td class=\\\"  sorting_1\\\">1592</td>\\n</tr><tr id=\\\"pkg_QUBIC\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/QUBIC.html\\\">QUBIC</a></td>\\n\\t<td class=\\\" \\\">Yu Zhang</td>\\n\\t<td class=\\\" \\\">An R package for qualitative biclustering in support of gene co-expression analyses</td>\\n\\t<td class=\\\"  sorting_1\\\">1592</td>\\n</tr><tr id=\\\"pkg_scone\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/scone.html\\\">scone</a></td>\\n\\t<td class=\\\" \\\">Davide Risso</td>\\n\\t<td class=\\\" \\\">Single Cell Overview of Normalized Expression data</td>\\n\\t<td class=\\\"  sorting_1\\\">1592</td>\\n</tr><tr id=\\\"pkg_yarn\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/yarn.html\\\">yarn</a></td>\\n\\t<td class=\\\" \\\">Joseph N Paulson</td>\\n\\t<td class=\\\" \\\">YARN: Robust Multi-Condition RNA-Seq Preprocessing and Normalization</td>\\n\\t<td class=\\\"  sorting_1\\\">1592</td>\\n</tr><tr id=\\\"pkg_exomePeak2\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/exomePeak2.html\\\">exomePeak2</a></td>\\n\\t<td class=\\\" \\\">Zhen Wei</td>\\n\\t<td class=\\\" \\\">Peak Calling and differential analysis for MeRIP-Seq</td>\\n\\t<td class=\\\"  sorting_1\\\">1598</td>\\n</tr><tr id=\\\"pkg_Modstrings\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/Modstrings.html\\\">Modstrings</a></td>\\n\\t<td class=\\\" \\\">Felix G.M. Ernst</td>\\n\\t<td class=\\\" \\\">Working with modified nucleotide sequences</td>\\n\\t<td class=\\\"  sorting_1\\\">1598</td>\\n</tr><tr id=\\\"pkg_pandaR\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/pandaR.html\\\">pandaR</a></td>\\n\\t<td class=\\\" \\\">Joseph N. Paulson, Dan Schlauch</td>\\n\\t<td class=\\\" \\\">PANDA Algorithm</td>\\n\\t<td class=\\\"  sorting_1\\\">1598</td>\\n</tr><tr id=\\\"pkg_paxtoolsr\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/paxtoolsr.html\\\">paxtoolsr</a></td>\\n\\t<td class=\\\" \\\">Augustin Luna</td>\\n\\t<td class=\\\" \\\">Access Pathways from Multiple Databases Through BioPAX and Pathway Commons</td>\\n\\t<td class=\\\"  sorting_1\\\">1598</td>\\n</tr><tr id=\\\"pkg_pdInfoBuilder\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/pdInfoBuilder.html\\\">pdInfoBuilder</a></td>\\n\\t<td class=\\\" \\\">Benilton Carvalho</td>\\n\\t<td class=\\\" \\\">Platform Design Information Package Builder</td>\\n\\t<td class=\\\"  sorting_1\\\">1598</td>\\n</tr><tr id=\\\"pkg_rebook\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/rebook.html\\\">rebook</a></td>\\n\\t<td class=\\\" \\\">Aaron Lun</td>\\n\\t<td class=\\\" \\\">Re-using Content in Bioconductor Books</td>\\n\\t<td class=\\\"  sorting_1\\\">1598</td>\\n</tr><tr id=\\\"pkg_maser\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/maser.html\\\">maser</a></td>\\n\\t<td class=\\\" \\\">Diogo F.T. Veiga</td>\\n\\t<td class=\\\" \\\">Mapping Alternative Splicing Events to pRoteins</td>\\n\\t<td class=\\\"  sorting_1\\\">1604</td>\\n</tr><tr id=\\\"pkg_regionReport\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/regionReport.html\\\">regionReport</a></td>\\n\\t<td class=\\\" \\\">Leonardo Collado-Torres</td>\\n\\t<td class=\\\" \\\">Generate HTML or PDF reports for a set of genomic regions or DESeq2/edgeR results</td>\\n\\t<td class=\\\"  sorting_1\\\">1604</td>\\n</tr><tr id=\\\"pkg_scry\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/scry.html\\\">scry</a></td>\\n\\t<td class=\\\" \\\">Kelly Street</td>\\n\\t<td class=\\\" \\\">Small-Count Analysis Methods for High-Dimensional Data</td>\\n\\t<td class=\\\"  sorting_1\\\">1604</td>\\n</tr><tr id=\\\"pkg_VariantTools\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/VariantTools.html\\\">VariantTools</a></td>\\n\\t<td class=\\\" \\\">Michael Lawrence</td>\\n\\t<td class=\\\" \\\">Tools for Exploratory Analysis of Variant Calls</td>\\n\\t<td class=\\\"  sorting_1\\\">1604</td>\\n</tr><tr id=\\\"pkg_methylclock\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/methylclock.html\\\">methylclock</a></td>\\n\\t<td class=\\\" \\\">Dolors Pelegri-Siso</td>\\n\\t<td class=\\\" \\\">Methylclock - DNA methylation-based clocks</td>\\n\\t<td class=\\\"  sorting_1\\\">1608</td>\\n</tr><tr id=\\\"pkg_mosaics\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/mosaics.html\\\">mosaics</a></td>\\n\\t<td class=\\\" \\\">Dongjun Chung</td>\\n\\t<td class=\\\" \\\">MOSAiCS (MOdel-based one and two Sample Analysis and Inference for ChIP-Seq)</td>\\n\\t<td class=\\\"  sorting_1\\\">1608</td>\\n</tr><tr id=\\\"pkg_PeacoQC\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/PeacoQC.html\\\">PeacoQC</a></td>\\n\\t<td class=\\\" \\\">Annelies Emmaneel</td>\\n\\t<td class=\\\" \\\">Peak-based selection of high quality cytometry data</td>\\n\\t<td class=\\\"  sorting_1\\\">1608</td>\\n</tr><tr id=\\\"pkg_qrqc\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/qrqc.html\\\">qrqc</a></td>\\n\\t<td class=\\\" \\\">Vince Buffalo</td>\\n\\t<td class=\\\" \\\">Quick Read Quality Control</td>\\n\\t<td class=\\\"  sorting_1\\\">1608</td>\\n</tr><tr id=\\\"pkg_ccrepe\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ccrepe.html\\\">ccrepe</a></td>\\n\\t<td class=\\\" \\\">Emma Schwager,Craig Bielski, George Weingart</td>\\n\\t<td class=\\\" \\\">ccrepe_and_nc.score</td>\\n\\t<td class=\\\"  sorting_1\\\">1612</td>\\n</tr><tr id=\\\"pkg_ExploreModelMatrix\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ExploreModelMatrix.html\\\">ExploreModelMatrix</a></td>\\n\\t<td class=\\\" \\\">Charlotte Soneson</td>\\n\\t<td class=\\\" \\\">Graphical Exploration of Design Matrices</td>\\n\\t<td class=\\\"  sorting_1\\\">1612</td>\\n</tr><tr id=\\\"pkg_omicade4\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/omicade4.html\\\">omicade4</a></td>\\n\\t<td class=\\\" \\\">Chen Meng</td>\\n\\t<td class=\\\" \\\">Multiple co-inertia analysis of omics datasets</td>\\n\\t<td class=\\\"  sorting_1\\\">1612</td>\\n</tr><tr id=\\\"pkg_TCGAbiolinksGUI\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/TCGAbiolinksGUI.html\\\">TCGAbiolinksGUI</a></td>\\n\\t<td class=\\\" \\\">Tiago C. Silva</td>\\n\\t<td class=\\\" \\\">\\\"TCGAbiolinksGUI: A Graphical User Interface to analyze cancer molecular and clinical data\\\"</td>\\n\\t<td class=\\\"  sorting_1\\\">1612</td>\\n</tr><tr id=\\\"pkg_tidySingleCellExperiment\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/tidySingleCellExperiment.html\\\">tidySingleCellExperiment</a></td>\\n\\t<td class=\\\" \\\">Stefano Mangiola</td>\\n\\t<td class=\\\" \\\">Brings SingleCellExperiment to the Tidyverse</td>\\n\\t<td class=\\\"  sorting_1\\\">1612</td>\\n</tr><tr id=\\\"pkg_BRAIN\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/BRAIN.html\\\">BRAIN</a></td>\\n\\t<td class=\\\" \\\">Piotr Dittwald</td>\\n\\t<td class=\\\" \\\">Baffling Recursive Algorithm for Isotope distributioN calculations</td>\\n\\t<td class=\\\"  sorting_1\\\">1617</td>\\n</tr><tr id=\\\"pkg_MEDIPS\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/MEDIPS.html\\\">MEDIPS</a></td>\\n\\t<td class=\\\" \\\">Lukas Chavez</td>\\n\\t<td class=\\\" \\\">DNA IP-seq data analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">1618</td>\\n</tr><tr id=\\\"pkg_arrayQuality\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/arrayQuality.html\\\">arrayQuality</a></td>\\n\\t<td class=\\\" \\\">Agnes Paquet</td>\\n\\t<td class=\\\" \\\">Assessing array quality on spotted arrays</td>\\n\\t<td class=\\\"  sorting_1\\\">1619</td>\\n</tr><tr id=\\\"pkg_CellBench\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/CellBench.html\\\">CellBench</a></td>\\n\\t<td class=\\\" \\\">Shian Su</td>\\n\\t<td class=\\\" \\\">Construct Benchmarks for Single Cell Analysis Methods</td>\\n\\t<td class=\\\"  sorting_1\\\">1619</td>\\n</tr><tr id=\\\"pkg_flowMeans\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/flowMeans.html\\\">flowMeans</a></td>\\n\\t<td class=\\\" \\\">Nima Aghaeepour</td>\\n\\t<td class=\\\" \\\">Non-parametric Flow Cytometry Data Gating</td>\\n\\t<td class=\\\"  sorting_1\\\">1619</td>\\n</tr><tr id=\\\"pkg_imcRtools\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/imcRtools.html\\\">imcRtools</a></td>\\n\\t<td class=\\\" \\\">Nils Eling</td>\\n\\t<td class=\\\" \\\">Methods for imaging mass cytometry data analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">1619</td>\\n</tr><tr id=\\\"pkg_methylGSA\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/methylGSA.html\\\">methylGSA</a></td>\\n\\t<td class=\\\" \\\">Xu Ren</td>\\n\\t<td class=\\\" \\\">Gene Set Analysis Using the Outcome of Differential Methylation</td>\\n\\t<td class=\\\"  sorting_1\\\">1619</td>\\n</tr><tr id=\\\"pkg_ontoProc\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ontoProc.html\\\">ontoProc</a></td>\\n\\t<td class=\\\" \\\">Vincent Carey</td>\\n\\t<td class=\\\" \\\">processing of ontologies of anatomy, cell lines, and so on</td>\\n\\t<td class=\\\"  sorting_1\\\">1619</td>\\n</tr><tr id=\\\"pkg_seqTools\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/seqTools.html\\\">seqTools</a></td>\\n\\t<td class=\\\" \\\">Wolfgang Kaisers</td>\\n\\t<td class=\\\" \\\">Analysis of nucleotide, sequence and quality content on fastq files</td>\\n\\t<td class=\\\"  sorting_1\\\">1619</td>\\n</tr><tr id=\\\"pkg_snifter\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/snifter.html\\\">snifter</a></td>\\n\\t<td class=\\\" \\\">Alan O'Callaghan</td>\\n\\t<td class=\\\" \\\">R wrapper for the python openTSNE library</td>\\n\\t<td class=\\\"  sorting_1\\\">1619</td>\\n</tr><tr id=\\\"pkg_BioQC\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/BioQC.html\\\">BioQC</a></td>\\n\\t<td class=\\\" \\\">Jitao David Zhang</td>\\n\\t<td class=\\\" \\\">Detect tissue heterogeneity in expression profiles with gene sets</td>\\n\\t<td class=\\\"  sorting_1\\\">1627</td>\\n</tr><tr id=\\\"pkg_edge\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/edge.html\\\">edge</a></td>\\n\\t<td class=\\\" \\\">John D. Storey, Andrew J. Bass</td>\\n\\t<td class=\\\" \\\">Extraction of Differential Gene Expression</td>\\n\\t<td class=\\\"  sorting_1\\\">1627</td>\\n</tr><tr id=\\\"pkg_Rfastp\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/Rfastp.html\\\">Rfastp</a></td>\\n\\t<td class=\\\" \\\">Thomas Carroll</td>\\n\\t<td class=\\\" \\\">An Ultra-Fast and All-in-One Fastq Preprocessor (Quality Control, Adapter, low quality and polyX trimming) and UMI Sequence Parsing).</td>\\n\\t<td class=\\\"  sorting_1\\\">1627</td>\\n</tr><tr id=\\\"pkg_BHC\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/BHC.html\\\">BHC</a></td>\\n\\t<td class=\\\" \\\">Rich Savage</td>\\n\\t<td class=\\\" \\\">Bayesian Hierarchical Clustering</td>\\n\\t<td class=\\\"  sorting_1\\\">1630</td>\\n</tr><tr id=\\\"pkg_CRISPRseek\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/CRISPRseek.html\\\">CRISPRseek</a></td>\\n\\t<td class=\\\" \\\">Lihua Julie Zhu</td>\\n\\t<td class=\\\" \\\">Design of target-specific guide RNAs in CRISPR-Cas9, genome-editing systems</td>\\n\\t<td class=\\\"  sorting_1\\\">1630</td>\\n</tr><tr id=\\\"pkg_distinct\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/distinct.html\\\">distinct</a></td>\\n\\t<td class=\\\" \\\">Simone Tiberi</td>\\n\\t<td class=\\\" \\\">distinct: a method for differential analyses via hierarchical permutation tests</td>\\n\\t<td class=\\\"  sorting_1\\\">1630</td>\\n</tr><tr id=\\\"pkg_flowPeaks\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/flowPeaks.html\\\">flowPeaks</a></td>\\n\\t<td class=\\\" \\\">Yongchao Ge</td>\\n\\t<td class=\\\" \\\">An R package for flow data clustering</td>\\n\\t<td class=\\\"  sorting_1\\\">1630</td>\\n</tr><tr id=\\\"pkg_GSAR\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/GSAR.html\\\">GSAR</a></td>\\n\\t<td class=\\\" \\\">Yasir Rahmatallah, Galina Glazko</td>\\n\\t<td class=\\\" \\\">Gene Set Analysis in R</td>\\n\\t<td class=\\\"  sorting_1\\\">1630</td>\\n</tr><tr id=\\\"pkg_Guitar\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/Guitar.html\\\">Guitar</a></td>\\n\\t<td class=\\\" \\\">Jia Meng</td>\\n\\t<td class=\\\" \\\">Guitar</td>\\n\\t<td class=\\\"  sorting_1\\\">1630</td>\\n</tr><tr id=\\\"pkg_NormalyzerDE\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/NormalyzerDE.html\\\">NormalyzerDE</a></td>\\n\\t<td class=\\\" \\\">Jakob Willforss</td>\\n\\t<td class=\\\" \\\">Evaluation of normalization methods and calculation of differential expression analysis statistics</td>\\n\\t<td class=\\\"  sorting_1\\\">1630</td>\\n</tr><tr id=\\\"pkg_a4Preproc\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/a4Preproc.html\\\">a4Preproc</a></td>\\n\\t<td class=\\\" \\\">Laure Cougnaud</td>\\n\\t<td class=\\\" \\\">Automated Affymetrix Array Analysis Preprocessing Package</td>\\n\\t<td class=\\\"  sorting_1\\\">1637</td>\\n</tr><tr id=\\\"pkg_DAPAR\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/DAPAR.html\\\">DAPAR</a></td>\\n\\t<td class=\\\" \\\">Samuel Wieczorek</td>\\n\\t<td class=\\\" \\\">Tools for the Differential Analysis of Proteins Abundance with R</td>\\n\\t<td class=\\\"  sorting_1\\\">1637</td>\\n</tr><tr id=\\\"pkg_lipidr\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/lipidr.html\\\">lipidr</a></td>\\n\\t<td class=\\\" \\\">Ahmed Mohamed</td>\\n\\t<td class=\\\" \\\">Data Mining and Analysis of Lipidomics Datasets</td>\\n\\t<td class=\\\"  sorting_1\\\">1637</td>\\n</tr><tr id=\\\"pkg_BatchQC\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/BatchQC.html\\\">BatchQC</a></td>\\n\\t<td class=\\\" \\\">Solaiappan Manimaran</td>\\n\\t<td class=\\\" \\\">Batch Effects Quality Control Software</td>\\n\\t<td class=\\\"  sorting_1\\\">1640</td>\\n</tr><tr id=\\\"pkg_agilp\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/agilp.html\\\">agilp</a></td>\\n\\t<td class=\\\" \\\">Benny Chain</td>\\n\\t<td class=\\\" \\\">Agilent expression array processing package</td>\\n\\t<td class=\\\"  sorting_1\\\">1641</td>\\n</tr><tr id=\\\"pkg_GeneTonic\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/GeneTonic.html\\\">GeneTonic</a></td>\\n\\t<td class=\\\" \\\">Federico Marini</td>\\n\\t<td class=\\\" \\\">Enjoy Analyzing And Integrating The Results From Differential Expression Analysis And Functional Enrichment Analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">1641</td>\\n</tr><tr id=\\\"pkg_FELLA\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/FELLA.html\\\">FELLA</a></td>\\n\\t<td class=\\\" \\\">Sergio Picart-Armada</td>\\n\\t<td class=\\\" \\\">Interpretation and enrichment for metabolomics data</td>\\n\\t<td class=\\\"  sorting_1\\\">1643</td>\\n</tr><tr id=\\\"pkg_FRASER\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/FRASER.html\\\">FRASER</a></td>\\n\\t<td class=\\\" \\\">Christian Mertes</td>\\n\\t<td class=\\\" \\\">Find RAre Splicing Events in RNA-Seq Data</td>\\n\\t<td class=\\\"  sorting_1\\\">1643</td>\\n</tr><tr id=\\\"pkg_makecdfenv\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/makecdfenv.html\\\">makecdfenv</a></td>\\n\\t<td class=\\\" \\\">James W. MacDonald</td>\\n\\t<td class=\\\" \\\">CDF Environment Maker</td>\\n\\t<td class=\\\"  sorting_1\\\">1643</td>\\n</tr><tr id=\\\"pkg_Cardinal\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/Cardinal.html\\\">Cardinal</a></td>\\n\\t<td class=\\\" \\\">Kylie A. Bemis</td>\\n\\t<td class=\\\" \\\">A mass spectrometry imaging toolbox for statistical analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">1646</td>\\n</tr><tr id=\\\"pkg_miRNAtap\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/miRNAtap.html\\\">miRNAtap</a></td>\\n\\t<td class=\\\" \\\">T. Ian Simpson</td>\\n\\t<td class=\\\" \\\">miRNAtap: microRNA Targets - Aggregated Predictions</td>\\n\\t<td class=\\\"  sorting_1\\\">1646</td>\\n</tr><tr id=\\\"pkg_mogsa\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/mogsa.html\\\">mogsa</a></td>\\n\\t<td class=\\\" \\\">Chen Meng</td>\\n\\t<td class=\\\" \\\">Multiple omics data integrative clustering and gene set analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">1646</td>\\n</tr><tr id=\\\"pkg_RTN\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/RTN.html\\\">RTN</a></td>\\n\\t<td class=\\\" \\\">Mauro Castro</td>\\n\\t<td class=\\\" \\\">RTN: Reconstruction of Transcriptional regulatory Networks and analysis of regulons</td>\\n\\t<td class=\\\"  sorting_1\\\">1646</td>\\n</tr><tr id=\\\"pkg_sigPathway\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/sigPathway.html\\\">sigPathway</a></td>\\n\\t<td class=\\\" \\\">Weil Lai</td>\\n\\t<td class=\\\" \\\">Pathway Analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">1646</td>\\n</tr><tr id=\\\"pkg_BASiCS\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/BASiCS.html\\\">BASiCS</a></td>\\n\\t<td class=\\\" \\\">Catalina Vallejos</td>\\n\\t<td class=\\\" \\\">Bayesian Analysis of Single-Cell Sequencing data</td>\\n\\t<td class=\\\"  sorting_1\\\">1651</td>\\n</tr><tr id=\\\"pkg_CelliD\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/CelliD.html\\\">CelliD</a></td>\\n\\t<td class=\\\" \\\">Akira Cortal</td>\\n\\t<td class=\\\" \\\">Unbiased Extraction of Single Cell gene signatures using Multiple Correspondence Analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">1651</td>\\n</tr><tr id=\\\"pkg_crlmm\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/crlmm.html\\\">crlmm</a></td>\\n\\t<td class=\\\" \\\">Benilton S Carvalho, Robert Scharpf, Matt Ritchie</td>\\n\\t<td class=\\\" \\\">Genotype Calling (CRLMM) and Copy Number Analysis tool for Affymetrix SNP 5.0 and 6.0 and Illumina arrays</td>\\n\\t<td class=\\\"  sorting_1\\\">1651</td>\\n</tr><tr id=\\\"pkg_matter\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/matter.html\\\">matter</a></td>\\n\\t<td class=\\\" \\\">Kylie A. Bemis</td>\\n\\t<td class=\\\" \\\">A framework for rapid prototyping with file-based data structures</td>\\n\\t<td class=\\\"  sorting_1\\\">1651</td>\\n</tr><tr id=\\\"pkg_polyester\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/polyester.html\\\">polyester</a></td>\\n\\t<td class=\\\" \\\">Jack Fu, Jeff Leek</td>\\n\\t<td class=\\\" \\\">Simulate RNA-seq reads</td>\\n\\t<td class=\\\"  sorting_1\\\">1651</td>\\n</tr><tr id=\\\"pkg_kebabs\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/kebabs.html\\\">kebabs</a></td>\\n\\t<td class=\\\" \\\">Ulrich Bodenhofer</td>\\n\\t<td class=\\\" \\\">Kernel-Based Analysis Of Biological Sequences</td>\\n\\t<td class=\\\"  sorting_1\\\">1656</td>\\n</tr><tr id=\\\"pkg_Rcpi\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/Rcpi.html\\\">Rcpi</a></td>\\n\\t<td class=\\\" \\\">Nan Xiao</td>\\n\\t<td class=\\\" \\\">Molecular Informatics Toolkit for Compound-Protein Interaction in Drug Discovery</td>\\n\\t<td class=\\\"  sorting_1\\\">1656</td>\\n</tr><tr id=\\\"pkg_SIAMCAT\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/SIAMCAT.html\\\">SIAMCAT</a></td>\\n\\t<td class=\\\" \\\">Jakob Wirbel</td>\\n\\t<td class=\\\" \\\">Statistical Inference of Associations between Microbial Communities And host phenoTypes</td>\\n\\t<td class=\\\"  sorting_1\\\">1656</td>\\n</tr><tr id=\\\"pkg_GOSim\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/GOSim.html\\\">GOSim</a></td>\\n\\t<td class=\\\" \\\">Holger Froehlich</td>\\n\\t<td class=\\\" \\\">Computation of functional similarities between GO terms and gene products; GO enrichment analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">1659</td>\\n</tr><tr id=\\\"pkg_gtrellis\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/gtrellis.html\\\">gtrellis</a></td>\\n\\t<td class=\\\" \\\">Zuguang Gu</td>\\n\\t<td class=\\\" \\\">Genome Level Trellis Layout</td>\\n\\t<td class=\\\"  sorting_1\\\">1659</td>\\n</tr><tr id=\\\"pkg_MethylMix\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/MethylMix.html\\\">MethylMix</a></td>\\n\\t<td class=\\\" \\\">Olivier Gevaert</td>\\n\\t<td class=\\\" \\\">MethylMix: Identifying methylation driven cancer genes</td>\\n\\t<td class=\\\"  sorting_1\\\">1659</td>\\n</tr><tr id=\\\"pkg_cellHTS2\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/cellHTS2.html\\\">cellHTS2</a></td>\\n\\t<td class=\\\" \\\">Joseph Barry</td>\\n\\t<td class=\\\" \\\">Analysis of cell-based screens - revised version of cellHTS</td>\\n\\t<td class=\\\"  sorting_1\\\">1662</td>\\n</tr><tr id=\\\"pkg_coRdon\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/coRdon.html\\\">coRdon</a></td>\\n\\t<td class=\\\" \\\">Anamaria Elek</td>\\n\\t<td class=\\\" \\\">Codon Usage Analysis and Prediction of Gene Expressivity</td>\\n\\t<td class=\\\"  sorting_1\\\">1662</td>\\n</tr><tr id=\\\"pkg_HiTC\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/HiTC.html\\\">HiTC</a></td>\\n\\t<td class=\\\" \\\">Nicolas Servant</td>\\n\\t<td class=\\\" \\\">High Throughput Chromosome Conformation Capture analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">1662</td>\\n</tr><tr id=\\\"pkg_CARNIVAL\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/CARNIVAL.html\\\">CARNIVAL</a></td>\\n\\t<td class=\\\" \\\">Attila Gabor</td>\\n\\t<td class=\\\" \\\">A CAusal Reasoning tool for Network Identification (from gene expression data) using Integer VALue programming</td>\\n\\t<td class=\\\"  sorting_1\\\">1665</td>\\n</tr><tr id=\\\"pkg_qpgraph\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/qpgraph.html\\\">qpgraph</a></td>\\n\\t<td class=\\\" \\\">Robert Castelo</td>\\n\\t<td class=\\\" \\\">Estimation of genetic and molecular regulatory networks from high-throughput genomics data</td>\\n\\t<td class=\\\"  sorting_1\\\">1665</td>\\n</tr><tr id=\\\"pkg_TCseq\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/TCseq.html\\\">TCseq</a></td>\\n\\t<td class=\\\" \\\">Mengjun Wu</td>\\n\\t<td class=\\\" \\\">Time course sequencing data analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">1665</td>\\n</tr><tr id=\\\"pkg_BrowserViz\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/BrowserViz.html\\\">BrowserViz</a></td>\\n\\t<td class=\\\" \\\">Paul Shannon</td>\\n\\t<td class=\\\" \\\">BrowserViz: interactive R/browser graphics using websockets and JSON</td>\\n\\t<td class=\\\"  sorting_1\\\">1668</td>\\n</tr><tr id=\\\"pkg_meshes\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/meshes.html\\\">meshes</a></td>\\n\\t<td class=\\\" \\\">Guangchuang Yu</td>\\n\\t<td class=\\\" \\\">MeSH Enrichment and Semantic analyses</td>\\n\\t<td class=\\\"  sorting_1\\\">1668</td>\\n</tr><tr id=\\\"pkg_wiggleplotr\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/wiggleplotr.html\\\">wiggleplotr</a></td>\\n\\t<td class=\\\" \\\">Kaur Alasoo</td>\\n\\t<td class=\\\" \\\">Make read coverage plots from BigWig files</td>\\n\\t<td class=\\\"  sorting_1\\\">1668</td>\\n</tr><tr id=\\\"pkg_convert\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/convert.html\\\">convert</a></td>\\n\\t<td class=\\\" \\\">Yee Hwa (Jean) Yang</td>\\n\\t<td class=\\\" \\\">Convert Microarray Data Objects</td>\\n\\t<td class=\\\"  sorting_1\\\">1671</td>\\n</tr><tr id=\\\"pkg_ensemblVEP\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ensemblVEP.html\\\">ensemblVEP</a></td>\\n\\t<td class=\\\" \\\">Bioconductor Package Maintainer</td>\\n\\t<td class=\\\" \\\">R Interface to Ensembl Variant Effect Predictor</td>\\n\\t<td class=\\\"  sorting_1\\\">1671</td>\\n</tr><tr id=\\\"pkg_pipeFrame\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/pipeFrame.html\\\">pipeFrame</a></td>\\n\\t<td class=\\\" \\\">Zheng Wei</td>\\n\\t<td class=\\\" \\\">Pipeline framework for bioinformatics in R</td>\\n\\t<td class=\\\"  sorting_1\\\">1671</td>\\n</tr><tr id=\\\"pkg_conumee\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/conumee.html\\\">conumee</a></td>\\n\\t<td class=\\\" \\\">Volker Hovestadt</td>\\n\\t<td class=\\\" \\\">Enhanced copy-number variation analysis using Illumina DNA methylation arrays</td>\\n\\t<td class=\\\"  sorting_1\\\">1674</td>\\n</tr><tr id=\\\"pkg_microRNA\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/microRNA.html\\\">microRNA</a></td>\\n\\t<td class=\\\" \\\">\\\"James F. Reid\\\"</td>\\n\\t<td class=\\\" \\\">Data and functions for dealing with microRNAs</td>\\n\\t<td class=\\\"  sorting_1\\\">1674</td>\\n</tr><tr id=\\\"pkg_tidySummarizedExperiment\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/tidySummarizedExperiment.html\\\">tidySummarizedExperiment</a></td>\\n\\t<td class=\\\" \\\">Stefano Mangiola</td>\\n\\t<td class=\\\" \\\">Brings SummarizedExperiment to the Tidyverse</td>\\n\\t<td class=\\\"  sorting_1\\\">1674</td>\\n</tr><tr id=\\\"pkg_CMA\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/CMA.html\\\">CMA</a></td>\\n\\t<td class=\\\" \\\">Roman Hornung</td>\\n\\t<td class=\\\" \\\">Synthesis of microarray-based classification</td>\\n\\t<td class=\\\"  sorting_1\\\">1677</td>\\n</tr><tr id=\\\"pkg_flowUtils\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/flowUtils.html\\\">flowUtils</a></td>\\n\\t<td class=\\\" \\\">Josef Spidlen</td>\\n\\t<td class=\\\" \\\">Utilities for flow cytometry</td>\\n\\t<td class=\\\"  sorting_1\\\">1677</td>\\n</tr><tr id=\\\"pkg_tidybulk\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/tidybulk.html\\\">tidybulk</a></td>\\n\\t<td class=\\\" \\\">Stefano Mangiola</td>\\n\\t<td class=\\\" \\\">Brings transcriptomics to the tidyverse</td>\\n\\t<td class=\\\"  sorting_1\\\">1677</td>\\n</tr><tr id=\\\"pkg_cellxgenedp\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/cellxgenedp.html\\\">cellxgenedp</a></td>\\n\\t<td class=\\\" \\\">Martin Morgan</td>\\n\\t<td class=\\\" \\\">Discover and Access Single Cell Data Sets in the cellxgene Data Portal</td>\\n\\t<td class=\\\"  sorting_1\\\">1680</td>\\n</tr><tr id=\\\"pkg_SpatialDecon\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/SpatialDecon.html\\\">SpatialDecon</a></td>\\n\\t<td class=\\\" \\\">Maddy Griswold</td>\\n\\t<td class=\\\" \\\">Deconvolution of mixed cells from spatial and/or bulk gene expression data</td>\\n\\t<td class=\\\"  sorting_1\\\">1680</td>\\n</tr><tr id=\\\"pkg_IPO\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/IPO.html\\\">IPO</a></td>\\n\\t<td class=\\\" \\\">Thomas Lieb</td>\\n\\t<td class=\\\" \\\">Automated Optimization of XCMS Data Processing parameters</td>\\n\\t<td class=\\\"  sorting_1\\\">1682</td>\\n</tr><tr id=\\\"pkg_ViSEAGO\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ViSEAGO.html\\\">ViSEAGO</a></td>\\n\\t<td class=\\\" \\\">Aurelien Brionne</td>\\n\\t<td class=\\\" \\\">ViSEAGO: a Bioconductor package for clustering biological functions using Gene Ontology and semantic similarity</td>\\n\\t<td class=\\\"  sorting_1\\\">1682</td>\\n</tr><tr id=\\\"pkg_debrowser\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/debrowser.html\\\">debrowser</a></td>\\n\\t<td class=\\\" \\\">Alper Kucukural</td>\\n\\t<td class=\\\" \\\">Interactive Differential Expresion Analysis Browser</td>\\n\\t<td class=\\\"  sorting_1\\\">1684</td>\\n</tr><tr id=\\\"pkg_MLSeq\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/MLSeq.html\\\">MLSeq</a></td>\\n\\t<td class=\\\" \\\">Gokmen Zararsiz</td>\\n\\t<td class=\\\" \\\">Machine Learning Interface for RNA-Seq Data</td>\\n\\t<td class=\\\"  sorting_1\\\">1684</td>\\n</tr><tr id=\\\"pkg_tracktables\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/tracktables.html\\\">tracktables</a></td>\\n\\t<td class=\\\" \\\">Tom Carroll</td>\\n\\t<td class=\\\" \\\">Build IGV tracks and HTML reports</td>\\n\\t<td class=\\\"  sorting_1\\\">1684</td>\\n</tr><tr id=\\\"pkg_cola\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/cola.html\\\">cola</a></td>\\n\\t<td class=\\\" \\\">Zuguang Gu</td>\\n\\t<td class=\\\" \\\">A Framework for Consensus Partitioning</td>\\n\\t<td class=\\\"  sorting_1\\\">1687</td>\\n</tr><tr id=\\\"pkg_schex\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/schex.html\\\">schex</a></td>\\n\\t<td class=\\\" \\\">Saskia Freytag</td>\\n\\t<td class=\\\" \\\">Hexbin plots for single cell omics data</td>\\n\\t<td class=\\\"  sorting_1\\\">1687</td>\\n</tr><tr id=\\\"pkg_splots\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/splots.html\\\">splots</a></td>\\n\\t<td class=\\\" \\\">Wolfgang Huber</td>\\n\\t<td class=\\\" \\\">Visualization of high-throughput assays in microtitre plate or slide format</td>\\n\\t<td class=\\\"  sorting_1\\\">1687</td>\\n</tr><tr id=\\\"pkg_DNABarcodes\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/DNABarcodes.html\\\">DNABarcodes</a></td>\\n\\t<td class=\\\" \\\">Tilo Buschmann</td>\\n\\t<td class=\\\" \\\">A tool for creating and analysing DNA barcodes used in Next Generation Sequencing multiplexing experiments</td>\\n\\t<td class=\\\"  sorting_1\\\">1690</td>\\n</tr><tr id=\\\"pkg_genomeIntervals\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/genomeIntervals.html\\\">genomeIntervals</a></td>\\n\\t<td class=\\\" \\\">Julien Gagneur</td>\\n\\t<td class=\\\" \\\">Operations on genomic intervals</td>\\n\\t<td class=\\\"  sorting_1\\\">1690</td>\\n</tr><tr id=\\\"pkg_normr\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/normr.html\\\">normr</a></td>\\n\\t<td class=\\\" \\\">Johannes Helmuth</td>\\n\\t<td class=\\\" \\\">Normalization and difference calling in ChIP-seq data</td>\\n\\t<td class=\\\"  sorting_1\\\">1690</td>\\n</tr><tr id=\\\"pkg_plier\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/plier.html\\\">plier</a></td>\\n\\t<td class=\\\" \\\">Crispin Miller</td>\\n\\t<td class=\\\" \\\">Implements the Affymetrix PLIER algorithm</td>\\n\\t<td class=\\\"  sorting_1\\\">1690</td>\\n</tr><tr id=\\\"pkg_RNASeqPower\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/RNASeqPower.html\\\">RNASeqPower</a></td>\\n\\t<td class=\\\" \\\">Terry M Therneau</td>\\n\\t<td class=\\\" \\\">Sample size for RNAseq studies</td>\\n\\t<td class=\\\"  sorting_1\\\">1690</td>\\n</tr><tr id=\\\"pkg_aCGH\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/aCGH.html\\\">aCGH</a></td>\\n\\t<td class=\\\" \\\">Peter Dimitrov</td>\\n\\t<td class=\\\" \\\">Classes and functions for Array Comparative Genomic Hybridization data</td>\\n\\t<td class=\\\"  sorting_1\\\">1695</td>\\n</tr><tr id=\\\"pkg_fabia\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/fabia.html\\\">fabia</a></td>\\n\\t<td class=\\\" \\\">Andreas Mitterecker</td>\\n\\t<td class=\\\" \\\">FABIA: Factor Analysis for Bicluster Acquisition</td>\\n\\t<td class=\\\"  sorting_1\\\">1695</td>\\n</tr><tr id=\\\"pkg_miRBaseConverter\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/miRBaseConverter.html\\\">miRBaseConverter</a></td>\\n\\t<td class=\\\" \\\">Taosheng Xu</td>\\n\\t<td class=\\\" \\\">A comprehensive and high-efficiency tool for converting and retrieving the information of miRNAs in different miRBase versions</td>\\n\\t<td class=\\\"  sorting_1\\\">1695</td>\\n</tr><tr id=\\\"pkg_dmrseq\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/dmrseq.html\\\">dmrseq</a></td>\\n\\t<td class=\\\" \\\">Keegan Korthauer</td>\\n\\t<td class=\\\" \\\">Detection and inference of differentially methylated regions from Whole Genome Bisulfite Sequencing</td>\\n\\t<td class=\\\"  sorting_1\\\">1698</td>\\n</tr><tr id=\\\"pkg_EBarrays\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/EBarrays.html\\\">EBarrays</a></td>\\n\\t<td class=\\\" \\\">Ming Yuan</td>\\n\\t<td class=\\\" \\\">Unified Approach for Simultaneous Gene Clustering and Differential Expression Identification</td>\\n\\t<td class=\\\"  sorting_1\\\">1698</td>\\n</tr><tr id=\\\"pkg_MethylSeekR\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/MethylSeekR.html\\\">MethylSeekR</a></td>\\n\\t<td class=\\\" \\\">Lukas Burger</td>\\n\\t<td class=\\\" \\\">Segmentation of Bis-seq data</td>\\n\\t<td class=\\\"  sorting_1\\\">1700</td>\\n</tr><tr id=\\\"pkg_SingleCellSignalR\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/SingleCellSignalR.html\\\">SingleCellSignalR</a></td>\\n\\t<td class=\\\" \\\">Jacques Colinge</td>\\n\\t<td class=\\\" \\\">Cell Signalling Using Single Cell RNAseq Data Analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">1700</td>\\n</tr><tr id=\\\"pkg_topconfects\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/topconfects.html\\\">topconfects</a></td>\\n\\t<td class=\\\" \\\">Paul Harrison</td>\\n\\t<td class=\\\" \\\">Top Confident Effect Sizes</td>\\n\\t<td class=\\\"  sorting_1\\\">1700</td>\\n</tr><tr id=\\\"pkg_Harman\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/Harman.html\\\">Harman</a></td>\\n\\t<td class=\\\" \\\">Jason Ross</td>\\n\\t<td class=\\\" \\\">The removal of batch effects from datasets using a PCA and constrained optimisation based technique</td>\\n\\t<td class=\\\"  sorting_1\\\">1703</td>\\n</tr><tr id=\\\"pkg_iCOBRA\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/iCOBRA.html\\\">iCOBRA</a></td>\\n\\t<td class=\\\" \\\">Charlotte Soneson</td>\\n\\t<td class=\\\" \\\">Comparison and Visualization of Ranking and Assignment Methods</td>\\n\\t<td class=\\\"  sorting_1\\\">1703</td>\\n</tr><tr id=\\\"pkg_clustifyr\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/clustifyr.html\\\">clustifyr</a></td>\\n\\t<td class=\\\" \\\">Kent Riemondy</td>\\n\\t<td class=\\\" \\\">Classifier for Single-cell RNA-seq Using Cell Clusters</td>\\n\\t<td class=\\\"  sorting_1\\\">1705</td>\\n</tr><tr id=\\\"pkg_StructuralVariantAnnotation\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/StructuralVariantAnnotation.html\\\">StructuralVariantAnnotation</a></td>\\n\\t<td class=\\\" \\\">Daniel Cameron</td>\\n\\t<td class=\\\" \\\">Variant annotations for structural variants</td>\\n\\t<td class=\\\"  sorting_1\\\">1705</td>\\n</tr><tr id=\\\"pkg_BRGenomics\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/BRGenomics.html\\\">BRGenomics</a></td>\\n\\t<td class=\\\" \\\">Mike DeBerardine</td>\\n\\t<td class=\\\" \\\">Tools for the Efficient Analysis of High-Resolution Genomics Data</td>\\n\\t<td class=\\\"  sorting_1\\\">1707</td>\\n</tr><tr id=\\\"pkg_CoGAPS\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/CoGAPS.html\\\">CoGAPS</a></td>\\n\\t<td class=\\\" \\\">Elana J. Fertig, Thomas D. Sherman, Jeanette Johnson</td>\\n\\t<td class=\\\" \\\">Coordinated Gene Activity in Pattern Sets</td>\\n\\t<td class=\\\"  sorting_1\\\">1707</td>\\n</tr><tr id=\\\"pkg_snm\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/snm.html\\\">snm</a></td>\\n\\t<td class=\\\" \\\">John D. Storey</td>\\n\\t<td class=\\\" \\\">Supervised Normalization of Microarrays</td>\\n\\t<td class=\\\"  sorting_1\\\">1707</td>\\n</tr><tr id=\\\"pkg_scDD\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/scDD.html\\\">scDD</a></td>\\n\\t<td class=\\\" \\\">Keegan Korthauer</td>\\n\\t<td class=\\\" \\\">Mixture modeling of single-cell RNA-seq data to identify genes with differential distributions</td>\\n\\t<td class=\\\"  sorting_1\\\">1710</td>\\n</tr><tr id=\\\"pkg_MetaboAnnotation\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/MetaboAnnotation.html\\\">MetaboAnnotation</a></td>\\n\\t<td class=\\\" \\\">Johannes Rainer</td>\\n\\t<td class=\\\" \\\">Utilities for Annotation of Metabolomics Data</td>\\n\\t<td class=\\\"  sorting_1\\\">1711</td>\\n</tr><tr id=\\\"pkg_sangeranalyseR\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/sangeranalyseR.html\\\">sangeranalyseR</a></td>\\n\\t<td class=\\\" \\\">Kuan-Hao Chao</td>\\n\\t<td class=\\\" \\\">sangeranalyseR: a suite of functions for the analysis of Sanger sequence data in R</td>\\n\\t<td class=\\\"  sorting_1\\\">1711</td>\\n</tr><tr id=\\\"pkg_sSeq\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/sSeq.html\\\">sSeq</a></td>\\n\\t<td class=\\\" \\\">Danni Yu</td>\\n\\t<td class=\\\" \\\">Shrinkage estimation of dispersion in Negative Binomial models for RNA-seq experiments with small sample size</td>\\n\\t<td class=\\\"  sorting_1\\\">1711</td>\\n</tr><tr id=\\\"pkg_drawProteins\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/drawProteins.html\\\">drawProteins</a></td>\\n\\t<td class=\\\" \\\">Paul Brennan</td>\\n\\t<td class=\\\" \\\">Package to Draw Protein Schematics from Uniprot API output</td>\\n\\t<td class=\\\"  sorting_1\\\">1714</td>\\n</tr><tr id=\\\"pkg_plotgardener\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/plotgardener.html\\\">plotgardener</a></td>\\n\\t<td class=\\\" \\\">Nicole Kramer</td>\\n\\t<td class=\\\" \\\">Coordinate-Based Genomic Visualization Package for R</td>\\n\\t<td class=\\\"  sorting_1\\\">1715</td>\\n</tr><tr id=\\\"pkg_miaViz\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/miaViz.html\\\">miaViz</a></td>\\n\\t<td class=\\\" \\\">Tuomas Borman</td>\\n\\t<td class=\\\" \\\">Microbiome Analysis Plotting and Visualization</td>\\n\\t<td class=\\\"  sorting_1\\\">1716</td>\\n</tr><tr id=\\\"pkg_MSnID\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/MSnID.html\\\">MSnID</a></td>\\n\\t<td class=\\\" \\\">Vlad Petyuk</td>\\n\\t<td class=\\\" \\\">Utilities for Exploration and Assessment of Confidence of LC-MSn Proteomics Identifications</td>\\n\\t<td class=\\\"  sorting_1\\\">1716</td>\\n</tr><tr id=\\\"pkg_SomaticSignatures\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/SomaticSignatures.html\\\">SomaticSignatures</a></td>\\n\\t<td class=\\\" \\\">Julian Gehring</td>\\n\\t<td class=\\\" \\\">Somatic Signatures</td>\\n\\t<td class=\\\"  sorting_1\\\">1716</td>\\n</tr><tr id=\\\"pkg_rBiopaxParser\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/rBiopaxParser.html\\\">rBiopaxParser</a></td>\\n\\t<td class=\\\" \\\">Frank Kramer</td>\\n\\t<td class=\\\" \\\">Parses BioPax files and represents them in R</td>\\n\\t<td class=\\\"  sorting_1\\\">1719</td>\\n</tr><tr id=\\\"pkg_AgiMicroRna\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/AgiMicroRna.html\\\">AgiMicroRna</a></td>\\n\\t<td class=\\\" \\\">Pedro Lopez-Romero</td>\\n\\t<td class=\\\" \\\">Processing and Differential Expression Analysis of Agilent microRNA chips</td>\\n\\t<td class=\\\"  sorting_1\\\">1720</td>\\n</tr><tr id=\\\"pkg_rawrr\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/rawrr.html\\\">rawrr</a></td>\\n\\t<td class=\\\" \\\">Christian Panse</td>\\n\\t<td class=\\\" \\\">Direct Access to Orbitrap Data and Beyond</td>\\n\\t<td class=\\\"  sorting_1\\\">1721</td>\\n</tr><tr id=\\\"pkg_biocthis\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/biocthis.html\\\">biocthis</a></td>\\n\\t<td class=\\\" \\\">Leonardo Collado-Torres</td>\\n\\t<td class=\\\" \\\">Automate package and project setup for Bioconductor packages</td>\\n\\t<td class=\\\"  sorting_1\\\">1722</td>\\n</tr><tr id=\\\"pkg_Linnorm\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/Linnorm.html\\\">Linnorm</a></td>\\n\\t<td class=\\\" \\\">Ken Shun Hang Yip</td>\\n\\t<td class=\\\" \\\">Linear model and normality based normalization and transformation method (Linnorm)</td>\\n\\t<td class=\\\"  sorting_1\\\">1722</td>\\n</tr><tr id=\\\"pkg_HTqPCR\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/HTqPCR.html\\\">HTqPCR</a></td>\\n\\t<td class=\\\" \\\">Heidi Dvinge</td>\\n\\t<td class=\\\" \\\">Automated analysis of high-throughput qPCR data</td>\\n\\t<td class=\\\"  sorting_1\\\">1724</td>\\n</tr><tr id=\\\"pkg_CompoundDb\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/CompoundDb.html\\\">CompoundDb</a></td>\\n\\t<td class=\\\" \\\">Johannes Rainer</td>\\n\\t<td class=\\\" \\\">Creating and Using (Chemical) Compound Annotation Databases</td>\\n\\t<td class=\\\"  sorting_1\\\">1725</td>\\n</tr><tr id=\\\"pkg_memes\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/memes.html\\\">memes</a></td>\\n\\t<td class=\\\" \\\">Spencer Nystrom</td>\\n\\t<td class=\\\" \\\">motif matching, comparison, and de novo discovery using the MEME Suite</td>\\n\\t<td class=\\\"  sorting_1\\\">1725</td>\\n</tr><tr id=\\\"pkg_ORFik\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ORFik.html\\\">ORFik</a></td>\\n\\t<td class=\\\" \\\">Haakon Tjeldnes</td>\\n\\t<td class=\\\" \\\">Open Reading Frames in Genomics</td>\\n\\t<td class=\\\"  sorting_1\\\">1725</td>\\n</tr><tr id=\\\"pkg_sigFeature\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/sigFeature.html\\\">sigFeature</a></td>\\n\\t<td class=\\\" \\\">Pijush Das Developer</td>\\n\\t<td class=\\\" \\\">sigFeature: Significant feature selection using SVM-RFE &amp; t-statistic</td>\\n\\t<td class=\\\"  sorting_1\\\">1725</td>\\n</tr><tr id=\\\"pkg_HiCcompare\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/HiCcompare.html\\\">HiCcompare</a></td>\\n\\t<td class=\\\" \\\">John Stansfield, Mikhail Dozmorov</td>\\n\\t<td class=\\\" \\\">HiCcompare: Joint normalization and comparative analysis of multiple Hi-C datasets</td>\\n\\t<td class=\\\"  sorting_1\\\">1729</td>\\n</tr><tr id=\\\"pkg_HilbertVis\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/HilbertVis.html\\\">HilbertVis</a></td>\\n\\t<td class=\\\" \\\">Simon Anders</td>\\n\\t<td class=\\\" \\\">Hilbert curve visualization</td>\\n\\t<td class=\\\"  sorting_1\\\">1729</td>\\n</tr><tr id=\\\"pkg_proDA\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/proDA.html\\\">proDA</a></td>\\n\\t<td class=\\\" \\\">Constantin Ahlmann-Eltze</td>\\n\\t<td class=\\\" \\\">Differential Abundance Analysis of Label-Free Mass Spectrometry Data</td>\\n\\t<td class=\\\"  sorting_1\\\">1731</td>\\n</tr><tr id=\\\"pkg_CEMiTool\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/CEMiTool.html\\\">CEMiTool</a></td>\\n\\t<td class=\\\" \\\">Helder Nakaya</td>\\n\\t<td class=\\\" \\\">Co-expression Modules identification Tool</td>\\n\\t<td class=\\\"  sorting_1\\\">1732</td>\\n</tr><tr id=\\\"pkg_miloR\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/miloR.html\\\">miloR</a></td>\\n\\t<td class=\\\" \\\">Mike Morgan</td>\\n\\t<td class=\\\" \\\">Differential neighbourhood abundance testing on a graph</td>\\n\\t<td class=\\\"  sorting_1\\\">1733</td>\\n</tr><tr id=\\\"pkg_msmsTests\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/msmsTests.html\\\">msmsTests</a></td>\\n\\t<td class=\\\" \\\">Josep Gregori i Font</td>\\n\\t<td class=\\\" \\\">LC-MS/MS Differential Expression Tests</td>\\n\\t<td class=\\\"  sorting_1\\\">1733</td>\\n</tr><tr id=\\\"pkg_pathifier\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/pathifier.html\\\">pathifier</a></td>\\n\\t<td class=\\\" \\\">Assif Yitzhaky</td>\\n\\t<td class=\\\" \\\">Quantify deregulation of pathways in cancer</td>\\n\\t<td class=\\\"  sorting_1\\\">1735</td>\\n</tr><tr id=\\\"pkg_PureCN\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/PureCN.html\\\">PureCN</a></td>\\n\\t<td class=\\\" \\\">Markus Riester</td>\\n\\t<td class=\\\" \\\">Copy number calling and SNV classification using targeted short read sequencing</td>\\n\\t<td class=\\\"  sorting_1\\\">1735</td>\\n</tr><tr id=\\\"pkg_OUTRIDER\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/OUTRIDER.html\\\">OUTRIDER</a></td>\\n\\t<td class=\\\" \\\">Christian Mertes</td>\\n\\t<td class=\\\" \\\">OUTRIDER - OUTlier in RNA-Seq fInDER</td>\\n\\t<td class=\\\"  sorting_1\\\">1737</td>\\n</tr><tr id=\\\"pkg_BiocWorkflowTools\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/BiocWorkflowTools.html\\\">BiocWorkflowTools</a></td>\\n\\t<td class=\\\" \\\">Mike Smith</td>\\n\\t<td class=\\\" \\\">Tools to aid the development of Bioconductor Workflow packages</td>\\n\\t<td class=\\\"  sorting_1\\\">1738</td>\\n</tr><tr id=\\\"pkg_PharmacoGx\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/PharmacoGx.html\\\">PharmacoGx</a></td>\\n\\t<td class=\\\" \\\">Benjamin Haibe-Kains</td>\\n\\t<td class=\\\" \\\">Analysis of Large-Scale Pharmacogenomic Data</td>\\n\\t<td class=\\\"  sorting_1\\\">1739</td>\\n</tr><tr id=\\\"pkg_ROTS\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ROTS.html\\\">ROTS</a></td>\\n\\t<td class=\\\" \\\">Tomi Suomi</td>\\n\\t<td class=\\\" \\\">Reproducibility-Optimized Test Statistic</td>\\n\\t<td class=\\\"  sorting_1\\\">1739</td>\\n</tr><tr id=\\\"pkg_BayesSpace\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/BayesSpace.html\\\">BayesSpace</a></td>\\n\\t<td class=\\\" \\\">Matt Stone</td>\\n\\t<td class=\\\" \\\">Clustering and Resolution Enhancement of Spatial Transcriptomes</td>\\n\\t<td class=\\\"  sorting_1\\\">1741</td>\\n</tr><tr id=\\\"pkg_profileplyr\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/profileplyr.html\\\">profileplyr</a></td>\\n\\t<td class=\\\" \\\">Tom Carroll, Doug Barrows</td>\\n\\t<td class=\\\" \\\">Visualization and annotation of read signal over genomic ranges with profileplyr</td>\\n\\t<td class=\\\"  sorting_1\\\">1741</td>\\n</tr><tr id=\\\"pkg_quantiseqr\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/quantiseqr.html\\\">quantiseqr</a></td>\\n\\t<td class=\\\" \\\">Federico Marini</td>\\n\\t<td class=\\\" \\\">Quantification of the Tumor Immune contexture from RNA-seq data</td>\\n\\t<td class=\\\"  sorting_1\\\">1743</td>\\n</tr><tr id=\\\"pkg_TOAST\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/TOAST.html\\\">TOAST</a></td>\\n\\t<td class=\\\" \\\">Ziyi Li</td>\\n\\t<td class=\\\" \\\">Tools for the analysis of heterogeneous tissues</td>\\n\\t<td class=\\\"  sorting_1\\\">1743</td>\\n</tr><tr id=\\\"pkg_LOLA\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/LOLA.html\\\">LOLA</a></td>\\n\\t<td class=\\\" \\\">Nathan Sheffield</td>\\n\\t<td class=\\\" \\\">Locus overlap analysis for enrichment of genomic ranges</td>\\n\\t<td class=\\\"  sorting_1\\\">1745</td>\\n</tr><tr id=\\\"pkg_msmsEDA\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/msmsEDA.html\\\">msmsEDA</a></td>\\n\\t<td class=\\\" \\\">Josep Gregori</td>\\n\\t<td class=\\\" \\\">Exploratory Data Analysis of LC-MS/MS data by spectral counts</td>\\n\\t<td class=\\\"  sorting_1\\\">1745</td>\\n</tr><tr id=\\\"pkg_ENmix\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ENmix.html\\\">ENmix</a></td>\\n\\t<td class=\\\" \\\">Zongli Xu</td>\\n\\t<td class=\\\" \\\">Quality control and analysis tools for Illumina DNA methylation BeadChip</td>\\n\\t<td class=\\\"  sorting_1\\\">1747</td>\\n</tr><tr id=\\\"pkg_orthogene\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/orthogene.html\\\">orthogene</a></td>\\n\\t<td class=\\\" \\\">Brian Schilder</td>\\n\\t<td class=\\\" \\\">Interspecies gene mapping</td>\\n\\t<td class=\\\"  sorting_1\\\">1748</td>\\n</tr><tr id=\\\"pkg_singleCellTK\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/singleCellTK.html\\\">singleCellTK</a></td>\\n\\t<td class=\\\" \\\">Yichen Wang</td>\\n\\t<td class=\\\" \\\">Comprehensive and Interactive Analysis of Single Cell RNA-Seq Data</td>\\n\\t<td class=\\\"  sorting_1\\\">1748</td>\\n</tr><tr id=\\\"pkg_EGSEA\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/EGSEA.html\\\">EGSEA</a></td>\\n\\t<td class=\\\" \\\">Monther Alhamdoosh</td>\\n\\t<td class=\\\" \\\">Ensemble of Gene Set Enrichment Analyses</td>\\n\\t<td class=\\\"  sorting_1\\\">1750</td>\\n</tr><tr id=\\\"pkg_VennDetail\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/VennDetail.html\\\">VennDetail</a></td>\\n\\t<td class=\\\" \\\">Kai Guo</td>\\n\\t<td class=\\\" \\\">A package for visualization and extract details</td>\\n\\t<td class=\\\"  sorting_1\\\">1750</td>\\n</tr><tr id=\\\"pkg_stageR\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/stageR.html\\\">stageR</a></td>\\n\\t<td class=\\\" \\\">Koen Van den Berge</td>\\n\\t<td class=\\\" \\\">stageR: stage-wise analysis of high throughput gene expression data in R</td>\\n\\t<td class=\\\"  sorting_1\\\">1752</td>\\n</tr><tr id=\\\"pkg_synergyfinder\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/synergyfinder.html\\\">synergyfinder</a></td>\\n\\t<td class=\\\" \\\">Shuyu Zheng</td>\\n\\t<td class=\\\" \\\">Calculate and Visualize Synergy Scores for Drug Combinations</td>\\n\\t<td class=\\\"  sorting_1\\\">1753</td>\\n</tr><tr id=\\\"pkg_biobroom\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/biobroom.html\\\">biobroom</a></td>\\n\\t<td class=\\\" \\\">John D. Storey and Andrew J. Bass</td>\\n\\t<td class=\\\" \\\">Turn Bioconductor objects into tidy data frames</td>\\n\\t<td class=\\\"  sorting_1\\\">1754</td>\\n</tr><tr id=\\\"pkg_cleaver\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/cleaver.html\\\">cleaver</a></td>\\n\\t<td class=\\\" \\\">Sebastian Gibb</td>\\n\\t<td class=\\\" \\\">Cleavage of Polypeptide Sequences</td>\\n\\t<td class=\\\"  sorting_1\\\">1754</td>\\n</tr><tr id=\\\"pkg_cytomapper\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/cytomapper.html\\\">cytomapper</a></td>\\n\\t<td class=\\\" \\\">Nils Eling</td>\\n\\t<td class=\\\" \\\">Visualization of highly multiplexed imaging data in R</td>\\n\\t<td class=\\\"  sorting_1\\\">1754</td>\\n</tr><tr id=\\\"pkg_DeconRNASeq\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/DeconRNASeq.html\\\">DeconRNASeq</a></td>\\n\\t<td class=\\\" \\\">Ting Gong</td>\\n\\t<td class=\\\" \\\">Deconvolution of Heterogeneous Tissue Samples for mRNA-Seq data</td>\\n\\t<td class=\\\"  sorting_1\\\">1754</td>\\n</tr><tr id=\\\"pkg_ExperimentHubData\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ExperimentHubData.html\\\">ExperimentHubData</a></td>\\n\\t<td class=\\\" \\\">Bioconductor Package Maintainer</td>\\n\\t<td class=\\\" \\\">Add resources to ExperimentHub</td>\\n\\t<td class=\\\"  sorting_1\\\">1758</td>\\n</tr><tr id=\\\"pkg_HTSFilter\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/HTSFilter.html\\\">HTSFilter</a></td>\\n\\t<td class=\\\" \\\">Andrea Rau</td>\\n\\t<td class=\\\" \\\">Filter replicated high-throughput transcriptome sequencing data</td>\\n\\t<td class=\\\"  sorting_1\\\">1758</td>\\n</tr><tr id=\\\"pkg_genbankr\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/genbankr.html\\\">genbankr</a></td>\\n\\t<td class=\\\" \\\">Gabriel Becker</td>\\n\\t<td class=\\\" \\\">Parsing GenBank files into semantically useful objects</td>\\n\\t<td class=\\\"  sorting_1\\\">1760</td>\\n</tr><tr id=\\\"pkg_CoreGx\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/CoreGx.html\\\">CoreGx</a></td>\\n\\t<td class=\\\" \\\">Benjamin Haibe-Kains</td>\\n\\t<td class=\\\" \\\">Classes and Functions to Serve as the Basis for Other 'Gx' Packages</td>\\n\\t<td class=\\\"  sorting_1\\\">1761</td>\\n</tr><tr id=\\\"pkg_PADOG\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/PADOG.html\\\">PADOG</a></td>\\n\\t<td class=\\\" \\\">Adi L. Tarca</td>\\n\\t<td class=\\\" \\\">Pathway Analysis with Down-weighting of Overlapping Genes (PADOG)</td>\\n\\t<td class=\\\"  sorting_1\\\">1761</td>\\n</tr><tr id=\\\"pkg_Rhisat2\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/Rhisat2.html\\\">Rhisat2</a></td>\\n\\t<td class=\\\" \\\">Charlotte Soneson</td>\\n\\t<td class=\\\" \\\">R Wrapper for HISAT2 Aligner</td>\\n\\t<td class=\\\"  sorting_1\\\">1761</td>\\n</tr><tr id=\\\"pkg_MetaboCoreUtils\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/MetaboCoreUtils.html\\\">MetaboCoreUtils</a></td>\\n\\t<td class=\\\" \\\">Johannes Rainer</td>\\n\\t<td class=\\\" \\\">Core Utils for Metabolomics Data</td>\\n\\t<td class=\\\"  sorting_1\\\">1764</td>\\n</tr><tr id=\\\"pkg_cn.mops\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/cn.mops.html\\\">cn.mops</a></td>\\n\\t<td class=\\\" \\\">Gundula Povysil</td>\\n\\t<td class=\\\" \\\">cn.mops - Mixture of Poissons for CNV detection in NGS data</td>\\n\\t<td class=\\\"  sorting_1\\\">1765</td>\\n</tr><tr id=\\\"pkg_philr\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/philr.html\\\">philr</a></td>\\n\\t<td class=\\\" \\\">Justin Silverman</td>\\n\\t<td class=\\\" \\\">Phylogenetic partitioning based ILR transform for metagenomics data</td>\\n\\t<td class=\\\"  sorting_1\\\">1765</td>\\n</tr><tr id=\\\"pkg_BiocSet\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/BiocSet.html\\\">BiocSet</a></td>\\n\\t<td class=\\\" \\\">Kayla Morrell</td>\\n\\t<td class=\\\" \\\">Representing Different Biological Sets</td>\\n\\t<td class=\\\"  sorting_1\\\">1767</td>\\n</tr><tr id=\\\"pkg_cicero\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/cicero.html\\\">cicero</a></td>\\n\\t<td class=\\\" \\\">Hannah Pliner</td>\\n\\t<td class=\\\" \\\">Predict cis-co-accessibility from single-cell chromatin accessibility data</td>\\n\\t<td class=\\\"  sorting_1\\\">1767</td>\\n</tr><tr id=\\\"pkg_flowDensity\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/flowDensity.html\\\">flowDensity</a></td>\\n\\t<td class=\\\" \\\">Mehrnoush Malek</td>\\n\\t<td class=\\\" \\\">Sequential Flow Cytometry Data Gating</td>\\n\\t<td class=\\\"  sorting_1\\\">1769</td>\\n</tr><tr id=\\\"pkg_MSstatsTMT\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/MSstatsTMT.html\\\">MSstatsTMT</a></td>\\n\\t<td class=\\\" \\\">Ting Huang</td>\\n\\t<td class=\\\" \\\">Protein Significance Analysis in shotgun mass spectrometry-based proteomic experiments with tandem mass tag (TMT) labeling</td>\\n\\t<td class=\\\"  sorting_1\\\">1769</td>\\n</tr><tr id=\\\"pkg_GEOmetadb\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/GEOmetadb.html\\\">GEOmetadb</a></td>\\n\\t<td class=\\\" \\\">Jack Zhu</td>\\n\\t<td class=\\\" \\\">A compilation of metadata from NCBI GEO</td>\\n\\t<td class=\\\"  sorting_1\\\">1771</td>\\n</tr><tr id=\\\"pkg_quantro\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/quantro.html\\\">quantro</a></td>\\n\\t<td class=\\\" \\\">Stephanie Hicks</td>\\n\\t<td class=\\\" \\\">A test for when to use quantile normalization</td>\\n\\t<td class=\\\"  sorting_1\\\">1771</td>\\n</tr><tr id=\\\"pkg_muscle\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/muscle.html\\\">muscle</a></td>\\n\\t<td class=\\\" \\\">Alex T. Kalinka</td>\\n\\t<td class=\\\" \\\">Multiple Sequence Alignment with MUSCLE</td>\\n\\t<td class=\\\"  sorting_1\\\">1773</td>\\n</tr><tr id=\\\"pkg_ReactomeGSA\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ReactomeGSA.html\\\">ReactomeGSA</a></td>\\n\\t<td class=\\\" \\\">Johannes Griss</td>\\n\\t<td class=\\\" \\\">Client for the Reactome Analysis Service for comparative multi-omics gene set analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">1773</td>\\n</tr><tr id=\\\"pkg_DEGseq\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/DEGseq.html\\\">DEGseq</a></td>\\n\\t<td class=\\\" \\\">Likun Wang</td>\\n\\t<td class=\\\" \\\">Identify Differentially Expressed Genes from RNA-seq data</td>\\n\\t<td class=\\\"  sorting_1\\\">1775</td>\\n</tr><tr id=\\\"pkg_SIMLR\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/SIMLR.html\\\">SIMLR</a></td>\\n\\t<td class=\\\" \\\">Luca De Sano</td>\\n\\t<td class=\\\" \\\">Single-cell Interpretation via Multi-kernel LeaRning (SIMLR)</td>\\n\\t<td class=\\\"  sorting_1\\\">1775</td>\\n</tr><tr id=\\\"pkg_densvis\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/densvis.html\\\">densvis</a></td>\\n\\t<td class=\\\" \\\">Alan O'Callaghan</td>\\n\\t<td class=\\\" \\\">Density-Preserving Data Visualization via Non-Linear Dimensionality Reduction</td>\\n\\t<td class=\\\"  sorting_1\\\">1777</td>\\n</tr><tr id=\\\"pkg_GOfuncR\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/GOfuncR.html\\\">GOfuncR</a></td>\\n\\t<td class=\\\" \\\">Steffi Grote</td>\\n\\t<td class=\\\" \\\">Gene ontology enrichment using FUNC</td>\\n\\t<td class=\\\"  sorting_1\\\">1777</td>\\n</tr><tr id=\\\"pkg_MeSHDbi\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/MeSHDbi.html\\\">MeSHDbi</a></td>\\n\\t<td class=\\\" \\\">Koki Tsuyuzaki</td>\\n\\t<td class=\\\" \\\">DBI to construct MeSH-related package from sqlite file</td>\\n\\t<td class=\\\"  sorting_1\\\">1777</td>\\n</tr><tr id=\\\"pkg_DEqMS\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/DEqMS.html\\\">DEqMS</a></td>\\n\\t<td class=\\\" \\\">Yafeng Zhu</td>\\n\\t<td class=\\\" \\\">a tool to perform statistical analysis of differential protein expression for quantitative proteomics data.</td>\\n\\t<td class=\\\"  sorting_1\\\">1780</td>\\n</tr><tr id=\\\"pkg_fmcsR\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/fmcsR.html\\\">fmcsR</a></td>\\n\\t<td class=\\\" \\\">Thomas Girke</td>\\n\\t<td class=\\\" \\\">Mismatch Tolerant Maximum Common Substructure Searching</td>\\n\\t<td class=\\\"  sorting_1\\\">1781</td>\\n</tr><tr id=\\\"pkg_RnBeads\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/RnBeads.html\\\">RnBeads</a></td>\\n\\t<td class=\\\" \\\">Fabian Mueller</td>\\n\\t<td class=\\\" \\\">RnBeads</td>\\n\\t<td class=\\\"  sorting_1\\\">1781</td>\\n</tr><tr id=\\\"pkg_lefser\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/lefser.html\\\">lefser</a></td>\\n\\t<td class=\\\" \\\">Asya Khleborodova</td>\\n\\t<td class=\\\" \\\">R implementation of the LEfSE method for microbiome biomarker discovery</td>\\n\\t<td class=\\\"  sorting_1\\\">1783</td>\\n</tr><tr id=\\\"pkg_bioDist\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/bioDist.html\\\">bioDist</a></td>\\n\\t<td class=\\\" \\\">Bioconductor Package Maintainer</td>\\n\\t<td class=\\\" \\\">Different distance measures</td>\\n\\t<td class=\\\"  sorting_1\\\">1784</td>\\n</tr><tr id=\\\"pkg_CancerSubtypes\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/CancerSubtypes.html\\\">CancerSubtypes</a></td>\\n\\t<td class=\\\" \\\">Taosheng Xu</td>\\n\\t<td class=\\\" \\\">Cancer subtypes identification, validation and visualization based on multiple genomic data sets</td>\\n\\t<td class=\\\"  sorting_1\\\">1784</td>\\n</tr><tr id=\\\"pkg_flowClean\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/flowClean.html\\\">flowClean</a></td>\\n\\t<td class=\\\" \\\">Kipper Fletez-Brant</td>\\n\\t<td class=\\\" \\\">flowClean</td>\\n\\t<td class=\\\"  sorting_1\\\">1784</td>\\n</tr><tr id=\\\"pkg_Rqc\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/Rqc.html\\\">Rqc</a></td>\\n\\t<td class=\\\" \\\">Welliton Souza</td>\\n\\t<td class=\\\" \\\">Quality Control Tool for High-Throughput Sequencing Data</td>\\n\\t<td class=\\\"  sorting_1\\\">1784</td>\\n</tr><tr id=\\\"pkg_hopach\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/hopach.html\\\">hopach</a></td>\\n\\t<td class=\\\" \\\">Katherine S. Pollard</td>\\n\\t<td class=\\\" \\\">Hierarchical Ordered Partitioning and Collapsing Hybrid (HOPACH)</td>\\n\\t<td class=\\\"  sorting_1\\\">1788</td>\\n</tr><tr id=\\\"pkg_OmicCircos\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/OmicCircos.html\\\">OmicCircos</a></td>\\n\\t<td class=\\\" \\\">Ying Hu</td>\\n\\t<td class=\\\" \\\">High-quality circular visualization of omics data</td>\\n\\t<td class=\\\"  sorting_1\\\">1788</td>\\n</tr><tr id=\\\"pkg_NormqPCR\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/NormqPCR.html\\\">NormqPCR</a></td>\\n\\t<td class=\\\" \\\">James Perkins</td>\\n\\t<td class=\\\" \\\">Functions for normalisation of RT-qPCR data</td>\\n\\t<td class=\\\"  sorting_1\\\">1790</td>\\n</tr><tr id=\\\"pkg_GenomicInteractions\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/GenomicInteractions.html\\\">GenomicInteractions</a></td>\\n\\t<td class=\\\" \\\">Liz Ing-Simmons</td>\\n\\t<td class=\\\" \\\">Utilities for handling genomic interaction data</td>\\n\\t<td class=\\\"  sorting_1\\\">1791</td>\\n</tr><tr id=\\\"pkg_ChemmineOB\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ChemmineOB.html\\\">ChemmineOB</a></td>\\n\\t<td class=\\\" \\\">Thomas Girke</td>\\n\\t<td class=\\\" \\\">R interface to a subset of OpenBabel functionalities</td>\\n\\t<td class=\\\"  sorting_1\\\">1792</td>\\n</tr><tr id=\\\"pkg_BUSpaRse\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/BUSpaRse.html\\\">BUSpaRse</a></td>\\n\\t<td class=\\\" \\\">Lambda Moses</td>\\n\\t<td class=\\\" \\\">kallisto | bustools R utilities</td>\\n\\t<td class=\\\"  sorting_1\\\">1793</td>\\n</tr><tr id=\\\"pkg_MsBackendMgf\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/MsBackendMgf.html\\\">MsBackendMgf</a></td>\\n\\t<td class=\\\" \\\">RforMassSpectrometry Package Maintainer</td>\\n\\t<td class=\\\" \\\">Mass Spectrometry Data Backend for Mascot Generic Format (mgf) Files</td>\\n\\t<td class=\\\"  sorting_1\\\">1793</td>\\n</tr><tr id=\\\"pkg_muscat\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/muscat.html\\\">muscat</a></td>\\n\\t<td class=\\\" \\\">Helena L. Crowell</td>\\n\\t<td class=\\\" \\\">Multi-sample multi-group scRNA-seq data analysis tools</td>\\n\\t<td class=\\\"  sorting_1\\\">1793</td>\\n</tr><tr id=\\\"pkg_AnnotationHubData\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/AnnotationHubData.html\\\">AnnotationHubData</a></td>\\n\\t<td class=\\\" \\\">Bioconductor Package Maintainer</td>\\n\\t<td class=\\\" \\\">Transform public data resources into Bioconductor Data Structures</td>\\n\\t<td class=\\\"  sorting_1\\\">1796</td>\\n</tr><tr id=\\\"pkg_DEFormats\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/DEFormats.html\\\">DEFormats</a></td>\\n\\t<td class=\\\" \\\">Andrzej Ole</td>\\n\\t<td class=\\\" \\\">Differential gene expression data formats converter</td>\\n\\t<td class=\\\"  sorting_1\\\">1796</td>\\n</tr><tr id=\\\"pkg_multiMiR\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/multiMiR.html\\\">multiMiR</a></td>\\n\\t<td class=\\\" \\\">Matt Mulvahill</td>\\n\\t<td class=\\\" \\\">Integration of multiple microRNA-target databases with their disease and drug associations</td>\\n\\t<td class=\\\"  sorting_1\\\">1798</td>\\n</tr><tr id=\\\"pkg_escape\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/escape.html\\\">escape</a></td>\\n\\t<td class=\\\" \\\">Nick Borcherding</td>\\n\\t<td class=\\\" \\\">Easy single cell analysis platform for enrichment</td>\\n\\t<td class=\\\"  sorting_1\\\">1799</td>\\n</tr><tr id=\\\"pkg_MungeSumstats\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/MungeSumstats.html\\\">MungeSumstats</a></td>\\n\\t<td class=\\\" \\\">Alan Murphy</td>\\n\\t<td class=\\\" \\\">Standardise summary statistics from GWAS</td>\\n\\t<td class=\\\"  sorting_1\\\">1799</td>\\n</tr><tr id=\\\"pkg_bambu\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/bambu.html\\\">bambu</a></td>\\n\\t<td class=\\\" \\\">Ying Chen</td>\\n\\t<td class=\\\" \\\">Context-Aware Transcript Quantification from Long Read RNA-Seq data</td>\\n\\t<td class=\\\"  sorting_1\\\">1801</td>\\n</tr><tr id=\\\"pkg_RedeR\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/RedeR.html\\\">RedeR</a></td>\\n\\t<td class=\\\" \\\">Mauro Castro</td>\\n\\t<td class=\\\" \\\">Interactive visualization and manipulation of nested networks</td>\\n\\t<td class=\\\"  sorting_1\\\">1801</td>\\n</tr><tr id=\\\"pkg_GeomxTools\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/GeomxTools.html\\\">GeomxTools</a></td>\\n\\t<td class=\\\" \\\">Nicole Ortogero</td>\\n\\t<td class=\\\" \\\">NanoString GeoMx Tools</td>\\n\\t<td class=\\\"  sorting_1\\\">1803</td>\\n</tr><tr id=\\\"pkg_ReadqPCR\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ReadqPCR.html\\\">ReadqPCR</a></td>\\n\\t<td class=\\\" \\\">James Perkins</td>\\n\\t<td class=\\\" \\\">Read qPCR data</td>\\n\\t<td class=\\\"  sorting_1\\\">1804</td>\\n</tr><tr id=\\\"pkg_annaffy\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/annaffy.html\\\">annaffy</a></td>\\n\\t<td class=\\\" \\\">Colin A. Smith</td>\\n\\t<td class=\\\" \\\">Annotation tools for Affymetrix biological metadata</td>\\n\\t<td class=\\\"  sorting_1\\\">1805</td>\\n</tr><tr id=\\\"pkg_exomeCopy\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/exomeCopy.html\\\">exomeCopy</a></td>\\n\\t<td class=\\\" \\\">Michael Love</td>\\n\\t<td class=\\\" \\\">Copy number variant detection from exome sequencing read depth</td>\\n\\t<td class=\\\"  sorting_1\\\">1805</td>\\n</tr><tr id=\\\"pkg_microbiomeMarker\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/microbiomeMarker.html\\\">microbiomeMarker</a></td>\\n\\t<td class=\\\" \\\">Yang Cao</td>\\n\\t<td class=\\\" \\\">microbiome biomarker analysis toolkit</td>\\n\\t<td class=\\\"  sorting_1\\\">1805</td>\\n</tr><tr id=\\\"pkg_NanoStringNCTools\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/NanoStringNCTools.html\\\">NanoStringNCTools</a></td>\\n\\t<td class=\\\" \\\">Nicole Ortogero</td>\\n\\t<td class=\\\" \\\">NanoString nCounter Tools</td>\\n\\t<td class=\\\"  sorting_1\\\">1808</td>\\n</tr><tr id=\\\"pkg_pRoloc\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/pRoloc.html\\\">pRoloc</a></td>\\n\\t<td class=\\\" \\\">Laurent Gatto</td>\\n\\t<td class=\\\" \\\">A unifying bioinformatics framework for spatial proteomics</td>\\n\\t<td class=\\\"  sorting_1\\\">1808</td>\\n</tr><tr id=\\\"pkg_InteractiveComplexHeatmap\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/InteractiveComplexHeatmap.html\\\">InteractiveComplexHeatmap</a></td>\\n\\t<td class=\\\" \\\">Zuguang Gu</td>\\n\\t<td class=\\\" \\\">Make Interactive Complex Heatmaps</td>\\n\\t<td class=\\\"  sorting_1\\\">1810</td>\\n</tr><tr id=\\\"pkg_IsoformSwitchAnalyzeR\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/IsoformSwitchAnalyzeR.html\\\">IsoformSwitchAnalyzeR</a></td>\\n\\t<td class=\\\" \\\">Kristoffer Vitting-Seerup</td>\\n\\t<td class=\\\" \\\">Identify, Annotate and Visualize Alternative Splicing and Isoform Switches with Functional Consequences from both short- and long-read RNA-seq data.</td>\\n\\t<td class=\\\"  sorting_1\\\">1811</td>\\n</tr><tr id=\\\"pkg_pvca\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/pvca.html\\\">pvca</a></td>\\n\\t<td class=\\\" \\\">Jianying LI</td>\\n\\t<td class=\\\" \\\">Principal Variance Component Analysis (PVCA)</td>\\n\\t<td class=\\\"  sorting_1\\\">1812</td>\\n</tr><tr id=\\\"pkg_hpar\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/hpar.html\\\">hpar</a></td>\\n\\t<td class=\\\" \\\">Laurent Gatto</td>\\n\\t<td class=\\\" \\\">Human Protein Atlas in R</td>\\n\\t<td class=\\\"  sorting_1\\\">1813</td>\\n</tr><tr id=\\\"pkg_diffcyt\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/diffcyt.html\\\">diffcyt</a></td>\\n\\t<td class=\\\" \\\">Lukas M. Weber</td>\\n\\t<td class=\\\" \\\">Differential discovery in high-dimensional cytometry via high-resolution clustering</td>\\n\\t<td class=\\\"  sorting_1\\\">1814</td>\\n</tr><tr id=\\\"pkg_made4\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/made4.html\\\">made4</a></td>\\n\\t<td class=\\\" \\\">Aedin Culhane</td>\\n\\t<td class=\\\" \\\">Multivariate analysis of microarray data using ADE4</td>\\n\\t<td class=\\\"  sorting_1\\\">1815</td>\\n</tr><tr id=\\\"pkg_cqn\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/cqn.html\\\">cqn</a></td>\\n\\t<td class=\\\" \\\">Kasper Daniel Hansen</td>\\n\\t<td class=\\\" \\\">Conditional quantile normalization</td>\\n\\t<td class=\\\"  sorting_1\\\">1816</td>\\n</tr><tr id=\\\"pkg_HMMcopy\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/HMMcopy.html\\\">HMMcopy</a></td>\\n\\t<td class=\\\" \\\">Daniel Lai</td>\\n\\t<td class=\\\" \\\">Copy number prediction with correction for GC and mappability bias for HTS data</td>\\n\\t<td class=\\\"  sorting_1\\\">1816</td>\\n</tr><tr id=\\\"pkg_ELMER\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ELMER.html\\\">ELMER</a></td>\\n\\t<td class=\\\" \\\">Tiago Chedraoui Silva</td>\\n\\t<td class=\\\" \\\">Inferring Regulatory Element Landscapes and Transcription Factor Networks Using Cancer Methylomes</td>\\n\\t<td class=\\\"  sorting_1\\\">1818</td>\\n</tr><tr id=\\\"pkg_GLAD\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/GLAD.html\\\">GLAD</a></td>\\n\\t<td class=\\\" \\\">Philippe Hupe</td>\\n\\t<td class=\\\" \\\">Gain and Loss Analysis of DNA</td>\\n\\t<td class=\\\"  sorting_1\\\">1818</td>\\n</tr><tr id=\\\"pkg_BioNet\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/BioNet.html\\\">BioNet</a></td>\\n\\t<td class=\\\" \\\">Marcus Dittrich</td>\\n\\t<td class=\\\" \\\">Routines for the functional analysis of biological networks</td>\\n\\t<td class=\\\"  sorting_1\\\">1820</td>\\n</tr><tr id=\\\"pkg_heatmaps\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/heatmaps.html\\\">heatmaps</a></td>\\n\\t<td class=\\\" \\\">Malcolm Perry</td>\\n\\t<td class=\\\" \\\">Flexible Heatmaps for Functional Genomics and Sequence Features</td>\\n\\t<td class=\\\"  sorting_1\\\">1820</td>\\n</tr><tr id=\\\"pkg_cummeRbund\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/cummeRbund.html\\\">cummeRbund</a></td>\\n\\t<td class=\\\" \\\">Loyal A. Goff</td>\\n\\t<td class=\\\" \\\">Analysis, exploration, manipulation, and visualization of Cufflinks high-throughput sequencing data.</td>\\n\\t<td class=\\\"  sorting_1\\\">1822</td>\\n</tr><tr id=\\\"pkg_EpiDISH\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/EpiDISH.html\\\">EpiDISH</a></td>\\n\\t<td class=\\\" \\\">Shijie C. Zheng</td>\\n\\t<td class=\\\" \\\">Epigenetic Dissection of Intra-Sample-Heterogeneity</td>\\n\\t<td class=\\\"  sorting_1\\\">1823</td>\\n</tr><tr id=\\\"pkg_soGGi\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/soGGi.html\\\">soGGi</a></td>\\n\\t<td class=\\\" \\\">Tom Carroll</td>\\n\\t<td class=\\\" \\\">Visualise ChIP-seq, MNase-seq and motif occurrence as aggregate plots Summarised Over Grouped Genomic Intervals</td>\\n\\t<td class=\\\"  sorting_1\\\">1824</td>\\n</tr><tr id=\\\"pkg_RankProd\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/RankProd.html\\\">RankProd</a></td>\\n\\t<td class=\\\" \\\">Francesco Del Carratore</td>\\n\\t<td class=\\\" \\\">Rank Product method for identifying differentially expressed genes with application in meta-analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">1825</td>\\n</tr><tr id=\\\"pkg_scRepertoire\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/scRepertoire.html\\\">scRepertoire</a></td>\\n\\t<td class=\\\" \\\">Nick Borcherding</td>\\n\\t<td class=\\\" \\\">A toolkit for single-cell immune receptor profiling</td>\\n\\t<td class=\\\"  sorting_1\\\">1826</td>\\n</tr><tr id=\\\"pkg_maSigPro\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/maSigPro.html\\\">maSigPro</a></td>\\n\\t<td class=\\\" \\\">Maria Jose Nueda</td>\\n\\t<td class=\\\" \\\">Significant Gene Expression Profile Differences in Time Course Gene Expression Data</td>\\n\\t<td class=\\\"  sorting_1\\\">1827</td>\\n</tr><tr id=\\\"pkg_GeneOverlap\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/GeneOverlap.html\\\">GeneOverlap</a></td>\\n\\t<td class=\\\" \\\">Antnio Miguel de Jesus Domingues, Max-Planck Institute for Cell Biology and Genetics</td>\\n\\t<td class=\\\" \\\">Test and visualize gene overlaps</td>\\n\\t<td class=\\\"  sorting_1\\\">1828</td>\\n</tr><tr id=\\\"pkg_recount3\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/recount3.html\\\">recount3</a></td>\\n\\t<td class=\\\" \\\">Leonardo Collado-Torres</td>\\n\\t<td class=\\\" \\\">Explore and download data from the recount3 project</td>\\n\\t<td class=\\\"  sorting_1\\\">1829</td>\\n</tr><tr id=\\\"pkg_SGSeq\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/SGSeq.html\\\">SGSeq</a></td>\\n\\t<td class=\\\" \\\">Leonard Goldstein</td>\\n\\t<td class=\\\" \\\">Splice event prediction and quantification from RNA-seq data</td>\\n\\t<td class=\\\"  sorting_1\\\">1829</td>\\n</tr><tr id=\\\"pkg_cmapR\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/cmapR.html\\\">cmapR</a></td>\\n\\t<td class=\\\" \\\">Ted Natoli</td>\\n\\t<td class=\\\" \\\">CMap Tools in R</td>\\n\\t<td class=\\\"  sorting_1\\\">1831</td>\\n</tr><tr id=\\\"pkg_Icens\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/Icens.html\\\">Icens</a></td>\\n\\t<td class=\\\" \\\">Bioconductor Package Maintainer</td>\\n\\t<td class=\\\" \\\">NPMLE for Censored and Truncated Data</td>\\n\\t<td class=\\\"  sorting_1\\\">1831</td>\\n</tr><tr id=\\\"pkg_iSEE\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/iSEE.html\\\">iSEE</a></td>\\n\\t<td class=\\\" \\\">Kevin Rue-Albrecht</td>\\n\\t<td class=\\\" \\\">Interactive SummarizedExperiment Explorer</td>\\n\\t<td class=\\\"  sorting_1\\\">1833</td>\\n</tr><tr id=\\\"pkg_TCC\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/TCC.html\\\">TCC</a></td>\\n\\t<td class=\\\" \\\">Jianqiang Sun, Tomoaki Nishiyama</td>\\n\\t<td class=\\\" \\\">TCC: Differential expression analysis for tag count data with robust normalization strategies</td>\\n\\t<td class=\\\"  sorting_1\\\">1833</td>\\n</tr><tr id=\\\"pkg_Rbowtie2\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/Rbowtie2.html\\\">Rbowtie2</a></td>\\n\\t<td class=\\\" \\\">Zheng Wei</td>\\n\\t<td class=\\\" \\\">An R Wrapper for Bowtie2 and AdapterRemoval</td>\\n\\t<td class=\\\"  sorting_1\\\">1835</td>\\n</tr><tr id=\\\"pkg_rpx\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/rpx.html\\\">rpx</a></td>\\n\\t<td class=\\\" \\\">Laurent Gatto</td>\\n\\t<td class=\\\" \\\">R Interface to the ProteomeXchange Repository</td>\\n\\t<td class=\\\"  sorting_1\\\">1835</td>\\n</tr><tr id=\\\"pkg_ctc\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ctc.html\\\">ctc</a></td>\\n\\t<td class=\\\" \\\">Antoine Lucas</td>\\n\\t<td class=\\\" \\\">Cluster and Tree Conversion.</td>\\n\\t<td class=\\\"  sorting_1\\\">1837</td>\\n</tr><tr id=\\\"pkg_EnrichmentBrowser\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/EnrichmentBrowser.html\\\">EnrichmentBrowser</a></td>\\n\\t<td class=\\\" \\\">Ludwig Geistlinger</td>\\n\\t<td class=\\\" \\\">Seamless navigation through combined results of set-based and network-based enrichment analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">1838</td>\\n</tr><tr id=\\\"pkg_rrvgo\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/rrvgo.html\\\">rrvgo</a></td>\\n\\t<td class=\\\" \\\">Sergi Sayols</td>\\n\\t<td class=\\\" \\\">Reduce + Visualize GO</td>\\n\\t<td class=\\\"  sorting_1\\\">1838</td>\\n</tr><tr id=\\\"pkg_DEsingle\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/DEsingle.html\\\">DEsingle</a></td>\\n\\t<td class=\\\" \\\">Zhun Miao</td>\\n\\t<td class=\\\" \\\">DEsingle for detecting three types of differential expression in single-cell RNA-seq data</td>\\n\\t<td class=\\\"  sorting_1\\\">1840</td>\\n</tr><tr id=\\\"pkg_LBE\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/LBE.html\\\">LBE</a></td>\\n\\t<td class=\\\" \\\">Cyril Dalmasso</td>\\n\\t<td class=\\\" \\\">Estimation of the false discovery rate.</td>\\n\\t<td class=\\\"  sorting_1\\\">1841</td>\\n</tr><tr id=\\\"pkg_QDNAseq\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/QDNAseq.html\\\">QDNAseq</a></td>\\n\\t<td class=\\\" \\\">Daoud Sie</td>\\n\\t<td class=\\\" \\\">Quantitative DNA Sequencing for Chromosomal Aberrations</td>\\n\\t<td class=\\\"  sorting_1\\\">1842</td>\\n</tr><tr id=\\\"pkg_MLInterfaces\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/MLInterfaces.html\\\">MLInterfaces</a></td>\\n\\t<td class=\\\" \\\">V. Carey</td>\\n\\t<td class=\\\" \\\">Uniform interfaces to R machine learning procedures for data in Bioconductor containers</td>\\n\\t<td class=\\\"  sorting_1\\\">1843</td>\\n</tr><tr id=\\\"pkg_BumpyMatrix\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/BumpyMatrix.html\\\">BumpyMatrix</a></td>\\n\\t<td class=\\\" \\\">Aaron Lun</td>\\n\\t<td class=\\\" \\\">Bumpy Matrix of Non-Scalar Objects</td>\\n\\t<td class=\\\"  sorting_1\\\">1844</td>\\n</tr><tr id=\\\"pkg_rWikiPathways\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/rWikiPathways.html\\\">rWikiPathways</a></td>\\n\\t<td class=\\\" \\\">Egon Willighagen</td>\\n\\t<td class=\\\" \\\">rWikiPathways - R client library for the WikiPathways API</td>\\n\\t<td class=\\\"  sorting_1\\\">1845</td>\\n</tr><tr id=\\\"pkg_trackViewer\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/trackViewer.html\\\">trackViewer</a></td>\\n\\t<td class=\\\" \\\">Jianhong Ou</td>\\n\\t<td class=\\\" \\\">A R/Bioconductor package with web interface for drawing elegant interactive tracks or lollipop plot to facilitate integrated analysis of multi-omics data</td>\\n\\t<td class=\\\"  sorting_1\\\">1846</td>\\n</tr><tr id=\\\"pkg_GDCRNATools\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/GDCRNATools.html\\\">GDCRNATools</a></td>\\n\\t<td class=\\\" \\\">Ruidong Li, Han Qu</td>\\n\\t<td class=\\\" \\\">GDCRNATools: an R/Bioconductor package for integrative analysis of lncRNA, mRNA, and miRNA data in GDC</td>\\n\\t<td class=\\\"  sorting_1\\\">1847</td>\\n</tr><tr id=\\\"pkg_UniProt.ws\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/UniProt.ws.html\\\">UniProt.ws</a></td>\\n\\t<td class=\\\" \\\">Marcel Ramos</td>\\n\\t<td class=\\\" \\\">R Interface to UniProt Web Services</td>\\n\\t<td class=\\\"  sorting_1\\\">1848</td>\\n</tr><tr id=\\\"pkg_rols\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/rols.html\\\">rols</a></td>\\n\\t<td class=\\\" \\\">Laurent Gatto</td>\\n\\t<td class=\\\" \\\">An R interface to the Ontology Lookup Service</td>\\n\\t<td class=\\\"  sorting_1\\\">1849</td>\\n</tr><tr id=\\\"pkg_CGHcall\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/CGHcall.html\\\">CGHcall</a></td>\\n\\t<td class=\\\" \\\">Mark van de Wiel</td>\\n\\t<td class=\\\" \\\">Calling aberrations for array CGH tumor profiles.</td>\\n\\t<td class=\\\"  sorting_1\\\">1850</td>\\n</tr><tr id=\\\"pkg_iClusterPlus\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/iClusterPlus.html\\\">iClusterPlus</a></td>\\n\\t<td class=\\\" \\\">Qianxing Mo, Ronglai Shen</td>\\n\\t<td class=\\\" \\\">Integrative clustering of multi-type genomic data</td>\\n\\t<td class=\\\"  sorting_1\\\">1850</td>\\n</tr><tr id=\\\"pkg_clusterExperiment\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/clusterExperiment.html\\\">clusterExperiment</a></td>\\n\\t<td class=\\\" \\\">Elizabeth Purdom</td>\\n\\t<td class=\\\" \\\">Compare Clusterings for Single-Cell Sequencing</td>\\n\\t<td class=\\\"  sorting_1\\\">1852</td>\\n</tr><tr id=\\\"pkg_GENESIS\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/GENESIS.html\\\">GENESIS</a></td>\\n\\t<td class=\\\" \\\">Stephanie M. Gogarten</td>\\n\\t<td class=\\\" \\\">GENetic EStimation and Inference in Structured samples (GENESIS): Statistical methods for analyzing genetic data from samples with population structure and/or relatedness</td>\\n\\t<td class=\\\"  sorting_1\\\">1852</td>\\n</tr><tr id=\\\"pkg_MAGeCKFlute\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/MAGeCKFlute.html\\\">MAGeCKFlute</a></td>\\n\\t<td class=\\\" \\\">Wubing Zhang</td>\\n\\t<td class=\\\" \\\">Integrative Analysis Pipeline for Pooled CRISPR Functional Genetic Screens</td>\\n\\t<td class=\\\"  sorting_1\\\">1852</td>\\n</tr><tr id=\\\"pkg_powerTCR\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/powerTCR.html\\\">powerTCR</a></td>\\n\\t<td class=\\\" \\\">Hillary Koch</td>\\n\\t<td class=\\\" \\\">Model-Based Comparative Analysis of the TCR Repertoire</td>\\n\\t<td class=\\\"  sorting_1\\\">1852</td>\\n</tr><tr id=\\\"pkg_simplifyEnrichment\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/simplifyEnrichment.html\\\">simplifyEnrichment</a></td>\\n\\t<td class=\\\" \\\">Zuguang Gu</td>\\n\\t<td class=\\\" \\\">Simplify Functional Enrichment Results</td>\\n\\t<td class=\\\"  sorting_1\\\">1856</td>\\n</tr><tr id=\\\"pkg_MOFA2\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/MOFA2.html\\\">MOFA2</a></td>\\n\\t<td class=\\\" \\\">Ricard Argelaguet</td>\\n\\t<td class=\\\" \\\">Multi-Omics Factor Analysis v2</td>\\n\\t<td class=\\\"  sorting_1\\\">1857</td>\\n</tr><tr id=\\\"pkg_scMerge\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/scMerge.html\\\">scMerge</a></td>\\n\\t<td class=\\\" \\\">Yingxin Lin</td>\\n\\t<td class=\\\" \\\">scMerge: Merging multiple batches of scRNA-seq data</td>\\n\\t<td class=\\\"  sorting_1\\\">1858</td>\\n</tr><tr id=\\\"pkg_cBioPortalData\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/cBioPortalData.html\\\">cBioPortalData</a></td>\\n\\t<td class=\\\" \\\">Marcel Ramos</td>\\n\\t<td class=\\\" \\\">Exposes and makes available data from the cBioPortal web resources</td>\\n\\t<td class=\\\"  sorting_1\\\">1859</td>\\n</tr><tr id=\\\"pkg_ggmsa\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ggmsa.html\\\">ggmsa</a></td>\\n\\t<td class=\\\" \\\">Lang Zhou</td>\\n\\t<td class=\\\" \\\">Plot Multiple Sequence Alignment using 'ggplot2'</td>\\n\\t<td class=\\\"  sorting_1\\\">1860</td>\\n</tr><tr id=\\\"pkg_MicrobiotaProcess\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/MicrobiotaProcess.html\\\">MicrobiotaProcess</a></td>\\n\\t<td class=\\\" \\\">Shuangbin Xu</td>\\n\\t<td class=\\\" \\\">A comprehensive R package for managing and analyzing microbiome and other ecological data within the tidy framework</td>\\n\\t<td class=\\\"  sorting_1\\\">1860</td>\\n</tr><tr id=\\\"pkg_decoupleR\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/decoupleR.html\\\">decoupleR</a></td>\\n\\t<td class=\\\" \\\">Pau Badia-i-Mompel</td>\\n\\t<td class=\\\" \\\">decoupleR: Ensemble of computational methods to infer biological activities from omics data</td>\\n\\t<td class=\\\"  sorting_1\\\">1862</td>\\n</tr><tr id=\\\"pkg_scde\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/scde.html\\\">scde</a></td>\\n\\t<td class=\\\" \\\">Jean Fan</td>\\n\\t<td class=\\\" \\\">Single Cell Differential Expression</td>\\n\\t<td class=\\\"  sorting_1\\\">1863</td>\\n</tr><tr id=\\\"pkg_ATACseqQC\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ATACseqQC.html\\\">ATACseqQC</a></td>\\n\\t<td class=\\\" \\\">Jianhong Ou</td>\\n\\t<td class=\\\" \\\">ATAC-seq Quality Control</td>\\n\\t<td class=\\\"  sorting_1\\\">1864</td>\\n</tr><tr id=\\\"pkg_MSstatsConvert\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/MSstatsConvert.html\\\">MSstatsConvert</a></td>\\n\\t<td class=\\\" \\\">Mateusz Staniak</td>\\n\\t<td class=\\\" \\\">Import Data from Various Mass Spectrometry Signal Processing Tools to MSstats Format</td>\\n\\t<td class=\\\"  sorting_1\\\">1865</td>\\n</tr><tr id=\\\"pkg_QuasR\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/QuasR.html\\\">QuasR</a></td>\\n\\t<td class=\\\" \\\">Michael Stadler</td>\\n\\t<td class=\\\" \\\">Quantify and Annotate Short Reads in R</td>\\n\\t<td class=\\\"  sorting_1\\\">1866</td>\\n</tr><tr id=\\\"pkg_LoomExperiment\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/LoomExperiment.html\\\">LoomExperiment</a></td>\\n\\t<td class=\\\" \\\">Bioconductor Package Maintainer</td>\\n\\t<td class=\\\" \\\">LoomExperiment container</td>\\n\\t<td class=\\\"  sorting_1\\\">1867</td>\\n</tr><tr id=\\\"pkg_CGHbase\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/CGHbase.html\\\">CGHbase</a></td>\\n\\t<td class=\\\" \\\">Mark van de Wiel</td>\\n\\t<td class=\\\" \\\">CGHbase: Base functions and classes for arrayCGH data analysis.</td>\\n\\t<td class=\\\"  sorting_1\\\">1868</td>\\n</tr><tr id=\\\"pkg_progeny\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/progeny.html\\\">progeny</a></td>\\n\\t<td class=\\\" \\\">Aurlien Dugourd</td>\\n\\t<td class=\\\" \\\">Pathway RespOnsive GENes for activity inference from gene expression</td>\\n\\t<td class=\\\"  sorting_1\\\">1868</td>\\n</tr><tr id=\\\"pkg_OmnipathR\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/OmnipathR.html\\\">OmnipathR</a></td>\\n\\t<td class=\\\" \\\">Denes Turei</td>\\n\\t<td class=\\\" \\\">OmniPath web service client and more</td>\\n\\t<td class=\\\"  sorting_1\\\">1870</td>\\n</tr><tr id=\\\"pkg_csaw\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/csaw.html\\\">csaw</a></td>\\n\\t<td class=\\\" \\\">Aaron Lun</td>\\n\\t<td class=\\\" \\\">ChIP-Seq Analysis with Windows</td>\\n\\t<td class=\\\"  sorting_1\\\">1871</td>\\n</tr><tr id=\\\"pkg_supraHex\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/supraHex.html\\\">supraHex</a></td>\\n\\t<td class=\\\" \\\">Hai Fang</td>\\n\\t<td class=\\\" \\\">supraHex: a supra-hexagonal map for analysing tabular omics data</td>\\n\\t<td class=\\\"  sorting_1\\\">1872</td>\\n</tr><tr id=\\\"pkg_celda\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/celda.html\\\">celda</a></td>\\n\\t<td class=\\\" \\\">Joshua Campbell</td>\\n\\t<td class=\\\" \\\">CEllular Latent Dirichlet Allocation</td>\\n\\t<td class=\\\"  sorting_1\\\">1873</td>\\n</tr><tr id=\\\"pkg_EBSeq\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/EBSeq.html\\\">EBSeq</a></td>\\n\\t<td class=\\\" \\\">Ning Leng</td>\\n\\t<td class=\\\" \\\">An R package for gene and isoform differential expression analysis of RNA-seq data</td>\\n\\t<td class=\\\"  sorting_1\\\">1874</td>\\n</tr><tr id=\\\"pkg_mygene\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/mygene.html\\\">mygene</a></td>\\n\\t<td class=\\\" \\\">Adam Mark, Cyrus Afrasiabi, Chunlei Wu</td>\\n\\t<td class=\\\" \\\">Access MyGene.Info_ services</td>\\n\\t<td class=\\\"  sorting_1\\\">1875</td>\\n</tr><tr id=\\\"pkg_piano\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/piano.html\\\">piano</a></td>\\n\\t<td class=\\\" \\\">Leif Varemo Wigge</td>\\n\\t<td class=\\\" \\\">Platform for integrative analysis of omics data</td>\\n\\t<td class=\\\"  sorting_1\\\">1876</td>\\n</tr><tr id=\\\"pkg_genefu\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/genefu.html\\\">genefu</a></td>\\n\\t<td class=\\\" \\\">Benjamin Haibe-Kains</td>\\n\\t<td class=\\\" \\\">Computation of Gene Expression-Based Signatures in Breast Cancer</td>\\n\\t<td class=\\\"  sorting_1\\\">1877</td>\\n</tr><tr id=\\\"pkg_ChIPQC\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ChIPQC.html\\\">ChIPQC</a></td>\\n\\t<td class=\\\" \\\">Tom Carroll, Rory Stark</td>\\n\\t<td class=\\\" \\\">Quality metrics for ChIPseq data</td>\\n\\t<td class=\\\"  sorting_1\\\">1878</td>\\n</tr><tr id=\\\"pkg_qusage\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/qusage.html\\\">qusage</a></td>\\n\\t<td class=\\\" \\\">Christopher Bolen</td>\\n\\t<td class=\\\" \\\">qusage: Quantitative Set Analysis for Gene Expression</td>\\n\\t<td class=\\\"  sorting_1\\\">1878</td>\\n</tr><tr id=\\\"pkg_SRAdb\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/SRAdb.html\\\">SRAdb</a></td>\\n\\t<td class=\\\" \\\">Jack Zhu</td>\\n\\t<td class=\\\" \\\">A compilation of metadata from NCBI SRA and tools</td>\\n\\t<td class=\\\"  sorting_1\\\">1880</td>\\n</tr><tr id=\\\"pkg_AIMS\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/AIMS.html\\\">AIMS</a></td>\\n\\t<td class=\\\" \\\">Eric R Paquet</td>\\n\\t<td class=\\\" \\\">AIMS : Absolute Assignment of Breast Cancer Intrinsic Molecular Subtype</td>\\n\\t<td class=\\\"  sorting_1\\\">1881</td>\\n</tr><tr id=\\\"pkg_Heatplus\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/Heatplus.html\\\">Heatplus</a></td>\\n\\t<td class=\\\" \\\">Alexander Ploner</td>\\n\\t<td class=\\\" \\\">Heatmaps with row and/or column covariates and colored clusters</td>\\n\\t<td class=\\\"  sorting_1\\\">1882</td>\\n</tr><tr id=\\\"pkg_lfa\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/lfa.html\\\">lfa</a></td>\\n\\t<td class=\\\" \\\">Wei Hao, John D. Storey</td>\\n\\t<td class=\\\" \\\">Logistic Factor Analysis for Categorical Data</td>\\n\\t<td class=\\\"  sorting_1\\\">1882</td>\\n</tr><tr id=\\\"pkg_tradeSeq\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/tradeSeq.html\\\">tradeSeq</a></td>\\n\\t<td class=\\\" \\\">Hector Roux de Bezieux</td>\\n\\t<td class=\\\" \\\">trajectory-based differential expression analysis for sequencing data</td>\\n\\t<td class=\\\"  sorting_1\\\">1882</td>\\n</tr><tr id=\\\"pkg_sangerseqR\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/sangerseqR.html\\\">sangerseqR</a></td>\\n\\t<td class=\\\" \\\">Jonathon Hill</td>\\n\\t<td class=\\\" \\\">Tools for Sanger Sequencing Data in R</td>\\n\\t<td class=\\\"  sorting_1\\\">1885</td>\\n</tr><tr id=\\\"pkg_DRIMSeq\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/DRIMSeq.html\\\">DRIMSeq</a></td>\\n\\t<td class=\\\" \\\">Malgorzata Nowicka</td>\\n\\t<td class=\\\" \\\">Differential transcript usage and tuQTL analyses with Dirichlet-multinomial model in RNA-seq</td>\\n\\t<td class=\\\"  sorting_1\\\">1886</td>\\n</tr><tr id=\\\"pkg_QFeatures\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/QFeatures.html\\\">QFeatures</a></td>\\n\\t<td class=\\\" \\\">Laurent Gatto</td>\\n\\t<td class=\\\" \\\">Quantitative features for mass spectrometry data</td>\\n\\t<td class=\\\"  sorting_1\\\">1886</td>\\n</tr><tr id=\\\"pkg_TSCAN\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/TSCAN.html\\\">TSCAN</a></td>\\n\\t<td class=\\\" \\\">Zhicheng Ji</td>\\n\\t<td class=\\\" \\\">Tools for Single-Cell Analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">1886</td>\\n</tr><tr id=\\\"pkg_fishpond\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/fishpond.html\\\">fishpond</a></td>\\n\\t<td class=\\\" \\\">Michael Love</td>\\n\\t<td class=\\\" \\\">Fishpond: downstream methods and tools for expression data</td>\\n\\t<td class=\\\"  sorting_1\\\">1889</td>\\n</tr><tr id=\\\"pkg_Organism.dplyr\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/Organism.dplyr.html\\\">Organism.dplyr</a></td>\\n\\t<td class=\\\" \\\">Martin Morgan</td>\\n\\t<td class=\\\" \\\">dplyr-based Access to Bioconductor Annotation Resources</td>\\n\\t<td class=\\\"  sorting_1\\\">1890</td>\\n</tr><tr id=\\\"pkg_affycoretools\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/affycoretools.html\\\">affycoretools</a></td>\\n\\t<td class=\\\" \\\">James W. MacDonald</td>\\n\\t<td class=\\\" \\\">Functions useful for those doing repetitive analyses with Affymetrix GeneChips</td>\\n\\t<td class=\\\"  sorting_1\\\">1891</td>\\n</tr><tr id=\\\"pkg_splatter\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/splatter.html\\\">splatter</a></td>\\n\\t<td class=\\\" \\\">Luke Zappia</td>\\n\\t<td class=\\\" \\\">Simple Simulation of Single-cell RNA Sequencing Data</td>\\n\\t<td class=\\\"  sorting_1\\\">1892</td>\\n</tr><tr id=\\\"pkg_recount\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/recount.html\\\">recount</a></td>\\n\\t<td class=\\\" \\\">Leonardo Collado-Torres</td>\\n\\t<td class=\\\" \\\">Explore and download data from the recount project</td>\\n\\t<td class=\\\"  sorting_1\\\">1893</td>\\n</tr><tr id=\\\"pkg_mbkmeans\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/mbkmeans.html\\\">mbkmeans</a></td>\\n\\t<td class=\\\" \\\">Davide Risso</td>\\n\\t<td class=\\\" \\\">Mini-batch K-means Clustering for Single-Cell RNA-seq</td>\\n\\t<td class=\\\"  sorting_1\\\">1894</td>\\n</tr><tr id=\\\"pkg_rGADEM\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/rGADEM.html\\\">rGADEM</a></td>\\n\\t<td class=\\\" \\\">Arnaud Droit</td>\\n\\t<td class=\\\" \\\">de novo motif discovery</td>\\n\\t<td class=\\\"  sorting_1\\\">1895</td>\\n</tr><tr id=\\\"pkg_MSstats\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/MSstats.html\\\">MSstats</a></td>\\n\\t<td class=\\\" \\\">Meena Choi</td>\\n\\t<td class=\\\" \\\">Protein Significance Analysis in DDA, SRM and DIA for Label-free or Label-based Proteomics Experiments</td>\\n\\t<td class=\\\"  sorting_1\\\">1896</td>\\n</tr><tr id=\\\"pkg_MutationalPatterns\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/MutationalPatterns.html\\\">MutationalPatterns</a></td>\\n\\t<td class=\\\" \\\">Mark van Roosmalen</td>\\n\\t<td class=\\\" \\\">Comprehensive genome-wide analysis of mutational processes</td>\\n\\t<td class=\\\"  sorting_1\\\">1897</td>\\n</tr><tr id=\\\"pkg_Spectra\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/Spectra.html\\\">Spectra</a></td>\\n\\t<td class=\\\" \\\">RforMassSpectrometry Package Maintainer</td>\\n\\t<td class=\\\" \\\">Spectra Infrastructure for Mass Spectrometry Data</td>\\n\\t<td class=\\\"  sorting_1\\\">1898</td>\\n</tr><tr id=\\\"pkg_copynumber\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/copynumber.html\\\">copynumber</a></td>\\n\\t<td class=\\\" \\\">Gro Nilsen</td>\\n\\t<td class=\\\" \\\">Segmentation of single- and multi-track copy number data by penalized least squares regression.</td>\\n\\t<td class=\\\"  sorting_1\\\">1899</td>\\n</tr><tr id=\\\"pkg_pcaExplorer\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/pcaExplorer.html\\\">pcaExplorer</a></td>\\n\\t<td class=\\\" \\\">Federico Marini</td>\\n\\t<td class=\\\" \\\">Interactive Visualization of RNA-seq Data Using a Principal Components Approach</td>\\n\\t<td class=\\\"  sorting_1\\\">1900</td>\\n</tr><tr id=\\\"pkg_CATALYST\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/CATALYST.html\\\">CATALYST</a></td>\\n\\t<td class=\\\" \\\">Helena L. Crowell</td>\\n\\t<td class=\\\" \\\">Cytometry dATa anALYSis Tools</td>\\n\\t<td class=\\\"  sorting_1\\\">1901</td>\\n</tr><tr id=\\\"pkg_universalmotif\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/universalmotif.html\\\">universalmotif</a></td>\\n\\t<td class=\\\" \\\">Benjamin Jean-Marie Tremblay</td>\\n\\t<td class=\\\" \\\">Import, Modify, and Export Motifs with R</td>\\n\\t<td class=\\\"  sorting_1\\\">1902</td>\\n</tr><tr id=\\\"pkg_SC3\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/SC3.html\\\">SC3</a></td>\\n\\t<td class=\\\" \\\">Vladimir Kiselev</td>\\n\\t<td class=\\\" \\\">Single-Cell Consensus Clustering</td>\\n\\t<td class=\\\"  sorting_1\\\">1903</td>\\n</tr><tr id=\\\"pkg_gwascat\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/gwascat.html\\\">gwascat</a></td>\\n\\t<td class=\\\" \\\">VJ Carey</td>\\n\\t<td class=\\\" \\\">representing and modeling data in the EMBL-EBI GWAS catalog</td>\\n\\t<td class=\\\"  sorting_1\\\">1904</td>\\n</tr><tr id=\\\"pkg_EnrichedHeatmap\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/EnrichedHeatmap.html\\\">EnrichedHeatmap</a></td>\\n\\t<td class=\\\" \\\">Zuguang Gu</td>\\n\\t<td class=\\\" \\\">Making Enriched Heatmaps</td>\\n\\t<td class=\\\"  sorting_1\\\">1905</td>\\n</tr><tr id=\\\"pkg_M3Drop\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/M3Drop.html\\\">M3Drop</a></td>\\n\\t<td class=\\\" \\\">Tallulah Andrews</td>\\n\\t<td class=\\\" \\\">Michaelis-Menten Modelling of Dropouts in single-cell RNASeq</td>\\n\\t<td class=\\\"  sorting_1\\\">1905</td>\\n</tr><tr id=\\\"pkg_safe\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/safe.html\\\">safe</a></td>\\n\\t<td class=\\\" \\\">Ludwig Geistlinger</td>\\n\\t<td class=\\\" \\\">Significance Analysis of Function and Expression</td>\\n\\t<td class=\\\"  sorting_1\\\">1905</td>\\n</tr><tr id=\\\"pkg_scmap\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/scmap.html\\\">scmap</a></td>\\n\\t<td class=\\\" \\\">Vladimir Kiselev</td>\\n\\t<td class=\\\" \\\">A tool for unsupervised projection of single cell RNA-seq data</td>\\n\\t<td class=\\\"  sorting_1\\\">1905</td>\\n</tr><tr id=\\\"pkg_GenVisR\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/GenVisR.html\\\">GenVisR</a></td>\\n\\t<td class=\\\" \\\">Zachary Skidmore</td>\\n\\t<td class=\\\" \\\">Genomic Visualizations in R</td>\\n\\t<td class=\\\"  sorting_1\\\">1909</td>\\n</tr><tr id=\\\"pkg_Rbowtie\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/Rbowtie.html\\\">Rbowtie</a></td>\\n\\t<td class=\\\" \\\">Michael Stadler</td>\\n\\t<td class=\\\" \\\">R bowtie wrapper</td>\\n\\t<td class=\\\"  sorting_1\\\">1910</td>\\n</tr><tr id=\\\"pkg_annotatr\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/annotatr.html\\\">annotatr</a></td>\\n\\t<td class=\\\" \\\">Raymond G. Cavalcante</td>\\n\\t<td class=\\\" \\\">Annotation of Genomic Regions to Genomic Annotations</td>\\n\\t<td class=\\\"  sorting_1\\\">1911</td>\\n</tr><tr id=\\\"pkg_LEA\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/LEA.html\\\">LEA</a></td>\\n\\t<td class=\\\" \\\">Olivier Francois, Eric Frichot</td>\\n\\t<td class=\\\" \\\">LEA: an R package for Landscape and Ecological Association Studies</td>\\n\\t<td class=\\\"  sorting_1\\\">1912</td>\\n</tr><tr id=\\\"pkg_Rdisop\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/Rdisop.html\\\">Rdisop</a></td>\\n\\t<td class=\\\" \\\">Steffen Neumann</td>\\n\\t<td class=\\\" \\\">Decomposition of Isotopic Patterns</td>\\n\\t<td class=\\\"  sorting_1\\\">1913</td>\\n</tr><tr id=\\\"pkg_sesame\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/sesame.html\\\">sesame</a></td>\\n\\t<td class=\\\" \\\">Wanding Zhou</td>\\n\\t<td class=\\\" \\\">SEnsible Step-wise Analysis of DNA MEthylation BeadChips</td>\\n\\t<td class=\\\"  sorting_1\\\">1913</td>\\n</tr><tr id=\\\"pkg_UCell\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/UCell.html\\\">UCell</a></td>\\n\\t<td class=\\\" \\\">Massimo Andreatta</td>\\n\\t<td class=\\\" \\\">Rank-based signature enrichment analysis for single-cell data</td>\\n\\t<td class=\\\"  sorting_1\\\">1913</td>\\n</tr><tr id=\\\"pkg_DEP\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/DEP.html\\\">DEP</a></td>\\n\\t<td class=\\\" \\\">Arne Smits</td>\\n\\t<td class=\\\" \\\">Differential Enrichment analysis of Proteomics data</td>\\n\\t<td class=\\\"  sorting_1\\\">1916</td>\\n</tr><tr id=\\\"pkg_DepecheR\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/DepecheR.html\\\">DepecheR</a></td>\\n\\t<td class=\\\" \\\">Jakob Theorell</td>\\n\\t<td class=\\\" \\\">Determination of essential phenotypic elements of clusters in high-dimensional entities</td>\\n\\t<td class=\\\"  sorting_1\\\">1916</td>\\n</tr><tr id=\\\"pkg_GenomicScores\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/GenomicScores.html\\\">GenomicScores</a></td>\\n\\t<td class=\\\" \\\">Robert Castelo</td>\\n\\t<td class=\\\" \\\">Infrastructure to work with genomewide position-specific scores</td>\\n\\t<td class=\\\"  sorting_1\\\">1918</td>\\n</tr><tr id=\\\"pkg_SeqVarTools\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/SeqVarTools.html\\\">SeqVarTools</a></td>\\n\\t<td class=\\\" \\\">Stephanie M. Gogarten</td>\\n\\t<td class=\\\" \\\">Tools for variant data</td>\\n\\t<td class=\\\"  sorting_1\\\">1918</td>\\n</tr><tr id=\\\"pkg_Maaslin2\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/Maaslin2.html\\\">Maaslin2</a></td>\\n\\t<td class=\\\" \\\">Lauren McIver</td>\\n\\t<td class=\\\" \\\">\\\"Multivariable Association Discovery in Population-scale Meta-omics Studies\\\"</td>\\n\\t<td class=\\\"  sorting_1\\\">1920</td>\\n</tr><tr id=\\\"pkg_SPIA\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/SPIA.html\\\">SPIA</a></td>\\n\\t<td class=\\\" \\\">Adi Laurentiu Tarca</td>\\n\\t<td class=\\\" \\\">Signaling Pathway Impact Analysis (SPIA) using combined evidence of pathway over-representation and unusual signaling perturbations</td>\\n\\t<td class=\\\"  sorting_1\\\">1921</td>\\n</tr><tr id=\\\"pkg_CAMERA\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/CAMERA.html\\\">CAMERA</a></td>\\n\\t<td class=\\\" \\\">Steffen Neumann</td>\\n\\t<td class=\\\" \\\">Collection of annotation related methods for mass spectrometry data</td>\\n\\t<td class=\\\"  sorting_1\\\">1922</td>\\n</tr><tr id=\\\"pkg_derfinderHelper\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/derfinderHelper.html\\\">derfinderHelper</a></td>\\n\\t<td class=\\\" \\\">Leonardo Collado-Torres</td>\\n\\t<td class=\\\" \\\">derfinder helper package</td>\\n\\t<td class=\\\"  sorting_1\\\">1923</td>\\n</tr><tr id=\\\"pkg_rGREAT\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/rGREAT.html\\\">rGREAT</a></td>\\n\\t<td class=\\\" \\\">Zuguang Gu</td>\\n\\t<td class=\\\" \\\">GREAT Analysis - Functional Enrichment on Genomic Regions</td>\\n\\t<td class=\\\"  sorting_1\\\">1923</td>\\n</tr><tr id=\\\"pkg_derfinder\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/derfinder.html\\\">derfinder</a></td>\\n\\t<td class=\\\" \\\">Leonardo Collado-Torres</td>\\n\\t<td class=\\\" \\\">Annotation-agnostic differential expression analysis of RNA-seq data at base-pair resolution via the DER Finder approach</td>\\n\\t<td class=\\\"  sorting_1\\\">1925</td>\\n</tr><tr id=\\\"pkg_flowAI\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/flowAI.html\\\">flowAI</a></td>\\n\\t<td class=\\\" \\\">Gianni Monaco</td>\\n\\t<td class=\\\" \\\">Automatic and interactive quality control for flow cytometry data</td>\\n\\t<td class=\\\"  sorting_1\\\">1926</td>\\n</tr><tr id=\\\"pkg_DEGreport\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/DEGreport.html\\\">DEGreport</a></td>\\n\\t<td class=\\\" \\\">Lorena Pantano</td>\\n\\t<td class=\\\" \\\">Report of DEG analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">1927</td>\\n</tr><tr id=\\\"pkg_MotifDb\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/MotifDb.html\\\">MotifDb</a></td>\\n\\t<td class=\\\" \\\">Paul Shannon</td>\\n\\t<td class=\\\" \\\">An Annotated Collection of Protein-DNA Binding Sequence Motifs</td>\\n\\t<td class=\\\"  sorting_1\\\">1928</td>\\n</tr><tr id=\\\"pkg_R4RNA\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/R4RNA.html\\\">R4RNA</a></td>\\n\\t<td class=\\\" \\\">Daniel Lai</td>\\n\\t<td class=\\\" \\\">An R package for RNA visualization and analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">1929</td>\\n</tr><tr id=\\\"pkg_variancePartition\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/variancePartition.html\\\">variancePartition</a></td>\\n\\t<td class=\\\" \\\">Gabriel E. Hoffman</td>\\n\\t<td class=\\\" \\\">Quantify and interpret drivers of variation in multilevel gene expression experiments</td>\\n\\t<td class=\\\"  sorting_1\\\">1930</td>\\n</tr><tr id=\\\"pkg_scds\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/scds.html\\\">scds</a></td>\\n\\t<td class=\\\" \\\">Dennis Kostka</td>\\n\\t<td class=\\\" \\\">In-Silico Annotation of Doublets for Single Cell RNA Sequencing Data</td>\\n\\t<td class=\\\"  sorting_1\\\">1931</td>\\n</tr><tr id=\\\"pkg_chipseq\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/chipseq.html\\\">chipseq</a></td>\\n\\t<td class=\\\" \\\">Bioconductor Package Maintainer</td>\\n\\t<td class=\\\" \\\">chipseq: A package for analyzing chipseq data</td>\\n\\t<td class=\\\"  sorting_1\\\">1932</td>\\n</tr><tr id=\\\"pkg_arrayQualityMetrics\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/arrayQualityMetrics.html\\\">arrayQualityMetrics</a></td>\\n\\t<td class=\\\" \\\">Mike Smith</td>\\n\\t<td class=\\\" \\\">Quality metrics report for microarray data sets</td>\\n\\t<td class=\\\"  sorting_1\\\">1933</td>\\n</tr><tr id=\\\"pkg_methylKit\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/methylKit.html\\\">methylKit</a></td>\\n\\t<td class=\\\" \\\">Altuna Akalin, Alexander Gosdschan</td>\\n\\t<td class=\\\" \\\">DNA methylation analysis from high-throughput bisulfite sequencing results</td>\\n\\t<td class=\\\"  sorting_1\\\">1934</td>\\n</tr><tr id=\\\"pkg_RTCGA\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/RTCGA.html\\\">RTCGA</a></td>\\n\\t<td class=\\\" \\\">Marcin Kosinski</td>\\n\\t<td class=\\\" \\\">The Cancer Genome Atlas Data Integration</td>\\n\\t<td class=\\\"  sorting_1\\\">1934</td>\\n</tr><tr id=\\\"pkg_ArrayExpress\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ArrayExpress.html\\\">ArrayExpress</a></td>\\n\\t<td class=\\\" \\\">Suhaib Mohammed</td>\\n\\t<td class=\\\" \\\">Access the ArrayExpress Microarray Database at EBI and build Bioconductor data structures: ExpressionSet, AffyBatch, NChannelSet</td>\\n\\t<td class=\\\"  sorting_1\\\">1936</td>\\n</tr><tr id=\\\"pkg_NOISeq\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/NOISeq.html\\\">NOISeq</a></td>\\n\\t<td class=\\\" \\\">Sonia Tarazona</td>\\n\\t<td class=\\\" \\\">Exploratory analysis and differential expression for RNA-seq data</td>\\n\\t<td class=\\\"  sorting_1\\\">1937</td>\\n</tr><tr id=\\\"pkg_RTCGAToolbox\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/RTCGAToolbox.html\\\">RTCGAToolbox</a></td>\\n\\t<td class=\\\" \\\">Marcel Ramos</td>\\n\\t<td class=\\\" \\\">A new tool for exporting TCGA Firehose data</td>\\n\\t<td class=\\\"  sorting_1\\\">1937</td>\\n</tr><tr id=\\\"pkg_ballgown\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ballgown.html\\\">ballgown</a></td>\\n\\t<td class=\\\" \\\">Jack Fu</td>\\n\\t<td class=\\\" \\\">Flexible, isoform-level differential expression analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">1939</td>\\n</tr><tr id=\\\"pkg_baySeq\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/baySeq.html\\\">baySeq</a></td>\\n\\t<td class=\\\" \\\">Thomas J. Hardcastle</td>\\n\\t<td class=\\\" \\\">Empirical Bayesian analysis of patterns of differential expression in count data</td>\\n\\t<td class=\\\"  sorting_1\\\">1940</td>\\n</tr><tr id=\\\"pkg_genomation\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/genomation.html\\\">genomation</a></td>\\n\\t<td class=\\\" \\\">Altuna Akalin, Vedran Franke, Katarzyna Wreczycka</td>\\n\\t<td class=\\\" \\\">Summary, annotation and visualization of genomic data</td>\\n\\t<td class=\\\"  sorting_1\\\">1941</td>\\n</tr><tr id=\\\"pkg_seqPattern\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/seqPattern.html\\\">seqPattern</a></td>\\n\\t<td class=\\\" \\\">Vanja Haberle</td>\\n\\t<td class=\\\" \\\">Visualising oligonucleotide patterns and motif occurrences across a set of sorted sequences</td>\\n\\t<td class=\\\"  sorting_1\\\">1942</td>\\n</tr><tr id=\\\"pkg_zellkonverter\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/zellkonverter.html\\\">zellkonverter</a></td>\\n\\t<td class=\\\" \\\">Luke Zappia</td>\\n\\t<td class=\\\" \\\">Conversion Between scRNA-seq Objects</td>\\n\\t<td class=\\\"  sorting_1\\\">1943</td>\\n</tr><tr id=\\\"pkg_minet\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/minet.html\\\">minet</a></td>\\n\\t<td class=\\\" \\\">Patrick E. Meyer</td>\\n\\t<td class=\\\" \\\">Mutual Information NETworks</td>\\n\\t<td class=\\\"  sorting_1\\\">1944</td>\\n</tr><tr id=\\\"pkg_GWASTools\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/GWASTools.html\\\">GWASTools</a></td>\\n\\t<td class=\\\" \\\">Stephanie M. Gogarten</td>\\n\\t<td class=\\\" \\\">Tools for Genome Wide Association Studies</td>\\n\\t<td class=\\\"  sorting_1\\\">1945</td>\\n</tr><tr id=\\\"pkg_motifStack\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/motifStack.html\\\">motifStack</a></td>\\n\\t<td class=\\\" \\\">Jianhong Ou</td>\\n\\t<td class=\\\" \\\">Plot stacked logos for single or multiple DNA, RNA and amino acid sequence</td>\\n\\t<td class=\\\"  sorting_1\\\">1945</td>\\n</tr><tr id=\\\"pkg_ChemmineR\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ChemmineR.html\\\">ChemmineR</a></td>\\n\\t<td class=\\\" \\\">Thomas Girke</td>\\n\\t<td class=\\\" \\\">Cheminformatics Toolkit for R</td>\\n\\t<td class=\\\"  sorting_1\\\">1947</td>\\n</tr><tr id=\\\"pkg_IHW\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/IHW.html\\\">IHW</a></td>\\n\\t<td class=\\\" \\\">Nikos Ignatiadis</td>\\n\\t<td class=\\\" \\\">Independent Hypothesis Weighting</td>\\n\\t<td class=\\\"  sorting_1\\\">1948</td>\\n</tr><tr id=\\\"pkg_quantsmooth\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/quantsmooth.html\\\">quantsmooth</a></td>\\n\\t<td class=\\\" \\\">Jan Oosting</td>\\n\\t<td class=\\\" \\\">Quantile smoothing and genomic visualization of array data</td>\\n\\t<td class=\\\"  sorting_1\\\">1949</td>\\n</tr><tr id=\\\"pkg_ChAMP\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ChAMP.html\\\">ChAMP</a></td>\\n\\t<td class=\\\" \\\">Yuan Tian</td>\\n\\t<td class=\\\" \\\">Chip Analysis Methylation Pipeline for Illumina HumanMethylation450 and EPIC</td>\\n\\t<td class=\\\"  sorting_1\\\">1950</td>\\n</tr><tr id=\\\"pkg_viper\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/viper.html\\\">viper</a></td>\\n\\t<td class=\\\" \\\">Mariano J Alvarez</td>\\n\\t<td class=\\\" \\\">Virtual Inference of Protein-activity by Enriched Regulon analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">1951</td>\\n</tr><tr id=\\\"pkg_GlobalAncova\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/GlobalAncova.html\\\">GlobalAncova</a></td>\\n\\t<td class=\\\" \\\">Manuela Hummel</td>\\n\\t<td class=\\\" \\\">Global test for groups of variables via model comparisons</td>\\n\\t<td class=\\\"  sorting_1\\\">1952</td>\\n</tr><tr id=\\\"pkg_AnVIL\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/AnVIL.html\\\">AnVIL</a></td>\\n\\t<td class=\\\" \\\">Martin Morgan</td>\\n\\t<td class=\\\" \\\">Bioconductor on the AnVIL compute environment</td>\\n\\t<td class=\\\"  sorting_1\\\">1953</td>\\n</tr><tr id=\\\"pkg_zinbwave\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/zinbwave.html\\\">zinbwave</a></td>\\n\\t<td class=\\\" \\\">Davide Risso</td>\\n\\t<td class=\\\" \\\">Zero-Inflated Negative Binomial Model for RNA-Seq Data</td>\\n\\t<td class=\\\"  sorting_1\\\">1953</td>\\n</tr><tr id=\\\"pkg_Nebulosa\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/Nebulosa.html\\\">Nebulosa</a></td>\\n\\t<td class=\\\" \\\">Jose Alquicira-Hernandez</td>\\n\\t<td class=\\\" \\\">Single-Cell Data Visualisation Using Kernel Gene-Weighted Density Estimation</td>\\n\\t<td class=\\\"  sorting_1\\\">1955</td>\\n</tr><tr id=\\\"pkg_Repitools\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/Repitools.html\\\">Repitools</a></td>\\n\\t<td class=\\\" \\\">Mark Robinson</td>\\n\\t<td class=\\\" \\\">Epigenomic tools</td>\\n\\t<td class=\\\"  sorting_1\\\">1956</td>\\n</tr><tr id=\\\"pkg_beadarray\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/beadarray.html\\\">beadarray</a></td>\\n\\t<td class=\\\" \\\">Mark Dunning</td>\\n\\t<td class=\\\" \\\">Quality assessment and low-level analysis for Illumina BeadArray data</td>\\n\\t<td class=\\\"  sorting_1\\\">1957</td>\\n</tr><tr id=\\\"pkg_mia\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/mia.html\\\">mia</a></td>\\n\\t<td class=\\\" \\\">Tuomas Borman</td>\\n\\t<td class=\\\" \\\">Microbiome analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">1957</td>\\n</tr><tr id=\\\"pkg_fastseg\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/fastseg.html\\\">fastseg</a></td>\\n\\t<td class=\\\" \\\">Alexander Blume</td>\\n\\t<td class=\\\" \\\">fastseg - a fast segmentation algorithm</td>\\n\\t<td class=\\\"  sorting_1\\\">1959</td>\\n</tr><tr id=\\\"pkg_BeadDataPackR\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/BeadDataPackR.html\\\">BeadDataPackR</a></td>\\n\\t<td class=\\\" \\\">Mike Smith</td>\\n\\t<td class=\\\" \\\">Compression of Illumina BeadArray data</td>\\n\\t<td class=\\\"  sorting_1\\\">1960</td>\\n</tr><tr id=\\\"pkg_destiny\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/destiny.html\\\">destiny</a></td>\\n\\t<td class=\\\" \\\">Philipp Angerer</td>\\n\\t<td class=\\\" \\\">Creates diffusion maps</td>\\n\\t<td class=\\\"  sorting_1\\\">1961</td>\\n</tr><tr id=\\\"pkg_Ringo\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/Ringo.html\\\">Ringo</a></td>\\n\\t<td class=\\\" \\\">J. Toedling</td>\\n\\t<td class=\\\" \\\">R Investigation of ChIP-chip Oligoarrays</td>\\n\\t<td class=\\\"  sorting_1\\\">1961</td>\\n</tr><tr id=\\\"pkg_dittoSeq\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/dittoSeq.html\\\">dittoSeq</a></td>\\n\\t<td class=\\\" \\\">Daniel Bunis</td>\\n\\t<td class=\\\" \\\">User Friendly Single-Cell and Bulk RNA Sequencing Visualization</td>\\n\\t<td class=\\\"  sorting_1\\\">1963</td>\\n</tr><tr id=\\\"pkg_TreeSummarizedExperiment\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/TreeSummarizedExperiment.html\\\">TreeSummarizedExperiment</a></td>\\n\\t<td class=\\\" \\\">Ruizhu Huang</td>\\n\\t<td class=\\\" \\\">TreeSummarizedExperiment: a S4 Class for Data with Tree Structures</td>\\n\\t<td class=\\\"  sorting_1\\\">1964</td>\\n</tr><tr id=\\\"pkg_RCy3\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/RCy3.html\\\">RCy3</a></td>\\n\\t<td class=\\\" \\\">Alex Pico</td>\\n\\t<td class=\\\" \\\">Functions to Access and Control Cytoscape</td>\\n\\t<td class=\\\"  sorting_1\\\">1965</td>\\n</tr><tr id=\\\"pkg_SpatialExperiment\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/SpatialExperiment.html\\\">SpatialExperiment</a></td>\\n\\t<td class=\\\" \\\">Dario Righelli</td>\\n\\t<td class=\\\" \\\">S4 Class for Spatially Resolved Transcriptomics Data</td>\\n\\t<td class=\\\"  sorting_1\\\">1966</td>\\n</tr><tr id=\\\"pkg_TCGAutils\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/TCGAutils.html\\\">TCGAutils</a></td>\\n\\t<td class=\\\" \\\">Marcel Ramos</td>\\n\\t<td class=\\\" \\\">TCGA utility functions for data management</td>\\n\\t<td class=\\\"  sorting_1\\\">1967</td>\\n</tr><tr id=\\\"pkg_MultiDataSet\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/MultiDataSet.html\\\">MultiDataSet</a></td>\\n\\t<td class=\\\" \\\">Xavier Escrib Montagut</td>\\n\\t<td class=\\\" \\\">Implementation of MultiDataSet and ResultSet</td>\\n\\t<td class=\\\"  sorting_1\\\">1968</td>\\n</tr><tr id=\\\"pkg_ALDEx2\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ALDEx2.html\\\">ALDEx2</a></td>\\n\\t<td class=\\\" \\\">Greg Gloor</td>\\n\\t<td class=\\\" \\\">Analysis Of Differential Abundance Taking Sample Variation Into Account</td>\\n\\t<td class=\\\"  sorting_1\\\">1969</td>\\n</tr><tr id=\\\"pkg_ggtreeExtra\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ggtreeExtra.html\\\">ggtreeExtra</a></td>\\n\\t<td class=\\\" \\\">Shuangbin Xu</td>\\n\\t<td class=\\\" \\\">An R Package To Add Geometric Layers On Circular Or Other Layout Tree Of \\\"ggtree\\\"</td>\\n\\t<td class=\\\"  sorting_1\\\">1970</td>\\n</tr><tr id=\\\"pkg_ropls\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ropls.html\\\">ropls</a></td>\\n\\t<td class=\\\" \\\">Etienne A. Thevenot</td>\\n\\t<td class=\\\" \\\">PCA, PLS(-DA) and OPLS(-DA) for multivariate analysis and feature selection of omics data</td>\\n\\t<td class=\\\"  sorting_1\\\">1971</td>\\n</tr><tr id=\\\"pkg_wateRmelon\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/wateRmelon.html\\\">wateRmelon</a></td>\\n\\t<td class=\\\" \\\">Leo C Schalkwyk</td>\\n\\t<td class=\\\" \\\">Illumina 450 and EPIC methylation array normalization and metrics</td>\\n\\t<td class=\\\"  sorting_1\\\">1972</td>\\n</tr><tr id=\\\"pkg_ANCOMBC\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ANCOMBC.html\\\">ANCOMBC</a></td>\\n\\t<td class=\\\" \\\">Huang Lin</td>\\n\\t<td class=\\\" \\\">Microbiome differential abudance and correlation analyses with bias correction</td>\\n\\t<td class=\\\"  sorting_1\\\">1973</td>\\n</tr><tr id=\\\"pkg_ReportingTools\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ReportingTools.html\\\">ReportingTools</a></td>\\n\\t<td class=\\\" \\\">Jason A. Hackney, Gabriel Becker, Jessica L. Larson</td>\\n\\t<td class=\\\" \\\">Tools for making reports in various formats</td>\\n\\t<td class=\\\"  sorting_1\\\">1974</td>\\n</tr><tr id=\\\"pkg_karyoploteR\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/karyoploteR.html\\\">karyoploteR</a></td>\\n\\t<td class=\\\" \\\">Bernat Gel</td>\\n\\t<td class=\\\" \\\">Plot customizable linear genomes displaying arbitrary data</td>\\n\\t<td class=\\\"  sorting_1\\\">1975</td>\\n</tr><tr id=\\\"pkg_survcomp\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/survcomp.html\\\">survcomp</a></td>\\n\\t<td class=\\\" \\\">Benjamin Haibe-Kains</td>\\n\\t<td class=\\\" \\\">Performance Assessment and Comparison for Survival Analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">1976</td>\\n</tr><tr id=\\\"pkg_GreyListChIP\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/GreyListChIP.html\\\">GreyListChIP</a></td>\\n\\t<td class=\\\" \\\">Rory Stark</td>\\n\\t<td class=\\\" \\\">Grey Lists -- Mask Artefact Regions Based on ChIP Inputs</td>\\n\\t<td class=\\\"  sorting_1\\\">1977</td>\\n</tr><tr id=\\\"pkg_SeqArray\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/SeqArray.html\\\">SeqArray</a></td>\\n\\t<td class=\\\" \\\">Xiuwen Zheng</td>\\n\\t<td class=\\\" \\\">Data Management of Large-Scale Whole-Genome Sequence Variant Calls</td>\\n\\t<td class=\\\"  sorting_1\\\">1978</td>\\n</tr><tr id=\\\"pkg_scDblFinder\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/scDblFinder.html\\\">scDblFinder</a></td>\\n\\t<td class=\\\" \\\">Pierre-Luc Germain</td>\\n\\t<td class=\\\" \\\">scDblFinder</td>\\n\\t<td class=\\\"  sorting_1\\\">1979</td>\\n</tr><tr id=\\\"pkg_Mfuzz\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/Mfuzz.html\\\">Mfuzz</a></td>\\n\\t<td class=\\\" \\\">Matthias Futschik</td>\\n\\t<td class=\\\" \\\">Soft clustering of time series gene expression data</td>\\n\\t<td class=\\\"  sorting_1\\\">1980</td>\\n</tr><tr id=\\\"pkg_DMRcate\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/DMRcate.html\\\">DMRcate</a></td>\\n\\t<td class=\\\" \\\">Tim Peters</td>\\n\\t<td class=\\\" \\\">Methylation array and sequencing spatial analysis methods</td>\\n\\t<td class=\\\"  sorting_1\\\">1981</td>\\n</tr><tr id=\\\"pkg_STRINGdb\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/STRINGdb.html\\\">STRINGdb</a></td>\\n\\t<td class=\\\" \\\">Damian Szklarczyk</td>\\n\\t<td class=\\\" \\\">STRINGdb - Protein-Protein Interaction Networks and Functional Enrichment Analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">1982</td>\\n</tr><tr id=\\\"pkg_infercnv\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/infercnv.html\\\">infercnv</a></td>\\n\\t<td class=\\\" \\\">Christophe Georgescu</td>\\n\\t<td class=\\\" \\\">Infer Copy Number Variation from Single-Cell RNA-Seq Data</td>\\n\\t<td class=\\\"  sorting_1\\\">1983</td>\\n</tr><tr id=\\\"pkg_singscore\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/singscore.html\\\">singscore</a></td>\\n\\t<td class=\\\" \\\">Dharmesh D. Bhuva</td>\\n\\t<td class=\\\" \\\">Rank-based single-sample gene set scoring method</td>\\n\\t<td class=\\\"  sorting_1\\\">1984</td>\\n</tr><tr id=\\\"pkg_BiocBaseUtils\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/BiocBaseUtils.html\\\">BiocBaseUtils</a></td>\\n\\t<td class=\\\" \\\">Marcel Ramos</td>\\n\\t<td class=\\\" \\\">General utility functions for developing Bioconductor packages</td>\\n\\t<td class=\\\"  sorting_1\\\">1985</td>\\n</tr><tr id=\\\"pkg_RaggedExperiment\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/RaggedExperiment.html\\\">RaggedExperiment</a></td>\\n\\t<td class=\\\" \\\">Marcel Ramos</td>\\n\\t<td class=\\\" \\\">Representation of Sparse Experiments and Assays Across Samples</td>\\n\\t<td class=\\\"  sorting_1\\\">1986</td>\\n</tr><tr id=\\\"pkg_RUVSeq\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/RUVSeq.html\\\">RUVSeq</a></td>\\n\\t<td class=\\\" \\\">Davide Risso</td>\\n\\t<td class=\\\" \\\">Remove Unwanted Variation from RNA-Seq Data</td>\\n\\t<td class=\\\"  sorting_1\\\">1987</td>\\n</tr><tr id=\\\"pkg_decontam\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/decontam.html\\\">decontam</a></td>\\n\\t<td class=\\\" \\\">Benjamin Callahan</td>\\n\\t<td class=\\\" \\\">Identify Contaminants in Marker-gene and Metagenomics Sequencing Data</td>\\n\\t<td class=\\\"  sorting_1\\\">1988</td>\\n</tr><tr id=\\\"pkg_GENIE3\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/GENIE3.html\\\">GENIE3</a></td>\\n\\t<td class=\\\" \\\">Van Anh Huynh-Thu</td>\\n\\t<td class=\\\" \\\">GEne Network Inference with Ensemble of trees</td>\\n\\t<td class=\\\"  sorting_1\\\">1989</td>\\n</tr><tr id=\\\"pkg_plyranges\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/plyranges.html\\\">plyranges</a></td>\\n\\t<td class=\\\" \\\">Stuart Lee</td>\\n\\t<td class=\\\" \\\">A fluent interface for manipulating GenomicRanges</td>\\n\\t<td class=\\\"  sorting_1\\\">1990</td>\\n</tr><tr id=\\\"pkg_RcisTarget\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/RcisTarget.html\\\">RcisTarget</a></td>\\n\\t<td class=\\\" \\\">Sara Aibar</td>\\n\\t<td class=\\\" \\\">RcisTarget Identify transcription factor binding motifs enriched on a list of genes or genomic regions</td>\\n\\t<td class=\\\"  sorting_1\\\">1991</td>\\n</tr><tr id=\\\"pkg_GenomicFiles\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/GenomicFiles.html\\\">GenomicFiles</a></td>\\n\\t<td class=\\\" \\\">Bioconductor Package Maintainer</td>\\n\\t<td class=\\\" \\\">Distributed computing by file or by range</td>\\n\\t<td class=\\\"  sorting_1\\\">1992</td>\\n</tr><tr id=\\\"pkg_missMethyl\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/missMethyl.html\\\">missMethyl</a></td>\\n\\t<td class=\\\" \\\">Belinda Phipson, Jovana Maksimovic, Andrew Lonsdale</td>\\n\\t<td class=\\\" \\\">Analysing Illumina HumanMethylation BeadChip Data</td>\\n\\t<td class=\\\"  sorting_1\\\">1993</td>\\n</tr><tr id=\\\"pkg_ChIPpeakAnno\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ChIPpeakAnno.html\\\">ChIPpeakAnno</a></td>\\n\\t<td class=\\\" \\\">Jianhong Ou, Lihua Julie Zhu</td>\\n\\t<td class=\\\" \\\">Batch annotation of the peaks identified from either ChIP-seq, ChIP-chip experiments or any experiments resulted in large number of chromosome ranges</td>\\n\\t<td class=\\\"  sorting_1\\\">1994</td>\\n</tr><tr id=\\\"pkg_chromVAR\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/chromVAR.html\\\">chromVAR</a></td>\\n\\t<td class=\\\" \\\">Alicia Schep</td>\\n\\t<td class=\\\" \\\">Chromatin Variation Across Regions</td>\\n\\t<td class=\\\"  sorting_1\\\">1995</td>\\n</tr><tr id=\\\"pkg_FlowSOM\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/FlowSOM.html\\\">FlowSOM</a></td>\\n\\t<td class=\\\" \\\">Sofie Van Gassen</td>\\n\\t<td class=\\\" \\\">Using self-organizing maps for visualization and interpretation of cytometry data</td>\\n\\t<td class=\\\"  sorting_1\\\">1995</td>\\n</tr><tr id=\\\"pkg_bamsignals\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/bamsignals.html\\\">bamsignals</a></td>\\n\\t<td class=\\\" \\\">Johannes Helmuth</td>\\n\\t<td class=\\\" \\\">Extract read count signals from bam files</td>\\n\\t<td class=\\\"  sorting_1\\\">1997</td>\\n</tr><tr id=\\\"pkg_CytoML\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/CytoML.html\\\">CytoML</a></td>\\n\\t<td class=\\\" \\\">Mike Jiang</td>\\n\\t<td class=\\\" \\\">A GatingML Interface for Cross Platform Cytometry Data Sharing</td>\\n\\t<td class=\\\"  sorting_1\\\">1998</td>\\n</tr><tr id=\\\"pkg_gage\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/gage.html\\\">gage</a></td>\\n\\t<td class=\\\" \\\">Weijun Luo</td>\\n\\t<td class=\\\" \\\">Generally Applicable Gene-set Enrichment for Pathway Analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">1999</td>\\n</tr><tr id=\\\"pkg_metagenomeSeq\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/metagenomeSeq.html\\\">metagenomeSeq</a></td>\\n\\t<td class=\\\" \\\">Joseph N. Paulson</td>\\n\\t<td class=\\\" \\\">Statistical analysis for sparse high-throughput sequencing</td>\\n\\t<td class=\\\"  sorting_1\\\">2000</td>\\n</tr><tr id=\\\"pkg_motifmatchr\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/motifmatchr.html\\\">motifmatchr</a></td>\\n\\t<td class=\\\" \\\">Alicia Schep</td>\\n\\t<td class=\\\" \\\">Fast Motif Matching in R</td>\\n\\t<td class=\\\"  sorting_1\\\">2001</td>\\n</tr><tr id=\\\"pkg_tkWidgets\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/tkWidgets.html\\\">tkWidgets</a></td>\\n\\t<td class=\\\" \\\">J. Zhang</td>\\n\\t<td class=\\\" \\\">R based tk widgets</td>\\n\\t<td class=\\\"  sorting_1\\\">2001</td>\\n</tr><tr id=\\\"pkg_Wrench\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/Wrench.html\\\">Wrench</a></td>\\n\\t<td class=\\\" \\\">Hector Corrada Bravo</td>\\n\\t<td class=\\\" \\\">Wrench normalization for sparse count data</td>\\n\\t<td class=\\\"  sorting_1\\\">2003</td>\\n</tr><tr id=\\\"pkg_PCAtools\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/PCAtools.html\\\">PCAtools</a></td>\\n\\t<td class=\\\" \\\">Kevin Blighe</td>\\n\\t<td class=\\\" \\\">PCAtools: Everything Principal Components Analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">2004</td>\\n</tr><tr id=\\\"pkg_DynDoc\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/DynDoc.html\\\">DynDoc</a></td>\\n\\t<td class=\\\" \\\">Bioconductor Package Maintainer</td>\\n\\t<td class=\\\" \\\">Dynamic document tools</td>\\n\\t<td class=\\\"  sorting_1\\\">2005</td>\\n</tr><tr id=\\\"pkg_widgetTools\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/widgetTools.html\\\">widgetTools</a></td>\\n\\t<td class=\\\" \\\">Jianhua Zhang</td>\\n\\t<td class=\\\" \\\">Creates an interactive tcltk widget</td>\\n\\t<td class=\\\"  sorting_1\\\">2005</td>\\n</tr><tr id=\\\"pkg_DSS\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/DSS.html\\\">DSS</a></td>\\n\\t<td class=\\\" \\\">Hao Wu, Hao Feng</td>\\n\\t<td class=\\\" \\\">Dispersion shrinkage for sequencing data</td>\\n\\t<td class=\\\"  sorting_1\\\">2007</td>\\n</tr><tr id=\\\"pkg_tximeta\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/tximeta.html\\\">tximeta</a></td>\\n\\t<td class=\\\" \\\">Michael Love</td>\\n\\t<td class=\\\" \\\">Transcript Quantification Import with Automatic Metadata</td>\\n\\t<td class=\\\"  sorting_1\\\">2008</td>\\n</tr><tr id=\\\"pkg_flowStats\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/flowStats.html\\\">flowStats</a></td>\\n\\t<td class=\\\" \\\">Greg Finak, Mike Jiang</td>\\n\\t<td class=\\\" \\\">Statistical methods for the analysis of flow cytometry data</td>\\n\\t<td class=\\\"  sorting_1\\\">2009</td>\\n</tr><tr id=\\\"pkg_lpsymphony\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/lpsymphony.html\\\">lpsymphony</a></td>\\n\\t<td class=\\\" \\\">Vladislav Kim</td>\\n\\t<td class=\\\" \\\">Symphony integer linear programming solver in R</td>\\n\\t<td class=\\\"  sorting_1\\\">2010</td>\\n</tr><tr id=\\\"pkg_openCyto\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/openCyto.html\\\">openCyto</a></td>\\n\\t<td class=\\\" \\\">Mike Jiang</td>\\n\\t<td class=\\\" \\\">Hierarchical Gating Pipeline for flow cytometry data</td>\\n\\t<td class=\\\"  sorting_1\\\">2011</td>\\n</tr><tr id=\\\"pkg_flowClust\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/flowClust.html\\\">flowClust</a></td>\\n\\t<td class=\\\" \\\">Greg Finak, Mike Jiang</td>\\n\\t<td class=\\\" \\\">Clustering for Flow Cytometry</td>\\n\\t<td class=\\\"  sorting_1\\\">2012</td>\\n</tr><tr id=\\\"pkg_DiffBind\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/DiffBind.html\\\">DiffBind</a></td>\\n\\t<td class=\\\" \\\">Rory Stark</td>\\n\\t<td class=\\\" \\\">Differential Binding Analysis of ChIP-Seq Peak Data</td>\\n\\t<td class=\\\"  sorting_1\\\">2013</td>\\n</tr><tr id=\\\"pkg_GenomicDataCommons\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/GenomicDataCommons.html\\\">GenomicDataCommons</a></td>\\n\\t<td class=\\\" \\\">Sean Davis</td>\\n\\t<td class=\\\" \\\">NIH / NCI Genomic Data Commons Access</td>\\n\\t<td class=\\\"  sorting_1\\\">2014</td>\\n</tr><tr id=\\\"pkg_ROC\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ROC.html\\\">ROC</a></td>\\n\\t<td class=\\\" \\\">Vince Carey</td>\\n\\t<td class=\\\" \\\">utilities for ROC, with microarray focus</td>\\n\\t<td class=\\\"  sorting_1\\\">2015</td>\\n</tr><tr id=\\\"pkg_lumi\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/lumi.html\\\">lumi</a></td>\\n\\t<td class=\\\" \\\">Lei Huang</td>\\n\\t<td class=\\\" \\\">BeadArray Specific Methods for Illumina Methylation and Expression Microarrays</td>\\n\\t<td class=\\\"  sorting_1\\\">2016</td>\\n</tr><tr id=\\\"pkg_M3C\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/M3C.html\\\">M3C</a></td>\\n\\t<td class=\\\" \\\">Christopher John</td>\\n\\t<td class=\\\" \\\">Monte Carlo Reference-based Consensus Clustering</td>\\n\\t<td class=\\\"  sorting_1\\\">2017</td>\\n</tr><tr id=\\\"pkg_slingshot\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/slingshot.html\\\">slingshot</a></td>\\n\\t<td class=\\\" \\\">Kelly Street</td>\\n\\t<td class=\\\" \\\">Tools for ordering single-cell sequencing</td>\\n\\t<td class=\\\"  sorting_1\\\">2018</td>\\n</tr><tr id=\\\"pkg_MsFeatures\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/MsFeatures.html\\\">MsFeatures</a></td>\\n\\t<td class=\\\" \\\">Johannes Rainer</td>\\n\\t<td class=\\\" \\\">Functionality for Mass Spectrometry Features</td>\\n\\t<td class=\\\"  sorting_1\\\">2019</td>\\n</tr><tr id=\\\"pkg_DEXSeq\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/DEXSeq.html\\\">DEXSeq</a></td>\\n\\t<td class=\\\" \\\">Alejandro Reyes</td>\\n\\t<td class=\\\" \\\">Inference of differential exon usage in RNA-Seq</td>\\n\\t<td class=\\\"  sorting_1\\\">2020</td>\\n</tr><tr id=\\\"pkg_systemPipeR\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/systemPipeR.html\\\">systemPipeR</a></td>\\n\\t<td class=\\\" \\\">Thomas Girke</td>\\n\\t<td class=\\\" \\\">systemPipeR: NGS workflow and report generation environment</td>\\n\\t<td class=\\\"  sorting_1\\\">2021</td>\\n</tr><tr id=\\\"pkg_TrajectoryUtils\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/TrajectoryUtils.html\\\">TrajectoryUtils</a></td>\\n\\t<td class=\\\" \\\">Aaron Lun</td>\\n\\t<td class=\\\" \\\">Single-Cell Trajectory Analysis Utilities</td>\\n\\t<td class=\\\"  sorting_1\\\">2022</td>\\n</tr><tr id=\\\"pkg_ggcyto\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ggcyto.html\\\">ggcyto</a></td>\\n\\t<td class=\\\" \\\">Mike Jiang</td>\\n\\t<td class=\\\" \\\">Visualize Cytometry data with ggplot</td>\\n\\t<td class=\\\"  sorting_1\\\">2023</td>\\n</tr><tr id=\\\"pkg_msa\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/msa.html\\\">msa</a></td>\\n\\t<td class=\\\" \\\">Ulrich Bodenhofer</td>\\n\\t<td class=\\\" \\\">Multiple Sequence Alignment</td>\\n\\t<td class=\\\"  sorting_1\\\">2024</td>\\n</tr><tr id=\\\"pkg_goseq\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/goseq.html\\\">goseq</a></td>\\n\\t<td class=\\\" \\\">Matthew Young, Nadia Davidson</td>\\n\\t<td class=\\\" \\\">Gene Ontology analyser for RNA-seq and other length biased data</td>\\n\\t<td class=\\\"  sorting_1\\\">2025</td>\\n</tr><tr id=\\\"pkg_bsseq\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/bsseq.html\\\">bsseq</a></td>\\n\\t<td class=\\\" \\\">Kasper Daniel Hansen</td>\\n\\t<td class=\\\" \\\">Analyze, manage and store bisulfite sequencing data</td>\\n\\t<td class=\\\"  sorting_1\\\">2026</td>\\n</tr><tr id=\\\"pkg_Glimma\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/Glimma.html\\\">Glimma</a></td>\\n\\t<td class=\\\" \\\">Shian Su</td>\\n\\t<td class=\\\" \\\">Interactive HTML graphics</td>\\n\\t<td class=\\\"  sorting_1\\\">2027</td>\\n</tr><tr id=\\\"pkg_InteractionSet\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/InteractionSet.html\\\">InteractionSet</a></td>\\n\\t<td class=\\\" \\\">Aaron Lun</td>\\n\\t<td class=\\\" \\\">Base Classes for Storing Genomic Interaction Data</td>\\n\\t<td class=\\\"  sorting_1\\\">2028</td>\\n</tr><tr id=\\\"pkg_dir.expiry\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/dir.expiry.html\\\">dir.expiry</a></td>\\n\\t<td class=\\\" \\\">Aaron Lun</td>\\n\\t<td class=\\\" \\\">Managing Expiration for Cache Directories</td>\\n\\t<td class=\\\"  sorting_1\\\">2029</td>\\n</tr><tr id=\\\"pkg_basilisk.utils\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/basilisk.utils.html\\\">basilisk.utils</a></td>\\n\\t<td class=\\\" \\\">Aaron Lun</td>\\n\\t<td class=\\\" \\\">Basilisk Installation Utilities</td>\\n\\t<td class=\\\"  sorting_1\\\">2030</td>\\n</tr><tr id=\\\"pkg_flowViz\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/flowViz.html\\\">flowViz</a></td>\\n\\t<td class=\\\" \\\">Mike Jiang</td>\\n\\t<td class=\\\" \\\">Visualization for flow cytometry</td>\\n\\t<td class=\\\"  sorting_1\\\">2031</td>\\n</tr><tr id=\\\"pkg_SNPRelate\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/SNPRelate.html\\\">SNPRelate</a></td>\\n\\t<td class=\\\" \\\">Xiuwen Zheng</td>\\n\\t<td class=\\\" \\\">Parallel Computing Toolset for Relatedness and Principal Component Analysis of SNP Data</td>\\n\\t<td class=\\\"  sorting_1\\\">2032</td>\\n</tr><tr id=\\\"pkg_xcms\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/xcms.html\\\">xcms</a></td>\\n\\t<td class=\\\" \\\">Steffen Neumann</td>\\n\\t<td class=\\\" \\\">LC-MS and GC-MS Data Analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">2033</td>\\n</tr><tr id=\\\"pkg_basilisk\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/basilisk.html\\\">basilisk</a></td>\\n\\t<td class=\\\" \\\">Aaron Lun</td>\\n\\t<td class=\\\" \\\">Freezing Python Dependencies Inside Bioconductor Packages</td>\\n\\t<td class=\\\"  sorting_1\\\">2034</td>\\n</tr><tr id=\\\"pkg_globaltest\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/globaltest.html\\\">globaltest</a></td>\\n\\t<td class=\\\" \\\">Jelle Goeman</td>\\n\\t<td class=\\\" \\\">Testing Groups of Covariates/Features for Association with a Response Variable, with Applications to Gene Set Testing</td>\\n\\t<td class=\\\"  sorting_1\\\">2035</td>\\n</tr><tr id=\\\"pkg_methylumi\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/methylumi.html\\\">methylumi</a></td>\\n\\t<td class=\\\" \\\">Sean Davis</td>\\n\\t<td class=\\\" \\\">Handle Illumina methylation data</td>\\n\\t<td class=\\\"  sorting_1\\\">2036</td>\\n</tr><tr id=\\\"pkg_affyPLM\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/affyPLM.html\\\">affyPLM</a></td>\\n\\t<td class=\\\" \\\">Ben Bolstad</td>\\n\\t<td class=\\\" \\\">Methods for fitting probe-level models</td>\\n\\t<td class=\\\"  sorting_1\\\">2037</td>\\n</tr><tr id=\\\"pkg_flowWorkspace\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/flowWorkspace.html\\\">flowWorkspace</a></td>\\n\\t<td class=\\\" \\\">Greg Finak, Mike Jiang</td>\\n\\t<td class=\\\" \\\">Infrastructure for representing and interacting with gated and ungated cytometry data sets.</td>\\n\\t<td class=\\\"  sorting_1\\\">2038</td>\\n</tr><tr id=\\\"pkg_ncdfFlow\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ncdfFlow.html\\\">ncdfFlow</a></td>\\n\\t<td class=\\\" \\\">Mike Jiang</td>\\n\\t<td class=\\\" \\\">ncdfFlow: A package that provides HDF5 based storage for flow cytometry data.</td>\\n\\t<td class=\\\"  sorting_1\\\">2039</td>\\n</tr><tr id=\\\"pkg_MassSpecWavelet\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/MassSpecWavelet.html\\\">MassSpecWavelet</a></td>\\n\\t<td class=\\\" \\\">Sergio Oller Moreno</td>\\n\\t<td class=\\\" \\\">Peak Detection for Mass Spectrometry data using wavelet-based algorithms</td>\\n\\t<td class=\\\"  sorting_1\\\">2040</td>\\n</tr><tr id=\\\"pkg_gcrma\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/gcrma.html\\\">gcrma</a></td>\\n\\t<td class=\\\" \\\">Z. Wu</td>\\n\\t<td class=\\\" \\\">Background Adjustment Using Sequence Information</td>\\n\\t<td class=\\\"  sorting_1\\\">2041</td>\\n</tr><tr id=\\\"pkg_affxparser\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/affxparser.html\\\">affxparser</a></td>\\n\\t<td class=\\\" \\\">Kasper Daniel Hansen</td>\\n\\t<td class=\\\" \\\">Affymetrix File Parsing SDK</td>\\n\\t<td class=\\\"  sorting_1\\\">2042</td>\\n</tr><tr id=\\\"pkg_oligo\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/oligo.html\\\">oligo</a></td>\\n\\t<td class=\\\" \\\">Benilton Carvalho</td>\\n\\t<td class=\\\" \\\">Preprocessing tools for oligonucleotide arrays</td>\\n\\t<td class=\\\"  sorting_1\\\">2043</td>\\n</tr><tr id=\\\"pkg_CNEr\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/CNEr.html\\\">CNEr</a></td>\\n\\t<td class=\\\" \\\">Ge Tan</td>\\n\\t<td class=\\\" \\\">CNE Detection and Visualization</td>\\n\\t<td class=\\\"  sorting_1\\\">2044</td>\\n</tr><tr id=\\\"pkg_TFBSTools\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/TFBSTools.html\\\">TFBSTools</a></td>\\n\\t<td class=\\\" \\\">Ge Tan</td>\\n\\t<td class=\\\" \\\">Software Package for Transcription Factor Binding Site (TFBS) Analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">2045</td>\\n</tr><tr id=\\\"pkg_microbiome\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/microbiome.html\\\">microbiome</a></td>\\n\\t<td class=\\\" \\\">Leo Lahti</td>\\n\\t<td class=\\\" \\\">Microbiome Analytics</td>\\n\\t<td class=\\\"  sorting_1\\\">2046</td>\\n</tr><tr id=\\\"pkg_glmGamPoi\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/glmGamPoi.html\\\">glmGamPoi</a></td>\\n\\t<td class=\\\" \\\">Constantin Ahlmann-Eltze</td>\\n\\t<td class=\\\" \\\">Fit a Gamma-Poisson Generalized Linear Model</td>\\n\\t<td class=\\\"  sorting_1\\\">2047</td>\\n</tr><tr id=\\\"pkg_oligoClasses\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/oligoClasses.html\\\">oligoClasses</a></td>\\n\\t<td class=\\\" \\\">Benilton Carvalho and Robert Scharpf</td>\\n\\t<td class=\\\" \\\">Classes for high-throughput arrays supported by oligo and crlmm</td>\\n\\t<td class=\\\"  sorting_1\\\">2048</td>\\n</tr><tr id=\\\"pkg_MAST\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/MAST.html\\\">MAST</a></td>\\n\\t<td class=\\\" \\\">Andrew McDavid</td>\\n\\t<td class=\\\" \\\">Model-based Analysis of Single Cell Transcriptomics</td>\\n\\t<td class=\\\"  sorting_1\\\">2049</td>\\n</tr><tr id=\\\"pkg_EDASeq\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/EDASeq.html\\\">EDASeq</a></td>\\n\\t<td class=\\\" \\\">Davide Risso</td>\\n\\t<td class=\\\" \\\">Exploratory Data Analysis and Normalization for RNA-Seq</td>\\n\\t<td class=\\\"  sorting_1\\\">2050</td>\\n</tr><tr id=\\\"pkg_AUCell\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/AUCell.html\\\">AUCell</a></td>\\n\\t<td class=\\\" \\\">Sara Aibar</td>\\n\\t<td class=\\\" \\\">AUCell: Analysis of 'gene set' activity in single-cell RNA-seq data (e.g. identify cells with specific gene signatures)</td>\\n\\t<td class=\\\"  sorting_1\\\">2051</td>\\n</tr><tr id=\\\"pkg_BiocCheck\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/BiocCheck.html\\\">BiocCheck</a></td>\\n\\t<td class=\\\" \\\">Marcel Ramos</td>\\n\\t<td class=\\\" \\\">Bioconductor-specific package checks</td>\\n\\t<td class=\\\"  sorting_1\\\">2052</td>\\n</tr><tr id=\\\"pkg_marray\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/marray.html\\\">marray</a></td>\\n\\t<td class=\\\" \\\">Yee Hwa (Jean) Yang</td>\\n\\t<td class=\\\" \\\">Exploratory analysis for two-color spotted microarray data</td>\\n\\t<td class=\\\"  sorting_1\\\">2053</td>\\n</tr><tr id=\\\"pkg_ChIPseeker\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ChIPseeker.html\\\">ChIPseeker</a></td>\\n\\t<td class=\\\" \\\">Guangchuang Yu</td>\\n\\t<td class=\\\" \\\">ChIPseeker for ChIP peak Annotation, Comparison, and Visualization</td>\\n\\t<td class=\\\"  sorting_1\\\">2054</td>\\n</tr><tr id=\\\"pkg_snpStats\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/snpStats.html\\\">snpStats</a></td>\\n\\t<td class=\\\" \\\">David Clayton</td>\\n\\t<td class=\\\" \\\">SnpMatrix and XSnpMatrix classes and methods</td>\\n\\t<td class=\\\"  sorting_1\\\">2055</td>\\n</tr><tr id=\\\"pkg_aroma.light\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/aroma.light.html\\\">aroma.light</a></td>\\n\\t<td class=\\\" \\\">Henrik Bengtsson</td>\\n\\t<td class=\\\" \\\">Light-Weight Methods for Normalization and Visualization of Microarray Data using Only Basic R Data Types</td>\\n\\t<td class=\\\"  sorting_1\\\">2056</td>\\n</tr><tr id=\\\"pkg_GOstats\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/GOstats.html\\\">GOstats</a></td>\\n\\t<td class=\\\" \\\">Bioconductor Package Maintainer</td>\\n\\t<td class=\\\" \\\">Tools for manipulating GO and microarrays</td>\\n\\t<td class=\\\"  sorting_1\\\">2056</td>\\n</tr><tr id=\\\"pkg_gdsfmt\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/gdsfmt.html\\\">gdsfmt</a></td>\\n\\t<td class=\\\" \\\">Xiuwen Zheng</td>\\n\\t<td class=\\\" \\\">R Interface to CoreArray Genomic Data Structure (GDS) Files</td>\\n\\t<td class=\\\"  sorting_1\\\">2058</td>\\n</tr><tr id=\\\"pkg_regioneR\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/regioneR.html\\\">regioneR</a></td>\\n\\t<td class=\\\" \\\">Bernat Gel</td>\\n\\t<td class=\\\" \\\">Association analysis of genomic regions based on permutation tests</td>\\n\\t<td class=\\\"  sorting_1\\\">2059</td>\\n</tr><tr id=\\\"pkg_ReactomePA\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ReactomePA.html\\\">ReactomePA</a></td>\\n\\t<td class=\\\" \\\">Guangchuang Yu</td>\\n\\t<td class=\\\" \\\">Reactome Pathway Analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">2060</td>\\n</tr><tr id=\\\"pkg_ggbio\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ggbio.html\\\">ggbio</a></td>\\n\\t<td class=\\\" \\\">Michael Lawrence</td>\\n\\t<td class=\\\" \\\">Visualization tools for genomic data</td>\\n\\t<td class=\\\"  sorting_1\\\">2061</td>\\n</tr><tr id=\\\"pkg_DECIPHER\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/DECIPHER.html\\\">DECIPHER</a></td>\\n\\t<td class=\\\" \\\">Erik Wright</td>\\n\\t<td class=\\\" \\\">Tools for curating, analyzing, and manipulating biological sequences</td>\\n\\t<td class=\\\"  sorting_1\\\">2062</td>\\n</tr><tr id=\\\"pkg_Category\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/Category.html\\\">Category</a></td>\\n\\t<td class=\\\" \\\">Bioconductor Package Maintainer</td>\\n\\t<td class=\\\" \\\">Category Analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">2063</td>\\n</tr><tr id=\\\"pkg_minfi\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/minfi.html\\\">minfi</a></td>\\n\\t<td class=\\\" \\\">Kasper Daniel Hansen</td>\\n\\t<td class=\\\" \\\">Analyze Illumina Infinium DNA methylation arrays</td>\\n\\t<td class=\\\"  sorting_1\\\">2064</td>\\n</tr><tr id=\\\"pkg_DropletUtils\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/DropletUtils.html\\\">DropletUtils</a></td>\\n\\t<td class=\\\" \\\">Jonathan Griffiths</td>\\n\\t<td class=\\\" \\\">Utilities for Handling Single-Cell Droplet Data</td>\\n\\t<td class=\\\"  sorting_1\\\">2065</td>\\n</tr><tr id=\\\"pkg_seqLogo\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/seqLogo.html\\\">seqLogo</a></td>\\n\\t<td class=\\\" \\\">Robert Ivanek</td>\\n\\t<td class=\\\" \\\">Sequence logos for DNA sequence alignments</td>\\n\\t<td class=\\\"  sorting_1\\\">2066</td>\\n</tr><tr id=\\\"pkg_RProtoBufLib\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/RProtoBufLib.html\\\">RProtoBufLib</a></td>\\n\\t<td class=\\\" \\\">Mike Jiang</td>\\n\\t<td class=\\\" \\\">C++ headers and static libraries of Protocol buffers</td>\\n\\t<td class=\\\"  sorting_1\\\">2067</td>\\n</tr><tr id=\\\"pkg_dada2\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/dada2.html\\\">dada2</a></td>\\n\\t<td class=\\\" \\\">Benjamin Callahan</td>\\n\\t<td class=\\\" \\\">Accurate, high-resolution sample inference from amplicon sequencing data</td>\\n\\t<td class=\\\"  sorting_1\\\">2068</td>\\n</tr><tr id=\\\"pkg_DirichletMultinomial\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/DirichletMultinomial.html\\\">DirichletMultinomial</a></td>\\n\\t<td class=\\\" \\\">Martin Morgan</td>\\n\\t<td class=\\\" \\\">Dirichlet-Multinomial Mixture Model Machine Learning for Microbiome Data</td>\\n\\t<td class=\\\"  sorting_1\\\">2069</td>\\n</tr><tr id=\\\"pkg_Rsubread\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/Rsubread.html\\\">Rsubread</a></td>\\n\\t<td class=\\\" \\\">Wei Shi, Yang Liao and Gordon K Smyth</td>\\n\\t<td class=\\\" \\\">Mapping, quantification and variant analysis of sequencing data</td>\\n\\t<td class=\\\"  sorting_1\\\">2070</td>\\n</tr><tr id=\\\"pkg_flowCore\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/flowCore.html\\\">flowCore</a></td>\\n\\t<td class=\\\" \\\">Mike Jiang</td>\\n\\t<td class=\\\" \\\">flowCore: Basic structures for flow cytometry data</td>\\n\\t<td class=\\\"  sorting_1\\\">2071</td>\\n</tr><tr id=\\\"pkg_graphite\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/graphite.html\\\">graphite</a></td>\\n\\t<td class=\\\" \\\">Gabriele Sales</td>\\n\\t<td class=\\\" \\\">GRAPH Interaction from pathway Topological Environment</td>\\n\\t<td class=\\\"  sorting_1\\\">2072</td>\\n</tr><tr id=\\\"pkg_EBImage\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/EBImage.html\\\">EBImage</a></td>\\n\\t<td class=\\\" \\\">Andrzej Ole</td>\\n\\t<td class=\\\" \\\">Image processing and analysis toolbox for R</td>\\n\\t<td class=\\\"  sorting_1\\\">2073</td>\\n</tr><tr id=\\\"pkg_bumphunter\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/bumphunter.html\\\">bumphunter</a></td>\\n\\t<td class=\\\" \\\">Tamilselvi Guharaj</td>\\n\\t<td class=\\\" \\\">Bump Hunter</td>\\n\\t<td class=\\\"  sorting_1\\\">2074</td>\\n</tr><tr id=\\\"pkg_cytolib\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/cytolib.html\\\">cytolib</a></td>\\n\\t<td class=\\\" \\\">Mike Jiang</td>\\n\\t<td class=\\\" \\\">C++ infrastructure for representing and interacting with the gated cytometry data</td>\\n\\t<td class=\\\"  sorting_1\\\">2075</td>\\n</tr><tr id=\\\"pkg_AnnotationForge\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/AnnotationForge.html\\\">AnnotationForge</a></td>\\n\\t<td class=\\\" \\\">Bioconductor Package Maintainer</td>\\n\\t<td class=\\\" \\\">Tools for building SQLite-based annotation data packages</td>\\n\\t<td class=\\\"  sorting_1\\\">2076</td>\\n</tr><tr id=\\\"pkg_mzID\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/mzID.html\\\">mzID</a></td>\\n\\t<td class=\\\" \\\">Laurent Gatto</td>\\n\\t<td class=\\\" \\\">An mzIdentML parser for R</td>\\n\\t<td class=\\\"  sorting_1\\\">2077</td>\\n</tr><tr id=\\\"pkg_monocle\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/monocle.html\\\">monocle</a></td>\\n\\t<td class=\\\" \\\">Cole Trapnell</td>\\n\\t<td class=\\\" \\\">Clustering, differential expression, and trajectory analysis for single- cell RNA-Seq</td>\\n\\t<td class=\\\"  sorting_1\\\">2078</td>\\n</tr><tr id=\\\"pkg_mixOmics\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/mixOmics.html\\\">mixOmics</a></td>\\n\\t<td class=\\\" \\\">Al J Abadi</td>\\n\\t<td class=\\\" \\\">Omics Data Integration Project</td>\\n\\t<td class=\\\"  sorting_1\\\">2079</td>\\n</tr><tr id=\\\"pkg_MsCoreUtils\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/MsCoreUtils.html\\\">MsCoreUtils</a></td>\\n\\t<td class=\\\" \\\">RforMassSpectrometry Package Maintainer</td>\\n\\t<td class=\\\" \\\">Core Utils for Mass Spectrometry Data</td>\\n\\t<td class=\\\"  sorting_1\\\">2080</td>\\n</tr><tr id=\\\"pkg_maftools\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/maftools.html\\\">maftools</a></td>\\n\\t<td class=\\\" \\\">Anand Mayakonda</td>\\n\\t<td class=\\\" \\\">Summarize, Analyze and Visualize MAF Files</td>\\n\\t<td class=\\\"  sorting_1\\\">2081</td>\\n</tr><tr id=\\\"pkg_illuminaio\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/illuminaio.html\\\">illuminaio</a></td>\\n\\t<td class=\\\" \\\">Kasper Daniel Hansen</td>\\n\\t<td class=\\\" \\\">Parsing Illumina Microarray Output Files</td>\\n\\t<td class=\\\"  sorting_1\\\">2082</td>\\n</tr><tr id=\\\"pkg_siggenes\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/siggenes.html\\\">siggenes</a></td>\\n\\t<td class=\\\" \\\">Holger Schwender</td>\\n\\t<td class=\\\" \\\">Multiple Testing using SAM and Efron's Empirical Bayes Approaches</td>\\n\\t<td class=\\\"  sorting_1\\\">2083</td>\\n</tr><tr id=\\\"pkg_mzR\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/mzR.html\\\">mzR</a></td>\\n\\t<td class=\\\" \\\">Steffen Neumann</td>\\n\\t<td class=\\\" \\\">parser for netCDF, mzXML, mzData and mzML and mzIdentML files (mass spectrometry data)</td>\\n\\t<td class=\\\"  sorting_1\\\">2084</td>\\n</tr><tr id=\\\"pkg_MultiAssayExperiment\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/MultiAssayExperiment.html\\\">MultiAssayExperiment</a></td>\\n\\t<td class=\\\" \\\">Marcel Ramos</td>\\n\\t<td class=\\\" \\\">Software for the integration of multi-omics experiments in Bioconductor</td>\\n\\t<td class=\\\"  sorting_1\\\">2085</td>\\n</tr><tr id=\\\"pkg_SingleR\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/SingleR.html\\\">SingleR</a></td>\\n\\t<td class=\\\" \\\">Aaron Lun</td>\\n\\t<td class=\\\" \\\">Reference-Based Single-Cell RNA-Seq Annotation</td>\\n\\t<td class=\\\"  sorting_1\\\">2086</td>\\n</tr><tr id=\\\"pkg_ConsensusClusterPlus\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ConsensusClusterPlus.html\\\">ConsensusClusterPlus</a></td>\\n\\t<td class=\\\" \\\">Matt Wilkerson</td>\\n\\t<td class=\\\" \\\">ConsensusClusterPlus</td>\\n\\t<td class=\\\"  sorting_1\\\">2087</td>\\n</tr><tr id=\\\"pkg_ResidualMatrix\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ResidualMatrix.html\\\">ResidualMatrix</a></td>\\n\\t<td class=\\\" \\\">Aaron Lun</td>\\n\\t<td class=\\\" \\\">Creating a DelayedMatrix of Regression Residuals</td>\\n\\t<td class=\\\"  sorting_1\\\">2088</td>\\n</tr><tr id=\\\"pkg_OrganismDbi\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/OrganismDbi.html\\\">OrganismDbi</a></td>\\n\\t<td class=\\\" \\\">Bioconductor Package Maintainer</td>\\n\\t<td class=\\\" \\\">Software to enable the smooth interfacing of different database packages</td>\\n\\t<td class=\\\"  sorting_1\\\">2089</td>\\n</tr><tr id=\\\"pkg_topGO\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/topGO.html\\\">topGO</a></td>\\n\\t<td class=\\\" \\\">Adrian Alexa</td>\\n\\t<td class=\\\" \\\">Enrichment Analysis for Gene Ontology</td>\\n\\t<td class=\\\"  sorting_1\\\">2090</td>\\n</tr><tr id=\\\"pkg_MSnbase\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/MSnbase.html\\\">MSnbase</a></td>\\n\\t<td class=\\\" \\\">Laurent Gatto</td>\\n\\t<td class=\\\" \\\">Base Functions and Classes for Mass Spectrometry and Proteomics</td>\\n\\t<td class=\\\"  sorting_1\\\">2091</td>\\n</tr><tr id=\\\"pkg_batchelor\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/batchelor.html\\\">batchelor</a></td>\\n\\t<td class=\\\" \\\">Aaron Lun</td>\\n\\t<td class=\\\" \\\">Single-Cell Batch Correction Methods</td>\\n\\t<td class=\\\"  sorting_1\\\">2092</td>\\n</tr><tr id=\\\"pkg_TCGAbiolinks\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/TCGAbiolinks.html\\\">TCGAbiolinks</a></td>\\n\\t<td class=\\\" \\\">Tiago Chedraoui Silva, Antonio Colaprico</td>\\n\\t<td class=\\\" \\\">TCGAbiolinks: An R/Bioconductor package for integrative analysis with GDC data</td>\\n\\t<td class=\\\"  sorting_1\\\">2093</td>\\n</tr><tr id=\\\"pkg_tximport\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/tximport.html\\\">tximport</a></td>\\n\\t<td class=\\\" \\\">Michael Love</td>\\n\\t<td class=\\\" \\\">Import and summarize transcript-level estimates for transcript- and gene-level analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">2094</td>\\n</tr><tr id=\\\"pkg_apeglm\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/apeglm.html\\\">apeglm</a></td>\\n\\t<td class=\\\" \\\">Anqi Zhu</td>\\n\\t<td class=\\\" \\\">Approximate posterior estimation for GLM coefficients</td>\\n\\t<td class=\\\"  sorting_1\\\">2095</td>\\n</tr><tr id=\\\"pkg_Gviz\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/Gviz.html\\\">Gviz</a></td>\\n\\t<td class=\\\" \\\">Robert Ivanek</td>\\n\\t<td class=\\\" \\\">Plotting data and annotation information along genomic coordinates</td>\\n\\t<td class=\\\"  sorting_1\\\">2096</td>\\n</tr><tr id=\\\"pkg_metapod\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/metapod.html\\\">metapod</a></td>\\n\\t<td class=\\\" \\\">Aaron Lun</td>\\n\\t<td class=\\\" \\\">Meta-Analyses on P-Values of Differential Analyses</td>\\n\\t<td class=\\\"  sorting_1\\\">2097</td>\\n</tr><tr id=\\\"pkg_EnhancedVolcano\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/EnhancedVolcano.html\\\">EnhancedVolcano</a></td>\\n\\t<td class=\\\" \\\">Kevin Blighe</td>\\n\\t<td class=\\\" \\\">Publication-ready volcano plots with enhanced colouring and labeling</td>\\n\\t<td class=\\\"  sorting_1\\\">2098</td>\\n</tr><tr id=\\\"pkg_bluster\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/bluster.html\\\">bluster</a></td>\\n\\t<td class=\\\" \\\">Aaron Lun</td>\\n\\t<td class=\\\" \\\">Clustering Algorithms for Bioconductor</td>\\n\\t<td class=\\\"  sorting_1\\\">2099</td>\\n</tr><tr id=\\\"pkg_scran\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/scran.html\\\">scran</a></td>\\n\\t<td class=\\\" \\\">Aaron Lun</td>\\n\\t<td class=\\\" \\\">Methods for Single-Cell RNA-Seq Data Analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">2100</td>\\n</tr><tr id=\\\"pkg_biocViews\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/biocViews.html\\\">biocViews</a></td>\\n\\t<td class=\\\" \\\">Bioconductor Package Maintainer</td>\\n\\t<td class=\\\" \\\">Categorized views of R package repositories</td>\\n\\t<td class=\\\"  sorting_1\\\">2101</td>\\n</tr><tr id=\\\"pkg_DNAcopy\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/DNAcopy.html\\\">DNAcopy</a></td>\\n\\t<td class=\\\" \\\">Venkatraman E. Seshan</td>\\n\\t<td class=\\\" \\\">DNA copy number data analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">2102</td>\\n</tr><tr id=\\\"pkg_pathview\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/pathview.html\\\">pathview</a></td>\\n\\t<td class=\\\" \\\">Weijun Luo</td>\\n\\t<td class=\\\" \\\">a tool set for pathway based data integration and visualization</td>\\n\\t<td class=\\\"  sorting_1\\\">2103</td>\\n</tr><tr id=\\\"pkg_phyloseq\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/phyloseq.html\\\">phyloseq</a></td>\\n\\t<td class=\\\" \\\">Paul J. McMurdie</td>\\n\\t<td class=\\\" \\\">Handling and analysis of high-throughput microbiome census data</td>\\n\\t<td class=\\\"  sorting_1\\\">2104</td>\\n</tr><tr id=\\\"pkg_pcaMethods\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/pcaMethods.html\\\">pcaMethods</a></td>\\n\\t<td class=\\\" \\\">Henning Redestig</td>\\n\\t<td class=\\\" \\\">A collection of PCA methods</td>\\n\\t<td class=\\\"  sorting_1\\\">2105</td>\\n</tr><tr id=\\\"pkg_biomformat\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/biomformat.html\\\">biomformat</a></td>\\n\\t<td class=\\\" \\\">Paul J. McMurdie</td>\\n\\t<td class=\\\" \\\">An interface package for the BIOM file format</td>\\n\\t<td class=\\\"  sorting_1\\\">2106</td>\\n</tr><tr id=\\\"pkg_GSVA\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/GSVA.html\\\">GSVA</a></td>\\n\\t<td class=\\\" \\\">Robert Castelo</td>\\n\\t<td class=\\\" \\\">Gene Set Variation Analysis for microarray and RNA-seq data</td>\\n\\t<td class=\\\"  sorting_1\\\">2107</td>\\n</tr><tr id=\\\"pkg_biovizBase\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/biovizBase.html\\\">biovizBase</a></td>\\n\\t<td class=\\\" \\\">Michael Lawrence</td>\\n\\t<td class=\\\" \\\">Basic graphic utilities for visualization of genomic data.</td>\\n\\t<td class=\\\"  sorting_1\\\">2108</td>\\n</tr><tr id=\\\"pkg_KEGGgraph\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/KEGGgraph.html\\\">KEGGgraph</a></td>\\n\\t<td class=\\\" \\\">Jitao David Zhang</td>\\n\\t<td class=\\\" \\\">KEGGgraph: A graph approach to KEGG PATHWAY in R and Bioconductor</td>\\n\\t<td class=\\\"  sorting_1\\\">2109</td>\\n</tr><tr id=\\\"pkg_vsn\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/vsn.html\\\">vsn</a></td>\\n\\t<td class=\\\" \\\">Wolfgang Huber</td>\\n\\t<td class=\\\" \\\">Variance stabilization and calibration for microarray data</td>\\n\\t<td class=\\\"  sorting_1\\\">2110</td>\\n</tr><tr id=\\\"pkg_scater\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/scater.html\\\">scater</a></td>\\n\\t<td class=\\\" \\\">Alan O'Callaghan</td>\\n\\t<td class=\\\" \\\">Single-Cell Analysis Toolkit for Gene Expression Data in R</td>\\n\\t<td class=\\\"  sorting_1\\\">2111</td>\\n</tr><tr id=\\\"pkg_ExperimentHub\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ExperimentHub.html\\\">ExperimentHub</a></td>\\n\\t<td class=\\\" \\\">Bioconductor Package Maintainer</td>\\n\\t<td class=\\\" \\\">Client to access ExperimentHub resources</td>\\n\\t<td class=\\\"  sorting_1\\\">2112</td>\\n</tr><tr id=\\\"pkg_ShortRead\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ShortRead.html\\\">ShortRead</a></td>\\n\\t<td class=\\\" \\\">Bioconductor Package Maintainer</td>\\n\\t<td class=\\\" \\\">FASTQ input and manipulation</td>\\n\\t<td class=\\\"  sorting_1\\\">2113</td>\\n</tr><tr id=\\\"pkg_sva\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/sva.html\\\">sva</a></td>\\n\\t<td class=\\\" \\\">Jeffrey T. Leek, John D. Storey, W. Evan Johnson</td>\\n\\t<td class=\\\" \\\">Surrogate Variable Analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">2114</td>\\n</tr><tr id=\\\"pkg_BiocStyle\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/BiocStyle.html\\\">BiocStyle</a></td>\\n\\t<td class=\\\" \\\">Bioconductor Package</td>\\n\\t<td class=\\\" \\\">Standard styles for vignettes and other Bioconductor documents</td>\\n\\t<td class=\\\"  sorting_1\\\">2115</td>\\n</tr><tr id=\\\"pkg_VariantAnnotation\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/VariantAnnotation.html\\\">VariantAnnotation</a></td>\\n\\t<td class=\\\" \\\">Bioconductor Package Maintainer</td>\\n\\t<td class=\\\" \\\">Annotation of Genetic Variants</td>\\n\\t<td class=\\\"  sorting_1\\\">2116</td>\\n</tr><tr id=\\\"pkg_BiocNeighbors\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/BiocNeighbors.html\\\">BiocNeighbors</a></td>\\n\\t<td class=\\\" \\\">Aaron Lun</td>\\n\\t<td class=\\\" \\\">Nearest Neighbor Detection for Bioconductor Packages</td>\\n\\t<td class=\\\"  sorting_1\\\">2117</td>\\n</tr><tr id=\\\"pkg_scuttle\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/scuttle.html\\\">scuttle</a></td>\\n\\t<td class=\\\" \\\">Aaron Lun</td>\\n\\t<td class=\\\" \\\">Single-Cell RNA-Seq Analysis Utilities</td>\\n\\t<td class=\\\"  sorting_1\\\">2118</td>\\n</tr><tr id=\\\"pkg_GSEABase\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/GSEABase.html\\\">GSEABase</a></td>\\n\\t<td class=\\\" \\\">Bioconductor Package Maintainer</td>\\n\\t<td class=\\\" \\\">Gene set enrichment data structures and methods</td>\\n\\t<td class=\\\"  sorting_1\\\">2119</td>\\n</tr><tr id=\\\"pkg_AnnotationFilter\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/AnnotationFilter.html\\\">AnnotationFilter</a></td>\\n\\t<td class=\\\" \\\">Bioconductor Package Maintainer</td>\\n\\t<td class=\\\" \\\">Facilities for Filtering Bioconductor Annotation Resources</td>\\n\\t<td class=\\\"  sorting_1\\\">2120</td>\\n</tr><tr id=\\\"pkg_interactiveDisplayBase\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/interactiveDisplayBase.html\\\">interactiveDisplayBase</a></td>\\n\\t<td class=\\\" \\\">Bioconductor Package Maintainer</td>\\n\\t<td class=\\\" \\\">Base package for enabling powerful shiny web displays of Bioconductor objects</td>\\n\\t<td class=\\\"  sorting_1\\\">2121</td>\\n</tr><tr id=\\\"pkg_affy\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/affy.html\\\">affy</a></td>\\n\\t<td class=\\\" \\\">Rafael A. Irizarry</td>\\n\\t<td class=\\\" \\\">Methods for Affymetrix Oligonucleotide Arrays</td>\\n\\t<td class=\\\"  sorting_1\\\">2122</td>\\n</tr><tr id=\\\"pkg_affyio\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/affyio.html\\\">affyio</a></td>\\n\\t<td class=\\\" \\\">Ben Bolstad</td>\\n\\t<td class=\\\" \\\">Tools for parsing Affymetrix data files</td>\\n\\t<td class=\\\"  sorting_1\\\">2123</td>\\n</tr><tr id=\\\"pkg_ensembldb\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ensembldb.html\\\">ensembldb</a></td>\\n\\t<td class=\\\" \\\">Johannes Rainer</td>\\n\\t<td class=\\\" \\\">Utilities to create and use Ensembl-based annotation databases</td>\\n\\t<td class=\\\"  sorting_1\\\">2124</td>\\n</tr><tr id=\\\"pkg_AnnotationHub\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/AnnotationHub.html\\\">AnnotationHub</a></td>\\n\\t<td class=\\\" \\\">Bioconductor Package Maintainer</td>\\n\\t<td class=\\\" \\\">Client to access AnnotationHub resources</td>\\n\\t<td class=\\\"  sorting_1\\\">2125</td>\\n</tr><tr id=\\\"pkg_impute\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/impute.html\\\">impute</a></td>\\n\\t<td class=\\\" \\\">Balasubramanian Narasimhan</td>\\n\\t<td class=\\\" \\\">impute: Imputation for microarray data</td>\\n\\t<td class=\\\"  sorting_1\\\">2126</td>\\n</tr><tr id=\\\"pkg_GEOquery\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/GEOquery.html\\\">GEOquery</a></td>\\n\\t<td class=\\\" \\\">Sean Davis</td>\\n\\t<td class=\\\" \\\">Get data from NCBI Gene Expression Omnibus (GEO)</td>\\n\\t<td class=\\\"  sorting_1\\\">2127</td>\\n</tr><tr id=\\\"pkg_RBGL\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/RBGL.html\\\">RBGL</a></td>\\n\\t<td class=\\\" \\\">Bioconductor Package Maintainer</td>\\n\\t<td class=\\\" \\\">An interface to the BOOST graph library</td>\\n\\t<td class=\\\"  sorting_1\\\">2128</td>\\n</tr><tr id=\\\"pkg_Rgraphviz\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/Rgraphviz.html\\\">Rgraphviz</a></td>\\n\\t<td class=\\\" \\\">Kasper Daniel Hansen</td>\\n\\t<td class=\\\" \\\">Provides plotting capabilities for R graph objects</td>\\n\\t<td class=\\\"  sorting_1\\\">2129</td>\\n</tr><tr id=\\\"pkg_ProtGenerics\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ProtGenerics.html\\\">ProtGenerics</a></td>\\n\\t<td class=\\\" \\\">Laurent Gatto</td>\\n\\t<td class=\\\" \\\">Generic infrastructure for Bioconductor mass spectrometry packages</td>\\n\\t<td class=\\\"  sorting_1\\\">2130</td>\\n</tr><tr id=\\\"pkg_ScaledMatrix\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ScaledMatrix.html\\\">ScaledMatrix</a></td>\\n\\t<td class=\\\" \\\">Aaron Lun</td>\\n\\t<td class=\\\" \\\">Creating a DelayedMatrix of Scaled and Centered Values</td>\\n\\t<td class=\\\"  sorting_1\\\">2131</td>\\n</tr><tr id=\\\"pkg_BiocSingular\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/BiocSingular.html\\\">BiocSingular</a></td>\\n\\t<td class=\\\" \\\">Aaron Lun</td>\\n\\t<td class=\\\" \\\">Singular Value Decomposition for Bioconductor Packages</td>\\n\\t<td class=\\\"  sorting_1\\\">2132</td>\\n</tr><tr id=\\\"pkg_multtest\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/multtest.html\\\">multtest</a></td>\\n\\t<td class=\\\" \\\">Katherine S. Pollard</td>\\n\\t<td class=\\\" \\\">Resampling-based multiple hypothesis testing</td>\\n\\t<td class=\\\"  sorting_1\\\">2133</td>\\n</tr><tr id=\\\"pkg_HDF5Array\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/HDF5Array.html\\\">HDF5Array</a></td>\\n\\t<td class=\\\" \\\">Herv Pags</td>\\n\\t<td class=\\\" \\\">HDF5 backend for DelayedArray objects</td>\\n\\t<td class=\\\"  sorting_1\\\">2134</td>\\n</tr><tr id=\\\"pkg_BSgenome\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/BSgenome.html\\\">BSgenome</a></td>\\n\\t<td class=\\\" \\\">Herv Pags</td>\\n\\t<td class=\\\" \\\">Software infrastructure for efficient representation of full genomes and their SNPs</td>\\n\\t<td class=\\\"  sorting_1\\\">2135</td>\\n</tr><tr id=\\\"pkg_beachmat\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/beachmat.html\\\">beachmat</a></td>\\n\\t<td class=\\\" \\\">Aaron Lun</td>\\n\\t<td class=\\\" \\\">Compiling Bioconductor to Handle Each Matrix Type</td>\\n\\t<td class=\\\"  sorting_1\\\">2136</td>\\n</tr><tr id=\\\"pkg_ComplexHeatmap\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ComplexHeatmap.html\\\">ComplexHeatmap</a></td>\\n\\t<td class=\\\" \\\">Zuguang Gu</td>\\n\\t<td class=\\\" \\\">Make Complex Heatmaps</td>\\n\\t<td class=\\\"  sorting_1\\\">2137</td>\\n</tr><tr id=\\\"pkg_SingleCellExperiment\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/SingleCellExperiment.html\\\">SingleCellExperiment</a></td>\\n\\t<td class=\\\" \\\">Davide Risso</td>\\n\\t<td class=\\\" \\\">S4 Classes for Single Cell Data</td>\\n\\t<td class=\\\"  sorting_1\\\">2138</td>\\n</tr><tr id=\\\"pkg_sparseMatrixStats\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/sparseMatrixStats.html\\\">sparseMatrixStats</a></td>\\n\\t<td class=\\\" \\\">Constantin Ahlmann-Eltze</td>\\n\\t<td class=\\\" \\\">Summary Statistics for Rows and Columns of Sparse Matrices</td>\\n\\t<td class=\\\"  sorting_1\\\">2139</td>\\n</tr><tr id=\\\"pkg_GOSemSim\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/GOSemSim.html\\\">GOSemSim</a></td>\\n\\t<td class=\\\" \\\">Guangchuang Yu</td>\\n\\t<td class=\\\" \\\">GO-terms Semantic Similarity Measures</td>\\n\\t<td class=\\\"  sorting_1\\\">2140</td>\\n</tr><tr id=\\\"pkg_clusterProfiler\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/clusterProfiler.html\\\">clusterProfiler</a></td>\\n\\t<td class=\\\" \\\">Guangchuang Yu</td>\\n\\t<td class=\\\" \\\">A universal enrichment tool for interpreting omics data</td>\\n\\t<td class=\\\"  sorting_1\\\">2141</td>\\n</tr><tr id=\\\"pkg_DOSE\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/DOSE.html\\\">DOSE</a></td>\\n\\t<td class=\\\" \\\">Guangchuang Yu</td>\\n\\t<td class=\\\" \\\">Disease Ontology Semantic and Enrichment analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">2142</td>\\n</tr><tr id=\\\"pkg_DelayedMatrixStats\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/DelayedMatrixStats.html\\\">DelayedMatrixStats</a></td>\\n\\t<td class=\\\" \\\">Peter Hickey</td>\\n\\t<td class=\\\" \\\">Functions that Apply to Rows and Columns of 'DelayedMatrix' Objects</td>\\n\\t<td class=\\\"  sorting_1\\\">2143</td>\\n</tr><tr id=\\\"pkg_fgsea\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/fgsea.html\\\">fgsea</a></td>\\n\\t<td class=\\\" \\\">Alexey Sergushichev</td>\\n\\t<td class=\\\" \\\">Fast Gene Set Enrichment Analysis</td>\\n\\t<td class=\\\"  sorting_1\\\">2144</td>\\n</tr><tr id=\\\"pkg_enrichplot\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/enrichplot.html\\\">enrichplot</a></td>\\n\\t<td class=\\\" \\\">Guangchuang Yu</td>\\n\\t<td class=\\\" \\\">Visualization of Functional Enrichment Result</td>\\n\\t<td class=\\\"  sorting_1\\\">2145</td>\\n</tr><tr id=\\\"pkg_preprocessCore\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/preprocessCore.html\\\">preprocessCore</a></td>\\n\\t<td class=\\\" \\\">Ben Bolstad</td>\\n\\t<td class=\\\" \\\">A collection of pre-processing functions</td>\\n\\t<td class=\\\"  sorting_1\\\">2146</td>\\n</tr><tr id=\\\"pkg_qvalue\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/qvalue.html\\\">qvalue</a></td>\\n\\t<td class=\\\" \\\">John D. Storey, Andrew J. Bass</td>\\n\\t<td class=\\\" \\\">Q-value estimation for false discovery rate control</td>\\n\\t<td class=\\\"  sorting_1\\\">2147</td>\\n</tr><tr id=\\\"pkg_BiocIO\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/BiocIO.html\\\">BiocIO</a></td>\\n\\t<td class=\\\" \\\">Bioconductor Package Maintainer</td>\\n\\t<td class=\\\" \\\">Standard Input and Output for Bioconductor Packages</td>\\n\\t<td class=\\\"  sorting_1\\\">2148</td>\\n</tr><tr id=\\\"pkg_treeio\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/treeio.html\\\">treeio</a></td>\\n\\t<td class=\\\" \\\">Guangchuang Yu</td>\\n\\t<td class=\\\" \\\">Base Classes and Functions for Phylogenetic Tree Input and Output</td>\\n\\t<td class=\\\"  sorting_1\\\">2149</td>\\n</tr><tr id=\\\"pkg_rhdf5filters\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/rhdf5filters.html\\\">rhdf5filters</a></td>\\n\\t<td class=\\\" \\\">Mike Smith</td>\\n\\t<td class=\\\" \\\">HDF5 Compression Filters</td>\\n\\t<td class=\\\"  sorting_1\\\">2150</td>\\n</tr><tr id=\\\"pkg_ggtree\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/ggtree.html\\\">ggtree</a></td>\\n\\t<td class=\\\" \\\">Guangchuang Yu</td>\\n\\t<td class=\\\" \\\">an R package for visualization of tree and annotation data</td>\\n\\t<td class=\\\"  sorting_1\\\">2151</td>\\n</tr><tr id=\\\"pkg_GenomicFeatures\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/GenomicFeatures.html\\\">GenomicFeatures</a></td>\\n\\t<td class=\\\" \\\">H. Pags</td>\\n\\t<td class=\\\" \\\">Conveniently import and query gene models</td>\\n\\t<td class=\\\"  sorting_1\\\">2152</td>\\n</tr><tr id=\\\"pkg_rhdf5\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/rhdf5.html\\\">rhdf5</a></td>\\n\\t<td class=\\\" \\\">Mike Smith</td>\\n\\t<td class=\\\" \\\">R Interface to HDF5</td>\\n\\t<td class=\\\"  sorting_1\\\">2153</td>\\n</tr><tr id=\\\"pkg_geneplotter\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/geneplotter.html\\\">geneplotter</a></td>\\n\\t<td class=\\\" \\\">Bioconductor Package Maintainer</td>\\n\\t<td class=\\\" \\\">Graphics related functions for Bioconductor</td>\\n\\t<td class=\\\"  sorting_1\\\">2154</td>\\n</tr><tr id=\\\"pkg_DESeq2\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/DESeq2.html\\\">DESeq2</a></td>\\n\\t<td class=\\\" \\\">Michael Love</td>\\n\\t<td class=\\\" \\\">Differential gene expression analysis based on the negative binomial distribution</td>\\n\\t<td class=\\\"  sorting_1\\\">2155</td>\\n</tr><tr id=\\\"pkg_graph\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/graph.html\\\">graph</a></td>\\n\\t<td class=\\\" \\\">Bioconductor Package Maintainer</td>\\n\\t<td class=\\\" \\\">graph: A package to handle graph data structures</td>\\n\\t<td class=\\\"  sorting_1\\\">2156</td>\\n</tr><tr id=\\\"pkg_rtracklayer\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/rtracklayer.html\\\">rtracklayer</a></td>\\n\\t<td class=\\\" \\\">Michael Lawrence</td>\\n\\t<td class=\\\" \\\">R interface to genome annotation files and the UCSC genome browser</td>\\n\\t<td class=\\\"  sorting_1\\\">2157</td>\\n</tr><tr id=\\\"pkg_Rhdf5lib\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/Rhdf5lib.html\\\">Rhdf5lib</a></td>\\n\\t<td class=\\\" \\\">Mike Smith</td>\\n\\t<td class=\\\" \\\">hdf5 library as an R package</td>\\n\\t<td class=\\\"  sorting_1\\\">2158</td>\\n</tr><tr id=\\\"pkg_biomaRt\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/biomaRt.html\\\">biomaRt</a></td>\\n\\t<td class=\\\" \\\">Mike Smith</td>\\n\\t<td class=\\\" \\\">Interface to BioMart databases (i.e. Ensembl)</td>\\n\\t<td class=\\\"  sorting_1\\\">2159</td>\\n</tr><tr id=\\\"pkg_genefilter\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/genefilter.html\\\">genefilter</a></td>\\n\\t<td class=\\\" \\\">Bioconductor Package Maintainer</td>\\n\\t<td class=\\\" \\\">genefilter: methods for filtering genes from high-throughput experiments</td>\\n\\t<td class=\\\"  sorting_1\\\">2160</td>\\n</tr><tr id=\\\"pkg_GenomicAlignments\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/GenomicAlignments.html\\\">GenomicAlignments</a></td>\\n\\t<td class=\\\" \\\">Herv Pags</td>\\n\\t<td class=\\\" \\\">Representation and manipulation of short genomic alignments</td>\\n\\t<td class=\\\"  sorting_1\\\">2161</td>\\n</tr><tr id=\\\"pkg_edgeR\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/edgeR.html\\\">edgeR</a></td>\\n\\t<td class=\\\" \\\">Yunshun Chen, Gordon Smyth, Aaron Lun, Mark Robinson</td>\\n\\t<td class=\\\" \\\">Empirical Analysis of Digital Gene Expression Data in R</td>\\n\\t<td class=\\\"  sorting_1\\\">2162</td>\\n</tr><tr id=\\\"pkg_Rsamtools\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/Rsamtools.html\\\">Rsamtools</a></td>\\n\\t<td class=\\\" \\\">Bioconductor Package Maintainer</td>\\n\\t<td class=\\\" \\\">Binary alignment (BAM), FASTA, variant call (BCF), and tabix file import</td>\\n\\t<td class=\\\"  sorting_1\\\">2163</td>\\n</tr><tr id=\\\"pkg_Rhtslib\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/Rhtslib.html\\\">Rhtslib</a></td>\\n\\t<td class=\\\" \\\">Herv Pags</td>\\n\\t<td class=\\\" \\\">HTSlib high-throughput sequencing library as an R package</td>\\n\\t<td class=\\\"  sorting_1\\\">2164</td>\\n</tr><tr id=\\\"pkg_BiocFileCache\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/BiocFileCache.html\\\">BiocFileCache</a></td>\\n\\t<td class=\\\" \\\">Lori Shepherd</td>\\n\\t<td class=\\\" \\\">Manage Files Across Sessions</td>\\n\\t<td class=\\\"  sorting_1\\\">2165</td>\\n</tr><tr id=\\\"pkg_annotate\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/annotate.html\\\">annotate</a></td>\\n\\t<td class=\\\" \\\">Bioconductor Package Maintainer</td>\\n\\t<td class=\\\" \\\">Annotation for microarrays</td>\\n\\t<td class=\\\"  sorting_1\\\">2166</td>\\n</tr><tr id=\\\"pkg_KEGGREST\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/KEGGREST.html\\\">KEGGREST</a></td>\\n\\t<td class=\\\" \\\">Bioconductor Package Maintainer</td>\\n\\t<td class=\\\" \\\">Client-side REST access to the Kyoto Encyclopedia of Genes and Genomes (KEGG)</td>\\n\\t<td class=\\\"  sorting_1\\\">2167</td>\\n</tr><tr id=\\\"pkg_AnnotationDbi\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/AnnotationDbi.html\\\">AnnotationDbi</a></td>\\n\\t<td class=\\\" \\\">Bioconductor Package Maintainer</td>\\n\\t<td class=\\\" \\\">Manipulation of SQLite-based annotations in Bioconductor</td>\\n\\t<td class=\\\"  sorting_1\\\">2168</td>\\n</tr><tr id=\\\"pkg_limma\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/limma.html\\\">limma</a></td>\\n\\t<td class=\\\" \\\">Gordon Smyth</td>\\n\\t<td class=\\\" \\\">Linear Models for Microarray Data</td>\\n\\t<td class=\\\"  sorting_1\\\">2169</td>\\n</tr><tr id=\\\"pkg_MatrixGenerics\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/MatrixGenerics.html\\\">MatrixGenerics</a></td>\\n\\t<td class=\\\" \\\">Peter Hickey</td>\\n\\t<td class=\\\" \\\">S4 Generic Summary Statistic Functions that Operate on Matrix-Like Objects</td>\\n\\t<td class=\\\"  sorting_1\\\">2170</td>\\n</tr><tr id=\\\"pkg_SummarizedExperiment\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/SummarizedExperiment.html\\\">SummarizedExperiment</a></td>\\n\\t<td class=\\\" \\\">Herv Pags</td>\\n\\t<td class=\\\" \\\">SummarizedExperiment container</td>\\n\\t<td class=\\\"  sorting_1\\\">2171</td>\\n</tr><tr id=\\\"pkg_DelayedArray\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/DelayedArray.html\\\">DelayedArray</a></td>\\n\\t<td class=\\\" \\\">Herv Pags</td>\\n\\t<td class=\\\" \\\">A unified framework for working transparently with on-disk and in-memory array-like datasets</td>\\n\\t<td class=\\\"  sorting_1\\\">2172</td>\\n</tr><tr id=\\\"pkg_GenomicRanges\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/GenomicRanges.html\\\">GenomicRanges</a></td>\\n\\t<td class=\\\" \\\">Herv Pags</td>\\n\\t<td class=\\\" \\\">Representation and manipulation of genomic intervals</td>\\n\\t<td class=\\\"  sorting_1\\\">2173</td>\\n</tr><tr id=\\\"pkg_BiocParallel\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/BiocParallel.html\\\">BiocParallel</a></td>\\n\\t<td class=\\\" \\\">Martin Morgan</td>\\n\\t<td class=\\\" \\\">Bioconductor facilities for parallel evaluation</td>\\n\\t<td class=\\\"  sorting_1\\\">2174</td>\\n</tr><tr id=\\\"pkg_Biostrings\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/Biostrings.html\\\">Biostrings</a></td>\\n\\t<td class=\\\" \\\">H. Pags</td>\\n\\t<td class=\\\" \\\">Efficient manipulation of biological strings</td>\\n\\t<td class=\\\"  sorting_1\\\">2175</td>\\n</tr><tr id=\\\"pkg_XVector\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/XVector.html\\\">XVector</a></td>\\n\\t<td class=\\\" \\\">Herv Pags</td>\\n\\t<td class=\\\" \\\">Foundation of external vector representation and manipulation in Bioconductor</td>\\n\\t<td class=\\\"  sorting_1\\\">2176</td>\\n</tr><tr id=\\\"pkg_zlibbioc\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/zlibbioc.html\\\">zlibbioc</a></td>\\n\\t<td class=\\\" \\\">Bioconductor Package Maintainer</td>\\n\\t<td class=\\\" \\\">An R packaged zlib-1.2.5</td>\\n\\t<td class=\\\"  sorting_1\\\">2177</td>\\n</tr><tr id=\\\"pkg_Biobase\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/Biobase.html\\\">Biobase</a></td>\\n\\t<td class=\\\" \\\">Bioconductor Package Maintainer</td>\\n\\t<td class=\\\" \\\">Biobase: Base functions for Bioconductor</td>\\n\\t<td class=\\\"  sorting_1\\\">2178</td>\\n</tr><tr id=\\\"pkg_IRanges\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/IRanges.html\\\">IRanges</a></td>\\n\\t<td class=\\\" \\\">Herv Pags</td>\\n\\t<td class=\\\" \\\">Foundation of integer range manipulation in Bioconductor</td>\\n\\t<td class=\\\"  sorting_1\\\">2179</td>\\n</tr><tr id=\\\"pkg_GenomeInfoDb\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/GenomeInfoDb.html\\\">GenomeInfoDb</a></td>\\n\\t<td class=\\\" \\\">Herv Pags</td>\\n\\t<td class=\\\" \\\">Utilities for manipulating chromosome names, including modifying them to follow a particular naming style</td>\\n\\t<td class=\\\"  sorting_1\\\">2180</td>\\n</tr><tr id=\\\"pkg_BiocVersion\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/BiocVersion.html\\\">BiocVersion</a></td>\\n\\t<td class=\\\" \\\">Bioconductor Package Maintainer</td>\\n\\t<td class=\\\" \\\">Set the appropriate version of Bioconductor packages</td>\\n\\t<td class=\\\"  sorting_1\\\">2181</td>\\n</tr><tr id=\\\"pkg_S4Vectors\\\" class=\\\"row_even\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/S4Vectors.html\\\">S4Vectors</a></td>\\n\\t<td class=\\\" \\\">Herv Pags</td>\\n\\t<td class=\\\" \\\">Foundation of vector-like and list-like containers in Bioconductor</td>\\n\\t<td class=\\\"  sorting_1\\\">2182</td>\\n</tr><tr id=\\\"pkg_BiocGenerics\\\" class=\\\"row_odd\\\">\\n\\t<td class=\\\" \\\"><a href=\\\"https://www.bioconductor.org/packages/release/bioc/html/BiocGenerics.html\\\">BiocGenerics</a></td>\\n\\t<td class=\\\" \\\">Herv Pags</td>\\n\\t<td class=\\\" \\\">S4 generic functions used in Bioconductor</td>\\n\\t<td class=\\\"  sorting_1\\\">2183</td>\\n</tr></tbody>\"} | headers=HTTPHeaderDict({'content-type': 'application/json; charset=utf-8', 'cache-control': 'no-cache', 'content-length': '724764', 'date': 'Mon, 20 Mar 2023 20:11:47 GMT'}) (remote_connection.py:319)\n",
      "2023-03-20 21:11:47 [   DEBUG] Finished Request (remote_connection.py:346)\n",
      "2023-03-20 21:11:47 [   DEBUG] DELETE http://localhost:46743/session/a5e86af6-7734-4cab-a830-816a0e7a134b/window {} (remote_connection.py:303)\n",
      "2023-03-20 21:11:48 [   DEBUG] http://localhost:46743 \"DELETE /session/a5e86af6-7734-4cab-a830-816a0e7a134b/window HTTP/1.1\" 200 12 (connectionpool.py:452)\n",
      "2023-03-20 21:11:48 [   DEBUG] Remote response: status=200 | data={\"value\":[]} | headers=HTTPHeaderDict({'content-type': 'application/json; charset=utf-8', 'cache-control': 'no-cache', 'content-length': '12', 'date': 'Mon, 20 Mar 2023 20:11:47 GMT'}) (remote_connection.py:319)\n",
      "2023-03-20 21:11:48 [   DEBUG] Finished Request (remote_connection.py:346)\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "['ABSSeq',\n",
       " 'ABarray',\n",
       " 'ACE',\n",
       " 'ACME',\n",
       " 'ADAM',\n",
       " 'ADAMgui',\n",
       " 'ADImpute',\n",
       " 'ADaCGH2',\n",
       " 'AGDEX',\n",
       " 'AIMS']"
      ]
     },
     "execution_count": 7,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "package_list = bioc_scraper_ds.obtain_package_names()\n",
    "package_list[:10]"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "We can obtain the data from a specific package, for example the **`DeepBlueR`** package"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "2023-03-20 21:11:02 [   DEBUG] Scraping package DeepBlueR (logger.py:140)\n",
      "2023-03-20 21:11:02 [   DEBUG] Getting next proxy (logger.py:140)\n",
      "2023-03-20 21:11:02 [   DEBUG] Proxy list rotated, using 213.6.68.94:5678, next will be 54.39.118.40:7497 (logger.py:140)\n",
      "2023-03-20 21:11:02 [   DEBUG] Using proxy: {'http': 'http://213.6.68.94:5678'} (logger.py:140)\n",
      "2023-03-20 21:11:02 [   DEBUG] Getting next useragent (logger.py:140)\n",
      "2023-03-20 21:11:02 [   DEBUG] Using user agent: Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:37.0) Gecko/20100101 Firefox/37.0 (logger.py:140)\n",
      "2023-03-20 21:11:02 [   DEBUG] Starting new HTTPS connection (1): www.bioconductor.org:443 (connectionpool.py:973)\n",
      "2023-03-20 21:11:02 [   DEBUG] https://www.bioconductor.org:443 \"GET /packages/release/bioc/html/DeepBlueR.html HTTP/1.1\" 200 5671 (connectionpool.py:452)\n",
      "2023-03-20 21:11:02 [   DEBUG] Response status code: 200 (logger.py:140)\n",
      "2023-03-20 21:11:02 [   DEBUG] Package DeepBlueR scraped successfully (logger.py:140)\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "{'name': 'DeepBlueR',\n",
       " 'version': '1.24.1',\n",
       " 'dependencies': [{'name': 'R', 'version': '>= 3.3'},\n",
       "  {'name': 'XML', 'version': ''},\n",
       "  {'name': 'RCurl', 'version': ''},\n",
       "  {'name': 'GenomicRanges', 'version': ''},\n",
       "  {'name': 'data.table', 'version': ''},\n",
       "  {'name': 'stringr', 'version': ''},\n",
       "  {'name': 'diffr', 'version': ''},\n",
       "  {'name': 'dplyr', 'version': ''},\n",
       "  {'name': 'methods', 'version': ''},\n",
       "  {'name': 'rjson', 'version': ''},\n",
       "  {'name': 'utils', 'version': ''},\n",
       "  {'name': 'R.utils', 'version': ''},\n",
       "  {'name': 'foreach', 'version': ''},\n",
       "  {'name': 'withr', 'version': ''},\n",
       "  {'name': 'rtracklayer', 'version': ''},\n",
       "  {'name': 'GenomeInfoDb', 'version': ''},\n",
       "  {'name': 'settings', 'version': ''},\n",
       "  {'name': 'filehash', 'version': ''}],\n",
       " 'url': 'https://www.bioconductor.org/packages/release/bioc/html/DeepBlueR.html'}"
      ]
     },
     "execution_count": 3,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "deepbluer = bioc_scraper_ds.obtain_package_data(\"DeepBlueR\")\n",
    "deepbluer"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Be careful with the sensitivity to **caps**, if the package has not been found, an **ScraperError** exception is raised"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "2023-03-20 21:11:22 [   DEBUG] Scraping package deepbluer (logger.py:140)\n",
      "2023-03-20 21:11:22 [   DEBUG] Getting next proxy (logger.py:140)\n",
      "2023-03-20 21:11:22 [   DEBUG] Proxy list rotated, using 54.39.118.40:7497, next will be 46.105.35.193:8080 (logger.py:140)\n",
      "2023-03-20 21:11:22 [   DEBUG] Using proxy: {'http': 'http://54.39.118.40:7497'} (logger.py:140)\n",
      "2023-03-20 21:11:22 [   DEBUG] Getting next useragent (logger.py:140)\n",
      "2023-03-20 21:11:22 [   DEBUG] Using user agent: Mozilla/5.0 (Windows NT 6.1; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/47.0.2503.0 Safari/537.36 (logger.py:140)\n",
      "2023-03-20 21:11:22 [   DEBUG] Starting new HTTPS connection (1): www.bioconductor.org:443 (connectionpool.py:973)\n",
      "2023-03-20 21:11:22 [   DEBUG] https://www.bioconductor.org:443 \"GET /packages/release/bioc/html/deepbluer.html HTTP/1.1\" 404 6873 (connectionpool.py:452)\n",
      "2023-03-20 21:11:22 [   DEBUG] Response status code: 404 (logger.py:140)\n",
      "2023-03-20 21:11:22 [   DEBUG] ScraperError: Package deepbluer not found (logger.py:140)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "ScraperError: Package deepbluer not found\n"
     ]
    }
   ],
   "source": [
    "try:\n",
    "    deepbluer2 = bioc_scraper_ds.obtain_package_data(\"deepbluer\")\n",
    "except Exception as e:\n",
    "    print(e)"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "We can get the data from a list of package names using the function:\n",
    "-   ```python\n",
    "    obtain_packages_data(list[str])\n",
    "    ```"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "2023-03-20 21:11:55 [   DEBUG] Using package names from param list (logger.py:140)\n",
      "2023-03-20 21:11:55 [   DEBUG] Building urls (logger.py:140)\n",
      "2023-03-20 21:11:55 [   DEBUG] Getting next proxy (logger.py:140)\n",
      "2023-03-20 21:11:55 [   DEBUG] Proxy list rotated, using 46.105.35.193:8080, next will be 103.85.67.114:4145 (logger.py:140)\n",
      "2023-03-20 21:11:55 [   DEBUG] Using proxy: {'http': 'http://46.105.35.193:8080'} (logger.py:140)\n",
      "2023-03-20 21:11:55 [   DEBUG] Getting next useragent (logger.py:140)\n",
      "2023-03-20 21:11:55 [   DEBUG] Using user agent: Mozilla/5.0 (X11; CrOS x86_64 7077.111.0) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/44.0.2403.125 Safari/537.36 (logger.py:140)\n",
      "2023-03-20 21:11:55 [   DEBUG] Getting next proxy (logger.py:140)\n",
      "2023-03-20 21:11:55 [   DEBUG] Starting new HTTPS connection (1): www.bioconductor.org:443 (connectionpool.py:973)\n",
      "2023-03-20 21:11:55 [   DEBUG] Proxy list rotated, using 103.85.67.114:4145, next will be 181.129.52.155:44665 (logger.py:140)\n",
      "2023-03-20 21:11:55 [   DEBUG] Using proxy: {'http': 'http://103.85.67.114:4145'} (logger.py:140)\n",
      "2023-03-20 21:11:55 [   DEBUG] Getting next useragent (logger.py:140)\n",
      "2023-03-20 21:11:55 [   DEBUG] Using user agent: Mozilla/5.0 (Android; Mobile; rv:40.0) Gecko/40.0 Firefox/40.0 (logger.py:140)\n",
      "2023-03-20 21:11:55 [   DEBUG] Getting next proxy (logger.py:140)\n",
      "2023-03-20 21:11:55 [   DEBUG] Starting new HTTPS connection (1): www.bioconductor.org:443 (connectionpool.py:973)\n",
      "2023-03-20 21:11:55 [   DEBUG] Proxy list rotated, using 181.129.52.155:44665, next will be 125.141.151.83:80 (logger.py:140)\n",
      "2023-03-20 21:11:55 [   DEBUG] Using proxy: {'http': 'http://181.129.52.155:44665'} (logger.py:140)\n",
      "2023-03-20 21:11:55 [   DEBUG] Getting next useragent (logger.py:140)\n",
      "2023-03-20 21:11:55 [   DEBUG] Using user agent: Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Ubuntu Chromium/44.0.2403.89 Chrome/44.0.2403.89 Safari/537.36 (logger.py:140)\n",
      "2023-03-20 21:11:55 [   DEBUG] Starting new HTTPS connection (1): www.bioconductor.org:443 (connectionpool.py:973)\n",
      "2023-03-20 21:11:55 [   DEBUG] https://www.bioconductor.org:443 \"GET /packages/release/bioc/html/ACE.html HTTP/1.1\" 200 5347 (connectionpool.py:452)\n",
      "2023-03-20 21:11:55 [   DEBUG] Response status code: 200 (logger.py:140)\n",
      "2023-03-20 21:11:55 [   DEBUG] https://www.bioconductor.org:443 \"GET /packages/release/bioc/html/ABarray.html HTTP/1.1\" 200 5349 (connectionpool.py:452)\n",
      "2023-03-20 21:11:55 [   DEBUG] Response status code: 200 (logger.py:140)\n",
      "2023-03-20 21:11:55 [   DEBUG] https://www.bioconductor.org:443 \"GET /packages/release/bioc/html/ABSSeq.html HTTP/1.1\" 200 5275 (connectionpool.py:452)\n",
      "2023-03-20 21:11:55 [   DEBUG] Response status code: 200 (logger.py:140)\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "[{'name': 'ABSSeq',\n",
       "  'version': '1.52.0',\n",
       "  'dependencies': [{'name': 'R', 'version': '>= 2.10'},\n",
       "   {'name': 'methods', 'version': ''},\n",
       "   {'name': 'locfit', 'version': ''},\n",
       "   {'name': 'limma', 'version': ''}],\n",
       "  'url': 'https://www.bioconductor.org/packages/release/bioc/html/ABSSeq.html'},\n",
       " {'name': 'ABarray',\n",
       "  'version': '1.66.0',\n",
       "  'dependencies': [{'name': 'Biobase', 'version': ''},\n",
       "   {'name': 'graphics', 'version': ''},\n",
       "   {'name': 'grDevices', 'version': ''},\n",
       "   {'name': 'methods', 'version': ''},\n",
       "   {'name': 'multtest', 'version': ''},\n",
       "   {'name': 'stats', 'version': ''},\n",
       "   {'name': 'tcltk', 'version': ''},\n",
       "   {'name': 'utils', 'version': ''}],\n",
       "  'url': 'https://www.bioconductor.org/packages/release/bioc/html/ABarray.html'},\n",
       " {'name': 'ACE',\n",
       "  'version': '1.16.0',\n",
       "  'dependencies': [{'name': 'R', 'version': '>= 3.4'},\n",
       "   {'name': 'Biobase', 'version': ''},\n",
       "   {'name': 'QDNAseq', 'version': ''},\n",
       "   {'name': 'ggplot2', 'version': ''},\n",
       "   {'name': 'grid', 'version': ''},\n",
       "   {'name': 'stats', 'version': ''},\n",
       "   {'name': 'utils', 'version': ''},\n",
       "   {'name': 'methods', 'version': ''},\n",
       "   {'name': 'grDevices', 'version': ''},\n",
       "   {'name': 'GenomicRanges', 'version': ''}],\n",
       "  'url': 'https://www.bioconductor.org/packages/release/bioc/html/ACE.html'}]"
      ]
     },
     "execution_count": 8,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "pkgs_data, not_found = bioc_scraper_ds.obtain_packages_data(package_list[:3])\n",
    "pkgs_data"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Package: ABSSeq (1.52.0)\n",
      "-   Dependency: R >= 2.10\n",
      "-   Dependency: methods \n",
      "-   Dependency: locfit \n",
      "-   Dependency: limma \n",
      "Package: ABarray (1.66.0)\n",
      "-   Dependency: Biobase \n",
      "-   Dependency: graphics \n",
      "-   Dependency: grDevices \n",
      "-   Dependency: methods \n",
      "-   Dependency: multtest \n",
      "-   Dependency: stats \n",
      "-   Dependency: tcltk \n",
      "-   Dependency: utils \n",
      "Package: ACE (1.16.0)\n",
      "-   Dependency: R >= 3.4\n",
      "-   Dependency: Biobase \n",
      "-   Dependency: QDNAseq \n",
      "-   Dependency: ggplot2 \n",
      "-   Dependency: grid \n",
      "-   Dependency: stats \n",
      "-   Dependency: utils \n",
      "-   Dependency: methods \n",
      "-   Dependency: grDevices \n",
      "-   Dependency: GenomicRanges \n"
     ]
    }
   ],
   "source": [
    "for p in pkgs_data:\n",
    "    print(f'Package: {p[\"name\"]} ({p[\"version\"]})')\n",
    "\n",
    "    for d in p[\"dependencies\"]:\n",
    "        print(f'-   Dependency: {d[\"name\"]} {d[\"version\"]}')"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Packages not found appear as the second object of the tuple"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "2023-03-20 21:12:11 [   DEBUG] Using package names from param list (logger.py:140)\n",
      "2023-03-20 21:12:11 [   DEBUG] Building urls (logger.py:140)\n",
      "2023-03-20 21:12:11 [   DEBUG] Getting next proxy (logger.py:140)\n",
      "2023-03-20 21:12:11 [   DEBUG] Proxy list rotated, using 125.141.151.83:80, next will be 216.105.130.33:39593 (logger.py:140)\n",
      "2023-03-20 21:12:11 [   DEBUG] Using proxy: {'http': 'http://125.141.151.83:80'} (logger.py:140)\n",
      "2023-03-20 21:12:11 [   DEBUG] Getting next useragent (logger.py:140)\n",
      "2023-03-20 21:12:11 [   DEBUG] Using user agent: Mozilla/5.0 (Windows NT 6.1) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/36.0.1985.125 Safari/537.36 (logger.py:140)\n",
      "2023-03-20 21:12:11 [   DEBUG] Getting next proxy (logger.py:140)\n",
      "2023-03-20 21:12:11 [   DEBUG] Starting new HTTPS connection (1): www.bioconductor.org:443 (connectionpool.py:973)\n",
      "2023-03-20 21:12:11 [   DEBUG] Proxy list rotated, using 216.105.130.33:39593, next will be 203.210.84.198:1080 (logger.py:140)\n",
      "2023-03-20 21:12:11 [   DEBUG] Using proxy: {'http': 'http://216.105.130.33:39593'} (logger.py:140)\n",
      "2023-03-20 21:12:11 [   DEBUG] Getting next useragent (logger.py:140)\n",
      "2023-03-20 21:12:11 [   DEBUG] Using user agent: Mozilla/5.0 (iPad; CPU OS 7_0 like Mac OS X) AppleWebKit/537.51.1 (KHTML, like Gecko) Version/7.0 Mobile/11A465 Safari/9537.53 (logger.py:140)\n",
      "2023-03-20 21:12:11 [   DEBUG] Starting new HTTPS connection (1): www.bioconductor.org:443 (connectionpool.py:973)\n",
      "2023-03-20 21:12:11 [   DEBUG] https://www.bioconductor.org:443 \"GET /packages/release/bioc/html/DeepBlueR.html HTTP/1.1\" 200 5671 (connectionpool.py:452)\n",
      "2023-03-20 21:12:11 [   DEBUG] Response status code: 200 (logger.py:140)\n",
      "2023-03-20 21:12:11 [   DEBUG] https://www.bioconductor.org:443 \"GET /packages/release/bioc/html/deepbluer.html HTTP/1.1\" 404 6873 (connectionpool.py:452)\n",
      "2023-03-20 21:12:11 [   DEBUG] Response status code: 404 (logger.py:140)\n",
      "2023-03-20 21:12:11 [   DEBUG] Package deepbluer not found, status code: 404, url: https://www.bioconductor.org/packages/release/bioc/html/deepbluer.html, skipping... (logger.py:140)\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "['deepbluer']"
      ]
     },
     "execution_count": 10,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "pkgs_data, not_found = bioc_scraper_ds.obtain_packages_data([\"deepbluer\", \"DeepBlueR\"])\n",
    "not_found"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Data obtaining from CSV files"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "from olivia_finder.data_source.csv_network import CSVNetwork"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Load the network\n",
    "bioc_csv_ds = CSVNetwork(\n",
    "    \"results/csv_datasets/bioconductor_adjlist_scraping.csv\",  # Path to the CSV file\n",
    "    \"Bioconductor\",                         # Name of the data source\n",
    "    \"Bioconductor as a CSV file\",            # Description of the data source\n",
    "    dependent_field=\"name\",                 # Name of the field that contains the dependencies\n",
    "    dependency_field=\"dependency\",          # Name of the field that contains the name of the package\n",
    "    dependent_version_field=\"version\",      # Name of the field that contains the version of the package\n",
    "    dependency_version_field=\"dependency_version\",     # Name of the field that contains the version of the dependency\n",
    "    dependent_url_field=\"url\",              # Name of the field that contains the URL of the package\n",
    ")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['ABSSeq',\n",
       " 'ABarray',\n",
       " 'ACE',\n",
       " 'ACME',\n",
       " 'ADAM',\n",
       " 'ADAMgui',\n",
       " 'ADImpute',\n",
       " 'ADaCGH2',\n",
       " 'AGDEX',\n",
       " 'AIMS']"
      ]
     },
     "execution_count": 5,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "package_list = bioc_csv_ds.obtain_package_names()\n",
    "package_list[:10]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'name': 'DeepBlueR',\n",
       " 'version': '1.24.1',\n",
       " 'url': 'https://www.bioconductor.org/packages/release/bioc/html/DeepBlueR.html',\n",
       " 'dependencies': [{'name': 'R', 'version': '>= 3.3'},\n",
       "  {'name': 'XML', 'version': nan},\n",
       "  {'name': 'RCurl', 'version': nan},\n",
       "  {'name': 'GenomicRanges', 'version': nan},\n",
       "  {'name': 'data.table', 'version': nan},\n",
       "  {'name': 'stringr', 'version': nan},\n",
       "  {'name': 'diffr', 'version': nan},\n",
       "  {'name': 'dplyr', 'version': nan},\n",
       "  {'name': 'methods', 'version': nan},\n",
       "  {'name': 'rjson', 'version': nan},\n",
       "  {'name': 'utils', 'version': nan},\n",
       "  {'name': 'R.utils', 'version': nan},\n",
       "  {'name': 'foreach', 'version': nan},\n",
       "  {'name': 'withr', 'version': nan},\n",
       "  {'name': 'rtracklayer', 'version': nan},\n",
       "  {'name': 'GenomeInfoDb', 'version': nan},\n",
       "  {'name': 'settings', 'version': nan},\n",
       "  {'name': 'filehash', 'version': nan}]}"
      ]
     },
     "execution_count": 6,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "deepbluer = bioc_csv_ds.obtain_package_data(\"DeepBlueR\")\n",
    "deepbluer"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[{'name': 'ABSSeq',\n",
       "  'version': '1.52.0',\n",
       "  'url': 'https://www.bioconductor.org/packages/release/bioc/html/ABSSeq.html',\n",
       "  'dependencies': [{'name': 'R', 'version': '>= 2.10'},\n",
       "   {'name': 'methods', 'version': nan},\n",
       "   {'name': 'locfit', 'version': nan},\n",
       "   {'name': 'limma', 'version': nan}]},\n",
       " {'name': 'ABarray',\n",
       "  'version': '1.66.0',\n",
       "  'url': 'https://www.bioconductor.org/packages/release/bioc/html/ABarray.html',\n",
       "  'dependencies': [{'name': 'Biobase', 'version': nan},\n",
       "   {'name': 'graphics', 'version': nan},\n",
       "   {'name': 'grDevices', 'version': nan},\n",
       "   {'name': 'methods', 'version': nan},\n",
       "   {'name': 'multtest', 'version': nan},\n",
       "   {'name': 'stats', 'version': nan},\n",
       "   {'name': 'tcltk', 'version': nan},\n",
       "   {'name': 'utils', 'version': nan}]},\n",
       " {'name': 'ACE',\n",
       "  'version': '1.16.0',\n",
       "  'url': 'https://www.bioconductor.org/packages/release/bioc/html/ACE.html',\n",
       "  'dependencies': [{'name': 'R', 'version': '>= 3.4'},\n",
       "   {'name': 'Biobase', 'version': nan},\n",
       "   {'name': 'QDNAseq', 'version': nan},\n",
       "   {'name': 'ggplot2', 'version': nan},\n",
       "   {'name': 'grid', 'version': nan},\n",
       "   {'name': 'stats', 'version': nan},\n",
       "   {'name': 'utils', 'version': nan},\n",
       "   {'name': 'methods', 'version': nan},\n",
       "   {'name': 'grDevices', 'version': nan},\n",
       "   {'name': 'GenomicRanges', 'version': nan}]}]"
      ]
     },
     "execution_count": 7,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "packages = bioc_csv_ds.obtain_packages_data(package_list[:3])\n",
    "packages"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "---"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 2 - Initialization of a package manager"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [],
   "source": [
    "from olivia_finder.package_manager import PackageManager"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "**Declare the class**\n",
    "\n",
    "Initialize the packagemanager class with the implementation of the scraper we want"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "2023-03-20 21:26:08 [   DEBUG] Added SSLProxies to proxy builders (logger.py:140)\n",
      "2023-03-20 21:26:08 [   DEBUG] Added FreeProxyList to proxy builders (logger.py:140)\n",
      "2023-03-20 21:26:08 [   DEBUG] Added GeonodeProxy to proxy builders (logger.py:140)\n",
      "2023-03-20 21:26:08 [   DEBUG] Starting new HTTPS connection (1): www.sslproxies.org:443 (connectionpool.py:973)\n",
      "2023-03-20 21:26:09 [   DEBUG] https://www.sslproxies.org:443 \"GET / HTTP/1.1\" 200 None (connectionpool.py:452)\n",
      "2023-03-20 21:26:09 [   DEBUG] Found 100 proxies from SSLProxies (logger.py:140)\n",
      "2023-03-20 21:26:09 [   DEBUG] Starting new HTTPS connection (1): free-proxy-list.net:443 (connectionpool.py:973)\n",
      "2023-03-20 21:26:09 [   DEBUG] https://free-proxy-list.net:443 \"GET /anonymous-proxy.html HTTP/1.1\" 200 None (connectionpool.py:452)\n",
      "2023-03-20 21:26:09 [   DEBUG] Found 100 proxies from FreeProxyList (logger.py:140)\n",
      "2023-03-20 21:26:09 [   DEBUG] Starting new HTTPS connection (1): proxylist.geonode.com:443 (connectionpool.py:973)\n",
      "2023-03-20 21:26:16 [   DEBUG] https://proxylist.geonode.com:443 \"GET /api/proxy-list?limit=500&page=1&sort_by=lastChecked&sort_type=desc HTTP/1.1\" 200 None (connectionpool.py:452)\n",
      "2023-03-20 21:26:16 [   DEBUG] Found 500 proxies from GeonodeProxy (logger.py:140)\n",
      "2023-03-20 21:26:16 [   DEBUG] Proxies len: 679 (logger.py:140)\n",
      "2023-03-20 21:26:16 [   DEBUG] Proxy Handler initialized with 679 proxies (logger.py:140)\n",
      "2023-03-20 21:26:16 [   DEBUG] Useragents loaded from file: /home/dnllns/Documentos/repositorios/olivia-finder/olivia_finder/olivia_finder/myrequests/data/useragents.txt (logger.py:140)\n"
     ]
    }
   ],
   "source": [
    "from olivia_finder.data_source.scrapers.pypi import PypiScraper\n",
    "pypi_scraper_pm = PackageManager(PypiScraper())"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Or init the class from csv file"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [],
   "source": [
    "from olivia_finder.data_source.csv_network import CSVNetwork"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Scraped CRAN packages: 18195\n"
     ]
    }
   ],
   "source": [
    "cran_scraped_csv_pm = PackageManager(\n",
    "    CSVNetwork(\n",
    "        \"results/csv_datasets/cran_adjlist_scraping.csv\",   # Path to the CSV file\n",
    "        \"CRAN\",                                             # Name of the data source\n",
    "        \"CRAN as a CSV file\",                               # Description of the data source\n",
    "        dependent_field=\"name\",                             # Name of the field that contains the dependencies\n",
    "        dependency_field=\"dependency\",                      # Name of the field that contains the name of the package\n",
    "        dependent_version_field=\"version\",                  # Name of the field that contains the version of the package\n",
    "        dependency_version_field=\"dependency_version\",     # Name of the field that contains the version of the dependency\n",
    "        dependent_url_field=\"url\",                          # Name of the field that contains the URL of the package\n",
    "    )\n",
    ")\n",
    "\n",
    "print(f'Scraped CRAN packages: {len(cran_scraped_csv_pm.data_source.obtain_package_names())}')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Libraries.io CRAN packages: 15522\n"
     ]
    }
   ],
   "source": [
    "cran_librariesio_csv_pm = PackageManager(\n",
    "    CSVNetwork(\n",
    "        \"results/csv_datasets/cran_librariesio_dependencies.csv\",   # Path to the CSV file  \n",
    "        \"CRAN\",                                                     # Name of the data source\n",
    "        \"CRAN as a CSV file\",                                       # Description of the data source\n",
    "        dependent_field=\"Project Name\",                             # Name of the field that contains the dependencies\n",
    "        dependency_field=\"Dependency Name\",                         # Name of the field that contains the name of the package\n",
    "        dependent_version_field=\"Version Number\",                   # Name of the field that contains the version of the package\n",
    "        dependency_version_field=\"Dependency Requirements\"          # Name of the field that contains the version of the dependency\n",
    "    )\n",
    ")\n",
    "print(f'Libraries.io CRAN packages: {len(cran_librariesio_csv_pm.data_source.obtain_package_names())}')"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "**Get a package from package manager**"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "2023-03-20 21:26:45 [   DEBUG] Scraping package networkx (logger.py:140)\n",
      "2023-03-20 21:26:45 [   DEBUG] Getting next proxy (logger.py:140)\n",
      "2023-03-20 21:26:45 [   DEBUG] Proxy list rotated, using 41.38.124.59:3128, next will be 116.203.77.144:8080 (logger.py:140)\n",
      "2023-03-20 21:26:45 [   DEBUG] Using proxy: {'http': 'http://41.38.124.59:3128'} (logger.py:140)\n",
      "2023-03-20 21:26:45 [   DEBUG] Getting next useragent (logger.py:140)\n",
      "2023-03-20 21:26:45 [   DEBUG] Using user agent: Mozilla/5.0 (Windows NT 6.1; rv:33.0) Gecko/20100101 Firefox/33.0 (logger.py:140)\n",
      "2023-03-20 21:26:45 [   DEBUG] Starting new HTTPS connection (1): pypi.org:443 (connectionpool.py:973)\n",
      "2023-03-20 21:26:45 [   DEBUG] https://pypi.org:443 \"GET /pypi/networkx/json HTTP/1.1\" 200 35312 (connectionpool.py:452)\n",
      "2023-03-20 21:26:45 [   DEBUG] Response status code: 200 (logger.py:140)\n",
      "2023-03-20 21:26:45 [   DEBUG] Package networkx scraped successfully (logger.py:140)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Package:\n",
      "  name: networkx\n",
      "  version: 3.0\n",
      "  url: https://pypi.org/project/networkx/\n",
      "  dependencies:\n",
      "    numpy:(>=1.20)\n",
      "    scipy:(>=1.8)\n",
      "    matplotlib:(>=3.4)\n",
      "    pandas:(>=1.3)\n",
      "    pre-commit:(>=2.20)\n",
      "    mypy:(>=0.991)\n",
      "    sphinx:(==5.2.3)\n",
      "    pydata-sphinx-theme:(>=0.11)\n",
      "    sphinx-gallery:(>=0.11)\n",
      "    numpydoc:(>=1.5)\n",
      "    pillow:(>=9.2)\n",
      "    nb2plots:(>=0.6)\n",
      "    texext:(>=0.6.7)\n",
      "    lxml:(>=4.6)\n",
      "    pygraphviz:(>=1.10)\n",
      "    pydot:(>=1.4.2)\n",
      "    sympy:(>=1.10)\n",
      "    pytest:(>=7.2)\n",
      "    pytest-cov:(>=4.0)\n",
      "    codecov:(>=2.1)\n"
     ]
    }
   ],
   "source": [
    "networkx = pypi_scraper_pm.obtain_package(\"networkx\")\n",
    "networkx.print()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Package:\n",
      "  name: A3\n",
      "  version: 1.0.0\n",
      "  url: https://cran.r-project.org/package=A3\n",
      "  dependencies:\n",
      "    R: 2.15.0\n",
      "    xtable:nan\n",
      "    pbapply:nan\n",
      "\n",
      "------------------\n",
      "\n",
      "Package:\n",
      "  name: A3\n",
      "  version: 1.0.0\n",
      "  url: None\n",
      "  dependencies:\n",
      "    R:>= 2.15.0\n",
      "    randomForest:*\n"
     ]
    }
   ],
   "source": [
    "cran_scraped_csv_pm.obtain_package(\"A3\").print()\n",
    "print(\"\\n------------------\\n\")\n",
    "cran_librariesio_csv_pm.obtain_package(\"A3\").print()"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "**Get packages from a list of package names**"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Webscraping-based implementation obtains the data manager website data"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "2023-03-20 21:37:25 [   DEBUG] Using package names from param list (logger.py:140)\n",
      "2023-03-20 21:37:25 [   DEBUG] Building urls (logger.py:140)\n",
      "2023-03-20 21:37:25 [   DEBUG] Getting next proxy (logger.py:140)\n",
      "2023-03-20 21:37:25 [   DEBUG] Proxy list rotated, using 116.203.77.144:8080, next will be 93.171.224.48:4153 (logger.py:140)\n",
      "2023-03-20 21:37:25 [   DEBUG] Using proxy: {'http': 'http://116.203.77.144:8080'} (logger.py:140)\n",
      "2023-03-20 21:37:25 [   DEBUG] Getting next useragent (logger.py:140)\n",
      "2023-03-20 21:37:25 [   DEBUG] Using user agent: Mozilla/5.0 (iPad; CPU OS 6_1_3 like Mac OS X) AppleWebKit/536.26 (KHTML, like Gecko) Version/6.0 Mobile/10B329 Safari/8536.25 (logger.py:140)\n",
      "2023-03-20 21:37:25 [   DEBUG] Getting next proxy (logger.py:140)\n",
      "2023-03-20 21:37:25 [   DEBUG] Proxy list rotated, using 93.171.224.48:4153, next will be 185.78.16.76:5678 (logger.py:140)\n",
      "2023-03-20 21:37:25 [   DEBUG] Starting new HTTPS connection (1): pypi.org:443 (connectionpool.py:973)\n",
      "2023-03-20 21:37:25 [   DEBUG] Using proxy: {'http': 'http://93.171.224.48:4153'} (logger.py:140)\n",
      "2023-03-20 21:37:25 [   DEBUG] Getting next useragent (logger.py:140)\n",
      "2023-03-20 21:37:25 [   DEBUG] Using user agent: Mozilla/5.0 (Linux; U; Android 4.4.3; en-us; KFSAWI Build/KTU84M) AppleWebKit/537.36 (KHTML, like Gecko) Silk/3.68 like Chrome/39.0.2171.93 Safari/537.36 (logger.py:140)\n",
      "2023-03-20 21:37:25 [   DEBUG] Getting next proxy (logger.py:140)\n",
      "2023-03-20 21:37:25 [   DEBUG] Starting new HTTPS connection (1): pypi.org:443 (connectionpool.py:973)\n",
      "2023-03-20 21:37:25 [   DEBUG] Proxy list rotated, using 185.78.16.76:5678, next will be 122.129.107.54:5678 (logger.py:140)\n",
      "2023-03-20 21:37:25 [   DEBUG] Using proxy: {'http': 'http://185.78.16.76:5678'} (logger.py:140)\n",
      "2023-03-20 21:37:25 [   DEBUG] Getting next useragent (logger.py:140)\n",
      "2023-03-20 21:37:25 [   DEBUG] Using user agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_9_2) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/45.0.2454.85 Safari/537.36 (logger.py:140)\n",
      "2023-03-20 21:37:25 [   DEBUG] Starting new HTTPS connection (1): pypi.org:443 (connectionpool.py:973)\n",
      "2023-03-20 21:37:26 [   DEBUG] https://pypi.org:443 \"GET /pypi/networkx/json HTTP/1.1\" 200 35312 (connectionpool.py:452)\n",
      "2023-03-20 21:37:26 [   DEBUG] https://pypi.org:443 \"GET /pypi/pandas/json HTTP/1.1\" 200 232995 (connectionpool.py:452)\n",
      "2023-03-20 21:37:26 [   DEBUG] Response status code: 200 (logger.py:140)\n",
      "2023-03-20 21:37:26 [   DEBUG] https://pypi.org:443 \"GET /pypi/numpy/json HTTP/1.1\" 200 310143 (connectionpool.py:452)\n",
      "2023-03-20 21:37:26 [   DEBUG] Response status code: 200 (logger.py:140)\n",
      "2023-03-20 21:37:27 [   DEBUG] Response status code: 200 (logger.py:140)\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "[<olivia_finder.package.Package at 0x7fecf1cc40d0>,\n",
       " <olivia_finder.package.Package at 0x7fecf1cdc340>,\n",
       " <olivia_finder.package.Package at 0x7fecf1cddf00>]"
      ]
     },
     "execution_count": 19,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "packages = pypi_scraper_pm.obtain_packages([\"networkx\", \"numpy\", \"pandas\"])\n",
    "packages"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Package:\n",
      "  name: pandas\n",
      "  version: 1.5.3\n",
      "  url: https://pypi.org/project/pandas/\n",
      "  dependencies:\n",
      "    python-dateutil:(>=2.8.1)\n",
      "    pytz:(>=2020.1)\n",
      "    numpy:(>=1.23.2)\n",
      "    hypothesis:(>=5.5.3)\n",
      "    pytest:(>=6.0)\n",
      "    pytest-xdist:(>=1.31)\n"
     ]
    }
   ],
   "source": [
    "packages[2].print()"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "CSV file-based implementation obtains file data from the csv"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "------------------\n",
      "\n",
      "Package:\n",
      "  name: A3\n",
      "  version: 1.0.0\n",
      "  url: https://cran.r-project.org/package=A3\n",
      "  dependencies:\n",
      "    R: 2.15.0\n",
      "    xtable:nan\n",
      "    pbapply:nan\n",
      "\n",
      "------------------\n",
      "\n",
      "Package:\n",
      "  name: pbapply\n",
      "  version: 1.7-0\n",
      "  url: https://cran.r-project.org/package=pbapply\n",
      "  dependencies:\n",
      "    R: 3.2.0\n",
      "    parallel:nan\n",
      "\n",
      "------------------\n",
      "\n",
      "Package:\n",
      "  name: xtable\n",
      "  version: 1.8-4\n",
      "  url: https://cran.r-project.org/package=xtable\n",
      "  dependencies:\n",
      "    R: 2.10.0\n",
      "    stats:nan\n",
      "    utils:nan\n"
     ]
    }
   ],
   "source": [
    "cran_packages = cran_scraped_csv_pm.obtain_packages([\"A3\", \"pbapply\", \"xtable\"])\n",
    "for p in cran_packages:\n",
    "    print(\"\\n------------------\\n\")\n",
    "    p.print()"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "**Get all packages from a package manager**"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "***Note:***\n",
    "\n",
    "The functionality of storing packages in the PackageManager object has been implemented\n",
    "\n",
    "-   Can be activated by flag\n",
    "\n",
    "    ```python\n",
    "    extend=True\n",
    "    ```\n",
    "\n",
    "The functionality of showing the progress of obtaining packages has been implemented\n",
    "\n",
    "-   Can be activated by flag\n",
    "\n",
    "    ```python\n",
    "    show_progress=True\n",
    "    ```"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Getting all the packages from a package manager can take a while, so it is recommended to save the data to a CSV file for later use.\n",
    "\n",
    "We can see that the execution time for half a million packages (Pypi) is around 7 hours.\n",
    "In the case of Bioconductor, to obtain the 2000 packages it contains, the execution time is around 4 minutes."
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "-   From **Spraper** data source implementation"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      " 15%|        | 66423/438514 [1:14:50<6:40:39, 15.48it/s] "
     ]
    }
   ],
   "source": [
    "pypi_packages = pypi_scraper_pm.obtain_packages(extend=True, show_progress=True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "100%|| 2183/2183 [03:47<00:00,  9.59it/s]\n"
     ]
    }
   ],
   "source": [
    "bioc_scraper_pm = PackageManager(BiocScraper())\n",
    "bioconductor_packages = bioc_scraper_pm.obtain_packages(extend=True, show_progress=True)"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "-   From **CSVNetwork** data source implementation"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "100%|| 18195/18195 [02:04<00:00, 145.67it/s]\n"
     ]
    }
   ],
   "source": [
    "cran_packages = cran_scraped_csv_pm.obtain_packages(extend=True, show_progress=True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Package:\n",
      "  name: A3\n",
      "  version: 1.0.0\n",
      "  url: https://cran.r-project.org/package=A3\n",
      "  dependencies:\n",
      "    R: 2.15.0\n",
      "    xtable:nan\n",
      "    pbapply:nan\n"
     ]
    }
   ],
   "source": [
    "cran_scraped_csv_pm.obtain_package(\"A3\").print()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Package:\n",
      "  name: A3\n",
      "  version: 1.0.0\n",
      "  url: None\n",
      "  dependencies:\n",
      "    R:>= 2.15.0\n",
      "    randomForest:*\n"
     ]
    }
   ],
   "source": [
    "cran_librariesio_csv_pm.obtain_package(\"A3\").print()"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "As can be seen there is inconsistency among the different data sources, it is recommended to use the most up-to-date source"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "#### Data persistence"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "The functionality of saving the PackageManager object in disk and loading of it has also been implemented, in order to maintain persistence and not repeat processes such as WebScraping."
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "**Save the PackageManager object**\n",
    "\n",
    "We can save the object through the `save` function\n",
    "\n",
    "The file extension is irrelevant since it is a binary serialization, but by agreement the extension has been chosen **.olvpm** \"to identify the PackageManager files"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {},
   "outputs": [],
   "source": [
    "cran_scraped_csv_pm.save(\"results/package_managers/cran.olvpm\")"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "**Load the PackageManager object**\n",
    "\n",
    "We can load the PackageManager object through the static method\n",
    "\n",
    "```python \n",
    "    PackageManager.load(path:str)\n",
    "```"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Package:\n",
      "  name: A3\n",
      "  version: 1.0.0\n",
      "  url: https://cran.r-project.org/package=A3\n",
      "  dependencies:\n",
      "    R: 2.15.0\n",
      "    xtable:nan\n",
      "    pbapply:nan\n"
     ]
    }
   ],
   "source": [
    "cran_loaded_csv_pm = PackageManager.load(\"results/package_managers/cran.olvpm\")\n",
    "cran_loaded_csv_pm.obtain_package(\"A3\").print()"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "**Export the CSV format**\n",
    "\n",
    "We can export the data of the packages to a CSV, with a structure similar to that of the data of Libraries.\n",
    "\n",
    "We can use the following function to generate a Pandas Dataframe and then write the file as Dis CSV\n",
    "\n",
    "-   \n",
    "    ```python\n",
    "    pandas_df = package_manager.export_full_adjlist()\n",
    "    ```\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>name</th>\n",
       "      <th>version</th>\n",
       "      <th>url</th>\n",
       "      <th>dependency</th>\n",
       "      <th>dependency_version</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "Empty DataFrame\n",
       "Columns: [name, version, url, dependency, dependency_version]\n",
       "Index: []"
      ]
     },
     "execution_count": 22,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# Store the package manager as a adjacency list\n",
    "cran_df = cran_loaded_csv_pm.get_package_list()\n",
    "cran_df.to_csv(\"results/csv_datasets/cran_full_adjlist.csv\", index=False)\n",
    "cran_df.head()"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "gt",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.10.6"
  },
  "orig_nbformat": 4
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
